{
  "cells": [
    {
      "cell_type": "code",
      "source": [
        "from google.colab import drive\n",
        "drive.mount('/content/drive')"
      ],
      "metadata": {
        "id": "MeEijz-zOuEj",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "4a20208b-05e6-44f5-8722-d71c203be564"
      },
      "execution_count": 37,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Drive already mounted at /content/drive; to attempt to forcibly remount, call drive.mount(\"/content/drive\", force_remount=True).\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 38,
      "metadata": {
        "id": "JnjeW8fc8Oul"
      },
      "outputs": [],
      "source": [
        "import requests\n",
        "import json"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 39,
      "metadata": {
        "id": "rEp986b48Oun"
      },
      "outputs": [],
      "source": [
        "def query_pdc(query):\n",
        "\n",
        "    url = 'https://pdc.cancer.gov/graphql'\n",
        "\n",
        "    print('Sending query.')\n",
        "\n",
        "    pdc_response = requests.post(url, json={'query': query})\n",
        "\n",
        "    if pdc_response.ok:\n",
        "        return pdc_response.json()\n",
        "    else:\n",
        "        return pdc_response.raise_for_status()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 40,
      "metadata": {
        "id": "vDEQ7dKO8Ouo"
      },
      "outputs": [],
      "source": [
        "study_metadata_query = '''\n",
        "{\n",
        "  programsProjectsStudies {\n",
        "    program_id\n",
        "    program_submitter_id\n",
        "    name\n",
        "    sponsor\n",
        "    start_date\n",
        "    end_date\n",
        "    program_manager\n",
        "    projects {\n",
        "      project_id\n",
        "      project_submitter_id\n",
        "      name\n",
        "      studies {\n",
        "        study_id\n",
        "        study_submitter_id\n",
        "        submitter_id_name\n",
        "        study_name\n",
        "        program_name\n",
        "        project_name\n",
        "        program_id\n",
        "        project_id\n",
        "        project_submitter_id\n",
        "        disease_type\n",
        "        primary_site\n",
        "        analytical_fraction\n",
        "        experiment_type\n",
        "        acquisition_type\n",
        "        cases_count\n",
        "        aliquots_count\n",
        "      }\n",
        "    }\n",
        "  }\n",
        "}'''"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 41,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "w5yV9zGz8Oup",
        "outputId": "5b2cdc7c-cf96-46f9-ccb9-cd746f150951"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Sending query.\n"
          ]
        }
      ],
      "source": [
        "study_mdata = query_pdc(study_metadata_query)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 42,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "xmiSp_eH8Our",
        "outputId": "cd6d3248-aec3-4292-9d16-890b6fe1729f"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "{\n",
            "  \"data\": {\n",
            "    \"programsProjectsStudies\": [\n",
            "      {\n",
            "        \"program_id\": \"029f67dd-5cc8-11ea-bf04-0ef15c86e253\",\n",
            "        \"program_submitter_id\": \"Georgetown Proteomics Research Program\",\n",
            "        \"name\": \"Georgetown Proteomics Research Program\",\n",
            "        \"sponsor\": null,\n",
            "        \"start_date\": \"2020-03-11\",\n",
            "        \"end_date\": null,\n",
            "        \"program_manager\": \"Ratna Thangudu\",\n",
            "        \"projects\": [\n",
            "          {\n",
            "            \"project_id\": \"8ffb768a-648c-11ea-b1fd-0aad30af8a83\",\n",
            "            \"project_submitter_id\": \"Georgetown Lung Cancer Proteomics Study\",\n",
            "            \"name\": \"Georgetown Lung Cancer Proteomics Study\",\n",
            "            \"studies\": [\n",
            "              {\n",
            "                \"study_id\": \"17d5bccf-d028-11ea-b1fd-0aad30af8a83\",\n",
            "                \"study_submitter_id\": \"Georgetown Lung Cancer Proteomics Study\",\n",
            "                \"submitter_id_name\": \"Georgetown Lung Cancer Proteomics Study\",\n",
            "                \"study_name\": \"Georgetown Lung Cancer Proteomics Study\",\n",
            "                \"program_name\": null,\n",
            "                \"project_name\": null,\n",
            "                \"program_id\": null,\n",
            "                \"project_id\": null,\n",
            "                \"project_submitter_id\": null,\n",
            "                \"disease_type\": null,\n",
            "                \"primary_site\": null,\n",
            "                \"analytical_fraction\": \"Proteome\",\n",
            "                \"experiment_type\": \"iTRAQ8\",\n",
            "                \"acquisition_type\": \"DDA\",\n",
            "                \"cases_count\": 11,\n",
            "                \"aliquots_count\": 21\n",
            "              }\n",
            "            ]\n",
            "          }\n",
            "        ]\n",
            "      },\n",
            "      {\n",
            "        \"program_id\": \"10251935-5540-11e8-b664-00a098d917f8\",\n",
            "        \"program_submitter_id\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "        \"name\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "        \"sponsor\": null,\n",
            "        \"start_date\": \"2018-06-29\",\n",
            "        \"end_date\": null,\n",
            "        \"program_manager\": \"null null\",\n",
            "        \"projects\": [\n",
            "          {\n",
            "            \"project_id\": \"7231aab6-7603-499b-a48d-3e3411143e18\",\n",
            "            \"project_submitter_id\": \"Proteogenomic Translational Research Centers (PTRC)\",\n",
            "            \"name\": \"Proteogenomic Translational Research Centers (PTRC)\",\n",
            "            \"studies\": [\n",
            "              {\n",
            "                \"study_id\": \"d9b5770b-2d17-432b-8d64-621677653da8\",\n",
            "                \"study_submitter_id\": \"AML Ex Vivo Drug Response - Combination Treatment - Proteome\",\n",
            "                \"submitter_id_name\": \"AML Ex Vivo Drug Response - Combination Treatment - Proteome\",\n",
            "                \"study_name\": \"AML Ex Vivo Drug Response - Combination Treatment - Proteome\",\n",
            "                \"program_name\": null,\n",
            "                \"project_name\": null,\n",
            "                \"program_id\": null,\n",
            "                \"project_id\": null,\n",
            "                \"project_submitter_id\": null,\n",
            "                \"disease_type\": null,\n",
            "                \"primary_site\": null,\n",
            "                \"analytical_fraction\": \"Proteome\",\n",
            "                \"experiment_type\": \"TMT11\",\n",
            "                \"acquisition_type\": \"DDA\",\n",
            "                \"cases_count\": 20,\n",
            "                \"aliquots_count\": 20\n"
          ]
        }
      ],
      "source": [
        "formatted = json.dumps(study_mdata, indent=2)\n",
        "print(formatted[0:3000])"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "output_file = 'studymdata.json'\n",
        "with open(output_file, 'w') as outfile:\n",
        "    outfile.write(formatted)\n",
        "print(f\"Study metadata saved to {output_file}\")"
      ],
      "metadata": {
        "id": "kkAJljNpC_g7",
        "outputId": "e9611925-9b20-435f-de2d-879639cd9a27",
        "colab": {
          "base_uri": "https://localhost:8080/"
        }
      },
      "execution_count": 43,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Study metadata saved to studymdata.json\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "import csv\n",
        "\n",
        "def flatten_data(data):\n",
        "    flattened_data = []\n",
        "\n",
        "    programs_projects_studies = data.get('data', {}).get('programsProjectsStudies', [])\n",
        "    for program in programs_projects_studies:\n",
        "        for project in program.get('projects', []):\n",
        "            for study in project.get('studies', []):\n",
        "                # Construct a flat dictionary for each study\n",
        "                row = {\n",
        "                    'Program Name': program['name'],\n",
        "                    'Project Name': project['name'],\n",
        "                    'Study Name': study['study_name'],\n",
        "                    'Cases Count': study['cases_count'],\n",
        "                    'Disease Type': study['disease_type'],\n",
        "                    'Primary Site': study['primary_site'],\n",
        "                    'Experiment Type': study['experiment_type']\n",
        "                }\n",
        "                flattened_data.append(row)\n",
        "    return flattened_data\n"
      ],
      "metadata": {
        "id": "I_i5tm7AC_Dq"
      },
      "execution_count": 44,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "def write_to_csv(data, filename='/content/drive/MyDrive/Capstone/output.csv'):\n",
        "    if not data:\n",
        "        print(\"No data provided to write to CSV.\")\n",
        "        return\n",
        "\n",
        "    try:\n",
        "        fieldnames = data[0].keys()\n",
        "        with open(filename, 'w', newline='') as file:\n",
        "            writer = csv.DictWriter(file, fieldnames=fieldnames)\n",
        "            writer.writeheader()\n",
        "            writer.writerows(data)\n",
        "        print(f\"Data successfully saved to {filename}.\")\n",
        "    except IOError as e:\n",
        "        print(f\"An error occurred while writing to the file: {e}\")\n",
        "    except Exception as e:\n",
        "        print(f\"An unexpected error occurred: {e}\")\n"
      ],
      "metadata": {
        "id": "WB6vACCLDBPj"
      },
      "execution_count": 45,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "!pip install datasets\n",
        "from datasets import load_dataset\n",
        "\n",
        "data = load_dataset('csv', data_files='/content/drive/MyDrive/Capstone/output.csv')"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "mPpt7rTxySsC",
        "outputId": "72243d38-b495-4c68-967d-27be031e230b"
      },
      "execution_count": 46,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Requirement already satisfied: datasets in /usr/local/lib/python3.10/dist-packages (2.20.0)\n",
            "Requirement already satisfied: filelock in /usr/local/lib/python3.10/dist-packages (from datasets) (3.15.3)\n",
            "Requirement already satisfied: numpy>=1.17 in /usr/local/lib/python3.10/dist-packages (from datasets) (1.25.2)\n",
            "Requirement already satisfied: pyarrow>=15.0.0 in /usr/local/lib/python3.10/dist-packages (from datasets) (16.1.0)\n",
            "Requirement already satisfied: pyarrow-hotfix in /usr/local/lib/python3.10/dist-packages (from datasets) (0.6)\n",
            "Requirement already satisfied: dill<0.3.9,>=0.3.0 in /usr/local/lib/python3.10/dist-packages (from datasets) (0.3.8)\n",
            "Requirement already satisfied: pandas in /usr/local/lib/python3.10/dist-packages (from datasets) (2.0.3)\n",
            "Requirement already satisfied: requests>=2.32.2 in /usr/local/lib/python3.10/dist-packages (from datasets) (2.32.3)\n",
            "Requirement already satisfied: tqdm>=4.66.3 in /usr/local/lib/python3.10/dist-packages (from datasets) (4.66.4)\n",
            "Requirement already satisfied: xxhash in /usr/local/lib/python3.10/dist-packages (from datasets) (3.4.1)\n",
            "Requirement already satisfied: multiprocess in /usr/local/lib/python3.10/dist-packages (from datasets) (0.70.16)\n",
            "Requirement already satisfied: fsspec[http]<=2024.5.0,>=2023.1.0 in /usr/local/lib/python3.10/dist-packages (from datasets) (2023.6.0)\n",
            "Requirement already satisfied: aiohttp in /usr/local/lib/python3.10/dist-packages (from datasets) (3.9.5)\n",
            "Requirement already satisfied: huggingface-hub>=0.21.2 in /usr/local/lib/python3.10/dist-packages (from datasets) (0.23.4)\n",
            "Requirement already satisfied: packaging in /usr/local/lib/python3.10/dist-packages (from datasets) (24.1)\n",
            "Requirement already satisfied: pyyaml>=5.1 in /usr/local/lib/python3.10/dist-packages (from datasets) (6.0.1)\n",
            "Requirement already satisfied: aiosignal>=1.1.2 in /usr/local/lib/python3.10/dist-packages (from aiohttp->datasets) (1.3.1)\n",
            "Requirement already satisfied: attrs>=17.3.0 in /usr/local/lib/python3.10/dist-packages (from aiohttp->datasets) (23.2.0)\n",
            "Requirement already satisfied: frozenlist>=1.1.1 in /usr/local/lib/python3.10/dist-packages (from aiohttp->datasets) (1.4.1)\n",
            "Requirement already satisfied: multidict<7.0,>=4.5 in /usr/local/lib/python3.10/dist-packages (from aiohttp->datasets) (6.0.5)\n",
            "Requirement already satisfied: yarl<2.0,>=1.0 in /usr/local/lib/python3.10/dist-packages (from aiohttp->datasets) (1.9.4)\n",
            "Requirement already satisfied: async-timeout<5.0,>=4.0 in /usr/local/lib/python3.10/dist-packages (from aiohttp->datasets) (4.0.3)\n",
            "Requirement already satisfied: typing-extensions>=3.7.4.3 in /usr/local/lib/python3.10/dist-packages (from huggingface-hub>=0.21.2->datasets) (4.12.2)\n",
            "Requirement already satisfied: charset-normalizer<4,>=2 in /usr/local/lib/python3.10/dist-packages (from requests>=2.32.2->datasets) (3.3.2)\n",
            "Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.10/dist-packages (from requests>=2.32.2->datasets) (3.7)\n",
            "Requirement already satisfied: urllib3<3,>=1.21.1 in /usr/local/lib/python3.10/dist-packages (from requests>=2.32.2->datasets) (2.0.7)\n",
            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.10/dist-packages (from requests>=2.32.2->datasets) (2024.6.2)\n",
            "Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.10/dist-packages (from pandas->datasets) (2.8.2)\n",
            "Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.10/dist-packages (from pandas->datasets) (2023.4)\n",
            "Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.10/dist-packages (from pandas->datasets) (2024.1)\n",
            "Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.10/dist-packages (from python-dateutil>=2.8.2->pandas->datasets) (1.16.0)\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "pip install transformers datasets torch"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "zUBmWBUfB3o-",
        "outputId": "229f29e4-12e6-4384-d257-921e1370ad66"
      },
      "execution_count": 47,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Requirement already satisfied: transformers in /usr/local/lib/python3.10/dist-packages (4.41.2)\n",
            "Requirement already satisfied: datasets in /usr/local/lib/python3.10/dist-packages (2.20.0)\n",
            "Requirement already satisfied: torch in /usr/local/lib/python3.10/dist-packages (2.3.0+cu121)\n",
            "Requirement already satisfied: filelock in /usr/local/lib/python3.10/dist-packages (from transformers) (3.15.3)\n",
            "Requirement already satisfied: huggingface-hub<1.0,>=0.23.0 in /usr/local/lib/python3.10/dist-packages (from transformers) (0.23.4)\n",
            "Requirement already satisfied: numpy>=1.17 in /usr/local/lib/python3.10/dist-packages (from transformers) (1.25.2)\n",
            "Requirement already satisfied: packaging>=20.0 in /usr/local/lib/python3.10/dist-packages (from transformers) (24.1)\n",
            "Requirement already satisfied: pyyaml>=5.1 in /usr/local/lib/python3.10/dist-packages (from transformers) (6.0.1)\n",
            "Requirement already satisfied: regex!=2019.12.17 in /usr/local/lib/python3.10/dist-packages (from transformers) (2024.5.15)\n",
            "Requirement already satisfied: requests in /usr/local/lib/python3.10/dist-packages (from transformers) (2.32.3)\n",
            "Requirement already satisfied: tokenizers<0.20,>=0.19 in /usr/local/lib/python3.10/dist-packages (from transformers) (0.19.1)\n",
            "Requirement already satisfied: safetensors>=0.4.1 in /usr/local/lib/python3.10/dist-packages (from transformers) (0.4.3)\n",
            "Requirement already satisfied: tqdm>=4.27 in /usr/local/lib/python3.10/dist-packages (from transformers) (4.66.4)\n",
            "Requirement already satisfied: pyarrow>=15.0.0 in /usr/local/lib/python3.10/dist-packages (from datasets) (16.1.0)\n",
            "Requirement already satisfied: pyarrow-hotfix in /usr/local/lib/python3.10/dist-packages (from datasets) (0.6)\n",
            "Requirement already satisfied: dill<0.3.9,>=0.3.0 in /usr/local/lib/python3.10/dist-packages (from datasets) (0.3.8)\n",
            "Requirement already satisfied: pandas in /usr/local/lib/python3.10/dist-packages (from datasets) (2.0.3)\n",
            "Requirement already satisfied: xxhash in /usr/local/lib/python3.10/dist-packages (from datasets) (3.4.1)\n",
            "Requirement already satisfied: multiprocess in /usr/local/lib/python3.10/dist-packages (from datasets) (0.70.16)\n",
            "Requirement already satisfied: fsspec[http]<=2024.5.0,>=2023.1.0 in /usr/local/lib/python3.10/dist-packages (from datasets) (2023.6.0)\n",
            "Requirement already satisfied: aiohttp in /usr/local/lib/python3.10/dist-packages (from datasets) (3.9.5)\n",
            "Requirement already satisfied: typing-extensions>=4.8.0 in /usr/local/lib/python3.10/dist-packages (from torch) (4.12.2)\n",
            "Requirement already satisfied: sympy in /usr/local/lib/python3.10/dist-packages (from torch) (1.12.1)\n",
            "Requirement already satisfied: networkx in /usr/local/lib/python3.10/dist-packages (from torch) (3.3)\n",
            "Requirement already satisfied: jinja2 in /usr/local/lib/python3.10/dist-packages (from torch) (3.1.4)\n",
            "Requirement already satisfied: nvidia-cuda-nvrtc-cu12==12.1.105 in /usr/local/lib/python3.10/dist-packages (from torch) (12.1.105)\n",
            "Requirement already satisfied: nvidia-cuda-runtime-cu12==12.1.105 in /usr/local/lib/python3.10/dist-packages (from torch) (12.1.105)\n",
            "Requirement already satisfied: nvidia-cuda-cupti-cu12==12.1.105 in /usr/local/lib/python3.10/dist-packages (from torch) (12.1.105)\n",
            "Requirement already satisfied: nvidia-cudnn-cu12==8.9.2.26 in /usr/local/lib/python3.10/dist-packages (from torch) (8.9.2.26)\n",
            "Requirement already satisfied: nvidia-cublas-cu12==12.1.3.1 in /usr/local/lib/python3.10/dist-packages (from torch) (12.1.3.1)\n",
            "Requirement already satisfied: nvidia-cufft-cu12==11.0.2.54 in /usr/local/lib/python3.10/dist-packages (from torch) (11.0.2.54)\n",
            "Requirement already satisfied: nvidia-curand-cu12==10.3.2.106 in /usr/local/lib/python3.10/dist-packages (from torch) (10.3.2.106)\n",
            "Requirement already satisfied: nvidia-cusolver-cu12==11.4.5.107 in /usr/local/lib/python3.10/dist-packages (from torch) (11.4.5.107)\n",
            "Requirement already satisfied: nvidia-cusparse-cu12==12.1.0.106 in /usr/local/lib/python3.10/dist-packages (from torch) (12.1.0.106)\n",
            "Requirement already satisfied: nvidia-nccl-cu12==2.20.5 in /usr/local/lib/python3.10/dist-packages (from torch) (2.20.5)\n",
            "Requirement already satisfied: nvidia-nvtx-cu12==12.1.105 in /usr/local/lib/python3.10/dist-packages (from torch) (12.1.105)\n",
            "Requirement already satisfied: triton==2.3.0 in /usr/local/lib/python3.10/dist-packages (from torch) (2.3.0)\n",
            "Requirement already satisfied: nvidia-nvjitlink-cu12 in /usr/local/lib/python3.10/dist-packages (from nvidia-cusolver-cu12==11.4.5.107->torch) (12.5.40)\n",
            "Requirement already satisfied: aiosignal>=1.1.2 in /usr/local/lib/python3.10/dist-packages (from aiohttp->datasets) (1.3.1)\n",
            "Requirement already satisfied: attrs>=17.3.0 in /usr/local/lib/python3.10/dist-packages (from aiohttp->datasets) (23.2.0)\n",
            "Requirement already satisfied: frozenlist>=1.1.1 in /usr/local/lib/python3.10/dist-packages (from aiohttp->datasets) (1.4.1)\n",
            "Requirement already satisfied: multidict<7.0,>=4.5 in /usr/local/lib/python3.10/dist-packages (from aiohttp->datasets) (6.0.5)\n",
            "Requirement already satisfied: yarl<2.0,>=1.0 in /usr/local/lib/python3.10/dist-packages (from aiohttp->datasets) (1.9.4)\n",
            "Requirement already satisfied: async-timeout<5.0,>=4.0 in /usr/local/lib/python3.10/dist-packages (from aiohttp->datasets) (4.0.3)\n",
            "Requirement already satisfied: charset-normalizer<4,>=2 in /usr/local/lib/python3.10/dist-packages (from requests->transformers) (3.3.2)\n",
            "Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.10/dist-packages (from requests->transformers) (3.7)\n",
            "Requirement already satisfied: urllib3<3,>=1.21.1 in /usr/local/lib/python3.10/dist-packages (from requests->transformers) (2.0.7)\n",
            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.10/dist-packages (from requests->transformers) (2024.6.2)\n",
            "Requirement already satisfied: MarkupSafe>=2.0 in /usr/local/lib/python3.10/dist-packages (from jinja2->torch) (2.1.5)\n",
            "Requirement already satisfied: python-dateutil>=2.8.2 in /usr/local/lib/python3.10/dist-packages (from pandas->datasets) (2.8.2)\n",
            "Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.10/dist-packages (from pandas->datasets) (2023.4)\n",
            "Requirement already satisfied: tzdata>=2022.1 in /usr/local/lib/python3.10/dist-packages (from pandas->datasets) (2024.1)\n",
            "Requirement already satisfied: mpmath<1.4.0,>=1.1.0 in /usr/local/lib/python3.10/dist-packages (from sympy->torch) (1.3.0)\n",
            "Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.10/dist-packages (from python-dateutil>=2.8.2->pandas->datasets) (1.16.0)\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!pip install weaviate-client"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "w7kynfAzcig4",
        "outputId": "fb7f53fd-210a-4748-e99c-9fd2856ed0e6"
      },
      "execution_count": 48,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Requirement already satisfied: weaviate-client in /usr/local/lib/python3.10/dist-packages (4.6.5)\n",
            "Requirement already satisfied: requests<3.0.0,>=2.30.0 in /usr/local/lib/python3.10/dist-packages (from weaviate-client) (2.32.3)\n",
            "Requirement already satisfied: httpx<=0.27.0,>=0.25.0 in /usr/local/lib/python3.10/dist-packages (from weaviate-client) (0.27.0)\n",
            "Requirement already satisfied: validators==0.28.3 in /usr/local/lib/python3.10/dist-packages (from weaviate-client) (0.28.3)\n",
            "Requirement already satisfied: authlib<2.0.0,>=1.2.1 in /usr/local/lib/python3.10/dist-packages (from weaviate-client) (1.3.1)\n",
            "Requirement already satisfied: pydantic<3.0.0,>=2.5.0 in /usr/local/lib/python3.10/dist-packages (from weaviate-client) (2.7.4)\n",
            "Requirement already satisfied: grpcio<2.0.0,>=1.57.0 in /usr/local/lib/python3.10/dist-packages (from weaviate-client) (1.64.1)\n",
            "Requirement already satisfied: grpcio-tools<2.0.0,>=1.57.0 in /usr/local/lib/python3.10/dist-packages (from weaviate-client) (1.64.1)\n",
            "Requirement already satisfied: grpcio-health-checking<2.0.0,>=1.57.0 in /usr/local/lib/python3.10/dist-packages (from weaviate-client) (1.64.1)\n",
            "Requirement already satisfied: cryptography in /usr/local/lib/python3.10/dist-packages (from authlib<2.0.0,>=1.2.1->weaviate-client) (42.0.8)\n",
            "Requirement already satisfied: protobuf<6.0dev,>=5.26.1 in /usr/local/lib/python3.10/dist-packages (from grpcio-health-checking<2.0.0,>=1.57.0->weaviate-client) (5.27.2)\n",
            "Requirement already satisfied: setuptools in /usr/local/lib/python3.10/dist-packages (from grpcio-tools<2.0.0,>=1.57.0->weaviate-client) (67.7.2)\n",
            "Requirement already satisfied: anyio in /usr/local/lib/python3.10/dist-packages (from httpx<=0.27.0,>=0.25.0->weaviate-client) (3.7.1)\n",
            "Requirement already satisfied: certifi in /usr/local/lib/python3.10/dist-packages (from httpx<=0.27.0,>=0.25.0->weaviate-client) (2024.6.2)\n",
            "Requirement already satisfied: httpcore==1.* in /usr/local/lib/python3.10/dist-packages (from httpx<=0.27.0,>=0.25.0->weaviate-client) (1.0.5)\n",
            "Requirement already satisfied: idna in /usr/local/lib/python3.10/dist-packages (from httpx<=0.27.0,>=0.25.0->weaviate-client) (3.7)\n",
            "Requirement already satisfied: sniffio in /usr/local/lib/python3.10/dist-packages (from httpx<=0.27.0,>=0.25.0->weaviate-client) (1.3.1)\n",
            "Requirement already satisfied: h11<0.15,>=0.13 in /usr/local/lib/python3.10/dist-packages (from httpcore==1.*->httpx<=0.27.0,>=0.25.0->weaviate-client) (0.14.0)\n",
            "Requirement already satisfied: annotated-types>=0.4.0 in /usr/local/lib/python3.10/dist-packages (from pydantic<3.0.0,>=2.5.0->weaviate-client) (0.7.0)\n",
            "Requirement already satisfied: pydantic-core==2.18.4 in /usr/local/lib/python3.10/dist-packages (from pydantic<3.0.0,>=2.5.0->weaviate-client) (2.18.4)\n",
            "Requirement already satisfied: typing-extensions>=4.6.1 in /usr/local/lib/python3.10/dist-packages (from pydantic<3.0.0,>=2.5.0->weaviate-client) (4.12.2)\n",
            "Requirement already satisfied: charset-normalizer<4,>=2 in /usr/local/lib/python3.10/dist-packages (from requests<3.0.0,>=2.30.0->weaviate-client) (3.3.2)\n",
            "Requirement already satisfied: urllib3<3,>=1.21.1 in /usr/local/lib/python3.10/dist-packages (from requests<3.0.0,>=2.30.0->weaviate-client) (2.0.7)\n",
            "Requirement already satisfied: exceptiongroup in /usr/local/lib/python3.10/dist-packages (from anyio->httpx<=0.27.0,>=0.25.0->weaviate-client) (1.2.1)\n",
            "Requirement already satisfied: cffi>=1.12 in /usr/local/lib/python3.10/dist-packages (from cryptography->authlib<2.0.0,>=1.2.1->weaviate-client) (1.16.0)\n",
            "Requirement already satisfied: pycparser in /usr/local/lib/python3.10/dist-packages (from cffi>=1.12->cryptography->authlib<2.0.0,>=1.2.1->weaviate-client) (2.22)\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "import weaviate\n",
        "\n",
        "def connect_to_weaviate(weaviate_url, api_key):\n",
        "    client = weaviate.Client(\n",
        "    url= \"https://pp769-l99q1fyt.weaviate.network\",\n",
        "    auth_client_secret=weaviate.AuthApiKey(api_key=\"zvDbZFC0SwRBPqy94OdlmvpLR5QjuO8FogFv\")\n",
        "    )\n",
        "    return client\n",
        "\n",
        "client = connect_to_weaviate(\"https://pp769-l99q1fyt.weaviate.network\", \"zvDbZFC0SwRBPqy94OdlmvpLR5QjuO8FogFv\" )\n",
        "\n",
        "if client.is_ready():\n",
        "    print(\"Connection successful!\")\n",
        "else:\n",
        "    print(\"Connection failed. Please check your URL and API key.\")"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "5aOIW2SCDUWE",
        "outputId": "c199832e-483d-4292-b29b-a4b4ec02365d"
      },
      "execution_count": 49,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Connection successful!\n"
          ]
        }
      ]
    },
    {
      "source": [
        "def get_weaviate_version(client):\n",
        "    if client.is_ready():\n",
        "        meta_info = client.get_meta()\n",
        "        return meta_info.get('version', 'Version not found')\n",
        "    else:\n",
        "        return \"Connection failed. Please check your URL and API key.\"\n",
        "\n",
        "version = get_weaviate_version(client)\n",
        "print(\"Weaviate version:\", version)\n"
      ],
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "E0Og0zSg3_qu",
        "outputId": "fed2b563-265f-432b-b53b-1a5cb4648f9b"
      },
      "execution_count": 50,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Weaviate version: 1.25.4\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "def create_schema(client):\n",
        "    if client is None:\n",
        "        print(\"Client is not initialized.\")\n",
        "        return\n",
        "    study_schema = {\n",
        "        \"class\": \"Study\",\n",
        "        \"properties\": [\n",
        "            {\"name\": \"study_id\", \"dataType\": [\"string\"]},\n",
        "            {\"name\": \"study_submitter_id\", \"dataType\": [\"string\"]},\n",
        "            {\"name\": \"submitter_id_name\", \"dataType\": [\"string\"]},\n",
        "            {\"name\": \"study_name\", \"dataType\": [\"string\"]},\n",
        "            {\"name\": \"program_name\", \"dataType\": [\"string\"]},\n",
        "            {\"name\": \"project_name\", \"dataType\": [\"string\"]},\n",
        "            {\"name\": \"program_id\", \"dataType\": [\"string\"]},\n",
        "            {\"name\": \"project_id\", \"dataType\": [\"string\"]},\n",
        "            {\"name\": \"project_submitter_id\", \"dataType\": [\"string\"]},\n",
        "            {\"name\": \"disease_type\", \"dataType\": [\"string\"]},\n",
        "            {\"name\": \"primary_site\", \"dataType\": [\"string\"]},\n",
        "            {\"name\": \"analytical_fraction\", \"dataType\": [\"string\"]},\n",
        "            {\"name\": \"experiment_type\", \"dataType\": [\"string\"]},\n",
        "            {\"name\": \"acquisition_type\", \"dataType\": [\"string\"]},\n",
        "            {\"name\": \"cases_count\", \"dataType\": [\"int\"]},\n",
        "            {\"name\": \"aliquots_count\", \"dataType\": [\"int\"]}\n",
        "        ]\n",
        "    }\n",
        "\n",
        "    publication_schema = {\n",
        "        \"class\": \"Publication\",\n",
        "        \"properties\": [\n",
        "            {\"name\": \"publicationId\", \"dataType\": [\"string\"]},\n",
        "            {\"name\": \"pubmedId\", \"dataType\": [\"string\"]},\n",
        "            {\"name\": \"doi\", \"dataType\": [\"string\"]},\n",
        "            {\"name\": \"author\", \"dataType\": [\"string[]\"]},\n",
        "            {\"name\": \"title\", \"dataType\": [\"text\"], \"indexInverted\": True},\n",
        "            {\"name\": \"journal\", \"dataType\": [\"string\"]},\n",
        "            {\"name\": \"journalUrl\", \"dataType\": [\"string\"]},\n",
        "            {\"name\": \"year\", \"dataType\": [\"int\"]},\n",
        "            {\"name\": \"abstract\", \"dataType\": [\"text\"], \"indexInverted\": True},\n",
        "            {\"name\": \"citation\", \"dataType\": [\"string\"]},\n",
        "            {\"name\": \"diseaseTypes\", \"dataType\": [\"string[]\"]},\n",
        "            {\"name\": \"relatedStudies\", \"dataType\": [\"Study\"], \"cardinality\": \"hasMany\"}\n",
        "        ]\n",
        "    }\n",
        "    try:\n",
        "        existing_classes = client.schema.get()['classes']\n",
        "        existing_class_names = [cls['class'] for cls in existing_classes]\n",
        "\n",
        "        if \"Study\" not in existing_class_names:\n",
        "            client.schema.create_class(study_schema)\n",
        "            print(\"Study schema created successfully.\")\n",
        "        else:\n",
        "            print(\"Study class already exists.\")\n",
        "\n",
        "        if \"Publication\" not in existing_class_names:\n",
        "            client.schema.create_class(publication_schema)\n",
        "            print(\"Publication schema created successfully.\")\n",
        "        else:\n",
        "            print(\"Publication class already exists.\")\n",
        "\n",
        "    except Exception as e:\n",
        "        print(f\"Failed to check or create schema: {e}\")"
      ],
      "metadata": {
        "id": "04URt0Vu6KDr"
      },
      "execution_count": 52,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "def fetch_and_insert_publications(client, url, graphql_query):\n",
        "    headers = {\"Content-Type\": \"application/json\"}\n",
        "    response = requests.post(url, json={'query': graphql_query}, headers=headers)\n",
        "    if response.status_code == 200:\n",
        "        data = response.json()['data']['getPaginatedUIPublication']['uiPublication']\n",
        "        print(f\"Retrieved {len(data)} publications.\")\n",
        "\n",
        "        for pub in data:\n",
        "            authors = [author.strip() for author in pub.get('author', '').split(',')] if pub.get('author') else []\n",
        "            print(f\"Processing publication: {pub['title']} with {len(pub.get('studies', []))} studies\")\n",
        "            publication_object = {\n",
        "                \"publicationId\": pub['publication_id'],\n",
        "                \"pubmedId\": pub['pubmed_id'],\n",
        "                \"doi\": pub['doi'],\n",
        "                \"author\": [author.strip() for author in pub.get('author', '').split(',')] if pub.get('author') else [],\n",
        "                \"title\": pub['title'],\n",
        "                \"journal\": pub['journal'],\n",
        "                \"journalUrl\": pub['journal_url'],\n",
        "                \"year\": int(pub.get('year', 0)),\n",
        "                \"abstract\": pub.get('abstract', \"\"),\n",
        "                \"citation\": pub.get('citation', \"\"),\n",
        "                \"diseaseTypes\": pub.get('disease_types', [])\n",
        "            }\n",
        "            try:\n",
        "                client.data_object.create(publication_object, \"Publication\")\n",
        "                print(f\"Publication Abstract: {pub.get('abstract', '')}\")\n",
        "            except Exception as e:\n",
        "                print(f\"Failed to insert publication: {pub.get('abstract', '')} - {str(e)}\")\n",
        "    else:\n",
        "        print(f\"Failed to fetch publications: {response.status_code} - {response.text}\")"
      ],
      "metadata": {
        "id": "02EsbXQSUV5C"
      },
      "execution_count": 53,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "def fetch_and_insert_studies(client, url, graphql_query):\n",
        "    headers = {\"Content-Type\": \"application/json\"}\n",
        "    response = requests.post(url, json={'query': graphql_query}, headers=headers)\n",
        "    if response.status_code == 200:\n",
        "        programs = response.json()['data']['programsProjectsStudies']\n",
        "        for program in programs:\n",
        "            for project in program['projects']:\n",
        "                for study in project['studies']:\n",
        "                    study_object = {\n",
        "                        \"study_id\": study['study_id'],\n",
        "                        \"study_submitter_id\": study.get('study_submitter_id', \"\"),\n",
        "                        \"submitter_id_name\": study.get('submitter_id_name', \"\"),\n",
        "                        \"study_name\": study['study_name'],\n",
        "                        \"program_name\": study['program_name'],\n",
        "                        \"project_name\": study['project_name'],\n",
        "                        \"program_id\": study['program_id'],\n",
        "                        \"project_id\": study['project_id'],\n",
        "                        \"project_submitter_id\": study.get('project_submitter_id', \"\"),\n",
        "                        \"disease_type\": study['disease_type'],\n",
        "                        \"primary_site\": study['primary_site'],\n",
        "                        \"analytical_fraction\": study.get('analytical_fraction', \"\"),\n",
        "                        \"experiment_type\": study.get('experiment_type', \"\"),\n",
        "                        \"acquisition_type\": study.get('acquisition_type', \"\"),\n",
        "                        \"cases_count\": int(study.get('cases_count', 0)),\n",
        "                        \"aliquots_count\": int(study.get('aliquots_count', 0))\n",
        "                    }\n",
        "                    try:\n",
        "                        client.data_object.create(study_object, \"Study\")\n",
        "                        print(f\"Inserted study: {study['study_name']} linked to project {project['name']}\")\n",
        "                    except Exception as e:\n",
        "                        print(f\"Failed to insert study {study['study_name']}: {str(e)}\")\n",
        "    else:\n",
        "        print(f\"Failed to fetch studies: {response.status_code} - {response.text}\")"
      ],
      "metadata": {
        "id": "XddVpJP3Uvk1"
      },
      "execution_count": 54,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "if __name__ == \"__main__\":\n",
        "    client = connect_to_weaviate(weaviate_url, api_key)\n",
        "    weaviate_url = 'https://pp769-l99q1fyt.weaviate.network'\n",
        "    api_key = 'zvDbZFC0SwRBPqy94OdlmvpLR5QjuO8FogFv'\n",
        "\n",
        "    fetch_and_insert_publications(client=client, url=graphql_url, graphql_query=publication_query)\n",
        "    fetch_and_insert_studies(client, graphql_url, study_metadata_query)"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "dQsCA8EybZD0",
        "outputId": "a709ad14-c8c5-44aa-cad4-38a306102746"
      },
      "execution_count": 55,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Retrieved 44 publications.\n",
            "Processing publication: Integrated proteogenomic characterization of glioblastoma evolution with 2 studies\n",
            "Publication Abstract: The evolutionary trajectory of glioblastoma (GBM) is a multifaceted biological process that extends beyond\r genetic alterations alone. Here, we perform an integrative proteogenomic analysis of 123 longitudinal glioblastoma\r pairs and identify a highly proliferative cellular state at diagnosis and replacement by activation\r of neuronal transition and synaptogenic pathways in recurrent tumors. Proteomic and phosphoproteomic\r analyses reveal that the molecular transition to neuronal state at recurrence is marked by post-translational\r activation of the wingless-related integration site (WNT)/ planar cell polarity (PCP) signaling pathway and\r BRAF protein kinase. Consistently, multi-omic analysis of patient-derived xenograft (PDX) models mirror\r similar patterns of evolutionary trajectory. Inhibition of B-raf proto-oncogene (BRAF) kinase impairs both\r neuronal transition and migration capability of recurrent tumor cells, phenotypic hallmarks of post-therapy\r progression. Combinatorial treatment of temozolomide (TMZ) with BRAF inhibitor, vemurafenib, significantly\r extends the survival of PDX models. This study provides comprehensive insights into the biological mechanisms\r of glioblastoma evolution and treatment resistance, highlighting promising therapeutic strategies for\r clinical intervention.\n",
            "Processing publication: Comprehensive proteogenomic characterization of rare kidney tumors with 4 studies\n",
            "Publication Abstract: Non-clear cell renal cell carcinomas (non-ccRCCs) encompass diverse malignant and benign tumors. Refinement of differential diagnosis biomarkers, markers for early prognosis of aggressive disease, and therapeutic targets to complement immunotherapy are current clinical needs. Multi-omics analyses of 48 non-ccRCCs compared with 103 ccRCCs reveal proteogenomic, phosphorylation, glycosylation, and metabolic aberrations in RCC subtypes. RCCs with high genome instability display overexpression of IGF2BP3 and PYCR1. Integration of single-cell and bulk transcriptome data predicts diverse cell-of-origin and clarifies RCC subtype-specific proteogenomic signatures. Expression of biomarkers MAPRE3, ADGRF5, and GPNMB differentiates renal oncocytoma from chromophobe RCC, and PIGR and SOSTDC1 distinguish papillary RCC from MTSCC. This study expands our knowledge of proteogenomic signatures, biomarkers, and potential therapeutic targets in non-ccRCC.\n",
            "Processing publication: Mapping the proteogenomic landscape enables prediction of drug response in acute myeloid leukemia with 2 studies\n",
            "Publication Abstract: Acute myeloid leukemia is a poor-prognosis cancer commonly stratified by genetic aberrations, but these mutations are often heterogeneous and fail to consistently predict therapeutic response. Here, we combine transcriptomic, proteomic, and phosphoproteomic datasets with ex vivo drug sensitivity data to help understand the underlying pathophysiology of AML beyond mutations. We measure the proteome and phosphoproteome of 210 patients and combine them with genomic and transcriptomic measurements to identify four proteogenomic subtypes that complement existing genetic subtypes. We build a predictor to classify samples into subtypes and map them to a \"landscape\" that identifies specific drug response patterns. We then build a drug response prediction model to identify drugs that target distinct subtypes and validate our findings on cell lines representing various stages of quizartinib resistance. Our results show how multiomics data together with drug sensitivity data can inform therapy stratification and drug combinations in AML.\n",
            "Processing publication: Frozen tissue coring and layered histological analysis improves cell type-specific proteogenomic characterization of pancreatic adenocarcinoma with 1 studies\n",
            "Publication Abstract: Background: Omics characterization of pancreatic adenocarcinoma tissue is complicated by the highly heterogeneous and mixed populations of cells. We evaluate the feasibility and potential benefit of using a coring method to enrich specific regions from bulk tissue and then perform proteogenomic analyses.  Methods: We used the Biopsy Trifecta Extraction (BioTExt) technique to isolate cores of epithelial-enriched and stroma-enriched tissue from pancreatic tumor and adjacent tissue blocks. Histology was assessed at multiple depths throughout each core. DNA sequencing, RNA sequencing, and proteomics were performed on the cored and bulk tissue samples. Supervised and unsupervised analyses were performed based on integrated molecular and histology data.  Results: Tissue cores had mixed cell composition at varying depths throughout. Average cell type percentages assessed by histology throughout the core were better associated with KRAS variant allele frequencies than standard histology assessment of the cut surface. Clustering based on serial histology data separated the cores into three groups with enrichment of neoplastic epithelium, stroma, and acinar cells, respectively. Using this classification, tumor overexpressed proteins identified in bulk tissue analysis were assigned into epithelial- or stroma-specific categories, which revealed novel epithelial-specific tumor overexpressed proteins.  Conclusions: Our study demonstrates the feasibility of multi-omics data generation from tissue cores, the necessity of interval H&E stains in serial histology sections, and the utility of coring to improve analysis over bulk tissue data.\n",
            "Processing publication: Detection of Pancreatic Ductal Adenocarcinoma-Associated Proteins in Serum with 2 studies\n",
            "Publication Abstract: Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal cancer types, partly because it is frequently identified at an advanced stage, when surgery is no longer feasible. Therefore, early detection using minimally invasive methods such as blood tests may improve outcomes. However, studies to discover molecular signatures for the early detection of PDAC using blood tests have only been marginally successful. In the current study, a quantitative glycoproteomic approach via data-independent acquisition mass spectrometry was utilized to detect glycoproteins in 29 patient-matched PDAC tissues and sera. A total of 892 N-linked glycopeptides originating from 141 glycoproteins had PDAC-associated changes beyond normal variation. We further evaluated the specificity of these serum-detectable glycoproteins by comparing their abundance in 53 independent PDAC patient sera and 65 cancer-free controls. The PDAC tissue-associated glycoproteins we have identified represent an inventory of serum-detectable PDAC-associated glycoproteins as candidate biomarkers that can be potentially used for the detection of PDAC using blood tests.\n",
            "Processing publication: Proteogenomic insights suggest druggable pathways in endometrial carcinoma with 7 studies\n",
            "Publication Abstract: We characterized a prospective endometrial carcinoma (EC) cohort containing 138 tumors and 20 enriched normal tissues using 10 different omics platforms. Targeted quantitation of two peptides can predict antigen processing and presentation machinery activity, and may inform patient selection for immunotherapy. Association analysis between MYC activity and metformin treatment in both patients and cell lines suggests a potential role for metformin treatment in non-diabetic patients with elevated MYC activity. PIK3R1 in-frame indels are associated with elevated AKT phosphorylation and increased sensitivity to AKT inhibitors. CTNNB1 hotspot mutations are concentrated near phosphorylation sites mediating pS45-induced degradation of β-catenin, which may render Wnt-FZD antagonists ineffective. Deep learning accurately predicts EC subtypes and mutations from histopathology images, which may be useful for rapid diagnosis. Overall, this study identified molecular and imaging markers that can be further investigated to guide patient stratification for more precise treatment of EC.\n",
            "Processing publication: Proteogenomic landscape of human pancreatic ductal adenocarcinoma in an Asian population reveals tumor cell-enriched and immune-rich subtypes with 2 studies\n",
            "Publication Abstract: We report a proteogenomic analysis of pancreatic ductal adenocarcinoma (PDAC). Mutation–phosphorylation correlations identified signaling pathways associated with somatic mutations in significantly mutated genes. Messenger RNA–protein abundance correlations revealed potential prognostic biomarkers correlated with patient survival. Integrated clustering of mRNA, protein and phosphorylation data identified six PDAC subtypes. Cellular pathways represented by mRNA and protein signatures, defining the subtypes and compositions of cell types in the subtypes, characterized them as classical progenitor (TS1), squamous (TS2–4), immunogenic progenitor (IS1) and exocrine-like (IS2) subtypes. Compared with the mRNA data, protein and phosphorylation data further classified the squamous subtypes into activated stroma-enriched (TS2), invasive (TS3) and invasive-proliferative (TS4) squamous subtypes. Orthotopic mouse PDAC models revealed a higher number of pro-tumorigenic immune cells in TS4, inhibiting T cell proliferation. Our proteogenomic analysis provides significantly mutated genes/biomarkers, cellular pathways and cell types as potential therapeutic targets to improve stratification of patients with PDAC.\n",
            "Processing publication: Proteogenomic analysis of chemo-refractory high-grade serous ovarian cancer. with 6 studies\n",
            "Publication Abstract: To improve the understanding of chemo-refractory high-grade serous ovarian cancers (HGSOCs), we characterized the proteogenomic landscape of 242 (refractory and sensitive) HGSOCs, representing one discovery and two validation cohorts across two biospecimen types (formalin-fixed paraffin-embedded and frozen). We identified a 64-protein signature that predicts with high specificity a subset of HGSOCs refractory to initial platinum-based therapy and is validated in two independent patient cohorts. We detected significant association between lack of Ch17 loss of heterozygosity (LOH) and chemo-refractoriness. Based on pathway protein expression, we identified 5 clusters of HGSOC, which validated across two independent patient cohorts and patient-derived xenograft (PDX) models. These clusters may represent different mechanisms of refractoriness and implicate putative therapeutic vulnerabilities.\n",
            "Processing publication: Refining Classification of Cholangiocarcinoma Subtypes via Proteogenomic Integration Reveals New Therapeutic Prospects with 2 studies\n",
            "Publication Abstract: Background & Aims  Intrahepatic cholangiocarcinomas (iCCs) are characterized by their rarity, difficult diagnosis, and overall poor prognosis. The iCC molecular classification for developing precision medicine strategies was investigated.  Methods  Comprehensive genomic, transcriptomic, proteomic, and phosphoproteomic analyses were performed on treatment-naïve tumor samples from 102 patients with iCC who underwent surgical resection with curative intent. An organoid model was constructed for testing therapeutic potential.  Results  Three clinically supported subtypes (stem-like, poorly immunogenic, and metabolism) were identified. NCT-501 (aldehyde dehydrogenase 1 family member A1 [ALDH1A1] inhibitor) exhibited synergism with nanoparticle albumin-bound–paclitaxel in the organoid model for the stem-like subtype. The oncometabolite dysregulations were associated with different clinical outcomes in the stem-like and metabolism subtypes. The poorly immunogenic subtype harbors the non–T-cell tumor infiltration. Integrated multiomics analysis not only reproduced the 3 subtypes but also showed heterogeneity in iCC.  Conclusions  This large-scale proteogenomic analysis provides information beyond that obtained with genomic analysis, allowing the functional impact of genomic alterations to be discerned. These findings may assist in the stratification of patients with iCC and in developing rational therapeutic strategies.\n",
            "Processing publication: Integrated glycoproteomic characterization of clear cell renal cell carcinoma with 1 studies\n",
            "Publication Abstract: Clear cell renal cell carcinoma (ccRCC), a common form of RCC, is responsible for the high mortality rate of kidney cancer. Dysregulations of glycoproteins have been shown to associate with ccRCC. However, the molecular mechanism has not been well characterized. Here, a comprehensive glycoproteomic analysis is conducted using 103 tumors and 80 paired normal adjacent tissues. Altered glycosylation enzymes and corresponding protein glycosylation are observed, while two of the major ccRCC mutations, BAP1 and PBRM1, show distinct glycosylation profiles. Additionally, inter-tumor heterogeneity and cross-correlation between glycosylation and phosphorylation are observed. The relation of glycoproteomic features to genomic, transcriptomic, proteomic, and phosphoproteomic changes shows the role of glycosylation in ccRCC development with potential for therapeutic interventions. This study reports a large-scale tandem mass tag (TMT)-based quantitative glycoproteomic analysis of ccRCC that can serve as a valuable resource for the community.\n",
            "Processing publication: Histopathologic and proteogenomic heterogeneity reveals features of clear cell renal cell carcinoma aggressiveness with 7 studies\n",
            "Publication Abstract: Clear cell renal cell carcinomas (ccRCCs) represent ~75% of RCC cases and account for most RCC-associated deaths. Inter- and intratumoral heterogeneity (ITH) results in varying prognosis and treatment outcomes. To obtain the most comprehensive profile of ccRCC, we perform integrative histopathologic, proteogenomic, and metabolomic analyses on 305 ccRCC tumor segments and 166 paired adjacent normal tissues from 213 cases. Combining histologic and molecular profiles reveals ITH in 90% of ccRCCs, with 50% demonstrating immune signature heterogeneity. High tumor grade, along with BAP1 mutation, genome instability, increased hypermethylation, and a specific protein glycosylation signature define a high-risk disease subset, where UCHL1 expression displays prognostic value. Single-nuclei RNA sequencing of the adverse sarcomatoid and rhabdoid phenotypes uncover gene signatures and potential insights into tumor evolution. In vitro cell line studies confirm the potential of inhibiting identified phosphoproteome targets. This study molecularly stratifies aggressive histopathologic subtypes that may inform more effective treatment strategies.\n",
            "Processing publication: Neoplastic cell enrichment of tumor tissues using coring and laser microdissection for proteomic and genomic analyses of pancreatic ductal adenocarcinoma with 1 studies\n",
            "Publication Abstract: The identification of differentially expressed tumor-associated proteins and genomic alterations driving neoplasia is critical in the development of clinical assays to detect cancers and forms the foundation for understanding cancer biology. One of the challenges in the analysis of pancreatic ductal adenocarcinoma (PDAC) is the low neoplastic cellularity and heterogeneous composition of bulk tumors. To enrich neoplastic cells from bulk tumor tissue, coring, and laser microdissection (LMD) sampling techniques have been employed. In this study, we assessed the protein and KRAS mutation changes associated with samples obtained by these enrichment techniques and evaluated the fraction of neoplastic cells in PDAC for proteomic and genomic analyses.\n",
            "Processing publication: Proteogenomic markers of chemotherapy resistance and response in triple negative breast cancer with 3 studies\n",
            "Publication Abstract: Microscaled proteogenomics was deployed to probe the molecular basis for differential response to neoadjuvant carboplatin and docetaxel combination chemotherapy for triple-negative breast cancer (TNBC). Proteomic analyses of pre-treatment patient biopsies uniquely revealed metabolic pathways, including oxidative phosphorylation, adipogenesis and fatty acid metabolism, that were resistance-associated. Both proteomics and transcriptomics revealed sensitivity was marked by elevation of DNA repair, E2F targets, G2M checkpoint, interferon-gamma signaling and immune checkpoint components. Proteogenomic analyses of somatic copy number aberrations identified a resistance-associated 19q13.31-33 deletion where LIG1, POLD1 and XRCC1 are located. In orthogonal datasets, LIG1 (DNA ligase I) gene deletion and/or low mRNA expression levels were associated with lack of pathological complete response, higher chromosomal instability (CIN) and poor prognosis in TNBC, as well as carboplatin-selective resistance in TNBC pre-clinical models. Hemizygous loss of LIG1 was also associated with higher CIN and poor prognosis in other cancer types, demonstrating broader clinical implications.\n",
            "Processing publication: Proteomic and phosphoproteomic measurements enhance ability to predict ex vivo drug response in AML with 6 studies\n",
            "Publication Abstract: Acute Myeloid Leukemia (AML) affects 20,000 patients in the US annually with a five-year survival rate of approximately 25%. One reason for the low survival rate is the high prevalence of clonal evolution that gives rise to heterogeneous sub-populations of leukemic cells with diverse mutation spectra, which eventually leads to disease relapse. This genetic heterogeneity drives the activation of complex signaling pathways that is reflected at the protein level. This diversity makes it difficult to treat AML with targeted therapy, requiring custom patient treatment protocols tailored to each individual’s leukemia. Toward this end, the Beat AML research program prospectively collected genomic and transcriptomic data from over 1000 AML patients and carried out ex vivo drug sensitivity assays to identify genomic signatures that could predict patient-specific drug responses. However, there are inherent weaknesses in using only genetic and transcriptomic measurements as surrogates of drug response, particularly the absence of direct information about phosphorylation-mediated signal transduction. As a member of the Clinical Proteomic Tumor Analysis Consortium, we have extended the molecular characterization of this cohort by collecting proteomic and phosphoproteomic measurements from a subset of these patient samples (38 in total) to evaluate the hypothesis that proteomic signatures can improve the ability to predict response to 26 drugs in AML ex vivo samples. In this work we describe our systematic, multi-omic approach to evaluate proteomic signatures of drug response and compare protein levels to other markers of drug response such as mutational patterns. We explore the nuances of this approach using two drugs that target key pathways activated in AML: quizartinib (FLT3) and trametinib (Ras/MEK), and show how patient-derived signatures can be interpreted biologically and validated in cell lines. In conclusion, this pilot study demonstrates strong promise for proteomics-based patient stratification to assess drug sensitivity in AML.\n",
            "Processing publication: Proteogenomic analysis of lung adenocarcinoma reveals tumor heterogeneity, survival determinants, and therapeutically relevant pathways with 3 studies\n",
            "Publication Abstract: We present a deep proteogenomic profiling study of 87 lung adenocarcinoma (LUAD) tumors from the United States, integrating whole-genome sequencing, transcriptome sequencing, proteomics and phosphoproteomics by mass spectrometry, and reverse-phase protein arrays. We identify three subtypes from somatic genome signature analysis, including a transition-high subtype enriched with never smokers, a transversion-high subtype enriched with current smokers, and a structurally altered subtype enriched with former smokers, TP53 alterations, and genome-wide structural alterations. We show that within-tumor correlations of RNA and protein expression associate with tumor purity and immune cell profiles. We detect and independently validate expression signatures of RNA and protein that predict patient survival. Additionally, among co-measured genes, we found that protein expression is more often associated with patient survival than RNA. Finally, integrative analysis characterizes three expression subtypes with divergent mutations, proteomic regulatory networks, and therapeutic vulnerabilities. This proteogenomic characterization provides a foundation for molecularly informed medicine in LUAD.\n",
            "Processing publication: A proteogenomic portrait of lung squamous cell carcinoma with 4 studies\n",
            "Publication Abstract: Lung squamous cell carcinoma (LSCC) remains a leading cause of cancer death with few therapeutic options. We characterized the proteogenomic landscape of LSCC, providing a deeper exposition of LSCC biology with potential therapeutic implications. We identify NSD3 as an alternative driver in FGFR1-amplified tumors and low-p63 tumors overexpressing the therapeutic target survivin. SOX2 is considered undruggable, but our analyses provide rationale for exploring chromatin modifiers such as LSD1 and EZH2 to target SOX2-overexpressing tumors. Our data support complex regulation of metabolic pathways by crosstalk between post-translational modifications including ubiquitylation. Numerous immune-related proteogenomic observations suggest directions for further investigation. Proteogenomic dissection of CDKN2A mutations argue for more nuanced assessment of RB1 protein expression and phosphorylation before declaring CDK4/6 inhibition unsuccessful. Finally, triangulation between LSCC, LUAD, and HNSCC identified both unique and common therapeutic vulnerabilities. These observations and proteogenomics data resources may guide research into the biology and treatment of LSCC.\n",
            "Processing publication: Proteogenomic insights into the biology and treatment of HPV-negative head and neck squamous cell carcinoma with 2 studies\n",
            "Publication Abstract: We present a proteogenomic study of 108 human papilloma virus (HPV)-negative head and neck squamous cell carcinomas (HNSCCs). Proteomic analysis systematically catalogs HNSCC-associated proteins and phosphosites, prioritizes copy number drivers, and highlights an oncogenic role for RNA processing genes. Proteomic investigation of mutual exclusivity between FAT1 truncating mutations and 11q13.3 amplifications reveals dysregulated actin dynamics as a common functional consequence. Phosphoproteomics characterizes two modes of EGFR activation, suggesting a new strategy to stratify HNSCCs based on EGFR ligand abundance for effective treatment with inhibitory EGFR monoclonal antibodies. Widespread deletion of immune modulatory genes accounts for low immune infiltration in immune-cold tumors, whereas concordant upregulation of multiple immune checkpoint proteins may underlie resistance to anti-programmed cell death protein 1 monotherapy in immune-hot tumors. Multi-omic analysis identifies three molecular subtypes with high potential for treatment with CDK inhibitors, anti-EGFR antibody therapy, and immunotherapy, respectively. Altogether, proteogenomics provides a systematic framework to inform HNSCC biology and treatment.\n",
            "Processing publication: Proteogenomic and metabolomic characterization of human glioblastoma with 6 studies\n",
            "Publication Abstract: Glioblastoma (GBM) is the most aggressive nervous system cancer. Understanding its molecular pathogenesis is crucial to improving diagnosis and treatment. Integrated analysis of genomic, proteomic, post-translational modification and metabolomic data on 99 treatment-naive GBMs provides insights to GBM biology. We identify key phosphorylation events (e.g., phosphorylated PTPN11 and PLCG1) as potential switches mediating oncogenic pathway activation, as well as potential targets for EGFR-, TP53-, and RB1-altered tumors. Immune subtypes with distinct immune cell types are discovered using bulk omics methodologies, validated by snRNA-seq, and correlated with specific expression and histone acetylation patterns. Histone H2B acetylation in classical-like and immune-low GBM is driven largely by BRDs, CREBBP, and EP300. Integrated metabolomic and proteomic data identify specific lipid distributions across subtypes and distinct global metabolic changes in IDH-mutated tumors. This work highlights biological relationships that could contribute to stratification of GBM patients for more effective treatment.\n",
            "Processing publication: Proteogenomic Characterization of Pancreatic Ductal Adenocarcinoma with 4 studies\n",
            "Publication Abstract: Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive cancer with poor patient survival. Toward understanding the underlying molecular alterations that drive PDAC oncogenesis, we conducted comprehensive proteogenomic analysis of 140 pancreatic cancers, 67 normal adjacent tissues, and 9 normal pancreatic ductal tissues. Proteomic, phosphoproteomic, and glycoproteomic analyses were used to characterize proteins and their modifications. In addition, whole-genome sequencing, whole-exome sequencing, methylation, RNA sequencing (RNA-seq), and microRNA sequencing (miRNA-seq) were performed on the same tissues to facilitate an integrated proteogenomic analysis and determine the impact of genomic alterations on protein expression, signaling pathways, and post-translational modifications. To ensure robust downstream analyses, tumor neoplastic cellularity was assessed via multiple orthogonal strategies using molecular features and verified via pathological estimation of tumor cellularity based on histological review. This integrated proteogenomic characterization of PDAC will serve as a valuable resource for the community, paving the way for early detection and identification of novel therapeutic targets.\n",
            "Processing publication: The AML Microenvironment Catalyzes a Step-wise Evolution to Gilteritinib Resistance with 6 studies\n",
            "Publication Abstract: Our study details the stepwise evolution of gilteritinib resistance in FLT3-mutated acute myeloid leukemia (AML). Early resistance is mediated by the bone marrow microenvironment, which protects residual leukemia cells. Over time, leukemia cells evolve intrinsic mechanisms of resistance, or late resistance. Whole exome sequencing, CRISPR/Cas, metabolomics, proteomics, and pharmacologic approaches were used to mechanistically define both early and late resistance. Early resistant cells undergo metabolic reprogramming, grow more slowly, and are dependent upon Aurora kinase B (AURKB). Late resistant cells are characterized by expansion of pre-existing NRAS mutant subclones and continued metabolic reprogramming. Our model closely mirrors the timing and mutations of AML patients treated with gilteritinib. Pharmacological inhibition of AURKB resensitized both early resistant cell cultures and primary leukemia cells from gilteritinib-treated AML patients. These findings support a combinatorial strategy to target early resistant AML cells with AURKB inhibitors and gilteritinib before the expansion of pre-existing resistance mutations occurs.\n",
            "Processing publication: Proteogenomic Landscape of Breast Cancer Tumorigenesis and Targeted Therapy with 6 studies\n",
            "Publication Abstract: The integration of mass spectrometry-based proteomics with next-generation DNA and RNA sequencing profiles tumors more comprehensively. Herein this “proteogenomic” approach was applied to 122 treatment-naive primary breast cancers purposely accrued to preserve post-translational modifications, including protein phosphorylation and acetylation. Proteogenomics challenged standard breast cancer diagnoses, provided detailed analysis of the ERBB2 amplicon, defined tumor subsets that could benefit from immune checkpoint therapy and allowed more accurate assessment of Rb status for the prediction of CDK4/6 inhibitor responsiveness. Phosphoproteomic profiles uncovered novel associations between tumor suppressor loss and targetable kinases. Acetylproteome analysis highlighted acetylation on key nuclear proteins involved in the DNA damage response and revealed cross-talk between cytoplasmic and mitochondrial acetylation and metabolism. Our results underscore the potential of proteogenomics for the clinical investigation of breast cancer through more accurate annotation of targetable pathways and biological features of this remarkably heterogeneous malignancy.\n",
            "Processing publication: Proteogenomic Characterization of Endometrial Carcinoma with 6 studies\n",
            "Publication Abstract: We undertook a comprehensive proteogenomic characterization of 95 prospectively collected endometrial carcinomas, comprising 83 endometrioid and 12 serous tumors. This analysis revealed possible new consequences of perturbations to the p53 and Wnt/Î²-catenin pathways, identified a potential role for circRNAs in the epithelial-mesenchymal transition, and provided new information about proteomic markers of clinical and genomic tumor subgroups, including relationships to known druggable pathways. An extensive genome-wide acetylation survey yielded insights into regulatory mechanisms linking Wnt signaling and histone acetylation. We also characterized aspects of the tumor immune landscape, including immunogenic alterations, neoantigens, common cancer/testis antigens, and the immune microenvironment, all of which can inform immunotherapy decisions. Collectively, our multi-omic analyses provide a valuable resource for researchers and clinicians, identify new molecular associations of potential mechanistic significance in the development of endometrial cancers, and suggest novel approaches for identifying potential therapeutic targets.\n",
            "Processing publication: Deep Proteomics Using Two Dimensional Data Independent Acquisition Mass Spectrometry with 1 studies\n",
            "Publication Abstract: Methodologies that facilitate high-throughput proteomic analysis are a key step toward moving proteome investigations into clinical translation. Data independent acquisition (DIA) has potential as a high-throughput analytical method due to the reduced time needed for sample analysis, as well as its highly quantitative accuracy. However, a limiting feature of DIA methods is the sensitivity of detection of low abundant proteins and depth of coverage, which other mass spectrometry approaches address by two-dimensional fractionation (2D) to reduce sample complexity during data acquisition. In this study, we developed a 2D-DIA method intended for rapid- and deeper-proteome analysis compared to conventional 1D-DIA analysis. First, we characterized 96 individual fractions obtained from the protein standard, NCI-7, using a data-dependent approach (DDA), identifying a total of 151,366 unique peptides from 11,273 protein groups. We observed that the majority of the proteins can be identified from just a few selected fractions. By performing an optimization analysis, we identified six fractions with high peptide number and uniqueness that can account for 80% of the proteins identified in the entire experiment. These selected fractions were combined into a single sample which was then subjected to DIA (referred to as 2D-DIA) quantitative analysis. Furthermore, improved DIA quantification was achieved using a hybrid spectral library, obtained by combining peptides identified from DDA data with peptides identified directly from the DIA runs with the help of DIA-Umpire. The optimized 2D-DIA method allowed for improved identification and quantification of low abundant proteins compared to conventional unfractionated DIA analysis (1D-DIA). We then applied the 2D-DIA method to profile the proteomes of two breast cancer patient-derived xenograft (PDX) models, quantifying 6,217 and 6,167 unique proteins in basal- and luminal- tumors, respectively. Overall, this study demonstrates the potential of high-throughput quantitative proteomics using a novel 2D-DIA method.\n",
            "Processing publication: Proteogenomics of Non-smoking Lung Cancer in East Asia Delineates Molecular Signatures of Pathogenesis and Progression with 2 studies\n",
            "Publication Abstract: Lung cancer in East Asia is characterized by a high percentage of never-smokers, early onset and predominant EGFR mutations. To illuminate the molecular phenotype of this demographically distinct disease, we performed a deep comprehensive proteogenomic study on a prospectively collected cohort in Taiwan, representing early stage, predominantly female, non-smoking lung adenocarcinoma. Integrated genomic, proteomic, and phosphoproteomic analysis delineated the demographically distinct molecular attributes and hallmarks of tumor progression. Mutational signature analysis revealed age- and gender-related mutagenesis mechanisms, characterized by high prevalence of APOBEC mutational signature in younger females and over-representation of environmental carcinogen-like mutational signatures in older females. A proteomics-informed classification distinguished the clinical characteristics of early stage patients with EGFR mutations. Furthermore, integrated protein network analysis revealed the cellular remodeling underpinning clinical trajectories and nominated candidate biomarkers for patient stratification and therapeutic intervention. This multi-omic molecular architecture may help develop strategies for management of early stage never-smoker lung adenocarcinoma.\n",
            "Processing publication: Proteogenomic Characterization Reveals Therapeutic Vulnerabilities in Lung Adenocarcinoma with 6 studies\n",
            "Publication Abstract: To explore the biology of lung adenocarcinoma (LUAD) and identify new therapeutic opportunities, we performed comprehensive proteogenomic characterization of 110 tumors and 101 matched normal adjacent tissues (NATs) incorporating genomics, epigenomics, deep-scale proteomics, phosphoproteomics, and acetylproteomics. Multi-omics clustering revealed four subgroups defined by key driver mutations, country, and gender. Proteomic and phosphoproteomic data illuminated biology downstream of copy number aberrations, somatic mutations, and fusions and identified therapeutic vulnerabilities associated with driver events involving KRAS, EGFR, and ALK. Immune subtyping revealed a complex landscape, reinforced the association of STK11 with immune-cold behavior, and underscored a potential immunosuppressive role of neutrophil degranulation. Smoking-associated LUADs showed correlation with other environmental exposure signatures and a field effect in NATs. Matched NATs allowed identification of differentially expressed proteins with potential diagnostic and therapeutic utility. This proteogenomics dataset represents a unique public resource for researchers and clinicians seeking to better understand and treat lung adenocarcinomas.\n",
            "Processing publication: Proteogenomic Characterization of Ovarian HGSC Implicates Mitotic Kinases, Replication Stress in Observed Chromosomal Instability with 5 studies\n",
            "Publication Abstract: In the absence of a dominant driving mutation other than uniformly present TP53 mutations, deeper understanding of the biology driving ovarian high-grade serous cancer (HGSC) requires analysis at a functional level, including post-translational modifications. Comprehensive proteogenomic and phosphoproteomic characterization of 83 prospectively collected ovarian HGSC and appropriate normal precursor tissue samples (fallopian tube) under strict control of ischemia time reveals pathways that significantly differentiate between HGSC and relevant normal tissues in the context of homologous repair deficiency (HRD) status. In addition to confirming key features of HGSC from previous studies, including a potential survival-associated signature and histone acetylation as a marker of HRD, deep phosphoproteomics provides insights regarding the potential role of proliferation-induced replication stress in promoting the characteristic chromosomal instability of HGSC and suggests potential therapeutic targets for use in precision medicine trials.\n",
            "Processing publication: Microscaled proteogenomic methods for precision oncology with 6 studies\n",
            "Publication Abstract: Cancer proteogenomics promises new insights into cancer biology and treatment efficacy by integrating genomics, transcriptomics and protein profiling including modifications by mass spectrometry (MS). A critical limitation is sample input requirements that exceed many sources of clinically important material. Here we report a proteogenomics approach for core biopsies using tissue-sparing specimen processing and microscaled proteomics. As a demonstration, we analyze core needle biopsies from ERBB2 positive breast cancers before and 48-72 h after initiating neoadjuvant trastuzumab-based chemotherapy. We show greater suppression of ERBB2 protein and both ERBB2 and mTOR target phosphosite levels in cases associated with pathological complete response, and identify potential causes of treatment resistance including the absence of ERBB2 amplification, insufficient ERBB2 activity for therapeutic sensitivity despite ERBB2 amplification, and candidate resistance mechanisms including androgen receptor signaling, mucin overexpression and an inactive immune microenvironment. The clinical utility and discovery potential of proteogenomics at biopsy-scale warrants further investigation.\n",
            "Processing publication: Integrated Proteomic and Glycoproteomic Characterization of Human High-Grade Serous Ovarian Carcinoma with 3 studies\n",
            "Publication Abstract: Many gene products exhibit great structural heterogeneity because of an array of modifications. These modifications are not directly encoded in the genomic template but often affect the functionality of proteins. Protein glycosylation plays a vital role in proper protein functions. However, the analysis of glycoproteins has been challenging compared with other protein modifications, such as phosphorylation. Here, we perform an integrated proteomic and glycoproteomic analysis of 83 prospectively collected high-grade serous ovarian carcinoma (HGSC) and 23 non-tumor tissues. Integration of the expression data from global proteomics and glycoproteomics reveals tumor-specific glycosylation, uncovers different glycosylation associated with three tumor clusters, and identifies glycosylation enzymes that were correlated with the altered glycosylation. In addition to providing a valuable resource, these results provide insights into the potential roles of glycosylation in the pathogenesis of HGSC, with the possibility of distinguishing pathological outcomes of ovarian tumors from non-tumors, as well as classifying tumor clusters.\n",
            "Processing publication: Integrated Proteogenomic Characterization across Major Histological Types of Pediatric Brain Cancer with 2 studies\n",
            "Publication Abstract: We report a comprehensive proteogenomics analysis, including whole-genome sequencing, RNA sequencing, and proteomics and phosphoproteomics profiling, of 218 tumors across 7 histological types of childhood brain cancer: low-grade glioma (nÂ = 93), ependymoma (32), high-grade glioma (25), medulloblastoma (22), ganglioglioma (18), craniopharyngioma (16), and atypical teratoid rhabdoid tumor (12). Proteomics data identify common biological themes that span histological boundaries, suggesting that treatments used for one histological type may be applied effectively to other tumors sharing similar proteomics features. Immune landscape characterization reveals diverse tumor microenvironments across and within diagnoses. Proteomics data further reveal functional effects of somatic mutations and copy number variations (CNVs) not evident in transcriptomics data. Kinase-substrate association and co-expression network analysis identify important biological mechanisms of tumorigenesis. This is the first large-scale proteogenomics analysis across traditional histological boundaries to uncover foundational pediatric brain tumor biology and inform rational treatment selection.\n",
            "Processing publication: Integrated Proteogenomic Characterization of HBV-Related Hepatocellular Carcinoma with 2 studies\n",
            "Publication Abstract: We performed the first proteogenomic characterization of hepatitis B virus (HBV)-related hepatocellular carcinoma (HCC) using paired tumor and adjacent liver tissues from 159 patients. Integrated proteogenomic analyses revealed consistency and discordance among multi-omics, activation status of key signaling pathways, and liver-specific metabolic reprogramming in HBV-related HCC. Proteomic profiling identified three subgroups associated with clinical and molecular attributes including patient survival, tumor thrombus, genetic profile, and the liver-specific proteome. These proteomic subgroups have distinct features in metabolic reprogramming, microenvironment dysregulation, cell proliferation, and potential therapeutics. Two prognostic biomarkers, PYCR2 and ADH1A, related to proteomic subgrouping and involved in HCC metabolic reprogramming, were identified. CTNNB1 and TP53 mutation-associated signaling and metabolic profiles were revealed, among which mutated CTNNB1-associated ALDOA phosphorylation was validated to promote glycolysis and cell proliferation. Our study provides a valuable resource that significantly expands the knowledge of HBV-related HCC and may eventually benefit clinical practice.\n",
            "Processing publication: Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities with 3 studies\n",
            "Publication Abstract: We performed the first proteogenomic study on a prospectively collected colon cancer cohort. Comparative proteomic and phosphoproteomic analysis of paired tumor and normal adjacent tissues produced a catalog of colon cancer-associated proteins and phosphosites, including known and putative new biomarkers, drug targets, and cancer/testis antigens. Proteogenomic integration not only prioritized genomically inferred targets, such as copy-number drivers and mutation-derived neoantigens, but also yielded novel findings. Phosphoproteomics data associated Rb phosphorylation with increased proliferation and decreased apoptosis in colon cancer, which explains why this classical tumor suppressor is amplified in colon tumors and suggests a rationale for targeting Rb phosphorylation in colon cancer. Proteomics identified an association between decreased CD8 T cell infiltration and increased glycolysis in microsatellite instability-high (MSI-H) tumors, suggesting glycolysis as a potential target to overcome the resistance of MSI-H tumors to immune checkpoint blockade. Proteogenomics presents new avenues for biological discoveries and therapeutic development.\n",
            "Processing publication: Integrated Proteogenomic Characterization of Clear Cell Renal Cell Carcinoma with 3 studies\n",
            "Publication Abstract: To elucidate the deregulated functional modules that drive clear cell renal cell carcinoma (ccRCC), we performed comprehensive genomic, epigenomic, transcriptomic, proteomic, and phosphoproteomic characterization of treatment-naive ccRCC and paired normal adjacent tissue samples. Genomic analyses identified a distinct molecular subgroup associated with genomic instability. Integration of proteogenomic measurements uniquely identified protein dysregulation of cellular mechanisms impacted by genomic alterations, including oxidative phosphorylation-related metabolism, protein translation processes, and phospho-signaling modules. To assess the degree of immune infiltration in individual tumors, we identified microenvironment cell signatures that delineated four immune-based ccRCC subtypes characterized by distinct cellular pathways. This study reports a large-scale proteogenomic analysis of ccRCC to discern the functional impact of genomic alterations and provides evidence for rational treatment selection stemming from ccRCC pathobiology.\n",
            "Processing publication: Proteogenomic Characterization of Human Early-Onset Gastric Cancer with 3 studies\n",
            "Publication Abstract: We report proteogenomic analysis of diffuse gastric cancers (GCs) in young populations. Phosphoproteome data elucidated signaling pathways associated with somatic mutations based on mutation-phosphorylation correlations. Moreover, correlations between mRNA and protein abundances provided potential oncogenes and tumor suppressors associated with patient survival. Furthermore, integrated clustering of mRNA, protein, phosphorylation, and N-glycosylation data identified four subtypes of diffuse GCs. Distinguishing these subtypes was possible by proteomic data. Four subtypes were associated with proliferation, immune response, metabolism, and invasion, respectively; and associations of the subtypes with immune- and invasion-related pathways were identified mainly by phosphorylation and N-glycosylation data. Therefore, our proteogenomic analysis provides additional information beyond genomic analyses, which can improve understanding of cancer biology and patient stratification in diffuse GCs.\n",
            "Processing publication: Reproducible workflow for multiplexed deep-scale proteome and phosphoproteome analysis of tumor tissues by liquid chromatography-mass spectrometry with 6 studies\n",
            "Publication Abstract: Here we present an optimized workflow for global proteome and phosphoproteome analysis of tissues or cell lines that uses isobaric tags (TMT (tandem mass tags)-10) for multiplexed analysis and relative quantification, and provides 3√ó higher throughput than iTRAQ (isobaric tags for absolute and relative quantification)-4-based methods with high intra- and inter-laboratory reproducibility. The workflow was systematically characterized and benchmarked across three independent laboratories using two distinct breast cancer subtypes from patient-derived xenograft models to enable assessment of proteome and phosphoproteome depth and quantitative reproducibility. Each plex consisted of ten samples, each being 300 Œºg of peptide derived from <50 mg of wet-weight tissue. Of the 10,000 proteins quantified per sample, we could distinguish 7,700 human proteins derived from tumor cells and 3100 mouse proteins derived from the surrounding stroma and blood. The maximum deviation across replicates and laboratories was <7%, and the inter-laboratory correlation for TMT ratio-based comparison of the two breast cancer subtypes was r > 0.88. The maximum deviation for the phosphoproteome coverage was <24% across laboratories, with an average of >37,000 quantified phosphosites per sample and differential quantification correlations of r > 0.72. The full procedure, including sample processing and data generation, can be completed within 10 d for ten tissue samples, and 100 samples can be analyzed in ~4 months using a single LC-MS/MS instrument. The high quality, depth, and reproducibility of the data obtained both within and across laboratories should enable new biological insights to be obtained from mass spectrometry-based proteomics analyses of cells and tissues together with proteogenomic data integration.\n",
            "Processing publication: Evaluation of NCI-7 Cell Line Panel as a Reference Material for Clinical Proteomics with 3 studies\n",
            "Publication Abstract: Reference materials are vital to benchmarking the reproducibility of clinical tests and essential for monitoring laboratory performance for clinical proteomics. The reference material utilized for mass spectrometric analysis of the human proteome would ideally contain enough proteins to be suitably representative of the human proteome, as well as exhibit a stable protein composition in different batches of sample regeneration. Previously, The Clinical Proteomic Tumor Analysis Consortium (CPTAC) utilized a PDX-derived comparative reference (CompRef) materials for the longitudinal assessment of proteomic performance; however, inherent drawbacks of PDX-derived material, including extended time needed to grow tumors and high level of expertise needed, have resulted in efforts to identify a new source of CompRef material. In this study, we examined the utility of using a panel of seven cancer cell lines, NCI-7 Cell Line Panel, as a reference material for mass spectrometric analysis of human proteome. Our results showed that not only is the NCI-7 material suitable for benchmarking laboratory sample preparation methods, but also NCI-7 sample generation is highly reproducible at both the global and phosphoprotein levels. In addition, the predicted genomic and experimental coverage of the NCI-7 proteome suggests the NCI-7 material may also have applications as a universal standard proteomic reference.\n",
            "Processing publication: Mass Spectrometry-Based Proteomics Reveals Potential Roles of NEK9 and MAP2K4 in Resistance to PI3K Inhibition in Triple-Negative Breast Cancers with 2 studies\n",
            "Publication Abstract: Activation of PI3K signaling is frequently observed in triple-negative breast cancer (TNBC), yet PI3K inhibitors have shown limited clinical activity. To investigate intrinsic and adaptive mechanisms of resistance, we analyzed a panel of patient-derived xenograft models of TNBC with varying responsiveness to buparlisib, a pan-PI3K inhibitor. In a subset of patient-derived xenografts, resistance was associated with incomplete inhibition of PI3K signaling and upregulated MAPK/MEK signaling in response to buparlisib. Outlier phosphoproteome and kinome analyses identified novel candidates functionally important to buparlisib resistance, including NEK9 and MAP2K4. Knockdown of NEK9 or MAP2K4 reduced both baseline and feedback MAPK/MEK signaling and showed synthetic lethality with buparlisib in vitro. A complex in/del frameshift in PIK3CA decreased sensitivity to buparlisib via NEK9/MAP2K4√¢‚Ç¨‚Äúdependent mechanisms. In summary, our study supports a role for NEK9 and MAP2K4 in mediating buparlisib resistance and demonstrates the value of unbiased omic analyses in uncovering resistance mechanisms to targeted therapy.\n",
            "Processing publication: Proteomics, Post-translational Modifications, and Integrative Analyses Reveal Molecular Heterogeneity within Medulloblastoma Subgroups with 4 studies\n",
            "Publication Abstract: There is a pressing need to identify therapeutic targets in tumors with low mutation rates such as the malignant pediatric brain tumor medulloblastoma. To address this challenge, we quantitatively profiled global proteomes and phospho-proteomes of 45 medulloblastoma samples. Integrated analyses revealed that tumors with similar RNA expression vary extensively at the post-transcriptional and post-translational levels. We identified distinct pathways associated with two subsets of SHH tumors, and found post-translational modifications of MYC that are associated with poor outcomes in group 3 tumors. We found kinases associated with subtypes and showed that inhibiting PRKDC sensitizes MYC-driven cells to radiation. Our study shows that proteomics enables a more comprehensive, functional readout, providing a foundation for future therapeutic strategies.\n",
            "Processing publication: APOBEC3A is an oral cancer prognostic biomarker in Taiwanese carriers of an APOBEC deletion polymorphism with 1 studies\n",
            "Publication Abstract: Oral squamous cell carcinoma is a prominent cancer worldwide, particularly in Taiwan. By integrating omics analyses in 50 matched samples, we uncover in Taiwanese patients a predominant mutation signature associated with cytidine deaminase APOBEC, which correlates with the upregulation of APOBEC3A expression in the APOBEC3 gene cluster at 22q13. APOBEC3A expression is significantly higher in tumors carrying APOBEC3B-deletion allele(s). High-level APOBEC3A expression is associated with better overall survival, especially among patients carrying APOBEC3B-deletion alleles, as examined in a second cohort (n = 188; p = 0.004). The frequency of APOBEC3B-deletion alleles is ~50% in 143 genotyped oral squamous cell carcinoma -Taiwan samples (27A3B −/−:89A3B +/−:27A3B +/+), compared to the 5.8% found in 314 OSCC-TCGA samples. We thus report a frequent APOBEC mutational profile, which relates to a APOBEC3B-deletion germline polymorphism in Taiwanese oral squamous cell carcinoma that impacts expression of APOBEC3A, and is shown to be of clinical prognostic relevance. Our finding might be recapitulated by genomic studies in other cancer types.\n",
            "Processing publication: Proteogenomic integration reveals therapeutic targets in breast cancer xenografts with 2 studies\n",
            "Publication Abstract: Recent advances in mass spectrometry (MS) have enabled extensive analysis of cancer proteomes. Here, we employed quantitative proteomics to profile protein expression across 24 breast cancer patient-derived xenograft (PDX) models. Integrated proteogenomic analysis shows positive correlation between expression measurements from transcriptomic and proteomic analyses; further, gene expression-based intrinsic subtypes are largely re-capitulated using non-stromal protein markers. Proteogenomic analysis also validates a number of predicted genomic targets in multiple receptor tyrosine kinases. However, several protein/phosphoprotein events such as overexpression of AKT proteins and ARAF, BRAF, HSP90AB1 phosphosites are not readily explainable by genomic analysis, suggesting that druggable translational and/or post-translational regulatory events may be uniquely diagnosed by MS. Drug treatment experiments targeting HER2 and components of the PI3K pathway supported proteogenomic response predictions in seven xenograft models. Our study demonstrates that MS-based proteomics can identify therapeutic targets and highlights the potential of PDX drug response evaluation to annotate MS-based pathway activities.\n",
            "Processing publication: Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer with 4 studies\n",
            "Publication Abstract: To provide a detailed analysis of the molecular components and underlying mechanisms associated with ovarian cancer, we performed a comprehensive mass-spectrometry-based proteomic characterization of 174 ovarian tumors previously analyzed by The Cancer Genome Atlas (TCGA), of which 169 were high-grade serous carcinomas (HGSCs). Integrating our proteomic measurements with the genomic data yielded a number of insights into disease, such as how different copy-number alternations influence the proteome, the proteins associated with chromosomal instability, the sets of signaling pathways that diverse genome rearrangements converge on, and the ones most associated with short overall survival. Specific protein acetylations associated with homologous recombination deficiency suggest a potential means for stratifying patients for therapy. In addition to providing a valuable resource, these findings provide a view of how the somatic genome drives the cancer proteome and associations between protein and post-translational modification levels and clinical outcomes in HGSC.\n",
            "Processing publication: Proteogenomics connects somatic mutations to signalling in breast cancer with 4 studies\n",
            "Publication Abstract: Somatic mutations have been extensively characterized in breast cancer, but the effects of these genetic alterations on the proteomic landscape remain poorly understood. Here we describe quantitative mass-spectrometry-based proteomic and phosphoproteomic analyses of 105 genomically annotated breast cancers, of which 77 provided high-quality data. Integrated analyses provided insights into the somatic cancer genome including the consequences of chromosomal loss, such as the 5q deletion characteristic of basal-like breast cancer. Interrogation of the 5q trans-effects against the Library of Integrated Network-based Cellular Signatures, connected loss of CETN3 and SKP1 to elevated expression of epidermal growth factor receptor (EGFR), and SKP1 loss also to increased SRC tyrosine kinase. Global proteomic data confirmed a stromal-enriched group of proteins in addition to basal and luminal clusters, and pathway analysis of the phosphoproteome identified a G-protein-coupled receptor cluster that was not readily identified at the mRNA level. In addition to ERBB2, other amplicon-associated highly phosphorylated kinases were identified, including CDK12, PAK1, PTK2, RIPK2 and TLK2. We demonstrate that proteogenomic analysis of breast cancer elucidates the functional consequences of somatic mutations, narrows candidate nominations for driver genes within large deletions and amplified regions, and identifies therapeutic targets.\n",
            "Processing publication: Proteomic analysis of colon and rectal carcinoma using standard and customized databases with 1 studies\n",
            "Publication Abstract: Understanding proteomic differences underlying the different phenotypic classes of colon and rectal carcinoma is important and may eventually lead to a better assessment of clinical behavior of these cancers. We here present a comprehensive description of the proteomic data obtained from 90 colon and rectal carcinomas previously subjected to genomic analysis by The Cancer Genome Atlas (TCGA). Here, the primary instrument files and derived secondary data files are compiled and presented in forms that will allow further analyses of the biology of colon and rectal carcinoma. We also discuss new challenges in processing these large proteomic datasets for relevant proteins and protein variants.\n",
            "Processing publication: Rapid mass spectrometric conversion of tissue biopsy samples into permanent quantitative digital proteome maps with 1 studies\n",
            "Publication Abstract: Clinical specimens are each inherently unique, limited and nonrenewable. Small samples such as tissue biopsies are often completely consumed after a limited number of analyses. Here we present a method that enables fast and reproducible conversion of a small amount of tissue (approximating the quantity obtained by a biopsy) into a single, permanent digital file representing the mass spectrometry (MS)-measurable proteome of the sample. The method combines pressure cycling technology (PCT) and sequential window acquisition of all theoretical fragment ion spectra (SWATH)-MS. The resulting proteome maps can be analyzed, re-analyzed, compared and mined in silico to detect and quantify specific proteins across multiple samples. We used this method to process and convert 18 biopsy samples from nine patients with renal cell carcinoma into SWATH-MS fragment ion maps. From these proteome maps we detected and quantified more than 2,000 proteins with a high degree of reproducibility across all samples. The measured proteins clearly distinguished tumorous kidney tissues from healthy tissues and differentiated distinct histomorphological kidney cancer subtypes.\n",
            "Processing publication: Proteogenomic characterization of human colon and rectal cancer with 1 studies\n",
            "Publication Abstract: Extensive genomic characterization of human cancers presents the problem of inference from genomic abnormalities to cancer phenotypes. To address this problem, we analysed proteomes of colon and rectal tumours characterized previously by The Cancer Genome Atlas (TCGA) and perform integrated proteogenomic analyses. Somatic variants displayed reduced protein abundance compared to germline variants. Messenger RNA transcript abundance did not reliably predict protein abundance differences between tumours. Proteomics identified five proteomic subtypes in the TCGA cohort, two of which overlapped with the TCGA 'microsatellite instability/CpG island methylation phenotype' transcriptomic subtype, but had distinct mutation, methylation and protein expression patterns associated with different clinical outcomes. Although copy number alterations showed strong cis- and trans-effects on mRNA abundance, relatively few of these extend to the protein level. Thus, proteomics data enabled prioritization of candidate driver genes. The chromosome 20q amplicon was associated with the largest global changes at both mRNA and protein levels; proteomics data highlighted potential 20q candidates, including HNF4A (hepatocyte nuclear factor 4, alpha), TOMM34 (translocase of outer mitochondrial membrane 34) and SRC (SRC proto-oncogene, non-receptor tyrosine kinase). Integrated proteogenomic analysis provides functional context to interpret genomic abnormalities and affords a new paradigm for understanding cancer biology.\n",
            "Inserted study: Georgetown Lung Cancer Proteomics Study linked to project Georgetown Lung Cancer Proteomics Study\n",
            "Inserted study: AML Ex Vivo Drug Response - Combination Treatment - Proteome linked to project Proteogenomic Translational Research Centers (PTRC)\n",
            "Inserted study: Beat AML Baseline Clinical - Phosphoproteome linked to project Proteogenomic Translational Research Centers (PTRC)\n",
            "Inserted study: PTRC TNBC - Phosphoproteome linked to project Proteogenomic Translational Research Centers (PTRC)\n",
            "Inserted study: PTRC HGSOC FFPE Discovery - Phosphoproteome linked to project Proteogenomic Translational Research Centers (PTRC)\n",
            "Inserted study: Beat AML Baseline Clinical - Proteome linked to project Proteogenomic Translational Research Centers (PTRC)\n",
            "Inserted study: PTRC HGSOC FFPE Validation - Proteome linked to project Proteogenomic Translational Research Centers (PTRC)\n",
            "Inserted study: AML Ex Vivo Drug Response - Primary Cohort - Proteome linked to project Proteogenomic Translational Research Centers (PTRC)\n",
            "Inserted study: AML Ex Vivo Drug Response - Combination Treatment - Phosphoproteome linked to project Proteogenomic Translational Research Centers (PTRC)\n",
            "Inserted study: AML Gilteritinib TimeCourse -  Proteome linked to project Proteogenomic Translational Research Centers (PTRC)\n",
            "Inserted study: AML Gilteritinib Resistance -  Phosphoproteome linked to project Proteogenomic Translational Research Centers (PTRC)\n",
            "Inserted study: AML Quizartinib Resistance -  Proteome linked to project Proteogenomic Translational Research Centers (PTRC)\n",
            "Inserted study: PTRC TNBC PDX - Proteome linked to project Proteogenomic Translational Research Centers (PTRC)\n",
            "Inserted study: AML Gilteritinib TimeCourse -  Phosphoproteome linked to project Proteogenomic Translational Research Centers (PTRC)\n",
            "Inserted study: AML Ex Vivo Drug Response - Sorafenib Treatment - Phosphoproteome linked to project Proteogenomic Translational Research Centers (PTRC)\n",
            "Inserted study: PTRC TNBC - Proteome linked to project Proteogenomic Translational Research Centers (PTRC)\n",
            "Inserted study: PTRC HGSOC FFPE Discovery - Proteome linked to project Proteogenomic Translational Research Centers (PTRC)\n",
            "Inserted study: AML Quizartinib Resistance -  Phosphoproteome linked to project Proteogenomic Translational Research Centers (PTRC)\n",
            "Inserted study: AML Ex Vivo Drug Response - Primary Cohort - Phosphoproteome linked to project Proteogenomic Translational Research Centers (PTRC)\n",
            "Inserted study: AML Ex Vivo Drug Response - Sorafenib Treatment - Proteome linked to project Proteogenomic Translational Research Centers (PTRC)\n",
            "Inserted study: PTRC HGSOC Frozen Validation - Phosphoproteome linked to project Proteogenomic Translational Research Centers (PTRC)\n",
            "Inserted study: PTRC HGSOC Frozen Validation - Proteome linked to project Proteogenomic Translational Research Centers (PTRC)\n",
            "Inserted study: AML Gilteritinib Resistance -  Proteome linked to project Proteogenomic Translational Research Centers (PTRC)\n",
            "Inserted study: PTRC HGSOC FFPE Validation - Phosphoproteome linked to project Proteogenomic Translational Research Centers (PTRC)\n",
            "Inserted study: CPTAC GBM Discovery Study - Proteome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC CCRCC Confirmatory Study - Intratumor Heterogeneity - DIA Proteome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC LSCC Discovery Study - Acetylome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC UCEC Discovery Study - Proteome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC UCEC Discovery Study - Phosphoproteome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC CCRCC Discovery Study - Proteome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC CCRCC Discovery Study - Phosphoproteome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC non-ccRCC Study - Intact Glycoproteome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC UCEC Confirmatory Study - CompRef Proteome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: Sampling techniques for enrichment of PDAC - Proteome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC UCEC Discovery Study - CompRef Proteome  linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC UCEC Discovery Study - CompRef Phosphoproteome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC CCRCC Discovery Study - CompRef Proteome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC CCRCC Discovery Study - CompRef Phosphoproteome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC LUAD Discovery Study - CompRef Proteome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC LUAD Discovery Study - CompRef Phosphoproteome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC GBM Discovery Study - Phosphoproteome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC LUAD Discovery Study - Acetylome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC CCRCC Confirmatory Study - DIA Intact Glycoproteome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC LUAD Discovery Study - Proteome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC LUAD Discovery Study - Phosphoproteome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC LSCC Discovery Study - Proteome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC HNSCC Discovery Study - Proteome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC non-ccRCC Study - Proteome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC CCRCC Discovery Study - DIA Proteome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC PDA Discovery Study - Intact Glycoproteome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC CCRCC Confirmatory Study - DIA Proteome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC non-ccRCC Study - Metabolome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC HNSCC Discovery Study - Phosphoproteome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC CCRCC Discovery Study - Intact Glycoproteome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC UCEC Confirmatory Study - Glycoproteome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC CCRCC Confirmatory Study - Training - Metabolome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC GBM Discovery Study - Acetylome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC LSCC Discovery Study - Ubiquitylome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC non-ccRCC Study - Phosphoproteome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC UCEC Discovery Study - Acetylome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC GBM Discovery Study - CompRef Proteome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC UCEC Confirmatory Study - Acetylome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC GBM Discovery Study - CompRef Phosphoproteome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC UCEC Confirmatory Study - Phosphoproteome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC PDA Discovery Study - DIA Proteome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC PDA Discovery Study - Proteome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC GBM Discovery Study - CompRef Acetylome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC UCEC Confirmatory Study - Proteome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC CCRCC Confirmatory Study - DIA Phosphoproteome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC CCRCC Confirmatory Study - Validation - Metabolome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC LUAD Discovery Study - CompRef Acetylome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC UCEC Confirmatory Study - CompRef Acetylome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC UCEC Confirmatory Study - CompRef Phosphoproteome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC LSCC Discovery Study - Phosphoproteome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC UCEC Discovery Study - CompRef Acetylome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC PDA Discovery Study - Phosphoproteome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: CPTAC CCRCC Confirmatory Study - Kinase Inhibition - DIA Phosphoproteome linked to project CPTAC3 Discovery and Confirmatory\n",
            "Inserted study: Reproducible Proteome and Phosphoproteome Workflow PNNL - Proteome linked to project CPTAC3-Other\n",
            "Inserted study: Reproducible Proteome and Phosphoproteome Workflow BI - Phosphoproteome linked to project CPTAC3-Other\n",
            "Inserted study: Microscaled Proteogenomic Methods for Precision Oncology DP1 Clinical Trial - Proteome linked to project CPTAC3-Other\n",
            "Inserted study: Reproducible Proteome and Phosphoproteome Workflow JHU - Proteome linked to project CPTAC3-Other\n",
            "Inserted study: NCI-7 Cell Line Panel - Phosphoproteome linked to project CPTAC3-Other\n",
            "Inserted study: CPTAC Deep Proteomics 2D-DIA linked to project CPTAC3-Other\n",
            "Inserted study: Microscaled Proteogenomic Methods for Precision Oncology PDX bulk - Phosphoproteome linked to project CPTAC3-Other\n",
            "Inserted study: Microscaled Proteogenomic Methods for Precision Oncology PDX cores - Proteome linked to project CPTAC3-Other\n",
            "Inserted study: NCI-7 Cell Line Panel - Proteome linked to project CPTAC3-Other\n",
            "Inserted study: NCI-7 Cell Line Panel Experimental Application - Proteome linked to project CPTAC3-Other\n",
            "Inserted study: CPTAC PDAC Proteins in Serum - Proteome linked to project CPTAC3-Other\n",
            "Inserted study: Microscaled Proteogenomic Methods for Precision Oncology PDX cores - Phosphoproteome linked to project CPTAC3-Other\n",
            "Inserted study: Microscaled Proteogenomic Methods for Precision Oncology DP1 Clinical Trial - Phosphoproteome linked to project CPTAC3-Other\n",
            "Inserted study: Reproducible Proteome and Phosphoproteome Workflow PNNL - Phosphoproteome linked to project CPTAC3-Other\n",
            "Inserted study: Reproducible Proteome and Phosphoproteome Workflow BI - Proteome linked to project CPTAC3-Other\n",
            "Inserted study: CPTAC PDAC Proteins in Serum - Glycoproteome linked to project CPTAC3-Other\n",
            "Inserted study: Reproducible Proteome and Phosphoproteome Workflow JHU - Phosphoproteome linked to project CPTAC3-Other\n",
            "Inserted study: CPTAC PDAC BioTExt - Proteome linked to project CPTAC3-Other\n",
            "Inserted study: Microscaled Proteogenomic Methods for Precision Oncology PDX bulk - Proteome linked to project CPTAC3-Other\n",
            "Inserted study: TCGA Ovarian CompRef Samples PNNL Phosphoproteome Velos Qexactive linked to project CPTAC2 Retrospective\n",
            "Inserted study: TCGA Ovarian CompRef Samples PNNL Proteome linked to project CPTAC2 Retrospective\n",
            "Inserted study: TCGA Colorectal Cancer CompRef Samples - Proteome linked to project CPTAC2 Retrospective\n",
            "Inserted study: TCGA Ovarian CompRef Samples JHU Proteome linked to project CPTAC2 Retrospective\n",
            "Inserted study: TCGA Breast Cancer CompRef Samples - Proteome linked to project CPTAC2 Retrospective\n",
            "Inserted study: TCGA Breast Cancer CompRef Samples - Phosphoproteome linked to project CPTAC2 Retrospective\n",
            "Inserted study: TCGA Breast Cancer Proteome linked to project CPTAC2 Retrospective\n",
            "Inserted study: TCGA Breast Cancer Phosphoproteome linked to project CPTAC2 Retrospective\n",
            "Inserted study: TCGA Colon Cancer Proteome linked to project CPTAC2 Retrospective\n",
            "Inserted study: TCGA Ovarian JHU Glycoproteome linked to project CPTAC2 Retrospective\n",
            "Inserted study: TCGA Ovarian JHU Proteome linked to project CPTAC2 Retrospective\n",
            "Inserted study: TCGA Ovarian PNNL Proteome linked to project CPTAC2 Retrospective\n",
            "Inserted study: TCGA Ovarian PNNL Phosphoproteome Velos Qexactive linked to project CPTAC2 Retrospective\n",
            "Inserted study: VU Normal Colon Epithelium - Proteome linked to project CPTAC2 Retrospective\n",
            "Inserted study: Prospective Colon VU Proteome linked to project CPTAC2 Confirmatory\n",
            "Inserted study: Prospective Colon PNNL Proteome Qeplus linked to project CPTAC2 Confirmatory\n",
            "Inserted study: Prospective Colon PNNL Phosphoproteome Lumos linked to project CPTAC2 Confirmatory\n",
            "Inserted study: Prospective Ovarian PNNL Proteome Qeplus linked to project CPTAC2 Confirmatory\n",
            "Inserted study: Prospective Ovarian PNNL Phosphoproteome Lumos linked to project CPTAC2 Confirmatory\n",
            "Inserted study: Prospective Ovarian JHU Intact Glycoproteome linked to project CPTAC2 Confirmatory\n",
            "Inserted study: Prospective Breast BI Phosphoproteome linked to project CPTAC2 Confirmatory\n",
            "Inserted study: Prospective Breast BI Acetylome linked to project CPTAC2 Confirmatory\n",
            "Inserted study: Prospective Ovarian JHU Proteome linked to project CPTAC2 Confirmatory\n",
            "Inserted study: Prospective Ovarian JHU N-linked Glycosite-containing peptide linked to project CPTAC2 Confirmatory\n",
            "Inserted study: Prospective Breast BI Proteome linked to project CPTAC2 Confirmatory\n",
            "Inserted study: Prospective Breast BI - CompRef Acetylome linked to project CPTAC2 Confirmatory\n",
            "Inserted study: Prospective Breast BI - CompRef Phosphoproteome linked to project CPTAC2 Confirmatory\n",
            "Inserted study: Prospective Breast BI - CompRef Proteome linked to project CPTAC2 Confirmatory\n",
            "Inserted study: Buparlisib Treated Xenograft Tumors of TNBC - Phosphoproteome linked to project CPTAC2-Other\n",
            "Inserted study: Therapeutic Targets in Breast Cancer Xenografts -  Phosphoproteome linked to project CPTAC2-Other\n",
            "Inserted study: Buparlisib Treated Xenograft Tumors of TNBC - Proteome linked to project CPTAC2-Other\n",
            "Inserted study: Therapeutic Targets in Breast Cancer Xenografts -  Proteome linked to project CPTAC2-Other\n",
            "Inserted study: PCT SWATH Kidney linked to project Quantitative digital maps of tissue biopsies\n",
            "Inserted study: Broad Institute - Medulloblastoma - Proteome linked to project Broad\n",
            "Inserted study: Broad Institute - Medulloblastoma - Acetylome linked to project Broad\n",
            "Inserted study: Broad Institute - Medulloblastoma - Phospho-tyrosine-enrichments linked to project Broad\n",
            "Inserted study: Broad Institute - Medulloblastoma - Phosphoproteome linked to project Broad\n",
            "Inserted study: NCC iCC - Proteome linked to project National Cancer Center Korea - iCC\n",
            "Inserted study: NCC iCC - Phosphoproteome linked to project National Cancer Center Korea - iCC\n",
            "Inserted study: HBV-Related Hepatocellular Carcinoma - Proteome linked to project Integrated Proteogenomic Characterization of HBV-related Hepatocellular carcinoma\n",
            "Inserted study: HBV-Related Hepatocellular Carcinoma - Phosphoproteome linked to project Integrated Proteogenomic Characterization of HBV-related Hepatocellular carcinoma\n",
            "Inserted study: KNCC Glioblastoma Evolution - Proteome linked to project Cancer Proteogenomics  Group of National Cancer Center Korea\n",
            "Inserted study: KNCC Glioblastoma Evolution - Phosphoproteome linked to project Cancer Proteogenomics  Group of National Cancer Center Korea\n",
            "Inserted study: KU PDAC Discovery Study - Phosphoproteome linked to project Proteogenomics Analysis to Develop Precision Medicine for Treatment-Resistant Pancreatic Cancer\n",
            "Inserted study: KU PDAC Discovery Study - Global proteome linked to project Proteogenomics Analysis to Develop Precision Medicine for Treatment-Resistant Pancreatic Cancer\n",
            "Inserted study: Proteogenomics of Gastric Cancer - Proteome linked to project Human Early-Onset Gastric Cancer - Korea University\n",
            "Inserted study: Proteogenomics of Gastric Cancer - Phosphoproteome linked to project Human Early-Onset Gastric Cancer - Korea University\n",
            "Inserted study: Proteogenomics of Gastric Cancer - Glycoproteome linked to project Human Early-Onset Gastric Cancer - Korea University\n",
            "Inserted study: Oral Squamous Cell Carcinoma Study - Proteome linked to project Oral Squamous Cell Carcinoma - Chang Gung University\n",
            "Inserted study: Academia Sinica LUAD100-Proteome linked to project Academia Sinica LUAD-100\n",
            "Inserted study: Academia Sinica LUAD100-Phosphoproteome linked to project Academia Sinica LUAD-100\n",
            "Inserted study: Pediatric Brain Cancer Pilot Study - Proteome linked to project Proteogenomic Analysis of Pediatric Brain Cancer Tumors Pilot Study\n",
            "Inserted study: Pediatric Brain Cancer Pilot Study - Phosphoproteome linked to project Proteogenomic Analysis of Pediatric Brain Cancer Tumors Pilot Study\n",
            "Inserted study: APOLLO LUAD - Proteome linked to project APOLLO1\n",
            "Inserted study: APOLLO LUAD - Phosphoproteome - FeNTA linked to project APOLLO1\n",
            "Inserted study: APOLLO LUAD - Phosphoproteome - TiO2 linked to project APOLLO1\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "def query_publications_by_year(client, year):\n",
        "    gql_query = f\"\"\"\n",
        "    {{\n",
        "        Get {{\n",
        "            Publication(\n",
        "                where: {{\n",
        "                    path: [\"year\"],\n",
        "                    operator: Equal,\n",
        "                    valueInt: {year}\n",
        "                }}\n",
        "            ) {{\n",
        "                publicationId\n",
        "                title\n",
        "                year\n",
        "            }}\n",
        "        }}\n",
        "    }}\n",
        "    \"\"\"\n",
        "    return client.query.raw(gql_query)\n",
        "\n",
        "def semantic_search_publications(client, keyword):\n",
        "    gql_query = f\"\"\"\n",
        "    {{\n",
        "        Get {{\n",
        "            Publication(\n",
        "                where: {{\n",
        "                    path: [\"abstract\"],\n",
        "                    operator: Like,\n",
        "                    valueString: \"{keyword}\"\n",
        "                }}\n",
        "            ) {{\n",
        "                publicationId\n",
        "                title\n",
        "                abstract\n",
        "            }}\n",
        "        }}\n",
        "    }}\n",
        "    \"\"\"\n",
        "    return client.query.raw(gql_query)\n",
        "\n",
        "def find_similar_publications(client, abstract):\n",
        "    gql_query = f\"\"\"\n",
        "    {{\n",
        "        Get {{\n",
        "            Publication(\n",
        "                nearText: {{\n",
        "                    concepts: [\"{abstract}\"],\n",
        "                    certainty: 0.7\n",
        "                }}\n",
        "            ) {{\n",
        "                publicationId\n",
        "                title\n",
        "                abstract\n",
        "            }}\n",
        "        }}\n",
        "    }}\n",
        "    \"\"\"\n",
        "    return client.query.raw(gql_query)\n",
        "\n",
        "def main():\n",
        "    client = connect_to_weaviate(\"https://pp769-l99q1fyt.weaviate.network\", \"zvDbZFC0SwRBPqy94OdlmvpLR5QjuO8FogFv\")\n",
        "\n",
        "    if client.is_ready():\n",
        "        print(\"Connection successful!\")\n",
        "\n",
        "        print(\"\\nQuerying all publications from the year 2020...\")\n",
        "        publications_from_2020 = query_publications_by_year(client, 2020)\n",
        "        print(publications_from_2020)\n",
        "\n",
        "        print(\"\\nQuerying publications related to 'Cancer'...\")\n",
        "        cancer_related_publications = semantic_search_publications(client, \"Cancer\")\n",
        "        print(cancer_related_publications)\n",
        "\n",
        "        print(\"\\nFinding similar publications to a given abstract...\")\n",
        "        similar_publications = find_similar_publications(client, \"example abstract here\")\n",
        "        print(similar_publications)\n",
        "\n",
        "    else:\n",
        "        print(\"Connection failed. Please check your URL and API key.\")\n",
        "\n",
        "if __name__ == \"__main__\":\n",
        "    main()\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "P4IxcxsRGhtr",
        "outputId": "de9398df-86fc-4bb1-eab0-e70bc84f746b"
      },
      "execution_count": 56,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Connection successful!\n",
            "\n",
            "Querying all publications from the year 2020...\n",
            "{'data': {'Get': {'Publication': [{'publicationId': '90f114bd-2b57-11eb-93d9-0adc6d0c0251', 'title': 'Proteogenomic Landscape of Breast Cancer Tumorigenesis and Targeted Therapy', 'year': 2020}, {'publicationId': '161370ab-4f57-11ea-b5ed-0a7ef35af225', 'title': 'Proteogenomic Characterization of Endometrial Carcinoma', 'year': 2020}, {'publicationId': 'fe52a762-74ad-11ec-bde9-0a4e2186f121', 'title': 'Deep Proteomics Using Two Dimensional Data Independent Acquisition Mass Spectrometry', 'year': 2020}, {'publicationId': '245451cc-c519-11ea-9ab4-0ae52bb11cb7', 'title': 'Proteogenomics of Non-smoking Lung Cancer in East Asia Delineates Molecular Signatures of Pathogenesis and Progression', 'year': 2020}, {'publicationId': '24548009-c519-11ea-9ab4-0ae52bb11cb7', 'title': 'Proteogenomic Characterization Reveals Therapeutic Vulnerabilities in Lung Adenocarcinoma', 'year': 2020}, {'publicationId': '3c25e6ab-1507-11eb-8dc2-0a3dadff3d13', 'title': 'Proteogenomic Characterization of Ovarian HGSC Implicates Mitotic Kinases, Replication Stress in Observed Chromosomal Instability', 'year': 2020}, {'publicationId': 'ccd9fc46-d912-11eb-9f63-0a2b20c3a9e9', 'title': 'Microscaled proteogenomic methods for precision oncology', 'year': 2020}, {'publicationId': '3c25ea7d-1507-11eb-8dc2-0a3dadff3d13', 'title': 'Integrated Proteomic and Glycoproteomic Characterization of Human High-Grade Serous Ovarian Carcinoma', 'year': 2020}, {'publicationId': '90c53958-3333-11eb-ab70-0abc48da233b', 'title': 'Integrated Proteogenomic Characterization across Major Histological Types of Pediatric Brain Cancer', 'year': 2020}, {'publicationId': '90f114bd-2b57-11eb-93d9-0adc6d0c0251', 'title': 'Proteogenomic Landscape of Breast Cancer Tumorigenesis and Targeted Therapy', 'year': 2020}, {'publicationId': '161370ab-4f57-11ea-b5ed-0a7ef35af225', 'title': 'Proteogenomic Characterization of Endometrial Carcinoma', 'year': 2020}, {'publicationId': 'fe52a762-74ad-11ec-bde9-0a4e2186f121', 'title': 'Deep Proteomics Using Two Dimensional Data Independent Acquisition Mass Spectrometry', 'year': 2020}, {'publicationId': '245451cc-c519-11ea-9ab4-0ae52bb11cb7', 'title': 'Proteogenomics of Non-smoking Lung Cancer in East Asia Delineates Molecular Signatures of Pathogenesis and Progression', 'year': 2020}, {'publicationId': '24548009-c519-11ea-9ab4-0ae52bb11cb7', 'title': 'Proteogenomic Characterization Reveals Therapeutic Vulnerabilities in Lung Adenocarcinoma', 'year': 2020}, {'publicationId': '3c25e6ab-1507-11eb-8dc2-0a3dadff3d13', 'title': 'Proteogenomic Characterization of Ovarian HGSC Implicates Mitotic Kinases, Replication Stress in Observed Chromosomal Instability', 'year': 2020}, {'publicationId': 'ccd9fc46-d912-11eb-9f63-0a2b20c3a9e9', 'title': 'Microscaled proteogenomic methods for precision oncology', 'year': 2020}, {'publicationId': '3c25ea7d-1507-11eb-8dc2-0a3dadff3d13', 'title': 'Integrated Proteomic and Glycoproteomic Characterization of Human High-Grade Serous Ovarian Carcinoma', 'year': 2020}, {'publicationId': '90c53958-3333-11eb-ab70-0abc48da233b', 'title': 'Integrated Proteogenomic Characterization across Major Histological Types of Pediatric Brain Cancer', 'year': 2020}, {'publicationId': '90f114bd-2b57-11eb-93d9-0adc6d0c0251', 'title': 'Proteogenomic Landscape of Breast Cancer Tumorigenesis and Targeted Therapy', 'year': 2020}, {'publicationId': '161370ab-4f57-11ea-b5ed-0a7ef35af225', 'title': 'Proteogenomic Characterization of Endometrial Carcinoma', 'year': 2020}, {'publicationId': 'fe52a762-74ad-11ec-bde9-0a4e2186f121', 'title': 'Deep Proteomics Using Two Dimensional Data Independent Acquisition Mass Spectrometry', 'year': 2020}, {'publicationId': '245451cc-c519-11ea-9ab4-0ae52bb11cb7', 'title': 'Proteogenomics of Non-smoking Lung Cancer in East Asia Delineates Molecular Signatures of Pathogenesis and Progression', 'year': 2020}, {'publicationId': '24548009-c519-11ea-9ab4-0ae52bb11cb7', 'title': 'Proteogenomic Characterization Reveals Therapeutic Vulnerabilities in Lung Adenocarcinoma', 'year': 2020}, {'publicationId': '3c25e6ab-1507-11eb-8dc2-0a3dadff3d13', 'title': 'Proteogenomic Characterization of Ovarian HGSC Implicates Mitotic Kinases, Replication Stress in Observed Chromosomal Instability', 'year': 2020}, {'publicationId': 'ccd9fc46-d912-11eb-9f63-0a2b20c3a9e9', 'title': 'Microscaled proteogenomic methods for precision oncology', 'year': 2020}, {'publicationId': '3c25ea7d-1507-11eb-8dc2-0a3dadff3d13', 'title': 'Integrated Proteomic and Glycoproteomic Characterization of Human High-Grade Serous Ovarian Carcinoma', 'year': 2020}, {'publicationId': '90c53958-3333-11eb-ab70-0abc48da233b', 'title': 'Integrated Proteogenomic Characterization across Major Histological Types of Pediatric Brain Cancer', 'year': 2020}, {'publicationId': '90f114bd-2b57-11eb-93d9-0adc6d0c0251', 'title': 'Proteogenomic Landscape of Breast Cancer Tumorigenesis and Targeted Therapy', 'year': 2020}, {'publicationId': '161370ab-4f57-11ea-b5ed-0a7ef35af225', 'title': 'Proteogenomic Characterization of Endometrial Carcinoma', 'year': 2020}, {'publicationId': 'fe52a762-74ad-11ec-bde9-0a4e2186f121', 'title': 'Deep Proteomics Using Two Dimensional Data Independent Acquisition Mass Spectrometry', 'year': 2020}, {'publicationId': '245451cc-c519-11ea-9ab4-0ae52bb11cb7', 'title': 'Proteogenomics of Non-smoking Lung Cancer in East Asia Delineates Molecular Signatures of Pathogenesis and Progression', 'year': 2020}, {'publicationId': '24548009-c519-11ea-9ab4-0ae52bb11cb7', 'title': 'Proteogenomic Characterization Reveals Therapeutic Vulnerabilities in Lung Adenocarcinoma', 'year': 2020}, {'publicationId': '3c25e6ab-1507-11eb-8dc2-0a3dadff3d13', 'title': 'Proteogenomic Characterization of Ovarian HGSC Implicates Mitotic Kinases, Replication Stress in Observed Chromosomal Instability', 'year': 2020}, {'publicationId': 'ccd9fc46-d912-11eb-9f63-0a2b20c3a9e9', 'title': 'Microscaled proteogenomic methods for precision oncology', 'year': 2020}, {'publicationId': '3c25ea7d-1507-11eb-8dc2-0a3dadff3d13', 'title': 'Integrated Proteomic and Glycoproteomic Characterization of Human High-Grade Serous Ovarian Carcinoma', 'year': 2020}, {'publicationId': '90c53958-3333-11eb-ab70-0abc48da233b', 'title': 'Integrated Proteogenomic Characterization across Major Histological Types of Pediatric Brain Cancer', 'year': 2020}, {'publicationId': '90f114bd-2b57-11eb-93d9-0adc6d0c0251', 'title': 'Proteogenomic Landscape of Breast Cancer Tumorigenesis and Targeted Therapy', 'year': 2020}, {'publicationId': '161370ab-4f57-11ea-b5ed-0a7ef35af225', 'title': 'Proteogenomic Characterization of Endometrial Carcinoma', 'year': 2020}, {'publicationId': 'fe52a762-74ad-11ec-bde9-0a4e2186f121', 'title': 'Deep Proteomics Using Two Dimensional Data Independent Acquisition Mass Spectrometry', 'year': 2020}, {'publicationId': '245451cc-c519-11ea-9ab4-0ae52bb11cb7', 'title': 'Proteogenomics of Non-smoking Lung Cancer in East Asia Delineates Molecular Signatures of Pathogenesis and Progression', 'year': 2020}, {'publicationId': '24548009-c519-11ea-9ab4-0ae52bb11cb7', 'title': 'Proteogenomic Characterization Reveals Therapeutic Vulnerabilities in Lung Adenocarcinoma', 'year': 2020}, {'publicationId': '3c25e6ab-1507-11eb-8dc2-0a3dadff3d13', 'title': 'Proteogenomic Characterization of Ovarian HGSC Implicates Mitotic Kinases, Replication Stress in Observed Chromosomal Instability', 'year': 2020}, {'publicationId': 'ccd9fc46-d912-11eb-9f63-0a2b20c3a9e9', 'title': 'Microscaled proteogenomic methods for precision oncology', 'year': 2020}, {'publicationId': '3c25ea7d-1507-11eb-8dc2-0a3dadff3d13', 'title': 'Integrated Proteomic and Glycoproteomic Characterization of Human High-Grade Serous Ovarian Carcinoma', 'year': 2020}, {'publicationId': '90c53958-3333-11eb-ab70-0abc48da233b', 'title': 'Integrated Proteogenomic Characterization across Major Histological Types of Pediatric Brain Cancer', 'year': 2020}, {'publicationId': '90f114bd-2b57-11eb-93d9-0adc6d0c0251', 'title': 'Proteogenomic Landscape of Breast Cancer Tumorigenesis and Targeted Therapy', 'year': 2020}, {'publicationId': '161370ab-4f57-11ea-b5ed-0a7ef35af225', 'title': 'Proteogenomic Characterization of Endometrial Carcinoma', 'year': 2020}, {'publicationId': 'fe52a762-74ad-11ec-bde9-0a4e2186f121', 'title': 'Deep Proteomics Using Two Dimensional Data Independent Acquisition Mass Spectrometry', 'year': 2020}, {'publicationId': '245451cc-c519-11ea-9ab4-0ae52bb11cb7', 'title': 'Proteogenomics of Non-smoking Lung Cancer in East Asia Delineates Molecular Signatures of Pathogenesis and Progression', 'year': 2020}, {'publicationId': '24548009-c519-11ea-9ab4-0ae52bb11cb7', 'title': 'Proteogenomic Characterization Reveals Therapeutic Vulnerabilities in Lung Adenocarcinoma', 'year': 2020}, {'publicationId': '3c25e6ab-1507-11eb-8dc2-0a3dadff3d13', 'title': 'Proteogenomic Characterization of Ovarian HGSC Implicates Mitotic Kinases, Replication Stress in Observed Chromosomal Instability', 'year': 2020}, {'publicationId': 'ccd9fc46-d912-11eb-9f63-0a2b20c3a9e9', 'title': 'Microscaled proteogenomic methods for precision oncology', 'year': 2020}, {'publicationId': '3c25ea7d-1507-11eb-8dc2-0a3dadff3d13', 'title': 'Integrated Proteomic and Glycoproteomic Characterization of Human High-Grade Serous Ovarian Carcinoma', 'year': 2020}, {'publicationId': '90c53958-3333-11eb-ab70-0abc48da233b', 'title': 'Integrated Proteogenomic Characterization across Major Histological Types of Pediatric Brain Cancer', 'year': 2020}, {'publicationId': '90f114bd-2b57-11eb-93d9-0adc6d0c0251', 'title': 'Proteogenomic Landscape of Breast Cancer Tumorigenesis and Targeted Therapy', 'year': 2020}, {'publicationId': '161370ab-4f57-11ea-b5ed-0a7ef35af225', 'title': 'Proteogenomic Characterization of Endometrial Carcinoma', 'year': 2020}, {'publicationId': 'fe52a762-74ad-11ec-bde9-0a4e2186f121', 'title': 'Deep Proteomics Using Two Dimensional Data Independent Acquisition Mass Spectrometry', 'year': 2020}, {'publicationId': '245451cc-c519-11ea-9ab4-0ae52bb11cb7', 'title': 'Proteogenomics of Non-smoking Lung Cancer in East Asia Delineates Molecular Signatures of Pathogenesis and Progression', 'year': 2020}, {'publicationId': '24548009-c519-11ea-9ab4-0ae52bb11cb7', 'title': 'Proteogenomic Characterization Reveals Therapeutic Vulnerabilities in Lung Adenocarcinoma', 'year': 2020}, {'publicationId': '3c25e6ab-1507-11eb-8dc2-0a3dadff3d13', 'title': 'Proteogenomic Characterization of Ovarian HGSC Implicates Mitotic Kinases, Replication Stress in Observed Chromosomal Instability', 'year': 2020}, {'publicationId': 'ccd9fc46-d912-11eb-9f63-0a2b20c3a9e9', 'title': 'Microscaled proteogenomic methods for precision oncology', 'year': 2020}, {'publicationId': '3c25ea7d-1507-11eb-8dc2-0a3dadff3d13', 'title': 'Integrated Proteomic and Glycoproteomic Characterization of Human High-Grade Serous Ovarian Carcinoma', 'year': 2020}, {'publicationId': '90c53958-3333-11eb-ab70-0abc48da233b', 'title': 'Integrated Proteogenomic Characterization across Major Histological Types of Pediatric Brain Cancer', 'year': 2020}, {'publicationId': '90f114bd-2b57-11eb-93d9-0adc6d0c0251', 'title': 'Proteogenomic Landscape of Breast Cancer Tumorigenesis and Targeted Therapy', 'year': 2020}, {'publicationId': '161370ab-4f57-11ea-b5ed-0a7ef35af225', 'title': 'Proteogenomic Characterization of Endometrial Carcinoma', 'year': 2020}, {'publicationId': 'fe52a762-74ad-11ec-bde9-0a4e2186f121', 'title': 'Deep Proteomics Using Two Dimensional Data Independent Acquisition Mass Spectrometry', 'year': 2020}, {'publicationId': '245451cc-c519-11ea-9ab4-0ae52bb11cb7', 'title': 'Proteogenomics of Non-smoking Lung Cancer in East Asia Delineates Molecular Signatures of Pathogenesis and Progression', 'year': 2020}, {'publicationId': '24548009-c519-11ea-9ab4-0ae52bb11cb7', 'title': 'Proteogenomic Characterization Reveals Therapeutic Vulnerabilities in Lung Adenocarcinoma', 'year': 2020}, {'publicationId': '3c25e6ab-1507-11eb-8dc2-0a3dadff3d13', 'title': 'Proteogenomic Characterization of Ovarian HGSC Implicates Mitotic Kinases, Replication Stress in Observed Chromosomal Instability', 'year': 2020}, {'publicationId': 'ccd9fc46-d912-11eb-9f63-0a2b20c3a9e9', 'title': 'Microscaled proteogenomic methods for precision oncology', 'year': 2020}, {'publicationId': '3c25ea7d-1507-11eb-8dc2-0a3dadff3d13', 'title': 'Integrated Proteomic and Glycoproteomic Characterization of Human High-Grade Serous Ovarian Carcinoma', 'year': 2020}, {'publicationId': '90c53958-3333-11eb-ab70-0abc48da233b', 'title': 'Integrated Proteogenomic Characterization across Major Histological Types of Pediatric Brain Cancer', 'year': 2020}, {'publicationId': '90f114bd-2b57-11eb-93d9-0adc6d0c0251', 'title': 'Proteogenomic Landscape of Breast Cancer Tumorigenesis and Targeted Therapy', 'year': 2020}, {'publicationId': '161370ab-4f57-11ea-b5ed-0a7ef35af225', 'title': 'Proteogenomic Characterization of Endometrial Carcinoma', 'year': 2020}, {'publicationId': 'fe52a762-74ad-11ec-bde9-0a4e2186f121', 'title': 'Deep Proteomics Using Two Dimensional Data Independent Acquisition Mass Spectrometry', 'year': 2020}, {'publicationId': '245451cc-c519-11ea-9ab4-0ae52bb11cb7', 'title': 'Proteogenomics of Non-smoking Lung Cancer in East Asia Delineates Molecular Signatures of Pathogenesis and Progression', 'year': 2020}, {'publicationId': '24548009-c519-11ea-9ab4-0ae52bb11cb7', 'title': 'Proteogenomic Characterization Reveals Therapeutic Vulnerabilities in Lung Adenocarcinoma', 'year': 2020}, {'publicationId': '3c25e6ab-1507-11eb-8dc2-0a3dadff3d13', 'title': 'Proteogenomic Characterization of Ovarian HGSC Implicates Mitotic Kinases, Replication Stress in Observed Chromosomal Instability', 'year': 2020}, {'publicationId': 'ccd9fc46-d912-11eb-9f63-0a2b20c3a9e9', 'title': 'Microscaled proteogenomic methods for precision oncology', 'year': 2020}, {'publicationId': '3c25ea7d-1507-11eb-8dc2-0a3dadff3d13', 'title': 'Integrated Proteomic and Glycoproteomic Characterization of Human High-Grade Serous Ovarian Carcinoma', 'year': 2020}, {'publicationId': '90c53958-3333-11eb-ab70-0abc48da233b', 'title': 'Integrated Proteogenomic Characterization across Major Histological Types of Pediatric Brain Cancer', 'year': 2020}, {'publicationId': '90f114bd-2b57-11eb-93d9-0adc6d0c0251', 'title': 'Proteogenomic Landscape of Breast Cancer Tumorigenesis and Targeted Therapy', 'year': 2020}, {'publicationId': '161370ab-4f57-11ea-b5ed-0a7ef35af225', 'title': 'Proteogenomic Characterization of Endometrial Carcinoma', 'year': 2020}, {'publicationId': 'fe52a762-74ad-11ec-bde9-0a4e2186f121', 'title': 'Deep Proteomics Using Two Dimensional Data Independent Acquisition Mass Spectrometry', 'year': 2020}, {'publicationId': '245451cc-c519-11ea-9ab4-0ae52bb11cb7', 'title': 'Proteogenomics of Non-smoking Lung Cancer in East Asia Delineates Molecular Signatures of Pathogenesis and Progression', 'year': 2020}, {'publicationId': '24548009-c519-11ea-9ab4-0ae52bb11cb7', 'title': 'Proteogenomic Characterization Reveals Therapeutic Vulnerabilities in Lung Adenocarcinoma', 'year': 2020}, {'publicationId': '3c25e6ab-1507-11eb-8dc2-0a3dadff3d13', 'title': 'Proteogenomic Characterization of Ovarian HGSC Implicates Mitotic Kinases, Replication Stress in Observed Chromosomal Instability', 'year': 2020}, {'publicationId': 'ccd9fc46-d912-11eb-9f63-0a2b20c3a9e9', 'title': 'Microscaled proteogenomic methods for precision oncology', 'year': 2020}, {'publicationId': '3c25ea7d-1507-11eb-8dc2-0a3dadff3d13', 'title': 'Integrated Proteomic and Glycoproteomic Characterization of Human High-Grade Serous Ovarian Carcinoma', 'year': 2020}, {'publicationId': '90c53958-3333-11eb-ab70-0abc48da233b', 'title': 'Integrated Proteogenomic Characterization across Major Histological Types of Pediatric Brain Cancer', 'year': 2020}, {'publicationId': '90f114bd-2b57-11eb-93d9-0adc6d0c0251', 'title': 'Proteogenomic Landscape of Breast Cancer Tumorigenesis and Targeted Therapy', 'year': 2020}, {'publicationId': '161370ab-4f57-11ea-b5ed-0a7ef35af225', 'title': 'Proteogenomic Characterization of Endometrial Carcinoma', 'year': 2020}, {'publicationId': 'fe52a762-74ad-11ec-bde9-0a4e2186f121', 'title': 'Deep Proteomics Using Two Dimensional Data Independent Acquisition Mass Spectrometry', 'year': 2020}, {'publicationId': '245451cc-c519-11ea-9ab4-0ae52bb11cb7', 'title': 'Proteogenomics of Non-smoking Lung Cancer in East Asia Delineates Molecular Signatures of Pathogenesis and Progression', 'year': 2020}, {'publicationId': '24548009-c519-11ea-9ab4-0ae52bb11cb7', 'title': 'Proteogenomic Characterization Reveals Therapeutic Vulnerabilities in Lung Adenocarcinoma', 'year': 2020}, {'publicationId': '3c25e6ab-1507-11eb-8dc2-0a3dadff3d13', 'title': 'Proteogenomic Characterization of Ovarian HGSC Implicates Mitotic Kinases, Replication Stress in Observed Chromosomal Instability', 'year': 2020}, {'publicationId': 'ccd9fc46-d912-11eb-9f63-0a2b20c3a9e9', 'title': 'Microscaled proteogenomic methods for precision oncology', 'year': 2020}, {'publicationId': '3c25ea7d-1507-11eb-8dc2-0a3dadff3d13', 'title': 'Integrated Proteomic and Glycoproteomic Characterization of Human High-Grade Serous Ovarian Carcinoma', 'year': 2020}, {'publicationId': '90c53958-3333-11eb-ab70-0abc48da233b', 'title': 'Integrated Proteogenomic Characterization across Major Histological Types of Pediatric Brain Cancer', 'year': 2020}, {'publicationId': '90f114bd-2b57-11eb-93d9-0adc6d0c0251', 'title': 'Proteogenomic Landscape of Breast Cancer Tumorigenesis and Targeted Therapy', 'year': 2020}]}}}\n",
            "\n",
            "Querying publications related to 'Cancer'...\n",
            "{'data': {'Get': {'Publication': [{'abstract': 'Acute myeloid leukemia is a poor-prognosis cancer commonly stratified by genetic aberrations, but these mutations are often heterogeneous and fail to consistently predict therapeutic response. Here, we combine transcriptomic, proteomic, and phosphoproteomic datasets with ex vivo drug sensitivity data to help understand the underlying pathophysiology of AML beyond mutations. We measure the proteome and phosphoproteome of 210 patients and combine them with genomic and transcriptomic measurements to identify four proteogenomic subtypes that complement existing genetic subtypes. We build a predictor to classify samples into subtypes and map them to a \"landscape\" that identifies specific drug response patterns. We then build a drug response prediction model to identify drugs that target distinct subtypes and validate our findings on cell lines representing various stages of quizartinib resistance. Our results show how multiomics data together with drug sensitivity data can inform therapy stratification and drug combinations in AML.', 'publicationId': '60806ae5-c064-11ee-babf-0aaebe6fe27b', 'title': 'Mapping the proteogenomic landscape enables prediction of drug response in acute myeloid leukemia'}, {'abstract': 'Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal cancer types, partly because it is frequently identified at an advanced stage, when surgery is no longer feasible. Therefore, early detection using minimally invasive methods such as blood tests may improve outcomes. However, studies to discover molecular signatures for the early detection of PDAC using blood tests have only been marginally successful. In the current study, a quantitative glycoproteomic approach via data-independent acquisition mass spectrometry was utilized to detect glycoproteins in 29 patient-matched PDAC tissues and sera. A total of 892 N-linked glycopeptides originating from 141 glycoproteins had PDAC-associated changes beyond normal variation. We further evaluated the specificity of these serum-detectable glycoproteins by comparing their abundance in 53 independent PDAC patient sera and 65 cancer-free controls. The PDAC tissue-associated glycoproteins we have identified represent an inventory of serum-detectable PDAC-associated glycoproteins as candidate biomarkers that can be potentially used for the detection of PDAC using blood tests.', 'publicationId': '86134adf-072f-11ef-b92f-0a200668d9ef', 'title': 'Detection of Pancreatic Ductal Adenocarcinoma-Associated Proteins in Serum'}, {'abstract': 'Clear cell renal cell carcinoma (ccRCC), a common form of RCC, is responsible for the high mortality rate of kidney cancer. Dysregulations of glycoproteins have been shown to associate with ccRCC. However, the molecular mechanism has not been well characterized. Here, a comprehensive glycoproteomic analysis is conducted using 103 tumors and 80 paired normal adjacent tissues. Altered glycosylation enzymes and corresponding protein glycosylation are observed, while two of the major ccRCC mutations, BAP1 and PBRM1, show distinct glycosylation profiles. Additionally, inter-tumor heterogeneity and cross-correlation between glycosylation and phosphorylation are observed. The relation of glycoproteomic features to genomic, transcriptomic, proteomic, and phosphoproteomic changes shows the role of glycosylation in ccRCC development with potential for therapeutic interventions. This study reports a large-scale tandem mass tag (TMT)-based quantitative glycoproteomic analysis of ccRCC that can serve as a valuable resource for the community.', 'publicationId': '129b3284-e36c-11ed-b3ec-0a7bf0b2090f', 'title': 'Integrated glycoproteomic characterization of clear cell renal cell carcinoma'}, {'abstract': 'The identification of differentially expressed tumor-associated proteins and genomic alterations driving neoplasia is critical in the development of clinical assays to detect cancers and forms the foundation for understanding cancer biology. One of the challenges in the analysis of pancreatic ductal adenocarcinoma (PDAC) is the low neoplastic cellularity and heterogeneous composition of bulk tumors. To enrich neoplastic cells from bulk tumor tissue, coring, and laser microdissection (LMD) sampling techniques have been employed. In this study, we assessed the protein and KRAS mutation changes associated with samples obtained by these enrichment techniques and evaluated the fraction of neoplastic cells in PDAC for proteomic and genomic analyses.', 'publicationId': 'bef019a4-50b1-11ed-a6f8-0a79dd92157b', 'title': 'Neoplastic cell enrichment of tumor tissues using coring and laser microdissection for proteomic and genomic analyses of pancreatic ductal adenocarcinoma'}, {'abstract': 'Microscaled proteogenomics was deployed to probe the molecular basis for differential response to neoadjuvant carboplatin and docetaxel combination chemotherapy for triple-negative breast cancer (TNBC). Proteomic analyses of pre-treatment patient biopsies uniquely revealed metabolic pathways, including oxidative phosphorylation, adipogenesis and fatty acid metabolism, that were resistance-associated. Both proteomics and transcriptomics revealed sensitivity was marked by elevation of DNA repair, E2F targets, G2M checkpoint, interferon-gamma signaling and immune checkpoint components. Proteogenomic analyses of somatic copy number aberrations identified a resistance-associated 19q13.31-33 deletion where LIG1, POLD1 and XRCC1 are located. In orthogonal datasets, LIG1 (DNA ligase I) gene deletion and/or low mRNA expression levels were associated with lack of pathological complete response, higher chromosomal instability (CIN) and poor prognosis in TNBC, as well as carboplatin-selective resistance in TNBC pre-clinical models. Hemizygous loss of LIG1 was also associated with higher CIN and poor prognosis in other cancer types, demonstrating broader clinical implications.', 'publicationId': '76a651c5-23da-11ed-adcf-0eeb2b18e995', 'title': 'Proteogenomic markers of chemotherapy resistance and response in triple negative breast cancer'}, {'abstract': 'Lung squamous cell carcinoma (LSCC) remains a leading cause of cancer death with few therapeutic options. We characterized the proteogenomic landscape of LSCC, providing a deeper exposition of LSCC biology with potential therapeutic implications. We identify NSD3 as an alternative driver in FGFR1-amplified tumors and low-p63 tumors overexpressing the therapeutic target survivin. SOX2 is considered undruggable, but our analyses provide rationale for exploring chromatin modifiers such as LSD1 and EZH2 to target SOX2-overexpressing tumors. Our data support complex regulation of metabolic pathways by crosstalk between post-translational modifications including ubiquitylation. Numerous immune-related proteogenomic observations suggest directions for further investigation. Proteogenomic dissection of CDKN2A mutations argue for more nuanced assessment of RB1 protein expression and phosphorylation before declaring CDK4/6 inhibition unsuccessful. Finally, triangulation between LSCC, LUAD, and HNSCC identified both unique and common therapeutic vulnerabilities. These observations and proteogenomics data resources may guide research into the biology and treatment of LSCC.', 'publicationId': 'ebbe2318-f601-11eb-9f63-0a2b20c3a9e9', 'title': 'A proteogenomic portrait of lung squamous cell carcinoma'}, {'abstract': 'Glioblastoma (GBM) is the most aggressive nervous system cancer. Understanding its molecular pathogenesis is crucial to improving diagnosis and treatment. Integrated analysis of genomic, proteomic, post-translational modification and metabolomic data on 99 treatment-naive GBMs provides insights to GBM biology. We identify key phosphorylation events (e.g., phosphorylated PTPN11 and PLCG1) as potential switches mediating oncogenic pathway activation, as well as potential targets for EGFR-, TP53-, and RB1-altered tumors. Immune subtypes with distinct immune cell types are discovered using bulk omics methodologies, validated by snRNA-seq, and correlated with specific expression and histone acetylation patterns. Histone H2B acetylation in classical-like and immune-low GBM is driven largely by BRDs, CREBBP, and EP300. Integrated metabolomic and proteomic data identify specific lipid distributions across subtypes and distinct global metabolic changes in IDH-mutated tumors. This work highlights biological relationships that could contribute to stratification of GBM patients for more effective treatment.', 'publicationId': 'ff35c89b-7071-11eb-9f63-0a2b20c3a9e9', 'title': 'Proteogenomic and metabolomic characterization of human glioblastoma'}, {'abstract': 'Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive cancer with poor patient survival. Toward understanding the underlying molecular alterations that drive PDAC oncogenesis, we conducted comprehensive proteogenomic analysis of 140 pancreatic cancers, 67 normal adjacent tissues, and 9 normal pancreatic ductal tissues. Proteomic, phosphoproteomic, and glycoproteomic analyses were used to characterize proteins and their modifications. In addition, whole-genome sequencing, whole-exome sequencing, methylation, RNA sequencing (RNA-seq), and microRNA sequencing (miRNA-seq) were performed on the same tissues to facilitate an integrated proteogenomic analysis and determine the impact of genomic alterations on protein expression, signaling pathways, and post-translational modifications. To ensure robust downstream analyses, tumor neoplastic cellularity was assessed via multiple orthogonal strategies using molecular features and verified via pathological estimation of tumor cellularity based on histological review. This integrated proteogenomic characterization of PDAC will serve as a valuable resource for the community, paving the way for early detection and identification of novel therapeutic targets.', 'publicationId': 'ffe3fbc9-165b-11ec-b712-0a4e2186f121', 'title': 'Proteogenomic Characterization of Pancreatic Ductal Adenocarcinoma'}, {'abstract': 'The integration of mass spectrometry-based proteomics with next-generation DNA and RNA sequencing profiles tumors more comprehensively. Herein this “proteogenomic” approach was applied to 122 treatment-naive primary breast cancers purposely accrued to preserve post-translational modifications, including protein phosphorylation and acetylation. Proteogenomics challenged standard breast cancer diagnoses, provided detailed analysis of the ERBB2 amplicon, defined tumor subsets that could benefit from immune checkpoint therapy and allowed more accurate assessment of Rb status for the prediction of CDK4/6 inhibitor responsiveness. Phosphoproteomic profiles uncovered novel associations between tumor suppressor loss and targetable kinases. Acetylproteome analysis highlighted acetylation on key nuclear proteins involved in the DNA damage response and revealed cross-talk between cytoplasmic and mitochondrial acetylation and metabolism. Our results underscore the potential of proteogenomics for the clinical investigation of breast cancer through more accurate annotation of targetable pathways and biological features of this remarkably heterogeneous malignancy.', 'publicationId': '90f114bd-2b57-11eb-93d9-0adc6d0c0251', 'title': 'Proteogenomic Landscape of Breast Cancer Tumorigenesis and Targeted Therapy'}, {'abstract': 'We undertook a comprehensive proteogenomic characterization of 95 prospectively collected endometrial carcinomas, comprising 83 endometrioid and 12 serous tumors. This analysis revealed possible new consequences of perturbations to the p53 and Wnt/Î²-catenin pathways, identified a potential role for circRNAs in the epithelial-mesenchymal transition, and provided new information about proteomic markers of clinical and genomic tumor subgroups, including relationships to known druggable pathways. An extensive genome-wide acetylation survey yielded insights into regulatory mechanisms linking Wnt signaling and histone acetylation. We also characterized aspects of the tumor immune landscape, including immunogenic alterations, neoantigens, common cancer/testis antigens, and the immune microenvironment, all of which can inform immunotherapy decisions. Collectively, our multi-omic analyses provide a valuable resource for researchers and clinicians, identify new molecular associations of potential mechanistic significance in the development of endometrial cancers, and suggest novel approaches for identifying potential therapeutic targets.', 'publicationId': '161370ab-4f57-11ea-b5ed-0a7ef35af225', 'title': 'Proteogenomic Characterization of Endometrial Carcinoma'}, {'abstract': 'Methodologies that facilitate high-throughput proteomic analysis are a key step toward moving proteome investigations into clinical translation. Data independent acquisition (DIA) has potential as a high-throughput analytical method due to the reduced time needed for sample analysis, as well as its highly quantitative accuracy. However, a limiting feature of DIA methods is the sensitivity of detection of low abundant proteins and depth of coverage, which other mass spectrometry approaches address by two-dimensional fractionation (2D) to reduce sample complexity during data acquisition. In this study, we developed a 2D-DIA method intended for rapid- and deeper-proteome analysis compared to conventional 1D-DIA analysis. First, we characterized 96 individual fractions obtained from the protein standard, NCI-7, using a data-dependent approach (DDA), identifying a total of 151,366 unique peptides from 11,273 protein groups. We observed that the majority of the proteins can be identified from just a few selected fractions. By performing an optimization analysis, we identified six fractions with high peptide number and uniqueness that can account for 80% of the proteins identified in the entire experiment. These selected fractions were combined into a single sample which was then subjected to DIA (referred to as 2D-DIA) quantitative analysis. Furthermore, improved DIA quantification was achieved using a hybrid spectral library, obtained by combining peptides identified from DDA data with peptides identified directly from the DIA runs with the help of DIA-Umpire. The optimized 2D-DIA method allowed for improved identification and quantification of low abundant proteins compared to conventional unfractionated DIA analysis (1D-DIA). We then applied the 2D-DIA method to profile the proteomes of two breast cancer patient-derived xenograft (PDX) models, quantifying 6,217 and 6,167 unique proteins in basal- and luminal- tumors, respectively. Overall, this study demonstrates the potential of high-throughput quantitative proteomics using a novel 2D-DIA method.', 'publicationId': 'fe52a762-74ad-11ec-bde9-0a4e2186f121', 'title': 'Deep Proteomics Using Two Dimensional Data Independent Acquisition Mass Spectrometry'}, {'abstract': 'Lung cancer in East Asia is characterized by a high percentage of never-smokers, early onset and predominant EGFR mutations. To illuminate the molecular phenotype of this demographically distinct disease, we performed a deep comprehensive proteogenomic study on a prospectively collected cohort in Taiwan, representing early stage, predominantly female, non-smoking lung adenocarcinoma. Integrated genomic, proteomic, and phosphoproteomic analysis delineated the demographically distinct molecular attributes and hallmarks of tumor progression. Mutational signature analysis revealed age- and gender-related mutagenesis mechanisms, characterized by high prevalence of APOBEC mutational signature in younger females and over-representation of environmental carcinogen-like mutational signatures in older females. A proteomics-informed classification distinguished the clinical characteristics of early stage patients with EGFR mutations. Furthermore, integrated protein network analysis revealed the cellular remodeling underpinning clinical trajectories and nominated candidate biomarkers for patient stratification and therapeutic intervention. This multi-omic molecular architecture may help develop strategies for management of early stage never-smoker lung adenocarcinoma.', 'publicationId': '245451cc-c519-11ea-9ab4-0ae52bb11cb7', 'title': 'Proteogenomics of Non-smoking Lung Cancer in East Asia Delineates Molecular Signatures of Pathogenesis and Progression'}, {'abstract': 'In the absence of a dominant driving mutation other than uniformly present TP53 mutations, deeper understanding of the biology driving ovarian high-grade serous cancer (HGSC) requires analysis at a functional level, including post-translational modifications. Comprehensive proteogenomic and phosphoproteomic characterization of 83 prospectively collected ovarian HGSC and appropriate normal precursor tissue samples (fallopian tube) under strict control of ischemia time reveals pathways that significantly differentiate between HGSC and relevant normal tissues in the context of homologous repair deficiency (HRD) status. In addition to confirming key features of HGSC from previous studies, including a potential survival-associated signature and histone acetylation as a marker of HRD, deep phosphoproteomics provides insights regarding the potential role of proliferation-induced replication stress in promoting the characteristic chromosomal instability of HGSC and suggests potential therapeutic targets for use in precision medicine trials.', 'publicationId': '3c25e6ab-1507-11eb-8dc2-0a3dadff3d13', 'title': 'Proteogenomic Characterization of Ovarian HGSC Implicates Mitotic Kinases, Replication Stress in Observed Chromosomal Instability'}, {'abstract': 'Cancer proteogenomics promises new insights into cancer biology and treatment efficacy by integrating genomics, transcriptomics and protein profiling including modifications by mass spectrometry (MS). A critical limitation is sample input requirements that exceed many sources of clinically important material. Here we report a proteogenomics approach for core biopsies using tissue-sparing specimen processing and microscaled proteomics. As a demonstration, we analyze core needle biopsies from ERBB2 positive breast cancers before and 48-72\\u2009h after initiating neoadjuvant trastuzumab-based chemotherapy. We show greater suppression of ERBB2 protein and both ERBB2 and mTOR target phosphosite levels in cases associated with pathological complete response, and identify potential causes of treatment resistance including the absence of ERBB2 amplification, insufficient ERBB2 activity for therapeutic sensitivity despite ERBB2 amplification, and candidate resistance mechanisms including androgen receptor signaling, mucin overexpression and an inactive immune microenvironment. The clinical utility and discovery potential of proteogenomics at biopsy-scale warrants further investigation.', 'publicationId': 'ccd9fc46-d912-11eb-9f63-0a2b20c3a9e9', 'title': 'Microscaled proteogenomic methods for precision oncology'}, {'abstract': 'We report a comprehensive proteogenomics analysis, including whole-genome sequencing, RNA sequencing, and proteomics and phosphoproteomics profiling, of 218 tumors across 7 histological types of childhood brain cancer: low-grade glioma (nÂ = 93), ependymoma (32), high-grade glioma (25), medulloblastoma (22), ganglioglioma (18), craniopharyngioma (16), and atypical teratoid rhabdoid tumor (12). Proteomics data identify common biological themes that span histological boundaries, suggesting that treatments used for one histological type may be applied effectively to other tumors sharing similar proteomics features. Immune landscape characterization reveals diverse tumor microenvironments across and within diagnoses. Proteomics data further reveal functional effects of somatic mutations and copy number variations (CNVs) not evident in transcriptomics data. Kinase-substrate association and co-expression network analysis identify important biological mechanisms of tumorigenesis. This is the first large-scale proteogenomics analysis across traditional histological boundaries to uncover foundational pediatric brain tumor biology and inform rational treatment selection.', 'publicationId': '90c53958-3333-11eb-ab70-0abc48da233b', 'title': 'Integrated Proteogenomic Characterization across Major Histological Types of Pediatric Brain Cancer'}, {'abstract': 'We performed the first proteogenomic study on a prospectively collected colon cancer cohort. Comparative proteomic and phosphoproteomic analysis of paired tumor and normal adjacent tissues produced a catalog of colon cancer-associated proteins and phosphosites, including known and putative new biomarkers, drug targets, and cancer/testis antigens. Proteogenomic integration not only prioritized genomically inferred targets, such as copy-number drivers and mutation-derived neoantigens, but also yielded novel findings. Phosphoproteomics data associated Rb phosphorylation with increased proliferation and decreased apoptosis in colon cancer, which explains why this classical tumor suppressor is amplified in colon tumors and suggests a rationale for targeting Rb phosphorylation in colon cancer. Proteomics identified an association between decreased CD8 T cell infiltration and increased glycolysis in microsatellite instability-high (MSI-H) tumors, suggesting glycolysis as a potential target to overcome the resistance of MSI-H tumors to immune checkpoint blockade. Proteogenomics presents new avenues for biological discoveries and therapeutic development.', 'publicationId': '7257a194-ba5e-11eb-9f63-0a2b20c3a9e9', 'title': 'Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities'}, {'abstract': 'We report proteogenomic analysis of diffuse gastric cancers (GCs) in young populations. Phosphoproteome data elucidated signaling pathways associated with somatic mutations based on mutation-phosphorylation correlations. Moreover, correlations between mRNA and protein abundances provided potential oncogenes and tumor suppressors associated with patient survival. Furthermore, integrated clustering of mRNA, protein, phosphorylation, and N-glycosylation data identified four subtypes of diffuse GCs. Distinguishing these subtypes was possible by proteomic data. Four subtypes were associated with proliferation, immune response, metabolism, and invasion, respectively; and associations of the subtypes with immune- and invasion-related pathways were identified mainly by phosphorylation and N-glycosylation data. Therefore, our proteogenomic analysis provides additional information beyond genomic analyses, which can improve understanding of cancer biology and patient stratification in diffuse GCs.', 'publicationId': 'f2121b29-7828-11ea-b5ed-0a7ef35af225', 'title': 'Proteogenomic Characterization of Human Early-Onset Gastric Cancer'}, {'abstract': 'Here we present an optimized workflow for global proteome and phosphoproteome analysis of tissues or cell lines that uses isobaric tags (TMT (tandem mass tags)-10) for multiplexed analysis and relative quantification, and provides 3√ó higher throughput than iTRAQ (isobaric tags for absolute and relative quantification)-4-based methods with high intra- and inter-laboratory reproducibility. The workflow was systematically characterized and benchmarked across three independent laboratories using two distinct breast cancer subtypes from patient-derived xenograft models to enable assessment of proteome and phosphoproteome depth and quantitative reproducibility. Each plex consisted of ten samples, each being 300 Œºg of peptide derived from <50 mg of wet-weight tissue. Of the 10,000 proteins quantified per sample, we could distinguish 7,700 human proteins derived from tumor cells and 3100 mouse proteins derived from the surrounding stroma and blood. The maximum deviation across replicates and laboratories was <7%, and the inter-laboratory correlation for TMT ratio-based comparison of the two breast cancer subtypes was r > 0.88. The maximum deviation for the phosphoproteome coverage was <24% across laboratories, with an average of >37,000 quantified phosphosites per sample and differential quantification correlations of r > 0.72. The full procedure, including sample processing and data generation, can be completed within 10 d for ten tissue samples, and 100 samples can be analyzed in ~4 months using a single LC-MS/MS instrument. The high quality, depth, and reproducibility of the data obtained both within and across laboratories should enable new biological insights to be obtained from mass spectrometry-based proteomics analyses of cells and tissues together with proteogenomic data integration.', 'publicationId': '80459042-c3d7-11eb-9f63-0a2b20c3a9e9', 'title': 'Reproducible workflow for multiplexed deep-scale proteome and phosphoproteome analysis of tumor tissues by liquid chromatography-mass spectrometry'}, {'abstract': 'Reference materials are vital to benchmarking the reproducibility of clinical tests and essential for monitoring laboratory performance for clinical proteomics. The reference material utilized for mass spectrometric analysis of the human proteome would ideally contain enough proteins to be suitably representative of the human proteome, as well as exhibit a stable protein composition in different batches of sample regeneration. Previously, The Clinical Proteomic Tumor Analysis Consortium (CPTAC) utilized a PDX-derived comparative reference (CompRef) materials for the longitudinal assessment of proteomic performance; however, inherent drawbacks of PDX-derived material, including extended time needed to grow tumors and high level of expertise needed, have resulted in efforts to identify a new source of CompRef material. In this study, we examined the utility of using a panel of seven cancer cell lines, NCI-7 Cell Line Panel, as a reference material for mass spectrometric analysis of human proteome. Our results showed that not only is the NCI-7 material suitable for benchmarking laboratory sample preparation methods, but also NCI-7 sample generation is highly reproducible at both the global and phosphoprotein levels. In addition, the predicted genomic and experimental coverage of the NCI-7 proteome suggests the NCI-7 material may also have applications as a universal standard proteomic reference.', 'publicationId': '45de306b-a861-11eb-9f63-0a2b20c3a9e9', 'title': 'Evaluation of NCI-7 Cell Line Panel as a Reference Material for Clinical Proteomics'}, {'abstract': 'Activation of PI3K signaling is frequently observed in triple-negative breast cancer (TNBC), yet PI3K inhibitors have shown limited clinical activity. To investigate intrinsic and adaptive mechanisms of resistance, we analyzed a panel of patient-derived xenograft models of TNBC with varying responsiveness to buparlisib, a pan-PI3K inhibitor. In a subset of patient-derived xenografts, resistance was associated with incomplete inhibition of PI3K signaling and upregulated MAPK/MEK signaling in response to buparlisib. Outlier phosphoproteome and kinome analyses identified novel candidates functionally important to buparlisib resistance, including NEK9 and MAP2K4. Knockdown of NEK9 or MAP2K4 reduced both baseline and feedback MAPK/MEK signaling and showed synthetic lethality with buparlisib in vitro. A complex in/del frameshift in PIK3CA decreased sensitivity to buparlisib via NEK9/MAP2K4√¢‚Ç¨‚Äúdependent mechanisms. In summary, our study supports a role for NEK9 and MAP2K4 in mediating buparlisib resistance and demonstrates the value of unbiased omic analyses in uncovering resistance mechanisms to targeted therapy.', 'publicationId': 'd37418f1-b87e-11eb-9f63-0a2b20c3a9e9', 'title': 'Mass Spectrometry-Based Proteomics Reveals Potential Roles of NEK9 and MAP2K4 in Resistance to PI3K Inhibition in Triple-Negative Breast Cancers'}, {'abstract': 'Oral squamous cell carcinoma is a prominent cancer worldwide, particularly in Taiwan. By integrating omics analyses in 50 matched samples, we uncover in Taiwanese patients a predominant mutation signature associated with cytidine deaminase APOBEC, which correlates with the upregulation of APOBEC3A expression in the APOBEC3 gene cluster at 22q13. APOBEC3A expression is significantly higher in tumors carrying APOBEC3B-deletion allele(s). High-level APOBEC3A expression is associated with better overall survival, especially among patients carrying APOBEC3B-deletion alleles, as examined in a second cohort (n = 188; p = 0.004). The frequency of APOBEC3B-deletion alleles is ~50% in 143 genotyped oral squamous cell carcinoma -Taiwan samples (27A3B −/−:89A3B +/−:27A3B +/+), compared to the 5.8% found in 314 OSCC-TCGA samples. We thus report a frequent APOBEC mutational profile, which relates to a APOBEC3B-deletion germline polymorphism in Taiwanese oral squamous cell carcinoma that impacts expression of APOBEC3A, and is shown to be of clinical prognostic relevance. Our finding might be recapitulated by genomic studies in other cancer types.', 'publicationId': 'd2b6c2b7-4370-11eb-bcbc-0abc48da233b', 'title': 'APOBEC3A is an oral cancer prognostic biomarker in Taiwanese carriers of an APOBEC deletion polymorphism'}, {'abstract': 'Recent advances in mass spectrometry (MS) have enabled extensive analysis of cancer proteomes. Here, we employed quantitative proteomics to profile protein expression across 24 breast cancer patient-derived xenograft (PDX) models. Integrated proteogenomic analysis shows positive correlation between expression measurements from transcriptomic and proteomic analyses; further, gene expression-based intrinsic subtypes are largely re-capitulated using non-stromal protein markers. Proteogenomic analysis also validates a number of predicted genomic targets in multiple receptor tyrosine kinases. However, several protein/phosphoprotein events such as overexpression of AKT proteins and ARAF, BRAF, HSP90AB1 phosphosites are not readily explainable by genomic analysis, suggesting that druggable translational and/or post-translational regulatory events may be uniquely diagnosed by MS. Drug treatment experiments targeting HER2 and components of the PI3K pathway supported proteogenomic response predictions in seven xenograft models. Our study demonstrates that MS-based proteomics can identify therapeutic targets and highlights the potential of PDX drug response evaluation to annotate MS-based pathway activities.', 'publicationId': '718f77fb-b329-11eb-9f63-0a2b20c3a9e9', 'title': 'Proteogenomic integration reveals therapeutic targets in breast cancer xenografts'}, {'abstract': 'To provide a detailed analysis of the molecular components and underlying mechanisms associated with ovarian cancer, we performed a comprehensive mass-spectrometry-based proteomic characterization of 174 ovarian tumors previously analyzed by The Cancer Genome Atlas (TCGA), of which 169 were high-grade serous carcinomas (HGSCs). Integrating our proteomic measurements with the genomic data yielded a number of insights into disease, such as how different copy-number alternations influence the proteome, the proteins associated with chromosomal instability, the sets of signaling pathways that diverse genome rearrangements converge on, and the ones most associated with short overall survival. Specific protein acetylations associated with homologous recombination deficiency suggest a potential means for stratifying patients for therapy. In addition to providing a valuable resource, these findings provide a view of how the somatic genome drives the cancer proteome and associations between protein and post-translational modification levels and clinical outcomes in HGSC.', 'publicationId': '44e2e658-89c7-11e8-bcf1-0a2705229b82', 'title': 'Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer'}, {'abstract': 'Somatic mutations have been extensively characterized in breast cancer, but the effects of these genetic alterations on the proteomic landscape remain poorly understood. Here we describe quantitative mass-spectrometry-based proteomic and phosphoproteomic analyses of 105 genomically annotated breast cancers, of which 77 provided high-quality data. Integrated analyses provided insights into the somatic cancer genome including the consequences of chromosomal loss, such as the 5q deletion characteristic of basal-like breast cancer. Interrogation of the 5q trans-effects against the Library of Integrated Network-based Cellular Signatures, connected loss of CETN3 and SKP1 to elevated expression of epidermal growth factor receptor (EGFR), and SKP1 loss also to increased SRC tyrosine kinase. Global proteomic data confirmed a stromal-enriched group of proteins in addition to basal and luminal clusters, and pathway analysis of the phosphoproteome identified a G-protein-coupled receptor cluster that was not readily identified at the mRNA level. In addition to ERBB2, other amplicon-associated highly phosphorylated kinases were identified, including CDK12, PAK1, PTK2, RIPK2 and TLK2. We demonstrate that proteogenomic analysis of breast cancer elucidates the functional consequences of somatic mutations, narrows candidate nominations for driver genes within large deletions and amplified regions, and identifies therapeutic targets.', 'publicationId': '44e2e766-89c7-11e8-bcf1-0a2705229b82', 'title': 'Proteogenomics connects somatic mutations to signalling in breast cancer'}, {'abstract': 'Understanding proteomic differences underlying the different phenotypic classes of colon and rectal carcinoma is important and may eventually lead to a better assessment of clinical behavior of these cancers. We here present a comprehensive description of the proteomic data obtained from 90 colon and rectal carcinomas previously subjected to genomic analysis by The Cancer Genome Atlas (TCGA). Here, the primary instrument files and derived secondary data files are compiled and presented in forms that will allow further analyses of the biology of colon and rectal carcinoma. We also discuss new challenges in processing these large proteomic datasets for relevant proteins and protein variants.', 'publicationId': '44e2e50c-89c7-11e8-bcf1-0a2705229b82', 'title': 'Proteomic analysis of colon and rectal carcinoma using standard and customized databases'}, {'abstract': 'Clinical specimens are each inherently unique, limited and nonrenewable. Small samples such as tissue biopsies are often completely consumed after a limited number of analyses. Here we present a method that enables fast and reproducible conversion of a small amount of tissue (approximating the quantity obtained by a biopsy) into a single, permanent digital file representing the mass spectrometry (MS)-measurable proteome of the sample. The method combines pressure cycling technology (PCT) and sequential window acquisition of all theoretical fragment ion spectra (SWATH)-MS. The resulting proteome maps can be analyzed, re-analyzed, compared and mined in silico to detect and quantify specific proteins across multiple samples. We used this method to process and convert 18 biopsy samples from nine patients with renal cell carcinoma into SWATH-MS fragment ion maps. From these proteome maps we detected and quantified more than 2,000 proteins with a high degree of reproducibility across all samples. The measured proteins clearly distinguished tumorous kidney tissues from healthy tissues and differentiated distinct histomorphological kidney cancer subtypes.', 'publicationId': '18a56fbd-10d5-11e9-afb9-0a9c39d33490', 'title': 'Rapid mass spectrometric conversion of tissue biopsy samples into permanent quantitative digital proteome maps'}, {'abstract': \"Extensive genomic characterization of human cancers presents the problem of inference from genomic abnormalities to cancer phenotypes. To address this problem, we analysed proteomes of colon and rectal tumours characterized previously by The Cancer Genome Atlas (TCGA) and perform integrated proteogenomic analyses. Somatic variants displayed reduced protein abundance compared to germline variants. Messenger RNA transcript abundance did not reliably predict protein abundance differences between tumours. Proteomics identified five proteomic subtypes in the TCGA cohort, two of which overlapped with the TCGA 'microsatellite instability/CpG island methylation phenotype' transcriptomic subtype, but had distinct mutation, methylation and protein expression patterns associated with different clinical outcomes. Although copy number alterations showed strong cis- and trans-effects on mRNA abundance, relatively few of these extend to the protein level. Thus, proteomics data enabled prioritization of candidate driver genes. The chromosome 20q amplicon was associated with the largest global changes at both mRNA and protein levels; proteomics data highlighted potential 20q candidates, including HNF4A (hepatocyte nuclear factor 4, alpha), TOMM34 (translocase of outer mitochondrial membrane 34) and SRC (SRC proto-oncogene, non-receptor tyrosine kinase). Integrated proteogenomic analysis provides functional context to interpret genomic abnormalities and affords a new paradigm for understanding cancer biology.\", 'publicationId': '44e2e0ce-89c7-11e8-bcf1-0a2705229b82', 'title': 'Proteogenomic characterization of human colon and rectal cancer'}, {'abstract': 'Acute myeloid leukemia is a poor-prognosis cancer commonly stratified by genetic aberrations, but these mutations are often heterogeneous and fail to consistently predict therapeutic response. Here, we combine transcriptomic, proteomic, and phosphoproteomic datasets with ex vivo drug sensitivity data to help understand the underlying pathophysiology of AML beyond mutations. We measure the proteome and phosphoproteome of 210 patients and combine them with genomic and transcriptomic measurements to identify four proteogenomic subtypes that complement existing genetic subtypes. We build a predictor to classify samples into subtypes and map them to a \"landscape\" that identifies specific drug response patterns. We then build a drug response prediction model to identify drugs that target distinct subtypes and validate our findings on cell lines representing various stages of quizartinib resistance. Our results show how multiomics data together with drug sensitivity data can inform therapy stratification and drug combinations in AML.', 'publicationId': '60806ae5-c064-11ee-babf-0aaebe6fe27b', 'title': 'Mapping the proteogenomic landscape enables prediction of drug response in acute myeloid leukemia'}, {'abstract': 'Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal cancer types, partly because it is frequently identified at an advanced stage, when surgery is no longer feasible. Therefore, early detection using minimally invasive methods such as blood tests may improve outcomes. However, studies to discover molecular signatures for the early detection of PDAC using blood tests have only been marginally successful. In the current study, a quantitative glycoproteomic approach via data-independent acquisition mass spectrometry was utilized to detect glycoproteins in 29 patient-matched PDAC tissues and sera. A total of 892 N-linked glycopeptides originating from 141 glycoproteins had PDAC-associated changes beyond normal variation. We further evaluated the specificity of these serum-detectable glycoproteins by comparing their abundance in 53 independent PDAC patient sera and 65 cancer-free controls. The PDAC tissue-associated glycoproteins we have identified represent an inventory of serum-detectable PDAC-associated glycoproteins as candidate biomarkers that can be potentially used for the detection of PDAC using blood tests.', 'publicationId': '86134adf-072f-11ef-b92f-0a200668d9ef', 'title': 'Detection of Pancreatic Ductal Adenocarcinoma-Associated Proteins in Serum'}, {'abstract': 'Clear cell renal cell carcinoma (ccRCC), a common form of RCC, is responsible for the high mortality rate of kidney cancer. Dysregulations of glycoproteins have been shown to associate with ccRCC. However, the molecular mechanism has not been well characterized. Here, a comprehensive glycoproteomic analysis is conducted using 103 tumors and 80 paired normal adjacent tissues. Altered glycosylation enzymes and corresponding protein glycosylation are observed, while two of the major ccRCC mutations, BAP1 and PBRM1, show distinct glycosylation profiles. Additionally, inter-tumor heterogeneity and cross-correlation between glycosylation and phosphorylation are observed. The relation of glycoproteomic features to genomic, transcriptomic, proteomic, and phosphoproteomic changes shows the role of glycosylation in ccRCC development with potential for therapeutic interventions. This study reports a large-scale tandem mass tag (TMT)-based quantitative glycoproteomic analysis of ccRCC that can serve as a valuable resource for the community.', 'publicationId': '129b3284-e36c-11ed-b3ec-0a7bf0b2090f', 'title': 'Integrated glycoproteomic characterization of clear cell renal cell carcinoma'}, {'abstract': 'The identification of differentially expressed tumor-associated proteins and genomic alterations driving neoplasia is critical in the development of clinical assays to detect cancers and forms the foundation for understanding cancer biology. One of the challenges in the analysis of pancreatic ductal adenocarcinoma (PDAC) is the low neoplastic cellularity and heterogeneous composition of bulk tumors. To enrich neoplastic cells from bulk tumor tissue, coring, and laser microdissection (LMD) sampling techniques have been employed. In this study, we assessed the protein and KRAS mutation changes associated with samples obtained by these enrichment techniques and evaluated the fraction of neoplastic cells in PDAC for proteomic and genomic analyses.', 'publicationId': 'bef019a4-50b1-11ed-a6f8-0a79dd92157b', 'title': 'Neoplastic cell enrichment of tumor tissues using coring and laser microdissection for proteomic and genomic analyses of pancreatic ductal adenocarcinoma'}, {'abstract': 'Microscaled proteogenomics was deployed to probe the molecular basis for differential response to neoadjuvant carboplatin and docetaxel combination chemotherapy for triple-negative breast cancer (TNBC). Proteomic analyses of pre-treatment patient biopsies uniquely revealed metabolic pathways, including oxidative phosphorylation, adipogenesis and fatty acid metabolism, that were resistance-associated. Both proteomics and transcriptomics revealed sensitivity was marked by elevation of DNA repair, E2F targets, G2M checkpoint, interferon-gamma signaling and immune checkpoint components. Proteogenomic analyses of somatic copy number aberrations identified a resistance-associated 19q13.31-33 deletion where LIG1, POLD1 and XRCC1 are located. In orthogonal datasets, LIG1 (DNA ligase I) gene deletion and/or low mRNA expression levels were associated with lack of pathological complete response, higher chromosomal instability (CIN) and poor prognosis in TNBC, as well as carboplatin-selective resistance in TNBC pre-clinical models. Hemizygous loss of LIG1 was also associated with higher CIN and poor prognosis in other cancer types, demonstrating broader clinical implications.', 'publicationId': '76a651c5-23da-11ed-adcf-0eeb2b18e995', 'title': 'Proteogenomic markers of chemotherapy resistance and response in triple negative breast cancer'}, {'abstract': 'Lung squamous cell carcinoma (LSCC) remains a leading cause of cancer death with few therapeutic options. We characterized the proteogenomic landscape of LSCC, providing a deeper exposition of LSCC biology with potential therapeutic implications. We identify NSD3 as an alternative driver in FGFR1-amplified tumors and low-p63 tumors overexpressing the therapeutic target survivin. SOX2 is considered undruggable, but our analyses provide rationale for exploring chromatin modifiers such as LSD1 and EZH2 to target SOX2-overexpressing tumors. Our data support complex regulation of metabolic pathways by crosstalk between post-translational modifications including ubiquitylation. Numerous immune-related proteogenomic observations suggest directions for further investigation. Proteogenomic dissection of CDKN2A mutations argue for more nuanced assessment of RB1 protein expression and phosphorylation before declaring CDK4/6 inhibition unsuccessful. Finally, triangulation between LSCC, LUAD, and HNSCC identified both unique and common therapeutic vulnerabilities. These observations and proteogenomics data resources may guide research into the biology and treatment of LSCC.', 'publicationId': 'ebbe2318-f601-11eb-9f63-0a2b20c3a9e9', 'title': 'A proteogenomic portrait of lung squamous cell carcinoma'}, {'abstract': 'Glioblastoma (GBM) is the most aggressive nervous system cancer. Understanding its molecular pathogenesis is crucial to improving diagnosis and treatment. Integrated analysis of genomic, proteomic, post-translational modification and metabolomic data on 99 treatment-naive GBMs provides insights to GBM biology. We identify key phosphorylation events (e.g., phosphorylated PTPN11 and PLCG1) as potential switches mediating oncogenic pathway activation, as well as potential targets for EGFR-, TP53-, and RB1-altered tumors. Immune subtypes with distinct immune cell types are discovered using bulk omics methodologies, validated by snRNA-seq, and correlated with specific expression and histone acetylation patterns. Histone H2B acetylation in classical-like and immune-low GBM is driven largely by BRDs, CREBBP, and EP300. Integrated metabolomic and proteomic data identify specific lipid distributions across subtypes and distinct global metabolic changes in IDH-mutated tumors. This work highlights biological relationships that could contribute to stratification of GBM patients for more effective treatment.', 'publicationId': 'ff35c89b-7071-11eb-9f63-0a2b20c3a9e9', 'title': 'Proteogenomic and metabolomic characterization of human glioblastoma'}, {'abstract': 'Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive cancer with poor patient survival. Toward understanding the underlying molecular alterations that drive PDAC oncogenesis, we conducted comprehensive proteogenomic analysis of 140 pancreatic cancers, 67 normal adjacent tissues, and 9 normal pancreatic ductal tissues. Proteomic, phosphoproteomic, and glycoproteomic analyses were used to characterize proteins and their modifications. In addition, whole-genome sequencing, whole-exome sequencing, methylation, RNA sequencing (RNA-seq), and microRNA sequencing (miRNA-seq) were performed on the same tissues to facilitate an integrated proteogenomic analysis and determine the impact of genomic alterations on protein expression, signaling pathways, and post-translational modifications. To ensure robust downstream analyses, tumor neoplastic cellularity was assessed via multiple orthogonal strategies using molecular features and verified via pathological estimation of tumor cellularity based on histological review. This integrated proteogenomic characterization of PDAC will serve as a valuable resource for the community, paving the way for early detection and identification of novel therapeutic targets.', 'publicationId': 'ffe3fbc9-165b-11ec-b712-0a4e2186f121', 'title': 'Proteogenomic Characterization of Pancreatic Ductal Adenocarcinoma'}, {'abstract': 'The integration of mass spectrometry-based proteomics with next-generation DNA and RNA sequencing profiles tumors more comprehensively. Herein this “proteogenomic” approach was applied to 122 treatment-naive primary breast cancers purposely accrued to preserve post-translational modifications, including protein phosphorylation and acetylation. Proteogenomics challenged standard breast cancer diagnoses, provided detailed analysis of the ERBB2 amplicon, defined tumor subsets that could benefit from immune checkpoint therapy and allowed more accurate assessment of Rb status for the prediction of CDK4/6 inhibitor responsiveness. Phosphoproteomic profiles uncovered novel associations between tumor suppressor loss and targetable kinases. Acetylproteome analysis highlighted acetylation on key nuclear proteins involved in the DNA damage response and revealed cross-talk between cytoplasmic and mitochondrial acetylation and metabolism. Our results underscore the potential of proteogenomics for the clinical investigation of breast cancer through more accurate annotation of targetable pathways and biological features of this remarkably heterogeneous malignancy.', 'publicationId': '90f114bd-2b57-11eb-93d9-0adc6d0c0251', 'title': 'Proteogenomic Landscape of Breast Cancer Tumorigenesis and Targeted Therapy'}, {'abstract': 'We undertook a comprehensive proteogenomic characterization of 95 prospectively collected endometrial carcinomas, comprising 83 endometrioid and 12 serous tumors. This analysis revealed possible new consequences of perturbations to the p53 and Wnt/Î²-catenin pathways, identified a potential role for circRNAs in the epithelial-mesenchymal transition, and provided new information about proteomic markers of clinical and genomic tumor subgroups, including relationships to known druggable pathways. An extensive genome-wide acetylation survey yielded insights into regulatory mechanisms linking Wnt signaling and histone acetylation. We also characterized aspects of the tumor immune landscape, including immunogenic alterations, neoantigens, common cancer/testis antigens, and the immune microenvironment, all of which can inform immunotherapy decisions. Collectively, our multi-omic analyses provide a valuable resource for researchers and clinicians, identify new molecular associations of potential mechanistic significance in the development of endometrial cancers, and suggest novel approaches for identifying potential therapeutic targets.', 'publicationId': '161370ab-4f57-11ea-b5ed-0a7ef35af225', 'title': 'Proteogenomic Characterization of Endometrial Carcinoma'}, {'abstract': 'Methodologies that facilitate high-throughput proteomic analysis are a key step toward moving proteome investigations into clinical translation. Data independent acquisition (DIA) has potential as a high-throughput analytical method due to the reduced time needed for sample analysis, as well as its highly quantitative accuracy. However, a limiting feature of DIA methods is the sensitivity of detection of low abundant proteins and depth of coverage, which other mass spectrometry approaches address by two-dimensional fractionation (2D) to reduce sample complexity during data acquisition. In this study, we developed a 2D-DIA method intended for rapid- and deeper-proteome analysis compared to conventional 1D-DIA analysis. First, we characterized 96 individual fractions obtained from the protein standard, NCI-7, using a data-dependent approach (DDA), identifying a total of 151,366 unique peptides from 11,273 protein groups. We observed that the majority of the proteins can be identified from just a few selected fractions. By performing an optimization analysis, we identified six fractions with high peptide number and uniqueness that can account for 80% of the proteins identified in the entire experiment. These selected fractions were combined into a single sample which was then subjected to DIA (referred to as 2D-DIA) quantitative analysis. Furthermore, improved DIA quantification was achieved using a hybrid spectral library, obtained by combining peptides identified from DDA data with peptides identified directly from the DIA runs with the help of DIA-Umpire. The optimized 2D-DIA method allowed for improved identification and quantification of low abundant proteins compared to conventional unfractionated DIA analysis (1D-DIA). We then applied the 2D-DIA method to profile the proteomes of two breast cancer patient-derived xenograft (PDX) models, quantifying 6,217 and 6,167 unique proteins in basal- and luminal- tumors, respectively. Overall, this study demonstrates the potential of high-throughput quantitative proteomics using a novel 2D-DIA method.', 'publicationId': 'fe52a762-74ad-11ec-bde9-0a4e2186f121', 'title': 'Deep Proteomics Using Two Dimensional Data Independent Acquisition Mass Spectrometry'}, {'abstract': 'Lung cancer in East Asia is characterized by a high percentage of never-smokers, early onset and predominant EGFR mutations. To illuminate the molecular phenotype of this demographically distinct disease, we performed a deep comprehensive proteogenomic study on a prospectively collected cohort in Taiwan, representing early stage, predominantly female, non-smoking lung adenocarcinoma. Integrated genomic, proteomic, and phosphoproteomic analysis delineated the demographically distinct molecular attributes and hallmarks of tumor progression. Mutational signature analysis revealed age- and gender-related mutagenesis mechanisms, characterized by high prevalence of APOBEC mutational signature in younger females and over-representation of environmental carcinogen-like mutational signatures in older females. A proteomics-informed classification distinguished the clinical characteristics of early stage patients with EGFR mutations. Furthermore, integrated protein network analysis revealed the cellular remodeling underpinning clinical trajectories and nominated candidate biomarkers for patient stratification and therapeutic intervention. This multi-omic molecular architecture may help develop strategies for management of early stage never-smoker lung adenocarcinoma.', 'publicationId': '245451cc-c519-11ea-9ab4-0ae52bb11cb7', 'title': 'Proteogenomics of Non-smoking Lung Cancer in East Asia Delineates Molecular Signatures of Pathogenesis and Progression'}, {'abstract': 'In the absence of a dominant driving mutation other than uniformly present TP53 mutations, deeper understanding of the biology driving ovarian high-grade serous cancer (HGSC) requires analysis at a functional level, including post-translational modifications. Comprehensive proteogenomic and phosphoproteomic characterization of 83 prospectively collected ovarian HGSC and appropriate normal precursor tissue samples (fallopian tube) under strict control of ischemia time reveals pathways that significantly differentiate between HGSC and relevant normal tissues in the context of homologous repair deficiency (HRD) status. In addition to confirming key features of HGSC from previous studies, including a potential survival-associated signature and histone acetylation as a marker of HRD, deep phosphoproteomics provides insights regarding the potential role of proliferation-induced replication stress in promoting the characteristic chromosomal instability of HGSC and suggests potential therapeutic targets for use in precision medicine trials.', 'publicationId': '3c25e6ab-1507-11eb-8dc2-0a3dadff3d13', 'title': 'Proteogenomic Characterization of Ovarian HGSC Implicates Mitotic Kinases, Replication Stress in Observed Chromosomal Instability'}, {'abstract': 'Cancer proteogenomics promises new insights into cancer biology and treatment efficacy by integrating genomics, transcriptomics and protein profiling including modifications by mass spectrometry (MS). A critical limitation is sample input requirements that exceed many sources of clinically important material. Here we report a proteogenomics approach for core biopsies using tissue-sparing specimen processing and microscaled proteomics. As a demonstration, we analyze core needle biopsies from ERBB2 positive breast cancers before and 48-72\\u2009h after initiating neoadjuvant trastuzumab-based chemotherapy. We show greater suppression of ERBB2 protein and both ERBB2 and mTOR target phosphosite levels in cases associated with pathological complete response, and identify potential causes of treatment resistance including the absence of ERBB2 amplification, insufficient ERBB2 activity for therapeutic sensitivity despite ERBB2 amplification, and candidate resistance mechanisms including androgen receptor signaling, mucin overexpression and an inactive immune microenvironment. The clinical utility and discovery potential of proteogenomics at biopsy-scale warrants further investigation.', 'publicationId': 'ccd9fc46-d912-11eb-9f63-0a2b20c3a9e9', 'title': 'Microscaled proteogenomic methods for precision oncology'}, {'abstract': 'We report a comprehensive proteogenomics analysis, including whole-genome sequencing, RNA sequencing, and proteomics and phosphoproteomics profiling, of 218 tumors across 7 histological types of childhood brain cancer: low-grade glioma (nÂ = 93), ependymoma (32), high-grade glioma (25), medulloblastoma (22), ganglioglioma (18), craniopharyngioma (16), and atypical teratoid rhabdoid tumor (12). Proteomics data identify common biological themes that span histological boundaries, suggesting that treatments used for one histological type may be applied effectively to other tumors sharing similar proteomics features. Immune landscape characterization reveals diverse tumor microenvironments across and within diagnoses. Proteomics data further reveal functional effects of somatic mutations and copy number variations (CNVs) not evident in transcriptomics data. Kinase-substrate association and co-expression network analysis identify important biological mechanisms of tumorigenesis. This is the first large-scale proteogenomics analysis across traditional histological boundaries to uncover foundational pediatric brain tumor biology and inform rational treatment selection.', 'publicationId': '90c53958-3333-11eb-ab70-0abc48da233b', 'title': 'Integrated Proteogenomic Characterization across Major Histological Types of Pediatric Brain Cancer'}, {'abstract': 'We performed the first proteogenomic study on a prospectively collected colon cancer cohort. Comparative proteomic and phosphoproteomic analysis of paired tumor and normal adjacent tissues produced a catalog of colon cancer-associated proteins and phosphosites, including known and putative new biomarkers, drug targets, and cancer/testis antigens. Proteogenomic integration not only prioritized genomically inferred targets, such as copy-number drivers and mutation-derived neoantigens, but also yielded novel findings. Phosphoproteomics data associated Rb phosphorylation with increased proliferation and decreased apoptosis in colon cancer, which explains why this classical tumor suppressor is amplified in colon tumors and suggests a rationale for targeting Rb phosphorylation in colon cancer. Proteomics identified an association between decreased CD8 T cell infiltration and increased glycolysis in microsatellite instability-high (MSI-H) tumors, suggesting glycolysis as a potential target to overcome the resistance of MSI-H tumors to immune checkpoint blockade. Proteogenomics presents new avenues for biological discoveries and therapeutic development.', 'publicationId': '7257a194-ba5e-11eb-9f63-0a2b20c3a9e9', 'title': 'Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities'}, {'abstract': 'We report proteogenomic analysis of diffuse gastric cancers (GCs) in young populations. Phosphoproteome data elucidated signaling pathways associated with somatic mutations based on mutation-phosphorylation correlations. Moreover, correlations between mRNA and protein abundances provided potential oncogenes and tumor suppressors associated with patient survival. Furthermore, integrated clustering of mRNA, protein, phosphorylation, and N-glycosylation data identified four subtypes of diffuse GCs. Distinguishing these subtypes was possible by proteomic data. Four subtypes were associated with proliferation, immune response, metabolism, and invasion, respectively; and associations of the subtypes with immune- and invasion-related pathways were identified mainly by phosphorylation and N-glycosylation data. Therefore, our proteogenomic analysis provides additional information beyond genomic analyses, which can improve understanding of cancer biology and patient stratification in diffuse GCs.', 'publicationId': 'f2121b29-7828-11ea-b5ed-0a7ef35af225', 'title': 'Proteogenomic Characterization of Human Early-Onset Gastric Cancer'}, {'abstract': 'Here we present an optimized workflow for global proteome and phosphoproteome analysis of tissues or cell lines that uses isobaric tags (TMT (tandem mass tags)-10) for multiplexed analysis and relative quantification, and provides 3√ó higher throughput than iTRAQ (isobaric tags for absolute and relative quantification)-4-based methods with high intra- and inter-laboratory reproducibility. The workflow was systematically characterized and benchmarked across three independent laboratories using two distinct breast cancer subtypes from patient-derived xenograft models to enable assessment of proteome and phosphoproteome depth and quantitative reproducibility. Each plex consisted of ten samples, each being 300 Œºg of peptide derived from <50 mg of wet-weight tissue. Of the 10,000 proteins quantified per sample, we could distinguish 7,700 human proteins derived from tumor cells and 3100 mouse proteins derived from the surrounding stroma and blood. The maximum deviation across replicates and laboratories was <7%, and the inter-laboratory correlation for TMT ratio-based comparison of the two breast cancer subtypes was r > 0.88. The maximum deviation for the phosphoproteome coverage was <24% across laboratories, with an average of >37,000 quantified phosphosites per sample and differential quantification correlations of r > 0.72. The full procedure, including sample processing and data generation, can be completed within 10 d for ten tissue samples, and 100 samples can be analyzed in ~4 months using a single LC-MS/MS instrument. The high quality, depth, and reproducibility of the data obtained both within and across laboratories should enable new biological insights to be obtained from mass spectrometry-based proteomics analyses of cells and tissues together with proteogenomic data integration.', 'publicationId': '80459042-c3d7-11eb-9f63-0a2b20c3a9e9', 'title': 'Reproducible workflow for multiplexed deep-scale proteome and phosphoproteome analysis of tumor tissues by liquid chromatography-mass spectrometry'}, {'abstract': 'Reference materials are vital to benchmarking the reproducibility of clinical tests and essential for monitoring laboratory performance for clinical proteomics. The reference material utilized for mass spectrometric analysis of the human proteome would ideally contain enough proteins to be suitably representative of the human proteome, as well as exhibit a stable protein composition in different batches of sample regeneration. Previously, The Clinical Proteomic Tumor Analysis Consortium (CPTAC) utilized a PDX-derived comparative reference (CompRef) materials for the longitudinal assessment of proteomic performance; however, inherent drawbacks of PDX-derived material, including extended time needed to grow tumors and high level of expertise needed, have resulted in efforts to identify a new source of CompRef material. In this study, we examined the utility of using a panel of seven cancer cell lines, NCI-7 Cell Line Panel, as a reference material for mass spectrometric analysis of human proteome. Our results showed that not only is the NCI-7 material suitable for benchmarking laboratory sample preparation methods, but also NCI-7 sample generation is highly reproducible at both the global and phosphoprotein levels. In addition, the predicted genomic and experimental coverage of the NCI-7 proteome suggests the NCI-7 material may also have applications as a universal standard proteomic reference.', 'publicationId': '45de306b-a861-11eb-9f63-0a2b20c3a9e9', 'title': 'Evaluation of NCI-7 Cell Line Panel as a Reference Material for Clinical Proteomics'}, {'abstract': 'Activation of PI3K signaling is frequently observed in triple-negative breast cancer (TNBC), yet PI3K inhibitors have shown limited clinical activity. To investigate intrinsic and adaptive mechanisms of resistance, we analyzed a panel of patient-derived xenograft models of TNBC with varying responsiveness to buparlisib, a pan-PI3K inhibitor. In a subset of patient-derived xenografts, resistance was associated with incomplete inhibition of PI3K signaling and upregulated MAPK/MEK signaling in response to buparlisib. Outlier phosphoproteome and kinome analyses identified novel candidates functionally important to buparlisib resistance, including NEK9 and MAP2K4. Knockdown of NEK9 or MAP2K4 reduced both baseline and feedback MAPK/MEK signaling and showed synthetic lethality with buparlisib in vitro. A complex in/del frameshift in PIK3CA decreased sensitivity to buparlisib via NEK9/MAP2K4√¢‚Ç¨‚Äúdependent mechanisms. In summary, our study supports a role for NEK9 and MAP2K4 in mediating buparlisib resistance and demonstrates the value of unbiased omic analyses in uncovering resistance mechanisms to targeted therapy.', 'publicationId': 'd37418f1-b87e-11eb-9f63-0a2b20c3a9e9', 'title': 'Mass Spectrometry-Based Proteomics Reveals Potential Roles of NEK9 and MAP2K4 in Resistance to PI3K Inhibition in Triple-Negative Breast Cancers'}, {'abstract': 'Oral squamous cell carcinoma is a prominent cancer worldwide, particularly in Taiwan. By integrating omics analyses in 50 matched samples, we uncover in Taiwanese patients a predominant mutation signature associated with cytidine deaminase APOBEC, which correlates with the upregulation of APOBEC3A expression in the APOBEC3 gene cluster at 22q13. APOBEC3A expression is significantly higher in tumors carrying APOBEC3B-deletion allele(s). High-level APOBEC3A expression is associated with better overall survival, especially among patients carrying APOBEC3B-deletion alleles, as examined in a second cohort (n = 188; p = 0.004). The frequency of APOBEC3B-deletion alleles is ~50% in 143 genotyped oral squamous cell carcinoma -Taiwan samples (27A3B −/−:89A3B +/−:27A3B +/+), compared to the 5.8% found in 314 OSCC-TCGA samples. We thus report a frequent APOBEC mutational profile, which relates to a APOBEC3B-deletion germline polymorphism in Taiwanese oral squamous cell carcinoma that impacts expression of APOBEC3A, and is shown to be of clinical prognostic relevance. Our finding might be recapitulated by genomic studies in other cancer types.', 'publicationId': 'd2b6c2b7-4370-11eb-bcbc-0abc48da233b', 'title': 'APOBEC3A is an oral cancer prognostic biomarker in Taiwanese carriers of an APOBEC deletion polymorphism'}, {'abstract': 'Recent advances in mass spectrometry (MS) have enabled extensive analysis of cancer proteomes. Here, we employed quantitative proteomics to profile protein expression across 24 breast cancer patient-derived xenograft (PDX) models. Integrated proteogenomic analysis shows positive correlation between expression measurements from transcriptomic and proteomic analyses; further, gene expression-based intrinsic subtypes are largely re-capitulated using non-stromal protein markers. Proteogenomic analysis also validates a number of predicted genomic targets in multiple receptor tyrosine kinases. However, several protein/phosphoprotein events such as overexpression of AKT proteins and ARAF, BRAF, HSP90AB1 phosphosites are not readily explainable by genomic analysis, suggesting that druggable translational and/or post-translational regulatory events may be uniquely diagnosed by MS. Drug treatment experiments targeting HER2 and components of the PI3K pathway supported proteogenomic response predictions in seven xenograft models. Our study demonstrates that MS-based proteomics can identify therapeutic targets and highlights the potential of PDX drug response evaluation to annotate MS-based pathway activities.', 'publicationId': '718f77fb-b329-11eb-9f63-0a2b20c3a9e9', 'title': 'Proteogenomic integration reveals therapeutic targets in breast cancer xenografts'}, {'abstract': 'To provide a detailed analysis of the molecular components and underlying mechanisms associated with ovarian cancer, we performed a comprehensive mass-spectrometry-based proteomic characterization of 174 ovarian tumors previously analyzed by The Cancer Genome Atlas (TCGA), of which 169 were high-grade serous carcinomas (HGSCs). Integrating our proteomic measurements with the genomic data yielded a number of insights into disease, such as how different copy-number alternations influence the proteome, the proteins associated with chromosomal instability, the sets of signaling pathways that diverse genome rearrangements converge on, and the ones most associated with short overall survival. Specific protein acetylations associated with homologous recombination deficiency suggest a potential means for stratifying patients for therapy. In addition to providing a valuable resource, these findings provide a view of how the somatic genome drives the cancer proteome and associations between protein and post-translational modification levels and clinical outcomes in HGSC.', 'publicationId': '44e2e658-89c7-11e8-bcf1-0a2705229b82', 'title': 'Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer'}, {'abstract': 'Somatic mutations have been extensively characterized in breast cancer, but the effects of these genetic alterations on the proteomic landscape remain poorly understood. Here we describe quantitative mass-spectrometry-based proteomic and phosphoproteomic analyses of 105 genomically annotated breast cancers, of which 77 provided high-quality data. Integrated analyses provided insights into the somatic cancer genome including the consequences of chromosomal loss, such as the 5q deletion characteristic of basal-like breast cancer. Interrogation of the 5q trans-effects against the Library of Integrated Network-based Cellular Signatures, connected loss of CETN3 and SKP1 to elevated expression of epidermal growth factor receptor (EGFR), and SKP1 loss also to increased SRC tyrosine kinase. Global proteomic data confirmed a stromal-enriched group of proteins in addition to basal and luminal clusters, and pathway analysis of the phosphoproteome identified a G-protein-coupled receptor cluster that was not readily identified at the mRNA level. In addition to ERBB2, other amplicon-associated highly phosphorylated kinases were identified, including CDK12, PAK1, PTK2, RIPK2 and TLK2. We demonstrate that proteogenomic analysis of breast cancer elucidates the functional consequences of somatic mutations, narrows candidate nominations for driver genes within large deletions and amplified regions, and identifies therapeutic targets.', 'publicationId': '44e2e766-89c7-11e8-bcf1-0a2705229b82', 'title': 'Proteogenomics connects somatic mutations to signalling in breast cancer'}, {'abstract': 'Understanding proteomic differences underlying the different phenotypic classes of colon and rectal carcinoma is important and may eventually lead to a better assessment of clinical behavior of these cancers. We here present a comprehensive description of the proteomic data obtained from 90 colon and rectal carcinomas previously subjected to genomic analysis by The Cancer Genome Atlas (TCGA). Here, the primary instrument files and derived secondary data files are compiled and presented in forms that will allow further analyses of the biology of colon and rectal carcinoma. We also discuss new challenges in processing these large proteomic datasets for relevant proteins and protein variants.', 'publicationId': '44e2e50c-89c7-11e8-bcf1-0a2705229b82', 'title': 'Proteomic analysis of colon and rectal carcinoma using standard and customized databases'}, {'abstract': 'Clinical specimens are each inherently unique, limited and nonrenewable. Small samples such as tissue biopsies are often completely consumed after a limited number of analyses. Here we present a method that enables fast and reproducible conversion of a small amount of tissue (approximating the quantity obtained by a biopsy) into a single, permanent digital file representing the mass spectrometry (MS)-measurable proteome of the sample. The method combines pressure cycling technology (PCT) and sequential window acquisition of all theoretical fragment ion spectra (SWATH)-MS. The resulting proteome maps can be analyzed, re-analyzed, compared and mined in silico to detect and quantify specific proteins across multiple samples. We used this method to process and convert 18 biopsy samples from nine patients with renal cell carcinoma into SWATH-MS fragment ion maps. From these proteome maps we detected and quantified more than 2,000 proteins with a high degree of reproducibility across all samples. The measured proteins clearly distinguished tumorous kidney tissues from healthy tissues and differentiated distinct histomorphological kidney cancer subtypes.', 'publicationId': '18a56fbd-10d5-11e9-afb9-0a9c39d33490', 'title': 'Rapid mass spectrometric conversion of tissue biopsy samples into permanent quantitative digital proteome maps'}, {'abstract': \"Extensive genomic characterization of human cancers presents the problem of inference from genomic abnormalities to cancer phenotypes. To address this problem, we analysed proteomes of colon and rectal tumours characterized previously by The Cancer Genome Atlas (TCGA) and perform integrated proteogenomic analyses. Somatic variants displayed reduced protein abundance compared to germline variants. Messenger RNA transcript abundance did not reliably predict protein abundance differences between tumours. Proteomics identified five proteomic subtypes in the TCGA cohort, two of which overlapped with the TCGA 'microsatellite instability/CpG island methylation phenotype' transcriptomic subtype, but had distinct mutation, methylation and protein expression patterns associated with different clinical outcomes. Although copy number alterations showed strong cis- and trans-effects on mRNA abundance, relatively few of these extend to the protein level. Thus, proteomics data enabled prioritization of candidate driver genes. The chromosome 20q amplicon was associated with the largest global changes at both mRNA and protein levels; proteomics data highlighted potential 20q candidates, including HNF4A (hepatocyte nuclear factor 4, alpha), TOMM34 (translocase of outer mitochondrial membrane 34) and SRC (SRC proto-oncogene, non-receptor tyrosine kinase). Integrated proteogenomic analysis provides functional context to interpret genomic abnormalities and affords a new paradigm for understanding cancer biology.\", 'publicationId': '44e2e0ce-89c7-11e8-bcf1-0a2705229b82', 'title': 'Proteogenomic characterization of human colon and rectal cancer'}, {'abstract': 'Acute myeloid leukemia is a poor-prognosis cancer commonly stratified by genetic aberrations, but these mutations are often heterogeneous and fail to consistently predict therapeutic response. Here, we combine transcriptomic, proteomic, and phosphoproteomic datasets with ex vivo drug sensitivity data to help understand the underlying pathophysiology of AML beyond mutations. We measure the proteome and phosphoproteome of 210 patients and combine them with genomic and transcriptomic measurements to identify four proteogenomic subtypes that complement existing genetic subtypes. We build a predictor to classify samples into subtypes and map them to a \"landscape\" that identifies specific drug response patterns. We then build a drug response prediction model to identify drugs that target distinct subtypes and validate our findings on cell lines representing various stages of quizartinib resistance. Our results show how multiomics data together with drug sensitivity data can inform therapy stratification and drug combinations in AML.', 'publicationId': '60806ae5-c064-11ee-babf-0aaebe6fe27b', 'title': 'Mapping the proteogenomic landscape enables prediction of drug response in acute myeloid leukemia'}, {'abstract': 'Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal cancer types, partly because it is frequently identified at an advanced stage, when surgery is no longer feasible. Therefore, early detection using minimally invasive methods such as blood tests may improve outcomes. However, studies to discover molecular signatures for the early detection of PDAC using blood tests have only been marginally successful. In the current study, a quantitative glycoproteomic approach via data-independent acquisition mass spectrometry was utilized to detect glycoproteins in 29 patient-matched PDAC tissues and sera. A total of 892 N-linked glycopeptides originating from 141 glycoproteins had PDAC-associated changes beyond normal variation. We further evaluated the specificity of these serum-detectable glycoproteins by comparing their abundance in 53 independent PDAC patient sera and 65 cancer-free controls. The PDAC tissue-associated glycoproteins we have identified represent an inventory of serum-detectable PDAC-associated glycoproteins as candidate biomarkers that can be potentially used for the detection of PDAC using blood tests.', 'publicationId': '86134adf-072f-11ef-b92f-0a200668d9ef', 'title': 'Detection of Pancreatic Ductal Adenocarcinoma-Associated Proteins in Serum'}, {'abstract': 'Clear cell renal cell carcinoma (ccRCC), a common form of RCC, is responsible for the high mortality rate of kidney cancer. Dysregulations of glycoproteins have been shown to associate with ccRCC. However, the molecular mechanism has not been well characterized. Here, a comprehensive glycoproteomic analysis is conducted using 103 tumors and 80 paired normal adjacent tissues. Altered glycosylation enzymes and corresponding protein glycosylation are observed, while two of the major ccRCC mutations, BAP1 and PBRM1, show distinct glycosylation profiles. Additionally, inter-tumor heterogeneity and cross-correlation between glycosylation and phosphorylation are observed. The relation of glycoproteomic features to genomic, transcriptomic, proteomic, and phosphoproteomic changes shows the role of glycosylation in ccRCC development with potential for therapeutic interventions. This study reports a large-scale tandem mass tag (TMT)-based quantitative glycoproteomic analysis of ccRCC that can serve as a valuable resource for the community.', 'publicationId': '129b3284-e36c-11ed-b3ec-0a7bf0b2090f', 'title': 'Integrated glycoproteomic characterization of clear cell renal cell carcinoma'}, {'abstract': 'The identification of differentially expressed tumor-associated proteins and genomic alterations driving neoplasia is critical in the development of clinical assays to detect cancers and forms the foundation for understanding cancer biology. One of the challenges in the analysis of pancreatic ductal adenocarcinoma (PDAC) is the low neoplastic cellularity and heterogeneous composition of bulk tumors. To enrich neoplastic cells from bulk tumor tissue, coring, and laser microdissection (LMD) sampling techniques have been employed. In this study, we assessed the protein and KRAS mutation changes associated with samples obtained by these enrichment techniques and evaluated the fraction of neoplastic cells in PDAC for proteomic and genomic analyses.', 'publicationId': 'bef019a4-50b1-11ed-a6f8-0a79dd92157b', 'title': 'Neoplastic cell enrichment of tumor tissues using coring and laser microdissection for proteomic and genomic analyses of pancreatic ductal adenocarcinoma'}, {'abstract': 'Microscaled proteogenomics was deployed to probe the molecular basis for differential response to neoadjuvant carboplatin and docetaxel combination chemotherapy for triple-negative breast cancer (TNBC). Proteomic analyses of pre-treatment patient biopsies uniquely revealed metabolic pathways, including oxidative phosphorylation, adipogenesis and fatty acid metabolism, that were resistance-associated. Both proteomics and transcriptomics revealed sensitivity was marked by elevation of DNA repair, E2F targets, G2M checkpoint, interferon-gamma signaling and immune checkpoint components. Proteogenomic analyses of somatic copy number aberrations identified a resistance-associated 19q13.31-33 deletion where LIG1, POLD1 and XRCC1 are located. In orthogonal datasets, LIG1 (DNA ligase I) gene deletion and/or low mRNA expression levels were associated with lack of pathological complete response, higher chromosomal instability (CIN) and poor prognosis in TNBC, as well as carboplatin-selective resistance in TNBC pre-clinical models. Hemizygous loss of LIG1 was also associated with higher CIN and poor prognosis in other cancer types, demonstrating broader clinical implications.', 'publicationId': '76a651c5-23da-11ed-adcf-0eeb2b18e995', 'title': 'Proteogenomic markers of chemotherapy resistance and response in triple negative breast cancer'}, {'abstract': 'Lung squamous cell carcinoma (LSCC) remains a leading cause of cancer death with few therapeutic options. We characterized the proteogenomic landscape of LSCC, providing a deeper exposition of LSCC biology with potential therapeutic implications. We identify NSD3 as an alternative driver in FGFR1-amplified tumors and low-p63 tumors overexpressing the therapeutic target survivin. SOX2 is considered undruggable, but our analyses provide rationale for exploring chromatin modifiers such as LSD1 and EZH2 to target SOX2-overexpressing tumors. Our data support complex regulation of metabolic pathways by crosstalk between post-translational modifications including ubiquitylation. Numerous immune-related proteogenomic observations suggest directions for further investigation. Proteogenomic dissection of CDKN2A mutations argue for more nuanced assessment of RB1 protein expression and phosphorylation before declaring CDK4/6 inhibition unsuccessful. Finally, triangulation between LSCC, LUAD, and HNSCC identified both unique and common therapeutic vulnerabilities. These observations and proteogenomics data resources may guide research into the biology and treatment of LSCC.', 'publicationId': 'ebbe2318-f601-11eb-9f63-0a2b20c3a9e9', 'title': 'A proteogenomic portrait of lung squamous cell carcinoma'}, {'abstract': 'Glioblastoma (GBM) is the most aggressive nervous system cancer. Understanding its molecular pathogenesis is crucial to improving diagnosis and treatment. Integrated analysis of genomic, proteomic, post-translational modification and metabolomic data on 99 treatment-naive GBMs provides insights to GBM biology. We identify key phosphorylation events (e.g., phosphorylated PTPN11 and PLCG1) as potential switches mediating oncogenic pathway activation, as well as potential targets for EGFR-, TP53-, and RB1-altered tumors. Immune subtypes with distinct immune cell types are discovered using bulk omics methodologies, validated by snRNA-seq, and correlated with specific expression and histone acetylation patterns. Histone H2B acetylation in classical-like and immune-low GBM is driven largely by BRDs, CREBBP, and EP300. Integrated metabolomic and proteomic data identify specific lipid distributions across subtypes and distinct global metabolic changes in IDH-mutated tumors. This work highlights biological relationships that could contribute to stratification of GBM patients for more effective treatment.', 'publicationId': 'ff35c89b-7071-11eb-9f63-0a2b20c3a9e9', 'title': 'Proteogenomic and metabolomic characterization of human glioblastoma'}, {'abstract': 'Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive cancer with poor patient survival. Toward understanding the underlying molecular alterations that drive PDAC oncogenesis, we conducted comprehensive proteogenomic analysis of 140 pancreatic cancers, 67 normal adjacent tissues, and 9 normal pancreatic ductal tissues. Proteomic, phosphoproteomic, and glycoproteomic analyses were used to characterize proteins and their modifications. In addition, whole-genome sequencing, whole-exome sequencing, methylation, RNA sequencing (RNA-seq), and microRNA sequencing (miRNA-seq) were performed on the same tissues to facilitate an integrated proteogenomic analysis and determine the impact of genomic alterations on protein expression, signaling pathways, and post-translational modifications. To ensure robust downstream analyses, tumor neoplastic cellularity was assessed via multiple orthogonal strategies using molecular features and verified via pathological estimation of tumor cellularity based on histological review. This integrated proteogenomic characterization of PDAC will serve as a valuable resource for the community, paving the way for early detection and identification of novel therapeutic targets.', 'publicationId': 'ffe3fbc9-165b-11ec-b712-0a4e2186f121', 'title': 'Proteogenomic Characterization of Pancreatic Ductal Adenocarcinoma'}, {'abstract': 'The integration of mass spectrometry-based proteomics with next-generation DNA and RNA sequencing profiles tumors more comprehensively. Herein this “proteogenomic” approach was applied to 122 treatment-naive primary breast cancers purposely accrued to preserve post-translational modifications, including protein phosphorylation and acetylation. Proteogenomics challenged standard breast cancer diagnoses, provided detailed analysis of the ERBB2 amplicon, defined tumor subsets that could benefit from immune checkpoint therapy and allowed more accurate assessment of Rb status for the prediction of CDK4/6 inhibitor responsiveness. Phosphoproteomic profiles uncovered novel associations between tumor suppressor loss and targetable kinases. Acetylproteome analysis highlighted acetylation on key nuclear proteins involved in the DNA damage response and revealed cross-talk between cytoplasmic and mitochondrial acetylation and metabolism. Our results underscore the potential of proteogenomics for the clinical investigation of breast cancer through more accurate annotation of targetable pathways and biological features of this remarkably heterogeneous malignancy.', 'publicationId': '90f114bd-2b57-11eb-93d9-0adc6d0c0251', 'title': 'Proteogenomic Landscape of Breast Cancer Tumorigenesis and Targeted Therapy'}, {'abstract': 'We undertook a comprehensive proteogenomic characterization of 95 prospectively collected endometrial carcinomas, comprising 83 endometrioid and 12 serous tumors. This analysis revealed possible new consequences of perturbations to the p53 and Wnt/Î²-catenin pathways, identified a potential role for circRNAs in the epithelial-mesenchymal transition, and provided new information about proteomic markers of clinical and genomic tumor subgroups, including relationships to known druggable pathways. An extensive genome-wide acetylation survey yielded insights into regulatory mechanisms linking Wnt signaling and histone acetylation. We also characterized aspects of the tumor immune landscape, including immunogenic alterations, neoantigens, common cancer/testis antigens, and the immune microenvironment, all of which can inform immunotherapy decisions. Collectively, our multi-omic analyses provide a valuable resource for researchers and clinicians, identify new molecular associations of potential mechanistic significance in the development of endometrial cancers, and suggest novel approaches for identifying potential therapeutic targets.', 'publicationId': '161370ab-4f57-11ea-b5ed-0a7ef35af225', 'title': 'Proteogenomic Characterization of Endometrial Carcinoma'}, {'abstract': 'Methodologies that facilitate high-throughput proteomic analysis are a key step toward moving proteome investigations into clinical translation. Data independent acquisition (DIA) has potential as a high-throughput analytical method due to the reduced time needed for sample analysis, as well as its highly quantitative accuracy. However, a limiting feature of DIA methods is the sensitivity of detection of low abundant proteins and depth of coverage, which other mass spectrometry approaches address by two-dimensional fractionation (2D) to reduce sample complexity during data acquisition. In this study, we developed a 2D-DIA method intended for rapid- and deeper-proteome analysis compared to conventional 1D-DIA analysis. First, we characterized 96 individual fractions obtained from the protein standard, NCI-7, using a data-dependent approach (DDA), identifying a total of 151,366 unique peptides from 11,273 protein groups. We observed that the majority of the proteins can be identified from just a few selected fractions. By performing an optimization analysis, we identified six fractions with high peptide number and uniqueness that can account for 80% of the proteins identified in the entire experiment. These selected fractions were combined into a single sample which was then subjected to DIA (referred to as 2D-DIA) quantitative analysis. Furthermore, improved DIA quantification was achieved using a hybrid spectral library, obtained by combining peptides identified from DDA data with peptides identified directly from the DIA runs with the help of DIA-Umpire. The optimized 2D-DIA method allowed for improved identification and quantification of low abundant proteins compared to conventional unfractionated DIA analysis (1D-DIA). We then applied the 2D-DIA method to profile the proteomes of two breast cancer patient-derived xenograft (PDX) models, quantifying 6,217 and 6,167 unique proteins in basal- and luminal- tumors, respectively. Overall, this study demonstrates the potential of high-throughput quantitative proteomics using a novel 2D-DIA method.', 'publicationId': 'fe52a762-74ad-11ec-bde9-0a4e2186f121', 'title': 'Deep Proteomics Using Two Dimensional Data Independent Acquisition Mass Spectrometry'}, {'abstract': 'Lung cancer in East Asia is characterized by a high percentage of never-smokers, early onset and predominant EGFR mutations. To illuminate the molecular phenotype of this demographically distinct disease, we performed a deep comprehensive proteogenomic study on a prospectively collected cohort in Taiwan, representing early stage, predominantly female, non-smoking lung adenocarcinoma. Integrated genomic, proteomic, and phosphoproteomic analysis delineated the demographically distinct molecular attributes and hallmarks of tumor progression. Mutational signature analysis revealed age- and gender-related mutagenesis mechanisms, characterized by high prevalence of APOBEC mutational signature in younger females and over-representation of environmental carcinogen-like mutational signatures in older females. A proteomics-informed classification distinguished the clinical characteristics of early stage patients with EGFR mutations. Furthermore, integrated protein network analysis revealed the cellular remodeling underpinning clinical trajectories and nominated candidate biomarkers for patient stratification and therapeutic intervention. This multi-omic molecular architecture may help develop strategies for management of early stage never-smoker lung adenocarcinoma.', 'publicationId': '245451cc-c519-11ea-9ab4-0ae52bb11cb7', 'title': 'Proteogenomics of Non-smoking Lung Cancer in East Asia Delineates Molecular Signatures of Pathogenesis and Progression'}, {'abstract': 'In the absence of a dominant driving mutation other than uniformly present TP53 mutations, deeper understanding of the biology driving ovarian high-grade serous cancer (HGSC) requires analysis at a functional level, including post-translational modifications. Comprehensive proteogenomic and phosphoproteomic characterization of 83 prospectively collected ovarian HGSC and appropriate normal precursor tissue samples (fallopian tube) under strict control of ischemia time reveals pathways that significantly differentiate between HGSC and relevant normal tissues in the context of homologous repair deficiency (HRD) status. In addition to confirming key features of HGSC from previous studies, including a potential survival-associated signature and histone acetylation as a marker of HRD, deep phosphoproteomics provides insights regarding the potential role of proliferation-induced replication stress in promoting the characteristic chromosomal instability of HGSC and suggests potential therapeutic targets for use in precision medicine trials.', 'publicationId': '3c25e6ab-1507-11eb-8dc2-0a3dadff3d13', 'title': 'Proteogenomic Characterization of Ovarian HGSC Implicates Mitotic Kinases, Replication Stress in Observed Chromosomal Instability'}, {'abstract': 'Cancer proteogenomics promises new insights into cancer biology and treatment efficacy by integrating genomics, transcriptomics and protein profiling including modifications by mass spectrometry (MS). A critical limitation is sample input requirements that exceed many sources of clinically important material. Here we report a proteogenomics approach for core biopsies using tissue-sparing specimen processing and microscaled proteomics. As a demonstration, we analyze core needle biopsies from ERBB2 positive breast cancers before and 48-72\\u2009h after initiating neoadjuvant trastuzumab-based chemotherapy. We show greater suppression of ERBB2 protein and both ERBB2 and mTOR target phosphosite levels in cases associated with pathological complete response, and identify potential causes of treatment resistance including the absence of ERBB2 amplification, insufficient ERBB2 activity for therapeutic sensitivity despite ERBB2 amplification, and candidate resistance mechanisms including androgen receptor signaling, mucin overexpression and an inactive immune microenvironment. The clinical utility and discovery potential of proteogenomics at biopsy-scale warrants further investigation.', 'publicationId': 'ccd9fc46-d912-11eb-9f63-0a2b20c3a9e9', 'title': 'Microscaled proteogenomic methods for precision oncology'}, {'abstract': 'We report a comprehensive proteogenomics analysis, including whole-genome sequencing, RNA sequencing, and proteomics and phosphoproteomics profiling, of 218 tumors across 7 histological types of childhood brain cancer: low-grade glioma (nÂ = 93), ependymoma (32), high-grade glioma (25), medulloblastoma (22), ganglioglioma (18), craniopharyngioma (16), and atypical teratoid rhabdoid tumor (12). Proteomics data identify common biological themes that span histological boundaries, suggesting that treatments used for one histological type may be applied effectively to other tumors sharing similar proteomics features. Immune landscape characterization reveals diverse tumor microenvironments across and within diagnoses. Proteomics data further reveal functional effects of somatic mutations and copy number variations (CNVs) not evident in transcriptomics data. Kinase-substrate association and co-expression network analysis identify important biological mechanisms of tumorigenesis. This is the first large-scale proteogenomics analysis across traditional histological boundaries to uncover foundational pediatric brain tumor biology and inform rational treatment selection.', 'publicationId': '90c53958-3333-11eb-ab70-0abc48da233b', 'title': 'Integrated Proteogenomic Characterization across Major Histological Types of Pediatric Brain Cancer'}, {'abstract': 'We performed the first proteogenomic study on a prospectively collected colon cancer cohort. Comparative proteomic and phosphoproteomic analysis of paired tumor and normal adjacent tissues produced a catalog of colon cancer-associated proteins and phosphosites, including known and putative new biomarkers, drug targets, and cancer/testis antigens. Proteogenomic integration not only prioritized genomically inferred targets, such as copy-number drivers and mutation-derived neoantigens, but also yielded novel findings. Phosphoproteomics data associated Rb phosphorylation with increased proliferation and decreased apoptosis in colon cancer, which explains why this classical tumor suppressor is amplified in colon tumors and suggests a rationale for targeting Rb phosphorylation in colon cancer. Proteomics identified an association between decreased CD8 T cell infiltration and increased glycolysis in microsatellite instability-high (MSI-H) tumors, suggesting glycolysis as a potential target to overcome the resistance of MSI-H tumors to immune checkpoint blockade. Proteogenomics presents new avenues for biological discoveries and therapeutic development.', 'publicationId': '7257a194-ba5e-11eb-9f63-0a2b20c3a9e9', 'title': 'Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities'}, {'abstract': 'We report proteogenomic analysis of diffuse gastric cancers (GCs) in young populations. Phosphoproteome data elucidated signaling pathways associated with somatic mutations based on mutation-phosphorylation correlations. Moreover, correlations between mRNA and protein abundances provided potential oncogenes and tumor suppressors associated with patient survival. Furthermore, integrated clustering of mRNA, protein, phosphorylation, and N-glycosylation data identified four subtypes of diffuse GCs. Distinguishing these subtypes was possible by proteomic data. Four subtypes were associated with proliferation, immune response, metabolism, and invasion, respectively; and associations of the subtypes with immune- and invasion-related pathways were identified mainly by phosphorylation and N-glycosylation data. Therefore, our proteogenomic analysis provides additional information beyond genomic analyses, which can improve understanding of cancer biology and patient stratification in diffuse GCs.', 'publicationId': 'f2121b29-7828-11ea-b5ed-0a7ef35af225', 'title': 'Proteogenomic Characterization of Human Early-Onset Gastric Cancer'}, {'abstract': 'Here we present an optimized workflow for global proteome and phosphoproteome analysis of tissues or cell lines that uses isobaric tags (TMT (tandem mass tags)-10) for multiplexed analysis and relative quantification, and provides 3√ó higher throughput than iTRAQ (isobaric tags for absolute and relative quantification)-4-based methods with high intra- and inter-laboratory reproducibility. The workflow was systematically characterized and benchmarked across three independent laboratories using two distinct breast cancer subtypes from patient-derived xenograft models to enable assessment of proteome and phosphoproteome depth and quantitative reproducibility. Each plex consisted of ten samples, each being 300 Œºg of peptide derived from <50 mg of wet-weight tissue. Of the 10,000 proteins quantified per sample, we could distinguish 7,700 human proteins derived from tumor cells and 3100 mouse proteins derived from the surrounding stroma and blood. The maximum deviation across replicates and laboratories was <7%, and the inter-laboratory correlation for TMT ratio-based comparison of the two breast cancer subtypes was r > 0.88. The maximum deviation for the phosphoproteome coverage was <24% across laboratories, with an average of >37,000 quantified phosphosites per sample and differential quantification correlations of r > 0.72. The full procedure, including sample processing and data generation, can be completed within 10 d for ten tissue samples, and 100 samples can be analyzed in ~4 months using a single LC-MS/MS instrument. The high quality, depth, and reproducibility of the data obtained both within and across laboratories should enable new biological insights to be obtained from mass spectrometry-based proteomics analyses of cells and tissues together with proteogenomic data integration.', 'publicationId': '80459042-c3d7-11eb-9f63-0a2b20c3a9e9', 'title': 'Reproducible workflow for multiplexed deep-scale proteome and phosphoproteome analysis of tumor tissues by liquid chromatography-mass spectrometry'}, {'abstract': 'Reference materials are vital to benchmarking the reproducibility of clinical tests and essential for monitoring laboratory performance for clinical proteomics. The reference material utilized for mass spectrometric analysis of the human proteome would ideally contain enough proteins to be suitably representative of the human proteome, as well as exhibit a stable protein composition in different batches of sample regeneration. Previously, The Clinical Proteomic Tumor Analysis Consortium (CPTAC) utilized a PDX-derived comparative reference (CompRef) materials for the longitudinal assessment of proteomic performance; however, inherent drawbacks of PDX-derived material, including extended time needed to grow tumors and high level of expertise needed, have resulted in efforts to identify a new source of CompRef material. In this study, we examined the utility of using a panel of seven cancer cell lines, NCI-7 Cell Line Panel, as a reference material for mass spectrometric analysis of human proteome. Our results showed that not only is the NCI-7 material suitable for benchmarking laboratory sample preparation methods, but also NCI-7 sample generation is highly reproducible at both the global and phosphoprotein levels. In addition, the predicted genomic and experimental coverage of the NCI-7 proteome suggests the NCI-7 material may also have applications as a universal standard proteomic reference.', 'publicationId': '45de306b-a861-11eb-9f63-0a2b20c3a9e9', 'title': 'Evaluation of NCI-7 Cell Line Panel as a Reference Material for Clinical Proteomics'}, {'abstract': 'Activation of PI3K signaling is frequently observed in triple-negative breast cancer (TNBC), yet PI3K inhibitors have shown limited clinical activity. To investigate intrinsic and adaptive mechanisms of resistance, we analyzed a panel of patient-derived xenograft models of TNBC with varying responsiveness to buparlisib, a pan-PI3K inhibitor. In a subset of patient-derived xenografts, resistance was associated with incomplete inhibition of PI3K signaling and upregulated MAPK/MEK signaling in response to buparlisib. Outlier phosphoproteome and kinome analyses identified novel candidates functionally important to buparlisib resistance, including NEK9 and MAP2K4. Knockdown of NEK9 or MAP2K4 reduced both baseline and feedback MAPK/MEK signaling and showed synthetic lethality with buparlisib in vitro. A complex in/del frameshift in PIK3CA decreased sensitivity to buparlisib via NEK9/MAP2K4√¢‚Ç¨‚Äúdependent mechanisms. In summary, our study supports a role for NEK9 and MAP2K4 in mediating buparlisib resistance and demonstrates the value of unbiased omic analyses in uncovering resistance mechanisms to targeted therapy.', 'publicationId': 'd37418f1-b87e-11eb-9f63-0a2b20c3a9e9', 'title': 'Mass Spectrometry-Based Proteomics Reveals Potential Roles of NEK9 and MAP2K4 in Resistance to PI3K Inhibition in Triple-Negative Breast Cancers'}, {'abstract': 'Oral squamous cell carcinoma is a prominent cancer worldwide, particularly in Taiwan. By integrating omics analyses in 50 matched samples, we uncover in Taiwanese patients a predominant mutation signature associated with cytidine deaminase APOBEC, which correlates with the upregulation of APOBEC3A expression in the APOBEC3 gene cluster at 22q13. APOBEC3A expression is significantly higher in tumors carrying APOBEC3B-deletion allele(s). High-level APOBEC3A expression is associated with better overall survival, especially among patients carrying APOBEC3B-deletion alleles, as examined in a second cohort (n = 188; p = 0.004). The frequency of APOBEC3B-deletion alleles is ~50% in 143 genotyped oral squamous cell carcinoma -Taiwan samples (27A3B −/−:89A3B +/−:27A3B +/+), compared to the 5.8% found in 314 OSCC-TCGA samples. We thus report a frequent APOBEC mutational profile, which relates to a APOBEC3B-deletion germline polymorphism in Taiwanese oral squamous cell carcinoma that impacts expression of APOBEC3A, and is shown to be of clinical prognostic relevance. Our finding might be recapitulated by genomic studies in other cancer types.', 'publicationId': 'd2b6c2b7-4370-11eb-bcbc-0abc48da233b', 'title': 'APOBEC3A is an oral cancer prognostic biomarker in Taiwanese carriers of an APOBEC deletion polymorphism'}, {'abstract': 'Recent advances in mass spectrometry (MS) have enabled extensive analysis of cancer proteomes. Here, we employed quantitative proteomics to profile protein expression across 24 breast cancer patient-derived xenograft (PDX) models. Integrated proteogenomic analysis shows positive correlation between expression measurements from transcriptomic and proteomic analyses; further, gene expression-based intrinsic subtypes are largely re-capitulated using non-stromal protein markers. Proteogenomic analysis also validates a number of predicted genomic targets in multiple receptor tyrosine kinases. However, several protein/phosphoprotein events such as overexpression of AKT proteins and ARAF, BRAF, HSP90AB1 phosphosites are not readily explainable by genomic analysis, suggesting that druggable translational and/or post-translational regulatory events may be uniquely diagnosed by MS. Drug treatment experiments targeting HER2 and components of the PI3K pathway supported proteogenomic response predictions in seven xenograft models. Our study demonstrates that MS-based proteomics can identify therapeutic targets and highlights the potential of PDX drug response evaluation to annotate MS-based pathway activities.', 'publicationId': '718f77fb-b329-11eb-9f63-0a2b20c3a9e9', 'title': 'Proteogenomic integration reveals therapeutic targets in breast cancer xenografts'}, {'abstract': 'To provide a detailed analysis of the molecular components and underlying mechanisms associated with ovarian cancer, we performed a comprehensive mass-spectrometry-based proteomic characterization of 174 ovarian tumors previously analyzed by The Cancer Genome Atlas (TCGA), of which 169 were high-grade serous carcinomas (HGSCs). Integrating our proteomic measurements with the genomic data yielded a number of insights into disease, such as how different copy-number alternations influence the proteome, the proteins associated with chromosomal instability, the sets of signaling pathways that diverse genome rearrangements converge on, and the ones most associated with short overall survival. Specific protein acetylations associated with homologous recombination deficiency suggest a potential means for stratifying patients for therapy. In addition to providing a valuable resource, these findings provide a view of how the somatic genome drives the cancer proteome and associations between protein and post-translational modification levels and clinical outcomes in HGSC.', 'publicationId': '44e2e658-89c7-11e8-bcf1-0a2705229b82', 'title': 'Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer'}, {'abstract': 'Somatic mutations have been extensively characterized in breast cancer, but the effects of these genetic alterations on the proteomic landscape remain poorly understood. Here we describe quantitative mass-spectrometry-based proteomic and phosphoproteomic analyses of 105 genomically annotated breast cancers, of which 77 provided high-quality data. Integrated analyses provided insights into the somatic cancer genome including the consequences of chromosomal loss, such as the 5q deletion characteristic of basal-like breast cancer. Interrogation of the 5q trans-effects against the Library of Integrated Network-based Cellular Signatures, connected loss of CETN3 and SKP1 to elevated expression of epidermal growth factor receptor (EGFR), and SKP1 loss also to increased SRC tyrosine kinase. Global proteomic data confirmed a stromal-enriched group of proteins in addition to basal and luminal clusters, and pathway analysis of the phosphoproteome identified a G-protein-coupled receptor cluster that was not readily identified at the mRNA level. In addition to ERBB2, other amplicon-associated highly phosphorylated kinases were identified, including CDK12, PAK1, PTK2, RIPK2 and TLK2. We demonstrate that proteogenomic analysis of breast cancer elucidates the functional consequences of somatic mutations, narrows candidate nominations for driver genes within large deletions and amplified regions, and identifies therapeutic targets.', 'publicationId': '44e2e766-89c7-11e8-bcf1-0a2705229b82', 'title': 'Proteogenomics connects somatic mutations to signalling in breast cancer'}, {'abstract': 'Understanding proteomic differences underlying the different phenotypic classes of colon and rectal carcinoma is important and may eventually lead to a better assessment of clinical behavior of these cancers. We here present a comprehensive description of the proteomic data obtained from 90 colon and rectal carcinomas previously subjected to genomic analysis by The Cancer Genome Atlas (TCGA). Here, the primary instrument files and derived secondary data files are compiled and presented in forms that will allow further analyses of the biology of colon and rectal carcinoma. We also discuss new challenges in processing these large proteomic datasets for relevant proteins and protein variants.', 'publicationId': '44e2e50c-89c7-11e8-bcf1-0a2705229b82', 'title': 'Proteomic analysis of colon and rectal carcinoma using standard and customized databases'}, {'abstract': 'Clinical specimens are each inherently unique, limited and nonrenewable. Small samples such as tissue biopsies are often completely consumed after a limited number of analyses. Here we present a method that enables fast and reproducible conversion of a small amount of tissue (approximating the quantity obtained by a biopsy) into a single, permanent digital file representing the mass spectrometry (MS)-measurable proteome of the sample. The method combines pressure cycling technology (PCT) and sequential window acquisition of all theoretical fragment ion spectra (SWATH)-MS. The resulting proteome maps can be analyzed, re-analyzed, compared and mined in silico to detect and quantify specific proteins across multiple samples. We used this method to process and convert 18 biopsy samples from nine patients with renal cell carcinoma into SWATH-MS fragment ion maps. From these proteome maps we detected and quantified more than 2,000 proteins with a high degree of reproducibility across all samples. The measured proteins clearly distinguished tumorous kidney tissues from healthy tissues and differentiated distinct histomorphological kidney cancer subtypes.', 'publicationId': '18a56fbd-10d5-11e9-afb9-0a9c39d33490', 'title': 'Rapid mass spectrometric conversion of tissue biopsy samples into permanent quantitative digital proteome maps'}, {'abstract': \"Extensive genomic characterization of human cancers presents the problem of inference from genomic abnormalities to cancer phenotypes. To address this problem, we analysed proteomes of colon and rectal tumours characterized previously by The Cancer Genome Atlas (TCGA) and perform integrated proteogenomic analyses. Somatic variants displayed reduced protein abundance compared to germline variants. Messenger RNA transcript abundance did not reliably predict protein abundance differences between tumours. Proteomics identified five proteomic subtypes in the TCGA cohort, two of which overlapped with the TCGA 'microsatellite instability/CpG island methylation phenotype' transcriptomic subtype, but had distinct mutation, methylation and protein expression patterns associated with different clinical outcomes. Although copy number alterations showed strong cis- and trans-effects on mRNA abundance, relatively few of these extend to the protein level. Thus, proteomics data enabled prioritization of candidate driver genes. The chromosome 20q amplicon was associated with the largest global changes at both mRNA and protein levels; proteomics data highlighted potential 20q candidates, including HNF4A (hepatocyte nuclear factor 4, alpha), TOMM34 (translocase of outer mitochondrial membrane 34) and SRC (SRC proto-oncogene, non-receptor tyrosine kinase). Integrated proteogenomic analysis provides functional context to interpret genomic abnormalities and affords a new paradigm for understanding cancer biology.\", 'publicationId': '44e2e0ce-89c7-11e8-bcf1-0a2705229b82', 'title': 'Proteogenomic characterization of human colon and rectal cancer'}, {'abstract': 'Acute myeloid leukemia is a poor-prognosis cancer commonly stratified by genetic aberrations, but these mutations are often heterogeneous and fail to consistently predict therapeutic response. Here, we combine transcriptomic, proteomic, and phosphoproteomic datasets with ex vivo drug sensitivity data to help understand the underlying pathophysiology of AML beyond mutations. We measure the proteome and phosphoproteome of 210 patients and combine them with genomic and transcriptomic measurements to identify four proteogenomic subtypes that complement existing genetic subtypes. We build a predictor to classify samples into subtypes and map them to a \"landscape\" that identifies specific drug response patterns. We then build a drug response prediction model to identify drugs that target distinct subtypes and validate our findings on cell lines representing various stages of quizartinib resistance. Our results show how multiomics data together with drug sensitivity data can inform therapy stratification and drug combinations in AML.', 'publicationId': '60806ae5-c064-11ee-babf-0aaebe6fe27b', 'title': 'Mapping the proteogenomic landscape enables prediction of drug response in acute myeloid leukemia'}, {'abstract': 'Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal cancer types, partly because it is frequently identified at an advanced stage, when surgery is no longer feasible. Therefore, early detection using minimally invasive methods such as blood tests may improve outcomes. However, studies to discover molecular signatures for the early detection of PDAC using blood tests have only been marginally successful. In the current study, a quantitative glycoproteomic approach via data-independent acquisition mass spectrometry was utilized to detect glycoproteins in 29 patient-matched PDAC tissues and sera. A total of 892 N-linked glycopeptides originating from 141 glycoproteins had PDAC-associated changes beyond normal variation. We further evaluated the specificity of these serum-detectable glycoproteins by comparing their abundance in 53 independent PDAC patient sera and 65 cancer-free controls. The PDAC tissue-associated glycoproteins we have identified represent an inventory of serum-detectable PDAC-associated glycoproteins as candidate biomarkers that can be potentially used for the detection of PDAC using blood tests.', 'publicationId': '86134adf-072f-11ef-b92f-0a200668d9ef', 'title': 'Detection of Pancreatic Ductal Adenocarcinoma-Associated Proteins in Serum'}, {'abstract': 'Clear cell renal cell carcinoma (ccRCC), a common form of RCC, is responsible for the high mortality rate of kidney cancer. Dysregulations of glycoproteins have been shown to associate with ccRCC. However, the molecular mechanism has not been well characterized. Here, a comprehensive glycoproteomic analysis is conducted using 103 tumors and 80 paired normal adjacent tissues. Altered glycosylation enzymes and corresponding protein glycosylation are observed, while two of the major ccRCC mutations, BAP1 and PBRM1, show distinct glycosylation profiles. Additionally, inter-tumor heterogeneity and cross-correlation between glycosylation and phosphorylation are observed. The relation of glycoproteomic features to genomic, transcriptomic, proteomic, and phosphoproteomic changes shows the role of glycosylation in ccRCC development with potential for therapeutic interventions. This study reports a large-scale tandem mass tag (TMT)-based quantitative glycoproteomic analysis of ccRCC that can serve as a valuable resource for the community.', 'publicationId': '129b3284-e36c-11ed-b3ec-0a7bf0b2090f', 'title': 'Integrated glycoproteomic characterization of clear cell renal cell carcinoma'}, {'abstract': 'The identification of differentially expressed tumor-associated proteins and genomic alterations driving neoplasia is critical in the development of clinical assays to detect cancers and forms the foundation for understanding cancer biology. One of the challenges in the analysis of pancreatic ductal adenocarcinoma (PDAC) is the low neoplastic cellularity and heterogeneous composition of bulk tumors. To enrich neoplastic cells from bulk tumor tissue, coring, and laser microdissection (LMD) sampling techniques have been employed. In this study, we assessed the protein and KRAS mutation changes associated with samples obtained by these enrichment techniques and evaluated the fraction of neoplastic cells in PDAC for proteomic and genomic analyses.', 'publicationId': 'bef019a4-50b1-11ed-a6f8-0a79dd92157b', 'title': 'Neoplastic cell enrichment of tumor tissues using coring and laser microdissection for proteomic and genomic analyses of pancreatic ductal adenocarcinoma'}, {'abstract': 'Microscaled proteogenomics was deployed to probe the molecular basis for differential response to neoadjuvant carboplatin and docetaxel combination chemotherapy for triple-negative breast cancer (TNBC). Proteomic analyses of pre-treatment patient biopsies uniquely revealed metabolic pathways, including oxidative phosphorylation, adipogenesis and fatty acid metabolism, that were resistance-associated. Both proteomics and transcriptomics revealed sensitivity was marked by elevation of DNA repair, E2F targets, G2M checkpoint, interferon-gamma signaling and immune checkpoint components. Proteogenomic analyses of somatic copy number aberrations identified a resistance-associated 19q13.31-33 deletion where LIG1, POLD1 and XRCC1 are located. In orthogonal datasets, LIG1 (DNA ligase I) gene deletion and/or low mRNA expression levels were associated with lack of pathological complete response, higher chromosomal instability (CIN) and poor prognosis in TNBC, as well as carboplatin-selective resistance in TNBC pre-clinical models. Hemizygous loss of LIG1 was also associated with higher CIN and poor prognosis in other cancer types, demonstrating broader clinical implications.', 'publicationId': '76a651c5-23da-11ed-adcf-0eeb2b18e995', 'title': 'Proteogenomic markers of chemotherapy resistance and response in triple negative breast cancer'}, {'abstract': 'Lung squamous cell carcinoma (LSCC) remains a leading cause of cancer death with few therapeutic options. We characterized the proteogenomic landscape of LSCC, providing a deeper exposition of LSCC biology with potential therapeutic implications. We identify NSD3 as an alternative driver in FGFR1-amplified tumors and low-p63 tumors overexpressing the therapeutic target survivin. SOX2 is considered undruggable, but our analyses provide rationale for exploring chromatin modifiers such as LSD1 and EZH2 to target SOX2-overexpressing tumors. Our data support complex regulation of metabolic pathways by crosstalk between post-translational modifications including ubiquitylation. Numerous immune-related proteogenomic observations suggest directions for further investigation. Proteogenomic dissection of CDKN2A mutations argue for more nuanced assessment of RB1 protein expression and phosphorylation before declaring CDK4/6 inhibition unsuccessful. Finally, triangulation between LSCC, LUAD, and HNSCC identified both unique and common therapeutic vulnerabilities. These observations and proteogenomics data resources may guide research into the biology and treatment of LSCC.', 'publicationId': 'ebbe2318-f601-11eb-9f63-0a2b20c3a9e9', 'title': 'A proteogenomic portrait of lung squamous cell carcinoma'}, {'abstract': 'Glioblastoma (GBM) is the most aggressive nervous system cancer. Understanding its molecular pathogenesis is crucial to improving diagnosis and treatment. Integrated analysis of genomic, proteomic, post-translational modification and metabolomic data on 99 treatment-naive GBMs provides insights to GBM biology. We identify key phosphorylation events (e.g., phosphorylated PTPN11 and PLCG1) as potential switches mediating oncogenic pathway activation, as well as potential targets for EGFR-, TP53-, and RB1-altered tumors. Immune subtypes with distinct immune cell types are discovered using bulk omics methodologies, validated by snRNA-seq, and correlated with specific expression and histone acetylation patterns. Histone H2B acetylation in classical-like and immune-low GBM is driven largely by BRDs, CREBBP, and EP300. Integrated metabolomic and proteomic data identify specific lipid distributions across subtypes and distinct global metabolic changes in IDH-mutated tumors. This work highlights biological relationships that could contribute to stratification of GBM patients for more effective treatment.', 'publicationId': 'ff35c89b-7071-11eb-9f63-0a2b20c3a9e9', 'title': 'Proteogenomic and metabolomic characterization of human glioblastoma'}, {'abstract': 'Pancreatic ductal adenocarcinoma (PDAC) is a highly aggressive cancer with poor patient survival. Toward understanding the underlying molecular alterations that drive PDAC oncogenesis, we conducted comprehensive proteogenomic analysis of 140 pancreatic cancers, 67 normal adjacent tissues, and 9 normal pancreatic ductal tissues. Proteomic, phosphoproteomic, and glycoproteomic analyses were used to characterize proteins and their modifications. In addition, whole-genome sequencing, whole-exome sequencing, methylation, RNA sequencing (RNA-seq), and microRNA sequencing (miRNA-seq) were performed on the same tissues to facilitate an integrated proteogenomic analysis and determine the impact of genomic alterations on protein expression, signaling pathways, and post-translational modifications. To ensure robust downstream analyses, tumor neoplastic cellularity was assessed via multiple orthogonal strategies using molecular features and verified via pathological estimation of tumor cellularity based on histological review. This integrated proteogenomic characterization of PDAC will serve as a valuable resource for the community, paving the way for early detection and identification of novel therapeutic targets.', 'publicationId': 'ffe3fbc9-165b-11ec-b712-0a4e2186f121', 'title': 'Proteogenomic Characterization of Pancreatic Ductal Adenocarcinoma'}, {'abstract': 'The integration of mass spectrometry-based proteomics with next-generation DNA and RNA sequencing profiles tumors more comprehensively. Herein this “proteogenomic” approach was applied to 122 treatment-naive primary breast cancers purposely accrued to preserve post-translational modifications, including protein phosphorylation and acetylation. Proteogenomics challenged standard breast cancer diagnoses, provided detailed analysis of the ERBB2 amplicon, defined tumor subsets that could benefit from immune checkpoint therapy and allowed more accurate assessment of Rb status for the prediction of CDK4/6 inhibitor responsiveness. Phosphoproteomic profiles uncovered novel associations between tumor suppressor loss and targetable kinases. Acetylproteome analysis highlighted acetylation on key nuclear proteins involved in the DNA damage response and revealed cross-talk between cytoplasmic and mitochondrial acetylation and metabolism. Our results underscore the potential of proteogenomics for the clinical investigation of breast cancer through more accurate annotation of targetable pathways and biological features of this remarkably heterogeneous malignancy.', 'publicationId': '90f114bd-2b57-11eb-93d9-0adc6d0c0251', 'title': 'Proteogenomic Landscape of Breast Cancer Tumorigenesis and Targeted Therapy'}, {'abstract': 'We undertook a comprehensive proteogenomic characterization of 95 prospectively collected endometrial carcinomas, comprising 83 endometrioid and 12 serous tumors. This analysis revealed possible new consequences of perturbations to the p53 and Wnt/Î²-catenin pathways, identified a potential role for circRNAs in the epithelial-mesenchymal transition, and provided new information about proteomic markers of clinical and genomic tumor subgroups, including relationships to known druggable pathways. An extensive genome-wide acetylation survey yielded insights into regulatory mechanisms linking Wnt signaling and histone acetylation. We also characterized aspects of the tumor immune landscape, including immunogenic alterations, neoantigens, common cancer/testis antigens, and the immune microenvironment, all of which can inform immunotherapy decisions. Collectively, our multi-omic analyses provide a valuable resource for researchers and clinicians, identify new molecular associations of potential mechanistic significance in the development of endometrial cancers, and suggest novel approaches for identifying potential therapeutic targets.', 'publicationId': '161370ab-4f57-11ea-b5ed-0a7ef35af225', 'title': 'Proteogenomic Characterization of Endometrial Carcinoma'}, {'abstract': 'Methodologies that facilitate high-throughput proteomic analysis are a key step toward moving proteome investigations into clinical translation. Data independent acquisition (DIA) has potential as a high-throughput analytical method due to the reduced time needed for sample analysis, as well as its highly quantitative accuracy. However, a limiting feature of DIA methods is the sensitivity of detection of low abundant proteins and depth of coverage, which other mass spectrometry approaches address by two-dimensional fractionation (2D) to reduce sample complexity during data acquisition. In this study, we developed a 2D-DIA method intended for rapid- and deeper-proteome analysis compared to conventional 1D-DIA analysis. First, we characterized 96 individual fractions obtained from the protein standard, NCI-7, using a data-dependent approach (DDA), identifying a total of 151,366 unique peptides from 11,273 protein groups. We observed that the majority of the proteins can be identified from just a few selected fractions. By performing an optimization analysis, we identified six fractions with high peptide number and uniqueness that can account for 80% of the proteins identified in the entire experiment. These selected fractions were combined into a single sample which was then subjected to DIA (referred to as 2D-DIA) quantitative analysis. Furthermore, improved DIA quantification was achieved using a hybrid spectral library, obtained by combining peptides identified from DDA data with peptides identified directly from the DIA runs with the help of DIA-Umpire. The optimized 2D-DIA method allowed for improved identification and quantification of low abundant proteins compared to conventional unfractionated DIA analysis (1D-DIA). We then applied the 2D-DIA method to profile the proteomes of two breast cancer patient-derived xenograft (PDX) models, quantifying 6,217 and 6,167 unique proteins in basal- and luminal- tumors, respectively. Overall, this study demonstrates the potential of high-throughput quantitative proteomics using a novel 2D-DIA method.', 'publicationId': 'fe52a762-74ad-11ec-bde9-0a4e2186f121', 'title': 'Deep Proteomics Using Two Dimensional Data Independent Acquisition Mass Spectrometry'}, {'abstract': 'Lung cancer in East Asia is characterized by a high percentage of never-smokers, early onset and predominant EGFR mutations. To illuminate the molecular phenotype of this demographically distinct disease, we performed a deep comprehensive proteogenomic study on a prospectively collected cohort in Taiwan, representing early stage, predominantly female, non-smoking lung adenocarcinoma. Integrated genomic, proteomic, and phosphoproteomic analysis delineated the demographically distinct molecular attributes and hallmarks of tumor progression. Mutational signature analysis revealed age- and gender-related mutagenesis mechanisms, characterized by high prevalence of APOBEC mutational signature in younger females and over-representation of environmental carcinogen-like mutational signatures in older females. A proteomics-informed classification distinguished the clinical characteristics of early stage patients with EGFR mutations. Furthermore, integrated protein network analysis revealed the cellular remodeling underpinning clinical trajectories and nominated candidate biomarkers for patient stratification and therapeutic intervention. This multi-omic molecular architecture may help develop strategies for management of early stage never-smoker lung adenocarcinoma.', 'publicationId': '245451cc-c519-11ea-9ab4-0ae52bb11cb7', 'title': 'Proteogenomics of Non-smoking Lung Cancer in East Asia Delineates Molecular Signatures of Pathogenesis and Progression'}, {'abstract': 'In the absence of a dominant driving mutation other than uniformly present TP53 mutations, deeper understanding of the biology driving ovarian high-grade serous cancer (HGSC) requires analysis at a functional level, including post-translational modifications. Comprehensive proteogenomic and phosphoproteomic characterization of 83 prospectively collected ovarian HGSC and appropriate normal precursor tissue samples (fallopian tube) under strict control of ischemia time reveals pathways that significantly differentiate between HGSC and relevant normal tissues in the context of homologous repair deficiency (HRD) status. In addition to confirming key features of HGSC from previous studies, including a potential survival-associated signature and histone acetylation as a marker of HRD, deep phosphoproteomics provides insights regarding the potential role of proliferation-induced replication stress in promoting the characteristic chromosomal instability of HGSC and suggests potential therapeutic targets for use in precision medicine trials.', 'publicationId': '3c25e6ab-1507-11eb-8dc2-0a3dadff3d13', 'title': 'Proteogenomic Characterization of Ovarian HGSC Implicates Mitotic Kinases, Replication Stress in Observed Chromosomal Instability'}, {'abstract': 'Cancer proteogenomics promises new insights into cancer biology and treatment efficacy by integrating genomics, transcriptomics and protein profiling including modifications by mass spectrometry (MS). A critical limitation is sample input requirements that exceed many sources of clinically important material. Here we report a proteogenomics approach for core biopsies using tissue-sparing specimen processing and microscaled proteomics. As a demonstration, we analyze core needle biopsies from ERBB2 positive breast cancers before and 48-72\\u2009h after initiating neoadjuvant trastuzumab-based chemotherapy. We show greater suppression of ERBB2 protein and both ERBB2 and mTOR target phosphosite levels in cases associated with pathological complete response, and identify potential causes of treatment resistance including the absence of ERBB2 amplification, insufficient ERBB2 activity for therapeutic sensitivity despite ERBB2 amplification, and candidate resistance mechanisms including androgen receptor signaling, mucin overexpression and an inactive immune microenvironment. The clinical utility and discovery potential of proteogenomics at biopsy-scale warrants further investigation.', 'publicationId': 'ccd9fc46-d912-11eb-9f63-0a2b20c3a9e9', 'title': 'Microscaled proteogenomic methods for precision oncology'}, {'abstract': 'We report a comprehensive proteogenomics analysis, including whole-genome sequencing, RNA sequencing, and proteomics and phosphoproteomics profiling, of 218 tumors across 7 histological types of childhood brain cancer: low-grade glioma (nÂ = 93), ependymoma (32), high-grade glioma (25), medulloblastoma (22), ganglioglioma (18), craniopharyngioma (16), and atypical teratoid rhabdoid tumor (12). Proteomics data identify common biological themes that span histological boundaries, suggesting that treatments used for one histological type may be applied effectively to other tumors sharing similar proteomics features. Immune landscape characterization reveals diverse tumor microenvironments across and within diagnoses. Proteomics data further reveal functional effects of somatic mutations and copy number variations (CNVs) not evident in transcriptomics data. Kinase-substrate association and co-expression network analysis identify important biological mechanisms of tumorigenesis. This is the first large-scale proteogenomics analysis across traditional histological boundaries to uncover foundational pediatric brain tumor biology and inform rational treatment selection.', 'publicationId': '90c53958-3333-11eb-ab70-0abc48da233b', 'title': 'Integrated Proteogenomic Characterization across Major Histological Types of Pediatric Brain Cancer'}, {'abstract': 'We performed the first proteogenomic study on a prospectively collected colon cancer cohort. Comparative proteomic and phosphoproteomic analysis of paired tumor and normal adjacent tissues produced a catalog of colon cancer-associated proteins and phosphosites, including known and putative new biomarkers, drug targets, and cancer/testis antigens. Proteogenomic integration not only prioritized genomically inferred targets, such as copy-number drivers and mutation-derived neoantigens, but also yielded novel findings. Phosphoproteomics data associated Rb phosphorylation with increased proliferation and decreased apoptosis in colon cancer, which explains why this classical tumor suppressor is amplified in colon tumors and suggests a rationale for targeting Rb phosphorylation in colon cancer. Proteomics identified an association between decreased CD8 T cell infiltration and increased glycolysis in microsatellite instability-high (MSI-H) tumors, suggesting glycolysis as a potential target to overcome the resistance of MSI-H tumors to immune checkpoint blockade. Proteogenomics presents new avenues for biological discoveries and therapeutic development.', 'publicationId': '7257a194-ba5e-11eb-9f63-0a2b20c3a9e9', 'title': 'Proteogenomic Analysis of Human Colon Cancer Reveals New Therapeutic Opportunities'}, {'abstract': 'We report proteogenomic analysis of diffuse gastric cancers (GCs) in young populations. Phosphoproteome data elucidated signaling pathways associated with somatic mutations based on mutation-phosphorylation correlations. Moreover, correlations between mRNA and protein abundances provided potential oncogenes and tumor suppressors associated with patient survival. Furthermore, integrated clustering of mRNA, protein, phosphorylation, and N-glycosylation data identified four subtypes of diffuse GCs. Distinguishing these subtypes was possible by proteomic data. Four subtypes were associated with proliferation, immune response, metabolism, and invasion, respectively; and associations of the subtypes with immune- and invasion-related pathways were identified mainly by phosphorylation and N-glycosylation data. Therefore, our proteogenomic analysis provides additional information beyond genomic analyses, which can improve understanding of cancer biology and patient stratification in diffuse GCs.', 'publicationId': 'f2121b29-7828-11ea-b5ed-0a7ef35af225', 'title': 'Proteogenomic Characterization of Human Early-Onset Gastric Cancer'}, {'abstract': 'Here we present an optimized workflow for global proteome and phosphoproteome analysis of tissues or cell lines that uses isobaric tags (TMT (tandem mass tags)-10) for multiplexed analysis and relative quantification, and provides 3√ó higher throughput than iTRAQ (isobaric tags for absolute and relative quantification)-4-based methods with high intra- and inter-laboratory reproducibility. The workflow was systematically characterized and benchmarked across three independent laboratories using two distinct breast cancer subtypes from patient-derived xenograft models to enable assessment of proteome and phosphoproteome depth and quantitative reproducibility. Each plex consisted of ten samples, each being 300 Œºg of peptide derived from <50 mg of wet-weight tissue. Of the 10,000 proteins quantified per sample, we could distinguish 7,700 human proteins derived from tumor cells and 3100 mouse proteins derived from the surrounding stroma and blood. The maximum deviation across replicates and laboratories was <7%, and the inter-laboratory correlation for TMT ratio-based comparison of the two breast cancer subtypes was r > 0.88. The maximum deviation for the phosphoproteome coverage was <24% across laboratories, with an average of >37,000 quantified phosphosites per sample and differential quantification correlations of r > 0.72. The full procedure, including sample processing and data generation, can be completed within 10 d for ten tissue samples, and 100 samples can be analyzed in ~4 months using a single LC-MS/MS instrument. The high quality, depth, and reproducibility of the data obtained both within and across laboratories should enable new biological insights to be obtained from mass spectrometry-based proteomics analyses of cells and tissues together with proteogenomic data integration.', 'publicationId': '80459042-c3d7-11eb-9f63-0a2b20c3a9e9', 'title': 'Reproducible workflow for multiplexed deep-scale proteome and phosphoproteome analysis of tumor tissues by liquid chromatography-mass spectrometry'}, {'abstract': 'Reference materials are vital to benchmarking the reproducibility of clinical tests and essential for monitoring laboratory performance for clinical proteomics. The reference material utilized for mass spectrometric analysis of the human proteome would ideally contain enough proteins to be suitably representative of the human proteome, as well as exhibit a stable protein composition in different batches of sample regeneration. Previously, The Clinical Proteomic Tumor Analysis Consortium (CPTAC) utilized a PDX-derived comparative reference (CompRef) materials for the longitudinal assessment of proteomic performance; however, inherent drawbacks of PDX-derived material, including extended time needed to grow tumors and high level of expertise needed, have resulted in efforts to identify a new source of CompRef material. In this study, we examined the utility of using a panel of seven cancer cell lines, NCI-7 Cell Line Panel, as a reference material for mass spectrometric analysis of human proteome. Our results showed that not only is the NCI-7 material suitable for benchmarking laboratory sample preparation methods, but also NCI-7 sample generation is highly reproducible at both the global and phosphoprotein levels. In addition, the predicted genomic and experimental coverage of the NCI-7 proteome suggests the NCI-7 material may also have applications as a universal standard proteomic reference.', 'publicationId': '45de306b-a861-11eb-9f63-0a2b20c3a9e9', 'title': 'Evaluation of NCI-7 Cell Line Panel as a Reference Material for Clinical Proteomics'}]}}}\n",
            "\n",
            "Finding similar publications to a given abstract...\n",
            "{'errors': [{'locations': [{'column': 17, 'line': 5}], 'message': 'Unknown argument \"nearText\" on field \"Publication\" of type \"GetObjectsObj\". Did you mean \"nearVector\" or \"nearObject\"?', 'path': None}]}\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!pip install transformers"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "2ORcaaBhEAyj",
        "outputId": "e0a71a48-e3f4-48a9-9150-1878a25d171b"
      },
      "execution_count": 58,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Requirement already satisfied: transformers in /usr/local/lib/python3.10/dist-packages (4.41.2)\n",
            "Requirement already satisfied: filelock in /usr/local/lib/python3.10/dist-packages (from transformers) (3.15.3)\n",
            "Requirement already satisfied: huggingface-hub<1.0,>=0.23.0 in /usr/local/lib/python3.10/dist-packages (from transformers) (0.23.4)\n",
            "Requirement already satisfied: numpy>=1.17 in /usr/local/lib/python3.10/dist-packages (from transformers) (1.25.2)\n",
            "Requirement already satisfied: packaging>=20.0 in /usr/local/lib/python3.10/dist-packages (from transformers) (24.1)\n",
            "Requirement already satisfied: pyyaml>=5.1 in /usr/local/lib/python3.10/dist-packages (from transformers) (6.0.1)\n",
            "Requirement already satisfied: regex!=2019.12.17 in /usr/local/lib/python3.10/dist-packages (from transformers) (2024.5.15)\n",
            "Requirement already satisfied: requests in /usr/local/lib/python3.10/dist-packages (from transformers) (2.32.3)\n",
            "Requirement already satisfied: tokenizers<0.20,>=0.19 in /usr/local/lib/python3.10/dist-packages (from transformers) (0.19.1)\n",
            "Requirement already satisfied: safetensors>=0.4.1 in /usr/local/lib/python3.10/dist-packages (from transformers) (0.4.3)\n",
            "Requirement already satisfied: tqdm>=4.27 in /usr/local/lib/python3.10/dist-packages (from transformers) (4.66.4)\n",
            "Requirement already satisfied: fsspec>=2023.5.0 in /usr/local/lib/python3.10/dist-packages (from huggingface-hub<1.0,>=0.23.0->transformers) (2023.6.0)\n",
            "Requirement already satisfied: typing-extensions>=3.7.4.3 in /usr/local/lib/python3.10/dist-packages (from huggingface-hub<1.0,>=0.23.0->transformers) (4.12.2)\n",
            "Requirement already satisfied: charset-normalizer<4,>=2 in /usr/local/lib/python3.10/dist-packages (from requests->transformers) (3.3.2)\n",
            "Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.10/dist-packages (from requests->transformers) (3.7)\n",
            "Requirement already satisfied: urllib3<3,>=1.21.1 in /usr/local/lib/python3.10/dist-packages (from requests->transformers) (2.0.7)\n",
            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.10/dist-packages (from requests->transformers) (2024.6.2)\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "from transformers import AutoTokenizer, AutoModel\n",
        "import torch\n",
        "\n",
        "# Load the tokenizer and model from Hugging Face's Transformers\n",
        "tokenizer = AutoTokenizer.from_pretrained('bert-base-uncased')\n",
        "model = AutoModel.from_pretrained('bert-base-uncased')\n",
        "\n",
        "# Function to generate embeddings\n",
        "def get_embedding_function(text):\n",
        "    inputs = tokenizer(text, return_tensors=\"pt\", truncation=True, padding=True)\n",
        "    with torch.no_grad():\n",
        "        outputs = model(**inputs)\n",
        "    embeddings = outputs.last_hidden_state.mean(dim=1).squeeze().tolist()\n",
        "    return embeddings\n",
        "\n",
        "text = \"Example abstract text for embedding\"\n",
        "# Generate embeddings\n",
        "embeddings = get_embedding_function(text)\n",
        "\n",
        "publication_object = {\n",
        "    \"pubmed_id\": \"example_pub\",\n",
        "    \"abstract\": text,\n",
        "    \"embeddings\": embeddings\n",
        "}\n",
        "print(\"Generated Embeddings:\", embeddings)\n",
        "\n",
        "try:\n",
        "    client.data_object.create(data_object=publication_object, class_name=\"Publication\")\n",
        "    print(\"Data inserted successfully.\")\n",
        "except Exception as e:\n",
        "    print(f\"Failed to insert data into Weaviate: {str(e)}\")\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "kktKGNctl6OD",
        "outputId": "3a1eb497-9719-4254-b69f-0d7239424574"
      },
      "execution_count": 64,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Generated Embeddings: [-0.004773537162691355, -0.07366238534450531, -0.24704734981060028, -0.22698190808296204, 0.030707087367773056, -0.2608434855937958, -0.024355215951800346, 0.2256229668855667, 0.15915220975875854, -0.17719651758670807, -0.19198530912399292, 0.06828570365905762, -0.13849997520446777, 0.006158149801194668, -0.2831863462924957, 0.1900382786989212, -0.17914994060993195, 0.1722506582736969, 0.051133424043655396, -0.007557927630841732, -0.10050210356712341, -0.0022351464722305536, -0.49910974502563477, 0.16080442070960999, 0.6187970042228699, -0.2574285566806793, 0.003770755371078849, 0.04784826934337616, -0.5063055157661438, 0.030827246606349945, 0.06813422590494156, 0.25597453117370605, 0.008969760499894619, -0.2160501778125763, -0.20582297444343567, 0.029385603964328766, 0.19034753739833832, -0.09060962498188019, -0.20969301462173462, 0.1414690613746643, -0.4959445595741272, -0.331805557012558, 0.4533627927303314, -0.0029274059925228357, 0.023881515488028526, -0.47210693359375, -0.2513149082660675, -0.030783653259277344, -0.036583587527275085, -0.10316245257854462, -0.8016223907470703, 0.19279508292675018, 0.10277097672224045, 0.44000497460365295, 0.04005799815058708, 0.39517420530319214, 0.2494695782661438, -0.46168816089630127, 0.5263959169387817, -0.18070966005325317, 0.08264264464378357, 0.202775776386261, -0.06806693971157074, -0.30417829751968384, 0.5307193994522095, 0.3789900839328766, -0.12879709899425507, 0.10998237133026123, -0.906109094619751, 0.14309090375900269, -0.5914031863212585, -0.20499607920646667, -0.10434333235025406, -0.06976988911628723, 0.06435717642307281, 0.12460923194885254, -0.3943043053150177, 0.1496170610189438, -0.01037159189581871, -0.22452524304389954, -0.11245841532945633, 0.4402267634868622, 0.0012790776090696454, -0.07720479369163513, 0.7563000321388245, -0.1298176646232605, -0.23067504167556763, 0.11419204622507095, -0.34459489583969116, 0.6027095913887024, 0.2364266961812973, -0.23939095437526703, -0.2515600919723511, 0.06775407493114471, 0.7379931807518005, 0.07842770218849182, -0.4024704396724701, 0.15592366456985474, 0.4090234339237213, 0.5310749411582947, 0.2177850753068924, -0.6397239565849304, 0.27098703384399414, 0.14190785586833954, -0.6313067674636841, -0.051034558564424515, -0.2423361986875534, 0.2820175886154175, 0.01944449543952942, -0.24539023637771606, 0.09236195683479309, -0.17774787545204163, 0.10090679675340652, -0.34859147667884827, -0.16563759744167328, -0.023678146302700043, 0.026900628581643105, 0.10739192366600037, 0.024740666151046753, -0.3359924256801605, -0.02618621475994587, 0.43106600642204285, 0.25273895263671875, 1.2423290014266968, 0.012867378070950508, 0.24722306430339813, -0.01709722727537155, 0.837782084941864, 0.3495517373085022, 0.0913650318980217, 0.037736907601356506, 0.33705636858940125, 0.7548288106918335, -0.307587206363678, 0.09478701651096344, 0.00943856779485941, -0.048091087490320206, 0.006891586817800999, -0.4178909659385681, -0.017176350578665733, -0.5711618661880493, 0.17269736528396606, 0.01677742972970009, 0.025257574394345284, 0.6415612697601318, 0.1177285760641098, -0.02889166958630085, -0.2670772671699524, 0.009714338928461075, 0.42445722222328186, 0.34975314140319824, 0.21962718665599823, -0.7066762447357178, 0.007330531720072031, -0.6947780251502991, 0.579947829246521, 0.34728556871414185, 0.3561910390853882, 0.11388754844665527, -0.04646007716655731, 0.4960058033466339, 0.3752251863479614, -0.14812760055065155, 0.21348947286605835, -0.18834879994392395, -0.11168357729911804, -0.21088175475597382, 0.42872536182403564, 0.20350544154644012, 0.21062225103378296, -0.2823370695114136, -0.17185227572917938, 0.19453653693199158, -0.28124698996543884, -0.14152111113071442, -0.005699959117919207, 0.030763085931539536, -0.2447502613067627, 0.35433876514434814, 0.2086053043603897, -1.2662067413330078, 0.2583712041378021, 0.12058838456869125, -0.25123360753059387, 0.2159331887960434, -0.08718161284923553, 0.22757942974567413, -0.6522876024246216, -0.0946209654211998, 0.035423118621110916, -0.23387373983860016, -0.19326908886432648, -0.43521925806999207, -0.2775498032569885, 0.704760730266571, -0.033972933888435364, -0.3536202013492584, -0.15553665161132812, -0.26684677600860596, 0.027128810063004494, -0.0070023974403738976, 0.15462540090084076, -0.04307633265852928, 0.19604896008968353, -0.30024439096450806, 0.45410048961639404, 0.08723831176757812, -0.1293584257364273, -0.3216416537761688, 0.08959659188985825, 0.04877450689673424, 0.7309564352035522, 0.3075862228870392, -0.1842740923166275, 0.3099173903465271, -0.16847874224185944, -0.168104350566864, 0.035553641617298126, -0.011482272297143936, -0.002366652013733983, 0.001565482933074236, -0.0440436489880085, -0.6026142835617065, 0.062004454433918, -0.4026916027069092, 0.6785517334938049, 0.25741496682167053, 0.04800078645348549, 0.47641605138778687, -0.1855340301990509, 0.06917165219783783, 0.048419393599033356, 0.5003483891487122, 0.09738805890083313, -0.2807076871395111, 0.2638462781906128, 0.08694696426391602, 0.3161051869392395, 0.020305730402469635, 0.17564080655574799, -0.07510220259428024, 0.4772210717201233, 0.31880876421928406, -0.278464138507843, -0.2668238878250122, -0.3584354519844055, -0.01369401440024376, 0.07735531777143478, 0.2770506739616394, -0.3568294942378998, -0.083838552236557, -0.4598856568336487, -0.04411618784070015, -0.7056963443756104, -0.3367539942264557, -0.014152970165014267, -0.43593552708625793, -0.2461492270231247, 0.00946315098553896, 0.7532075643539429, 0.2128690630197525, -0.17993348836898804, 0.11462759971618652, 0.1213318258523941, -0.6632418036460876, -0.29612815380096436, -0.2638387084007263, 0.5280694961547852, 0.8543341755867004, 0.23052151501178741, -0.49937883019447327, -0.4551893472671509, -0.41048991680145264, -0.44902700185775757, -0.30406174063682556, -0.15711277723312378, -0.10769552737474442, 0.06261948496103287, -0.027660204097628593, -0.23019708693027496, -0.12495151162147522, 0.36033403873443604, 0.04813801497220993, -0.0931679904460907, -0.0542253702878952, -0.2696293294429779, -0.09118065237998962, -0.05294523015618324, -0.2633776068687439, -0.4029826521873474, -0.16548162698745728, 0.40302345156669617, -0.14244718849658966, 0.024347428232431412, 0.28768467903137207, 0.523603618144989, 0.5012550950050354, -0.20019331574440002, -0.0840647891163826, -0.5621437430381775, -0.2743733823299408, -0.21391095221042633, -0.12288447469472885, 0.13379032909870148, -0.37336432933807373, 0.26606664061546326, -0.2958563566207886, -0.12013028562068939, -3.178177833557129, 0.28123047947883606, -0.06560268998146057, -0.08327525109052658, 0.16012659668922424, -0.2199803739786148, -0.6117354035377502, -0.185200035572052, -0.3248830735683441, -0.1343047022819519, -0.15828093886375427, -0.3153020143508911, -0.09488894790410995, 0.4201800525188446, 0.07916344702243805, 0.4365987479686737, -0.23338182270526886, -0.04632819443941116, -0.10210508853197098, 0.2475159913301468, -0.10139764845371246, -0.17321306467056274, -0.15419000387191772, -0.21259579062461853, 0.5115930438041687, -0.2636862099170685, -0.7115906476974487, 0.3129478096961975, -0.2726863622665405, -0.3921176493167877, 0.04724772274494171, -0.1471920907497406, 0.3012799620628357, 0.26702603697776794, -0.3636065125465393, 0.15997062623500824, 0.011711706407368183, -0.009344015270471573, 0.504437267780304, -0.2274555116891861, -0.15975238382816315, 0.3866519331932068, -0.10831987857818604, 0.1482001692056656, 0.49038130044937134, -0.4515801668167114, 0.018118873238563538, -0.20047764480113983, 0.11829677224159241, 0.19540825486183167, -0.16795141994953156, -0.25229400396347046, 0.17685192823410034, -0.23629042506217957, -0.059625957161188126, -0.16464130580425262, 0.3409057855606079, 0.3378138542175293, -0.30975106358528137, 0.22668470442295074, 0.30065643787384033, -0.5790297389030457, 0.061689164489507675, -0.0921030193567276, -0.16579720377922058, -0.17735998332500458, -0.9442829489707947, -0.32885822653770447, -0.18030039966106415, 0.11067616939544678, -0.2948833405971527, -0.04947458952665329, -0.3762120008468628, -0.9063961505889893, 0.06372770667076111, -0.16314838826656342, 0.13635319471359253, -0.44864803552627563, 0.14233554899692535, -0.29145175218582153, -0.533868134021759, -0.6373703479766846, 0.10337969660758972, 0.14260724186897278, -0.10799116641283035, 0.3350011110305786, -0.049665287137031555, -0.3893287777900696, 0.06555499881505966, -0.1751127541065216, 0.14428144693374634, 0.2887987792491913, 0.19807294011116028, 0.22175975143909454, -0.15867255628108978, -0.06887229532003403, 0.4642682671546936, -0.6612264513969421, 0.5235476493835449, -0.0005811576847918332, 0.3748910129070282, -0.32579201459884644, 0.1417468786239624, -0.0791654959321022, -0.18168534338474274, 0.21930547058582306, -0.7975807189941406, 0.02929501421749592, -0.02324015647172928, 0.23177075386047363, 0.2210131287574768, -0.007116118911653757, 0.22073981165885925, -0.22763827443122864, -0.2080957293510437, 0.4666169583797455, 0.01804409734904766, 0.36620277166366577, 0.3517724275588989, -0.17897449433803558, 0.13317377865314484, 0.45298323035240173, -0.0736551433801651, -0.21521933376789093, -0.10001467913389206, -0.09837953001260757, 0.03318420425057411, 0.005823049228638411, -0.006568295881152153, -0.3777308464050293, -0.2261488139629364, -0.26749610900878906, 0.017795778810977936, -0.30662569403648376, 0.2406696230173111, -0.42913228273391724, 0.027223745360970497, 0.7276060581207275, -0.14930826425552368, -0.21787740290164948, 0.4286799430847168, 0.17186301946640015, 0.10258399695158005, -0.34918543696403503, -0.1515330672264099, 0.2736714780330658, -0.22554169595241547, 0.29788854718208313, 0.13567259907722473, 0.3157699704170227, -0.20985692739486694, 0.041894812136888504, 0.0037313965149223804, -0.28673020005226135, 0.11894182115793228, -0.39883291721343994, 0.07759390026330948, -0.36311984062194824, 0.05574416369199753, -0.5604164004325867, 0.2833912968635559, 0.6190540790557861, 0.6084270477294922, 0.1221693754196167, -0.10117528587579727, 0.1066589429974556, 0.14821211993694305, 0.04570268839597702, -0.3698849081993103, 0.19731786847114563, -0.10505212843418121, -0.2791224718093872, 0.42270442843437195, 0.034884262830019, 0.12741048634052277, 0.5992529392242432, 0.24161645770072937, 0.07096201926469803, 0.4425767958164215, 1.0526232719421387, 0.2534419894218445, 0.031276702880859375, 0.21317994594573975, 0.09657827764749527, 0.4292469620704651, 0.48659616708755493, -0.39894095063209534, -0.04774849861860275, -0.06532485038042068, 0.17784297466278076, -0.4411599636077881, 0.03686841204762459, -0.12541362643241882, -0.6639444828033447, -0.34424030780792236, -0.0025770599022507668, 0.3932969272136688, 0.13060294091701508, 0.1635836958885193, -0.33134549856185913, 0.2541840970516205, -0.1768231987953186, -0.8140914440155029, -0.12517140805721283, 0.0588889978826046, -0.5894185900688171, 0.5492771863937378, 0.014286777935922146, -0.08409532159566879, -0.15646471083164215, 0.19237875938415527, -0.5452587604522705, -0.47476449608802795, -1.0156774520874023, 0.3414498567581177, -0.16317418217658997, 0.15753485262393951, 0.10813387483358383, -0.06242823600769043, -0.2952757775783539, -0.43149125576019287, -0.020087841898202896, 0.21560388803482056, 0.6184669733047485, -0.20206056535243988, -0.040024977177381516, -0.5302854180335999, -0.388792484998703, -0.836505115032196, 0.23438794910907745, -0.1714940369129181, -0.23461097478866577, 0.24194063246250153, 0.09241267293691635, 0.20775215327739716, -0.44669172167778015, -0.34205150604248047, -0.10240808874368668, 0.07768390327692032, -0.5315855741500854, -0.18038763105869293, 0.0015598692698404193, -0.2673301696777344, -0.5363814234733582, 0.25176069140434265, -0.20040263235569, -0.05463737994432449, 0.30502116680145264, -0.30768290162086487, 0.4137945771217346, -0.13952253758907318, -0.10065245628356934, -0.15381121635437012, -0.38787761330604553, 0.15455375611782074, -0.09453275799751282, -0.5959376692771912, -0.08357175439596176, -0.3862594664096832, 0.3092963397502899, 0.28230488300323486, -0.09178756922483444, -0.6324250102043152, 0.43310657143592834, 0.3947027623653412, 0.2870403528213501, 0.3286755681037903, 0.28816351294517517, 0.20610150694847107, 0.028487155213952065, -0.35468581318855286, -0.4090496301651001, 0.11397948861122131, 0.09737507998943329, 0.011827349662780762, -0.6960173845291138, 0.001459568738937378, -0.26087552309036255, 0.2925378382205963, -0.04329267516732216, 0.01639106124639511, 0.09060566127300262, 0.09964166581630707, 0.19600889086723328, 0.37933459877967834, 0.234381303191185, -0.026907198131084442, 0.22043932974338531, -0.02781994827091694, -0.2943801283836365, 0.21201500296592712, 0.2695043087005615, -0.20934802293777466, -0.4934383034706116, -0.25524160265922546, 0.37168848514556885, 0.4477499723434448, 0.12945252656936646, 0.1257922351360321, -0.12288568913936615, -0.12801778316497803, 0.7288113832473755, -0.027378065511584282, -0.03479040414094925, -0.1216411218047142, 0.07419747859239578, 0.029977798461914062, -0.09367834776639938, 0.05658012628555298, 0.22756065428256989, -0.1984071582555771, -0.2859850227832794, 0.3707926273345947, 0.318454772233963, -0.41495323181152344, -0.17827166616916656, 0.2392405867576599, -0.2413223832845688, 0.3038199841976166, 0.02150832675397396, -0.14551475644111633, -0.031039007008075714, 0.6168343424797058, 0.09013944119215012, -0.08964768052101135, 0.22151102125644684, -0.3323756158351898, -0.2608027458190918, -0.1943763941526413, 0.09817375242710114, 0.024247365072369576, 0.4054005444049835, 0.20330902934074402, 0.339108943939209, -0.27596747875213623, -0.28140366077423096, 0.052726566791534424, 0.2700139284133911, 0.40940648317337036, -0.18407851457595825, 0.23856228590011597, 0.4050526022911072, -0.0403129905462265, 0.31703588366508484, -0.13506585359573364, -0.1489497572183609, -0.2084692120552063, 0.3416787385940552, 0.08549182116985321, 0.7007935047149658, 0.5452704429626465, -0.02494695410132408, 0.4601617455482483, -0.15650063753128052, 0.046695128083229065, -0.03176360949873924, 0.5551736354827881, 0.4066142141819, 0.4905884563922882, -0.16787725687026978, 0.1057165190577507, 0.08744202554225922, 0.0860619843006134, -0.534214198589325, 0.1719319224357605, 0.3142610788345337, 0.22809331119060516, 0.1624908298254013, -0.2819324731826782, -0.047202128916978836, -0.0034472744446247816, 0.3411674499511719, -0.1533273309469223, 0.09022262692451477, -0.2970191538333893, 0.23884810507297516, 0.05073408782482147, 0.12429920583963394, -0.5150727033615112, -0.12619099020957947, -0.10840418934822083, -0.14412695169448853, -0.20003744959831238, -0.3259880244731903, -0.14844779670238495, -0.37933650612831116, 0.17871332168579102, 0.32077300548553467, -0.21990658342838287, -0.4092126488685608, -0.09439201653003693, -0.036688968539237976, 0.520562469959259, -0.3627282381057739, 0.48330163955688477, -0.03263167291879654, -0.3251660466194153, 0.3931685984134674, 0.42751961946487427, -0.19995075464248657, 0.13267292082309723, -0.08972231298685074, -0.06244371831417084, -0.007345878519117832, -0.04843100905418396, 0.1395680457353592, -0.0047126347199082375, 0.463409423828125, -0.5887267589569092, -0.3844931721687317, 0.472568154335022, 0.2328605353832245, -0.5899361371994019, -0.3416799008846283, -0.10428176075220108, 0.1242745965719223, -0.2845195531845093, -0.03155498206615448, -0.2543717622756958, 0.4358425736427307, -0.46670001745224, -0.10387367755174637, 0.43157169222831726, 0.35983383655548096, -0.09524598717689514, 0.01926456205546856, -0.07981356978416443, 0.4094965159893036, 0.179312065243721, -0.18965713679790497, 0.5743433833122253, 0.23732121288776398, -0.06059987097978592, 0.5850181579589844, 0.08120909333229065, 0.11997534334659576, -0.2693415880203247, 0.2535378336906433, 0.14137215912342072, -0.05087925121188164, -0.41419151425361633, 0.33649706840515137, 0.03948766738176346, 0.26425397396087646, -1.5951991081237793, 0.11063564568758011, 0.0005923708085902035, -0.257681667804718, -0.29461368918418884, -0.22804729640483856, 0.505540132522583, -0.259371280670166, 0.06845149397850037, -0.3423007130622864, 0.04012271389365196, 0.36992672085762024, 0.19828099012374878, -0.2300453782081604, -0.41609126329421997, 0.27678290009498596]\n",
            "Data inserted successfully.\n"
          ]
        }
      ]
    }
  ],
  "metadata": {
    "kernelspec": {
      "display_name": "Python 3",
      "name": "python3"
    },
    "language_info": {
      "codemirror_mode": {
        "name": "ipython",
        "version": 3
      },
      "file_extension": ".py",
      "mimetype": "text/x-python",
      "name": "python",
      "nbconvert_exporter": "python",
      "pygments_lexer": "ipython3",
      "version": "3.7.1"
    },
    "colab": {
      "provenance": [],
      "gpuType": "T4"
    },
    "accelerator": "GPU"
  },
  "nbformat": 4,
  "nbformat_minor": 0
}