{
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "JnsoEPY5z6V_"
      },
      "source": [
        "Install Weaviate"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "k-UDlYcUVub6",
        "outputId": "84499af2-4906-409f-b5bd-20074f965537"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Requirement already satisfied: weaviate-client in /usr/local/lib/python3.10/dist-packages (4.7.1)\n",
            "Requirement already satisfied: requests<3.0.0,>=2.30.0 in /usr/local/lib/python3.10/dist-packages (from weaviate-client) (2.31.0)\n",
            "Requirement already satisfied: httpx<=0.27.0,>=0.25.0 in /usr/local/lib/python3.10/dist-packages (from weaviate-client) (0.27.0)\n",
            "Requirement already satisfied: validators==0.33.0 in /usr/local/lib/python3.10/dist-packages (from weaviate-client) (0.33.0)\n",
            "Requirement already satisfied: authlib<2.0.0,>=1.2.1 in /usr/local/lib/python3.10/dist-packages (from weaviate-client) (1.3.1)\n",
            "Requirement already satisfied: pydantic<3.0.0,>=2.5.0 in /usr/local/lib/python3.10/dist-packages (from weaviate-client) (2.8.2)\n",
            "Requirement already satisfied: grpcio<2.0.0,>=1.57.0 in /usr/local/lib/python3.10/dist-packages (from weaviate-client) (1.65.1)\n",
            "Requirement already satisfied: grpcio-tools<2.0.0,>=1.57.0 in /usr/local/lib/python3.10/dist-packages (from weaviate-client) (1.65.1)\n",
            "Requirement already satisfied: grpcio-health-checking<2.0.0,>=1.57.0 in /usr/local/lib/python3.10/dist-packages (from weaviate-client) (1.65.1)\n",
            "Requirement already satisfied: cryptography in /usr/local/lib/python3.10/dist-packages (from authlib<2.0.0,>=1.2.1->weaviate-client) (43.0.0)\n",
            "Requirement already satisfied: protobuf<6.0dev,>=5.26.1 in /usr/local/lib/python3.10/dist-packages (from grpcio-health-checking<2.0.0,>=1.57.0->weaviate-client) (5.27.2)\n",
            "Requirement already satisfied: setuptools in /usr/local/lib/python3.10/dist-packages (from grpcio-tools<2.0.0,>=1.57.0->weaviate-client) (71.0.4)\n",
            "Requirement already satisfied: anyio in /usr/local/lib/python3.10/dist-packages (from httpx<=0.27.0,>=0.25.0->weaviate-client) (3.7.1)\n",
            "Requirement already satisfied: certifi in /usr/local/lib/python3.10/dist-packages (from httpx<=0.27.0,>=0.25.0->weaviate-client) (2024.7.4)\n",
            "Requirement already satisfied: httpcore==1.* in /usr/local/lib/python3.10/dist-packages (from httpx<=0.27.0,>=0.25.0->weaviate-client) (1.0.5)\n",
            "Requirement already satisfied: idna in /usr/local/lib/python3.10/dist-packages (from httpx<=0.27.0,>=0.25.0->weaviate-client) (3.7)\n",
            "Requirement already satisfied: sniffio in /usr/local/lib/python3.10/dist-packages (from httpx<=0.27.0,>=0.25.0->weaviate-client) (1.3.1)\n",
            "Requirement already satisfied: h11<0.15,>=0.13 in /usr/local/lib/python3.10/dist-packages (from httpcore==1.*->httpx<=0.27.0,>=0.25.0->weaviate-client) (0.14.0)\n",
            "Requirement already satisfied: annotated-types>=0.4.0 in /usr/local/lib/python3.10/dist-packages (from pydantic<3.0.0,>=2.5.0->weaviate-client) (0.7.0)\n",
            "Requirement already satisfied: pydantic-core==2.20.1 in /usr/local/lib/python3.10/dist-packages (from pydantic<3.0.0,>=2.5.0->weaviate-client) (2.20.1)\n",
            "Requirement already satisfied: typing-extensions>=4.6.1 in /usr/local/lib/python3.10/dist-packages (from pydantic<3.0.0,>=2.5.0->weaviate-client) (4.12.2)\n",
            "Requirement already satisfied: charset-normalizer<4,>=2 in /usr/local/lib/python3.10/dist-packages (from requests<3.0.0,>=2.30.0->weaviate-client) (3.3.2)\n",
            "Requirement already satisfied: urllib3<3,>=1.21.1 in /usr/local/lib/python3.10/dist-packages (from requests<3.0.0,>=2.30.0->weaviate-client) (2.0.7)\n",
            "Requirement already satisfied: exceptiongroup in /usr/local/lib/python3.10/dist-packages (from anyio->httpx<=0.27.0,>=0.25.0->weaviate-client) (1.2.2)\n",
            "Requirement already satisfied: cffi>=1.12 in /usr/local/lib/python3.10/dist-packages (from cryptography->authlib<2.0.0,>=1.2.1->weaviate-client) (1.16.0)\n",
            "Requirement already satisfied: pycparser in /usr/local/lib/python3.10/dist-packages (from cffi>=1.12->cryptography->authlib<2.0.0,>=1.2.1->weaviate-client) (2.22)\n"
          ]
        }
      ],
      "source": [
        "!pip install weaviate-client"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "SPqKDD-JV1eV"
      },
      "outputs": [],
      "source": [
        "import weaviate\n",
        "from weaviate.embedded import EmbeddedOptions"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "XmDJf6YwmT9V",
        "outputId": "d997cf98-3b88-4b2d-efd8-3eb9ff1fe4c7"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "INFO:weaviate-client:Binary /root/.cache/weaviate-embedded did not exist. Downloading binary from https://github.com/weaviate/weaviate/releases/download/v1.26.1/weaviate-v1.26.1-Linux-amd64.tar.gz\n",
            "INFO:weaviate-client:Started /root/.cache/weaviate-embedded: process ID 731\n"
          ]
        }
      ],
      "source": [
        "client = weaviate.Client(\n",
        "  embedded_options=EmbeddedOptions(),\n",
        "  additional_headers={\n",
        "    \"X-HuggingFace-Api-Key\": \"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\"\n",
        "  }\n",
        ")"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "M6WdwCk_pu6Z",
        "outputId": "db1cb7a8-3a45-48a1-8772-6db23d170bb0"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Collecting langchain\n",
            "  Downloading langchain-0.2.11-py3-none-any.whl.metadata (7.1 kB)\n",
            "Requirement already satisfied: PyYAML>=5.3 in /usr/local/lib/python3.10/dist-packages (from langchain) (6.0.1)\n",
            "Requirement already satisfied: SQLAlchemy<3,>=1.4 in /usr/local/lib/python3.10/dist-packages (from langchain) (2.0.31)\n",
            "Requirement already satisfied: aiohttp<4.0.0,>=3.8.3 in /usr/local/lib/python3.10/dist-packages (from langchain) (3.9.5)\n",
            "Requirement already satisfied: async-timeout<5.0.0,>=4.0.0 in /usr/local/lib/python3.10/dist-packages (from langchain) (4.0.3)\n",
            "Collecting langchain-core<0.3.0,>=0.2.23 (from langchain)\n",
            "  Downloading langchain_core-0.2.23-py3-none-any.whl.metadata (6.2 kB)\n",
            "Collecting langchain-text-splitters<0.3.0,>=0.2.0 (from langchain)\n",
            "  Downloading langchain_text_splitters-0.2.2-py3-none-any.whl.metadata (2.1 kB)\n",
            "Collecting langsmith<0.2.0,>=0.1.17 (from langchain)\n",
            "  Downloading langsmith-0.1.93-py3-none-any.whl.metadata (13 kB)\n",
            "Requirement already satisfied: numpy<2,>=1 in /usr/local/lib/python3.10/dist-packages (from langchain) (1.25.2)\n",
            "Requirement already satisfied: pydantic<3,>=1 in /usr/local/lib/python3.10/dist-packages (from langchain) (2.8.2)\n",
            "Requirement already satisfied: requests<3,>=2 in /usr/local/lib/python3.10/dist-packages (from langchain) (2.31.0)\n",
            "Requirement already satisfied: tenacity!=8.4.0,<9.0.0,>=8.1.0 in /usr/local/lib/python3.10/dist-packages (from langchain) (8.5.0)\n",
            "Requirement already satisfied: aiosignal>=1.1.2 in /usr/local/lib/python3.10/dist-packages (from aiohttp<4.0.0,>=3.8.3->langchain) (1.3.1)\n",
            "Requirement already satisfied: attrs>=17.3.0 in /usr/local/lib/python3.10/dist-packages (from aiohttp<4.0.0,>=3.8.3->langchain) (23.2.0)\n",
            "Requirement already satisfied: frozenlist>=1.1.1 in /usr/local/lib/python3.10/dist-packages (from aiohttp<4.0.0,>=3.8.3->langchain) (1.4.1)\n",
            "Requirement already satisfied: multidict<7.0,>=4.5 in /usr/local/lib/python3.10/dist-packages (from aiohttp<4.0.0,>=3.8.3->langchain) (6.0.5)\n",
            "Requirement already satisfied: yarl<2.0,>=1.0 in /usr/local/lib/python3.10/dist-packages (from aiohttp<4.0.0,>=3.8.3->langchain) (1.9.4)\n",
            "Collecting jsonpatch<2.0,>=1.33 (from langchain-core<0.3.0,>=0.2.23->langchain)\n",
            "  Downloading jsonpatch-1.33-py2.py3-none-any.whl.metadata (3.0 kB)\n",
            "Requirement already satisfied: packaging<25,>=23.2 in /usr/local/lib/python3.10/dist-packages (from langchain-core<0.3.0,>=0.2.23->langchain) (24.1)\n",
            "Collecting orjson<4.0.0,>=3.9.14 (from langsmith<0.2.0,>=0.1.17->langchain)\n",
            "  Downloading orjson-3.10.6-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (50 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m50.4/50.4 kB\u001b[0m \u001b[31m978.0 kB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hRequirement already satisfied: annotated-types>=0.4.0 in /usr/local/lib/python3.10/dist-packages (from pydantic<3,>=1->langchain) (0.7.0)\n",
            "Requirement already satisfied: pydantic-core==2.20.1 in /usr/local/lib/python3.10/dist-packages (from pydantic<3,>=1->langchain) (2.20.1)\n",
            "Requirement already satisfied: typing-extensions>=4.6.1 in /usr/local/lib/python3.10/dist-packages (from pydantic<3,>=1->langchain) (4.12.2)\n",
            "Requirement already satisfied: charset-normalizer<4,>=2 in /usr/local/lib/python3.10/dist-packages (from requests<3,>=2->langchain) (3.3.2)\n",
            "Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.10/dist-packages (from requests<3,>=2->langchain) (3.7)\n",
            "Requirement already satisfied: urllib3<3,>=1.21.1 in /usr/local/lib/python3.10/dist-packages (from requests<3,>=2->langchain) (2.0.7)\n",
            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.10/dist-packages (from requests<3,>=2->langchain) (2024.7.4)\n",
            "Requirement already satisfied: greenlet!=0.4.17 in /usr/local/lib/python3.10/dist-packages (from SQLAlchemy<3,>=1.4->langchain) (3.0.3)\n",
            "Collecting jsonpointer>=1.9 (from jsonpatch<2.0,>=1.33->langchain-core<0.3.0,>=0.2.23->langchain)\n",
            "  Downloading jsonpointer-3.0.0-py2.py3-none-any.whl.metadata (2.3 kB)\n",
            "Downloading langchain-0.2.11-py3-none-any.whl (990 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m990.3/990.3 kB\u001b[0m \u001b[31m7.9 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading langchain_core-0.2.23-py3-none-any.whl (374 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m374.2/374.2 kB\u001b[0m \u001b[31m17.8 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading langchain_text_splitters-0.2.2-py3-none-any.whl (25 kB)\n",
            "Downloading langsmith-0.1.93-py3-none-any.whl (139 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m139.8/139.8 kB\u001b[0m \u001b[31m9.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading jsonpatch-1.33-py2.py3-none-any.whl (12 kB)\n",
            "Downloading orjson-3.10.6-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (141 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m141.1/141.1 kB\u001b[0m \u001b[31m9.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading jsonpointer-3.0.0-py2.py3-none-any.whl (7.6 kB)\n",
            "Installing collected packages: orjson, jsonpointer, jsonpatch, langsmith, langchain-core, langchain-text-splitters, langchain\n",
            "Successfully installed jsonpatch-1.33 jsonpointer-3.0.0 langchain-0.2.11 langchain-core-0.2.23 langchain-text-splitters-0.2.2 langsmith-0.1.93 orjson-3.10.6\n"
          ]
        }
      ],
      "source": [
        "!pip install langchain"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "ZTRcG4i-0TVn"
      },
      "source": [
        "Fetch data"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "7nhsM0RXV8Vt",
        "outputId": "d6b7afde-ab02-43fe-f892-70164afa6188"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[1;30;43mStreaming output truncated to the last 5000 lines.\u001b[0m\n",
            "          \"sampling techniques on the proteomic findings. In the genomic analysis, effective enrichment of neoplastic cellularity revealed by high KRAS VAF scores was found in LMD as well as cored tumor tissues obtained from certain bulk tumor tissues containing\",\n",
            "          \"tissues obtained from certain bulk tumor tissues containing different amount of cellularity. Knowledge of the effects and differences of sampling techniques warrant further investigation.\",\n",
            "          \"Supplementary Information\\n\\nAbbreviations\\n\\nACN\\n\\nAcetonitrile\\n\\nAGC\\n\\nAutomatic gain control\\n\\nbRPLC\\n\\nBasic reversed-phase liquid chromatography\\n\\nCHMP\\n\\nChromatin-modifying protein/charged multivesicular body protein\\n\\nCPTAC\",\n",
            "          \"CPTAC\\n\\nClinical proteomic tumor analysis consortium\\n\\nDDA\\n\\nData-dependent acquisition\\n\\nDTT\\n\\nDithiothreitol; FA: formic acid\\n\\nFDR\\n\\nFalse discovery rate\\n\\nH&E\\n\\nHematoxylin and eosin\\n\\nHCD\\n\\nCollision dissociation activation energy\\n\\nHPA\\n\\nHuman Proteome Atlas\",\n",
            "          \"HPA\\n\\nHuman Proteome Atlas\\n\\nIAA\\n\\nIodoacetamide\\n\\niRT\\n\\nIndexed retention time\\n\\nLMD\\n\\nLaser microdissection\\n\\nMeOH\\n\\nMethanol\\n\\nMS\\n\\nMass spectrometry\\n\\nNAT\\n\\nNormal adjacent tissues\\n\\nNCE\\n\\nNormalized collision energy\\n\\nPCA\\n\\nPrincipal component analysis\\n\\nPDAC\",\n",
            "          \"PCA\\n\\nPrincipal component analysis\\n\\nPDAC\\n\\nPancreatic adenocarcinoma\\n\\nPSM\\n\\nPeptide spectrum match\\n\\nQC\\n\\nQuality control\\n\\nRT\\n\\nRoom temperature\\n\\nSD\\n\\nStandard deviation\\n\\nTCGA\\n\\nThe Cancer Genome Atlas\\n\\nTCEP\\n\\nTris (2-carboxyethyl) phosphine\\n\\nTFA\",\n",
            "          \"TCEP\\n\\nTris (2-carboxyethyl) phosphine\\n\\nTFA\\n\\nTrifluoroacetic acid\\n\\nTMT\\n\\nTandem mass tag\\n\\nVAF\\n\\nVariant allele frequency\\n\\nWES\\n\\nWhole-exome sequencing\\n\\nPublisher's Note\",\n",
            "          \"WES\\n\\nWhole-exome sequencing\\n\\nPublisher's Note\\n\\nSpringer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\\n\\nAuthor contributions\",\n",
            "          \"QKL, HZ, YEH, AIR, GM, MM, OFB, RHH, DWC contributed to experimental design, organization of data and drafting the manuscript; YEH, YZL, RL, CZ, LD, BZ, GH, CJN, SDJ, QKL, HZ, MS, LC, contributed to data analysis. All authors read and approved the final\",\n",
            "          \"to data analysis. All authors read and approved the final manuscript.\",\n",
            "          \"Funding\\n\\nThis work is supported in part by the National Institutes of Health, National Cancer Institute, Clinical Proteomics Tumor Analysis Consortium (CPTAC, U24CA210985).\\n\\nAvailability of data and materials\",\n",
            "          \"All data generated and analyzed during this study are included in this published article and its supplementary information files. Pathology and radiology images can be accessed via Imaging Data Commons (IDC) at\",\n",
            "          \"images can be accessed via Imaging Data Commons (IDC) at https://portal.imaging.datacommons.cancer.gov/explore/filters/?collection_id=cptac_ccrcc and The Cancer Imaging Archive at 10.7937/K9/TCIA.2018.OBLAMN27.\",\n",
            "          \"Declarations\\n\\nEthics approval and consent to participate\\n\\nThese criteria are followed by the guidelines of the Cancer Clinical Proteomics Research, National Cancer Institute \\n\\nConsent for publication\",\n",
            "          \"Consent for publication\\n\\nThis manuscript has been read and approved by all the authors to publish and is not submitted or under consideration for publication elsewhere.\\n\\nCompeting interests\\n\\nThe authors declare no competing interests.\"\n",
            "        ]\n",
            "      }\n",
            "    ]\n",
            "  },\n",
            "  \"PDC000291\": {\n",
            "    \"study_id\": \"a2df3cc3-2c40-4da6-ac5a-ff016cce4bbd\",\n",
            "    \"pdc_study_id\": \"PDC000291\",\n",
            "    \"study_submitter_id\": \"TCGA Breast Cancer CompRef Samples - Phosphoproteome\",\n",
            "    \"program_id\": \"10251935-5540-11e8-b664-00a098d917f8\",\n",
            "    \"project_id\": \"c96eca75-8b73-4018-9bf7-b20e31a965cd\",\n",
            "    \"study_name\": \"TCGA Breast Cancer CompRef Samples - Phosphoproteome\",\n",
            "    \"program_name\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"project_name\": \"CPTAC2 Retrospective\",\n",
            "    \"disease_type\": \"Other\",\n",
            "    \"primary_site\": \"Not Reported\",\n",
            "    \"analytical_fraction\": \"Phosphoproteome\",\n",
            "    \"experiment_type\": \"iTRAQ4\",\n",
            "    \"embargo_date\": null,\n",
            "    \"cases_count\": 2,\n",
            "    \"aliquots_count\": 20,\n",
            "    \"protocol_id\": \"e9812283-3c88-4d7a-990a-15402dc4759d\",\n",
            "    \"protocol_submitter_id\": \"TCGA Breast Cancer CompRef Samples Phosphoproteome\",\n",
            "    \"program_submitter_id\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"protocol_name\": \"TCGA Breast Cancer CompRef Samples Phosphoproteome\",\n",
            "    \"protocol_date\": \"2021-04-07\",\n",
            "    \"document_name\": \"CPTAC-SOP-Broad_Proteome/Phosphoproteome_3.0.txt\",\n",
            "    \"quantitation_strategy\": \"Isobaric label quantitation analysis\",\n",
            "    \"label_free_quantitation\": \"N/A\",\n",
            "    \"labeled_quantitation\": \"iTRAQ\",\n",
            "    \"isobaric_labeling_reagent\": \"iTRAQ4\",\n",
            "    \"reporter_ion_ms_level\": \"MS2\",\n",
            "    \"starting_amount\": \"4.00\",\n",
            "    \"starting_amount_uom\": \"mg\",\n",
            "    \"digestion_reagent\": \"Lys-C,Trypsin\",\n",
            "    \"alkylation_reagent\": \"Iodoacetamide (IAA)\",\n",
            "    \"enrichment_strategy\": \"Phosphoproteome\",\n",
            "    \"enrichment\": \"Phosphopeptide enrichment with immobilized metal affinity chromatography (NiNTA beads stripped with EDTA and loaded with FeCl3; Qiagen)\",\n",
            "    \"chromatography_dimensions_count\": \"2\",\n",
            "    \"one_d_chromatography_type\": \"bRPLC (pH 10)\",\n",
            "    \"two_d_chromatography_type\": \"RPLC\",\n",
            "    \"fractions_analyzed_count\": \"13\",\n",
            "    \"column_type\": \"PicoFrit SELF/P\",\n",
            "    \"amount_on_column\": \"1.0\",\n",
            "    \"amount_on_column_uom\": \"\\u00b5g\",\n",
            "    \"column_length\": \"20.0\",\n",
            "    \"column_length_uom\": \"cm\",\n",
            "    \"column_inner_diameter\": \"75.00\",\n",
            "    \"column_inner_diameter_uom\": \"\\u00b5m\",\n",
            "    \"particle_size\": \"1.9\",\n",
            "    \"particle_size_uom\": \"\\u00b5m\",\n",
            "    \"particle_type\": \"ReproSil-Pur C18-AQ\",\n",
            "    \"gradient_length\": \"110.0\",\n",
            "    \"gradient_length_uom\": \"min\",\n",
            "    \"instrument_make\": \"Thermo Fisher Scientific\",\n",
            "    \"instrument_model\": \"Q Exactive\",\n",
            "    \"dissociation_type\": \"HCD\",\n",
            "    \"ms1_resolution\": \"70000\",\n",
            "    \"ms2_resolution\": \"17500\",\n",
            "    \"dda_topn\": \"Top 12\",\n",
            "    \"normalized_collision_energy\": \"28\",\n",
            "    \"acquistion_type\": \"DDA\",\n",
            "    \"dia_multiplexing\": \"N/A\",\n",
            "    \"dia_ims\": \"N/A\",\n",
            "    \"analytical_technique\": null,\n",
            "    \"chromatography_instrument_make\": null,\n",
            "    \"chromatography_instrument_model\": null,\n",
            "    \"polarity\": null,\n",
            "    \"reconstitution_solvent\": null,\n",
            "    \"reconstitution_volume\": null,\n",
            "    \"reconstitution_volume_uom\": null,\n",
            "    \"internal_standards\": null,\n",
            "    \"extraction_method\": null,\n",
            "    \"ionization_mode\": null,\n",
            "    \"publications\": [\n",
            "      {\n",
            "        \"publication_id\": \"44e2e766-89c7-11e8-bcf1-0a2705229b82\",\n",
            "        \"pubmed_id\": \"27251275\",\n",
            "        \"doi\": \"10.1038/nature18003\",\n",
            "        \"author\": \"Mertins, P.; Mani, D. R.; Ruggles, K. V.; Gillette, M. A.; Clauser, K. R.; Wang, P.; Wang, X.; Qiao, J. W.; Cao, S.; Petralia, F.; Kawaler, E.; Mundt, F.; Krug, K.; Tu, Z.; Lei, J. T.; Gatza, M. L.; Wilkerson, M.; Perou, C. M.; Yellapantula, V.; Huang, K. L.; Lin, C.; McLellan, M. D.; Yan, P.; Davies, S. R.; Townsend, R. R.; Skates, S. J.; Wang, J.; Zhang, B.; Kinsinger, C. R.; Mesri, M.; Rodriguez, H.; Ding, L.; Paulovich, A. G.; Feny\\u00f6, D.; Ellis, M. J.; Carr, S. A.; Gil Kuhn, E.; Mani, D. R.; Mertins, P.; Ketchum, K. A.; Thangudu, R. R.; Cai, S.; Oberti, M.; Whiteaker, J. R.; Wang, X.; Lin, C.; Ping, Y.; Edwards, N. J.; Madhavan, S.; McGarvey, P. B.; Wang, P.; Petralia, F.; Tu, Z.; Chan, D.; Pandey, A.; Shih, L. M.; Zhang, H.; Zhang, Z.; Thomas, S.; Zhu, H.; Whiteley, G. A.; Skates, S. J.; White, F. M.; Levine, D. A.; Boja, E. S.; Kinsinger, C. R.; Hiltke, T.; Mesri, M.; Rivers, R. C.; Rodriguez, H.; Shaw, K. M.; Stein, S. E.; Fenyo, D.; Liu, T.; McDermott, J. E.; Payne, S. H.; Rodland, K. D.; Smith, R. D.; Rudnick, P.; Snyder, M.; Zhao, Y.; Chen, X.; Ransohoff, D. F.; Hoofnagle, A. N.; Liebler, D. C.; Sanders, M. E.; Shi, Z.; Slebos, R. J.; Tabb, D. L.; Zhang, B.; Zimmerman, L. J.; Wang, Y.; Li, S.; Davies, S. R.; Ding, L.; Maher, C.; Townsend, R.; Ellis, M. J.; Lei, J. T.; Luo, J.\",\n",
            "        \"title\": \"Proteogenomics connects somatic mutations to signalling in breast cancer\",\n",
            "        \"journal\": \"Nature\",\n",
            "        \"journal_url\": \"https://www.nature.com/articles/nature18003\",\n",
            "        \"year\": \"2016\",\n",
            "        \"abstract\": \"Somatic mutations have been extensively characterized in breast cancer, but the effects of these genetic alterations on the proteomic landscape remain poorly understood. Here we describe quantitative mass-spectrometry-based proteomic and phosphoproteomic analyses of 105 genomically annotated breast cancers, of which 77 provided high-quality data. Integrated analyses provided insights into the somatic cancer genome including the consequences of chromosomal loss, such as the 5q deletion characteristic of basal-like breast cancer. Interrogation of the 5q trans-effects against the Library of Integrated Network-based Cellular Signatures, connected loss of CETN3 and SKP1 to elevated expression of epidermal growth factor receptor (EGFR), and SKP1 loss also to increased SRC tyrosine kinase. Global proteomic data confirmed a stromal-enriched group of proteins in addition to basal and luminal clusters, and pathway analysis of the phosphoproteome identified a G-protein-coupled receptor cluster that was not readily identified at the mRNA level. In addition to ERBB2, other amplicon-associated highly phosphorylated kinases were identified, including CDK12, PAK1, PTK2, RIPK2 and TLK2. We demonstrate that proteogenomic analysis of breast cancer elucidates the functional consequences of somatic mutations, narrows candidate nominations for driver genes within large deletions and amplified regions, and identifies therapeutic targets.\",\n",
            "        \"full_text\": [\n",
            "          \"Proteogenomics connects somatic mutations to signaling in breast cancer\\n\\nSummary\",\n",
            "          \"Somatic mutations have been extensively characterized in breast cancer, but the effects of these genetic alterations on the proteomic landscape remain poorly understood. We describe quantitative mass spectrometry-based proteomic and phosphoproteomic\",\n",
            "          \"mass spectrometry-based proteomic and phosphoproteomic analyses of 105 genomically annotated breast cancers of which 77 provided high-quality data. Integrated analyses allowed insights into the somatic cancer genome including the consequences of\",\n",
            "          \"into the somatic cancer genome including the consequences of chromosomal loss, such as the 5q deletion characteristic of basal-like breast cancer. The 5q trans effects were interrogated against the Library of Integrated Network-based Cellular Signatures,\",\n",
            "          \"the Library of Integrated Network-based Cellular Signatures, thereby connecting CETN3 and SKP1 loss to elevated expression of EGFR, and SKP1 loss also to increased SRC. Global proteomic data confirmed a stromal-enriched group in addition to basal and\",\n",
            "          \"confirmed a stromal-enriched group in addition to basal and luminal clusters and pathway analysis of the phosphoproteome identified a G Protein-coupled receptor cluster that was not readily identified at the mRNA level. Besides ERBB2, other\",\n",
            "          \"not readily identified at the mRNA level. Besides ERBB2, other amplicon-associated, highly phosphorylated kinases were identified, including CDK12, PAK1, PTK2, RIPK2 and TLK2. We demonstrate that proteogenomic analysis of breast cancer elucidates\",\n",
            "          \"that proteogenomic analysis of breast cancer elucidates functional consequences of somatic mutations, narrows candidate nominations for driver genes within large deletions and amplified regions, and identifies therapeutic targets.\",\n",
            "          \"Introduction\",\n",
            "          \"A central deficiency in our knowledge of cancer concerns how genomic changes drive the proteome and phosphoproteome to execute phenotypic characteristics. The initial proteomic characterization in the TCGA breast study was performed using reversed phase\",\n",
            "          \"in the TCGA breast study was performed using reversed phase protein arrays; however this approach is restricted by antibody availability. To provide greater analytical breadth, the NCI Clinical Proteomic Tumor Analysis Consortium (CPTAC) is analyzing the\",\n",
            "          \"Proteomic Tumor Analysis Consortium (CPTAC) is analyzing the proteomes of genome-annotated TCGA tumor specimens using mass spectrometry. Herein we describe integrated proteogenomic analyses of TCGA breast cancer samples representing the four principal\",\n",
            "          \"of TCGA breast cancer samples representing the four principal mRNA-defined breast cancer intrinsic subtypes.\",\n",
            "          \"Proteogenomic analysis of TCGA samples\",\n",
            "          \"105 breast tumors previously characterized by the TCGA were selected for proteomic analysis after histopathological documentation (Supplementary Tables 1 and 2). The cohort included a balanced representation of PAM50-defined intrinsic subtypes including\",\n",
            "          \"representation of PAM50-defined intrinsic subtypes including 25 basal-like, 29 luminal A, 33 luminal B, and 18 HER2 (ERBB2)-enriched tumors, along with 3 normal breast tissue samples. Samples were analyzed by high-resolution, accurate mass, tandem mass\",\n",
            "          \"were analyzed by high-resolution, accurate mass, tandem mass spectrometry (MS) that included extensive peptide fractionation and phosphopeptide enrichment (Extended Data Fig. 1a). An isobaric peptide labeling approach (iTRAQ) was employed to quantify\",\n",
            "          \"peptide labeling approach (iTRAQ) was employed to quantify protein and phosphosite levels across samples, with 37 iTRAQ 4-plexes analyzed in total. A total of 15,369 proteins (12,405 genes) and 62,679 phosphosites were confidently identified with 11,632\",\n",
            "          \"and 62,679 phosphosites were confidently identified with 11,632 proteins/tumor and 26,310 phosphosites/tumor on average (Supplementary Tables 3, 4 and Supplementary Methods). After filtering for observation in at least a quarter of the samples\",\n",
            "          \"filtering for observation in at least a quarter of the samples (Supplementary Methods, Extended Data Fig. 1b) 12,553 proteins (10,062 genes) and 33,239 phosphosites, with their relative abundances quantified across tumors, were used in subsequent analyses\",\n",
            "          \"quantified across tumors, were used in subsequent analyses in this study. Stable longitudinal performance and low technical noise were demonstrated by repeated, interspersed analyses of a single batch of patient-derived luminal and basal breast cancer\",\n",
            "          \"single batch of patient-derived luminal and basal breast cancer xenograft samples (Extended Data Fig. 1d,e). Due to the heterogeneous nature of breast tumors, and because proteomic analyses were performed on tumor fragments that were different from those\",\n",
            "          \"performed on tumor fragments that were different from those used in the genomic analyses, rigorous pre-specified sample and data QC metrics were implemented (Supplementary Discussion and Extended Data Figures 2, 3). Extensive analyses concluded that 28 of\",\n",
            "          \"Data Figures 2, 3). Extensive analyses concluded that 28 of the 105 samples were compromised by protein degradation. These samples were excluded from further analysis with subsequent informatics focused on the 77 tumor samples and three biological\",\n",
            "          \"focused on the 77 tumor samples and three biological replicates.\",\n",
            "          \"Genome and transcriptomic variation was observed at the peptide level by searching MS/MS spectra not matched to RefSeq against a patient-specific sequence database (Fig. 1a). The database was constructed using the QUILTS software package leveraging RefSeq\",\n",
            "          \"constructed using the QUILTS software package leveraging RefSeq gene models based on whole exome and RNA-seq data generated from portions of the same tumors and matched germline DNA (Fig. 1a, Supplementary Table 5). While these analyses detected a number\",\n",
            "          \"Supplementary Table 5). While these analyses detected a number of single amino-acid variants (SAAVs), frameshifts, and splice junctions, including splice isoforms that had been detected as only single transcript reads by RNA-seq (Fig. 1b, Supplementary\",\n",
            "          \"only single transcript reads by RNA-seq (Fig. 1b, Supplementary Table 5), the number of genomic and transcriptomic variants that were confirmed as peptides by MS was low (Supplementary Discussion). Sparse detection of individual genomic variants by\",\n",
            "          \"Discussion). Sparse detection of individual genomic variants by peptide sequencing has been noted in our previous studies and reflects limited coverage at the single amino-acid level with current technology. However quantitative MS analysis of multiple\",\n",
            "          \"technology. However quantitative MS analysis of multiple peptides for each protein is used to reliably infer overall protein levels. This is an advantage for MS since antibody-based protein expression analysis is typically based on a single epitope. To\",\n",
            "          \"expression analysis is typically based on a single epitope. To illustrate this capability in the current data set an initial analysis of three frequently mutated genes in breast cancer (TP53, PIK3CA, and GATA3) and three clinical biomarkers (ER, PGR, and\",\n",
            "          \"PIK3CA, and GATA3) and three clinical biomarkers (ER, PGR, and ERBB2) was conducted (Fig. 1c, Supplementary Table 6, 7 and Supplementary Discussion). As expected, TP53 missense mutations were associated with elevated MS-based protein levels, as observed\",\n",
            "          \"associated with elevated MS-based protein levels, as observed by RPPA (Reverse Phase Protein Array), especially in basal-like breast cancer. TP53 nonsense and frame-shift mutations were associated with a decrease in TP53 protein levels that was\",\n",
            "          \"were associated with a decrease in TP53 protein levels that was particularly striking in the MS data. In contrast, the mostly C-terminal GATA3 frame-shift alterations did not result in decreased protein expression when measured by the median of all GATA3\",\n",
            "          \"protein expression when measured by the median of all GATA3 peptides, suggesting these proteins are expressed despite truncation. No consistent effect of somatic PIK3CA mutation was observed at the level of protein expression. Good correlations between\",\n",
            "          \"at the level of protein expression. Good correlations between RNA-seq and MS-protein expression levels were found for ESR1 (r=0.74), PGR (r=0.74), ERBB2 (r=0.84) and GATA3 (r=0.83) with moderate correlations observed for PIK3CA (r=0.45) and TP53 (r=0.36).\",\n",
            "          \"correlations observed for PIK3CA (r=0.45) and TP53 (r=0.36). Lower TP53 protein abundance levels compared to mRNA levels were especially prevalent in luminal tumors, suggesting post-transcriptional regulatory mechanisms such as proteasomal degradation. To\",\n",
            "          \"regulatory mechanisms such as proteasomal degradation. To explore this hypothesis a search was made for E3 ligases that showed negative correlation to p53 protein (Supplementary Table 8). These analyses identified UBE3A (r=\\u22120.42; adj. pval=0.05) (Extended\",\n",
            "          \"analyses identified UBE3A (r=\\u22120.42; adj. pval=0.05) (Extended Data Fig. 4a), an established TP53 E3 ligase. In comparing CNA, RNA, and protein levels for GATA3, copy number gains in chromosome 10q were anti-correlated with RNA and protein levels in\",\n",
            "          \"10q were anti-correlated with RNA and protein levels in basal-like tumors. This observation prompted a search for other gains or losses that were anticorrelated with RNA and/or protein levels (see Extended Data Fig. 4b for further analyses). Overall, six\",\n",
            "          \"(see Extended Data Fig. 4b for further analyses). Overall, six genes were identified that significantly anti-correlated at an FDR<0.05 on both RNA and protein level to their CNA signals (Extended Data Fig. 4b). GATA3 amplification on 10q in basal-like\",\n",
            "          \"Data Fig. 4b). GATA3 amplification on 10q in basal-like breast cancer showed the strongest anti-correlation, followed by the hexosamine and glycolysis pathway enzymes GFPT2 and HK3, which are upregulated in basal-like breast cancer despite being subjected\",\n",
            "          \"upregulated in basal-like breast cancer despite being subjected to frequent chromosomal deletion on 5q. Global analysis of the correlation of mRNA-to-protein yielded a median Pearson value of r=0.39, with 6,135 out of 9,302 protein/mRNA pairs (66.0%)\",\n",
            "          \"of r=0.39, with 6,135 out of 9,302 protein/mRNA pairs (66.0%) correlating significantly at an FDR<0.05 (Extended Data Fig. 4c, Supplementary Table 9 and Supplementary Discussion). Similar to the colon cancer analysis metabolic functions such as amino\",\n",
            "          \"to the colon cancer analysis metabolic functions such as amino acid, sugar and fatty acid metabolism were found to be enriched among positively correlated genes whereas ribosomal, RNA polymerase and mRNA splicing functions were negatively correlated.\",\n",
            "          \"and mRNA splicing functions were negatively correlated. Overall these analyses demonstrate the utility of global proteome correlation analysis for both confirmation of suspected regulatory mechanisms and identification of candidate regulators meriting\",\n",
            "          \"mechanisms and identification of candidate regulators meriting further investigation.\",\n",
            "          \"Copy Number Alterations\",\n",
            "          \"To determine the consequences of CNA on mRNA, protein and phosphoprotein abundance, both in \\u201ccis\\u201d on genes within the aberrant locus and in \\u201ctrans\\u201d on genes encoded elsewhere, univariate correlation analysis was used as previously described. A total of\",\n",
            "          \"analysis was used as previously described. A total of 7,776 genes with CNA, mRNA and protein measurements were analyzed by calculating Pearson correlation and associated statistical significance (Benjamini-Hochberg corrected p-value) for all possible\",\n",
            "          \"(Benjamini-Hochberg corrected p-value) for all possible CNA-mRNA and CNA-protein pairs (Fig. 2a, Supplementary Table 10, Extended Data Fig. 5a, see Methods). For the phosphoproteome, 4,472 CNA-phosphoprotein pairs were analyzed (Extended Data Fig. 5b).\",\n",
            "          \"CNA-phosphoprotein pairs were analyzed (Extended Data Fig. 5b). Significant positive correlations (cis) were observed for 64% of all CNA-mRNA, 31% of all CNA-protein, and 20% of all CNA-phosphoprotein pairs Fig. 2b. Proteins and phosphoproteins correlated\",\n",
            "          \"pairs Fig. 2b. Proteins and phosphoproteins correlated in cis to CNA were, for the most part, a subset of the cis effects observed in mRNA-to-CNA correlation (Fig. 2b, Supplementary Table 10). The fractional difference of well-annotated oncogenes and\",\n",
            "          \"10). The fractional difference of well-annotated oncogenes and tumor suppressor genes among the significantly cis-correlated CNA-to-mRNA and -protein gene pairs was analyzed. Based on a reference list of 487 oncogenes and tumor suppressors (Supplementary\",\n",
            "          \"list of 487 oncogenes and tumor suppressors (Supplementary Table 10), these cancer relevant genes occur 37.6% more frequently in the subset of genes that correlate both on CNA/mRNA and CNA/protein levels than in the subset that only correlate on CNA/mRNA\",\n",
            "          \"levels than in the subset that only correlate on CNA/mRNA but not on CNA/protein level (Fisher exact p-value=0.02). This suggests that CNA events with a tumor promoting outcome more likely lead to cis regulatory effects on both the protein and mRNA level,\",\n",
            "          \"to cis regulatory effects on both the protein and mRNA level, whereas CNA events with no documented role in tumorigenesis are more likely to be neutralized on the protein level than on the RNA level. Trans effects (Fig. 2a) appear as vertical bands, with\",\n",
            "          \"level. Trans effects (Fig. 2a) appear as vertical bands, with accompanying frequency histograms (in blue) highlighting \\u201chot spots\\u201d of significant trans effects. Using a minimum threshold of 50 trans-affected genes, 68% of the tested genes were associated\",\n",
            "          \"trans-affected genes, 68% of the tested genes were associated with trans effects on the mRNA level, whereas only 13% were associated with effects on the protein level and 8% on the phosphoprotein level. Importantly, CNA-protein correlations appeared to be\",\n",
            "          \"level. Importantly, CNA-protein correlations appeared to be a reduced representation of CNA-mRNA correlations. Furthermore, for many CNA regions correlations were more directionally uniform on the protein level than on the mRNA level. CNA regions\",\n",
            "          \"on the protein level than on the mRNA level. CNA regions exhibiting the most trans associations at the protein level were found on chromosomes 5q (LOH in basal; gain in LumB), 10p (gain in basal), 12 (gain in basal), 16q (LumA deletion), 17q (LumB\",\n",
            "          \"in basal), 12 (gain in basal), 16q (LumA deletion), 17q (LumB amplification), and 22q (LOH in luminal and basal) (Extended Data Fig. 5a).\",\n",
            "          \"Trans associations are not necessarily direct consequences of the chromosomal aberration. For example since 5q loss occurs in at least 50% of basal-like breast cancers, many of the trans effects involve genes that mark the basal-subtype. To identify\",\n",
            "          \"effects involve genes that mark the basal-subtype. To identify candidate driver genes whose copy number alterations are direct drivers of trans effects, results were compared with functional knock-down data on 3,797 genes in the Library of Integrated\",\n",
            "          \"knock-down data on 3,797 genes in the Library of Integrated Cellular Signatures (LINCS) database (http://www.lincsproject.org/). For any given gene with copy number alterations (\\u201cCNA-gene\\u201d), sets of genes were identified corresponding to proteins that\",\n",
            "          \"sets of genes were identified corresponding to proteins that changed where there was gain (\\u201cCNA-gain trans gene set\\u201d) or loss (\\u201cCNA-loss trans gene set\\u201d). These gene sets were then compared to the effects of gene knock down in the LINCS database (see\",\n",
            "          \"to the effects of gene knock down in the LINCS database (see Methods). Queries for 502 different CNA-genes meeting the criteria defined above identified 10 CNA-genes that could be functionally connected to both CNA-gain and CNA-loss trans protein-level\",\n",
            "          \"connected to both CNA-gain and CNA-loss trans protein-level effects (Extended Data Fig. 5c, Supplementary Table 11). A permutation-based approach implemented to test significance (see Supplementary Methods) yielded an FDR < 0.05 for 10 genes affected by\",\n",
            "          \"Methods) yielded an FDR < 0.05 for 10 genes affected by both CNA gains and losses (Fig. 2c). These proteins were defined as potential regulatory candidates for the CNA trans effects observed on the proteome level in this study, since in a gene-dependent\",\n",
            "          \"on the proteome level in this study, since in a gene-dependent manner on average 17% of these trans effects were consistent with the knockdown profiles. Notably, the established oncogenic receptor tyrosine kinase ERBB2 was functionally connected only to\",\n",
            "          \"tyrosine kinase ERBB2 was functionally connected only to CNA gain trans effects (Supplementary Table 11). The E3 ligase SKP1 and the ribonucleoprotein export factor CETN3, both located on chromosome arm 5q with frequent losses in basal-like breast cancer\",\n",
            "          \"arm 5q with frequent losses in basal-like breast cancer and less frequent gains in luminal B breast cancer, were detected as potential regulators affecting the expression of the tyrosine kinase and therapeutic target EGFR, and SKP1 also was linked to SRC\",\n",
            "          \"and therapeutic target EGFR, and SKP1 also was linked to SRC (Extended Data Fig. 5d). Another potential regulator, FBXO7, (a substrate recognition component of the SCF (SKP1-CUL1-F-box protein)-type E3 ubiquitin ligase complex), was affected mostly by LOH\",\n",
            "          \"E3 ubiquitin ligase complex), was affected mostly by LOH events on chromosome 22q. Interestingly, in a recent human interaction proteome study SKP1 and FBXO7 were listed as interaction partners.\",\n",
            "          \"Clustering and Network analysis\",\n",
            "          \"Transcriptional profiling has converged on four major breast cancer subtypes: Luminal A and B, basal and HER2-enriched. To investigate the extent to which the PAM50 \\u201cintrinsic\\u201d breast cancer classification scheme is reflected or refined on the proteome\",\n",
            "          \"classification scheme is reflected or refined on the proteome level in the CPTAC samples, clustering analyses were first restricted to the reduced set of PAM50 genes. When RNA data for the 50 PAM50 genes were clustered directly (without using a\",\n",
            "          \"for the 50 PAM50 genes were clustered directly (without using a classifier), the clustering was similar to the TCGA PAM50 annotation (second annotation bar in Fig. 3a). Restricting both the RNA and proteome data to the set of 35 PAM50 genes observed in\",\n",
            "          \"RNA and proteome data to the set of 35 PAM50 genes observed in the proteome produced a similar result (bottom two annotation bars in Fig. 3a), and all the major PAM50 groups were recapitulated in the proteome almost as well as in the RNA data. This\",\n",
            "          \"in the proteome almost as well as in the RNA data. This indicates that although different tissue sections of the same tumors were used for RNA-seq and protein analysis, very similar subtype-defining features can be observed in both data types. Global\",\n",
            "          \"features can be observed in both data types. Global proteome and phosphoproteome data were then used to identify proteome subtypes in an unsupervised manner. Consensus clustering identified basal-enriched, luminal-enriched, and stromal-enriched clusters\",\n",
            "          \"basal-enriched, luminal-enriched, and stromal-enriched clusters (Extended Data Figs. 6a\\u2013d, 7a). Unlike the clustering observed with PAM50 genes, mRNA-defined HER2-enriched tumors were distributed across these three proteomic subgroups. The basal-enriched\",\n",
            "          \"across these three proteomic subgroups. The basal-enriched and luminal-enriched groups showed a strong overlap with the mRNA-based PAM50 basal-like and luminal subgroups, whereas stromal-enriched proteome subtype represented a mix of all PAM50 mRNA-based\",\n",
            "          \"proteome subtype represented a mix of all PAM50 mRNA-based subtypes, and has a significantly enriched stromal signature (Extended Data Fig. 3e). Among the stromal-enriched tumors there was strong representation of reactive type I tumors as classified by\",\n",
            "          \"representation of reactive type I tumors as classified by RPPA (Supplementary Table 12), showing agreement between the RPPA and mass spectrometry-based protein analyses for the detection of a tumor subgroup characterized by stromal gene expression.\",\n",
            "          \"Since the basal- and luminal-enriched proteome subgroups are coherent, pathway analyses were conducted on these two subtypes, using the stromal-enriched subgroup as a control to assess specificity. (Fig. 3c, Extended Data Fig. 7b, Supplementary Table 13).\",\n",
            "          \"(Fig. 3c, Extended Data Fig. 7b, Supplementary Table 13). The luminal-enriched subgroup was exclusively enriched for estradiol and ESR1-driven gene sets. In contrast, multiple gene sets were enriched and upregulated specifically in the basal-like tumors.\",\n",
            "          \"enriched and upregulated specifically in the basal-like tumors. Particularly extensive basal-like enrichment was seen for MYC target genes; for cell cycle, checkpoint, and DNA repair pathways including regulators AURKA/B, ATM, ATR, CHEK1/2, and BRCA1/2;\",\n",
            "          \"including regulators AURKA/B, ATM, ATR, CHEK1/2, and BRCA1/2; and for immune response/inflammation, including T-cell, B-cell, and neutrophil signatures. The complementarity of transcriptional, proteomic, and phosphoproteomic data was also highlighted in\",\n",
            "          \"proteomic, and phosphoproteomic data was also highlighted in these analyses (Extended Data Fig. 7c, d).\",\n",
            "          \"Using phosphorylation status as a proxy for activity, phosphoproteome profiling can theoretically be used to develop a signaling pathway-based cancer classification. K-means consensus clustering was therefore performed on pathways derived from single\",\n",
            "          \"was therefore performed on pathways derived from single sample GSEA analysis of phosphopeptide data (Methods, Supplementary Tables 14 and 15). Of four robustly segregated groups, subgroups 2 and 3 substantially recapitulated the stromal- and\",\n",
            "          \"subgroups 2 and 3 substantially recapitulated the stromal- and luminal-enriched proteomic subgroups, respectively (Fig. 3d, Extended Data Fig. 8a). Subgroup 4 included a majority of tumors from the basal-enriched proteomic subgroup, but was admixed\",\n",
            "          \"from the basal-enriched proteomic subgroup, but was admixed particularly with luminal-enriched samples. This subgroup was defined by high levels of cell cycle and checkpoint activity. All basal and a majority of non-basal samples in this subgroup had TP53\",\n",
            "          \"and a majority of non-basal samples in this subgroup had TP53 mutations. Consistent with high levels of cell cycle activity, a multivariate kinase-to-phosphosite abundance regression analysis highlighted CDK1 as one of the most highly connected kinases in\",\n",
            "          \"highlighted CDK1 as one of the most highly connected kinases in this study (Extended Data Fig. 8b, Supplementary Table 16). Subgroup 1 was a novel subgroup defined exclusively in the phosphopeptide/pathway activity domain, with no enrichment for either\",\n",
            "          \"activity domain, with no enrichment for either proteomic or PAM50 subtypes. It was defined by G-protein, G-protein coupled receptor, and inositol phosphate metabolism signatures, as well as ionotropic glutamate signaling (Fig. 3d). Co-expression patterns\",\n",
            "          \"glutamate signaling (Fig. 3d). Co-expression patterns among genes/proteins across different subgroups were also analyzed using a Joint Random Forest (JRF) method that identified network modules, such as an MMP9 module, with different interaction patterns\",\n",
            "          \"such as an MMP9 module, with different interaction patterns between basal-enriched and luminal-enriched subgroups. These latter patterns appeared specific to the proteome-level data (Extended Data Fig. 8 c\\u2013f, Supplementary Table 17 and Supplementary\",\n",
            "          \"Data Fig. 8 c\\u2013f, Supplementary Table 17 and Supplementary Methods).\",\n",
            "          \"Phosphosite markers in PIK3CA and TP53 mutant breast cancer\",\n",
            "          \"TP53 and PIK3CA are the most recurrently mutated genes in breast cancer, with frequencies for PIK3CA at 43% in luminal tumors and for TP53 at 84% in basal-like tumors. Most of the PIK3CA missense mutations were gain of function mutations and therefore\",\n",
            "          \"mutations were gain of function mutations and therefore expected to lead to an activation of the PI3K signaling cascade, but the extent to which this occurs has been controversial and there is uncertainty which pathway components are effectors. Marker\",\n",
            "          \"is uncertainty which pathway components are effectors. Marker selection analysis was therefore performed for upregulated phosphosites in PIK3CA-mutated tumors. In total, 62 phosphosites were identified that were positively associated with PIK3CA mutation\",\n",
            "          \"identified that were positively associated with PIK3CA mutation (FDR<0.05), including the kinases RPS6KA5 and EIF2AK4 (Extended Data Fig. 9a, Supplementary Table 18). Calculating the average phosphorylation signal of these marker phosphosites provided a\",\n",
            "          \"phosphorylation signal of these marker phosphosites provided a read-out for PI3K pathway activity in PIK3CA-mutated tumors, with 15 of the 26 mutated tumors (58%) exhibiting an activated PIK3CA mutation signature. Of note, the identified PIK3CA mutant\",\n",
            "          \"mutation signature. Of note, the identified PIK3CA mutant phosphoproteome signature was activated in all tumors harboring helical domain PIK3CA mutations but only 2 of 10 tumors harboring kinase domain mutations. To test if the identified differences in\",\n",
            "          \"domain mutations. To test if the identified differences in the phosphoproteome of PI3K mutant versus wild-type tumors could be explained by mutation of PIK3CA, the tumor data were compared to phosphosite signatures derived from isogenic PIK3CA mutant cell\",\n",
            "          \"phosphosite signatures derived from isogenic PIK3CA mutant cell lines (Extended Data Fig. 9b, Supplementary Table 18). There was an enrichment of signatures derived from helical domain-mutated isogenic cell lines, but not from kinase domain-mutated cells,\",\n",
            "          \"isogenic cell lines, but not from kinase domain-mutated cells, supporting the observations in primary tumors.\",\n",
            "          \"The same strategy was used to identify phosphorylation signaling events connected to TP53 mutation. A total of 56 phosphosites upregulated in TP53 mutant tumors were identified that were independent of basal-like subtype association (Extended Data Fig.\",\n",
            "          \"of basal-like subtype association (Extended Data Fig. 9c, Supplementary Table 18). Using the average phosphorylation signal of these marker phosphosites as a proxy for TP53 mutation-driven cell cycle control, 22 of 41 mutated tumors (54%) showed\",\n",
            "          \"cell cycle control, 22 of 41 mutated tumors (54%) showed upregulated signals. This TP53 mutant phosphosignature was somewhat enhanced in tumors in which mutations occurred almost exclusively in the DNA binding region compared to those with\",\n",
            "          \"exclusively in the DNA binding region compared to those with non-sense/frameshift mutations. In addition to the well-described checkpoint kinase CHEK2, significantly upregulated phosphosites were identified for the kinases MASTL and EEF2K in TP53-mutated\",\n",
            "          \"were identified for the kinases MASTL and EEF2K in TP53-mutated tumors. Single sample GSEA analysis of isogenic p53-mutant phosphosignatures showed an enrichment of a phosphosignature derived from R273H mutated isogenic cells (Extended Data Fig. 9d),\",\n",
            "          \"from R273H mutated isogenic cells (Extended Data Fig. 9d), confirming the pronounced effect of missense mutations in the DNA-binding region on phosphorylation pathways.\",\n",
            "          \"Identification of gene amplification and breast cancer subtype-specific activated kinases in human breast cancer\",\n",
            "          \"CNA spans many driver gene candidates and RNA expression has been frequently used to narrow candidate nominations. Proteogenomic analysis should further promote this nomination process. In this candidate refinement a focus on protein kinases is warranted,\",\n",
            "          \"candidate refinement a focus on protein kinases is warranted, since many are drug targets. An in-depth proteogenomic pipeline was developed that flagged kinases, expression levels of which were at least 1.5 interquartile ranges higher than the median\",\n",
            "          \"were at least 1.5 interquartile ranges higher than the median (Supplemental Table 19). A proteogenomic circos-like plot (termed a \\u201cpircos\\u201d plot) was used to map these outlier kinase values onto the genome (Fig. 4a,b, Extended Data Fig. 10a). The ERBB2\",\n",
            "          \"onto the genome (Fig. 4a,b, Extended Data Fig. 10a). The ERBB2 locus showed the strongest effect of increased phosphoprotein levels associated with gene amplification-driven RNA and protein over expression (Fig. 4a). CDK12 is a positive transcriptional\",\n",
            "          \"over expression (Fig. 4a). CDK12 is a positive transcriptional regulator of homologous recombination repair genes with its partner Cyclin K, and is often encompassed by the ERBB2 amplicon. This gene was also found to be upregulated at the RNA, protein and\",\n",
            "          \"gene was also found to be upregulated at the RNA, protein and phosphosite level indicating that CDK12 is highly active in the majority of ERBB2 positive tumors (Fig. 4a). The analysis of the ERBB2 amplicon also uncovered co-outlier phosphorylation status\",\n",
            "          \"ERBB2 amplicon also uncovered co-outlier phosphorylation status for MED1, GRB7, MSL1, CASC3 and TOP2A, all previously described in association with ERBB2 amplification. To better understand the downstream effects of ERBB2 amplification, additional\",\n",
            "          \"the downstream effects of ERBB2 amplification, additional phosphosite outliers were identified in 41 known ERBB2 signaling genes for the 15 samples that had ERBB2 phosphosite outlier expression (Extended Data Fig. 10b).\",\n",
            "          \"These canonical findings stimulated a proteogenomic analysis to identify additional outlier kinases in the breast cancer genome. A proteogenomic dissection of chromosome 11q based on PAK1 amplification (Fig. 4b,c), a breast cancer driver kinase,\",\n",
            "          \"PAK1 amplification (Fig. 4b,c), a breast cancer driver kinase, illustrated that PAK1 is hyperphosphorylated in PAK1 amplified tumors, along with CLNS1A, RFS1 and GAB2. Additional examples of outlier kinases included PTK2 and RIPK2 in association with\",\n",
            "          \"of outlier kinases included PTK2 and RIPK2 in association with amplification of chromosome 8q (Fig. 4c; Extended Data Fig. 10a,c). PAK1 and TLK2 (17q23) appear to be luminal breast cancer specific events Fig. 4c; Extended Data Fig. 10c). To further\",\n",
            "          \"specific events Fig. 4c; Extended Data Fig. 10c). To further examine whether outlier kinases were breast cancer subtype-specific, independent of amplification status, the BH-corrected probability was calculated of finding that number of phosphosite\",\n",
            "          \"was calculated of finding that number of phosphosite outliers within a subtype, given the total number of outliers across all subtypes, the subtype sample size and the total sample size. (Fig. 4d). These analyses led to the expected identification of\",\n",
            "          \"(Fig. 4d). These analyses led to the expected identification of ERBB2 in the HER2-enriched subtype at the 5% FDR level, as well as the new finding of CDC42BPG (MRGKG), an effector kinase for RHO-family GTPases. In basal-like breast cancer, two kinases,\",\n",
            "          \"RHO-family GTPases. In basal-like breast cancer, two kinases, PRKDC and SPEG, were significant at the 5% FDR level. PRKDC is a non-homologous end-joining (NHEJ) factor that can be phosphorylated by ATM kinase, and is therefore a logical finding in this\",\n",
            "          \"by ATM kinase, and is therefore a logical finding in this disease subset. However SPEG, a kinase associated with severe dilated cardiomyopathy when suppressed, has not been previously reported in association with breast cancer. A larger number of\",\n",
            "          \"reported in association with breast cancer. A larger number of subtype-specific kinases were detected at the 10% FDR level, several of which have recently described relevance in breast cancer, including PRKD3 in basal-like breast cancer, the LKB-regulated\",\n",
            "          \"including PRKD3 in basal-like breast cancer, the LKB-regulated SIK3 in luminal A breast cancer and CDK13 in luminal B breast cancer, which, similar to CDK12, can interact with Cyclin K.\",\n",
            "          \"Discussion\",\n",
            "          \"The analytic breadth and depth of proteomic and phosphoproteomic analyses displayed in this study demonstrates the strengths of mass spectrometry-based proteomics, but also some of the limitations inherent in proteolytic peptide sequencing (see\",\n",
            "          \"the limitations inherent in proteolytic peptide sequencing (see Supplementary Discussion). An example of how high-dimensional proteomic analysis provides insight into unresolved genomic issues concerns the study of loss of the long arm of chromosome 5\",\n",
            "          \"concerns the study of loss of the long arm of chromosome 5 (5q). Analysis of RNA and protein correlations narrowed the list of potential trans-deregulated proteins. Orthogonal candidate screening using functional genomics methods identified loss of CETN3\",\n",
            "          \"using functional genomics methods identified loss of CETN3 and SKP1 as potential transregulators, with upregulation of EGFR as a downstream consequence in basal-like breast cancers. While further experimental evidence must be sought for these proposed\",\n",
            "          \"further experimental evidence must be sought for these proposed regulatory relationships, SKP1/Cullin complex has already been linked to EGFR activation in Glioma. Unfortunately EGFR targeting has not to date proven to be effective therapy in basal-like\",\n",
            "          \"has not to date proven to be effective therapy in basal-like breast cancer. This might be due to the fact the SKP1 loss deregulates multiple targets requiring a much broader inhibitory strategy.\",\n",
            "          \"It is recognized that PIK3CA mutations do not strongly activate canonical downstream effectors. Mass spectrometry-based phosphoproteomics provides an opportunity for unbiased examination of downstream signaling events consequent upon PIK3CA mutational\",\n",
            "          \"downstream signaling events consequent upon PIK3CA mutational activation. These studies revealed that common PIK3CA mutations affect a large number of targets with diverse functionalities including the kinases RPS6KA5 and EIF2AK4. Thus, the data and\",\n",
            "          \"including the kinases RPS6KA5 and EIF2AK4. Thus, the data and analyses reported here extend our knowledge of the effectors that promote tumorigenesis in response to constitutive activation of PI3 kinase. Similarly, TP53 mutation-associated phosphopeptides\",\n",
            "          \"PI3 kinase. Similarly, TP53 mutation-associated phosphopeptides point towards novel functionalities, including regulation of the kinases MASTL and EEF2K.\",\n",
            "          \"A central goal in breast cancer research has been the identification of druggable kinases beyond HER2. Candidate genes that exhibited similar gene amplification-driven proteogenomic patterns to HER2 included CDK12, TLK2, PAK1 and RIPK2. The proteogenomic\",\n",
            "          \"to HER2 included CDK12, TLK2, PAK1 and RIPK2. The proteogenomic link with gene amplification was particularly strong for CDK12, in keeping with its location in the ERBB2 amplicon, while the strengths of correlation between DNA amplification, RNA, protein\",\n",
            "          \"of correlation between DNA amplification, RNA, protein and phosphoprotein for the other examples were more variable. The presence of activated CDK12 in the ERBB2 amplicon might explain why tumors arising in BRCA1 carriers are usually ERBB2 negative. As a\",\n",
            "          \"arising in BRCA1 carriers are usually ERBB2 negative. As a positive transcriptional regulator of BRCA1 and multiple FANC family members, CDK12 promotes DNA repair by homologous recombination. CDK12 amplification would, therefore, oppose the functional\",\n",
            "          \"CDK12 amplification would, therefore, oppose the functional effects of BRCA1 haploinsufficiency during tumor evolution. Overall, multiple outlier kinases generate testable therapeutic hypotheses for which enabling inhibitors are in development. For\",\n",
            "          \"for which enabling inhibitors are in development. For example PAK1 has recently been confirmed to be a therapeutic target and poor prognosis factor in luminal breast cancer.\",\n",
            "          \"Although incomplete outcome data and the remarkable heterogeneity of breast cancer are additional relevant constraints, the number of TCGA specimens analyzed here is insufficient to support conclusive clinical correlations. Only 8 deaths occurred among\",\n",
            "          \"conclusive clinical correlations. Only 8 deaths occurred among the 77 patients, which are too few samples to provide sufficient statistical power for association analysis. Adequately powered MS-based clinical investigation will require targeted\",\n",
            "          \"powered MS-based clinical investigation will require targeted approaches, especially given the highly limited amount of patient material available from clinical trials and the mostly formalin-fixed nature of the specimens. The current analysis is\",\n",
            "          \"formalin-fixed nature of the specimens. The current analysis is therefore centered on biological findings and correlations, with orthogonal validation and false discovery concerns addressed through an examination of cell-line databases of the effects of\",\n",
            "          \"through an examination of cell-line databases of the effects of individual gene perturbations. Typical of a multi-tiered analysis of this complexity, there are many hypotheses to test, and many findings that require further investigation.\",\n",
            "          \"In conclusion, this study provides a high-quality proteomic resource for human breast cancer investigation and illustrates technologies and analytical approaches that provide an important new opportunity to connect the genome to the proteome. Larger-scale\",\n",
            "          \"opportunity to connect the genome to the proteome. Larger-scale exploration of discovery proteomics in the clinical setting will require improvements in clinical investigation, including acquisition of adequate amounts of optimally collected tumor tissue\",\n",
            "          \"of adequate amounts of optimally collected tumor tissue both before and during therapy as well as advances in MS proteomics to reduce sample input and increase sensitivity for low abundance proteins and modified peptides.\",\n",
            "          \"Extended Data\\n\\nExperimental and data analysis workflows and longitudinal data generation quality control\",\n",
            "          \"a, iTRAQ 4-plex global proteome and phosphoproteome analysis workflow. 105 TCGA breast tumors were analyzed in 35 iTRAQ 4-plex experiments (plus 1 replicate and 1 normal sample experiment), with three tumors of different subtypes compared to a fourth\",\n",
            "          \"with three tumors of different subtypes compared to a fourth common internal reference sample in each experiment. The reference sample comprised 10 individual tumors of each of the 4 major breast cancer intrinsic subtypes and served as an internal\",\n",
            "          \"breast cancer intrinsic subtypes and served as an internal standard for all proteins and phosphoproteins quantified in this study. Each iTRAQ MS/MS spectrum measures a peptide from 4 samples (3 individual patients and the reference sample mix of 40\",\n",
            "          \"(3 individual patients and the reference sample mix of 40 patients). More than 400,000 distinct peptides were identified and quantified in ~14 million MS/MS spectra. Personalized tumor-specific protein databases were generated in the QUILTS software\",\n",
            "          \"protein databases were generated in the QUILTS software package using whole exome sequencing-derived variant calls and RNAseq-derived transcript information. All mass spectrometry data was analyzed using the Spectrum Mill software package. b, Overview of\",\n",
            "          \"using the Spectrum Mill software package. b, Overview of proteome and phosphoproteome datasets. The table provides a summary of the datasets used in specific analyses, including the filters applied to derive the proteins and phosphosites/phosphoproteins\",\n",
            "          \"applied to derive the proteins and phosphosites/phosphoproteins that constitute each dataset; the protein, phosphosite or phosphoprotein count; and the methods that employ the respective datasets. c, Distribution of sequence coverage of the identified\",\n",
            "          \"c, Distribution of sequence coverage of the identified proteins with tryptic peptides detected by MS/MS, whiskers show the 5\\u201395 percentiles. d and e, Robust and accurate proteome/phosphoproteome platform. Longitudinal performance was tested by repeated\",\n",
            "          \"platform. Longitudinal performance was tested by repeated proteome and phosphoproteome analysis of patient-derived xenograft tumors. Scatterplots, histograms and Pearson correlations comparing individual replicate measurements are shown.\",\n",
            "          \"Tumor sample quality control (I)\",\n",
            "          \"a, Remark diagram showing sample processing and partitioning. Initial quality review encompassed histopathological examination of H&E stained tissue slices. *For 3 samples no tumor cells were seen on histopathology (BH-A0E9, BH-A0C1, A2-A0SW). These\",\n",
            "          \"were seen on histopathology (BH-A0E9, BH-A0C1, A2-A0SW). These samples were nevertheless included in the proteome analysis since other quality control standards were met (see below) and samples with 0% tumor cellularity on top or bottom sections were\",\n",
            "          \"with 0% tumor cellularity on top or bottom sections were included in TCGA analyses. b, correlation of TCGA (top or bottom sections) and CPTAC histological assessment of neoplastic cellularity for samples (n = 105). The average and range of neoplastic\",\n",
            "          \"for samples (n = 105). The average and range of neoplastic cellularities were identical for CPTAC and TCGA histological assessments. Averages (standard deviations) for neoplastic cellularity were 76% (+/\\u2212 17) for CPTAC, 76% (+/\\u2212 15) for TCGA_Top, and 75%\",\n",
            "          \"were 76% (+/\\u2212 17) for CPTAC, 76% (+/\\u2212 15) for TCGA_Top, and 75% (+/\\u2212 18) for TCGA_Bottom histopathology slides (Supplementary Table 2). Note that in three CPTAC cases where no tumor cells were identified by histopathological assessment, numbers of\",\n",
            "          \"were identified by histopathological assessment, numbers of protein-level somatic variants were similar to all other tumors. The identified mutated proteins were TP53_R273C, NOP58_Q23E, TAGLN2_G154R, TUBA1B_D116H, and MRPL48_I173K (Supplementary Table 5),\",\n",
            "          \"TUBA1B_D116H, and MRPL48_I173K (Supplementary Table 5), indicating presence of tumor cells in these samples. c, Proteome iTRAQ tumor/internal reference ratio heatmap for all CPTAC samples (8,028 proteins without missing values) including passed and failed\",\n",
            "          \"proteins without missing values) including passed and failed proteomic quality control (QC) samples. d, Global tumor/reference proteome ratio distributions for samples that passed and failed proteomic quality control analysis. e, Degradation-related gene\",\n",
            "          \"proteomic quality control analysis. e, Degradation-related gene sets were enriched in tumors that failed proteomic quality control analysis. f, Variant allele frequency (VAF) analysis of re-sequenced CPTAC tumors and comparison to original TCGA data.\",\n",
            "          \"re-sequenced CPTAC tumors and comparison to original TCGA data. Overall VAFs for failed QC samples were lower compared to passed samples suggesting lower purity.\",\n",
            "          \"Tumor sample quality control (II)\",\n",
            "          \"a, There was high concordance (94.6%) between DNA variants reported by TCGA and CPTAC re-sequenced tumors. Most point mutations reported by TCGA could be identified across the 8 re-sequenced samples used in the study. b, A high overall correlation\",\n",
            "          \"samples used in the study. b, A high overall correlation (mean=0.77) was observed for the CPTAC Variant Allele Fraction (VAF) (X-axis) and TCGA VAF (Y-axis) across the 8 samples used in the study. c, Agglomerative hierarchical clustering (Supplementary\",\n",
            "          \"study. c, Agglomerative hierarchical clustering (Supplementary Methods Section 3.8) used to co-cluster protein and RNA tumor expression data after filtering to retain 4,291 proteins and genes with moderate to high protein-RNA correlation (Pearson\",\n",
            "          \"genes with moderate to high protein-RNA correlation (Pearson correlation > 0.4) with results displayed as a circular dendrogram (fanplot). The proteome (.P) and RNA (.R) components of each sample are labeled using the same color. The outer ring shows\",\n",
            "          \"sample are labeled using the same color. The outer ring shows proteome samples in light grey and RNA samples in dark grey. High concordance between RNA and protein expression is evident from the color adjacency in the inner ring and alternating color in\",\n",
            "          \"the color adjacency in the inner ring and alternating color in the outer ring showing that RNA and protein components co-cluster for a large proportion of samples (62/80). d, Co-clustering of MS and RPPA tumor data. 126 RPPA readouts were mapped to gene\",\n",
            "          \"MS and RPPA tumor data. 126 RPPA readouts were mapped to gene names. These genes were intersected with the genes observed in the MS proteome, filtered to 48 proteins with moderate or higher RPPA-MS protein correlation, and analyzed for co-clustering as in\",\n",
            "          \"protein correlation, and analyzed for co-clustering as in c. 47 of 80 RPPA-MS protein pairs co-cluster. While this is a smaller proportion than for RNA-protein analysis, the number of genes used in the clustering is significantly smaller for RPPA (48 vs.\",\n",
            "          \"in the clustering is significantly smaller for RPPA (48 vs. 4,291 for RNA). e, ESTIMATE tumor purity comparison between mRNA, RNAseq, and proteome data. ANOVA is used to assess the difference in distribution (\\u2212log10(p-value)) of ESTIMATE, stromal, immune,\",\n",
            "          \"in distribution (\\u2212log10(p-value)) of ESTIMATE, stromal, immune, and tumor purity scores across mRNA (microarray), RNA-seq and proteome data. The only significant p-value (=0.02) is for the Cluster 3 stromal score, and higher stromal scores for the\",\n",
            "          \"the Cluster 3 stromal score, and higher stromal scores for the proteome drive that difference. f, Ischemia score analysis. Comparison of ischemia scores of 77 CPTAC tumors, 3 normal samples, and patient-derived xenografts. CPTAC tumors had generally lower\",\n",
            "          \"patient-derived xenografts. CPTAC tumors had generally lower ischemia scores than PDX samples subjected to 30 minutes of cold ischemia. Median ischemia scores are less than 30 minutes for each subtype and no significant differences were observed across\",\n",
            "          \"subtype and no significant differences were observed across subtypes. Effects due to cold ischemia therefore appear to be negligible in this CPTAC sample collection.\",\n",
            "          \"Protein-to-Protein, -CNA, and -mRNA correlation analyses\",\n",
            "          \"a, Identification of UBE3A as an E3 ubiquitin ligase that negatively correlates to p53 on the protein level. Pearson correlation and Benjamini-Hochberg corrected p-value are shown. b, Analysis of counter-regulated genes with negative correlation of\",\n",
            "          \"of counter-regulated genes with negative correlation of CNA-to-RNA as well as CNA-to-protein levels. Negative Pearson correlations are shown with Benjamini-Hochberg corrected p-values for CNA-to-protein correlations. Depicted genes have significant\",\n",
            "          \"CNA-to-protein correlations. Depicted genes have significant negative correlations at FDR<0.05 in the CNA-to-RNA and CNA-to-protein analyses. c, Global mRNA-to-protein correlation and gene set enrichment analysis.\",\n",
            "          \"Global CNA effects and comparison of CNA TRANS effects to knockdown signatures in the LINCS database\",\n",
            "          \"a, CNA landscape in the CPTAC tumor collection. The segment-based CNAs of 77 samples were downloaded from TCGA Firehose, including 18 Basal, 12 Her2, 23 Luminal A and 24 Luminal B subtypes. Copy number amplifications were marked in red and deletions in\",\n",
            "          \"Copy number amplifications were marked in red and deletions in blue. The bottom color key represents the log2 transformed copy number value, with CNA=2 centered at 0. Specific CNA events are seen for chromosome 5q and 10p regions in basal-like tumors. b,\",\n",
            "          \"seen for chromosome 5q and 10p regions in basal-like tumors. b, Correlations of copy number alterations (x-axis) to phosphoprotein levels (y-axis) highlight new CNA cis and trans effects. Significant (FDR<0.05) positive (red) and negative (green)\",\n",
            "          \"Significant (FDR<0.05) positive (red) and negative (green) correlations between CNA and phosphoproteins are indicated. Histograms show the fraction [%] of significant CNA trans effects for each CNA gene. c, LINCS CMap analysis facilitates identification\",\n",
            "          \"CNA gene. c, LINCS CMap analysis facilitates identification of novel functional candidates for CNA trans effects. Knockdown profiles were compared with CNA/protein trans effects for 502 genes. Genes with a connectivity score >|90| were considered\",\n",
            "          \"genes. Genes with a connectivity score >|90| were considered connected and significant cis effects were annotated at an FDR<0.05. d, Basal-like tumor-specific CNAs are candidate regulatory events for EGFR and SRC expression levels. Oncogenic kinases with\",\n",
            "          \"for EGFR and SRC expression levels. Oncogenic kinases with significant CNA/protein trans effects (left panel), that were regulated in LINCS shRNA experiments (right panel; 4 cell lines,) and directly measured as LINCS landmark genes, are shown alongside\",\n",
            "          \"directly measured as LINCS landmark genes, are shown alongside candidate regulatory genes CETN3 and SKP1. Clinical ER, PR, and HER2 annotation and PAM50 classification are shown in the header rows of each column.\",\n",
            "          \"Proteome cluster heatmap and stability analysis\",\n",
            "          \"a, K-means consensus clustering of proteome and phosphoproteome data identifies three subgroups: basal-enriched, luminal-enriched, and stromal-enriched. The heatmap represents all 1,521 proteins used for clustering (Dataset G8). b, Identification of\",\n",
            "          \"proteins used for clustering (Dataset G8). b, Identification of optimal proteome clusters for QC-passed CPTAC breast cancer tumors. Proteome clusters were derived using consensus clustering based on 1000 resampled datasets, exploring the range of 2 to 6\",\n",
            "          \"based on 1000 resampled datasets, exploring the range of 2 to 6 k-means clusters. Visualization of consensus matrices from k-means consensus clustering for k=3, 4, 5 and 6 target clusters. Consensus clustering was performed on 1,521 proteins with no\",\n",
            "          \"Consensus clustering was performed on 1,521 proteins with no missing values and SD>1.5. c, Silhouette plots were generated to evaluate the coherence of the clustering. Silhouette plots for k=3 and k=4 clusters showing a cleaner separation of clusters for\",\n",
            "          \"and k=4 clusters showing a cleaner separation of clusters for k=3. d, Based on both visual inspection of the consensus matrix and the delta plot assessing change in consensus cumulative distribution function (CDF) area, three robustly segregated groups\",\n",
            "          \"function (CDF) area, three robustly segregated groups were observed. Consensus cumulative distribution function (CDF) and delta area (change in CDF area) plots for 2\\u20136 clusters.\",\n",
            "          \"Proteome cluster markers and enriched pathways\",\n",
            "          \"a, Markers (based on SAM analysis; FDR<0.01) discriminate between proteome clusters 1, 2 and 3 (compare to heatmap of proteins used to derive clusters depicted in Extended Data Fig. 6a). b, Applying a Fisher exact test-based enrichment analysis to the\",\n",
            "          \"Applying a Fisher exact test-based enrichment analysis to the proteome, phosphoproteome and mRNA data, gene sets from MSigDB were identified that were unique for each proteome cluster. Heat map showing specific pathways comprising dominant biological\",\n",
            "          \"map showing specific pathways comprising dominant biological themes that are significantly differential by enrichment analysis between basal-enriched and luminal-enriched tumors (Fisher Exact Test Benjamini-Hochberg corrected p-values are shown;\",\n",
            "          \"Exact Test Benjamini-Hochberg corrected p-values are shown; enrichment test performed on marker sets identified using SAM analysis; see Methods; compare to Figure 3c). c, Heatmap showing a selection of gene sets significant in basal-enriched or\",\n",
            "          \"a selection of gene sets significant in basal-enriched or luminal-enriched tumors exclusively by mRNA, protein or phosphoprotein expression. Cytokine signatures, for example, were strongly captured at the mRNA level, but were seen to only a limited degree\",\n",
            "          \"at the mRNA level, but were seen to only a limited degree at the global protein level, likely because of their typically low protein abundance. By contrast, the vast majority of significant gene sets annotated as \\\"signaling\\\" were enriched only at the\",\n",
            "          \"gene sets annotated as \\\"signaling\\\" were enriched only at the phosphoprotein level. d, Global heat map representing all gene sets significantly enriched in at least one of the proteomic breast cancer subtypes. The stromal-enriched group was characterized\",\n",
            "          \"cancer subtypes. The stromal-enriched group was characterized by breast cancer normal-like, adipocyte differentiation, smooth muscle, toll-like receptor signaling and endothelin gene sets, supporting the clustering-based annotation of high stromal and/or\",\n",
            "          \"the clustering-based annotation of high stromal and/or adipose content in these tumors (see Supplemental Table 13).\",\n",
            "          \"Phosphoproteome pathway clustering, kinase-phosphosite multivariate regression, and protein co-expression networks\",\n",
            "          \"a, Phosphoproteome pathway clustering. Using phosphorylation state as a proxy for activity, deep phosphoproteome profiling allows development of a breast cancer molecular taxonomy based on signaling pathways. K-means consensus clustering was performed on\",\n",
            "          \"pathways. K-means consensus clustering was performed on pathways derived from single sample GSEA analysis of phosphopeptide data (908 pathways shown). Of four robustly segregated groups, subgroups 2 and 3 substantially recapitulated the stromal- and\",\n",
            "          \"subgroups 2 and 3 substantially recapitulated the stromal- and luminal-enriched proteomic subgroups, respectively. Subgroup 4 included a significant majority of tumors from the basal-enriched proteomic subgroup, but was admixed particularly with\",\n",
            "          \"proteomic subgroup, but was admixed particularly with luminal-enriched samples. This subgroup was defined by high levels of cell cycle and checkpoint activity. All basal and a majority of non-basal samples in this subgroup had TP53 mutations. Subgroup 1\",\n",
            "          \"samples in this subgroup had TP53 mutations. Subgroup 1 was a novel subgroup defined exclusively in the phosphopeptide / pathway activity domain, with no enrichment for either proteomic or PAM50 subtypes. It was defined by G-protein, G-protein coupled\",\n",
            "          \"PAM50 subtypes. It was defined by G-protein, G-protein coupled receptor, and inositol phosphate metabolism signatures, as well as ionotropic glutamate signaling. b, Analysis of the regulatory relationship between outlier kinases (see Supplementary Table\",\n",
            "          \"relationship between outlier kinases (see Supplementary Table 19) and phosphopeptides by regulatory multivariate regression analysis (see Methods) identified CDK1 as the most highly connected of the outlier Cyclin-Dependent Kinases, with highest\",\n",
            "          \"connected of the outlier Cyclin-Dependent Kinases, with highest centrality (based on node-degree; see Methods) among the outlier CDKs and seventh highest centrality among all the outlier kinases considered in the remMap analysis. Each line represents a\",\n",
            "          \"considered in the remMap analysis. Each line represents a phosphosite-kinase relationship. c\\u2013f, Analysis of differences in the co-expression patterns among genes/proteins across different subgroups. A Joint Random Forest (JRF) method was applied to\",\n",
            "          \"subgroups. A Joint Random Forest (JRF) method was applied to simultaneously build gene co-expression and protein co-expression networks (Supplementary Table 17, and Methods). Modules in these networks revealed different interaction patterns between\",\n",
            "          \"these networks revealed different interaction patterns between basal-enriched and luminal-enriched subgroups. c, Network module P1 of the protein co-expression network, defined chiefly in the proteome space. This module contained 12 genes connected by 39\",\n",
            "          \"proteome space. This module contained 12 genes connected by 39 edges, among which 34 were protein-specific and 5 were shared by both the protein and mRNA co-expression networks. Many edges were supported by published information and were contained in the\",\n",
            "          \"supported by published information and were contained in the STRING database. Edges in red are specific to the protein co-expression network; edges in green are shared by both protein and gene co-expression networks; edges indicated by double lines are\",\n",
            "          \"co-expression networks; edges indicated by double lines are contained in the STRING database with confidence score greater than 0.15. MMP9, one of the central proteins in this module, contributes to metastatic progression and is a potential target for\",\n",
            "          \"to metastatic progression and is a potential target for anti-metastatic therapies for basal-like / triple negative breast cancer. d, Heatmaps of the absolute correlation across each pair of genes in module P1 (shown in Panel c), based on either protein or\",\n",
            "          \"in module P1 (shown in Panel c), based on either protein or gene expression data for samples in the basal-enriched and luminal-enriched subgroups, respectively. The MMP9 protein was strongly co-expressed with the other members of the module only in the\",\n",
            "          \"co-expressed with the other members of the module only in the basal-enriched subgroup. Notably, this observation is dependent on protein data; the correlation at the mRNA level for this module was consistently low in both the basal-enriched and luminal\",\n",
            "          \"was consistently low in both the basal-enriched and luminal enriched subgroups indicating that these events coherently occur at the proteomic level. e, Co-expression network based on proteomics data. The network contains 693 proteomic network-specific\",\n",
            "          \"data. The network contains 693 proteomic network-specific edges (grey) and 792 edges shared with the RNAseq network (green). For each module, the most enriched category and corresponding Benjamini-Hochberg adjusted p-value is reported. Pie charts adjacent\",\n",
            "          \"adjusted p-value is reported. Pie charts adjacent to each module show the proportion of proteomics-specific edges (grey area) and edges shared between proteomics and RNAseq data (green area). f, RNAseq network.\",\n",
            "          \"Phosphoproteome signatures of PIK3CA (a,b) and TP53 (c,d) mutated tumors highlight activated key regulators and indicate frequency of activation\",\n",
            "          \"a and c, Phosphosites upregulated in mutated tumors (SAM FDR<0.05 across all tumors and independently also across luminal tumors; average phosphosite signal for all markers shown as bar graph). To avoid confounding by intrinsic subtype-specific\",\n",
            "          \"bar graph). To avoid confounding by intrinsic subtype-specific distinctions, only markers that were significantly identified both in analyses covering all tumors and analyses restricted to luminal tumors were selected (FDR <0.05). Color bars in the\",\n",
            "          \"to luminal tumors were selected (FDR <0.05). Color bars in the margins indicate FDRs for grouped analysis of different mutation classes and indicate kinase substrates of known kinases in the respective pathways. Significantly regulated kinase phosphosites\",\n",
            "          \"pathways. Significantly regulated kinase phosphosites are annotated. The average phosphorylation signal of the marker phosphosites provides a read-out for PI3K and TP53 pathway activity in mutated tumors (histogram below heatmap). A 95% prediction\",\n",
            "          \"in mutated tumors (histogram below heatmap). A 95% prediction confidence interval (indicated by dashed lines) across the average signal in non-mutated tumors was chosen in order to discriminate active from non-active tumors. The most strongly activated\",\n",
            "          \"active from non-active tumors. The most strongly activated PIK3CA kinase domain mutant tumor differed from the other 9 kinase domain mutant tumors, as it contained an amino acid side chain charge neutral H1047L instead of the more common positively\",\n",
            "          \"charge neutral H1047L instead of the more common positively charged H1047R mutation. Among the 62 phosphosites identified that were significantly upregulated in PIK3CA mutated tumors, 13 phosphosites were found on phosphoproteins that are known substrates\",\n",
            "          \"were found on phosphoproteins that are known substrates of well-annotated kinases in the PIK3CA pathway (panel a, right column). In the mutant TP53 analysis a total 20 phosphosites were found on phosphoproteins that are known substrates of well annotated\",\n",
            "          \"on phosphoproteins that are known substrates of well annotated kinases in the p53 pathway (panel c, right column). b and d, Upregulated phosphosite sets were derived from isogenic PIK3CA and TP53 mutant versus wild-type cell line pairs and tested for\",\n",
            "          \"and TP53 mutant versus wild-type cell line pairs and tested for enrichment within mutant versus wild-type CPTAC tumors using single sample GSEA. Significantly enriched phosphosite sets are shown (p<0.05).\",\n",
            "          \"PIRCOS plots, kinase outliers and outliers in the ERBB2 pathway\",\n",
            "          \"a, Pircos (Proteogenomics CIRCOS) plots for 8q and 17q showing median CNA, RNA, protein, and phosphosite expression for 20 tumors with amplification in 8q based on RIPK2 CNA>1; 23 tumors with amplification in 8q based on PTK2 CNA>1; 15 tumors with\",\n",
            "          \"with amplification in 8q based on PTK2 CNA>1; 15 tumors with amplification in 17q based on CDK12 CNA >1; and 10 tumors with amplification in 17q based on TLK2 CNA>1. Red indicates expression >1, blue < \\u22121, and grey between \\u22121 and 1. Genes with both copy\",\n",
            "          \">1, blue < \\u22121, and grey between \\u22121 and 1. Genes with both copy number amplification (CNA>1) and increased phosphosite expression (p-site>1) are labeled. b, Phosphosite outliers in known ERBB2 signaling genes. To better understand the downstream effects of\",\n",
            "          \"signaling genes. To better understand the downstream effects of ERBB2 amplification, phosphosite outliers in known ERBB2 signaling genes (MSigDB pathway set, KEGG_ERBB_SIGNALING PATHWAY) were identified for the 15 samples that had ERBB2 phosphosite\",\n",
            "          \"were identified for the 15 samples that had ERBB2 phosphosite outlier status. Forty-one genes were identified as having a phosphosite outlier in at least one of the ERBB2 amplified samples. PAK4 and ARAF phosphosite outlier status were found in seven of\",\n",
            "          \"PAK4 and ARAF phosphosite outlier status were found in seven of the 15 ERBB2 kinase outlier samples; GSK3B outliers were found in 6 samples; and EIF4EBP1, MAP2K2, ABL1 and AKT1 outlier status was found in 5 of the 15 samples. c, Proteogenomic outlier\",\n",
            "          \"was found in 5 of the 15 samples. c, Proteogenomic outlier expression analysis for TLK2 and RIPK2. Samples with outlier phosphosite (red), protein (yellow), RNA (green) and copy number (purple) expression are shown. Phosphosite squares indicate per-sample\",\n",
            "          \"expression are shown. Phosphosite squares indicate per-sample outlier phosphosites.\",\n",
            "          \"Supplementary Material\\n\\nSupplementary Information is available in the online version of the paper.\",\n",
            "          \"Author Contributions P.M., D.R.M., M.A.G., K.R.C., and S.A.C. designed the proteomic analysis experiments, data analysis workflow, and proteomic-genomic data comparisons. P.M., M.A.G., J.W.Q., and S.A.C. directed and performed proteomic analysis of breast\",\n",
            "          \"and S.A.C. directed and performed proteomic analysis of breast tumor and quality control samples. P.M., D.R.M., K.V.R., K.R.C., P.W., X.W., S.C., E.K., F.P., Z.T., J.L., M.L.G., M.W., V.Y., K.H., C.L., M.D.M., P.Y., J.W., B.Z., and D.F. performed\",\n",
            "          \"V.Y., K.H., C.L., M.D.M., P.Y., J.W., B.Z., and D.F. performed proteomic-genomic data analyses. D.R.M., P.W., and S.J.S. provided statistical support. D.R.M., K.V.R., K.R.C., K.K. and D.F. performed analyses of mass spectrometry data and adapted\",\n",
            "          \"D.F. performed analyses of mass spectrometry data and adapted algorithms and software for data analysis. S.R.D., R.R.T and M.J.E. developed and prepared breast xenografts used as quality control samples. P.M. and F.M prepared and analyzed cell lines for\",\n",
            "          \"samples. P.M. and F.M prepared and analyzed cell lines for correlative functional annotation of frequently mutated genes. P.M., D.R.M., M.A.G., and S.A.C designed strategy for quality control analyses. M.A.G., S.R.D., C.R.K., M.M., and H.R. coordinated\",\n",
            "          \"analyses. M.A.G., S.R.D., C.R.K., M.M., and H.R. coordinated acquisition, distribution and quality control evaluation of TCGA tumor samples. P.M., M.A.G., C.P., L.D., A.G.P., and M.J.E. interpreted data in the context of breast cancer biology. P.M.,\",\n",
            "          \"interpreted data in the context of breast cancer biology. P.M., D.R.M, M.A.G., K.R.C., P.W., A.G.P, M.J.E. and S.A.C. wrote the manuscript.\",\n",
            "          \"All primary mass spectrometry data are deposited at the CPTAC Data Portal as raw and mzML files and complete protein assembly data sets for public access (https://cptac-data-portal.georgetown.edu/cptac/s/S029). In addition, a set of ancillary files such\",\n",
            "          \"In addition, a set of ancillary files such as dataset G1/P1, G3/P3, G4/P4, G5/P5, G7/P7, CNA correlation tables for CNA-mRNA, CNA-proteome and CNA-phosphoproteome; CNA data, RNA-seq expression data have also been deposited at the DCC. Two browsers have\",\n",
            "          \"data have also been deposited at the DCC. Two browsers have been created to assist the interested reader in exploring the results. One provides track hubs for viewing the identified peptides in the UCSD genome browser\",\n",
            "          \"for viewing the identified peptides in the UCSD genome browser (http://fenyolab.org/cptac_breast_ucsc). The second is an on-line tool for proteogenomic data exploration and can be accessed at http://prot-shiny-vm.broadinstitute.org:3838/BC2016/. See\",\n",
            "          \"at http://prot-shiny-vm.broadinstitute.org:3838/BC2016/. See Supplemental Methods for descriptions.\",\n",
            "          \"The authors declare no competing financial interests.\\n\\nProteogenomic analysis of human breast cancer. Direct effects of genomic alterations on protein level\",\n",
            "          \"Overlap of a, protein coding single amino acid variants (SAAVs) and b, RNA splice junctions not present in RefSeq v60 detected by DNA exome sequencing, RNA-seq, and LC-MS/MS. Proportions of novel variants are noted. c, Heatmap of mutations/CNA and their\",\n",
            "          \"novel variants are noted. c, Heatmap of mutations/CNA and their effects on RNA and protein expression of breast cancer-relevant genes across tumor and normal samples. ER, PR, HER2 and PAM50 status are annotated. Median iTRAQ protein abundance ratio and\",\n",
            "          \"status are annotated. Median iTRAQ protein abundance ratio and the most frequently detected and differential phosphosite ratio are shown for each gene. Pearson correlations between MS protein vs RNA-seq and MS protein vs RPPA are indicated.\",\n",
            "          \"Effects of copy number alterations (CNA) on mRNA, protein, and phosphoprotein abundance\",\n",
            "          \"a, Correlations of CNA (x-axes) to RNA and protein expression levels (y-axes) highlight new CNA cis and trans effects. Significant (FDR<0.05) positive (red) and negative (green) correlations between CNA and mRNAs or proteins are indicated. CNA cis effects\",\n",
            "          \"CNA and mRNAs or proteins are indicated. CNA cis effects appear as a red diagonal line, CNA trans effects as vertical stripes. Histograms show the fraction [%] of significant CNA trans effects for each CNA gene. b, Overlap of cis effects observed at RNA,\",\n",
            "          \"for each CNA gene. b, Overlap of cis effects observed at RNA, protein, and phosphoprotein levels (FDR<0.05). c, Trans-effect regulatory candidates identified among those with significant protein cis-effects using LINCS CMap. Bars indicate total numbers of\",\n",
            "          \"cis-effects using LINCS CMap. Bars indicate total numbers of significant CNA/protein trans effects (gray; FDR<0.05) and overlap with regulated genes in LINCS knock-down profiles (red; 4 cell lines; moderated T-test FDR<0.1).\",\n",
            "          \"Proteomic and phosphoproteomic subtypes of breast cancer and subtype-specific pathway enrichment\",\n",
            "          \"a, Unsupervised clustering of RNA-seq and proteomics data restricted to PAM50 genes and subset of 35 detected proteins reveal high similarity to PAM50 (TCGA) sample annotation. b, K-means consensus clustering of proteome and phosphoproteome data\",\n",
            "          \"consensus clustering of proteome and phosphoproteome data identifies basal-enriched, luminal-enriched, and stromal-enriched subgroups. c, Gene set enrichment analysis highlights sets of pathways significantly differential between basal-enriched and\",\n",
            "          \"pathways significantly differential between basal-enriched and luminal-enriched tumors (detailed in Extended Data Fig. 7b). d, K-means consensus clustering performed on pathways derived from single sample GSEA analysis of phosphopeptide data identifies\",\n",
            "          \"single sample GSEA analysis of phosphopeptide data identifies four distinct clusters.\",\n",
            "          \"Example analyses of aberrantly regulated kinases in human breast cancer\",\n",
            "          \"a and b, PIRCOS (Proteogenomics CIRCOS) plots showing CNA, RNA, protein and phosphosite expression for 17 tumors with amplification in 17q (ERBB2 CNA>1) and 8 tumors with amplification in 11q (PAK1 CNA>1). Labeled genes have CNA>1 and phosphosite>1. c,\",\n",
            "          \"(PAK1 CNA>1). Labeled genes have CNA>1 and phosphosite>1. c, Proteogenomic outlier expression analysis for ERBB2, CDK12, and PAK1. Samples with outlier phosphosite (red), protein (yellow), RNA (green) and copy number (purple) expression are shown.\",\n",
            "          \"RNA (green) and copy number (purple) expression are shown. Phosphosite squares indicate per-sample outlier phosphosites. d, Outlier kinase events by PAM50 subtype (>35% of subtype samples contain a phosphosite outlier; <10% FDR using Benjamini-Hochberg\",\n",
            "          \"a phosphosite outlier; <10% FDR using Benjamini-Hochberg adjusted p-values).\"\n",
            "        ]\n",
            "      }\n",
            "    ]\n",
            "  },\n",
            "  \"PDC000290\": {\n",
            "    \"study_id\": \"57b07c60-c9e1-424f-bfd6-9262e06a77d8\",\n",
            "    \"pdc_study_id\": \"PDC000290\",\n",
            "    \"study_submitter_id\": \"TCGA Breast Cancer CompRef Samples - Proteome\",\n",
            "    \"program_id\": \"10251935-5540-11e8-b664-00a098d917f8\",\n",
            "    \"project_id\": \"c96eca75-8b73-4018-9bf7-b20e31a965cd\",\n",
            "    \"study_name\": \"TCGA Breast Cancer CompRef Samples - Proteome\",\n",
            "    \"program_name\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"project_name\": \"CPTAC2 Retrospective\",\n",
            "    \"disease_type\": \"Other\",\n",
            "    \"primary_site\": \"Not Reported\",\n",
            "    \"analytical_fraction\": \"Proteome\",\n",
            "    \"experiment_type\": \"iTRAQ4\",\n",
            "    \"embargo_date\": null,\n",
            "    \"cases_count\": 2,\n",
            "    \"aliquots_count\": 20,\n",
            "    \"protocol_id\": \"7b457134-224f-4c69-b5a8-f62a99666489\",\n",
            "    \"protocol_submitter_id\": \"TCGA Breast Cancer CompRef Proteome\",\n",
            "    \"program_submitter_id\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"protocol_name\": \"TCGA Breast Cancer CompRef Proteome\",\n",
            "    \"protocol_date\": \"2021-04-07\",\n",
            "    \"document_name\": \"CPTAC-SOP-Broad_Proteome/Phosphoproteome_3.0.txt\",\n",
            "    \"quantitation_strategy\": \"Isobaric label quantitation analysis\",\n",
            "    \"label_free_quantitation\": \"N/A\",\n",
            "    \"labeled_quantitation\": \"iTRAQ\",\n",
            "    \"isobaric_labeling_reagent\": \"iTRAQ4\",\n",
            "    \"reporter_ion_ms_level\": \"MS2\",\n",
            "    \"starting_amount\": \"4.00\",\n",
            "    \"starting_amount_uom\": \"mg\",\n",
            "    \"digestion_reagent\": \"Lys-C,Trypsin\",\n",
            "    \"alkylation_reagent\": \"Iodoacetamide (IAA)\",\n",
            "    \"enrichment_strategy\": \"Proteome\",\n",
            "    \"enrichment\": \"None\",\n",
            "    \"chromatography_dimensions_count\": \"2\",\n",
            "    \"one_d_chromatography_type\": \"bRPLC (pH 10)\",\n",
            "    \"two_d_chromatography_type\": \"RPLC\",\n",
            "    \"fractions_analyzed_count\": \"25\",\n",
            "    \"column_type\": \"PicoFrit SELF/P\",\n",
            "    \"amount_on_column\": \"1.0\",\n",
            "    \"amount_on_column_uom\": \"\\u00b5g\",\n",
            "    \"column_length\": \"20.0\",\n",
            "    \"column_length_uom\": \"cm\",\n",
            "    \"column_inner_diameter\": \"75.00\",\n",
            "    \"column_inner_diameter_uom\": \"\\u00b5m\",\n",
            "    \"particle_size\": \"1.9\",\n",
            "    \"particle_size_uom\": \"\\u00b5m\",\n",
            "    \"particle_type\": \"ReproSil-Pur C18-AQ\",\n",
            "    \"gradient_length\": \"110.0\",\n",
            "    \"gradient_length_uom\": \"min\",\n",
            "    \"instrument_make\": \"Thermo Fisher Scientific\",\n",
            "    \"instrument_model\": \"Q Exactive\",\n",
            "    \"dissociation_type\": \"HCD\",\n",
            "    \"ms1_resolution\": \"70000\",\n",
            "    \"ms2_resolution\": \"17500\",\n",
            "    \"dda_topn\": \"Top 12\",\n",
            "    \"normalized_collision_energy\": \"28\",\n",
            "    \"acquistion_type\": \"DDA\",\n",
            "    \"dia_multiplexing\": \"N/A\",\n",
            "    \"dia_ims\": \"N/A\",\n",
            "    \"analytical_technique\": null,\n",
            "    \"chromatography_instrument_make\": null,\n",
            "    \"chromatography_instrument_model\": null,\n",
            "    \"polarity\": null,\n",
            "    \"reconstitution_solvent\": null,\n",
            "    \"reconstitution_volume\": null,\n",
            "    \"reconstitution_volume_uom\": null,\n",
            "    \"internal_standards\": null,\n",
            "    \"extraction_method\": null,\n",
            "    \"ionization_mode\": null,\n",
            "    \"publications\": [\n",
            "      {\n",
            "        \"publication_id\": \"44e2e766-89c7-11e8-bcf1-0a2705229b82\",\n",
            "        \"pubmed_id\": \"27251275\",\n",
            "        \"doi\": \"10.1038/nature18003\",\n",
            "        \"author\": \"Mertins, P.; Mani, D. R.; Ruggles, K. V.; Gillette, M. A.; Clauser, K. R.; Wang, P.; Wang, X.; Qiao, J. W.; Cao, S.; Petralia, F.; Kawaler, E.; Mundt, F.; Krug, K.; Tu, Z.; Lei, J. T.; Gatza, M. L.; Wilkerson, M.; Perou, C. M.; Yellapantula, V.; Huang, K. L.; Lin, C.; McLellan, M. D.; Yan, P.; Davies, S. R.; Townsend, R. R.; Skates, S. J.; Wang, J.; Zhang, B.; Kinsinger, C. R.; Mesri, M.; Rodriguez, H.; Ding, L.; Paulovich, A. G.; Feny\\u00f6, D.; Ellis, M. J.; Carr, S. A.; Gil Kuhn, E.; Mani, D. R.; Mertins, P.; Ketchum, K. A.; Thangudu, R. R.; Cai, S.; Oberti, M.; Whiteaker, J. R.; Wang, X.; Lin, C.; Ping, Y.; Edwards, N. J.; Madhavan, S.; McGarvey, P. B.; Wang, P.; Petralia, F.; Tu, Z.; Chan, D.; Pandey, A.; Shih, L. M.; Zhang, H.; Zhang, Z.; Thomas, S.; Zhu, H.; Whiteley, G. A.; Skates, S. J.; White, F. M.; Levine, D. A.; Boja, E. S.; Kinsinger, C. R.; Hiltke, T.; Mesri, M.; Rivers, R. C.; Rodriguez, H.; Shaw, K. M.; Stein, S. E.; Fenyo, D.; Liu, T.; McDermott, J. E.; Payne, S. H.; Rodland, K. D.; Smith, R. D.; Rudnick, P.; Snyder, M.; Zhao, Y.; Chen, X.; Ransohoff, D. F.; Hoofnagle, A. N.; Liebler, D. C.; Sanders, M. E.; Shi, Z.; Slebos, R. J.; Tabb, D. L.; Zhang, B.; Zimmerman, L. J.; Wang, Y.; Li, S.; Davies, S. R.; Ding, L.; Maher, C.; Townsend, R.; Ellis, M. J.; Lei, J. T.; Luo, J.\",\n",
            "        \"title\": \"Proteogenomics connects somatic mutations to signalling in breast cancer\",\n",
            "        \"journal\": \"Nature\",\n",
            "        \"journal_url\": \"https://www.nature.com/articles/nature18003\",\n",
            "        \"year\": \"2016\",\n",
            "        \"abstract\": \"Somatic mutations have been extensively characterized in breast cancer, but the effects of these genetic alterations on the proteomic landscape remain poorly understood. Here we describe quantitative mass-spectrometry-based proteomic and phosphoproteomic analyses of 105 genomically annotated breast cancers, of which 77 provided high-quality data. Integrated analyses provided insights into the somatic cancer genome including the consequences of chromosomal loss, such as the 5q deletion characteristic of basal-like breast cancer. Interrogation of the 5q trans-effects against the Library of Integrated Network-based Cellular Signatures, connected loss of CETN3 and SKP1 to elevated expression of epidermal growth factor receptor (EGFR), and SKP1 loss also to increased SRC tyrosine kinase. Global proteomic data confirmed a stromal-enriched group of proteins in addition to basal and luminal clusters, and pathway analysis of the phosphoproteome identified a G-protein-coupled receptor cluster that was not readily identified at the mRNA level. In addition to ERBB2, other amplicon-associated highly phosphorylated kinases were identified, including CDK12, PAK1, PTK2, RIPK2 and TLK2. We demonstrate that proteogenomic analysis of breast cancer elucidates the functional consequences of somatic mutations, narrows candidate nominations for driver genes within large deletions and amplified regions, and identifies therapeutic targets.\",\n",
            "        \"full_text\": [\n",
            "          \"Proteogenomics connects somatic mutations to signaling in breast cancer\\n\\nSummary\",\n",
            "          \"Somatic mutations have been extensively characterized in breast cancer, but the effects of these genetic alterations on the proteomic landscape remain poorly understood. We describe quantitative mass spectrometry-based proteomic and phosphoproteomic\",\n",
            "          \"mass spectrometry-based proteomic and phosphoproteomic analyses of 105 genomically annotated breast cancers of which 77 provided high-quality data. Integrated analyses allowed insights into the somatic cancer genome including the consequences of\",\n",
            "          \"into the somatic cancer genome including the consequences of chromosomal loss, such as the 5q deletion characteristic of basal-like breast cancer. The 5q trans effects were interrogated against the Library of Integrated Network-based Cellular Signatures,\",\n",
            "          \"the Library of Integrated Network-based Cellular Signatures, thereby connecting CETN3 and SKP1 loss to elevated expression of EGFR, and SKP1 loss also to increased SRC. Global proteomic data confirmed a stromal-enriched group in addition to basal and\",\n",
            "          \"confirmed a stromal-enriched group in addition to basal and luminal clusters and pathway analysis of the phosphoproteome identified a G Protein-coupled receptor cluster that was not readily identified at the mRNA level. Besides ERBB2, other\",\n",
            "          \"not readily identified at the mRNA level. Besides ERBB2, other amplicon-associated, highly phosphorylated kinases were identified, including CDK12, PAK1, PTK2, RIPK2 and TLK2. We demonstrate that proteogenomic analysis of breast cancer elucidates\",\n",
            "          \"that proteogenomic analysis of breast cancer elucidates functional consequences of somatic mutations, narrows candidate nominations for driver genes within large deletions and amplified regions, and identifies therapeutic targets.\",\n",
            "          \"Introduction\",\n",
            "          \"A central deficiency in our knowledge of cancer concerns how genomic changes drive the proteome and phosphoproteome to execute phenotypic characteristics. The initial proteomic characterization in the TCGA breast study was performed using reversed phase\",\n",
            "          \"in the TCGA breast study was performed using reversed phase protein arrays; however this approach is restricted by antibody availability. To provide greater analytical breadth, the NCI Clinical Proteomic Tumor Analysis Consortium (CPTAC) is analyzing the\",\n",
            "          \"Proteomic Tumor Analysis Consortium (CPTAC) is analyzing the proteomes of genome-annotated TCGA tumor specimens using mass spectrometry. Herein we describe integrated proteogenomic analyses of TCGA breast cancer samples representing the four principal\",\n",
            "          \"of TCGA breast cancer samples representing the four principal mRNA-defined breast cancer intrinsic subtypes.\",\n",
            "          \"Proteogenomic analysis of TCGA samples\",\n",
            "          \"105 breast tumors previously characterized by the TCGA were selected for proteomic analysis after histopathological documentation (Supplementary Tables 1 and 2). The cohort included a balanced representation of PAM50-defined intrinsic subtypes including\",\n",
            "          \"representation of PAM50-defined intrinsic subtypes including 25 basal-like, 29 luminal A, 33 luminal B, and 18 HER2 (ERBB2)-enriched tumors, along with 3 normal breast tissue samples. Samples were analyzed by high-resolution, accurate mass, tandem mass\",\n",
            "          \"were analyzed by high-resolution, accurate mass, tandem mass spectrometry (MS) that included extensive peptide fractionation and phosphopeptide enrichment (Extended Data Fig. 1a). An isobaric peptide labeling approach (iTRAQ) was employed to quantify\",\n",
            "          \"peptide labeling approach (iTRAQ) was employed to quantify protein and phosphosite levels across samples, with 37 iTRAQ 4-plexes analyzed in total. A total of 15,369 proteins (12,405 genes) and 62,679 phosphosites were confidently identified with 11,632\",\n",
            "          \"and 62,679 phosphosites were confidently identified with 11,632 proteins/tumor and 26,310 phosphosites/tumor on average (Supplementary Tables 3, 4 and Supplementary Methods). After filtering for observation in at least a quarter of the samples\",\n",
            "          \"filtering for observation in at least a quarter of the samples (Supplementary Methods, Extended Data Fig. 1b) 12,553 proteins (10,062 genes) and 33,239 phosphosites, with their relative abundances quantified across tumors, were used in subsequent analyses\",\n",
            "          \"quantified across tumors, were used in subsequent analyses in this study. Stable longitudinal performance and low technical noise were demonstrated by repeated, interspersed analyses of a single batch of patient-derived luminal and basal breast cancer\",\n",
            "          \"single batch of patient-derived luminal and basal breast cancer xenograft samples (Extended Data Fig. 1d,e). Due to the heterogeneous nature of breast tumors, and because proteomic analyses were performed on tumor fragments that were different from those\",\n",
            "          \"performed on tumor fragments that were different from those used in the genomic analyses, rigorous pre-specified sample and data QC metrics were implemented (Supplementary Discussion and Extended Data Figures 2, 3). Extensive analyses concluded that 28 of\",\n",
            "          \"Data Figures 2, 3). Extensive analyses concluded that 28 of the 105 samples were compromised by protein degradation. These samples were excluded from further analysis with subsequent informatics focused on the 77 tumor samples and three biological\",\n",
            "          \"focused on the 77 tumor samples and three biological replicates.\",\n",
            "          \"Genome and transcriptomic variation was observed at the peptide level by searching MS/MS spectra not matched to RefSeq against a patient-specific sequence database (Fig. 1a). The database was constructed using the QUILTS software package leveraging RefSeq\",\n",
            "          \"constructed using the QUILTS software package leveraging RefSeq gene models based on whole exome and RNA-seq data generated from portions of the same tumors and matched germline DNA (Fig. 1a, Supplementary Table 5). While these analyses detected a number\",\n",
            "          \"Supplementary Table 5). While these analyses detected a number of single amino-acid variants (SAAVs), frameshifts, and splice junctions, including splice isoforms that had been detected as only single transcript reads by RNA-seq (Fig. 1b, Supplementary\",\n",
            "          \"only single transcript reads by RNA-seq (Fig. 1b, Supplementary Table 5), the number of genomic and transcriptomic variants that were confirmed as peptides by MS was low (Supplementary Discussion). Sparse detection of individual genomic variants by\",\n",
            "          \"Discussion). Sparse detection of individual genomic variants by peptide sequencing has been noted in our previous studies and reflects limited coverage at the single amino-acid level with current technology. However quantitative MS analysis of multiple\",\n",
            "          \"technology. However quantitative MS analysis of multiple peptides for each protein is used to reliably infer overall protein levels. This is an advantage for MS since antibody-based protein expression analysis is typically based on a single epitope. To\",\n",
            "          \"expression analysis is typically based on a single epitope. To illustrate this capability in the current data set an initial analysis of three frequently mutated genes in breast cancer (TP53, PIK3CA, and GATA3) and three clinical biomarkers (ER, PGR, and\",\n",
            "          \"PIK3CA, and GATA3) and three clinical biomarkers (ER, PGR, and ERBB2) was conducted (Fig. 1c, Supplementary Table 6, 7 and Supplementary Discussion). As expected, TP53 missense mutations were associated with elevated MS-based protein levels, as observed\",\n",
            "          \"associated with elevated MS-based protein levels, as observed by RPPA (Reverse Phase Protein Array), especially in basal-like breast cancer. TP53 nonsense and frame-shift mutations were associated with a decrease in TP53 protein levels that was\",\n",
            "          \"were associated with a decrease in TP53 protein levels that was particularly striking in the MS data. In contrast, the mostly C-terminal GATA3 frame-shift alterations did not result in decreased protein expression when measured by the median of all GATA3\",\n",
            "          \"protein expression when measured by the median of all GATA3 peptides, suggesting these proteins are expressed despite truncation. No consistent effect of somatic PIK3CA mutation was observed at the level of protein expression. Good correlations between\",\n",
            "          \"at the level of protein expression. Good correlations between RNA-seq and MS-protein expression levels were found for ESR1 (r=0.74), PGR (r=0.74), ERBB2 (r=0.84) and GATA3 (r=0.83) with moderate correlations observed for PIK3CA (r=0.45) and TP53 (r=0.36).\",\n",
            "          \"correlations observed for PIK3CA (r=0.45) and TP53 (r=0.36). Lower TP53 protein abundance levels compared to mRNA levels were especially prevalent in luminal tumors, suggesting post-transcriptional regulatory mechanisms such as proteasomal degradation. To\",\n",
            "          \"regulatory mechanisms such as proteasomal degradation. To explore this hypothesis a search was made for E3 ligases that showed negative correlation to p53 protein (Supplementary Table 8). These analyses identified UBE3A (r=\\u22120.42; adj. pval=0.05) (Extended\",\n",
            "          \"analyses identified UBE3A (r=\\u22120.42; adj. pval=0.05) (Extended Data Fig. 4a), an established TP53 E3 ligase. In comparing CNA, RNA, and protein levels for GATA3, copy number gains in chromosome 10q were anti-correlated with RNA and protein levels in\",\n",
            "          \"10q were anti-correlated with RNA and protein levels in basal-like tumors. This observation prompted a search for other gains or losses that were anticorrelated with RNA and/or protein levels (see Extended Data Fig. 4b for further analyses). Overall, six\",\n",
            "          \"(see Extended Data Fig. 4b for further analyses). Overall, six genes were identified that significantly anti-correlated at an FDR<0.05 on both RNA and protein level to their CNA signals (Extended Data Fig. 4b). GATA3 amplification on 10q in basal-like\",\n",
            "          \"Data Fig. 4b). GATA3 amplification on 10q in basal-like breast cancer showed the strongest anti-correlation, followed by the hexosamine and glycolysis pathway enzymes GFPT2 and HK3, which are upregulated in basal-like breast cancer despite being subjected\",\n",
            "          \"upregulated in basal-like breast cancer despite being subjected to frequent chromosomal deletion on 5q. Global analysis of the correlation of mRNA-to-protein yielded a median Pearson value of r=0.39, with 6,135 out of 9,302 protein/mRNA pairs (66.0%)\",\n",
            "          \"of r=0.39, with 6,135 out of 9,302 protein/mRNA pairs (66.0%) correlating significantly at an FDR<0.05 (Extended Data Fig. 4c, Supplementary Table 9 and Supplementary Discussion). Similar to the colon cancer analysis metabolic functions such as amino\",\n",
            "          \"to the colon cancer analysis metabolic functions such as amino acid, sugar and fatty acid metabolism were found to be enriched among positively correlated genes whereas ribosomal, RNA polymerase and mRNA splicing functions were negatively correlated.\",\n",
            "          \"and mRNA splicing functions were negatively correlated. Overall these analyses demonstrate the utility of global proteome correlation analysis for both confirmation of suspected regulatory mechanisms and identification of candidate regulators meriting\",\n",
            "          \"mechanisms and identification of candidate regulators meriting further investigation.\",\n",
            "          \"Copy Number Alterations\",\n",
            "          \"To determine the consequences of CNA on mRNA, protein and phosphoprotein abundance, both in \\u201ccis\\u201d on genes within the aberrant locus and in \\u201ctrans\\u201d on genes encoded elsewhere, univariate correlation analysis was used as previously described. A total of\",\n",
            "          \"analysis was used as previously described. A total of 7,776 genes with CNA, mRNA and protein measurements were analyzed by calculating Pearson correlation and associated statistical significance (Benjamini-Hochberg corrected p-value) for all possible\",\n",
            "          \"(Benjamini-Hochberg corrected p-value) for all possible CNA-mRNA and CNA-protein pairs (Fig. 2a, Supplementary Table 10, Extended Data Fig. 5a, see Methods). For the phosphoproteome, 4,472 CNA-phosphoprotein pairs were analyzed (Extended Data Fig. 5b).\",\n",
            "          \"CNA-phosphoprotein pairs were analyzed (Extended Data Fig. 5b). Significant positive correlations (cis) were observed for 64% of all CNA-mRNA, 31% of all CNA-protein, and 20% of all CNA-phosphoprotein pairs Fig. 2b. Proteins and phosphoproteins correlated\",\n",
            "          \"pairs Fig. 2b. Proteins and phosphoproteins correlated in cis to CNA were, for the most part, a subset of the cis effects observed in mRNA-to-CNA correlation (Fig. 2b, Supplementary Table 10). The fractional difference of well-annotated oncogenes and\",\n",
            "          \"10). The fractional difference of well-annotated oncogenes and tumor suppressor genes among the significantly cis-correlated CNA-to-mRNA and -protein gene pairs was analyzed. Based on a reference list of 487 oncogenes and tumor suppressors (Supplementary\",\n",
            "          \"list of 487 oncogenes and tumor suppressors (Supplementary Table 10), these cancer relevant genes occur 37.6% more frequently in the subset of genes that correlate both on CNA/mRNA and CNA/protein levels than in the subset that only correlate on CNA/mRNA\",\n",
            "          \"levels than in the subset that only correlate on CNA/mRNA but not on CNA/protein level (Fisher exact p-value=0.02). This suggests that CNA events with a tumor promoting outcome more likely lead to cis regulatory effects on both the protein and mRNA level,\",\n",
            "          \"to cis regulatory effects on both the protein and mRNA level, whereas CNA events with no documented role in tumorigenesis are more likely to be neutralized on the protein level than on the RNA level. Trans effects (Fig. 2a) appear as vertical bands, with\",\n",
            "          \"level. Trans effects (Fig. 2a) appear as vertical bands, with accompanying frequency histograms (in blue) highlighting \\u201chot spots\\u201d of significant trans effects. Using a minimum threshold of 50 trans-affected genes, 68% of the tested genes were associated\",\n",
            "          \"trans-affected genes, 68% of the tested genes were associated with trans effects on the mRNA level, whereas only 13% were associated with effects on the protein level and 8% on the phosphoprotein level. Importantly, CNA-protein correlations appeared to be\",\n",
            "          \"level. Importantly, CNA-protein correlations appeared to be a reduced representation of CNA-mRNA correlations. Furthermore, for many CNA regions correlations were more directionally uniform on the protein level than on the mRNA level. CNA regions\",\n",
            "          \"on the protein level than on the mRNA level. CNA regions exhibiting the most trans associations at the protein level were found on chromosomes 5q (LOH in basal; gain in LumB), 10p (gain in basal), 12 (gain in basal), 16q (LumA deletion), 17q (LumB\",\n",
            "          \"in basal), 12 (gain in basal), 16q (LumA deletion), 17q (LumB amplification), and 22q (LOH in luminal and basal) (Extended Data Fig. 5a).\",\n",
            "          \"Trans associations are not necessarily direct consequences of the chromosomal aberration. For example since 5q loss occurs in at least 50% of basal-like breast cancers, many of the trans effects involve genes that mark the basal-subtype. To identify\",\n",
            "          \"effects involve genes that mark the basal-subtype. To identify candidate driver genes whose copy number alterations are direct drivers of trans effects, results were compared with functional knock-down data on 3,797 genes in the Library of Integrated\",\n",
            "          \"knock-down data on 3,797 genes in the Library of Integrated Cellular Signatures (LINCS) database (http://www.lincsproject.org/). For any given gene with copy number alterations (\\u201cCNA-gene\\u201d), sets of genes were identified corresponding to proteins that\",\n",
            "          \"sets of genes were identified corresponding to proteins that changed where there was gain (\\u201cCNA-gain trans gene set\\u201d) or loss (\\u201cCNA-loss trans gene set\\u201d). These gene sets were then compared to the effects of gene knock down in the LINCS database (see\",\n",
            "          \"to the effects of gene knock down in the LINCS database (see Methods). Queries for 502 different CNA-genes meeting the criteria defined above identified 10 CNA-genes that could be functionally connected to both CNA-gain and CNA-loss trans protein-level\",\n",
            "          \"connected to both CNA-gain and CNA-loss trans protein-level effects (Extended Data Fig. 5c, Supplementary Table 11). A permutation-based approach implemented to test significance (see Supplementary Methods) yielded an FDR < 0.05 for 10 genes affected by\",\n",
            "          \"Methods) yielded an FDR < 0.05 for 10 genes affected by both CNA gains and losses (Fig. 2c). These proteins were defined as potential regulatory candidates for the CNA trans effects observed on the proteome level in this study, since in a gene-dependent\",\n",
            "          \"on the proteome level in this study, since in a gene-dependent manner on average 17% of these trans effects were consistent with the knockdown profiles. Notably, the established oncogenic receptor tyrosine kinase ERBB2 was functionally connected only to\",\n",
            "          \"tyrosine kinase ERBB2 was functionally connected only to CNA gain trans effects (Supplementary Table 11). The E3 ligase SKP1 and the ribonucleoprotein export factor CETN3, both located on chromosome arm 5q with frequent losses in basal-like breast cancer\",\n",
            "          \"arm 5q with frequent losses in basal-like breast cancer and less frequent gains in luminal B breast cancer, were detected as potential regulators affecting the expression of the tyrosine kinase and therapeutic target EGFR, and SKP1 also was linked to SRC\",\n",
            "          \"and therapeutic target EGFR, and SKP1 also was linked to SRC (Extended Data Fig. 5d). Another potential regulator, FBXO7, (a substrate recognition component of the SCF (SKP1-CUL1-F-box protein)-type E3 ubiquitin ligase complex), was affected mostly by LOH\",\n",
            "          \"E3 ubiquitin ligase complex), was affected mostly by LOH events on chromosome 22q. Interestingly, in a recent human interaction proteome study SKP1 and FBXO7 were listed as interaction partners.\",\n",
            "          \"Clustering and Network analysis\",\n",
            "          \"Transcriptional profiling has converged on four major breast cancer subtypes: Luminal A and B, basal and HER2-enriched. To investigate the extent to which the PAM50 \\u201cintrinsic\\u201d breast cancer classification scheme is reflected or refined on the proteome\",\n",
            "          \"classification scheme is reflected or refined on the proteome level in the CPTAC samples, clustering analyses were first restricted to the reduced set of PAM50 genes. When RNA data for the 50 PAM50 genes were clustered directly (without using a\",\n",
            "          \"for the 50 PAM50 genes were clustered directly (without using a classifier), the clustering was similar to the TCGA PAM50 annotation (second annotation bar in Fig. 3a). Restricting both the RNA and proteome data to the set of 35 PAM50 genes observed in\",\n",
            "          \"RNA and proteome data to the set of 35 PAM50 genes observed in the proteome produced a similar result (bottom two annotation bars in Fig. 3a), and all the major PAM50 groups were recapitulated in the proteome almost as well as in the RNA data. This\",\n",
            "          \"in the proteome almost as well as in the RNA data. This indicates that although different tissue sections of the same tumors were used for RNA-seq and protein analysis, very similar subtype-defining features can be observed in both data types. Global\",\n",
            "          \"features can be observed in both data types. Global proteome and phosphoproteome data were then used to identify proteome subtypes in an unsupervised manner. Consensus clustering identified basal-enriched, luminal-enriched, and stromal-enriched clusters\",\n",
            "          \"basal-enriched, luminal-enriched, and stromal-enriched clusters (Extended Data Figs. 6a\\u2013d, 7a). Unlike the clustering observed with PAM50 genes, mRNA-defined HER2-enriched tumors were distributed across these three proteomic subgroups. The basal-enriched\",\n",
            "          \"across these three proteomic subgroups. The basal-enriched and luminal-enriched groups showed a strong overlap with the mRNA-based PAM50 basal-like and luminal subgroups, whereas stromal-enriched proteome subtype represented a mix of all PAM50 mRNA-based\",\n",
            "          \"proteome subtype represented a mix of all PAM50 mRNA-based subtypes, and has a significantly enriched stromal signature (Extended Data Fig. 3e). Among the stromal-enriched tumors there was strong representation of reactive type I tumors as classified by\",\n",
            "          \"representation of reactive type I tumors as classified by RPPA (Supplementary Table 12), showing agreement between the RPPA and mass spectrometry-based protein analyses for the detection of a tumor subgroup characterized by stromal gene expression.\",\n",
            "          \"Since the basal- and luminal-enriched proteome subgroups are coherent, pathway analyses were conducted on these two subtypes, using the stromal-enriched subgroup as a control to assess specificity. (Fig. 3c, Extended Data Fig. 7b, Supplementary Table 13).\",\n",
            "          \"(Fig. 3c, Extended Data Fig. 7b, Supplementary Table 13). The luminal-enriched subgroup was exclusively enriched for estradiol and ESR1-driven gene sets. In contrast, multiple gene sets were enriched and upregulated specifically in the basal-like tumors.\",\n",
            "          \"enriched and upregulated specifically in the basal-like tumors. Particularly extensive basal-like enrichment was seen for MYC target genes; for cell cycle, checkpoint, and DNA repair pathways including regulators AURKA/B, ATM, ATR, CHEK1/2, and BRCA1/2;\",\n",
            "          \"including regulators AURKA/B, ATM, ATR, CHEK1/2, and BRCA1/2; and for immune response/inflammation, including T-cell, B-cell, and neutrophil signatures. The complementarity of transcriptional, proteomic, and phosphoproteomic data was also highlighted in\",\n",
            "          \"proteomic, and phosphoproteomic data was also highlighted in these analyses (Extended Data Fig. 7c, d).\",\n",
            "          \"Using phosphorylation status as a proxy for activity, phosphoproteome profiling can theoretically be used to develop a signaling pathway-based cancer classification. K-means consensus clustering was therefore performed on pathways derived from single\",\n",
            "          \"was therefore performed on pathways derived from single sample GSEA analysis of phosphopeptide data (Methods, Supplementary Tables 14 and 15). Of four robustly segregated groups, subgroups 2 and 3 substantially recapitulated the stromal- and\",\n",
            "          \"subgroups 2 and 3 substantially recapitulated the stromal- and luminal-enriched proteomic subgroups, respectively (Fig. 3d, Extended Data Fig. 8a). Subgroup 4 included a majority of tumors from the basal-enriched proteomic subgroup, but was admixed\",\n",
            "          \"from the basal-enriched proteomic subgroup, but was admixed particularly with luminal-enriched samples. This subgroup was defined by high levels of cell cycle and checkpoint activity. All basal and a majority of non-basal samples in this subgroup had TP53\",\n",
            "          \"and a majority of non-basal samples in this subgroup had TP53 mutations. Consistent with high levels of cell cycle activity, a multivariate kinase-to-phosphosite abundance regression analysis highlighted CDK1 as one of the most highly connected kinases in\",\n",
            "          \"highlighted CDK1 as one of the most highly connected kinases in this study (Extended Data Fig. 8b, Supplementary Table 16). Subgroup 1 was a novel subgroup defined exclusively in the phosphopeptide/pathway activity domain, with no enrichment for either\",\n",
            "          \"activity domain, with no enrichment for either proteomic or PAM50 subtypes. It was defined by G-protein, G-protein coupled receptor, and inositol phosphate metabolism signatures, as well as ionotropic glutamate signaling (Fig. 3d). Co-expression patterns\",\n",
            "          \"glutamate signaling (Fig. 3d). Co-expression patterns among genes/proteins across different subgroups were also analyzed using a Joint Random Forest (JRF) method that identified network modules, such as an MMP9 module, with different interaction patterns\",\n",
            "          \"such as an MMP9 module, with different interaction patterns between basal-enriched and luminal-enriched subgroups. These latter patterns appeared specific to the proteome-level data (Extended Data Fig. 8 c\\u2013f, Supplementary Table 17 and Supplementary\",\n",
            "          \"Data Fig. 8 c\\u2013f, Supplementary Table 17 and Supplementary Methods).\",\n",
            "          \"Phosphosite markers in PIK3CA and TP53 mutant breast cancer\",\n",
            "          \"TP53 and PIK3CA are the most recurrently mutated genes in breast cancer, with frequencies for PIK3CA at 43% in luminal tumors and for TP53 at 84% in basal-like tumors. Most of the PIK3CA missense mutations were gain of function mutations and therefore\",\n",
            "          \"mutations were gain of function mutations and therefore expected to lead to an activation of the PI3K signaling cascade, but the extent to which this occurs has been controversial and there is uncertainty which pathway components are effectors. Marker\",\n",
            "          \"is uncertainty which pathway components are effectors. Marker selection analysis was therefore performed for upregulated phosphosites in PIK3CA-mutated tumors. In total, 62 phosphosites were identified that were positively associated with PIK3CA mutation\",\n",
            "          \"identified that were positively associated with PIK3CA mutation (FDR<0.05), including the kinases RPS6KA5 and EIF2AK4 (Extended Data Fig. 9a, Supplementary Table 18). Calculating the average phosphorylation signal of these marker phosphosites provided a\",\n",
            "          \"phosphorylation signal of these marker phosphosites provided a read-out for PI3K pathway activity in PIK3CA-mutated tumors, with 15 of the 26 mutated tumors (58%) exhibiting an activated PIK3CA mutation signature. Of note, the identified PIK3CA mutant\",\n",
            "          \"mutation signature. Of note, the identified PIK3CA mutant phosphoproteome signature was activated in all tumors harboring helical domain PIK3CA mutations but only 2 of 10 tumors harboring kinase domain mutations. To test if the identified differences in\",\n",
            "          \"domain mutations. To test if the identified differences in the phosphoproteome of PI3K mutant versus wild-type tumors could be explained by mutation of PIK3CA, the tumor data were compared to phosphosite signatures derived from isogenic PIK3CA mutant cell\",\n",
            "          \"phosphosite signatures derived from isogenic PIK3CA mutant cell lines (Extended Data Fig. 9b, Supplementary Table 18). There was an enrichment of signatures derived from helical domain-mutated isogenic cell lines, but not from kinase domain-mutated cells,\",\n",
            "          \"isogenic cell lines, but not from kinase domain-mutated cells, supporting the observations in primary tumors.\",\n",
            "          \"The same strategy was used to identify phosphorylation signaling events connected to TP53 mutation. A total of 56 phosphosites upregulated in TP53 mutant tumors were identified that were independent of basal-like subtype association (Extended Data Fig.\",\n",
            "          \"of basal-like subtype association (Extended Data Fig. 9c, Supplementary Table 18). Using the average phosphorylation signal of these marker phosphosites as a proxy for TP53 mutation-driven cell cycle control, 22 of 41 mutated tumors (54%) showed\",\n",
            "          \"cell cycle control, 22 of 41 mutated tumors (54%) showed upregulated signals. This TP53 mutant phosphosignature was somewhat enhanced in tumors in which mutations occurred almost exclusively in the DNA binding region compared to those with\",\n",
            "          \"exclusively in the DNA binding region compared to those with non-sense/frameshift mutations. In addition to the well-described checkpoint kinase CHEK2, significantly upregulated phosphosites were identified for the kinases MASTL and EEF2K in TP53-mutated\",\n",
            "          \"were identified for the kinases MASTL and EEF2K in TP53-mutated tumors. Single sample GSEA analysis of isogenic p53-mutant phosphosignatures showed an enrichment of a phosphosignature derived from R273H mutated isogenic cells (Extended Data Fig. 9d),\",\n",
            "          \"from R273H mutated isogenic cells (Extended Data Fig. 9d), confirming the pronounced effect of missense mutations in the DNA-binding region on phosphorylation pathways.\",\n",
            "          \"Identification of gene amplification and breast cancer subtype-specific activated kinases in human breast cancer\",\n",
            "          \"CNA spans many driver gene candidates and RNA expression has been frequently used to narrow candidate nominations. Proteogenomic analysis should further promote this nomination process. In this candidate refinement a focus on protein kinases is warranted,\",\n",
            "          \"candidate refinement a focus on protein kinases is warranted, since many are drug targets. An in-depth proteogenomic pipeline was developed that flagged kinases, expression levels of which were at least 1.5 interquartile ranges higher than the median\",\n",
            "          \"were at least 1.5 interquartile ranges higher than the median (Supplemental Table 19). A proteogenomic circos-like plot (termed a \\u201cpircos\\u201d plot) was used to map these outlier kinase values onto the genome (Fig. 4a,b, Extended Data Fig. 10a). The ERBB2\",\n",
            "          \"onto the genome (Fig. 4a,b, Extended Data Fig. 10a). The ERBB2 locus showed the strongest effect of increased phosphoprotein levels associated with gene amplification-driven RNA and protein over expression (Fig. 4a). CDK12 is a positive transcriptional\",\n",
            "          \"over expression (Fig. 4a). CDK12 is a positive transcriptional regulator of homologous recombination repair genes with its partner Cyclin K, and is often encompassed by the ERBB2 amplicon. This gene was also found to be upregulated at the RNA, protein and\",\n",
            "          \"gene was also found to be upregulated at the RNA, protein and phosphosite level indicating that CDK12 is highly active in the majority of ERBB2 positive tumors (Fig. 4a). The analysis of the ERBB2 amplicon also uncovered co-outlier phosphorylation status\",\n",
            "          \"ERBB2 amplicon also uncovered co-outlier phosphorylation status for MED1, GRB7, MSL1, CASC3 and TOP2A, all previously described in association with ERBB2 amplification. To better understand the downstream effects of ERBB2 amplification, additional\",\n",
            "          \"the downstream effects of ERBB2 amplification, additional phosphosite outliers were identified in 41 known ERBB2 signaling genes for the 15 samples that had ERBB2 phosphosite outlier expression (Extended Data Fig. 10b).\",\n",
            "          \"These canonical findings stimulated a proteogenomic analysis to identify additional outlier kinases in the breast cancer genome. A proteogenomic dissection of chromosome 11q based on PAK1 amplification (Fig. 4b,c), a breast cancer driver kinase,\",\n",
            "          \"PAK1 amplification (Fig. 4b,c), a breast cancer driver kinase, illustrated that PAK1 is hyperphosphorylated in PAK1 amplified tumors, along with CLNS1A, RFS1 and GAB2. Additional examples of outlier kinases included PTK2 and RIPK2 in association with\",\n",
            "          \"of outlier kinases included PTK2 and RIPK2 in association with amplification of chromosome 8q (Fig. 4c; Extended Data Fig. 10a,c). PAK1 and TLK2 (17q23) appear to be luminal breast cancer specific events Fig. 4c; Extended Data Fig. 10c). To further\",\n",
            "          \"specific events Fig. 4c; Extended Data Fig. 10c). To further examine whether outlier kinases were breast cancer subtype-specific, independent of amplification status, the BH-corrected probability was calculated of finding that number of phosphosite\",\n",
            "          \"was calculated of finding that number of phosphosite outliers within a subtype, given the total number of outliers across all subtypes, the subtype sample size and the total sample size. (Fig. 4d). These analyses led to the expected identification of\",\n",
            "          \"(Fig. 4d). These analyses led to the expected identification of ERBB2 in the HER2-enriched subtype at the 5% FDR level, as well as the new finding of CDC42BPG (MRGKG), an effector kinase for RHO-family GTPases. In basal-like breast cancer, two kinases,\",\n",
            "          \"RHO-family GTPases. In basal-like breast cancer, two kinases, PRKDC and SPEG, were significant at the 5% FDR level. PRKDC is a non-homologous end-joining (NHEJ) factor that can be phosphorylated by ATM kinase, and is therefore a logical finding in this\",\n",
            "          \"by ATM kinase, and is therefore a logical finding in this disease subset. However SPEG, a kinase associated with severe dilated cardiomyopathy when suppressed, has not been previously reported in association with breast cancer. A larger number of\",\n",
            "          \"reported in association with breast cancer. A larger number of subtype-specific kinases were detected at the 10% FDR level, several of which have recently described relevance in breast cancer, including PRKD3 in basal-like breast cancer, the LKB-regulated\",\n",
            "          \"including PRKD3 in basal-like breast cancer, the LKB-regulated SIK3 in luminal A breast cancer and CDK13 in luminal B breast cancer, which, similar to CDK12, can interact with Cyclin K.\",\n",
            "          \"Discussion\",\n",
            "          \"The analytic breadth and depth of proteomic and phosphoproteomic analyses displayed in this study demonstrates the strengths of mass spectrometry-based proteomics, but also some of the limitations inherent in proteolytic peptide sequencing (see\",\n",
            "          \"the limitations inherent in proteolytic peptide sequencing (see Supplementary Discussion). An example of how high-dimensional proteomic analysis provides insight into unresolved genomic issues concerns the study of loss of the long arm of chromosome 5\",\n",
            "          \"concerns the study of loss of the long arm of chromosome 5 (5q). Analysis of RNA and protein correlations narrowed the list of potential trans-deregulated proteins. Orthogonal candidate screening using functional genomics methods identified loss of CETN3\",\n",
            "          \"using functional genomics methods identified loss of CETN3 and SKP1 as potential transregulators, with upregulation of EGFR as a downstream consequence in basal-like breast cancers. While further experimental evidence must be sought for these proposed\",\n",
            "          \"further experimental evidence must be sought for these proposed regulatory relationships, SKP1/Cullin complex has already been linked to EGFR activation in Glioma. Unfortunately EGFR targeting has not to date proven to be effective therapy in basal-like\",\n",
            "          \"has not to date proven to be effective therapy in basal-like breast cancer. This might be due to the fact the SKP1 loss deregulates multiple targets requiring a much broader inhibitory strategy.\",\n",
            "          \"It is recognized that PIK3CA mutations do not strongly activate canonical downstream effectors. Mass spectrometry-based phosphoproteomics provides an opportunity for unbiased examination of downstream signaling events consequent upon PIK3CA mutational\",\n",
            "          \"downstream signaling events consequent upon PIK3CA mutational activation. These studies revealed that common PIK3CA mutations affect a large number of targets with diverse functionalities including the kinases RPS6KA5 and EIF2AK4. Thus, the data and\",\n",
            "          \"including the kinases RPS6KA5 and EIF2AK4. Thus, the data and analyses reported here extend our knowledge of the effectors that promote tumorigenesis in response to constitutive activation of PI3 kinase. Similarly, TP53 mutation-associated phosphopeptides\",\n",
            "          \"PI3 kinase. Similarly, TP53 mutation-associated phosphopeptides point towards novel functionalities, including regulation of the kinases MASTL and EEF2K.\",\n",
            "          \"A central goal in breast cancer research has been the identification of druggable kinases beyond HER2. Candidate genes that exhibited similar gene amplification-driven proteogenomic patterns to HER2 included CDK12, TLK2, PAK1 and RIPK2. The proteogenomic\",\n",
            "          \"to HER2 included CDK12, TLK2, PAK1 and RIPK2. The proteogenomic link with gene amplification was particularly strong for CDK12, in keeping with its location in the ERBB2 amplicon, while the strengths of correlation between DNA amplification, RNA, protein\",\n",
            "          \"of correlation between DNA amplification, RNA, protein and phosphoprotein for the other examples were more variable. The presence of activated CDK12 in the ERBB2 amplicon might explain why tumors arising in BRCA1 carriers are usually ERBB2 negative. As a\",\n",
            "          \"arising in BRCA1 carriers are usually ERBB2 negative. As a positive transcriptional regulator of BRCA1 and multiple FANC family members, CDK12 promotes DNA repair by homologous recombination. CDK12 amplification would, therefore, oppose the functional\",\n",
            "          \"CDK12 amplification would, therefore, oppose the functional effects of BRCA1 haploinsufficiency during tumor evolution. Overall, multiple outlier kinases generate testable therapeutic hypotheses for which enabling inhibitors are in development. For\",\n",
            "          \"for which enabling inhibitors are in development. For example PAK1 has recently been confirmed to be a therapeutic target and poor prognosis factor in luminal breast cancer.\",\n",
            "          \"Although incomplete outcome data and the remarkable heterogeneity of breast cancer are additional relevant constraints, the number of TCGA specimens analyzed here is insufficient to support conclusive clinical correlations. Only 8 deaths occurred among\",\n",
            "          \"conclusive clinical correlations. Only 8 deaths occurred among the 77 patients, which are too few samples to provide sufficient statistical power for association analysis. Adequately powered MS-based clinical investigation will require targeted\",\n",
            "          \"powered MS-based clinical investigation will require targeted approaches, especially given the highly limited amount of patient material available from clinical trials and the mostly formalin-fixed nature of the specimens. The current analysis is\",\n",
            "          \"formalin-fixed nature of the specimens. The current analysis is therefore centered on biological findings and correlations, with orthogonal validation and false discovery concerns addressed through an examination of cell-line databases of the effects of\",\n",
            "          \"through an examination of cell-line databases of the effects of individual gene perturbations. Typical of a multi-tiered analysis of this complexity, there are many hypotheses to test, and many findings that require further investigation.\",\n",
            "          \"In conclusion, this study provides a high-quality proteomic resource for human breast cancer investigation and illustrates technologies and analytical approaches that provide an important new opportunity to connect the genome to the proteome. Larger-scale\",\n",
            "          \"opportunity to connect the genome to the proteome. Larger-scale exploration of discovery proteomics in the clinical setting will require improvements in clinical investigation, including acquisition of adequate amounts of optimally collected tumor tissue\",\n",
            "          \"of adequate amounts of optimally collected tumor tissue both before and during therapy as well as advances in MS proteomics to reduce sample input and increase sensitivity for low abundance proteins and modified peptides.\",\n",
            "          \"Extended Data\\n\\nExperimental and data analysis workflows and longitudinal data generation quality control\",\n",
            "          \"a, iTRAQ 4-plex global proteome and phosphoproteome analysis workflow. 105 TCGA breast tumors were analyzed in 35 iTRAQ 4-plex experiments (plus 1 replicate and 1 normal sample experiment), with three tumors of different subtypes compared to a fourth\",\n",
            "          \"with three tumors of different subtypes compared to a fourth common internal reference sample in each experiment. The reference sample comprised 10 individual tumors of each of the 4 major breast cancer intrinsic subtypes and served as an internal\",\n",
            "          \"breast cancer intrinsic subtypes and served as an internal standard for all proteins and phosphoproteins quantified in this study. Each iTRAQ MS/MS spectrum measures a peptide from 4 samples (3 individual patients and the reference sample mix of 40\",\n",
            "          \"(3 individual patients and the reference sample mix of 40 patients). More than 400,000 distinct peptides were identified and quantified in ~14 million MS/MS spectra. Personalized tumor-specific protein databases were generated in the QUILTS software\",\n",
            "          \"protein databases were generated in the QUILTS software package using whole exome sequencing-derived variant calls and RNAseq-derived transcript information. All mass spectrometry data was analyzed using the Spectrum Mill software package. b, Overview of\",\n",
            "          \"using the Spectrum Mill software package. b, Overview of proteome and phosphoproteome datasets. The table provides a summary of the datasets used in specific analyses, including the filters applied to derive the proteins and phosphosites/phosphoproteins\",\n",
            "          \"applied to derive the proteins and phosphosites/phosphoproteins that constitute each dataset; the protein, phosphosite or phosphoprotein count; and the methods that employ the respective datasets. c, Distribution of sequence coverage of the identified\",\n",
            "          \"c, Distribution of sequence coverage of the identified proteins with tryptic peptides detected by MS/MS, whiskers show the 5\\u201395 percentiles. d and e, Robust and accurate proteome/phosphoproteome platform. Longitudinal performance was tested by repeated\",\n",
            "          \"platform. Longitudinal performance was tested by repeated proteome and phosphoproteome analysis of patient-derived xenograft tumors. Scatterplots, histograms and Pearson correlations comparing individual replicate measurements are shown.\",\n",
            "          \"Tumor sample quality control (I)\",\n",
            "          \"a, Remark diagram showing sample processing and partitioning. Initial quality review encompassed histopathological examination of H&E stained tissue slices. *For 3 samples no tumor cells were seen on histopathology (BH-A0E9, BH-A0C1, A2-A0SW). These\",\n",
            "          \"were seen on histopathology (BH-A0E9, BH-A0C1, A2-A0SW). These samples were nevertheless included in the proteome analysis since other quality control standards were met (see below) and samples with 0% tumor cellularity on top or bottom sections were\",\n",
            "          \"with 0% tumor cellularity on top or bottom sections were included in TCGA analyses. b, correlation of TCGA (top or bottom sections) and CPTAC histological assessment of neoplastic cellularity for samples (n = 105). The average and range of neoplastic\",\n",
            "          \"for samples (n = 105). The average and range of neoplastic cellularities were identical for CPTAC and TCGA histological assessments. Averages (standard deviations) for neoplastic cellularity were 76% (+/\\u2212 17) for CPTAC, 76% (+/\\u2212 15) for TCGA_Top, and 75%\",\n",
            "          \"were 76% (+/\\u2212 17) for CPTAC, 76% (+/\\u2212 15) for TCGA_Top, and 75% (+/\\u2212 18) for TCGA_Bottom histopathology slides (Supplementary Table 2). Note that in three CPTAC cases where no tumor cells were identified by histopathological assessment, numbers of\",\n",
            "          \"were identified by histopathological assessment, numbers of protein-level somatic variants were similar to all other tumors. The identified mutated proteins were TP53_R273C, NOP58_Q23E, TAGLN2_G154R, TUBA1B_D116H, and MRPL48_I173K (Supplementary Table 5),\",\n",
            "          \"TUBA1B_D116H, and MRPL48_I173K (Supplementary Table 5), indicating presence of tumor cells in these samples. c, Proteome iTRAQ tumor/internal reference ratio heatmap for all CPTAC samples (8,028 proteins without missing values) including passed and failed\",\n",
            "          \"proteins without missing values) including passed and failed proteomic quality control (QC) samples. d, Global tumor/reference proteome ratio distributions for samples that passed and failed proteomic quality control analysis. e, Degradation-related gene\",\n",
            "          \"proteomic quality control analysis. e, Degradation-related gene sets were enriched in tumors that failed proteomic quality control analysis. f, Variant allele frequency (VAF) analysis of re-sequenced CPTAC tumors and comparison to original TCGA data.\",\n",
            "          \"re-sequenced CPTAC tumors and comparison to original TCGA data. Overall VAFs for failed QC samples were lower compared to passed samples suggesting lower purity.\",\n",
            "          \"Tumor sample quality control (II)\",\n",
            "          \"a, There was high concordance (94.6%) between DNA variants reported by TCGA and CPTAC re-sequenced tumors. Most point mutations reported by TCGA could be identified across the 8 re-sequenced samples used in the study. b, A high overall correlation\",\n",
            "          \"samples used in the study. b, A high overall correlation (mean=0.77) was observed for the CPTAC Variant Allele Fraction (VAF) (X-axis) and TCGA VAF (Y-axis) across the 8 samples used in the study. c, Agglomerative hierarchical clustering (Supplementary\",\n",
            "          \"study. c, Agglomerative hierarchical clustering (Supplementary Methods Section 3.8) used to co-cluster protein and RNA tumor expression data after filtering to retain 4,291 proteins and genes with moderate to high protein-RNA correlation (Pearson\",\n",
            "          \"genes with moderate to high protein-RNA correlation (Pearson correlation > 0.4) with results displayed as a circular dendrogram (fanplot). The proteome (.P) and RNA (.R) components of each sample are labeled using the same color. The outer ring shows\",\n",
            "          \"sample are labeled using the same color. The outer ring shows proteome samples in light grey and RNA samples in dark grey. High concordance between RNA and protein expression is evident from the color adjacency in the inner ring and alternating color in\",\n",
            "          \"the color adjacency in the inner ring and alternating color in the outer ring showing that RNA and protein components co-cluster for a large proportion of samples (62/80). d, Co-clustering of MS and RPPA tumor data. 126 RPPA readouts were mapped to gene\",\n",
            "          \"MS and RPPA tumor data. 126 RPPA readouts were mapped to gene names. These genes were intersected with the genes observed in the MS proteome, filtered to 48 proteins with moderate or higher RPPA-MS protein correlation, and analyzed for co-clustering as in\",\n",
            "          \"protein correlation, and analyzed for co-clustering as in c. 47 of 80 RPPA-MS protein pairs co-cluster. While this is a smaller proportion than for RNA-protein analysis, the number of genes used in the clustering is significantly smaller for RPPA (48 vs.\",\n",
            "          \"in the clustering is significantly smaller for RPPA (48 vs. 4,291 for RNA). e, ESTIMATE tumor purity comparison between mRNA, RNAseq, and proteome data. ANOVA is used to assess the difference in distribution (\\u2212log10(p-value)) of ESTIMATE, stromal, immune,\",\n",
            "          \"in distribution (\\u2212log10(p-value)) of ESTIMATE, stromal, immune, and tumor purity scores across mRNA (microarray), RNA-seq and proteome data. The only significant p-value (=0.02) is for the Cluster 3 stromal score, and higher stromal scores for the\",\n",
            "          \"the Cluster 3 stromal score, and higher stromal scores for the proteome drive that difference. f, Ischemia score analysis. Comparison of ischemia scores of 77 CPTAC tumors, 3 normal samples, and patient-derived xenografts. CPTAC tumors had generally lower\",\n",
            "          \"patient-derived xenografts. CPTAC tumors had generally lower ischemia scores than PDX samples subjected to 30 minutes of cold ischemia. Median ischemia scores are less than 30 minutes for each subtype and no significant differences were observed across\",\n",
            "          \"subtype and no significant differences were observed across subtypes. Effects due to cold ischemia therefore appear to be negligible in this CPTAC sample collection.\",\n",
            "          \"Protein-to-Protein, -CNA, and -mRNA correlation analyses\",\n",
            "          \"a, Identification of UBE3A as an E3 ubiquitin ligase that negatively correlates to p53 on the protein level. Pearson correlation and Benjamini-Hochberg corrected p-value are shown. b, Analysis of counter-regulated genes with negative correlation of\",\n",
            "          \"of counter-regulated genes with negative correlation of CNA-to-RNA as well as CNA-to-protein levels. Negative Pearson correlations are shown with Benjamini-Hochberg corrected p-values for CNA-to-protein correlations. Depicted genes have significant\",\n",
            "          \"CNA-to-protein correlations. Depicted genes have significant negative correlations at FDR<0.05 in the CNA-to-RNA and CNA-to-protein analyses. c, Global mRNA-to-protein correlation and gene set enrichment analysis.\",\n",
            "          \"Global CNA effects and comparison of CNA TRANS effects to knockdown signatures in the LINCS database\",\n",
            "          \"a, CNA landscape in the CPTAC tumor collection. The segment-based CNAs of 77 samples were downloaded from TCGA Firehose, including 18 Basal, 12 Her2, 23 Luminal A and 24 Luminal B subtypes. Copy number amplifications were marked in red and deletions in\",\n",
            "          \"Copy number amplifications were marked in red and deletions in blue. The bottom color key represents the log2 transformed copy number value, with CNA=2 centered at 0. Specific CNA events are seen for chromosome 5q and 10p regions in basal-like tumors. b,\",\n",
            "          \"seen for chromosome 5q and 10p regions in basal-like tumors. b, Correlations of copy number alterations (x-axis) to phosphoprotein levels (y-axis) highlight new CNA cis and trans effects. Significant (FDR<0.05) positive (red) and negative (green)\",\n",
            "          \"Significant (FDR<0.05) positive (red) and negative (green) correlations between CNA and phosphoproteins are indicated. Histograms show the fraction [%] of significant CNA trans effects for each CNA gene. c, LINCS CMap analysis facilitates identification\",\n",
            "          \"CNA gene. c, LINCS CMap analysis facilitates identification of novel functional candidates for CNA trans effects. Knockdown profiles were compared with CNA/protein trans effects for 502 genes. Genes with a connectivity score >|90| were considered\",\n",
            "          \"genes. Genes with a connectivity score >|90| were considered connected and significant cis effects were annotated at an FDR<0.05. d, Basal-like tumor-specific CNAs are candidate regulatory events for EGFR and SRC expression levels. Oncogenic kinases with\",\n",
            "          \"for EGFR and SRC expression levels. Oncogenic kinases with significant CNA/protein trans effects (left panel), that were regulated in LINCS shRNA experiments (right panel; 4 cell lines,) and directly measured as LINCS landmark genes, are shown alongside\",\n",
            "          \"directly measured as LINCS landmark genes, are shown alongside candidate regulatory genes CETN3 and SKP1. Clinical ER, PR, and HER2 annotation and PAM50 classification are shown in the header rows of each column.\",\n",
            "          \"Proteome cluster heatmap and stability analysis\",\n",
            "          \"a, K-means consensus clustering of proteome and phosphoproteome data identifies three subgroups: basal-enriched, luminal-enriched, and stromal-enriched. The heatmap represents all 1,521 proteins used for clustering (Dataset G8). b, Identification of\",\n",
            "          \"proteins used for clustering (Dataset G8). b, Identification of optimal proteome clusters for QC-passed CPTAC breast cancer tumors. Proteome clusters were derived using consensus clustering based on 1000 resampled datasets, exploring the range of 2 to 6\",\n",
            "          \"based on 1000 resampled datasets, exploring the range of 2 to 6 k-means clusters. Visualization of consensus matrices from k-means consensus clustering for k=3, 4, 5 and 6 target clusters. Consensus clustering was performed on 1,521 proteins with no\",\n",
            "          \"Consensus clustering was performed on 1,521 proteins with no missing values and SD>1.5. c, Silhouette plots were generated to evaluate the coherence of the clustering. Silhouette plots for k=3 and k=4 clusters showing a cleaner separation of clusters for\",\n",
            "          \"and k=4 clusters showing a cleaner separation of clusters for k=3. d, Based on both visual inspection of the consensus matrix and the delta plot assessing change in consensus cumulative distribution function (CDF) area, three robustly segregated groups\",\n",
            "          \"function (CDF) area, three robustly segregated groups were observed. Consensus cumulative distribution function (CDF) and delta area (change in CDF area) plots for 2\\u20136 clusters.\",\n",
            "          \"Proteome cluster markers and enriched pathways\",\n",
            "          \"a, Markers (based on SAM analysis; FDR<0.01) discriminate between proteome clusters 1, 2 and 3 (compare to heatmap of proteins used to derive clusters depicted in Extended Data Fig. 6a). b, Applying a Fisher exact test-based enrichment analysis to the\",\n",
            "          \"Applying a Fisher exact test-based enrichment analysis to the proteome, phosphoproteome and mRNA data, gene sets from MSigDB were identified that were unique for each proteome cluster. Heat map showing specific pathways comprising dominant biological\",\n",
            "          \"map showing specific pathways comprising dominant biological themes that are significantly differential by enrichment analysis between basal-enriched and luminal-enriched tumors (Fisher Exact Test Benjamini-Hochberg corrected p-values are shown;\",\n",
            "          \"Exact Test Benjamini-Hochberg corrected p-values are shown; enrichment test performed on marker sets identified using SAM analysis; see Methods; compare to Figure 3c). c, Heatmap showing a selection of gene sets significant in basal-enriched or\",\n",
            "          \"a selection of gene sets significant in basal-enriched or luminal-enriched tumors exclusively by mRNA, protein or phosphoprotein expression. Cytokine signatures, for example, were strongly captured at the mRNA level, but were seen to only a limited degree\",\n",
            "          \"at the mRNA level, but were seen to only a limited degree at the global protein level, likely because of their typically low protein abundance. By contrast, the vast majority of significant gene sets annotated as \\\"signaling\\\" were enriched only at the\",\n",
            "          \"gene sets annotated as \\\"signaling\\\" were enriched only at the phosphoprotein level. d, Global heat map representing all gene sets significantly enriched in at least one of the proteomic breast cancer subtypes. The stromal-enriched group was characterized\",\n",
            "          \"cancer subtypes. The stromal-enriched group was characterized by breast cancer normal-like, adipocyte differentiation, smooth muscle, toll-like receptor signaling and endothelin gene sets, supporting the clustering-based annotation of high stromal and/or\",\n",
            "          \"the clustering-based annotation of high stromal and/or adipose content in these tumors (see Supplemental Table 13).\",\n",
            "          \"Phosphoproteome pathway clustering, kinase-phosphosite multivariate regression, and protein co-expression networks\",\n",
            "          \"a, Phosphoproteome pathway clustering. Using phosphorylation state as a proxy for activity, deep phosphoproteome profiling allows development of a breast cancer molecular taxonomy based on signaling pathways. K-means consensus clustering was performed on\",\n",
            "          \"pathways. K-means consensus clustering was performed on pathways derived from single sample GSEA analysis of phosphopeptide data (908 pathways shown). Of four robustly segregated groups, subgroups 2 and 3 substantially recapitulated the stromal- and\",\n",
            "          \"subgroups 2 and 3 substantially recapitulated the stromal- and luminal-enriched proteomic subgroups, respectively. Subgroup 4 included a significant majority of tumors from the basal-enriched proteomic subgroup, but was admixed particularly with\",\n",
            "          \"proteomic subgroup, but was admixed particularly with luminal-enriched samples. This subgroup was defined by high levels of cell cycle and checkpoint activity. All basal and a majority of non-basal samples in this subgroup had TP53 mutations. Subgroup 1\",\n",
            "          \"samples in this subgroup had TP53 mutations. Subgroup 1 was a novel subgroup defined exclusively in the phosphopeptide / pathway activity domain, with no enrichment for either proteomic or PAM50 subtypes. It was defined by G-protein, G-protein coupled\",\n",
            "          \"PAM50 subtypes. It was defined by G-protein, G-protein coupled receptor, and inositol phosphate metabolism signatures, as well as ionotropic glutamate signaling. b, Analysis of the regulatory relationship between outlier kinases (see Supplementary Table\",\n",
            "          \"relationship between outlier kinases (see Supplementary Table 19) and phosphopeptides by regulatory multivariate regression analysis (see Methods) identified CDK1 as the most highly connected of the outlier Cyclin-Dependent Kinases, with highest\",\n",
            "          \"connected of the outlier Cyclin-Dependent Kinases, with highest centrality (based on node-degree; see Methods) among the outlier CDKs and seventh highest centrality among all the outlier kinases considered in the remMap analysis. Each line represents a\",\n",
            "          \"considered in the remMap analysis. Each line represents a phosphosite-kinase relationship. c\\u2013f, Analysis of differences in the co-expression patterns among genes/proteins across different subgroups. A Joint Random Forest (JRF) method was applied to\",\n",
            "          \"subgroups. A Joint Random Forest (JRF) method was applied to simultaneously build gene co-expression and protein co-expression networks (Supplementary Table 17, and Methods). Modules in these networks revealed different interaction patterns between\",\n",
            "          \"these networks revealed different interaction patterns between basal-enriched and luminal-enriched subgroups. c, Network module P1 of the protein co-expression network, defined chiefly in the proteome space. This module contained 12 genes connected by 39\",\n",
            "          \"proteome space. This module contained 12 genes connected by 39 edges, among which 34 were protein-specific and 5 were shared by both the protein and mRNA co-expression networks. Many edges were supported by published information and were contained in the\",\n",
            "          \"supported by published information and were contained in the STRING database. Edges in red are specific to the protein co-expression network; edges in green are shared by both protein and gene co-expression networks; edges indicated by double lines are\",\n",
            "          \"co-expression networks; edges indicated by double lines are contained in the STRING database with confidence score greater than 0.15. MMP9, one of the central proteins in this module, contributes to metastatic progression and is a potential target for\",\n",
            "          \"to metastatic progression and is a potential target for anti-metastatic therapies for basal-like / triple negative breast cancer. d, Heatmaps of the absolute correlation across each pair of genes in module P1 (shown in Panel c), based on either protein or\",\n",
            "          \"in module P1 (shown in Panel c), based on either protein or gene expression data for samples in the basal-enriched and luminal-enriched subgroups, respectively. The MMP9 protein was strongly co-expressed with the other members of the module only in the\",\n",
            "          \"co-expressed with the other members of the module only in the basal-enriched subgroup. Notably, this observation is dependent on protein data; the correlation at the mRNA level for this module was consistently low in both the basal-enriched and luminal\",\n",
            "          \"was consistently low in both the basal-enriched and luminal enriched subgroups indicating that these events coherently occur at the proteomic level. e, Co-expression network based on proteomics data. The network contains 693 proteomic network-specific\",\n",
            "          \"data. The network contains 693 proteomic network-specific edges (grey) and 792 edges shared with the RNAseq network (green). For each module, the most enriched category and corresponding Benjamini-Hochberg adjusted p-value is reported. Pie charts adjacent\",\n",
            "          \"adjusted p-value is reported. Pie charts adjacent to each module show the proportion of proteomics-specific edges (grey area) and edges shared between proteomics and RNAseq data (green area). f, RNAseq network.\",\n",
            "          \"Phosphoproteome signatures of PIK3CA (a,b) and TP53 (c,d) mutated tumors highlight activated key regulators and indicate frequency of activation\",\n",
            "          \"a and c, Phosphosites upregulated in mutated tumors (SAM FDR<0.05 across all tumors and independently also across luminal tumors; average phosphosite signal for all markers shown as bar graph). To avoid confounding by intrinsic subtype-specific\",\n",
            "          \"bar graph). To avoid confounding by intrinsic subtype-specific distinctions, only markers that were significantly identified both in analyses covering all tumors and analyses restricted to luminal tumors were selected (FDR <0.05). Color bars in the\",\n",
            "          \"to luminal tumors were selected (FDR <0.05). Color bars in the margins indicate FDRs for grouped analysis of different mutation classes and indicate kinase substrates of known kinases in the respective pathways. Significantly regulated kinase phosphosites\",\n",
            "          \"pathways. Significantly regulated kinase phosphosites are annotated. The average phosphorylation signal of the marker phosphosites provides a read-out for PI3K and TP53 pathway activity in mutated tumors (histogram below heatmap). A 95% prediction\",\n",
            "          \"in mutated tumors (histogram below heatmap). A 95% prediction confidence interval (indicated by dashed lines) across the average signal in non-mutated tumors was chosen in order to discriminate active from non-active tumors. The most strongly activated\",\n",
            "          \"active from non-active tumors. The most strongly activated PIK3CA kinase domain mutant tumor differed from the other 9 kinase domain mutant tumors, as it contained an amino acid side chain charge neutral H1047L instead of the more common positively\",\n",
            "          \"charge neutral H1047L instead of the more common positively charged H1047R mutation. Among the 62 phosphosites identified that were significantly upregulated in PIK3CA mutated tumors, 13 phosphosites were found on phosphoproteins that are known substrates\",\n",
            "          \"were found on phosphoproteins that are known substrates of well-annotated kinases in the PIK3CA pathway (panel a, right column). In the mutant TP53 analysis a total 20 phosphosites were found on phosphoproteins that are known substrates of well annotated\",\n",
            "          \"on phosphoproteins that are known substrates of well annotated kinases in the p53 pathway (panel c, right column). b and d, Upregulated phosphosite sets were derived from isogenic PIK3CA and TP53 mutant versus wild-type cell line pairs and tested for\",\n",
            "          \"and TP53 mutant versus wild-type cell line pairs and tested for enrichment within mutant versus wild-type CPTAC tumors using single sample GSEA. Significantly enriched phosphosite sets are shown (p<0.05).\",\n",
            "          \"PIRCOS plots, kinase outliers and outliers in the ERBB2 pathway\",\n",
            "          \"a, Pircos (Proteogenomics CIRCOS) plots for 8q and 17q showing median CNA, RNA, protein, and phosphosite expression for 20 tumors with amplification in 8q based on RIPK2 CNA>1; 23 tumors with amplification in 8q based on PTK2 CNA>1; 15 tumors with\",\n",
            "          \"with amplification in 8q based on PTK2 CNA>1; 15 tumors with amplification in 17q based on CDK12 CNA >1; and 10 tumors with amplification in 17q based on TLK2 CNA>1. Red indicates expression >1, blue < \\u22121, and grey between \\u22121 and 1. Genes with both copy\",\n",
            "          \">1, blue < \\u22121, and grey between \\u22121 and 1. Genes with both copy number amplification (CNA>1) and increased phosphosite expression (p-site>1) are labeled. b, Phosphosite outliers in known ERBB2 signaling genes. To better understand the downstream effects of\",\n",
            "          \"signaling genes. To better understand the downstream effects of ERBB2 amplification, phosphosite outliers in known ERBB2 signaling genes (MSigDB pathway set, KEGG_ERBB_SIGNALING PATHWAY) were identified for the 15 samples that had ERBB2 phosphosite\",\n",
            "          \"were identified for the 15 samples that had ERBB2 phosphosite outlier status. Forty-one genes were identified as having a phosphosite outlier in at least one of the ERBB2 amplified samples. PAK4 and ARAF phosphosite outlier status were found in seven of\",\n",
            "          \"PAK4 and ARAF phosphosite outlier status were found in seven of the 15 ERBB2 kinase outlier samples; GSK3B outliers were found in 6 samples; and EIF4EBP1, MAP2K2, ABL1 and AKT1 outlier status was found in 5 of the 15 samples. c, Proteogenomic outlier\",\n",
            "          \"was found in 5 of the 15 samples. c, Proteogenomic outlier expression analysis for TLK2 and RIPK2. Samples with outlier phosphosite (red), protein (yellow), RNA (green) and copy number (purple) expression are shown. Phosphosite squares indicate per-sample\",\n",
            "          \"expression are shown. Phosphosite squares indicate per-sample outlier phosphosites.\",\n",
            "          \"Supplementary Material\\n\\nSupplementary Information is available in the online version of the paper.\",\n",
            "          \"Author Contributions P.M., D.R.M., M.A.G., K.R.C., and S.A.C. designed the proteomic analysis experiments, data analysis workflow, and proteomic-genomic data comparisons. P.M., M.A.G., J.W.Q., and S.A.C. directed and performed proteomic analysis of breast\",\n",
            "          \"and S.A.C. directed and performed proteomic analysis of breast tumor and quality control samples. P.M., D.R.M., K.V.R., K.R.C., P.W., X.W., S.C., E.K., F.P., Z.T., J.L., M.L.G., M.W., V.Y., K.H., C.L., M.D.M., P.Y., J.W., B.Z., and D.F. performed\",\n",
            "          \"V.Y., K.H., C.L., M.D.M., P.Y., J.W., B.Z., and D.F. performed proteomic-genomic data analyses. D.R.M., P.W., and S.J.S. provided statistical support. D.R.M., K.V.R., K.R.C., K.K. and D.F. performed analyses of mass spectrometry data and adapted\",\n",
            "          \"D.F. performed analyses of mass spectrometry data and adapted algorithms and software for data analysis. S.R.D., R.R.T and M.J.E. developed and prepared breast xenografts used as quality control samples. P.M. and F.M prepared and analyzed cell lines for\",\n",
            "          \"samples. P.M. and F.M prepared and analyzed cell lines for correlative functional annotation of frequently mutated genes. P.M., D.R.M., M.A.G., and S.A.C designed strategy for quality control analyses. M.A.G., S.R.D., C.R.K., M.M., and H.R. coordinated\",\n",
            "          \"analyses. M.A.G., S.R.D., C.R.K., M.M., and H.R. coordinated acquisition, distribution and quality control evaluation of TCGA tumor samples. P.M., M.A.G., C.P., L.D., A.G.P., and M.J.E. interpreted data in the context of breast cancer biology. P.M.,\",\n",
            "          \"interpreted data in the context of breast cancer biology. P.M., D.R.M, M.A.G., K.R.C., P.W., A.G.P, M.J.E. and S.A.C. wrote the manuscript.\",\n",
            "          \"All primary mass spectrometry data are deposited at the CPTAC Data Portal as raw and mzML files and complete protein assembly data sets for public access (https://cptac-data-portal.georgetown.edu/cptac/s/S029). In addition, a set of ancillary files such\",\n",
            "          \"In addition, a set of ancillary files such as dataset G1/P1, G3/P3, G4/P4, G5/P5, G7/P7, CNA correlation tables for CNA-mRNA, CNA-proteome and CNA-phosphoproteome; CNA data, RNA-seq expression data have also been deposited at the DCC. Two browsers have\",\n",
            "          \"data have also been deposited at the DCC. Two browsers have been created to assist the interested reader in exploring the results. One provides track hubs for viewing the identified peptides in the UCSD genome browser\",\n",
            "          \"for viewing the identified peptides in the UCSD genome browser (http://fenyolab.org/cptac_breast_ucsc). The second is an on-line tool for proteogenomic data exploration and can be accessed at http://prot-shiny-vm.broadinstitute.org:3838/BC2016/. See\",\n",
            "          \"at http://prot-shiny-vm.broadinstitute.org:3838/BC2016/. See Supplemental Methods for descriptions.\",\n",
            "          \"The authors declare no competing financial interests.\\n\\nProteogenomic analysis of human breast cancer. Direct effects of genomic alterations on protein level\",\n",
            "          \"Overlap of a, protein coding single amino acid variants (SAAVs) and b, RNA splice junctions not present in RefSeq v60 detected by DNA exome sequencing, RNA-seq, and LC-MS/MS. Proportions of novel variants are noted. c, Heatmap of mutations/CNA and their\",\n",
            "          \"novel variants are noted. c, Heatmap of mutations/CNA and their effects on RNA and protein expression of breast cancer-relevant genes across tumor and normal samples. ER, PR, HER2 and PAM50 status are annotated. Median iTRAQ protein abundance ratio and\",\n",
            "          \"status are annotated. Median iTRAQ protein abundance ratio and the most frequently detected and differential phosphosite ratio are shown for each gene. Pearson correlations between MS protein vs RNA-seq and MS protein vs RPPA are indicated.\",\n",
            "          \"Effects of copy number alterations (CNA) on mRNA, protein, and phosphoprotein abundance\",\n",
            "          \"a, Correlations of CNA (x-axes) to RNA and protein expression levels (y-axes) highlight new CNA cis and trans effects. Significant (FDR<0.05) positive (red) and negative (green) correlations between CNA and mRNAs or proteins are indicated. CNA cis effects\",\n",
            "          \"CNA and mRNAs or proteins are indicated. CNA cis effects appear as a red diagonal line, CNA trans effects as vertical stripes. Histograms show the fraction [%] of significant CNA trans effects for each CNA gene. b, Overlap of cis effects observed at RNA,\",\n",
            "          \"for each CNA gene. b, Overlap of cis effects observed at RNA, protein, and phosphoprotein levels (FDR<0.05). c, Trans-effect regulatory candidates identified among those with significant protein cis-effects using LINCS CMap. Bars indicate total numbers of\",\n",
            "          \"cis-effects using LINCS CMap. Bars indicate total numbers of significant CNA/protein trans effects (gray; FDR<0.05) and overlap with regulated genes in LINCS knock-down profiles (red; 4 cell lines; moderated T-test FDR<0.1).\",\n",
            "          \"Proteomic and phosphoproteomic subtypes of breast cancer and subtype-specific pathway enrichment\",\n",
            "          \"a, Unsupervised clustering of RNA-seq and proteomics data restricted to PAM50 genes and subset of 35 detected proteins reveal high similarity to PAM50 (TCGA) sample annotation. b, K-means consensus clustering of proteome and phosphoproteome data\",\n",
            "          \"consensus clustering of proteome and phosphoproteome data identifies basal-enriched, luminal-enriched, and stromal-enriched subgroups. c, Gene set enrichment analysis highlights sets of pathways significantly differential between basal-enriched and\",\n",
            "          \"pathways significantly differential between basal-enriched and luminal-enriched tumors (detailed in Extended Data Fig. 7b). d, K-means consensus clustering performed on pathways derived from single sample GSEA analysis of phosphopeptide data identifies\",\n",
            "          \"single sample GSEA analysis of phosphopeptide data identifies four distinct clusters.\",\n",
            "          \"Example analyses of aberrantly regulated kinases in human breast cancer\",\n",
            "          \"a and b, PIRCOS (Proteogenomics CIRCOS) plots showing CNA, RNA, protein and phosphosite expression for 17 tumors with amplification in 17q (ERBB2 CNA>1) and 8 tumors with amplification in 11q (PAK1 CNA>1). Labeled genes have CNA>1 and phosphosite>1. c,\",\n",
            "          \"(PAK1 CNA>1). Labeled genes have CNA>1 and phosphosite>1. c, Proteogenomic outlier expression analysis for ERBB2, CDK12, and PAK1. Samples with outlier phosphosite (red), protein (yellow), RNA (green) and copy number (purple) expression are shown.\",\n",
            "          \"RNA (green) and copy number (purple) expression are shown. Phosphosite squares indicate per-sample outlier phosphosites. d, Outlier kinase events by PAM50 subtype (>35% of subtype samples contain a phosphosite outlier; <10% FDR using Benjamini-Hochberg\",\n",
            "          \"a phosphosite outlier; <10% FDR using Benjamini-Hochberg adjusted p-values).\"\n",
            "        ]\n",
            "      }\n",
            "    ]\n",
            "  },\n",
            "  \"PDC000174\": {\n",
            "    \"study_id\": \"b93bb1e9-57b8-11e8-b07a-00a098d917f8\",\n",
            "    \"pdc_study_id\": \"PDC000174\",\n",
            "    \"study_submitter_id\": \"TCGA BRCA Phosphoproteome S015-2\",\n",
            "    \"program_id\": \"10251935-5540-11e8-b664-00a098d917f8\",\n",
            "    \"project_id\": \"48af5040-5546-11e8-b664-00a098d917f8\",\n",
            "    \"study_name\": \"TCGA Breast Cancer Phosphoproteome\",\n",
            "    \"program_name\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"project_name\": \"CPTAC2 Retrospective\",\n",
            "    \"disease_type\": \"Breast Invasive Carcinoma;Other\",\n",
            "    \"primary_site\": \"Breast;Not Reported\",\n",
            "    \"analytical_fraction\": \"Phosphoproteome\",\n",
            "    \"experiment_type\": \"iTRAQ4\",\n",
            "    \"embargo_date\": null,\n",
            "    \"cases_count\": 109,\n",
            "    \"aliquots_count\": 109,\n",
            "    \"protocol_id\": \"1c06aff5-3b26-11e9-9a07-0a80fada099c\",\n",
            "    \"protocol_submitter_id\": \"P-S015-2\",\n",
            "    \"program_submitter_id\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"protocol_name\": \"TCGA_Breast_Cancer_Phosphoproteome\",\n",
            "    \"protocol_date\": \"2013-02-02\",\n",
            "    \"document_name\": \"\",\n",
            "    \"quantitation_strategy\": \"Isobaric label quantitation\",\n",
            "    \"label_free_quantitation\": \"\",\n",
            "    \"labeled_quantitation\": \"iTRAQ quantitation\",\n",
            "    \"isobaric_labeling_reagent\": \"iTRAQ\",\n",
            "    \"reporter_ion_ms_level\": \"2\",\n",
            "    \"starting_amount\": \"4.00\",\n",
            "    \"starting_amount_uom\": \"mg\",\n",
            "    \"digestion_reagent\": \"Lys-C/Trypsin\",\n",
            "    \"alkylation_reagent\": \"Iodoacetamide\",\n",
            "    \"enrichment_strategy\": \"Phosphoproteome\",\n",
            "    \"enrichment\": \"Phosphopeptide enrichment with immobilized metal affinity chromatography (NiNTA beads stripped with EDTA and loaded with FeCl3; Qiagen)\",\n",
            "    \"chromatography_dimensions_count\": \"2\",\n",
            "    \"one_d_chromatography_type\": \"bRPLC (pH 10)\",\n",
            "    \"two_d_chromatography_type\": \"RPLC\",\n",
            "    \"fractions_analyzed_count\": \"13\",\n",
            "    \"column_type\": \"New Objective; PicoFrit SELF/P\",\n",
            "    \"amount_on_column\": \"\",\n",
            "    \"amount_on_column_uom\": \"\",\n",
            "    \"column_length\": \"20\",\n",
            "    \"column_length_uom\": \"cm\",\n",
            "    \"column_inner_diameter\": \"75.00\",\n",
            "    \"column_inner_diameter_uom\": \"\\u00b5m\",\n",
            "    \"particle_size\": \"1.9\",\n",
            "    \"particle_size_uom\": \"\\u00b5m\",\n",
            "    \"particle_type\": \"Dr Maisch GmbH; ReproSil-Pur 120 C18-AQ; 1.9 um\",\n",
            "    \"gradient_length\": \"110\",\n",
            "    \"gradient_length_uom\": \"min\",\n",
            "    \"instrument_make\": \"Thermo Scientific\",\n",
            "    \"instrument_model\": \"Q Exactive\",\n",
            "    \"dissociation_type\": \"HCD\",\n",
            "    \"ms1_resolution\": \"70000\",\n",
            "    \"ms2_resolution\": \"17500\",\n",
            "    \"dda_topn\": \"Top12\",\n",
            "    \"normalized_collision_energy\": \"28%\",\n",
            "    \"acquistion_type\": \"DDA\",\n",
            "    \"dia_multiplexing\": \"N/A\",\n",
            "    \"dia_ims\": \"N/A\",\n",
            "    \"analytical_technique\": null,\n",
            "    \"chromatography_instrument_make\": null,\n",
            "    \"chromatography_instrument_model\": null,\n",
            "    \"polarity\": null,\n",
            "    \"reconstitution_solvent\": null,\n",
            "    \"reconstitution_volume\": null,\n",
            "    \"reconstitution_volume_uom\": null,\n",
            "    \"internal_standards\": null,\n",
            "    \"extraction_method\": null,\n",
            "    \"ionization_mode\": null,\n",
            "    \"publications\": [\n",
            "      {\n",
            "        \"publication_id\": \"44e2e766-89c7-11e8-bcf1-0a2705229b82\",\n",
            "        \"pubmed_id\": \"27251275\",\n",
            "        \"doi\": \"10.1038/nature18003\",\n",
            "        \"author\": \"Mertins, P.; Mani, D. R.; Ruggles, K. V.; Gillette, M. A.; Clauser, K. R.; Wang, P.; Wang, X.; Qiao, J. W.; Cao, S.; Petralia, F.; Kawaler, E.; Mundt, F.; Krug, K.; Tu, Z.; Lei, J. T.; Gatza, M. L.; Wilkerson, M.; Perou, C. M.; Yellapantula, V.; Huang, K. L.; Lin, C.; McLellan, M. D.; Yan, P.; Davies, S. R.; Townsend, R. R.; Skates, S. J.; Wang, J.; Zhang, B.; Kinsinger, C. R.; Mesri, M.; Rodriguez, H.; Ding, L.; Paulovich, A. G.; Feny\\u00f6, D.; Ellis, M. J.; Carr, S. A.; Gil Kuhn, E.; Mani, D. R.; Mertins, P.; Ketchum, K. A.; Thangudu, R. R.; Cai, S.; Oberti, M.; Whiteaker, J. R.; Wang, X.; Lin, C.; Ping, Y.; Edwards, N. J.; Madhavan, S.; McGarvey, P. B.; Wang, P.; Petralia, F.; Tu, Z.; Chan, D.; Pandey, A.; Shih, L. M.; Zhang, H.; Zhang, Z.; Thomas, S.; Zhu, H.; Whiteley, G. A.; Skates, S. J.; White, F. M.; Levine, D. A.; Boja, E. S.; Kinsinger, C. R.; Hiltke, T.; Mesri, M.; Rivers, R. C.; Rodriguez, H.; Shaw, K. M.; Stein, S. E.; Fenyo, D.; Liu, T.; McDermott, J. E.; Payne, S. H.; Rodland, K. D.; Smith, R. D.; Rudnick, P.; Snyder, M.; Zhao, Y.; Chen, X.; Ransohoff, D. F.; Hoofnagle, A. N.; Liebler, D. C.; Sanders, M. E.; Shi, Z.; Slebos, R. J.; Tabb, D. L.; Zhang, B.; Zimmerman, L. J.; Wang, Y.; Li, S.; Davies, S. R.; Ding, L.; Maher, C.; Townsend, R.; Ellis, M. J.; Lei, J. T.; Luo, J.\",\n",
            "        \"title\": \"Proteogenomics connects somatic mutations to signalling in breast cancer\",\n",
            "        \"journal\": \"Nature\",\n",
            "        \"journal_url\": \"https://www.nature.com/articles/nature18003\",\n",
            "        \"year\": \"2016\",\n",
            "        \"abstract\": \"Somatic mutations have been extensively characterized in breast cancer, but the effects of these genetic alterations on the proteomic landscape remain poorly understood. Here we describe quantitative mass-spectrometry-based proteomic and phosphoproteomic analyses of 105 genomically annotated breast cancers, of which 77 provided high-quality data. Integrated analyses provided insights into the somatic cancer genome including the consequences of chromosomal loss, such as the 5q deletion characteristic of basal-like breast cancer. Interrogation of the 5q trans-effects against the Library of Integrated Network-based Cellular Signatures, connected loss of CETN3 and SKP1 to elevated expression of epidermal growth factor receptor (EGFR), and SKP1 loss also to increased SRC tyrosine kinase. Global proteomic data confirmed a stromal-enriched group of proteins in addition to basal and luminal clusters, and pathway analysis of the phosphoproteome identified a G-protein-coupled receptor cluster that was not readily identified at the mRNA level. In addition to ERBB2, other amplicon-associated highly phosphorylated kinases were identified, including CDK12, PAK1, PTK2, RIPK2 and TLK2. We demonstrate that proteogenomic analysis of breast cancer elucidates the functional consequences of somatic mutations, narrows candidate nominations for driver genes within large deletions and amplified regions, and identifies therapeutic targets.\",\n",
            "        \"full_text\": [\n",
            "          \"Proteogenomics connects somatic mutations to signaling in breast cancer\\n\\nSummary\",\n",
            "          \"Somatic mutations have been extensively characterized in breast cancer, but the effects of these genetic alterations on the proteomic landscape remain poorly understood. We describe quantitative mass spectrometry-based proteomic and phosphoproteomic\",\n",
            "          \"mass spectrometry-based proteomic and phosphoproteomic analyses of 105 genomically annotated breast cancers of which 77 provided high-quality data. Integrated analyses allowed insights into the somatic cancer genome including the consequences of\",\n",
            "          \"into the somatic cancer genome including the consequences of chromosomal loss, such as the 5q deletion characteristic of basal-like breast cancer. The 5q trans effects were interrogated against the Library of Integrated Network-based Cellular Signatures,\",\n",
            "          \"the Library of Integrated Network-based Cellular Signatures, thereby connecting CETN3 and SKP1 loss to elevated expression of EGFR, and SKP1 loss also to increased SRC. Global proteomic data confirmed a stromal-enriched group in addition to basal and\",\n",
            "          \"confirmed a stromal-enriched group in addition to basal and luminal clusters and pathway analysis of the phosphoproteome identified a G Protein-coupled receptor cluster that was not readily identified at the mRNA level. Besides ERBB2, other\",\n",
            "          \"not readily identified at the mRNA level. Besides ERBB2, other amplicon-associated, highly phosphorylated kinases were identified, including CDK12, PAK1, PTK2, RIPK2 and TLK2. We demonstrate that proteogenomic analysis of breast cancer elucidates\",\n",
            "          \"that proteogenomic analysis of breast cancer elucidates functional consequences of somatic mutations, narrows candidate nominations for driver genes within large deletions and amplified regions, and identifies therapeutic targets.\",\n",
            "          \"Introduction\",\n",
            "          \"A central deficiency in our knowledge of cancer concerns how genomic changes drive the proteome and phosphoproteome to execute phenotypic characteristics. The initial proteomic characterization in the TCGA breast study was performed using reversed phase\",\n",
            "          \"in the TCGA breast study was performed using reversed phase protein arrays; however this approach is restricted by antibody availability. To provide greater analytical breadth, the NCI Clinical Proteomic Tumor Analysis Consortium (CPTAC) is analyzing the\",\n",
            "          \"Proteomic Tumor Analysis Consortium (CPTAC) is analyzing the proteomes of genome-annotated TCGA tumor specimens using mass spectrometry. Herein we describe integrated proteogenomic analyses of TCGA breast cancer samples representing the four principal\",\n",
            "          \"of TCGA breast cancer samples representing the four principal mRNA-defined breast cancer intrinsic subtypes.\",\n",
            "          \"Proteogenomic analysis of TCGA samples\",\n",
            "          \"105 breast tumors previously characterized by the TCGA were selected for proteomic analysis after histopathological documentation (Supplementary Tables 1 and 2). The cohort included a balanced representation of PAM50-defined intrinsic subtypes including\",\n",
            "          \"representation of PAM50-defined intrinsic subtypes including 25 basal-like, 29 luminal A, 33 luminal B, and 18 HER2 (ERBB2)-enriched tumors, along with 3 normal breast tissue samples. Samples were analyzed by high-resolution, accurate mass, tandem mass\",\n",
            "          \"were analyzed by high-resolution, accurate mass, tandem mass spectrometry (MS) that included extensive peptide fractionation and phosphopeptide enrichment (Extended Data Fig. 1a). An isobaric peptide labeling approach (iTRAQ) was employed to quantify\",\n",
            "          \"peptide labeling approach (iTRAQ) was employed to quantify protein and phosphosite levels across samples, with 37 iTRAQ 4-plexes analyzed in total. A total of 15,369 proteins (12,405 genes) and 62,679 phosphosites were confidently identified with 11,632\",\n",
            "          \"and 62,679 phosphosites were confidently identified with 11,632 proteins/tumor and 26,310 phosphosites/tumor on average (Supplementary Tables 3, 4 and Supplementary Methods). After filtering for observation in at least a quarter of the samples\",\n",
            "          \"filtering for observation in at least a quarter of the samples (Supplementary Methods, Extended Data Fig. 1b) 12,553 proteins (10,062 genes) and 33,239 phosphosites, with their relative abundances quantified across tumors, were used in subsequent analyses\",\n",
            "          \"quantified across tumors, were used in subsequent analyses in this study. Stable longitudinal performance and low technical noise were demonstrated by repeated, interspersed analyses of a single batch of patient-derived luminal and basal breast cancer\",\n",
            "          \"single batch of patient-derived luminal and basal breast cancer xenograft samples (Extended Data Fig. 1d,e). Due to the heterogeneous nature of breast tumors, and because proteomic analyses were performed on tumor fragments that were different from those\",\n",
            "          \"performed on tumor fragments that were different from those used in the genomic analyses, rigorous pre-specified sample and data QC metrics were implemented (Supplementary Discussion and Extended Data Figures 2, 3). Extensive analyses concluded that 28 of\",\n",
            "          \"Data Figures 2, 3). Extensive analyses concluded that 28 of the 105 samples were compromised by protein degradation. These samples were excluded from further analysis with subsequent informatics focused on the 77 tumor samples and three biological\",\n",
            "          \"focused on the 77 tumor samples and three biological replicates.\",\n",
            "          \"Genome and transcriptomic variation was observed at the peptide level by searching MS/MS spectra not matched to RefSeq against a patient-specific sequence database (Fig. 1a). The database was constructed using the QUILTS software package leveraging RefSeq\",\n",
            "          \"constructed using the QUILTS software package leveraging RefSeq gene models based on whole exome and RNA-seq data generated from portions of the same tumors and matched germline DNA (Fig. 1a, Supplementary Table 5). While these analyses detected a number\",\n",
            "          \"Supplementary Table 5). While these analyses detected a number of single amino-acid variants (SAAVs), frameshifts, and splice junctions, including splice isoforms that had been detected as only single transcript reads by RNA-seq (Fig. 1b, Supplementary\",\n",
            "          \"only single transcript reads by RNA-seq (Fig. 1b, Supplementary Table 5), the number of genomic and transcriptomic variants that were confirmed as peptides by MS was low (Supplementary Discussion). Sparse detection of individual genomic variants by\",\n",
            "          \"Discussion). Sparse detection of individual genomic variants by peptide sequencing has been noted in our previous studies and reflects limited coverage at the single amino-acid level with current technology. However quantitative MS analysis of multiple\",\n",
            "          \"technology. However quantitative MS analysis of multiple peptides for each protein is used to reliably infer overall protein levels. This is an advantage for MS since antibody-based protein expression analysis is typically based on a single epitope. To\",\n",
            "          \"expression analysis is typically based on a single epitope. To illustrate this capability in the current data set an initial analysis of three frequently mutated genes in breast cancer (TP53, PIK3CA, and GATA3) and three clinical biomarkers (ER, PGR, and\",\n",
            "          \"PIK3CA, and GATA3) and three clinical biomarkers (ER, PGR, and ERBB2) was conducted (Fig. 1c, Supplementary Table 6, 7 and Supplementary Discussion). As expected, TP53 missense mutations were associated with elevated MS-based protein levels, as observed\",\n",
            "          \"associated with elevated MS-based protein levels, as observed by RPPA (Reverse Phase Protein Array), especially in basal-like breast cancer. TP53 nonsense and frame-shift mutations were associated with a decrease in TP53 protein levels that was\",\n",
            "          \"were associated with a decrease in TP53 protein levels that was particularly striking in the MS data. In contrast, the mostly C-terminal GATA3 frame-shift alterations did not result in decreased protein expression when measured by the median of all GATA3\",\n",
            "          \"protein expression when measured by the median of all GATA3 peptides, suggesting these proteins are expressed despite truncation. No consistent effect of somatic PIK3CA mutation was observed at the level of protein expression. Good correlations between\",\n",
            "          \"at the level of protein expression. Good correlations between RNA-seq and MS-protein expression levels were found for ESR1 (r=0.74), PGR (r=0.74), ERBB2 (r=0.84) and GATA3 (r=0.83) with moderate correlations observed for PIK3CA (r=0.45) and TP53 (r=0.36).\",\n",
            "          \"correlations observed for PIK3CA (r=0.45) and TP53 (r=0.36). Lower TP53 protein abundance levels compared to mRNA levels were especially prevalent in luminal tumors, suggesting post-transcriptional regulatory mechanisms such as proteasomal degradation. To\",\n",
            "          \"regulatory mechanisms such as proteasomal degradation. To explore this hypothesis a search was made for E3 ligases that showed negative correlation to p53 protein (Supplementary Table 8). These analyses identified UBE3A (r=\\u22120.42; adj. pval=0.05) (Extended\",\n",
            "          \"analyses identified UBE3A (r=\\u22120.42; adj. pval=0.05) (Extended Data Fig. 4a), an established TP53 E3 ligase. In comparing CNA, RNA, and protein levels for GATA3, copy number gains in chromosome 10q were anti-correlated with RNA and protein levels in\",\n",
            "          \"10q were anti-correlated with RNA and protein levels in basal-like tumors. This observation prompted a search for other gains or losses that were anticorrelated with RNA and/or protein levels (see Extended Data Fig. 4b for further analyses). Overall, six\",\n",
            "          \"(see Extended Data Fig. 4b for further analyses). Overall, six genes were identified that significantly anti-correlated at an FDR<0.05 on both RNA and protein level to their CNA signals (Extended Data Fig. 4b). GATA3 amplification on 10q in basal-like\",\n",
            "          \"Data Fig. 4b). GATA3 amplification on 10q in basal-like breast cancer showed the strongest anti-correlation, followed by the hexosamine and glycolysis pathway enzymes GFPT2 and HK3, which are upregulated in basal-like breast cancer despite being subjected\",\n",
            "          \"upregulated in basal-like breast cancer despite being subjected to frequent chromosomal deletion on 5q. Global analysis of the correlation of mRNA-to-protein yielded a median Pearson value of r=0.39, with 6,135 out of 9,302 protein/mRNA pairs (66.0%)\",\n",
            "          \"of r=0.39, with 6,135 out of 9,302 protein/mRNA pairs (66.0%) correlating significantly at an FDR<0.05 (Extended Data Fig. 4c, Supplementary Table 9 and Supplementary Discussion). Similar to the colon cancer analysis metabolic functions such as amino\",\n",
            "          \"to the colon cancer analysis metabolic functions such as amino acid, sugar and fatty acid metabolism were found to be enriched among positively correlated genes whereas ribosomal, RNA polymerase and mRNA splicing functions were negatively correlated.\",\n",
            "          \"and mRNA splicing functions were negatively correlated. Overall these analyses demonstrate the utility of global proteome correlation analysis for both confirmation of suspected regulatory mechanisms and identification of candidate regulators meriting\",\n",
            "          \"mechanisms and identification of candidate regulators meriting further investigation.\",\n",
            "          \"Copy Number Alterations\",\n",
            "          \"To determine the consequences of CNA on mRNA, protein and phosphoprotein abundance, both in \\u201ccis\\u201d on genes within the aberrant locus and in \\u201ctrans\\u201d on genes encoded elsewhere, univariate correlation analysis was used as previously described. A total of\",\n",
            "          \"analysis was used as previously described. A total of 7,776 genes with CNA, mRNA and protein measurements were analyzed by calculating Pearson correlation and associated statistical significance (Benjamini-Hochberg corrected p-value) for all possible\",\n",
            "          \"(Benjamini-Hochberg corrected p-value) for all possible CNA-mRNA and CNA-protein pairs (Fig. 2a, Supplementary Table 10, Extended Data Fig. 5a, see Methods). For the phosphoproteome, 4,472 CNA-phosphoprotein pairs were analyzed (Extended Data Fig. 5b).\",\n",
            "          \"CNA-phosphoprotein pairs were analyzed (Extended Data Fig. 5b). Significant positive correlations (cis) were observed for 64% of all CNA-mRNA, 31% of all CNA-protein, and 20% of all CNA-phosphoprotein pairs Fig. 2b. Proteins and phosphoproteins correlated\",\n",
            "          \"pairs Fig. 2b. Proteins and phosphoproteins correlated in cis to CNA were, for the most part, a subset of the cis effects observed in mRNA-to-CNA correlation (Fig. 2b, Supplementary Table 10). The fractional difference of well-annotated oncogenes and\",\n",
            "          \"10). The fractional difference of well-annotated oncogenes and tumor suppressor genes among the significantly cis-correlated CNA-to-mRNA and -protein gene pairs was analyzed. Based on a reference list of 487 oncogenes and tumor suppressors (Supplementary\",\n",
            "          \"list of 487 oncogenes and tumor suppressors (Supplementary Table 10), these cancer relevant genes occur 37.6% more frequently in the subset of genes that correlate both on CNA/mRNA and CNA/protein levels than in the subset that only correlate on CNA/mRNA\",\n",
            "          \"levels than in the subset that only correlate on CNA/mRNA but not on CNA/protein level (Fisher exact p-value=0.02). This suggests that CNA events with a tumor promoting outcome more likely lead to cis regulatory effects on both the protein and mRNA level,\",\n",
            "          \"to cis regulatory effects on both the protein and mRNA level, whereas CNA events with no documented role in tumorigenesis are more likely to be neutralized on the protein level than on the RNA level. Trans effects (Fig. 2a) appear as vertical bands, with\",\n",
            "          \"level. Trans effects (Fig. 2a) appear as vertical bands, with accompanying frequency histograms (in blue) highlighting \\u201chot spots\\u201d of significant trans effects. Using a minimum threshold of 50 trans-affected genes, 68% of the tested genes were associated\",\n",
            "          \"trans-affected genes, 68% of the tested genes were associated with trans effects on the mRNA level, whereas only 13% were associated with effects on the protein level and 8% on the phosphoprotein level. Importantly, CNA-protein correlations appeared to be\",\n",
            "          \"level. Importantly, CNA-protein correlations appeared to be a reduced representation of CNA-mRNA correlations. Furthermore, for many CNA regions correlations were more directionally uniform on the protein level than on the mRNA level. CNA regions\",\n",
            "          \"on the protein level than on the mRNA level. CNA regions exhibiting the most trans associations at the protein level were found on chromosomes 5q (LOH in basal; gain in LumB), 10p (gain in basal), 12 (gain in basal), 16q (LumA deletion), 17q (LumB\",\n",
            "          \"in basal), 12 (gain in basal), 16q (LumA deletion), 17q (LumB amplification), and 22q (LOH in luminal and basal) (Extended Data Fig. 5a).\",\n",
            "          \"Trans associations are not necessarily direct consequences of the chromosomal aberration. For example since 5q loss occurs in at least 50% of basal-like breast cancers, many of the trans effects involve genes that mark the basal-subtype. To identify\",\n",
            "          \"effects involve genes that mark the basal-subtype. To identify candidate driver genes whose copy number alterations are direct drivers of trans effects, results were compared with functional knock-down data on 3,797 genes in the Library of Integrated\",\n",
            "          \"knock-down data on 3,797 genes in the Library of Integrated Cellular Signatures (LINCS) database (http://www.lincsproject.org/). For any given gene with copy number alterations (\\u201cCNA-gene\\u201d), sets of genes were identified corresponding to proteins that\",\n",
            "          \"sets of genes were identified corresponding to proteins that changed where there was gain (\\u201cCNA-gain trans gene set\\u201d) or loss (\\u201cCNA-loss trans gene set\\u201d). These gene sets were then compared to the effects of gene knock down in the LINCS database (see\",\n",
            "          \"to the effects of gene knock down in the LINCS database (see Methods). Queries for 502 different CNA-genes meeting the criteria defined above identified 10 CNA-genes that could be functionally connected to both CNA-gain and CNA-loss trans protein-level\",\n",
            "          \"connected to both CNA-gain and CNA-loss trans protein-level effects (Extended Data Fig. 5c, Supplementary Table 11). A permutation-based approach implemented to test significance (see Supplementary Methods) yielded an FDR < 0.05 for 10 genes affected by\",\n",
            "          \"Methods) yielded an FDR < 0.05 for 10 genes affected by both CNA gains and losses (Fig. 2c). These proteins were defined as potential regulatory candidates for the CNA trans effects observed on the proteome level in this study, since in a gene-dependent\",\n",
            "          \"on the proteome level in this study, since in a gene-dependent manner on average 17% of these trans effects were consistent with the knockdown profiles. Notably, the established oncogenic receptor tyrosine kinase ERBB2 was functionally connected only to\",\n",
            "          \"tyrosine kinase ERBB2 was functionally connected only to CNA gain trans effects (Supplementary Table 11). The E3 ligase SKP1 and the ribonucleoprotein export factor CETN3, both located on chromosome arm 5q with frequent losses in basal-like breast cancer\",\n",
            "          \"arm 5q with frequent losses in basal-like breast cancer and less frequent gains in luminal B breast cancer, were detected as potential regulators affecting the expression of the tyrosine kinase and therapeutic target EGFR, and SKP1 also was linked to SRC\",\n",
            "          \"and therapeutic target EGFR, and SKP1 also was linked to SRC (Extended Data Fig. 5d). Another potential regulator, FBXO7, (a substrate recognition component of the SCF (SKP1-CUL1-F-box protein)-type E3 ubiquitin ligase complex), was affected mostly by LOH\",\n",
            "          \"E3 ubiquitin ligase complex), was affected mostly by LOH events on chromosome 22q. Interestingly, in a recent human interaction proteome study SKP1 and FBXO7 were listed as interaction partners.\",\n",
            "          \"Clustering and Network analysis\",\n",
            "          \"Transcriptional profiling has converged on four major breast cancer subtypes: Luminal A and B, basal and HER2-enriched. To investigate the extent to which the PAM50 \\u201cintrinsic\\u201d breast cancer classification scheme is reflected or refined on the proteome\",\n",
            "          \"classification scheme is reflected or refined on the proteome level in the CPTAC samples, clustering analyses were first restricted to the reduced set of PAM50 genes. When RNA data for the 50 PAM50 genes were clustered directly (without using a\",\n",
            "          \"for the 50 PAM50 genes were clustered directly (without using a classifier), the clustering was similar to the TCGA PAM50 annotation (second annotation bar in Fig. 3a). Restricting both the RNA and proteome data to the set of 35 PAM50 genes observed in\",\n",
            "          \"RNA and proteome data to the set of 35 PAM50 genes observed in the proteome produced a similar result (bottom two annotation bars in Fig. 3a), and all the major PAM50 groups were recapitulated in the proteome almost as well as in the RNA data. This\",\n",
            "          \"in the proteome almost as well as in the RNA data. This indicates that although different tissue sections of the same tumors were used for RNA-seq and protein analysis, very similar subtype-defining features can be observed in both data types. Global\",\n",
            "          \"features can be observed in both data types. Global proteome and phosphoproteome data were then used to identify proteome subtypes in an unsupervised manner. Consensus clustering identified basal-enriched, luminal-enriched, and stromal-enriched clusters\",\n",
            "          \"basal-enriched, luminal-enriched, and stromal-enriched clusters (Extended Data Figs. 6a\\u2013d, 7a). Unlike the clustering observed with PAM50 genes, mRNA-defined HER2-enriched tumors were distributed across these three proteomic subgroups. The basal-enriched\",\n",
            "          \"across these three proteomic subgroups. The basal-enriched and luminal-enriched groups showed a strong overlap with the mRNA-based PAM50 basal-like and luminal subgroups, whereas stromal-enriched proteome subtype represented a mix of all PAM50 mRNA-based\",\n",
            "          \"proteome subtype represented a mix of all PAM50 mRNA-based subtypes, and has a significantly enriched stromal signature (Extended Data Fig. 3e). Among the stromal-enriched tumors there was strong representation of reactive type I tumors as classified by\",\n",
            "          \"representation of reactive type I tumors as classified by RPPA (Supplementary Table 12), showing agreement between the RPPA and mass spectrometry-based protein analyses for the detection of a tumor subgroup characterized by stromal gene expression.\",\n",
            "          \"Since the basal- and luminal-enriched proteome subgroups are coherent, pathway analyses were conducted on these two subtypes, using the stromal-enriched subgroup as a control to assess specificity. (Fig. 3c, Extended Data Fig. 7b, Supplementary Table 13).\",\n",
            "          \"(Fig. 3c, Extended Data Fig. 7b, Supplementary Table 13). The luminal-enriched subgroup was exclusively enriched for estradiol and ESR1-driven gene sets. In contrast, multiple gene sets were enriched and upregulated specifically in the basal-like tumors.\",\n",
            "          \"enriched and upregulated specifically in the basal-like tumors. Particularly extensive basal-like enrichment was seen for MYC target genes; for cell cycle, checkpoint, and DNA repair pathways including regulators AURKA/B, ATM, ATR, CHEK1/2, and BRCA1/2;\",\n",
            "          \"including regulators AURKA/B, ATM, ATR, CHEK1/2, and BRCA1/2; and for immune response/inflammation, including T-cell, B-cell, and neutrophil signatures. The complementarity of transcriptional, proteomic, and phosphoproteomic data was also highlighted in\",\n",
            "          \"proteomic, and phosphoproteomic data was also highlighted in these analyses (Extended Data Fig. 7c, d).\",\n",
            "          \"Using phosphorylation status as a proxy for activity, phosphoproteome profiling can theoretically be used to develop a signaling pathway-based cancer classification. K-means consensus clustering was therefore performed on pathways derived from single\",\n",
            "          \"was therefore performed on pathways derived from single sample GSEA analysis of phosphopeptide data (Methods, Supplementary Tables 14 and 15). Of four robustly segregated groups, subgroups 2 and 3 substantially recapitulated the stromal- and\",\n",
            "          \"subgroups 2 and 3 substantially recapitulated the stromal- and luminal-enriched proteomic subgroups, respectively (Fig. 3d, Extended Data Fig. 8a). Subgroup 4 included a majority of tumors from the basal-enriched proteomic subgroup, but was admixed\",\n",
            "          \"from the basal-enriched proteomic subgroup, but was admixed particularly with luminal-enriched samples. This subgroup was defined by high levels of cell cycle and checkpoint activity. All basal and a majority of non-basal samples in this subgroup had TP53\",\n",
            "          \"and a majority of non-basal samples in this subgroup had TP53 mutations. Consistent with high levels of cell cycle activity, a multivariate kinase-to-phosphosite abundance regression analysis highlighted CDK1 as one of the most highly connected kinases in\",\n",
            "          \"highlighted CDK1 as one of the most highly connected kinases in this study (Extended Data Fig. 8b, Supplementary Table 16). Subgroup 1 was a novel subgroup defined exclusively in the phosphopeptide/pathway activity domain, with no enrichment for either\",\n",
            "          \"activity domain, with no enrichment for either proteomic or PAM50 subtypes. It was defined by G-protein, G-protein coupled receptor, and inositol phosphate metabolism signatures, as well as ionotropic glutamate signaling (Fig. 3d). Co-expression patterns\",\n",
            "          \"glutamate signaling (Fig. 3d). Co-expression patterns among genes/proteins across different subgroups were also analyzed using a Joint Random Forest (JRF) method that identified network modules, such as an MMP9 module, with different interaction patterns\",\n",
            "          \"such as an MMP9 module, with different interaction patterns between basal-enriched and luminal-enriched subgroups. These latter patterns appeared specific to the proteome-level data (Extended Data Fig. 8 c\\u2013f, Supplementary Table 17 and Supplementary\",\n",
            "          \"Data Fig. 8 c\\u2013f, Supplementary Table 17 and Supplementary Methods).\",\n",
            "          \"Phosphosite markers in PIK3CA and TP53 mutant breast cancer\",\n",
            "          \"TP53 and PIK3CA are the most recurrently mutated genes in breast cancer, with frequencies for PIK3CA at 43% in luminal tumors and for TP53 at 84% in basal-like tumors. Most of the PIK3CA missense mutations were gain of function mutations and therefore\",\n",
            "          \"mutations were gain of function mutations and therefore expected to lead to an activation of the PI3K signaling cascade, but the extent to which this occurs has been controversial and there is uncertainty which pathway components are effectors. Marker\",\n",
            "          \"is uncertainty which pathway components are effectors. Marker selection analysis was therefore performed for upregulated phosphosites in PIK3CA-mutated tumors. In total, 62 phosphosites were identified that were positively associated with PIK3CA mutation\",\n",
            "          \"identified that were positively associated with PIK3CA mutation (FDR<0.05), including the kinases RPS6KA5 and EIF2AK4 (Extended Data Fig. 9a, Supplementary Table 18). Calculating the average phosphorylation signal of these marker phosphosites provided a\",\n",
            "          \"phosphorylation signal of these marker phosphosites provided a read-out for PI3K pathway activity in PIK3CA-mutated tumors, with 15 of the 26 mutated tumors (58%) exhibiting an activated PIK3CA mutation signature. Of note, the identified PIK3CA mutant\",\n",
            "          \"mutation signature. Of note, the identified PIK3CA mutant phosphoproteome signature was activated in all tumors harboring helical domain PIK3CA mutations but only 2 of 10 tumors harboring kinase domain mutations. To test if the identified differences in\",\n",
            "          \"domain mutations. To test if the identified differences in the phosphoproteome of PI3K mutant versus wild-type tumors could be explained by mutation of PIK3CA, the tumor data were compared to phosphosite signatures derived from isogenic PIK3CA mutant cell\",\n",
            "          \"phosphosite signatures derived from isogenic PIK3CA mutant cell lines (Extended Data Fig. 9b, Supplementary Table 18). There was an enrichment of signatures derived from helical domain-mutated isogenic cell lines, but not from kinase domain-mutated cells,\",\n",
            "          \"isogenic cell lines, but not from kinase domain-mutated cells, supporting the observations in primary tumors.\",\n",
            "          \"The same strategy was used to identify phosphorylation signaling events connected to TP53 mutation. A total of 56 phosphosites upregulated in TP53 mutant tumors were identified that were independent of basal-like subtype association (Extended Data Fig.\",\n",
            "          \"of basal-like subtype association (Extended Data Fig. 9c, Supplementary Table 18). Using the average phosphorylation signal of these marker phosphosites as a proxy for TP53 mutation-driven cell cycle control, 22 of 41 mutated tumors (54%) showed\",\n",
            "          \"cell cycle control, 22 of 41 mutated tumors (54%) showed upregulated signals. This TP53 mutant phosphosignature was somewhat enhanced in tumors in which mutations occurred almost exclusively in the DNA binding region compared to those with\",\n",
            "          \"exclusively in the DNA binding region compared to those with non-sense/frameshift mutations. In addition to the well-described checkpoint kinase CHEK2, significantly upregulated phosphosites were identified for the kinases MASTL and EEF2K in TP53-mutated\",\n",
            "          \"were identified for the kinases MASTL and EEF2K in TP53-mutated tumors. Single sample GSEA analysis of isogenic p53-mutant phosphosignatures showed an enrichment of a phosphosignature derived from R273H mutated isogenic cells (Extended Data Fig. 9d),\",\n",
            "          \"from R273H mutated isogenic cells (Extended Data Fig. 9d), confirming the pronounced effect of missense mutations in the DNA-binding region on phosphorylation pathways.\",\n",
            "          \"Identification of gene amplification and breast cancer subtype-specific activated kinases in human breast cancer\",\n",
            "          \"CNA spans many driver gene candidates and RNA expression has been frequently used to narrow candidate nominations. Proteogenomic analysis should further promote this nomination process. In this candidate refinement a focus on protein kinases is warranted,\",\n",
            "          \"candidate refinement a focus on protein kinases is warranted, since many are drug targets. An in-depth proteogenomic pipeline was developed that flagged kinases, expression levels of which were at least 1.5 interquartile ranges higher than the median\",\n",
            "          \"were at least 1.5 interquartile ranges higher than the median (Supplemental Table 19). A proteogenomic circos-like plot (termed a \\u201cpircos\\u201d plot) was used to map these outlier kinase values onto the genome (Fig. 4a,b, Extended Data Fig. 10a). The ERBB2\",\n",
            "          \"onto the genome (Fig. 4a,b, Extended Data Fig. 10a). The ERBB2 locus showed the strongest effect of increased phosphoprotein levels associated with gene amplification-driven RNA and protein over expression (Fig. 4a). CDK12 is a positive transcriptional\",\n",
            "          \"over expression (Fig. 4a). CDK12 is a positive transcriptional regulator of homologous recombination repair genes with its partner Cyclin K, and is often encompassed by the ERBB2 amplicon. This gene was also found to be upregulated at the RNA, protein and\",\n",
            "          \"gene was also found to be upregulated at the RNA, protein and phosphosite level indicating that CDK12 is highly active in the majority of ERBB2 positive tumors (Fig. 4a). The analysis of the ERBB2 amplicon also uncovered co-outlier phosphorylation status\",\n",
            "          \"ERBB2 amplicon also uncovered co-outlier phosphorylation status for MED1, GRB7, MSL1, CASC3 and TOP2A, all previously described in association with ERBB2 amplification. To better understand the downstream effects of ERBB2 amplification, additional\",\n",
            "          \"the downstream effects of ERBB2 amplification, additional phosphosite outliers were identified in 41 known ERBB2 signaling genes for the 15 samples that had ERBB2 phosphosite outlier expression (Extended Data Fig. 10b).\",\n",
            "          \"These canonical findings stimulated a proteogenomic analysis to identify additional outlier kinases in the breast cancer genome. A proteogenomic dissection of chromosome 11q based on PAK1 amplification (Fig. 4b,c), a breast cancer driver kinase,\",\n",
            "          \"PAK1 amplification (Fig. 4b,c), a breast cancer driver kinase, illustrated that PAK1 is hyperphosphorylated in PAK1 amplified tumors, along with CLNS1A, RFS1 and GAB2. Additional examples of outlier kinases included PTK2 and RIPK2 in association with\",\n",
            "          \"of outlier kinases included PTK2 and RIPK2 in association with amplification of chromosome 8q (Fig. 4c; Extended Data Fig. 10a,c). PAK1 and TLK2 (17q23) appear to be luminal breast cancer specific events Fig. 4c; Extended Data Fig. 10c). To further\",\n",
            "          \"specific events Fig. 4c; Extended Data Fig. 10c). To further examine whether outlier kinases were breast cancer subtype-specific, independent of amplification status, the BH-corrected probability was calculated of finding that number of phosphosite\",\n",
            "          \"was calculated of finding that number of phosphosite outliers within a subtype, given the total number of outliers across all subtypes, the subtype sample size and the total sample size. (Fig. 4d). These analyses led to the expected identification of\",\n",
            "          \"(Fig. 4d). These analyses led to the expected identification of ERBB2 in the HER2-enriched subtype at the 5% FDR level, as well as the new finding of CDC42BPG (MRGKG), an effector kinase for RHO-family GTPases. In basal-like breast cancer, two kinases,\",\n",
            "          \"RHO-family GTPases. In basal-like breast cancer, two kinases, PRKDC and SPEG, were significant at the 5% FDR level. PRKDC is a non-homologous end-joining (NHEJ) factor that can be phosphorylated by ATM kinase, and is therefore a logical finding in this\",\n",
            "          \"by ATM kinase, and is therefore a logical finding in this disease subset. However SPEG, a kinase associated with severe dilated cardiomyopathy when suppressed, has not been previously reported in association with breast cancer. A larger number of\",\n",
            "          \"reported in association with breast cancer. A larger number of subtype-specific kinases were detected at the 10% FDR level, several of which have recently described relevance in breast cancer, including PRKD3 in basal-like breast cancer, the LKB-regulated\",\n",
            "          \"including PRKD3 in basal-like breast cancer, the LKB-regulated SIK3 in luminal A breast cancer and CDK13 in luminal B breast cancer, which, similar to CDK12, can interact with Cyclin K.\",\n",
            "          \"Discussion\",\n",
            "          \"The analytic breadth and depth of proteomic and phosphoproteomic analyses displayed in this study demonstrates the strengths of mass spectrometry-based proteomics, but also some of the limitations inherent in proteolytic peptide sequencing (see\",\n",
            "          \"the limitations inherent in proteolytic peptide sequencing (see Supplementary Discussion). An example of how high-dimensional proteomic analysis provides insight into unresolved genomic issues concerns the study of loss of the long arm of chromosome 5\",\n",
            "          \"concerns the study of loss of the long arm of chromosome 5 (5q). Analysis of RNA and protein correlations narrowed the list of potential trans-deregulated proteins. Orthogonal candidate screening using functional genomics methods identified loss of CETN3\",\n",
            "          \"using functional genomics methods identified loss of CETN3 and SKP1 as potential transregulators, with upregulation of EGFR as a downstream consequence in basal-like breast cancers. While further experimental evidence must be sought for these proposed\",\n",
            "          \"further experimental evidence must be sought for these proposed regulatory relationships, SKP1/Cullin complex has already been linked to EGFR activation in Glioma. Unfortunately EGFR targeting has not to date proven to be effective therapy in basal-like\",\n",
            "          \"has not to date proven to be effective therapy in basal-like breast cancer. This might be due to the fact the SKP1 loss deregulates multiple targets requiring a much broader inhibitory strategy.\",\n",
            "          \"It is recognized that PIK3CA mutations do not strongly activate canonical downstream effectors. Mass spectrometry-based phosphoproteomics provides an opportunity for unbiased examination of downstream signaling events consequent upon PIK3CA mutational\",\n",
            "          \"downstream signaling events consequent upon PIK3CA mutational activation. These studies revealed that common PIK3CA mutations affect a large number of targets with diverse functionalities including the kinases RPS6KA5 and EIF2AK4. Thus, the data and\",\n",
            "          \"including the kinases RPS6KA5 and EIF2AK4. Thus, the data and analyses reported here extend our knowledge of the effectors that promote tumorigenesis in response to constitutive activation of PI3 kinase. Similarly, TP53 mutation-associated phosphopeptides\",\n",
            "          \"PI3 kinase. Similarly, TP53 mutation-associated phosphopeptides point towards novel functionalities, including regulation of the kinases MASTL and EEF2K.\",\n",
            "          \"A central goal in breast cancer research has been the identification of druggable kinases beyond HER2. Candidate genes that exhibited similar gene amplification-driven proteogenomic patterns to HER2 included CDK12, TLK2, PAK1 and RIPK2. The proteogenomic\",\n",
            "          \"to HER2 included CDK12, TLK2, PAK1 and RIPK2. The proteogenomic link with gene amplification was particularly strong for CDK12, in keeping with its location in the ERBB2 amplicon, while the strengths of correlation between DNA amplification, RNA, protein\",\n",
            "          \"of correlation between DNA amplification, RNA, protein and phosphoprotein for the other examples were more variable. The presence of activated CDK12 in the ERBB2 amplicon might explain why tumors arising in BRCA1 carriers are usually ERBB2 negative. As a\",\n",
            "          \"arising in BRCA1 carriers are usually ERBB2 negative. As a positive transcriptional regulator of BRCA1 and multiple FANC family members, CDK12 promotes DNA repair by homologous recombination. CDK12 amplification would, therefore, oppose the functional\",\n",
            "          \"CDK12 amplification would, therefore, oppose the functional effects of BRCA1 haploinsufficiency during tumor evolution. Overall, multiple outlier kinases generate testable therapeutic hypotheses for which enabling inhibitors are in development. For\",\n",
            "          \"for which enabling inhibitors are in development. For example PAK1 has recently been confirmed to be a therapeutic target and poor prognosis factor in luminal breast cancer.\",\n",
            "          \"Although incomplete outcome data and the remarkable heterogeneity of breast cancer are additional relevant constraints, the number of TCGA specimens analyzed here is insufficient to support conclusive clinical correlations. Only 8 deaths occurred among\",\n",
            "          \"conclusive clinical correlations. Only 8 deaths occurred among the 77 patients, which are too few samples to provide sufficient statistical power for association analysis. Adequately powered MS-based clinical investigation will require targeted\",\n",
            "          \"powered MS-based clinical investigation will require targeted approaches, especially given the highly limited amount of patient material available from clinical trials and the mostly formalin-fixed nature of the specimens. The current analysis is\",\n",
            "          \"formalin-fixed nature of the specimens. The current analysis is therefore centered on biological findings and correlations, with orthogonal validation and false discovery concerns addressed through an examination of cell-line databases of the effects of\",\n",
            "          \"through an examination of cell-line databases of the effects of individual gene perturbations. Typical of a multi-tiered analysis of this complexity, there are many hypotheses to test, and many findings that require further investigation.\",\n",
            "          \"In conclusion, this study provides a high-quality proteomic resource for human breast cancer investigation and illustrates technologies and analytical approaches that provide an important new opportunity to connect the genome to the proteome. Larger-scale\",\n",
            "          \"opportunity to connect the genome to the proteome. Larger-scale exploration of discovery proteomics in the clinical setting will require improvements in clinical investigation, including acquisition of adequate amounts of optimally collected tumor tissue\",\n",
            "          \"of adequate amounts of optimally collected tumor tissue both before and during therapy as well as advances in MS proteomics to reduce sample input and increase sensitivity for low abundance proteins and modified peptides.\",\n",
            "          \"Extended Data\\n\\nExperimental and data analysis workflows and longitudinal data generation quality control\",\n",
            "          \"a, iTRAQ 4-plex global proteome and phosphoproteome analysis workflow. 105 TCGA breast tumors were analyzed in 35 iTRAQ 4-plex experiments (plus 1 replicate and 1 normal sample experiment), with three tumors of different subtypes compared to a fourth\",\n",
            "          \"with three tumors of different subtypes compared to a fourth common internal reference sample in each experiment. The reference sample comprised 10 individual tumors of each of the 4 major breast cancer intrinsic subtypes and served as an internal\",\n",
            "          \"breast cancer intrinsic subtypes and served as an internal standard for all proteins and phosphoproteins quantified in this study. Each iTRAQ MS/MS spectrum measures a peptide from 4 samples (3 individual patients and the reference sample mix of 40\",\n",
            "          \"(3 individual patients and the reference sample mix of 40 patients). More than 400,000 distinct peptides were identified and quantified in ~14 million MS/MS spectra. Personalized tumor-specific protein databases were generated in the QUILTS software\",\n",
            "          \"protein databases were generated in the QUILTS software package using whole exome sequencing-derived variant calls and RNAseq-derived transcript information. All mass spectrometry data was analyzed using the Spectrum Mill software package. b, Overview of\",\n",
            "          \"using the Spectrum Mill software package. b, Overview of proteome and phosphoproteome datasets. The table provides a summary of the datasets used in specific analyses, including the filters applied to derive the proteins and phosphosites/phosphoproteins\",\n",
            "          \"applied to derive the proteins and phosphosites/phosphoproteins that constitute each dataset; the protein, phosphosite or phosphoprotein count; and the methods that employ the respective datasets. c, Distribution of sequence coverage of the identified\",\n",
            "          \"c, Distribution of sequence coverage of the identified proteins with tryptic peptides detected by MS/MS, whiskers show the 5\\u201395 percentiles. d and e, Robust and accurate proteome/phosphoproteome platform. Longitudinal performance was tested by repeated\",\n",
            "          \"platform. Longitudinal performance was tested by repeated proteome and phosphoproteome analysis of patient-derived xenograft tumors. Scatterplots, histograms and Pearson correlations comparing individual replicate measurements are shown.\",\n",
            "          \"Tumor sample quality control (I)\",\n",
            "          \"a, Remark diagram showing sample processing and partitioning. Initial quality review encompassed histopathological examination of H&E stained tissue slices. *For 3 samples no tumor cells were seen on histopathology (BH-A0E9, BH-A0C1, A2-A0SW). These\",\n",
            "          \"were seen on histopathology (BH-A0E9, BH-A0C1, A2-A0SW). These samples were nevertheless included in the proteome analysis since other quality control standards were met (see below) and samples with 0% tumor cellularity on top or bottom sections were\",\n",
            "          \"with 0% tumor cellularity on top or bottom sections were included in TCGA analyses. b, correlation of TCGA (top or bottom sections) and CPTAC histological assessment of neoplastic cellularity for samples (n = 105). The average and range of neoplastic\",\n",
            "          \"for samples (n = 105). The average and range of neoplastic cellularities were identical for CPTAC and TCGA histological assessments. Averages (standard deviations) for neoplastic cellularity were 76% (+/\\u2212 17) for CPTAC, 76% (+/\\u2212 15) for TCGA_Top, and 75%\",\n",
            "          \"were 76% (+/\\u2212 17) for CPTAC, 76% (+/\\u2212 15) for TCGA_Top, and 75% (+/\\u2212 18) for TCGA_Bottom histopathology slides (Supplementary Table 2). Note that in three CPTAC cases where no tumor cells were identified by histopathological assessment, numbers of\",\n",
            "          \"were identified by histopathological assessment, numbers of protein-level somatic variants were similar to all other tumors. The identified mutated proteins were TP53_R273C, NOP58_Q23E, TAGLN2_G154R, TUBA1B_D116H, and MRPL48_I173K (Supplementary Table 5),\",\n",
            "          \"TUBA1B_D116H, and MRPL48_I173K (Supplementary Table 5), indicating presence of tumor cells in these samples. c, Proteome iTRAQ tumor/internal reference ratio heatmap for all CPTAC samples (8,028 proteins without missing values) including passed and failed\",\n",
            "          \"proteins without missing values) including passed and failed proteomic quality control (QC) samples. d, Global tumor/reference proteome ratio distributions for samples that passed and failed proteomic quality control analysis. e, Degradation-related gene\",\n",
            "          \"proteomic quality control analysis. e, Degradation-related gene sets were enriched in tumors that failed proteomic quality control analysis. f, Variant allele frequency (VAF) analysis of re-sequenced CPTAC tumors and comparison to original TCGA data.\",\n",
            "          \"re-sequenced CPTAC tumors and comparison to original TCGA data. Overall VAFs for failed QC samples were lower compared to passed samples suggesting lower purity.\",\n",
            "          \"Tumor sample quality control (II)\",\n",
            "          \"a, There was high concordance (94.6%) between DNA variants reported by TCGA and CPTAC re-sequenced tumors. Most point mutations reported by TCGA could be identified across the 8 re-sequenced samples used in the study. b, A high overall correlation\",\n",
            "          \"samples used in the study. b, A high overall correlation (mean=0.77) was observed for the CPTAC Variant Allele Fraction (VAF) (X-axis) and TCGA VAF (Y-axis) across the 8 samples used in the study. c, Agglomerative hierarchical clustering (Supplementary\",\n",
            "          \"study. c, Agglomerative hierarchical clustering (Supplementary Methods Section 3.8) used to co-cluster protein and RNA tumor expression data after filtering to retain 4,291 proteins and genes with moderate to high protein-RNA correlation (Pearson\",\n",
            "          \"genes with moderate to high protein-RNA correlation (Pearson correlation > 0.4) with results displayed as a circular dendrogram (fanplot). The proteome (.P) and RNA (.R) components of each sample are labeled using the same color. The outer ring shows\",\n",
            "          \"sample are labeled using the same color. The outer ring shows proteome samples in light grey and RNA samples in dark grey. High concordance between RNA and protein expression is evident from the color adjacency in the inner ring and alternating color in\",\n",
            "          \"the color adjacency in the inner ring and alternating color in the outer ring showing that RNA and protein components co-cluster for a large proportion of samples (62/80). d, Co-clustering of MS and RPPA tumor data. 126 RPPA readouts were mapped to gene\",\n",
            "          \"MS and RPPA tumor data. 126 RPPA readouts were mapped to gene names. These genes were intersected with the genes observed in the MS proteome, filtered to 48 proteins with moderate or higher RPPA-MS protein correlation, and analyzed for co-clustering as in\",\n",
            "          \"protein correlation, and analyzed for co-clustering as in c. 47 of 80 RPPA-MS protein pairs co-cluster. While this is a smaller proportion than for RNA-protein analysis, the number of genes used in the clustering is significantly smaller for RPPA (48 vs.\",\n",
            "          \"in the clustering is significantly smaller for RPPA (48 vs. 4,291 for RNA). e, ESTIMATE tumor purity comparison between mRNA, RNAseq, and proteome data. ANOVA is used to assess the difference in distribution (\\u2212log10(p-value)) of ESTIMATE, stromal, immune,\",\n",
            "          \"in distribution (\\u2212log10(p-value)) of ESTIMATE, stromal, immune, and tumor purity scores across mRNA (microarray), RNA-seq and proteome data. The only significant p-value (=0.02) is for the Cluster 3 stromal score, and higher stromal scores for the\",\n",
            "          \"the Cluster 3 stromal score, and higher stromal scores for the proteome drive that difference. f, Ischemia score analysis. Comparison of ischemia scores of 77 CPTAC tumors, 3 normal samples, and patient-derived xenografts. CPTAC tumors had generally lower\",\n",
            "          \"patient-derived xenografts. CPTAC tumors had generally lower ischemia scores than PDX samples subjected to 30 minutes of cold ischemia. Median ischemia scores are less than 30 minutes for each subtype and no significant differences were observed across\",\n",
            "          \"subtype and no significant differences were observed across subtypes. Effects due to cold ischemia therefore appear to be negligible in this CPTAC sample collection.\",\n",
            "          \"Protein-to-Protein, -CNA, and -mRNA correlation analyses\",\n",
            "          \"a, Identification of UBE3A as an E3 ubiquitin ligase that negatively correlates to p53 on the protein level. Pearson correlation and Benjamini-Hochberg corrected p-value are shown. b, Analysis of counter-regulated genes with negative correlation of\",\n",
            "          \"of counter-regulated genes with negative correlation of CNA-to-RNA as well as CNA-to-protein levels. Negative Pearson correlations are shown with Benjamini-Hochberg corrected p-values for CNA-to-protein correlations. Depicted genes have significant\",\n",
            "          \"CNA-to-protein correlations. Depicted genes have significant negative correlations at FDR<0.05 in the CNA-to-RNA and CNA-to-protein analyses. c, Global mRNA-to-protein correlation and gene set enrichment analysis.\",\n",
            "          \"Global CNA effects and comparison of CNA TRANS effects to knockdown signatures in the LINCS database\",\n",
            "          \"a, CNA landscape in the CPTAC tumor collection. The segment-based CNAs of 77 samples were downloaded from TCGA Firehose, including 18 Basal, 12 Her2, 23 Luminal A and 24 Luminal B subtypes. Copy number amplifications were marked in red and deletions in\",\n",
            "          \"Copy number amplifications were marked in red and deletions in blue. The bottom color key represents the log2 transformed copy number value, with CNA=2 centered at 0. Specific CNA events are seen for chromosome 5q and 10p regions in basal-like tumors. b,\",\n",
            "          \"seen for chromosome 5q and 10p regions in basal-like tumors. b, Correlations of copy number alterations (x-axis) to phosphoprotein levels (y-axis) highlight new CNA cis and trans effects. Significant (FDR<0.05) positive (red) and negative (green)\",\n",
            "          \"Significant (FDR<0.05) positive (red) and negative (green) correlations between CNA and phosphoproteins are indicated. Histograms show the fraction [%] of significant CNA trans effects for each CNA gene. c, LINCS CMap analysis facilitates identification\",\n",
            "          \"CNA gene. c, LINCS CMap analysis facilitates identification of novel functional candidates for CNA trans effects. Knockdown profiles were compared with CNA/protein trans effects for 502 genes. Genes with a connectivity score >|90| were considered\",\n",
            "          \"genes. Genes with a connectivity score >|90| were considered connected and significant cis effects were annotated at an FDR<0.05. d, Basal-like tumor-specific CNAs are candidate regulatory events for EGFR and SRC expression levels. Oncogenic kinases with\",\n",
            "          \"for EGFR and SRC expression levels. Oncogenic kinases with significant CNA/protein trans effects (left panel), that were regulated in LINCS shRNA experiments (right panel; 4 cell lines,) and directly measured as LINCS landmark genes, are shown alongside\",\n",
            "          \"directly measured as LINCS landmark genes, are shown alongside candidate regulatory genes CETN3 and SKP1. Clinical ER, PR, and HER2 annotation and PAM50 classification are shown in the header rows of each column.\",\n",
            "          \"Proteome cluster heatmap and stability analysis\",\n",
            "          \"a, K-means consensus clustering of proteome and phosphoproteome data identifies three subgroups: basal-enriched, luminal-enriched, and stromal-enriched. The heatmap represents all 1,521 proteins used for clustering (Dataset G8). b, Identification of\",\n",
            "          \"proteins used for clustering (Dataset G8). b, Identification of optimal proteome clusters for QC-passed CPTAC breast cancer tumors. Proteome clusters were derived using consensus clustering based on 1000 resampled datasets, exploring the range of 2 to 6\",\n",
            "          \"based on 1000 resampled datasets, exploring the range of 2 to 6 k-means clusters. Visualization of consensus matrices from k-means consensus clustering for k=3, 4, 5 and 6 target clusters. Consensus clustering was performed on 1,521 proteins with no\",\n",
            "          \"Consensus clustering was performed on 1,521 proteins with no missing values and SD>1.5. c, Silhouette plots were generated to evaluate the coherence of the clustering. Silhouette plots for k=3 and k=4 clusters showing a cleaner separation of clusters for\",\n",
            "          \"and k=4 clusters showing a cleaner separation of clusters for k=3. d, Based on both visual inspection of the consensus matrix and the delta plot assessing change in consensus cumulative distribution function (CDF) area, three robustly segregated groups\",\n",
            "          \"function (CDF) area, three robustly segregated groups were observed. Consensus cumulative distribution function (CDF) and delta area (change in CDF area) plots for 2\\u20136 clusters.\",\n",
            "          \"Proteome cluster markers and enriched pathways\",\n",
            "          \"a, Markers (based on SAM analysis; FDR<0.01) discriminate between proteome clusters 1, 2 and 3 (compare to heatmap of proteins used to derive clusters depicted in Extended Data Fig. 6a). b, Applying a Fisher exact test-based enrichment analysis to the\",\n",
            "          \"Applying a Fisher exact test-based enrichment analysis to the proteome, phosphoproteome and mRNA data, gene sets from MSigDB were identified that were unique for each proteome cluster. Heat map showing specific pathways comprising dominant biological\",\n",
            "          \"map showing specific pathways comprising dominant biological themes that are significantly differential by enrichment analysis between basal-enriched and luminal-enriched tumors (Fisher Exact Test Benjamini-Hochberg corrected p-values are shown;\",\n",
            "          \"Exact Test Benjamini-Hochberg corrected p-values are shown; enrichment test performed on marker sets identified using SAM analysis; see Methods; compare to Figure 3c). c, Heatmap showing a selection of gene sets significant in basal-enriched or\",\n",
            "          \"a selection of gene sets significant in basal-enriched or luminal-enriched tumors exclusively by mRNA, protein or phosphoprotein expression. Cytokine signatures, for example, were strongly captured at the mRNA level, but were seen to only a limited degree\",\n",
            "          \"at the mRNA level, but were seen to only a limited degree at the global protein level, likely because of their typically low protein abundance. By contrast, the vast majority of significant gene sets annotated as \\\"signaling\\\" were enriched only at the\",\n",
            "          \"gene sets annotated as \\\"signaling\\\" were enriched only at the phosphoprotein level. d, Global heat map representing all gene sets significantly enriched in at least one of the proteomic breast cancer subtypes. The stromal-enriched group was characterized\",\n",
            "          \"cancer subtypes. The stromal-enriched group was characterized by breast cancer normal-like, adipocyte differentiation, smooth muscle, toll-like receptor signaling and endothelin gene sets, supporting the clustering-based annotation of high stromal and/or\",\n",
            "          \"the clustering-based annotation of high stromal and/or adipose content in these tumors (see Supplemental Table 13).\",\n",
            "          \"Phosphoproteome pathway clustering, kinase-phosphosite multivariate regression, and protein co-expression networks\",\n",
            "          \"a, Phosphoproteome pathway clustering. Using phosphorylation state as a proxy for activity, deep phosphoproteome profiling allows development of a breast cancer molecular taxonomy based on signaling pathways. K-means consensus clustering was performed on\",\n",
            "          \"pathways. K-means consensus clustering was performed on pathways derived from single sample GSEA analysis of phosphopeptide data (908 pathways shown). Of four robustly segregated groups, subgroups 2 and 3 substantially recapitulated the stromal- and\",\n",
            "          \"subgroups 2 and 3 substantially recapitulated the stromal- and luminal-enriched proteomic subgroups, respectively. Subgroup 4 included a significant majority of tumors from the basal-enriched proteomic subgroup, but was admixed particularly with\",\n",
            "          \"proteomic subgroup, but was admixed particularly with luminal-enriched samples. This subgroup was defined by high levels of cell cycle and checkpoint activity. All basal and a majority of non-basal samples in this subgroup had TP53 mutations. Subgroup 1\",\n",
            "          \"samples in this subgroup had TP53 mutations. Subgroup 1 was a novel subgroup defined exclusively in the phosphopeptide / pathway activity domain, with no enrichment for either proteomic or PAM50 subtypes. It was defined by G-protein, G-protein coupled\",\n",
            "          \"PAM50 subtypes. It was defined by G-protein, G-protein coupled receptor, and inositol phosphate metabolism signatures, as well as ionotropic glutamate signaling. b, Analysis of the regulatory relationship between outlier kinases (see Supplementary Table\",\n",
            "          \"relationship between outlier kinases (see Supplementary Table 19) and phosphopeptides by regulatory multivariate regression analysis (see Methods) identified CDK1 as the most highly connected of the outlier Cyclin-Dependent Kinases, with highest\",\n",
            "          \"connected of the outlier Cyclin-Dependent Kinases, with highest centrality (based on node-degree; see Methods) among the outlier CDKs and seventh highest centrality among all the outlier kinases considered in the remMap analysis. Each line represents a\",\n",
            "          \"considered in the remMap analysis. Each line represents a phosphosite-kinase relationship. c\\u2013f, Analysis of differences in the co-expression patterns among genes/proteins across different subgroups. A Joint Random Forest (JRF) method was applied to\",\n",
            "          \"subgroups. A Joint Random Forest (JRF) method was applied to simultaneously build gene co-expression and protein co-expression networks (Supplementary Table 17, and Methods). Modules in these networks revealed different interaction patterns between\",\n",
            "          \"these networks revealed different interaction patterns between basal-enriched and luminal-enriched subgroups. c, Network module P1 of the protein co-expression network, defined chiefly in the proteome space. This module contained 12 genes connected by 39\",\n",
            "          \"proteome space. This module contained 12 genes connected by 39 edges, among which 34 were protein-specific and 5 were shared by both the protein and mRNA co-expression networks. Many edges were supported by published information and were contained in the\",\n",
            "          \"supported by published information and were contained in the STRING database. Edges in red are specific to the protein co-expression network; edges in green are shared by both protein and gene co-expression networks; edges indicated by double lines are\",\n",
            "          \"co-expression networks; edges indicated by double lines are contained in the STRING database with confidence score greater than 0.15. MMP9, one of the central proteins in this module, contributes to metastatic progression and is a potential target for\",\n",
            "          \"to metastatic progression and is a potential target for anti-metastatic therapies for basal-like / triple negative breast cancer. d, Heatmaps of the absolute correlation across each pair of genes in module P1 (shown in Panel c), based on either protein or\",\n",
            "          \"in module P1 (shown in Panel c), based on either protein or gene expression data for samples in the basal-enriched and luminal-enriched subgroups, respectively. The MMP9 protein was strongly co-expressed with the other members of the module only in the\",\n",
            "          \"co-expressed with the other members of the module only in the basal-enriched subgroup. Notably, this observation is dependent on protein data; the correlation at the mRNA level for this module was consistently low in both the basal-enriched and luminal\",\n",
            "          \"was consistently low in both the basal-enriched and luminal enriched subgroups indicating that these events coherently occur at the proteomic level. e, Co-expression network based on proteomics data. The network contains 693 proteomic network-specific\",\n",
            "          \"data. The network contains 693 proteomic network-specific edges (grey) and 792 edges shared with the RNAseq network (green). For each module, the most enriched category and corresponding Benjamini-Hochberg adjusted p-value is reported. Pie charts adjacent\",\n",
            "          \"adjusted p-value is reported. Pie charts adjacent to each module show the proportion of proteomics-specific edges (grey area) and edges shared between proteomics and RNAseq data (green area). f, RNAseq network.\",\n",
            "          \"Phosphoproteome signatures of PIK3CA (a,b) and TP53 (c,d) mutated tumors highlight activated key regulators and indicate frequency of activation\",\n",
            "          \"a and c, Phosphosites upregulated in mutated tumors (SAM FDR<0.05 across all tumors and independently also across luminal tumors; average phosphosite signal for all markers shown as bar graph). To avoid confounding by intrinsic subtype-specific\",\n",
            "          \"bar graph). To avoid confounding by intrinsic subtype-specific distinctions, only markers that were significantly identified both in analyses covering all tumors and analyses restricted to luminal tumors were selected (FDR <0.05). Color bars in the\",\n",
            "          \"to luminal tumors were selected (FDR <0.05). Color bars in the margins indicate FDRs for grouped analysis of different mutation classes and indicate kinase substrates of known kinases in the respective pathways. Significantly regulated kinase phosphosites\",\n",
            "          \"pathways. Significantly regulated kinase phosphosites are annotated. The average phosphorylation signal of the marker phosphosites provides a read-out for PI3K and TP53 pathway activity in mutated tumors (histogram below heatmap). A 95% prediction\",\n",
            "          \"in mutated tumors (histogram below heatmap). A 95% prediction confidence interval (indicated by dashed lines) across the average signal in non-mutated tumors was chosen in order to discriminate active from non-active tumors. The most strongly activated\",\n",
            "          \"active from non-active tumors. The most strongly activated PIK3CA kinase domain mutant tumor differed from the other 9 kinase domain mutant tumors, as it contained an amino acid side chain charge neutral H1047L instead of the more common positively\",\n",
            "          \"charge neutral H1047L instead of the more common positively charged H1047R mutation. Among the 62 phosphosites identified that were significantly upregulated in PIK3CA mutated tumors, 13 phosphosites were found on phosphoproteins that are known substrates\",\n",
            "          \"were found on phosphoproteins that are known substrates of well-annotated kinases in the PIK3CA pathway (panel a, right column). In the mutant TP53 analysis a total 20 phosphosites were found on phosphoproteins that are known substrates of well annotated\",\n",
            "          \"on phosphoproteins that are known substrates of well annotated kinases in the p53 pathway (panel c, right column). b and d, Upregulated phosphosite sets were derived from isogenic PIK3CA and TP53 mutant versus wild-type cell line pairs and tested for\",\n",
            "          \"and TP53 mutant versus wild-type cell line pairs and tested for enrichment within mutant versus wild-type CPTAC tumors using single sample GSEA. Significantly enriched phosphosite sets are shown (p<0.05).\",\n",
            "          \"PIRCOS plots, kinase outliers and outliers in the ERBB2 pathway\",\n",
            "          \"a, Pircos (Proteogenomics CIRCOS) plots for 8q and 17q showing median CNA, RNA, protein, and phosphosite expression for 20 tumors with amplification in 8q based on RIPK2 CNA>1; 23 tumors with amplification in 8q based on PTK2 CNA>1; 15 tumors with\",\n",
            "          \"with amplification in 8q based on PTK2 CNA>1; 15 tumors with amplification in 17q based on CDK12 CNA >1; and 10 tumors with amplification in 17q based on TLK2 CNA>1. Red indicates expression >1, blue < \\u22121, and grey between \\u22121 and 1. Genes with both copy\",\n",
            "          \">1, blue < \\u22121, and grey between \\u22121 and 1. Genes with both copy number amplification (CNA>1) and increased phosphosite expression (p-site>1) are labeled. b, Phosphosite outliers in known ERBB2 signaling genes. To better understand the downstream effects of\",\n",
            "          \"signaling genes. To better understand the downstream effects of ERBB2 amplification, phosphosite outliers in known ERBB2 signaling genes (MSigDB pathway set, KEGG_ERBB_SIGNALING PATHWAY) were identified for the 15 samples that had ERBB2 phosphosite\",\n",
            "          \"were identified for the 15 samples that had ERBB2 phosphosite outlier status. Forty-one genes were identified as having a phosphosite outlier in at least one of the ERBB2 amplified samples. PAK4 and ARAF phosphosite outlier status were found in seven of\",\n",
            "          \"PAK4 and ARAF phosphosite outlier status were found in seven of the 15 ERBB2 kinase outlier samples; GSK3B outliers were found in 6 samples; and EIF4EBP1, MAP2K2, ABL1 and AKT1 outlier status was found in 5 of the 15 samples. c, Proteogenomic outlier\",\n",
            "          \"was found in 5 of the 15 samples. c, Proteogenomic outlier expression analysis for TLK2 and RIPK2. Samples with outlier phosphosite (red), protein (yellow), RNA (green) and copy number (purple) expression are shown. Phosphosite squares indicate per-sample\",\n",
            "          \"expression are shown. Phosphosite squares indicate per-sample outlier phosphosites.\",\n",
            "          \"Supplementary Material\\n\\nSupplementary Information is available in the online version of the paper.\",\n",
            "          \"Author Contributions P.M., D.R.M., M.A.G., K.R.C., and S.A.C. designed the proteomic analysis experiments, data analysis workflow, and proteomic-genomic data comparisons. P.M., M.A.G., J.W.Q., and S.A.C. directed and performed proteomic analysis of breast\",\n",
            "          \"and S.A.C. directed and performed proteomic analysis of breast tumor and quality control samples. P.M., D.R.M., K.V.R., K.R.C., P.W., X.W., S.C., E.K., F.P., Z.T., J.L., M.L.G., M.W., V.Y., K.H., C.L., M.D.M., P.Y., J.W., B.Z., and D.F. performed\",\n",
            "          \"V.Y., K.H., C.L., M.D.M., P.Y., J.W., B.Z., and D.F. performed proteomic-genomic data analyses. D.R.M., P.W., and S.J.S. provided statistical support. D.R.M., K.V.R., K.R.C., K.K. and D.F. performed analyses of mass spectrometry data and adapted\",\n",
            "          \"D.F. performed analyses of mass spectrometry data and adapted algorithms and software for data analysis. S.R.D., R.R.T and M.J.E. developed and prepared breast xenografts used as quality control samples. P.M. and F.M prepared and analyzed cell lines for\",\n",
            "          \"samples. P.M. and F.M prepared and analyzed cell lines for correlative functional annotation of frequently mutated genes. P.M., D.R.M., M.A.G., and S.A.C designed strategy for quality control analyses. M.A.G., S.R.D., C.R.K., M.M., and H.R. coordinated\",\n",
            "          \"analyses. M.A.G., S.R.D., C.R.K., M.M., and H.R. coordinated acquisition, distribution and quality control evaluation of TCGA tumor samples. P.M., M.A.G., C.P., L.D., A.G.P., and M.J.E. interpreted data in the context of breast cancer biology. P.M.,\",\n",
            "          \"interpreted data in the context of breast cancer biology. P.M., D.R.M, M.A.G., K.R.C., P.W., A.G.P, M.J.E. and S.A.C. wrote the manuscript.\",\n",
            "          \"All primary mass spectrometry data are deposited at the CPTAC Data Portal as raw and mzML files and complete protein assembly data sets for public access (https://cptac-data-portal.georgetown.edu/cptac/s/S029). In addition, a set of ancillary files such\",\n",
            "          \"In addition, a set of ancillary files such as dataset G1/P1, G3/P3, G4/P4, G5/P5, G7/P7, CNA correlation tables for CNA-mRNA, CNA-proteome and CNA-phosphoproteome; CNA data, RNA-seq expression data have also been deposited at the DCC. Two browsers have\",\n",
            "          \"data have also been deposited at the DCC. Two browsers have been created to assist the interested reader in exploring the results. One provides track hubs for viewing the identified peptides in the UCSD genome browser\",\n",
            "          \"for viewing the identified peptides in the UCSD genome browser (http://fenyolab.org/cptac_breast_ucsc). The second is an on-line tool for proteogenomic data exploration and can be accessed at http://prot-shiny-vm.broadinstitute.org:3838/BC2016/. See\",\n",
            "          \"at http://prot-shiny-vm.broadinstitute.org:3838/BC2016/. See Supplemental Methods for descriptions.\",\n",
            "          \"The authors declare no competing financial interests.\\n\\nProteogenomic analysis of human breast cancer. Direct effects of genomic alterations on protein level\",\n",
            "          \"Overlap of a, protein coding single amino acid variants (SAAVs) and b, RNA splice junctions not present in RefSeq v60 detected by DNA exome sequencing, RNA-seq, and LC-MS/MS. Proportions of novel variants are noted. c, Heatmap of mutations/CNA and their\",\n",
            "          \"novel variants are noted. c, Heatmap of mutations/CNA and their effects on RNA and protein expression of breast cancer-relevant genes across tumor and normal samples. ER, PR, HER2 and PAM50 status are annotated. Median iTRAQ protein abundance ratio and\",\n",
            "          \"status are annotated. Median iTRAQ protein abundance ratio and the most frequently detected and differential phosphosite ratio are shown for each gene. Pearson correlations between MS protein vs RNA-seq and MS protein vs RPPA are indicated.\",\n",
            "          \"Effects of copy number alterations (CNA) on mRNA, protein, and phosphoprotein abundance\",\n",
            "          \"a, Correlations of CNA (x-axes) to RNA and protein expression levels (y-axes) highlight new CNA cis and trans effects. Significant (FDR<0.05) positive (red) and negative (green) correlations between CNA and mRNAs or proteins are indicated. CNA cis effects\",\n",
            "          \"CNA and mRNAs or proteins are indicated. CNA cis effects appear as a red diagonal line, CNA trans effects as vertical stripes. Histograms show the fraction [%] of significant CNA trans effects for each CNA gene. b, Overlap of cis effects observed at RNA,\",\n",
            "          \"for each CNA gene. b, Overlap of cis effects observed at RNA, protein, and phosphoprotein levels (FDR<0.05). c, Trans-effect regulatory candidates identified among those with significant protein cis-effects using LINCS CMap. Bars indicate total numbers of\",\n",
            "          \"cis-effects using LINCS CMap. Bars indicate total numbers of significant CNA/protein trans effects (gray; FDR<0.05) and overlap with regulated genes in LINCS knock-down profiles (red; 4 cell lines; moderated T-test FDR<0.1).\",\n",
            "          \"Proteomic and phosphoproteomic subtypes of breast cancer and subtype-specific pathway enrichment\",\n",
            "          \"a, Unsupervised clustering of RNA-seq and proteomics data restricted to PAM50 genes and subset of 35 detected proteins reveal high similarity to PAM50 (TCGA) sample annotation. b, K-means consensus clustering of proteome and phosphoproteome data\",\n",
            "          \"consensus clustering of proteome and phosphoproteome data identifies basal-enriched, luminal-enriched, and stromal-enriched subgroups. c, Gene set enrichment analysis highlights sets of pathways significantly differential between basal-enriched and\",\n",
            "          \"pathways significantly differential between basal-enriched and luminal-enriched tumors (detailed in Extended Data Fig. 7b). d, K-means consensus clustering performed on pathways derived from single sample GSEA analysis of phosphopeptide data identifies\",\n",
            "          \"single sample GSEA analysis of phosphopeptide data identifies four distinct clusters.\",\n",
            "          \"Example analyses of aberrantly regulated kinases in human breast cancer\",\n",
            "          \"a and b, PIRCOS (Proteogenomics CIRCOS) plots showing CNA, RNA, protein and phosphosite expression for 17 tumors with amplification in 17q (ERBB2 CNA>1) and 8 tumors with amplification in 11q (PAK1 CNA>1). Labeled genes have CNA>1 and phosphosite>1. c,\",\n",
            "          \"(PAK1 CNA>1). Labeled genes have CNA>1 and phosphosite>1. c, Proteogenomic outlier expression analysis for ERBB2, CDK12, and PAK1. Samples with outlier phosphosite (red), protein (yellow), RNA (green) and copy number (purple) expression are shown.\",\n",
            "          \"RNA (green) and copy number (purple) expression are shown. Phosphosite squares indicate per-sample outlier phosphosites. d, Outlier kinase events by PAM50 subtype (>35% of subtype samples contain a phosphosite outlier; <10% FDR using Benjamini-Hochberg\",\n",
            "          \"a phosphosite outlier; <10% FDR using Benjamini-Hochberg adjusted p-values).\"\n",
            "        ]\n",
            "      }\n",
            "    ]\n",
            "  },\n",
            "  \"PDC000173\": {\n",
            "    \"study_id\": \"b8da9eeb-57b8-11e8-b07a-00a098d917f8\",\n",
            "    \"pdc_study_id\": \"PDC000173\",\n",
            "    \"study_submitter_id\": \"TCGA BRCA Proteome S015-1\",\n",
            "    \"program_id\": \"10251935-5540-11e8-b664-00a098d917f8\",\n",
            "    \"project_id\": \"48af5040-5546-11e8-b664-00a098d917f8\",\n",
            "    \"study_name\": \"TCGA Breast Cancer Proteome\",\n",
            "    \"program_name\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"project_name\": \"CPTAC2 Retrospective\",\n",
            "    \"disease_type\": \"Breast Invasive Carcinoma;Other\",\n",
            "    \"primary_site\": \"Breast;Not Reported\",\n",
            "    \"analytical_fraction\": \"Proteome\",\n",
            "    \"experiment_type\": \"iTRAQ4\",\n",
            "    \"embargo_date\": null,\n",
            "    \"cases_count\": 109,\n",
            "    \"aliquots_count\": 109,\n",
            "    \"protocol_id\": \"1c06aab9-3b26-11e9-9a07-0a80fada099c\",\n",
            "    \"protocol_submitter_id\": \"P-S015-1\",\n",
            "    \"program_submitter_id\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"protocol_name\": \"TCGA_Breast_Cancer_Proteome\",\n",
            "    \"protocol_date\": \"2013-02-02\",\n",
            "    \"document_name\": \"\",\n",
            "    \"quantitation_strategy\": \"Isobaric label quantitation\",\n",
            "    \"label_free_quantitation\": \"\",\n",
            "    \"labeled_quantitation\": \"iTRAQ quantitation\",\n",
            "    \"isobaric_labeling_reagent\": \"iTRAQ\",\n",
            "    \"reporter_ion_ms_level\": \"2\",\n",
            "    \"starting_amount\": \"4.00\",\n",
            "    \"starting_amount_uom\": \"mg\",\n",
            "    \"digestion_reagent\": \"Lys-C/Trypsin\",\n",
            "    \"alkylation_reagent\": \"Iodoacetamide\",\n",
            "    \"enrichment_strategy\": \"Proteome\",\n",
            "    \"enrichment\": \"N/A\",\n",
            "    \"chromatography_dimensions_count\": \"2\",\n",
            "    \"one_d_chromatography_type\": \"bRPLC (pH 10)\",\n",
            "    \"two_d_chromatography_type\": \"RPLC\",\n",
            "    \"fractions_analyzed_count\": \"25\",\n",
            "    \"column_type\": \"New Objective; PicoFrit SELF/P\",\n",
            "    \"amount_on_column\": \"1\",\n",
            "    \"amount_on_column_uom\": \"\\u00b5g\",\n",
            "    \"column_length\": \"20\",\n",
            "    \"column_length_uom\": \"cm\",\n",
            "    \"column_inner_diameter\": \"75.00\",\n",
            "    \"column_inner_diameter_uom\": \"\\u00b5m\",\n",
            "    \"particle_size\": \"1.9\",\n",
            "    \"particle_size_uom\": \"\\u00b5m\",\n",
            "    \"particle_type\": \"Dr Maisch GmbH; ReproSil-Pur 120 C18-AQ; 1.9 um\",\n",
            "    \"gradient_length\": \"110\",\n",
            "    \"gradient_length_uom\": \"min\",\n",
            "    \"instrument_make\": \"Thermo Scientific\",\n",
            "    \"instrument_model\": \"Q Exactive\",\n",
            "    \"dissociation_type\": \"HCD\",\n",
            "    \"ms1_resolution\": \"70000\",\n",
            "    \"ms2_resolution\": \"17500\",\n",
            "    \"dda_topn\": \"Top12\",\n",
            "    \"normalized_collision_energy\": \"28%\",\n",
            "    \"acquistion_type\": \"DDA\",\n",
            "    \"dia_multiplexing\": \"N/A\",\n",
            "    \"dia_ims\": \"N/A\",\n",
            "    \"analytical_technique\": null,\n",
            "    \"chromatography_instrument_make\": null,\n",
            "    \"chromatography_instrument_model\": null,\n",
            "    \"polarity\": null,\n",
            "    \"reconstitution_solvent\": null,\n",
            "    \"reconstitution_volume\": null,\n",
            "    \"reconstitution_volume_uom\": null,\n",
            "    \"internal_standards\": null,\n",
            "    \"extraction_method\": null,\n",
            "    \"ionization_mode\": null,\n",
            "    \"publications\": [\n",
            "      {\n",
            "        \"publication_id\": \"44e2e766-89c7-11e8-bcf1-0a2705229b82\",\n",
            "        \"pubmed_id\": \"27251275\",\n",
            "        \"doi\": \"10.1038/nature18003\",\n",
            "        \"author\": \"Mertins, P.; Mani, D. R.; Ruggles, K. V.; Gillette, M. A.; Clauser, K. R.; Wang, P.; Wang, X.; Qiao, J. W.; Cao, S.; Petralia, F.; Kawaler, E.; Mundt, F.; Krug, K.; Tu, Z.; Lei, J. T.; Gatza, M. L.; Wilkerson, M.; Perou, C. M.; Yellapantula, V.; Huang, K. L.; Lin, C.; McLellan, M. D.; Yan, P.; Davies, S. R.; Townsend, R. R.; Skates, S. J.; Wang, J.; Zhang, B.; Kinsinger, C. R.; Mesri, M.; Rodriguez, H.; Ding, L.; Paulovich, A. G.; Feny\\u00f6, D.; Ellis, M. J.; Carr, S. A.; Gil Kuhn, E.; Mani, D. R.; Mertins, P.; Ketchum, K. A.; Thangudu, R. R.; Cai, S.; Oberti, M.; Whiteaker, J. R.; Wang, X.; Lin, C.; Ping, Y.; Edwards, N. J.; Madhavan, S.; McGarvey, P. B.; Wang, P.; Petralia, F.; Tu, Z.; Chan, D.; Pandey, A.; Shih, L. M.; Zhang, H.; Zhang, Z.; Thomas, S.; Zhu, H.; Whiteley, G. A.; Skates, S. J.; White, F. M.; Levine, D. A.; Boja, E. S.; Kinsinger, C. R.; Hiltke, T.; Mesri, M.; Rivers, R. C.; Rodriguez, H.; Shaw, K. M.; Stein, S. E.; Fenyo, D.; Liu, T.; McDermott, J. E.; Payne, S. H.; Rodland, K. D.; Smith, R. D.; Rudnick, P.; Snyder, M.; Zhao, Y.; Chen, X.; Ransohoff, D. F.; Hoofnagle, A. N.; Liebler, D. C.; Sanders, M. E.; Shi, Z.; Slebos, R. J.; Tabb, D. L.; Zhang, B.; Zimmerman, L. J.; Wang, Y.; Li, S.; Davies, S. R.; Ding, L.; Maher, C.; Townsend, R.; Ellis, M. J.; Lei, J. T.; Luo, J.\",\n",
            "        \"title\": \"Proteogenomics connects somatic mutations to signalling in breast cancer\",\n",
            "        \"journal\": \"Nature\",\n",
            "        \"journal_url\": \"https://www.nature.com/articles/nature18003\",\n",
            "        \"year\": \"2016\",\n",
            "        \"abstract\": \"Somatic mutations have been extensively characterized in breast cancer, but the effects of these genetic alterations on the proteomic landscape remain poorly understood. Here we describe quantitative mass-spectrometry-based proteomic and phosphoproteomic analyses of 105 genomically annotated breast cancers, of which 77 provided high-quality data. Integrated analyses provided insights into the somatic cancer genome including the consequences of chromosomal loss, such as the 5q deletion characteristic of basal-like breast cancer. Interrogation of the 5q trans-effects against the Library of Integrated Network-based Cellular Signatures, connected loss of CETN3 and SKP1 to elevated expression of epidermal growth factor receptor (EGFR), and SKP1 loss also to increased SRC tyrosine kinase. Global proteomic data confirmed a stromal-enriched group of proteins in addition to basal and luminal clusters, and pathway analysis of the phosphoproteome identified a G-protein-coupled receptor cluster that was not readily identified at the mRNA level. In addition to ERBB2, other amplicon-associated highly phosphorylated kinases were identified, including CDK12, PAK1, PTK2, RIPK2 and TLK2. We demonstrate that proteogenomic analysis of breast cancer elucidates the functional consequences of somatic mutations, narrows candidate nominations for driver genes within large deletions and amplified regions, and identifies therapeutic targets.\",\n",
            "        \"full_text\": [\n",
            "          \"Proteogenomics connects somatic mutations to signaling in breast cancer\\n\\nSummary\",\n",
            "          \"Somatic mutations have been extensively characterized in breast cancer, but the effects of these genetic alterations on the proteomic landscape remain poorly understood. We describe quantitative mass spectrometry-based proteomic and phosphoproteomic\",\n",
            "          \"mass spectrometry-based proteomic and phosphoproteomic analyses of 105 genomically annotated breast cancers of which 77 provided high-quality data. Integrated analyses allowed insights into the somatic cancer genome including the consequences of\",\n",
            "          \"into the somatic cancer genome including the consequences of chromosomal loss, such as the 5q deletion characteristic of basal-like breast cancer. The 5q trans effects were interrogated against the Library of Integrated Network-based Cellular Signatures,\",\n",
            "          \"the Library of Integrated Network-based Cellular Signatures, thereby connecting CETN3 and SKP1 loss to elevated expression of EGFR, and SKP1 loss also to increased SRC. Global proteomic data confirmed a stromal-enriched group in addition to basal and\",\n",
            "          \"confirmed a stromal-enriched group in addition to basal and luminal clusters and pathway analysis of the phosphoproteome identified a G Protein-coupled receptor cluster that was not readily identified at the mRNA level. Besides ERBB2, other\",\n",
            "          \"not readily identified at the mRNA level. Besides ERBB2, other amplicon-associated, highly phosphorylated kinases were identified, including CDK12, PAK1, PTK2, RIPK2 and TLK2. We demonstrate that proteogenomic analysis of breast cancer elucidates\",\n",
            "          \"that proteogenomic analysis of breast cancer elucidates functional consequences of somatic mutations, narrows candidate nominations for driver genes within large deletions and amplified regions, and identifies therapeutic targets.\",\n",
            "          \"Introduction\",\n",
            "          \"A central deficiency in our knowledge of cancer concerns how genomic changes drive the proteome and phosphoproteome to execute phenotypic characteristics. The initial proteomic characterization in the TCGA breast study was performed using reversed phase\",\n",
            "          \"in the TCGA breast study was performed using reversed phase protein arrays; however this approach is restricted by antibody availability. To provide greater analytical breadth, the NCI Clinical Proteomic Tumor Analysis Consortium (CPTAC) is analyzing the\",\n",
            "          \"Proteomic Tumor Analysis Consortium (CPTAC) is analyzing the proteomes of genome-annotated TCGA tumor specimens using mass spectrometry. Herein we describe integrated proteogenomic analyses of TCGA breast cancer samples representing the four principal\",\n",
            "          \"of TCGA breast cancer samples representing the four principal mRNA-defined breast cancer intrinsic subtypes.\",\n",
            "          \"Proteogenomic analysis of TCGA samples\",\n",
            "          \"105 breast tumors previously characterized by the TCGA were selected for proteomic analysis after histopathological documentation (Supplementary Tables 1 and 2). The cohort included a balanced representation of PAM50-defined intrinsic subtypes including\",\n",
            "          \"representation of PAM50-defined intrinsic subtypes including 25 basal-like, 29 luminal A, 33 luminal B, and 18 HER2 (ERBB2)-enriched tumors, along with 3 normal breast tissue samples. Samples were analyzed by high-resolution, accurate mass, tandem mass\",\n",
            "          \"were analyzed by high-resolution, accurate mass, tandem mass spectrometry (MS) that included extensive peptide fractionation and phosphopeptide enrichment (Extended Data Fig. 1a). An isobaric peptide labeling approach (iTRAQ) was employed to quantify\",\n",
            "          \"peptide labeling approach (iTRAQ) was employed to quantify protein and phosphosite levels across samples, with 37 iTRAQ 4-plexes analyzed in total. A total of 15,369 proteins (12,405 genes) and 62,679 phosphosites were confidently identified with 11,632\",\n",
            "          \"and 62,679 phosphosites were confidently identified with 11,632 proteins/tumor and 26,310 phosphosites/tumor on average (Supplementary Tables 3, 4 and Supplementary Methods). After filtering for observation in at least a quarter of the samples\",\n",
            "          \"filtering for observation in at least a quarter of the samples (Supplementary Methods, Extended Data Fig. 1b) 12,553 proteins (10,062 genes) and 33,239 phosphosites, with their relative abundances quantified across tumors, were used in subsequent analyses\",\n",
            "          \"quantified across tumors, were used in subsequent analyses in this study. Stable longitudinal performance and low technical noise were demonstrated by repeated, interspersed analyses of a single batch of patient-derived luminal and basal breast cancer\",\n",
            "          \"single batch of patient-derived luminal and basal breast cancer xenograft samples (Extended Data Fig. 1d,e). Due to the heterogeneous nature of breast tumors, and because proteomic analyses were performed on tumor fragments that were different from those\",\n",
            "          \"performed on tumor fragments that were different from those used in the genomic analyses, rigorous pre-specified sample and data QC metrics were implemented (Supplementary Discussion and Extended Data Figures 2, 3). Extensive analyses concluded that 28 of\",\n",
            "          \"Data Figures 2, 3). Extensive analyses concluded that 28 of the 105 samples were compromised by protein degradation. These samples were excluded from further analysis with subsequent informatics focused on the 77 tumor samples and three biological\",\n",
            "          \"focused on the 77 tumor samples and three biological replicates.\",\n",
            "          \"Genome and transcriptomic variation was observed at the peptide level by searching MS/MS spectra not matched to RefSeq against a patient-specific sequence database (Fig. 1a). The database was constructed using the QUILTS software package leveraging RefSeq\",\n",
            "          \"constructed using the QUILTS software package leveraging RefSeq gene models based on whole exome and RNA-seq data generated from portions of the same tumors and matched germline DNA (Fig. 1a, Supplementary Table 5). While these analyses detected a number\",\n",
            "          \"Supplementary Table 5). While these analyses detected a number of single amino-acid variants (SAAVs), frameshifts, and splice junctions, including splice isoforms that had been detected as only single transcript reads by RNA-seq (Fig. 1b, Supplementary\",\n",
            "          \"only single transcript reads by RNA-seq (Fig. 1b, Supplementary Table 5), the number of genomic and transcriptomic variants that were confirmed as peptides by MS was low (Supplementary Discussion). Sparse detection of individual genomic variants by\",\n",
            "          \"Discussion). Sparse detection of individual genomic variants by peptide sequencing has been noted in our previous studies and reflects limited coverage at the single amino-acid level with current technology. However quantitative MS analysis of multiple\",\n",
            "          \"technology. However quantitative MS analysis of multiple peptides for each protein is used to reliably infer overall protein levels. This is an advantage for MS since antibody-based protein expression analysis is typically based on a single epitope. To\",\n",
            "          \"expression analysis is typically based on a single epitope. To illustrate this capability in the current data set an initial analysis of three frequently mutated genes in breast cancer (TP53, PIK3CA, and GATA3) and three clinical biomarkers (ER, PGR, and\",\n",
            "          \"PIK3CA, and GATA3) and three clinical biomarkers (ER, PGR, and ERBB2) was conducted (Fig. 1c, Supplementary Table 6, 7 and Supplementary Discussion). As expected, TP53 missense mutations were associated with elevated MS-based protein levels, as observed\",\n",
            "          \"associated with elevated MS-based protein levels, as observed by RPPA (Reverse Phase Protein Array), especially in basal-like breast cancer. TP53 nonsense and frame-shift mutations were associated with a decrease in TP53 protein levels that was\",\n",
            "          \"were associated with a decrease in TP53 protein levels that was particularly striking in the MS data. In contrast, the mostly C-terminal GATA3 frame-shift alterations did not result in decreased protein expression when measured by the median of all GATA3\",\n",
            "          \"protein expression when measured by the median of all GATA3 peptides, suggesting these proteins are expressed despite truncation. No consistent effect of somatic PIK3CA mutation was observed at the level of protein expression. Good correlations between\",\n",
            "          \"at the level of protein expression. Good correlations between RNA-seq and MS-protein expression levels were found for ESR1 (r=0.74), PGR (r=0.74), ERBB2 (r=0.84) and GATA3 (r=0.83) with moderate correlations observed for PIK3CA (r=0.45) and TP53 (r=0.36).\",\n",
            "          \"correlations observed for PIK3CA (r=0.45) and TP53 (r=0.36). Lower TP53 protein abundance levels compared to mRNA levels were especially prevalent in luminal tumors, suggesting post-transcriptional regulatory mechanisms such as proteasomal degradation. To\",\n",
            "          \"regulatory mechanisms such as proteasomal degradation. To explore this hypothesis a search was made for E3 ligases that showed negative correlation to p53 protein (Supplementary Table 8). These analyses identified UBE3A (r=\\u22120.42; adj. pval=0.05) (Extended\",\n",
            "          \"analyses identified UBE3A (r=\\u22120.42; adj. pval=0.05) (Extended Data Fig. 4a), an established TP53 E3 ligase. In comparing CNA, RNA, and protein levels for GATA3, copy number gains in chromosome 10q were anti-correlated with RNA and protein levels in\",\n",
            "          \"10q were anti-correlated with RNA and protein levels in basal-like tumors. This observation prompted a search for other gains or losses that were anticorrelated with RNA and/or protein levels (see Extended Data Fig. 4b for further analyses). Overall, six\",\n",
            "          \"(see Extended Data Fig. 4b for further analyses). Overall, six genes were identified that significantly anti-correlated at an FDR<0.05 on both RNA and protein level to their CNA signals (Extended Data Fig. 4b). GATA3 amplification on 10q in basal-like\",\n",
            "          \"Data Fig. 4b). GATA3 amplification on 10q in basal-like breast cancer showed the strongest anti-correlation, followed by the hexosamine and glycolysis pathway enzymes GFPT2 and HK3, which are upregulated in basal-like breast cancer despite being subjected\",\n",
            "          \"upregulated in basal-like breast cancer despite being subjected to frequent chromosomal deletion on 5q. Global analysis of the correlation of mRNA-to-protein yielded a median Pearson value of r=0.39, with 6,135 out of 9,302 protein/mRNA pairs (66.0%)\",\n",
            "          \"of r=0.39, with 6,135 out of 9,302 protein/mRNA pairs (66.0%) correlating significantly at an FDR<0.05 (Extended Data Fig. 4c, Supplementary Table 9 and Supplementary Discussion). Similar to the colon cancer analysis metabolic functions such as amino\",\n",
            "          \"to the colon cancer analysis metabolic functions such as amino acid, sugar and fatty acid metabolism were found to be enriched among positively correlated genes whereas ribosomal, RNA polymerase and mRNA splicing functions were negatively correlated.\",\n",
            "          \"and mRNA splicing functions were negatively correlated. Overall these analyses demonstrate the utility of global proteome correlation analysis for both confirmation of suspected regulatory mechanisms and identification of candidate regulators meriting\",\n",
            "          \"mechanisms and identification of candidate regulators meriting further investigation.\",\n",
            "          \"Copy Number Alterations\",\n",
            "          \"To determine the consequences of CNA on mRNA, protein and phosphoprotein abundance, both in \\u201ccis\\u201d on genes within the aberrant locus and in \\u201ctrans\\u201d on genes encoded elsewhere, univariate correlation analysis was used as previously described. A total of\",\n",
            "          \"analysis was used as previously described. A total of 7,776 genes with CNA, mRNA and protein measurements were analyzed by calculating Pearson correlation and associated statistical significance (Benjamini-Hochberg corrected p-value) for all possible\",\n",
            "          \"(Benjamini-Hochberg corrected p-value) for all possible CNA-mRNA and CNA-protein pairs (Fig. 2a, Supplementary Table 10, Extended Data Fig. 5a, see Methods). For the phosphoproteome, 4,472 CNA-phosphoprotein pairs were analyzed (Extended Data Fig. 5b).\",\n",
            "          \"CNA-phosphoprotein pairs were analyzed (Extended Data Fig. 5b). Significant positive correlations (cis) were observed for 64% of all CNA-mRNA, 31% of all CNA-protein, and 20% of all CNA-phosphoprotein pairs Fig. 2b. Proteins and phosphoproteins correlated\",\n",
            "          \"pairs Fig. 2b. Proteins and phosphoproteins correlated in cis to CNA were, for the most part, a subset of the cis effects observed in mRNA-to-CNA correlation (Fig. 2b, Supplementary Table 10). The fractional difference of well-annotated oncogenes and\",\n",
            "          \"10). The fractional difference of well-annotated oncogenes and tumor suppressor genes among the significantly cis-correlated CNA-to-mRNA and -protein gene pairs was analyzed. Based on a reference list of 487 oncogenes and tumor suppressors (Supplementary\",\n",
            "          \"list of 487 oncogenes and tumor suppressors (Supplementary Table 10), these cancer relevant genes occur 37.6% more frequently in the subset of genes that correlate both on CNA/mRNA and CNA/protein levels than in the subset that only correlate on CNA/mRNA\",\n",
            "          \"levels than in the subset that only correlate on CNA/mRNA but not on CNA/protein level (Fisher exact p-value=0.02). This suggests that CNA events with a tumor promoting outcome more likely lead to cis regulatory effects on both the protein and mRNA level,\",\n",
            "          \"to cis regulatory effects on both the protein and mRNA level, whereas CNA events with no documented role in tumorigenesis are more likely to be neutralized on the protein level than on the RNA level. Trans effects (Fig. 2a) appear as vertical bands, with\",\n",
            "          \"level. Trans effects (Fig. 2a) appear as vertical bands, with accompanying frequency histograms (in blue) highlighting \\u201chot spots\\u201d of significant trans effects. Using a minimum threshold of 50 trans-affected genes, 68% of the tested genes were associated\",\n",
            "          \"trans-affected genes, 68% of the tested genes were associated with trans effects on the mRNA level, whereas only 13% were associated with effects on the protein level and 8% on the phosphoprotein level. Importantly, CNA-protein correlations appeared to be\",\n",
            "          \"level. Importantly, CNA-protein correlations appeared to be a reduced representation of CNA-mRNA correlations. Furthermore, for many CNA regions correlations were more directionally uniform on the protein level than on the mRNA level. CNA regions\",\n",
            "          \"on the protein level than on the mRNA level. CNA regions exhibiting the most trans associations at the protein level were found on chromosomes 5q (LOH in basal; gain in LumB), 10p (gain in basal), 12 (gain in basal), 16q (LumA deletion), 17q (LumB\",\n",
            "          \"in basal), 12 (gain in basal), 16q (LumA deletion), 17q (LumB amplification), and 22q (LOH in luminal and basal) (Extended Data Fig. 5a).\",\n",
            "          \"Trans associations are not necessarily direct consequences of the chromosomal aberration. For example since 5q loss occurs in at least 50% of basal-like breast cancers, many of the trans effects involve genes that mark the basal-subtype. To identify\",\n",
            "          \"effects involve genes that mark the basal-subtype. To identify candidate driver genes whose copy number alterations are direct drivers of trans effects, results were compared with functional knock-down data on 3,797 genes in the Library of Integrated\",\n",
            "          \"knock-down data on 3,797 genes in the Library of Integrated Cellular Signatures (LINCS) database (http://www.lincsproject.org/). For any given gene with copy number alterations (\\u201cCNA-gene\\u201d), sets of genes were identified corresponding to proteins that\",\n",
            "          \"sets of genes were identified corresponding to proteins that changed where there was gain (\\u201cCNA-gain trans gene set\\u201d) or loss (\\u201cCNA-loss trans gene set\\u201d). These gene sets were then compared to the effects of gene knock down in the LINCS database (see\",\n",
            "          \"to the effects of gene knock down in the LINCS database (see Methods). Queries for 502 different CNA-genes meeting the criteria defined above identified 10 CNA-genes that could be functionally connected to both CNA-gain and CNA-loss trans protein-level\",\n",
            "          \"connected to both CNA-gain and CNA-loss trans protein-level effects (Extended Data Fig. 5c, Supplementary Table 11). A permutation-based approach implemented to test significance (see Supplementary Methods) yielded an FDR < 0.05 for 10 genes affected by\",\n",
            "          \"Methods) yielded an FDR < 0.05 for 10 genes affected by both CNA gains and losses (Fig. 2c). These proteins were defined as potential regulatory candidates for the CNA trans effects observed on the proteome level in this study, since in a gene-dependent\",\n",
            "          \"on the proteome level in this study, since in a gene-dependent manner on average 17% of these trans effects were consistent with the knockdown profiles. Notably, the established oncogenic receptor tyrosine kinase ERBB2 was functionally connected only to\",\n",
            "          \"tyrosine kinase ERBB2 was functionally connected only to CNA gain trans effects (Supplementary Table 11). The E3 ligase SKP1 and the ribonucleoprotein export factor CETN3, both located on chromosome arm 5q with frequent losses in basal-like breast cancer\",\n",
            "          \"arm 5q with frequent losses in basal-like breast cancer and less frequent gains in luminal B breast cancer, were detected as potential regulators affecting the expression of the tyrosine kinase and therapeutic target EGFR, and SKP1 also was linked to SRC\",\n",
            "          \"and therapeutic target EGFR, and SKP1 also was linked to SRC (Extended Data Fig. 5d). Another potential regulator, FBXO7, (a substrate recognition component of the SCF (SKP1-CUL1-F-box protein)-type E3 ubiquitin ligase complex), was affected mostly by LOH\",\n",
            "          \"E3 ubiquitin ligase complex), was affected mostly by LOH events on chromosome 22q. Interestingly, in a recent human interaction proteome study SKP1 and FBXO7 were listed as interaction partners.\",\n",
            "          \"Clustering and Network analysis\",\n",
            "          \"Transcriptional profiling has converged on four major breast cancer subtypes: Luminal A and B, basal and HER2-enriched. To investigate the extent to which the PAM50 \\u201cintrinsic\\u201d breast cancer classification scheme is reflected or refined on the proteome\",\n",
            "          \"classification scheme is reflected or refined on the proteome level in the CPTAC samples, clustering analyses were first restricted to the reduced set of PAM50 genes. When RNA data for the 50 PAM50 genes were clustered directly (without using a\",\n",
            "          \"for the 50 PAM50 genes were clustered directly (without using a classifier), the clustering was similar to the TCGA PAM50 annotation (second annotation bar in Fig. 3a). Restricting both the RNA and proteome data to the set of 35 PAM50 genes observed in\",\n",
            "          \"RNA and proteome data to the set of 35 PAM50 genes observed in the proteome produced a similar result (bottom two annotation bars in Fig. 3a), and all the major PAM50 groups were recapitulated in the proteome almost as well as in the RNA data. This\",\n",
            "          \"in the proteome almost as well as in the RNA data. This indicates that although different tissue sections of the same tumors were used for RNA-seq and protein analysis, very similar subtype-defining features can be observed in both data types. Global\",\n",
            "          \"features can be observed in both data types. Global proteome and phosphoproteome data were then used to identify proteome subtypes in an unsupervised manner. Consensus clustering identified basal-enriched, luminal-enriched, and stromal-enriched clusters\",\n",
            "          \"basal-enriched, luminal-enriched, and stromal-enriched clusters (Extended Data Figs. 6a\\u2013d, 7a). Unlike the clustering observed with PAM50 genes, mRNA-defined HER2-enriched tumors were distributed across these three proteomic subgroups. The basal-enriched\",\n",
            "          \"across these three proteomic subgroups. The basal-enriched and luminal-enriched groups showed a strong overlap with the mRNA-based PAM50 basal-like and luminal subgroups, whereas stromal-enriched proteome subtype represented a mix of all PAM50 mRNA-based\",\n",
            "          \"proteome subtype represented a mix of all PAM50 mRNA-based subtypes, and has a significantly enriched stromal signature (Extended Data Fig. 3e). Among the stromal-enriched tumors there was strong representation of reactive type I tumors as classified by\",\n",
            "          \"representation of reactive type I tumors as classified by RPPA (Supplementary Table 12), showing agreement between the RPPA and mass spectrometry-based protein analyses for the detection of a tumor subgroup characterized by stromal gene expression.\",\n",
            "          \"Since the basal- and luminal-enriched proteome subgroups are coherent, pathway analyses were conducted on these two subtypes, using the stromal-enriched subgroup as a control to assess specificity. (Fig. 3c, Extended Data Fig. 7b, Supplementary Table 13).\",\n",
            "          \"(Fig. 3c, Extended Data Fig. 7b, Supplementary Table 13). The luminal-enriched subgroup was exclusively enriched for estradiol and ESR1-driven gene sets. In contrast, multiple gene sets were enriched and upregulated specifically in the basal-like tumors.\",\n",
            "          \"enriched and upregulated specifically in the basal-like tumors. Particularly extensive basal-like enrichment was seen for MYC target genes; for cell cycle, checkpoint, and DNA repair pathways including regulators AURKA/B, ATM, ATR, CHEK1/2, and BRCA1/2;\",\n",
            "          \"including regulators AURKA/B, ATM, ATR, CHEK1/2, and BRCA1/2; and for immune response/inflammation, including T-cell, B-cell, and neutrophil signatures. The complementarity of transcriptional, proteomic, and phosphoproteomic data was also highlighted in\",\n",
            "          \"proteomic, and phosphoproteomic data was also highlighted in these analyses (Extended Data Fig. 7c, d).\",\n",
            "          \"Using phosphorylation status as a proxy for activity, phosphoproteome profiling can theoretically be used to develop a signaling pathway-based cancer classification. K-means consensus clustering was therefore performed on pathways derived from single\",\n",
            "          \"was therefore performed on pathways derived from single sample GSEA analysis of phosphopeptide data (Methods, Supplementary Tables 14 and 15). Of four robustly segregated groups, subgroups 2 and 3 substantially recapitulated the stromal- and\",\n",
            "          \"subgroups 2 and 3 substantially recapitulated the stromal- and luminal-enriched proteomic subgroups, respectively (Fig. 3d, Extended Data Fig. 8a). Subgroup 4 included a majority of tumors from the basal-enriched proteomic subgroup, but was admixed\",\n",
            "          \"from the basal-enriched proteomic subgroup, but was admixed particularly with luminal-enriched samples. This subgroup was defined by high levels of cell cycle and checkpoint activity. All basal and a majority of non-basal samples in this subgroup had TP53\",\n",
            "          \"and a majority of non-basal samples in this subgroup had TP53 mutations. Consistent with high levels of cell cycle activity, a multivariate kinase-to-phosphosite abundance regression analysis highlighted CDK1 as one of the most highly connected kinases in\",\n",
            "          \"highlighted CDK1 as one of the most highly connected kinases in this study (Extended Data Fig. 8b, Supplementary Table 16). Subgroup 1 was a novel subgroup defined exclusively in the phosphopeptide/pathway activity domain, with no enrichment for either\",\n",
            "          \"activity domain, with no enrichment for either proteomic or PAM50 subtypes. It was defined by G-protein, G-protein coupled receptor, and inositol phosphate metabolism signatures, as well as ionotropic glutamate signaling (Fig. 3d). Co-expression patterns\",\n",
            "          \"glutamate signaling (Fig. 3d). Co-expression patterns among genes/proteins across different subgroups were also analyzed using a Joint Random Forest (JRF) method that identified network modules, such as an MMP9 module, with different interaction patterns\",\n",
            "          \"such as an MMP9 module, with different interaction patterns between basal-enriched and luminal-enriched subgroups. These latter patterns appeared specific to the proteome-level data (Extended Data Fig. 8 c\\u2013f, Supplementary Table 17 and Supplementary\",\n",
            "          \"Data Fig. 8 c\\u2013f, Supplementary Table 17 and Supplementary Methods).\",\n",
            "          \"Phosphosite markers in PIK3CA and TP53 mutant breast cancer\",\n",
            "          \"TP53 and PIK3CA are the most recurrently mutated genes in breast cancer, with frequencies for PIK3CA at 43% in luminal tumors and for TP53 at 84% in basal-like tumors. Most of the PIK3CA missense mutations were gain of function mutations and therefore\",\n",
            "          \"mutations were gain of function mutations and therefore expected to lead to an activation of the PI3K signaling cascade, but the extent to which this occurs has been controversial and there is uncertainty which pathway components are effectors. Marker\",\n",
            "          \"is uncertainty which pathway components are effectors. Marker selection analysis was therefore performed for upregulated phosphosites in PIK3CA-mutated tumors. In total, 62 phosphosites were identified that were positively associated with PIK3CA mutation\",\n",
            "          \"identified that were positively associated with PIK3CA mutation (FDR<0.05), including the kinases RPS6KA5 and EIF2AK4 (Extended Data Fig. 9a, Supplementary Table 18). Calculating the average phosphorylation signal of these marker phosphosites provided a\",\n",
            "          \"phosphorylation signal of these marker phosphosites provided a read-out for PI3K pathway activity in PIK3CA-mutated tumors, with 15 of the 26 mutated tumors (58%) exhibiting an activated PIK3CA mutation signature. Of note, the identified PIK3CA mutant\",\n",
            "          \"mutation signature. Of note, the identified PIK3CA mutant phosphoproteome signature was activated in all tumors harboring helical domain PIK3CA mutations but only 2 of 10 tumors harboring kinase domain mutations. To test if the identified differences in\",\n",
            "          \"domain mutations. To test if the identified differences in the phosphoproteome of PI3K mutant versus wild-type tumors could be explained by mutation of PIK3CA, the tumor data were compared to phosphosite signatures derived from isogenic PIK3CA mutant cell\",\n",
            "          \"phosphosite signatures derived from isogenic PIK3CA mutant cell lines (Extended Data Fig. 9b, Supplementary Table 18). There was an enrichment of signatures derived from helical domain-mutated isogenic cell lines, but not from kinase domain-mutated cells,\",\n",
            "          \"isogenic cell lines, but not from kinase domain-mutated cells, supporting the observations in primary tumors.\",\n",
            "          \"The same strategy was used to identify phosphorylation signaling events connected to TP53 mutation. A total of 56 phosphosites upregulated in TP53 mutant tumors were identified that were independent of basal-like subtype association (Extended Data Fig.\",\n",
            "          \"of basal-like subtype association (Extended Data Fig. 9c, Supplementary Table 18). Using the average phosphorylation signal of these marker phosphosites as a proxy for TP53 mutation-driven cell cycle control, 22 of 41 mutated tumors (54%) showed\",\n",
            "          \"cell cycle control, 22 of 41 mutated tumors (54%) showed upregulated signals. This TP53 mutant phosphosignature was somewhat enhanced in tumors in which mutations occurred almost exclusively in the DNA binding region compared to those with\",\n",
            "          \"exclusively in the DNA binding region compared to those with non-sense/frameshift mutations. In addition to the well-described checkpoint kinase CHEK2, significantly upregulated phosphosites were identified for the kinases MASTL and EEF2K in TP53-mutated\",\n",
            "          \"were identified for the kinases MASTL and EEF2K in TP53-mutated tumors. Single sample GSEA analysis of isogenic p53-mutant phosphosignatures showed an enrichment of a phosphosignature derived from R273H mutated isogenic cells (Extended Data Fig. 9d),\",\n",
            "          \"from R273H mutated isogenic cells (Extended Data Fig. 9d), confirming the pronounced effect of missense mutations in the DNA-binding region on phosphorylation pathways.\",\n",
            "          \"Identification of gene amplification and breast cancer subtype-specific activated kinases in human breast cancer\",\n",
            "          \"CNA spans many driver gene candidates and RNA expression has been frequently used to narrow candidate nominations. Proteogenomic analysis should further promote this nomination process. In this candidate refinement a focus on protein kinases is warranted,\",\n",
            "          \"candidate refinement a focus on protein kinases is warranted, since many are drug targets. An in-depth proteogenomic pipeline was developed that flagged kinases, expression levels of which were at least 1.5 interquartile ranges higher than the median\",\n",
            "          \"were at least 1.5 interquartile ranges higher than the median (Supplemental Table 19). A proteogenomic circos-like plot (termed a \\u201cpircos\\u201d plot) was used to map these outlier kinase values onto the genome (Fig. 4a,b, Extended Data Fig. 10a). The ERBB2\",\n",
            "          \"onto the genome (Fig. 4a,b, Extended Data Fig. 10a). The ERBB2 locus showed the strongest effect of increased phosphoprotein levels associated with gene amplification-driven RNA and protein over expression (Fig. 4a). CDK12 is a positive transcriptional\",\n",
            "          \"over expression (Fig. 4a). CDK12 is a positive transcriptional regulator of homologous recombination repair genes with its partner Cyclin K, and is often encompassed by the ERBB2 amplicon. This gene was also found to be upregulated at the RNA, protein and\",\n",
            "          \"gene was also found to be upregulated at the RNA, protein and phosphosite level indicating that CDK12 is highly active in the majority of ERBB2 positive tumors (Fig. 4a). The analysis of the ERBB2 amplicon also uncovered co-outlier phosphorylation status\",\n",
            "          \"ERBB2 amplicon also uncovered co-outlier phosphorylation status for MED1, GRB7, MSL1, CASC3 and TOP2A, all previously described in association with ERBB2 amplification. To better understand the downstream effects of ERBB2 amplification, additional\",\n",
            "          \"the downstream effects of ERBB2 amplification, additional phosphosite outliers were identified in 41 known ERBB2 signaling genes for the 15 samples that had ERBB2 phosphosite outlier expression (Extended Data Fig. 10b).\",\n",
            "          \"These canonical findings stimulated a proteogenomic analysis to identify additional outlier kinases in the breast cancer genome. A proteogenomic dissection of chromosome 11q based on PAK1 amplification (Fig. 4b,c), a breast cancer driver kinase,\",\n",
            "          \"PAK1 amplification (Fig. 4b,c), a breast cancer driver kinase, illustrated that PAK1 is hyperphosphorylated in PAK1 amplified tumors, along with CLNS1A, RFS1 and GAB2. Additional examples of outlier kinases included PTK2 and RIPK2 in association with\",\n",
            "          \"of outlier kinases included PTK2 and RIPK2 in association with amplification of chromosome 8q (Fig. 4c; Extended Data Fig. 10a,c). PAK1 and TLK2 (17q23) appear to be luminal breast cancer specific events Fig. 4c; Extended Data Fig. 10c). To further\",\n",
            "          \"specific events Fig. 4c; Extended Data Fig. 10c). To further examine whether outlier kinases were breast cancer subtype-specific, independent of amplification status, the BH-corrected probability was calculated of finding that number of phosphosite\",\n",
            "          \"was calculated of finding that number of phosphosite outliers within a subtype, given the total number of outliers across all subtypes, the subtype sample size and the total sample size. (Fig. 4d). These analyses led to the expected identification of\",\n",
            "          \"(Fig. 4d). These analyses led to the expected identification of ERBB2 in the HER2-enriched subtype at the 5% FDR level, as well as the new finding of CDC42BPG (MRGKG), an effector kinase for RHO-family GTPases. In basal-like breast cancer, two kinases,\",\n",
            "          \"RHO-family GTPases. In basal-like breast cancer, two kinases, PRKDC and SPEG, were significant at the 5% FDR level. PRKDC is a non-homologous end-joining (NHEJ) factor that can be phosphorylated by ATM kinase, and is therefore a logical finding in this\",\n",
            "          \"by ATM kinase, and is therefore a logical finding in this disease subset. However SPEG, a kinase associated with severe dilated cardiomyopathy when suppressed, has not been previously reported in association with breast cancer. A larger number of\",\n",
            "          \"reported in association with breast cancer. A larger number of subtype-specific kinases were detected at the 10% FDR level, several of which have recently described relevance in breast cancer, including PRKD3 in basal-like breast cancer, the LKB-regulated\",\n",
            "          \"including PRKD3 in basal-like breast cancer, the LKB-regulated SIK3 in luminal A breast cancer and CDK13 in luminal B breast cancer, which, similar to CDK12, can interact with Cyclin K.\",\n",
            "          \"Discussion\",\n",
            "          \"The analytic breadth and depth of proteomic and phosphoproteomic analyses displayed in this study demonstrates the strengths of mass spectrometry-based proteomics, but also some of the limitations inherent in proteolytic peptide sequencing (see\",\n",
            "          \"the limitations inherent in proteolytic peptide sequencing (see Supplementary Discussion). An example of how high-dimensional proteomic analysis provides insight into unresolved genomic issues concerns the study of loss of the long arm of chromosome 5\",\n",
            "          \"concerns the study of loss of the long arm of chromosome 5 (5q). Analysis of RNA and protein correlations narrowed the list of potential trans-deregulated proteins. Orthogonal candidate screening using functional genomics methods identified loss of CETN3\",\n",
            "          \"using functional genomics methods identified loss of CETN3 and SKP1 as potential transregulators, with upregulation of EGFR as a downstream consequence in basal-like breast cancers. While further experimental evidence must be sought for these proposed\",\n",
            "          \"further experimental evidence must be sought for these proposed regulatory relationships, SKP1/Cullin complex has already been linked to EGFR activation in Glioma. Unfortunately EGFR targeting has not to date proven to be effective therapy in basal-like\",\n",
            "          \"has not to date proven to be effective therapy in basal-like breast cancer. This might be due to the fact the SKP1 loss deregulates multiple targets requiring a much broader inhibitory strategy.\",\n",
            "          \"It is recognized that PIK3CA mutations do not strongly activate canonical downstream effectors. Mass spectrometry-based phosphoproteomics provides an opportunity for unbiased examination of downstream signaling events consequent upon PIK3CA mutational\",\n",
            "          \"downstream signaling events consequent upon PIK3CA mutational activation. These studies revealed that common PIK3CA mutations affect a large number of targets with diverse functionalities including the kinases RPS6KA5 and EIF2AK4. Thus, the data and\",\n",
            "          \"including the kinases RPS6KA5 and EIF2AK4. Thus, the data and analyses reported here extend our knowledge of the effectors that promote tumorigenesis in response to constitutive activation of PI3 kinase. Similarly, TP53 mutation-associated phosphopeptides\",\n",
            "          \"PI3 kinase. Similarly, TP53 mutation-associated phosphopeptides point towards novel functionalities, including regulation of the kinases MASTL and EEF2K.\",\n",
            "          \"A central goal in breast cancer research has been the identification of druggable kinases beyond HER2. Candidate genes that exhibited similar gene amplification-driven proteogenomic patterns to HER2 included CDK12, TLK2, PAK1 and RIPK2. The proteogenomic\",\n",
            "          \"to HER2 included CDK12, TLK2, PAK1 and RIPK2. The proteogenomic link with gene amplification was particularly strong for CDK12, in keeping with its location in the ERBB2 amplicon, while the strengths of correlation between DNA amplification, RNA, protein\",\n",
            "          \"of correlation between DNA amplification, RNA, protein and phosphoprotein for the other examples were more variable. The presence of activated CDK12 in the ERBB2 amplicon might explain why tumors arising in BRCA1 carriers are usually ERBB2 negative. As a\",\n",
            "          \"arising in BRCA1 carriers are usually ERBB2 negative. As a positive transcriptional regulator of BRCA1 and multiple FANC family members, CDK12 promotes DNA repair by homologous recombination. CDK12 amplification would, therefore, oppose the functional\",\n",
            "          \"CDK12 amplification would, therefore, oppose the functional effects of BRCA1 haploinsufficiency during tumor evolution. Overall, multiple outlier kinases generate testable therapeutic hypotheses for which enabling inhibitors are in development. For\",\n",
            "          \"for which enabling inhibitors are in development. For example PAK1 has recently been confirmed to be a therapeutic target and poor prognosis factor in luminal breast cancer.\",\n",
            "          \"Although incomplete outcome data and the remarkable heterogeneity of breast cancer are additional relevant constraints, the number of TCGA specimens analyzed here is insufficient to support conclusive clinical correlations. Only 8 deaths occurred among\",\n",
            "          \"conclusive clinical correlations. Only 8 deaths occurred among the 77 patients, which are too few samples to provide sufficient statistical power for association analysis. Adequately powered MS-based clinical investigation will require targeted\",\n",
            "          \"powered MS-based clinical investigation will require targeted approaches, especially given the highly limited amount of patient material available from clinical trials and the mostly formalin-fixed nature of the specimens. The current analysis is\",\n",
            "          \"formalin-fixed nature of the specimens. The current analysis is therefore centered on biological findings and correlations, with orthogonal validation and false discovery concerns addressed through an examination of cell-line databases of the effects of\",\n",
            "          \"through an examination of cell-line databases of the effects of individual gene perturbations. Typical of a multi-tiered analysis of this complexity, there are many hypotheses to test, and many findings that require further investigation.\",\n",
            "          \"In conclusion, this study provides a high-quality proteomic resource for human breast cancer investigation and illustrates technologies and analytical approaches that provide an important new opportunity to connect the genome to the proteome. Larger-scale\",\n",
            "          \"opportunity to connect the genome to the proteome. Larger-scale exploration of discovery proteomics in the clinical setting will require improvements in clinical investigation, including acquisition of adequate amounts of optimally collected tumor tissue\",\n",
            "          \"of adequate amounts of optimally collected tumor tissue both before and during therapy as well as advances in MS proteomics to reduce sample input and increase sensitivity for low abundance proteins and modified peptides.\",\n",
            "          \"Extended Data\\n\\nExperimental and data analysis workflows and longitudinal data generation quality control\",\n",
            "          \"a, iTRAQ 4-plex global proteome and phosphoproteome analysis workflow. 105 TCGA breast tumors were analyzed in 35 iTRAQ 4-plex experiments (plus 1 replicate and 1 normal sample experiment), with three tumors of different subtypes compared to a fourth\",\n",
            "          \"with three tumors of different subtypes compared to a fourth common internal reference sample in each experiment. The reference sample comprised 10 individual tumors of each of the 4 major breast cancer intrinsic subtypes and served as an internal\",\n",
            "          \"breast cancer intrinsic subtypes and served as an internal standard for all proteins and phosphoproteins quantified in this study. Each iTRAQ MS/MS spectrum measures a peptide from 4 samples (3 individual patients and the reference sample mix of 40\",\n",
            "          \"(3 individual patients and the reference sample mix of 40 patients). More than 400,000 distinct peptides were identified and quantified in ~14 million MS/MS spectra. Personalized tumor-specific protein databases were generated in the QUILTS software\",\n",
            "          \"protein databases were generated in the QUILTS software package using whole exome sequencing-derived variant calls and RNAseq-derived transcript information. All mass spectrometry data was analyzed using the Spectrum Mill software package. b, Overview of\",\n",
            "          \"using the Spectrum Mill software package. b, Overview of proteome and phosphoproteome datasets. The table provides a summary of the datasets used in specific analyses, including the filters applied to derive the proteins and phosphosites/phosphoproteins\",\n",
            "          \"applied to derive the proteins and phosphosites/phosphoproteins that constitute each dataset; the protein, phosphosite or phosphoprotein count; and the methods that employ the respective datasets. c, Distribution of sequence coverage of the identified\",\n",
            "          \"c, Distribution of sequence coverage of the identified proteins with tryptic peptides detected by MS/MS, whiskers show the 5\\u201395 percentiles. d and e, Robust and accurate proteome/phosphoproteome platform. Longitudinal performance was tested by repeated\",\n",
            "          \"platform. Longitudinal performance was tested by repeated proteome and phosphoproteome analysis of patient-derived xenograft tumors. Scatterplots, histograms and Pearson correlations comparing individual replicate measurements are shown.\",\n",
            "          \"Tumor sample quality control (I)\",\n",
            "          \"a, Remark diagram showing sample processing and partitioning. Initial quality review encompassed histopathological examination of H&E stained tissue slices. *For 3 samples no tumor cells were seen on histopathology (BH-A0E9, BH-A0C1, A2-A0SW). These\",\n",
            "          \"were seen on histopathology (BH-A0E9, BH-A0C1, A2-A0SW). These samples were nevertheless included in the proteome analysis since other quality control standards were met (see below) and samples with 0% tumor cellularity on top or bottom sections were\",\n",
            "          \"with 0% tumor cellularity on top or bottom sections were included in TCGA analyses. b, correlation of TCGA (top or bottom sections) and CPTAC histological assessment of neoplastic cellularity for samples (n = 105). The average and range of neoplastic\",\n",
            "          \"for samples (n = 105). The average and range of neoplastic cellularities were identical for CPTAC and TCGA histological assessments. Averages (standard deviations) for neoplastic cellularity were 76% (+/\\u2212 17) for CPTAC, 76% (+/\\u2212 15) for TCGA_Top, and 75%\",\n",
            "          \"were 76% (+/\\u2212 17) for CPTAC, 76% (+/\\u2212 15) for TCGA_Top, and 75% (+/\\u2212 18) for TCGA_Bottom histopathology slides (Supplementary Table 2). Note that in three CPTAC cases where no tumor cells were identified by histopathological assessment, numbers of\",\n",
            "          \"were identified by histopathological assessment, numbers of protein-level somatic variants were similar to all other tumors. The identified mutated proteins were TP53_R273C, NOP58_Q23E, TAGLN2_G154R, TUBA1B_D116H, and MRPL48_I173K (Supplementary Table 5),\",\n",
            "          \"TUBA1B_D116H, and MRPL48_I173K (Supplementary Table 5), indicating presence of tumor cells in these samples. c, Proteome iTRAQ tumor/internal reference ratio heatmap for all CPTAC samples (8,028 proteins without missing values) including passed and failed\",\n",
            "          \"proteins without missing values) including passed and failed proteomic quality control (QC) samples. d, Global tumor/reference proteome ratio distributions for samples that passed and failed proteomic quality control analysis. e, Degradation-related gene\",\n",
            "          \"proteomic quality control analysis. e, Degradation-related gene sets were enriched in tumors that failed proteomic quality control analysis. f, Variant allele frequency (VAF) analysis of re-sequenced CPTAC tumors and comparison to original TCGA data.\",\n",
            "          \"re-sequenced CPTAC tumors and comparison to original TCGA data. Overall VAFs for failed QC samples were lower compared to passed samples suggesting lower purity.\",\n",
            "          \"Tumor sample quality control (II)\",\n",
            "          \"a, There was high concordance (94.6%) between DNA variants reported by TCGA and CPTAC re-sequenced tumors. Most point mutations reported by TCGA could be identified across the 8 re-sequenced samples used in the study. b, A high overall correlation\",\n",
            "          \"samples used in the study. b, A high overall correlation (mean=0.77) was observed for the CPTAC Variant Allele Fraction (VAF) (X-axis) and TCGA VAF (Y-axis) across the 8 samples used in the study. c, Agglomerative hierarchical clustering (Supplementary\",\n",
            "          \"study. c, Agglomerative hierarchical clustering (Supplementary Methods Section 3.8) used to co-cluster protein and RNA tumor expression data after filtering to retain 4,291 proteins and genes with moderate to high protein-RNA correlation (Pearson\",\n",
            "          \"genes with moderate to high protein-RNA correlation (Pearson correlation > 0.4) with results displayed as a circular dendrogram (fanplot). The proteome (.P) and RNA (.R) components of each sample are labeled using the same color. The outer ring shows\",\n",
            "          \"sample are labeled using the same color. The outer ring shows proteome samples in light grey and RNA samples in dark grey. High concordance between RNA and protein expression is evident from the color adjacency in the inner ring and alternating color in\",\n",
            "          \"the color adjacency in the inner ring and alternating color in the outer ring showing that RNA and protein components co-cluster for a large proportion of samples (62/80). d, Co-clustering of MS and RPPA tumor data. 126 RPPA readouts were mapped to gene\",\n",
            "          \"MS and RPPA tumor data. 126 RPPA readouts were mapped to gene names. These genes were intersected with the genes observed in the MS proteome, filtered to 48 proteins with moderate or higher RPPA-MS protein correlation, and analyzed for co-clustering as in\",\n",
            "          \"protein correlation, and analyzed for co-clustering as in c. 47 of 80 RPPA-MS protein pairs co-cluster. While this is a smaller proportion than for RNA-protein analysis, the number of genes used in the clustering is significantly smaller for RPPA (48 vs.\",\n",
            "          \"in the clustering is significantly smaller for RPPA (48 vs. 4,291 for RNA). e, ESTIMATE tumor purity comparison between mRNA, RNAseq, and proteome data. ANOVA is used to assess the difference in distribution (\\u2212log10(p-value)) of ESTIMATE, stromal, immune,\",\n",
            "          \"in distribution (\\u2212log10(p-value)) of ESTIMATE, stromal, immune, and tumor purity scores across mRNA (microarray), RNA-seq and proteome data. The only significant p-value (=0.02) is for the Cluster 3 stromal score, and higher stromal scores for the\",\n",
            "          \"the Cluster 3 stromal score, and higher stromal scores for the proteome drive that difference. f, Ischemia score analysis. Comparison of ischemia scores of 77 CPTAC tumors, 3 normal samples, and patient-derived xenografts. CPTAC tumors had generally lower\",\n",
            "          \"patient-derived xenografts. CPTAC tumors had generally lower ischemia scores than PDX samples subjected to 30 minutes of cold ischemia. Median ischemia scores are less than 30 minutes for each subtype and no significant differences were observed across\",\n",
            "          \"subtype and no significant differences were observed across subtypes. Effects due to cold ischemia therefore appear to be negligible in this CPTAC sample collection.\",\n",
            "          \"Protein-to-Protein, -CNA, and -mRNA correlation analyses\",\n",
            "          \"a, Identification of UBE3A as an E3 ubiquitin ligase that negatively correlates to p53 on the protein level. Pearson correlation and Benjamini-Hochberg corrected p-value are shown. b, Analysis of counter-regulated genes with negative correlation of\",\n",
            "          \"of counter-regulated genes with negative correlation of CNA-to-RNA as well as CNA-to-protein levels. Negative Pearson correlations are shown with Benjamini-Hochberg corrected p-values for CNA-to-protein correlations. Depicted genes have significant\",\n",
            "          \"CNA-to-protein correlations. Depicted genes have significant negative correlations at FDR<0.05 in the CNA-to-RNA and CNA-to-protein analyses. c, Global mRNA-to-protein correlation and gene set enrichment analysis.\",\n",
            "          \"Global CNA effects and comparison of CNA TRANS effects to knockdown signatures in the LINCS database\",\n",
            "          \"a, CNA landscape in the CPTAC tumor collection. The segment-based CNAs of 77 samples were downloaded from TCGA Firehose, including 18 Basal, 12 Her2, 23 Luminal A and 24 Luminal B subtypes. Copy number amplifications were marked in red and deletions in\",\n",
            "          \"Copy number amplifications were marked in red and deletions in blue. The bottom color key represents the log2 transformed copy number value, with CNA=2 centered at 0. Specific CNA events are seen for chromosome 5q and 10p regions in basal-like tumors. b,\",\n",
            "          \"seen for chromosome 5q and 10p regions in basal-like tumors. b, Correlations of copy number alterations (x-axis) to phosphoprotein levels (y-axis) highlight new CNA cis and trans effects. Significant (FDR<0.05) positive (red) and negative (green)\",\n",
            "          \"Significant (FDR<0.05) positive (red) and negative (green) correlations between CNA and phosphoproteins are indicated. Histograms show the fraction [%] of significant CNA trans effects for each CNA gene. c, LINCS CMap analysis facilitates identification\",\n",
            "          \"CNA gene. c, LINCS CMap analysis facilitates identification of novel functional candidates for CNA trans effects. Knockdown profiles were compared with CNA/protein trans effects for 502 genes. Genes with a connectivity score >|90| were considered\",\n",
            "          \"genes. Genes with a connectivity score >|90| were considered connected and significant cis effects were annotated at an FDR<0.05. d, Basal-like tumor-specific CNAs are candidate regulatory events for EGFR and SRC expression levels. Oncogenic kinases with\",\n",
            "          \"for EGFR and SRC expression levels. Oncogenic kinases with significant CNA/protein trans effects (left panel), that were regulated in LINCS shRNA experiments (right panel; 4 cell lines,) and directly measured as LINCS landmark genes, are shown alongside\",\n",
            "          \"directly measured as LINCS landmark genes, are shown alongside candidate regulatory genes CETN3 and SKP1. Clinical ER, PR, and HER2 annotation and PAM50 classification are shown in the header rows of each column.\",\n",
            "          \"Proteome cluster heatmap and stability analysis\",\n",
            "          \"a, K-means consensus clustering of proteome and phosphoproteome data identifies three subgroups: basal-enriched, luminal-enriched, and stromal-enriched. The heatmap represents all 1,521 proteins used for clustering (Dataset G8). b, Identification of\",\n",
            "          \"proteins used for clustering (Dataset G8). b, Identification of optimal proteome clusters for QC-passed CPTAC breast cancer tumors. Proteome clusters were derived using consensus clustering based on 1000 resampled datasets, exploring the range of 2 to 6\",\n",
            "          \"based on 1000 resampled datasets, exploring the range of 2 to 6 k-means clusters. Visualization of consensus matrices from k-means consensus clustering for k=3, 4, 5 and 6 target clusters. Consensus clustering was performed on 1,521 proteins with no\",\n",
            "          \"Consensus clustering was performed on 1,521 proteins with no missing values and SD>1.5. c, Silhouette plots were generated to evaluate the coherence of the clustering. Silhouette plots for k=3 and k=4 clusters showing a cleaner separation of clusters for\",\n",
            "          \"and k=4 clusters showing a cleaner separation of clusters for k=3. d, Based on both visual inspection of the consensus matrix and the delta plot assessing change in consensus cumulative distribution function (CDF) area, three robustly segregated groups\",\n",
            "          \"function (CDF) area, three robustly segregated groups were observed. Consensus cumulative distribution function (CDF) and delta area (change in CDF area) plots for 2\\u20136 clusters.\",\n",
            "          \"Proteome cluster markers and enriched pathways\",\n",
            "          \"a, Markers (based on SAM analysis; FDR<0.01) discriminate between proteome clusters 1, 2 and 3 (compare to heatmap of proteins used to derive clusters depicted in Extended Data Fig. 6a). b, Applying a Fisher exact test-based enrichment analysis to the\",\n",
            "          \"Applying a Fisher exact test-based enrichment analysis to the proteome, phosphoproteome and mRNA data, gene sets from MSigDB were identified that were unique for each proteome cluster. Heat map showing specific pathways comprising dominant biological\",\n",
            "          \"map showing specific pathways comprising dominant biological themes that are significantly differential by enrichment analysis between basal-enriched and luminal-enriched tumors (Fisher Exact Test Benjamini-Hochberg corrected p-values are shown;\",\n",
            "          \"Exact Test Benjamini-Hochberg corrected p-values are shown; enrichment test performed on marker sets identified using SAM analysis; see Methods; compare to Figure 3c). c, Heatmap showing a selection of gene sets significant in basal-enriched or\",\n",
            "          \"a selection of gene sets significant in basal-enriched or luminal-enriched tumors exclusively by mRNA, protein or phosphoprotein expression. Cytokine signatures, for example, were strongly captured at the mRNA level, but were seen to only a limited degree\",\n",
            "          \"at the mRNA level, but were seen to only a limited degree at the global protein level, likely because of their typically low protein abundance. By contrast, the vast majority of significant gene sets annotated as \\\"signaling\\\" were enriched only at the\",\n",
            "          \"gene sets annotated as \\\"signaling\\\" were enriched only at the phosphoprotein level. d, Global heat map representing all gene sets significantly enriched in at least one of the proteomic breast cancer subtypes. The stromal-enriched group was characterized\",\n",
            "          \"cancer subtypes. The stromal-enriched group was characterized by breast cancer normal-like, adipocyte differentiation, smooth muscle, toll-like receptor signaling and endothelin gene sets, supporting the clustering-based annotation of high stromal and/or\",\n",
            "          \"the clustering-based annotation of high stromal and/or adipose content in these tumors (see Supplemental Table 13).\",\n",
            "          \"Phosphoproteome pathway clustering, kinase-phosphosite multivariate regression, and protein co-expression networks\",\n",
            "          \"a, Phosphoproteome pathway clustering. Using phosphorylation state as a proxy for activity, deep phosphoproteome profiling allows development of a breast cancer molecular taxonomy based on signaling pathways. K-means consensus clustering was performed on\",\n",
            "          \"pathways. K-means consensus clustering was performed on pathways derived from single sample GSEA analysis of phosphopeptide data (908 pathways shown). Of four robustly segregated groups, subgroups 2 and 3 substantially recapitulated the stromal- and\",\n",
            "          \"subgroups 2 and 3 substantially recapitulated the stromal- and luminal-enriched proteomic subgroups, respectively. Subgroup 4 included a significant majority of tumors from the basal-enriched proteomic subgroup, but was admixed particularly with\",\n",
            "          \"proteomic subgroup, but was admixed particularly with luminal-enriched samples. This subgroup was defined by high levels of cell cycle and checkpoint activity. All basal and a majority of non-basal samples in this subgroup had TP53 mutations. Subgroup 1\",\n",
            "          \"samples in this subgroup had TP53 mutations. Subgroup 1 was a novel subgroup defined exclusively in the phosphopeptide / pathway activity domain, with no enrichment for either proteomic or PAM50 subtypes. It was defined by G-protein, G-protein coupled\",\n",
            "          \"PAM50 subtypes. It was defined by G-protein, G-protein coupled receptor, and inositol phosphate metabolism signatures, as well as ionotropic glutamate signaling. b, Analysis of the regulatory relationship between outlier kinases (see Supplementary Table\",\n",
            "          \"relationship between outlier kinases (see Supplementary Table 19) and phosphopeptides by regulatory multivariate regression analysis (see Methods) identified CDK1 as the most highly connected of the outlier Cyclin-Dependent Kinases, with highest\",\n",
            "          \"connected of the outlier Cyclin-Dependent Kinases, with highest centrality (based on node-degree; see Methods) among the outlier CDKs and seventh highest centrality among all the outlier kinases considered in the remMap analysis. Each line represents a\",\n",
            "          \"considered in the remMap analysis. Each line represents a phosphosite-kinase relationship. c\\u2013f, Analysis of differences in the co-expression patterns among genes/proteins across different subgroups. A Joint Random Forest (JRF) method was applied to\",\n",
            "          \"subgroups. A Joint Random Forest (JRF) method was applied to simultaneously build gene co-expression and protein co-expression networks (Supplementary Table 17, and Methods). Modules in these networks revealed different interaction patterns between\",\n",
            "          \"these networks revealed different interaction patterns between basal-enriched and luminal-enriched subgroups. c, Network module P1 of the protein co-expression network, defined chiefly in the proteome space. This module contained 12 genes connected by 39\",\n",
            "          \"proteome space. This module contained 12 genes connected by 39 edges, among which 34 were protein-specific and 5 were shared by both the protein and mRNA co-expression networks. Many edges were supported by published information and were contained in the\",\n",
            "          \"supported by published information and were contained in the STRING database. Edges in red are specific to the protein co-expression network; edges in green are shared by both protein and gene co-expression networks; edges indicated by double lines are\",\n",
            "          \"co-expression networks; edges indicated by double lines are contained in the STRING database with confidence score greater than 0.15. MMP9, one of the central proteins in this module, contributes to metastatic progression and is a potential target for\",\n",
            "          \"to metastatic progression and is a potential target for anti-metastatic therapies for basal-like / triple negative breast cancer. d, Heatmaps of the absolute correlation across each pair of genes in module P1 (shown in Panel c), based on either protein or\",\n",
            "          \"in module P1 (shown in Panel c), based on either protein or gene expression data for samples in the basal-enriched and luminal-enriched subgroups, respectively. The MMP9 protein was strongly co-expressed with the other members of the module only in the\",\n",
            "          \"co-expressed with the other members of the module only in the basal-enriched subgroup. Notably, this observation is dependent on protein data; the correlation at the mRNA level for this module was consistently low in both the basal-enriched and luminal\",\n",
            "          \"was consistently low in both the basal-enriched and luminal enriched subgroups indicating that these events coherently occur at the proteomic level. e, Co-expression network based on proteomics data. The network contains 693 proteomic network-specific\",\n",
            "          \"data. The network contains 693 proteomic network-specific edges (grey) and 792 edges shared with the RNAseq network (green). For each module, the most enriched category and corresponding Benjamini-Hochberg adjusted p-value is reported. Pie charts adjacent\",\n",
            "          \"adjusted p-value is reported. Pie charts adjacent to each module show the proportion of proteomics-specific edges (grey area) and edges shared between proteomics and RNAseq data (green area). f, RNAseq network.\",\n",
            "          \"Phosphoproteome signatures of PIK3CA (a,b) and TP53 (c,d) mutated tumors highlight activated key regulators and indicate frequency of activation\",\n",
            "          \"a and c, Phosphosites upregulated in mutated tumors (SAM FDR<0.05 across all tumors and independently also across luminal tumors; average phosphosite signal for all markers shown as bar graph). To avoid confounding by intrinsic subtype-specific\",\n",
            "          \"bar graph). To avoid confounding by intrinsic subtype-specific distinctions, only markers that were significantly identified both in analyses covering all tumors and analyses restricted to luminal tumors were selected (FDR <0.05). Color bars in the\",\n",
            "          \"to luminal tumors were selected (FDR <0.05). Color bars in the margins indicate FDRs for grouped analysis of different mutation classes and indicate kinase substrates of known kinases in the respective pathways. Significantly regulated kinase phosphosites\",\n",
            "          \"pathways. Significantly regulated kinase phosphosites are annotated. The average phosphorylation signal of the marker phosphosites provides a read-out for PI3K and TP53 pathway activity in mutated tumors (histogram below heatmap). A 95% prediction\",\n",
            "          \"in mutated tumors (histogram below heatmap). A 95% prediction confidence interval (indicated by dashed lines) across the average signal in non-mutated tumors was chosen in order to discriminate active from non-active tumors. The most strongly activated\",\n",
            "          \"active from non-active tumors. The most strongly activated PIK3CA kinase domain mutant tumor differed from the other 9 kinase domain mutant tumors, as it contained an amino acid side chain charge neutral H1047L instead of the more common positively\",\n",
            "          \"charge neutral H1047L instead of the more common positively charged H1047R mutation. Among the 62 phosphosites identified that were significantly upregulated in PIK3CA mutated tumors, 13 phosphosites were found on phosphoproteins that are known substrates\",\n",
            "          \"were found on phosphoproteins that are known substrates of well-annotated kinases in the PIK3CA pathway (panel a, right column). In the mutant TP53 analysis a total 20 phosphosites were found on phosphoproteins that are known substrates of well annotated\",\n",
            "          \"on phosphoproteins that are known substrates of well annotated kinases in the p53 pathway (panel c, right column). b and d, Upregulated phosphosite sets were derived from isogenic PIK3CA and TP53 mutant versus wild-type cell line pairs and tested for\",\n",
            "          \"and TP53 mutant versus wild-type cell line pairs and tested for enrichment within mutant versus wild-type CPTAC tumors using single sample GSEA. Significantly enriched phosphosite sets are shown (p<0.05).\",\n",
            "          \"PIRCOS plots, kinase outliers and outliers in the ERBB2 pathway\",\n",
            "          \"a, Pircos (Proteogenomics CIRCOS) plots for 8q and 17q showing median CNA, RNA, protein, and phosphosite expression for 20 tumors with amplification in 8q based on RIPK2 CNA>1; 23 tumors with amplification in 8q based on PTK2 CNA>1; 15 tumors with\",\n",
            "          \"with amplification in 8q based on PTK2 CNA>1; 15 tumors with amplification in 17q based on CDK12 CNA >1; and 10 tumors with amplification in 17q based on TLK2 CNA>1. Red indicates expression >1, blue < \\u22121, and grey between \\u22121 and 1. Genes with both copy\",\n",
            "          \">1, blue < \\u22121, and grey between \\u22121 and 1. Genes with both copy number amplification (CNA>1) and increased phosphosite expression (p-site>1) are labeled. b, Phosphosite outliers in known ERBB2 signaling genes. To better understand the downstream effects of\",\n",
            "          \"signaling genes. To better understand the downstream effects of ERBB2 amplification, phosphosite outliers in known ERBB2 signaling genes (MSigDB pathway set, KEGG_ERBB_SIGNALING PATHWAY) were identified for the 15 samples that had ERBB2 phosphosite\",\n",
            "          \"were identified for the 15 samples that had ERBB2 phosphosite outlier status. Forty-one genes were identified as having a phosphosite outlier in at least one of the ERBB2 amplified samples. PAK4 and ARAF phosphosite outlier status were found in seven of\",\n",
            "          \"PAK4 and ARAF phosphosite outlier status were found in seven of the 15 ERBB2 kinase outlier samples; GSK3B outliers were found in 6 samples; and EIF4EBP1, MAP2K2, ABL1 and AKT1 outlier status was found in 5 of the 15 samples. c, Proteogenomic outlier\",\n",
            "          \"was found in 5 of the 15 samples. c, Proteogenomic outlier expression analysis for TLK2 and RIPK2. Samples with outlier phosphosite (red), protein (yellow), RNA (green) and copy number (purple) expression are shown. Phosphosite squares indicate per-sample\",\n",
            "          \"expression are shown. Phosphosite squares indicate per-sample outlier phosphosites.\",\n",
            "          \"Supplementary Material\\n\\nSupplementary Information is available in the online version of the paper.\",\n",
            "          \"Author Contributions P.M., D.R.M., M.A.G., K.R.C., and S.A.C. designed the proteomic analysis experiments, data analysis workflow, and proteomic-genomic data comparisons. P.M., M.A.G., J.W.Q., and S.A.C. directed and performed proteomic analysis of breast\",\n",
            "          \"and S.A.C. directed and performed proteomic analysis of breast tumor and quality control samples. P.M., D.R.M., K.V.R., K.R.C., P.W., X.W., S.C., E.K., F.P., Z.T., J.L., M.L.G., M.W., V.Y., K.H., C.L., M.D.M., P.Y., J.W., B.Z., and D.F. performed\",\n",
            "          \"V.Y., K.H., C.L., M.D.M., P.Y., J.W., B.Z., and D.F. performed proteomic-genomic data analyses. D.R.M., P.W., and S.J.S. provided statistical support. D.R.M., K.V.R., K.R.C., K.K. and D.F. performed analyses of mass spectrometry data and adapted\",\n",
            "          \"D.F. performed analyses of mass spectrometry data and adapted algorithms and software for data analysis. S.R.D., R.R.T and M.J.E. developed and prepared breast xenografts used as quality control samples. P.M. and F.M prepared and analyzed cell lines for\",\n",
            "          \"samples. P.M. and F.M prepared and analyzed cell lines for correlative functional annotation of frequently mutated genes. P.M., D.R.M., M.A.G., and S.A.C designed strategy for quality control analyses. M.A.G., S.R.D., C.R.K., M.M., and H.R. coordinated\",\n",
            "          \"analyses. M.A.G., S.R.D., C.R.K., M.M., and H.R. coordinated acquisition, distribution and quality control evaluation of TCGA tumor samples. P.M., M.A.G., C.P., L.D., A.G.P., and M.J.E. interpreted data in the context of breast cancer biology. P.M.,\",\n",
            "          \"interpreted data in the context of breast cancer biology. P.M., D.R.M, M.A.G., K.R.C., P.W., A.G.P, M.J.E. and S.A.C. wrote the manuscript.\",\n",
            "          \"All primary mass spectrometry data are deposited at the CPTAC Data Portal as raw and mzML files and complete protein assembly data sets for public access (https://cptac-data-portal.georgetown.edu/cptac/s/S029). In addition, a set of ancillary files such\",\n",
            "          \"In addition, a set of ancillary files such as dataset G1/P1, G3/P3, G4/P4, G5/P5, G7/P7, CNA correlation tables for CNA-mRNA, CNA-proteome and CNA-phosphoproteome; CNA data, RNA-seq expression data have also been deposited at the DCC. Two browsers have\",\n",
            "          \"data have also been deposited at the DCC. Two browsers have been created to assist the interested reader in exploring the results. One provides track hubs for viewing the identified peptides in the UCSD genome browser\",\n",
            "          \"for viewing the identified peptides in the UCSD genome browser (http://fenyolab.org/cptac_breast_ucsc). The second is an on-line tool for proteogenomic data exploration and can be accessed at http://prot-shiny-vm.broadinstitute.org:3838/BC2016/. See\",\n",
            "          \"at http://prot-shiny-vm.broadinstitute.org:3838/BC2016/. See Supplemental Methods for descriptions.\",\n",
            "          \"The authors declare no competing financial interests.\\n\\nProteogenomic analysis of human breast cancer. Direct effects of genomic alterations on protein level\",\n",
            "          \"Overlap of a, protein coding single amino acid variants (SAAVs) and b, RNA splice junctions not present in RefSeq v60 detected by DNA exome sequencing, RNA-seq, and LC-MS/MS. Proportions of novel variants are noted. c, Heatmap of mutations/CNA and their\",\n",
            "          \"novel variants are noted. c, Heatmap of mutations/CNA and their effects on RNA and protein expression of breast cancer-relevant genes across tumor and normal samples. ER, PR, HER2 and PAM50 status are annotated. Median iTRAQ protein abundance ratio and\",\n",
            "          \"status are annotated. Median iTRAQ protein abundance ratio and the most frequently detected and differential phosphosite ratio are shown for each gene. Pearson correlations between MS protein vs RNA-seq and MS protein vs RPPA are indicated.\",\n",
            "          \"Effects of copy number alterations (CNA) on mRNA, protein, and phosphoprotein abundance\",\n",
            "          \"a, Correlations of CNA (x-axes) to RNA and protein expression levels (y-axes) highlight new CNA cis and trans effects. Significant (FDR<0.05) positive (red) and negative (green) correlations between CNA and mRNAs or proteins are indicated. CNA cis effects\",\n",
            "          \"CNA and mRNAs or proteins are indicated. CNA cis effects appear as a red diagonal line, CNA trans effects as vertical stripes. Histograms show the fraction [%] of significant CNA trans effects for each CNA gene. b, Overlap of cis effects observed at RNA,\",\n",
            "          \"for each CNA gene. b, Overlap of cis effects observed at RNA, protein, and phosphoprotein levels (FDR<0.05). c, Trans-effect regulatory candidates identified among those with significant protein cis-effects using LINCS CMap. Bars indicate total numbers of\",\n",
            "          \"cis-effects using LINCS CMap. Bars indicate total numbers of significant CNA/protein trans effects (gray; FDR<0.05) and overlap with regulated genes in LINCS knock-down profiles (red; 4 cell lines; moderated T-test FDR<0.1).\",\n",
            "          \"Proteomic and phosphoproteomic subtypes of breast cancer and subtype-specific pathway enrichment\",\n",
            "          \"a, Unsupervised clustering of RNA-seq and proteomics data restricted to PAM50 genes and subset of 35 detected proteins reveal high similarity to PAM50 (TCGA) sample annotation. b, K-means consensus clustering of proteome and phosphoproteome data\",\n",
            "          \"consensus clustering of proteome and phosphoproteome data identifies basal-enriched, luminal-enriched, and stromal-enriched subgroups. c, Gene set enrichment analysis highlights sets of pathways significantly differential between basal-enriched and\",\n",
            "          \"pathways significantly differential between basal-enriched and luminal-enriched tumors (detailed in Extended Data Fig. 7b). d, K-means consensus clustering performed on pathways derived from single sample GSEA analysis of phosphopeptide data identifies\",\n",
            "          \"single sample GSEA analysis of phosphopeptide data identifies four distinct clusters.\",\n",
            "          \"Example analyses of aberrantly regulated kinases in human breast cancer\",\n",
            "          \"a and b, PIRCOS (Proteogenomics CIRCOS) plots showing CNA, RNA, protein and phosphosite expression for 17 tumors with amplification in 17q (ERBB2 CNA>1) and 8 tumors with amplification in 11q (PAK1 CNA>1). Labeled genes have CNA>1 and phosphosite>1. c,\",\n",
            "          \"(PAK1 CNA>1). Labeled genes have CNA>1 and phosphosite>1. c, Proteogenomic outlier expression analysis for ERBB2, CDK12, and PAK1. Samples with outlier phosphosite (red), protein (yellow), RNA (green) and copy number (purple) expression are shown.\",\n",
            "          \"RNA (green) and copy number (purple) expression are shown. Phosphosite squares indicate per-sample outlier phosphosites. d, Outlier kinase events by PAM50 subtype (>35% of subtype samples contain a phosphosite outlier; <10% FDR using Benjamini-Hochberg\",\n",
            "          \"a phosphosite outlier; <10% FDR using Benjamini-Hochberg adjusted p-values).\"\n",
            "        ]\n",
            "      }\n",
            "    ]\n",
            "  },\n",
            "  \"PDC000111\": {\n",
            "    \"study_id\": \"b998098f-57b8-11e8-b07a-00a098d917f8\",\n",
            "    \"pdc_study_id\": \"PDC000111\",\n",
            "    \"study_submitter_id\": \"TCGA COAD Proteome S016-1\",\n",
            "    \"program_id\": \"10251935-5540-11e8-b664-00a098d917f8\",\n",
            "    \"project_id\": \"48af5040-5546-11e8-b664-00a098d917f8\",\n",
            "    \"study_name\": \"TCGA Colon Cancer Proteome\",\n",
            "    \"program_name\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"project_name\": \"CPTAC2 Retrospective\",\n",
            "    \"disease_type\": \"Colon Adenocarcinoma;Rectum Adenocarcinoma\",\n",
            "    \"primary_site\": \"Colon;Rectum\",\n",
            "    \"analytical_fraction\": \"Proteome\",\n",
            "    \"experiment_type\": \"Label Free\",\n",
            "    \"embargo_date\": null,\n",
            "    \"cases_count\": 90,\n",
            "    \"aliquots_count\": 95,\n",
            "    \"protocol_id\": \"1c06b336-3b26-11e9-9a07-0a80fada099c\",\n",
            "    \"protocol_submitter_id\": \"P-S016-1\",\n",
            "    \"program_submitter_id\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"protocol_name\": \"TCGA_Colon_Cancer_Proteome\",\n",
            "    \"protocol_date\": \"2012-08-28\",\n",
            "    \"document_name\": \"VU_Tissue_Prep_By_bRPLC_1.0_20120828\",\n",
            "    \"quantitation_strategy\": \"LC-MS label-free quantitation\",\n",
            "    \"label_free_quantitation\": \"Label-free protein level quantitation\",\n",
            "    \"labeled_quantitation\": \"\",\n",
            "    \"isobaric_labeling_reagent\": \"N/A\",\n",
            "    \"reporter_ion_ms_level\": \"N/A\",\n",
            "    \"starting_amount\": \"200.00\",\n",
            "    \"starting_amount_uom\": \"\\u00b5g\",\n",
            "    \"digestion_reagent\": \"Trypsin\",\n",
            "    \"alkylation_reagent\": \"Iodoacetamide\",\n",
            "    \"enrichment_strategy\": \"Proteome\",\n",
            "    \"enrichment\": \"N/A\",\n",
            "    \"chromatography_dimensions_count\": \"2\",\n",
            "    \"one_d_chromatography_type\": \"bRPLC\",\n",
            "    \"two_d_chromatography_type\": \"RPLC\",\n",
            "    \"fractions_analyzed_count\": \"15\",\n",
            "    \"column_type\": \"C18\",\n",
            "    \"amount_on_column\": \"6\",\n",
            "    \"amount_on_column_uom\": \"\\u00b5g\",\n",
            "    \"column_length\": \"11\",\n",
            "    \"column_length_uom\": \"cm\",\n",
            "    \"column_inner_diameter\": \"100.00\",\n",
            "    \"column_inner_diameter_uom\": \"\\u00b5m\",\n",
            "    \"particle_size\": \"5\",\n",
            "    \"particle_size_uom\": \"\\u00b5m\",\n",
            "    \"particle_type\": \"300 \\u00c5,  Phenomenex\",\n",
            "    \"gradient_length\": \"95\",\n",
            "    \"gradient_length_uom\": \"min\",\n",
            "    \"instrument_make\": \"Thermo Scientific\",\n",
            "    \"instrument_model\": \"Orbitrap Velos\",\n",
            "    \"dissociation_type\": \"CID\",\n",
            "    \"ms1_resolution\": \"60000\",\n",
            "    \"ms2_resolution\": \"1\",\n",
            "    \"dda_topn\": \"Top 8\",\n",
            "    \"normalized_collision_energy\": \"35%\",\n",
            "    \"acquistion_type\": \"DDA\",\n",
            "    \"dia_multiplexing\": \"N/A\",\n",
            "    \"dia_ims\": \"N/A\",\n",
            "    \"analytical_technique\": null,\n",
            "    \"chromatography_instrument_make\": null,\n",
            "    \"chromatography_instrument_model\": null,\n",
            "    \"polarity\": null,\n",
            "    \"reconstitution_solvent\": null,\n",
            "    \"reconstitution_volume\": null,\n",
            "    \"reconstitution_volume_uom\": null,\n",
            "    \"internal_standards\": null,\n",
            "    \"extraction_method\": null,\n",
            "    \"ionization_mode\": null,\n",
            "    \"publications\": [\n",
            "      {\n",
            "        \"publication_id\": \"44e2e50c-89c7-11e8-bcf1-0a2705229b82\",\n",
            "        \"pubmed_id\": \"26110064\",\n",
            "        \"doi\": \"10.1038/sdata.2015.22\",\n",
            "        \"author\": \"Slebos, R. J.; Wang, X.; Wang, X.; Wang, X.; Zhang, B.; Tabb, D. L.; Liebler, D. C.\",\n",
            "        \"title\": \"Proteomic analysis of colon and rectal carcinoma using standard and customized databases\",\n",
            "        \"journal\": \"Sci Data\",\n",
            "        \"journal_url\": \"https://www.nature.com/articles/sdata201522\",\n",
            "        \"year\": \"2015\",\n",
            "        \"abstract\": \"Understanding proteomic differences underlying the different phenotypic classes of colon and rectal carcinoma is important and may eventually lead to a better assessment of clinical behavior of these cancers. We here present a comprehensive description of the proteomic data obtained from 90 colon and rectal carcinomas previously subjected to genomic analysis by The Cancer Genome Atlas (TCGA). Here, the primary instrument files and derived secondary data files are compiled and presented in forms that will allow further analyses of the biology of colon and rectal carcinoma. We also discuss new challenges in processing these large proteomic datasets for relevant proteins and protein variants.\",\n",
            "        \"full_text\": [\n",
            "          \"Proteomic analysis of colon and rectal carcinoma using standard and customized databases\",\n",
            "          \"Understanding proteomic differences underlying the different phenotypic classes of colon and rectal carcinoma is important and may eventually lead to a better assessment of clinical behavior of these cancers. We here present a comprehensive description of\",\n",
            "          \"these cancers. We here present a comprehensive description of the proteomic data obtained from 90 colon and rectal carcinomas previously subjected to genomic analysis by The Cancer Genome Atlas (TCGA). Here, the primary instrument files and derived\",\n",
            "          \"Atlas (TCGA). Here, the primary instrument files and derived secondary data files are compiled and presented in forms that will allow further analyses of the biology of colon and rectal carcinoma. We also discuss new challenges in processing these large\",\n",
            "          \"We also discuss new challenges in processing these large proteomic datasets for relevant proteins and protein variants.\",\n",
            "          \"Background & Summary\",\n",
            "          \"Emerging collections of genomics data for human cancers present the formidable challenge of understanding how genomic abnormalities drive the biological and clinical characteristics of cancer. This task will be facilitated by proteomic analyses, which\",\n",
            "          \"This task will be facilitated by proteomic analyses, which provides an intermediate layer of biological information that is more directly connected to phenotype. The National Cancer Institute (NCI) Clinical Proteomic Tumor Assessment Consortium (CPTAC)\",\n",
            "          \"(NCI) Clinical Proteomic Tumor Assessment Consortium (CPTAC) network employs proteomic technology platforms to analyze colorectal, breast and ovarian tumors previously analyzed by the NCI The Cancer Genome Atlas (TCGA) network, with the objective of\",\n",
            "          \"The Cancer Genome Atlas (TCGA) network, with the objective of integrating proteomic and genomic data for the same tissue specimens. Here we describe datasets and data analysis methods for the proteomic analysis of colon and rectal tumors from the TCGA,\",\n",
            "          \"proteomic analysis of colon and rectal tumors from the TCGA, which were combined with TCGA genomics data to produce the first integrated proteogenomic analysis of a human cancer.\",\n",
            "          \"We analyzed 95 samples representing 90 TCGA colon and rectal (CRC) tumors, which comprise a subset of the 224 sample collection subjected to multiplatform genomic analyses by the TCGA. Proteomic analyses by multidimensional liquid chromatography-tandem\",\n",
            "          \"analyses by multidimensional liquid chromatography-tandem mass spectrometry (LC-MS/MS) provide peptide and protein identifications from MS/MS data and estimates of relative abundance, from which expression differences can be inferred. Our data analysis\",\n",
            "          \"which expression differences can be inferred. Our data analysis employed a workflow that is well-established for global proteomics, but is typically used for much smaller datasets. The large size of our proteomic dataset created unanticipated challenges\",\n",
            "          \"size of our proteomic dataset created unanticipated challenges and required additional evaluation of alternative analyses strategies. The final protein assembly summarizes a total of more than 13 million successfully assigned MS/MS spectra representing\",\n",
            "          \"13 million successfully assigned MS/MS spectra representing more than 7,000 non-redundant human protein identifications. We used this dataset to create a proteomics-based classification of colon and rectal carcinoma and demonstrate its use in relation to\",\n",
            "          \"and rectal carcinoma and demonstrate its use in relation to genomics-based classifications. Chromosomal mapping of protein expression profiles revealed both cis- and trans-effects of chromosomal loci commonly involved in amplification and deletion events.\",\n",
            "          \"loci commonly involved in amplification and deletion events. Protein expression changes may be used to create a better understanding of the driving forces behind such chromosomal changes. In addition, we employed a novel strategy that integrates\",\n",
            "          \"In addition, we employed a novel strategy that integrates individual sequence variation as determined by RNA-seq with proteomic identification to improve protein identification and discover single amino acid variation (SAAVs) in the proteome. The current\",\n",
            "          \"amino acid variation (SAAVs) in the proteome. The current paper describes this dataset in detail and makes available additional files that will facilitate further analyses based on our proteomics data. These additional files include search results from\",\n",
            "          \"data. These additional files include search results from the three different protein-search engines and customized databases containing all sequence variations for each of the 90 TCGA tumors. We also discuss the different options at each of the data\",\n",
            "          \"We also discuss the different options at each of the data processing steps and the consequences for the eventual protein inventory.\",\n",
            "          \"Methods\\n\\nSummary\",\n",
            "          \"Description of data acquisition and processing were expanded from our previous work. After receipt of the tumor blocks, protein was extracted from the samples, digested with trypsin, and the resulting peptides were fractionated by off-line basic-reverse\",\n",
            "          \"resulting peptides were fractionated by off-line basic-reverse phase high-performance liquid chromatography. Each tumor sample yielded 15 concatenated peptide fractions, which then were analyzed by reverse phase liquid chromatography-tandem mass\",\n",
            "          \"analyzed by reverse phase liquid chromatography-tandem mass spectrometry. Analysis of the 95 carcinoma and 60 normal epithelial samples took 9 months on a single MS instrument. To monitor system stability and performance, we also analyzed benchmark\",\n",
            "          \"system stability and performance, we also analyzed benchmark quality control (QC) samples from one basal and one luminal human breast tumor xenograft. These were analyzed in alternating order after every 5 sets of colorectal carcinoma and normal colon\",\n",
            "          \"after every 5 sets of colorectal carcinoma and normal colon epithelium samples. This produced proteomics QC datasets for both the 16 basal and luminal xenografts (Data Citation 1). The QC dataset not only enabled assessment of platform performance and\",\n",
            "          \"dataset not only enabled assessment of platform performance and consistency throughout the project, but also provided a representative large proteomics dataset with which to evaluate data normalization. Raw MS/MS data were processed for peptide\",\n",
            "          \"data normalization. Raw MS/MS data were processed for peptide identification by database and spectral library searching using three different search engines (Myrimatch, Pepitome and MS-GF+). Identified peptides were assembled as proteins and mapped to\",\n",
            "          \"Identified peptides were assembled as proteins and mapped to gene identifiers for proteogenomic comparisons using IDPicker 3.0. Quantitative comparisons were based on spectral count data. Details for each of the preparation steps are given below.\",\n",
            "          \"TCGA tumor samples\",\n",
            "          \"All samples for the current study were obtained through the TCGA Biospecimen Core Resource (BCR) as described previously. Of the 276 samples described in the TCGA study, 95 specimens from 90 patients were available for the current study (5 patients\\u2019s\",\n",
            "          \"90 patients were available for the current study (5 patients\\u2019s tumors were provided as duplicate pieces). No other selection criteria other than availability were applied for this study. Specimens for proteomic study were sectioned sequentially from the\",\n",
            "          \"for proteomic study were sectioned sequentially from the tumor blocks used for genomic studies, hence the \\u2018bottom\\u2019 slides from the genomic studies are representative of the material used for proteomics. The slides are available from the TCGA data portal\",\n",
            "          \"proteomics. The slides are available from the TCGA data portal (http://tcga-data.nci.nih.gov/tcga/tcgaHome2.jsp). All samples were frozen within 1\\u2009h of collection and histopathological examination confirmed that they contained at least 60% tumor nuclei as\",\n",
            "          \"confirmed that they contained at least 60% tumor nuclei as described earlier. Prior CPTAC studies have shown that global protein levels, but not phosphorylation states, are stable within this 1\\u2009h time period. Samples were shipped from the TCGA BCR on dry\",\n",
            "          \"1\\u2009h time period. Samples were shipped from the TCGA BCR on dry ice and kept frozen at \\u221280\\u2009\\u00b0C until processing. All TCGA colorectal tissue samples were washed prior to digestion to eliminate any residual optimal cutting temperature compound (OCT). The\",\n",
            "          \"any residual optimal cutting temperature compound (OCT). The tissue was placed in a 1.5\\u2009ml micro-tube was washed with 1\\u2009ml 70% EtOH/30% H2O for 30\\u2009s with vortexing. The supernatant was then discarded and the tissue was washed with 1\\u2009ml of 100% H2O 30\\u2009s\",\n",
            "          \"discarded and the tissue was washed with 1\\u2009ml of 100% H2O 30\\u2009s with vortexing and again the supernatant was discarded. One milliliter of 70% EtOH/30% H2O was added to the tissue sample and incubated for 5\\u2009min at room temperature following by\",\n",
            "          \"sample and incubated for 5\\u2009min at room temperature following by centrifugation at 20,000\\u00d7g for 2\\u2009min at 20\\u2009\\u00b0C. The supernatant was removed, and this wash step was repeated. Next, 1\\u2009ml of 85% EtOH/15% H2O was added to the tissue and incubated for 5\\u2009min at\",\n",
            "          \"EtOH/15% H2O was added to the tissue and incubated for 5\\u2009min at room temperature following by centrifugation at 20,000\\u00d7g for 2\\u2009min at 20\\u2009\\u00b0C. The supernatant was removed, and the wash was repeated. For the final wash, the tissue was washed in 1\\u2009ml 100%\",\n",
            "          \"For the final wash, the tissue was washed in 1\\u2009ml 100% EtOH, incubated 5\\u2009min at room temperature, and centrifuged at 20,000\\u00d7g for 2\\u2009min at 20\\u2009\\u00b0C. The supernatant was removed, and the final wash was repeated.\",\n",
            "          \"Normal colon epithelium\",\n",
            "          \"Normal colon epithelium biopsies were obtained through the Cooperative Human Tissue Network (CHTN) Western Division at Vanderbilt University. Samples were collected from screening colonoscopies performed between July 2006 and October 2010 under Vanderbilt\",\n",
            "          \"performed between July 2006 and October 2010 under Vanderbilt University IRB approval #061096. During colonoscopy, multiple pinch biopsies were obtained from both ascending and descending colon and immediately frozen in liquid nitrogen. Biopsies obtained\",\n",
            "          \"and immediately frozen in liquid nitrogen. Biopsies obtained from 30 subjects with completely normal findings during colonoscopy were included in the study. A total of 60 specimens, one from ascending and one from descending colon, were used for proteomic\",\n",
            "          \"and one from descending colon, were used for proteomic analysis.\",\n",
            "          \"Protein extraction and digestion of tissue specimens\",\n",
            "          \"Following OCT removal, tissue specimens were placed in 1.5\\u2009ml micro-centrifuge tubes and re-suspended 100\\u2009\\u03bcl of trifluoroethanol (TFE) and 100\\u2009\\u03bcl of 100\\u2009mM ammonium bicarbonate, pH 8. If additional buffer was required, equal volumes of TFE and 100\\u2009mM\",\n",
            "          \"additional buffer was required, equal volumes of TFE and 100\\u2009mM ammonium bicarbonate pH 8.0 were added accordingly. In addition, powderized xenograft tumor tissue representing the Comparison/Reference (CompRef) samples for Luminal (WHIM16) and Basal\",\n",
            "          \"(CompRef) samples for Luminal (WHIM16) and Basal (WHIM2) breast cancer subtypes were analyzed in alternating order after every 5 TCGA colorectal tissue samples. Samples were sonicated using a Fisher Scientific Sonic Dismembrator Model 100 at a setting of\",\n",
            "          \"Fisher Scientific Sonic Dismembrator Model 100 at a setting of 20\\u2009watts for 20\\u2009s followed by 30\\u2009s incubation on ice. This sonication step was repeated twice, and samples were placed on ice between sonications. The resulting homogenate was heated with\",\n",
            "          \"between sonications. The resulting homogenate was heated with shaking at 1,000\\u2009rpm for 1\\u2009h at 60\\u2009\\u00b0C followed by a second series of sonication steps, as described above. A protein measurement then was obtained for each sample using the BCA Protein Assay\",\n",
            "          \"then was obtained for each sample using the BCA Protein Assay (ThermoFisher Pierce, Rockford, IL) using the manufacturer\\u2019s protocol.\",\n",
            "          \"An aliquot equivalent to 200\\u2009\\u03bcg was removed and reduced with tris(2-carboxyethyl)phosphine (TCEP, 20\\u2009mM) and dithiothreitol (DTT, 50\\u2009mM) at 60\\u2009\\u00b0C for 30\\u2009min, followed by alkylation with iodoacetamide (IAM, 100\\u2009mM) in the dark at room temperature for\",\n",
            "          \"iodoacetamide (IAM, 100\\u2009mM) in the dark at room temperature for 20\\u2009min. The lysate was diluted with the appropriate volume of 50\\u2009mM ammonium bicarbonate, pH 8.0, to reduce the TFE concentration to 10%. Trypsin was added at a ratio of 1:50 (w:w) and\",\n",
            "          \"to 10%. Trypsin was added at a ratio of 1:50 (w:w) and digested overnight at 37\\u2009\\u00b0C. The digested mixture was frozen at \\u221280\\u2009\\u00b0C and lyophilized to dryness. The lyophilized samples were re-suspended in 350\\u2009\\u03bcl of HPLC-grade water and vortexed vigorously for\",\n",
            "          \"in 350\\u2009\\u03bcl of HPLC-grade water and vortexed vigorously for one minute and desalted using an Oasis HLB 96-well \\u03bcElution plate (30\\u2009\\u03bcm, 5\\u2009mg, Waters Corp., Milford, MA), which was pre-washed with 500\\u2009\\u03bcl of acetonitrile and equilibrated with 750\\u2009\\u03bcl of\",\n",
            "          \"with 500\\u2009\\u03bcl of acetonitrile and equilibrated with 750\\u2009\\u03bcl of HPLC-grade water. The flow-through was discarded, and the plates were washed with 500\\u2009\\u03bcl of HPLC-grade water. The peptides were eluted with 80% acetonitrile, and the eluates were evaporated to\",\n",
            "          \"with 80% acetonitrile, and the eluates were evaporated to dryness in vacuo. Samples were stored in the freezer until further analysis.\",\n",
            "          \"Peptide fractionation by basic reverse phase liquid chromatography (bRPLC)\",\n",
            "          \"Samples were reconstituted in 300\\u2009\\u03bcl of Solvent A (1.0\\u2009M triethylamine bicarbonate (TEAB), pH 7.5). The reconstituted sample was then diluted with an additional 100\\u2009\\u03bcl of Solvent A and the entire 400\\u2009\\u03bcl of solution was injected into the bRPLC column.\",\n",
            "          \"entire 400\\u2009\\u03bcl of solution was injected into the bRPLC column. Tryptic peptides were fractionated using bRPLC separation with an XBridge BEH C18, 250\\u2009mm\\u00d74.6\\u2009mm analytical column (130A, 5\\u2009\\u03bcm particle size) equipped with a XBridge BEH C18 Sentry guard\",\n",
            "          \"particle size) equipped with a XBridge BEH C18 Sentry guard cartridge at a flow rate of 0.5\\u2009ml/min. The solvents were 10\\u2009mM TEAB pH 7.5 in water as mobile phase A and 100% acetonitrile as mobile phase B. Sample fractionation was accomplished using the\",\n",
            "          \"mobile phase B. Sample fractionation was accomplished using the following multistep linear gradient: from 0 to 5% B in 10\\u2009min, from 5 to 35% B in 60\\u2009min, from 35\\u201370% in 15\\u2009min and held at 70% B for an additional 10\\u2009min before returning to initial\",\n",
            "          \"at 70% B for an additional 10\\u2009min before returning to initial conditions. A total of 60 fractions were collected over the 105\\u2009min gradient and concatenated into 15 fractions by combining fractions 1, 16, 31, 46; 2, 17, 32, 47; and so on up to fractions\",\n",
            "          \"1, 16, 31, 46; 2, 17, 32, 47; and so on up to fractions 15, 30, 45, and 60. The samples were evaporated to dryness in a Speed-Vac sample concentrator and stored at \\u221280C until LC-MS/MS analysis.\",\n",
            "          \"LC-MS/MS analysis\",\n",
            "          \"Resulting peptide fractions were resuspended in 50\\u2009\\u03bcl of 2% acetonitrile/0.1% formic acid and analyzed using a Thermo LTQ Orbitrap Velos ion trap mass spectrometer equipped with an Eksigent NanoLC 2D pump and AS-1 autosampler. A 2\\u2009\\u03bcl injection volume of\",\n",
            "          \"NanoLC 2D pump and AS-1 autosampler. A 2\\u2009\\u03bcl injection volume of the peptide solution was separated on a packed capillary tip (Polymicro Technologies, 100\\u2009\\u03bcm\\u00d711\\u2009cm) containing Jupiter C18 resin (5\\u2009\\u03bcm, 300\\u2009\\u00c5, Phenomenex) using an in-line solid-phase\",\n",
            "          \"resin (5\\u2009\\u03bcm, 300\\u2009\\u00c5, Phenomenex) using an in-line solid-phase extraction column (100\\u2009\\u03bcm\\u00d76\\u2009cm) packed with the same C18 resin using a frit generated with liquid silicate Kasil. Mobile phase A consisted of 0.1% formic acid and Mobile phase B consisted of\",\n",
            "          \"A consisted of 0.1% formic acid and Mobile phase B consisted of 0.1% formic acid in acetonitrile. A 95\\u2009min gradient was preceded by a 15\\u2009min washing period (100% A) at a flow-rate of 1.5\\u2009\\u03bcl\\u2009min\\u22121\\u2009to remove residual salt. Following the wash, the mobile\",\n",
            "          \"remove residual salt. Following the wash, the mobile phase was programmed to 25% B by 50\\u2009min, followed by an increase to 90% B by 65\\u2009min and held for 9\\u2009min before returning to the initial conditions. A full MS scan was collected for peptides from\",\n",
            "          \"conditions. A full MS scan was collected for peptides from 400\\u20132000\\u2009m/z on the Orbitrap at a resolution of 60,000 followed by eight data-dependent MS/MS scans from lowest to highest signal intensity on the LTQ. Centroided MS/MS scans were acquired using\",\n",
            "          \"on the LTQ. Centroided MS/MS scans were acquired using an isolation width of 3\\u2009m/z, an activation time of 30\\u2009ms, an activation q of 0.250 and 35% normalized collision energy. One microscan with a max ion time of 100 and 1000\\u2009ms was used for each MS/MS and\",\n",
            "          \"a max ion time of 100 and 1000\\u2009ms was used for each MS/MS and full MS scan, respectively. MS/MS spectra were collected using a dynamic exclusion of 60\\u2009s with a repeat of 1 and repeat duration of 1.\",\n",
            "          \"All TCGA and normal colon epithelial samples were analyzed on the same Thermo LTQ Orbitrap Velos instrument, with sample analysis beginning on July 26, 2012 and concluding on June 27, 2013. Each sample comprised the 15 bRPLC fractions prepared as\",\n",
            "          \"2013. Each sample comprised the 15 bRPLC fractions prepared as described above. Benchmark quality control (QC) samples from one basal and one luminal human breast tumor xenograft (Comparison/Reference, \\u2018CompRef\\u2019 samples) were analyzed in alternating order\",\n",
            "          \"\\u2018CompRef\\u2019 samples) were analyzed in alternating order after every five CRC samples. Specifically, five TCGA samples were run on the instrument, followed by a luminal CompRef sample, and then another five TCGA samples followed by a basal CompRef sample for\",\n",
            "          \"five TCGA samples followed by a basal CompRef sample for a total of 32 CompRef samples. Normal epithelial samples were analyzed using the same schema whereby each 5 normal epithelial samples were followed by a CompRef sample. Bovine serum albumin (BSA)\",\n",
            "          \"were followed by a CompRef sample. Bovine serum albumin (BSA) tryptic digest standards were analyzed before and after every ten TCGA samples and were used to monitor instrument sensitivity, BSA standard sequence coverage and chromatographic performance to\",\n",
            "          \"standard sequence coverage and chromatographic performance to determine acceptance or rejection of the acquired data.\",\n",
            "          \"LC-MS/MS peptide identification\",\n",
            "          \"Peptide identification employed the RefSeq Human protein sequence database, release version 54, and both database and peptide library search strategies. Two bovine trypsin sequences and one porcine trypsin sequence were appended to these 34,586 sequences.\",\n",
            "          \"trypsin sequence were appended to these 34,586 sequences. The June 14, 2011 NIST human spectral library for ion traps (617,000 spectra, counting paired decoys) was indexed against this sequence database. Thermo RAW files were converted to mzML peaklists\",\n",
            "          \"database. Thermo RAW files were converted to mzML peaklists by the ScanSifter algorithm\\u2009or by ProteoWizard msConvert. The ScanSifter files were employed by Pepitome 1.0.42 for spectral library search and MyriMatch 2.1.87 for database search, while the\",\n",
            "          \"search and MyriMatch 2.1.87 for database search, while the msConvert files were used by MS-GF+ v9176. Pepitome and MyriMatch employed precursor tolerances of 10\\u2009ppm, while MS-GF+ used a 20\\u2009ppm window; all three algorithms allowed fragments to vary by up\",\n",
            "          \"window; all three algorithms allowed fragments to vary by up to 0.5\\u2009m/z, and both database search engines considered semi-tryptic peptides equally with fully-tryptic peptides, allowed for isotopic error in precursor ion selection, conducted on-the-fly\",\n",
            "          \"isotopic error in precursor ion selection, conducted on-the-fly peptide sequence reversal, and applied static +57 modifications to cysteines and dynamic +16 oxidations to methionines. MS-GF+ considered acetylation for protein N-termini, while MyriMatch\",\n",
            "          \"considered acetylation for protein N-termini, while MyriMatch added pyroglutamine modifications to the N-termini of peptides starting with Gln residues. Pepitome considered any modification variants and trypsin specificities that were included in the\",\n",
            "          \"variants and trypsin specificities that were included in the spectral library.\",\n",
            "          \"Protein assembly\",\n",
            "          \"Spectral identification files from each of the search engines (pepXML for Myrimatch and Pepitome, mzIdent for MS-GF+) were converted to IDPicker 3 SQLite databases (idpDB) using IDP3. The resulting 4,275 idpDB files (95 samples\\u00d715 fractions\\u00d73 search\",\n",
            "          \"resulting 4,275 idpDB files (95 samples\\u00d715 fractions\\u00d73 search engines) were used for a final protein assembly using IDP3. For initial protein assembly, peptide identification stringency was set at a maximum of 1% reversed peptide matches, i.e., 2%\",\n",
            "          \"was set at a maximum of 1% reversed peptide matches, i.e., 2% peptide-to-spectrum matches (PSM) FDR and a minimum of 2 distinct peptides to identify a given protein within the full data set. To optimize the number of proteins identified, we applied a very\",\n",
            "          \"optimize the number of proteins identified, we applied a very stringent filter at 0.1% PSM FDR and a minimum of 2 distinct peptides identified for each protein. This filter resulted in the identification of a total of 94,442 distinct peptides among the 95\",\n",
            "          \"of a total of 94,442 distinct peptides among the 95 samples representing 7,526 protein groups with a protein-level FDR of 2.64%. To rescue high quality PSMs that were excluded by the stringent PSM FDR threshold, we relaxed the PSM FDR threshold to 1% for\",\n",
            "          \"PSM FDR threshold, we relaxed the PSM FDR threshold to 1% for the confidently identified proteins. This step increased the number of distinct peptides identified to 124,823, corresponding to a total number of 6,299,756 spectra; the rescued PSMs were of\",\n",
            "          \"a total number of 6,299,756 spectra; the rescued PSMs were of high quality, with a median PSM FDR of less than 0.2%, confirming that the integrity of the data set was maintained. To facilitate the integration between genomic and proteomic data, we further\",\n",
            "          \"the integration between genomic and proteomic data, we further assembled the peptides at the gene level. This assembly resulted in 7,211 gene groups with a gene-level FDR of 2.7%. Genes identified from each sample ranged from 3,372 to 5,456, with a median\",\n",
            "          \"from each sample ranged from 3,372 to 5,456, with a median gene count of 4,656 for the 95 samples. 1,530 genes (21%) were found in all 95 samples, 4,628 genes (64%) in more than half of the samples, and only 10 genes (0.1%) in just a single sample. In\",\n",
            "          \"samples, and only 10 genes (0.1%) in just a single sample. In addition, we created a combined protein assembly using all TCGA colorectal carcinoma and the 60 normal colon epithelium samples. This assembly identified 7,548 protein groups with a protein FDR\",\n",
            "          \"assembly identified 7,548 protein groups with a protein FDR of 2.7%. All IDP3 protein assembly files are available through ProteomeXchange (Data Citation 13) or from the CPTAC data portal (cptac-data-portal.georgetown.edu/cptacPublic).\",\n",
            "          \"Customized database construction\",\n",
            "          \"We developed an R package customProDB \\u2009to annotate variations predicted from RNA-Seq, including mapping to dbSNP135 and COSMIC64 databases. For each sample, customProDB generates a protein FASTA database by appending proteins with nonsynonymous protein\",\n",
            "          \"FASTA database by appending proteins with nonsynonymous protein coding single nucleotide variants (SNVs) and aberrant proteins to the end of the standard RefSeq Human protein sequence database. Due to the low coverage of this RNA-Seq data set, we did not\",\n",
            "          \"Due to the low coverage of this RNA-Seq data set, we did not remove the low abundance transcripts from the standard RefSeq database. Peptide identification was performed for each sample separately using the corresponding customized FASTA database with\",\n",
            "          \"using the corresponding customized FASTA database with MyriMatch 2.1.87. IDPicker 3 was produced a conjoint protein assembly across all the customized database searches, with filtering set at 1% PSM FDR and a minimum of 5 spectra identified per protein.\",\n",
            "          \"1% PSM FDR and a minimum of 5 spectra identified per protein. The full data set consisted of 8,352 protein groups with 1.8% protein FDR. Identified single amino acid variations (SAAVs) were further annotated from the previous report of the TCGA study\",\n",
            "          \"further annotated from the previous report of the TCGA study (i.e., TCGA-somatic variants), existence in the COSMIC64 database (i.e., COSMIC-supported variants), and existence in the dbSNP135 database (i.e., dbSNP-supported variants). To identify\",\n",
            "          \"dbSNP135 database (i.e., dbSNP-supported variants). To identify TCGA-somatic variants, we downloaded the MAF (Mutation Annotation Format) files from the FIREHOSE website (Data Citation 2 and Data Citation 3). Since the coordinates in MAF files were based\",\n",
            "          \"Data Citation 3). Since the coordinates in MAF files were based on hg18, liftOver (http://hgdownload.cse.ucsc.edu/admin/exe/) from UCSC was used to convert genome coordinates to hg19. All identified variant peptides as well as SAAVs and their annotations\",\n",
            "          \"variant peptides as well as SAAVs and their annotations can be found in the original publication.\",\n",
            "          \"Variant peptide identification\",\n",
            "          \"To identify variant peptides, we performed database searches using the above-constructed sample-specific customized databases as previously described. Identifying SNVs and INDELs from RNA-Seq data BAM files for 86 of the 90 patients were downloaded from\",\n",
            "          \"data BAM files for 86 of the 90 patients were downloaded from CGHub in February 2013. Although 87 samples were analyzed by RNA-Seq, we were unable to obtain the BAM file for one of the 87 samples. Tophat (v2.0.7) was used to re-align reads to human\",\n",
            "          \"87 samples. Tophat (v2.0.7) was used to re-align reads to human reference genome (hg19) in a spliced mode using default parameters, allowing a maximum of 10 hits per read. The resulting BAM files were indexed using samtools (0.1.18,\",\n",
            "          \"The resulting BAM files were indexed using samtools (0.1.18, http://samtools.sourceforge.net/). We used an in-house script to summarize the reads mapping information and calculate the exon coverage. Seventy-six percent of exons were covered by RNA-Seq\",\n",
            "          \"coverage. Seventy-six percent of exons were covered by RNA-Seq reads, and 64% had an average coverage greater than 1. Putative single nucleotide variants (SNVs) and short INDELs were called one library at a time using mpileup and varFilter from samtools.\",\n",
            "          \"library at a time using mpileup and varFilter from samtools. Putative SNVs were further filtered based on the following criteria: (i) SNP quality\\u226520; (ii) mapping quality\\u226520; and (iii) read depth\\u22653 and then put into a VCF file. For short INDEL candidates,\",\n",
            "          \"and then put into a VCF file. For short INDEL candidates, gapped reads\\u22653 were required.\",\n",
            "          \"Protein quantification\",\n",
            "          \"We used spectral count, or the total number of MS/MS spectra acquired for peptides from a given protein in a given LC/LC\\u2212MS/MS analysis, as the basis for protein quantification. Spectral count is linearly correlated with the protein abundance over a large\",\n",
            "          \"is linearly correlated with the protein abundance over a large dynamic range. This simple but practical quantification method has found broad application in detecting differential or correlated protein expression, and multiple groups have concluded that\",\n",
            "          \"protein expression, and multiple groups have concluded that spectral counting achieves similar accuracy to more complex methods such as the intensity-based techniques. Previously, we have confirmed proteomic changes detected from spectral count data by\",\n",
            "          \"proteomic changes detected from spectral count data by targeted proteomics with multiple reaction monitoring (MRM) in different data sets.\",\n",
            "          \"For the CompRef QC sample data set, spectral count data were summarized at the protein group level, where a protein group is defined as the set of proteins that are indiscernible on the basis of the observed peptides. For each group, a random protein was\",\n",
            "          \"of the observed peptides. For each group, a random protein was selected to represent the group. The spectral count table has data for 8,029 mouse/human protein groups from 32 samples, with 16 basal samples and 16 luminal samples. For the TCGA tumor data\",\n",
            "          \"basal samples and 16 luminal samples. For the TCGA tumor data set, to facilitate the integration between genomic and proteomic data, spectral count data were summarized at the gene group level, where a gene group is defined as the set of genes that are\",\n",
            "          \"where a gene group is defined as the set of genes that are indiscernible on the basis of the observed peptides. To ensure reproducibility, the longest protein was selected for each gene in the calculation of protein length. For each gene group, the gene\",\n",
            "          \"calculation of protein length. For each gene group, the gene with the shortest protein length was selected to represent the group following the Occam\\u2019s razor principle. The final spectral count Table has data for 7,211 genes and 95 samples (5 tumors have\",\n",
            "          \"Table has data for 7,211 genes and 95 samples (5 tumors have duplicated samples, Supplementary Table S1). For analyses that require only one sample from the duplicates, the sample with a larger total spectral count was selected.\",\n",
            "          \"Quality control using QuaMeter\",\n",
            "          \"The raw data for each of the 95 bRPLC experiments were subjected to quality control assessment in the QuaMeter \\u2018IDFree\\u2019 mode, producing forty-four metrics for each file along with its timestamp. Metrics included peak height and width distributions, MS\",\n",
            "          \"Metrics included peak height and width distributions, MS scan rates, precursor charge states, and total ion current distributions, and identifications were not a factor in any of the metrics. The table of metrics included 1,425 rows, one for each RAW file\",\n",
            "          \"The table of metrics included 1,425 rows, one for each RAW file (Supplementary Table S2).\",\n",
            "          \"Scripts were created in the R statistical environment for multivariate analysis of these metrics. The software performed a robust Principal Components Analysis (PCA) on the values, excluding the following fields due to insufficient variation: XIC-FWHM.Q1,\",\n",
            "          \"following fields due to insufficient variation: XIC-FWHM.Q1, XIC.FWHM.Q3, RT.Duration, and all precursor charge state fields. In PC space, the normalized Euclidean distance between pairs of RAW files was computed for the fifteen files produced for each\",\n",
            "          \"RAW files was computed for the fifteen files produced for each bRPLC. When a LC-MS/MS experiment yields a large median standard distance from the other files, that distance suggests the experiment is an outlier in QC. When all the files of a bRPLC\",\n",
            "          \"experiment is an outlier in QC. When all the files of a bRPLC experiment have high median distances, they are indicative of high variability in instrument performance for that sample.\",\n",
            "          \"Code availability\\n\\nExact copies of all open source software, customized scripts and data processing procedures are provided through the ProteomeXchange Consortium via the PRIDE partner repository as 'other' files (Data Citation 13).\\n\\nData Records\",\n",
            "          \"Data Records\\n\\nPrimary MS instrument and database search files\",\n",
            "          \"Primary files for this study are in the Thermo \\u2018RAW\\u2019 file format accessible through the ProteomeXchange repository (Data Citation 1 and Data Citation 4, Data Citation 5, Data Citation 6, Data Citation 7, Data Citation 8, Data Citation 9, Data Citation 10,\",\n",
            "          \"Citation 7, Data Citation 8, Data Citation 9, Data Citation 10, Data Citation 11, Data Citation 12, Data Citation 13 and Data Citation 14) or from the CPTAC data portal (cptac-data-portal.georgetown.edu/cptacPublic). Included are data files from the 95\",\n",
            "          \"Included are data files from the 95 TCGA carcinoma samples, the 60 normal epithelial biopsy samples and the CompRef control samples. A complete list of files is provided in Supplementary Table S3. Protein identification from these files was accomplished\",\n",
            "          \"S3. Protein identification from these files was accomplished through database searching using Myrimatch and MS-GF+, and spectral library searching using Pepitome (fully human samples only). Resulting search files are in the pepXML file format for\",\n",
            "          \"only). Resulting search files are in the pepXML file format for Myrimatch and Pepitome searches and in the mzID file format for MS-GF+ searches. Output files from all searches using the standard human RefSeq database are provided in a single compressed\",\n",
            "          \"human RefSeq database are provided in a single compressed file. Likewise, all search results from normal epithelium and search results using each of the customized databases are summarized in single compressed files. These databases are available from the\",\n",
            "          \"single compressed files. These databases are available from the CPTAC data portal. In addition to the IDPicker 3 (IDP3) protein assembly file combining search results from the original publication, we here present IDP3 files from each of the individual\",\n",
            "          \"we here present IDP3 files from each of the individual search engines and the IDP3 file containing only the normal epithelium control samples. IDP3 is an open-source protein assembly and filtering tool that queries search results summarized in a SQLite\",\n",
            "          \"tool that queries search results summarized in a SQLite database format. The protein databases against which all searches were performed are provided separately: 1) the standard human database file (RefSeq version 54), 2) the accompanying standard\",\n",
            "          \"database file (RefSeq version 54), 2) the accompanying standard spectral library file, 3) a single compressed file containing each of the 86 customized databases that include the standard search information plus all variant sequences identified in TCGA\",\n",
            "          \"information plus all variant sequences identified in TCGA RNA-seq data from the CPTAC data portal, and 4) the combined human/mouse RefSeq database. The processing and result files have been deposited to the ProteomeXchange Consortium via the PRIDE partner\",\n",
            "          \"to the ProteomeXchange Consortium via the PRIDE partner repository (Data Citation 1, Data Citation 13 and Data Citation 14).\",\n",
            "          \"Data processing strategies\",\n",
            "          \"One of the goals of bioinformatics analysis of shotgun proteomic dataset is to maximize the number of successfully identified MS/MS spectra without compromising the quality of the data. Almost all spectrum identification strategies are based on comparing\",\n",
            "          \"all spectrum identification strategies are based on comparing obtained spectra with predicted spectra from known amino acid peptide sequences (\\u2018database searching\\u2019) or with previously observed and identified spectra (\\u2018spectral library searching\\u2019) (for\",\n",
            "          \"and identified spectra (\\u2018spectral library searching\\u2019) (for review see). Over the years, several algorithms have been developed that accomplish this goal with varying levels of sophistication, and these have been incorporated into commercially available or\",\n",
            "          \"and these have been incorporated into commercially available or open-source software tools for peptide identification. For instance, most algorithms calculate from known amino acid masses where one would expect peptide fragments to appear on a MS/MS\",\n",
            "          \"where one would expect peptide fragments to appear on a MS/MS spectrum, but not all would take signal intensity into account when matching to the observed spectrum. Newer algorithms also take complementarity of the fragment ions into account (e.g.,\",\n",
            "          \"take complementarity of the fragment ions into account (e.g., MS-GF+), looking for the remaining part of the precursor mass when a given peptide fragment is observed. A commonly accepted strategy to avoid false-positive matches is to include distractor\",\n",
            "          \"to avoid false-positive matches is to include distractor (decoy) sequences in the database and to set limits on the proportion of such decoy matches in the final list of identified peptides. These distractor sequences can be generated by adding reversed\",\n",
            "          \"These distractor sequences can be generated by adding reversed amino acid sequences in the database (reverse-peptides), an approach that has the advantage of preserving the distribution of precursor masses in the (reversed) search space. The various\",\n",
            "          \"of precursor masses in the (reversed) search space. The various strategies in search algorithms cause differences in their identification profiles and it is helpful to combine search results from multiple search engines to come to an optimal annotation of\",\n",
            "          \"multiple search engines to come to an optimal annotation of a given set of MS/MS spectra. For this study, we chose three open-source search tools (Myrimatch, MS-GF+ and Pepitome) and combined their results into a single protein assembly using IDP3. Figure\",\n",
            "          \"their results into a single protein assembly using IDP3. Figure 1 shows the distribution of spectra and protein group identifications between the three different search engines. Note that even though the number of identified proteins is not greatly\",\n",
            "          \"even though the number of identified proteins is not greatly increased by combining the data, each of the search engines contributes a significant number of engine-specific spectral matches. The spectral library search engine Pepitome matches spectra to\",\n",
            "          \"The spectral library search engine Pepitome matches spectra to previously identified spectra from the library and is better at finding such known spectra than the other two search engines. This leads to higher numbers of spectral counts, but not\",\n",
            "          \"This leads to higher numbers of spectral counts, but not necessarily of distinct peptides or protein groups. On the other hand, MS-GF+ identifies a distinct group of additional peptide-spectrum matches, which leads to a large number of added counts from\",\n",
            "          \"matches, which leads to a large number of added counts from distinct peptides uniquely discovered by this search engine. These results indicate that it is important to combine results from different approaches to increase the variety and number of\",\n",
            "          \"from different approaches to increase the variety and number of identifications and thus the universe of proteins that are quantifiable by spectral counting.\",\n",
            "          \"IDP3 uses an inclusive combination algorithm whereby all PSMs that pass filtering criteria for an individual search engine are included in the assembly, using the highest scoring PSM from the search engines. IDP3 uses protein parsimony filtering to\",\n",
            "          \"the search engines. IDP3 uses protein parsimony filtering to assemble identified peptide lists into a list of non-redundant proteins (\\u2018protein groups\\u2019) as described previously. To allow easy comparison to genomic data, IDP3 layers genome data on top of\",\n",
            "          \"comparison to genomic data, IDP3 layers genome data on top of the RefSeq protein identifications, so that the main output becomes a non-redundant list of RefSeq gene IDs (\\u2018gene groups\\u2019) for which peptide evidence was obtained. The 95 TCGA colon cancer\",\n",
            "          \"which peptide evidence was obtained. The 95 TCGA colon cancer specimens yielded a dataset of 7,526 protein groups representing 7,211 gene groups.\",\n",
            "          \"Cut off points for filtering, stringencies PSM/peptide/protein FDR\",\n",
            "          \"The goal of our bioinformatics analysis of the TCGA colorectal cancer dataset was to maximize the number and diversity of identified peptides without sacrificing confidence in the obtained spectral matches. A common strategy to determine false-positive\",\n",
            "          \"spectral matches. A common strategy to determine false-positive rates in shotgun proteomics datasets is to include decoy sequences in the database. Decoy sequence identifications enable modeling of the score distribution for false matches in a given\",\n",
            "          \"modeling of the score distribution for false matches in a given dataset. The maximum allowable fraction of decoy peptide-spectrum matches in shotgun proteomic datasets is commonly set at 1 or 2%, although these are arbitrarily chosen cut-off values. In\",\n",
            "          \"or 2%, although these are arbitrarily chosen cut-off values. In addition, the requirement of having at least 2 distinct peptide sequences for the identification of a single protein can also limit the number of falsely identified proteins. However,\",\n",
            "          \"also limit the number of falsely identified proteins. However, applying these two criteria may not be sufficient to limit the number of falsely identified proteins in a dataset, as many studies do not report protein FDR values in separate calculations.\",\n",
            "          \"do not report protein FDR values in separate calculations. This is especially true with larger protein assemblies where the vast majority of unique peptide sequences have been identified already and any additional peptide identifications are therefore\",\n",
            "          \"and any additional peptide identifications are therefore more likely to be false. One strategy we employed in moderately large datasets is to increase the minimum number of matched spectra required for protein acceptance into the assembly. This strategy\",\n",
            "          \"for protein acceptance into the assembly. This strategy works well and protects against inclusion of proteins that have little spectral evidence, as well as against false-positive peptide identifications. The drawbacks of using a minimum number of\",\n",
            "          \"identifications. The drawbacks of using a minimum number of spectral counts per protein are that low-abundance proteins may be missed and that the number of identified proteins is greatly reduced. In very large datasets, such as the one reported here,\",\n",
            "          \"reduced. In very large datasets, such as the one reported here, proteins with high expression in just a few samples also may be missed. With the ever increasing size of shotgun experiments (our TCGA colorectal carcinoma assembly was based on 4,275 search\",\n",
            "          \"TCGA colorectal carcinoma assembly was based on 4,275 search result files), limiting PSM FDR and spectral counts is not sufficient to limit false-discovery rates to acceptable levels.\",\n",
            "          \"To deal with the false discovery challenge in our large dataset, we developed a new strategy based on the counter-intuitive finding that increasing the stringency of PSMs actually increased the number of proteins identified. This is illustrated in Fig. 2,\",\n",
            "          \"number of proteins identified. This is illustrated in Fig. 2, which presents the number of protein groups identified at different PSM and protein false-discovery rates at different minimum numbers of spectral counts. The lower curve shows the values\",\n",
            "          \"numbers of spectral counts. The lower curve shows the values obtained at the commonly accepted PSM value of 2%, where a minimum number of spectral counts per protein needs to be set at 175 to achieve a protein FDR of less than 5%. Because of the loss of a\",\n",
            "          \"achieve a protein FDR of less than 5%. Because of the loss of a large number of proteins with lower numbers of spectral counts, this setting also yields the lowest number of protein groups identified (~4,000). When PSM FDR rate is set more stringently,\",\n",
            "          \"identified (~4,000). When PSM FDR rate is set more stringently, lower-cut off values for the minimum number of spectral counts per protein can be selected without sacrificing protein FDR. This leads to higher numbers of identified protein groups at an\",\n",
            "          \"This leads to higher numbers of identified protein groups at an acceptable protein FDR, albeit with lower spectral counts per protein than using the more lenient PSM FDR settings.\",\n",
            "          \"Based on these considerations, we adopted a 2-step procedure, whereby we initially set a very stringent PSM FDR of 0.1%, which maximized the number of identified protein groups at an acceptable protein FDR. This defined the inventory of proteins that\",\n",
            "          \"protein FDR. This defined the inventory of proteins that could be confidently identified in this dataset. The second step was then to increase the PSM FDR 10-fold to 1% for only the protein groups already identified using the more stringent setting. Thus,\",\n",
            "          \"already identified using the more stringent setting. Thus, additional PSMs could be rescued without adversely affecting the protein FDR, which remained locked at the level from the first step of the procedure. Examination of the rescued peptides revealed\",\n",
            "          \"of the procedure. Examination of the rescued peptides revealed that the majority had PSM FDR well below acceptable levels (median 0.2%), thus demonstrating that these additional peptide matches did not compromise the accuracy of the protein\",\n",
            "          \"peptide matches did not compromise the accuracy of the protein identifications. Our analysis of this dataset illustrates challenges associated with very large proteomic studies. Although our approach adapts established target-decoy methodology, new\",\n",
            "          \"our approach adapts established target-decoy methodology, new approaches to this challenge merit further study.\",\n",
            "          \"Detection of novel single amino acid variants (SAAVs) in the dataset\",\n",
            "          \"We previously designed and implemented a new strategy for the detection of protein variant sequences by using genomic data for peptide identification. This new approach was used to leverage mRNA sequencing data for the discovery of 796 SAAVs in the TCGA\",\n",
            "          \"mRNA sequencing data for the discovery of 796 SAAVs in the TCGA colorectal carcinoma dataset. About 70% (561 of 796) of these had been previously described as single nucleotide polymorphisms (SNPs) occurring as natural variation in the human population.\",\n",
            "          \"(SNPs) occurring as natural variation in the human population. SNPs leading to amino acid changes in proteins occur with varying frequencies and are generally not associated with disease phenotypes. Of the remaining 235 SAAVs, 73 were previously described\",\n",
            "          \"Of the remaining 235 SAAVs, 73 were previously described in either the COSMIC database or tagged as somatic mutation in the genomic analysis of the TCGA dataset. Thus, the remaining 162 SAAVs detected in our analysis could not be explained by existing\",\n",
            "          \"detected in our analysis could not be explained by existing knowledge and were determined to be \\u2018new\\u2019 (Supplementary Table S4). Given the low false-discovery rate in our proteomic dataset and the fact that independent, RNAseq-based, evidence existed, the\",\n",
            "          \"the fact that independent, RNAseq-based, evidence existed, the likelihood that these new SAAVs were real findings was high. To determine the reliability of the identifications we manually examined the top SAAVs with at least two spectra identified (N=70)\",\n",
            "          \"the top SAAVs with at least two spectra identified (N=70) from the dataset using BLAST searches and matching to the most recent National Center for Biotechnology Information (NCBI) database of SNPs.\",\n",
            "          \"The results from this follow up analysis demonstrate that many of the most abundant of these SAAVs are correct assignments, which appear to be truly novel (summarized in Supplementary Table S4). The analyses also revealed apparently trivial explanations\",\n",
            "          \"S4). The analyses also revealed apparently trivial explanations for some incorrect assignments, which nevertheless are instructive. For example, peptide \\u2018INLVQR\\u2019, representing the F76I variant of DWR67 with the highest number of spectral counts (375) is\",\n",
            "          \"of DWR67 with the highest number of spectral counts (375) is an isobar, indistinguishable by mass spectrometry from the peptide \\u2018LNLVQR\\u2019 from LDHA, an abundant metabolic protein. Three other peptides had a similar L to I variation which, given the RNA-seq\",\n",
            "          \"had a similar L to I variation which, given the RNA-seq data, were likely to be true but could not be confirmed by MS because of the lack of a mass difference. There were 14 peptides that matched to SNPs not present in the original databases and 6\",\n",
            "          \"matched to SNPs not present in the original databases and 6 peptides that matched to published immunoglobulin and major histocompatability complex sequence variants. The remaining 46 of the originally selected 70 peptides appear to be truly novel. Two\",\n",
            "          \"originally selected 70 peptides appear to be truly novel. Two variant sequences each were found for 2 proteins (MYH9 and PKM), resulting in a total of 44 proteins with novel variant sequences. Future research will be needed to examine the biological\",\n",
            "          \"Future research will be needed to examine the biological relevance of each of the observed variant sequences. Furthermore, these results underscore the need to perform additional manual analyses on variant peptides identified through automated procedures\",\n",
            "          \"on variant peptides identified through automated procedures as recently reviewed by Nesvizhski. These include manual inspection for trivial explanations, comparisons to sequence databases and comparison with chromatography and spectra obtained from\",\n",
            "          \"and comparison with chromatography and spectra obtained from synthetic variant sequence peptides.\",\n",
            "          \"Germline versus somatic single amino acid variations\",\n",
            "          \"One observation we originally made on the basis of SAAV detection in 10 colorectal carcinoma cell lines was that germline variants were about twice as likely to be detectable at the protein level as somatic variants. The larger set of TCGA colorectal\",\n",
            "          \"level as somatic variants. The larger set of TCGA colorectal carcinomas provided a second confirmation of this original observation, again, the proportion of germline variant sequences observed in the mass spectrometry data was about twice as high as the\",\n",
            "          \"in the mass spectrometry data was about twice as high as the proportion of somatic variant sequences that would have been expected based on their abundance in the full dataset. These numbers are potentially influenced by the limitations of detection by\",\n",
            "          \"are potentially influenced by the limitations of detection by mass spectrometry, although those limitations apply to both types of protein variants. Somatic variants may also be present at lower levels than germline variants because they are expected to\",\n",
            "          \"levels than germline variants because they are expected to only occur in the tumor cells and not in adjacent normal cells that also contribute to the overall proteomic inventory from the TCGA tumors. It is also possible that somatic variants are only\",\n",
            "          \"TCGA tumors. It is also possible that somatic variants are only present in a sub-set of the tumor cells due to tumor heterogeneity. However, both lines of reasoning do not apply to our earlier observation in the more homogeneous colorectal carcinoma cell\",\n",
            "          \"observation in the more homogeneous colorectal carcinoma cell lines. A direct comparison of germline- and somatically-derived variant proteins will be needed to confirm the validity of our original observations.\",\n",
            "          \"Technical Validation\\n\\nSummary\",\n",
            "          \"We used three quality control strategies throughout the analysis of TCGA colorectal carcinoma samples. The first utilized a standard BSA tryptic digest after every colon carcinoma sample of 15 fractions, followed by manual inspection to assess peptide\",\n",
            "          \"15 fractions, followed by manual inspection to assess peptide peak intensities and chromatographic features. The second was achieved by analyzing a CompRef\\u2009sample after each set of 5 colorectal carcinoma samples and the third was by utilizing the\",\n",
            "          \"colorectal carcinoma samples and the third was by utilizing the identification-independent quality metric software package QuaMeter.\",\n",
            "          \"Day-to-day quality control\",\n",
            "          \"The BSA runs were evaluated by mass calibration check and using intensity and coverage of the BSA PSMs identified by database search. Mass axis shift was determined by comparing the experimental mass to the known monoisotopic mass of five different\",\n",
            "          \"mass to the known monoisotopic mass of five different peptides found in the BSA standard. These peptides are: SLHTLFGDELCK (z=3), LVNELTEFAK (z=2), HLVDEPQNLIK (z=2), YICDNQDTISSK (z=2), KVPQVSTPTLVEVSR (z=2 and z=3). If the mass axis shift exceeded\",\n",
            "          \"KVPQVSTPTLVEVSR (z=2 and z=3). If the mass axis shift exceeded 10\\u2009ppm, the instrument was recalibrated prior to any sample runs. In addition, the BSA base peak chromatogram was viewed to determine that peak shape and retention times are appropriate.\",\n",
            "          \"determine that peak shape and retention times are appropriate. Specifically, a total intensity of 107\\u2009derived from the BSA chromatogram and BSA coverage of at least 65% was required for acceptance of a set of colon carcinoma runs. Any failure to achieve\",\n",
            "          \"of a set of colon carcinoma runs. Any failure to achieve these quality control measures prompted instrument maintenance before data acquisition was allowed to continue.\",\n",
            "          \"Use of standardized CompRef sample\",\n",
            "          \"CompRef samples obtained from WHIM2 (basal type) and WHIM16 (luminal type) breast carcinoma xenografts\\u2009were alternated and analyzed after every 5 colorectal carcinoma and normal epithelium samples throughout the full analysis period of the data set (from\",\n",
            "          \"throughout the full analysis period of the data set (from July 2012 until August 2013). Each of these runs was searched using MyriMatch against a combined human/mouse RefSeq-based database and the results compared to the initial CompRef samples used to\",\n",
            "          \"and the results compared to the initial CompRef samples used to establish our shotgun proteomic platform (not published). QuaMeter was used as a tool to evaluate instrument performance for each sample run. The quality control goals for each of these new\",\n",
            "          \"sample run. The quality control goals for each of these new runs were: 1) more than 5,000 identified spectra per run, 2) third quartile MS2 density of more than 200, and 3) more than 40% of identified spectra have a charge state of 2. These metrics\",\n",
            "          \"of identified spectra have a charge state of 2. These metrics guarantee that sufficient numbers of scans are detected, that they have sufficient signal to allow identification, and that stable precursor charge state ratios were achieved.\",\n",
            "          \"One of the main reasons to include the CompRef samples in this dataset was to compare the consistency of the observed biological differences between the basal and luminal subtypes. To test the consistency of such observations, we compiled a dataset\",\n",
            "          \"the consistency of such observations, we compiled a dataset containing all 32 CompRef samples that were analyzed with the TCGA carcinomas and normal biopsies. This dataset included a total of 8,029 identified protein groups with a protein FDR of 3.7%\",\n",
            "          \"of 8,029 identified protein groups with a protein FDR of 3.7% using a combined human/mouse RefSeq database (provided as an IDP3 database available through ProteomeXchange (Data Citation 13) or from the CPTAC data portal\",\n",
            "          \"(Data Citation 13) or from the CPTAC data portal (cptac-data-portal.georgetown.edu/cptacPublic)). We compared the WHIM2 basal with the WHIM16 luminal samples using QuasiTel and selected 172 human proteins that distinguished the two xenograft types.\",\n",
            "          \"172 human proteins that distinguished the two xenograft types. Spectral counts for these proteins were at least 32-fold higher in one group over the other with an FDR- of less than 0.001. The full dataset for this analysis consisted of 5,633 proteins that\",\n",
            "          \"full dataset for this analysis consisted of 5,633 proteins that had at least 45 counts across all 32 samples, according to the 1.4 spectral count average we earlier established to be necessary for appropriate quantification by spectral counting. We then\",\n",
            "          \"for appropriate quantification by spectral counting. We then created comparisons for all 256 possible combinations of WHIM2 and WHIM16 (16\\u00d716 samples) between single pairs of the WHIM2 and WHIM16 samples and obtained spectral count ratios for all\",\n",
            "          \"and WHIM16 samples and obtained spectral count ratios for all proteins. The predictive value of the set of 172 differential proteins (the \\u2018proteomic signature\\u2019) was tested against these combinations using Receiver Operating Characteristic (ROC) analysis\",\n",
            "          \"using Receiver Operating Characteristic (ROC) analysis using \\u2018area under the curve\\u2019 (AUC) as a measure of accuracy. The median AUC for this analysis was 0.931 (+s.d.=0.028), indicating that the most prominent differential proteins are captured when\",\n",
            "          \"that the most prominent differential proteins are captured when comparing single runs from each of the CompRef subtypes (Fig. 3). This suggests that biologically relevant phenotype differences remain observable even when individual MS analysis results are\",\n",
            "          \"remain observable even when individual MS analysis results are used.\",\n",
            "          \"Use of instrument values for quality control\",\n",
            "          \"The QuaMeter performance metrics have been computed for the full set of 1425 LC-MS/MS experiments for the 95 TCGA colon samples (see Methods). The software computed quality metrics for each file that were independent of identifications from Pepitome or\",\n",
            "          \"file that were independent of identifications from Pepitome or any other search engine. These metrics enabled recognition of outliers among LC-MS/MS experiments for a particular sample and of batch effects from instrument drift over the ten-month interval\",\n",
            "          \"batch effects from instrument drift over the ten-month interval during which these experiments were conducted. The 95 samples were run sequentially and are separated into 10 \\u2018batches\\u2019 for presentation purposes, with each sample represented by fifteen\",\n",
            "          \"presentation purposes, with each sample represented by fifteen LC-MS/MS experiments. Each batch is represented by a different color in Fig. 4, with the ten samples each represented by a letter from A to J. QuaMeter reduced each LC-MS/MS experiment to a\",\n",
            "          \"from A to J. QuaMeter reduced each LC-MS/MS experiment to a vector of 44 metrics, and multivariate analysis in the R statistical environment transformed the metrics by robust Principal Components Analysis (PCA). The letters in each batch generally occupy\",\n",
            "          \"Analysis (PCA). The letters in each batch generally occupy the same region of the principal component space. The first two principal components accounted for 42.5% of the variability (Fig. 4), suggesting diversity in the forces causing variation. The\",\n",
            "          \"4), suggesting diversity in the forces causing variation. The first principal component placed the most positive loading on extracted ion chromatogram height ratios (XIC.Height.Q2, XIC.Height.Q3) and the most negative loading on the number of peaks in\",\n",
            "          \"and the most negative loading on the number of peaks in MS/MS scans (MS2.Density.Q*). Component two, on the other hand, emphasized positive loadings on the retention time interval during which the second quartile of MS scans were acquired (RT.MS.Q2) and a\",\n",
            "          \"the second quartile of MS scans were acquired (RT.MS.Q2) and a negative loading on the third quartile of MS scan intensity (MS1.TIC.Q3) and of MS scan intensity change (MS1.TIC.Change.Q3). The lack of a clear-cut source of variation indicates that\",\n",
            "          \"The lack of a clear-cut source of variation indicates that multiple, interacting components determine quality metrics stability, and thus do not point towards any particular part of the procedure that could be stabilized further to improve the quality\",\n",
            "          \"that could be stabilized further to improve the quality metrics as determined by QuaMeter.\",\n",
            "          \"Within each set of fifteen LC-MS/MS experiments, statistical scripts were used to determine the distance separating all possible pairs of the 15 fractions (Fig. 5, panel a). For most samples, the median value was close to the mean and relatively few\",\n",
            "          \"the median value was close to the mean and relatively few distances were substantially lower or higher than the median. For each tumor sample, we compared Euclidian distances of each of the 15 fractions to the sample median and used the T2\\u2009statistic in a\",\n",
            "          \"fractions to the sample median and used the T2\\u2009statistic in a Chi2-test to identify outlier values. When we set the cut-off at P<0.01, 82 of the 1425 fractions are identified as outliers (indicated in blue). Figure 5 also shows the number of identified\",\n",
            "          \"in blue). Figure 5 also shows the number of identified spectra, unique peptides and protein groups for each of the fractions. These plots show the variability within each sample and the variability between fractions. A 2-way ANOVA analysis of these\",\n",
            "          \"variability between fractions. A 2-way ANOVA analysis of these numbers showed that more than 80% of the variability was due to differences between samples, with just 3\\u20135% of the variability due to fraction and the remainder was the error component. Thus,\",\n",
            "          \"to fraction and the remainder was the error component. Thus, differences in protein composition of the various tumor samples make up the vast majority of the difference observed in spectral counts, distinct peptides and protein groups. When we compared\",\n",
            "          \"counts, distinct peptides and protein groups. When we compared the search results from the 82 fractions identified as outliers by the performance metrics, these fractions had lower identifications of spectra, peptides and proteins compared to the\",\n",
            "          \"of spectra, peptides and proteins compared to the non-outlier fractions (Fig. 6). However, individual fractions with normal or above-normal identifications were also part of this group. Thus, our conclusion from these analyses is that outliers identified\",\n",
            "          \"our conclusion from these analyses is that outliers identified with the performance metrics do not necessarily identify sub-standard fractions in this dataset, but that they may indicate sources of variability that could be better controlled in future\",\n",
            "          \"of variability that could be better controlled in future studies.\",\n",
            "          \"Usage Notes\",\n",
            "          \"To date, our compilation of TCGA colorectal carcinomas yielded the first large proteomic dataset for a human cancer. The use of alternative analysis strategies on this dataset have the potential to identify a larger number of proteins and protein forms.\",\n",
            "          \"to identify a larger number of proteins and protein forms. The datasets included with the original publication consisted of all instrument (RAW) files and the final end product files in the form of IDP3 database (idpDB) files. The pipeline that leads from\",\n",
            "          \"of IDP3 database (idpDB) files. The pipeline that leads from the source data to the final IDP3 file, described in detail under \\u2018Methods\\u2019, represents one of the possible strategies of dealing with large proteomic datasets. In this publication, we make\",\n",
            "          \"with large proteomic datasets. In this publication, we make available all intermediate files that were generated in the process of analyzing this dataset. These include the standard and individual customized FASTA databases used for protein search, the\",\n",
            "          \"customized FASTA databases used for protein search, the standard spectral library, and the search result files (pepXML/mzID) from Myrimatch, Pepitome and MS-GF+ searches. These files can serve as direct input to the various other proteomic analysis tools\",\n",
            "          \"as direct input to the various other proteomic analysis tools such as the Trans Proteomic Pipeline (TPP), or commercial packages such as Scaffold.\",\n",
            "          \"To facilitate access to the TCGA colorectal dataset we created a Colorectal Cancer Portal in NetGestalt\\u2009that encompasses data from all analyses types performed on this set of clinical specimens (http://www.netgestalt.org), including genomic, epigenomic,\",\n",
            "          \"(http://www.netgestalt.org), including genomic, epigenomic, transcriptomic, proteomic, and clinical data. The portal also includes microarray gene expression data from the Gene Expression Omnibus GEO for five CRC cohorts; mRNA expression, copy number,\",\n",
            "          \"Omnibus GEO for five CRC cohorts; mRNA expression, copy number, mutation, and drug response data for CRC cell lines from the Cancer Cell Line Encyclopedia project; and shRNA-based high-throughput screen data for CRC cell lines from the Achilles project.\",\n",
            "          \"screen data for CRC cell lines from the Achilles project. The portal provides for easy data access and visualization, data integration of proteomic data with other colorectal carcinoma data types, and biological interrogation against multiple biological\",\n",
            "          \"types, and biological interrogation against multiple biological knowledge bases (protein interaction networks, MsigDB, KEGG etc.). Examples of such analyses include the integration of all data types for a single gene or a set of genes that allow\",\n",
            "          \"all data types for a single gene or a set of genes that allow comparisons of epigenetic, copy number, mutational, mRNA expression, and protein expression data; the query of protein networks that are associated with specific sub-types of colorectal\",\n",
            "          \"that are associated with specific sub-types of colorectal carcinoma; and the prioritization of candidate drivers by integrating multiple types of data.\",\n",
            "          \"Additional Information\\n\\nHow to cite this article: Slebos, R. J. C. et al. Proteomic analysis of colon and rectal carcinoma using standard and customized databases. Sci. Data 2:150022 doi: 10.1038/sdata.2015.22 (2015).\\n\\nSupplementary Material\",\n",
            "          \"Supplementary Material\\n\\nData Citations\\n\\nThe authors declare no competing financial interests.\\n\\nComparison of identified spectra, peptides and proteins by the three search engines Myrimatch, Pepitome and MS-GF+.\",\n",
            "          \"Over 93% of the proteins in the dataset are identified by all 3 search engines, while the spectra and peptide inventories benefit more from each of the individual contributions of the different search engines. The spectral library search engine Pepitome\",\n",
            "          \"search engines. The spectral library search engine Pepitome increases the overall spectral count totals by 13% through the identification of previously observed spectra that were not identified by the other search engines. The contribution of Pepitome is\",\n",
            "          \"by the other search engines. The contribution of Pepitome is less in the area of unique peptides and proteins. MS-GF+ contributes a large fraction of all spectral counts (9.4%) and distinct peptides (16.1%).\",\n",
            "          \"Impact of peptide-to-spectrum match (PSM) false discovery rate (FDR) threshold at different levels of overall protein FDR on the total protein inventories.\",\n",
            "          \"Surprisingly, increased PSM FDR stringency (lower values) increased the number of identifiable proteins. Protein FDR was maintained below 5% by requiring a minimum number of spectral counts for each protein across the dataset. The spectral count minimum\",\n",
            "          \"for each protein across the dataset. The spectral count minimum requires to maintain a protein FDR below 5% increased with the applied PSM FDR.\",\n",
            "          \"Performance of a 172 protein signature chosen to distinguish proteomic differences between single pairs of basal and luminal breast cancer xenografts.\",\n",
            "          \"The protein signature distinguished the two breast tumor subtypes in almost all paired combinations of proteomic datasets generated for the two types of xenografts.\\n\\nPrincipal component analysis of LC-MS/MS system performance metrics.\",\n",
            "          \"Forty-four metrics from a total of 1,425 LC-MS/MS experiments were collapsed into two principal components, which accounted for 42.5% of the total variation.\",\n",
            "          \"Comparison of all TCGA samples with respect to normalized Euclidean distance based on performance metrics and numbers of filtered spectra, distinct peptides and protein groups.\",\n",
            "          \"Comparison of protein identification values between LC-MS/MS experiments classified as \\u2018outlier\\u2019 and \\u2018non-outlier\\u2019 based on performance metrics.\",\n",
            "          \"The mean values for spectral counts, peptide and protein identification were lower in the analyses classified as outliers.\"\n",
            "        ]\n",
            "      },\n",
            "      {\n",
            "        \"publication_id\": \"44e2e0ce-89c7-11e8-bcf1-0a2705229b82\",\n",
            "        \"pubmed_id\": \"25043054\",\n",
            "        \"doi\": \"10.1038/nature13438\",\n",
            "        \"author\": \"Zhang, B.; Wang, J.; Wang, X.; Zhu, J.; Liu, Q.; Shi, Z.; Chambers, M. C.; Zimmerman, L. J.; Shaddox, K. F.; Kim, S.; Davies, S. R.; Wang, S.; Wang, P.; Kinsinger, C. R.; Rivers, R. C.; Rodriguez, H.; Townsend, R. R.; Ellis, M. J.; Carr, S. A.; Tabb, D. L.; Coffey, R. J.; Slebos, R. J.; Liebler, D. C.; Carr, S. A.; Gillette, M. A.; Klauser, K. R.; Kuhn, E.; Mani, D. R.; Mertins, P.; Ketchum, K. A.; Paulovich, A. G.; Whiteaker, J. R.; Edwards, N. J.; McGarvey, P. B.; Madhavan, S.; Wang, P.; Chan, D.; Pandey, A.; Shih, I. e. M.; Zhang, H.; Zhang, Z.; Zhu, H.; Whiteley, G. A.; Skates, S. J.; White, F. M.; Levine, D. A.; Boja, E. S.; Kinsinger, C. R.; Hiltke, T.; Mesri, M.; Rivers, R. C.; Rodriguez, H.; Shaw, K. M.; Stein, S. E.; Fenyo, D.; Liu, T.; McDermott, J. E.; Payne, S. H.; Rodland, K. D.; Smith, R. D.; Rudnick, P.; Snyder, M.; Zhao, Y.; Chen, X.; Ransohoff, D. F.; Hoofnagle, A. N.; Liebler, D. C.; Sanders, M. E.; Shi, Z.; Slebos, R. J.; Tabb, D. L.; Zhang, B.; Zimmerman, L. J.; Wang, Y.; Davies, S. R.; Ding, L.; Ellis, M. J.; Townsend, R. R.\",\n",
            "        \"title\": \"Proteogenomic characterization of human colon and rectal cancer\",\n",
            "        \"journal\": \"Nature\",\n",
            "        \"journal_url\": \"https://www.nature.com/articles/nature13438\",\n",
            "        \"year\": \"2014\",\n",
            "        \"abstract\": \"Extensive genomic characterization of human cancers presents the problem of inference from genomic abnormalities to cancer phenotypes. To address this problem, we analysed proteomes of colon and rectal tumours characterized previously by The Cancer Genome Atlas (TCGA) and perform integrated proteogenomic analyses. Somatic variants displayed reduced protein abundance compared to germline variants. Messenger RNA transcript abundance did not reliably predict protein abundance differences between tumours. Proteomics identified five proteomic subtypes in the TCGA cohort, two of which overlapped with the TCGA 'microsatellite instability/CpG island methylation phenotype' transcriptomic subtype, but had distinct mutation, methylation and protein expression patterns associated with different clinical outcomes. Although copy number alterations showed strong cis- and trans-effects on mRNA abundance, relatively few of these extend to the protein level. Thus, proteomics data enabled prioritization of candidate driver genes. The chromosome 20q amplicon was associated with the largest global changes at both mRNA and protein levels; proteomics data highlighted potential 20q candidates, including HNF4A (hepatocyte nuclear factor 4, alpha), TOMM34 (translocase of outer mitochondrial membrane 34) and SRC (SRC proto-oncogene, non-receptor tyrosine kinase). Integrated proteogenomic analysis provides functional context to interpret genomic abnormalities and affords a new paradigm for understanding cancer biology.\",\n",
            "        \"full_text\": [\n",
            "          \"Proteogenomic characterization of human colon and rectal cancer\\n\\nSummary\",\n",
            "          \"We analyzed proteomes of colon and rectal tumors previously characterized by the Cancer Genome Atlas (TCGA) and performed integrated proteogenomic analyses. Somatic variants displayed reduced protein abundance compared to germline variants. mRNA\",\n",
            "          \"reduced protein abundance compared to germline variants. mRNA transcript abundance did not reliably predict protein abundance differences between tumors. Proteomics identified five proteomic subtypes in the TCGA cohort, two of which overlapped with the\",\n",
            "          \"subtypes in the TCGA cohort, two of which overlapped with the TCGA \\u201cMSI/CIMP\\u201d transcriptomic subtype, but had distinct mutation, methylation, and protein expression patterns associated with different clinical outcomes. Although copy number alterations\",\n",
            "          \"different clinical outcomes. Although copy number alterations showed strong cis- and trans-effects on mRNA abundance, relatively few of these extend to the protein level. Thus, proteomics data enabled prioritization of candidate driver genes. The\",\n",
            "          \"data enabled prioritization of candidate driver genes. The chromosome 20q amplicon was associated with the largest global changes at both mRNA and protein levels; proteomics data highlighted potential 20q candidates including HNF4A, TOMM34 and SRC.\",\n",
            "          \"potential 20q candidates including HNF4A, TOMM34 and SRC. Integrated proteogenomic analysis provides functional context to interpret genomic abnormalities and affords a new paradigm for understanding cancer biology.\",\n",
            "          \"Introduction\",\n",
            "          \"The Cancer Genome Atlas (TCGA) has characterized the genomic features of human cancers and thereby presents the challenge of explaining how genomic alterations drive cancers. Because proteins link genotypes to phenotypes, the Clinical Proteomic Tumor\",\n",
            "          \"link genotypes to phenotypes, the Clinical Proteomic Tumor Analysis Consortium (CPTAC) is performing proteomic analyses of TCGA tumor specimens for selected cancer types. Here we present the first integrated proteogenomic characterization of human cancer\",\n",
            "          \"first integrated proteogenomic characterization of human cancer with an analysis of the TCGA colorectal cancer (CRC) specimens.\",\n",
            "          \"The TCGA study affirmed well-established genomic features of CRC and described three transcriptional subtypes, 17 chromosomal regions of significant focal amplification and 28 regions of significant focal deletion, and linked genomic features of CRC to\",\n",
            "          \"focal deletion, and linked genomic features of CRC to critical signaling pathways. The drivers underlying copy number alterations (CNAs) and transcriptional subtypes are largely unknown, and an integrative analysis of both genomic and proteomic data may\",\n",
            "          \"an integrative analysis of both genomic and proteomic data may provide a more comprehensive understanding of the information flow from DNA to protein to phenotype.\",\n",
            "          \"Identification of peptides, proteins and variant sequences\",\n",
            "          \"We performed liquid chromatography-tandem mass spectrometry (LC-MS/MS)-based shotgun proteomic analyses on 95 TCGA tumor samples (Extended Data Fig. 1, Supplementary Methods 1\\u20134), the clinical and pathological characteristics and TCGA datasets for which\",\n",
            "          \"and pathological characteristics and TCGA datasets for which are summarized in Supplementary Table 1. Benchmark quality control (QC) samples from one basal and one luminal human breast tumor xenograft were analyzed in alternating order after every five\",\n",
            "          \"xenograft were analyzed in alternating order after every five CRC samples (Supplementary Methods 2).\",\n",
            "          \"We identified a total of 124,823 distinct peptides among the 95 samples, corresponding to 6,299,756 spectra in an assembly of 7,526 protein groups with a protein-level False Discovery Rate (FDR) of 2.64% (Supplementary Methods 5.1\\u20135.2, Extended Data Fig.\",\n",
            "          \"of 2.64% (Supplementary Methods 5.1\\u20135.2, Extended Data Fig. 2). To facilitate integration between genomic and proteomic data, a gene-level assembly of the peptides identified 7,211 genes.\",\n",
            "          \"A fundamental question in proteogenomics is which protein coding alterations are expressed at the protein level. Because standard database search approaches cannot identify variant pe3ptides from MS/MS data, we also performed database searches with\",\n",
            "          \"from MS/MS data, we also performed database searches with customized sequence databases from matched RNA-Seq data for individual samples (Supplementary Methods 5.3, Extended Data Fig. 3).\",\n",
            "          \"We identified 796 single amino acid variants (SAAVs) across all 86 tumors for which matched RNA-Seq data was available (Fig. 1a\\u2013b, Supplementary Table 2\\u20133), among which 64 corresponded to somatic variants reported by TCGA and 101 were reported in the\",\n",
            "          \"somatic variants reported by TCGA and 101 were reported in the COSMIC database (i.e. COSMIC-supported variants). Of the remaining SAAVs, 526 were listed in the dbSNP database (i.e. dbSNP-supported variants) and are likely to be germline variants. The 162\",\n",
            "          \"variants) and are likely to be germline variants. The 162 previously unreported SAAVs might be explained by novel somatic or germline variants, RNA editing, or, in some cases, false discovery.\",\n",
            "          \"The identified somatic variants were clearly enriched in the hypermutated samples, whereas the germline variants showed no association with hypermutation (Fig. 1a). Although 58% of the germline variants occurred in two or more samples, almost all somatic\",\n",
            "          \"variants occurred in two or more samples, almost all somatic variants occurred in only one sample (Fig. 1c). The low identification rate for somatic variants may reflect relatively low sequence coverage in shotgun proteomics; however, somatic variants\",\n",
            "          \"coverage in shotgun proteomics; however, somatic variants also might negatively impact protein abundance, possibly by reducing translational efficiency or protein stability. Using the protein abundance quantification method described below and detailed in\",\n",
            "          \"abundance quantification method described below and detailed in Supplementary Methods 5.4, we found that somatic variants exerted a significantly stronger negative impact on protein abundance than did dbSNP-supported variants (p<2.2 \\u00d7 10-16,\",\n",
            "          \"abundance than did dbSNP-supported variants (p<2.2 \\u00d7 10-16, Kolmogorov-Smirnov (KS) test, Fig. 1d, Supplementary Methods 5.5). The percentage of variants with an impact score of less than \\u22122 was doubled for somatic variants compared to dbSNP-supported\",\n",
            "          \"\\u22122 was doubled for somatic variants compared to dbSNP-supported variants (p<2.2 \\u00d7 10-16, Chi-squared test, Fig. 1d).\",\n",
            "          \"Cancer-related variant proteins may serve as candidate protein biomarkers or therapeutic targets. The 108 somatic or COSMIC-supported protein variants mapped to 105 genes, including known cancer genes in the Cancer Gene Census database such as KRAS,\",\n",
            "          \"cancer genes in the Cancer Gene Census database such as KRAS, CTNNB1, SF3B1, ALDH2, and FH. The list also included 14 targets of FDA-approved drugs or drugs in clinical trials, such as ALDH2, HSD17B4, PARP1, P4HB, TST, GAK, SLC25A24, and SUPT16H. A subset\",\n",
            "          \"HSD17B4, PARP1, P4HB, TST, GAK, SLC25A24, and SUPT16H. A subset of variant peptide sequences, including K-ras (G12D) were verified by targeted analyses of tumor lysates spiked with synthetic, isotope-labeled peptide standards (Supplementary Methods 6).\",\n",
            "          \"isotope-labeled peptide standards (Supplementary Methods 6). One example is shown in Extended Data Fig. 4.\",\n",
            "          \"Quantification of protein abundance\",\n",
            "          \"To quantify protein abundance, we used spectral counts, which are the total number of MS/MS spectra acquired for peptides from a given protein (Supplementary Methods 5.4, Supplementary Table 4). Analysis of data from benchmark QC samples demonstrated\",\n",
            "          \"4). Analysis of data from benchmark QC samples demonstrated platform reproducibility throughout the analyses and enabled evaluation of data normalization methods (Extended Data Fig. 5a\\u2013b). Based on the minimal spectral count requirement established using\",\n",
            "          \"on the minimal spectral count requirement established using the QC data set (Extended Data Fig. 5c), 3,899 genes with a protein-level FDR of 0.43% were used to compare relative protein abundance across tumor samples.\",\n",
            "          \"mRNA abundance does not reliably predict protein abundance\",\n",
            "          \"The matched proteomic and RNA-Seq measurements from the TCGA CRC tumors allowed the first global analysis of transcript-protein relationships in a large human tumor cohort (Supplementary Methods 7). First, we compared the steady state mRNA and protein\",\n",
            "          \"7). First, we compared the steady state mRNA and protein abundance for each gene within individual samples (Supplementary Methods 7.2\\u20137.3, Extended Data Fig. 6a). All samples showed significant positive mRNA-protein correlation (multiple-test adjusted p\",\n",
            "          \"positive mRNA-protein correlation (multiple-test adjusted p value < 0.01, Spearman\\u2019s correlation coefficient) and the average correlation between steady state mRNA and protein abundance in individual samples was 0.47 (Fig. 2a), which is comparable to\",\n",
            "          \"individual samples was 0.47 (Fig. 2a), which is comparable to previous reports in multi-cellular organisms.\",\n",
            "          \"Next, we examined the concordance between mRNA and protein variation of individual genes across the 87 tumors for which 3,764 genes had both mRNA and protein measurements suitable for relative abundance comparison (Supplementary Methods 7.2, 7.4).\",\n",
            "          \"relative abundance comparison (Supplementary Methods 7.2, 7.4). Although 89% of the genes showed a positive mRNA-protein correlation, only 32% had statistically significant correlations (Fig. 2b). The average Spearman\\u2019s correlation between mRNA and\",\n",
            "          \"(Fig. 2b). The average Spearman\\u2019s correlation between mRNA and protein variation was 0.23, which was comparable to reported values for yeast, mouse and human cell lines.\",\n",
            "          \"To test whether the concordance between protein and mRNA variation is related to the biological function of the gene product, we performed KEGG enrichment analysis (Supplementary Methods 7.5, Supplementary Table 5). Genes involved in several metabolic\",\n",
            "          \"Supplementary Table 5). Genes involved in several metabolic processes showed concordant mRNA and protein variation, whereas other gene classes showed low or even negative concordance in mRNA and protein variation (Figure 2c). We also found that genes with\",\n",
            "          \"protein variation (Figure 2c). We also found that genes with stable mRNA and stable protein tend to have higher mRNA-protein correlation than those with unstable mRNA and unstable protein (p = 5.27 \\u00d7 10-6, two-sided Wilcoxon rank-sum test, Supplementary\",\n",
            "          \"= 5.27 \\u00d7 10-6, two-sided Wilcoxon rank-sum test, Supplementary Methods 7.6, Extended Data Fig. 6b). mRNA measurements thus are poor predictors of protein abundance variations and both biological functions of the gene products and mRNA and protein\",\n",
            "          \"biological functions of the gene products and mRNA and protein stability may govern mRNA-protein correlation.\",\n",
            "          \"Impact of copy number alterations on mRNA and protein abundance\",\n",
            "          \"The TCGA study identified 17 regions of significant focal amplification and 28 regions of significant focal deletion. We examined the impact of CNAs on mRNA and protein abundance, including both cis-effects on the abundance of genes in the same loci and\",\n",
            "          \"both cis-effects on the abundance of genes in the same loci and trans-effects on the abundance of genes at other loci in the genome (Supplementary Methods 8).\",\n",
            "          \"For all 23,125 genes with a CNA measurement in the TCGA data set, we calculated Spearman\\u2019s correlation with mRNA and protein abundance, respectively for the 3,764 genes with both mRNA and protein measurements (Supplementary Methods 8.1). Examination of\",\n",
            "          \"measurements (Supplementary Methods 8.1). Examination of the matrix visualizing significant CNA-mRNA correlations (multiple-test adjusted p value < 0.01) revealed strong positive correlations along the diagonal (Fig. 3a), suggesting strong cis-effects of\",\n",
            "          \"along the diagonal (Fig. 3a), suggesting strong cis-effects of CNAs on mRNA abundance. Most of the diagonal signals corresponded to previously reported arm-level changes. In contrast, the diagonal pattern was much weaker for CNA-protein correlations (Fig.\",\n",
            "          \"pattern was much weaker for CNA-protein correlations (Fig. 3b).\",\n",
            "          \"To further investigate the cis-effects of CNAs, we separated all genes with CNA, mRNA, and protein measurements into those in focal amplification regions, focal deletion regions, and non-focal regions (i.e., chromosomal regions without focal amplification\",\n",
            "          \"regions (i.e., chromosomal regions without focal amplification or deletion). As shown in Extended Data Figure 7, CNA-mRNA correlations were significantly higher than CNA-protein correlations for genes in all three groups (p<1.0 \\u00d7 10-10, KS test).\",\n",
            "          \"for genes in all three groups (p<1.0 \\u00d7 10-10, KS test). Moreover, genes in the focal amplification regions showed significantly higher CNA-mRNA and CNA-protein correlations than genes in the non-focal regions (p=4.4 \\u00d7 10-4 and 0.02, respectively, KS\",\n",
            "          \"the non-focal regions (p=4.4 \\u00d7 10-4 and 0.02, respectively, KS test). However, the same trend was not observed for genes in the focal deletion regions. Therefore, focal amplifications have the strongest cis-effects on both mRNA and protein abundance,\",\n",
            "          \"the strongest cis-effects on both mRNA and protein abundance, suggesting that selection for high protein abundance may drive CNA in regions of focal amplification. On the other hand, many CNA-driven mRNA level increases do not translate into increased\",\n",
            "          \"CNA-driven mRNA level increases do not translate into increased abundance of the corresponding proteins.\",\n",
            "          \"Figure 3a\\u2013b also revealed multiple trans-acting CNA hot spots, defined as chromosomal loci whose amplification is significantly associated with abundance changes of many transcripts or proteins at other loci. Chromosomes 20q, 18, 16, 13 and 7 contained\",\n",
            "          \"at other loci. Chromosomes 20q, 18, 16, 13 and 7 contained the five strongest hot spots driving global mRNA abundance variation. These hot spots also were strongest at the protein level. Most hot spot-related transcript changes did not propagate to the\",\n",
            "          \"hot spot-related transcript changes did not propagate to the protein level, presumably reflecting buffering of protein abundance by post-transcriptional regulation. Notably, many hot spot-associated protein level alterations occurred in the absence of\",\n",
            "          \"protein level alterations occurred in the absence of corresponding mRNA alterations, suggesting that the same trans-acting hot spot may exert independent effects at both the transcriptome and proteome levels.\",\n",
            "          \"The 20q amplification was associated with the largest global changes in both mRNA and protein levels in this univariate analysis. The same conclusion was reached with a regularized multivariate regression analysis method, remMap (Supplementary Methods\",\n",
            "          \"regression analysis method, remMap (Supplementary Methods 8.2, Supplementary Table 6\\u20139). These data highlight the importance of 20q amplification in CRC, which has not been well documented in previous studies. Among the 79 genes in the 20q region with\",\n",
            "          \"in previous studies. Among the 79 genes in the 20q region with quantifiable protein measurements, 67 (85%) showed significant CNA-mRNA correlation, but only 40 (51%) showed significant CNA-protein correlation (multiple-test adjusted p value < 0.01,\",\n",
            "          \"CNA-protein correlation (multiple-test adjusted p value < 0.01, Spearman\\u2019s correlation coefficient, Supplementary Table 10).\",\n",
            "          \"Because significant CNA-protein correlations identify amplified sequences that translate to high protein abundance, proteomic measurements can help prioritize genes in amplified regions for further examination. Of particular interest among the 40 genes is\",\n",
            "          \"examination. Of particular interest among the 40 genes is HNF4A (Fig. 3c), a candidate driver gene nominated by TCGA for the 20q13.12 focal amplification peak. HNF4\\u03b1 is a transcription factor with a key role in normal gastrointestinal development and is\",\n",
            "          \"with a key role in normal gastrointestinal development and is increasingly being linked to CRC. However, there are contradictory reports on whether HNF4\\u03b1 acts as an oncogene or a tumor suppressor gene in CRC. Upon reanalysis of the HNF4A shRNA knockdown\",\n",
            "          \"gene in CRC. Upon reanalysis of the HNF4A shRNA knockdown data for CRC cell lines from the Achilles project, we found that the dependency of CRC cells on HNF4\\u03b1 correlated significantly with the amplification level of HNF4A (Supplementary Methods 8.3,\",\n",
            "          \"the amplification level of HNF4A (Supplementary Methods 8.3, Extended Data Fig. 8), which may partially explain the contradictory roles reported for HNF4\\u03b1 in CRC. Other interesting candidates included TOMM34 (Fig. 3d), which is over-expressed frequently\",\n",
            "          \"included TOMM34 (Fig. 3d), which is over-expressed frequently in CRC tumors and is involved in the growth of CRC cells, and SRC (Fig. 3e), which encodes a non-receptor tyrosine kinase implicated in several human cancers including CRC.\",\n",
            "          \"Proteomic subtypes of CRC\",\n",
            "          \"The TCGA study reported three transcriptomic subtypes of CRC, designated \\u201cMSI/CIMP\\u201d (microsatellite instability/CpG island methylator phenotype), \\u201cInvasive\\u201d, and \\u201cCIN\\u201d (chromosomal instability). Given the limited correlation between mRNA and protein\",\n",
            "          \"Given the limited correlation between mRNA and protein levels, we asked whether CRC subtypes can be better represented with proteomics data. Using the Consensus Clustering method (Supplementary Methods 9.1\\u20139.2, Extended Data Fig. 9), we identified five\",\n",
            "          \"Methods 9.1\\u20139.2, Extended Data Fig. 9), we identified five major proteomic subtypes in this tumor cohort, with 15, 9, 25, 11, and 19 cases in subtypes A through E, respectively (Fig. 4a\\u2013b).\",\n",
            "          \"We tested the association between the subtype classification and established genomic and epigenomic features of CRC using Fisher\\u2019s exact test (Fig. 4c, Supplementary Table 11). Almost all hypermutated and MSI-high tumors were included in subtypes B and C,\",\n",
            "          \"and MSI-high tumors were included in subtypes B and C, as well as tumors with POLE and BRAF mutations. However, statistically significant association with these features was only observed for subtype B (multiple-test adjusted p value < 0.05). Moreover,\",\n",
            "          \"subtype B (multiple-test adjusted p value < 0.05). Moreover, whereas subtype B was significantly associated with the TCGA CIMP-H (CIMP-high) methylation subtype, subtype C was significantly associated with a non-CIMP subtype (cluster 4). Another unique\",\n",
            "          \"associated with a non-CIMP subtype (cluster 4). Another unique feature of subtype B was the lack of TP53 mutations and chromosome 18q loss. These results clearly established the association between proteomic subtype B and MSI-High and CIMP, but suggest\",\n",
            "          \"between proteomic subtype B and MSI-High and CIMP, but suggest that subtype C might have different biological underpinnings.\",\n",
            "          \"The remaining three subtypes were associated with CIN, another well-accepted genetic property of CRC. In particular, subtype E was significantly associated with both TP53 mutations and 18q loss, genomic features frequently associated with CIN tumors.\",\n",
            "          \"loss, genomic features frequently associated with CIN tumors. Interestingly, subtype E was also associated with HNF4A amplification and relatively higher abundance of HNF4\\u03b1 protein (Fig. 4d). HNF4\\u03b1 abundance was significantly higher in subtype E tumors\",\n",
            "          \"HNF4\\u03b1 abundance was significantly higher in subtype E tumors compared to normal colon samples (multiple-test adjusted p value = 1.09 \\u00d7 10-6, two-sided Wilcoxon rank-sum test); however, significant up-regulation of HNF4\\u03b1 was not observed for other subtypes\",\n",
            "          \"up-regulation of HNF4\\u03b1 was not observed for other subtypes (Supplementary Methods 10). This result, together with our reanalysis of shRNA knockdown data from the Achilles project (Extended Data Fig. 8), suggest that HNF4\\u03b1 dependency might be particularly\",\n",
            "          \"Fig. 8), suggest that HNF4\\u03b1 dependency might be particularly associated with the subset of tumors or cells with HNF4A amplification.\",\n",
            "          \"We also examined the association between the subtype classification and clinical features and found only that stage II tumors were significantly enriched in subtype C (multiple-test adjusted p value < 0.05, Supplementary Table 11). Supervised statistical\",\n",
            "          \"p value < 0.05, Supplementary Table 11). Supervised statistical analyses at the individual protein level for 13 clinical and genomic features also identified few, if any significant protein effects of these features, except for hypermutation status, MSI\",\n",
            "          \"effects of these features, except for hypermutation status, MSI status, and 18q loss (Supplementary Table 12), suggesting that the proteomic subtypes identified by the unsupervised clustering analysis captured the major proteome variations across the\",\n",
            "          \"analysis captured the major proteome variations across the tumors.\",\n",
            "          \"Next, we compared the proteomic subtype classification with the TCGA transcriptional subtype classification for the 62 samples that had both subtype labels. Proteomic subtypes B and C both showed significant association with the TCGA subtype \\u201cMSI/CIMP\\u201d\",\n",
            "          \"showed significant association with the TCGA subtype \\u201cMSI/CIMP\\u201d (Fig. 4b, Supplementary Table 11); however, they differ considerably at genomic, epigenomic, and proteomic levels (Fig. 4b). We also examined alternative classifications of the TCGA samples\",\n",
            "          \"also examined alternative classifications of the TCGA samples based on two recently published transcriptomic subtype classifiers. Proteomic subtype C, but not subtype B, showed enriched overlap with the \\u201cStem-like\\u201d subtype described by Sadanandam et al.\",\n",
            "          \"with the \\u201cStem-like\\u201d subtype described by Sadanandam et al. and the \\u201cCCS3\\u201d subtype described by De Sousa et al.. Interestingly, tumors with \\u201cStem-like\\u201d and \\u201cCCS3\\u201d classifications both have poor prognosis, which suggests that proteome subtype C also may be\",\n",
            "          \"prognosis, which suggests that proteome subtype C also may be associated with poor prognosis. Therefore, the ability to distinguish subtype B from C through proteomics data is important, because MSI-High tumors typically have better prognosis.\",\n",
            "          \"Protein signatures and networks associated with proteomic subtypes\",\n",
            "          \"To better understand the biology underlying the proteomic subtypes, we identified protein signatures for each subtype by supervised comparison of protein abundance in that subtype against all others; we also required signature proteins for a subtype to be\",\n",
            "          \"others; we also required signature proteins for a subtype to be significantly different in abundance compared to normal colon samples from 30 individuals analyzed on the same proteome analysis platform (Supplementary Methods 10, Supplementary Table\",\n",
            "          \"platform (Supplementary Methods 10, Supplementary Table 13\\u201314). As shown in Extended Data Figure 10a, all CRC subtypes displayed more than 2,000 (>60%) significant protein abundance differences compared to normal colon. Although a full validation of the\",\n",
            "          \"compared to normal colon. Although a full validation of the proteomic subtypes and protein signatures for the subtypes will require proteomic profiling data from an independent tumor cohort, a low cross-validation error rate of 3.8% demonstrated good\",\n",
            "          \"a low cross-validation error rate of 3.8% demonstrated good generalizability of the subtypes and their signature proteins (Supplementary Methods 11).\",\n",
            "          \"We performed Gene Ontology enrichment analysis for the subtype signatures using WebGestalt (Supplementary Methods 11, Supplementary Table 15). Genes involved in \\u201cresponse to wounding\\u201d were significantly enriched in the up-signature of subtype C\",\n",
            "          \"were significantly enriched in the up-signature of subtype C (multiple-test adjusted p value < 2.2 \\u00d7 10-16, Fisher\\u2019s exact test). The wound-response gene signature is a powerful predictor of poor clinical outcome in patients with early stage breast\",\n",
            "          \"of poor clinical outcome in patients with early stage breast cancers. This result further links our subtype C to poor prognosis.\",\n",
            "          \"To better understand the functional networks underlying this subtype with potential clinical importance, we uploaded the up- and down-signatures of subtype C to NetGestalt for enriched protein-protein interaction network module analysis. Four network\",\n",
            "          \"interaction network module analysis. Four network modules were enriched with genes in the up-signature for subtype C, whereas two modules were enriched with genes in the down-signature (multiple-test adjusted p value < 0.01, Fisher\\u2019s exact test, Extended\",\n",
            "          \"adjusted p value < 0.01, Fisher\\u2019s exact test, Extended Data Fig. 10b). Notably, the down-signature enriched module (III) included the E-cadherin (CDH1)-\\u03b2-catenin (CTNNB1)-\\u03b1-catenin (CTNNA1) complex (Extended Data Fig. 10c, 10e). E-cadherin, the most\",\n",
            "          \"complex (Extended Data Fig. 10c, 10e). E-cadherin, the most under-expressed protein in the sub-network, suppresses invasion in lobular breast carcinoma and is a switch for the epithelial-to-mesenchymal transition (EMT), which is associated with poor\",\n",
            "          \"transition (EMT), which is associated with poor prognosis in colon cancer. Other components of the module were desmosomal proteins (PKP2, JUP and DSG2) and cytokeratins (KRT18, KRT6A and KRT8). Reduction in both desmosome formation and cytokeratin\",\n",
            "          \"KRT8). Reduction in both desmosome formation and cytokeratin expression is associated with EMT. Moreover, proteins in the most significantly up-regulated network module (Extended Data Fig. 10d, 10f) included collagens (COL1A1 and COL3A1) and extracellular\",\n",
            "          \"10f) included collagens (COL1A1 and COL3A1) and extracellular matrix glycoproteins (FN1, BGN, FBN1, and FBN2) that also are markers of EMT. These data strengthen the association of subtype C with poor prognosis and relate it to EMT activation.\",\n",
            "          \"Discussion\",\n",
            "          \"Our proteomic characterization of the genomically-annotated TCGA colon tumors illustrates the power of integrated proteogenomic analysis. The data demonstrate that protein abundance cannot be reliably predicted from DNA or RNA-level measurements. mRNA and\",\n",
            "          \"reliably predicted from DNA or RNA-level measurements. mRNA and protein levels were modestly correlated, as earlier cell and animal model studies suggest, but over two thirds of these correlations were not statistically significant in the TCGA tumor set.\",\n",
            "          \"were not statistically significant in the TCGA tumor set. Although most CNAs in CRC drive mRNA abundance changes, relatively few translated to consistent changes in protein abundance.\",\n",
            "          \"Genomic and proteomic technologies provide reinforcing data. RNA-Seq data facilitates the discovery of variant proteins, which could serve as possible biomarker candidates or therapeutic targets. Combined mRNA and protein profiling data can identify\",\n",
            "          \"targets. Combined mRNA and protein profiling data can identify potentially relevant genes in amplified chromosomal regions. This approach, which revealed the importance of chromosome 20q amplification and provided new insights into the role of HNF4\\u03b1 in\",\n",
            "          \"and provided new insights into the role of HNF4\\u03b1 in CRC, can be broadly extended to understand roles of CNAs in other cancers. Proteomics identified CRC subtypes similar to those detectable by transcriptome profiles, but further captured features not\",\n",
            "          \"by transcriptome profiles, but further captured features not detectable in transcript profiles. The separation of the TCGA \\u201cMSI/CIMP\\u201d subtype into distinct proteotypes illustrates the unique potential of proteomics-based subtyping. After validation in\",\n",
            "          \"potential of proteomics-based subtyping. After validation in independent cohorts, protein subtype signatures could be directly translated into laboratory tests for tumor classification. Integrated proteogenomic analysis, as demonstrated in this study,\",\n",
            "          \"proteogenomic analysis, as demonstrated in this study, will enable new advances in cancer biology, diagnostics and therapeutics.\",\n",
            "          \"Methods Summary\",\n",
            "          \"All tumor samples for the current study were obtained through the TCGA Biospecimen Core Resource (BCR) as described previously. No other selection criteria other than availability were applied for this study. Patient-derived xenograft tumors from\",\n",
            "          \"applied for this study. Patient-derived xenograft tumors from established Basal and Luminal-B breast cancer intrinsic subtypes  were raised subcutaneously in 8 week old NOD.Cg-Prkdcscid Il2rgtm1Wjl/SzJ mice (Jackson Labs, Bar Harbor, Maine) as previously\",\n",
            "          \"mice (Jackson Labs, Bar Harbor, Maine) as previously described. Normal colon biopsies were obtained from screening colonoscopies performed between July 2006 and October 2010 under Vanderbilt University IRB approval #061096.\",\n",
            "          \"Tissue proteins were extracted and tryptic peptide digests were analyzed by multidimensional liquid chromatography-tandem mass spectrometry. Xenograft QC samples were run after every 5 colorectal tumor samples. Raw data were processed for peptide\",\n",
            "          \"5 colorectal tumor samples. Raw data were processed for peptide identification by database and spectral library searching and identified peptides were assembled as proteins and mapped to gene identifiers for proteogenomic comparisons. Quantitative\",\n",
            "          \"to gene identifiers for proteogenomic comparisons. Quantitative proteomic comparisons were based on spectral count data. Detailed descriptions of the samples, LC-MS/MS analysis, and data analysis methods can be found in Supplementary Methods. All of the\",\n",
            "          \"methods can be found in Supplementary Methods. All of the primary mass spectrometry data on TCGA tumor samples are deposited at the CPTAC Data Coordinating Center as raw and mzML files and complete protein assembly datasets for public access\",\n",
            "          \"files and complete protein assembly datasets for public access (https://cptac-data-portal.georgetown.edu).\",\n",
            "          \"Extended Data\\n\\nMass spectrometry (MS)-based proteomics workflow\",\n",
            "          \"Protein was extracted from frozen tumor tissue and used to generate tryptic digests. The resulting tryptic peptides were fractionated using off-line basic-reverse phase high-pressure liquid chromatography (bRPLC). Collected fractions were pooled and used\",\n",
            "          \"(bRPLC). Collected fractions were pooled and used for reverse phase HPLC in-line with a Thermo Orbitrap-Velos MS instrument. Raw data was processed by MSConvert and then used for database and spectral library searching using three different search engines\",\n",
            "          \"spectral library searching using three different search engines (Myrimatch, Pepitome and MS-GF+). Identified peptides were assembled using IDPicker 3 with selected filters as described in the methods. IDPicker 3 stores its protein assemblies for a\",\n",
            "          \"in the methods. IDPicker 3 stores its protein assemblies for a specified set of filters in the idpDB format. These SQLite databases associate spectra with peptides, peptides with proteins, and LC-MS/MS experiments with a hierarchy of experiments.\",\n",
            "          \"Relaxing the false discovery rate (FDR) of peptide-spectrum match (PSM) for high-confident proteins increases spectral counts\",\n",
            "          \"To increase spectral counts and improve statistical comparisons, we first created a protein assembly that maximized the number of proteins identified (at 0.1% PSM FDR) and then relaxed the PSM FDR to 1% exclusively for the set of confidently identified\",\n",
            "          \"PSM FDR to 1% exclusively for the set of confidently identified proteins. This strategy led to increased spectral counts from 4,896,831 to 6,299,756, a 29% increase. a, Spectral count plot of all 7,526 confidently identified proteins demonstrates the\",\n",
            "          \"of all 7,526 confidently identified proteins demonstrates the increase in the absolute number of spectra identified for each protein, but no decrease for any of the proteins. Each dot in the figure represents one of the 7,526 proteins; x-axis and y-axis\",\n",
            "          \"figure represents one of the 7,526 proteins; x-axis and y-axis represent the spectral counts obtained in the data sets with 0.1% and 1% PSM FDR, respectively, both plotted on a log scale. b, Density plot showing the distribution of PSM FDR scores for all\",\n",
            "          \"Density plot showing the distribution of PSM FDR scores for all rescued PSMs. Rescued PSMs are of high quality with a median PSM FDR score of less than 0.2%, indicating the maintained integrity of the data set.\",\n",
            "          \"Reads mapping, exon coverage and missense somatic variants in RNA-Seq data\",\n",
            "          \"a, Summary of total RNA-Seq read counts and mapping results for individual samples. b, Distribution of percentage sequence coverage in exons for individual samples. Among all 228,157 exons, 76% were expressed, but only 64% had an average coverage greater\",\n",
            "          \"were expressed, but only 64% had an average coverage greater than 1. Exons with no coverage were not included in the box plots. c, Number of missense somatic variants detected by RNA-Seq in individual samples. Approximately 54% of the mutation positions\",\n",
            "          \"individual samples. Approximately 54% of the mutation positions were covered by RNA-Seq reads and only 43% were covered by three or more reads.\",\n",
            "          \"PRM (Parallel reaction monitoring) validation results\",\n",
            "          \"a, PRM data for the variant sequence LVVVGADGVGK (KRAS G12D in TCGA-AA-3818). b, PRM data for the variant sequence LVVVGADGVGK (KRAS G12D in TCGA-AG-A00Y). c, PRM data for the variant sequence TPVLFDVYEIK (ANXA11 I278V in TCGA-AF-3400). d, PRM data for\",\n",
            "          \"TPVLFDVYEIK (ANXA11 I278V in TCGA-AF-3400). d, PRM data for the variant sequence DLEDLFFK (SRSF9 Y35F in TCGA-AA-A01P). Single amino acid variants (SAAVs) identified in the TCGA shotgun data set were validated using PRM analyses. Three distinct SAAVs in\",\n",
            "          \"set were validated using PRM analyses. Three distinct SAAVs in four TCGA samples were selected for validation. The TCGA samples were freshly prepared in the same manner as the original samples analyzed by shotgun proteomics. Each sample was spiked with\",\n",
            "          \"analyzed by shotgun proteomics. Each sample was spiked with 12.5 fmol/\\u03bcL of a mixture of all isotopically labeled peptides. Using an inclusion list containing the precursor m/z values representing both unlabeled (endogenous) and labeled peptides, each\",\n",
            "          \"both unlabeled (endogenous) and labeled peptides, each fraction was analyzed by PRM for the variant peptides. For each variant shown in a\\u2013d, the top MS/MS spectra display represents the spectrum identified in the initial shotgun analyses of the TCGA\",\n",
            "          \"spectrum identified in the initial shotgun analyses of the TCGA samples. The two annotated spectra shown below the original spectra represent the MS/MS of the unlabeled endogenous variant peptide and the spiked respective labeled peptide in the PRM\",\n",
            "          \"peptide and the spiked respective labeled peptide in the PRM analysis of the TCGA sample, respectively. The chromatographic traces show the overlapping transitions and retention time of both the endogenous and labeled variant peptide, respectively.\",\n",
            "          \"Platform evaluation and analysis method selection using quality control (QC) samples\",\n",
            "          \"a, The lower-left half (uncolored) depicts pair-wise scatter plots of the samples, with x- and y-axes representing quantile-normalized spectral counts for samples in corresponding columns and rows, respectively. The upper-right half (red colored) depicts\",\n",
            "          \"rows, respectively. The upper-right half (red colored) depicts pair-wise Spearman\\u2019s correlation coefficients for the same comparisons. b, For each normalization method (none, global, quantile, and NSAF), we calculated the intraclass correlation\",\n",
            "          \"quantile, and NSAF), we calculated the intraclass correlation coefficients (ICCs) for individual proteins in the QC data set. The analysis was done for the top 1000, 500, or 100 proteins with the largest variance and the cumulative fraction curves were\",\n",
            "          \"the largest variance and the cumulative fraction curves were plotted. In most scenarios, quantile normalization generated slightly higher ICC scores than global normalization, and both methods clearly outperformed the NSAF normalization. c, We sorted all\",\n",
            "          \"clearly outperformed the NSAF normalization. c, We sorted all proteins in the QC data set based on their total spectral counts and then divided the proteins into 10 bins with equal number of proteins. Average spectral count ranges for each bin are shown\",\n",
            "          \"proteins. Average spectral count ranges for each bin are shown in the brackets in the legend box. For each bin, we calculated the ICCs for individual proteins in the bin. The analysis was done for the top 300, 200, or 100 proteins with the largest\",\n",
            "          \"was done for the top 300, 200, or 100 proteins with the largest variance in each bin. The cumulative fraction curves were plotted. Protein bins with spectral counts less than 1.4 showed clearly lower ICC scores, whereas the ICC score curves started to\",\n",
            "          \"lower ICC scores, whereas the ICC score curves started to converge when the average spectral count was greater than 1.4.\",\n",
            "          \"Extended data for mRNA-protein correlation analysis\",\n",
            "          \"a, Evaluation of the length-bias in different RNA-Seq-based gene abundance estimation methods. The plot shows the distribution of correlation between gene length and estimated transcript abundance based on FPKM (Fragments Per Kilobase of exon per Million\",\n",
            "          \"based on FPKM (Fragments Per Kilobase of exon per Million fragments mapped, blue curve) and RSEM (RNA-Seq Expectation Maximization, red curve), respectively. FPKM measure is independent of gene length, whereas the RSEM measure strongly correlates with\",\n",
            "          \"gene length, whereas the RSEM measure strongly correlates with gene length. b, Relationship between mRNA-protein correlation and the stability of the molecules. Human genes were separated into four categories based on the mRNA and protein half-lives of\",\n",
            "          \"four categories based on the mRNA and protein half-lives of their mouse orthologs: stable mRNA/stable protein; stable mRNA/unstable protein, unstable mRNA/stable protein, and unstable mRNA/unstable protein. Distribution of mRNA-protein correlations for\",\n",
            "          \"protein. Distribution of mRNA-protein correlations for genes in each category was plotted in the box plots. Genes with stable mRNA and stable protein showed relatively higher mRNA-protein correlation whereas those with unstable mRNA and unstable protein\",\n",
            "          \"whereas those with unstable mRNA and unstable protein showed relatively lower mRNA-protein correlation. Only common genes in both our study and the mouse study were included in the analysis. The total number of genes in each category (N) is labeled in the\",\n",
            "          \"total number of genes in each category (N) is labeled in the figure. The p value indicating correlation difference among the four categories was calculated based on the Kruskal-Wallis non-parametric ANOVA test. The p value indicating correlation\",\n",
            "          \"non-parametric ANOVA test. The p value indicating correlation difference between the stable mRNA/stable protein group and the unstable mRNA/unstable protein group was calculated based on the two-sided Wilcoxon rank-sum test.\",\n",
            "          \"mRNA and protein-level cis-effect of copy number alterations (CNAs) in focal amplification, focal deletion and non-focal regions\",\n",
            "          \"The figure plots cumulative fraction curves of CNA-mRNA (dashed lines) and CNA-protein (solid lines) expression correlations for genes in the focal amplification regions (red), focal deletion regions (green), and non-focal regions (blue), respectively.\",\n",
            "          \"regions (green), and non-focal regions (blue), respectively. Focal amplification regions were defined in the TCGA study. Any chromosomal regions outside the focal amplification and deletion regions were considered as non-focal regions. CNA-mRNA\",\n",
            "          \"deletion regions were considered as non-focal regions. CNA-mRNA correlations were significantly higher than CNA-protein correlations for genes in any of the three groups. Moreover, genes in the focal amplification regions showed the highest level of\",\n",
            "          \"in the focal amplification regions showed the highest level of CNA-mRNA and CNA-protein correlations among the three groups of genes. P values were based on the two-sided kolmogorov-smirnov test.\",\n",
            "          \"HNF4\\u03b1 isoforms and the effect of HNF4A shRNA on the proliferation of colon cancer cells\",\n",
            "          \"a, Multiple sequence alignment of the HNF4\\u03b1 isoforms, with peptides detected by shotgun proteomics and sequences corresponding to the shRNA target sequences highlighted. Different colors of the letters indicate different levels of sequence coverage in the\",\n",
            "          \"letters indicate different levels of sequence coverage in the shotgun proteomics study, as indicated by the color scale bar. Yellow boxes highlight sequences corresponding to the shRNA target sequences. TRCN0000019193 specifically targets P1\",\n",
            "          \"shRNA target sequences. TRCN0000019193 specifically targets P1 promoter-driven isoforms, whereas the other two target both types of isoforms. b\\u2013d, The P1- HNF4\\u03b1 specific shRNA showed mixed impacts (b), whereas shRNAs simultaneously targeting both P1- and\",\n",
            "          \"(b), whereas shRNAs simultaneously targeting both P1- and P2- HNF4\\u03b1 showed a primarily negative impact on cell proliferation (c,d). Moreover, a stronger negative impact was associated with increased copy number, both for the P1- HNF4\\u03b1 specific shRNA\",\n",
            "          \"increased copy number, both for the P1- HNF4\\u03b1 specific shRNA (p=0.04, Spearman\\u2019s correlation [r]) and for all shRNAs (p=0.01, Spearman\\u2019s correlation p-values for individual shRNAs summarized by the Fisher\\u2019s combined probability test).\",\n",
            "          \"Consensus matrices, the empirical cumulative distribution function (CDF) plot and core sample identification\",\n",
            "          \"a, Consensus matrices of the 90 CRC samples for k = 2 to k = 8. The consensus matrices show the robustness of the discovered clusters to sampling variability (resampling 80% samples) for cluster numbers k = 2 to 8. In each consensus matrix, both the rows\",\n",
            "          \"numbers k = 2 to 8. In each consensus matrix, both the rows and the columns were indexed with the same sample order and samples belonging to the same cluster frequently are adjacent to each other. For each pair of samples, a consensus index, which is the\",\n",
            "          \"For each pair of samples, a consensus index, which is the percentage of times they belong to the same cluster during 1,000 runs of the clustering algorithm based on resampling was calculated. The consensus index for each pair of samples was represented by\",\n",
            "          \"The consensus index for each pair of samples was represented by color gradient from white (0%) to red (100%) in the consensus matrix. b, CDF plots corresponding to the consensus matrices for k = 2 to k = 8. This plot shows the cumulative distribution of\",\n",
            "          \"k = 2 to k = 8. This plot shows the cumulative distribution of the entries of the consensus matrices within the 0\\u20131 range. Skew toward 0 and 1 indicates good clustering. As k increases, the area under the CDF is hypothesized to increase markedly until k\",\n",
            "          \"area under the CDF is hypothesized to increase markedly until k reaches the ktrue. In this case, 7 was considered as ktrue because the change of the area under the CDF was close to zero when k increased from 7 to 8. c, Silhouette plot for core sample\",\n",
            "          \"k increased from 7 to 8. c, Silhouette plot for core sample identification. For each sample (y-axis), the silhouette width (x-axis) compares its similarity to its assigned class and to any other classes. Samples with higher similarity to their assigned\",\n",
            "          \"other classes. Samples with higher similarity to their assigned class than to any other classes will get positive silhouette width score and be selected as core samples.\",\n",
            "          \"Network analysis of the subtype signature proteins\",\n",
            "          \"a, The number of signature proteins for each subtype. For a given subtype, the red circle represents proteins that were different in abundance between the subtype and all other subtypes, the green circle represents proteins that were different in\",\n",
            "          \"the green circle represents proteins that were different in abundance between the subtype and normal colon tissues. The intersection between red and green circles contains the signature proteins for each subtype. b, Visualizing subtype C signature\",\n",
            "          \"proteins for each subtype. b, Visualizing subtype C signature proteins in NetGestalt. Proteins in the iRef protein-protein interaction network are placed in a linear order together with the hierarchical modular organization of the network. Alternating bar\",\n",
            "          \"modular organization of the network. Alternating bar colors (green and orange) are used to distinguish neighboring modules. Proteins in the up- and down-signatures of subtype C were visualized as two separate tracks below the network modules, where each\",\n",
            "          \"as two separate tracks below the network modules, where each bar represents a protein. These proteins are not randomly distributed in the network. Highlighted by red or blue arrows are four Network modules (I, IV, V, VI) significantly enriched with\",\n",
            "          \"four Network modules (I, IV, V, VI) significantly enriched with up-signature proteins and two modules (II and III) significantly enriched with down-signature proteins (adjusted p value < 0.01). c\\u2013d, Heat maps depicting relative abundance of down- and\",\n",
            "          \"0.01). c\\u2013d, Heat maps depicting relative abundance of down- and up-signature proteins of subtype C in modules III and I, respectively. Tumors are displayed as rows, grouped by normal controls (N) and proteomic subtypes (A\\u2013E) as indicated by different side\",\n",
            "          \"(N) and proteomic subtypes (A\\u2013E) as indicated by different side bar colors. Proteins are displayed as columns. e\\u2013f, Network diagrams depicting the interaction of down- and up-signature proteins of subtype C in modules III and I, respectively. Node and\",\n",
            "          \"of subtype C in modules III and I, respectively. Node and node-border colors represent relatively higher or lower abundance in the subtype compared to other subtypes and normal colon tissues, respectively. Red and blue in the heat maps and the network\",\n",
            "          \"respectively. Red and blue in the heat maps and the network diagrams represent relatively higher or lower abundance, respectively.\",\n",
            "          \"Supplementary Material\\n\\nSupplementary Information is linked to the online version of this paper at www.nature.com/nature.\",\n",
            "          \"Author Contributions B.Z., R.J.C.S., D.L.T., L.J.Z. and D.C.L. designed the proteomic analysis experiments, data analysis workflow, and proteomic-genomic data comparisons. K.F.S., L.J.Z., R.J.C.S. and D.C.L. directed and performed proteomic analysis of\",\n",
            "          \"and D.C.L. directed and performed proteomic analysis of colon tumor and quality control samples. J.W., X.W., J.Z., Q.L., Z.S., P.W., S.W., R.J.C.S. and B.Z. performed proteomic-genomic data analyses. M.C.C., S.K., R.J.C.S. and D.L.T. performed analyses of\",\n",
            "          \"M.C.C., S.K., R.J.C.S. and D.L.T. performed analyses of mass spectrometry data and adapted algorithms and software for data analysis. S.R.D., R.R.T and M.J.C.E. developed and prepared breast xenografts used as quality control samples. S.A.C, K.F.S. and\",\n",
            "          \"xenografts used as quality control samples. S.A.C, K.F.S. and D.C.L. designed strategy for quality control analyses. R.J.C.S., C.R.K, R.C.R, and H.R. coordinated acquisition, distribution and quality control evaluation of TCGA tumor samples. B.Z., J.W.,\",\n",
            "          \"quality control evaluation of TCGA tumor samples. B.Z., J.W., R.J.C.S., R.J.C. and D.C.L. interpreted data in context of colon cancer biology. B.Z., R.J.C.S. and D.C.L. wrote the manuscript.\",\n",
            "          \"All of the primary mass spectrometry data on TCGA tumor samples are deposited at the CPTAC Data Coordinating Center as raw and mzML files for public access (https://cptac-data-portal.georgetown.edu).\\n\\nThe authors declare no competing financial interests.\",\n",
            "          \"The authors declare no competing financial interests.\\n\\nReprints and permissions information is available at www.nature.com/reprints.\\n\\nReaders are welcome to comment on the online version of this article at www.nature.com/nature.\",\n",
            "          \"Summary of detected single amino acid variants (SAAVs) and the impact of single nucleotide variants (SNVs) on protein abundance\",\n",
            "          \"a, The number of different types of SAAVs (TCGA-reported somatic variants, COSMIC-supported variants, dbSNP-supported variants, and new variants) in individual tumor samples. The samples are ordered by the number of detected somatic variants, then\",\n",
            "          \"are ordered by the number of detected somatic variants, then COSMIC-supported variants, and then dbSNP-supported variants. The Microsatellite instability (MSI) and hypermutation (Hyper) status are labeled below the bar charts for each sample (MSI-High:\",\n",
            "          \"are labeled below the bar charts for each sample (MSI-High: red, MSI-Low: orange, Microsatellite Stable: yellow; hypermutated: blue, non-hypermutated: sky blue; no data: grey). The number of somatic variants and COSMIC-supported variants were\",\n",
            "          \"number of somatic variants and COSMIC-supported variants were significantly higher in MSI-High and hypermutated tumors, whereas the other two types of SAAVs were randomly distributed across the data set. b, The total numbers for different types of SAAVs\",\n",
            "          \"the data set. b, The total numbers for different types of SAAVs and their overlapping relations. All 796 detected SAAVs were annotated based on previous reports in dbSNP (left circle), COSMIC (middle circle), or TCGA-reported somatic variants (right\",\n",
            "          \"(middle circle), or TCGA-reported somatic variants (right circle), and their overlapping relations are shown in the Venn diagram. There are 162 SAAVs that have not been reported previously in these databases (new). c, Distribution of the frequency of\",\n",
            "          \"in these databases (new). c, Distribution of the frequency of occurrence (1 sample: light grey, 2\\u20139 samples: grey, >=10 samples: dark grey) for different types of SAAVs. Border colors of the pie charts correspond to different SAAV types using the same\",\n",
            "          \"pie charts correspond to different SAAV types using the same color scheme as in (a). Whereas 58% of dbSNP-supported variants occurred in two or more samples, almost all somatic variants each occurred in only one sample. d, SNVs detected in RNA-Seq data\",\n",
            "          \"occurred in only one sample. d, SNVs detected in RNA-Seq data were separated into three categories (dbSNP-supported, COSMIC-supported, and TCGA-Somatic). The impact of individual SNVs on protein abundance was calculated (see supplementary methods) and the\",\n",
            "          \"abundance was calculated (see supplementary methods) and the impact scores for different categories of SNVs were plotted as cumulative fraction curves with two-sided p values from the Kolmogorov-Smirnov test labeled. The percentage of SNVs with an\",\n",
            "          \"Kolmogorov-Smirnov test labeled. The percentage of SNVs with an absolute impact score greater than 2 was also plotted as an inset, with p values from the Chi-squared test. Sample size for the dbSNP-supported, COSMIC-supported and TCGA-Somatic variants\",\n",
            "          \"the dbSNP-supported, COSMIC-supported and TCGA-Somatic variants were 12184, 7492, and 3302, respectively.\",\n",
            "          \"Correlations between mRNA and protein abundance in TCGA tumors\",\n",
            "          \"a, Steady state mRNA and protein abundance were positively correlated in all 86 samples (multiple-test adjusted p value < 0.01) with a mean Spearman\\u2019s correlation coefficient of 0.47. b, mRNA and protein variation were positively correlated for most\",\n",
            "          \"mRNA and protein variation were positively correlated for most (89.4%) mRNA-protein pairs across the 87 samples, but only 32% showed significant correlation (multiple-test adjusted p value < 0.01), with a mean Spearman\\u2019s correlation coefficient of 0.23.\",\n",
            "          \"0.01), with a mean Spearman\\u2019s correlation coefficient of 0.23. c, mRNA and protein levels displayed dramatically different correlation for genes involved in different biological processes. Genes encoding intermediary metabolism functions showed high\",\n",
            "          \"Genes encoding intermediary metabolism functions showed high mRNA-protein correlations, whereas genes involved in oxidative phosphorylation, RNA splicing and ribosome components showed low or negative correlations. Multiple-test adjusted two-sided\",\n",
            "          \"low or negative correlations. Multiple-test adjusted two-sided p-values from the Kolmogorov-Smirnov test were provided in the parentheses following the KEGG pathway names. Red and green in the figures indicate positive- and negative-correlations,\",\n",
            "          \"in the figures indicate positive- and negative-correlations, respectively.\",\n",
            "          \"Effects of copy number alterations (CNAs) on mRNA and protein abundance\",\n",
            "          \"a,b, The top panels show copy number-abundance correlation matrices for mRNA abundance (a) and protein abundance (b) with significant positive and negative correlations (multiple-test adjusted p value < 0.01, Spearman\\u2019s correlation coefficient) indicated\",\n",
            "          \"p value < 0.01, Spearman\\u2019s correlation coefficient) indicated by red and green colors, respectively, and genes ordered by chromosomal location on both x and y-axes. The bottom panels show the frequency of mRNAs/proteins associated with a particular copy\",\n",
            "          \"frequency of mRNAs/proteins associated with a particular copy number alteration, where blue and black bars represent associations specific to mRNA/protein or common to both mRNA and protein, respectively. c\\u2013e,\",\n",
            "          \"HNF4A, TOMM34 and SRC showed significant CNA-mRNA, mRNA-protein, and CNA-protein correlations (Spearman\\u2019s correlation coefficient). The color grade from light yellow to red indicates relatively low-level to high-level CNA, relative mRNA abundance or\",\n",
            "          \"low-level to high-level CNA, relative mRNA abundance or relative protein abundance among the 85 samples, which were ordered by copy number data.\",\n",
            "          \"Proteomic subtypes of colon and rectal cancers, associated genomic features, and relative abundance of HNF4\\u03b1\",\n",
            "          \"a, Figure legends for b, c and d. b, Identification of five proteomic subtypes. Tumors are displayed as columns, grouped by proteomic subtypes as indicated by different colors. Proteins used for the subtype classification are displayed as rows. The heat\",\n",
            "          \"for the subtype classification are displayed as rows. The heat map presents relative abundance of the proteins (logarithmic scale in base 2) in the 90 tumor cohort. c, Association of proteomic subtypes with major colorectal cancer-associated genomic\",\n",
            "          \"subtypes with major colorectal cancer-associated genomic alterations and previously published transcriptomic and methylation subtypes. Subtypes significantly overlapped with a transcriptomic or methylation subtype are highlighted by pink boxes. Both\",\n",
            "          \"or methylation subtype are highlighted by pink boxes. Both proteomic subtypes B and C showed significant overlap with the TCGA MSI/CIMP subtype. In addition, they showed significant overlap with the CCS2 and CCS3 subtypes in the De Sousa et al.\",\n",
            "          \"overlap with the CCS2 and CCS3 subtypes in the De Sousa et al. classification, respectively. Proteomic subtype B significantly overlapped with the TCGA CIMP-H methylation subtype, whereas subtype C significantly overlapped with a non-methylation subtype\",\n",
            "          \"C significantly overlapped with a non-methylation subtype (TCGA cluster 4 methylation subtype). Subtypes over-represented with a specific genomic alteration are also highlighted by pink boxes. The green box highlights the absence of TP53 mutations and 18q\",\n",
            "          \"The green box highlights the absence of TP53 mutations and 18q loss in subtype B. d, The top panel shows HNF4A copy number and relative abundance of HNF4\\u03b1 in the five subtypes; the bottom panel compares relative abundance of HNF4\\u03b1 in the five subtypes to\",\n",
            "          \"compares relative abundance of HNF4\\u03b1 in the five subtypes to that in normal colon samples, respectively, and the adjP values are based on the two-sided Wilcoxon rank-sum test followed by multiple-test adjustment.\"\n",
            "        ]\n",
            "      }\n",
            "    ]\n",
            "  },\n",
            "  \"PDC000289\": {\n",
            "    \"study_id\": \"5d019771-c39c-4dec-b2a9-f0aec54a796f\",\n",
            "    \"pdc_study_id\": \"PDC000289\",\n",
            "    \"study_submitter_id\": \"TCGA Colorectal Cancer CompRef Samples - Proteome\",\n",
            "    \"program_id\": \"10251935-5540-11e8-b664-00a098d917f8\",\n",
            "    \"project_id\": \"c96eca75-8b73-4018-9bf7-b20e31a965cd\",\n",
            "    \"study_name\": \"TCGA Colorectal Cancer CompRef Samples - Proteome\",\n",
            "    \"program_name\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"project_name\": \"CPTAC2 Retrospective\",\n",
            "    \"disease_type\": \"Other\",\n",
            "    \"primary_site\": \"Not Reported\",\n",
            "    \"analytical_fraction\": \"Proteome\",\n",
            "    \"experiment_type\": \"Label Free\",\n",
            "    \"embargo_date\": null,\n",
            "    \"cases_count\": 2,\n",
            "    \"aliquots_count\": 8,\n",
            "    \"protocol_id\": \"26719c58-7efe-42e5-bb7b-c3667f231762\",\n",
            "    \"protocol_submitter_id\": \"TCGA Colorectal Cancer CompRef Proteome\",\n",
            "    \"program_submitter_id\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"protocol_name\": \"TCGA Colorectal Cancer CompRef Proteome\",\n",
            "    \"protocol_date\": \"2021-04-07\",\n",
            "    \"document_name\": null,\n",
            "    \"quantitation_strategy\": \"LC-MS label-free quantitation analysis\",\n",
            "    \"label_free_quantitation\": \"Label-free protein level quantitation\",\n",
            "    \"labeled_quantitation\": \"N/A\",\n",
            "    \"isobaric_labeling_reagent\": \"N/A\",\n",
            "    \"reporter_ion_ms_level\": \"N/A\",\n",
            "    \"starting_amount\": \"200.00\",\n",
            "    \"starting_amount_uom\": \"\\u00b5g\",\n",
            "    \"digestion_reagent\": \"Trypsin\",\n",
            "    \"alkylation_reagent\": \"Iodoacetamide (IAA)\",\n",
            "    \"enrichment_strategy\": \"Proteome\",\n",
            "    \"enrichment\": \"None\",\n",
            "    \"chromatography_dimensions_count\": \"2\",\n",
            "    \"one_d_chromatography_type\": \"bRPLC\",\n",
            "    \"two_d_chromatography_type\": \"RPLC\",\n",
            "    \"fractions_analyzed_count\": \"15\",\n",
            "    \"column_type\": \"C18\",\n",
            "    \"amount_on_column\": null,\n",
            "    \"amount_on_column_uom\": null,\n",
            "    \"column_length\": \"11.0\",\n",
            "    \"column_length_uom\": \"cm\",\n",
            "    \"column_inner_diameter\": \"100.00\",\n",
            "    \"column_inner_diameter_uom\": \"\\u00b5m\",\n",
            "    \"particle_size\": \"5.0\",\n",
            "    \"particle_size_uom\": \"\\u00b5m\",\n",
            "    \"particle_type\": \"300 \\u00c5, Phenomenex \",\n",
            "    \"gradient_length\": \"95.0\",\n",
            "    \"gradient_length_uom\": \"min\",\n",
            "    \"instrument_make\": \"Thermo Fisher Scientific\",\n",
            "    \"instrument_model\": \"LTQ Orbitrap Velos\",\n",
            "    \"dissociation_type\": \"CID\",\n",
            "    \"ms1_resolution\": \"60000\",\n",
            "    \"ms2_resolution\": null,\n",
            "    \"dda_topn\": \"Top 8\",\n",
            "    \"normalized_collision_energy\": \"35\",\n",
            "    \"acquistion_type\": \"DDA\",\n",
            "    \"dia_multiplexing\": \"N/A\",\n",
            "    \"dia_ims\": \"N/A\",\n",
            "    \"analytical_technique\": null,\n",
            "    \"chromatography_instrument_make\": null,\n",
            "    \"chromatography_instrument_model\": null,\n",
            "    \"polarity\": null,\n",
            "    \"reconstitution_solvent\": null,\n",
            "    \"reconstitution_volume\": null,\n",
            "    \"reconstitution_volume_uom\": null,\n",
            "    \"internal_standards\": null,\n",
            "    \"extraction_method\": null,\n",
            "    \"ionization_mode\": null\n",
            "  },\n",
            "  \"PDC000292\": {\n",
            "    \"study_id\": \"fef54f2c-f792-455c-b5d2-41f7cb0b071c\",\n",
            "    \"pdc_study_id\": \"PDC000292\",\n",
            "    \"study_submitter_id\": \"TCGA Ovarian CompRef Samples JHU Proteome\",\n",
            "    \"program_id\": \"10251935-5540-11e8-b664-00a098d917f8\",\n",
            "    \"project_id\": \"c96eca75-8b73-4018-9bf7-b20e31a965cd\",\n",
            "    \"study_name\": \"TCGA Ovarian CompRef Samples JHU Proteome\",\n",
            "    \"program_name\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"project_name\": \"CPTAC2 Retrospective\",\n",
            "    \"disease_type\": \"Other\",\n",
            "    \"primary_site\": \"Not Reported\",\n",
            "    \"analytical_fraction\": \"Proteome\",\n",
            "    \"experiment_type\": \"iTRAQ4\",\n",
            "    \"embargo_date\": null,\n",
            "    \"cases_count\": 2,\n",
            "    \"aliquots_count\": 2,\n",
            "    \"protocol_id\": \"5f78dd60-a969-4e41-93fd-a87a2bf597ce\",\n",
            "    \"protocol_submitter_id\": \"TCGA Ovarian CompRef JHU Proteome\",\n",
            "    \"program_submitter_id\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"protocol_name\": \"TCGA Ovarian CompRef JHU Proteome\",\n",
            "    \"protocol_date\": \"2021-04-08\",\n",
            "    \"document_name\": null,\n",
            "    \"quantitation_strategy\": \"Isobaric label quantitation analysis\",\n",
            "    \"label_free_quantitation\": \"N/A\",\n",
            "    \"labeled_quantitation\": \"iTRAQ\",\n",
            "    \"isobaric_labeling_reagent\": \"iTRAQ4\",\n",
            "    \"reporter_ion_ms_level\": \"MS2\",\n",
            "    \"starting_amount\": \"2.00\",\n",
            "    \"starting_amount_uom\": \"mg\",\n",
            "    \"digestion_reagent\": \"Trypsin\",\n",
            "    \"alkylation_reagent\": \"Iodoacetamide (IAA)\",\n",
            "    \"enrichment_strategy\": \"Proteome\",\n",
            "    \"enrichment\": \"None\",\n",
            "    \"chromatography_dimensions_count\": \"2\",\n",
            "    \"one_d_chromatography_type\": \"bRPLC\",\n",
            "    \"two_d_chromatography_type\": \"RPLC\",\n",
            "    \"fractions_analyzed_count\": \"24\",\n",
            "    \"column_type\": \"C18\",\n",
            "    \"amount_on_column\": \"2.0\",\n",
            "    \"amount_on_column_uom\": \"\\u00b5g\",\n",
            "    \"column_length\": \"15.0\",\n",
            "    \"column_length_uom\": \"cm\",\n",
            "    \"column_inner_diameter\": \"75.00\",\n",
            "    \"column_inner_diameter_uom\": \"\\u00b5m\",\n",
            "    \"particle_size\": \"3.0\",\n",
            "    \"particle_size_uom\": \"\\u00b5m\",\n",
            "    \"particle_type\": \"Jupiter C18\",\n",
            "    \"gradient_length\": \"90.0\",\n",
            "    \"gradient_length_uom\": \"min\",\n",
            "    \"instrument_make\": \"Thermo Fisher Scientific\",\n",
            "    \"instrument_model\": \"LTQ Orbitrap Velos\",\n",
            "    \"dissociation_type\": \"HCD\",\n",
            "    \"ms1_resolution\": \"30000\",\n",
            "    \"ms2_resolution\": \"7500\",\n",
            "    \"dda_topn\": \"Top 10\",\n",
            "    \"normalized_collision_energy\": \"35\",\n",
            "    \"acquistion_type\": \"DDA\",\n",
            "    \"dia_multiplexing\": \"N/A\",\n",
            "    \"dia_ims\": \"N/A\",\n",
            "    \"analytical_technique\": null,\n",
            "    \"chromatography_instrument_make\": null,\n",
            "    \"chromatography_instrument_model\": null,\n",
            "    \"polarity\": null,\n",
            "    \"reconstitution_solvent\": null,\n",
            "    \"reconstitution_volume\": null,\n",
            "    \"reconstitution_volume_uom\": null,\n",
            "    \"internal_standards\": null,\n",
            "    \"extraction_method\": null,\n",
            "    \"ionization_mode\": null\n",
            "  },\n",
            "  \"PDC000294\": {\n",
            "    \"study_id\": \"d865ed3e-1057-4f31-b018-85d87db9dcce\",\n",
            "    \"pdc_study_id\": \"PDC000294\",\n",
            "    \"study_submitter_id\": \"TCGA Ovarian CompRef Samples PNNL Phosphoproteome Velos Qexactive\",\n",
            "    \"program_id\": \"10251935-5540-11e8-b664-00a098d917f8\",\n",
            "    \"project_id\": \"c96eca75-8b73-4018-9bf7-b20e31a965cd\",\n",
            "    \"study_name\": \"TCGA Ovarian CompRef Samples PNNL Phosphoproteome Velos Qexactive\",\n",
            "    \"program_name\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"project_name\": \"CPTAC2 Retrospective\",\n",
            "    \"disease_type\": \"Other\",\n",
            "    \"primary_site\": \"Not Reported\",\n",
            "    \"analytical_fraction\": \"Phosphoproteome\",\n",
            "    \"experiment_type\": \"iTRAQ4\",\n",
            "    \"embargo_date\": null,\n",
            "    \"cases_count\": 2,\n",
            "    \"aliquots_count\": 2,\n",
            "    \"protocol_id\": \"12453f97-3f53-4bb5-9258-d34909d16c77\",\n",
            "    \"protocol_submitter_id\": \"TCGA Ovarian CompRef PNNL Phosphoproteome Velos\",\n",
            "    \"program_submitter_id\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"protocol_name\": \"TCGA Ovarian CompRef PNNL Phosphoproteome Velos\",\n",
            "    \"protocol_date\": \"2021-04-08\",\n",
            "    \"document_name\": \"PNNL_CPTAC_SampleProtocol_CompRef\",\n",
            "    \"quantitation_strategy\": \"Isobaric label quantitation analysis\",\n",
            "    \"label_free_quantitation\": \"N/A\",\n",
            "    \"labeled_quantitation\": \"iTRAQ\",\n",
            "    \"isobaric_labeling_reagent\": \"iTRAQ4\",\n",
            "    \"reporter_ion_ms_level\": \"MS2\",\n",
            "    \"starting_amount\": \"45.00\",\n",
            "    \"starting_amount_uom\": \"mg\",\n",
            "    \"digestion_reagent\": \"Trypsin\",\n",
            "    \"alkylation_reagent\": \"Iodoacetamide (IAA)\",\n",
            "    \"enrichment_strategy\": \"Phosphoproteome\",\n",
            "    \"enrichment\": \"Fe3+ NTA-Agarose (Qiagen)\",\n",
            "    \"chromatography_dimensions_count\": \"2\",\n",
            "    \"one_d_chromatography_type\": \"bRPLC (pH 7.5)\",\n",
            "    \"two_d_chromatography_type\": \"RPLC\",\n",
            "    \"fractions_analyzed_count\": \"12\",\n",
            "    \"column_type\": \"C18\",\n",
            "    \"amount_on_column\": \"0.5\",\n",
            "    \"amount_on_column_uom\": \"\\u00b5g\",\n",
            "    \"column_length\": \"60.0\",\n",
            "    \"column_length_uom\": \"cm\",\n",
            "    \"column_inner_diameter\": \"50.00\",\n",
            "    \"column_inner_diameter_uom\": \"\\u00b5m\",\n",
            "    \"particle_size\": \"3.0\",\n",
            "    \"particle_size_uom\": \"\\u00b5m\",\n",
            "    \"particle_type\": \"Phenomenex Jupiter C18\",\n",
            "    \"gradient_length\": \"130.0\",\n",
            "    \"gradient_length_uom\": \"min\",\n",
            "    \"instrument_make\": \"Thermo Fisher Scientific\",\n",
            "    \"instrument_model\": \"LTQ Orbitrap Velos\",\n",
            "    \"dissociation_type\": \"HCD\",\n",
            "    \"ms1_resolution\": \"70000\",\n",
            "    \"ms2_resolution\": \"7500\",\n",
            "    \"dda_topn\": \"Top 10\",\n",
            "    \"normalized_collision_energy\": \"45\",\n",
            "    \"acquistion_type\": \"DDA\",\n",
            "    \"dia_multiplexing\": \"N/A\",\n",
            "    \"dia_ims\": \"N/A\",\n",
            "    \"analytical_technique\": null,\n",
            "    \"chromatography_instrument_make\": null,\n",
            "    \"chromatography_instrument_model\": null,\n",
            "    \"polarity\": null,\n",
            "    \"reconstitution_solvent\": null,\n",
            "    \"reconstitution_volume\": null,\n",
            "    \"reconstitution_volume_uom\": null,\n",
            "    \"internal_standards\": null,\n",
            "    \"extraction_method\": null,\n",
            "    \"ionization_mode\": null\n",
            "  },\n",
            "  \"PDC000293\": {\n",
            "    \"study_id\": \"6fd7959e-bd78-4e42-b59f-0cfca9bef779\",\n",
            "    \"pdc_study_id\": \"PDC000293\",\n",
            "    \"study_submitter_id\": \"TCGA Ovarian CompRef Samples PNNL Proteome\",\n",
            "    \"program_id\": \"10251935-5540-11e8-b664-00a098d917f8\",\n",
            "    \"project_id\": \"c96eca75-8b73-4018-9bf7-b20e31a965cd\",\n",
            "    \"study_name\": \"TCGA Ovarian CompRef Samples PNNL Proteome\",\n",
            "    \"program_name\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"project_name\": \"CPTAC2 Retrospective\",\n",
            "    \"disease_type\": \"Other\",\n",
            "    \"primary_site\": \"Not Reported\",\n",
            "    \"analytical_fraction\": \"Proteome\",\n",
            "    \"experiment_type\": \"iTRAQ4\",\n",
            "    \"embargo_date\": null,\n",
            "    \"cases_count\": 2,\n",
            "    \"aliquots_count\": 2,\n",
            "    \"protocol_id\": \"236f18e1-b6c5-43af-841c-285358764ac3\",\n",
            "    \"protocol_submitter_id\": \"TCGA Ovarian CompRef PNNL Proteome\",\n",
            "    \"program_submitter_id\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"protocol_name\": \"TCGA Ovarian CompRef PNNL Proteome\",\n",
            "    \"protocol_date\": \"2021-04-08\",\n",
            "    \"document_name\": \"PNNL_CPTAC_SampleProtocol_CompRef\",\n",
            "    \"quantitation_strategy\": \"Isobaric label quantitation analysis\",\n",
            "    \"label_free_quantitation\": \"N/A\",\n",
            "    \"labeled_quantitation\": \"iTRAQ\",\n",
            "    \"isobaric_labeling_reagent\": \"iTRAQ4\",\n",
            "    \"reporter_ion_ms_level\": \"MS2\",\n",
            "    \"starting_amount\": \"5.00\",\n",
            "    \"starting_amount_uom\": \"mg\",\n",
            "    \"digestion_reagent\": \"Trypsin\",\n",
            "    \"alkylation_reagent\": \"Iodoacetamide (IAA)\",\n",
            "    \"enrichment_strategy\": \"Proteome\",\n",
            "    \"enrichment\": \"None\",\n",
            "    \"chromatography_dimensions_count\": \"2\",\n",
            "    \"one_d_chromatography_type\": \"bRPLC (pH 7.5)\",\n",
            "    \"two_d_chromatography_type\": \"RPLC\",\n",
            "    \"fractions_analyzed_count\": \"24\",\n",
            "    \"column_type\": \"C18\",\n",
            "    \"amount_on_column\": \"0.75\",\n",
            "    \"amount_on_column_uom\": \"\\u00b5g\",\n",
            "    \"column_length\": \"35.0\",\n",
            "    \"column_length_uom\": \"cm\",\n",
            "    \"column_inner_diameter\": \"75.00\",\n",
            "    \"column_inner_diameter_uom\": \"\\u00b5m\",\n",
            "    \"particle_size\": \"3.0\",\n",
            "    \"particle_size_uom\": \"\\u00b5m\",\n",
            "    \"particle_type\": \"Phenomenex Jupiter C18\",\n",
            "    \"gradient_length\": \"100.0\",\n",
            "    \"gradient_length_uom\": \"min\",\n",
            "    \"instrument_make\": \"Thermo Fisher Scientific\",\n",
            "    \"instrument_model\": \"LTQ Orbitrap Velos\",\n",
            "    \"dissociation_type\": \"HCD\",\n",
            "    \"ms1_resolution\": \"30000\",\n",
            "    \"ms2_resolution\": \"7500\",\n",
            "    \"dda_topn\": \"Top 10\",\n",
            "    \"normalized_collision_energy\": \"45\",\n",
            "    \"acquistion_type\": \"DDA\",\n",
            "    \"dia_multiplexing\": \"N/A\",\n",
            "    \"dia_ims\": \"N/A\",\n",
            "    \"analytical_technique\": null,\n",
            "    \"chromatography_instrument_make\": null,\n",
            "    \"chromatography_instrument_model\": null,\n",
            "    \"polarity\": null,\n",
            "    \"reconstitution_solvent\": null,\n",
            "    \"reconstitution_volume\": null,\n",
            "    \"reconstitution_volume_uom\": null,\n",
            "    \"internal_standards\": null,\n",
            "    \"extraction_method\": null,\n",
            "    \"ionization_mode\": null\n",
            "  },\n",
            "  \"PDC000112\": {\n",
            "    \"study_id\": \"b9f2ccc5-57b8-11e8-b07a-00a098d917f8\",\n",
            "    \"pdc_study_id\": \"PDC000112\",\n",
            "    \"study_submitter_id\": \"TCGA OV Glycoproteome S020-1\",\n",
            "    \"program_id\": \"10251935-5540-11e8-b664-00a098d917f8\",\n",
            "    \"project_id\": \"48af5040-5546-11e8-b664-00a098d917f8\",\n",
            "    \"study_name\": \"TCGA Ovarian JHU Glycoproteome\",\n",
            "    \"program_name\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"project_name\": \"CPTAC2 Retrospective\",\n",
            "    \"disease_type\": \"Other;Ovarian Serous Cystadenocarcinoma\",\n",
            "    \"primary_site\": \"Not Reported;Ovary\",\n",
            "    \"analytical_fraction\": \"Glycoproteome\",\n",
            "    \"experiment_type\": \"iTRAQ4\",\n",
            "    \"embargo_date\": null,\n",
            "    \"cases_count\": 124,\n",
            "    \"aliquots_count\": 124,\n",
            "    \"protocol_id\": \"1c06b658-3b26-11e9-9a07-0a80fada099c\",\n",
            "    \"protocol_submitter_id\": \"P-S020-1\",\n",
            "    \"program_submitter_id\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"protocol_name\": \"TCGA_Ovarian_JHU_Glycoproteome\",\n",
            "    \"protocol_date\": \"2014-04-01\",\n",
            "    \"document_name\": \"\",\n",
            "    \"quantitation_strategy\": \"Isobaric label quantitation\",\n",
            "    \"label_free_quantitation\": \"\",\n",
            "    \"labeled_quantitation\": \"iTRAQ quantitation\",\n",
            "    \"isobaric_labeling_reagent\": \"iTRAQ\",\n",
            "    \"reporter_ion_ms_level\": \"2\",\n",
            "    \"starting_amount\": \"3.60\",\n",
            "    \"starting_amount_uom\": \"mg\",\n",
            "    \"digestion_reagent\": \"Trypsin\",\n",
            "    \"alkylation_reagent\": \"Iodoacetamide \",\n",
            "    \"enrichment_strategy\": \"Glycoproteome\",\n",
            "    \"enrichment\": \"Hydrazide Beads (Glycopeptides)\",\n",
            "    \"chromatography_dimensions_count\": \"2\",\n",
            "    \"one_d_chromatography_type\": \"bRPLC\",\n",
            "    \"two_d_chromatography_type\": \"RPLC\",\n",
            "    \"fractions_analyzed_count\": \"1\",\n",
            "    \"column_type\": \"C18\",\n",
            "    \"amount_on_column\": \"2\",\n",
            "    \"amount_on_column_uom\": \"\\u00b5g\",\n",
            "    \"column_length\": \"15\",\n",
            "    \"column_length_uom\": \"cm\",\n",
            "    \"column_inner_diameter\": \"75.00\",\n",
            "    \"column_inner_diameter_uom\": \"\\u00b5m\",\n",
            "    \"particle_size\": \"3\",\n",
            "    \"particle_size_uom\": \"\\u00b5m\",\n",
            "    \"particle_type\": \"Jupiter C18\",\n",
            "    \"gradient_length\": \"90\",\n",
            "    \"gradient_length_uom\": \"min\",\n",
            "    \"instrument_make\": \"Thermo Scientific\",\n",
            "    \"instrument_model\": \"Orbitrap Velos\",\n",
            "    \"dissociation_type\": \"HCD\",\n",
            "    \"ms1_resolution\": \"30000\",\n",
            "    \"ms2_resolution\": \"7500\",\n",
            "    \"dda_topn\": \"Top 10\",\n",
            "    \"normalized_collision_energy\": \"35%\",\n",
            "    \"acquistion_type\": \"DDA\",\n",
            "    \"dia_multiplexing\": \"N/A\",\n",
            "    \"dia_ims\": \"N/A\",\n",
            "    \"analytical_technique\": null,\n",
            "    \"chromatography_instrument_make\": null,\n",
            "    \"chromatography_instrument_model\": null,\n",
            "    \"polarity\": null,\n",
            "    \"reconstitution_solvent\": null,\n",
            "    \"reconstitution_volume\": null,\n",
            "    \"reconstitution_volume_uom\": null,\n",
            "    \"internal_standards\": null,\n",
            "    \"extraction_method\": null,\n",
            "    \"ionization_mode\": null,\n",
            "    \"publications\": [\n",
            "      {\n",
            "        \"publication_id\": \"44e2e658-89c7-11e8-bcf1-0a2705229b82\",\n",
            "        \"pubmed_id\": \"27372738\",\n",
            "        \"doi\": \"10.1016/j.cell.2016.05.069\",\n",
            "        \"author\": \"Zhang, H.; Liu, T.; Zhang, Z.; Payne, S. H.; Zhang, B.; McDermott, J. E.; Zhou, J. Y.; Petyuk, V. A.; Chen, L.; Ray, D.; Sun, S.; Yang, F.; Chen, L.; Wang, J.; Shah, P.; Cha, S. W.; Aiyetan, P.; Woo, S.; Tian, Y.; Gritsenko, M. A.; Clauss, T. R.; Choi, C.; Monroe, M. E.; Thomas, S.; Nie, S.; Wu, C.; Moore, R. J.; Yu, K. H.; Tabb, D. L.; Feny\\u00f6, D.; Bafna, V.; Wang, Y.; Rodriguez, H.; Boja, E. S.; Hiltke, T.; Rivers, R. C.; Sokoll, L.; Zhu, H.; Shih, I. M.; Cope, L.; Pandey, A.; Zhang, B.; Snyder, M. P.; Levine, D. A.; Smith, R. D.; Chan, D. W.; Rodland, K. D.; Carr, S. A.; Gillette, M. A.; Klauser, K. R.; Kuhn, E.; Mani, D. R.; Mertins, P.; Ketchum, K. A.; Thangudu, R.; Cai, S.; Oberti, M.; Paulovich, A. G.; Whiteaker, J. R.; Edwards, N. J.; McGarvey, P. B.; Madhavan, S.; Wang, P.; Chan, D. W.; Pandey, A.; Shih, I. M.; Zhang, H.; Zhang, Z.; Zhu, H.; Cope, L.; Whiteley, G. A.; Skates, S. J.; White, F. M.; Levine, D. A.; Boja, E. S.; Kinsinger, C. R.; Hiltke, T.; Mesri, M.; Rivers, R. C.; Rodriguez, H.; Shaw, K. M.; Stein, S. E.; Fenyo, D.; Liu, T.; McDermott, J. E.; Payne, S. H.; Rodland, K. D.; Smith, R. D.; Rudnick, P.; Snyder, M.; Zhao, Y.; Chen, X.; Ransohoff, D. F.; Hoofnagle, A. N.; Liebler, D. C.; Sanders, M. E.; Shi, Z.; Slebos, R. J. C.; Tabb, D. L.; Zhang, B.; Zimmerman, L. J.; Wang, Y.; Davies, S. R.; Ding, L.; Ellis, M. J. C.; Townsend, R. R.\",\n",
            "        \"title\": \"Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer\",\n",
            "        \"journal\": \"Cell\",\n",
            "        \"journal_url\": \"https://www.cell.com/cell/fulltext/S0092-8674(16)30673-0?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0092867416306730%3Fshowall%3Dtrue\",\n",
            "        \"year\": \"2016\",\n",
            "        \"abstract\": \"To provide a detailed analysis of the molecular components and underlying mechanisms associated with ovarian cancer, we performed a comprehensive mass-spectrometry-based proteomic characterization of 174 ovarian tumors previously analyzed by The Cancer Genome Atlas (TCGA), of which 169 were high-grade serous carcinomas (HGSCs). Integrating our proteomic measurements with the genomic data yielded a number of insights into disease, such as how different copy-number alternations influence the proteome, the proteins associated with chromosomal instability, the sets of signaling pathways that diverse genome rearrangements converge on, and the ones most associated with short overall survival. Specific protein acetylations associated with homologous recombination deficiency suggest a potential means for stratifying patients for therapy. In addition to providing a valuable resource, these findings provide a view of how the somatic genome drives the cancer proteome and associations between protein and post-translational modification levels and clinical outcomes in HGSC.\",\n",
            "        \"full_text\": [\n",
            "          \"Integrated proteogenomic characterization of human high grade serous ovarian cancer\\n\\nSUMMARY\",\n",
            "          \"To provide a detailed analysis of the molecular components and underlying mechanisms associated with ovarian cancer, we performed a comprehensive mass spectrometry-based proteomic characterization of 174 ovarian tumors previously analyzed by The Cancer\",\n",
            "          \"of 174 ovarian tumors previously analyzed by The Cancer Genome Atlas (TCGA), of which 169 were high-grade serous carcinomas (HGSC). Integrating our proteomic measurements with the genomic data yielded a number of insights into disease such as how\",\n",
            "          \"data yielded a number of insights into disease such as how different copy number alternations influence the proteome, the proteins associated with chromosomal instability, the sets of signaling pathways that diverse genome rearrangements converge on, as\",\n",
            "          \"pathways that diverse genome rearrangements converge on, as well as the ones most associated with short overall survival. Specific protein acetylations associated with homologous recombination deficiency suggest a potential means for stratifying patients\",\n",
            "          \"deficiency suggest a potential means for stratifying patients for therapy. In addition to providing a valuable resource, these findings provide a view of how the somatic genome drives the cancer proteome and associations between protein and\",\n",
            "          \"drives the cancer proteome and associations between protein and post-translational modification levels and clinical outcomes in HGSC.\",\n",
            "          \"INTRODUCTION\",\n",
            "          \"A comprehensive molecular view of cancer is necessary for understanding the underlying mechanisms of disease, improving prognosis, and ultimately guiding treatment. The Cancer Genome Atlas (TCGA) conducted an extensive genomic and transcriptomic\",\n",
            "          \"Atlas (TCGA) conducted an extensive genomic and transcriptomic characterization of ovarian high-grade serous carcinoma (HGSC) aimed at defining the genomic landscape and aiding the development of targeted therapies for this highly lethal malignancy. Key\",\n",
            "          \"of targeted therapies for this highly lethal malignancy. Key findings from TCGA were: 1) the prevalent role of TP53 mutations, 2) extensive DNA copy alterations, 3) preliminary transcriptional signatures associated with survival, 4) varied mechanisms of\",\n",
            "          \"signatures associated with survival, 4) varied mechanisms of BRCA1/2 inactivation, and lastly, 5) CCNE1 aberrations. Subsequent analysis of genomic data from the TCGA consortium led to the refinement of the transcript-defined signatures, improving the\",\n",
            "          \"refinement of the transcript-defined signatures, improving the statistical association with patient outcome, and integrating microRNA and mRNA expression profiles associated with HGSC to identify candidate microRNA targets.\",\n",
            "          \"While the insights from genomic analyses are substantial, functions encoded in the genome are generally executed at the protein level, and are often further modulated by post-translational modifications (PTMs). For example, TCGA used reverse phase protein\",\n",
            "          \"(PTMs). For example, TCGA used reverse phase protein array (RPPA) analysis of 172 proteins (including 31 phosphoproteins with phospho-specific antibodies) to generate a signature associated with the risk of tumor recurrence. To obtain a more comprehensive\",\n",
            "          \"the risk of tumor recurrence. To obtain a more comprehensive assessment of the complex ovarian HGSC phenotype, the Clinical Proteomic Tumor Analysis Consortium (CPTAC) conducted an extensive mass spectrometry (MS)-based proteomic and phosphoproteomic\",\n",
            "          \"mass spectrometry (MS)-based proteomic and phosphoproteomic characterization of HGSC tumors characterized by TCGA, providing quantitative measurements for a combined total of 9,600 proteins from 174 tumors (an average of 7,952 proteins per tumor), and a\",\n",
            "          \"from 174 tumors (an average of 7,952 proteins per tumor), and a total of 24,429 phosphosites from 6,769 phosphoproteins in a subset of 69 tumors (an average of 7,677 phosphosites per tumor). Our results provide insights on HGSC biology and correlate\",\n",
            "          \"Our results provide insights on HGSC biology and correlate differences in protein and PTM levels with clinical outcome complementary to TCGA genomic analyses.\",\n",
            "          \"RESULTS\\n\\nProteomic analysis of TCGA HGSC samples\",\n",
            "          \"HGSC biospecimens and clinical data from 174 patients collected by TCGA were analyzed at two independent CPTAC Centers, Johns Hopkins University (JHU) and Pacific Northwest National Laboratory (PNNL); thirty-two samples were analyzed at both JHU and PNNL.\",\n",
            "          \"(PNNL); thirty-two samples were analyzed at both JHU and PNNL. Tumors were selected by examining the associated TCGA metadata to select tumors either 1) on the basis of putative homologous recombination deficiency (HRD), defined by the presence of\",\n",
            "          \"recombination deficiency (HRD), defined by the presence of germline or somatic BRCA1 or BRCA2 mutations, BRCA1 promoter methylation, or homozygous deletion of PTEN (122 samples; 67 classified as HRD, 55 as non-HRD by the above criteria; JHU), or 2) to\",\n",
            "          \"as HRD, 55 as non-HRD by the above criteria; JHU), or 2) to maximize differences in overall survival (84 samples; PNNL). For selection purposes, short survival was defined as overall survival of less than 3 years, and long survival as greater than 5\",\n",
            "          \"of less than 3 years, and long survival as greater than 5 years. All but five tumors had somatic TP53 mutations, a characteristic feature of HGSC; these five tumors were subsequently reclassified as other than HGSC, and removed from protein functional\",\n",
            "          \"as other than HGSC, and removed from protein functional analyses (e.g., subtyping and survival analyses). The tumor selection criteria and the associated metadata are provided in the Supplemental Information Table S1, available online.\",\n",
            "          \"Proteomics measurements used isobaric tags for relative and absolute quantitation (iTRAQ) in conjunction with offline liquid chromatography fractionation via high-pH reversed phase liquid chromatography (RPLC) and online RPLC with high-resolution tandem\",\n",
            "          \"(RPLC) and online RPLC with high-resolution tandem MS to provide broad coverage for peptide and protein identification and quantification (Supplemental Information); this also alleviated quantitative interference potentially associated with the use of\",\n",
            "          \"interference potentially associated with the use of isobaric tags. We used the relative abundance measurements for each protein in the 32 patient samples analyzed at both JHU and PNNL to normalize across the two analysis sites, and then used clustering,\",\n",
            "          \"across the two analysis sites, and then used clustering, principle component analysis (PCA), and statistical tests to identify any significant batch effects associated with the site of analysis (a detailed comparison of within site, between site, between\",\n",
            "          \"(a detailed comparison of within site, between site, between sample measurement variability and the process used to merge the JHU and PNNL data is given in Supplemental Information, Figure S1). As shown in Figure S1C, the median coefficient of variation\",\n",
            "          \"As shown in Figure S1C, the median coefficient of variation (CV) between measurements at the two sites was 16%.\",\n",
            "          \"A total of 9,600 proteins were identified with high confidence in all tumors, and the relative abundances in each tumor are given in Table S2. Functional analyses and proteome-transcriptome associations were restricted to 3,586 proteins observed and\",\n",
            "          \"associations were restricted to 3,586 proteins observed and quantified in all 169 HGSC samples used for protein functional analyses and where sample variability (signal) exceeded technical variability (noise) in the merged data (Table S2), calculated as\",\n",
            "          \"(noise) in the merged data (Table S2), calculated as described in Supplemental Information. On average we identify peptides covering 29% of the amino acids in any of these 3,586 proteins in a given sample, with a range from 10% to 47%. In addition to\",\n",
            "          \"in a given sample, with a range from 10% to 47%. In addition to protein abundance levels, phosphoproteomics data were acquired for 69 tumors with sufficient sample (Table S2). As with proteins, phosphopeptide abundances were calculated relative to the\",\n",
            "          \"phosphopeptide abundances were calculated relative to the pooled reference sample. Because isobaric labeling was performed prior to splitting the samples for proteome and phosphoproteome analyses, phosphopeptide abundance could be corrected for changes in\",\n",
            "          \"phosphopeptide abundance could be corrected for changes in parent protein abundance to identify differences in the relative extent of phosphorylation at specific sites for each protein (Table S2). Overall, we achieved a protein dynamic range encompassing\",\n",
            "          \"S2). Overall, we achieved a protein dynamic range encompassing >4 orders of magnitude, ranging from low-level transcription factors to abundant structural proteins i.e., actin and tubulin.\",\n",
            "          \"Proteogenomic landscape of HGSC\",\n",
            "          \"The degree to which alterations observed at the genome and transcriptome levels are manifested at the protein level is variable, both qualitatively and quantitatively, and partially driven by multiple levels of post-transcriptional regulation. To identify\",\n",
            "          \"multiple levels of post-transcriptional regulation. To identify peptides encoded by single amino acid variants (SAAVs), splice variants, and novel exons documented by TCGA, mass spectra were searched against a custom graph database containing all peptide\",\n",
            "          \"searched against a custom graph database containing all peptide variations projected from the TCGA genomic analyses of the cohort using a multi-stage analysis pipeline (Supplemental Information). The most frequently observed variant peptides represented\",\n",
            "          \"The most frequently observed variant peptides represented SAAVs and gene-level events. The evidence supporting each novel event is detailed in Table S2, including event type, genomic location, abundance information, and estimated false discovery rate\",\n",
            "          \"abundance information, and estimated false discovery rate (FDR). The validity of these variant peptides was further evaluated by the synthesis of twenty examples selected at random across the entire range of spectrum match scores. We obtained tandem mass\",\n",
            "          \"entire range of spectrum match scores. We obtained tandem mass spectra for all twenty synthetic peptides that matched spectra from our analyses, strongly supporting our observation of these variant peptides; three representative examples are given in\",\n",
            "          \"variant peptides; three representative examples are given in Figure S1E. These results demonstrate the ability to confidently detect, identify and validate genome-level predictions at the protein level. More novel peptides would likely be observed if\",\n",
            "          \"protein level. More novel peptides would likely be observed if there was sufficient sample for more extensive fractionation and/or replicate analyses; these limitations preclude any conclusions about the biological significance of unobserved events. For\",\n",
            "          \"about the biological significance of unobserved events. For example, only 2 mutant p53 peptides were identified despite the presence of p53 mutations in all tumors examined. Such low coverage can occur for reasons that include: excessively large or small\",\n",
            "          \"can occur for reasons that include: excessively large or small tryptic fragments, inability to distinguish some amino acids, some possible biases against highly hydrophobic and hydrophilic peptides, or low abundance peptides co-eluting with very high\",\n",
            "          \"peptides, or low abundance peptides co-eluting with very high abundance peptides.\",\n",
            "          \"To assess the potential for post-transcriptional regulation (e.g., translational efficiency or protein degradation), we compared each protein to its corresponding transcript across all tumors, and correlation was assessed for those pairs with reliable\",\n",
            "          \"and correlation was assessed for those pairs with reliable measurements for both mRNA and protein, i.e., proteins with a corresponding mRNA measurement observed in all 169 HGSC tumors where sample variability exceeded technical variability (3,196 pairs).\",\n",
            "          \"variability exceeded technical variability (3,196 pairs). We excluded 391 proteins observed without a corresponding mRNA (e.g., HBA1) or discordant gene symbol annotation in the protein database (e.g., THOC4). Overall, 79.4% of the mRNA:protein pairs\",\n",
            "          \"(e.g., THOC4). Overall, 79.4% of the mRNA:protein pairs showed statistically significant positive Spearman correlations (Benjamini-Hochberg adjusted p value <0.01; Figure 1) when changes in mRNA abundance were compared to changes in relative protein\",\n",
            "          \"in mRNA abundance were compared to changes in relative protein abundance. The observed median r value of 0.45 for each mRNA-protein pair across all 169 tumors is similar to observations in colorectal cancer, breast cancer and mouse tissues, although less\",\n",
            "          \"cancer, breast cancer and mouse tissues, although less than found for cell lines. In comparison, the median correlation of paired protein measurements from the same sample, but measured at two sites, was substantially higher, at 0.69 (Figure S1).\",\n",
            "          \"A wide range of mRNA:protein correlations was observed. In general, weaker correlations were observed for highly stable and abundant proteins associated with housekeeping or non-intrinsic functions (e.g., ribosomes, mRNA splicing, oxidative\",\n",
            "          \"functions (e.g., ribosomes, mRNA splicing, oxidative phosphorylation, complement cascade), while more dynamic proteins known to be transcriptionally regulated in response to nutrient demand or other perturbations (e.g., nucleotide metabolism, amino acid\",\n",
            "          \"or other perturbations (e.g., nucleotide metabolism, amino acid metabolism, acute inflammatory responses) were more highly correlated (Figure 1 and Table S3). This result is consistent with a previous colorectal cancer study and supports the hypothesis\",\n",
            "          \"a previous colorectal cancer study and supports the hypothesis that while many biological functions are primarily regulated by mRNA abundance, post-transcriptional mechanisms likely have an important role in regulating certain house-keeping functions.\",\n",
            "          \"important role in regulating certain house-keeping functions. Additionally, we found that 23 mRNAs lacking polyA tails displayed lower correlation (mean \\u22120.15) with their cognate proteins than did polyadenylated mRNAs, consistent with the decreased\",\n",
            "          \"than did polyadenylated mRNAs, consistent with the decreased stability of mRNAs lacking polyA tails.\",\n",
            "          \"Clustering of tumors based on protein abundance\",\n",
            "          \"HGSC is the most common of the four major histological subtypes of epithelial ovarian cancer, and is characterized by distinct morphological features. Recent studies using mRNA abundance data have suggested four transcriptomic HGSC subtypes designated as\",\n",
            "          \"have suggested four transcriptomic HGSC subtypes designated as differentiated, immunoreactive, mesenchymal, and proliferative. To build an unbiased molecular taxonomy of ovarian HGSC, we used protein abundance data from the 169 tumors to identify subtypes\",\n",
            "          \"protein abundance data from the 169 tumors to identify subtypes that might show biological differences that could be exploited in future studies. Figure 2 shows the resulting clustering analysis of individual tumors (vertical columns) by protein abundance\",\n",
            "          \"of individual tumors (vertical columns) by protein abundance (horizontal rows). The cluster assignment for each sample is provided in Table S4, and a consensus value matrix for the subtype comparisons is shown in Figure S2A. The results of a Weighted Gene\",\n",
            "          \"is shown in Figure S2A. The results of a Weighted Gene Co-expression Network Analysis (WGCNA) of protein functional enrichment by subgroups are provided for the protein clusters, in Figure S2B and Table S4. The enrichment of KEGG and Reactome ontologies\",\n",
            "          \"and Table S4. The enrichment of KEGG and Reactome ontologies in the WGCNA-derived modules are shown in Figures S2C and S2D, respectively; membership of enriched pathways is provided in Table S3.\",\n",
            "          \"Four of the proteomic clusters showed a clear correspondence to the mesenchymal, proliferative, immunoreactive, and differentiated subtypes defined by the TCGA transcriptome analysis (Figure 2 and Figure S2E). A relatively small fifth cluster of tumors\",\n",
            "          \"2 and Figure S2E). A relatively small fifth cluster of tumors significantly enriched in proteins associated with extracellular matrix interactions, erythrocyte and platelet functions, and the complement cascade was also observed using multiple approaches,\",\n",
            "          \"complement cascade was also observed using multiple approaches, including model based clustering with Bayesian information criteria, consensus clustering, and VISDA-based sub-phenotype clustering. This new group could not be attributed to tissue source\",\n",
            "          \"This new group could not be attributed to tissue source site sampling bias or any other meta-data category, but may be related to tumor characteristics, including vascularization and tumor content, as the Tumor Purity score for this subtype (and the\",\n",
            "          \"content, as the Tumor Purity score for this subtype (and the Mesenchymal subtype) was significantly lower than that of the other three subtypes (Figure S2F). The clinical relevance of these protein-based clusters will require validation in independent\",\n",
            "          \"protein-based clusters will require validation in independent HGSC sample sets, particularly as no significant difference in survival was observed (Figure S2G), similar to mRNA-based clustering analysis.\",\n",
            "          \"Proteomic analysis of CNA effects\",\n",
            "          \"Chromosomal instability, marked by extensive copy number alterations (CNAs) in each tumor, is a hallmark of HGSC, and a likely source of driver alterations in this disease. CNAs can affect the abundance of proteins at the same locus (cis effects), and may\",\n",
            "          \"abundance of proteins at the same locus (cis effects), and may also act in trans, either directly or indirectly.\",\n",
            "          \"Hypothesizing that CNAs with strong trans effects are more likely to elicit a molecular phenotype and confer selective advantages, we sought to identify those CNA regions that have the broadest effect on protein expression. In all, 29,393 CNA loci (Table\",\n",
            "          \"effect on protein expression. In all, 29,393 CNA loci (Table S5) were compared to our global proteomics data with 950,209 CNA/protein pairs (0.72% of the total) exhibiting significant association (Benjamini-Hochberg adjusted p value <0.01). We provide a\",\n",
            "          \"(Benjamini-Hochberg adjusted p value <0.01). We provide a complete list of the significantly associated proteins for each CNA in Table S5. The diagonal line evident in Figure 3 corresponds to cis effects, and vertical stripes correspond to trans effects\",\n",
            "          \"cis effects, and vertical stripes correspond to trans effects where changes in copy number affect expression of numerous proteins across the genome. We performed the same analysis for mRNA to identify sites where associations are transcriptionally\",\n",
            "          \"mRNA to identify sites where associations are transcriptionally mediated. A similar number of CNA/mRNA pairs were found to be significantly associated (1,113,164 at Benjamini-Hochberg adjusted p value <0.01; Figure 3). This analysis revealed regions on\",\n",
            "          \"p value <0.01; Figure 3). This analysis revealed regions on chromosomes 2, 7, 20, and 22 correlated in trans with more than 200 proteins. In contrast to colorectal cancer where most of the trans-regulation of protein by CNA was accompanied by similar\",\n",
            "          \"trans-regulation of protein by CNA was accompanied by similar changes in mRNA, we observed several loci associated with differences in protein abundance without a corresponding change in mRNA. For example, large regions on chromosome 2 have relatively\",\n",
            "          \"For example, large regions on chromosome 2 have relatively little trans effect on mRNA levels, but are associated with more than 200 proteins in trans. Dissecting the mechanisms by which a specific CNA can alter protein levels in trans, without affecting\",\n",
            "          \"CNA can alter protein levels in trans, without affecting the corresponding mRNA is difficult, given the extent of the amplified or deleted regions and the numerous genes affected at a given locus. Possible mechanisms include cis regulation of proteins\",\n",
            "          \"locus. Possible mechanisms include cis regulation of proteins associated with mRNA stability and translational efficiency, such as microRNAs and RNA binding proteins.\",\n",
            "          \"Given the complex pattern of CNAs observed in HGSC, it has been difficult to identify a limited number of high impact genomic alterations that could function as drivers of the disease. We interrogated the trans-affected proteins associated with each\",\n",
            "          \"interrogated the trans-affected proteins associated with each putative CNA for common functions using pathways defined by the KEGG, NCI pathway interaction database (PID) and Reactome databases (Supplemental Information). Proteins associated with cell\",\n",
            "          \"(Supplemental Information). Proteins associated with cell invasion and migration, and proteins related to immune function appeared to be enriched in association with multiple CNAs (Figure S3; Table S3). These observations suggest a convergence of multiple\",\n",
            "          \"Table S3). These observations suggest a convergence of multiple CNA targets on a common set of biological functions, namely motility/invasion and immune regulation, functions that are among the hallmarks of cancer.\",\n",
            "          \"Association of CNA trans-affected proteins with overall survival\",\n",
            "          \"Availability of the TCGA survival data allowed us to use trans-affected protein data from the most influential CNAs (e.g., the four altered regions described on chromosomes 2, 7, 20, and 22; Figure 3) to build a model of overall survival. Because each CNA\",\n",
            "          \"3) to build a model of overall survival. Because each CNA affects many proteins, we used a regression approach that identifies parsimonious Cox proportional hazards models with maximal predictive ability from the list of significantly correlated proteins\",\n",
            "          \"ability from the list of significantly correlated proteins for that CNA. We trained models on the proteomics data from PNNL (82 tumors) and tested the ability of the model to predict survival times in the data from JHU (87 tumors, not including the 32\",\n",
            "          \"times in the data from JHU (87 tumors, not including the 32 overlapping). Each of the four most influential CNA regions produced models that were strongly associated with patient survival (p value <0.01, FDR <0.5% based on randomly selected proteins). A\",\n",
            "          \"value <0.01, FDR <0.5% based on randomly selected proteins). A Kaplan-Meier plot illustrating the predictive value of each of the locus-specific models after validation is shown in Figure S4A; for the Kaplan-Meier plots, \\u2018high\\u2019 and \\u2018low\\u2019 expression were\",\n",
            "          \"for the Kaplan-Meier plots, \\u2018high\\u2019 and \\u2018low\\u2019 expression were defined relative to the median for that signature across all samples, splitting patients into two equal groups of 45 (Table S6). We examined the overlap in predictions for patients for the four\",\n",
            "          \"examined the overlap in predictions for patients for the four signatures and found that the predictions were unanimous (either high or low signature) for 62% of patients and 16% were evenly split, with the remaining having one dissenting signature. This\",\n",
            "          \"split, with the remaining having one dissenting signature. This suggested the utility of combining these signatures, using a voting method where the number of high or low calls was counted for each patient; this substantially improved prediction of\",\n",
            "          \"for each patient; this substantially improved prediction of survival time (p value 1.9e-6; Figure 4). We also examined the effects of tumor stage, tumor grade, patient age, surgical outcome, and platinum status, and found that our proteomic signatures\",\n",
            "          \"and platinum status, and found that our proteomic signatures were not improved by inclusion of these variables.\",\n",
            "          \"For each of these locus-specific models we analyzed the enrichment of genes and their regulatory sequences associated with outcome, and found all four models had genes significantly enriched (Benjamini-Hochberg adjusted p value <0.05) in binding sites for\",\n",
            "          \"adjusted p value <0.05) in binding sites for the proliferation-associated serum response factor (SRF), suggesting that SRF activity may be important in ovarian cancer outcome. Additionally, there was significant enrichment of proteins involved in the\",\n",
            "          \"there was significant enrichment of proteins involved in the regulation of actin cytoskeleton, apoptosis, and adherens junction (Benjamini-Hochberg adjusted p value <0.05). We examined the protein abundance and phosphorylation status of SRF between short\",\n",
            "          \"abundance and phosphorylation status of SRF between short and long surviving groups and observed that they were higher in short surviving patients, but only slightly. Thus, although SRF alone was not predictive, the trends in SRF abundance and\",\n",
            "          \"SRF alone was not predictive, the trends in SRF abundance and phosphorylation are consistent with the observation of enrichment for proteins potentially regulated by SRF binding. Details of the larger gene set analysis are provided in Supplemental\",\n",
            "          \"of the larger gene set analysis are provided in Supplemental Information.\",\n",
            "          \"Several proteins shared across all the signatures are known to be involved in cancer processes. Catenin B2 (CTNNA2) is a cell-cell adhesion protein and tumor suppressor. The Rho GDP dissociation inhibitor (GDI) beta (ARHGDIB) is involved in invasion and\",\n",
            "          \"inhibitor (GDI) beta (ARHGDIB) is involved in invasion and migration in many cancers, and overexpression correlates with progression in pancreatic carcinoma. Protein kinase C and casein kinase substrate in neurons protein 2 (PACSIN2) is a repressor of\",\n",
            "          \"substrate in neurons protein 2 (PACSIN2) is a repressor of cellular migration. Finally, the GTP-binding nuclear protein Ran, is a prognostic marker associated with increased survival in epithelial ovarian cancer. The association of these previously\",\n",
            "          \"epithelial ovarian cancer. The association of these previously described survival-related proteins with genes affected in trans by CNAs suggests a potential mechanism for the parallel activation of multiple pathways associated with poor prognosis in HGSC.\",\n",
            "          \"As a comparison we applied the previously described Provar signature, comprising five protein and four phosphoproteins that showed good survival prediction in the TCGA ovarian cancer dataset. We observed all proteins in the Provar signature in our\",\n",
            "          \"We observed all proteins in the Provar signature in our proteomic data, but only one of the phosphosites (EGFR Y1173). Thus, we used the RPPA data from the original signature (Figure S4B). We found that the Provar signature was prognostic of survival\",\n",
            "          \"We found that the Provar signature was prognostic of survival (Benjamini-Hochberg adjusted p value = 0.11) in the 67 patients examined here (those with phosphoprotein data and had somatic TP53 mutations), but the statistical power of Provar is not as high\",\n",
            "          \"mutations), but the statistical power of Provar is not as high as the statistical power of signatures derived from the CNA trans-affected proteins in this dataset. In addition, integrating the Provar signature with the CNA signatures did not improve\",\n",
            "          \"the Provar signature with the CNA signatures did not improve survival prediction (Figure S4C).\",\n",
            "          \"Identification of proteins associated with chromosomal structural abnormality\",\n",
            "          \"The degree of chromosomal instability exhibited by a tumor can be represented by a calculated chromosome instability (CIN) index, as described previously for lung cancer. Identification of proteins associated with CIN may provide information on the\",\n",
            "          \"of proteins associated with CIN may provide information on the processes contributing to chromosomal instability, while the analysis of trans-affected proteins described above is more closely related to the downstream consequences of specific CNAs. Using\",\n",
            "          \"related to the downstream consequences of specific CNAs. Using a bootstrapping method described in Supplemental Information, we identified a proteomic signature including 128 proteins (Table S7) showing significant correlation with CIN index\",\n",
            "          \"(Table S7) showing significant correlation with CIN index (Benjamini-Hochberg adjusted p value <1e-4, Spearman correlation; Figure S5A). Functional annotation of these proteins (Figure S5B) showed that proteins up-regulated in tumors with high CIN index\",\n",
            "          \"showed that proteins up-regulated in tumors with high CIN index were preferentially involved in chromatin organization (p value 6.90e-5), whereas proteins up-regulated in tumors with low CIN index were more often associated with cell death (p value\",\n",
            "          \"CIN index were more often associated with cell death (p value 2.13e-5). Correlation analysis of protein abundances and CIN indicated that a small number of proteins could account for the majority of the CIN (Figure 5); two of the most strongly associated\",\n",
            "          \"of the CIN (Figure 5); two of the most strongly associated proteins, CHD4 and CHD5, are known to be involved in chromatin organization.\",\n",
            "          \"Identification of proteins associated with HRD status\",\n",
            "          \"Since HRD is associated with susceptibility to PARP inhibitors and improved survival, we sought to elucidate systemic changes associated with HRD and identify biomarkers that might be used to stratify patients for treatment. We defined tumors with HRD as\",\n",
            "          \"stratify patients for treatment. We defined tumors with HRD as having either germline or somatic mutations in BRCA1 or BRCA2, or BRCA1 promoter methylation, or homozygous deletion of PTEN, with an overall survival >1.5 years, while non-HRD patients were\",\n",
            "          \"an overall survival >1.5 years, while non-HRD patients were defined as lacking these genomic aberrations and with a follow-up or time-to-death <2.5 years; additional selection criteria include available residual tissue volume and a tumor tissue\",\n",
            "          \"include available residual tissue volume and a tumor tissue contamination score estimated using copy number alterations. Applying differential dependency network (DDN) analysis on a set of 171 BRCA1- or BRCA2-related proteins curated from the literature\",\n",
            "          \"BRCA1- or BRCA2-related proteins curated from the literature and from the cBio portal, we identified a sub-network of 30 proteins that displayed co-expression patterns differentiating HRD from non-HRD patients (Table S7; Figure 6). Several of the proteins\",\n",
            "          \"non-HRD patients (Table S7; Figure 6). Several of the proteins in these modules are known to be involved in histone acetylation or deacetylation, e.g., HDAC1, RBBP4, RBBP7, and EP300, and HUS1.\",\n",
            "          \"Although statistical association cannot distinguish between drivers and consequences of HRD status, the observed enrichment of proteins associated with histone acetylation motivated us to use an effective database search strategy (Supplemental\",\n",
            "          \"us to use an effective database search strategy (Supplemental Information) to identify and quantify acetylated peptides from the global proteomic data. Comparative analysis of 399 acetylated peptides identified 15 acetylated peptides with significant\",\n",
            "          \"peptides identified 15 acetylated peptides with significant differences between the HRD and non-HRD samples (Table S7). Among these, dual acetylation at K12 and K16 of histone H4 showed a significant difference between HRD and non-HRD samples (Figure 6);\",\n",
            "          \"difference between HRD and non-HRD samples (Figure 6); differential acetylation of K12 and K16 was verified using synthetic peptides and targeted analysis using Sequential Windowed data independent Acquisition of the Total High-resolution Mass Spectra\",\n",
            "          \"Acquisition of the Total High-resolution Mass Spectra (SWATH-MS). In cell line data, acetylation of H4 has previously been reported to be involved in the choice of DNA double strand break (DSB) repair pathways (homologous recombination or non-homologous\",\n",
            "          \"repair pathways (homologous recombination or non-homologous end joining. This relationship is regulated partially by HDAC1, a protein also identified in the DDN analysis. We observed a significant enrichment of HDAC1 and its co-regulated proteins in\",\n",
            "          \"enrichment of HDAC1 and its co-regulated proteins in tumors with HRD and low H4 acetylation relative to non-HRD tumors with high H4 acetylation (DDN analysis: permutation tests, p value <0.05; differentially acetylated peptides: t-tests, p value <0.05\",\n",
            "          \"differentially acetylated peptides: t-tests, p value <0.05 with an estimated FDR <0.5% by bootstrap/permutation tests). The combined observations of increased HDAC1 and associated proteins at the pathway level, together with decreased acetylation of H4 in\",\n",
            "          \"the pathway level, together with decreased acetylation of H4 in HRD patients at the PTM level, provide insight regarding the potential role of HDAC1 in modulating the choice of DSB repair pathways.\",\n",
            "          \"Phosphoproteomic analysis of pathways associated with survival\",\n",
            "          \"An initial set of 24,464 different phosphopeptides (21,298 unique phosphorylation sites) contained within 4,420 proteins having data on net differences in phosphorylation (Table S2) was obtained. Since ischemia of the TCGA tumor samples may alter\",\n",
            "          \"obtained. Since ischemia of the TCGA tumor samples may alter phosphopeptide abundance, we removed any phosphorylation sites previously shown to be altered by ischemia. We also removed three tumor samples having substantially higher than average missing\",\n",
            "          \"tumor samples having substantially higher than average missing values and two tumor samples lacking somatic TP53 mutations (Supplemental Information). This yielded a final set of 7,675 different phosphopeptides (6,802 unique phosphorylation sites) from\",\n",
            "          \"phosphopeptides (6,802 unique phosphorylation sites) from 2,324 proteins (Table S2) that were mapped to pathways via the NCI PID. The average net phosphorylation of all phosphopeptides mapping to a given pathway (i.e., the statistical average across all\",\n",
            "          \"to a given pathway (i.e., the statistical average across all phosphoproteins known to be in the pathway, rather than the statistically significant individual proteins) was used as a measure of pathway activity and compared between short survivors\",\n",
            "          \"of pathway activity and compared between short survivors (deceased surviving <3 years; N=17) and long survivors (patients surviving >5 years; N=19) using both proteomic and phosphoproteomic data.\",\n",
            "          \"An issue in ranking the pathways is the distinction between phosphopeptides that are unique to a single pathway, such as the receptor peptides themselves, vs. peptides that map to proteins shared between pathways. In order to address this issue, we\",\n",
            "          \"shared between pathways. In order to address this issue, we performed two analyses: one of all proteins associated with a specific pathway in the NCID database, and a second analysis in which proteins common to multiple pathways were excluded. Figure 7A\",\n",
            "          \"proteins common to multiple pathways were excluded. Figure 7A shows a ranking of activated pathways, as inferred from the analysis of pathway specific components (i.e., differences in mRNA, protein and phosphoprotein levels) in short vs. long survivors.\",\n",
            "          \"protein and phosphoprotein levels) in short vs. long survivors. Fifteen pathways showed increased phosphorylation at Benjamini-Hochberg adjusted p values <0.01, in contrast with transcriptional data for the same samples which yielded only one\",\n",
            "          \"data for the same samples which yielded only one statistically significantly increased pathway, androgen receptor signaling, which was also increased at the phosphoprotein level (Figure 7A and Supplemental Information). All pathways indicated in Figure 7A\",\n",
            "          \"Supplemental Information). All pathways indicated in Figure 7A were increased in association with short survival relative to long survival, though some individual components within a pathway showed opposite changes (e.g. JNK1 transcript and protein\",\n",
            "          \"showed opposite changes (e.g. JNK1 transcript and protein levels, see Figure 7B). The RhoA-regulatory, PDGFRB, and integrin-like kinase pathways emerged as the most activated in this analysis (Figure 7A). Figure 7B shows the common and unique elements of\",\n",
            "          \"(Figure 7A). Figure 7B shows the common and unique elements of the PDGFRB pathway, and the trends in mRNA, protein, and phosphoprotein levels in the comparison of short vs. long survivors, illustrating the benefit of combined protein and phosphoprotein\",\n",
            "          \"illustrating the benefit of combined protein and phosphoprotein measurements, as well as the substantial contribution of protein phosphorylation data to the overall analysis.\",\n",
            "          \"We compared our results to the 31 phosphoproteins represented on the RPPA arrays used in the original TCGA ovarian cancer analysis; RPPA identified differential phosphorylation of ERK1, RAF1, and STAT3 in the same patient samples analyzed by MS. For short\",\n",
            "          \"and STAT3 in the same patient samples analyzed by MS. For short surviving vs. long surviving patients, ERK1 showed statistically significantly increased phosphorylation by RPPA, on the same ERK1 phosphopeptides as found using MS, while AKT1, RAF1 and\",\n",
            "          \"ERK1 phosphopeptides as found using MS, while AKT1, RAF1 and STAT3 showed increased phosphorylation by both RPPA and MS-based analyses, but based upon distinct phosphopeptides (Table S2). Good overlap was observed between RPPA and MS results for\",\n",
            "          \"S2). Good overlap was observed between RPPA and MS results for phosphoproteins on the RPPA array, but we did not observe three of the specific phosphosites from the Provar survival signature in our MS data, suggesting that proteomics is more sensitive to\",\n",
            "          \"in our MS data, suggesting that proteomics is more sensitive to overall pathway activity than specific regulatory events in the pathway itself. However, many of the phosphopeptides identified by MS phosphoproteomics as components of the PDGFR-beta pathway\",\n",
            "          \"by MS phosphoproteomics as components of the PDGFR-beta pathway were not represented in the RPPA, illustrating the ability to identify phosphorylation events not readily accessible with currently available antibodies (Figure 7B).\",\n",
            "          \"DISCUSSION\",\n",
            "          \"The addition of proteomic information, including post-translational modifications, to the rich genomic and transcriptomic data available from the ovarian HGSC samples analyzed by TCGA has provided additional insights into the biology of HGSC, including\",\n",
            "          \"additional insights into the biology of HGSC, including identification of changes at the level of pathway activation. Integration of genomic, proteomic, and phosphoproteomic measurements identified differentially regulated pathways and functional modules\",\n",
            "          \"differentially regulated pathways and functional modules that displayed significant associations with patient outcomes, including survival and HRD status. Cox proportional hazard analysis of proteins associated with CNAs identified overall survival\",\n",
            "          \"of proteins associated with CNAs identified overall survival signatures enriched for targets of the proliferation-associated transcription factor SRF, and integrated proteomic, phosphoproteomic and transcriptomic data identified pathways that\",\n",
            "          \"and transcriptomic data identified pathways that differentiated patients on the basis of survival, including the PDGFR-beta signaling pathway associated with angiogenesis, the RhoA regulatory and integrin-linked kinase pathways associated with cell\",\n",
            "          \"and integrin-linked kinase pathways associated with cell mobility and invasion, and pathways associated with chemokine signaling and adaptive immunity. Proteins associated with cell invasion and motility emerged from both the integration of CNAs with\",\n",
            "          \"and motility emerged from both the integration of CNAs with protein abundance data and the phosphoproteomic investigation of up-regulated pathways in short versus long survivors. Despite the complexity of genomic alterations that characterize HGSC, these\",\n",
            "          \"complexity of genomic alterations that characterize HGSC, these analyses suggest a functional convergence on a subset of key pathways. The association of increased invasiveness and motility with short overall survival in this study may help to explain\",\n",
            "          \"with short overall survival in this study may help to explain more aggressive mechanisms of dissemination in ovarian cancer, including recently reported hematogenous metastasis, in addition to previously described lymphatic spread and known peritoneal\",\n",
            "          \"to previously described lymphatic spread and known peritoneal spread by direct extension.\",\n",
            "          \"Focusing on functional modules has also revealed potential drivers of HGSC and more robust signatures for potentially stratifying ovarian HGSC patients into distinct cancer phenotypes to inform therapeutic management. For example, the identification of\",\n",
            "          \"therapeutic management. For example, the identification of specific acetylation events associated with HRD, e.g., the simultaneous acetylation of K12 and K16 on histone H4, may provide an alternative biomarker of HRD and a rationale for the selection of\",\n",
            "          \"biomarker of HRD and a rationale for the selection of patients in future clinical trials of HDAC inhibitors alone or in combination with PARP inhibition. This may help to resolve the current discrepancy between the initial observation of limited single\",\n",
            "          \"discrepancy between the initial observation of limited single agent activity of HDAC inhibitors in ovarian cancer, and more recent findings of a >40% response rate when used in combination with cytotoxic chemotherapy in platinum-resistant patients.\",\n",
            "          \"with cytotoxic chemotherapy in platinum-resistant patients. Similarly, comprehensive interrogation of protein phosphorylation identified multiple pathways with significantly increased phosphorylation in patients with poor clinical outcomes. Specifically,\",\n",
            "          \"in patients with poor clinical outcomes. Specifically, the observed activation of PDGFR pathways in a subset of patients with short overall survival could potentially stratify patients for selective enrollment in trials of anti-angiogenic therapy, which\",\n",
            "          \"enrollment in trials of anti-angiogenic therapy, which is particularly relevant to current controversies over the use of bevacuzimab as a first-line therapy in ovarian cancer.\",\n",
            "          \"Overall, this work illustrates the ability of proteomics to complement genomics in providing additional insights into pathways and processes that drive ovarian cancer biology and how these pathways are altered in correspondence with clinical phenotypes.\",\n",
            "          \"are altered in correspondence with clinical phenotypes. The comprehensive proteomic measurements for the HGSC tumor samples provide a public resource of information. Importantly, the ability to identify PTMs revealed a strong association between specific\",\n",
            "          \"to identify PTMs revealed a strong association between specific histone acetylation events and the HRD phenotype. In addition, the enhanced view of pathway activity provided by measurements of protein phosphorylation provides a foundation for linking\",\n",
            "          \"of protein phosphorylation provides a foundation for linking genotype to proteotype, and ultimately to phenotype for understanding the molecular basis of cancer.\",\n",
            "          \"EXPERIMENTAL PROCEDURES\\n\\nFull methods are available in Extended Experimental Procedures.\\n\\nTumor Samples\",\n",
            "          \"Tumor Samples\\n\\nAll tumor samples for the current study were obtained through the TCGA Biospecimen Core Resource as described previously. Samples were selected based on overall survival, HRD status and availability.\\n\\nQuantitative Proteomics\",\n",
            "          \"Tissue proteins were extracted and digested with trypsin; at each site, a portion of each resulting sample was also used to create a pooled reference in which each tumor sample contributed an equal percentage by peptide mass. The pooled reference sample\",\n",
            "          \"equal percentage by peptide mass. The pooled reference sample was included in each multiplexed, isobaric labeling experiment to enable cross-experiment comparison in the entire sample cohort. The patient samples and the pooled reference sample were each\",\n",
            "          \"The patient samples and the pooled reference sample were each labeled by different iTRAQ reagents (Sciex), combined, fractionated, and split for integrated, quantitative global proteome (10%) and phosphoproteome (90%) analysis using LC-MS/MS on an\",\n",
            "          \"(10%) and phosphoproteome (90%) analysis using LC-MS/MS on an Orbitrap Velos mass spectrometer (Thermo Scientific). Raw data were processed for peptide identification by database searching and identified peptides were assembled as proteins and mapped to\",\n",
            "          \"identified peptides were assembled as proteins and mapped to gene identifiers for proteogenomic comparisons. Quantitation was achieved by comparing the iTRAQ reporter ion intensities in each sample.\",\n",
            "          \"Comparison of mRNA and Protein Subtypes\",\n",
            "          \"A model-based clustering approach was used to model the protein abundance data as a mixture of subtypes. Bayesian information criteria, statistical resampling and VISDA-based sub-phenotype clustering approaches provided similar results with a stable\",\n",
            "          \"clustering approaches provided similar results with a stable optimization on five clusters as determined by consensus clustering. WGCNA analysis was performed to infer genes or gene networks that drive subtyping into 5 clusters, followed by correlation\",\n",
            "          \"that drive subtyping into 5 clusters, followed by correlation with subtype as a trait.\",\n",
            "          \"Integration of Proteomics and CNA data\",\n",
            "          \"Spearman correlation coefficients and corresponding adjusted p values were calculated for each protein/transcript by CNA locus, and gene set enrichment analysis was used to infer function for groups of proteins significantly correlated with a given CNA\",\n",
            "          \"groups of proteins significantly correlated with a given CNA locus. Regression analysis was applied to the list of trans-affected protein correlated with each CNA to generate parsimonious Cox proportional hazards models with maximal predictive ability,\",\n",
            "          \"proportional hazards models with maximal predictive ability, using the PNNL data for training and JHU data for testing. Kaplan-Meier plots were used to visualize performance in predicting overall survival and progression-free survival. A CIN index was\",\n",
            "          \"overall survival and progression-free survival. A CIN index was calculated for each sample as the mean absolute values of copy number measurements at the 29,393 selected loci. Bootstrap resampling was used to select proteins correlated with CIN index at\",\n",
            "          \"was used to select proteins correlated with CIN index at high confidence.\",\n",
            "          \"Protein Acetylation and HRD\",\n",
            "          \"Acetylated peptides were identified by searching the global proteomics data for dynamic acetylation to lysines (+42 Da). Acetylation levels were compared between HRD and non-HRD cases by t-test. Targeted proteomics (SWATH) was used to orthogonally\",\n",
            "          \"by t-test. Targeted proteomics (SWATH) was used to orthogonally quantify the acetylated peptide with synthetic peptides as internal standard.\",\n",
            "          \"Survival-related Pathways Analysis\",\n",
            "          \"Phosphoproteome data from the short and long survivors were mapped to signaling pathways in the NCI pathway information database,(PID (http://pid.nci.nih.gov) using the gene names. For each signaling pathway in the PID, relative abundances for all\",\n",
            "          \"each signaling pathway in the PID, relative abundances for all phosphopeptides mapping to any pathway component were identified and separated into short and long survivor groups. The difference in distributions between the two sets of pathway-specific\",\n",
            "          \"in distributions between the two sets of pathway-specific peptides, i.e., those associated with either short survivors or long long survivors, was then assessed using a two-tailed t test. Similar enrichment analyses were also performed using protein\",\n",
            "          \"Similar enrichment analyses were also performed using protein abundance, mRNA abundance, and CNA.\",\n",
            "          \"Data Repository\",\n",
            "          \"All of the primary mass spectrometry data on TCGA tumor samples are deposited at the CPTAC Data Coordinating Center as raw and mzML files and complete protein assembly data sets for public access (https://cptac-data-portal.georgetown.edu).\",\n",
            "          \"Supplementary Material\",\n",
            "          \"This is a PDF file of an unedited manuscript that has been accepted for publication. As a service to our customers we are providing this early version of the manuscript. The manuscript will undergo copyediting, typesetting, and review of the resulting\",\n",
            "          \"undergo copyediting, typesetting, and review of the resulting proof before it is published in its final form. Please note that during the production process errors may be discovered which could affect the content, and all legal disclaimers that apply to\",\n",
            "          \"affect the content, and all legal disclaimers that apply to the journal pertain.\",\n",
            "          \"Author Contributions\",\n",
            "          \"Study conception and design: HZ, TL, ZZ, RDS, DWC, KDR. Investigation: performed experiment or data collection: TL, SS, FY, JZ, LijunC, PS, PA, YT, MAG, TRC, CC, ST, SN, RJM, HengZ. Computation and statistical analysis: ZZ, SHP, BaiZ, JEM, VAP, LiC, DR,\",\n",
            "          \"and statistical analysis: ZZ, SHP, BaiZ, JEM, VAP, LiC, DR, MEM, SWC, SW, JW, DLT, DF, VB, YW, BingZ. Data interpretation and biological analysis: TL, HZ, SHP, ZZ, BaiZ, JEM, VAP, LiC, DR, SN, CW, IMS, AP, MPS, DAL, RDS, DWC, KDR. Writing/manuscript\",\n",
            "          \"SN, CW, IMS, AP, MPS, DAL, RDS, DWC, KDR. Writing/manuscript preparation and revision: HZ, TL, ZZ, SHP, JEM, DR, VAP, LeslieC, HR, IMS, AP, MPS, DAL, RDS, DWC, KDR. Supervision and administration: HR, ESB, TH, RCR, LS, RDS, DWC, KDR.\",\n",
            "          \"The authors declare no competing financial interests.\\n\\n           SUPPLEMENTAL INFORMATION         \\n\\nSupplemental information includes Extended Experimental Procedures, 5 figures, and 8 tables and can be found with this article online.\",\n",
            "          \"Correlations between mRNA and protein abundance in TCGA tumors\",\n",
            "          \"mRNA and protein were correlated across all the samples, resulting in 90.6% positive correlations, and 79.4% were significantly correlated (Benjamini-Hochberg adjusted p value <0.01), with a mean Spearman\\u2019s correlation of 0.38 and a median value of 0.45\",\n",
            "          \"mean Spearman\\u2019s correlation of 0.38 and a median value of 0.45 (top panel). Different biological pathways and processes showed significantly different levels of correlation (bottom panel). Metabolic pathways and the interferon response displayed high\",\n",
            "          \"Metabolic pathways and the interferon response displayed high mRNA-protein correlation, and ribosome, mRNA splicing, oxidative phosphorylation, and complement and coagulation cascade were poorly correlated. The mean correlation is shown in parentheses\",\n",
            "          \"poorly correlated. The mean correlation is shown in parentheses followed by Benjamini-Hochberg adjusted p values calculated using a Kolmogorov\\u2013Smirnov test following the functional group names from MSIGDB. Blue bars indicate positive correlations and\",\n",
            "          \"names from MSIGDB. Blue bars indicate positive correlations and yellow indicates negative correlations; individual proteins (represented as bars on the x-axis) are sorted by correlation from low to high (bottom panel).\",\n",
            "          \"Proteomic subtypes and corresponding driving protein modules\",\n",
            "          \"Global proteomics abundance is shown in a heatmap with TCGA samples represented by columns ordered by protein subtype where rows represent proteins. Color of each cell indicates z-score (log2 of relative abundance scaled by proteins\\u2019 standard deviation)\",\n",
            "          \"of relative abundance scaled by proteins\\u2019 standard deviation) of the protein in that sample; red is increased and blue is decreased (relative to the pooled reference). Transcriptome-based subtypes (Verhaak et al., 2013) and the proposed proteomic subtypes\",\n",
            "          \"(Verhaak et al., 2013) and the proposed proteomic subtypes are indicated in color above the heatmap. WGCNA-derived modules are delineated with the row color panel and annotated according to the pathways based on enrichment of KEGG and Reactome ontologies.\",\n",
            "          \"Functional impact of copy number alterations\",\n",
            "          \"The top panel shows the correlation of CNAs to protein abundance (right) or mRNA (left) with significant positive correlations in red and negative correlations in blue (Benjamini-Hochberg adjusted p value <0.01, Spearman\\u2019s correlation coefficient). The\",\n",
            "          \"p value <0.01, Spearman\\u2019s correlation coefficient). The x-axis plots the 29,393 CNAs obtained from TCGA. The y-axis plots 3,202 proteins. Genes are ordered by chromosomal location on both the x- and y-axes. The bottom panel shows the summed number of\",\n",
            "          \"the x- and y-axes. The bottom panel shows the summed number of significantly correlated proteins (or mRNA) for each individual CNA. In blue is shown the total number, in black are those genes significantly correlated as both mRNA and protein. Where blue\",\n",
            "          \"significantly correlated as both mRNA and protein. Where blue and black lines have similar magnitude, e.g., the hotspot on chromosome 20, the CNA associations are shared between protein and mRNA. Where a strong blue line has no mirrored black line, e.g.,\",\n",
            "          \"Where a strong blue line has no mirrored black line, e.g., the protein hotspot on chromosome 2, the associations with CNA are largely unique.\",\n",
            "          \"Kaplan-Meier plot of overall survival stratified by CNA-derived signatures\",\n",
            "          \"A survival plot is shown for a consensus of the four best signatures (see Figure S4A and Table S6) identified from analysis of proteins affected in trans by CNAs. Models were trained using a lasso-based Cox proportional hazards model on the samples from\",\n",
            "          \"lasso-based Cox proportional hazards model on the samples from PNNL, and shown are the survival curves from these models applied to the non-overlapping data from the JHU analysis, with the up (red; above the median) and down (blue, below the median)\",\n",
            "          \"up (red; above the median) and down (blue, below the median) signatures each representing 45 patients. A vote was taken among the four most predictive signatures, and the results of this vote are shown. Probability of death is shown on the y-axis vs.\",\n",
            "          \"vote are shown. Probability of death is shown on the y-axis vs. survival time in years on the x-axis. Shaded ribbons denote 95% confidence intervals.\",\n",
            "          \"Number of CNAs statistically explained by proteins significantly associated with CIN index\",\n",
            "          \"A total of 128 proteins selected for their association with CIN were arranged along the chromosomal locations of their corresponding genes (y-axis). The length of the light grey horizontal lines indicates the number of CNAs significantly correlated with\",\n",
            "          \"indicates the number of CNAs significantly correlated with the protein across all chromosomes except for the protein\\u2019s corresponding gene coding region (x-axis). The top-ranked proteins are annotated and highlighted with dark lines to show the\",\n",
            "          \"are annotated and highlighted with dark lines to show the bootstrap-estimated 95% confidence intervals.\",\n",
            "          \"DDN analysis and lysine-acetylation analysis between HRD and non-HRD patients\",\n",
            "          \"DDN analysis revealed a sub-network of proteins that displayed distinct co-expression patterns between HRD and non-HRD patients, where purple connections indicate protein correlations that exist only in HRD samples and blue connections indicate protein\",\n",
            "          \"exist only in HRD samples and blue connections indicate protein correlations that exist only in non-HRD patients. The links between the nodes are drawn with two different thickness indicative of whether the connections meet the significance threshold of\",\n",
            "          \"of whether the connections meet the significance threshold of 0.05 (light lines) or 0.01 (bold lines). Proteins with blue dotted circles are known to be involved in histone acetylation or deacetylation. Further, identification and quantitation of\",\n",
            "          \"or deacetylation. Further, identification and quantitation of lysine-acetylated peptides in global proteomics data showed that acetylation level at K12 and K16 of histone H4 are significantly different between HRD and non-HRD samples, suggesting a role of\",\n",
            "          \"different between HRD and non-HRD samples, suggesting a role of histone H4 acetylation (together with HDAC1) in modulating the choice of DSB repair mechanisms.\",\n",
            "          \"Pathway analysis associated with patient survival\",\n",
            "          \"A) Pathway components were statistically analyzed using a two-sided t test between samples from patients surviving <3 years (short survival) and patients surviving >5 years (long survival) for differences in CNA, mRNA expression, protein or\",\n",
            "          \"survival) for differences in CNA, mRNA expression, protein or phosphorylation abundance. All significant pathways for mRNA and protein are shown (Benjamini-Hochberg adjusted p value <0.05) and the most significant pathways for phosphoproteomics are\",\n",
            "          \"and the most significant pathways for phosphoproteomics are plotted on the x axis as the \\u2212log of the p value. Results show that phosphorylation provides additional information about the functional state of tumors. B) Simplified PDGFR-beta pathway showing\",\n",
            "          \"state of tumors. B) Simplified PDGFR-beta pathway showing differences between short and long survivor groups for protein abundance (p), transcript abundance (m), and phosphopeptide abundance (circled P).\"\n",
            "        ]\n",
            "      }\n",
            "    ]\n",
            "  },\n",
            "  \"PDC000113\": {\n",
            "    \"study_id\": \"ba4e17a5-57b8-11e8-b07a-00a098d917f8\",\n",
            "    \"pdc_study_id\": \"PDC000113\",\n",
            "    \"study_submitter_id\": \"TCGA OV Proteome S020-2\",\n",
            "    \"program_id\": \"10251935-5540-11e8-b664-00a098d917f8\",\n",
            "    \"project_id\": \"48af5040-5546-11e8-b664-00a098d917f8\",\n",
            "    \"study_name\": \"TCGA Ovarian JHU Proteome\",\n",
            "    \"program_name\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"project_name\": \"CPTAC2 Retrospective\",\n",
            "    \"disease_type\": \"Other;Ovarian Serous Cystadenocarcinoma\",\n",
            "    \"primary_site\": \"Not Reported;Ovary\",\n",
            "    \"analytical_fraction\": \"Proteome\",\n",
            "    \"experiment_type\": \"iTRAQ4\",\n",
            "    \"embargo_date\": null,\n",
            "    \"cases_count\": 124,\n",
            "    \"aliquots_count\": 124,\n",
            "    \"protocol_id\": \"1c06b8e7-3b26-11e9-9a07-0a80fada099c\",\n",
            "    \"protocol_submitter_id\": \"P-S020-2\",\n",
            "    \"program_submitter_id\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"protocol_name\": \"TCGA_Ovarian_JHU_Proteome\",\n",
            "    \"protocol_date\": \"2014-04-01\",\n",
            "    \"document_name\": \"\",\n",
            "    \"quantitation_strategy\": \"Isobaric label quantitation\",\n",
            "    \"label_free_quantitation\": \"\",\n",
            "    \"labeled_quantitation\": \"iTRAQ quantitation\",\n",
            "    \"isobaric_labeling_reagent\": \"iTRAQ\",\n",
            "    \"reporter_ion_ms_level\": \"2\",\n",
            "    \"starting_amount\": \"0.40\",\n",
            "    \"starting_amount_uom\": \"mg\",\n",
            "    \"digestion_reagent\": \"Trypsin\",\n",
            "    \"alkylation_reagent\": \"Iodoacetamide\",\n",
            "    \"enrichment_strategy\": \"Proteome\",\n",
            "    \"enrichment\": \"N/A\",\n",
            "    \"chromatography_dimensions_count\": \"2\",\n",
            "    \"one_d_chromatography_type\": \"bRPLC\",\n",
            "    \"two_d_chromatography_type\": \"RPLC\",\n",
            "    \"fractions_analyzed_count\": \"25\",\n",
            "    \"column_type\": \"C18\",\n",
            "    \"amount_on_column\": \"2\",\n",
            "    \"amount_on_column_uom\": \"\\u00b5g\",\n",
            "    \"column_length\": \"15\",\n",
            "    \"column_length_uom\": \"cm\",\n",
            "    \"column_inner_diameter\": \"75.00\",\n",
            "    \"column_inner_diameter_uom\": \"\\u00b5m\",\n",
            "    \"particle_size\": \"3\",\n",
            "    \"particle_size_uom\": \"\\u00b5m\",\n",
            "    \"particle_type\": \"Jupiter C18\",\n",
            "    \"gradient_length\": \"90\",\n",
            "    \"gradient_length_uom\": \"min\",\n",
            "    \"instrument_make\": \"Thermo Scientific\",\n",
            "    \"instrument_model\": \"Orbitrap Velos\",\n",
            "    \"dissociation_type\": \"HCD\",\n",
            "    \"ms1_resolution\": \"30000\",\n",
            "    \"ms2_resolution\": \"7500\",\n",
            "    \"dda_topn\": \"Top 10\",\n",
            "    \"normalized_collision_energy\": \"35%\",\n",
            "    \"acquistion_type\": \"DDA\",\n",
            "    \"dia_multiplexing\": \"N/A\",\n",
            "    \"dia_ims\": \"N/A\",\n",
            "    \"analytical_technique\": null,\n",
            "    \"chromatography_instrument_make\": null,\n",
            "    \"chromatography_instrument_model\": null,\n",
            "    \"polarity\": null,\n",
            "    \"reconstitution_solvent\": null,\n",
            "    \"reconstitution_volume\": null,\n",
            "    \"reconstitution_volume_uom\": null,\n",
            "    \"internal_standards\": null,\n",
            "    \"extraction_method\": null,\n",
            "    \"ionization_mode\": null,\n",
            "    \"publications\": [\n",
            "      {\n",
            "        \"publication_id\": \"44e2e658-89c7-11e8-bcf1-0a2705229b82\",\n",
            "        \"pubmed_id\": \"27372738\",\n",
            "        \"doi\": \"10.1016/j.cell.2016.05.069\",\n",
            "        \"author\": \"Zhang, H.; Liu, T.; Zhang, Z.; Payne, S. H.; Zhang, B.; McDermott, J. E.; Zhou, J. Y.; Petyuk, V. A.; Chen, L.; Ray, D.; Sun, S.; Yang, F.; Chen, L.; Wang, J.; Shah, P.; Cha, S. W.; Aiyetan, P.; Woo, S.; Tian, Y.; Gritsenko, M. A.; Clauss, T. R.; Choi, C.; Monroe, M. E.; Thomas, S.; Nie, S.; Wu, C.; Moore, R. J.; Yu, K. H.; Tabb, D. L.; Feny\\u00f6, D.; Bafna, V.; Wang, Y.; Rodriguez, H.; Boja, E. S.; Hiltke, T.; Rivers, R. C.; Sokoll, L.; Zhu, H.; Shih, I. M.; Cope, L.; Pandey, A.; Zhang, B.; Snyder, M. P.; Levine, D. A.; Smith, R. D.; Chan, D. W.; Rodland, K. D.; Carr, S. A.; Gillette, M. A.; Klauser, K. R.; Kuhn, E.; Mani, D. R.; Mertins, P.; Ketchum, K. A.; Thangudu, R.; Cai, S.; Oberti, M.; Paulovich, A. G.; Whiteaker, J. R.; Edwards, N. J.; McGarvey, P. B.; Madhavan, S.; Wang, P.; Chan, D. W.; Pandey, A.; Shih, I. M.; Zhang, H.; Zhang, Z.; Zhu, H.; Cope, L.; Whiteley, G. A.; Skates, S. J.; White, F. M.; Levine, D. A.; Boja, E. S.; Kinsinger, C. R.; Hiltke, T.; Mesri, M.; Rivers, R. C.; Rodriguez, H.; Shaw, K. M.; Stein, S. E.; Fenyo, D.; Liu, T.; McDermott, J. E.; Payne, S. H.; Rodland, K. D.; Smith, R. D.; Rudnick, P.; Snyder, M.; Zhao, Y.; Chen, X.; Ransohoff, D. F.; Hoofnagle, A. N.; Liebler, D. C.; Sanders, M. E.; Shi, Z.; Slebos, R. J. C.; Tabb, D. L.; Zhang, B.; Zimmerman, L. J.; Wang, Y.; Davies, S. R.; Ding, L.; Ellis, M. J. C.; Townsend, R. R.\",\n",
            "        \"title\": \"Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer\",\n",
            "        \"journal\": \"Cell\",\n",
            "        \"journal_url\": \"https://www.cell.com/cell/fulltext/S0092-8674(16)30673-0?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0092867416306730%3Fshowall%3Dtrue\",\n",
            "        \"year\": \"2016\",\n",
            "        \"abstract\": \"To provide a detailed analysis of the molecular components and underlying mechanisms associated with ovarian cancer, we performed a comprehensive mass-spectrometry-based proteomic characterization of 174 ovarian tumors previously analyzed by The Cancer Genome Atlas (TCGA), of which 169 were high-grade serous carcinomas (HGSCs). Integrating our proteomic measurements with the genomic data yielded a number of insights into disease, such as how different copy-number alternations influence the proteome, the proteins associated with chromosomal instability, the sets of signaling pathways that diverse genome rearrangements converge on, and the ones most associated with short overall survival. Specific protein acetylations associated with homologous recombination deficiency suggest a potential means for stratifying patients for therapy. In addition to providing a valuable resource, these findings provide a view of how the somatic genome drives the cancer proteome and associations between protein and post-translational modification levels and clinical outcomes in HGSC.\",\n",
            "        \"full_text\": [\n",
            "          \"Integrated proteogenomic characterization of human high grade serous ovarian cancer\\n\\nSUMMARY\",\n",
            "          \"To provide a detailed analysis of the molecular components and underlying mechanisms associated with ovarian cancer, we performed a comprehensive mass spectrometry-based proteomic characterization of 174 ovarian tumors previously analyzed by The Cancer\",\n",
            "          \"of 174 ovarian tumors previously analyzed by The Cancer Genome Atlas (TCGA), of which 169 were high-grade serous carcinomas (HGSC). Integrating our proteomic measurements with the genomic data yielded a number of insights into disease such as how\",\n",
            "          \"data yielded a number of insights into disease such as how different copy number alternations influence the proteome, the proteins associated with chromosomal instability, the sets of signaling pathways that diverse genome rearrangements converge on, as\",\n",
            "          \"pathways that diverse genome rearrangements converge on, as well as the ones most associated with short overall survival. Specific protein acetylations associated with homologous recombination deficiency suggest a potential means for stratifying patients\",\n",
            "          \"deficiency suggest a potential means for stratifying patients for therapy. In addition to providing a valuable resource, these findings provide a view of how the somatic genome drives the cancer proteome and associations between protein and\",\n",
            "          \"drives the cancer proteome and associations between protein and post-translational modification levels and clinical outcomes in HGSC.\",\n",
            "          \"INTRODUCTION\",\n",
            "          \"A comprehensive molecular view of cancer is necessary for understanding the underlying mechanisms of disease, improving prognosis, and ultimately guiding treatment. The Cancer Genome Atlas (TCGA) conducted an extensive genomic and transcriptomic\",\n",
            "          \"Atlas (TCGA) conducted an extensive genomic and transcriptomic characterization of ovarian high-grade serous carcinoma (HGSC) aimed at defining the genomic landscape and aiding the development of targeted therapies for this highly lethal malignancy. Key\",\n",
            "          \"of targeted therapies for this highly lethal malignancy. Key findings from TCGA were: 1) the prevalent role of TP53 mutations, 2) extensive DNA copy alterations, 3) preliminary transcriptional signatures associated with survival, 4) varied mechanisms of\",\n",
            "          \"signatures associated with survival, 4) varied mechanisms of BRCA1/2 inactivation, and lastly, 5) CCNE1 aberrations. Subsequent analysis of genomic data from the TCGA consortium led to the refinement of the transcript-defined signatures, improving the\",\n",
            "          \"refinement of the transcript-defined signatures, improving the statistical association with patient outcome, and integrating microRNA and mRNA expression profiles associated with HGSC to identify candidate microRNA targets.\",\n",
            "          \"While the insights from genomic analyses are substantial, functions encoded in the genome are generally executed at the protein level, and are often further modulated by post-translational modifications (PTMs). For example, TCGA used reverse phase protein\",\n",
            "          \"(PTMs). For example, TCGA used reverse phase protein array (RPPA) analysis of 172 proteins (including 31 phosphoproteins with phospho-specific antibodies) to generate a signature associated with the risk of tumor recurrence. To obtain a more comprehensive\",\n",
            "          \"the risk of tumor recurrence. To obtain a more comprehensive assessment of the complex ovarian HGSC phenotype, the Clinical Proteomic Tumor Analysis Consortium (CPTAC) conducted an extensive mass spectrometry (MS)-based proteomic and phosphoproteomic\",\n",
            "          \"mass spectrometry (MS)-based proteomic and phosphoproteomic characterization of HGSC tumors characterized by TCGA, providing quantitative measurements for a combined total of 9,600 proteins from 174 tumors (an average of 7,952 proteins per tumor), and a\",\n",
            "          \"from 174 tumors (an average of 7,952 proteins per tumor), and a total of 24,429 phosphosites from 6,769 phosphoproteins in a subset of 69 tumors (an average of 7,677 phosphosites per tumor). Our results provide insights on HGSC biology and correlate\",\n",
            "          \"Our results provide insights on HGSC biology and correlate differences in protein and PTM levels with clinical outcome complementary to TCGA genomic analyses.\",\n",
            "          \"RESULTS\\n\\nProteomic analysis of TCGA HGSC samples\",\n",
            "          \"HGSC biospecimens and clinical data from 174 patients collected by TCGA were analyzed at two independent CPTAC Centers, Johns Hopkins University (JHU) and Pacific Northwest National Laboratory (PNNL); thirty-two samples were analyzed at both JHU and PNNL.\",\n",
            "          \"(PNNL); thirty-two samples were analyzed at both JHU and PNNL. Tumors were selected by examining the associated TCGA metadata to select tumors either 1) on the basis of putative homologous recombination deficiency (HRD), defined by the presence of\",\n",
            "          \"recombination deficiency (HRD), defined by the presence of germline or somatic BRCA1 or BRCA2 mutations, BRCA1 promoter methylation, or homozygous deletion of PTEN (122 samples; 67 classified as HRD, 55 as non-HRD by the above criteria; JHU), or 2) to\",\n",
            "          \"as HRD, 55 as non-HRD by the above criteria; JHU), or 2) to maximize differences in overall survival (84 samples; PNNL). For selection purposes, short survival was defined as overall survival of less than 3 years, and long survival as greater than 5\",\n",
            "          \"of less than 3 years, and long survival as greater than 5 years. All but five tumors had somatic TP53 mutations, a characteristic feature of HGSC; these five tumors were subsequently reclassified as other than HGSC, and removed from protein functional\",\n",
            "          \"as other than HGSC, and removed from protein functional analyses (e.g., subtyping and survival analyses). The tumor selection criteria and the associated metadata are provided in the Supplemental Information Table S1, available online.\",\n",
            "          \"Proteomics measurements used isobaric tags for relative and absolute quantitation (iTRAQ) in conjunction with offline liquid chromatography fractionation via high-pH reversed phase liquid chromatography (RPLC) and online RPLC with high-resolution tandem\",\n",
            "          \"(RPLC) and online RPLC with high-resolution tandem MS to provide broad coverage for peptide and protein identification and quantification (Supplemental Information); this also alleviated quantitative interference potentially associated with the use of\",\n",
            "          \"interference potentially associated with the use of isobaric tags. We used the relative abundance measurements for each protein in the 32 patient samples analyzed at both JHU and PNNL to normalize across the two analysis sites, and then used clustering,\",\n",
            "          \"across the two analysis sites, and then used clustering, principle component analysis (PCA), and statistical tests to identify any significant batch effects associated with the site of analysis (a detailed comparison of within site, between site, between\",\n",
            "          \"(a detailed comparison of within site, between site, between sample measurement variability and the process used to merge the JHU and PNNL data is given in Supplemental Information, Figure S1). As shown in Figure S1C, the median coefficient of variation\",\n",
            "          \"As shown in Figure S1C, the median coefficient of variation (CV) between measurements at the two sites was 16%.\",\n",
            "          \"A total of 9,600 proteins were identified with high confidence in all tumors, and the relative abundances in each tumor are given in Table S2. Functional analyses and proteome-transcriptome associations were restricted to 3,586 proteins observed and\",\n",
            "          \"associations were restricted to 3,586 proteins observed and quantified in all 169 HGSC samples used for protein functional analyses and where sample variability (signal) exceeded technical variability (noise) in the merged data (Table S2), calculated as\",\n",
            "          \"(noise) in the merged data (Table S2), calculated as described in Supplemental Information. On average we identify peptides covering 29% of the amino acids in any of these 3,586 proteins in a given sample, with a range from 10% to 47%. In addition to\",\n",
            "          \"in a given sample, with a range from 10% to 47%. In addition to protein abundance levels, phosphoproteomics data were acquired for 69 tumors with sufficient sample (Table S2). As with proteins, phosphopeptide abundances were calculated relative to the\",\n",
            "          \"phosphopeptide abundances were calculated relative to the pooled reference sample. Because isobaric labeling was performed prior to splitting the samples for proteome and phosphoproteome analyses, phosphopeptide abundance could be corrected for changes in\",\n",
            "          \"phosphopeptide abundance could be corrected for changes in parent protein abundance to identify differences in the relative extent of phosphorylation at specific sites for each protein (Table S2). Overall, we achieved a protein dynamic range encompassing\",\n",
            "          \"S2). Overall, we achieved a protein dynamic range encompassing >4 orders of magnitude, ranging from low-level transcription factors to abundant structural proteins i.e., actin and tubulin.\",\n",
            "          \"Proteogenomic landscape of HGSC\",\n",
            "          \"The degree to which alterations observed at the genome and transcriptome levels are manifested at the protein level is variable, both qualitatively and quantitatively, and partially driven by multiple levels of post-transcriptional regulation. To identify\",\n",
            "          \"multiple levels of post-transcriptional regulation. To identify peptides encoded by single amino acid variants (SAAVs), splice variants, and novel exons documented by TCGA, mass spectra were searched against a custom graph database containing all peptide\",\n",
            "          \"searched against a custom graph database containing all peptide variations projected from the TCGA genomic analyses of the cohort using a multi-stage analysis pipeline (Supplemental Information). The most frequently observed variant peptides represented\",\n",
            "          \"The most frequently observed variant peptides represented SAAVs and gene-level events. The evidence supporting each novel event is detailed in Table S2, including event type, genomic location, abundance information, and estimated false discovery rate\",\n",
            "          \"abundance information, and estimated false discovery rate (FDR). The validity of these variant peptides was further evaluated by the synthesis of twenty examples selected at random across the entire range of spectrum match scores. We obtained tandem mass\",\n",
            "          \"entire range of spectrum match scores. We obtained tandem mass spectra for all twenty synthetic peptides that matched spectra from our analyses, strongly supporting our observation of these variant peptides; three representative examples are given in\",\n",
            "          \"variant peptides; three representative examples are given in Figure S1E. These results demonstrate the ability to confidently detect, identify and validate genome-level predictions at the protein level. More novel peptides would likely be observed if\",\n",
            "          \"protein level. More novel peptides would likely be observed if there was sufficient sample for more extensive fractionation and/or replicate analyses; these limitations preclude any conclusions about the biological significance of unobserved events. For\",\n",
            "          \"about the biological significance of unobserved events. For example, only 2 mutant p53 peptides were identified despite the presence of p53 mutations in all tumors examined. Such low coverage can occur for reasons that include: excessively large or small\",\n",
            "          \"can occur for reasons that include: excessively large or small tryptic fragments, inability to distinguish some amino acids, some possible biases against highly hydrophobic and hydrophilic peptides, or low abundance peptides co-eluting with very high\",\n",
            "          \"peptides, or low abundance peptides co-eluting with very high abundance peptides.\",\n",
            "          \"To assess the potential for post-transcriptional regulation (e.g., translational efficiency or protein degradation), we compared each protein to its corresponding transcript across all tumors, and correlation was assessed for those pairs with reliable\",\n",
            "          \"and correlation was assessed for those pairs with reliable measurements for both mRNA and protein, i.e., proteins with a corresponding mRNA measurement observed in all 169 HGSC tumors where sample variability exceeded technical variability (3,196 pairs).\",\n",
            "          \"variability exceeded technical variability (3,196 pairs). We excluded 391 proteins observed without a corresponding mRNA (e.g., HBA1) or discordant gene symbol annotation in the protein database (e.g., THOC4). Overall, 79.4% of the mRNA:protein pairs\",\n",
            "          \"(e.g., THOC4). Overall, 79.4% of the mRNA:protein pairs showed statistically significant positive Spearman correlations (Benjamini-Hochberg adjusted p value <0.01; Figure 1) when changes in mRNA abundance were compared to changes in relative protein\",\n",
            "          \"in mRNA abundance were compared to changes in relative protein abundance. The observed median r value of 0.45 for each mRNA-protein pair across all 169 tumors is similar to observations in colorectal cancer, breast cancer and mouse tissues, although less\",\n",
            "          \"cancer, breast cancer and mouse tissues, although less than found for cell lines. In comparison, the median correlation of paired protein measurements from the same sample, but measured at two sites, was substantially higher, at 0.69 (Figure S1).\",\n",
            "          \"A wide range of mRNA:protein correlations was observed. In general, weaker correlations were observed for highly stable and abundant proteins associated with housekeeping or non-intrinsic functions (e.g., ribosomes, mRNA splicing, oxidative\",\n",
            "          \"functions (e.g., ribosomes, mRNA splicing, oxidative phosphorylation, complement cascade), while more dynamic proteins known to be transcriptionally regulated in response to nutrient demand or other perturbations (e.g., nucleotide metabolism, amino acid\",\n",
            "          \"or other perturbations (e.g., nucleotide metabolism, amino acid metabolism, acute inflammatory responses) were more highly correlated (Figure 1 and Table S3). This result is consistent with a previous colorectal cancer study and supports the hypothesis\",\n",
            "          \"a previous colorectal cancer study and supports the hypothesis that while many biological functions are primarily regulated by mRNA abundance, post-transcriptional mechanisms likely have an important role in regulating certain house-keeping functions.\",\n",
            "          \"important role in regulating certain house-keeping functions. Additionally, we found that 23 mRNAs lacking polyA tails displayed lower correlation (mean \\u22120.15) with their cognate proteins than did polyadenylated mRNAs, consistent with the decreased\",\n",
            "          \"than did polyadenylated mRNAs, consistent with the decreased stability of mRNAs lacking polyA tails.\",\n",
            "          \"Clustering of tumors based on protein abundance\",\n",
            "          \"HGSC is the most common of the four major histological subtypes of epithelial ovarian cancer, and is characterized by distinct morphological features. Recent studies using mRNA abundance data have suggested four transcriptomic HGSC subtypes designated as\",\n",
            "          \"have suggested four transcriptomic HGSC subtypes designated as differentiated, immunoreactive, mesenchymal, and proliferative. To build an unbiased molecular taxonomy of ovarian HGSC, we used protein abundance data from the 169 tumors to identify subtypes\",\n",
            "          \"protein abundance data from the 169 tumors to identify subtypes that might show biological differences that could be exploited in future studies. Figure 2 shows the resulting clustering analysis of individual tumors (vertical columns) by protein abundance\",\n",
            "          \"of individual tumors (vertical columns) by protein abundance (horizontal rows). The cluster assignment for each sample is provided in Table S4, and a consensus value matrix for the subtype comparisons is shown in Figure S2A. The results of a Weighted Gene\",\n",
            "          \"is shown in Figure S2A. The results of a Weighted Gene Co-expression Network Analysis (WGCNA) of protein functional enrichment by subgroups are provided for the protein clusters, in Figure S2B and Table S4. The enrichment of KEGG and Reactome ontologies\",\n",
            "          \"and Table S4. The enrichment of KEGG and Reactome ontologies in the WGCNA-derived modules are shown in Figures S2C and S2D, respectively; membership of enriched pathways is provided in Table S3.\",\n",
            "          \"Four of the proteomic clusters showed a clear correspondence to the mesenchymal, proliferative, immunoreactive, and differentiated subtypes defined by the TCGA transcriptome analysis (Figure 2 and Figure S2E). A relatively small fifth cluster of tumors\",\n",
            "          \"2 and Figure S2E). A relatively small fifth cluster of tumors significantly enriched in proteins associated with extracellular matrix interactions, erythrocyte and platelet functions, and the complement cascade was also observed using multiple approaches,\",\n",
            "          \"complement cascade was also observed using multiple approaches, including model based clustering with Bayesian information criteria, consensus clustering, and VISDA-based sub-phenotype clustering. This new group could not be attributed to tissue source\",\n",
            "          \"This new group could not be attributed to tissue source site sampling bias or any other meta-data category, but may be related to tumor characteristics, including vascularization and tumor content, as the Tumor Purity score for this subtype (and the\",\n",
            "          \"content, as the Tumor Purity score for this subtype (and the Mesenchymal subtype) was significantly lower than that of the other three subtypes (Figure S2F). The clinical relevance of these protein-based clusters will require validation in independent\",\n",
            "          \"protein-based clusters will require validation in independent HGSC sample sets, particularly as no significant difference in survival was observed (Figure S2G), similar to mRNA-based clustering analysis.\",\n",
            "          \"Proteomic analysis of CNA effects\",\n",
            "          \"Chromosomal instability, marked by extensive copy number alterations (CNAs) in each tumor, is a hallmark of HGSC, and a likely source of driver alterations in this disease. CNAs can affect the abundance of proteins at the same locus (cis effects), and may\",\n",
            "          \"abundance of proteins at the same locus (cis effects), and may also act in trans, either directly or indirectly.\",\n",
            "          \"Hypothesizing that CNAs with strong trans effects are more likely to elicit a molecular phenotype and confer selective advantages, we sought to identify those CNA regions that have the broadest effect on protein expression. In all, 29,393 CNA loci (Table\",\n",
            "          \"effect on protein expression. In all, 29,393 CNA loci (Table S5) were compared to our global proteomics data with 950,209 CNA/protein pairs (0.72% of the total) exhibiting significant association (Benjamini-Hochberg adjusted p value <0.01). We provide a\",\n",
            "          \"(Benjamini-Hochberg adjusted p value <0.01). We provide a complete list of the significantly associated proteins for each CNA in Table S5. The diagonal line evident in Figure 3 corresponds to cis effects, and vertical stripes correspond to trans effects\",\n",
            "          \"cis effects, and vertical stripes correspond to trans effects where changes in copy number affect expression of numerous proteins across the genome. We performed the same analysis for mRNA to identify sites where associations are transcriptionally\",\n",
            "          \"mRNA to identify sites where associations are transcriptionally mediated. A similar number of CNA/mRNA pairs were found to be significantly associated (1,113,164 at Benjamini-Hochberg adjusted p value <0.01; Figure 3). This analysis revealed regions on\",\n",
            "          \"p value <0.01; Figure 3). This analysis revealed regions on chromosomes 2, 7, 20, and 22 correlated in trans with more than 200 proteins. In contrast to colorectal cancer where most of the trans-regulation of protein by CNA was accompanied by similar\",\n",
            "          \"trans-regulation of protein by CNA was accompanied by similar changes in mRNA, we observed several loci associated with differences in protein abundance without a corresponding change in mRNA. For example, large regions on chromosome 2 have relatively\",\n",
            "          \"For example, large regions on chromosome 2 have relatively little trans effect on mRNA levels, but are associated with more than 200 proteins in trans. Dissecting the mechanisms by which a specific CNA can alter protein levels in trans, without affecting\",\n",
            "          \"CNA can alter protein levels in trans, without affecting the corresponding mRNA is difficult, given the extent of the amplified or deleted regions and the numerous genes affected at a given locus. Possible mechanisms include cis regulation of proteins\",\n",
            "          \"locus. Possible mechanisms include cis regulation of proteins associated with mRNA stability and translational efficiency, such as microRNAs and RNA binding proteins.\",\n",
            "          \"Given the complex pattern of CNAs observed in HGSC, it has been difficult to identify a limited number of high impact genomic alterations that could function as drivers of the disease. We interrogated the trans-affected proteins associated with each\",\n",
            "          \"interrogated the trans-affected proteins associated with each putative CNA for common functions using pathways defined by the KEGG, NCI pathway interaction database (PID) and Reactome databases (Supplemental Information). Proteins associated with cell\",\n",
            "          \"(Supplemental Information). Proteins associated with cell invasion and migration, and proteins related to immune function appeared to be enriched in association with multiple CNAs (Figure S3; Table S3). These observations suggest a convergence of multiple\",\n",
            "          \"Table S3). These observations suggest a convergence of multiple CNA targets on a common set of biological functions, namely motility/invasion and immune regulation, functions that are among the hallmarks of cancer.\",\n",
            "          \"Association of CNA trans-affected proteins with overall survival\",\n",
            "          \"Availability of the TCGA survival data allowed us to use trans-affected protein data from the most influential CNAs (e.g., the four altered regions described on chromosomes 2, 7, 20, and 22; Figure 3) to build a model of overall survival. Because each CNA\",\n",
            "          \"3) to build a model of overall survival. Because each CNA affects many proteins, we used a regression approach that identifies parsimonious Cox proportional hazards models with maximal predictive ability from the list of significantly correlated proteins\",\n",
            "          \"ability from the list of significantly correlated proteins for that CNA. We trained models on the proteomics data from PNNL (82 tumors) and tested the ability of the model to predict survival times in the data from JHU (87 tumors, not including the 32\",\n",
            "          \"times in the data from JHU (87 tumors, not including the 32 overlapping). Each of the four most influential CNA regions produced models that were strongly associated with patient survival (p value <0.01, FDR <0.5% based on randomly selected proteins). A\",\n",
            "          \"value <0.01, FDR <0.5% based on randomly selected proteins). A Kaplan-Meier plot illustrating the predictive value of each of the locus-specific models after validation is shown in Figure S4A; for the Kaplan-Meier plots, \\u2018high\\u2019 and \\u2018low\\u2019 expression were\",\n",
            "          \"for the Kaplan-Meier plots, \\u2018high\\u2019 and \\u2018low\\u2019 expression were defined relative to the median for that signature across all samples, splitting patients into two equal groups of 45 (Table S6). We examined the overlap in predictions for patients for the four\",\n",
            "          \"examined the overlap in predictions for patients for the four signatures and found that the predictions were unanimous (either high or low signature) for 62% of patients and 16% were evenly split, with the remaining having one dissenting signature. This\",\n",
            "          \"split, with the remaining having one dissenting signature. This suggested the utility of combining these signatures, using a voting method where the number of high or low calls was counted for each patient; this substantially improved prediction of\",\n",
            "          \"for each patient; this substantially improved prediction of survival time (p value 1.9e-6; Figure 4). We also examined the effects of tumor stage, tumor grade, patient age, surgical outcome, and platinum status, and found that our proteomic signatures\",\n",
            "          \"and platinum status, and found that our proteomic signatures were not improved by inclusion of these variables.\",\n",
            "          \"For each of these locus-specific models we analyzed the enrichment of genes and their regulatory sequences associated with outcome, and found all four models had genes significantly enriched (Benjamini-Hochberg adjusted p value <0.05) in binding sites for\",\n",
            "          \"adjusted p value <0.05) in binding sites for the proliferation-associated serum response factor (SRF), suggesting that SRF activity may be important in ovarian cancer outcome. Additionally, there was significant enrichment of proteins involved in the\",\n",
            "          \"there was significant enrichment of proteins involved in the regulation of actin cytoskeleton, apoptosis, and adherens junction (Benjamini-Hochberg adjusted p value <0.05). We examined the protein abundance and phosphorylation status of SRF between short\",\n",
            "          \"abundance and phosphorylation status of SRF between short and long surviving groups and observed that they were higher in short surviving patients, but only slightly. Thus, although SRF alone was not predictive, the trends in SRF abundance and\",\n",
            "          \"SRF alone was not predictive, the trends in SRF abundance and phosphorylation are consistent with the observation of enrichment for proteins potentially regulated by SRF binding. Details of the larger gene set analysis are provided in Supplemental\",\n",
            "          \"of the larger gene set analysis are provided in Supplemental Information.\",\n",
            "          \"Several proteins shared across all the signatures are known to be involved in cancer processes. Catenin B2 (CTNNA2) is a cell-cell adhesion protein and tumor suppressor. The Rho GDP dissociation inhibitor (GDI) beta (ARHGDIB) is involved in invasion and\",\n",
            "          \"inhibitor (GDI) beta (ARHGDIB) is involved in invasion and migration in many cancers, and overexpression correlates with progression in pancreatic carcinoma. Protein kinase C and casein kinase substrate in neurons protein 2 (PACSIN2) is a repressor of\",\n",
            "          \"substrate in neurons protein 2 (PACSIN2) is a repressor of cellular migration. Finally, the GTP-binding nuclear protein Ran, is a prognostic marker associated with increased survival in epithelial ovarian cancer. The association of these previously\",\n",
            "          \"epithelial ovarian cancer. The association of these previously described survival-related proteins with genes affected in trans by CNAs suggests a potential mechanism for the parallel activation of multiple pathways associated with poor prognosis in HGSC.\",\n",
            "          \"As a comparison we applied the previously described Provar signature, comprising five protein and four phosphoproteins that showed good survival prediction in the TCGA ovarian cancer dataset. We observed all proteins in the Provar signature in our\",\n",
            "          \"We observed all proteins in the Provar signature in our proteomic data, but only one of the phosphosites (EGFR Y1173). Thus, we used the RPPA data from the original signature (Figure S4B). We found that the Provar signature was prognostic of survival\",\n",
            "          \"We found that the Provar signature was prognostic of survival (Benjamini-Hochberg adjusted p value = 0.11) in the 67 patients examined here (those with phosphoprotein data and had somatic TP53 mutations), but the statistical power of Provar is not as high\",\n",
            "          \"mutations), but the statistical power of Provar is not as high as the statistical power of signatures derived from the CNA trans-affected proteins in this dataset. In addition, integrating the Provar signature with the CNA signatures did not improve\",\n",
            "          \"the Provar signature with the CNA signatures did not improve survival prediction (Figure S4C).\",\n",
            "          \"Identification of proteins associated with chromosomal structural abnormality\",\n",
            "          \"The degree of chromosomal instability exhibited by a tumor can be represented by a calculated chromosome instability (CIN) index, as described previously for lung cancer. Identification of proteins associated with CIN may provide information on the\",\n",
            "          \"of proteins associated with CIN may provide information on the processes contributing to chromosomal instability, while the analysis of trans-affected proteins described above is more closely related to the downstream consequences of specific CNAs. Using\",\n",
            "          \"related to the downstream consequences of specific CNAs. Using a bootstrapping method described in Supplemental Information, we identified a proteomic signature including 128 proteins (Table S7) showing significant correlation with CIN index\",\n",
            "          \"(Table S7) showing significant correlation with CIN index (Benjamini-Hochberg adjusted p value <1e-4, Spearman correlation; Figure S5A). Functional annotation of these proteins (Figure S5B) showed that proteins up-regulated in tumors with high CIN index\",\n",
            "          \"showed that proteins up-regulated in tumors with high CIN index were preferentially involved in chromatin organization (p value 6.90e-5), whereas proteins up-regulated in tumors with low CIN index were more often associated with cell death (p value\",\n",
            "          \"CIN index were more often associated with cell death (p value 2.13e-5). Correlation analysis of protein abundances and CIN indicated that a small number of proteins could account for the majority of the CIN (Figure 5); two of the most strongly associated\",\n",
            "          \"of the CIN (Figure 5); two of the most strongly associated proteins, CHD4 and CHD5, are known to be involved in chromatin organization.\",\n",
            "          \"Identification of proteins associated with HRD status\",\n",
            "          \"Since HRD is associated with susceptibility to PARP inhibitors and improved survival, we sought to elucidate systemic changes associated with HRD and identify biomarkers that might be used to stratify patients for treatment. We defined tumors with HRD as\",\n",
            "          \"stratify patients for treatment. We defined tumors with HRD as having either germline or somatic mutations in BRCA1 or BRCA2, or BRCA1 promoter methylation, or homozygous deletion of PTEN, with an overall survival >1.5 years, while non-HRD patients were\",\n",
            "          \"an overall survival >1.5 years, while non-HRD patients were defined as lacking these genomic aberrations and with a follow-up or time-to-death <2.5 years; additional selection criteria include available residual tissue volume and a tumor tissue\",\n",
            "          \"include available residual tissue volume and a tumor tissue contamination score estimated using copy number alterations. Applying differential dependency network (DDN) analysis on a set of 171 BRCA1- or BRCA2-related proteins curated from the literature\",\n",
            "          \"BRCA1- or BRCA2-related proteins curated from the literature and from the cBio portal, we identified a sub-network of 30 proteins that displayed co-expression patterns differentiating HRD from non-HRD patients (Table S7; Figure 6). Several of the proteins\",\n",
            "          \"non-HRD patients (Table S7; Figure 6). Several of the proteins in these modules are known to be involved in histone acetylation or deacetylation, e.g., HDAC1, RBBP4, RBBP7, and EP300, and HUS1.\",\n",
            "          \"Although statistical association cannot distinguish between drivers and consequences of HRD status, the observed enrichment of proteins associated with histone acetylation motivated us to use an effective database search strategy (Supplemental\",\n",
            "          \"us to use an effective database search strategy (Supplemental Information) to identify and quantify acetylated peptides from the global proteomic data. Comparative analysis of 399 acetylated peptides identified 15 acetylated peptides with significant\",\n",
            "          \"peptides identified 15 acetylated peptides with significant differences between the HRD and non-HRD samples (Table S7). Among these, dual acetylation at K12 and K16 of histone H4 showed a significant difference between HRD and non-HRD samples (Figure 6);\",\n",
            "          \"difference between HRD and non-HRD samples (Figure 6); differential acetylation of K12 and K16 was verified using synthetic peptides and targeted analysis using Sequential Windowed data independent Acquisition of the Total High-resolution Mass Spectra\",\n",
            "          \"Acquisition of the Total High-resolution Mass Spectra (SWATH-MS). In cell line data, acetylation of H4 has previously been reported to be involved in the choice of DNA double strand break (DSB) repair pathways (homologous recombination or non-homologous\",\n",
            "          \"repair pathways (homologous recombination or non-homologous end joining. This relationship is regulated partially by HDAC1, a protein also identified in the DDN analysis. We observed a significant enrichment of HDAC1 and its co-regulated proteins in\",\n",
            "          \"enrichment of HDAC1 and its co-regulated proteins in tumors with HRD and low H4 acetylation relative to non-HRD tumors with high H4 acetylation (DDN analysis: permutation tests, p value <0.05; differentially acetylated peptides: t-tests, p value <0.05\",\n",
            "          \"differentially acetylated peptides: t-tests, p value <0.05 with an estimated FDR <0.5% by bootstrap/permutation tests). The combined observations of increased HDAC1 and associated proteins at the pathway level, together with decreased acetylation of H4 in\",\n",
            "          \"the pathway level, together with decreased acetylation of H4 in HRD patients at the PTM level, provide insight regarding the potential role of HDAC1 in modulating the choice of DSB repair pathways.\",\n",
            "          \"Phosphoproteomic analysis of pathways associated with survival\",\n",
            "          \"An initial set of 24,464 different phosphopeptides (21,298 unique phosphorylation sites) contained within 4,420 proteins having data on net differences in phosphorylation (Table S2) was obtained. Since ischemia of the TCGA tumor samples may alter\",\n",
            "          \"obtained. Since ischemia of the TCGA tumor samples may alter phosphopeptide abundance, we removed any phosphorylation sites previously shown to be altered by ischemia. We also removed three tumor samples having substantially higher than average missing\",\n",
            "          \"tumor samples having substantially higher than average missing values and two tumor samples lacking somatic TP53 mutations (Supplemental Information). This yielded a final set of 7,675 different phosphopeptides (6,802 unique phosphorylation sites) from\",\n",
            "          \"phosphopeptides (6,802 unique phosphorylation sites) from 2,324 proteins (Table S2) that were mapped to pathways via the NCI PID. The average net phosphorylation of all phosphopeptides mapping to a given pathway (i.e., the statistical average across all\",\n",
            "          \"to a given pathway (i.e., the statistical average across all phosphoproteins known to be in the pathway, rather than the statistically significant individual proteins) was used as a measure of pathway activity and compared between short survivors\",\n",
            "          \"of pathway activity and compared between short survivors (deceased surviving <3 years; N=17) and long survivors (patients surviving >5 years; N=19) using both proteomic and phosphoproteomic data.\",\n",
            "          \"An issue in ranking the pathways is the distinction between phosphopeptides that are unique to a single pathway, such as the receptor peptides themselves, vs. peptides that map to proteins shared between pathways. In order to address this issue, we\",\n",
            "          \"shared between pathways. In order to address this issue, we performed two analyses: one of all proteins associated with a specific pathway in the NCID database, and a second analysis in which proteins common to multiple pathways were excluded. Figure 7A\",\n",
            "          \"proteins common to multiple pathways were excluded. Figure 7A shows a ranking of activated pathways, as inferred from the analysis of pathway specific components (i.e., differences in mRNA, protein and phosphoprotein levels) in short vs. long survivors.\",\n",
            "          \"protein and phosphoprotein levels) in short vs. long survivors. Fifteen pathways showed increased phosphorylation at Benjamini-Hochberg adjusted p values <0.01, in contrast with transcriptional data for the same samples which yielded only one\",\n",
            "          \"data for the same samples which yielded only one statistically significantly increased pathway, androgen receptor signaling, which was also increased at the phosphoprotein level (Figure 7A and Supplemental Information). All pathways indicated in Figure 7A\",\n",
            "          \"Supplemental Information). All pathways indicated in Figure 7A were increased in association with short survival relative to long survival, though some individual components within a pathway showed opposite changes (e.g. JNK1 transcript and protein\",\n",
            "          \"showed opposite changes (e.g. JNK1 transcript and protein levels, see Figure 7B). The RhoA-regulatory, PDGFRB, and integrin-like kinase pathways emerged as the most activated in this analysis (Figure 7A). Figure 7B shows the common and unique elements of\",\n",
            "          \"(Figure 7A). Figure 7B shows the common and unique elements of the PDGFRB pathway, and the trends in mRNA, protein, and phosphoprotein levels in the comparison of short vs. long survivors, illustrating the benefit of combined protein and phosphoprotein\",\n",
            "          \"illustrating the benefit of combined protein and phosphoprotein measurements, as well as the substantial contribution of protein phosphorylation data to the overall analysis.\",\n",
            "          \"We compared our results to the 31 phosphoproteins represented on the RPPA arrays used in the original TCGA ovarian cancer analysis; RPPA identified differential phosphorylation of ERK1, RAF1, and STAT3 in the same patient samples analyzed by MS. For short\",\n",
            "          \"and STAT3 in the same patient samples analyzed by MS. For short surviving vs. long surviving patients, ERK1 showed statistically significantly increased phosphorylation by RPPA, on the same ERK1 phosphopeptides as found using MS, while AKT1, RAF1 and\",\n",
            "          \"ERK1 phosphopeptides as found using MS, while AKT1, RAF1 and STAT3 showed increased phosphorylation by both RPPA and MS-based analyses, but based upon distinct phosphopeptides (Table S2). Good overlap was observed between RPPA and MS results for\",\n",
            "          \"S2). Good overlap was observed between RPPA and MS results for phosphoproteins on the RPPA array, but we did not observe three of the specific phosphosites from the Provar survival signature in our MS data, suggesting that proteomics is more sensitive to\",\n",
            "          \"in our MS data, suggesting that proteomics is more sensitive to overall pathway activity than specific regulatory events in the pathway itself. However, many of the phosphopeptides identified by MS phosphoproteomics as components of the PDGFR-beta pathway\",\n",
            "          \"by MS phosphoproteomics as components of the PDGFR-beta pathway were not represented in the RPPA, illustrating the ability to identify phosphorylation events not readily accessible with currently available antibodies (Figure 7B).\",\n",
            "          \"DISCUSSION\",\n",
            "          \"The addition of proteomic information, including post-translational modifications, to the rich genomic and transcriptomic data available from the ovarian HGSC samples analyzed by TCGA has provided additional insights into the biology of HGSC, including\",\n",
            "          \"additional insights into the biology of HGSC, including identification of changes at the level of pathway activation. Integration of genomic, proteomic, and phosphoproteomic measurements identified differentially regulated pathways and functional modules\",\n",
            "          \"differentially regulated pathways and functional modules that displayed significant associations with patient outcomes, including survival and HRD status. Cox proportional hazard analysis of proteins associated with CNAs identified overall survival\",\n",
            "          \"of proteins associated with CNAs identified overall survival signatures enriched for targets of the proliferation-associated transcription factor SRF, and integrated proteomic, phosphoproteomic and transcriptomic data identified pathways that\",\n",
            "          \"and transcriptomic data identified pathways that differentiated patients on the basis of survival, including the PDGFR-beta signaling pathway associated with angiogenesis, the RhoA regulatory and integrin-linked kinase pathways associated with cell\",\n",
            "          \"and integrin-linked kinase pathways associated with cell mobility and invasion, and pathways associated with chemokine signaling and adaptive immunity. Proteins associated with cell invasion and motility emerged from both the integration of CNAs with\",\n",
            "          \"and motility emerged from both the integration of CNAs with protein abundance data and the phosphoproteomic investigation of up-regulated pathways in short versus long survivors. Despite the complexity of genomic alterations that characterize HGSC, these\",\n",
            "          \"complexity of genomic alterations that characterize HGSC, these analyses suggest a functional convergence on a subset of key pathways. The association of increased invasiveness and motility with short overall survival in this study may help to explain\",\n",
            "          \"with short overall survival in this study may help to explain more aggressive mechanisms of dissemination in ovarian cancer, including recently reported hematogenous metastasis, in addition to previously described lymphatic spread and known peritoneal\",\n",
            "          \"to previously described lymphatic spread and known peritoneal spread by direct extension.\",\n",
            "          \"Focusing on functional modules has also revealed potential drivers of HGSC and more robust signatures for potentially stratifying ovarian HGSC patients into distinct cancer phenotypes to inform therapeutic management. For example, the identification of\",\n",
            "          \"therapeutic management. For example, the identification of specific acetylation events associated with HRD, e.g., the simultaneous acetylation of K12 and K16 on histone H4, may provide an alternative biomarker of HRD and a rationale for the selection of\",\n",
            "          \"biomarker of HRD and a rationale for the selection of patients in future clinical trials of HDAC inhibitors alone or in combination with PARP inhibition. This may help to resolve the current discrepancy between the initial observation of limited single\",\n",
            "          \"discrepancy between the initial observation of limited single agent activity of HDAC inhibitors in ovarian cancer, and more recent findings of a >40% response rate when used in combination with cytotoxic chemotherapy in platinum-resistant patients.\",\n",
            "          \"with cytotoxic chemotherapy in platinum-resistant patients. Similarly, comprehensive interrogation of protein phosphorylation identified multiple pathways with significantly increased phosphorylation in patients with poor clinical outcomes. Specifically,\",\n",
            "          \"in patients with poor clinical outcomes. Specifically, the observed activation of PDGFR pathways in a subset of patients with short overall survival could potentially stratify patients for selective enrollment in trials of anti-angiogenic therapy, which\",\n",
            "          \"enrollment in trials of anti-angiogenic therapy, which is particularly relevant to current controversies over the use of bevacuzimab as a first-line therapy in ovarian cancer.\",\n",
            "          \"Overall, this work illustrates the ability of proteomics to complement genomics in providing additional insights into pathways and processes that drive ovarian cancer biology and how these pathways are altered in correspondence with clinical phenotypes.\",\n",
            "          \"are altered in correspondence with clinical phenotypes. The comprehensive proteomic measurements for the HGSC tumor samples provide a public resource of information. Importantly, the ability to identify PTMs revealed a strong association between specific\",\n",
            "          \"to identify PTMs revealed a strong association between specific histone acetylation events and the HRD phenotype. In addition, the enhanced view of pathway activity provided by measurements of protein phosphorylation provides a foundation for linking\",\n",
            "          \"of protein phosphorylation provides a foundation for linking genotype to proteotype, and ultimately to phenotype for understanding the molecular basis of cancer.\",\n",
            "          \"EXPERIMENTAL PROCEDURES\\n\\nFull methods are available in Extended Experimental Procedures.\\n\\nTumor Samples\",\n",
            "          \"Tumor Samples\\n\\nAll tumor samples for the current study were obtained through the TCGA Biospecimen Core Resource as described previously. Samples were selected based on overall survival, HRD status and availability.\\n\\nQuantitative Proteomics\",\n",
            "          \"Tissue proteins were extracted and digested with trypsin; at each site, a portion of each resulting sample was also used to create a pooled reference in which each tumor sample contributed an equal percentage by peptide mass. The pooled reference sample\",\n",
            "          \"equal percentage by peptide mass. The pooled reference sample was included in each multiplexed, isobaric labeling experiment to enable cross-experiment comparison in the entire sample cohort. The patient samples and the pooled reference sample were each\",\n",
            "          \"The patient samples and the pooled reference sample were each labeled by different iTRAQ reagents (Sciex), combined, fractionated, and split for integrated, quantitative global proteome (10%) and phosphoproteome (90%) analysis using LC-MS/MS on an\",\n",
            "          \"(10%) and phosphoproteome (90%) analysis using LC-MS/MS on an Orbitrap Velos mass spectrometer (Thermo Scientific). Raw data were processed for peptide identification by database searching and identified peptides were assembled as proteins and mapped to\",\n",
            "          \"identified peptides were assembled as proteins and mapped to gene identifiers for proteogenomic comparisons. Quantitation was achieved by comparing the iTRAQ reporter ion intensities in each sample.\",\n",
            "          \"Comparison of mRNA and Protein Subtypes\",\n",
            "          \"A model-based clustering approach was used to model the protein abundance data as a mixture of subtypes. Bayesian information criteria, statistical resampling and VISDA-based sub-phenotype clustering approaches provided similar results with a stable\",\n",
            "          \"clustering approaches provided similar results with a stable optimization on five clusters as determined by consensus clustering. WGCNA analysis was performed to infer genes or gene networks that drive subtyping into 5 clusters, followed by correlation\",\n",
            "          \"that drive subtyping into 5 clusters, followed by correlation with subtype as a trait.\",\n",
            "          \"Integration of Proteomics and CNA data\",\n",
            "          \"Spearman correlation coefficients and corresponding adjusted p values were calculated for each protein/transcript by CNA locus, and gene set enrichment analysis was used to infer function for groups of proteins significantly correlated with a given CNA\",\n",
            "          \"groups of proteins significantly correlated with a given CNA locus. Regression analysis was applied to the list of trans-affected protein correlated with each CNA to generate parsimonious Cox proportional hazards models with maximal predictive ability,\",\n",
            "          \"proportional hazards models with maximal predictive ability, using the PNNL data for training and JHU data for testing. Kaplan-Meier plots were used to visualize performance in predicting overall survival and progression-free survival. A CIN index was\",\n",
            "          \"overall survival and progression-free survival. A CIN index was calculated for each sample as the mean absolute values of copy number measurements at the 29,393 selected loci. Bootstrap resampling was used to select proteins correlated with CIN index at\",\n",
            "          \"was used to select proteins correlated with CIN index at high confidence.\",\n",
            "          \"Protein Acetylation and HRD\",\n",
            "          \"Acetylated peptides were identified by searching the global proteomics data for dynamic acetylation to lysines (+42 Da). Acetylation levels were compared between HRD and non-HRD cases by t-test. Targeted proteomics (SWATH) was used to orthogonally\",\n",
            "          \"by t-test. Targeted proteomics (SWATH) was used to orthogonally quantify the acetylated peptide with synthetic peptides as internal standard.\",\n",
            "          \"Survival-related Pathways Analysis\",\n",
            "          \"Phosphoproteome data from the short and long survivors were mapped to signaling pathways in the NCI pathway information database,(PID (http://pid.nci.nih.gov) using the gene names. For each signaling pathway in the PID, relative abundances for all\",\n",
            "          \"each signaling pathway in the PID, relative abundances for all phosphopeptides mapping to any pathway component were identified and separated into short and long survivor groups. The difference in distributions between the two sets of pathway-specific\",\n",
            "          \"in distributions between the two sets of pathway-specific peptides, i.e., those associated with either short survivors or long long survivors, was then assessed using a two-tailed t test. Similar enrichment analyses were also performed using protein\",\n",
            "          \"Similar enrichment analyses were also performed using protein abundance, mRNA abundance, and CNA.\",\n",
            "          \"Data Repository\",\n",
            "          \"All of the primary mass spectrometry data on TCGA tumor samples are deposited at the CPTAC Data Coordinating Center as raw and mzML files and complete protein assembly data sets for public access (https://cptac-data-portal.georgetown.edu).\",\n",
            "          \"Supplementary Material\",\n",
            "          \"This is a PDF file of an unedited manuscript that has been accepted for publication. As a service to our customers we are providing this early version of the manuscript. The manuscript will undergo copyediting, typesetting, and review of the resulting\",\n",
            "          \"undergo copyediting, typesetting, and review of the resulting proof before it is published in its final form. Please note that during the production process errors may be discovered which could affect the content, and all legal disclaimers that apply to\",\n",
            "          \"affect the content, and all legal disclaimers that apply to the journal pertain.\",\n",
            "          \"Author Contributions\",\n",
            "          \"Study conception and design: HZ, TL, ZZ, RDS, DWC, KDR. Investigation: performed experiment or data collection: TL, SS, FY, JZ, LijunC, PS, PA, YT, MAG, TRC, CC, ST, SN, RJM, HengZ. Computation and statistical analysis: ZZ, SHP, BaiZ, JEM, VAP, LiC, DR,\",\n",
            "          \"and statistical analysis: ZZ, SHP, BaiZ, JEM, VAP, LiC, DR, MEM, SWC, SW, JW, DLT, DF, VB, YW, BingZ. Data interpretation and biological analysis: TL, HZ, SHP, ZZ, BaiZ, JEM, VAP, LiC, DR, SN, CW, IMS, AP, MPS, DAL, RDS, DWC, KDR. Writing/manuscript\",\n",
            "          \"SN, CW, IMS, AP, MPS, DAL, RDS, DWC, KDR. Writing/manuscript preparation and revision: HZ, TL, ZZ, SHP, JEM, DR, VAP, LeslieC, HR, IMS, AP, MPS, DAL, RDS, DWC, KDR. Supervision and administration: HR, ESB, TH, RCR, LS, RDS, DWC, KDR.\",\n",
            "          \"The authors declare no competing financial interests.\\n\\n           SUPPLEMENTAL INFORMATION         \\n\\nSupplemental information includes Extended Experimental Procedures, 5 figures, and 8 tables and can be found with this article online.\",\n",
            "          \"Correlations between mRNA and protein abundance in TCGA tumors\",\n",
            "          \"mRNA and protein were correlated across all the samples, resulting in 90.6% positive correlations, and 79.4% were significantly correlated (Benjamini-Hochberg adjusted p value <0.01), with a mean Spearman\\u2019s correlation of 0.38 and a median value of 0.45\",\n",
            "          \"mean Spearman\\u2019s correlation of 0.38 and a median value of 0.45 (top panel). Different biological pathways and processes showed significantly different levels of correlation (bottom panel). Metabolic pathways and the interferon response displayed high\",\n",
            "          \"Metabolic pathways and the interferon response displayed high mRNA-protein correlation, and ribosome, mRNA splicing, oxidative phosphorylation, and complement and coagulation cascade were poorly correlated. The mean correlation is shown in parentheses\",\n",
            "          \"poorly correlated. The mean correlation is shown in parentheses followed by Benjamini-Hochberg adjusted p values calculated using a Kolmogorov\\u2013Smirnov test following the functional group names from MSIGDB. Blue bars indicate positive correlations and\",\n",
            "          \"names from MSIGDB. Blue bars indicate positive correlations and yellow indicates negative correlations; individual proteins (represented as bars on the x-axis) are sorted by correlation from low to high (bottom panel).\",\n",
            "          \"Proteomic subtypes and corresponding driving protein modules\",\n",
            "          \"Global proteomics abundance is shown in a heatmap with TCGA samples represented by columns ordered by protein subtype where rows represent proteins. Color of each cell indicates z-score (log2 of relative abundance scaled by proteins\\u2019 standard deviation)\",\n",
            "          \"of relative abundance scaled by proteins\\u2019 standard deviation) of the protein in that sample; red is increased and blue is decreased (relative to the pooled reference). Transcriptome-based subtypes (Verhaak et al., 2013) and the proposed proteomic subtypes\",\n",
            "          \"(Verhaak et al., 2013) and the proposed proteomic subtypes are indicated in color above the heatmap. WGCNA-derived modules are delineated with the row color panel and annotated according to the pathways based on enrichment of KEGG and Reactome ontologies.\",\n",
            "          \"Functional impact of copy number alterations\",\n",
            "          \"The top panel shows the correlation of CNAs to protein abundance (right) or mRNA (left) with significant positive correlations in red and negative correlations in blue (Benjamini-Hochberg adjusted p value <0.01, Spearman\\u2019s correlation coefficient). The\",\n",
            "          \"p value <0.01, Spearman\\u2019s correlation coefficient). The x-axis plots the 29,393 CNAs obtained from TCGA. The y-axis plots 3,202 proteins. Genes are ordered by chromosomal location on both the x- and y-axes. The bottom panel shows the summed number of\",\n",
            "          \"the x- and y-axes. The bottom panel shows the summed number of significantly correlated proteins (or mRNA) for each individual CNA. In blue is shown the total number, in black are those genes significantly correlated as both mRNA and protein. Where blue\",\n",
            "          \"significantly correlated as both mRNA and protein. Where blue and black lines have similar magnitude, e.g., the hotspot on chromosome 20, the CNA associations are shared between protein and mRNA. Where a strong blue line has no mirrored black line, e.g.,\",\n",
            "          \"Where a strong blue line has no mirrored black line, e.g., the protein hotspot on chromosome 2, the associations with CNA are largely unique.\",\n",
            "          \"Kaplan-Meier plot of overall survival stratified by CNA-derived signatures\",\n",
            "          \"A survival plot is shown for a consensus of the four best signatures (see Figure S4A and Table S6) identified from analysis of proteins affected in trans by CNAs. Models were trained using a lasso-based Cox proportional hazards model on the samples from\",\n",
            "          \"lasso-based Cox proportional hazards model on the samples from PNNL, and shown are the survival curves from these models applied to the non-overlapping data from the JHU analysis, with the up (red; above the median) and down (blue, below the median)\",\n",
            "          \"up (red; above the median) and down (blue, below the median) signatures each representing 45 patients. A vote was taken among the four most predictive signatures, and the results of this vote are shown. Probability of death is shown on the y-axis vs.\",\n",
            "          \"vote are shown. Probability of death is shown on the y-axis vs. survival time in years on the x-axis. Shaded ribbons denote 95% confidence intervals.\",\n",
            "          \"Number of CNAs statistically explained by proteins significantly associated with CIN index\",\n",
            "          \"A total of 128 proteins selected for their association with CIN were arranged along the chromosomal locations of their corresponding genes (y-axis). The length of the light grey horizontal lines indicates the number of CNAs significantly correlated with\",\n",
            "          \"indicates the number of CNAs significantly correlated with the protein across all chromosomes except for the protein\\u2019s corresponding gene coding region (x-axis). The top-ranked proteins are annotated and highlighted with dark lines to show the\",\n",
            "          \"are annotated and highlighted with dark lines to show the bootstrap-estimated 95% confidence intervals.\",\n",
            "          \"DDN analysis and lysine-acetylation analysis between HRD and non-HRD patients\",\n",
            "          \"DDN analysis revealed a sub-network of proteins that displayed distinct co-expression patterns between HRD and non-HRD patients, where purple connections indicate protein correlations that exist only in HRD samples and blue connections indicate protein\",\n",
            "          \"exist only in HRD samples and blue connections indicate protein correlations that exist only in non-HRD patients. The links between the nodes are drawn with two different thickness indicative of whether the connections meet the significance threshold of\",\n",
            "          \"of whether the connections meet the significance threshold of 0.05 (light lines) or 0.01 (bold lines). Proteins with blue dotted circles are known to be involved in histone acetylation or deacetylation. Further, identification and quantitation of\",\n",
            "          \"or deacetylation. Further, identification and quantitation of lysine-acetylated peptides in global proteomics data showed that acetylation level at K12 and K16 of histone H4 are significantly different between HRD and non-HRD samples, suggesting a role of\",\n",
            "          \"different between HRD and non-HRD samples, suggesting a role of histone H4 acetylation (together with HDAC1) in modulating the choice of DSB repair mechanisms.\",\n",
            "          \"Pathway analysis associated with patient survival\",\n",
            "          \"A) Pathway components were statistically analyzed using a two-sided t test between samples from patients surviving <3 years (short survival) and patients surviving >5 years (long survival) for differences in CNA, mRNA expression, protein or\",\n",
            "          \"survival) for differences in CNA, mRNA expression, protein or phosphorylation abundance. All significant pathways for mRNA and protein are shown (Benjamini-Hochberg adjusted p value <0.05) and the most significant pathways for phosphoproteomics are\",\n",
            "          \"and the most significant pathways for phosphoproteomics are plotted on the x axis as the \\u2212log of the p value. Results show that phosphorylation provides additional information about the functional state of tumors. B) Simplified PDGFR-beta pathway showing\",\n",
            "          \"state of tumors. B) Simplified PDGFR-beta pathway showing differences between short and long survivor groups for protein abundance (p), transcript abundance (m), and phosphopeptide abundance (circled P).\"\n",
            "        ]\n",
            "      }\n",
            "    ]\n",
            "  },\n",
            "  \"PDC000115\": {\n",
            "    \"study_id\": \"bb076b33-57b8-11e8-b07a-00a098d917f8\",\n",
            "    \"pdc_study_id\": \"PDC000115\",\n",
            "    \"study_submitter_id\": \"TCGA OV Phosphoproteome S020-4\",\n",
            "    \"program_id\": \"10251935-5540-11e8-b664-00a098d917f8\",\n",
            "    \"project_id\": \"48af5040-5546-11e8-b664-00a098d917f8\",\n",
            "    \"study_name\": \"TCGA Ovarian PNNL Phosphoproteome Velos Qexactive\",\n",
            "    \"program_name\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"project_name\": \"CPTAC2 Retrospective\",\n",
            "    \"disease_type\": \"Other;Ovarian Serous Cystadenocarcinoma\",\n",
            "    \"primary_site\": \"Not Reported;Ovary\",\n",
            "    \"analytical_fraction\": \"Phosphoproteome\",\n",
            "    \"experiment_type\": \"iTRAQ4\",\n",
            "    \"embargo_date\": null,\n",
            "    \"cases_count\": 70,\n",
            "    \"aliquots_count\": 70,\n",
            "    \"protocol_id\": \"1c06bed3-3b26-11e9-9a07-0a80fada099c\",\n",
            "    \"protocol_submitter_id\": \"P-S020-4-1\",\n",
            "    \"program_submitter_id\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"protocol_name\": \"TCGA_Ovarian_PNNL_Phosphoproteome_Velos\",\n",
            "    \"protocol_date\": \"2013-08-31\",\n",
            "    \"document_name\": \"PNNL_CPTAC_SampleProtocol_TCGA\",\n",
            "    \"quantitation_strategy\": \"Isobaric label quantitation\",\n",
            "    \"label_free_quantitation\": \"\",\n",
            "    \"labeled_quantitation\": \"iTRAQ quantitation\",\n",
            "    \"isobaric_labeling_reagent\": \"iTRAQ\",\n",
            "    \"reporter_ion_ms_level\": \"2\",\n",
            "    \"starting_amount\": \"45.00\",\n",
            "    \"starting_amount_uom\": \"mg\",\n",
            "    \"digestion_reagent\": \"Trypsin\",\n",
            "    \"alkylation_reagent\": \"Iodoacetamide \",\n",
            "    \"enrichment_strategy\": \"Phosphoproteome\",\n",
            "    \"enrichment\": \"Fe3+ NTA-Agarose (Qiagen)\",\n",
            "    \"chromatography_dimensions_count\": \"2\",\n",
            "    \"one_d_chromatography_type\": \"bRPLC (pH 7.5)\",\n",
            "    \"two_d_chromatography_type\": \"RPLC\",\n",
            "    \"fractions_analyzed_count\": \"12\",\n",
            "    \"column_type\": \"C18\",\n",
            "    \"amount_on_column\": \"0.5\",\n",
            "    \"amount_on_column_uom\": \"\\u00b5g\",\n",
            "    \"column_length\": \"60\",\n",
            "    \"column_length_uom\": \"cm\",\n",
            "    \"column_inner_diameter\": \"50.00\",\n",
            "    \"column_inner_diameter_uom\": \"\\u00b5m\",\n",
            "    \"particle_size\": \"3\",\n",
            "    \"particle_size_uom\": \"\\u00b5m\",\n",
            "    \"particle_type\": \"Phenomenex Jupiter C18\",\n",
            "    \"gradient_length\": \"130\",\n",
            "    \"gradient_length_uom\": \"min\",\n",
            "    \"instrument_make\": \"Thermo Scientific\",\n",
            "    \"instrument_model\": \"Orbitrap Velos\",\n",
            "    \"dissociation_type\": \"HCD\",\n",
            "    \"ms1_resolution\": \"30000\",\n",
            "    \"ms2_resolution\": \"7500\",\n",
            "    \"dda_topn\": \"Top 10\",\n",
            "    \"normalized_collision_energy\": \"45%\",\n",
            "    \"acquistion_type\": \"DDA\",\n",
            "    \"dia_multiplexing\": \"N/A\",\n",
            "    \"dia_ims\": \"N/A\",\n",
            "    \"analytical_technique\": null,\n",
            "    \"chromatography_instrument_make\": null,\n",
            "    \"chromatography_instrument_model\": null,\n",
            "    \"polarity\": null,\n",
            "    \"reconstitution_solvent\": null,\n",
            "    \"reconstitution_volume\": null,\n",
            "    \"reconstitution_volume_uom\": null,\n",
            "    \"internal_standards\": null,\n",
            "    \"extraction_method\": null,\n",
            "    \"ionization_mode\": null,\n",
            "    \"publications\": [\n",
            "      {\n",
            "        \"publication_id\": \"44e2e658-89c7-11e8-bcf1-0a2705229b82\",\n",
            "        \"pubmed_id\": \"27372738\",\n",
            "        \"doi\": \"10.1016/j.cell.2016.05.069\",\n",
            "        \"author\": \"Zhang, H.; Liu, T.; Zhang, Z.; Payne, S. H.; Zhang, B.; McDermott, J. E.; Zhou, J. Y.; Petyuk, V. A.; Chen, L.; Ray, D.; Sun, S.; Yang, F.; Chen, L.; Wang, J.; Shah, P.; Cha, S. W.; Aiyetan, P.; Woo, S.; Tian, Y.; Gritsenko, M. A.; Clauss, T. R.; Choi, C.; Monroe, M. E.; Thomas, S.; Nie, S.; Wu, C.; Moore, R. J.; Yu, K. H.; Tabb, D. L.; Feny\\u00f6, D.; Bafna, V.; Wang, Y.; Rodriguez, H.; Boja, E. S.; Hiltke, T.; Rivers, R. C.; Sokoll, L.; Zhu, H.; Shih, I. M.; Cope, L.; Pandey, A.; Zhang, B.; Snyder, M. P.; Levine, D. A.; Smith, R. D.; Chan, D. W.; Rodland, K. D.; Carr, S. A.; Gillette, M. A.; Klauser, K. R.; Kuhn, E.; Mani, D. R.; Mertins, P.; Ketchum, K. A.; Thangudu, R.; Cai, S.; Oberti, M.; Paulovich, A. G.; Whiteaker, J. R.; Edwards, N. J.; McGarvey, P. B.; Madhavan, S.; Wang, P.; Chan, D. W.; Pandey, A.; Shih, I. M.; Zhang, H.; Zhang, Z.; Zhu, H.; Cope, L.; Whiteley, G. A.; Skates, S. J.; White, F. M.; Levine, D. A.; Boja, E. S.; Kinsinger, C. R.; Hiltke, T.; Mesri, M.; Rivers, R. C.; Rodriguez, H.; Shaw, K. M.; Stein, S. E.; Fenyo, D.; Liu, T.; McDermott, J. E.; Payne, S. H.; Rodland, K. D.; Smith, R. D.; Rudnick, P.; Snyder, M.; Zhao, Y.; Chen, X.; Ransohoff, D. F.; Hoofnagle, A. N.; Liebler, D. C.; Sanders, M. E.; Shi, Z.; Slebos, R. J. C.; Tabb, D. L.; Zhang, B.; Zimmerman, L. J.; Wang, Y.; Davies, S. R.; Ding, L.; Ellis, M. J. C.; Townsend, R. R.\",\n",
            "        \"title\": \"Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer\",\n",
            "        \"journal\": \"Cell\",\n",
            "        \"journal_url\": \"https://www.cell.com/cell/fulltext/S0092-8674(16)30673-0?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0092867416306730%3Fshowall%3Dtrue\",\n",
            "        \"year\": \"2016\",\n",
            "        \"abstract\": \"To provide a detailed analysis of the molecular components and underlying mechanisms associated with ovarian cancer, we performed a comprehensive mass-spectrometry-based proteomic characterization of 174 ovarian tumors previously analyzed by The Cancer Genome Atlas (TCGA), of which 169 were high-grade serous carcinomas (HGSCs). Integrating our proteomic measurements with the genomic data yielded a number of insights into disease, such as how different copy-number alternations influence the proteome, the proteins associated with chromosomal instability, the sets of signaling pathways that diverse genome rearrangements converge on, and the ones most associated with short overall survival. Specific protein acetylations associated with homologous recombination deficiency suggest a potential means for stratifying patients for therapy. In addition to providing a valuable resource, these findings provide a view of how the somatic genome drives the cancer proteome and associations between protein and post-translational modification levels and clinical outcomes in HGSC.\",\n",
            "        \"full_text\": [\n",
            "          \"Integrated proteogenomic characterization of human high grade serous ovarian cancer\\n\\nSUMMARY\",\n",
            "          \"To provide a detailed analysis of the molecular components and underlying mechanisms associated with ovarian cancer, we performed a comprehensive mass spectrometry-based proteomic characterization of 174 ovarian tumors previously analyzed by The Cancer\",\n",
            "          \"of 174 ovarian tumors previously analyzed by The Cancer Genome Atlas (TCGA), of which 169 were high-grade serous carcinomas (HGSC). Integrating our proteomic measurements with the genomic data yielded a number of insights into disease such as how\",\n",
            "          \"data yielded a number of insights into disease such as how different copy number alternations influence the proteome, the proteins associated with chromosomal instability, the sets of signaling pathways that diverse genome rearrangements converge on, as\",\n",
            "          \"pathways that diverse genome rearrangements converge on, as well as the ones most associated with short overall survival. Specific protein acetylations associated with homologous recombination deficiency suggest a potential means for stratifying patients\",\n",
            "          \"deficiency suggest a potential means for stratifying patients for therapy. In addition to providing a valuable resource, these findings provide a view of how the somatic genome drives the cancer proteome and associations between protein and\",\n",
            "          \"drives the cancer proteome and associations between protein and post-translational modification levels and clinical outcomes in HGSC.\",\n",
            "          \"INTRODUCTION\",\n",
            "          \"A comprehensive molecular view of cancer is necessary for understanding the underlying mechanisms of disease, improving prognosis, and ultimately guiding treatment. The Cancer Genome Atlas (TCGA) conducted an extensive genomic and transcriptomic\",\n",
            "          \"Atlas (TCGA) conducted an extensive genomic and transcriptomic characterization of ovarian high-grade serous carcinoma (HGSC) aimed at defining the genomic landscape and aiding the development of targeted therapies for this highly lethal malignancy. Key\",\n",
            "          \"of targeted therapies for this highly lethal malignancy. Key findings from TCGA were: 1) the prevalent role of TP53 mutations, 2) extensive DNA copy alterations, 3) preliminary transcriptional signatures associated with survival, 4) varied mechanisms of\",\n",
            "          \"signatures associated with survival, 4) varied mechanisms of BRCA1/2 inactivation, and lastly, 5) CCNE1 aberrations. Subsequent analysis of genomic data from the TCGA consortium led to the refinement of the transcript-defined signatures, improving the\",\n",
            "          \"refinement of the transcript-defined signatures, improving the statistical association with patient outcome, and integrating microRNA and mRNA expression profiles associated with HGSC to identify candidate microRNA targets.\",\n",
            "          \"While the insights from genomic analyses are substantial, functions encoded in the genome are generally executed at the protein level, and are often further modulated by post-translational modifications (PTMs). For example, TCGA used reverse phase protein\",\n",
            "          \"(PTMs). For example, TCGA used reverse phase protein array (RPPA) analysis of 172 proteins (including 31 phosphoproteins with phospho-specific antibodies) to generate a signature associated with the risk of tumor recurrence. To obtain a more comprehensive\",\n",
            "          \"the risk of tumor recurrence. To obtain a more comprehensive assessment of the complex ovarian HGSC phenotype, the Clinical Proteomic Tumor Analysis Consortium (CPTAC) conducted an extensive mass spectrometry (MS)-based proteomic and phosphoproteomic\",\n",
            "          \"mass spectrometry (MS)-based proteomic and phosphoproteomic characterization of HGSC tumors characterized by TCGA, providing quantitative measurements for a combined total of 9,600 proteins from 174 tumors (an average of 7,952 proteins per tumor), and a\",\n",
            "          \"from 174 tumors (an average of 7,952 proteins per tumor), and a total of 24,429 phosphosites from 6,769 phosphoproteins in a subset of 69 tumors (an average of 7,677 phosphosites per tumor). Our results provide insights on HGSC biology and correlate\",\n",
            "          \"Our results provide insights on HGSC biology and correlate differences in protein and PTM levels with clinical outcome complementary to TCGA genomic analyses.\",\n",
            "          \"RESULTS\\n\\nProteomic analysis of TCGA HGSC samples\",\n",
            "          \"HGSC biospecimens and clinical data from 174 patients collected by TCGA were analyzed at two independent CPTAC Centers, Johns Hopkins University (JHU) and Pacific Northwest National Laboratory (PNNL); thirty-two samples were analyzed at both JHU and PNNL.\",\n",
            "          \"(PNNL); thirty-two samples were analyzed at both JHU and PNNL. Tumors were selected by examining the associated TCGA metadata to select tumors either 1) on the basis of putative homologous recombination deficiency (HRD), defined by the presence of\",\n",
            "          \"recombination deficiency (HRD), defined by the presence of germline or somatic BRCA1 or BRCA2 mutations, BRCA1 promoter methylation, or homozygous deletion of PTEN (122 samples; 67 classified as HRD, 55 as non-HRD by the above criteria; JHU), or 2) to\",\n",
            "          \"as HRD, 55 as non-HRD by the above criteria; JHU), or 2) to maximize differences in overall survival (84 samples; PNNL). For selection purposes, short survival was defined as overall survival of less than 3 years, and long survival as greater than 5\",\n",
            "          \"of less than 3 years, and long survival as greater than 5 years. All but five tumors had somatic TP53 mutations, a characteristic feature of HGSC; these five tumors were subsequently reclassified as other than HGSC, and removed from protein functional\",\n",
            "          \"as other than HGSC, and removed from protein functional analyses (e.g., subtyping and survival analyses). The tumor selection criteria and the associated metadata are provided in the Supplemental Information Table S1, available online.\",\n",
            "          \"Proteomics measurements used isobaric tags for relative and absolute quantitation (iTRAQ) in conjunction with offline liquid chromatography fractionation via high-pH reversed phase liquid chromatography (RPLC) and online RPLC with high-resolution tandem\",\n",
            "          \"(RPLC) and online RPLC with high-resolution tandem MS to provide broad coverage for peptide and protein identification and quantification (Supplemental Information); this also alleviated quantitative interference potentially associated with the use of\",\n",
            "          \"interference potentially associated with the use of isobaric tags. We used the relative abundance measurements for each protein in the 32 patient samples analyzed at both JHU and PNNL to normalize across the two analysis sites, and then used clustering,\",\n",
            "          \"across the two analysis sites, and then used clustering, principle component analysis (PCA), and statistical tests to identify any significant batch effects associated with the site of analysis (a detailed comparison of within site, between site, between\",\n",
            "          \"(a detailed comparison of within site, between site, between sample measurement variability and the process used to merge the JHU and PNNL data is given in Supplemental Information, Figure S1). As shown in Figure S1C, the median coefficient of variation\",\n",
            "          \"As shown in Figure S1C, the median coefficient of variation (CV) between measurements at the two sites was 16%.\",\n",
            "          \"A total of 9,600 proteins were identified with high confidence in all tumors, and the relative abundances in each tumor are given in Table S2. Functional analyses and proteome-transcriptome associations were restricted to 3,586 proteins observed and\",\n",
            "          \"associations were restricted to 3,586 proteins observed and quantified in all 169 HGSC samples used for protein functional analyses and where sample variability (signal) exceeded technical variability (noise) in the merged data (Table S2), calculated as\",\n",
            "          \"(noise) in the merged data (Table S2), calculated as described in Supplemental Information. On average we identify peptides covering 29% of the amino acids in any of these 3,586 proteins in a given sample, with a range from 10% to 47%. In addition to\",\n",
            "          \"in a given sample, with a range from 10% to 47%. In addition to protein abundance levels, phosphoproteomics data were acquired for 69 tumors with sufficient sample (Table S2). As with proteins, phosphopeptide abundances were calculated relative to the\",\n",
            "          \"phosphopeptide abundances were calculated relative to the pooled reference sample. Because isobaric labeling was performed prior to splitting the samples for proteome and phosphoproteome analyses, phosphopeptide abundance could be corrected for changes in\",\n",
            "          \"phosphopeptide abundance could be corrected for changes in parent protein abundance to identify differences in the relative extent of phosphorylation at specific sites for each protein (Table S2). Overall, we achieved a protein dynamic range encompassing\",\n",
            "          \"S2). Overall, we achieved a protein dynamic range encompassing >4 orders of magnitude, ranging from low-level transcription factors to abundant structural proteins i.e., actin and tubulin.\",\n",
            "          \"Proteogenomic landscape of HGSC\",\n",
            "          \"The degree to which alterations observed at the genome and transcriptome levels are manifested at the protein level is variable, both qualitatively and quantitatively, and partially driven by multiple levels of post-transcriptional regulation. To identify\",\n",
            "          \"multiple levels of post-transcriptional regulation. To identify peptides encoded by single amino acid variants (SAAVs), splice variants, and novel exons documented by TCGA, mass spectra were searched against a custom graph database containing all peptide\",\n",
            "          \"searched against a custom graph database containing all peptide variations projected from the TCGA genomic analyses of the cohort using a multi-stage analysis pipeline (Supplemental Information). The most frequently observed variant peptides represented\",\n",
            "          \"The most frequently observed variant peptides represented SAAVs and gene-level events. The evidence supporting each novel event is detailed in Table S2, including event type, genomic location, abundance information, and estimated false discovery rate\",\n",
            "          \"abundance information, and estimated false discovery rate (FDR). The validity of these variant peptides was further evaluated by the synthesis of twenty examples selected at random across the entire range of spectrum match scores. We obtained tandem mass\",\n",
            "          \"entire range of spectrum match scores. We obtained tandem mass spectra for all twenty synthetic peptides that matched spectra from our analyses, strongly supporting our observation of these variant peptides; three representative examples are given in\",\n",
            "          \"variant peptides; three representative examples are given in Figure S1E. These results demonstrate the ability to confidently detect, identify and validate genome-level predictions at the protein level. More novel peptides would likely be observed if\",\n",
            "          \"protein level. More novel peptides would likely be observed if there was sufficient sample for more extensive fractionation and/or replicate analyses; these limitations preclude any conclusions about the biological significance of unobserved events. For\",\n",
            "          \"about the biological significance of unobserved events. For example, only 2 mutant p53 peptides were identified despite the presence of p53 mutations in all tumors examined. Such low coverage can occur for reasons that include: excessively large or small\",\n",
            "          \"can occur for reasons that include: excessively large or small tryptic fragments, inability to distinguish some amino acids, some possible biases against highly hydrophobic and hydrophilic peptides, or low abundance peptides co-eluting with very high\",\n",
            "          \"peptides, or low abundance peptides co-eluting with very high abundance peptides.\",\n",
            "          \"To assess the potential for post-transcriptional regulation (e.g., translational efficiency or protein degradation), we compared each protein to its corresponding transcript across all tumors, and correlation was assessed for those pairs with reliable\",\n",
            "          \"and correlation was assessed for those pairs with reliable measurements for both mRNA and protein, i.e., proteins with a corresponding mRNA measurement observed in all 169 HGSC tumors where sample variability exceeded technical variability (3,196 pairs).\",\n",
            "          \"variability exceeded technical variability (3,196 pairs). We excluded 391 proteins observed without a corresponding mRNA (e.g., HBA1) or discordant gene symbol annotation in the protein database (e.g., THOC4). Overall, 79.4% of the mRNA:protein pairs\",\n",
            "          \"(e.g., THOC4). Overall, 79.4% of the mRNA:protein pairs showed statistically significant positive Spearman correlations (Benjamini-Hochberg adjusted p value <0.01; Figure 1) when changes in mRNA abundance were compared to changes in relative protein\",\n",
            "          \"in mRNA abundance were compared to changes in relative protein abundance. The observed median r value of 0.45 for each mRNA-protein pair across all 169 tumors is similar to observations in colorectal cancer, breast cancer and mouse tissues, although less\",\n",
            "          \"cancer, breast cancer and mouse tissues, although less than found for cell lines. In comparison, the median correlation of paired protein measurements from the same sample, but measured at two sites, was substantially higher, at 0.69 (Figure S1).\",\n",
            "          \"A wide range of mRNA:protein correlations was observed. In general, weaker correlations were observed for highly stable and abundant proteins associated with housekeeping or non-intrinsic functions (e.g., ribosomes, mRNA splicing, oxidative\",\n",
            "          \"functions (e.g., ribosomes, mRNA splicing, oxidative phosphorylation, complement cascade), while more dynamic proteins known to be transcriptionally regulated in response to nutrient demand or other perturbations (e.g., nucleotide metabolism, amino acid\",\n",
            "          \"or other perturbations (e.g., nucleotide metabolism, amino acid metabolism, acute inflammatory responses) were more highly correlated (Figure 1 and Table S3). This result is consistent with a previous colorectal cancer study and supports the hypothesis\",\n",
            "          \"a previous colorectal cancer study and supports the hypothesis that while many biological functions are primarily regulated by mRNA abundance, post-transcriptional mechanisms likely have an important role in regulating certain house-keeping functions.\",\n",
            "          \"important role in regulating certain house-keeping functions. Additionally, we found that 23 mRNAs lacking polyA tails displayed lower correlation (mean \\u22120.15) with their cognate proteins than did polyadenylated mRNAs, consistent with the decreased\",\n",
            "          \"than did polyadenylated mRNAs, consistent with the decreased stability of mRNAs lacking polyA tails.\",\n",
            "          \"Clustering of tumors based on protein abundance\",\n",
            "          \"HGSC is the most common of the four major histological subtypes of epithelial ovarian cancer, and is characterized by distinct morphological features. Recent studies using mRNA abundance data have suggested four transcriptomic HGSC subtypes designated as\",\n",
            "          \"have suggested four transcriptomic HGSC subtypes designated as differentiated, immunoreactive, mesenchymal, and proliferative. To build an unbiased molecular taxonomy of ovarian HGSC, we used protein abundance data from the 169 tumors to identify subtypes\",\n",
            "          \"protein abundance data from the 169 tumors to identify subtypes that might show biological differences that could be exploited in future studies. Figure 2 shows the resulting clustering analysis of individual tumors (vertical columns) by protein abundance\",\n",
            "          \"of individual tumors (vertical columns) by protein abundance (horizontal rows). The cluster assignment for each sample is provided in Table S4, and a consensus value matrix for the subtype comparisons is shown in Figure S2A. The results of a Weighted Gene\",\n",
            "          \"is shown in Figure S2A. The results of a Weighted Gene Co-expression Network Analysis (WGCNA) of protein functional enrichment by subgroups are provided for the protein clusters, in Figure S2B and Table S4. The enrichment of KEGG and Reactome ontologies\",\n",
            "          \"and Table S4. The enrichment of KEGG and Reactome ontologies in the WGCNA-derived modules are shown in Figures S2C and S2D, respectively; membership of enriched pathways is provided in Table S3.\",\n",
            "          \"Four of the proteomic clusters showed a clear correspondence to the mesenchymal, proliferative, immunoreactive, and differentiated subtypes defined by the TCGA transcriptome analysis (Figure 2 and Figure S2E). A relatively small fifth cluster of tumors\",\n",
            "          \"2 and Figure S2E). A relatively small fifth cluster of tumors significantly enriched in proteins associated with extracellular matrix interactions, erythrocyte and platelet functions, and the complement cascade was also observed using multiple approaches,\",\n",
            "          \"complement cascade was also observed using multiple approaches, including model based clustering with Bayesian information criteria, consensus clustering, and VISDA-based sub-phenotype clustering. This new group could not be attributed to tissue source\",\n",
            "          \"This new group could not be attributed to tissue source site sampling bias or any other meta-data category, but may be related to tumor characteristics, including vascularization and tumor content, as the Tumor Purity score for this subtype (and the\",\n",
            "          \"content, as the Tumor Purity score for this subtype (and the Mesenchymal subtype) was significantly lower than that of the other three subtypes (Figure S2F). The clinical relevance of these protein-based clusters will require validation in independent\",\n",
            "          \"protein-based clusters will require validation in independent HGSC sample sets, particularly as no significant difference in survival was observed (Figure S2G), similar to mRNA-based clustering analysis.\",\n",
            "          \"Proteomic analysis of CNA effects\",\n",
            "          \"Chromosomal instability, marked by extensive copy number alterations (CNAs) in each tumor, is a hallmark of HGSC, and a likely source of driver alterations in this disease. CNAs can affect the abundance of proteins at the same locus (cis effects), and may\",\n",
            "          \"abundance of proteins at the same locus (cis effects), and may also act in trans, either directly or indirectly.\",\n",
            "          \"Hypothesizing that CNAs with strong trans effects are more likely to elicit a molecular phenotype and confer selective advantages, we sought to identify those CNA regions that have the broadest effect on protein expression. In all, 29,393 CNA loci (Table\",\n",
            "          \"effect on protein expression. In all, 29,393 CNA loci (Table S5) were compared to our global proteomics data with 950,209 CNA/protein pairs (0.72% of the total) exhibiting significant association (Benjamini-Hochberg adjusted p value <0.01). We provide a\",\n",
            "          \"(Benjamini-Hochberg adjusted p value <0.01). We provide a complete list of the significantly associated proteins for each CNA in Table S5. The diagonal line evident in Figure 3 corresponds to cis effects, and vertical stripes correspond to trans effects\",\n",
            "          \"cis effects, and vertical stripes correspond to trans effects where changes in copy number affect expression of numerous proteins across the genome. We performed the same analysis for mRNA to identify sites where associations are transcriptionally\",\n",
            "          \"mRNA to identify sites where associations are transcriptionally mediated. A similar number of CNA/mRNA pairs were found to be significantly associated (1,113,164 at Benjamini-Hochberg adjusted p value <0.01; Figure 3). This analysis revealed regions on\",\n",
            "          \"p value <0.01; Figure 3). This analysis revealed regions on chromosomes 2, 7, 20, and 22 correlated in trans with more than 200 proteins. In contrast to colorectal cancer where most of the trans-regulation of protein by CNA was accompanied by similar\",\n",
            "          \"trans-regulation of protein by CNA was accompanied by similar changes in mRNA, we observed several loci associated with differences in protein abundance without a corresponding change in mRNA. For example, large regions on chromosome 2 have relatively\",\n",
            "          \"For example, large regions on chromosome 2 have relatively little trans effect on mRNA levels, but are associated with more than 200 proteins in trans. Dissecting the mechanisms by which a specific CNA can alter protein levels in trans, without affecting\",\n",
            "          \"CNA can alter protein levels in trans, without affecting the corresponding mRNA is difficult, given the extent of the amplified or deleted regions and the numerous genes affected at a given locus. Possible mechanisms include cis regulation of proteins\",\n",
            "          \"locus. Possible mechanisms include cis regulation of proteins associated with mRNA stability and translational efficiency, such as microRNAs and RNA binding proteins.\",\n",
            "          \"Given the complex pattern of CNAs observed in HGSC, it has been difficult to identify a limited number of high impact genomic alterations that could function as drivers of the disease. We interrogated the trans-affected proteins associated with each\",\n",
            "          \"interrogated the trans-affected proteins associated with each putative CNA for common functions using pathways defined by the KEGG, NCI pathway interaction database (PID) and Reactome databases (Supplemental Information). Proteins associated with cell\",\n",
            "          \"(Supplemental Information). Proteins associated with cell invasion and migration, and proteins related to immune function appeared to be enriched in association with multiple CNAs (Figure S3; Table S3). These observations suggest a convergence of multiple\",\n",
            "          \"Table S3). These observations suggest a convergence of multiple CNA targets on a common set of biological functions, namely motility/invasion and immune regulation, functions that are among the hallmarks of cancer.\",\n",
            "          \"Association of CNA trans-affected proteins with overall survival\",\n",
            "          \"Availability of the TCGA survival data allowed us to use trans-affected protein data from the most influential CNAs (e.g., the four altered regions described on chromosomes 2, 7, 20, and 22; Figure 3) to build a model of overall survival. Because each CNA\",\n",
            "          \"3) to build a model of overall survival. Because each CNA affects many proteins, we used a regression approach that identifies parsimonious Cox proportional hazards models with maximal predictive ability from the list of significantly correlated proteins\",\n",
            "          \"ability from the list of significantly correlated proteins for that CNA. We trained models on the proteomics data from PNNL (82 tumors) and tested the ability of the model to predict survival times in the data from JHU (87 tumors, not including the 32\",\n",
            "          \"times in the data from JHU (87 tumors, not including the 32 overlapping). Each of the four most influential CNA regions produced models that were strongly associated with patient survival (p value <0.01, FDR <0.5% based on randomly selected proteins). A\",\n",
            "          \"value <0.01, FDR <0.5% based on randomly selected proteins). A Kaplan-Meier plot illustrating the predictive value of each of the locus-specific models after validation is shown in Figure S4A; for the Kaplan-Meier plots, \\u2018high\\u2019 and \\u2018low\\u2019 expression were\",\n",
            "          \"for the Kaplan-Meier plots, \\u2018high\\u2019 and \\u2018low\\u2019 expression were defined relative to the median for that signature across all samples, splitting patients into two equal groups of 45 (Table S6). We examined the overlap in predictions for patients for the four\",\n",
            "          \"examined the overlap in predictions for patients for the four signatures and found that the predictions were unanimous (either high or low signature) for 62% of patients and 16% were evenly split, with the remaining having one dissenting signature. This\",\n",
            "          \"split, with the remaining having one dissenting signature. This suggested the utility of combining these signatures, using a voting method where the number of high or low calls was counted for each patient; this substantially improved prediction of\",\n",
            "          \"for each patient; this substantially improved prediction of survival time (p value 1.9e-6; Figure 4). We also examined the effects of tumor stage, tumor grade, patient age, surgical outcome, and platinum status, and found that our proteomic signatures\",\n",
            "          \"and platinum status, and found that our proteomic signatures were not improved by inclusion of these variables.\",\n",
            "          \"For each of these locus-specific models we analyzed the enrichment of genes and their regulatory sequences associated with outcome, and found all four models had genes significantly enriched (Benjamini-Hochberg adjusted p value <0.05) in binding sites for\",\n",
            "          \"adjusted p value <0.05) in binding sites for the proliferation-associated serum response factor (SRF), suggesting that SRF activity may be important in ovarian cancer outcome. Additionally, there was significant enrichment of proteins involved in the\",\n",
            "          \"there was significant enrichment of proteins involved in the regulation of actin cytoskeleton, apoptosis, and adherens junction (Benjamini-Hochberg adjusted p value <0.05). We examined the protein abundance and phosphorylation status of SRF between short\",\n",
            "          \"abundance and phosphorylation status of SRF between short and long surviving groups and observed that they were higher in short surviving patients, but only slightly. Thus, although SRF alone was not predictive, the trends in SRF abundance and\",\n",
            "          \"SRF alone was not predictive, the trends in SRF abundance and phosphorylation are consistent with the observation of enrichment for proteins potentially regulated by SRF binding. Details of the larger gene set analysis are provided in Supplemental\",\n",
            "          \"of the larger gene set analysis are provided in Supplemental Information.\",\n",
            "          \"Several proteins shared across all the signatures are known to be involved in cancer processes. Catenin B2 (CTNNA2) is a cell-cell adhesion protein and tumor suppressor. The Rho GDP dissociation inhibitor (GDI) beta (ARHGDIB) is involved in invasion and\",\n",
            "          \"inhibitor (GDI) beta (ARHGDIB) is involved in invasion and migration in many cancers, and overexpression correlates with progression in pancreatic carcinoma. Protein kinase C and casein kinase substrate in neurons protein 2 (PACSIN2) is a repressor of\",\n",
            "          \"substrate in neurons protein 2 (PACSIN2) is a repressor of cellular migration. Finally, the GTP-binding nuclear protein Ran, is a prognostic marker associated with increased survival in epithelial ovarian cancer. The association of these previously\",\n",
            "          \"epithelial ovarian cancer. The association of these previously described survival-related proteins with genes affected in trans by CNAs suggests a potential mechanism for the parallel activation of multiple pathways associated with poor prognosis in HGSC.\",\n",
            "          \"As a comparison we applied the previously described Provar signature, comprising five protein and four phosphoproteins that showed good survival prediction in the TCGA ovarian cancer dataset. We observed all proteins in the Provar signature in our\",\n",
            "          \"We observed all proteins in the Provar signature in our proteomic data, but only one of the phosphosites (EGFR Y1173). Thus, we used the RPPA data from the original signature (Figure S4B). We found that the Provar signature was prognostic of survival\",\n",
            "          \"We found that the Provar signature was prognostic of survival (Benjamini-Hochberg adjusted p value = 0.11) in the 67 patients examined here (those with phosphoprotein data and had somatic TP53 mutations), but the statistical power of Provar is not as high\",\n",
            "          \"mutations), but the statistical power of Provar is not as high as the statistical power of signatures derived from the CNA trans-affected proteins in this dataset. In addition, integrating the Provar signature with the CNA signatures did not improve\",\n",
            "          \"the Provar signature with the CNA signatures did not improve survival prediction (Figure S4C).\",\n",
            "          \"Identification of proteins associated with chromosomal structural abnormality\",\n",
            "          \"The degree of chromosomal instability exhibited by a tumor can be represented by a calculated chromosome instability (CIN) index, as described previously for lung cancer. Identification of proteins associated with CIN may provide information on the\",\n",
            "          \"of proteins associated with CIN may provide information on the processes contributing to chromosomal instability, while the analysis of trans-affected proteins described above is more closely related to the downstream consequences of specific CNAs. Using\",\n",
            "          \"related to the downstream consequences of specific CNAs. Using a bootstrapping method described in Supplemental Information, we identified a proteomic signature including 128 proteins (Table S7) showing significant correlation with CIN index\",\n",
            "          \"(Table S7) showing significant correlation with CIN index (Benjamini-Hochberg adjusted p value <1e-4, Spearman correlation; Figure S5A). Functional annotation of these proteins (Figure S5B) showed that proteins up-regulated in tumors with high CIN index\",\n",
            "          \"showed that proteins up-regulated in tumors with high CIN index were preferentially involved in chromatin organization (p value 6.90e-5), whereas proteins up-regulated in tumors with low CIN index were more often associated with cell death (p value\",\n",
            "          \"CIN index were more often associated with cell death (p value 2.13e-5). Correlation analysis of protein abundances and CIN indicated that a small number of proteins could account for the majority of the CIN (Figure 5); two of the most strongly associated\",\n",
            "          \"of the CIN (Figure 5); two of the most strongly associated proteins, CHD4 and CHD5, are known to be involved in chromatin organization.\",\n",
            "          \"Identification of proteins associated with HRD status\",\n",
            "          \"Since HRD is associated with susceptibility to PARP inhibitors and improved survival, we sought to elucidate systemic changes associated with HRD and identify biomarkers that might be used to stratify patients for treatment. We defined tumors with HRD as\",\n",
            "          \"stratify patients for treatment. We defined tumors with HRD as having either germline or somatic mutations in BRCA1 or BRCA2, or BRCA1 promoter methylation, or homozygous deletion of PTEN, with an overall survival >1.5 years, while non-HRD patients were\",\n",
            "          \"an overall survival >1.5 years, while non-HRD patients were defined as lacking these genomic aberrations and with a follow-up or time-to-death <2.5 years; additional selection criteria include available residual tissue volume and a tumor tissue\",\n",
            "          \"include available residual tissue volume and a tumor tissue contamination score estimated using copy number alterations. Applying differential dependency network (DDN) analysis on a set of 171 BRCA1- or BRCA2-related proteins curated from the literature\",\n",
            "          \"BRCA1- or BRCA2-related proteins curated from the literature and from the cBio portal, we identified a sub-network of 30 proteins that displayed co-expression patterns differentiating HRD from non-HRD patients (Table S7; Figure 6). Several of the proteins\",\n",
            "          \"non-HRD patients (Table S7; Figure 6). Several of the proteins in these modules are known to be involved in histone acetylation or deacetylation, e.g., HDAC1, RBBP4, RBBP7, and EP300, and HUS1.\",\n",
            "          \"Although statistical association cannot distinguish between drivers and consequences of HRD status, the observed enrichment of proteins associated with histone acetylation motivated us to use an effective database search strategy (Supplemental\",\n",
            "          \"us to use an effective database search strategy (Supplemental Information) to identify and quantify acetylated peptides from the global proteomic data. Comparative analysis of 399 acetylated peptides identified 15 acetylated peptides with significant\",\n",
            "          \"peptides identified 15 acetylated peptides with significant differences between the HRD and non-HRD samples (Table S7). Among these, dual acetylation at K12 and K16 of histone H4 showed a significant difference between HRD and non-HRD samples (Figure 6);\",\n",
            "          \"difference between HRD and non-HRD samples (Figure 6); differential acetylation of K12 and K16 was verified using synthetic peptides and targeted analysis using Sequential Windowed data independent Acquisition of the Total High-resolution Mass Spectra\",\n",
            "          \"Acquisition of the Total High-resolution Mass Spectra (SWATH-MS). In cell line data, acetylation of H4 has previously been reported to be involved in the choice of DNA double strand break (DSB) repair pathways (homologous recombination or non-homologous\",\n",
            "          \"repair pathways (homologous recombination or non-homologous end joining. This relationship is regulated partially by HDAC1, a protein also identified in the DDN analysis. We observed a significant enrichment of HDAC1 and its co-regulated proteins in\",\n",
            "          \"enrichment of HDAC1 and its co-regulated proteins in tumors with HRD and low H4 acetylation relative to non-HRD tumors with high H4 acetylation (DDN analysis: permutation tests, p value <0.05; differentially acetylated peptides: t-tests, p value <0.05\",\n",
            "          \"differentially acetylated peptides: t-tests, p value <0.05 with an estimated FDR <0.5% by bootstrap/permutation tests). The combined observations of increased HDAC1 and associated proteins at the pathway level, together with decreased acetylation of H4 in\",\n",
            "          \"the pathway level, together with decreased acetylation of H4 in HRD patients at the PTM level, provide insight regarding the potential role of HDAC1 in modulating the choice of DSB repair pathways.\",\n",
            "          \"Phosphoproteomic analysis of pathways associated with survival\",\n",
            "          \"An initial set of 24,464 different phosphopeptides (21,298 unique phosphorylation sites) contained within 4,420 proteins having data on net differences in phosphorylation (Table S2) was obtained. Since ischemia of the TCGA tumor samples may alter\",\n",
            "          \"obtained. Since ischemia of the TCGA tumor samples may alter phosphopeptide abundance, we removed any phosphorylation sites previously shown to be altered by ischemia. We also removed three tumor samples having substantially higher than average missing\",\n",
            "          \"tumor samples having substantially higher than average missing values and two tumor samples lacking somatic TP53 mutations (Supplemental Information). This yielded a final set of 7,675 different phosphopeptides (6,802 unique phosphorylation sites) from\",\n",
            "          \"phosphopeptides (6,802 unique phosphorylation sites) from 2,324 proteins (Table S2) that were mapped to pathways via the NCI PID. The average net phosphorylation of all phosphopeptides mapping to a given pathway (i.e., the statistical average across all\",\n",
            "          \"to a given pathway (i.e., the statistical average across all phosphoproteins known to be in the pathway, rather than the statistically significant individual proteins) was used as a measure of pathway activity and compared between short survivors\",\n",
            "          \"of pathway activity and compared between short survivors (deceased surviving <3 years; N=17) and long survivors (patients surviving >5 years; N=19) using both proteomic and phosphoproteomic data.\",\n",
            "          \"An issue in ranking the pathways is the distinction between phosphopeptides that are unique to a single pathway, such as the receptor peptides themselves, vs. peptides that map to proteins shared between pathways. In order to address this issue, we\",\n",
            "          \"shared between pathways. In order to address this issue, we performed two analyses: one of all proteins associated with a specific pathway in the NCID database, and a second analysis in which proteins common to multiple pathways were excluded. Figure 7A\",\n",
            "          \"proteins common to multiple pathways were excluded. Figure 7A shows a ranking of activated pathways, as inferred from the analysis of pathway specific components (i.e., differences in mRNA, protein and phosphoprotein levels) in short vs. long survivors.\",\n",
            "          \"protein and phosphoprotein levels) in short vs. long survivors. Fifteen pathways showed increased phosphorylation at Benjamini-Hochberg adjusted p values <0.01, in contrast with transcriptional data for the same samples which yielded only one\",\n",
            "          \"data for the same samples which yielded only one statistically significantly increased pathway, androgen receptor signaling, which was also increased at the phosphoprotein level (Figure 7A and Supplemental Information). All pathways indicated in Figure 7A\",\n",
            "          \"Supplemental Information). All pathways indicated in Figure 7A were increased in association with short survival relative to long survival, though some individual components within a pathway showed opposite changes (e.g. JNK1 transcript and protein\",\n",
            "          \"showed opposite changes (e.g. JNK1 transcript and protein levels, see Figure 7B). The RhoA-regulatory, PDGFRB, and integrin-like kinase pathways emerged as the most activated in this analysis (Figure 7A). Figure 7B shows the common and unique elements of\",\n",
            "          \"(Figure 7A). Figure 7B shows the common and unique elements of the PDGFRB pathway, and the trends in mRNA, protein, and phosphoprotein levels in the comparison of short vs. long survivors, illustrating the benefit of combined protein and phosphoprotein\",\n",
            "          \"illustrating the benefit of combined protein and phosphoprotein measurements, as well as the substantial contribution of protein phosphorylation data to the overall analysis.\",\n",
            "          \"We compared our results to the 31 phosphoproteins represented on the RPPA arrays used in the original TCGA ovarian cancer analysis; RPPA identified differential phosphorylation of ERK1, RAF1, and STAT3 in the same patient samples analyzed by MS. For short\",\n",
            "          \"and STAT3 in the same patient samples analyzed by MS. For short surviving vs. long surviving patients, ERK1 showed statistically significantly increased phosphorylation by RPPA, on the same ERK1 phosphopeptides as found using MS, while AKT1, RAF1 and\",\n",
            "          \"ERK1 phosphopeptides as found using MS, while AKT1, RAF1 and STAT3 showed increased phosphorylation by both RPPA and MS-based analyses, but based upon distinct phosphopeptides (Table S2). Good overlap was observed between RPPA and MS results for\",\n",
            "          \"S2). Good overlap was observed between RPPA and MS results for phosphoproteins on the RPPA array, but we did not observe three of the specific phosphosites from the Provar survival signature in our MS data, suggesting that proteomics is more sensitive to\",\n",
            "          \"in our MS data, suggesting that proteomics is more sensitive to overall pathway activity than specific regulatory events in the pathway itself. However, many of the phosphopeptides identified by MS phosphoproteomics as components of the PDGFR-beta pathway\",\n",
            "          \"by MS phosphoproteomics as components of the PDGFR-beta pathway were not represented in the RPPA, illustrating the ability to identify phosphorylation events not readily accessible with currently available antibodies (Figure 7B).\",\n",
            "          \"DISCUSSION\",\n",
            "          \"The addition of proteomic information, including post-translational modifications, to the rich genomic and transcriptomic data available from the ovarian HGSC samples analyzed by TCGA has provided additional insights into the biology of HGSC, including\",\n",
            "          \"additional insights into the biology of HGSC, including identification of changes at the level of pathway activation. Integration of genomic, proteomic, and phosphoproteomic measurements identified differentially regulated pathways and functional modules\",\n",
            "          \"differentially regulated pathways and functional modules that displayed significant associations with patient outcomes, including survival and HRD status. Cox proportional hazard analysis of proteins associated with CNAs identified overall survival\",\n",
            "          \"of proteins associated with CNAs identified overall survival signatures enriched for targets of the proliferation-associated transcription factor SRF, and integrated proteomic, phosphoproteomic and transcriptomic data identified pathways that\",\n",
            "          \"and transcriptomic data identified pathways that differentiated patients on the basis of survival, including the PDGFR-beta signaling pathway associated with angiogenesis, the RhoA regulatory and integrin-linked kinase pathways associated with cell\",\n",
            "          \"and integrin-linked kinase pathways associated with cell mobility and invasion, and pathways associated with chemokine signaling and adaptive immunity. Proteins associated with cell invasion and motility emerged from both the integration of CNAs with\",\n",
            "          \"and motility emerged from both the integration of CNAs with protein abundance data and the phosphoproteomic investigation of up-regulated pathways in short versus long survivors. Despite the complexity of genomic alterations that characterize HGSC, these\",\n",
            "          \"complexity of genomic alterations that characterize HGSC, these analyses suggest a functional convergence on a subset of key pathways. The association of increased invasiveness and motility with short overall survival in this study may help to explain\",\n",
            "          \"with short overall survival in this study may help to explain more aggressive mechanisms of dissemination in ovarian cancer, including recently reported hematogenous metastasis, in addition to previously described lymphatic spread and known peritoneal\",\n",
            "          \"to previously described lymphatic spread and known peritoneal spread by direct extension.\",\n",
            "          \"Focusing on functional modules has also revealed potential drivers of HGSC and more robust signatures for potentially stratifying ovarian HGSC patients into distinct cancer phenotypes to inform therapeutic management. For example, the identification of\",\n",
            "          \"therapeutic management. For example, the identification of specific acetylation events associated with HRD, e.g., the simultaneous acetylation of K12 and K16 on histone H4, may provide an alternative biomarker of HRD and a rationale for the selection of\",\n",
            "          \"biomarker of HRD and a rationale for the selection of patients in future clinical trials of HDAC inhibitors alone or in combination with PARP inhibition. This may help to resolve the current discrepancy between the initial observation of limited single\",\n",
            "          \"discrepancy between the initial observation of limited single agent activity of HDAC inhibitors in ovarian cancer, and more recent findings of a >40% response rate when used in combination with cytotoxic chemotherapy in platinum-resistant patients.\",\n",
            "          \"with cytotoxic chemotherapy in platinum-resistant patients. Similarly, comprehensive interrogation of protein phosphorylation identified multiple pathways with significantly increased phosphorylation in patients with poor clinical outcomes. Specifically,\",\n",
            "          \"in patients with poor clinical outcomes. Specifically, the observed activation of PDGFR pathways in a subset of patients with short overall survival could potentially stratify patients for selective enrollment in trials of anti-angiogenic therapy, which\",\n",
            "          \"enrollment in trials of anti-angiogenic therapy, which is particularly relevant to current controversies over the use of bevacuzimab as a first-line therapy in ovarian cancer.\",\n",
            "          \"Overall, this work illustrates the ability of proteomics to complement genomics in providing additional insights into pathways and processes that drive ovarian cancer biology and how these pathways are altered in correspondence with clinical phenotypes.\",\n",
            "          \"are altered in correspondence with clinical phenotypes. The comprehensive proteomic measurements for the HGSC tumor samples provide a public resource of information. Importantly, the ability to identify PTMs revealed a strong association between specific\",\n",
            "          \"to identify PTMs revealed a strong association between specific histone acetylation events and the HRD phenotype. In addition, the enhanced view of pathway activity provided by measurements of protein phosphorylation provides a foundation for linking\",\n",
            "          \"of protein phosphorylation provides a foundation for linking genotype to proteotype, and ultimately to phenotype for understanding the molecular basis of cancer.\",\n",
            "          \"EXPERIMENTAL PROCEDURES\\n\\nFull methods are available in Extended Experimental Procedures.\\n\\nTumor Samples\",\n",
            "          \"Tumor Samples\\n\\nAll tumor samples for the current study were obtained through the TCGA Biospecimen Core Resource as described previously. Samples were selected based on overall survival, HRD status and availability.\\n\\nQuantitative Proteomics\",\n",
            "          \"Tissue proteins were extracted and digested with trypsin; at each site, a portion of each resulting sample was also used to create a pooled reference in which each tumor sample contributed an equal percentage by peptide mass. The pooled reference sample\",\n",
            "          \"equal percentage by peptide mass. The pooled reference sample was included in each multiplexed, isobaric labeling experiment to enable cross-experiment comparison in the entire sample cohort. The patient samples and the pooled reference sample were each\",\n",
            "          \"The patient samples and the pooled reference sample were each labeled by different iTRAQ reagents (Sciex), combined, fractionated, and split for integrated, quantitative global proteome (10%) and phosphoproteome (90%) analysis using LC-MS/MS on an\",\n",
            "          \"(10%) and phosphoproteome (90%) analysis using LC-MS/MS on an Orbitrap Velos mass spectrometer (Thermo Scientific). Raw data were processed for peptide identification by database searching and identified peptides were assembled as proteins and mapped to\",\n",
            "          \"identified peptides were assembled as proteins and mapped to gene identifiers for proteogenomic comparisons. Quantitation was achieved by comparing the iTRAQ reporter ion intensities in each sample.\",\n",
            "          \"Comparison of mRNA and Protein Subtypes\",\n",
            "          \"A model-based clustering approach was used to model the protein abundance data as a mixture of subtypes. Bayesian information criteria, statistical resampling and VISDA-based sub-phenotype clustering approaches provided similar results with a stable\",\n",
            "          \"clustering approaches provided similar results with a stable optimization on five clusters as determined by consensus clustering. WGCNA analysis was performed to infer genes or gene networks that drive subtyping into 5 clusters, followed by correlation\",\n",
            "          \"that drive subtyping into 5 clusters, followed by correlation with subtype as a trait.\",\n",
            "          \"Integration of Proteomics and CNA data\",\n",
            "          \"Spearman correlation coefficients and corresponding adjusted p values were calculated for each protein/transcript by CNA locus, and gene set enrichment analysis was used to infer function for groups of proteins significantly correlated with a given CNA\",\n",
            "          \"groups of proteins significantly correlated with a given CNA locus. Regression analysis was applied to the list of trans-affected protein correlated with each CNA to generate parsimonious Cox proportional hazards models with maximal predictive ability,\",\n",
            "          \"proportional hazards models with maximal predictive ability, using the PNNL data for training and JHU data for testing. Kaplan-Meier plots were used to visualize performance in predicting overall survival and progression-free survival. A CIN index was\",\n",
            "          \"overall survival and progression-free survival. A CIN index was calculated for each sample as the mean absolute values of copy number measurements at the 29,393 selected loci. Bootstrap resampling was used to select proteins correlated with CIN index at\",\n",
            "          \"was used to select proteins correlated with CIN index at high confidence.\",\n",
            "          \"Protein Acetylation and HRD\",\n",
            "          \"Acetylated peptides were identified by searching the global proteomics data for dynamic acetylation to lysines (+42 Da). Acetylation levels were compared between HRD and non-HRD cases by t-test. Targeted proteomics (SWATH) was used to orthogonally\",\n",
            "          \"by t-test. Targeted proteomics (SWATH) was used to orthogonally quantify the acetylated peptide with synthetic peptides as internal standard.\",\n",
            "          \"Survival-related Pathways Analysis\",\n",
            "          \"Phosphoproteome data from the short and long survivors were mapped to signaling pathways in the NCI pathway information database,(PID (http://pid.nci.nih.gov) using the gene names. For each signaling pathway in the PID, relative abundances for all\",\n",
            "          \"each signaling pathway in the PID, relative abundances for all phosphopeptides mapping to any pathway component were identified and separated into short and long survivor groups. The difference in distributions between the two sets of pathway-specific\",\n",
            "          \"in distributions between the two sets of pathway-specific peptides, i.e., those associated with either short survivors or long long survivors, was then assessed using a two-tailed t test. Similar enrichment analyses were also performed using protein\",\n",
            "          \"Similar enrichment analyses were also performed using protein abundance, mRNA abundance, and CNA.\",\n",
            "          \"Data Repository\",\n",
            "          \"All of the primary mass spectrometry data on TCGA tumor samples are deposited at the CPTAC Data Coordinating Center as raw and mzML files and complete protein assembly data sets for public access (https://cptac-data-portal.georgetown.edu).\",\n",
            "          \"Supplementary Material\",\n",
            "          \"This is a PDF file of an unedited manuscript that has been accepted for publication. As a service to our customers we are providing this early version of the manuscript. The manuscript will undergo copyediting, typesetting, and review of the resulting\",\n",
            "          \"undergo copyediting, typesetting, and review of the resulting proof before it is published in its final form. Please note that during the production process errors may be discovered which could affect the content, and all legal disclaimers that apply to\",\n",
            "          \"affect the content, and all legal disclaimers that apply to the journal pertain.\",\n",
            "          \"Author Contributions\",\n",
            "          \"Study conception and design: HZ, TL, ZZ, RDS, DWC, KDR. Investigation: performed experiment or data collection: TL, SS, FY, JZ, LijunC, PS, PA, YT, MAG, TRC, CC, ST, SN, RJM, HengZ. Computation and statistical analysis: ZZ, SHP, BaiZ, JEM, VAP, LiC, DR,\",\n",
            "          \"and statistical analysis: ZZ, SHP, BaiZ, JEM, VAP, LiC, DR, MEM, SWC, SW, JW, DLT, DF, VB, YW, BingZ. Data interpretation and biological analysis: TL, HZ, SHP, ZZ, BaiZ, JEM, VAP, LiC, DR, SN, CW, IMS, AP, MPS, DAL, RDS, DWC, KDR. Writing/manuscript\",\n",
            "          \"SN, CW, IMS, AP, MPS, DAL, RDS, DWC, KDR. Writing/manuscript preparation and revision: HZ, TL, ZZ, SHP, JEM, DR, VAP, LeslieC, HR, IMS, AP, MPS, DAL, RDS, DWC, KDR. Supervision and administration: HR, ESB, TH, RCR, LS, RDS, DWC, KDR.\",\n",
            "          \"The authors declare no competing financial interests.\\n\\n           SUPPLEMENTAL INFORMATION         \\n\\nSupplemental information includes Extended Experimental Procedures, 5 figures, and 8 tables and can be found with this article online.\",\n",
            "          \"Correlations between mRNA and protein abundance in TCGA tumors\",\n",
            "          \"mRNA and protein were correlated across all the samples, resulting in 90.6% positive correlations, and 79.4% were significantly correlated (Benjamini-Hochberg adjusted p value <0.01), with a mean Spearman\\u2019s correlation of 0.38 and a median value of 0.45\",\n",
            "          \"mean Spearman\\u2019s correlation of 0.38 and a median value of 0.45 (top panel). Different biological pathways and processes showed significantly different levels of correlation (bottom panel). Metabolic pathways and the interferon response displayed high\",\n",
            "          \"Metabolic pathways and the interferon response displayed high mRNA-protein correlation, and ribosome, mRNA splicing, oxidative phosphorylation, and complement and coagulation cascade were poorly correlated. The mean correlation is shown in parentheses\",\n",
            "          \"poorly correlated. The mean correlation is shown in parentheses followed by Benjamini-Hochberg adjusted p values calculated using a Kolmogorov\\u2013Smirnov test following the functional group names from MSIGDB. Blue bars indicate positive correlations and\",\n",
            "          \"names from MSIGDB. Blue bars indicate positive correlations and yellow indicates negative correlations; individual proteins (represented as bars on the x-axis) are sorted by correlation from low to high (bottom panel).\",\n",
            "          \"Proteomic subtypes and corresponding driving protein modules\",\n",
            "          \"Global proteomics abundance is shown in a heatmap with TCGA samples represented by columns ordered by protein subtype where rows represent proteins. Color of each cell indicates z-score (log2 of relative abundance scaled by proteins\\u2019 standard deviation)\",\n",
            "          \"of relative abundance scaled by proteins\\u2019 standard deviation) of the protein in that sample; red is increased and blue is decreased (relative to the pooled reference). Transcriptome-based subtypes (Verhaak et al., 2013) and the proposed proteomic subtypes\",\n",
            "          \"(Verhaak et al., 2013) and the proposed proteomic subtypes are indicated in color above the heatmap. WGCNA-derived modules are delineated with the row color panel and annotated according to the pathways based on enrichment of KEGG and Reactome ontologies.\",\n",
            "          \"Functional impact of copy number alterations\",\n",
            "          \"The top panel shows the correlation of CNAs to protein abundance (right) or mRNA (left) with significant positive correlations in red and negative correlations in blue (Benjamini-Hochberg adjusted p value <0.01, Spearman\\u2019s correlation coefficient). The\",\n",
            "          \"p value <0.01, Spearman\\u2019s correlation coefficient). The x-axis plots the 29,393 CNAs obtained from TCGA. The y-axis plots 3,202 proteins. Genes are ordered by chromosomal location on both the x- and y-axes. The bottom panel shows the summed number of\",\n",
            "          \"the x- and y-axes. The bottom panel shows the summed number of significantly correlated proteins (or mRNA) for each individual CNA. In blue is shown the total number, in black are those genes significantly correlated as both mRNA and protein. Where blue\",\n",
            "          \"significantly correlated as both mRNA and protein. Where blue and black lines have similar magnitude, e.g., the hotspot on chromosome 20, the CNA associations are shared between protein and mRNA. Where a strong blue line has no mirrored black line, e.g.,\",\n",
            "          \"Where a strong blue line has no mirrored black line, e.g., the protein hotspot on chromosome 2, the associations with CNA are largely unique.\",\n",
            "          \"Kaplan-Meier plot of overall survival stratified by CNA-derived signatures\",\n",
            "          \"A survival plot is shown for a consensus of the four best signatures (see Figure S4A and Table S6) identified from analysis of proteins affected in trans by CNAs. Models were trained using a lasso-based Cox proportional hazards model on the samples from\",\n",
            "          \"lasso-based Cox proportional hazards model on the samples from PNNL, and shown are the survival curves from these models applied to the non-overlapping data from the JHU analysis, with the up (red; above the median) and down (blue, below the median)\",\n",
            "          \"up (red; above the median) and down (blue, below the median) signatures each representing 45 patients. A vote was taken among the four most predictive signatures, and the results of this vote are shown. Probability of death is shown on the y-axis vs.\",\n",
            "          \"vote are shown. Probability of death is shown on the y-axis vs. survival time in years on the x-axis. Shaded ribbons denote 95% confidence intervals.\",\n",
            "          \"Number of CNAs statistically explained by proteins significantly associated with CIN index\",\n",
            "          \"A total of 128 proteins selected for their association with CIN were arranged along the chromosomal locations of their corresponding genes (y-axis). The length of the light grey horizontal lines indicates the number of CNAs significantly correlated with\",\n",
            "          \"indicates the number of CNAs significantly correlated with the protein across all chromosomes except for the protein\\u2019s corresponding gene coding region (x-axis). The top-ranked proteins are annotated and highlighted with dark lines to show the\",\n",
            "          \"are annotated and highlighted with dark lines to show the bootstrap-estimated 95% confidence intervals.\",\n",
            "          \"DDN analysis and lysine-acetylation analysis between HRD and non-HRD patients\",\n",
            "          \"DDN analysis revealed a sub-network of proteins that displayed distinct co-expression patterns between HRD and non-HRD patients, where purple connections indicate protein correlations that exist only in HRD samples and blue connections indicate protein\",\n",
            "          \"exist only in HRD samples and blue connections indicate protein correlations that exist only in non-HRD patients. The links between the nodes are drawn with two different thickness indicative of whether the connections meet the significance threshold of\",\n",
            "          \"of whether the connections meet the significance threshold of 0.05 (light lines) or 0.01 (bold lines). Proteins with blue dotted circles are known to be involved in histone acetylation or deacetylation. Further, identification and quantitation of\",\n",
            "          \"or deacetylation. Further, identification and quantitation of lysine-acetylated peptides in global proteomics data showed that acetylation level at K12 and K16 of histone H4 are significantly different between HRD and non-HRD samples, suggesting a role of\",\n",
            "          \"different between HRD and non-HRD samples, suggesting a role of histone H4 acetylation (together with HDAC1) in modulating the choice of DSB repair mechanisms.\",\n",
            "          \"Pathway analysis associated with patient survival\",\n",
            "          \"A) Pathway components were statistically analyzed using a two-sided t test between samples from patients surviving <3 years (short survival) and patients surviving >5 years (long survival) for differences in CNA, mRNA expression, protein or\",\n",
            "          \"survival) for differences in CNA, mRNA expression, protein or phosphorylation abundance. All significant pathways for mRNA and protein are shown (Benjamini-Hochberg adjusted p value <0.05) and the most significant pathways for phosphoproteomics are\",\n",
            "          \"and the most significant pathways for phosphoproteomics are plotted on the x axis as the \\u2212log of the p value. Results show that phosphorylation provides additional information about the functional state of tumors. B) Simplified PDGFR-beta pathway showing\",\n",
            "          \"state of tumors. B) Simplified PDGFR-beta pathway showing differences between short and long survivor groups for protein abundance (p), transcript abundance (m), and phosphopeptide abundance (circled P).\"\n",
            "        ]\n",
            "      }\n",
            "    ]\n",
            "  },\n",
            "  \"PDC000114\": {\n",
            "    \"study_id\": \"baa8ae46-57b8-11e8-b07a-00a098d917f8\",\n",
            "    \"pdc_study_id\": \"PDC000114\",\n",
            "    \"study_submitter_id\": \"TCGA OV Proteome S020-3\",\n",
            "    \"program_id\": \"10251935-5540-11e8-b664-00a098d917f8\",\n",
            "    \"project_id\": \"48af5040-5546-11e8-b664-00a098d917f8\",\n",
            "    \"study_name\": \"TCGA Ovarian PNNL Proteome\",\n",
            "    \"program_name\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"project_name\": \"CPTAC2 Retrospective\",\n",
            "    \"disease_type\": \"Other;Ovarian Serous Cystadenocarcinoma\",\n",
            "    \"primary_site\": \"Not Reported;Ovary\",\n",
            "    \"analytical_fraction\": \"Proteome\",\n",
            "    \"experiment_type\": \"iTRAQ4\",\n",
            "    \"embargo_date\": null,\n",
            "    \"cases_count\": 85,\n",
            "    \"aliquots_count\": 85,\n",
            "    \"protocol_id\": \"1c06bbb7-3b26-11e9-9a07-0a80fada099c\",\n",
            "    \"protocol_submitter_id\": \"P-S020-3\",\n",
            "    \"program_submitter_id\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"protocol_name\": \"TCGA_Ovarian_PNNL_Proteome\",\n",
            "    \"protocol_date\": \"2013-08-31\",\n",
            "    \"document_name\": \"PNNL_CPTAC_SampleProtocol_TCGA\",\n",
            "    \"quantitation_strategy\": \"Isobaric label quantitation\",\n",
            "    \"label_free_quantitation\": \"\",\n",
            "    \"labeled_quantitation\": \"iTRAQ quantitation\",\n",
            "    \"isobaric_labeling_reagent\": \"iTRAQ\",\n",
            "    \"reporter_ion_ms_level\": \"2\",\n",
            "    \"starting_amount\": \"5.00\",\n",
            "    \"starting_amount_uom\": \"mg\",\n",
            "    \"digestion_reagent\": \"Trypsin\",\n",
            "    \"alkylation_reagent\": \"Iodoacetamide \",\n",
            "    \"enrichment_strategy\": \"Proteome\",\n",
            "    \"enrichment\": \"N/A\",\n",
            "    \"chromatography_dimensions_count\": \"2\",\n",
            "    \"one_d_chromatography_type\": \"bRPLC (pH 7.5)\",\n",
            "    \"two_d_chromatography_type\": \"RPLC\",\n",
            "    \"fractions_analyzed_count\": \"24\",\n",
            "    \"column_type\": \"C18\",\n",
            "    \"amount_on_column\": \"0.75\",\n",
            "    \"amount_on_column_uom\": \"\\u00b5g\",\n",
            "    \"column_length\": \"35\",\n",
            "    \"column_length_uom\": \"cm\",\n",
            "    \"column_inner_diameter\": \"75.00\",\n",
            "    \"column_inner_diameter_uom\": \"\\u00b5m\",\n",
            "    \"particle_size\": \"3\",\n",
            "    \"particle_size_uom\": \"\\u00b5m\",\n",
            "    \"particle_type\": \"Phenomenex Jupiter C18\",\n",
            "    \"gradient_length\": \"100\",\n",
            "    \"gradient_length_uom\": \"min\",\n",
            "    \"instrument_make\": \"Thermo Scientific\",\n",
            "    \"instrument_model\": \"Orbitrap Velos\",\n",
            "    \"dissociation_type\": \"HCD\",\n",
            "    \"ms1_resolution\": \"30000\",\n",
            "    \"ms2_resolution\": \"7500\",\n",
            "    \"dda_topn\": \"Top 10\",\n",
            "    \"normalized_collision_energy\": \"45%\",\n",
            "    \"acquistion_type\": \"DDA\",\n",
            "    \"dia_multiplexing\": \"N/A\",\n",
            "    \"dia_ims\": \"N/A\",\n",
            "    \"analytical_technique\": null,\n",
            "    \"chromatography_instrument_make\": null,\n",
            "    \"chromatography_instrument_model\": null,\n",
            "    \"polarity\": null,\n",
            "    \"reconstitution_solvent\": null,\n",
            "    \"reconstitution_volume\": null,\n",
            "    \"reconstitution_volume_uom\": null,\n",
            "    \"internal_standards\": null,\n",
            "    \"extraction_method\": null,\n",
            "    \"ionization_mode\": null,\n",
            "    \"publications\": [\n",
            "      {\n",
            "        \"publication_id\": \"44e2e658-89c7-11e8-bcf1-0a2705229b82\",\n",
            "        \"pubmed_id\": \"27372738\",\n",
            "        \"doi\": \"10.1016/j.cell.2016.05.069\",\n",
            "        \"author\": \"Zhang, H.; Liu, T.; Zhang, Z.; Payne, S. H.; Zhang, B.; McDermott, J. E.; Zhou, J. Y.; Petyuk, V. A.; Chen, L.; Ray, D.; Sun, S.; Yang, F.; Chen, L.; Wang, J.; Shah, P.; Cha, S. W.; Aiyetan, P.; Woo, S.; Tian, Y.; Gritsenko, M. A.; Clauss, T. R.; Choi, C.; Monroe, M. E.; Thomas, S.; Nie, S.; Wu, C.; Moore, R. J.; Yu, K. H.; Tabb, D. L.; Feny\\u00f6, D.; Bafna, V.; Wang, Y.; Rodriguez, H.; Boja, E. S.; Hiltke, T.; Rivers, R. C.; Sokoll, L.; Zhu, H.; Shih, I. M.; Cope, L.; Pandey, A.; Zhang, B.; Snyder, M. P.; Levine, D. A.; Smith, R. D.; Chan, D. W.; Rodland, K. D.; Carr, S. A.; Gillette, M. A.; Klauser, K. R.; Kuhn, E.; Mani, D. R.; Mertins, P.; Ketchum, K. A.; Thangudu, R.; Cai, S.; Oberti, M.; Paulovich, A. G.; Whiteaker, J. R.; Edwards, N. J.; McGarvey, P. B.; Madhavan, S.; Wang, P.; Chan, D. W.; Pandey, A.; Shih, I. M.; Zhang, H.; Zhang, Z.; Zhu, H.; Cope, L.; Whiteley, G. A.; Skates, S. J.; White, F. M.; Levine, D. A.; Boja, E. S.; Kinsinger, C. R.; Hiltke, T.; Mesri, M.; Rivers, R. C.; Rodriguez, H.; Shaw, K. M.; Stein, S. E.; Fenyo, D.; Liu, T.; McDermott, J. E.; Payne, S. H.; Rodland, K. D.; Smith, R. D.; Rudnick, P.; Snyder, M.; Zhao, Y.; Chen, X.; Ransohoff, D. F.; Hoofnagle, A. N.; Liebler, D. C.; Sanders, M. E.; Shi, Z.; Slebos, R. J. C.; Tabb, D. L.; Zhang, B.; Zimmerman, L. J.; Wang, Y.; Davies, S. R.; Ding, L.; Ellis, M. J. C.; Townsend, R. R.\",\n",
            "        \"title\": \"Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer\",\n",
            "        \"journal\": \"Cell\",\n",
            "        \"journal_url\": \"https://www.cell.com/cell/fulltext/S0092-8674(16)30673-0?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0092867416306730%3Fshowall%3Dtrue\",\n",
            "        \"year\": \"2016\",\n",
            "        \"abstract\": \"To provide a detailed analysis of the molecular components and underlying mechanisms associated with ovarian cancer, we performed a comprehensive mass-spectrometry-based proteomic characterization of 174 ovarian tumors previously analyzed by The Cancer Genome Atlas (TCGA), of which 169 were high-grade serous carcinomas (HGSCs). Integrating our proteomic measurements with the genomic data yielded a number of insights into disease, such as how different copy-number alternations influence the proteome, the proteins associated with chromosomal instability, the sets of signaling pathways that diverse genome rearrangements converge on, and the ones most associated with short overall survival. Specific protein acetylations associated with homologous recombination deficiency suggest a potential means for stratifying patients for therapy. In addition to providing a valuable resource, these findings provide a view of how the somatic genome drives the cancer proteome and associations between protein and post-translational modification levels and clinical outcomes in HGSC.\",\n",
            "        \"full_text\": [\n",
            "          \"Integrated proteogenomic characterization of human high grade serous ovarian cancer\\n\\nSUMMARY\",\n",
            "          \"To provide a detailed analysis of the molecular components and underlying mechanisms associated with ovarian cancer, we performed a comprehensive mass spectrometry-based proteomic characterization of 174 ovarian tumors previously analyzed by The Cancer\",\n",
            "          \"of 174 ovarian tumors previously analyzed by The Cancer Genome Atlas (TCGA), of which 169 were high-grade serous carcinomas (HGSC). Integrating our proteomic measurements with the genomic data yielded a number of insights into disease such as how\",\n",
            "          \"data yielded a number of insights into disease such as how different copy number alternations influence the proteome, the proteins associated with chromosomal instability, the sets of signaling pathways that diverse genome rearrangements converge on, as\",\n",
            "          \"pathways that diverse genome rearrangements converge on, as well as the ones most associated with short overall survival. Specific protein acetylations associated with homologous recombination deficiency suggest a potential means for stratifying patients\",\n",
            "          \"deficiency suggest a potential means for stratifying patients for therapy. In addition to providing a valuable resource, these findings provide a view of how the somatic genome drives the cancer proteome and associations between protein and\",\n",
            "          \"drives the cancer proteome and associations between protein and post-translational modification levels and clinical outcomes in HGSC.\",\n",
            "          \"INTRODUCTION\",\n",
            "          \"A comprehensive molecular view of cancer is necessary for understanding the underlying mechanisms of disease, improving prognosis, and ultimately guiding treatment. The Cancer Genome Atlas (TCGA) conducted an extensive genomic and transcriptomic\",\n",
            "          \"Atlas (TCGA) conducted an extensive genomic and transcriptomic characterization of ovarian high-grade serous carcinoma (HGSC) aimed at defining the genomic landscape and aiding the development of targeted therapies for this highly lethal malignancy. Key\",\n",
            "          \"of targeted therapies for this highly lethal malignancy. Key findings from TCGA were: 1) the prevalent role of TP53 mutations, 2) extensive DNA copy alterations, 3) preliminary transcriptional signatures associated with survival, 4) varied mechanisms of\",\n",
            "          \"signatures associated with survival, 4) varied mechanisms of BRCA1/2 inactivation, and lastly, 5) CCNE1 aberrations. Subsequent analysis of genomic data from the TCGA consortium led to the refinement of the transcript-defined signatures, improving the\",\n",
            "          \"refinement of the transcript-defined signatures, improving the statistical association with patient outcome, and integrating microRNA and mRNA expression profiles associated with HGSC to identify candidate microRNA targets.\",\n",
            "          \"While the insights from genomic analyses are substantial, functions encoded in the genome are generally executed at the protein level, and are often further modulated by post-translational modifications (PTMs). For example, TCGA used reverse phase protein\",\n",
            "          \"(PTMs). For example, TCGA used reverse phase protein array (RPPA) analysis of 172 proteins (including 31 phosphoproteins with phospho-specific antibodies) to generate a signature associated with the risk of tumor recurrence. To obtain a more comprehensive\",\n",
            "          \"the risk of tumor recurrence. To obtain a more comprehensive assessment of the complex ovarian HGSC phenotype, the Clinical Proteomic Tumor Analysis Consortium (CPTAC) conducted an extensive mass spectrometry (MS)-based proteomic and phosphoproteomic\",\n",
            "          \"mass spectrometry (MS)-based proteomic and phosphoproteomic characterization of HGSC tumors characterized by TCGA, providing quantitative measurements for a combined total of 9,600 proteins from 174 tumors (an average of 7,952 proteins per tumor), and a\",\n",
            "          \"from 174 tumors (an average of 7,952 proteins per tumor), and a total of 24,429 phosphosites from 6,769 phosphoproteins in a subset of 69 tumors (an average of 7,677 phosphosites per tumor). Our results provide insights on HGSC biology and correlate\",\n",
            "          \"Our results provide insights on HGSC biology and correlate differences in protein and PTM levels with clinical outcome complementary to TCGA genomic analyses.\",\n",
            "          \"RESULTS\\n\\nProteomic analysis of TCGA HGSC samples\",\n",
            "          \"HGSC biospecimens and clinical data from 174 patients collected by TCGA were analyzed at two independent CPTAC Centers, Johns Hopkins University (JHU) and Pacific Northwest National Laboratory (PNNL); thirty-two samples were analyzed at both JHU and PNNL.\",\n",
            "          \"(PNNL); thirty-two samples were analyzed at both JHU and PNNL. Tumors were selected by examining the associated TCGA metadata to select tumors either 1) on the basis of putative homologous recombination deficiency (HRD), defined by the presence of\",\n",
            "          \"recombination deficiency (HRD), defined by the presence of germline or somatic BRCA1 or BRCA2 mutations, BRCA1 promoter methylation, or homozygous deletion of PTEN (122 samples; 67 classified as HRD, 55 as non-HRD by the above criteria; JHU), or 2) to\",\n",
            "          \"as HRD, 55 as non-HRD by the above criteria; JHU), or 2) to maximize differences in overall survival (84 samples; PNNL). For selection purposes, short survival was defined as overall survival of less than 3 years, and long survival as greater than 5\",\n",
            "          \"of less than 3 years, and long survival as greater than 5 years. All but five tumors had somatic TP53 mutations, a characteristic feature of HGSC; these five tumors were subsequently reclassified as other than HGSC, and removed from protein functional\",\n",
            "          \"as other than HGSC, and removed from protein functional analyses (e.g., subtyping and survival analyses). The tumor selection criteria and the associated metadata are provided in the Supplemental Information Table S1, available online.\",\n",
            "          \"Proteomics measurements used isobaric tags for relative and absolute quantitation (iTRAQ) in conjunction with offline liquid chromatography fractionation via high-pH reversed phase liquid chromatography (RPLC) and online RPLC with high-resolution tandem\",\n",
            "          \"(RPLC) and online RPLC with high-resolution tandem MS to provide broad coverage for peptide and protein identification and quantification (Supplemental Information); this also alleviated quantitative interference potentially associated with the use of\",\n",
            "          \"interference potentially associated with the use of isobaric tags. We used the relative abundance measurements for each protein in the 32 patient samples analyzed at both JHU and PNNL to normalize across the two analysis sites, and then used clustering,\",\n",
            "          \"across the two analysis sites, and then used clustering, principle component analysis (PCA), and statistical tests to identify any significant batch effects associated with the site of analysis (a detailed comparison of within site, between site, between\",\n",
            "          \"(a detailed comparison of within site, between site, between sample measurement variability and the process used to merge the JHU and PNNL data is given in Supplemental Information, Figure S1). As shown in Figure S1C, the median coefficient of variation\",\n",
            "          \"As shown in Figure S1C, the median coefficient of variation (CV) between measurements at the two sites was 16%.\",\n",
            "          \"A total of 9,600 proteins were identified with high confidence in all tumors, and the relative abundances in each tumor are given in Table S2. Functional analyses and proteome-transcriptome associations were restricted to 3,586 proteins observed and\",\n",
            "          \"associations were restricted to 3,586 proteins observed and quantified in all 169 HGSC samples used for protein functional analyses and where sample variability (signal) exceeded technical variability (noise) in the merged data (Table S2), calculated as\",\n",
            "          \"(noise) in the merged data (Table S2), calculated as described in Supplemental Information. On average we identify peptides covering 29% of the amino acids in any of these 3,586 proteins in a given sample, with a range from 10% to 47%. In addition to\",\n",
            "          \"in a given sample, with a range from 10% to 47%. In addition to protein abundance levels, phosphoproteomics data were acquired for 69 tumors with sufficient sample (Table S2). As with proteins, phosphopeptide abundances were calculated relative to the\",\n",
            "          \"phosphopeptide abundances were calculated relative to the pooled reference sample. Because isobaric labeling was performed prior to splitting the samples for proteome and phosphoproteome analyses, phosphopeptide abundance could be corrected for changes in\",\n",
            "          \"phosphopeptide abundance could be corrected for changes in parent protein abundance to identify differences in the relative extent of phosphorylation at specific sites for each protein (Table S2). Overall, we achieved a protein dynamic range encompassing\",\n",
            "          \"S2). Overall, we achieved a protein dynamic range encompassing >4 orders of magnitude, ranging from low-level transcription factors to abundant structural proteins i.e., actin and tubulin.\",\n",
            "          \"Proteogenomic landscape of HGSC\",\n",
            "          \"The degree to which alterations observed at the genome and transcriptome levels are manifested at the protein level is variable, both qualitatively and quantitatively, and partially driven by multiple levels of post-transcriptional regulation. To identify\",\n",
            "          \"multiple levels of post-transcriptional regulation. To identify peptides encoded by single amino acid variants (SAAVs), splice variants, and novel exons documented by TCGA, mass spectra were searched against a custom graph database containing all peptide\",\n",
            "          \"searched against a custom graph database containing all peptide variations projected from the TCGA genomic analyses of the cohort using a multi-stage analysis pipeline (Supplemental Information). The most frequently observed variant peptides represented\",\n",
            "          \"The most frequently observed variant peptides represented SAAVs and gene-level events. The evidence supporting each novel event is detailed in Table S2, including event type, genomic location, abundance information, and estimated false discovery rate\",\n",
            "          \"abundance information, and estimated false discovery rate (FDR). The validity of these variant peptides was further evaluated by the synthesis of twenty examples selected at random across the entire range of spectrum match scores. We obtained tandem mass\",\n",
            "          \"entire range of spectrum match scores. We obtained tandem mass spectra for all twenty synthetic peptides that matched spectra from our analyses, strongly supporting our observation of these variant peptides; three representative examples are given in\",\n",
            "          \"variant peptides; three representative examples are given in Figure S1E. These results demonstrate the ability to confidently detect, identify and validate genome-level predictions at the protein level. More novel peptides would likely be observed if\",\n",
            "          \"protein level. More novel peptides would likely be observed if there was sufficient sample for more extensive fractionation and/or replicate analyses; these limitations preclude any conclusions about the biological significance of unobserved events. For\",\n",
            "          \"about the biological significance of unobserved events. For example, only 2 mutant p53 peptides were identified despite the presence of p53 mutations in all tumors examined. Such low coverage can occur for reasons that include: excessively large or small\",\n",
            "          \"can occur for reasons that include: excessively large or small tryptic fragments, inability to distinguish some amino acids, some possible biases against highly hydrophobic and hydrophilic peptides, or low abundance peptides co-eluting with very high\",\n",
            "          \"peptides, or low abundance peptides co-eluting with very high abundance peptides.\",\n",
            "          \"To assess the potential for post-transcriptional regulation (e.g., translational efficiency or protein degradation), we compared each protein to its corresponding transcript across all tumors, and correlation was assessed for those pairs with reliable\",\n",
            "          \"and correlation was assessed for those pairs with reliable measurements for both mRNA and protein, i.e., proteins with a corresponding mRNA measurement observed in all 169 HGSC tumors where sample variability exceeded technical variability (3,196 pairs).\",\n",
            "          \"variability exceeded technical variability (3,196 pairs). We excluded 391 proteins observed without a corresponding mRNA (e.g., HBA1) or discordant gene symbol annotation in the protein database (e.g., THOC4). Overall, 79.4% of the mRNA:protein pairs\",\n",
            "          \"(e.g., THOC4). Overall, 79.4% of the mRNA:protein pairs showed statistically significant positive Spearman correlations (Benjamini-Hochberg adjusted p value <0.01; Figure 1) when changes in mRNA abundance were compared to changes in relative protein\",\n",
            "          \"in mRNA abundance were compared to changes in relative protein abundance. The observed median r value of 0.45 for each mRNA-protein pair across all 169 tumors is similar to observations in colorectal cancer, breast cancer and mouse tissues, although less\",\n",
            "          \"cancer, breast cancer and mouse tissues, although less than found for cell lines. In comparison, the median correlation of paired protein measurements from the same sample, but measured at two sites, was substantially higher, at 0.69 (Figure S1).\",\n",
            "          \"A wide range of mRNA:protein correlations was observed. In general, weaker correlations were observed for highly stable and abundant proteins associated with housekeeping or non-intrinsic functions (e.g., ribosomes, mRNA splicing, oxidative\",\n",
            "          \"functions (e.g., ribosomes, mRNA splicing, oxidative phosphorylation, complement cascade), while more dynamic proteins known to be transcriptionally regulated in response to nutrient demand or other perturbations (e.g., nucleotide metabolism, amino acid\",\n",
            "          \"or other perturbations (e.g., nucleotide metabolism, amino acid metabolism, acute inflammatory responses) were more highly correlated (Figure 1 and Table S3). This result is consistent with a previous colorectal cancer study and supports the hypothesis\",\n",
            "          \"a previous colorectal cancer study and supports the hypothesis that while many biological functions are primarily regulated by mRNA abundance, post-transcriptional mechanisms likely have an important role in regulating certain house-keeping functions.\",\n",
            "          \"important role in regulating certain house-keeping functions. Additionally, we found that 23 mRNAs lacking polyA tails displayed lower correlation (mean \\u22120.15) with their cognate proteins than did polyadenylated mRNAs, consistent with the decreased\",\n",
            "          \"than did polyadenylated mRNAs, consistent with the decreased stability of mRNAs lacking polyA tails.\",\n",
            "          \"Clustering of tumors based on protein abundance\",\n",
            "          \"HGSC is the most common of the four major histological subtypes of epithelial ovarian cancer, and is characterized by distinct morphological features. Recent studies using mRNA abundance data have suggested four transcriptomic HGSC subtypes designated as\",\n",
            "          \"have suggested four transcriptomic HGSC subtypes designated as differentiated, immunoreactive, mesenchymal, and proliferative. To build an unbiased molecular taxonomy of ovarian HGSC, we used protein abundance data from the 169 tumors to identify subtypes\",\n",
            "          \"protein abundance data from the 169 tumors to identify subtypes that might show biological differences that could be exploited in future studies. Figure 2 shows the resulting clustering analysis of individual tumors (vertical columns) by protein abundance\",\n",
            "          \"of individual tumors (vertical columns) by protein abundance (horizontal rows). The cluster assignment for each sample is provided in Table S4, and a consensus value matrix for the subtype comparisons is shown in Figure S2A. The results of a Weighted Gene\",\n",
            "          \"is shown in Figure S2A. The results of a Weighted Gene Co-expression Network Analysis (WGCNA) of protein functional enrichment by subgroups are provided for the protein clusters, in Figure S2B and Table S4. The enrichment of KEGG and Reactome ontologies\",\n",
            "          \"and Table S4. The enrichment of KEGG and Reactome ontologies in the WGCNA-derived modules are shown in Figures S2C and S2D, respectively; membership of enriched pathways is provided in Table S3.\",\n",
            "          \"Four of the proteomic clusters showed a clear correspondence to the mesenchymal, proliferative, immunoreactive, and differentiated subtypes defined by the TCGA transcriptome analysis (Figure 2 and Figure S2E). A relatively small fifth cluster of tumors\",\n",
            "          \"2 and Figure S2E). A relatively small fifth cluster of tumors significantly enriched in proteins associated with extracellular matrix interactions, erythrocyte and platelet functions, and the complement cascade was also observed using multiple approaches,\",\n",
            "          \"complement cascade was also observed using multiple approaches, including model based clustering with Bayesian information criteria, consensus clustering, and VISDA-based sub-phenotype clustering. This new group could not be attributed to tissue source\",\n",
            "          \"This new group could not be attributed to tissue source site sampling bias or any other meta-data category, but may be related to tumor characteristics, including vascularization and tumor content, as the Tumor Purity score for this subtype (and the\",\n",
            "          \"content, as the Tumor Purity score for this subtype (and the Mesenchymal subtype) was significantly lower than that of the other three subtypes (Figure S2F). The clinical relevance of these protein-based clusters will require validation in independent\",\n",
            "          \"protein-based clusters will require validation in independent HGSC sample sets, particularly as no significant difference in survival was observed (Figure S2G), similar to mRNA-based clustering analysis.\",\n",
            "          \"Proteomic analysis of CNA effects\",\n",
            "          \"Chromosomal instability, marked by extensive copy number alterations (CNAs) in each tumor, is a hallmark of HGSC, and a likely source of driver alterations in this disease. CNAs can affect the abundance of proteins at the same locus (cis effects), and may\",\n",
            "          \"abundance of proteins at the same locus (cis effects), and may also act in trans, either directly or indirectly.\",\n",
            "          \"Hypothesizing that CNAs with strong trans effects are more likely to elicit a molecular phenotype and confer selective advantages, we sought to identify those CNA regions that have the broadest effect on protein expression. In all, 29,393 CNA loci (Table\",\n",
            "          \"effect on protein expression. In all, 29,393 CNA loci (Table S5) were compared to our global proteomics data with 950,209 CNA/protein pairs (0.72% of the total) exhibiting significant association (Benjamini-Hochberg adjusted p value <0.01). We provide a\",\n",
            "          \"(Benjamini-Hochberg adjusted p value <0.01). We provide a complete list of the significantly associated proteins for each CNA in Table S5. The diagonal line evident in Figure 3 corresponds to cis effects, and vertical stripes correspond to trans effects\",\n",
            "          \"cis effects, and vertical stripes correspond to trans effects where changes in copy number affect expression of numerous proteins across the genome. We performed the same analysis for mRNA to identify sites where associations are transcriptionally\",\n",
            "          \"mRNA to identify sites where associations are transcriptionally mediated. A similar number of CNA/mRNA pairs were found to be significantly associated (1,113,164 at Benjamini-Hochberg adjusted p value <0.01; Figure 3). This analysis revealed regions on\",\n",
            "          \"p value <0.01; Figure 3). This analysis revealed regions on chromosomes 2, 7, 20, and 22 correlated in trans with more than 200 proteins. In contrast to colorectal cancer where most of the trans-regulation of protein by CNA was accompanied by similar\",\n",
            "          \"trans-regulation of protein by CNA was accompanied by similar changes in mRNA, we observed several loci associated with differences in protein abundance without a corresponding change in mRNA. For example, large regions on chromosome 2 have relatively\",\n",
            "          \"For example, large regions on chromosome 2 have relatively little trans effect on mRNA levels, but are associated with more than 200 proteins in trans. Dissecting the mechanisms by which a specific CNA can alter protein levels in trans, without affecting\",\n",
            "          \"CNA can alter protein levels in trans, without affecting the corresponding mRNA is difficult, given the extent of the amplified or deleted regions and the numerous genes affected at a given locus. Possible mechanisms include cis regulation of proteins\",\n",
            "          \"locus. Possible mechanisms include cis regulation of proteins associated with mRNA stability and translational efficiency, such as microRNAs and RNA binding proteins.\",\n",
            "          \"Given the complex pattern of CNAs observed in HGSC, it has been difficult to identify a limited number of high impact genomic alterations that could function as drivers of the disease. We interrogated the trans-affected proteins associated with each\",\n",
            "          \"interrogated the trans-affected proteins associated with each putative CNA for common functions using pathways defined by the KEGG, NCI pathway interaction database (PID) and Reactome databases (Supplemental Information). Proteins associated with cell\",\n",
            "          \"(Supplemental Information). Proteins associated with cell invasion and migration, and proteins related to immune function appeared to be enriched in association with multiple CNAs (Figure S3; Table S3). These observations suggest a convergence of multiple\",\n",
            "          \"Table S3). These observations suggest a convergence of multiple CNA targets on a common set of biological functions, namely motility/invasion and immune regulation, functions that are among the hallmarks of cancer.\",\n",
            "          \"Association of CNA trans-affected proteins with overall survival\",\n",
            "          \"Availability of the TCGA survival data allowed us to use trans-affected protein data from the most influential CNAs (e.g., the four altered regions described on chromosomes 2, 7, 20, and 22; Figure 3) to build a model of overall survival. Because each CNA\",\n",
            "          \"3) to build a model of overall survival. Because each CNA affects many proteins, we used a regression approach that identifies parsimonious Cox proportional hazards models with maximal predictive ability from the list of significantly correlated proteins\",\n",
            "          \"ability from the list of significantly correlated proteins for that CNA. We trained models on the proteomics data from PNNL (82 tumors) and tested the ability of the model to predict survival times in the data from JHU (87 tumors, not including the 32\",\n",
            "          \"times in the data from JHU (87 tumors, not including the 32 overlapping). Each of the four most influential CNA regions produced models that were strongly associated with patient survival (p value <0.01, FDR <0.5% based on randomly selected proteins). A\",\n",
            "          \"value <0.01, FDR <0.5% based on randomly selected proteins). A Kaplan-Meier plot illustrating the predictive value of each of the locus-specific models after validation is shown in Figure S4A; for the Kaplan-Meier plots, \\u2018high\\u2019 and \\u2018low\\u2019 expression were\",\n",
            "          \"for the Kaplan-Meier plots, \\u2018high\\u2019 and \\u2018low\\u2019 expression were defined relative to the median for that signature across all samples, splitting patients into two equal groups of 45 (Table S6). We examined the overlap in predictions for patients for the four\",\n",
            "          \"examined the overlap in predictions for patients for the four signatures and found that the predictions were unanimous (either high or low signature) for 62% of patients and 16% were evenly split, with the remaining having one dissenting signature. This\",\n",
            "          \"split, with the remaining having one dissenting signature. This suggested the utility of combining these signatures, using a voting method where the number of high or low calls was counted for each patient; this substantially improved prediction of\",\n",
            "          \"for each patient; this substantially improved prediction of survival time (p value 1.9e-6; Figure 4). We also examined the effects of tumor stage, tumor grade, patient age, surgical outcome, and platinum status, and found that our proteomic signatures\",\n",
            "          \"and platinum status, and found that our proteomic signatures were not improved by inclusion of these variables.\",\n",
            "          \"For each of these locus-specific models we analyzed the enrichment of genes and their regulatory sequences associated with outcome, and found all four models had genes significantly enriched (Benjamini-Hochberg adjusted p value <0.05) in binding sites for\",\n",
            "          \"adjusted p value <0.05) in binding sites for the proliferation-associated serum response factor (SRF), suggesting that SRF activity may be important in ovarian cancer outcome. Additionally, there was significant enrichment of proteins involved in the\",\n",
            "          \"there was significant enrichment of proteins involved in the regulation of actin cytoskeleton, apoptosis, and adherens junction (Benjamini-Hochberg adjusted p value <0.05). We examined the protein abundance and phosphorylation status of SRF between short\",\n",
            "          \"abundance and phosphorylation status of SRF between short and long surviving groups and observed that they were higher in short surviving patients, but only slightly. Thus, although SRF alone was not predictive, the trends in SRF abundance and\",\n",
            "          \"SRF alone was not predictive, the trends in SRF abundance and phosphorylation are consistent with the observation of enrichment for proteins potentially regulated by SRF binding. Details of the larger gene set analysis are provided in Supplemental\",\n",
            "          \"of the larger gene set analysis are provided in Supplemental Information.\",\n",
            "          \"Several proteins shared across all the signatures are known to be involved in cancer processes. Catenin B2 (CTNNA2) is a cell-cell adhesion protein and tumor suppressor. The Rho GDP dissociation inhibitor (GDI) beta (ARHGDIB) is involved in invasion and\",\n",
            "          \"inhibitor (GDI) beta (ARHGDIB) is involved in invasion and migration in many cancers, and overexpression correlates with progression in pancreatic carcinoma. Protein kinase C and casein kinase substrate in neurons protein 2 (PACSIN2) is a repressor of\",\n",
            "          \"substrate in neurons protein 2 (PACSIN2) is a repressor of cellular migration. Finally, the GTP-binding nuclear protein Ran, is a prognostic marker associated with increased survival in epithelial ovarian cancer. The association of these previously\",\n",
            "          \"epithelial ovarian cancer. The association of these previously described survival-related proteins with genes affected in trans by CNAs suggests a potential mechanism for the parallel activation of multiple pathways associated with poor prognosis in HGSC.\",\n",
            "          \"As a comparison we applied the previously described Provar signature, comprising five protein and four phosphoproteins that showed good survival prediction in the TCGA ovarian cancer dataset. We observed all proteins in the Provar signature in our\",\n",
            "          \"We observed all proteins in the Provar signature in our proteomic data, but only one of the phosphosites (EGFR Y1173). Thus, we used the RPPA data from the original signature (Figure S4B). We found that the Provar signature was prognostic of survival\",\n",
            "          \"We found that the Provar signature was prognostic of survival (Benjamini-Hochberg adjusted p value = 0.11) in the 67 patients examined here (those with phosphoprotein data and had somatic TP53 mutations), but the statistical power of Provar is not as high\",\n",
            "          \"mutations), but the statistical power of Provar is not as high as the statistical power of signatures derived from the CNA trans-affected proteins in this dataset. In addition, integrating the Provar signature with the CNA signatures did not improve\",\n",
            "          \"the Provar signature with the CNA signatures did not improve survival prediction (Figure S4C).\",\n",
            "          \"Identification of proteins associated with chromosomal structural abnormality\",\n",
            "          \"The degree of chromosomal instability exhibited by a tumor can be represented by a calculated chromosome instability (CIN) index, as described previously for lung cancer. Identification of proteins associated with CIN may provide information on the\",\n",
            "          \"of proteins associated with CIN may provide information on the processes contributing to chromosomal instability, while the analysis of trans-affected proteins described above is more closely related to the downstream consequences of specific CNAs. Using\",\n",
            "          \"related to the downstream consequences of specific CNAs. Using a bootstrapping method described in Supplemental Information, we identified a proteomic signature including 128 proteins (Table S7) showing significant correlation with CIN index\",\n",
            "          \"(Table S7) showing significant correlation with CIN index (Benjamini-Hochberg adjusted p value <1e-4, Spearman correlation; Figure S5A). Functional annotation of these proteins (Figure S5B) showed that proteins up-regulated in tumors with high CIN index\",\n",
            "          \"showed that proteins up-regulated in tumors with high CIN index were preferentially involved in chromatin organization (p value 6.90e-5), whereas proteins up-regulated in tumors with low CIN index were more often associated with cell death (p value\",\n",
            "          \"CIN index were more often associated with cell death (p value 2.13e-5). Correlation analysis of protein abundances and CIN indicated that a small number of proteins could account for the majority of the CIN (Figure 5); two of the most strongly associated\",\n",
            "          \"of the CIN (Figure 5); two of the most strongly associated proteins, CHD4 and CHD5, are known to be involved in chromatin organization.\",\n",
            "          \"Identification of proteins associated with HRD status\",\n",
            "          \"Since HRD is associated with susceptibility to PARP inhibitors and improved survival, we sought to elucidate systemic changes associated with HRD and identify biomarkers that might be used to stratify patients for treatment. We defined tumors with HRD as\",\n",
            "          \"stratify patients for treatment. We defined tumors with HRD as having either germline or somatic mutations in BRCA1 or BRCA2, or BRCA1 promoter methylation, or homozygous deletion of PTEN, with an overall survival >1.5 years, while non-HRD patients were\",\n",
            "          \"an overall survival >1.5 years, while non-HRD patients were defined as lacking these genomic aberrations and with a follow-up or time-to-death <2.5 years; additional selection criteria include available residual tissue volume and a tumor tissue\",\n",
            "          \"include available residual tissue volume and a tumor tissue contamination score estimated using copy number alterations. Applying differential dependency network (DDN) analysis on a set of 171 BRCA1- or BRCA2-related proteins curated from the literature\",\n",
            "          \"BRCA1- or BRCA2-related proteins curated from the literature and from the cBio portal, we identified a sub-network of 30 proteins that displayed co-expression patterns differentiating HRD from non-HRD patients (Table S7; Figure 6). Several of the proteins\",\n",
            "          \"non-HRD patients (Table S7; Figure 6). Several of the proteins in these modules are known to be involved in histone acetylation or deacetylation, e.g., HDAC1, RBBP4, RBBP7, and EP300, and HUS1.\",\n",
            "          \"Although statistical association cannot distinguish between drivers and consequences of HRD status, the observed enrichment of proteins associated with histone acetylation motivated us to use an effective database search strategy (Supplemental\",\n",
            "          \"us to use an effective database search strategy (Supplemental Information) to identify and quantify acetylated peptides from the global proteomic data. Comparative analysis of 399 acetylated peptides identified 15 acetylated peptides with significant\",\n",
            "          \"peptides identified 15 acetylated peptides with significant differences between the HRD and non-HRD samples (Table S7). Among these, dual acetylation at K12 and K16 of histone H4 showed a significant difference between HRD and non-HRD samples (Figure 6);\",\n",
            "          \"difference between HRD and non-HRD samples (Figure 6); differential acetylation of K12 and K16 was verified using synthetic peptides and targeted analysis using Sequential Windowed data independent Acquisition of the Total High-resolution Mass Spectra\",\n",
            "          \"Acquisition of the Total High-resolution Mass Spectra (SWATH-MS). In cell line data, acetylation of H4 has previously been reported to be involved in the choice of DNA double strand break (DSB) repair pathways (homologous recombination or non-homologous\",\n",
            "          \"repair pathways (homologous recombination or non-homologous end joining. This relationship is regulated partially by HDAC1, a protein also identified in the DDN analysis. We observed a significant enrichment of HDAC1 and its co-regulated proteins in\",\n",
            "          \"enrichment of HDAC1 and its co-regulated proteins in tumors with HRD and low H4 acetylation relative to non-HRD tumors with high H4 acetylation (DDN analysis: permutation tests, p value <0.05; differentially acetylated peptides: t-tests, p value <0.05\",\n",
            "          \"differentially acetylated peptides: t-tests, p value <0.05 with an estimated FDR <0.5% by bootstrap/permutation tests). The combined observations of increased HDAC1 and associated proteins at the pathway level, together with decreased acetylation of H4 in\",\n",
            "          \"the pathway level, together with decreased acetylation of H4 in HRD patients at the PTM level, provide insight regarding the potential role of HDAC1 in modulating the choice of DSB repair pathways.\",\n",
            "          \"Phosphoproteomic analysis of pathways associated with survival\",\n",
            "          \"An initial set of 24,464 different phosphopeptides (21,298 unique phosphorylation sites) contained within 4,420 proteins having data on net differences in phosphorylation (Table S2) was obtained. Since ischemia of the TCGA tumor samples may alter\",\n",
            "          \"obtained. Since ischemia of the TCGA tumor samples may alter phosphopeptide abundance, we removed any phosphorylation sites previously shown to be altered by ischemia. We also removed three tumor samples having substantially higher than average missing\",\n",
            "          \"tumor samples having substantially higher than average missing values and two tumor samples lacking somatic TP53 mutations (Supplemental Information). This yielded a final set of 7,675 different phosphopeptides (6,802 unique phosphorylation sites) from\",\n",
            "          \"phosphopeptides (6,802 unique phosphorylation sites) from 2,324 proteins (Table S2) that were mapped to pathways via the NCI PID. The average net phosphorylation of all phosphopeptides mapping to a given pathway (i.e., the statistical average across all\",\n",
            "          \"to a given pathway (i.e., the statistical average across all phosphoproteins known to be in the pathway, rather than the statistically significant individual proteins) was used as a measure of pathway activity and compared between short survivors\",\n",
            "          \"of pathway activity and compared between short survivors (deceased surviving <3 years; N=17) and long survivors (patients surviving >5 years; N=19) using both proteomic and phosphoproteomic data.\",\n",
            "          \"An issue in ranking the pathways is the distinction between phosphopeptides that are unique to a single pathway, such as the receptor peptides themselves, vs. peptides that map to proteins shared between pathways. In order to address this issue, we\",\n",
            "          \"shared between pathways. In order to address this issue, we performed two analyses: one of all proteins associated with a specific pathway in the NCID database, and a second analysis in which proteins common to multiple pathways were excluded. Figure 7A\",\n",
            "          \"proteins common to multiple pathways were excluded. Figure 7A shows a ranking of activated pathways, as inferred from the analysis of pathway specific components (i.e., differences in mRNA, protein and phosphoprotein levels) in short vs. long survivors.\",\n",
            "          \"protein and phosphoprotein levels) in short vs. long survivors. Fifteen pathways showed increased phosphorylation at Benjamini-Hochberg adjusted p values <0.01, in contrast with transcriptional data for the same samples which yielded only one\",\n",
            "          \"data for the same samples which yielded only one statistically significantly increased pathway, androgen receptor signaling, which was also increased at the phosphoprotein level (Figure 7A and Supplemental Information). All pathways indicated in Figure 7A\",\n",
            "          \"Supplemental Information). All pathways indicated in Figure 7A were increased in association with short survival relative to long survival, though some individual components within a pathway showed opposite changes (e.g. JNK1 transcript and protein\",\n",
            "          \"showed opposite changes (e.g. JNK1 transcript and protein levels, see Figure 7B). The RhoA-regulatory, PDGFRB, and integrin-like kinase pathways emerged as the most activated in this analysis (Figure 7A). Figure 7B shows the common and unique elements of\",\n",
            "          \"(Figure 7A). Figure 7B shows the common and unique elements of the PDGFRB pathway, and the trends in mRNA, protein, and phosphoprotein levels in the comparison of short vs. long survivors, illustrating the benefit of combined protein and phosphoprotein\",\n",
            "          \"illustrating the benefit of combined protein and phosphoprotein measurements, as well as the substantial contribution of protein phosphorylation data to the overall analysis.\",\n",
            "          \"We compared our results to the 31 phosphoproteins represented on the RPPA arrays used in the original TCGA ovarian cancer analysis; RPPA identified differential phosphorylation of ERK1, RAF1, and STAT3 in the same patient samples analyzed by MS. For short\",\n",
            "          \"and STAT3 in the same patient samples analyzed by MS. For short surviving vs. long surviving patients, ERK1 showed statistically significantly increased phosphorylation by RPPA, on the same ERK1 phosphopeptides as found using MS, while AKT1, RAF1 and\",\n",
            "          \"ERK1 phosphopeptides as found using MS, while AKT1, RAF1 and STAT3 showed increased phosphorylation by both RPPA and MS-based analyses, but based upon distinct phosphopeptides (Table S2). Good overlap was observed between RPPA and MS results for\",\n",
            "          \"S2). Good overlap was observed between RPPA and MS results for phosphoproteins on the RPPA array, but we did not observe three of the specific phosphosites from the Provar survival signature in our MS data, suggesting that proteomics is more sensitive to\",\n",
            "          \"in our MS data, suggesting that proteomics is more sensitive to overall pathway activity than specific regulatory events in the pathway itself. However, many of the phosphopeptides identified by MS phosphoproteomics as components of the PDGFR-beta pathway\",\n",
            "          \"by MS phosphoproteomics as components of the PDGFR-beta pathway were not represented in the RPPA, illustrating the ability to identify phosphorylation events not readily accessible with currently available antibodies (Figure 7B).\",\n",
            "          \"DISCUSSION\",\n",
            "          \"The addition of proteomic information, including post-translational modifications, to the rich genomic and transcriptomic data available from the ovarian HGSC samples analyzed by TCGA has provided additional insights into the biology of HGSC, including\",\n",
            "          \"additional insights into the biology of HGSC, including identification of changes at the level of pathway activation. Integration of genomic, proteomic, and phosphoproteomic measurements identified differentially regulated pathways and functional modules\",\n",
            "          \"differentially regulated pathways and functional modules that displayed significant associations with patient outcomes, including survival and HRD status. Cox proportional hazard analysis of proteins associated with CNAs identified overall survival\",\n",
            "          \"of proteins associated with CNAs identified overall survival signatures enriched for targets of the proliferation-associated transcription factor SRF, and integrated proteomic, phosphoproteomic and transcriptomic data identified pathways that\",\n",
            "          \"and transcriptomic data identified pathways that differentiated patients on the basis of survival, including the PDGFR-beta signaling pathway associated with angiogenesis, the RhoA regulatory and integrin-linked kinase pathways associated with cell\",\n",
            "          \"and integrin-linked kinase pathways associated with cell mobility and invasion, and pathways associated with chemokine signaling and adaptive immunity. Proteins associated with cell invasion and motility emerged from both the integration of CNAs with\",\n",
            "          \"and motility emerged from both the integration of CNAs with protein abundance data and the phosphoproteomic investigation of up-regulated pathways in short versus long survivors. Despite the complexity of genomic alterations that characterize HGSC, these\",\n",
            "          \"complexity of genomic alterations that characterize HGSC, these analyses suggest a functional convergence on a subset of key pathways. The association of increased invasiveness and motility with short overall survival in this study may help to explain\",\n",
            "          \"with short overall survival in this study may help to explain more aggressive mechanisms of dissemination in ovarian cancer, including recently reported hematogenous metastasis, in addition to previously described lymphatic spread and known peritoneal\",\n",
            "          \"to previously described lymphatic spread and known peritoneal spread by direct extension.\",\n",
            "          \"Focusing on functional modules has also revealed potential drivers of HGSC and more robust signatures for potentially stratifying ovarian HGSC patients into distinct cancer phenotypes to inform therapeutic management. For example, the identification of\",\n",
            "          \"therapeutic management. For example, the identification of specific acetylation events associated with HRD, e.g., the simultaneous acetylation of K12 and K16 on histone H4, may provide an alternative biomarker of HRD and a rationale for the selection of\",\n",
            "          \"biomarker of HRD and a rationale for the selection of patients in future clinical trials of HDAC inhibitors alone or in combination with PARP inhibition. This may help to resolve the current discrepancy between the initial observation of limited single\",\n",
            "          \"discrepancy between the initial observation of limited single agent activity of HDAC inhibitors in ovarian cancer, and more recent findings of a >40% response rate when used in combination with cytotoxic chemotherapy in platinum-resistant patients.\",\n",
            "          \"with cytotoxic chemotherapy in platinum-resistant patients. Similarly, comprehensive interrogation of protein phosphorylation identified multiple pathways with significantly increased phosphorylation in patients with poor clinical outcomes. Specifically,\",\n",
            "          \"in patients with poor clinical outcomes. Specifically, the observed activation of PDGFR pathways in a subset of patients with short overall survival could potentially stratify patients for selective enrollment in trials of anti-angiogenic therapy, which\",\n",
            "          \"enrollment in trials of anti-angiogenic therapy, which is particularly relevant to current controversies over the use of bevacuzimab as a first-line therapy in ovarian cancer.\",\n",
            "          \"Overall, this work illustrates the ability of proteomics to complement genomics in providing additional insights into pathways and processes that drive ovarian cancer biology and how these pathways are altered in correspondence with clinical phenotypes.\",\n",
            "          \"are altered in correspondence with clinical phenotypes. The comprehensive proteomic measurements for the HGSC tumor samples provide a public resource of information. Importantly, the ability to identify PTMs revealed a strong association between specific\",\n",
            "          \"to identify PTMs revealed a strong association between specific histone acetylation events and the HRD phenotype. In addition, the enhanced view of pathway activity provided by measurements of protein phosphorylation provides a foundation for linking\",\n",
            "          \"of protein phosphorylation provides a foundation for linking genotype to proteotype, and ultimately to phenotype for understanding the molecular basis of cancer.\",\n",
            "          \"EXPERIMENTAL PROCEDURES\\n\\nFull methods are available in Extended Experimental Procedures.\\n\\nTumor Samples\",\n",
            "          \"Tumor Samples\\n\\nAll tumor samples for the current study were obtained through the TCGA Biospecimen Core Resource as described previously. Samples were selected based on overall survival, HRD status and availability.\\n\\nQuantitative Proteomics\",\n",
            "          \"Tissue proteins were extracted and digested with trypsin; at each site, a portion of each resulting sample was also used to create a pooled reference in which each tumor sample contributed an equal percentage by peptide mass. The pooled reference sample\",\n",
            "          \"equal percentage by peptide mass. The pooled reference sample was included in each multiplexed, isobaric labeling experiment to enable cross-experiment comparison in the entire sample cohort. The patient samples and the pooled reference sample were each\",\n",
            "          \"The patient samples and the pooled reference sample were each labeled by different iTRAQ reagents (Sciex), combined, fractionated, and split for integrated, quantitative global proteome (10%) and phosphoproteome (90%) analysis using LC-MS/MS on an\",\n",
            "          \"(10%) and phosphoproteome (90%) analysis using LC-MS/MS on an Orbitrap Velos mass spectrometer (Thermo Scientific). Raw data were processed for peptide identification by database searching and identified peptides were assembled as proteins and mapped to\",\n",
            "          \"identified peptides were assembled as proteins and mapped to gene identifiers for proteogenomic comparisons. Quantitation was achieved by comparing the iTRAQ reporter ion intensities in each sample.\",\n",
            "          \"Comparison of mRNA and Protein Subtypes\",\n",
            "          \"A model-based clustering approach was used to model the protein abundance data as a mixture of subtypes. Bayesian information criteria, statistical resampling and VISDA-based sub-phenotype clustering approaches provided similar results with a stable\",\n",
            "          \"clustering approaches provided similar results with a stable optimization on five clusters as determined by consensus clustering. WGCNA analysis was performed to infer genes or gene networks that drive subtyping into 5 clusters, followed by correlation\",\n",
            "          \"that drive subtyping into 5 clusters, followed by correlation with subtype as a trait.\",\n",
            "          \"Integration of Proteomics and CNA data\",\n",
            "          \"Spearman correlation coefficients and corresponding adjusted p values were calculated for each protein/transcript by CNA locus, and gene set enrichment analysis was used to infer function for groups of proteins significantly correlated with a given CNA\",\n",
            "          \"groups of proteins significantly correlated with a given CNA locus. Regression analysis was applied to the list of trans-affected protein correlated with each CNA to generate parsimonious Cox proportional hazards models with maximal predictive ability,\",\n",
            "          \"proportional hazards models with maximal predictive ability, using the PNNL data for training and JHU data for testing. Kaplan-Meier plots were used to visualize performance in predicting overall survival and progression-free survival. A CIN index was\",\n",
            "          \"overall survival and progression-free survival. A CIN index was calculated for each sample as the mean absolute values of copy number measurements at the 29,393 selected loci. Bootstrap resampling was used to select proteins correlated with CIN index at\",\n",
            "          \"was used to select proteins correlated with CIN index at high confidence.\",\n",
            "          \"Protein Acetylation and HRD\",\n",
            "          \"Acetylated peptides were identified by searching the global proteomics data for dynamic acetylation to lysines (+42 Da). Acetylation levels were compared between HRD and non-HRD cases by t-test. Targeted proteomics (SWATH) was used to orthogonally\",\n",
            "          \"by t-test. Targeted proteomics (SWATH) was used to orthogonally quantify the acetylated peptide with synthetic peptides as internal standard.\",\n",
            "          \"Survival-related Pathways Analysis\",\n",
            "          \"Phosphoproteome data from the short and long survivors were mapped to signaling pathways in the NCI pathway information database,(PID (http://pid.nci.nih.gov) using the gene names. For each signaling pathway in the PID, relative abundances for all\",\n",
            "          \"each signaling pathway in the PID, relative abundances for all phosphopeptides mapping to any pathway component were identified and separated into short and long survivor groups. The difference in distributions between the two sets of pathway-specific\",\n",
            "          \"in distributions between the two sets of pathway-specific peptides, i.e., those associated with either short survivors or long long survivors, was then assessed using a two-tailed t test. Similar enrichment analyses were also performed using protein\",\n",
            "          \"Similar enrichment analyses were also performed using protein abundance, mRNA abundance, and CNA.\",\n",
            "          \"Data Repository\",\n",
            "          \"All of the primary mass spectrometry data on TCGA tumor samples are deposited at the CPTAC Data Coordinating Center as raw and mzML files and complete protein assembly data sets for public access (https://cptac-data-portal.georgetown.edu).\",\n",
            "          \"Supplementary Material\",\n",
            "          \"This is a PDF file of an unedited manuscript that has been accepted for publication. As a service to our customers we are providing this early version of the manuscript. The manuscript will undergo copyediting, typesetting, and review of the resulting\",\n",
            "          \"undergo copyediting, typesetting, and review of the resulting proof before it is published in its final form. Please note that during the production process errors may be discovered which could affect the content, and all legal disclaimers that apply to\",\n",
            "          \"affect the content, and all legal disclaimers that apply to the journal pertain.\",\n",
            "          \"Author Contributions\",\n",
            "          \"Study conception and design: HZ, TL, ZZ, RDS, DWC, KDR. Investigation: performed experiment or data collection: TL, SS, FY, JZ, LijunC, PS, PA, YT, MAG, TRC, CC, ST, SN, RJM, HengZ. Computation and statistical analysis: ZZ, SHP, BaiZ, JEM, VAP, LiC, DR,\",\n",
            "          \"and statistical analysis: ZZ, SHP, BaiZ, JEM, VAP, LiC, DR, MEM, SWC, SW, JW, DLT, DF, VB, YW, BingZ. Data interpretation and biological analysis: TL, HZ, SHP, ZZ, BaiZ, JEM, VAP, LiC, DR, SN, CW, IMS, AP, MPS, DAL, RDS, DWC, KDR. Writing/manuscript\",\n",
            "          \"SN, CW, IMS, AP, MPS, DAL, RDS, DWC, KDR. Writing/manuscript preparation and revision: HZ, TL, ZZ, SHP, JEM, DR, VAP, LeslieC, HR, IMS, AP, MPS, DAL, RDS, DWC, KDR. Supervision and administration: HR, ESB, TH, RCR, LS, RDS, DWC, KDR.\",\n",
            "          \"The authors declare no competing financial interests.\\n\\n           SUPPLEMENTAL INFORMATION         \\n\\nSupplemental information includes Extended Experimental Procedures, 5 figures, and 8 tables and can be found with this article online.\",\n",
            "          \"Correlations between mRNA and protein abundance in TCGA tumors\",\n",
            "          \"mRNA and protein were correlated across all the samples, resulting in 90.6% positive correlations, and 79.4% were significantly correlated (Benjamini-Hochberg adjusted p value <0.01), with a mean Spearman\\u2019s correlation of 0.38 and a median value of 0.45\",\n",
            "          \"mean Spearman\\u2019s correlation of 0.38 and a median value of 0.45 (top panel). Different biological pathways and processes showed significantly different levels of correlation (bottom panel). Metabolic pathways and the interferon response displayed high\",\n",
            "          \"Metabolic pathways and the interferon response displayed high mRNA-protein correlation, and ribosome, mRNA splicing, oxidative phosphorylation, and complement and coagulation cascade were poorly correlated. The mean correlation is shown in parentheses\",\n",
            "          \"poorly correlated. The mean correlation is shown in parentheses followed by Benjamini-Hochberg adjusted p values calculated using a Kolmogorov\\u2013Smirnov test following the functional group names from MSIGDB. Blue bars indicate positive correlations and\",\n",
            "          \"names from MSIGDB. Blue bars indicate positive correlations and yellow indicates negative correlations; individual proteins (represented as bars on the x-axis) are sorted by correlation from low to high (bottom panel).\",\n",
            "          \"Proteomic subtypes and corresponding driving protein modules\",\n",
            "          \"Global proteomics abundance is shown in a heatmap with TCGA samples represented by columns ordered by protein subtype where rows represent proteins. Color of each cell indicates z-score (log2 of relative abundance scaled by proteins\\u2019 standard deviation)\",\n",
            "          \"of relative abundance scaled by proteins\\u2019 standard deviation) of the protein in that sample; red is increased and blue is decreased (relative to the pooled reference). Transcriptome-based subtypes (Verhaak et al., 2013) and the proposed proteomic subtypes\",\n",
            "          \"(Verhaak et al., 2013) and the proposed proteomic subtypes are indicated in color above the heatmap. WGCNA-derived modules are delineated with the row color panel and annotated according to the pathways based on enrichment of KEGG and Reactome ontologies.\",\n",
            "          \"Functional impact of copy number alterations\",\n",
            "          \"The top panel shows the correlation of CNAs to protein abundance (right) or mRNA (left) with significant positive correlations in red and negative correlations in blue (Benjamini-Hochberg adjusted p value <0.01, Spearman\\u2019s correlation coefficient). The\",\n",
            "          \"p value <0.01, Spearman\\u2019s correlation coefficient). The x-axis plots the 29,393 CNAs obtained from TCGA. The y-axis plots 3,202 proteins. Genes are ordered by chromosomal location on both the x- and y-axes. The bottom panel shows the summed number of\",\n",
            "          \"the x- and y-axes. The bottom panel shows the summed number of significantly correlated proteins (or mRNA) for each individual CNA. In blue is shown the total number, in black are those genes significantly correlated as both mRNA and protein. Where blue\",\n",
            "          \"significantly correlated as both mRNA and protein. Where blue and black lines have similar magnitude, e.g., the hotspot on chromosome 20, the CNA associations are shared between protein and mRNA. Where a strong blue line has no mirrored black line, e.g.,\",\n",
            "          \"Where a strong blue line has no mirrored black line, e.g., the protein hotspot on chromosome 2, the associations with CNA are largely unique.\",\n",
            "          \"Kaplan-Meier plot of overall survival stratified by CNA-derived signatures\",\n",
            "          \"A survival plot is shown for a consensus of the four best signatures (see Figure S4A and Table S6) identified from analysis of proteins affected in trans by CNAs. Models were trained using a lasso-based Cox proportional hazards model on the samples from\",\n",
            "          \"lasso-based Cox proportional hazards model on the samples from PNNL, and shown are the survival curves from these models applied to the non-overlapping data from the JHU analysis, with the up (red; above the median) and down (blue, below the median)\",\n",
            "          \"up (red; above the median) and down (blue, below the median) signatures each representing 45 patients. A vote was taken among the four most predictive signatures, and the results of this vote are shown. Probability of death is shown on the y-axis vs.\",\n",
            "          \"vote are shown. Probability of death is shown on the y-axis vs. survival time in years on the x-axis. Shaded ribbons denote 95% confidence intervals.\",\n",
            "          \"Number of CNAs statistically explained by proteins significantly associated with CIN index\",\n",
            "          \"A total of 128 proteins selected for their association with CIN were arranged along the chromosomal locations of their corresponding genes (y-axis). The length of the light grey horizontal lines indicates the number of CNAs significantly correlated with\",\n",
            "          \"indicates the number of CNAs significantly correlated with the protein across all chromosomes except for the protein\\u2019s corresponding gene coding region (x-axis). The top-ranked proteins are annotated and highlighted with dark lines to show the\",\n",
            "          \"are annotated and highlighted with dark lines to show the bootstrap-estimated 95% confidence intervals.\",\n",
            "          \"DDN analysis and lysine-acetylation analysis between HRD and non-HRD patients\",\n",
            "          \"DDN analysis revealed a sub-network of proteins that displayed distinct co-expression patterns between HRD and non-HRD patients, where purple connections indicate protein correlations that exist only in HRD samples and blue connections indicate protein\",\n",
            "          \"exist only in HRD samples and blue connections indicate protein correlations that exist only in non-HRD patients. The links between the nodes are drawn with two different thickness indicative of whether the connections meet the significance threshold of\",\n",
            "          \"of whether the connections meet the significance threshold of 0.05 (light lines) or 0.01 (bold lines). Proteins with blue dotted circles are known to be involved in histone acetylation or deacetylation. Further, identification and quantitation of\",\n",
            "          \"or deacetylation. Further, identification and quantitation of lysine-acetylated peptides in global proteomics data showed that acetylation level at K12 and K16 of histone H4 are significantly different between HRD and non-HRD samples, suggesting a role of\",\n",
            "          \"different between HRD and non-HRD samples, suggesting a role of histone H4 acetylation (together with HDAC1) in modulating the choice of DSB repair mechanisms.\",\n",
            "          \"Pathway analysis associated with patient survival\",\n",
            "          \"A) Pathway components were statistically analyzed using a two-sided t test between samples from patients surviving <3 years (short survival) and patients surviving >5 years (long survival) for differences in CNA, mRNA expression, protein or\",\n",
            "          \"survival) for differences in CNA, mRNA expression, protein or phosphorylation abundance. All significant pathways for mRNA and protein are shown (Benjamini-Hochberg adjusted p value <0.05) and the most significant pathways for phosphoproteomics are\",\n",
            "          \"and the most significant pathways for phosphoproteomics are plotted on the x axis as the \\u2212log of the p value. Results show that phosphorylation provides additional information about the functional state of tumors. B) Simplified PDGFR-beta pathway showing\",\n",
            "          \"state of tumors. B) Simplified PDGFR-beta pathway showing differences between short and long survivor groups for protein abundance (p), transcript abundance (m), and phosphopeptide abundance (circled P).\"\n",
            "        ]\n",
            "      }\n",
            "    ]\n",
            "  },\n",
            "  \"PDC000304\": {\n",
            "    \"study_id\": \"debd14db-1702-4bd2-b3cf-7349739f7c6b\",\n",
            "    \"pdc_study_id\": \"PDC000304\",\n",
            "    \"study_submitter_id\": \"Therapeutic Targets in Breast Cancer Xenografts -  Phosphoproteome\",\n",
            "    \"program_id\": \"10251935-5540-11e8-b664-00a098d917f8\",\n",
            "    \"project_id\": \"8dbcf809-3df2-4c57-a9d5-8d58d5824ffa\",\n",
            "    \"study_name\": \"Therapeutic Targets in Breast Cancer Xenografts -  Phosphoproteome\",\n",
            "    \"program_name\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"project_name\": \"CPTAC2-Other\",\n",
            "    \"disease_type\": \"Other\",\n",
            "    \"primary_site\": \"Not Reported\",\n",
            "    \"analytical_fraction\": \"Phosphoproteome\",\n",
            "    \"experiment_type\": \"iTRAQ4\",\n",
            "    \"embargo_date\": null,\n",
            "    \"cases_count\": 27,\n",
            "    \"aliquots_count\": 27,\n",
            "    \"protocol_id\": \"481c6411-677c-4748-8a6e-b569ab524861\",\n",
            "    \"protocol_submitter_id\": \"Breast Cancer Xenografts -  Phosphoproteome\",\n",
            "    \"program_submitter_id\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"protocol_name\": \"Breast Cancer Xenografts -  Phosphoproteome\",\n",
            "    \"protocol_date\": \"2021-05-06\",\n",
            "    \"document_name\": null,\n",
            "    \"quantitation_strategy\": \"Isobaric label quantitation analysis\",\n",
            "    \"label_free_quantitation\": \"N/A\",\n",
            "    \"labeled_quantitation\": \"iTRAQ\",\n",
            "    \"isobaric_labeling_reagent\": \"iTRAQ4\",\n",
            "    \"reporter_ion_ms_level\": \"MS2\",\n",
            "    \"starting_amount\": \"2.00\",\n",
            "    \"starting_amount_uom\": \"mg\",\n",
            "    \"digestion_reagent\": \"Lys-C,Trypsin\",\n",
            "    \"alkylation_reagent\": \"Iodoacetamide (IAA)\",\n",
            "    \"enrichment_strategy\": \"Phosphoproteome\",\n",
            "    \"enrichment\": \"80% acetonitrile/0.1% trifluoroacetic acid binding buffer and a 500?mM dibasic sodium phosphate, pH 7.0, (Sigma, S9763) elution buffer\",\n",
            "    \"chromatography_dimensions_count\": \"2\",\n",
            "    \"one_d_chromatography_type\": \"bRPLC\",\n",
            "    \"two_d_chromatography_type\": \"RPLC\",\n",
            "    \"fractions_analyzed_count\": \"12\",\n",
            "    \"column_type\": \"C18\",\n",
            "    \"amount_on_column\": null,\n",
            "    \"amount_on_column_uom\": null,\n",
            "    \"column_length\": \"20.0\",\n",
            "    \"column_length_uom\": \"cm\",\n",
            "    \"column_inner_diameter\": \"75.00\",\n",
            "    \"column_inner_diameter_uom\": \"\\u00b5m\",\n",
            "    \"particle_size\": \"1.9\",\n",
            "    \"particle_size_uom\": \"\\u00b5m\",\n",
            "    \"particle_type\": \"ReproSil-Pur C18\",\n",
            "    \"gradient_length\": \"110.0\",\n",
            "    \"gradient_length_uom\": \"min\",\n",
            "    \"instrument_make\": \"Thermo Fisher Scientific\",\n",
            "    \"instrument_model\": \"Q Exactive\",\n",
            "    \"dissociation_type\": \"HCD\",\n",
            "    \"ms1_resolution\": \"70000\",\n",
            "    \"ms2_resolution\": \"17500\",\n",
            "    \"dda_topn\": \"Top 12\",\n",
            "    \"normalized_collision_energy\": \"28\",\n",
            "    \"acquistion_type\": \"DDA\",\n",
            "    \"dia_multiplexing\": \"N/A\",\n",
            "    \"dia_ims\": \"N/A\",\n",
            "    \"analytical_technique\": null,\n",
            "    \"chromatography_instrument_make\": null,\n",
            "    \"chromatography_instrument_model\": null,\n",
            "    \"polarity\": null,\n",
            "    \"reconstitution_solvent\": null,\n",
            "    \"reconstitution_volume\": null,\n",
            "    \"reconstitution_volume_uom\": null,\n",
            "    \"internal_standards\": null,\n",
            "    \"extraction_method\": null,\n",
            "    \"ionization_mode\": null,\n",
            "    \"publications\": [\n",
            "      {\n",
            "        \"publication_id\": \"718f77fb-b329-11eb-9f63-0a2b20c3a9e9\",\n",
            "        \"pubmed_id\": \"28348404\",\n",
            "        \"doi\": \"10.1038/ncomms14864\",\n",
            "        \"author\": \"Huang, K. L.; Li, S.; Mertins, P.; Cao, S.; Gunawardena, H. P.; Ruggles, K. V.; Mani, D. R.; Clauser, K. R.; Tanioka, M.; Usary, J.; Kavuri, S. M.; Xie, L.; Yoon, C.; Qiao, J. W.; Wrobel, J.; Wyczalkowski, M. A.; Erdmann-Gilmore, P.; Snider, J. E.; Hoog, J.; Singh, P.; Niu, B.; Guo, Z.; Sun, S.Q.; Sanati, S.; Kawaler, E.; Wang, X.; Scott, A.; Ye, K.; McLellan, M.D.; Wendl, M.C.; Malovannaya, A.; Held, J.M.; Gillette, M.A.; Feny\\u221a\\u00c9\\u00ac\\u2202, D.; Kinsinger, C.R.; Mesri, M.; Rodriguez, H.; Davies, S.R.; Perou, C.M.; Ma, C.; Reid Townsend, R.; Chen, X.; Carr, S.A.; Ellis, M.J.; Ding L.\",\n",
            "        \"title\": \"Proteogenomic integration reveals therapeutic targets in breast cancer xenografts\",\n",
            "        \"journal\": \"Nature Communications\",\n",
            "        \"journal_url\": \"https://www.nature.com/articles/ncomms14864\",\n",
            "        \"year\": \"2017\",\n",
            "        \"abstract\": \"Recent advances in mass spectrometry (MS) have enabled extensive analysis of cancer proteomes. Here, we employed quantitative proteomics to profile protein expression across 24 breast cancer patient-derived xenograft (PDX) models. Integrated proteogenomic analysis shows positive correlation between expression measurements from transcriptomic and proteomic analyses; further, gene expression-based intrinsic subtypes are largely re-capitulated using non-stromal protein markers. Proteogenomic analysis also validates a number of predicted genomic targets in multiple receptor tyrosine kinases. However, several protein/phosphoprotein events such as overexpression of AKT proteins and ARAF, BRAF, HSP90AB1 phosphosites are not readily explainable by genomic analysis, suggesting that druggable translational and/or post-translational regulatory events may be uniquely diagnosed by MS. Drug treatment experiments targeting HER2 and components of the PI3K pathway supported proteogenomic response predictions in seven xenograft models. Our study demonstrates that MS-based proteomics can identify therapeutic targets and highlights the potential of PDX drug response evaluation to annotate MS-based pathway activities.\",\n",
            "        \"full_text\": [\n",
            "          \"Proteogenomic integration reveals therapeutic targets in breast cancer xenografts\",\n",
            "          \"Recent advances in mass spectrometry (MS) have enabled extensive analysis of cancer proteomes. Here, we employed quantitative proteomics to profile protein expression across 24 breast cancer patient-derived xenograft (PDX) models. Integrated proteogenomic\",\n",
            "          \"xenograft (PDX) models. Integrated proteogenomic analysis shows positive correlation between expression measurements from transcriptomic and proteomic analyses; further, gene expression-based intrinsic subtypes are largely re-capitulated using non-stromal\",\n",
            "          \"intrinsic subtypes are largely re-capitulated using non-stromal protein markers. Proteogenomic analysis also validates a number of predicted genomic targets in multiple receptor tyrosine kinases. However, several protein/phosphoprotein events such as\",\n",
            "          \"kinases. However, several protein/phosphoprotein events such as overexpression of AKT proteins and ARAF, BRAF, HSP90AB1 phosphosites are not readily explainable by genomic analysis, suggesting that druggable translational and/or post-translational\",\n",
            "          \"that druggable translational and/or post-translational regulatory events may be uniquely diagnosed by MS. Drug treatment experiments targeting HER2 and components of the PI3K pathway supported proteogenomic response predictions in seven xenograft models.\",\n",
            "          \"proteogenomic response predictions in seven xenograft models. Our study demonstrates that MS-based proteomics can identify therapeutic targets and highlights the potential of PDX drug response evaluation to annotate MS-based pathway activities.\",\n",
            "          \"Patient-derived xenografts recapitulate major genomic signatures and transcriptome profiles of their original tumours. Here, the authors, performing proteomic and phosphoproteomic analyses of 24 breast cancer PDX models, demonstrate that druggable\",\n",
            "          \"of 24 breast cancer PDX models, demonstrate that druggable candidates can be identified based on a comprehensive proteogenomic profiling.\",\n",
            "          \"Profiling of somatic alteration by next-generation DNA sequencing (NGS) has entered clinical practice with the promise of rapid diagnosis of druggable somatic genomic alterations for personalized cancer treatment. For example, recent analysis of 4,068\",\n",
            "          \"cancer treatment. For example, recent analysis of 4,068 samples from 16 cancer types suggested that repurposing approved drugs based on genomic alterations could provide individualized treatment options for around 40% of tumours. However, clinical\",\n",
            "          \"treatment options for around 40% of tumours. However, clinical evidence for this proposition is limited and has been slow to develop. Further, the signalling and biological effects of somatic mutations are not routinely determined in human tumour samples\",\n",
            "          \"mutations are not routinely determined in human tumour samples even though this is a consideration for rational drug design, response prediction and target prioritization. Finally, druggable genomic alterations are not detected in the majority of cases\",\n",
            "          \"genomic alterations are not detected in the majority of cases tested by NGS. Comprehensive proteomic analyses provide a potentially valuable approach to validate genomic findings as likely biological drivers and to discover discover opportunities for\",\n",
            "          \"biological drivers and to discover discover opportunities for targeted treatment.\",\n",
            "          \"Patient-derived xenografts (PDXs) in immunodeficient mice maintain the histological and molecular heterogeneity of the progenitor human tumour and cytotoxic drug responsiveness is often a transplantable phenotype. Our previous studies have shown that\",\n",
            "          \"transplantable phenotype. Our previous studies have shown that breast tumour PDXs recapitulate major genomic signatures and transcriptome profiles of their original breast tumours. Moreover, drug responses to endocrine therapy in breast cancer PDXs\",\n",
            "          \"drug responses to endocrine therapy in breast cancer PDXs resembled that observed in the corresponding patient and endocrine therapy resistance patterns were associated with aberrations in the ESR1 gene. While comprehensive proteomic characterization of\",\n",
            "          \"ESR1 gene. While comprehensive proteomic characterization of PDX is still lacking, recent studies using reverse phase protein array have identified similar protein profiles between PDX and primary tumours. These studies collectively suggest that the PDX\",\n",
            "          \"tumours. These studies collectively suggest that the PDX approach is a potentially valuable preclinical model for identification and testing of therapeutic targets.\",\n",
            "          \"Recent advances in mass-spectrometry provide an unprecedented opportunity for antibody-independent proteome profiling with approximately 80% of all proteins in major human tissues quantifiable by this technique. Analyses of the global proteome in tandem\",\n",
            "          \"by this technique. Analyses of the global proteome in tandem with genomic profiles generated through TCGA have yielded key insights on the molecular etiology of colorectal cancer, breast cancer and ovarian cancer. Our recent proteogenomic characterization\",\n",
            "          \"and ovarian cancer. Our recent proteogenomic characterization of 105 human breast cancers linked 5q-loss to elevated EGFR expression and identified overexpression of phosphorylated kinases including CDK12, PAK1 and ARAF as potential biological drivers.\",\n",
            "          \"including CDK12, PAK1 and ARAF as potential biological drivers. However, current proteogenomic analyses of human tissues are limited by the fact that therapeutic hypotheses were not being addressed in these initial analyses. As patient treatments were\",\n",
            "          \"addressed in these initial analyses. As patient treatments were heterogeneous, links between treatment and proteogenomic findings are difficult to establish in banked tumour specimens. The PDX system therefore provides an early opportunity to determine\",\n",
            "          \"PDX system therefore provides an early opportunity to determine whether a proteogenomic approach can produce more robust therapeutic hypotheses and predictive biomarkers for individual patients than that achieved by genomic analysis alone.\",\n",
            "          \"In this study, we generated high-coverage proteome and phosphoproteome data for 24 PDX models representing different breast cancer subtypes using both isobaric mass-tag labelling and label-free proteomic methods for cross-validated identification of\",\n",
            "          \"proteomic methods for cross-validated identification of 12,794 proteins and 56,874 phosphorylation sites. After filtering for observation in at least 10 out of 24 samples 10,069 proteins and 36,609 phosphorylation sites with their relative abundances\",\n",
            "          \"and 36,609 phosphorylation sites with their relative abundances quantified across tumours, were used in subsequent analyses in this study. A substantial number of druggable protein/phosphoprotein events were uniquely identified. Further, similar\",\n",
            "          \"events were uniquely identified. Further, similar proteomic/phosphoproteomic signatures were observed in patient breast cancer samples indicating potential clinical relevance. In selected PDX models exhibiting proteomic and/or phosphoproteomic\",\n",
            "          \"PDX models exhibiting proteomic and/or phosphoproteomic up-regulation events of HER2 or the PI3K pathway, we validated the efficacy of targeted inhibitors in suppressing PDX growth. Taken together, these observations show that comprehensive proteogenomic\",\n",
            "          \"these observations show that comprehensive proteogenomic profiling has potential to identify new targets for individualized treatment approaches in cancer.\",\n",
            "          \"Results\\n\\nProteogenomic coverage of breast cancer xenografts\",\n",
            "          \"We selected 24 PDX models established from primary or metastatic breast tumours for comprehensive proteogenomic characterization (Fig. 1a). The human patient cohort was composed of 10 basal, 1 claudin-low (CLDN-low), 9 luminal B and 4 HER2-enriched\",\n",
            "          \"1 claudin-low (CLDN-low), 9 luminal B and 4 HER2-enriched (HER2-E) breast tumours based on PAM50 expression subtyping (Supplementary Data 1). We conducted DNA and RNA sequencing respectively for 23 PDX models and in one case Sanger DNA sequencing of\",\n",
            "          \"for 23 PDX models and in one case Sanger DNA sequencing of hotspot mutations. Isotope Tagging for Relative and Absolute Quantitation (iTRAQ 4-plex) was completed for all 24 PDXs for discovery and label-free quantification (LFQ) for 18 PDXs for validation\",\n",
            "          \"and label-free quantification (LFQ) for 18 PDXs for validation and confirmation (Supplementary Data 2). The comprehensive proteogenomic coverage allowed us to systematically assess the somatic mutation profile, copy-number variation (CNV), mRNA\",\n",
            "          \"the somatic mutation profile, copy-number variation (CNV), mRNA expression, protein expression, and phosphosite levels and localizations.\",\n",
            "          \"We examined somatic mutations in significantly mutated genes of breast cancer in 12 human tumours and their matching PDXs (Supplementary Data 2). Most key somatic mutations in these genes were preserved (Fig. 1b), validating the genomic fidelity of these\",\n",
            "          \"preserved (Fig. 1b), validating the genomic fidelity of these PDX models. However, recurrent breast cancer mutations were not detected in two PDX models (WHIM17 and WHIM46). While sequencing data were not available for the matched progenitor human\",\n",
            "          \"data were not available for the matched progenitor human tumours, the germline SNPs of human blood normal samples matched with the PDX tumours, validating their patients of origin. Follow-up histological and RNA-seq analyses suggested WHIM17 and WHIM46\",\n",
            "          \"histological and RNA-seq analyses suggested WHIM17 and WHIM46 are EBV-positive human lymphoproliferative cells arising in the NOD/SCID/\\u03b3 mouse strain (Methods, Supplementary Fig. 1).\",\n",
            "          \"We then compared the variant allele fractions (VAFs) of exonic somatic mutations in human tumours and derived xenografts when both are available. We found comparable or higher VAFs in xenografts, potentially due to higher tumour purity and selection of\",\n",
            "          \"potentially due to higher tumour purity and selection of some mutant alleles by loss of heterozygosity in PDX models (Fig. 1c). The relatively high, positive VAF correlation (R=0.66) implied similarity between human samples and their respective\",\n",
            "          \"implied similarity between human samples and their respective xenografts, consistent with our previous report. Importantly, all PIK3CA and TP53 somatic mutations in the 12 originating human tumours were detected in respective xenografts with comparable or\",\n",
            "          \"were detected in respective xenografts with comparable or increased VAFs (Fig. 1c). Among the 22 breast cancer PDX models, 14, including 7 of the 8 basal tumours, harboured TP53 mutations, 5 luminal B PDXs had ESR1 mutations, and 5 luminal B and 1\",\n",
            "          \"5 luminal B PDXs had ESR1 mutations, and 5 luminal B and 1 CLDN-low xenografts carried PIK3CA mutations.\",\n",
            "          \"We quantified mRNA expression of 16,209 unique human genes from RNA-seq data for these 24 models. We also applied two distinct MS data acquisition approaches (Supplementary Fig. 2a, Methods), iTRAQ (Supplementary Data 3) and LFQ (Supplementary Data 4), to\",\n",
            "          \"iTRAQ (Supplementary Data 3) and LFQ (Supplementary Data 4), to quantify the expression ratios of 12,794 human proteins (11,879 genes) and 8,648 proteins (8,035 genes), respectively. A total of 56,874 phosphosites were also confidently identified using\",\n",
            "          \"of 56,874 phosphosites were also confidently identified using iTRAQ (Supplementary Data 5). After filtering for observation in at least 10 out of 24 samples (4 out of 9 of the iTRAQ experiments, Supplementary Fig. 3), the relative abundances of 10,069\",\n",
            "          \"Supplementary Fig. 3), the relative abundances of 10,069 proteins and 36,609 phosphorylation sites were quantified across tumours by iTRAQ and used in subsequent analyses in this study. The technical replicates in the LFQ (WHIM2 and WHIM16) and the iTRAQ\",\n",
            "          \"replicates in the LFQ (WHIM2 and WHIM16) and the iTRAQ (WHIM13) sets showed high correlation in protein expression levels (R>0.85, Supplementary Fig. 2b,c,e). Further, phosphosite expressions also showed high correlations in technical replicates (WHIM13)\",\n",
            "          \"also showed high correlations in technical replicates (WHIM13) of the iTRAQ experiment (R=0.82, Supplementary Fig. 2d), validating the technical reproducibility of our proteomic and phosphoproteomic datasets. Of note, while iTRAQ and LFQ quantification\",\n",
            "          \"datasets. Of note, while iTRAQ and LFQ quantification were conducted separately and based on different features of the LC\\u2013MS data collected from the Q Exactive mass spectrometer (that is, reporter ion intensity and MS1 peak area, respectively), our\",\n",
            "          \"reporter ion intensity and MS1 peak area, respectively), our analysis showed reasonable correlation between the two measurements after normalization (R=0.61). The two datasets utilizing the same PDX models but different workflows enabled cross-method\",\n",
            "          \"same PDX models but different workflows enabled cross-method validation of global proteomic results.\",\n",
            "          \"Proteogenomic integration and comparison\",\n",
            "          \"We first evaluated the correlation between mRNA expression and protein abundance measurements from the iTRAQ experiment (Fig. 2a) for these 24 PDX models: 83.6% of the genes with sufficient data showed positive correlations with a median Pearson R=0.536\",\n",
            "          \"data showed positive correlations with a median Pearson R=0.536 (Supplementary Fig. 4). We investigated whether the trend of mRNA-protein correlations were associated with specific KEGG pathways, finding that metabolic pathways involved in house-keeping\",\n",
            "          \"finding that metabolic pathways involved in house-keeping functions are enriched for genes showing high, positive correlation. For example, genes in the glutathione metabolism pathway showed the highest enrichment for positive correlations\",\n",
            "          \"pathway showed the highest enrichment for positive correlations (Kolmogorov\\u2013Smirnov test, false discovery rate (FDR)=5.6e-07). Interestingly, we observed that genes in the ribosome (FDR<2.2e-16), spliceosome (FDR=2.0e-13), RNA transport (FDR=1.7e-5), RNA\",\n",
            "          \"spliceosome (FDR=2.0e-13), RNA transport (FDR=1.7e-5), RNA polymerase (FDR=1.8e-4), and oxidative phosphorylation (FDR<2.2e-16) pathways showed relatively lower correlation between mRNA and protein level. These pathways were enriched for genes that do not\",\n",
            "          \"level. These pathways were enriched for genes that do not require translated proteins for their biological functions. Similar pathway-specific pattern of positive and negative enrichment of correlations were observed by LFQ (Supplementary Fig. 5). The\",\n",
            "          \"correlations were observed by LFQ (Supplementary Fig. 5). The high degree of mRNA\\u2013protein correlation observed in the PDX samples is consistent with recent results obtained for human breast tumours and colorectal cancer, suggesting that PDXs closely mimic\",\n",
            "          \"and colorectal cancer, suggesting that PDXs closely mimic the respective human breast tumours in their relationship between mRNA and protein.\",\n",
            "          \"We then examined the correlation of CNV, mRNA, and protein expression levels for several key genes for breast cancer biology: EGFR, ERBB2 (HER2), ESR1, GATA3, PGR, PIK3CA, AKT1/2/3, MTOR and TP53. In most cases, we observed consistent relationships\",\n",
            "          \"and TP53. In most cases, we observed consistent relationships between CNV, mRNA, and protein expression levels. Compared with PDXs of non-luminal subtypes, luminal B breast cancer xenografts, as expected, showed higher mRNA and protein expression of ESR1\",\n",
            "          \"as expected, showed higher mRNA and protein expression of ESR1 and PGR, consistent with their positive ER and PR status (Fig. 2b). Five out of the six PIK3CA mutations were observed in luminal B PDXs, which tended to also show higher mRNA and protein\",\n",
            "          \"B PDXs, which tended to also show higher mRNA and protein expression levels of GATA3. In contrast, a larger proportion of basal PDXs expressed higher protein levels of EGFR. Strong HER2 expression at the mRNA, protein and phosphoprotein levels were\",\n",
            "          \"expression at the mRNA, protein and phosphoprotein levels were detected in WHIM8 and WHIM35, both derived from HER2-positive breast cancers. Overall expression patterns for key genes were consistent with the clinical subtype diagnosis across CNV, mRNA,\",\n",
            "          \"with the clinical subtype diagnosis across CNV, mRNA, protein and phosphosite analyses (Supplementary Fig. 6).\",\n",
            "          \"Proteomic subtyping of xenografts and human breast tumours\",\n",
            "          \"Molecular subtyping of breast cancer based on mRNA expression profiles has been shown to correlate with prognosis and has treatment implications. As expected, transcriptome clustering based on PAM50 genes from RNA-seq data largely reproduced intrinsic\",\n",
            "          \"on PAM50 genes from RNA-seq data largely reproduced intrinsic subtypes of the breast tumours (Fig. 3a; Supplementary Data 6). To explore proteomic and phosphoproteomic subtype classifications in xenograft models, we conducted unsupervised clustering of\",\n",
            "          \"in xenograft models, we conducted unsupervised clustering of proteomes and phosphoproteomes of the 24 xenograft samples based upon the top 436 variably expressed proteins showing a s.d. greater than 2 from the iTRAQ proteome (Supplementary Data 7). Two\",\n",
            "          \"than 2 from the iTRAQ proteome (Supplementary Data 7). Two distinct clusters emerged: one contained all basal tumours and the only CLDN-low tumour (WHIM12), while the other included all luminal B and HER2-E breast tumours; HER2-E tumours did not show a\",\n",
            "          \"B and HER2-E breast tumours; HER2-E tumours did not show a proteomic expression profile distinct from luminal B samples (Fig. 3b). Clustering analysis using the same gene markers from the transcriptome (Supplementary Fig. 7a) and the LFQ proteome\",\n",
            "          \"the transcriptome (Supplementary Fig. 7a) and the LFQ proteome (Supplementary Fig. 7b; Supplementary Data 8) further supported the separation into these two proteomic subtypes, although the minor differences between the transcriptome and the proteome\",\n",
            "          \"minor differences between the transcriptome and the proteome clustering suggested distinctions between mRNA and protein levels. The proteomic subtypes defined by the top 968 most variably expressed mouse host proteins did not segregate based on luminal\",\n",
            "          \"mouse host proteins did not segregate based on luminal and basal subtypes (Supplementary Fig. 8; Supplementary Data 9). Besides differing in mRNA expression, luminal and basal breast cancer PDXs also showed consistently distinct proteomic expression\",\n",
            "          \"PDXs also showed consistently distinct proteomic expression profiles, supporting their distinct biological origins.\",\n",
            "          \"We then utilized the iTRAQ phosphopeptide expression data to infer phosphoproteomic subtypes. 1,737 unique phosphosites with s.d. greater than 2.5 were included to conduct hierarchical clustering (Fig. 3c; Supplementary Data 10). These analyses of the\",\n",
            "          \"(Fig. 3c; Supplementary Data 10). These analyses of the phosphoproteomic data produced two major clusters segregating the luminal B and basal subtypes. Again, the WHIM37/WHIM47/WHIM26 group and the WHIM17/WHIM46 group, as observed in the proteome\",\n",
            "          \"group and the WHIM17/WHIM46 group, as observed in the proteome clustering, grouped closely together. Gene and protein expression of lymphoid lineage markers showed high expression of CD20 and JAK3 in WHIM17 and WHIM46 (Methods, Supplementary Fig. 9),\",\n",
            "          \"and JAK3 in WHIM17 and WHIM46 (Methods, Supplementary Fig. 9), consistent with their positive EBV status and histological diagnosis (Supplementary Fig. 1) as human lymphoproliferative cells arising in an immunocompromised mouse background. Overall, while\",\n",
            "          \"in an immunocompromised mouse background. Overall, while the basal and luminal clusters remain consistent, the hierarchical distances between PDX samples within the two major clusters differ between data types. The departure of proteomic and\",\n",
            "          \"differ between data types. The departure of proteomic and phosphoproteome subtypes from mRNA expression-defined subtypes suggests independent layers of molecular heterogeneity provided by distinct proteomic analyses.\",\n",
            "          \"We then investigated whether human proteomic subtypes could be recapitulated in PDX models by including an additional 77 proteomes from TCGA human tumours that were processed concurrently in the iTRAQ experiment (Methods). To reduce clustering bias imbued\",\n",
            "          \"iTRAQ experiment (Methods). To reduce clustering bias imbued by the mouse contribution to the proteome in the PDX samples, we excluded proteins showing differential expression between human tumour and xenograft (t-test, FDR\\u22640.3) (Supplementary Data 11).\",\n",
            "          \"tumour and xenograft (t-test, FDR\\u22640.3) (Supplementary Data 11). Additional requirements, including the presence of detectable protein in more than 10 samples and minimum difference of 2 s.d.'s in the merged proteome, resulted in 133 proteins qualifying\",\n",
            "          \"in the merged proteome, resulted in 133 proteins qualifying for un-supervised clustering. Consistent with the subtyping analysis of the 24 PDXs alone, we identified two major clusters: one that included all but one basal breast tumour and the other\",\n",
            "          \"one that included all but one basal breast tumour and the other comprised mostly of luminal tumours (Fig. 3d). Similar to the proteomic subtypes of the PDX cohorts, luminal tumours and HER-2 tumours did not show clear separation, although several\",\n",
            "          \"HER-2 tumours did not show clear separation, although several sub-clusters were identified. Importantly, xenograft proteomes clustered adjacently to the human proteomes of their respective subtypes, validating the fidelity of basal and luminal proteomic\",\n",
            "          \"validating the fidelity of basal and luminal proteomic signatures discovered in PDXs.\",\n",
            "          \"To search for defining markers between the basal and luminal B subtypes, we conducted differential expression analysis between the PDXs of the respective subtypes. We found several proteomic markers that were differentially expressed in both the LFQ and\",\n",
            "          \"markers that were differentially expressed in both the LFQ and iTRAQ datasets (t-test, FDR\\u22640.05), including SPR, GSTP1 and SERPINB5 (Supplementary Fig. 10; Supplementary Data 12). We then conducted gene-set enrichment analysis based on the Reactome\",\n",
            "          \"conducted gene-set enrichment analysis based on the Reactome pathway database to investigate patterns of differential expression (Methods). The basal subtype breast tumours were up-regulated in most of the significantly differentially-expressed pathways\",\n",
            "          \"in most of the significantly differentially-expressed pathways (FDR\\u22640.01) identified by both LFQ and iTRAQ datasets, including extracellular matrix organization, cell cycle, and collagen formation. In comparison, the luminal B breast tumours showed higher\",\n",
            "          \"In comparison, the luminal B breast tumours showed higher expression in genes related to organelle biogenesis and membrane trafficking (Supplementary Data 13).\",\n",
            "          \"Activated pathways revealed by phosphorylation profiles\",\n",
            "          \"Cancer driving somatic events trigger major changes in downstream signalling to launch the tumorigenic cascade. To search for tumour-specific activated pathways in PDX tumours, we systematically evaluated and compared phosphoproteome profiles of gene sets\",\n",
            "          \"evaluated and compared phosphoproteome profiles of gene sets from KEGG signalling pathways (Methods). Phosphorylation enrichment analysis identified 12 significantly activated pathways, including Ras, MAPK and NF\\u03baB signalling, in 4 xenografts (FDR\\u22640.01,\",\n",
            "          \"Ras, MAPK and NF\\u03baB signalling, in 4 xenografts (FDR\\u22640.01, Supplementary Data 14). WHIM9 exhibited elevated phosphorylation of the MAPK signalling pathway (FDR=9.69e-6; Fig. 4a). Interestingly, WHIM9 carried a recurrent somatic mutation, KRAS p.A146V\",\n",
            "          \"WHIM9 carried a recurrent somatic mutation, KRAS p.A146V (refs), which may have driven canonical MAPK pathway activation. Further, WHIM12 exhibited activation of the Ras signalling pathway (FDR=4.28e-5), along with an outlier protein expression of MET, a\",\n",
            "          \"along with an outlier protein expression of MET, a receptor tyrosine kinase upstream of the Ras signalling pathway (Fig. 4b). Interestingly, WHIM17 and WHIM46, both harboured BTK and PLCG2 protein overexpression, exhibited overall high phosphorylation of\",\n",
            "          \"overexpression, exhibited overall high phosphorylation of the NF\\u03baB signalling pathway (Fig. 4c,d; FDR=6.94e-3, 4.53e-5 respectively). This observation further supports the strong similarity between these two PDX models based on protein/phosphoprotein\",\n",
            "          \"between these two PDX models based on protein/phosphoprotein clustering (Fig. 3b,c) and their classification as EBV-positive lymphoproliferation.\",\n",
            "          \"While pathways such as PI3K/AKT/MTOR are known to be activated across the majority of breast tumours, our analysis shows that other complementary tumorigenesis-related pathways including RAS/MAPK are also activated in a small set of breast tumours due to\",\n",
            "          \"are also activated in a small set of breast tumours due to specific genomic or proteomic alterations, representing alternative treatment opportunities. In addition, our phospho-proteomic analysis revealed activation of signalling pathways not readily\",\n",
            "          \"analysis revealed activation of signalling pathways not readily predicted by genomic data.\",\n",
            "          \"Complementary genomic/proteomic druggable targets\",\n",
            "          \"We examined promising drug targets in each tumour by surveying the genome and expressed proteomes. Specifically, we compiled a list 76 druggable genes (Supplementary Data 15), along with their respective drugs, from established public databases (Methods).\",\n",
            "          \"respective drugs, from established public databases (Methods). Six PDXs, representing 20.8% of the tumours in this study, harboured druggable somatic mutations, including PIK3CA p.H1047R and KRAS p.A146V in WHIM9, PIK3CA p.H1047R in WHIM16 and WHIM24,\",\n",
            "          \"KRAS p.A146V in WHIM9, PIK3CA p.H1047R in WHIM16 and WHIM24, PIK3CA p.E545K in WHIM18, PIK3CA p.E542K in WHIM20 and SF3B1 p.K700E in WHIM26 (Supplementary Data 16).\",\n",
            "          \"While activating mutations in oncogenes can be targeted by treatments, aberrantly overexpressed or activated protein products, such as HER2, also presents exploitable treatment opportunities. We sought genomic and proteomic evidence of overexpressed\",\n",
            "          \"We sought genomic and proteomic evidence of overexpressed genes/proteins or proteins with highly phosphorylated sites. We defined outliers as expression values exceeding the 1.5 interquartile ranges (IQR) above the third quartile of the cohort (Variations\",\n",
            "          \"ranges (IQR) above the third quartile of the cohort (Variations of Box Plots), and rank ordered them by the outlier score. We further required CNV outliers to be validated with outlier expression in either the mRNA or protein level to rule out\",\n",
            "          \"expression in either the mRNA or protein level to rule out up-regulation events due to technical artifacts or passenger events (Methods). mRNA and protein expression outlier scores showed a moderate positive correlation (Fig. 5a, R=0.516), but mRNA\",\n",
            "          \"a moderate positive correlation (Fig. 5a, R=0.516), but mRNA outlier expression did not guarantee high protein expression (for example, AKT2 and FGFR2, Fig. 5a). Similarly, we observed a fraction of phosphosite outliers not detected at the protein level\",\n",
            "          \"of phosphosite outliers not detected at the protein level (Fig. 5b, R=0.548). Consequently, identifying post-transcriptional and post-translational events to capture potential druggable treatment opportunities requires consideration of protein expression\",\n",
            "          \"opportunities requires consideration of protein expression as well as gene expression.\",\n",
            "          \"Applying this druggable outlier detection strategy across CNV, mRNA, protein and protein phosphorylation levels, we identified overexpressed druggable genes in 26.1% and 47.8% of PDXs at the CNV and mRNA levels, respectively (Fig. 6; Supplementary Data\",\n",
            "          \"CNV and mRNA levels, respectively (Fig. 6; Supplementary Data 16). These events recapitulated known druggable opportunities, such as the PIK3CA copy-number amplification in WHIM4 and ERBB2 (HER2) copy-number amplification in WHIM35. Expanding to iTRAQ\",\n",
            "          \"(HER2) copy-number amplification in WHIM35. Expanding to iTRAQ protein expression outliers allowed us to uncover druggable targets in 19 out of the 24 PDXs (79.2%), while considering phosphosite outliers covered 22 of the 24 PDXs (91.7%). A significantly\",\n",
            "          \"outliers covered 22 of the 24 PDXs (91.7%). A significantly high fraction of protein outliers overlapped between the LFQ and iTRAQ datasets (Fisher's Exact Test, P=2.013e-05, Supplementary Fig. 11), providing validation of our findings.\",\n",
            "          \"The identified proteomic outliers included multiple known druggable targets involved in the PI3K, RTK, MAPK signalling pathways and other oncogenic processes (Fig. 6; Supplementary Data 16). For instance, HER2 was found to be the top outlier at the\",\n",
            "          \"16). For instance, HER2 was found to be the top outlier at the protein expression level in 2 HER2-E xenografts, WHIM8 and WHIM35; the phosphosites of HER2 were also identified as outliers. Immunohistochemistry experiments validated the high HER2 protein\",\n",
            "          \"experiments validated the high HER2 protein expression on the cell membranes of WHIM8 and WHIM35 (Fig. 5c; Supplementary Fig. 12). Further, AKT1, AKT2 and AKT3 proteins were outlier expression candidates in WHIM35, WHIM20 and WHIM43 and WHIM4,\",\n",
            "          \"expression candidates in WHIM35, WHIM20 and WHIM43 and WHIM4, respectively, providing proteomic validation of previously observed AKT up-regulation at mRNA expression level in breast cancer. Other outlier proteins included IDH1 in WHIM9 and WHIM30, FGFR4\",\n",
            "          \"Other outlier proteins included IDH1 in WHIM9 and WHIM30, FGFR4 in WHIM11 and WHIM26, and both BTK and JAK3 in the EBV-positive WHIM17 and WHIM46. Many of these protein outliers aligned with up-regulation in their corresponding phosphosites, including\",\n",
            "          \"up-regulation in their corresponding phosphosites, including phosphosites on AKT1, AKT3, BRAF, FGFR4 and HSP90AB1 (Fig. 5c, selected phosphosite spectra in Supplementary Fig. 13). We also discovered additional outlier phosphosites in other proteins\",\n",
            "          \"discovered additional outlier phosphosites in other proteins including CTNNB1, ARAF, and HSP90B1 (Supplementary Data 16). Finally, we identified outlier FGFR2 in WHIM16 and high RAF1 in WHIM9 and further validated their high protein expression status of\",\n",
            "          \"and further validated their high protein expression status of by immunohistochemistry analyses (Fig. 5c; Supplementary Fig. 12). As diverse sets of proteomic outliers are identified across PDXs, effectively inhibiting such activation events will be\",\n",
            "          \"PDXs, effectively inhibiting such activation events will be required when designing targeted treatment strategies to each individual breast tumour.\",\n",
            "          \"Notably, for 80.6% of proteomic outlier events, we were able to identify human tumours from the 77 TCGA samples showing the same outlier protein expression through outlier score or ranking (Methods, Fig. 6b, Supplementary Data 16). For example, we\",\n",
            "          \"(Methods, Fig. 6b, Supplementary Data 16). For example, we observed outlier HER2 expression in five HER2-E and four luminal B human breast tumours. Further, a basal human sample carried both the outlier FGFR2 and FGFR4 expression, validating the findings\",\n",
            "          \"the outlier FGFR2 and FGFR4 expression, validating the findings in WHIM16, WHIM26 and WHIM11. Basal human breast tumours also carried protein expression outliers in IDH1, EGFR and MAP2K1 (Fig. 6b). Phosphosite outlier expression events showed a moderate\",\n",
            "          \"6b). Phosphosite outlier expression events showed a moderate rate of validation in the same human cohort (48.8%, Fig. 6c), suggesting its transient nature and potential micro-environmental effects on protein phosphorylation. As expected, HER2-E and a few\",\n",
            "          \"on protein phosphorylation. As expected, HER2-E and a few HER2-positive luminal B tumours carried outliers in HER2 phosphosites including p.T701, p.T1240 and p.Y1248. ARAF phosphosite outliers, such as p.S299, were found in both human and PDX samples,\",\n",
            "          \"such as p.S299, were found in both human and PDX samples, validating our previous finding in the human cohort. Interestingly, we identified outlier phosphosites in genes not previously implicated in breast cancer through genomic profiles, such as BRAF\",\n",
            "          \"in breast cancer through genomic profiles, such as BRAF p.S447, p.S750 and HSP90AB1 p.Y56 and p.S169 (Fig. 6c). Our results demonstrated that proteomic outlier events, like genomic driver mutations, are consistently observed in PDXs and human tumours.\",\n",
            "          \"mutations, are consistently observed in PDXs and human tumours. Some protein outlier events might represent \\u2018proteomic drivers' of tumorigenesis and therefore potential drug targets in breast tumours.\",\n",
            "          \"Targeted treatments using breast cancer xenograft models\",\n",
            "          \"To validate the identified druggable events, we conducted treatment experiments targeting HER2 and PI3K pathways on selected PDX models. Four PDX models were chosen to address HER2 targeting using lapatinib, an oral HER2 kinase inhibitor. These include 2\",\n",
            "          \"using lapatinib, an oral HER2 kinase inhibitor. These include 2 HER2-E PDX models WHIM8 and WHIM35, both with high HER2 protein and phosphoprotein expression, and 2 low HER2 expressing, basal-like PDX models WHIM6 and WHIM14. Western blotting suggested\",\n",
            "          \"PDX models WHIM6 and WHIM14. Western blotting suggested high HER2 expression and phosphorylation of HER2 p.Y1248 in both HER2 positive tumours, WHIM8 and WHIM35 (Fig. 7a; Supplementary Fig. 14). Unexpectedly, high levels of HER2 p.Y1248 phosphorylation\",\n",
            "          \"14). Unexpectedly, high levels of HER2 p.Y1248 phosphorylation was also detected in WHIM14 by western blotting. In contrast MS-based phosphoproteomics also detected high levels of HER2p.Y1248 in WHIM8 and WHIM35, but not in WHIM14, which was more\",\n",
            "          \"in WHIM8 and WHIM35, but not in WHIM14, which was more consistent with the known biology of the models. Antibody-based diagnosis of HER2 activation in WHIM14 was possibly due to cross-reaction of the Y1248 antibody with the pY1172 site of EGFR that bears\",\n",
            "          \"of the Y1248 antibody with the pY1172 site of EGFR that bears high sequence similarity around the pY residue (..GTPTAENPEy1248LGLDVPV-CO2H vs...GSTAENAEy1172LRVAPQ.., Supplementary Fig. 6). As expected, WHIM8 and WHIM35 were growth inhibited (Wilcoxon\",\n",
            "          \"As expected, WHIM8 and WHIM35 were growth inhibited (Wilcoxon Rank Sum Test, P=2.7e-5, 4.8e-7) by lapatinib, whereas WHIM6 was not (P=0.65) (Fig. 7b). Interestingly, WHIM14 also showed significant reduction in tumour growth by lapatinib (P=4.3e-3). Upon\",\n",
            "          \"reduction in tumour growth by lapatinib (P=4.3e-3). Upon further exploration with a lower, clinically achievable dose of lapatinib for chronic treatment (30\\u2009mg\\u2009kg\\u22121 compared with 220\\u2009mg\\u2009kg\\u22121 in the previous experiment) over 48 days, significant but weak\",\n",
            "          \"in the previous experiment) over 48 days, significant but weak tumour growth inhibition was again achieved (P=0.0311, Supplementary Fig. 15). However, WHIM14 did not respond to HER2/HER3 antibodies trastuzumab or pertuzumab (P=0.250 and 0.181,\",\n",
            "          \"antibodies trastuzumab or pertuzumab (P=0.250 and 0.181, respectively; Supplementary Fig. 15). Thus, the response of WHIM14 to lapatinib was likely due to inhibition of EGFR not HER2 (refs). While WHIM6 also showed elevated EGFR protein level, WHIM14\",\n",
            "          \"While WHIM6 also showed elevated EGFR protein level, WHIM14 showed notably higher EGFR phosphorylation based on both mass spectrometry (Fig. 2b) and western blotting (Fig. 7a), which could account for their different response to lapatinib. Two basal and\",\n",
            "          \"for their different response to lapatinib. Two basal and one HER2-E human breast cancers harboured outlier EGFR expression (Fig. 6b), suggesting EGFR remains a potential therapeutic target in a subset of breast cancers that has yet to be fully realized\",\n",
            "          \"in a subset of breast cancers that has yet to be fully realized clinically.\",\n",
            "          \"The PI3K/AKT/mTOR signalling pathway is altered in approximately 77% of breast tumours and multiple drugs targeting its components, including Class I PI3Ks, AKTs, and mTORs, are already in clinical trials. Among these, a combination of everolimus and\",\n",
            "          \"clinical trials. Among these, a combination of everolimus and exemestane has been approved for treating advanced ER-positive breast cancers resistant to non-steroidal aromatase inhibitors. Promising activity has also been reported for direct inhibitors of\",\n",
            "          \"activity has also been reported for direct inhibitors of PI3K. However, mutations in PIK3CA and other genetic alterations at the genomic level have so far failed to closely predict therapeutic responsiveness to PI3K pathway inhibitors. We therefore\",\n",
            "          \"responsiveness to PI3K pathway inhibitors. We therefore hypothesized that combined genomic and proteomic indication of PI3K signalling activation is necessary for the prediction of treatment response. Our integrated approach identified 6 xenografts that\",\n",
            "          \"response. Our integrated approach identified 6 xenografts that harboured complementary genomic and proteomic druggable events in the PI3K-AKT pathway. In particular, WHIM16 harboured a hotspot PIK3CA p.H1047R mutation, whereas WHIM18 and WHIM20 each\",\n",
            "          \"PIK3CA p.H1047R mutation, whereas WHIM18 and WHIM20 each carried a hotspot PIK3CA p.E545K mutation. WHIM20 also showed an additional outlier protein expression of AKT2 and WHIM18 also expressed AKT2 at a near outlier level that may combine to activate\",\n",
            "          \"AKT2 at a near outlier level that may combine to activate PI3K pathway signalling (Fig. 6). Since the treatments were applied to later-passage PDX models relative to the ones we conducted proteogenomic analysis on, we performed immunohistochemistry to\",\n",
            "          \"proteogenomic analysis on, we performed immunohistochemistry to validate the phosphorylation of AKT p.S473. WHIM18 and WHIM20 showed detectable AKT p.S473, which was not observed in WHIM16 (Fig. 7c).\",\n",
            "          \"We conducted combinatorial treatment experiments by applying an alpha specific PI3K inhibitor (TAK-117) and/or an mTORC1/2 inhibitor (TAK-228) to three PDX models of luminal B breast cancer. Consistent with previous reports, PIK3CA mutation status alone\",\n",
            "          \"Consistent with previous reports, PIK3CA mutation status alone did not accurately predict outcome; WHIM18 and WHIM20 showed reduced tumour growth upon application of the inhibitors, whereas WHIM16 did not (Fig. 7d). mTOR inhibition repressed tumour growth\",\n",
            "          \"did not (Fig. 7d). mTOR inhibition repressed tumour growth in WHIM18 and WHIM20 (ANOVA followed by Tukey's post hoc test, P=9.5e-10, 1.6e-03 respectively), but not in WHIM16 (P=0.97), showing that inhibition of mTORC1/2 may effectively suppress breast\",\n",
            "          \"that inhibition of mTORC1/2 may effectively suppress breast tumours with activated AKTs and validating a previous study showing the efficacy of mTOR inhibition in PDX models of triple-negative breast cancer. Importantly, the combinatorial treatment\",\n",
            "          \"breast cancer. Importantly, the combinatorial treatment achieved the greatest effect in WHIM18 and WHIM20 (P<2.2e-16 for both comparisons). While neither PI3K nor mTOR inhibitor drug treatment alone suppressed tumour growth in WHIM20 completely, the\",\n",
            "          \"alone suppressed tumour growth in WHIM20 completely, the combination of both PI3K and mTOR inhibitors significantly reduced tumour growth to a nearly static state (Fig. 7d). Based on our proteomic characterization, WHIM20 exhibited the strongest AKT1 and\",\n",
            "          \"characterization, WHIM20 exhibited the strongest AKT1 and AKT2 protein expression, as well as AKT1 p.S122 and p.S475 phosphorylation signatures followed by WHIM18 and then WHIM16 (Fig. 6b,c). Further, both PI3K and mTOR inhibitors significantly reduced\",\n",
            "          \"Further, both PI3K and mTOR inhibitors significantly reduced AKT p.S473 (Fig. 7c). Our treatment results showed that the magnitude of drug response may be associated with overexpression and phosphorylation of the downstream signalling targets such as AKT\",\n",
            "          \"of the downstream signalling targets such as AKT proteins.\",\n",
            "          \"In addition to this validation of druggable hypotheses in luminal tumours, our previous report also demonstrated the effectiveness of combinatorial therapy of AKT and mTOR inhibitors in two other basal breast cancer xenografts. One of the treated\",\n",
            "          \"in two other basal breast cancer xenografts. One of the treated xenografts, WHIM4, was also characterized in this study, and showed copy-number amplification of PIK3CA and AKT3 protein outlier expression (Fig. 6). Overall, proteogenomic analysis revealed\",\n",
            "          \"expression (Fig. 6). Overall, proteogenomic analysis revealed that the dual activation of PIK3CA at the genomic level and AKTs at the protein level may be a common signature of breast tumours, affecting more than 20% of PDXs in this cohort. Importantly,\",\n",
            "          \"affecting more than 20% of PDXs in this cohort. Importantly, our results demonstrate the potential utility of combinatorial inhibitor treatments to treat breast tumours showing these proteogenomic signatures.\",\n",
            "          \"Discussion\",\n",
            "          \"Breast cancer has been traditionally characterized in the clinic through hormone receptor status and selected genes' expressions, and more recently by genomic sequencing. However, druggable genomic driver events are detectable in only a limited percentage\",\n",
            "          \"driver events are detectable in only a limited percentage of patients. As the majority of drugs target proteins, a systematic evaluation of breast cancer proteomes would seem ultimately to be necessary for selecting targeted treatment and predicting drug\",\n",
            "          \"necessary for selecting targeted treatment and predicting drug response. Recent advances in MS-based proteomics allow extensive and quantitative surveys of the global proteome. Here, we have systematically analyzed proteogenomic profiles of 22\",\n",
            "          \"we have systematically analyzed proteogenomic profiles of 22 patient-derived breast cancer xenografts and 2 EBV-positive lymphoproliferations that are likely artifacts of engraftment of human lymphocytes into NSG mice.\",\n",
            "          \"This study shows that proteogenomic signatures of PDXs resemble most findings from breast cancer patients. While some discrepancies exist, we established a normalization strategy at both the genomic and proteomic levels that enabled direct comparison. PDX\",\n",
            "          \"and proteomic levels that enabled direct comparison. PDX tumours recapitulated the proteomic diversity of human breast cancers (Fig. 3). We also identified multiple druggable targets for each tumour model (Fig. 6; Supplementary Data 17). Proteomic events\",\n",
            "          \"tumour model (Fig. 6; Supplementary Data 17). Proteomic events validated a significant number of CNV or mRNA up-regulations. For example, HER2 protein and phosphosite outlier expression was observed in HER2-E WHIM8 and WHIM35 (Fig. 5), which were\",\n",
            "          \"was observed in HER2-E WHIM8 and WHIM35 (Fig. 5), which were effectively treated using lapatinib (Fig. 7b). Interestingly, we also identified overexpressed proteomic events not evident in the genomic level in both PDX and human samples, including outlier\",\n",
            "          \"genomic level in both PDX and human samples, including outlier protein expression of EGFR, and outlier phosphosite expressions of ARAF, BRAF, HSP90AB1, PTPN11 and TOP2A (Fig. 6), highlighting potential new treatment opportunities in breast cancer. In the\",\n",
            "          \"potential new treatment opportunities in breast cancer. In the two PDX models subsequently diagnosed as EBV-positive lymphoproliferations, we observed outlier BTK expression (Fig. 6) and activation of the NF\\u03baB pathway (Fig. 4), validating BTK as a\",\n",
            "          \"activation of the NF\\u03baB pathway (Fig. 4), validating BTK as a druggable target in EBV+ lymphomas. While more than 80% of the proteomic outlier events in PDX were also found in human tumours, a lessor 48.8% of phosphosite outlier events were validated,\",\n",
            "          \"a lessor 48.8% of phosphosite outlier events were validated, potentially due to different tumour micro-environments. Thus, transient phosphoproteomic events identified in PDX tumours would likely require further verification in their corresponding primary\",\n",
            "          \"require further verification in their corresponding primary tumours.\",\n",
            "          \"Outlier protein expression events can likely lead to downstream pathway activation, such as MET outlier protein expression (Fig. 6) and activated Ras pathway observed in WHIM12 (Fig. 4). While genomic analysis has utilized mutual occurrence or exclusivity\",\n",
            "          \"genomic analysis has utilized mutual occurrence or exclusivity in patient cohorts to deduce pathway relationships, phosphorylation profile analyses allowed us to directly interrogate signalling in these established pathways in a single sample. Our pathway\",\n",
            "          \"in these established pathways in a single sample. Our pathway activation results suggest these events may be crucial to tumorigenesis and that some are likely the proteomic \\u2018smoking guns' that originally triggered the oncogeneic cascade.\",\n",
            "          \"Roughly 77% of breast tumours showed alterations in the PI3K-AKT signalling pathway, representing a potentially important path forward for drug-based treatment. Yet, genomic alterations of PI3K pathway components have not shown themselves to predict\",\n",
            "          \"of PI3K pathway components have not shown themselves to predict treatment responsiveness to PI3K pathway inhibitors. In this study, we defined complementary druggable targets of the pathway using proteogenomic analysis, including several events of a\",\n",
            "          \"using proteogenomic analysis, including several events of a co-occurring PIK3CA mutation or copy-number amplification, and AKT protein outlier expression coupled by elevated AKT phosphorylation. In two such breast tumours, we successfully inhibited tumour\",\n",
            "          \"In two such breast tumours, we successfully inhibited tumour growth using a combination of PI3K and MTOR inhibitors (Fig. 7c). While these results show potential functional implications, additional, systematic treatment experiments are required to\",\n",
            "          \"additional, systematic treatment experiments are required to validate the identified proteomic druggable targets.\",\n",
            "          \"In conclusion, this initial work using proteogenomic integration coupled with patient-derived xenograft validation, has demonstrated a strategy that, in principle, may enable more accurate prediction of the efficacy of mechanism-based cancer therapeutics.\",\n",
            "          \"Methods\\n\\nXenograft model generation\",\n",
            "          \"Patient-derived xenografts were generated from primary or metastatic breast tumours using previously described procedures. All human tissues for these experiments were processed in compliance with NIH regulations and institutional guidelines, and approved\",\n",
            "          \"with NIH regulations and institutional guidelines, and approved by the institutional review board at Washington University. All animal procedures were reviewed and approved by the institutional animal care and use committee at Washington University in St.\",\n",
            "          \"animal care and use committee at Washington University in St. Louis. PDX models are available through the application to the Human and Mouse-Linked Evaluation of Tumors core at http://digitalcommons.wustl.edu/hamlet/.\",\n",
            "          \"We selected 24 of the established breast tumour samples, including 9 luminal B, 10 basal, 4 HER2-E, and 1 CLDN-low breast tumours for further proteogenomic characterization. Receptor statuses of xenograft tumours were validated using IHC after\",\n",
            "          \"statuses of xenograft tumours were validated using IHC after engraftment.\",\n",
            "          \"Immunohistochemistry\",\n",
            "          \"Xenografts were formalin-fixed at least for 24\\u2009h and paraffin-embedded. Sections were evaluated by hematoxylin & eosin (H&E) staining. Immunohistochemistry (IHC) was performed on additional sections for HER2 (Dako), Phospho-Akt (Ser473) (Cell signalling),\",\n",
            "          \"for HER2 (Dako), Phospho-Akt (Ser473) (Cell signalling), FGFR2 (Abcam), and Raf-1 (Santa Cruz) following the manufacturer's instructions. Western blotting of phosphorylated HER2 (p.Y1248) was performed using antibody cat Nr-06-229 (Millipore).\",\n",
            "          \"In vivo drug treatment experiments\",\n",
            "          \"For the targeted treatment of the PI3K pathway, PI3K alpha inhibitor TAK-117 and TORC1/2 inhibitor TAK-228 were provided by Millennium Pharmaceuticals, Inc. The compounds were dissolved in Peg400. Tumours were engrafted in NSG (NOD.Cg-Prkdcscid\",\n",
            "          \"in Peg400. Tumours were engrafted in NSG (NOD.Cg-Prkdcscid Il2rgtm1Wjl/SzJ) mice (The Jackson Laboratory) by subcutaneous injection of 2\\u20135 \\u00d7 106 PDX cells in PBS supplemented with 30% Matrigel (BD Biosciences, cat. no. 354234). When tumours reached an\",\n",
            "          \"(BD Biosciences, cat. no. 354234). When tumours reached an average size of 250\\u2013300\\u2009mm3, animals were assigned randomly to control and various treatment groups (n=8\\u20139 each group). For treatment of WHIM16 (passage 7), 18 (passage 8), and 20 (passage 5) were\",\n",
            "          \"of WHIM16 (passage 7), 18 (passage 8), and 20 (passage 5) were used. Tumour bearing mice were gavaged with: (1) Peg400; (2) TAK-117, 140\\u2009mg\\u2009kg\\u22121\\u2009day\\u22121; (3) TAK-228, 1\\u2009mg\\u2009kg\\u22121\\u2009day\\u22121; (4) TAK-117, 140\\u2009mg\\u2009kg\\u22121\\u2009day\\u22121, TAK-228, 1\\u2009mg\\u2009kg\\u22121\\u2009day\\u22121. The mice were\",\n",
            "          \"140\\u2009mg\\u2009kg\\u22121\\u2009day\\u22121, TAK-228, 1\\u2009mg\\u2009kg\\u22121\\u2009day\\u22121. The mice were treated on three consecutive days once daily and then had a 4-day interval. Tumours were measured with external caliper, and volume was calculated as (4\\u03c0/3) \\u00d7 (width/2)2 \\u00d7 (length/2).\",\n",
            "          \"For the lapatinib therapeutic experiments of WHIM6 (passage 7), WHIM 14 (passage 11), WHIM 8 (passage 6) and WHIM 35 (passage 6), 1 \\u00d7 106 tumour cells were added to equal volume of 1:1 mixture of Matrigel and 10% RPMI plus 10% FBS to 4th mammary fat pads\",\n",
            "          \"of Matrigel and 10% RPMI plus 10% FBS to 4th mammary fat pads in female SCID/bg mice (ENVIGO). We then established tumours to an average volume 250\\u2013300\\u2009mm3, and randomized the mice into control and lapatinib groups. We treated the treated group mice with\",\n",
            "          \"and lapatinib groups. We treated the treated group mice with lapatinib chow diet by formulating lapatinib (220\\u2009mg\\u2009kg\\u22121) and processing them into food pellets (by Research Diets Inc), which is supplied for 2 weeks. The numbers of replicates for each group\",\n",
            "          \"supplied for 2 weeks. The numbers of replicates for each group are as followed: WHIM 14 control (n=6) and lapatinib treated (n=5); WHIM 6 control (n=11) and lapatinib treated (n=12); WHIM 8 control (n=16) and lapatinib treated (n=6); WHIM 35 control\",\n",
            "          \"8 control (n=16) and lapatinib treated (n=6); WHIM 35 control (n=17) and lapatinib treated (n=18). We also treated WHIM14 with 100\\u2009mg\\u2009kg\\u22121 lapatinib treatment for 48 days, including both control (n=8) and lapatinib treated (n=9) groups. Further, we tested\",\n",
            "          \"(n=8) and lapatinib treated (n=9) groups. Further, we tested the effect of trastuzumab (30\\u2009mg\\u2009kg\\u22121 weekly with IP injections) and pertuzumab (30\\u2009mg\\u2009kg\\u22121 weekly with IP injections) on PDX tumour growth for 48 days. The experimental groups are as following:\",\n",
            "          \"growth for 48 days. The experimental groups are as following: control group (n=6) treated with physiological saline (vehicle); trastuzumab treated group (n=12); pertuzumab treated group (n=11).\",\n",
            "          \"Statistical testing of the resulting data was conducted using the R programming language. Wilcoxon rank sum test was applied to compare the fold change in tumour volumes after two weeks in the lapatinib treatment experiment, and after 44 days or 48 days\",\n",
            "          \"lapatinib treatment experiment, and after 44 days or 48 days for the additional WHIM14 experiment using low-dose lapatinib and trastuzumab/pertuzumab. One-way ANOVA was applied to compare the treated versus control groups in the PI3K targeted-therapy\",\n",
            "          \"the treated versus control groups in the PI3K targeted-therapy experiments, and a follow-up Tukey's post hoc test was used for the various comparisons in PI3K inhibition experiments. All human tissues for these experiments were processed in compliance\",\n",
            "          \"tissues for these experiments were processed in compliance with NIH regulations and institutional guidelines, and approved by the Washington University, University of North Carolina, or Baylor College of Medicine Institutional Review Board (IRB). All\",\n",
            "          \"College of Medicine Institutional Review Board (IRB). All animal procedures were reviewed and approved by the institutional animal care and use committee at Washington University in St. Louis, University of North Carolina, or Baylor College of Medicine.\",\n",
            "          \"Genomic and proteomic data generation.\\n\\nSomatic mutation\",\n",
            "          \"Sequencing reads were aligned using BWA. Somatic variants were identified using VarScan2 (refs), GATK and Pindel, and annotated based on Ensembl release 70_37. We then filtered out common variants by using variants from the 1000 Genomes and NHLBI\",\n",
            "          \"variants by using variants from the 1000 Genomes and NHLBI projects. We further eliminated mouse contamination by filtering somatic variants that were mapped to mouse reference genome. Somatic mutation calls were validated either using a custom array or\",\n",
            "          \"mutation calls were validated either using a custom array or manually reviewed in IGV. The genomic data of 17 out of the 24 PDXs have been presented before in our previous studies (Supplementary Data 2).\",\n",
            "          \"Copy-number variation\",\n",
            "          \"The segment-based copy-number data were generated using the whole-genome sequencing and exome sequencing data. We then converted the segment-based copy-number data to the gene-based copy-number data by using the RefSeq database (version 20130727). The\",\n",
            "          \"data by using the RefSeq database (version 20130727). The copy-number values were further transformed to the log-R ratio, using the cohort mean for the gene as the reference.\",\n",
            "          \"mRNA expression and virus detection through RNA-seq\",\n",
            "          \"mRNA expression values were calculated from mRNA sequencing data using MapSplice. The resulting RSEM values were normalized within samples to a fixed upper quartile. Upper quartile normalized RSEM data were log2 transformed and the data were median\",\n",
            "          \"RSEM data were log2 transformed and the data were median centred by gene. To quantify virus abundance, we used the VirusScan pipeline (https://github.com/ding-lab/VirusScan) to detect viruses by numbers of virus-supporting reads from RNA-seq data.\",\n",
            "          \"LFQ proteome\",\n",
            "          \"Tumour Sample Generation and Protein Extraction. Patient-derived xenograft breast tumours were processed to cryopulverized powders as described previously. The powders (100\\u2009mg wet weight) were subjected to lysis and protein extraction using a buffer\",\n",
            "          \"were subjected to lysis and protein extraction using a buffer composed of 8 M urea, 50\\u2009mM Tris pH 8.0, 75\\u2009mM NaCl, 1\\u2009mM MgCl2, and 500 units Benzonase. Approximately 1\\u2009mg of total protein extracted was reduced with DTT and subsequently alkylated with\",\n",
            "          \"extracted was reduced with DTT and subsequently alkylated with iodoacetamide. The proteins were then subjected to proteolysis with endoproteinase Lys-C (Wako Chemicals, USA) for \\u223c4\\u2009h at 37\\u2009\\u00b0C. The solution was diluted 4-fold with 25\\u2009mM Tris pH 8.0, 1\\u2009mM\",\n",
            "          \"The solution was diluted 4-fold with 25\\u2009mM Tris pH 8.0, 1\\u2009mM CaCl2 and further digested with trypsin (Promega) for \\u223c12\\u2009h at 37\\u2009\\u00b0C. Digestion was stopped by the addition of TFA to 0.4%, and the precipitate was removed by centrifugation. The peptide\",\n",
            "          \"and the precipitate was removed by centrifugation. The peptide solutions were desalted on Sep-Pak Light C18 cartridges (Waters) and dissolved in 30% ACN, 0.1% TFA before loading on a 300\\u2009\\u03bcm Source 15\\u2009S (GE Healthcare) column for Basic Reversed Phase\",\n",
            "          \"Source 15\\u2009S (GE Healthcare) column for Basic Reversed Phase Chromatography (bRPLC). A linear LC gradient was performed by increasing buffer B from 0 to 70% within 60\\u2009min, where buffer A was aqueous 10\\u2009mM ammonium formate, and buffer B was 90% ACN in 10\\u2009mM\",\n",
            "          \"10\\u2009mM ammonium formate, and buffer B was 90% ACN in 10\\u2009mM ammonium formate. A total of 30 fractions were collected for each WHIM sample (18 WHIMs). Five fractions were then prepared by combining non-contiguously fractions. We analyzed an additional\",\n",
            "          \"combining non-contiguously fractions. We analyzed an additional technical replicate for WHIM2 and WHIM16. The fractions were dried and desalted using a stop-and-go-extraction tip (StageTip) protocol containing 4 \\u00d7 1\\u2009mm C18 extraction disk (3M).\",\n",
            "          \"Liquid Chromatography-Tandem Mass Spectrometry and Protein Identification. Sample analysis was performed via reversed phase LC-MS/MS using a Proxeon 1000 nano LC system coupled to a Q Exactive mass spectrometer (Thermo Scientific, San Jose, CA). The\",\n",
            "          \"mass spectrometer (Thermo Scientific, San Jose, CA). The Proxeon system was configured to trap peptides using a C18 column (3\\u2009cm \\u00d7 100\\u2009\\u03bcm i.d.) with a diverted flow rate (5\\u2009\\u03bcl\\u2009min\\u22121) The trap column was placed in line with the analytical column (15\\u2009cm \\u00d7\",\n",
            "          \"column was placed in line with the analytical column (15\\u2009cm \\u00d7 75\\u2009\\u03bcm i.d., 3.5\\u2009\\u03bcm, 300\\u2009\\u00c5 particle C18, Thermo Scientific, San Jose, CA, USA) before gradient elution of peptides. Analytical separation of all the tryptic peptides was achieved with a linear\",\n",
            "          \"of all the tryptic peptides was achieved with a linear gradient of 2\\u201330% buffer B over 240\\u2009min (250\\u2009nl\\u2009min\\u22121), where buffer A was aqueous 0.1% formic acid, and buffer B was acetonitrile in 0.1% formic acid.\",\n",
            "          \"LC-MS experiments were performed in a data-dependent mode with Full-MS (externally calibrated to a mass accuracy of <5 p.p.m., and a resolution of 70,000 at m/z=200) followed by HCD-MS/MS of the top 20 most intense ions. High-energy collision activated\",\n",
            "          \"the top 20 most intense ions. High-energy collision activated dissociation (HCD)-MS/MS was used to fragment peptides at a normalized collision energy of 27\\u2009eV in the presence of nitrogen. One LC-MS run was performed for each fraction (from 1 process\",\n",
            "          \"One LC-MS run was performed for each fraction (from 1 process technical replicate), except for WHIM2 and WHIM16 where 2 LC\\u2013MS runs were conducted (from 3 process technical replicates), resulting in the production of 100 LC\\u2013MS runs for global peptide\",\n",
            "          \"in the production of 100 LC\\u2013MS runs for global peptide analysis. Mass spectra were processed, and peptide identification was performed using the Andromeda search engine found in MaxQuant software ver. 1.5.0.25. (Max Planck Institute, Germany). All protein\",\n",
            "          \"ver. 1.5.0.25. (Max Planck Institute, Germany). All protein database searches were performed against the RefSeq database (version 20140707). Peptides were identified with a target-decoy approach using a combined database consisting of reverse protein\",\n",
            "          \"using a combined database consisting of reverse protein sequences of the RefSeq human, mouse and common repository of adventitious proteins (cRAP). The cRAP database was obtained from the Global Proteome Machine (ftp://ftp.thegpm.org/fasta/cRAP). Peptide\",\n",
            "          \"Proteome Machine (ftp://ftp.thegpm.org/fasta/cRAP). Peptide inference was made with a FDR of 1% while peptides were assigned to proteins with a protein FDR of 5%. A precursor ion mass tolerance of 20 ppm was used for the first search that allowed for m/z\",\n",
            "          \"of 20 ppm was used for the first search that allowed for m/z retention time recalibration of precursor ions that were then subjected to a main search using a precursor ion mass tolerance of 6 p.p.m. and a product ion mass tolerance 0.5\\u2009Da. Search\",\n",
            "          \"of 6 p.p.m. and a product ion mass tolerance 0.5\\u2009Da. Search parameters included up to two missed cleavages at KR on the sequence, and oxidation of methionine, and protein N terminus acetylation as a dynamic modification. Carbamidomethylation of cysteine\",\n",
            "          \"as a dynamic modification. Carbamidomethylation of cysteine residues was considered as a static modification. Peptide identifications are reported by filtering of reverse and contaminant entries and assigning to their leading razor protein according to\",\n",
            "          \"and assigning to their leading razor protein according to the Occams razor principal. The mass spectrometric data are deposited at the CPTAC Data Coordinating Center as raw and mzML files (https://cptac-data-portal.georgetown.edu).\",\n",
            "          \"Peptide and Protein Quantitation. LFQ was performed based on peak areas. The measured area under the curve of m/z and the retention time-aligned extracted ion chromatograms of peptides were performed via the label-free quantitation module in MaxQuant\",\n",
            "          \"performed via the label-free quantitation module in MaxQuant [ver. 1.5.0.25]. All replicates for each PDX were included in the LFQ experimental design with peptide-level quantitation performed using unique and razor peptide features corresponding to\",\n",
            "          \"using unique and razor peptide features corresponding to identifications filtered with a posterior error probability (PEP) of 0.01, peptide FDR of 0.01 and protein FDR of 0.05. The expression values were median centred in the Perseus software for further\",\n",
            "          \"values were median centred in the Perseus software for further analysis [version 1.5.0.9].\",\n",
            "          \"iTRAQ proteome and phosphoproteome\",\n",
            "          \"We included all 24 of the established breast tumour samples for proteomic characterization using iTRAQ. Tumour tissue samples were maintained in cryovials at \\u221280\\u2009\\u00b0C until cryopulverization using a CP02 Cryprep Pulverizer (Covaris, Woburn, MA). 90\\u2009mg\",\n",
            "          \"using a CP02 Cryprep Pulverizer (Covaris, Woburn, MA). 90\\u2009mg aliquots of cryofractured material were prepared for proteomic processing in aluminum weighing dishes on dry ice using spatulas kept cold in liquid nitrogen, with remaining material reserved for\",\n",
            "          \"cold in liquid nitrogen, with remaining material reserved for other applications. The 90\\u2009mg target was designed to include 40\\u2009mg for each of the collaborating research teams, with an anticipated yield for each team of 1.5\\u20132\\u2009mg protein based on 4\\u20135%\",\n",
            "          \"yield for each team of 1.5\\u20132\\u2009mg protein based on 4\\u20135% recovery. To avoid systematic bias, sample processing was block randomized, with each intrinsic subtype proportionally represented in each processing tranche.\",\n",
            "          \"The reproducibility of the iTRAQ4-plex global proteome and phosphoproteome analysis workflow used in this study has been extensively tested for quantitative reproducibility both within and across laboratories in the CPTAC program. Over a period of several\",\n",
            "          \"laboratories in the CPTAC program. Over a period of several months 5 iTRAQ4-plex replicates were measured at each of the 3 CPTAC proteome analysis centres. Each of these iTRAQ4-plexes contained duplicate measurements for both a basal WHIM2 and a luminal\",\n",
            "          \"duplicate measurements for both a basal WHIM2 and a luminal WHIM16 PDX samples that are also part of this study. A high degree of consistency in the number of proteins identified and correlation in the protein expression was obtained. Pearson correlations\",\n",
            "          \"in the protein expression was obtained. Pearson correlations for replicate proteome and phosphoproteome measurements were very high with a R=0.9 in our previous study and very similar to the correlation observed here for the WHIM13 replicate measurement.\",\n",
            "          \"correlation observed here for the WHIM13 replicate measurement. These data show that our platform provides highly reproducible quantitative measurements for global proteomes and phosphoproteomes.\",\n",
            "          \"Protein extraction, digestion and iTRAQ labelling of peptides from breast cancer tumours. Cryopulverized breast cancer tumour samples tissues (\\u223c2 combined aliquots of 90\\u2009mg tissue weight each) were homogenized in 1,000\\u2009\\u03bcl lysis buffer containing 8M urea,\",\n",
            "          \"were homogenized in 1,000\\u2009\\u03bcl lysis buffer containing 8M urea, 75\\u2009mM NaCl, 1\\u2009mM EDTA in 50\\u2009mM Tris HCl (pH 8), 10\\u2009mM NaF, phosphatase inhibitor cocktail 2 (1:100; Sigma, P5726) and cocktail 3 (1:100; Sigma, P0044), 2\\u2009\\u03bcg\\u2009ml\\u22121 aprotinin (Sigma, A6103),\",\n",
            "          \"3 (1:100; Sigma, P0044), 2\\u2009\\u03bcg\\u2009ml\\u22121 aprotinin (Sigma, A6103), 10\\u2009\\u03bcg\\u2009ml\\u22121 Leupeptin (Roche, #11017101001), and 1\\u2009mM PMSF (Sigma, 78830). Lysates were centrifuged at 20,000 g for 10\\u2009min before measuring protein concentration of the clarified lysates by BCA\",\n",
            "          \"measuring protein concentration of the clarified lysates by BCA assay (Pierce). Protein lysates were subsequently reduced with 5\\u2009mM dithiothreitol (Thermo Scientific, 20291) for 45\\u2009min at room temperature, and alkylated with 10\\u2009mM iodoacetamide (Sigma,\",\n",
            "          \"temperature, and alkylated with 10\\u2009mM iodoacetamide (Sigma, A3221) for 45\\u2009min in the dark. Samples were diluted 4-fold with 50\\u2009mM Tris HCl (pH 8) prior to digesting them with LysC (Wako, 129-02541) for 4\\u2009h and trypsin (Promega, V511X) overnight at a 1:50\",\n",
            "          \"for 4\\u2009h and trypsin (Promega, V511X) overnight at a 1:50 enzyme-to-protein ratio at room temperature overnight on a shaker.\",\n",
            "          \"Digested samples were acidified with formic acid (FA; Fluka, 56302) to a final volumetric concentration of 1% or final pH of \\u223c3\\u20135, and centrifuged at 2,000\\u2009g for 5\\u2009min to clear precipitated urea from peptide lysates. Samples were desalted on C18 SepPak\",\n",
            "          \"urea from peptide lysates. Samples were desalted on C18 SepPak columns (Waters, 100\\u2009mg, WAT036820) and 1\\u2009mg peptide aliquots were dried down using a SpeedVac apparatus.\",\n",
            "          \"Construction of the Common internal Reference Pool. The proteomic and phosphoproteomic analyses of xenograft samples were performed as iTRAQ 4-plex experiments. Quantitative comparison between all samples analyzed was facilitated by the use of iTRAQ\",\n",
            "          \"all samples analyzed was facilitated by the use of iTRAQ reporter ion ratios between each individual sample and a common internal reference sample present in each 4-plex. The reference sample was comprised of 16 of the 24 WHIM tumours analyzed in this\",\n",
            "          \"was comprised of 16 of the 24 WHIM tumours analyzed in this study with equal contribution for each tumour (WHIM numbers 2, 4, 6, 8, 11, 12, 13, 14, 16, 18, 20, 21, 24, 25, 30 and 46). The 24 tumour samples were analyzed in nine independent 4-plex\",\n",
            "          \"The 24 tumour samples were analyzed in nine independent 4-plex experiments, with three individual samples occupying the first three channels of each experiment and the 4th channel being reserved for the reference sample. While eight iTRAQ 4-plex\",\n",
            "          \"reserved for the reference sample. While eight iTRAQ 4-plex experiments were used to analyze the 24 individual WHIM tumour samples, an additional 4-plex experiment was designed to include the WHIM13 sample for process replicate analysis and also internal\",\n",
            "          \"WHIM13 sample for process replicate analysis and also internal reference samples from our human primary breast cancer study and a taxol drug response study (unpublished) to allow cross-referencing of the different datasets.\",\n",
            "          \"iTRAQ labelling, high pH reversed-phase separation and phosphopeptide enrichment of peptide samples. Desalted peptides were labelled with 4-plex iTRAQ reagents according to the manufacturer's instructions (AB Sciex, Foster City, CA). For each 1\\u2009mg peptide\",\n",
            "          \"instructions (AB Sciex, Foster City, CA). For each 1\\u2009mg peptide from each breast tumour sample, 10 units of labelling reagent were used. Peptides were dissolved in 300\\u2009\\u03bcl of 0.5 M triethylammonium bicarbonate (TEAB) (pH 8.5) solution and labelling reagent\",\n",
            "          \"bicarbonate (TEAB) (pH 8.5) solution and labelling reagent was added in 700\\u2009\\u03bcl of ethanol. After 1\\u2009h incubation, 1.5\\u2009ml of 0.05% TFA was added to stop the reaction. Differentially labelled peptides were mixed and subsequently desalted on 500\\u2009mg tC18\",\n",
            "          \"peptides were mixed and subsequently desalted on 500\\u2009mg tC18 SepPak columns. The combined 4\\u2009mg iTRAQ samples per experiment were separated into 24 proteome fractions and 12 phosphoproteome fractions using a 4.6\\u2009mm \\u00d7 250\\u2009mm column RP Zorbax 300 A ExtendC18\",\n",
            "          \"using a 4.6\\u2009mm \\u00d7 250\\u2009mm column RP Zorbax 300 A ExtendC18 column (Agilent, 3.5\\u2009\\u03bcm bead size) on an Agilent 1100 Series HPLC instrument by basic reversed-phase chromatography as described previously. Peptides were separated according to their hydrophobicity\",\n",
            "          \"Peptides were separated according to their hydrophobicity using solvent A (2% acetonitrile, 5\\u2009mM ammonium formate, pH 10) and a nonlinear increasing concentration of solvent B (90% acetonitrile, 5\\u2009mM ammonium formate, pH 10). Phosphopeptides were enriched\",\n",
            "          \"5\\u2009mM ammonium formate, pH 10). Phosphopeptides were enriched using Ni-NTA superflow agarose beads (Qiagen, #1018611) that were stripped of nickel with 100\\u2009mM EDTA and incubated in an aqueous solution of 10\\u2009mM FeCl3 (Sigma, 451649) as described previously.\",\n",
            "          \"of 10\\u2009mM FeCl3 (Sigma, 451649) as described previously. For phosphopeptide enrichment a 80% acetonitrile/0.1% trifluoroacetic acid binding buffer and a 500\\u2009mM dibasic sodium phosphate, pH 7.0, (Sigma, S9763) elution buffer were used. Enriched samples were\",\n",
            "          \"(Sigma, S9763) elution buffer were used. Enriched samples were desalted on StageTips as described before analysis by LC\\u2013MS/MS.\",\n",
            "          \"Analysis of tumour samples by high performance liquid chromatography tandem mass spectrometry (LC\\u2013MS/MS). All peptides were separated with an online nanoflow Proxeon EASY-nLC 1000 UHPLC system (Thermo Fisher Scientific) and analyzed on a benchtop Orbitrap\",\n",
            "          \"(Thermo Fisher Scientific) and analyzed on a benchtop Orbitrap Q Exactive mass spectrometer (Thermo Fisher Scientific) equipped with a nanoflow ionization source (James A. Hill Instrument Services, Arlington, MA, USA). The LC system, column, and platinum\",\n",
            "          \"Arlington, MA, USA). The LC system, column, and platinum wire to deliver electrospray source voltage were connected via a stainless-steel cross (360\\u2009\\u03bcm, IDEX Health & Science, UH-906x). The column was heated to 50\\u2009\\u00b0C using a column heater sleeve\",\n",
            "          \"The column was heated to 50\\u2009\\u00b0C using a column heater sleeve (Phoenix-ST) to prevent overpressurizing of columns during UHPLC separation. 10% of each global proteome sample in a 2\\u2009\\u03bcl injection volume, or 50% of each phosphoproteome sample in a 4 ul\",\n",
            "          \"volume, or 50% of each phosphoproteome sample in a 4 ul injection volume was injected onto an in-house packed 20\\u2009cm \\u00d7 75\\u2009um diameter C18 silica picofrit capillary column (1.9\\u2009\\u03bcm ReproSil-Pur C18-AQ beads, Dr Maisch GmbH, r119.aq; Picofrit 10\\u2009um tip\",\n",
            "          \"C18-AQ beads, Dr Maisch GmbH, r119.aq; Picofrit 10\\u2009um tip opening, New Objective, PF360-75-10-N-5). Mobile phase flow rate was 200\\u2009nl\\u2009min\\u22121, comprised of 3% acetonitrile/0.1% formic acid (Solvent A) and 90% acetonitrile /0.1% formic acid (Solvent B), and\",\n",
            "          \"A) and 90% acetonitrile /0.1% formic acid (Solvent B), and the 110-minute LC-MS/MS method consisted of a 10-min column-equilibration procedure, a 20-min sample-loading procedure, and the following gradient profile: (min:%B) 0:2; 1:6; 85:30; 94:60; 95;90;\",\n",
            "          \"gradient profile: (min:%B) 0:2; 1:6; 85:30; 94:60; 95;90; 100:90; 101:50; 110:50 (last two steps at 500\\u2009nl\\u2009min\\u22121 flowrate). Data-dependent acquisition was performed using Xcalibur QExactive v2.1 software in positive ion mode at a spray voltage of 2.00\\u2009kV.\",\n",
            "          \"software in positive ion mode at a spray voltage of 2.00\\u2009kV. MS1 Spectra were measured with a resolution of 70,000, an AGC target of 3e6 and a mass range from 300 to 1,800\\u2009m/z. Up to 12 MS2 spectra per duty cycle were triggered at a resolution of 17,500,\",\n",
            "          \"per duty cycle were triggered at a resolution of 17,500, an AGC target of 5e4, an isolation window of 2.5\\u2009m/z, a maximum ion time of 120\\u2009msec, and a normalized collision energy of 28. Peptides that triggered MS2 scans were dynamically excluded from\",\n",
            "          \"that triggered MS2 scans were dynamically excluded from further MS2 scans for 20\\u2009s. Charge state screening was enabled to reject precursor charge states that were unassigned, 1, or >6. Peptide match was enabled for monoisotopic precursor mass assignment.\",\n",
            "          \"Protein-peptide identification, phosphosite localization, and quantitation. All MS data were interpreted using the Spectrum Mill software package v5.1 (for comparison with proteomes of human breast tumours from our previous study) and v6.0 pre-release\",\n",
            "          \"breast tumours from our previous study) and v6.0 pre-release (Agilent Technologies, Santa Clara, CA, USA) co-developed by the authors. Similar MS/MS spectra acquired on the same precursor m/z within \\u00b1 45\\u2009s were merged. MS/MS spectra were excluded from\",\n",
            "          \"m/z within \\u00b1 45\\u2009s were merged. MS/MS spectra were excluded from searching if they failed the quality filter by not having a sequence tag length>0 (that is, minimum of two masses separated by the in-chain mass of an amino acid) or did not have a precursor\",\n",
            "          \"the in-chain mass of an amino acid) or did not have a precursor MH+ in the range of 750\\u20136,000. MS/MS spectra from were searched against a database consisting of RefSeq release 60 containing 31,767 human proteins, 24,821 mouse proteins, and an appended set\",\n",
            "          \"human proteins, 24,821 mouse proteins, and an appended set of 85 common laboratory contaminant proteins (RefSeq.20130727-Human.20130730-MouseNR.mm13.contams). Scoring parameters were ESI-QEXACTIVE-HCD-v2, for whole proteome datasets, and\",\n",
            "          \"were ESI-QEXACTIVE-HCD-v2, for whole proteome datasets, and ESI-QEXACTIVE-HCD-v3 parameters were for phosphoproteome datasets. All spectra were allowed \\u00b120 p.p.m. mass tolerance for precursor and product ions, 40% minimum matched peak intensity, and\",\n",
            "          \"and product ions, 40% minimum matched peak intensity, and trypsin allow P enzyme specificity with up to four missed cleavages. Fixed modifications were carbamidomethylation at cysteine. iTRAQ labelling was required at lysine, but peptide N-termini were\",\n",
            "          \"labelling was required at lysine, but peptide N-termini were allowed to be either labelled or unlabelled. Allowed variable modifications for whole proteome datasets were acetylation of protein N-termini, oxidized methionine, deamidation of asparagine,\",\n",
            "          \"N-termini, oxidized methionine, deamidation of asparagine, pyro-glutamic acid at peptide N-terminal glutamine, and pyro-carbamidomethylation at peptide N-terminal cysteine with a precursor MH+ shift range of \\u221218 to 64\\u2009Da. Allowed variable modifications\",\n",
            "          \"MH+ shift range of \\u221218 to 64\\u2009Da. Allowed variable modifications for phosphoproteome dataset were revised to disallow deamidation and allow phosphorylation of serine, threonine, and tyrosine with a precursor MH+ shift range of 0 to 272\\u2009Da.\",\n",
            "          \"Identities interpreted for individual spectra were automatically designated as confidently assigned using the Spectrum Mill autovalidation module to use target-decoy based FDR estimates to apply score threshold criteria via two-step strategies. For the\",\n",
            "          \"apply score threshold criteria via two-step strategies. For the whole proteome datasets thresholding was done at the spectral and protein levels. For the phosphoproteome datasets thresholding was done at the spectral level. In step 1, peptide\",\n",
            "          \"thresholding was done at the spectral level. In step 1, peptide autovalidation was done first and separately for each iTRAQ 4-plex experiment consisting of either 25 LC\\u2013MS/MS runs (whole proteome) or 13 LC\\u2013MS/MS runs (phosphoproteome) using an auto\",\n",
            "          \"proteome) or 13 LC\\u2013MS/MS runs (phosphoproteome) using an auto thresholds strategy with a minimum sequence length of 7(whole proteome) or 8 (phosphoproteome), automatic variable range precursor mass filtering, and score and delta Rank1\\u2013Rank2 score\",\n",
            "          \"precursor mass filtering, and score and delta Rank1\\u2013Rank2 score thresholds optimized to yield a spectral level FDR estimate for precursor charges 2 through 4 of <0.6% for each precursor charge state in each LC\\u2013MS/MS run. For precursor charges 5\\u20136,\",\n",
            "          \"charge state in each LC\\u2013MS/MS run. For precursor charges 5\\u20136, thresholds were optimized to yield a spectral level FDR estimate of <0.3% across all runs per iTRAQ 4-plex experiment (instead of each run), to achieve reasonable statistics, since many fewer\",\n",
            "          \"each run), to achieve reasonable statistics, since many fewer spectra are generated for the higher charge states.\",\n",
            "          \"In step 2 for the whole proteome datasets, protein polishing autovalidation was applied separately to each iTRAQ 4-plex experiment to further filter the PSM's using a target protein-level FDR threshold of zero. The primary goal of this step was to\",\n",
            "          \"FDR threshold of zero. The primary goal of this step was to eliminate peptides identified with low scoring peptide spectrum matches (PSM's) that represent proteins identified by a single peptide, so-called \\u2018one-hit wonders'. After assembling protein\",\n",
            "          \"peptide, so-called \\u2018one-hit wonders'. After assembling protein groups from the autovalidated PSM's, protein polishing determined the maximum protein level score of a protein group that consisted entirely of distinct peptides estimated to be false-positive\",\n",
            "          \"entirely of distinct peptides estimated to be false-positive identifications (PSM's with negative delta forward-reverse scores). PSM's were removed from the set obtained in the initial peptide-level autovalidation step if they contributed to protein\",\n",
            "          \"autovalidation step if they contributed to protein groups that have protein scores below the max false-positive protein score. In the filtered results each identified protein detected in an iTRAQ 4-plex experiment was comprised of multiple peptides unless\",\n",
            "          \"4-plex experiment was comprised of multiple peptides unless a single excellent scoring peptide was the sole match. For the whole proteome datasets the above criteria yielded FDR of <0.5% at the peptide-spectrum match level and <0.8% at the distinct\",\n",
            "          \"at the peptide-spectrum match level and <0.8% at the distinct peptide level for each iTRAQ 4-plex experiment. After assembling proteins with all the PSMs from all the iTRAQ 4-plex experiments together the aggregate FDR estimates were 0.42% at the at the\",\n",
            "          \"together the aggregate FDR estimates were 0.42% at the at the peptide-spectrum match level, 1.5% at the distinct peptide level, and <0.01% (1/11,372) at the protein group level. Since the protein level FDR estimate neither explicitly required a minimum\",\n",
            "          \"level FDR estimate neither explicitly required a minimum number of distinct peptides per protein nor adjusted for the number of possible tryptic peptides per protein, it may underestimate false positive protein identifications for large proteins observed\",\n",
            "          \"positive protein identifications for large proteins observed only on the basis of multiple low scoring PSMs.\",\n",
            "          \"In calculating scores at the protein level and reporting the identified proteins, redundancy was addressed in the following manner: the protein score was the sum of the scores of distinct peptides. A distinct peptide was the single highest scoring\",\n",
            "          \"peptides. A distinct peptide was the single highest scoring instance of a peptide detected through an MS/MS spectrum. MS/MS spectra for a particular peptide may have been recorded multiple times, (that is, as different precursor charge states, in adjacent\",\n",
            "          \"(that is, as different precursor charge states, in adjacent bRP fractions, modified by deamidation at Asn or oxidation of Met, or different phosphosite localization) but were still counted as a single distinct peptide. When a peptide sequence >8 residues\",\n",
            "          \"a single distinct peptide. When a peptide sequence >8 residues long was contained in multiple protein entries in the sequence database, the proteins were grouped together and the highest scoring one and its accession number were reported. In some cases\",\n",
            "          \"one and its accession number were reported. In some cases when the protein sequences were grouped in this manner there were distinct peptides that uniquely represented a lower scoring member of the group (isoforms, family members, and different species).\",\n",
            "          \"of the group (isoforms, family members, and different species). Each of these instances spawned a subgroup. Multiple subgroups were reported and counted towards the total number of proteins, and were given related protein subgroup numbers (for example,\",\n",
            "          \"and were given related protein subgroup numbers (for example, 3.1 and 3.2: group 3, subgroups 1 and 2). To better dissect the tumour/stroma (human/mouse) origin of orthologous proteins in this xenograft experiment, the inclusion of peptides contributing\",\n",
            "          \"xenograft experiment, the inclusion of peptides contributing to each subgroup was restricted by enabling the subgroup-specific (SGS) option in Spectrum Mill. Only subgroup-specific peptide sequences were counted toward each subgroup's count of distinct\",\n",
            "          \"sequences were counted toward each subgroup's count of distinct peptides and protein level TMT quantitation. The SGS option omits peptides that are shared between subgroups. If evidence for BOTH human and mouse peptides from an orthologous protein were\",\n",
            "          \"BOTH human and mouse peptides from an orthologous protein were observed, then peptides that cannot distinguish the two (shared) were ignored. However, the peptides shared between species were retained if there was specific evidence for only one of the\",\n",
            "          \"retained if there was specific evidence for only one of the species, thus yielding a single subgroup attributed to only the single species consistent with the specific peptides. Furthermore, if all peptides observed for a protein group were shared between\",\n",
            "          \"all peptides observed for a protein group were shared between species, thus yielding a single subgroup composed of indistinguishable species, then all peptides were retained (the column in Supplementary Data 3 numSpeciesPresentR1 will have a value of 2 in\",\n",
            "          \"Data 3 numSpeciesPresentR1 will have a value of 2 in such cases). Assembly of confidently identified PSM's yielded 20,480 total protein subgroups from 11,372 protein groups. Human and mouse ortholog proteins were typically arranged into individual\",\n",
            "          \"mouse ortholog proteins were typically arranged into individual subgroups.\",\n",
            "          \"In step 2 for the phosphoproteome datasets a phosphosite table were assembled with columns for individual iTRAQ 4-plex experiments and rows for individual phosphosites. PSM's were combined into a single row for all non-conflicting observations of a\",\n",
            "          \"into a single row for all non-conflicting observations of a particular phosphosite. (that is, different missed cleavage forms, different precursor charges, confident and ambiguous localizations, different sample handling modifications). For related\",\n",
            "          \"different sample handling modifications). For related peptides neither observations with a different number of phosphosites nor different confident localizations were allowed to be combined. Selecting the representative peptide from the combined\",\n",
            "          \"Selecting the representative peptide from the combined observations was done such that once confident phosphosite localization was established, higher identification scores and longer peptide lengths are preferable. After assembling the phosphosite table\",\n",
            "          \"lengths are preferable. After assembling the phosphosite table a polishing step was applied to further filter the phosphosites with the primary goal of eliminating phosphosites with representative peptides identified through low scoring peptide spectrum\",\n",
            "          \"peptides identified through low scoring peptide spectrum matches (PSM's) that were observed in only a few experiments. The initial table of representative peptides for 82,030 phosphosites had an aggregate FDR of 3.3% at phosphosite-level. The table was\",\n",
            "          \"an aggregate FDR of 3.3% at phosphosite-level. The table was sorted by identification score and then by number of iTRAQ 4-plex experiments in which the phosphosite was observed. The cumulative FDR trend showed inflection points at an identification score\",\n",
            "          \"FDR trend showed inflection points at an identification score of \\u223c8. Phosphosites with an identification score<8.0 observed in <3/9 experiments were therefore removed, yielding 68,385 phosphosites with an aggregate FDR of 0.34% at the phosphosite level.\",\n",
            "          \"with an aggregate FDR of 0.34% at the phosphosite level. While the Spectrum Mill identification score is based on the number of matching peaks, their ion type assignment, and the relative height of unmatched peaks, the phosphosite localization score is\",\n",
            "          \"of unmatched peaks, the phosphosite localization score is the difference in identification score between the top two localizations. The score threshold for confident localization (>1.1) essentially corresponds to at least 1 b or y ion located between two\",\n",
            "          \"corresponds to at least 1 b or y ion located between two candidate sites that has a peak height 10% of the tallest fragment ion (neutral losses of phosphate from the precursor and related ions as well as immonium and iTRAQ reporter ions are excluded from\",\n",
            "          \"as well as immonium and iTRAQ reporter ions are excluded from the relative height calculation). The ion type scores for b-H3PO4, y-H3PO4, b-H2O, and y-H2O ion types are all set to 0.5. This prevents inappropriate confident localization assignment when a\",\n",
            "          \"prevents inappropriate confident localization assignment when a spectrum lacks primary b or y ions between two possible sites but contains ions that can be assigned as either phosphate loss ions for one localization or water loss ions for another\",\n",
            "          \"loss ions for one localization or water loss ions for another localization. In aggregate, 66.3% of the reported phosphosites were fully localized to a particular serine, threonine, or tyrosine residue.\",\n",
            "          \"Relative abundances of proteins and phosphosites were determined in Spectrum Mill using iTRAQ reporter ion intensity ratios from each PSM. A protein-level or phosphosite-level iTRAQ ratio was calculated as the median of all PSM level ratios contributing\",\n",
            "          \"calculated as the median of all PSM level ratios contributing to a protein subgroup or phosphosite remaining after excluding those PSM's lacking an iTRAQ label, having a negative delta forward-reverse score (half of all false-positive identifications), or\",\n",
            "          \"score (half of all false-positive identifications), or having a precursor ion purity<50% (MS/MS has significant precursor isolation contamination from co-eluting peptides). Unless stated otherwise for a particular analysis, the following considerations\",\n",
            "          \"for a particular analysis, the following considerations apply to the tumour/stroma (human/mouse) origin of a protein in this xenograft experiment. For the proteome dataset, only PSM's from subgroup-specific peptide sequences contributed to the protein\",\n",
            "          \"subgroup-specific peptide sequences contributed to the protein level quantitation (see protein subgrouping description above). A protein detected with all contributing PSM's shared between human and mouse was considered to be human. For the\",\n",
            "          \"between human and mouse was considered to be human. For the phosphoproteome dataset, a phosphosite was considered to be mouse if the contributing PSM's were distinctly mouse and human if they were either distinctly human or shared between human and mouse.\",\n",
            "          \"were either distinctly human or shared between human and mouse. A 2-component Gaussian mixture model-based normalization approach was used to centre the distribution of iTRAQ log-ratios around zero to nullify the effect of differential protein loading\",\n",
            "          \"zero to nullify the effect of differential protein loading and/or systematic MS variation. Downstream analyses presented in the main figures were restricted to proteins/phosphosites quantified in at least 10 out of the 24 samples with non-missing values\",\n",
            "          \"in at least 10 out of the 24 samples with non-missing values (Supplementary Fig. 3), with the exception of the previously described mRNA-protein correlation analysis requiring quantification in 30%, or 8 out of 24, PDX samples. Specific filtering\",\n",
            "          \"in 30%, or 8 out of 24, PDX samples. Specific filtering procedures are noted in descriptions of the relevant methods.\",\n",
            "          \"Bioinformatics analyses.\\n\\nCross data-type integration\",\n",
            "          \"All gene names were converted to HUGO Gene Nomenclature Committee's approved gene names for comparison across levels and datasets. For mRNA and protein expressions, expression values were collapsed across transcripts or isoforms to the corresponding gene\",\n",
            "          \"across transcripts or isoforms to the corresponding gene using the highest mean when there were two transcripts or isoforms, or the value with the highest connectivity when there were more than three transcripts or isoforms as implemented in the WGCNA R\",\n",
            "          \"three transcripts or isoforms as implemented in the WGCNA R package.\",\n",
            "          \"mRNA-protein correlation\",\n",
            "          \"Spearman correlations between normalized RSEM values and protein quantifications were calculated for genes that were observed in at least 30% of samples for both RNA-seq and mass-spectrometry as previously described. KEGG pathway enrichment analysis of\",\n",
            "          \"as previously described. KEGG pathway enrichment analysis of the correlation was carried out using a Kolmogorov\\u2013Smirnov test, and P values were adjusted using the Benjamini-Hochberg procedure.\",\n",
            "          \"Proteomic clustering and subtyping\",\n",
            "          \"We first applied filtering for protein and phosphosite markers observed in at least 10 samples and sufficient deviations across samples. We used a 2 s.d. threshold for iTRAQ proteome and 2.5 s.d. for the phosphoproteome. We applied the same protein marker\",\n",
            "          \"for the phosphoproteome. We applied the same protein marker to conduct LFQ proteome and PDX vs. human proteome co-clustering. For the co-clustering with human proteome, we further selected for markers that had higher than 2 s.d.'s in the merged proteome\",\n",
            "          \"markers that had higher than 2 s.d.'s in the merged proteome and showed non-differential expression between human and PDX (FDR>0.3, t-test). The subsequent hierarchical clustering was conducted using the complete agglomeration method of hclust as\",\n",
            "          \"conducted using the complete agglomeration method of hclust as implemented in the heatmap2 R package.\",\n",
            "          \"Differential expression analysis\",\n",
            "          \"Differential expression testing of each protein in the LFQ and iTRAQ datasets was conducted using the student's t-test, and P values were adjusted using the Benjamini-Hochberg procedure. For gene set enrichment analysis, we conducted the Wilcoxon Rank Sum\",\n",
            "          \"set enrichment analysis, we conducted the Wilcoxon Rank Sum Test to test for changes in the t-statistics ranks of protein members in each of the KEGG signalling pathway, and again adjusted P values using the Benjamini-Hochberg procedure.\",\n",
            "          \"Druggable genes and mutations\",\n",
            "          \"We compiled a list of druggable genes based on extensive curation of public databases: Tumor Alterations Relevant for GEnomics-driven Therapy (TARGET, version 3, assessed on 6/15/2015), Personalized Cancer Therapy (PCT, assessed on 3/15/2015), GDKD\",\n",
            "          \"Personalized Cancer Therapy (PCT, assessed on 3/15/2015), GDKD (Gene-Drug Knowledge Database, version 11.0, assessed on 4/10/2015), CancerDR (assessed on 2/6/2015), My Cancer Genome (assessed on 9/11/2014), and DrugBank (assessed on 9/21/2015). We curated\",\n",
            "          \"on 9/11/2014), and DrugBank (assessed on 9/21/2015). We curated the list based on evidence level and literature, as well as IC50 data when available. The final list used for the analysis included 76 druggable genes (Supplementary Data 15).\",\n",
            "          \"Druggable outlier analysis\",\n",
            "          \"To discover expression outliers, we utilized a strategy incorporating multiple steps: first, we limited our search to genes that are in the druggable gene list. We then narrowed down the list to genes that are observed in at least 10 samples in the\",\n",
            "          \"list to genes that are observed in at least 10 samples in the dataset under investigation. Outlier expressions were defined as values that are greater than 1.5 interquartile ranges (IQRs) above the third quartile (Q3). To rank order outlier expression for\",\n",
            "          \"the third quartile (Q3). To rank order outlier expression for each gene, we calculated an outlier score defined as:\",\n",
            "          \"By definition, genes with outlier score greater than 1.5 are considered as expression outliers. Outlier score for each gene were ranked within the sample to select the most promising druggable targets. For CNV outliers, we required them to have outlier\",\n",
            "          \"targets. For CNV outliers, we required them to have outlier scores above 1 in at least another expression level. For validation of the proteomic druggable outliers, we counted the numbers of the same protein or phosphosite outliers observed in the\",\n",
            "          \"of the same protein or phosphosite outliers observed in the parallel-processed cohort of 77 human breast cancer samples. Due to the lower numbers in this larger cohort, we considered both proteins with outlier score greater than 1 and the top 2 outliers\",\n",
            "          \"with outlier score greater than 1 and the top 2 outliers of each human sample as validating outliers.\",\n",
            "          \"Pathway activation analysis\",\n",
            "          \"We first collapsed the phosphosites to gene-level phosphorylation values by averaging the phosphosite expressions observed for each gene. Then, we converted the phosphoproteomic expression values from iTRAQ to modified z-scores normalized against the\",\n",
            "          \"values from iTRAQ to modified z-scores normalized against the cohort as described in Hoaglin et al. (How to Detect and Handle Outliers). We then used the Wilcoxon Rank Sum Test to test for changes in the phosphorylation z-score ranks of protein members in\",\n",
            "          \"in the phosphorylation z-score ranks of protein members in each of the KEGG signalling pathway. The resulting P values were adjusted using the Benjamini\\u2013Hochberg procedure.\",\n",
            "          \"Data availability\\n\\nAll mass spectra, in the original instrument vendor format, contributing to this study may be downloaded from: https://cptac-data-portal.georgetown.edu/cptacPublic/ for the study name: TCGA Breast Cancer.\\n\\nAdditional information\",\n",
            "          \"Additional information\\n\\nHow to cite this article: Huang, K.-l. et al. Proteogenomic integration reveals therapeutic targets in breast cancer xenografts. Nat. Commun. 8, 14864 doi: 10.1038/ncomms14864 (2017).\",\n",
            "          \"Publisher's note: Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\\n\\nSupplementary Material\",\n",
            "          \"Supplementary Material\\n\\nC.M. Perou is on the board of directors, has ownership interest (including patents), and is a consultant/advisory board member for bioclassifier LLC. The other authors declare no competing financial interests.\",\n",
            "          \"Author contributions L.D. directed data analysis. L.D., M.J.E., S.A.C., X.C., H.R. and R.R.T. designed and supervised research and experiment. K.-l.H., S.L., P.M., H.P.G., S.C., C.Y., D.R.M., K.R.C., C.R.K., J.W., K.V.R., D.F., M.D.M., S.M.K., S.S. and\",\n",
            "          \"K.R.C., C.R.K., J.W., K.V.R., D.F., M.D.M., S.M.K., S.S. and B.N. analyzed the data. K.-l.H., M.C.W. and D.R.M. performed statistical analysis. S.L. established the PDX models. S.L., Z.G., S.M.K., M.T., J.U., J.E.S., J.H., P.E.-G., P.S., C.M. and C.M.P.\",\n",
            "          \"M.T., J.U., J.E.S., J.H., P.E.-G., P.S., C.M. and C.M.P. performed functional validation experiments. J.W.Q., P.M., C.M., L.X. and H.P.G. performed data collection and tissue acquisition. M.A.W., K.-l.H., C.Y., S.C. and L.D. prepared figures and tables.\",\n",
            "          \"K.-l.H., C.Y., S.C. and L.D. prepared figures and tables. M.J.E., S.R.D. and S.L. provided samples. K.-l.H. and L.D. wrote the manuscript. M.J.E., M.C.W., P.M., S.A.C., X.C., R.R.T., C.M., K.-l.H. and L.D. revised the manuscript. All authors approved and\",\n",
            "          \"and L.D. revised the manuscript. All authors approved and contributed to the preparation of the manuscript.\",\n",
            "          \"04/25/2017\\n\\nThe original version of this Article contained a typographical error in the spelling of the author Beifang Niu, which was incorrectly given as Beifung Niu. This has now been corrected in both the PDF and HTML versions of the Article.\",\n",
            "          \"Modelling human breast cancer with patient-derived xenografts (n=24).\",\n",
            "          \"(a) Illustration of generation and proteogenomic characterization of breast cancer xenograft models. (b) Somatic mutations of significantly mutated genes of human breast tumour were recapitulated in xenograft models. Mutation data for 23 WHIMs are shown\",\n",
            "          \"in xenograft models. Mutation data for 23 WHIMs are shown (exome data were not available for WHIM47). (c) Variant allele fraction analysis showed clonal representation was consistent between human breast tumour and xenografts. Genomic driver events,\",\n",
            "          \"human breast tumour and xenografts. Genomic driver events, including missenses and truncations in TP53 and PIK3CA, were retained in the xenograft models. Each colour represents one xenograft sample.\",\n",
            "          \"Proteogenomic correlation analysis in PDX samples.\",\n",
            "          \"(a) Correlation between mRNA and iTRAQ protein expression levels identified pathways with significantly concordant or discordant mRNA-protein expressions. Genes were aligned along the x axis by the rank of their Spearman correlation coefficient between\",\n",
            "          \"by the rank of their Spearman correlation coefficient between mRNA and protein expression levels. Each colour represents one significantly associated pathway, and each bar represents one gene in the pathway. (b) Proteogenomic summary of xenograft shows\",\n",
            "          \"in the pathway. (b) Proteogenomic summary of xenograft shows relationships among mutation, CNV (normalized log-R ratio), mRNA (log-transformed and normalized RSEM values), proteomic (normalized log2 ratio to reference), and phosphoproteomic expression\",\n",
            "          \"log2 ratio to reference), and phosphoproteomic expression (normalized log2 ratio to reference) levels of breast cancer-related genes in 24 PDX samples across 4 intrinsic subtypes. Expression values from each dataset were calculated as described (Methods)\",\n",
            "          \"values from each dataset were calculated as described (Methods) and truncated to a maximum of 10 and a minimum of \\u221210 for visualization.\",\n",
            "          \"Transcriptomic and proteomic clustering of breast cancer PDX and human samples.\",\n",
            "          \"(a) Transcriptomic clustering of PDX breast tumours based on the PAM50 gene expression markers. (b) Proteomic clustering of PDX breast tumours based on the top 436 variably expressed proteins. (c) Phosphoproteomic clustering of PDX breast tumours based on\",\n",
            "          \"(c) Phosphoproteomic clustering of PDX breast tumours based on the top 1,737 variably expressed phosphosites. (d) Proteomic clustering using only 133 non-differential expressed proteins between WHIM and human breast tumour samples. The clustering\",\n",
            "          \"between WHIM and human breast tumour samples. The clustering reproduced the basal-enriched and luminal-enriched clusters, where PDX (n=24) and TCGA human breast tumour samples (n=77) cluster based on their subtypes. The non-differentially expressed\",\n",
            "          \"based on their subtypes. The non-differentially expressed proteins were identified through a t-test with FDR>0.3 between the PDX and the TCGA human tumour samples. The PDX tumours are labelled by their WHIM IDs whereas the human tumours are not labelled.\",\n",
            "          \"Activated signalling pathways detected through pathway phosphorylation enrichment analysis.\",\n",
            "          \"(a) Activation of the MAPK signalling pathway in WHIM9. (b) Activation of the Ras signalling pathway in WHIM12. (c) Activation of the NF\\u03baB pathway in WHIM17. (d) Activation of the NF\\u03baB pathway in WHIM46. Phosphorylation levels of each protein in the\",\n",
            "          \"in WHIM46. Phosphorylation levels of each protein in the pathway relative to the cohort of 24 PDX models are shown by the colour scale of red (high) to blue (low). Proteins with no phosphorylation data are coloured in green.\",\n",
            "          \"Druggable events identified by expression outlier analysis.\",\n",
            "          \"Druggable outlier events identified at (a) the mRNA and protein and (b) protein and phosphopeptide expression levels. Each colour represents one xenograft sample. Key outlier events validated in this study are labelled by text. (c) Immunochemistry\",\n",
            "          \"in this study are labelled by text. (c) Immunochemistry staining verified outlier expression of HER2 in WHIM8 and WHIM35, RAF1 in WHIM9, and FGFR2 in WHIM16. Scale bar: 50\\u2009\\u03bcm.\",\n",
            "          \"Druggable targets identified through proteogenomic analysis in PDX and human breast cancer.\",\n",
            "          \"(a) Outlier analysis revealed potentially druggable events in the RTK, PI3K, MAPK signalling, genome integrity pathways at various frequency and magnitudes across four breast cancer subtypes. Selected genes with any outlier score greater than 2.5 or in\",\n",
            "          \"Selected genes with any outlier score greater than 2.5 or in the key oncogenic pathways, including the PI3K, RTK, MAPK signalling pathways, are shown. (b) Comparison of protein expression outliers of selected druggable genes in PDX and human breast\",\n",
            "          \"outliers of selected druggable genes in PDX and human breast tumours. (c) Comparison of overexpressed phosphosite outliers of selected druggable genes in PDX and human breast tumours. Key outlier events reaching the outlier definition threshold or\",\n",
            "          \"Key outlier events reaching the outlier definition threshold or validated in this study are labelled by text. For the box plots in (b) and (c), the center line indicates median of the protein/phosphosite expression in the human and PDX cohort. The upper\",\n",
            "          \"expression in the human and PDX cohort. The upper and lower hinges correspond to the 25th and 75th percentiles; the upper whisker corresponds to 1.5 \\u00d7 IQR (inter-quartile range) above the 25 percentile and the lower whisker corresponds to 1.5 \\u00d7 IQR below\",\n",
            "          \"percentile and the lower whisker corresponds to 1.5 \\u00d7 IQR below the 75 percentile.\",\n",
            "          \"Targeted treatments of breast cancer xenografts.\",\n",
            "          \"(a) Western blot of HER2 protein and HER2 p.Y1248 expression levels in 5 WHIM models (WHIM6, WHIM8, WHIM12, WHIM14 and WHIM35). (b) In vivo treatment responses to lapatinib in 4 PDX models including two HER2 positive lines (WHIM8 and WHIM35) and two basal\",\n",
            "          \"two HER2 positive lines (WHIM8 and WHIM35) and two basal lines (WHIM6 and WHIM14). The response is measured in fold change (%) of tumour volumes after 2 weeks of vehicle or lapatinib treatment. (c) Immunochemistry staining of AKT phosphorylation status in\",\n",
            "          \"(c) Immunochemistry staining of AKT phosphorylation status in WHIM16, WHIM18 and WHIM20 in response to PI3K inhibitor TAK-117, mTOR inhibitor TAK128, and TAK-117/TAK-128 combined. (d) In vivo treatment responses to PI3K/mTOR inhibitors (TAK-117/TAK-128)\",\n",
            "          \"treatment responses to PI3K/mTOR inhibitors (TAK-117/TAK-128) of WHIM16, WHIM18 and WHIM20 (n=8-9 for each control or experimental group). Values were represented by average tumour volume (mm3) every 3 days following treatment (error bars: s.e.m.).\"\n",
            "        ]\n",
            "      }\n",
            "    ]\n",
            "  },\n",
            "  \"PDC000303\": {\n",
            "    \"study_id\": \"f0f337b3-563b-4335-83d8-7e65146102a0\",\n",
            "    \"pdc_study_id\": \"PDC000303\",\n",
            "    \"study_submitter_id\": \"Therapeutic Targets in Breast Cancer Xenografts -  Proteome\",\n",
            "    \"program_id\": \"10251935-5540-11e8-b664-00a098d917f8\",\n",
            "    \"project_id\": \"8dbcf809-3df2-4c57-a9d5-8d58d5824ffa\",\n",
            "    \"study_name\": \"Therapeutic Targets in Breast Cancer Xenografts -  Proteome\",\n",
            "    \"program_name\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"project_name\": \"CPTAC2-Other\",\n",
            "    \"disease_type\": \"Other\",\n",
            "    \"primary_site\": \"Not Reported\",\n",
            "    \"analytical_fraction\": \"Proteome\",\n",
            "    \"experiment_type\": \"iTRAQ4\",\n",
            "    \"embargo_date\": null,\n",
            "    \"cases_count\": 27,\n",
            "    \"aliquots_count\": 27,\n",
            "    \"protocol_id\": \"93e99c96-49ac-40ee-9466-b113f4c62cf8\",\n",
            "    \"protocol_submitter_id\": \"Breast Cancer Xenografts -  Proteome\",\n",
            "    \"program_submitter_id\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"protocol_name\": \"Breast Cancer Xenografts -  Proteome\",\n",
            "    \"protocol_date\": \"2021-05-06\",\n",
            "    \"document_name\": null,\n",
            "    \"quantitation_strategy\": \"Isobaric label quantitation analysis\",\n",
            "    \"label_free_quantitation\": \"N/A\",\n",
            "    \"labeled_quantitation\": \"iTRAQ\",\n",
            "    \"isobaric_labeling_reagent\": \"iTRAQ4\",\n",
            "    \"reporter_ion_ms_level\": \"MS2\",\n",
            "    \"starting_amount\": \"2.00\",\n",
            "    \"starting_amount_uom\": \"mg\",\n",
            "    \"digestion_reagent\": \"Lys-C,Trypsin\",\n",
            "    \"alkylation_reagent\": \"Iodoacetamide (IAA)\",\n",
            "    \"enrichment_strategy\": \"Proteome\",\n",
            "    \"enrichment\": \"None\",\n",
            "    \"chromatography_dimensions_count\": \"2\",\n",
            "    \"one_d_chromatography_type\": \"bRPLC\",\n",
            "    \"two_d_chromatography_type\": \"RPLC\",\n",
            "    \"fractions_analyzed_count\": \"24\",\n",
            "    \"column_type\": \"C18\",\n",
            "    \"amount_on_column\": null,\n",
            "    \"amount_on_column_uom\": null,\n",
            "    \"column_length\": \"20.0\",\n",
            "    \"column_length_uom\": \"cm\",\n",
            "    \"column_inner_diameter\": \"75.00\",\n",
            "    \"column_inner_diameter_uom\": \"\\u00b5m\",\n",
            "    \"particle_size\": \"1.9\",\n",
            "    \"particle_size_uom\": \"\\u00b5m\",\n",
            "    \"particle_type\": \"ReproSil-Pur C18\",\n",
            "    \"gradient_length\": \"110.0\",\n",
            "    \"gradient_length_uom\": \"min\",\n",
            "    \"instrument_make\": \"Thermo Fisher Scientific\",\n",
            "    \"instrument_model\": \"Q Exactive\",\n",
            "    \"dissociation_type\": \"HCD\",\n",
            "    \"ms1_resolution\": \"70000\",\n",
            "    \"ms2_resolution\": \"17500\",\n",
            "    \"dda_topn\": \"Top 12\",\n",
            "    \"normalized_collision_energy\": \"28\",\n",
            "    \"acquistion_type\": \"DDA\",\n",
            "    \"dia_multiplexing\": \"N/A\",\n",
            "    \"dia_ims\": \"N/A\",\n",
            "    \"analytical_technique\": null,\n",
            "    \"chromatography_instrument_make\": null,\n",
            "    \"chromatography_instrument_model\": null,\n",
            "    \"polarity\": null,\n",
            "    \"reconstitution_solvent\": null,\n",
            "    \"reconstitution_volume\": null,\n",
            "    \"reconstitution_volume_uom\": null,\n",
            "    \"internal_standards\": null,\n",
            "    \"extraction_method\": null,\n",
            "    \"ionization_mode\": null,\n",
            "    \"publications\": [\n",
            "      {\n",
            "        \"publication_id\": \"718f77fb-b329-11eb-9f63-0a2b20c3a9e9\",\n",
            "        \"pubmed_id\": \"28348404\",\n",
            "        \"doi\": \"10.1038/ncomms14864\",\n",
            "        \"author\": \"Huang, K. L.; Li, S.; Mertins, P.; Cao, S.; Gunawardena, H. P.; Ruggles, K. V.; Mani, D. R.; Clauser, K. R.; Tanioka, M.; Usary, J.; Kavuri, S. M.; Xie, L.; Yoon, C.; Qiao, J. W.; Wrobel, J.; Wyczalkowski, M. A.; Erdmann-Gilmore, P.; Snider, J. E.; Hoog, J.; Singh, P.; Niu, B.; Guo, Z.; Sun, S.Q.; Sanati, S.; Kawaler, E.; Wang, X.; Scott, A.; Ye, K.; McLellan, M.D.; Wendl, M.C.; Malovannaya, A.; Held, J.M.; Gillette, M.A.; Feny\\u221a\\u00c9\\u00ac\\u2202, D.; Kinsinger, C.R.; Mesri, M.; Rodriguez, H.; Davies, S.R.; Perou, C.M.; Ma, C.; Reid Townsend, R.; Chen, X.; Carr, S.A.; Ellis, M.J.; Ding L.\",\n",
            "        \"title\": \"Proteogenomic integration reveals therapeutic targets in breast cancer xenografts\",\n",
            "        \"journal\": \"Nature Communications\",\n",
            "        \"journal_url\": \"https://www.nature.com/articles/ncomms14864\",\n",
            "        \"year\": \"2017\",\n",
            "        \"abstract\": \"Recent advances in mass spectrometry (MS) have enabled extensive analysis of cancer proteomes. Here, we employed quantitative proteomics to profile protein expression across 24 breast cancer patient-derived xenograft (PDX) models. Integrated proteogenomic analysis shows positive correlation between expression measurements from transcriptomic and proteomic analyses; further, gene expression-based intrinsic subtypes are largely re-capitulated using non-stromal protein markers. Proteogenomic analysis also validates a number of predicted genomic targets in multiple receptor tyrosine kinases. However, several protein/phosphoprotein events such as overexpression of AKT proteins and ARAF, BRAF, HSP90AB1 phosphosites are not readily explainable by genomic analysis, suggesting that druggable translational and/or post-translational regulatory events may be uniquely diagnosed by MS. Drug treatment experiments targeting HER2 and components of the PI3K pathway supported proteogenomic response predictions in seven xenograft models. Our study demonstrates that MS-based proteomics can identify therapeutic targets and highlights the potential of PDX drug response evaluation to annotate MS-based pathway activities.\",\n",
            "        \"full_text\": [\n",
            "          \"Proteogenomic integration reveals therapeutic targets in breast cancer xenografts\",\n",
            "          \"Recent advances in mass spectrometry (MS) have enabled extensive analysis of cancer proteomes. Here, we employed quantitative proteomics to profile protein expression across 24 breast cancer patient-derived xenograft (PDX) models. Integrated proteogenomic\",\n",
            "          \"xenograft (PDX) models. Integrated proteogenomic analysis shows positive correlation between expression measurements from transcriptomic and proteomic analyses; further, gene expression-based intrinsic subtypes are largely re-capitulated using non-stromal\",\n",
            "          \"intrinsic subtypes are largely re-capitulated using non-stromal protein markers. Proteogenomic analysis also validates a number of predicted genomic targets in multiple receptor tyrosine kinases. However, several protein/phosphoprotein events such as\",\n",
            "          \"kinases. However, several protein/phosphoprotein events such as overexpression of AKT proteins and ARAF, BRAF, HSP90AB1 phosphosites are not readily explainable by genomic analysis, suggesting that druggable translational and/or post-translational\",\n",
            "          \"that druggable translational and/or post-translational regulatory events may be uniquely diagnosed by MS. Drug treatment experiments targeting HER2 and components of the PI3K pathway supported proteogenomic response predictions in seven xenograft models.\",\n",
            "          \"proteogenomic response predictions in seven xenograft models. Our study demonstrates that MS-based proteomics can identify therapeutic targets and highlights the potential of PDX drug response evaluation to annotate MS-based pathway activities.\",\n",
            "          \"Patient-derived xenografts recapitulate major genomic signatures and transcriptome profiles of their original tumours. Here, the authors, performing proteomic and phosphoproteomic analyses of 24 breast cancer PDX models, demonstrate that druggable\",\n",
            "          \"of 24 breast cancer PDX models, demonstrate that druggable candidates can be identified based on a comprehensive proteogenomic profiling.\",\n",
            "          \"Profiling of somatic alteration by next-generation DNA sequencing (NGS) has entered clinical practice with the promise of rapid diagnosis of druggable somatic genomic alterations for personalized cancer treatment. For example, recent analysis of 4,068\",\n",
            "          \"cancer treatment. For example, recent analysis of 4,068 samples from 16 cancer types suggested that repurposing approved drugs based on genomic alterations could provide individualized treatment options for around 40% of tumours. However, clinical\",\n",
            "          \"treatment options for around 40% of tumours. However, clinical evidence for this proposition is limited and has been slow to develop. Further, the signalling and biological effects of somatic mutations are not routinely determined in human tumour samples\",\n",
            "          \"mutations are not routinely determined in human tumour samples even though this is a consideration for rational drug design, response prediction and target prioritization. Finally, druggable genomic alterations are not detected in the majority of cases\",\n",
            "          \"genomic alterations are not detected in the majority of cases tested by NGS. Comprehensive proteomic analyses provide a potentially valuable approach to validate genomic findings as likely biological drivers and to discover discover opportunities for\",\n",
            "          \"biological drivers and to discover discover opportunities for targeted treatment.\",\n",
            "          \"Patient-derived xenografts (PDXs) in immunodeficient mice maintain the histological and molecular heterogeneity of the progenitor human tumour and cytotoxic drug responsiveness is often a transplantable phenotype. Our previous studies have shown that\",\n",
            "          \"transplantable phenotype. Our previous studies have shown that breast tumour PDXs recapitulate major genomic signatures and transcriptome profiles of their original breast tumours. Moreover, drug responses to endocrine therapy in breast cancer PDXs\",\n",
            "          \"drug responses to endocrine therapy in breast cancer PDXs resembled that observed in the corresponding patient and endocrine therapy resistance patterns were associated with aberrations in the ESR1 gene. While comprehensive proteomic characterization of\",\n",
            "          \"ESR1 gene. While comprehensive proteomic characterization of PDX is still lacking, recent studies using reverse phase protein array have identified similar protein profiles between PDX and primary tumours. These studies collectively suggest that the PDX\",\n",
            "          \"tumours. These studies collectively suggest that the PDX approach is a potentially valuable preclinical model for identification and testing of therapeutic targets.\",\n",
            "          \"Recent advances in mass-spectrometry provide an unprecedented opportunity for antibody-independent proteome profiling with approximately 80% of all proteins in major human tissues quantifiable by this technique. Analyses of the global proteome in tandem\",\n",
            "          \"by this technique. Analyses of the global proteome in tandem with genomic profiles generated through TCGA have yielded key insights on the molecular etiology of colorectal cancer, breast cancer and ovarian cancer. Our recent proteogenomic characterization\",\n",
            "          \"and ovarian cancer. Our recent proteogenomic characterization of 105 human breast cancers linked 5q-loss to elevated EGFR expression and identified overexpression of phosphorylated kinases including CDK12, PAK1 and ARAF as potential biological drivers.\",\n",
            "          \"including CDK12, PAK1 and ARAF as potential biological drivers. However, current proteogenomic analyses of human tissues are limited by the fact that therapeutic hypotheses were not being addressed in these initial analyses. As patient treatments were\",\n",
            "          \"addressed in these initial analyses. As patient treatments were heterogeneous, links between treatment and proteogenomic findings are difficult to establish in banked tumour specimens. The PDX system therefore provides an early opportunity to determine\",\n",
            "          \"PDX system therefore provides an early opportunity to determine whether a proteogenomic approach can produce more robust therapeutic hypotheses and predictive biomarkers for individual patients than that achieved by genomic analysis alone.\",\n",
            "          \"In this study, we generated high-coverage proteome and phosphoproteome data for 24 PDX models representing different breast cancer subtypes using both isobaric mass-tag labelling and label-free proteomic methods for cross-validated identification of\",\n",
            "          \"proteomic methods for cross-validated identification of 12,794 proteins and 56,874 phosphorylation sites. After filtering for observation in at least 10 out of 24 samples 10,069 proteins and 36,609 phosphorylation sites with their relative abundances\",\n",
            "          \"and 36,609 phosphorylation sites with their relative abundances quantified across tumours, were used in subsequent analyses in this study. A substantial number of druggable protein/phosphoprotein events were uniquely identified. Further, similar\",\n",
            "          \"events were uniquely identified. Further, similar proteomic/phosphoproteomic signatures were observed in patient breast cancer samples indicating potential clinical relevance. In selected PDX models exhibiting proteomic and/or phosphoproteomic\",\n",
            "          \"PDX models exhibiting proteomic and/or phosphoproteomic up-regulation events of HER2 or the PI3K pathway, we validated the efficacy of targeted inhibitors in suppressing PDX growth. Taken together, these observations show that comprehensive proteogenomic\",\n",
            "          \"these observations show that comprehensive proteogenomic profiling has potential to identify new targets for individualized treatment approaches in cancer.\",\n",
            "          \"Results\\n\\nProteogenomic coverage of breast cancer xenografts\",\n",
            "          \"We selected 24 PDX models established from primary or metastatic breast tumours for comprehensive proteogenomic characterization (Fig. 1a). The human patient cohort was composed of 10 basal, 1 claudin-low (CLDN-low), 9 luminal B and 4 HER2-enriched\",\n",
            "          \"1 claudin-low (CLDN-low), 9 luminal B and 4 HER2-enriched (HER2-E) breast tumours based on PAM50 expression subtyping (Supplementary Data 1). We conducted DNA and RNA sequencing respectively for 23 PDX models and in one case Sanger DNA sequencing of\",\n",
            "          \"for 23 PDX models and in one case Sanger DNA sequencing of hotspot mutations. Isotope Tagging for Relative and Absolute Quantitation (iTRAQ 4-plex) was completed for all 24 PDXs for discovery and label-free quantification (LFQ) for 18 PDXs for validation\",\n",
            "          \"and label-free quantification (LFQ) for 18 PDXs for validation and confirmation (Supplementary Data 2). The comprehensive proteogenomic coverage allowed us to systematically assess the somatic mutation profile, copy-number variation (CNV), mRNA\",\n",
            "          \"the somatic mutation profile, copy-number variation (CNV), mRNA expression, protein expression, and phosphosite levels and localizations.\",\n",
            "          \"We examined somatic mutations in significantly mutated genes of breast cancer in 12 human tumours and their matching PDXs (Supplementary Data 2). Most key somatic mutations in these genes were preserved (Fig. 1b), validating the genomic fidelity of these\",\n",
            "          \"preserved (Fig. 1b), validating the genomic fidelity of these PDX models. However, recurrent breast cancer mutations were not detected in two PDX models (WHIM17 and WHIM46). While sequencing data were not available for the matched progenitor human\",\n",
            "          \"data were not available for the matched progenitor human tumours, the germline SNPs of human blood normal samples matched with the PDX tumours, validating their patients of origin. Follow-up histological and RNA-seq analyses suggested WHIM17 and WHIM46\",\n",
            "          \"histological and RNA-seq analyses suggested WHIM17 and WHIM46 are EBV-positive human lymphoproliferative cells arising in the NOD/SCID/\\u03b3 mouse strain (Methods, Supplementary Fig. 1).\",\n",
            "          \"We then compared the variant allele fractions (VAFs) of exonic somatic mutations in human tumours and derived xenografts when both are available. We found comparable or higher VAFs in xenografts, potentially due to higher tumour purity and selection of\",\n",
            "          \"potentially due to higher tumour purity and selection of some mutant alleles by loss of heterozygosity in PDX models (Fig. 1c). The relatively high, positive VAF correlation (R=0.66) implied similarity between human samples and their respective\",\n",
            "          \"implied similarity between human samples and their respective xenografts, consistent with our previous report. Importantly, all PIK3CA and TP53 somatic mutations in the 12 originating human tumours were detected in respective xenografts with comparable or\",\n",
            "          \"were detected in respective xenografts with comparable or increased VAFs (Fig. 1c). Among the 22 breast cancer PDX models, 14, including 7 of the 8 basal tumours, harboured TP53 mutations, 5 luminal B PDXs had ESR1 mutations, and 5 luminal B and 1\",\n",
            "          \"5 luminal B PDXs had ESR1 mutations, and 5 luminal B and 1 CLDN-low xenografts carried PIK3CA mutations.\",\n",
            "          \"We quantified mRNA expression of 16,209 unique human genes from RNA-seq data for these 24 models. We also applied two distinct MS data acquisition approaches (Supplementary Fig. 2a, Methods), iTRAQ (Supplementary Data 3) and LFQ (Supplementary Data 4), to\",\n",
            "          \"iTRAQ (Supplementary Data 3) and LFQ (Supplementary Data 4), to quantify the expression ratios of 12,794 human proteins (11,879 genes) and 8,648 proteins (8,035 genes), respectively. A total of 56,874 phosphosites were also confidently identified using\",\n",
            "          \"of 56,874 phosphosites were also confidently identified using iTRAQ (Supplementary Data 5). After filtering for observation in at least 10 out of 24 samples (4 out of 9 of the iTRAQ experiments, Supplementary Fig. 3), the relative abundances of 10,069\",\n",
            "          \"Supplementary Fig. 3), the relative abundances of 10,069 proteins and 36,609 phosphorylation sites were quantified across tumours by iTRAQ and used in subsequent analyses in this study. The technical replicates in the LFQ (WHIM2 and WHIM16) and the iTRAQ\",\n",
            "          \"replicates in the LFQ (WHIM2 and WHIM16) and the iTRAQ (WHIM13) sets showed high correlation in protein expression levels (R>0.85, Supplementary Fig. 2b,c,e). Further, phosphosite expressions also showed high correlations in technical replicates (WHIM13)\",\n",
            "          \"also showed high correlations in technical replicates (WHIM13) of the iTRAQ experiment (R=0.82, Supplementary Fig. 2d), validating the technical reproducibility of our proteomic and phosphoproteomic datasets. Of note, while iTRAQ and LFQ quantification\",\n",
            "          \"datasets. Of note, while iTRAQ and LFQ quantification were conducted separately and based on different features of the LC\\u2013MS data collected from the Q Exactive mass spectrometer (that is, reporter ion intensity and MS1 peak area, respectively), our\",\n",
            "          \"reporter ion intensity and MS1 peak area, respectively), our analysis showed reasonable correlation between the two measurements after normalization (R=0.61). The two datasets utilizing the same PDX models but different workflows enabled cross-method\",\n",
            "          \"same PDX models but different workflows enabled cross-method validation of global proteomic results.\",\n",
            "          \"Proteogenomic integration and comparison\",\n",
            "          \"We first evaluated the correlation between mRNA expression and protein abundance measurements from the iTRAQ experiment (Fig. 2a) for these 24 PDX models: 83.6% of the genes with sufficient data showed positive correlations with a median Pearson R=0.536\",\n",
            "          \"data showed positive correlations with a median Pearson R=0.536 (Supplementary Fig. 4). We investigated whether the trend of mRNA-protein correlations were associated with specific KEGG pathways, finding that metabolic pathways involved in house-keeping\",\n",
            "          \"finding that metabolic pathways involved in house-keeping functions are enriched for genes showing high, positive correlation. For example, genes in the glutathione metabolism pathway showed the highest enrichment for positive correlations\",\n",
            "          \"pathway showed the highest enrichment for positive correlations (Kolmogorov\\u2013Smirnov test, false discovery rate (FDR)=5.6e-07). Interestingly, we observed that genes in the ribosome (FDR<2.2e-16), spliceosome (FDR=2.0e-13), RNA transport (FDR=1.7e-5), RNA\",\n",
            "          \"spliceosome (FDR=2.0e-13), RNA transport (FDR=1.7e-5), RNA polymerase (FDR=1.8e-4), and oxidative phosphorylation (FDR<2.2e-16) pathways showed relatively lower correlation between mRNA and protein level. These pathways were enriched for genes that do not\",\n",
            "          \"level. These pathways were enriched for genes that do not require translated proteins for their biological functions. Similar pathway-specific pattern of positive and negative enrichment of correlations were observed by LFQ (Supplementary Fig. 5). The\",\n",
            "          \"correlations were observed by LFQ (Supplementary Fig. 5). The high degree of mRNA\\u2013protein correlation observed in the PDX samples is consistent with recent results obtained for human breast tumours and colorectal cancer, suggesting that PDXs closely mimic\",\n",
            "          \"and colorectal cancer, suggesting that PDXs closely mimic the respective human breast tumours in their relationship between mRNA and protein.\",\n",
            "          \"We then examined the correlation of CNV, mRNA, and protein expression levels for several key genes for breast cancer biology: EGFR, ERBB2 (HER2), ESR1, GATA3, PGR, PIK3CA, AKT1/2/3, MTOR and TP53. In most cases, we observed consistent relationships\",\n",
            "          \"and TP53. In most cases, we observed consistent relationships between CNV, mRNA, and protein expression levels. Compared with PDXs of non-luminal subtypes, luminal B breast cancer xenografts, as expected, showed higher mRNA and protein expression of ESR1\",\n",
            "          \"as expected, showed higher mRNA and protein expression of ESR1 and PGR, consistent with their positive ER and PR status (Fig. 2b). Five out of the six PIK3CA mutations were observed in luminal B PDXs, which tended to also show higher mRNA and protein\",\n",
            "          \"B PDXs, which tended to also show higher mRNA and protein expression levels of GATA3. In contrast, a larger proportion of basal PDXs expressed higher protein levels of EGFR. Strong HER2 expression at the mRNA, protein and phosphoprotein levels were\",\n",
            "          \"expression at the mRNA, protein and phosphoprotein levels were detected in WHIM8 and WHIM35, both derived from HER2-positive breast cancers. Overall expression patterns for key genes were consistent with the clinical subtype diagnosis across CNV, mRNA,\",\n",
            "          \"with the clinical subtype diagnosis across CNV, mRNA, protein and phosphosite analyses (Supplementary Fig. 6).\",\n",
            "          \"Proteomic subtyping of xenografts and human breast tumours\",\n",
            "          \"Molecular subtyping of breast cancer based on mRNA expression profiles has been shown to correlate with prognosis and has treatment implications. As expected, transcriptome clustering based on PAM50 genes from RNA-seq data largely reproduced intrinsic\",\n",
            "          \"on PAM50 genes from RNA-seq data largely reproduced intrinsic subtypes of the breast tumours (Fig. 3a; Supplementary Data 6). To explore proteomic and phosphoproteomic subtype classifications in xenograft models, we conducted unsupervised clustering of\",\n",
            "          \"in xenograft models, we conducted unsupervised clustering of proteomes and phosphoproteomes of the 24 xenograft samples based upon the top 436 variably expressed proteins showing a s.d. greater than 2 from the iTRAQ proteome (Supplementary Data 7). Two\",\n",
            "          \"than 2 from the iTRAQ proteome (Supplementary Data 7). Two distinct clusters emerged: one contained all basal tumours and the only CLDN-low tumour (WHIM12), while the other included all luminal B and HER2-E breast tumours; HER2-E tumours did not show a\",\n",
            "          \"B and HER2-E breast tumours; HER2-E tumours did not show a proteomic expression profile distinct from luminal B samples (Fig. 3b). Clustering analysis using the same gene markers from the transcriptome (Supplementary Fig. 7a) and the LFQ proteome\",\n",
            "          \"the transcriptome (Supplementary Fig. 7a) and the LFQ proteome (Supplementary Fig. 7b; Supplementary Data 8) further supported the separation into these two proteomic subtypes, although the minor differences between the transcriptome and the proteome\",\n",
            "          \"minor differences between the transcriptome and the proteome clustering suggested distinctions between mRNA and protein levels. The proteomic subtypes defined by the top 968 most variably expressed mouse host proteins did not segregate based on luminal\",\n",
            "          \"mouse host proteins did not segregate based on luminal and basal subtypes (Supplementary Fig. 8; Supplementary Data 9). Besides differing in mRNA expression, luminal and basal breast cancer PDXs also showed consistently distinct proteomic expression\",\n",
            "          \"PDXs also showed consistently distinct proteomic expression profiles, supporting their distinct biological origins.\",\n",
            "          \"We then utilized the iTRAQ phosphopeptide expression data to infer phosphoproteomic subtypes. 1,737 unique phosphosites with s.d. greater than 2.5 were included to conduct hierarchical clustering (Fig. 3c; Supplementary Data 10). These analyses of the\",\n",
            "          \"(Fig. 3c; Supplementary Data 10). These analyses of the phosphoproteomic data produced two major clusters segregating the luminal B and basal subtypes. Again, the WHIM37/WHIM47/WHIM26 group and the WHIM17/WHIM46 group, as observed in the proteome\",\n",
            "          \"group and the WHIM17/WHIM46 group, as observed in the proteome clustering, grouped closely together. Gene and protein expression of lymphoid lineage markers showed high expression of CD20 and JAK3 in WHIM17 and WHIM46 (Methods, Supplementary Fig. 9),\",\n",
            "          \"and JAK3 in WHIM17 and WHIM46 (Methods, Supplementary Fig. 9), consistent with their positive EBV status and histological diagnosis (Supplementary Fig. 1) as human lymphoproliferative cells arising in an immunocompromised mouse background. Overall, while\",\n",
            "          \"in an immunocompromised mouse background. Overall, while the basal and luminal clusters remain consistent, the hierarchical distances between PDX samples within the two major clusters differ between data types. The departure of proteomic and\",\n",
            "          \"differ between data types. The departure of proteomic and phosphoproteome subtypes from mRNA expression-defined subtypes suggests independent layers of molecular heterogeneity provided by distinct proteomic analyses.\",\n",
            "          \"We then investigated whether human proteomic subtypes could be recapitulated in PDX models by including an additional 77 proteomes from TCGA human tumours that were processed concurrently in the iTRAQ experiment (Methods). To reduce clustering bias imbued\",\n",
            "          \"iTRAQ experiment (Methods). To reduce clustering bias imbued by the mouse contribution to the proteome in the PDX samples, we excluded proteins showing differential expression between human tumour and xenograft (t-test, FDR\\u22640.3) (Supplementary Data 11).\",\n",
            "          \"tumour and xenograft (t-test, FDR\\u22640.3) (Supplementary Data 11). Additional requirements, including the presence of detectable protein in more than 10 samples and minimum difference of 2 s.d.'s in the merged proteome, resulted in 133 proteins qualifying\",\n",
            "          \"in the merged proteome, resulted in 133 proteins qualifying for un-supervised clustering. Consistent with the subtyping analysis of the 24 PDXs alone, we identified two major clusters: one that included all but one basal breast tumour and the other\",\n",
            "          \"one that included all but one basal breast tumour and the other comprised mostly of luminal tumours (Fig. 3d). Similar to the proteomic subtypes of the PDX cohorts, luminal tumours and HER-2 tumours did not show clear separation, although several\",\n",
            "          \"HER-2 tumours did not show clear separation, although several sub-clusters were identified. Importantly, xenograft proteomes clustered adjacently to the human proteomes of their respective subtypes, validating the fidelity of basal and luminal proteomic\",\n",
            "          \"validating the fidelity of basal and luminal proteomic signatures discovered in PDXs.\",\n",
            "          \"To search for defining markers between the basal and luminal B subtypes, we conducted differential expression analysis between the PDXs of the respective subtypes. We found several proteomic markers that were differentially expressed in both the LFQ and\",\n",
            "          \"markers that were differentially expressed in both the LFQ and iTRAQ datasets (t-test, FDR\\u22640.05), including SPR, GSTP1 and SERPINB5 (Supplementary Fig. 10; Supplementary Data 12). We then conducted gene-set enrichment analysis based on the Reactome\",\n",
            "          \"conducted gene-set enrichment analysis based on the Reactome pathway database to investigate patterns of differential expression (Methods). The basal subtype breast tumours were up-regulated in most of the significantly differentially-expressed pathways\",\n",
            "          \"in most of the significantly differentially-expressed pathways (FDR\\u22640.01) identified by both LFQ and iTRAQ datasets, including extracellular matrix organization, cell cycle, and collagen formation. In comparison, the luminal B breast tumours showed higher\",\n",
            "          \"In comparison, the luminal B breast tumours showed higher expression in genes related to organelle biogenesis and membrane trafficking (Supplementary Data 13).\",\n",
            "          \"Activated pathways revealed by phosphorylation profiles\",\n",
            "          \"Cancer driving somatic events trigger major changes in downstream signalling to launch the tumorigenic cascade. To search for tumour-specific activated pathways in PDX tumours, we systematically evaluated and compared phosphoproteome profiles of gene sets\",\n",
            "          \"evaluated and compared phosphoproteome profiles of gene sets from KEGG signalling pathways (Methods). Phosphorylation enrichment analysis identified 12 significantly activated pathways, including Ras, MAPK and NF\\u03baB signalling, in 4 xenografts (FDR\\u22640.01,\",\n",
            "          \"Ras, MAPK and NF\\u03baB signalling, in 4 xenografts (FDR\\u22640.01, Supplementary Data 14). WHIM9 exhibited elevated phosphorylation of the MAPK signalling pathway (FDR=9.69e-6; Fig. 4a). Interestingly, WHIM9 carried a recurrent somatic mutation, KRAS p.A146V\",\n",
            "          \"WHIM9 carried a recurrent somatic mutation, KRAS p.A146V (refs), which may have driven canonical MAPK pathway activation. Further, WHIM12 exhibited activation of the Ras signalling pathway (FDR=4.28e-5), along with an outlier protein expression of MET, a\",\n",
            "          \"along with an outlier protein expression of MET, a receptor tyrosine kinase upstream of the Ras signalling pathway (Fig. 4b). Interestingly, WHIM17 and WHIM46, both harboured BTK and PLCG2 protein overexpression, exhibited overall high phosphorylation of\",\n",
            "          \"overexpression, exhibited overall high phosphorylation of the NF\\u03baB signalling pathway (Fig. 4c,d; FDR=6.94e-3, 4.53e-5 respectively). This observation further supports the strong similarity between these two PDX models based on protein/phosphoprotein\",\n",
            "          \"between these two PDX models based on protein/phosphoprotein clustering (Fig. 3b,c) and their classification as EBV-positive lymphoproliferation.\",\n",
            "          \"While pathways such as PI3K/AKT/MTOR are known to be activated across the majority of breast tumours, our analysis shows that other complementary tumorigenesis-related pathways including RAS/MAPK are also activated in a small set of breast tumours due to\",\n",
            "          \"are also activated in a small set of breast tumours due to specific genomic or proteomic alterations, representing alternative treatment opportunities. In addition, our phospho-proteomic analysis revealed activation of signalling pathways not readily\",\n",
            "          \"analysis revealed activation of signalling pathways not readily predicted by genomic data.\",\n",
            "          \"Complementary genomic/proteomic druggable targets\",\n",
            "          \"We examined promising drug targets in each tumour by surveying the genome and expressed proteomes. Specifically, we compiled a list 76 druggable genes (Supplementary Data 15), along with their respective drugs, from established public databases (Methods).\",\n",
            "          \"respective drugs, from established public databases (Methods). Six PDXs, representing 20.8% of the tumours in this study, harboured druggable somatic mutations, including PIK3CA p.H1047R and KRAS p.A146V in WHIM9, PIK3CA p.H1047R in WHIM16 and WHIM24,\",\n",
            "          \"KRAS p.A146V in WHIM9, PIK3CA p.H1047R in WHIM16 and WHIM24, PIK3CA p.E545K in WHIM18, PIK3CA p.E542K in WHIM20 and SF3B1 p.K700E in WHIM26 (Supplementary Data 16).\",\n",
            "          \"While activating mutations in oncogenes can be targeted by treatments, aberrantly overexpressed or activated protein products, such as HER2, also presents exploitable treatment opportunities. We sought genomic and proteomic evidence of overexpressed\",\n",
            "          \"We sought genomic and proteomic evidence of overexpressed genes/proteins or proteins with highly phosphorylated sites. We defined outliers as expression values exceeding the 1.5 interquartile ranges (IQR) above the third quartile of the cohort (Variations\",\n",
            "          \"ranges (IQR) above the third quartile of the cohort (Variations of Box Plots), and rank ordered them by the outlier score. We further required CNV outliers to be validated with outlier expression in either the mRNA or protein level to rule out\",\n",
            "          \"expression in either the mRNA or protein level to rule out up-regulation events due to technical artifacts or passenger events (Methods). mRNA and protein expression outlier scores showed a moderate positive correlation (Fig. 5a, R=0.516), but mRNA\",\n",
            "          \"a moderate positive correlation (Fig. 5a, R=0.516), but mRNA outlier expression did not guarantee high protein expression (for example, AKT2 and FGFR2, Fig. 5a). Similarly, we observed a fraction of phosphosite outliers not detected at the protein level\",\n",
            "          \"of phosphosite outliers not detected at the protein level (Fig. 5b, R=0.548). Consequently, identifying post-transcriptional and post-translational events to capture potential druggable treatment opportunities requires consideration of protein expression\",\n",
            "          \"opportunities requires consideration of protein expression as well as gene expression.\",\n",
            "          \"Applying this druggable outlier detection strategy across CNV, mRNA, protein and protein phosphorylation levels, we identified overexpressed druggable genes in 26.1% and 47.8% of PDXs at the CNV and mRNA levels, respectively (Fig. 6; Supplementary Data\",\n",
            "          \"CNV and mRNA levels, respectively (Fig. 6; Supplementary Data 16). These events recapitulated known druggable opportunities, such as the PIK3CA copy-number amplification in WHIM4 and ERBB2 (HER2) copy-number amplification in WHIM35. Expanding to iTRAQ\",\n",
            "          \"(HER2) copy-number amplification in WHIM35. Expanding to iTRAQ protein expression outliers allowed us to uncover druggable targets in 19 out of the 24 PDXs (79.2%), while considering phosphosite outliers covered 22 of the 24 PDXs (91.7%). A significantly\",\n",
            "          \"outliers covered 22 of the 24 PDXs (91.7%). A significantly high fraction of protein outliers overlapped between the LFQ and iTRAQ datasets (Fisher's Exact Test, P=2.013e-05, Supplementary Fig. 11), providing validation of our findings.\",\n",
            "          \"The identified proteomic outliers included multiple known druggable targets involved in the PI3K, RTK, MAPK signalling pathways and other oncogenic processes (Fig. 6; Supplementary Data 16). For instance, HER2 was found to be the top outlier at the\",\n",
            "          \"16). For instance, HER2 was found to be the top outlier at the protein expression level in 2 HER2-E xenografts, WHIM8 and WHIM35; the phosphosites of HER2 were also identified as outliers. Immunohistochemistry experiments validated the high HER2 protein\",\n",
            "          \"experiments validated the high HER2 protein expression on the cell membranes of WHIM8 and WHIM35 (Fig. 5c; Supplementary Fig. 12). Further, AKT1, AKT2 and AKT3 proteins were outlier expression candidates in WHIM35, WHIM20 and WHIM43 and WHIM4,\",\n",
            "          \"expression candidates in WHIM35, WHIM20 and WHIM43 and WHIM4, respectively, providing proteomic validation of previously observed AKT up-regulation at mRNA expression level in breast cancer. Other outlier proteins included IDH1 in WHIM9 and WHIM30, FGFR4\",\n",
            "          \"Other outlier proteins included IDH1 in WHIM9 and WHIM30, FGFR4 in WHIM11 and WHIM26, and both BTK and JAK3 in the EBV-positive WHIM17 and WHIM46. Many of these protein outliers aligned with up-regulation in their corresponding phosphosites, including\",\n",
            "          \"up-regulation in their corresponding phosphosites, including phosphosites on AKT1, AKT3, BRAF, FGFR4 and HSP90AB1 (Fig. 5c, selected phosphosite spectra in Supplementary Fig. 13). We also discovered additional outlier phosphosites in other proteins\",\n",
            "          \"discovered additional outlier phosphosites in other proteins including CTNNB1, ARAF, and HSP90B1 (Supplementary Data 16). Finally, we identified outlier FGFR2 in WHIM16 and high RAF1 in WHIM9 and further validated their high protein expression status of\",\n",
            "          \"and further validated their high protein expression status of by immunohistochemistry analyses (Fig. 5c; Supplementary Fig. 12). As diverse sets of proteomic outliers are identified across PDXs, effectively inhibiting such activation events will be\",\n",
            "          \"PDXs, effectively inhibiting such activation events will be required when designing targeted treatment strategies to each individual breast tumour.\",\n",
            "          \"Notably, for 80.6% of proteomic outlier events, we were able to identify human tumours from the 77 TCGA samples showing the same outlier protein expression through outlier score or ranking (Methods, Fig. 6b, Supplementary Data 16). For example, we\",\n",
            "          \"(Methods, Fig. 6b, Supplementary Data 16). For example, we observed outlier HER2 expression in five HER2-E and four luminal B human breast tumours. Further, a basal human sample carried both the outlier FGFR2 and FGFR4 expression, validating the findings\",\n",
            "          \"the outlier FGFR2 and FGFR4 expression, validating the findings in WHIM16, WHIM26 and WHIM11. Basal human breast tumours also carried protein expression outliers in IDH1, EGFR and MAP2K1 (Fig. 6b). Phosphosite outlier expression events showed a moderate\",\n",
            "          \"6b). Phosphosite outlier expression events showed a moderate rate of validation in the same human cohort (48.8%, Fig. 6c), suggesting its transient nature and potential micro-environmental effects on protein phosphorylation. As expected, HER2-E and a few\",\n",
            "          \"on protein phosphorylation. As expected, HER2-E and a few HER2-positive luminal B tumours carried outliers in HER2 phosphosites including p.T701, p.T1240 and p.Y1248. ARAF phosphosite outliers, such as p.S299, were found in both human and PDX samples,\",\n",
            "          \"such as p.S299, were found in both human and PDX samples, validating our previous finding in the human cohort. Interestingly, we identified outlier phosphosites in genes not previously implicated in breast cancer through genomic profiles, such as BRAF\",\n",
            "          \"in breast cancer through genomic profiles, such as BRAF p.S447, p.S750 and HSP90AB1 p.Y56 and p.S169 (Fig. 6c). Our results demonstrated that proteomic outlier events, like genomic driver mutations, are consistently observed in PDXs and human tumours.\",\n",
            "          \"mutations, are consistently observed in PDXs and human tumours. Some protein outlier events might represent \\u2018proteomic drivers' of tumorigenesis and therefore potential drug targets in breast tumours.\",\n",
            "          \"Targeted treatments using breast cancer xenograft models\",\n",
            "          \"To validate the identified druggable events, we conducted treatment experiments targeting HER2 and PI3K pathways on selected PDX models. Four PDX models were chosen to address HER2 targeting using lapatinib, an oral HER2 kinase inhibitor. These include 2\",\n",
            "          \"using lapatinib, an oral HER2 kinase inhibitor. These include 2 HER2-E PDX models WHIM8 and WHIM35, both with high HER2 protein and phosphoprotein expression, and 2 low HER2 expressing, basal-like PDX models WHIM6 and WHIM14. Western blotting suggested\",\n",
            "          \"PDX models WHIM6 and WHIM14. Western blotting suggested high HER2 expression and phosphorylation of HER2 p.Y1248 in both HER2 positive tumours, WHIM8 and WHIM35 (Fig. 7a; Supplementary Fig. 14). Unexpectedly, high levels of HER2 p.Y1248 phosphorylation\",\n",
            "          \"14). Unexpectedly, high levels of HER2 p.Y1248 phosphorylation was also detected in WHIM14 by western blotting. In contrast MS-based phosphoproteomics also detected high levels of HER2p.Y1248 in WHIM8 and WHIM35, but not in WHIM14, which was more\",\n",
            "          \"in WHIM8 and WHIM35, but not in WHIM14, which was more consistent with the known biology of the models. Antibody-based diagnosis of HER2 activation in WHIM14 was possibly due to cross-reaction of the Y1248 antibody with the pY1172 site of EGFR that bears\",\n",
            "          \"of the Y1248 antibody with the pY1172 site of EGFR that bears high sequence similarity around the pY residue (..GTPTAENPEy1248LGLDVPV-CO2H vs...GSTAENAEy1172LRVAPQ.., Supplementary Fig. 6). As expected, WHIM8 and WHIM35 were growth inhibited (Wilcoxon\",\n",
            "          \"As expected, WHIM8 and WHIM35 were growth inhibited (Wilcoxon Rank Sum Test, P=2.7e-5, 4.8e-7) by lapatinib, whereas WHIM6 was not (P=0.65) (Fig. 7b). Interestingly, WHIM14 also showed significant reduction in tumour growth by lapatinib (P=4.3e-3). Upon\",\n",
            "          \"reduction in tumour growth by lapatinib (P=4.3e-3). Upon further exploration with a lower, clinically achievable dose of lapatinib for chronic treatment (30\\u2009mg\\u2009kg\\u22121 compared with 220\\u2009mg\\u2009kg\\u22121 in the previous experiment) over 48 days, significant but weak\",\n",
            "          \"in the previous experiment) over 48 days, significant but weak tumour growth inhibition was again achieved (P=0.0311, Supplementary Fig. 15). However, WHIM14 did not respond to HER2/HER3 antibodies trastuzumab or pertuzumab (P=0.250 and 0.181,\",\n",
            "          \"antibodies trastuzumab or pertuzumab (P=0.250 and 0.181, respectively; Supplementary Fig. 15). Thus, the response of WHIM14 to lapatinib was likely due to inhibition of EGFR not HER2 (refs). While WHIM6 also showed elevated EGFR protein level, WHIM14\",\n",
            "          \"While WHIM6 also showed elevated EGFR protein level, WHIM14 showed notably higher EGFR phosphorylation based on both mass spectrometry (Fig. 2b) and western blotting (Fig. 7a), which could account for their different response to lapatinib. Two basal and\",\n",
            "          \"for their different response to lapatinib. Two basal and one HER2-E human breast cancers harboured outlier EGFR expression (Fig. 6b), suggesting EGFR remains a potential therapeutic target in a subset of breast cancers that has yet to be fully realized\",\n",
            "          \"in a subset of breast cancers that has yet to be fully realized clinically.\",\n",
            "          \"The PI3K/AKT/mTOR signalling pathway is altered in approximately 77% of breast tumours and multiple drugs targeting its components, including Class I PI3Ks, AKTs, and mTORs, are already in clinical trials. Among these, a combination of everolimus and\",\n",
            "          \"clinical trials. Among these, a combination of everolimus and exemestane has been approved for treating advanced ER-positive breast cancers resistant to non-steroidal aromatase inhibitors. Promising activity has also been reported for direct inhibitors of\",\n",
            "          \"activity has also been reported for direct inhibitors of PI3K. However, mutations in PIK3CA and other genetic alterations at the genomic level have so far failed to closely predict therapeutic responsiveness to PI3K pathway inhibitors. We therefore\",\n",
            "          \"responsiveness to PI3K pathway inhibitors. We therefore hypothesized that combined genomic and proteomic indication of PI3K signalling activation is necessary for the prediction of treatment response. Our integrated approach identified 6 xenografts that\",\n",
            "          \"response. Our integrated approach identified 6 xenografts that harboured complementary genomic and proteomic druggable events in the PI3K-AKT pathway. In particular, WHIM16 harboured a hotspot PIK3CA p.H1047R mutation, whereas WHIM18 and WHIM20 each\",\n",
            "          \"PIK3CA p.H1047R mutation, whereas WHIM18 and WHIM20 each carried a hotspot PIK3CA p.E545K mutation. WHIM20 also showed an additional outlier protein expression of AKT2 and WHIM18 also expressed AKT2 at a near outlier level that may combine to activate\",\n",
            "          \"AKT2 at a near outlier level that may combine to activate PI3K pathway signalling (Fig. 6). Since the treatments were applied to later-passage PDX models relative to the ones we conducted proteogenomic analysis on, we performed immunohistochemistry to\",\n",
            "          \"proteogenomic analysis on, we performed immunohistochemistry to validate the phosphorylation of AKT p.S473. WHIM18 and WHIM20 showed detectable AKT p.S473, which was not observed in WHIM16 (Fig. 7c).\",\n",
            "          \"We conducted combinatorial treatment experiments by applying an alpha specific PI3K inhibitor (TAK-117) and/or an mTORC1/2 inhibitor (TAK-228) to three PDX models of luminal B breast cancer. Consistent with previous reports, PIK3CA mutation status alone\",\n",
            "          \"Consistent with previous reports, PIK3CA mutation status alone did not accurately predict outcome; WHIM18 and WHIM20 showed reduced tumour growth upon application of the inhibitors, whereas WHIM16 did not (Fig. 7d). mTOR inhibition repressed tumour growth\",\n",
            "          \"did not (Fig. 7d). mTOR inhibition repressed tumour growth in WHIM18 and WHIM20 (ANOVA followed by Tukey's post hoc test, P=9.5e-10, 1.6e-03 respectively), but not in WHIM16 (P=0.97), showing that inhibition of mTORC1/2 may effectively suppress breast\",\n",
            "          \"that inhibition of mTORC1/2 may effectively suppress breast tumours with activated AKTs and validating a previous study showing the efficacy of mTOR inhibition in PDX models of triple-negative breast cancer. Importantly, the combinatorial treatment\",\n",
            "          \"breast cancer. Importantly, the combinatorial treatment achieved the greatest effect in WHIM18 and WHIM20 (P<2.2e-16 for both comparisons). While neither PI3K nor mTOR inhibitor drug treatment alone suppressed tumour growth in WHIM20 completely, the\",\n",
            "          \"alone suppressed tumour growth in WHIM20 completely, the combination of both PI3K and mTOR inhibitors significantly reduced tumour growth to a nearly static state (Fig. 7d). Based on our proteomic characterization, WHIM20 exhibited the strongest AKT1 and\",\n",
            "          \"characterization, WHIM20 exhibited the strongest AKT1 and AKT2 protein expression, as well as AKT1 p.S122 and p.S475 phosphorylation signatures followed by WHIM18 and then WHIM16 (Fig. 6b,c). Further, both PI3K and mTOR inhibitors significantly reduced\",\n",
            "          \"Further, both PI3K and mTOR inhibitors significantly reduced AKT p.S473 (Fig. 7c). Our treatment results showed that the magnitude of drug response may be associated with overexpression and phosphorylation of the downstream signalling targets such as AKT\",\n",
            "          \"of the downstream signalling targets such as AKT proteins.\",\n",
            "          \"In addition to this validation of druggable hypotheses in luminal tumours, our previous report also demonstrated the effectiveness of combinatorial therapy of AKT and mTOR inhibitors in two other basal breast cancer xenografts. One of the treated\",\n",
            "          \"in two other basal breast cancer xenografts. One of the treated xenografts, WHIM4, was also characterized in this study, and showed copy-number amplification of PIK3CA and AKT3 protein outlier expression (Fig. 6). Overall, proteogenomic analysis revealed\",\n",
            "          \"expression (Fig. 6). Overall, proteogenomic analysis revealed that the dual activation of PIK3CA at the genomic level and AKTs at the protein level may be a common signature of breast tumours, affecting more than 20% of PDXs in this cohort. Importantly,\",\n",
            "          \"affecting more than 20% of PDXs in this cohort. Importantly, our results demonstrate the potential utility of combinatorial inhibitor treatments to treat breast tumours showing these proteogenomic signatures.\",\n",
            "          \"Discussion\",\n",
            "          \"Breast cancer has been traditionally characterized in the clinic through hormone receptor status and selected genes' expressions, and more recently by genomic sequencing. However, druggable genomic driver events are detectable in only a limited percentage\",\n",
            "          \"driver events are detectable in only a limited percentage of patients. As the majority of drugs target proteins, a systematic evaluation of breast cancer proteomes would seem ultimately to be necessary for selecting targeted treatment and predicting drug\",\n",
            "          \"necessary for selecting targeted treatment and predicting drug response. Recent advances in MS-based proteomics allow extensive and quantitative surveys of the global proteome. Here, we have systematically analyzed proteogenomic profiles of 22\",\n",
            "          \"we have systematically analyzed proteogenomic profiles of 22 patient-derived breast cancer xenografts and 2 EBV-positive lymphoproliferations that are likely artifacts of engraftment of human lymphocytes into NSG mice.\",\n",
            "          \"This study shows that proteogenomic signatures of PDXs resemble most findings from breast cancer patients. While some discrepancies exist, we established a normalization strategy at both the genomic and proteomic levels that enabled direct comparison. PDX\",\n",
            "          \"and proteomic levels that enabled direct comparison. PDX tumours recapitulated the proteomic diversity of human breast cancers (Fig. 3). We also identified multiple druggable targets for each tumour model (Fig. 6; Supplementary Data 17). Proteomic events\",\n",
            "          \"tumour model (Fig. 6; Supplementary Data 17). Proteomic events validated a significant number of CNV or mRNA up-regulations. For example, HER2 protein and phosphosite outlier expression was observed in HER2-E WHIM8 and WHIM35 (Fig. 5), which were\",\n",
            "          \"was observed in HER2-E WHIM8 and WHIM35 (Fig. 5), which were effectively treated using lapatinib (Fig. 7b). Interestingly, we also identified overexpressed proteomic events not evident in the genomic level in both PDX and human samples, including outlier\",\n",
            "          \"genomic level in both PDX and human samples, including outlier protein expression of EGFR, and outlier phosphosite expressions of ARAF, BRAF, HSP90AB1, PTPN11 and TOP2A (Fig. 6), highlighting potential new treatment opportunities in breast cancer. In the\",\n",
            "          \"potential new treatment opportunities in breast cancer. In the two PDX models subsequently diagnosed as EBV-positive lymphoproliferations, we observed outlier BTK expression (Fig. 6) and activation of the NF\\u03baB pathway (Fig. 4), validating BTK as a\",\n",
            "          \"activation of the NF\\u03baB pathway (Fig. 4), validating BTK as a druggable target in EBV+ lymphomas. While more than 80% of the proteomic outlier events in PDX were also found in human tumours, a lessor 48.8% of phosphosite outlier events were validated,\",\n",
            "          \"a lessor 48.8% of phosphosite outlier events were validated, potentially due to different tumour micro-environments. Thus, transient phosphoproteomic events identified in PDX tumours would likely require further verification in their corresponding primary\",\n",
            "          \"require further verification in their corresponding primary tumours.\",\n",
            "          \"Outlier protein expression events can likely lead to downstream pathway activation, such as MET outlier protein expression (Fig. 6) and activated Ras pathway observed in WHIM12 (Fig. 4). While genomic analysis has utilized mutual occurrence or exclusivity\",\n",
            "          \"genomic analysis has utilized mutual occurrence or exclusivity in patient cohorts to deduce pathway relationships, phosphorylation profile analyses allowed us to directly interrogate signalling in these established pathways in a single sample. Our pathway\",\n",
            "          \"in these established pathways in a single sample. Our pathway activation results suggest these events may be crucial to tumorigenesis and that some are likely the proteomic \\u2018smoking guns' that originally triggered the oncogeneic cascade.\",\n",
            "          \"Roughly 77% of breast tumours showed alterations in the PI3K-AKT signalling pathway, representing a potentially important path forward for drug-based treatment. Yet, genomic alterations of PI3K pathway components have not shown themselves to predict\",\n",
            "          \"of PI3K pathway components have not shown themselves to predict treatment responsiveness to PI3K pathway inhibitors. In this study, we defined complementary druggable targets of the pathway using proteogenomic analysis, including several events of a\",\n",
            "          \"using proteogenomic analysis, including several events of a co-occurring PIK3CA mutation or copy-number amplification, and AKT protein outlier expression coupled by elevated AKT phosphorylation. In two such breast tumours, we successfully inhibited tumour\",\n",
            "          \"In two such breast tumours, we successfully inhibited tumour growth using a combination of PI3K and MTOR inhibitors (Fig. 7c). While these results show potential functional implications, additional, systematic treatment experiments are required to\",\n",
            "          \"additional, systematic treatment experiments are required to validate the identified proteomic druggable targets.\",\n",
            "          \"In conclusion, this initial work using proteogenomic integration coupled with patient-derived xenograft validation, has demonstrated a strategy that, in principle, may enable more accurate prediction of the efficacy of mechanism-based cancer therapeutics.\",\n",
            "          \"Methods\\n\\nXenograft model generation\",\n",
            "          \"Patient-derived xenografts were generated from primary or metastatic breast tumours using previously described procedures. All human tissues for these experiments were processed in compliance with NIH regulations and institutional guidelines, and approved\",\n",
            "          \"with NIH regulations and institutional guidelines, and approved by the institutional review board at Washington University. All animal procedures were reviewed and approved by the institutional animal care and use committee at Washington University in St.\",\n",
            "          \"animal care and use committee at Washington University in St. Louis. PDX models are available through the application to the Human and Mouse-Linked Evaluation of Tumors core at http://digitalcommons.wustl.edu/hamlet/.\",\n",
            "          \"We selected 24 of the established breast tumour samples, including 9 luminal B, 10 basal, 4 HER2-E, and 1 CLDN-low breast tumours for further proteogenomic characterization. Receptor statuses of xenograft tumours were validated using IHC after\",\n",
            "          \"statuses of xenograft tumours were validated using IHC after engraftment.\",\n",
            "          \"Immunohistochemistry\",\n",
            "          \"Xenografts were formalin-fixed at least for 24\\u2009h and paraffin-embedded. Sections were evaluated by hematoxylin & eosin (H&E) staining. Immunohistochemistry (IHC) was performed on additional sections for HER2 (Dako), Phospho-Akt (Ser473) (Cell signalling),\",\n",
            "          \"for HER2 (Dako), Phospho-Akt (Ser473) (Cell signalling), FGFR2 (Abcam), and Raf-1 (Santa Cruz) following the manufacturer's instructions. Western blotting of phosphorylated HER2 (p.Y1248) was performed using antibody cat Nr-06-229 (Millipore).\",\n",
            "          \"In vivo drug treatment experiments\",\n",
            "          \"For the targeted treatment of the PI3K pathway, PI3K alpha inhibitor TAK-117 and TORC1/2 inhibitor TAK-228 were provided by Millennium Pharmaceuticals, Inc. The compounds were dissolved in Peg400. Tumours were engrafted in NSG (NOD.Cg-Prkdcscid\",\n",
            "          \"in Peg400. Tumours were engrafted in NSG (NOD.Cg-Prkdcscid Il2rgtm1Wjl/SzJ) mice (The Jackson Laboratory) by subcutaneous injection of 2\\u20135 \\u00d7 106 PDX cells in PBS supplemented with 30% Matrigel (BD Biosciences, cat. no. 354234). When tumours reached an\",\n",
            "          \"(BD Biosciences, cat. no. 354234). When tumours reached an average size of 250\\u2013300\\u2009mm3, animals were assigned randomly to control and various treatment groups (n=8\\u20139 each group). For treatment of WHIM16 (passage 7), 18 (passage 8), and 20 (passage 5) were\",\n",
            "          \"of WHIM16 (passage 7), 18 (passage 8), and 20 (passage 5) were used. Tumour bearing mice were gavaged with: (1) Peg400; (2) TAK-117, 140\\u2009mg\\u2009kg\\u22121\\u2009day\\u22121; (3) TAK-228, 1\\u2009mg\\u2009kg\\u22121\\u2009day\\u22121; (4) TAK-117, 140\\u2009mg\\u2009kg\\u22121\\u2009day\\u22121, TAK-228, 1\\u2009mg\\u2009kg\\u22121\\u2009day\\u22121. The mice were\",\n",
            "          \"140\\u2009mg\\u2009kg\\u22121\\u2009day\\u22121, TAK-228, 1\\u2009mg\\u2009kg\\u22121\\u2009day\\u22121. The mice were treated on three consecutive days once daily and then had a 4-day interval. Tumours were measured with external caliper, and volume was calculated as (4\\u03c0/3) \\u00d7 (width/2)2 \\u00d7 (length/2).\",\n",
            "          \"For the lapatinib therapeutic experiments of WHIM6 (passage 7), WHIM 14 (passage 11), WHIM 8 (passage 6) and WHIM 35 (passage 6), 1 \\u00d7 106 tumour cells were added to equal volume of 1:1 mixture of Matrigel and 10% RPMI plus 10% FBS to 4th mammary fat pads\",\n",
            "          \"of Matrigel and 10% RPMI plus 10% FBS to 4th mammary fat pads in female SCID/bg mice (ENVIGO). We then established tumours to an average volume 250\\u2013300\\u2009mm3, and randomized the mice into control and lapatinib groups. We treated the treated group mice with\",\n",
            "          \"and lapatinib groups. We treated the treated group mice with lapatinib chow diet by formulating lapatinib (220\\u2009mg\\u2009kg\\u22121) and processing them into food pellets (by Research Diets Inc), which is supplied for 2 weeks. The numbers of replicates for each group\",\n",
            "          \"supplied for 2 weeks. The numbers of replicates for each group are as followed: WHIM 14 control (n=6) and lapatinib treated (n=5); WHIM 6 control (n=11) and lapatinib treated (n=12); WHIM 8 control (n=16) and lapatinib treated (n=6); WHIM 35 control\",\n",
            "          \"8 control (n=16) and lapatinib treated (n=6); WHIM 35 control (n=17) and lapatinib treated (n=18). We also treated WHIM14 with 100\\u2009mg\\u2009kg\\u22121 lapatinib treatment for 48 days, including both control (n=8) and lapatinib treated (n=9) groups. Further, we tested\",\n",
            "          \"(n=8) and lapatinib treated (n=9) groups. Further, we tested the effect of trastuzumab (30\\u2009mg\\u2009kg\\u22121 weekly with IP injections) and pertuzumab (30\\u2009mg\\u2009kg\\u22121 weekly with IP injections) on PDX tumour growth for 48 days. The experimental groups are as following:\",\n",
            "          \"growth for 48 days. The experimental groups are as following: control group (n=6) treated with physiological saline (vehicle); trastuzumab treated group (n=12); pertuzumab treated group (n=11).\",\n",
            "          \"Statistical testing of the resulting data was conducted using the R programming language. Wilcoxon rank sum test was applied to compare the fold change in tumour volumes after two weeks in the lapatinib treatment experiment, and after 44 days or 48 days\",\n",
            "          \"lapatinib treatment experiment, and after 44 days or 48 days for the additional WHIM14 experiment using low-dose lapatinib and trastuzumab/pertuzumab. One-way ANOVA was applied to compare the treated versus control groups in the PI3K targeted-therapy\",\n",
            "          \"the treated versus control groups in the PI3K targeted-therapy experiments, and a follow-up Tukey's post hoc test was used for the various comparisons in PI3K inhibition experiments. All human tissues for these experiments were processed in compliance\",\n",
            "          \"tissues for these experiments were processed in compliance with NIH regulations and institutional guidelines, and approved by the Washington University, University of North Carolina, or Baylor College of Medicine Institutional Review Board (IRB). All\",\n",
            "          \"College of Medicine Institutional Review Board (IRB). All animal procedures were reviewed and approved by the institutional animal care and use committee at Washington University in St. Louis, University of North Carolina, or Baylor College of Medicine.\",\n",
            "          \"Genomic and proteomic data generation.\\n\\nSomatic mutation\",\n",
            "          \"Sequencing reads were aligned using BWA. Somatic variants were identified using VarScan2 (refs), GATK and Pindel, and annotated based on Ensembl release 70_37. We then filtered out common variants by using variants from the 1000 Genomes and NHLBI\",\n",
            "          \"variants by using variants from the 1000 Genomes and NHLBI projects. We further eliminated mouse contamination by filtering somatic variants that were mapped to mouse reference genome. Somatic mutation calls were validated either using a custom array or\",\n",
            "          \"mutation calls were validated either using a custom array or manually reviewed in IGV. The genomic data of 17 out of the 24 PDXs have been presented before in our previous studies (Supplementary Data 2).\",\n",
            "          \"Copy-number variation\",\n",
            "          \"The segment-based copy-number data were generated using the whole-genome sequencing and exome sequencing data. We then converted the segment-based copy-number data to the gene-based copy-number data by using the RefSeq database (version 20130727). The\",\n",
            "          \"data by using the RefSeq database (version 20130727). The copy-number values were further transformed to the log-R ratio, using the cohort mean for the gene as the reference.\",\n",
            "          \"mRNA expression and virus detection through RNA-seq\",\n",
            "          \"mRNA expression values were calculated from mRNA sequencing data using MapSplice. The resulting RSEM values were normalized within samples to a fixed upper quartile. Upper quartile normalized RSEM data were log2 transformed and the data were median\",\n",
            "          \"RSEM data were log2 transformed and the data were median centred by gene. To quantify virus abundance, we used the VirusScan pipeline (https://github.com/ding-lab/VirusScan) to detect viruses by numbers of virus-supporting reads from RNA-seq data.\",\n",
            "          \"LFQ proteome\",\n",
            "          \"Tumour Sample Generation and Protein Extraction. Patient-derived xenograft breast tumours were processed to cryopulverized powders as described previously. The powders (100\\u2009mg wet weight) were subjected to lysis and protein extraction using a buffer\",\n",
            "          \"were subjected to lysis and protein extraction using a buffer composed of 8 M urea, 50\\u2009mM Tris pH 8.0, 75\\u2009mM NaCl, 1\\u2009mM MgCl2, and 500 units Benzonase. Approximately 1\\u2009mg of total protein extracted was reduced with DTT and subsequently alkylated with\",\n",
            "          \"extracted was reduced with DTT and subsequently alkylated with iodoacetamide. The proteins were then subjected to proteolysis with endoproteinase Lys-C (Wako Chemicals, USA) for \\u223c4\\u2009h at 37\\u2009\\u00b0C. The solution was diluted 4-fold with 25\\u2009mM Tris pH 8.0, 1\\u2009mM\",\n",
            "          \"The solution was diluted 4-fold with 25\\u2009mM Tris pH 8.0, 1\\u2009mM CaCl2 and further digested with trypsin (Promega) for \\u223c12\\u2009h at 37\\u2009\\u00b0C. Digestion was stopped by the addition of TFA to 0.4%, and the precipitate was removed by centrifugation. The peptide\",\n",
            "          \"and the precipitate was removed by centrifugation. The peptide solutions were desalted on Sep-Pak Light C18 cartridges (Waters) and dissolved in 30% ACN, 0.1% TFA before loading on a 300\\u2009\\u03bcm Source 15\\u2009S (GE Healthcare) column for Basic Reversed Phase\",\n",
            "          \"Source 15\\u2009S (GE Healthcare) column for Basic Reversed Phase Chromatography (bRPLC). A linear LC gradient was performed by increasing buffer B from 0 to 70% within 60\\u2009min, where buffer A was aqueous 10\\u2009mM ammonium formate, and buffer B was 90% ACN in 10\\u2009mM\",\n",
            "          \"10\\u2009mM ammonium formate, and buffer B was 90% ACN in 10\\u2009mM ammonium formate. A total of 30 fractions were collected for each WHIM sample (18 WHIMs). Five fractions were then prepared by combining non-contiguously fractions. We analyzed an additional\",\n",
            "          \"combining non-contiguously fractions. We analyzed an additional technical replicate for WHIM2 and WHIM16. The fractions were dried and desalted using a stop-and-go-extraction tip (StageTip) protocol containing 4 \\u00d7 1\\u2009mm C18 extraction disk (3M).\",\n",
            "          \"Liquid Chromatography-Tandem Mass Spectrometry and Protein Identification. Sample analysis was performed via reversed phase LC-MS/MS using a Proxeon 1000 nano LC system coupled to a Q Exactive mass spectrometer (Thermo Scientific, San Jose, CA). The\",\n",
            "          \"mass spectrometer (Thermo Scientific, San Jose, CA). The Proxeon system was configured to trap peptides using a C18 column (3\\u2009cm \\u00d7 100\\u2009\\u03bcm i.d.) with a diverted flow rate (5\\u2009\\u03bcl\\u2009min\\u22121) The trap column was placed in line with the analytical column (15\\u2009cm \\u00d7\",\n",
            "          \"column was placed in line with the analytical column (15\\u2009cm \\u00d7 75\\u2009\\u03bcm i.d., 3.5\\u2009\\u03bcm, 300\\u2009\\u00c5 particle C18, Thermo Scientific, San Jose, CA, USA) before gradient elution of peptides. Analytical separation of all the tryptic peptides was achieved with a linear\",\n",
            "          \"of all the tryptic peptides was achieved with a linear gradient of 2\\u201330% buffer B over 240\\u2009min (250\\u2009nl\\u2009min\\u22121), where buffer A was aqueous 0.1% formic acid, and buffer B was acetonitrile in 0.1% formic acid.\",\n",
            "          \"LC-MS experiments were performed in a data-dependent mode with Full-MS (externally calibrated to a mass accuracy of <5 p.p.m., and a resolution of 70,000 at m/z=200) followed by HCD-MS/MS of the top 20 most intense ions. High-energy collision activated\",\n",
            "          \"the top 20 most intense ions. High-energy collision activated dissociation (HCD)-MS/MS was used to fragment peptides at a normalized collision energy of 27\\u2009eV in the presence of nitrogen. One LC-MS run was performed for each fraction (from 1 process\",\n",
            "          \"One LC-MS run was performed for each fraction (from 1 process technical replicate), except for WHIM2 and WHIM16 where 2 LC\\u2013MS runs were conducted (from 3 process technical replicates), resulting in the production of 100 LC\\u2013MS runs for global peptide\",\n",
            "          \"in the production of 100 LC\\u2013MS runs for global peptide analysis. Mass spectra were processed, and peptide identification was performed using the Andromeda search engine found in MaxQuant software ver. 1.5.0.25. (Max Planck Institute, Germany). All protein\",\n",
            "          \"ver. 1.5.0.25. (Max Planck Institute, Germany). All protein database searches were performed against the RefSeq database (version 20140707). Peptides were identified with a target-decoy approach using a combined database consisting of reverse protein\",\n",
            "          \"using a combined database consisting of reverse protein sequences of the RefSeq human, mouse and common repository of adventitious proteins (cRAP). The cRAP database was obtained from the Global Proteome Machine (ftp://ftp.thegpm.org/fasta/cRAP). Peptide\",\n",
            "          \"Proteome Machine (ftp://ftp.thegpm.org/fasta/cRAP). Peptide inference was made with a FDR of 1% while peptides were assigned to proteins with a protein FDR of 5%. A precursor ion mass tolerance of 20 ppm was used for the first search that allowed for m/z\",\n",
            "          \"of 20 ppm was used for the first search that allowed for m/z retention time recalibration of precursor ions that were then subjected to a main search using a precursor ion mass tolerance of 6 p.p.m. and a product ion mass tolerance 0.5\\u2009Da. Search\",\n",
            "          \"of 6 p.p.m. and a product ion mass tolerance 0.5\\u2009Da. Search parameters included up to two missed cleavages at KR on the sequence, and oxidation of methionine, and protein N terminus acetylation as a dynamic modification. Carbamidomethylation of cysteine\",\n",
            "          \"as a dynamic modification. Carbamidomethylation of cysteine residues was considered as a static modification. Peptide identifications are reported by filtering of reverse and contaminant entries and assigning to their leading razor protein according to\",\n",
            "          \"and assigning to their leading razor protein according to the Occams razor principal. The mass spectrometric data are deposited at the CPTAC Data Coordinating Center as raw and mzML files (https://cptac-data-portal.georgetown.edu).\",\n",
            "          \"Peptide and Protein Quantitation. LFQ was performed based on peak areas. The measured area under the curve of m/z and the retention time-aligned extracted ion chromatograms of peptides were performed via the label-free quantitation module in MaxQuant\",\n",
            "          \"performed via the label-free quantitation module in MaxQuant [ver. 1.5.0.25]. All replicates for each PDX were included in the LFQ experimental design with peptide-level quantitation performed using unique and razor peptide features corresponding to\",\n",
            "          \"using unique and razor peptide features corresponding to identifications filtered with a posterior error probability (PEP) of 0.01, peptide FDR of 0.01 and protein FDR of 0.05. The expression values were median centred in the Perseus software for further\",\n",
            "          \"values were median centred in the Perseus software for further analysis [version 1.5.0.9].\",\n",
            "          \"iTRAQ proteome and phosphoproteome\",\n",
            "          \"We included all 24 of the established breast tumour samples for proteomic characterization using iTRAQ. Tumour tissue samples were maintained in cryovials at \\u221280\\u2009\\u00b0C until cryopulverization using a CP02 Cryprep Pulverizer (Covaris, Woburn, MA). 90\\u2009mg\",\n",
            "          \"using a CP02 Cryprep Pulverizer (Covaris, Woburn, MA). 90\\u2009mg aliquots of cryofractured material were prepared for proteomic processing in aluminum weighing dishes on dry ice using spatulas kept cold in liquid nitrogen, with remaining material reserved for\",\n",
            "          \"cold in liquid nitrogen, with remaining material reserved for other applications. The 90\\u2009mg target was designed to include 40\\u2009mg for each of the collaborating research teams, with an anticipated yield for each team of 1.5\\u20132\\u2009mg protein based on 4\\u20135%\",\n",
            "          \"yield for each team of 1.5\\u20132\\u2009mg protein based on 4\\u20135% recovery. To avoid systematic bias, sample processing was block randomized, with each intrinsic subtype proportionally represented in each processing tranche.\",\n",
            "          \"The reproducibility of the iTRAQ4-plex global proteome and phosphoproteome analysis workflow used in this study has been extensively tested for quantitative reproducibility both within and across laboratories in the CPTAC program. Over a period of several\",\n",
            "          \"laboratories in the CPTAC program. Over a period of several months 5 iTRAQ4-plex replicates were measured at each of the 3 CPTAC proteome analysis centres. Each of these iTRAQ4-plexes contained duplicate measurements for both a basal WHIM2 and a luminal\",\n",
            "          \"duplicate measurements for both a basal WHIM2 and a luminal WHIM16 PDX samples that are also part of this study. A high degree of consistency in the number of proteins identified and correlation in the protein expression was obtained. Pearson correlations\",\n",
            "          \"in the protein expression was obtained. Pearson correlations for replicate proteome and phosphoproteome measurements were very high with a R=0.9 in our previous study and very similar to the correlation observed here for the WHIM13 replicate measurement.\",\n",
            "          \"correlation observed here for the WHIM13 replicate measurement. These data show that our platform provides highly reproducible quantitative measurements for global proteomes and phosphoproteomes.\",\n",
            "          \"Protein extraction, digestion and iTRAQ labelling of peptides from breast cancer tumours. Cryopulverized breast cancer tumour samples tissues (\\u223c2 combined aliquots of 90\\u2009mg tissue weight each) were homogenized in 1,000\\u2009\\u03bcl lysis buffer containing 8M urea,\",\n",
            "          \"were homogenized in 1,000\\u2009\\u03bcl lysis buffer containing 8M urea, 75\\u2009mM NaCl, 1\\u2009mM EDTA in 50\\u2009mM Tris HCl (pH 8), 10\\u2009mM NaF, phosphatase inhibitor cocktail 2 (1:100; Sigma, P5726) and cocktail 3 (1:100; Sigma, P0044), 2\\u2009\\u03bcg\\u2009ml\\u22121 aprotinin (Sigma, A6103),\",\n",
            "          \"3 (1:100; Sigma, P0044), 2\\u2009\\u03bcg\\u2009ml\\u22121 aprotinin (Sigma, A6103), 10\\u2009\\u03bcg\\u2009ml\\u22121 Leupeptin (Roche, #11017101001), and 1\\u2009mM PMSF (Sigma, 78830). Lysates were centrifuged at 20,000 g for 10\\u2009min before measuring protein concentration of the clarified lysates by BCA\",\n",
            "          \"measuring protein concentration of the clarified lysates by BCA assay (Pierce). Protein lysates were subsequently reduced with 5\\u2009mM dithiothreitol (Thermo Scientific, 20291) for 45\\u2009min at room temperature, and alkylated with 10\\u2009mM iodoacetamide (Sigma,\",\n",
            "          \"temperature, and alkylated with 10\\u2009mM iodoacetamide (Sigma, A3221) for 45\\u2009min in the dark. Samples were diluted 4-fold with 50\\u2009mM Tris HCl (pH 8) prior to digesting them with LysC (Wako, 129-02541) for 4\\u2009h and trypsin (Promega, V511X) overnight at a 1:50\",\n",
            "          \"for 4\\u2009h and trypsin (Promega, V511X) overnight at a 1:50 enzyme-to-protein ratio at room temperature overnight on a shaker.\",\n",
            "          \"Digested samples were acidified with formic acid (FA; Fluka, 56302) to a final volumetric concentration of 1% or final pH of \\u223c3\\u20135, and centrifuged at 2,000\\u2009g for 5\\u2009min to clear precipitated urea from peptide lysates. Samples were desalted on C18 SepPak\",\n",
            "          \"urea from peptide lysates. Samples were desalted on C18 SepPak columns (Waters, 100\\u2009mg, WAT036820) and 1\\u2009mg peptide aliquots were dried down using a SpeedVac apparatus.\",\n",
            "          \"Construction of the Common internal Reference Pool. The proteomic and phosphoproteomic analyses of xenograft samples were performed as iTRAQ 4-plex experiments. Quantitative comparison between all samples analyzed was facilitated by the use of iTRAQ\",\n",
            "          \"all samples analyzed was facilitated by the use of iTRAQ reporter ion ratios between each individual sample and a common internal reference sample present in each 4-plex. The reference sample was comprised of 16 of the 24 WHIM tumours analyzed in this\",\n",
            "          \"was comprised of 16 of the 24 WHIM tumours analyzed in this study with equal contribution for each tumour (WHIM numbers 2, 4, 6, 8, 11, 12, 13, 14, 16, 18, 20, 21, 24, 25, 30 and 46). The 24 tumour samples were analyzed in nine independent 4-plex\",\n",
            "          \"The 24 tumour samples were analyzed in nine independent 4-plex experiments, with three individual samples occupying the first three channels of each experiment and the 4th channel being reserved for the reference sample. While eight iTRAQ 4-plex\",\n",
            "          \"reserved for the reference sample. While eight iTRAQ 4-plex experiments were used to analyze the 24 individual WHIM tumour samples, an additional 4-plex experiment was designed to include the WHIM13 sample for process replicate analysis and also internal\",\n",
            "          \"WHIM13 sample for process replicate analysis and also internal reference samples from our human primary breast cancer study and a taxol drug response study (unpublished) to allow cross-referencing of the different datasets.\",\n",
            "          \"iTRAQ labelling, high pH reversed-phase separation and phosphopeptide enrichment of peptide samples. Desalted peptides were labelled with 4-plex iTRAQ reagents according to the manufacturer's instructions (AB Sciex, Foster City, CA). For each 1\\u2009mg peptide\",\n",
            "          \"instructions (AB Sciex, Foster City, CA). For each 1\\u2009mg peptide from each breast tumour sample, 10 units of labelling reagent were used. Peptides were dissolved in 300\\u2009\\u03bcl of 0.5 M triethylammonium bicarbonate (TEAB) (pH 8.5) solution and labelling reagent\",\n",
            "          \"bicarbonate (TEAB) (pH 8.5) solution and labelling reagent was added in 700\\u2009\\u03bcl of ethanol. After 1\\u2009h incubation, 1.5\\u2009ml of 0.05% TFA was added to stop the reaction. Differentially labelled peptides were mixed and subsequently desalted on 500\\u2009mg tC18\",\n",
            "          \"peptides were mixed and subsequently desalted on 500\\u2009mg tC18 SepPak columns. The combined 4\\u2009mg iTRAQ samples per experiment were separated into 24 proteome fractions and 12 phosphoproteome fractions using a 4.6\\u2009mm \\u00d7 250\\u2009mm column RP Zorbax 300 A ExtendC18\",\n",
            "          \"using a 4.6\\u2009mm \\u00d7 250\\u2009mm column RP Zorbax 300 A ExtendC18 column (Agilent, 3.5\\u2009\\u03bcm bead size) on an Agilent 1100 Series HPLC instrument by basic reversed-phase chromatography as described previously. Peptides were separated according to their hydrophobicity\",\n",
            "          \"Peptides were separated according to their hydrophobicity using solvent A (2% acetonitrile, 5\\u2009mM ammonium formate, pH 10) and a nonlinear increasing concentration of solvent B (90% acetonitrile, 5\\u2009mM ammonium formate, pH 10). Phosphopeptides were enriched\",\n",
            "          \"5\\u2009mM ammonium formate, pH 10). Phosphopeptides were enriched using Ni-NTA superflow agarose beads (Qiagen, #1018611) that were stripped of nickel with 100\\u2009mM EDTA and incubated in an aqueous solution of 10\\u2009mM FeCl3 (Sigma, 451649) as described previously.\",\n",
            "          \"of 10\\u2009mM FeCl3 (Sigma, 451649) as described previously. For phosphopeptide enrichment a 80% acetonitrile/0.1% trifluoroacetic acid binding buffer and a 500\\u2009mM dibasic sodium phosphate, pH 7.0, (Sigma, S9763) elution buffer were used. Enriched samples were\",\n",
            "          \"(Sigma, S9763) elution buffer were used. Enriched samples were desalted on StageTips as described before analysis by LC\\u2013MS/MS.\",\n",
            "          \"Analysis of tumour samples by high performance liquid chromatography tandem mass spectrometry (LC\\u2013MS/MS). All peptides were separated with an online nanoflow Proxeon EASY-nLC 1000 UHPLC system (Thermo Fisher Scientific) and analyzed on a benchtop Orbitrap\",\n",
            "          \"(Thermo Fisher Scientific) and analyzed on a benchtop Orbitrap Q Exactive mass spectrometer (Thermo Fisher Scientific) equipped with a nanoflow ionization source (James A. Hill Instrument Services, Arlington, MA, USA). The LC system, column, and platinum\",\n",
            "          \"Arlington, MA, USA). The LC system, column, and platinum wire to deliver electrospray source voltage were connected via a stainless-steel cross (360\\u2009\\u03bcm, IDEX Health & Science, UH-906x). The column was heated to 50\\u2009\\u00b0C using a column heater sleeve\",\n",
            "          \"The column was heated to 50\\u2009\\u00b0C using a column heater sleeve (Phoenix-ST) to prevent overpressurizing of columns during UHPLC separation. 10% of each global proteome sample in a 2\\u2009\\u03bcl injection volume, or 50% of each phosphoproteome sample in a 4 ul\",\n",
            "          \"volume, or 50% of each phosphoproteome sample in a 4 ul injection volume was injected onto an in-house packed 20\\u2009cm \\u00d7 75\\u2009um diameter C18 silica picofrit capillary column (1.9\\u2009\\u03bcm ReproSil-Pur C18-AQ beads, Dr Maisch GmbH, r119.aq; Picofrit 10\\u2009um tip\",\n",
            "          \"C18-AQ beads, Dr Maisch GmbH, r119.aq; Picofrit 10\\u2009um tip opening, New Objective, PF360-75-10-N-5). Mobile phase flow rate was 200\\u2009nl\\u2009min\\u22121, comprised of 3% acetonitrile/0.1% formic acid (Solvent A) and 90% acetonitrile /0.1% formic acid (Solvent B), and\",\n",
            "          \"A) and 90% acetonitrile /0.1% formic acid (Solvent B), and the 110-minute LC-MS/MS method consisted of a 10-min column-equilibration procedure, a 20-min sample-loading procedure, and the following gradient profile: (min:%B) 0:2; 1:6; 85:30; 94:60; 95;90;\",\n",
            "          \"gradient profile: (min:%B) 0:2; 1:6; 85:30; 94:60; 95;90; 100:90; 101:50; 110:50 (last two steps at 500\\u2009nl\\u2009min\\u22121 flowrate). Data-dependent acquisition was performed using Xcalibur QExactive v2.1 software in positive ion mode at a spray voltage of 2.00\\u2009kV.\",\n",
            "          \"software in positive ion mode at a spray voltage of 2.00\\u2009kV. MS1 Spectra were measured with a resolution of 70,000, an AGC target of 3e6 and a mass range from 300 to 1,800\\u2009m/z. Up to 12 MS2 spectra per duty cycle were triggered at a resolution of 17,500,\",\n",
            "          \"per duty cycle were triggered at a resolution of 17,500, an AGC target of 5e4, an isolation window of 2.5\\u2009m/z, a maximum ion time of 120\\u2009msec, and a normalized collision energy of 28. Peptides that triggered MS2 scans were dynamically excluded from\",\n",
            "          \"that triggered MS2 scans were dynamically excluded from further MS2 scans for 20\\u2009s. Charge state screening was enabled to reject precursor charge states that were unassigned, 1, or >6. Peptide match was enabled for monoisotopic precursor mass assignment.\",\n",
            "          \"Protein-peptide identification, phosphosite localization, and quantitation. All MS data were interpreted using the Spectrum Mill software package v5.1 (for comparison with proteomes of human breast tumours from our previous study) and v6.0 pre-release\",\n",
            "          \"breast tumours from our previous study) and v6.0 pre-release (Agilent Technologies, Santa Clara, CA, USA) co-developed by the authors. Similar MS/MS spectra acquired on the same precursor m/z within \\u00b1 45\\u2009s were merged. MS/MS spectra were excluded from\",\n",
            "          \"m/z within \\u00b1 45\\u2009s were merged. MS/MS spectra were excluded from searching if they failed the quality filter by not having a sequence tag length>0 (that is, minimum of two masses separated by the in-chain mass of an amino acid) or did not have a precursor\",\n",
            "          \"the in-chain mass of an amino acid) or did not have a precursor MH+ in the range of 750\\u20136,000. MS/MS spectra from were searched against a database consisting of RefSeq release 60 containing 31,767 human proteins, 24,821 mouse proteins, and an appended set\",\n",
            "          \"human proteins, 24,821 mouse proteins, and an appended set of 85 common laboratory contaminant proteins (RefSeq.20130727-Human.20130730-MouseNR.mm13.contams). Scoring parameters were ESI-QEXACTIVE-HCD-v2, for whole proteome datasets, and\",\n",
            "          \"were ESI-QEXACTIVE-HCD-v2, for whole proteome datasets, and ESI-QEXACTIVE-HCD-v3 parameters were for phosphoproteome datasets. All spectra were allowed \\u00b120 p.p.m. mass tolerance for precursor and product ions, 40% minimum matched peak intensity, and\",\n",
            "          \"and product ions, 40% minimum matched peak intensity, and trypsin allow P enzyme specificity with up to four missed cleavages. Fixed modifications were carbamidomethylation at cysteine. iTRAQ labelling was required at lysine, but peptide N-termini were\",\n",
            "          \"labelling was required at lysine, but peptide N-termini were allowed to be either labelled or unlabelled. Allowed variable modifications for whole proteome datasets were acetylation of protein N-termini, oxidized methionine, deamidation of asparagine,\",\n",
            "          \"N-termini, oxidized methionine, deamidation of asparagine, pyro-glutamic acid at peptide N-terminal glutamine, and pyro-carbamidomethylation at peptide N-terminal cysteine with a precursor MH+ shift range of \\u221218 to 64\\u2009Da. Allowed variable modifications\",\n",
            "          \"MH+ shift range of \\u221218 to 64\\u2009Da. Allowed variable modifications for phosphoproteome dataset were revised to disallow deamidation and allow phosphorylation of serine, threonine, and tyrosine with a precursor MH+ shift range of 0 to 272\\u2009Da.\",\n",
            "          \"Identities interpreted for individual spectra were automatically designated as confidently assigned using the Spectrum Mill autovalidation module to use target-decoy based FDR estimates to apply score threshold criteria via two-step strategies. For the\",\n",
            "          \"apply score threshold criteria via two-step strategies. For the whole proteome datasets thresholding was done at the spectral and protein levels. For the phosphoproteome datasets thresholding was done at the spectral level. In step 1, peptide\",\n",
            "          \"thresholding was done at the spectral level. In step 1, peptide autovalidation was done first and separately for each iTRAQ 4-plex experiment consisting of either 25 LC\\u2013MS/MS runs (whole proteome) or 13 LC\\u2013MS/MS runs (phosphoproteome) using an auto\",\n",
            "          \"proteome) or 13 LC\\u2013MS/MS runs (phosphoproteome) using an auto thresholds strategy with a minimum sequence length of 7(whole proteome) or 8 (phosphoproteome), automatic variable range precursor mass filtering, and score and delta Rank1\\u2013Rank2 score\",\n",
            "          \"precursor mass filtering, and score and delta Rank1\\u2013Rank2 score thresholds optimized to yield a spectral level FDR estimate for precursor charges 2 through 4 of <0.6% for each precursor charge state in each LC\\u2013MS/MS run. For precursor charges 5\\u20136,\",\n",
            "          \"charge state in each LC\\u2013MS/MS run. For precursor charges 5\\u20136, thresholds were optimized to yield a spectral level FDR estimate of <0.3% across all runs per iTRAQ 4-plex experiment (instead of each run), to achieve reasonable statistics, since many fewer\",\n",
            "          \"each run), to achieve reasonable statistics, since many fewer spectra are generated for the higher charge states.\",\n",
            "          \"In step 2 for the whole proteome datasets, protein polishing autovalidation was applied separately to each iTRAQ 4-plex experiment to further filter the PSM's using a target protein-level FDR threshold of zero. The primary goal of this step was to\",\n",
            "          \"FDR threshold of zero. The primary goal of this step was to eliminate peptides identified with low scoring peptide spectrum matches (PSM's) that represent proteins identified by a single peptide, so-called \\u2018one-hit wonders'. After assembling protein\",\n",
            "          \"peptide, so-called \\u2018one-hit wonders'. After assembling protein groups from the autovalidated PSM's, protein polishing determined the maximum protein level score of a protein group that consisted entirely of distinct peptides estimated to be false-positive\",\n",
            "          \"entirely of distinct peptides estimated to be false-positive identifications (PSM's with negative delta forward-reverse scores). PSM's were removed from the set obtained in the initial peptide-level autovalidation step if they contributed to protein\",\n",
            "          \"autovalidation step if they contributed to protein groups that have protein scores below the max false-positive protein score. In the filtered results each identified protein detected in an iTRAQ 4-plex experiment was comprised of multiple peptides unless\",\n",
            "          \"4-plex experiment was comprised of multiple peptides unless a single excellent scoring peptide was the sole match. For the whole proteome datasets the above criteria yielded FDR of <0.5% at the peptide-spectrum match level and <0.8% at the distinct\",\n",
            "          \"at the peptide-spectrum match level and <0.8% at the distinct peptide level for each iTRAQ 4-plex experiment. After assembling proteins with all the PSMs from all the iTRAQ 4-plex experiments together the aggregate FDR estimates were 0.42% at the at the\",\n",
            "          \"together the aggregate FDR estimates were 0.42% at the at the peptide-spectrum match level, 1.5% at the distinct peptide level, and <0.01% (1/11,372) at the protein group level. Since the protein level FDR estimate neither explicitly required a minimum\",\n",
            "          \"level FDR estimate neither explicitly required a minimum number of distinct peptides per protein nor adjusted for the number of possible tryptic peptides per protein, it may underestimate false positive protein identifications for large proteins observed\",\n",
            "          \"positive protein identifications for large proteins observed only on the basis of multiple low scoring PSMs.\",\n",
            "          \"In calculating scores at the protein level and reporting the identified proteins, redundancy was addressed in the following manner: the protein score was the sum of the scores of distinct peptides. A distinct peptide was the single highest scoring\",\n",
            "          \"peptides. A distinct peptide was the single highest scoring instance of a peptide detected through an MS/MS spectrum. MS/MS spectra for a particular peptide may have been recorded multiple times, (that is, as different precursor charge states, in adjacent\",\n",
            "          \"(that is, as different precursor charge states, in adjacent bRP fractions, modified by deamidation at Asn or oxidation of Met, or different phosphosite localization) but were still counted as a single distinct peptide. When a peptide sequence >8 residues\",\n",
            "          \"a single distinct peptide. When a peptide sequence >8 residues long was contained in multiple protein entries in the sequence database, the proteins were grouped together and the highest scoring one and its accession number were reported. In some cases\",\n",
            "          \"one and its accession number were reported. In some cases when the protein sequences were grouped in this manner there were distinct peptides that uniquely represented a lower scoring member of the group (isoforms, family members, and different species).\",\n",
            "          \"of the group (isoforms, family members, and different species). Each of these instances spawned a subgroup. Multiple subgroups were reported and counted towards the total number of proteins, and were given related protein subgroup numbers (for example,\",\n",
            "          \"and were given related protein subgroup numbers (for example, 3.1 and 3.2: group 3, subgroups 1 and 2). To better dissect the tumour/stroma (human/mouse) origin of orthologous proteins in this xenograft experiment, the inclusion of peptides contributing\",\n",
            "          \"xenograft experiment, the inclusion of peptides contributing to each subgroup was restricted by enabling the subgroup-specific (SGS) option in Spectrum Mill. Only subgroup-specific peptide sequences were counted toward each subgroup's count of distinct\",\n",
            "          \"sequences were counted toward each subgroup's count of distinct peptides and protein level TMT quantitation. The SGS option omits peptides that are shared between subgroups. If evidence for BOTH human and mouse peptides from an orthologous protein were\",\n",
            "          \"BOTH human and mouse peptides from an orthologous protein were observed, then peptides that cannot distinguish the two (shared) were ignored. However, the peptides shared between species were retained if there was specific evidence for only one of the\",\n",
            "          \"retained if there was specific evidence for only one of the species, thus yielding a single subgroup attributed to only the single species consistent with the specific peptides. Furthermore, if all peptides observed for a protein group were shared between\",\n",
            "          \"all peptides observed for a protein group were shared between species, thus yielding a single subgroup composed of indistinguishable species, then all peptides were retained (the column in Supplementary Data 3 numSpeciesPresentR1 will have a value of 2 in\",\n",
            "          \"Data 3 numSpeciesPresentR1 will have a value of 2 in such cases). Assembly of confidently identified PSM's yielded 20,480 total protein subgroups from 11,372 protein groups. Human and mouse ortholog proteins were typically arranged into individual\",\n",
            "          \"mouse ortholog proteins were typically arranged into individual subgroups.\",\n",
            "          \"In step 2 for the phosphoproteome datasets a phosphosite table were assembled with columns for individual iTRAQ 4-plex experiments and rows for individual phosphosites. PSM's were combined into a single row for all non-conflicting observations of a\",\n",
            "          \"into a single row for all non-conflicting observations of a particular phosphosite. (that is, different missed cleavage forms, different precursor charges, confident and ambiguous localizations, different sample handling modifications). For related\",\n",
            "          \"different sample handling modifications). For related peptides neither observations with a different number of phosphosites nor different confident localizations were allowed to be combined. Selecting the representative peptide from the combined\",\n",
            "          \"Selecting the representative peptide from the combined observations was done such that once confident phosphosite localization was established, higher identification scores and longer peptide lengths are preferable. After assembling the phosphosite table\",\n",
            "          \"lengths are preferable. After assembling the phosphosite table a polishing step was applied to further filter the phosphosites with the primary goal of eliminating phosphosites with representative peptides identified through low scoring peptide spectrum\",\n",
            "          \"peptides identified through low scoring peptide spectrum matches (PSM's) that were observed in only a few experiments. The initial table of representative peptides for 82,030 phosphosites had an aggregate FDR of 3.3% at phosphosite-level. The table was\",\n",
            "          \"an aggregate FDR of 3.3% at phosphosite-level. The table was sorted by identification score and then by number of iTRAQ 4-plex experiments in which the phosphosite was observed. The cumulative FDR trend showed inflection points at an identification score\",\n",
            "          \"FDR trend showed inflection points at an identification score of \\u223c8. Phosphosites with an identification score<8.0 observed in <3/9 experiments were therefore removed, yielding 68,385 phosphosites with an aggregate FDR of 0.34% at the phosphosite level.\",\n",
            "          \"with an aggregate FDR of 0.34% at the phosphosite level. While the Spectrum Mill identification score is based on the number of matching peaks, their ion type assignment, and the relative height of unmatched peaks, the phosphosite localization score is\",\n",
            "          \"of unmatched peaks, the phosphosite localization score is the difference in identification score between the top two localizations. The score threshold for confident localization (>1.1) essentially corresponds to at least 1 b or y ion located between two\",\n",
            "          \"corresponds to at least 1 b or y ion located between two candidate sites that has a peak height 10% of the tallest fragment ion (neutral losses of phosphate from the precursor and related ions as well as immonium and iTRAQ reporter ions are excluded from\",\n",
            "          \"as well as immonium and iTRAQ reporter ions are excluded from the relative height calculation). The ion type scores for b-H3PO4, y-H3PO4, b-H2O, and y-H2O ion types are all set to 0.5. This prevents inappropriate confident localization assignment when a\",\n",
            "          \"prevents inappropriate confident localization assignment when a spectrum lacks primary b or y ions between two possible sites but contains ions that can be assigned as either phosphate loss ions for one localization or water loss ions for another\",\n",
            "          \"loss ions for one localization or water loss ions for another localization. In aggregate, 66.3% of the reported phosphosites were fully localized to a particular serine, threonine, or tyrosine residue.\",\n",
            "          \"Relative abundances of proteins and phosphosites were determined in Spectrum Mill using iTRAQ reporter ion intensity ratios from each PSM. A protein-level or phosphosite-level iTRAQ ratio was calculated as the median of all PSM level ratios contributing\",\n",
            "          \"calculated as the median of all PSM level ratios contributing to a protein subgroup or phosphosite remaining after excluding those PSM's lacking an iTRAQ label, having a negative delta forward-reverse score (half of all false-positive identifications), or\",\n",
            "          \"score (half of all false-positive identifications), or having a precursor ion purity<50% (MS/MS has significant precursor isolation contamination from co-eluting peptides). Unless stated otherwise for a particular analysis, the following considerations\",\n",
            "          \"for a particular analysis, the following considerations apply to the tumour/stroma (human/mouse) origin of a protein in this xenograft experiment. For the proteome dataset, only PSM's from subgroup-specific peptide sequences contributed to the protein\",\n",
            "          \"subgroup-specific peptide sequences contributed to the protein level quantitation (see protein subgrouping description above). A protein detected with all contributing PSM's shared between human and mouse was considered to be human. For the\",\n",
            "          \"between human and mouse was considered to be human. For the phosphoproteome dataset, a phosphosite was considered to be mouse if the contributing PSM's were distinctly mouse and human if they were either distinctly human or shared between human and mouse.\",\n",
            "          \"were either distinctly human or shared between human and mouse. A 2-component Gaussian mixture model-based normalization approach was used to centre the distribution of iTRAQ log-ratios around zero to nullify the effect of differential protein loading\",\n",
            "          \"zero to nullify the effect of differential protein loading and/or systematic MS variation. Downstream analyses presented in the main figures were restricted to proteins/phosphosites quantified in at least 10 out of the 24 samples with non-missing values\",\n",
            "          \"in at least 10 out of the 24 samples with non-missing values (Supplementary Fig. 3), with the exception of the previously described mRNA-protein correlation analysis requiring quantification in 30%, or 8 out of 24, PDX samples. Specific filtering\",\n",
            "          \"in 30%, or 8 out of 24, PDX samples. Specific filtering procedures are noted in descriptions of the relevant methods.\",\n",
            "          \"Bioinformatics analyses.\\n\\nCross data-type integration\",\n",
            "          \"All gene names were converted to HUGO Gene Nomenclature Committee's approved gene names for comparison across levels and datasets. For mRNA and protein expressions, expression values were collapsed across transcripts or isoforms to the corresponding gene\",\n",
            "          \"across transcripts or isoforms to the corresponding gene using the highest mean when there were two transcripts or isoforms, or the value with the highest connectivity when there were more than three transcripts or isoforms as implemented in the WGCNA R\",\n",
            "          \"three transcripts or isoforms as implemented in the WGCNA R package.\",\n",
            "          \"mRNA-protein correlation\",\n",
            "          \"Spearman correlations between normalized RSEM values and protein quantifications were calculated for genes that were observed in at least 30% of samples for both RNA-seq and mass-spectrometry as previously described. KEGG pathway enrichment analysis of\",\n",
            "          \"as previously described. KEGG pathway enrichment analysis of the correlation was carried out using a Kolmogorov\\u2013Smirnov test, and P values were adjusted using the Benjamini-Hochberg procedure.\",\n",
            "          \"Proteomic clustering and subtyping\",\n",
            "          \"We first applied filtering for protein and phosphosite markers observed in at least 10 samples and sufficient deviations across samples. We used a 2 s.d. threshold for iTRAQ proteome and 2.5 s.d. for the phosphoproteome. We applied the same protein marker\",\n",
            "          \"for the phosphoproteome. We applied the same protein marker to conduct LFQ proteome and PDX vs. human proteome co-clustering. For the co-clustering with human proteome, we further selected for markers that had higher than 2 s.d.'s in the merged proteome\",\n",
            "          \"markers that had higher than 2 s.d.'s in the merged proteome and showed non-differential expression between human and PDX (FDR>0.3, t-test). The subsequent hierarchical clustering was conducted using the complete agglomeration method of hclust as\",\n",
            "          \"conducted using the complete agglomeration method of hclust as implemented in the heatmap2 R package.\",\n",
            "          \"Differential expression analysis\",\n",
            "          \"Differential expression testing of each protein in the LFQ and iTRAQ datasets was conducted using the student's t-test, and P values were adjusted using the Benjamini-Hochberg procedure. For gene set enrichment analysis, we conducted the Wilcoxon Rank Sum\",\n",
            "          \"set enrichment analysis, we conducted the Wilcoxon Rank Sum Test to test for changes in the t-statistics ranks of protein members in each of the KEGG signalling pathway, and again adjusted P values using the Benjamini-Hochberg procedure.\",\n",
            "          \"Druggable genes and mutations\",\n",
            "          \"We compiled a list of druggable genes based on extensive curation of public databases: Tumor Alterations Relevant for GEnomics-driven Therapy (TARGET, version 3, assessed on 6/15/2015), Personalized Cancer Therapy (PCT, assessed on 3/15/2015), GDKD\",\n",
            "          \"Personalized Cancer Therapy (PCT, assessed on 3/15/2015), GDKD (Gene-Drug Knowledge Database, version 11.0, assessed on 4/10/2015), CancerDR (assessed on 2/6/2015), My Cancer Genome (assessed on 9/11/2014), and DrugBank (assessed on 9/21/2015). We curated\",\n",
            "          \"on 9/11/2014), and DrugBank (assessed on 9/21/2015). We curated the list based on evidence level and literature, as well as IC50 data when available. The final list used for the analysis included 76 druggable genes (Supplementary Data 15).\",\n",
            "          \"Druggable outlier analysis\",\n",
            "          \"To discover expression outliers, we utilized a strategy incorporating multiple steps: first, we limited our search to genes that are in the druggable gene list. We then narrowed down the list to genes that are observed in at least 10 samples in the\",\n",
            "          \"list to genes that are observed in at least 10 samples in the dataset under investigation. Outlier expressions were defined as values that are greater than 1.5 interquartile ranges (IQRs) above the third quartile (Q3). To rank order outlier expression for\",\n",
            "          \"the third quartile (Q3). To rank order outlier expression for each gene, we calculated an outlier score defined as:\",\n",
            "          \"By definition, genes with outlier score greater than 1.5 are considered as expression outliers. Outlier score for each gene were ranked within the sample to select the most promising druggable targets. For CNV outliers, we required them to have outlier\",\n",
            "          \"targets. For CNV outliers, we required them to have outlier scores above 1 in at least another expression level. For validation of the proteomic druggable outliers, we counted the numbers of the same protein or phosphosite outliers observed in the\",\n",
            "          \"of the same protein or phosphosite outliers observed in the parallel-processed cohort of 77 human breast cancer samples. Due to the lower numbers in this larger cohort, we considered both proteins with outlier score greater than 1 and the top 2 outliers\",\n",
            "          \"with outlier score greater than 1 and the top 2 outliers of each human sample as validating outliers.\",\n",
            "          \"Pathway activation analysis\",\n",
            "          \"We first collapsed the phosphosites to gene-level phosphorylation values by averaging the phosphosite expressions observed for each gene. Then, we converted the phosphoproteomic expression values from iTRAQ to modified z-scores normalized against the\",\n",
            "          \"values from iTRAQ to modified z-scores normalized against the cohort as described in Hoaglin et al. (How to Detect and Handle Outliers). We then used the Wilcoxon Rank Sum Test to test for changes in the phosphorylation z-score ranks of protein members in\",\n",
            "          \"in the phosphorylation z-score ranks of protein members in each of the KEGG signalling pathway. The resulting P values were adjusted using the Benjamini\\u2013Hochberg procedure.\",\n",
            "          \"Data availability\\n\\nAll mass spectra, in the original instrument vendor format, contributing to this study may be downloaded from: https://cptac-data-portal.georgetown.edu/cptacPublic/ for the study name: TCGA Breast Cancer.\\n\\nAdditional information\",\n",
            "          \"Additional information\\n\\nHow to cite this article: Huang, K.-l. et al. Proteogenomic integration reveals therapeutic targets in breast cancer xenografts. Nat. Commun. 8, 14864 doi: 10.1038/ncomms14864 (2017).\",\n",
            "          \"Publisher's note: Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.\\n\\nSupplementary Material\",\n",
            "          \"Supplementary Material\\n\\nC.M. Perou is on the board of directors, has ownership interest (including patents), and is a consultant/advisory board member for bioclassifier LLC. The other authors declare no competing financial interests.\",\n",
            "          \"Author contributions L.D. directed data analysis. L.D., M.J.E., S.A.C., X.C., H.R. and R.R.T. designed and supervised research and experiment. K.-l.H., S.L., P.M., H.P.G., S.C., C.Y., D.R.M., K.R.C., C.R.K., J.W., K.V.R., D.F., M.D.M., S.M.K., S.S. and\",\n",
            "          \"K.R.C., C.R.K., J.W., K.V.R., D.F., M.D.M., S.M.K., S.S. and B.N. analyzed the data. K.-l.H., M.C.W. and D.R.M. performed statistical analysis. S.L. established the PDX models. S.L., Z.G., S.M.K., M.T., J.U., J.E.S., J.H., P.E.-G., P.S., C.M. and C.M.P.\",\n",
            "          \"M.T., J.U., J.E.S., J.H., P.E.-G., P.S., C.M. and C.M.P. performed functional validation experiments. J.W.Q., P.M., C.M., L.X. and H.P.G. performed data collection and tissue acquisition. M.A.W., K.-l.H., C.Y., S.C. and L.D. prepared figures and tables.\",\n",
            "          \"K.-l.H., C.Y., S.C. and L.D. prepared figures and tables. M.J.E., S.R.D. and S.L. provided samples. K.-l.H. and L.D. wrote the manuscript. M.J.E., M.C.W., P.M., S.A.C., X.C., R.R.T., C.M., K.-l.H. and L.D. revised the manuscript. All authors approved and\",\n",
            "          \"and L.D. revised the manuscript. All authors approved and contributed to the preparation of the manuscript.\",\n",
            "          \"04/25/2017\\n\\nThe original version of this Article contained a typographical error in the spelling of the author Beifang Niu, which was incorrectly given as Beifung Niu. This has now been corrected in both the PDF and HTML versions of the Article.\",\n",
            "          \"Modelling human breast cancer with patient-derived xenografts (n=24).\",\n",
            "          \"(a) Illustration of generation and proteogenomic characterization of breast cancer xenograft models. (b) Somatic mutations of significantly mutated genes of human breast tumour were recapitulated in xenograft models. Mutation data for 23 WHIMs are shown\",\n",
            "          \"in xenograft models. Mutation data for 23 WHIMs are shown (exome data were not available for WHIM47). (c) Variant allele fraction analysis showed clonal representation was consistent between human breast tumour and xenografts. Genomic driver events,\",\n",
            "          \"human breast tumour and xenografts. Genomic driver events, including missenses and truncations in TP53 and PIK3CA, were retained in the xenograft models. Each colour represents one xenograft sample.\",\n",
            "          \"Proteogenomic correlation analysis in PDX samples.\",\n",
            "          \"(a) Correlation between mRNA and iTRAQ protein expression levels identified pathways with significantly concordant or discordant mRNA-protein expressions. Genes were aligned along the x axis by the rank of their Spearman correlation coefficient between\",\n",
            "          \"by the rank of their Spearman correlation coefficient between mRNA and protein expression levels. Each colour represents one significantly associated pathway, and each bar represents one gene in the pathway. (b) Proteogenomic summary of xenograft shows\",\n",
            "          \"in the pathway. (b) Proteogenomic summary of xenograft shows relationships among mutation, CNV (normalized log-R ratio), mRNA (log-transformed and normalized RSEM values), proteomic (normalized log2 ratio to reference), and phosphoproteomic expression\",\n",
            "          \"log2 ratio to reference), and phosphoproteomic expression (normalized log2 ratio to reference) levels of breast cancer-related genes in 24 PDX samples across 4 intrinsic subtypes. Expression values from each dataset were calculated as described (Methods)\",\n",
            "          \"values from each dataset were calculated as described (Methods) and truncated to a maximum of 10 and a minimum of \\u221210 for visualization.\",\n",
            "          \"Transcriptomic and proteomic clustering of breast cancer PDX and human samples.\",\n",
            "          \"(a) Transcriptomic clustering of PDX breast tumours based on the PAM50 gene expression markers. (b) Proteomic clustering of PDX breast tumours based on the top 436 variably expressed proteins. (c) Phosphoproteomic clustering of PDX breast tumours based on\",\n",
            "          \"(c) Phosphoproteomic clustering of PDX breast tumours based on the top 1,737 variably expressed phosphosites. (d) Proteomic clustering using only 133 non-differential expressed proteins between WHIM and human breast tumour samples. The clustering\",\n",
            "          \"between WHIM and human breast tumour samples. The clustering reproduced the basal-enriched and luminal-enriched clusters, where PDX (n=24) and TCGA human breast tumour samples (n=77) cluster based on their subtypes. The non-differentially expressed\",\n",
            "          \"based on their subtypes. The non-differentially expressed proteins were identified through a t-test with FDR>0.3 between the PDX and the TCGA human tumour samples. The PDX tumours are labelled by their WHIM IDs whereas the human tumours are not labelled.\",\n",
            "          \"Activated signalling pathways detected through pathway phosphorylation enrichment analysis.\",\n",
            "          \"(a) Activation of the MAPK signalling pathway in WHIM9. (b) Activation of the Ras signalling pathway in WHIM12. (c) Activation of the NF\\u03baB pathway in WHIM17. (d) Activation of the NF\\u03baB pathway in WHIM46. Phosphorylation levels of each protein in the\",\n",
            "          \"in WHIM46. Phosphorylation levels of each protein in the pathway relative to the cohort of 24 PDX models are shown by the colour scale of red (high) to blue (low). Proteins with no phosphorylation data are coloured in green.\",\n",
            "          \"Druggable events identified by expression outlier analysis.\",\n",
            "          \"Druggable outlier events identified at (a) the mRNA and protein and (b) protein and phosphopeptide expression levels. Each colour represents one xenograft sample. Key outlier events validated in this study are labelled by text. (c) Immunochemistry\",\n",
            "          \"in this study are labelled by text. (c) Immunochemistry staining verified outlier expression of HER2 in WHIM8 and WHIM35, RAF1 in WHIM9, and FGFR2 in WHIM16. Scale bar: 50\\u2009\\u03bcm.\",\n",
            "          \"Druggable targets identified through proteogenomic analysis in PDX and human breast cancer.\",\n",
            "          \"(a) Outlier analysis revealed potentially druggable events in the RTK, PI3K, MAPK signalling, genome integrity pathways at various frequency and magnitudes across four breast cancer subtypes. Selected genes with any outlier score greater than 2.5 or in\",\n",
            "          \"Selected genes with any outlier score greater than 2.5 or in the key oncogenic pathways, including the PI3K, RTK, MAPK signalling pathways, are shown. (b) Comparison of protein expression outliers of selected druggable genes in PDX and human breast\",\n",
            "          \"outliers of selected druggable genes in PDX and human breast tumours. (c) Comparison of overexpressed phosphosite outliers of selected druggable genes in PDX and human breast tumours. Key outlier events reaching the outlier definition threshold or\",\n",
            "          \"Key outlier events reaching the outlier definition threshold or validated in this study are labelled by text. For the box plots in (b) and (c), the center line indicates median of the protein/phosphosite expression in the human and PDX cohort. The upper\",\n",
            "          \"expression in the human and PDX cohort. The upper and lower hinges correspond to the 25th and 75th percentiles; the upper whisker corresponds to 1.5 \\u00d7 IQR (inter-quartile range) above the 25 percentile and the lower whisker corresponds to 1.5 \\u00d7 IQR below\",\n",
            "          \"percentile and the lower whisker corresponds to 1.5 \\u00d7 IQR below the 75 percentile.\",\n",
            "          \"Targeted treatments of breast cancer xenografts.\",\n",
            "          \"(a) Western blot of HER2 protein and HER2 p.Y1248 expression levels in 5 WHIM models (WHIM6, WHIM8, WHIM12, WHIM14 and WHIM35). (b) In vivo treatment responses to lapatinib in 4 PDX models including two HER2 positive lines (WHIM8 and WHIM35) and two basal\",\n",
            "          \"two HER2 positive lines (WHIM8 and WHIM35) and two basal lines (WHIM6 and WHIM14). The response is measured in fold change (%) of tumour volumes after 2 weeks of vehicle or lapatinib treatment. (c) Immunochemistry staining of AKT phosphorylation status in\",\n",
            "          \"(c) Immunochemistry staining of AKT phosphorylation status in WHIM16, WHIM18 and WHIM20 in response to PI3K inhibitor TAK-117, mTOR inhibitor TAK128, and TAK-117/TAK-128 combined. (d) In vivo treatment responses to PI3K/mTOR inhibitors (TAK-117/TAK-128)\",\n",
            "          \"treatment responses to PI3K/mTOR inhibitors (TAK-117/TAK-128) of WHIM16, WHIM18 and WHIM20 (n=8-9 for each control or experimental group). Values were represented by average tumour volume (mm3) every 3 days following treatment (error bars: s.e.m.).\"\n",
            "        ]\n",
            "      }\n",
            "    ]\n",
            "  },\n",
            "  \"PDC000278\": {\n",
            "    \"study_id\": \"3ea1fbb1-0f05-447e-853e-639338442830\",\n",
            "    \"pdc_study_id\": \"PDC000278\",\n",
            "    \"study_submitter_id\": \"VU Normal Colon Epithelium - Proteome\",\n",
            "    \"program_id\": \"10251935-5540-11e8-b664-00a098d917f8\",\n",
            "    \"project_id\": \"48af5040-5546-11e8-b664-00a098d917f8\",\n",
            "    \"study_name\": \"VU Normal Colon Epithelium - Proteome\",\n",
            "    \"program_name\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"project_name\": \"CPTAC2 Retrospective\",\n",
            "    \"disease_type\": \"Other\",\n",
            "    \"primary_site\": \"Not Reported\",\n",
            "    \"analytical_fraction\": \"Proteome\",\n",
            "    \"experiment_type\": \"Label Free\",\n",
            "    \"embargo_date\": null,\n",
            "    \"cases_count\": 30,\n",
            "    \"aliquots_count\": 60,\n",
            "    \"protocol_id\": \"67e1fa76-3932-42cd-ba8a-01d71090674e\",\n",
            "    \"protocol_submitter_id\": \"VU_Protocol_Summary_S019\",\n",
            "    \"program_submitter_id\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"protocol_name\": \"VU_Protocol_Summary_S019\",\n",
            "    \"protocol_date\": \"2021-02-22\",\n",
            "    \"document_name\": \"VU_Tissue_Prep_By_bRPLC_1.0_20120828\",\n",
            "    \"quantitation_strategy\": \"LC-MS label-free quantitation analysis\",\n",
            "    \"label_free_quantitation\": \"Label-free protein level quantitation\",\n",
            "    \"labeled_quantitation\": \"N/A\",\n",
            "    \"isobaric_labeling_reagent\": \"N/A\",\n",
            "    \"reporter_ion_ms_level\": \"N/A\",\n",
            "    \"starting_amount\": \"200.00\",\n",
            "    \"starting_amount_uom\": \"\\u00b5g\",\n",
            "    \"digestion_reagent\": \"Trypsin\",\n",
            "    \"alkylation_reagent\": \"Iodoacetamide (IAA)\",\n",
            "    \"enrichment_strategy\": \"Proteome\",\n",
            "    \"enrichment\": \"None\",\n",
            "    \"chromatography_dimensions_count\": \"2\",\n",
            "    \"one_d_chromatography_type\": \"bRPLC\",\n",
            "    \"two_d_chromatography_type\": \"RPLC\",\n",
            "    \"fractions_analyzed_count\": \"15\",\n",
            "    \"column_type\": \"C18\",\n",
            "    \"amount_on_column\": null,\n",
            "    \"amount_on_column_uom\": null,\n",
            "    \"column_length\": \"11.0\",\n",
            "    \"column_length_uom\": \"cm\",\n",
            "    \"column_inner_diameter\": \"100.00\",\n",
            "    \"column_inner_diameter_uom\": \"\\u00b5m\",\n",
            "    \"particle_size\": \"5.0\",\n",
            "    \"particle_size_uom\": \"\\u00b5m\",\n",
            "    \"particle_type\": \"300 \\u00c5, Phenomenex \",\n",
            "    \"gradient_length\": \"95.0\",\n",
            "    \"gradient_length_uom\": \"min\",\n",
            "    \"instrument_make\": \"Thermo Fisher Scientific\",\n",
            "    \"instrument_model\": \"LTQ Orbitrap Velos\",\n",
            "    \"dissociation_type\": \"CID\",\n",
            "    \"ms1_resolution\": \"60000\",\n",
            "    \"ms2_resolution\": null,\n",
            "    \"dda_topn\": \"Top 8\",\n",
            "    \"normalized_collision_energy\": \"35\",\n",
            "    \"acquistion_type\": \"DDA\",\n",
            "    \"dia_multiplexing\": \"N/A\",\n",
            "    \"dia_ims\": \"N/A\",\n",
            "    \"analytical_technique\": null,\n",
            "    \"chromatography_instrument_make\": null,\n",
            "    \"chromatography_instrument_model\": null,\n",
            "    \"polarity\": null,\n",
            "    \"reconstitution_solvent\": null,\n",
            "    \"reconstitution_volume\": null,\n",
            "    \"reconstitution_volume_uom\": null,\n",
            "    \"internal_standards\": null,\n",
            "    \"extraction_method\": null,\n",
            "    \"ionization_mode\": null\n",
            "  }\n",
            "}\n"
          ]
        }
      ],
      "source": [
        "import requests\n",
        "import json\n",
        "from langchain.text_splitter import RecursiveCharacterTextSplitter\n",
        "\n",
        "# Step 1: Fetch the study and protocol data\n",
        "study_protocol_url = \"https://pdc.cancer.gov/graphql\"  # The URL to fetch study and protocol data\n",
        "study_protocol_query = \"\"\"\n",
        "query($acceptDUA: Boolean!) {\n",
        "  study(acceptDUA: $acceptDUA) {\n",
        "    study_id\n",
        "    pdc_study_id\n",
        "    study_submitter_id\n",
        "    program_id\n",
        "    project_id\n",
        "    study_name\n",
        "    program_name\n",
        "    project_name\n",
        "    disease_type\n",
        "    primary_site\n",
        "    analytical_fraction\n",
        "    experiment_type\n",
        "    embargo_date\n",
        "    cases_count\n",
        "    aliquots_count\n",
        "  }\n",
        "  protocolPerStudy {\n",
        "    protocol_id\n",
        "    protocol_submitter_id\n",
        "    study_id\n",
        "    pdc_study_id\n",
        "    study_submitter_id\n",
        "    program_id\n",
        "    program_submitter_id\n",
        "    protocol_name\n",
        "    protocol_date\n",
        "    document_name\n",
        "    quantitation_strategy\n",
        "    experiment_type\n",
        "    label_free_quantitation\n",
        "    labeled_quantitation\n",
        "    isobaric_labeling_reagent\n",
        "    reporter_ion_ms_level\n",
        "    starting_amount\n",
        "    starting_amount_uom\n",
        "    digestion_reagent\n",
        "    alkylation_reagent\n",
        "    enrichment_strategy\n",
        "    enrichment\n",
        "    chromatography_dimensions_count\n",
        "    one_d_chromatography_type\n",
        "    two_d_chromatography_type\n",
        "    fractions_analyzed_count\n",
        "    column_type\n",
        "    amount_on_column\n",
        "    amount_on_column_uom\n",
        "    column_length\n",
        "    column_length_uom\n",
        "    column_inner_diameter\n",
        "    column_inner_diameter_uom\n",
        "    particle_size\n",
        "    particle_size_uom\n",
        "    particle_type\n",
        "    gradient_length\n",
        "    gradient_length_uom\n",
        "    instrument_make\n",
        "    instrument_model\n",
        "    dissociation_type\n",
        "    ms1_resolution\n",
        "    ms2_resolution\n",
        "    dda_topn\n",
        "    normalized_collision_energy\n",
        "    acquistion_type\n",
        "    dia_multiplexing\n",
        "    dia_ims\n",
        "    analytical_technique\n",
        "    chromatography_instrument_make\n",
        "    chromatography_instrument_model\n",
        "    polarity\n",
        "    reconstitution_solvent\n",
        "    reconstitution_volume\n",
        "    reconstitution_volume_uom\n",
        "    internal_standards\n",
        "    extraction_method\n",
        "    ionization_mode\n",
        "  }\n",
        "}\n",
        "\"\"\"\n",
        "variables = {\n",
        "    \"acceptDUA\": True  # Accept DUA (Data Use Agreement) as a variable\n",
        "}\n",
        "response = requests.post(study_protocol_url, json={'query': study_protocol_query, 'variables': variables})\n",
        "data = response.json()\n",
        "\n",
        "# Step 2: Merge study and protocol data\n",
        "merged_data = {}\n",
        "\n",
        "for study in data['data']['study']:\n",
        "    pdc_study_id = study['pdc_study_id']\n",
        "    if pdc_study_id not in merged_data:\n",
        "        merged_data[pdc_study_id] = study\n",
        "    else:\n",
        "        merged_data[pdc_study_id].update(study)\n",
        "\n",
        "for protocol in data['data']['protocolPerStudy']:\n",
        "    pdc_study_id = protocol['pdc_study_id']\n",
        "    if pdc_study_id not in merged_data:\n",
        "        merged_data[pdc_study_id] = protocol\n",
        "    else:\n",
        "        for key, value in protocol.items():\n",
        "            if key not in merged_data[pdc_study_id]:\n",
        "                merged_data[pdc_study_id][key] = value\n",
        "\n",
        "# Step 3: Fetch publication data\n",
        "publication_url = \"https://pdc.cancer.gov/graphql\"  # The URL to fetch publication data\n",
        "publication_query = \"\"\"\n",
        "{\n",
        "  getPaginatedUIPublication(offset: 0, limit: 50) {\n",
        "    uiPublication {\n",
        "      publication_id\n",
        "      pubmed_id\n",
        "      doi\n",
        "      author\n",
        "      title\n",
        "      journal\n",
        "      journal_url\n",
        "      year\n",
        "      abstract\n",
        "      studies {\n",
        "        pdc_study_id\n",
        "        submitter_id_name\n",
        "      }\n",
        "    }\n",
        "  }\n",
        "}\n",
        "\"\"\"\n",
        "pub_response = requests.post(publication_url, json={'query': publication_query})\n",
        "pub_data = pub_response.json()\n",
        "\n",
        "# Step 4: Create a mapping of pdc_study_id to publication info\n",
        "publication_mapping = {}\n",
        "\n",
        "for publication in pub_data['data']['getPaginatedUIPublication']['uiPublication']:\n",
        "    for study in publication['studies']:\n",
        "        pdc_study_id = study['pdc_study_id']\n",
        "        if pdc_study_id not in publication_mapping:\n",
        "            publication_mapping[pdc_study_id] = []\n",
        "        publication_mapping[pdc_study_id].append({\n",
        "            'publication_id': publication['publication_id'],\n",
        "            'pubmed_id': publication['pubmed_id'],\n",
        "            'doi': publication['doi'],\n",
        "            'author': publication['author'],\n",
        "            'title': publication['title'],\n",
        "            'journal': publication['journal'],\n",
        "            'journal_url': publication['journal_url'],\n",
        "            'year': publication['year'],\n",
        "            'abstract': publication['abstract'],\n",
        "        })\n",
        "\n",
        "# Step 5: Add publication info to merged data\n",
        "for pdc_study_id, publications in publication_mapping.items():\n",
        "    if pdc_study_id in merged_data:\n",
        "        merged_data[pdc_study_id]['publications'] = publications\n",
        "    else:\n",
        "        merged_data[pdc_study_id] = {'publications': publications}\n",
        "\n",
        "# Convert merged_data back to a list if needed\n",
        "merged_list = list(merged_data.values())\n",
        "\n",
        "# Step 6: Fetch the list of PubMed IDs\n",
        "publication_query_pubmed_ids = \"\"\"\n",
        "{\n",
        "  getPaginatedUIPublication(offset: 0, limit: 50) {\n",
        "    total\n",
        "    uiPublication {\n",
        "      pubmed_id\n",
        "    }\n",
        "  }\n",
        "}\n",
        "\"\"\"\n",
        "pub_response_pubmed_ids = requests.post(publication_url, json={'query': publication_query_pubmed_ids})\n",
        "response_json_pubmed_ids = pub_response_pubmed_ids.json()\n",
        "\n",
        "# Extract the list of publications\n",
        "publications = response_json_pubmed_ids['data']['getPaginatedUIPublication']['uiPublication']\n",
        "\n",
        "# Extract pubmed_id from each publication and create a list\n",
        "pubmed_ids = [publication['pubmed_id'] for publication in publications]\n",
        "\n",
        "# Step 7: Fetch article JSON from PubMed API and extract text\n",
        "def fetch_article_json(pubmed_id):\n",
        "    url = f\"https://www.ncbi.nlm.nih.gov/research/bionlp/RESTful/pmcoa.cgi/BioC_json/{pubmed_id}/unicode\"\n",
        "    response = requests.get(url)\n",
        "    response.raise_for_status()  # Check for request errors\n",
        "    return response.json()\n",
        "\n",
        "def extract_article_text(article_json):\n",
        "    article_text = []\n",
        "    for document in article_json[0]['documents']:\n",
        "        for passage in document['passages']:\n",
        "            if passage['infons'].get('section_type') != 'REF':\n",
        "                article_text.append(passage['text'])\n",
        "    return \"\\n\\n\".join(article_text)\n",
        "\n",
        "def split_article_text(article_text, chunk_size=256, chunk_overlap=64):\n",
        "    splitter = RecursiveCharacterTextSplitter(chunk_size=chunk_size, chunk_overlap=chunk_overlap)\n",
        "    return splitter.split_text(article_text)\n",
        "\n",
        "all_articles = []\n",
        "\n",
        "for pubmed_id in pubmed_ids:\n",
        "    try:\n",
        "        article_json = fetch_article_json(pubmed_id)\n",
        "        article_text = extract_article_text(article_json)\n",
        "        article_text_split = split_article_text(article_text)\n",
        "        all_articles.append({\n",
        "            'pubmed_id': pubmed_id,\n",
        "            'full_text': article_text_split\n",
        "        })\n",
        "    except requests.exceptions.RequestException as e:\n",
        "        all_articles.append({ 'pubmed_id': pubmed_id, 'full_text': 'Full text is not available in this publication' })\n",
        "        print(f\"Failed to fetch data for PubMed ID {pubmed_id}: {e}\")\n",
        "\n",
        "# Step 8: Merge all_articles with merged_list based on pubmed_id\n",
        "pubmed_to_full_text = {article['pubmed_id']: article['full_text'] for article in all_articles}\n",
        "\n",
        "for study in merged_list:\n",
        "    if 'publications' in study:\n",
        "        valid_publications = []\n",
        "        for publication in study['publications']:\n",
        "            pubmed_id = publication['pubmed_id']\n",
        "            if pubmed_id in pubmed_to_full_text:\n",
        "                publication['full_text'] = pubmed_to_full_text[pubmed_id]\n",
        "                valid_publications.append(publication)\n",
        "        study['publications'] = valid_publications\n",
        "\n",
        "# Output the merged data\n",
        "full_data = {\n",
        "    study['pdc_study_id']: study for study in merged_list\n",
        "}\n",
        "\n",
        "print(json.dumps(full_data, indent=2))"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "Define Schema"
      ],
      "metadata": {
        "id": "Led2SJzNug17"
      }
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "0u6Ld7ksXCgf",
        "outputId": "7a8780fe-96e5-44bb-e5c4-1cb58cb87298"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[38;5;3m⚠ Document class existed and was removed.\u001b[0m\n",
            "\u001b[38;5;2m✔ Document class was re-created successfully.\u001b[0m\n"
          ]
        }
      ],
      "source": [
        "from weaviate.classes.config import Configure, Property, DataType, Tokenization\n",
        "from wasabi import msg\n",
        "\n",
        "# Define a schema\n",
        "class_definition = {\n",
        "   \"classes\": [\n",
        "  {\n",
        "    \"class\": \"Document\",\n",
        "    \"description\": \"A class called document\",\n",
        "    \"vectorizer\": \"text2vec-huggingface\",\n",
        "    \"moduleConfig\": {\n",
        "        \"text2vec-huggingface\": {\n",
        "            \"model\": \"sentence-transformers/all-MiniLM-L6-v2\",\n",
        "            \"vectorizeClassName\": False\n",
        "            }\n",
        "        },\n",
        "            \"properties\": [\n",
        "                {\n",
        "                    \"name\": \"study_id\",\n",
        "            \"dataType\": [\"string\"],\n",
        "            \"description\": \"A unique identifier for a study.\"\n",
        "        },\n",
        "                {\n",
        "                    \"name\": \"pdc_study_id\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"The PDC-specific study ID (e.g., PDC000121).\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"study_submitter_id\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"An identifier submitted by the study’s submitter.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"program_id\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"Identifier for the program associated with the study.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"project_id\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"Identifier for the project within the program.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"study_name\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"Name of the study.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"program_name\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"Name of the program.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"project_name\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"Name of the project.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"disease_type\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"Type of disease being studied.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"primary_site\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"Anatomic site of the tumor or disease.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"analytical_fraction\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"The type of analytical fraction used (e.g., proteome, phosphoproteome).\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"experiment_type\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"Type of labeling reagent used. Indicate number of channels.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"cases_count\",\n",
        "                    \"dataType\": [\"int\"],\n",
        "                    \"description\": \"Number of cases in the study (null if not available).\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"aliquots_count\",\n",
        "                    \"dataType\": [\"int\"],\n",
        "                    \"description\": \"Number of aliquots (samples) in the study (null if not available).\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"publication_id\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"Publication identifier.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"pubmed_id\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"PubMed identifier.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"doi\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"Digital Object Identifier (DOI) for the publication.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"author\",\n",
        "                    \"dataType\": [\"text\"],\n",
        "                    \"description\": \"Author(s) of the publication.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"title\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"Title of the publication.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"journal\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"Journal where the publication appeared.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"journal_url\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"URL to the journal.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"year\",\n",
        "                    \"dataType\": [\"number\"],\n",
        "                    \"description\": \"Year of publication.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"abstract\",\n",
        "                    \"dataType\": [\"text\"],\n",
        "                    \"description\": \"Abstract of the publication.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"full_text\",\n",
        "                    \"dataType\": [\"text\"],\n",
        "                    \"description\": \"Full text of the publication.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"embedding\",\n",
        "                    \"dataType\": [\"blob\"],\n",
        "                    \"description\": \"Text embedding of the document.\"\n",
        "                }\n",
        "                ]\n",
        "            }\n",
        "   ]\n",
        "        }\n",
        "# Check if the class exists\n",
        "if not client.schema.exists(\"Document\"):\n",
        "            # Create the class if it does not exist\n",
        "            client.schema.create(class_definition)\n",
        "            msg.good(\"Document class was created successfully.\")\n",
        "else:\n",
        "            # Optionally, delete and recreate the class\n",
        "            client.schema.delete_class(\"Document\")\n",
        "            msg.warn(\"Document class existed and was removed.\")\n",
        "            client.schema.create(class_definition)\n",
        "            msg.good(\"Document class was re-created successfully.\")"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "def jprint(json_in):\n",
        "    import json\n",
        "    print(json.dumps(json_in, indent=2))\n",
        "jprint(client.schema.get())"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "EmL09-Jm9Rp9",
        "outputId": "f0ed6a7a-2cfc-4cb5-b40c-0be104334397"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "{\n",
            "  \"classes\": [\n",
            "    {\n",
            "      \"class\": \"Document\",\n",
            "      \"description\": \"A class called document\",\n",
            "      \"invertedIndexConfig\": {\n",
            "        \"bm25\": {\n",
            "          \"b\": 0.75,\n",
            "          \"k1\": 1.2\n",
            "        },\n",
            "        \"cleanupIntervalSeconds\": 60,\n",
            "        \"stopwords\": {\n",
            "          \"additions\": null,\n",
            "          \"preset\": \"en\",\n",
            "          \"removals\": null\n",
            "        }\n",
            "      },\n",
            "      \"moduleConfig\": {\n",
            "        \"text2vec-huggingface\": {\n",
            "          \"model\": \"sentence-transformers/all-MiniLM-L6-v2\",\n",
            "          \"vectorizeClassName\": false\n",
            "        }\n",
            "      },\n",
            "      \"multiTenancyConfig\": {\n",
            "        \"autoTenantActivation\": false,\n",
            "        \"autoTenantCreation\": false,\n",
            "        \"enabled\": false\n",
            "      },\n",
            "      \"properties\": [\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"A unique identifier for a study.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"study_id\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"The PDC-specific study ID (e.g., PDC000121).\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"pdc_study_id\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"An identifier submitted by the study\\u2019s submitter.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"study_submitter_id\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"Identifier for the program associated with the study.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"program_id\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"Identifier for the project within the program.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"project_id\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"Name of the study.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"study_name\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"Name of the program.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"program_name\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"Name of the project.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"project_name\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"Type of disease being studied.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"disease_type\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"Anatomic site of the tumor or disease.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"primary_site\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"The type of analytical fraction used (e.g., proteome, phosphoproteome).\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"analytical_fraction\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"Type of labeling reagent used. Indicate number of channels.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"experiment_type\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"int\"\n",
            "          ],\n",
            "          \"description\": \"Number of cases in the study (null if not available).\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": false,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"cases_count\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"int\"\n",
            "          ],\n",
            "          \"description\": \"Number of aliquots (samples) in the study (null if not available).\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": false,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"aliquots_count\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"Publication identifier.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"publication_id\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"PubMed identifier.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"pubmed_id\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"Digital Object Identifier (DOI) for the publication.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"doi\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"Author(s) of the publication.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"author\",\n",
            "          \"tokenization\": \"word\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"Title of the publication.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"title\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"Journal where the publication appeared.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"journal\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"URL to the journal.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"journal_url\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"number\"\n",
            "          ],\n",
            "          \"description\": \"Year of publication.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": false,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"year\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"Abstract of the publication.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"abstract\",\n",
            "          \"tokenization\": \"word\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"Full text of the publication.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"full_text\",\n",
            "          \"tokenization\": \"word\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"blob\"\n",
            "          ],\n",
            "          \"description\": \"Text embedding of the document.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": false,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"embedding\"\n",
            "        }\n",
            "      ],\n",
            "      \"replicationConfig\": {\n",
            "        \"asyncEnabled\": false,\n",
            "        \"factor\": 1\n",
            "      },\n",
            "      \"shardingConfig\": {\n",
            "        \"actualCount\": 1,\n",
            "        \"actualVirtualCount\": 128,\n",
            "        \"desiredCount\": 1,\n",
            "        \"desiredVirtualCount\": 128,\n",
            "        \"function\": \"murmur3\",\n",
            "        \"key\": \"_id\",\n",
            "        \"strategy\": \"hash\",\n",
            "        \"virtualPerPhysical\": 128\n",
            "      },\n",
            "      \"vectorIndexConfig\": {\n",
            "        \"bq\": {\n",
            "          \"enabled\": false\n",
            "        },\n",
            "        \"cleanupIntervalSeconds\": 300,\n",
            "        \"distance\": \"cosine\",\n",
            "        \"dynamicEfFactor\": 8,\n",
            "        \"dynamicEfMax\": 500,\n",
            "        \"dynamicEfMin\": 100,\n",
            "        \"ef\": -1,\n",
            "        \"efConstruction\": 128,\n",
            "        \"flatSearchCutoff\": 40000,\n",
            "        \"maxConnections\": 32,\n",
            "        \"pq\": {\n",
            "          \"bitCompression\": false,\n",
            "          \"centroids\": 256,\n",
            "          \"enabled\": false,\n",
            "          \"encoder\": {\n",
            "            \"distribution\": \"log-normal\",\n",
            "            \"type\": \"kmeans\"\n",
            "          },\n",
            "          \"segments\": 0,\n",
            "          \"trainingLimit\": 100000\n",
            "        },\n",
            "        \"skip\": false,\n",
            "        \"sq\": {\n",
            "          \"enabled\": false,\n",
            "          \"rescoreLimit\": 20,\n",
            "          \"trainingLimit\": 100000\n",
            "        },\n",
            "        \"vectorCacheMaxObjects\": 1000000000000\n",
            "      },\n",
            "      \"vectorIndexType\": \"hnsw\",\n",
            "      \"vectorizer\": \"text2vec-huggingface\"\n",
            "    }\n",
            "  ]\n",
            "}\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "Import Data to Weaviate"
      ],
      "metadata": {
        "id": "ljf9iBX_urxG"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "from transformers import AutoTokenizer, AutoModel\n",
        "import torch\n",
        "import logging\n",
        "import numpy as np\n",
        "import base64\n",
        "\n",
        "model_name = \"sentence-transformers/all-MiniLM-L6-v2\"\n",
        "tokenizer = AutoTokenizer.from_pretrained(model_name)\n",
        "model = AutoModel.from_pretrained(model_name)\n",
        "\n",
        "logging.basicConfig(level=logging.INFO)\n",
        "msg = logging.getLogger()\n",
        "\n",
        "def generate_embedding(text):\n",
        "    inputs = tokenizer(text, return_tensors=\"pt\", truncation=True, padding=True, max_length=512)\n",
        "    with torch.no_grad():\n",
        "        outputs = model(**inputs)\n",
        "    return outputs.last_hidden_state.mean(dim=1).squeeze().numpy()\n",
        "\n",
        "def encode_embedding(embedding):\n",
        "    return base64.b64encode(embedding.tobytes()).decode('utf-8')\n",
        "\n",
        "with client.batch as batch:\n",
        "    batch.batch_size = 100\n",
        "    for i, (pdc_study_id, doc) in enumerate(full_data.items()):\n",
        "        msg.info(f\"({i+1}/{len(full_data)}) Importing Study {doc.get('pdc_study_id', 'Unknown ID')}\")\n",
        "\n",
        "        publications = doc.get(\"publications\", [])\n",
        "        for publication in publications:\n",
        "            full_text = publication.get(\"full_text\", \"\")\n",
        "            if isinstance(full_text, list):\n",
        "                full_text = \" \".join(full_text)\n",
        "\n",
        "            embedding = generate_embedding(full_text)\n",
        "            encoded_embedding = encode_embedding(embedding)\n",
        "\n",
        "            year = publication.get(\"year\")\n",
        "            try:\n",
        "                year = float(year) if year is not None else None\n",
        "            except ValueError:\n",
        "                year = None\n",
        "                msg.warning(f\"Invalid year value '{publication.get('year')}' encountered and set to None\")\n",
        "\n",
        "            properties = {\n",
        "                \"study_id\": doc.get(\"study_id\", \"\"),\n",
        "                \"pdc_study_id\": doc.get(\"pdc_study_id\", \"\"),\n",
        "                \"study_submitter_id\": doc.get(\"study_submitter_id\", \"\"),\n",
        "                \"program_id\": doc.get(\"program_id\", \"\"),\n",
        "                \"project_id\": doc.get(\"project_id\", \"\"),\n",
        "                \"study_name\": doc.get(\"study_name\", \"\"),\n",
        "                \"program_name\": doc.get(\"program_name\", \"\"),\n",
        "                \"project_name\": doc.get(\"project_name\", \"\"),\n",
        "                \"disease_type\": doc.get(\"disease_type\", \"\"),\n",
        "                \"primary_site\": doc.get(\"primary_site\", \"\"),\n",
        "                \"analytical_fraction\": doc.get(\"analytical_fraction\", \"\"),\n",
        "                \"experiment_type\": doc.get(\"experiment_type\", \"\"),\n",
        "                \"cases_count\": doc.get(\"cases_count\", None),\n",
        "                \"aliquots_count\": doc.get(\"aliquots_count\", None),\n",
        "                \"publication_id\": publication.get(\"publication_id\", \"\"),\n",
        "                \"pubmed_id\": publication.get(\"pubmed_id\", \"\"),\n",
        "                \"doi\": publication.get(\"doi\", \"\"),\n",
        "                \"author\": publication.get(\"author\", \"\"),\n",
        "                \"title\": publication.get(\"title\", \"\"),\n",
        "                \"journal\": publication.get(\"journal\", \"\"),\n",
        "                \"journal_url\": publication.get(\"journal_url\", \"\"),\n",
        "                \"year\": year,\n",
        "                \"abstract\": publication.get(\"abstract\", \"\"),\n",
        "                \"full_text\": full_text,\n",
        "                \"embedding\": encoded_embedding\n",
        "            }\n",
        "            client.batch.add_data_object(properties, \"Document\")\n",
        "            msg.info(\"Data imported\")"
      ],
      "metadata": {
        "id": "y2WHHKFSwxEM",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 333,
          "referenced_widgets": [
            "5ec8e1a6e1284e70a08e5bee99246d93",
            "7caff46ba1a44fdbbda0526a4446066b",
            "62b9bd5f6f75432985482593074f576b",
            "87c8b42a2bde46eaba1323d574ecf952",
            "f98ae7514faa4c6fb86d4bdb7002fc03",
            "17a64412b6324428b48a271d92e4bf90",
            "32dfff721ff5472eb1531e8af230aef3",
            "c6e1cca880d24ec5a92d369ba6ccdcb1",
            "6d275cd58fb4455d9ddcd76f2a80fa7e",
            "1fb2cda0de70436f93f563099bd03e45",
            "fa6760ba8dec4bd7a23bffce6ee4f52c",
            "9d5e46a422e74e2ebeabd37214cd9da9",
            "29a9dbc295484540b475bf496aeb7e16",
            "7363e379219f4e948e3a7e04d8c938ac",
            "4d33c4b2de30400c97fb10d933c52c99",
            "693dad55763340899e8b87fb38b63da4",
            "c6a11961669443ad8d15f4a92baa23a8",
            "0b4f3abe0e3749a695ab5407b96d0cb2",
            "7e070566c44f409c80c9371551a18fff",
            "c4dd7e98e36447c2b42fc4ce5fb3f0fe",
            "3f9f711504d54766b04df13907416358",
            "4fb04110dfb24eabb2e415b88dc9e99b",
            "afd7ef7693414cbc9ef31f2c01cf3338",
            "fce47e32606a4f5885f547f7a6e7fc91",
            "50130f71472f46bb8a17c7bbe263333b",
            "a34999e8265e45b481a9f21e236e015a",
            "bd0cdf4a88c34863a468437fa3a12b46",
            "e06537ef6cb3429aac69bdd1bde4c3ce",
            "cea5fd3a576b4a899baeaa785bbe61ba",
            "629956ba73a949b4b4ddd5f3d8215f54",
            "6f7e87a6b53e43c6a211d2908a3e5ca0",
            "0ed8a6c2c3f5444ba4a68fd31dad37f9",
            "16de9dd8747d4bafa995463659701758",
            "95edc7a7720e43fa82f91c2a6488a182",
            "9b63b5e556df42dc9340efd9b97f65a8",
            "149ba45f3956453bbbd18f2403265722",
            "83ca301cb83f41008115bb9aabd093a5",
            "88fade17c0cd43688dabd5165abd2ba3",
            "0c6e7f85e9f04695a0d490ee069726cc",
            "8696e33ff081426382179869af171582",
            "a8e8ecfe0cd5454cb7442092cb140b6e",
            "06bf883790c34ea19539f2c69cd83f72",
            "a5bc2f222eaa4a1cb5fbc6d1357d3753",
            "f4456a291e534e8385ecb61edc70670d",
            "0c911f51644541cba54018da18275f2f",
            "c13670e29d9b4930b5bc225bf7f7b71a",
            "91ece160ff7041308c0623bc6a9b76ed",
            "a87fe9ee202d44ebba67815a4a6b2629",
            "a1aed037af964ef18a0f3fa3941ee437",
            "546ac419f6754e27abdb734d3b6b7843",
            "e562edf27cfc4efb94aed410cb4d3aa1",
            "da0ce35c16e14fd7a85978cade3e3e84",
            "fa60a789763142ec80d13a9a184a5841",
            "d82f08f51c9e46aeaa00d48bbcb2de06",
            "dd837693968a4efba11d1b7fef0c93ea",
            "f803e56bf9e94430bb0839408ee34fb9",
            "fa0c6769c333436a900c924edd7f9229",
            "2974f5e504b14a5cb981ba11a6e672a3",
            "6db2234ab60244118e48fddcab425857",
            "ed885ea3b46b471b81a19362935fc331",
            "d694c3c2ea204e79b6bd690ea6ae77e8",
            "e0eda87d9cb84e64b1fda056e980f022",
            "0d44242851ca4036823e8873ef5569fe",
            "e00eedf0d25b42ed875ad8ad4e653800",
            "6bf0bfd11902470ba49568d2421a5abd",
            "1c586a9d27ac472cbfb98b69512a71cb"
          ]
        },
        "outputId": "5dbffa17-1b09-4864-e0ec-e425884bd7de"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/huggingface_hub/utils/_token.py:89: UserWarning: \n",
            "The secret `HF_TOKEN` does not exist in your Colab secrets.\n",
            "To authenticate with the Hugging Face Hub, create a token in your settings tab (https://huggingface.co/settings/tokens), set it as secret in your Google Colab and restart your session.\n",
            "You will be able to reuse this secret in all of your notebooks.\n",
            "Please note that authentication is recommended but still optional to access public models or datasets.\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "tokenizer_config.json:   0%|          | 0.00/350 [00:00<?, ?B/s]"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "5ec8e1a6e1284e70a08e5bee99246d93"
            }
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "vocab.txt:   0%|          | 0.00/232k [00:00<?, ?B/s]"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "9d5e46a422e74e2ebeabd37214cd9da9"
            }
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "tokenizer.json:   0%|          | 0.00/466k [00:00<?, ?B/s]"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "afd7ef7693414cbc9ef31f2c01cf3338"
            }
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "special_tokens_map.json:   0%|          | 0.00/112 [00:00<?, ?B/s]"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "95edc7a7720e43fa82f91c2a6488a182"
            }
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "config.json:   0%|          | 0.00/612 [00:00<?, ?B/s]"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "0c911f51644541cba54018da18275f2f"
            }
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "model.safetensors:   0%|          | 0.00/90.9M [00:00<?, ?B/s]"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "f803e56bf9e94430bb0839408ee34fb9"
            }
          },
          "metadata": {}
        }
      ]
    },
    {
      "source": [
        "#example\n",
        "publications = doc.get(\"publications\", [])\n",
        "for publication in publications:\n",
        "    full_text = publication.get(\"full_text\", \"\")\n",
        "    embedding = generate_embedding(full_text)\n",
        "print(embedding)"
      ],
      "cell_type": "code",
      "metadata": {
        "id": "SREqBbvKkc6z",
        "outputId": "2122ee9d-8391-4d1e-8dbb-b5377e3c1315",
        "colab": {
          "base_uri": "https://localhost:8080/"
        }
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "[-9.41009149e-02 -1.89930767e-01 -1.43516019e-01 -8.77505690e-02\n",
            "  2.77019113e-01 -1.12023115e-01 -4.57238853e-02  8.36083293e-02\n",
            "  7.25025982e-02 -1.04528606e-01 -7.34966993e-02 -2.59311087e-02\n",
            " -6.23768494e-02  1.20297089e-01  6.30323496e-03  1.85521960e-01\n",
            " -1.71304256e-01  1.86843783e-01 -4.39400785e-02  8.39416534e-02\n",
            "  2.59517640e-01 -1.62528008e-02  6.68800771e-02  2.25069433e-01\n",
            " -1.03203878e-02 -1.27455845e-01  1.36206210e-01 -8.90714582e-03\n",
            " -2.80195743e-01  6.10978380e-02  4.65099625e-02  7.65417814e-02\n",
            "  2.51891837e-02  1.21539995e-01 -2.70181578e-02  7.54228532e-02\n",
            " -5.31649068e-02 -8.51387531e-02 -1.58204600e-01 -1.75579101e-01\n",
            "  1.77027836e-01 -2.66491976e-02 -2.48716921e-02  1.17250718e-04\n",
            "  9.92754400e-02 -1.86344951e-01 -4.58728559e-02  7.70096108e-02\n",
            "  1.03084659e-02  1.86921030e-01 -6.19113520e-02 -6.67831302e-02\n",
            " -1.40255317e-01  1.02814391e-01  3.21376920e-02 -6.19568955e-03\n",
            "  1.05215780e-01 -8.85296799e-03  1.77134201e-02 -6.21088073e-02\n",
            " -2.25320105e-02 -1.73517466e-01  6.22113720e-02  1.27104810e-02\n",
            "  1.67465627e-01  4.53776568e-02 -4.64100689e-02  1.47305988e-02\n",
            " -1.74943537e-01 -6.01865537e-03 -5.57744354e-02  3.46104056e-02\n",
            "  6.05297424e-02  1.80218250e-01  4.36290503e-02  1.99922264e-01\n",
            "  2.65913516e-01  2.70532906e-01  2.16072258e-02 -1.64155126e-01\n",
            "  6.33265451e-02  7.89609700e-02  3.63082998e-02 -1.16502315e-01\n",
            "  3.13628614e-02  1.06223328e-02 -5.16599044e-02  1.73814058e-01\n",
            "  6.20865785e-02  1.44536346e-01  1.82714593e-03  3.17482688e-02\n",
            "  6.93700612e-02  1.17261134e-01  7.02998554e-03  4.36597429e-02\n",
            " -2.66412571e-02 -3.57191600e-02  2.51974296e-02 -1.35159850e-01\n",
            " -1.46566659e-01  7.25304782e-02  2.07954347e-02 -4.83451784e-03\n",
            "  1.30350798e-01 -3.01757711e-03  9.08633024e-02  6.28543273e-02\n",
            " -1.05525121e-01 -2.23647244e-02  1.89079791e-01 -9.47645456e-02\n",
            "  1.84087783e-01  2.28723213e-02  2.28264853e-02  3.40545364e-03\n",
            "  4.60427254e-03 -8.94440711e-02 -3.17189172e-02 -9.10407901e-02\n",
            " -2.66331639e-02  2.40187012e-02  4.40815538e-02  4.41617630e-02\n",
            " -5.65131567e-02  2.28576526e-01 -2.10450277e-01  1.39950052e-32\n",
            "  7.33475685e-02  6.40507340e-02  2.59430081e-01  1.50968567e-01\n",
            " -1.28133902e-02 -6.75852299e-02  3.64833400e-02 -5.49745560e-03\n",
            " -7.91010484e-02 -3.19128763e-03 -6.09461218e-02  1.05649337e-01\n",
            " -1.32496068e-02 -2.91048223e-03  8.37818533e-02 -1.68308884e-01\n",
            "  5.27663901e-02  1.62081122e-01 -6.04709238e-02 -4.22094800e-02\n",
            "  4.90619242e-02 -8.68605003e-02 -1.24816120e-01 -1.50787681e-01\n",
            " -3.18029243e-03  2.22416103e-01 -3.21552902e-03 -5.36473207e-02\n",
            " -1.74272418e-01  3.48589905e-02 -4.18140352e-01 -8.91616568e-02\n",
            "  1.83409959e-01  6.75924122e-02 -2.81073749e-02 -1.80506885e-01\n",
            "  9.05376375e-02 -2.80248672e-01  1.16719738e-01  5.22125401e-02\n",
            "  2.10878737e-02 -1.67015836e-01 -1.27849475e-01 -3.27468142e-02\n",
            " -8.58278722e-02 -2.90850163e-01 -8.26651510e-03 -4.99892095e-03\n",
            " -2.01433420e-01 -4.39046770e-02 -7.31717199e-02 -4.58154455e-02\n",
            " -6.61288388e-04  1.61896497e-01 -2.95474589e-01 -3.13845128e-02\n",
            " -1.70798570e-01 -2.72843242e-02  1.33778185e-01 -1.76555645e-02\n",
            " -5.84487151e-03 -9.60047841e-02  5.59714660e-02  8.76923054e-02\n",
            "  1.72477305e-01  5.15155345e-02 -1.56504571e-01  1.83434486e-02\n",
            " -9.87664685e-02  2.45256245e-01 -1.77744646e-02  3.50885317e-02\n",
            "  3.47147614e-01  7.62926042e-02  6.91938400e-02  4.36825827e-02\n",
            "  2.11661533e-02  1.05035201e-01 -8.25950410e-03  3.09317529e-01\n",
            " -9.88496915e-02  1.39208376e-01 -3.31134349e-01  1.64454877e-01\n",
            " -1.50857002e-01 -1.59736186e-01  1.36150107e-01 -5.47292158e-02\n",
            " -1.59014821e-01 -7.55023807e-02 -3.13476361e-02 -1.21163696e-01\n",
            "  1.32961068e-02  1.47948135e-02 -2.66191997e-02 -1.42117355e-32\n",
            " -3.36207077e-02  8.66645277e-02  9.51169711e-03 -2.03716829e-01\n",
            " -1.40325259e-03 -1.18609145e-02  1.44405976e-01 -5.68138547e-02\n",
            "  9.76969823e-02 -2.56878376e-01  3.07580419e-02 -1.95061654e-01\n",
            "  6.44859448e-02 -1.39246047e-01 -2.00008333e-01  1.52602987e-02\n",
            " -1.79794908e-01  8.91883373e-02 -6.77742809e-02  5.69221079e-02\n",
            "  4.23704125e-02  1.63876131e-01 -1.15421727e-01  9.11281705e-02\n",
            " -1.58579320e-01  9.24379155e-02  7.60130286e-02  2.36716382e-02\n",
            " -6.13507405e-02  4.58003804e-02 -1.57859623e-02  2.28548467e-01\n",
            " -1.97622404e-01 -5.35907857e-02  1.29674040e-02 -5.66373132e-02\n",
            "  5.82004674e-02 -5.70824072e-02  1.14688352e-01  1.60161350e-02\n",
            "  5.60207516e-02  8.79822671e-02 -1.61202058e-01 -1.39287919e-01\n",
            "  1.92121267e-02  1.64217681e-01  4.91309129e-02 -6.35524094e-03\n",
            "  2.42663488e-01 -1.16273146e-02 -9.66684669e-02  1.41510323e-01\n",
            " -1.65068716e-01  9.71796662e-02 -1.09785356e-01 -1.33703589e-01\n",
            " -1.40924007e-03 -4.51672226e-02  3.10078654e-02 -4.36741412e-02\n",
            " -2.16664046e-01 -1.94223057e-02  3.85380872e-02  6.28799424e-02\n",
            "  8.60621408e-02  1.16016105e-01  8.09872299e-02 -1.90828234e-01\n",
            "  7.75165558e-02 -3.40785012e-02  5.13680950e-02 -1.15663707e-02\n",
            "  7.40470290e-02  1.62172318e-01 -1.85614526e-02 -9.82371420e-02\n",
            " -9.53076854e-02 -1.24851212e-01  1.15736052e-02 -7.69828483e-02\n",
            "  1.96543522e-02 -1.74489334e-01  1.48526598e-02 -6.06980063e-02\n",
            "  5.52934781e-02  9.10893641e-03 -8.10021013e-02  1.70801476e-01\n",
            " -4.34965156e-02  1.70345958e-02 -1.74981058e-01 -1.53386265e-01\n",
            " -2.07281888e-01  1.38397723e-01 -8.34839195e-02 -9.96254670e-08\n",
            "  2.38293745e-02  6.45451546e-02  9.83086303e-02 -9.80777591e-02\n",
            " -2.10954305e-02  7.26243407e-02 -9.92479026e-02  1.05772652e-02\n",
            "  8.87416974e-02 -6.63075373e-02  8.08839500e-02 -1.60054967e-01\n",
            " -1.05646566e-01 -2.68025637e-01  4.19189818e-02  1.00233108e-01\n",
            "  7.64933974e-02 -4.35646251e-02 -6.73784167e-02  6.10098913e-02\n",
            " -1.58689208e-02 -6.60549290e-03 -2.23473143e-02 -7.51410127e-02\n",
            "  2.13167608e-01 -4.31011990e-03 -1.00646272e-01  2.23587155e-01\n",
            "  1.03148021e-01 -2.20752098e-02  5.50722219e-02 -1.03574507e-01\n",
            "  4.71075699e-02  4.92775813e-03  1.37374729e-01  1.52911961e-01\n",
            "  4.56292182e-02 -5.78188524e-02  5.06212413e-02  1.41430616e-01\n",
            " -8.56594294e-02 -1.03141844e-01 -1.58226624e-01 -5.56543320e-02\n",
            " -6.26222268e-02  1.18435636e-01  1.22192204e-02 -2.61506792e-02\n",
            "  1.48551553e-01 -5.50342053e-02 -1.50148824e-01 -5.36075942e-02\n",
            " -4.49585617e-02 -3.06440648e-02  8.82939249e-02  1.90120012e-01\n",
            " -6.49751574e-02  2.70501953e-02  2.30456442e-01 -9.23487693e-02\n",
            "  1.89622700e-01 -4.69763577e-02  1.46879882e-01 -1.14223436e-01]\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "Querying vector database"
      ],
      "metadata": {
        "id": "lqgN9atKwHut"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "result = (\n",
        "    client.query\n",
        "    .get(\"Document\",[\"pdc_study_id\", \"title\"])\n",
        "    .with_near_text({\"concepts\": [\"glioblastoma\"]})\n",
        "    .with_limit(5)\n",
        "    .do()\n",
        ")\n",
        "print(json.dumps(result, indent=4))"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "vYUsaeKjlCkB",
        "outputId": "a6fa386c-487f-44ce-c14d-5e81341d03ea"
      },
      "execution_count": 42,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "{\n",
            "    \"data\": {\n",
            "        \"Get\": {\n",
            "            \"Document\": [\n",
            "                {\n",
            "                    \"pdc_study_id\": \"PDC000514\",\n",
            "                    \"title\": \"Integrated proteogenomic characterization of glioblastoma evolution\"\n",
            "                },\n",
            "                {\n",
            "                    \"pdc_study_id\": \"PDC000515\",\n",
            "                    \"title\": \"Integrated proteogenomic characterization of glioblastoma evolution\"\n",
            "                },\n",
            "                {\n",
            "                    \"pdc_study_id\": \"PDC000245\",\n",
            "                    \"title\": \"Proteogenomic and metabolomic characterization of human glioblastoma\"\n",
            "                },\n",
            "                {\n",
            "                    \"pdc_study_id\": \"PDC000205\",\n",
            "                    \"title\": \"Proteogenomic and metabolomic characterization of human glioblastoma\"\n",
            "                },\n",
            "                {\n",
            "                    \"pdc_study_id\": \"PDC000204\",\n",
            "                    \"title\": \"Proteogenomic and metabolomic characterization of human glioblastoma\"\n",
            "                }\n",
            "            ]\n",
            "        }\n",
            "    }\n",
            "}\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "where_filter= {\n",
        "    \"path\":[\"full_text\"],\n",
        "    \"operator\":\"Like\",\n",
        "    \"valueText\":\"*pancreatic cancer*\"\n",
        "}\n",
        "result1 = (\n",
        "    client.query\n",
        "    .get(\"Document\",[\"pdc_study_id\", \"title\", \"full_text\"])\n",
        "    .with_limit(5)\n",
        "    .with_where(where_filter)\n",
        "    .do()\n",
        ")\n",
        "print(json.dumps(result1, indent=4))"
      ],
      "metadata": {
        "id": "nNgC6UAsn4BP",
        "outputId": "95b1ebef-2229-43a9-e9c8-9960c1279ee1",
        "colab": {
          "base_uri": "https://localhost:8080/"
        }
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "{\n",
            "    \"data\": {\n",
            "        \"Get\": {\n",
            "            \"Document\": [\n",
            "                {\n",
            "                    \"full_text\": \"A proteogenomic portrait of lung squamous cell carcinoma\\n\\nSummary Lung squamous cell carcinoma (LSCC) remains a leading cause of cancer death with few therapeutic options. We characterized the proteogenomic landscape of LSCC, providing a deeper exposition of LSCC biology with potential therapeutic implications. We of LSCC biology with potential therapeutic implications. We identify NSD3 as an alternative driver in FGFR1-amplified tumors and low-p63 tumors overexpressing the therapeutic target survivin. SOX2 is considered undruggable, but our analyses provide SOX2 is considered undruggable, but our analyses provide rationale for exploring chromatin modifiers such as LSD1 and EZH2 to target SOX2-overexpressing tumors. Our data support complex regulation of metabolic pathways by crosstalk between complex regulation of metabolic pathways by crosstalk between post-translational modifications including ubiquitylation. Numerous immune-related proteogenomic observations suggest directions for further investigation. Proteogenomic dissection of CDKN2A for further investigation. Proteogenomic dissection of CDKN2A mutations argue for more nuanced assessment of RB1 protein expression and phosphorylation before declaring CDK4/6 inhibition unsuccessful. Finally, triangulation between LSCC, LUAD and HNSCC Finally, triangulation between LSCC, LUAD and HNSCC identified both unique and common therapeutic vulnerabilities. These observations and proteogenomics data resources may guide research into the biology and treatment of LSCC. In Brief Comprehensive proteogenomic characterization of lung squamous cell carcinomas and paired normal adjacent tissues identifies taxonomic subclasses, alternative driver events and insights into immune modulation, as well as putative biomarkers and potential immune modulation, as well as putative biomarkers and potential therapeutic targets. Graphical Abstract\\n\\nIntroduction Lung cancer is the leading cause of cancer-associated mortality. Unlike lung adenocarcinomas (LUAD), patients with lung squamous cell carcinoma (LSCC) have not benefited from targeted therapies. Potentially druggable genetic events in three primary Potentially druggable genetic events in three primary pathways (FGFR1, PI3K, or G1/S checkpoint genes such as CDKN2A) are found in upwards of 60% but targeting these clinically has largely failed. While other therapeutic regimens are being explored as failed. While other therapeutic regimens are being explored as part of the LungMAP consortium, to date only immunotherapy has evolved into a successful therapeutic strategy for patients with LSCC. Given the lack of established actionable genomic targets, LSCC. Given the lack of established actionable genomic targets, a comprehensive characterization of the proteogenomic landscape of LSCC, including the associated post-translational modifications (PTMs) that are key to protein activity and signaling, is a (PTMs) that are key to protein activity and signaling, is a crucial step forward. Such studies will aid in connecting genomic aberrations to molecular and clinical phenotypes and in subsequently identifying therapeutic vulnerabilities and effective, identifying therapeutic vulnerabilities and effective, biomarker-based patient stratification. We characterized 108 prospectively-collected, treatment-na\\u00efve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs), identifying actionable therapeutic protein targets and elucidating cellular signaling pathways and crosstalk between multiple cellular signaling pathways and crosstalk between multiple PTMs. The dataset provides an exceptional resource to guide further research and support development of therapeutic modalities in LSCC. Results\\n\\nProteogenomic landscape of LSCC We performed deep-scale molecular analysis spanning nine different data types on LSCC tumors and NATs prospectively collected from 108 patients (Figure 1A, S1A, Table S1\\u20133). Proteomics data were generated using TMT11-based multiplexing linked by a common generated using TMT11-based multiplexing linked by a common reference (Figure S1A) and demonstrated high technical quality (Figure S1B\\u2013C). The cohort demographics are summarized in Figure 1B. Most self-reported never-smokers displayed high genomic smoking Most self-reported never-smokers displayed high genomic smoking scores (Figure S1D), highlighting occasional discrepancy between self-reported smoking status and genomic evidence, as previously reported. The genomic landscape reaffirmed previously reported. The genomic landscape reaffirmed previously reported somatic alterations in LSCC (Figure 1C). Principal component analysis (PCA) of global proteomic and PTM data showed clear separation of tumors and NATs (Figure S1E). Proteogenomic data helped annotate the impact of copy number alteration (CNA) events (Figure 1D,E). Of 5,523 significant (FDR<0.01) cis CNA-mRNA events, 2,154 were concordantly correlated with protein expression, including 138 \\u201ccancer associated genes\\u201d protein expression, including 138 \\u201ccancer associated genes\\u201d (CAG) (Figure 1E, Table S1). We identified six amplified (including WHSC1L1, CCND1, and SOX2) and 29 deleted (including NCOR1, SETD2, and CBL) CAGs in significant focal events (Q<0.25) (Figure and CBL) CAGs in significant focal events (Q<0.25) (Figure 1E, Table S4). Intriguingly, WHSC1L1 (NSD3) is part of a recurrent focal amplicon (8p11.23) in LSCC that encompasses FGFR1. Therapies targeted against FGFR1 have been unsuccessful. Proteomics data targeted against FGFR1 have been unsuccessful. Proteomics data suggested that NSD3, rather than FGFR1, could be the critical driver oncogene within this amplicon (Figure 1F), nominating it as a potential therapeutic target. While this paper was under as a potential therapeutic target. While this paper was under review, NSD3 was demonstrated to be a key regulator of LSCC tumorigenesis, mediating oncogenic chromatin changes. To investigate the impact of CNAs on noncognate gene products, we matched patterns of these significant trans-effects (vertical stripes in Figure 1D) to perturbation profiles from the Connectivity Map (CMap) (https://clue.io/cmap). Six CNAs (IKBKAP, PIN1, Map (CMap) (https://clue.io/cmap). Six CNAs (IKBKAP, PIN1, DNAJB1, IL18, NR2F6, AKAP) (Table S4) were enriched for both amplification and deletion (FDR=0.06) and associated (Fisher\\u2019s exact p<0.01) with clinical metadata. IL-18 deletion and amplification with clinical metadata. IL-18 deletion and amplification correlated with decreased and increased IL-18 protein expression and xCell immune score, respectively, (Figure S1F (upper panel)), consistent with its playing a key modulatory role in the tumor consistent with its playing a key modulatory role in the tumor microenvironment of NSCLC. NR2F6 amplification and deletion were correlated with NR2F6 protein expression and anticorrelated with xCell immune score (Figure S1F (lower panel)). NR2F6 acts as a xCell immune score (Figure S1F (lower panel)). NR2F6 acts as a non-redundant immune checkpoint in cancer, and even partial knockdown works synergistically with PD-L1 blockade. LSCC tumor DNA showed overall hypermethylation relative to NATs (Figure S1G), with tumors separating into CIMP (CpG island methylator phenotype) clusters (Figure S1H). Multi-omic data allowed the identification of \\u201ccascading\\u201d promoter methylation allowed the identification of \\u201ccascading\\u201d promoter methylation cis-effects across cognate mRNA, protein and PTM abundances, supporting their functional significance. Of 90 genes that showed such significant (FDR<0.1) cis-effects in tumors (Table S4), 20 such significant (FDR<0.1) cis-effects in tumors (Table S4), 20 were also differentially expressed (FDR<0.01) between tumors and NATs (Figure 1G). While most were newly associated with LSCC, methylation-driven expression of FAM110A, PTGES3, PLAU and methylation-driven expression of FAM110A, PTGES3, PLAU and SLC16A3 has been implicated in lung and other cancers. Multi-omic clustering identifies five LSCC molecular subtypes, including one that is EMT-Enriched We performed non-negative matrix factorization (NMF)-based single- and multi-omic unsupervised clustering on CNA, RNA, protein, phosphoprotein and acetylprotein datasets from 108 tumors, excluding ubiquitylprotein data as it was not available for the excluding ubiquitylprotein data as it was not available for the entire cohort (Figure 1A). The five resulting multi-omic subtypes (Figure 2A\\u2013B, Figure S2A) were named based on their predominant pathway associations and similarities to previously defined pathway associations and similarities to previously defined RNA clusters. Significant associations between NMF subtypes and cohort metadata are presented in Table S1. Two NMF subtypes, \\u201cBasal-Inclusive\\u201d (B-I) and \\u201cEpithelial to Mesenchymal \\u201cBasal-Inclusive\\u201d (B-I) and \\u201cEpithelial to Mesenchymal Transition-enriched\\u201d (EMT-E), emerged from partitioning of samples that showed similarity to the TCGA-derived RNA Basal cluster, although only B-I retained significant association after Bonferroni only B-I retained significant association after Bonferroni correction (Fisher\\u2019s p<0.01, Table S1). B-I tumors showed basaloid histology and upregulated metabolic, immune and estrogen receptor signaling (Figure 2B). EMT-E showed upregulation of EMT, signaling (Figure 2B). EMT-E showed upregulation of EMT, Angiogenesis and Myogenesis (Figure 2B), with myxoid histologic features and fibroblast infiltration. A third subtype characterized by mutations in KEAP1, CUL3 and NFE2L2 genes and high-level by mutations in KEAP1, CUL3 and NFE2L2 genes and high-level amplification (GISTIC threshold value = 2) of SOX2 and TP63 (Figure 2A\\u2013B, Table S2) was labeled the \\u201cClassical\\u201d subtype, consistent with TCGA nomenclature and with a previous publication. These with TCGA nomenclature and with a previous publication. These showed classical histologic features, CIMP-high enrichment, upregulation of OxPhos- and proliferation-related pathways, and downregulation of immune signaling (Figure 2A\\u2013B). A fourth NMF downregulation of immune signaling (Figure 2A\\u2013B). A fourth NMF subtype was designated \\u201cInflamed-Secretory\\u201d (I-S) due to its alignment with the RNA-based Secretory cluster and strong upregulation of immune-related pathways. The fifth, and strong upregulation of immune-related pathways. The fifth, \\u201cProliferative-Primitive\\u201d (P-P) NMF subtype displayed upregulated proliferation-related pathways, downregulated immune signaling, and enrichment of CIMP-low samples. Within each of these five and enrichment of CIMP-low samples. Within each of these five NMF clusters were samples with low NMF cluster membership scores, which could be considered as belonging to a \\u201cmixed\\u201d subgroup, since they showed features of more than one subtype. Mixed since they showed features of more than one subtype. Mixed subgroup membership, a possible readout of tumor heterogeneity, conferred significantly worse survival (Figure 2C), an observation also made in a recent proteogenomic analysis of glioblastoma. The made in a recent proteogenomic analysis of glioblastoma. The mixed class was associated in the current cohort with increased frequency of SOX2 amplifications (p = 0.0038) but not grade, mutation burden, stemness or other tested variables. We compared B-I (Figure S2B) and EMT-E (Figure S2C) to other subtypes to identify distinguishing features. B-I had hyperphosphorylated M phase Histone 1 T11 and elevated expression of both TACSTD2 (TROP2), a cell-surface glycoprotein that drives growth TACSTD2 (TROP2), a cell-surface glycoprotein that drives growth and metastasis, and MARK2, a kinase that regulates cell polarity and leads to cisplatin resistance. Transcription activator PBX3, which is upregulated only at the protein level (Figure S2B), which is upregulated only at the protein level (Figure S2B), promotes invasion, proliferation, and chemoresistance and is associated with poor prognosis, including in laryngeal squamous cell carcinoma. Pathway analysis of proteins highly correlated with carcinoma. Pathway analysis of proteins highly correlated with PBX3 highlighted neutrophil activation and degranulation (Figure S2D), consistent with the significant upregulation of neutrophilic granule proteins and chemotactic factors (Figure S2E) and granule proteins and chemotactic factors (Figure S2E) and with the xCell-based deconvolution signature (Figure S2F). This correlation was not observed in paired NATs, suggesting a tumor-specific role. In addition to a strong EMT signature (Figure 2B), the EMT-E subtype showed upregulated VCAN (Figure S2C), a tumor-promoting target of SNAIL expressed in cancer-associated fibroblasts (CAFs), and FHL3, a protein that stabilizes EMT-associated (CAFs), and FHL3, a protein that stabilizes EMT-associated transcription factors (Figure S2C). DVL3, FN1 and FHL2, all Wnt pathway regulators, were upregulated in the proteomics dataset, and cell-cell signaling by Wnt was one of the significantly and cell-cell signaling by Wnt was one of the significantly (FDR=0.04) enriched pathways in this subtype. As shown in Figure 2D\\u2013E, coordinated high expression of EMT and CAF proteins, xCell fibroblast scores, and a \\u201cfibroblast proliferation\\u201d signature fibroblast scores, and a \\u201cfibroblast proliferation\\u201d signature suggested that CAFs and tumor epithelium might collaborate in the EMT-E phenotype. This was further supported by enrichment of EMT-related receptor tyrosine kinases (RTKs), as described below. receptor tyrosine kinases (RTKs), as described below. Immunohistochemical co-staining of a small subset of tumors for the epithelial marker pan-cytokeratin (CK) and CAF marker \\u03b1-smooth muscle actin (\\u03b1-SMA) demonstrated the presence in EMT-E tumors of both actin (\\u03b1-SMA) demonstrated the presence in EMT-E tumors of both CAFs and tumor cells undergoing EMT (Figure S2G), with significant enrichment of co-stained cells in the EMT-E subtype (Wilcoxon p=0.025), but not of cells showing single staining of either p=0.025), but not of cells showing single staining of either CK or \\u03b1-SMA alone. TGF\\u03b2 is a well-established and potent inducer of EMT, and a Library of Integrated Network-Based Cellular Signatures-based (LINCS) query for compounds that reversed the EMT-E (LINCS) query for compounds that reversed the EMT-E signature showed enrichment for TGF\\u03b2 inhibitors (Figure S2H, Table S4H). Integrating our global proteome data with a prior LSCC dataset resulted in six clusters largely recapitulating RNA, proteomic and multi-omic clusters defined in the current study (Figure S2I\\u2013J). NMF EMT-E subtype tumors show phosphorylation-driven PDGFR and ROR2 signaling. Though LSCC lacks the RTK mutations for which targeted inhibition has dramatically improved therapeutic options in LUAD, other modes of RTK activation, inferred from phosphoproteomic data, may nonetheless nominate targeted inhibition for some LSCC may nonetheless nominate targeted inhibition for some LSCC patients. Hence, we derived a serine/threonine-predominant correlation-based phosphosite enrichment score (CBPE score) for all RTKs in our tumor cohort. Of nine RTKs with high CBPE scores in LSCC in our tumor cohort. Of nine RTKs with high CBPE scores in LSCC tumors, seven were significantly associated with NMF subtypes (Kruskal-Wallis p <0.01). Scores for PDGFRB and ROR2, markedly elevated in EMT-E (Figure 2F), were highly correlated with the elevated in EMT-E (Figure 2F), were highly correlated with the loss of epithelial cell-cell junction markers and upregulation of mesenchymal proteins involved in remodeling of extracellular matrix, induction of EMT, and promotion of cell migration, which induction of EMT, and promotion of cell migration, which mark the transition from epithelial to mesenchymal state (Figure 2G). As summarized in Figure S2K, we found phosphosite-based evidence for upregulation of both the planar cell polarity and evidence for upregulation of both the planar cell polarity and calcium-dependent branches of non-canonical Wnt signaling, initiated through ROR2 or PDGFR, both targetable and recently implicated in the progression from a hybrid E/M to a highly mesenchymal in the progression from a hybrid E/M to a highly mesenchymal state. Further functional characterization is warranted to confirm the contribution of these RTKs to the EMT-E phenotype and to test the impact of their modulation. Loss of CDK4/6 pathway inhibitors is a universal feature of LSCC but Rb1 expression is variable We investigated the impact of recurrent mutations on cognate RNA, proteins and PTMs (cis-effects) and on a set of cancer associated genes (trans-effects). Significant (FDR<0.05) Cis/Trans pairs are shown in Figure 3A\\u2013D. RB1 mutated tumors had upregulated are shown in Figure 3A\\u2013D. RB1 mutated tumors had upregulated cell cycle-related protein expression. NOTCH1 mutation led to elevated GSK3B protein and phosphorylation (T433), downregulation of inhibitory sites on GSK3A (S21) and upregulation of NLK. NLK of inhibitory sites on GSK3A (S21) and upregulation of NLK. NLK and GSK3B inhibit NOTCH1, recently described as a tumor suppressor in LSCC, suggesting that both might inhibit NOTCH1 downstream signaling in tumors with NOTCH1 mutations. ARID1A is one of signaling in tumors with NOTCH1 mutations. ARID1A is one of the most frequently mutated SWItch Sucrose Non-Fermentable (SWI/SNF) ATP-dependent chromatin remodeling complex members. Mutations were associated with higher tumor mutational burden (Figure S3A) associated with higher tumor mutational burden (Figure S3A) and led to protein downregulation, associated with worse overall survival in NSCLC. Paradoxically, mutations in CDKN2A, proposed as a potential biomarker for CDK4/6 inhibitors, resulted in increased RNA expression (Figure 3A). We analyzed the effects of genetic and epigenetic alterations of CDKN2A on both its major isoforms, p16INK4a alterations of CDKN2A on both its major isoforms, p16INK4a (p16) and p14ARF (p14) (Table S4). As expected, its homozygous deletion resulted in the loss of expression of both isoforms (Figure 3E). However, only eight of the 58 CDKN2A wild-type (WT) samples However, only eight of the 58 CDKN2A wild-type (WT) samples showed p16 RNA expression levels that were comparable to samples with CDKN2A/p16 missense mutations (log2(FPKM+1)>2). Many of those CDKN2A/p16 mutations were unlikely to result in Many of those CDKN2A/p16 mutations were unlikely to result in nonsense-mediated decay (Figure S3B), accounting for preserved expression in those samples. Notably, 28 WT tumors showed hypermethylation of the p16 but not the p14 promoter and suppression of of the p16 but not the p14 promoter and suppression of RNA and protein expression for CDKN2A/p16, while 18 additional WT tumors had reduced expression despite having no clear genetic or epigenetic alteration. Furthermore, the WT samples with high or epigenetic alteration. Furthermore, the WT samples with high CDKN2A/p16 expression had RB1 mutations or deletions and/or low Rb protein levels, consistent with a recently published pan-cancer analysis of the CDK4/6 pathway that showed mutual pan-cancer analysis of the CDK4/6 pathway that showed mutual exclusivity between mutations in these two genes. This suggests that loss of one of these two key CDK4/6 pathway inhibitors is a universal feature of LSCC. Conversely, while amplification of a universal feature of LSCC. Conversely, while amplification of Cyclin D - CDK4/6 complex genes is frequent in LSCC, there was no association between the amplification of CCND2, CCND3, CDK4, or CDK6 and either CDKN2A/p16 protein or Rb CCND2, CCND3, CDK4, or CDK6 and either CDKN2A/p16 protein or Rb protein/phosphoprotein levels (Table S4). CCND1 amplification resulted in significantly higher mean levels of Rb protein and phosphoprotein compared to WT samples, though the distributions phosphoprotein compared to WT samples, though the distributions were overlapping (Figure S3C). Intriguingly, phospho-Rb levels, which reflect CDK4/6 activity, correlated with response to CDK4/6 inhibitors in LSCC cell lines despite the heterogeneous CDK4/6 inhibitors in LSCC cell lines despite the heterogeneous distribution in response for cells with mutations in CDKN2A and copy number alterations in CCND1 and CDKN2A (Figure S3D). LSCC trial data have generally shown minimal efficacy of CDK4/6 LSCC trial data have generally shown minimal efficacy of CDK4/6 inhibition, but outlier responses have been present in trials with abemaciclib and palbociclib. While we found that CCND1 amplification resulted in higher average levels of phosphorylated Rb, resulted in higher average levels of phosphorylated Rb, the heterogeneity of Rb expression and phosphorylation in samples with CCND1 amplification provides a potential explanation for some patients\\u2019 lack of response to CDK4/6 inhibitor therapies. Improved lack of response to CDK4/6 inhibitor therapies. Improved prediction based on a downstream functional assessment of G1/S checkpoint alterations (i.e. RB1 protein expression and phosphorylation) may identify a subset of tumors sensitive to CDK4/6 may identify a subset of tumors sensitive to CDK4/6 inhibition. NRF2 pathway activation in tumors with and without NRF2 pathway mutations LSCC showed mutations of three key genes in the NRF2 antioxidant response pathway, NFE2L2 (NRF2), CUL3 and KEAP1. NFE2L2 phosphorylation was observed in samples with any of these mutations (Figure S3E). LSCC tumors were annotated as \\u201cNRF2 pathway mutated\\u201d S3E). LSCC tumors were annotated as \\u201cNRF2 pathway mutated\\u201d (n=33; with either one hit [n=28] or two hits [n=5] in NFE2L2, KEAP1, and CUL3), or \\u201cNRF2 WT\\u201d (n=68). Compared to WT, NRF2 pathway mutated tumors showed highly concordant differential expression mutated tumors showed highly concordant differential expression of mRNA, protein and phosphoprotein levels of NRF2 pathway genes (Figure 3F, Figure S3F\\u2013G, Table S4), some with therapeutic implications. A subset of NRF2 gene products showed a dosage effect A subset of NRF2 gene products showed a dosage effect (SQSTM1, NR0B1, AKR1B10, CARD11, FFAR2) with the five cases harboring two hits (Table S4) demonstrating increased up-/down-regulation of NRF2 pathway genes relative to one-hit cases (Figure S3H). Using pathway genes relative to one-hit cases (Figure S3H). Using the proteogenomic signature (Table S4) that defined NFE2L2 mutated tumors, we derived an ssGSEA-based NRF2 pathway score that was enriched not only in NRF2 pathway mutated tumors, but also in enriched not only in NRF2 pathway mutated tumors, but also in samples without NRF2 pathway mutations, especially in the NMF Classical subtype (Figure 3G). These samples showed increased NFE2L2 phosphorylation, indicative of NRF2 pathway activation. NFE2L2 phosphorylation, indicative of NRF2 pathway activation. Although the kinase(s) responsible for phosphorylation of NRF2 in LSCC are unclear, CDK5 protein (Figure 3H) and PTM-SEA-derived activity (Figure 3I) were significantly upregulated (FDR<0.05) activity (Figure 3I) were significantly upregulated (FDR<0.05) in the Classical subtype. CDK5 phosphorylation of NFE2L2 S433 mediates its activation in astrocytes and can likely play a role in CDK5-mediated NRF2 activation in LSCC tumors. Intriguingly, in CDK5-mediated NRF2 activation in LSCC tumors. Intriguingly, unlike LUAD, KEAP1 mutations did not result in significantly reduced protein expression in LSCC, although missense mutations were mostly nonoverlapping, suggesting heterogeneity in NRF2 were mostly nonoverlapping, suggesting heterogeneity in NRF2 pathway dysregulation in NSCLC subtypes. Proteogenomic analysis of chromosome 3 prioritizes therapeutic targets in LSCC Chromosome 3q, which harbors key squamous differentiation markers SOX2 and TP63, showed the most dramatic arm-level amplification in this cohort (Figure S4A). To assess potential drivers in the 3q amplicon, we identified 3q genes for which CNA correlated 3q amplicon, we identified 3q genes for which CNA correlated with RNA and protein and expression differed between tumors and NATs. TP63 showed the highest elevation in tumors (Figure 4A). LSCC cell lines amplified for TP63 and SOX2 were highly dependent cell lines amplified for TP63 and SOX2 were highly dependent upon them, supporting oncogene addiction (Figure S4B). As previously reported, \\u0394Np63\\u03b1 was the dominant TP63 isoform in this dataset (Figure S4C) and was highly correlated with TP63 expression (Figure S4D). Its amplification was associated with improved survival in the TCGA dataset (Figure S4E). In contrast to survival in the TCGA dataset (Figure S4E). In contrast to the general elevation of TP63 copy number, transcript, and protein abundance in LSCC tumors compared to NATs, 10 \\u201cNp63-low\\u201d samples histologically confirmed to be LSCC showed no elevation at RNA or confirmed to be LSCC showed no elevation at RNA or protein levels (Figure 4B). The apoptosis inhibitor survivin (BIRC5) was among the most overexpressed proteins in this Np63-low group (Figure 4C). Notably, a substantial number of LSCC cell lines also had 4C). Notably, a substantial number of LSCC cell lines also had low Np63 expression (Figure S4F), and those TP63-low cell lines were significantly more vulnerable to the survivin inhibitor YM-155 (Figure S4G). Survivin promotes cell proliferation and G2M (Figure S4G). Survivin promotes cell proliferation and G2M checkpoints, evident in Np63-low samples (Figure S4H). Small molecule inhibitors and inhibitory peptides targeting survivin are being explored as part of ongoing clinical trials including in being explored as part of ongoing clinical trials including in NSCLC, but have shown modest or no improvement. Our data on the newly identified Np63-low LSCC tumors suggests that TP63 status may identify patients with potential response to survivin may identify patients with potential response to survivin inhibition. Overexpression of miR-205, a biomarker to distinguish LSCC from other NSCLC (Figure S4I), was associated with decreased promoter DNA methylation (Figure S4J) and elevation of \\u0394Np63\\u03b1, a transcriptional regulator of this miRNA. miR-205 showed high a transcriptional regulator of this miRNA. miR-205 showed high correlation with TP63 expression and its expression was markedly reduced in Np63-low samples (Figure 4D, Figure S4K). As expected, the abundance of most target gene products was negatively the abundance of most target gene products was negatively correlated with miR-205 expression, including ZEB1/2 and PTEN (Figure 4E). TP73 positively correlated with miR-205 at the transcript (Figure 4E) and still more strongly at the protein level (Figure 4E) and still more strongly at the protein level (Figure 4F) and with TP63 protein but not RNA (Figure S4L). TP73 overexpression mediates \\u0394Np63\\u03b1-dependent cell survival in squamous cell carcinoma, suggesting a mechanistic model in which functional and suggesting a mechanistic model in which functional and stoichiometric interaction between TP63 and TP73 leads to miR-205 overexpression in LSCC. Finally, consistent with the downregulation of key EMT-related proteins, the TP63 target miR-205 showed of key EMT-related proteins, the TP63 target miR-205 showed significant negative correlation with EMT activity but positive correlation with DNA replication (Figure 4G). The seemingly paradoxical role of TP63 in repressing EMT has been confirmed in role of TP63 in repressing EMT has been confirmed in previous studies. Of note, although \\u0394Np63\\u03b1 and miR-205 were consistently highly overexpressed in the B-I and Classical subtypes compared to NATs, their abundances were variable and not particularly low NATs, their abundances were variable and not particularly low in the EMT-E subtype. Thus, \\u0394Np63\\u03b1, either via miR-205 or indirectly, may induce a strong squamous epithelial cell phenotype in the B-I and Classical tumors, with additional mechanisms leading B-I and Classical tumors, with additional mechanisms leading to derepression and a metastatic phenotype, and other mechanisms beyond miR-205 may be involved in promoting the EMT phenotype in EMT-E samples. SOX2, a recurrently amplified squamous cell marker often co-amplified with TP63, was also overexpressed in tumors (Figure 4A). SOX2 is considered undruggable, intensifying therapeutic interest in upstream or downstream targets. An unbiased search (Figure in upstream or downstream targets. An unbiased search (Figure 4H) identified a significant correlation of LSD1 (KDM1A) with SOX2 protein. LSD1 regulates SOX2 expression and its inhibition in LSCC is currently being explored in a clinical setting in in LSCC is currently being explored in a clinical setting in conjunction with immunotherapy (NCT04350463). Other identified chromatin regulators included KDM3A, known to regulate SOX2 expression in ovarian cancer, and EZH2, whose inhibition in triple in ovarian cancer, and EZH2, whose inhibition in triple negative breast cancer leads to downregulation of SOX2 expression. EZH2 dependency was observed in SOX2 amplified LSCC cell lines (Figure S4M). The positive correlations of SOX2 with ALDH1A1, (Figure S4M). The positive correlations of SOX2 with ALDH1A1, ALDH3A1, and WNT5A (Figure 4H) suggest stemness-like features in these tumors. Interestingly, JAK1 was the protein most negatively correlated with SOX2 among other interferon-signaling proteins correlated with SOX2 among other interferon-signaling proteins such as IRF3 and IFNGR1 (Figure 4H), and negative correlation was seen between SOX2 and JAK-STAT signaling (Figure 4I). Crosstalk between lysine acetylation and ubiquitylation impacts cancer metabolism Consensus clustering of K-GG peptide abundances after correcting for cognate protein abundance revealed two stable ubiquitylproteome clusters (Figure 5A, Table S5) associated with country of origin and ethnicity (Fisher\\u2019s exact p<0.01). Proteins in of origin and ethnicity (Fisher\\u2019s exact p<0.01). Proteins in pathways such as glycolysis, JAK-STAT, MAPK, and immune signaling were differentially modified between these clusters (Figure 5A, Figure S5A\\u2013B, Table S5). To identify candidate enzymes driving Ub and UbL modifications in LSCC, we correlated E3 ubiquitin ligases or deubiquitylases (DUBs) to KGG-sites (Table S5). Eighteen DUBs with at least one negatively correlated K-GG site and 35 E3 ligases with at least negatively correlated K-GG site and 35 E3 ligases with at least one positively correlated K-GG site were identified, with top sites shown in Figure 5B and Figure S5C. HERC5 is the major E3 ligase involved in conjugation of the UbL modification ISG15 in ligase involved in conjugation of the UbL modification ISG15 in humans; hence the numerous positively correlated Ub/UbL sites (n=474; FDR<0.01) are likely ISG15 modifications. Consistent with this, we observed positive correlation between ISG15 and HERC5 this, we observed positive correlation between ISG15 and HERC5 abundance (Figure S5D). Positively-correlating Ub/UbL sites were enriched for glycolysis and TCA cycle annotations (Figure S5E). We observed several positively-correlating K-GG sites in the We observed several positively-correlating K-GG sites in the key glycolytic enzymes PKM, PGK1, and ENO1 (Figure 5C, Figure S5F). No significant negative correlations were observed between total protein abundance and their respective K-GG sites or HERC5, protein abundance and their respective K-GG sites or HERC5, suggesting a regulatory rather than degradative role, as expected for ISGylation (Figure S5G, Figure S5H). ISG15 modifications have been described in these glycolytic enzymes, and ISG15 knockdown been described in these glycolytic enzymes, and ISG15 knockdown in pancreatic cancer cells reduced oxidative phosphorylation and glycolysis. ISGylation can be induced by interferon in cancer; accordingly, HERC5 and ISG15 were positively associated with accordingly, HERC5 and ISG15 were positively associated with proteome-derived IFN-G signaling scores (Figure S5I, Figure S5J), and HERC5 was increased in tumors of the IFN-gamma dominant subtype (Figure 5D). Recently, a direct link between immune subtype (Figure 5D). Recently, a direct link between immune signaling and glucose metabolism through ISGylation was shown in the context of pathogenic invasion. Collectively, our data supports a potential role for HERC5 as a bridge between immune supports a potential role for HERC5 as a bridge between immune signaling and metabolic regulation in LSCC. This dataset allowed us to investigate crosstalk between lysine PTMs. We employed a modified version of CLUMPS to detect clustering of either acetylation or ubiquitylation sites within protein 3D structures. Significant ubiquitylation clusters were protein 3D structures. Significant ubiquitylation clusters were detected in 33 proteins and significant acetylation clusters in 17 proteins (p<0.05) (Table S5). Top hits revealed enrichment of glycolysis, fatty acid metabolism and OxPhos (Table S5), of glycolysis, fatty acid metabolism and OxPhos (Table S5), supporting modulation of these metabolic pathways by ubiquitylation and acetylation. In tumor-NAT comparison, major glycolytic enzymes PGK1 and PKM showed reduced acetylation at multiple known PGK1 and PKM showed reduced acetylation at multiple known inhibitory sites closely-positioned on the 3D structure, as well as on adjacent lysines surrounding the enzymatic pocket  (Figure 5E, Figure S5K). Tumors also had decreased ubiquitylation at 5E, Figure S5K). Tumors also had decreased ubiquitylation at closely-positioned K-GG-sites for TXN1, an important redox regulator (Figures 5F). PGK1 and PKM deacetylation suggests increased activity of these key glycolytic enzymes in tumors, a conclusion of these key glycolytic enzymes in tumors, a conclusion supported by increased phosphorylation on known activating sites of regulatory proteins (PFKFB3 pS461, PKM pS37). Upregulation of a glycolytic metabolic phenotype confers selective advantage to a glycolytic metabolic phenotype confers selective advantage to cancer cells by supporting uninterrupted growth, dynamically modulated by PTMs. While site-level roles of most Ub/UbL modifications and acetylsites are unclear, our data on acetylation, and acetylsites are unclear, our data on acetylation, phosphorylation, and HERC5-mediated ISGylation of glycolytic enzymes hints at the complexity of regulation of this cancer hallmark. Rapid growth of cells often leads to oxidative stress within tumors, which adapt by upregulating redox systems. Although thioredoxin (TXN) did not show overall differential protein expression between tumors and NATs, we observed increased TXN1 protein between tumors and NATs, we observed increased TXN1 protein combined with decreased Ub sites, particularly in the NMF Classical subtype (Kruskal Wallis FDR<0.01). The TXN1 activator TXNRD1 was also increased in the Classical subtype, while TXNIP, which was also increased in the Classical subtype, while TXNIP, which modulates cellular redox by binding and inhibiting TXN1, was decreased (Figure 5F). TXN and TXNRD1 were highly correlated with NRF2 score (TXN: R=0.85; TXNRD1: R=0.8, both p<2.2\\u00d710\\u221216), with NRF2 score (TXN: R=0.85; TXNRD1: R=0.8, both p<2.2\\u00d710\\u221216), consistent with their role in response to oxidative stress. Redox modulation through this pathway is associated with tumor proliferation, differentiation and prognosis in multiple epithelial differentiation and prognosis in multiple epithelial cancers, including NSCLC. The small proportion of glucose directed to the tumor mitochondria and subsequently to the TCA and OXPHOS cycles leads to increased acetyl CoA within the mitochondria and leads to increased acetyl CoA within the mitochondria and thence to non-enzymatic acetylation and inhibition of key enzymes in the fatty acid oxidation pathway such as ECH1 and ACADVL (also identified in the CLUMPS analysis). We recapitulate the finding identified in the CLUMPS analysis). We recapitulate the finding of increased ECH1 and ACADVL acetylation, including specific increase at the binding site of ACADVL in tumors (Figure 5E, Figure S5K). PTM-based modulation of these key metabolic and ROS S5K). PTM-based modulation of these key metabolic and ROS pathways is summarized in Figure 5G. As a complementary assessment of the crosstalk between lysine ubiquitylation and acetylation, we identified K-Ac or K-GG sites showing significant differential expression in tumors relative to NATs. Figure 5H shows 8 lysine sites where K-GG was relative to NATs. Figure 5H shows 8 lysine sites where K-GG was upregulated and K-Ac downregulated in tumors (upper panel) and 36 sites demonstrating the opposite trend (lower panel). Among the latter was the GTP-binding nuclear protein RAN, involved in the latter was the GTP-binding nuclear protein RAN, involved in nucleocytoplasmic transport, mitotic progression and spindle assembly. Acetylation on multiple lysine residues can affect RAN localization, GTP/GDP cycle and import / export complex RAN localization, GTP/GDP cycle and import / export complex formation. Consistent with the role of RAN in mitosis, RAN K127-acK showed dramatic upregulation in the P-P subtype (Figure 5I), named in part for its generally high proliferation (Figure 2B). named in part for its generally high proliferation (Figure 2B). The specific functional relevance of K127-ac has not been previously demonstrated, but its selective upregulation suggests a functional role. While of great interest to the PTM community and role. While of great interest to the PTM community and potentially very powerful, such crosstalk analyses are nascent, and interpretations remain suitably cautious. Immune landscape and regulation in LSCC Consensus clustering based on the xCell signatures (Table S6) identified three sets of tumors representing immunologically Hot, Warm, and Cold clusters (Figure 6A\\u2013C), as well as an NAT-enriched cluster. The NMF I-S subtype was significantly associated cluster. The NMF I-S subtype was significantly associated with the immune Hot cluster (Fisher\\u2019s exact p=6.371\\u00d710\\u221209) (Figure 6A). The Hot cluster was characterized by increased macrophages, CD4+, CD8+ and regulatory T cells, and dendritic cells in both CD4+, CD8+ and regulatory T cells, and dendritic cells in both deconvolution and immunohistochemistry (IHC) analyses (Figures 6A,C, S6C,D), and showed upregulation of immune-related pathways (Figure 6A\\u2013B, Figure S6A, Table S6). Immunosuppressive mediators 6A\\u2013B, Figure S6A, Table S6). Immunosuppressive mediators CTLA4, PD-1 (PDCD1), PD-L1 (CD274) and IDO1 and the key Treg transcriptional regulator FOXP3 were upregulated in Hot compared to Cold tumors, suggesting immune checkpoint-related therapeutic options suggesting immune checkpoint-related therapeutic options in these tumors. The Hot tumor acetylproteome was chiefly enriched for OXPHOS, Mitochondrial Complex and TCA Cycle pathways, consistent with increased metabolism or increased mitochondrial consistent with increased metabolism or increased mitochondrial acetyl-CoA leading to non-enzymatic acetylation of mitochondrial proteins (Figure 6D). Notably, NATs did not display consonant immune signaling (Figure S6B). The intermediate, Warm cluster immune signaling (Figure S6B). The intermediate, Warm cluster displayed upregulation of immune-related pathways such as PD-1 signaling, Interferon gamma (IFN-G) response, and Allograft rejection relative to Cold tumors, but did not show the downregulation relative to Cold tumors, but did not show the downregulation of cell cycle-related pathways observed in Hot tumors (Figure 6A\\u2013B, Figure S6A). In addition, canonical Wnt Signaling, active in tumor proliferation and immune evasion, was comparably in tumor proliferation and immune evasion, was comparably upregulated in the Warm and Cold tumor proteomes (Figure 6B). Fifteen pQTLs were also identified in proteins differentially abundant between Hot and Cold tumors (Table S6, Figure S6E) Rho GTPase signaling was upregulated in Hot tumors (Figure 6A, Table S6). ARHGDIB (RhoGDIb), the central regulatory molecule in GTPase activation, was the most significantly acetylated protein in the Hot cluster, followed by metabolic mitochondrial protein in the Hot cluster, followed by metabolic mitochondrial enzymes (Figure 6D). Acetylation of its homolog RhoGDIa at the conserved K135 site regulates F-actin assembly by activating RhoA. In contrast to RhoGDIa, the affinity of RhoGDIb for RhoA is In contrast to RhoGDIa, the affinity of RhoGDIb for RhoA is low, and the K135 acetylation of RhoGDIb may act as a positive regulator of Rac signaling via its interaction with Rac2 instead. In support of this hypothesis, we found significant upregulation support of this hypothesis, we found significant upregulation in the Hot cluster of Rho GTPase Rac2 and numerous Rac-specific regulators (Figures 6F, S6F, Table S6). Notably, these proteins are primarily immune-specific, indicating a possible role for are primarily immune-specific, indicating a possible role for lysine acetylation of RhoGDIb in promoting immune cell functions in the Hot cluster. Immunohistochemical staining for RhoGDIb (ARHGDIB) in the subset of immune Hot tumors confirmed strong (ARHGDIB) in the subset of immune Hot tumors confirmed strong signals in infiltrating immune cells and mesenchymal cells (Figure S6G), similar to dispersed RhoGDIb-expressing cells identified as lymphocytes seen in renal cell carcinomas. Thus, we propose as lymphocytes seen in renal cell carcinomas. Thus, we propose that K135 acetylation of RhoGDIb is a potential regulatory mechanism of immune cell functions that is in line with the inflammasome signature and immune-enriched networks. We explored the immunological associations of the RTK CBPE scores described above (Figure 2I,J) in LSCC tumors (Figure 6G). CSF1R, predominantly expressed in macrophages, showed the highest CBPE scores, and was positively associated with the Hot cluster CBPE scores, and was positively associated with the Hot cluster (Figure 6H) and consequently with immune-related pathways and cell types (Figure 6G). CSF1R correlated with immunosuppressive chemokine CCL5, phosphorylation of PIK3R1 Y580, downstream chemokine CCL5, phosphorylation of PIK3R1 Y580, downstream regulators of the actin cytoskeleton including WAS and WASF2 S103, and upregulation of AKAP13, among other immune-related proteins (Figure 6I). Tumor-associated macrophages (, CSF1R signaling, and 6I). Tumor-associated macrophages (, CSF1R signaling, and AKAP13 have each been proposed as anti-cancer targets. Proteomic Biomarker Candidates for Prognosis, Diagnosis, and Treatment Tumors and paired NATs revealed remarkable differences in protein expression and pathway enrichment (Figure 7A, Table S7). The top 50 of 502 differential proteins are shown in Figure S7A. In addition to 206 upregulated oncogenic proteins (OncoKB, Table addition to 206 upregulated oncogenic proteins (OncoKB, Table S7), several CT antigens, attractive targets of CT-antigen therapies, were overexpressed in tumors (Figure S7B), with the I-S samples showing the fewest CT antigens. As complete long-term samples showing the fewest CT antigens. As complete long-term follow-up was not yet available for this cohort, we leveraged the TCGA LSCC dataset to examine the association of these 502 candidate tumor biomarkers with overall survival (OS) or disease-free tumor biomarkers with overall survival (OS) or disease-free survival (DFS). Expression of four of the most highly differential genes showed significant association with poor OS and another 15 genes with poor DFS (Figure 7B, Figure S7C). Furthermore, 15 genes with poor DFS (Figure 7B, Figure S7C). Furthermore, knockdown of these tumor biomarker proteins reduced fitness across 16 LSCC cell lines (https://depmap.org/), suggesting critical roles in key cellular transformation and proliferation processes in key cellular transformation and proliferation processes (Figure 7C, Table S7). There is increasing interest in molecular taxonomies that help position cancer types in the context of related cancers. CNAs were more frequent in LSCC than in LUAD or head and neck squamous cell carcinoma (HNSCC), which respectively share tissue and cell carcinoma (HNSCC), which respectively share tissue and cell type of origin (Figure S7D). Widespread deletions of immune-related genes in LSCC correlated with ESTIMATE immune scores (Figure 7D). Immune-related deletions were less prominent in HNSCC and 7D). Immune-related deletions were less prominent in HNSCC and especially LUAD (Figure S7E), except for Chromosome 3p deletion, encompassing five chemokine receptors. Squamous cancers shared 3q amplification, JAK2 deletion and skin-development proteins 3q amplification, JAK2 deletion and skin-development proteins from across the genome. Finally, deletions of 4p14 and 5q, including multiple Toll-like receptor and interleukin signaling genes, were unique to LSCC. Among key tumor-specific phosphorylation were unique to LSCC. Among key tumor-specific phosphorylation events in LSCC, we identified upregulation of activating sites on 27 kinases, including MAPK14 and three others (DCK, EGFR, SRC) targetable by FDA-approved drugs (Figure S7F), with several targetable by FDA-approved drugs (Figure S7F), with several being shared by LUAD and HNSCC tumors. Notably, EGFR protein was significantly upregulated in the squamous cancers but not LUAD (Figure S7G), although LUAD had many activating EGFR mutations, and S7G), although LUAD had many activating EGFR mutations, and activating phosphorylation of EGFR was increased in both LUAD and LSCC. High EGFR amplification in the LSCC cohort did not correlate with EGFR pathway activity as assessed by mRNA-based PROGENy with EGFR pathway activity as assessed by mRNA-based PROGENy scoring (Figure 7E). Instead, this activity measurement was highly correlated to mRNA abundance of the five EGFR ligands that are prominent in tumors (Figure 7F). Furthermore, EGFR amplified prominent in tumors (Figure 7F). Furthermore, EGFR amplified samples displayed increased phosphorylation on proteins related to actin filament and cell junction organization as well as cell-substrate adhesion (Figure 7G). These findings and similar cell-substrate adhesion (Figure 7G). These findings and similar results recently reported in HNSCC support a squamous cell cancer feature in which EGFR ligand abundance drives canonical EGFR pathway activity and tumors with very high levels of EGFR may no activity and tumors with very high levels of EGFR may no longer require the ligand to drive activity. EGFR inhibition guided by EGFR abundance has been unsuccessful in squamous tumors. Since EGFR inhibitors such as cetuximab affect ligand-induced EGFR EGFR inhibitors such as cetuximab affect ligand-induced EGFR activity, ligand abundance, rather than EGFR amplification, might better predict EGFR inhibitor response in this population. Figure 7H graphically represents these and other key findings Figure 7H graphically represents these and other key findings presented in this manuscript, emphasizing translatable implications. Discussion In this study, we provide detailed proteogenomic profiles of treatment-na\\u00efve, primary LSCC tumors and paired NATs, with unprecedented coverage of post-translational modifications including phosphorylation, acetylation and for the first time, including phosphorylation, acetylation and for the first time, ubiquitylation. Unsupervised NMF-based multi-omic clustering suggested a refinement to prior RNA-based clustering, dividing basal tumors into Basal-inclusive and EMT-enriched subtypes with tumors into Basal-inclusive and EMT-enriched subtypes with biological differences of potential therapeutic significance. Molecular events downstream of copy number alterations showed NSD3 rather than FGFR1 protein overexpression in tumors with 8p11.23 rather than FGFR1 protein overexpression in tumors with 8p11.23 amplification, providing both a potential explanation for the limited effectiveness of FGFR1-targeted treatment in this population and a potential alternative therapy. Parallel downstream and a potential alternative therapy. Parallel downstream analysis of DNA promoter methylation events provided insight into regulation of many proteins differential between cancer and NATs. Comprehensive proteogenomic data provided a deeper exposition of established LSCC biology, often with potential therapeutic implications. We identified a subset of low-p63 tumors that were characterized by high levels of the known therapeutic target characterized by high levels of the known therapeutic target survivin. SOX2 is considered undruggable, but our analyses provide rationale for exploring LSD1 or other chromatin modifiers such as EZH2 to target SOX2 amplified / overexpressing tumors. The as EZH2 to target SOX2 amplified / overexpressing tumors. The importance of glycolysis and oxidative stress in LSCC are well-appreciated, but our data support complex regulation of these metabolic pathways by crosstalk between ubiquitylation (or metabolic pathways by crosstalk between ubiquitylation (or ubiquitin-like modifications), phosphorylation and acetylation. Immunotherapy represents the greatest advance in LSCC therapy in decades, but outcomes lag behind those seen in patients with LUAD but outcomes lag behind those seen in patients with LUAD and still only a minority of patients exhibit long-term responses; numerous immune-related proteogenomic observations suggest directions for further investigation. Proteogenomic dissection of the for further investigation. Proteogenomic dissection of the downstream effects of CDKN2A mutations had clinical implications related both to the interpretation of trials utilizing CDK4/6 inhibitors in LSCC patients and to biomarker selection for future in LSCC patients and to biomarker selection for future studies: though CDK4/6 inhibition has shown limited efficacy in LSCC trials to date, our analysis suggested that a more nuanced assessment of RB1 protein expression and phosphorylation is required of RB1 protein expression and phosphorylation is required before declaring this approach unsuccessful. Finally, triangulation between LSCC, LUAD and HNSCC demonstrated the influence of both tissue and cell type of origin on cancer biology, with of both tissue and cell type of origin on cancer biology, with therapeutic vulnerabilities both unique and common across these cancer types. Limitations of the study Many important clinical advances in NSCLC and other cancers have been driven by genomic profiling of bulk tumor material, and we anticipate that the same will prove true of bulk proteogenomic characterization, as performed here. Nevertheless, tumor characterization, as performed here. Nevertheless, tumor heterogeneity is an important complication, and emerging methods for proteomics approaches to heterogeneity, such as pairing microdissection with mass spectrometry or using mass cytometry will be with mass spectrometry or using mass cytometry will be useful adjuncts. It is typically metastatic disease that proves lethal, and in-depth proteogenomic analysis of primary tumors paired with metastatic lesions will be needed to provide critical insights metastatic lesions will be needed to provide critical insights into metastatic biology. Importantly, the scope of a landscape study such as this necessitates that it be understood as hypothesis-generating, and a wider community effort will be required to and a wider community effort will be required to validate biological observations and suggested therapeutic alternatives. Nevertheless, we hope this study will prove a valuable resource to the research and clinical communities, and advance the to the research and clinical communities, and advance the understanding and treatment of LSCC. STAR\\u2605METHODS\\n\\nRESOURCE AVAILABILITY\\n\\nLead contact\\n\\nFurther information and requests should be directed to and will be fulfilled by the lead author Michael A. Gillette (gillette@broadinstitute.org)\\n\\nMaterial availability Material availability\\n\\nThis study did not generate new unique reagents.\\n\\nData and Code Availability\\n\\nCPTAC LSCC proteomics data:\\n\\nProteomic Data Commons (PDC); https://pdc.cancer.gov/pdc/ with identifiers PDC000232, PDC000233, PDC000234 and PDC000237 CPTAC data portal LSCC: https://cptac-data-portal.georgetown.edu/study-summary/S063\\n\\nCPTAC HNSCC proteomics data:\\n\\nPDC: https://pdc.cancer.gov/pdc/ with identifiers PDC000221 and PDC000222\\n\\n https://cptac-data-portal.georgetown.edu/study-summary/S054 https://cptac-data-portal.georgetown.edu/study-summary/S054 \\n\\nCPATC LUAD proteomics data:\\n\\nPDC: https://pdc.cancer.gov/pdc/ with identifiers PDC000153, PDC000149, PDC000224)\\n\\n https://cptac-data-portal.georgetown.edu/study-summary/S056 https://cptac-data-portal.georgetown.edu/study-summary/S056 \\n\\nGenomic and transcriptomic data files can be accessed at the Genomic Data Commons (GDC); https://portal.gdc.cancer.gov/, via dbGaP Study Accession: phs001287.v10.p5 All histologic https://www.cancerimagingarchive.net/datascope/cptac/home/ and radiologic details can be accessed from the The Cancer Imaging Archive (TCIA) Public Access https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC. Sample annotation, processed and normalized data files are provided as Tables S1\\u2013S3.\\n\\nSoftware and code used in this study are referenced in their corresponding STAR Method sections and also the Key Resource Table.\\n\\nEXPERIMENTAL MODEL AND SUBJECT DETAILS EXPERIMENTAL MODEL AND SUBJECT DETAILS\\n\\nHuman Subjects A total of 113 participants (90 males, 23 females, 40\\u201388 years old) were collected for this study by 13 different tissue source sites from seven different countries between May 2016 and August 2018, with a median follow-up time of 2.5 years. Five (one 2018, with a median follow-up time of 2.5 years. Five (one female and four male) samples were excluded based on further pathological assessment and 108 tumors and 99 paired NATs were used in this study. Only histopathologically-defined adult lung squamous study. Only histopathologically-defined adult lung squamous tumors were considered for analysis, with an age range of 40\\u201388. Institutional review boards at tissue source sites reviewed protocols and consent documentation adhering to the Clinical Proteomic and consent documentation adhering to the Clinical Proteomic Tumor Analysis Consortium (CPTAC) guidelines. Clinical data were obtained from tissue source sites and aggregated by an internal database called the CDR (Comprehensive Data Resource) that database called the CDR (Comprehensive Data Resource) that synchronizes with the CPTAC DCC (https://cptac-data-portal.georgetown.edu/). Clinical data can be accessed and downloaded from the DCC (Data Coordinating Center). Demographics, histopathologic DCC (Data Coordinating Center). Demographics, histopathologic information, and treatment details were collected. LSCC histopathology was confirmed for all cases by at least 2 expert pathologists based on high resolution images of H&E sections. For samples based on high resolution images of H&E sections. For samples with low TP63 status, additional pathologic confirmation was performed. The histologic, genotypic, clinical, geographical and other associated metadata is summarized in Table S1. METHOD DETAILS\\n\\nSpecimen Acquisition The tumor, normal adjacent tissue (NAT), and whole blood samples used in this manuscript were prospectively collected for the CPTAC project. Biospecimens were collected from newly diagnosed patients with LSCC who underwent surgical resection and had patients with LSCC who underwent surgical resection and had received no prior treatment for their disease, including chemotherapy or radiotherapy. All cases had to be of acceptable LSCC histology but were collected regardless of surgical stage or histology but were collected regardless of surgical stage or histologic grade. Cases were staged using the AJCC cancer staging system 7th edition and the 8th edition. The tumor specimen weights ranged from 125 to 1560 milligrams. The average tissue mass ranged from 125 to 1560 milligrams. The average tissue mass was 220 mg. For most cases, three to four tumor specimens were collected. Paired histologically-normal adjacent lung tissues (NATs) were collected from the same patient at tumor resection. Each were collected from the same patient at tumor resection. Each tissue specimen endured cold ischemia for less than 40 minutes prior to freezing in liquid nitrogen; the average ischemic time was 16 minutes from resection/collection to freezing. Specimens was 16 minutes from resection/collection to freezing. Specimens were flash frozen in liquid nitrogen. Histologic sections obtained from top and bottom portions from each case were reviewed by a board-certified pathologist to confirm the assigned by a board-certified pathologist to confirm the assigned pathology. For samples to be deemed acceptable, the top and bottom sections had to contain an average of 50% tumor cell nuclei with less than 20% necrosis. Specimens were shipped overnight from the than 20% necrosis. Specimens were shipped overnight from the tissue source sites to the biospecimen core resource (BCR) located at Van Andel Research Institute, Grand Rapids, MI using a cryoport that maintained an average temperature of less than \\u2212140\\u00b0C. that maintained an average temperature of less than \\u2212140\\u00b0C. At the biospecimen core resource, specimens were confirmed for pathology qualification and prepared for genomic, transcriptomic, and proteomic analyses. Selected specimens were cryopulverized and proteomic analyses. Selected specimens were cryopulverized using a Covaris CryoPREP instrument and material aliquoted for subsequent molecular characterization. Genomic DNA and total RNA were extracted and sent to the genome sequencing centers. The were extracted and sent to the genome sequencing centers. The whole exome and whole genome DNA sequencing and methylation EPIC array analyses were performed at the Broad Institute, Cambridge, MA. TotalRNA and miRNA sequencing were performed at the MA. TotalRNA and miRNA sequencing were performed at the University of North Carolina, Chapel Hill, NC. Material for proteomic analyses were sent to the Proteomic Characterization Center (PCC) at the Broad Institute, Cambridge, MA. Sequencing sample preparation Our study sampled a single site of the primary tumor from surgical resections, with an internal requirement to process a minimum of 125mg of tumor issue and 50mg of NAT. DNA and RNA were extracted from tumor and NAT specimens in a co-isolation protocol from tumor and NAT specimens in a co-isolation protocol using Qiagen\\u2019s QIAsymphony DNA Mini Kit and QIAsymphony RNA Kit. Genomic DNA was also isolated from peripheral blood (3\\u20135mL) to serve as matched normal reference material. The Qubit\\u2122 dsDNA BR Assay as matched normal reference material. The Qubit\\u2122 dsDNA BR Assay Kit was used with the Qubit\\u00ae 2.0 Fluorimeter to determine the concentration of dsDNA in an aqueous solution. Any sample that passed quality control and produced enough DNA yield to go through quality control and produced enough DNA yield to go through the multiple planned genomic assays was sent for genomic characterization. RNA quality was quantified using the NanoDrop 8000 and quality assessed using an Agilent Bioanalyzer. A sample of and quality assessed using an Agilent Bioanalyzer. A sample of sufficient quantity that passed RNA quality control and had a minimum RIN (RNA integrity number) score of 7 was subjected to RNA sequencing. Identity matches for germline, normal adjacent RNA sequencing. Identity matches for germline, normal adjacent tissue, and tumor tissue were confirmed at the BCR using the Illumina Infinium QC array. This beadchip contains 15,949 markers designed to prioritize sample tracking, quality control, and designed to prioritize sample tracking, quality control, and stratification. Whole exome Sequencing (WES)\\n\\nLibrary construction and Hybrid Selection Library construction was performed as described in, with the following modifications: initial genomic DNA input into shearing was reduced from 3\\u03bcg to 20\\u2013250ng in 50\\u03bcL of solution. For adapter ligation, Illumina paired-end adapters were replaced with ligation, Illumina paired-end adapters were replaced with palindromic forked adapters, purchased from Integrated DNA Technologies (IDT), with unique dual-indexed molecular barcode sequences to facilitate downstream pooling. Kapa HyperPrep reagents in to facilitate downstream pooling. Kapa HyperPrep reagents in 96-reaction kit format were used for end repair/A-tailing, adapter ligation, and library enrichment PCR. In addition, during the post-enrichment SPRI cleanup, elution volume was reduced to 30\\u03bcL SPRI cleanup, elution volume was reduced to 30\\u03bcL to maximize library concentration, and a vortexing step was added to maximize the amount of template eluted. After library construction, libraries were pooled into groups of up to 96 samples. Hybridization were pooled into groups of up to 96 samples. Hybridization and capture were performed using the relevant components of Illumina\\u2019s Nextera Exome Kit and following the manufacturer\\u2019s suggested protocol, with the following exceptions: First, all libraries protocol, with the following exceptions: First, all libraries within a library construction plate were pooled prior to hybridization. Second, the Midi plate from Illumina\\u2019s Nextera Exome Kit was replaced with a skirted PCR plate to facilitate automation. was replaced with a skirted PCR plate to facilitate automation. All hybridization and capture steps were automated on the Agilent Bravo liquid handling system. Cluster Amplification and Sequencing After post-capture enrichment, library pools were quantified using qPCR (KAPA Biosystems) using an automated assay on the Agilent Bravo with probes specific to the ends of the adapters. Based on qPCR quantification, libraries were normalized to 2nM. Based on qPCR quantification, libraries were normalized to 2nM. Cluster amplification of DNA libraries was performed following manufacturer\\u2019s protocol (Illumina) using exclusion amplification chemistry and flowcells. Flow cells were sequenced utilizing chemistry and flowcells. Flow cells were sequenced utilizing sequencing-by-synthesis chemistry. The flow cells were then analyzed using RTA v.2.7.3 or later. Each pool of whole exome libraries was sequenced on paired 76-cycle runs with two 8-cycle index was sequenced on paired 76-cycle runs with two 8-cycle index reads across the number of lanes needed to meet coverage for all libraries in the pool. Pooled libraries were run on HiSeq4000 paired-end runs to achieve a minimum of 150x on-target coverage per runs to achieve a minimum of 150x on-target coverage per library. The raw Illumina sequence data were demultiplexed and converted to FASTQ files; adapter and low-quality sequences were trimmed. The raw reads were mapped to the GRCh38/hg38 human reference The raw reads were mapped to the GRCh38/hg38 human reference genome and the validated BAMs were used for downstream analysis and variant calling. Whole genome sequencing (WGS)\\n\\nCluster Amplification and Sequencing An aliquot of genomic DNA (350ng in 50\\u03bcL) was used as the input into DNA fragmentation (aka shearing). Shearing was performed acoustically using a Covaris focused-ultrasonicator, targeting 385bp fragments. Following fragmentation, additional size 385bp fragments. Following fragmentation, additional size selection was performed using SPRI cleanup. Library preparation was performed using a commercially available KAPA Hyper Prep without amplification module kit (KAPA Biosystems) and with palindromic amplification module kit (KAPA Biosystems) and with palindromic forked adapters with unique 8-base index sequences embedded within the adapter (IDT). Following sample preparation, libraries were quantified using quantitative PCR (KAPA Biosystems), with were quantified using quantitative PCR (KAPA Biosystems), with probes specific to the ends of the adapters using the automated Agilent\\u2019s Bravo liquid handling platform. Based on qPCR quantification, libraries were normalized to 1.7nM and pooled into libraries were normalized to 1.7nM and pooled into 24-plexes. Sample pools were combined with HiSeqX Cluster Amp Reagents EPX1, EPX2, and EPX3 into single wells on a strip tube using the Hamilton Starlet Liquid Handling system. Cluster amplification of the templates was performed according to the manufacturer\\u2019s of the templates was performed according to the manufacturer\\u2019s protocol (Illumina) with the Illumina cBot. Flow cells were sequenced to a minimum of 15x on HiSeqX utilizing sequencing-by-synthesis kits to produce 151bp paired-end reads. Output from kits to produce 151bp paired-end reads. Output from Illumina software was processed by the Picard data processing pipeline to yield BAM files containing demultiplexed, aggregated, aligned reads. All sample information tracking was performed by automated All sample information tracking was performed by automated LIMS messaging. Array Based Methylation Analysis The Methylation EPIC array uses an 8-sample version of the Illumina Beadchip capturing >850,000 methylation sites per sample. Two hundred and fifty nanograms of DNA was used for the bisulfite conversion using Infinium MethylationEPIC BeadChip Kit conversion using Infinium MethylationEPIC BeadChip Kit (Illumina). The EPIC array includes sample plating, bisulfite conversion, and methylation array processing. After scanning, the data was processed through an automated genotype-calling pipeline. Data processed through an automated genotype-calling pipeline. Data output consisted of raw idats and a sample sheet. RNA and miRNA sequencing\\n\\nQuality Assurance and Control of RNA Analytes All RNA analytes were assayed for RNA integrity, concentration, and fragment size. Samples for total RNA-seq were quantified on a TapeStation system (Agilent, Inc. Santa Clara, CA). Samples with RINs >7.0 were considered high quality and were considered with RINs >7.0 were considered high quality and were considered for sequencing. Total RNA-seq libraries were generated using 300 nanograms of total RNA using the TruSeq Stranded Total RNA Library Prep Kit with Ribo-Zero Gold and bar-coded with individual Prep Kit with Ribo-Zero Gold and bar-coded with individual tags following the manufacturer\\u2019s instructions (Illumina). Total RNA Libraries were prepared on an Agilent Bravo automated liquid handling system. Quality control was performed at every step, and system. Quality control was performed at every step, and the libraries were quantified using a TapeStation system. Total RNA Sequencing Indexed libraries were prepared and run on HiSeq4000 paired-end 75 base pairs to generate a minimum of 120 million reads per sample library with a target of greater than 90% mapped reads. The raw Illumina sequence data were demultiplexed and converted to raw Illumina sequence data were demultiplexed and converted to FASTQ files, and adapter and low-quality sequences were trimmed. Samples were then assessed for quality by mapping reads to GRCh38/hg38, estimating the total number of mapped reads, amount of estimating the total number of mapped reads, amount of RNA mapping to coding regions, amount of rRNA in the sample, number of genes expressed, and relative expression of housekeeping genes. Samples passing this QA/QC were then clustered with other Samples passing this QA/QC were then clustered with other expression data from similar and distinct tumor types to confirm expected expression patterns. Atypical samples were then SNP typed from the RNA data to confirm source analyte. FASTQ files of all from the RNA data to confirm source analyte. FASTQ files of all reads were then uploaded to the GDC repository. miRNA-seq Library Construction miRNA-seq library construction was performed from the RNA samples using the NEXTflex Small RNA-Seq Kit (v3, PerkinElmer, Waltham, MA) and barcoded with individual tags following the manufacturer\\u2019s instructions. Libraries were prepared on a Sciclone Liquid instructions. Libraries were prepared on a Sciclone Liquid Handling Workstation. Quality control was performed at every step, and the libraries were quantified using a TapeStation system and an Agilent Bioanalyzer using the Small RNA analysis kit. Pooled an Agilent Bioanalyzer using the Small RNA analysis kit. Pooled libraries were then size selected according to NEXTflex kit specifications using a Pippin Prep system (Sage Science, Beverly, MA). miRNA Sequencing Indexed libraries were loaded on the HiSeq4000 to generate a minimum of 10 million reads per library with a minimum of 90% reads mapped. The raw Illumina sequence data were demultiplexed and converted to FASTQ files for downstream analysis. Resultant data to FASTQ files for downstream analysis. Resultant data were analyzed using a variant of the small RNA quantification pipeline developed for TCGA (Chu et al., 2016). Data from samples were assessed for the number of miRNAs called, species diversity, and for the number of miRNAs called, species diversity, and total abundance before uploading to the GDC repository. Mass Spectrometry methods The protocols below for protein extraction, tryptic digestion, TMT-11 labeling of peptides, peptide fractionation by basic reversed-phase liquid chromatography, phosphopeptide enrichment using immobilized metal affinity chromatography, and LC-MS/MS were immobilized metal affinity chromatography, and LC-MS/MS were performed as previously described in depth. Acetyl-enrichment was performed as described before with modifications as indicated below. Protein Extraction and Tryptic Digestion Approximately fifty milligrams (wet weight) of cryopulverized human LSCC and NAT samples were homogenized in the lysis buffer for 30 minutes at a ratio of about 200 uL lysis buffer for every 50 mg wet weight tissue. The lysis buffer consisted of 8 M urea, mg wet weight tissue. The lysis buffer consisted of 8 M urea, 75 mM NaCl, 1 mM EDTA, 50 mM Tris HCl (pH 8), 10 mM NaF, phosphatase inhibitor cocktail 2 (1:100; Sigma, P5726) and cocktail 3 (1:100; Sigma, P0044), 2 \\u03bcg/mL aprotinin (Sigma, A6103), 10 \\u03bcg/mL Sigma, P0044), 2 \\u03bcg/mL aprotinin (Sigma, A6103), 10 \\u03bcg/mL leupeptin (Roche, 11017101001), and 1 mM PMSF (Sigma, 78830). Lysates were centrifuged at 20,000 g for 10 minutes and protein concentrations of the clarified lysates were measured by BCA assay of the clarified lysates were measured by BCA assay (Pierce). Protein lysates were subsequently reduced with 5 mM dithiothreitol (Thermo Scientific, 20291) for an hour at 37C and alkylated with 10 mM iodoacetamide (Sigma, A3221) for 45 minutes in the dark 10 mM iodoacetamide (Sigma, A3221) for 45 minutes in the dark at room temperature. Prior to digestion, samples were diluted 4-fold to achieve 2 M urea with 50mM Tris HCl (pH 8). Digestion was performed with LysC (Wako, 129\\u201302541) for 2 hours and with was performed with LysC (Wako, 129\\u201302541) for 2 hours and with trypsin (Promega, V511X) overnight, both at a 1:50 enzyme-to-protein ratio and at room temperature. Digested samples were acidified with formic acid (FA; Sigma, F0507) to achieve a final with formic acid (FA; Sigma, F0507) to achieve a final volumetric concentration of 1% (final pH of ~3), and centrifuged at 1,500 g for 15 minutes to clear precipitated urea from peptide lysates. Samples were desalted on tC18 SepPak columns (Waters, 200mg, Samples were desalted on tC18 SepPak columns (Waters, 200mg, WAT054925) and dried down using a SpeedVac apparatus. Common Reference (CR) Pool and Plex layout\\n\\nProteome, Phosphoproteome, and Acetyprotome: The proteomic, phosphoproteomic and acetylproteome analyses of LSCC samples consisted of 22 TMT-11 plex experiments. To facilitate quantitative comparison between all samples across experiments, a common reference (CR) sample was included in each 11-plex. a common reference (CR) sample was included in each 11-plex. A common physical, rather than in silico reference was used for this purpose for optimal quantitative precision between TMT-11 experiments. Considerations prior to creating the reference sample Considerations prior to creating the reference sample were that this sample needed to be of adequate quantity to cover all planned experiments for both the discovery and future confirmatory sets (not part of this study) with overhead for additional sets (not part of this study) with overhead for additional possible experiments. The CR included all (212/212) of the samples analyzed in the discovery set TMT experiments, yielding a CR that was representative of all the samples in the study. Making the was representative of all the samples in the study. Making the CR as representative of the study as a whole was particularly important since by definition only analytes represented in the reference sample would be included in the final ratio-based data sample would be included in the final ratio-based data analyses. 110 unique tumor samples with 102 paired NAT samples are distributed among 22 11-plex experiments, with 10 individual samples occupying the first 10 channels of each experiment and the 11th channel being reserved for the reference sample. All the tumors channel being reserved for the reference sample. All the tumors were in the C channels and all the normal samples were in the N channels. The first 8 channels of each experiment contained 4 tumor/normal pairs, with each pair of patient samples adjacent to pairs, with each pair of patient samples adjacent to each other. Channels 9 and 10 (130C and 131N) either contained a 5th tumor/normal pair, or an unpaired tumor sample in channel 9 and an alternate common reference sample in channel 10. These included 3 common reference sample in channel 10. These included 3 global LUAD common references, 3 HNSCC common references from the team at JHU, and 2 Tumor only common references from this LSCC discovery set. To ensure capacity for additional samples or experiments given a target input of 300 \\u03bcg peptide per channel per experiment, 18 mg total was targeted for reference material. To meet these collective requirements, all the samples in the discovery set that requirements, all the samples in the discovery set that had enough material were included in the CR. After reserving a conservative 400 \\u03bcg peptide / sample for individual sample analysis, an additional amount of 120 \\u03bcg for each of the samples with an additional amount of 120 \\u03bcg for each of the samples with adequate quantities were pooled. All 212 samples were selected for the combined tumor/normal CR. To make the combined CR, tumor only and normal only CRs were first created separately, with 110 and normal only CRs were first created separately, with 110 tumor samples and 102 normal samples. After creating tumor and NAT only CRs at 150 ug and 150ug amounts, a pool of combined CR was made, comprised of 8.6 mg tumor only reference and 8.6 mg normal comprised of 8.6 mg tumor only reference and 8.6 mg normal only reference. The 17.2 mg pooled reference material was divided into 55 300 \\u03bcg aliquots, dried down, and frozen at \\u221280\\u00b0C until use. Ubiquitylproteome: 750 ug of unlabeled peptides was required from each sample to ensure depth of coverage in the ubiquitylproteome, which restricted the number of samples available for profiling. 147/212 samples from the proteome, phosphoproteome, acetylproteome set were from the proteome, phosphoproteome, acetylproteome set were included in the ubiquitylproteome analysis. A separate ubiquitylproteome CR was constructed prior to KGG enrichment, due to limitations in sample material (both patient material and remaining CR in sample material (both patient material and remaining CR aliquots to be used in future LSCC proteome, phosphoproteome, and acetylproteome processing.. In order to make TMT ratio-based quantitation comparable across omes, the ubiquitylproteome CR was comparable across omes, the ubiquitylproteome CR was constructed to be as similar as practically possible to the proteome, phosphoproteome, acetylproteome CR. The ubiquitylproteome CR contains all samples from the proteome, phosphoproteome, acetylproteome all samples from the proteome, phosphoproteome, acetylproteome set with material remaining to contribute, 209/212. The ubiquitylproteome CR contributions were of 3 types: 104/147 samples included in ubiquitylproteome study and made full CR contribution, >1000ug available.\\n\\n42/147 samples included in ubiquitylproteome study and made partial CR contribution, 750ug \\u22121000 ug available. 63/65 samples from the proteome, phosphoproteome, acetylproteome set not included in ubiquitylproteome study, but made full CR contribution, <750 ug available. CR tumor and normal peptides were combined at a ratio of 55:45 to reflect the higher proportion of tumor samples in the experimental design (as opposed to proteome, phosphoproteome, acetylproteome CR\\u2019s proportion of 50:50), and aliquoted into 750 ug CR\\u2019s proportion of 50:50), and aliquoted into 750 ug aliquots. In the ubiquitylproteome experimental design, 15 11-plex experiments contained 87 unique tumor samples in the C channels and 60 normal samples in the N channels, with the first 8 channels of every 11-plex consisting of patient-paired tumor/normal samples every 11-plex consisting of patient-paired tumor/normal samples adjacent to each other. The 11th channel was reserved for the ubiquitylproteome common reference sample. Channel 9 (130C) contained an unpaired tumor, while channel 10 (131N) contained either an unpaired tumor, while channel 10 (131N) contained either an unpaired tumor, or in 3 plexes, a LUAD ubiquitylproteome common reference (which was constructed similarly for that experiment, unpublished). TMT-11 Labeling of Peptides\\n\\nProteome, Phosphoproteome, Acetylproteome: 300 \\u03bcg of desalted peptides per sample (based on peptide-level BCA after digestion) were labeled with 11-plex TMT reagents and combined for multiplexed analysis using a \\u201creduced labeling\\u201d approach (Zecha et al., 2019). To conservatively yield extra approach (Zecha et al., 2019). To conservatively yield extra labeled peptide in case a given sample demonstrated unsuitably low TMT reporter ion intensity during channel mixing control LC-MS/MS runs, 400 ug peptides per sample were initially labeled, with runs, 400 ug peptides per sample were initially labeled, with only 300 ug combined per sample for the final multiplexed experiments upon successful quality control. For each 400 \\u03bcg peptide aliquot of an individual sample, peptides were reconstituted to 5 of an individual sample, peptides were reconstituted to 5 mg/ml with 50 mM HEPES (pH 8.5) solution and 400 ug labeling reagent dissolved in 20 uL anhydrous acetonitrile was added. After 1 h incubation with shaking, samples were diluted 1:1 with a solution with shaking, samples were diluted 1:1 with a solution of 80% 50 mM HEPES and 20% acetonitrile. After confirming good label incorporation, 5 \\u03bcL of 5% hydroxylamine was added to quench the unreacted TMT reagents. Good label incorporation was defined as unreacted TMT reagents. Good label incorporation was defined as having less than 3% isobaric under-labeled MS/MS spectra in each sample, as measured by LC-MS/MS after taking out a 2 \\u03bcg aliquot from each sample and analyzing 0.25 \\u03bcg. If a sample did not from each sample and analyzing 0.25 \\u03bcg. If a sample did not have sufficient label incorporation, additional TMT was added to the sample and another 1 h incubation was performed with shaking. At the time that the labeling efficiency quality control samples the time that the labeling efficiency quality control samples were taken out, an additional 4 \\u03bcg of material from each sample was taken out and combined as a mixing control. After analyzing the mixing control sample by LC-MS/MS, intensity values of the the mixing control sample by LC-MS/MS, intensity values of the individual TMT reporter ions were summed across all peptide spectrum matches and compared to ensure that the total reporter ion intensity of each sample met a threshold of +/\\u2212 15% of the ion intensity of each sample met a threshold of +/\\u2212 15% of the internal reference. If necessary, adjustments were made by either using additional labeled material or reducing an individual sample\\u2019s contribution to the mixture, and analyzing a subsequent contribution to the mixture, and analyzing a subsequent mixing control, until all samples met the threshold and were thus approximately 1:1:1.... Differentially labeled peptides were then mixed (11 \\u00d7 300 \\u03bcg), dried down via vacuum centrifuge, and the mixed (11 \\u00d7 300 \\u03bcg), dried down via vacuum centrifuge, and the quenched, combined sample was subsequently desalted on a 200 mg tC18 SepPak column. The same lot of TMT reagents was used for all samples (UA280170/TL272399). Peptide Fractionation\\n\\nFor Proteome, Phosphoproteome, Acetylproteome: To reduce sample complexity, peptide samples were separated by high pH reversed phase (RP) separation as described previously (Mertins et al., 2018). A desalted 3.3 mg, 11-plex TMT-labeled experiment (based on peptide-level BCA after digestion) was experiment (based on peptide-level BCA after digestion) was reconstituted in 935 \\u03bcL 4.5mM ammonium formate (pH 10) and 2% acetonitrile, centrifuged for 7 min at 20000 g to precipitate insoluble peptides, loaded on a 4.6 mm x 250 mm column RP Zorbax 300 A peptides, loaded on a 4.6 mm x 250 mm column RP Zorbax 300 A Extend-C18 column (Agilent, 3.5 \\u03bcm bead size), and separated on an Agilent 1260 Series HPLC instrument using basic reversed-phase chromatography. Solvent A (2% acetonitrile, 4.5 mM ammonium chromatography. Solvent A (2% acetonitrile, 4.5 mM ammonium formate, pH 10) and a nonlinear increasing concentration of solvent B (90% acetonitrile, 4.5 mM ammonium formate, pH 10) were used to separate peptides. The 4.5 mM ammonium formate solvents were to separate peptides. The 4.5 mM ammonium formate solvents were made by 40\\u2013fold dilution of a stock solution of 180 mM ammonium formate, pH 10. To make 1L of stock solution, add 25 mL of 28% (wt/vol) ammonium hydroxide (28%, density 0.9 g/ml, mL of 28% (wt/vol) ammonium hydroxide (28%, density 0.9 g/ml, Sigma-Aldrich) to ~850ml of HPLC grade water, then add ~35 mL of 10 % (vol/vol) formic acid (>95% Sigma-Aldrich) to titrate the pH to 10.0; bring the final volume to 1 liter with HPLC grade pH to 10.0; bring the final volume to 1 liter with HPLC grade water. The 96 minute separation LC gradient followed this profile: (min: %B) 0:0; 7:0; 13:16; 73:40; 77:44; 82:60; 96:60. The flow rate was 1 mL/min. Per 3.3 mg separation, 82 fractions were rate was 1 mL/min. Per 3.3 mg separation, 82 fractions were collected into a 96 deep-well x 2mL plate (Whatman, #7701\\u20135200), with fractions combined in a step-wise concatenation strategy and acidified to a final concentration of 0.1% FA as reported and acidified to a final concentration of 0.1% FA as reported previously. An additional 14 fractions were collected from the 96 deep-well plate for fraction A, which are the early eluting fractions that tend to contain multi-phosphorylated peptides. 5% of that tend to contain multi-phosphorylated peptides. 5% of the volume of each of the 24+A proteome fractions was allocated for proteome analysis, dried down, and re-suspended in 3% MeCN/0.1% FA (MeCN; acetonitrile) to a peptide concentration of 0.25 \\u03bcg/ \\u03bcL (MeCN; acetonitrile) to a peptide concentration of 0.25 \\u03bcg/ \\u03bcL for LC-MS/MS analysis. The remaining 95% of concatenated 24 fractions were further combined into 12 fractions, with fraction A as a separate fraction. These 13 fractions were then enriched for a separate fraction. These 13 fractions were then enriched for phosphopeptides as described below. Phosphopeptide Enrichment Ni-NTA agarose beads were used to prepare Fe3+-NTA agarose beads. In each phosphoproteome fraction, ~261.25 \\u03bcg peptides (based on peptide-level BCA after digestion with uniformly-distributed fractionation presumed) were reconstituted in 522.5 \\u03bcL 80% fractionation presumed) were reconstituted in 522.5 \\u03bcL 80% MeCN/0.1% TFA (trifluoroacetic acid) solvent and incubated with 10 \\u03bcL of the IMAC beads for 30 minutes with end-over-end rotation at RT. After incubation, samples were briefly spun down on a at RT. After incubation, samples were briefly spun down on a tabletop centrifuge; clarified peptide flow-throughs were separated from the beads; and the beads were reconstituted in 200 \\u03bcL IMAC binding/wash buffer (80 MeCN/0.1% TFA) and loaded onto \\u03bcL IMAC binding/wash buffer (80 MeCN/0.1% TFA) and loaded onto equilibrated Empore C18 silica-packed stage tips (3M, 2315). Samples were then washed twice with 50 \\u03bcL of IMAC binding/wash buffer and once with 50 uL 1% FA, and were eluted from the IMAC buffer and once with 50 uL 1% FA, and were eluted from the IMAC beads to the stage tips with 3 \\u00d7 70 uL washes of 500 mM dibasic sodium phosphate (pH 7.0, Sigma S9763). Stage tips were then washed once with 100 \\u03bcL 1% FA and phosphopeptides were eluted from once with 100 \\u03bcL 1% FA and phosphopeptides were eluted from the stage tips with 60 \\u03bcL 50% MeCN/0.1% FA. Phosphopeptides were dried down and re-suspended in 9 \\u03bcL 50% MeCN/0.1%FA for LC-MS/MS analysis, where 4 \\u03bcL was injected per run. Acetylpeptide Enrichment Acetylated lysine peptides were enriched using an antibody against the acetyl-lysine motif (CST PTM-SCAN Catalogue No. 13416). IMAC eluents were concatenated into 4 fractions (~783.75 \\u03bcg peptides per fraction) and dried down using a SpeedVac apparatus. per fraction) and dried down using a SpeedVac apparatus. Peptide fractions were reconstituted with 1.4ml of IAP buffer (5 mM MOPS pH 7.2, 1 mM Sodium Phosphate (dibasic), 5 mM NaCl) per fraction and incubated for 2 hours at 4oC with pre-washed (3 times and incubated for 2 hours at 4oC with pre-washed (3 times with IAP buffer) agarose beads bound to acetyl-lysine motif antibody. Peptide-bound beads were washed 4 times with ice-cold PBS followed by elution with 2 \\u00d7 100ul of 0.15% TFA. Eluents were followed by elution with 2 \\u00d7 100ul of 0.15% TFA. Eluents were desalted using C18 stage tips, eluted with 50% ACN and dried down. Acetylpeptides were suspended in 7ul of 0.1% FA and 3% ACN and 4ul were injected per run. Ubiquitylpeptide Enrichment with on-bead TMT labeling Ubiquitin enrichment was performed based on the recently published UbiFast protocol. Before enrichment, anti-K-\\u03b5-GG bead-bound antibodies from the PTM-Scan ubiquitin remnant motif kit (Cell Signaling Technologies, Kit #5562) were cross-linked. Beads were Technologies, Kit #5562) were cross-linked. Beads were briefly washed 3X with 100 mM sodium borate (pH 9.0) and incubated with 20 mM DMP for 30 min at RT. Beads were then washed 2X with 200 mM ethanolamine and incubated overnight at 4\\u00b0C in 200 mM 200 mM ethanolamine and incubated overnight at 4\\u00b0C in 200 mM ethanolamine with end-over-end rotation. Following incubation, beads were washed 3X with immunoprecipitation (IAP) buffer (50 mM MOPS, pH 7.2, 10 mM sodium phosphate, 50 mM NaCl) and stored at MOPS, pH 7.2, 10 mM sodium phosphate, 50 mM NaCl) and stored at 4\\u00b0C at a concentration of 0.5 ug/uL. For each 11-plex experiment, 31.25 ug of cross-linked anti-K-GG bead-bound antibody at 0.5 ug/uL in IAP per channel was aliquoted into 1.5 mL Eppendorf tubes on ice. 750 ug peptide samples were reconstituted to 0.5 mg/mL concentration in IAP buffer and were reconstituted to 0.5 mg/mL concentration in IAP buffer and vortexed for 10 min. Peptides were then centrifuged for 5 min at 5,000g. Each peptide solution was added to a tube of antibody and gently rotated end-over-end at 4oC for 1 hour. Following and gently rotated end-over-end at 4oC for 1 hour. Following enrichment, samples were spun at 2000 rcf for ~1 min to settle and the supernatant was removed and stored as flow-through, and from this point beads were kept on ice unless being handled. from this point beads were kept on ice unless being handled. Antibody beads were washed with 1.5 mL ice cold IAP followed by 1.5 mL ice cold PBS. For all washes, after adding wash reagent each sample was inverted ~5 times, agitated by tapping pairs of each sample was inverted ~5 times, agitated by tapping pairs of tubes together, spun at 2000 rcf for about 30\\u201340 s, left to settle on ice for 10\\u201320 s, and supernatant was removed. Importantly, all washes were completed as quickly as possible. Importantly, all washes were completed as quickly as possible. Pre-aliquoted TMT @ 40 ug/uL in 100% MeCN was retrieved and briefly equilibrated to RT while beads were reconstituted in 200 uL room temp. 100 mM HEPES buffer. For each sample, 400 ug of TMT room temp. 100 mM HEPES buffer. For each sample, 400 ug of TMT labeling reagent in 10 uL acetonitrile was added. Samples were labeled on-antibody while shaking vigorously (1400 rpm) at 20\\u00b0C for 10 minutes, then quenched with 8 uL 5% hydroxylamine and for 10 minutes, then quenched with 8 uL 5% hydroxylamine and shaken vigorously for another 5 minutes. Labeled antibody-bound peptides were then put back on ice, washed once with 1.3 mL cold IAP, and again with 1.5 mL cold IAP. Each channel was resuspended and again with 1.5 mL cold IAP. Each channel was resuspended and transferred to a combination tube with 130 uL cold IAP. Following combination, each now-empty tube was serially washed with 1.5 mL cold IAP to remove remaining beads, and this 1.5 mL IAP was mL cold IAP to remove remaining beads, and this 1.5 mL IAP was added to the combination tube and used to wash the combined beads. Combined beads were washed one final time with 1.5 mL ice cold PBS. Once the channels were combined and washed, peptides were PBS. Once the channels were combined and washed, peptides were eluted twice from the beads by resuspending with 150 uL room temp. 0.15% TFA and incubating 5 min at RT. Each round of acid-eluted KGG-modified peptides was desalted on an equilibrated KGG-modified peptides was desalted on an equilibrated two-punch C18 stage tip. Both elutions of KGG peptides were loaded sequentially, washed 2X with 100 uL 0.1% FA, and eluted into an MS vial with 50 uL 50% ACN/0.1% FA. The eluted peptides were frozen, with 50 uL 50% ACN/0.1% FA. The eluted peptides were frozen, lyophilized, and reconstituted in 9 uL 3% ACN/0.1% FA, with 4 uL injected twice for two LC-MS/MS runs on a Thermo Lumos instrument. While the same lot of TMT reagents was used for all instrument. While the same lot of TMT reagents was used for all ubiquitylproteome samples (TE270748-TD264064), it is a different lot than that used for Proteome, Phosphoproteome, and acetylproteome samples. LC-MS/MS for Proteomics Analyses Online separation was done with a nanoflow Proxeon EASY-nLC 1200 UHPLC system (Thermo Fisher Scientific). In this set up, the LC system, column, and platinum wire used to deliver electrospray source voltage were connected via a stainless steel cross source voltage were connected via a stainless steel cross (360\\u03bcm, IDEX Health & Science, UH-906x). The column was heated to 50\\u00b0C using a column heater sleeve (Phoenix-ST) to prevent over-pressuring of columns during UHPLC separation. From each peptide of columns during UHPLC separation. From each peptide fraction, ~1ug (based on protein-level BCA prior to digestion with uniformly-distributed fractionation presumed), the equivalent of 12% of each global proteome sample in a 2 ul injection volume or 50% each global proteome sample in a 2 ul injection volume or 50% of each phosphoproteome, acetylproteome, or ubiquitylproteome sample in a 4 ul injection volume, was injected onto an in-house packed 22cm x 75um internal diameter C18 silica picofrit capillary 22cm x 75um internal diameter C18 silica picofrit capillary column (1.9 \\u03bcm ReproSil-Pur C18-AQ beads, Dr. Maisch GmbH, r119.aq; Picofrit 10um tip opening, New Objective, PF360\\u201375-10-N-5). Mobile phase flow rate was 200 nL/min, comprised of 3% Mobile phase flow rate was 200 nL/min, comprised of 3% acetonitrile/0.1% formic acid (Solvent A) and 90% acetonitrile /0.1% formic acid (Solvent B). The 110-minute LC-MS/MS method consisted of a 10-min column-equilibration procedure; a 20-min consisted of a 10-min column-equilibration procedure; a 20-min sample-loading procedure; and the following gradient profile: (min:%B) 0:2; 1:6; 85:30; 94:60; 95;90; 100:90; 101:50; 110:50 (the last two steps at 500 nL/min flow rate). For acetylproteome last two steps at 500 nL/min flow rate). For acetylproteome analysis, the same LC and column setup was used, but the gradient was extended to 260 minutes with the following gradient profile: (min:%B) 0:2; 1:6; 235:30; 244:60; 245;90; 250:90; 251:50; (min:%B) 0:2; 1:6; 235:30; 244:60; 245;90; 250:90; 251:50; 260:50 (the last two steps at 500 nL/min flow rate). For ubiquitylproteome analysis, the same LC and column setup was used, but the gradient was 154-minutes with the following gradient profile: gradient was 154-minutes with the following gradient profile: (min:%B) 0:2; 2:6; 122:35; 130:60; 133;90; 143:90; 144:50; 154:50 (the last two steps at 500 nL/min flow rate). For proteome and acetylproteome analysis, samples were analyzed with a benchtop Q Exactive HF-X mass spectrometer (Thermo Fisher Scientific) equipped with a nanoflow ionization source (James A. Hill Instrument Services, Arlington, MA). Data-dependent A. Hill Instrument Services, Arlington, MA). Data-dependent acquisition was performed using Q Exactive HF-X Orbitrap v 2.9 software in positive ion mode at a spray voltage of 1.5 kV. MS1 Spectra were measured with a resolution of 60,000, an AGC target of were measured with a resolution of 60,000, an AGC target of 3e6 and a mass range from 350 to 1800 m/z. The data-dependent mode cycle was set to trigger MS/MS on up to the top 20 most abundant precursors per cycle at an MS2 resolution of 45,000, an AGC precursors per cycle at an MS2 resolution of 45,000, an AGC target of 5e4, an isolation window of 0.7 m/z, a maximum injection time of 105 msec, and an HCD collision energy of 31%. Peptides that triggered MS/MS scans were dynamically excluded from further triggered MS/MS scans were dynamically excluded from further MS/MS scans for 45 sec for proteome or 30 sec for acetylproteome. Peptide match was set to preferred for monoisotopic peak determination, and charge state screening was enabled to only include and charge state screening was enabled to only include precursor charge states 2\\u20136, with an intensity threshold of 9.5e4 for proteome or 8e4 for acetylproteome. Advanced precursor determination feature (APD) (Myers et al., 2018) was turned off using a feature (APD) (Myers et al., 2018) was turned off using a software patch provided to us by Thermo Fisher Scientific allowing us to turn APD off in the tune file, Tune version 2.9.0.2926 (later versions of Exactive Tune 2.9 sp2 for the HFX have this option versions of Exactive Tune 2.9 sp2 for the HFX have this option as standard). For phosphoproteome and ubiquitylproteome analysis, samples were analyzed with a benchtop Orbitrap Fusion Lumos mass spectrometer (Thermo Fisher Scientific) equipped with a NanoSpray Flex NG ion source. Data-dependent acquisition was performed using NG ion source. Data-dependent acquisition was performed using Xcalibur Orbitrap Fusion Lumos v3.0 software in positive ion mode at a spray voltage of 1.8 kV. MS1 Spectra were measured with a resolution of 60,000, an AGC target of 4e5 and a mass range from of 60,000, an AGC target of 4e5 and a mass range from 350 to 1800 m/z. The data-dependent mode cycle time was set at 2 seconds with a MS2 resolution of 50,000, an AGC target of 6e4, an isolation window of 0.7 m/z, a maximum injection time of 105 msec (125 window of 0.7 m/z, a maximum injection time of 105 msec (125 msec for ubiquitylproteome), and an HCD collision energy of 36%. Peptide mode was selected for monoisotopic peak determination, and charge state screening was enabled to only include precursor charge state screening was enabled to only include precursor charge states 2\\u20136, with an intensity threshold of 1e4. Peptides that triggered MS/MS scans were dynamically excluded from further MS/MS scans for 45 sec (20 sec for ubiquitylproteome), with a MS/MS scans for 45 sec (20 sec for ubiquitylproteome), with a +/\\u2212 10 ppm mass tolerance. \\u201cPerform dependent scan on single charge state per precursor only\\u201d was enabled for phosphoproteome analysis, but disabled for ubiquitylproteome analysis. Immunohistochemistry (IHC) analysis.\\n\\nTMA construction Seventy-one LSCC paraffin tissue blocks were available for tissue microarray (TMA) construction and whole sections by immunohistochemistry (IHC) assay. TMAs were constructed in duplicate, and each included two (2) 1.0 mm cores providing up to four (4) and each included two (2) 1.0 mm cores providing up to four (4) cores per case for IHC analysis. Common control tissue on each TMA included patient-matched normal adjacent lung from 14 cases. Immune cell marker, Immunohistochemistry (IHC) Seventeen (17) samples were scored for immune cell markers. Sequential TMA sections were obtained for immune marker assessment to best approximate native spatial relationships. Chromogenic (DAB) IHC by single marker analysis was performed on 5-micron (DAB) IHC by single marker analysis was performed on 5-micron thick sections on standard charged slides. Automated immunostaining was performed by Autostainer Link 48, Dako, Inc. Heat induced epitope retrieval (HIER) was performed and biomarker dilutions epitope retrieval (HIER) was performed and biomarker dilutions as follows: CD4 clone 4B12, Bio-Rad 1:50; CD8 clone 4B11, Bio-Rad, 1:50; CD163, Abcam, 1:500 and ARHGDIB, HPA051235 Sigma-Aldrich, 1:50. IHC analysis was performed independently by three 1:50. IHC analysis was performed independently by three independent pathologists by staining intensity of 0\\u20133 scale for each TMA core separately and then by average score / case for a total of 15 cases. Case scores compared across independent pathologists 15 cases. Case scores compared across independent pathologists showed very good agreement. For evaluating statistical significance of IHC scores across the three immune clusters, ordinal regression was used, implemented using the \\u201cordinal\\u201d package in R. was used, implemented using the \\u201cordinal\\u201d package in R. Random effects accounted for the correlation between multiple cores from each sample and multiple pathologists. Multiplex Immunofluorescence (mIF) staining Development and optimization of the mIF platform has been previously described. mIF staining was performed in a 4-\\u03bcm-thick section obtained from a FFPE TMA block, using the Opal 7-Color Kit (Akoya Biosciences, Waltham, MA, USA) and scanned using a Vectra Biosciences, Waltham, MA, USA) and scanned using a Vectra multispectral microscope (Akoya Biosciences). The IF markers used were pan-Cytokeratin (CK, clone AE1/AE3, (DAKO) and Alpha-SMA ab5694 (Abcam). Multiplexed stained sections were imaged using the (Abcam). Multiplexed stained sections were imaged using the VECTRA multispectral imaging system (Akoya Biosciences). TMA cores were annotated using InForm 2.4.8 image analysis software (Akoya Biosciences). IHC based assessment of TP63 low samples (Figure 4) For samples with low TP63 status, additional pathologic confirmation was performed. Pathology reassessment confirmed 8/10 samples to be unequivocal LSCC; one sample (C3N.00247) had adenosquamous but predominantly squamous histology; the remaining sample but predominantly squamous histology; the remaining sample (C3N.02283) showed basaloid squamous cell features and despite some atypical characteristics retained its LSCC attribution after review by four separate pathologists - with histology very similar by four separate pathologists - with histology very similar to a TP63-low TCGA tumor (TCGA-66\\u20132754) deemed LSCC by TCGA pathology. QUANTIFICATION AND STATISTICAL ANALYSIS\\n\\nGenomic Data Analysis\\n\\nCopy Number Calling Copy-number analysis was performed jointly leveraging both whole-genome sequencing (WGS) and whole-exome sequencing (WES) data of the tumor and germline DNA, using CNVEX (https://github.com/mctp/cnvex). CNVEX uses whole-genome aligned reads to estimate CNVEX uses whole-genome aligned reads to estimate coverage within fixed genomic intervals, and whole-genome and whole-exome variant calls to compute B-allele frequencies at variable positions (we used TNScope germline calls). Coverages were computed in (we used TNScope germline calls). Coverages were computed in 10kb bins, and the resulting log coverage ratios between tumor and normal samples were adjusted for GC bias using weighted LOESS smoothing across mappable and non-blacklisted genomic intervals smoothing across mappable and non-blacklisted genomic intervals within the GC range 0.3\\u20130.7, with a span of 0.5 (the target, blacklist, and configuration files are provided with CNVEX). The adjusted log coverage ratios (LR) and B-allele frequencies (BAF) log coverage ratios (LR) and B-allele frequencies (BAF) were jointly segmented by custom algorithm based on Circular Binary Segmentation (CBS). Alternative probabilistic algorithms were implemented in CNVEX, including algorithms based on recursive binary in CNVEX, including algorithms based on recursive binary segmentation (RBS), and dynamic programming, as implemented in the R-package jointseg. For the CBS-based algorithm, first LR and mirrored BAF were independently segmented using CBS (parameters mirrored BAF were independently segmented using CBS (parameters alpha=0.01, trim=0.025) and all candidate breakpoints collected. The resulting segmentation track was iteratively \\u201cpruned\\u201d by merging segments that had similar LR, BAFs and short lengths. For segments that had similar LR, BAFs and short lengths. For the RBS- and DP-based algorithms, joint-breakpoints were \\u201cpruned\\u201d using a statistical model selection method. For the final set of CNA segments, we chose the CBS-based results as they did not of CNA segments, we chose the CBS-based results as they did not require specifying a prior on the number of expected segments (K) per chromosome arm, were robust to unequal variances between the LR and BAF tracks, and provided empirically the best fit to the LR and BAF tracks, and provided empirically the best fit to the underlying data. Somatic Variant Calling Somatic variant calling was performed on exome-capture BAM files aligned to the GDC GRCh38 reference. Following de-duplication and co-realignment around known indels \\u201cGATK bundle known\\u201d and \\u201cMills 1000G\\u201d somatic variants were called using Sentieon TNScope 1000G\\u201d somatic variants were called using Sentieon TNScope with the following settings \\u201c--max_fisher_pv_active 0.05 --min_tumor_allele_frac 0.0075 --min_init_tumor_lod 2.5 --assemble_mode 4 --SVIntrusionThres 5\\u201d, with a provided germline variants from 4 --SVIntrusionThres 5\\u201d, with a provided germline variants from dbSNP138. The resulting VCF files were annotated and filtered, in a series of post-processing steps which included: 1) restriction of variants to the hybrid-capture target regions. 2) Phasing of variants to the hybrid-capture target regions. 2) Phasing of adjacent and separated by 1bp SNVs into multi-nucleotide variants based on the haplotype information provided by TNScope. 3) Annotation using VEP with the following settings \\u201c--assembly 3) Annotation using VEP with the following settings \\u201c--assembly GRCh38 --species homo_sapiens --cache_version 97 --format vcf --gene_phenotype --symbol --canonical --ccds --hgvs --biotype --tsl --uniprot --domains --appris --protein --variant_class --sift --uniprot --domains --appris --protein --variant_class --sift b, --polyphen b --no_stats --total_length --allele_number --no_escape --flag_pick_allele --pick_order canonical,tsl,biotype,rank,ccds,length --buffer_size 20000\\u201d. Additional annotations were --buffer_size 20000\\u201d. Additional annotations were attached to the variants using VcfAnno including allele frequencies in GNOMAD (v2.0.1), ClinVar (downloaded Dec. 2019), Cosmic (downloaded Dec. 2019), dbSNP (20180418). The resulting variants are annotated 2019), dbSNP (20180418). The resulting variants are annotated against both RefSeq and Ensembl transcripts and contain multiple predicted consequences per-variant. For most downstream applications these annotations were reduced to a single predicted these annotations were reduced to a single predicted consequence per-variant and converted into a MAF file using custom software. The choice of the most-relevant consequence was based on a number of features. In general, the most \\u2018severe\\u2019 consequence a number of features. In general, the most \\u2018severe\\u2019 consequence occuring in a \\u2018reliable\\u2019 transcript was chosen. To accomplish this the computed consequences were ranked by \\u2018severity\\u2019 and \\u2018priority\\u2019 reflecting how deleterious/impactful a variant and \\u2018priority\\u2019 reflecting how deleterious/impactful a variant consequence is and how well-supported the existence and biological relevance of an isoform is. The specific rules differed for RefSeq and Ensembl annotations. RefSeq transcripts were ranked RefSeq and Ensembl annotations. RefSeq transcripts were ranked higher if they were known to CCDS, LRG or ranked \\u2018CANONICAL\\u2019 or \\u2018PICK\\u2019 by VEP. Ensembl transcripts were further prioritized by Basic, Appris and TSL annotations. Ensembl-based consequences Basic, Appris and TSL annotations. Ensembl-based consequences were prefered in case of ties. The resulting variant sets were filtered to only include variants with a high likelihood of being somatic based on a set of probabilistic and heuristic of being somatic based on a set of probabilistic and heuristic thresholds. A reported variant had to pass all of the following filters: 5 or more variant reads, fewer than 2 variant reads in the germline allele frequency in the normal 0.005, allele in the germline allele frequency in the normal 0.005, allele frequency in the tumor greater than 0.025, TLOD > 5, NLOD > 6, Fisher\\u2019s test p-value < 0.05, NFLOD > 1.5. Additionally heuristic filters were implemented to remove strand-bias, variants with filters were implemented to remove strand-bias, variants with mapping quality bias, and variants with high population allele frequency in GNOMAD. Germline Variant Calling Germline variant calling and annotation was carried out analogously using DNAScope using the following settings \\u201c--emit_conf 10 --call_conf 10\\u201d to generate a VCF file and \\u201c--emit_mode GVCF\\u201d to generate a gVCF file. The variants were custom-filtered based generate a gVCF file. The variants were custom-filtered based on the specific applications. GISTIC and MutSig analysis The Genomic Identification of Significant Targets in Cancer (GISTIC2.0) algorithm was used to identify significantly amplified or deleted focal-level and arm-level events, with Q value <0.25 considered significant. The following parameters were used: Amplification Threshold = 0.1\\n\\nDeletion Threshold = 0.1\\n\\nCap Values = 1.5\\n\\nBroad Length Cutoff = 0.98\\n\\nRemove X-Chromosome = 0\\n\\nConfidence Level = 0.99\\n\\nJoin Segment Size = 4\\n\\nArm Level Peel Off = 1\\n\\nMaximum Sample Segments = 2000\\n\\nGene GISTIC = 1 Each gene of every sample is assigned a thresholded copy number level that reflects the magnitude of its deletion or amplification. These are integer values ranging from \\u22122 to 2, where 0 means no amplification or deletion of magnitude greater than the no amplification or deletion of magnitude greater than the threshold parameters described above. Amplifications are represented by positive numbers: 1 means amplification above the amplification threshold; 2 means amplification larger than the arm level threshold; 2 means amplification larger than the arm level amplifications observed in the sample. Deletions are represented by negative numbers: \\u22121 means deletion beyond the threshold; \\u22122 means deletions greater than the minimum arm-level copy number means deletions greater than the minimum arm-level copy number observed in the sample. The somatic variants were filtered through a panel of normals to remove potential sequencing artifacts and undetected germline variants. MutSig2CV was run on these filtered results to evaluate the significance of mutated genes and estimate mutation the significance of mutated genes and estimate mutation densities of samples. These results were constrained to genes in the Cancer Gene Census, with false discovery rates (q values) recalculated. Genes of q value < 0.1 were declared significant. RNAseq and miRNAseq Quantification\\n\\nRNAseq Quantification The readcount is generated by featureCounts (subread v1.6.4) under stranded mode with parameters: `-g gene_id -t exon -Q 10 -p -B -s 2`. Gene annotation in use is identical to GDC (`gencode.v22.annotation.gtf.gz`; md5: `291330bdcff1094bc4d5645de35e0871`), md5: `291330bdcff1094bc4d5645de35e0871`), which is available on the GDC Reference Files page (https://gdc.cancer.gov/about-data/gdc-data-processing/gdc-reference-files). The readcount is later converted to FPKM and FPKM-UQ using GDC\\u2019s formula is later converted to FPKM and FPKM-UQ using GDC\\u2019s formula (https://docs.gdc.cancer.gov/Data/Bioinformatics_Pipelines/Expression_mRNA_Pipeline/). Isoform specific RNA quantification For TP63 isoform centric analysis presented in Figure 4, the hg38 reference genome and RefSeq annotations were used for the RNAseq data analysis and were downloaded from the UCSC table browser. First, CIRI (v2.0.6) was used to call circular RNA with First, CIRI (v2.0.6) was used to call circular RNA with default parameters and BWA (version 0.7.17-r1188) was used as the mapping tool. The cutoff of supporting reads for circRNAs was set to 10. Then we used a pseudo-linear transcript strategy to quantify Then we used a pseudo-linear transcript strategy to quantify gene and circular RNA expression. In brief, for each sample, linear transcripts of circular RNAs were extracted and 75bp (read length) from the 3\\u2019 end was copied to the 5\\u2019 end. The modified length) from the 3\\u2019 end was copied to the 5\\u2019 end. The modified transcripts were called pseudo-linear transcripts. Transcripts of linear genes were also extracted and mixed with pseudo-linear transcripts. RSEM (version 1.3.1) with Bowtie2 (version 2.3.3) transcripts. RSEM (version 1.3.1) with Bowtie2 (version 2.3.3) as the mapping tool was used to quantify gene, isoform, and circular RNA expression based on the mixed transcripts. DNA methylation data preprocessing Raw methylation image files were downloaded from the CPTAC DCC (See data availability). We calculated and analyzed methylated (M) and unmethylated (U) intensities for LSCC samples as described previously. We flagged locus as NA where probes did not meet a previously. We flagged locus as NA where probes did not meet a detection p-value of 0.01. Probes with MAF more than 0.1 were removed, and samples with more than 85% NA values were removed. Resulting beta values of methylation were utilized for subsequent beta values of methylation were utilized for subsequent analysis. Gene-level methylation scores were generated by taking the mean beta values of probes in the CpG islands of promoters and 5\\u2019 UTR regions of the gene. Methylation profiles (i.e., density plots) of some samples had unexpectedly skewed distributions of plots) of some samples had unexpectedly skewed distributions of methylation beta values, in addition to significantly more missing values. To systematically determine the subset of methylation samples with these evident data QC issues, we subjected all samples with these evident data QC issues, we subjected all the samples to model-based clustering using the Mclust package in R, using the interquartile range over all the genes as the representative metric. The clustering automatically determined the metric. The clustering automatically determined the optimal number of clusters, and identified 3 clusters. Two of these clusters (with centroids at 0.029 and 0.038) captured the bulk of the samples (202). The third cluster (centroid at 0.165, of the samples (202). The third cluster (centroid at 0.165, significantly higher than the other two clusters) consisted of 4 samples that belong to this cluster with high confidence (uncertainty < 5%), each of which had a skewed distribution of beta < 5%), each of which had a skewed distribution of beta values with a mean of 4,407 missing values per sample (in contrast to 1.8 missing values per sample for clusters 1 and 2 combined). Based on this analysis, we concluded that the 4 samples in cluster 3 on this analysis, we concluded that the 4 samples in cluster 3 represent samples with poor data quality. These have been excluded from all methylation analysis. miRNA-seq Data Analysis miRNA-seq unaligned bam files were downloaded from the CPTAC GDC API (https://docs.gdc.cancer.gov). Unaligned bams were first converted to fastq.gz by samtools bam2fq (samtools version 1.9). TPM (Transcripts per million) values of mature miRNA and 1.9). TPM (Transcripts per million) values of mature miRNA and precursor miRNA were reported after adaptor trimming, quality check, alignment, annotation, and read counting (https://github.com/ding-lab/CPTAC_miRNA/blob/master/cptac_mirna_analysis.md). The The mature miRNA expression was calculated irrespective of its gene of origin by summing the expression. Proteomics Data Analysis\\n\\nSpectrum quality filtering and searching All MS data were interpreted using the Spectrum Mill software package v7.0 pre-release (Agilent Technologies, Santa Clara, CA) co-developed by Karl Clauser of the Carr laboratory (https://www.broadinstitute.org/proteomics). Similar MS/MS spectra acquired Similar MS/MS spectra acquired on the same precursor m/z within +/\\u2212 45 sec were merged. MS/MS spectra were excluded from searching if they failed the quality filter by not having a sequence tag length > 0 (i.e., minimum of two masses separated by the tag length > 0 (i.e., minimum of two masses separated by the in-chain mass of an amino acid) or did not have a precursor MH+ in the range of 800\\u20136000. MS/MS spectra were searched against a RefSeq-based sequence database containing 41,457 proteins mapped sequence database containing 41,457 proteins mapped to the human reference genome (GRCh38/hg38) obtained via the UCSC Table Browser (https://genome.ucsc.edu/cgi-bin/hgTables) on June 29, 2018, with the addition of 13 proteins encoded in the human 29, 2018, with the addition of 13 proteins encoded in the human mitochondrial genome, 264 common laboratory contaminant proteins, and 553 non-canonical small open reading frames. Scoring parameters were ESI-QEXACTIVE-HCD-v2, for whole proteome datasets, were ESI-QEXACTIVE-HCD-v2, for whole proteome datasets, and ESI-QEXACTIVE-HCD-v3, for phosphoproteome, acetylproteome, and ubiquitylproteome datasets. All spectra were allowed +/\\u2212 20 ppm mass tolerance for precursor and product ions, 30% minimum matched tolerance for precursor and product ions, 30% minimum matched peak intensity, and \\u201ctrypsin allow P\\u201d enzyme specificity with up to 4 missed cleavages. Allowed fixed modifications included carbamidomethylation of cysteine and selenocysteine. TMT labeling of cysteine and selenocysteine. TMT labeling was required at lysine, but peptide N-termini were allowed to be either labeled or unlabeled. Allowed variable modifications for whole proteome datasets were acetylation of protein N-termini, oxidized datasets were acetylation of protein N-termini, oxidized methionine, deamidation of asparagine, hydroxylation of proline in PG motifs, pyroglutamic acid at peptide N-terminal glutamine, and pyro-carbamidomethylation at peptide N-terminal cysteine with a pyro-carbamidomethylation at peptide N-terminal cysteine with a precursor MH+ shift range of \\u221218 to 97 Da. For the phosphoproteome dataset the allowed variable modifications were revised to allow phosphorylation of serine, threonine, and tyrosine, allow allow phosphorylation of serine, threonine, and tyrosine, allow deamidation only in NG motifs, and disallow hydroxylation of proline with a precursor MH+ shift range of \\u221218 to 272 Da. For the acetylproteome dataset the allowed variable modifications were acetylproteome dataset the allowed variable modifications were revised to allow acetylation of lysine, allow deamidation only in NG motifs, and disallow hydroxylation of proline with a precursor MH+ shift range of \\u2212400 to 70 Da. For the ubiquitylproteome MH+ shift range of \\u2212400 to 70 Da. For the ubiquitylproteome dataset the allowed variable modifications were revised to allow diglycine modification of lysine, allow deamidation only in NG motifs, and disallow hydroxylation of proline with a precursor MH+ and disallow hydroxylation of proline with a precursor MH+ shift range of \\u2212690 to 70 Da. Protein grouping, and localization of PTMs Identities interpreted for individual spectra were automatically designated as confidently assigned using the Spectrum Mill autovalidation module to use target-decoy based false discovery rate (FDR) estimates to apply score threshold criteria. For the rate (FDR) estimates to apply score threshold criteria. For the whole proteome dataset thresholding was done in 3 steps: at the peptide spectrum match (PSM) level, the protein level for each TMT-plex, and the protein level for all 22 TMT-plexes. For the TMT-plex, and the protein level for all 22 TMT-plexes. For the phosphoproteome, acetylproteome, and ubiquitylproteome datasets thresholding was done in two steps: at the PSM and variable modification (VM) site levels. In step 1 for all datasets, PSM-level autovalidation was done first and separately for each TMT-plex experiment consisting of either 25 LC-MS/MS runs (whole proteome), 13 LC-MS/MS runs (phosphoproteome), 4 LC-MS/MS runs (acetylproteome), or 2 LC-MS/MS 4 LC-MS/MS runs (acetylproteome), or 2 LC-MS/MS runs (ubiquitylproteome) using an auto-thresholds strategy with a minimum sequence length of 7; automatic variable range precursor mass filtering; and score and delta Rank1 \\u2013 Rank2 score thresholds optimized and score and delta Rank1 \\u2013 Rank2 score thresholds optimized to yield a PSM-level FDR estimate for precursor charges 2 through 4 of <0.8% for each precursor charge state in each LC-MS/MS run. To achieve reasonable statistics for precursor charges 5\\u20136, To achieve reasonable statistics for precursor charges 5\\u20136, thresholds were optimized to yield a PSM-level FDR estimate of <0.4% across all runs per TMT-plex experiment (instead of per each run), since many fewer spectra are generated for the higher run), since many fewer spectra are generated for the higher charge states. In step 2 for the whole proteome dataset, protein-polishing autovalidation was applied separately to each TMTplex experiment to further filter the PSMs using a target protein-level FDR threshold of zero. The primary goal of this step was to eliminate of zero. The primary goal of this step was to eliminate peptides identified with low scoring PSMs that represent proteins identified by a single peptide, so-called \\u201cone-hit wonders.\\u201d After assembling protein groups from the autovalidated PSMs, protein assembling protein groups from the autovalidated PSMs, protein polishing determined the maximum protein level score of a protein group that consisted entirely of distinct peptides estimated to be false-positive identifications (PSMs with negative delta to be false-positive identifications (PSMs with negative delta forward-reverse scores). PSMs were removed from the set obtained in the initial peptide-level autovalidation step if they contributed to protein groups that had protein scores below the contributed to protein groups that had protein scores below the maximum false-positive protein score. Step 3 was then applied, consisting of protein-polishing autovalidation across all TMT plexes together using the protein grouping method \\u201cexpand TMT plexes together using the protein grouping method \\u201cexpand subgroups, top uses shared\\u201d to retain protein subgroups with either a minimum protein score of 25 or observation in at least 4 TMT plexes. The primary goal of this step was to eliminate low TMT plexes. The primary goal of this step was to eliminate low scoring proteins that were infrequently detected in the sample cohort. As a consequence of these two protein-polishing steps, each identified protein reported in the study comprised multiple identified protein reported in the study comprised multiple peptides, unless a single excellent scoring peptide was the sole match and that peptide was observed in at least 4 TMT-plexes. In calculating scores at the protein level and reporting the In calculating scores at the protein level and reporting the identified proteins, peptide redundancy was addressed in Spectrum Mill as follows: The protein score was the sum of the scores of distinct peptides. A distinct peptide was the single highest of distinct peptides. A distinct peptide was the single highest scoring instance of a peptide detected through an MS/MS spectrum. MS/MS spectra for a particular peptide may have been recorded multiple times (e.g. as different precursor charge states, in multiple times (e.g. as different precursor charge states, in adjacent bRP fractions, modified by deamidation at Asn or oxidation of Met, or with different phosphosite localization), but were still counted as a single distinct peptide. When a peptide were still counted as a single distinct peptide. When a peptide sequence of >8 residues was contained in multiple protein entries in the sequence database, the proteins were grouped together and the highest scoring one and its accession number were and the highest scoring one and its accession number were reported. In some cases when the protein sequences were grouped in this manner, there were distinct peptides that uniquely represent a lower scoring member of the group (isoforms, family members, a lower scoring member of the group (isoforms, family members, and different species). Each of these instances spawned a subgroup. Multiple subgroups were reported, counted towards the total number of proteins, and given related protein subgroup numbers number of proteins, and given related protein subgroup numbers (e.g. 3.1 and 3.2 for group 3, subgroups 1 and 2). For the whole proteome datasets the above criteria yielded false discovery rates (FDR) for each TMT-plex experiment of <0.6% at the rates (FDR) for each TMT-plex experiment of <0.6% at the peptide-spectrum match level and <0.8% at the distinct peptide level. After assembling proteins with all the PSMs from all the TMT-plex experiments together, the aggregate FDR estimates were 0.58% experiments together, the aggregate FDR estimates were 0.58% at the peptide-spectrum match level, 2.3% at the distinct peptide level, and <0.01% (1/11,029) at the protein group level. Since the protein level FDR estimate neither explicitly required a the protein level FDR estimate neither explicitly required a minimum number of distinct peptides per protein nor adjusted for the number of possible tryptic peptides per protein, it may underestimate false positive protein identifications for large underestimate false positive protein identifications for large proteins observed only on the basis of multiple low scoring PSMs. In step 2 for the phosphoproteome and acetylproteome datasets, variable modification (VM) site polishing autovalidation was applied across all 22 TMT plexes to retain all VM-site identifications with either a minimum id score of 8.0 or observation in at with either a minimum id score of 8.0 or observation in at least 4 TMT plexes. For the ubiquitylproteome datasets, it was applied across all 15 TMT plexes to retain all VM-site identifications with either a minimum id score of 8.0 or observation in at with either a minimum id score of 8.0 or observation in at least 3 TMT plexes. The intention of the VM site polishing step is to control FDR by eliminating unreliable VM site-level identifications, particularly low scoring VM sites that are only detected particularly low scoring VM sites that are only detected as low scoring peptides that are also infrequently detected across all of the TMT plexes in the study. In calculating scores at the VM-site level and reporting the identified VM sites, redundancy VM-site level and reporting the identified VM sites, redundancy was addressed in Spectrum Mill as follows: A VM-site table was assembled with columns for individual TMT-plex experiments and rows for individual VM-sites. PSMs were combined into a single rows for individual VM-sites. PSMs were combined into a single row for all non-conflicting observations of a particular VM-site (e.g. different missed cleavage forms, different precursor charges, confident and ambiguous localizations, and different charges, confident and ambiguous localizations, and different sample-handling modifications). For related peptides, neither observations with a different number of VM-sites nor different confident localizations were allowed to be combined. Selecting the localizations were allowed to be combined. Selecting the representative peptide from the combined observations was done such that once confident VM-site localization was established, higher identification scores and longer peptide lengths were preferred. scores and longer peptide lengths were preferred. While a Spectrum Mill identification score was based on the number of matching peaks, their ion type assignment, and the relative height of unmatched peaks, the VM site localization score was the of unmatched peaks, the VM site localization score was the difference in identification score between the top two localizations. The score threshold for confident localization, >1.1, essentially corresponded to at least 1 b or y ion located between two corresponded to at least 1 b or y ion located between two candidate sites that has a peak height >10% of the tallest fragment ion (neutral losses of phosphate from the precursor and related ions as well as immonium and TMT reporter ions were excluded from as well as immonium and TMT reporter ions were excluded from the relative height calculation). The ion type scores for b-H3PO4, y-H3PO4, b-H2O, and y-H2O ion types were all set to 0.5. This prevented inappropriate confident localization assignment when a inappropriate confident localization assignment when a spectrum lacked primary b or y ions between two possible sites but contained ions that could be assigned as either phosphate-loss ions for one localization or water loss ions for another localization. one localization or water loss ions for another localization. VM-site polishing yielded 68, 674 phosphosites with an aggregate FDR of 0.73% at the phosphosite level. In aggregate, 71% of the reported phosphosites in this study were fully localized to a reported phosphosites in this study were fully localized to a particular serine, threonine, or tyrosine residue. VM-site polishing yielded 15,186 acetylsites with an aggregate FDR of 0.80% at the acetylsite level. In aggregate, 99% of the reported at the acetylsite level. In aggregate, 99% of the reported acetylsites in this study were fully localized to a particular lysine residue. VM-site polishing yielded 25,489 ubiquitylsites with an aggregate FDR of 0.40% at the ubiquitylsite level. In with an aggregate FDR of 0.40% at the ubiquitylsite level. In aggregate, >99% of the reported ubiquitylsites in this study were fully localized to a particular lysine residue. Quantitation using TMT ratios Using the Spectrum Mill Protein/Peptide Summary module, a protein comparison report was generated for the proteome dataset using the protein grouping method \\u201cexpand subgroups, top uses shared\\u201d (SGT). For the phosphoproteome and acetylproteome datasets a (SGT). For the phosphoproteome and acetylproteome datasets a Variable Modification site comparison report limited to either phospho or acetyl sites, respectively, was generated using the protein grouping method \\u201cunexpand subgroups\\u201d. Relative abundances of grouping method \\u201cunexpand subgroups\\u201d. Relative abundances of proteins and VM-sites were determined in Spectrum Mill using TMT reporter ion intensity ratios from each PSM. TMT reporter ion intensities were corrected for isotopic impurities in the Spectrum were corrected for isotopic impurities in the Spectrum Mill Protein/Peptide summary module using the afRICA correction method, which implements determinant calculations according to Cramer\\u2019s Rule and correction factors obtained from the reagent Cramer\\u2019s Rule and correction factors obtained from the reagent manufacturer\\u2019s certificate of analysis (https://www.thermofisher.com/order/catalog/product/90406) for TMT11_lot number UA280170/TL272399 (Proteome, Phosphoproteme, and Acetylproteome) and (Proteome, Phosphoproteme, and Acetylproteome) and TE270748-TD264064 (Ubiquitylproteome). A protein-level, phosphosite-level, or acetylsite-level TMT ratio is calculated as the median of all PSM-level ratios contributing to a protein subgroup, of all PSM-level ratios contributing to a protein subgroup, phosphosite, or acetylsite. PSMs were excluded from the calculation if they lacked a TMT label, had a precursor ion purity < 50% (MS/MS has significant precursor isolation contamination from (MS/MS has significant precursor isolation contamination from co-eluting peptides), or had a negative delta forward-reverse identification score (half of all false-positive identifications). Lack of TMT label led to exclusion of PSMs per TMT plex with a Lack of TMT label led to exclusion of PSMs per TMT plex with a range of 1.4 to 2.3% for the proteome, 1.8 to 3.7% for the phosphoproteome, 1.0 to 3.5% for the acetylproteome, and 4.5 to 18.0% for the ubiquitylproteome datasets. Low precursor ion purity for the ubiquitylproteome datasets. Low precursor ion purity led to exclusion of PSMs per TMT plex with a range of 1.1 to 1.4% for the proteome, 1.7 to 2.8% for the phosphoproteome, 0.5 to 1.7% for the acetylproteome, and 2.8 to 3.5% for the 0.5 to 1.7% for the acetylproteome, and 2.8 to 3.5% for the ubiquitylproteome datasets. Two-component normalization of TMT ratios It was assumed that for every sample there would be a set of unregulated proteins, phosphosites, acetylsites or ubiquitylsites that have abundance comparable to the common reference (CR) sample. In the normalized sample, these proteins, phosphosites, sample. In the normalized sample, these proteins, phosphosites, acetylsites or ubiquitylsites should have a log TMT ratio centered at zero. In addition, there were proteins, phosphosites, acetylsites and ubiquitylsites that were either up- or acetylsites and ubiquitylsites that were either up- or down-regulated compared to the CR. A normalization scheme was employed that attempted to identify the unregulated proteins, phosphosites, acetylsites or ubiquitylsites, and centered the distribution acetylsites or ubiquitylsites, and centered the distribution of these log-ratios around zero in order to nullify the effect of differential protein loading and/or systematic MS variation. A 2-component Gaussian mixture model-based normalization algorithm Gaussian mixture model-based normalization algorithm was used to achieve this effect. The two Gaussians N(\\u03bci1,\\u03c3i1) and N(\\u03bci2,\\u03c3i2) for a sample i were fitted and used in the normalization process as follows: the mode mi of the log-ratio distribution was as follows: the mode mi of the log-ratio distribution was determined for each sample using kernel density estimation with a Gaussian kernel and Shafer-Jones bandwidth. A two-component Gaussian mixture model was then fit with the mean of both Gaussians mixture model was then fit with the mean of both Gaussians constrained to be mi, i.e., \\u03bci1=\\u03bci2=mi. The Gaussian with the smaller estimated standard deviation  was assumed to represent the unregulated component of was assumed to represent the unregulated component of proteins/phosphosites/acetylsites/ubiquitylsites, and was used to normalize the sample. The sample was standardized using (mi,\\u03c3i) by subtracting the mean mi from each standardized using (mi,\\u03c3i) by subtracting the mean mi from each protein/phosphosite/acetylsite/ubiquitylsite and dividing by the standard deviation \\u03c3i. Normalization of acetyl- and ubiquitylproteome To correct for differential underlying protein levels when performing certain acetylproteome- and ubiquitylproteome-based analyses, a global linear model approach was employed to produce protein-normalized versions of both of these PTM datasets. PTM sites versions of both of these PTM datasets. PTM sites were first matched to underlying proteins in the proteome dataset using RefSeq accession numbers. If a site\\u2019s main accession number did not yield a match in the proteome, all accession numbers associated yield a match in the proteome, all accession numbers associated with the site were then tried and the protein with the highest score was selected as a match. For the acetylproteome and ubiquitylproteome separately, a linear model was then fit (R function separately, a linear model was then fit (R function lm, PTM ~ protein) to all matched PTM-protein data points in the dataset. The residuals of each model were then used as protein-normalized acetylsite or ubiquitylsite abundances. Proteogenomic analysis\\n\\nQuality control and Batch effect assessment We used guided principal component analysis (gPCA) to assess the presence of a batch effect in the sample/CR ratio data based on TMT-plex. There was no overall apparent batch effect (gPCA p-value=1) in the global proteome, phosphoproteome, acetylproteome in the global proteome, phosphoproteome, acetylproteome or ubiquitylproteome datasets. To assess composition similarities between CR used for proteome, phosphoproteome and acetylproteome analysis, and ubiquitylproteome, we performed label-free proteomic analysis of each CRs in triplicates. Both CRs showed excellent correlation with median triplicates. Both CRs showed excellent correlation with median pearson correlation > 0.85. In addition, to assess data quality, we assessed the Pearson correlations of log TMT reporter ion intensities between all pairs of TMT plexes, separately for sample and CR channels. For each given pair of TMT plexes, correlations between CRs were each given pair of TMT plexes, correlations between CRs were consistently higher than correlations of respective sample channels enabling precise relative quantitation of proteins, phosphorylation and acetylation sites across 22 TMT plexes using the CR and acetylation sites across 22 TMT plexes using the CR channels as common denominator (Figure S1C). In the ubiquitylproteome data we observed lower correlations of the CR channel (in comparison to samples channels) in three of the 18 TMT plexes analyzed to samples channels) in three of the 18 TMT plexes analyzed (plexes 7, 8, 10). Clustering of the CR channels using pairwise correlations as a similarity metric pointed to a batch effect in the CR channels (but not in the sample channels) in these plexes. CR channels (but not in the sample channels) in these plexes. The presence of a batch effect was further confirmed by applying the gPCA test described above to only the reporter intensities of the CR channels (gPCA p-value < 0.001). Even though of the CR channels (gPCA p-value < 0.001). Even though undetectable with a global test, we observed this CR batch effect propagate to the sample/CR ratio data in preliminary analyses (data not shown). To counter the propagation of this CR batch effect to shown). To counter the propagation of this CR batch effect to the sample/CR ratio for samples in these plexes, batch correction was applied to the sample/CR ratio data for plexes 7, 8, 10 (batch 1) and all other plexes (batch 2) using limma\\u2019s 7, 8, 10 (batch 1) and all other plexes (batch 2) using limma\\u2019s removeBatchEffect (https://rdrr.io/bioc/limma/man/removeBatchEffect.html) on the sample/CR ratio data including the sample type (tumor, NAT) as a covariate. Dataset filtering Genes (RNA-seq), proteins (global proteome), phosphosites, acetylsites and ubiquitin sites present in fewer than 30% of samples (i.e., missing in >70% of samples) were removed from the respective datasets. Furthermore, proteins were required to have at datasets. Furthermore, proteins were required to have at least two observed TMT ratios in >25% of samples in order to be included in the proteome dataset. Phosphosites, acetylsites and ubiquitin sites were required to have at least one observed TMT ratio sites were required to have at least one observed TMT ratio in >25% of samples. Some of the filtering steps were modified for specific analyses in the study. For many of the marker selection and gene set enrichment analyses, at least 50% of samples were required to have non-missing values for proteins/phosphosites/acetyl/ubiquitin non-missing values for proteins/phosphosites/acetyl/ubiquitin sites, since missing values were imputed, and excessive missing values can result in poor imputation. Alternate filtering has been noted in descriptions of the relevant methods. Unsupervised multi-omic clustering using NMF We used non-negative matrix factorization (NMF)-based multi-omic clustering using protein, phosphosite, acetylsite, RNA transcript and gene copy number variants (CNV) as previously described (. Briefly, given a factorization rank k (where k is the number (. Briefly, given a factorization rank k (where k is the number of clusters), NMF decomposes a p x n data matrix V into two matrices W and H such that multiplication of W and H approximates V. Matrix H is a k x n matrix whose entries represent weights for Matrix H is a k x n matrix whose entries represent weights for each sample (1 to N) to contribute to each cluster (1 to k), whereas matrix W is a p x k matrix representing weights for each feature (1 to p) to contribute to each cluster (1 to k). Matrix H (1 to p) to contribute to each cluster (1 to k). Matrix H was used to assign samples to clusters by choosing the k with maximum score in each column of H. For each sample we calculated a cluster membership score as the maximal fractional score of the cluster membership score as the maximal fractional score of the corresponding column in matrix H. We defined a \\u201dcluster core\\u201d as the set of samples with cluster membership score > 0.5. To enable integrative multi-omic clustering we enforced all data types (and converted if necessary) to represent ratios to: (i) a common reference measured in each TMT plex (proteome, phosphoproteome, acetylproteome), (ii) an in silico common reference acetylproteome), (ii) an in silico common reference calculated as the median abundance across all samples (RNA expression) or (iii) to matching blood normal for CNA data. The CNA data was further filtered to only retain genes significantly altered was further filtered to only retain genes significantly altered (GISTIC2 thresholded of +2 or \\u22122) in at least 5% of all tumors. All data tables were then concatenated and all rows with missing values were removed. To remove uninformative features from the values were removed. To remove uninformative features from the dataset prior to NMF clustering we removed features with the lowest standard deviation (bottom 5th percentile) across all samples. Each column in the data matrix was further scaled and samples. Each column in the data matrix was further scaled and standardized such that all features from different data types were represented as z-scores. The resulting data matrix of z-scores into was converted to a non-negative input matrix required by into was converted to a non-negative input matrix required by NMF as follows: Create one data matrix with all negative numbers zeroed.\\n\\nCreate another data matrix with all positive numbers zeroed and the signs of all negative numbers removed. Concatenate both matrices resulting in a data matrix twice as large as the original, but containing only positive values and zeros and hence appropriate for NMF. The resulting matrix was then subjected to NMF analysis leveraging the NMF R-package and using the factorization method described in. To determine the optimal factorization rank k (number of clusters) for the multi-omic data matrix we tested a range of clusters) for the multi-omic data matrix we tested a range of clusters between k=2 and 8. For each k we factorized matrix V using 50 iterations with random initializations of W and H. To determine the optimal factorization rank we calculated two metrics the optimal factorization rank we calculated two metrics for each k: 1) cophenetic correlation coefficient measuring how well the intrinsic structure of the data was recapitulated after clustering and 2) the dispersion coefficient of the consensus matrix and 2) the dispersion coefficient of the consensus matrix as defined in measuring the reproducibility of the clustering across 50 iterations. The optimal k was defined as the maximum of the product of both metrics for cluster numbers between k=3 and 8 Having determined the optimal factorization rank k, in order to achieve robust factorization of the multi-omic data matrix V, we repeated the NMF analysis using 500 iterations with random initializations of W and H and performed the partitioning of initializations of W and H and performed the partitioning of samples into clusters as described above. Due to the non-negative transformation applied to the z-scored data matrix as described above, matrix W of feature weights contained two separate above, matrix W of feature weights contained two separate weights for positive and negative z-scores of each feature, respectively. In order to revert the non-negative transformation and to derive a single signed weight for each feature, we first and to derive a single signed weight for each feature, we first normalized each row in matrix W by dividing by the sum of feature weights in each row, aggregated both weights per feature and cluster by keeping the maximal normalized weight and and cluster by keeping the maximal normalized weight and multiplication with the sign of the z-score in the initial data matrix. Thus, the resulting transformed version of matrix Wsigned contained signed cluster weights for each feature in the input contained signed cluster weights for each feature in the input matrix. For functional characterization of clustering results by single sample Gene Set Enrichment Analysis (ssGSEA), we calculated normalized enrichment scores (NES) of cancer-relevant gene sets by projecting the matrix of signed multi-omic feature weights by projecting the matrix of signed multi-omic feature weights (Wsigned) onto Hallmark pathway gene sets using ssGSEA. To derive a single weight for each gene measured across multiple omics data types (protein, RNA, phosphorylation site, acetylation site, types (protein, RNA, phosphorylation site, acetylation site, CNA) we retained the weight with maximal absolute amplitude. We used the ssGSEA implementation available on https://github.com/broadinstitute/ssGSEA2.0 using the following parameters: gene.set.database=\\u201ch.all.v6.2.symbols.gmt\\u201d\\n\\nsample.norm.type=\\u201crank\\u201d\\n\\nweight=1\\n\\nstatistic=\\u201darea.under.RES\\u201d\\n\\noutput.score.type=\\u201cNES\\u201d\\n\\nnperm=1000\\n\\nglobal.fdr=TRUE\\n\\nmin.overlap=5\\n\\ncorrel.type=\\u201dz.score\\u201d correl.type=\\u201dz.score\\u201d\\n\\nTo test the association of the resulting clusters to clinical variables we used Fisher\\u2019s exact test (R function fisher.test) to test for overrepresentation in the set of samples defining the cluster core as described above. The entire workflow described above has been implemented as a module for PANOPLY which runs on Broad\\u2019s Cloud platform Terra (https://app.terra.bio/). The docker containers encapsulating the source code and required R-packages for NMF clustering and ssGSEA code and required R-packages for NMF clustering and ssGSEA have been submitted to Dockerhub (broadcptacdev/pgdac_mo_nmf:15, broadcptac/pgdac_ssgsea:5). The source code for ssGSEA is available on GitHub: https://github.com/broadinstitute/ssGSEA2.0. Integrative analysis with Stewart et al. To enable integrative analysis of our protein data with the protein data in Stewart et al. we first aggregated the protein-level data to gene-level data matrices by retaining the dominant isoform (identified by the lowest protein subgroup number) isoform (identified by the lowest protein subgroup number) associated with each gene symbol. To aggregate the Stewart et al. protein data we used the mean expression across all proteins mapping to the same gene symbol. Both gene-level protein data mapping to the same gene symbol. Both gene-level protein data matrices were then separately subjected to gene-level z-score transformation before joining the matrices using the unique gene symbols as keys. The resulting integrated protein data matrix was as keys. The resulting integrated protein data matrix was then subjected to the NMF-based clustering pipeline described above using the 3,366 genes quantified across all 216 tumors. RNA subtyping Previously reported LSCC gene expression-subtype signatures were downloaded from the original publication and applied to our RNA expression dataset using the centroid-based method in a manner similar to the TCGA study. Specifically, the RNA expression of similar to the TCGA study. Specifically, the RNA expression of the 208 signature genes, represented as FPKM, was mean-centered in the gene-wise manner, and each sample was correlated to each of the four centroid vectors representing the average signature of the four centroid vectors representing the average signature gene expression for the four subtypes. The samples were assigned to one of the four transcriptomics subtypes according to the highest correlations. Samples with insignificance correlation (p correlations. Samples with insignificance correlation (p > 0.01) were marked to have \\u2018undecided\\u2019 subtype. Chromosomal instability index The CIN score reflects the overall copy number aberration across the whole genome. From the segment-level CNA result, we used a weighted-sum approach to summarize the chromosome instability for each sample. The absolute segment-level log2 ratios of all for each sample. The absolute segment-level log2 ratios of all copy number segments (indicating the copy number aberration of these segments) within a chromosome were weighted by the segment length and summed up to derive the instability score for the length and summed up to derive the instability score for the chromosome. The genome-wide chromosome instability index was calculated by summing up the instability score of all 22 autosomes. Fusion detection and analysis Fusion calling was performed using a combination of CRISP, CODAC MI-ONCOSEQ pipeline, fusioncatcher_v1.10 and arriba_v1.1.0 (https://github.com/suhrig/arriba/). The fusions calls were compiled and visualized using AGFusion to display the predicted domain and visualized using AGFusion to display the predicted domain and exon structure of the fusion proteins as described. mRNA and Protein correlation To compare mRNA expression and protein abundance across samples we focused on the RNAseq data with 21,792 genes, and global proteome with 11,575 quantified proteins. Protein IDs were mapped to gene names. In total, 10,890 genes common to both RNAseq and to gene names. In total, 10,890 genes common to both RNAseq and proteome data spanning 108 tumor samples were used in the analysis. The analyses were carried out on normalized data: RNAseq data were log2 transformed, upper quartile normalized RPKM values, were log2 transformed, upper quartile normalized RPKM values, which were median-centered by row (i.e., gene); proteome data was two-component normalized as described earlier. Pearson correlation was calculated for each gene. Both correlation coefficient was calculated for each gene. Both correlation coefficient and p-value were recorded. Further, adjusted p-values were calculated using the Benjamini\\u2013Hochberg procedure. CNA-driven cis and trans effects Correlations between copy number alterations (CNA) and RNA, proteome, phosphoproteome acetylproteome and ubiquitylproteome (with proteome and PTM data mapped to genes, by choosing the most variable protein isoform/PTM site as the gene-level the most variable protein isoform/PTM site as the gene-level representative) were determined using Pearson correlation of common genes present in CNA-RNA-proteome (9,988 genes), CNA-RNA-phosphoproteome (5,144 genes), CNA-RNA-acetylproteome (1,344 genes) (5,144 genes), CNA-RNA-acetylproteome (1,344 genes) and CNA-RNA-ubiquitylproteome (2,616 genes). In addition, p-values (corrected for multiple testing using Benjamini-Hochberg FDR) for assessing the statistical significance of the correlation values were the statistical significance of the correlation values were also calculated. CNA trans-effects for a given gene were determined by identifying genes with statistically significant (FDR < 0.05) positive or negative correlations. CMAP analysis Candidate genes driving response to copy number alterations were identified using large-scale Connectivity Map (CMAP) queries. The CMAP is a collection of about 1.3 million gene expression profiles from cell lines treated with bioactive small molecules profiles from cell lines treated with bioactive small molecules (~20,000 drug perturbagens), shRNA gene knockdowns (~4,300) and ectopic expression of genes. The CMAP dataset is available on GEO (Series GSE92742). For this analysis, we use the Level 5 on GEO (Series GSE92742). For this analysis, we use the Level 5 (signatures from aggregating replicates) TouchStone dataset with 473,647 total profiles, containing 36,720 gene knock-down profiles, with measurements for 12,328 genes. See knock-down profiles, with measurements for 12,328 genes. See https://clue.io/GEO-guide for more information. To identify candidate driver genes, proteome profiles of copy number-altered samples were correlated with gene knockdown mRNA profiles in the above CMAP dataset, and enrichment of up/downregulated genes was evaluated. Normalized log2 copy num- ber values genes was evaluated. Normalized log2 copy num- ber values less than 0.3 defined deletion (loss), and values greater than +0.3 defined copy number amplifications (gains). In the copy number-altered samples (separately for CNA amplification and CNA samples (separately for CNA amplification and CNA deletion), the trans-genes (identified by significant correlation in \\u201cCNA driven cis and trans effects\\u201d above) were grouped into UP and DOWN categories by comparing the protein ratios of these genes to categories by comparing the protein ratios of these genes to their ratios in the copy number neutral samples (normalized log2 copy number between 0.3 and +0.3). The lists of UP and DOWN trans-genes were then used as queries to interrogate CMAP signatures were then used as queries to interrogate CMAP signatures and calculate weighted connectivity scores (WTCS) using the single-sample GSEA algorithm. The weighted connectivity scores were then normalized for each perturbation type and cell line to obtain normalized for each perturbation type and cell line to obtain normalized connectivity scores (NCS). See for details on WTCS and NCS. For each query we then identified outlier NCS scores, where a score was considered an outlier if it lay beyond 1.5 times a score was considered an outlier if it lay beyond 1.5 times the interquartile range of score distribution for the query. The query gene was designated a candidate driver if (i) the score outliers were statistically cis-enriched (Fisher test with BH-FDR were statistically cis-enriched (Fisher test with BH-FDR multiple testing correction) and (ii) the gene had statistically significant and positive cis-correlation. For a gene to be considered for inclusion in a CMAP query it needed to i) have a copy number change (amplification or deletion) in at least 15 samples; ii) have at least 20 significant trans genes; and iii) be on the list of shRNA knockdowns in the CMAP. genes; and iii) be on the list of shRNA knockdowns in the CMAP. 502 genes satisfied these conditions and resulted in 976 queries (CNA amplification and deletion combined) that were tested for enrichment. Thirty two (32) candidate driver genes were for enrichment. Thirty two (32) candidate driver genes were identified with Fisher\\u2019s test FDR < 0.25, using this process. In order to ensure that the identified candidate driver genes were not a random occurrence, we performed a permutation test to determine how many candidate driver genes would be identified with random input (Mertins et al., 2016). For the 976 queries with random input (Mertins et al., 2016). For the 976 queries used, we substituted the bona-fide trans-genes with randomly chosen genes, and repeated the CMAP enrichment process. To determine FDR, each permutation run was treated as a Poisson sample with FDR, each permutation run was treated as a Poisson sample with rate \\u03bb, counting the number of identified candidate driver genes. Given the small n ( = 10) and \\u03bb, a Score confidence interval was calculated (Barker, 2002) and the midpoint of the confidence calculated (Barker, 2002) and the midpoint of the confidence interval used to estimate the expected number of false positives. Using 10 random permutations, we determined the overall false discovery rate to be FDR = 0.33, with a 95% CI of (0.27, 0.40) for rate to be FDR = 0.33, with a 95% CI of (0.27, 0.40) for genes with either CNA amplification or deletion, and an FDR=0.06 (95% CI: 0.01, 0.12) for six (6) genes with both CNA amplification and deletion. To identify how many trans-correlated genes for all candidate regulatory genes could be directly explained by gene expression changes measured in the CMAP shRNA perturbation experiments, knockdown gene expression consensus signature z-scores knockdown gene expression consensus signature z-scores (knockdown/control) were used to identify regulated genes with \\u03b1 = 0.05, followed by counting the number of trans-genes in this list of regulated genes. To obtain biological insight into the list of candidate driver genes, we performed (i) enrichment analysis on samples with extreme CNA values (amplification or deletion) to identify statistically enriched sample annotation subgroups; and (ii) GSEA on enriched sample annotation subgroups; and (ii) GSEA on cis/trans-correlation values to find enriched pathways. LINCS analysis Differentially expressed genes between the 5 NMF LSCC subtypes were identified using the limma package. In order to identify genes that were uniquely expressed in each subtype, all pairwise comparisons between the 5 subtypes were computed. In addition, comparisons between the 5 subtypes were computed. In addition, genes that were differentially expressed (adjusted p.value < 0.05) in at least 3 out of the 4 comparisons and had a concordant fold change among all comparisons were identified as concordant fold change among all comparisons were identified as subtype-specific and were used in the subsequent analysis. Protein abundance comparisons were performed between all 5 NMF subtypes using the Wilcoxon rank-sum test and p-values were adjusted using the Benjamini & Hochberg method. Subtype-specific differentially expressed proteins were identified based on their expressed proteins were identified based on their differential expression (adjusted p.value < 0.05) in at least 2 out of the 4 comparisons, and by having a concordant fold change among all comparisons. The identified differentially expressed genes and comparisons. The identified differentially expressed genes and proteins were then filtered for gene symbols measured in the L1000 assay (978 landmark genes). These NMF-specific signatures were used as input to calculate normalized weighted connectivity used as input to calculate normalized weighted connectivity scores (WTCS) against the Library of Integrated Network-Based Cellular Signatures (LINCS) L1000 perturbation-response signatures. The scores were computed using the sig_queryl1k_tool pipeline The scores were computed using the sig_queryl1k_tool pipeline (https://hub.docker.com/u/cmap) and the LINCS L1000 Level 5 compound (trt_cp) signatures from CLUE (clue.io, \\u201cExpanded CMap LINCS Resource 2020 Release\\u201d). The resulting normalized connectivity Resource 2020 Release\\u201d). The resulting normalized connectivity scores were summarized across cell lines using the maximum quantile of all the scores of the same compound (Qhi = 67,Qlow = 33) as previously described. The corresponding compound metadata as previously described. The corresponding compound metadata were obtained from CLUE (clue.io, \\u201cExpanded CMap LINCS Resource 2020 Release\\u201d) and were used to filter and identify compounds with existing annotations (drug name, MOA, Target). The top 20 with existing annotations (drug name, MOA, Target). The top 20 negatively connected compounds to each NMF signature were selected for visualization. Defining cancer-associated genes\\n\\nCancer-associated genes (CAG) were compiled using the Census website https://cancer.sanger.ac.uk/census. The list of genes is provided in Table S1.\\n\\nCpG Island Methylator Phenotype The methylation analysis started with all 103 tumor samples, which excluded 4 samples from the cohort with high missing rate (see above), to generate the CpG island methylator phenotypes (CIMP). Specifically, we first generated the gene-level methylation Specifically, we first generated the gene-level methylation score, by taking the averaged beta values of all probes harboring in the islands of promotor or 5 UTR regions of the gene. Then, we preprocessed the data by filtering genes that were Then, we preprocessed the data by filtering genes that were hypermethylated, i.e. the gene-level methylation score >0.2, transformed the score into M-values, normalized the transformed values, and then imputed the missing values with k-nearest neighbors and then imputed the missing values with k-nearest neighbors (KNN). With preprocessed data, we performed consensus clustering 1000 times, each taking 80% of the samples and all genes, and calculated the consensus matrix (probabilities of two samples calculated the consensus matrix (probabilities of two samples clustering together) for each predetermined number of clusters K. K ranges from 2 to 10. In each value of K, we visualized the consensus matrix using K means with Euclidean distance as the consensus matrix using K means with Euclidean distance as the distance metric. Finally, we determined the optimal number of clusters by considering the relative change in area under the consensus cumulative density function (CDF) curve. In the end, three cumulative density function (CDF) curve. In the end, three distinct clusters were identified, one was hypermethylated with mean M value 0.393, and two were hypomethylated with mean M value \\u22120.041 and \\u22120.359, respectively. We labeled these three clusters and \\u22120.359, respectively. We labeled these three clusters as CIMP high, CIMP intermediate, and CIMP low groups. iProFun Based Cis Association Analysis We used iProFun, an integrative analysis tool to identify multi-omic molecular quantitative traits (QT) perturbed by DNA-level variations. We considered five functional molecular quantitative traits (gene expression, protein, phosphoprotein, quantitative traits (gene expression, protein, phosphoprotein, acetylprotein, ubiquitylproteome levels) for their associations with DNA methylation, accounting for mutation, copy number variation, age, gender, tumor purity and smoking status. Tumor purity age, gender, tumor purity and smoking status. Tumor purity was determined using TSNet from RNA-seq data. The iProFun procedure was applied to a total of 12,666 genes that was available in at least one predictor (mutation, CNA, DNA methylation) and one at least one predictor (mutation, CNA, DNA methylation) and one outcome (mRNA, global protein, phospholation, acetylation and ubiquitylation) for their cis regulatory patterns in tumors. For each of the 12,666 genes, we started with linear regression to each of the 12,666 genes, we started with linear regression to capture the covariate-adjusted associations. The resulting association summary statistics was used to call probabilities of belonging to each of the 2^5=32 possible configurations. We of belonging to each of the 2^5=32 possible configurations. We calculated the probabilities of associating with each of the five outcomes by combining probabilities from relevant configurations, and calculated FDR controlled associations. The significant and calculated FDR controlled associations. The significant genes need to pass three criteria: (1) the satisfaction of biological filtering procedure, (2) posterior probabilities > 75%, and (3) empirical false discovery rate (eFDR)<10%. As DNA methylation empirical false discovery rate (eFDR)<10%. As DNA methylation at the promoter regions of a gene down regulates its expression level, biological filtering procedure requires DNA methylations that exist either negative significant associations or that exist either negative significant associations or insignificant associations with all 5 the types of molecular QTs for significance call. DNA methylations with significant positive associations with QTs were filtered out for significance call, as it with QTs were filtered out for significance call, as it was not the pattern of biological associations that were interesting. Secondly, a significance was called only if the posterior probabilities > 75% of a predictor being associated with a molecular > 75% of a predictor being associated with a molecular QT, by summing over all configurations that are consistent with the association of interest. Lastly, we calculated empirical FDR via 100 permutations per molecular QTs by shuffling the label of the permutations per molecular QTs by shuffling the label of the molecular QTs, and requested empirical FDR (eFDR) <10% by selecting a minimal cutoff value of alpha such that 75%<alpha<100%. The eFDR is calculated by: eFDR= (Averaged No. of genes with The eFDR is calculated by: eFDR= (Averaged No. of genes with posterior probabilities > alpha in permuted data) / (Averaged No. of genes with posterior probabilities > alpha in original data). Figure 1G annotated those genes whose DNA methylation had Figure 1G annotated those genes whose DNA methylation had cascading effects (associated with all QTs under investigation), and among them those whose protein abundances significantly differ between tumor and NAT were annotated. EMT-specific cluster and fibroblast enrichment NMF derived EMT-enriched subtype was interrogated for EMT and fibroblast related proteins in LSCC tumor patients. The relative protein expression was calculated using z-score. The single ssGSEA was performed on a protein dataset using GSVA (v3.11) to ssGSEA was performed on a protein dataset using GSVA (v3.11) to calculate normalized enrichment score for EMT and fibroblast proliferation using MSigdb (v6.1) and gene ontology geneset respectively. Similar analysis was also performed to interrogate SOX2 Similar analysis was also performed to interrogate SOX2 associated genes and its relation with IL6_JAK_STAT signaling (MSigdb hallmarks geneset V6.1). Differential marker analysis A Wilcoxon signed rank test was performed on TMT-based global proteomic data between tumor and matched normal samples to determine differential abundance of proteins between tumor and NAT samples. Proteins having < 50% missing values were considered for Proteins having < 50% missing values were considered for downstream analysis. Proteins with log2-fold-change (FC)> 2 in tumors and Benjamini-Hochberg FDR < 0.01 were considered to be tumor-associated proteins. Over-representation analysis of Gene Ontology proteins. Over-representation analysis of Gene Ontology Biological Process terms was performed on a background of quantified proteins in WebGestaltR. The selected gene list was either significant proteins or proteins containing a significantly altered proteins or proteins containing a significantly altered phosphorylation site from the differential marker analyses. ORA was performed separately for increased and decreased proteins. Terms were considered significant with a Benjamini-Hochberg adjusted were considered significant with a Benjamini-Hochberg adjusted p-value <0.01. Among tumor associated proteins, the top 50 differential proteins with more than 90% of samples showing high expression in tumors were selected and shown. Differential proteins with more than 99% of samples showing tumor specific expression were more than 99% of samples showing tumor specific expression were highlighted. Immunohistochemistry-based antibody-specific staining scores in lung tumors were obtained from the Human Protein Atlas (HPA, https://www.proteinatlas.org), in which Protein Atlas (HPA, https://www.proteinatlas.org), in which tumor-specific staining is reported in four levels, i.e., high, medium, low, and not detected. The protein-specific annotations such as enzymes, transcription factor, transporters, secreted, such as enzymes, transcription factor, transporters, secreted, transmembrane and FDA-approved drugs targeting the protein or reported in drugbank were designated. Similarly, differential analysis was performed between RNA, phosphosite and acetylproteome was performed between RNA, phosphosite and acetylproteome datasets between tumor and matched NATs. The differential genes/sites with log2-fold-change (FC)> 2 in tumors and Benjamini-Hochberg FDR < 0.01 were considered to be tumor-associated. The FDR < 0.01 were considered to be tumor-associated. The overlapping genes/sites with proteins were represented in a checkerbox plot. Overall and disease-free survival in TCGA lung squamous carcinoma Survival for lung squamous tumor patients in TCGA LSCC cohort was calculated using the time from date of diagnosis to death or last contact (Overall survival, OS) and disease-free interval time (Disease-free survival, DFS). Normalized RNA expression was (Disease-free survival, DFS). Normalized RNA expression was categorized into low and high expression group based on mean. Univariate Cox proportional hazards models were fitted to OS, and DFI endpoints to calculate the hazard ratios for tumor-associated endpoints to calculate the hazard ratios for tumor-associated genes/proteins using coxph function in Survival package (version 2.44) in R. The p-values were determined using a log-rank test. The hazard ratio [exp(cox coefficient)] was used to compare poor hazard ratio [exp(cox coefficient)] was used to compare poor survival in low and high-expression group. Kaplan-Meier survival analysis We used time-to-death or last contact data for the current cohort (recorded as number of days from diagnosis date) to assess survival differences based on various sample subgroups and clinical annotations. We used Kaplan-Meier analysis to explore survival annotations. We used Kaplan-Meier analysis to explore survival differences associated with tumor grade, mutation burden, mutation status for SMGs, CUL3-NFE2L2-KEAP1 combined mutation status, CNA for KAT6A/SOX2/TP63/FGFR1/CDKN2A, immune subtype, NMF CNA for KAT6A/SOX2/TP63/FGFR1/CDKN2A, immune subtype, NMF cluster, CIN, tumor grade and ploidy. We identified NMF mixed tumors (i.e., tumors that are not part of the core NMF clusters) had significantly worse survival (log-rank p-value = 0.005). We used a Fisher test to find enriched annotations (among those mentioned in the previous paragraph, with (among those mentioned in the previous paragraph, with tumor stemness index added) associated with NMF mixed tumors. The frequency of SOX2 amplifications were significantly enriched in NMF mixed tumors (p-value = 0.0038). None of the other annotations had tumors (p-value = 0.0038). None of the other annotations had significant enrichment in the NMF mixed tumors. Continuous Smoking Score Non-negative matrix factorization (NMF) was used in deciphering mutation signatures in cancer somatic mutations stratified by 96 base substitutions in tri-nucleotide sequence contexts. To obtain a reliable signature profile, we used SomaticWrapper to call a reliable signature profile, we used SomaticWrapper to call mutations from WGS data. SignatureAnalyzer exploited the Bayesian variant of the NMF algorithm and enabled an inference for the optimal number of signatures from the data itself at a balance optimal number of signatures from the data itself at a balance between the data fidelity (likelihood) and the model complexity (regularization). After decomposing into three signatures, signatures were compared against known signatures derived from COSMIC were compared against known signatures derived from COSMIC and cosine similarity was calculated to identify the best match. We also sought to integrate count of total mutations, t, percentage that are signature mutations, c, and count of DNPs, n, into a continuous score, 0 < S < 1, to quantify the degree of confidence that a sample was associated with smoking signature. We that a sample was associated with smoking signature. We referred to these quantities as the data, namely D = C \\u2229 T \\u2229 N, and used A and A\\u2019 to indicate smoking signature or lack thereof, respectively. In a Bayesian framework, it is readily shown that a In a Bayesian framework, it is readily shown that a suitable form is S = 1 / (1 + R), where R is the ratio of the joint probability of A\\u2019 and D to the joint probability of A and D. For example, the latter can be written P(A)\\u30fbP(C|A)\\u30fbP(T|A)\\u30fbP(N|A) and the the latter can be written P(A)\\u30fbP(C|A)\\u30fbP(T|A)\\u30fbP(N|A) and the former similarly, where each term of the former is the complement of its respective term in this expression. Common risk statistics are invoked as priors, i.e. P(A) = 0.9. We consider S to be a score because rigorous conditioned probabilities are difficult to establish. (For example, the data types themselves are not entirely independent of one another and models using common distributions like the Poisson do not and models using common distributions like the Poisson do not recapitulate realistic variances.) Instead, we adopt a data-driven approach of estimating contributions of each data type based on 2-point empirical fitting of roughly the low and high extremes 2-point empirical fitting of roughly the low and high extremes using shape functions. The general model for data type G is P(G|A) = [x \\u30fberf (g/y) + 1] / (x + 2) and P(G|A\\u2019) = 1 - P(G|A), where erf is the Gaussian error function, g is the observed value erf is the Gaussian error function, g is the observed value for the given data type, and x and y are empirically-determined weights. The shape function for total mutations, T, has two modifiers. First, it includes a simple expected-value correction for purity, u. Namely, assuming mutation-calling does not capture all mutations because of impurities, t is taken as the observed number of because of impurities, t is taken as the observed number of mutations divided by a purity shape function, f, where f \\u2264 1. Although one might model f according to common characteristics of mutation callers, e.g. close to 100% sensitivity for pure samples callers, e.g. close to 100% sensitivity for pure samples and very low calling rate for low variant allele fractions (VAFs), the purity estimates for these data are based on RNA-Seq and are not highly correlated with total mutation counts. Consequently, we highly correlated with total mutation counts. Consequently, we use a line function, f = 0.2\\u30fbu + 0.8, which does not strongly impact the adjustment of low-purity samples. Second, before choosing weights, we removed outliers, as determined by Peirce\\u2019s weights, we removed outliers, as determined by Peirce\\u2019s criterion. Based on the distribution of continuous smoking score, we set 0.15 as the lower bound cutoff and 0.6 as the upper bound cutoff for defining the inferred smoking status. We classified the for defining the inferred smoking status. We classified the samples with strong smoking evidence if the smoking score > 0.6; samples with weak smoking evidence if smoking score < 0.15. Identification of differentially regulated events in NRF2 mutant tumors. For each gene, normalized levels of mRNA/protein/phosphoprotein (log transformed data) were fit into a linear regression model. In addition to NFR2 mutation status, gender, tumor purity and ethnicity were also included in the model. Coefficient and and ethnicity were also included in the model. Coefficient and p-value of NRF2 mutation status variables from the fitted model were used to evaluate the extent and significance of the difference between patients with and without aberration. To adjust for between patients with and without aberration. To adjust for multi-test, p-values were adjusted with the BH method. Genes that showed significant upregulation (up) in more than one dataset (Table S4) were used to generate a GMT file for ssGSEA based NRF2 S4) were used to generate a GMT file for ssGSEA based NRF2 activity score calculation using ssGSEA (https://github.com/broadinstitute/ssGSEA2.0). The NRF2 activity score contains a total of 54 genes upregulated both at protein and RNA level, identified by genes upregulated both at protein and RNA level, identified by our NRF2 mutant versus wildtype tumor differential analysis (p-value <0.05 and logFC>1). Among the 54 genes, 44 have been previously associated with NRF2 signaling (either in NRF2 concept in associated with NRF2 signaling (either in NRF2 concept in msigdb such as Gene Sets NFE2L2.V2 and SINGH_NFE2L2_TARGETS (, BIOCARTA_ARENRF2_PATHWAY], NRF2 wikipathway or Lacher et al. Ten of the 54 genes were novel targets identified in this study. Immune cluster identification based on cell type composition The abundance of 64 different cell types were computed via xCell based on transcriptomic profiles based on 108 tumors and 94 NAT samples. Table S6 contains the final score computed by xCell of different cell types. Consensus clustering was performed based different cell types. Consensus clustering was performed based on immune cells, fibroblasts, endothelial and epithelial cells from xCell. In particular, we only considered cells that were detected in at least 10% of the patients (FDR < 5%). This filtering in at least 10% of the patients (FDR < 5%). This filtering resulted in 25 cell types annotated in Table S6. Based on these signatures, consensus clustering was performed in order to identify groups of samples with similar immune/stromal characteristics. groups of samples with similar immune/stromal characteristics. Consensus clustering was performed using the R packages ConsensusClusterPlus based on z-score normalized signatures. Specifically, signatures were partitioned into three major clusters using signatures were partitioned into three major clusters using the Partitioning Around Medoids (PAM) algorithm, which was repeated 1000 times (Figure 6A, Table S6). TCGA pan-cancer immune subtyping Tumors were classified into the six distinct immune subtypes \\u2014 wound healing, IFN-\\u03b3 dominant, inflammatory, lymphocyte depleted, immunologically quiet, and TGF-\\u03b2 dominant \\u2014 identified by a TCGA pan-cancer analysis presented in. Gene expression data (log2 pan-cancer analysis presented in. Gene expression data (log2 FPKM; lscc-v3.0-rnaseq-uq-fpkm-log2-NArm) was input to the ImmuneSubtypeClassifier R package (https://github.com/CRI-iAtlas/ImmuneSubtypeClassifier) to assign an immune subtype to each of the to assign an immune subtype to each of the tumors. Ranking tumors by inferred activity of IFN-\\u03b3 pathway We assumed that true biological activity of a pathway is regulated by collective changes of expression levels of majority of proteins involved in this pathway. Then, a difference of a pathway activity between tumors can be assessed by a difference in activity between tumors can be assessed by a difference in positioning of expression levels of proteins involved in this pathway in ranked list of expression levels of all proteins in each of tumors. Following this idea, we assessed relative positioning tumors. Following this idea, we assessed relative positioning of pathway proteins between tumor by determining two probabilities: a probability of pathway proteins to occupy by random the observed positions in a list of tumor proteins ranked by expression positions in a list of tumor proteins ranked by expression levels from the top to the bottom and, similarly, a probability to occupy by random the observed positions in a list of expression levels ranked from the bottom to the top. Then, the inferred levels ranked from the bottom to the top. Then, the inferred relative activation of a given pathway across tumors was assessed as a negative logarithm of the ratio of the above \\u201ctop\\u201d and \\u201cbottom\\u201d probabilities. Thus, for a pathway of a single protein, its probabilities. Thus, for a pathway of a single protein, its relative activity across tumors was assessed as a negative log of ratio of two numbers: a number of proteins with expression level bigger than an expression level of given protein, and a number bigger than an expression level of given protein, and a number of proteins with expression levels less than an expression level of given protein. For pathways of multiple proteins, the \\u201ctop\\u201d and \\u201cbottom\\u201d probabilities were computed as geometrically \\u201ctop\\u201d and \\u201cbottom\\u201d probabilities were computed as geometrically averaged P values computed for each of proteins using Fisher\\u2019s exact test, given protein\\u2019s ranks in a list of pathway proteins and in a list of ranked proteins of a tumor, a number of and in a list of ranked proteins of a tumor, a number of proteins in a pathway, and the total number of proteins with the assessed expression level in a given tumor. The thermodynamic interpretation of the inferred pathway activity scoring function is a of the inferred pathway activity scoring function is a free energy associated with deviation of the system from equilibrium either as a result of activation or suppression. Thus, the scoring function is positive, when expression levels of pathway\\u2019s function is positive, when expression levels of pathway\\u2019s proteins are overrepresented among top expressed proteins of a tumor, and it is negative, when pathway\\u2019s proteins are at the bottom of expressed proteins of a tumor; the scoring function is close of expressed proteins of a tumor; the scoring function is close to zero, when expression levels are distributed by random or equally shifted towards top or bottom. In our analysis, we used a gene list of IFN-\\u03b3 signaling pathway from. We settled with this 15 genes signaling pathway because it has all necessary components \\u2013 a cytokine mediator, receptors, signal transduction, transcription factors and final effectors signal transduction, transcription factors and final effectors (PD-L1/2, MHC-I), and all those genes are expressed in cancer cells. Estimation of Tumor Purity, Stromal and Immune Scores Besides xCell, we utilized ESTIMATE to infer immune and stromal scores based on gene expression data (Table S6). Cibersort absolute immune scores were obtained by evaluating upper-quartile normalized RNA-seq FPKM data with the Cibersort web app normalized RNA-seq FPKM data with the Cibersort web app (cibersort.stanford.edu) in absolute mode. To infer tumor purity, TSNet was utilized (Table S6). Differentially Expressed Genes and Pathway Analysis Genes upregulated in Hot and Warm clusters compared to the Cold cluster were identified based on gene expression data, global proteomic, phosphoproteomic and acetylation data. For this analysis, markers with at most 50% missing values were utilized. For markers with at most 50% missing values were utilized. For each data type, the expression level of gene/protein/sites was modeled as a function of immune cluster via linear regression. P-values were then adjusted for multiple comparisons via P-values were then adjusted for multiple comparisons via Benjamini-Hochberg adjustment. For each immune cluster, considering the set of genes up(down)-regulated with Benjamini-Hochberg adjusted p-value lower than 10%, a fisher exact test was implemented p-value lower than 10%, a fisher exact test was implemented to derive enriched pathways. ssGSEA was utilized to obtain pathway scores based on RNA-seq and global proteomics data using the R package GSVA. For this analysis, pathways from the Reactome, KEGG GSVA. For this analysis, pathways from the Reactome, KEGG and Hallmark databases were considered and as background the full list of gene/proteins observed under each data type was utilized. For phosphorylation and acetylproteome data, a gene was For phosphorylation and acetylproteome data, a gene was considered upregulated if at least one substrate of the gene was found upregulated at 10% FDR. The pathway analysis results for different data types are contained in Table S6. Deriving RTK CBPE scores A total of 42 human RTKs were present in our proteomics data set. For each phosphosite in our data set we computed a linear association with each of the RTKs in two ways. A phosphosite profile across samples (for tumor and NAT separately) was modeled as a across samples (for tumor and NAT separately) was modeled as a linear combination of 1) an RTK protein profile and a profile of a protein that a given phosphosite belonged to or 2) an RTK protein profile, a profile of a protein that a given phosphosite profile, a profile of a protein that a given phosphosite belonged to and a profile of tumor purity estimates obtained from RNAseq data via TSNet. Each association computation was carried out only if at least 50% of samples had non-missing data of all out only if at least 50% of samples had non-missing data of all three of RTK protein, phosphosite, and protein in question. The phosphosite:RTK associations were considered significant if their FDR adjusted p-value was below 0.1. Additionally, for each FDR adjusted p-value was below 0.1. Additionally, for each phosphosite the z-scores of their log2 phosphoprotein abundance were computed across samples (separately for tumor and NAT). A raw RTK CBPE score was computed for each RTK in each sample as a RTK CBPE score was computed for each RTK in each sample as a number of all phosphosites with significant association with this RTK that also had a z-score higher than 1. These raw RTK CBPE scores in each sample were normalized across RTKs so the final RTK in each sample were normalized across RTKs so the final RTK CBPE scores of all RTKs in one sample would add up to 1. We selected the most relevant subset of RTKs by requiring that each had a normalized CBPE score of at least 0.1 in at least 5 samples. a normalized CBPE score of at least 0.1 in at least 5 samples. Nine RTKs satisfied this condition in the tumor data set, and seven in NAT. Independent component analysis ICA was performed with a workflow modified from previously described. Decomposition was run for 100 times on the matrix of protein abundance difference between tumor/NAT pairs (n=99). Independent components were in the form of vectors comprised with components were in the form of vectors comprised with weights of all genes in the original data. Components extracted from each run were clustered using HDBSCAN algorithm with cosine distance as dissimilarity metric, min_cluster_size=50 and distance as dissimilarity metric, min_cluster_size=50 and min_samples=20. The centroids of clusters (n=37) were considered as representative of stable signatures, and mean mixing scores (activity of each signature over all samples) of each cluster were of each signature over all samples) of each cluster were used to represent the activity levels of corresponding signature in each sample. Correlation between the extracted signatures and known clinical characteristics were examined by regressing the known clinical characteristics were examined by regressing the corresponding mixing scores for all members of a component cluster against 64 sample annotations to obtain within-cluster average of log10 P-values. Significance was controlled for multiple of log10 P-values. Significance was controlled for multiple testing at 0.01 level (log10 (p-value)=\\u22125.3). Each signature vector (cluster centroid) was submitted to GSEA pre-ranked test for functional annotations. Mutation-based cis- and trans-effects We examined the cis- and trans-effects of 22 genes with somatic mutations that were significant in a previous large-scale TCGA LSCC study on the RNA, proteome, and phosphoproteome of known interactome DBs including Omnipath, Phosphositeplus, DEPOD, interactome DBs including Omnipath, Phosphositeplus, DEPOD, Signor, and CORUM. After excluding silent mutations, samples were separated into mutated and WT groups. We used the Wilcoxon rank-sum test to report differentially expressed features (RNA, rank-sum test to report differentially expressed features (RNA, proteins, phosphosites, acetylsites, and ubiquitylation sites) between the two groups. Differentially enriched features passing an FDR <0.05 cut-off were separated into two categories based an FDR <0.05 cut-off were separated into two categories based on cis- and trans- effects. Germline quantitative trait loci (QTL) analysis To identify germline genetic variants that explain variation in tumor gene (eQTL) and protein (pQTL) expression, we utilized the gold-standard mapping pipeline at https://github.com/molgenis/systemsgenetics/wiki/eQTL-mapping-analysis-cookbook-(eQTLGen). Briefly, we followed the default steps that include correcting for population stratification removing outliers, normalizing and mapping cis-QTLs. For cis-eQTL analysis input, we used i) germline genotype data; and ii) RNA-seq raw read count data for 105 genotype data; and ii) RNA-seq raw read count data for 105 samples, and performed analyses only for genes with sum of read count \\u2265 10. For cis-pQTL analysis input, we used i) germline genotype data; and ii) normalized and filtered protein TMT ratio data; and ii) normalized and filtered protein TMT ratio aggregated to gene level for 108 samples. The genotype data were harmonized using Genotype Harmonizer v1.4.9. To adjust for population stratification, we identified the multi dimensional scaling stratification, we identified the multi dimensional scaling (MDS) components using the genotype data using PLINK v1.07. For both types of cis-QTL mapping, we used eqtl-mapping-pipeline-v1.3.9 using its standard settings (HWE > 0.0001, MAF > 0.01, and call using its standard settings (HWE > 0.0001, MAF > 0.01, and call rate > 0.95, maximum distance between the SNP and the middle of the probe is 250,000 bp). To control for multiple testing, we performed 10 permutations. We deemed QTLs at \\u2264 FDR 5% as we performed 10 permutations. We deemed QTLs at \\u2264 FDR 5% as significant. Finally, we performed pathway analysis of the identified significant eGenes (genes whose expression is impacted by at least one cis-QTL) using QIAGEN Ingenuity Pathway Analysis miRNA analysis presented in Figure 4 Targets of miRNAs were downloaded from the miRNA targets database miRTarBase and only the miRNA/target pairs with strong experimental evidence were retained. Spearman correlation was used to calculate the correlations between miRNA and its target genes. Pathway projection using ssGSEA The single sample Gene Set Enrichment Analysis (ssGSEA) implementation available on https://github.com/broadinstitute/ssGSEA2.0 was used to project log2(FPKM) mRNA abundances to MSigDB cancer hallmark gene sets using the following parameters: gene.set.database=\\u201ch.all.v6.2.symbols.gmt\\u201d\\n\\nsample.norm.type=\\u201crank\\u201d\\n\\nweight=0.75\\n\\nstatistic=\\u201darea.under.RES\\u201d\\n\\noutput.score.type=\\u201cNES\\u201d\\n\\nnperm=1000\\n\\nglobal.fdr=TRUE\\n\\nmin.overlap=10\\n\\ncorrel.type=\\u201dz.score\\u201d\\n\\nPhosphorylation-driven signature analysis We performed phosphosite-specific signature enrichment analysis (PTM-SEA) to identify dysregulated phosphorylation-driven pathways. To adequately account for both magnitude and variance of measured phosphosite abundance, we used p-values derived from measured phosphosite abundance, we used p-values derived from application of the Wilcoxon rank-sum test to phosphorylation data as ranking for PTM-SEA. To that end, p-values were log-transformed and signed according to the fold change (signed \\u2212log10 and signed according to the fold change (signed \\u2212log10 (p-value)) such that large positive values indicated phosphosite abundance in classical or NFE2L2, CUL3, KEAP1 mutated samples, and large negative values indicated phosphosite abundance in samples negative values indicated phosphosite abundance in samples that do not belong to classical NMF subtype or wild-type samples (in Figure 3I)  PTM-SEA relies on site-specific annotation provided by PTMsigDB and thus a single site-centric data matrix data is by PTMsigDB and thus a single site-centric data matrix data is required such that each row corresponds to a single phosphosite. We note that in this analysis the data matrix comprised a single data column and each row represented a confidently localized data column and each row represented a confidently localized phosphosite assigned by Spectrum Mill software. We employed the heuristic method introduced by Krug et al. to deconvolute multiple phosphorylated peptides to separate data points (log-transformed and signed p-values). Briefly, phosphosites measured on different phospho-proteoform peptides were resolved measured on different phospho-proteoform peptides were resolved by using the p-value derived from the least modified version of the peptide. For instance, if a site T4 measured on a doubly phosphorylated (T4, S8) peptide (PEPtIDEsR) was also measured on a (T4, S8) peptide (PEPtIDEsR) was also measured on a mono-phosphorylated version (PEPtIDESR), we assigned the p-value derived from the mono-phosphorylated peptide proteoform to T4, and the p-value derived from PEPtIDEsR to S8. If only the doubly the p-value derived from PEPtIDEsR to S8. If only the doubly phosphorylated proteoform was present in the dataset, we assigned the same p-value to both sites T4 and S8. We queried the PTM signatures database (PTMsigDB) v1.9.0 downloaded from https://github.com/broadinstitute/ssGSEA2.0/tree/master/db/ptmsigdb using the flanking amino acid sequence (+/\\u2212 7 aa) as primary identifier. We used the implementation of PTM-SEA as primary identifier. We used the implementation of PTM-SEA available on GitHub (https://github.com/broadinstitute/ssGSEA2.0) using the command interface R-script (ssgsea-cli.R). The following parameters were used to run PTM-SEA: weight: 1\\n\\nstatistic: \\u201carea.under.RES\\u201d\\n\\noutput.score.type: \\u201cNES\\u201d\\n\\nnperm: 1000\\n\\nmin.overlap: 5\\n\\ncorrel.type: \\u201cZ-score\\u201d The sign of the normalized enrichment score (NES) calculated for each signature corresponds to the sign of the tumor-NAT log fold change. P-values for each signature were derived from 1,000 random permutations and further adjusted for multiple hypothesis permutations and further adjusted for multiple hypothesis testing using the method proposed by Benjamini & and Hochberg. Signatures with FDR-corrected p-values < 0.05 were considered to be differential between tumor and NAT. CDKN2A and RB1 annotations and pathway analysis Comprehensive tumor annotation for CDKN2A and RB1 genomic status was carried out using multiple molecular features for each patient. Specifically, we considered the following: 1) mutation types (missense/in-frame indels mutations, nonsense (stop gain, types (missense/in-frame indels mutations, nonsense (stop gain, frameshift indels) mutations, and splice site (splice donor, splice acceptor) mutations) as separate categories and 2) copy number data. For cases with mutation, we considered both the number data. For cases with mutation, we considered both the variant allele frequency of the mutation from whole exome sequencing and copy number data (log ratio, absolute copy number, and B-allele frequency). Only mutations that occurred in cases with B-allele frequency). Only mutations that occurred in cases with loss of the wild-type allele were annotated as loss of heterozygosity (LOH) mutations and used for subsequent analysis. 2) Based on CDKN2A copy number data, we next annotated tumors without on CDKN2A copy number data, we next annotated tumors without mutations as homozygous deletions or no loss (WT). Samples were re-classified as having p16INK4a promoter hypermethylation if three p16INK4a promoter-associated methylation probes (cg11075751, p16INK4a promoter-associated methylation probes (cg11075751, cg02008397, and cg01694391) all had beta values >= 0.2. Multi-gene proliferation scores were calculated as described previously as the means of the gene normalized RNA levels for cell cycle as the means of the gene normalized RNA levels for cell cycle associated genes characterized by Whitfield et al. p16INK4a RNA levels were obtained from the isoform specific RSEM data for Ensembl transcript ENST00000304494.8, while ENST00000579755.1 was transcript ENST00000304494.8, while ENST00000579755.1 was used for p14ARF, and log2 (RSEM + 1) values were used for barplots. Isoform specific protein data included Refseq NP_000068.1 for p16INK4a and Refseq NP_478102.2 for p14ARF. Calculation of CDK1, 2, and Refseq NP_478102.2 for p14ARF. Calculation of CDK1, 2, and 4 target site scores from the phosphorylation site data and Hallmark E2F targets and G2M scores from the RNA-seq data is described above (Phosphorylation-driven signature analysis and Pathway above (Phosphorylation-driven signature analysis and Pathway projection using ssGSEA, respectively). Lolliplots for CDKN2A mutations in the CPTAC head and neck squamous cell carcinoma and this lung squamous cell carcinoma cohort were generated using the lung squamous cell carcinoma cohort were generated using the ProteinPaint web application to visualize mutations. Mutations annotated as nonsense mediated decay (NMD) mutations were those that resulted in low expression of p16INK4a RNA in the tumors that resulted in low expression of p16INK4a RNA in the tumors harboring the mutations. Association analysis between KGG-site abundances and E3 ligases and DUBs A list of known human E3 ubiquitin and ubiquitin-like ligases and DUBs was compiled from. We then fit a linear model using limma in R with the formula kgg_site_abundance ~ protein_abundance, followed by empirical bayes shrinkage. The coefficient p-values followed by empirical bayes shrinkage. The coefficient p-values were used to determine significant associations after FDR correction (FDR < 0.01). Cluster and pathway analysis of significantly modulated K-GG sites in tumors. Consensus clustering of the K-GG dataset after protein abundance correction was performed on tumor samples, resulting in 3 clusters. Next, marker selection for each cluster was performed by moderated t-test between samples that belong to the clusters by moderated t-test between samples that belong to the clusters against the remaining samples. These K-GG sites showing differential abundance across sample clusters were further clustered into 3 site-wise clusters. Genes in these site-wise clusters were used for pathway enrichment analysis against the KEGG, Reactome, and WikiPathways databases using g:profiler. Pathway enrichment was performed using a gene background containing all observable genes in the K-GG a gene background containing all observable genes in the K-GG dataset. Pathway enrichment results were imported into Cytoscape using the Enrichment Map app for network analysis of pathways. Pathways were connected using the gene overlap and clustered Pathways were connected using the gene overlap and clustered (pathway cutoff q-val < 0.1; jaccard overlap > 0.375). Each cluster was manually annotated from the pathways contained in it to facilitate interpretation. PTM CLUMPS analysis We employ two methods to select tumor-specific sites to include in structural analysis. First, we take PTM-sites for solely tumor-derived samples and binarize modifications by negative vs. positive normalized signal. These are considered as individual positive normalized signal. These are considered as individual \\u201cevents\\u201d as done with mutations in CLUMPS v1. For a more robust approximation of tumor-specific acetylation or ubiquitination, we perform differential expression using Limma between NAT and we perform differential expression using Limma between NAT and Tumor samples. Sites are selected with an FDR < 0.1 (Benjamini-Hochberg) and LFC > 0.1. We then binarize these tumor sites as done in the first approach. We first map the PTM-sites to as done in the first approach. We first map the PTM-sites to corresponding UNIPROT ids (ID) with available PDBs. For each crystal structure, we compute an initial WAP score and randomly sample sites as done in CLUMPS. We generate an empirical p-value sites as done in CLUMPS. We generate an empirical p-value based on a random sampling of lysines in each crystal structure to limit the selection to residues capable of ubiquitination or acetylation. The null hypothesis we define is each random sampling of The null hypothesis we define is each random sampling of lysines will have a WAP score less than or equal to the initially computed WAP score. We run 1e6 permutations to generate an empirical p-value. DepMap genetic dependency and drug response analysis\\n\\nCell lines annotated as \\u201cNSCLC_squamous\\u201d from DepMap were considered as LSCC for which molecular profiles, dependency scores, and drug response were used. To determine copy number status in LSCC cell lines, copy number segmentation files for LSCC cell lines were processed with GISTIC2.0 (using the same parameters used to process this study\\u2019s LSCC cohort as described in STAR METHODS). Samples were considered cohort as described in STAR METHODS). Samples were considered to be amplified for a given gene if GISTIC thresholded values = 1 or 2 and not amplified if GISTIC thresholded values = \\u22122, \\u22121, or 0. Mutation status for CCND1, CDKN2A, and RB1, Np63 transcript 0. Mutation status for CCND1, CDKN2A, and RB1, Np63 transcript isoform expression, phospho-Rb-Ser807/811 RPPA abundance, and drug responses to a survivin inhibitor (YM-155) and CDK4/6i inhibitors (abemaciclib and palbociclib) were also retrieved from (abemaciclib and palbociclib) were also retrieved from DepMap. Figure 7: For identifying if top protein biomarkers (502 proteins significantly overexpressed (log2(FC) >2, FDR <0.01) in tumors relative to their matched NATs, most with coherent overexpression in multi-omic analysis) also conferred altered dependencies in multi-omic analysis) also conferred altered dependencies in LSCC cell lines, we leveraged DepMap genetic dependency dataset (CRISPR Avana Public 20Q3) that contained 18119 genes and 789 cell lines (https://depmap.org/portal/download/ file: and 789 cell lines (https://depmap.org/portal/download/ file: Achilles_gene_effect.csv). Only 16 cell lines were classified as LSCC (Cell Line Sample Info.csv). Median dependencies were calculated and for every gene and density plot in Figure 7C shows and for every gene and density plot in Figure 7C shows dependency score of the 502 genes corresponding to 502 protein biomarkers relative to other genes. Figure S4B: To investigate copy number and dependency associations in LSCC cell lines, each gene up-regulated at the protein level with recurrent copy number gain in tumors vs NATs (from Figure 4A with Log2FC>0 & FDR < 0.05) was used as a query to compare 4A with Log2FC>0 & FDR < 0.05) was used as a query to compare dependency scores of the given gene (Combined CRISPR KO screen, DepMapPublic 21Q1) in LSCC cell lines where the query gene was amplified vs not-amplified. A lower mean change in dependency amplified vs not-amplified. A lower mean change in dependency scores for a given gene in amplified vs not-amplified samples indicates LSCC cell lines are more dependent upon that gene when it is amplified. Continuous log2 copy number data from DepMap was used in Figure S4M to correlate SOX2 copy number with EZH2 shRNA dependency data (Combined shRNA screen from DEMETER2 Data v6 in DepMap). Drug responses to YM-155 (PRISM secondary screen) were compared by t-test in LSCC cell lines with low vs high levels of Np63 by transcript isoform expression (Figure S4G). Pearson correlation was also performed with phospho-Rb-Ser807/811 RPPA abundance was also performed with phospho-Rb-Ser807/811 RPPA abundance and drug responses to CDK4/6i inhibitors, abemaciclib (PRISM secondary screen) and palbociclib (Sanger GDSC1) (Figure S3D). CausalPath analysis CausalPath searches for known biological mechanisms that can explain correlated proteomic changes in terms of causal hypotheses. We set CausalPath parameters to compare tumors and NATs with a paired t-test, used 0.1 as FDR threshold for proteomic change a paired t-test, used 0.1 as FDR threshold for proteomic change significance and network significance, and detected 5917 potential causal relations between proteins. We repeated the same analysis for each NMF subtype separately and identified 4378 analysis for each NMF subtype separately and identified 4378 (basal-inclusive), 5334 (classical), 3048 (EMT-enriched), 3744 (inflamed-secretory), and 4332 (proliferative-primitive) relations. We used these CausalPath network results in the preparation of We used these CausalPath network results in the preparation of Figure 7C, identifying potential upstream regulators of oncogenic phosphoproteomic changes. Here an oncogenic phosphoproteomic change can be any of the following 4 events: increase of change can be any of the following 4 events: increase of activating phosphorylation of an oncoprotein, decrease of inactivating phosphorylation of an oncoprotein, decrease of activating phosphorylation of a tumor suppressor protein, and increase of phosphorylation of a tumor suppressor protein, and increase of inactivating phosphorylation of a tumor suppressor protein. We used the OncoKB database for oncoprotein and tumor suppressor classification (excluded proteins that have both annotations), and (excluded proteins that have both annotations), and used PhosphoSitePlus and Signor databases for the activating/inhibiting classification of phosphorylation sites. In the phosphorylation regulation networks, we included only the targetable regulators regulation networks, we included only the targetable regulators (activated proteins) and excluded the untargetable regulators (inactivated proteins). Variant Peptide Identification We used NeoFlow (https://github.com/bzhanglab/neoflow) for neoantigen prediction. Specifically, Optitype was used to find human leukocyte antigens (HLA) for each sample based on WES data. Then we used netMHCpan to predict HLA peptide binding affinity for we used netMHCpan to predict HLA peptide binding affinity for somatic mutation\\u2013derived variant peptides with a length between 8\\u201311 amino acids. The cutoff of IC50 binding affinity was set to 150 nM. HLA peptides with binding affinity higher than 150 nM 150 nM. HLA peptides with binding affinity higher than 150 nM were removed. Variant identification was also performed at both mRNA and protein levels using RNA-Seq data and MS/MS data, respectively. Variant identification and gene quantification using Variant identification and gene quantification using RNA-Seq data were performed following the methods used in the previous study. To identify variant peptides, we used a customized protein sequence database approach. We built a customized database for sequence database approach. We built a customized database for each TMT experiment based on somatic variants from WES data. We used Customprodbj (https://github.com/bzhanglab/customprodbj) for customized database construction. MS-GF+ was used for variant customized database construction. MS-GF+ was used for variant peptide identification for all global proteome and phosphorylation data. Results from MS-GF+ were filtered with 1% FDR at peptide level. Remaining variant peptides were further filtered using level. Remaining variant peptides were further filtered using PepQuery (http://www.pepquery.org) with the p-value cutoff <= 0.01. Competitive filtering based on unrestricted posttranslational modification searching was enabled in PepQuery validation. The modification searching was enabled in PepQuery validation. The spectra of variant peptides were annotated using PDV (http://www.zhang-lab.org/). Cancer/testis Antigen Prediction\\n\\nCancer/testis (CT) antigens were downloaded from the CTdatabase. CT antigens with a median >4-fold increase in tumor from NAT were highlighted.\\n\\nPROGENy Scores PROGENy Scores\\n\\nPROGENy was used to generate activity scores for EGFR based on RNA expression data. Tumor RNA expression values were submitted to PROGENy.\\n\\nLSCC, HNSCC and LUAD integrative analysis LUAD data for 110 lung adenocarcinoma samples and 102 NAT were acquired from the published manuscript. HNSCC data for 108 head and neck squamous cell carcinoma samples and 66 NAT were acquired from the submitted manuscript. The LUAD and HNSCC proteomics from the submitted manuscript. The LUAD and HNSCC proteomics data were processed, quantified, and normalized using the same pipeline as described for the LSCC samples. Differential expression analysis for all three cohorts was performed as described in analysis for all three cohorts was performed as described in the Differential marker analysis method. Proteins missing in more than 50% of the paired tumor/NAT samples were excluded. Copy number drivers for all three were assessed by filtering the 8309 genes that were quantified in all three cohorts in the copy number, RNA, and proteomic data to those found in focal amplification regions (q value < .25). Spearman correlation was amplification regions (q value < .25). Spearman correlation was performed for these genes between CNA and RNA and between CNA and protein. Proteins were considered drivers if the correlation between both the CNA and RNA and CNA and protein were between both the CNA and RNA and CNA and protein were significantly positively associated (BH adjusted p value <0.01). Proteins were also required to be significantly increased in tumor vs paired NAT in the same cohort (Wilcoxon signed rank BH adjusted p NAT in the same cohort (Wilcoxon signed rank BH adjusted p value <0.01). To identify genes associated with the immune score, correlation between CNA, RNA, protein, and the immune score was performed for the genes present in both CNA and RNA in a cohort (20,313 genes in LSCC, 18,091 genes in LUAD, and 21,616 genes in HNSCC). in LSCC, 18,091 genes in LUAD, and 21,616 genes in HNSCC). The immune score was calculated as the z-score transformation of the ESTIMATE Immune score, which was calculated for all three cohorts as described in the method Estimation of Tumor Purity, cohorts as described in the method Estimation of Tumor Purity, Stromal and Immune Scores. To be considered copy number drivers of the immune score, genes had to have a positive correlation between CNA & RNA, CNA & immune score, and RNA & immune score. If CNA & RNA, CNA & immune score, and RNA & immune score. If the protein was quantified in the cohort, the gene was also required to have a positive correlation between CNA & protein and protein & immune score. Spearman correlations with a BH adjusted p & immune score. Spearman correlations with a BH adjusted p value < 0.01 were considered significant. Supplementary Material\\n\\nDeclaration of Interests\\n\\nThe authors declare no competing interests\\n\\nADDITIONAL RESOURCES ADDITIONAL RESOURCES\\n\\nThe CPTAC program website, detailing program initiatives, investigators, and datasets, is found at https://proteomics.cancer.gov/programs/cptac. A website for interactive visualization of the multi-omic dataset is available at: https://rstudio-connect.broadapps.org/CPTAC-LSCC2021/ All processed data matrices are available at LinkedOmics (http://www.linkedomics.org), where computational tools are available for further exploration of this dataset.\\n\\nConsortia Alex Green, Alexey Nesvizhskii, Alfredo Molinolo, Alicia Francis, Amanda Paulovich, Ana Robles, Andrii Karnuta, Antonio Colaprico, Arul Chinnaiyan, Azra Krek, Barbara Hindenach, Barbara Pruetz, Bartosz Kubisa, Bing Zhang, Bo Wen, Boris Reva, Brian Druker, Bartosz Kubisa, Bing Zhang, Bo Wen, Boris Reva, Brian Druker, Carissa Huynh, Chandan Kumar-Sinha, Charles Goldthwaite Jr., Chelsea Newton, Chen Huang, Chet Birger, Chia-Kuei Mo, Christopher Kinsinger, Corbin Jones, DR Mani, Dan Rohrer, Dana Valley, Daniel Corbin Jones, DR Mani, Dan Rohrer, Dana Valley, Daniel Chan, David Chesla, David Feny\\u00f6, David Heiman, Dmitry Rykunov, Donna Hansel, Elena Ponomareva, Elizabeth Duffy, Eric Burks, Eric Jaehnig, Eric Schadt, Erik Bergstrom, Eugene Fedorov, Eunkyung An, Fei Eric Schadt, Erik Bergstrom, Eugene Fedorov, Eunkyung An, Fei Ding, Felipe da Veiga Levoprost, Fernanda Martins Rodrigues, Francesca Petralia, Gad Getz, Galen Hostetter, George Wilson, Gilbert Omenn, Henry Rodriguez, Houxiang Zhu, Hui Zhang, Jackson Omenn, Henry Rodriguez, Houxiang Zhu, Hui Zhang, Jackson White, James Suh, Jennifer Eschbacher, Jennifer Maas, Jonathan Lei, Jiayi Ji, Kai Li, Karen Christianson, Karen Ketchum, Karin Rodland, Karl Clauser, Karsten Krug, Katherine Hoadley, Kei Suzuki, Karl Clauser, Karsten Krug, Katherine Hoadley, Kei Suzuki, Kelly Ruggles, Ki Sung Um, Li Ding, Liqun Qi, Lori Bernard, Harry Kane, Maciej Wiznerowicz, MacIntosh Cornwell, Ma\\u0142gorzata Wojty\\u015b, Marcin Cieslik, Marcin Domagalski, Mathangi Thiagarajan, Matthew Cieslik, Marcin Domagalski, Mathangi Thiagarajan, Matthew Ellis, Matthew Wyczalkowski, Maureen Dyer, Meenskshi Anurag, Mehdi Mesri, Melissa Borucki, Michael Birrer, Michael Gillette, Midie Xu, Mikhail Krotevich, Myvizhi Esai Selvan, Namrata Udeshi, Nancy Mikhail Krotevich, Myvizhi Esai Selvan, Namrata Udeshi, Nancy Roche, Nathan Edwards, Negin Vatanian, Neil Mucci, Nicollette Maunganidze, Nikolay Gabrovski, Olga Potapova, Oluwole Fadare, \\u00d6zg\\u00fcn Babur, Pamela Grady, Pankaj Vats, Paul Paik, Pei Wang, Peter Babur, Pamela Grady, Pankaj Vats, Paul Paik, Pei Wang, Peter McGarvey, Pierre Jean Beltran, Pushpa Hariharan, Qing Kay Li, Ramaswamy Govindan, Ratna Thangudu, Rebecca Montgomery, Richard Smith,Robert Welsh, Runyu Hong, Sailaja Mareedu, Samuel Payne, Smith,Robert Welsh, Runyu Hong, Sailaja Mareedu, Samuel Payne, Sandra Cottingham, Sara Savage, Saravana Dhanasekaran, Scott Jewell, Sendurai Mani, Seungyeul Yoo, Shankara Anand, Shankha Satpathy, Shayan Avanessian, Shilpi Singh, Shirley Tsang, Shuang Cai, Shayan Avanessian, Shilpi Singh, Shirley Tsang, Shuang Cai, Song Cao, Stacey Gabriel, Stephan Schurer, Steven Carr, Suhas Vasaikar, Tao Liu, Tara Hiltke, Tatiana Omelchencko, Thomas Bauer, Tobias Schraink, Vasileios Stathias, Volodymyr Sovenko, Warren Tobias Schraink, Vasileios Stathias, Volodymyr Sovenko, Warren Tourtellotte, Weiping Ma, Wenke Liu, William Bocik, Wohaib Hasan, Xiaoyu Song, Yifat Geffen, Yize Li, Yongchao Dou, Yuping Zhang, Yuxing Liao, Yvonne Shutack, Zeynep Gumus, Zhen Zhang, Ziad Yuxing Liao, Yvonne Shutack, Zeynep Gumus, Zhen Zhang, Ziad Hanhan. This is a PDF file of an article that has undergone enhancements after acceptance, such as the addition of a cover page and metadata, and formatting for readability, but it is not yet the definitive version of record. This version will undergo additional version of record. This version will undergo additional copyediting, typesetting and review before it is published in its final form, but we are providing this version to give early visibility of the article. Please note that, during the production of the article. Please note that, during the production process, errors may be discovered which could affect the content, and all legal disclaimers that apply to the journal pertain. Proteogenomic Landscape of LSCC A. Schematic showing the number of tumors and NATs profiled and the various data types generated in this study. The lower panel represents data completeness. WGS: Whole Genome Sequencing, WES: Whole Exome Sequencing. CNA: copy number alteration. DNAme: Whole Exome Sequencing. CNA: copy number alteration. DNAme: DNA methylation. pSTY: phosphoproteome. Ac: acetylproteome. Ub: ubiquitylproteome. B. Stacked histograms indicating the distribution of patient phenotypes. Smoking History reflects self-report. C. Co-occuring mutation plot indicating cancer-relevant genes. MutSig-based significantly mutated genes (SMGs, q-value < 0.1) in this dataset are highlighted in red font. D. Heatmaps showing correlation between copy number alterations (CNA) and RNA (left) or proteomics (right). Red and green events represent significant (FDR <0.01) positive and negative correlations, respectively. E. Flow chart for identification of cancer-associated genes (CAGs) that showed GISTIC-based focal amplification or deletion (q<0.25) and cis-effects in both mRNA and protein (FDR<0.05). F. Differential protein expression (Log2 fold-change (FC)) in tumors with and without high-level amplification of the FGFR1 gene (GISTIC thresholded value =2). G. Genes whose DNA methylation was significantly associated with cascading cis regulation of their cognate mRNA expression, protein level, phosphopeptide and acetylpeptide abundance. Shapes indicate the cis-effects across the indicated datasets. Named indicate the cis-effects across the indicated datasets. Named genes also showed differential expression between tumors and NATs. See also Figure S1 and Table S1\\u20133\\n\\nLSCC Molecular Subtypes and Associations\\n\\nA. NMF-based clustering of tumor CNA, RNA, protein, phosphosite and acetylsite profiles, showing five primary NMF subtypes (top sample annotation row). B. Heatmap representing significantly enriched pathways (MSigDB Hallmark) in five multi-omic subtypes. C. Kaplan-Meier plot comparing survival probability of patients whose tumors were core members of a specific NMF subtype (NMF Core) to those whose tumors had characteristics of more than one NMF subtype (NMF mixed). D. Heatmap showing relative overexpression of mesenchymal proteins in the EMT-E subtype compared to others. E. Correlation between ssGSEA-based enrichment of EMT (Hallmark genesets) and fibroblast proliferation (GO: Gene Ontology) genesets (Pearson correlation=0.65, p=2.8\\u00d710\\u221214). F. Distribution of RTK correlation-based phosphosite enrichment (RTK CBPE) scores for PDGFRB and ROR2 across the five NMF subtypes. Wilcoxon p values for CBPE scores in EMT-E vs other subtypes are 1.5\\u00d710\\u22126 for PDGFRB and 2.7\\u00d710\\u22127 for ROR2. G. Proteins and phosphosites significantly associated with PDGFRB or ROR2 CBPE scores, known to play a role in EMT and extracellular matrix reorganization. The left panel shows Spearman correlation between PDGFRB CBPE scores and protein/phosphosite correlation between PDGFRB CBPE scores and protein/phosphosite abundance profiles. See also Figure S2 and Table S1\\u20134\\n\\nImpact of Somatic Mutations on Proteogenomic Features A. Significant (Wilcoxon FDR<0.05) cis- (circles) and trans-effects (squares) of selected mutations (x axis) on the expression of cancer-associated gene products, with mRNA in blue and proteins in green.\\n\\nB. Similar to panel A but showing phosphosites. B. Similar to panel A but showing phosphosites.\\n\\nC. Similar to panel A but showing acetylsites.\\n\\nD. Similar to panel A but showing ubiquitylsites. E. CNA data for CDKN2A and RB1 was used to classify tumors as having homozygous deletions or three classes of loss of heterozygosity mutations: nonsense/frameshift indel, missense/inframe indel, and splicing (see Table S4). CDKN2A genetic and indel, and splicing (see Table S4). CDKN2A genetic and hypermethylation annotations were based on the effect of the aberration on the p16INK4a (p16) gene product, but the effects of these CDKN2A/p16 aberrations on both major isoforms (p16INK4a and p14ARF aberrations on both major isoforms (p16INK4a and p14ARF (p14)) at the RNA (barplot) and protein (heatmap directly below barplot) levels are shown. Samples with amplification of CCND1\\u20133, CDK4, and CDK6 were assessed by GISTIC (threshold = 2), and the CDK4, and CDK6 were assessed by GISTIC (threshold = 2), and the genomic status, protein, and phosphoprotein levels for RB1 are included. Also shown are RNA-based scores for the cell cycle (MGPS, the mean of cell cycle genes and E2F target and G2M (MGPS, the mean of cell cycle genes and E2F target and G2M checkpoint gene set scores derived from ssGSEA of Hallmark gene sets) and phosphosite-based CDK kinase activity scores for CDKs 1, 2, and 4 derived from single sample post translational CDKs 1, 2, and 4 derived from single sample post translational modifications - signature enrichment analysis (ssPTM-SEA) of known kinase targets. Three tumors with copy number gain of CDKN2A are not included. F. Correlation between differential regulation of protein abundance (Log2 Fold-change (FC)) versus phosphoprotein log2 FC in tumors with NRF2 pathway mutation (one or two mutations in KEAP1, CUL3, or NFE2L2) versus NRF2 WT tumors (no NRF2 pathway KEAP1, CUL3, or NFE2L2) versus NRF2 WT tumors (no NRF2 pathway aberration). G. NRF2 pathway score and RNA, protein and phosphoprotein expression of key NRF pathway members according to NMF subtype.\\n\\nH. CDK5 protein expression (Log2 TMT ratio) by NMF subtype. P-values are from the Anova test. I. PTM-SEA-derived normalized enrichment scores (NES) for pathways enriched in NRF2 pathway-mutated (Mutant) vs wild-type samples (WT) plotted against NES for pathways enriched in NMF Classical vs other subtypes. Significantly upregulated (FDR<0.05) vs other subtypes. Significantly upregulated (FDR<0.05) PTM-SEA terms in the Classical subtype are indicated by red dots and labeled. See also Figure S3 and Table S4\\n\\nProteogenomic Impact of Chromosome 3q Amplification\\n\\nA. Differential protein expression (Log2 FC) between tumors and NATs for genes on chromosome arm 3q. B. Frequency distribution of \\u0394Np63\\u03b1 RNA expression in tumors and NATs. C. Differential protein expression in samples classified as \\u0394Np63\\u03b1-low vs -high based on \\u0394Np63\\u03b1 transcript level. The outlier upregulated gene product in red is BIRC5, also known as survivin. D. Differential expression of microRNAs in \\u0394Np63\\u03b1-low vs -high samples.\\n\\nE. Pearson correlation between expression of miR-205 and mRNA expression of its cognate, experimentally validated targets. F. Pearson correlation between expression of miR-205 and protein abundance of its cognate targets. G. Relationship between miR-205 expression (log2 TPM) and EMT (top, p=2.3\\u00d710\\u221208, Correlation = \\u22120.53) and DNA replication (bottom, p=1.1\\u00d710\\u221206, Correlation= 0.47) scores. H. Heatmap showing relative protein expression (TMT log ratio) of selected proteins with significant (FDR<0.01) Pearson correlation values (positive or negative) with SOX2 protein expression. I. Pearson correlation (p= 5.2\\u00d710\\u221207, Correlation = \\u22120.46) between SOX2 protein expression and HALLMARK_IL6_JAK-STAT_signaling NES.\\n\\nSee also Figure S4 and Table S4\\n\\nUbiquitylation landscape in LSCC A. Consensus clustering of protein ratio-corrected K-GG (di-glycine) site abundances in tumor samples and their associations. Heatmap shows only protein ratio-corrected K-GG sites with differential abundance across clusters (FDR<0.01). Enriched pathways abundance across clusters (FDR<0.01). Enriched pathways and molecular and clinical annotations are indicated. B. Number of K-GG sites showing significant correlations (FDR< 0.01) with E3 ligases. Shown are the five E3 ligases with the highest proportion of positive correlations. C. Pearson correlations between HERC5 protein expression and K-GG site protein-corrected abundance in key glycolytic enzymes PKM, PGK1, and ENO1. D. HERC5 protein expression (log2 TMT ratio) with samples grouped by immune subtype. Significant (Kruskal\\u2212Wallis, p = 2.8\\u00d710\\u221205) association is seen between HERC5 abundance and immune subtypes. E. Representative examples of significant spatial clustering of lysine acetylsites (purple) on PGK1 (left) and ACADVL (right) protein 3-D structure space-filling models (cyan) as determined by PTM CLUMPS. (PGK1 structure = PDB ID:3ZOZ. ACADVL structure = PTM CLUMPS. (PGK1 structure = PDB ID:3ZOZ. ACADVL structure = PDB ID:3B9). F. TXN protein levels in the NMF Classical subtype relative to NATs and other NMF subtypes (top left). Protein-corrected ubiquitylation (K-GG) sites are decreased on TXN1 in tumor subtypes relative to NATs (top right). TXNIP is decreased in tumor subtypes to NATs (top right). TXNIP is decreased in tumor subtypes relative to NATs (lower left). TXN1 activator TXNRD1 is increased in the Classical subtype relative to NATs and other NMF subtypes (lower right). Kruskal\\u2212Wallis p-values are indicated in the (lower right). Kruskal\\u2212Wallis p-values are indicated in the respective plots. G. Schematic representation of PTM-based modulation in LSCC tumors showing key enzymes in the metabolic and reactive oxygen species (ROS) pathways. Green and red arrows indicate higher and lower abundance of the corresponding PTMs in tumors. Putative lower abundance of the corresponding PTMs in tumors. Putative ISGylation targets of HERC5 are indicated by dotted lines. A known regulatory PKM phosphosite observed to be modulated in LSCC tumors is also highlighted. H. Lollipop charts showing the Log2 FC of acetylated (K-Ac) and ubiquitylated sites (K-GG) between tumors and NATs (Hyper: log2(FC)>1 or Hypo: <\\u22121, FDR<0.01). The upper panel shows specific sites that were hyper-ubiquitylated and hypoacetylated in tumors; that were hyper-ubiquitylated and hypoacetylated in tumors; the lower panel shows specific sites that were hyperacetylated and hypo-ubiquitylated in tumors. Dot colors indicate protein fold change between tumors and NATs. \\u201ck\\u201d represents modified lysine. I. Relative abundances of RAN K127 acetylation (K-Ac), ubiquitylation (K-GG) and RAN protein levels across NMF subtypes and NATs. Wilcoxon p-values are indicated above; ns represents p>0.05.\\n\\nSee also Figure S5 and Table S5\\n\\nImmune Landscape of LSCC A. Heatmaps illustrate cell type compositions and activities of selected individual genes/proteins and pathways across the four immune clusters: Hot, Warm, and Cold tumor and NAT-enriched. Successive heatmaps illustrate xCell immune signatures, mRNA and heatmaps illustrate xCell immune signatures, mRNA and protein expression of key immune-related markers, and ssGSEA pathway scores based on global proteomic data for biological pathways that were differentially regulated in immune groups based on both mRNA differentially regulated in immune groups based on both mRNA and global protein abundance (Common) or on global protein abundance alone (Proteomics only). B. Pathway scores of key pathways differentially expressed across the immune clusters. Wilcoxon p-values for the individual comparisons are provided on top. C. Contour plot of two-dimensional density based on Macrophage (x-axis) and CD8 T-cell scores (y-axis) showing the variation in these cell types\\u2019 distributions observed across the different immune clusters. D. Acetylsites differentially expressed between Hot and Cold tumors. Acetylsites of genes contained in the Hallmark Oxidative Phosphorylation pathway are highlighted in blue, ARHGDIB K135 is highlighted in red, and remaining sites are in gray. Darker is highlighted in red, and remaining sites are in gray. Darker color designates significant sites (FDR < 0.1). E. Regulation of Rho GTPase signaling including K135 acetylation of ARHGDIB. F. Global protein abundance of RAC2, DOCK2 and ELMO1, acetylproteome abundance of ARHGDIB K135k and phosphorylation abundance of ARHGEF6 at Serine 225 in immune clusters. Wilcoxon p-values are reported. G. RTK CBPE scores for 108 tumor samples and associated xCell signatures and pathway scores. The first heatmap shows CBPE scores of key RTKs, the second xCell signatures and the third pathway scores based on global protein abundance. H. Distribution of RTK CBPE scores for CSF1R, PDGFRB and FGFR2 stratified by immune clusters. Significance values (two-sided Wilcoxon test) between Hot clusters and combined Warm and Cold clusters are indicated on the violin plots. I. Heatmap showing proteins and phosphosites correlated with CSF1R CBPE scores that are known to be involved in immune evasion. See also Figure S6 and Table S6\\n\\nProteomic Features Related to Diagnosis, Prognosis, or Treatment A. Differentially expressed proteins between tumors and NATs. B. Significantly increased proteins (larger font indicates >4 fold) in the study LSCC cohort that are associated with poor overall survival (OS) or disease-free survival (DFS) in the TCGA LSCC cohort mRNA. C. Genetic dependencies of 502 proteins (log2 FC >2, FDR<0.01 and NAs <50%) in LSCC cell lines (n=16) profiled as part of the Achilles Dependency-Map project. D. Genes, ordered by their chromosomal location, that are deleted in at least 25% of the samples and significantly correlated to the immune score. Immune-related genes are highlighted. E. EGFR protein and tyrosine phosphorylation levels compared to EGFR copy number and an EGFR activity score (PROGENy). F. Heatmap showing Pearson correlation between the EGFR activity score represented by PROGENy (top) and RNA expression of EGFR ligands. *p<0.05 G. GO Biological Process enrichment for proteins with increased phosphorylation in EGFR amplified samples compared to non-amplified samples.\\n\\nH. Summary roadmap figure partitioned into five major categories, indicated by different colors. See also Figure S7 and Table S7 REAGENT or RESOURCE\\tSOURCE\\tIDENTIFIER\\t \\tAntibodies\\t \\tPTMScan\\u00ae Ubiquitin Remnant Motif (K-\\u03b5-GG) Kit\\tCell Signaling Technology\\tCatalog 5562\\t \\tPTMScan Acetyllysine Kit\\tCell Signaling Technology\\tCatalog: 13416\\t \\tCD4\\tDako\\tClone 4B12, RRID:AB_2728838 13416\\t \\tCD4\\tDako\\tClone 4B12, RRID:AB_2728838\\t \\tCD8\\tBio-Rad\\tClone 4B11, RRID:AB_322868\\t \\tCD163\\tAbcam\\tAb182422, RRID:AB_2753196\\t \\tARHGDIB\\tSigma-Aldrich\\tHPA051235, RRID:AB_2681398\\t \\tCK (pan-Cytokeratin)\\tDako\\tclone AE1/AE3, RRID:AB_2132885 CK (pan-Cytokeratin)\\tDako\\tclone AE1/AE3, RRID:AB_2132885\\t \\talpha-SMA\\tAbcam\\tab5694, RRID:AB_2223021\\t \\tBiological Samples\\t \\tPrimary tumor samples\\tSee Experimental Model and Subject Details\\tN/A\\t \\tChemicals and Reagents\\t \\tHPLC-grade water\\tJ.T. Baker\\tCatalog: Chemicals and Reagents\\t \\tHPLC-grade water\\tJ.T. Baker\\tCatalog: 4218-03\\t \\tUrea\\tSigma\\tCatalog: U0631\\t \\tSodium chloride\\tSigma\\tCatalog: 71376\\t \\t1M Tris, pH 8.0\\tInvitrogen\\tCatalog: AM9855G\\t \\tEthylenediaminetetraacetic acid\\tSigma\\tCatalog: E7889 AM9855G\\t \\tEthylenediaminetetraacetic acid\\tSigma\\tCatalog: E7889\\t \\tAprotinin\\tSigma\\tCatalog: A6103\\t \\tLeupeptin\\tRoche\\tCatalog: 11017101001\\t \\tPhenylmethylsulfonyl fluoride\\tSigma\\tCatalog: 78830\\t \\tSodium fluoride\\tSigma\\tCatalog: S7920\\t \\tPhosphatase inhibitor Sodium fluoride\\tSigma\\tCatalog: S7920\\t \\tPhosphatase inhibitor cocktail 2\\tSigma\\tCatalog: P5726\\t \\tPhosphatase inhibitor cocktail 3\\tSigma\\tCatalog: P0044\\t \\tDithiothretiol, No-Weigh Format\\tThermoScientific\\tCatalog: 20291\\t \\tIodoacetamide\\tSigma\\tCatalog: A3221 20291\\t \\tIodoacetamide\\tSigma\\tCatalog: A3221\\t \\tLysyl endopeptidase\\tWako Chemicals\\tCatalog: 129-02541\\t \\tSequencing-grade modified trypsin\\tPromega\\tCatalog: V511X\\t \\tFormic acid\\tSigma\\tCatalog: F0507\\t \\tAcetonitrile, LC-MS grade\\tHoneywell\\tCatalog: 34967 F0507\\t \\tAcetonitrile, LC-MS grade\\tHoneywell\\tCatalog: 34967\\t \\tAcetonitrile, anhydrous\\tSigma\\tCatalog: 271004\\t \\tTrifluoroacetic acid\\tSigma\\tCatalog: 302031\\t \\tTandem Mass Tag reagent kit \\u2013 11plex\\tThermoFisher\\tCatalog: A34808\\t \\t0.5M HEPES, pH 8.5\\tAlfa \\u2013 11plex\\tThermoFisher\\tCatalog: A34808\\t \\t0.5M HEPES, pH 8.5\\tAlfa Aesar\\tCatalog: J63218\\t \\tHydroxylamine solution, 50% (vol/vol) in H2O\\tAldrich\\tCatalog: 467804\\t \\tMethanol\\tHoneywell\\tCatalog: 34966\\t \\tAmmonium hydroxide solution, 28% (wt/vol) in 34966\\t \\tAmmonium hydroxide solution, 28% (wt/vol) in H2O\\tSigma\\tCatalog: 338818\\t \\tNi-NTA agarose beads\\tQiagen\\tCatalog: 30410\\t \\tIron (III) chloride\\tSigma\\tCatalog: 451649\\t \\tAcetic acid, glacial\\tSigma\\tCatalog: AX0073\\t \\tPotassium phosphate, acid, glacial\\tSigma\\tCatalog: AX0073\\t \\tPotassium phosphate, monobasic\\tSigma\\tCatalog: P0662\\t \\tPotassium phosphate, dibasic\\tSigma\\tCatalog: P3786\\t \\tMOPS\\tSigma\\tCatalog: M5162\\t \\tSodium hydroxide\\tVWR\\tCatalog: BDH7225\\t \\tSodium phosphate, dibasic\\tSigma\\tCatalog: BDH7225\\t \\tSodium phosphate, dibasic\\tSigma\\tCatalog: S9763\\t \\tPhosphate-buffered saline\\tFisher Scientific\\tCatalog: 10010023\\t \\tiVIEW DAB Detection Kit\\tRoche\\tCatalog: 760-091\\t \\tEquipment\\t \\tReversed-phase tC18 SepPak, 3cc 200mg\\tWaters\\tCatalog: WAT054925 tC18 SepPak, 3cc 200mg\\tWaters\\tCatalog: WAT054925\\t \\tSolid-phase C18 disk, for Stage-tips\\tEmpore\\tCatalog: 66883-U\\t \\tStage-tip needle\\tCadence\\tCatalog: 7928\\t \\tStage-tip puncher, PEEK tubing\\tIdex Health & Science\\tCatalog: 1581\\t \\tPicoFrit LC-MS column\\tNew Health & Science\\tCatalog: 1581\\t \\tPicoFrit LC-MS column\\tNew Objective\\tCatalog: PF360-75-10-N-5\\t \\tReproSil-Pur, 120 \\u212b, C18-AQ, 1.9-\\u03bcm resin\\tDr. Maisch\\tCatalog: r119.aq\\t \\tNanospray column heater\\tPhoenix S&T\\tCatalog: PST-CH-20U\\t \\tColumn heater column heater\\tPhoenix S&T\\tCatalog: PST-CH-20U\\t \\tColumn heater controller\\tPhoenix S&T\\tCatalog: PST-CHC\\t \\t300 \\u03bcL LC-MS autosampler vial and cap\\tWaters\\tCatalog: 186002639\\t \\tOffline HPLC column, 3.5-\\u03bcm particle size, 4.6 um \\u00d7 250 mm\\tAgilent\\tCatalog: Custom 3.5-\\u03bcm particle size, 4.6 um \\u00d7 250 mm\\tAgilent\\tCatalog: Custom order\\t \\tOffline 96-well fractionation plate\\tWhatman\\tCatalog: 77015200\\t \\t700 \\u03bcL bRP fractionation autosampler vial\\tThermoFisher\\tCatalog: C4010-14\\t \\t700 \\u03bcL bRP fractionation autosampler C4010-14\\t \\t700 \\u03bcL bRP fractionation autosampler cap\\tThermoFisher\\tCatalog: C4010-55A\\t \\t96-well microplate for BCA\\tGreiner\\tCatalog: 655101\\t \\tMicroplate foil cover\\tCorning\\tCatalog: PCR-AS-200\\t \\tVacuum centrifuge\\tThermoFisher\\tCatalog: SPD121P-115 Vacuum centrifuge\\tThermoFisher\\tCatalog: SPD121P-115\\t \\tCentrifuge\\tEppendorf\\tCatalog: 5427 R\\t \\tBenchtop mini centrifuge\\tCorning\\tCatalog: 6765\\t \\tBenchtop vortex\\tScientific Industries\\tCatalog: SI-0236\\t \\tIncubating shaker\\tVWR\\tCatalog: 12620-942\\t \\t15 mL SI-0236\\t \\tIncubating shaker\\tVWR\\tCatalog: 12620-942\\t \\t15 mL centrifuge tube\\tCorning\\tCatalog: 352097\\t \\t50 mL centrifuge tube\\tCorning\\tCatalog: 352070\\t \\t1.5 mL microtube w/o cap\\tSarstedt\\tCatalog: 72.607\\t \\t2.0 mL microtube w/o cap\\tSarstedt\\tCatalog: 72.608 72.607\\t \\t2.0 mL microtube w/o cap\\tSarstedt\\tCatalog: 72.608\\t \\tMicrotube caps\\tSarstedt\\tCatalog: 72.692\\t \\t1.5 mL snapcap tube\\tThermoFisher\\tCatalog: AM12450\\t \\t2.0 mL snapcap tube\\tThermoFisher\\tCatalog: AM12475\\t \\tInstrumentation\\t \\tMicroplate Reader\\tMolecular AM12475\\t \\tInstrumentation\\t \\tMicroplate Reader\\tMolecular Devices\\tCatalog: M2\\t \\tOffline HPLC System for bRP fractionation\\tAgilent 1260\\tCatalog: G1380-90000\\t \\tOnline LC for LC-MS\\tThermoFisher\\tCatalog: LC140\\t \\tQ Exactive Plus Mass LC for LC-MS\\tThermoFisher\\tCatalog: LC140\\t \\tQ Exactive Plus Mass Spectrometer\\tThermoFisher\\tCatalog: IQLAAEGAAPFALGMBDK\\t \\tQ Exactive HF-X Mass Spectrometer\\tThermoFisher\\tCatalog: 0726042\\t \\tOrbitrap Fusion Lumos Tribrid Mass Spectrometer\\tThermoFisher\\tCatalog: Fusion Lumos Tribrid Mass Spectrometer\\tThermoFisher\\tCatalog: IQLAAEGAAPFADBMBHQ\\t \\tCritical Commercial Assays\\t \\tTruSeq Stranded Total RNA Library Prep Kit with Ribo-Zero Gold\\tIllumina\\tCatalog: RS-122-2301\\t \\tInfinium MethylationEPIC Kit\\tIllumina\\tCatalog: RS-122-2301\\t \\tInfinium MethylationEPIC Kit\\tIllumina\\tCatalog: WG-317-1003\\t \\tNextera DNA Exosome Kit\\tIllumina\\tCatalog: 20020617\\t \\tKAPA Hyper Prep Kit, PCR-free\\tRoche\\tCatalog: 07962371001\\t \\tBCA Protein Assay Kit\\tThermoFisher\\tCatalog: 23225\\t \\tDeposited Data Protein Assay Kit\\tThermoFisher\\tCatalog: 23225\\t \\tDeposited Data\\t \\tDepMap: Mutation\\tDepMapPublic 21Q1 (PMID: 31068700; Dataset doi:10.6084/m9.figshare.13681534.v1)\\thttps://depmap.org/portal/download/\\t \\tDepMap: Segmented copy number\\tDepMapPublic 21Q1 (PMID: DepMap: Segmented copy number\\tDepMapPublic 21Q1 (PMID: 31068700; Dataset doi:10.6084/m9.figshare.13681534.v1)\\thttps://depmap.org/portal/download/\\t \\tDepMap: Gene level copy number\\tDepMapPublic 21Q1 (PMID: 31068700; Dataset level copy number\\tDepMapPublic 21Q1 (PMID: 31068700; Dataset doi:10.6084/m9.figshare.13681534.v1)\\thttps://depmap.org/portal/download/\\t \\tDepMap: RNAseq (transcript isoform)\\tDepMapPublic 21Q1 (PMID: 31068700; Dataset (transcript isoform)\\tDepMapPublic 21Q1 (PMID: 31068700; Dataset doi:10.6084/m9.figshare.13681534.v1)\\thttps://depmap.org/portal/download/\\t \\tDepMap: Proteomics (RPPA)\\tCCLE 2019 (PMID: 31068700)\\thttps://depmap.org/portal/download/\\t \\tDepMap: CRISPR KO screen DepMap: CRISPR KO screen (combined)\\tDepMapPublic 21Q1 (bioRxiv 2020.05.22.110247; doi: https://doi.org/10.1101/2020.05.22.110247)\\thttps://depmap.org/portal/download/\\t \\tDepMap: shRNA screen (combined)\\tDEMETER2 Data v6 (PMID: DepMap: shRNA screen (combined)\\tDEMETER2 Data v6 (PMID: 30389920)\\thttps://depmap.org/portal/download/\\t \\tDepMap: GDSC drug screen\\tSanger GDSC 1 (PMID: 27397505)\\thttps://depmap.org/portal/download/\\t \\tDepMap: PRISM drug screen\\tPRISM Repurposing 19Q4 DepMap: PRISM drug screen\\tPRISM Repurposing 19Q4 Secondary Screen (PMID: 32613204)\\thttps://depmap.org/portal/download/\\t \\tPhosphoSitePlus\\t\\thttps://www.phosphosite.org\\t \\tConnectivity Map (CMAP)\\t\\thttps://www.broadinstitute.org/connectivity-map-cmap\\t \\tHuman Human Protein Atlas (HPA)\\t\\thttps://www.proteinatlas.org\\t \\tCT Antigen database\\t\\thttp://www.cta.lncc.br\\t \\tDependency map (DepMap)\\t\\thttps://depmap.org/portal/v3.3.8 is a GECKOv2 Achilles dataset\\t \\tLibrary of Integrated Network-based Cellular Signatures Library of Integrated Network-based Cellular Signatures (LINCS)\\t\\thttps://clue.io/dataExpanded CMap LINCS Resource 2020 (1/28/2021 update)\\t \\tCPTAC HNSCC cohort\\t\\thttps://cptac-data-portal.georgetown.edu/study-summary/S054\\t \\tCPTAC LSCC cohort\\tThis CPTAC LSCC cohort\\tThis study\\thttps://cptac-data-portal.georgetown.edu/study-summary/S063\\t \\tSoftware and Algorithms\\t \\tmethylationArrayAnalysis (version 3.9)\\t\\thttps://master.bioconductor.org/packages/release/workflows/html/methylationArrayAnalysis.html Illumina EPIC methylation array (3.9)\\tHansen KD, 2019\\thttps://bioconductor.org/packages/release/data/annotation/html/IlluminaHumanMethylationEPICanno.ilm10b2.hg19.html\\t \\tMethylation array analysis pipeline for CPTAC\\tLi Ding Methylation array analysis pipeline for CPTAC\\tLi Ding Lab\\thttps://github.com/ding-lab/cptac_methylation\\t \\tmiRNA-Seq analysis pipeline for CPTAC\\tLi Ding Lab\\thttps://github.com/ding-lab/CPTAC_miRNA\\t \\tVEP\\t\\thttps://github.com/Ensembl/ensembl-vep/tags VEP\\t\\thttps://github.com/Ensembl/ensembl-vep/tags\\t \\tTNScope / DNAScope (Sentieon)\\t\\tsentieon.com\\t \\tvcfAnno\\t\\thttps://github.com/brentp/vcfanno\\t \\tVariantAnnotation (Bioconductor)\\t\\thttps://bioconductor.org/packages/release/bioc/html/VariantAnnotation.html arriba_v1.1.0\\t\\thttps://github.com/suhrig/arriba/\\t \\tfusioncatcher_v1.10\\t\\thttps://github.com/ndaniel/fusioncatcher/blob/master/doc/manual.md\\t \\teQTLGen\\t\\thttps://github.com/molgenis/systemsgenetics/wiki/eQTL-mapping-analysis-cookbook-(eQTLGen) Pindel0.2.5\\t\\thttp://gmt.genome.wustl.edu/packages/pindel/\\t \\tSignatureAnalyzer\\t\\thttps://software.broadinstitute.org/cancer/cga/msp\\t \\tCNVEX\\tMarcin Cieslik Lab\\thttps://github.com/mctp/cnvex\\t \\tCRISP\\tMarcin Cieslik Lab\\thttps://github.com/mctp/cnvex\\t \\tCRISP\\tMarcin Cieslik Lab\\thttps://github.com/mcieslik-mctp/crisp-build\\t \\tSpectrum Mill\\tKarl R. Clauser, Steven Carr Lab\\thttps://proteomics.broadinstitute.org/\\t \\tComBat Steven Carr Lab\\thttps://proteomics.broadinstitute.org/\\t \\tComBat (v3.20.0)\\t\\thttps://bioconductor.org/packages/release/bioc/html/sva.html\\t \\tgPCA\\t\\thttps://cran.r-project.org/web/packages/gPCA/index.html gPCA\\t\\thttps://cran.r-project.org/web/packages/gPCA/index.html\\t \\tGISTIC2.0\\t\\tftp://ftp.broadinstitute.org/pub/GISTIC2.0/GISTIC_2_0_23.tar.gz\\t \\tiProFun\\t\\thttps://github.com/WangLab-MSSM/iProFun iProFun\\t\\thttps://github.com/WangLab-MSSM/iProFun\\t \\tESTIMATE\\t\\thttps://bioinformatics.mdanderson.org/public-software/estimate/\\t \\tWebGestaltR\\t\\thttp://www.webgestalt.org/\\t \\tGSVA\\t\\thttps://bioconductor.org/packages/release/bioc/html/GSVA.html TSNet\\t\\thttps://github.com/WangLab-MSSM/TSNet\\t \\txCell\\t\\thttp://xcell.ucsf.edu/\\t \\tCPTAC LSCC Data Viewer\\tSteven Carr lab\\thttps://rstudio-connect.broadapps.org/CPTAC-LSCC2021/ Carr lab\\thttps://rstudio-connect.broadapps.org/CPTAC-LSCC2021/\\t \\tConsensusClusterPlus\\t\\thttp://bioconductor.org/packages/release/bioc/html/CancerSubtypes.html\\t \\tMS-GF+\\t\\thttps://github.com/MSGFPlus/msgfplus\\t \\tNeoFlow\\t\\thttps://github.com/bzhanglab/neoflow NeoFlow\\t\\thttps://github.com/bzhanglab/neoflow\\t \\tnetMHCpan\\t\\thttp://www.cbs.dtu.dk/services/NetMHCpan/\\t \\tOptitype\\t\\thttps://github.com/FRED-2/OptiType\\t \\tCustomprodbj\\t\\thttps://github.com/bzhanglab/customprodbj\\t \\tPDV\\t\\thttps://github.com/wenbostar/PDV PDV\\t\\thttps://github.com/wenbostar/PDV\\t \\tPepQuery\\t\\thttp://pepquery.org\\t \\tPTM-SEA\\t)\\thttps://github.com/broadinstitute/ssGSEA2.0\\t \\tPTMsigDB\\t)\\thttp://prot-shiny-vm.broadinstitute.org:3838/ptmsigdb-app/\\t \\tTerra\\tBroad Institute data science Terra\\tBroad Institute data science platform.\\thttps://terra.bio/\\t \\tPanoply\\tBroad Institute Proteomics Platform\\thttps://github.com/broadinstitute/PANOPLY\\t \\tCMap\\t\\thttps://clue.io/cmap\\t \\tLIMMA v3.36 (R CMap\\t\\thttps://clue.io/cmap\\t \\tLIMMA v3.36 (R Package)\\t\\thttps://bioconductor.org/packages/release/bioc/html/limma.html\\t \\tFactoMineR v1.41NMF(R - package)\\t\\thttps://cran.r-project.org/web/packages/FactoMineR/index.html\\t \\tMClust v5.4 (R package)\\t(Scrucca, MClust v5.4 (R package)\\t(Scrucca, Fop, Murphy and Raftery, 2017)\\thttps://cran.r-project.org/web/packages/mclust/index.html\\t \\tg:Profiler\\t\\thttps://biit.cs.ut.ee/gprofiler/gost\\t \\tCytoscape\\t\\thttps://cytoscape.org/ Cytoscape\\t\\thttps://cytoscape.org/\\t \\tImmuneSubtypeClassifier\\t\\thttps://github.com/CRI-iAtlas/ImmuneSubtypeClassifier\\t \\tProteinPaint\\t\\thttps://pecan.stjude.doud/proteinpaint/\\t \\tOrdinal\\tChristensen RHB (2019)\\thttps://CRAN.R-project.org/package=ordinal\",\n",
            "                    \"pdc_study_id\": \"PDC000234\",\n",
            "                    \"title\": \"A proteogenomic portrait of lung squamous cell carcinoma\"\n",
            "                },\n",
            "                {\n",
            "                    \"full_text\": \"Integrated Glycoproteomic Characterization of Clear Cell Renal Cell Carcinoma\\n\\nSummary Clear cell renal cell carcinoma (ccRCC), a common form of RCC, is responsible for the high mortality rate of kidney cancer. Dysregulations of glycoproteins have been shown to associate with ccRCC. However, the molecular mechanism has not been well with ccRCC. However, the molecular mechanism has not been well characterized. Here, a comprehensive glycoproteomic analysis was conducted using 103 tumors and 80 paired normal adjacent tissues. Altered glycosylation enzymes and corresponding protein Altered glycosylation enzymes and corresponding protein glycosylation were observed, while two of the major ccRCC mutations, BAP1 and PBRM1, showed distinct glycosylation profiles. Additionally, inter-tumor heterogeneity and cross-correlation between inter-tumor heterogeneity and cross-correlation between glycosylation and phosphorylation were observed. The relation of glycoproteomic features to genomic, transcriptomic, proteomic, and phosphoproteomic changes reveals roles of glycosylation in ccRCC changes reveals roles of glycosylation in ccRCC development with potential for therapeutic interventions. This study reports a large-scale TMT-based quantitative glycoproteomic analysis of ccRCC which can serve as a valuable resource for the community. Introduction Renal cell carcinoma (RCC) is a highly heterogeneous cancer of which malignant cells were formed from renal tubular epithelial cells. There are estimated 79,000 new cases and 13,920 deaths in 2022 with kidney and renal pelvis cancer in United States. in 2022 with kidney and renal pelvis cancer in United States. Among the various subtypes of RCC, clear cell renal cell carcinoma (ccRCC) represents the most common form of RCC (~75%) and is responsible for the high mortality rate of kidney cancer. The responsible for the high mortality rate of kidney cancer. The primary treatment for localized ccRCC tumors is surgical resection because conventional radiotherapy and chemotherapy have a little effect. Although, several Food and Drug Administration a little effect. Although, several Food and Drug Administration (FDA)-approved agents have been applied to patients with advanced RCC targeting various intracellular pathways (e.g., mTOR), the patients\\u2019 response to these treatments is still limited. the patients\\u2019 response to these treatments is still limited. Therefore, it is essential to understand the molecular alteration associated with the development and the progression of ccRCC to derive suitable therapeutic interventions for the management of derive suitable therapeutic interventions for the management of ccRCC patients. Our previous study characterized ccRCC by utilizing proteogenomic approach, which enhanced our understanding of the impact of genomic alterations of ccRCC on protein networks the impact of genomic alterations of ccRCC on protein networks and signaling pathways. Dysregulations of glycoproteins have been shown to associate with ccRCC development . However, the molecular mechanism has not been well characterized. Glycosylation is one of the most prevalent protein modifications. Dysregulation of glycoproteins have been shown to affect biological functions or disease development. Many currently FDA-approved biomarkers for cancer diagnosis and monitoring are FDA-approved biomarkers for cancer diagnosis and monitoring are glycoproteins, such as fucosylated alpha-fetoprotein for hepatocellular carcinoma, CA125 for ovarian cancer, and CA19\\u20139 for pancreatic cancer. Glycoproteins are often found on the cell pancreatic cancer. Glycoproteins are often found on the cell surface or secreted from cells; thus, discovery of dysregulated glycoproteins/glycosylation events and their corresponding signaling pathways could potentially target the ccRCC cells for signaling pathways could potentially target the ccRCC cells for treatment. Owing to the challenges posted by the heterogeneity and complexity of glycosylation, large-scale glycoproteomic analysis has not been feasible until recently the advances in mass has not been feasible until recently the advances in mass spectrometry (MS)-based glycoproteomic technology. Glycoproteomic studies carried out on several cancer types have revealed the functions of glycosylation in cancer development. Thus, a the functions of glycosylation in cancer development. Thus, a comprehensive characterization of ccRCC using quantitative glycoproteomic approach would allow the investigation of the roles of protein glycosylation specific to ccRCC. In this study, we utilized TMT-based MS data to quantitatively analyze N-linked glycoproteins of 103 ccRCC tumors and 80 paired normal adjacent tissues (NATs) and quantified glycosite-specific glycosylation. Alteration in glycosylation was observed in glycosylation. Alteration in glycosylation was observed in ccRCC tumors compared to NATs with the responsible glycosylation enzymes were identified. Moreover, BAP1 and PBRM1 are two of the five major mutations of ccRCC that are mutually exclusive, where major mutations of ccRCC that are mutually exclusive, where BAP1-mutant tumors are often associated with poor clinical outcome. In this particular cohort, the aggressiveness of BAP1-mutant tumors may be explained by the variations in glycosylation tumors may be explained by the variations in glycosylation patterns that were found in comparison to the wildtype and PBRM1-mutant tumors. The inter-tumor heterogeneity of ccRCC was detected by subtyping the tumors into three distinct glyco-subtypes. by subtyping the tumors into three distinct glyco-subtypes. Furthermore, different protein modifications can co-occur on the proteins that a crosstalk/cross-correlation among the protein modifications on the same or different proteins can impact the modifications on the same or different proteins can impact the functional regulation of the proteins. Protein phosphorylation is another common protein modifications involved in many cellular processes. The association between N-linked glycosylation and processes. The association between N-linked glycosylation and phosphorylation is poorly understood. Thus, we performed an integrated analysis to predict whether crosstalk of glycosylation and phosphorylation occurred in ccRCC. We observed the association phosphorylation occurred in ccRCC. We observed the association between glycan types and phosphoproteins as well as revealing distinct cross-correlation clusters among the ccRCC tumors. Overall, aforementioned findings indicate the role of protein Overall, aforementioned findings indicate the role of protein glycosylation specific to ccRCC. Additionally, this dataset is a rich source for future studies focused on development of potential therapeutic interventions to treat the cancer. Results\\n\\nGlycoproteomic profile of ccRCC We investigated the glycoproteomic landscape of ccRCC by analyzing 103 treatment-na\\u00efve ccRCC tumors and 80 paired-matched NATs in the current study. Figure S1A shows the experimental workflow of generating proteomic, phosphoproteomic, and glycoproteomic of generating proteomic, phosphoproteomic, and glycoproteomic data for the study. The summarized clinical data can be found in Table S1, where detailed clinical information is in our previous publication. Intact glycopeptide abundances from all tissue samples, eight quality control (QC) samples, and five NCI-7 cell samples were measured in 23 TMT-10plexes. The results showed a large number of intact N-linked glycopeptides were identified in each TMT set intact N-linked glycopeptides were identified in each TMT set (Figure S1B). A total of 44,181 intact glycopeptides (FDR < 1%) from 1,429 glycoproteins were identified in at least one sample across 23 TMT sets. Majority of the intact glycopeptides were across 23 TMT sets. Majority of the intact glycopeptides were from the proteins in the extracellular matrix, cell surface, lysosome, and plasma member region (Figure S1C). For the downstream analysis, intact glycopeptides quantified in greater than 50% of intact glycopeptides quantified in greater than 50% of the samples were retained and analyzed. We examined four glycan categories according to the monosaccharide composition of N-glycans associated with the identified glycopeptides as follows: oligomannose (High-Man), fucosylated glycans (Fucose), sialylated glycans (Sialic, including glycans with (Fucose), sialylated glycans (Sialic, including glycans with sialylation only and those with both sialylation and fucosylation), and other glycoforms (Others) that did not fit the first three categories. The percentage of identified intact N-linked three categories. The percentage of identified intact N-linked glycopeptides according to glycoforms were 24.9 %, 24.3 %, 40.2 %, and 10.6% for High-Man, Fucose, Sialic, and Others, respectively. The reproducibility among inter-TMT experiments was The reproducibility among inter-TMT experiments was evaluated using the QC and NCI-7. Among the QC samples, the median Spearman correlation and the median coefficient of variation (CV) was 0.95 and 18.5%, respectively (Figures 1D\\u2013E, Table S1). Similar 0.95 and 18.5%, respectively (Figures 1D\\u2013E, Table S1). Similar results were observed for the NCI-7 samples with a median Spearman correlation of 0.95 and CV of 15.2% (Figures 1F\\u2013G, Table S1) indicating high reproducibility of the glycoproteomic data of indicating high reproducibility of the glycoproteomic data of ccRCC. Altered glycosylation in ccRCC compared to NATs We first analyzed quantitative glycoproteomic data to investigate whether ccRCC tissues showed distinct glycopeptide features in comparison to non-cancerous tissues. A clear separation between 103 tumors and 80 NATs was observed based on the abundances of 103 tumors and 80 NATs was observed based on the abundances of intact glycopeptides by principal component analysis (Figure 1A). Further carrying out the differential analysis between ccRCCs and NATs revealed a total of 2,064 intact glycopeptides with and NATs revealed a total of 2,064 intact glycopeptides with significantly larger than 2-fold changes (FDR < 0.01), where 625 upregulated and 1,439 downregulated in tumors relative to NATs (Figure 1B, Table S2). We found a high proportion of upregulated 1B, Table S2). We found a high proportion of upregulated intact glycopeptides with High-Man (44.8%) followed by Sialic (32.5%), whereas more fucosylated glycans (41.6%) were found in downregulated intact glycopeptides (Figure 1C). The differentially expressed intact glycopeptides originated from 317 glycoproteins, where 121 and 160 glycoproteins were unique to the upregulated and downregulated glycopeptides, respectively (Figure S2A). Unique glycoproteins were subjected to the KEGG (Figure S2A). Unique glycoproteins were subjected to the KEGG pathway enrichment analysis (Figure S2B). Pathways, such as extracellular matrix (ECM) receptor interaction, cell adhesion molecules (CAMs), and focal adhesion, were only enriched from the (CAMs), and focal adhesion, were only enriched from the glycoproteins of the upregulated glycopeptides. On the other hand, glycoproteins of downregulated glycopeptides were mainly from lysosome, complement and coagulation cascades, and cholesterol lysosome, complement and coagulation cascades, and cholesterol metabolism pathways. In addition, distinct glycan categories were observed in the glycopeptides of the glycoproteins mapped to the aforementioned pathways (Figure S2C). For instance, in the the aforementioned pathways (Figure S2C). For instance, in the cholesterol metabolism pathway, most of the downregulated glycopeptides were modified by fucosylated glycans, whereas high-mannose and sialylated glycans were found in the upregulated and sialylated glycans were found in the upregulated glycopeptides. The distinct glycan profiles for differentially expressed intact glycopeptides were potentially associated with glycosylation enzymes expressed in the tumors. The altered glycosylation enzymes could provide prognostic values and/or additional therapeutic could provide prognostic values and/or additional therapeutic targets for treating ccRCC. Upregulation of MGAT1 and MAN1B1 and downregulation of MAN1C1 and MAN1A1 were observed in tumors relative to NATs (Figure 1D, Table S2). Similar expression patterns to NATs (Figure 1D, Table S2). Similar expression patterns were found on the transcriptomic level (Figure S2D, Table S2). Decrease in MAN1C1 and MAN1A1 and elevation of MGAT1 may explain the high percentage of High-Man and complex glycans with sialic acid percentage of High-Man and complex glycans with sialic acid in tumors since MAN1C1 is responsible for trimming \\u03b1-1,2-linked mannose residues from N2H8 (N=HexNAc and H=Hex) to produce N2H5 along with MAN1A1 and MAN1A2, while MGAT1 relates to the synthesis with MAN1A1 and MAN1A2, while MGAT1 relates to the synthesis of complex glycans. Variation in glycosylation patterns between high-grade and low-grade ccRCC tumors In this study, we found glycopeptides with high-mannose and complex glycans with sialic acid were highly expressed in tumors compared to NATs (Figure 1C). Previous studies have shown that high-mannose or sialylated glycans are associated with cancer high-mannose or sialylated glycans are associated with cancer progression in different cancer types. It was intriguing to investigate whether glycosylation patterns varied between low-grade (G1 and G2) and high-grade (G3 and G4) ccRCC tumors. Among the (G1 and G2) and high-grade (G3 and G4) ccRCC tumors. Among the 220 differentially expressed intact glycopeptides (FDR<0.05) between low-grade and high-grade tumors, High-Man were dominated in high-grade tumors, where most of the high-mannose glycans were high-grade tumors, where most of the high-mannose glycans were N2H8 followed by N2H7 and N2H6 (Figure 1E, Table S2). We speculated that the elevated MAN1B1 (FDR<0.05) in high-grade relative to low-grade contributed to the High-Man patterns that were found low-grade contributed to the High-Man patterns that were found in high-grade tumors (Table S2). The upregulated intact glycopeptides with High-Man glycans were originated from glycoproteins with biological processes such as regulation of leukocyte activation (e.g., Transmembrane glycoprotein NMB, GPNMB), receptor-mediated endocytosis, (e.g., glycoprotein NMB, GPNMB), receptor-mediated endocytosis, (e.g., Scavenger receptor cysteine-rich type 1 protein M130, CD163), and integrin-mediated signaling pathway (e.g., Integrin alpha-V, ITGAV). GPNMB has been widely studied in various cancer types, ITGAV). GPNMB has been widely studied in various cancer types, which plays a role in metastasis of cancer and a potential therapeutic target for metastatic ccRCC. In this study, the global expression of GPNMB was unchanged regardless of tumor grade groups of GPNMB was unchanged regardless of tumor grade groups (Figure 1F, Table S2). Among the four identified glycosylation sites on GPNMB, majority of the glycosylation events showed no significant changes between high-grade and low-grade, except High-Man changes between high-grade and low-grade, except High-Man glycans (N2H7 and N2H8) on the glycosite, NDRN249SSDETFLK of GPNMB, showed significant elevation in high-grade compared to low-grade tumors (Figure 1F). On the other hand, low-grade tumors were enriched with Sialic glycans (Figure 1E). The sialylated glycopeptides were from glycoproteins that were associated with biological processes such as cell-substrate adhesion (e.g., Cadherin-13, CDH13) and such as cell-substrate adhesion (e.g., Cadherin-13, CDH13) and angiogenesis (e.g., Platelet endothelial cell adhesion molecule, PECAM1). In our previous work, PECAM1 was discovered as one of the gene signatures in VEGF immune desert subtype and this of the gene signatures in VEGF immune desert subtype and this particular immune subtype had better survival probability. We observed that sialylation was predominated on the four identified glycopeptides of PECAM1, where a significant downregulation of glycopeptides of PECAM1, where a significant downregulation of the glycopeptide, VN151CSVPEEK, attached with N5H6F1S1 (F=fucose and S=sialic acid) was observed in high-grade tumors (Figure S2E, Table S2). Glycosylation in tumors with specific mutations PBRM1 and BAP1 are two of the five commonly identified mutated genes in ccRCC that patients with BAP1 mutations tend to have poorer clinical outcomes than patients with PBRM1 mutations. Moreover, PBRM1 were found to be mutually exclusive with BAP1 Moreover, PBRM1 were found to be mutually exclusive with BAP1 mutations in ccRCC. Therefore, we investigated the glycosylation patterns in specific mutations. A comparative analysis was first conducted between mutant and wildtype tumors on both intact glycopeptide and global protein levels. The altered intact glycopeptides were mainly positively associated with the expression of the corresponding proteins on associated with the expression of the corresponding proteins on the global level for BAP1-mutant tumors (Figure 2A, Table S3) and PBRM1-mutant tumors (Figure 2B, Table S3) in comparison to BAP1 wildtype (BAP1-WT) and PBRM1 wildtype (PBRM1-WT) tumors, BAP1 wildtype (BAP1-WT) and PBRM1 wildtype (PBRM1-WT) tumors, respectively. However, elevation in High-Man and Sialic glycans was observed in BAP1-mutant, whereas PBRM1-mutant tumors tended to show upregulation in Fucose and Others glycans (>1.5-fold to show upregulation in Fucose and Others glycans (>1.5-fold changes with FDR < 0.05, Figure 2C, Table S3). Examination of the intact glycopeptides with altered abundance in BAP1-mutant tumors indicated the corresponding glycoproteins were associated with immunity, inflammation, and leukocyte migration, including Angiotensin-converting enzyme 2 (ACE2), migration, including Angiotensin-converting enzyme 2 (ACE2), Lysosome-associated membrane glycoprotein 2 (LAMP2), and Golgi apparatus protein 1 (GLG1) (top panel of Figure 2D, Table S3). On the other hand, the glycoproteins of differentially expressed the other hand, the glycoproteins of differentially expressed intact glycopeptides in PBRM1-mutant compared to PBRM1-WT tumors were related to metabolic process, including Agrin (AGRN), Glutamyl aminopeptidase (ENPEP), and Low-density lipoprotein Glutamyl aminopeptidase (ENPEP), and Low-density lipoprotein receptor-related protein 2 (LRP2). By directly comparing BAP1-mutant and PBRM1-mutant tumors, we further verified that the significantly enriched glycosylation events observed in the comparison enriched glycosylation events observed in the comparison to WT tumors were undoubtedly unique to the BAP1-mutant and PBRM1-mutant tumors (bottom panel of Figure 2D, Table S3). Subtyping ccRCC using glycoproteomics Immune-based and global proteomic-based subtyping of ccRCC were performed in our previous study. In this study, subtyping analysis was conducted using intact glycopeptides with CVs in > 25% quantile (8,111 intact glycopeptides) to examine the in > 25% quantile (8,111 intact glycopeptides) to examine the glycoproteomic-based inter-tumoral heterogeneity of ccRCC. Three glyco subtypes, Glyco 1 (n=33 tumors), Glyco 2 (n=33), and Glyco 3 (n=37), were identified (Figure 3A, Table S4). Glyco 1 tumors were characterized by high grade and stage tumors and a high frequency of chromosome 14 loss, genome instability, and a high frequency of chromosome 14 loss, genome instability, and BAP1 mutation. The relationship between protein-based and glyco-based ccRCC subtypes was delineated according to the enrichment scores and p-value (p<0.05) from a hypergeometric test, which scores and p-value (p<0.05) from a hypergeometric test, which revealed Glyco 1 tumors were associated with Global 1 subtype (Figure 3B). Similar analysis was performed between immune-based and glyco-based subtypes, we found that Glyco 1 overlapped with and glyco-based subtypes, we found that Glyco 1 overlapped with two immune subtypes, CD8+ inflamed and Metabolic immune-desert (Figure 3C). On the other hand, majority of the tumors in Glyco 2 and Glyco 3 with lower grade and stage compared to Glyco 1. 2 and Glyco 3 with lower grade and stage compared to Glyco 1. However, Glyco 2 overlapped with Global 2, while Glyco 3 showed a higher frequency of PBRM1 mutation as well as coincided with Global 3 and VEGF immune desert (Figures 3A\\u2013C). Four intact glycopeptide clusters (IPC) were established to investigate the association between the glyco subtypes and glycan types. As shown in Figure 3D, the intact glycopeptides classified in IPC 1 mainly contain Sialic and Fucose glycans. In contrast, in IPC 1 mainly contain Sialic and Fucose glycans. In contrast, IPC 2 was dominated by High-Man glycans, where IPC 3 was a cluster occupied mostly by intact glycopeptides with Fucose glycans. IPC 4 was the predominant cluster of Sialic glycans. To IPC 4 was the predominant cluster of Sialic glycans. To establish the relationship between the glyco subtypes and the glycan types, we utilized the glycans associated with IPCs and averaged Z-score of the intact glycopeptides from each of the four IPCs Z-score of the intact glycopeptides from each of the four IPCs (Figure 3E, Table S4). Intact glycopeptides in IPC 2 were highly expressed in Glyco 1 compared to the other IPCs and tumor subtypes; thus, tumors in Glyco 1 were considered as High-Man subtypes; thus, tumors in Glyco 1 were considered as High-Man associated tumors. Moreover, an increase in the expression of intact glycopeptides from IPC 3 was observed in Glyco 2, while lower in both Glyco 1 and Glyco 3. A high frequency of intact lower in both Glyco 1 and Glyco 3. A high frequency of intact glycopeptides was fucosylated in IPC3, therefore, Glyco 2 was regarded as Fucose-associated tumors. On the other hand, Glyco 3 showed an association with both IPC 1 and IPC 4. Since sialylated an association with both IPC 1 and IPC 4. Since sialylated glycopeptides were dominated in IPC 1 and IPC 4, especially in IPC4, Glyco 3 was considered as Sialic glycan associated glyco subtype. Furthermore, we conducted comparative analysis between a glyco subtype and the other subtypes to find intact glycopeptide signatures for each glyco subtype. To ensure the signatures were distinct for a particular subtype, each subtype was further compared for a particular subtype, each subtype was further compared to the NATs. Among the 8,111 intact glycopeptides, 150 intact glycopeptides with elevated expression profiles (>1.5-fold increase and FDR < 0.05) in Glyco 1 relative to Glyco 2, Glyco 3, and and FDR < 0.05) in Glyco 1 relative to Glyco 2, Glyco 3, and NATs. We also found 28 and 22 intact glycopeptides upregulated (>1.5-fold increase and FDR < 0.05) in Glyco 2 and Glyco 3, respectively (Table S4). Many of the glycopeptides were from 3, respectively (Table S4). Many of the glycopeptides were from glycoproteins related to immunity in Glyco 1 (e.g., CD63 antigen, CD63, Figure 3F). We observed glycopeptides from glycoproteins associated with urogenital system development (e.g., Laminin associated with urogenital system development (e.g., Laminin subunit alpha-5, LAMA5) and complement activation (e.g., Microfibril-associated glycoprotein 4, MFAP4) in Glyco 2 and Glyco 3, respectively (Figure 3F). The probability of patient survival based on the three glyco subtypes revealed that Glyco 1 tumors were associated with the worst patient survival among the three Glyco subtypes (Figure 3G). The low survival rate of Glyco 1 subtype correlates with the The low survival rate of Glyco 1 subtype correlates with the features that are regarded as poor prognosticators in ccRCC, including CD8+ inflamed immune subtype, higher frequency of BAP1 mutations, and high-grade tumors. In addition, the patient outcome and high-grade tumors. In addition, the patient outcome was evaluated based on the glycan types. We did not detect an association between survival and glycan types, except for High-Man of which high expression level of High-Man glycan type would result in high expression level of High-Man glycan type would result in poor survival probability (Figure 3H). Cross-correlation between glycosylation and phosphorylation in ccRCC Glycosylation and phosphorylation are two prevalent and broadly studied protein modifications, however, the cross-correlation between these two protein modifications is poorly understood. Therefore, we investigated whether there were certain interactions we investigated whether there were certain interactions between N-linked glycosylation and phosphorylation that were associated with ccRCC. It is intriguing to first explore whether glycan types affect the activation of the downstream signaling pathways of phosphoproteins in ccRCC. In this study, we classified the glycans into four major glycan types as aforementioned. The Spearman into four major glycan types as aforementioned. The Spearman correlation was calculated between each glycan type and phosphoproteins, while only correlation > \\u00b10.3 were selected for subsequent analysis. The associations between the glycan types and analysis. The associations between the glycan types and phosphoproteins could be classified into three different pathways, including metabolic pathways, RNA transport, and complement and coagulation cascades (Figure 4A, Table S5). Fucose glycans and other cascades (Figure 4A, Table S5). Fucose glycans and other glycans (i.e., none High-Man and Sialic) were positively correlated with phosphoproteins in the metabolic pathways, whereas High-Man glycans were associated with phosphoproteins in the RNA transport were associated with phosphoproteins in the RNA transport pathway. High-Man glycans can contribute to the spread of cancer cells; thus, High-Man glycans may alter the regulation of RNA transport enhancing cancer cell proliferation. Moreover, we found that enhancing cancer cell proliferation. Moreover, we found that Sialic glycans were positively associated with phosphoproteins from the pathway of complement and coagulation cascades, whereas the High-Man glycans were negatively correlated with this the High-Man glycans were negatively correlated with this particular pathway (Figure 4A). Studies showed that low complement activity was observed when the cell surface with abundant sialic acid, which enhanced cancer progression by allowing cancer cells which enhanced cancer progression by allowing cancer cells to avoid complement attack. We further performed linear regression between intact glycopeptides and phosphopeptides based on the aforementioned correlated glycan types and phosphoproteins with aforementioned correlated glycan types and phosphoproteins with consideration of the global expressions of corresponding glycoproteins and phosphoproteins. Potential connections among the phosphorylation and glycosylation events in metabolic pathways, RNA and glycosylation events in metabolic pathways, RNA transport, and complement and coagulation cascades were observed (Figures 4B, S3A\\u2013B, Table S5). For example, S555 of Complement C1r subcomponent (C1R) showed associations with glycopeptides from various (C1R) showed associations with glycopeptides from various glycoproteins, including those involved in neutrophil mediated immunity, such as Lysosome-associated membrane glycoprotein 1 (LAMP1), Haptoglobin (HP), and Adhesion G protein-coupled receptor E5 Haptoglobin (HP), and Adhesion G protein-coupled receptor E5 (ADGRE5). These observations suggest that the phosphorylation of C1R-S555 may depend on the glycosylation activities of presented glycoproteins (Figure 4B). Furthermore, some proteins had been detected with both phosphorylation and glycosylation activities that these proteins were found in different biological processes, including Epidermal growth factor receptor (EGFR), PECAM1, and kidney tissue-associated factor receptor (EGFR), PECAM1, and kidney tissue-associated proteins (e.g., LRP2). An upregulation of EGFR was observed in the majority of the ccRCC tumors, which made it a potential therapeutic target. Here, a linear relationship was noted between the target. Here, a linear relationship was noted between the phosphorylation at EGFR-S1018 and glycosylation at EGFR-N352 (N2H5) (Figure 4C, Table S5). The glycosylation at EGFR-N352 is reported to be essential for EGFR to maintain its functional is reported to be essential for EGFR to maintain its functional conformation to allow EGF binding. The phosphorylation of EGFR-S1018 is important for regulating desensitization, internalization and degradation of EGFR. The association between EGFR-S1018 and degradation of EGFR. The association between EGFR-S1018 and EGFR-N352 suggests a potential regulation of EGFR-S1018 activity via glycosylation of EGFR-N352. To further delineate the tumors based on the interactions among the intact glycopeptides and phosphopeptides, a multi-omics non-negative matrix factorization (NMF)-based clustering was conducted. Three Cross-Correlation clusters (CC 1\\u20133) were derived and Three Cross-Correlation clusters (CC 1\\u20133) were derived and overlaid with the Glyco subtypes (Figure 4D, Table S5). Intriguingly, a high percentage of CC 1 tumors (n=34) overlapped with Glyco 3, where most of the CC 2 tumors (n=35) were annotated as Glyco 3, where most of the CC 2 tumors (n=35) were annotated as Glyco 1 and CC 3 tumors (n=34) were associated with Glyco 2. Each CC cluster had its own unique features composed of both intact glycopeptides and phosphopeptides, which may reflect the intact glycopeptides and phosphopeptides, which may reflect the tumor-intrinsic biological differences among the CC clusters (Figure S3C, Table S5). Indeed, CC 1 cluster showed associations with extracellular matrix organization and receptor tyrosine with extracellular matrix organization and receptor tyrosine kinase-related signaling pathways. On the other hand, CC 2 cluster was identified as an immune-related cluster. CC 3 cluster was mapped to molecular transport-related pathways and negatively mapped to molecular transport-related pathways and negatively correlated with most of the pathways associated with CC 1 and CC 2 clusters. We further examined each CC cluster features that were differentially expressed (>1.5-fold change and FDR<0.05) compared to the other CC clusters (Table S5). In this study, an increase in FLT1-N251 with Sialic glycan (N4H5S1) was detected in CC 1 relative with Sialic glycan (N4H5S1) was detected in CC 1 relative to the other CC clusters (Figure 4E, Table S5). The glycosylation site, N251, is in the Ig-like domain 3 of FLT1 which is responsible for its ligand binding ability. Moreover, S373 of Alpha-enolase for its ligand binding ability. Moreover, S373 of Alpha-enolase (ENO1), downstream of FLT1 in the HIF-1 signaling pathway, was upregulated in CC1 compared to CC 2 and CC 3 (Figure 4E). Thus, we speculated that the elevation in FLT1-N251 and ENO1-S373 we speculated that the elevation in FLT1-N251 and ENO1-S373 expression might promote tumor vascularization and alter metabolic pathways under hypoxia condition driven by HIF-1. Majority of the CC 2 tumors overlapped with Glyco 1 subtype, which was correlated with CD8+ inflamed immune subtype (Figure 3C). Many significantly upregulated features of CC 2 tumors came from proteins involved in immune-related activities. We found from proteins involved in immune-related activities. We found elevated glycosylation at N240 of Cytochrome b-245 heavy chain (CYBB) with high-mannose glycan (N2H8) and phosphorylation at S102 of DEAD box protein 3, X-chromosomal (DDX3X) (Figure 4F, Table of DEAD box protein 3, X-chromosomal (DDX3X) (Figure 4F, Table S5). CYBB is expressed by myeloid cells predominately and produces reactive oxygen species for immune response to pathogens. CYBB can promote tumor growth and cancer metastasis which may serve can promote tumor growth and cancer metastasis which may serve as an immune checkpoint target. Phosphorylation of S102 of DDX3X mediates the activation of type I interferon signaling pathway. Studies have shown the association between DDX3X and cancer Studies have shown the association between DDX3X and cancer cell proliferation. Although the effects of glycosylation on CYBB-N240 and phosphorylation on DDX3X-S102 are not fully understood yet, however, we found that the global expressions of CYBB and yet, however, we found that the global expressions of CYBB and DDX3X were unchanged in CC 2 tumors. Therefore, CYBB-N240 and/or DDX3X-S102 could be potential therapeutic targets for CC 2 tumors. Furthermore, CC 3 was mapped to molecular transport-related biological processes involving proteins such as Serotransferrin (TF) and Hemoglobin subunit alpha (HBA2). HBA2 is responsible for oxygen transport and TF is an iron binding transport protein. oxygen transport and TF is an iron binding transport protein. Significant upregulation of glycosylation at TF-N630 (N3H6S1) and phosphorylation at HBA2-T138 were observed in CC 3 tumors compared to the remaining tumors (Figure 4G, Table S5). TF supplies to the remaining tumors (Figure 4G, Table S5). TF supplies irons to immature red blood cells to make hemoglobin. The N630 is a major glycosylation site for TF that sialylation can play a role in promoting tumor growth as observed in cholangiocarcinoma. in promoting tumor growth as observed in cholangiocarcinoma. Little is known about the phosphorylation of HBA2; however, we found several phosphosites on HBA2 were upregulated in CC 3 tumors relative to others in addition to T138 (Table S5). A positive relative to others in addition to T138 (Table S5). A positive correlation between phosphorylation and the function of the protein was possible in response to the oxidative stress by cancer cells. In summary, we investigated the possibility of crosstalk between glycosylation and phosphorylation in ccRCC. Various associations were observed between glycan types and phosphoproteins as well as between intact glycopeptides and phosphopeptides. Three CC as between intact glycopeptides and phosphopeptides. Three CC clusters were derived in addition to the Glyco subtypes. Furthermore, distinct feature profiles among ccRCC tumors were observed (Figure 4H) suggesting the potential of developing new observed (Figure 4H) suggesting the potential of developing new strategies to stratify the patients for treating ccRCC. Discussion In this study, 44,181 intact N-linked glycopeptides (corresponding to 1,429 glycoprotein) were identified and quantified from 103 ccRCC tumors and 80 paired NATs. The differences in significantly upregulated and downregulated intact glycopeptides were upregulated and downregulated intact glycopeptides were observed in glycan type distribution as well as the pathways associated with the corresponding glycoproteins in ccRCC tumors compared to NATs. We found that upregulated glycopeptides mostly to NATs. We found that upregulated glycopeptides mostly originated from glycoproteins involved in ECM-receptor interaction, cell adhesion molecules, or focal adhesion, which correlate well with the previous studies showing the role of N-linked well with the previous studies showing the role of N-linked glycosylation in cell adhesion and proliferation. Moreover, changes in glycan types/patterns may associate with the cancer progression. In this particular study, a high percentage of the progression. In this particular study, a high percentage of the upregulated glycopeptides contained High-Man followed by sialylated and fucosylated glycans. Increase in high-mannose glycans can impact the protein functions and enhance cancer progression. impact the protein functions and enhance cancer progression. The increased level of High-Man glycans indicated incomplete/premature termination of N-glycan biosynthesis, which was correlated to the downregulation of MAN1C1 in ccRCC as observed in our to the downregulation of MAN1C1 in ccRCC as observed in our data. The suppression of MAN1C1 is associated with decreased cell apoptosis in ccRCC and contributes to tumor invasion and metastasis. Intriguingly, we observed that intact glycopeptides with Intriguingly, we observed that intact glycopeptides with High-Man glycans were highly expressed in high-grade tumors, while elevated levels of sialylated glycopeptides were observed in low-grade tumors. Mutations on BAP1 and PBRM1 are two of the major genomic alteration events in ccRCC, while BAP1 mutation often leads to poor clinical outcome. The attached glycan structures of differentially expressed intact glycopeptides in BAP1-mutant tumors were expressed intact glycopeptides in BAP1-mutant tumors were different from PBRM1-mutant tumors. A high percentage of upregulated glycopeptides had High-Man glycans followed by Sialic glycans in BAP1-mutant tumors, whereas the expression levels of glycans in BAP1-mutant tumors, whereas the expression levels of fucosylated glycopeptides were increased in PBRM1-mutant tumors. Since high abundance of High-Man glycans can promote metastasis of cancer by enhancing the translocation and invasion of of cancer by enhancing the translocation and invasion of cancer cells into surrounding microenvironment, which may explain the aggressiveness of BAP1-mutant tumors compared to PBRM1-mutant tumors. Additionally, High-Man glycan can be used for survival tumors. Additionally, High-Man glycan can be used for survival prediction, while insignificant difference in patient survival was observed for other glycan types in this particular cohort. Significant inter-tumor heterogeneity was detected across ccRCC tumors using glycoproteome stratifying the tumors into three glycoproteomic subtypes. Each of the Glyco subtypes had a distinct profile that most of the features (e.g., BAP1 mutation, profile that most of the features (e.g., BAP1 mutation, High-Man glycans, and high-grade tumors) in Glyco 1 likely contributed to the poor patient outcome of this particular subtype through the survival analysis. By further exploring the glycosylation the survival analysis. By further exploring the glycosylation along with phosphorylation suggested potential crosstalk between the two protein modifications in ccRCC. The correlation between glycans and phosphoproteins revealed that sialic glycans were glycans and phosphoproteins revealed that sialic glycans were associated with phosphoproteins in complement and coagulation cascades. This result matched well with previous studies that lower complement activity was observed when cell surface was covered complement activity was observed when cell surface was covered with abundant sialic acid allowing the cancer cells to escape complement attack while enhancing the progression of cancer. On the other hand, high-mannose were correlated well with RNA On the other hand, high-mannose were correlated well with RNA transport. As aforementioned, High-Man glycans can promote the spread of cancer cells; thus, High-Man glycans may play a role in altering the regulation of RNA transport by increasing cancer altering the regulation of RNA transport by increasing cancer cell proliferation. Furthermore, cross-correlation among glycopeptides and phosphopeptides were derived using linear regression models indicating the possibility of regulating phospho-signaling pathways through N-linked glycosylation. For instance, association between N352 N-linked glycosylation. For instance, association between N352 and S1018 on EGFR was noted in our cohort. Glycosylation of N352 is important for stabilizing EGFR in a conformation which allows EGF binding. Although future studies will be required to allows EGF binding. Although future studies will be required to confirm our speculation, we hypothesized that the glycosylation of N352 will mediate the phosphorylation of S1018, which would affect the EGFR function since S1018 is responsible for the affect the EGFR function since S1018 is responsible for the desensitization, internalization and degradation of EGFR. By utilizing multi-omics NMF-based clustering, we were able to further differentiate the ccRCC tumors into three cross-correlation differentiate the ccRCC tumors into three cross-correlation clusters (CC1\\u20133) inferring interactions between glycosylation and phosphorylation. Variations in glycosylation patterns and phosphorylation events contributed to different biological pathways events contributed to different biological pathways that each CC cluster was associated, further suggesting the need for developing different diagnosis strategies and/or treatment options for ccRCC patients. This study describes glycoproteomic characterization of ccRCC using a large-scale clinical cohort with the largest number of quantified glycopeptides that has ever been reported for ccRCC. Current front-line treatment for advanced ccRCC either targets Current front-line treatment for advanced ccRCC either targets VEGF/mTOR or uses immunotherapy. Our results showed specific glycan types in association with cancer progression and aggressiveness along with the glycoproteomic subtyping and crosstalk of along with the glycoproteomic subtyping and crosstalk of glycosylation and phosphorylation may provide new approaches to stratify ccRCC patients, with the goal of developing diagnostic and/or treatment options in future. In summary, this study demonstrates that glycoproteome can expand our perception of the ccRCC by revealing altered biological events that were not captured in our previous proteogenomic analysis. Additionally, this study provides a rich resource to the Additionally, this study provides a rich resource to the ccRCC and broader scientific communities for translational research. Limitations of the study The objectives of this study were to comprehensively characterize ccRCC tumors and NATs using glycoproteomic approach as well as to understand the relation of glycoproteomic features to the changes observed on genomic, transcriptomic, proteomic, and the changes observed on genomic, transcriptomic, proteomic, and phosphoproteomic levels. Tissues collected by the Clinical Proteomic Tumor Analysis Consortium (CPTAC) program were treatment-naive and surgically resected to fulfill the above purposes. and surgically resected to fulfill the above purposes. Thus, there is a limitation of extrapolation to advanced/metastatic disease treated with systemic therapy since only treatment-na\\u00efve samples available in the present cohort. Furthermore, samples available in the present cohort. Furthermore, glycoproteomic data provide additional resources to investigate biological insights of ccRCC and allow investigation of glycoproteins that are located on cell surface, membrane, and secreted from that are located on cell surface, membrane, and secreted from cells. However, causal effects of correlated molecular alterations cannot be determined from this study and further validation using cell lines, patient-derived xenograft (PDX) models, or using cell lines, patient-derived xenograft (PDX) models, or clinical trials may be required. STAR METHODS\\n\\nRESOURCE AVAILABILITY\\n\\nLead contact\\n\\nFurther information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, T. Mamie Lih (tlih1@jhmi.edu)\\n\\nMaterials availability Materials availability\\n\\nThis study did not generate new, unique reagents\\n\\nData and code availability Clinical data and raw data files can be accessed via the Proteomic Data Commons (PDC) at: https://pdc.cancer.gov/ (PDC Study ID: PDC000127, PDC000128, PDC000471). All other data (e.g., processed data) can also be found in the PDC or in the supplementary data) can also be found in the PDC or in the supplementary materials. This paper does not report original code.\\n\\nAny additional information required to reanalyze the data reported in this paper is available in the STAR Methods or upon request.\\n\\nEXPERIMENTAL MODEL AND SUBJECT DETAILS\\n\\nTissue specimens The initial tissue cohort contained 110 RCC tumors, which was analyzed via proteogenomic in our previous work. Since the current study only focused on glycoproteomic analysis of ccRCC, therefore, a total of 103 histopathologically defined ccRCC cases with a total of 103 histopathologically defined ccRCC cases with an age range of 30\\u201390 were used in the downstream data analyses. This ccRCC cohort (n=103) contained 77 males and 26 females reflecting the gender distribution of ccRCC and the previous Clinical the gender distribution of ccRCC and the previous Clinical Proteomic Tumor Analysis Consortium (CPTAC) ccRCC cohort. All human specimens were existing specimens and subjects cannot be identified. Institutional review boards at each Tissue Source Site Institutional review boards at each Tissue Source Site (TSS) reviewed protocols and consent documentation, in adherence to CPTAC guidelines. The summarized clinical data can be found in Supplementary Table 1. Detailed clinical data can be accessed and Table 1. Detailed clinical data can be accessed and downloaded from the CPTAC Data Portal. METHOD DETAILS\\n\\nExperimental Design To conduct a comprehensive glycoproteomic analysis of ccRCC, the experimental workflow of generating global proteomic, phosphoproteomic, and glycoproteomic data of ccRCC for the current study are briefly described as following (Figure S1A): (i) proteins are briefly described as following (Figure S1A): (i) proteins from each cryopulverized tissue specimen were extracted and trypsin digested into peptides; (ii) peptides from each sample were labeled with TMT-10plex and then fractionated by basic were labeled with TMT-10plex and then fractionated by basic reversed-phase liquid chromatography (bRPLC); (iii) a portion of the TMT labeled peptides was used for global proteomic analysis and the remaining peptides were used for phosphopeptide enrichment the remaining peptides were used for phosphopeptide enrichment via immobilized metal affinity chromatography (IMAC). The flow through from the IMAC enrichment was further utilized for intact glycopeptide enrichment via mixed anion exchange (MAX) solid glycopeptide enrichment via mixed anion exchange (MAX) solid phase extraction column; (iv) The enriched glycopeptides were subjected to liquid chromatography-tandem mass spectrometry (LC-MS/MS) analysis in data-dependent acquisition (DDA) mode; (v) since analysis in data-dependent acquisition (DDA) mode; (v) since IMAC enrichment can capture both phosphopeptides and intact glycopeptides, therefore, we used GPQuest to identify and quantify the glycopeptides from both IMAC phospho-enriched data and MAX the glycopeptides from both IMAC phospho-enriched data and MAX glyco-enriched data. A total of 23 TMT-10plex sets were acquired and analyzed together in this study. Detailed information of materials and sample preparation (tryptic digestion, TMT labeling, and sample preparation (tryptic digestion, TMT labeling, and fractionation) can be found in our previous published work. Sequential enrichment of intact glycopeptides In this study, we utilized a sequential enrichment procedure to enrich intact glycopeptides after phosphopeptide enrichment as previously described. In brief, flow through and washing solution from IMAC enrichment process as described in our previous from IMAC enrichment process as described in our previous study were collected together to perform intact glycopeptide enrichment using MAX cartridge. Peptides were diluted with 80 % ACN in 0.1 % TFA and then incubated with Fe3+-NTA beads for 30 min at 0.1 % TFA and then incubated with Fe3+-NTA beads for 30 min at room temperature. The flow through from IMAC enrichment was collected by taking the supernatant of peptides-beads mixture after centrifuging for 1 min at 1000 \\u00d7 g. The pellet of peptides-beads for 1 min at 1000 \\u00d7 g. The pellet of peptides-beads was added into the C18 stage tip and then non-phosphopeptides were washed out by washing solution (80 % ACN in 0.1 % TFA). The \\u201cflow through\\u201d and \\u201cwashing solution\\u201d were dried together and stored at \\u22128\\u00b0C and \\u201cwashing solution\\u201d were dried together and stored at \\u22128\\u00b0C before MAX enrichment. To enrich glycopeptides, MAX cartridges were sequentially conditioned with 1mL of methanol (2 times), 1mL of ACN (2 times), 1mL of 100 mM triethylammonium acetate (3 times), 1 mL of water (Fisher chemical, LC/MS grade, 5 times), and 1 mL of 95% ACN (v/v) chemical, LC/MS grade, 5 times), and 1 mL of 95% ACN (v/v) 1% TFA (v/v) (5 times). Samples were diluted with 1 mL of 95 % ACN (v/v) 1 % TFA (v/v) and loaded onto cartridge twice. The non-glycopeptides were eluted out by 95% ACN (v/v) 1% TFA (v/v) * 3 were eluted out by 95% ACN (v/v) 1% TFA (v/v) * 3 times. Finally, glycopeptides were eluted by 50 \\u03bcL of 50% ACN (v/v) 0.1% TFA (v/v), dried and stored at \\u221280\\u00b0C prior to LC-MS/MS analysis. LC-MS/MS analysis Intact glycopeptides were analyzed by Orbitrap Fusion Lumos Tribrid\\u2122 (Thermo Scientific) combined with Easy nLC 1200 UPLC system (Thermo Scientific). Samples were reconstituted with 3 % ACN in 0.1 % formic acid (solvent A) and loaded onto an in-house in 0.1 % formic acid (solvent A) and loaded onto an in-house packed column (0.75 \\u03bcm I.D. \\u00d7 27.5 cm length packed with ReproSil-Pur 120 C18-AQ, 1.9 \\u03bcm). Loaded peptides were subjected to the gradient with 200 \\u03bcL/min of flow rate as follows: 2 to 6 % B (90 with 200 \\u03bcL/min of flow rate as follows: 2 to 6 % B (90 % ACN 0.1 % F.A) for 1 min, 6 to 30 % B for 84 min, 30 to 60 % B for 9 min, 60 to 90 % B for 1min, isocratic 90 % B for 5 min, 90 to 50 % B for 1 min, and isocratic 50 % B for 9 min. The temperature % B for 1 min, and isocratic 50 % B for 9 min. The temperature of the column was maintained by a column heater (Phoenix-ST) at 50\\u00b0C. The ion source of the mass spectrometer was set up with 1.8 kV of electrospray voltage and 305\\u00b0C of ion transfer tube 1.8 kV of electrospray voltage and 305\\u00b0C of ion transfer tube temperature. The precursor ion scan was acquired with 60 K resolution at 200 m/z for from 500 to 2000 m/z range and AGC value was set as 5\\u00d7105. Precursor ions isolated from 0.7 m/z width were set as 5\\u00d7105. Precursor ions isolated from 0.7 m/z width were fragmented by HCD with 35 % NCE and fragment ions were acquired with 50 K resolution with 1\\u00d7105 of AGC value for 100 ms of injection time for a total duty cycle (2 sec). The peptide charge time for a total duty cycle (2 sec). The peptide charge state screening was enabled to include 2 to 6+ ions with a dynamic exclusion time of 45 sec to discriminate against previously analyzed ions between +/\\u2212 10 ppm. QUANTIFICATION AND STATISTICAL ANALYSIS\\n\\nIdentification and quantification of N-linked intact glycopeptides The DDA raw files were first converted to mzML format via the ProteoWizard 3.0 with the Peak Picking option selected for all MS levels prior to the database searching against a customized N-linked glycopeptide database containing over 30,000 known N-linked glycopeptide database containing over 30,000 known glycosite-containing peptide sequences of human species and a glycan database containing 253 glycan compositions (GlycomeDB, http://www.glycome-db.org/) via GPQuest for the identification of via GPQuest for the identification of intact glycopeptides. The theoretical b-/y-ions of each targeted/decoy N-linked glycopeptide in the customized database were calculated and used as fragment ion index during database search. MS2 spectra were as fragment ion index during database search. MS2 spectra were preprocessed in a series of procedures including spectrum de-noising, oxonium ion evaluation, and glycan composition prediction. For each preprocessed potential MS2 spectrum of an intact For each preprocessed potential MS2 spectrum of an intact glycopeptide, the top 100 peaks were matched to the fragment ion index of candidate peptides in the database. A candidate peptide was retained if at least 6 peaks were matched. Further evaluations retained if at least 6 peaks were matched. Further evaluations were performed on the spectrum and the matched candidate peptides by assessing the isotopic peaks and fragment ions (b-/y-ions and Y-type ions) to compute Morpheus score, where the candidate and Y-type ions) to compute Morpheus score, where the candidate peptide with the highest Morpheus score was assigned to that particular MS2 spectrum. The glycan composition of the intact glycopeptide was determined by first computing the mass difference was determined by first computing the mass difference between the peptide sequence and the precursor mass followed by finding the glycan matching that mass difference in the glycan database. PSMs with summed b-/y-ion intensity less than 10% of total PSMs with summed b-/y-ion intensity less than 10% of total intensity of the spectrum were filtered out. FDR at PSM level was set at <1% to ensure the precise identification of the intact glycopeptides. Other parameters used in GPQuest as follows: 10 ppm Other parameters used in GPQuest as follows: 10 ppm for MS1 tolerance, 20 ppm for MS2 tolerance, maximum 1 missed cleavage and peptide length of 6 to 30 amino acids were allowed, TMT-10plex (N-terminal and lysine) and carbamidomethylation on cysteine set and lysine) and carbamidomethylation on cysteine set as fixed modifications, and oxidation on methionine set as variable modification. The intact glycopeptides were quantified using the report ions of TMT-10plex. The missing values were imputed (only for ions of TMT-10plex. The missing values were imputed (only for intact glycopeptides quantified in >50% of the 207 samples across the 23 TMT sets, where 11 samples were excluded from downstream data analyses for the reasons as described in our previous data analyses for the reasons as described in our previous publication) using DreamAI (https://github.com/WangLab-MSSM/DreamAI), which was the tool used in our previous study for the imputation of the phosphoproteomic data. All the downstream analyses of the phosphoproteomic data. All the downstream analyses were conducted in R (version 4.0). The processed data can be downloaded from the PDC. Quality control The reproducibility of 23 TMT-10plexes was evaluated using eight QC and five NCI-7 cell samples. Spearman correlation calculation along with the histograms and scatter plots were conducted using psych (version 2.0.9). The CV was calculated using goeveg using psych (version 2.0.9). The CV was calculated using goeveg (version 0.4.2) and plotted using ggplot2 (version 3.3.5). Differential analysis The differential analysis was carried out by calculating the median log2 fold changes between (i) tumors and NATs, (ii) low-grade and high-grade tumors, (iii) mutant tumors and NATs, (iv) mutant tumors and tumors without the particular mutation (iv) mutant tumors and tumors without the particular mutation (wildtype), (v) BAP1-mutant tumors and PBRM1-mutant tumors, (vi) one Glyco subtype and the remaining Glyco subtypes, and (vii) one Cross-Correlation (CC) cluster and the remaining CC clusters Cross-Correlation (CC) cluster and the remaining CC clusters on intact glycopeptide level as well as on transcriptomic, global proteomic, or phosphoproteomic level when applicable. The p-values were computed using two-sided Wilcoxon rank sum test and were computed using two-sided Wilcoxon rank sum test and adjusted (false discovery rate, FDR) using Benjamini-Hochberg method when applicable. Intact glycopeptides with >2-fold changes and FDR<0.01 were considered as significantly altered in tumors and FDR<0.01 were considered as significantly altered in tumors relative to NATs. Glycosylation enzymes (global proteomics and transcriptomics) with >1.5-fold changes and FDR<0.05 were considered as significantly altered in tumors compared to NATs. Intact as significantly altered in tumors compared to NATs. Intact glycopeptides with FDR<0.05 were used in high-grade and low-grade tumor comparison. Intact glycopeptides and/or global proteins with >1.5-fold changes and FDR<0.05 in mutant tumors relative to >1.5-fold changes and FDR<0.05 in mutant tumors relative to NATs were used in the comparison of mutant and wildtype. For comparisons (iv), (v), (vi), and (vii), differentially expressed intact glycopeptides were defined as >1.5-fold changes and FDR<0.05. Glycoproteomic subtyping Intact glycopeptides with CVs in the >25% quantile were utilized for subtyping analysis. By using CancerSubtypes for consensus clustering of tumor subtypes, 80% of the original sample pool was randomly subsampled without replacement and partitioned into randomly subsampled without replacement and partitioned into three major clusters (Glyco subtypes 1 to 3) via hierarchical clustering, which was repeated 2000 times. The subtyping heatmap was generated using ComplexHeatmap (version 2.4.3). The heatmap was generated using ComplexHeatmap (version 2.4.3). The consensus-clustered samples were overlaid with clinical features (e.g., grade, stage) and other omics subtypes derived in our previous publication (immune subtypes and global proteomic our previous publication (immune subtypes and global proteomic subtypes). The intact glycopeptides were grouped into four intact glycopeptide clusters (IPCs 1 to 4) using K-means clustering. Enrichment scores between immune-based/proteomic-based subtypes Enrichment scores between immune-based/proteomic-based subtypes in relation to the glycoproteomic-based subtypes were calculated according to the hypergeometric distribution. A hypergeometric test p-value < 0.05 was required for the consideration of test p-value < 0.05 was required for the consideration of overlap between different omic subtypes. Cross-correlation analysis The cross-correlation analysis between glycan types and phosphoproteins was based on the Spearman correlation. A correlation >\\u00b10.3 was considered as positively/negatively correlated between a glycan type and a phosphoprotein. For those correlated a glycan type and a phosphoprotein. For those correlated phosphoproteins and glycan types, the corresponding phosphopeptides and intact glycopeptides were subjected to linear regression analysis using lm function in R to examine the relation between analysis using lm function in R to examine the relation between glycosylation and phosphorylation. The resulting p-values (t-statistic) were adjusted via Benjamini-Hochberg method. Only linear regression models with FDR <0.01 were utilized to construct regression models with FDR <0.01 were utilized to construct the networks via Cytoscape (version 3.9.1). The linear regression was applied to phosphorylation and glycosylation events occurring on the same proteins as well. For the linear regression on the same proteins as well. For the linear regression analysis, only intact glycopeptides and phosphopeptides differentially expressed between tumors and NATs were considered. To further analyze the association between glycosylation and phosphorylation, the association between glycosylation and phosphorylation, we utilized non-negative matrix factorization (NMF)-based multi-omics clustering, which was conducted similar to our previous work on pancreatic ductal adenocarcinoma. Briefly, NMF was used to on pancreatic ductal adenocarcinoma. Briefly, NMF was used to perform unsupervised clustering of tumor samples using the abundances of phosphopeptides and intact glycopeptides. Only features (i.e., phosphopeptides and intact glycopeptides) with CVs in (i.e., phosphopeptides and intact glycopeptides) with CVs in >50% quantile were used for subsequent analysis. The feature matrix was scaled and standardized, thus all features were represented as Z-scores. Since NMF requires a non-negative input matrix, as Z-scores. Since NMF requires a non-negative input matrix, the feature matrix was further converted as follows: (i) create one data matrix with all negative numbers zeroed, (ii) create another data matrix with all positive numbers zeroed and the signs data matrix with all positive numbers zeroed and the signs of all negative numbers removed, and (iii) concatenate both matrices resulting in a data matrix with positive values and zeros only. The resulting matrix was then subjected to NMF analysis only. The resulting matrix was then subjected to NMF analysis leveraging the NMF R-package. To determine the optimal factorization rank k (i.e., number of clusters), a range of k from 2 to 10 was tested using default settings with 50 iterations. The to 10 was tested using default settings with 50 iterations. The optimal factorization rank k=3 (i.e., CC cluster) was selected since the product of cophenetic correlation coefficient and dispersion coefficient of the consensus matrix was the maximum dispersion coefficient of the consensus matrix was the maximum compared to other tested ks. The NMF analysis was repeated using 500 iterations for the optimal factorization rank k. A list of representative features for each CC cluster was derived that of representative features for each CC cluster was derived that differential analysis was carried out using cluster-specific features by comparing one CC cluster to the remaining CC clusters as described in the Differential Analysis section. Heatmap of CC described in the Differential Analysis section. Heatmap of CC clusters was generated using ComplexHeatmap (version 2.4.3), where subgroup hierarchical clustering was performed for the cluster-specific features, separately for intact glycopeptides and features, separately for intact glycopeptides and phosphopeptides. Pathway enrichment and Gene Ontology (GO) analysis All the pathway enrichment (KEGG/Reactome) and GO were performed via WebGestalt. The p-values were calculated using hypergeometric test and adjusted (FDR) using Benjamini-Hochberg method. For intact glycopeptides/phosphopeptides, we used the corresponding intact glycopeptides/phosphopeptides, we used the corresponding proteins to identify enriched pathways, biological processes, or cellular components. Survival analysis The Kaplan-Meier survival analysis was carried out and log-rank p-value was calculated using the survival (version 3.2\\u201313) R package. For the survival analysis of Glyco subtypes, the samples were differentiated based on their subtyping assignment. For the differentiated based on their subtyping assignment. For the survival analysis of glycan type (High-Man), the samples were divided into two groups based on the median abundance across 103 cases, where the samples with High-Man abundance above the median where the samples with High-Man abundance above the median were assigned to the High group and the remaining were considered as the Low group. Supplementary Material\\n\\nDeclaration of Interests\\n\\nAll other authors declare they have no competing interests. Comparative analysis to examine alter glycosylation in ccRCC. (A) Principal component analysis between ccRCC tumors and NATs. (B) Differential analysis between tumors and NATs. Significantly altered intact glycopeptides were defined as >2-fold changes altered intact glycopeptides were defined as >2-fold changes with false discovery rate (FDR)<0.01. (C) Distribution of glycan types for upregulated and downregulated intact glycopeptides. (D) Differential analysis of glycosylation enzymes on global level. Differential analysis of glycosylation enzymes on global level. Significantly altered glycosylation enzymes were defined as >1.5-fold changes with FDR<0.05. (E) Glycan types observed in differentially expressed intact glycopeptides (FDR<0.05) between expressed intact glycopeptides (FDR<0.05) between high-grade and low-grade tumors. (F) Glycosylation changes in NDRN249SSDETFLK of GPNMB in high-grade tumors relative to low-grade tumors (FDR<0.05 as significant changes). (B, D, E, and F) The p-values as significant changes). (B, D, E, and F) The p-values were computed using two-sided Wilcoxon rank sum test and adjusted (i.e., FDR) using Benjamini-Hochberg method. See also Figure S2 and Table S2. Alteration in glycosylation in BAP1-mutant and PBRM1-mutant tumors. (A) Median log2 fold change between intact glycopeptides and corresponding glycoproteins of BAP1-mutant and BAP1-WT tumors. (B) Median log2 fold change between intact glycopeptides and (B) Median log2 fold change between intact glycopeptides and corresponding glycoproteins of PBRM1-mutant and PBRM1-WT tumors. (C) Distribution of glycan types for upregulated and downregulated intact glycopeptides in BAP1-mutant and PBRM1-mutant compared intact glycopeptides in BAP1-mutant and PBRM1-mutant compared to wildtypes. (D) Biological processes of BAP1-mutant-associated and PBRM1-mutant-associated glycoproteins based on the differential analysis of intact glycopeptides of BAP1-mutant vs BAP1-WT, analysis of intact glycopeptides of BAP1-mutant vs BAP1-WT, PBRM1-mutant vs PBRM1-WT, and BAP1-mutant vs PBRM1-mutant. Significantly altered intact glycopeptides were defined as >1.5-fold changes with FDR<0.05. The p-values were computed using two-sided with FDR<0.05. The p-values were computed using two-sided Wilcoxon rank sum test and adjusted (FDR) using Benjamini-Hochberg method. See also Table S3. Inter-tumor heterogeneity of ccRCC. (A) Glycoproteomic subtyping of ccRCC. (B) Association between Glyco subtypes and global subtypes (* p-value<0.05, hypergeometric test). (C) Association between Glyco subtypes and immune subtypes (* p-value<0.05, between Glyco subtypes and immune subtypes (* p-value<0.05, hypergeometric test). (D) Glycan type distribution of each intact glycopeptide cluster (IPC). (E) Association among Glyco subtypes, glycan type, and IPC. (F) Examples of intact glycopeptide glycan type, and IPC. (F) Examples of intact glycopeptide signatures of each glyco subtype. (G) Survival analysis of Glyco subtypes. Log-rank test p=0.015 (H) Survival analysis based on abundance of High-Man where samples with abundance above median were of High-Man where samples with abundance above median were assigned to High group (n=51), otherwise assigned to Low group (n=52). Log-rank test p=0.013. See also Table S4. Cross-correlation between glycosylation and phosphorylation in ccRCC. (A) Correlation between glycan types and phosphoproteins. (B) Networks based on the linear models constructed using intact glycopeptides and phosphopeptides that were associated with glycopeptides and phosphopeptides that were associated with complement and coagulation cascades. (C) Association between EGFR-N352 (N2H5) and EGFR-S1018 with p=0.037 (t-statistic). (D) Cross-Correlation (CC) clusters derived using NMF multi-omic Cross-Correlation (CC) clusters derived using NMF multi-omic clustering. (E) Distinct glycosylation (FLT1-N251, N4H5S1) and phosphorylation (ENO1-S373) events in CC 1 tumors relative to the other CC clusters. (F) Distinct glycosylation (CYBB-N240, N2H8) other CC clusters. (F) Distinct glycosylation (CYBB-N240, N2H8) and phosphorylation (DDX3X-S102) events in CC 2 tumors relative to the other CC clusters. (G) Distinct glycosylation (TF-N630, N3H6S1) and phosphorylation (HBA2-T138) events in CC 3 tumors N3H6S1) and phosphorylation (HBA2-T138) events in CC 3 tumors relative to the other CC clusters. (H) Summarized key observations for ccRCC in this study. See also Figure S3 and Table S5. Key resources table REAGENT or RESOURCE\\tSOURCE\\tIDENTIFIER\\t \\tBiological Samples\\t\\t\\t \\tPrimary tumor and normal adjacent tissue samples\\tSee Experimental Model and Subject Details\\tN/A\\t \\t\\t\\t\\t \\tChemicals, Peptides, and Recombinant Proteins\\t \\tAprotinin\\tSigma\\tCatalog: A6103 and Recombinant Proteins\\t \\tAprotinin\\tSigma\\tCatalog: A6103\\t \\tLeupeptin\\tRoche\\tCatalog: 11017101001\\t \\tPhenylmethylsulfonyl fluoride\\tSigma\\tCatalog:93482\\t \\tSodium fluoride\\tSigma\\tCatalog: S7920\\t \\tPhosphatase Inhibitor Cocktail 2\\tSigma\\tCatalog: P5726 S7920\\t \\tPhosphatase Inhibitor Cocktail 2\\tSigma\\tCatalog: P5726\\t \\tPhosphatase Inhibitor Cocktail 3\\tSigma\\tCatalog: P0044\\t \\tUrea\\tSigma\\tCatalog: U0631\\t \\tTris(hydroxymethyl)aminomethane\\tInvitrogen\\tCatalog: AM9855G\\t \\tEthylenediaminetetraacetic AM9855G\\t \\tEthylenediaminetetraacetic acid\\tSigma\\tCatalog: E7889\\t \\tSodium chloride\\tSanta Cruz Biotechnology\\tCatalog: sc-295833\\t \\tPUGNAc\\tSigma\\tCatalog: A7229\\t \\tDithiothretiol\\tThermoFisher Scientific\\tCatalog: 20291\\t \\tIodoacetamide\\tThermoFisher Scientific\\tCatalog: 20291\\t \\tIodoacetamide\\tThermoFisher Scientific\\tCatalog: A3221\\t \\tSequencing grade modified trypsin\\tPromega\\tCatalog: V511X\\t \\tLysyl endopeptidase, Mass spectrometry grade\\tWako Chemicals\\tCatalog: 125\\u201305061\\t \\tFormic acid\\tFisher grade\\tWako Chemicals\\tCatalog: 125\\u201305061\\t \\tFormic acid\\tFisher Chemical\\tCatalog: A117\\u201350\\t \\tAcetonitrile\\tFisher Chemical\\tCatalog: A955\\u20134\\t \\tReversed-phase C18 SepPak\\tWaters\\tCatalog: WAT054925\\t \\t4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid\\tAlfa 4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid\\tAlfa Aesar\\tCatalog: J63218\\t \\tTandem mass tags \\u2013 10plex\\tThermoFisher Scientific\\tCatalog: 90110\\t \\tTrifluoroacetic acid\\tSigma\\tCatalog: 302031\\t \\tAmmonium Hydroxide solution\\tSigma\\tCatalog: 338818 302031\\t \\tAmmonium Hydroxide solution\\tSigma\\tCatalog: 338818\\t \\tHydroxylamine solution\\tAldrich\\tCatalog: 467804\\t \\tNi-NTA agarose beads\\tQIAGEN\\tCatalog: 30410\\t \\tIron (III) chloride\\tSigma\\tCatalog:451649\\t \\tOasis MAX Cartridge\\tWaters\\tCatalog: 186000366 Oasis MAX Cartridge\\tWaters\\tCatalog: 186000366\\t \\tTriethylammonium acetate buffer\\tSigma\\tCatalog: 90358\\t \\t\\t\\t\\t \\tCritical Commercial Assays\\t\\t\\t \\tBCA Protein Assay Kit\\tThermoFisher Scientific\\tCatalog: 23225\\t \\t\\t\\t\\t \\tDeposited data\\t\\t\\t \\tCPTAC ccRCC clinical data 23225\\t \\t\\t\\t\\t \\tDeposited data\\t\\t\\t \\tCPTAC ccRCC clinical data and proteomic data\\tThis manuscript; Clark et al.\\thttps://pdc.cancer.gov/ (PDC000127, PDC000128, PDC000471)\\t \\t\\t\\t\\t \\tSoftware and Algorithms\\t\\t\\t \\tProteowizard\\tChambers et Software and Algorithms\\t\\t\\t \\tProteowizard\\tChambers et al.\\thttp://proteowizard.sourceforge.net/\\t \\tGPQuest\\tHu et al.; Toghi Eshghi et al.; Toghi Eshghi et al.\\thttps://github.com/huizhanglab-jhu/GPQuest\\t \\tR v4.0\\tR Development Core R v4.0\\tR Development Core Team\\thttps://www.R-project.org\",\n",
            "                    \"pdc_study_id\": \"PDC000471\",\n",
            "                    \"title\": \"Integrated glycoproteomic characterization of clear cell renal cell carcinoma\"\n",
            "                },\n",
            "                {\n",
            "                    \"full_text\": \"A proteogenomic portrait of lung squamous cell carcinoma\\n\\nSummary Lung squamous cell carcinoma (LSCC) remains a leading cause of cancer death with few therapeutic options. We characterized the proteogenomic landscape of LSCC, providing a deeper exposition of LSCC biology with potential therapeutic implications. We of LSCC biology with potential therapeutic implications. We identify NSD3 as an alternative driver in FGFR1-amplified tumors and low-p63 tumors overexpressing the therapeutic target survivin. SOX2 is considered undruggable, but our analyses provide SOX2 is considered undruggable, but our analyses provide rationale for exploring chromatin modifiers such as LSD1 and EZH2 to target SOX2-overexpressing tumors. Our data support complex regulation of metabolic pathways by crosstalk between complex regulation of metabolic pathways by crosstalk between post-translational modifications including ubiquitylation. Numerous immune-related proteogenomic observations suggest directions for further investigation. Proteogenomic dissection of CDKN2A for further investigation. Proteogenomic dissection of CDKN2A mutations argue for more nuanced assessment of RB1 protein expression and phosphorylation before declaring CDK4/6 inhibition unsuccessful. Finally, triangulation between LSCC, LUAD and HNSCC Finally, triangulation between LSCC, LUAD and HNSCC identified both unique and common therapeutic vulnerabilities. These observations and proteogenomics data resources may guide research into the biology and treatment of LSCC. In Brief Comprehensive proteogenomic characterization of lung squamous cell carcinomas and paired normal adjacent tissues identifies taxonomic subclasses, alternative driver events and insights into immune modulation, as well as putative biomarkers and potential immune modulation, as well as putative biomarkers and potential therapeutic targets. Graphical Abstract\\n\\nIntroduction Lung cancer is the leading cause of cancer-associated mortality. Unlike lung adenocarcinomas (LUAD), patients with lung squamous cell carcinoma (LSCC) have not benefited from targeted therapies. Potentially druggable genetic events in three primary Potentially druggable genetic events in three primary pathways (FGFR1, PI3K, or G1/S checkpoint genes such as CDKN2A) are found in upwards of 60% but targeting these clinically has largely failed. While other therapeutic regimens are being explored as failed. While other therapeutic regimens are being explored as part of the LungMAP consortium, to date only immunotherapy has evolved into a successful therapeutic strategy for patients with LSCC. Given the lack of established actionable genomic targets, LSCC. Given the lack of established actionable genomic targets, a comprehensive characterization of the proteogenomic landscape of LSCC, including the associated post-translational modifications (PTMs) that are key to protein activity and signaling, is a (PTMs) that are key to protein activity and signaling, is a crucial step forward. Such studies will aid in connecting genomic aberrations to molecular and clinical phenotypes and in subsequently identifying therapeutic vulnerabilities and effective, identifying therapeutic vulnerabilities and effective, biomarker-based patient stratification. We characterized 108 prospectively-collected, treatment-na\\u00efve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs), identifying actionable therapeutic protein targets and elucidating cellular signaling pathways and crosstalk between multiple cellular signaling pathways and crosstalk between multiple PTMs. The dataset provides an exceptional resource to guide further research and support development of therapeutic modalities in LSCC. Results\\n\\nProteogenomic landscape of LSCC We performed deep-scale molecular analysis spanning nine different data types on LSCC tumors and NATs prospectively collected from 108 patients (Figure 1A, S1A, Table S1\\u20133). Proteomics data were generated using TMT11-based multiplexing linked by a common generated using TMT11-based multiplexing linked by a common reference (Figure S1A) and demonstrated high technical quality (Figure S1B\\u2013C). The cohort demographics are summarized in Figure 1B. Most self-reported never-smokers displayed high genomic smoking Most self-reported never-smokers displayed high genomic smoking scores (Figure S1D), highlighting occasional discrepancy between self-reported smoking status and genomic evidence, as previously reported. The genomic landscape reaffirmed previously reported. The genomic landscape reaffirmed previously reported somatic alterations in LSCC (Figure 1C). Principal component analysis (PCA) of global proteomic and PTM data showed clear separation of tumors and NATs (Figure S1E). Proteogenomic data helped annotate the impact of copy number alteration (CNA) events (Figure 1D,E). Of 5,523 significant (FDR<0.01) cis CNA-mRNA events, 2,154 were concordantly correlated with protein expression, including 138 \\u201ccancer associated genes\\u201d protein expression, including 138 \\u201ccancer associated genes\\u201d (CAG) (Figure 1E, Table S1). We identified six amplified (including WHSC1L1, CCND1, and SOX2) and 29 deleted (including NCOR1, SETD2, and CBL) CAGs in significant focal events (Q<0.25) (Figure and CBL) CAGs in significant focal events (Q<0.25) (Figure 1E, Table S4). Intriguingly, WHSC1L1 (NSD3) is part of a recurrent focal amplicon (8p11.23) in LSCC that encompasses FGFR1. Therapies targeted against FGFR1 have been unsuccessful. Proteomics data targeted against FGFR1 have been unsuccessful. Proteomics data suggested that NSD3, rather than FGFR1, could be the critical driver oncogene within this amplicon (Figure 1F), nominating it as a potential therapeutic target. While this paper was under as a potential therapeutic target. While this paper was under review, NSD3 was demonstrated to be a key regulator of LSCC tumorigenesis, mediating oncogenic chromatin changes. To investigate the impact of CNAs on noncognate gene products, we matched patterns of these significant trans-effects (vertical stripes in Figure 1D) to perturbation profiles from the Connectivity Map (CMap) (https://clue.io/cmap). Six CNAs (IKBKAP, PIN1, Map (CMap) (https://clue.io/cmap). Six CNAs (IKBKAP, PIN1, DNAJB1, IL18, NR2F6, AKAP) (Table S4) were enriched for both amplification and deletion (FDR=0.06) and associated (Fisher\\u2019s exact p<0.01) with clinical metadata. IL-18 deletion and amplification with clinical metadata. IL-18 deletion and amplification correlated with decreased and increased IL-18 protein expression and xCell immune score, respectively, (Figure S1F (upper panel)), consistent with its playing a key modulatory role in the tumor consistent with its playing a key modulatory role in the tumor microenvironment of NSCLC. NR2F6 amplification and deletion were correlated with NR2F6 protein expression and anticorrelated with xCell immune score (Figure S1F (lower panel)). NR2F6 acts as a xCell immune score (Figure S1F (lower panel)). NR2F6 acts as a non-redundant immune checkpoint in cancer, and even partial knockdown works synergistically with PD-L1 blockade. LSCC tumor DNA showed overall hypermethylation relative to NATs (Figure S1G), with tumors separating into CIMP (CpG island methylator phenotype) clusters (Figure S1H). Multi-omic data allowed the identification of \\u201ccascading\\u201d promoter methylation allowed the identification of \\u201ccascading\\u201d promoter methylation cis-effects across cognate mRNA, protein and PTM abundances, supporting their functional significance. Of 90 genes that showed such significant (FDR<0.1) cis-effects in tumors (Table S4), 20 such significant (FDR<0.1) cis-effects in tumors (Table S4), 20 were also differentially expressed (FDR<0.01) between tumors and NATs (Figure 1G). While most were newly associated with LSCC, methylation-driven expression of FAM110A, PTGES3, PLAU and methylation-driven expression of FAM110A, PTGES3, PLAU and SLC16A3 has been implicated in lung and other cancers. Multi-omic clustering identifies five LSCC molecular subtypes, including one that is EMT-Enriched We performed non-negative matrix factorization (NMF)-based single- and multi-omic unsupervised clustering on CNA, RNA, protein, phosphoprotein and acetylprotein datasets from 108 tumors, excluding ubiquitylprotein data as it was not available for the excluding ubiquitylprotein data as it was not available for the entire cohort (Figure 1A). The five resulting multi-omic subtypes (Figure 2A\\u2013B, Figure S2A) were named based on their predominant pathway associations and similarities to previously defined pathway associations and similarities to previously defined RNA clusters. Significant associations between NMF subtypes and cohort metadata are presented in Table S1. Two NMF subtypes, \\u201cBasal-Inclusive\\u201d (B-I) and \\u201cEpithelial to Mesenchymal \\u201cBasal-Inclusive\\u201d (B-I) and \\u201cEpithelial to Mesenchymal Transition-enriched\\u201d (EMT-E), emerged from partitioning of samples that showed similarity to the TCGA-derived RNA Basal cluster, although only B-I retained significant association after Bonferroni only B-I retained significant association after Bonferroni correction (Fisher\\u2019s p<0.01, Table S1). B-I tumors showed basaloid histology and upregulated metabolic, immune and estrogen receptor signaling (Figure 2B). EMT-E showed upregulation of EMT, signaling (Figure 2B). EMT-E showed upregulation of EMT, Angiogenesis and Myogenesis (Figure 2B), with myxoid histologic features and fibroblast infiltration. A third subtype characterized by mutations in KEAP1, CUL3 and NFE2L2 genes and high-level by mutations in KEAP1, CUL3 and NFE2L2 genes and high-level amplification (GISTIC threshold value = 2) of SOX2 and TP63 (Figure 2A\\u2013B, Table S2) was labeled the \\u201cClassical\\u201d subtype, consistent with TCGA nomenclature and with a previous publication. These with TCGA nomenclature and with a previous publication. These showed classical histologic features, CIMP-high enrichment, upregulation of OxPhos- and proliferation-related pathways, and downregulation of immune signaling (Figure 2A\\u2013B). A fourth NMF downregulation of immune signaling (Figure 2A\\u2013B). A fourth NMF subtype was designated \\u201cInflamed-Secretory\\u201d (I-S) due to its alignment with the RNA-based Secretory cluster and strong upregulation of immune-related pathways. The fifth, and strong upregulation of immune-related pathways. The fifth, \\u201cProliferative-Primitive\\u201d (P-P) NMF subtype displayed upregulated proliferation-related pathways, downregulated immune signaling, and enrichment of CIMP-low samples. Within each of these five and enrichment of CIMP-low samples. Within each of these five NMF clusters were samples with low NMF cluster membership scores, which could be considered as belonging to a \\u201cmixed\\u201d subgroup, since they showed features of more than one subtype. Mixed since they showed features of more than one subtype. Mixed subgroup membership, a possible readout of tumor heterogeneity, conferred significantly worse survival (Figure 2C), an observation also made in a recent proteogenomic analysis of glioblastoma. The made in a recent proteogenomic analysis of glioblastoma. The mixed class was associated in the current cohort with increased frequency of SOX2 amplifications (p = 0.0038) but not grade, mutation burden, stemness or other tested variables. We compared B-I (Figure S2B) and EMT-E (Figure S2C) to other subtypes to identify distinguishing features. B-I had hyperphosphorylated M phase Histone 1 T11 and elevated expression of both TACSTD2 (TROP2), a cell-surface glycoprotein that drives growth TACSTD2 (TROP2), a cell-surface glycoprotein that drives growth and metastasis, and MARK2, a kinase that regulates cell polarity and leads to cisplatin resistance. Transcription activator PBX3, which is upregulated only at the protein level (Figure S2B), which is upregulated only at the protein level (Figure S2B), promotes invasion, proliferation, and chemoresistance and is associated with poor prognosis, including in laryngeal squamous cell carcinoma. Pathway analysis of proteins highly correlated with carcinoma. Pathway analysis of proteins highly correlated with PBX3 highlighted neutrophil activation and degranulation (Figure S2D), consistent with the significant upregulation of neutrophilic granule proteins and chemotactic factors (Figure S2E) and granule proteins and chemotactic factors (Figure S2E) and with the xCell-based deconvolution signature (Figure S2F). This correlation was not observed in paired NATs, suggesting a tumor-specific role. In addition to a strong EMT signature (Figure 2B), the EMT-E subtype showed upregulated VCAN (Figure S2C), a tumor-promoting target of SNAIL expressed in cancer-associated fibroblasts (CAFs), and FHL3, a protein that stabilizes EMT-associated (CAFs), and FHL3, a protein that stabilizes EMT-associated transcription factors (Figure S2C). DVL3, FN1 and FHL2, all Wnt pathway regulators, were upregulated in the proteomics dataset, and cell-cell signaling by Wnt was one of the significantly and cell-cell signaling by Wnt was one of the significantly (FDR=0.04) enriched pathways in this subtype. As shown in Figure 2D\\u2013E, coordinated high expression of EMT and CAF proteins, xCell fibroblast scores, and a \\u201cfibroblast proliferation\\u201d signature fibroblast scores, and a \\u201cfibroblast proliferation\\u201d signature suggested that CAFs and tumor epithelium might collaborate in the EMT-E phenotype. This was further supported by enrichment of EMT-related receptor tyrosine kinases (RTKs), as described below. receptor tyrosine kinases (RTKs), as described below. Immunohistochemical co-staining of a small subset of tumors for the epithelial marker pan-cytokeratin (CK) and CAF marker \\u03b1-smooth muscle actin (\\u03b1-SMA) demonstrated the presence in EMT-E tumors of both actin (\\u03b1-SMA) demonstrated the presence in EMT-E tumors of both CAFs and tumor cells undergoing EMT (Figure S2G), with significant enrichment of co-stained cells in the EMT-E subtype (Wilcoxon p=0.025), but not of cells showing single staining of either p=0.025), but not of cells showing single staining of either CK or \\u03b1-SMA alone. TGF\\u03b2 is a well-established and potent inducer of EMT, and a Library of Integrated Network-Based Cellular Signatures-based (LINCS) query for compounds that reversed the EMT-E (LINCS) query for compounds that reversed the EMT-E signature showed enrichment for TGF\\u03b2 inhibitors (Figure S2H, Table S4H). Integrating our global proteome data with a prior LSCC dataset resulted in six clusters largely recapitulating RNA, proteomic and multi-omic clusters defined in the current study (Figure S2I\\u2013J). NMF EMT-E subtype tumors show phosphorylation-driven PDGFR and ROR2 signaling. Though LSCC lacks the RTK mutations for which targeted inhibition has dramatically improved therapeutic options in LUAD, other modes of RTK activation, inferred from phosphoproteomic data, may nonetheless nominate targeted inhibition for some LSCC may nonetheless nominate targeted inhibition for some LSCC patients. Hence, we derived a serine/threonine-predominant correlation-based phosphosite enrichment score (CBPE score) for all RTKs in our tumor cohort. Of nine RTKs with high CBPE scores in LSCC in our tumor cohort. Of nine RTKs with high CBPE scores in LSCC tumors, seven were significantly associated with NMF subtypes (Kruskal-Wallis p <0.01). Scores for PDGFRB and ROR2, markedly elevated in EMT-E (Figure 2F), were highly correlated with the elevated in EMT-E (Figure 2F), were highly correlated with the loss of epithelial cell-cell junction markers and upregulation of mesenchymal proteins involved in remodeling of extracellular matrix, induction of EMT, and promotion of cell migration, which induction of EMT, and promotion of cell migration, which mark the transition from epithelial to mesenchymal state (Figure 2G). As summarized in Figure S2K, we found phosphosite-based evidence for upregulation of both the planar cell polarity and evidence for upregulation of both the planar cell polarity and calcium-dependent branches of non-canonical Wnt signaling, initiated through ROR2 or PDGFR, both targetable and recently implicated in the progression from a hybrid E/M to a highly mesenchymal in the progression from a hybrid E/M to a highly mesenchymal state. Further functional characterization is warranted to confirm the contribution of these RTKs to the EMT-E phenotype and to test the impact of their modulation. Loss of CDK4/6 pathway inhibitors is a universal feature of LSCC but Rb1 expression is variable We investigated the impact of recurrent mutations on cognate RNA, proteins and PTMs (cis-effects) and on a set of cancer associated genes (trans-effects). Significant (FDR<0.05) Cis/Trans pairs are shown in Figure 3A\\u2013D. RB1 mutated tumors had upregulated are shown in Figure 3A\\u2013D. RB1 mutated tumors had upregulated cell cycle-related protein expression. NOTCH1 mutation led to elevated GSK3B protein and phosphorylation (T433), downregulation of inhibitory sites on GSK3A (S21) and upregulation of NLK. NLK of inhibitory sites on GSK3A (S21) and upregulation of NLK. NLK and GSK3B inhibit NOTCH1, recently described as a tumor suppressor in LSCC, suggesting that both might inhibit NOTCH1 downstream signaling in tumors with NOTCH1 mutations. ARID1A is one of signaling in tumors with NOTCH1 mutations. ARID1A is one of the most frequently mutated SWItch Sucrose Non-Fermentable (SWI/SNF) ATP-dependent chromatin remodeling complex members. Mutations were associated with higher tumor mutational burden (Figure S3A) associated with higher tumor mutational burden (Figure S3A) and led to protein downregulation, associated with worse overall survival in NSCLC. Paradoxically, mutations in CDKN2A, proposed as a potential biomarker for CDK4/6 inhibitors, resulted in increased RNA expression (Figure 3A). We analyzed the effects of genetic and epigenetic alterations of CDKN2A on both its major isoforms, p16INK4a alterations of CDKN2A on both its major isoforms, p16INK4a (p16) and p14ARF (p14) (Table S4). As expected, its homozygous deletion resulted in the loss of expression of both isoforms (Figure 3E). However, only eight of the 58 CDKN2A wild-type (WT) samples However, only eight of the 58 CDKN2A wild-type (WT) samples showed p16 RNA expression levels that were comparable to samples with CDKN2A/p16 missense mutations (log2(FPKM+1)>2). Many of those CDKN2A/p16 mutations were unlikely to result in Many of those CDKN2A/p16 mutations were unlikely to result in nonsense-mediated decay (Figure S3B), accounting for preserved expression in those samples. Notably, 28 WT tumors showed hypermethylation of the p16 but not the p14 promoter and suppression of of the p16 but not the p14 promoter and suppression of RNA and protein expression for CDKN2A/p16, while 18 additional WT tumors had reduced expression despite having no clear genetic or epigenetic alteration. Furthermore, the WT samples with high or epigenetic alteration. Furthermore, the WT samples with high CDKN2A/p16 expression had RB1 mutations or deletions and/or low Rb protein levels, consistent with a recently published pan-cancer analysis of the CDK4/6 pathway that showed mutual pan-cancer analysis of the CDK4/6 pathway that showed mutual exclusivity between mutations in these two genes. This suggests that loss of one of these two key CDK4/6 pathway inhibitors is a universal feature of LSCC. Conversely, while amplification of a universal feature of LSCC. Conversely, while amplification of Cyclin D - CDK4/6 complex genes is frequent in LSCC, there was no association between the amplification of CCND2, CCND3, CDK4, or CDK6 and either CDKN2A/p16 protein or Rb CCND2, CCND3, CDK4, or CDK6 and either CDKN2A/p16 protein or Rb protein/phosphoprotein levels (Table S4). CCND1 amplification resulted in significantly higher mean levels of Rb protein and phosphoprotein compared to WT samples, though the distributions phosphoprotein compared to WT samples, though the distributions were overlapping (Figure S3C). Intriguingly, phospho-Rb levels, which reflect CDK4/6 activity, correlated with response to CDK4/6 inhibitors in LSCC cell lines despite the heterogeneous CDK4/6 inhibitors in LSCC cell lines despite the heterogeneous distribution in response for cells with mutations in CDKN2A and copy number alterations in CCND1 and CDKN2A (Figure S3D). LSCC trial data have generally shown minimal efficacy of CDK4/6 LSCC trial data have generally shown minimal efficacy of CDK4/6 inhibition, but outlier responses have been present in trials with abemaciclib and palbociclib. While we found that CCND1 amplification resulted in higher average levels of phosphorylated Rb, resulted in higher average levels of phosphorylated Rb, the heterogeneity of Rb expression and phosphorylation in samples with CCND1 amplification provides a potential explanation for some patients\\u2019 lack of response to CDK4/6 inhibitor therapies. Improved lack of response to CDK4/6 inhibitor therapies. Improved prediction based on a downstream functional assessment of G1/S checkpoint alterations (i.e. RB1 protein expression and phosphorylation) may identify a subset of tumors sensitive to CDK4/6 may identify a subset of tumors sensitive to CDK4/6 inhibition. NRF2 pathway activation in tumors with and without NRF2 pathway mutations LSCC showed mutations of three key genes in the NRF2 antioxidant response pathway, NFE2L2 (NRF2), CUL3 and KEAP1. NFE2L2 phosphorylation was observed in samples with any of these mutations (Figure S3E). LSCC tumors were annotated as \\u201cNRF2 pathway mutated\\u201d S3E). LSCC tumors were annotated as \\u201cNRF2 pathway mutated\\u201d (n=33; with either one hit [n=28] or two hits [n=5] in NFE2L2, KEAP1, and CUL3), or \\u201cNRF2 WT\\u201d (n=68). Compared to WT, NRF2 pathway mutated tumors showed highly concordant differential expression mutated tumors showed highly concordant differential expression of mRNA, protein and phosphoprotein levels of NRF2 pathway genes (Figure 3F, Figure S3F\\u2013G, Table S4), some with therapeutic implications. A subset of NRF2 gene products showed a dosage effect A subset of NRF2 gene products showed a dosage effect (SQSTM1, NR0B1, AKR1B10, CARD11, FFAR2) with the five cases harboring two hits (Table S4) demonstrating increased up-/down-regulation of NRF2 pathway genes relative to one-hit cases (Figure S3H). Using pathway genes relative to one-hit cases (Figure S3H). Using the proteogenomic signature (Table S4) that defined NFE2L2 mutated tumors, we derived an ssGSEA-based NRF2 pathway score that was enriched not only in NRF2 pathway mutated tumors, but also in enriched not only in NRF2 pathway mutated tumors, but also in samples without NRF2 pathway mutations, especially in the NMF Classical subtype (Figure 3G). These samples showed increased NFE2L2 phosphorylation, indicative of NRF2 pathway activation. NFE2L2 phosphorylation, indicative of NRF2 pathway activation. Although the kinase(s) responsible for phosphorylation of NRF2 in LSCC are unclear, CDK5 protein (Figure 3H) and PTM-SEA-derived activity (Figure 3I) were significantly upregulated (FDR<0.05) activity (Figure 3I) were significantly upregulated (FDR<0.05) in the Classical subtype. CDK5 phosphorylation of NFE2L2 S433 mediates its activation in astrocytes and can likely play a role in CDK5-mediated NRF2 activation in LSCC tumors. Intriguingly, in CDK5-mediated NRF2 activation in LSCC tumors. Intriguingly, unlike LUAD, KEAP1 mutations did not result in significantly reduced protein expression in LSCC, although missense mutations were mostly nonoverlapping, suggesting heterogeneity in NRF2 were mostly nonoverlapping, suggesting heterogeneity in NRF2 pathway dysregulation in NSCLC subtypes. Proteogenomic analysis of chromosome 3 prioritizes therapeutic targets in LSCC Chromosome 3q, which harbors key squamous differentiation markers SOX2 and TP63, showed the most dramatic arm-level amplification in this cohort (Figure S4A). To assess potential drivers in the 3q amplicon, we identified 3q genes for which CNA correlated 3q amplicon, we identified 3q genes for which CNA correlated with RNA and protein and expression differed between tumors and NATs. TP63 showed the highest elevation in tumors (Figure 4A). LSCC cell lines amplified for TP63 and SOX2 were highly dependent cell lines amplified for TP63 and SOX2 were highly dependent upon them, supporting oncogene addiction (Figure S4B). As previously reported, \\u0394Np63\\u03b1 was the dominant TP63 isoform in this dataset (Figure S4C) and was highly correlated with TP63 expression (Figure S4D). Its amplification was associated with improved survival in the TCGA dataset (Figure S4E). In contrast to survival in the TCGA dataset (Figure S4E). In contrast to the general elevation of TP63 copy number, transcript, and protein abundance in LSCC tumors compared to NATs, 10 \\u201cNp63-low\\u201d samples histologically confirmed to be LSCC showed no elevation at RNA or confirmed to be LSCC showed no elevation at RNA or protein levels (Figure 4B). The apoptosis inhibitor survivin (BIRC5) was among the most overexpressed proteins in this Np63-low group (Figure 4C). Notably, a substantial number of LSCC cell lines also had 4C). Notably, a substantial number of LSCC cell lines also had low Np63 expression (Figure S4F), and those TP63-low cell lines were significantly more vulnerable to the survivin inhibitor YM-155 (Figure S4G). Survivin promotes cell proliferation and G2M (Figure S4G). Survivin promotes cell proliferation and G2M checkpoints, evident in Np63-low samples (Figure S4H). Small molecule inhibitors and inhibitory peptides targeting survivin are being explored as part of ongoing clinical trials including in being explored as part of ongoing clinical trials including in NSCLC, but have shown modest or no improvement. Our data on the newly identified Np63-low LSCC tumors suggests that TP63 status may identify patients with potential response to survivin may identify patients with potential response to survivin inhibition. Overexpression of miR-205, a biomarker to distinguish LSCC from other NSCLC (Figure S4I), was associated with decreased promoter DNA methylation (Figure S4J) and elevation of \\u0394Np63\\u03b1, a transcriptional regulator of this miRNA. miR-205 showed high a transcriptional regulator of this miRNA. miR-205 showed high correlation with TP63 expression and its expression was markedly reduced in Np63-low samples (Figure 4D, Figure S4K). As expected, the abundance of most target gene products was negatively the abundance of most target gene products was negatively correlated with miR-205 expression, including ZEB1/2 and PTEN (Figure 4E). TP73 positively correlated with miR-205 at the transcript (Figure 4E) and still more strongly at the protein level (Figure 4E) and still more strongly at the protein level (Figure 4F) and with TP63 protein but not RNA (Figure S4L). TP73 overexpression mediates \\u0394Np63\\u03b1-dependent cell survival in squamous cell carcinoma, suggesting a mechanistic model in which functional and suggesting a mechanistic model in which functional and stoichiometric interaction between TP63 and TP73 leads to miR-205 overexpression in LSCC. Finally, consistent with the downregulation of key EMT-related proteins, the TP63 target miR-205 showed of key EMT-related proteins, the TP63 target miR-205 showed significant negative correlation with EMT activity but positive correlation with DNA replication (Figure 4G). The seemingly paradoxical role of TP63 in repressing EMT has been confirmed in role of TP63 in repressing EMT has been confirmed in previous studies. Of note, although \\u0394Np63\\u03b1 and miR-205 were consistently highly overexpressed in the B-I and Classical subtypes compared to NATs, their abundances were variable and not particularly low NATs, their abundances were variable and not particularly low in the EMT-E subtype. Thus, \\u0394Np63\\u03b1, either via miR-205 or indirectly, may induce a strong squamous epithelial cell phenotype in the B-I and Classical tumors, with additional mechanisms leading B-I and Classical tumors, with additional mechanisms leading to derepression and a metastatic phenotype, and other mechanisms beyond miR-205 may be involved in promoting the EMT phenotype in EMT-E samples. SOX2, a recurrently amplified squamous cell marker often co-amplified with TP63, was also overexpressed in tumors (Figure 4A). SOX2 is considered undruggable, intensifying therapeutic interest in upstream or downstream targets. An unbiased search (Figure in upstream or downstream targets. An unbiased search (Figure 4H) identified a significant correlation of LSD1 (KDM1A) with SOX2 protein. LSD1 regulates SOX2 expression and its inhibition in LSCC is currently being explored in a clinical setting in in LSCC is currently being explored in a clinical setting in conjunction with immunotherapy (NCT04350463). Other identified chromatin regulators included KDM3A, known to regulate SOX2 expression in ovarian cancer, and EZH2, whose inhibition in triple in ovarian cancer, and EZH2, whose inhibition in triple negative breast cancer leads to downregulation of SOX2 expression. EZH2 dependency was observed in SOX2 amplified LSCC cell lines (Figure S4M). The positive correlations of SOX2 with ALDH1A1, (Figure S4M). The positive correlations of SOX2 with ALDH1A1, ALDH3A1, and WNT5A (Figure 4H) suggest stemness-like features in these tumors. Interestingly, JAK1 was the protein most negatively correlated with SOX2 among other interferon-signaling proteins correlated with SOX2 among other interferon-signaling proteins such as IRF3 and IFNGR1 (Figure 4H), and negative correlation was seen between SOX2 and JAK-STAT signaling (Figure 4I). Crosstalk between lysine acetylation and ubiquitylation impacts cancer metabolism Consensus clustering of K-GG peptide abundances after correcting for cognate protein abundance revealed two stable ubiquitylproteome clusters (Figure 5A, Table S5) associated with country of origin and ethnicity (Fisher\\u2019s exact p<0.01). Proteins in of origin and ethnicity (Fisher\\u2019s exact p<0.01). Proteins in pathways such as glycolysis, JAK-STAT, MAPK, and immune signaling were differentially modified between these clusters (Figure 5A, Figure S5A\\u2013B, Table S5). To identify candidate enzymes driving Ub and UbL modifications in LSCC, we correlated E3 ubiquitin ligases or deubiquitylases (DUBs) to KGG-sites (Table S5). Eighteen DUBs with at least one negatively correlated K-GG site and 35 E3 ligases with at least negatively correlated K-GG site and 35 E3 ligases with at least one positively correlated K-GG site were identified, with top sites shown in Figure 5B and Figure S5C. HERC5 is the major E3 ligase involved in conjugation of the UbL modification ISG15 in ligase involved in conjugation of the UbL modification ISG15 in humans; hence the numerous positively correlated Ub/UbL sites (n=474; FDR<0.01) are likely ISG15 modifications. Consistent with this, we observed positive correlation between ISG15 and HERC5 this, we observed positive correlation between ISG15 and HERC5 abundance (Figure S5D). Positively-correlating Ub/UbL sites were enriched for glycolysis and TCA cycle annotations (Figure S5E). We observed several positively-correlating K-GG sites in the We observed several positively-correlating K-GG sites in the key glycolytic enzymes PKM, PGK1, and ENO1 (Figure 5C, Figure S5F). No significant negative correlations were observed between total protein abundance and their respective K-GG sites or HERC5, protein abundance and their respective K-GG sites or HERC5, suggesting a regulatory rather than degradative role, as expected for ISGylation (Figure S5G, Figure S5H). ISG15 modifications have been described in these glycolytic enzymes, and ISG15 knockdown been described in these glycolytic enzymes, and ISG15 knockdown in pancreatic cancer cells reduced oxidative phosphorylation and glycolysis. ISGylation can be induced by interferon in cancer; accordingly, HERC5 and ISG15 were positively associated with accordingly, HERC5 and ISG15 were positively associated with proteome-derived IFN-G signaling scores (Figure S5I, Figure S5J), and HERC5 was increased in tumors of the IFN-gamma dominant subtype (Figure 5D). Recently, a direct link between immune subtype (Figure 5D). Recently, a direct link between immune signaling and glucose metabolism through ISGylation was shown in the context of pathogenic invasion. Collectively, our data supports a potential role for HERC5 as a bridge between immune supports a potential role for HERC5 as a bridge between immune signaling and metabolic regulation in LSCC. This dataset allowed us to investigate crosstalk between lysine PTMs. We employed a modified version of CLUMPS to detect clustering of either acetylation or ubiquitylation sites within protein 3D structures. Significant ubiquitylation clusters were protein 3D structures. Significant ubiquitylation clusters were detected in 33 proteins and significant acetylation clusters in 17 proteins (p<0.05) (Table S5). Top hits revealed enrichment of glycolysis, fatty acid metabolism and OxPhos (Table S5), of glycolysis, fatty acid metabolism and OxPhos (Table S5), supporting modulation of these metabolic pathways by ubiquitylation and acetylation. In tumor-NAT comparison, major glycolytic enzymes PGK1 and PKM showed reduced acetylation at multiple known PGK1 and PKM showed reduced acetylation at multiple known inhibitory sites closely-positioned on the 3D structure, as well as on adjacent lysines surrounding the enzymatic pocket  (Figure 5E, Figure S5K). Tumors also had decreased ubiquitylation at 5E, Figure S5K). Tumors also had decreased ubiquitylation at closely-positioned K-GG-sites for TXN1, an important redox regulator (Figures 5F). PGK1 and PKM deacetylation suggests increased activity of these key glycolytic enzymes in tumors, a conclusion of these key glycolytic enzymes in tumors, a conclusion supported by increased phosphorylation on known activating sites of regulatory proteins (PFKFB3 pS461, PKM pS37). Upregulation of a glycolytic metabolic phenotype confers selective advantage to a glycolytic metabolic phenotype confers selective advantage to cancer cells by supporting uninterrupted growth, dynamically modulated by PTMs. While site-level roles of most Ub/UbL modifications and acetylsites are unclear, our data on acetylation, and acetylsites are unclear, our data on acetylation, phosphorylation, and HERC5-mediated ISGylation of glycolytic enzymes hints at the complexity of regulation of this cancer hallmark. Rapid growth of cells often leads to oxidative stress within tumors, which adapt by upregulating redox systems. Although thioredoxin (TXN) did not show overall differential protein expression between tumors and NATs, we observed increased TXN1 protein between tumors and NATs, we observed increased TXN1 protein combined with decreased Ub sites, particularly in the NMF Classical subtype (Kruskal Wallis FDR<0.01). The TXN1 activator TXNRD1 was also increased in the Classical subtype, while TXNIP, which was also increased in the Classical subtype, while TXNIP, which modulates cellular redox by binding and inhibiting TXN1, was decreased (Figure 5F). TXN and TXNRD1 were highly correlated with NRF2 score (TXN: R=0.85; TXNRD1: R=0.8, both p<2.2\\u00d710\\u221216), with NRF2 score (TXN: R=0.85; TXNRD1: R=0.8, both p<2.2\\u00d710\\u221216), consistent with their role in response to oxidative stress. Redox modulation through this pathway is associated with tumor proliferation, differentiation and prognosis in multiple epithelial differentiation and prognosis in multiple epithelial cancers, including NSCLC. The small proportion of glucose directed to the tumor mitochondria and subsequently to the TCA and OXPHOS cycles leads to increased acetyl CoA within the mitochondria and leads to increased acetyl CoA within the mitochondria and thence to non-enzymatic acetylation and inhibition of key enzymes in the fatty acid oxidation pathway such as ECH1 and ACADVL (also identified in the CLUMPS analysis). We recapitulate the finding identified in the CLUMPS analysis). We recapitulate the finding of increased ECH1 and ACADVL acetylation, including specific increase at the binding site of ACADVL in tumors (Figure 5E, Figure S5K). PTM-based modulation of these key metabolic and ROS S5K). PTM-based modulation of these key metabolic and ROS pathways is summarized in Figure 5G. As a complementary assessment of the crosstalk between lysine ubiquitylation and acetylation, we identified K-Ac or K-GG sites showing significant differential expression in tumors relative to NATs. Figure 5H shows 8 lysine sites where K-GG was relative to NATs. Figure 5H shows 8 lysine sites where K-GG was upregulated and K-Ac downregulated in tumors (upper panel) and 36 sites demonstrating the opposite trend (lower panel). Among the latter was the GTP-binding nuclear protein RAN, involved in the latter was the GTP-binding nuclear protein RAN, involved in nucleocytoplasmic transport, mitotic progression and spindle assembly. Acetylation on multiple lysine residues can affect RAN localization, GTP/GDP cycle and import / export complex RAN localization, GTP/GDP cycle and import / export complex formation. Consistent with the role of RAN in mitosis, RAN K127-acK showed dramatic upregulation in the P-P subtype (Figure 5I), named in part for its generally high proliferation (Figure 2B). named in part for its generally high proliferation (Figure 2B). The specific functional relevance of K127-ac has not been previously demonstrated, but its selective upregulation suggests a functional role. While of great interest to the PTM community and role. While of great interest to the PTM community and potentially very powerful, such crosstalk analyses are nascent, and interpretations remain suitably cautious. Immune landscape and regulation in LSCC Consensus clustering based on the xCell signatures (Table S6) identified three sets of tumors representing immunologically Hot, Warm, and Cold clusters (Figure 6A\\u2013C), as well as an NAT-enriched cluster. The NMF I-S subtype was significantly associated cluster. The NMF I-S subtype was significantly associated with the immune Hot cluster (Fisher\\u2019s exact p=6.371\\u00d710\\u221209) (Figure 6A). The Hot cluster was characterized by increased macrophages, CD4+, CD8+ and regulatory T cells, and dendritic cells in both CD4+, CD8+ and regulatory T cells, and dendritic cells in both deconvolution and immunohistochemistry (IHC) analyses (Figures 6A,C, S6C,D), and showed upregulation of immune-related pathways (Figure 6A\\u2013B, Figure S6A, Table S6). Immunosuppressive mediators 6A\\u2013B, Figure S6A, Table S6). Immunosuppressive mediators CTLA4, PD-1 (PDCD1), PD-L1 (CD274) and IDO1 and the key Treg transcriptional regulator FOXP3 were upregulated in Hot compared to Cold tumors, suggesting immune checkpoint-related therapeutic options suggesting immune checkpoint-related therapeutic options in these tumors. The Hot tumor acetylproteome was chiefly enriched for OXPHOS, Mitochondrial Complex and TCA Cycle pathways, consistent with increased metabolism or increased mitochondrial consistent with increased metabolism or increased mitochondrial acetyl-CoA leading to non-enzymatic acetylation of mitochondrial proteins (Figure 6D). Notably, NATs did not display consonant immune signaling (Figure S6B). The intermediate, Warm cluster immune signaling (Figure S6B). The intermediate, Warm cluster displayed upregulation of immune-related pathways such as PD-1 signaling, Interferon gamma (IFN-G) response, and Allograft rejection relative to Cold tumors, but did not show the downregulation relative to Cold tumors, but did not show the downregulation of cell cycle-related pathways observed in Hot tumors (Figure 6A\\u2013B, Figure S6A). In addition, canonical Wnt Signaling, active in tumor proliferation and immune evasion, was comparably in tumor proliferation and immune evasion, was comparably upregulated in the Warm and Cold tumor proteomes (Figure 6B). Fifteen pQTLs were also identified in proteins differentially abundant between Hot and Cold tumors (Table S6, Figure S6E) Rho GTPase signaling was upregulated in Hot tumors (Figure 6A, Table S6). ARHGDIB (RhoGDIb), the central regulatory molecule in GTPase activation, was the most significantly acetylated protein in the Hot cluster, followed by metabolic mitochondrial protein in the Hot cluster, followed by metabolic mitochondrial enzymes (Figure 6D). Acetylation of its homolog RhoGDIa at the conserved K135 site regulates F-actin assembly by activating RhoA. In contrast to RhoGDIa, the affinity of RhoGDIb for RhoA is In contrast to RhoGDIa, the affinity of RhoGDIb for RhoA is low, and the K135 acetylation of RhoGDIb may act as a positive regulator of Rac signaling via its interaction with Rac2 instead. In support of this hypothesis, we found significant upregulation support of this hypothesis, we found significant upregulation in the Hot cluster of Rho GTPase Rac2 and numerous Rac-specific regulators (Figures 6F, S6F, Table S6). Notably, these proteins are primarily immune-specific, indicating a possible role for are primarily immune-specific, indicating a possible role for lysine acetylation of RhoGDIb in promoting immune cell functions in the Hot cluster. Immunohistochemical staining for RhoGDIb (ARHGDIB) in the subset of immune Hot tumors confirmed strong (ARHGDIB) in the subset of immune Hot tumors confirmed strong signals in infiltrating immune cells and mesenchymal cells (Figure S6G), similar to dispersed RhoGDIb-expressing cells identified as lymphocytes seen in renal cell carcinomas. Thus, we propose as lymphocytes seen in renal cell carcinomas. Thus, we propose that K135 acetylation of RhoGDIb is a potential regulatory mechanism of immune cell functions that is in line with the inflammasome signature and immune-enriched networks. We explored the immunological associations of the RTK CBPE scores described above (Figure 2I,J) in LSCC tumors (Figure 6G). CSF1R, predominantly expressed in macrophages, showed the highest CBPE scores, and was positively associated with the Hot cluster CBPE scores, and was positively associated with the Hot cluster (Figure 6H) and consequently with immune-related pathways and cell types (Figure 6G). CSF1R correlated with immunosuppressive chemokine CCL5, phosphorylation of PIK3R1 Y580, downstream chemokine CCL5, phosphorylation of PIK3R1 Y580, downstream regulators of the actin cytoskeleton including WAS and WASF2 S103, and upregulation of AKAP13, among other immune-related proteins (Figure 6I). Tumor-associated macrophages (, CSF1R signaling, and 6I). Tumor-associated macrophages (, CSF1R signaling, and AKAP13 have each been proposed as anti-cancer targets. Proteomic Biomarker Candidates for Prognosis, Diagnosis, and Treatment Tumors and paired NATs revealed remarkable differences in protein expression and pathway enrichment (Figure 7A, Table S7). The top 50 of 502 differential proteins are shown in Figure S7A. In addition to 206 upregulated oncogenic proteins (OncoKB, Table addition to 206 upregulated oncogenic proteins (OncoKB, Table S7), several CT antigens, attractive targets of CT-antigen therapies, were overexpressed in tumors (Figure S7B), with the I-S samples showing the fewest CT antigens. As complete long-term samples showing the fewest CT antigens. As complete long-term follow-up was not yet available for this cohort, we leveraged the TCGA LSCC dataset to examine the association of these 502 candidate tumor biomarkers with overall survival (OS) or disease-free tumor biomarkers with overall survival (OS) or disease-free survival (DFS). Expression of four of the most highly differential genes showed significant association with poor OS and another 15 genes with poor DFS (Figure 7B, Figure S7C). Furthermore, 15 genes with poor DFS (Figure 7B, Figure S7C). Furthermore, knockdown of these tumor biomarker proteins reduced fitness across 16 LSCC cell lines (https://depmap.org/), suggesting critical roles in key cellular transformation and proliferation processes in key cellular transformation and proliferation processes (Figure 7C, Table S7). There is increasing interest in molecular taxonomies that help position cancer types in the context of related cancers. CNAs were more frequent in LSCC than in LUAD or head and neck squamous cell carcinoma (HNSCC), which respectively share tissue and cell carcinoma (HNSCC), which respectively share tissue and cell type of origin (Figure S7D). Widespread deletions of immune-related genes in LSCC correlated with ESTIMATE immune scores (Figure 7D). Immune-related deletions were less prominent in HNSCC and 7D). Immune-related deletions were less prominent in HNSCC and especially LUAD (Figure S7E), except for Chromosome 3p deletion, encompassing five chemokine receptors. Squamous cancers shared 3q amplification, JAK2 deletion and skin-development proteins 3q amplification, JAK2 deletion and skin-development proteins from across the genome. Finally, deletions of 4p14 and 5q, including multiple Toll-like receptor and interleukin signaling genes, were unique to LSCC. Among key tumor-specific phosphorylation were unique to LSCC. Among key tumor-specific phosphorylation events in LSCC, we identified upregulation of activating sites on 27 kinases, including MAPK14 and three others (DCK, EGFR, SRC) targetable by FDA-approved drugs (Figure S7F), with several targetable by FDA-approved drugs (Figure S7F), with several being shared by LUAD and HNSCC tumors. Notably, EGFR protein was significantly upregulated in the squamous cancers but not LUAD (Figure S7G), although LUAD had many activating EGFR mutations, and S7G), although LUAD had many activating EGFR mutations, and activating phosphorylation of EGFR was increased in both LUAD and LSCC. High EGFR amplification in the LSCC cohort did not correlate with EGFR pathway activity as assessed by mRNA-based PROGENy with EGFR pathway activity as assessed by mRNA-based PROGENy scoring (Figure 7E). Instead, this activity measurement was highly correlated to mRNA abundance of the five EGFR ligands that are prominent in tumors (Figure 7F). Furthermore, EGFR amplified prominent in tumors (Figure 7F). Furthermore, EGFR amplified samples displayed increased phosphorylation on proteins related to actin filament and cell junction organization as well as cell-substrate adhesion (Figure 7G). These findings and similar cell-substrate adhesion (Figure 7G). These findings and similar results recently reported in HNSCC support a squamous cell cancer feature in which EGFR ligand abundance drives canonical EGFR pathway activity and tumors with very high levels of EGFR may no activity and tumors with very high levels of EGFR may no longer require the ligand to drive activity. EGFR inhibition guided by EGFR abundance has been unsuccessful in squamous tumors. Since EGFR inhibitors such as cetuximab affect ligand-induced EGFR EGFR inhibitors such as cetuximab affect ligand-induced EGFR activity, ligand abundance, rather than EGFR amplification, might better predict EGFR inhibitor response in this population. Figure 7H graphically represents these and other key findings Figure 7H graphically represents these and other key findings presented in this manuscript, emphasizing translatable implications. Discussion In this study, we provide detailed proteogenomic profiles of treatment-na\\u00efve, primary LSCC tumors and paired NATs, with unprecedented coverage of post-translational modifications including phosphorylation, acetylation and for the first time, including phosphorylation, acetylation and for the first time, ubiquitylation. Unsupervised NMF-based multi-omic clustering suggested a refinement to prior RNA-based clustering, dividing basal tumors into Basal-inclusive and EMT-enriched subtypes with tumors into Basal-inclusive and EMT-enriched subtypes with biological differences of potential therapeutic significance. Molecular events downstream of copy number alterations showed NSD3 rather than FGFR1 protein overexpression in tumors with 8p11.23 rather than FGFR1 protein overexpression in tumors with 8p11.23 amplification, providing both a potential explanation for the limited effectiveness of FGFR1-targeted treatment in this population and a potential alternative therapy. Parallel downstream and a potential alternative therapy. Parallel downstream analysis of DNA promoter methylation events provided insight into regulation of many proteins differential between cancer and NATs. Comprehensive proteogenomic data provided a deeper exposition of established LSCC biology, often with potential therapeutic implications. We identified a subset of low-p63 tumors that were characterized by high levels of the known therapeutic target characterized by high levels of the known therapeutic target survivin. SOX2 is considered undruggable, but our analyses provide rationale for exploring LSD1 or other chromatin modifiers such as EZH2 to target SOX2 amplified / overexpressing tumors. The as EZH2 to target SOX2 amplified / overexpressing tumors. The importance of glycolysis and oxidative stress in LSCC are well-appreciated, but our data support complex regulation of these metabolic pathways by crosstalk between ubiquitylation (or metabolic pathways by crosstalk between ubiquitylation (or ubiquitin-like modifications), phosphorylation and acetylation. Immunotherapy represents the greatest advance in LSCC therapy in decades, but outcomes lag behind those seen in patients with LUAD but outcomes lag behind those seen in patients with LUAD and still only a minority of patients exhibit long-term responses; numerous immune-related proteogenomic observations suggest directions for further investigation. Proteogenomic dissection of the for further investigation. Proteogenomic dissection of the downstream effects of CDKN2A mutations had clinical implications related both to the interpretation of trials utilizing CDK4/6 inhibitors in LSCC patients and to biomarker selection for future in LSCC patients and to biomarker selection for future studies: though CDK4/6 inhibition has shown limited efficacy in LSCC trials to date, our analysis suggested that a more nuanced assessment of RB1 protein expression and phosphorylation is required of RB1 protein expression and phosphorylation is required before declaring this approach unsuccessful. Finally, triangulation between LSCC, LUAD and HNSCC demonstrated the influence of both tissue and cell type of origin on cancer biology, with of both tissue and cell type of origin on cancer biology, with therapeutic vulnerabilities both unique and common across these cancer types. Limitations of the study Many important clinical advances in NSCLC and other cancers have been driven by genomic profiling of bulk tumor material, and we anticipate that the same will prove true of bulk proteogenomic characterization, as performed here. Nevertheless, tumor characterization, as performed here. Nevertheless, tumor heterogeneity is an important complication, and emerging methods for proteomics approaches to heterogeneity, such as pairing microdissection with mass spectrometry or using mass cytometry will be with mass spectrometry or using mass cytometry will be useful adjuncts. It is typically metastatic disease that proves lethal, and in-depth proteogenomic analysis of primary tumors paired with metastatic lesions will be needed to provide critical insights metastatic lesions will be needed to provide critical insights into metastatic biology. Importantly, the scope of a landscape study such as this necessitates that it be understood as hypothesis-generating, and a wider community effort will be required to and a wider community effort will be required to validate biological observations and suggested therapeutic alternatives. Nevertheless, we hope this study will prove a valuable resource to the research and clinical communities, and advance the to the research and clinical communities, and advance the understanding and treatment of LSCC. STAR\\u2605METHODS\\n\\nRESOURCE AVAILABILITY\\n\\nLead contact\\n\\nFurther information and requests should be directed to and will be fulfilled by the lead author Michael A. Gillette (gillette@broadinstitute.org)\\n\\nMaterial availability Material availability\\n\\nThis study did not generate new unique reagents.\\n\\nData and Code Availability\\n\\nCPTAC LSCC proteomics data:\\n\\nProteomic Data Commons (PDC); https://pdc.cancer.gov/pdc/ with identifiers PDC000232, PDC000233, PDC000234 and PDC000237 CPTAC data portal LSCC: https://cptac-data-portal.georgetown.edu/study-summary/S063\\n\\nCPTAC HNSCC proteomics data:\\n\\nPDC: https://pdc.cancer.gov/pdc/ with identifiers PDC000221 and PDC000222\\n\\n https://cptac-data-portal.georgetown.edu/study-summary/S054 https://cptac-data-portal.georgetown.edu/study-summary/S054 \\n\\nCPATC LUAD proteomics data:\\n\\nPDC: https://pdc.cancer.gov/pdc/ with identifiers PDC000153, PDC000149, PDC000224)\\n\\n https://cptac-data-portal.georgetown.edu/study-summary/S056 https://cptac-data-portal.georgetown.edu/study-summary/S056 \\n\\nGenomic and transcriptomic data files can be accessed at the Genomic Data Commons (GDC); https://portal.gdc.cancer.gov/, via dbGaP Study Accession: phs001287.v10.p5 All histologic https://www.cancerimagingarchive.net/datascope/cptac/home/ and radiologic details can be accessed from the The Cancer Imaging Archive (TCIA) Public Access https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC. Sample annotation, processed and normalized data files are provided as Tables S1\\u2013S3.\\n\\nSoftware and code used in this study are referenced in their corresponding STAR Method sections and also the Key Resource Table.\\n\\nEXPERIMENTAL MODEL AND SUBJECT DETAILS EXPERIMENTAL MODEL AND SUBJECT DETAILS\\n\\nHuman Subjects A total of 113 participants (90 males, 23 females, 40\\u201388 years old) were collected for this study by 13 different tissue source sites from seven different countries between May 2016 and August 2018, with a median follow-up time of 2.5 years. Five (one 2018, with a median follow-up time of 2.5 years. Five (one female and four male) samples were excluded based on further pathological assessment and 108 tumors and 99 paired NATs were used in this study. Only histopathologically-defined adult lung squamous study. Only histopathologically-defined adult lung squamous tumors were considered for analysis, with an age range of 40\\u201388. Institutional review boards at tissue source sites reviewed protocols and consent documentation adhering to the Clinical Proteomic and consent documentation adhering to the Clinical Proteomic Tumor Analysis Consortium (CPTAC) guidelines. Clinical data were obtained from tissue source sites and aggregated by an internal database called the CDR (Comprehensive Data Resource) that database called the CDR (Comprehensive Data Resource) that synchronizes with the CPTAC DCC (https://cptac-data-portal.georgetown.edu/). Clinical data can be accessed and downloaded from the DCC (Data Coordinating Center). Demographics, histopathologic DCC (Data Coordinating Center). Demographics, histopathologic information, and treatment details were collected. LSCC histopathology was confirmed for all cases by at least 2 expert pathologists based on high resolution images of H&E sections. For samples based on high resolution images of H&E sections. For samples with low TP63 status, additional pathologic confirmation was performed. The histologic, genotypic, clinical, geographical and other associated metadata is summarized in Table S1. METHOD DETAILS\\n\\nSpecimen Acquisition The tumor, normal adjacent tissue (NAT), and whole blood samples used in this manuscript were prospectively collected for the CPTAC project. Biospecimens were collected from newly diagnosed patients with LSCC who underwent surgical resection and had patients with LSCC who underwent surgical resection and had received no prior treatment for their disease, including chemotherapy or radiotherapy. All cases had to be of acceptable LSCC histology but were collected regardless of surgical stage or histology but were collected regardless of surgical stage or histologic grade. Cases were staged using the AJCC cancer staging system 7th edition and the 8th edition. The tumor specimen weights ranged from 125 to 1560 milligrams. The average tissue mass ranged from 125 to 1560 milligrams. The average tissue mass was 220 mg. For most cases, three to four tumor specimens were collected. Paired histologically-normal adjacent lung tissues (NATs) were collected from the same patient at tumor resection. Each were collected from the same patient at tumor resection. Each tissue specimen endured cold ischemia for less than 40 minutes prior to freezing in liquid nitrogen; the average ischemic time was 16 minutes from resection/collection to freezing. Specimens was 16 minutes from resection/collection to freezing. Specimens were flash frozen in liquid nitrogen. Histologic sections obtained from top and bottom portions from each case were reviewed by a board-certified pathologist to confirm the assigned by a board-certified pathologist to confirm the assigned pathology. For samples to be deemed acceptable, the top and bottom sections had to contain an average of 50% tumor cell nuclei with less than 20% necrosis. Specimens were shipped overnight from the than 20% necrosis. Specimens were shipped overnight from the tissue source sites to the biospecimen core resource (BCR) located at Van Andel Research Institute, Grand Rapids, MI using a cryoport that maintained an average temperature of less than \\u2212140\\u00b0C. that maintained an average temperature of less than \\u2212140\\u00b0C. At the biospecimen core resource, specimens were confirmed for pathology qualification and prepared for genomic, transcriptomic, and proteomic analyses. Selected specimens were cryopulverized and proteomic analyses. Selected specimens were cryopulverized using a Covaris CryoPREP instrument and material aliquoted for subsequent molecular characterization. Genomic DNA and total RNA were extracted and sent to the genome sequencing centers. The were extracted and sent to the genome sequencing centers. The whole exome and whole genome DNA sequencing and methylation EPIC array analyses were performed at the Broad Institute, Cambridge, MA. TotalRNA and miRNA sequencing were performed at the MA. TotalRNA and miRNA sequencing were performed at the University of North Carolina, Chapel Hill, NC. Material for proteomic analyses were sent to the Proteomic Characterization Center (PCC) at the Broad Institute, Cambridge, MA. Sequencing sample preparation Our study sampled a single site of the primary tumor from surgical resections, with an internal requirement to process a minimum of 125mg of tumor issue and 50mg of NAT. DNA and RNA were extracted from tumor and NAT specimens in a co-isolation protocol from tumor and NAT specimens in a co-isolation protocol using Qiagen\\u2019s QIAsymphony DNA Mini Kit and QIAsymphony RNA Kit. Genomic DNA was also isolated from peripheral blood (3\\u20135mL) to serve as matched normal reference material. The Qubit\\u2122 dsDNA BR Assay as matched normal reference material. The Qubit\\u2122 dsDNA BR Assay Kit was used with the Qubit\\u00ae 2.0 Fluorimeter to determine the concentration of dsDNA in an aqueous solution. Any sample that passed quality control and produced enough DNA yield to go through quality control and produced enough DNA yield to go through the multiple planned genomic assays was sent for genomic characterization. RNA quality was quantified using the NanoDrop 8000 and quality assessed using an Agilent Bioanalyzer. A sample of and quality assessed using an Agilent Bioanalyzer. A sample of sufficient quantity that passed RNA quality control and had a minimum RIN (RNA integrity number) score of 7 was subjected to RNA sequencing. Identity matches for germline, normal adjacent RNA sequencing. Identity matches for germline, normal adjacent tissue, and tumor tissue were confirmed at the BCR using the Illumina Infinium QC array. This beadchip contains 15,949 markers designed to prioritize sample tracking, quality control, and designed to prioritize sample tracking, quality control, and stratification. Whole exome Sequencing (WES)\\n\\nLibrary construction and Hybrid Selection Library construction was performed as described in, with the following modifications: initial genomic DNA input into shearing was reduced from 3\\u03bcg to 20\\u2013250ng in 50\\u03bcL of solution. For adapter ligation, Illumina paired-end adapters were replaced with ligation, Illumina paired-end adapters were replaced with palindromic forked adapters, purchased from Integrated DNA Technologies (IDT), with unique dual-indexed molecular barcode sequences to facilitate downstream pooling. Kapa HyperPrep reagents in to facilitate downstream pooling. Kapa HyperPrep reagents in 96-reaction kit format were used for end repair/A-tailing, adapter ligation, and library enrichment PCR. In addition, during the post-enrichment SPRI cleanup, elution volume was reduced to 30\\u03bcL SPRI cleanup, elution volume was reduced to 30\\u03bcL to maximize library concentration, and a vortexing step was added to maximize the amount of template eluted. After library construction, libraries were pooled into groups of up to 96 samples. Hybridization were pooled into groups of up to 96 samples. Hybridization and capture were performed using the relevant components of Illumina\\u2019s Nextera Exome Kit and following the manufacturer\\u2019s suggested protocol, with the following exceptions: First, all libraries protocol, with the following exceptions: First, all libraries within a library construction plate were pooled prior to hybridization. Second, the Midi plate from Illumina\\u2019s Nextera Exome Kit was replaced with a skirted PCR plate to facilitate automation. was replaced with a skirted PCR plate to facilitate automation. All hybridization and capture steps were automated on the Agilent Bravo liquid handling system. Cluster Amplification and Sequencing After post-capture enrichment, library pools were quantified using qPCR (KAPA Biosystems) using an automated assay on the Agilent Bravo with probes specific to the ends of the adapters. Based on qPCR quantification, libraries were normalized to 2nM. Based on qPCR quantification, libraries were normalized to 2nM. Cluster amplification of DNA libraries was performed following manufacturer\\u2019s protocol (Illumina) using exclusion amplification chemistry and flowcells. Flow cells were sequenced utilizing chemistry and flowcells. Flow cells were sequenced utilizing sequencing-by-synthesis chemistry. The flow cells were then analyzed using RTA v.2.7.3 or later. Each pool of whole exome libraries was sequenced on paired 76-cycle runs with two 8-cycle index was sequenced on paired 76-cycle runs with two 8-cycle index reads across the number of lanes needed to meet coverage for all libraries in the pool. Pooled libraries were run on HiSeq4000 paired-end runs to achieve a minimum of 150x on-target coverage per runs to achieve a minimum of 150x on-target coverage per library. The raw Illumina sequence data were demultiplexed and converted to FASTQ files; adapter and low-quality sequences were trimmed. The raw reads were mapped to the GRCh38/hg38 human reference The raw reads were mapped to the GRCh38/hg38 human reference genome and the validated BAMs were used for downstream analysis and variant calling. Whole genome sequencing (WGS)\\n\\nCluster Amplification and Sequencing An aliquot of genomic DNA (350ng in 50\\u03bcL) was used as the input into DNA fragmentation (aka shearing). Shearing was performed acoustically using a Covaris focused-ultrasonicator, targeting 385bp fragments. Following fragmentation, additional size 385bp fragments. Following fragmentation, additional size selection was performed using SPRI cleanup. Library preparation was performed using a commercially available KAPA Hyper Prep without amplification module kit (KAPA Biosystems) and with palindromic amplification module kit (KAPA Biosystems) and with palindromic forked adapters with unique 8-base index sequences embedded within the adapter (IDT). Following sample preparation, libraries were quantified using quantitative PCR (KAPA Biosystems), with were quantified using quantitative PCR (KAPA Biosystems), with probes specific to the ends of the adapters using the automated Agilent\\u2019s Bravo liquid handling platform. Based on qPCR quantification, libraries were normalized to 1.7nM and pooled into libraries were normalized to 1.7nM and pooled into 24-plexes. Sample pools were combined with HiSeqX Cluster Amp Reagents EPX1, EPX2, and EPX3 into single wells on a strip tube using the Hamilton Starlet Liquid Handling system. Cluster amplification of the templates was performed according to the manufacturer\\u2019s of the templates was performed according to the manufacturer\\u2019s protocol (Illumina) with the Illumina cBot. Flow cells were sequenced to a minimum of 15x on HiSeqX utilizing sequencing-by-synthesis kits to produce 151bp paired-end reads. Output from kits to produce 151bp paired-end reads. Output from Illumina software was processed by the Picard data processing pipeline to yield BAM files containing demultiplexed, aggregated, aligned reads. All sample information tracking was performed by automated All sample information tracking was performed by automated LIMS messaging. Array Based Methylation Analysis The Methylation EPIC array uses an 8-sample version of the Illumina Beadchip capturing >850,000 methylation sites per sample. Two hundred and fifty nanograms of DNA was used for the bisulfite conversion using Infinium MethylationEPIC BeadChip Kit conversion using Infinium MethylationEPIC BeadChip Kit (Illumina). The EPIC array includes sample plating, bisulfite conversion, and methylation array processing. After scanning, the data was processed through an automated genotype-calling pipeline. Data processed through an automated genotype-calling pipeline. Data output consisted of raw idats and a sample sheet. RNA and miRNA sequencing\\n\\nQuality Assurance and Control of RNA Analytes All RNA analytes were assayed for RNA integrity, concentration, and fragment size. Samples for total RNA-seq were quantified on a TapeStation system (Agilent, Inc. Santa Clara, CA). Samples with RINs >7.0 were considered high quality and were considered with RINs >7.0 were considered high quality and were considered for sequencing. Total RNA-seq libraries were generated using 300 nanograms of total RNA using the TruSeq Stranded Total RNA Library Prep Kit with Ribo-Zero Gold and bar-coded with individual Prep Kit with Ribo-Zero Gold and bar-coded with individual tags following the manufacturer\\u2019s instructions (Illumina). Total RNA Libraries were prepared on an Agilent Bravo automated liquid handling system. Quality control was performed at every step, and system. Quality control was performed at every step, and the libraries were quantified using a TapeStation system. Total RNA Sequencing Indexed libraries were prepared and run on HiSeq4000 paired-end 75 base pairs to generate a minimum of 120 million reads per sample library with a target of greater than 90% mapped reads. The raw Illumina sequence data were demultiplexed and converted to raw Illumina sequence data were demultiplexed and converted to FASTQ files, and adapter and low-quality sequences were trimmed. Samples were then assessed for quality by mapping reads to GRCh38/hg38, estimating the total number of mapped reads, amount of estimating the total number of mapped reads, amount of RNA mapping to coding regions, amount of rRNA in the sample, number of genes expressed, and relative expression of housekeeping genes. Samples passing this QA/QC were then clustered with other Samples passing this QA/QC were then clustered with other expression data from similar and distinct tumor types to confirm expected expression patterns. Atypical samples were then SNP typed from the RNA data to confirm source analyte. FASTQ files of all from the RNA data to confirm source analyte. FASTQ files of all reads were then uploaded to the GDC repository. miRNA-seq Library Construction miRNA-seq library construction was performed from the RNA samples using the NEXTflex Small RNA-Seq Kit (v3, PerkinElmer, Waltham, MA) and barcoded with individual tags following the manufacturer\\u2019s instructions. Libraries were prepared on a Sciclone Liquid instructions. Libraries were prepared on a Sciclone Liquid Handling Workstation. Quality control was performed at every step, and the libraries were quantified using a TapeStation system and an Agilent Bioanalyzer using the Small RNA analysis kit. Pooled an Agilent Bioanalyzer using the Small RNA analysis kit. Pooled libraries were then size selected according to NEXTflex kit specifications using a Pippin Prep system (Sage Science, Beverly, MA). miRNA Sequencing Indexed libraries were loaded on the HiSeq4000 to generate a minimum of 10 million reads per library with a minimum of 90% reads mapped. The raw Illumina sequence data were demultiplexed and converted to FASTQ files for downstream analysis. Resultant data to FASTQ files for downstream analysis. Resultant data were analyzed using a variant of the small RNA quantification pipeline developed for TCGA (Chu et al., 2016). Data from samples were assessed for the number of miRNAs called, species diversity, and for the number of miRNAs called, species diversity, and total abundance before uploading to the GDC repository. Mass Spectrometry methods The protocols below for protein extraction, tryptic digestion, TMT-11 labeling of peptides, peptide fractionation by basic reversed-phase liquid chromatography, phosphopeptide enrichment using immobilized metal affinity chromatography, and LC-MS/MS were immobilized metal affinity chromatography, and LC-MS/MS were performed as previously described in depth. Acetyl-enrichment was performed as described before with modifications as indicated below. Protein Extraction and Tryptic Digestion Approximately fifty milligrams (wet weight) of cryopulverized human LSCC and NAT samples were homogenized in the lysis buffer for 30 minutes at a ratio of about 200 uL lysis buffer for every 50 mg wet weight tissue. The lysis buffer consisted of 8 M urea, mg wet weight tissue. The lysis buffer consisted of 8 M urea, 75 mM NaCl, 1 mM EDTA, 50 mM Tris HCl (pH 8), 10 mM NaF, phosphatase inhibitor cocktail 2 (1:100; Sigma, P5726) and cocktail 3 (1:100; Sigma, P0044), 2 \\u03bcg/mL aprotinin (Sigma, A6103), 10 \\u03bcg/mL Sigma, P0044), 2 \\u03bcg/mL aprotinin (Sigma, A6103), 10 \\u03bcg/mL leupeptin (Roche, 11017101001), and 1 mM PMSF (Sigma, 78830). Lysates were centrifuged at 20,000 g for 10 minutes and protein concentrations of the clarified lysates were measured by BCA assay of the clarified lysates were measured by BCA assay (Pierce). Protein lysates were subsequently reduced with 5 mM dithiothreitol (Thermo Scientific, 20291) for an hour at 37C and alkylated with 10 mM iodoacetamide (Sigma, A3221) for 45 minutes in the dark 10 mM iodoacetamide (Sigma, A3221) for 45 minutes in the dark at room temperature. Prior to digestion, samples were diluted 4-fold to achieve 2 M urea with 50mM Tris HCl (pH 8). Digestion was performed with LysC (Wako, 129\\u201302541) for 2 hours and with was performed with LysC (Wako, 129\\u201302541) for 2 hours and with trypsin (Promega, V511X) overnight, both at a 1:50 enzyme-to-protein ratio and at room temperature. Digested samples were acidified with formic acid (FA; Sigma, F0507) to achieve a final with formic acid (FA; Sigma, F0507) to achieve a final volumetric concentration of 1% (final pH of ~3), and centrifuged at 1,500 g for 15 minutes to clear precipitated urea from peptide lysates. Samples were desalted on tC18 SepPak columns (Waters, 200mg, Samples were desalted on tC18 SepPak columns (Waters, 200mg, WAT054925) and dried down using a SpeedVac apparatus. Common Reference (CR) Pool and Plex layout\\n\\nProteome, Phosphoproteome, and Acetyprotome: The proteomic, phosphoproteomic and acetylproteome analyses of LSCC samples consisted of 22 TMT-11 plex experiments. To facilitate quantitative comparison between all samples across experiments, a common reference (CR) sample was included in each 11-plex. a common reference (CR) sample was included in each 11-plex. A common physical, rather than in silico reference was used for this purpose for optimal quantitative precision between TMT-11 experiments. Considerations prior to creating the reference sample Considerations prior to creating the reference sample were that this sample needed to be of adequate quantity to cover all planned experiments for both the discovery and future confirmatory sets (not part of this study) with overhead for additional sets (not part of this study) with overhead for additional possible experiments. The CR included all (212/212) of the samples analyzed in the discovery set TMT experiments, yielding a CR that was representative of all the samples in the study. Making the was representative of all the samples in the study. Making the CR as representative of the study as a whole was particularly important since by definition only analytes represented in the reference sample would be included in the final ratio-based data sample would be included in the final ratio-based data analyses. 110 unique tumor samples with 102 paired NAT samples are distributed among 22 11-plex experiments, with 10 individual samples occupying the first 10 channels of each experiment and the 11th channel being reserved for the reference sample. All the tumors channel being reserved for the reference sample. All the tumors were in the C channels and all the normal samples were in the N channels. The first 8 channels of each experiment contained 4 tumor/normal pairs, with each pair of patient samples adjacent to pairs, with each pair of patient samples adjacent to each other. Channels 9 and 10 (130C and 131N) either contained a 5th tumor/normal pair, or an unpaired tumor sample in channel 9 and an alternate common reference sample in channel 10. These included 3 common reference sample in channel 10. These included 3 global LUAD common references, 3 HNSCC common references from the team at JHU, and 2 Tumor only common references from this LSCC discovery set. To ensure capacity for additional samples or experiments given a target input of 300 \\u03bcg peptide per channel per experiment, 18 mg total was targeted for reference material. To meet these collective requirements, all the samples in the discovery set that requirements, all the samples in the discovery set that had enough material were included in the CR. After reserving a conservative 400 \\u03bcg peptide / sample for individual sample analysis, an additional amount of 120 \\u03bcg for each of the samples with an additional amount of 120 \\u03bcg for each of the samples with adequate quantities were pooled. All 212 samples were selected for the combined tumor/normal CR. To make the combined CR, tumor only and normal only CRs were first created separately, with 110 and normal only CRs were first created separately, with 110 tumor samples and 102 normal samples. After creating tumor and NAT only CRs at 150 ug and 150ug amounts, a pool of combined CR was made, comprised of 8.6 mg tumor only reference and 8.6 mg normal comprised of 8.6 mg tumor only reference and 8.6 mg normal only reference. The 17.2 mg pooled reference material was divided into 55 300 \\u03bcg aliquots, dried down, and frozen at \\u221280\\u00b0C until use. Ubiquitylproteome: 750 ug of unlabeled peptides was required from each sample to ensure depth of coverage in the ubiquitylproteome, which restricted the number of samples available for profiling. 147/212 samples from the proteome, phosphoproteome, acetylproteome set were from the proteome, phosphoproteome, acetylproteome set were included in the ubiquitylproteome analysis. A separate ubiquitylproteome CR was constructed prior to KGG enrichment, due to limitations in sample material (both patient material and remaining CR in sample material (both patient material and remaining CR aliquots to be used in future LSCC proteome, phosphoproteome, and acetylproteome processing.. In order to make TMT ratio-based quantitation comparable across omes, the ubiquitylproteome CR was comparable across omes, the ubiquitylproteome CR was constructed to be as similar as practically possible to the proteome, phosphoproteome, acetylproteome CR. The ubiquitylproteome CR contains all samples from the proteome, phosphoproteome, acetylproteome all samples from the proteome, phosphoproteome, acetylproteome set with material remaining to contribute, 209/212. The ubiquitylproteome CR contributions were of 3 types: 104/147 samples included in ubiquitylproteome study and made full CR contribution, >1000ug available.\\n\\n42/147 samples included in ubiquitylproteome study and made partial CR contribution, 750ug \\u22121000 ug available. 63/65 samples from the proteome, phosphoproteome, acetylproteome set not included in ubiquitylproteome study, but made full CR contribution, <750 ug available. CR tumor and normal peptides were combined at a ratio of 55:45 to reflect the higher proportion of tumor samples in the experimental design (as opposed to proteome, phosphoproteome, acetylproteome CR\\u2019s proportion of 50:50), and aliquoted into 750 ug CR\\u2019s proportion of 50:50), and aliquoted into 750 ug aliquots. In the ubiquitylproteome experimental design, 15 11-plex experiments contained 87 unique tumor samples in the C channels and 60 normal samples in the N channels, with the first 8 channels of every 11-plex consisting of patient-paired tumor/normal samples every 11-plex consisting of patient-paired tumor/normal samples adjacent to each other. The 11th channel was reserved for the ubiquitylproteome common reference sample. Channel 9 (130C) contained an unpaired tumor, while channel 10 (131N) contained either an unpaired tumor, while channel 10 (131N) contained either an unpaired tumor, or in 3 plexes, a LUAD ubiquitylproteome common reference (which was constructed similarly for that experiment, unpublished). TMT-11 Labeling of Peptides\\n\\nProteome, Phosphoproteome, Acetylproteome: 300 \\u03bcg of desalted peptides per sample (based on peptide-level BCA after digestion) were labeled with 11-plex TMT reagents and combined for multiplexed analysis using a \\u201creduced labeling\\u201d approach (Zecha et al., 2019). To conservatively yield extra approach (Zecha et al., 2019). To conservatively yield extra labeled peptide in case a given sample demonstrated unsuitably low TMT reporter ion intensity during channel mixing control LC-MS/MS runs, 400 ug peptides per sample were initially labeled, with runs, 400 ug peptides per sample were initially labeled, with only 300 ug combined per sample for the final multiplexed experiments upon successful quality control. For each 400 \\u03bcg peptide aliquot of an individual sample, peptides were reconstituted to 5 of an individual sample, peptides were reconstituted to 5 mg/ml with 50 mM HEPES (pH 8.5) solution and 400 ug labeling reagent dissolved in 20 uL anhydrous acetonitrile was added. After 1 h incubation with shaking, samples were diluted 1:1 with a solution with shaking, samples were diluted 1:1 with a solution of 80% 50 mM HEPES and 20% acetonitrile. After confirming good label incorporation, 5 \\u03bcL of 5% hydroxylamine was added to quench the unreacted TMT reagents. Good label incorporation was defined as unreacted TMT reagents. Good label incorporation was defined as having less than 3% isobaric under-labeled MS/MS spectra in each sample, as measured by LC-MS/MS after taking out a 2 \\u03bcg aliquot from each sample and analyzing 0.25 \\u03bcg. If a sample did not from each sample and analyzing 0.25 \\u03bcg. If a sample did not have sufficient label incorporation, additional TMT was added to the sample and another 1 h incubation was performed with shaking. At the time that the labeling efficiency quality control samples the time that the labeling efficiency quality control samples were taken out, an additional 4 \\u03bcg of material from each sample was taken out and combined as a mixing control. After analyzing the mixing control sample by LC-MS/MS, intensity values of the the mixing control sample by LC-MS/MS, intensity values of the individual TMT reporter ions were summed across all peptide spectrum matches and compared to ensure that the total reporter ion intensity of each sample met a threshold of +/\\u2212 15% of the ion intensity of each sample met a threshold of +/\\u2212 15% of the internal reference. If necessary, adjustments were made by either using additional labeled material or reducing an individual sample\\u2019s contribution to the mixture, and analyzing a subsequent contribution to the mixture, and analyzing a subsequent mixing control, until all samples met the threshold and were thus approximately 1:1:1.... Differentially labeled peptides were then mixed (11 \\u00d7 300 \\u03bcg), dried down via vacuum centrifuge, and the mixed (11 \\u00d7 300 \\u03bcg), dried down via vacuum centrifuge, and the quenched, combined sample was subsequently desalted on a 200 mg tC18 SepPak column. The same lot of TMT reagents was used for all samples (UA280170/TL272399). Peptide Fractionation\\n\\nFor Proteome, Phosphoproteome, Acetylproteome: To reduce sample complexity, peptide samples were separated by high pH reversed phase (RP) separation as described previously (Mertins et al., 2018). A desalted 3.3 mg, 11-plex TMT-labeled experiment (based on peptide-level BCA after digestion) was experiment (based on peptide-level BCA after digestion) was reconstituted in 935 \\u03bcL 4.5mM ammonium formate (pH 10) and 2% acetonitrile, centrifuged for 7 min at 20000 g to precipitate insoluble peptides, loaded on a 4.6 mm x 250 mm column RP Zorbax 300 A peptides, loaded on a 4.6 mm x 250 mm column RP Zorbax 300 A Extend-C18 column (Agilent, 3.5 \\u03bcm bead size), and separated on an Agilent 1260 Series HPLC instrument using basic reversed-phase chromatography. Solvent A (2% acetonitrile, 4.5 mM ammonium chromatography. Solvent A (2% acetonitrile, 4.5 mM ammonium formate, pH 10) and a nonlinear increasing concentration of solvent B (90% acetonitrile, 4.5 mM ammonium formate, pH 10) were used to separate peptides. The 4.5 mM ammonium formate solvents were to separate peptides. The 4.5 mM ammonium formate solvents were made by 40\\u2013fold dilution of a stock solution of 180 mM ammonium formate, pH 10. To make 1L of stock solution, add 25 mL of 28% (wt/vol) ammonium hydroxide (28%, density 0.9 g/ml, mL of 28% (wt/vol) ammonium hydroxide (28%, density 0.9 g/ml, Sigma-Aldrich) to ~850ml of HPLC grade water, then add ~35 mL of 10 % (vol/vol) formic acid (>95% Sigma-Aldrich) to titrate the pH to 10.0; bring the final volume to 1 liter with HPLC grade pH to 10.0; bring the final volume to 1 liter with HPLC grade water. The 96 minute separation LC gradient followed this profile: (min: %B) 0:0; 7:0; 13:16; 73:40; 77:44; 82:60; 96:60. The flow rate was 1 mL/min. Per 3.3 mg separation, 82 fractions were rate was 1 mL/min. Per 3.3 mg separation, 82 fractions were collected into a 96 deep-well x 2mL plate (Whatman, #7701\\u20135200), with fractions combined in a step-wise concatenation strategy and acidified to a final concentration of 0.1% FA as reported and acidified to a final concentration of 0.1% FA as reported previously. An additional 14 fractions were collected from the 96 deep-well plate for fraction A, which are the early eluting fractions that tend to contain multi-phosphorylated peptides. 5% of that tend to contain multi-phosphorylated peptides. 5% of the volume of each of the 24+A proteome fractions was allocated for proteome analysis, dried down, and re-suspended in 3% MeCN/0.1% FA (MeCN; acetonitrile) to a peptide concentration of 0.25 \\u03bcg/ \\u03bcL (MeCN; acetonitrile) to a peptide concentration of 0.25 \\u03bcg/ \\u03bcL for LC-MS/MS analysis. The remaining 95% of concatenated 24 fractions were further combined into 12 fractions, with fraction A as a separate fraction. These 13 fractions were then enriched for a separate fraction. These 13 fractions were then enriched for phosphopeptides as described below. Phosphopeptide Enrichment Ni-NTA agarose beads were used to prepare Fe3+-NTA agarose beads. In each phosphoproteome fraction, ~261.25 \\u03bcg peptides (based on peptide-level BCA after digestion with uniformly-distributed fractionation presumed) were reconstituted in 522.5 \\u03bcL 80% fractionation presumed) were reconstituted in 522.5 \\u03bcL 80% MeCN/0.1% TFA (trifluoroacetic acid) solvent and incubated with 10 \\u03bcL of the IMAC beads for 30 minutes with end-over-end rotation at RT. After incubation, samples were briefly spun down on a at RT. After incubation, samples were briefly spun down on a tabletop centrifuge; clarified peptide flow-throughs were separated from the beads; and the beads were reconstituted in 200 \\u03bcL IMAC binding/wash buffer (80 MeCN/0.1% TFA) and loaded onto \\u03bcL IMAC binding/wash buffer (80 MeCN/0.1% TFA) and loaded onto equilibrated Empore C18 silica-packed stage tips (3M, 2315). Samples were then washed twice with 50 \\u03bcL of IMAC binding/wash buffer and once with 50 uL 1% FA, and were eluted from the IMAC buffer and once with 50 uL 1% FA, and were eluted from the IMAC beads to the stage tips with 3 \\u00d7 70 uL washes of 500 mM dibasic sodium phosphate (pH 7.0, Sigma S9763). Stage tips were then washed once with 100 \\u03bcL 1% FA and phosphopeptides were eluted from once with 100 \\u03bcL 1% FA and phosphopeptides were eluted from the stage tips with 60 \\u03bcL 50% MeCN/0.1% FA. Phosphopeptides were dried down and re-suspended in 9 \\u03bcL 50% MeCN/0.1%FA for LC-MS/MS analysis, where 4 \\u03bcL was injected per run. Acetylpeptide Enrichment Acetylated lysine peptides were enriched using an antibody against the acetyl-lysine motif (CST PTM-SCAN Catalogue No. 13416). IMAC eluents were concatenated into 4 fractions (~783.75 \\u03bcg peptides per fraction) and dried down using a SpeedVac apparatus. per fraction) and dried down using a SpeedVac apparatus. Peptide fractions were reconstituted with 1.4ml of IAP buffer (5 mM MOPS pH 7.2, 1 mM Sodium Phosphate (dibasic), 5 mM NaCl) per fraction and incubated for 2 hours at 4oC with pre-washed (3 times and incubated for 2 hours at 4oC with pre-washed (3 times with IAP buffer) agarose beads bound to acetyl-lysine motif antibody. Peptide-bound beads were washed 4 times with ice-cold PBS followed by elution with 2 \\u00d7 100ul of 0.15% TFA. Eluents were followed by elution with 2 \\u00d7 100ul of 0.15% TFA. Eluents were desalted using C18 stage tips, eluted with 50% ACN and dried down. Acetylpeptides were suspended in 7ul of 0.1% FA and 3% ACN and 4ul were injected per run. Ubiquitylpeptide Enrichment with on-bead TMT labeling Ubiquitin enrichment was performed based on the recently published UbiFast protocol. Before enrichment, anti-K-\\u03b5-GG bead-bound antibodies from the PTM-Scan ubiquitin remnant motif kit (Cell Signaling Technologies, Kit #5562) were cross-linked. Beads were Technologies, Kit #5562) were cross-linked. Beads were briefly washed 3X with 100 mM sodium borate (pH 9.0) and incubated with 20 mM DMP for 30 min at RT. Beads were then washed 2X with 200 mM ethanolamine and incubated overnight at 4\\u00b0C in 200 mM 200 mM ethanolamine and incubated overnight at 4\\u00b0C in 200 mM ethanolamine with end-over-end rotation. Following incubation, beads were washed 3X with immunoprecipitation (IAP) buffer (50 mM MOPS, pH 7.2, 10 mM sodium phosphate, 50 mM NaCl) and stored at MOPS, pH 7.2, 10 mM sodium phosphate, 50 mM NaCl) and stored at 4\\u00b0C at a concentration of 0.5 ug/uL. For each 11-plex experiment, 31.25 ug of cross-linked anti-K-GG bead-bound antibody at 0.5 ug/uL in IAP per channel was aliquoted into 1.5 mL Eppendorf tubes on ice. 750 ug peptide samples were reconstituted to 0.5 mg/mL concentration in IAP buffer and were reconstituted to 0.5 mg/mL concentration in IAP buffer and vortexed for 10 min. Peptides were then centrifuged for 5 min at 5,000g. Each peptide solution was added to a tube of antibody and gently rotated end-over-end at 4oC for 1 hour. Following and gently rotated end-over-end at 4oC for 1 hour. Following enrichment, samples were spun at 2000 rcf for ~1 min to settle and the supernatant was removed and stored as flow-through, and from this point beads were kept on ice unless being handled. from this point beads were kept on ice unless being handled. Antibody beads were washed with 1.5 mL ice cold IAP followed by 1.5 mL ice cold PBS. For all washes, after adding wash reagent each sample was inverted ~5 times, agitated by tapping pairs of each sample was inverted ~5 times, agitated by tapping pairs of tubes together, spun at 2000 rcf for about 30\\u201340 s, left to settle on ice for 10\\u201320 s, and supernatant was removed. Importantly, all washes were completed as quickly as possible. Importantly, all washes were completed as quickly as possible. Pre-aliquoted TMT @ 40 ug/uL in 100% MeCN was retrieved and briefly equilibrated to RT while beads were reconstituted in 200 uL room temp. 100 mM HEPES buffer. For each sample, 400 ug of TMT room temp. 100 mM HEPES buffer. For each sample, 400 ug of TMT labeling reagent in 10 uL acetonitrile was added. Samples were labeled on-antibody while shaking vigorously (1400 rpm) at 20\\u00b0C for 10 minutes, then quenched with 8 uL 5% hydroxylamine and for 10 minutes, then quenched with 8 uL 5% hydroxylamine and shaken vigorously for another 5 minutes. Labeled antibody-bound peptides were then put back on ice, washed once with 1.3 mL cold IAP, and again with 1.5 mL cold IAP. Each channel was resuspended and again with 1.5 mL cold IAP. Each channel was resuspended and transferred to a combination tube with 130 uL cold IAP. Following combination, each now-empty tube was serially washed with 1.5 mL cold IAP to remove remaining beads, and this 1.5 mL IAP was mL cold IAP to remove remaining beads, and this 1.5 mL IAP was added to the combination tube and used to wash the combined beads. Combined beads were washed one final time with 1.5 mL ice cold PBS. Once the channels were combined and washed, peptides were PBS. Once the channels were combined and washed, peptides were eluted twice from the beads by resuspending with 150 uL room temp. 0.15% TFA and incubating 5 min at RT. Each round of acid-eluted KGG-modified peptides was desalted on an equilibrated KGG-modified peptides was desalted on an equilibrated two-punch C18 stage tip. Both elutions of KGG peptides were loaded sequentially, washed 2X with 100 uL 0.1% FA, and eluted into an MS vial with 50 uL 50% ACN/0.1% FA. The eluted peptides were frozen, with 50 uL 50% ACN/0.1% FA. The eluted peptides were frozen, lyophilized, and reconstituted in 9 uL 3% ACN/0.1% FA, with 4 uL injected twice for two LC-MS/MS runs on a Thermo Lumos instrument. While the same lot of TMT reagents was used for all instrument. While the same lot of TMT reagents was used for all ubiquitylproteome samples (TE270748-TD264064), it is a different lot than that used for Proteome, Phosphoproteome, and acetylproteome samples. LC-MS/MS for Proteomics Analyses Online separation was done with a nanoflow Proxeon EASY-nLC 1200 UHPLC system (Thermo Fisher Scientific). In this set up, the LC system, column, and platinum wire used to deliver electrospray source voltage were connected via a stainless steel cross source voltage were connected via a stainless steel cross (360\\u03bcm, IDEX Health & Science, UH-906x). The column was heated to 50\\u00b0C using a column heater sleeve (Phoenix-ST) to prevent over-pressuring of columns during UHPLC separation. From each peptide of columns during UHPLC separation. From each peptide fraction, ~1ug (based on protein-level BCA prior to digestion with uniformly-distributed fractionation presumed), the equivalent of 12% of each global proteome sample in a 2 ul injection volume or 50% each global proteome sample in a 2 ul injection volume or 50% of each phosphoproteome, acetylproteome, or ubiquitylproteome sample in a 4 ul injection volume, was injected onto an in-house packed 22cm x 75um internal diameter C18 silica picofrit capillary 22cm x 75um internal diameter C18 silica picofrit capillary column (1.9 \\u03bcm ReproSil-Pur C18-AQ beads, Dr. Maisch GmbH, r119.aq; Picofrit 10um tip opening, New Objective, PF360\\u201375-10-N-5). Mobile phase flow rate was 200 nL/min, comprised of 3% Mobile phase flow rate was 200 nL/min, comprised of 3% acetonitrile/0.1% formic acid (Solvent A) and 90% acetonitrile /0.1% formic acid (Solvent B). The 110-minute LC-MS/MS method consisted of a 10-min column-equilibration procedure; a 20-min consisted of a 10-min column-equilibration procedure; a 20-min sample-loading procedure; and the following gradient profile: (min:%B) 0:2; 1:6; 85:30; 94:60; 95;90; 100:90; 101:50; 110:50 (the last two steps at 500 nL/min flow rate). For acetylproteome last two steps at 500 nL/min flow rate). For acetylproteome analysis, the same LC and column setup was used, but the gradient was extended to 260 minutes with the following gradient profile: (min:%B) 0:2; 1:6; 235:30; 244:60; 245;90; 250:90; 251:50; (min:%B) 0:2; 1:6; 235:30; 244:60; 245;90; 250:90; 251:50; 260:50 (the last two steps at 500 nL/min flow rate). For ubiquitylproteome analysis, the same LC and column setup was used, but the gradient was 154-minutes with the following gradient profile: gradient was 154-minutes with the following gradient profile: (min:%B) 0:2; 2:6; 122:35; 130:60; 133;90; 143:90; 144:50; 154:50 (the last two steps at 500 nL/min flow rate). For proteome and acetylproteome analysis, samples were analyzed with a benchtop Q Exactive HF-X mass spectrometer (Thermo Fisher Scientific) equipped with a nanoflow ionization source (James A. Hill Instrument Services, Arlington, MA). Data-dependent A. Hill Instrument Services, Arlington, MA). Data-dependent acquisition was performed using Q Exactive HF-X Orbitrap v 2.9 software in positive ion mode at a spray voltage of 1.5 kV. MS1 Spectra were measured with a resolution of 60,000, an AGC target of were measured with a resolution of 60,000, an AGC target of 3e6 and a mass range from 350 to 1800 m/z. The data-dependent mode cycle was set to trigger MS/MS on up to the top 20 most abundant precursors per cycle at an MS2 resolution of 45,000, an AGC precursors per cycle at an MS2 resolution of 45,000, an AGC target of 5e4, an isolation window of 0.7 m/z, a maximum injection time of 105 msec, and an HCD collision energy of 31%. Peptides that triggered MS/MS scans were dynamically excluded from further triggered MS/MS scans were dynamically excluded from further MS/MS scans for 45 sec for proteome or 30 sec for acetylproteome. Peptide match was set to preferred for monoisotopic peak determination, and charge state screening was enabled to only include and charge state screening was enabled to only include precursor charge states 2\\u20136, with an intensity threshold of 9.5e4 for proteome or 8e4 for acetylproteome. Advanced precursor determination feature (APD) (Myers et al., 2018) was turned off using a feature (APD) (Myers et al., 2018) was turned off using a software patch provided to us by Thermo Fisher Scientific allowing us to turn APD off in the tune file, Tune version 2.9.0.2926 (later versions of Exactive Tune 2.9 sp2 for the HFX have this option versions of Exactive Tune 2.9 sp2 for the HFX have this option as standard). For phosphoproteome and ubiquitylproteome analysis, samples were analyzed with a benchtop Orbitrap Fusion Lumos mass spectrometer (Thermo Fisher Scientific) equipped with a NanoSpray Flex NG ion source. Data-dependent acquisition was performed using NG ion source. Data-dependent acquisition was performed using Xcalibur Orbitrap Fusion Lumos v3.0 software in positive ion mode at a spray voltage of 1.8 kV. MS1 Spectra were measured with a resolution of 60,000, an AGC target of 4e5 and a mass range from of 60,000, an AGC target of 4e5 and a mass range from 350 to 1800 m/z. The data-dependent mode cycle time was set at 2 seconds with a MS2 resolution of 50,000, an AGC target of 6e4, an isolation window of 0.7 m/z, a maximum injection time of 105 msec (125 window of 0.7 m/z, a maximum injection time of 105 msec (125 msec for ubiquitylproteome), and an HCD collision energy of 36%. Peptide mode was selected for monoisotopic peak determination, and charge state screening was enabled to only include precursor charge state screening was enabled to only include precursor charge states 2\\u20136, with an intensity threshold of 1e4. Peptides that triggered MS/MS scans were dynamically excluded from further MS/MS scans for 45 sec (20 sec for ubiquitylproteome), with a MS/MS scans for 45 sec (20 sec for ubiquitylproteome), with a +/\\u2212 10 ppm mass tolerance. \\u201cPerform dependent scan on single charge state per precursor only\\u201d was enabled for phosphoproteome analysis, but disabled for ubiquitylproteome analysis. Immunohistochemistry (IHC) analysis.\\n\\nTMA construction Seventy-one LSCC paraffin tissue blocks were available for tissue microarray (TMA) construction and whole sections by immunohistochemistry (IHC) assay. TMAs were constructed in duplicate, and each included two (2) 1.0 mm cores providing up to four (4) and each included two (2) 1.0 mm cores providing up to four (4) cores per case for IHC analysis. Common control tissue on each TMA included patient-matched normal adjacent lung from 14 cases. Immune cell marker, Immunohistochemistry (IHC) Seventeen (17) samples were scored for immune cell markers. Sequential TMA sections were obtained for immune marker assessment to best approximate native spatial relationships. Chromogenic (DAB) IHC by single marker analysis was performed on 5-micron (DAB) IHC by single marker analysis was performed on 5-micron thick sections on standard charged slides. Automated immunostaining was performed by Autostainer Link 48, Dako, Inc. Heat induced epitope retrieval (HIER) was performed and biomarker dilutions epitope retrieval (HIER) was performed and biomarker dilutions as follows: CD4 clone 4B12, Bio-Rad 1:50; CD8 clone 4B11, Bio-Rad, 1:50; CD163, Abcam, 1:500 and ARHGDIB, HPA051235 Sigma-Aldrich, 1:50. IHC analysis was performed independently by three 1:50. IHC analysis was performed independently by three independent pathologists by staining intensity of 0\\u20133 scale for each TMA core separately and then by average score / case for a total of 15 cases. Case scores compared across independent pathologists 15 cases. Case scores compared across independent pathologists showed very good agreement. For evaluating statistical significance of IHC scores across the three immune clusters, ordinal regression was used, implemented using the \\u201cordinal\\u201d package in R. was used, implemented using the \\u201cordinal\\u201d package in R. Random effects accounted for the correlation between multiple cores from each sample and multiple pathologists. Multiplex Immunofluorescence (mIF) staining Development and optimization of the mIF platform has been previously described. mIF staining was performed in a 4-\\u03bcm-thick section obtained from a FFPE TMA block, using the Opal 7-Color Kit (Akoya Biosciences, Waltham, MA, USA) and scanned using a Vectra Biosciences, Waltham, MA, USA) and scanned using a Vectra multispectral microscope (Akoya Biosciences). The IF markers used were pan-Cytokeratin (CK, clone AE1/AE3, (DAKO) and Alpha-SMA ab5694 (Abcam). Multiplexed stained sections were imaged using the (Abcam). Multiplexed stained sections were imaged using the VECTRA multispectral imaging system (Akoya Biosciences). TMA cores were annotated using InForm 2.4.8 image analysis software (Akoya Biosciences). IHC based assessment of TP63 low samples (Figure 4) For samples with low TP63 status, additional pathologic confirmation was performed. Pathology reassessment confirmed 8/10 samples to be unequivocal LSCC; one sample (C3N.00247) had adenosquamous but predominantly squamous histology; the remaining sample but predominantly squamous histology; the remaining sample (C3N.02283) showed basaloid squamous cell features and despite some atypical characteristics retained its LSCC attribution after review by four separate pathologists - with histology very similar by four separate pathologists - with histology very similar to a TP63-low TCGA tumor (TCGA-66\\u20132754) deemed LSCC by TCGA pathology. QUANTIFICATION AND STATISTICAL ANALYSIS\\n\\nGenomic Data Analysis\\n\\nCopy Number Calling Copy-number analysis was performed jointly leveraging both whole-genome sequencing (WGS) and whole-exome sequencing (WES) data of the tumor and germline DNA, using CNVEX (https://github.com/mctp/cnvex). CNVEX uses whole-genome aligned reads to estimate CNVEX uses whole-genome aligned reads to estimate coverage within fixed genomic intervals, and whole-genome and whole-exome variant calls to compute B-allele frequencies at variable positions (we used TNScope germline calls). Coverages were computed in (we used TNScope germline calls). Coverages were computed in 10kb bins, and the resulting log coverage ratios between tumor and normal samples were adjusted for GC bias using weighted LOESS smoothing across mappable and non-blacklisted genomic intervals smoothing across mappable and non-blacklisted genomic intervals within the GC range 0.3\\u20130.7, with a span of 0.5 (the target, blacklist, and configuration files are provided with CNVEX). The adjusted log coverage ratios (LR) and B-allele frequencies (BAF) log coverage ratios (LR) and B-allele frequencies (BAF) were jointly segmented by custom algorithm based on Circular Binary Segmentation (CBS). Alternative probabilistic algorithms were implemented in CNVEX, including algorithms based on recursive binary in CNVEX, including algorithms based on recursive binary segmentation (RBS), and dynamic programming, as implemented in the R-package jointseg. For the CBS-based algorithm, first LR and mirrored BAF were independently segmented using CBS (parameters mirrored BAF were independently segmented using CBS (parameters alpha=0.01, trim=0.025) and all candidate breakpoints collected. The resulting segmentation track was iteratively \\u201cpruned\\u201d by merging segments that had similar LR, BAFs and short lengths. For segments that had similar LR, BAFs and short lengths. For the RBS- and DP-based algorithms, joint-breakpoints were \\u201cpruned\\u201d using a statistical model selection method. For the final set of CNA segments, we chose the CBS-based results as they did not of CNA segments, we chose the CBS-based results as they did not require specifying a prior on the number of expected segments (K) per chromosome arm, were robust to unequal variances between the LR and BAF tracks, and provided empirically the best fit to the LR and BAF tracks, and provided empirically the best fit to the underlying data. Somatic Variant Calling Somatic variant calling was performed on exome-capture BAM files aligned to the GDC GRCh38 reference. Following de-duplication and co-realignment around known indels \\u201cGATK bundle known\\u201d and \\u201cMills 1000G\\u201d somatic variants were called using Sentieon TNScope 1000G\\u201d somatic variants were called using Sentieon TNScope with the following settings \\u201c--max_fisher_pv_active 0.05 --min_tumor_allele_frac 0.0075 --min_init_tumor_lod 2.5 --assemble_mode 4 --SVIntrusionThres 5\\u201d, with a provided germline variants from 4 --SVIntrusionThres 5\\u201d, with a provided germline variants from dbSNP138. The resulting VCF files were annotated and filtered, in a series of post-processing steps which included: 1) restriction of variants to the hybrid-capture target regions. 2) Phasing of variants to the hybrid-capture target regions. 2) Phasing of adjacent and separated by 1bp SNVs into multi-nucleotide variants based on the haplotype information provided by TNScope. 3) Annotation using VEP with the following settings \\u201c--assembly 3) Annotation using VEP with the following settings \\u201c--assembly GRCh38 --species homo_sapiens --cache_version 97 --format vcf --gene_phenotype --symbol --canonical --ccds --hgvs --biotype --tsl --uniprot --domains --appris --protein --variant_class --sift --uniprot --domains --appris --protein --variant_class --sift b, --polyphen b --no_stats --total_length --allele_number --no_escape --flag_pick_allele --pick_order canonical,tsl,biotype,rank,ccds,length --buffer_size 20000\\u201d. Additional annotations were --buffer_size 20000\\u201d. Additional annotations were attached to the variants using VcfAnno including allele frequencies in GNOMAD (v2.0.1), ClinVar (downloaded Dec. 2019), Cosmic (downloaded Dec. 2019), dbSNP (20180418). The resulting variants are annotated 2019), dbSNP (20180418). The resulting variants are annotated against both RefSeq and Ensembl transcripts and contain multiple predicted consequences per-variant. For most downstream applications these annotations were reduced to a single predicted these annotations were reduced to a single predicted consequence per-variant and converted into a MAF file using custom software. The choice of the most-relevant consequence was based on a number of features. In general, the most \\u2018severe\\u2019 consequence a number of features. In general, the most \\u2018severe\\u2019 consequence occuring in a \\u2018reliable\\u2019 transcript was chosen. To accomplish this the computed consequences were ranked by \\u2018severity\\u2019 and \\u2018priority\\u2019 reflecting how deleterious/impactful a variant and \\u2018priority\\u2019 reflecting how deleterious/impactful a variant consequence is and how well-supported the existence and biological relevance of an isoform is. The specific rules differed for RefSeq and Ensembl annotations. RefSeq transcripts were ranked RefSeq and Ensembl annotations. RefSeq transcripts were ranked higher if they were known to CCDS, LRG or ranked \\u2018CANONICAL\\u2019 or \\u2018PICK\\u2019 by VEP. Ensembl transcripts were further prioritized by Basic, Appris and TSL annotations. Ensembl-based consequences Basic, Appris and TSL annotations. Ensembl-based consequences were prefered in case of ties. The resulting variant sets were filtered to only include variants with a high likelihood of being somatic based on a set of probabilistic and heuristic of being somatic based on a set of probabilistic and heuristic thresholds. A reported variant had to pass all of the following filters: 5 or more variant reads, fewer than 2 variant reads in the germline allele frequency in the normal 0.005, allele in the germline allele frequency in the normal 0.005, allele frequency in the tumor greater than 0.025, TLOD > 5, NLOD > 6, Fisher\\u2019s test p-value < 0.05, NFLOD > 1.5. Additionally heuristic filters were implemented to remove strand-bias, variants with filters were implemented to remove strand-bias, variants with mapping quality bias, and variants with high population allele frequency in GNOMAD. Germline Variant Calling Germline variant calling and annotation was carried out analogously using DNAScope using the following settings \\u201c--emit_conf 10 --call_conf 10\\u201d to generate a VCF file and \\u201c--emit_mode GVCF\\u201d to generate a gVCF file. The variants were custom-filtered based generate a gVCF file. The variants were custom-filtered based on the specific applications. GISTIC and MutSig analysis The Genomic Identification of Significant Targets in Cancer (GISTIC2.0) algorithm was used to identify significantly amplified or deleted focal-level and arm-level events, with Q value <0.25 considered significant. The following parameters were used: Amplification Threshold = 0.1\\n\\nDeletion Threshold = 0.1\\n\\nCap Values = 1.5\\n\\nBroad Length Cutoff = 0.98\\n\\nRemove X-Chromosome = 0\\n\\nConfidence Level = 0.99\\n\\nJoin Segment Size = 4\\n\\nArm Level Peel Off = 1\\n\\nMaximum Sample Segments = 2000\\n\\nGene GISTIC = 1 Each gene of every sample is assigned a thresholded copy number level that reflects the magnitude of its deletion or amplification. These are integer values ranging from \\u22122 to 2, where 0 means no amplification or deletion of magnitude greater than the no amplification or deletion of magnitude greater than the threshold parameters described above. Amplifications are represented by positive numbers: 1 means amplification above the amplification threshold; 2 means amplification larger than the arm level threshold; 2 means amplification larger than the arm level amplifications observed in the sample. Deletions are represented by negative numbers: \\u22121 means deletion beyond the threshold; \\u22122 means deletions greater than the minimum arm-level copy number means deletions greater than the minimum arm-level copy number observed in the sample. The somatic variants were filtered through a panel of normals to remove potential sequencing artifacts and undetected germline variants. MutSig2CV was run on these filtered results to evaluate the significance of mutated genes and estimate mutation the significance of mutated genes and estimate mutation densities of samples. These results were constrained to genes in the Cancer Gene Census, with false discovery rates (q values) recalculated. Genes of q value < 0.1 were declared significant. RNAseq and miRNAseq Quantification\\n\\nRNAseq Quantification The readcount is generated by featureCounts (subread v1.6.4) under stranded mode with parameters: `-g gene_id -t exon -Q 10 -p -B -s 2`. Gene annotation in use is identical to GDC (`gencode.v22.annotation.gtf.gz`; md5: `291330bdcff1094bc4d5645de35e0871`), md5: `291330bdcff1094bc4d5645de35e0871`), which is available on the GDC Reference Files page (https://gdc.cancer.gov/about-data/gdc-data-processing/gdc-reference-files). The readcount is later converted to FPKM and FPKM-UQ using GDC\\u2019s formula is later converted to FPKM and FPKM-UQ using GDC\\u2019s formula (https://docs.gdc.cancer.gov/Data/Bioinformatics_Pipelines/Expression_mRNA_Pipeline/). Isoform specific RNA quantification For TP63 isoform centric analysis presented in Figure 4, the hg38 reference genome and RefSeq annotations were used for the RNAseq data analysis and were downloaded from the UCSC table browser. First, CIRI (v2.0.6) was used to call circular RNA with First, CIRI (v2.0.6) was used to call circular RNA with default parameters and BWA (version 0.7.17-r1188) was used as the mapping tool. The cutoff of supporting reads for circRNAs was set to 10. Then we used a pseudo-linear transcript strategy to quantify Then we used a pseudo-linear transcript strategy to quantify gene and circular RNA expression. In brief, for each sample, linear transcripts of circular RNAs were extracted and 75bp (read length) from the 3\\u2019 end was copied to the 5\\u2019 end. The modified length) from the 3\\u2019 end was copied to the 5\\u2019 end. The modified transcripts were called pseudo-linear transcripts. Transcripts of linear genes were also extracted and mixed with pseudo-linear transcripts. RSEM (version 1.3.1) with Bowtie2 (version 2.3.3) transcripts. RSEM (version 1.3.1) with Bowtie2 (version 2.3.3) as the mapping tool was used to quantify gene, isoform, and circular RNA expression based on the mixed transcripts. DNA methylation data preprocessing Raw methylation image files were downloaded from the CPTAC DCC (See data availability). We calculated and analyzed methylated (M) and unmethylated (U) intensities for LSCC samples as described previously. We flagged locus as NA where probes did not meet a previously. We flagged locus as NA where probes did not meet a detection p-value of 0.01. Probes with MAF more than 0.1 were removed, and samples with more than 85% NA values were removed. Resulting beta values of methylation were utilized for subsequent beta values of methylation were utilized for subsequent analysis. Gene-level methylation scores were generated by taking the mean beta values of probes in the CpG islands of promoters and 5\\u2019 UTR regions of the gene. Methylation profiles (i.e., density plots) of some samples had unexpectedly skewed distributions of plots) of some samples had unexpectedly skewed distributions of methylation beta values, in addition to significantly more missing values. To systematically determine the subset of methylation samples with these evident data QC issues, we subjected all samples with these evident data QC issues, we subjected all the samples to model-based clustering using the Mclust package in R, using the interquartile range over all the genes as the representative metric. The clustering automatically determined the metric. The clustering automatically determined the optimal number of clusters, and identified 3 clusters. Two of these clusters (with centroids at 0.029 and 0.038) captured the bulk of the samples (202). The third cluster (centroid at 0.165, of the samples (202). The third cluster (centroid at 0.165, significantly higher than the other two clusters) consisted of 4 samples that belong to this cluster with high confidence (uncertainty < 5%), each of which had a skewed distribution of beta < 5%), each of which had a skewed distribution of beta values with a mean of 4,407 missing values per sample (in contrast to 1.8 missing values per sample for clusters 1 and 2 combined). Based on this analysis, we concluded that the 4 samples in cluster 3 on this analysis, we concluded that the 4 samples in cluster 3 represent samples with poor data quality. These have been excluded from all methylation analysis. miRNA-seq Data Analysis miRNA-seq unaligned bam files were downloaded from the CPTAC GDC API (https://docs.gdc.cancer.gov). Unaligned bams were first converted to fastq.gz by samtools bam2fq (samtools version 1.9). TPM (Transcripts per million) values of mature miRNA and 1.9). TPM (Transcripts per million) values of mature miRNA and precursor miRNA were reported after adaptor trimming, quality check, alignment, annotation, and read counting (https://github.com/ding-lab/CPTAC_miRNA/blob/master/cptac_mirna_analysis.md). The The mature miRNA expression was calculated irrespective of its gene of origin by summing the expression. Proteomics Data Analysis\\n\\nSpectrum quality filtering and searching All MS data were interpreted using the Spectrum Mill software package v7.0 pre-release (Agilent Technologies, Santa Clara, CA) co-developed by Karl Clauser of the Carr laboratory (https://www.broadinstitute.org/proteomics). Similar MS/MS spectra acquired Similar MS/MS spectra acquired on the same precursor m/z within +/\\u2212 45 sec were merged. MS/MS spectra were excluded from searching if they failed the quality filter by not having a sequence tag length > 0 (i.e., minimum of two masses separated by the tag length > 0 (i.e., minimum of two masses separated by the in-chain mass of an amino acid) or did not have a precursor MH+ in the range of 800\\u20136000. MS/MS spectra were searched against a RefSeq-based sequence database containing 41,457 proteins mapped sequence database containing 41,457 proteins mapped to the human reference genome (GRCh38/hg38) obtained via the UCSC Table Browser (https://genome.ucsc.edu/cgi-bin/hgTables) on June 29, 2018, with the addition of 13 proteins encoded in the human 29, 2018, with the addition of 13 proteins encoded in the human mitochondrial genome, 264 common laboratory contaminant proteins, and 553 non-canonical small open reading frames. Scoring parameters were ESI-QEXACTIVE-HCD-v2, for whole proteome datasets, were ESI-QEXACTIVE-HCD-v2, for whole proteome datasets, and ESI-QEXACTIVE-HCD-v3, for phosphoproteome, acetylproteome, and ubiquitylproteome datasets. All spectra were allowed +/\\u2212 20 ppm mass tolerance for precursor and product ions, 30% minimum matched tolerance for precursor and product ions, 30% minimum matched peak intensity, and \\u201ctrypsin allow P\\u201d enzyme specificity with up to 4 missed cleavages. Allowed fixed modifications included carbamidomethylation of cysteine and selenocysteine. TMT labeling of cysteine and selenocysteine. TMT labeling was required at lysine, but peptide N-termini were allowed to be either labeled or unlabeled. Allowed variable modifications for whole proteome datasets were acetylation of protein N-termini, oxidized datasets were acetylation of protein N-termini, oxidized methionine, deamidation of asparagine, hydroxylation of proline in PG motifs, pyroglutamic acid at peptide N-terminal glutamine, and pyro-carbamidomethylation at peptide N-terminal cysteine with a pyro-carbamidomethylation at peptide N-terminal cysteine with a precursor MH+ shift range of \\u221218 to 97 Da. For the phosphoproteome dataset the allowed variable modifications were revised to allow phosphorylation of serine, threonine, and tyrosine, allow allow phosphorylation of serine, threonine, and tyrosine, allow deamidation only in NG motifs, and disallow hydroxylation of proline with a precursor MH+ shift range of \\u221218 to 272 Da. For the acetylproteome dataset the allowed variable modifications were acetylproteome dataset the allowed variable modifications were revised to allow acetylation of lysine, allow deamidation only in NG motifs, and disallow hydroxylation of proline with a precursor MH+ shift range of \\u2212400 to 70 Da. For the ubiquitylproteome MH+ shift range of \\u2212400 to 70 Da. For the ubiquitylproteome dataset the allowed variable modifications were revised to allow diglycine modification of lysine, allow deamidation only in NG motifs, and disallow hydroxylation of proline with a precursor MH+ and disallow hydroxylation of proline with a precursor MH+ shift range of \\u2212690 to 70 Da. Protein grouping, and localization of PTMs Identities interpreted for individual spectra were automatically designated as confidently assigned using the Spectrum Mill autovalidation module to use target-decoy based false discovery rate (FDR) estimates to apply score threshold criteria. For the rate (FDR) estimates to apply score threshold criteria. For the whole proteome dataset thresholding was done in 3 steps: at the peptide spectrum match (PSM) level, the protein level for each TMT-plex, and the protein level for all 22 TMT-plexes. For the TMT-plex, and the protein level for all 22 TMT-plexes. For the phosphoproteome, acetylproteome, and ubiquitylproteome datasets thresholding was done in two steps: at the PSM and variable modification (VM) site levels. In step 1 for all datasets, PSM-level autovalidation was done first and separately for each TMT-plex experiment consisting of either 25 LC-MS/MS runs (whole proteome), 13 LC-MS/MS runs (phosphoproteome), 4 LC-MS/MS runs (acetylproteome), or 2 LC-MS/MS 4 LC-MS/MS runs (acetylproteome), or 2 LC-MS/MS runs (ubiquitylproteome) using an auto-thresholds strategy with a minimum sequence length of 7; automatic variable range precursor mass filtering; and score and delta Rank1 \\u2013 Rank2 score thresholds optimized and score and delta Rank1 \\u2013 Rank2 score thresholds optimized to yield a PSM-level FDR estimate for precursor charges 2 through 4 of <0.8% for each precursor charge state in each LC-MS/MS run. To achieve reasonable statistics for precursor charges 5\\u20136, To achieve reasonable statistics for precursor charges 5\\u20136, thresholds were optimized to yield a PSM-level FDR estimate of <0.4% across all runs per TMT-plex experiment (instead of per each run), since many fewer spectra are generated for the higher run), since many fewer spectra are generated for the higher charge states. In step 2 for the whole proteome dataset, protein-polishing autovalidation was applied separately to each TMTplex experiment to further filter the PSMs using a target protein-level FDR threshold of zero. The primary goal of this step was to eliminate of zero. The primary goal of this step was to eliminate peptides identified with low scoring PSMs that represent proteins identified by a single peptide, so-called \\u201cone-hit wonders.\\u201d After assembling protein groups from the autovalidated PSMs, protein assembling protein groups from the autovalidated PSMs, protein polishing determined the maximum protein level score of a protein group that consisted entirely of distinct peptides estimated to be false-positive identifications (PSMs with negative delta to be false-positive identifications (PSMs with negative delta forward-reverse scores). PSMs were removed from the set obtained in the initial peptide-level autovalidation step if they contributed to protein groups that had protein scores below the contributed to protein groups that had protein scores below the maximum false-positive protein score. Step 3 was then applied, consisting of protein-polishing autovalidation across all TMT plexes together using the protein grouping method \\u201cexpand TMT plexes together using the protein grouping method \\u201cexpand subgroups, top uses shared\\u201d to retain protein subgroups with either a minimum protein score of 25 or observation in at least 4 TMT plexes. The primary goal of this step was to eliminate low TMT plexes. The primary goal of this step was to eliminate low scoring proteins that were infrequently detected in the sample cohort. As a consequence of these two protein-polishing steps, each identified protein reported in the study comprised multiple identified protein reported in the study comprised multiple peptides, unless a single excellent scoring peptide was the sole match and that peptide was observed in at least 4 TMT-plexes. In calculating scores at the protein level and reporting the In calculating scores at the protein level and reporting the identified proteins, peptide redundancy was addressed in Spectrum Mill as follows: The protein score was the sum of the scores of distinct peptides. A distinct peptide was the single highest of distinct peptides. A distinct peptide was the single highest scoring instance of a peptide detected through an MS/MS spectrum. MS/MS spectra for a particular peptide may have been recorded multiple times (e.g. as different precursor charge states, in multiple times (e.g. as different precursor charge states, in adjacent bRP fractions, modified by deamidation at Asn or oxidation of Met, or with different phosphosite localization), but were still counted as a single distinct peptide. When a peptide were still counted as a single distinct peptide. When a peptide sequence of >8 residues was contained in multiple protein entries in the sequence database, the proteins were grouped together and the highest scoring one and its accession number were and the highest scoring one and its accession number were reported. In some cases when the protein sequences were grouped in this manner, there were distinct peptides that uniquely represent a lower scoring member of the group (isoforms, family members, a lower scoring member of the group (isoforms, family members, and different species). Each of these instances spawned a subgroup. Multiple subgroups were reported, counted towards the total number of proteins, and given related protein subgroup numbers number of proteins, and given related protein subgroup numbers (e.g. 3.1 and 3.2 for group 3, subgroups 1 and 2). For the whole proteome datasets the above criteria yielded false discovery rates (FDR) for each TMT-plex experiment of <0.6% at the rates (FDR) for each TMT-plex experiment of <0.6% at the peptide-spectrum match level and <0.8% at the distinct peptide level. After assembling proteins with all the PSMs from all the TMT-plex experiments together, the aggregate FDR estimates were 0.58% experiments together, the aggregate FDR estimates were 0.58% at the peptide-spectrum match level, 2.3% at the distinct peptide level, and <0.01% (1/11,029) at the protein group level. Since the protein level FDR estimate neither explicitly required a the protein level FDR estimate neither explicitly required a minimum number of distinct peptides per protein nor adjusted for the number of possible tryptic peptides per protein, it may underestimate false positive protein identifications for large underestimate false positive protein identifications for large proteins observed only on the basis of multiple low scoring PSMs. In step 2 for the phosphoproteome and acetylproteome datasets, variable modification (VM) site polishing autovalidation was applied across all 22 TMT plexes to retain all VM-site identifications with either a minimum id score of 8.0 or observation in at with either a minimum id score of 8.0 or observation in at least 4 TMT plexes. For the ubiquitylproteome datasets, it was applied across all 15 TMT plexes to retain all VM-site identifications with either a minimum id score of 8.0 or observation in at with either a minimum id score of 8.0 or observation in at least 3 TMT plexes. The intention of the VM site polishing step is to control FDR by eliminating unreliable VM site-level identifications, particularly low scoring VM sites that are only detected particularly low scoring VM sites that are only detected as low scoring peptides that are also infrequently detected across all of the TMT plexes in the study. In calculating scores at the VM-site level and reporting the identified VM sites, redundancy VM-site level and reporting the identified VM sites, redundancy was addressed in Spectrum Mill as follows: A VM-site table was assembled with columns for individual TMT-plex experiments and rows for individual VM-sites. PSMs were combined into a single rows for individual VM-sites. PSMs were combined into a single row for all non-conflicting observations of a particular VM-site (e.g. different missed cleavage forms, different precursor charges, confident and ambiguous localizations, and different charges, confident and ambiguous localizations, and different sample-handling modifications). For related peptides, neither observations with a different number of VM-sites nor different confident localizations were allowed to be combined. Selecting the localizations were allowed to be combined. Selecting the representative peptide from the combined observations was done such that once confident VM-site localization was established, higher identification scores and longer peptide lengths were preferred. scores and longer peptide lengths were preferred. While a Spectrum Mill identification score was based on the number of matching peaks, their ion type assignment, and the relative height of unmatched peaks, the VM site localization score was the of unmatched peaks, the VM site localization score was the difference in identification score between the top two localizations. The score threshold for confident localization, >1.1, essentially corresponded to at least 1 b or y ion located between two corresponded to at least 1 b or y ion located between two candidate sites that has a peak height >10% of the tallest fragment ion (neutral losses of phosphate from the precursor and related ions as well as immonium and TMT reporter ions were excluded from as well as immonium and TMT reporter ions were excluded from the relative height calculation). The ion type scores for b-H3PO4, y-H3PO4, b-H2O, and y-H2O ion types were all set to 0.5. This prevented inappropriate confident localization assignment when a inappropriate confident localization assignment when a spectrum lacked primary b or y ions between two possible sites but contained ions that could be assigned as either phosphate-loss ions for one localization or water loss ions for another localization. one localization or water loss ions for another localization. VM-site polishing yielded 68, 674 phosphosites with an aggregate FDR of 0.73% at the phosphosite level. In aggregate, 71% of the reported phosphosites in this study were fully localized to a reported phosphosites in this study were fully localized to a particular serine, threonine, or tyrosine residue. VM-site polishing yielded 15,186 acetylsites with an aggregate FDR of 0.80% at the acetylsite level. In aggregate, 99% of the reported at the acetylsite level. In aggregate, 99% of the reported acetylsites in this study were fully localized to a particular lysine residue. VM-site polishing yielded 25,489 ubiquitylsites with an aggregate FDR of 0.40% at the ubiquitylsite level. In with an aggregate FDR of 0.40% at the ubiquitylsite level. In aggregate, >99% of the reported ubiquitylsites in this study were fully localized to a particular lysine residue. Quantitation using TMT ratios Using the Spectrum Mill Protein/Peptide Summary module, a protein comparison report was generated for the proteome dataset using the protein grouping method \\u201cexpand subgroups, top uses shared\\u201d (SGT). For the phosphoproteome and acetylproteome datasets a (SGT). For the phosphoproteome and acetylproteome datasets a Variable Modification site comparison report limited to either phospho or acetyl sites, respectively, was generated using the protein grouping method \\u201cunexpand subgroups\\u201d. Relative abundances of grouping method \\u201cunexpand subgroups\\u201d. Relative abundances of proteins and VM-sites were determined in Spectrum Mill using TMT reporter ion intensity ratios from each PSM. TMT reporter ion intensities were corrected for isotopic impurities in the Spectrum were corrected for isotopic impurities in the Spectrum Mill Protein/Peptide summary module using the afRICA correction method, which implements determinant calculations according to Cramer\\u2019s Rule and correction factors obtained from the reagent Cramer\\u2019s Rule and correction factors obtained from the reagent manufacturer\\u2019s certificate of analysis (https://www.thermofisher.com/order/catalog/product/90406) for TMT11_lot number UA280170/TL272399 (Proteome, Phosphoproteme, and Acetylproteome) and (Proteome, Phosphoproteme, and Acetylproteome) and TE270748-TD264064 (Ubiquitylproteome). A protein-level, phosphosite-level, or acetylsite-level TMT ratio is calculated as the median of all PSM-level ratios contributing to a protein subgroup, of all PSM-level ratios contributing to a protein subgroup, phosphosite, or acetylsite. PSMs were excluded from the calculation if they lacked a TMT label, had a precursor ion purity < 50% (MS/MS has significant precursor isolation contamination from (MS/MS has significant precursor isolation contamination from co-eluting peptides), or had a negative delta forward-reverse identification score (half of all false-positive identifications). Lack of TMT label led to exclusion of PSMs per TMT plex with a Lack of TMT label led to exclusion of PSMs per TMT plex with a range of 1.4 to 2.3% for the proteome, 1.8 to 3.7% for the phosphoproteome, 1.0 to 3.5% for the acetylproteome, and 4.5 to 18.0% for the ubiquitylproteome datasets. Low precursor ion purity for the ubiquitylproteome datasets. Low precursor ion purity led to exclusion of PSMs per TMT plex with a range of 1.1 to 1.4% for the proteome, 1.7 to 2.8% for the phosphoproteome, 0.5 to 1.7% for the acetylproteome, and 2.8 to 3.5% for the 0.5 to 1.7% for the acetylproteome, and 2.8 to 3.5% for the ubiquitylproteome datasets. Two-component normalization of TMT ratios It was assumed that for every sample there would be a set of unregulated proteins, phosphosites, acetylsites or ubiquitylsites that have abundance comparable to the common reference (CR) sample. In the normalized sample, these proteins, phosphosites, sample. In the normalized sample, these proteins, phosphosites, acetylsites or ubiquitylsites should have a log TMT ratio centered at zero. In addition, there were proteins, phosphosites, acetylsites and ubiquitylsites that were either up- or acetylsites and ubiquitylsites that were either up- or down-regulated compared to the CR. A normalization scheme was employed that attempted to identify the unregulated proteins, phosphosites, acetylsites or ubiquitylsites, and centered the distribution acetylsites or ubiquitylsites, and centered the distribution of these log-ratios around zero in order to nullify the effect of differential protein loading and/or systematic MS variation. A 2-component Gaussian mixture model-based normalization algorithm Gaussian mixture model-based normalization algorithm was used to achieve this effect. The two Gaussians N(\\u03bci1,\\u03c3i1) and N(\\u03bci2,\\u03c3i2) for a sample i were fitted and used in the normalization process as follows: the mode mi of the log-ratio distribution was as follows: the mode mi of the log-ratio distribution was determined for each sample using kernel density estimation with a Gaussian kernel and Shafer-Jones bandwidth. A two-component Gaussian mixture model was then fit with the mean of both Gaussians mixture model was then fit with the mean of both Gaussians constrained to be mi, i.e., \\u03bci1=\\u03bci2=mi. The Gaussian with the smaller estimated standard deviation  was assumed to represent the unregulated component of was assumed to represent the unregulated component of proteins/phosphosites/acetylsites/ubiquitylsites, and was used to normalize the sample. The sample was standardized using (mi,\\u03c3i) by subtracting the mean mi from each standardized using (mi,\\u03c3i) by subtracting the mean mi from each protein/phosphosite/acetylsite/ubiquitylsite and dividing by the standard deviation \\u03c3i. Normalization of acetyl- and ubiquitylproteome To correct for differential underlying protein levels when performing certain acetylproteome- and ubiquitylproteome-based analyses, a global linear model approach was employed to produce protein-normalized versions of both of these PTM datasets. PTM sites versions of both of these PTM datasets. PTM sites were first matched to underlying proteins in the proteome dataset using RefSeq accession numbers. If a site\\u2019s main accession number did not yield a match in the proteome, all accession numbers associated yield a match in the proteome, all accession numbers associated with the site were then tried and the protein with the highest score was selected as a match. For the acetylproteome and ubiquitylproteome separately, a linear model was then fit (R function separately, a linear model was then fit (R function lm, PTM ~ protein) to all matched PTM-protein data points in the dataset. The residuals of each model were then used as protein-normalized acetylsite or ubiquitylsite abundances. Proteogenomic analysis\\n\\nQuality control and Batch effect assessment We used guided principal component analysis (gPCA) to assess the presence of a batch effect in the sample/CR ratio data based on TMT-plex. There was no overall apparent batch effect (gPCA p-value=1) in the global proteome, phosphoproteome, acetylproteome in the global proteome, phosphoproteome, acetylproteome or ubiquitylproteome datasets. To assess composition similarities between CR used for proteome, phosphoproteome and acetylproteome analysis, and ubiquitylproteome, we performed label-free proteomic analysis of each CRs in triplicates. Both CRs showed excellent correlation with median triplicates. Both CRs showed excellent correlation with median pearson correlation > 0.85. In addition, to assess data quality, we assessed the Pearson correlations of log TMT reporter ion intensities between all pairs of TMT plexes, separately for sample and CR channels. For each given pair of TMT plexes, correlations between CRs were each given pair of TMT plexes, correlations between CRs were consistently higher than correlations of respective sample channels enabling precise relative quantitation of proteins, phosphorylation and acetylation sites across 22 TMT plexes using the CR and acetylation sites across 22 TMT plexes using the CR channels as common denominator (Figure S1C). In the ubiquitylproteome data we observed lower correlations of the CR channel (in comparison to samples channels) in three of the 18 TMT plexes analyzed to samples channels) in three of the 18 TMT plexes analyzed (plexes 7, 8, 10). Clustering of the CR channels using pairwise correlations as a similarity metric pointed to a batch effect in the CR channels (but not in the sample channels) in these plexes. CR channels (but not in the sample channels) in these plexes. The presence of a batch effect was further confirmed by applying the gPCA test described above to only the reporter intensities of the CR channels (gPCA p-value < 0.001). Even though of the CR channels (gPCA p-value < 0.001). Even though undetectable with a global test, we observed this CR batch effect propagate to the sample/CR ratio data in preliminary analyses (data not shown). To counter the propagation of this CR batch effect to shown). To counter the propagation of this CR batch effect to the sample/CR ratio for samples in these plexes, batch correction was applied to the sample/CR ratio data for plexes 7, 8, 10 (batch 1) and all other plexes (batch 2) using limma\\u2019s 7, 8, 10 (batch 1) and all other plexes (batch 2) using limma\\u2019s removeBatchEffect (https://rdrr.io/bioc/limma/man/removeBatchEffect.html) on the sample/CR ratio data including the sample type (tumor, NAT) as a covariate. Dataset filtering Genes (RNA-seq), proteins (global proteome), phosphosites, acetylsites and ubiquitin sites present in fewer than 30% of samples (i.e., missing in >70% of samples) were removed from the respective datasets. Furthermore, proteins were required to have at datasets. Furthermore, proteins were required to have at least two observed TMT ratios in >25% of samples in order to be included in the proteome dataset. Phosphosites, acetylsites and ubiquitin sites were required to have at least one observed TMT ratio sites were required to have at least one observed TMT ratio in >25% of samples. Some of the filtering steps were modified for specific analyses in the study. For many of the marker selection and gene set enrichment analyses, at least 50% of samples were required to have non-missing values for proteins/phosphosites/acetyl/ubiquitin non-missing values for proteins/phosphosites/acetyl/ubiquitin sites, since missing values were imputed, and excessive missing values can result in poor imputation. Alternate filtering has been noted in descriptions of the relevant methods. Unsupervised multi-omic clustering using NMF We used non-negative matrix factorization (NMF)-based multi-omic clustering using protein, phosphosite, acetylsite, RNA transcript and gene copy number variants (CNV) as previously described (. Briefly, given a factorization rank k (where k is the number (. Briefly, given a factorization rank k (where k is the number of clusters), NMF decomposes a p x n data matrix V into two matrices W and H such that multiplication of W and H approximates V. Matrix H is a k x n matrix whose entries represent weights for Matrix H is a k x n matrix whose entries represent weights for each sample (1 to N) to contribute to each cluster (1 to k), whereas matrix W is a p x k matrix representing weights for each feature (1 to p) to contribute to each cluster (1 to k). Matrix H (1 to p) to contribute to each cluster (1 to k). Matrix H was used to assign samples to clusters by choosing the k with maximum score in each column of H. For each sample we calculated a cluster membership score as the maximal fractional score of the cluster membership score as the maximal fractional score of the corresponding column in matrix H. We defined a \\u201dcluster core\\u201d as the set of samples with cluster membership score > 0.5. To enable integrative multi-omic clustering we enforced all data types (and converted if necessary) to represent ratios to: (i) a common reference measured in each TMT plex (proteome, phosphoproteome, acetylproteome), (ii) an in silico common reference acetylproteome), (ii) an in silico common reference calculated as the median abundance across all samples (RNA expression) or (iii) to matching blood normal for CNA data. The CNA data was further filtered to only retain genes significantly altered was further filtered to only retain genes significantly altered (GISTIC2 thresholded of +2 or \\u22122) in at least 5% of all tumors. All data tables were then concatenated and all rows with missing values were removed. To remove uninformative features from the values were removed. To remove uninformative features from the dataset prior to NMF clustering we removed features with the lowest standard deviation (bottom 5th percentile) across all samples. Each column in the data matrix was further scaled and samples. Each column in the data matrix was further scaled and standardized such that all features from different data types were represented as z-scores. The resulting data matrix of z-scores into was converted to a non-negative input matrix required by into was converted to a non-negative input matrix required by NMF as follows: Create one data matrix with all negative numbers zeroed.\\n\\nCreate another data matrix with all positive numbers zeroed and the signs of all negative numbers removed. Concatenate both matrices resulting in a data matrix twice as large as the original, but containing only positive values and zeros and hence appropriate for NMF. The resulting matrix was then subjected to NMF analysis leveraging the NMF R-package and using the factorization method described in. To determine the optimal factorization rank k (number of clusters) for the multi-omic data matrix we tested a range of clusters) for the multi-omic data matrix we tested a range of clusters between k=2 and 8. For each k we factorized matrix V using 50 iterations with random initializations of W and H. To determine the optimal factorization rank we calculated two metrics the optimal factorization rank we calculated two metrics for each k: 1) cophenetic correlation coefficient measuring how well the intrinsic structure of the data was recapitulated after clustering and 2) the dispersion coefficient of the consensus matrix and 2) the dispersion coefficient of the consensus matrix as defined in measuring the reproducibility of the clustering across 50 iterations. The optimal k was defined as the maximum of the product of both metrics for cluster numbers between k=3 and 8 Having determined the optimal factorization rank k, in order to achieve robust factorization of the multi-omic data matrix V, we repeated the NMF analysis using 500 iterations with random initializations of W and H and performed the partitioning of initializations of W and H and performed the partitioning of samples into clusters as described above. Due to the non-negative transformation applied to the z-scored data matrix as described above, matrix W of feature weights contained two separate above, matrix W of feature weights contained two separate weights for positive and negative z-scores of each feature, respectively. In order to revert the non-negative transformation and to derive a single signed weight for each feature, we first and to derive a single signed weight for each feature, we first normalized each row in matrix W by dividing by the sum of feature weights in each row, aggregated both weights per feature and cluster by keeping the maximal normalized weight and and cluster by keeping the maximal normalized weight and multiplication with the sign of the z-score in the initial data matrix. Thus, the resulting transformed version of matrix Wsigned contained signed cluster weights for each feature in the input contained signed cluster weights for each feature in the input matrix. For functional characterization of clustering results by single sample Gene Set Enrichment Analysis (ssGSEA), we calculated normalized enrichment scores (NES) of cancer-relevant gene sets by projecting the matrix of signed multi-omic feature weights by projecting the matrix of signed multi-omic feature weights (Wsigned) onto Hallmark pathway gene sets using ssGSEA. To derive a single weight for each gene measured across multiple omics data types (protein, RNA, phosphorylation site, acetylation site, types (protein, RNA, phosphorylation site, acetylation site, CNA) we retained the weight with maximal absolute amplitude. We used the ssGSEA implementation available on https://github.com/broadinstitute/ssGSEA2.0 using the following parameters: gene.set.database=\\u201ch.all.v6.2.symbols.gmt\\u201d\\n\\nsample.norm.type=\\u201crank\\u201d\\n\\nweight=1\\n\\nstatistic=\\u201darea.under.RES\\u201d\\n\\noutput.score.type=\\u201cNES\\u201d\\n\\nnperm=1000\\n\\nglobal.fdr=TRUE\\n\\nmin.overlap=5\\n\\ncorrel.type=\\u201dz.score\\u201d correl.type=\\u201dz.score\\u201d\\n\\nTo test the association of the resulting clusters to clinical variables we used Fisher\\u2019s exact test (R function fisher.test) to test for overrepresentation in the set of samples defining the cluster core as described above. The entire workflow described above has been implemented as a module for PANOPLY which runs on Broad\\u2019s Cloud platform Terra (https://app.terra.bio/). The docker containers encapsulating the source code and required R-packages for NMF clustering and ssGSEA code and required R-packages for NMF clustering and ssGSEA have been submitted to Dockerhub (broadcptacdev/pgdac_mo_nmf:15, broadcptac/pgdac_ssgsea:5). The source code for ssGSEA is available on GitHub: https://github.com/broadinstitute/ssGSEA2.0. Integrative analysis with Stewart et al. To enable integrative analysis of our protein data with the protein data in Stewart et al. we first aggregated the protein-level data to gene-level data matrices by retaining the dominant isoform (identified by the lowest protein subgroup number) isoform (identified by the lowest protein subgroup number) associated with each gene symbol. To aggregate the Stewart et al. protein data we used the mean expression across all proteins mapping to the same gene symbol. Both gene-level protein data mapping to the same gene symbol. Both gene-level protein data matrices were then separately subjected to gene-level z-score transformation before joining the matrices using the unique gene symbols as keys. The resulting integrated protein data matrix was as keys. The resulting integrated protein data matrix was then subjected to the NMF-based clustering pipeline described above using the 3,366 genes quantified across all 216 tumors. RNA subtyping Previously reported LSCC gene expression-subtype signatures were downloaded from the original publication and applied to our RNA expression dataset using the centroid-based method in a manner similar to the TCGA study. Specifically, the RNA expression of similar to the TCGA study. Specifically, the RNA expression of the 208 signature genes, represented as FPKM, was mean-centered in the gene-wise manner, and each sample was correlated to each of the four centroid vectors representing the average signature of the four centroid vectors representing the average signature gene expression for the four subtypes. The samples were assigned to one of the four transcriptomics subtypes according to the highest correlations. Samples with insignificance correlation (p correlations. Samples with insignificance correlation (p > 0.01) were marked to have \\u2018undecided\\u2019 subtype. Chromosomal instability index The CIN score reflects the overall copy number aberration across the whole genome. From the segment-level CNA result, we used a weighted-sum approach to summarize the chromosome instability for each sample. The absolute segment-level log2 ratios of all for each sample. The absolute segment-level log2 ratios of all copy number segments (indicating the copy number aberration of these segments) within a chromosome were weighted by the segment length and summed up to derive the instability score for the length and summed up to derive the instability score for the chromosome. The genome-wide chromosome instability index was calculated by summing up the instability score of all 22 autosomes. Fusion detection and analysis Fusion calling was performed using a combination of CRISP, CODAC MI-ONCOSEQ pipeline, fusioncatcher_v1.10 and arriba_v1.1.0 (https://github.com/suhrig/arriba/). The fusions calls were compiled and visualized using AGFusion to display the predicted domain and visualized using AGFusion to display the predicted domain and exon structure of the fusion proteins as described. mRNA and Protein correlation To compare mRNA expression and protein abundance across samples we focused on the RNAseq data with 21,792 genes, and global proteome with 11,575 quantified proteins. Protein IDs were mapped to gene names. In total, 10,890 genes common to both RNAseq and to gene names. In total, 10,890 genes common to both RNAseq and proteome data spanning 108 tumor samples were used in the analysis. The analyses were carried out on normalized data: RNAseq data were log2 transformed, upper quartile normalized RPKM values, were log2 transformed, upper quartile normalized RPKM values, which were median-centered by row (i.e., gene); proteome data was two-component normalized as described earlier. Pearson correlation was calculated for each gene. Both correlation coefficient was calculated for each gene. Both correlation coefficient and p-value were recorded. Further, adjusted p-values were calculated using the Benjamini\\u2013Hochberg procedure. CNA-driven cis and trans effects Correlations between copy number alterations (CNA) and RNA, proteome, phosphoproteome acetylproteome and ubiquitylproteome (with proteome and PTM data mapped to genes, by choosing the most variable protein isoform/PTM site as the gene-level the most variable protein isoform/PTM site as the gene-level representative) were determined using Pearson correlation of common genes present in CNA-RNA-proteome (9,988 genes), CNA-RNA-phosphoproteome (5,144 genes), CNA-RNA-acetylproteome (1,344 genes) (5,144 genes), CNA-RNA-acetylproteome (1,344 genes) and CNA-RNA-ubiquitylproteome (2,616 genes). In addition, p-values (corrected for multiple testing using Benjamini-Hochberg FDR) for assessing the statistical significance of the correlation values were the statistical significance of the correlation values were also calculated. CNA trans-effects for a given gene were determined by identifying genes with statistically significant (FDR < 0.05) positive or negative correlations. CMAP analysis Candidate genes driving response to copy number alterations were identified using large-scale Connectivity Map (CMAP) queries. The CMAP is a collection of about 1.3 million gene expression profiles from cell lines treated with bioactive small molecules profiles from cell lines treated with bioactive small molecules (~20,000 drug perturbagens), shRNA gene knockdowns (~4,300) and ectopic expression of genes. The CMAP dataset is available on GEO (Series GSE92742). For this analysis, we use the Level 5 on GEO (Series GSE92742). For this analysis, we use the Level 5 (signatures from aggregating replicates) TouchStone dataset with 473,647 total profiles, containing 36,720 gene knock-down profiles, with measurements for 12,328 genes. See knock-down profiles, with measurements for 12,328 genes. See https://clue.io/GEO-guide for more information. To identify candidate driver genes, proteome profiles of copy number-altered samples were correlated with gene knockdown mRNA profiles in the above CMAP dataset, and enrichment of up/downregulated genes was evaluated. Normalized log2 copy num- ber values genes was evaluated. Normalized log2 copy num- ber values less than 0.3 defined deletion (loss), and values greater than +0.3 defined copy number amplifications (gains). In the copy number-altered samples (separately for CNA amplification and CNA samples (separately for CNA amplification and CNA deletion), the trans-genes (identified by significant correlation in \\u201cCNA driven cis and trans effects\\u201d above) were grouped into UP and DOWN categories by comparing the protein ratios of these genes to categories by comparing the protein ratios of these genes to their ratios in the copy number neutral samples (normalized log2 copy number between 0.3 and +0.3). The lists of UP and DOWN trans-genes were then used as queries to interrogate CMAP signatures were then used as queries to interrogate CMAP signatures and calculate weighted connectivity scores (WTCS) using the single-sample GSEA algorithm. The weighted connectivity scores were then normalized for each perturbation type and cell line to obtain normalized for each perturbation type and cell line to obtain normalized connectivity scores (NCS). See for details on WTCS and NCS. For each query we then identified outlier NCS scores, where a score was considered an outlier if it lay beyond 1.5 times a score was considered an outlier if it lay beyond 1.5 times the interquartile range of score distribution for the query. The query gene was designated a candidate driver if (i) the score outliers were statistically cis-enriched (Fisher test with BH-FDR were statistically cis-enriched (Fisher test with BH-FDR multiple testing correction) and (ii) the gene had statistically significant and positive cis-correlation. For a gene to be considered for inclusion in a CMAP query it needed to i) have a copy number change (amplification or deletion) in at least 15 samples; ii) have at least 20 significant trans genes; and iii) be on the list of shRNA knockdowns in the CMAP. genes; and iii) be on the list of shRNA knockdowns in the CMAP. 502 genes satisfied these conditions and resulted in 976 queries (CNA amplification and deletion combined) that were tested for enrichment. Thirty two (32) candidate driver genes were for enrichment. Thirty two (32) candidate driver genes were identified with Fisher\\u2019s test FDR < 0.25, using this process. In order to ensure that the identified candidate driver genes were not a random occurrence, we performed a permutation test to determine how many candidate driver genes would be identified with random input (Mertins et al., 2016). For the 976 queries with random input (Mertins et al., 2016). For the 976 queries used, we substituted the bona-fide trans-genes with randomly chosen genes, and repeated the CMAP enrichment process. To determine FDR, each permutation run was treated as a Poisson sample with FDR, each permutation run was treated as a Poisson sample with rate \\u03bb, counting the number of identified candidate driver genes. Given the small n ( = 10) and \\u03bb, a Score confidence interval was calculated (Barker, 2002) and the midpoint of the confidence calculated (Barker, 2002) and the midpoint of the confidence interval used to estimate the expected number of false positives. Using 10 random permutations, we determined the overall false discovery rate to be FDR = 0.33, with a 95% CI of (0.27, 0.40) for rate to be FDR = 0.33, with a 95% CI of (0.27, 0.40) for genes with either CNA amplification or deletion, and an FDR=0.06 (95% CI: 0.01, 0.12) for six (6) genes with both CNA amplification and deletion. To identify how many trans-correlated genes for all candidate regulatory genes could be directly explained by gene expression changes measured in the CMAP shRNA perturbation experiments, knockdown gene expression consensus signature z-scores knockdown gene expression consensus signature z-scores (knockdown/control) were used to identify regulated genes with \\u03b1 = 0.05, followed by counting the number of trans-genes in this list of regulated genes. To obtain biological insight into the list of candidate driver genes, we performed (i) enrichment analysis on samples with extreme CNA values (amplification or deletion) to identify statistically enriched sample annotation subgroups; and (ii) GSEA on enriched sample annotation subgroups; and (ii) GSEA on cis/trans-correlation values to find enriched pathways. LINCS analysis Differentially expressed genes between the 5 NMF LSCC subtypes were identified using the limma package. In order to identify genes that were uniquely expressed in each subtype, all pairwise comparisons between the 5 subtypes were computed. In addition, comparisons between the 5 subtypes were computed. In addition, genes that were differentially expressed (adjusted p.value < 0.05) in at least 3 out of the 4 comparisons and had a concordant fold change among all comparisons were identified as concordant fold change among all comparisons were identified as subtype-specific and were used in the subsequent analysis. Protein abundance comparisons were performed between all 5 NMF subtypes using the Wilcoxon rank-sum test and p-values were adjusted using the Benjamini & Hochberg method. Subtype-specific differentially expressed proteins were identified based on their expressed proteins were identified based on their differential expression (adjusted p.value < 0.05) in at least 2 out of the 4 comparisons, and by having a concordant fold change among all comparisons. The identified differentially expressed genes and comparisons. The identified differentially expressed genes and proteins were then filtered for gene symbols measured in the L1000 assay (978 landmark genes). These NMF-specific signatures were used as input to calculate normalized weighted connectivity used as input to calculate normalized weighted connectivity scores (WTCS) against the Library of Integrated Network-Based Cellular Signatures (LINCS) L1000 perturbation-response signatures. The scores were computed using the sig_queryl1k_tool pipeline The scores were computed using the sig_queryl1k_tool pipeline (https://hub.docker.com/u/cmap) and the LINCS L1000 Level 5 compound (trt_cp) signatures from CLUE (clue.io, \\u201cExpanded CMap LINCS Resource 2020 Release\\u201d). The resulting normalized connectivity Resource 2020 Release\\u201d). The resulting normalized connectivity scores were summarized across cell lines using the maximum quantile of all the scores of the same compound (Qhi = 67,Qlow = 33) as previously described. The corresponding compound metadata as previously described. The corresponding compound metadata were obtained from CLUE (clue.io, \\u201cExpanded CMap LINCS Resource 2020 Release\\u201d) and were used to filter and identify compounds with existing annotations (drug name, MOA, Target). The top 20 with existing annotations (drug name, MOA, Target). The top 20 negatively connected compounds to each NMF signature were selected for visualization. Defining cancer-associated genes\\n\\nCancer-associated genes (CAG) were compiled using the Census website https://cancer.sanger.ac.uk/census. The list of genes is provided in Table S1.\\n\\nCpG Island Methylator Phenotype The methylation analysis started with all 103 tumor samples, which excluded 4 samples from the cohort with high missing rate (see above), to generate the CpG island methylator phenotypes (CIMP). Specifically, we first generated the gene-level methylation Specifically, we first generated the gene-level methylation score, by taking the averaged beta values of all probes harboring in the islands of promotor or 5 UTR regions of the gene. Then, we preprocessed the data by filtering genes that were Then, we preprocessed the data by filtering genes that were hypermethylated, i.e. the gene-level methylation score >0.2, transformed the score into M-values, normalized the transformed values, and then imputed the missing values with k-nearest neighbors and then imputed the missing values with k-nearest neighbors (KNN). With preprocessed data, we performed consensus clustering 1000 times, each taking 80% of the samples and all genes, and calculated the consensus matrix (probabilities of two samples calculated the consensus matrix (probabilities of two samples clustering together) for each predetermined number of clusters K. K ranges from 2 to 10. In each value of K, we visualized the consensus matrix using K means with Euclidean distance as the consensus matrix using K means with Euclidean distance as the distance metric. Finally, we determined the optimal number of clusters by considering the relative change in area under the consensus cumulative density function (CDF) curve. In the end, three cumulative density function (CDF) curve. In the end, three distinct clusters were identified, one was hypermethylated with mean M value 0.393, and two were hypomethylated with mean M value \\u22120.041 and \\u22120.359, respectively. We labeled these three clusters and \\u22120.359, respectively. We labeled these three clusters as CIMP high, CIMP intermediate, and CIMP low groups. iProFun Based Cis Association Analysis We used iProFun, an integrative analysis tool to identify multi-omic molecular quantitative traits (QT) perturbed by DNA-level variations. We considered five functional molecular quantitative traits (gene expression, protein, phosphoprotein, quantitative traits (gene expression, protein, phosphoprotein, acetylprotein, ubiquitylproteome levels) for their associations with DNA methylation, accounting for mutation, copy number variation, age, gender, tumor purity and smoking status. Tumor purity age, gender, tumor purity and smoking status. Tumor purity was determined using TSNet from RNA-seq data. The iProFun procedure was applied to a total of 12,666 genes that was available in at least one predictor (mutation, CNA, DNA methylation) and one at least one predictor (mutation, CNA, DNA methylation) and one outcome (mRNA, global protein, phospholation, acetylation and ubiquitylation) for their cis regulatory patterns in tumors. For each of the 12,666 genes, we started with linear regression to each of the 12,666 genes, we started with linear regression to capture the covariate-adjusted associations. The resulting association summary statistics was used to call probabilities of belonging to each of the 2^5=32 possible configurations. We of belonging to each of the 2^5=32 possible configurations. We calculated the probabilities of associating with each of the five outcomes by combining probabilities from relevant configurations, and calculated FDR controlled associations. The significant and calculated FDR controlled associations. The significant genes need to pass three criteria: (1) the satisfaction of biological filtering procedure, (2) posterior probabilities > 75%, and (3) empirical false discovery rate (eFDR)<10%. As DNA methylation empirical false discovery rate (eFDR)<10%. As DNA methylation at the promoter regions of a gene down regulates its expression level, biological filtering procedure requires DNA methylations that exist either negative significant associations or that exist either negative significant associations or insignificant associations with all 5 the types of molecular QTs for significance call. DNA methylations with significant positive associations with QTs were filtered out for significance call, as it with QTs were filtered out for significance call, as it was not the pattern of biological associations that were interesting. Secondly, a significance was called only if the posterior probabilities > 75% of a predictor being associated with a molecular > 75% of a predictor being associated with a molecular QT, by summing over all configurations that are consistent with the association of interest. Lastly, we calculated empirical FDR via 100 permutations per molecular QTs by shuffling the label of the permutations per molecular QTs by shuffling the label of the molecular QTs, and requested empirical FDR (eFDR) <10% by selecting a minimal cutoff value of alpha such that 75%<alpha<100%. The eFDR is calculated by: eFDR= (Averaged No. of genes with The eFDR is calculated by: eFDR= (Averaged No. of genes with posterior probabilities > alpha in permuted data) / (Averaged No. of genes with posterior probabilities > alpha in original data). Figure 1G annotated those genes whose DNA methylation had Figure 1G annotated those genes whose DNA methylation had cascading effects (associated with all QTs under investigation), and among them those whose protein abundances significantly differ between tumor and NAT were annotated. EMT-specific cluster and fibroblast enrichment NMF derived EMT-enriched subtype was interrogated for EMT and fibroblast related proteins in LSCC tumor patients. The relative protein expression was calculated using z-score. The single ssGSEA was performed on a protein dataset using GSVA (v3.11) to ssGSEA was performed on a protein dataset using GSVA (v3.11) to calculate normalized enrichment score for EMT and fibroblast proliferation using MSigdb (v6.1) and gene ontology geneset respectively. Similar analysis was also performed to interrogate SOX2 Similar analysis was also performed to interrogate SOX2 associated genes and its relation with IL6_JAK_STAT signaling (MSigdb hallmarks geneset V6.1). Differential marker analysis A Wilcoxon signed rank test was performed on TMT-based global proteomic data between tumor and matched normal samples to determine differential abundance of proteins between tumor and NAT samples. Proteins having < 50% missing values were considered for Proteins having < 50% missing values were considered for downstream analysis. Proteins with log2-fold-change (FC)> 2 in tumors and Benjamini-Hochberg FDR < 0.01 were considered to be tumor-associated proteins. Over-representation analysis of Gene Ontology proteins. Over-representation analysis of Gene Ontology Biological Process terms was performed on a background of quantified proteins in WebGestaltR. The selected gene list was either significant proteins or proteins containing a significantly altered proteins or proteins containing a significantly altered phosphorylation site from the differential marker analyses. ORA was performed separately for increased and decreased proteins. Terms were considered significant with a Benjamini-Hochberg adjusted were considered significant with a Benjamini-Hochberg adjusted p-value <0.01. Among tumor associated proteins, the top 50 differential proteins with more than 90% of samples showing high expression in tumors were selected and shown. Differential proteins with more than 99% of samples showing tumor specific expression were more than 99% of samples showing tumor specific expression were highlighted. Immunohistochemistry-based antibody-specific staining scores in lung tumors were obtained from the Human Protein Atlas (HPA, https://www.proteinatlas.org), in which Protein Atlas (HPA, https://www.proteinatlas.org), in which tumor-specific staining is reported in four levels, i.e., high, medium, low, and not detected. The protein-specific annotations such as enzymes, transcription factor, transporters, secreted, such as enzymes, transcription factor, transporters, secreted, transmembrane and FDA-approved drugs targeting the protein or reported in drugbank were designated. Similarly, differential analysis was performed between RNA, phosphosite and acetylproteome was performed between RNA, phosphosite and acetylproteome datasets between tumor and matched NATs. The differential genes/sites with log2-fold-change (FC)> 2 in tumors and Benjamini-Hochberg FDR < 0.01 were considered to be tumor-associated. The FDR < 0.01 were considered to be tumor-associated. The overlapping genes/sites with proteins were represented in a checkerbox plot. Overall and disease-free survival in TCGA lung squamous carcinoma Survival for lung squamous tumor patients in TCGA LSCC cohort was calculated using the time from date of diagnosis to death or last contact (Overall survival, OS) and disease-free interval time (Disease-free survival, DFS). Normalized RNA expression was (Disease-free survival, DFS). Normalized RNA expression was categorized into low and high expression group based on mean. Univariate Cox proportional hazards models were fitted to OS, and DFI endpoints to calculate the hazard ratios for tumor-associated endpoints to calculate the hazard ratios for tumor-associated genes/proteins using coxph function in Survival package (version 2.44) in R. The p-values were determined using a log-rank test. The hazard ratio [exp(cox coefficient)] was used to compare poor hazard ratio [exp(cox coefficient)] was used to compare poor survival in low and high-expression group. Kaplan-Meier survival analysis We used time-to-death or last contact data for the current cohort (recorded as number of days from diagnosis date) to assess survival differences based on various sample subgroups and clinical annotations. We used Kaplan-Meier analysis to explore survival annotations. We used Kaplan-Meier analysis to explore survival differences associated with tumor grade, mutation burden, mutation status for SMGs, CUL3-NFE2L2-KEAP1 combined mutation status, CNA for KAT6A/SOX2/TP63/FGFR1/CDKN2A, immune subtype, NMF CNA for KAT6A/SOX2/TP63/FGFR1/CDKN2A, immune subtype, NMF cluster, CIN, tumor grade and ploidy. We identified NMF mixed tumors (i.e., tumors that are not part of the core NMF clusters) had significantly worse survival (log-rank p-value = 0.005). We used a Fisher test to find enriched annotations (among those mentioned in the previous paragraph, with (among those mentioned in the previous paragraph, with tumor stemness index added) associated with NMF mixed tumors. The frequency of SOX2 amplifications were significantly enriched in NMF mixed tumors (p-value = 0.0038). None of the other annotations had tumors (p-value = 0.0038). None of the other annotations had significant enrichment in the NMF mixed tumors. Continuous Smoking Score Non-negative matrix factorization (NMF) was used in deciphering mutation signatures in cancer somatic mutations stratified by 96 base substitutions in tri-nucleotide sequence contexts. To obtain a reliable signature profile, we used SomaticWrapper to call a reliable signature profile, we used SomaticWrapper to call mutations from WGS data. SignatureAnalyzer exploited the Bayesian variant of the NMF algorithm and enabled an inference for the optimal number of signatures from the data itself at a balance optimal number of signatures from the data itself at a balance between the data fidelity (likelihood) and the model complexity (regularization). After decomposing into three signatures, signatures were compared against known signatures derived from COSMIC were compared against known signatures derived from COSMIC and cosine similarity was calculated to identify the best match. We also sought to integrate count of total mutations, t, percentage that are signature mutations, c, and count of DNPs, n, into a continuous score, 0 < S < 1, to quantify the degree of confidence that a sample was associated with smoking signature. We that a sample was associated with smoking signature. We referred to these quantities as the data, namely D = C \\u2229 T \\u2229 N, and used A and A\\u2019 to indicate smoking signature or lack thereof, respectively. In a Bayesian framework, it is readily shown that a In a Bayesian framework, it is readily shown that a suitable form is S = 1 / (1 + R), where R is the ratio of the joint probability of A\\u2019 and D to the joint probability of A and D. For example, the latter can be written P(A)\\u30fbP(C|A)\\u30fbP(T|A)\\u30fbP(N|A) and the the latter can be written P(A)\\u30fbP(C|A)\\u30fbP(T|A)\\u30fbP(N|A) and the former similarly, where each term of the former is the complement of its respective term in this expression. Common risk statistics are invoked as priors, i.e. P(A) = 0.9. We consider S to be a score because rigorous conditioned probabilities are difficult to establish. (For example, the data types themselves are not entirely independent of one another and models using common distributions like the Poisson do not and models using common distributions like the Poisson do not recapitulate realistic variances.) Instead, we adopt a data-driven approach of estimating contributions of each data type based on 2-point empirical fitting of roughly the low and high extremes 2-point empirical fitting of roughly the low and high extremes using shape functions. The general model for data type G is P(G|A) = [x \\u30fberf (g/y) + 1] / (x + 2) and P(G|A\\u2019) = 1 - P(G|A), where erf is the Gaussian error function, g is the observed value erf is the Gaussian error function, g is the observed value for the given data type, and x and y are empirically-determined weights. The shape function for total mutations, T, has two modifiers. First, it includes a simple expected-value correction for purity, u. Namely, assuming mutation-calling does not capture all mutations because of impurities, t is taken as the observed number of because of impurities, t is taken as the observed number of mutations divided by a purity shape function, f, where f \\u2264 1. Although one might model f according to common characteristics of mutation callers, e.g. close to 100% sensitivity for pure samples callers, e.g. close to 100% sensitivity for pure samples and very low calling rate for low variant allele fractions (VAFs), the purity estimates for these data are based on RNA-Seq and are not highly correlated with total mutation counts. Consequently, we highly correlated with total mutation counts. Consequently, we use a line function, f = 0.2\\u30fbu + 0.8, which does not strongly impact the adjustment of low-purity samples. Second, before choosing weights, we removed outliers, as determined by Peirce\\u2019s weights, we removed outliers, as determined by Peirce\\u2019s criterion. Based on the distribution of continuous smoking score, we set 0.15 as the lower bound cutoff and 0.6 as the upper bound cutoff for defining the inferred smoking status. We classified the for defining the inferred smoking status. We classified the samples with strong smoking evidence if the smoking score > 0.6; samples with weak smoking evidence if smoking score < 0.15. Identification of differentially regulated events in NRF2 mutant tumors. For each gene, normalized levels of mRNA/protein/phosphoprotein (log transformed data) were fit into a linear regression model. In addition to NFR2 mutation status, gender, tumor purity and ethnicity were also included in the model. Coefficient and and ethnicity were also included in the model. Coefficient and p-value of NRF2 mutation status variables from the fitted model were used to evaluate the extent and significance of the difference between patients with and without aberration. To adjust for between patients with and without aberration. To adjust for multi-test, p-values were adjusted with the BH method. Genes that showed significant upregulation (up) in more than one dataset (Table S4) were used to generate a GMT file for ssGSEA based NRF2 S4) were used to generate a GMT file for ssGSEA based NRF2 activity score calculation using ssGSEA (https://github.com/broadinstitute/ssGSEA2.0). The NRF2 activity score contains a total of 54 genes upregulated both at protein and RNA level, identified by genes upregulated both at protein and RNA level, identified by our NRF2 mutant versus wildtype tumor differential analysis (p-value <0.05 and logFC>1). Among the 54 genes, 44 have been previously associated with NRF2 signaling (either in NRF2 concept in associated with NRF2 signaling (either in NRF2 concept in msigdb such as Gene Sets NFE2L2.V2 and SINGH_NFE2L2_TARGETS (, BIOCARTA_ARENRF2_PATHWAY], NRF2 wikipathway or Lacher et al. Ten of the 54 genes were novel targets identified in this study. Immune cluster identification based on cell type composition The abundance of 64 different cell types were computed via xCell based on transcriptomic profiles based on 108 tumors and 94 NAT samples. Table S6 contains the final score computed by xCell of different cell types. Consensus clustering was performed based different cell types. Consensus clustering was performed based on immune cells, fibroblasts, endothelial and epithelial cells from xCell. In particular, we only considered cells that were detected in at least 10% of the patients (FDR < 5%). This filtering in at least 10% of the patients (FDR < 5%). This filtering resulted in 25 cell types annotated in Table S6. Based on these signatures, consensus clustering was performed in order to identify groups of samples with similar immune/stromal characteristics. groups of samples with similar immune/stromal characteristics. Consensus clustering was performed using the R packages ConsensusClusterPlus based on z-score normalized signatures. Specifically, signatures were partitioned into three major clusters using signatures were partitioned into three major clusters using the Partitioning Around Medoids (PAM) algorithm, which was repeated 1000 times (Figure 6A, Table S6). TCGA pan-cancer immune subtyping Tumors were classified into the six distinct immune subtypes \\u2014 wound healing, IFN-\\u03b3 dominant, inflammatory, lymphocyte depleted, immunologically quiet, and TGF-\\u03b2 dominant \\u2014 identified by a TCGA pan-cancer analysis presented in. Gene expression data (log2 pan-cancer analysis presented in. Gene expression data (log2 FPKM; lscc-v3.0-rnaseq-uq-fpkm-log2-NArm) was input to the ImmuneSubtypeClassifier R package (https://github.com/CRI-iAtlas/ImmuneSubtypeClassifier) to assign an immune subtype to each of the to assign an immune subtype to each of the tumors. Ranking tumors by inferred activity of IFN-\\u03b3 pathway We assumed that true biological activity of a pathway is regulated by collective changes of expression levels of majority of proteins involved in this pathway. Then, a difference of a pathway activity between tumors can be assessed by a difference in activity between tumors can be assessed by a difference in positioning of expression levels of proteins involved in this pathway in ranked list of expression levels of all proteins in each of tumors. Following this idea, we assessed relative positioning tumors. Following this idea, we assessed relative positioning of pathway proteins between tumor by determining two probabilities: a probability of pathway proteins to occupy by random the observed positions in a list of tumor proteins ranked by expression positions in a list of tumor proteins ranked by expression levels from the top to the bottom and, similarly, a probability to occupy by random the observed positions in a list of expression levels ranked from the bottom to the top. Then, the inferred levels ranked from the bottom to the top. Then, the inferred relative activation of a given pathway across tumors was assessed as a negative logarithm of the ratio of the above \\u201ctop\\u201d and \\u201cbottom\\u201d probabilities. Thus, for a pathway of a single protein, its probabilities. Thus, for a pathway of a single protein, its relative activity across tumors was assessed as a negative log of ratio of two numbers: a number of proteins with expression level bigger than an expression level of given protein, and a number bigger than an expression level of given protein, and a number of proteins with expression levels less than an expression level of given protein. For pathways of multiple proteins, the \\u201ctop\\u201d and \\u201cbottom\\u201d probabilities were computed as geometrically \\u201ctop\\u201d and \\u201cbottom\\u201d probabilities were computed as geometrically averaged P values computed for each of proteins using Fisher\\u2019s exact test, given protein\\u2019s ranks in a list of pathway proteins and in a list of ranked proteins of a tumor, a number of and in a list of ranked proteins of a tumor, a number of proteins in a pathway, and the total number of proteins with the assessed expression level in a given tumor. The thermodynamic interpretation of the inferred pathway activity scoring function is a of the inferred pathway activity scoring function is a free energy associated with deviation of the system from equilibrium either as a result of activation or suppression. Thus, the scoring function is positive, when expression levels of pathway\\u2019s function is positive, when expression levels of pathway\\u2019s proteins are overrepresented among top expressed proteins of a tumor, and it is negative, when pathway\\u2019s proteins are at the bottom of expressed proteins of a tumor; the scoring function is close of expressed proteins of a tumor; the scoring function is close to zero, when expression levels are distributed by random or equally shifted towards top or bottom. In our analysis, we used a gene list of IFN-\\u03b3 signaling pathway from. We settled with this 15 genes signaling pathway because it has all necessary components \\u2013 a cytokine mediator, receptors, signal transduction, transcription factors and final effectors signal transduction, transcription factors and final effectors (PD-L1/2, MHC-I), and all those genes are expressed in cancer cells. Estimation of Tumor Purity, Stromal and Immune Scores Besides xCell, we utilized ESTIMATE to infer immune and stromal scores based on gene expression data (Table S6). Cibersort absolute immune scores were obtained by evaluating upper-quartile normalized RNA-seq FPKM data with the Cibersort web app normalized RNA-seq FPKM data with the Cibersort web app (cibersort.stanford.edu) in absolute mode. To infer tumor purity, TSNet was utilized (Table S6). Differentially Expressed Genes and Pathway Analysis Genes upregulated in Hot and Warm clusters compared to the Cold cluster were identified based on gene expression data, global proteomic, phosphoproteomic and acetylation data. For this analysis, markers with at most 50% missing values were utilized. For markers with at most 50% missing values were utilized. For each data type, the expression level of gene/protein/sites was modeled as a function of immune cluster via linear regression. P-values were then adjusted for multiple comparisons via P-values were then adjusted for multiple comparisons via Benjamini-Hochberg adjustment. For each immune cluster, considering the set of genes up(down)-regulated with Benjamini-Hochberg adjusted p-value lower than 10%, a fisher exact test was implemented p-value lower than 10%, a fisher exact test was implemented to derive enriched pathways. ssGSEA was utilized to obtain pathway scores based on RNA-seq and global proteomics data using the R package GSVA. For this analysis, pathways from the Reactome, KEGG GSVA. For this analysis, pathways from the Reactome, KEGG and Hallmark databases were considered and as background the full list of gene/proteins observed under each data type was utilized. For phosphorylation and acetylproteome data, a gene was For phosphorylation and acetylproteome data, a gene was considered upregulated if at least one substrate of the gene was found upregulated at 10% FDR. The pathway analysis results for different data types are contained in Table S6. Deriving RTK CBPE scores A total of 42 human RTKs were present in our proteomics data set. For each phosphosite in our data set we computed a linear association with each of the RTKs in two ways. A phosphosite profile across samples (for tumor and NAT separately) was modeled as a across samples (for tumor and NAT separately) was modeled as a linear combination of 1) an RTK protein profile and a profile of a protein that a given phosphosite belonged to or 2) an RTK protein profile, a profile of a protein that a given phosphosite profile, a profile of a protein that a given phosphosite belonged to and a profile of tumor purity estimates obtained from RNAseq data via TSNet. Each association computation was carried out only if at least 50% of samples had non-missing data of all out only if at least 50% of samples had non-missing data of all three of RTK protein, phosphosite, and protein in question. The phosphosite:RTK associations were considered significant if their FDR adjusted p-value was below 0.1. Additionally, for each FDR adjusted p-value was below 0.1. Additionally, for each phosphosite the z-scores of their log2 phosphoprotein abundance were computed across samples (separately for tumor and NAT). A raw RTK CBPE score was computed for each RTK in each sample as a RTK CBPE score was computed for each RTK in each sample as a number of all phosphosites with significant association with this RTK that also had a z-score higher than 1. These raw RTK CBPE scores in each sample were normalized across RTKs so the final RTK in each sample were normalized across RTKs so the final RTK CBPE scores of all RTKs in one sample would add up to 1. We selected the most relevant subset of RTKs by requiring that each had a normalized CBPE score of at least 0.1 in at least 5 samples. a normalized CBPE score of at least 0.1 in at least 5 samples. Nine RTKs satisfied this condition in the tumor data set, and seven in NAT. Independent component analysis ICA was performed with a workflow modified from previously described. Decomposition was run for 100 times on the matrix of protein abundance difference between tumor/NAT pairs (n=99). Independent components were in the form of vectors comprised with components were in the form of vectors comprised with weights of all genes in the original data. Components extracted from each run were clustered using HDBSCAN algorithm with cosine distance as dissimilarity metric, min_cluster_size=50 and distance as dissimilarity metric, min_cluster_size=50 and min_samples=20. The centroids of clusters (n=37) were considered as representative of stable signatures, and mean mixing scores (activity of each signature over all samples) of each cluster were of each signature over all samples) of each cluster were used to represent the activity levels of corresponding signature in each sample. Correlation between the extracted signatures and known clinical characteristics were examined by regressing the known clinical characteristics were examined by regressing the corresponding mixing scores for all members of a component cluster against 64 sample annotations to obtain within-cluster average of log10 P-values. Significance was controlled for multiple of log10 P-values. Significance was controlled for multiple testing at 0.01 level (log10 (p-value)=\\u22125.3). Each signature vector (cluster centroid) was submitted to GSEA pre-ranked test for functional annotations. Mutation-based cis- and trans-effects We examined the cis- and trans-effects of 22 genes with somatic mutations that were significant in a previous large-scale TCGA LSCC study on the RNA, proteome, and phosphoproteome of known interactome DBs including Omnipath, Phosphositeplus, DEPOD, interactome DBs including Omnipath, Phosphositeplus, DEPOD, Signor, and CORUM. After excluding silent mutations, samples were separated into mutated and WT groups. We used the Wilcoxon rank-sum test to report differentially expressed features (RNA, rank-sum test to report differentially expressed features (RNA, proteins, phosphosites, acetylsites, and ubiquitylation sites) between the two groups. Differentially enriched features passing an FDR <0.05 cut-off were separated into two categories based an FDR <0.05 cut-off were separated into two categories based on cis- and trans- effects. Germline quantitative trait loci (QTL) analysis To identify germline genetic variants that explain variation in tumor gene (eQTL) and protein (pQTL) expression, we utilized the gold-standard mapping pipeline at https://github.com/molgenis/systemsgenetics/wiki/eQTL-mapping-analysis-cookbook-(eQTLGen). Briefly, we followed the default steps that include correcting for population stratification removing outliers, normalizing and mapping cis-QTLs. For cis-eQTL analysis input, we used i) germline genotype data; and ii) RNA-seq raw read count data for 105 genotype data; and ii) RNA-seq raw read count data for 105 samples, and performed analyses only for genes with sum of read count \\u2265 10. For cis-pQTL analysis input, we used i) germline genotype data; and ii) normalized and filtered protein TMT ratio data; and ii) normalized and filtered protein TMT ratio aggregated to gene level for 108 samples. The genotype data were harmonized using Genotype Harmonizer v1.4.9. To adjust for population stratification, we identified the multi dimensional scaling stratification, we identified the multi dimensional scaling (MDS) components using the genotype data using PLINK v1.07. For both types of cis-QTL mapping, we used eqtl-mapping-pipeline-v1.3.9 using its standard settings (HWE > 0.0001, MAF > 0.01, and call using its standard settings (HWE > 0.0001, MAF > 0.01, and call rate > 0.95, maximum distance between the SNP and the middle of the probe is 250,000 bp). To control for multiple testing, we performed 10 permutations. We deemed QTLs at \\u2264 FDR 5% as we performed 10 permutations. We deemed QTLs at \\u2264 FDR 5% as significant. Finally, we performed pathway analysis of the identified significant eGenes (genes whose expression is impacted by at least one cis-QTL) using QIAGEN Ingenuity Pathway Analysis miRNA analysis presented in Figure 4 Targets of miRNAs were downloaded from the miRNA targets database miRTarBase and only the miRNA/target pairs with strong experimental evidence were retained. Spearman correlation was used to calculate the correlations between miRNA and its target genes. Pathway projection using ssGSEA The single sample Gene Set Enrichment Analysis (ssGSEA) implementation available on https://github.com/broadinstitute/ssGSEA2.0 was used to project log2(FPKM) mRNA abundances to MSigDB cancer hallmark gene sets using the following parameters: gene.set.database=\\u201ch.all.v6.2.symbols.gmt\\u201d\\n\\nsample.norm.type=\\u201crank\\u201d\\n\\nweight=0.75\\n\\nstatistic=\\u201darea.under.RES\\u201d\\n\\noutput.score.type=\\u201cNES\\u201d\\n\\nnperm=1000\\n\\nglobal.fdr=TRUE\\n\\nmin.overlap=10\\n\\ncorrel.type=\\u201dz.score\\u201d\\n\\nPhosphorylation-driven signature analysis We performed phosphosite-specific signature enrichment analysis (PTM-SEA) to identify dysregulated phosphorylation-driven pathways. To adequately account for both magnitude and variance of measured phosphosite abundance, we used p-values derived from measured phosphosite abundance, we used p-values derived from application of the Wilcoxon rank-sum test to phosphorylation data as ranking for PTM-SEA. To that end, p-values were log-transformed and signed according to the fold change (signed \\u2212log10 and signed according to the fold change (signed \\u2212log10 (p-value)) such that large positive values indicated phosphosite abundance in classical or NFE2L2, CUL3, KEAP1 mutated samples, and large negative values indicated phosphosite abundance in samples negative values indicated phosphosite abundance in samples that do not belong to classical NMF subtype or wild-type samples (in Figure 3I)  PTM-SEA relies on site-specific annotation provided by PTMsigDB and thus a single site-centric data matrix data is by PTMsigDB and thus a single site-centric data matrix data is required such that each row corresponds to a single phosphosite. We note that in this analysis the data matrix comprised a single data column and each row represented a confidently localized data column and each row represented a confidently localized phosphosite assigned by Spectrum Mill software. We employed the heuristic method introduced by Krug et al. to deconvolute multiple phosphorylated peptides to separate data points (log-transformed and signed p-values). Briefly, phosphosites measured on different phospho-proteoform peptides were resolved measured on different phospho-proteoform peptides were resolved by using the p-value derived from the least modified version of the peptide. For instance, if a site T4 measured on a doubly phosphorylated (T4, S8) peptide (PEPtIDEsR) was also measured on a (T4, S8) peptide (PEPtIDEsR) was also measured on a mono-phosphorylated version (PEPtIDESR), we assigned the p-value derived from the mono-phosphorylated peptide proteoform to T4, and the p-value derived from PEPtIDEsR to S8. If only the doubly the p-value derived from PEPtIDEsR to S8. If only the doubly phosphorylated proteoform was present in the dataset, we assigned the same p-value to both sites T4 and S8. We queried the PTM signatures database (PTMsigDB) v1.9.0 downloaded from https://github.com/broadinstitute/ssGSEA2.0/tree/master/db/ptmsigdb using the flanking amino acid sequence (+/\\u2212 7 aa) as primary identifier. We used the implementation of PTM-SEA as primary identifier. We used the implementation of PTM-SEA available on GitHub (https://github.com/broadinstitute/ssGSEA2.0) using the command interface R-script (ssgsea-cli.R). The following parameters were used to run PTM-SEA: weight: 1\\n\\nstatistic: \\u201carea.under.RES\\u201d\\n\\noutput.score.type: \\u201cNES\\u201d\\n\\nnperm: 1000\\n\\nmin.overlap: 5\\n\\ncorrel.type: \\u201cZ-score\\u201d The sign of the normalized enrichment score (NES) calculated for each signature corresponds to the sign of the tumor-NAT log fold change. P-values for each signature were derived from 1,000 random permutations and further adjusted for multiple hypothesis permutations and further adjusted for multiple hypothesis testing using the method proposed by Benjamini & and Hochberg. Signatures with FDR-corrected p-values < 0.05 were considered to be differential between tumor and NAT. CDKN2A and RB1 annotations and pathway analysis Comprehensive tumor annotation for CDKN2A and RB1 genomic status was carried out using multiple molecular features for each patient. Specifically, we considered the following: 1) mutation types (missense/in-frame indels mutations, nonsense (stop gain, types (missense/in-frame indels mutations, nonsense (stop gain, frameshift indels) mutations, and splice site (splice donor, splice acceptor) mutations) as separate categories and 2) copy number data. For cases with mutation, we considered both the number data. For cases with mutation, we considered both the variant allele frequency of the mutation from whole exome sequencing and copy number data (log ratio, absolute copy number, and B-allele frequency). Only mutations that occurred in cases with B-allele frequency). Only mutations that occurred in cases with loss of the wild-type allele were annotated as loss of heterozygosity (LOH) mutations and used for subsequent analysis. 2) Based on CDKN2A copy number data, we next annotated tumors without on CDKN2A copy number data, we next annotated tumors without mutations as homozygous deletions or no loss (WT). Samples were re-classified as having p16INK4a promoter hypermethylation if three p16INK4a promoter-associated methylation probes (cg11075751, p16INK4a promoter-associated methylation probes (cg11075751, cg02008397, and cg01694391) all had beta values >= 0.2. Multi-gene proliferation scores were calculated as described previously as the means of the gene normalized RNA levels for cell cycle as the means of the gene normalized RNA levels for cell cycle associated genes characterized by Whitfield et al. p16INK4a RNA levels were obtained from the isoform specific RSEM data for Ensembl transcript ENST00000304494.8, while ENST00000579755.1 was transcript ENST00000304494.8, while ENST00000579755.1 was used for p14ARF, and log2 (RSEM + 1) values were used for barplots. Isoform specific protein data included Refseq NP_000068.1 for p16INK4a and Refseq NP_478102.2 for p14ARF. Calculation of CDK1, 2, and Refseq NP_478102.2 for p14ARF. Calculation of CDK1, 2, and 4 target site scores from the phosphorylation site data and Hallmark E2F targets and G2M scores from the RNA-seq data is described above (Phosphorylation-driven signature analysis and Pathway above (Phosphorylation-driven signature analysis and Pathway projection using ssGSEA, respectively). Lolliplots for CDKN2A mutations in the CPTAC head and neck squamous cell carcinoma and this lung squamous cell carcinoma cohort were generated using the lung squamous cell carcinoma cohort were generated using the ProteinPaint web application to visualize mutations. Mutations annotated as nonsense mediated decay (NMD) mutations were those that resulted in low expression of p16INK4a RNA in the tumors that resulted in low expression of p16INK4a RNA in the tumors harboring the mutations. Association analysis between KGG-site abundances and E3 ligases and DUBs A list of known human E3 ubiquitin and ubiquitin-like ligases and DUBs was compiled from. We then fit a linear model using limma in R with the formula kgg_site_abundance ~ protein_abundance, followed by empirical bayes shrinkage. The coefficient p-values followed by empirical bayes shrinkage. The coefficient p-values were used to determine significant associations after FDR correction (FDR < 0.01). Cluster and pathway analysis of significantly modulated K-GG sites in tumors. Consensus clustering of the K-GG dataset after protein abundance correction was performed on tumor samples, resulting in 3 clusters. Next, marker selection for each cluster was performed by moderated t-test between samples that belong to the clusters by moderated t-test between samples that belong to the clusters against the remaining samples. These K-GG sites showing differential abundance across sample clusters were further clustered into 3 site-wise clusters. Genes in these site-wise clusters were used for pathway enrichment analysis against the KEGG, Reactome, and WikiPathways databases using g:profiler. Pathway enrichment was performed using a gene background containing all observable genes in the K-GG a gene background containing all observable genes in the K-GG dataset. Pathway enrichment results were imported into Cytoscape using the Enrichment Map app for network analysis of pathways. Pathways were connected using the gene overlap and clustered Pathways were connected using the gene overlap and clustered (pathway cutoff q-val < 0.1; jaccard overlap > 0.375). Each cluster was manually annotated from the pathways contained in it to facilitate interpretation. PTM CLUMPS analysis We employ two methods to select tumor-specific sites to include in structural analysis. First, we take PTM-sites for solely tumor-derived samples and binarize modifications by negative vs. positive normalized signal. These are considered as individual positive normalized signal. These are considered as individual \\u201cevents\\u201d as done with mutations in CLUMPS v1. For a more robust approximation of tumor-specific acetylation or ubiquitination, we perform differential expression using Limma between NAT and we perform differential expression using Limma between NAT and Tumor samples. Sites are selected with an FDR < 0.1 (Benjamini-Hochberg) and LFC > 0.1. We then binarize these tumor sites as done in the first approach. We first map the PTM-sites to as done in the first approach. We first map the PTM-sites to corresponding UNIPROT ids (ID) with available PDBs. For each crystal structure, we compute an initial WAP score and randomly sample sites as done in CLUMPS. We generate an empirical p-value sites as done in CLUMPS. We generate an empirical p-value based on a random sampling of lysines in each crystal structure to limit the selection to residues capable of ubiquitination or acetylation. The null hypothesis we define is each random sampling of The null hypothesis we define is each random sampling of lysines will have a WAP score less than or equal to the initially computed WAP score. We run 1e6 permutations to generate an empirical p-value. DepMap genetic dependency and drug response analysis\\n\\nCell lines annotated as \\u201cNSCLC_squamous\\u201d from DepMap were considered as LSCC for which molecular profiles, dependency scores, and drug response were used. To determine copy number status in LSCC cell lines, copy number segmentation files for LSCC cell lines were processed with GISTIC2.0 (using the same parameters used to process this study\\u2019s LSCC cohort as described in STAR METHODS). Samples were considered cohort as described in STAR METHODS). Samples were considered to be amplified for a given gene if GISTIC thresholded values = 1 or 2 and not amplified if GISTIC thresholded values = \\u22122, \\u22121, or 0. Mutation status for CCND1, CDKN2A, and RB1, Np63 transcript 0. Mutation status for CCND1, CDKN2A, and RB1, Np63 transcript isoform expression, phospho-Rb-Ser807/811 RPPA abundance, and drug responses to a survivin inhibitor (YM-155) and CDK4/6i inhibitors (abemaciclib and palbociclib) were also retrieved from (abemaciclib and palbociclib) were also retrieved from DepMap. Figure 7: For identifying if top protein biomarkers (502 proteins significantly overexpressed (log2(FC) >2, FDR <0.01) in tumors relative to their matched NATs, most with coherent overexpression in multi-omic analysis) also conferred altered dependencies in multi-omic analysis) also conferred altered dependencies in LSCC cell lines, we leveraged DepMap genetic dependency dataset (CRISPR Avana Public 20Q3) that contained 18119 genes and 789 cell lines (https://depmap.org/portal/download/ file: and 789 cell lines (https://depmap.org/portal/download/ file: Achilles_gene_effect.csv). Only 16 cell lines were classified as LSCC (Cell Line Sample Info.csv). Median dependencies were calculated and for every gene and density plot in Figure 7C shows and for every gene and density plot in Figure 7C shows dependency score of the 502 genes corresponding to 502 protein biomarkers relative to other genes. Figure S4B: To investigate copy number and dependency associations in LSCC cell lines, each gene up-regulated at the protein level with recurrent copy number gain in tumors vs NATs (from Figure 4A with Log2FC>0 & FDR < 0.05) was used as a query to compare 4A with Log2FC>0 & FDR < 0.05) was used as a query to compare dependency scores of the given gene (Combined CRISPR KO screen, DepMapPublic 21Q1) in LSCC cell lines where the query gene was amplified vs not-amplified. A lower mean change in dependency amplified vs not-amplified. A lower mean change in dependency scores for a given gene in amplified vs not-amplified samples indicates LSCC cell lines are more dependent upon that gene when it is amplified. Continuous log2 copy number data from DepMap was used in Figure S4M to correlate SOX2 copy number with EZH2 shRNA dependency data (Combined shRNA screen from DEMETER2 Data v6 in DepMap). Drug responses to YM-155 (PRISM secondary screen) were compared by t-test in LSCC cell lines with low vs high levels of Np63 by transcript isoform expression (Figure S4G). Pearson correlation was also performed with phospho-Rb-Ser807/811 RPPA abundance was also performed with phospho-Rb-Ser807/811 RPPA abundance and drug responses to CDK4/6i inhibitors, abemaciclib (PRISM secondary screen) and palbociclib (Sanger GDSC1) (Figure S3D). CausalPath analysis CausalPath searches for known biological mechanisms that can explain correlated proteomic changes in terms of causal hypotheses. We set CausalPath parameters to compare tumors and NATs with a paired t-test, used 0.1 as FDR threshold for proteomic change a paired t-test, used 0.1 as FDR threshold for proteomic change significance and network significance, and detected 5917 potential causal relations between proteins. We repeated the same analysis for each NMF subtype separately and identified 4378 analysis for each NMF subtype separately and identified 4378 (basal-inclusive), 5334 (classical), 3048 (EMT-enriched), 3744 (inflamed-secretory), and 4332 (proliferative-primitive) relations. We used these CausalPath network results in the preparation of We used these CausalPath network results in the preparation of Figure 7C, identifying potential upstream regulators of oncogenic phosphoproteomic changes. Here an oncogenic phosphoproteomic change can be any of the following 4 events: increase of change can be any of the following 4 events: increase of activating phosphorylation of an oncoprotein, decrease of inactivating phosphorylation of an oncoprotein, decrease of activating phosphorylation of a tumor suppressor protein, and increase of phosphorylation of a tumor suppressor protein, and increase of inactivating phosphorylation of a tumor suppressor protein. We used the OncoKB database for oncoprotein and tumor suppressor classification (excluded proteins that have both annotations), and (excluded proteins that have both annotations), and used PhosphoSitePlus and Signor databases for the activating/inhibiting classification of phosphorylation sites. In the phosphorylation regulation networks, we included only the targetable regulators regulation networks, we included only the targetable regulators (activated proteins) and excluded the untargetable regulators (inactivated proteins). Variant Peptide Identification We used NeoFlow (https://github.com/bzhanglab/neoflow) for neoantigen prediction. Specifically, Optitype was used to find human leukocyte antigens (HLA) for each sample based on WES data. Then we used netMHCpan to predict HLA peptide binding affinity for we used netMHCpan to predict HLA peptide binding affinity for somatic mutation\\u2013derived variant peptides with a length between 8\\u201311 amino acids. The cutoff of IC50 binding affinity was set to 150 nM. HLA peptides with binding affinity higher than 150 nM 150 nM. HLA peptides with binding affinity higher than 150 nM were removed. Variant identification was also performed at both mRNA and protein levels using RNA-Seq data and MS/MS data, respectively. Variant identification and gene quantification using Variant identification and gene quantification using RNA-Seq data were performed following the methods used in the previous study. To identify variant peptides, we used a customized protein sequence database approach. We built a customized database for sequence database approach. We built a customized database for each TMT experiment based on somatic variants from WES data. We used Customprodbj (https://github.com/bzhanglab/customprodbj) for customized database construction. MS-GF+ was used for variant customized database construction. MS-GF+ was used for variant peptide identification for all global proteome and phosphorylation data. Results from MS-GF+ were filtered with 1% FDR at peptide level. Remaining variant peptides were further filtered using level. Remaining variant peptides were further filtered using PepQuery (http://www.pepquery.org) with the p-value cutoff <= 0.01. Competitive filtering based on unrestricted posttranslational modification searching was enabled in PepQuery validation. The modification searching was enabled in PepQuery validation. The spectra of variant peptides were annotated using PDV (http://www.zhang-lab.org/). Cancer/testis Antigen Prediction\\n\\nCancer/testis (CT) antigens were downloaded from the CTdatabase. CT antigens with a median >4-fold increase in tumor from NAT were highlighted.\\n\\nPROGENy Scores PROGENy Scores\\n\\nPROGENy was used to generate activity scores for EGFR based on RNA expression data. Tumor RNA expression values were submitted to PROGENy.\\n\\nLSCC, HNSCC and LUAD integrative analysis LUAD data for 110 lung adenocarcinoma samples and 102 NAT were acquired from the published manuscript. HNSCC data for 108 head and neck squamous cell carcinoma samples and 66 NAT were acquired from the submitted manuscript. The LUAD and HNSCC proteomics from the submitted manuscript. The LUAD and HNSCC proteomics data were processed, quantified, and normalized using the same pipeline as described for the LSCC samples. Differential expression analysis for all three cohorts was performed as described in analysis for all three cohorts was performed as described in the Differential marker analysis method. Proteins missing in more than 50% of the paired tumor/NAT samples were excluded. Copy number drivers for all three were assessed by filtering the 8309 genes that were quantified in all three cohorts in the copy number, RNA, and proteomic data to those found in focal amplification regions (q value < .25). Spearman correlation was amplification regions (q value < .25). Spearman correlation was performed for these genes between CNA and RNA and between CNA and protein. Proteins were considered drivers if the correlation between both the CNA and RNA and CNA and protein were between both the CNA and RNA and CNA and protein were significantly positively associated (BH adjusted p value <0.01). Proteins were also required to be significantly increased in tumor vs paired NAT in the same cohort (Wilcoxon signed rank BH adjusted p NAT in the same cohort (Wilcoxon signed rank BH adjusted p value <0.01). To identify genes associated with the immune score, correlation between CNA, RNA, protein, and the immune score was performed for the genes present in both CNA and RNA in a cohort (20,313 genes in LSCC, 18,091 genes in LUAD, and 21,616 genes in HNSCC). in LSCC, 18,091 genes in LUAD, and 21,616 genes in HNSCC). The immune score was calculated as the z-score transformation of the ESTIMATE Immune score, which was calculated for all three cohorts as described in the method Estimation of Tumor Purity, cohorts as described in the method Estimation of Tumor Purity, Stromal and Immune Scores. To be considered copy number drivers of the immune score, genes had to have a positive correlation between CNA & RNA, CNA & immune score, and RNA & immune score. If CNA & RNA, CNA & immune score, and RNA & immune score. If the protein was quantified in the cohort, the gene was also required to have a positive correlation between CNA & protein and protein & immune score. Spearman correlations with a BH adjusted p & immune score. Spearman correlations with a BH adjusted p value < 0.01 were considered significant. Supplementary Material\\n\\nDeclaration of Interests\\n\\nThe authors declare no competing interests\\n\\nADDITIONAL RESOURCES ADDITIONAL RESOURCES\\n\\nThe CPTAC program website, detailing program initiatives, investigators, and datasets, is found at https://proteomics.cancer.gov/programs/cptac. A website for interactive visualization of the multi-omic dataset is available at: https://rstudio-connect.broadapps.org/CPTAC-LSCC2021/ All processed data matrices are available at LinkedOmics (http://www.linkedomics.org), where computational tools are available for further exploration of this dataset.\\n\\nConsortia Alex Green, Alexey Nesvizhskii, Alfredo Molinolo, Alicia Francis, Amanda Paulovich, Ana Robles, Andrii Karnuta, Antonio Colaprico, Arul Chinnaiyan, Azra Krek, Barbara Hindenach, Barbara Pruetz, Bartosz Kubisa, Bing Zhang, Bo Wen, Boris Reva, Brian Druker, Bartosz Kubisa, Bing Zhang, Bo Wen, Boris Reva, Brian Druker, Carissa Huynh, Chandan Kumar-Sinha, Charles Goldthwaite Jr., Chelsea Newton, Chen Huang, Chet Birger, Chia-Kuei Mo, Christopher Kinsinger, Corbin Jones, DR Mani, Dan Rohrer, Dana Valley, Daniel Corbin Jones, DR Mani, Dan Rohrer, Dana Valley, Daniel Chan, David Chesla, David Feny\\u00f6, David Heiman, Dmitry Rykunov, Donna Hansel, Elena Ponomareva, Elizabeth Duffy, Eric Burks, Eric Jaehnig, Eric Schadt, Erik Bergstrom, Eugene Fedorov, Eunkyung An, Fei Eric Schadt, Erik Bergstrom, Eugene Fedorov, Eunkyung An, Fei Ding, Felipe da Veiga Levoprost, Fernanda Martins Rodrigues, Francesca Petralia, Gad Getz, Galen Hostetter, George Wilson, Gilbert Omenn, Henry Rodriguez, Houxiang Zhu, Hui Zhang, Jackson Omenn, Henry Rodriguez, Houxiang Zhu, Hui Zhang, Jackson White, James Suh, Jennifer Eschbacher, Jennifer Maas, Jonathan Lei, Jiayi Ji, Kai Li, Karen Christianson, Karen Ketchum, Karin Rodland, Karl Clauser, Karsten Krug, Katherine Hoadley, Kei Suzuki, Karl Clauser, Karsten Krug, Katherine Hoadley, Kei Suzuki, Kelly Ruggles, Ki Sung Um, Li Ding, Liqun Qi, Lori Bernard, Harry Kane, Maciej Wiznerowicz, MacIntosh Cornwell, Ma\\u0142gorzata Wojty\\u015b, Marcin Cieslik, Marcin Domagalski, Mathangi Thiagarajan, Matthew Cieslik, Marcin Domagalski, Mathangi Thiagarajan, Matthew Ellis, Matthew Wyczalkowski, Maureen Dyer, Meenskshi Anurag, Mehdi Mesri, Melissa Borucki, Michael Birrer, Michael Gillette, Midie Xu, Mikhail Krotevich, Myvizhi Esai Selvan, Namrata Udeshi, Nancy Mikhail Krotevich, Myvizhi Esai Selvan, Namrata Udeshi, Nancy Roche, Nathan Edwards, Negin Vatanian, Neil Mucci, Nicollette Maunganidze, Nikolay Gabrovski, Olga Potapova, Oluwole Fadare, \\u00d6zg\\u00fcn Babur, Pamela Grady, Pankaj Vats, Paul Paik, Pei Wang, Peter Babur, Pamela Grady, Pankaj Vats, Paul Paik, Pei Wang, Peter McGarvey, Pierre Jean Beltran, Pushpa Hariharan, Qing Kay Li, Ramaswamy Govindan, Ratna Thangudu, Rebecca Montgomery, Richard Smith,Robert Welsh, Runyu Hong, Sailaja Mareedu, Samuel Payne, Smith,Robert Welsh, Runyu Hong, Sailaja Mareedu, Samuel Payne, Sandra Cottingham, Sara Savage, Saravana Dhanasekaran, Scott Jewell, Sendurai Mani, Seungyeul Yoo, Shankara Anand, Shankha Satpathy, Shayan Avanessian, Shilpi Singh, Shirley Tsang, Shuang Cai, Shayan Avanessian, Shilpi Singh, Shirley Tsang, Shuang Cai, Song Cao, Stacey Gabriel, Stephan Schurer, Steven Carr, Suhas Vasaikar, Tao Liu, Tara Hiltke, Tatiana Omelchencko, Thomas Bauer, Tobias Schraink, Vasileios Stathias, Volodymyr Sovenko, Warren Tobias Schraink, Vasileios Stathias, Volodymyr Sovenko, Warren Tourtellotte, Weiping Ma, Wenke Liu, William Bocik, Wohaib Hasan, Xiaoyu Song, Yifat Geffen, Yize Li, Yongchao Dou, Yuping Zhang, Yuxing Liao, Yvonne Shutack, Zeynep Gumus, Zhen Zhang, Ziad Yuxing Liao, Yvonne Shutack, Zeynep Gumus, Zhen Zhang, Ziad Hanhan. This is a PDF file of an article that has undergone enhancements after acceptance, such as the addition of a cover page and metadata, and formatting for readability, but it is not yet the definitive version of record. This version will undergo additional version of record. This version will undergo additional copyediting, typesetting and review before it is published in its final form, but we are providing this version to give early visibility of the article. Please note that, during the production of the article. Please note that, during the production process, errors may be discovered which could affect the content, and all legal disclaimers that apply to the journal pertain. Proteogenomic Landscape of LSCC A. Schematic showing the number of tumors and NATs profiled and the various data types generated in this study. The lower panel represents data completeness. WGS: Whole Genome Sequencing, WES: Whole Exome Sequencing. CNA: copy number alteration. DNAme: Whole Exome Sequencing. CNA: copy number alteration. DNAme: DNA methylation. pSTY: phosphoproteome. Ac: acetylproteome. Ub: ubiquitylproteome. B. Stacked histograms indicating the distribution of patient phenotypes. Smoking History reflects self-report. C. Co-occuring mutation plot indicating cancer-relevant genes. MutSig-based significantly mutated genes (SMGs, q-value < 0.1) in this dataset are highlighted in red font. D. Heatmaps showing correlation between copy number alterations (CNA) and RNA (left) or proteomics (right). Red and green events represent significant (FDR <0.01) positive and negative correlations, respectively. E. Flow chart for identification of cancer-associated genes (CAGs) that showed GISTIC-based focal amplification or deletion (q<0.25) and cis-effects in both mRNA and protein (FDR<0.05). F. Differential protein expression (Log2 fold-change (FC)) in tumors with and without high-level amplification of the FGFR1 gene (GISTIC thresholded value =2). G. Genes whose DNA methylation was significantly associated with cascading cis regulation of their cognate mRNA expression, protein level, phosphopeptide and acetylpeptide abundance. Shapes indicate the cis-effects across the indicated datasets. Named indicate the cis-effects across the indicated datasets. Named genes also showed differential expression between tumors and NATs. See also Figure S1 and Table S1\\u20133\\n\\nLSCC Molecular Subtypes and Associations\\n\\nA. NMF-based clustering of tumor CNA, RNA, protein, phosphosite and acetylsite profiles, showing five primary NMF subtypes (top sample annotation row). B. Heatmap representing significantly enriched pathways (MSigDB Hallmark) in five multi-omic subtypes. C. Kaplan-Meier plot comparing survival probability of patients whose tumors were core members of a specific NMF subtype (NMF Core) to those whose tumors had characteristics of more than one NMF subtype (NMF mixed). D. Heatmap showing relative overexpression of mesenchymal proteins in the EMT-E subtype compared to others. E. Correlation between ssGSEA-based enrichment of EMT (Hallmark genesets) and fibroblast proliferation (GO: Gene Ontology) genesets (Pearson correlation=0.65, p=2.8\\u00d710\\u221214). F. Distribution of RTK correlation-based phosphosite enrichment (RTK CBPE) scores for PDGFRB and ROR2 across the five NMF subtypes. Wilcoxon p values for CBPE scores in EMT-E vs other subtypes are 1.5\\u00d710\\u22126 for PDGFRB and 2.7\\u00d710\\u22127 for ROR2. G. Proteins and phosphosites significantly associated with PDGFRB or ROR2 CBPE scores, known to play a role in EMT and extracellular matrix reorganization. The left panel shows Spearman correlation between PDGFRB CBPE scores and protein/phosphosite correlation between PDGFRB CBPE scores and protein/phosphosite abundance profiles. See also Figure S2 and Table S1\\u20134\\n\\nImpact of Somatic Mutations on Proteogenomic Features A. Significant (Wilcoxon FDR<0.05) cis- (circles) and trans-effects (squares) of selected mutations (x axis) on the expression of cancer-associated gene products, with mRNA in blue and proteins in green.\\n\\nB. Similar to panel A but showing phosphosites. B. Similar to panel A but showing phosphosites.\\n\\nC. Similar to panel A but showing acetylsites.\\n\\nD. Similar to panel A but showing ubiquitylsites. E. CNA data for CDKN2A and RB1 was used to classify tumors as having homozygous deletions or three classes of loss of heterozygosity mutations: nonsense/frameshift indel, missense/inframe indel, and splicing (see Table S4). CDKN2A genetic and indel, and splicing (see Table S4). CDKN2A genetic and hypermethylation annotations were based on the effect of the aberration on the p16INK4a (p16) gene product, but the effects of these CDKN2A/p16 aberrations on both major isoforms (p16INK4a and p14ARF aberrations on both major isoforms (p16INK4a and p14ARF (p14)) at the RNA (barplot) and protein (heatmap directly below barplot) levels are shown. Samples with amplification of CCND1\\u20133, CDK4, and CDK6 were assessed by GISTIC (threshold = 2), and the CDK4, and CDK6 were assessed by GISTIC (threshold = 2), and the genomic status, protein, and phosphoprotein levels for RB1 are included. Also shown are RNA-based scores for the cell cycle (MGPS, the mean of cell cycle genes and E2F target and G2M (MGPS, the mean of cell cycle genes and E2F target and G2M checkpoint gene set scores derived from ssGSEA of Hallmark gene sets) and phosphosite-based CDK kinase activity scores for CDKs 1, 2, and 4 derived from single sample post translational CDKs 1, 2, and 4 derived from single sample post translational modifications - signature enrichment analysis (ssPTM-SEA) of known kinase targets. Three tumors with copy number gain of CDKN2A are not included. F. Correlation between differential regulation of protein abundance (Log2 Fold-change (FC)) versus phosphoprotein log2 FC in tumors with NRF2 pathway mutation (one or two mutations in KEAP1, CUL3, or NFE2L2) versus NRF2 WT tumors (no NRF2 pathway KEAP1, CUL3, or NFE2L2) versus NRF2 WT tumors (no NRF2 pathway aberration). G. NRF2 pathway score and RNA, protein and phosphoprotein expression of key NRF pathway members according to NMF subtype.\\n\\nH. CDK5 protein expression (Log2 TMT ratio) by NMF subtype. P-values are from the Anova test. I. PTM-SEA-derived normalized enrichment scores (NES) for pathways enriched in NRF2 pathway-mutated (Mutant) vs wild-type samples (WT) plotted against NES for pathways enriched in NMF Classical vs other subtypes. Significantly upregulated (FDR<0.05) vs other subtypes. Significantly upregulated (FDR<0.05) PTM-SEA terms in the Classical subtype are indicated by red dots and labeled. See also Figure S3 and Table S4\\n\\nProteogenomic Impact of Chromosome 3q Amplification\\n\\nA. Differential protein expression (Log2 FC) between tumors and NATs for genes on chromosome arm 3q. B. Frequency distribution of \\u0394Np63\\u03b1 RNA expression in tumors and NATs. C. Differential protein expression in samples classified as \\u0394Np63\\u03b1-low vs -high based on \\u0394Np63\\u03b1 transcript level. The outlier upregulated gene product in red is BIRC5, also known as survivin. D. Differential expression of microRNAs in \\u0394Np63\\u03b1-low vs -high samples.\\n\\nE. Pearson correlation between expression of miR-205 and mRNA expression of its cognate, experimentally validated targets. F. Pearson correlation between expression of miR-205 and protein abundance of its cognate targets. G. Relationship between miR-205 expression (log2 TPM) and EMT (top, p=2.3\\u00d710\\u221208, Correlation = \\u22120.53) and DNA replication (bottom, p=1.1\\u00d710\\u221206, Correlation= 0.47) scores. H. Heatmap showing relative protein expression (TMT log ratio) of selected proteins with significant (FDR<0.01) Pearson correlation values (positive or negative) with SOX2 protein expression. I. Pearson correlation (p= 5.2\\u00d710\\u221207, Correlation = \\u22120.46) between SOX2 protein expression and HALLMARK_IL6_JAK-STAT_signaling NES.\\n\\nSee also Figure S4 and Table S4\\n\\nUbiquitylation landscape in LSCC A. Consensus clustering of protein ratio-corrected K-GG (di-glycine) site abundances in tumor samples and their associations. Heatmap shows only protein ratio-corrected K-GG sites with differential abundance across clusters (FDR<0.01). Enriched pathways abundance across clusters (FDR<0.01). Enriched pathways and molecular and clinical annotations are indicated. B. Number of K-GG sites showing significant correlations (FDR< 0.01) with E3 ligases. Shown are the five E3 ligases with the highest proportion of positive correlations. C. Pearson correlations between HERC5 protein expression and K-GG site protein-corrected abundance in key glycolytic enzymes PKM, PGK1, and ENO1. D. HERC5 protein expression (log2 TMT ratio) with samples grouped by immune subtype. Significant (Kruskal\\u2212Wallis, p = 2.8\\u00d710\\u221205) association is seen between HERC5 abundance and immune subtypes. E. Representative examples of significant spatial clustering of lysine acetylsites (purple) on PGK1 (left) and ACADVL (right) protein 3-D structure space-filling models (cyan) as determined by PTM CLUMPS. (PGK1 structure = PDB ID:3ZOZ. ACADVL structure = PTM CLUMPS. (PGK1 structure = PDB ID:3ZOZ. ACADVL structure = PDB ID:3B9). F. TXN protein levels in the NMF Classical subtype relative to NATs and other NMF subtypes (top left). Protein-corrected ubiquitylation (K-GG) sites are decreased on TXN1 in tumor subtypes relative to NATs (top right). TXNIP is decreased in tumor subtypes to NATs (top right). TXNIP is decreased in tumor subtypes relative to NATs (lower left). TXN1 activator TXNRD1 is increased in the Classical subtype relative to NATs and other NMF subtypes (lower right). Kruskal\\u2212Wallis p-values are indicated in the (lower right). Kruskal\\u2212Wallis p-values are indicated in the respective plots. G. Schematic representation of PTM-based modulation in LSCC tumors showing key enzymes in the metabolic and reactive oxygen species (ROS) pathways. Green and red arrows indicate higher and lower abundance of the corresponding PTMs in tumors. Putative lower abundance of the corresponding PTMs in tumors. Putative ISGylation targets of HERC5 are indicated by dotted lines. A known regulatory PKM phosphosite observed to be modulated in LSCC tumors is also highlighted. H. Lollipop charts showing the Log2 FC of acetylated (K-Ac) and ubiquitylated sites (K-GG) between tumors and NATs (Hyper: log2(FC)>1 or Hypo: <\\u22121, FDR<0.01). The upper panel shows specific sites that were hyper-ubiquitylated and hypoacetylated in tumors; that were hyper-ubiquitylated and hypoacetylated in tumors; the lower panel shows specific sites that were hyperacetylated and hypo-ubiquitylated in tumors. Dot colors indicate protein fold change between tumors and NATs. \\u201ck\\u201d represents modified lysine. I. Relative abundances of RAN K127 acetylation (K-Ac), ubiquitylation (K-GG) and RAN protein levels across NMF subtypes and NATs. Wilcoxon p-values are indicated above; ns represents p>0.05.\\n\\nSee also Figure S5 and Table S5\\n\\nImmune Landscape of LSCC A. Heatmaps illustrate cell type compositions and activities of selected individual genes/proteins and pathways across the four immune clusters: Hot, Warm, and Cold tumor and NAT-enriched. Successive heatmaps illustrate xCell immune signatures, mRNA and heatmaps illustrate xCell immune signatures, mRNA and protein expression of key immune-related markers, and ssGSEA pathway scores based on global proteomic data for biological pathways that were differentially regulated in immune groups based on both mRNA differentially regulated in immune groups based on both mRNA and global protein abundance (Common) or on global protein abundance alone (Proteomics only). B. Pathway scores of key pathways differentially expressed across the immune clusters. Wilcoxon p-values for the individual comparisons are provided on top. C. Contour plot of two-dimensional density based on Macrophage (x-axis) and CD8 T-cell scores (y-axis) showing the variation in these cell types\\u2019 distributions observed across the different immune clusters. D. Acetylsites differentially expressed between Hot and Cold tumors. Acetylsites of genes contained in the Hallmark Oxidative Phosphorylation pathway are highlighted in blue, ARHGDIB K135 is highlighted in red, and remaining sites are in gray. Darker is highlighted in red, and remaining sites are in gray. Darker color designates significant sites (FDR < 0.1). E. Regulation of Rho GTPase signaling including K135 acetylation of ARHGDIB. F. Global protein abundance of RAC2, DOCK2 and ELMO1, acetylproteome abundance of ARHGDIB K135k and phosphorylation abundance of ARHGEF6 at Serine 225 in immune clusters. Wilcoxon p-values are reported. G. RTK CBPE scores for 108 tumor samples and associated xCell signatures and pathway scores. The first heatmap shows CBPE scores of key RTKs, the second xCell signatures and the third pathway scores based on global protein abundance. H. Distribution of RTK CBPE scores for CSF1R, PDGFRB and FGFR2 stratified by immune clusters. Significance values (two-sided Wilcoxon test) between Hot clusters and combined Warm and Cold clusters are indicated on the violin plots. I. Heatmap showing proteins and phosphosites correlated with CSF1R CBPE scores that are known to be involved in immune evasion. See also Figure S6 and Table S6\\n\\nProteomic Features Related to Diagnosis, Prognosis, or Treatment A. Differentially expressed proteins between tumors and NATs. B. Significantly increased proteins (larger font indicates >4 fold) in the study LSCC cohort that are associated with poor overall survival (OS) or disease-free survival (DFS) in the TCGA LSCC cohort mRNA. C. Genetic dependencies of 502 proteins (log2 FC >2, FDR<0.01 and NAs <50%) in LSCC cell lines (n=16) profiled as part of the Achilles Dependency-Map project. D. Genes, ordered by their chromosomal location, that are deleted in at least 25% of the samples and significantly correlated to the immune score. Immune-related genes are highlighted. E. EGFR protein and tyrosine phosphorylation levels compared to EGFR copy number and an EGFR activity score (PROGENy). F. Heatmap showing Pearson correlation between the EGFR activity score represented by PROGENy (top) and RNA expression of EGFR ligands. *p<0.05 G. GO Biological Process enrichment for proteins with increased phosphorylation in EGFR amplified samples compared to non-amplified samples.\\n\\nH. Summary roadmap figure partitioned into five major categories, indicated by different colors. See also Figure S7 and Table S7 REAGENT or RESOURCE\\tSOURCE\\tIDENTIFIER\\t \\tAntibodies\\t \\tPTMScan\\u00ae Ubiquitin Remnant Motif (K-\\u03b5-GG) Kit\\tCell Signaling Technology\\tCatalog 5562\\t \\tPTMScan Acetyllysine Kit\\tCell Signaling Technology\\tCatalog: 13416\\t \\tCD4\\tDako\\tClone 4B12, RRID:AB_2728838 13416\\t \\tCD4\\tDako\\tClone 4B12, RRID:AB_2728838\\t \\tCD8\\tBio-Rad\\tClone 4B11, RRID:AB_322868\\t \\tCD163\\tAbcam\\tAb182422, RRID:AB_2753196\\t \\tARHGDIB\\tSigma-Aldrich\\tHPA051235, RRID:AB_2681398\\t \\tCK (pan-Cytokeratin)\\tDako\\tclone AE1/AE3, RRID:AB_2132885 CK (pan-Cytokeratin)\\tDako\\tclone AE1/AE3, RRID:AB_2132885\\t \\talpha-SMA\\tAbcam\\tab5694, RRID:AB_2223021\\t \\tBiological Samples\\t \\tPrimary tumor samples\\tSee Experimental Model and Subject Details\\tN/A\\t \\tChemicals and Reagents\\t \\tHPLC-grade water\\tJ.T. Baker\\tCatalog: Chemicals and Reagents\\t \\tHPLC-grade water\\tJ.T. Baker\\tCatalog: 4218-03\\t \\tUrea\\tSigma\\tCatalog: U0631\\t \\tSodium chloride\\tSigma\\tCatalog: 71376\\t \\t1M Tris, pH 8.0\\tInvitrogen\\tCatalog: AM9855G\\t \\tEthylenediaminetetraacetic acid\\tSigma\\tCatalog: E7889 AM9855G\\t \\tEthylenediaminetetraacetic acid\\tSigma\\tCatalog: E7889\\t \\tAprotinin\\tSigma\\tCatalog: A6103\\t \\tLeupeptin\\tRoche\\tCatalog: 11017101001\\t \\tPhenylmethylsulfonyl fluoride\\tSigma\\tCatalog: 78830\\t \\tSodium fluoride\\tSigma\\tCatalog: S7920\\t \\tPhosphatase inhibitor Sodium fluoride\\tSigma\\tCatalog: S7920\\t \\tPhosphatase inhibitor cocktail 2\\tSigma\\tCatalog: P5726\\t \\tPhosphatase inhibitor cocktail 3\\tSigma\\tCatalog: P0044\\t \\tDithiothretiol, No-Weigh Format\\tThermoScientific\\tCatalog: 20291\\t \\tIodoacetamide\\tSigma\\tCatalog: A3221 20291\\t \\tIodoacetamide\\tSigma\\tCatalog: A3221\\t \\tLysyl endopeptidase\\tWako Chemicals\\tCatalog: 129-02541\\t \\tSequencing-grade modified trypsin\\tPromega\\tCatalog: V511X\\t \\tFormic acid\\tSigma\\tCatalog: F0507\\t \\tAcetonitrile, LC-MS grade\\tHoneywell\\tCatalog: 34967 F0507\\t \\tAcetonitrile, LC-MS grade\\tHoneywell\\tCatalog: 34967\\t \\tAcetonitrile, anhydrous\\tSigma\\tCatalog: 271004\\t \\tTrifluoroacetic acid\\tSigma\\tCatalog: 302031\\t \\tTandem Mass Tag reagent kit \\u2013 11plex\\tThermoFisher\\tCatalog: A34808\\t \\t0.5M HEPES, pH 8.5\\tAlfa \\u2013 11plex\\tThermoFisher\\tCatalog: A34808\\t \\t0.5M HEPES, pH 8.5\\tAlfa Aesar\\tCatalog: J63218\\t \\tHydroxylamine solution, 50% (vol/vol) in H2O\\tAldrich\\tCatalog: 467804\\t \\tMethanol\\tHoneywell\\tCatalog: 34966\\t \\tAmmonium hydroxide solution, 28% (wt/vol) in 34966\\t \\tAmmonium hydroxide solution, 28% (wt/vol) in H2O\\tSigma\\tCatalog: 338818\\t \\tNi-NTA agarose beads\\tQiagen\\tCatalog: 30410\\t \\tIron (III) chloride\\tSigma\\tCatalog: 451649\\t \\tAcetic acid, glacial\\tSigma\\tCatalog: AX0073\\t \\tPotassium phosphate, acid, glacial\\tSigma\\tCatalog: AX0073\\t \\tPotassium phosphate, monobasic\\tSigma\\tCatalog: P0662\\t \\tPotassium phosphate, dibasic\\tSigma\\tCatalog: P3786\\t \\tMOPS\\tSigma\\tCatalog: M5162\\t \\tSodium hydroxide\\tVWR\\tCatalog: BDH7225\\t \\tSodium phosphate, dibasic\\tSigma\\tCatalog: BDH7225\\t \\tSodium phosphate, dibasic\\tSigma\\tCatalog: S9763\\t \\tPhosphate-buffered saline\\tFisher Scientific\\tCatalog: 10010023\\t \\tiVIEW DAB Detection Kit\\tRoche\\tCatalog: 760-091\\t \\tEquipment\\t \\tReversed-phase tC18 SepPak, 3cc 200mg\\tWaters\\tCatalog: WAT054925 tC18 SepPak, 3cc 200mg\\tWaters\\tCatalog: WAT054925\\t \\tSolid-phase C18 disk, for Stage-tips\\tEmpore\\tCatalog: 66883-U\\t \\tStage-tip needle\\tCadence\\tCatalog: 7928\\t \\tStage-tip puncher, PEEK tubing\\tIdex Health & Science\\tCatalog: 1581\\t \\tPicoFrit LC-MS column\\tNew Health & Science\\tCatalog: 1581\\t \\tPicoFrit LC-MS column\\tNew Objective\\tCatalog: PF360-75-10-N-5\\t \\tReproSil-Pur, 120 \\u212b, C18-AQ, 1.9-\\u03bcm resin\\tDr. Maisch\\tCatalog: r119.aq\\t \\tNanospray column heater\\tPhoenix S&T\\tCatalog: PST-CH-20U\\t \\tColumn heater column heater\\tPhoenix S&T\\tCatalog: PST-CH-20U\\t \\tColumn heater controller\\tPhoenix S&T\\tCatalog: PST-CHC\\t \\t300 \\u03bcL LC-MS autosampler vial and cap\\tWaters\\tCatalog: 186002639\\t \\tOffline HPLC column, 3.5-\\u03bcm particle size, 4.6 um \\u00d7 250 mm\\tAgilent\\tCatalog: Custom 3.5-\\u03bcm particle size, 4.6 um \\u00d7 250 mm\\tAgilent\\tCatalog: Custom order\\t \\tOffline 96-well fractionation plate\\tWhatman\\tCatalog: 77015200\\t \\t700 \\u03bcL bRP fractionation autosampler vial\\tThermoFisher\\tCatalog: C4010-14\\t \\t700 \\u03bcL bRP fractionation autosampler C4010-14\\t \\t700 \\u03bcL bRP fractionation autosampler cap\\tThermoFisher\\tCatalog: C4010-55A\\t \\t96-well microplate for BCA\\tGreiner\\tCatalog: 655101\\t \\tMicroplate foil cover\\tCorning\\tCatalog: PCR-AS-200\\t \\tVacuum centrifuge\\tThermoFisher\\tCatalog: SPD121P-115 Vacuum centrifuge\\tThermoFisher\\tCatalog: SPD121P-115\\t \\tCentrifuge\\tEppendorf\\tCatalog: 5427 R\\t \\tBenchtop mini centrifuge\\tCorning\\tCatalog: 6765\\t \\tBenchtop vortex\\tScientific Industries\\tCatalog: SI-0236\\t \\tIncubating shaker\\tVWR\\tCatalog: 12620-942\\t \\t15 mL SI-0236\\t \\tIncubating shaker\\tVWR\\tCatalog: 12620-942\\t \\t15 mL centrifuge tube\\tCorning\\tCatalog: 352097\\t \\t50 mL centrifuge tube\\tCorning\\tCatalog: 352070\\t \\t1.5 mL microtube w/o cap\\tSarstedt\\tCatalog: 72.607\\t \\t2.0 mL microtube w/o cap\\tSarstedt\\tCatalog: 72.608 72.607\\t \\t2.0 mL microtube w/o cap\\tSarstedt\\tCatalog: 72.608\\t \\tMicrotube caps\\tSarstedt\\tCatalog: 72.692\\t \\t1.5 mL snapcap tube\\tThermoFisher\\tCatalog: AM12450\\t \\t2.0 mL snapcap tube\\tThermoFisher\\tCatalog: AM12475\\t \\tInstrumentation\\t \\tMicroplate Reader\\tMolecular AM12475\\t \\tInstrumentation\\t \\tMicroplate Reader\\tMolecular Devices\\tCatalog: M2\\t \\tOffline HPLC System for bRP fractionation\\tAgilent 1260\\tCatalog: G1380-90000\\t \\tOnline LC for LC-MS\\tThermoFisher\\tCatalog: LC140\\t \\tQ Exactive Plus Mass LC for LC-MS\\tThermoFisher\\tCatalog: LC140\\t \\tQ Exactive Plus Mass Spectrometer\\tThermoFisher\\tCatalog: IQLAAEGAAPFALGMBDK\\t \\tQ Exactive HF-X Mass Spectrometer\\tThermoFisher\\tCatalog: 0726042\\t \\tOrbitrap Fusion Lumos Tribrid Mass Spectrometer\\tThermoFisher\\tCatalog: Fusion Lumos Tribrid Mass Spectrometer\\tThermoFisher\\tCatalog: IQLAAEGAAPFADBMBHQ\\t \\tCritical Commercial Assays\\t \\tTruSeq Stranded Total RNA Library Prep Kit with Ribo-Zero Gold\\tIllumina\\tCatalog: RS-122-2301\\t \\tInfinium MethylationEPIC Kit\\tIllumina\\tCatalog: RS-122-2301\\t \\tInfinium MethylationEPIC Kit\\tIllumina\\tCatalog: WG-317-1003\\t \\tNextera DNA Exosome Kit\\tIllumina\\tCatalog: 20020617\\t \\tKAPA Hyper Prep Kit, PCR-free\\tRoche\\tCatalog: 07962371001\\t \\tBCA Protein Assay Kit\\tThermoFisher\\tCatalog: 23225\\t \\tDeposited Data Protein Assay Kit\\tThermoFisher\\tCatalog: 23225\\t \\tDeposited Data\\t \\tDepMap: Mutation\\tDepMapPublic 21Q1 (PMID: 31068700; Dataset doi:10.6084/m9.figshare.13681534.v1)\\thttps://depmap.org/portal/download/\\t \\tDepMap: Segmented copy number\\tDepMapPublic 21Q1 (PMID: DepMap: Segmented copy number\\tDepMapPublic 21Q1 (PMID: 31068700; Dataset doi:10.6084/m9.figshare.13681534.v1)\\thttps://depmap.org/portal/download/\\t \\tDepMap: Gene level copy number\\tDepMapPublic 21Q1 (PMID: 31068700; Dataset level copy number\\tDepMapPublic 21Q1 (PMID: 31068700; Dataset doi:10.6084/m9.figshare.13681534.v1)\\thttps://depmap.org/portal/download/\\t \\tDepMap: RNAseq (transcript isoform)\\tDepMapPublic 21Q1 (PMID: 31068700; Dataset (transcript isoform)\\tDepMapPublic 21Q1 (PMID: 31068700; Dataset doi:10.6084/m9.figshare.13681534.v1)\\thttps://depmap.org/portal/download/\\t \\tDepMap: Proteomics (RPPA)\\tCCLE 2019 (PMID: 31068700)\\thttps://depmap.org/portal/download/\\t \\tDepMap: CRISPR KO screen DepMap: CRISPR KO screen (combined)\\tDepMapPublic 21Q1 (bioRxiv 2020.05.22.110247; doi: https://doi.org/10.1101/2020.05.22.110247)\\thttps://depmap.org/portal/download/\\t \\tDepMap: shRNA screen (combined)\\tDEMETER2 Data v6 (PMID: DepMap: shRNA screen (combined)\\tDEMETER2 Data v6 (PMID: 30389920)\\thttps://depmap.org/portal/download/\\t \\tDepMap: GDSC drug screen\\tSanger GDSC 1 (PMID: 27397505)\\thttps://depmap.org/portal/download/\\t \\tDepMap: PRISM drug screen\\tPRISM Repurposing 19Q4 DepMap: PRISM drug screen\\tPRISM Repurposing 19Q4 Secondary Screen (PMID: 32613204)\\thttps://depmap.org/portal/download/\\t \\tPhosphoSitePlus\\t\\thttps://www.phosphosite.org\\t \\tConnectivity Map (CMAP)\\t\\thttps://www.broadinstitute.org/connectivity-map-cmap\\t \\tHuman Human Protein Atlas (HPA)\\t\\thttps://www.proteinatlas.org\\t \\tCT Antigen database\\t\\thttp://www.cta.lncc.br\\t \\tDependency map (DepMap)\\t\\thttps://depmap.org/portal/v3.3.8 is a GECKOv2 Achilles dataset\\t \\tLibrary of Integrated Network-based Cellular Signatures Library of Integrated Network-based Cellular Signatures (LINCS)\\t\\thttps://clue.io/dataExpanded CMap LINCS Resource 2020 (1/28/2021 update)\\t \\tCPTAC HNSCC cohort\\t\\thttps://cptac-data-portal.georgetown.edu/study-summary/S054\\t \\tCPTAC LSCC cohort\\tThis CPTAC LSCC cohort\\tThis study\\thttps://cptac-data-portal.georgetown.edu/study-summary/S063\\t \\tSoftware and Algorithms\\t \\tmethylationArrayAnalysis (version 3.9)\\t\\thttps://master.bioconductor.org/packages/release/workflows/html/methylationArrayAnalysis.html Illumina EPIC methylation array (3.9)\\tHansen KD, 2019\\thttps://bioconductor.org/packages/release/data/annotation/html/IlluminaHumanMethylationEPICanno.ilm10b2.hg19.html\\t \\tMethylation array analysis pipeline for CPTAC\\tLi Ding Methylation array analysis pipeline for CPTAC\\tLi Ding Lab\\thttps://github.com/ding-lab/cptac_methylation\\t \\tmiRNA-Seq analysis pipeline for CPTAC\\tLi Ding Lab\\thttps://github.com/ding-lab/CPTAC_miRNA\\t \\tVEP\\t\\thttps://github.com/Ensembl/ensembl-vep/tags VEP\\t\\thttps://github.com/Ensembl/ensembl-vep/tags\\t \\tTNScope / DNAScope (Sentieon)\\t\\tsentieon.com\\t \\tvcfAnno\\t\\thttps://github.com/brentp/vcfanno\\t \\tVariantAnnotation (Bioconductor)\\t\\thttps://bioconductor.org/packages/release/bioc/html/VariantAnnotation.html arriba_v1.1.0\\t\\thttps://github.com/suhrig/arriba/\\t \\tfusioncatcher_v1.10\\t\\thttps://github.com/ndaniel/fusioncatcher/blob/master/doc/manual.md\\t \\teQTLGen\\t\\thttps://github.com/molgenis/systemsgenetics/wiki/eQTL-mapping-analysis-cookbook-(eQTLGen) Pindel0.2.5\\t\\thttp://gmt.genome.wustl.edu/packages/pindel/\\t \\tSignatureAnalyzer\\t\\thttps://software.broadinstitute.org/cancer/cga/msp\\t \\tCNVEX\\tMarcin Cieslik Lab\\thttps://github.com/mctp/cnvex\\t \\tCRISP\\tMarcin Cieslik Lab\\thttps://github.com/mctp/cnvex\\t \\tCRISP\\tMarcin Cieslik Lab\\thttps://github.com/mcieslik-mctp/crisp-build\\t \\tSpectrum Mill\\tKarl R. Clauser, Steven Carr Lab\\thttps://proteomics.broadinstitute.org/\\t \\tComBat Steven Carr Lab\\thttps://proteomics.broadinstitute.org/\\t \\tComBat (v3.20.0)\\t\\thttps://bioconductor.org/packages/release/bioc/html/sva.html\\t \\tgPCA\\t\\thttps://cran.r-project.org/web/packages/gPCA/index.html gPCA\\t\\thttps://cran.r-project.org/web/packages/gPCA/index.html\\t \\tGISTIC2.0\\t\\tftp://ftp.broadinstitute.org/pub/GISTIC2.0/GISTIC_2_0_23.tar.gz\\t \\tiProFun\\t\\thttps://github.com/WangLab-MSSM/iProFun iProFun\\t\\thttps://github.com/WangLab-MSSM/iProFun\\t \\tESTIMATE\\t\\thttps://bioinformatics.mdanderson.org/public-software/estimate/\\t \\tWebGestaltR\\t\\thttp://www.webgestalt.org/\\t \\tGSVA\\t\\thttps://bioconductor.org/packages/release/bioc/html/GSVA.html TSNet\\t\\thttps://github.com/WangLab-MSSM/TSNet\\t \\txCell\\t\\thttp://xcell.ucsf.edu/\\t \\tCPTAC LSCC Data Viewer\\tSteven Carr lab\\thttps://rstudio-connect.broadapps.org/CPTAC-LSCC2021/ Carr lab\\thttps://rstudio-connect.broadapps.org/CPTAC-LSCC2021/\\t \\tConsensusClusterPlus\\t\\thttp://bioconductor.org/packages/release/bioc/html/CancerSubtypes.html\\t \\tMS-GF+\\t\\thttps://github.com/MSGFPlus/msgfplus\\t \\tNeoFlow\\t\\thttps://github.com/bzhanglab/neoflow NeoFlow\\t\\thttps://github.com/bzhanglab/neoflow\\t \\tnetMHCpan\\t\\thttp://www.cbs.dtu.dk/services/NetMHCpan/\\t \\tOptitype\\t\\thttps://github.com/FRED-2/OptiType\\t \\tCustomprodbj\\t\\thttps://github.com/bzhanglab/customprodbj\\t \\tPDV\\t\\thttps://github.com/wenbostar/PDV PDV\\t\\thttps://github.com/wenbostar/PDV\\t \\tPepQuery\\t\\thttp://pepquery.org\\t \\tPTM-SEA\\t)\\thttps://github.com/broadinstitute/ssGSEA2.0\\t \\tPTMsigDB\\t)\\thttp://prot-shiny-vm.broadinstitute.org:3838/ptmsigdb-app/\\t \\tTerra\\tBroad Institute data science Terra\\tBroad Institute data science platform.\\thttps://terra.bio/\\t \\tPanoply\\tBroad Institute Proteomics Platform\\thttps://github.com/broadinstitute/PANOPLY\\t \\tCMap\\t\\thttps://clue.io/cmap\\t \\tLIMMA v3.36 (R CMap\\t\\thttps://clue.io/cmap\\t \\tLIMMA v3.36 (R Package)\\t\\thttps://bioconductor.org/packages/release/bioc/html/limma.html\\t \\tFactoMineR v1.41NMF(R - package)\\t\\thttps://cran.r-project.org/web/packages/FactoMineR/index.html\\t \\tMClust v5.4 (R package)\\t(Scrucca, MClust v5.4 (R package)\\t(Scrucca, Fop, Murphy and Raftery, 2017)\\thttps://cran.r-project.org/web/packages/mclust/index.html\\t \\tg:Profiler\\t\\thttps://biit.cs.ut.ee/gprofiler/gost\\t \\tCytoscape\\t\\thttps://cytoscape.org/ Cytoscape\\t\\thttps://cytoscape.org/\\t \\tImmuneSubtypeClassifier\\t\\thttps://github.com/CRI-iAtlas/ImmuneSubtypeClassifier\\t \\tProteinPaint\\t\\thttps://pecan.stjude.doud/proteinpaint/\\t \\tOrdinal\\tChristensen RHB (2019)\\thttps://CRAN.R-project.org/package=ordinal\",\n",
            "                    \"pdc_study_id\": \"PDC000233\",\n",
            "                    \"title\": \"A proteogenomic portrait of lung squamous cell carcinoma\"\n",
            "                },\n",
            "                {\n",
            "                    \"full_text\": \"A proteogenomic portrait of lung squamous cell carcinoma\\n\\nSummary Lung squamous cell carcinoma (LSCC) remains a leading cause of cancer death with few therapeutic options. We characterized the proteogenomic landscape of LSCC, providing a deeper exposition of LSCC biology with potential therapeutic implications. We of LSCC biology with potential therapeutic implications. We identify NSD3 as an alternative driver in FGFR1-amplified tumors and low-p63 tumors overexpressing the therapeutic target survivin. SOX2 is considered undruggable, but our analyses provide SOX2 is considered undruggable, but our analyses provide rationale for exploring chromatin modifiers such as LSD1 and EZH2 to target SOX2-overexpressing tumors. Our data support complex regulation of metabolic pathways by crosstalk between complex regulation of metabolic pathways by crosstalk between post-translational modifications including ubiquitylation. Numerous immune-related proteogenomic observations suggest directions for further investigation. Proteogenomic dissection of CDKN2A for further investigation. Proteogenomic dissection of CDKN2A mutations argue for more nuanced assessment of RB1 protein expression and phosphorylation before declaring CDK4/6 inhibition unsuccessful. Finally, triangulation between LSCC, LUAD and HNSCC Finally, triangulation between LSCC, LUAD and HNSCC identified both unique and common therapeutic vulnerabilities. These observations and proteogenomics data resources may guide research into the biology and treatment of LSCC. In Brief Comprehensive proteogenomic characterization of lung squamous cell carcinomas and paired normal adjacent tissues identifies taxonomic subclasses, alternative driver events and insights into immune modulation, as well as putative biomarkers and potential immune modulation, as well as putative biomarkers and potential therapeutic targets. Graphical Abstract\\n\\nIntroduction Lung cancer is the leading cause of cancer-associated mortality. Unlike lung adenocarcinomas (LUAD), patients with lung squamous cell carcinoma (LSCC) have not benefited from targeted therapies. Potentially druggable genetic events in three primary Potentially druggable genetic events in three primary pathways (FGFR1, PI3K, or G1/S checkpoint genes such as CDKN2A) are found in upwards of 60% but targeting these clinically has largely failed. While other therapeutic regimens are being explored as failed. While other therapeutic regimens are being explored as part of the LungMAP consortium, to date only immunotherapy has evolved into a successful therapeutic strategy for patients with LSCC. Given the lack of established actionable genomic targets, LSCC. Given the lack of established actionable genomic targets, a comprehensive characterization of the proteogenomic landscape of LSCC, including the associated post-translational modifications (PTMs) that are key to protein activity and signaling, is a (PTMs) that are key to protein activity and signaling, is a crucial step forward. Such studies will aid in connecting genomic aberrations to molecular and clinical phenotypes and in subsequently identifying therapeutic vulnerabilities and effective, identifying therapeutic vulnerabilities and effective, biomarker-based patient stratification. We characterized 108 prospectively-collected, treatment-na\\u00efve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs), identifying actionable therapeutic protein targets and elucidating cellular signaling pathways and crosstalk between multiple cellular signaling pathways and crosstalk between multiple PTMs. The dataset provides an exceptional resource to guide further research and support development of therapeutic modalities in LSCC. Results\\n\\nProteogenomic landscape of LSCC We performed deep-scale molecular analysis spanning nine different data types on LSCC tumors and NATs prospectively collected from 108 patients (Figure 1A, S1A, Table S1\\u20133). Proteomics data were generated using TMT11-based multiplexing linked by a common generated using TMT11-based multiplexing linked by a common reference (Figure S1A) and demonstrated high technical quality (Figure S1B\\u2013C). The cohort demographics are summarized in Figure 1B. Most self-reported never-smokers displayed high genomic smoking Most self-reported never-smokers displayed high genomic smoking scores (Figure S1D), highlighting occasional discrepancy between self-reported smoking status and genomic evidence, as previously reported. The genomic landscape reaffirmed previously reported. The genomic landscape reaffirmed previously reported somatic alterations in LSCC (Figure 1C). Principal component analysis (PCA) of global proteomic and PTM data showed clear separation of tumors and NATs (Figure S1E). Proteogenomic data helped annotate the impact of copy number alteration (CNA) events (Figure 1D,E). Of 5,523 significant (FDR<0.01) cis CNA-mRNA events, 2,154 were concordantly correlated with protein expression, including 138 \\u201ccancer associated genes\\u201d protein expression, including 138 \\u201ccancer associated genes\\u201d (CAG) (Figure 1E, Table S1). We identified six amplified (including WHSC1L1, CCND1, and SOX2) and 29 deleted (including NCOR1, SETD2, and CBL) CAGs in significant focal events (Q<0.25) (Figure and CBL) CAGs in significant focal events (Q<0.25) (Figure 1E, Table S4). Intriguingly, WHSC1L1 (NSD3) is part of a recurrent focal amplicon (8p11.23) in LSCC that encompasses FGFR1. Therapies targeted against FGFR1 have been unsuccessful. Proteomics data targeted against FGFR1 have been unsuccessful. Proteomics data suggested that NSD3, rather than FGFR1, could be the critical driver oncogene within this amplicon (Figure 1F), nominating it as a potential therapeutic target. While this paper was under as a potential therapeutic target. While this paper was under review, NSD3 was demonstrated to be a key regulator of LSCC tumorigenesis, mediating oncogenic chromatin changes. To investigate the impact of CNAs on noncognate gene products, we matched patterns of these significant trans-effects (vertical stripes in Figure 1D) to perturbation profiles from the Connectivity Map (CMap) (https://clue.io/cmap). Six CNAs (IKBKAP, PIN1, Map (CMap) (https://clue.io/cmap). Six CNAs (IKBKAP, PIN1, DNAJB1, IL18, NR2F6, AKAP) (Table S4) were enriched for both amplification and deletion (FDR=0.06) and associated (Fisher\\u2019s exact p<0.01) with clinical metadata. IL-18 deletion and amplification with clinical metadata. IL-18 deletion and amplification correlated with decreased and increased IL-18 protein expression and xCell immune score, respectively, (Figure S1F (upper panel)), consistent with its playing a key modulatory role in the tumor consistent with its playing a key modulatory role in the tumor microenvironment of NSCLC. NR2F6 amplification and deletion were correlated with NR2F6 protein expression and anticorrelated with xCell immune score (Figure S1F (lower panel)). NR2F6 acts as a xCell immune score (Figure S1F (lower panel)). NR2F6 acts as a non-redundant immune checkpoint in cancer, and even partial knockdown works synergistically with PD-L1 blockade. LSCC tumor DNA showed overall hypermethylation relative to NATs (Figure S1G), with tumors separating into CIMP (CpG island methylator phenotype) clusters (Figure S1H). Multi-omic data allowed the identification of \\u201ccascading\\u201d promoter methylation allowed the identification of \\u201ccascading\\u201d promoter methylation cis-effects across cognate mRNA, protein and PTM abundances, supporting their functional significance. Of 90 genes that showed such significant (FDR<0.1) cis-effects in tumors (Table S4), 20 such significant (FDR<0.1) cis-effects in tumors (Table S4), 20 were also differentially expressed (FDR<0.01) between tumors and NATs (Figure 1G). While most were newly associated with LSCC, methylation-driven expression of FAM110A, PTGES3, PLAU and methylation-driven expression of FAM110A, PTGES3, PLAU and SLC16A3 has been implicated in lung and other cancers. Multi-omic clustering identifies five LSCC molecular subtypes, including one that is EMT-Enriched We performed non-negative matrix factorization (NMF)-based single- and multi-omic unsupervised clustering on CNA, RNA, protein, phosphoprotein and acetylprotein datasets from 108 tumors, excluding ubiquitylprotein data as it was not available for the excluding ubiquitylprotein data as it was not available for the entire cohort (Figure 1A). The five resulting multi-omic subtypes (Figure 2A\\u2013B, Figure S2A) were named based on their predominant pathway associations and similarities to previously defined pathway associations and similarities to previously defined RNA clusters. Significant associations between NMF subtypes and cohort metadata are presented in Table S1. Two NMF subtypes, \\u201cBasal-Inclusive\\u201d (B-I) and \\u201cEpithelial to Mesenchymal \\u201cBasal-Inclusive\\u201d (B-I) and \\u201cEpithelial to Mesenchymal Transition-enriched\\u201d (EMT-E), emerged from partitioning of samples that showed similarity to the TCGA-derived RNA Basal cluster, although only B-I retained significant association after Bonferroni only B-I retained significant association after Bonferroni correction (Fisher\\u2019s p<0.01, Table S1). B-I tumors showed basaloid histology and upregulated metabolic, immune and estrogen receptor signaling (Figure 2B). EMT-E showed upregulation of EMT, signaling (Figure 2B). EMT-E showed upregulation of EMT, Angiogenesis and Myogenesis (Figure 2B), with myxoid histologic features and fibroblast infiltration. A third subtype characterized by mutations in KEAP1, CUL3 and NFE2L2 genes and high-level by mutations in KEAP1, CUL3 and NFE2L2 genes and high-level amplification (GISTIC threshold value = 2) of SOX2 and TP63 (Figure 2A\\u2013B, Table S2) was labeled the \\u201cClassical\\u201d subtype, consistent with TCGA nomenclature and with a previous publication. These with TCGA nomenclature and with a previous publication. These showed classical histologic features, CIMP-high enrichment, upregulation of OxPhos- and proliferation-related pathways, and downregulation of immune signaling (Figure 2A\\u2013B). A fourth NMF downregulation of immune signaling (Figure 2A\\u2013B). A fourth NMF subtype was designated \\u201cInflamed-Secretory\\u201d (I-S) due to its alignment with the RNA-based Secretory cluster and strong upregulation of immune-related pathways. The fifth, and strong upregulation of immune-related pathways. The fifth, \\u201cProliferative-Primitive\\u201d (P-P) NMF subtype displayed upregulated proliferation-related pathways, downregulated immune signaling, and enrichment of CIMP-low samples. Within each of these five and enrichment of CIMP-low samples. Within each of these five NMF clusters were samples with low NMF cluster membership scores, which could be considered as belonging to a \\u201cmixed\\u201d subgroup, since they showed features of more than one subtype. Mixed since they showed features of more than one subtype. Mixed subgroup membership, a possible readout of tumor heterogeneity, conferred significantly worse survival (Figure 2C), an observation also made in a recent proteogenomic analysis of glioblastoma. The made in a recent proteogenomic analysis of glioblastoma. The mixed class was associated in the current cohort with increased frequency of SOX2 amplifications (p = 0.0038) but not grade, mutation burden, stemness or other tested variables. We compared B-I (Figure S2B) and EMT-E (Figure S2C) to other subtypes to identify distinguishing features. B-I had hyperphosphorylated M phase Histone 1 T11 and elevated expression of both TACSTD2 (TROP2), a cell-surface glycoprotein that drives growth TACSTD2 (TROP2), a cell-surface glycoprotein that drives growth and metastasis, and MARK2, a kinase that regulates cell polarity and leads to cisplatin resistance. Transcription activator PBX3, which is upregulated only at the protein level (Figure S2B), which is upregulated only at the protein level (Figure S2B), promotes invasion, proliferation, and chemoresistance and is associated with poor prognosis, including in laryngeal squamous cell carcinoma. Pathway analysis of proteins highly correlated with carcinoma. Pathway analysis of proteins highly correlated with PBX3 highlighted neutrophil activation and degranulation (Figure S2D), consistent with the significant upregulation of neutrophilic granule proteins and chemotactic factors (Figure S2E) and granule proteins and chemotactic factors (Figure S2E) and with the xCell-based deconvolution signature (Figure S2F). This correlation was not observed in paired NATs, suggesting a tumor-specific role. In addition to a strong EMT signature (Figure 2B), the EMT-E subtype showed upregulated VCAN (Figure S2C), a tumor-promoting target of SNAIL expressed in cancer-associated fibroblasts (CAFs), and FHL3, a protein that stabilizes EMT-associated (CAFs), and FHL3, a protein that stabilizes EMT-associated transcription factors (Figure S2C). DVL3, FN1 and FHL2, all Wnt pathway regulators, were upregulated in the proteomics dataset, and cell-cell signaling by Wnt was one of the significantly and cell-cell signaling by Wnt was one of the significantly (FDR=0.04) enriched pathways in this subtype. As shown in Figure 2D\\u2013E, coordinated high expression of EMT and CAF proteins, xCell fibroblast scores, and a \\u201cfibroblast proliferation\\u201d signature fibroblast scores, and a \\u201cfibroblast proliferation\\u201d signature suggested that CAFs and tumor epithelium might collaborate in the EMT-E phenotype. This was further supported by enrichment of EMT-related receptor tyrosine kinases (RTKs), as described below. receptor tyrosine kinases (RTKs), as described below. Immunohistochemical co-staining of a small subset of tumors for the epithelial marker pan-cytokeratin (CK) and CAF marker \\u03b1-smooth muscle actin (\\u03b1-SMA) demonstrated the presence in EMT-E tumors of both actin (\\u03b1-SMA) demonstrated the presence in EMT-E tumors of both CAFs and tumor cells undergoing EMT (Figure S2G), with significant enrichment of co-stained cells in the EMT-E subtype (Wilcoxon p=0.025), but not of cells showing single staining of either p=0.025), but not of cells showing single staining of either CK or \\u03b1-SMA alone. TGF\\u03b2 is a well-established and potent inducer of EMT, and a Library of Integrated Network-Based Cellular Signatures-based (LINCS) query for compounds that reversed the EMT-E (LINCS) query for compounds that reversed the EMT-E signature showed enrichment for TGF\\u03b2 inhibitors (Figure S2H, Table S4H). Integrating our global proteome data with a prior LSCC dataset resulted in six clusters largely recapitulating RNA, proteomic and multi-omic clusters defined in the current study (Figure S2I\\u2013J). NMF EMT-E subtype tumors show phosphorylation-driven PDGFR and ROR2 signaling. Though LSCC lacks the RTK mutations for which targeted inhibition has dramatically improved therapeutic options in LUAD, other modes of RTK activation, inferred from phosphoproteomic data, may nonetheless nominate targeted inhibition for some LSCC may nonetheless nominate targeted inhibition for some LSCC patients. Hence, we derived a serine/threonine-predominant correlation-based phosphosite enrichment score (CBPE score) for all RTKs in our tumor cohort. Of nine RTKs with high CBPE scores in LSCC in our tumor cohort. Of nine RTKs with high CBPE scores in LSCC tumors, seven were significantly associated with NMF subtypes (Kruskal-Wallis p <0.01). Scores for PDGFRB and ROR2, markedly elevated in EMT-E (Figure 2F), were highly correlated with the elevated in EMT-E (Figure 2F), were highly correlated with the loss of epithelial cell-cell junction markers and upregulation of mesenchymal proteins involved in remodeling of extracellular matrix, induction of EMT, and promotion of cell migration, which induction of EMT, and promotion of cell migration, which mark the transition from epithelial to mesenchymal state (Figure 2G). As summarized in Figure S2K, we found phosphosite-based evidence for upregulation of both the planar cell polarity and evidence for upregulation of both the planar cell polarity and calcium-dependent branches of non-canonical Wnt signaling, initiated through ROR2 or PDGFR, both targetable and recently implicated in the progression from a hybrid E/M to a highly mesenchymal in the progression from a hybrid E/M to a highly mesenchymal state. Further functional characterization is warranted to confirm the contribution of these RTKs to the EMT-E phenotype and to test the impact of their modulation. Loss of CDK4/6 pathway inhibitors is a universal feature of LSCC but Rb1 expression is variable We investigated the impact of recurrent mutations on cognate RNA, proteins and PTMs (cis-effects) and on a set of cancer associated genes (trans-effects). Significant (FDR<0.05) Cis/Trans pairs are shown in Figure 3A\\u2013D. RB1 mutated tumors had upregulated are shown in Figure 3A\\u2013D. RB1 mutated tumors had upregulated cell cycle-related protein expression. NOTCH1 mutation led to elevated GSK3B protein and phosphorylation (T433), downregulation of inhibitory sites on GSK3A (S21) and upregulation of NLK. NLK of inhibitory sites on GSK3A (S21) and upregulation of NLK. NLK and GSK3B inhibit NOTCH1, recently described as a tumor suppressor in LSCC, suggesting that both might inhibit NOTCH1 downstream signaling in tumors with NOTCH1 mutations. ARID1A is one of signaling in tumors with NOTCH1 mutations. ARID1A is one of the most frequently mutated SWItch Sucrose Non-Fermentable (SWI/SNF) ATP-dependent chromatin remodeling complex members. Mutations were associated with higher tumor mutational burden (Figure S3A) associated with higher tumor mutational burden (Figure S3A) and led to protein downregulation, associated with worse overall survival in NSCLC. Paradoxically, mutations in CDKN2A, proposed as a potential biomarker for CDK4/6 inhibitors, resulted in increased RNA expression (Figure 3A). We analyzed the effects of genetic and epigenetic alterations of CDKN2A on both its major isoforms, p16INK4a alterations of CDKN2A on both its major isoforms, p16INK4a (p16) and p14ARF (p14) (Table S4). As expected, its homozygous deletion resulted in the loss of expression of both isoforms (Figure 3E). However, only eight of the 58 CDKN2A wild-type (WT) samples However, only eight of the 58 CDKN2A wild-type (WT) samples showed p16 RNA expression levels that were comparable to samples with CDKN2A/p16 missense mutations (log2(FPKM+1)>2). Many of those CDKN2A/p16 mutations were unlikely to result in Many of those CDKN2A/p16 mutations were unlikely to result in nonsense-mediated decay (Figure S3B), accounting for preserved expression in those samples. Notably, 28 WT tumors showed hypermethylation of the p16 but not the p14 promoter and suppression of of the p16 but not the p14 promoter and suppression of RNA and protein expression for CDKN2A/p16, while 18 additional WT tumors had reduced expression despite having no clear genetic or epigenetic alteration. Furthermore, the WT samples with high or epigenetic alteration. Furthermore, the WT samples with high CDKN2A/p16 expression had RB1 mutations or deletions and/or low Rb protein levels, consistent with a recently published pan-cancer analysis of the CDK4/6 pathway that showed mutual pan-cancer analysis of the CDK4/6 pathway that showed mutual exclusivity between mutations in these two genes. This suggests that loss of one of these two key CDK4/6 pathway inhibitors is a universal feature of LSCC. Conversely, while amplification of a universal feature of LSCC. Conversely, while amplification of Cyclin D - CDK4/6 complex genes is frequent in LSCC, there was no association between the amplification of CCND2, CCND3, CDK4, or CDK6 and either CDKN2A/p16 protein or Rb CCND2, CCND3, CDK4, or CDK6 and either CDKN2A/p16 protein or Rb protein/phosphoprotein levels (Table S4). CCND1 amplification resulted in significantly higher mean levels of Rb protein and phosphoprotein compared to WT samples, though the distributions phosphoprotein compared to WT samples, though the distributions were overlapping (Figure S3C). Intriguingly, phospho-Rb levels, which reflect CDK4/6 activity, correlated with response to CDK4/6 inhibitors in LSCC cell lines despite the heterogeneous CDK4/6 inhibitors in LSCC cell lines despite the heterogeneous distribution in response for cells with mutations in CDKN2A and copy number alterations in CCND1 and CDKN2A (Figure S3D). LSCC trial data have generally shown minimal efficacy of CDK4/6 LSCC trial data have generally shown minimal efficacy of CDK4/6 inhibition, but outlier responses have been present in trials with abemaciclib and palbociclib. While we found that CCND1 amplification resulted in higher average levels of phosphorylated Rb, resulted in higher average levels of phosphorylated Rb, the heterogeneity of Rb expression and phosphorylation in samples with CCND1 amplification provides a potential explanation for some patients\\u2019 lack of response to CDK4/6 inhibitor therapies. Improved lack of response to CDK4/6 inhibitor therapies. Improved prediction based on a downstream functional assessment of G1/S checkpoint alterations (i.e. RB1 protein expression and phosphorylation) may identify a subset of tumors sensitive to CDK4/6 may identify a subset of tumors sensitive to CDK4/6 inhibition. NRF2 pathway activation in tumors with and without NRF2 pathway mutations LSCC showed mutations of three key genes in the NRF2 antioxidant response pathway, NFE2L2 (NRF2), CUL3 and KEAP1. NFE2L2 phosphorylation was observed in samples with any of these mutations (Figure S3E). LSCC tumors were annotated as \\u201cNRF2 pathway mutated\\u201d S3E). LSCC tumors were annotated as \\u201cNRF2 pathway mutated\\u201d (n=33; with either one hit [n=28] or two hits [n=5] in NFE2L2, KEAP1, and CUL3), or \\u201cNRF2 WT\\u201d (n=68). Compared to WT, NRF2 pathway mutated tumors showed highly concordant differential expression mutated tumors showed highly concordant differential expression of mRNA, protein and phosphoprotein levels of NRF2 pathway genes (Figure 3F, Figure S3F\\u2013G, Table S4), some with therapeutic implications. A subset of NRF2 gene products showed a dosage effect A subset of NRF2 gene products showed a dosage effect (SQSTM1, NR0B1, AKR1B10, CARD11, FFAR2) with the five cases harboring two hits (Table S4) demonstrating increased up-/down-regulation of NRF2 pathway genes relative to one-hit cases (Figure S3H). Using pathway genes relative to one-hit cases (Figure S3H). Using the proteogenomic signature (Table S4) that defined NFE2L2 mutated tumors, we derived an ssGSEA-based NRF2 pathway score that was enriched not only in NRF2 pathway mutated tumors, but also in enriched not only in NRF2 pathway mutated tumors, but also in samples without NRF2 pathway mutations, especially in the NMF Classical subtype (Figure 3G). These samples showed increased NFE2L2 phosphorylation, indicative of NRF2 pathway activation. NFE2L2 phosphorylation, indicative of NRF2 pathway activation. Although the kinase(s) responsible for phosphorylation of NRF2 in LSCC are unclear, CDK5 protein (Figure 3H) and PTM-SEA-derived activity (Figure 3I) were significantly upregulated (FDR<0.05) activity (Figure 3I) were significantly upregulated (FDR<0.05) in the Classical subtype. CDK5 phosphorylation of NFE2L2 S433 mediates its activation in astrocytes and can likely play a role in CDK5-mediated NRF2 activation in LSCC tumors. Intriguingly, in CDK5-mediated NRF2 activation in LSCC tumors. Intriguingly, unlike LUAD, KEAP1 mutations did not result in significantly reduced protein expression in LSCC, although missense mutations were mostly nonoverlapping, suggesting heterogeneity in NRF2 were mostly nonoverlapping, suggesting heterogeneity in NRF2 pathway dysregulation in NSCLC subtypes. Proteogenomic analysis of chromosome 3 prioritizes therapeutic targets in LSCC Chromosome 3q, which harbors key squamous differentiation markers SOX2 and TP63, showed the most dramatic arm-level amplification in this cohort (Figure S4A). To assess potential drivers in the 3q amplicon, we identified 3q genes for which CNA correlated 3q amplicon, we identified 3q genes for which CNA correlated with RNA and protein and expression differed between tumors and NATs. TP63 showed the highest elevation in tumors (Figure 4A). LSCC cell lines amplified for TP63 and SOX2 were highly dependent cell lines amplified for TP63 and SOX2 were highly dependent upon them, supporting oncogene addiction (Figure S4B). As previously reported, \\u0394Np63\\u03b1 was the dominant TP63 isoform in this dataset (Figure S4C) and was highly correlated with TP63 expression (Figure S4D). Its amplification was associated with improved survival in the TCGA dataset (Figure S4E). In contrast to survival in the TCGA dataset (Figure S4E). In contrast to the general elevation of TP63 copy number, transcript, and protein abundance in LSCC tumors compared to NATs, 10 \\u201cNp63-low\\u201d samples histologically confirmed to be LSCC showed no elevation at RNA or confirmed to be LSCC showed no elevation at RNA or protein levels (Figure 4B). The apoptosis inhibitor survivin (BIRC5) was among the most overexpressed proteins in this Np63-low group (Figure 4C). Notably, a substantial number of LSCC cell lines also had 4C). Notably, a substantial number of LSCC cell lines also had low Np63 expression (Figure S4F), and those TP63-low cell lines were significantly more vulnerable to the survivin inhibitor YM-155 (Figure S4G). Survivin promotes cell proliferation and G2M (Figure S4G). Survivin promotes cell proliferation and G2M checkpoints, evident in Np63-low samples (Figure S4H). Small molecule inhibitors and inhibitory peptides targeting survivin are being explored as part of ongoing clinical trials including in being explored as part of ongoing clinical trials including in NSCLC, but have shown modest or no improvement. Our data on the newly identified Np63-low LSCC tumors suggests that TP63 status may identify patients with potential response to survivin may identify patients with potential response to survivin inhibition. Overexpression of miR-205, a biomarker to distinguish LSCC from other NSCLC (Figure S4I), was associated with decreased promoter DNA methylation (Figure S4J) and elevation of \\u0394Np63\\u03b1, a transcriptional regulator of this miRNA. miR-205 showed high a transcriptional regulator of this miRNA. miR-205 showed high correlation with TP63 expression and its expression was markedly reduced in Np63-low samples (Figure 4D, Figure S4K). As expected, the abundance of most target gene products was negatively the abundance of most target gene products was negatively correlated with miR-205 expression, including ZEB1/2 and PTEN (Figure 4E). TP73 positively correlated with miR-205 at the transcript (Figure 4E) and still more strongly at the protein level (Figure 4E) and still more strongly at the protein level (Figure 4F) and with TP63 protein but not RNA (Figure S4L). TP73 overexpression mediates \\u0394Np63\\u03b1-dependent cell survival in squamous cell carcinoma, suggesting a mechanistic model in which functional and suggesting a mechanistic model in which functional and stoichiometric interaction between TP63 and TP73 leads to miR-205 overexpression in LSCC. Finally, consistent with the downregulation of key EMT-related proteins, the TP63 target miR-205 showed of key EMT-related proteins, the TP63 target miR-205 showed significant negative correlation with EMT activity but positive correlation with DNA replication (Figure 4G). The seemingly paradoxical role of TP63 in repressing EMT has been confirmed in role of TP63 in repressing EMT has been confirmed in previous studies. Of note, although \\u0394Np63\\u03b1 and miR-205 were consistently highly overexpressed in the B-I and Classical subtypes compared to NATs, their abundances were variable and not particularly low NATs, their abundances were variable and not particularly low in the EMT-E subtype. Thus, \\u0394Np63\\u03b1, either via miR-205 or indirectly, may induce a strong squamous epithelial cell phenotype in the B-I and Classical tumors, with additional mechanisms leading B-I and Classical tumors, with additional mechanisms leading to derepression and a metastatic phenotype, and other mechanisms beyond miR-205 may be involved in promoting the EMT phenotype in EMT-E samples. SOX2, a recurrently amplified squamous cell marker often co-amplified with TP63, was also overexpressed in tumors (Figure 4A). SOX2 is considered undruggable, intensifying therapeutic interest in upstream or downstream targets. An unbiased search (Figure in upstream or downstream targets. An unbiased search (Figure 4H) identified a significant correlation of LSD1 (KDM1A) with SOX2 protein. LSD1 regulates SOX2 expression and its inhibition in LSCC is currently being explored in a clinical setting in in LSCC is currently being explored in a clinical setting in conjunction with immunotherapy (NCT04350463). Other identified chromatin regulators included KDM3A, known to regulate SOX2 expression in ovarian cancer, and EZH2, whose inhibition in triple in ovarian cancer, and EZH2, whose inhibition in triple negative breast cancer leads to downregulation of SOX2 expression. EZH2 dependency was observed in SOX2 amplified LSCC cell lines (Figure S4M). The positive correlations of SOX2 with ALDH1A1, (Figure S4M). The positive correlations of SOX2 with ALDH1A1, ALDH3A1, and WNT5A (Figure 4H) suggest stemness-like features in these tumors. Interestingly, JAK1 was the protein most negatively correlated with SOX2 among other interferon-signaling proteins correlated with SOX2 among other interferon-signaling proteins such as IRF3 and IFNGR1 (Figure 4H), and negative correlation was seen between SOX2 and JAK-STAT signaling (Figure 4I). Crosstalk between lysine acetylation and ubiquitylation impacts cancer metabolism Consensus clustering of K-GG peptide abundances after correcting for cognate protein abundance revealed two stable ubiquitylproteome clusters (Figure 5A, Table S5) associated with country of origin and ethnicity (Fisher\\u2019s exact p<0.01). Proteins in of origin and ethnicity (Fisher\\u2019s exact p<0.01). Proteins in pathways such as glycolysis, JAK-STAT, MAPK, and immune signaling were differentially modified between these clusters (Figure 5A, Figure S5A\\u2013B, Table S5). To identify candidate enzymes driving Ub and UbL modifications in LSCC, we correlated E3 ubiquitin ligases or deubiquitylases (DUBs) to KGG-sites (Table S5). Eighteen DUBs with at least one negatively correlated K-GG site and 35 E3 ligases with at least negatively correlated K-GG site and 35 E3 ligases with at least one positively correlated K-GG site were identified, with top sites shown in Figure 5B and Figure S5C. HERC5 is the major E3 ligase involved in conjugation of the UbL modification ISG15 in ligase involved in conjugation of the UbL modification ISG15 in humans; hence the numerous positively correlated Ub/UbL sites (n=474; FDR<0.01) are likely ISG15 modifications. Consistent with this, we observed positive correlation between ISG15 and HERC5 this, we observed positive correlation between ISG15 and HERC5 abundance (Figure S5D). Positively-correlating Ub/UbL sites were enriched for glycolysis and TCA cycle annotations (Figure S5E). We observed several positively-correlating K-GG sites in the We observed several positively-correlating K-GG sites in the key glycolytic enzymes PKM, PGK1, and ENO1 (Figure 5C, Figure S5F). No significant negative correlations were observed between total protein abundance and their respective K-GG sites or HERC5, protein abundance and their respective K-GG sites or HERC5, suggesting a regulatory rather than degradative role, as expected for ISGylation (Figure S5G, Figure S5H). ISG15 modifications have been described in these glycolytic enzymes, and ISG15 knockdown been described in these glycolytic enzymes, and ISG15 knockdown in pancreatic cancer cells reduced oxidative phosphorylation and glycolysis. ISGylation can be induced by interferon in cancer; accordingly, HERC5 and ISG15 were positively associated with accordingly, HERC5 and ISG15 were positively associated with proteome-derived IFN-G signaling scores (Figure S5I, Figure S5J), and HERC5 was increased in tumors of the IFN-gamma dominant subtype (Figure 5D). Recently, a direct link between immune subtype (Figure 5D). Recently, a direct link between immune signaling and glucose metabolism through ISGylation was shown in the context of pathogenic invasion. Collectively, our data supports a potential role for HERC5 as a bridge between immune supports a potential role for HERC5 as a bridge between immune signaling and metabolic regulation in LSCC. This dataset allowed us to investigate crosstalk between lysine PTMs. We employed a modified version of CLUMPS to detect clustering of either acetylation or ubiquitylation sites within protein 3D structures. Significant ubiquitylation clusters were protein 3D structures. Significant ubiquitylation clusters were detected in 33 proteins and significant acetylation clusters in 17 proteins (p<0.05) (Table S5). Top hits revealed enrichment of glycolysis, fatty acid metabolism and OxPhos (Table S5), of glycolysis, fatty acid metabolism and OxPhos (Table S5), supporting modulation of these metabolic pathways by ubiquitylation and acetylation. In tumor-NAT comparison, major glycolytic enzymes PGK1 and PKM showed reduced acetylation at multiple known PGK1 and PKM showed reduced acetylation at multiple known inhibitory sites closely-positioned on the 3D structure, as well as on adjacent lysines surrounding the enzymatic pocket  (Figure 5E, Figure S5K). Tumors also had decreased ubiquitylation at 5E, Figure S5K). Tumors also had decreased ubiquitylation at closely-positioned K-GG-sites for TXN1, an important redox regulator (Figures 5F). PGK1 and PKM deacetylation suggests increased activity of these key glycolytic enzymes in tumors, a conclusion of these key glycolytic enzymes in tumors, a conclusion supported by increased phosphorylation on known activating sites of regulatory proteins (PFKFB3 pS461, PKM pS37). Upregulation of a glycolytic metabolic phenotype confers selective advantage to a glycolytic metabolic phenotype confers selective advantage to cancer cells by supporting uninterrupted growth, dynamically modulated by PTMs. While site-level roles of most Ub/UbL modifications and acetylsites are unclear, our data on acetylation, and acetylsites are unclear, our data on acetylation, phosphorylation, and HERC5-mediated ISGylation of glycolytic enzymes hints at the complexity of regulation of this cancer hallmark. Rapid growth of cells often leads to oxidative stress within tumors, which adapt by upregulating redox systems. Although thioredoxin (TXN) did not show overall differential protein expression between tumors and NATs, we observed increased TXN1 protein between tumors and NATs, we observed increased TXN1 protein combined with decreased Ub sites, particularly in the NMF Classical subtype (Kruskal Wallis FDR<0.01). The TXN1 activator TXNRD1 was also increased in the Classical subtype, while TXNIP, which was also increased in the Classical subtype, while TXNIP, which modulates cellular redox by binding and inhibiting TXN1, was decreased (Figure 5F). TXN and TXNRD1 were highly correlated with NRF2 score (TXN: R=0.85; TXNRD1: R=0.8, both p<2.2\\u00d710\\u221216), with NRF2 score (TXN: R=0.85; TXNRD1: R=0.8, both p<2.2\\u00d710\\u221216), consistent with their role in response to oxidative stress. Redox modulation through this pathway is associated with tumor proliferation, differentiation and prognosis in multiple epithelial differentiation and prognosis in multiple epithelial cancers, including NSCLC. The small proportion of glucose directed to the tumor mitochondria and subsequently to the TCA and OXPHOS cycles leads to increased acetyl CoA within the mitochondria and leads to increased acetyl CoA within the mitochondria and thence to non-enzymatic acetylation and inhibition of key enzymes in the fatty acid oxidation pathway such as ECH1 and ACADVL (also identified in the CLUMPS analysis). We recapitulate the finding identified in the CLUMPS analysis). We recapitulate the finding of increased ECH1 and ACADVL acetylation, including specific increase at the binding site of ACADVL in tumors (Figure 5E, Figure S5K). PTM-based modulation of these key metabolic and ROS S5K). PTM-based modulation of these key metabolic and ROS pathways is summarized in Figure 5G. As a complementary assessment of the crosstalk between lysine ubiquitylation and acetylation, we identified K-Ac or K-GG sites showing significant differential expression in tumors relative to NATs. Figure 5H shows 8 lysine sites where K-GG was relative to NATs. Figure 5H shows 8 lysine sites where K-GG was upregulated and K-Ac downregulated in tumors (upper panel) and 36 sites demonstrating the opposite trend (lower panel). Among the latter was the GTP-binding nuclear protein RAN, involved in the latter was the GTP-binding nuclear protein RAN, involved in nucleocytoplasmic transport, mitotic progression and spindle assembly. Acetylation on multiple lysine residues can affect RAN localization, GTP/GDP cycle and import / export complex RAN localization, GTP/GDP cycle and import / export complex formation. Consistent with the role of RAN in mitosis, RAN K127-acK showed dramatic upregulation in the P-P subtype (Figure 5I), named in part for its generally high proliferation (Figure 2B). named in part for its generally high proliferation (Figure 2B). The specific functional relevance of K127-ac has not been previously demonstrated, but its selective upregulation suggests a functional role. While of great interest to the PTM community and role. While of great interest to the PTM community and potentially very powerful, such crosstalk analyses are nascent, and interpretations remain suitably cautious. Immune landscape and regulation in LSCC Consensus clustering based on the xCell signatures (Table S6) identified three sets of tumors representing immunologically Hot, Warm, and Cold clusters (Figure 6A\\u2013C), as well as an NAT-enriched cluster. The NMF I-S subtype was significantly associated cluster. The NMF I-S subtype was significantly associated with the immune Hot cluster (Fisher\\u2019s exact p=6.371\\u00d710\\u221209) (Figure 6A). The Hot cluster was characterized by increased macrophages, CD4+, CD8+ and regulatory T cells, and dendritic cells in both CD4+, CD8+ and regulatory T cells, and dendritic cells in both deconvolution and immunohistochemistry (IHC) analyses (Figures 6A,C, S6C,D), and showed upregulation of immune-related pathways (Figure 6A\\u2013B, Figure S6A, Table S6). Immunosuppressive mediators 6A\\u2013B, Figure S6A, Table S6). Immunosuppressive mediators CTLA4, PD-1 (PDCD1), PD-L1 (CD274) and IDO1 and the key Treg transcriptional regulator FOXP3 were upregulated in Hot compared to Cold tumors, suggesting immune checkpoint-related therapeutic options suggesting immune checkpoint-related therapeutic options in these tumors. The Hot tumor acetylproteome was chiefly enriched for OXPHOS, Mitochondrial Complex and TCA Cycle pathways, consistent with increased metabolism or increased mitochondrial consistent with increased metabolism or increased mitochondrial acetyl-CoA leading to non-enzymatic acetylation of mitochondrial proteins (Figure 6D). Notably, NATs did not display consonant immune signaling (Figure S6B). The intermediate, Warm cluster immune signaling (Figure S6B). The intermediate, Warm cluster displayed upregulation of immune-related pathways such as PD-1 signaling, Interferon gamma (IFN-G) response, and Allograft rejection relative to Cold tumors, but did not show the downregulation relative to Cold tumors, but did not show the downregulation of cell cycle-related pathways observed in Hot tumors (Figure 6A\\u2013B, Figure S6A). In addition, canonical Wnt Signaling, active in tumor proliferation and immune evasion, was comparably in tumor proliferation and immune evasion, was comparably upregulated in the Warm and Cold tumor proteomes (Figure 6B). Fifteen pQTLs were also identified in proteins differentially abundant between Hot and Cold tumors (Table S6, Figure S6E) Rho GTPase signaling was upregulated in Hot tumors (Figure 6A, Table S6). ARHGDIB (RhoGDIb), the central regulatory molecule in GTPase activation, was the most significantly acetylated protein in the Hot cluster, followed by metabolic mitochondrial protein in the Hot cluster, followed by metabolic mitochondrial enzymes (Figure 6D). Acetylation of its homolog RhoGDIa at the conserved K135 site regulates F-actin assembly by activating RhoA. In contrast to RhoGDIa, the affinity of RhoGDIb for RhoA is In contrast to RhoGDIa, the affinity of RhoGDIb for RhoA is low, and the K135 acetylation of RhoGDIb may act as a positive regulator of Rac signaling via its interaction with Rac2 instead. In support of this hypothesis, we found significant upregulation support of this hypothesis, we found significant upregulation in the Hot cluster of Rho GTPase Rac2 and numerous Rac-specific regulators (Figures 6F, S6F, Table S6). Notably, these proteins are primarily immune-specific, indicating a possible role for are primarily immune-specific, indicating a possible role for lysine acetylation of RhoGDIb in promoting immune cell functions in the Hot cluster. Immunohistochemical staining for RhoGDIb (ARHGDIB) in the subset of immune Hot tumors confirmed strong (ARHGDIB) in the subset of immune Hot tumors confirmed strong signals in infiltrating immune cells and mesenchymal cells (Figure S6G), similar to dispersed RhoGDIb-expressing cells identified as lymphocytes seen in renal cell carcinomas. Thus, we propose as lymphocytes seen in renal cell carcinomas. Thus, we propose that K135 acetylation of RhoGDIb is a potential regulatory mechanism of immune cell functions that is in line with the inflammasome signature and immune-enriched networks. We explored the immunological associations of the RTK CBPE scores described above (Figure 2I,J) in LSCC tumors (Figure 6G). CSF1R, predominantly expressed in macrophages, showed the highest CBPE scores, and was positively associated with the Hot cluster CBPE scores, and was positively associated with the Hot cluster (Figure 6H) and consequently with immune-related pathways and cell types (Figure 6G). CSF1R correlated with immunosuppressive chemokine CCL5, phosphorylation of PIK3R1 Y580, downstream chemokine CCL5, phosphorylation of PIK3R1 Y580, downstream regulators of the actin cytoskeleton including WAS and WASF2 S103, and upregulation of AKAP13, among other immune-related proteins (Figure 6I). Tumor-associated macrophages (, CSF1R signaling, and 6I). Tumor-associated macrophages (, CSF1R signaling, and AKAP13 have each been proposed as anti-cancer targets. Proteomic Biomarker Candidates for Prognosis, Diagnosis, and Treatment Tumors and paired NATs revealed remarkable differences in protein expression and pathway enrichment (Figure 7A, Table S7). The top 50 of 502 differential proteins are shown in Figure S7A. In addition to 206 upregulated oncogenic proteins (OncoKB, Table addition to 206 upregulated oncogenic proteins (OncoKB, Table S7), several CT antigens, attractive targets of CT-antigen therapies, were overexpressed in tumors (Figure S7B), with the I-S samples showing the fewest CT antigens. As complete long-term samples showing the fewest CT antigens. As complete long-term follow-up was not yet available for this cohort, we leveraged the TCGA LSCC dataset to examine the association of these 502 candidate tumor biomarkers with overall survival (OS) or disease-free tumor biomarkers with overall survival (OS) or disease-free survival (DFS). Expression of four of the most highly differential genes showed significant association with poor OS and another 15 genes with poor DFS (Figure 7B, Figure S7C). Furthermore, 15 genes with poor DFS (Figure 7B, Figure S7C). Furthermore, knockdown of these tumor biomarker proteins reduced fitness across 16 LSCC cell lines (https://depmap.org/), suggesting critical roles in key cellular transformation and proliferation processes in key cellular transformation and proliferation processes (Figure 7C, Table S7). There is increasing interest in molecular taxonomies that help position cancer types in the context of related cancers. CNAs were more frequent in LSCC than in LUAD or head and neck squamous cell carcinoma (HNSCC), which respectively share tissue and cell carcinoma (HNSCC), which respectively share tissue and cell type of origin (Figure S7D). Widespread deletions of immune-related genes in LSCC correlated with ESTIMATE immune scores (Figure 7D). Immune-related deletions were less prominent in HNSCC and 7D). Immune-related deletions were less prominent in HNSCC and especially LUAD (Figure S7E), except for Chromosome 3p deletion, encompassing five chemokine receptors. Squamous cancers shared 3q amplification, JAK2 deletion and skin-development proteins 3q amplification, JAK2 deletion and skin-development proteins from across the genome. Finally, deletions of 4p14 and 5q, including multiple Toll-like receptor and interleukin signaling genes, were unique to LSCC. Among key tumor-specific phosphorylation were unique to LSCC. Among key tumor-specific phosphorylation events in LSCC, we identified upregulation of activating sites on 27 kinases, including MAPK14 and three others (DCK, EGFR, SRC) targetable by FDA-approved drugs (Figure S7F), with several targetable by FDA-approved drugs (Figure S7F), with several being shared by LUAD and HNSCC tumors. Notably, EGFR protein was significantly upregulated in the squamous cancers but not LUAD (Figure S7G), although LUAD had many activating EGFR mutations, and S7G), although LUAD had many activating EGFR mutations, and activating phosphorylation of EGFR was increased in both LUAD and LSCC. High EGFR amplification in the LSCC cohort did not correlate with EGFR pathway activity as assessed by mRNA-based PROGENy with EGFR pathway activity as assessed by mRNA-based PROGENy scoring (Figure 7E). Instead, this activity measurement was highly correlated to mRNA abundance of the five EGFR ligands that are prominent in tumors (Figure 7F). Furthermore, EGFR amplified prominent in tumors (Figure 7F). Furthermore, EGFR amplified samples displayed increased phosphorylation on proteins related to actin filament and cell junction organization as well as cell-substrate adhesion (Figure 7G). These findings and similar cell-substrate adhesion (Figure 7G). These findings and similar results recently reported in HNSCC support a squamous cell cancer feature in which EGFR ligand abundance drives canonical EGFR pathway activity and tumors with very high levels of EGFR may no activity and tumors with very high levels of EGFR may no longer require the ligand to drive activity. EGFR inhibition guided by EGFR abundance has been unsuccessful in squamous tumors. Since EGFR inhibitors such as cetuximab affect ligand-induced EGFR EGFR inhibitors such as cetuximab affect ligand-induced EGFR activity, ligand abundance, rather than EGFR amplification, might better predict EGFR inhibitor response in this population. Figure 7H graphically represents these and other key findings Figure 7H graphically represents these and other key findings presented in this manuscript, emphasizing translatable implications. Discussion In this study, we provide detailed proteogenomic profiles of treatment-na\\u00efve, primary LSCC tumors and paired NATs, with unprecedented coverage of post-translational modifications including phosphorylation, acetylation and for the first time, including phosphorylation, acetylation and for the first time, ubiquitylation. Unsupervised NMF-based multi-omic clustering suggested a refinement to prior RNA-based clustering, dividing basal tumors into Basal-inclusive and EMT-enriched subtypes with tumors into Basal-inclusive and EMT-enriched subtypes with biological differences of potential therapeutic significance. Molecular events downstream of copy number alterations showed NSD3 rather than FGFR1 protein overexpression in tumors with 8p11.23 rather than FGFR1 protein overexpression in tumors with 8p11.23 amplification, providing both a potential explanation for the limited effectiveness of FGFR1-targeted treatment in this population and a potential alternative therapy. Parallel downstream and a potential alternative therapy. Parallel downstream analysis of DNA promoter methylation events provided insight into regulation of many proteins differential between cancer and NATs. Comprehensive proteogenomic data provided a deeper exposition of established LSCC biology, often with potential therapeutic implications. We identified a subset of low-p63 tumors that were characterized by high levels of the known therapeutic target characterized by high levels of the known therapeutic target survivin. SOX2 is considered undruggable, but our analyses provide rationale for exploring LSD1 or other chromatin modifiers such as EZH2 to target SOX2 amplified / overexpressing tumors. The as EZH2 to target SOX2 amplified / overexpressing tumors. The importance of glycolysis and oxidative stress in LSCC are well-appreciated, but our data support complex regulation of these metabolic pathways by crosstalk between ubiquitylation (or metabolic pathways by crosstalk between ubiquitylation (or ubiquitin-like modifications), phosphorylation and acetylation. Immunotherapy represents the greatest advance in LSCC therapy in decades, but outcomes lag behind those seen in patients with LUAD but outcomes lag behind those seen in patients with LUAD and still only a minority of patients exhibit long-term responses; numerous immune-related proteogenomic observations suggest directions for further investigation. Proteogenomic dissection of the for further investigation. Proteogenomic dissection of the downstream effects of CDKN2A mutations had clinical implications related both to the interpretation of trials utilizing CDK4/6 inhibitors in LSCC patients and to biomarker selection for future in LSCC patients and to biomarker selection for future studies: though CDK4/6 inhibition has shown limited efficacy in LSCC trials to date, our analysis suggested that a more nuanced assessment of RB1 protein expression and phosphorylation is required of RB1 protein expression and phosphorylation is required before declaring this approach unsuccessful. Finally, triangulation between LSCC, LUAD and HNSCC demonstrated the influence of both tissue and cell type of origin on cancer biology, with of both tissue and cell type of origin on cancer biology, with therapeutic vulnerabilities both unique and common across these cancer types. Limitations of the study Many important clinical advances in NSCLC and other cancers have been driven by genomic profiling of bulk tumor material, and we anticipate that the same will prove true of bulk proteogenomic characterization, as performed here. Nevertheless, tumor characterization, as performed here. Nevertheless, tumor heterogeneity is an important complication, and emerging methods for proteomics approaches to heterogeneity, such as pairing microdissection with mass spectrometry or using mass cytometry will be with mass spectrometry or using mass cytometry will be useful adjuncts. It is typically metastatic disease that proves lethal, and in-depth proteogenomic analysis of primary tumors paired with metastatic lesions will be needed to provide critical insights metastatic lesions will be needed to provide critical insights into metastatic biology. Importantly, the scope of a landscape study such as this necessitates that it be understood as hypothesis-generating, and a wider community effort will be required to and a wider community effort will be required to validate biological observations and suggested therapeutic alternatives. Nevertheless, we hope this study will prove a valuable resource to the research and clinical communities, and advance the to the research and clinical communities, and advance the understanding and treatment of LSCC. STAR\\u2605METHODS\\n\\nRESOURCE AVAILABILITY\\n\\nLead contact\\n\\nFurther information and requests should be directed to and will be fulfilled by the lead author Michael A. Gillette (gillette@broadinstitute.org)\\n\\nMaterial availability Material availability\\n\\nThis study did not generate new unique reagents.\\n\\nData and Code Availability\\n\\nCPTAC LSCC proteomics data:\\n\\nProteomic Data Commons (PDC); https://pdc.cancer.gov/pdc/ with identifiers PDC000232, PDC000233, PDC000234 and PDC000237 CPTAC data portal LSCC: https://cptac-data-portal.georgetown.edu/study-summary/S063\\n\\nCPTAC HNSCC proteomics data:\\n\\nPDC: https://pdc.cancer.gov/pdc/ with identifiers PDC000221 and PDC000222\\n\\n https://cptac-data-portal.georgetown.edu/study-summary/S054 https://cptac-data-portal.georgetown.edu/study-summary/S054 \\n\\nCPATC LUAD proteomics data:\\n\\nPDC: https://pdc.cancer.gov/pdc/ with identifiers PDC000153, PDC000149, PDC000224)\\n\\n https://cptac-data-portal.georgetown.edu/study-summary/S056 https://cptac-data-portal.georgetown.edu/study-summary/S056 \\n\\nGenomic and transcriptomic data files can be accessed at the Genomic Data Commons (GDC); https://portal.gdc.cancer.gov/, via dbGaP Study Accession: phs001287.v10.p5 All histologic https://www.cancerimagingarchive.net/datascope/cptac/home/ and radiologic details can be accessed from the The Cancer Imaging Archive (TCIA) Public Access https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC. Sample annotation, processed and normalized data files are provided as Tables S1\\u2013S3.\\n\\nSoftware and code used in this study are referenced in their corresponding STAR Method sections and also the Key Resource Table.\\n\\nEXPERIMENTAL MODEL AND SUBJECT DETAILS EXPERIMENTAL MODEL AND SUBJECT DETAILS\\n\\nHuman Subjects A total of 113 participants (90 males, 23 females, 40\\u201388 years old) were collected for this study by 13 different tissue source sites from seven different countries between May 2016 and August 2018, with a median follow-up time of 2.5 years. Five (one 2018, with a median follow-up time of 2.5 years. Five (one female and four male) samples were excluded based on further pathological assessment and 108 tumors and 99 paired NATs were used in this study. Only histopathologically-defined adult lung squamous study. Only histopathologically-defined adult lung squamous tumors were considered for analysis, with an age range of 40\\u201388. Institutional review boards at tissue source sites reviewed protocols and consent documentation adhering to the Clinical Proteomic and consent documentation adhering to the Clinical Proteomic Tumor Analysis Consortium (CPTAC) guidelines. Clinical data were obtained from tissue source sites and aggregated by an internal database called the CDR (Comprehensive Data Resource) that database called the CDR (Comprehensive Data Resource) that synchronizes with the CPTAC DCC (https://cptac-data-portal.georgetown.edu/). Clinical data can be accessed and downloaded from the DCC (Data Coordinating Center). Demographics, histopathologic DCC (Data Coordinating Center). Demographics, histopathologic information, and treatment details were collected. LSCC histopathology was confirmed for all cases by at least 2 expert pathologists based on high resolution images of H&E sections. For samples based on high resolution images of H&E sections. For samples with low TP63 status, additional pathologic confirmation was performed. The histologic, genotypic, clinical, geographical and other associated metadata is summarized in Table S1. METHOD DETAILS\\n\\nSpecimen Acquisition The tumor, normal adjacent tissue (NAT), and whole blood samples used in this manuscript were prospectively collected for the CPTAC project. Biospecimens were collected from newly diagnosed patients with LSCC who underwent surgical resection and had patients with LSCC who underwent surgical resection and had received no prior treatment for their disease, including chemotherapy or radiotherapy. All cases had to be of acceptable LSCC histology but were collected regardless of surgical stage or histology but were collected regardless of surgical stage or histologic grade. Cases were staged using the AJCC cancer staging system 7th edition and the 8th edition. The tumor specimen weights ranged from 125 to 1560 milligrams. The average tissue mass ranged from 125 to 1560 milligrams. The average tissue mass was 220 mg. For most cases, three to four tumor specimens were collected. Paired histologically-normal adjacent lung tissues (NATs) were collected from the same patient at tumor resection. Each were collected from the same patient at tumor resection. Each tissue specimen endured cold ischemia for less than 40 minutes prior to freezing in liquid nitrogen; the average ischemic time was 16 minutes from resection/collection to freezing. Specimens was 16 minutes from resection/collection to freezing. Specimens were flash frozen in liquid nitrogen. Histologic sections obtained from top and bottom portions from each case were reviewed by a board-certified pathologist to confirm the assigned by a board-certified pathologist to confirm the assigned pathology. For samples to be deemed acceptable, the top and bottom sections had to contain an average of 50% tumor cell nuclei with less than 20% necrosis. Specimens were shipped overnight from the than 20% necrosis. Specimens were shipped overnight from the tissue source sites to the biospecimen core resource (BCR) located at Van Andel Research Institute, Grand Rapids, MI using a cryoport that maintained an average temperature of less than \\u2212140\\u00b0C. that maintained an average temperature of less than \\u2212140\\u00b0C. At the biospecimen core resource, specimens were confirmed for pathology qualification and prepared for genomic, transcriptomic, and proteomic analyses. Selected specimens were cryopulverized and proteomic analyses. Selected specimens were cryopulverized using a Covaris CryoPREP instrument and material aliquoted for subsequent molecular characterization. Genomic DNA and total RNA were extracted and sent to the genome sequencing centers. The were extracted and sent to the genome sequencing centers. The whole exome and whole genome DNA sequencing and methylation EPIC array analyses were performed at the Broad Institute, Cambridge, MA. TotalRNA and miRNA sequencing were performed at the MA. TotalRNA and miRNA sequencing were performed at the University of North Carolina, Chapel Hill, NC. Material for proteomic analyses were sent to the Proteomic Characterization Center (PCC) at the Broad Institute, Cambridge, MA. Sequencing sample preparation Our study sampled a single site of the primary tumor from surgical resections, with an internal requirement to process a minimum of 125mg of tumor issue and 50mg of NAT. DNA and RNA were extracted from tumor and NAT specimens in a co-isolation protocol from tumor and NAT specimens in a co-isolation protocol using Qiagen\\u2019s QIAsymphony DNA Mini Kit and QIAsymphony RNA Kit. Genomic DNA was also isolated from peripheral blood (3\\u20135mL) to serve as matched normal reference material. The Qubit\\u2122 dsDNA BR Assay as matched normal reference material. The Qubit\\u2122 dsDNA BR Assay Kit was used with the Qubit\\u00ae 2.0 Fluorimeter to determine the concentration of dsDNA in an aqueous solution. Any sample that passed quality control and produced enough DNA yield to go through quality control and produced enough DNA yield to go through the multiple planned genomic assays was sent for genomic characterization. RNA quality was quantified using the NanoDrop 8000 and quality assessed using an Agilent Bioanalyzer. A sample of and quality assessed using an Agilent Bioanalyzer. A sample of sufficient quantity that passed RNA quality control and had a minimum RIN (RNA integrity number) score of 7 was subjected to RNA sequencing. Identity matches for germline, normal adjacent RNA sequencing. Identity matches for germline, normal adjacent tissue, and tumor tissue were confirmed at the BCR using the Illumina Infinium QC array. This beadchip contains 15,949 markers designed to prioritize sample tracking, quality control, and designed to prioritize sample tracking, quality control, and stratification. Whole exome Sequencing (WES)\\n\\nLibrary construction and Hybrid Selection Library construction was performed as described in, with the following modifications: initial genomic DNA input into shearing was reduced from 3\\u03bcg to 20\\u2013250ng in 50\\u03bcL of solution. For adapter ligation, Illumina paired-end adapters were replaced with ligation, Illumina paired-end adapters were replaced with palindromic forked adapters, purchased from Integrated DNA Technologies (IDT), with unique dual-indexed molecular barcode sequences to facilitate downstream pooling. Kapa HyperPrep reagents in to facilitate downstream pooling. Kapa HyperPrep reagents in 96-reaction kit format were used for end repair/A-tailing, adapter ligation, and library enrichment PCR. In addition, during the post-enrichment SPRI cleanup, elution volume was reduced to 30\\u03bcL SPRI cleanup, elution volume was reduced to 30\\u03bcL to maximize library concentration, and a vortexing step was added to maximize the amount of template eluted. After library construction, libraries were pooled into groups of up to 96 samples. Hybridization were pooled into groups of up to 96 samples. Hybridization and capture were performed using the relevant components of Illumina\\u2019s Nextera Exome Kit and following the manufacturer\\u2019s suggested protocol, with the following exceptions: First, all libraries protocol, with the following exceptions: First, all libraries within a library construction plate were pooled prior to hybridization. Second, the Midi plate from Illumina\\u2019s Nextera Exome Kit was replaced with a skirted PCR plate to facilitate automation. was replaced with a skirted PCR plate to facilitate automation. All hybridization and capture steps were automated on the Agilent Bravo liquid handling system. Cluster Amplification and Sequencing After post-capture enrichment, library pools were quantified using qPCR (KAPA Biosystems) using an automated assay on the Agilent Bravo with probes specific to the ends of the adapters. Based on qPCR quantification, libraries were normalized to 2nM. Based on qPCR quantification, libraries were normalized to 2nM. Cluster amplification of DNA libraries was performed following manufacturer\\u2019s protocol (Illumina) using exclusion amplification chemistry and flowcells. Flow cells were sequenced utilizing chemistry and flowcells. Flow cells were sequenced utilizing sequencing-by-synthesis chemistry. The flow cells were then analyzed using RTA v.2.7.3 or later. Each pool of whole exome libraries was sequenced on paired 76-cycle runs with two 8-cycle index was sequenced on paired 76-cycle runs with two 8-cycle index reads across the number of lanes needed to meet coverage for all libraries in the pool. Pooled libraries were run on HiSeq4000 paired-end runs to achieve a minimum of 150x on-target coverage per runs to achieve a minimum of 150x on-target coverage per library. The raw Illumina sequence data were demultiplexed and converted to FASTQ files; adapter and low-quality sequences were trimmed. The raw reads were mapped to the GRCh38/hg38 human reference The raw reads were mapped to the GRCh38/hg38 human reference genome and the validated BAMs were used for downstream analysis and variant calling. Whole genome sequencing (WGS)\\n\\nCluster Amplification and Sequencing An aliquot of genomic DNA (350ng in 50\\u03bcL) was used as the input into DNA fragmentation (aka shearing). Shearing was performed acoustically using a Covaris focused-ultrasonicator, targeting 385bp fragments. Following fragmentation, additional size 385bp fragments. Following fragmentation, additional size selection was performed using SPRI cleanup. Library preparation was performed using a commercially available KAPA Hyper Prep without amplification module kit (KAPA Biosystems) and with palindromic amplification module kit (KAPA Biosystems) and with palindromic forked adapters with unique 8-base index sequences embedded within the adapter (IDT). Following sample preparation, libraries were quantified using quantitative PCR (KAPA Biosystems), with were quantified using quantitative PCR (KAPA Biosystems), with probes specific to the ends of the adapters using the automated Agilent\\u2019s Bravo liquid handling platform. Based on qPCR quantification, libraries were normalized to 1.7nM and pooled into libraries were normalized to 1.7nM and pooled into 24-plexes. Sample pools were combined with HiSeqX Cluster Amp Reagents EPX1, EPX2, and EPX3 into single wells on a strip tube using the Hamilton Starlet Liquid Handling system. Cluster amplification of the templates was performed according to the manufacturer\\u2019s of the templates was performed according to the manufacturer\\u2019s protocol (Illumina) with the Illumina cBot. Flow cells were sequenced to a minimum of 15x on HiSeqX utilizing sequencing-by-synthesis kits to produce 151bp paired-end reads. Output from kits to produce 151bp paired-end reads. Output from Illumina software was processed by the Picard data processing pipeline to yield BAM files containing demultiplexed, aggregated, aligned reads. All sample information tracking was performed by automated All sample information tracking was performed by automated LIMS messaging. Array Based Methylation Analysis The Methylation EPIC array uses an 8-sample version of the Illumina Beadchip capturing >850,000 methylation sites per sample. Two hundred and fifty nanograms of DNA was used for the bisulfite conversion using Infinium MethylationEPIC BeadChip Kit conversion using Infinium MethylationEPIC BeadChip Kit (Illumina). The EPIC array includes sample plating, bisulfite conversion, and methylation array processing. After scanning, the data was processed through an automated genotype-calling pipeline. Data processed through an automated genotype-calling pipeline. Data output consisted of raw idats and a sample sheet. RNA and miRNA sequencing\\n\\nQuality Assurance and Control of RNA Analytes All RNA analytes were assayed for RNA integrity, concentration, and fragment size. Samples for total RNA-seq were quantified on a TapeStation system (Agilent, Inc. Santa Clara, CA). Samples with RINs >7.0 were considered high quality and were considered with RINs >7.0 were considered high quality and were considered for sequencing. Total RNA-seq libraries were generated using 300 nanograms of total RNA using the TruSeq Stranded Total RNA Library Prep Kit with Ribo-Zero Gold and bar-coded with individual Prep Kit with Ribo-Zero Gold and bar-coded with individual tags following the manufacturer\\u2019s instructions (Illumina). Total RNA Libraries were prepared on an Agilent Bravo automated liquid handling system. Quality control was performed at every step, and system. Quality control was performed at every step, and the libraries were quantified using a TapeStation system. Total RNA Sequencing Indexed libraries were prepared and run on HiSeq4000 paired-end 75 base pairs to generate a minimum of 120 million reads per sample library with a target of greater than 90% mapped reads. The raw Illumina sequence data were demultiplexed and converted to raw Illumina sequence data were demultiplexed and converted to FASTQ files, and adapter and low-quality sequences were trimmed. Samples were then assessed for quality by mapping reads to GRCh38/hg38, estimating the total number of mapped reads, amount of estimating the total number of mapped reads, amount of RNA mapping to coding regions, amount of rRNA in the sample, number of genes expressed, and relative expression of housekeeping genes. Samples passing this QA/QC were then clustered with other Samples passing this QA/QC were then clustered with other expression data from similar and distinct tumor types to confirm expected expression patterns. Atypical samples were then SNP typed from the RNA data to confirm source analyte. FASTQ files of all from the RNA data to confirm source analyte. FASTQ files of all reads were then uploaded to the GDC repository. miRNA-seq Library Construction miRNA-seq library construction was performed from the RNA samples using the NEXTflex Small RNA-Seq Kit (v3, PerkinElmer, Waltham, MA) and barcoded with individual tags following the manufacturer\\u2019s instructions. Libraries were prepared on a Sciclone Liquid instructions. Libraries were prepared on a Sciclone Liquid Handling Workstation. Quality control was performed at every step, and the libraries were quantified using a TapeStation system and an Agilent Bioanalyzer using the Small RNA analysis kit. Pooled an Agilent Bioanalyzer using the Small RNA analysis kit. Pooled libraries were then size selected according to NEXTflex kit specifications using a Pippin Prep system (Sage Science, Beverly, MA). miRNA Sequencing Indexed libraries were loaded on the HiSeq4000 to generate a minimum of 10 million reads per library with a minimum of 90% reads mapped. The raw Illumina sequence data were demultiplexed and converted to FASTQ files for downstream analysis. Resultant data to FASTQ files for downstream analysis. Resultant data were analyzed using a variant of the small RNA quantification pipeline developed for TCGA (Chu et al., 2016). Data from samples were assessed for the number of miRNAs called, species diversity, and for the number of miRNAs called, species diversity, and total abundance before uploading to the GDC repository. Mass Spectrometry methods The protocols below for protein extraction, tryptic digestion, TMT-11 labeling of peptides, peptide fractionation by basic reversed-phase liquid chromatography, phosphopeptide enrichment using immobilized metal affinity chromatography, and LC-MS/MS were immobilized metal affinity chromatography, and LC-MS/MS were performed as previously described in depth. Acetyl-enrichment was performed as described before with modifications as indicated below. Protein Extraction and Tryptic Digestion Approximately fifty milligrams (wet weight) of cryopulverized human LSCC and NAT samples were homogenized in the lysis buffer for 30 minutes at a ratio of about 200 uL lysis buffer for every 50 mg wet weight tissue. The lysis buffer consisted of 8 M urea, mg wet weight tissue. The lysis buffer consisted of 8 M urea, 75 mM NaCl, 1 mM EDTA, 50 mM Tris HCl (pH 8), 10 mM NaF, phosphatase inhibitor cocktail 2 (1:100; Sigma, P5726) and cocktail 3 (1:100; Sigma, P0044), 2 \\u03bcg/mL aprotinin (Sigma, A6103), 10 \\u03bcg/mL Sigma, P0044), 2 \\u03bcg/mL aprotinin (Sigma, A6103), 10 \\u03bcg/mL leupeptin (Roche, 11017101001), and 1 mM PMSF (Sigma, 78830). Lysates were centrifuged at 20,000 g for 10 minutes and protein concentrations of the clarified lysates were measured by BCA assay of the clarified lysates were measured by BCA assay (Pierce). Protein lysates were subsequently reduced with 5 mM dithiothreitol (Thermo Scientific, 20291) for an hour at 37C and alkylated with 10 mM iodoacetamide (Sigma, A3221) for 45 minutes in the dark 10 mM iodoacetamide (Sigma, A3221) for 45 minutes in the dark at room temperature. Prior to digestion, samples were diluted 4-fold to achieve 2 M urea with 50mM Tris HCl (pH 8). Digestion was performed with LysC (Wako, 129\\u201302541) for 2 hours and with was performed with LysC (Wako, 129\\u201302541) for 2 hours and with trypsin (Promega, V511X) overnight, both at a 1:50 enzyme-to-protein ratio and at room temperature. Digested samples were acidified with formic acid (FA; Sigma, F0507) to achieve a final with formic acid (FA; Sigma, F0507) to achieve a final volumetric concentration of 1% (final pH of ~3), and centrifuged at 1,500 g for 15 minutes to clear precipitated urea from peptide lysates. Samples were desalted on tC18 SepPak columns (Waters, 200mg, Samples were desalted on tC18 SepPak columns (Waters, 200mg, WAT054925) and dried down using a SpeedVac apparatus. Common Reference (CR) Pool and Plex layout\\n\\nProteome, Phosphoproteome, and Acetyprotome: The proteomic, phosphoproteomic and acetylproteome analyses of LSCC samples consisted of 22 TMT-11 plex experiments. To facilitate quantitative comparison between all samples across experiments, a common reference (CR) sample was included in each 11-plex. a common reference (CR) sample was included in each 11-plex. A common physical, rather than in silico reference was used for this purpose for optimal quantitative precision between TMT-11 experiments. Considerations prior to creating the reference sample Considerations prior to creating the reference sample were that this sample needed to be of adequate quantity to cover all planned experiments for both the discovery and future confirmatory sets (not part of this study) with overhead for additional sets (not part of this study) with overhead for additional possible experiments. The CR included all (212/212) of the samples analyzed in the discovery set TMT experiments, yielding a CR that was representative of all the samples in the study. Making the was representative of all the samples in the study. Making the CR as representative of the study as a whole was particularly important since by definition only analytes represented in the reference sample would be included in the final ratio-based data sample would be included in the final ratio-based data analyses. 110 unique tumor samples with 102 paired NAT samples are distributed among 22 11-plex experiments, with 10 individual samples occupying the first 10 channels of each experiment and the 11th channel being reserved for the reference sample. All the tumors channel being reserved for the reference sample. All the tumors were in the C channels and all the normal samples were in the N channels. The first 8 channels of each experiment contained 4 tumor/normal pairs, with each pair of patient samples adjacent to pairs, with each pair of patient samples adjacent to each other. Channels 9 and 10 (130C and 131N) either contained a 5th tumor/normal pair, or an unpaired tumor sample in channel 9 and an alternate common reference sample in channel 10. These included 3 common reference sample in channel 10. These included 3 global LUAD common references, 3 HNSCC common references from the team at JHU, and 2 Tumor only common references from this LSCC discovery set. To ensure capacity for additional samples or experiments given a target input of 300 \\u03bcg peptide per channel per experiment, 18 mg total was targeted for reference material. To meet these collective requirements, all the samples in the discovery set that requirements, all the samples in the discovery set that had enough material were included in the CR. After reserving a conservative 400 \\u03bcg peptide / sample for individual sample analysis, an additional amount of 120 \\u03bcg for each of the samples with an additional amount of 120 \\u03bcg for each of the samples with adequate quantities were pooled. All 212 samples were selected for the combined tumor/normal CR. To make the combined CR, tumor only and normal only CRs were first created separately, with 110 and normal only CRs were first created separately, with 110 tumor samples and 102 normal samples. After creating tumor and NAT only CRs at 150 ug and 150ug amounts, a pool of combined CR was made, comprised of 8.6 mg tumor only reference and 8.6 mg normal comprised of 8.6 mg tumor only reference and 8.6 mg normal only reference. The 17.2 mg pooled reference material was divided into 55 300 \\u03bcg aliquots, dried down, and frozen at \\u221280\\u00b0C until use. Ubiquitylproteome: 750 ug of unlabeled peptides was required from each sample to ensure depth of coverage in the ubiquitylproteome, which restricted the number of samples available for profiling. 147/212 samples from the proteome, phosphoproteome, acetylproteome set were from the proteome, phosphoproteome, acetylproteome set were included in the ubiquitylproteome analysis. A separate ubiquitylproteome CR was constructed prior to KGG enrichment, due to limitations in sample material (both patient material and remaining CR in sample material (both patient material and remaining CR aliquots to be used in future LSCC proteome, phosphoproteome, and acetylproteome processing.. In order to make TMT ratio-based quantitation comparable across omes, the ubiquitylproteome CR was comparable across omes, the ubiquitylproteome CR was constructed to be as similar as practically possible to the proteome, phosphoproteome, acetylproteome CR. The ubiquitylproteome CR contains all samples from the proteome, phosphoproteome, acetylproteome all samples from the proteome, phosphoproteome, acetylproteome set with material remaining to contribute, 209/212. The ubiquitylproteome CR contributions were of 3 types: 104/147 samples included in ubiquitylproteome study and made full CR contribution, >1000ug available.\\n\\n42/147 samples included in ubiquitylproteome study and made partial CR contribution, 750ug \\u22121000 ug available. 63/65 samples from the proteome, phosphoproteome, acetylproteome set not included in ubiquitylproteome study, but made full CR contribution, <750 ug available. CR tumor and normal peptides were combined at a ratio of 55:45 to reflect the higher proportion of tumor samples in the experimental design (as opposed to proteome, phosphoproteome, acetylproteome CR\\u2019s proportion of 50:50), and aliquoted into 750 ug CR\\u2019s proportion of 50:50), and aliquoted into 750 ug aliquots. In the ubiquitylproteome experimental design, 15 11-plex experiments contained 87 unique tumor samples in the C channels and 60 normal samples in the N channels, with the first 8 channels of every 11-plex consisting of patient-paired tumor/normal samples every 11-plex consisting of patient-paired tumor/normal samples adjacent to each other. The 11th channel was reserved for the ubiquitylproteome common reference sample. Channel 9 (130C) contained an unpaired tumor, while channel 10 (131N) contained either an unpaired tumor, while channel 10 (131N) contained either an unpaired tumor, or in 3 plexes, a LUAD ubiquitylproteome common reference (which was constructed similarly for that experiment, unpublished). TMT-11 Labeling of Peptides\\n\\nProteome, Phosphoproteome, Acetylproteome: 300 \\u03bcg of desalted peptides per sample (based on peptide-level BCA after digestion) were labeled with 11-plex TMT reagents and combined for multiplexed analysis using a \\u201creduced labeling\\u201d approach (Zecha et al., 2019). To conservatively yield extra approach (Zecha et al., 2019). To conservatively yield extra labeled peptide in case a given sample demonstrated unsuitably low TMT reporter ion intensity during channel mixing control LC-MS/MS runs, 400 ug peptides per sample were initially labeled, with runs, 400 ug peptides per sample were initially labeled, with only 300 ug combined per sample for the final multiplexed experiments upon successful quality control. For each 400 \\u03bcg peptide aliquot of an individual sample, peptides were reconstituted to 5 of an individual sample, peptides were reconstituted to 5 mg/ml with 50 mM HEPES (pH 8.5) solution and 400 ug labeling reagent dissolved in 20 uL anhydrous acetonitrile was added. After 1 h incubation with shaking, samples were diluted 1:1 with a solution with shaking, samples were diluted 1:1 with a solution of 80% 50 mM HEPES and 20% acetonitrile. After confirming good label incorporation, 5 \\u03bcL of 5% hydroxylamine was added to quench the unreacted TMT reagents. Good label incorporation was defined as unreacted TMT reagents. Good label incorporation was defined as having less than 3% isobaric under-labeled MS/MS spectra in each sample, as measured by LC-MS/MS after taking out a 2 \\u03bcg aliquot from each sample and analyzing 0.25 \\u03bcg. If a sample did not from each sample and analyzing 0.25 \\u03bcg. If a sample did not have sufficient label incorporation, additional TMT was added to the sample and another 1 h incubation was performed with shaking. At the time that the labeling efficiency quality control samples the time that the labeling efficiency quality control samples were taken out, an additional 4 \\u03bcg of material from each sample was taken out and combined as a mixing control. After analyzing the mixing control sample by LC-MS/MS, intensity values of the the mixing control sample by LC-MS/MS, intensity values of the individual TMT reporter ions were summed across all peptide spectrum matches and compared to ensure that the total reporter ion intensity of each sample met a threshold of +/\\u2212 15% of the ion intensity of each sample met a threshold of +/\\u2212 15% of the internal reference. If necessary, adjustments were made by either using additional labeled material or reducing an individual sample\\u2019s contribution to the mixture, and analyzing a subsequent contribution to the mixture, and analyzing a subsequent mixing control, until all samples met the threshold and were thus approximately 1:1:1.... Differentially labeled peptides were then mixed (11 \\u00d7 300 \\u03bcg), dried down via vacuum centrifuge, and the mixed (11 \\u00d7 300 \\u03bcg), dried down via vacuum centrifuge, and the quenched, combined sample was subsequently desalted on a 200 mg tC18 SepPak column. The same lot of TMT reagents was used for all samples (UA280170/TL272399). Peptide Fractionation\\n\\nFor Proteome, Phosphoproteome, Acetylproteome: To reduce sample complexity, peptide samples were separated by high pH reversed phase (RP) separation as described previously (Mertins et al., 2018). A desalted 3.3 mg, 11-plex TMT-labeled experiment (based on peptide-level BCA after digestion) was experiment (based on peptide-level BCA after digestion) was reconstituted in 935 \\u03bcL 4.5mM ammonium formate (pH 10) and 2% acetonitrile, centrifuged for 7 min at 20000 g to precipitate insoluble peptides, loaded on a 4.6 mm x 250 mm column RP Zorbax 300 A peptides, loaded on a 4.6 mm x 250 mm column RP Zorbax 300 A Extend-C18 column (Agilent, 3.5 \\u03bcm bead size), and separated on an Agilent 1260 Series HPLC instrument using basic reversed-phase chromatography. Solvent A (2% acetonitrile, 4.5 mM ammonium chromatography. Solvent A (2% acetonitrile, 4.5 mM ammonium formate, pH 10) and a nonlinear increasing concentration of solvent B (90% acetonitrile, 4.5 mM ammonium formate, pH 10) were used to separate peptides. The 4.5 mM ammonium formate solvents were to separate peptides. The 4.5 mM ammonium formate solvents were made by 40\\u2013fold dilution of a stock solution of 180 mM ammonium formate, pH 10. To make 1L of stock solution, add 25 mL of 28% (wt/vol) ammonium hydroxide (28%, density 0.9 g/ml, mL of 28% (wt/vol) ammonium hydroxide (28%, density 0.9 g/ml, Sigma-Aldrich) to ~850ml of HPLC grade water, then add ~35 mL of 10 % (vol/vol) formic acid (>95% Sigma-Aldrich) to titrate the pH to 10.0; bring the final volume to 1 liter with HPLC grade pH to 10.0; bring the final volume to 1 liter with HPLC grade water. The 96 minute separation LC gradient followed this profile: (min: %B) 0:0; 7:0; 13:16; 73:40; 77:44; 82:60; 96:60. The flow rate was 1 mL/min. Per 3.3 mg separation, 82 fractions were rate was 1 mL/min. Per 3.3 mg separation, 82 fractions were collected into a 96 deep-well x 2mL plate (Whatman, #7701\\u20135200), with fractions combined in a step-wise concatenation strategy and acidified to a final concentration of 0.1% FA as reported and acidified to a final concentration of 0.1% FA as reported previously. An additional 14 fractions were collected from the 96 deep-well plate for fraction A, which are the early eluting fractions that tend to contain multi-phosphorylated peptides. 5% of that tend to contain multi-phosphorylated peptides. 5% of the volume of each of the 24+A proteome fractions was allocated for proteome analysis, dried down, and re-suspended in 3% MeCN/0.1% FA (MeCN; acetonitrile) to a peptide concentration of 0.25 \\u03bcg/ \\u03bcL (MeCN; acetonitrile) to a peptide concentration of 0.25 \\u03bcg/ \\u03bcL for LC-MS/MS analysis. The remaining 95% of concatenated 24 fractions were further combined into 12 fractions, with fraction A as a separate fraction. These 13 fractions were then enriched for a separate fraction. These 13 fractions were then enriched for phosphopeptides as described below. Phosphopeptide Enrichment Ni-NTA agarose beads were used to prepare Fe3+-NTA agarose beads. In each phosphoproteome fraction, ~261.25 \\u03bcg peptides (based on peptide-level BCA after digestion with uniformly-distributed fractionation presumed) were reconstituted in 522.5 \\u03bcL 80% fractionation presumed) were reconstituted in 522.5 \\u03bcL 80% MeCN/0.1% TFA (trifluoroacetic acid) solvent and incubated with 10 \\u03bcL of the IMAC beads for 30 minutes with end-over-end rotation at RT. After incubation, samples were briefly spun down on a at RT. After incubation, samples were briefly spun down on a tabletop centrifuge; clarified peptide flow-throughs were separated from the beads; and the beads were reconstituted in 200 \\u03bcL IMAC binding/wash buffer (80 MeCN/0.1% TFA) and loaded onto \\u03bcL IMAC binding/wash buffer (80 MeCN/0.1% TFA) and loaded onto equilibrated Empore C18 silica-packed stage tips (3M, 2315). Samples were then washed twice with 50 \\u03bcL of IMAC binding/wash buffer and once with 50 uL 1% FA, and were eluted from the IMAC buffer and once with 50 uL 1% FA, and were eluted from the IMAC beads to the stage tips with 3 \\u00d7 70 uL washes of 500 mM dibasic sodium phosphate (pH 7.0, Sigma S9763). Stage tips were then washed once with 100 \\u03bcL 1% FA and phosphopeptides were eluted from once with 100 \\u03bcL 1% FA and phosphopeptides were eluted from the stage tips with 60 \\u03bcL 50% MeCN/0.1% FA. Phosphopeptides were dried down and re-suspended in 9 \\u03bcL 50% MeCN/0.1%FA for LC-MS/MS analysis, where 4 \\u03bcL was injected per run. Acetylpeptide Enrichment Acetylated lysine peptides were enriched using an antibody against the acetyl-lysine motif (CST PTM-SCAN Catalogue No. 13416). IMAC eluents were concatenated into 4 fractions (~783.75 \\u03bcg peptides per fraction) and dried down using a SpeedVac apparatus. per fraction) and dried down using a SpeedVac apparatus. Peptide fractions were reconstituted with 1.4ml of IAP buffer (5 mM MOPS pH 7.2, 1 mM Sodium Phosphate (dibasic), 5 mM NaCl) per fraction and incubated for 2 hours at 4oC with pre-washed (3 times and incubated for 2 hours at 4oC with pre-washed (3 times with IAP buffer) agarose beads bound to acetyl-lysine motif antibody. Peptide-bound beads were washed 4 times with ice-cold PBS followed by elution with 2 \\u00d7 100ul of 0.15% TFA. Eluents were followed by elution with 2 \\u00d7 100ul of 0.15% TFA. Eluents were desalted using C18 stage tips, eluted with 50% ACN and dried down. Acetylpeptides were suspended in 7ul of 0.1% FA and 3% ACN and 4ul were injected per run. Ubiquitylpeptide Enrichment with on-bead TMT labeling Ubiquitin enrichment was performed based on the recently published UbiFast protocol. Before enrichment, anti-K-\\u03b5-GG bead-bound antibodies from the PTM-Scan ubiquitin remnant motif kit (Cell Signaling Technologies, Kit #5562) were cross-linked. Beads were Technologies, Kit #5562) were cross-linked. Beads were briefly washed 3X with 100 mM sodium borate (pH 9.0) and incubated with 20 mM DMP for 30 min at RT. Beads were then washed 2X with 200 mM ethanolamine and incubated overnight at 4\\u00b0C in 200 mM 200 mM ethanolamine and incubated overnight at 4\\u00b0C in 200 mM ethanolamine with end-over-end rotation. Following incubation, beads were washed 3X with immunoprecipitation (IAP) buffer (50 mM MOPS, pH 7.2, 10 mM sodium phosphate, 50 mM NaCl) and stored at MOPS, pH 7.2, 10 mM sodium phosphate, 50 mM NaCl) and stored at 4\\u00b0C at a concentration of 0.5 ug/uL. For each 11-plex experiment, 31.25 ug of cross-linked anti-K-GG bead-bound antibody at 0.5 ug/uL in IAP per channel was aliquoted into 1.5 mL Eppendorf tubes on ice. 750 ug peptide samples were reconstituted to 0.5 mg/mL concentration in IAP buffer and were reconstituted to 0.5 mg/mL concentration in IAP buffer and vortexed for 10 min. Peptides were then centrifuged for 5 min at 5,000g. Each peptide solution was added to a tube of antibody and gently rotated end-over-end at 4oC for 1 hour. Following and gently rotated end-over-end at 4oC for 1 hour. Following enrichment, samples were spun at 2000 rcf for ~1 min to settle and the supernatant was removed and stored as flow-through, and from this point beads were kept on ice unless being handled. from this point beads were kept on ice unless being handled. Antibody beads were washed with 1.5 mL ice cold IAP followed by 1.5 mL ice cold PBS. For all washes, after adding wash reagent each sample was inverted ~5 times, agitated by tapping pairs of each sample was inverted ~5 times, agitated by tapping pairs of tubes together, spun at 2000 rcf for about 30\\u201340 s, left to settle on ice for 10\\u201320 s, and supernatant was removed. Importantly, all washes were completed as quickly as possible. Importantly, all washes were completed as quickly as possible. Pre-aliquoted TMT @ 40 ug/uL in 100% MeCN was retrieved and briefly equilibrated to RT while beads were reconstituted in 200 uL room temp. 100 mM HEPES buffer. For each sample, 400 ug of TMT room temp. 100 mM HEPES buffer. For each sample, 400 ug of TMT labeling reagent in 10 uL acetonitrile was added. Samples were labeled on-antibody while shaking vigorously (1400 rpm) at 20\\u00b0C for 10 minutes, then quenched with 8 uL 5% hydroxylamine and for 10 minutes, then quenched with 8 uL 5% hydroxylamine and shaken vigorously for another 5 minutes. Labeled antibody-bound peptides were then put back on ice, washed once with 1.3 mL cold IAP, and again with 1.5 mL cold IAP. Each channel was resuspended and again with 1.5 mL cold IAP. Each channel was resuspended and transferred to a combination tube with 130 uL cold IAP. Following combination, each now-empty tube was serially washed with 1.5 mL cold IAP to remove remaining beads, and this 1.5 mL IAP was mL cold IAP to remove remaining beads, and this 1.5 mL IAP was added to the combination tube and used to wash the combined beads. Combined beads were washed one final time with 1.5 mL ice cold PBS. Once the channels were combined and washed, peptides were PBS. Once the channels were combined and washed, peptides were eluted twice from the beads by resuspending with 150 uL room temp. 0.15% TFA and incubating 5 min at RT. Each round of acid-eluted KGG-modified peptides was desalted on an equilibrated KGG-modified peptides was desalted on an equilibrated two-punch C18 stage tip. Both elutions of KGG peptides were loaded sequentially, washed 2X with 100 uL 0.1% FA, and eluted into an MS vial with 50 uL 50% ACN/0.1% FA. The eluted peptides were frozen, with 50 uL 50% ACN/0.1% FA. The eluted peptides were frozen, lyophilized, and reconstituted in 9 uL 3% ACN/0.1% FA, with 4 uL injected twice for two LC-MS/MS runs on a Thermo Lumos instrument. While the same lot of TMT reagents was used for all instrument. While the same lot of TMT reagents was used for all ubiquitylproteome samples (TE270748-TD264064), it is a different lot than that used for Proteome, Phosphoproteome, and acetylproteome samples. LC-MS/MS for Proteomics Analyses Online separation was done with a nanoflow Proxeon EASY-nLC 1200 UHPLC system (Thermo Fisher Scientific). In this set up, the LC system, column, and platinum wire used to deliver electrospray source voltage were connected via a stainless steel cross source voltage were connected via a stainless steel cross (360\\u03bcm, IDEX Health & Science, UH-906x). The column was heated to 50\\u00b0C using a column heater sleeve (Phoenix-ST) to prevent over-pressuring of columns during UHPLC separation. From each peptide of columns during UHPLC separation. From each peptide fraction, ~1ug (based on protein-level BCA prior to digestion with uniformly-distributed fractionation presumed), the equivalent of 12% of each global proteome sample in a 2 ul injection volume or 50% each global proteome sample in a 2 ul injection volume or 50% of each phosphoproteome, acetylproteome, or ubiquitylproteome sample in a 4 ul injection volume, was injected onto an in-house packed 22cm x 75um internal diameter C18 silica picofrit capillary 22cm x 75um internal diameter C18 silica picofrit capillary column (1.9 \\u03bcm ReproSil-Pur C18-AQ beads, Dr. Maisch GmbH, r119.aq; Picofrit 10um tip opening, New Objective, PF360\\u201375-10-N-5). Mobile phase flow rate was 200 nL/min, comprised of 3% Mobile phase flow rate was 200 nL/min, comprised of 3% acetonitrile/0.1% formic acid (Solvent A) and 90% acetonitrile /0.1% formic acid (Solvent B). The 110-minute LC-MS/MS method consisted of a 10-min column-equilibration procedure; a 20-min consisted of a 10-min column-equilibration procedure; a 20-min sample-loading procedure; and the following gradient profile: (min:%B) 0:2; 1:6; 85:30; 94:60; 95;90; 100:90; 101:50; 110:50 (the last two steps at 500 nL/min flow rate). For acetylproteome last two steps at 500 nL/min flow rate). For acetylproteome analysis, the same LC and column setup was used, but the gradient was extended to 260 minutes with the following gradient profile: (min:%B) 0:2; 1:6; 235:30; 244:60; 245;90; 250:90; 251:50; (min:%B) 0:2; 1:6; 235:30; 244:60; 245;90; 250:90; 251:50; 260:50 (the last two steps at 500 nL/min flow rate). For ubiquitylproteome analysis, the same LC and column setup was used, but the gradient was 154-minutes with the following gradient profile: gradient was 154-minutes with the following gradient profile: (min:%B) 0:2; 2:6; 122:35; 130:60; 133;90; 143:90; 144:50; 154:50 (the last two steps at 500 nL/min flow rate). For proteome and acetylproteome analysis, samples were analyzed with a benchtop Q Exactive HF-X mass spectrometer (Thermo Fisher Scientific) equipped with a nanoflow ionization source (James A. Hill Instrument Services, Arlington, MA). Data-dependent A. Hill Instrument Services, Arlington, MA). Data-dependent acquisition was performed using Q Exactive HF-X Orbitrap v 2.9 software in positive ion mode at a spray voltage of 1.5 kV. MS1 Spectra were measured with a resolution of 60,000, an AGC target of were measured with a resolution of 60,000, an AGC target of 3e6 and a mass range from 350 to 1800 m/z. The data-dependent mode cycle was set to trigger MS/MS on up to the top 20 most abundant precursors per cycle at an MS2 resolution of 45,000, an AGC precursors per cycle at an MS2 resolution of 45,000, an AGC target of 5e4, an isolation window of 0.7 m/z, a maximum injection time of 105 msec, and an HCD collision energy of 31%. Peptides that triggered MS/MS scans were dynamically excluded from further triggered MS/MS scans were dynamically excluded from further MS/MS scans for 45 sec for proteome or 30 sec for acetylproteome. Peptide match was set to preferred for monoisotopic peak determination, and charge state screening was enabled to only include and charge state screening was enabled to only include precursor charge states 2\\u20136, with an intensity threshold of 9.5e4 for proteome or 8e4 for acetylproteome. Advanced precursor determination feature (APD) (Myers et al., 2018) was turned off using a feature (APD) (Myers et al., 2018) was turned off using a software patch provided to us by Thermo Fisher Scientific allowing us to turn APD off in the tune file, Tune version 2.9.0.2926 (later versions of Exactive Tune 2.9 sp2 for the HFX have this option versions of Exactive Tune 2.9 sp2 for the HFX have this option as standard). For phosphoproteome and ubiquitylproteome analysis, samples were analyzed with a benchtop Orbitrap Fusion Lumos mass spectrometer (Thermo Fisher Scientific) equipped with a NanoSpray Flex NG ion source. Data-dependent acquisition was performed using NG ion source. Data-dependent acquisition was performed using Xcalibur Orbitrap Fusion Lumos v3.0 software in positive ion mode at a spray voltage of 1.8 kV. MS1 Spectra were measured with a resolution of 60,000, an AGC target of 4e5 and a mass range from of 60,000, an AGC target of 4e5 and a mass range from 350 to 1800 m/z. The data-dependent mode cycle time was set at 2 seconds with a MS2 resolution of 50,000, an AGC target of 6e4, an isolation window of 0.7 m/z, a maximum injection time of 105 msec (125 window of 0.7 m/z, a maximum injection time of 105 msec (125 msec for ubiquitylproteome), and an HCD collision energy of 36%. Peptide mode was selected for monoisotopic peak determination, and charge state screening was enabled to only include precursor charge state screening was enabled to only include precursor charge states 2\\u20136, with an intensity threshold of 1e4. Peptides that triggered MS/MS scans were dynamically excluded from further MS/MS scans for 45 sec (20 sec for ubiquitylproteome), with a MS/MS scans for 45 sec (20 sec for ubiquitylproteome), with a +/\\u2212 10 ppm mass tolerance. \\u201cPerform dependent scan on single charge state per precursor only\\u201d was enabled for phosphoproteome analysis, but disabled for ubiquitylproteome analysis. Immunohistochemistry (IHC) analysis.\\n\\nTMA construction Seventy-one LSCC paraffin tissue blocks were available for tissue microarray (TMA) construction and whole sections by immunohistochemistry (IHC) assay. TMAs were constructed in duplicate, and each included two (2) 1.0 mm cores providing up to four (4) and each included two (2) 1.0 mm cores providing up to four (4) cores per case for IHC analysis. Common control tissue on each TMA included patient-matched normal adjacent lung from 14 cases. Immune cell marker, Immunohistochemistry (IHC) Seventeen (17) samples were scored for immune cell markers. Sequential TMA sections were obtained for immune marker assessment to best approximate native spatial relationships. Chromogenic (DAB) IHC by single marker analysis was performed on 5-micron (DAB) IHC by single marker analysis was performed on 5-micron thick sections on standard charged slides. Automated immunostaining was performed by Autostainer Link 48, Dako, Inc. Heat induced epitope retrieval (HIER) was performed and biomarker dilutions epitope retrieval (HIER) was performed and biomarker dilutions as follows: CD4 clone 4B12, Bio-Rad 1:50; CD8 clone 4B11, Bio-Rad, 1:50; CD163, Abcam, 1:500 and ARHGDIB, HPA051235 Sigma-Aldrich, 1:50. IHC analysis was performed independently by three 1:50. IHC analysis was performed independently by three independent pathologists by staining intensity of 0\\u20133 scale for each TMA core separately and then by average score / case for a total of 15 cases. Case scores compared across independent pathologists 15 cases. Case scores compared across independent pathologists showed very good agreement. For evaluating statistical significance of IHC scores across the three immune clusters, ordinal regression was used, implemented using the \\u201cordinal\\u201d package in R. was used, implemented using the \\u201cordinal\\u201d package in R. Random effects accounted for the correlation between multiple cores from each sample and multiple pathologists. Multiplex Immunofluorescence (mIF) staining Development and optimization of the mIF platform has been previously described. mIF staining was performed in a 4-\\u03bcm-thick section obtained from a FFPE TMA block, using the Opal 7-Color Kit (Akoya Biosciences, Waltham, MA, USA) and scanned using a Vectra Biosciences, Waltham, MA, USA) and scanned using a Vectra multispectral microscope (Akoya Biosciences). The IF markers used were pan-Cytokeratin (CK, clone AE1/AE3, (DAKO) and Alpha-SMA ab5694 (Abcam). Multiplexed stained sections were imaged using the (Abcam). Multiplexed stained sections were imaged using the VECTRA multispectral imaging system (Akoya Biosciences). TMA cores were annotated using InForm 2.4.8 image analysis software (Akoya Biosciences). IHC based assessment of TP63 low samples (Figure 4) For samples with low TP63 status, additional pathologic confirmation was performed. Pathology reassessment confirmed 8/10 samples to be unequivocal LSCC; one sample (C3N.00247) had adenosquamous but predominantly squamous histology; the remaining sample but predominantly squamous histology; the remaining sample (C3N.02283) showed basaloid squamous cell features and despite some atypical characteristics retained its LSCC attribution after review by four separate pathologists - with histology very similar by four separate pathologists - with histology very similar to a TP63-low TCGA tumor (TCGA-66\\u20132754) deemed LSCC by TCGA pathology. QUANTIFICATION AND STATISTICAL ANALYSIS\\n\\nGenomic Data Analysis\\n\\nCopy Number Calling Copy-number analysis was performed jointly leveraging both whole-genome sequencing (WGS) and whole-exome sequencing (WES) data of the tumor and germline DNA, using CNVEX (https://github.com/mctp/cnvex). CNVEX uses whole-genome aligned reads to estimate CNVEX uses whole-genome aligned reads to estimate coverage within fixed genomic intervals, and whole-genome and whole-exome variant calls to compute B-allele frequencies at variable positions (we used TNScope germline calls). Coverages were computed in (we used TNScope germline calls). Coverages were computed in 10kb bins, and the resulting log coverage ratios between tumor and normal samples were adjusted for GC bias using weighted LOESS smoothing across mappable and non-blacklisted genomic intervals smoothing across mappable and non-blacklisted genomic intervals within the GC range 0.3\\u20130.7, with a span of 0.5 (the target, blacklist, and configuration files are provided with CNVEX). The adjusted log coverage ratios (LR) and B-allele frequencies (BAF) log coverage ratios (LR) and B-allele frequencies (BAF) were jointly segmented by custom algorithm based on Circular Binary Segmentation (CBS). Alternative probabilistic algorithms were implemented in CNVEX, including algorithms based on recursive binary in CNVEX, including algorithms based on recursive binary segmentation (RBS), and dynamic programming, as implemented in the R-package jointseg. For the CBS-based algorithm, first LR and mirrored BAF were independently segmented using CBS (parameters mirrored BAF were independently segmented using CBS (parameters alpha=0.01, trim=0.025) and all candidate breakpoints collected. The resulting segmentation track was iteratively \\u201cpruned\\u201d by merging segments that had similar LR, BAFs and short lengths. For segments that had similar LR, BAFs and short lengths. For the RBS- and DP-based algorithms, joint-breakpoints were \\u201cpruned\\u201d using a statistical model selection method. For the final set of CNA segments, we chose the CBS-based results as they did not of CNA segments, we chose the CBS-based results as they did not require specifying a prior on the number of expected segments (K) per chromosome arm, were robust to unequal variances between the LR and BAF tracks, and provided empirically the best fit to the LR and BAF tracks, and provided empirically the best fit to the underlying data. Somatic Variant Calling Somatic variant calling was performed on exome-capture BAM files aligned to the GDC GRCh38 reference. Following de-duplication and co-realignment around known indels \\u201cGATK bundle known\\u201d and \\u201cMills 1000G\\u201d somatic variants were called using Sentieon TNScope 1000G\\u201d somatic variants were called using Sentieon TNScope with the following settings \\u201c--max_fisher_pv_active 0.05 --min_tumor_allele_frac 0.0075 --min_init_tumor_lod 2.5 --assemble_mode 4 --SVIntrusionThres 5\\u201d, with a provided germline variants from 4 --SVIntrusionThres 5\\u201d, with a provided germline variants from dbSNP138. The resulting VCF files were annotated and filtered, in a series of post-processing steps which included: 1) restriction of variants to the hybrid-capture target regions. 2) Phasing of variants to the hybrid-capture target regions. 2) Phasing of adjacent and separated by 1bp SNVs into multi-nucleotide variants based on the haplotype information provided by TNScope. 3) Annotation using VEP with the following settings \\u201c--assembly 3) Annotation using VEP with the following settings \\u201c--assembly GRCh38 --species homo_sapiens --cache_version 97 --format vcf --gene_phenotype --symbol --canonical --ccds --hgvs --biotype --tsl --uniprot --domains --appris --protein --variant_class --sift --uniprot --domains --appris --protein --variant_class --sift b, --polyphen b --no_stats --total_length --allele_number --no_escape --flag_pick_allele --pick_order canonical,tsl,biotype,rank,ccds,length --buffer_size 20000\\u201d. Additional annotations were --buffer_size 20000\\u201d. Additional annotations were attached to the variants using VcfAnno including allele frequencies in GNOMAD (v2.0.1), ClinVar (downloaded Dec. 2019), Cosmic (downloaded Dec. 2019), dbSNP (20180418). The resulting variants are annotated 2019), dbSNP (20180418). The resulting variants are annotated against both RefSeq and Ensembl transcripts and contain multiple predicted consequences per-variant. For most downstream applications these annotations were reduced to a single predicted these annotations were reduced to a single predicted consequence per-variant and converted into a MAF file using custom software. The choice of the most-relevant consequence was based on a number of features. In general, the most \\u2018severe\\u2019 consequence a number of features. In general, the most \\u2018severe\\u2019 consequence occuring in a \\u2018reliable\\u2019 transcript was chosen. To accomplish this the computed consequences were ranked by \\u2018severity\\u2019 and \\u2018priority\\u2019 reflecting how deleterious/impactful a variant and \\u2018priority\\u2019 reflecting how deleterious/impactful a variant consequence is and how well-supported the existence and biological relevance of an isoform is. The specific rules differed for RefSeq and Ensembl annotations. RefSeq transcripts were ranked RefSeq and Ensembl annotations. RefSeq transcripts were ranked higher if they were known to CCDS, LRG or ranked \\u2018CANONICAL\\u2019 or \\u2018PICK\\u2019 by VEP. Ensembl transcripts were further prioritized by Basic, Appris and TSL annotations. Ensembl-based consequences Basic, Appris and TSL annotations. Ensembl-based consequences were prefered in case of ties. The resulting variant sets were filtered to only include variants with a high likelihood of being somatic based on a set of probabilistic and heuristic of being somatic based on a set of probabilistic and heuristic thresholds. A reported variant had to pass all of the following filters: 5 or more variant reads, fewer than 2 variant reads in the germline allele frequency in the normal 0.005, allele in the germline allele frequency in the normal 0.005, allele frequency in the tumor greater than 0.025, TLOD > 5, NLOD > 6, Fisher\\u2019s test p-value < 0.05, NFLOD > 1.5. Additionally heuristic filters were implemented to remove strand-bias, variants with filters were implemented to remove strand-bias, variants with mapping quality bias, and variants with high population allele frequency in GNOMAD. Germline Variant Calling Germline variant calling and annotation was carried out analogously using DNAScope using the following settings \\u201c--emit_conf 10 --call_conf 10\\u201d to generate a VCF file and \\u201c--emit_mode GVCF\\u201d to generate a gVCF file. The variants were custom-filtered based generate a gVCF file. The variants were custom-filtered based on the specific applications. GISTIC and MutSig analysis The Genomic Identification of Significant Targets in Cancer (GISTIC2.0) algorithm was used to identify significantly amplified or deleted focal-level and arm-level events, with Q value <0.25 considered significant. The following parameters were used: Amplification Threshold = 0.1\\n\\nDeletion Threshold = 0.1\\n\\nCap Values = 1.5\\n\\nBroad Length Cutoff = 0.98\\n\\nRemove X-Chromosome = 0\\n\\nConfidence Level = 0.99\\n\\nJoin Segment Size = 4\\n\\nArm Level Peel Off = 1\\n\\nMaximum Sample Segments = 2000\\n\\nGene GISTIC = 1 Each gene of every sample is assigned a thresholded copy number level that reflects the magnitude of its deletion or amplification. These are integer values ranging from \\u22122 to 2, where 0 means no amplification or deletion of magnitude greater than the no amplification or deletion of magnitude greater than the threshold parameters described above. Amplifications are represented by positive numbers: 1 means amplification above the amplification threshold; 2 means amplification larger than the arm level threshold; 2 means amplification larger than the arm level amplifications observed in the sample. Deletions are represented by negative numbers: \\u22121 means deletion beyond the threshold; \\u22122 means deletions greater than the minimum arm-level copy number means deletions greater than the minimum arm-level copy number observed in the sample. The somatic variants were filtered through a panel of normals to remove potential sequencing artifacts and undetected germline variants. MutSig2CV was run on these filtered results to evaluate the significance of mutated genes and estimate mutation the significance of mutated genes and estimate mutation densities of samples. These results were constrained to genes in the Cancer Gene Census, with false discovery rates (q values) recalculated. Genes of q value < 0.1 were declared significant. RNAseq and miRNAseq Quantification\\n\\nRNAseq Quantification The readcount is generated by featureCounts (subread v1.6.4) under stranded mode with parameters: `-g gene_id -t exon -Q 10 -p -B -s 2`. Gene annotation in use is identical to GDC (`gencode.v22.annotation.gtf.gz`; md5: `291330bdcff1094bc4d5645de35e0871`), md5: `291330bdcff1094bc4d5645de35e0871`), which is available on the GDC Reference Files page (https://gdc.cancer.gov/about-data/gdc-data-processing/gdc-reference-files). The readcount is later converted to FPKM and FPKM-UQ using GDC\\u2019s formula is later converted to FPKM and FPKM-UQ using GDC\\u2019s formula (https://docs.gdc.cancer.gov/Data/Bioinformatics_Pipelines/Expression_mRNA_Pipeline/). Isoform specific RNA quantification For TP63 isoform centric analysis presented in Figure 4, the hg38 reference genome and RefSeq annotations were used for the RNAseq data analysis and were downloaded from the UCSC table browser. First, CIRI (v2.0.6) was used to call circular RNA with First, CIRI (v2.0.6) was used to call circular RNA with default parameters and BWA (version 0.7.17-r1188) was used as the mapping tool. The cutoff of supporting reads for circRNAs was set to 10. Then we used a pseudo-linear transcript strategy to quantify Then we used a pseudo-linear transcript strategy to quantify gene and circular RNA expression. In brief, for each sample, linear transcripts of circular RNAs were extracted and 75bp (read length) from the 3\\u2019 end was copied to the 5\\u2019 end. The modified length) from the 3\\u2019 end was copied to the 5\\u2019 end. The modified transcripts were called pseudo-linear transcripts. Transcripts of linear genes were also extracted and mixed with pseudo-linear transcripts. RSEM (version 1.3.1) with Bowtie2 (version 2.3.3) transcripts. RSEM (version 1.3.1) with Bowtie2 (version 2.3.3) as the mapping tool was used to quantify gene, isoform, and circular RNA expression based on the mixed transcripts. DNA methylation data preprocessing Raw methylation image files were downloaded from the CPTAC DCC (See data availability). We calculated and analyzed methylated (M) and unmethylated (U) intensities for LSCC samples as described previously. We flagged locus as NA where probes did not meet a previously. We flagged locus as NA where probes did not meet a detection p-value of 0.01. Probes with MAF more than 0.1 were removed, and samples with more than 85% NA values were removed. Resulting beta values of methylation were utilized for subsequent beta values of methylation were utilized for subsequent analysis. Gene-level methylation scores were generated by taking the mean beta values of probes in the CpG islands of promoters and 5\\u2019 UTR regions of the gene. Methylation profiles (i.e., density plots) of some samples had unexpectedly skewed distributions of plots) of some samples had unexpectedly skewed distributions of methylation beta values, in addition to significantly more missing values. To systematically determine the subset of methylation samples with these evident data QC issues, we subjected all samples with these evident data QC issues, we subjected all the samples to model-based clustering using the Mclust package in R, using the interquartile range over all the genes as the representative metric. The clustering automatically determined the metric. The clustering automatically determined the optimal number of clusters, and identified 3 clusters. Two of these clusters (with centroids at 0.029 and 0.038) captured the bulk of the samples (202). The third cluster (centroid at 0.165, of the samples (202). The third cluster (centroid at 0.165, significantly higher than the other two clusters) consisted of 4 samples that belong to this cluster with high confidence (uncertainty < 5%), each of which had a skewed distribution of beta < 5%), each of which had a skewed distribution of beta values with a mean of 4,407 missing values per sample (in contrast to 1.8 missing values per sample for clusters 1 and 2 combined). Based on this analysis, we concluded that the 4 samples in cluster 3 on this analysis, we concluded that the 4 samples in cluster 3 represent samples with poor data quality. These have been excluded from all methylation analysis. miRNA-seq Data Analysis miRNA-seq unaligned bam files were downloaded from the CPTAC GDC API (https://docs.gdc.cancer.gov). Unaligned bams were first converted to fastq.gz by samtools bam2fq (samtools version 1.9). TPM (Transcripts per million) values of mature miRNA and 1.9). TPM (Transcripts per million) values of mature miRNA and precursor miRNA were reported after adaptor trimming, quality check, alignment, annotation, and read counting (https://github.com/ding-lab/CPTAC_miRNA/blob/master/cptac_mirna_analysis.md). The The mature miRNA expression was calculated irrespective of its gene of origin by summing the expression. Proteomics Data Analysis\\n\\nSpectrum quality filtering and searching All MS data were interpreted using the Spectrum Mill software package v7.0 pre-release (Agilent Technologies, Santa Clara, CA) co-developed by Karl Clauser of the Carr laboratory (https://www.broadinstitute.org/proteomics). Similar MS/MS spectra acquired Similar MS/MS spectra acquired on the same precursor m/z within +/\\u2212 45 sec were merged. MS/MS spectra were excluded from searching if they failed the quality filter by not having a sequence tag length > 0 (i.e., minimum of two masses separated by the tag length > 0 (i.e., minimum of two masses separated by the in-chain mass of an amino acid) or did not have a precursor MH+ in the range of 800\\u20136000. MS/MS spectra were searched against a RefSeq-based sequence database containing 41,457 proteins mapped sequence database containing 41,457 proteins mapped to the human reference genome (GRCh38/hg38) obtained via the UCSC Table Browser (https://genome.ucsc.edu/cgi-bin/hgTables) on June 29, 2018, with the addition of 13 proteins encoded in the human 29, 2018, with the addition of 13 proteins encoded in the human mitochondrial genome, 264 common laboratory contaminant proteins, and 553 non-canonical small open reading frames. Scoring parameters were ESI-QEXACTIVE-HCD-v2, for whole proteome datasets, were ESI-QEXACTIVE-HCD-v2, for whole proteome datasets, and ESI-QEXACTIVE-HCD-v3, for phosphoproteome, acetylproteome, and ubiquitylproteome datasets. All spectra were allowed +/\\u2212 20 ppm mass tolerance for precursor and product ions, 30% minimum matched tolerance for precursor and product ions, 30% minimum matched peak intensity, and \\u201ctrypsin allow P\\u201d enzyme specificity with up to 4 missed cleavages. Allowed fixed modifications included carbamidomethylation of cysteine and selenocysteine. TMT labeling of cysteine and selenocysteine. TMT labeling was required at lysine, but peptide N-termini were allowed to be either labeled or unlabeled. Allowed variable modifications for whole proteome datasets were acetylation of protein N-termini, oxidized datasets were acetylation of protein N-termini, oxidized methionine, deamidation of asparagine, hydroxylation of proline in PG motifs, pyroglutamic acid at peptide N-terminal glutamine, and pyro-carbamidomethylation at peptide N-terminal cysteine with a pyro-carbamidomethylation at peptide N-terminal cysteine with a precursor MH+ shift range of \\u221218 to 97 Da. For the phosphoproteome dataset the allowed variable modifications were revised to allow phosphorylation of serine, threonine, and tyrosine, allow allow phosphorylation of serine, threonine, and tyrosine, allow deamidation only in NG motifs, and disallow hydroxylation of proline with a precursor MH+ shift range of \\u221218 to 272 Da. For the acetylproteome dataset the allowed variable modifications were acetylproteome dataset the allowed variable modifications were revised to allow acetylation of lysine, allow deamidation only in NG motifs, and disallow hydroxylation of proline with a precursor MH+ shift range of \\u2212400 to 70 Da. For the ubiquitylproteome MH+ shift range of \\u2212400 to 70 Da. For the ubiquitylproteome dataset the allowed variable modifications were revised to allow diglycine modification of lysine, allow deamidation only in NG motifs, and disallow hydroxylation of proline with a precursor MH+ and disallow hydroxylation of proline with a precursor MH+ shift range of \\u2212690 to 70 Da. Protein grouping, and localization of PTMs Identities interpreted for individual spectra were automatically designated as confidently assigned using the Spectrum Mill autovalidation module to use target-decoy based false discovery rate (FDR) estimates to apply score threshold criteria. For the rate (FDR) estimates to apply score threshold criteria. For the whole proteome dataset thresholding was done in 3 steps: at the peptide spectrum match (PSM) level, the protein level for each TMT-plex, and the protein level for all 22 TMT-plexes. For the TMT-plex, and the protein level for all 22 TMT-plexes. For the phosphoproteome, acetylproteome, and ubiquitylproteome datasets thresholding was done in two steps: at the PSM and variable modification (VM) site levels. In step 1 for all datasets, PSM-level autovalidation was done first and separately for each TMT-plex experiment consisting of either 25 LC-MS/MS runs (whole proteome), 13 LC-MS/MS runs (phosphoproteome), 4 LC-MS/MS runs (acetylproteome), or 2 LC-MS/MS 4 LC-MS/MS runs (acetylproteome), or 2 LC-MS/MS runs (ubiquitylproteome) using an auto-thresholds strategy with a minimum sequence length of 7; automatic variable range precursor mass filtering; and score and delta Rank1 \\u2013 Rank2 score thresholds optimized and score and delta Rank1 \\u2013 Rank2 score thresholds optimized to yield a PSM-level FDR estimate for precursor charges 2 through 4 of <0.8% for each precursor charge state in each LC-MS/MS run. To achieve reasonable statistics for precursor charges 5\\u20136, To achieve reasonable statistics for precursor charges 5\\u20136, thresholds were optimized to yield a PSM-level FDR estimate of <0.4% across all runs per TMT-plex experiment (instead of per each run), since many fewer spectra are generated for the higher run), since many fewer spectra are generated for the higher charge states. In step 2 for the whole proteome dataset, protein-polishing autovalidation was applied separately to each TMTplex experiment to further filter the PSMs using a target protein-level FDR threshold of zero. The primary goal of this step was to eliminate of zero. The primary goal of this step was to eliminate peptides identified with low scoring PSMs that represent proteins identified by a single peptide, so-called \\u201cone-hit wonders.\\u201d After assembling protein groups from the autovalidated PSMs, protein assembling protein groups from the autovalidated PSMs, protein polishing determined the maximum protein level score of a protein group that consisted entirely of distinct peptides estimated to be false-positive identifications (PSMs with negative delta to be false-positive identifications (PSMs with negative delta forward-reverse scores). PSMs were removed from the set obtained in the initial peptide-level autovalidation step if they contributed to protein groups that had protein scores below the contributed to protein groups that had protein scores below the maximum false-positive protein score. Step 3 was then applied, consisting of protein-polishing autovalidation across all TMT plexes together using the protein grouping method \\u201cexpand TMT plexes together using the protein grouping method \\u201cexpand subgroups, top uses shared\\u201d to retain protein subgroups with either a minimum protein score of 25 or observation in at least 4 TMT plexes. The primary goal of this step was to eliminate low TMT plexes. The primary goal of this step was to eliminate low scoring proteins that were infrequently detected in the sample cohort. As a consequence of these two protein-polishing steps, each identified protein reported in the study comprised multiple identified protein reported in the study comprised multiple peptides, unless a single excellent scoring peptide was the sole match and that peptide was observed in at least 4 TMT-plexes. In calculating scores at the protein level and reporting the In calculating scores at the protein level and reporting the identified proteins, peptide redundancy was addressed in Spectrum Mill as follows: The protein score was the sum of the scores of distinct peptides. A distinct peptide was the single highest of distinct peptides. A distinct peptide was the single highest scoring instance of a peptide detected through an MS/MS spectrum. MS/MS spectra for a particular peptide may have been recorded multiple times (e.g. as different precursor charge states, in multiple times (e.g. as different precursor charge states, in adjacent bRP fractions, modified by deamidation at Asn or oxidation of Met, or with different phosphosite localization), but were still counted as a single distinct peptide. When a peptide were still counted as a single distinct peptide. When a peptide sequence of >8 residues was contained in multiple protein entries in the sequence database, the proteins were grouped together and the highest scoring one and its accession number were and the highest scoring one and its accession number were reported. In some cases when the protein sequences were grouped in this manner, there were distinct peptides that uniquely represent a lower scoring member of the group (isoforms, family members, a lower scoring member of the group (isoforms, family members, and different species). Each of these instances spawned a subgroup. Multiple subgroups were reported, counted towards the total number of proteins, and given related protein subgroup numbers number of proteins, and given related protein subgroup numbers (e.g. 3.1 and 3.2 for group 3, subgroups 1 and 2). For the whole proteome datasets the above criteria yielded false discovery rates (FDR) for each TMT-plex experiment of <0.6% at the rates (FDR) for each TMT-plex experiment of <0.6% at the peptide-spectrum match level and <0.8% at the distinct peptide level. After assembling proteins with all the PSMs from all the TMT-plex experiments together, the aggregate FDR estimates were 0.58% experiments together, the aggregate FDR estimates were 0.58% at the peptide-spectrum match level, 2.3% at the distinct peptide level, and <0.01% (1/11,029) at the protein group level. Since the protein level FDR estimate neither explicitly required a the protein level FDR estimate neither explicitly required a minimum number of distinct peptides per protein nor adjusted for the number of possible tryptic peptides per protein, it may underestimate false positive protein identifications for large underestimate false positive protein identifications for large proteins observed only on the basis of multiple low scoring PSMs. In step 2 for the phosphoproteome and acetylproteome datasets, variable modification (VM) site polishing autovalidation was applied across all 22 TMT plexes to retain all VM-site identifications with either a minimum id score of 8.0 or observation in at with either a minimum id score of 8.0 or observation in at least 4 TMT plexes. For the ubiquitylproteome datasets, it was applied across all 15 TMT plexes to retain all VM-site identifications with either a minimum id score of 8.0 or observation in at with either a minimum id score of 8.0 or observation in at least 3 TMT plexes. The intention of the VM site polishing step is to control FDR by eliminating unreliable VM site-level identifications, particularly low scoring VM sites that are only detected particularly low scoring VM sites that are only detected as low scoring peptides that are also infrequently detected across all of the TMT plexes in the study. In calculating scores at the VM-site level and reporting the identified VM sites, redundancy VM-site level and reporting the identified VM sites, redundancy was addressed in Spectrum Mill as follows: A VM-site table was assembled with columns for individual TMT-plex experiments and rows for individual VM-sites. PSMs were combined into a single rows for individual VM-sites. PSMs were combined into a single row for all non-conflicting observations of a particular VM-site (e.g. different missed cleavage forms, different precursor charges, confident and ambiguous localizations, and different charges, confident and ambiguous localizations, and different sample-handling modifications). For related peptides, neither observations with a different number of VM-sites nor different confident localizations were allowed to be combined. Selecting the localizations were allowed to be combined. Selecting the representative peptide from the combined observations was done such that once confident VM-site localization was established, higher identification scores and longer peptide lengths were preferred. scores and longer peptide lengths were preferred. While a Spectrum Mill identification score was based on the number of matching peaks, their ion type assignment, and the relative height of unmatched peaks, the VM site localization score was the of unmatched peaks, the VM site localization score was the difference in identification score between the top two localizations. The score threshold for confident localization, >1.1, essentially corresponded to at least 1 b or y ion located between two corresponded to at least 1 b or y ion located between two candidate sites that has a peak height >10% of the tallest fragment ion (neutral losses of phosphate from the precursor and related ions as well as immonium and TMT reporter ions were excluded from as well as immonium and TMT reporter ions were excluded from the relative height calculation). The ion type scores for b-H3PO4, y-H3PO4, b-H2O, and y-H2O ion types were all set to 0.5. This prevented inappropriate confident localization assignment when a inappropriate confident localization assignment when a spectrum lacked primary b or y ions between two possible sites but contained ions that could be assigned as either phosphate-loss ions for one localization or water loss ions for another localization. one localization or water loss ions for another localization. VM-site polishing yielded 68, 674 phosphosites with an aggregate FDR of 0.73% at the phosphosite level. In aggregate, 71% of the reported phosphosites in this study were fully localized to a reported phosphosites in this study were fully localized to a particular serine, threonine, or tyrosine residue. VM-site polishing yielded 15,186 acetylsites with an aggregate FDR of 0.80% at the acetylsite level. In aggregate, 99% of the reported at the acetylsite level. In aggregate, 99% of the reported acetylsites in this study were fully localized to a particular lysine residue. VM-site polishing yielded 25,489 ubiquitylsites with an aggregate FDR of 0.40% at the ubiquitylsite level. In with an aggregate FDR of 0.40% at the ubiquitylsite level. In aggregate, >99% of the reported ubiquitylsites in this study were fully localized to a particular lysine residue. Quantitation using TMT ratios Using the Spectrum Mill Protein/Peptide Summary module, a protein comparison report was generated for the proteome dataset using the protein grouping method \\u201cexpand subgroups, top uses shared\\u201d (SGT). For the phosphoproteome and acetylproteome datasets a (SGT). For the phosphoproteome and acetylproteome datasets a Variable Modification site comparison report limited to either phospho or acetyl sites, respectively, was generated using the protein grouping method \\u201cunexpand subgroups\\u201d. Relative abundances of grouping method \\u201cunexpand subgroups\\u201d. Relative abundances of proteins and VM-sites were determined in Spectrum Mill using TMT reporter ion intensity ratios from each PSM. TMT reporter ion intensities were corrected for isotopic impurities in the Spectrum were corrected for isotopic impurities in the Spectrum Mill Protein/Peptide summary module using the afRICA correction method, which implements determinant calculations according to Cramer\\u2019s Rule and correction factors obtained from the reagent Cramer\\u2019s Rule and correction factors obtained from the reagent manufacturer\\u2019s certificate of analysis (https://www.thermofisher.com/order/catalog/product/90406) for TMT11_lot number UA280170/TL272399 (Proteome, Phosphoproteme, and Acetylproteome) and (Proteome, Phosphoproteme, and Acetylproteome) and TE270748-TD264064 (Ubiquitylproteome). A protein-level, phosphosite-level, or acetylsite-level TMT ratio is calculated as the median of all PSM-level ratios contributing to a protein subgroup, of all PSM-level ratios contributing to a protein subgroup, phosphosite, or acetylsite. PSMs were excluded from the calculation if they lacked a TMT label, had a precursor ion purity < 50% (MS/MS has significant precursor isolation contamination from (MS/MS has significant precursor isolation contamination from co-eluting peptides), or had a negative delta forward-reverse identification score (half of all false-positive identifications). Lack of TMT label led to exclusion of PSMs per TMT plex with a Lack of TMT label led to exclusion of PSMs per TMT plex with a range of 1.4 to 2.3% for the proteome, 1.8 to 3.7% for the phosphoproteome, 1.0 to 3.5% for the acetylproteome, and 4.5 to 18.0% for the ubiquitylproteome datasets. Low precursor ion purity for the ubiquitylproteome datasets. Low precursor ion purity led to exclusion of PSMs per TMT plex with a range of 1.1 to 1.4% for the proteome, 1.7 to 2.8% for the phosphoproteome, 0.5 to 1.7% for the acetylproteome, and 2.8 to 3.5% for the 0.5 to 1.7% for the acetylproteome, and 2.8 to 3.5% for the ubiquitylproteome datasets. Two-component normalization of TMT ratios It was assumed that for every sample there would be a set of unregulated proteins, phosphosites, acetylsites or ubiquitylsites that have abundance comparable to the common reference (CR) sample. In the normalized sample, these proteins, phosphosites, sample. In the normalized sample, these proteins, phosphosites, acetylsites or ubiquitylsites should have a log TMT ratio centered at zero. In addition, there were proteins, phosphosites, acetylsites and ubiquitylsites that were either up- or acetylsites and ubiquitylsites that were either up- or down-regulated compared to the CR. A normalization scheme was employed that attempted to identify the unregulated proteins, phosphosites, acetylsites or ubiquitylsites, and centered the distribution acetylsites or ubiquitylsites, and centered the distribution of these log-ratios around zero in order to nullify the effect of differential protein loading and/or systematic MS variation. A 2-component Gaussian mixture model-based normalization algorithm Gaussian mixture model-based normalization algorithm was used to achieve this effect. The two Gaussians N(\\u03bci1,\\u03c3i1) and N(\\u03bci2,\\u03c3i2) for a sample i were fitted and used in the normalization process as follows: the mode mi of the log-ratio distribution was as follows: the mode mi of the log-ratio distribution was determined for each sample using kernel density estimation with a Gaussian kernel and Shafer-Jones bandwidth. A two-component Gaussian mixture model was then fit with the mean of both Gaussians mixture model was then fit with the mean of both Gaussians constrained to be mi, i.e., \\u03bci1=\\u03bci2=mi. The Gaussian with the smaller estimated standard deviation  was assumed to represent the unregulated component of was assumed to represent the unregulated component of proteins/phosphosites/acetylsites/ubiquitylsites, and was used to normalize the sample. The sample was standardized using (mi,\\u03c3i) by subtracting the mean mi from each standardized using (mi,\\u03c3i) by subtracting the mean mi from each protein/phosphosite/acetylsite/ubiquitylsite and dividing by the standard deviation \\u03c3i. Normalization of acetyl- and ubiquitylproteome To correct for differential underlying protein levels when performing certain acetylproteome- and ubiquitylproteome-based analyses, a global linear model approach was employed to produce protein-normalized versions of both of these PTM datasets. PTM sites versions of both of these PTM datasets. PTM sites were first matched to underlying proteins in the proteome dataset using RefSeq accession numbers. If a site\\u2019s main accession number did not yield a match in the proteome, all accession numbers associated yield a match in the proteome, all accession numbers associated with the site were then tried and the protein with the highest score was selected as a match. For the acetylproteome and ubiquitylproteome separately, a linear model was then fit (R function separately, a linear model was then fit (R function lm, PTM ~ protein) to all matched PTM-protein data points in the dataset. The residuals of each model were then used as protein-normalized acetylsite or ubiquitylsite abundances. Proteogenomic analysis\\n\\nQuality control and Batch effect assessment We used guided principal component analysis (gPCA) to assess the presence of a batch effect in the sample/CR ratio data based on TMT-plex. There was no overall apparent batch effect (gPCA p-value=1) in the global proteome, phosphoproteome, acetylproteome in the global proteome, phosphoproteome, acetylproteome or ubiquitylproteome datasets. To assess composition similarities between CR used for proteome, phosphoproteome and acetylproteome analysis, and ubiquitylproteome, we performed label-free proteomic analysis of each CRs in triplicates. Both CRs showed excellent correlation with median triplicates. Both CRs showed excellent correlation with median pearson correlation > 0.85. In addition, to assess data quality, we assessed the Pearson correlations of log TMT reporter ion intensities between all pairs of TMT plexes, separately for sample and CR channels. For each given pair of TMT plexes, correlations between CRs were each given pair of TMT plexes, correlations between CRs were consistently higher than correlations of respective sample channels enabling precise relative quantitation of proteins, phosphorylation and acetylation sites across 22 TMT plexes using the CR and acetylation sites across 22 TMT plexes using the CR channels as common denominator (Figure S1C). In the ubiquitylproteome data we observed lower correlations of the CR channel (in comparison to samples channels) in three of the 18 TMT plexes analyzed to samples channels) in three of the 18 TMT plexes analyzed (plexes 7, 8, 10). Clustering of the CR channels using pairwise correlations as a similarity metric pointed to a batch effect in the CR channels (but not in the sample channels) in these plexes. CR channels (but not in the sample channels) in these plexes. The presence of a batch effect was further confirmed by applying the gPCA test described above to only the reporter intensities of the CR channels (gPCA p-value < 0.001). Even though of the CR channels (gPCA p-value < 0.001). Even though undetectable with a global test, we observed this CR batch effect propagate to the sample/CR ratio data in preliminary analyses (data not shown). To counter the propagation of this CR batch effect to shown). To counter the propagation of this CR batch effect to the sample/CR ratio for samples in these plexes, batch correction was applied to the sample/CR ratio data for plexes 7, 8, 10 (batch 1) and all other plexes (batch 2) using limma\\u2019s 7, 8, 10 (batch 1) and all other plexes (batch 2) using limma\\u2019s removeBatchEffect (https://rdrr.io/bioc/limma/man/removeBatchEffect.html) on the sample/CR ratio data including the sample type (tumor, NAT) as a covariate. Dataset filtering Genes (RNA-seq), proteins (global proteome), phosphosites, acetylsites and ubiquitin sites present in fewer than 30% of samples (i.e., missing in >70% of samples) were removed from the respective datasets. Furthermore, proteins were required to have at datasets. Furthermore, proteins were required to have at least two observed TMT ratios in >25% of samples in order to be included in the proteome dataset. Phosphosites, acetylsites and ubiquitin sites were required to have at least one observed TMT ratio sites were required to have at least one observed TMT ratio in >25% of samples. Some of the filtering steps were modified for specific analyses in the study. For many of the marker selection and gene set enrichment analyses, at least 50% of samples were required to have non-missing values for proteins/phosphosites/acetyl/ubiquitin non-missing values for proteins/phosphosites/acetyl/ubiquitin sites, since missing values were imputed, and excessive missing values can result in poor imputation. Alternate filtering has been noted in descriptions of the relevant methods. Unsupervised multi-omic clustering using NMF We used non-negative matrix factorization (NMF)-based multi-omic clustering using protein, phosphosite, acetylsite, RNA transcript and gene copy number variants (CNV) as previously described (. Briefly, given a factorization rank k (where k is the number (. Briefly, given a factorization rank k (where k is the number of clusters), NMF decomposes a p x n data matrix V into two matrices W and H such that multiplication of W and H approximates V. Matrix H is a k x n matrix whose entries represent weights for Matrix H is a k x n matrix whose entries represent weights for each sample (1 to N) to contribute to each cluster (1 to k), whereas matrix W is a p x k matrix representing weights for each feature (1 to p) to contribute to each cluster (1 to k). Matrix H (1 to p) to contribute to each cluster (1 to k). Matrix H was used to assign samples to clusters by choosing the k with maximum score in each column of H. For each sample we calculated a cluster membership score as the maximal fractional score of the cluster membership score as the maximal fractional score of the corresponding column in matrix H. We defined a \\u201dcluster core\\u201d as the set of samples with cluster membership score > 0.5. To enable integrative multi-omic clustering we enforced all data types (and converted if necessary) to represent ratios to: (i) a common reference measured in each TMT plex (proteome, phosphoproteome, acetylproteome), (ii) an in silico common reference acetylproteome), (ii) an in silico common reference calculated as the median abundance across all samples (RNA expression) or (iii) to matching blood normal for CNA data. The CNA data was further filtered to only retain genes significantly altered was further filtered to only retain genes significantly altered (GISTIC2 thresholded of +2 or \\u22122) in at least 5% of all tumors. All data tables were then concatenated and all rows with missing values were removed. To remove uninformative features from the values were removed. To remove uninformative features from the dataset prior to NMF clustering we removed features with the lowest standard deviation (bottom 5th percentile) across all samples. Each column in the data matrix was further scaled and samples. Each column in the data matrix was further scaled and standardized such that all features from different data types were represented as z-scores. The resulting data matrix of z-scores into was converted to a non-negative input matrix required by into was converted to a non-negative input matrix required by NMF as follows: Create one data matrix with all negative numbers zeroed.\\n\\nCreate another data matrix with all positive numbers zeroed and the signs of all negative numbers removed. Concatenate both matrices resulting in a data matrix twice as large as the original, but containing only positive values and zeros and hence appropriate for NMF. The resulting matrix was then subjected to NMF analysis leveraging the NMF R-package and using the factorization method described in. To determine the optimal factorization rank k (number of clusters) for the multi-omic data matrix we tested a range of clusters) for the multi-omic data matrix we tested a range of clusters between k=2 and 8. For each k we factorized matrix V using 50 iterations with random initializations of W and H. To determine the optimal factorization rank we calculated two metrics the optimal factorization rank we calculated two metrics for each k: 1) cophenetic correlation coefficient measuring how well the intrinsic structure of the data was recapitulated after clustering and 2) the dispersion coefficient of the consensus matrix and 2) the dispersion coefficient of the consensus matrix as defined in measuring the reproducibility of the clustering across 50 iterations. The optimal k was defined as the maximum of the product of both metrics for cluster numbers between k=3 and 8 Having determined the optimal factorization rank k, in order to achieve robust factorization of the multi-omic data matrix V, we repeated the NMF analysis using 500 iterations with random initializations of W and H and performed the partitioning of initializations of W and H and performed the partitioning of samples into clusters as described above. Due to the non-negative transformation applied to the z-scored data matrix as described above, matrix W of feature weights contained two separate above, matrix W of feature weights contained two separate weights for positive and negative z-scores of each feature, respectively. In order to revert the non-negative transformation and to derive a single signed weight for each feature, we first and to derive a single signed weight for each feature, we first normalized each row in matrix W by dividing by the sum of feature weights in each row, aggregated both weights per feature and cluster by keeping the maximal normalized weight and and cluster by keeping the maximal normalized weight and multiplication with the sign of the z-score in the initial data matrix. Thus, the resulting transformed version of matrix Wsigned contained signed cluster weights for each feature in the input contained signed cluster weights for each feature in the input matrix. For functional characterization of clustering results by single sample Gene Set Enrichment Analysis (ssGSEA), we calculated normalized enrichment scores (NES) of cancer-relevant gene sets by projecting the matrix of signed multi-omic feature weights by projecting the matrix of signed multi-omic feature weights (Wsigned) onto Hallmark pathway gene sets using ssGSEA. To derive a single weight for each gene measured across multiple omics data types (protein, RNA, phosphorylation site, acetylation site, types (protein, RNA, phosphorylation site, acetylation site, CNA) we retained the weight with maximal absolute amplitude. We used the ssGSEA implementation available on https://github.com/broadinstitute/ssGSEA2.0 using the following parameters: gene.set.database=\\u201ch.all.v6.2.symbols.gmt\\u201d\\n\\nsample.norm.type=\\u201crank\\u201d\\n\\nweight=1\\n\\nstatistic=\\u201darea.under.RES\\u201d\\n\\noutput.score.type=\\u201cNES\\u201d\\n\\nnperm=1000\\n\\nglobal.fdr=TRUE\\n\\nmin.overlap=5\\n\\ncorrel.type=\\u201dz.score\\u201d correl.type=\\u201dz.score\\u201d\\n\\nTo test the association of the resulting clusters to clinical variables we used Fisher\\u2019s exact test (R function fisher.test) to test for overrepresentation in the set of samples defining the cluster core as described above. The entire workflow described above has been implemented as a module for PANOPLY which runs on Broad\\u2019s Cloud platform Terra (https://app.terra.bio/). The docker containers encapsulating the source code and required R-packages for NMF clustering and ssGSEA code and required R-packages for NMF clustering and ssGSEA have been submitted to Dockerhub (broadcptacdev/pgdac_mo_nmf:15, broadcptac/pgdac_ssgsea:5). The source code for ssGSEA is available on GitHub: https://github.com/broadinstitute/ssGSEA2.0. Integrative analysis with Stewart et al. To enable integrative analysis of our protein data with the protein data in Stewart et al. we first aggregated the protein-level data to gene-level data matrices by retaining the dominant isoform (identified by the lowest protein subgroup number) isoform (identified by the lowest protein subgroup number) associated with each gene symbol. To aggregate the Stewart et al. protein data we used the mean expression across all proteins mapping to the same gene symbol. Both gene-level protein data mapping to the same gene symbol. Both gene-level protein data matrices were then separately subjected to gene-level z-score transformation before joining the matrices using the unique gene symbols as keys. The resulting integrated protein data matrix was as keys. The resulting integrated protein data matrix was then subjected to the NMF-based clustering pipeline described above using the 3,366 genes quantified across all 216 tumors. RNA subtyping Previously reported LSCC gene expression-subtype signatures were downloaded from the original publication and applied to our RNA expression dataset using the centroid-based method in a manner similar to the TCGA study. Specifically, the RNA expression of similar to the TCGA study. Specifically, the RNA expression of the 208 signature genes, represented as FPKM, was mean-centered in the gene-wise manner, and each sample was correlated to each of the four centroid vectors representing the average signature of the four centroid vectors representing the average signature gene expression for the four subtypes. The samples were assigned to one of the four transcriptomics subtypes according to the highest correlations. Samples with insignificance correlation (p correlations. Samples with insignificance correlation (p > 0.01) were marked to have \\u2018undecided\\u2019 subtype. Chromosomal instability index The CIN score reflects the overall copy number aberration across the whole genome. From the segment-level CNA result, we used a weighted-sum approach to summarize the chromosome instability for each sample. The absolute segment-level log2 ratios of all for each sample. The absolute segment-level log2 ratios of all copy number segments (indicating the copy number aberration of these segments) within a chromosome were weighted by the segment length and summed up to derive the instability score for the length and summed up to derive the instability score for the chromosome. The genome-wide chromosome instability index was calculated by summing up the instability score of all 22 autosomes. Fusion detection and analysis Fusion calling was performed using a combination of CRISP, CODAC MI-ONCOSEQ pipeline, fusioncatcher_v1.10 and arriba_v1.1.0 (https://github.com/suhrig/arriba/). The fusions calls were compiled and visualized using AGFusion to display the predicted domain and visualized using AGFusion to display the predicted domain and exon structure of the fusion proteins as described. mRNA and Protein correlation To compare mRNA expression and protein abundance across samples we focused on the RNAseq data with 21,792 genes, and global proteome with 11,575 quantified proteins. Protein IDs were mapped to gene names. In total, 10,890 genes common to both RNAseq and to gene names. In total, 10,890 genes common to both RNAseq and proteome data spanning 108 tumor samples were used in the analysis. The analyses were carried out on normalized data: RNAseq data were log2 transformed, upper quartile normalized RPKM values, were log2 transformed, upper quartile normalized RPKM values, which were median-centered by row (i.e., gene); proteome data was two-component normalized as described earlier. Pearson correlation was calculated for each gene. Both correlation coefficient was calculated for each gene. Both correlation coefficient and p-value were recorded. Further, adjusted p-values were calculated using the Benjamini\\u2013Hochberg procedure. CNA-driven cis and trans effects Correlations between copy number alterations (CNA) and RNA, proteome, phosphoproteome acetylproteome and ubiquitylproteome (with proteome and PTM data mapped to genes, by choosing the most variable protein isoform/PTM site as the gene-level the most variable protein isoform/PTM site as the gene-level representative) were determined using Pearson correlation of common genes present in CNA-RNA-proteome (9,988 genes), CNA-RNA-phosphoproteome (5,144 genes), CNA-RNA-acetylproteome (1,344 genes) (5,144 genes), CNA-RNA-acetylproteome (1,344 genes) and CNA-RNA-ubiquitylproteome (2,616 genes). In addition, p-values (corrected for multiple testing using Benjamini-Hochberg FDR) for assessing the statistical significance of the correlation values were the statistical significance of the correlation values were also calculated. CNA trans-effects for a given gene were determined by identifying genes with statistically significant (FDR < 0.05) positive or negative correlations. CMAP analysis Candidate genes driving response to copy number alterations were identified using large-scale Connectivity Map (CMAP) queries. The CMAP is a collection of about 1.3 million gene expression profiles from cell lines treated with bioactive small molecules profiles from cell lines treated with bioactive small molecules (~20,000 drug perturbagens), shRNA gene knockdowns (~4,300) and ectopic expression of genes. The CMAP dataset is available on GEO (Series GSE92742). For this analysis, we use the Level 5 on GEO (Series GSE92742). For this analysis, we use the Level 5 (signatures from aggregating replicates) TouchStone dataset with 473,647 total profiles, containing 36,720 gene knock-down profiles, with measurements for 12,328 genes. See knock-down profiles, with measurements for 12,328 genes. See https://clue.io/GEO-guide for more information. To identify candidate driver genes, proteome profiles of copy number-altered samples were correlated with gene knockdown mRNA profiles in the above CMAP dataset, and enrichment of up/downregulated genes was evaluated. Normalized log2 copy num- ber values genes was evaluated. Normalized log2 copy num- ber values less than 0.3 defined deletion (loss), and values greater than +0.3 defined copy number amplifications (gains). In the copy number-altered samples (separately for CNA amplification and CNA samples (separately for CNA amplification and CNA deletion), the trans-genes (identified by significant correlation in \\u201cCNA driven cis and trans effects\\u201d above) were grouped into UP and DOWN categories by comparing the protein ratios of these genes to categories by comparing the protein ratios of these genes to their ratios in the copy number neutral samples (normalized log2 copy number between 0.3 and +0.3). The lists of UP and DOWN trans-genes were then used as queries to interrogate CMAP signatures were then used as queries to interrogate CMAP signatures and calculate weighted connectivity scores (WTCS) using the single-sample GSEA algorithm. The weighted connectivity scores were then normalized for each perturbation type and cell line to obtain normalized for each perturbation type and cell line to obtain normalized connectivity scores (NCS). See for details on WTCS and NCS. For each query we then identified outlier NCS scores, where a score was considered an outlier if it lay beyond 1.5 times a score was considered an outlier if it lay beyond 1.5 times the interquartile range of score distribution for the query. The query gene was designated a candidate driver if (i) the score outliers were statistically cis-enriched (Fisher test with BH-FDR were statistically cis-enriched (Fisher test with BH-FDR multiple testing correction) and (ii) the gene had statistically significant and positive cis-correlation. For a gene to be considered for inclusion in a CMAP query it needed to i) have a copy number change (amplification or deletion) in at least 15 samples; ii) have at least 20 significant trans genes; and iii) be on the list of shRNA knockdowns in the CMAP. genes; and iii) be on the list of shRNA knockdowns in the CMAP. 502 genes satisfied these conditions and resulted in 976 queries (CNA amplification and deletion combined) that were tested for enrichment. Thirty two (32) candidate driver genes were for enrichment. Thirty two (32) candidate driver genes were identified with Fisher\\u2019s test FDR < 0.25, using this process. In order to ensure that the identified candidate driver genes were not a random occurrence, we performed a permutation test to determine how many candidate driver genes would be identified with random input (Mertins et al., 2016). For the 976 queries with random input (Mertins et al., 2016). For the 976 queries used, we substituted the bona-fide trans-genes with randomly chosen genes, and repeated the CMAP enrichment process. To determine FDR, each permutation run was treated as a Poisson sample with FDR, each permutation run was treated as a Poisson sample with rate \\u03bb, counting the number of identified candidate driver genes. Given the small n ( = 10) and \\u03bb, a Score confidence interval was calculated (Barker, 2002) and the midpoint of the confidence calculated (Barker, 2002) and the midpoint of the confidence interval used to estimate the expected number of false positives. Using 10 random permutations, we determined the overall false discovery rate to be FDR = 0.33, with a 95% CI of (0.27, 0.40) for rate to be FDR = 0.33, with a 95% CI of (0.27, 0.40) for genes with either CNA amplification or deletion, and an FDR=0.06 (95% CI: 0.01, 0.12) for six (6) genes with both CNA amplification and deletion. To identify how many trans-correlated genes for all candidate regulatory genes could be directly explained by gene expression changes measured in the CMAP shRNA perturbation experiments, knockdown gene expression consensus signature z-scores knockdown gene expression consensus signature z-scores (knockdown/control) were used to identify regulated genes with \\u03b1 = 0.05, followed by counting the number of trans-genes in this list of regulated genes. To obtain biological insight into the list of candidate driver genes, we performed (i) enrichment analysis on samples with extreme CNA values (amplification or deletion) to identify statistically enriched sample annotation subgroups; and (ii) GSEA on enriched sample annotation subgroups; and (ii) GSEA on cis/trans-correlation values to find enriched pathways. LINCS analysis Differentially expressed genes between the 5 NMF LSCC subtypes were identified using the limma package. In order to identify genes that were uniquely expressed in each subtype, all pairwise comparisons between the 5 subtypes were computed. In addition, comparisons between the 5 subtypes were computed. In addition, genes that were differentially expressed (adjusted p.value < 0.05) in at least 3 out of the 4 comparisons and had a concordant fold change among all comparisons were identified as concordant fold change among all comparisons were identified as subtype-specific and were used in the subsequent analysis. Protein abundance comparisons were performed between all 5 NMF subtypes using the Wilcoxon rank-sum test and p-values were adjusted using the Benjamini & Hochberg method. Subtype-specific differentially expressed proteins were identified based on their expressed proteins were identified based on their differential expression (adjusted p.value < 0.05) in at least 2 out of the 4 comparisons, and by having a concordant fold change among all comparisons. The identified differentially expressed genes and comparisons. The identified differentially expressed genes and proteins were then filtered for gene symbols measured in the L1000 assay (978 landmark genes). These NMF-specific signatures were used as input to calculate normalized weighted connectivity used as input to calculate normalized weighted connectivity scores (WTCS) against the Library of Integrated Network-Based Cellular Signatures (LINCS) L1000 perturbation-response signatures. The scores were computed using the sig_queryl1k_tool pipeline The scores were computed using the sig_queryl1k_tool pipeline (https://hub.docker.com/u/cmap) and the LINCS L1000 Level 5 compound (trt_cp) signatures from CLUE (clue.io, \\u201cExpanded CMap LINCS Resource 2020 Release\\u201d). The resulting normalized connectivity Resource 2020 Release\\u201d). The resulting normalized connectivity scores were summarized across cell lines using the maximum quantile of all the scores of the same compound (Qhi = 67,Qlow = 33) as previously described. The corresponding compound metadata as previously described. The corresponding compound metadata were obtained from CLUE (clue.io, \\u201cExpanded CMap LINCS Resource 2020 Release\\u201d) and were used to filter and identify compounds with existing annotations (drug name, MOA, Target). The top 20 with existing annotations (drug name, MOA, Target). The top 20 negatively connected compounds to each NMF signature were selected for visualization. Defining cancer-associated genes\\n\\nCancer-associated genes (CAG) were compiled using the Census website https://cancer.sanger.ac.uk/census. The list of genes is provided in Table S1.\\n\\nCpG Island Methylator Phenotype The methylation analysis started with all 103 tumor samples, which excluded 4 samples from the cohort with high missing rate (see above), to generate the CpG island methylator phenotypes (CIMP). Specifically, we first generated the gene-level methylation Specifically, we first generated the gene-level methylation score, by taking the averaged beta values of all probes harboring in the islands of promotor or 5 UTR regions of the gene. Then, we preprocessed the data by filtering genes that were Then, we preprocessed the data by filtering genes that were hypermethylated, i.e. the gene-level methylation score >0.2, transformed the score into M-values, normalized the transformed values, and then imputed the missing values with k-nearest neighbors and then imputed the missing values with k-nearest neighbors (KNN). With preprocessed data, we performed consensus clustering 1000 times, each taking 80% of the samples and all genes, and calculated the consensus matrix (probabilities of two samples calculated the consensus matrix (probabilities of two samples clustering together) for each predetermined number of clusters K. K ranges from 2 to 10. In each value of K, we visualized the consensus matrix using K means with Euclidean distance as the consensus matrix using K means with Euclidean distance as the distance metric. Finally, we determined the optimal number of clusters by considering the relative change in area under the consensus cumulative density function (CDF) curve. In the end, three cumulative density function (CDF) curve. In the end, three distinct clusters were identified, one was hypermethylated with mean M value 0.393, and two were hypomethylated with mean M value \\u22120.041 and \\u22120.359, respectively. We labeled these three clusters and \\u22120.359, respectively. We labeled these three clusters as CIMP high, CIMP intermediate, and CIMP low groups. iProFun Based Cis Association Analysis We used iProFun, an integrative analysis tool to identify multi-omic molecular quantitative traits (QT) perturbed by DNA-level variations. We considered five functional molecular quantitative traits (gene expression, protein, phosphoprotein, quantitative traits (gene expression, protein, phosphoprotein, acetylprotein, ubiquitylproteome levels) for their associations with DNA methylation, accounting for mutation, copy number variation, age, gender, tumor purity and smoking status. Tumor purity age, gender, tumor purity and smoking status. Tumor purity was determined using TSNet from RNA-seq data. The iProFun procedure was applied to a total of 12,666 genes that was available in at least one predictor (mutation, CNA, DNA methylation) and one at least one predictor (mutation, CNA, DNA methylation) and one outcome (mRNA, global protein, phospholation, acetylation and ubiquitylation) for their cis regulatory patterns in tumors. For each of the 12,666 genes, we started with linear regression to each of the 12,666 genes, we started with linear regression to capture the covariate-adjusted associations. The resulting association summary statistics was used to call probabilities of belonging to each of the 2^5=32 possible configurations. We of belonging to each of the 2^5=32 possible configurations. We calculated the probabilities of associating with each of the five outcomes by combining probabilities from relevant configurations, and calculated FDR controlled associations. The significant and calculated FDR controlled associations. The significant genes need to pass three criteria: (1) the satisfaction of biological filtering procedure, (2) posterior probabilities > 75%, and (3) empirical false discovery rate (eFDR)<10%. As DNA methylation empirical false discovery rate (eFDR)<10%. As DNA methylation at the promoter regions of a gene down regulates its expression level, biological filtering procedure requires DNA methylations that exist either negative significant associations or that exist either negative significant associations or insignificant associations with all 5 the types of molecular QTs for significance call. DNA methylations with significant positive associations with QTs were filtered out for significance call, as it with QTs were filtered out for significance call, as it was not the pattern of biological associations that were interesting. Secondly, a significance was called only if the posterior probabilities > 75% of a predictor being associated with a molecular > 75% of a predictor being associated with a molecular QT, by summing over all configurations that are consistent with the association of interest. Lastly, we calculated empirical FDR via 100 permutations per molecular QTs by shuffling the label of the permutations per molecular QTs by shuffling the label of the molecular QTs, and requested empirical FDR (eFDR) <10% by selecting a minimal cutoff value of alpha such that 75%<alpha<100%. The eFDR is calculated by: eFDR= (Averaged No. of genes with The eFDR is calculated by: eFDR= (Averaged No. of genes with posterior probabilities > alpha in permuted data) / (Averaged No. of genes with posterior probabilities > alpha in original data). Figure 1G annotated those genes whose DNA methylation had Figure 1G annotated those genes whose DNA methylation had cascading effects (associated with all QTs under investigation), and among them those whose protein abundances significantly differ between tumor and NAT were annotated. EMT-specific cluster and fibroblast enrichment NMF derived EMT-enriched subtype was interrogated for EMT and fibroblast related proteins in LSCC tumor patients. The relative protein expression was calculated using z-score. The single ssGSEA was performed on a protein dataset using GSVA (v3.11) to ssGSEA was performed on a protein dataset using GSVA (v3.11) to calculate normalized enrichment score for EMT and fibroblast proliferation using MSigdb (v6.1) and gene ontology geneset respectively. Similar analysis was also performed to interrogate SOX2 Similar analysis was also performed to interrogate SOX2 associated genes and its relation with IL6_JAK_STAT signaling (MSigdb hallmarks geneset V6.1). Differential marker analysis A Wilcoxon signed rank test was performed on TMT-based global proteomic data between tumor and matched normal samples to determine differential abundance of proteins between tumor and NAT samples. Proteins having < 50% missing values were considered for Proteins having < 50% missing values were considered for downstream analysis. Proteins with log2-fold-change (FC)> 2 in tumors and Benjamini-Hochberg FDR < 0.01 were considered to be tumor-associated proteins. Over-representation analysis of Gene Ontology proteins. Over-representation analysis of Gene Ontology Biological Process terms was performed on a background of quantified proteins in WebGestaltR. The selected gene list was either significant proteins or proteins containing a significantly altered proteins or proteins containing a significantly altered phosphorylation site from the differential marker analyses. ORA was performed separately for increased and decreased proteins. Terms were considered significant with a Benjamini-Hochberg adjusted were considered significant with a Benjamini-Hochberg adjusted p-value <0.01. Among tumor associated proteins, the top 50 differential proteins with more than 90% of samples showing high expression in tumors were selected and shown. Differential proteins with more than 99% of samples showing tumor specific expression were more than 99% of samples showing tumor specific expression were highlighted. Immunohistochemistry-based antibody-specific staining scores in lung tumors were obtained from the Human Protein Atlas (HPA, https://www.proteinatlas.org), in which Protein Atlas (HPA, https://www.proteinatlas.org), in which tumor-specific staining is reported in four levels, i.e., high, medium, low, and not detected. The protein-specific annotations such as enzymes, transcription factor, transporters, secreted, such as enzymes, transcription factor, transporters, secreted, transmembrane and FDA-approved drugs targeting the protein or reported in drugbank were designated. Similarly, differential analysis was performed between RNA, phosphosite and acetylproteome was performed between RNA, phosphosite and acetylproteome datasets between tumor and matched NATs. The differential genes/sites with log2-fold-change (FC)> 2 in tumors and Benjamini-Hochberg FDR < 0.01 were considered to be tumor-associated. The FDR < 0.01 were considered to be tumor-associated. The overlapping genes/sites with proteins were represented in a checkerbox plot. Overall and disease-free survival in TCGA lung squamous carcinoma Survival for lung squamous tumor patients in TCGA LSCC cohort was calculated using the time from date of diagnosis to death or last contact (Overall survival, OS) and disease-free interval time (Disease-free survival, DFS). Normalized RNA expression was (Disease-free survival, DFS). Normalized RNA expression was categorized into low and high expression group based on mean. Univariate Cox proportional hazards models were fitted to OS, and DFI endpoints to calculate the hazard ratios for tumor-associated endpoints to calculate the hazard ratios for tumor-associated genes/proteins using coxph function in Survival package (version 2.44) in R. The p-values were determined using a log-rank test. The hazard ratio [exp(cox coefficient)] was used to compare poor hazard ratio [exp(cox coefficient)] was used to compare poor survival in low and high-expression group. Kaplan-Meier survival analysis We used time-to-death or last contact data for the current cohort (recorded as number of days from diagnosis date) to assess survival differences based on various sample subgroups and clinical annotations. We used Kaplan-Meier analysis to explore survival annotations. We used Kaplan-Meier analysis to explore survival differences associated with tumor grade, mutation burden, mutation status for SMGs, CUL3-NFE2L2-KEAP1 combined mutation status, CNA for KAT6A/SOX2/TP63/FGFR1/CDKN2A, immune subtype, NMF CNA for KAT6A/SOX2/TP63/FGFR1/CDKN2A, immune subtype, NMF cluster, CIN, tumor grade and ploidy. We identified NMF mixed tumors (i.e., tumors that are not part of the core NMF clusters) had significantly worse survival (log-rank p-value = 0.005). We used a Fisher test to find enriched annotations (among those mentioned in the previous paragraph, with (among those mentioned in the previous paragraph, with tumor stemness index added) associated with NMF mixed tumors. The frequency of SOX2 amplifications were significantly enriched in NMF mixed tumors (p-value = 0.0038). None of the other annotations had tumors (p-value = 0.0038). None of the other annotations had significant enrichment in the NMF mixed tumors. Continuous Smoking Score Non-negative matrix factorization (NMF) was used in deciphering mutation signatures in cancer somatic mutations stratified by 96 base substitutions in tri-nucleotide sequence contexts. To obtain a reliable signature profile, we used SomaticWrapper to call a reliable signature profile, we used SomaticWrapper to call mutations from WGS data. SignatureAnalyzer exploited the Bayesian variant of the NMF algorithm and enabled an inference for the optimal number of signatures from the data itself at a balance optimal number of signatures from the data itself at a balance between the data fidelity (likelihood) and the model complexity (regularization). After decomposing into three signatures, signatures were compared against known signatures derived from COSMIC were compared against known signatures derived from COSMIC and cosine similarity was calculated to identify the best match. We also sought to integrate count of total mutations, t, percentage that are signature mutations, c, and count of DNPs, n, into a continuous score, 0 < S < 1, to quantify the degree of confidence that a sample was associated with smoking signature. We that a sample was associated with smoking signature. We referred to these quantities as the data, namely D = C \\u2229 T \\u2229 N, and used A and A\\u2019 to indicate smoking signature or lack thereof, respectively. In a Bayesian framework, it is readily shown that a In a Bayesian framework, it is readily shown that a suitable form is S = 1 / (1 + R), where R is the ratio of the joint probability of A\\u2019 and D to the joint probability of A and D. For example, the latter can be written P(A)\\u30fbP(C|A)\\u30fbP(T|A)\\u30fbP(N|A) and the the latter can be written P(A)\\u30fbP(C|A)\\u30fbP(T|A)\\u30fbP(N|A) and the former similarly, where each term of the former is the complement of its respective term in this expression. Common risk statistics are invoked as priors, i.e. P(A) = 0.9. We consider S to be a score because rigorous conditioned probabilities are difficult to establish. (For example, the data types themselves are not entirely independent of one another and models using common distributions like the Poisson do not and models using common distributions like the Poisson do not recapitulate realistic variances.) Instead, we adopt a data-driven approach of estimating contributions of each data type based on 2-point empirical fitting of roughly the low and high extremes 2-point empirical fitting of roughly the low and high extremes using shape functions. The general model for data type G is P(G|A) = [x \\u30fberf (g/y) + 1] / (x + 2) and P(G|A\\u2019) = 1 - P(G|A), where erf is the Gaussian error function, g is the observed value erf is the Gaussian error function, g is the observed value for the given data type, and x and y are empirically-determined weights. The shape function for total mutations, T, has two modifiers. First, it includes a simple expected-value correction for purity, u. Namely, assuming mutation-calling does not capture all mutations because of impurities, t is taken as the observed number of because of impurities, t is taken as the observed number of mutations divided by a purity shape function, f, where f \\u2264 1. Although one might model f according to common characteristics of mutation callers, e.g. close to 100% sensitivity for pure samples callers, e.g. close to 100% sensitivity for pure samples and very low calling rate for low variant allele fractions (VAFs), the purity estimates for these data are based on RNA-Seq and are not highly correlated with total mutation counts. Consequently, we highly correlated with total mutation counts. Consequently, we use a line function, f = 0.2\\u30fbu + 0.8, which does not strongly impact the adjustment of low-purity samples. Second, before choosing weights, we removed outliers, as determined by Peirce\\u2019s weights, we removed outliers, as determined by Peirce\\u2019s criterion. Based on the distribution of continuous smoking score, we set 0.15 as the lower bound cutoff and 0.6 as the upper bound cutoff for defining the inferred smoking status. We classified the for defining the inferred smoking status. We classified the samples with strong smoking evidence if the smoking score > 0.6; samples with weak smoking evidence if smoking score < 0.15. Identification of differentially regulated events in NRF2 mutant tumors. For each gene, normalized levels of mRNA/protein/phosphoprotein (log transformed data) were fit into a linear regression model. In addition to NFR2 mutation status, gender, tumor purity and ethnicity were also included in the model. Coefficient and and ethnicity were also included in the model. Coefficient and p-value of NRF2 mutation status variables from the fitted model were used to evaluate the extent and significance of the difference between patients with and without aberration. To adjust for between patients with and without aberration. To adjust for multi-test, p-values were adjusted with the BH method. Genes that showed significant upregulation (up) in more than one dataset (Table S4) were used to generate a GMT file for ssGSEA based NRF2 S4) were used to generate a GMT file for ssGSEA based NRF2 activity score calculation using ssGSEA (https://github.com/broadinstitute/ssGSEA2.0). The NRF2 activity score contains a total of 54 genes upregulated both at protein and RNA level, identified by genes upregulated both at protein and RNA level, identified by our NRF2 mutant versus wildtype tumor differential analysis (p-value <0.05 and logFC>1). Among the 54 genes, 44 have been previously associated with NRF2 signaling (either in NRF2 concept in associated with NRF2 signaling (either in NRF2 concept in msigdb such as Gene Sets NFE2L2.V2 and SINGH_NFE2L2_TARGETS (, BIOCARTA_ARENRF2_PATHWAY], NRF2 wikipathway or Lacher et al. Ten of the 54 genes were novel targets identified in this study. Immune cluster identification based on cell type composition The abundance of 64 different cell types were computed via xCell based on transcriptomic profiles based on 108 tumors and 94 NAT samples. Table S6 contains the final score computed by xCell of different cell types. Consensus clustering was performed based different cell types. Consensus clustering was performed based on immune cells, fibroblasts, endothelial and epithelial cells from xCell. In particular, we only considered cells that were detected in at least 10% of the patients (FDR < 5%). This filtering in at least 10% of the patients (FDR < 5%). This filtering resulted in 25 cell types annotated in Table S6. Based on these signatures, consensus clustering was performed in order to identify groups of samples with similar immune/stromal characteristics. groups of samples with similar immune/stromal characteristics. Consensus clustering was performed using the R packages ConsensusClusterPlus based on z-score normalized signatures. Specifically, signatures were partitioned into three major clusters using signatures were partitioned into three major clusters using the Partitioning Around Medoids (PAM) algorithm, which was repeated 1000 times (Figure 6A, Table S6). TCGA pan-cancer immune subtyping Tumors were classified into the six distinct immune subtypes \\u2014 wound healing, IFN-\\u03b3 dominant, inflammatory, lymphocyte depleted, immunologically quiet, and TGF-\\u03b2 dominant \\u2014 identified by a TCGA pan-cancer analysis presented in. Gene expression data (log2 pan-cancer analysis presented in. Gene expression data (log2 FPKM; lscc-v3.0-rnaseq-uq-fpkm-log2-NArm) was input to the ImmuneSubtypeClassifier R package (https://github.com/CRI-iAtlas/ImmuneSubtypeClassifier) to assign an immune subtype to each of the to assign an immune subtype to each of the tumors. Ranking tumors by inferred activity of IFN-\\u03b3 pathway We assumed that true biological activity of a pathway is regulated by collective changes of expression levels of majority of proteins involved in this pathway. Then, a difference of a pathway activity between tumors can be assessed by a difference in activity between tumors can be assessed by a difference in positioning of expression levels of proteins involved in this pathway in ranked list of expression levels of all proteins in each of tumors. Following this idea, we assessed relative positioning tumors. Following this idea, we assessed relative positioning of pathway proteins between tumor by determining two probabilities: a probability of pathway proteins to occupy by random the observed positions in a list of tumor proteins ranked by expression positions in a list of tumor proteins ranked by expression levels from the top to the bottom and, similarly, a probability to occupy by random the observed positions in a list of expression levels ranked from the bottom to the top. Then, the inferred levels ranked from the bottom to the top. Then, the inferred relative activation of a given pathway across tumors was assessed as a negative logarithm of the ratio of the above \\u201ctop\\u201d and \\u201cbottom\\u201d probabilities. Thus, for a pathway of a single protein, its probabilities. Thus, for a pathway of a single protein, its relative activity across tumors was assessed as a negative log of ratio of two numbers: a number of proteins with expression level bigger than an expression level of given protein, and a number bigger than an expression level of given protein, and a number of proteins with expression levels less than an expression level of given protein. For pathways of multiple proteins, the \\u201ctop\\u201d and \\u201cbottom\\u201d probabilities were computed as geometrically \\u201ctop\\u201d and \\u201cbottom\\u201d probabilities were computed as geometrically averaged P values computed for each of proteins using Fisher\\u2019s exact test, given protein\\u2019s ranks in a list of pathway proteins and in a list of ranked proteins of a tumor, a number of and in a list of ranked proteins of a tumor, a number of proteins in a pathway, and the total number of proteins with the assessed expression level in a given tumor. The thermodynamic interpretation of the inferred pathway activity scoring function is a of the inferred pathway activity scoring function is a free energy associated with deviation of the system from equilibrium either as a result of activation or suppression. Thus, the scoring function is positive, when expression levels of pathway\\u2019s function is positive, when expression levels of pathway\\u2019s proteins are overrepresented among top expressed proteins of a tumor, and it is negative, when pathway\\u2019s proteins are at the bottom of expressed proteins of a tumor; the scoring function is close of expressed proteins of a tumor; the scoring function is close to zero, when expression levels are distributed by random or equally shifted towards top or bottom. In our analysis, we used a gene list of IFN-\\u03b3 signaling pathway from. We settled with this 15 genes signaling pathway because it has all necessary components \\u2013 a cytokine mediator, receptors, signal transduction, transcription factors and final effectors signal transduction, transcription factors and final effectors (PD-L1/2, MHC-I), and all those genes are expressed in cancer cells. Estimation of Tumor Purity, Stromal and Immune Scores Besides xCell, we utilized ESTIMATE to infer immune and stromal scores based on gene expression data (Table S6). Cibersort absolute immune scores were obtained by evaluating upper-quartile normalized RNA-seq FPKM data with the Cibersort web app normalized RNA-seq FPKM data with the Cibersort web app (cibersort.stanford.edu) in absolute mode. To infer tumor purity, TSNet was utilized (Table S6). Differentially Expressed Genes and Pathway Analysis Genes upregulated in Hot and Warm clusters compared to the Cold cluster were identified based on gene expression data, global proteomic, phosphoproteomic and acetylation data. For this analysis, markers with at most 50% missing values were utilized. For markers with at most 50% missing values were utilized. For each data type, the expression level of gene/protein/sites was modeled as a function of immune cluster via linear regression. P-values were then adjusted for multiple comparisons via P-values were then adjusted for multiple comparisons via Benjamini-Hochberg adjustment. For each immune cluster, considering the set of genes up(down)-regulated with Benjamini-Hochberg adjusted p-value lower than 10%, a fisher exact test was implemented p-value lower than 10%, a fisher exact test was implemented to derive enriched pathways. ssGSEA was utilized to obtain pathway scores based on RNA-seq and global proteomics data using the R package GSVA. For this analysis, pathways from the Reactome, KEGG GSVA. For this analysis, pathways from the Reactome, KEGG and Hallmark databases were considered and as background the full list of gene/proteins observed under each data type was utilized. For phosphorylation and acetylproteome data, a gene was For phosphorylation and acetylproteome data, a gene was considered upregulated if at least one substrate of the gene was found upregulated at 10% FDR. The pathway analysis results for different data types are contained in Table S6. Deriving RTK CBPE scores A total of 42 human RTKs were present in our proteomics data set. For each phosphosite in our data set we computed a linear association with each of the RTKs in two ways. A phosphosite profile across samples (for tumor and NAT separately) was modeled as a across samples (for tumor and NAT separately) was modeled as a linear combination of 1) an RTK protein profile and a profile of a protein that a given phosphosite belonged to or 2) an RTK protein profile, a profile of a protein that a given phosphosite profile, a profile of a protein that a given phosphosite belonged to and a profile of tumor purity estimates obtained from RNAseq data via TSNet. Each association computation was carried out only if at least 50% of samples had non-missing data of all out only if at least 50% of samples had non-missing data of all three of RTK protein, phosphosite, and protein in question. The phosphosite:RTK associations were considered significant if their FDR adjusted p-value was below 0.1. Additionally, for each FDR adjusted p-value was below 0.1. Additionally, for each phosphosite the z-scores of their log2 phosphoprotein abundance were computed across samples (separately for tumor and NAT). A raw RTK CBPE score was computed for each RTK in each sample as a RTK CBPE score was computed for each RTK in each sample as a number of all phosphosites with significant association with this RTK that also had a z-score higher than 1. These raw RTK CBPE scores in each sample were normalized across RTKs so the final RTK in each sample were normalized across RTKs so the final RTK CBPE scores of all RTKs in one sample would add up to 1. We selected the most relevant subset of RTKs by requiring that each had a normalized CBPE score of at least 0.1 in at least 5 samples. a normalized CBPE score of at least 0.1 in at least 5 samples. Nine RTKs satisfied this condition in the tumor data set, and seven in NAT. Independent component analysis ICA was performed with a workflow modified from previously described. Decomposition was run for 100 times on the matrix of protein abundance difference between tumor/NAT pairs (n=99). Independent components were in the form of vectors comprised with components were in the form of vectors comprised with weights of all genes in the original data. Components extracted from each run were clustered using HDBSCAN algorithm with cosine distance as dissimilarity metric, min_cluster_size=50 and distance as dissimilarity metric, min_cluster_size=50 and min_samples=20. The centroids of clusters (n=37) were considered as representative of stable signatures, and mean mixing scores (activity of each signature over all samples) of each cluster were of each signature over all samples) of each cluster were used to represent the activity levels of corresponding signature in each sample. Correlation between the extracted signatures and known clinical characteristics were examined by regressing the known clinical characteristics were examined by regressing the corresponding mixing scores for all members of a component cluster against 64 sample annotations to obtain within-cluster average of log10 P-values. Significance was controlled for multiple of log10 P-values. Significance was controlled for multiple testing at 0.01 level (log10 (p-value)=\\u22125.3). Each signature vector (cluster centroid) was submitted to GSEA pre-ranked test for functional annotations. Mutation-based cis- and trans-effects We examined the cis- and trans-effects of 22 genes with somatic mutations that were significant in a previous large-scale TCGA LSCC study on the RNA, proteome, and phosphoproteome of known interactome DBs including Omnipath, Phosphositeplus, DEPOD, interactome DBs including Omnipath, Phosphositeplus, DEPOD, Signor, and CORUM. After excluding silent mutations, samples were separated into mutated and WT groups. We used the Wilcoxon rank-sum test to report differentially expressed features (RNA, rank-sum test to report differentially expressed features (RNA, proteins, phosphosites, acetylsites, and ubiquitylation sites) between the two groups. Differentially enriched features passing an FDR <0.05 cut-off were separated into two categories based an FDR <0.05 cut-off were separated into two categories based on cis- and trans- effects. Germline quantitative trait loci (QTL) analysis To identify germline genetic variants that explain variation in tumor gene (eQTL) and protein (pQTL) expression, we utilized the gold-standard mapping pipeline at https://github.com/molgenis/systemsgenetics/wiki/eQTL-mapping-analysis-cookbook-(eQTLGen). Briefly, we followed the default steps that include correcting for population stratification removing outliers, normalizing and mapping cis-QTLs. For cis-eQTL analysis input, we used i) germline genotype data; and ii) RNA-seq raw read count data for 105 genotype data; and ii) RNA-seq raw read count data for 105 samples, and performed analyses only for genes with sum of read count \\u2265 10. For cis-pQTL analysis input, we used i) germline genotype data; and ii) normalized and filtered protein TMT ratio data; and ii) normalized and filtered protein TMT ratio aggregated to gene level for 108 samples. The genotype data were harmonized using Genotype Harmonizer v1.4.9. To adjust for population stratification, we identified the multi dimensional scaling stratification, we identified the multi dimensional scaling (MDS) components using the genotype data using PLINK v1.07. For both types of cis-QTL mapping, we used eqtl-mapping-pipeline-v1.3.9 using its standard settings (HWE > 0.0001, MAF > 0.01, and call using its standard settings (HWE > 0.0001, MAF > 0.01, and call rate > 0.95, maximum distance between the SNP and the middle of the probe is 250,000 bp). To control for multiple testing, we performed 10 permutations. We deemed QTLs at \\u2264 FDR 5% as we performed 10 permutations. We deemed QTLs at \\u2264 FDR 5% as significant. Finally, we performed pathway analysis of the identified significant eGenes (genes whose expression is impacted by at least one cis-QTL) using QIAGEN Ingenuity Pathway Analysis miRNA analysis presented in Figure 4 Targets of miRNAs were downloaded from the miRNA targets database miRTarBase and only the miRNA/target pairs with strong experimental evidence were retained. Spearman correlation was used to calculate the correlations between miRNA and its target genes. Pathway projection using ssGSEA The single sample Gene Set Enrichment Analysis (ssGSEA) implementation available on https://github.com/broadinstitute/ssGSEA2.0 was used to project log2(FPKM) mRNA abundances to MSigDB cancer hallmark gene sets using the following parameters: gene.set.database=\\u201ch.all.v6.2.symbols.gmt\\u201d\\n\\nsample.norm.type=\\u201crank\\u201d\\n\\nweight=0.75\\n\\nstatistic=\\u201darea.under.RES\\u201d\\n\\noutput.score.type=\\u201cNES\\u201d\\n\\nnperm=1000\\n\\nglobal.fdr=TRUE\\n\\nmin.overlap=10\\n\\ncorrel.type=\\u201dz.score\\u201d\\n\\nPhosphorylation-driven signature analysis We performed phosphosite-specific signature enrichment analysis (PTM-SEA) to identify dysregulated phosphorylation-driven pathways. To adequately account for both magnitude and variance of measured phosphosite abundance, we used p-values derived from measured phosphosite abundance, we used p-values derived from application of the Wilcoxon rank-sum test to phosphorylation data as ranking for PTM-SEA. To that end, p-values were log-transformed and signed according to the fold change (signed \\u2212log10 and signed according to the fold change (signed \\u2212log10 (p-value)) such that large positive values indicated phosphosite abundance in classical or NFE2L2, CUL3, KEAP1 mutated samples, and large negative values indicated phosphosite abundance in samples negative values indicated phosphosite abundance in samples that do not belong to classical NMF subtype or wild-type samples (in Figure 3I)  PTM-SEA relies on site-specific annotation provided by PTMsigDB and thus a single site-centric data matrix data is by PTMsigDB and thus a single site-centric data matrix data is required such that each row corresponds to a single phosphosite. We note that in this analysis the data matrix comprised a single data column and each row represented a confidently localized data column and each row represented a confidently localized phosphosite assigned by Spectrum Mill software. We employed the heuristic method introduced by Krug et al. to deconvolute multiple phosphorylated peptides to separate data points (log-transformed and signed p-values). Briefly, phosphosites measured on different phospho-proteoform peptides were resolved measured on different phospho-proteoform peptides were resolved by using the p-value derived from the least modified version of the peptide. For instance, if a site T4 measured on a doubly phosphorylated (T4, S8) peptide (PEPtIDEsR) was also measured on a (T4, S8) peptide (PEPtIDEsR) was also measured on a mono-phosphorylated version (PEPtIDESR), we assigned the p-value derived from the mono-phosphorylated peptide proteoform to T4, and the p-value derived from PEPtIDEsR to S8. If only the doubly the p-value derived from PEPtIDEsR to S8. If only the doubly phosphorylated proteoform was present in the dataset, we assigned the same p-value to both sites T4 and S8. We queried the PTM signatures database (PTMsigDB) v1.9.0 downloaded from https://github.com/broadinstitute/ssGSEA2.0/tree/master/db/ptmsigdb using the flanking amino acid sequence (+/\\u2212 7 aa) as primary identifier. We used the implementation of PTM-SEA as primary identifier. We used the implementation of PTM-SEA available on GitHub (https://github.com/broadinstitute/ssGSEA2.0) using the command interface R-script (ssgsea-cli.R). The following parameters were used to run PTM-SEA: weight: 1\\n\\nstatistic: \\u201carea.under.RES\\u201d\\n\\noutput.score.type: \\u201cNES\\u201d\\n\\nnperm: 1000\\n\\nmin.overlap: 5\\n\\ncorrel.type: \\u201cZ-score\\u201d The sign of the normalized enrichment score (NES) calculated for each signature corresponds to the sign of the tumor-NAT log fold change. P-values for each signature were derived from 1,000 random permutations and further adjusted for multiple hypothesis permutations and further adjusted for multiple hypothesis testing using the method proposed by Benjamini & and Hochberg. Signatures with FDR-corrected p-values < 0.05 were considered to be differential between tumor and NAT. CDKN2A and RB1 annotations and pathway analysis Comprehensive tumor annotation for CDKN2A and RB1 genomic status was carried out using multiple molecular features for each patient. Specifically, we considered the following: 1) mutation types (missense/in-frame indels mutations, nonsense (stop gain, types (missense/in-frame indels mutations, nonsense (stop gain, frameshift indels) mutations, and splice site (splice donor, splice acceptor) mutations) as separate categories and 2) copy number data. For cases with mutation, we considered both the number data. For cases with mutation, we considered both the variant allele frequency of the mutation from whole exome sequencing and copy number data (log ratio, absolute copy number, and B-allele frequency). Only mutations that occurred in cases with B-allele frequency). Only mutations that occurred in cases with loss of the wild-type allele were annotated as loss of heterozygosity (LOH) mutations and used for subsequent analysis. 2) Based on CDKN2A copy number data, we next annotated tumors without on CDKN2A copy number data, we next annotated tumors without mutations as homozygous deletions or no loss (WT). Samples were re-classified as having p16INK4a promoter hypermethylation if three p16INK4a promoter-associated methylation probes (cg11075751, p16INK4a promoter-associated methylation probes (cg11075751, cg02008397, and cg01694391) all had beta values >= 0.2. Multi-gene proliferation scores were calculated as described previously as the means of the gene normalized RNA levels for cell cycle as the means of the gene normalized RNA levels for cell cycle associated genes characterized by Whitfield et al. p16INK4a RNA levels were obtained from the isoform specific RSEM data for Ensembl transcript ENST00000304494.8, while ENST00000579755.1 was transcript ENST00000304494.8, while ENST00000579755.1 was used for p14ARF, and log2 (RSEM + 1) values were used for barplots. Isoform specific protein data included Refseq NP_000068.1 for p16INK4a and Refseq NP_478102.2 for p14ARF. Calculation of CDK1, 2, and Refseq NP_478102.2 for p14ARF. Calculation of CDK1, 2, and 4 target site scores from the phosphorylation site data and Hallmark E2F targets and G2M scores from the RNA-seq data is described above (Phosphorylation-driven signature analysis and Pathway above (Phosphorylation-driven signature analysis and Pathway projection using ssGSEA, respectively). Lolliplots for CDKN2A mutations in the CPTAC head and neck squamous cell carcinoma and this lung squamous cell carcinoma cohort were generated using the lung squamous cell carcinoma cohort were generated using the ProteinPaint web application to visualize mutations. Mutations annotated as nonsense mediated decay (NMD) mutations were those that resulted in low expression of p16INK4a RNA in the tumors that resulted in low expression of p16INK4a RNA in the tumors harboring the mutations. Association analysis between KGG-site abundances and E3 ligases and DUBs A list of known human E3 ubiquitin and ubiquitin-like ligases and DUBs was compiled from. We then fit a linear model using limma in R with the formula kgg_site_abundance ~ protein_abundance, followed by empirical bayes shrinkage. The coefficient p-values followed by empirical bayes shrinkage. The coefficient p-values were used to determine significant associations after FDR correction (FDR < 0.01). Cluster and pathway analysis of significantly modulated K-GG sites in tumors. Consensus clustering of the K-GG dataset after protein abundance correction was performed on tumor samples, resulting in 3 clusters. Next, marker selection for each cluster was performed by moderated t-test between samples that belong to the clusters by moderated t-test between samples that belong to the clusters against the remaining samples. These K-GG sites showing differential abundance across sample clusters were further clustered into 3 site-wise clusters. Genes in these site-wise clusters were used for pathway enrichment analysis against the KEGG, Reactome, and WikiPathways databases using g:profiler. Pathway enrichment was performed using a gene background containing all observable genes in the K-GG a gene background containing all observable genes in the K-GG dataset. Pathway enrichment results were imported into Cytoscape using the Enrichment Map app for network analysis of pathways. Pathways were connected using the gene overlap and clustered Pathways were connected using the gene overlap and clustered (pathway cutoff q-val < 0.1; jaccard overlap > 0.375). Each cluster was manually annotated from the pathways contained in it to facilitate interpretation. PTM CLUMPS analysis We employ two methods to select tumor-specific sites to include in structural analysis. First, we take PTM-sites for solely tumor-derived samples and binarize modifications by negative vs. positive normalized signal. These are considered as individual positive normalized signal. These are considered as individual \\u201cevents\\u201d as done with mutations in CLUMPS v1. For a more robust approximation of tumor-specific acetylation or ubiquitination, we perform differential expression using Limma between NAT and we perform differential expression using Limma between NAT and Tumor samples. Sites are selected with an FDR < 0.1 (Benjamini-Hochberg) and LFC > 0.1. We then binarize these tumor sites as done in the first approach. We first map the PTM-sites to as done in the first approach. We first map the PTM-sites to corresponding UNIPROT ids (ID) with available PDBs. For each crystal structure, we compute an initial WAP score and randomly sample sites as done in CLUMPS. We generate an empirical p-value sites as done in CLUMPS. We generate an empirical p-value based on a random sampling of lysines in each crystal structure to limit the selection to residues capable of ubiquitination or acetylation. The null hypothesis we define is each random sampling of The null hypothesis we define is each random sampling of lysines will have a WAP score less than or equal to the initially computed WAP score. We run 1e6 permutations to generate an empirical p-value. DepMap genetic dependency and drug response analysis\\n\\nCell lines annotated as \\u201cNSCLC_squamous\\u201d from DepMap were considered as LSCC for which molecular profiles, dependency scores, and drug response were used. To determine copy number status in LSCC cell lines, copy number segmentation files for LSCC cell lines were processed with GISTIC2.0 (using the same parameters used to process this study\\u2019s LSCC cohort as described in STAR METHODS). Samples were considered cohort as described in STAR METHODS). Samples were considered to be amplified for a given gene if GISTIC thresholded values = 1 or 2 and not amplified if GISTIC thresholded values = \\u22122, \\u22121, or 0. Mutation status for CCND1, CDKN2A, and RB1, Np63 transcript 0. Mutation status for CCND1, CDKN2A, and RB1, Np63 transcript isoform expression, phospho-Rb-Ser807/811 RPPA abundance, and drug responses to a survivin inhibitor (YM-155) and CDK4/6i inhibitors (abemaciclib and palbociclib) were also retrieved from (abemaciclib and palbociclib) were also retrieved from DepMap. Figure 7: For identifying if top protein biomarkers (502 proteins significantly overexpressed (log2(FC) >2, FDR <0.01) in tumors relative to their matched NATs, most with coherent overexpression in multi-omic analysis) also conferred altered dependencies in multi-omic analysis) also conferred altered dependencies in LSCC cell lines, we leveraged DepMap genetic dependency dataset (CRISPR Avana Public 20Q3) that contained 18119 genes and 789 cell lines (https://depmap.org/portal/download/ file: and 789 cell lines (https://depmap.org/portal/download/ file: Achilles_gene_effect.csv). Only 16 cell lines were classified as LSCC (Cell Line Sample Info.csv). Median dependencies were calculated and for every gene and density plot in Figure 7C shows and for every gene and density plot in Figure 7C shows dependency score of the 502 genes corresponding to 502 protein biomarkers relative to other genes. Figure S4B: To investigate copy number and dependency associations in LSCC cell lines, each gene up-regulated at the protein level with recurrent copy number gain in tumors vs NATs (from Figure 4A with Log2FC>0 & FDR < 0.05) was used as a query to compare 4A with Log2FC>0 & FDR < 0.05) was used as a query to compare dependency scores of the given gene (Combined CRISPR KO screen, DepMapPublic 21Q1) in LSCC cell lines where the query gene was amplified vs not-amplified. A lower mean change in dependency amplified vs not-amplified. A lower mean change in dependency scores for a given gene in amplified vs not-amplified samples indicates LSCC cell lines are more dependent upon that gene when it is amplified. Continuous log2 copy number data from DepMap was used in Figure S4M to correlate SOX2 copy number with EZH2 shRNA dependency data (Combined shRNA screen from DEMETER2 Data v6 in DepMap). Drug responses to YM-155 (PRISM secondary screen) were compared by t-test in LSCC cell lines with low vs high levels of Np63 by transcript isoform expression (Figure S4G). Pearson correlation was also performed with phospho-Rb-Ser807/811 RPPA abundance was also performed with phospho-Rb-Ser807/811 RPPA abundance and drug responses to CDK4/6i inhibitors, abemaciclib (PRISM secondary screen) and palbociclib (Sanger GDSC1) (Figure S3D). CausalPath analysis CausalPath searches for known biological mechanisms that can explain correlated proteomic changes in terms of causal hypotheses. We set CausalPath parameters to compare tumors and NATs with a paired t-test, used 0.1 as FDR threshold for proteomic change a paired t-test, used 0.1 as FDR threshold for proteomic change significance and network significance, and detected 5917 potential causal relations between proteins. We repeated the same analysis for each NMF subtype separately and identified 4378 analysis for each NMF subtype separately and identified 4378 (basal-inclusive), 5334 (classical), 3048 (EMT-enriched), 3744 (inflamed-secretory), and 4332 (proliferative-primitive) relations. We used these CausalPath network results in the preparation of We used these CausalPath network results in the preparation of Figure 7C, identifying potential upstream regulators of oncogenic phosphoproteomic changes. Here an oncogenic phosphoproteomic change can be any of the following 4 events: increase of change can be any of the following 4 events: increase of activating phosphorylation of an oncoprotein, decrease of inactivating phosphorylation of an oncoprotein, decrease of activating phosphorylation of a tumor suppressor protein, and increase of phosphorylation of a tumor suppressor protein, and increase of inactivating phosphorylation of a tumor suppressor protein. We used the OncoKB database for oncoprotein and tumor suppressor classification (excluded proteins that have both annotations), and (excluded proteins that have both annotations), and used PhosphoSitePlus and Signor databases for the activating/inhibiting classification of phosphorylation sites. In the phosphorylation regulation networks, we included only the targetable regulators regulation networks, we included only the targetable regulators (activated proteins) and excluded the untargetable regulators (inactivated proteins). Variant Peptide Identification We used NeoFlow (https://github.com/bzhanglab/neoflow) for neoantigen prediction. Specifically, Optitype was used to find human leukocyte antigens (HLA) for each sample based on WES data. Then we used netMHCpan to predict HLA peptide binding affinity for we used netMHCpan to predict HLA peptide binding affinity for somatic mutation\\u2013derived variant peptides with a length between 8\\u201311 amino acids. The cutoff of IC50 binding affinity was set to 150 nM. HLA peptides with binding affinity higher than 150 nM 150 nM. HLA peptides with binding affinity higher than 150 nM were removed. Variant identification was also performed at both mRNA and protein levels using RNA-Seq data and MS/MS data, respectively. Variant identification and gene quantification using Variant identification and gene quantification using RNA-Seq data were performed following the methods used in the previous study. To identify variant peptides, we used a customized protein sequence database approach. We built a customized database for sequence database approach. We built a customized database for each TMT experiment based on somatic variants from WES data. We used Customprodbj (https://github.com/bzhanglab/customprodbj) for customized database construction. MS-GF+ was used for variant customized database construction. MS-GF+ was used for variant peptide identification for all global proteome and phosphorylation data. Results from MS-GF+ were filtered with 1% FDR at peptide level. Remaining variant peptides were further filtered using level. Remaining variant peptides were further filtered using PepQuery (http://www.pepquery.org) with the p-value cutoff <= 0.01. Competitive filtering based on unrestricted posttranslational modification searching was enabled in PepQuery validation. The modification searching was enabled in PepQuery validation. The spectra of variant peptides were annotated using PDV (http://www.zhang-lab.org/). Cancer/testis Antigen Prediction\\n\\nCancer/testis (CT) antigens were downloaded from the CTdatabase. CT antigens with a median >4-fold increase in tumor from NAT were highlighted.\\n\\nPROGENy Scores PROGENy Scores\\n\\nPROGENy was used to generate activity scores for EGFR based on RNA expression data. Tumor RNA expression values were submitted to PROGENy.\\n\\nLSCC, HNSCC and LUAD integrative analysis LUAD data for 110 lung adenocarcinoma samples and 102 NAT were acquired from the published manuscript. HNSCC data for 108 head and neck squamous cell carcinoma samples and 66 NAT were acquired from the submitted manuscript. The LUAD and HNSCC proteomics from the submitted manuscript. The LUAD and HNSCC proteomics data were processed, quantified, and normalized using the same pipeline as described for the LSCC samples. Differential expression analysis for all three cohorts was performed as described in analysis for all three cohorts was performed as described in the Differential marker analysis method. Proteins missing in more than 50% of the paired tumor/NAT samples were excluded. Copy number drivers for all three were assessed by filtering the 8309 genes that were quantified in all three cohorts in the copy number, RNA, and proteomic data to those found in focal amplification regions (q value < .25). Spearman correlation was amplification regions (q value < .25). Spearman correlation was performed for these genes between CNA and RNA and between CNA and protein. Proteins were considered drivers if the correlation between both the CNA and RNA and CNA and protein were between both the CNA and RNA and CNA and protein were significantly positively associated (BH adjusted p value <0.01). Proteins were also required to be significantly increased in tumor vs paired NAT in the same cohort (Wilcoxon signed rank BH adjusted p NAT in the same cohort (Wilcoxon signed rank BH adjusted p value <0.01). To identify genes associated with the immune score, correlation between CNA, RNA, protein, and the immune score was performed for the genes present in both CNA and RNA in a cohort (20,313 genes in LSCC, 18,091 genes in LUAD, and 21,616 genes in HNSCC). in LSCC, 18,091 genes in LUAD, and 21,616 genes in HNSCC). The immune score was calculated as the z-score transformation of the ESTIMATE Immune score, which was calculated for all three cohorts as described in the method Estimation of Tumor Purity, cohorts as described in the method Estimation of Tumor Purity, Stromal and Immune Scores. To be considered copy number drivers of the immune score, genes had to have a positive correlation between CNA & RNA, CNA & immune score, and RNA & immune score. If CNA & RNA, CNA & immune score, and RNA & immune score. If the protein was quantified in the cohort, the gene was also required to have a positive correlation between CNA & protein and protein & immune score. Spearman correlations with a BH adjusted p & immune score. Spearman correlations with a BH adjusted p value < 0.01 were considered significant. Supplementary Material\\n\\nDeclaration of Interests\\n\\nThe authors declare no competing interests\\n\\nADDITIONAL RESOURCES ADDITIONAL RESOURCES\\n\\nThe CPTAC program website, detailing program initiatives, investigators, and datasets, is found at https://proteomics.cancer.gov/programs/cptac. A website for interactive visualization of the multi-omic dataset is available at: https://rstudio-connect.broadapps.org/CPTAC-LSCC2021/ All processed data matrices are available at LinkedOmics (http://www.linkedomics.org), where computational tools are available for further exploration of this dataset.\\n\\nConsortia Alex Green, Alexey Nesvizhskii, Alfredo Molinolo, Alicia Francis, Amanda Paulovich, Ana Robles, Andrii Karnuta, Antonio Colaprico, Arul Chinnaiyan, Azra Krek, Barbara Hindenach, Barbara Pruetz, Bartosz Kubisa, Bing Zhang, Bo Wen, Boris Reva, Brian Druker, Bartosz Kubisa, Bing Zhang, Bo Wen, Boris Reva, Brian Druker, Carissa Huynh, Chandan Kumar-Sinha, Charles Goldthwaite Jr., Chelsea Newton, Chen Huang, Chet Birger, Chia-Kuei Mo, Christopher Kinsinger, Corbin Jones, DR Mani, Dan Rohrer, Dana Valley, Daniel Corbin Jones, DR Mani, Dan Rohrer, Dana Valley, Daniel Chan, David Chesla, David Feny\\u00f6, David Heiman, Dmitry Rykunov, Donna Hansel, Elena Ponomareva, Elizabeth Duffy, Eric Burks, Eric Jaehnig, Eric Schadt, Erik Bergstrom, Eugene Fedorov, Eunkyung An, Fei Eric Schadt, Erik Bergstrom, Eugene Fedorov, Eunkyung An, Fei Ding, Felipe da Veiga Levoprost, Fernanda Martins Rodrigues, Francesca Petralia, Gad Getz, Galen Hostetter, George Wilson, Gilbert Omenn, Henry Rodriguez, Houxiang Zhu, Hui Zhang, Jackson Omenn, Henry Rodriguez, Houxiang Zhu, Hui Zhang, Jackson White, James Suh, Jennifer Eschbacher, Jennifer Maas, Jonathan Lei, Jiayi Ji, Kai Li, Karen Christianson, Karen Ketchum, Karin Rodland, Karl Clauser, Karsten Krug, Katherine Hoadley, Kei Suzuki, Karl Clauser, Karsten Krug, Katherine Hoadley, Kei Suzuki, Kelly Ruggles, Ki Sung Um, Li Ding, Liqun Qi, Lori Bernard, Harry Kane, Maciej Wiznerowicz, MacIntosh Cornwell, Ma\\u0142gorzata Wojty\\u015b, Marcin Cieslik, Marcin Domagalski, Mathangi Thiagarajan, Matthew Cieslik, Marcin Domagalski, Mathangi Thiagarajan, Matthew Ellis, Matthew Wyczalkowski, Maureen Dyer, Meenskshi Anurag, Mehdi Mesri, Melissa Borucki, Michael Birrer, Michael Gillette, Midie Xu, Mikhail Krotevich, Myvizhi Esai Selvan, Namrata Udeshi, Nancy Mikhail Krotevich, Myvizhi Esai Selvan, Namrata Udeshi, Nancy Roche, Nathan Edwards, Negin Vatanian, Neil Mucci, Nicollette Maunganidze, Nikolay Gabrovski, Olga Potapova, Oluwole Fadare, \\u00d6zg\\u00fcn Babur, Pamela Grady, Pankaj Vats, Paul Paik, Pei Wang, Peter Babur, Pamela Grady, Pankaj Vats, Paul Paik, Pei Wang, Peter McGarvey, Pierre Jean Beltran, Pushpa Hariharan, Qing Kay Li, Ramaswamy Govindan, Ratna Thangudu, Rebecca Montgomery, Richard Smith,Robert Welsh, Runyu Hong, Sailaja Mareedu, Samuel Payne, Smith,Robert Welsh, Runyu Hong, Sailaja Mareedu, Samuel Payne, Sandra Cottingham, Sara Savage, Saravana Dhanasekaran, Scott Jewell, Sendurai Mani, Seungyeul Yoo, Shankara Anand, Shankha Satpathy, Shayan Avanessian, Shilpi Singh, Shirley Tsang, Shuang Cai, Shayan Avanessian, Shilpi Singh, Shirley Tsang, Shuang Cai, Song Cao, Stacey Gabriel, Stephan Schurer, Steven Carr, Suhas Vasaikar, Tao Liu, Tara Hiltke, Tatiana Omelchencko, Thomas Bauer, Tobias Schraink, Vasileios Stathias, Volodymyr Sovenko, Warren Tobias Schraink, Vasileios Stathias, Volodymyr Sovenko, Warren Tourtellotte, Weiping Ma, Wenke Liu, William Bocik, Wohaib Hasan, Xiaoyu Song, Yifat Geffen, Yize Li, Yongchao Dou, Yuping Zhang, Yuxing Liao, Yvonne Shutack, Zeynep Gumus, Zhen Zhang, Ziad Yuxing Liao, Yvonne Shutack, Zeynep Gumus, Zhen Zhang, Ziad Hanhan. This is a PDF file of an article that has undergone enhancements after acceptance, such as the addition of a cover page and metadata, and formatting for readability, but it is not yet the definitive version of record. This version will undergo additional version of record. This version will undergo additional copyediting, typesetting and review before it is published in its final form, but we are providing this version to give early visibility of the article. Please note that, during the production of the article. Please note that, during the production process, errors may be discovered which could affect the content, and all legal disclaimers that apply to the journal pertain. Proteogenomic Landscape of LSCC A. Schematic showing the number of tumors and NATs profiled and the various data types generated in this study. The lower panel represents data completeness. WGS: Whole Genome Sequencing, WES: Whole Exome Sequencing. CNA: copy number alteration. DNAme: Whole Exome Sequencing. CNA: copy number alteration. DNAme: DNA methylation. pSTY: phosphoproteome. Ac: acetylproteome. Ub: ubiquitylproteome. B. Stacked histograms indicating the distribution of patient phenotypes. Smoking History reflects self-report. C. Co-occuring mutation plot indicating cancer-relevant genes. MutSig-based significantly mutated genes (SMGs, q-value < 0.1) in this dataset are highlighted in red font. D. Heatmaps showing correlation between copy number alterations (CNA) and RNA (left) or proteomics (right). Red and green events represent significant (FDR <0.01) positive and negative correlations, respectively. E. Flow chart for identification of cancer-associated genes (CAGs) that showed GISTIC-based focal amplification or deletion (q<0.25) and cis-effects in both mRNA and protein (FDR<0.05). F. Differential protein expression (Log2 fold-change (FC)) in tumors with and without high-level amplification of the FGFR1 gene (GISTIC thresholded value =2). G. Genes whose DNA methylation was significantly associated with cascading cis regulation of their cognate mRNA expression, protein level, phosphopeptide and acetylpeptide abundance. Shapes indicate the cis-effects across the indicated datasets. Named indicate the cis-effects across the indicated datasets. Named genes also showed differential expression between tumors and NATs. See also Figure S1 and Table S1\\u20133\\n\\nLSCC Molecular Subtypes and Associations\\n\\nA. NMF-based clustering of tumor CNA, RNA, protein, phosphosite and acetylsite profiles, showing five primary NMF subtypes (top sample annotation row). B. Heatmap representing significantly enriched pathways (MSigDB Hallmark) in five multi-omic subtypes. C. Kaplan-Meier plot comparing survival probability of patients whose tumors were core members of a specific NMF subtype (NMF Core) to those whose tumors had characteristics of more than one NMF subtype (NMF mixed). D. Heatmap showing relative overexpression of mesenchymal proteins in the EMT-E subtype compared to others. E. Correlation between ssGSEA-based enrichment of EMT (Hallmark genesets) and fibroblast proliferation (GO: Gene Ontology) genesets (Pearson correlation=0.65, p=2.8\\u00d710\\u221214). F. Distribution of RTK correlation-based phosphosite enrichment (RTK CBPE) scores for PDGFRB and ROR2 across the five NMF subtypes. Wilcoxon p values for CBPE scores in EMT-E vs other subtypes are 1.5\\u00d710\\u22126 for PDGFRB and 2.7\\u00d710\\u22127 for ROR2. G. Proteins and phosphosites significantly associated with PDGFRB or ROR2 CBPE scores, known to play a role in EMT and extracellular matrix reorganization. The left panel shows Spearman correlation between PDGFRB CBPE scores and protein/phosphosite correlation between PDGFRB CBPE scores and protein/phosphosite abundance profiles. See also Figure S2 and Table S1\\u20134\\n\\nImpact of Somatic Mutations on Proteogenomic Features A. Significant (Wilcoxon FDR<0.05) cis- (circles) and trans-effects (squares) of selected mutations (x axis) on the expression of cancer-associated gene products, with mRNA in blue and proteins in green.\\n\\nB. Similar to panel A but showing phosphosites. B. Similar to panel A but showing phosphosites.\\n\\nC. Similar to panel A but showing acetylsites.\\n\\nD. Similar to panel A but showing ubiquitylsites. E. CNA data for CDKN2A and RB1 was used to classify tumors as having homozygous deletions or three classes of loss of heterozygosity mutations: nonsense/frameshift indel, missense/inframe indel, and splicing (see Table S4). CDKN2A genetic and indel, and splicing (see Table S4). CDKN2A genetic and hypermethylation annotations were based on the effect of the aberration on the p16INK4a (p16) gene product, but the effects of these CDKN2A/p16 aberrations on both major isoforms (p16INK4a and p14ARF aberrations on both major isoforms (p16INK4a and p14ARF (p14)) at the RNA (barplot) and protein (heatmap directly below barplot) levels are shown. Samples with amplification of CCND1\\u20133, CDK4, and CDK6 were assessed by GISTIC (threshold = 2), and the CDK4, and CDK6 were assessed by GISTIC (threshold = 2), and the genomic status, protein, and phosphoprotein levels for RB1 are included. Also shown are RNA-based scores for the cell cycle (MGPS, the mean of cell cycle genes and E2F target and G2M (MGPS, the mean of cell cycle genes and E2F target and G2M checkpoint gene set scores derived from ssGSEA of Hallmark gene sets) and phosphosite-based CDK kinase activity scores for CDKs 1, 2, and 4 derived from single sample post translational CDKs 1, 2, and 4 derived from single sample post translational modifications - signature enrichment analysis (ssPTM-SEA) of known kinase targets. Three tumors with copy number gain of CDKN2A are not included. F. Correlation between differential regulation of protein abundance (Log2 Fold-change (FC)) versus phosphoprotein log2 FC in tumors with NRF2 pathway mutation (one or two mutations in KEAP1, CUL3, or NFE2L2) versus NRF2 WT tumors (no NRF2 pathway KEAP1, CUL3, or NFE2L2) versus NRF2 WT tumors (no NRF2 pathway aberration). G. NRF2 pathway score and RNA, protein and phosphoprotein expression of key NRF pathway members according to NMF subtype.\\n\\nH. CDK5 protein expression (Log2 TMT ratio) by NMF subtype. P-values are from the Anova test. I. PTM-SEA-derived normalized enrichment scores (NES) for pathways enriched in NRF2 pathway-mutated (Mutant) vs wild-type samples (WT) plotted against NES for pathways enriched in NMF Classical vs other subtypes. Significantly upregulated (FDR<0.05) vs other subtypes. Significantly upregulated (FDR<0.05) PTM-SEA terms in the Classical subtype are indicated by red dots and labeled. See also Figure S3 and Table S4\\n\\nProteogenomic Impact of Chromosome 3q Amplification\\n\\nA. Differential protein expression (Log2 FC) between tumors and NATs for genes on chromosome arm 3q. B. Frequency distribution of \\u0394Np63\\u03b1 RNA expression in tumors and NATs. C. Differential protein expression in samples classified as \\u0394Np63\\u03b1-low vs -high based on \\u0394Np63\\u03b1 transcript level. The outlier upregulated gene product in red is BIRC5, also known as survivin. D. Differential expression of microRNAs in \\u0394Np63\\u03b1-low vs -high samples.\\n\\nE. Pearson correlation between expression of miR-205 and mRNA expression of its cognate, experimentally validated targets. F. Pearson correlation between expression of miR-205 and protein abundance of its cognate targets. G. Relationship between miR-205 expression (log2 TPM) and EMT (top, p=2.3\\u00d710\\u221208, Correlation = \\u22120.53) and DNA replication (bottom, p=1.1\\u00d710\\u221206, Correlation= 0.47) scores. H. Heatmap showing relative protein expression (TMT log ratio) of selected proteins with significant (FDR<0.01) Pearson correlation values (positive or negative) with SOX2 protein expression. I. Pearson correlation (p= 5.2\\u00d710\\u221207, Correlation = \\u22120.46) between SOX2 protein expression and HALLMARK_IL6_JAK-STAT_signaling NES.\\n\\nSee also Figure S4 and Table S4\\n\\nUbiquitylation landscape in LSCC A. Consensus clustering of protein ratio-corrected K-GG (di-glycine) site abundances in tumor samples and their associations. Heatmap shows only protein ratio-corrected K-GG sites with differential abundance across clusters (FDR<0.01). Enriched pathways abundance across clusters (FDR<0.01). Enriched pathways and molecular and clinical annotations are indicated. B. Number of K-GG sites showing significant correlations (FDR< 0.01) with E3 ligases. Shown are the five E3 ligases with the highest proportion of positive correlations. C. Pearson correlations between HERC5 protein expression and K-GG site protein-corrected abundance in key glycolytic enzymes PKM, PGK1, and ENO1. D. HERC5 protein expression (log2 TMT ratio) with samples grouped by immune subtype. Significant (Kruskal\\u2212Wallis, p = 2.8\\u00d710\\u221205) association is seen between HERC5 abundance and immune subtypes. E. Representative examples of significant spatial clustering of lysine acetylsites (purple) on PGK1 (left) and ACADVL (right) protein 3-D structure space-filling models (cyan) as determined by PTM CLUMPS. (PGK1 structure = PDB ID:3ZOZ. ACADVL structure = PTM CLUMPS. (PGK1 structure = PDB ID:3ZOZ. ACADVL structure = PDB ID:3B9). F. TXN protein levels in the NMF Classical subtype relative to NATs and other NMF subtypes (top left). Protein-corrected ubiquitylation (K-GG) sites are decreased on TXN1 in tumor subtypes relative to NATs (top right). TXNIP is decreased in tumor subtypes to NATs (top right). TXNIP is decreased in tumor subtypes relative to NATs (lower left). TXN1 activator TXNRD1 is increased in the Classical subtype relative to NATs and other NMF subtypes (lower right). Kruskal\\u2212Wallis p-values are indicated in the (lower right). Kruskal\\u2212Wallis p-values are indicated in the respective plots. G. Schematic representation of PTM-based modulation in LSCC tumors showing key enzymes in the metabolic and reactive oxygen species (ROS) pathways. Green and red arrows indicate higher and lower abundance of the corresponding PTMs in tumors. Putative lower abundance of the corresponding PTMs in tumors. Putative ISGylation targets of HERC5 are indicated by dotted lines. A known regulatory PKM phosphosite observed to be modulated in LSCC tumors is also highlighted. H. Lollipop charts showing the Log2 FC of acetylated (K-Ac) and ubiquitylated sites (K-GG) between tumors and NATs (Hyper: log2(FC)>1 or Hypo: <\\u22121, FDR<0.01). The upper panel shows specific sites that were hyper-ubiquitylated and hypoacetylated in tumors; that were hyper-ubiquitylated and hypoacetylated in tumors; the lower panel shows specific sites that were hyperacetylated and hypo-ubiquitylated in tumors. Dot colors indicate protein fold change between tumors and NATs. \\u201ck\\u201d represents modified lysine. I. Relative abundances of RAN K127 acetylation (K-Ac), ubiquitylation (K-GG) and RAN protein levels across NMF subtypes and NATs. Wilcoxon p-values are indicated above; ns represents p>0.05.\\n\\nSee also Figure S5 and Table S5\\n\\nImmune Landscape of LSCC A. Heatmaps illustrate cell type compositions and activities of selected individual genes/proteins and pathways across the four immune clusters: Hot, Warm, and Cold tumor and NAT-enriched. Successive heatmaps illustrate xCell immune signatures, mRNA and heatmaps illustrate xCell immune signatures, mRNA and protein expression of key immune-related markers, and ssGSEA pathway scores based on global proteomic data for biological pathways that were differentially regulated in immune groups based on both mRNA differentially regulated in immune groups based on both mRNA and global protein abundance (Common) or on global protein abundance alone (Proteomics only). B. Pathway scores of key pathways differentially expressed across the immune clusters. Wilcoxon p-values for the individual comparisons are provided on top. C. Contour plot of two-dimensional density based on Macrophage (x-axis) and CD8 T-cell scores (y-axis) showing the variation in these cell types\\u2019 distributions observed across the different immune clusters. D. Acetylsites differentially expressed between Hot and Cold tumors. Acetylsites of genes contained in the Hallmark Oxidative Phosphorylation pathway are highlighted in blue, ARHGDIB K135 is highlighted in red, and remaining sites are in gray. Darker is highlighted in red, and remaining sites are in gray. Darker color designates significant sites (FDR < 0.1). E. Regulation of Rho GTPase signaling including K135 acetylation of ARHGDIB. F. Global protein abundance of RAC2, DOCK2 and ELMO1, acetylproteome abundance of ARHGDIB K135k and phosphorylation abundance of ARHGEF6 at Serine 225 in immune clusters. Wilcoxon p-values are reported. G. RTK CBPE scores for 108 tumor samples and associated xCell signatures and pathway scores. The first heatmap shows CBPE scores of key RTKs, the second xCell signatures and the third pathway scores based on global protein abundance. H. Distribution of RTK CBPE scores for CSF1R, PDGFRB and FGFR2 stratified by immune clusters. Significance values (two-sided Wilcoxon test) between Hot clusters and combined Warm and Cold clusters are indicated on the violin plots. I. Heatmap showing proteins and phosphosites correlated with CSF1R CBPE scores that are known to be involved in immune evasion. See also Figure S6 and Table S6\\n\\nProteomic Features Related to Diagnosis, Prognosis, or Treatment A. Differentially expressed proteins between tumors and NATs. B. Significantly increased proteins (larger font indicates >4 fold) in the study LSCC cohort that are associated with poor overall survival (OS) or disease-free survival (DFS) in the TCGA LSCC cohort mRNA. C. Genetic dependencies of 502 proteins (log2 FC >2, FDR<0.01 and NAs <50%) in LSCC cell lines (n=16) profiled as part of the Achilles Dependency-Map project. D. Genes, ordered by their chromosomal location, that are deleted in at least 25% of the samples and significantly correlated to the immune score. Immune-related genes are highlighted. E. EGFR protein and tyrosine phosphorylation levels compared to EGFR copy number and an EGFR activity score (PROGENy). F. Heatmap showing Pearson correlation between the EGFR activity score represented by PROGENy (top) and RNA expression of EGFR ligands. *p<0.05 G. GO Biological Process enrichment for proteins with increased phosphorylation in EGFR amplified samples compared to non-amplified samples.\\n\\nH. Summary roadmap figure partitioned into five major categories, indicated by different colors. See also Figure S7 and Table S7 REAGENT or RESOURCE\\tSOURCE\\tIDENTIFIER\\t \\tAntibodies\\t \\tPTMScan\\u00ae Ubiquitin Remnant Motif (K-\\u03b5-GG) Kit\\tCell Signaling Technology\\tCatalog 5562\\t \\tPTMScan Acetyllysine Kit\\tCell Signaling Technology\\tCatalog: 13416\\t \\tCD4\\tDako\\tClone 4B12, RRID:AB_2728838 13416\\t \\tCD4\\tDako\\tClone 4B12, RRID:AB_2728838\\t \\tCD8\\tBio-Rad\\tClone 4B11, RRID:AB_322868\\t \\tCD163\\tAbcam\\tAb182422, RRID:AB_2753196\\t \\tARHGDIB\\tSigma-Aldrich\\tHPA051235, RRID:AB_2681398\\t \\tCK (pan-Cytokeratin)\\tDako\\tclone AE1/AE3, RRID:AB_2132885 CK (pan-Cytokeratin)\\tDako\\tclone AE1/AE3, RRID:AB_2132885\\t \\talpha-SMA\\tAbcam\\tab5694, RRID:AB_2223021\\t \\tBiological Samples\\t \\tPrimary tumor samples\\tSee Experimental Model and Subject Details\\tN/A\\t \\tChemicals and Reagents\\t \\tHPLC-grade water\\tJ.T. Baker\\tCatalog: Chemicals and Reagents\\t \\tHPLC-grade water\\tJ.T. Baker\\tCatalog: 4218-03\\t \\tUrea\\tSigma\\tCatalog: U0631\\t \\tSodium chloride\\tSigma\\tCatalog: 71376\\t \\t1M Tris, pH 8.0\\tInvitrogen\\tCatalog: AM9855G\\t \\tEthylenediaminetetraacetic acid\\tSigma\\tCatalog: E7889 AM9855G\\t \\tEthylenediaminetetraacetic acid\\tSigma\\tCatalog: E7889\\t \\tAprotinin\\tSigma\\tCatalog: A6103\\t \\tLeupeptin\\tRoche\\tCatalog: 11017101001\\t \\tPhenylmethylsulfonyl fluoride\\tSigma\\tCatalog: 78830\\t \\tSodium fluoride\\tSigma\\tCatalog: S7920\\t \\tPhosphatase inhibitor Sodium fluoride\\tSigma\\tCatalog: S7920\\t \\tPhosphatase inhibitor cocktail 2\\tSigma\\tCatalog: P5726\\t \\tPhosphatase inhibitor cocktail 3\\tSigma\\tCatalog: P0044\\t \\tDithiothretiol, No-Weigh Format\\tThermoScientific\\tCatalog: 20291\\t \\tIodoacetamide\\tSigma\\tCatalog: A3221 20291\\t \\tIodoacetamide\\tSigma\\tCatalog: A3221\\t \\tLysyl endopeptidase\\tWako Chemicals\\tCatalog: 129-02541\\t \\tSequencing-grade modified trypsin\\tPromega\\tCatalog: V511X\\t \\tFormic acid\\tSigma\\tCatalog: F0507\\t \\tAcetonitrile, LC-MS grade\\tHoneywell\\tCatalog: 34967 F0507\\t \\tAcetonitrile, LC-MS grade\\tHoneywell\\tCatalog: 34967\\t \\tAcetonitrile, anhydrous\\tSigma\\tCatalog: 271004\\t \\tTrifluoroacetic acid\\tSigma\\tCatalog: 302031\\t \\tTandem Mass Tag reagent kit \\u2013 11plex\\tThermoFisher\\tCatalog: A34808\\t \\t0.5M HEPES, pH 8.5\\tAlfa \\u2013 11plex\\tThermoFisher\\tCatalog: A34808\\t \\t0.5M HEPES, pH 8.5\\tAlfa Aesar\\tCatalog: J63218\\t \\tHydroxylamine solution, 50% (vol/vol) in H2O\\tAldrich\\tCatalog: 467804\\t \\tMethanol\\tHoneywell\\tCatalog: 34966\\t \\tAmmonium hydroxide solution, 28% (wt/vol) in 34966\\t \\tAmmonium hydroxide solution, 28% (wt/vol) in H2O\\tSigma\\tCatalog: 338818\\t \\tNi-NTA agarose beads\\tQiagen\\tCatalog: 30410\\t \\tIron (III) chloride\\tSigma\\tCatalog: 451649\\t \\tAcetic acid, glacial\\tSigma\\tCatalog: AX0073\\t \\tPotassium phosphate, acid, glacial\\tSigma\\tCatalog: AX0073\\t \\tPotassium phosphate, monobasic\\tSigma\\tCatalog: P0662\\t \\tPotassium phosphate, dibasic\\tSigma\\tCatalog: P3786\\t \\tMOPS\\tSigma\\tCatalog: M5162\\t \\tSodium hydroxide\\tVWR\\tCatalog: BDH7225\\t \\tSodium phosphate, dibasic\\tSigma\\tCatalog: BDH7225\\t \\tSodium phosphate, dibasic\\tSigma\\tCatalog: S9763\\t \\tPhosphate-buffered saline\\tFisher Scientific\\tCatalog: 10010023\\t \\tiVIEW DAB Detection Kit\\tRoche\\tCatalog: 760-091\\t \\tEquipment\\t \\tReversed-phase tC18 SepPak, 3cc 200mg\\tWaters\\tCatalog: WAT054925 tC18 SepPak, 3cc 200mg\\tWaters\\tCatalog: WAT054925\\t \\tSolid-phase C18 disk, for Stage-tips\\tEmpore\\tCatalog: 66883-U\\t \\tStage-tip needle\\tCadence\\tCatalog: 7928\\t \\tStage-tip puncher, PEEK tubing\\tIdex Health & Science\\tCatalog: 1581\\t \\tPicoFrit LC-MS column\\tNew Health & Science\\tCatalog: 1581\\t \\tPicoFrit LC-MS column\\tNew Objective\\tCatalog: PF360-75-10-N-5\\t \\tReproSil-Pur, 120 \\u212b, C18-AQ, 1.9-\\u03bcm resin\\tDr. Maisch\\tCatalog: r119.aq\\t \\tNanospray column heater\\tPhoenix S&T\\tCatalog: PST-CH-20U\\t \\tColumn heater column heater\\tPhoenix S&T\\tCatalog: PST-CH-20U\\t \\tColumn heater controller\\tPhoenix S&T\\tCatalog: PST-CHC\\t \\t300 \\u03bcL LC-MS autosampler vial and cap\\tWaters\\tCatalog: 186002639\\t \\tOffline HPLC column, 3.5-\\u03bcm particle size, 4.6 um \\u00d7 250 mm\\tAgilent\\tCatalog: Custom 3.5-\\u03bcm particle size, 4.6 um \\u00d7 250 mm\\tAgilent\\tCatalog: Custom order\\t \\tOffline 96-well fractionation plate\\tWhatman\\tCatalog: 77015200\\t \\t700 \\u03bcL bRP fractionation autosampler vial\\tThermoFisher\\tCatalog: C4010-14\\t \\t700 \\u03bcL bRP fractionation autosampler C4010-14\\t \\t700 \\u03bcL bRP fractionation autosampler cap\\tThermoFisher\\tCatalog: C4010-55A\\t \\t96-well microplate for BCA\\tGreiner\\tCatalog: 655101\\t \\tMicroplate foil cover\\tCorning\\tCatalog: PCR-AS-200\\t \\tVacuum centrifuge\\tThermoFisher\\tCatalog: SPD121P-115 Vacuum centrifuge\\tThermoFisher\\tCatalog: SPD121P-115\\t \\tCentrifuge\\tEppendorf\\tCatalog: 5427 R\\t \\tBenchtop mini centrifuge\\tCorning\\tCatalog: 6765\\t \\tBenchtop vortex\\tScientific Industries\\tCatalog: SI-0236\\t \\tIncubating shaker\\tVWR\\tCatalog: 12620-942\\t \\t15 mL SI-0236\\t \\tIncubating shaker\\tVWR\\tCatalog: 12620-942\\t \\t15 mL centrifuge tube\\tCorning\\tCatalog: 352097\\t \\t50 mL centrifuge tube\\tCorning\\tCatalog: 352070\\t \\t1.5 mL microtube w/o cap\\tSarstedt\\tCatalog: 72.607\\t \\t2.0 mL microtube w/o cap\\tSarstedt\\tCatalog: 72.608 72.607\\t \\t2.0 mL microtube w/o cap\\tSarstedt\\tCatalog: 72.608\\t \\tMicrotube caps\\tSarstedt\\tCatalog: 72.692\\t \\t1.5 mL snapcap tube\\tThermoFisher\\tCatalog: AM12450\\t \\t2.0 mL snapcap tube\\tThermoFisher\\tCatalog: AM12475\\t \\tInstrumentation\\t \\tMicroplate Reader\\tMolecular AM12475\\t \\tInstrumentation\\t \\tMicroplate Reader\\tMolecular Devices\\tCatalog: M2\\t \\tOffline HPLC System for bRP fractionation\\tAgilent 1260\\tCatalog: G1380-90000\\t \\tOnline LC for LC-MS\\tThermoFisher\\tCatalog: LC140\\t \\tQ Exactive Plus Mass LC for LC-MS\\tThermoFisher\\tCatalog: LC140\\t \\tQ Exactive Plus Mass Spectrometer\\tThermoFisher\\tCatalog: IQLAAEGAAPFALGMBDK\\t \\tQ Exactive HF-X Mass Spectrometer\\tThermoFisher\\tCatalog: 0726042\\t \\tOrbitrap Fusion Lumos Tribrid Mass Spectrometer\\tThermoFisher\\tCatalog: Fusion Lumos Tribrid Mass Spectrometer\\tThermoFisher\\tCatalog: IQLAAEGAAPFADBMBHQ\\t \\tCritical Commercial Assays\\t \\tTruSeq Stranded Total RNA Library Prep Kit with Ribo-Zero Gold\\tIllumina\\tCatalog: RS-122-2301\\t \\tInfinium MethylationEPIC Kit\\tIllumina\\tCatalog: RS-122-2301\\t \\tInfinium MethylationEPIC Kit\\tIllumina\\tCatalog: WG-317-1003\\t \\tNextera DNA Exosome Kit\\tIllumina\\tCatalog: 20020617\\t \\tKAPA Hyper Prep Kit, PCR-free\\tRoche\\tCatalog: 07962371001\\t \\tBCA Protein Assay Kit\\tThermoFisher\\tCatalog: 23225\\t \\tDeposited Data Protein Assay Kit\\tThermoFisher\\tCatalog: 23225\\t \\tDeposited Data\\t \\tDepMap: Mutation\\tDepMapPublic 21Q1 (PMID: 31068700; Dataset doi:10.6084/m9.figshare.13681534.v1)\\thttps://depmap.org/portal/download/\\t \\tDepMap: Segmented copy number\\tDepMapPublic 21Q1 (PMID: DepMap: Segmented copy number\\tDepMapPublic 21Q1 (PMID: 31068700; Dataset doi:10.6084/m9.figshare.13681534.v1)\\thttps://depmap.org/portal/download/\\t \\tDepMap: Gene level copy number\\tDepMapPublic 21Q1 (PMID: 31068700; Dataset level copy number\\tDepMapPublic 21Q1 (PMID: 31068700; Dataset doi:10.6084/m9.figshare.13681534.v1)\\thttps://depmap.org/portal/download/\\t \\tDepMap: RNAseq (transcript isoform)\\tDepMapPublic 21Q1 (PMID: 31068700; Dataset (transcript isoform)\\tDepMapPublic 21Q1 (PMID: 31068700; Dataset doi:10.6084/m9.figshare.13681534.v1)\\thttps://depmap.org/portal/download/\\t \\tDepMap: Proteomics (RPPA)\\tCCLE 2019 (PMID: 31068700)\\thttps://depmap.org/portal/download/\\t \\tDepMap: CRISPR KO screen DepMap: CRISPR KO screen (combined)\\tDepMapPublic 21Q1 (bioRxiv 2020.05.22.110247; doi: https://doi.org/10.1101/2020.05.22.110247)\\thttps://depmap.org/portal/download/\\t \\tDepMap: shRNA screen (combined)\\tDEMETER2 Data v6 (PMID: DepMap: shRNA screen (combined)\\tDEMETER2 Data v6 (PMID: 30389920)\\thttps://depmap.org/portal/download/\\t \\tDepMap: GDSC drug screen\\tSanger GDSC 1 (PMID: 27397505)\\thttps://depmap.org/portal/download/\\t \\tDepMap: PRISM drug screen\\tPRISM Repurposing 19Q4 DepMap: PRISM drug screen\\tPRISM Repurposing 19Q4 Secondary Screen (PMID: 32613204)\\thttps://depmap.org/portal/download/\\t \\tPhosphoSitePlus\\t\\thttps://www.phosphosite.org\\t \\tConnectivity Map (CMAP)\\t\\thttps://www.broadinstitute.org/connectivity-map-cmap\\t \\tHuman Human Protein Atlas (HPA)\\t\\thttps://www.proteinatlas.org\\t \\tCT Antigen database\\t\\thttp://www.cta.lncc.br\\t \\tDependency map (DepMap)\\t\\thttps://depmap.org/portal/v3.3.8 is a GECKOv2 Achilles dataset\\t \\tLibrary of Integrated Network-based Cellular Signatures Library of Integrated Network-based Cellular Signatures (LINCS)\\t\\thttps://clue.io/dataExpanded CMap LINCS Resource 2020 (1/28/2021 update)\\t \\tCPTAC HNSCC cohort\\t\\thttps://cptac-data-portal.georgetown.edu/study-summary/S054\\t \\tCPTAC LSCC cohort\\tThis CPTAC LSCC cohort\\tThis study\\thttps://cptac-data-portal.georgetown.edu/study-summary/S063\\t \\tSoftware and Algorithms\\t \\tmethylationArrayAnalysis (version 3.9)\\t\\thttps://master.bioconductor.org/packages/release/workflows/html/methylationArrayAnalysis.html Illumina EPIC methylation array (3.9)\\tHansen KD, 2019\\thttps://bioconductor.org/packages/release/data/annotation/html/IlluminaHumanMethylationEPICanno.ilm10b2.hg19.html\\t \\tMethylation array analysis pipeline for CPTAC\\tLi Ding Methylation array analysis pipeline for CPTAC\\tLi Ding Lab\\thttps://github.com/ding-lab/cptac_methylation\\t \\tmiRNA-Seq analysis pipeline for CPTAC\\tLi Ding Lab\\thttps://github.com/ding-lab/CPTAC_miRNA\\t \\tVEP\\t\\thttps://github.com/Ensembl/ensembl-vep/tags VEP\\t\\thttps://github.com/Ensembl/ensembl-vep/tags\\t \\tTNScope / DNAScope (Sentieon)\\t\\tsentieon.com\\t \\tvcfAnno\\t\\thttps://github.com/brentp/vcfanno\\t \\tVariantAnnotation (Bioconductor)\\t\\thttps://bioconductor.org/packages/release/bioc/html/VariantAnnotation.html arriba_v1.1.0\\t\\thttps://github.com/suhrig/arriba/\\t \\tfusioncatcher_v1.10\\t\\thttps://github.com/ndaniel/fusioncatcher/blob/master/doc/manual.md\\t \\teQTLGen\\t\\thttps://github.com/molgenis/systemsgenetics/wiki/eQTL-mapping-analysis-cookbook-(eQTLGen) Pindel0.2.5\\t\\thttp://gmt.genome.wustl.edu/packages/pindel/\\t \\tSignatureAnalyzer\\t\\thttps://software.broadinstitute.org/cancer/cga/msp\\t \\tCNVEX\\tMarcin Cieslik Lab\\thttps://github.com/mctp/cnvex\\t \\tCRISP\\tMarcin Cieslik Lab\\thttps://github.com/mctp/cnvex\\t \\tCRISP\\tMarcin Cieslik Lab\\thttps://github.com/mcieslik-mctp/crisp-build\\t \\tSpectrum Mill\\tKarl R. Clauser, Steven Carr Lab\\thttps://proteomics.broadinstitute.org/\\t \\tComBat Steven Carr Lab\\thttps://proteomics.broadinstitute.org/\\t \\tComBat (v3.20.0)\\t\\thttps://bioconductor.org/packages/release/bioc/html/sva.html\\t \\tgPCA\\t\\thttps://cran.r-project.org/web/packages/gPCA/index.html gPCA\\t\\thttps://cran.r-project.org/web/packages/gPCA/index.html\\t \\tGISTIC2.0\\t\\tftp://ftp.broadinstitute.org/pub/GISTIC2.0/GISTIC_2_0_23.tar.gz\\t \\tiProFun\\t\\thttps://github.com/WangLab-MSSM/iProFun iProFun\\t\\thttps://github.com/WangLab-MSSM/iProFun\\t \\tESTIMATE\\t\\thttps://bioinformatics.mdanderson.org/public-software/estimate/\\t \\tWebGestaltR\\t\\thttp://www.webgestalt.org/\\t \\tGSVA\\t\\thttps://bioconductor.org/packages/release/bioc/html/GSVA.html TSNet\\t\\thttps://github.com/WangLab-MSSM/TSNet\\t \\txCell\\t\\thttp://xcell.ucsf.edu/\\t \\tCPTAC LSCC Data Viewer\\tSteven Carr lab\\thttps://rstudio-connect.broadapps.org/CPTAC-LSCC2021/ Carr lab\\thttps://rstudio-connect.broadapps.org/CPTAC-LSCC2021/\\t \\tConsensusClusterPlus\\t\\thttp://bioconductor.org/packages/release/bioc/html/CancerSubtypes.html\\t \\tMS-GF+\\t\\thttps://github.com/MSGFPlus/msgfplus\\t \\tNeoFlow\\t\\thttps://github.com/bzhanglab/neoflow NeoFlow\\t\\thttps://github.com/bzhanglab/neoflow\\t \\tnetMHCpan\\t\\thttp://www.cbs.dtu.dk/services/NetMHCpan/\\t \\tOptitype\\t\\thttps://github.com/FRED-2/OptiType\\t \\tCustomprodbj\\t\\thttps://github.com/bzhanglab/customprodbj\\t \\tPDV\\t\\thttps://github.com/wenbostar/PDV PDV\\t\\thttps://github.com/wenbostar/PDV\\t \\tPepQuery\\t\\thttp://pepquery.org\\t \\tPTM-SEA\\t)\\thttps://github.com/broadinstitute/ssGSEA2.0\\t \\tPTMsigDB\\t)\\thttp://prot-shiny-vm.broadinstitute.org:3838/ptmsigdb-app/\\t \\tTerra\\tBroad Institute data science Terra\\tBroad Institute data science platform.\\thttps://terra.bio/\\t \\tPanoply\\tBroad Institute Proteomics Platform\\thttps://github.com/broadinstitute/PANOPLY\\t \\tCMap\\t\\thttps://clue.io/cmap\\t \\tLIMMA v3.36 (R CMap\\t\\thttps://clue.io/cmap\\t \\tLIMMA v3.36 (R Package)\\t\\thttps://bioconductor.org/packages/release/bioc/html/limma.html\\t \\tFactoMineR v1.41NMF(R - package)\\t\\thttps://cran.r-project.org/web/packages/FactoMineR/index.html\\t \\tMClust v5.4 (R package)\\t(Scrucca, MClust v5.4 (R package)\\t(Scrucca, Fop, Murphy and Raftery, 2017)\\thttps://cran.r-project.org/web/packages/mclust/index.html\\t \\tg:Profiler\\t\\thttps://biit.cs.ut.ee/gprofiler/gost\\t \\tCytoscape\\t\\thttps://cytoscape.org/ Cytoscape\\t\\thttps://cytoscape.org/\\t \\tImmuneSubtypeClassifier\\t\\thttps://github.com/CRI-iAtlas/ImmuneSubtypeClassifier\\t \\tProteinPaint\\t\\thttps://pecan.stjude.doud/proteinpaint/\\t \\tOrdinal\\tChristensen RHB (2019)\\thttps://CRAN.R-project.org/package=ordinal\",\n",
            "                    \"pdc_study_id\": \"PDC000232\",\n",
            "                    \"title\": \"A proteogenomic portrait of lung squamous cell carcinoma\"\n",
            "                },\n",
            "                {\n",
            "                    \"full_text\": \"A proteogenomic portrait of lung squamous cell carcinoma\\n\\nSummary Lung squamous cell carcinoma (LSCC) remains a leading cause of cancer death with few therapeutic options. We characterized the proteogenomic landscape of LSCC, providing a deeper exposition of LSCC biology with potential therapeutic implications. We of LSCC biology with potential therapeutic implications. We identify NSD3 as an alternative driver in FGFR1-amplified tumors and low-p63 tumors overexpressing the therapeutic target survivin. SOX2 is considered undruggable, but our analyses provide SOX2 is considered undruggable, but our analyses provide rationale for exploring chromatin modifiers such as LSD1 and EZH2 to target SOX2-overexpressing tumors. Our data support complex regulation of metabolic pathways by crosstalk between complex regulation of metabolic pathways by crosstalk between post-translational modifications including ubiquitylation. Numerous immune-related proteogenomic observations suggest directions for further investigation. Proteogenomic dissection of CDKN2A for further investigation. Proteogenomic dissection of CDKN2A mutations argue for more nuanced assessment of RB1 protein expression and phosphorylation before declaring CDK4/6 inhibition unsuccessful. Finally, triangulation between LSCC, LUAD and HNSCC Finally, triangulation between LSCC, LUAD and HNSCC identified both unique and common therapeutic vulnerabilities. These observations and proteogenomics data resources may guide research into the biology and treatment of LSCC. In Brief Comprehensive proteogenomic characterization of lung squamous cell carcinomas and paired normal adjacent tissues identifies taxonomic subclasses, alternative driver events and insights into immune modulation, as well as putative biomarkers and potential immune modulation, as well as putative biomarkers and potential therapeutic targets. Graphical Abstract\\n\\nIntroduction Lung cancer is the leading cause of cancer-associated mortality. Unlike lung adenocarcinomas (LUAD), patients with lung squamous cell carcinoma (LSCC) have not benefited from targeted therapies. Potentially druggable genetic events in three primary Potentially druggable genetic events in three primary pathways (FGFR1, PI3K, or G1/S checkpoint genes such as CDKN2A) are found in upwards of 60% but targeting these clinically has largely failed. While other therapeutic regimens are being explored as failed. While other therapeutic regimens are being explored as part of the LungMAP consortium, to date only immunotherapy has evolved into a successful therapeutic strategy for patients with LSCC. Given the lack of established actionable genomic targets, LSCC. Given the lack of established actionable genomic targets, a comprehensive characterization of the proteogenomic landscape of LSCC, including the associated post-translational modifications (PTMs) that are key to protein activity and signaling, is a (PTMs) that are key to protein activity and signaling, is a crucial step forward. Such studies will aid in connecting genomic aberrations to molecular and clinical phenotypes and in subsequently identifying therapeutic vulnerabilities and effective, identifying therapeutic vulnerabilities and effective, biomarker-based patient stratification. We characterized 108 prospectively-collected, treatment-na\\u00efve, primary LSCC tumors and 99 paired normal adjacent tissues (NATs), identifying actionable therapeutic protein targets and elucidating cellular signaling pathways and crosstalk between multiple cellular signaling pathways and crosstalk between multiple PTMs. The dataset provides an exceptional resource to guide further research and support development of therapeutic modalities in LSCC. Results\\n\\nProteogenomic landscape of LSCC We performed deep-scale molecular analysis spanning nine different data types on LSCC tumors and NATs prospectively collected from 108 patients (Figure 1A, S1A, Table S1\\u20133). Proteomics data were generated using TMT11-based multiplexing linked by a common generated using TMT11-based multiplexing linked by a common reference (Figure S1A) and demonstrated high technical quality (Figure S1B\\u2013C). The cohort demographics are summarized in Figure 1B. Most self-reported never-smokers displayed high genomic smoking Most self-reported never-smokers displayed high genomic smoking scores (Figure S1D), highlighting occasional discrepancy between self-reported smoking status and genomic evidence, as previously reported. The genomic landscape reaffirmed previously reported. The genomic landscape reaffirmed previously reported somatic alterations in LSCC (Figure 1C). Principal component analysis (PCA) of global proteomic and PTM data showed clear separation of tumors and NATs (Figure S1E). Proteogenomic data helped annotate the impact of copy number alteration (CNA) events (Figure 1D,E). Of 5,523 significant (FDR<0.01) cis CNA-mRNA events, 2,154 were concordantly correlated with protein expression, including 138 \\u201ccancer associated genes\\u201d protein expression, including 138 \\u201ccancer associated genes\\u201d (CAG) (Figure 1E, Table S1). We identified six amplified (including WHSC1L1, CCND1, and SOX2) and 29 deleted (including NCOR1, SETD2, and CBL) CAGs in significant focal events (Q<0.25) (Figure and CBL) CAGs in significant focal events (Q<0.25) (Figure 1E, Table S4). Intriguingly, WHSC1L1 (NSD3) is part of a recurrent focal amplicon (8p11.23) in LSCC that encompasses FGFR1. Therapies targeted against FGFR1 have been unsuccessful. Proteomics data targeted against FGFR1 have been unsuccessful. Proteomics data suggested that NSD3, rather than FGFR1, could be the critical driver oncogene within this amplicon (Figure 1F), nominating it as a potential therapeutic target. While this paper was under as a potential therapeutic target. While this paper was under review, NSD3 was demonstrated to be a key regulator of LSCC tumorigenesis, mediating oncogenic chromatin changes. To investigate the impact of CNAs on noncognate gene products, we matched patterns of these significant trans-effects (vertical stripes in Figure 1D) to perturbation profiles from the Connectivity Map (CMap) (https://clue.io/cmap). Six CNAs (IKBKAP, PIN1, Map (CMap) (https://clue.io/cmap). Six CNAs (IKBKAP, PIN1, DNAJB1, IL18, NR2F6, AKAP) (Table S4) were enriched for both amplification and deletion (FDR=0.06) and associated (Fisher\\u2019s exact p<0.01) with clinical metadata. IL-18 deletion and amplification with clinical metadata. IL-18 deletion and amplification correlated with decreased and increased IL-18 protein expression and xCell immune score, respectively, (Figure S1F (upper panel)), consistent with its playing a key modulatory role in the tumor consistent with its playing a key modulatory role in the tumor microenvironment of NSCLC. NR2F6 amplification and deletion were correlated with NR2F6 protein expression and anticorrelated with xCell immune score (Figure S1F (lower panel)). NR2F6 acts as a xCell immune score (Figure S1F (lower panel)). NR2F6 acts as a non-redundant immune checkpoint in cancer, and even partial knockdown works synergistically with PD-L1 blockade. LSCC tumor DNA showed overall hypermethylation relative to NATs (Figure S1G), with tumors separating into CIMP (CpG island methylator phenotype) clusters (Figure S1H). Multi-omic data allowed the identification of \\u201ccascading\\u201d promoter methylation allowed the identification of \\u201ccascading\\u201d promoter methylation cis-effects across cognate mRNA, protein and PTM abundances, supporting their functional significance. Of 90 genes that showed such significant (FDR<0.1) cis-effects in tumors (Table S4), 20 such significant (FDR<0.1) cis-effects in tumors (Table S4), 20 were also differentially expressed (FDR<0.01) between tumors and NATs (Figure 1G). While most were newly associated with LSCC, methylation-driven expression of FAM110A, PTGES3, PLAU and methylation-driven expression of FAM110A, PTGES3, PLAU and SLC16A3 has been implicated in lung and other cancers. Multi-omic clustering identifies five LSCC molecular subtypes, including one that is EMT-Enriched We performed non-negative matrix factorization (NMF)-based single- and multi-omic unsupervised clustering on CNA, RNA, protein, phosphoprotein and acetylprotein datasets from 108 tumors, excluding ubiquitylprotein data as it was not available for the excluding ubiquitylprotein data as it was not available for the entire cohort (Figure 1A). The five resulting multi-omic subtypes (Figure 2A\\u2013B, Figure S2A) were named based on their predominant pathway associations and similarities to previously defined pathway associations and similarities to previously defined RNA clusters. Significant associations between NMF subtypes and cohort metadata are presented in Table S1. Two NMF subtypes, \\u201cBasal-Inclusive\\u201d (B-I) and \\u201cEpithelial to Mesenchymal \\u201cBasal-Inclusive\\u201d (B-I) and \\u201cEpithelial to Mesenchymal Transition-enriched\\u201d (EMT-E), emerged from partitioning of samples that showed similarity to the TCGA-derived RNA Basal cluster, although only B-I retained significant association after Bonferroni only B-I retained significant association after Bonferroni correction (Fisher\\u2019s p<0.01, Table S1). B-I tumors showed basaloid histology and upregulated metabolic, immune and estrogen receptor signaling (Figure 2B). EMT-E showed upregulation of EMT, signaling (Figure 2B). EMT-E showed upregulation of EMT, Angiogenesis and Myogenesis (Figure 2B), with myxoid histologic features and fibroblast infiltration. A third subtype characterized by mutations in KEAP1, CUL3 and NFE2L2 genes and high-level by mutations in KEAP1, CUL3 and NFE2L2 genes and high-level amplification (GISTIC threshold value = 2) of SOX2 and TP63 (Figure 2A\\u2013B, Table S2) was labeled the \\u201cClassical\\u201d subtype, consistent with TCGA nomenclature and with a previous publication. These with TCGA nomenclature and with a previous publication. These showed classical histologic features, CIMP-high enrichment, upregulation of OxPhos- and proliferation-related pathways, and downregulation of immune signaling (Figure 2A\\u2013B). A fourth NMF downregulation of immune signaling (Figure 2A\\u2013B). A fourth NMF subtype was designated \\u201cInflamed-Secretory\\u201d (I-S) due to its alignment with the RNA-based Secretory cluster and strong upregulation of immune-related pathways. The fifth, and strong upregulation of immune-related pathways. The fifth, \\u201cProliferative-Primitive\\u201d (P-P) NMF subtype displayed upregulated proliferation-related pathways, downregulated immune signaling, and enrichment of CIMP-low samples. Within each of these five and enrichment of CIMP-low samples. Within each of these five NMF clusters were samples with low NMF cluster membership scores, which could be considered as belonging to a \\u201cmixed\\u201d subgroup, since they showed features of more than one subtype. Mixed since they showed features of more than one subtype. Mixed subgroup membership, a possible readout of tumor heterogeneity, conferred significantly worse survival (Figure 2C), an observation also made in a recent proteogenomic analysis of glioblastoma. The made in a recent proteogenomic analysis of glioblastoma. The mixed class was associated in the current cohort with increased frequency of SOX2 amplifications (p = 0.0038) but not grade, mutation burden, stemness or other tested variables. We compared B-I (Figure S2B) and EMT-E (Figure S2C) to other subtypes to identify distinguishing features. B-I had hyperphosphorylated M phase Histone 1 T11 and elevated expression of both TACSTD2 (TROP2), a cell-surface glycoprotein that drives growth TACSTD2 (TROP2), a cell-surface glycoprotein that drives growth and metastasis, and MARK2, a kinase that regulates cell polarity and leads to cisplatin resistance. Transcription activator PBX3, which is upregulated only at the protein level (Figure S2B), which is upregulated only at the protein level (Figure S2B), promotes invasion, proliferation, and chemoresistance and is associated with poor prognosis, including in laryngeal squamous cell carcinoma. Pathway analysis of proteins highly correlated with carcinoma. Pathway analysis of proteins highly correlated with PBX3 highlighted neutrophil activation and degranulation (Figure S2D), consistent with the significant upregulation of neutrophilic granule proteins and chemotactic factors (Figure S2E) and granule proteins and chemotactic factors (Figure S2E) and with the xCell-based deconvolution signature (Figure S2F). This correlation was not observed in paired NATs, suggesting a tumor-specific role. In addition to a strong EMT signature (Figure 2B), the EMT-E subtype showed upregulated VCAN (Figure S2C), a tumor-promoting target of SNAIL expressed in cancer-associated fibroblasts (CAFs), and FHL3, a protein that stabilizes EMT-associated (CAFs), and FHL3, a protein that stabilizes EMT-associated transcription factors (Figure S2C). DVL3, FN1 and FHL2, all Wnt pathway regulators, were upregulated in the proteomics dataset, and cell-cell signaling by Wnt was one of the significantly and cell-cell signaling by Wnt was one of the significantly (FDR=0.04) enriched pathways in this subtype. As shown in Figure 2D\\u2013E, coordinated high expression of EMT and CAF proteins, xCell fibroblast scores, and a \\u201cfibroblast proliferation\\u201d signature fibroblast scores, and a \\u201cfibroblast proliferation\\u201d signature suggested that CAFs and tumor epithelium might collaborate in the EMT-E phenotype. This was further supported by enrichment of EMT-related receptor tyrosine kinases (RTKs), as described below. receptor tyrosine kinases (RTKs), as described below. Immunohistochemical co-staining of a small subset of tumors for the epithelial marker pan-cytokeratin (CK) and CAF marker \\u03b1-smooth muscle actin (\\u03b1-SMA) demonstrated the presence in EMT-E tumors of both actin (\\u03b1-SMA) demonstrated the presence in EMT-E tumors of both CAFs and tumor cells undergoing EMT (Figure S2G), with significant enrichment of co-stained cells in the EMT-E subtype (Wilcoxon p=0.025), but not of cells showing single staining of either p=0.025), but not of cells showing single staining of either CK or \\u03b1-SMA alone. TGF\\u03b2 is a well-established and potent inducer of EMT, and a Library of Integrated Network-Based Cellular Signatures-based (LINCS) query for compounds that reversed the EMT-E (LINCS) query for compounds that reversed the EMT-E signature showed enrichment for TGF\\u03b2 inhibitors (Figure S2H, Table S4H). Integrating our global proteome data with a prior LSCC dataset resulted in six clusters largely recapitulating RNA, proteomic and multi-omic clusters defined in the current study (Figure S2I\\u2013J). NMF EMT-E subtype tumors show phosphorylation-driven PDGFR and ROR2 signaling. Though LSCC lacks the RTK mutations for which targeted inhibition has dramatically improved therapeutic options in LUAD, other modes of RTK activation, inferred from phosphoproteomic data, may nonetheless nominate targeted inhibition for some LSCC may nonetheless nominate targeted inhibition for some LSCC patients. Hence, we derived a serine/threonine-predominant correlation-based phosphosite enrichment score (CBPE score) for all RTKs in our tumor cohort. Of nine RTKs with high CBPE scores in LSCC in our tumor cohort. Of nine RTKs with high CBPE scores in LSCC tumors, seven were significantly associated with NMF subtypes (Kruskal-Wallis p <0.01). Scores for PDGFRB and ROR2, markedly elevated in EMT-E (Figure 2F), were highly correlated with the elevated in EMT-E (Figure 2F), were highly correlated with the loss of epithelial cell-cell junction markers and upregulation of mesenchymal proteins involved in remodeling of extracellular matrix, induction of EMT, and promotion of cell migration, which induction of EMT, and promotion of cell migration, which mark the transition from epithelial to mesenchymal state (Figure 2G). As summarized in Figure S2K, we found phosphosite-based evidence for upregulation of both the planar cell polarity and evidence for upregulation of both the planar cell polarity and calcium-dependent branches of non-canonical Wnt signaling, initiated through ROR2 or PDGFR, both targetable and recently implicated in the progression from a hybrid E/M to a highly mesenchymal in the progression from a hybrid E/M to a highly mesenchymal state. Further functional characterization is warranted to confirm the contribution of these RTKs to the EMT-E phenotype and to test the impact of their modulation. Loss of CDK4/6 pathway inhibitors is a universal feature of LSCC but Rb1 expression is variable We investigated the impact of recurrent mutations on cognate RNA, proteins and PTMs (cis-effects) and on a set of cancer associated genes (trans-effects). Significant (FDR<0.05) Cis/Trans pairs are shown in Figure 3A\\u2013D. RB1 mutated tumors had upregulated are shown in Figure 3A\\u2013D. RB1 mutated tumors had upregulated cell cycle-related protein expression. NOTCH1 mutation led to elevated GSK3B protein and phosphorylation (T433), downregulation of inhibitory sites on GSK3A (S21) and upregulation of NLK. NLK of inhibitory sites on GSK3A (S21) and upregulation of NLK. NLK and GSK3B inhibit NOTCH1, recently described as a tumor suppressor in LSCC, suggesting that both might inhibit NOTCH1 downstream signaling in tumors with NOTCH1 mutations. ARID1A is one of signaling in tumors with NOTCH1 mutations. ARID1A is one of the most frequently mutated SWItch Sucrose Non-Fermentable (SWI/SNF) ATP-dependent chromatin remodeling complex members. Mutations were associated with higher tumor mutational burden (Figure S3A) associated with higher tumor mutational burden (Figure S3A) and led to protein downregulation, associated with worse overall survival in NSCLC. Paradoxically, mutations in CDKN2A, proposed as a potential biomarker for CDK4/6 inhibitors, resulted in increased RNA expression (Figure 3A). We analyzed the effects of genetic and epigenetic alterations of CDKN2A on both its major isoforms, p16INK4a alterations of CDKN2A on both its major isoforms, p16INK4a (p16) and p14ARF (p14) (Table S4). As expected, its homozygous deletion resulted in the loss of expression of both isoforms (Figure 3E). However, only eight of the 58 CDKN2A wild-type (WT) samples However, only eight of the 58 CDKN2A wild-type (WT) samples showed p16 RNA expression levels that were comparable to samples with CDKN2A/p16 missense mutations (log2(FPKM+1)>2). Many of those CDKN2A/p16 mutations were unlikely to result in Many of those CDKN2A/p16 mutations were unlikely to result in nonsense-mediated decay (Figure S3B), accounting for preserved expression in those samples. Notably, 28 WT tumors showed hypermethylation of the p16 but not the p14 promoter and suppression of of the p16 but not the p14 promoter and suppression of RNA and protein expression for CDKN2A/p16, while 18 additional WT tumors had reduced expression despite having no clear genetic or epigenetic alteration. Furthermore, the WT samples with high or epigenetic alteration. Furthermore, the WT samples with high CDKN2A/p16 expression had RB1 mutations or deletions and/or low Rb protein levels, consistent with a recently published pan-cancer analysis of the CDK4/6 pathway that showed mutual pan-cancer analysis of the CDK4/6 pathway that showed mutual exclusivity between mutations in these two genes. This suggests that loss of one of these two key CDK4/6 pathway inhibitors is a universal feature of LSCC. Conversely, while amplification of a universal feature of LSCC. Conversely, while amplification of Cyclin D - CDK4/6 complex genes is frequent in LSCC, there was no association between the amplification of CCND2, CCND3, CDK4, or CDK6 and either CDKN2A/p16 protein or Rb CCND2, CCND3, CDK4, or CDK6 and either CDKN2A/p16 protein or Rb protein/phosphoprotein levels (Table S4). CCND1 amplification resulted in significantly higher mean levels of Rb protein and phosphoprotein compared to WT samples, though the distributions phosphoprotein compared to WT samples, though the distributions were overlapping (Figure S3C). Intriguingly, phospho-Rb levels, which reflect CDK4/6 activity, correlated with response to CDK4/6 inhibitors in LSCC cell lines despite the heterogeneous CDK4/6 inhibitors in LSCC cell lines despite the heterogeneous distribution in response for cells with mutations in CDKN2A and copy number alterations in CCND1 and CDKN2A (Figure S3D). LSCC trial data have generally shown minimal efficacy of CDK4/6 LSCC trial data have generally shown minimal efficacy of CDK4/6 inhibition, but outlier responses have been present in trials with abemaciclib and palbociclib. While we found that CCND1 amplification resulted in higher average levels of phosphorylated Rb, resulted in higher average levels of phosphorylated Rb, the heterogeneity of Rb expression and phosphorylation in samples with CCND1 amplification provides a potential explanation for some patients\\u2019 lack of response to CDK4/6 inhibitor therapies. Improved lack of response to CDK4/6 inhibitor therapies. Improved prediction based on a downstream functional assessment of G1/S checkpoint alterations (i.e. RB1 protein expression and phosphorylation) may identify a subset of tumors sensitive to CDK4/6 may identify a subset of tumors sensitive to CDK4/6 inhibition. NRF2 pathway activation in tumors with and without NRF2 pathway mutations LSCC showed mutations of three key genes in the NRF2 antioxidant response pathway, NFE2L2 (NRF2), CUL3 and KEAP1. NFE2L2 phosphorylation was observed in samples with any of these mutations (Figure S3E). LSCC tumors were annotated as \\u201cNRF2 pathway mutated\\u201d S3E). LSCC tumors were annotated as \\u201cNRF2 pathway mutated\\u201d (n=33; with either one hit [n=28] or two hits [n=5] in NFE2L2, KEAP1, and CUL3), or \\u201cNRF2 WT\\u201d (n=68). Compared to WT, NRF2 pathway mutated tumors showed highly concordant differential expression mutated tumors showed highly concordant differential expression of mRNA, protein and phosphoprotein levels of NRF2 pathway genes (Figure 3F, Figure S3F\\u2013G, Table S4), some with therapeutic implications. A subset of NRF2 gene products showed a dosage effect A subset of NRF2 gene products showed a dosage effect (SQSTM1, NR0B1, AKR1B10, CARD11, FFAR2) with the five cases harboring two hits (Table S4) demonstrating increased up-/down-regulation of NRF2 pathway genes relative to one-hit cases (Figure S3H). Using pathway genes relative to one-hit cases (Figure S3H). Using the proteogenomic signature (Table S4) that defined NFE2L2 mutated tumors, we derived an ssGSEA-based NRF2 pathway score that was enriched not only in NRF2 pathway mutated tumors, but also in enriched not only in NRF2 pathway mutated tumors, but also in samples without NRF2 pathway mutations, especially in the NMF Classical subtype (Figure 3G). These samples showed increased NFE2L2 phosphorylation, indicative of NRF2 pathway activation. NFE2L2 phosphorylation, indicative of NRF2 pathway activation. Although the kinase(s) responsible for phosphorylation of NRF2 in LSCC are unclear, CDK5 protein (Figure 3H) and PTM-SEA-derived activity (Figure 3I) were significantly upregulated (FDR<0.05) activity (Figure 3I) were significantly upregulated (FDR<0.05) in the Classical subtype. CDK5 phosphorylation of NFE2L2 S433 mediates its activation in astrocytes and can likely play a role in CDK5-mediated NRF2 activation in LSCC tumors. Intriguingly, in CDK5-mediated NRF2 activation in LSCC tumors. Intriguingly, unlike LUAD, KEAP1 mutations did not result in significantly reduced protein expression in LSCC, although missense mutations were mostly nonoverlapping, suggesting heterogeneity in NRF2 were mostly nonoverlapping, suggesting heterogeneity in NRF2 pathway dysregulation in NSCLC subtypes. Proteogenomic analysis of chromosome 3 prioritizes therapeutic targets in LSCC Chromosome 3q, which harbors key squamous differentiation markers SOX2 and TP63, showed the most dramatic arm-level amplification in this cohort (Figure S4A). To assess potential drivers in the 3q amplicon, we identified 3q genes for which CNA correlated 3q amplicon, we identified 3q genes for which CNA correlated with RNA and protein and expression differed between tumors and NATs. TP63 showed the highest elevation in tumors (Figure 4A). LSCC cell lines amplified for TP63 and SOX2 were highly dependent cell lines amplified for TP63 and SOX2 were highly dependent upon them, supporting oncogene addiction (Figure S4B). As previously reported, \\u0394Np63\\u03b1 was the dominant TP63 isoform in this dataset (Figure S4C) and was highly correlated with TP63 expression (Figure S4D). Its amplification was associated with improved survival in the TCGA dataset (Figure S4E). In contrast to survival in the TCGA dataset (Figure S4E). In contrast to the general elevation of TP63 copy number, transcript, and protein abundance in LSCC tumors compared to NATs, 10 \\u201cNp63-low\\u201d samples histologically confirmed to be LSCC showed no elevation at RNA or confirmed to be LSCC showed no elevation at RNA or protein levels (Figure 4B). The apoptosis inhibitor survivin (BIRC5) was among the most overexpressed proteins in this Np63-low group (Figure 4C). Notably, a substantial number of LSCC cell lines also had 4C). Notably, a substantial number of LSCC cell lines also had low Np63 expression (Figure S4F), and those TP63-low cell lines were significantly more vulnerable to the survivin inhibitor YM-155 (Figure S4G). Survivin promotes cell proliferation and G2M (Figure S4G). Survivin promotes cell proliferation and G2M checkpoints, evident in Np63-low samples (Figure S4H). Small molecule inhibitors and inhibitory peptides targeting survivin are being explored as part of ongoing clinical trials including in being explored as part of ongoing clinical trials including in NSCLC, but have shown modest or no improvement. Our data on the newly identified Np63-low LSCC tumors suggests that TP63 status may identify patients with potential response to survivin may identify patients with potential response to survivin inhibition. Overexpression of miR-205, a biomarker to distinguish LSCC from other NSCLC (Figure S4I), was associated with decreased promoter DNA methylation (Figure S4J) and elevation of \\u0394Np63\\u03b1, a transcriptional regulator of this miRNA. miR-205 showed high a transcriptional regulator of this miRNA. miR-205 showed high correlation with TP63 expression and its expression was markedly reduced in Np63-low samples (Figure 4D, Figure S4K). As expected, the abundance of most target gene products was negatively the abundance of most target gene products was negatively correlated with miR-205 expression, including ZEB1/2 and PTEN (Figure 4E). TP73 positively correlated with miR-205 at the transcript (Figure 4E) and still more strongly at the protein level (Figure 4E) and still more strongly at the protein level (Figure 4F) and with TP63 protein but not RNA (Figure S4L). TP73 overexpression mediates \\u0394Np63\\u03b1-dependent cell survival in squamous cell carcinoma, suggesting a mechanistic model in which functional and suggesting a mechanistic model in which functional and stoichiometric interaction between TP63 and TP73 leads to miR-205 overexpression in LSCC. Finally, consistent with the downregulation of key EMT-related proteins, the TP63 target miR-205 showed of key EMT-related proteins, the TP63 target miR-205 showed significant negative correlation with EMT activity but positive correlation with DNA replication (Figure 4G). The seemingly paradoxical role of TP63 in repressing EMT has been confirmed in role of TP63 in repressing EMT has been confirmed in previous studies. Of note, although \\u0394Np63\\u03b1 and miR-205 were consistently highly overexpressed in the B-I and Classical subtypes compared to NATs, their abundances were variable and not particularly low NATs, their abundances were variable and not particularly low in the EMT-E subtype. Thus, \\u0394Np63\\u03b1, either via miR-205 or indirectly, may induce a strong squamous epithelial cell phenotype in the B-I and Classical tumors, with additional mechanisms leading B-I and Classical tumors, with additional mechanisms leading to derepression and a metastatic phenotype, and other mechanisms beyond miR-205 may be involved in promoting the EMT phenotype in EMT-E samples. SOX2, a recurrently amplified squamous cell marker often co-amplified with TP63, was also overexpressed in tumors (Figure 4A). SOX2 is considered undruggable, intensifying therapeutic interest in upstream or downstream targets. An unbiased search (Figure in upstream or downstream targets. An unbiased search (Figure 4H) identified a significant correlation of LSD1 (KDM1A) with SOX2 protein. LSD1 regulates SOX2 expression and its inhibition in LSCC is currently being explored in a clinical setting in in LSCC is currently being explored in a clinical setting in conjunction with immunotherapy (NCT04350463). Other identified chromatin regulators included KDM3A, known to regulate SOX2 expression in ovarian cancer, and EZH2, whose inhibition in triple in ovarian cancer, and EZH2, whose inhibition in triple negative breast cancer leads to downregulation of SOX2 expression. EZH2 dependency was observed in SOX2 amplified LSCC cell lines (Figure S4M). The positive correlations of SOX2 with ALDH1A1, (Figure S4M). The positive correlations of SOX2 with ALDH1A1, ALDH3A1, and WNT5A (Figure 4H) suggest stemness-like features in these tumors. Interestingly, JAK1 was the protein most negatively correlated with SOX2 among other interferon-signaling proteins correlated with SOX2 among other interferon-signaling proteins such as IRF3 and IFNGR1 (Figure 4H), and negative correlation was seen between SOX2 and JAK-STAT signaling (Figure 4I). Crosstalk between lysine acetylation and ubiquitylation impacts cancer metabolism Consensus clustering of K-GG peptide abundances after correcting for cognate protein abundance revealed two stable ubiquitylproteome clusters (Figure 5A, Table S5) associated with country of origin and ethnicity (Fisher\\u2019s exact p<0.01). Proteins in of origin and ethnicity (Fisher\\u2019s exact p<0.01). Proteins in pathways such as glycolysis, JAK-STAT, MAPK, and immune signaling were differentially modified between these clusters (Figure 5A, Figure S5A\\u2013B, Table S5). To identify candidate enzymes driving Ub and UbL modifications in LSCC, we correlated E3 ubiquitin ligases or deubiquitylases (DUBs) to KGG-sites (Table S5). Eighteen DUBs with at least one negatively correlated K-GG site and 35 E3 ligases with at least negatively correlated K-GG site and 35 E3 ligases with at least one positively correlated K-GG site were identified, with top sites shown in Figure 5B and Figure S5C. HERC5 is the major E3 ligase involved in conjugation of the UbL modification ISG15 in ligase involved in conjugation of the UbL modification ISG15 in humans; hence the numerous positively correlated Ub/UbL sites (n=474; FDR<0.01) are likely ISG15 modifications. Consistent with this, we observed positive correlation between ISG15 and HERC5 this, we observed positive correlation between ISG15 and HERC5 abundance (Figure S5D). Positively-correlating Ub/UbL sites were enriched for glycolysis and TCA cycle annotations (Figure S5E). We observed several positively-correlating K-GG sites in the We observed several positively-correlating K-GG sites in the key glycolytic enzymes PKM, PGK1, and ENO1 (Figure 5C, Figure S5F). No significant negative correlations were observed between total protein abundance and their respective K-GG sites or HERC5, protein abundance and their respective K-GG sites or HERC5, suggesting a regulatory rather than degradative role, as expected for ISGylation (Figure S5G, Figure S5H). ISG15 modifications have been described in these glycolytic enzymes, and ISG15 knockdown been described in these glycolytic enzymes, and ISG15 knockdown in pancreatic cancer cells reduced oxidative phosphorylation and glycolysis. ISGylation can be induced by interferon in cancer; accordingly, HERC5 and ISG15 were positively associated with accordingly, HERC5 and ISG15 were positively associated with proteome-derived IFN-G signaling scores (Figure S5I, Figure S5J), and HERC5 was increased in tumors of the IFN-gamma dominant subtype (Figure 5D). Recently, a direct link between immune subtype (Figure 5D). Recently, a direct link between immune signaling and glucose metabolism through ISGylation was shown in the context of pathogenic invasion. Collectively, our data supports a potential role for HERC5 as a bridge between immune supports a potential role for HERC5 as a bridge between immune signaling and metabolic regulation in LSCC. This dataset allowed us to investigate crosstalk between lysine PTMs. We employed a modified version of CLUMPS to detect clustering of either acetylation or ubiquitylation sites within protein 3D structures. Significant ubiquitylation clusters were protein 3D structures. Significant ubiquitylation clusters were detected in 33 proteins and significant acetylation clusters in 17 proteins (p<0.05) (Table S5). Top hits revealed enrichment of glycolysis, fatty acid metabolism and OxPhos (Table S5), of glycolysis, fatty acid metabolism and OxPhos (Table S5), supporting modulation of these metabolic pathways by ubiquitylation and acetylation. In tumor-NAT comparison, major glycolytic enzymes PGK1 and PKM showed reduced acetylation at multiple known PGK1 and PKM showed reduced acetylation at multiple known inhibitory sites closely-positioned on the 3D structure, as well as on adjacent lysines surrounding the enzymatic pocket  (Figure 5E, Figure S5K). Tumors also had decreased ubiquitylation at 5E, Figure S5K). Tumors also had decreased ubiquitylation at closely-positioned K-GG-sites for TXN1, an important redox regulator (Figures 5F). PGK1 and PKM deacetylation suggests increased activity of these key glycolytic enzymes in tumors, a conclusion of these key glycolytic enzymes in tumors, a conclusion supported by increased phosphorylation on known activating sites of regulatory proteins (PFKFB3 pS461, PKM pS37). Upregulation of a glycolytic metabolic phenotype confers selective advantage to a glycolytic metabolic phenotype confers selective advantage to cancer cells by supporting uninterrupted growth, dynamically modulated by PTMs. While site-level roles of most Ub/UbL modifications and acetylsites are unclear, our data on acetylation, and acetylsites are unclear, our data on acetylation, phosphorylation, and HERC5-mediated ISGylation of glycolytic enzymes hints at the complexity of regulation of this cancer hallmark. Rapid growth of cells often leads to oxidative stress within tumors, which adapt by upregulating redox systems. Although thioredoxin (TXN) did not show overall differential protein expression between tumors and NATs, we observed increased TXN1 protein between tumors and NATs, we observed increased TXN1 protein combined with decreased Ub sites, particularly in the NMF Classical subtype (Kruskal Wallis FDR<0.01). The TXN1 activator TXNRD1 was also increased in the Classical subtype, while TXNIP, which was also increased in the Classical subtype, while TXNIP, which modulates cellular redox by binding and inhibiting TXN1, was decreased (Figure 5F). TXN and TXNRD1 were highly correlated with NRF2 score (TXN: R=0.85; TXNRD1: R=0.8, both p<2.2\\u00d710\\u221216), with NRF2 score (TXN: R=0.85; TXNRD1: R=0.8, both p<2.2\\u00d710\\u221216), consistent with their role in response to oxidative stress. Redox modulation through this pathway is associated with tumor proliferation, differentiation and prognosis in multiple epithelial differentiation and prognosis in multiple epithelial cancers, including NSCLC. The small proportion of glucose directed to the tumor mitochondria and subsequently to the TCA and OXPHOS cycles leads to increased acetyl CoA within the mitochondria and leads to increased acetyl CoA within the mitochondria and thence to non-enzymatic acetylation and inhibition of key enzymes in the fatty acid oxidation pathway such as ECH1 and ACADVL (also identified in the CLUMPS analysis). We recapitulate the finding identified in the CLUMPS analysis). We recapitulate the finding of increased ECH1 and ACADVL acetylation, including specific increase at the binding site of ACADVL in tumors (Figure 5E, Figure S5K). PTM-based modulation of these key metabolic and ROS S5K). PTM-based modulation of these key metabolic and ROS pathways is summarized in Figure 5G. As a complementary assessment of the crosstalk between lysine ubiquitylation and acetylation, we identified K-Ac or K-GG sites showing significant differential expression in tumors relative to NATs. Figure 5H shows 8 lysine sites where K-GG was relative to NATs. Figure 5H shows 8 lysine sites where K-GG was upregulated and K-Ac downregulated in tumors (upper panel) and 36 sites demonstrating the opposite trend (lower panel). Among the latter was the GTP-binding nuclear protein RAN, involved in the latter was the GTP-binding nuclear protein RAN, involved in nucleocytoplasmic transport, mitotic progression and spindle assembly. Acetylation on multiple lysine residues can affect RAN localization, GTP/GDP cycle and import / export complex RAN localization, GTP/GDP cycle and import / export complex formation. Consistent with the role of RAN in mitosis, RAN K127-acK showed dramatic upregulation in the P-P subtype (Figure 5I), named in part for its generally high proliferation (Figure 2B). named in part for its generally high proliferation (Figure 2B). The specific functional relevance of K127-ac has not been previously demonstrated, but its selective upregulation suggests a functional role. While of great interest to the PTM community and role. While of great interest to the PTM community and potentially very powerful, such crosstalk analyses are nascent, and interpretations remain suitably cautious. Immune landscape and regulation in LSCC Consensus clustering based on the xCell signatures (Table S6) identified three sets of tumors representing immunologically Hot, Warm, and Cold clusters (Figure 6A\\u2013C), as well as an NAT-enriched cluster. The NMF I-S subtype was significantly associated cluster. The NMF I-S subtype was significantly associated with the immune Hot cluster (Fisher\\u2019s exact p=6.371\\u00d710\\u221209) (Figure 6A). The Hot cluster was characterized by increased macrophages, CD4+, CD8+ and regulatory T cells, and dendritic cells in both CD4+, CD8+ and regulatory T cells, and dendritic cells in both deconvolution and immunohistochemistry (IHC) analyses (Figures 6A,C, S6C,D), and showed upregulation of immune-related pathways (Figure 6A\\u2013B, Figure S6A, Table S6). Immunosuppressive mediators 6A\\u2013B, Figure S6A, Table S6). Immunosuppressive mediators CTLA4, PD-1 (PDCD1), PD-L1 (CD274) and IDO1 and the key Treg transcriptional regulator FOXP3 were upregulated in Hot compared to Cold tumors, suggesting immune checkpoint-related therapeutic options suggesting immune checkpoint-related therapeutic options in these tumors. The Hot tumor acetylproteome was chiefly enriched for OXPHOS, Mitochondrial Complex and TCA Cycle pathways, consistent with increased metabolism or increased mitochondrial consistent with increased metabolism or increased mitochondrial acetyl-CoA leading to non-enzymatic acetylation of mitochondrial proteins (Figure 6D). Notably, NATs did not display consonant immune signaling (Figure S6B). The intermediate, Warm cluster immune signaling (Figure S6B). The intermediate, Warm cluster displayed upregulation of immune-related pathways such as PD-1 signaling, Interferon gamma (IFN-G) response, and Allograft rejection relative to Cold tumors, but did not show the downregulation relative to Cold tumors, but did not show the downregulation of cell cycle-related pathways observed in Hot tumors (Figure 6A\\u2013B, Figure S6A). In addition, canonical Wnt Signaling, active in tumor proliferation and immune evasion, was comparably in tumor proliferation and immune evasion, was comparably upregulated in the Warm and Cold tumor proteomes (Figure 6B). Fifteen pQTLs were also identified in proteins differentially abundant between Hot and Cold tumors (Table S6, Figure S6E) Rho GTPase signaling was upregulated in Hot tumors (Figure 6A, Table S6). ARHGDIB (RhoGDIb), the central regulatory molecule in GTPase activation, was the most significantly acetylated protein in the Hot cluster, followed by metabolic mitochondrial protein in the Hot cluster, followed by metabolic mitochondrial enzymes (Figure 6D). Acetylation of its homolog RhoGDIa at the conserved K135 site regulates F-actin assembly by activating RhoA. In contrast to RhoGDIa, the affinity of RhoGDIb for RhoA is In contrast to RhoGDIa, the affinity of RhoGDIb for RhoA is low, and the K135 acetylation of RhoGDIb may act as a positive regulator of Rac signaling via its interaction with Rac2 instead. In support of this hypothesis, we found significant upregulation support of this hypothesis, we found significant upregulation in the Hot cluster of Rho GTPase Rac2 and numerous Rac-specific regulators (Figures 6F, S6F, Table S6). Notably, these proteins are primarily immune-specific, indicating a possible role for are primarily immune-specific, indicating a possible role for lysine acetylation of RhoGDIb in promoting immune cell functions in the Hot cluster. Immunohistochemical staining for RhoGDIb (ARHGDIB) in the subset of immune Hot tumors confirmed strong (ARHGDIB) in the subset of immune Hot tumors confirmed strong signals in infiltrating immune cells and mesenchymal cells (Figure S6G), similar to dispersed RhoGDIb-expressing cells identified as lymphocytes seen in renal cell carcinomas. Thus, we propose as lymphocytes seen in renal cell carcinomas. Thus, we propose that K135 acetylation of RhoGDIb is a potential regulatory mechanism of immune cell functions that is in line with the inflammasome signature and immune-enriched networks. We explored the immunological associations of the RTK CBPE scores described above (Figure 2I,J) in LSCC tumors (Figure 6G). CSF1R, predominantly expressed in macrophages, showed the highest CBPE scores, and was positively associated with the Hot cluster CBPE scores, and was positively associated with the Hot cluster (Figure 6H) and consequently with immune-related pathways and cell types (Figure 6G). CSF1R correlated with immunosuppressive chemokine CCL5, phosphorylation of PIK3R1 Y580, downstream chemokine CCL5, phosphorylation of PIK3R1 Y580, downstream regulators of the actin cytoskeleton including WAS and WASF2 S103, and upregulation of AKAP13, among other immune-related proteins (Figure 6I). Tumor-associated macrophages (, CSF1R signaling, and 6I). Tumor-associated macrophages (, CSF1R signaling, and AKAP13 have each been proposed as anti-cancer targets. Proteomic Biomarker Candidates for Prognosis, Diagnosis, and Treatment Tumors and paired NATs revealed remarkable differences in protein expression and pathway enrichment (Figure 7A, Table S7). The top 50 of 502 differential proteins are shown in Figure S7A. In addition to 206 upregulated oncogenic proteins (OncoKB, Table addition to 206 upregulated oncogenic proteins (OncoKB, Table S7), several CT antigens, attractive targets of CT-antigen therapies, were overexpressed in tumors (Figure S7B), with the I-S samples showing the fewest CT antigens. As complete long-term samples showing the fewest CT antigens. As complete long-term follow-up was not yet available for this cohort, we leveraged the TCGA LSCC dataset to examine the association of these 502 candidate tumor biomarkers with overall survival (OS) or disease-free tumor biomarkers with overall survival (OS) or disease-free survival (DFS). Expression of four of the most highly differential genes showed significant association with poor OS and another 15 genes with poor DFS (Figure 7B, Figure S7C). Furthermore, 15 genes with poor DFS (Figure 7B, Figure S7C). Furthermore, knockdown of these tumor biomarker proteins reduced fitness across 16 LSCC cell lines (https://depmap.org/), suggesting critical roles in key cellular transformation and proliferation processes in key cellular transformation and proliferation processes (Figure 7C, Table S7). There is increasing interest in molecular taxonomies that help position cancer types in the context of related cancers. CNAs were more frequent in LSCC than in LUAD or head and neck squamous cell carcinoma (HNSCC), which respectively share tissue and cell carcinoma (HNSCC), which respectively share tissue and cell type of origin (Figure S7D). Widespread deletions of immune-related genes in LSCC correlated with ESTIMATE immune scores (Figure 7D). Immune-related deletions were less prominent in HNSCC and 7D). Immune-related deletions were less prominent in HNSCC and especially LUAD (Figure S7E), except for Chromosome 3p deletion, encompassing five chemokine receptors. Squamous cancers shared 3q amplification, JAK2 deletion and skin-development proteins 3q amplification, JAK2 deletion and skin-development proteins from across the genome. Finally, deletions of 4p14 and 5q, including multiple Toll-like receptor and interleukin signaling genes, were unique to LSCC. Among key tumor-specific phosphorylation were unique to LSCC. Among key tumor-specific phosphorylation events in LSCC, we identified upregulation of activating sites on 27 kinases, including MAPK14 and three others (DCK, EGFR, SRC) targetable by FDA-approved drugs (Figure S7F), with several targetable by FDA-approved drugs (Figure S7F), with several being shared by LUAD and HNSCC tumors. Notably, EGFR protein was significantly upregulated in the squamous cancers but not LUAD (Figure S7G), although LUAD had many activating EGFR mutations, and S7G), although LUAD had many activating EGFR mutations, and activating phosphorylation of EGFR was increased in both LUAD and LSCC. High EGFR amplification in the LSCC cohort did not correlate with EGFR pathway activity as assessed by mRNA-based PROGENy with EGFR pathway activity as assessed by mRNA-based PROGENy scoring (Figure 7E). Instead, this activity measurement was highly correlated to mRNA abundance of the five EGFR ligands that are prominent in tumors (Figure 7F). Furthermore, EGFR amplified prominent in tumors (Figure 7F). Furthermore, EGFR amplified samples displayed increased phosphorylation on proteins related to actin filament and cell junction organization as well as cell-substrate adhesion (Figure 7G). These findings and similar cell-substrate adhesion (Figure 7G). These findings and similar results recently reported in HNSCC support a squamous cell cancer feature in which EGFR ligand abundance drives canonical EGFR pathway activity and tumors with very high levels of EGFR may no activity and tumors with very high levels of EGFR may no longer require the ligand to drive activity. EGFR inhibition guided by EGFR abundance has been unsuccessful in squamous tumors. Since EGFR inhibitors such as cetuximab affect ligand-induced EGFR EGFR inhibitors such as cetuximab affect ligand-induced EGFR activity, ligand abundance, rather than EGFR amplification, might better predict EGFR inhibitor response in this population. Figure 7H graphically represents these and other key findings Figure 7H graphically represents these and other key findings presented in this manuscript, emphasizing translatable implications. Discussion In this study, we provide detailed proteogenomic profiles of treatment-na\\u00efve, primary LSCC tumors and paired NATs, with unprecedented coverage of post-translational modifications including phosphorylation, acetylation and for the first time, including phosphorylation, acetylation and for the first time, ubiquitylation. Unsupervised NMF-based multi-omic clustering suggested a refinement to prior RNA-based clustering, dividing basal tumors into Basal-inclusive and EMT-enriched subtypes with tumors into Basal-inclusive and EMT-enriched subtypes with biological differences of potential therapeutic significance. Molecular events downstream of copy number alterations showed NSD3 rather than FGFR1 protein overexpression in tumors with 8p11.23 rather than FGFR1 protein overexpression in tumors with 8p11.23 amplification, providing both a potential explanation for the limited effectiveness of FGFR1-targeted treatment in this population and a potential alternative therapy. Parallel downstream and a potential alternative therapy. Parallel downstream analysis of DNA promoter methylation events provided insight into regulation of many proteins differential between cancer and NATs. Comprehensive proteogenomic data provided a deeper exposition of established LSCC biology, often with potential therapeutic implications. We identified a subset of low-p63 tumors that were characterized by high levels of the known therapeutic target characterized by high levels of the known therapeutic target survivin. SOX2 is considered undruggable, but our analyses provide rationale for exploring LSD1 or other chromatin modifiers such as EZH2 to target SOX2 amplified / overexpressing tumors. The as EZH2 to target SOX2 amplified / overexpressing tumors. The importance of glycolysis and oxidative stress in LSCC are well-appreciated, but our data support complex regulation of these metabolic pathways by crosstalk between ubiquitylation (or metabolic pathways by crosstalk between ubiquitylation (or ubiquitin-like modifications), phosphorylation and acetylation. Immunotherapy represents the greatest advance in LSCC therapy in decades, but outcomes lag behind those seen in patients with LUAD but outcomes lag behind those seen in patients with LUAD and still only a minority of patients exhibit long-term responses; numerous immune-related proteogenomic observations suggest directions for further investigation. Proteogenomic dissection of the for further investigation. Proteogenomic dissection of the downstream effects of CDKN2A mutations had clinical implications related both to the interpretation of trials utilizing CDK4/6 inhibitors in LSCC patients and to biomarker selection for future in LSCC patients and to biomarker selection for future studies: though CDK4/6 inhibition has shown limited efficacy in LSCC trials to date, our analysis suggested that a more nuanced assessment of RB1 protein expression and phosphorylation is required of RB1 protein expression and phosphorylation is required before declaring this approach unsuccessful. Finally, triangulation between LSCC, LUAD and HNSCC demonstrated the influence of both tissue and cell type of origin on cancer biology, with of both tissue and cell type of origin on cancer biology, with therapeutic vulnerabilities both unique and common across these cancer types. Limitations of the study Many important clinical advances in NSCLC and other cancers have been driven by genomic profiling of bulk tumor material, and we anticipate that the same will prove true of bulk proteogenomic characterization, as performed here. Nevertheless, tumor characterization, as performed here. Nevertheless, tumor heterogeneity is an important complication, and emerging methods for proteomics approaches to heterogeneity, such as pairing microdissection with mass spectrometry or using mass cytometry will be with mass spectrometry or using mass cytometry will be useful adjuncts. It is typically metastatic disease that proves lethal, and in-depth proteogenomic analysis of primary tumors paired with metastatic lesions will be needed to provide critical insights metastatic lesions will be needed to provide critical insights into metastatic biology. Importantly, the scope of a landscape study such as this necessitates that it be understood as hypothesis-generating, and a wider community effort will be required to and a wider community effort will be required to validate biological observations and suggested therapeutic alternatives. Nevertheless, we hope this study will prove a valuable resource to the research and clinical communities, and advance the to the research and clinical communities, and advance the understanding and treatment of LSCC. STAR\\u2605METHODS\\n\\nRESOURCE AVAILABILITY\\n\\nLead contact\\n\\nFurther information and requests should be directed to and will be fulfilled by the lead author Michael A. Gillette (gillette@broadinstitute.org)\\n\\nMaterial availability Material availability\\n\\nThis study did not generate new unique reagents.\\n\\nData and Code Availability\\n\\nCPTAC LSCC proteomics data:\\n\\nProteomic Data Commons (PDC); https://pdc.cancer.gov/pdc/ with identifiers PDC000232, PDC000233, PDC000234 and PDC000237 CPTAC data portal LSCC: https://cptac-data-portal.georgetown.edu/study-summary/S063\\n\\nCPTAC HNSCC proteomics data:\\n\\nPDC: https://pdc.cancer.gov/pdc/ with identifiers PDC000221 and PDC000222\\n\\n https://cptac-data-portal.georgetown.edu/study-summary/S054 https://cptac-data-portal.georgetown.edu/study-summary/S054 \\n\\nCPATC LUAD proteomics data:\\n\\nPDC: https://pdc.cancer.gov/pdc/ with identifiers PDC000153, PDC000149, PDC000224)\\n\\n https://cptac-data-portal.georgetown.edu/study-summary/S056 https://cptac-data-portal.georgetown.edu/study-summary/S056 \\n\\nGenomic and transcriptomic data files can be accessed at the Genomic Data Commons (GDC); https://portal.gdc.cancer.gov/, via dbGaP Study Accession: phs001287.v10.p5 All histologic https://www.cancerimagingarchive.net/datascope/cptac/home/ and radiologic details can be accessed from the The Cancer Imaging Archive (TCIA) Public Access https://wiki.cancerimagingarchive.net/display/Public/CPTAC-LSCC. Sample annotation, processed and normalized data files are provided as Tables S1\\u2013S3.\\n\\nSoftware and code used in this study are referenced in their corresponding STAR Method sections and also the Key Resource Table.\\n\\nEXPERIMENTAL MODEL AND SUBJECT DETAILS EXPERIMENTAL MODEL AND SUBJECT DETAILS\\n\\nHuman Subjects A total of 113 participants (90 males, 23 females, 40\\u201388 years old) were collected for this study by 13 different tissue source sites from seven different countries between May 2016 and August 2018, with a median follow-up time of 2.5 years. Five (one 2018, with a median follow-up time of 2.5 years. Five (one female and four male) samples were excluded based on further pathological assessment and 108 tumors and 99 paired NATs were used in this study. Only histopathologically-defined adult lung squamous study. Only histopathologically-defined adult lung squamous tumors were considered for analysis, with an age range of 40\\u201388. Institutional review boards at tissue source sites reviewed protocols and consent documentation adhering to the Clinical Proteomic and consent documentation adhering to the Clinical Proteomic Tumor Analysis Consortium (CPTAC) guidelines. Clinical data were obtained from tissue source sites and aggregated by an internal database called the CDR (Comprehensive Data Resource) that database called the CDR (Comprehensive Data Resource) that synchronizes with the CPTAC DCC (https://cptac-data-portal.georgetown.edu/). Clinical data can be accessed and downloaded from the DCC (Data Coordinating Center). Demographics, histopathologic DCC (Data Coordinating Center). Demographics, histopathologic information, and treatment details were collected. LSCC histopathology was confirmed for all cases by at least 2 expert pathologists based on high resolution images of H&E sections. For samples based on high resolution images of H&E sections. For samples with low TP63 status, additional pathologic confirmation was performed. The histologic, genotypic, clinical, geographical and other associated metadata is summarized in Table S1. METHOD DETAILS\\n\\nSpecimen Acquisition The tumor, normal adjacent tissue (NAT), and whole blood samples used in this manuscript were prospectively collected for the CPTAC project. Biospecimens were collected from newly diagnosed patients with LSCC who underwent surgical resection and had patients with LSCC who underwent surgical resection and had received no prior treatment for their disease, including chemotherapy or radiotherapy. All cases had to be of acceptable LSCC histology but were collected regardless of surgical stage or histology but were collected regardless of surgical stage or histologic grade. Cases were staged using the AJCC cancer staging system 7th edition and the 8th edition. The tumor specimen weights ranged from 125 to 1560 milligrams. The average tissue mass ranged from 125 to 1560 milligrams. The average tissue mass was 220 mg. For most cases, three to four tumor specimens were collected. Paired histologically-normal adjacent lung tissues (NATs) were collected from the same patient at tumor resection. Each were collected from the same patient at tumor resection. Each tissue specimen endured cold ischemia for less than 40 minutes prior to freezing in liquid nitrogen; the average ischemic time was 16 minutes from resection/collection to freezing. Specimens was 16 minutes from resection/collection to freezing. Specimens were flash frozen in liquid nitrogen. Histologic sections obtained from top and bottom portions from each case were reviewed by a board-certified pathologist to confirm the assigned by a board-certified pathologist to confirm the assigned pathology. For samples to be deemed acceptable, the top and bottom sections had to contain an average of 50% tumor cell nuclei with less than 20% necrosis. Specimens were shipped overnight from the than 20% necrosis. Specimens were shipped overnight from the tissue source sites to the biospecimen core resource (BCR) located at Van Andel Research Institute, Grand Rapids, MI using a cryoport that maintained an average temperature of less than \\u2212140\\u00b0C. that maintained an average temperature of less than \\u2212140\\u00b0C. At the biospecimen core resource, specimens were confirmed for pathology qualification and prepared for genomic, transcriptomic, and proteomic analyses. Selected specimens were cryopulverized and proteomic analyses. Selected specimens were cryopulverized using a Covaris CryoPREP instrument and material aliquoted for subsequent molecular characterization. Genomic DNA and total RNA were extracted and sent to the genome sequencing centers. The were extracted and sent to the genome sequencing centers. The whole exome and whole genome DNA sequencing and methylation EPIC array analyses were performed at the Broad Institute, Cambridge, MA. TotalRNA and miRNA sequencing were performed at the MA. TotalRNA and miRNA sequencing were performed at the University of North Carolina, Chapel Hill, NC. Material for proteomic analyses were sent to the Proteomic Characterization Center (PCC) at the Broad Institute, Cambridge, MA. Sequencing sample preparation Our study sampled a single site of the primary tumor from surgical resections, with an internal requirement to process a minimum of 125mg of tumor issue and 50mg of NAT. DNA and RNA were extracted from tumor and NAT specimens in a co-isolation protocol from tumor and NAT specimens in a co-isolation protocol using Qiagen\\u2019s QIAsymphony DNA Mini Kit and QIAsymphony RNA Kit. Genomic DNA was also isolated from peripheral blood (3\\u20135mL) to serve as matched normal reference material. The Qubit\\u2122 dsDNA BR Assay as matched normal reference material. The Qubit\\u2122 dsDNA BR Assay Kit was used with the Qubit\\u00ae 2.0 Fluorimeter to determine the concentration of dsDNA in an aqueous solution. Any sample that passed quality control and produced enough DNA yield to go through quality control and produced enough DNA yield to go through the multiple planned genomic assays was sent for genomic characterization. RNA quality was quantified using the NanoDrop 8000 and quality assessed using an Agilent Bioanalyzer. A sample of and quality assessed using an Agilent Bioanalyzer. A sample of sufficient quantity that passed RNA quality control and had a minimum RIN (RNA integrity number) score of 7 was subjected to RNA sequencing. Identity matches for germline, normal adjacent RNA sequencing. Identity matches for germline, normal adjacent tissue, and tumor tissue were confirmed at the BCR using the Illumina Infinium QC array. This beadchip contains 15,949 markers designed to prioritize sample tracking, quality control, and designed to prioritize sample tracking, quality control, and stratification. Whole exome Sequencing (WES)\\n\\nLibrary construction and Hybrid Selection Library construction was performed as described in, with the following modifications: initial genomic DNA input into shearing was reduced from 3\\u03bcg to 20\\u2013250ng in 50\\u03bcL of solution. For adapter ligation, Illumina paired-end adapters were replaced with ligation, Illumina paired-end adapters were replaced with palindromic forked adapters, purchased from Integrated DNA Technologies (IDT), with unique dual-indexed molecular barcode sequences to facilitate downstream pooling. Kapa HyperPrep reagents in to facilitate downstream pooling. Kapa HyperPrep reagents in 96-reaction kit format were used for end repair/A-tailing, adapter ligation, and library enrichment PCR. In addition, during the post-enrichment SPRI cleanup, elution volume was reduced to 30\\u03bcL SPRI cleanup, elution volume was reduced to 30\\u03bcL to maximize library concentration, and a vortexing step was added to maximize the amount of template eluted. After library construction, libraries were pooled into groups of up to 96 samples. Hybridization were pooled into groups of up to 96 samples. Hybridization and capture were performed using the relevant components of Illumina\\u2019s Nextera Exome Kit and following the manufacturer\\u2019s suggested protocol, with the following exceptions: First, all libraries protocol, with the following exceptions: First, all libraries within a library construction plate were pooled prior to hybridization. Second, the Midi plate from Illumina\\u2019s Nextera Exome Kit was replaced with a skirted PCR plate to facilitate automation. was replaced with a skirted PCR plate to facilitate automation. All hybridization and capture steps were automated on the Agilent Bravo liquid handling system. Cluster Amplification and Sequencing After post-capture enrichment, library pools were quantified using qPCR (KAPA Biosystems) using an automated assay on the Agilent Bravo with probes specific to the ends of the adapters. Based on qPCR quantification, libraries were normalized to 2nM. Based on qPCR quantification, libraries were normalized to 2nM. Cluster amplification of DNA libraries was performed following manufacturer\\u2019s protocol (Illumina) using exclusion amplification chemistry and flowcells. Flow cells were sequenced utilizing chemistry and flowcells. Flow cells were sequenced utilizing sequencing-by-synthesis chemistry. The flow cells were then analyzed using RTA v.2.7.3 or later. Each pool of whole exome libraries was sequenced on paired 76-cycle runs with two 8-cycle index was sequenced on paired 76-cycle runs with two 8-cycle index reads across the number of lanes needed to meet coverage for all libraries in the pool. Pooled libraries were run on HiSeq4000 paired-end runs to achieve a minimum of 150x on-target coverage per runs to achieve a minimum of 150x on-target coverage per library. The raw Illumina sequence data were demultiplexed and converted to FASTQ files; adapter and low-quality sequences were trimmed. The raw reads were mapped to the GRCh38/hg38 human reference The raw reads were mapped to the GRCh38/hg38 human reference genome and the validated BAMs were used for downstream analysis and variant calling. Whole genome sequencing (WGS)\\n\\nCluster Amplification and Sequencing An aliquot of genomic DNA (350ng in 50\\u03bcL) was used as the input into DNA fragmentation (aka shearing). Shearing was performed acoustically using a Covaris focused-ultrasonicator, targeting 385bp fragments. Following fragmentation, additional size 385bp fragments. Following fragmentation, additional size selection was performed using SPRI cleanup. Library preparation was performed using a commercially available KAPA Hyper Prep without amplification module kit (KAPA Biosystems) and with palindromic amplification module kit (KAPA Biosystems) and with palindromic forked adapters with unique 8-base index sequences embedded within the adapter (IDT). Following sample preparation, libraries were quantified using quantitative PCR (KAPA Biosystems), with were quantified using quantitative PCR (KAPA Biosystems), with probes specific to the ends of the adapters using the automated Agilent\\u2019s Bravo liquid handling platform. Based on qPCR quantification, libraries were normalized to 1.7nM and pooled into libraries were normalized to 1.7nM and pooled into 24-plexes. Sample pools were combined with HiSeqX Cluster Amp Reagents EPX1, EPX2, and EPX3 into single wells on a strip tube using the Hamilton Starlet Liquid Handling system. Cluster amplification of the templates was performed according to the manufacturer\\u2019s of the templates was performed according to the manufacturer\\u2019s protocol (Illumina) with the Illumina cBot. Flow cells were sequenced to a minimum of 15x on HiSeqX utilizing sequencing-by-synthesis kits to produce 151bp paired-end reads. Output from kits to produce 151bp paired-end reads. Output from Illumina software was processed by the Picard data processing pipeline to yield BAM files containing demultiplexed, aggregated, aligned reads. All sample information tracking was performed by automated All sample information tracking was performed by automated LIMS messaging. Array Based Methylation Analysis The Methylation EPIC array uses an 8-sample version of the Illumina Beadchip capturing >850,000 methylation sites per sample. Two hundred and fifty nanograms of DNA was used for the bisulfite conversion using Infinium MethylationEPIC BeadChip Kit conversion using Infinium MethylationEPIC BeadChip Kit (Illumina). The EPIC array includes sample plating, bisulfite conversion, and methylation array processing. After scanning, the data was processed through an automated genotype-calling pipeline. Data processed through an automated genotype-calling pipeline. Data output consisted of raw idats and a sample sheet. RNA and miRNA sequencing\\n\\nQuality Assurance and Control of RNA Analytes All RNA analytes were assayed for RNA integrity, concentration, and fragment size. Samples for total RNA-seq were quantified on a TapeStation system (Agilent, Inc. Santa Clara, CA). Samples with RINs >7.0 were considered high quality and were considered with RINs >7.0 were considered high quality and were considered for sequencing. Total RNA-seq libraries were generated using 300 nanograms of total RNA using the TruSeq Stranded Total RNA Library Prep Kit with Ribo-Zero Gold and bar-coded with individual Prep Kit with Ribo-Zero Gold and bar-coded with individual tags following the manufacturer\\u2019s instructions (Illumina). Total RNA Libraries were prepared on an Agilent Bravo automated liquid handling system. Quality control was performed at every step, and system. Quality control was performed at every step, and the libraries were quantified using a TapeStation system. Total RNA Sequencing Indexed libraries were prepared and run on HiSeq4000 paired-end 75 base pairs to generate a minimum of 120 million reads per sample library with a target of greater than 90% mapped reads. The raw Illumina sequence data were demultiplexed and converted to raw Illumina sequence data were demultiplexed and converted to FASTQ files, and adapter and low-quality sequences were trimmed. Samples were then assessed for quality by mapping reads to GRCh38/hg38, estimating the total number of mapped reads, amount of estimating the total number of mapped reads, amount of RNA mapping to coding regions, amount of rRNA in the sample, number of genes expressed, and relative expression of housekeeping genes. Samples passing this QA/QC were then clustered with other Samples passing this QA/QC were then clustered with other expression data from similar and distinct tumor types to confirm expected expression patterns. Atypical samples were then SNP typed from the RNA data to confirm source analyte. FASTQ files of all from the RNA data to confirm source analyte. FASTQ files of all reads were then uploaded to the GDC repository. miRNA-seq Library Construction miRNA-seq library construction was performed from the RNA samples using the NEXTflex Small RNA-Seq Kit (v3, PerkinElmer, Waltham, MA) and barcoded with individual tags following the manufacturer\\u2019s instructions. Libraries were prepared on a Sciclone Liquid instructions. Libraries were prepared on a Sciclone Liquid Handling Workstation. Quality control was performed at every step, and the libraries were quantified using a TapeStation system and an Agilent Bioanalyzer using the Small RNA analysis kit. Pooled an Agilent Bioanalyzer using the Small RNA analysis kit. Pooled libraries were then size selected according to NEXTflex kit specifications using a Pippin Prep system (Sage Science, Beverly, MA). miRNA Sequencing Indexed libraries were loaded on the HiSeq4000 to generate a minimum of 10 million reads per library with a minimum of 90% reads mapped. The raw Illumina sequence data were demultiplexed and converted to FASTQ files for downstream analysis. Resultant data to FASTQ files for downstream analysis. Resultant data were analyzed using a variant of the small RNA quantification pipeline developed for TCGA (Chu et al., 2016). Data from samples were assessed for the number of miRNAs called, species diversity, and for the number of miRNAs called, species diversity, and total abundance before uploading to the GDC repository. Mass Spectrometry methods The protocols below for protein extraction, tryptic digestion, TMT-11 labeling of peptides, peptide fractionation by basic reversed-phase liquid chromatography, phosphopeptide enrichment using immobilized metal affinity chromatography, and LC-MS/MS were immobilized metal affinity chromatography, and LC-MS/MS were performed as previously described in depth. Acetyl-enrichment was performed as described before with modifications as indicated below. Protein Extraction and Tryptic Digestion Approximately fifty milligrams (wet weight) of cryopulverized human LSCC and NAT samples were homogenized in the lysis buffer for 30 minutes at a ratio of about 200 uL lysis buffer for every 50 mg wet weight tissue. The lysis buffer consisted of 8 M urea, mg wet weight tissue. The lysis buffer consisted of 8 M urea, 75 mM NaCl, 1 mM EDTA, 50 mM Tris HCl (pH 8), 10 mM NaF, phosphatase inhibitor cocktail 2 (1:100; Sigma, P5726) and cocktail 3 (1:100; Sigma, P0044), 2 \\u03bcg/mL aprotinin (Sigma, A6103), 10 \\u03bcg/mL Sigma, P0044), 2 \\u03bcg/mL aprotinin (Sigma, A6103), 10 \\u03bcg/mL leupeptin (Roche, 11017101001), and 1 mM PMSF (Sigma, 78830). Lysates were centrifuged at 20,000 g for 10 minutes and protein concentrations of the clarified lysates were measured by BCA assay of the clarified lysates were measured by BCA assay (Pierce). Protein lysates were subsequently reduced with 5 mM dithiothreitol (Thermo Scientific, 20291) for an hour at 37C and alkylated with 10 mM iodoacetamide (Sigma, A3221) for 45 minutes in the dark 10 mM iodoacetamide (Sigma, A3221) for 45 minutes in the dark at room temperature. Prior to digestion, samples were diluted 4-fold to achieve 2 M urea with 50mM Tris HCl (pH 8). Digestion was performed with LysC (Wako, 129\\u201302541) for 2 hours and with was performed with LysC (Wako, 129\\u201302541) for 2 hours and with trypsin (Promega, V511X) overnight, both at a 1:50 enzyme-to-protein ratio and at room temperature. Digested samples were acidified with formic acid (FA; Sigma, F0507) to achieve a final with formic acid (FA; Sigma, F0507) to achieve a final volumetric concentration of 1% (final pH of ~3), and centrifuged at 1,500 g for 15 minutes to clear precipitated urea from peptide lysates. Samples were desalted on tC18 SepPak columns (Waters, 200mg, Samples were desalted on tC18 SepPak columns (Waters, 200mg, WAT054925) and dried down using a SpeedVac apparatus. Common Reference (CR) Pool and Plex layout\\n\\nProteome, Phosphoproteome, and Acetyprotome: The proteomic, phosphoproteomic and acetylproteome analyses of LSCC samples consisted of 22 TMT-11 plex experiments. To facilitate quantitative comparison between all samples across experiments, a common reference (CR) sample was included in each 11-plex. a common reference (CR) sample was included in each 11-plex. A common physical, rather than in silico reference was used for this purpose for optimal quantitative precision between TMT-11 experiments. Considerations prior to creating the reference sample Considerations prior to creating the reference sample were that this sample needed to be of adequate quantity to cover all planned experiments for both the discovery and future confirmatory sets (not part of this study) with overhead for additional sets (not part of this study) with overhead for additional possible experiments. The CR included all (212/212) of the samples analyzed in the discovery set TMT experiments, yielding a CR that was representative of all the samples in the study. Making the was representative of all the samples in the study. Making the CR as representative of the study as a whole was particularly important since by definition only analytes represented in the reference sample would be included in the final ratio-based data sample would be included in the final ratio-based data analyses. 110 unique tumor samples with 102 paired NAT samples are distributed among 22 11-plex experiments, with 10 individual samples occupying the first 10 channels of each experiment and the 11th channel being reserved for the reference sample. All the tumors channel being reserved for the reference sample. All the tumors were in the C channels and all the normal samples were in the N channels. The first 8 channels of each experiment contained 4 tumor/normal pairs, with each pair of patient samples adjacent to pairs, with each pair of patient samples adjacent to each other. Channels 9 and 10 (130C and 131N) either contained a 5th tumor/normal pair, or an unpaired tumor sample in channel 9 and an alternate common reference sample in channel 10. These included 3 common reference sample in channel 10. These included 3 global LUAD common references, 3 HNSCC common references from the team at JHU, and 2 Tumor only common references from this LSCC discovery set. To ensure capacity for additional samples or experiments given a target input of 300 \\u03bcg peptide per channel per experiment, 18 mg total was targeted for reference material. To meet these collective requirements, all the samples in the discovery set that requirements, all the samples in the discovery set that had enough material were included in the CR. After reserving a conservative 400 \\u03bcg peptide / sample for individual sample analysis, an additional amount of 120 \\u03bcg for each of the samples with an additional amount of 120 \\u03bcg for each of the samples with adequate quantities were pooled. All 212 samples were selected for the combined tumor/normal CR. To make the combined CR, tumor only and normal only CRs were first created separately, with 110 and normal only CRs were first created separately, with 110 tumor samples and 102 normal samples. After creating tumor and NAT only CRs at 150 ug and 150ug amounts, a pool of combined CR was made, comprised of 8.6 mg tumor only reference and 8.6 mg normal comprised of 8.6 mg tumor only reference and 8.6 mg normal only reference. The 17.2 mg pooled reference material was divided into 55 300 \\u03bcg aliquots, dried down, and frozen at \\u221280\\u00b0C until use. Ubiquitylproteome: 750 ug of unlabeled peptides was required from each sample to ensure depth of coverage in the ubiquitylproteome, which restricted the number of samples available for profiling. 147/212 samples from the proteome, phosphoproteome, acetylproteome set were from the proteome, phosphoproteome, acetylproteome set were included in the ubiquitylproteome analysis. A separate ubiquitylproteome CR was constructed prior to KGG enrichment, due to limitations in sample material (both patient material and remaining CR in sample material (both patient material and remaining CR aliquots to be used in future LSCC proteome, phosphoproteome, and acetylproteome processing.. In order to make TMT ratio-based quantitation comparable across omes, the ubiquitylproteome CR was comparable across omes, the ubiquitylproteome CR was constructed to be as similar as practically possible to the proteome, phosphoproteome, acetylproteome CR. The ubiquitylproteome CR contains all samples from the proteome, phosphoproteome, acetylproteome all samples from the proteome, phosphoproteome, acetylproteome set with material remaining to contribute, 209/212. The ubiquitylproteome CR contributions were of 3 types: 104/147 samples included in ubiquitylproteome study and made full CR contribution, >1000ug available.\\n\\n42/147 samples included in ubiquitylproteome study and made partial CR contribution, 750ug \\u22121000 ug available. 63/65 samples from the proteome, phosphoproteome, acetylproteome set not included in ubiquitylproteome study, but made full CR contribution, <750 ug available. CR tumor and normal peptides were combined at a ratio of 55:45 to reflect the higher proportion of tumor samples in the experimental design (as opposed to proteome, phosphoproteome, acetylproteome CR\\u2019s proportion of 50:50), and aliquoted into 750 ug CR\\u2019s proportion of 50:50), and aliquoted into 750 ug aliquots. In the ubiquitylproteome experimental design, 15 11-plex experiments contained 87 unique tumor samples in the C channels and 60 normal samples in the N channels, with the first 8 channels of every 11-plex consisting of patient-paired tumor/normal samples every 11-plex consisting of patient-paired tumor/normal samples adjacent to each other. The 11th channel was reserved for the ubiquitylproteome common reference sample. Channel 9 (130C) contained an unpaired tumor, while channel 10 (131N) contained either an unpaired tumor, while channel 10 (131N) contained either an unpaired tumor, or in 3 plexes, a LUAD ubiquitylproteome common reference (which was constructed similarly for that experiment, unpublished). TMT-11 Labeling of Peptides\\n\\nProteome, Phosphoproteome, Acetylproteome: 300 \\u03bcg of desalted peptides per sample (based on peptide-level BCA after digestion) were labeled with 11-plex TMT reagents and combined for multiplexed analysis using a \\u201creduced labeling\\u201d approach (Zecha et al., 2019). To conservatively yield extra approach (Zecha et al., 2019). To conservatively yield extra labeled peptide in case a given sample demonstrated unsuitably low TMT reporter ion intensity during channel mixing control LC-MS/MS runs, 400 ug peptides per sample were initially labeled, with runs, 400 ug peptides per sample were initially labeled, with only 300 ug combined per sample for the final multiplexed experiments upon successful quality control. For each 400 \\u03bcg peptide aliquot of an individual sample, peptides were reconstituted to 5 of an individual sample, peptides were reconstituted to 5 mg/ml with 50 mM HEPES (pH 8.5) solution and 400 ug labeling reagent dissolved in 20 uL anhydrous acetonitrile was added. After 1 h incubation with shaking, samples were diluted 1:1 with a solution with shaking, samples were diluted 1:1 with a solution of 80% 50 mM HEPES and 20% acetonitrile. After confirming good label incorporation, 5 \\u03bcL of 5% hydroxylamine was added to quench the unreacted TMT reagents. Good label incorporation was defined as unreacted TMT reagents. Good label incorporation was defined as having less than 3% isobaric under-labeled MS/MS spectra in each sample, as measured by LC-MS/MS after taking out a 2 \\u03bcg aliquot from each sample and analyzing 0.25 \\u03bcg. If a sample did not from each sample and analyzing 0.25 \\u03bcg. If a sample did not have sufficient label incorporation, additional TMT was added to the sample and another 1 h incubation was performed with shaking. At the time that the labeling efficiency quality control samples the time that the labeling efficiency quality control samples were taken out, an additional 4 \\u03bcg of material from each sample was taken out and combined as a mixing control. After analyzing the mixing control sample by LC-MS/MS, intensity values of the the mixing control sample by LC-MS/MS, intensity values of the individual TMT reporter ions were summed across all peptide spectrum matches and compared to ensure that the total reporter ion intensity of each sample met a threshold of +/\\u2212 15% of the ion intensity of each sample met a threshold of +/\\u2212 15% of the internal reference. If necessary, adjustments were made by either using additional labeled material or reducing an individual sample\\u2019s contribution to the mixture, and analyzing a subsequent contribution to the mixture, and analyzing a subsequent mixing control, until all samples met the threshold and were thus approximately 1:1:1.... Differentially labeled peptides were then mixed (11 \\u00d7 300 \\u03bcg), dried down via vacuum centrifuge, and the mixed (11 \\u00d7 300 \\u03bcg), dried down via vacuum centrifuge, and the quenched, combined sample was subsequently desalted on a 200 mg tC18 SepPak column. The same lot of TMT reagents was used for all samples (UA280170/TL272399). Peptide Fractionation\\n\\nFor Proteome, Phosphoproteome, Acetylproteome: To reduce sample complexity, peptide samples were separated by high pH reversed phase (RP) separation as described previously (Mertins et al., 2018). A desalted 3.3 mg, 11-plex TMT-labeled experiment (based on peptide-level BCA after digestion) was experiment (based on peptide-level BCA after digestion) was reconstituted in 935 \\u03bcL 4.5mM ammonium formate (pH 10) and 2% acetonitrile, centrifuged for 7 min at 20000 g to precipitate insoluble peptides, loaded on a 4.6 mm x 250 mm column RP Zorbax 300 A peptides, loaded on a 4.6 mm x 250 mm column RP Zorbax 300 A Extend-C18 column (Agilent, 3.5 \\u03bcm bead size), and separated on an Agilent 1260 Series HPLC instrument using basic reversed-phase chromatography. Solvent A (2% acetonitrile, 4.5 mM ammonium chromatography. Solvent A (2% acetonitrile, 4.5 mM ammonium formate, pH 10) and a nonlinear increasing concentration of solvent B (90% acetonitrile, 4.5 mM ammonium formate, pH 10) were used to separate peptides. The 4.5 mM ammonium formate solvents were to separate peptides. The 4.5 mM ammonium formate solvents were made by 40\\u2013fold dilution of a stock solution of 180 mM ammonium formate, pH 10. To make 1L of stock solution, add 25 mL of 28% (wt/vol) ammonium hydroxide (28%, density 0.9 g/ml, mL of 28% (wt/vol) ammonium hydroxide (28%, density 0.9 g/ml, Sigma-Aldrich) to ~850ml of HPLC grade water, then add ~35 mL of 10 % (vol/vol) formic acid (>95% Sigma-Aldrich) to titrate the pH to 10.0; bring the final volume to 1 liter with HPLC grade pH to 10.0; bring the final volume to 1 liter with HPLC grade water. The 96 minute separation LC gradient followed this profile: (min: %B) 0:0; 7:0; 13:16; 73:40; 77:44; 82:60; 96:60. The flow rate was 1 mL/min. Per 3.3 mg separation, 82 fractions were rate was 1 mL/min. Per 3.3 mg separation, 82 fractions were collected into a 96 deep-well x 2mL plate (Whatman, #7701\\u20135200), with fractions combined in a step-wise concatenation strategy and acidified to a final concentration of 0.1% FA as reported and acidified to a final concentration of 0.1% FA as reported previously. An additional 14 fractions were collected from the 96 deep-well plate for fraction A, which are the early eluting fractions that tend to contain multi-phosphorylated peptides. 5% of that tend to contain multi-phosphorylated peptides. 5% of the volume of each of the 24+A proteome fractions was allocated for proteome analysis, dried down, and re-suspended in 3% MeCN/0.1% FA (MeCN; acetonitrile) to a peptide concentration of 0.25 \\u03bcg/ \\u03bcL (MeCN; acetonitrile) to a peptide concentration of 0.25 \\u03bcg/ \\u03bcL for LC-MS/MS analysis. The remaining 95% of concatenated 24 fractions were further combined into 12 fractions, with fraction A as a separate fraction. These 13 fractions were then enriched for a separate fraction. These 13 fractions were then enriched for phosphopeptides as described below. Phosphopeptide Enrichment Ni-NTA agarose beads were used to prepare Fe3+-NTA agarose beads. In each phosphoproteome fraction, ~261.25 \\u03bcg peptides (based on peptide-level BCA after digestion with uniformly-distributed fractionation presumed) were reconstituted in 522.5 \\u03bcL 80% fractionation presumed) were reconstituted in 522.5 \\u03bcL 80% MeCN/0.1% TFA (trifluoroacetic acid) solvent and incubated with 10 \\u03bcL of the IMAC beads for 30 minutes with end-over-end rotation at RT. After incubation, samples were briefly spun down on a at RT. After incubation, samples were briefly spun down on a tabletop centrifuge; clarified peptide flow-throughs were separated from the beads; and the beads were reconstituted in 200 \\u03bcL IMAC binding/wash buffer (80 MeCN/0.1% TFA) and loaded onto \\u03bcL IMAC binding/wash buffer (80 MeCN/0.1% TFA) and loaded onto equilibrated Empore C18 silica-packed stage tips (3M, 2315). Samples were then washed twice with 50 \\u03bcL of IMAC binding/wash buffer and once with 50 uL 1% FA, and were eluted from the IMAC buffer and once with 50 uL 1% FA, and were eluted from the IMAC beads to the stage tips with 3 \\u00d7 70 uL washes of 500 mM dibasic sodium phosphate (pH 7.0, Sigma S9763). Stage tips were then washed once with 100 \\u03bcL 1% FA and phosphopeptides were eluted from once with 100 \\u03bcL 1% FA and phosphopeptides were eluted from the stage tips with 60 \\u03bcL 50% MeCN/0.1% FA. Phosphopeptides were dried down and re-suspended in 9 \\u03bcL 50% MeCN/0.1%FA for LC-MS/MS analysis, where 4 \\u03bcL was injected per run. Acetylpeptide Enrichment Acetylated lysine peptides were enriched using an antibody against the acetyl-lysine motif (CST PTM-SCAN Catalogue No. 13416). IMAC eluents were concatenated into 4 fractions (~783.75 \\u03bcg peptides per fraction) and dried down using a SpeedVac apparatus. per fraction) and dried down using a SpeedVac apparatus. Peptide fractions were reconstituted with 1.4ml of IAP buffer (5 mM MOPS pH 7.2, 1 mM Sodium Phosphate (dibasic), 5 mM NaCl) per fraction and incubated for 2 hours at 4oC with pre-washed (3 times and incubated for 2 hours at 4oC with pre-washed (3 times with IAP buffer) agarose beads bound to acetyl-lysine motif antibody. Peptide-bound beads were washed 4 times with ice-cold PBS followed by elution with 2 \\u00d7 100ul of 0.15% TFA. Eluents were followed by elution with 2 \\u00d7 100ul of 0.15% TFA. Eluents were desalted using C18 stage tips, eluted with 50% ACN and dried down. Acetylpeptides were suspended in 7ul of 0.1% FA and 3% ACN and 4ul were injected per run. Ubiquitylpeptide Enrichment with on-bead TMT labeling Ubiquitin enrichment was performed based on the recently published UbiFast protocol. Before enrichment, anti-K-\\u03b5-GG bead-bound antibodies from the PTM-Scan ubiquitin remnant motif kit (Cell Signaling Technologies, Kit #5562) were cross-linked. Beads were Technologies, Kit #5562) were cross-linked. Beads were briefly washed 3X with 100 mM sodium borate (pH 9.0) and incubated with 20 mM DMP for 30 min at RT. Beads were then washed 2X with 200 mM ethanolamine and incubated overnight at 4\\u00b0C in 200 mM 200 mM ethanolamine and incubated overnight at 4\\u00b0C in 200 mM ethanolamine with end-over-end rotation. Following incubation, beads were washed 3X with immunoprecipitation (IAP) buffer (50 mM MOPS, pH 7.2, 10 mM sodium phosphate, 50 mM NaCl) and stored at MOPS, pH 7.2, 10 mM sodium phosphate, 50 mM NaCl) and stored at 4\\u00b0C at a concentration of 0.5 ug/uL. For each 11-plex experiment, 31.25 ug of cross-linked anti-K-GG bead-bound antibody at 0.5 ug/uL in IAP per channel was aliquoted into 1.5 mL Eppendorf tubes on ice. 750 ug peptide samples were reconstituted to 0.5 mg/mL concentration in IAP buffer and were reconstituted to 0.5 mg/mL concentration in IAP buffer and vortexed for 10 min. Peptides were then centrifuged for 5 min at 5,000g. Each peptide solution was added to a tube of antibody and gently rotated end-over-end at 4oC for 1 hour. Following and gently rotated end-over-end at 4oC for 1 hour. Following enrichment, samples were spun at 2000 rcf for ~1 min to settle and the supernatant was removed and stored as flow-through, and from this point beads were kept on ice unless being handled. from this point beads were kept on ice unless being handled. Antibody beads were washed with 1.5 mL ice cold IAP followed by 1.5 mL ice cold PBS. For all washes, after adding wash reagent each sample was inverted ~5 times, agitated by tapping pairs of each sample was inverted ~5 times, agitated by tapping pairs of tubes together, spun at 2000 rcf for about 30\\u201340 s, left to settle on ice for 10\\u201320 s, and supernatant was removed. Importantly, all washes were completed as quickly as possible. Importantly, all washes were completed as quickly as possible. Pre-aliquoted TMT @ 40 ug/uL in 100% MeCN was retrieved and briefly equilibrated to RT while beads were reconstituted in 200 uL room temp. 100 mM HEPES buffer. For each sample, 400 ug of TMT room temp. 100 mM HEPES buffer. For each sample, 400 ug of TMT labeling reagent in 10 uL acetonitrile was added. Samples were labeled on-antibody while shaking vigorously (1400 rpm) at 20\\u00b0C for 10 minutes, then quenched with 8 uL 5% hydroxylamine and for 10 minutes, then quenched with 8 uL 5% hydroxylamine and shaken vigorously for another 5 minutes. Labeled antibody-bound peptides were then put back on ice, washed once with 1.3 mL cold IAP, and again with 1.5 mL cold IAP. Each channel was resuspended and again with 1.5 mL cold IAP. Each channel was resuspended and transferred to a combination tube with 130 uL cold IAP. Following combination, each now-empty tube was serially washed with 1.5 mL cold IAP to remove remaining beads, and this 1.5 mL IAP was mL cold IAP to remove remaining beads, and this 1.5 mL IAP was added to the combination tube and used to wash the combined beads. Combined beads were washed one final time with 1.5 mL ice cold PBS. Once the channels were combined and washed, peptides were PBS. Once the channels were combined and washed, peptides were eluted twice from the beads by resuspending with 150 uL room temp. 0.15% TFA and incubating 5 min at RT. Each round of acid-eluted KGG-modified peptides was desalted on an equilibrated KGG-modified peptides was desalted on an equilibrated two-punch C18 stage tip. Both elutions of KGG peptides were loaded sequentially, washed 2X with 100 uL 0.1% FA, and eluted into an MS vial with 50 uL 50% ACN/0.1% FA. The eluted peptides were frozen, with 50 uL 50% ACN/0.1% FA. The eluted peptides were frozen, lyophilized, and reconstituted in 9 uL 3% ACN/0.1% FA, with 4 uL injected twice for two LC-MS/MS runs on a Thermo Lumos instrument. While the same lot of TMT reagents was used for all instrument. While the same lot of TMT reagents was used for all ubiquitylproteome samples (TE270748-TD264064), it is a different lot than that used for Proteome, Phosphoproteome, and acetylproteome samples. LC-MS/MS for Proteomics Analyses Online separation was done with a nanoflow Proxeon EASY-nLC 1200 UHPLC system (Thermo Fisher Scientific). In this set up, the LC system, column, and platinum wire used to deliver electrospray source voltage were connected via a stainless steel cross source voltage were connected via a stainless steel cross (360\\u03bcm, IDEX Health & Science, UH-906x). The column was heated to 50\\u00b0C using a column heater sleeve (Phoenix-ST) to prevent over-pressuring of columns during UHPLC separation. From each peptide of columns during UHPLC separation. From each peptide fraction, ~1ug (based on protein-level BCA prior to digestion with uniformly-distributed fractionation presumed), the equivalent of 12% of each global proteome sample in a 2 ul injection volume or 50% each global proteome sample in a 2 ul injection volume or 50% of each phosphoproteome, acetylproteome, or ubiquitylproteome sample in a 4 ul injection volume, was injected onto an in-house packed 22cm x 75um internal diameter C18 silica picofrit capillary 22cm x 75um internal diameter C18 silica picofrit capillary column (1.9 \\u03bcm ReproSil-Pur C18-AQ beads, Dr. Maisch GmbH, r119.aq; Picofrit 10um tip opening, New Objective, PF360\\u201375-10-N-5). Mobile phase flow rate was 200 nL/min, comprised of 3% Mobile phase flow rate was 200 nL/min, comprised of 3% acetonitrile/0.1% formic acid (Solvent A) and 90% acetonitrile /0.1% formic acid (Solvent B). The 110-minute LC-MS/MS method consisted of a 10-min column-equilibration procedure; a 20-min consisted of a 10-min column-equilibration procedure; a 20-min sample-loading procedure; and the following gradient profile: (min:%B) 0:2; 1:6; 85:30; 94:60; 95;90; 100:90; 101:50; 110:50 (the last two steps at 500 nL/min flow rate). For acetylproteome last two steps at 500 nL/min flow rate). For acetylproteome analysis, the same LC and column setup was used, but the gradient was extended to 260 minutes with the following gradient profile: (min:%B) 0:2; 1:6; 235:30; 244:60; 245;90; 250:90; 251:50; (min:%B) 0:2; 1:6; 235:30; 244:60; 245;90; 250:90; 251:50; 260:50 (the last two steps at 500 nL/min flow rate). For ubiquitylproteome analysis, the same LC and column setup was used, but the gradient was 154-minutes with the following gradient profile: gradient was 154-minutes with the following gradient profile: (min:%B) 0:2; 2:6; 122:35; 130:60; 133;90; 143:90; 144:50; 154:50 (the last two steps at 500 nL/min flow rate). For proteome and acetylproteome analysis, samples were analyzed with a benchtop Q Exactive HF-X mass spectrometer (Thermo Fisher Scientific) equipped with a nanoflow ionization source (James A. Hill Instrument Services, Arlington, MA). Data-dependent A. Hill Instrument Services, Arlington, MA). Data-dependent acquisition was performed using Q Exactive HF-X Orbitrap v 2.9 software in positive ion mode at a spray voltage of 1.5 kV. MS1 Spectra were measured with a resolution of 60,000, an AGC target of were measured with a resolution of 60,000, an AGC target of 3e6 and a mass range from 350 to 1800 m/z. The data-dependent mode cycle was set to trigger MS/MS on up to the top 20 most abundant precursors per cycle at an MS2 resolution of 45,000, an AGC precursors per cycle at an MS2 resolution of 45,000, an AGC target of 5e4, an isolation window of 0.7 m/z, a maximum injection time of 105 msec, and an HCD collision energy of 31%. Peptides that triggered MS/MS scans were dynamically excluded from further triggered MS/MS scans were dynamically excluded from further MS/MS scans for 45 sec for proteome or 30 sec for acetylproteome. Peptide match was set to preferred for monoisotopic peak determination, and charge state screening was enabled to only include and charge state screening was enabled to only include precursor charge states 2\\u20136, with an intensity threshold of 9.5e4 for proteome or 8e4 for acetylproteome. Advanced precursor determination feature (APD) (Myers et al., 2018) was turned off using a feature (APD) (Myers et al., 2018) was turned off using a software patch provided to us by Thermo Fisher Scientific allowing us to turn APD off in the tune file, Tune version 2.9.0.2926 (later versions of Exactive Tune 2.9 sp2 for the HFX have this option versions of Exactive Tune 2.9 sp2 for the HFX have this option as standard). For phosphoproteome and ubiquitylproteome analysis, samples were analyzed with a benchtop Orbitrap Fusion Lumos mass spectrometer (Thermo Fisher Scientific) equipped with a NanoSpray Flex NG ion source. Data-dependent acquisition was performed using NG ion source. Data-dependent acquisition was performed using Xcalibur Orbitrap Fusion Lumos v3.0 software in positive ion mode at a spray voltage of 1.8 kV. MS1 Spectra were measured with a resolution of 60,000, an AGC target of 4e5 and a mass range from of 60,000, an AGC target of 4e5 and a mass range from 350 to 1800 m/z. The data-dependent mode cycle time was set at 2 seconds with a MS2 resolution of 50,000, an AGC target of 6e4, an isolation window of 0.7 m/z, a maximum injection time of 105 msec (125 window of 0.7 m/z, a maximum injection time of 105 msec (125 msec for ubiquitylproteome), and an HCD collision energy of 36%. Peptide mode was selected for monoisotopic peak determination, and charge state screening was enabled to only include precursor charge state screening was enabled to only include precursor charge states 2\\u20136, with an intensity threshold of 1e4. Peptides that triggered MS/MS scans were dynamically excluded from further MS/MS scans for 45 sec (20 sec for ubiquitylproteome), with a MS/MS scans for 45 sec (20 sec for ubiquitylproteome), with a +/\\u2212 10 ppm mass tolerance. \\u201cPerform dependent scan on single charge state per precursor only\\u201d was enabled for phosphoproteome analysis, but disabled for ubiquitylproteome analysis. Immunohistochemistry (IHC) analysis.\\n\\nTMA construction Seventy-one LSCC paraffin tissue blocks were available for tissue microarray (TMA) construction and whole sections by immunohistochemistry (IHC) assay. TMAs were constructed in duplicate, and each included two (2) 1.0 mm cores providing up to four (4) and each included two (2) 1.0 mm cores providing up to four (4) cores per case for IHC analysis. Common control tissue on each TMA included patient-matched normal adjacent lung from 14 cases. Immune cell marker, Immunohistochemistry (IHC) Seventeen (17) samples were scored for immune cell markers. Sequential TMA sections were obtained for immune marker assessment to best approximate native spatial relationships. Chromogenic (DAB) IHC by single marker analysis was performed on 5-micron (DAB) IHC by single marker analysis was performed on 5-micron thick sections on standard charged slides. Automated immunostaining was performed by Autostainer Link 48, Dako, Inc. Heat induced epitope retrieval (HIER) was performed and biomarker dilutions epitope retrieval (HIER) was performed and biomarker dilutions as follows: CD4 clone 4B12, Bio-Rad 1:50; CD8 clone 4B11, Bio-Rad, 1:50; CD163, Abcam, 1:500 and ARHGDIB, HPA051235 Sigma-Aldrich, 1:50. IHC analysis was performed independently by three 1:50. IHC analysis was performed independently by three independent pathologists by staining intensity of 0\\u20133 scale for each TMA core separately and then by average score / case for a total of 15 cases. Case scores compared across independent pathologists 15 cases. Case scores compared across independent pathologists showed very good agreement. For evaluating statistical significance of IHC scores across the three immune clusters, ordinal regression was used, implemented using the \\u201cordinal\\u201d package in R. was used, implemented using the \\u201cordinal\\u201d package in R. Random effects accounted for the correlation between multiple cores from each sample and multiple pathologists. Multiplex Immunofluorescence (mIF) staining Development and optimization of the mIF platform has been previously described. mIF staining was performed in a 4-\\u03bcm-thick section obtained from a FFPE TMA block, using the Opal 7-Color Kit (Akoya Biosciences, Waltham, MA, USA) and scanned using a Vectra Biosciences, Waltham, MA, USA) and scanned using a Vectra multispectral microscope (Akoya Biosciences). The IF markers used were pan-Cytokeratin (CK, clone AE1/AE3, (DAKO) and Alpha-SMA ab5694 (Abcam). Multiplexed stained sections were imaged using the (Abcam). Multiplexed stained sections were imaged using the VECTRA multispectral imaging system (Akoya Biosciences). TMA cores were annotated using InForm 2.4.8 image analysis software (Akoya Biosciences). IHC based assessment of TP63 low samples (Figure 4) For samples with low TP63 status, additional pathologic confirmation was performed. Pathology reassessment confirmed 8/10 samples to be unequivocal LSCC; one sample (C3N.00247) had adenosquamous but predominantly squamous histology; the remaining sample but predominantly squamous histology; the remaining sample (C3N.02283) showed basaloid squamous cell features and despite some atypical characteristics retained its LSCC attribution after review by four separate pathologists - with histology very similar by four separate pathologists - with histology very similar to a TP63-low TCGA tumor (TCGA-66\\u20132754) deemed LSCC by TCGA pathology. QUANTIFICATION AND STATISTICAL ANALYSIS\\n\\nGenomic Data Analysis\\n\\nCopy Number Calling Copy-number analysis was performed jointly leveraging both whole-genome sequencing (WGS) and whole-exome sequencing (WES) data of the tumor and germline DNA, using CNVEX (https://github.com/mctp/cnvex). CNVEX uses whole-genome aligned reads to estimate CNVEX uses whole-genome aligned reads to estimate coverage within fixed genomic intervals, and whole-genome and whole-exome variant calls to compute B-allele frequencies at variable positions (we used TNScope germline calls). Coverages were computed in (we used TNScope germline calls). Coverages were computed in 10kb bins, and the resulting log coverage ratios between tumor and normal samples were adjusted for GC bias using weighted LOESS smoothing across mappable and non-blacklisted genomic intervals smoothing across mappable and non-blacklisted genomic intervals within the GC range 0.3\\u20130.7, with a span of 0.5 (the target, blacklist, and configuration files are provided with CNVEX). The adjusted log coverage ratios (LR) and B-allele frequencies (BAF) log coverage ratios (LR) and B-allele frequencies (BAF) were jointly segmented by custom algorithm based on Circular Binary Segmentation (CBS). Alternative probabilistic algorithms were implemented in CNVEX, including algorithms based on recursive binary in CNVEX, including algorithms based on recursive binary segmentation (RBS), and dynamic programming, as implemented in the R-package jointseg. For the CBS-based algorithm, first LR and mirrored BAF were independently segmented using CBS (parameters mirrored BAF were independently segmented using CBS (parameters alpha=0.01, trim=0.025) and all candidate breakpoints collected. The resulting segmentation track was iteratively \\u201cpruned\\u201d by merging segments that had similar LR, BAFs and short lengths. For segments that had similar LR, BAFs and short lengths. For the RBS- and DP-based algorithms, joint-breakpoints were \\u201cpruned\\u201d using a statistical model selection method. For the final set of CNA segments, we chose the CBS-based results as they did not of CNA segments, we chose the CBS-based results as they did not require specifying a prior on the number of expected segments (K) per chromosome arm, were robust to unequal variances between the LR and BAF tracks, and provided empirically the best fit to the LR and BAF tracks, and provided empirically the best fit to the underlying data. Somatic Variant Calling Somatic variant calling was performed on exome-capture BAM files aligned to the GDC GRCh38 reference. Following de-duplication and co-realignment around known indels \\u201cGATK bundle known\\u201d and \\u201cMills 1000G\\u201d somatic variants were called using Sentieon TNScope 1000G\\u201d somatic variants were called using Sentieon TNScope with the following settings \\u201c--max_fisher_pv_active 0.05 --min_tumor_allele_frac 0.0075 --min_init_tumor_lod 2.5 --assemble_mode 4 --SVIntrusionThres 5\\u201d, with a provided germline variants from 4 --SVIntrusionThres 5\\u201d, with a provided germline variants from dbSNP138. The resulting VCF files were annotated and filtered, in a series of post-processing steps which included: 1) restriction of variants to the hybrid-capture target regions. 2) Phasing of variants to the hybrid-capture target regions. 2) Phasing of adjacent and separated by 1bp SNVs into multi-nucleotide variants based on the haplotype information provided by TNScope. 3) Annotation using VEP with the following settings \\u201c--assembly 3) Annotation using VEP with the following settings \\u201c--assembly GRCh38 --species homo_sapiens --cache_version 97 --format vcf --gene_phenotype --symbol --canonical --ccds --hgvs --biotype --tsl --uniprot --domains --appris --protein --variant_class --sift --uniprot --domains --appris --protein --variant_class --sift b, --polyphen b --no_stats --total_length --allele_number --no_escape --flag_pick_allele --pick_order canonical,tsl,biotype,rank,ccds,length --buffer_size 20000\\u201d. Additional annotations were --buffer_size 20000\\u201d. Additional annotations were attached to the variants using VcfAnno including allele frequencies in GNOMAD (v2.0.1), ClinVar (downloaded Dec. 2019), Cosmic (downloaded Dec. 2019), dbSNP (20180418). The resulting variants are annotated 2019), dbSNP (20180418). The resulting variants are annotated against both RefSeq and Ensembl transcripts and contain multiple predicted consequences per-variant. For most downstream applications these annotations were reduced to a single predicted these annotations were reduced to a single predicted consequence per-variant and converted into a MAF file using custom software. The choice of the most-relevant consequence was based on a number of features. In general, the most \\u2018severe\\u2019 consequence a number of features. In general, the most \\u2018severe\\u2019 consequence occuring in a \\u2018reliable\\u2019 transcript was chosen. To accomplish this the computed consequences were ranked by \\u2018severity\\u2019 and \\u2018priority\\u2019 reflecting how deleterious/impactful a variant and \\u2018priority\\u2019 reflecting how deleterious/impactful a variant consequence is and how well-supported the existence and biological relevance of an isoform is. The specific rules differed for RefSeq and Ensembl annotations. RefSeq transcripts were ranked RefSeq and Ensembl annotations. RefSeq transcripts were ranked higher if they were known to CCDS, LRG or ranked \\u2018CANONICAL\\u2019 or \\u2018PICK\\u2019 by VEP. Ensembl transcripts were further prioritized by Basic, Appris and TSL annotations. Ensembl-based consequences Basic, Appris and TSL annotations. Ensembl-based consequences were prefered in case of ties. The resulting variant sets were filtered to only include variants with a high likelihood of being somatic based on a set of probabilistic and heuristic of being somatic based on a set of probabilistic and heuristic thresholds. A reported variant had to pass all of the following filters: 5 or more variant reads, fewer than 2 variant reads in the germline allele frequency in the normal 0.005, allele in the germline allele frequency in the normal 0.005, allele frequency in the tumor greater than 0.025, TLOD > 5, NLOD > 6, Fisher\\u2019s test p-value < 0.05, NFLOD > 1.5. Additionally heuristic filters were implemented to remove strand-bias, variants with filters were implemented to remove strand-bias, variants with mapping quality bias, and variants with high population allele frequency in GNOMAD. Germline Variant Calling Germline variant calling and annotation was carried out analogously using DNAScope using the following settings \\u201c--emit_conf 10 --call_conf 10\\u201d to generate a VCF file and \\u201c--emit_mode GVCF\\u201d to generate a gVCF file. The variants were custom-filtered based generate a gVCF file. The variants were custom-filtered based on the specific applications. GISTIC and MutSig analysis The Genomic Identification of Significant Targets in Cancer (GISTIC2.0) algorithm was used to identify significantly amplified or deleted focal-level and arm-level events, with Q value <0.25 considered significant. The following parameters were used: Amplification Threshold = 0.1\\n\\nDeletion Threshold = 0.1\\n\\nCap Values = 1.5\\n\\nBroad Length Cutoff = 0.98\\n\\nRemove X-Chromosome = 0\\n\\nConfidence Level = 0.99\\n\\nJoin Segment Size = 4\\n\\nArm Level Peel Off = 1\\n\\nMaximum Sample Segments = 2000\\n\\nGene GISTIC = 1 Each gene of every sample is assigned a thresholded copy number level that reflects the magnitude of its deletion or amplification. These are integer values ranging from \\u22122 to 2, where 0 means no amplification or deletion of magnitude greater than the no amplification or deletion of magnitude greater than the threshold parameters described above. Amplifications are represented by positive numbers: 1 means amplification above the amplification threshold; 2 means amplification larger than the arm level threshold; 2 means amplification larger than the arm level amplifications observed in the sample. Deletions are represented by negative numbers: \\u22121 means deletion beyond the threshold; \\u22122 means deletions greater than the minimum arm-level copy number means deletions greater than the minimum arm-level copy number observed in the sample. The somatic variants were filtered through a panel of normals to remove potential sequencing artifacts and undetected germline variants. MutSig2CV was run on these filtered results to evaluate the significance of mutated genes and estimate mutation the significance of mutated genes and estimate mutation densities of samples. These results were constrained to genes in the Cancer Gene Census, with false discovery rates (q values) recalculated. Genes of q value < 0.1 were declared significant. RNAseq and miRNAseq Quantification\\n\\nRNAseq Quantification The readcount is generated by featureCounts (subread v1.6.4) under stranded mode with parameters: `-g gene_id -t exon -Q 10 -p -B -s 2`. Gene annotation in use is identical to GDC (`gencode.v22.annotation.gtf.gz`; md5: `291330bdcff1094bc4d5645de35e0871`), md5: `291330bdcff1094bc4d5645de35e0871`), which is available on the GDC Reference Files page (https://gdc.cancer.gov/about-data/gdc-data-processing/gdc-reference-files). The readcount is later converted to FPKM and FPKM-UQ using GDC\\u2019s formula is later converted to FPKM and FPKM-UQ using GDC\\u2019s formula (https://docs.gdc.cancer.gov/Data/Bioinformatics_Pipelines/Expression_mRNA_Pipeline/). Isoform specific RNA quantification For TP63 isoform centric analysis presented in Figure 4, the hg38 reference genome and RefSeq annotations were used for the RNAseq data analysis and were downloaded from the UCSC table browser. First, CIRI (v2.0.6) was used to call circular RNA with First, CIRI (v2.0.6) was used to call circular RNA with default parameters and BWA (version 0.7.17-r1188) was used as the mapping tool. The cutoff of supporting reads for circRNAs was set to 10. Then we used a pseudo-linear transcript strategy to quantify Then we used a pseudo-linear transcript strategy to quantify gene and circular RNA expression. In brief, for each sample, linear transcripts of circular RNAs were extracted and 75bp (read length) from the 3\\u2019 end was copied to the 5\\u2019 end. The modified length) from the 3\\u2019 end was copied to the 5\\u2019 end. The modified transcripts were called pseudo-linear transcripts. Transcripts of linear genes were also extracted and mixed with pseudo-linear transcripts. RSEM (version 1.3.1) with Bowtie2 (version 2.3.3) transcripts. RSEM (version 1.3.1) with Bowtie2 (version 2.3.3) as the mapping tool was used to quantify gene, isoform, and circular RNA expression based on the mixed transcripts. DNA methylation data preprocessing Raw methylation image files were downloaded from the CPTAC DCC (See data availability). We calculated and analyzed methylated (M) and unmethylated (U) intensities for LSCC samples as described previously. We flagged locus as NA where probes did not meet a previously. We flagged locus as NA where probes did not meet a detection p-value of 0.01. Probes with MAF more than 0.1 were removed, and samples with more than 85% NA values were removed. Resulting beta values of methylation were utilized for subsequent beta values of methylation were utilized for subsequent analysis. Gene-level methylation scores were generated by taking the mean beta values of probes in the CpG islands of promoters and 5\\u2019 UTR regions of the gene. Methylation profiles (i.e., density plots) of some samples had unexpectedly skewed distributions of plots) of some samples had unexpectedly skewed distributions of methylation beta values, in addition to significantly more missing values. To systematically determine the subset of methylation samples with these evident data QC issues, we subjected all samples with these evident data QC issues, we subjected all the samples to model-based clustering using the Mclust package in R, using the interquartile range over all the genes as the representative metric. The clustering automatically determined the metric. The clustering automatically determined the optimal number of clusters, and identified 3 clusters. Two of these clusters (with centroids at 0.029 and 0.038) captured the bulk of the samples (202). The third cluster (centroid at 0.165, of the samples (202). The third cluster (centroid at 0.165, significantly higher than the other two clusters) consisted of 4 samples that belong to this cluster with high confidence (uncertainty < 5%), each of which had a skewed distribution of beta < 5%), each of which had a skewed distribution of beta values with a mean of 4,407 missing values per sample (in contrast to 1.8 missing values per sample for clusters 1 and 2 combined). Based on this analysis, we concluded that the 4 samples in cluster 3 on this analysis, we concluded that the 4 samples in cluster 3 represent samples with poor data quality. These have been excluded from all methylation analysis. miRNA-seq Data Analysis miRNA-seq unaligned bam files were downloaded from the CPTAC GDC API (https://docs.gdc.cancer.gov). Unaligned bams were first converted to fastq.gz by samtools bam2fq (samtools version 1.9). TPM (Transcripts per million) values of mature miRNA and 1.9). TPM (Transcripts per million) values of mature miRNA and precursor miRNA were reported after adaptor trimming, quality check, alignment, annotation, and read counting (https://github.com/ding-lab/CPTAC_miRNA/blob/master/cptac_mirna_analysis.md). The The mature miRNA expression was calculated irrespective of its gene of origin by summing the expression. Proteomics Data Analysis\\n\\nSpectrum quality filtering and searching All MS data were interpreted using the Spectrum Mill software package v7.0 pre-release (Agilent Technologies, Santa Clara, CA) co-developed by Karl Clauser of the Carr laboratory (https://www.broadinstitute.org/proteomics). Similar MS/MS spectra acquired Similar MS/MS spectra acquired on the same precursor m/z within +/\\u2212 45 sec were merged. MS/MS spectra were excluded from searching if they failed the quality filter by not having a sequence tag length > 0 (i.e., minimum of two masses separated by the tag length > 0 (i.e., minimum of two masses separated by the in-chain mass of an amino acid) or did not have a precursor MH+ in the range of 800\\u20136000. MS/MS spectra were searched against a RefSeq-based sequence database containing 41,457 proteins mapped sequence database containing 41,457 proteins mapped to the human reference genome (GRCh38/hg38) obtained via the UCSC Table Browser (https://genome.ucsc.edu/cgi-bin/hgTables) on June 29, 2018, with the addition of 13 proteins encoded in the human 29, 2018, with the addition of 13 proteins encoded in the human mitochondrial genome, 264 common laboratory contaminant proteins, and 553 non-canonical small open reading frames. Scoring parameters were ESI-QEXACTIVE-HCD-v2, for whole proteome datasets, were ESI-QEXACTIVE-HCD-v2, for whole proteome datasets, and ESI-QEXACTIVE-HCD-v3, for phosphoproteome, acetylproteome, and ubiquitylproteome datasets. All spectra were allowed +/\\u2212 20 ppm mass tolerance for precursor and product ions, 30% minimum matched tolerance for precursor and product ions, 30% minimum matched peak intensity, and \\u201ctrypsin allow P\\u201d enzyme specificity with up to 4 missed cleavages. Allowed fixed modifications included carbamidomethylation of cysteine and selenocysteine. TMT labeling of cysteine and selenocysteine. TMT labeling was required at lysine, but peptide N-termini were allowed to be either labeled or unlabeled. Allowed variable modifications for whole proteome datasets were acetylation of protein N-termini, oxidized datasets were acetylation of protein N-termini, oxidized methionine, deamidation of asparagine, hydroxylation of proline in PG motifs, pyroglutamic acid at peptide N-terminal glutamine, and pyro-carbamidomethylation at peptide N-terminal cysteine with a pyro-carbamidomethylation at peptide N-terminal cysteine with a precursor MH+ shift range of \\u221218 to 97 Da. For the phosphoproteome dataset the allowed variable modifications were revised to allow phosphorylation of serine, threonine, and tyrosine, allow allow phosphorylation of serine, threonine, and tyrosine, allow deamidation only in NG motifs, and disallow hydroxylation of proline with a precursor MH+ shift range of \\u221218 to 272 Da. For the acetylproteome dataset the allowed variable modifications were acetylproteome dataset the allowed variable modifications were revised to allow acetylation of lysine, allow deamidation only in NG motifs, and disallow hydroxylation of proline with a precursor MH+ shift range of \\u2212400 to 70 Da. For the ubiquitylproteome MH+ shift range of \\u2212400 to 70 Da. For the ubiquitylproteome dataset the allowed variable modifications were revised to allow diglycine modification of lysine, allow deamidation only in NG motifs, and disallow hydroxylation of proline with a precursor MH+ and disallow hydroxylation of proline with a precursor MH+ shift range of \\u2212690 to 70 Da. Protein grouping, and localization of PTMs Identities interpreted for individual spectra were automatically designated as confidently assigned using the Spectrum Mill autovalidation module to use target-decoy based false discovery rate (FDR) estimates to apply score threshold criteria. For the rate (FDR) estimates to apply score threshold criteria. For the whole proteome dataset thresholding was done in 3 steps: at the peptide spectrum match (PSM) level, the protein level for each TMT-plex, and the protein level for all 22 TMT-plexes. For the TMT-plex, and the protein level for all 22 TMT-plexes. For the phosphoproteome, acetylproteome, and ubiquitylproteome datasets thresholding was done in two steps: at the PSM and variable modification (VM) site levels. In step 1 for all datasets, PSM-level autovalidation was done first and separately for each TMT-plex experiment consisting of either 25 LC-MS/MS runs (whole proteome), 13 LC-MS/MS runs (phosphoproteome), 4 LC-MS/MS runs (acetylproteome), or 2 LC-MS/MS 4 LC-MS/MS runs (acetylproteome), or 2 LC-MS/MS runs (ubiquitylproteome) using an auto-thresholds strategy with a minimum sequence length of 7; automatic variable range precursor mass filtering; and score and delta Rank1 \\u2013 Rank2 score thresholds optimized and score and delta Rank1 \\u2013 Rank2 score thresholds optimized to yield a PSM-level FDR estimate for precursor charges 2 through 4 of <0.8% for each precursor charge state in each LC-MS/MS run. To achieve reasonable statistics for precursor charges 5\\u20136, To achieve reasonable statistics for precursor charges 5\\u20136, thresholds were optimized to yield a PSM-level FDR estimate of <0.4% across all runs per TMT-plex experiment (instead of per each run), since many fewer spectra are generated for the higher run), since many fewer spectra are generated for the higher charge states. In step 2 for the whole proteome dataset, protein-polishing autovalidation was applied separately to each TMTplex experiment to further filter the PSMs using a target protein-level FDR threshold of zero. The primary goal of this step was to eliminate of zero. The primary goal of this step was to eliminate peptides identified with low scoring PSMs that represent proteins identified by a single peptide, so-called \\u201cone-hit wonders.\\u201d After assembling protein groups from the autovalidated PSMs, protein assembling protein groups from the autovalidated PSMs, protein polishing determined the maximum protein level score of a protein group that consisted entirely of distinct peptides estimated to be false-positive identifications (PSMs with negative delta to be false-positive identifications (PSMs with negative delta forward-reverse scores). PSMs were removed from the set obtained in the initial peptide-level autovalidation step if they contributed to protein groups that had protein scores below the contributed to protein groups that had protein scores below the maximum false-positive protein score. Step 3 was then applied, consisting of protein-polishing autovalidation across all TMT plexes together using the protein grouping method \\u201cexpand TMT plexes together using the protein grouping method \\u201cexpand subgroups, top uses shared\\u201d to retain protein subgroups with either a minimum protein score of 25 or observation in at least 4 TMT plexes. The primary goal of this step was to eliminate low TMT plexes. The primary goal of this step was to eliminate low scoring proteins that were infrequently detected in the sample cohort. As a consequence of these two protein-polishing steps, each identified protein reported in the study comprised multiple identified protein reported in the study comprised multiple peptides, unless a single excellent scoring peptide was the sole match and that peptide was observed in at least 4 TMT-plexes. In calculating scores at the protein level and reporting the In calculating scores at the protein level and reporting the identified proteins, peptide redundancy was addressed in Spectrum Mill as follows: The protein score was the sum of the scores of distinct peptides. A distinct peptide was the single highest of distinct peptides. A distinct peptide was the single highest scoring instance of a peptide detected through an MS/MS spectrum. MS/MS spectra for a particular peptide may have been recorded multiple times (e.g. as different precursor charge states, in multiple times (e.g. as different precursor charge states, in adjacent bRP fractions, modified by deamidation at Asn or oxidation of Met, or with different phosphosite localization), but were still counted as a single distinct peptide. When a peptide were still counted as a single distinct peptide. When a peptide sequence of >8 residues was contained in multiple protein entries in the sequence database, the proteins were grouped together and the highest scoring one and its accession number were and the highest scoring one and its accession number were reported. In some cases when the protein sequences were grouped in this manner, there were distinct peptides that uniquely represent a lower scoring member of the group (isoforms, family members, a lower scoring member of the group (isoforms, family members, and different species). Each of these instances spawned a subgroup. Multiple subgroups were reported, counted towards the total number of proteins, and given related protein subgroup numbers number of proteins, and given related protein subgroup numbers (e.g. 3.1 and 3.2 for group 3, subgroups 1 and 2). For the whole proteome datasets the above criteria yielded false discovery rates (FDR) for each TMT-plex experiment of <0.6% at the rates (FDR) for each TMT-plex experiment of <0.6% at the peptide-spectrum match level and <0.8% at the distinct peptide level. After assembling proteins with all the PSMs from all the TMT-plex experiments together, the aggregate FDR estimates were 0.58% experiments together, the aggregate FDR estimates were 0.58% at the peptide-spectrum match level, 2.3% at the distinct peptide level, and <0.01% (1/11,029) at the protein group level. Since the protein level FDR estimate neither explicitly required a the protein level FDR estimate neither explicitly required a minimum number of distinct peptides per protein nor adjusted for the number of possible tryptic peptides per protein, it may underestimate false positive protein identifications for large underestimate false positive protein identifications for large proteins observed only on the basis of multiple low scoring PSMs. In step 2 for the phosphoproteome and acetylproteome datasets, variable modification (VM) site polishing autovalidation was applied across all 22 TMT plexes to retain all VM-site identifications with either a minimum id score of 8.0 or observation in at with either a minimum id score of 8.0 or observation in at least 4 TMT plexes. For the ubiquitylproteome datasets, it was applied across all 15 TMT plexes to retain all VM-site identifications with either a minimum id score of 8.0 or observation in at with either a minimum id score of 8.0 or observation in at least 3 TMT plexes. The intention of the VM site polishing step is to control FDR by eliminating unreliable VM site-level identifications, particularly low scoring VM sites that are only detected particularly low scoring VM sites that are only detected as low scoring peptides that are also infrequently detected across all of the TMT plexes in the study. In calculating scores at the VM-site level and reporting the identified VM sites, redundancy VM-site level and reporting the identified VM sites, redundancy was addressed in Spectrum Mill as follows: A VM-site table was assembled with columns for individual TMT-plex experiments and rows for individual VM-sites. PSMs were combined into a single rows for individual VM-sites. PSMs were combined into a single row for all non-conflicting observations of a particular VM-site (e.g. different missed cleavage forms, different precursor charges, confident and ambiguous localizations, and different charges, confident and ambiguous localizations, and different sample-handling modifications). For related peptides, neither observations with a different number of VM-sites nor different confident localizations were allowed to be combined. Selecting the localizations were allowed to be combined. Selecting the representative peptide from the combined observations was done such that once confident VM-site localization was established, higher identification scores and longer peptide lengths were preferred. scores and longer peptide lengths were preferred. While a Spectrum Mill identification score was based on the number of matching peaks, their ion type assignment, and the relative height of unmatched peaks, the VM site localization score was the of unmatched peaks, the VM site localization score was the difference in identification score between the top two localizations. The score threshold for confident localization, >1.1, essentially corresponded to at least 1 b or y ion located between two corresponded to at least 1 b or y ion located between two candidate sites that has a peak height >10% of the tallest fragment ion (neutral losses of phosphate from the precursor and related ions as well as immonium and TMT reporter ions were excluded from as well as immonium and TMT reporter ions were excluded from the relative height calculation). The ion type scores for b-H3PO4, y-H3PO4, b-H2O, and y-H2O ion types were all set to 0.5. This prevented inappropriate confident localization assignment when a inappropriate confident localization assignment when a spectrum lacked primary b or y ions between two possible sites but contained ions that could be assigned as either phosphate-loss ions for one localization or water loss ions for another localization. one localization or water loss ions for another localization. VM-site polishing yielded 68, 674 phosphosites with an aggregate FDR of 0.73% at the phosphosite level. In aggregate, 71% of the reported phosphosites in this study were fully localized to a reported phosphosites in this study were fully localized to a particular serine, threonine, or tyrosine residue. VM-site polishing yielded 15,186 acetylsites with an aggregate FDR of 0.80% at the acetylsite level. In aggregate, 99% of the reported at the acetylsite level. In aggregate, 99% of the reported acetylsites in this study were fully localized to a particular lysine residue. VM-site polishing yielded 25,489 ubiquitylsites with an aggregate FDR of 0.40% at the ubiquitylsite level. In with an aggregate FDR of 0.40% at the ubiquitylsite level. In aggregate, >99% of the reported ubiquitylsites in this study were fully localized to a particular lysine residue. Quantitation using TMT ratios Using the Spectrum Mill Protein/Peptide Summary module, a protein comparison report was generated for the proteome dataset using the protein grouping method \\u201cexpand subgroups, top uses shared\\u201d (SGT). For the phosphoproteome and acetylproteome datasets a (SGT). For the phosphoproteome and acetylproteome datasets a Variable Modification site comparison report limited to either phospho or acetyl sites, respectively, was generated using the protein grouping method \\u201cunexpand subgroups\\u201d. Relative abundances of grouping method \\u201cunexpand subgroups\\u201d. Relative abundances of proteins and VM-sites were determined in Spectrum Mill using TMT reporter ion intensity ratios from each PSM. TMT reporter ion intensities were corrected for isotopic impurities in the Spectrum were corrected for isotopic impurities in the Spectrum Mill Protein/Peptide summary module using the afRICA correction method, which implements determinant calculations according to Cramer\\u2019s Rule and correction factors obtained from the reagent Cramer\\u2019s Rule and correction factors obtained from the reagent manufacturer\\u2019s certificate of analysis (https://www.thermofisher.com/order/catalog/product/90406) for TMT11_lot number UA280170/TL272399 (Proteome, Phosphoproteme, and Acetylproteome) and (Proteome, Phosphoproteme, and Acetylproteome) and TE270748-TD264064 (Ubiquitylproteome). A protein-level, phosphosite-level, or acetylsite-level TMT ratio is calculated as the median of all PSM-level ratios contributing to a protein subgroup, of all PSM-level ratios contributing to a protein subgroup, phosphosite, or acetylsite. PSMs were excluded from the calculation if they lacked a TMT label, had a precursor ion purity < 50% (MS/MS has significant precursor isolation contamination from (MS/MS has significant precursor isolation contamination from co-eluting peptides), or had a negative delta forward-reverse identification score (half of all false-positive identifications). Lack of TMT label led to exclusion of PSMs per TMT plex with a Lack of TMT label led to exclusion of PSMs per TMT plex with a range of 1.4 to 2.3% for the proteome, 1.8 to 3.7% for the phosphoproteome, 1.0 to 3.5% for the acetylproteome, and 4.5 to 18.0% for the ubiquitylproteome datasets. Low precursor ion purity for the ubiquitylproteome datasets. Low precursor ion purity led to exclusion of PSMs per TMT plex with a range of 1.1 to 1.4% for the proteome, 1.7 to 2.8% for the phosphoproteome, 0.5 to 1.7% for the acetylproteome, and 2.8 to 3.5% for the 0.5 to 1.7% for the acetylproteome, and 2.8 to 3.5% for the ubiquitylproteome datasets. Two-component normalization of TMT ratios It was assumed that for every sample there would be a set of unregulated proteins, phosphosites, acetylsites or ubiquitylsites that have abundance comparable to the common reference (CR) sample. In the normalized sample, these proteins, phosphosites, sample. In the normalized sample, these proteins, phosphosites, acetylsites or ubiquitylsites should have a log TMT ratio centered at zero. In addition, there were proteins, phosphosites, acetylsites and ubiquitylsites that were either up- or acetylsites and ubiquitylsites that were either up- or down-regulated compared to the CR. A normalization scheme was employed that attempted to identify the unregulated proteins, phosphosites, acetylsites or ubiquitylsites, and centered the distribution acetylsites or ubiquitylsites, and centered the distribution of these log-ratios around zero in order to nullify the effect of differential protein loading and/or systematic MS variation. A 2-component Gaussian mixture model-based normalization algorithm Gaussian mixture model-based normalization algorithm was used to achieve this effect. The two Gaussians N(\\u03bci1,\\u03c3i1) and N(\\u03bci2,\\u03c3i2) for a sample i were fitted and used in the normalization process as follows: the mode mi of the log-ratio distribution was as follows: the mode mi of the log-ratio distribution was determined for each sample using kernel density estimation with a Gaussian kernel and Shafer-Jones bandwidth. A two-component Gaussian mixture model was then fit with the mean of both Gaussians mixture model was then fit with the mean of both Gaussians constrained to be mi, i.e., \\u03bci1=\\u03bci2=mi. The Gaussian with the smaller estimated standard deviation  was assumed to represent the unregulated component of was assumed to represent the unregulated component of proteins/phosphosites/acetylsites/ubiquitylsites, and was used to normalize the sample. The sample was standardized using (mi,\\u03c3i) by subtracting the mean mi from each standardized using (mi,\\u03c3i) by subtracting the mean mi from each protein/phosphosite/acetylsite/ubiquitylsite and dividing by the standard deviation \\u03c3i. Normalization of acetyl- and ubiquitylproteome To correct for differential underlying protein levels when performing certain acetylproteome- and ubiquitylproteome-based analyses, a global linear model approach was employed to produce protein-normalized versions of both of these PTM datasets. PTM sites versions of both of these PTM datasets. PTM sites were first matched to underlying proteins in the proteome dataset using RefSeq accession numbers. If a site\\u2019s main accession number did not yield a match in the proteome, all accession numbers associated yield a match in the proteome, all accession numbers associated with the site were then tried and the protein with the highest score was selected as a match. For the acetylproteome and ubiquitylproteome separately, a linear model was then fit (R function separately, a linear model was then fit (R function lm, PTM ~ protein) to all matched PTM-protein data points in the dataset. The residuals of each model were then used as protein-normalized acetylsite or ubiquitylsite abundances. Proteogenomic analysis\\n\\nQuality control and Batch effect assessment We used guided principal component analysis (gPCA) to assess the presence of a batch effect in the sample/CR ratio data based on TMT-plex. There was no overall apparent batch effect (gPCA p-value=1) in the global proteome, phosphoproteome, acetylproteome in the global proteome, phosphoproteome, acetylproteome or ubiquitylproteome datasets. To assess composition similarities between CR used for proteome, phosphoproteome and acetylproteome analysis, and ubiquitylproteome, we performed label-free proteomic analysis of each CRs in triplicates. Both CRs showed excellent correlation with median triplicates. Both CRs showed excellent correlation with median pearson correlation > 0.85. In addition, to assess data quality, we assessed the Pearson correlations of log TMT reporter ion intensities between all pairs of TMT plexes, separately for sample and CR channels. For each given pair of TMT plexes, correlations between CRs were each given pair of TMT plexes, correlations between CRs were consistently higher than correlations of respective sample channels enabling precise relative quantitation of proteins, phosphorylation and acetylation sites across 22 TMT plexes using the CR and acetylation sites across 22 TMT plexes using the CR channels as common denominator (Figure S1C). In the ubiquitylproteome data we observed lower correlations of the CR channel (in comparison to samples channels) in three of the 18 TMT plexes analyzed to samples channels) in three of the 18 TMT plexes analyzed (plexes 7, 8, 10). Clustering of the CR channels using pairwise correlations as a similarity metric pointed to a batch effect in the CR channels (but not in the sample channels) in these plexes. CR channels (but not in the sample channels) in these plexes. The presence of a batch effect was further confirmed by applying the gPCA test described above to only the reporter intensities of the CR channels (gPCA p-value < 0.001). Even though of the CR channels (gPCA p-value < 0.001). Even though undetectable with a global test, we observed this CR batch effect propagate to the sample/CR ratio data in preliminary analyses (data not shown). To counter the propagation of this CR batch effect to shown). To counter the propagation of this CR batch effect to the sample/CR ratio for samples in these plexes, batch correction was applied to the sample/CR ratio data for plexes 7, 8, 10 (batch 1) and all other plexes (batch 2) using limma\\u2019s 7, 8, 10 (batch 1) and all other plexes (batch 2) using limma\\u2019s removeBatchEffect (https://rdrr.io/bioc/limma/man/removeBatchEffect.html) on the sample/CR ratio data including the sample type (tumor, NAT) as a covariate. Dataset filtering Genes (RNA-seq), proteins (global proteome), phosphosites, acetylsites and ubiquitin sites present in fewer than 30% of samples (i.e., missing in >70% of samples) were removed from the respective datasets. Furthermore, proteins were required to have at datasets. Furthermore, proteins were required to have at least two observed TMT ratios in >25% of samples in order to be included in the proteome dataset. Phosphosites, acetylsites and ubiquitin sites were required to have at least one observed TMT ratio sites were required to have at least one observed TMT ratio in >25% of samples. Some of the filtering steps were modified for specific analyses in the study. For many of the marker selection and gene set enrichment analyses, at least 50% of samples were required to have non-missing values for proteins/phosphosites/acetyl/ubiquitin non-missing values for proteins/phosphosites/acetyl/ubiquitin sites, since missing values were imputed, and excessive missing values can result in poor imputation. Alternate filtering has been noted in descriptions of the relevant methods. Unsupervised multi-omic clustering using NMF We used non-negative matrix factorization (NMF)-based multi-omic clustering using protein, phosphosite, acetylsite, RNA transcript and gene copy number variants (CNV) as previously described (. Briefly, given a factorization rank k (where k is the number (. Briefly, given a factorization rank k (where k is the number of clusters), NMF decomposes a p x n data matrix V into two matrices W and H such that multiplication of W and H approximates V. Matrix H is a k x n matrix whose entries represent weights for Matrix H is a k x n matrix whose entries represent weights for each sample (1 to N) to contribute to each cluster (1 to k), whereas matrix W is a p x k matrix representing weights for each feature (1 to p) to contribute to each cluster (1 to k). Matrix H (1 to p) to contribute to each cluster (1 to k). Matrix H was used to assign samples to clusters by choosing the k with maximum score in each column of H. For each sample we calculated a cluster membership score as the maximal fractional score of the cluster membership score as the maximal fractional score of the corresponding column in matrix H. We defined a \\u201dcluster core\\u201d as the set of samples with cluster membership score > 0.5. To enable integrative multi-omic clustering we enforced all data types (and converted if necessary) to represent ratios to: (i) a common reference measured in each TMT plex (proteome, phosphoproteome, acetylproteome), (ii) an in silico common reference acetylproteome), (ii) an in silico common reference calculated as the median abundance across all samples (RNA expression) or (iii) to matching blood normal for CNA data. The CNA data was further filtered to only retain genes significantly altered was further filtered to only retain genes significantly altered (GISTIC2 thresholded of +2 or \\u22122) in at least 5% of all tumors. All data tables were then concatenated and all rows with missing values were removed. To remove uninformative features from the values were removed. To remove uninformative features from the dataset prior to NMF clustering we removed features with the lowest standard deviation (bottom 5th percentile) across all samples. Each column in the data matrix was further scaled and samples. Each column in the data matrix was further scaled and standardized such that all features from different data types were represented as z-scores. The resulting data matrix of z-scores into was converted to a non-negative input matrix required by into was converted to a non-negative input matrix required by NMF as follows: Create one data matrix with all negative numbers zeroed.\\n\\nCreate another data matrix with all positive numbers zeroed and the signs of all negative numbers removed. Concatenate both matrices resulting in a data matrix twice as large as the original, but containing only positive values and zeros and hence appropriate for NMF. The resulting matrix was then subjected to NMF analysis leveraging the NMF R-package and using the factorization method described in. To determine the optimal factorization rank k (number of clusters) for the multi-omic data matrix we tested a range of clusters) for the multi-omic data matrix we tested a range of clusters between k=2 and 8. For each k we factorized matrix V using 50 iterations with random initializations of W and H. To determine the optimal factorization rank we calculated two metrics the optimal factorization rank we calculated two metrics for each k: 1) cophenetic correlation coefficient measuring how well the intrinsic structure of the data was recapitulated after clustering and 2) the dispersion coefficient of the consensus matrix and 2) the dispersion coefficient of the consensus matrix as defined in measuring the reproducibility of the clustering across 50 iterations. The optimal k was defined as the maximum of the product of both metrics for cluster numbers between k=3 and 8 Having determined the optimal factorization rank k, in order to achieve robust factorization of the multi-omic data matrix V, we repeated the NMF analysis using 500 iterations with random initializations of W and H and performed the partitioning of initializations of W and H and performed the partitioning of samples into clusters as described above. Due to the non-negative transformation applied to the z-scored data matrix as described above, matrix W of feature weights contained two separate above, matrix W of feature weights contained two separate weights for positive and negative z-scores of each feature, respectively. In order to revert the non-negative transformation and to derive a single signed weight for each feature, we first and to derive a single signed weight for each feature, we first normalized each row in matrix W by dividing by the sum of feature weights in each row, aggregated both weights per feature and cluster by keeping the maximal normalized weight and and cluster by keeping the maximal normalized weight and multiplication with the sign of the z-score in the initial data matrix. Thus, the resulting transformed version of matrix Wsigned contained signed cluster weights for each feature in the input contained signed cluster weights for each feature in the input matrix. For functional characterization of clustering results by single sample Gene Set Enrichment Analysis (ssGSEA), we calculated normalized enrichment scores (NES) of cancer-relevant gene sets by projecting the matrix of signed multi-omic feature weights by projecting the matrix of signed multi-omic feature weights (Wsigned) onto Hallmark pathway gene sets using ssGSEA. To derive a single weight for each gene measured across multiple omics data types (protein, RNA, phosphorylation site, acetylation site, types (protein, RNA, phosphorylation site, acetylation site, CNA) we retained the weight with maximal absolute amplitude. We used the ssGSEA implementation available on https://github.com/broadinstitute/ssGSEA2.0 using the following parameters: gene.set.database=\\u201ch.all.v6.2.symbols.gmt\\u201d\\n\\nsample.norm.type=\\u201crank\\u201d\\n\\nweight=1\\n\\nstatistic=\\u201darea.under.RES\\u201d\\n\\noutput.score.type=\\u201cNES\\u201d\\n\\nnperm=1000\\n\\nglobal.fdr=TRUE\\n\\nmin.overlap=5\\n\\ncorrel.type=\\u201dz.score\\u201d correl.type=\\u201dz.score\\u201d\\n\\nTo test the association of the resulting clusters to clinical variables we used Fisher\\u2019s exact test (R function fisher.test) to test for overrepresentation in the set of samples defining the cluster core as described above. The entire workflow described above has been implemented as a module for PANOPLY which runs on Broad\\u2019s Cloud platform Terra (https://app.terra.bio/). The docker containers encapsulating the source code and required R-packages for NMF clustering and ssGSEA code and required R-packages for NMF clustering and ssGSEA have been submitted to Dockerhub (broadcptacdev/pgdac_mo_nmf:15, broadcptac/pgdac_ssgsea:5). The source code for ssGSEA is available on GitHub: https://github.com/broadinstitute/ssGSEA2.0. Integrative analysis with Stewart et al. To enable integrative analysis of our protein data with the protein data in Stewart et al. we first aggregated the protein-level data to gene-level data matrices by retaining the dominant isoform (identified by the lowest protein subgroup number) isoform (identified by the lowest protein subgroup number) associated with each gene symbol. To aggregate the Stewart et al. protein data we used the mean expression across all proteins mapping to the same gene symbol. Both gene-level protein data mapping to the same gene symbol. Both gene-level protein data matrices were then separately subjected to gene-level z-score transformation before joining the matrices using the unique gene symbols as keys. The resulting integrated protein data matrix was as keys. The resulting integrated protein data matrix was then subjected to the NMF-based clustering pipeline described above using the 3,366 genes quantified across all 216 tumors. RNA subtyping Previously reported LSCC gene expression-subtype signatures were downloaded from the original publication and applied to our RNA expression dataset using the centroid-based method in a manner similar to the TCGA study. Specifically, the RNA expression of similar to the TCGA study. Specifically, the RNA expression of the 208 signature genes, represented as FPKM, was mean-centered in the gene-wise manner, and each sample was correlated to each of the four centroid vectors representing the average signature of the four centroid vectors representing the average signature gene expression for the four subtypes. The samples were assigned to one of the four transcriptomics subtypes according to the highest correlations. Samples with insignificance correlation (p correlations. Samples with insignificance correlation (p > 0.01) were marked to have \\u2018undecided\\u2019 subtype. Chromosomal instability index The CIN score reflects the overall copy number aberration across the whole genome. From the segment-level CNA result, we used a weighted-sum approach to summarize the chromosome instability for each sample. The absolute segment-level log2 ratios of all for each sample. The absolute segment-level log2 ratios of all copy number segments (indicating the copy number aberration of these segments) within a chromosome were weighted by the segment length and summed up to derive the instability score for the length and summed up to derive the instability score for the chromosome. The genome-wide chromosome instability index was calculated by summing up the instability score of all 22 autosomes. Fusion detection and analysis Fusion calling was performed using a combination of CRISP, CODAC MI-ONCOSEQ pipeline, fusioncatcher_v1.10 and arriba_v1.1.0 (https://github.com/suhrig/arriba/). The fusions calls were compiled and visualized using AGFusion to display the predicted domain and visualized using AGFusion to display the predicted domain and exon structure of the fusion proteins as described. mRNA and Protein correlation To compare mRNA expression and protein abundance across samples we focused on the RNAseq data with 21,792 genes, and global proteome with 11,575 quantified proteins. Protein IDs were mapped to gene names. In total, 10,890 genes common to both RNAseq and to gene names. In total, 10,890 genes common to both RNAseq and proteome data spanning 108 tumor samples were used in the analysis. The analyses were carried out on normalized data: RNAseq data were log2 transformed, upper quartile normalized RPKM values, were log2 transformed, upper quartile normalized RPKM values, which were median-centered by row (i.e., gene); proteome data was two-component normalized as described earlier. Pearson correlation was calculated for each gene. Both correlation coefficient was calculated for each gene. Both correlation coefficient and p-value were recorded. Further, adjusted p-values were calculated using the Benjamini\\u2013Hochberg procedure. CNA-driven cis and trans effects Correlations between copy number alterations (CNA) and RNA, proteome, phosphoproteome acetylproteome and ubiquitylproteome (with proteome and PTM data mapped to genes, by choosing the most variable protein isoform/PTM site as the gene-level the most variable protein isoform/PTM site as the gene-level representative) were determined using Pearson correlation of common genes present in CNA-RNA-proteome (9,988 genes), CNA-RNA-phosphoproteome (5,144 genes), CNA-RNA-acetylproteome (1,344 genes) (5,144 genes), CNA-RNA-acetylproteome (1,344 genes) and CNA-RNA-ubiquitylproteome (2,616 genes). In addition, p-values (corrected for multiple testing using Benjamini-Hochberg FDR) for assessing the statistical significance of the correlation values were the statistical significance of the correlation values were also calculated. CNA trans-effects for a given gene were determined by identifying genes with statistically significant (FDR < 0.05) positive or negative correlations. CMAP analysis Candidate genes driving response to copy number alterations were identified using large-scale Connectivity Map (CMAP) queries. The CMAP is a collection of about 1.3 million gene expression profiles from cell lines treated with bioactive small molecules profiles from cell lines treated with bioactive small molecules (~20,000 drug perturbagens), shRNA gene knockdowns (~4,300) and ectopic expression of genes. The CMAP dataset is available on GEO (Series GSE92742). For this analysis, we use the Level 5 on GEO (Series GSE92742). For this analysis, we use the Level 5 (signatures from aggregating replicates) TouchStone dataset with 473,647 total profiles, containing 36,720 gene knock-down profiles, with measurements for 12,328 genes. See knock-down profiles, with measurements for 12,328 genes. See https://clue.io/GEO-guide for more information. To identify candidate driver genes, proteome profiles of copy number-altered samples were correlated with gene knockdown mRNA profiles in the above CMAP dataset, and enrichment of up/downregulated genes was evaluated. Normalized log2 copy num- ber values genes was evaluated. Normalized log2 copy num- ber values less than 0.3 defined deletion (loss), and values greater than +0.3 defined copy number amplifications (gains). In the copy number-altered samples (separately for CNA amplification and CNA samples (separately for CNA amplification and CNA deletion), the trans-genes (identified by significant correlation in \\u201cCNA driven cis and trans effects\\u201d above) were grouped into UP and DOWN categories by comparing the protein ratios of these genes to categories by comparing the protein ratios of these genes to their ratios in the copy number neutral samples (normalized log2 copy number between 0.3 and +0.3). The lists of UP and DOWN trans-genes were then used as queries to interrogate CMAP signatures were then used as queries to interrogate CMAP signatures and calculate weighted connectivity scores (WTCS) using the single-sample GSEA algorithm. The weighted connectivity scores were then normalized for each perturbation type and cell line to obtain normalized for each perturbation type and cell line to obtain normalized connectivity scores (NCS). See for details on WTCS and NCS. For each query we then identified outlier NCS scores, where a score was considered an outlier if it lay beyond 1.5 times a score was considered an outlier if it lay beyond 1.5 times the interquartile range of score distribution for the query. The query gene was designated a candidate driver if (i) the score outliers were statistically cis-enriched (Fisher test with BH-FDR were statistically cis-enriched (Fisher test with BH-FDR multiple testing correction) and (ii) the gene had statistically significant and positive cis-correlation. For a gene to be considered for inclusion in a CMAP query it needed to i) have a copy number change (amplification or deletion) in at least 15 samples; ii) have at least 20 significant trans genes; and iii) be on the list of shRNA knockdowns in the CMAP. genes; and iii) be on the list of shRNA knockdowns in the CMAP. 502 genes satisfied these conditions and resulted in 976 queries (CNA amplification and deletion combined) that were tested for enrichment. Thirty two (32) candidate driver genes were for enrichment. Thirty two (32) candidate driver genes were identified with Fisher\\u2019s test FDR < 0.25, using this process. In order to ensure that the identified candidate driver genes were not a random occurrence, we performed a permutation test to determine how many candidate driver genes would be identified with random input (Mertins et al., 2016). For the 976 queries with random input (Mertins et al., 2016). For the 976 queries used, we substituted the bona-fide trans-genes with randomly chosen genes, and repeated the CMAP enrichment process. To determine FDR, each permutation run was treated as a Poisson sample with FDR, each permutation run was treated as a Poisson sample with rate \\u03bb, counting the number of identified candidate driver genes. Given the small n ( = 10) and \\u03bb, a Score confidence interval was calculated (Barker, 2002) and the midpoint of the confidence calculated (Barker, 2002) and the midpoint of the confidence interval used to estimate the expected number of false positives. Using 10 random permutations, we determined the overall false discovery rate to be FDR = 0.33, with a 95% CI of (0.27, 0.40) for rate to be FDR = 0.33, with a 95% CI of (0.27, 0.40) for genes with either CNA amplification or deletion, and an FDR=0.06 (95% CI: 0.01, 0.12) for six (6) genes with both CNA amplification and deletion. To identify how many trans-correlated genes for all candidate regulatory genes could be directly explained by gene expression changes measured in the CMAP shRNA perturbation experiments, knockdown gene expression consensus signature z-scores knockdown gene expression consensus signature z-scores (knockdown/control) were used to identify regulated genes with \\u03b1 = 0.05, followed by counting the number of trans-genes in this list of regulated genes. To obtain biological insight into the list of candidate driver genes, we performed (i) enrichment analysis on samples with extreme CNA values (amplification or deletion) to identify statistically enriched sample annotation subgroups; and (ii) GSEA on enriched sample annotation subgroups; and (ii) GSEA on cis/trans-correlation values to find enriched pathways. LINCS analysis Differentially expressed genes between the 5 NMF LSCC subtypes were identified using the limma package. In order to identify genes that were uniquely expressed in each subtype, all pairwise comparisons between the 5 subtypes were computed. In addition, comparisons between the 5 subtypes were computed. In addition, genes that were differentially expressed (adjusted p.value < 0.05) in at least 3 out of the 4 comparisons and had a concordant fold change among all comparisons were identified as concordant fold change among all comparisons were identified as subtype-specific and were used in the subsequent analysis. Protein abundance comparisons were performed between all 5 NMF subtypes using the Wilcoxon rank-sum test and p-values were adjusted using the Benjamini & Hochberg method. Subtype-specific differentially expressed proteins were identified based on their expressed proteins were identified based on their differential expression (adjusted p.value < 0.05) in at least 2 out of the 4 comparisons, and by having a concordant fold change among all comparisons. The identified differentially expressed genes and comparisons. The identified differentially expressed genes and proteins were then filtered for gene symbols measured in the L1000 assay (978 landmark genes). These NMF-specific signatures were used as input to calculate normalized weighted connectivity used as input to calculate normalized weighted connectivity scores (WTCS) against the Library of Integrated Network-Based Cellular Signatures (LINCS) L1000 perturbation-response signatures. The scores were computed using the sig_queryl1k_tool pipeline The scores were computed using the sig_queryl1k_tool pipeline (https://hub.docker.com/u/cmap) and the LINCS L1000 Level 5 compound (trt_cp) signatures from CLUE (clue.io, \\u201cExpanded CMap LINCS Resource 2020 Release\\u201d). The resulting normalized connectivity Resource 2020 Release\\u201d). The resulting normalized connectivity scores were summarized across cell lines using the maximum quantile of all the scores of the same compound (Qhi = 67,Qlow = 33) as previously described. The corresponding compound metadata as previously described. The corresponding compound metadata were obtained from CLUE (clue.io, \\u201cExpanded CMap LINCS Resource 2020 Release\\u201d) and were used to filter and identify compounds with existing annotations (drug name, MOA, Target). The top 20 with existing annotations (drug name, MOA, Target). The top 20 negatively connected compounds to each NMF signature were selected for visualization. Defining cancer-associated genes\\n\\nCancer-associated genes (CAG) were compiled using the Census website https://cancer.sanger.ac.uk/census. The list of genes is provided in Table S1.\\n\\nCpG Island Methylator Phenotype The methylation analysis started with all 103 tumor samples, which excluded 4 samples from the cohort with high missing rate (see above), to generate the CpG island methylator phenotypes (CIMP). Specifically, we first generated the gene-level methylation Specifically, we first generated the gene-level methylation score, by taking the averaged beta values of all probes harboring in the islands of promotor or 5 UTR regions of the gene. Then, we preprocessed the data by filtering genes that were Then, we preprocessed the data by filtering genes that were hypermethylated, i.e. the gene-level methylation score >0.2, transformed the score into M-values, normalized the transformed values, and then imputed the missing values with k-nearest neighbors and then imputed the missing values with k-nearest neighbors (KNN). With preprocessed data, we performed consensus clustering 1000 times, each taking 80% of the samples and all genes, and calculated the consensus matrix (probabilities of two samples calculated the consensus matrix (probabilities of two samples clustering together) for each predetermined number of clusters K. K ranges from 2 to 10. In each value of K, we visualized the consensus matrix using K means with Euclidean distance as the consensus matrix using K means with Euclidean distance as the distance metric. Finally, we determined the optimal number of clusters by considering the relative change in area under the consensus cumulative density function (CDF) curve. In the end, three cumulative density function (CDF) curve. In the end, three distinct clusters were identified, one was hypermethylated with mean M value 0.393, and two were hypomethylated with mean M value \\u22120.041 and \\u22120.359, respectively. We labeled these three clusters and \\u22120.359, respectively. We labeled these three clusters as CIMP high, CIMP intermediate, and CIMP low groups. iProFun Based Cis Association Analysis We used iProFun, an integrative analysis tool to identify multi-omic molecular quantitative traits (QT) perturbed by DNA-level variations. We considered five functional molecular quantitative traits (gene expression, protein, phosphoprotein, quantitative traits (gene expression, protein, phosphoprotein, acetylprotein, ubiquitylproteome levels) for their associations with DNA methylation, accounting for mutation, copy number variation, age, gender, tumor purity and smoking status. Tumor purity age, gender, tumor purity and smoking status. Tumor purity was determined using TSNet from RNA-seq data. The iProFun procedure was applied to a total of 12,666 genes that was available in at least one predictor (mutation, CNA, DNA methylation) and one at least one predictor (mutation, CNA, DNA methylation) and one outcome (mRNA, global protein, phospholation, acetylation and ubiquitylation) for their cis regulatory patterns in tumors. For each of the 12,666 genes, we started with linear regression to each of the 12,666 genes, we started with linear regression to capture the covariate-adjusted associations. The resulting association summary statistics was used to call probabilities of belonging to each of the 2^5=32 possible configurations. We of belonging to each of the 2^5=32 possible configurations. We calculated the probabilities of associating with each of the five outcomes by combining probabilities from relevant configurations, and calculated FDR controlled associations. The significant and calculated FDR controlled associations. The significant genes need to pass three criteria: (1) the satisfaction of biological filtering procedure, (2) posterior probabilities > 75%, and (3) empirical false discovery rate (eFDR)<10%. As DNA methylation empirical false discovery rate (eFDR)<10%. As DNA methylation at the promoter regions of a gene down regulates its expression level, biological filtering procedure requires DNA methylations that exist either negative significant associations or that exist either negative significant associations or insignificant associations with all 5 the types of molecular QTs for significance call. DNA methylations with significant positive associations with QTs were filtered out for significance call, as it with QTs were filtered out for significance call, as it was not the pattern of biological associations that were interesting. Secondly, a significance was called only if the posterior probabilities > 75% of a predictor being associated with a molecular > 75% of a predictor being associated with a molecular QT, by summing over all configurations that are consistent with the association of interest. Lastly, we calculated empirical FDR via 100 permutations per molecular QTs by shuffling the label of the permutations per molecular QTs by shuffling the label of the molecular QTs, and requested empirical FDR (eFDR) <10% by selecting a minimal cutoff value of alpha such that 75%<alpha<100%. The eFDR is calculated by: eFDR= (Averaged No. of genes with The eFDR is calculated by: eFDR= (Averaged No. of genes with posterior probabilities > alpha in permuted data) / (Averaged No. of genes with posterior probabilities > alpha in original data). Figure 1G annotated those genes whose DNA methylation had Figure 1G annotated those genes whose DNA methylation had cascading effects (associated with all QTs under investigation), and among them those whose protein abundances significantly differ between tumor and NAT were annotated. EMT-specific cluster and fibroblast enrichment NMF derived EMT-enriched subtype was interrogated for EMT and fibroblast related proteins in LSCC tumor patients. The relative protein expression was calculated using z-score. The single ssGSEA was performed on a protein dataset using GSVA (v3.11) to ssGSEA was performed on a protein dataset using GSVA (v3.11) to calculate normalized enrichment score for EMT and fibroblast proliferation using MSigdb (v6.1) and gene ontology geneset respectively. Similar analysis was also performed to interrogate SOX2 Similar analysis was also performed to interrogate SOX2 associated genes and its relation with IL6_JAK_STAT signaling (MSigdb hallmarks geneset V6.1). Differential marker analysis A Wilcoxon signed rank test was performed on TMT-based global proteomic data between tumor and matched normal samples to determine differential abundance of proteins between tumor and NAT samples. Proteins having < 50% missing values were considered for Proteins having < 50% missing values were considered for downstream analysis. Proteins with log2-fold-change (FC)> 2 in tumors and Benjamini-Hochberg FDR < 0.01 were considered to be tumor-associated proteins. Over-representation analysis of Gene Ontology proteins. Over-representation analysis of Gene Ontology Biological Process terms was performed on a background of quantified proteins in WebGestaltR. The selected gene list was either significant proteins or proteins containing a significantly altered proteins or proteins containing a significantly altered phosphorylation site from the differential marker analyses. ORA was performed separately for increased and decreased proteins. Terms were considered significant with a Benjamini-Hochberg adjusted were considered significant with a Benjamini-Hochberg adjusted p-value <0.01. Among tumor associated proteins, the top 50 differential proteins with more than 90% of samples showing high expression in tumors were selected and shown. Differential proteins with more than 99% of samples showing tumor specific expression were more than 99% of samples showing tumor specific expression were highlighted. Immunohistochemistry-based antibody-specific staining scores in lung tumors were obtained from the Human Protein Atlas (HPA, https://www.proteinatlas.org), in which Protein Atlas (HPA, https://www.proteinatlas.org), in which tumor-specific staining is reported in four levels, i.e., high, medium, low, and not detected. The protein-specific annotations such as enzymes, transcription factor, transporters, secreted, such as enzymes, transcription factor, transporters, secreted, transmembrane and FDA-approved drugs targeting the protein or reported in drugbank were designated. Similarly, differential analysis was performed between RNA, phosphosite and acetylproteome was performed between RNA, phosphosite and acetylproteome datasets between tumor and matched NATs. The differential genes/sites with log2-fold-change (FC)> 2 in tumors and Benjamini-Hochberg FDR < 0.01 were considered to be tumor-associated. The FDR < 0.01 were considered to be tumor-associated. The overlapping genes/sites with proteins were represented in a checkerbox plot. Overall and disease-free survival in TCGA lung squamous carcinoma Survival for lung squamous tumor patients in TCGA LSCC cohort was calculated using the time from date of diagnosis to death or last contact (Overall survival, OS) and disease-free interval time (Disease-free survival, DFS). Normalized RNA expression was (Disease-free survival, DFS). Normalized RNA expression was categorized into low and high expression group based on mean. Univariate Cox proportional hazards models were fitted to OS, and DFI endpoints to calculate the hazard ratios for tumor-associated endpoints to calculate the hazard ratios for tumor-associated genes/proteins using coxph function in Survival package (version 2.44) in R. The p-values were determined using a log-rank test. The hazard ratio [exp(cox coefficient)] was used to compare poor hazard ratio [exp(cox coefficient)] was used to compare poor survival in low and high-expression group. Kaplan-Meier survival analysis We used time-to-death or last contact data for the current cohort (recorded as number of days from diagnosis date) to assess survival differences based on various sample subgroups and clinical annotations. We used Kaplan-Meier analysis to explore survival annotations. We used Kaplan-Meier analysis to explore survival differences associated with tumor grade, mutation burden, mutation status for SMGs, CUL3-NFE2L2-KEAP1 combined mutation status, CNA for KAT6A/SOX2/TP63/FGFR1/CDKN2A, immune subtype, NMF CNA for KAT6A/SOX2/TP63/FGFR1/CDKN2A, immune subtype, NMF cluster, CIN, tumor grade and ploidy. We identified NMF mixed tumors (i.e., tumors that are not part of the core NMF clusters) had significantly worse survival (log-rank p-value = 0.005). We used a Fisher test to find enriched annotations (among those mentioned in the previous paragraph, with (among those mentioned in the previous paragraph, with tumor stemness index added) associated with NMF mixed tumors. The frequency of SOX2 amplifications were significantly enriched in NMF mixed tumors (p-value = 0.0038). None of the other annotations had tumors (p-value = 0.0038). None of the other annotations had significant enrichment in the NMF mixed tumors. Continuous Smoking Score Non-negative matrix factorization (NMF) was used in deciphering mutation signatures in cancer somatic mutations stratified by 96 base substitutions in tri-nucleotide sequence contexts. To obtain a reliable signature profile, we used SomaticWrapper to call a reliable signature profile, we used SomaticWrapper to call mutations from WGS data. SignatureAnalyzer exploited the Bayesian variant of the NMF algorithm and enabled an inference for the optimal number of signatures from the data itself at a balance optimal number of signatures from the data itself at a balance between the data fidelity (likelihood) and the model complexity (regularization). After decomposing into three signatures, signatures were compared against known signatures derived from COSMIC were compared against known signatures derived from COSMIC and cosine similarity was calculated to identify the best match. We also sought to integrate count of total mutations, t, percentage that are signature mutations, c, and count of DNPs, n, into a continuous score, 0 < S < 1, to quantify the degree of confidence that a sample was associated with smoking signature. We that a sample was associated with smoking signature. We referred to these quantities as the data, namely D = C \\u2229 T \\u2229 N, and used A and A\\u2019 to indicate smoking signature or lack thereof, respectively. In a Bayesian framework, it is readily shown that a In a Bayesian framework, it is readily shown that a suitable form is S = 1 / (1 + R), where R is the ratio of the joint probability of A\\u2019 and D to the joint probability of A and D. For example, the latter can be written P(A)\\u30fbP(C|A)\\u30fbP(T|A)\\u30fbP(N|A) and the the latter can be written P(A)\\u30fbP(C|A)\\u30fbP(T|A)\\u30fbP(N|A) and the former similarly, where each term of the former is the complement of its respective term in this expression. Common risk statistics are invoked as priors, i.e. P(A) = 0.9. We consider S to be a score because rigorous conditioned probabilities are difficult to establish. (For example, the data types themselves are not entirely independent of one another and models using common distributions like the Poisson do not and models using common distributions like the Poisson do not recapitulate realistic variances.) Instead, we adopt a data-driven approach of estimating contributions of each data type based on 2-point empirical fitting of roughly the low and high extremes 2-point empirical fitting of roughly the low and high extremes using shape functions. The general model for data type G is P(G|A) = [x \\u30fberf (g/y) + 1] / (x + 2) and P(G|A\\u2019) = 1 - P(G|A), where erf is the Gaussian error function, g is the observed value erf is the Gaussian error function, g is the observed value for the given data type, and x and y are empirically-determined weights. The shape function for total mutations, T, has two modifiers. First, it includes a simple expected-value correction for purity, u. Namely, assuming mutation-calling does not capture all mutations because of impurities, t is taken as the observed number of because of impurities, t is taken as the observed number of mutations divided by a purity shape function, f, where f \\u2264 1. Although one might model f according to common characteristics of mutation callers, e.g. close to 100% sensitivity for pure samples callers, e.g. close to 100% sensitivity for pure samples and very low calling rate for low variant allele fractions (VAFs), the purity estimates for these data are based on RNA-Seq and are not highly correlated with total mutation counts. Consequently, we highly correlated with total mutation counts. Consequently, we use a line function, f = 0.2\\u30fbu + 0.8, which does not strongly impact the adjustment of low-purity samples. Second, before choosing weights, we removed outliers, as determined by Peirce\\u2019s weights, we removed outliers, as determined by Peirce\\u2019s criterion. Based on the distribution of continuous smoking score, we set 0.15 as the lower bound cutoff and 0.6 as the upper bound cutoff for defining the inferred smoking status. We classified the for defining the inferred smoking status. We classified the samples with strong smoking evidence if the smoking score > 0.6; samples with weak smoking evidence if smoking score < 0.15. Identification of differentially regulated events in NRF2 mutant tumors. For each gene, normalized levels of mRNA/protein/phosphoprotein (log transformed data) were fit into a linear regression model. In addition to NFR2 mutation status, gender, tumor purity and ethnicity were also included in the model. Coefficient and and ethnicity were also included in the model. Coefficient and p-value of NRF2 mutation status variables from the fitted model were used to evaluate the extent and significance of the difference between patients with and without aberration. To adjust for between patients with and without aberration. To adjust for multi-test, p-values were adjusted with the BH method. Genes that showed significant upregulation (up) in more than one dataset (Table S4) were used to generate a GMT file for ssGSEA based NRF2 S4) were used to generate a GMT file for ssGSEA based NRF2 activity score calculation using ssGSEA (https://github.com/broadinstitute/ssGSEA2.0). The NRF2 activity score contains a total of 54 genes upregulated both at protein and RNA level, identified by genes upregulated both at protein and RNA level, identified by our NRF2 mutant versus wildtype tumor differential analysis (p-value <0.05 and logFC>1). Among the 54 genes, 44 have been previously associated with NRF2 signaling (either in NRF2 concept in associated with NRF2 signaling (either in NRF2 concept in msigdb such as Gene Sets NFE2L2.V2 and SINGH_NFE2L2_TARGETS (, BIOCARTA_ARENRF2_PATHWAY], NRF2 wikipathway or Lacher et al. Ten of the 54 genes were novel targets identified in this study. Immune cluster identification based on cell type composition The abundance of 64 different cell types were computed via xCell based on transcriptomic profiles based on 108 tumors and 94 NAT samples. Table S6 contains the final score computed by xCell of different cell types. Consensus clustering was performed based different cell types. Consensus clustering was performed based on immune cells, fibroblasts, endothelial and epithelial cells from xCell. In particular, we only considered cells that were detected in at least 10% of the patients (FDR < 5%). This filtering in at least 10% of the patients (FDR < 5%). This filtering resulted in 25 cell types annotated in Table S6. Based on these signatures, consensus clustering was performed in order to identify groups of samples with similar immune/stromal characteristics. groups of samples with similar immune/stromal characteristics. Consensus clustering was performed using the R packages ConsensusClusterPlus based on z-score normalized signatures. Specifically, signatures were partitioned into three major clusters using signatures were partitioned into three major clusters using the Partitioning Around Medoids (PAM) algorithm, which was repeated 1000 times (Figure 6A, Table S6). TCGA pan-cancer immune subtyping Tumors were classified into the six distinct immune subtypes \\u2014 wound healing, IFN-\\u03b3 dominant, inflammatory, lymphocyte depleted, immunologically quiet, and TGF-\\u03b2 dominant \\u2014 identified by a TCGA pan-cancer analysis presented in. Gene expression data (log2 pan-cancer analysis presented in. Gene expression data (log2 FPKM; lscc-v3.0-rnaseq-uq-fpkm-log2-NArm) was input to the ImmuneSubtypeClassifier R package (https://github.com/CRI-iAtlas/ImmuneSubtypeClassifier) to assign an immune subtype to each of the to assign an immune subtype to each of the tumors. Ranking tumors by inferred activity of IFN-\\u03b3 pathway We assumed that true biological activity of a pathway is regulated by collective changes of expression levels of majority of proteins involved in this pathway. Then, a difference of a pathway activity between tumors can be assessed by a difference in activity between tumors can be assessed by a difference in positioning of expression levels of proteins involved in this pathway in ranked list of expression levels of all proteins in each of tumors. Following this idea, we assessed relative positioning tumors. Following this idea, we assessed relative positioning of pathway proteins between tumor by determining two probabilities: a probability of pathway proteins to occupy by random the observed positions in a list of tumor proteins ranked by expression positions in a list of tumor proteins ranked by expression levels from the top to the bottom and, similarly, a probability to occupy by random the observed positions in a list of expression levels ranked from the bottom to the top. Then, the inferred levels ranked from the bottom to the top. Then, the inferred relative activation of a given pathway across tumors was assessed as a negative logarithm of the ratio of the above \\u201ctop\\u201d and \\u201cbottom\\u201d probabilities. Thus, for a pathway of a single protein, its probabilities. Thus, for a pathway of a single protein, its relative activity across tumors was assessed as a negative log of ratio of two numbers: a number of proteins with expression level bigger than an expression level of given protein, and a number bigger than an expression level of given protein, and a number of proteins with expression levels less than an expression level of given protein. For pathways of multiple proteins, the \\u201ctop\\u201d and \\u201cbottom\\u201d probabilities were computed as geometrically \\u201ctop\\u201d and \\u201cbottom\\u201d probabilities were computed as geometrically averaged P values computed for each of proteins using Fisher\\u2019s exact test, given protein\\u2019s ranks in a list of pathway proteins and in a list of ranked proteins of a tumor, a number of and in a list of ranked proteins of a tumor, a number of proteins in a pathway, and the total number of proteins with the assessed expression level in a given tumor. The thermodynamic interpretation of the inferred pathway activity scoring function is a of the inferred pathway activity scoring function is a free energy associated with deviation of the system from equilibrium either as a result of activation or suppression. Thus, the scoring function is positive, when expression levels of pathway\\u2019s function is positive, when expression levels of pathway\\u2019s proteins are overrepresented among top expressed proteins of a tumor, and it is negative, when pathway\\u2019s proteins are at the bottom of expressed proteins of a tumor; the scoring function is close of expressed proteins of a tumor; the scoring function is close to zero, when expression levels are distributed by random or equally shifted towards top or bottom. In our analysis, we used a gene list of IFN-\\u03b3 signaling pathway from. We settled with this 15 genes signaling pathway because it has all necessary components \\u2013 a cytokine mediator, receptors, signal transduction, transcription factors and final effectors signal transduction, transcription factors and final effectors (PD-L1/2, MHC-I), and all those genes are expressed in cancer cells. Estimation of Tumor Purity, Stromal and Immune Scores Besides xCell, we utilized ESTIMATE to infer immune and stromal scores based on gene expression data (Table S6). Cibersort absolute immune scores were obtained by evaluating upper-quartile normalized RNA-seq FPKM data with the Cibersort web app normalized RNA-seq FPKM data with the Cibersort web app (cibersort.stanford.edu) in absolute mode. To infer tumor purity, TSNet was utilized (Table S6). Differentially Expressed Genes and Pathway Analysis Genes upregulated in Hot and Warm clusters compared to the Cold cluster were identified based on gene expression data, global proteomic, phosphoproteomic and acetylation data. For this analysis, markers with at most 50% missing values were utilized. For markers with at most 50% missing values were utilized. For each data type, the expression level of gene/protein/sites was modeled as a function of immune cluster via linear regression. P-values were then adjusted for multiple comparisons via P-values were then adjusted for multiple comparisons via Benjamini-Hochberg adjustment. For each immune cluster, considering the set of genes up(down)-regulated with Benjamini-Hochberg adjusted p-value lower than 10%, a fisher exact test was implemented p-value lower than 10%, a fisher exact test was implemented to derive enriched pathways. ssGSEA was utilized to obtain pathway scores based on RNA-seq and global proteomics data using the R package GSVA. For this analysis, pathways from the Reactome, KEGG GSVA. For this analysis, pathways from the Reactome, KEGG and Hallmark databases were considered and as background the full list of gene/proteins observed under each data type was utilized. For phosphorylation and acetylproteome data, a gene was For phosphorylation and acetylproteome data, a gene was considered upregulated if at least one substrate of the gene was found upregulated at 10% FDR. The pathway analysis results for different data types are contained in Table S6. Deriving RTK CBPE scores A total of 42 human RTKs were present in our proteomics data set. For each phosphosite in our data set we computed a linear association with each of the RTKs in two ways. A phosphosite profile across samples (for tumor and NAT separately) was modeled as a across samples (for tumor and NAT separately) was modeled as a linear combination of 1) an RTK protein profile and a profile of a protein that a given phosphosite belonged to or 2) an RTK protein profile, a profile of a protein that a given phosphosite profile, a profile of a protein that a given phosphosite belonged to and a profile of tumor purity estimates obtained from RNAseq data via TSNet. Each association computation was carried out only if at least 50% of samples had non-missing data of all out only if at least 50% of samples had non-missing data of all three of RTK protein, phosphosite, and protein in question. The phosphosite:RTK associations were considered significant if their FDR adjusted p-value was below 0.1. Additionally, for each FDR adjusted p-value was below 0.1. Additionally, for each phosphosite the z-scores of their log2 phosphoprotein abundance were computed across samples (separately for tumor and NAT). A raw RTK CBPE score was computed for each RTK in each sample as a RTK CBPE score was computed for each RTK in each sample as a number of all phosphosites with significant association with this RTK that also had a z-score higher than 1. These raw RTK CBPE scores in each sample were normalized across RTKs so the final RTK in each sample were normalized across RTKs so the final RTK CBPE scores of all RTKs in one sample would add up to 1. We selected the most relevant subset of RTKs by requiring that each had a normalized CBPE score of at least 0.1 in at least 5 samples. a normalized CBPE score of at least 0.1 in at least 5 samples. Nine RTKs satisfied this condition in the tumor data set, and seven in NAT. Independent component analysis ICA was performed with a workflow modified from previously described. Decomposition was run for 100 times on the matrix of protein abundance difference between tumor/NAT pairs (n=99). Independent components were in the form of vectors comprised with components were in the form of vectors comprised with weights of all genes in the original data. Components extracted from each run were clustered using HDBSCAN algorithm with cosine distance as dissimilarity metric, min_cluster_size=50 and distance as dissimilarity metric, min_cluster_size=50 and min_samples=20. The centroids of clusters (n=37) were considered as representative of stable signatures, and mean mixing scores (activity of each signature over all samples) of each cluster were of each signature over all samples) of each cluster were used to represent the activity levels of corresponding signature in each sample. Correlation between the extracted signatures and known clinical characteristics were examined by regressing the known clinical characteristics were examined by regressing the corresponding mixing scores for all members of a component cluster against 64 sample annotations to obtain within-cluster average of log10 P-values. Significance was controlled for multiple of log10 P-values. Significance was controlled for multiple testing at 0.01 level (log10 (p-value)=\\u22125.3). Each signature vector (cluster centroid) was submitted to GSEA pre-ranked test for functional annotations. Mutation-based cis- and trans-effects We examined the cis- and trans-effects of 22 genes with somatic mutations that were significant in a previous large-scale TCGA LSCC study on the RNA, proteome, and phosphoproteome of known interactome DBs including Omnipath, Phosphositeplus, DEPOD, interactome DBs including Omnipath, Phosphositeplus, DEPOD, Signor, and CORUM. After excluding silent mutations, samples were separated into mutated and WT groups. We used the Wilcoxon rank-sum test to report differentially expressed features (RNA, rank-sum test to report differentially expressed features (RNA, proteins, phosphosites, acetylsites, and ubiquitylation sites) between the two groups. Differentially enriched features passing an FDR <0.05 cut-off were separated into two categories based an FDR <0.05 cut-off were separated into two categories based on cis- and trans- effects. Germline quantitative trait loci (QTL) analysis To identify germline genetic variants that explain variation in tumor gene (eQTL) and protein (pQTL) expression, we utilized the gold-standard mapping pipeline at https://github.com/molgenis/systemsgenetics/wiki/eQTL-mapping-analysis-cookbook-(eQTLGen). Briefly, we followed the default steps that include correcting for population stratification removing outliers, normalizing and mapping cis-QTLs. For cis-eQTL analysis input, we used i) germline genotype data; and ii) RNA-seq raw read count data for 105 genotype data; and ii) RNA-seq raw read count data for 105 samples, and performed analyses only for genes with sum of read count \\u2265 10. For cis-pQTL analysis input, we used i) germline genotype data; and ii) normalized and filtered protein TMT ratio data; and ii) normalized and filtered protein TMT ratio aggregated to gene level for 108 samples. The genotype data were harmonized using Genotype Harmonizer v1.4.9. To adjust for population stratification, we identified the multi dimensional scaling stratification, we identified the multi dimensional scaling (MDS) components using the genotype data using PLINK v1.07. For both types of cis-QTL mapping, we used eqtl-mapping-pipeline-v1.3.9 using its standard settings (HWE > 0.0001, MAF > 0.01, and call using its standard settings (HWE > 0.0001, MAF > 0.01, and call rate > 0.95, maximum distance between the SNP and the middle of the probe is 250,000 bp). To control for multiple testing, we performed 10 permutations. We deemed QTLs at \\u2264 FDR 5% as we performed 10 permutations. We deemed QTLs at \\u2264 FDR 5% as significant. Finally, we performed pathway analysis of the identified significant eGenes (genes whose expression is impacted by at least one cis-QTL) using QIAGEN Ingenuity Pathway Analysis miRNA analysis presented in Figure 4 Targets of miRNAs were downloaded from the miRNA targets database miRTarBase and only the miRNA/target pairs with strong experimental evidence were retained. Spearman correlation was used to calculate the correlations between miRNA and its target genes. Pathway projection using ssGSEA The single sample Gene Set Enrichment Analysis (ssGSEA) implementation available on https://github.com/broadinstitute/ssGSEA2.0 was used to project log2(FPKM) mRNA abundances to MSigDB cancer hallmark gene sets using the following parameters: gene.set.database=\\u201ch.all.v6.2.symbols.gmt\\u201d\\n\\nsample.norm.type=\\u201crank\\u201d\\n\\nweight=0.75\\n\\nstatistic=\\u201darea.under.RES\\u201d\\n\\noutput.score.type=\\u201cNES\\u201d\\n\\nnperm=1000\\n\\nglobal.fdr=TRUE\\n\\nmin.overlap=10\\n\\ncorrel.type=\\u201dz.score\\u201d\\n\\nPhosphorylation-driven signature analysis We performed phosphosite-specific signature enrichment analysis (PTM-SEA) to identify dysregulated phosphorylation-driven pathways. To adequately account for both magnitude and variance of measured phosphosite abundance, we used p-values derived from measured phosphosite abundance, we used p-values derived from application of the Wilcoxon rank-sum test to phosphorylation data as ranking for PTM-SEA. To that end, p-values were log-transformed and signed according to the fold change (signed \\u2212log10 and signed according to the fold change (signed \\u2212log10 (p-value)) such that large positive values indicated phosphosite abundance in classical or NFE2L2, CUL3, KEAP1 mutated samples, and large negative values indicated phosphosite abundance in samples negative values indicated phosphosite abundance in samples that do not belong to classical NMF subtype or wild-type samples (in Figure 3I)  PTM-SEA relies on site-specific annotation provided by PTMsigDB and thus a single site-centric data matrix data is by PTMsigDB and thus a single site-centric data matrix data is required such that each row corresponds to a single phosphosite. We note that in this analysis the data matrix comprised a single data column and each row represented a confidently localized data column and each row represented a confidently localized phosphosite assigned by Spectrum Mill software. We employed the heuristic method introduced by Krug et al. to deconvolute multiple phosphorylated peptides to separate data points (log-transformed and signed p-values). Briefly, phosphosites measured on different phospho-proteoform peptides were resolved measured on different phospho-proteoform peptides were resolved by using the p-value derived from the least modified version of the peptide. For instance, if a site T4 measured on a doubly phosphorylated (T4, S8) peptide (PEPtIDEsR) was also measured on a (T4, S8) peptide (PEPtIDEsR) was also measured on a mono-phosphorylated version (PEPtIDESR), we assigned the p-value derived from the mono-phosphorylated peptide proteoform to T4, and the p-value derived from PEPtIDEsR to S8. If only the doubly the p-value derived from PEPtIDEsR to S8. If only the doubly phosphorylated proteoform was present in the dataset, we assigned the same p-value to both sites T4 and S8. We queried the PTM signatures database (PTMsigDB) v1.9.0 downloaded from https://github.com/broadinstitute/ssGSEA2.0/tree/master/db/ptmsigdb using the flanking amino acid sequence (+/\\u2212 7 aa) as primary identifier. We used the implementation of PTM-SEA as primary identifier. We used the implementation of PTM-SEA available on GitHub (https://github.com/broadinstitute/ssGSEA2.0) using the command interface R-script (ssgsea-cli.R). The following parameters were used to run PTM-SEA: weight: 1\\n\\nstatistic: \\u201carea.under.RES\\u201d\\n\\noutput.score.type: \\u201cNES\\u201d\\n\\nnperm: 1000\\n\\nmin.overlap: 5\\n\\ncorrel.type: \\u201cZ-score\\u201d The sign of the normalized enrichment score (NES) calculated for each signature corresponds to the sign of the tumor-NAT log fold change. P-values for each signature were derived from 1,000 random permutations and further adjusted for multiple hypothesis permutations and further adjusted for multiple hypothesis testing using the method proposed by Benjamini & and Hochberg. Signatures with FDR-corrected p-values < 0.05 were considered to be differential between tumor and NAT. CDKN2A and RB1 annotations and pathway analysis Comprehensive tumor annotation for CDKN2A and RB1 genomic status was carried out using multiple molecular features for each patient. Specifically, we considered the following: 1) mutation types (missense/in-frame indels mutations, nonsense (stop gain, types (missense/in-frame indels mutations, nonsense (stop gain, frameshift indels) mutations, and splice site (splice donor, splice acceptor) mutations) as separate categories and 2) copy number data. For cases with mutation, we considered both the number data. For cases with mutation, we considered both the variant allele frequency of the mutation from whole exome sequencing and copy number data (log ratio, absolute copy number, and B-allele frequency). Only mutations that occurred in cases with B-allele frequency). Only mutations that occurred in cases with loss of the wild-type allele were annotated as loss of heterozygosity (LOH) mutations and used for subsequent analysis. 2) Based on CDKN2A copy number data, we next annotated tumors without on CDKN2A copy number data, we next annotated tumors without mutations as homozygous deletions or no loss (WT). Samples were re-classified as having p16INK4a promoter hypermethylation if three p16INK4a promoter-associated methylation probes (cg11075751, p16INK4a promoter-associated methylation probes (cg11075751, cg02008397, and cg01694391) all had beta values >= 0.2. Multi-gene proliferation scores were calculated as described previously as the means of the gene normalized RNA levels for cell cycle as the means of the gene normalized RNA levels for cell cycle associated genes characterized by Whitfield et al. p16INK4a RNA levels were obtained from the isoform specific RSEM data for Ensembl transcript ENST00000304494.8, while ENST00000579755.1 was transcript ENST00000304494.8, while ENST00000579755.1 was used for p14ARF, and log2 (RSEM + 1) values were used for barplots. Isoform specific protein data included Refseq NP_000068.1 for p16INK4a and Refseq NP_478102.2 for p14ARF. Calculation of CDK1, 2, and Refseq NP_478102.2 for p14ARF. Calculation of CDK1, 2, and 4 target site scores from the phosphorylation site data and Hallmark E2F targets and G2M scores from the RNA-seq data is described above (Phosphorylation-driven signature analysis and Pathway above (Phosphorylation-driven signature analysis and Pathway projection using ssGSEA, respectively). Lolliplots for CDKN2A mutations in the CPTAC head and neck squamous cell carcinoma and this lung squamous cell carcinoma cohort were generated using the lung squamous cell carcinoma cohort were generated using the ProteinPaint web application to visualize mutations. Mutations annotated as nonsense mediated decay (NMD) mutations were those that resulted in low expression of p16INK4a RNA in the tumors that resulted in low expression of p16INK4a RNA in the tumors harboring the mutations. Association analysis between KGG-site abundances and E3 ligases and DUBs A list of known human E3 ubiquitin and ubiquitin-like ligases and DUBs was compiled from. We then fit a linear model using limma in R with the formula kgg_site_abundance ~ protein_abundance, followed by empirical bayes shrinkage. The coefficient p-values followed by empirical bayes shrinkage. The coefficient p-values were used to determine significant associations after FDR correction (FDR < 0.01). Cluster and pathway analysis of significantly modulated K-GG sites in tumors. Consensus clustering of the K-GG dataset after protein abundance correction was performed on tumor samples, resulting in 3 clusters. Next, marker selection for each cluster was performed by moderated t-test between samples that belong to the clusters by moderated t-test between samples that belong to the clusters against the remaining samples. These K-GG sites showing differential abundance across sample clusters were further clustered into 3 site-wise clusters. Genes in these site-wise clusters were used for pathway enrichment analysis against the KEGG, Reactome, and WikiPathways databases using g:profiler. Pathway enrichment was performed using a gene background containing all observable genes in the K-GG a gene background containing all observable genes in the K-GG dataset. Pathway enrichment results were imported into Cytoscape using the Enrichment Map app for network analysis of pathways. Pathways were connected using the gene overlap and clustered Pathways were connected using the gene overlap and clustered (pathway cutoff q-val < 0.1; jaccard overlap > 0.375). Each cluster was manually annotated from the pathways contained in it to facilitate interpretation. PTM CLUMPS analysis We employ two methods to select tumor-specific sites to include in structural analysis. First, we take PTM-sites for solely tumor-derived samples and binarize modifications by negative vs. positive normalized signal. These are considered as individual positive normalized signal. These are considered as individual \\u201cevents\\u201d as done with mutations in CLUMPS v1. For a more robust approximation of tumor-specific acetylation or ubiquitination, we perform differential expression using Limma between NAT and we perform differential expression using Limma between NAT and Tumor samples. Sites are selected with an FDR < 0.1 (Benjamini-Hochberg) and LFC > 0.1. We then binarize these tumor sites as done in the first approach. We first map the PTM-sites to as done in the first approach. We first map the PTM-sites to corresponding UNIPROT ids (ID) with available PDBs. For each crystal structure, we compute an initial WAP score and randomly sample sites as done in CLUMPS. We generate an empirical p-value sites as done in CLUMPS. We generate an empirical p-value based on a random sampling of lysines in each crystal structure to limit the selection to residues capable of ubiquitination or acetylation. The null hypothesis we define is each random sampling of The null hypothesis we define is each random sampling of lysines will have a WAP score less than or equal to the initially computed WAP score. We run 1e6 permutations to generate an empirical p-value. DepMap genetic dependency and drug response analysis\\n\\nCell lines annotated as \\u201cNSCLC_squamous\\u201d from DepMap were considered as LSCC for which molecular profiles, dependency scores, and drug response were used. To determine copy number status in LSCC cell lines, copy number segmentation files for LSCC cell lines were processed with GISTIC2.0 (using the same parameters used to process this study\\u2019s LSCC cohort as described in STAR METHODS). Samples were considered cohort as described in STAR METHODS). Samples were considered to be amplified for a given gene if GISTIC thresholded values = 1 or 2 and not amplified if GISTIC thresholded values = \\u22122, \\u22121, or 0. Mutation status for CCND1, CDKN2A, and RB1, Np63 transcript 0. Mutation status for CCND1, CDKN2A, and RB1, Np63 transcript isoform expression, phospho-Rb-Ser807/811 RPPA abundance, and drug responses to a survivin inhibitor (YM-155) and CDK4/6i inhibitors (abemaciclib and palbociclib) were also retrieved from (abemaciclib and palbociclib) were also retrieved from DepMap. Figure 7: For identifying if top protein biomarkers (502 proteins significantly overexpressed (log2(FC) >2, FDR <0.01) in tumors relative to their matched NATs, most with coherent overexpression in multi-omic analysis) also conferred altered dependencies in multi-omic analysis) also conferred altered dependencies in LSCC cell lines, we leveraged DepMap genetic dependency dataset (CRISPR Avana Public 20Q3) that contained 18119 genes and 789 cell lines (https://depmap.org/portal/download/ file: and 789 cell lines (https://depmap.org/portal/download/ file: Achilles_gene_effect.csv). Only 16 cell lines were classified as LSCC (Cell Line Sample Info.csv). Median dependencies were calculated and for every gene and density plot in Figure 7C shows and for every gene and density plot in Figure 7C shows dependency score of the 502 genes corresponding to 502 protein biomarkers relative to other genes. Figure S4B: To investigate copy number and dependency associations in LSCC cell lines, each gene up-regulated at the protein level with recurrent copy number gain in tumors vs NATs (from Figure 4A with Log2FC>0 & FDR < 0.05) was used as a query to compare 4A with Log2FC>0 & FDR < 0.05) was used as a query to compare dependency scores of the given gene (Combined CRISPR KO screen, DepMapPublic 21Q1) in LSCC cell lines where the query gene was amplified vs not-amplified. A lower mean change in dependency amplified vs not-amplified. A lower mean change in dependency scores for a given gene in amplified vs not-amplified samples indicates LSCC cell lines are more dependent upon that gene when it is amplified. Continuous log2 copy number data from DepMap was used in Figure S4M to correlate SOX2 copy number with EZH2 shRNA dependency data (Combined shRNA screen from DEMETER2 Data v6 in DepMap). Drug responses to YM-155 (PRISM secondary screen) were compared by t-test in LSCC cell lines with low vs high levels of Np63 by transcript isoform expression (Figure S4G). Pearson correlation was also performed with phospho-Rb-Ser807/811 RPPA abundance was also performed with phospho-Rb-Ser807/811 RPPA abundance and drug responses to CDK4/6i inhibitors, abemaciclib (PRISM secondary screen) and palbociclib (Sanger GDSC1) (Figure S3D). CausalPath analysis CausalPath searches for known biological mechanisms that can explain correlated proteomic changes in terms of causal hypotheses. We set CausalPath parameters to compare tumors and NATs with a paired t-test, used 0.1 as FDR threshold for proteomic change a paired t-test, used 0.1 as FDR threshold for proteomic change significance and network significance, and detected 5917 potential causal relations between proteins. We repeated the same analysis for each NMF subtype separately and identified 4378 analysis for each NMF subtype separately and identified 4378 (basal-inclusive), 5334 (classical), 3048 (EMT-enriched), 3744 (inflamed-secretory), and 4332 (proliferative-primitive) relations. We used these CausalPath network results in the preparation of We used these CausalPath network results in the preparation of Figure 7C, identifying potential upstream regulators of oncogenic phosphoproteomic changes. Here an oncogenic phosphoproteomic change can be any of the following 4 events: increase of change can be any of the following 4 events: increase of activating phosphorylation of an oncoprotein, decrease of inactivating phosphorylation of an oncoprotein, decrease of activating phosphorylation of a tumor suppressor protein, and increase of phosphorylation of a tumor suppressor protein, and increase of inactivating phosphorylation of a tumor suppressor protein. We used the OncoKB database for oncoprotein and tumor suppressor classification (excluded proteins that have both annotations), and (excluded proteins that have both annotations), and used PhosphoSitePlus and Signor databases for the activating/inhibiting classification of phosphorylation sites. In the phosphorylation regulation networks, we included only the targetable regulators regulation networks, we included only the targetable regulators (activated proteins) and excluded the untargetable regulators (inactivated proteins). Variant Peptide Identification We used NeoFlow (https://github.com/bzhanglab/neoflow) for neoantigen prediction. Specifically, Optitype was used to find human leukocyte antigens (HLA) for each sample based on WES data. Then we used netMHCpan to predict HLA peptide binding affinity for we used netMHCpan to predict HLA peptide binding affinity for somatic mutation\\u2013derived variant peptides with a length between 8\\u201311 amino acids. The cutoff of IC50 binding affinity was set to 150 nM. HLA peptides with binding affinity higher than 150 nM 150 nM. HLA peptides with binding affinity higher than 150 nM were removed. Variant identification was also performed at both mRNA and protein levels using RNA-Seq data and MS/MS data, respectively. Variant identification and gene quantification using Variant identification and gene quantification using RNA-Seq data were performed following the methods used in the previous study. To identify variant peptides, we used a customized protein sequence database approach. We built a customized database for sequence database approach. We built a customized database for each TMT experiment based on somatic variants from WES data. We used Customprodbj (https://github.com/bzhanglab/customprodbj) for customized database construction. MS-GF+ was used for variant customized database construction. MS-GF+ was used for variant peptide identification for all global proteome and phosphorylation data. Results from MS-GF+ were filtered with 1% FDR at peptide level. Remaining variant peptides were further filtered using level. Remaining variant peptides were further filtered using PepQuery (http://www.pepquery.org) with the p-value cutoff <= 0.01. Competitive filtering based on unrestricted posttranslational modification searching was enabled in PepQuery validation. The modification searching was enabled in PepQuery validation. The spectra of variant peptides were annotated using PDV (http://www.zhang-lab.org/). Cancer/testis Antigen Prediction\\n\\nCancer/testis (CT) antigens were downloaded from the CTdatabase. CT antigens with a median >4-fold increase in tumor from NAT were highlighted.\\n\\nPROGENy Scores PROGENy Scores\\n\\nPROGENy was used to generate activity scores for EGFR based on RNA expression data. Tumor RNA expression values were submitted to PROGENy.\\n\\nLSCC, HNSCC and LUAD integrative analysis LUAD data for 110 lung adenocarcinoma samples and 102 NAT were acquired from the published manuscript. HNSCC data for 108 head and neck squamous cell carcinoma samples and 66 NAT were acquired from the submitted manuscript. The LUAD and HNSCC proteomics from the submitted manuscript. The LUAD and HNSCC proteomics data were processed, quantified, and normalized using the same pipeline as described for the LSCC samples. Differential expression analysis for all three cohorts was performed as described in analysis for all three cohorts was performed as described in the Differential marker analysis method. Proteins missing in more than 50% of the paired tumor/NAT samples were excluded. Copy number drivers for all three were assessed by filtering the 8309 genes that were quantified in all three cohorts in the copy number, RNA, and proteomic data to those found in focal amplification regions (q value < .25). Spearman correlation was amplification regions (q value < .25). Spearman correlation was performed for these genes between CNA and RNA and between CNA and protein. Proteins were considered drivers if the correlation between both the CNA and RNA and CNA and protein were between both the CNA and RNA and CNA and protein were significantly positively associated (BH adjusted p value <0.01). Proteins were also required to be significantly increased in tumor vs paired NAT in the same cohort (Wilcoxon signed rank BH adjusted p NAT in the same cohort (Wilcoxon signed rank BH adjusted p value <0.01). To identify genes associated with the immune score, correlation between CNA, RNA, protein, and the immune score was performed for the genes present in both CNA and RNA in a cohort (20,313 genes in LSCC, 18,091 genes in LUAD, and 21,616 genes in HNSCC). in LSCC, 18,091 genes in LUAD, and 21,616 genes in HNSCC). The immune score was calculated as the z-score transformation of the ESTIMATE Immune score, which was calculated for all three cohorts as described in the method Estimation of Tumor Purity, cohorts as described in the method Estimation of Tumor Purity, Stromal and Immune Scores. To be considered copy number drivers of the immune score, genes had to have a positive correlation between CNA & RNA, CNA & immune score, and RNA & immune score. If CNA & RNA, CNA & immune score, and RNA & immune score. If the protein was quantified in the cohort, the gene was also required to have a positive correlation between CNA & protein and protein & immune score. Spearman correlations with a BH adjusted p & immune score. Spearman correlations with a BH adjusted p value < 0.01 were considered significant. Supplementary Material\\n\\nDeclaration of Interests\\n\\nThe authors declare no competing interests\\n\\nADDITIONAL RESOURCES ADDITIONAL RESOURCES\\n\\nThe CPTAC program website, detailing program initiatives, investigators, and datasets, is found at https://proteomics.cancer.gov/programs/cptac. A website for interactive visualization of the multi-omic dataset is available at: https://rstudio-connect.broadapps.org/CPTAC-LSCC2021/ All processed data matrices are available at LinkedOmics (http://www.linkedomics.org), where computational tools are available for further exploration of this dataset.\\n\\nConsortia Alex Green, Alexey Nesvizhskii, Alfredo Molinolo, Alicia Francis, Amanda Paulovich, Ana Robles, Andrii Karnuta, Antonio Colaprico, Arul Chinnaiyan, Azra Krek, Barbara Hindenach, Barbara Pruetz, Bartosz Kubisa, Bing Zhang, Bo Wen, Boris Reva, Brian Druker, Bartosz Kubisa, Bing Zhang, Bo Wen, Boris Reva, Brian Druker, Carissa Huynh, Chandan Kumar-Sinha, Charles Goldthwaite Jr., Chelsea Newton, Chen Huang, Chet Birger, Chia-Kuei Mo, Christopher Kinsinger, Corbin Jones, DR Mani, Dan Rohrer, Dana Valley, Daniel Corbin Jones, DR Mani, Dan Rohrer, Dana Valley, Daniel Chan, David Chesla, David Feny\\u00f6, David Heiman, Dmitry Rykunov, Donna Hansel, Elena Ponomareva, Elizabeth Duffy, Eric Burks, Eric Jaehnig, Eric Schadt, Erik Bergstrom, Eugene Fedorov, Eunkyung An, Fei Eric Schadt, Erik Bergstrom, Eugene Fedorov, Eunkyung An, Fei Ding, Felipe da Veiga Levoprost, Fernanda Martins Rodrigues, Francesca Petralia, Gad Getz, Galen Hostetter, George Wilson, Gilbert Omenn, Henry Rodriguez, Houxiang Zhu, Hui Zhang, Jackson Omenn, Henry Rodriguez, Houxiang Zhu, Hui Zhang, Jackson White, James Suh, Jennifer Eschbacher, Jennifer Maas, Jonathan Lei, Jiayi Ji, Kai Li, Karen Christianson, Karen Ketchum, Karin Rodland, Karl Clauser, Karsten Krug, Katherine Hoadley, Kei Suzuki, Karl Clauser, Karsten Krug, Katherine Hoadley, Kei Suzuki, Kelly Ruggles, Ki Sung Um, Li Ding, Liqun Qi, Lori Bernard, Harry Kane, Maciej Wiznerowicz, MacIntosh Cornwell, Ma\\u0142gorzata Wojty\\u015b, Marcin Cieslik, Marcin Domagalski, Mathangi Thiagarajan, Matthew Cieslik, Marcin Domagalski, Mathangi Thiagarajan, Matthew Ellis, Matthew Wyczalkowski, Maureen Dyer, Meenskshi Anurag, Mehdi Mesri, Melissa Borucki, Michael Birrer, Michael Gillette, Midie Xu, Mikhail Krotevich, Myvizhi Esai Selvan, Namrata Udeshi, Nancy Mikhail Krotevich, Myvizhi Esai Selvan, Namrata Udeshi, Nancy Roche, Nathan Edwards, Negin Vatanian, Neil Mucci, Nicollette Maunganidze, Nikolay Gabrovski, Olga Potapova, Oluwole Fadare, \\u00d6zg\\u00fcn Babur, Pamela Grady, Pankaj Vats, Paul Paik, Pei Wang, Peter Babur, Pamela Grady, Pankaj Vats, Paul Paik, Pei Wang, Peter McGarvey, Pierre Jean Beltran, Pushpa Hariharan, Qing Kay Li, Ramaswamy Govindan, Ratna Thangudu, Rebecca Montgomery, Richard Smith,Robert Welsh, Runyu Hong, Sailaja Mareedu, Samuel Payne, Smith,Robert Welsh, Runyu Hong, Sailaja Mareedu, Samuel Payne, Sandra Cottingham, Sara Savage, Saravana Dhanasekaran, Scott Jewell, Sendurai Mani, Seungyeul Yoo, Shankara Anand, Shankha Satpathy, Shayan Avanessian, Shilpi Singh, Shirley Tsang, Shuang Cai, Shayan Avanessian, Shilpi Singh, Shirley Tsang, Shuang Cai, Song Cao, Stacey Gabriel, Stephan Schurer, Steven Carr, Suhas Vasaikar, Tao Liu, Tara Hiltke, Tatiana Omelchencko, Thomas Bauer, Tobias Schraink, Vasileios Stathias, Volodymyr Sovenko, Warren Tobias Schraink, Vasileios Stathias, Volodymyr Sovenko, Warren Tourtellotte, Weiping Ma, Wenke Liu, William Bocik, Wohaib Hasan, Xiaoyu Song, Yifat Geffen, Yize Li, Yongchao Dou, Yuping Zhang, Yuxing Liao, Yvonne Shutack, Zeynep Gumus, Zhen Zhang, Ziad Yuxing Liao, Yvonne Shutack, Zeynep Gumus, Zhen Zhang, Ziad Hanhan. This is a PDF file of an article that has undergone enhancements after acceptance, such as the addition of a cover page and metadata, and formatting for readability, but it is not yet the definitive version of record. This version will undergo additional version of record. This version will undergo additional copyediting, typesetting and review before it is published in its final form, but we are providing this version to give early visibility of the article. Please note that, during the production of the article. Please note that, during the production process, errors may be discovered which could affect the content, and all legal disclaimers that apply to the journal pertain. Proteogenomic Landscape of LSCC A. Schematic showing the number of tumors and NATs profiled and the various data types generated in this study. The lower panel represents data completeness. WGS: Whole Genome Sequencing, WES: Whole Exome Sequencing. CNA: copy number alteration. DNAme: Whole Exome Sequencing. CNA: copy number alteration. DNAme: DNA methylation. pSTY: phosphoproteome. Ac: acetylproteome. Ub: ubiquitylproteome. B. Stacked histograms indicating the distribution of patient phenotypes. Smoking History reflects self-report. C. Co-occuring mutation plot indicating cancer-relevant genes. MutSig-based significantly mutated genes (SMGs, q-value < 0.1) in this dataset are highlighted in red font. D. Heatmaps showing correlation between copy number alterations (CNA) and RNA (left) or proteomics (right). Red and green events represent significant (FDR <0.01) positive and negative correlations, respectively. E. Flow chart for identification of cancer-associated genes (CAGs) that showed GISTIC-based focal amplification or deletion (q<0.25) and cis-effects in both mRNA and protein (FDR<0.05). F. Differential protein expression (Log2 fold-change (FC)) in tumors with and without high-level amplification of the FGFR1 gene (GISTIC thresholded value =2). G. Genes whose DNA methylation was significantly associated with cascading cis regulation of their cognate mRNA expression, protein level, phosphopeptide and acetylpeptide abundance. Shapes indicate the cis-effects across the indicated datasets. Named indicate the cis-effects across the indicated datasets. Named genes also showed differential expression between tumors and NATs. See also Figure S1 and Table S1\\u20133\\n\\nLSCC Molecular Subtypes and Associations\\n\\nA. NMF-based clustering of tumor CNA, RNA, protein, phosphosite and acetylsite profiles, showing five primary NMF subtypes (top sample annotation row). B. Heatmap representing significantly enriched pathways (MSigDB Hallmark) in five multi-omic subtypes. C. Kaplan-Meier plot comparing survival probability of patients whose tumors were core members of a specific NMF subtype (NMF Core) to those whose tumors had characteristics of more than one NMF subtype (NMF mixed). D. Heatmap showing relative overexpression of mesenchymal proteins in the EMT-E subtype compared to others. E. Correlation between ssGSEA-based enrichment of EMT (Hallmark genesets) and fibroblast proliferation (GO: Gene Ontology) genesets (Pearson correlation=0.65, p=2.8\\u00d710\\u221214). F. Distribution of RTK correlation-based phosphosite enrichment (RTK CBPE) scores for PDGFRB and ROR2 across the five NMF subtypes. Wilcoxon p values for CBPE scores in EMT-E vs other subtypes are 1.5\\u00d710\\u22126 for PDGFRB and 2.7\\u00d710\\u22127 for ROR2. G. Proteins and phosphosites significantly associated with PDGFRB or ROR2 CBPE scores, known to play a role in EMT and extracellular matrix reorganization. The left panel shows Spearman correlation between PDGFRB CBPE scores and protein/phosphosite correlation between PDGFRB CBPE scores and protein/phosphosite abundance profiles. See also Figure S2 and Table S1\\u20134\\n\\nImpact of Somatic Mutations on Proteogenomic Features A. Significant (Wilcoxon FDR<0.05) cis- (circles) and trans-effects (squares) of selected mutations (x axis) on the expression of cancer-associated gene products, with mRNA in blue and proteins in green.\\n\\nB. Similar to panel A but showing phosphosites. B. Similar to panel A but showing phosphosites.\\n\\nC. Similar to panel A but showing acetylsites.\\n\\nD. Similar to panel A but showing ubiquitylsites. E. CNA data for CDKN2A and RB1 was used to classify tumors as having homozygous deletions or three classes of loss of heterozygosity mutations: nonsense/frameshift indel, missense/inframe indel, and splicing (see Table S4). CDKN2A genetic and indel, and splicing (see Table S4). CDKN2A genetic and hypermethylation annotations were based on the effect of the aberration on the p16INK4a (p16) gene product, but the effects of these CDKN2A/p16 aberrations on both major isoforms (p16INK4a and p14ARF aberrations on both major isoforms (p16INK4a and p14ARF (p14)) at the RNA (barplot) and protein (heatmap directly below barplot) levels are shown. Samples with amplification of CCND1\\u20133, CDK4, and CDK6 were assessed by GISTIC (threshold = 2), and the CDK4, and CDK6 were assessed by GISTIC (threshold = 2), and the genomic status, protein, and phosphoprotein levels for RB1 are included. Also shown are RNA-based scores for the cell cycle (MGPS, the mean of cell cycle genes and E2F target and G2M (MGPS, the mean of cell cycle genes and E2F target and G2M checkpoint gene set scores derived from ssGSEA of Hallmark gene sets) and phosphosite-based CDK kinase activity scores for CDKs 1, 2, and 4 derived from single sample post translational CDKs 1, 2, and 4 derived from single sample post translational modifications - signature enrichment analysis (ssPTM-SEA) of known kinase targets. Three tumors with copy number gain of CDKN2A are not included. F. Correlation between differential regulation of protein abundance (Log2 Fold-change (FC)) versus phosphoprotein log2 FC in tumors with NRF2 pathway mutation (one or two mutations in KEAP1, CUL3, or NFE2L2) versus NRF2 WT tumors (no NRF2 pathway KEAP1, CUL3, or NFE2L2) versus NRF2 WT tumors (no NRF2 pathway aberration). G. NRF2 pathway score and RNA, protein and phosphoprotein expression of key NRF pathway members according to NMF subtype.\\n\\nH. CDK5 protein expression (Log2 TMT ratio) by NMF subtype. P-values are from the Anova test. I. PTM-SEA-derived normalized enrichment scores (NES) for pathways enriched in NRF2 pathway-mutated (Mutant) vs wild-type samples (WT) plotted against NES for pathways enriched in NMF Classical vs other subtypes. Significantly upregulated (FDR<0.05) vs other subtypes. Significantly upregulated (FDR<0.05) PTM-SEA terms in the Classical subtype are indicated by red dots and labeled. See also Figure S3 and Table S4\\n\\nProteogenomic Impact of Chromosome 3q Amplification\\n\\nA. Differential protein expression (Log2 FC) between tumors and NATs for genes on chromosome arm 3q. B. Frequency distribution of \\u0394Np63\\u03b1 RNA expression in tumors and NATs. C. Differential protein expression in samples classified as \\u0394Np63\\u03b1-low vs -high based on \\u0394Np63\\u03b1 transcript level. The outlier upregulated gene product in red is BIRC5, also known as survivin. D. Differential expression of microRNAs in \\u0394Np63\\u03b1-low vs -high samples.\\n\\nE. Pearson correlation between expression of miR-205 and mRNA expression of its cognate, experimentally validated targets. F. Pearson correlation between expression of miR-205 and protein abundance of its cognate targets. G. Relationship between miR-205 expression (log2 TPM) and EMT (top, p=2.3\\u00d710\\u221208, Correlation = \\u22120.53) and DNA replication (bottom, p=1.1\\u00d710\\u221206, Correlation= 0.47) scores. H. Heatmap showing relative protein expression (TMT log ratio) of selected proteins with significant (FDR<0.01) Pearson correlation values (positive or negative) with SOX2 protein expression. I. Pearson correlation (p= 5.2\\u00d710\\u221207, Correlation = \\u22120.46) between SOX2 protein expression and HALLMARK_IL6_JAK-STAT_signaling NES.\\n\\nSee also Figure S4 and Table S4\\n\\nUbiquitylation landscape in LSCC A. Consensus clustering of protein ratio-corrected K-GG (di-glycine) site abundances in tumor samples and their associations. Heatmap shows only protein ratio-corrected K-GG sites with differential abundance across clusters (FDR<0.01). Enriched pathways abundance across clusters (FDR<0.01). Enriched pathways and molecular and clinical annotations are indicated. B. Number of K-GG sites showing significant correlations (FDR< 0.01) with E3 ligases. Shown are the five E3 ligases with the highest proportion of positive correlations. C. Pearson correlations between HERC5 protein expression and K-GG site protein-corrected abundance in key glycolytic enzymes PKM, PGK1, and ENO1. D. HERC5 protein expression (log2 TMT ratio) with samples grouped by immune subtype. Significant (Kruskal\\u2212Wallis, p = 2.8\\u00d710\\u221205) association is seen between HERC5 abundance and immune subtypes. E. Representative examples of significant spatial clustering of lysine acetylsites (purple) on PGK1 (left) and ACADVL (right) protein 3-D structure space-filling models (cyan) as determined by PTM CLUMPS. (PGK1 structure = PDB ID:3ZOZ. ACADVL structure = PTM CLUMPS. (PGK1 structure = PDB ID:3ZOZ. ACADVL structure = PDB ID:3B9). F. TXN protein levels in the NMF Classical subtype relative to NATs and other NMF subtypes (top left). Protein-corrected ubiquitylation (K-GG) sites are decreased on TXN1 in tumor subtypes relative to NATs (top right). TXNIP is decreased in tumor subtypes to NATs (top right). TXNIP is decreased in tumor subtypes relative to NATs (lower left). TXN1 activator TXNRD1 is increased in the Classical subtype relative to NATs and other NMF subtypes (lower right). Kruskal\\u2212Wallis p-values are indicated in the (lower right). Kruskal\\u2212Wallis p-values are indicated in the respective plots. G. Schematic representation of PTM-based modulation in LSCC tumors showing key enzymes in the metabolic and reactive oxygen species (ROS) pathways. Green and red arrows indicate higher and lower abundance of the corresponding PTMs in tumors. Putative lower abundance of the corresponding PTMs in tumors. Putative ISGylation targets of HERC5 are indicated by dotted lines. A known regulatory PKM phosphosite observed to be modulated in LSCC tumors is also highlighted. H. Lollipop charts showing the Log2 FC of acetylated (K-Ac) and ubiquitylated sites (K-GG) between tumors and NATs (Hyper: log2(FC)>1 or Hypo: <\\u22121, FDR<0.01). The upper panel shows specific sites that were hyper-ubiquitylated and hypoacetylated in tumors; that were hyper-ubiquitylated and hypoacetylated in tumors; the lower panel shows specific sites that were hyperacetylated and hypo-ubiquitylated in tumors. Dot colors indicate protein fold change between tumors and NATs. \\u201ck\\u201d represents modified lysine. I. Relative abundances of RAN K127 acetylation (K-Ac), ubiquitylation (K-GG) and RAN protein levels across NMF subtypes and NATs. Wilcoxon p-values are indicated above; ns represents p>0.05.\\n\\nSee also Figure S5 and Table S5\\n\\nImmune Landscape of LSCC A. Heatmaps illustrate cell type compositions and activities of selected individual genes/proteins and pathways across the four immune clusters: Hot, Warm, and Cold tumor and NAT-enriched. Successive heatmaps illustrate xCell immune signatures, mRNA and heatmaps illustrate xCell immune signatures, mRNA and protein expression of key immune-related markers, and ssGSEA pathway scores based on global proteomic data for biological pathways that were differentially regulated in immune groups based on both mRNA differentially regulated in immune groups based on both mRNA and global protein abundance (Common) or on global protein abundance alone (Proteomics only). B. Pathway scores of key pathways differentially expressed across the immune clusters. Wilcoxon p-values for the individual comparisons are provided on top. C. Contour plot of two-dimensional density based on Macrophage (x-axis) and CD8 T-cell scores (y-axis) showing the variation in these cell types\\u2019 distributions observed across the different immune clusters. D. Acetylsites differentially expressed between Hot and Cold tumors. Acetylsites of genes contained in the Hallmark Oxidative Phosphorylation pathway are highlighted in blue, ARHGDIB K135 is highlighted in red, and remaining sites are in gray. Darker is highlighted in red, and remaining sites are in gray. Darker color designates significant sites (FDR < 0.1). E. Regulation of Rho GTPase signaling including K135 acetylation of ARHGDIB. F. Global protein abundance of RAC2, DOCK2 and ELMO1, acetylproteome abundance of ARHGDIB K135k and phosphorylation abundance of ARHGEF6 at Serine 225 in immune clusters. Wilcoxon p-values are reported. G. RTK CBPE scores for 108 tumor samples and associated xCell signatures and pathway scores. The first heatmap shows CBPE scores of key RTKs, the second xCell signatures and the third pathway scores based on global protein abundance. H. Distribution of RTK CBPE scores for CSF1R, PDGFRB and FGFR2 stratified by immune clusters. Significance values (two-sided Wilcoxon test) between Hot clusters and combined Warm and Cold clusters are indicated on the violin plots. I. Heatmap showing proteins and phosphosites correlated with CSF1R CBPE scores that are known to be involved in immune evasion. See also Figure S6 and Table S6\\n\\nProteomic Features Related to Diagnosis, Prognosis, or Treatment A. Differentially expressed proteins between tumors and NATs. B. Significantly increased proteins (larger font indicates >4 fold) in the study LSCC cohort that are associated with poor overall survival (OS) or disease-free survival (DFS) in the TCGA LSCC cohort mRNA. C. Genetic dependencies of 502 proteins (log2 FC >2, FDR<0.01 and NAs <50%) in LSCC cell lines (n=16) profiled as part of the Achilles Dependency-Map project. D. Genes, ordered by their chromosomal location, that are deleted in at least 25% of the samples and significantly correlated to the immune score. Immune-related genes are highlighted. E. EGFR protein and tyrosine phosphorylation levels compared to EGFR copy number and an EGFR activity score (PROGENy). F. Heatmap showing Pearson correlation between the EGFR activity score represented by PROGENy (top) and RNA expression of EGFR ligands. *p<0.05 G. GO Biological Process enrichment for proteins with increased phosphorylation in EGFR amplified samples compared to non-amplified samples.\\n\\nH. Summary roadmap figure partitioned into five major categories, indicated by different colors. See also Figure S7 and Table S7 REAGENT or RESOURCE\\tSOURCE\\tIDENTIFIER\\t \\tAntibodies\\t \\tPTMScan\\u00ae Ubiquitin Remnant Motif (K-\\u03b5-GG) Kit\\tCell Signaling Technology\\tCatalog 5562\\t \\tPTMScan Acetyllysine Kit\\tCell Signaling Technology\\tCatalog: 13416\\t \\tCD4\\tDako\\tClone 4B12, RRID:AB_2728838 13416\\t \\tCD4\\tDako\\tClone 4B12, RRID:AB_2728838\\t \\tCD8\\tBio-Rad\\tClone 4B11, RRID:AB_322868\\t \\tCD163\\tAbcam\\tAb182422, RRID:AB_2753196\\t \\tARHGDIB\\tSigma-Aldrich\\tHPA051235, RRID:AB_2681398\\t \\tCK (pan-Cytokeratin)\\tDako\\tclone AE1/AE3, RRID:AB_2132885 CK (pan-Cytokeratin)\\tDako\\tclone AE1/AE3, RRID:AB_2132885\\t \\talpha-SMA\\tAbcam\\tab5694, RRID:AB_2223021\\t \\tBiological Samples\\t \\tPrimary tumor samples\\tSee Experimental Model and Subject Details\\tN/A\\t \\tChemicals and Reagents\\t \\tHPLC-grade water\\tJ.T. Baker\\tCatalog: Chemicals and Reagents\\t \\tHPLC-grade water\\tJ.T. Baker\\tCatalog: 4218-03\\t \\tUrea\\tSigma\\tCatalog: U0631\\t \\tSodium chloride\\tSigma\\tCatalog: 71376\\t \\t1M Tris, pH 8.0\\tInvitrogen\\tCatalog: AM9855G\\t \\tEthylenediaminetetraacetic acid\\tSigma\\tCatalog: E7889 AM9855G\\t \\tEthylenediaminetetraacetic acid\\tSigma\\tCatalog: E7889\\t \\tAprotinin\\tSigma\\tCatalog: A6103\\t \\tLeupeptin\\tRoche\\tCatalog: 11017101001\\t \\tPhenylmethylsulfonyl fluoride\\tSigma\\tCatalog: 78830\\t \\tSodium fluoride\\tSigma\\tCatalog: S7920\\t \\tPhosphatase inhibitor Sodium fluoride\\tSigma\\tCatalog: S7920\\t \\tPhosphatase inhibitor cocktail 2\\tSigma\\tCatalog: P5726\\t \\tPhosphatase inhibitor cocktail 3\\tSigma\\tCatalog: P0044\\t \\tDithiothretiol, No-Weigh Format\\tThermoScientific\\tCatalog: 20291\\t \\tIodoacetamide\\tSigma\\tCatalog: A3221 20291\\t \\tIodoacetamide\\tSigma\\tCatalog: A3221\\t \\tLysyl endopeptidase\\tWako Chemicals\\tCatalog: 129-02541\\t \\tSequencing-grade modified trypsin\\tPromega\\tCatalog: V511X\\t \\tFormic acid\\tSigma\\tCatalog: F0507\\t \\tAcetonitrile, LC-MS grade\\tHoneywell\\tCatalog: 34967 F0507\\t \\tAcetonitrile, LC-MS grade\\tHoneywell\\tCatalog: 34967\\t \\tAcetonitrile, anhydrous\\tSigma\\tCatalog: 271004\\t \\tTrifluoroacetic acid\\tSigma\\tCatalog: 302031\\t \\tTandem Mass Tag reagent kit \\u2013 11plex\\tThermoFisher\\tCatalog: A34808\\t \\t0.5M HEPES, pH 8.5\\tAlfa \\u2013 11plex\\tThermoFisher\\tCatalog: A34808\\t \\t0.5M HEPES, pH 8.5\\tAlfa Aesar\\tCatalog: J63218\\t \\tHydroxylamine solution, 50% (vol/vol) in H2O\\tAldrich\\tCatalog: 467804\\t \\tMethanol\\tHoneywell\\tCatalog: 34966\\t \\tAmmonium hydroxide solution, 28% (wt/vol) in 34966\\t \\tAmmonium hydroxide solution, 28% (wt/vol) in H2O\\tSigma\\tCatalog: 338818\\t \\tNi-NTA agarose beads\\tQiagen\\tCatalog: 30410\\t \\tIron (III) chloride\\tSigma\\tCatalog: 451649\\t \\tAcetic acid, glacial\\tSigma\\tCatalog: AX0073\\t \\tPotassium phosphate, acid, glacial\\tSigma\\tCatalog: AX0073\\t \\tPotassium phosphate, monobasic\\tSigma\\tCatalog: P0662\\t \\tPotassium phosphate, dibasic\\tSigma\\tCatalog: P3786\\t \\tMOPS\\tSigma\\tCatalog: M5162\\t \\tSodium hydroxide\\tVWR\\tCatalog: BDH7225\\t \\tSodium phosphate, dibasic\\tSigma\\tCatalog: BDH7225\\t \\tSodium phosphate, dibasic\\tSigma\\tCatalog: S9763\\t \\tPhosphate-buffered saline\\tFisher Scientific\\tCatalog: 10010023\\t \\tiVIEW DAB Detection Kit\\tRoche\\tCatalog: 760-091\\t \\tEquipment\\t \\tReversed-phase tC18 SepPak, 3cc 200mg\\tWaters\\tCatalog: WAT054925 tC18 SepPak, 3cc 200mg\\tWaters\\tCatalog: WAT054925\\t \\tSolid-phase C18 disk, for Stage-tips\\tEmpore\\tCatalog: 66883-U\\t \\tStage-tip needle\\tCadence\\tCatalog: 7928\\t \\tStage-tip puncher, PEEK tubing\\tIdex Health & Science\\tCatalog: 1581\\t \\tPicoFrit LC-MS column\\tNew Health & Science\\tCatalog: 1581\\t \\tPicoFrit LC-MS column\\tNew Objective\\tCatalog: PF360-75-10-N-5\\t \\tReproSil-Pur, 120 \\u212b, C18-AQ, 1.9-\\u03bcm resin\\tDr. Maisch\\tCatalog: r119.aq\\t \\tNanospray column heater\\tPhoenix S&T\\tCatalog: PST-CH-20U\\t \\tColumn heater column heater\\tPhoenix S&T\\tCatalog: PST-CH-20U\\t \\tColumn heater controller\\tPhoenix S&T\\tCatalog: PST-CHC\\t \\t300 \\u03bcL LC-MS autosampler vial and cap\\tWaters\\tCatalog: 186002639\\t \\tOffline HPLC column, 3.5-\\u03bcm particle size, 4.6 um \\u00d7 250 mm\\tAgilent\\tCatalog: Custom 3.5-\\u03bcm particle size, 4.6 um \\u00d7 250 mm\\tAgilent\\tCatalog: Custom order\\t \\tOffline 96-well fractionation plate\\tWhatman\\tCatalog: 77015200\\t \\t700 \\u03bcL bRP fractionation autosampler vial\\tThermoFisher\\tCatalog: C4010-14\\t \\t700 \\u03bcL bRP fractionation autosampler C4010-14\\t \\t700 \\u03bcL bRP fractionation autosampler cap\\tThermoFisher\\tCatalog: C4010-55A\\t \\t96-well microplate for BCA\\tGreiner\\tCatalog: 655101\\t \\tMicroplate foil cover\\tCorning\\tCatalog: PCR-AS-200\\t \\tVacuum centrifuge\\tThermoFisher\\tCatalog: SPD121P-115 Vacuum centrifuge\\tThermoFisher\\tCatalog: SPD121P-115\\t \\tCentrifuge\\tEppendorf\\tCatalog: 5427 R\\t \\tBenchtop mini centrifuge\\tCorning\\tCatalog: 6765\\t \\tBenchtop vortex\\tScientific Industries\\tCatalog: SI-0236\\t \\tIncubating shaker\\tVWR\\tCatalog: 12620-942\\t \\t15 mL SI-0236\\t \\tIncubating shaker\\tVWR\\tCatalog: 12620-942\\t \\t15 mL centrifuge tube\\tCorning\\tCatalog: 352097\\t \\t50 mL centrifuge tube\\tCorning\\tCatalog: 352070\\t \\t1.5 mL microtube w/o cap\\tSarstedt\\tCatalog: 72.607\\t \\t2.0 mL microtube w/o cap\\tSarstedt\\tCatalog: 72.608 72.607\\t \\t2.0 mL microtube w/o cap\\tSarstedt\\tCatalog: 72.608\\t \\tMicrotube caps\\tSarstedt\\tCatalog: 72.692\\t \\t1.5 mL snapcap tube\\tThermoFisher\\tCatalog: AM12450\\t \\t2.0 mL snapcap tube\\tThermoFisher\\tCatalog: AM12475\\t \\tInstrumentation\\t \\tMicroplate Reader\\tMolecular AM12475\\t \\tInstrumentation\\t \\tMicroplate Reader\\tMolecular Devices\\tCatalog: M2\\t \\tOffline HPLC System for bRP fractionation\\tAgilent 1260\\tCatalog: G1380-90000\\t \\tOnline LC for LC-MS\\tThermoFisher\\tCatalog: LC140\\t \\tQ Exactive Plus Mass LC for LC-MS\\tThermoFisher\\tCatalog: LC140\\t \\tQ Exactive Plus Mass Spectrometer\\tThermoFisher\\tCatalog: IQLAAEGAAPFALGMBDK\\t \\tQ Exactive HF-X Mass Spectrometer\\tThermoFisher\\tCatalog: 0726042\\t \\tOrbitrap Fusion Lumos Tribrid Mass Spectrometer\\tThermoFisher\\tCatalog: Fusion Lumos Tribrid Mass Spectrometer\\tThermoFisher\\tCatalog: IQLAAEGAAPFADBMBHQ\\t \\tCritical Commercial Assays\\t \\tTruSeq Stranded Total RNA Library Prep Kit with Ribo-Zero Gold\\tIllumina\\tCatalog: RS-122-2301\\t \\tInfinium MethylationEPIC Kit\\tIllumina\\tCatalog: RS-122-2301\\t \\tInfinium MethylationEPIC Kit\\tIllumina\\tCatalog: WG-317-1003\\t \\tNextera DNA Exosome Kit\\tIllumina\\tCatalog: 20020617\\t \\tKAPA Hyper Prep Kit, PCR-free\\tRoche\\tCatalog: 07962371001\\t \\tBCA Protein Assay Kit\\tThermoFisher\\tCatalog: 23225\\t \\tDeposited Data Protein Assay Kit\\tThermoFisher\\tCatalog: 23225\\t \\tDeposited Data\\t \\tDepMap: Mutation\\tDepMapPublic 21Q1 (PMID: 31068700; Dataset doi:10.6084/m9.figshare.13681534.v1)\\thttps://depmap.org/portal/download/\\t \\tDepMap: Segmented copy number\\tDepMapPublic 21Q1 (PMID: DepMap: Segmented copy number\\tDepMapPublic 21Q1 (PMID: 31068700; Dataset doi:10.6084/m9.figshare.13681534.v1)\\thttps://depmap.org/portal/download/\\t \\tDepMap: Gene level copy number\\tDepMapPublic 21Q1 (PMID: 31068700; Dataset level copy number\\tDepMapPublic 21Q1 (PMID: 31068700; Dataset doi:10.6084/m9.figshare.13681534.v1)\\thttps://depmap.org/portal/download/\\t \\tDepMap: RNAseq (transcript isoform)\\tDepMapPublic 21Q1 (PMID: 31068700; Dataset (transcript isoform)\\tDepMapPublic 21Q1 (PMID: 31068700; Dataset doi:10.6084/m9.figshare.13681534.v1)\\thttps://depmap.org/portal/download/\\t \\tDepMap: Proteomics (RPPA)\\tCCLE 2019 (PMID: 31068700)\\thttps://depmap.org/portal/download/\\t \\tDepMap: CRISPR KO screen DepMap: CRISPR KO screen (combined)\\tDepMapPublic 21Q1 (bioRxiv 2020.05.22.110247; doi: https://doi.org/10.1101/2020.05.22.110247)\\thttps://depmap.org/portal/download/\\t \\tDepMap: shRNA screen (combined)\\tDEMETER2 Data v6 (PMID: DepMap: shRNA screen (combined)\\tDEMETER2 Data v6 (PMID: 30389920)\\thttps://depmap.org/portal/download/\\t \\tDepMap: GDSC drug screen\\tSanger GDSC 1 (PMID: 27397505)\\thttps://depmap.org/portal/download/\\t \\tDepMap: PRISM drug screen\\tPRISM Repurposing 19Q4 DepMap: PRISM drug screen\\tPRISM Repurposing 19Q4 Secondary Screen (PMID: 32613204)\\thttps://depmap.org/portal/download/\\t \\tPhosphoSitePlus\\t\\thttps://www.phosphosite.org\\t \\tConnectivity Map (CMAP)\\t\\thttps://www.broadinstitute.org/connectivity-map-cmap\\t \\tHuman Human Protein Atlas (HPA)\\t\\thttps://www.proteinatlas.org\\t \\tCT Antigen database\\t\\thttp://www.cta.lncc.br\\t \\tDependency map (DepMap)\\t\\thttps://depmap.org/portal/v3.3.8 is a GECKOv2 Achilles dataset\\t \\tLibrary of Integrated Network-based Cellular Signatures Library of Integrated Network-based Cellular Signatures (LINCS)\\t\\thttps://clue.io/dataExpanded CMap LINCS Resource 2020 (1/28/2021 update)\\t \\tCPTAC HNSCC cohort\\t\\thttps://cptac-data-portal.georgetown.edu/study-summary/S054\\t \\tCPTAC LSCC cohort\\tThis CPTAC LSCC cohort\\tThis study\\thttps://cptac-data-portal.georgetown.edu/study-summary/S063\\t \\tSoftware and Algorithms\\t \\tmethylationArrayAnalysis (version 3.9)\\t\\thttps://master.bioconductor.org/packages/release/workflows/html/methylationArrayAnalysis.html Illumina EPIC methylation array (3.9)\\tHansen KD, 2019\\thttps://bioconductor.org/packages/release/data/annotation/html/IlluminaHumanMethylationEPICanno.ilm10b2.hg19.html\\t \\tMethylation array analysis pipeline for CPTAC\\tLi Ding Methylation array analysis pipeline for CPTAC\\tLi Ding Lab\\thttps://github.com/ding-lab/cptac_methylation\\t \\tmiRNA-Seq analysis pipeline for CPTAC\\tLi Ding Lab\\thttps://github.com/ding-lab/CPTAC_miRNA\\t \\tVEP\\t\\thttps://github.com/Ensembl/ensembl-vep/tags VEP\\t\\thttps://github.com/Ensembl/ensembl-vep/tags\\t \\tTNScope / DNAScope (Sentieon)\\t\\tsentieon.com\\t \\tvcfAnno\\t\\thttps://github.com/brentp/vcfanno\\t \\tVariantAnnotation (Bioconductor)\\t\\thttps://bioconductor.org/packages/release/bioc/html/VariantAnnotation.html arriba_v1.1.0\\t\\thttps://github.com/suhrig/arriba/\\t \\tfusioncatcher_v1.10\\t\\thttps://github.com/ndaniel/fusioncatcher/blob/master/doc/manual.md\\t \\teQTLGen\\t\\thttps://github.com/molgenis/systemsgenetics/wiki/eQTL-mapping-analysis-cookbook-(eQTLGen) Pindel0.2.5\\t\\thttp://gmt.genome.wustl.edu/packages/pindel/\\t \\tSignatureAnalyzer\\t\\thttps://software.broadinstitute.org/cancer/cga/msp\\t \\tCNVEX\\tMarcin Cieslik Lab\\thttps://github.com/mctp/cnvex\\t \\tCRISP\\tMarcin Cieslik Lab\\thttps://github.com/mctp/cnvex\\t \\tCRISP\\tMarcin Cieslik Lab\\thttps://github.com/mcieslik-mctp/crisp-build\\t \\tSpectrum Mill\\tKarl R. Clauser, Steven Carr Lab\\thttps://proteomics.broadinstitute.org/\\t \\tComBat Steven Carr Lab\\thttps://proteomics.broadinstitute.org/\\t \\tComBat (v3.20.0)\\t\\thttps://bioconductor.org/packages/release/bioc/html/sva.html\\t \\tgPCA\\t\\thttps://cran.r-project.org/web/packages/gPCA/index.html gPCA\\t\\thttps://cran.r-project.org/web/packages/gPCA/index.html\\t \\tGISTIC2.0\\t\\tftp://ftp.broadinstitute.org/pub/GISTIC2.0/GISTIC_2_0_23.tar.gz\\t \\tiProFun\\t\\thttps://github.com/WangLab-MSSM/iProFun iProFun\\t\\thttps://github.com/WangLab-MSSM/iProFun\\t \\tESTIMATE\\t\\thttps://bioinformatics.mdanderson.org/public-software/estimate/\\t \\tWebGestaltR\\t\\thttp://www.webgestalt.org/\\t \\tGSVA\\t\\thttps://bioconductor.org/packages/release/bioc/html/GSVA.html TSNet\\t\\thttps://github.com/WangLab-MSSM/TSNet\\t \\txCell\\t\\thttp://xcell.ucsf.edu/\\t \\tCPTAC LSCC Data Viewer\\tSteven Carr lab\\thttps://rstudio-connect.broadapps.org/CPTAC-LSCC2021/ Carr lab\\thttps://rstudio-connect.broadapps.org/CPTAC-LSCC2021/\\t \\tConsensusClusterPlus\\t\\thttp://bioconductor.org/packages/release/bioc/html/CancerSubtypes.html\\t \\tMS-GF+\\t\\thttps://github.com/MSGFPlus/msgfplus\\t \\tNeoFlow\\t\\thttps://github.com/bzhanglab/neoflow NeoFlow\\t\\thttps://github.com/bzhanglab/neoflow\\t \\tnetMHCpan\\t\\thttp://www.cbs.dtu.dk/services/NetMHCpan/\\t \\tOptitype\\t\\thttps://github.com/FRED-2/OptiType\\t \\tCustomprodbj\\t\\thttps://github.com/bzhanglab/customprodbj\\t \\tPDV\\t\\thttps://github.com/wenbostar/PDV PDV\\t\\thttps://github.com/wenbostar/PDV\\t \\tPepQuery\\t\\thttp://pepquery.org\\t \\tPTM-SEA\\t)\\thttps://github.com/broadinstitute/ssGSEA2.0\\t \\tPTMsigDB\\t)\\thttp://prot-shiny-vm.broadinstitute.org:3838/ptmsigdb-app/\\t \\tTerra\\tBroad Institute data science Terra\\tBroad Institute data science platform.\\thttps://terra.bio/\\t \\tPanoply\\tBroad Institute Proteomics Platform\\thttps://github.com/broadinstitute/PANOPLY\\t \\tCMap\\t\\thttps://clue.io/cmap\\t \\tLIMMA v3.36 (R CMap\\t\\thttps://clue.io/cmap\\t \\tLIMMA v3.36 (R Package)\\t\\thttps://bioconductor.org/packages/release/bioc/html/limma.html\\t \\tFactoMineR v1.41NMF(R - package)\\t\\thttps://cran.r-project.org/web/packages/FactoMineR/index.html\\t \\tMClust v5.4 (R package)\\t(Scrucca, MClust v5.4 (R package)\\t(Scrucca, Fop, Murphy and Raftery, 2017)\\thttps://cran.r-project.org/web/packages/mclust/index.html\\t \\tg:Profiler\\t\\thttps://biit.cs.ut.ee/gprofiler/gost\\t \\tCytoscape\\t\\thttps://cytoscape.org/ Cytoscape\\t\\thttps://cytoscape.org/\\t \\tImmuneSubtypeClassifier\\t\\thttps://github.com/CRI-iAtlas/ImmuneSubtypeClassifier\\t \\tProteinPaint\\t\\thttps://pecan.stjude.doud/proteinpaint/\\t \\tOrdinal\\tChristensen RHB (2019)\\thttps://CRAN.R-project.org/package=ordinal\",\n",
            "                    \"pdc_study_id\": \"PDC000237\",\n",
            "                    \"title\": \"A proteogenomic portrait of lung squamous cell carcinoma\"\n",
            "                }\n",
            "            ]\n",
            "        }\n",
            "    }\n",
            "}\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "Load LLM"
      ],
      "metadata": {
        "id": "1fqmBKpexS-q"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "from google.colab import drive\n",
        "drive.mount('/content/drive')"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "CQM7iZQ0JsE5",
        "outputId": "fa42f5d1-5250-4956-ce27-81213b653c51"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Mounted at /content/drive\n"
          ]
        }
      ]
    },
    {
      "source": [
        "!pip install langchain_community\n",
        "from langchain_community.llms import LlamaCpp"
      ],
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "6Yxs1XVPNLJ6",
        "outputId": "5dd2c225-7ecb-4525-cd49-7ba01e331afb"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Collecting langchain_community\n",
            "  Downloading langchain_community-0.2.10-py3-none-any.whl.metadata (2.7 kB)\n",
            "Requirement already satisfied: PyYAML>=5.3 in /usr/local/lib/python3.10/dist-packages (from langchain_community) (6.0.1)\n",
            "Requirement already satisfied: SQLAlchemy<3,>=1.4 in /usr/local/lib/python3.10/dist-packages (from langchain_community) (2.0.31)\n",
            "Requirement already satisfied: aiohttp<4.0.0,>=3.8.3 in /usr/local/lib/python3.10/dist-packages (from langchain_community) (3.9.5)\n",
            "Collecting dataclasses-json<0.7,>=0.5.7 (from langchain_community)\n",
            "  Downloading dataclasses_json-0.6.7-py3-none-any.whl.metadata (25 kB)\n",
            "Requirement already satisfied: langchain<0.3.0,>=0.2.9 in /usr/local/lib/python3.10/dist-packages (from langchain_community) (0.2.11)\n",
            "Requirement already satisfied: langchain-core<0.3.0,>=0.2.23 in /usr/local/lib/python3.10/dist-packages (from langchain_community) (0.2.23)\n",
            "Requirement already satisfied: langsmith<0.2.0,>=0.1.0 in /usr/local/lib/python3.10/dist-packages (from langchain_community) (0.1.93)\n",
            "Requirement already satisfied: numpy<2,>=1 in /usr/local/lib/python3.10/dist-packages (from langchain_community) (1.25.2)\n",
            "Requirement already satisfied: requests<3,>=2 in /usr/local/lib/python3.10/dist-packages (from langchain_community) (2.31.0)\n",
            "Requirement already satisfied: tenacity!=8.4.0,<9.0.0,>=8.1.0 in /usr/local/lib/python3.10/dist-packages (from langchain_community) (8.5.0)\n",
            "Requirement already satisfied: aiosignal>=1.1.2 in /usr/local/lib/python3.10/dist-packages (from aiohttp<4.0.0,>=3.8.3->langchain_community) (1.3.1)\n",
            "Requirement already satisfied: attrs>=17.3.0 in /usr/local/lib/python3.10/dist-packages (from aiohttp<4.0.0,>=3.8.3->langchain_community) (23.2.0)\n",
            "Requirement already satisfied: frozenlist>=1.1.1 in /usr/local/lib/python3.10/dist-packages (from aiohttp<4.0.0,>=3.8.3->langchain_community) (1.4.1)\n",
            "Requirement already satisfied: multidict<7.0,>=4.5 in /usr/local/lib/python3.10/dist-packages (from aiohttp<4.0.0,>=3.8.3->langchain_community) (6.0.5)\n",
            "Requirement already satisfied: yarl<2.0,>=1.0 in /usr/local/lib/python3.10/dist-packages (from aiohttp<4.0.0,>=3.8.3->langchain_community) (1.9.4)\n",
            "Requirement already satisfied: async-timeout<5.0,>=4.0 in /usr/local/lib/python3.10/dist-packages (from aiohttp<4.0.0,>=3.8.3->langchain_community) (4.0.3)\n",
            "Collecting marshmallow<4.0.0,>=3.18.0 (from dataclasses-json<0.7,>=0.5.7->langchain_community)\n",
            "  Downloading marshmallow-3.21.3-py3-none-any.whl.metadata (7.1 kB)\n",
            "Collecting typing-inspect<1,>=0.4.0 (from dataclasses-json<0.7,>=0.5.7->langchain_community)\n",
            "  Downloading typing_inspect-0.9.0-py3-none-any.whl.metadata (1.5 kB)\n",
            "Requirement already satisfied: langchain-text-splitters<0.3.0,>=0.2.0 in /usr/local/lib/python3.10/dist-packages (from langchain<0.3.0,>=0.2.9->langchain_community) (0.2.2)\n",
            "Requirement already satisfied: pydantic<3,>=1 in /usr/local/lib/python3.10/dist-packages (from langchain<0.3.0,>=0.2.9->langchain_community) (2.8.2)\n",
            "Requirement already satisfied: jsonpatch<2.0,>=1.33 in /usr/local/lib/python3.10/dist-packages (from langchain-core<0.3.0,>=0.2.23->langchain_community) (1.33)\n",
            "Requirement already satisfied: packaging<25,>=23.2 in /usr/local/lib/python3.10/dist-packages (from langchain-core<0.3.0,>=0.2.23->langchain_community) (24.1)\n",
            "Requirement already satisfied: orjson<4.0.0,>=3.9.14 in /usr/local/lib/python3.10/dist-packages (from langsmith<0.2.0,>=0.1.0->langchain_community) (3.10.6)\n",
            "Requirement already satisfied: charset-normalizer<4,>=2 in /usr/local/lib/python3.10/dist-packages (from requests<3,>=2->langchain_community) (3.3.2)\n",
            "Requirement already satisfied: idna<4,>=2.5 in /usr/local/lib/python3.10/dist-packages (from requests<3,>=2->langchain_community) (3.7)\n",
            "Requirement already satisfied: urllib3<3,>=1.21.1 in /usr/local/lib/python3.10/dist-packages (from requests<3,>=2->langchain_community) (2.0.7)\n",
            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.10/dist-packages (from requests<3,>=2->langchain_community) (2024.7.4)\n",
            "Requirement already satisfied: typing-extensions>=4.6.0 in /usr/local/lib/python3.10/dist-packages (from SQLAlchemy<3,>=1.4->langchain_community) (4.12.2)\n",
            "Requirement already satisfied: greenlet!=0.4.17 in /usr/local/lib/python3.10/dist-packages (from SQLAlchemy<3,>=1.4->langchain_community) (3.0.3)\n",
            "Requirement already satisfied: jsonpointer>=1.9 in /usr/local/lib/python3.10/dist-packages (from jsonpatch<2.0,>=1.33->langchain-core<0.3.0,>=0.2.23->langchain_community) (3.0.0)\n",
            "Requirement already satisfied: annotated-types>=0.4.0 in /usr/local/lib/python3.10/dist-packages (from pydantic<3,>=1->langchain<0.3.0,>=0.2.9->langchain_community) (0.7.0)\n",
            "Requirement already satisfied: pydantic-core==2.20.1 in /usr/local/lib/python3.10/dist-packages (from pydantic<3,>=1->langchain<0.3.0,>=0.2.9->langchain_community) (2.20.1)\n",
            "Collecting mypy-extensions>=0.3.0 (from typing-inspect<1,>=0.4.0->dataclasses-json<0.7,>=0.5.7->langchain_community)\n",
            "  Downloading mypy_extensions-1.0.0-py3-none-any.whl.metadata (1.1 kB)\n",
            "Downloading langchain_community-0.2.10-py3-none-any.whl (2.3 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m2.3/2.3 MB\u001b[0m \u001b[31m11.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading dataclasses_json-0.6.7-py3-none-any.whl (28 kB)\n",
            "Downloading marshmallow-3.21.3-py3-none-any.whl (49 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m49.2/49.2 kB\u001b[0m \u001b[31m3.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading typing_inspect-0.9.0-py3-none-any.whl (8.8 kB)\n",
            "Downloading mypy_extensions-1.0.0-py3-none-any.whl (4.7 kB)\n",
            "Installing collected packages: mypy-extensions, marshmallow, typing-inspect, dataclasses-json, langchain_community\n",
            "Successfully installed dataclasses-json-0.6.7 langchain_community-0.2.10 marshmallow-3.21.3 mypy-extensions-1.0.0 typing-inspect-0.9.0\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!pip install llama-cpp-python"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "gqyhMThLW0ZZ",
        "outputId": "ac72903c-385b-49d3-afec-9f088e660d7e"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Collecting llama-cpp-python\n",
            "  Downloading llama_cpp_python-0.2.83.tar.gz (49.4 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m49.4/49.4 MB\u001b[0m \u001b[31m11.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25h  Installing build dependencies ... \u001b[?25l\u001b[?25hdone\n",
            "  Getting requirements to build wheel ... \u001b[?25l\u001b[?25hdone\n",
            "  Installing backend dependencies ... \u001b[?25l\u001b[?25hdone\n",
            "  Preparing metadata (pyproject.toml) ... \u001b[?25l\u001b[?25hdone\n",
            "Requirement already satisfied: typing-extensions>=4.5.0 in /usr/local/lib/python3.10/dist-packages (from llama-cpp-python) (4.12.2)\n",
            "Requirement already satisfied: numpy>=1.20.0 in /usr/local/lib/python3.10/dist-packages (from llama-cpp-python) (1.25.2)\n",
            "Collecting diskcache>=5.6.1 (from llama-cpp-python)\n",
            "  Downloading diskcache-5.6.3-py3-none-any.whl.metadata (20 kB)\n",
            "Requirement already satisfied: jinja2>=2.11.3 in /usr/local/lib/python3.10/dist-packages (from llama-cpp-python) (3.1.4)\n",
            "Requirement already satisfied: MarkupSafe>=2.0 in /usr/local/lib/python3.10/dist-packages (from jinja2>=2.11.3->llama-cpp-python) (2.1.5)\n",
            "Downloading diskcache-5.6.3-py3-none-any.whl (45 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m45.5/45.5 kB\u001b[0m \u001b[31m3.1 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hBuilding wheels for collected packages: llama-cpp-python\n",
            "  Building wheel for llama-cpp-python (pyproject.toml) ... \u001b[?25l\u001b[?25hdone\n",
            "  Created wheel for llama-cpp-python: filename=llama_cpp_python-0.2.83-cp310-cp310-linux_x86_64.whl size=2843660 sha256=cdd46bf28c9315d41a64f8a22dab59f362e2662bf5b22075efa1801755b5f235\n",
            "  Stored in directory: /root/.cache/pip/wheels/25/97/95/bd309ea454a04b3b46c2b6321749172cf68a0279d892f12534\n",
            "Successfully built llama-cpp-python\n",
            "Installing collected packages: diskcache, llama-cpp-python\n",
            "Successfully installed diskcache-5.6.3 llama-cpp-python-0.2.83\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "model_path=\"/content/drive/MyDrive/Meta-Llama-3-8B-Instruct-Q4_K_M.gguf\""
      ],
      "metadata": {
        "id": "zTuD4teJ3Z5v"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "source": [
        "from langchain_community.llms import LlamaCpp\n",
        "\n",
        "llm = LlamaCpp(\n",
        "    model_path=\"/content/drive/MyDrive/Meta-Llama-3-8B-Instruct-Q4_K_M.gguf\",\n",
        "    n_ctx=8192,\n",
        "    temperature=0.6,\n",
        "    max_tokens=2048,\n",
        "    top_p=1\n",
        ")"
      ],
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "WfSVYjl53tj6",
        "outputId": "e6c2e6d7-5487-4976-9a5c-5011d7698c18"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "llama_model_loader: loaded meta data with 22 key-value pairs and 291 tensors from /content/drive/MyDrive/Meta-Llama-3-8B-Instruct-Q4_K_M.gguf (version GGUF V3 (latest))\n",
            "llama_model_loader: Dumping metadata keys/values. Note: KV overrides do not apply in this output.\n",
            "llama_model_loader: - kv   0:                       general.architecture str              = llama\n",
            "llama_model_loader: - kv   1:                               general.name str              = ..\n",
            "llama_model_loader: - kv   2:                           llama.vocab_size u32              = 128256\n",
            "llama_model_loader: - kv   3:                       llama.context_length u32              = 8192\n",
            "llama_model_loader: - kv   4:                     llama.embedding_length u32              = 4096\n",
            "llama_model_loader: - kv   5:                          llama.block_count u32              = 32\n",
            "llama_model_loader: - kv   6:                  llama.feed_forward_length u32              = 14336\n",
            "llama_model_loader: - kv   7:                 llama.rope.dimension_count u32              = 128\n",
            "llama_model_loader: - kv   8:                 llama.attention.head_count u32              = 32\n",
            "llama_model_loader: - kv   9:              llama.attention.head_count_kv u32              = 8\n",
            "llama_model_loader: - kv  10:     llama.attention.layer_norm_rms_epsilon f32              = 0.000010\n",
            "llama_model_loader: - kv  11:                       llama.rope.freq_base f32              = 500000.000000\n",
            "llama_model_loader: - kv  12:                          general.file_type u32              = 15\n",
            "llama_model_loader: - kv  13:                       tokenizer.ggml.model str              = gpt2\n",
            "llama_model_loader: - kv  14:                      tokenizer.ggml.tokens arr[str,128256]  = [\"!\", \"\\\"\", \"#\", \"$\", \"%\", \"&\", \"'\", ...\n",
            "llama_model_loader: - kv  15:                      tokenizer.ggml.scores arr[f32,128256]  = [0.000000, 0.000000, 0.000000, 0.0000...\n",
            "llama_model_loader: - kv  16:                  tokenizer.ggml.token_type arr[i32,128256]  = [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, ...\n",
            "llama_model_loader: - kv  17:                      tokenizer.ggml.merges arr[str,280147]  = [\"Ġ Ġ\", \"Ġ ĠĠĠ\", \"ĠĠ ĠĠ\", \"...\n",
            "llama_model_loader: - kv  18:                tokenizer.ggml.bos_token_id u32              = 128000\n",
            "llama_model_loader: - kv  19:                tokenizer.ggml.eos_token_id u32              = 128009\n",
            "llama_model_loader: - kv  20:                    tokenizer.chat_template str              = {% set loop_messages = messages %}{% ...\n",
            "llama_model_loader: - kv  21:               general.quantization_version u32              = 2\n",
            "llama_model_loader: - type  f32:   65 tensors\n",
            "llama_model_loader: - type q4_K:  193 tensors\n",
            "llama_model_loader: - type q6_K:   33 tensors\n",
            "llm_load_vocab: missing pre-tokenizer type, using: 'default'\n",
            "llm_load_vocab:                                             \n",
            "llm_load_vocab: ************************************        \n",
            "llm_load_vocab: GENERATION QUALITY WILL BE DEGRADED!        \n",
            "llm_load_vocab: CONSIDER REGENERATING THE MODEL             \n",
            "llm_load_vocab: ************************************        \n",
            "llm_load_vocab:                                             \n",
            "llm_load_vocab: special tokens cache size = 256\n",
            "llm_load_vocab: token to piece cache size = 0.8000 MB\n",
            "llm_load_print_meta: format           = GGUF V3 (latest)\n",
            "llm_load_print_meta: arch             = llama\n",
            "llm_load_print_meta: vocab type       = BPE\n",
            "llm_load_print_meta: n_vocab          = 128256\n",
            "llm_load_print_meta: n_merges         = 280147\n",
            "llm_load_print_meta: vocab_only       = 0\n",
            "llm_load_print_meta: n_ctx_train      = 8192\n",
            "llm_load_print_meta: n_embd           = 4096\n",
            "llm_load_print_meta: n_layer          = 32\n",
            "llm_load_print_meta: n_head           = 32\n",
            "llm_load_print_meta: n_head_kv        = 8\n",
            "llm_load_print_meta: n_rot            = 128\n",
            "llm_load_print_meta: n_swa            = 0\n",
            "llm_load_print_meta: n_embd_head_k    = 128\n",
            "llm_load_print_meta: n_embd_head_v    = 128\n",
            "llm_load_print_meta: n_gqa            = 4\n",
            "llm_load_print_meta: n_embd_k_gqa     = 1024\n",
            "llm_load_print_meta: n_embd_v_gqa     = 1024\n",
            "llm_load_print_meta: f_norm_eps       = 0.0e+00\n",
            "llm_load_print_meta: f_norm_rms_eps   = 1.0e-05\n",
            "llm_load_print_meta: f_clamp_kqv      = 0.0e+00\n",
            "llm_load_print_meta: f_max_alibi_bias = 0.0e+00\n",
            "llm_load_print_meta: f_logit_scale    = 0.0e+00\n",
            "llm_load_print_meta: n_ff             = 14336\n",
            "llm_load_print_meta: n_expert         = 0\n",
            "llm_load_print_meta: n_expert_used    = 0\n",
            "llm_load_print_meta: causal attn      = 1\n",
            "llm_load_print_meta: pooling type     = 0\n",
            "llm_load_print_meta: rope type        = 0\n",
            "llm_load_print_meta: rope scaling     = linear\n",
            "llm_load_print_meta: freq_base_train  = 500000.0\n",
            "llm_load_print_meta: freq_scale_train = 1\n",
            "llm_load_print_meta: n_ctx_orig_yarn  = 8192\n",
            "llm_load_print_meta: rope_finetuned   = unknown\n",
            "llm_load_print_meta: ssm_d_conv       = 0\n",
            "llm_load_print_meta: ssm_d_inner      = 0\n",
            "llm_load_print_meta: ssm_d_state      = 0\n",
            "llm_load_print_meta: ssm_dt_rank      = 0\n",
            "llm_load_print_meta: model type       = 8B\n",
            "llm_load_print_meta: model ftype      = Q4_K - Medium\n",
            "llm_load_print_meta: model params     = 8.03 B\n",
            "llm_load_print_meta: model size       = 4.58 GiB (4.89 BPW) \n",
            "llm_load_print_meta: general.name     = ..\n",
            "llm_load_print_meta: BOS token        = 128000 '<|begin_of_text|>'\n",
            "llm_load_print_meta: EOS token        = 128009 '<|eot_id|>'\n",
            "llm_load_print_meta: LF token         = 128 'Ä'\n",
            "llm_load_print_meta: EOT token        = 128009 '<|eot_id|>'\n",
            "llm_load_print_meta: max token length = 256\n",
            "llm_load_tensors: ggml ctx size =    0.14 MiB\n",
            "llm_load_tensors:        CPU buffer size =  4685.30 MiB\n",
            "........................................................................................\n",
            "llama_new_context_with_model: n_batch is less than GGML_KQ_MASK_PAD - increasing to 32\n",
            "llama_new_context_with_model: n_ctx      = 8192\n",
            "llama_new_context_with_model: n_batch    = 32\n",
            "llama_new_context_with_model: n_ubatch   = 32\n",
            "llama_new_context_with_model: flash_attn = 0\n",
            "llama_new_context_with_model: freq_base  = 10000.0\n",
            "llama_new_context_with_model: freq_scale = 1\n",
            "llama_kv_cache_init:        CPU KV buffer size =  1024.00 MiB\n",
            "llama_new_context_with_model: KV self size  = 1024.00 MiB, K (f16):  512.00 MiB, V (f16):  512.00 MiB\n",
            "llama_new_context_with_model:        CPU  output buffer size =     0.49 MiB\n",
            "llama_new_context_with_model:        CPU compute buffer size =    35.00 MiB\n",
            "llama_new_context_with_model: graph nodes  = 1030\n",
            "llama_new_context_with_model: graph splits = 1\n",
            "AVX = 1 | AVX_VNNI = 0 | AVX2 = 1 | AVX512 = 0 | AVX512_VBMI = 0 | AVX512_VNNI = 0 | AVX512_BF16 = 0 | FMA = 1 | NEON = 0 | SVE = 0 | ARM_FMA = 0 | F16C = 1 | FP16_VA = 0 | WASM_SIMD = 0 | BLAS = 0 | SSE3 = 1 | SSSE3 = 1 | VSX = 0 | MATMUL_INT8 = 0 | LLAMAFILE = 0 | \n",
            "Model metadata: {'tokenizer.chat_template': \"{% set loop_messages = messages %}{% for message in loop_messages %}{% set content = '<|start_header_id|>' + message['role'] + '<|end_header_id|>\\n\\n'+ message['content'] | trim + '<|eot_id|>' %}{% if loop.index0 == 0 %}{% set content = bos_token + content %}{% endif %}{{ content }}{% endfor %}{{ '<|start_header_id|>assistant<|end_header_id|>\\n\\n' }}\", 'tokenizer.ggml.eos_token_id': '128009', 'general.quantization_version': '2', 'tokenizer.ggml.model': 'gpt2', 'general.architecture': 'llama', 'llama.rope.freq_base': '500000.000000', 'llama.context_length': '8192', 'general.name': '..', 'llama.vocab_size': '128256', 'general.file_type': '15', 'llama.embedding_length': '4096', 'llama.feed_forward_length': '14336', 'llama.attention.layer_norm_rms_epsilon': '0.000010', 'llama.rope.dimension_count': '128', 'tokenizer.ggml.bos_token_id': '128000', 'llama.attention.head_count': '32', 'llama.block_count': '32', 'llama.attention.head_count_kv': '8'}\n",
            "Available chat formats from metadata: chat_template.default\n",
            "Using gguf chat template: {% set loop_messages = messages %}{% for message in loop_messages %}{% set content = '<|start_header_id|>' + message['role'] + '<|end_header_id|>\n",
            "\n",
            "'+ message['content'] | trim + '<|eot_id|>' %}{% if loop.index0 == 0 %}{% set content = bos_token + content %}{% endif %}{{ content }}{% endfor %}{{ '<|start_header_id|>assistant<|end_header_id|>\n",
            "\n",
            "' }}\n",
            "Using chat eos_token: <|eot_id|>\n",
            "Using chat bos_token: <|begin_of_text|>\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "Prompt setup"
      ],
      "metadata": {
        "id": "up91WWPb00EN"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "system_prompt = \"\"\"\n",
        "You are a knowledgeable AI assistant specialized in providing detailed information from the Proteomics Data Commons (PDC). Your goal is to help researchers and users find specific studies related to cancer, understand the details of these studies, and answer questions based on the PDC's extensive data repositories. Always provide precise, relevant, and up-to-date information.\n",
        "\"\"\"\n",
        "#directly invokes a language model with a prompt composed of a system-defined context and the user's query\n",
        "def generate_response(question):\n",
        "    prompt = f\"{system_prompt}\\nUser: {question}\\nAssistant:\"\n",
        "    response = llm.invoke(prompt)\n",
        "    return response.strip()\n",
        "\n",
        "# Example questions\n",
        "questions = [\n",
        "    \"What is Study PDC000251?\",\n",
        "    \"Provide studies linked with gastric cancer.\",\n",
        "]\n",
        "# Generating responses for the given questions\n",
        "for question in questions:\n",
        "    print(f\"Question: {question}\")\n",
        "    print(\"Response:\", generate_response(question))\n",
        "    print(\"---\")\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "tszATbOOj8CY",
        "outputId": "3592a339-3e28-4fc5-d2c9-21373b8ba395"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Question: What is Study PDC000251?\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Llama.generate: prefix-match hit\n",
            "\n",
            "llama_print_timings:        load time =   10954.98 ms\n",
            "llama_print_timings:      sample time =     410.56 ms /   140 runs   (    2.93 ms per token,   341.00 tokens per second)\n",
            "llama_print_timings: prompt eval time =    7137.24 ms /     2 tokens ( 3568.62 ms per token,     0.28 tokens per second)\n",
            "llama_print_timings:        eval time =  140220.63 ms /   139 runs   ( 1008.78 ms per token,     0.99 tokens per second)\n",
            "llama_print_timings:       total time =  143769.31 ms /   141 tokens\n",
            "Llama.generate: prefix-match hit\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Response: According to the Proteomics Data Commons (PDC), Study PDC000251 is a mass spectrometry-based study focused on understanding the protein expression profiles of breast cancer tissues.\n",
            "\n",
            "This study was conducted by researchers at the University of California, San Francisco (UCSF) and published in the journal Nature Communications in 2019.\n",
            "\n",
            "The study used high-resolution mass spectrometry to analyze the proteomes of breast cancer tissues. The researchers identified over 10,000 proteins with high confidence and found that many of these proteins were differentially expressed between healthy and tumor tissues.\n",
            "\n",
            "The study's findings have important implications for our understanding of breast cancer biology and the development of new therapeutic strategies for this disease.\n",
            "---\n",
            "Question: Provide studies linked with gastric cancer.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "\n",
            "llama_print_timings:        load time =   10954.98 ms\n",
            "llama_print_timings:      sample time =     673.80 ms /   245 runs   (    2.75 ms per token,   363.61 tokens per second)\n",
            "llama_print_timings: prompt eval time =    7040.26 ms /    10 tokens (  704.03 ms per token,     1.42 tokens per second)\n",
            "llama_print_timings:        eval time =  239415.73 ms /   244 runs   (  981.21 ms per token,     1.02 tokens per second)\n",
            "llama_print_timings:       total time =  247756.10 ms /   254 tokens\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Response: I'd be happy to help! According to the Proteomics Data Commons (PDC), there are several studies related to gastric cancer.\n",
            "\n",
            "Here are a few examples:\n",
            "\n",
            "1. \"Proteomic analysis of gastric cancer\" - This study published in the journal Cancer Research used mass spectrometry to identify proteins that were differentially expressed between normal and cancerous gastric tissues.\n",
            "2. \"Gastric cancer biomarkers identified by proteomics\" - This study published in the Journal of Proteome Research used a combination of mass spectrometry and bioinformatics tools to identify potential biomarkers for gastric cancer.\n",
            "3. \"Proteomic analysis of the tumor microenvironment in gastric cancer\" - This study published in the journal Cancer & Metabolism used mass spectrometry to analyze the proteome of the tumor microenvironment in gastric cancer, with a focus on identifying potential therapeutic targets.\n",
            "\n",
            "These studies demonstrate the power of proteomics in understanding the biology and behavior of gastric cancer. By analyzing the proteomes of normal and cancerous tissues, researchers can identify potential biomarkers for early detection and diagnosis, as well as potential therapeutic targets for treatment.\n",
            "\n",
            "I hope this information is helpful! Let me know if you have any further questions.\n",
            "---\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "RAG approach"
      ],
      "metadata": {
        "id": "1GwpCT3V3Stk"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "def retrieve_documents(client, query_text, top_k=5):\n",
        "    query_embedding = generate_embedding(query_text)\n",
        "    results = client.query.get(\n",
        "        \"Document\",\n",
        "        properties=[\"study_id\", \"pdc_study_id\", \"study_submitter_id\",  \"study_name\", \"program_name\", \"project_name\",\n",
        "        \"disease_type\", \"primary_site\", \"analytical_fraction\",\n",
        "        \"experiment_type\", \"cases_count\", \"aliquots_count\",\n",
        "        \"pubmed_id\", \"doi\", \"author\", \"title\", \"journal\", \"journal_url\",\n",
        "        \"year\", \"abstract\", \"full_text\", \"embedding\"]\n",
        "    ).with_near_vector({\n",
        "        \"vector\": query_embedding,\n",
        "        \"certainty\": 0.7\n",
        "    }).with_limit(top_k).do()\n",
        "    documents = results['data']['Get']['Document']\n",
        "    # Create a list of dictionaries with only the desired fields\n",
        "    return [{'pdc_study_id': doc.get('pdc_study_id'), 'abstract': doc.get('abstract')} for doc in documents]\n",
        "\n",
        "def format_prompt(question, documents):\n",
        "    context = \"\\n\".join([f\"PDC Study ID: {doc['pdc_study_id']}, Abstract: {doc['abstract']}\" for doc in documents])\n",
        " #context = \"\\n\".join([f\"PDC Study ID: {doc['pdc_study_id']}, Abstract: {doc['abstract']}, Details: {doc.get('full_text', 'No additional information available.')}\" for doc in documents])\n",
        "    return f\"{system_prompt}\\nUser: {question}\\nContext: {context}\\nAssistant:\"\n",
        "\n",
        "def generate_response(question):\n",
        "    documents = retrieve_documents(client, question, top_k=3)\n",
        "    prompt = format_prompt(question, documents)\n",
        "    response = llm.invoke(prompt)\n",
        "    return response.strip()\n",
        "\n",
        "# Example questions and invoking the generate_response\n",
        "questions = [\"What is Study PDC000251?\", \"Provide studies linked with gastric cancer.\"]\n",
        "for question in questions:\n",
        "    print(f\"Question: {question}\")\n",
        "    print(\"Response:\", generate_response(question))\n",
        "    print(\"---\")"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "BiIf1vlo08xM",
        "outputId": "bc5583c0-f1ab-4a37-8ba5-012c4d4485fd"
      },
      "execution_count": null,
      "outputs": [
        {
          "metadata": {
            "tags": null
          },
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "INFO:weaviate-client:Embedded weaviate wasn't listening on ports http:8079 & grpc:50060, so starting embedded weaviate again\n",
            "INFO:weaviate-client:Started /root/.cache/weaviate-embedded: process ID 62413\n"
          ]
        },
        {
          "metadata": {
            "tags": null
          },
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Question: What is Study PDC000251?\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Llama.generate: prefix-match hit\n",
            "\n",
            "llama_print_timings:        load time =   10954.98 ms\n",
            "llama_print_timings:      sample time =     523.33 ms /   135 runs   (    3.88 ms per token,   257.96 tokens per second)\n",
            "llama_print_timings: prompt eval time =  324589.29 ms /    23 tokens (14112.58 ms per token,     0.07 tokens per second)\n",
            "llama_print_timings:        eval time =  185580.27 ms /   134 runs   ( 1384.93 ms per token,     0.72 tokens per second)\n",
            "llama_print_timings:       total time =  205194.71 ms /   157 tokens\n",
            "Llama.generate: prefix-match hit\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Response: Study PDC000251 is a proteomics study published in the Proteomics Data Commons (PDC). The study was conducted by researchers at the University of California, San Francisco (UCSF) and aimed to identify protein biomarkers for cancer diagnosis.\n",
            "\n",
            "The study used a combination of mass spectrometry-based proteomics and bioinformatics tools to analyze the protein profiles of cancer tissues and matched normal tissues. The results showed that several proteins were differentially expressed between cancer and normal tissues, suggesting potential biomarkers for cancer diagnosis.\n",
            "\n",
            "Overall, Study PDC000251 provides valuable insights into the proteomic changes associated with cancer development and progression. (Source: Proteomics Data Commons)\n",
            "---\n",
            "Question: Provide studies linked with gastric cancer.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "\n",
            "llama_print_timings:        load time =   10954.98 ms\n",
            "llama_print_timings:      sample time =     388.55 ms /   142 runs   (    2.74 ms per token,   365.46 tokens per second)\n",
            "llama_print_timings: prompt eval time =  339215.95 ms /   603 tokens (  562.55 ms per token,     1.78 tokens per second)\n",
            "llama_print_timings:        eval time =  149002.77 ms /   141 runs   ( 1056.76 ms per token,     0.95 tokens per second)\n",
            "llama_print_timings:       total time =  489043.66 ms /   744 tokens\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Response: I'm happy to help! Based on your request, it seems like you're looking for studies related to gastric cancer.\n",
            "\n",
            "Here are some relevant studies that might be of interest:\n",
            "\n",
            "1. Study ID: PDC000214, Title: Proteogenomic Analysis of Diffuse Gastric Cancers.\n",
            "2. Study ID: PDC000216, Title: Proteogenomic Analysis of Gastric Cancer.\n",
            "3. Study ID: PDC000215, Title: Proteogenomic Analysis of Gastric Cancer.\n",
            "\n",
            "These studies provide insights into the molecular mechanisms underlying gastric cancer development and progression.\n",
            "\n",
            "Please let me know if you have any further questions or if there's anything else I can assist you with!\n",
            "---\n"
          ]
        }
      ]
    }
  ],
  "metadata": {
    "colab": {
      "provenance": []
    },
    "kernelspec": {
      "display_name": "Python 3",
      "name": "python3"
    },
    "language_info": {
      "name": "python"
    },
    "widgets": {
      "application/vnd.jupyter.widget-state+json": {
        "5ec8e1a6e1284e70a08e5bee99246d93": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HBoxModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HBoxModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HBoxView",
            "box_style": "",
            "children": [
              "IPY_MODEL_7caff46ba1a44fdbbda0526a4446066b",
              "IPY_MODEL_62b9bd5f6f75432985482593074f576b",
              "IPY_MODEL_87c8b42a2bde46eaba1323d574ecf952"
            ],
            "layout": "IPY_MODEL_f98ae7514faa4c6fb86d4bdb7002fc03"
          }
        },
        "7caff46ba1a44fdbbda0526a4446066b": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_17a64412b6324428b48a271d92e4bf90",
            "placeholder": "​",
            "style": "IPY_MODEL_32dfff721ff5472eb1531e8af230aef3",
            "value": "tokenizer_config.json: 100%"
          }
        },
        "62b9bd5f6f75432985482593074f576b": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "FloatProgressModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "FloatProgressModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "ProgressView",
            "bar_style": "success",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_c6e1cca880d24ec5a92d369ba6ccdcb1",
            "max": 350,
            "min": 0,
            "orientation": "horizontal",
            "style": "IPY_MODEL_6d275cd58fb4455d9ddcd76f2a80fa7e",
            "value": 350
          }
        },
        "87c8b42a2bde46eaba1323d574ecf952": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_1fb2cda0de70436f93f563099bd03e45",
            "placeholder": "​",
            "style": "IPY_MODEL_fa6760ba8dec4bd7a23bffce6ee4f52c",
            "value": " 350/350 [00:00&lt;00:00, 8.87kB/s]"
          }
        },
        "f98ae7514faa4c6fb86d4bdb7002fc03": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "17a64412b6324428b48a271d92e4bf90": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "32dfff721ff5472eb1531e8af230aef3": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "c6e1cca880d24ec5a92d369ba6ccdcb1": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "6d275cd58fb4455d9ddcd76f2a80fa7e": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ProgressStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ProgressStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "bar_color": null,
            "description_width": ""
          }
        },
        "1fb2cda0de70436f93f563099bd03e45": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "fa6760ba8dec4bd7a23bffce6ee4f52c": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "9d5e46a422e74e2ebeabd37214cd9da9": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HBoxModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HBoxModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HBoxView",
            "box_style": "",
            "children": [
              "IPY_MODEL_29a9dbc295484540b475bf496aeb7e16",
              "IPY_MODEL_7363e379219f4e948e3a7e04d8c938ac",
              "IPY_MODEL_4d33c4b2de30400c97fb10d933c52c99"
            ],
            "layout": "IPY_MODEL_693dad55763340899e8b87fb38b63da4"
          }
        },
        "29a9dbc295484540b475bf496aeb7e16": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_c6a11961669443ad8d15f4a92baa23a8",
            "placeholder": "​",
            "style": "IPY_MODEL_0b4f3abe0e3749a695ab5407b96d0cb2",
            "value": "vocab.txt: 100%"
          }
        },
        "7363e379219f4e948e3a7e04d8c938ac": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "FloatProgressModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "FloatProgressModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "ProgressView",
            "bar_style": "success",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_7e070566c44f409c80c9371551a18fff",
            "max": 231508,
            "min": 0,
            "orientation": "horizontal",
            "style": "IPY_MODEL_c4dd7e98e36447c2b42fc4ce5fb3f0fe",
            "value": 231508
          }
        },
        "4d33c4b2de30400c97fb10d933c52c99": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_3f9f711504d54766b04df13907416358",
            "placeholder": "​",
            "style": "IPY_MODEL_4fb04110dfb24eabb2e415b88dc9e99b",
            "value": " 232k/232k [00:00&lt;00:00, 4.01MB/s]"
          }
        },
        "693dad55763340899e8b87fb38b63da4": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "c6a11961669443ad8d15f4a92baa23a8": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "0b4f3abe0e3749a695ab5407b96d0cb2": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "7e070566c44f409c80c9371551a18fff": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "c4dd7e98e36447c2b42fc4ce5fb3f0fe": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ProgressStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ProgressStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "bar_color": null,
            "description_width": ""
          }
        },
        "3f9f711504d54766b04df13907416358": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "4fb04110dfb24eabb2e415b88dc9e99b": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "afd7ef7693414cbc9ef31f2c01cf3338": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HBoxModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HBoxModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HBoxView",
            "box_style": "",
            "children": [
              "IPY_MODEL_fce47e32606a4f5885f547f7a6e7fc91",
              "IPY_MODEL_50130f71472f46bb8a17c7bbe263333b",
              "IPY_MODEL_a34999e8265e45b481a9f21e236e015a"
            ],
            "layout": "IPY_MODEL_bd0cdf4a88c34863a468437fa3a12b46"
          }
        },
        "fce47e32606a4f5885f547f7a6e7fc91": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_e06537ef6cb3429aac69bdd1bde4c3ce",
            "placeholder": "​",
            "style": "IPY_MODEL_cea5fd3a576b4a899baeaa785bbe61ba",
            "value": "tokenizer.json: 100%"
          }
        },
        "50130f71472f46bb8a17c7bbe263333b": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "FloatProgressModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "FloatProgressModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "ProgressView",
            "bar_style": "success",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_629956ba73a949b4b4ddd5f3d8215f54",
            "max": 466247,
            "min": 0,
            "orientation": "horizontal",
            "style": "IPY_MODEL_6f7e87a6b53e43c6a211d2908a3e5ca0",
            "value": 466247
          }
        },
        "a34999e8265e45b481a9f21e236e015a": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_0ed8a6c2c3f5444ba4a68fd31dad37f9",
            "placeholder": "​",
            "style": "IPY_MODEL_16de9dd8747d4bafa995463659701758",
            "value": " 466k/466k [00:00&lt;00:00, 14.6MB/s]"
          }
        },
        "bd0cdf4a88c34863a468437fa3a12b46": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "e06537ef6cb3429aac69bdd1bde4c3ce": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "cea5fd3a576b4a899baeaa785bbe61ba": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "629956ba73a949b4b4ddd5f3d8215f54": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "6f7e87a6b53e43c6a211d2908a3e5ca0": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ProgressStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ProgressStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "bar_color": null,
            "description_width": ""
          }
        },
        "0ed8a6c2c3f5444ba4a68fd31dad37f9": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "16de9dd8747d4bafa995463659701758": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "95edc7a7720e43fa82f91c2a6488a182": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HBoxModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HBoxModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HBoxView",
            "box_style": "",
            "children": [
              "IPY_MODEL_9b63b5e556df42dc9340efd9b97f65a8",
              "IPY_MODEL_149ba45f3956453bbbd18f2403265722",
              "IPY_MODEL_83ca301cb83f41008115bb9aabd093a5"
            ],
            "layout": "IPY_MODEL_88fade17c0cd43688dabd5165abd2ba3"
          }
        },
        "9b63b5e556df42dc9340efd9b97f65a8": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_0c6e7f85e9f04695a0d490ee069726cc",
            "placeholder": "​",
            "style": "IPY_MODEL_8696e33ff081426382179869af171582",
            "value": "special_tokens_map.json: 100%"
          }
        },
        "149ba45f3956453bbbd18f2403265722": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "FloatProgressModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "FloatProgressModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "ProgressView",
            "bar_style": "success",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_a8e8ecfe0cd5454cb7442092cb140b6e",
            "max": 112,
            "min": 0,
            "orientation": "horizontal",
            "style": "IPY_MODEL_06bf883790c34ea19539f2c69cd83f72",
            "value": 112
          }
        },
        "83ca301cb83f41008115bb9aabd093a5": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_a5bc2f222eaa4a1cb5fbc6d1357d3753",
            "placeholder": "​",
            "style": "IPY_MODEL_f4456a291e534e8385ecb61edc70670d",
            "value": " 112/112 [00:00&lt;00:00, 2.80kB/s]"
          }
        },
        "88fade17c0cd43688dabd5165abd2ba3": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "0c6e7f85e9f04695a0d490ee069726cc": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "8696e33ff081426382179869af171582": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "a8e8ecfe0cd5454cb7442092cb140b6e": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "06bf883790c34ea19539f2c69cd83f72": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ProgressStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ProgressStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "bar_color": null,
            "description_width": ""
          }
        },
        "a5bc2f222eaa4a1cb5fbc6d1357d3753": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "f4456a291e534e8385ecb61edc70670d": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "0c911f51644541cba54018da18275f2f": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HBoxModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HBoxModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HBoxView",
            "box_style": "",
            "children": [
              "IPY_MODEL_c13670e29d9b4930b5bc225bf7f7b71a",
              "IPY_MODEL_91ece160ff7041308c0623bc6a9b76ed",
              "IPY_MODEL_a87fe9ee202d44ebba67815a4a6b2629"
            ],
            "layout": "IPY_MODEL_a1aed037af964ef18a0f3fa3941ee437"
          }
        },
        "c13670e29d9b4930b5bc225bf7f7b71a": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_546ac419f6754e27abdb734d3b6b7843",
            "placeholder": "​",
            "style": "IPY_MODEL_e562edf27cfc4efb94aed410cb4d3aa1",
            "value": "config.json: 100%"
          }
        },
        "91ece160ff7041308c0623bc6a9b76ed": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "FloatProgressModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "FloatProgressModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "ProgressView",
            "bar_style": "success",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_da0ce35c16e14fd7a85978cade3e3e84",
            "max": 612,
            "min": 0,
            "orientation": "horizontal",
            "style": "IPY_MODEL_fa60a789763142ec80d13a9a184a5841",
            "value": 612
          }
        },
        "a87fe9ee202d44ebba67815a4a6b2629": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_d82f08f51c9e46aeaa00d48bbcb2de06",
            "placeholder": "​",
            "style": "IPY_MODEL_dd837693968a4efba11d1b7fef0c93ea",
            "value": " 612/612 [00:00&lt;00:00, 24.4kB/s]"
          }
        },
        "a1aed037af964ef18a0f3fa3941ee437": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "546ac419f6754e27abdb734d3b6b7843": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "e562edf27cfc4efb94aed410cb4d3aa1": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "da0ce35c16e14fd7a85978cade3e3e84": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "fa60a789763142ec80d13a9a184a5841": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ProgressStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ProgressStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "bar_color": null,
            "description_width": ""
          }
        },
        "d82f08f51c9e46aeaa00d48bbcb2de06": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "dd837693968a4efba11d1b7fef0c93ea": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "f803e56bf9e94430bb0839408ee34fb9": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HBoxModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HBoxModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HBoxView",
            "box_style": "",
            "children": [
              "IPY_MODEL_fa0c6769c333436a900c924edd7f9229",
              "IPY_MODEL_2974f5e504b14a5cb981ba11a6e672a3",
              "IPY_MODEL_6db2234ab60244118e48fddcab425857"
            ],
            "layout": "IPY_MODEL_ed885ea3b46b471b81a19362935fc331"
          }
        },
        "fa0c6769c333436a900c924edd7f9229": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_d694c3c2ea204e79b6bd690ea6ae77e8",
            "placeholder": "​",
            "style": "IPY_MODEL_e0eda87d9cb84e64b1fda056e980f022",
            "value": "model.safetensors: 100%"
          }
        },
        "2974f5e504b14a5cb981ba11a6e672a3": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "FloatProgressModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "FloatProgressModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "ProgressView",
            "bar_style": "success",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_0d44242851ca4036823e8873ef5569fe",
            "max": 90868376,
            "min": 0,
            "orientation": "horizontal",
            "style": "IPY_MODEL_e00eedf0d25b42ed875ad8ad4e653800",
            "value": 90868376
          }
        },
        "6db2234ab60244118e48fddcab425857": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_6bf0bfd11902470ba49568d2421a5abd",
            "placeholder": "​",
            "style": "IPY_MODEL_1c586a9d27ac472cbfb98b69512a71cb",
            "value": " 90.9M/90.9M [00:01&lt;00:00, 108MB/s]"
          }
        },
        "ed885ea3b46b471b81a19362935fc331": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "d694c3c2ea204e79b6bd690ea6ae77e8": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "e0eda87d9cb84e64b1fda056e980f022": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "0d44242851ca4036823e8873ef5569fe": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "e00eedf0d25b42ed875ad8ad4e653800": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ProgressStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ProgressStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "bar_color": null,
            "description_width": ""
          }
        },
        "6bf0bfd11902470ba49568d2421a5abd": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "1c586a9d27ac472cbfb98b69512a71cb": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        }
      }
    }
  },
  "nbformat": 4,
  "nbformat_minor": 0
}