{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "provenance": [],
      "toc_visible": true,
      "authorship_tag": "ABX9TyPUHN/nwUKp8N++JomoX0An",
      "include_colab_link": true
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "language_info": {
      "name": "python"
    },
    "widgets": {
      "application/vnd.jupyter.widget-state+json": {
        "f2effa109824477394e9cdf4d02e2ef8": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HBoxModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HBoxModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HBoxView",
            "box_style": "",
            "children": [
              "IPY_MODEL_f493e172d62449a0af6d6e17356577ee",
              "IPY_MODEL_7c89b29f901f40afa432f2a0a0ee2b7f",
              "IPY_MODEL_8bff5b1a19dd4457a6740ab1ee3d88c4"
            ],
            "layout": "IPY_MODEL_f82f2ddf2eba4e319186fa015fbf909d"
          }
        },
        "f493e172d62449a0af6d6e17356577ee": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_79c2349768134eab82edecef1b81d032",
            "placeholder": "​",
            "style": "IPY_MODEL_86c5e01197c4457491a236cc6136a54a",
            "value": "tokenizer_config.json: 100%"
          }
        },
        "7c89b29f901f40afa432f2a0a0ee2b7f": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "FloatProgressModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "FloatProgressModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "ProgressView",
            "bar_style": "success",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_a4af9a54a0384db7b2265ff701b71a65",
            "max": 28,
            "min": 0,
            "orientation": "horizontal",
            "style": "IPY_MODEL_f18699c03dc3448d82e48bd94131970e",
            "value": 28
          }
        },
        "8bff5b1a19dd4457a6740ab1ee3d88c4": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_7716bd03cb844d37a3947921a045b8c6",
            "placeholder": "​",
            "style": "IPY_MODEL_cb4d8a8e54dd4275bb3a027ddb31fff1",
            "value": " 28.0/28.0 [00:00&lt;00:00, 1.69kB/s]"
          }
        },
        "f82f2ddf2eba4e319186fa015fbf909d": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "79c2349768134eab82edecef1b81d032": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "86c5e01197c4457491a236cc6136a54a": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "a4af9a54a0384db7b2265ff701b71a65": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "f18699c03dc3448d82e48bd94131970e": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ProgressStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ProgressStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "bar_color": null,
            "description_width": ""
          }
        },
        "7716bd03cb844d37a3947921a045b8c6": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "cb4d8a8e54dd4275bb3a027ddb31fff1": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "81d50ba5d6424cb7b548bf0d747b386c": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HBoxModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HBoxModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HBoxView",
            "box_style": "",
            "children": [
              "IPY_MODEL_ff1e48f3bdc548bd8b0dc80df657287d",
              "IPY_MODEL_4da8303f8a8c4d6f98cfb8aa44fbf197",
              "IPY_MODEL_cdf9c1546be84fe387cf49e306f1aaaa"
            ],
            "layout": "IPY_MODEL_0f9a4f95f0884cd58f6b9f1cdf6f01a2"
          }
        },
        "ff1e48f3bdc548bd8b0dc80df657287d": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_e94c2a2991884a82b75425d5fe15c0a4",
            "placeholder": "​",
            "style": "IPY_MODEL_fa22c8f44d0646bbb568371809df90af",
            "value": "config.json: 100%"
          }
        },
        "4da8303f8a8c4d6f98cfb8aa44fbf197": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "FloatProgressModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "FloatProgressModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "ProgressView",
            "bar_style": "success",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_47f966d5dcb64dd7a26d077823c9d10e",
            "max": 385,
            "min": 0,
            "orientation": "horizontal",
            "style": "IPY_MODEL_b7cb1815160744afa231f61acccd1280",
            "value": 385
          }
        },
        "cdf9c1546be84fe387cf49e306f1aaaa": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_69204280d0bd4a9a8068597343386839",
            "placeholder": "​",
            "style": "IPY_MODEL_c0254ede3b1e4efb8978924de358ac4c",
            "value": " 385/385 [00:00&lt;00:00, 22.0kB/s]"
          }
        },
        "0f9a4f95f0884cd58f6b9f1cdf6f01a2": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "e94c2a2991884a82b75425d5fe15c0a4": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "fa22c8f44d0646bbb568371809df90af": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "47f966d5dcb64dd7a26d077823c9d10e": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "b7cb1815160744afa231f61acccd1280": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ProgressStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ProgressStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "bar_color": null,
            "description_width": ""
          }
        },
        "69204280d0bd4a9a8068597343386839": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "c0254ede3b1e4efb8978924de358ac4c": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "b93f7d774e044573a89e819eb76fe92a": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HBoxModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HBoxModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HBoxView",
            "box_style": "",
            "children": [
              "IPY_MODEL_e6ff09e32fd941dbac5cbf819a79c6b7",
              "IPY_MODEL_2d71c1900ead44fd89e3fd89dd7040ec",
              "IPY_MODEL_e1124d2f9264473d88768b3d9a5c7e83"
            ],
            "layout": "IPY_MODEL_aa6e53eb2868430d9c7df06c256265a4"
          }
        },
        "e6ff09e32fd941dbac5cbf819a79c6b7": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_7f42e38709fc435eb08c67af1ca811ca",
            "placeholder": "​",
            "style": "IPY_MODEL_7f5cc813c63549fa8444bea0e370096e",
            "value": "vocab.txt: 100%"
          }
        },
        "2d71c1900ead44fd89e3fd89dd7040ec": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "FloatProgressModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "FloatProgressModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "ProgressView",
            "bar_style": "success",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_647606068f9b4a55872fe5b89dc499b2",
            "max": 226150,
            "min": 0,
            "orientation": "horizontal",
            "style": "IPY_MODEL_cdfadf7836194184aa2b3bb5b2fb9251",
            "value": 226150
          }
        },
        "e1124d2f9264473d88768b3d9a5c7e83": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_6cd91df2cd5342249dd3952c7df21c08",
            "placeholder": "​",
            "style": "IPY_MODEL_cd5460bedbd34255bb38fd472a9ad1cb",
            "value": " 226k/226k [00:00&lt;00:00, 3.60MB/s]"
          }
        },
        "aa6e53eb2868430d9c7df06c256265a4": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "7f42e38709fc435eb08c67af1ca811ca": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "7f5cc813c63549fa8444bea0e370096e": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "647606068f9b4a55872fe5b89dc499b2": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "cdfadf7836194184aa2b3bb5b2fb9251": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ProgressStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ProgressStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "bar_color": null,
            "description_width": ""
          }
        },
        "6cd91df2cd5342249dd3952c7df21c08": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "cd5460bedbd34255bb38fd472a9ad1cb": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "6a31bd80d106457f9fb9815c4abfacab": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HBoxModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HBoxModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HBoxView",
            "box_style": "",
            "children": [
              "IPY_MODEL_ffa3da38deef4d69bfd7568549ec448d",
              "IPY_MODEL_0c3e8f1d60ad4e14a8f2fadc29bf7c4a",
              "IPY_MODEL_8810615977d44056baeae312b2cb3949"
            ],
            "layout": "IPY_MODEL_ee74b2ffd9e643888b3cd6edc291091a"
          }
        },
        "ffa3da38deef4d69bfd7568549ec448d": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_52eec76de628416daf23b5a604fe23e5",
            "placeholder": "​",
            "style": "IPY_MODEL_66a53cf98b9c418598807707caf26427",
            "value": "pytorch_model.bin: 100%"
          }
        },
        "0c3e8f1d60ad4e14a8f2fadc29bf7c4a": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "FloatProgressModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "FloatProgressModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "ProgressView",
            "bar_style": "success",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_ef5251e61e21485c8043d80127de7503",
            "max": 440474434,
            "min": 0,
            "orientation": "horizontal",
            "style": "IPY_MODEL_c14aaaa4a7bb49cfaa9907b0c12a41dd",
            "value": 440474434
          }
        },
        "8810615977d44056baeae312b2cb3949": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_3c6a08489c164ef4ae3e89f24d8b4373",
            "placeholder": "​",
            "style": "IPY_MODEL_c7b2a9fc66f5415e89f82bb5b21f02fe",
            "value": " 440M/440M [00:02&lt;00:00, 236MB/s]"
          }
        },
        "ee74b2ffd9e643888b3cd6edc291091a": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "52eec76de628416daf23b5a604fe23e5": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "66a53cf98b9c418598807707caf26427": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "ef5251e61e21485c8043d80127de7503": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "c14aaaa4a7bb49cfaa9907b0c12a41dd": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ProgressStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ProgressStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "bar_color": null,
            "description_width": ""
          }
        },
        "3c6a08489c164ef4ae3e89f24d8b4373": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "c7b2a9fc66f5415e89f82bb5b21f02fe": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        }
      }
    }
  },
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "view-in-github",
        "colab_type": "text"
      },
      "source": [
        "<a href=\"https://colab.research.google.com/github/hansheng-hsu/AI-chatbot-for-PDC/blob/main/RAG_with_full_JSON_20240728.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "# Load library"
      ],
      "metadata": {
        "id": "ohgrjdAeUxi_"
      }
    },
    {
      "cell_type": "code",
      "execution_count": 1,
      "metadata": {
        "id": "jPsv9xnHNzR3"
      },
      "outputs": [],
      "source": [
        "#connect to google drive\n",
        "from google.colab import drive"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 2,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "CIiVtQ17N1Pq",
        "outputId": "5bfd6e98-c387-49b8-ad2c-a9e4d917f8a6"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Mounted at /content/drive\n"
          ]
        }
      ],
      "source": [
        "drive.mount('/content/drive')"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 3,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "9AfuwKZebQ0p",
        "outputId": "5712f600-c94b-4478-f58a-2e9ebfd9ed51"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Collecting langchain\n",
            "  Downloading langchain-0.2.11-py3-none-any.whl.metadata (7.1 kB)\n",
            "Collecting weaviate_client\n",
            "  Downloading weaviate_client-4.7.1-py3-none-any.whl.metadata (3.3 kB)\n",
            "Requirement already satisfied: wasabi in /usr/local/lib/python3.10/dist-packages (1.1.3)\n",
            "Collecting langchain_community\n",
            "  Downloading langchain_community-0.2.10-py3-none-any.whl.metadata (2.7 kB)\n",
            "Collecting llama-cpp-python\n",
            "  Downloading llama_cpp_python-0.2.85.tar.gz (49.3 MB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m49.3/49.3 MB\u001b[0m \u001b[31m18.1 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25h  Installing build dependencies ... \u001b[?25l\u001b[?25hdone\n",
            "  Getting requirements to build wheel ... \u001b[?25l\u001b[?25hdone\n",
            "  Installing backend dependencies ... \u001b[?25l\u001b[?25hdone\n",
            "  Preparing metadata (pyproject.toml) ... \u001b[?25l\u001b[?25hdone\n",
            "Collecting langchain_weaviate\n",
            "  Downloading langchain_weaviate-0.0.2-py3-none-any.whl.metadata (2.7 kB)\n",
            "Requirement already satisfied: PyYAML>=5.3 in /usr/local/lib/python3.10/dist-packages (from langchain) (6.0.1)\n",
            "Requirement already satisfied: SQLAlchemy<3,>=1.4 in /usr/local/lib/python3.10/dist-packages (from langchain) (2.0.31)\n",
            "Requirement already satisfied: aiohttp<4.0.0,>=3.8.3 in /usr/local/lib/python3.10/dist-packages (from langchain) (3.9.5)\n",
            "Requirement already satisfied: async-timeout<5.0.0,>=4.0.0 in /usr/local/lib/python3.10/dist-packages (from langchain) (4.0.3)\n",
            "Collecting langchain-core<0.3.0,>=0.2.23 (from langchain)\n",
            "  Downloading langchain_core-0.2.26-py3-none-any.whl.metadata (6.2 kB)\n",
            "Collecting langchain-text-splitters<0.3.0,>=0.2.0 (from langchain)\n",
            "  Downloading langchain_text_splitters-0.2.2-py3-none-any.whl.metadata (2.1 kB)\n",
            "Collecting langsmith<0.2.0,>=0.1.17 (from langchain)\n",
            "  Downloading langsmith-0.1.95-py3-none-any.whl.metadata (13 kB)\n",
            "Requirement already satisfied: numpy<2,>=1 in /usr/local/lib/python3.10/dist-packages (from langchain) (1.26.4)\n",
            "Requirement already satisfied: pydantic<3,>=1 in /usr/local/lib/python3.10/dist-packages (from langchain) (2.8.2)\n",
            "Requirement already satisfied: requests<3,>=2 in /usr/local/lib/python3.10/dist-packages (from langchain) (2.31.0)\n",
            "Collecting tenacity!=8.4.0,<9.0.0,>=8.1.0 (from langchain)\n",
            "  Downloading tenacity-8.5.0-py3-none-any.whl.metadata (1.2 kB)\n",
            "Collecting httpx<=0.27.0,>=0.25.0 (from weaviate_client)\n",
            "  Downloading httpx-0.27.0-py3-none-any.whl.metadata (7.2 kB)\n",
            "Collecting validators==0.33.0 (from weaviate_client)\n",
            "  Downloading validators-0.33.0-py3-none-any.whl.metadata (3.8 kB)\n",
            "Collecting authlib<2.0.0,>=1.2.1 (from weaviate_client)\n",
            "  Downloading Authlib-1.3.1-py2.py3-none-any.whl.metadata (3.8 kB)\n",
            "Requirement already satisfied: grpcio<2.0.0,>=1.57.0 in /usr/local/lib/python3.10/dist-packages (from weaviate_client) (1.64.1)\n",
            "Collecting grpcio-tools<2.0.0,>=1.57.0 (from weaviate_client)\n",
            "  Downloading grpcio_tools-1.65.2-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (5.3 kB)\n",
            "Collecting grpcio-health-checking<2.0.0,>=1.57.0 (from weaviate_client)\n",
            "  Downloading grpcio_health_checking-1.65.2-py3-none-any.whl.metadata (1.1 kB)\n",
            "Collecting dataclasses-json<0.7,>=0.5.7 (from langchain_community)\n",
            "  Downloading dataclasses_json-0.6.7-py3-none-any.whl.metadata (25 kB)\n",
            "Requirement already satisfied: typing-extensions>=4.5.0 in /usr/local/lib/python3.10/dist-packages (from llama-cpp-python) (4.12.2)\n",
            "Collecting diskcache>=5.6.1 (from llama-cpp-python)\n",
            "  Downloading diskcache-5.6.3-py3-none-any.whl.metadata (20 kB)\n",
            "Requirement already satisfied: jinja2>=2.11.3 in /usr/local/lib/python3.10/dist-packages (from llama-cpp-python) (3.1.4)\n",
            "Collecting simsimd<5.0.0,>=3.6.1 (from langchain_weaviate)\n",
            "  Downloading simsimd-4.4.0-cp310-cp310-manylinux_2_28_x86_64.whl.metadata (31 kB)\n",
            "Requirement already satisfied: aiosignal>=1.1.2 in /usr/local/lib/python3.10/dist-packages (from aiohttp<4.0.0,>=3.8.3->langchain) (1.3.1)\n",
            "Requirement already satisfied: attrs>=17.3.0 in /usr/local/lib/python3.10/dist-packages (from aiohttp<4.0.0,>=3.8.3->langchain) (23.2.0)\n",
            "Requirement already satisfied: frozenlist>=1.1.1 in /usr/local/lib/python3.10/dist-packages (from aiohttp<4.0.0,>=3.8.3->langchain) (1.4.1)\n",
            "Requirement already satisfied: multidict<7.0,>=4.5 in /usr/local/lib/python3.10/dist-packages (from aiohttp<4.0.0,>=3.8.3->langchain) (6.0.5)\n",
            "Requirement already satisfied: yarl<2.0,>=1.0 in /usr/local/lib/python3.10/dist-packages (from aiohttp<4.0.0,>=3.8.3->langchain) (1.9.4)\n",
            "Requirement already satisfied: cryptography in /usr/local/lib/python3.10/dist-packages (from authlib<2.0.0,>=1.2.1->weaviate_client) (42.0.8)\n",
            "Collecting marshmallow<4.0.0,>=3.18.0 (from dataclasses-json<0.7,>=0.5.7->langchain_community)\n",
            "  Downloading marshmallow-3.21.3-py3-none-any.whl.metadata (7.1 kB)\n",
            "Collecting typing-inspect<1,>=0.4.0 (from dataclasses-json<0.7,>=0.5.7->langchain_community)\n",
            "  Downloading typing_inspect-0.9.0-py3-none-any.whl.metadata (1.5 kB)\n",
            "Collecting protobuf<6.0dev,>=5.26.1 (from grpcio-health-checking<2.0.0,>=1.57.0->weaviate_client)\n",
            "  Downloading protobuf-5.27.3-cp38-abi3-manylinux2014_x86_64.whl.metadata (592 bytes)\n",
            "Collecting grpcio<2.0.0,>=1.57.0 (from weaviate_client)\n",
            "  Downloading grpcio-1.65.2-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (3.3 kB)\n",
            "Requirement already satisfied: setuptools in /usr/local/lib/python3.10/dist-packages (from grpcio-tools<2.0.0,>=1.57.0->weaviate_client) (71.0.4)\n",
            "Requirement already satisfied: anyio in /usr/local/lib/python3.10/dist-packages (from httpx<=0.27.0,>=0.25.0->weaviate_client) (3.7.1)\n",
            "Requirement already satisfied: certifi in /usr/local/lib/python3.10/dist-packages (from httpx<=0.27.0,>=0.25.0->weaviate_client) (2024.7.4)\n",
            "Collecting httpcore==1.* (from httpx<=0.27.0,>=0.25.0->weaviate_client)\n",
            "  Downloading httpcore-1.0.5-py3-none-any.whl.metadata (20 kB)\n",
            "Requirement already satisfied: idna in /usr/local/lib/python3.10/dist-packages (from httpx<=0.27.0,>=0.25.0->weaviate_client) (3.7)\n",
            "Requirement already satisfied: sniffio in /usr/local/lib/python3.10/dist-packages (from httpx<=0.27.0,>=0.25.0->weaviate_client) (1.3.1)\n",
            "Collecting h11<0.15,>=0.13 (from httpcore==1.*->httpx<=0.27.0,>=0.25.0->weaviate_client)\n",
            "  Downloading h11-0.14.0-py3-none-any.whl.metadata (8.2 kB)\n",
            "Requirement already satisfied: MarkupSafe>=2.0 in /usr/local/lib/python3.10/dist-packages (from jinja2>=2.11.3->llama-cpp-python) (2.1.5)\n",
            "Collecting jsonpatch<2.0,>=1.33 (from langchain-core<0.3.0,>=0.2.23->langchain)\n",
            "  Downloading jsonpatch-1.33-py2.py3-none-any.whl.metadata (3.0 kB)\n",
            "Requirement already satisfied: packaging<25,>=23.2 in /usr/local/lib/python3.10/dist-packages (from langchain-core<0.3.0,>=0.2.23->langchain) (24.1)\n",
            "Collecting orjson<4.0.0,>=3.9.14 (from langsmith<0.2.0,>=0.1.17->langchain)\n",
            "  Downloading orjson-3.10.6-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (50 kB)\n",
            "\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m50.4/50.4 kB\u001b[0m \u001b[31m3.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hRequirement already satisfied: annotated-types>=0.4.0 in /usr/local/lib/python3.10/dist-packages (from pydantic<3,>=1->langchain) (0.7.0)\n",
            "Requirement already satisfied: pydantic-core==2.20.1 in /usr/local/lib/python3.10/dist-packages (from pydantic<3,>=1->langchain) (2.20.1)\n",
            "Requirement already satisfied: charset-normalizer<4,>=2 in /usr/local/lib/python3.10/dist-packages (from requests<3,>=2->langchain) (3.3.2)\n",
            "Requirement already satisfied: urllib3<3,>=1.21.1 in /usr/local/lib/python3.10/dist-packages (from requests<3,>=2->langchain) (2.0.7)\n",
            "Requirement already satisfied: greenlet!=0.4.17 in /usr/local/lib/python3.10/dist-packages (from SQLAlchemy<3,>=1.4->langchain) (3.0.3)\n",
            "Collecting jsonpointer>=1.9 (from jsonpatch<2.0,>=1.33->langchain-core<0.3.0,>=0.2.23->langchain)\n",
            "  Downloading jsonpointer-3.0.0-py2.py3-none-any.whl.metadata (2.3 kB)\n",
            "Collecting mypy-extensions>=0.3.0 (from typing-inspect<1,>=0.4.0->dataclasses-json<0.7,>=0.5.7->langchain_community)\n",
            "  Downloading mypy_extensions-1.0.0-py3-none-any.whl.metadata (1.1 kB)\n",
            "Requirement already satisfied: exceptiongroup in /usr/local/lib/python3.10/dist-packages (from anyio->httpx<=0.27.0,>=0.25.0->weaviate_client) (1.2.2)\n",
            "Requirement already satisfied: cffi>=1.12 in /usr/local/lib/python3.10/dist-packages (from cryptography->authlib<2.0.0,>=1.2.1->weaviate_client) (1.16.0)\n",
            "Requirement already satisfied: pycparser in /usr/local/lib/python3.10/dist-packages (from cffi>=1.12->cryptography->authlib<2.0.0,>=1.2.1->weaviate_client) (2.22)\n",
            "Downloading langchain-0.2.11-py3-none-any.whl (990 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m990.3/990.3 kB\u001b[0m \u001b[31m44.6 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading weaviate_client-4.7.1-py3-none-any.whl (368 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m368.3/368.3 kB\u001b[0m \u001b[31m22.8 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading validators-0.33.0-py3-none-any.whl (43 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m43.3/43.3 kB\u001b[0m \u001b[31m3.0 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading langchain_community-0.2.10-py3-none-any.whl (2.3 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m2.3/2.3 MB\u001b[0m \u001b[31m64.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading langchain_weaviate-0.0.2-py3-none-any.whl (10 kB)\n",
            "Downloading Authlib-1.3.1-py2.py3-none-any.whl (223 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m223.8/223.8 kB\u001b[0m \u001b[31m14.4 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading dataclasses_json-0.6.7-py3-none-any.whl (28 kB)\n",
            "Downloading diskcache-5.6.3-py3-none-any.whl (45 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m45.5/45.5 kB\u001b[0m \u001b[31m3.1 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading grpcio_health_checking-1.65.2-py3-none-any.whl (18 kB)\n",
            "Downloading grpcio-1.65.2-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (5.7 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m5.7/5.7 MB\u001b[0m \u001b[31m88.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading grpcio_tools-1.65.2-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (2.3 MB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m2.3/2.3 MB\u001b[0m \u001b[31m61.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading httpx-0.27.0-py3-none-any.whl (75 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m75.6/75.6 kB\u001b[0m \u001b[31m5.1 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading httpcore-1.0.5-py3-none-any.whl (77 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m77.9/77.9 kB\u001b[0m \u001b[31m5.8 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading langchain_core-0.2.26-py3-none-any.whl (378 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m378.9/378.9 kB\u001b[0m \u001b[31m23.9 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading langchain_text_splitters-0.2.2-py3-none-any.whl (25 kB)\n",
            "Downloading langsmith-0.1.95-py3-none-any.whl (275 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m275.8/275.8 kB\u001b[0m \u001b[31m18.3 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading simsimd-4.4.0-cp310-cp310-manylinux_2_28_x86_64.whl (519 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m519.1/519.1 kB\u001b[0m \u001b[31m27.5 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading tenacity-8.5.0-py3-none-any.whl (28 kB)\n",
            "Downloading jsonpatch-1.33-py2.py3-none-any.whl (12 kB)\n",
            "Downloading marshmallow-3.21.3-py3-none-any.whl (49 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m49.2/49.2 kB\u001b[0m \u001b[31m3.5 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading orjson-3.10.6-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (141 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m141.1/141.1 kB\u001b[0m \u001b[31m9.7 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading protobuf-5.27.3-cp38-abi3-manylinux2014_x86_64.whl (309 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m309.3/309.3 kB\u001b[0m \u001b[31m20.1 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading typing_inspect-0.9.0-py3-none-any.whl (8.8 kB)\n",
            "Downloading h11-0.14.0-py3-none-any.whl (58 kB)\n",
            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m58.3/58.3 kB\u001b[0m \u001b[31m4.1 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hDownloading jsonpointer-3.0.0-py2.py3-none-any.whl (7.6 kB)\n",
            "Downloading mypy_extensions-1.0.0-py3-none-any.whl (4.7 kB)\n",
            "Building wheels for collected packages: llama-cpp-python\n",
            "  Building wheel for llama-cpp-python (pyproject.toml) ... \u001b[?25l\u001b[?25hdone\n",
            "  Created wheel for llama-cpp-python: filename=llama_cpp_python-0.2.85-cp310-cp310-linux_x86_64.whl size=2857608 sha256=6f4fa3d305ba0b82d1e8fa2201f9ef0092e6f1332d8e3156dc3982dd2118d2cb\n",
            "  Stored in directory: /root/.cache/pip/wheels/3f/e8/4e/29a754f9175ef52b6481cd75e3af4de38bf6dfa9c2972f75d4\n",
            "Successfully built llama-cpp-python\n",
            "Installing collected packages: simsimd, validators, tenacity, protobuf, orjson, mypy-extensions, marshmallow, jsonpointer, h11, grpcio, diskcache, typing-inspect, llama-cpp-python, jsonpatch, httpcore, grpcio-tools, grpcio-health-checking, langsmith, httpx, dataclasses-json, authlib, weaviate_client, langchain-core, langchain_weaviate, langchain-text-splitters, langchain, langchain_community\n",
            "  Attempting uninstall: tenacity\n",
            "    Found existing installation: tenacity 9.0.0\n",
            "    Uninstalling tenacity-9.0.0:\n",
            "      Successfully uninstalled tenacity-9.0.0\n",
            "  Attempting uninstall: protobuf\n",
            "    Found existing installation: protobuf 3.20.3\n",
            "    Uninstalling protobuf-3.20.3:\n",
            "      Successfully uninstalled protobuf-3.20.3\n",
            "  Attempting uninstall: grpcio\n",
            "    Found existing installation: grpcio 1.64.1\n",
            "    Uninstalling grpcio-1.64.1:\n",
            "      Successfully uninstalled grpcio-1.64.1\n",
            "\u001b[31mERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.\n",
            "cudf-cu12 24.4.1 requires protobuf<5,>=3.20, but you have protobuf 5.27.3 which is incompatible.\n",
            "google-ai-generativelanguage 0.6.6 requires protobuf!=3.20.0,!=3.20.1,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5,<5.0.0dev,>=3.19.5, but you have protobuf 5.27.3 which is incompatible.\n",
            "google-cloud-aiplatform 1.59.0 requires protobuf!=3.20.0,!=3.20.1,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5,<5.0.0dev,>=3.19.5, but you have protobuf 5.27.3 which is incompatible.\n",
            "google-cloud-bigquery-storage 2.25.0 requires protobuf!=3.20.0,!=3.20.1,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5,<5.0.0dev,>=3.19.5, but you have protobuf 5.27.3 which is incompatible.\n",
            "google-cloud-datastore 2.19.0 requires protobuf!=3.20.0,!=3.20.1,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5,<5.0.0dev,>=3.19.5, but you have protobuf 5.27.3 which is incompatible.\n",
            "google-cloud-firestore 2.16.1 requires protobuf!=3.20.0,!=3.20.1,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5,<5.0.0dev,>=3.19.5, but you have protobuf 5.27.3 which is incompatible.\n",
            "tensorboard 2.17.0 requires protobuf!=4.24.0,<5.0.0,>=3.19.6, but you have protobuf 5.27.3 which is incompatible.\n",
            "tensorflow 2.17.0 requires protobuf!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5,<5.0.0dev,>=3.20.3, but you have protobuf 5.27.3 which is incompatible.\n",
            "tensorflow-metadata 1.15.0 requires protobuf<4.21,>=3.20.3; python_version < \"3.11\", but you have protobuf 5.27.3 which is incompatible.\u001b[0m\u001b[31m\n",
            "\u001b[0mSuccessfully installed authlib-1.3.1 dataclasses-json-0.6.7 diskcache-5.6.3 grpcio-1.65.2 grpcio-health-checking-1.65.2 grpcio-tools-1.65.2 h11-0.14.0 httpcore-1.0.5 httpx-0.27.0 jsonpatch-1.33 jsonpointer-3.0.0 langchain-0.2.11 langchain-core-0.2.26 langchain-text-splitters-0.2.2 langchain_community-0.2.10 langchain_weaviate-0.0.2 langsmith-0.1.95 llama-cpp-python-0.2.85 marshmallow-3.21.3 mypy-extensions-1.0.0 orjson-3.10.6 protobuf-5.27.3 simsimd-4.4.0 tenacity-8.5.0 typing-inspect-0.9.0 validators-0.33.0 weaviate_client-4.7.1\n"
          ]
        }
      ],
      "source": [
        "!pip install langchain weaviate_client wasabi langchain_community llama-cpp-python langchain_weaviate"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 4,
      "metadata": {
        "id": "Bgk79196bT-m"
      },
      "outputs": [],
      "source": [
        "import requests\n",
        "import json\n",
        "import weaviate\n",
        "from weaviate.embedded import EmbeddedOptions\n",
        "from wasabi import msg\n",
        "from langchain.text_splitter import RecursiveCharacterTextSplitter"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "h3VANGvSqQFM"
      },
      "source": [
        "# Fetch the Data"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 5,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "gIRQNsPGYRBq",
        "outputId": "24f93615-23dc-4878-b953-d04896193509"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[1;30;43m串流輸出內容已截斷至最後 5000 行。\u001b[0m\n",
            "          \"\\u03bcl of 50 mM HEPES (pH 8.5). Plate 10 \\u03bcl of the diluted sample in triplicate per sample in a clear, flat-bottom 96-well microplate. Plate a 10-\\u03bcl HPLC water blank and a 50 mM HEPES, pH 8.5, blank, each in triplicate. Plate eight wells of 10 \\u03bcl each of an\",\n",
            "          \"each in triplicate. Plate eight wells of 10 \\u03bcl each of an 8-point BSA curve, ranging from 50 \\u03bcg/ml to 2 mg/ml. Prepare the BCA reagent by mixing reagent B and reagent A in a 1:49 (vol/vol) ratio. \\u25b2 CRITICAL STEP This solution must be made fresh\",\n",
            "          \"ratio. \\u25b2 CRITICAL STEP This solution must be made fresh immediately before use. Add 200 \\u03bcl of freshly prepared BCA reagent to each well containing 10 \\u03bcl of sample. Incubate the microplate at 37 \\u00b0C for 30 min. Read the plate at 562 nm. We usually recover\",\n",
            "          \"37 \\u00b0C for 30 min. Read the plate at 562 nm. We usually recover 40\\u201350% of the starting material at this step. When using 1 mg of protein starting material, we usually recover 400\\u2013500 \\u03bcg of tryptic peptides. On the basis of the BCA results at the peptide\",\n",
            "          \"peptides. On the basis of the BCA results at the peptide level, dilute the samples with 50 mM HEPES (pH 8.5) for a concentration of 1 \\u03bcg/\\u03bcl. Vortex the well to mix. \\u25a0 PAUSE POINT Peptides in 50 mM HEPES (pH 8.5) can be stored in a \\u221280 \\u00b0C freezer space for\",\n",
            "          \"mM HEPES (pH 8.5) can be stored in a \\u221280 \\u00b0C freezer space for up to 1 week. TMT labeling of 300 \\u03bcg of peptide per channel \\u25cf Timing 90 min Remove 300 \\u03bcg of peptides, as measured by peptide-level BCA, which should be in 300 \\u03bcl of 50 mM HEPES (pH 8.5) at\",\n",
            "          \"BCA, which should be in 300 \\u03bcl of 50 mM HEPES (pH 8.5) at this point, and transfer it to a 2-ml screw-cap tube. Reconstitute each 5-mg TMT reagent vial in 256 \\u03bcl of anhydrous acetonitrile. Allow it to sit for 5 min. Vortex the reagent tubes briefly and\",\n",
            "          \"Allow it to sit for 5 min. Vortex the reagent tubes briefly and spin down on a benchtop centrifuge at 1,000g at RT for 30 s. Add 123 \\u03bcl of each reagent to the corresponding aliquot of peptides. Mix well and incubate at RT for 1 h with shaking on a\",\n",
            "          \"peptides. Mix well and incubate at RT for 1 h with shaking on a tabletop shaker set at 1,000 r.p.m. After 1 h, remove 2-\\u03bcg aliquots from each sample and combine these to create a mixing QC test sample. Using a vacuum centrifuge, dry down this mixture of\",\n",
            "          \"sample. Using a vacuum centrifuge, dry down this mixture of the ten channels and desalt these QC samples as described in Steps 63\\u201372. Freeze the rest of the samples at \\u221280 \\u00b0C until the mixing QC test is analyzed. \\u25b2 CRITICAL STEP Do not quench the labeling\",\n",
            "          \"QC test is analyzed. \\u25b2 CRITICAL STEP Do not quench the labeling reaction until after assessment of the label incorporation test results. \\u25a0 PAUSE POINT Peptides in 50 mM HEPES (pH 8.5) with TMT-labeling reagent can be stored in a \\u221280 \\u00b0C freezer space for\",\n",
            "          \"reagent can be stored in a \\u221280 \\u00b0C freezer space for up to 2 weeks. Desalting of the mixing QC test sample \\u25cf Timing 30 min Prepare C18 stage tips by using Empore C18 extraction disks, as described in ref. . Pack two plugs of C18 material into the tip of\",\n",
            "          \"in ref. . Pack two plugs of C18 material into the tip of each stage tip (200-\\u03bcl pipette tip) for a total binding capacity of ~40 \\u03bcg. Create extraction disks using a 16-gauge blunt-end metal needle to hole-punch the ~1-mm disks. We recommend using adapters\",\n",
            "          \"to hole-punch the ~1-mm disks. We recommend using adapters to hold the stage-tip pipette tip at the orifice of each 2 ml microcentrifuge tube. Condition the stage tip with 100 \\u03bcl of MeOH. Centrifuge at 3,000\\u20133,500g for 3 min at RT and discard the liquid\",\n",
            "          \"at 3,000\\u20133,500g for 3 min at RT and discard the liquid from the collection vial. All subsequent centrifugation steps are for the same duration or until all the solvent has passed through the stage tip at the same speed and at RT. \\u25b2 CRITICAL STEP Do not\",\n",
            "          \"stage tip at the same speed and at RT. \\u25b2 CRITICAL STEP Do not centrifuge the stage tips at speeds above 3,500g or else the C18 material will dry out. Wash the stage tips with 100 \\u03bcl of 50% (vol/vol) MeCN in 0.1% (vol/vol) FA (stage-tip desalting solvent\",\n",
            "          \"MeCN in 0.1% (vol/vol) FA (stage-tip desalting solvent B). Centrifuge the stage tip at 3,000\\u20133,500g for 3 min at RT. Discard the liquid. Equilibrate the stage tips twice with 100 \\u03bcl of 0.1% (vol/vol) FA (stage-tip desalting solvent A). Centrifuge the\",\n",
            "          \"(vol/vol) FA (stage-tip desalting solvent A). Centrifuge the stage tip at 3,000\\u20133,500g for 3 min at RT after each solvent loading. Discard the liquid. Reconstitute the sample in 100 \\u03bcl of 3% (vol/vol) MeCN/0.1% (vol/vol) FA and add the samples to the\",\n",
            "          \"3% (vol/vol) MeCN/0.1% (vol/vol) FA and add the samples to the stage tip. Centrifuge the stage tip at 3,000\\u20133,500g for 3 min at RT and discard the liquid from the collection vial. Wash the samples twice with 100 \\u03bcl of stage-tip desalting solvent A.\",\n",
            "          \"the samples twice with 100 \\u03bcl of stage-tip desalting solvent A. Centrifuge the stage tip at 3,000\\u20133,500g for 3 min at RT. Discard the liquid. Replace the collection vial with a new 1.5 ml screw-cap vial for the collection of the eluate. Elute the sample\",\n",
            "          \"vial for the collection of the eluate. Elute the sample with 60 \\u03bcl of stage-tip desalting solvent B. Centrifuge the stage tip at 3,000\\u20133,500g for 3 min at RT. Transfer the eluate to an HPLC vial, freeze it, and completely dry it by vacuum centrifugation.\",\n",
            "          \"freeze it, and completely dry it by vacuum centrifugation. Reconstitute the dried sample in 40 \\u03bcl of 3% (vol/vol) MeCN/0.1% (vol/vol) FA and analyze the sample by LC-MS/MS (using the parameters described in the \\u2018Equipment setup\\u2019 section) to check the\",\n",
            "          \"described in the \\u2018Equipment setup\\u2019 section) to check the label incorporation and relative channel abundance. For the LC-MS/MS analysis, inject 2 \\u03bcl onto the column. \\u25b2 CRITICAL STEP As it can take a considerable amount of instrument time to analyze each of\",\n",
            "          \"a considerable amount of instrument time to analyze each of the individual label-incorporation test samples, particularly for TMT-10, one can consider analyzing the mixing QC test sample for both labeling efficiency and mixing consistency, and only\",\n",
            "          \"for both labeling efficiency and mixing consistency, and only analyzing the individual label-incorporation test samples for troubleshooting purposes if the labeling efficiency is suspect in the mixing test sample. Data analysis for assessment of labeling\",\n",
            "          \"mixing test sample. Data analysis for assessment of labeling efficiency and mixing \\u25cf Timing 1 h To evaluate the completeness of free-amine labeling, database searches should be configured to allow for both peptide N termini and lysine side chains to be\",\n",
            "          \"allow for both peptide N termini and lysine side chains to be present in either labeled or unlabeled form. With Spectrum Mill, accounting for partial labeling is accomplished with a fixed/mix search cycle strategy that runs a search four consecutive times\",\n",
            "          \"search cycle strategy that runs a search four consecutive times with different sets of modifications in each round and then produces a single integrated output. The four cycles are as follows: all unmodified, both peptide N termini and lysines labeled,\",\n",
            "          \"all unmodified, both peptide N termini and lysines labeled, only lysines labeled, and only peptide N termini labeled. As the primary amine groups of lysine side chains (pKa ~10) are more reactive than peptide N-terminal amines (pKa ~7.5), incomplete\",\n",
            "          \"reactive than peptide N-terminal amines (pKa ~7.5), incomplete labeling tends to be observed predominantly in the form of unlabeled peptide N termini. Furthermore, so long as a peptide includes at least 1 label, reporter-ion quantitation is viable. After\",\n",
            "          \"at least 1 label, reporter-ion quantitation is viable. After running a four-cycle partial-labeling database search configuration with a suitable (1%) PSM-level false-discovery rate (FDR) threshold on a label-check aliquot, labeling efficiency percentage\",\n",
            "          \"on a label-check aliquot, labeling efficiency percentage metrics are calculated in the Spectrum Mill Quality Metrics module: full label % = 100 \\u00d7 fully labeled PSMs/total PSMs and label % = 100 \\u00d7 1 or more labeled PSMs/total PSMs. \\u25b2 CRITICAL STEP We\",\n",
            "          \"% = 100 \\u00d7 1 or more labeled PSMs/total PSMs. \\u25b2 CRITICAL STEP We typically apply a threshold for minimal labeling efficiency of >99% labeling by PSM identifications. If the threshold is not met, then the samples will be relabeled (see Troubleshooting\",\n",
            "          \"met, then the samples will be relabeled (see Troubleshooting section for relabeling procedure) before proceeding with further peptide-level fractionation. In our experience, TMT labeling is more robust and less likely to require relabeling than iTRAQ\",\n",
            "          \"is more robust and less likely to require relabeling than iTRAQ labeling. ? TROUBLESHOOTING Analyze the resulting MS data from the mixing test sample to check if the total protein amount from each sample is the same. The Spectrum Mill Quality Metrics\",\n",
            "          \"from each sample is the same. The Spectrum Mill Quality Metrics module does this by summing the reporter-ion intensity (after applying isotopic correction) for each channel across all the confidently identified PSMs in the dataset. Then, per-sample mixing\",\n",
            "          \"identified PSMs in the dataset. Then, per-sample mixing percentages are calculated using the most abundant reporter-ion channel sum as a common denominator and each of the other channels as a numerator. \\u25b2 CRITICAL STEP Unless particular samples are\",\n",
            "          \"as a numerator. \\u25b2 CRITICAL STEP Unless particular samples are available in limited quantity, we require each sample to have a reporter-ion intensity sum for all identified PSMs in the mixing test sample LC-MS/MS run that deviates <25% from the\",\n",
            "          \"the mixing test sample LC-MS/MS run that deviates <25% from the reporter-ion intensity sum of the reference channel, if present, or from the average reporter-ion intensity sum of all channels. ? TROUBLESHOOTING Quenching of the TMT-labeling reaction \\u25cf\",\n",
            "          \"? TROUBLESHOOTING Quenching of the TMT-labeling reaction \\u25cf Timing 20 min Quench all individual TMT-labeling reactions by adding 32 \\u03bcl of 5% (vol/vol) hydroxylamine, followed by incubation at RT for 15 min with shaking at 1,000 r.p.m. Combine the volumes\",\n",
            "          \"RT for 15 min with shaking at 1,000 r.p.m. Combine the volumes of each of the ten TMT-labeled samples in a 15-ml conical tube before freezing in liquid nitrogen or by placing into a \\u221280 \\u00b0C freezer space. Dry down the combined sample. \\u25a0 PAUSE POINT Dried\",\n",
            "          \"space. Dry down the combined sample. \\u25a0 PAUSE POINT Dried peptides can be stored in a \\u221280 \\u00b0C freezer space for up to 1 month. Peptide desalting of labeled peptides by SPE \\u25cf Timing 30 min Use a 200-mg tC18 SepPak cartridge with a vacuum manifold to desalt\",\n",
            "          \"a 200-mg tC18 SepPak cartridge with a vacuum manifold to desalt the peptide samples. \\u25b2 CRITICAL STEP Do not let the cartridges run dry during washing or loading, as this can increase sample losses during desalting from 5% to >20%. Reconstitute the\",\n",
            "          \"losses during desalting from 5% to >20%. Reconstitute the combined sample of TMT-labeled peptides in 3 ml of 3% (vol/vol) MeCN/0.1% (vol/vol) FA. \\u25b2 CRITICAL STEP Measure the pH with pH-indicator paper and adjust the pH to 3 with 100% (vol/vol) FA if\",\n",
            "          \"paper and adjust the pH to 3 with 100% (vol/vol) FA if necessary. Condition the cartridge with 3 ml of 100% (vol/vol) MeCN. Condition the cartridge with 3 ml of 50% (vol/vol) MeCN/0.1% (vol/vol) FA. Equilibrate the cartridge with 4 \\u00d7 3 ml of 0.1%\",\n",
            "          \"(vol/vol) FA. Equilibrate the cartridge with 4 \\u00d7 3 ml of 0.1% (vol/vol) TFA. Load the sample onto the cartridge. Desalt the sample with 3 \\u00d7 3 ml of 0.1% (vol/vol) TFA. Wash the cartridge with 1 \\u00d7 3 ml of 1% (vol/vol) FA to remove the TFA. Elute the sample\",\n",
            "          \"1 \\u00d7 3 ml of 1% (vol/vol) FA to remove the TFA. Elute the sample from the tC18 cartridge with 2 \\u00d7 1.5 ml of 50% (vol/vol) MeCN/0.1% (vol/vol) FA. Freeze the eluate in liquid nitrogen or in a \\u221280 \\u00b0C freezer space before drying down with a vacuum centrifuge.\",\n",
            "          \"\\u00b0C freezer space before drying down with a vacuum centrifuge. \\u25a0 PAUSE POINT Dried peptides can be stored in a \\u221280 \\u00b0C freezer space for up to 1 year. Offline HPLC fractionation \\u25cf Timing 60 min + overnight QC + 5 h Prepare the HPLC system by purging solvent\",\n",
            "          \"+ overnight QC + 5 h Prepare the HPLC system by purging solvent lines A and B of air. Equilibrate the bRP fractionation column with 100% (vol/vol) solvent B, then 100% (vol/vol) solvent A, and finally with 50% (vol/vol) solvent A/50% (vol/vol) solvent B.\",\n",
            "          \"finally with 50% (vol/vol) solvent A/50% (vol/vol) solvent B. Equilibrate with each of these conditions for 30 min, for a total of 1.5 h. The flow rate for equilibration is that of the method, which in our method is 1 ml/min. Perform a QC of the HPLC\",\n",
            "          \"which in our method is 1 ml/min. Perform a QC of the HPLC system before fractionation. We typically accomplish this by injecting peptide standards in duplicate before running the actual sample. For this scale of bRP, 200 pmol of a mixture of ten in-house\",\n",
            "          \"For this scale of bRP, 200 pmol of a mixture of ten in-house synthetic peptide standards (Reagent setup) is injected and evaluated for retention time reproducibility, signal intensity, and peak resolution (Supplementary Fig. 3a). Prepare the LC system for\",\n",
            "          \"resolution (Supplementary Fig. 3a). Prepare the LC system for QC sample injection: the composition of a single sample separation is a sequence of two methods: a blank gradient, used to condition the column, followed by the sample gradient. For the QC run,\",\n",
            "          \"the column, followed by the sample gradient. For the QC run, we typically use a method that is meant for label-free peptides for better separation and resolution peaks for the peptides. Once the QC of the peptide standards indicates the system is running\",\n",
            "          \"the QC of the peptide standards indicates the system is running properly, run a blank gradient to condition the column for the TMT-labeled sample (Equipment setup). This is achieved by injecting 50 \\u03bcl of bRP solvent A (refer to \\u2018Reagent setup\\u2019 for\",\n",
            "          \"injecting 50 \\u03bcl of bRP solvent A (refer to \\u2018Reagent setup\\u2019 for basic-pH RP solvents) from an HPLC vial with the gradient outlined in the \\u2018Equipment setup\\u2019 as a blank gradient. Upon completion of the blank run, reconstitute the sample in 900 \\u03bcl of bRP\",\n",
            "          \"of the blank run, reconstitute the sample in 900 \\u03bcl of bRP solvent A (see \\u2018Reagent setup\\u2019 for basic-pH RP solvents). Vortex until the sample is in solution. \\u25b2 CRITICAL STEP Do not pipette up and down to bring the sample into solution, as this will result\",\n",
            "          \"and down to bring the sample into solution, as this will result in the formation of bubbles. Transfer the sample to a 1.5-ml screw-cap vial and centrifuge at 20,000g at RT for 5 min to remove any material that did not go into solution. Transfer the\",\n",
            "          \"remove any material that did not go into solution. Transfer the sample, avoiding the pelleted insoluble peptides, to an HPLC vial. Inject 850 \\u03bcl of the sample into the sample loop of the HPLC system. \\u25b2 CRITICAL STEP The sample volume is brought to 900 \\u03bcl,\",\n",
            "          \"system. \\u25b2 CRITICAL STEP The sample volume is brought to 900 \\u03bcl, but only 850 \\u03bcl is injected, because extra volume is needed to avoid injecting air bubbles into the system during sample injection. For separation of the TMT-labeled samples, use the\",\n",
            "          \"injection. For separation of the TMT-labeled samples, use the corresponding gradient and flow rate settings outlined in the tables in the \\u2018Equipment setup\\u2019 section. For an example chromatogram, see Supplementary Fig. 3b. In the bRP separation, 96\",\n",
            "          \"see Supplementary Fig. 3b. In the bRP separation, 96 fractions are collected into a Whatman 2-ml 96-well plate at a flow rate of 1 ml/min. \\u25b2 CRITICAL STEP To store the HPLC column after separation, flow water through the column at 1 ml/min and then flow\",\n",
            "          \"flow water through the column at 1 ml/min and then flow methanol through the column, also at 1 ml/min. Do not store or maintain the column in basic solvent when samples are not being run, as this will substantially speed up the aging of the column and\",\n",
            "          \"as this will substantially speed up the aging of the column and cause premature widening of peaks. After separation, pool the bRP fractions (as described in Table 1) in 15-ml conical vials to generate 24 final fractions and fraction A. Fraction A does not\",\n",
            "          \"generate 24 final fractions and fraction A. Fraction A does not contain many peptides, acts as a negative control, and contains multiply phosphorylated peptides that do not bind well to the RP column. The plate layout describes the recombination of wells\",\n",
            "          \"column. The plate layout describes the recombination of wells in which B10 is combined with D10 and F10 as fraction 1, B9 is combined with D9 and F9 as fraction 2, and so on (Table 1). \\u25b2 CRITICAL STEP In our method of pooling, we have factored in the fact\",\n",
            "          \"STEP In our method of pooling, we have factored in the fact that our sample is eluted in a serpentine manner. For methods that do not eluate in a serpentine method, please correct the pooling scheme to still allow pooling of the fractions from the\",\n",
            "          \"pooling scheme to still allow pooling of the fractions from the beginning, middle, and end of the gradient in an even and recurring manner. Our method reduces 96 individual wells into 23 final fractions (three wells with disparate hydrophobicity) plus two\",\n",
            "          \"fractions (three wells with disparate hydrophobicity) plus two fractions from the early and late portions of the chromatogram that are sparser in peptide content. Acidify each pooled fraction using 10% (vol/vol) FA to achieve a final concentration of 0.1%\",\n",
            "          \"using 10% (vol/vol) FA to achieve a final concentration of 0.1% (vol/vol) FA and a pH of 3. From each pooled fraction, collect 5% of each fraction into an HPLC vial for proteome analysis by LC-MS/MS. Vacuum-centrifuge each HPLC vial with the proteome\",\n",
            "          \"by LC-MS/MS. Vacuum-centrifuge each HPLC vial with the proteome analysis fraction. For analysis, bring the sample up in 3% (vol/vol) MeCN/0.1% (vol/vol) FA to a final concentration of 0.5 \\u03bcg/\\u03bcl and inject 1 \\u03bcl onto the column. In practice, we assume there\",\n",
            "          \"and inject 1 \\u03bcl onto the column. In practice, we assume there is 6.25 \\u03bcg in each of the 24 fractions and we bring the sample up in 12.5 \\u03bcl of 3% (vol/vol) MeCN/0.1% (vol/vol) FA. We have previously found that the columns can accommodate 1 \\u03bcg without\",\n",
            "          \"previously found that the columns can accommodate 1 \\u03bcg without overloading when samples are measured on the protein level. In our current protocol, we are quantifying the amounts on the peptide level. Peptide yields are approximately half of protein\",\n",
            "          \"peptide level. Peptide yields are approximately half of protein yields. Therefore, by loading 0.5 \\u03bcg (peptide-level BCA), we are effectively reproducing our previous 1 \\u03bcg on the column (protein-level BCA). Experience from one of the three labs indicates\",\n",
            "          \"BCA). Experience from one of the three labs indicates that results with 0.5- and 1-\\u03bcg loads (by peptide BCA) are very similar. In general, it should be noted that the higher the peptide load, the greater the possibility of the chromatographic separation\",\n",
            "          \"the greater the possibility of the chromatographic separation decreasing because of overloading. ? TROUBLESHOOTING Combine the remaining 95% of each fraction into 12 fractions + fraction A for enrichment of phosphopeptides as follows: Fraction no.\\tbRP\",\n",
            "          \"for enrichment of phosphopeptides as follows: Fraction no.\\tbRP fractions\\t \\t1\\t1 + 13\\t \\t2\\t2 + 14\\t \\t3\\t3 + 15\\t \\t4\\t4 + 16\\t \\t5\\t5 + 17\\t \\t6\\t6 + 18\\t \\t7\\t7 + 19\\t \\t8\\t8 + 20\\t \\t9\\t9 + 21\\t \\t10\\t10 + 22\\t \\t11\\t11 + 23\\t \\t12\\t12 + 24\\t \\tA\\tA\\t \\t Freeze the 13 fractions with liquid\",\n",
            "          \"+ 23\\t \\t12\\t12 + 24\\t \\tA\\tA\\t \\t Freeze the 13 fractions with liquid nitrogen or in the \\u221280 \\u00b0C freezer space before drying the samples down in a vacuum centrifuge or lyophilizer. \\u25a0 PAUSE POINT Dried peptides can be stored in a \\u221280 \\u00b0C freezer space for up 1\",\n",
            "          \"Dried peptides can be stored in a \\u221280 \\u00b0C freezer space for up 1 week. Combined IMAC enrichment and phosphopeptide desalting: bead preparation \\u25cf Timing 90 min Fully resuspend the Ni-NTA Superflow Agarose beads before removing an appropriate aliquot of\",\n",
            "          \"Agarose beads before removing an appropriate aliquot of slurry, in which the beads/solvent ratio is 1:1 (vol/vol). This protocol is written for the preparation of up to 500 \\u03bcl of beads. Scale up appropriately if >300 \\u03bcl of beads need to be prepped at\",\n",
            "          \"up appropriately if >300 \\u03bcl of beads need to be prepped at once. For a 13-fraction IMAC enrichment, 130 \\u03bcl of beads is needed. To account for proper overhead, remove 160 \\u03bcl of beads or 320 \\u03bcl of slurry. \\u25b2 CRITICAL STEP Prepare the beads fresh on the day\",\n",
            "          \"of slurry. \\u25b2 CRITICAL STEP Prepare the beads fresh on the day of the enrichment. Centrifuge the slurry in a tabletop centrifuge at 1,000g at RT for 1 min. Remove the stock supernatant. Wash the beads 3\\u00d7 in 1 ml of HPLC water, centrifuging in the tabletop\",\n",
            "          \"beads 3\\u00d7 in 1 ml of HPLC water, centrifuging in the tabletop centrifuge (1,000g at RT for 1 min) each time to remove the supernatant. Incubate the beads in 1,200 \\u03bcl of 100 mM EDTA (1:5 (vol/vol) dilution of the stock 500 mM EDTA solution) for 30 min at RT\",\n",
            "          \"dilution of the stock 500 mM EDTA solution) for 30 min at RT with end-over-end turning to strip the beads of nickel. Wash the beads 3\\u00d7 in 1 ml of HPLC water, centrifuging in the tabletop centrifuge (1,000g at RT for 1 min) each time to remove the\",\n",
            "          \"centrifuge (1,000g at RT for 1 min) each time to remove the supernatant. Incubate the beads with 1,200 \\u03bcl of 10 mM iron (III) chloride aqueous solution (solid iron (III) chloride in HPLC water) for 30 min at RT with end-over-end turning. Wash the beads 3\\u00d7\",\n",
            "          \"for 30 min at RT with end-over-end turning. Wash the beads 3\\u00d7 in 1 ml of HPLC water, centrifuging in the tabletop centrifuge each time (1,000g at RT for 1 min) to remove the supernatant. Bring the beads up with a 1:1:1 (vol/vol/vol) ratio of\",\n",
            "          \"Bring the beads up with a 1:1:1 (vol/vol/vol) ratio of acetonitrile/methanol/0.01% (vol/vol) acetic acid so that it is a slurry of 1:1:1:1 beads/acetonitrile/methanol/0.01% (vol/vol) acetic acid. If 160 \\u03bcl of beads were originally removed, the total\",\n",
            "          \"acid. If 160 \\u03bcl of beads were originally removed, the total volume would be 640 \\u03bcl at this point. Pipette 40 \\u03bcl of bead slurry into each of 13 \\u00d7 1.5-ml screw-cap tubes, shaking the slurry from which you are drawing aliquots after every few aliquots to\",\n",
            "          \"from which you are drawing aliquots after every few aliquots to maintain the suspension. The 40 \\u03bcl of slurry in each tube corresponds to 10 \\u03bcl of beads for enrichment per fraction. \\u25b2 CRITICAL STEP Pipette 40 uL of bead slurry into each of thirteen \\u00d7 1.5\",\n",
            "          \"STEP Pipette 40 uL of bead slurry into each of thirteen \\u00d7 1.5 mL screw-cap tubes, shaking the slurry from which you are drawing aliquots after every few aliquots to maintain the suspension. The 40 uL of slurry in each tube corresponds to 10 uL of beads\",\n",
            "          \"The 40 uL of slurry in each tube corresponds to 10 uL of beads for enrichment per fraction. \\u25a0 PAUSE POINT Beads cannot be prepared before the day of the enrichment. Beads that are not prepared fresh on the day of the enrichment have been found to show\",\n",
            "          \"fresh on the day of the enrichment have been found to show decreased enrichment specificity. However, beads can be kept in this 1:1:1:1 slurry for up to 1\\u20132 h. Peptide sample preparation \\u25cf Timing 15 min Bring the peptide concentration to 0.5 \\u03bcg/\\u03bcl in 80%\",\n",
            "          \"15 min Bring the peptide concentration to 0.5 \\u03bcg/\\u03bcl in 80% (vol/vol) MeCN/0.1% (vol/vol) TFA. Because it is difficult to solubilize peptides in such high organic solvents, first solubilize the peptides in 50% (vol/vol) MeCN/0.1% (vol/vol) TFA and add 100%\",\n",
            "          \"peptides in 50% (vol/vol) MeCN/0.1% (vol/vol) TFA and add 100% (vol/vol) MeCN/0.1% (vol/vol) TFA to achieve an 80% (vol/vol) MeCN/0.1% (vol/vol) TFA concentration. For example, in a 3-mg sample that was fractionated into 24 fractions (with 5% removed for\",\n",
            "          \"that was fractionated into 24 fractions (with 5% removed for proteome analysis) and then pooled into 12 fractions, there is 237.5 \\u03bcg of peptides in each of the phosphofractions for enrichment. Fraction A is not counted in the calculations because it is\",\n",
            "          \"Fraction A is not counted in the calculations because it is nearly empty, except for a few multiphosphorylated peptides. Bring each fraction, including fraction A, into solution. First dissolve each fraction in 190 \\u03bcl of 50% (vol/vol) MeCN/0.1% (vol/vol)\",\n",
            "          \"each fraction in 190 \\u03bcl of 50% (vol/vol) MeCN/0.1% (vol/vol) TFA with vortexing. Once all the peptides are in solution, add 285 \\u03bcl of 100% (vol/vol) MeCN/0.1% (vol/vol) TFA. \\u25b2 CRITICAL STEP A concentration of 0.1% (vol/vol) TFA is used in the IMAC binding\",\n",
            "          \"concentration of 0.1% (vol/vol) TFA is used in the IMAC binding buffer to control the pH at 2. This ensures that the carboxyl groups of glutamic and aspartic acid, and the peptide C termini are protonated to avoid background binding of nonphosphorylated\",\n",
            "          \"are protonated to avoid background binding of nonphosphorylated peptides to the IMAC resin. At pH 2, S/T/Y-phosphorylated residues remain negatively charged and bind strongly to the resin. Spin down the samples (1,500g at RT for 10 min) to pellet any\",\n",
            "          \"Spin down the samples (1,500g at RT for 10 min) to pellet any un-reconstituted peptides. \\u25b2 CRITICAL STEP Check if the pH is 2 with pH-indicator paper. Combined IMAC enrichment and phosphopeptide desalting: phosphoenrichment \\u25cf Timing 45 min Add the peptide\",\n",
            "          \"desalting: phosphoenrichment \\u25cf Timing 45 min Add the peptide solution to the aliquoted beads and incubate for 30 min at RT on a shaker at 1,000 r.p.m. \\u25b2 CRITICAL STEP Do not let the incubation go beyond 30 min. Spin down the bead-peptide solution for 1\",\n",
            "          \"go beyond 30 min. Spin down the bead-peptide solution for 1 min at 1,000g at RT in a centrifuge. Remove the supernatant from each tube and keep this as a \\u2018flow through\\u2019 sample for potential subsequent enrichments. Add 200 \\u03bcl of 80% (vol/vol) MeCN/0.1%\",\n",
            "          \"subsequent enrichments. Add 200 \\u03bcl of 80% (vol/vol) MeCN/0.1% (vol/vol) TFA to the beads, creating a slurry that can later be pipetted and transferred in Step 118. Combined IMAC enrichment and phosphopeptide desalting: stage-tip desalting of\",\n",
            "          \"enrichment and phosphopeptide desalting: stage-tip desalting of phosphoenrichment \\u25cf Timing 90 min Prepare 13 2-plug C18 stage tips by using Empore C18 extraction disks as described in Step 63. Condition the stage tips with 100 \\u03bcl of MeOH. Centrifuge at\",\n",
            "          \"63. Condition the stage tips with 100 \\u03bcl of MeOH. Centrifuge at 3,000\\u20133,500g for 3 min at RT and discard the liquid from the collection vial. Condition the stage tips a second time with another 100 \\u03bcl of MeOH. All subsequent centrifugation steps are for\",\n",
            "          \"100 \\u03bcl of MeOH. All subsequent centrifugation steps are for the same duration or until all the solvent has passed through the stage tip at the same speed and at RT. \\u25b2 CRITICAL STEP Do not centrifuge the stage tips at speeds >3,500g or else the C18\",\n",
            "          \"not centrifuge the stage tips at speeds >3,500g or else the C18 material will dry out. Wash the stage tips with 50 \\u03bcl of 50% (vol/vol) MeCN in 0.1% (vol/vol) FA (stage tip solvent B). Centrifuge the stage tips at 3,000\\u20133,500g for 3 min at RT. Discard the\",\n",
            "          \"the stage tips at 3,000\\u20133,500g for 3 min at RT. Discard the liquid. Equilibrate the stage tips twice with 100 \\u03bcl of 1% (vol/vol) FA. Centrifuge the stage tips at 3,000\\u20133,500g for 3 min at RT after each solvent loading. Discard the liquid. Load the\",\n",
            "          \"at RT after each solvent loading. Discard the liquid. Load the enriched beads onto the stage tip. Centrifuge the stage tips at 3000\\u20133500g for 3 min at RT and discard the liquid from the collection vial. Wash the beads twice with 50 \\u03bcl of 80% (vol/vol)\",\n",
            "          \"vial. Wash the beads twice with 50 \\u03bcl of 80% (vol/vol) MeCN/0.1% (vol/vol) TFA. Centrifuge the stage tips at 3000\\u20133500g for 3 min at RT after each solvent loading. Discard the liquid. \\u25b2 CRITICAL STEP A concentration of 0.1% (vol/vol) TFA is used to ensure\",\n",
            "          \"STEP A concentration of 0.1% (vol/vol) TFA is used to ensure that the pH is 2. This ensures that the carboxyl groups of glutamic and aspartic acid, and the peptide C termini are protonated to avoid background binding of nonphosphorylated peptides to the\",\n",
            "          \"avoid background binding of nonphosphorylated peptides to the IMAC resin. At pH 2, S/T/Y-phosphorylated residues remain negatively charged and bind strongly to the resin. Wash the beads with 50 \\u03bcl of 1% (vol/vol) FA. Centrifuge the stage tip at\",\n",
            "          \"with 50 \\u03bcl of 1% (vol/vol) FA. Centrifuge the stage tip at 3,000\\u20133,500g for 3 min at RT. Discard the liquid. Elute the peptides from the beads and onto the C18 plugs with three iterations of 70 \\u03bcl of agarose-bead elution buffer. Centrifuge the stage tip\",\n",
            "          \"70 \\u03bcl of agarose-bead elution buffer. Centrifuge the stage tip at 3,000\\u20133,500g for 3 min at RT after each solvent loading. Discard the liquid. Wash the C18 material with 100 \\u03bcl of 1% (vol/vol) FA. Centrifuge the stage tip at 3,000\\u20133,500g for 3 min at RT\",\n",
            "          \"FA. Centrifuge the stage tip at 3,000\\u20133,500g for 3 min at RT and discard the liquid. Replace the collection vial with a new 1.5-ml screw-cap vial for the collection of the eluate. Elute the sample from the C18 plugs with 60 \\u03bcl of StageTip solvent B.\",\n",
            "          \"the sample from the C18 plugs with 60 \\u03bcl of StageTip solvent B. Centrifuge the stage tip at 3,000\\u20133,500g for 3 min at RT. Transfer the eluate to an HPLC vial, then freeze and completely dry the samples by vacuum centrifugation. Reconstitute the dried\",\n",
            "          \"the samples by vacuum centrifugation. Reconstitute the dried sample in 9 \\u03bcl of 3% (vol/vol) MeCN/0.1% (vol/vol) FA and analyze by LC-MS/MS (using the parameters described in the \\u2018Equipment setup\\u2019 section). For LC-MS/MS analysis, inject 4 \\u03bcl onto the\",\n",
            "          \"setup\\u2019 section). For LC-MS/MS analysis, inject 4 \\u03bcl onto the column. \\u25b2 CRITICAL STEP TFA reduces the ionization efficiency in the electrospray, so we wash it out of the stage tips using 1% (vol/vol) FA. The concentration of FA is reduced to 0.1% at the\",\n",
            "          \"(vol/vol) FA. The concentration of FA is reduced to 0.1% at the last elution step to ensure the best sensitivity before injection into the mass spectrometer. ? TROUBLESHOOTING Data analysis \\u25cf Timing <1 d We typically analyze the data using Spectrum Mill\",\n",
            "          \"\\u25cf Timing <1 d We typically analyze the data using Spectrum Mill MS Proteomics Workbench. However, other search engines can be used for searching the data, as long as the program can handle TMT-10 data. The table below shows the search parameters we use.\",\n",
            "          \"data. The table below shows the search parameters we use. Parameter\\tValue\\t \\tVariable modification\\tOxidation (M)\\t \\t\\tAcetyl (protein N-term)\\t \\t\\tDeamidation (N)\\t \\t\\tPyroGlu (Q)\\t \\tFixed modification\\tCarbamidomethyl (C)\\t \\t\\tTMT-10 (peptide N-term, K)\",\n",
            "          \"modification\\tCarbamidomethyl (C)\\t \\t\\tTMT-10 (peptide N-term, K)\\t \\tDigest\\tTrypsin Allow P\\t \\tMaximum missed cleavages\\t4\\t \\tMaximum charge\\t5\\t \\tPrecursor mass tolerance (p.p.m.)\\t20\\t \\tProduct mass tolerance (p.p.m.)\\t20\\t \\tPeptide FDR (%)\\t1\\t \\tProtein FDR (%)\\t0\",\n",
            "          \"(p.p.m.)\\t20\\t \\tPeptide FDR (%)\\t1\\t \\tProtein FDR (%)\\t0\\t \\t \\u25b2 CRITICAL STEP The digestion enzyme search parameter used, Trypsin Allow P, allows K-P and P cleavages that are typically disallowed by search engines when configured for trypsin specificity. These\",\n",
            "          \"search engines when configured for trypsin specificity. These two additional bond cleavages are allowed because the digestion protocol employs both Lys-C and trypsin. Lys-C often cleaves at K-P linkages, and trypsin can occasionally cleave at both R-P and\",\n",
            "          \"linkages, and trypsin can occasionally cleave at both R-P and K-P, Therefore, the missed cleavage allowance is set to 4 instead of a typical value of 2 when using trypsin. Other search engines may have the flexibility to configure custom digestion\",\n",
            "          \"engines may have the flexibility to configure custom digestion specificity to explicitly allow K-P cleavage when disallowing R-P cleavage. \\u25b2 CRITICAL STEP Deamidation of glutamine (Q) residues is much slower than deamidation of asparagine (N) residues,\",\n",
            "          \"is much slower than deamidation of asparagine (N) residues, because of the kinetics of a reaction mechanism intermediate with a six-membered ring (Q) versus a five-membered ring (N). However, some search engines may not readily allow for selection of a\",\n",
            "          \"some search engines may not readily allow for selection of a deamidation variable modification only at N. Deamidation reactions are kinetically more favored at the elevated pH (10) encountered during the basic RP step in this protocol after the samples\",\n",
            "          \"during the basic RP step in this protocol after the samples are mixed. Analyze the final sample dataset with a two-cycle fixed/mix modifications search in Spectrum Mill that runs two consecutive searches with different sets of fixed modifications in each\",\n",
            "          \"searches with different sets of fixed modifications in each round and then produces a single integrated output. The two cycles allow for (i) labeling of both peptide N termini and lysines, and (ii) labeling of only lysines. Other search engines might\",\n",
            "          \"and (ii) labeling of only lysines. Other search engines might allow similar strategies to be executed by configuring the label as a variable modification on peptide N termini and on lysines. \\u25b2 CRITICAL STEP Including variable modifications during peptide\",\n",
            "          \"\\u25b2 CRITICAL STEP Including variable modifications during peptide identification of MS/MS spectra to account for common foreseeable sample handling modifications is not only important for calculating the metrics to monitor sample-handling QC, but also\",\n",
            "          \"calculating the metrics to monitor sample-handling QC, but also diminishes the potential for falsepositive identification of otherwise lower-scoring unmodified sequences. Depending on the capabilities and calculation methods of the quantitation software\",\n",
            "          \"and calculation methods of the quantitation software used, it may be appropriate to exclude PSMs related to sample handling modifications from quantitative calculations that are expected to occur disproportionately in individual samples before the point\",\n",
            "          \"occur disproportionately in individual samples before the point of mixing. Correct the reporter-ion intensities for isotopic impurities before using the reporter-ion signals in each MS/MS spectrum for quantitative calculations. \\u25b2 CRITICAL STEP Each lot of\",\n",
            "          \"for quantitative calculations. \\u25b2 CRITICAL STEP Each lot of reagent obtained from its manufacturer is accompanied by a certificate of analysis (this is also available on the manufacturer\\u2019s website) that contains isotopic correction factors to be used by\",\n",
            "          \"that contains isotopic correction factors to be used by the quantitation software. These correction factors are primarily used to account for naturally occurring levels of 13C at the unlabeled carbons in the mass-tag portion of the labeling reagents.\",\n",
            "          \"carbons in the mass-tag portion of the labeling reagents. Because very-high-purity sources of 15N and 13C are routinely available and are used for the labeled positions in the reagent, near-full incorporation is typical. The correction factors provided by\",\n",
            "          \"incorporation is typical. The correction factors provided by the reagent manufacturer represent MS measurements of the isotope profile for each lot of reagent and thus comingle the contributions of the unlabeled carbon and the source of heavy isotope.\",\n",
            "          \"of the unlabeled carbon and the source of heavy isotope. Because there are more unlabeled carbons in the TMT structure than in the iTRAQ structure, attention to isotopic correction to achieve accurate quantitation is more important for TMT than for iTRAQ.\",\n",
            "          \"accurate quantitation is more important for TMT than for iTRAQ. \\u25b2 CRITICAL STEP In complex samples such as tissue, the typical precursor mass window used for MS/MS (i.e., 0.7\\u20132.0 m/z) passes more ions than just the dominant peak in any given m/z window.\",\n",
            "          \"more ions than just the dominant peak in any given m/z window. If these additional ions have an isobaric mass-tag reporter as part of the structure, then fragmentation of these ions will produce mass-tag ions that add to the reporter-ion series from the\",\n",
            "          \"mass-tag ions that add to the reporter-ion series from the dominant peak. As most peptides in a sample are derived from proteins whose levels are not changing (or not changing substantially), the interfering signals from this background in every mass-tag\",\n",
            "          \"the interfering signals from this background in every mass-tag channel compress the observed ratios of regulated peptides/proteins in the samples. When combining reporter-ion quantitation from multiple PSMs to the protein level, several software programs\",\n",
            "          \"multiple PSMs to the protein level, several software programs offer mechanisms to exclude PSMs that exhibit substantial interference. A precursor-ion purity filter is commonly used that determines the ratio of the intensity of the precursor ion and\",\n",
            "          \"determines the ratio of the intensity of the precursor ion and isotopes of the primary peptide identified in the MS/MS spectrum to the total intensity in the mass window isolated for precursor-ion transmission. Use of a precursor-ion purity of >50% to 70%\",\n",
            "          \"transmission. Use of a precursor-ion purity of >50% to 70% is recommended. Various software packages employ different mechanisms for combining the PSM-level measurements to the protein level from which the constituent peptides are derived. Spectrum Mill\",\n",
            "          \"from which the constituent peptides are derived. Spectrum Mill takes the ratios at the PSM level, then calculates the protein-level ratio as the median of all PSM ratios. This strategy diminishes the overall impact of outliers but otherwise gives each PSM\",\n",
            "          \"the overall impact of outliers but otherwise gives each PSM an equal contribution to the protein-level ratio. Another common strategy is to sum the intensity for each reporter-ion channel from multiple PSMs contributing to the protein before taking the\",\n",
            "          \"multiple PSMs contributing to the protein before taking the ratios. This strategy seeks to weight the contribution of each PSM by the reporter-ion signal strength, with the implicit assumption that low-abundance signals are less accurate. Other variants\",\n",
            "          \"that low-abundance signals are less accurate. Other variants of these basic approaches involve combining multiple PSMs to the peptide level first before combining to the protein level. These strategies seek to limit the bias of individual peptides that\",\n",
            "          \"strategies seek to limit the bias of individual peptides that are observed in multiple PSMs because of different precursor charge states or sample-handling modifications. Troubleshooting Troubleshooting advice can be found in Table 2. Timing All\",\n",
            "          \"Troubleshooting advice can be found in Table 2. Timing All approximate timing given below is for a single 10-plex sample. Steps 1\\u201310, (optional) cryopulverization of tissue blocks: 2.5 h for ten non-OCT tissue blocks; 3.5 h for ten OCT tissue blocks Steps\",\n",
            "          \"non-OCT tissue blocks; 3.5 h for ten OCT tissue blocks Steps 11\\u201318, tissue lysis: 90 min Steps 19\\u201327, estimation of protein yield using a Pierce BCA Kit: 60 min Steps 28 and 29, reduction and alkylation: 2 h Steps 30\\u201335, in-solution digestion: 2 h +\",\n",
            "          \"and alkylation: 2 h Steps 30\\u201335, in-solution digestion: 2 h + overnight digestion + 30 min Steps 36\\u201345, peptide desalting of the digest by SPE: 90\\u2013120 min per set of ten samples Steps 46\\u201356, estimation of peptide amount using a Pierce BCA Kit: 60 min\",\n",
            "          \"estimation of peptide amount using a Pierce BCA Kit: 60 min Steps 57\\u201362, TMT labeling of 300 \\u03bcg of peptide per channel: 90 min Steps 63\\u201372, desalting of mixing QC test sample: 30 min Steps 73 and 74, data analysis for the assessment of labeling efficiency\",\n",
            "          \"and 74, data analysis for the assessment of labeling efficiency and mixing: 1 h Steps 75 and 76, quenching of the TMT-labeling reaction: 20 min Steps 77\\u201386, peptide desalting of the labeled peptides by SPE: 30 min Steps 87\\u201399, offline HPLC fractionation:\",\n",
            "          \"by SPE: 30 min Steps 87\\u201399, offline HPLC fractionation: 60 min + overnight QC + 5 h Steps 100\\u2013107, combined IMAC enrichment and phosphopeptide desalting: bead preparation: 90 min Steps 108 and 109, peptide sample preparation: 15 min Steps 110\\u2013113,\",\n",
            "          \"108 and 109, peptide sample preparation: 15 min Steps 110\\u2013113, combined IMAC enrichment and phosphopeptide desalting: phosphoenrichment: 45 min Steps 114\\u2013126, combined IMAC enrichment and phosphopeptide desalting: stage-tip desalting of phosphoenrichment:\",\n",
            "          \"desalting: stage-tip desalting of phosphoenrichment: 90 min Steps 127\\u2013129, data analysis: <1 d Anticipated results Assessing intra-plex, inter-plex, and inter-laboratory variation The protocol was tested across three independent laboratories with full\",\n",
            "          \"was tested across three independent laboratories with full process replicates, starting with cryofractured tissue material, for a total of 20 patient-derived xenograft tumor tissue samples per laboratory. As established in previous methods comparison\",\n",
            "          \"per laboratory. As established in previous methods comparison studies, we analyzed the proteome differences in two very different breast cancer subtypes, basal-like (WHIM2) and luminal (WHIM16) breast samples. The experimental design for this benchmarking\",\n",
            "          \"breast samples. The experimental design for this benchmarking study across three laboratories is depicted in Fig. 1b. As all three groups used the same experimental design and samples, the Procedure can be tested for intra-plex, inter-plex, and\",\n",
            "          \"the Procedure can be tested for intra-plex, inter-plex, and inter-laboratory variation. All participating laboratories followed this protocol, with similar chromatography and MS setups. Although all three laboratories used the same model of mass\",\n",
            "          \"Although all three laboratories used the same model of mass spectrometer, a Thermo Fisher Scientific Orbitrap Fusion Lumos, the LC models were not all the same. The off-line systems used for bRP were an Agilent 1100 (Broad Institute), an Agilent 1220\",\n",
            "          \"for bRP were an Agilent 1100 (Broad Institute), an Agilent 1220 (Johns Hopkins), and an Agilent 1100 (PNNL). For online LC systems, a Thermo Fisher Scientific Easy nLC 1200 was used by both the Broad Institute and Johns Hopkins, whereas a Waters\",\n",
            "          \"by both the Broad Institute and Johns Hopkins, whereas a Waters nanoAcquity instrument was used by PNNL. The gradient-mixing performance inevitably varies between LC manufacturers. Data from all three laboratories were analyzed centrally using the\",\n",
            "          \"from all three laboratories were analyzed centrally using the Spectrum Mill software package, so that all the described variances are due to experimental factors rather than introduced data analysis differences between the participating laboratories.\",\n",
            "          \"analysis differences between the participating laboratories. Comparison of proteome and phosphoproteome datasets We observed highly reproducible proteome and phosphoproteome datasets across three independent laboratories after following the described\",\n",
            "          \"three independent laboratories after following the described protocol. Each laboratory achieved proteome coverages of, on average, 145,000 distinct tryptic peptides (Fig. 2a) and >10,000 proteins, with a maximum deviation across replicates and centers of\",\n",
            "          \"with a maximum deviation across replicates and centers of <7% for protein observations (Fig. 2b). For each protein, we required at least two distinct peptides for identification and two TMT ratio counts for quantification. Out of all quantified proteins,\",\n",
            "          \"counts for quantification. Out of all quantified proteins, 77% were detected by all three centers (Supplementary Fig. 5a). The protein analysis presented here was restricted to ortholog-specific peptides for protein identification and quantification, and\",\n",
            "          \"peptides for protein identification and quantification, and up to 7,700 human proteins derived from tumor cells and up to 3,100 mouse proteins derived from mouse stroma and blood were quantified (Fig. 2b). The phosphoproteome coverage at the peptide level\",\n",
            "          \"(Fig. 2b). The phosphoproteome coverage at the peptide level was, on average, 35,000 phosphopeptides per experiment across laboratories and >31,000 phosphosites in every analyzed TMT-plex experiment (Fig. 2c,d). As, in general, ~35% of human and mouse\",\n",
            "          \"experiment (Fig. 2c,d). As, in general, ~35% of human and mouse tryptic peptides are identical in their amino acid sequences, we did not filter out peptides shared between human and mouse orthologs. Most of the shared signal in the PDX samples is\",\n",
            "          \"orthologs. Most of the shared signal in the PDX samples is contributed by the human tumor material, which contributes up to 55\\u201364% of the total protein signal. The overlap of all quantified phosphosites across three laboratories is 40% (Supplementary Fig.\",\n",
            "          \"across three laboratories is 40% (Supplementary Fig. 5b). For each of the three laboratories, 64\\u201369% of phosphosites were observed in both replicate experiments (Supplementary Fig. 5c-e). In each experiment, more than half of all the phosphosites were\",\n",
            "          \"In each experiment, more than half of all the phosphosites were detected by only a single MS2 scan, whereas every quantified protein is detected by at least two distinct peptides. Reduced overlap at the phosphosite-level relative to protein-level\",\n",
            "          \"overlap at the phosphosite-level relative to protein-level observations may arise because of the variable recovery of phosphopeptides from the additional experimental steps associated with IMAC phosphopeptide enrichment, as well as stochasticity in\",\n",
            "          \"IMAC phosphopeptide enrichment, as well as stochasticity in sampling by the mass spectrometer and minor variability in LC-MS performance. Nonetheless, reproducible and overlapping quantification of >21,000 phosphosites covering 5,384 phosphoproteins\",\n",
            "          \"of >21,000 phosphosites covering 5,384 phosphoproteins across three laboratories was achieved (Supplementary Fig. 5b). Comparison of the reproducibility of protein and phosphoprotein quantification To compare the reproducibility of protein and phosphosite\",\n",
            "          \"To compare the reproducibility of protein and phosphosite quantification, we referenced the reporter intensity of each individual channel against the average across all ten channels at the PSM level. This strategy allows comparison of the quantification\",\n",
            "          \"level. This strategy allows comparison of the quantification data of each individual tumor sample within a TMT-10-plex experiment, as well as across multiplex experiments and across different laboratories. For both the proteome and the phosphoproteome\",\n",
            "          \"laboratories. For both the proteome and the phosphoproteome data, we observe better correlations for the human-only subset of the dataset, as compared with the combined human plus mouse dataset (Supplementary Fig. 6). This is due to the varying\",\n",
            "          \"dataset (Supplementary Fig. 6). This is due to the varying contributions of mouse stroma components in the different cryofractured aliquots used for this study (Supplementary Fig. 7); therefore, we focused on human proteins and phosphosites for subsequent\",\n",
            "          \"we focused on human proteins and phosphosites for subsequent analyses. As expected, the best correlations were observed within a TMT-10-plex, with average r values of 0.94 and 0.85 for protein and phosphosite quantification, respectively (Fig. 3). The\",\n",
            "          \"and phosphosite quantification, respectively (Fig. 3). The high degree of reproducibility of this approach was demonstrated by very similar correlations for the intra- and inter-laboratory comparisons, with median r values of 0.88\\u20130.94 and 0.72\\u20130.88 at\",\n",
            "          \"comparisons, with median r values of 0.88\\u20130.94 and 0.72\\u20130.88 at the protein and phosphosite levels, respectively. We also studied the relationship between minimal ratio counts of proteins and phosphosites with regard to quantification reproducibility\",\n",
            "          \"and phosphosites with regard to quantification reproducibility (Supplementary Fig. 8). For the analyzed datasets, we find the best tradeoff between the highest quantitative reproducibility and the best coverage at two ratio counts for proteins and one\",\n",
            "          \"and the best coverage at two ratio counts for proteins and one ratio count for phosphosites. Isobaric tagging methods have been shown to improve the precision of quantification, whereas label-free methods suffer from high technical variation for PTM\",\n",
            "          \"label-free methods suffer from high technical variation for PTM applications, in which the majority of quantification events rely on the quantification of single peptides. Using the optimal filtering steps described above, we found very consistent results\",\n",
            "          \"steps described above, we found very consistent results for the analysis of breast cancer-relevant driver and biomarker proteins such as ESR1, GATA3, FOXA1, TP53, EGFR, and KRT5 (Supplementary Fig. 9). These proteins were all quantified at similar\",\n",
            "          \"Fig. 9). These proteins were all quantified at similar amplitudes across experiments with levels matching the known luminal and basal subtypes. To identify proteins and phosphosites that are specific for the luminal and basal PDX samples, we used a\",\n",
            "          \"are specific for the luminal and basal PDX samples, we used a two-sample t test and compared the overlap of proteins that were called to be specific to either of the subtypes across the different laboratories (FDR 1%). On the protein level, we observed an\",\n",
            "          \"laboratories (FDR 1%). On the protein level, we observed an overlap of 62.3% and 61.8% for the luminal and basal proteomes, respectively (Fig. 4a,c). For phosphoproteome, the overlap was 27.7% and 28% for luminal and basal phosphosites, respectively (Fig.\",\n",
            "          \"and 28% for luminal and basal phosphosites, respectively (Fig. 4b,d). Using Gene Set Enrichment Analysis, we also converted the proteome and phosphoproteome data to pathways from the Molecular Signature database, MSigDB. As single protein and phophosite\",\n",
            "          \"Signature database, MSigDB. As single protein and phophosite measurements are further collapsed at the pathway level, even higher correlations can be observed for the proteome and phosphoproteome pathway datasets (Supplementary Fig. 10). Data from all\",\n",
            "          \"pathway datasets (Supplementary Fig. 10). Data from all laboratories confirmed the subtype identity of the analyzed PDX tumor sample at the pathway level. In studies of human-only breast tumor tissue, we have achieved typical coverage depth of >8,000\",\n",
            "          \"tumor tissue, we have achieved typical coverage depth of >8,000 proteins per experiment (>2 unique peptides/protein) and >25,000 phosphorylation sites per experiment using the protocol described here (P.M., L.C.T., K.K., K.R.C., M.A. Gillette, D.R.M.,\",\n",
            "          \"here (P.M., L.C.T., K.K., K.R.C., M.A. Gillette, D.R.M., F.M., H.K., S.R.D., S.A.C. & S. Satpathy, Broad Institute; M. Ellis, Baylor College of Medicine; L. Ding, Washington University School of Medicine; K. Ruggles and D. Fenyo, New York University; B.\",\n",
            "          \"of Medicine; K. Ruggles and D. Fenyo, New York University; B. Zhang, Baylor College of Medicine; H. Rodriguez, C. Kinsinger, E. Boja and M. Mesri, National Cancer Institute; data accessible at https://cptac-data-portal.georgetown.edu/cptac/s/S039?; Study\",\n",
            "          \"https://cptac-data-portal.georgetown.edu/cptac/s/S039?; Study name: CPTAC breast cancer confirmatory study). Concluding remarks The optimized protocol presented here enables deep-scale and reproducible proteomics data and results to be obtained within and\",\n",
            "          \"proteomics data and results to be obtained within and across laboratories conducting tissue or cell analyses. The workflow was systematically characterized and analytically validated across three independent laboratories using two distinct breast cancer\",\n",
            "          \"three independent laboratories using two distinct breast cancer subtypes. Proteome coverages of >10,000 proteins per sample and >37,000 quantified phosphosites per sample were obtained for each PDX-tumor sample with high reproducibility with respect to\",\n",
            "          \"each PDX-tumor sample with high reproducibility with respect to both the overall numbers, differentially detected proteins and phosphosites, and the biology represented. The high-quality data obtained are suitable for proteogenomic data integration and\",\n",
            "          \"obtained are suitable for proteogenomic data integration and will enable proteomics-based pan-cancer studies. The entire procedure, including sample processing and data generation, can be completed within 10 d for ten tissue samples, and 100 samples can\",\n",
            "          \"within 10 d for ten tissue samples, and 100 samples can be completed in ~4 months, using a single LC-MS/MS instrument. Although the present study focuses on tissue analysis, the procedure is equally applicable to cell lines. Supplementary Material\",\n",
            "          \"is equally applicable to cell lines. Supplementary Material Competing interests The authors declare no competing interests. Additional information Supplementary information is available for this paper at https://doi.org/10.1038/s41596-018-0006-9. Reprints\",\n",
            "          \"paper at https://doi.org/10.1038/s41596-018-0006-9. Reprints and permissions information is available at www.nature.com/reprints. Related links 1. Mertins, P. et al. Nature 534, 55\\u201362 (2016) http://dx.doi.org/10.1038/nature18003 2. Zhang, H. et al. Cell\",\n",
            "          \"http://dx.doi.org/10.1038/nature18003 2. Zhang, H. et al. Cell 166, 755\\u2013765 (2016) http://dx.doi.org/10.1016/j.cell.2016.05.069 3. Mundt, F. et al. Cancer Res. 78, 2732\\u20132746 (2018) http://dx.doi.org/10.1158/0008-5472.CAN-17-1990 Optimized workflow and\",\n",
            "          \"Optimized workflow and experimental design of global proteome and phosphoproteome analysis in tissues using TMT. a, Multiple aspects of sample handling were optimized based on a preexisting workflow for global proteome and phosphoproteome analysis\",\n",
            "          \"workflow for global proteome and phosphoproteome analysis (Mertins et al.). Some of the conditions tested relative to the preexisting workflow were (i) digestion at higher protein concentrations, which effectively increases the enzyme concentration during\",\n",
            "          \"which effectively increases the enzyme concentration during digestion, resulting in lower missed cleavage rates; (ii) reconstitution of lysyl endopeptidase in water, instead of 50 mM acetic acid, which better maintains the activity of the enzyme; (iii)\",\n",
            "          \"acid, which better maintains the activity of the enzyme; (iii) quantification of peptides by BCA before isobaric labeling, which yields more accurate input amounts than BCA at the protein level; (iv) offline basic RP fractionation using either Agilent or\",\n",
            "          \"(iv) offline basic RP fractionation using either Agilent or Waters columns, which yield equivalent results; and (v) optimization of HCD energy for each individual instrument, rather than the use of a common collision energy, which improved spectral\",\n",
            "          \"the use of a common collision energy, which improved spectral quality. The relevant steps of the Procedure are indicated in red. b, Multiple mice of basal (WHIM 2) and luminal (WHIM16) subtypes were grown, and the tumors of each subtype were pooled\",\n",
            "          \"subtypes were grown, and the tumors of each subtype were pooled together. Tumors of each subtype from multiple mice were cryofractured and aliquots of the homogenized powders were distributed to the three different laboratories for global proteome and\",\n",
            "          \"to the three different laboratories for global proteome and phosphoproteome analysis. Each laboratory analyzed 2\\u00d7 TMT-10 plexes. Intra-plex, intra-lab, and inter-lab comparisons were conducted to test depth of coverage and reproducibility. PCC1\\u20133 indicate\",\n",
            "          \"to test depth of coverage and reproducibility. PCC1\\u20133 indicate Protein Characterization Centers 1 (Broad Institute), 2 (Johns Hopkins University), and 3 (Pacific Northwest National Laboratory), respectively. BCA, bicinchoninic acid; HCD, higher-energy\",\n",
            "          \"respectively. BCA, bicinchoninic acid; HCD, higher-energy collision dissociation. a Adapted from Extended Data Fig. 1 in ref. , Springer Nature. Deep and reproducible coverage of tumor tissue proteomes and phosphoproteomes across three laboratories. a-d,\",\n",
            "          \"proteomes and phosphoproteomes across three laboratories. a-d, Bar charts depicting the number of quantified distinct peptide sequences (a) and proteins (b) identified in basic RP fractions of proteome measurements, and the number of distinct\",\n",
            "          \"fractions of proteome measurements, and the number of distinct phosphorylated peptides (c) and individual phosphorylation sites (d) quantified in the metal-affinity enriched fractions. Solid-colored bars represent the proportion of human features and\",\n",
            "          \"bars represent the proportion of human features and shaded bars represent the proportion of mouse-specific features. Numbers inside the bars represent the numbers of quantified human and mouse features, respectively. PDX models used in this study were\",\n",
            "          \"features, respectively. PDX models used in this study were approved by the institutional animal care and use committee at Washington University in St. Louis. Assessment of the variability of TMT quantitation. Our experimental design enables the assessment\",\n",
            "          \"quantitation. Our experimental design enables the assessment of intra-plex, inter-plex, and inter-laboratory variation of TMT quantitation. Pearson correlation coefficients between replicate measurements were calculated and visualized in box-and-whiskers\",\n",
            "          \"measurements were calculated and visualized in box-and-whiskers plots. a, Correlations calculated from proteome measurements comparing intra-plex replicates (left), inter-plex replicates (middle), and inter-laboratory replicates (right). b, Correlations\",\n",
            "          \"and inter-laboratory replicates (right). b, Correlations of quantified phosphorylation sites. PDX models used in this study were approved by the institutional animal care and use committee at Washington University in St. Louis. Breast cancer\",\n",
            "          \"committee at Washington University in St. Louis. Breast cancer subtype-specific protein and phosphorylation site expression identified by three laboratories. Differences in the expression of proteins and phosphorylation sites between luminal and basal\",\n",
            "          \"of proteins and phosphorylation sites between luminal and basal tumor subtypes were determined by a two-sample moderated t test at a 1% FDR. The results of the analysis are illustrated as \\u2018UpSet\\u2019 plots. Horizontal bars indicate total number of features\",\n",
            "          \"plots. Horizontal bars indicate total number of features detected by each laboratory; vertical bars depict the number of jointly detected features, as indicated by the layout matrix below. a,b, Comparison of proteins (a) and phosphorylation sites (b)\",\n",
            "          \"a,b, Comparison of proteins (a) and phosphorylation sites (b) highly expressed in the basal subtype. c,d, Comparison of proteins (c) and phosphorylation sites (d) highly expressed in the luminal subtype. Approximately two-thirds of the phosphosites that\",\n",
            "          \"subtype. Approximately two-thirds of the phosphosites that were quantified as differentially expressed by a single laboratory were also only detected by a single laboratory. PDX models used in this study were approved by the institutional animal care and\",\n",
            "          \"this study were approved by the institutional animal care and use committee at Washington University in St. Louis. Pooling fractions for proteome analysis \\t1\\t2\\t3\\t4\\t5\\t6\\t7\\t8\\t9\\t10\\t11\\t12\\t \\tA\\t\\t\\tA\\tA\\tA\\tA\\tA\\tA\\tA\\tA\\tA\\tA\\t \\tB\\t10\\t9\\t8\\t7\\t6\\t5\\t4\\t3\\t2\\t1\\tA\\tA\",\n",
            "          \"A\\t\\t\\tA\\tA\\tA\\tA\\tA\\tA\\tA\\tA\\tA\\tA\\t \\tB\\t10\\t9\\t8\\t7\\t6\\t5\\t4\\t3\\t2\\t1\\tA\\tA\\t \\tC\\t11\\t12\\t13\\t14\\t15\\t16\\t17\\t18\\t19\\t20\\t21\\t22\\t \\tD\\t10\\t9\\t8\\t7\\t6\\t5\\t4\\t3\\t2\\t1\\t24\\t23\\t \\tE\\t11\\t12\\t13\\t14\\t15\\t16\\t17\\t18\\t19\\t20\\t21\\t22\\t \\tF\\t10\\t9\\t8\\t7\\t6\\t5\\t4\\t3\\t2\\t1\\t24\\t23\\t \\tG\\t11\\t12\\t13\\t14\\t15\\t16\\t17\\t18\\t19\\t20\\t21\\t22\",\n",
            "          \"G\\t11\\t12\\t13\\t14\\t15\\t16\\t17\\t18\\t19\\t20\\t21\\t22\\t \\tH\\t\\t\\t\\t\\t\\t24\\t24\\t24\\t24\\t24\\t24\\t23\\t \\t Combining strategy for a 4.6-mm column. Underlined numbers indicate the fractions meant to be pooled into fraction 1. Italicized numbers indicate the fractions meant to be pooled into\",\n",
            "          \"numbers indicate the fractions meant to be pooled into fraction 2. These indications are meant to make pooling easier visually. Troubleshooting table Step\\tProblem\\tPossible reason\\tSolution\\t \\t44\\tLarge amounts of OCT contamination seen in QC mass\",\n",
            "          \"44\\tLarge amounts of OCT contamination seen in QC mass spectrometry runsHigh missed-cleavage rates for tryptic peptides (Supplementary Fig. 2)\\tLarge contamination of OCT in cryopulverized tissueDigest efficiency, especially for trypsin, is dependent on\",\n",
            "          \"efficiency, especially for trypsin, is dependent on enzyme concentration during digestion\\tUse basic RP fractionation to dilute OCT signal across multiple fractionsUse low volumes of extraction buffer to increase the initial protein concentrations\\t \\t73,\",\n",
            "          \"buffer to increase the initial protein concentrations\\t \\t73, 74\\tPoor labeling efficiency and/or poor mixing control\\tHigh levels of plasma contamination, leading to poor labeling efficiency or inaccurate protein BCA assay\\tIf a channel does not have\",\n",
            "          \"or inaccurate protein BCA assay\\tIf a channel does not have sufficient labeling incorporation, additional TMT is added to the sample and another 1-h incubation is performed with shaking. If the samples are too variable from each other in the mixing\",\n",
            "          \"If the samples are too variable from each other in the mixing control, additional material should be labeled and incorporated, or material should be removed, depending on the nature of the channel, until the channels are ~1:1:1\\u2026\\t \\t97\\tHigh-pH separation\",\n",
            "          \"channel, until the channels are ~1:1:1\\u2026\\t \\t97\\tHigh-pH separation resolution is <75-80% uniqueness per fraction. For optimal performance across three laboratories and different column types, see Supplementary Fig. 1\\tColumn is overloaded or deteriorated\\tUse\",\n",
            "          \"Supplementary Fig. 1\\tColumn is overloaded or deteriorated\\tUse standard runs of synthetic peptides to monitor column performance and do not load more than 4 mg of peptides onto 4.6 \\u00d7 250-mm columns\\t \\t126\\tPhosphopeptide enrichment yields <90% enrichment\",\n",
            "          \"columns\\t \\t126\\tPhosphopeptide enrichment yields <90% enrichment specificities. For optimal performance across three laboratories, see Supplementary Fig. 4\\tpH of peptides in IMAC-binding buffer is >2, IMAC beads were not prepared freshly, or IMAC incubation\",\n",
            "          \"is >2, IMAC beads were not prepared freshly, or IMAC incubation was longer than 30 min\\tCheck pH with filter paper, prepare new IMAC beads for each experiment, and do not incubate for >30 min\"\n",
            "        ]\n",
            "      }\n",
            "    ]\n",
            "  },\n",
            "  \"PDC000310\": {\n",
            "    \"study_id\": \"d7ced23a-2187-4b6c-9fb1-5bf75b6ff6f5\",\n",
            "    \"pdc_study_id\": \"PDC000310\",\n",
            "    \"study_submitter_id\": \"Reproducible Proteome and Phosphoproteome Workflow PNNL - Proteome\",\n",
            "    \"program_id\": \"10251935-5540-11e8-b664-00a098d917f8\",\n",
            "    \"project_id\": \"9c3e6e3f-ab9c-483d-9ac0-d13677e12331\",\n",
            "    \"study_name\": \"Reproducible Proteome and Phosphoproteome Workflow PNNL - Proteome\",\n",
            "    \"program_name\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"project_name\": \"CPTAC3-Other\",\n",
            "    \"disease_type\": \"Other\",\n",
            "    \"primary_site\": \"Not Reported\",\n",
            "    \"analytical_fraction\": \"Proteome\",\n",
            "    \"experiment_type\": \"TMT10\",\n",
            "    \"embargo_date\": null,\n",
            "    \"cases_count\": 2,\n",
            "    \"aliquots_count\": 20,\n",
            "    \"protocol_id\": \"07c08a35-2e8b-4719-8499-7c3d849eb707\",\n",
            "    \"protocol_submitter_id\": \"Reproducible Workflow PNNL - Proteome\",\n",
            "    \"program_submitter_id\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"protocol_name\": \"Reproducible Workflow PNNL - Proteome\",\n",
            "    \"protocol_date\": \"2021-05-28\",\n",
            "    \"publications\": [\n",
            "      {\n",
            "        \"publication_id\": \"80459042-c3d7-11eb-9f63-0a2b20c3a9e9\",\n",
            "        \"pubmed_id\": \"29988108\",\n",
            "        \"doi\": \"10.1038/s41596-018-0006-9\",\n",
            "        \"title\": \"Reproducible workflow for multiplexed deep-scale proteome and phosphoproteome analysis of tumor tissues by liquid chromatography-mass spectrometry\",\n",
            "        \"journal\": \"Nature protocols\",\n",
            "        \"journal_url\": \"https://www.nature.com/articles/s41596-018-0006-9\",\n",
            "        \"year\": \"2018\",\n",
            "        \"abstract\": \"Here we present an optimized workflow for global proteome and phosphoproteome analysis of tissues or cell lines that uses isobaric tags (TMT (tandem mass tags)-10) for multiplexed analysis and relative quantification, and provides 3\\u221a\\u00f3 higher throughput than iTRAQ (isobaric tags for absolute and relative quantification)-4-based methods with high intra- and inter-laboratory reproducibility. The workflow was systematically characterized and benchmarked across three independent laboratories using two distinct breast cancer subtypes from patient-derived xenograft models to enable assessment of proteome and phosphoproteome depth and quantitative reproducibility. Each plex consisted of ten samples, each being 300 \\u0152\\u00bag of peptide derived from <50 mg of wet-weight tissue. Of the 10,000 proteins quantified per sample, we could distinguish 7,700 human proteins derived from tumor cells and 3100 mouse proteins derived from the surrounding stroma and blood. The maximum deviation across replicates and laboratories was <7%, and the inter-laboratory correlation for TMT ratio-based comparison of the two breast cancer subtypes was r > 0.88. The maximum deviation for the phosphoproteome coverage was <24% across laboratories, with an average of >37,000 quantified phosphosites per sample and differential quantification correlations of r > 0.72. The full procedure, including sample processing and data generation, can be completed within 10 d for ten tissue samples, and 100 samples can be analyzed in ~4 months using a single LC-MS/MS instrument. The high quality, depth, and reproducibility of the data obtained both within and across laboratories should enable new biological insights to be obtained from mass spectrometry-based proteomics analyses of cells and tissues together with proteogenomic data integration.\",\n",
            "        \"full_text\": [\n",
            "          \"Reproducible workflow for multiplexed deep-scale proteome and phosphoproteome analysis of tumor tissues by liquid chromatography-mass spectrometry Here we present an optimized workflow for global proteome and phosphoproteome analysis of tissues or cell\",\n",
            "          \"global proteome and phosphoproteome analysis of tissues or cell lines that uses isobaric tags (TMT (tandem mass tags)-10) for multiplexed analysis and relative quantification, and provides 3\\u00d7 higher throughput than iTRAQ (isobaric tags for absolute and\",\n",
            "          \"3\\u00d7 higher throughput than iTRAQ (isobaric tags for absolute and relative quantification)-4-based methods with high intra- and inter-laboratory reproducibility. The workflow was systematically characterized and benchmarked across three independent\",\n",
            "          \"characterized and benchmarked across three independent laboratories using two distinct breast cancer subtypes from patient-derived xenograft models to enable assessment of proteome and phosphoproteome depth and quantitative reproducibility. Each plex\",\n",
            "          \"depth and quantitative reproducibility. Each plex consisted of ten samples, each being 300 \\u03bcg of peptide derived from <50 mg of wet-weight tissue. Of the 10,000 proteins quantified per sample, we could distinguish 7,700 human proteins derived from tumor\",\n",
            "          \"we could distinguish 7,700 human proteins derived from tumor cells and 3100 mouse proteins derived from the surrounding stroma and blood. The maximum deviation across replicates and laboratories was <7%, and the inter-laboratory correlation for TMT\",\n",
            "          \"was <7%, and the inter-laboratory correlation for TMT ratio-based comparison of the two breast cancer subtypes was r > 0.88. The maximum deviation for the phosphoproteome coverage was <24% across laboratories, with an average of >37,000 quantified\",\n",
            "          \"<24% across laboratories, with an average of >37,000 quantified phosphosites per sample and differential quantification correlations of r > 0.72. The full procedure, including sample processing and data generation, can be completed within 10 d for ten\",\n",
            "          \"and data generation, can be completed within 10 d for ten tissue samples, and 100 samples can be analyzed in \\u22124 months using a single LC-MS/MS instrument. The high quality, depth, and reproducibility of the data obtained both within and across\",\n",
            "          \"and reproducibility of the data obtained both within and across laboratories should enable new biological insights to be obtained from mass spectrometry-based proteomics analyses of cells and tissues together with proteogenomic data integration.\",\n",
            "          \"cells and tissues together with proteogenomic data integration. Introduction Genetic alterations in human cancer have been systematically mapped by genomic landscape studies in the past decade, but the direct consequences of these alterations on the\",\n",
            "          \"decade, but the direct consequences of these alterations on the functional proteome remain poorly understood. Deep-scale, mass spectrometry (MS)-based proteomic studies by the Clinical Proteomics Tumor Analysis Consortium (CPTAC) program have revealed\",\n",
            "          \"Tumor Analysis Consortium (CPTAC) program have revealed that integration of proteomic and phosphoproteomic data with genomic data for human tumor samples can improve specificity for identifying cancer-relevant pathways triggered by somatic DNA variants or\",\n",
            "          \"cancer-relevant pathways triggered by somatic DNA variants or DNA copy-number alterations, as compared with genomic characterization alone. The proteomics results also serve to elucidate pathways activated by genomic alterations and help to narrow target\",\n",
            "          \"activated by genomic alterations and help to narrow target selection for potential therapeutic intervention. Various proteomic methods have been developed for global proteome profiling by MS. Typically, proteins are reduced and alkylated to open disulfide\",\n",
            "          \"Typically, proteins are reduced and alkylated to open disulfide bonds and block newly formed and preexisting cysteine side-chain residues. These unfolded proteins are subsequently digested to peptides using the proteolytic enzyme trypsin or LysC followed\",\n",
            "          \"peptides using the proteolytic enzyme trypsin or LysC followed by trypsin to form tryptic peptides. This approach is referred to by various names, including \\u2018bottom-up\\u2019, \\u2018discovery\\u2019, \\u2018global\\u2019, and \\u2018shotgun\\u2019 proteomics. Methods exist that enable purified\",\n",
            "          \"and \\u2018shotgun\\u2019 proteomics. Methods exist that enable purified or simple mixtures of intact proteins to be analyzed (top down), but these approaches are, at present, inadequate for the comprehensive analysis of complex protein samples derived from cells or\",\n",
            "          \"analysis of complex protein samples derived from cells or tissues. For global profiling, proteolytic peptides are chromatographically separated in one or two dimensions to enable detection of low-abundance peptides, signals of which would otherwise be\",\n",
            "          \"of low-abundance peptides, signals of which would otherwise be suppressed and quantification of which would be affected due to interference by co-isolated high-abundance peptides. One-dimensional or \\u2018single-shot\\u2019 separation methods are typically performed\",\n",
            "          \"or \\u2018single-shot\\u2019 separation methods are typically performed in a label-free manner in which one sample is analyzed at a time, usually with two to three replicate injections to achieve statistical significance for confident detection of lower-abundance\",\n",
            "          \"significance for confident detection of lower-abundance proteins. Using ultra-high-pressure separations on 75-\\u03bcm inner diameter (i.d.) fused silica columns packed with sub-2 \\u03bcm C18 packing material and gradient durations of up to 5 h, samples of moderate\",\n",
            "          \"and gradient durations of up to 5 h, samples of moderate complexity, such as cancer cell lines, have been analyzed to a depth of ~6,100 proteins when using the standard, conservative metric of at least two unique peptides to identify a protein. Solid\",\n",
            "          \"of at least two unique peptides to identify a protein. Solid tumor tissues are compositionally more complex than cell lines, typically consisting of at least epithelial, stromal, and hematologic components. To detect low-abundance proteins such as\",\n",
            "          \"components. To detect low-abundance proteins such as oncogenes and tumor suppressors in a tumor tissue background or to reach a depth of >9,000 proteins in individual cell-line backgrounds, sample complexity must be reduced before LC-MS/MS. This is most\",\n",
            "          \"sample complexity must be reduced before LC-MS/MS. This is most often accomplished using a 2D separation approach involving a mode of off-line chromatography at the peptide level that is orthogonal to the final acidic reversed-phase liquid chromatography\",\n",
            "          \"to the final acidic reversed-phase liquid chromatography separation into the mass spectrometer. A variety of methods have been proposed to accomplish this at the peptide level, including off-gel electrophoresis, strong cation exchange chromatography, and\",\n",
            "          \"electrophoresis, strong cation exchange chromatography, and high pH/basic reversed-phase chromatography (bRP). Of these methods, bRP has proven to have the best balance of excellent chromatographic resolution, high peak capacity, reproducible retention\",\n",
            "          \"resolution, high peak capacity, reproducible retention times, and orthogonality to low-pH reversed-phase (RP) separation. These properties have also been shown to extend to phosphopeptide separation. A 2D separation method using bRP as the first dimension\",\n",
            "          \"A 2D separation method using bRP as the first dimension of separation, followed by analysis of concatenated fractions by LC-MS/MS, was developed and optimized by groups of the NCI-CPTAC consortium and others. These sequential high pH/low pH RP separation\",\n",
            "          \"and others. These sequential high pH/low pH RP separation approaches are equally suited for deep proteome and post-translational modification (PTM) analysis including those of phosphorylation, ubiquitination, and acetylation. This 2D approach has been\",\n",
            "          \"ubiquitination, and acetylation. This 2D approach has been demonstrated in our previous work to provide a coverage of >10,000 proteins (at \\u22652 peptides per protein) and 26,000 phosphosites per sample in cancer tissues. Proteins and phosphosites are\",\n",
            "          \"per sample in cancer tissues. Proteins and phosphosites are quantified indirectly by measuring the peptides derived from each protein by enzymatic digestion. Three methods are currently used for protein quantification. In label-free quantification, the\",\n",
            "          \"for protein quantification. In label-free quantification, the response (i.e., observed signal intensity in the mass spectrometer) of peptides or the number of peptide-to-spectrum matches (PSMs; spectral count) of peptides derived from a given protein are\",\n",
            "          \"spectral count) of peptides derived from a given protein are used to quantify that protein. In metabolic labeling, heavy isotopically labeled amino acids (typically Arg and Lys) are incorporated into proteins during cell culture (SILAC) or a mixture of\",\n",
            "          \"into proteins during cell culture (SILAC) or a mixture of labeled cell lines is used as an internal standard, a method referred to as \\u2018super SILAC\\u2019. Up to three states can be typically achieved with SILAC, and up to five can be achieved in specialized\",\n",
            "          \"with SILAC, and up to five can be achieved in specialized SILAC applications. Relative quantification is accomplished at the MS-1 level by calculating the ratios of the intensity of a given peptide between one label state and another. A limitation of the\",\n",
            "          \"between one label state and another. A limitation of the SILAC method is that humans cannot be labeled, and the multiplex level is therefore limited to two using the super SILAC approach. The third common method for protein and PTM peptide quantification\",\n",
            "          \"third common method for protein and PTM peptide quantification involves chemical labeling at the peptide level using isobaric mass-tag reagents that react with the free N-termini of peptides, as well as the side-chain primary amine of lysine residues. The\",\n",
            "          \"as well as the side-chain primary amine of lysine residues. The two most common reagents are iTRAQ and TMT. The iTRAQ reagents are available as kits with four or eight distinct labels. The TMT reagents are available as kits with six or ten distinct\",\n",
            "          \"The TMT reagents are available as kits with six or ten distinct labels, meaning that up to ten different samples can be labeled, mixed, fractionated, and analyzed by LC-MS/MS as a single \\u2018plex\\u2019. The differentially labeled peptides have identical (or\",\n",
            "          \"\\u2018plex\\u2019. The differentially labeled peptides have identical (or nearly identical) masses (i.e., they are isobaric), but their relative quantities become distinguishable after fragmentation in the mass spectrometer, which releases the mass tags from the\",\n",
            "          \"in the mass spectrometer, which releases the mass tags from the labeled peptides, generating low-mass reporter ions in the MS/MS spectra. Quantification using either the iTRAQ or TMT reagents is based on the relative intensities or ratios of the tags to\",\n",
            "          \"is based on the relative intensities or ratios of the tags to one another or to a common reference in one of the channels. Isobaric mass tagging has a number of important advantages relative to either label-free or SILAC methods. First, the isobaric\",\n",
            "          \"to either label-free or SILAC methods. First, the isobaric nature of TMT reagents leads to a summation of the abundances of peptides from individual samples. This increases the observed signals for labeled peptides and decreases the amount of any given\",\n",
            "          \"for labeled peptides and decreases the amount of any given sample required in a plex roughly by the number of channels in that plex. Second, there are few missing values in each multiplex, as each sequenced peptide can be quantified across all ten states.\",\n",
            "          \"each sequenced peptide can be quantified across all ten states. Third, for the analysis of PTMs, in which the majority of quantification events rely on the quantification of single peptides, isobaric tagging methods have been shown to improve the\",\n",
            "          \"isobaric tagging methods have been shown to improve the precision of quantification, whereas label-free methods suffer from high technical variation for PTM applications. Development of the protocol The CPTAC consortium has previously published landmark\",\n",
            "          \"protocol The CPTAC consortium has previously published landmark proteogenomic studies utilizing iTRAQ4-plex quantification for high-pH RP global proteome and phosphoproteome profiling that enabled a throughput for deep-scale profiling of 100 tumor samples\",\n",
            "          \"a throughput for deep-scale profiling of 100 tumor samples within 9 months per laboratory. In this detailed protocol, we present an improved method implementing TMT-10 quantification, which increases the throughput threefold relative to iTRAQ4, at similar\",\n",
            "          \"the throughput threefold relative to iTRAQ4, at similar proteome coverage and with high reproducibility of quantification (Fig. 1a). To study tumor patient cohorts, we typically use one isobaric channel for a common reference sample; therefore, three\",\n",
            "          \"channel for a common reference sample; therefore, three individual samples can be analyzed in an iTRAQ4-plex experiment and nine samples in a TMT-10-plex experiment. The procedure was developed for tumor samples that were cryo-fractured to particle sizes\",\n",
            "          \"for tumor samples that were cryo-fractured to particle sizes <100 \\u03bcm using a device from Covaris (see protocol for details). Typically, a 1\\u20133% weight-by-weight protein yield per wet-weight tissue can be expected with this protocol for breast tumor tissue,\",\n",
            "          \"can be expected with this protocol for breast tumor tissue, but yields can be higher depending on the type of tumor tissue analyzed. Wet weights of 50 mg per tumor sample provide sufficient amounts of total protein in the range of 0.5\\u20131.5 mg for\",\n",
            "          \"amounts of total protein in the range of 0.5\\u20131.5 mg for deep-scale proteome and phosphoproteome characterization. Proteins are extracted with an 8 M urea-based extraction buffer, reduced and alkylated, and digested first with LysC endopeptidase and\",\n",
            "          \"and alkylated, and digested first with LysC endopeptidase and subsequently with trypsin (Fig. 1a). We observed that digestion efficiencies, as measured by missed-cleavage rates, are improved by working with more-concentrated protein extracts, most likely\",\n",
            "          \"by working with more-concentrated protein extracts, most likely due to the higher enzyme concentrations in these samples (see troubleshooting section). To enable multiplexing, peptide samples are labeled with TMT-10 reagents. The TMT-labeled samples are\",\n",
            "          \"are labeled with TMT-10 reagents. The TMT-labeled samples are separated by high-pH reversed-phase liquid chromatography into 96 fractions that are combined in a step-wise manner, into 24 fractions for proteome analysis and 12 fractions for phosphoproteome\",\n",
            "          \"for proteome analysis and 12 fractions for phosphoproteome analysis. Early-, middle-, and late-eluting peptides are combined by mixing every 24th original fraction for the proteome (e.g., combining fractions 1, 25, 49, and so on) and every 12th original\",\n",
            "          \"fractions 1, 25, 49, and so on) and every 12th original fraction for the phosphoproteome analysis (e.g., combining fractions of 1, 13, 25, and so on). This step-wise concatenation strategy further improves the orthogonality of the high-pH and low-pH RP\",\n",
            "          \"further improves the orthogonality of the high-pH and low-pH RP separation steps. A total of 5% by volume of the material is used for proteome analysis, and the remaining 95% of the sample is enriched for phosphopeptides by immobilized affinity\",\n",
            "          \"sample is enriched for phosphopeptides by immobilized affinity chromatography (IMAC) with Fe3+-loaded nitrilotriacetic acid (NTA) beads. In the described protocol, off-line, high-pH RP separation is accomplished using standard HPLC equipment and columns\",\n",
            "          \"is accomplished using standard HPLC equipment and columns packed with 3.5-\\u03bcm C18 beads, whereas the second, online acidic RP separation uses ultra-high-performance HPLC (UHPLC) instruments and columns packed with 1.9-\\u03bcm C18 beads. The datasets described\",\n",
            "          \"columns packed with 1.9-\\u03bcm C18 beads. The datasets described in the Anticipated results section were acquired on high-performance Orbitrap Fusion Lumos instruments in three independent laboratories. All three laboratories generated highly reproducible\",\n",
            "          \"All three laboratories generated highly reproducible datasets by following this procedure, with coverages of >10,000 proteins (\\u22652 unique peptides per protein) and >37,000 phosphorylation sites. In studies of human-only breast tumor tissue, we have\",\n",
            "          \"sites. In studies of human-only breast tumor tissue, we have achieved typical coverage depth of >8,000 proteins per experiment (\\u22652 unique peptides per protein) and >25,000 phosphorylation sites per experiment using the protocol described here (P.M.,\",\n",
            "          \"sites per experiment using the protocol described here (P.M., L.C.T., K.R.C., D.R.M., F.M. and S.A.C., unpublished data; the data are accessible at https://cptac-data-portal.georgetown.edu/cptac/s/S039?: Study name, \\u2018CPTAC breast cancer confirmatory\",\n",
            "          \"Study name, \\u2018CPTAC breast cancer confirmatory study\\u2019). The optimized protocol enables deep-scale and reproducible results to be obtained within and across laboratories conducting tissue or cell-line analyses, producing high-quality data for proteogenomic\",\n",
            "          \"analyses, producing high-quality data for proteogenomic data integration and facilitating proteomics-based pan-cancer studies. Information on how to identify and manage common sources of variability, such as proteolytic digest efficiency and overlabeling\",\n",
            "          \"such as proteolytic digest efficiency and overlabeling with TMT-10 reagents, is also provided. Applications of the method Although this protocol was developed for tissue samples, it is also directly applicable to analysis of mammalian cell culture\",\n",
            "          \"also directly applicable to analysis of mammalian cell culture samples. The starting material amounts for 300 \\u03bcg of peptide per sample/TMT channel described here allow for reproducible generation of deep-scale proteome and phosphoproteome datasets. In\",\n",
            "          \"of deep-scale proteome and phosphoproteome datasets. In addition, the unbound material after phosphopeptide capture (Fig. 1a) can be used for deep lysine acetylation profiling, as has been described previously (refs ). If sample amounts are limited, lower\",\n",
            "          \"previously (refs ). If sample amounts are limited, lower peptide quantities (50\\u2013100 \\u03bcg of peptides per TMT channel) can be used. Reduced amounts do not affect overall proteome coverage or quantification but will reduce phosphoproteome coverage by 20\\u201330%\",\n",
            "          \"but will reduce phosphoproteome coverage by 20\\u201330% (P.M., L.C.T., K.R.C., D.R.M., F.M. and S.A.C., unpublished data). For proteome quantification, only 0.5\\u20131.0 \\u03bcg of peptides per bRP fraction is needed to achieve the depth of proteome coverage described\",\n",
            "          \"is needed to achieve the depth of proteome coverage described here, so for a 24-fraction proteome analysis, typically, a minimum of 50 \\u03bcg of peptides per TMT-10-plex or 5 \\u03bcg per sample is needed. The 300-\\u03bcg amount of input peptide is needed for deep-scale\",\n",
            "          \"The 300-\\u03bcg amount of input peptide is needed for deep-scale phosphoproteome analysis. Improvements in isobaric tagging chemistries already allow higher degrees of multiplexing, for example, by using TMT-11 reagents, which provide an 11th multiplexing\",\n",
            "          \"by using TMT-11 reagents, which provide an 11th multiplexing channel. The higher the degree of multiplexing, the more sample fractions and MS time can be dedicated to a given multiplex experiment to increase proteome coverage at a similar time and cost.\",\n",
            "          \"to increase proteome coverage at a similar time and cost. Current limitations and future directions Although the present protocol represents the current state of the art, advances in sample processing, chromatography, and MS technologies, as well as data\",\n",
            "          \"chromatography, and MS technologies, as well as data analysis methods, will inevitably enable this multistep protocol to be improved, including reducing the amount of input protein per sample required to achieve the depth we report. When considering\",\n",
            "          \"required to achieve the depth we report. When considering changes to any individual step, it is critical to consider the impact on the entire protocol in terms of sensitivity, dynamic range, throughput, speed, cost, time, and other relevant factors. Not\",\n",
            "          \"throughput, speed, cost, time, and other relevant factors. Not every possible sample-processing variable was examined in this study, and it is possible that further refinements resulting in additional incremental improvements can be realized. Many such\",\n",
            "          \"additional incremental improvements can be realized. Many such process modifications will represent tradeoffs, however. For example, increasing the temperature at which enzymatic digestions are carried out could decrease the overall time required for\",\n",
            "          \"are carried out could decrease the overall time required for digestion and decrease the percentage of missed cleavages observed. However, the extent of carbamylation of peptide N termini and lysines will also certainly increase at elevated temperatures,\",\n",
            "          \"lysines will also certainly increase at elevated temperatures, and this may have an effect on quantification. Similarly, the higher-multiplex reagents that are becoming available will further increase sample-analysis throughput, albeit at some\",\n",
            "          \"further increase sample-analysis throughput, albeit at some to-be-determined cost to depth of proteome and phosphoproteome coverage. The described protocol benefits from the use of UHPLC in the second separation dimension, but the first dimension high-pH\",\n",
            "          \"second separation dimension, but the first dimension high-pH RP separation is performed using conventional HPLC instrumentation. Further improvements in semipreparative column technology with smaller bead sizes and the use of the latest high-flow UHPLC\",\n",
            "          \"smaller bead sizes and the use of the latest high-flow UHPLC instrumentation may enhance the separation power and increase the uniqueness per fraction to greater than the 75% achieved here (Supplementary Fig. 1). The more unique each peptide fraction is,\",\n",
            "          \"Fig. 1). The more unique each peptide fraction is, the more efficiently each individual fraction\\u2019s LC-MS/MS run contributes to the unique peptide count of the overall experiment. The number of fractions collected per sample gates the overall analysis\",\n",
            "          \"of fractions collected per sample gates the overall analysis throughput, so reducing this number is highly desirable. Maintaining the sensitivity of the online LC-MS/MS analysis when analyzing fewer first-dimension high-pH RP fractions would require both\",\n",
            "          \"fewer first-dimension high-pH RP fractions would require both increased peak capacity of the online column (e.g., through the use of smaller-dimension packing material or a longer online column) and loading of more peptide weight per fraction to maintain\",\n",
            "          \"and loading of more peptide weight per fraction to maintain the same total weight of peptides used in the overall analysis. Reducing the number of fractions analyzed by LC-MS/MS would reduce the number of MS/MS spectra acquired and would thereby diminish\",\n",
            "          \"the number of MS/MS spectra acquired and would thereby diminish the depth of coverage, unless the repeat peptide observation rate falls, the MS/MS interpretation rate rises, or the MS/MS scan rate increases. TMT-labeling chemistry can result in low levels\",\n",
            "          \"rate increases. TMT-labeling chemistry can result in low levels of overlabeling, whereby an additional TMT label is incorporated into peptides via O-acylation of Ser, Thr, and Tyr sites with a nearby His residue, which increases the nucleophilicity of the\",\n",
            "          \"nearby His residue, which increases the nucleophilicity of the side-chain hydroxyl group via hydrogen bonding with the histidyl imidazole group. We do not routinely allow for these variable modifications in our database searches because allowing for\",\n",
            "          \"modifications in our database searches because allowing for overlabeling substantially increases the search space (3\\u201310\\u00d7). Instead, we are working to routinely measure the extent of overlabeling present and adjust our TMT-labeling methodology to diminish\",\n",
            "          \"present and adjust our TMT-labeling methodology to diminish the overlabeling rate. Adjustments in the labeling reaction conditions, such as use of lower pH, use of ethanol instead of acetonitrile in the labeling step, decreased TMT reagent-to-substrate\",\n",
            "          \"in the labeling step, decreased TMT reagent-to-substrate ratios, and/or shorter reaction times, may lead to a decrease in the proportion of MS/MS spectra attributable to overlabeling. Multiplexed reporter-ion quantification in MS2 mode provides many\",\n",
            "          \"reporter-ion quantification in MS2 mode provides many advantages, as listed above, and new chemistries enabling even higher levels of multiplexing beyond 11 plex are anticipated. However, some limitations have been noted. The reagents are expensive, but\",\n",
            "          \"limitations have been noted. The reagents are expensive, but lower-cost alternatives that match the multiplex level are not currently available. The tags also increase the overall charge state of the labeled peptides, which can decrease identification\",\n",
            "          \"of the labeled peptides, which can decrease identification rates, relative to label-free methods. Most importantly, the accuracy of quantification is distorted by co-isolation phenomena. Reporterion signals for a peptide of interest are contributed to by\",\n",
            "          \"signals for a peptide of interest are contributed to by not only the primary precursor ion isolated but also background signals with similar m/z values that are co-isolated. Co-isolation correction approaches have been described, but these need to be\",\n",
            "          \"correction approaches have been described, but these need to be further improved to enhance the accuracy of TMT ratio-based quantification. Strategies to mitigate this effect include decreasing the sample and background complexity by extensive\",\n",
            "          \"decreasing the sample and background complexity by extensive fractionation, decreasing the isolation m/z window size before MS2 scans, using MS3 fragmentation strategies, and, very importantly, designing TMT experiments and using TMT datasets in ways that\",\n",
            "          \"designing TMT experiments and using TMT datasets in ways that compensate for co-isolation problems. Other foreseeable data acquisition methods may also diminish co-isolation problems. Ion mobility (IM) is a gas-phase ion separation method that separates\",\n",
            "          \"(IM) is a gas-phase ion separation method that separates ions on a millisecond time scale before the microsecond time scale MS analyses that occur in the mass spectrometer. The addition of FAIMS (field asymmetric ion mobility spectrometry), which is\",\n",
            "          \"of FAIMS (field asymmetric ion mobility spectrometry), which is already commercially available, to the front end of current high-performance MS instruments can improve the accuracy of quantification using isobaric labeling methods by discriminating\",\n",
            "          \"using isobaric labeling methods by discriminating against singly charged interferences. This and other potential advantages of FAIMS in the context of proteomics experiments have been described. In the future, devices such as SLIM (structures for lossless\",\n",
            "          \"In the future, devices such as SLIM (structures for lossless ion manipulations) may provide even higher sensitivity, resolution, and speed, as compared with current devices. Diminished co-isolation and increased sensitivity should also be achievable by\",\n",
            "          \"and increased sensitivity should also be achievable by increasing the chromatographic resolution of the online chromatography with longer columns, more efficient and uniform column packing, or use of smaller bead sizes to the extent the column back\",\n",
            "          \"or use of smaller bead sizes to the extent the column back pressure can be handled by the LC pumping system. Narrower chromatographic peaks should increase sensitivity and decrease the spectrum acquisition rate by increasing each peptide\\u2019s effective\",\n",
            "          \"acquisition rate by increasing each peptide\\u2019s effective concentration at its chromatographic peak apex. Experimental design One of the greatest advantages of isotopic labeling techniques in MS is that peptides in multiplexed samples are measured\",\n",
            "          \"in MS is that peptides in multiplexed samples are measured simultaneously, limiting the effect of detection and quantification interference by variable background signals that is observed in label-free experiments. For this reason, we primarily use TMT\",\n",
            "          \"label-free experiments. For this reason, we primarily use TMT quantification as a relative quantification technique. Samples of interest are directly referenced to other samples or internal reference samples within the same multiplexed experiments, and\",\n",
            "          \"reference samples within the same multiplexed experiments, and intensity ratios are used for all following analyses. The use of internal reference samples is of particular use for larger sample cohorts, as has been demonstrated in our previous analysis of\",\n",
            "          \"cohorts, as has been demonstrated in our previous analysis of 80 breast tumor samples and 174 ovarian cancer samples. Internal reference samples should be designed to contain the average protein concentrations of all different sample types to be\",\n",
            "          \"protein concentrations of all different sample types to be investigated in a cohort of multiplexed experiments. To avoid batch effects, all different constituent sample types should be evenly distributed across multiple TMT-plex experiments. It is also\",\n",
            "          \"distributed across multiple TMT-plex experiments. It is also important to consider the effects of isotopic impurities that lead to reporter intensity cross-contributions of up to 5% to neighboring TMT channels. Correction approaches for isotopic\",\n",
            "          \"to neighboring TMT channels. Correction approaches for isotopic impurities based on the impurity factors provided by the manufacturer have been previously described and were adapted here for TMT-10 applications. Additional caution should be applied when\",\n",
            "          \"TMT-10 applications. Additional caution should be applied when samples of extremely different compositions, such as human and xenograft tumors, are placed in the same multiplexed experiments. To resolve and accurately measure the near-isobaric N- and\",\n",
            "          \"To resolve and accurately measure the near-isobaric N- and C-labeled reporter ions in the TMT-10 reagents, the instrument used must be able to achieve a minimum MS/MS resolution of 50,000 at m/z 150. Compatible instruments include the Thermo QE series of\",\n",
            "          \"m/z 150. Compatible instruments include the Thermo QE series of mass spectrometers (QE, QEplus, QE-HF, and QE-HF-X), as well as the Fusion and Lumos instruments also from Thermo. The QE series of instruments lack the linear ion trap that constitutes the\",\n",
            "          \"of instruments lack the linear ion trap that constitutes the \\u2018back-end\\u2019 of the Fusion and Lumos Tribrid mass spectrometers. As the linear trap portion of the Lumos was not used in this protocol, the less expensive QE series instruments may be used, with\",\n",
            "          \"the less expensive QE series instruments may be used, with results differing depending on the model employed. Of the QE systems, only the QE-HF-X has what is effectively the same ion optical front end as the Lumos, and therefore results obtained when\",\n",
            "          \"front end as the Lumos, and therefore results obtained when using this instrument are expected to be of similar depth and overall quality and reproducibility as those obtained when using the Lumos in the manner described in this protocol, provided that\",\n",
            "          \"Lumos in the manner described in this protocol, provided that the data acquisition control feature \\u2018advanced peak determination\\u2019 (APD) can be turned off, to minimize the acquisition of MS/MS spectra with co-isolated precursors. Some instrument types, such\",\n",
            "          \"with co-isolated precursors. Some instrument types, such as orthogonal time-of-flight mass spectrometers, may not have sufficient mass resolution and therefore may not be compatible with the use of TMT-10 reagents. In such cases, a lower multiplex reagent\",\n",
            "          \"of TMT-10 reagents. In such cases, a lower multiplex reagent such as TMT-6 that does not have as high a resolution requirement may be used, with consequent decrease in overall sample analysis throughput as well as differences in the depth of analysis.\",\n",
            "          \"throughput as well as differences in the depth of analysis. Materials Reagents Cells or tissue samples. The patient-derived xenograft tumors used in this study were from previously established basal (WHIM2) and luminal (WHIM16) breast cancers. The\",\n",
            "          \"basal (WHIM2) and luminal (WHIM16) breast cancers. The xenograft tumors were grown subcutaneously in 8-week-old NOD.Cg-Prkdescid Il2rgtm1Wjl/SzJ mice (Jackson Laboratories, strain code 005557). All patient-derived xenograft (PDX) models are available\",\n",
            "          \"All patient-derived xenograft (PDX) models are available through the application from the Human and Mouse-Linked Evaluation of Tumor\\u2019s Core at http://digitalcommons.wustl.edu/ hamlet/. Xenograft tumors were grown in multiple animals. Tumor pieces for each\",\n",
            "          \"tumors were grown in multiple animals. Tumor pieces for each subtype were cryopulverized, and the tissue powder was mixed in an aluminum weighing boat on dry ice, as previously described. Tissue was stored at \\u221280 \\u00b0C until shipment on dry ice to each of\",\n",
            "          \"was stored at \\u221280 \\u00b0C until shipment on dry ice to each of the three participating laboratories. A sufficient amount of material was generated for all three laboratories for the duration of the project to prevent batch effects with regard to the xenograft\",\n",
            "          \"project to prevent batch effects with regard to the xenograft material generation. Full proteome and phosphoproteome process replicates for each of the two xenografts were prepared as described in this study. The details for preparing bulk cryopulverized\",\n",
            "          \"in this study. The details for preparing bulk cryopulverized tissue in a sustained frozen state can be found in the supplementary methods of Mertins et al. ! CAUTION Informed consent must be in place when using patient tissues. Human material should be\",\n",
            "          \"in place when using patient tissues. Human material should be handled in a Biosafety Level 2 (or higher) environment and with strict attention to Biosafety Level 2 procedures. Refer to the Centers for Disease Control website\",\n",
            "          \"2 procedures. Refer to the Centers for Disease Control website (http://www.cdc.gov/biosafety/publications/bmbl5/BMBL5_sect_IV.pdf) for additional information ! CAUTION All experiments with live mice should be performed according to institutional and\",\n",
            "          \"live mice should be performed according to institutional and national regulations. The procedures in this protocol were reviewed and approved by the institutional animal care and use committee at Washington University in St. Louis. Urea (Sigma-Aldrich,\",\n",
            "          \"at Washington University in St. Louis. Urea (Sigma-Aldrich, cat. no. U0631\\u20131KG; or Thermo Scientific, cat. no. 29700\\u20131KG) ! CAUTION Urea is a health hazard level 2 compound. Avoid contact with skin and eyes, and avoid inhalation. Tris (hydroxymethyl)\",\n",
            "          \"with skin and eyes, and avoid inhalation. Tris (hydroxymethyl) aminomethane hydrochloride (Tris-HCl; Ambion, cat. no. AM9855G; or Sigma-Aldrich, cat. no. T2694\\u20131L) ! CAUTION Tris-HCl is a health hazard level 2 compound. Avoid contact with skin and eyes,\",\n",
            "          \"hazard level 2 compound. Avoid contact with skin and eyes, and avoid inhalation. Sodium chloride (NaCl; Sigma-Aldrich) Ethylenediaminetetraacetic acid (EDTA; Sigma-Aldrich, cat. no. E7889) ! CAUTION EDTA is a health hazard level 1 compound. Avoid contact\",\n",
            "          \"CAUTION EDTA is a health hazard level 1 compound. Avoid contact with skin and eyes, and avoid inhalation. Aprotinin (Sigma-Aldrich, cat. no. A610) Sodium fluoride (NaF; Sigma-Aldrich, cat. no. S7920) ! CAUTION NaF is a health hazard level 3 compound.\",\n",
            "          \"no. S7920) ! CAUTION NaF is a health hazard level 3 compound. Avoid contact with skin and eyes, and avoid inhalation. Leupeptin (Roche, cat. no. 11017101001) PMSF (Sigma-Aldrich, cat. no. 93482) Phosphatase inhibitor cocktail 2 (Sigma-Aldrich, cat. no.\",\n",
            "          \"Phosphatase inhibitor cocktail 2 (Sigma-Aldrich, cat. no. P5726) Phosphatase inhibitor cocktail 3 (Sigma-Aldrich, cat. no. P0044) O-(2-Acetamido-2-deoxy-D-glucopyranosylidenamino) N-phenylcarbamate (PUGNAc; Sigma-Aldrich, cat. no. A7229) DTT (Pierce, cat.\",\n",
            "          \"(PUGNAc; Sigma-Aldrich, cat. no. A7229) DTT (Pierce, cat. no. 20291) ! CAUTION DTT is a health hazard level 2 compound. Avoid contact with skin and eyes, and avoid inhalation. Iodoacetamide (IAM; Sigma-Aldrich, cat. no. A3221) ! CAUTION IAM is a health\",\n",
            "          \"(IAM; Sigma-Aldrich, cat. no. A3221) ! CAUTION IAM is a health hazard level 3 compound with a reactivity level of 1. Avoid contact with skin and eyes, and avoid inhalation. Lysyl endopeptidase (LysC; Wako Chemicals, cat. no. 129\\u201302541) Sequencing-grade\",\n",
            "          \"(LysC; Wako Chemicals, cat. no. 129\\u201302541) Sequencing-grade modified trypsin (500 \\u03bcg per vial or 5 \\u00d7 20 \\u03bcg per vial; Promega, cat. no. V511X or V5113) BCA bicinchoninic acid Protein Assay Kit (Pierce, cat. no. 23225) Formic acid (FA; Sigma-Aldrich, cat.\",\n",
            "          \"(Pierce, cat. no. 23225) Formic acid (FA; Sigma-Aldrich, cat. no. 56302) ! CAUTION FA is a health hazard level 4 compound with a flammability level of 3. Avoid contact with skin and eyes, and avoid inhalation. Use it in a well-ventilated area per MSDS\",\n",
            "          \"and avoid inhalation. Use it in a well-ventilated area per MSDS recommendations. Store at room temperature (RT; 25 \\u00b0C) for up to 6 months. Trifluoroacetic acid (TFA; Sigma-Aldrich, cat. no. 91707) ! CAUTION TFA is a health hazard level 3 compound with a\",\n",
            "          \"91707) ! CAUTION TFA is a health hazard level 3 compound with a flammability level of 1 and reactivity of 1. Avoid contact with skin or eyes, and avoid inhalation. Acetonitrile (MeCN; J.T. Baker, cat. no. 9829\\u201303) ! CAUTION MeCN is a health hazard level 4\",\n",
            "          \"cat. no. 9829\\u201303) ! CAUTION MeCN is a health hazard level 4 compound with a flammability level of 2. Avoid contact with skin and eyes, and avoid inhalation. Store and use it in a well-ventilated area per MSDS recommendations. Store at RT for up to 6\",\n",
            "          \"area per MSDS recommendations. Store at RT for up to 6 months. \\u25b2 CRITICAL Requires LC/MS-grade quality. Methanol (MeOH; Fluka, cat. no. 34966) ! CAUTION MeOH is a health hazard level 1 compound with a flammability level of 3 and an instability/reactivity\",\n",
            "          \"with a flammability level of 3 and an instability/reactivity level of 0. Avoid contact with skin and eyes, and avoid inhalation. Store and use it in a well-ventilated area per MSDS recommendations. Store at RT for up to 6 months. Ammonium hydroxide\",\n",
            "          \"Store at RT for up to 6 months. Ammonium hydroxide solution (28% (wt/vol) NH4OH; Sigma-Aldrich, cat. no. 338818) ! CAUTION Ammonium hydroxide is a health hazard level 4 compound. Avoid contact with skin and eyes, and avoid inhalation. HPLC-grade water\",\n",
            "          \"with skin and eyes, and avoid inhalation. HPLC-grade water (J.T. Baker, cat. no. 4218\\u201303) TMT-10 Reagent Kit (Thermo Fisher Scientific, cat. no. 90046) HEPES (0.5 M buffer solution, pH 8.5, liquid; Alfa Aesar, cat. no. J63218) 50% (vol/vol) Hydroxylamine\",\n",
            "          \"Alfa Aesar, cat. no. J63218) 50% (vol/vol) Hydroxylamine solution (Sigma-Aldrich, cat. no. 467804) ! CAUTION A 50% (vol/vol) hydroxylamine solution is a health hazard level 2 compound. Avoid contact with skin or eyes, and avoid inhalation. Acetic acid,\",\n",
            "          \"contact with skin or eyes, and avoid inhalation. Acetic acid, glacial (EMD Millipore, cat. no. AX0074\\u20136) ! CAUTION Acetic acid is a health hazard level 3 compound with a flammability level of 2 and reactivity of 0. Avoid contact with skin or eyes, and\",\n",
            "          \"of 2 and reactivity of 0. Avoid contact with skin or eyes, and avoid inhalation. mColorpHast pH test strips (5\\u201310 range; VWR, cat. no. EM1.09533.0001) Ni-NTA superflow agarose beads (Qiagen, cat. no. 30410) Iron (III) chloride (Sigma-Aldrich, cat. no.\",\n",
            "          \"cat. no. 30410) Iron (III) chloride (Sigma-Aldrich, cat. no. 451649) ! CAUTION Iron (III) chloride is a health hazard level 2 compound. May be corrosive to metals. Avoid contact with skin or eyes, and avoid inhalation/ingestion. Potassium phosphate,\",\n",
            "          \"or eyes, and avoid inhalation/ingestion. Potassium phosphate, dibasic (Sigma-Aldrich, cat. no. P3786) Potassium phosphate, monobasic (Sigma-Aldrich, cat. no. P0662) Ethanol (EtOH; Sigma-Aldrich, cat. no. 270741) ! CAUTION EtOH is a health grade level 2\",\n",
            "          \"cat. no. 270741) ! CAUTION EtOH is a health grade level 2 compound with a flammability level of 3 and reactivity level of 0. Synthetic peptide standards: we use an in-house set of synthetic peptides (Reagent setup), but commercial peptide standard\",\n",
            "          \"peptides (Reagent setup), but commercial peptide standard alternatives include the Pierce Peptide Retention Time Calibration Mixture (Thermo Fisher Scientific, cat. no. 88320) or custom synthetic peptides from commercial vendors such as New England\",\n",
            "          \"synthetic peptides from commercial vendors such as New England Peptide. ReproSil-Pur (120 A, C18-AQ, 1.9-\\u03bcm resin; Dr. Maisch, cat. no. r119.aq) Equipment Needle to make stage tips (laboratory pipetting needles with 90\\u00b0 blunt ends, 16-gauge, 2-inch\",\n",
            "          \"pipetting needles with 90\\u00b0 blunt ends, 16-gauge, 2-inch length; Cadence Science, cat. no. 7938) Puncher to make stage tips Tubing (PEEK, 25 \\u03bcm \\u00d7 1/32 \\u00d7 5 feet, natural; Idex Health & Science, cat. no. 1567) Stage-tip C18 material (solid-phase C18\",\n",
            "          \"Science, cat. no. 1567) Stage-tip C18 material (solid-phase C18 extraction disks, diam. = 47 mm, 20 pack; Empore, cat. no. 66883-U) Adapter for stage tipping (Glygen, cat. no. CEN.24) BCA plate (96-well microplate, flat bottom, clear; Greiner Bio-One,\",\n",
            "          \"plate (96-well microplate, flat bottom, clear; Greiner Bio-One, cat. no. 655101) SepPak tC18 3cc Vac cartridges (200 mg of sorbent per cartridge; Waters Technologies, cat. no. WAT054925) Offline HPLC system (Agilent, 1100 series HPLC instrument or\",\n",
            "          \"Offline HPLC system (Agilent, 1100 series HPLC instrument or similar) Offline HPLC column (3.5-\\u03bcm, 4.6 \\u00d7 250 mm; Agilent, model no. Zorbax 300 Extend-C18) Offline fractionation plate (96-well, 2-ml polypropylene, round-bottom; Whatman Microplate Devices\",\n",
            "          \"2-ml polypropylene, round-bottom; Whatman Microplate Devices Uniplate, cat. no. 7701\\u20135200) 1-ml Autosampler vial for bRP (National Scientific, cat. no. C4010\\u201314) 1-ml Autosampler cap for bRP (National Scientific, cat. no. C4010\\u201355A) LC system for online\",\n",
            "          \"(National Scientific, cat. no. C4010\\u201355A) LC system for online LC-MS analysis (Easy Nano liquid chromatography instrument, Thermo Fisher Scientific, model no. EASY-nLC 1200, or similar) \\u25b2 CRITICAL We use a Proxeon Easy-nLC 1200 and operate under\",\n",
            "          \"\\u25b2 CRITICAL We use a Proxeon Easy-nLC 1200 and operate under ultra-performance liquid chromatography (UPLC) conditions. However, any LC system that can deliver nanoflow rates and can operate up to a pressure of 1,000 bar can be used for peptide separation.\",\n",
            "          \"to a pressure of 1,000 bar can be used for peptide separation. MS system for online LC-MS analysis (Thermo Fisher Scientific, Orbitrap Fusion Lumos model) \\u25b2 CRITICAL Although we use an Orbitrap Fusion Lumos, other LC-MS/MS systems could be used, as long\",\n",
            "          \"Fusion Lumos, other LC-MS/MS systems could be used, as long as they have sufficient resolution in MS/MS mode to resolve the low-mass N and C series TMT-10 reporter ions. To resolve and accurately measure the near-isobaric N- and C-labeled reporter ions in\",\n",
            "          \"measure the near-isobaric N- and C-labeled reporter ions in the TMT-10 reagents, the instrument used must be able to achieve a minimum MS/MS resolution of 50,000 at m/z 150. Compatible instruments include the Thermo Fisher Scientific QE series of mass\",\n",
            "          \"include the Thermo Fisher Scientific QE series of mass spectrometers (QE, QEplus, QE-HF, and QE-HF-X), as well as the Fusion and Lumos instruments also from Thermo Fisher Scientific. If the data acquisition control feature APD is present, it should be\",\n",
            "          \"data acquisition control feature APD is present, it should be turned off to minimize the acquisition of MS/MS spectra with co-isolated precursors. PicoFrit column (360-\\u03bcm outer diameter (o.d.) \\u00d7 75-\\u03bcm i.d., 10-\\u03bcm i.d. tip, 50-cm length; New Objective,\",\n",
            "          \"\\u00d7 75-\\u03bcm i.d., 10-\\u03bcm i.d. tip, 50-cm length; New Objective, cat. no. PF360\\u201375-10-N-5) 300-\\u03bcl Autosampler vial for LC-MS (Waters, cat. no. 186002639) 300-\\u03bcl Autosampler cap for LC-MS (Waters, cat. no. 186000305) 20-cm Nanospray column heater (Phoenix S&T,\",\n",
            "          \"cat. no. 186000305) 20-cm Nanospray column heater (Phoenix S&T, cat. no. PST-CH-20U) Column heater controller (Phoenix S&T, cat. no. PST-CHC) Vacuum centrifuge (Electron Savant SpeedVac Concentrator; Thermo Fisher Scientific, model no. SPD121P)\",\n",
            "          \"Concentrator; Thermo Fisher Scientific, model no. SPD121P) Lyophilizer (Freezone 4.5; Labconco, cat. no. 7750020) Vortex (Scientific Industries, model no. Vortex-Genie 2) Incubator shaker (New Brunswick, model no. 22331) Shaker (Thermomixer; Eppendorf,\",\n",
            "          \"Brunswick, model no. 22331) Shaker (Thermomixer; Eppendorf, model no. C1213) Tabletop centrifuge (Galaxy Mini; VWR, model no. C1213) Tubes (microtubes, polypropylene; 1.5 ml, 2.0 ml, and 1.5 ml with cap; Sarstedt, cat. nos. 72.607, 72.693, 72.692)\",\n",
            "          \"1.5 ml with cap; Sarstedt, cat. nos. 72.607, 72.693, 72.692) Cryopulverizer (Covaris, model no. CP02 cryoPREP Impactor) Tissue bags (tissueTUBE TT1 and tissueTUBE TT1 plug) (Covaris, cat. nos. 520001, 520006) Software Spectrum Mill MS Proteomics Workbench\",\n",
            "          \"520001, 520006) Software Spectrum Mill MS Proteomics Workbench v6.0 (Agilent Technologies, https://www.agilent.com/en/ products/software-informatics/masshunter-suite/masshunter-for-life-science-research/spectrum-mill) \\u25b2 CRITICAL Although we used Spectrum\",\n",
            "          \"\\u25b2 CRITICAL Although we used Spectrum Mill, other software packages can be used that support the identification and quantitation of high-resolution LC-MS/MS with TMT-10 reagents. Throughout the data analysis portions of the protocol, we have tried to\",\n",
            "          \"the data analysis portions of the protocol, we have tried to explain the metrics, parameters, and approaches that may be somewhat different in other software packages. All mass spectra contributing to this study can be downloaded in the original\",\n",
            "          \"contributing to this study can be downloaded in the original instrument vendor format from: https://cptac-data-portal.georgetown.edu/cptac/s/S036 for the study name \\u2018reproducible workflow for multiplexed deep-scale analysis of the proteome and\",\n",
            "          \"for multiplexed deep-scale analysis of the proteome and phosphoproteome of tumor tissues by LC-MS\\u2019. Reagent setup Stock solutions for lysis buffer Stock solutions are: 1 M Tris HCl (pH 8.0), 1 M NaCl, 500 mM EDTA, 1 mg/ml aprotinin, 2 mg/ml leupeptin, 100\",\n",
            "          \"M NaCl, 500 mM EDTA, 1 mg/ml aprotinin, 2 mg/ml leupeptin, 100 mM PMSF in ethanol, 1 M NaF, Phosphatase Inhibitor Cocktail 2, Phosphatase Inhibitor Cocktail 3, 14 mM PUGNAc. Stock solutions for lysis buffer can be made in advance and stored for up to 6\",\n",
            "          \"for lysis buffer can be made in advance and stored for up to 6 months, if stored properly. 1 M Tris-HCl (pH 8.0), 1 M NaCl, and 500 mM EDTA can be stored at RT for up to 12 months. 1 mg/ml aprotinin, 1 M NaF, and Phosphatase Inhibitor Cocktail 2 can be\",\n",
            "          \"aprotinin, 1 M NaF, and Phosphatase Inhibitor Cocktail 2 can be stored at 4 \\u00b0C for up to 6 months. 2 mg/ml leupeptin, 100 mM PMSF in ethanol, 14 mM PUGNAc, and Phosphatase Inhibitor Cocktail 3 can be stored at \\u221220 \\u00b0C for up to 6 months. Urea lysis buffer\",\n",
            "          \"3 can be stored at \\u221220 \\u00b0C for up to 6 months. Urea lysis buffer Urea lysis buffer contains 8 M urea, 75 mM NaCl, 50 mM Tris (pH 8.0), 1 mM EDTA, 2 \\u03bcg/ml aprotinin, 10 \\u03bcg/ml leupeptin, 1 mM PMSF, 1:100 (vol/vol) Phosphatase Inhibitor Cocktail 2, 1:100\",\n",
            "          \"PMSF, 1:100 (vol/vol) Phosphatase Inhibitor Cocktail 2, 1:100 (vol/vol) Phosphatase Inhibitor Cocktail 3, 10 mM NaF, and 20 \\u03bcM PUGNAc. \\u25b2 CRITICAL Urea lysis buffer should be made fresh each time immediately before use. Make sure that the urea goes into\",\n",
            "          \"time immediately before use. Make sure that the urea goes into solution before adding the additional enzymes. Add aprotinin, leupeptin, PMSF, NaF, Phosphatase Inhibitor Cocktail 2, Phosphatase Inhibitor Cocktail 3, and PUGNAc to the urea solution while it\",\n",
            "          \"Inhibitor Cocktail 3, and PUGNAc to the urea solution while it is on ice. Do not vortex after adding the additives. Swirling is sufficient. In addition, the additives should be added only immediately before use. The solution should be kept on ice once the\",\n",
            "          \"before use. The solution should be kept on ice once the additives are added. BSA curve for BCA protein assay Using the 2 mg/ml albumin stock included in the Pierce BCA Protein Assay Kit, make a serial dilution curve with HPLC-grade water. The 8-point\",\n",
            "          \"make a serial dilution curve with HPLC-grade water. The 8-point curve we use includes albumin concentrations of 2 mg/ml, 1.5 mg/ml, 1.0 mg/ml, 0.75 mg/ml, 0.5 mg/ml, 0.25 mg/ml, 0.125 mg/ml, and 0.0625 mg/ml. The BSA curve can be made in advance and\",\n",
            "          \"and 0.0625 mg/ml. The BSA curve can be made in advance and stored at 4 \\u00b0C for up to a year. Preparation of digestion enzymes Dissolve lyophilized 10 AU Wako LysC in 20 ml of HPLC water for a final concentration of 0.5 AU/ml. Reconstituted LysC should be\",\n",
            "          \"final concentration of 0.5 AU/ml. Reconstituted LysC should be stored at \\u221280 \\u00b0C, can be made in advance, and is stable in storage for up to 6 months. Promega-sequencing grade modified trypsin can be purchased as 5 \\u00d7 20-\\u03bcg vials at 0.5 \\u03bcg/\\u03bcl already\",\n",
            "          \"can be purchased as 5 \\u00d7 20-\\u03bcg vials at 0.5 \\u03bcg/\\u03bcl already reconstituted in 50 mM acetic acid. To facilitate larger digestion batches and avoid pooling multiple small tubes together, we place a custom order of trypsin with Promega, in which the trypsin is\",\n",
            "          \"a custom order of trypsin with Promega, in which the trypsin is in 500-\\u03bcg aliquots at 0.5 \\u03bcg/\\u03bcl in 50 mM acetic acid solution. Reconstituted trypsin should be stored at \\u221280 \\u00b0C and is stable in storage for up to a year. Solid-phase extraction desalting\",\n",
            "          \"in storage for up to a year. Solid-phase extraction desalting solvents A total of 100% (vol/vol) MeCN is used for conditioning. The solvents for equilibration and washing the cartridge are 0.1% (vol/vol) TFA and 1% (vol/vol) FA. The elution solvent is 50%\",\n",
            "          \"(vol/vol) TFA and 1% (vol/vol) FA. The elution solvent is 50% MeCN/0.1% (vol/vol) FA. Solid-phase extraction (SPE) desalting solvents can be made in advance and stored at RT, and are stable for up to a year. Stage-tip desalting solvents Solvent A for\",\n",
            "          \"for up to a year. Stage-tip desalting solvents Solvent A for equilibration and washing is 0.1% (vol/vol) FA. Solvent B for elution is 50% (vol/vol) MeCN/0.1% (vol/vol) FA. Stage-tip desalting solvents can be made in advance and stored at RT, and are\",\n",
            "          \"solvents can be made in advance and stored at RT, and are stable for up to a year. Ammonium formate (NH4HCO2) stock solution (pH 10) for basic-pH RP solvents To make 1 liter of 180 mM ammonium formate, add 25 ml of 28% (wt/vol) ammonium hydroxide (density\",\n",
            "          \"formate, add 25 ml of 28% (wt/vol) ammonium hydroxide (density 0.9 g/ml) to ~500 ml of HPLC-grade water, then add ~45 ml of 10% (vol/vol) FA to titrate the pH to 10.0, and then bring the final volume to 1 liter with HPLC-grade water. Ammonium formate\",\n",
            "          \"final volume to 1 liter with HPLC-grade water. Ammonium formate stock solution is stable for up to a year at RT; check the pH before use to prepare basic-pH RP solvents. Basic-pH RP solvent A Basic-pH RP solvent A is 4.5 mM ammonium formate (pH 10) in 2%\",\n",
            "          \"Basic-pH RP solvent A is 4.5 mM ammonium formate (pH 10) in 2% (vol/vol) acetonitrile. To make 1 liter, add 25 ml of ammonium formate stock solution (pH 10) and 20 ml of acetonitrile (2% (vol/vol)) to 955 ml of water. Basic RP solvents are stable at RT\",\n",
            "          \"to 955 ml of water. Basic RP solvents are stable at RT for up to 2 weeks. Basic-pH RP solvent B Basic-pH RP solvent B is 4.5 mM ammonium formate (pH 10) in 90% (vol/vol) acetonitrile. To make 1 liter, add 25 ml of ammonium formate stock solution (pH 10)\",\n",
            "          \"1 liter, add 25 ml of ammonium formate stock solution (pH 10) and 900 ml of acetonitrile (90% (vol/vol)) to 75 ml of water. Basic RP solvents are stable at RT for up to 2 weeks. \\u25b2 CRITICAL The ionic strength of solvents A and B, ~5 mM, is deliberately\",\n",
            "          \"The ionic strength of solvents A and B, ~5 mM, is deliberately low, so that collected fractions can be readily desalted by vacuum centrifugation and are compatible with the subsequent IMAC-enrichment step of the protocol. Synthetic peptide standards\",\n",
            "          \"step of the protocol. Synthetic peptide standards Selection of synthetic peptides should be based on their availability as well as their chromatographic behavior, such as retention times spanning the duration of the gradient. For example, tryptic peptides\",\n",
            "          \"the duration of the gradient. For example, tryptic peptides of 10\\u201315 amino acids in length with good solubility in aqueous buffer and consistent elution behavior on RP C18 are ideal. We choose peptides that are either from a different species than our\",\n",
            "          \"peptides that are either from a different species than our sample of interest or are isotopically labeled with heavy amino acids to reduce any contamination in subsequent runs on the column. In addition, column back pressure is informative for system\",\n",
            "          \"In addition, column back pressure is informative for system performance. For this scale and configuration, back pressure for the 4.6-mm i.d. columns at 1 ml/min flow is typically in the 180\\u2013220 bar range. The synthetic peptide standards can be stored dry\",\n",
            "          \"bar range. The synthetic peptide standards can be stored dry at \\u221280 \\u00b0C indefinitely. Solvents for conditioning and equilibrating the stage tips during IMAC These include 100% (vol/vol) MeOH, 50% (vol/vol) MeCN/0.1% (vol/vol) FA, and 1% (vol/vol) FA.\",\n",
            "          \"50% (vol/vol) MeCN/0.1% (vol/vol) FA, and 1% (vol/vol) FA. Solvent for binding and washing agarose beads is 80% (vol/vol) MeCN/0.1% (vol/vol) TFA. The stage-tip elution buffer is 50% (vol/vol) MeCN/0.1% (vol/vol) FA. The stage-tip solvents are stable for\",\n",
            "          \"MeCN/0.1% (vol/vol) FA. The stage-tip solvents are stable for up to a year at RT. Agarose-bead elution buffer To make 500 ml of 500 mM potassium phosphate buffer, combine 96.25 ml of 1 M monobasic potassium phosphate with 153.75 ml of dibasic potassium\",\n",
            "          \"potassium phosphate with 153.75 ml of dibasic potassium phosphate in 250 ml of HPLC water. Monobasic and dibasic potassium phosphate stock solutions are stable for up to a year at RT. The 500 mM potassium phosphate buffer is stable for up to 6 months at\",\n",
            "          \"mM potassium phosphate buffer is stable for up to 6 months at RT. Ten mM iron (III) chloride solution Dissolve solid iron (III) chloride in HPLC water; this solution should be made fresh immediately before use. The agarose-bead slurry solution consists of\",\n",
            "          \"before use. The agarose-bead slurry solution consists of acetonitrile, methanol, and 0.01% (vol/vol) acetic acid in a 1:1:1 ratio by volume, which can be made in advance and stored at RT for up to year. UPLC-MS/MS solvents Solvent A is 3% (vol/vol)\",\n",
            "          \"for up to year. UPLC-MS/MS solvents Solvent A is 3% (vol/vol) MeCN/0.1% (vol/vol) FA and solvent B is 90% (vol/vol) MeCN/0.1% (vol/vol) FA. UPLC solvents are stable for up to a month at RT. Nanoflow C18 column The nanospray column for online UPLC-MS/MS\",\n",
            "          \"Nanoflow C18 column The nanospray column for online UPLC-MS/MS analysis is self-packed into a 75-\\u03bcm i.d. PicoFrit column with ReproSil-Pur 120 A, C18-AQ, 1.9 \\u03bcm to a length of 20\\u201324 cm using a pressure bomb set to 950 p.s.i. These columns can be stored at\",\n",
            "          \"pressure bomb set to 950 p.s.i. These columns can be stored at RT for up to 3 months without drying out. Equipment setup Basic RP chromatography blank gradient A timetable summarizing the gradient used for conditioning the column before the injection of\",\n",
            "          \"used for conditioning the column before the injection of the sample at flow rate of 1 ml/min (Step 90) is given below. Time interval (min)\\tGradient (%B)\\t \\t0\\t60\\t \\t2\\t0\\t \\t4\\t0\\t \\t19\\t100\\t \\t21\\t100\\t \\t22\\t0\\t \\t24\\t0\\t \\t39\\t100\\t \\t41\\t100\\t \\t42\\t0\\t \\t70\\t0\\t \\t Basic RP\",\n",
            "          \"22\\t0\\t \\t24\\t0\\t \\t39\\t100\\t \\t41\\t100\\t \\t42\\t0\\t \\t70\\t0\\t \\t Basic RP chromatography label-free/iTRAQ-labeled sample gradient A timetable summarizing the gradient used for fractionation of iTRAQ-labeled sample at a flow rate of 1 ml/min (Step 90) is given below. Time\",\n",
            "          \"at a flow rate of 1 ml/min (Step 90) is given below. Time interval (min)\\tGradient (%B)\\t \\t0\\t0\\t \\t9\\t0\\t \\t13\\t6\\t \\t63\\t28.5\\t \\t68.5\\t34\\t \\t81.5\\t60\\t \\t90\\t60\\t \\t Basic RP chromatography TMT-labeled sample gradient A timetable summarizing gradient used for fractionation\",\n",
            "          \"A timetable summarizing gradient used for fractionation of TMT-labeled sample at a flow rate of 1 ml/min (Step 94) is given below. Time interval (min)\\tGradient (%B)\\t \\t0\\t0\\t \\t7\\t0\\t \\t13\\t16\\t \\t73\\t40\\t \\t77\\t44\\t \\t82\\t60\\t \\t96\\t60\\t \\t UPLC-MS RP chromatography gradient\",\n",
            "          \"77\\t44\\t \\t82\\t60\\t \\t96\\t60\\t \\t UPLC-MS RP chromatography gradient A timetable summarizing the gradient for LC-MS/MS analysis of label-free samples, TMT-labeled basic RP fractions, and TMT-labeled phosphorus-enriched fractions (Steps 44, 72, 97, and 126) is\",\n",
            "          \"phosphorus-enriched fractions (Steps 44, 72, 97, and 126) is given below. Time interval (min)\\tGradient (%B)\\tFlow rate (nl/min)\\t \\t0\\t2\\t200\\t \\t1\\t6\\t200\\t \\t85\\t30\\t200\\t \\t94\\t60\\t200\\t \\t95\\t90\\t200\\t \\t100\\t90\\t200\\t \\t101\\t50\\t500\\t \\t110\\t50\\t500\\t \\t Orbitrap Fusion Lumos MS/MS\",\n",
            "          \"101\\t50\\t500\\t \\t110\\t50\\t500\\t \\t Orbitrap Fusion Lumos MS/MS method MS parameters used for the LC-MS/MS analysis of label-free samples (Step 44) are given below. Parameter\\tValue\\t \\tPolarity\\tPositive\\t \\tFull MS\\t\\t \\tMicroscans\\t1\\t \\tOrbitrap resolution\\t60,000 m/\\u0394(m)\",\n",
            "          \"Full MS\\t\\t \\tMicroscans\\t1\\t \\tOrbitrap resolution\\t60,000 m/\\u0394(m) at 200 m/z\\t \\tAutomatic Gain Control (AGC) target\\t4 \\u00d7 105 ion counts\\t \\tMaximum ion time\\t50 ms\\t \\tScan range\\t350\\u20131,800 m/z\\t \\tData-dependent dd-MS2\\t\\t \\tMicroscans\\t1\\t \\tDetector type\\tOrbitrap\",\n",
            "          \"dd-MS2\\t\\t \\tMicroscans\\t1\\t \\tDetector type\\tOrbitrap\\t \\tOrbitrap resolution\\t15,000 m/delta(m) at 200 m/z\\t \\tAGC target\\t5 \\u00d7 104 ion counts\\t \\tMaximum ion time\\t50 ms\\t \\tNumber of dependent scans\\t20\\t \\tIsolation window\\t0.7 m/z\\t \\tFixed first mass\\t110 m/z\\t \\tActivation\",\n",
            "          \"window\\t0.7 m/z\\t \\tFixed first mass\\t110 m/z\\t \\tActivation type\\tHigher-energy collision dissociation (HCD)\\t \\tHCD energy (%) dd settings\\t34\\t \\tCharge exclusion\\tInclude 2\\u20136 charge states\\t \\tPeptide match\\tPreferred\\t \\tExclude isotopes\\tOn\\t \\tDynamic exclusion\\t45 s\",\n",
            "          \"Exclude isotopes\\tOn\\t \\tDynamic exclusion\\t45 s\\t \\t Orbitrap Fusion Lumos MS/MS method MS parameters used for the LC-MS/MS analysis of TMT-labeled fractionated and phospho-enriched samples (Steps 72, 97, and 126) are given below. Parameter\\tValue\",\n",
            "          \"(Steps 72, 97, and 126) are given below. Parameter\\tValue\\t \\tPolarity\\tPositive\\t \\tFull MS\\t\\t \\tMicroscans\\t1\\t \\tOrbitrap resolution\\t60,000\\t \\tAGC Automatic Gain Control target\\t4 \\u00d7 105 ion counts\\t \\tMaximum ion time\\t50 ms\\t \\tScan range\\t350\\u20131,800 m/z\\t \\tdd-MS2\",\n",
            "          \"Maximum ion time\\t50 ms\\t \\tScan range\\t350\\u20131,800 m/z\\t \\tdd-MS2\\t\\t \\tMicroscans\\t1\\t \\tDetector type\\tOrbitrap\\t \\tOrbitrap resolution\\t50,000 (TMT-10)\\t \\tAGC target\\t1 \\u00d7 105 ion counts\\t \\tMaximum ion time\\t105 ms\\t \\tCycle time: time between master scans\\t2 s\\t \\tIsolation\",\n",
            "          \"ms\\t \\tCycle time: time between master scans\\t2 s\\t \\tIsolation window\\t0.7 m/z\\t \\tFixed first mass\\t110 m/z\\t \\tActivation type\\tHigher-energy collision dissociation (HCD)\\t \\tHCD energy (%) dd settings\\t38 (TMT)\\t \\tMonoisotopic peak determination\\tPeptide\\t \\tCharge\",\n",
            "          \"(TMT)\\t \\tMonoisotopic peak determination\\tPeptide\\t \\tCharge exclusion\\tInclude 2\\u20136 charge states\\t \\tExclude isotopes\\tOn\\t \\tDynamic exclusion\\t20 s\\t \\t \\u25b2 CRITICAL The performance of the combined LC-MS/MS system must be evaluated before the analysis of any precious\",\n",
            "          \"system must be evaluated before the analysis of any precious biological materials such as tissue samples. It is difficult to be prescriptive, as each laboratory will have its own metrics for optimal instrument performance. System performance is typically\",\n",
            "          \"optimal instrument performance. System performance is typically evaluated relative to the maximum obtainable system performance previously established, using a highly complex sample such as a cell-line digest that can be repeatedly produced or purchased.\",\n",
            "          \"cell-line digest that can be repeatedly produced or purchased. System performance is typically evaluated using single-shot analyses and a set gradient duration (e.g., 60\\u201390 min). The number of peptides obtained under these conditions is compared to prior\",\n",
            "          \"peptides obtained under these conditions is compared to prior \\u2018best performance\\u2019 to ascertain if the system is usable. A decrease on the order of 15\\u201320% in the number of peptides identified indicates that either the column, LC, or MS is underperforming\",\n",
            "          \"indicates that either the column, LC, or MS is underperforming and probably needs adjustment or maintenance. The overall appearance of the chromatogram, including start and end times for eluting peptides and chromatographic peak width, as compared with\",\n",
            "          \"peptides and chromatographic peak width, as compared with that obtained when the system is operating well, can help to determine if the problem is with chromatography (LC or column) or the mass spectrometer. \\u25b2 CRITICAL To resolve C and N series reporter\",\n",
            "          \"spectrometer. \\u25b2 CRITICAL To resolve C and N series reporter ions for the TMT-10-labeling strategy, 50,000 at 200 m/z resolution is used for MS2. \\u25b2 CRITICAL To minimize the acquisition of MS/MS spectra with co-isolated precursors, the data acquisition\",\n",
            "          \"MS/MS spectra with co-isolated precursors, the data acquisition control feature, APD, should be turned off. \\u25b2 CRITICAL Each center optimized the collision energy for its own instrument. The listed collision energy is one center\\u2019s value and can be used as\",\n",
            "          \"collision energy is one center\\u2019s value and can be used as a benchmark to further optimize individual instruments. To achieve optimal fragmentation of TMT-labeled peptides, collision energy should be optimized on the instrument used for analyses of those\",\n",
            "          \"be optimized on the instrument used for analyses of those samples. Procedure (Optional) Cryopulverization of tissue blocks \\u25cf Timing 2.5 h for ten non-OCT tissue blocks; 3.5 h for ten OCT tissue blocks. ! CAUTION We recommend that Steps 1\\u201310 be performed\",\n",
            "          \"blocks. ! CAUTION We recommend that Steps 1\\u201310 be performed in a chemical hood to protect against unintended exposure to tissue fragments. \\u25b2 CRITICAL If working with frozen cell pellets or already-cryopulverized tissue, Steps 1\\u201310 can be skipped. If the\",\n",
            "          \"tissue, Steps 1\\u201310 can be skipped. If the frozen tissue blocks are not embedded in optimal-cutting temperature compound (OCT), skip Steps 1\\u20134. Precool a sterile, unopened Petri dish on dry ice. Starting with a tissue block embedded in OCT, unwrap the\",\n",
            "          \"ice. Starting with a tissue block embedded in OCT, unwrap the packaging and place the block on the Petri dish. Do not open the Petri dish until immediately before starting this procedure. Dip a clean, sterile scalpel into liquid nitrogen for 10 s and then\",\n",
            "          \"a clean, sterile scalpel into liquid nitrogen for 10 s and then use it to shave-off slices of OCT away from the tissue block. Dip a pair of tweezers into liquid nitrogen for 10 s every few minutes to use to hold the tissue block down while cutting away\",\n",
            "          \"minutes to use to hold the tissue block down while cutting away the OCT slices. Before freezing the tweezers each time, first clean them with 70% (vol/vol) EtOH. \\u25b2 CRITICAL STEP The first few slices will be difficult to make because the OCT block may be\",\n",
            "          \"slices will be difficult to make because the OCT block may be too frozen. Sometimes it is helpful to wait for 15 s for the outer layers of OCT to become softer before proceeding to cut. Attempting to make cuts while the OCT block is still rock-solid\",\n",
            "          \"Attempting to make cuts while the OCT block is still rock-solid frozen may lead to sample losses and could possibly introduce contamination because the tissue block may flip out of the Petri dish. Slice the OCT away until the OCT constitutes <10% of the\",\n",
            "          \"dish. Slice the OCT away until the OCT constitutes <10% of the sample by volume. Any more than that risks the loss of too much tissue material. Use a new, freshly cooled scalpel every few cuts to ensure that the scalpel itself is not unnecessarily warming\",\n",
            "          \"to ensure that the scalpel itself is not unnecessarily warming the sample. Once a satisfactory amount of OCT has been cut away from the tissue block, clean and refreeze the tweezers, then use them to put the tissue block into a precooled Covaris\",\n",
            "          \"then use them to put the tissue block into a precooled Covaris cryopulverization bag. If the tissue block is too big, first cut the tissue block in half or thirds so that the tissue blocks fit through the mouth of the bag. \\u25b2 CRITICAL STEP If the frozen\",\n",
            "          \"fit through the mouth of the bag. \\u25b2 CRITICAL STEP If the frozen tissue blocks are not embedded in OCT, a frozen 300-mg tissue piece can be directly placed in the tissue bag. Once the tissue blocks are in the cryopulverization bag and the bag is capped,\",\n",
            "          \"blocks are in the cryopulverization bag and the bag is capped, redip the entire bag in liquid nitrogen. Keep the tissue on dry ice until one is ready for the next step. For the actual cryofractionation procedure, prepare dry ice, liquid nitrogen, and\",\n",
            "          \"procedure, prepare dry ice, liquid nitrogen, and tweezers to prepare, store, and handle the sample bag. The impact level should be adjusted depending on the amount of tissue in the tissue bag. For a tissue piece that is between 200 and 400 mg wet weight\",\n",
            "          \"For a tissue piece that is between 200 and 400 mg wet weight (approximated by eye), the impact level should be ~3. Smaller tissue blocks should use less impact and larger tissue blocks should use high impact. Tissue of ~300 mg wet weight is ~10 \\u00d7 10 \\u00d7 3\",\n",
            "          \"use high impact. Tissue of ~300 mg wet weight is ~10 \\u00d7 10 \\u00d7 3 mm. This approximation can vary depending on tissue type. For example, breast cancer is less dense than lung cancer because it has higher proportions of fat. \\u25b2 CRITICAL STEP In this protocol,\",\n",
            "          \"higher proportions of fat. \\u25b2 CRITICAL STEP In this protocol, we recommend a minimum of 50\\u2013100 mg of wet-weight tissue starting material because the protein yield of breast cancer tissue is ~1\\u20133%, and the peptide yield from protein is ~40\\u201350%. Note,\",\n",
            "          \"is ~1\\u20133%, and the peptide yield from protein is ~40\\u201350%. Note, however, that different tissue types will have different protein yields from wet-weight tissue, which should be taken into consideration when determining the amount of starting material\",\n",
            "          \"consideration when determining the amount of starting material needed. \\u25b2 CRITICAL STEP To achieve best cryofracture results, the bagged tissue sample must be transferred from dry ice to liquid nitrogen for 5 s and then immediately cryofractured. \\u25b2\",\n",
            "          \"liquid nitrogen for 5 s and then immediately cryofractured. \\u25b2 CRITICAL STEP Do not hammer a tissue piece more than once, or else the bag will be pierced, and the tissue will fall out. If the impact level is too low and the tissue is not cryopulverized\",\n",
            "          \"impact level is too low and the tissue is not cryopulverized properly, either transfer the sample to a new bag or perform manual pulverizing with a frozen spatula. If the impact is too high, the cryofracture bag will burst on the first hammer hit and the\",\n",
            "          \"the cryofracture bag will burst on the first hammer hit and the tissue will fall out. It may be helpful to use practice tissue samples to familiarize oneself with the relative impact levels. Note that the lid of the cryofracture bag must be screwed on\",\n",
            "          \"Note that the lid of the cryofracture bag must be screwed on loosely to allow air to move out of the bag when it is hit by the metal plates; otherwise the bag may rupture. To transfer the cryopulverized tissue from the cryofracture bag to a 1.5-ml tube,\",\n",
            "          \"tissue from the cryofracture bag to a 1.5-ml tube, dip a capped 1.5-ml tube in liquid nitrogen for 10 s or until the boiling stops. Keep the uncapped tube on dry ice. Dip a new or clean spatula into liquid nitrogen for 10 s and then transfer the tissue\",\n",
            "          \"into liquid nitrogen for 10 s and then transfer the tissue from the bag to a weighing boat kept on dry ice. Transfer the sample to the cooled tube after weighing. Change spatulas between different patient samples. Clean the spatula with 70% EtOH before\",\n",
            "          \"patient samples. Clean the spatula with 70% EtOH before using and after every two to three scoops. Cap the tube once the cryofracture bag is emptied into the tube and store the tube at \\u221280 \\u00b0C until ready for lysis. \\u25a0 PAUSE POINT Frozen tissue samples\",\n",
            "          \"\\u00b0C until ready for lysis. \\u25a0 PAUSE POINT Frozen tissue samples should be stored at \\u221280 \\u00b0C until ready for lysis. We recommend that the length of storage not exceed 1 year. Tissue lysis \\u25cf Timing 90 min Keep the cryopulverized tissue on ice and add 200 \\u03bcl of\",\n",
            "          \"90 min Keep the cryopulverized tissue on ice and add 200 \\u03bcl of chilled urea lysis buffer (4 \\u00b0C) for each ~50-mg portion of wet-weight tissue. \\u25b2 CRITICAL STEP Keep excess urea lysis buffer on ice to be used in later steps. Vortex the tissue-lysis buffer\",\n",
            "          \"ice to be used in later steps. Vortex the tissue-lysis buffer mixture on the highest setting for 5\\u201310 s. Lyse the cells at 4 \\u00b0C for 15 min by letting the tissue-lysis buffer mixture sit on ice. Vortex the tissue-lysis buffer mixture on the highest setting\",\n",
            "          \"Vortex the tissue-lysis buffer mixture on the highest setting for 5\\u201310 s. Incubate the sample for an additional 15 min on ice. If the cryopulverized tissue was originally in a cryovial that is not compatible with the centrifuge, transfer the contents of\",\n",
            "          \"is not compatible with the centrifuge, transfer the contents of the cryovial to a 2-ml screw-cap vial. Centrifuge the tissue-lysis buffer solution at 20,000g at 4 \\u00b0C for 10 min to clear the lysate. \\u25b2 CRITICAL STEP When this protocol was used on human\",\n",
            "          \"lysate. \\u25b2 CRITICAL STEP When this protocol was used on human breast cancer patient material, the fat would rise to the top at this step. It is largely at the lysis step that we eliminate excess amounts of lipid, if present. Transfer the supernatant to a\",\n",
            "          \"amounts of lipid, if present. Transfer the supernatant to a 2-ml conical tube, measuring the volume of the supernatant when transferring. ! CAUTION When removing the supernatant, slide the pipette tip down the side of the tube to avoid a potential fat\",\n",
            "          \"pipette tip down the side of the tube to avoid a potential fat layer that will float above the supernatant and not pellet out with the other cellular debris. \\u25b2 CRITICAL STEP Keep the insoluble pellet that collected at the bottom of the original tube on\",\n",
            "          \"pellet that collected at the bottom of the original tube on ice and store it at \\u221280 \\u00b0C. This pellet can be used for whole-exome sequencing at a quality comparable to that of the original samples. \\u25a0 PAUSE POINT The cell pellets can be stored at \\u221280 \\u00b0C for\",\n",
            "          \"\\u25a0 PAUSE POINT The cell pellets can be stored at \\u221280 \\u00b0C for up to 1 year. Estimation of the protein yield using a Pierce BCA Kit \\u25cf Timing 60 min Dilute the samples in water at a 1:19 (vol/vol) ratio, by removing 2 \\u03bcl of sample and putting it in a 1.5-ml\",\n",
            "          \"ratio, by removing 2 \\u03bcl of sample and putting it in a 1.5-ml conical tube containing 38 \\u03bcl of HPLC water. Plate 10 \\u03bcl of the diluted sample in triplicate per sample in a clear, flat-bottom 96-well microplate. Plate 10 \\u03bcl of HPLC water as a blank and plate\",\n",
            "          \"microplate. Plate 10 \\u03bcl of HPLC water as a blank and plate a urea lysis buffer dilution sample (1:19 (vol/vol), urea lysis buffer to water), each in triplicate. Plate eight wells of 10 \\u03bcl each of an 8-point BSA curve, ranging in concentration from 50\",\n",
            "          \"each of an 8-point BSA curve, ranging in concentration from 50 \\u03bcg/ml to 2 mg/ml. Prepare the BCA reagent by mixing reagent B and reagent A in a 1:49 (vol/vol) ratio. \\u25b2 CRITICAL STEP This solution must be made fresh right before use. Add 200 \\u03bcl of freshly\",\n",
            "          \"must be made fresh right before use. Add 200 \\u03bcl of freshly prepared BCA reagent to each well containing 10 \\u03bcl of sample. Incubate the microplate at 37 \\u00b0C for 30 min. Read the plate at 562 nm and determine the protein concentrations of the samples. Adjust\",\n",
            "          \"and determine the protein concentrations of the samples. Adjust each sample concentration to 8 \\u03bcg/\\u03bcl with urea lysis buffer. If the sample concentration does not reach 8 \\u03bcg/\\u03bcl, record the maximum concentration achieved and adjust subsequent amounts of\",\n",
            "          \"maximum concentration achieved and adjust subsequent amounts of reducing reagents accordingly. Prepare 1-mg aliquots per sample for reduction, alkylation, and digestion. \\u25a0 PAUSE POINT Samples can be frozen and stored at \\u221280 \\u00b0C for several weeks. Thawing\",\n",
            "          \"can be frozen and stored at \\u221280 \\u00b0C for several weeks. Thawing of previously frozen samples is performed gently on ice. Reduction and alkylation \\u25cf Timing 2 h Reduce the disulfide bonds in the denatured proteins by adding DTT to 1 mg of sample (one of the\",\n",
            "          \"denatured proteins by adding DTT to 1 mg of sample (one of the aliquots prepared in Step 27) at a final concentration of 5 mM DTT, followed by incubation for 1 h at 37 \\u00b0C in a Thermomixer. Alkylate the reduced cysteine residues by adding IAM to a final\",\n",
            "          \"Alkylate the reduced cysteine residues by adding IAM to a final concentration of 10 mM IAM, followed by incubation for 45 min in the dark at 25 \\u00b0C. In-solution digestion \\u25cf Timing 2 h + overnight digestion + 30 min Dilute the samples 1:3 (vol/vol) with 50\",\n",
            "          \"digestion + 30 min Dilute the samples 1:3 (vol/vol) with 50 mM Tris-HCl (pH 8.0) to decrease the urea concentration to <2 M. Add LysC in an enzyme/substrate ratio of 1 mAU LysC per 50 \\u03bcg of total protein, followed by incubation for 2 h at 25 \\u00b0C in the\",\n",
            "          \"total protein, followed by incubation for 2 h at 25 \\u00b0C in the incubator shaker. \\u25b2 CRITICAL STEP The manufacturer of the LysC (Wako) provides quantities only in milli-Anson units and not in micrograms. It is impossible for us to know exactly how much LysC\",\n",
            "          \"It is impossible for us to know exactly how much LysC we are adding when measured in micrograms. Our general rule is that for each milli-Anson unit of LysC, we use 1 \\u03bcg of enzyme. A sequencing-grade version of the enzyme is also available at substantially\",\n",
            "          \"version of the enzyme is also available at substantially higher cost from the same vendor, and it is indicated as being packaged at 20 \\u03bcg/vial. However, we have not used this version of the product. Add trypsin in an enzyme/substrate ratio of 1:49 (wt/wt)\",\n",
            "          \"Add trypsin in an enzyme/substrate ratio of 1:49 (wt/wt) for overnight digestion at 25 \\u00b0C in the incubator shaker. \\u25b2 CRITICAL STEP Use of higher temperatures will increase the extent of carbamylation of peptide N termini and lysines, and may have an\",\n",
            "          \"carbamylation of peptide N termini and lysines, and may have an unfavorable effect on quantification. Quench the digestion reaction by acidifying the digestion mixture to a final concentration of 1% (vol/vol) FA using 100% (vol/vol) FA. \\u25b2 CRITICAL STEP\",\n",
            "          \"of 1% (vol/vol) FA using 100% (vol/vol) FA. \\u25b2 CRITICAL STEP Using pH paper, check that the pH of the final solution is at pH 3. If not, add 100% (vol/vol) FA until a pH = 3 is reached. Dilute the samples with 0.1% (vol/vol) FA to a 1.5-ml volume for ease\",\n",
            "          \"the samples with 0.1% (vol/vol) FA to a 1.5-ml volume for ease of handling in subsequent steps. Centrifuge the solution at 1,500g at RT for 15 min. The supernatant containing the peptides to be analyzed is subsequently desalted (Step 41). The pellet is\",\n",
            "          \"be analyzed is subsequently desalted (Step 41). The pellet is generally discarded but can be saved for possible additional processing. Peptide desalting of the digest by SPE \\u25cf Timing 90\\u2013120 min per set of ten samples Use a 200-mg tC18 SepPak cartridge\",\n",
            "          \"min per set of ten samples Use a 200-mg tC18 SepPak cartridge with a vacuum manifold to desalt the peptide samples. \\u25b2 CRITICAL STEP Do not let the cartridges run dry during washing or loading, as this can increase sample loss during desalting from 5 to\",\n",
            "          \"as this can increase sample loss during desalting from 5 to >20%. Condition the cartridge with 3 ml of 100% (vol/vol) MeCN. Condition the cartridge with 3 ml of 50% (vol/vol) MeCN/0.1% (vol/vol) FA. Equilibrate the cartridge with 4 \\u00d7 3 ml of 0.1%\",\n",
            "          \"(vol/vol) FA. Equilibrate the cartridge with 4 \\u00d7 3 ml of 0.1% (vol/vol) TFA. Load the sample supernatant from after centrifugation (from Step 35). Desalt the sample with 3 \\u00d7 3 ml of 0.1% (vol/vol) TFA. Wash the cartridge with 1 \\u00d7 3 ml of 1% (vol/vol) FA\",\n",
            "          \"TFA. Wash the cartridge with 1 \\u00d7 3 ml of 1% (vol/vol) FA to remove the TFA. Elute the sample from the tC18 cartridge with 2 \\u00d7 1.5 ml of 50% (vol/vol) MeCN/0.1% (vol/vol) FA. (Optional) To check for digestion success of individual samples by metrics such\",\n",
            "          \"for digestion success of individual samples by metrics such as missed cleavage rates and peptide counts, as read by LC-MS/MS, prepare a single-shot quality control (QC) per sample. This QC sample can also be used to detect the amount of OCT contamination\",\n",
            "          \"can also be used to detect the amount of OCT contamination in each sample. To generate the QC sample, remove 15 \\u03bcg of the 3 ml eluate and dry it in an HPLC vial using a vacuum centrifuge. Reconstitute the sample in 30 \\u03bcl of 3% (vol/vol) MeCN/0.1%\",\n",
            "          \"Reconstitute the sample in 30 \\u03bcl of 3% (vol/vol) MeCN/0.1% (vol/vol) FA and analyze by LC-MS/MS, injecting 1 \\u03bcg onto the column. Run QC on the sample and digestion by checking parameters such as missed-cleavage rates (Supplementary Fig. 2). Skip this step\",\n",
            "          \"as missed-cleavage rates (Supplementary Fig. 2). Skip this step if instrument time on an LC-MS/MS system is limited. ? TROUBLESHOOTING Freeze the eluate in liquid nitrogen or in a \\u221280 \\u00b0C freezer space before drying down with a vacuum centrifuge. \\u25a0 PAUSE\",\n",
            "          \"space before drying down with a vacuum centrifuge. \\u25a0 PAUSE POINT Dried peptides can be stored in a \\u221280 \\u00b0C freezer space. Estimation of the peptide amount using a Pierce BCA Kit \\u25cf Timing 60 min Reconstitute the sample in 500 \\u03bcl of 50 mM HEPES (pH 8.5) by\",\n",
            "          \"Reconstitute the sample in 500 \\u03bcl of 50 mM HEPES (pH 8.5) by mixing well. Remove a 25-\\u03bcl aliquot for the peptide-level BCA estimation. Dilute the 25-\\u03bcl aliquot of sample 1:1 (vol/vol) with 25 \\u03bcl of 50 mM HEPES (pH 8.5). Plate 10 \\u03bcl of the diluted sample\",\n",
            "          \"\\u03bcl of 50 mM HEPES (pH 8.5). Plate 10 \\u03bcl of the diluted sample in triplicate per sample in a clear, flat-bottom 96-well microplate. Plate a 10-\\u03bcl HPLC water blank and a 50 mM HEPES, pH 8.5, blank, each in triplicate. Plate eight wells of 10 \\u03bcl each of an\",\n",
            "          \"each in triplicate. Plate eight wells of 10 \\u03bcl each of an 8-point BSA curve, ranging from 50 \\u03bcg/ml to 2 mg/ml. Prepare the BCA reagent by mixing reagent B and reagent A in a 1:49 (vol/vol) ratio. \\u25b2 CRITICAL STEP This solution must be made fresh\",\n",
            "          \"ratio. \\u25b2 CRITICAL STEP This solution must be made fresh immediately before use. Add 200 \\u03bcl of freshly prepared BCA reagent to each well containing 10 \\u03bcl of sample. Incubate the microplate at 37 \\u00b0C for 30 min. Read the plate at 562 nm. We usually recover\",\n",
            "          \"37 \\u00b0C for 30 min. Read the plate at 562 nm. We usually recover 40\\u201350% of the starting material at this step. When using 1 mg of protein starting material, we usually recover 400\\u2013500 \\u03bcg of tryptic peptides. On the basis of the BCA results at the peptide\",\n",
            "          \"peptides. On the basis of the BCA results at the peptide level, dilute the samples with 50 mM HEPES (pH 8.5) for a concentration of 1 \\u03bcg/\\u03bcl. Vortex the well to mix. \\u25a0 PAUSE POINT Peptides in 50 mM HEPES (pH 8.5) can be stored in a \\u221280 \\u00b0C freezer space for\",\n",
            "          \"mM HEPES (pH 8.5) can be stored in a \\u221280 \\u00b0C freezer space for up to 1 week. TMT labeling of 300 \\u03bcg of peptide per channel \\u25cf Timing 90 min Remove 300 \\u03bcg of peptides, as measured by peptide-level BCA, which should be in 300 \\u03bcl of 50 mM HEPES (pH 8.5) at\",\n",
            "          \"BCA, which should be in 300 \\u03bcl of 50 mM HEPES (pH 8.5) at this point, and transfer it to a 2-ml screw-cap tube. Reconstitute each 5-mg TMT reagent vial in 256 \\u03bcl of anhydrous acetonitrile. Allow it to sit for 5 min. Vortex the reagent tubes briefly and\",\n",
            "          \"Allow it to sit for 5 min. Vortex the reagent tubes briefly and spin down on a benchtop centrifuge at 1,000g at RT for 30 s. Add 123 \\u03bcl of each reagent to the corresponding aliquot of peptides. Mix well and incubate at RT for 1 h with shaking on a\",\n",
            "          \"peptides. Mix well and incubate at RT for 1 h with shaking on a tabletop shaker set at 1,000 r.p.m. After 1 h, remove 2-\\u03bcg aliquots from each sample and combine these to create a mixing QC test sample. Using a vacuum centrifuge, dry down this mixture of\",\n",
            "          \"sample. Using a vacuum centrifuge, dry down this mixture of the ten channels and desalt these QC samples as described in Steps 63\\u201372. Freeze the rest of the samples at \\u221280 \\u00b0C until the mixing QC test is analyzed. \\u25b2 CRITICAL STEP Do not quench the labeling\",\n",
            "          \"QC test is analyzed. \\u25b2 CRITICAL STEP Do not quench the labeling reaction until after assessment of the label incorporation test results. \\u25a0 PAUSE POINT Peptides in 50 mM HEPES (pH 8.5) with TMT-labeling reagent can be stored in a \\u221280 \\u00b0C freezer space for\",\n",
            "          \"reagent can be stored in a \\u221280 \\u00b0C freezer space for up to 2 weeks. Desalting of the mixing QC test sample \\u25cf Timing 30 min Prepare C18 stage tips by using Empore C18 extraction disks, as described in ref. . Pack two plugs of C18 material into the tip of\",\n",
            "          \"in ref. . Pack two plugs of C18 material into the tip of each stage tip (200-\\u03bcl pipette tip) for a total binding capacity of ~40 \\u03bcg. Create extraction disks using a 16-gauge blunt-end metal needle to hole-punch the ~1-mm disks. We recommend using adapters\",\n",
            "          \"to hole-punch the ~1-mm disks. We recommend using adapters to hold the stage-tip pipette tip at the orifice of each 2 ml microcentrifuge tube. Condition the stage tip with 100 \\u03bcl of MeOH. Centrifuge at 3,000\\u20133,500g for 3 min at RT and discard the liquid\",\n",
            "          \"at 3,000\\u20133,500g for 3 min at RT and discard the liquid from the collection vial. All subsequent centrifugation steps are for the same duration or until all the solvent has passed through the stage tip at the same speed and at RT. \\u25b2 CRITICAL STEP Do not\",\n",
            "          \"stage tip at the same speed and at RT. \\u25b2 CRITICAL STEP Do not centrifuge the stage tips at speeds above 3,500g or else the C18 material will dry out. Wash the stage tips with 100 \\u03bcl of 50% (vol/vol) MeCN in 0.1% (vol/vol) FA (stage-tip desalting solvent\",\n",
            "          \"MeCN in 0.1% (vol/vol) FA (stage-tip desalting solvent B). Centrifuge the stage tip at 3,000\\u20133,500g for 3 min at RT. Discard the liquid. Equilibrate the stage tips twice with 100 \\u03bcl of 0.1% (vol/vol) FA (stage-tip desalting solvent A). Centrifuge the\",\n",
            "          \"(vol/vol) FA (stage-tip desalting solvent A). Centrifuge the stage tip at 3,000\\u20133,500g for 3 min at RT after each solvent loading. Discard the liquid. Reconstitute the sample in 100 \\u03bcl of 3% (vol/vol) MeCN/0.1% (vol/vol) FA and add the samples to the\",\n",
            "          \"3% (vol/vol) MeCN/0.1% (vol/vol) FA and add the samples to the stage tip. Centrifuge the stage tip at 3,000\\u20133,500g for 3 min at RT and discard the liquid from the collection vial. Wash the samples twice with 100 \\u03bcl of stage-tip desalting solvent A.\",\n",
            "          \"the samples twice with 100 \\u03bcl of stage-tip desalting solvent A. Centrifuge the stage tip at 3,000\\u20133,500g for 3 min at RT. Discard the liquid. Replace the collection vial with a new 1.5 ml screw-cap vial for the collection of the eluate. Elute the sample\",\n",
            "          \"vial for the collection of the eluate. Elute the sample with 60 \\u03bcl of stage-tip desalting solvent B. Centrifuge the stage tip at 3,000\\u20133,500g for 3 min at RT. Transfer the eluate to an HPLC vial, freeze it, and completely dry it by vacuum centrifugation.\",\n",
            "          \"freeze it, and completely dry it by vacuum centrifugation. Reconstitute the dried sample in 40 \\u03bcl of 3% (vol/vol) MeCN/0.1% (vol/vol) FA and analyze the sample by LC-MS/MS (using the parameters described in the \\u2018Equipment setup\\u2019 section) to check the\",\n",
            "          \"described in the \\u2018Equipment setup\\u2019 section) to check the label incorporation and relative channel abundance. For the LC-MS/MS analysis, inject 2 \\u03bcl onto the column. \\u25b2 CRITICAL STEP As it can take a considerable amount of instrument time to analyze each of\",\n",
            "          \"a considerable amount of instrument time to analyze each of the individual label-incorporation test samples, particularly for TMT-10, one can consider analyzing the mixing QC test sample for both labeling efficiency and mixing consistency, and only\",\n",
            "          \"for both labeling efficiency and mixing consistency, and only analyzing the individual label-incorporation test samples for troubleshooting purposes if the labeling efficiency is suspect in the mixing test sample. Data analysis for assessment of labeling\",\n",
            "          \"mixing test sample. Data analysis for assessment of labeling efficiency and mixing \\u25cf Timing 1 h To evaluate the completeness of free-amine labeling, database searches should be configured to allow for both peptide N termini and lysine side chains to be\",\n",
            "          \"allow for both peptide N termini and lysine side chains to be present in either labeled or unlabeled form. With Spectrum Mill, accounting for partial labeling is accomplished with a fixed/mix search cycle strategy that runs a search four consecutive times\",\n",
            "          \"search cycle strategy that runs a search four consecutive times with different sets of modifications in each round and then produces a single integrated output. The four cycles are as follows: all unmodified, both peptide N termini and lysines labeled,\",\n",
            "          \"all unmodified, both peptide N termini and lysines labeled, only lysines labeled, and only peptide N termini labeled. As the primary amine groups of lysine side chains (pKa ~10) are more reactive than peptide N-terminal amines (pKa ~7.5), incomplete\",\n",
            "          \"reactive than peptide N-terminal amines (pKa ~7.5), incomplete labeling tends to be observed predominantly in the form of unlabeled peptide N termini. Furthermore, so long as a peptide includes at least 1 label, reporter-ion quantitation is viable. After\",\n",
            "          \"at least 1 label, reporter-ion quantitation is viable. After running a four-cycle partial-labeling database search configuration with a suitable (1%) PSM-level false-discovery rate (FDR) threshold on a label-check aliquot, labeling efficiency percentage\",\n",
            "          \"on a label-check aliquot, labeling efficiency percentage metrics are calculated in the Spectrum Mill Quality Metrics module: full label % = 100 \\u00d7 fully labeled PSMs/total PSMs and label % = 100 \\u00d7 1 or more labeled PSMs/total PSMs. \\u25b2 CRITICAL STEP We\",\n",
            "          \"% = 100 \\u00d7 1 or more labeled PSMs/total PSMs. \\u25b2 CRITICAL STEP We typically apply a threshold for minimal labeling efficiency of >99% labeling by PSM identifications. If the threshold is not met, then the samples will be relabeled (see Troubleshooting\",\n",
            "          \"met, then the samples will be relabeled (see Troubleshooting section for relabeling procedure) before proceeding with further peptide-level fractionation. In our experience, TMT labeling is more robust and less likely to require relabeling than iTRAQ\",\n",
            "          \"is more robust and less likely to require relabeling than iTRAQ labeling. ? TROUBLESHOOTING Analyze the resulting MS data from the mixing test sample to check if the total protein amount from each sample is the same. The Spectrum Mill Quality Metrics\",\n",
            "          \"from each sample is the same. The Spectrum Mill Quality Metrics module does this by summing the reporter-ion intensity (after applying isotopic correction) for each channel across all the confidently identified PSMs in the dataset. Then, per-sample mixing\",\n",
            "          \"identified PSMs in the dataset. Then, per-sample mixing percentages are calculated using the most abundant reporter-ion channel sum as a common denominator and each of the other channels as a numerator. \\u25b2 CRITICAL STEP Unless particular samples are\",\n",
            "          \"as a numerator. \\u25b2 CRITICAL STEP Unless particular samples are available in limited quantity, we require each sample to have a reporter-ion intensity sum for all identified PSMs in the mixing test sample LC-MS/MS run that deviates <25% from the\",\n",
            "          \"the mixing test sample LC-MS/MS run that deviates <25% from the reporter-ion intensity sum of the reference channel, if present, or from the average reporter-ion intensity sum of all channels. ? TROUBLESHOOTING Quenching of the TMT-labeling reaction \\u25cf\",\n",
            "          \"? TROUBLESHOOTING Quenching of the TMT-labeling reaction \\u25cf Timing 20 min Quench all individual TMT-labeling reactions by adding 32 \\u03bcl of 5% (vol/vol) hydroxylamine, followed by incubation at RT for 15 min with shaking at 1,000 r.p.m. Combine the volumes\",\n",
            "          \"RT for 15 min with shaking at 1,000 r.p.m. Combine the volumes of each of the ten TMT-labeled samples in a 15-ml conical tube before freezing in liquid nitrogen or by placing into a \\u221280 \\u00b0C freezer space. Dry down the combined sample. \\u25a0 PAUSE POINT Dried\",\n",
            "          \"space. Dry down the combined sample. \\u25a0 PAUSE POINT Dried peptides can be stored in a \\u221280 \\u00b0C freezer space for up to 1 month. Peptide desalting of labeled peptides by SPE \\u25cf Timing 30 min Use a 200-mg tC18 SepPak cartridge with a vacuum manifold to desalt\",\n",
            "          \"a 200-mg tC18 SepPak cartridge with a vacuum manifold to desalt the peptide samples. \\u25b2 CRITICAL STEP Do not let the cartridges run dry during washing or loading, as this can increase sample losses during desalting from 5% to >20%. Reconstitute the\",\n",
            "          \"losses during desalting from 5% to >20%. Reconstitute the combined sample of TMT-labeled peptides in 3 ml of 3% (vol/vol) MeCN/0.1% (vol/vol) FA. \\u25b2 CRITICAL STEP Measure the pH with pH-indicator paper and adjust the pH to 3 with 100% (vol/vol) FA if\",\n",
            "          \"paper and adjust the pH to 3 with 100% (vol/vol) FA if necessary. Condition the cartridge with 3 ml of 100% (vol/vol) MeCN. Condition the cartridge with 3 ml of 50% (vol/vol) MeCN/0.1% (vol/vol) FA. Equilibrate the cartridge with 4 \\u00d7 3 ml of 0.1%\",\n",
            "          \"(vol/vol) FA. Equilibrate the cartridge with 4 \\u00d7 3 ml of 0.1% (vol/vol) TFA. Load the sample onto the cartridge. Desalt the sample with 3 \\u00d7 3 ml of 0.1% (vol/vol) TFA. Wash the cartridge with 1 \\u00d7 3 ml of 1% (vol/vol) FA to remove the TFA. Elute the sample\",\n",
            "          \"1 \\u00d7 3 ml of 1% (vol/vol) FA to remove the TFA. Elute the sample from the tC18 cartridge with 2 \\u00d7 1.5 ml of 50% (vol/vol) MeCN/0.1% (vol/vol) FA. Freeze the eluate in liquid nitrogen or in a \\u221280 \\u00b0C freezer space before drying down with a vacuum centrifuge.\",\n",
            "          \"\\u00b0C freezer space before drying down with a vacuum centrifuge. \\u25a0 PAUSE POINT Dried peptides can be stored in a \\u221280 \\u00b0C freezer space for up to 1 year. Offline HPLC fractionation \\u25cf Timing 60 min + overnight QC + 5 h Prepare the HPLC system by purging solvent\",\n",
            "          \"+ overnight QC + 5 h Prepare the HPLC system by purging solvent lines A and B of air. Equilibrate the bRP fractionation column with 100% (vol/vol) solvent B, then 100% (vol/vol) solvent A, and finally with 50% (vol/vol) solvent A/50% (vol/vol) solvent B.\",\n",
            "          \"finally with 50% (vol/vol) solvent A/50% (vol/vol) solvent B. Equilibrate with each of these conditions for 30 min, for a total of 1.5 h. The flow rate for equilibration is that of the method, which in our method is 1 ml/min. Perform a QC of the HPLC\",\n",
            "          \"which in our method is 1 ml/min. Perform a QC of the HPLC system before fractionation. We typically accomplish this by injecting peptide standards in duplicate before running the actual sample. For this scale of bRP, 200 pmol of a mixture of ten in-house\",\n",
            "          \"For this scale of bRP, 200 pmol of a mixture of ten in-house synthetic peptide standards (Reagent setup) is injected and evaluated for retention time reproducibility, signal intensity, and peak resolution (Supplementary Fig. 3a). Prepare the LC system for\",\n",
            "          \"resolution (Supplementary Fig. 3a). Prepare the LC system for QC sample injection: the composition of a single sample separation is a sequence of two methods: a blank gradient, used to condition the column, followed by the sample gradient. For the QC run,\",\n",
            "          \"the column, followed by the sample gradient. For the QC run, we typically use a method that is meant for label-free peptides for better separation and resolution peaks for the peptides. Once the QC of the peptide standards indicates the system is running\",\n",
            "          \"the QC of the peptide standards indicates the system is running properly, run a blank gradient to condition the column for the TMT-labeled sample (Equipment setup). This is achieved by injecting 50 \\u03bcl of bRP solvent A (refer to \\u2018Reagent setup\\u2019 for\",\n",
            "          \"injecting 50 \\u03bcl of bRP solvent A (refer to \\u2018Reagent setup\\u2019 for basic-pH RP solvents) from an HPLC vial with the gradient outlined in the \\u2018Equipment setup\\u2019 as a blank gradient. Upon completion of the blank run, reconstitute the sample in 900 \\u03bcl of bRP\",\n",
            "          \"of the blank run, reconstitute the sample in 900 \\u03bcl of bRP solvent A (see \\u2018Reagent setup\\u2019 for basic-pH RP solvents). Vortex until the sample is in solution. \\u25b2 CRITICAL STEP Do not pipette up and down to bring the sample into solution, as this will result\",\n",
            "          \"and down to bring the sample into solution, as this will result in the formation of bubbles. Transfer the sample to a 1.5-ml screw-cap vial and centrifuge at 20,000g at RT for 5 min to remove any material that did not go into solution. Transfer the\",\n",
            "          \"remove any material that did not go into solution. Transfer the sample, avoiding the pelleted insoluble peptides, to an HPLC vial. Inject 850 \\u03bcl of the sample into the sample loop of the HPLC system. \\u25b2 CRITICAL STEP The sample volume is brought to 900 \\u03bcl,\",\n",
            "          \"system. \\u25b2 CRITICAL STEP The sample volume is brought to 900 \\u03bcl, but only 850 \\u03bcl is injected, because extra volume is needed to avoid injecting air bubbles into the system during sample injection. For separation of the TMT-labeled samples, use the\",\n",
            "          \"injection. For separation of the TMT-labeled samples, use the corresponding gradient and flow rate settings outlined in the tables in the \\u2018Equipment setup\\u2019 section. For an example chromatogram, see Supplementary Fig. 3b. In the bRP separation, 96\",\n",
            "          \"see Supplementary Fig. 3b. In the bRP separation, 96 fractions are collected into a Whatman 2-ml 96-well plate at a flow rate of 1 ml/min. \\u25b2 CRITICAL STEP To store the HPLC column after separation, flow water through the column at 1 ml/min and then flow\",\n",
            "          \"flow water through the column at 1 ml/min and then flow methanol through the column, also at 1 ml/min. Do not store or maintain the column in basic solvent when samples are not being run, as this will substantially speed up the aging of the column and\",\n",
            "          \"as this will substantially speed up the aging of the column and cause premature widening of peaks. After separation, pool the bRP fractions (as described in Table 1) in 15-ml conical vials to generate 24 final fractions and fraction A. Fraction A does not\",\n",
            "          \"generate 24 final fractions and fraction A. Fraction A does not contain many peptides, acts as a negative control, and contains multiply phosphorylated peptides that do not bind well to the RP column. The plate layout describes the recombination of wells\",\n",
            "          \"column. The plate layout describes the recombination of wells in which B10 is combined with D10 and F10 as fraction 1, B9 is combined with D9 and F9 as fraction 2, and so on (Table 1). \\u25b2 CRITICAL STEP In our method of pooling, we have factored in the fact\",\n",
            "          \"STEP In our method of pooling, we have factored in the fact that our sample is eluted in a serpentine manner. For methods that do not eluate in a serpentine method, please correct the pooling scheme to still allow pooling of the fractions from the\",\n",
            "          \"pooling scheme to still allow pooling of the fractions from the beginning, middle, and end of the gradient in an even and recurring manner. Our method reduces 96 individual wells into 23 final fractions (three wells with disparate hydrophobicity) plus two\",\n",
            "          \"fractions (three wells with disparate hydrophobicity) plus two fractions from the early and late portions of the chromatogram that are sparser in peptide content. Acidify each pooled fraction using 10% (vol/vol) FA to achieve a final concentration of 0.1%\",\n",
            "          \"using 10% (vol/vol) FA to achieve a final concentration of 0.1% (vol/vol) FA and a pH of 3. From each pooled fraction, collect 5% of each fraction into an HPLC vial for proteome analysis by LC-MS/MS. Vacuum-centrifuge each HPLC vial with the proteome\",\n",
            "          \"by LC-MS/MS. Vacuum-centrifuge each HPLC vial with the proteome analysis fraction. For analysis, bring the sample up in 3% (vol/vol) MeCN/0.1% (vol/vol) FA to a final concentration of 0.5 \\u03bcg/\\u03bcl and inject 1 \\u03bcl onto the column. In practice, we assume there\",\n",
            "          \"and inject 1 \\u03bcl onto the column. In practice, we assume there is 6.25 \\u03bcg in each of the 24 fractions and we bring the sample up in 12.5 \\u03bcl of 3% (vol/vol) MeCN/0.1% (vol/vol) FA. We have previously found that the columns can accommodate 1 \\u03bcg without\",\n",
            "          \"previously found that the columns can accommodate 1 \\u03bcg without overloading when samples are measured on the protein level. In our current protocol, we are quantifying the amounts on the peptide level. Peptide yields are approximately half of protein\",\n",
            "          \"peptide level. Peptide yields are approximately half of protein yields. Therefore, by loading 0.5 \\u03bcg (peptide-level BCA), we are effectively reproducing our previous 1 \\u03bcg on the column (protein-level BCA). Experience from one of the three labs indicates\",\n",
            "          \"BCA). Experience from one of the three labs indicates that results with 0.5- and 1-\\u03bcg loads (by peptide BCA) are very similar. In general, it should be noted that the higher the peptide load, the greater the possibility of the chromatographic separation\",\n",
            "          \"the greater the possibility of the chromatographic separation decreasing because of overloading. ? TROUBLESHOOTING Combine the remaining 95% of each fraction into 12 fractions + fraction A for enrichment of phosphopeptides as follows: Fraction no.\\tbRP\",\n",
            "          \"for enrichment of phosphopeptides as follows: Fraction no.\\tbRP fractions\\t \\t1\\t1 + 13\\t \\t2\\t2 + 14\\t \\t3\\t3 + 15\\t \\t4\\t4 + 16\\t \\t5\\t5 + 17\\t \\t6\\t6 + 18\\t \\t7\\t7 + 19\\t \\t8\\t8 + 20\\t \\t9\\t9 + 21\\t \\t10\\t10 + 22\\t \\t11\\t11 + 23\\t \\t12\\t12 + 24\\t \\tA\\tA\\t \\t Freeze the 13 fractions with liquid\",\n",
            "          \"+ 23\\t \\t12\\t12 + 24\\t \\tA\\tA\\t \\t Freeze the 13 fractions with liquid nitrogen or in the \\u221280 \\u00b0C freezer space before drying the samples down in a vacuum centrifuge or lyophilizer. \\u25a0 PAUSE POINT Dried peptides can be stored in a \\u221280 \\u00b0C freezer space for up 1\",\n",
            "          \"Dried peptides can be stored in a \\u221280 \\u00b0C freezer space for up 1 week. Combined IMAC enrichment and phosphopeptide desalting: bead preparation \\u25cf Timing 90 min Fully resuspend the Ni-NTA Superflow Agarose beads before removing an appropriate aliquot of\",\n",
            "          \"Agarose beads before removing an appropriate aliquot of slurry, in which the beads/solvent ratio is 1:1 (vol/vol). This protocol is written for the preparation of up to 500 \\u03bcl of beads. Scale up appropriately if >300 \\u03bcl of beads need to be prepped at\",\n",
            "          \"up appropriately if >300 \\u03bcl of beads need to be prepped at once. For a 13-fraction IMAC enrichment, 130 \\u03bcl of beads is needed. To account for proper overhead, remove 160 \\u03bcl of beads or 320 \\u03bcl of slurry. \\u25b2 CRITICAL STEP Prepare the beads fresh on the day\",\n",
            "          \"of slurry. \\u25b2 CRITICAL STEP Prepare the beads fresh on the day of the enrichment. Centrifuge the slurry in a tabletop centrifuge at 1,000g at RT for 1 min. Remove the stock supernatant. Wash the beads 3\\u00d7 in 1 ml of HPLC water, centrifuging in the tabletop\",\n",
            "          \"beads 3\\u00d7 in 1 ml of HPLC water, centrifuging in the tabletop centrifuge (1,000g at RT for 1 min) each time to remove the supernatant. Incubate the beads in 1,200 \\u03bcl of 100 mM EDTA (1:5 (vol/vol) dilution of the stock 500 mM EDTA solution) for 30 min at RT\",\n",
            "          \"dilution of the stock 500 mM EDTA solution) for 30 min at RT with end-over-end turning to strip the beads of nickel. Wash the beads 3\\u00d7 in 1 ml of HPLC water, centrifuging in the tabletop centrifuge (1,000g at RT for 1 min) each time to remove the\",\n",
            "          \"centrifuge (1,000g at RT for 1 min) each time to remove the supernatant. Incubate the beads with 1,200 \\u03bcl of 10 mM iron (III) chloride aqueous solution (solid iron (III) chloride in HPLC water) for 30 min at RT with end-over-end turning. Wash the beads 3\\u00d7\",\n",
            "          \"for 30 min at RT with end-over-end turning. Wash the beads 3\\u00d7 in 1 ml of HPLC water, centrifuging in the tabletop centrifuge each time (1,000g at RT for 1 min) to remove the supernatant. Bring the beads up with a 1:1:1 (vol/vol/vol) ratio of\",\n",
            "          \"Bring the beads up with a 1:1:1 (vol/vol/vol) ratio of acetonitrile/methanol/0.01% (vol/vol) acetic acid so that it is a slurry of 1:1:1:1 beads/acetonitrile/methanol/0.01% (vol/vol) acetic acid. If 160 \\u03bcl of beads were originally removed, the total\",\n",
            "          \"acid. If 160 \\u03bcl of beads were originally removed, the total volume would be 640 \\u03bcl at this point. Pipette 40 \\u03bcl of bead slurry into each of 13 \\u00d7 1.5-ml screw-cap tubes, shaking the slurry from which you are drawing aliquots after every few aliquots to\",\n",
            "          \"from which you are drawing aliquots after every few aliquots to maintain the suspension. The 40 \\u03bcl of slurry in each tube corresponds to 10 \\u03bcl of beads for enrichment per fraction. \\u25b2 CRITICAL STEP Pipette 40 uL of bead slurry into each of thirteen \\u00d7 1.5\",\n",
            "          \"STEP Pipette 40 uL of bead slurry into each of thirteen \\u00d7 1.5 mL screw-cap tubes, shaking the slurry from which you are drawing aliquots after every few aliquots to maintain the suspension. The 40 uL of slurry in each tube corresponds to 10 uL of beads\",\n",
            "          \"The 40 uL of slurry in each tube corresponds to 10 uL of beads for enrichment per fraction. \\u25a0 PAUSE POINT Beads cannot be prepared before the day of the enrichment. Beads that are not prepared fresh on the day of the enrichment have been found to show\",\n",
            "          \"fresh on the day of the enrichment have been found to show decreased enrichment specificity. However, beads can be kept in this 1:1:1:1 slurry for up to 1\\u20132 h. Peptide sample preparation \\u25cf Timing 15 min Bring the peptide concentration to 0.5 \\u03bcg/\\u03bcl in 80%\",\n",
            "          \"15 min Bring the peptide concentration to 0.5 \\u03bcg/\\u03bcl in 80% (vol/vol) MeCN/0.1% (vol/vol) TFA. Because it is difficult to solubilize peptides in such high organic solvents, first solubilize the peptides in 50% (vol/vol) MeCN/0.1% (vol/vol) TFA and add 100%\",\n",
            "          \"peptides in 50% (vol/vol) MeCN/0.1% (vol/vol) TFA and add 100% (vol/vol) MeCN/0.1% (vol/vol) TFA to achieve an 80% (vol/vol) MeCN/0.1% (vol/vol) TFA concentration. For example, in a 3-mg sample that was fractionated into 24 fractions (with 5% removed for\",\n",
            "          \"that was fractionated into 24 fractions (with 5% removed for proteome analysis) and then pooled into 12 fractions, there is 237.5 \\u03bcg of peptides in each of the phosphofractions for enrichment. Fraction A is not counted in the calculations because it is\",\n",
            "          \"Fraction A is not counted in the calculations because it is nearly empty, except for a few multiphosphorylated peptides. Bring each fraction, including fraction A, into solution. First dissolve each fraction in 190 \\u03bcl of 50% (vol/vol) MeCN/0.1% (vol/vol)\",\n",
            "          \"each fraction in 190 \\u03bcl of 50% (vol/vol) MeCN/0.1% (vol/vol) TFA with vortexing. Once all the peptides are in solution, add 285 \\u03bcl of 100% (vol/vol) MeCN/0.1% (vol/vol) TFA. \\u25b2 CRITICAL STEP A concentration of 0.1% (vol/vol) TFA is used in the IMAC binding\",\n",
            "          \"concentration of 0.1% (vol/vol) TFA is used in the IMAC binding buffer to control the pH at 2. This ensures that the carboxyl groups of glutamic and aspartic acid, and the peptide C termini are protonated to avoid background binding of nonphosphorylated\",\n",
            "          \"are protonated to avoid background binding of nonphosphorylated peptides to the IMAC resin. At pH 2, S/T/Y-phosphorylated residues remain negatively charged and bind strongly to the resin. Spin down the samples (1,500g at RT for 10 min) to pellet any\",\n",
            "          \"Spin down the samples (1,500g at RT for 10 min) to pellet any un-reconstituted peptides. \\u25b2 CRITICAL STEP Check if the pH is 2 with pH-indicator paper. Combined IMAC enrichment and phosphopeptide desalting: phosphoenrichment \\u25cf Timing 45 min Add the peptide\",\n",
            "          \"desalting: phosphoenrichment \\u25cf Timing 45 min Add the peptide solution to the aliquoted beads and incubate for 30 min at RT on a shaker at 1,000 r.p.m. \\u25b2 CRITICAL STEP Do not let the incubation go beyond 30 min. Spin down the bead-peptide solution for 1\",\n",
            "          \"go beyond 30 min. Spin down the bead-peptide solution for 1 min at 1,000g at RT in a centrifuge. Remove the supernatant from each tube and keep this as a \\u2018flow through\\u2019 sample for potential subsequent enrichments. Add 200 \\u03bcl of 80% (vol/vol) MeCN/0.1%\",\n",
            "          \"subsequent enrichments. Add 200 \\u03bcl of 80% (vol/vol) MeCN/0.1% (vol/vol) TFA to the beads, creating a slurry that can later be pipetted and transferred in Step 118. Combined IMAC enrichment and phosphopeptide desalting: stage-tip desalting of\",\n",
            "          \"enrichment and phosphopeptide desalting: stage-tip desalting of phosphoenrichment \\u25cf Timing 90 min Prepare 13 2-plug C18 stage tips by using Empore C18 extraction disks as described in Step 63. Condition the stage tips with 100 \\u03bcl of MeOH. Centrifuge at\",\n",
            "          \"63. Condition the stage tips with 100 \\u03bcl of MeOH. Centrifuge at 3,000\\u20133,500g for 3 min at RT and discard the liquid from the collection vial. Condition the stage tips a second time with another 100 \\u03bcl of MeOH. All subsequent centrifugation steps are for\",\n",
            "          \"100 \\u03bcl of MeOH. All subsequent centrifugation steps are for the same duration or until all the solvent has passed through the stage tip at the same speed and at RT. \\u25b2 CRITICAL STEP Do not centrifuge the stage tips at speeds >3,500g or else the C18\",\n",
            "          \"not centrifuge the stage tips at speeds >3,500g or else the C18 material will dry out. Wash the stage tips with 50 \\u03bcl of 50% (vol/vol) MeCN in 0.1% (vol/vol) FA (stage tip solvent B). Centrifuge the stage tips at 3,000\\u20133,500g for 3 min at RT. Discard the\",\n",
            "          \"the stage tips at 3,000\\u20133,500g for 3 min at RT. Discard the liquid. Equilibrate the stage tips twice with 100 \\u03bcl of 1% (vol/vol) FA. Centrifuge the stage tips at 3,000\\u20133,500g for 3 min at RT after each solvent loading. Discard the liquid. Load the\",\n",
            "          \"at RT after each solvent loading. Discard the liquid. Load the enriched beads onto the stage tip. Centrifuge the stage tips at 3000\\u20133500g for 3 min at RT and discard the liquid from the collection vial. Wash the beads twice with 50 \\u03bcl of 80% (vol/vol)\",\n",
            "          \"vial. Wash the beads twice with 50 \\u03bcl of 80% (vol/vol) MeCN/0.1% (vol/vol) TFA. Centrifuge the stage tips at 3000\\u20133500g for 3 min at RT after each solvent loading. Discard the liquid. \\u25b2 CRITICAL STEP A concentration of 0.1% (vol/vol) TFA is used to ensure\",\n",
            "          \"STEP A concentration of 0.1% (vol/vol) TFA is used to ensure that the pH is 2. This ensures that the carboxyl groups of glutamic and aspartic acid, and the peptide C termini are protonated to avoid background binding of nonphosphorylated peptides to the\",\n",
            "          \"avoid background binding of nonphosphorylated peptides to the IMAC resin. At pH 2, S/T/Y-phosphorylated residues remain negatively charged and bind strongly to the resin. Wash the beads with 50 \\u03bcl of 1% (vol/vol) FA. Centrifuge the stage tip at\",\n",
            "          \"with 50 \\u03bcl of 1% (vol/vol) FA. Centrifuge the stage tip at 3,000\\u20133,500g for 3 min at RT. Discard the liquid. Elute the peptides from the beads and onto the C18 plugs with three iterations of 70 \\u03bcl of agarose-bead elution buffer. Centrifuge the stage tip\",\n",
            "          \"70 \\u03bcl of agarose-bead elution buffer. Centrifuge the stage tip at 3,000\\u20133,500g for 3 min at RT after each solvent loading. Discard the liquid. Wash the C18 material with 100 \\u03bcl of 1% (vol/vol) FA. Centrifuge the stage tip at 3,000\\u20133,500g for 3 min at RT\",\n",
            "          \"FA. Centrifuge the stage tip at 3,000\\u20133,500g for 3 min at RT and discard the liquid. Replace the collection vial with a new 1.5-ml screw-cap vial for the collection of the eluate. Elute the sample from the C18 plugs with 60 \\u03bcl of StageTip solvent B.\",\n",
            "          \"the sample from the C18 plugs with 60 \\u03bcl of StageTip solvent B. Centrifuge the stage tip at 3,000\\u20133,500g for 3 min at RT. Transfer the eluate to an HPLC vial, then freeze and completely dry the samples by vacuum centrifugation. Reconstitute the dried\",\n",
            "          \"the samples by vacuum centrifugation. Reconstitute the dried sample in 9 \\u03bcl of 3% (vol/vol) MeCN/0.1% (vol/vol) FA and analyze by LC-MS/MS (using the parameters described in the \\u2018Equipment setup\\u2019 section). For LC-MS/MS analysis, inject 4 \\u03bcl onto the\",\n",
            "          \"setup\\u2019 section). For LC-MS/MS analysis, inject 4 \\u03bcl onto the column. \\u25b2 CRITICAL STEP TFA reduces the ionization efficiency in the electrospray, so we wash it out of the stage tips using 1% (vol/vol) FA. The concentration of FA is reduced to 0.1% at the\",\n",
            "          \"(vol/vol) FA. The concentration of FA is reduced to 0.1% at the last elution step to ensure the best sensitivity before injection into the mass spectrometer. ? TROUBLESHOOTING Data analysis \\u25cf Timing <1 d We typically analyze the data using Spectrum Mill\",\n",
            "          \"\\u25cf Timing <1 d We typically analyze the data using Spectrum Mill MS Proteomics Workbench. However, other search engines can be used for searching the data, as long as the program can handle TMT-10 data. The table below shows the search parameters we use.\",\n",
            "          \"data. The table below shows the search parameters we use. Parameter\\tValue\\t \\tVariable modification\\tOxidation (M)\\t \\t\\tAcetyl (protein N-term)\\t \\t\\tDeamidation (N)\\t \\t\\tPyroGlu (Q)\\t \\tFixed modification\\tCarbamidomethyl (C)\\t \\t\\tTMT-10 (peptide N-term, K)\",\n",
            "          \"modification\\tCarbamidomethyl (C)\\t \\t\\tTMT-10 (peptide N-term, K)\\t \\tDigest\\tTrypsin Allow P\\t \\tMaximum missed cleavages\\t4\\t \\tMaximum charge\\t5\\t \\tPrecursor mass tolerance (p.p.m.)\\t20\\t \\tProduct mass tolerance (p.p.m.)\\t20\\t \\tPeptide FDR (%)\\t1\\t \\tProtein FDR (%)\\t0\",\n",
            "          \"(p.p.m.)\\t20\\t \\tPeptide FDR (%)\\t1\\t \\tProtein FDR (%)\\t0\\t \\t \\u25b2 CRITICAL STEP The digestion enzyme search parameter used, Trypsin Allow P, allows K-P and P cleavages that are typically disallowed by search engines when configured for trypsin specificity. These\",\n",
            "          \"search engines when configured for trypsin specificity. These two additional bond cleavages are allowed because the digestion protocol employs both Lys-C and trypsin. Lys-C often cleaves at K-P linkages, and trypsin can occasionally cleave at both R-P and\",\n",
            "          \"linkages, and trypsin can occasionally cleave at both R-P and K-P, Therefore, the missed cleavage allowance is set to 4 instead of a typical value of 2 when using trypsin. Other search engines may have the flexibility to configure custom digestion\",\n",
            "          \"engines may have the flexibility to configure custom digestion specificity to explicitly allow K-P cleavage when disallowing R-P cleavage. \\u25b2 CRITICAL STEP Deamidation of glutamine (Q) residues is much slower than deamidation of asparagine (N) residues,\",\n",
            "          \"is much slower than deamidation of asparagine (N) residues, because of the kinetics of a reaction mechanism intermediate with a six-membered ring (Q) versus a five-membered ring (N). However, some search engines may not readily allow for selection of a\",\n",
            "          \"some search engines may not readily allow for selection of a deamidation variable modification only at N. Deamidation reactions are kinetically more favored at the elevated pH (10) encountered during the basic RP step in this protocol after the samples\",\n",
            "          \"during the basic RP step in this protocol after the samples are mixed. Analyze the final sample dataset with a two-cycle fixed/mix modifications search in Spectrum Mill that runs two consecutive searches with different sets of fixed modifications in each\",\n",
            "          \"searches with different sets of fixed modifications in each round and then produces a single integrated output. The two cycles allow for (i) labeling of both peptide N termini and lysines, and (ii) labeling of only lysines. Other search engines might\",\n",
            "          \"and (ii) labeling of only lysines. Other search engines might allow similar strategies to be executed by configuring the label as a variable modification on peptide N termini and on lysines. \\u25b2 CRITICAL STEP Including variable modifications during peptide\",\n",
            "          \"\\u25b2 CRITICAL STEP Including variable modifications during peptide identification of MS/MS spectra to account for common foreseeable sample handling modifications is not only important for calculating the metrics to monitor sample-handling QC, but also\",\n",
            "          \"calculating the metrics to monitor sample-handling QC, but also diminishes the potential for falsepositive identification of otherwise lower-scoring unmodified sequences. Depending on the capabilities and calculation methods of the quantitation software\",\n",
            "          \"and calculation methods of the quantitation software used, it may be appropriate to exclude PSMs related to sample handling modifications from quantitative calculations that are expected to occur disproportionately in individual samples before the point\",\n",
            "          \"occur disproportionately in individual samples before the point of mixing. Correct the reporter-ion intensities for isotopic impurities before using the reporter-ion signals in each MS/MS spectrum for quantitative calculations. \\u25b2 CRITICAL STEP Each lot of\",\n",
            "          \"for quantitative calculations. \\u25b2 CRITICAL STEP Each lot of reagent obtained from its manufacturer is accompanied by a certificate of analysis (this is also available on the manufacturer\\u2019s website) that contains isotopic correction factors to be used by\",\n",
            "          \"that contains isotopic correction factors to be used by the quantitation software. These correction factors are primarily used to account for naturally occurring levels of 13C at the unlabeled carbons in the mass-tag portion of the labeling reagents.\",\n",
            "          \"carbons in the mass-tag portion of the labeling reagents. Because very-high-purity sources of 15N and 13C are routinely available and are used for the labeled positions in the reagent, near-full incorporation is typical. The correction factors provided by\",\n",
            "          \"incorporation is typical. The correction factors provided by the reagent manufacturer represent MS measurements of the isotope profile for each lot of reagent and thus comingle the contributions of the unlabeled carbon and the source of heavy isotope.\",\n",
            "          \"of the unlabeled carbon and the source of heavy isotope. Because there are more unlabeled carbons in the TMT structure than in the iTRAQ structure, attention to isotopic correction to achieve accurate quantitation is more important for TMT than for iTRAQ.\",\n",
            "          \"accurate quantitation is more important for TMT than for iTRAQ. \\u25b2 CRITICAL STEP In complex samples such as tissue, the typical precursor mass window used for MS/MS (i.e., 0.7\\u20132.0 m/z) passes more ions than just the dominant peak in any given m/z window.\",\n",
            "          \"more ions than just the dominant peak in any given m/z window. If these additional ions have an isobaric mass-tag reporter as part of the structure, then fragmentation of these ions will produce mass-tag ions that add to the reporter-ion series from the\",\n",
            "          \"mass-tag ions that add to the reporter-ion series from the dominant peak. As most peptides in a sample are derived from proteins whose levels are not changing (or not changing substantially), the interfering signals from this background in every mass-tag\",\n",
            "          \"the interfering signals from this background in every mass-tag channel compress the observed ratios of regulated peptides/proteins in the samples. When combining reporter-ion quantitation from multiple PSMs to the protein level, several software programs\",\n",
            "          \"multiple PSMs to the protein level, several software programs offer mechanisms to exclude PSMs that exhibit substantial interference. A precursor-ion purity filter is commonly used that determines the ratio of the intensity of the precursor ion and\",\n",
            "          \"determines the ratio of the intensity of the precursor ion and isotopes of the primary peptide identified in the MS/MS spectrum to the total intensity in the mass window isolated for precursor-ion transmission. Use of a precursor-ion purity of >50% to 70%\",\n",
            "          \"transmission. Use of a precursor-ion purity of >50% to 70% is recommended. Various software packages employ different mechanisms for combining the PSM-level measurements to the protein level from which the constituent peptides are derived. Spectrum Mill\",\n",
            "          \"from which the constituent peptides are derived. Spectrum Mill takes the ratios at the PSM level, then calculates the protein-level ratio as the median of all PSM ratios. This strategy diminishes the overall impact of outliers but otherwise gives each PSM\",\n",
            "          \"the overall impact of outliers but otherwise gives each PSM an equal contribution to the protein-level ratio. Another common strategy is to sum the intensity for each reporter-ion channel from multiple PSMs contributing to the protein before taking the\",\n",
            "          \"multiple PSMs contributing to the protein before taking the ratios. This strategy seeks to weight the contribution of each PSM by the reporter-ion signal strength, with the implicit assumption that low-abundance signals are less accurate. Other variants\",\n",
            "          \"that low-abundance signals are less accurate. Other variants of these basic approaches involve combining multiple PSMs to the peptide level first before combining to the protein level. These strategies seek to limit the bias of individual peptides that\",\n",
            "          \"strategies seek to limit the bias of individual peptides that are observed in multiple PSMs because of different precursor charge states or sample-handling modifications. Troubleshooting Troubleshooting advice can be found in Table 2. Timing All\",\n",
            "          \"Troubleshooting advice can be found in Table 2. Timing All approximate timing given below is for a single 10-plex sample. Steps 1\\u201310, (optional) cryopulverization of tissue blocks: 2.5 h for ten non-OCT tissue blocks; 3.5 h for ten OCT tissue blocks Steps\",\n",
            "          \"non-OCT tissue blocks; 3.5 h for ten OCT tissue blocks Steps 11\\u201318, tissue lysis: 90 min Steps 19\\u201327, estimation of protein yield using a Pierce BCA Kit: 60 min Steps 28 and 29, reduction and alkylation: 2 h Steps 30\\u201335, in-solution digestion: 2 h +\",\n",
            "          \"and alkylation: 2 h Steps 30\\u201335, in-solution digestion: 2 h + overnight digestion + 30 min Steps 36\\u201345, peptide desalting of the digest by SPE: 90\\u2013120 min per set of ten samples Steps 46\\u201356, estimation of peptide amount using a Pierce BCA Kit: 60 min\",\n",
            "          \"estimation of peptide amount using a Pierce BCA Kit: 60 min Steps 57\\u201362, TMT labeling of 300 \\u03bcg of peptide per channel: 90 min Steps 63\\u201372, desalting of mixing QC test sample: 30 min Steps 73 and 74, data analysis for the assessment of labeling efficiency\",\n",
            "          \"and 74, data analysis for the assessment of labeling efficiency and mixing: 1 h Steps 75 and 76, quenching of the TMT-labeling reaction: 20 min Steps 77\\u201386, peptide desalting of the labeled peptides by SPE: 30 min Steps 87\\u201399, offline HPLC fractionation:\",\n",
            "          \"by SPE: 30 min Steps 87\\u201399, offline HPLC fractionation: 60 min + overnight QC + 5 h Steps 100\\u2013107, combined IMAC enrichment and phosphopeptide desalting: bead preparation: 90 min Steps 108 and 109, peptide sample preparation: 15 min Steps 110\\u2013113,\",\n",
            "          \"108 and 109, peptide sample preparation: 15 min Steps 110\\u2013113, combined IMAC enrichment and phosphopeptide desalting: phosphoenrichment: 45 min Steps 114\\u2013126, combined IMAC enrichment and phosphopeptide desalting: stage-tip desalting of phosphoenrichment:\",\n",
            "          \"desalting: stage-tip desalting of phosphoenrichment: 90 min Steps 127\\u2013129, data analysis: <1 d Anticipated results Assessing intra-plex, inter-plex, and inter-laboratory variation The protocol was tested across three independent laboratories with full\",\n",
            "          \"was tested across three independent laboratories with full process replicates, starting with cryofractured tissue material, for a total of 20 patient-derived xenograft tumor tissue samples per laboratory. As established in previous methods comparison\",\n",
            "          \"per laboratory. As established in previous methods comparison studies, we analyzed the proteome differences in two very different breast cancer subtypes, basal-like (WHIM2) and luminal (WHIM16) breast samples. The experimental design for this benchmarking\",\n",
            "          \"breast samples. The experimental design for this benchmarking study across three laboratories is depicted in Fig. 1b. As all three groups used the same experimental design and samples, the Procedure can be tested for intra-plex, inter-plex, and\",\n",
            "          \"the Procedure can be tested for intra-plex, inter-plex, and inter-laboratory variation. All participating laboratories followed this protocol, with similar chromatography and MS setups. Although all three laboratories used the same model of mass\",\n",
            "          \"Although all three laboratories used the same model of mass spectrometer, a Thermo Fisher Scientific Orbitrap Fusion Lumos, the LC models were not all the same. The off-line systems used for bRP were an Agilent 1100 (Broad Institute), an Agilent 1220\",\n",
            "          \"for bRP were an Agilent 1100 (Broad Institute), an Agilent 1220 (Johns Hopkins), and an Agilent 1100 (PNNL). For online LC systems, a Thermo Fisher Scientific Easy nLC 1200 was used by both the Broad Institute and Johns Hopkins, whereas a Waters\",\n",
            "          \"by both the Broad Institute and Johns Hopkins, whereas a Waters nanoAcquity instrument was used by PNNL. The gradient-mixing performance inevitably varies between LC manufacturers. Data from all three laboratories were analyzed centrally using the\",\n",
            "          \"from all three laboratories were analyzed centrally using the Spectrum Mill software package, so that all the described variances are due to experimental factors rather than introduced data analysis differences between the participating laboratories.\",\n",
            "          \"analysis differences between the participating laboratories. Comparison of proteome and phosphoproteome datasets We observed highly reproducible proteome and phosphoproteome datasets across three independent laboratories after following the described\",\n",
            "          \"three independent laboratories after following the described protocol. Each laboratory achieved proteome coverages of, on average, 145,000 distinct tryptic peptides (Fig. 2a) and >10,000 proteins, with a maximum deviation across replicates and centers of\",\n",
            "          \"with a maximum deviation across replicates and centers of <7% for protein observations (Fig. 2b). For each protein, we required at least two distinct peptides for identification and two TMT ratio counts for quantification. Out of all quantified proteins,\",\n",
            "          \"counts for quantification. Out of all quantified proteins, 77% were detected by all three centers (Supplementary Fig. 5a). The protein analysis presented here was restricted to ortholog-specific peptides for protein identification and quantification, and\",\n",
            "          \"peptides for protein identification and quantification, and up to 7,700 human proteins derived from tumor cells and up to 3,100 mouse proteins derived from mouse stroma and blood were quantified (Fig. 2b). The phosphoproteome coverage at the peptide level\",\n",
            "          \"(Fig. 2b). The phosphoproteome coverage at the peptide level was, on average, 35,000 phosphopeptides per experiment across laboratories and >31,000 phosphosites in every analyzed TMT-plex experiment (Fig. 2c,d). As, in general, ~35% of human and mouse\",\n",
            "          \"experiment (Fig. 2c,d). As, in general, ~35% of human and mouse tryptic peptides are identical in their amino acid sequences, we did not filter out peptides shared between human and mouse orthologs. Most of the shared signal in the PDX samples is\",\n",
            "          \"orthologs. Most of the shared signal in the PDX samples is contributed by the human tumor material, which contributes up to 55\\u201364% of the total protein signal. The overlap of all quantified phosphosites across three laboratories is 40% (Supplementary Fig.\",\n",
            "          \"across three laboratories is 40% (Supplementary Fig. 5b). For each of the three laboratories, 64\\u201369% of phosphosites were observed in both replicate experiments (Supplementary Fig. 5c-e). In each experiment, more than half of all the phosphosites were\",\n",
            "          \"In each experiment, more than half of all the phosphosites were detected by only a single MS2 scan, whereas every quantified protein is detected by at least two distinct peptides. Reduced overlap at the phosphosite-level relative to protein-level\",\n",
            "          \"overlap at the phosphosite-level relative to protein-level observations may arise because of the variable recovery of phosphopeptides from the additional experimental steps associated with IMAC phosphopeptide enrichment, as well as stochasticity in\",\n",
            "          \"IMAC phosphopeptide enrichment, as well as stochasticity in sampling by the mass spectrometer and minor variability in LC-MS performance. Nonetheless, reproducible and overlapping quantification of >21,000 phosphosites covering 5,384 phosphoproteins\",\n",
            "          \"of >21,000 phosphosites covering 5,384 phosphoproteins across three laboratories was achieved (Supplementary Fig. 5b). Comparison of the reproducibility of protein and phosphoprotein quantification To compare the reproducibility of protein and phosphosite\",\n",
            "          \"To compare the reproducibility of protein and phosphosite quantification, we referenced the reporter intensity of each individual channel against the average across all ten channels at the PSM level. This strategy allows comparison of the quantification\",\n",
            "          \"level. This strategy allows comparison of the quantification data of each individual tumor sample within a TMT-10-plex experiment, as well as across multiplex experiments and across different laboratories. For both the proteome and the phosphoproteome\",\n",
            "          \"laboratories. For both the proteome and the phosphoproteome data, we observe better correlations for the human-only subset of the dataset, as compared with the combined human plus mouse dataset (Supplementary Fig. 6). This is due to the varying\",\n",
            "          \"dataset (Supplementary Fig. 6). This is due to the varying contributions of mouse stroma components in the different cryofractured aliquots used for this study (Supplementary Fig. 7); therefore, we focused on human proteins and phosphosites for subsequent\",\n",
            "          \"we focused on human proteins and phosphosites for subsequent analyses. As expected, the best correlations were observed within a TMT-10-plex, with average r values of 0.94 and 0.85 for protein and phosphosite quantification, respectively (Fig. 3). The\",\n",
            "          \"and phosphosite quantification, respectively (Fig. 3). The high degree of reproducibility of this approach was demonstrated by very similar correlations for the intra- and inter-laboratory comparisons, with median r values of 0.88\\u20130.94 and 0.72\\u20130.88 at\",\n",
            "          \"comparisons, with median r values of 0.88\\u20130.94 and 0.72\\u20130.88 at the protein and phosphosite levels, respectively. We also studied the relationship between minimal ratio counts of proteins and phosphosites with regard to quantification reproducibility\",\n",
            "          \"and phosphosites with regard to quantification reproducibility (Supplementary Fig. 8). For the analyzed datasets, we find the best tradeoff between the highest quantitative reproducibility and the best coverage at two ratio counts for proteins and one\",\n",
            "          \"and the best coverage at two ratio counts for proteins and one ratio count for phosphosites. Isobaric tagging methods have been shown to improve the precision of quantification, whereas label-free methods suffer from high technical variation for PTM\",\n",
            "          \"label-free methods suffer from high technical variation for PTM applications, in which the majority of quantification events rely on the quantification of single peptides. Using the optimal filtering steps described above, we found very consistent results\",\n",
            "          \"steps described above, we found very consistent results for the analysis of breast cancer-relevant driver and biomarker proteins such as ESR1, GATA3, FOXA1, TP53, EGFR, and KRT5 (Supplementary Fig. 9). These proteins were all quantified at similar\",\n",
            "          \"Fig. 9). These proteins were all quantified at similar amplitudes across experiments with levels matching the known luminal and basal subtypes. To identify proteins and phosphosites that are specific for the luminal and basal PDX samples, we used a\",\n",
            "          \"are specific for the luminal and basal PDX samples, we used a two-sample t test and compared the overlap of proteins that were called to be specific to either of the subtypes across the different laboratories (FDR 1%). On the protein level, we observed an\",\n",
            "          \"laboratories (FDR 1%). On the protein level, we observed an overlap of 62.3% and 61.8% for the luminal and basal proteomes, respectively (Fig. 4a,c). For phosphoproteome, the overlap was 27.7% and 28% for luminal and basal phosphosites, respectively (Fig.\",\n",
            "          \"and 28% for luminal and basal phosphosites, respectively (Fig. 4b,d). Using Gene Set Enrichment Analysis, we also converted the proteome and phosphoproteome data to pathways from the Molecular Signature database, MSigDB. As single protein and phophosite\",\n",
            "          \"Signature database, MSigDB. As single protein and phophosite measurements are further collapsed at the pathway level, even higher correlations can be observed for the proteome and phosphoproteome pathway datasets (Supplementary Fig. 10). Data from all\",\n",
            "          \"pathway datasets (Supplementary Fig. 10). Data from all laboratories confirmed the subtype identity of the analyzed PDX tumor sample at the pathway level. In studies of human-only breast tumor tissue, we have achieved typical coverage depth of >8,000\",\n",
            "          \"tumor tissue, we have achieved typical coverage depth of >8,000 proteins per experiment (>2 unique peptides/protein) and >25,000 phosphorylation sites per experiment using the protocol described here (P.M., L.C.T., K.K., K.R.C., M.A. Gillette, D.R.M.,\",\n",
            "          \"here (P.M., L.C.T., K.K., K.R.C., M.A. Gillette, D.R.M., F.M., H.K., S.R.D., S.A.C. & S. Satpathy, Broad Institute; M. Ellis, Baylor College of Medicine; L. Ding, Washington University School of Medicine; K. Ruggles and D. Fenyo, New York University; B.\",\n",
            "          \"of Medicine; K. Ruggles and D. Fenyo, New York University; B. Zhang, Baylor College of Medicine; H. Rodriguez, C. Kinsinger, E. Boja and M. Mesri, National Cancer Institute; data accessible at https://cptac-data-portal.georgetown.edu/cptac/s/S039?; Study\",\n",
            "          \"https://cptac-data-portal.georgetown.edu/cptac/s/S039?; Study name: CPTAC breast cancer confirmatory study). Concluding remarks The optimized protocol presented here enables deep-scale and reproducible proteomics data and results to be obtained within and\",\n",
            "          \"proteomics data and results to be obtained within and across laboratories conducting tissue or cell analyses. The workflow was systematically characterized and analytically validated across three independent laboratories using two distinct breast cancer\",\n",
            "          \"three independent laboratories using two distinct breast cancer subtypes. Proteome coverages of >10,000 proteins per sample and >37,000 quantified phosphosites per sample were obtained for each PDX-tumor sample with high reproducibility with respect to\",\n",
            "          \"each PDX-tumor sample with high reproducibility with respect to both the overall numbers, differentially detected proteins and phosphosites, and the biology represented. The high-quality data obtained are suitable for proteogenomic data integration and\",\n",
            "          \"obtained are suitable for proteogenomic data integration and will enable proteomics-based pan-cancer studies. The entire procedure, including sample processing and data generation, can be completed within 10 d for ten tissue samples, and 100 samples can\",\n",
            "          \"within 10 d for ten tissue samples, and 100 samples can be completed in ~4 months, using a single LC-MS/MS instrument. Although the present study focuses on tissue analysis, the procedure is equally applicable to cell lines. Supplementary Material\",\n",
            "          \"is equally applicable to cell lines. Supplementary Material Competing interests The authors declare no competing interests. Additional information Supplementary information is available for this paper at https://doi.org/10.1038/s41596-018-0006-9. Reprints\",\n",
            "          \"paper at https://doi.org/10.1038/s41596-018-0006-9. Reprints and permissions information is available at www.nature.com/reprints. Related links 1. Mertins, P. et al. Nature 534, 55\\u201362 (2016) http://dx.doi.org/10.1038/nature18003 2. Zhang, H. et al. Cell\",\n",
            "          \"http://dx.doi.org/10.1038/nature18003 2. Zhang, H. et al. Cell 166, 755\\u2013765 (2016) http://dx.doi.org/10.1016/j.cell.2016.05.069 3. Mundt, F. et al. Cancer Res. 78, 2732\\u20132746 (2018) http://dx.doi.org/10.1158/0008-5472.CAN-17-1990 Optimized workflow and\",\n",
            "          \"Optimized workflow and experimental design of global proteome and phosphoproteome analysis in tissues using TMT. a, Multiple aspects of sample handling were optimized based on a preexisting workflow for global proteome and phosphoproteome analysis\",\n",
            "          \"workflow for global proteome and phosphoproteome analysis (Mertins et al.). Some of the conditions tested relative to the preexisting workflow were (i) digestion at higher protein concentrations, which effectively increases the enzyme concentration during\",\n",
            "          \"which effectively increases the enzyme concentration during digestion, resulting in lower missed cleavage rates; (ii) reconstitution of lysyl endopeptidase in water, instead of 50 mM acetic acid, which better maintains the activity of the enzyme; (iii)\",\n",
            "          \"acid, which better maintains the activity of the enzyme; (iii) quantification of peptides by BCA before isobaric labeling, which yields more accurate input amounts than BCA at the protein level; (iv) offline basic RP fractionation using either Agilent or\",\n",
            "          \"(iv) offline basic RP fractionation using either Agilent or Waters columns, which yield equivalent results; and (v) optimization of HCD energy for each individual instrument, rather than the use of a common collision energy, which improved spectral\",\n",
            "          \"the use of a common collision energy, which improved spectral quality. The relevant steps of the Procedure are indicated in red. b, Multiple mice of basal (WHIM 2) and luminal (WHIM16) subtypes were grown, and the tumors of each subtype were pooled\",\n",
            "          \"subtypes were grown, and the tumors of each subtype were pooled together. Tumors of each subtype from multiple mice were cryofractured and aliquots of the homogenized powders were distributed to the three different laboratories for global proteome and\",\n",
            "          \"to the three different laboratories for global proteome and phosphoproteome analysis. Each laboratory analyzed 2\\u00d7 TMT-10 plexes. Intra-plex, intra-lab, and inter-lab comparisons were conducted to test depth of coverage and reproducibility. PCC1\\u20133 indicate\",\n",
            "          \"to test depth of coverage and reproducibility. PCC1\\u20133 indicate Protein Characterization Centers 1 (Broad Institute), 2 (Johns Hopkins University), and 3 (Pacific Northwest National Laboratory), respectively. BCA, bicinchoninic acid; HCD, higher-energy\",\n",
            "          \"respectively. BCA, bicinchoninic acid; HCD, higher-energy collision dissociation. a Adapted from Extended Data Fig. 1 in ref. , Springer Nature. Deep and reproducible coverage of tumor tissue proteomes and phosphoproteomes across three laboratories. a-d,\",\n",
            "          \"proteomes and phosphoproteomes across three laboratories. a-d, Bar charts depicting the number of quantified distinct peptide sequences (a) and proteins (b) identified in basic RP fractions of proteome measurements, and the number of distinct\",\n",
            "          \"fractions of proteome measurements, and the number of distinct phosphorylated peptides (c) and individual phosphorylation sites (d) quantified in the metal-affinity enriched fractions. Solid-colored bars represent the proportion of human features and\",\n",
            "          \"bars represent the proportion of human features and shaded bars represent the proportion of mouse-specific features. Numbers inside the bars represent the numbers of quantified human and mouse features, respectively. PDX models used in this study were\",\n",
            "          \"features, respectively. PDX models used in this study were approved by the institutional animal care and use committee at Washington University in St. Louis. Assessment of the variability of TMT quantitation. Our experimental design enables the assessment\",\n",
            "          \"quantitation. Our experimental design enables the assessment of intra-plex, inter-plex, and inter-laboratory variation of TMT quantitation. Pearson correlation coefficients between replicate measurements were calculated and visualized in box-and-whiskers\",\n",
            "          \"measurements were calculated and visualized in box-and-whiskers plots. a, Correlations calculated from proteome measurements comparing intra-plex replicates (left), inter-plex replicates (middle), and inter-laboratory replicates (right). b, Correlations\",\n",
            "          \"and inter-laboratory replicates (right). b, Correlations of quantified phosphorylation sites. PDX models used in this study were approved by the institutional animal care and use committee at Washington University in St. Louis. Breast cancer\",\n",
            "          \"committee at Washington University in St. Louis. Breast cancer subtype-specific protein and phosphorylation site expression identified by three laboratories. Differences in the expression of proteins and phosphorylation sites between luminal and basal\",\n",
            "          \"of proteins and phosphorylation sites between luminal and basal tumor subtypes were determined by a two-sample moderated t test at a 1% FDR. The results of the analysis are illustrated as \\u2018UpSet\\u2019 plots. Horizontal bars indicate total number of features\",\n",
            "          \"plots. Horizontal bars indicate total number of features detected by each laboratory; vertical bars depict the number of jointly detected features, as indicated by the layout matrix below. a,b, Comparison of proteins (a) and phosphorylation sites (b)\",\n",
            "          \"a,b, Comparison of proteins (a) and phosphorylation sites (b) highly expressed in the basal subtype. c,d, Comparison of proteins (c) and phosphorylation sites (d) highly expressed in the luminal subtype. Approximately two-thirds of the phosphosites that\",\n",
            "          \"subtype. Approximately two-thirds of the phosphosites that were quantified as differentially expressed by a single laboratory were also only detected by a single laboratory. PDX models used in this study were approved by the institutional animal care and\",\n",
            "          \"this study were approved by the institutional animal care and use committee at Washington University in St. Louis. Pooling fractions for proteome analysis \\t1\\t2\\t3\\t4\\t5\\t6\\t7\\t8\\t9\\t10\\t11\\t12\\t \\tA\\t\\t\\tA\\tA\\tA\\tA\\tA\\tA\\tA\\tA\\tA\\tA\\t \\tB\\t10\\t9\\t8\\t7\\t6\\t5\\t4\\t3\\t2\\t1\\tA\\tA\",\n",
            "          \"A\\t\\t\\tA\\tA\\tA\\tA\\tA\\tA\\tA\\tA\\tA\\tA\\t \\tB\\t10\\t9\\t8\\t7\\t6\\t5\\t4\\t3\\t2\\t1\\tA\\tA\\t \\tC\\t11\\t12\\t13\\t14\\t15\\t16\\t17\\t18\\t19\\t20\\t21\\t22\\t \\tD\\t10\\t9\\t8\\t7\\t6\\t5\\t4\\t3\\t2\\t1\\t24\\t23\\t \\tE\\t11\\t12\\t13\\t14\\t15\\t16\\t17\\t18\\t19\\t20\\t21\\t22\\t \\tF\\t10\\t9\\t8\\t7\\t6\\t5\\t4\\t3\\t2\\t1\\t24\\t23\\t \\tG\\t11\\t12\\t13\\t14\\t15\\t16\\t17\\t18\\t19\\t20\\t21\\t22\",\n",
            "          \"G\\t11\\t12\\t13\\t14\\t15\\t16\\t17\\t18\\t19\\t20\\t21\\t22\\t \\tH\\t\\t\\t\\t\\t\\t24\\t24\\t24\\t24\\t24\\t24\\t23\\t \\t Combining strategy for a 4.6-mm column. Underlined numbers indicate the fractions meant to be pooled into fraction 1. Italicized numbers indicate the fractions meant to be pooled into\",\n",
            "          \"numbers indicate the fractions meant to be pooled into fraction 2. These indications are meant to make pooling easier visually. Troubleshooting table Step\\tProblem\\tPossible reason\\tSolution\\t \\t44\\tLarge amounts of OCT contamination seen in QC mass\",\n",
            "          \"44\\tLarge amounts of OCT contamination seen in QC mass spectrometry runsHigh missed-cleavage rates for tryptic peptides (Supplementary Fig. 2)\\tLarge contamination of OCT in cryopulverized tissueDigest efficiency, especially for trypsin, is dependent on\",\n",
            "          \"efficiency, especially for trypsin, is dependent on enzyme concentration during digestion\\tUse basic RP fractionation to dilute OCT signal across multiple fractionsUse low volumes of extraction buffer to increase the initial protein concentrations\\t \\t73,\",\n",
            "          \"buffer to increase the initial protein concentrations\\t \\t73, 74\\tPoor labeling efficiency and/or poor mixing control\\tHigh levels of plasma contamination, leading to poor labeling efficiency or inaccurate protein BCA assay\\tIf a channel does not have\",\n",
            "          \"or inaccurate protein BCA assay\\tIf a channel does not have sufficient labeling incorporation, additional TMT is added to the sample and another 1-h incubation is performed with shaking. If the samples are too variable from each other in the mixing\",\n",
            "          \"If the samples are too variable from each other in the mixing control, additional material should be labeled and incorporated, or material should be removed, depending on the nature of the channel, until the channels are ~1:1:1\\u2026\\t \\t97\\tHigh-pH separation\",\n",
            "          \"channel, until the channels are ~1:1:1\\u2026\\t \\t97\\tHigh-pH separation resolution is <75-80% uniqueness per fraction. For optimal performance across three laboratories and different column types, see Supplementary Fig. 1\\tColumn is overloaded or deteriorated\\tUse\",\n",
            "          \"Supplementary Fig. 1\\tColumn is overloaded or deteriorated\\tUse standard runs of synthetic peptides to monitor column performance and do not load more than 4 mg of peptides onto 4.6 \\u00d7 250-mm columns\\t \\t126\\tPhosphopeptide enrichment yields <90% enrichment\",\n",
            "          \"columns\\t \\t126\\tPhosphopeptide enrichment yields <90% enrichment specificities. For optimal performance across three laboratories, see Supplementary Fig. 4\\tpH of peptides in IMAC-binding buffer is >2, IMAC beads were not prepared freshly, or IMAC incubation\",\n",
            "          \"is >2, IMAC beads were not prepared freshly, or IMAC incubation was longer than 30 min\\tCheck pH with filter paper, prepare new IMAC beads for each experiment, and do not incubate for >30 min\"\n",
            "        ]\n",
            "      }\n",
            "    ]\n",
            "  },\n",
            "  \"PDC000393\": {\n",
            "    \"study_id\": \"26da108f-b554-4c81-8397-0aded036fe93\",\n",
            "    \"pdc_study_id\": \"PDC000393\",\n",
            "    \"study_submitter_id\": \"Sampling techniques for enrichment of PDAC - Proteome\",\n",
            "    \"program_id\": \"10251935-5540-11e8-b664-00a098d917f8\",\n",
            "    \"project_id\": \"267d6671-0e78-11e9-a064-0a9c39d33490\",\n",
            "    \"study_name\": \"Sampling techniques for enrichment of PDAC - Proteome\",\n",
            "    \"program_name\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"project_name\": \"CPTAC3 Discovery and Confirmatory\",\n",
            "    \"disease_type\": \"Other;Pancreatic Ductal Adenocarcinoma\",\n",
            "    \"primary_site\": \"Not Reported;Pancreas\",\n",
            "    \"analytical_fraction\": \"Proteome\",\n",
            "    \"experiment_type\": \"TMT10\",\n",
            "    \"embargo_date\": null,\n",
            "    \"cases_count\": 7,\n",
            "    \"aliquots_count\": 28,\n",
            "    \"protocol_id\": \"2ff8faf2-dbc7-490b-98d5-68607adac54f\",\n",
            "    \"protocol_submitter_id\": \"CPTAC JHU PDAC enrichment\",\n",
            "    \"program_submitter_id\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"protocol_name\": \"CPTAC JHU PDAC enrichment\",\n",
            "    \"protocol_date\": \"2022-05-09\",\n",
            "    \"publications\": [\n",
            "      {\n",
            "        \"publication_id\": \"bef019a4-50b1-11ed-a6f8-0a79dd92157b\",\n",
            "        \"pubmed_id\": \"36266629\",\n",
            "        \"doi\": \"10.1186/s12014-022-09373-x\",\n",
            "        \"title\": \"Neoplastic cell enrichment of tumor tissues using coring and laser microdissection for proteomic and genomic analyses of pancreatic ductal adenocarcinoma\",\n",
            "        \"journal\": \"Clin Proteomics\",\n",
            "        \"journal_url\": \"https://clinicalproteomicsjournal.biomedcentral.com/articles/10.1186/s12014-022-09373-x\",\n",
            "        \"year\": \"2022\",\n",
            "        \"abstract\": \"The identification of differentially expressed tumor-associated proteins and genomic alterations driving neoplasia is critical in the development of clinical assays to detect cancers and forms the foundation for understanding cancer biology. One of the challenges in the analysis of pancreatic ductal adenocarcinoma (PDAC) is the low neoplastic cellularity and heterogeneous composition of bulk tumors. To enrich neoplastic cells from bulk tumor tissue, coring, and laser microdissection (LMD) sampling techniques have been employed. In this study, we assessed the protein and KRAS mutation changes associated with samples obtained by these enrichment techniques and evaluated the fraction of neoplastic cells in PDAC for proteomic and genomic analyses.\",\n",
            "        \"full_text\": [\n",
            "          \"Neoplastic cell enrichment of tumor tissues using coring and laser microdissection for proteomic and genomic analyses of pancreatic ductal adenocarcinoma Background The identification of differentially expressed tumor-associated proteins and genomic\",\n",
            "          \"differentially expressed tumor-associated proteins and genomic alterations driving neoplasia is critical in the development of clinical assays to detect cancers and forms the foundation for understanding cancer biology. One of the challenges in the\",\n",
            "          \"for understanding cancer biology. One of the challenges in the analysis of pancreatic ductal adenocarcinoma (PDAC) is the low neoplastic cellularity and heterogeneous composition of bulk tumors. To enrich neoplastic cells from bulk tumor tissue, coring,\",\n",
            "          \"To enrich neoplastic cells from bulk tumor tissue, coring, and laser microdissection (LMD) sampling techniques have been employed. In this study, we assessed the protein and KRAS mutation changes associated with samples obtained by these enrichment\",\n",
            "          \"changes associated with samples obtained by these enrichment techniques and evaluated the fraction of neoplastic cells in PDAC for proteomic and genomic analyses. Methods Three fresh frozen PDAC tumors and their tumor-matched normal adjacent tissues\",\n",
            "          \"PDAC tumors and their tumor-matched normal adjacent tissues (NATs) were obtained from three sampling techniques using bulk, coring, and LMD; and analyzed by TMT-based quantitative proteomics. The protein profiles and characterizations of differentially\",\n",
            "          \"The protein profiles and characterizations of differentially expressed proteins in three sampling groups were determined. These three PDACs and samples of five additional PDACs obtained by the same three sampling techniques were also subjected to genomic\",\n",
            "          \"same three sampling techniques were also subjected to genomic analysis to characterize KRAS mutations. Results The neoplastic cellularity of eight PDACs ranged from less than 10% to over 80% based on morphological review. Distinctive proteomic patterns\",\n",
            "          \"based on morphological review. Distinctive proteomic patterns and abundances of certain tumor-associated proteins were revealed when comparing the tumors and NATs by different sampling techniques. Coring and bulk tissues had comparable proteome profiles,\",\n",
            "          \"Coring and bulk tissues had comparable proteome profiles, while LMD samples had the most distinct proteome composition compared to bulk tissues. Further genomic analysis of bulk, cored, or LMD samples demonstrated that KRAS mutations were significantly\",\n",
            "          \"LMD samples demonstrated that KRAS mutations were significantly enriched in LMD samples while coring was less effective in enriching for KRAS mutations when bulk tissues contained a relatively low neoplastic cellularity. Conclusions In addition to bulk\",\n",
            "          \"low neoplastic cellularity. Conclusions In addition to bulk tissues, samples from LMD and coring techniques can be used for proteogenomic studies. The greatest enrichment of neoplastic cellularity is obtained with the LMD technique. Supplementary\",\n",
            "          \"cellularity is obtained with the LMD technique. Supplementary Information The online version contains supplementary material available at 10.1186/s12014-022-09373-x. Introduction Pancreatic ductal adenocarcinoma (PDAC) accounts for\\u2009>\\u200990% of malignant\",\n",
            "          \"ductal adenocarcinoma (PDAC) accounts for\\u2009>\\u200990% of malignant tumors of the pancreas. Clinically, the majority of PDAC present at an advanced-stage with unresectable cancer at the time of diagnosis. Despite the rapid development of targeted- and\",\n",
            "          \"of diagnosis. Despite the rapid development of targeted- and immuno-therapies for cancers, the outcome of PDAC is still dismal with a 5-yr survival rate of 11%. It is well-known that PDAC is driven by the accumulation of genomic aberrations, and that\",\n",
            "          \"is driven by the accumulation of genomic aberrations, and that these, in turn, drive the phenotypical and molecular transformation from normal pancreatic epithelial cells into a non-invasive neoplastic precursor lesion and eventually to an infiltrating\",\n",
            "          \"neoplastic precursor lesion and eventually to an infiltrating malignant cancer. Recent large-scale proteogenomic studies such as the Clinical Proteomic Tumor Analysis Consortium (CPTAC) PDAC study have significantly advanced our knowledge of the molecular\",\n",
            "          \"have significantly advanced our knowledge of the molecular biology of PDAC. Somatic mutations in the KRAS gene have been identified in\\u2009>\\u200990% of PDACs. Other molecular events have also been identified as drivers in the development and progression of PDAC,\",\n",
            "          \"as drivers in the development and progression of PDAC, including the loss of function of tumor suppressor genes TP53, p16/CDKN2A and SMAD4, activation of oncogenic Her2/neu, and germline mutations of BRCA1/2, PALB2, ATM, MLH1 and others. In conjunction\",\n",
            "          \"of BRCA1/2, PALB2, ATM, MLH1 and others. In conjunction with genomic findings, proteomic studies have identified up- and down-regulated proteins and signaling pathways in PDAC. The recent comprehensive proteomic analysis of 135 PDACs has identified a\",\n",
            "          \"comprehensive proteomic analysis of 135 PDACs has identified a number of protein changes in tumors compared to NATs and normal ductal epithelium. Proteins such as HK2, LOXL2, COL12A1, C19orf33, TSPAN1 and MDK have been considered as diagnostic or\",\n",
            "          \"C19orf33, TSPAN1 and MDK have been considered as diagnostic or prognostic markers, as well as potential therapeutic targets. As these multi-omics studies, in particular proteomic analyses, have potential clinical applications, it is clear that\",\n",
            "          \"have potential clinical applications, it is clear that identification of differentially expressed tumor-associated proteins is a critical step for the study of cancer biology, pathophysiologic perturbations, and the development of potential clinical\",\n",
            "          \"perturbations, and the development of potential clinical assays. It is well established, however, that this process toward clinical application is challenging, since cellular heterogeneity, the mixture of neoplastic and non-neoplastic cells, can obscure\",\n",
            "          \"the mixture of neoplastic and non-neoplastic cells, can obscure neoplasm-specific protein expression patterns. The analysis of bulk tumor tissue is a conventional approach to profiling tumor-associated proteins; however, bulk tumor, since it relies on\",\n",
            "          \"proteins; however, bulk tumor, since it relies on gross tumor identification, can contain substantial amounts of non-neoplastic tissues. This is particularly a problem in PADC, where the majority of tumors have\\u2009<\\u200920% neoplastic cellularity, and only some,\",\n",
            "          \"of tumors have\\u2009<\\u200920% neoplastic cellularity, and only some, often unusual, tumors have reasonably high neoplastic cellularity. Bulk tumor tissue in the pancreas typically contains normal pancreas, mucin, collagen, fibroblasts, vascular endothelial cells,\",\n",
            "          \"mucin, collagen, fibroblasts, vascular endothelial cells, and inflammatory cells. Thus, enrichment of neoplastic cells is an important step in the proteomic analysis of tumors, particularly PDAC. To address the issue of low neoplastic cellularity, several\",\n",
            "          \"To address the issue of low neoplastic cellularity, several micro-dissection techniques, such as laser microdissection (LMD) and coring of tumor tissue, have been used to enrich samples for neoplastic cells. In the LMD approach, microscope sections are\",\n",
            "          \"neoplastic cells. In the LMD approach, microscope sections are examined and areas selected and excised using a laser. This technique can be used to isolate a relatively pure cellular population for further multi-omics analysis. However, it is\",\n",
            "          \"population for further multi-omics analysis. However, it is time-consuming and the yield of recovered tissue material is low. In addition, the laser can heat the tissues to be studied, and this heat can cause artifacts and degradation of nucleic acids. In\",\n",
            "          \"heat can cause artifacts and degradation of nucleic acids. In the coring technique, aspiration needles of different gauges are used to punch the selected targeted areas from larger blocks of tissue. Similar to LMD, it can provide a relatively pure\",\n",
            "          \"of tissue. Similar to LMD, it can provide a relatively pure cellular component from the top layer of tissue blocks. However, the cellular components beneath the top layer of cells in the cored tissues are difficult to determine as tissues are almost never\",\n",
            "          \"tissues are difficult to determine as tissues are almost never perfectly oriented vertically. Although these approaches are routinely applied to genetic analyses, they are often overlooked in proteomic analyses. In this study, we compared different\",\n",
            "          \"in proteomic analyses. In this study, we compared different sampling techniques, including bulk tumor tissue, LMD, and coring, for the proteomic and genomic analyses of PDAC. The goal was to evaluate the impact of the different sampling techniques on the\",\n",
            "          \"evaluate the impact of the different sampling techniques on the observed proteomic and genomic data, and to address the potential utility of LMD and coring techniques in the enrichment of neoplastic cells. Methods Materials and reagents BCA protein assay\",\n",
            "          \"cells. Methods Materials and reagents BCA protein assay kit (Pierce), urea, tris (2-carboxyethyl) phosphine (TCEP), tandem mass tag (TMT) reagents and dithiothreitol (DTT) were purchased from Thermo Fisher Scientific (Waltham, MA). Sequencing-grade\",\n",
            "          \"from Thermo Fisher Scientific (Waltham, MA). Sequencing-grade trypsin was purchased from Promega (Madison, WI). Lys-C was purchased from Wako Chemicals (Richmond, VA). C18 SPE columns were purchased from Waters (Milford, MA). All other reagents, including\",\n",
            "          \"from Waters (Milford, MA). All other reagents, including iodoacetamide (IAA), formic acid (FA), and anhydrous acetonitrile (ACN), were purchased from Sigma-Aldrich (St. Louis, MO). Samples collection and process Fresh-frozen blocks of primary PDACs from\",\n",
            "          \"and process Fresh-frozen blocks of primary PDACs from treatment na\\u00efve patients and tumor-matched NATs were prospectively collected from surgically resected specimens according to the CPTAC guidelines. All study cases had no prior history of other\",\n",
            "          \"CPTAC guidelines. All study cases had no prior history of other malignancies, and the patients had not received systemic chemotherapy, or immune-related therapy. The clinical information and the neoplastic cellularity of study cases were determined by\",\n",
            "          \"the neoplastic cellularity of study cases were determined by histology review and summarized in Additional file 1: Table S1. Informed consent was obtained and reviewed by Institutional Review Boards at tissue collection sites. The diagnoses were confirmed\",\n",
            "          \"Boards at tissue collection sites. The diagnoses were confirmed by re-reviewing digital images of H&E stained slides by board-certified pathologists. The tumor area and tumor-matched normal adjacent tissues (NATs) were marked on microscope slides and\",\n",
            "          \"adjacent tissues (NATs) were marked on microscope slides and matched to corresponding tissue blocks. Tissue blocks were sampled using three techniques, including the bulk sampling of the entire section of the block, LMD for neoplastic cells, and coring of\",\n",
            "          \"section of the block, LMD for neoplastic cells, and coring of neoplastic areas using a 3\\u00a0mm diameter biopsy needle. NATs contained acinar cells and ductal epithelium, and scattered stromal and inflammatory cells, but did not contain neoplastic cells. All\",\n",
            "          \"inflammatory cells, but did not contain neoplastic cells. All samples were cryo-pulverized, aliquoted and stored for subsequent proteomic and genomic analyses. Proteomic analyses were performed in the Mass Spectrometry Core Facility at the Johns Hopkins\",\n",
            "          \"in the Mass Spectrometry Core Facility at the Johns Hopkins Biomarker Discovery and Translation Center. DNA sequencing was performed at the Broad Institute, and genomic analyses were performed in the Oncology Center at Washington University. Protein\",\n",
            "          \"in the Oncology Center at Washington University. Protein extraction and tryptic digestion for proteomics Protein extraction and digestion were performed as previously described. Briefly, each sample was lysed in lysis buffer containing 8\\u00a0M urea, 75\\u00a0mM\",\n",
            "          \"sample was lysed in lysis buffer containing 8\\u00a0M urea, 75\\u00a0mM NaCl, 50\\u00a0mM Tris (pH 8.0), 1\\u00a0mM EDTA, 2\\u00a0\\u03bcg/mL aprotinin, 10\\u00a0\\u03bcg/mL leupeptin, 1\\u00a0mM PMSF, 10\\u00a0mM NaF, phosphatase inhibitor cocktail 2 and 3 [1:100 dilution], and 20\\u00a0\\u03bcM PUGNAc. The protein\",\n",
            "          \"2 and 3 [1:100 dilution], and 20\\u00a0\\u03bcM PUGNAc. The protein concentration in the supernatant was measured by BCA assay. Proteins were reduced and alkylated with DTT (5\\u00a0mM, 37\\u00a0\\u00b0C, 1\\u00a0h) and IAA (10\\u00a0mM, room temperature (RT) for 45\\u00a0min in the dark). The reduced\",\n",
            "          \"room temperature (RT) for 45\\u00a0min in the dark). The reduced proteins were diluted 1:4 with 50\\u00a0mM Tris\\u2013HCl (pH 8.0) and incubated with Lys-C followed by trypsin digestion with the enzyme-to-substrate ratio of 1:50 overnight at RT. The digestion was quenched\",\n",
            "          \"ratio of 1:50 overnight at RT. The digestion was quenched by adjusting pH to\\u2009<\\u20093 with 50% of formic acid (FA). The peptides were desalted on reversed-phase C18 SPE columns and dried using Speed-Vac. Tandem Mass Tag (TMT) labeling and peptide fractionation\",\n",
            "          \"Tandem Mass Tag (TMT) labeling and peptide fractionation Dried peptide samples were dissolved in 50\\u00a0mM HEPES. 50 ul of each sample from three sampling techniques, including 100\\u00a0\\u03bcg of proteins from bulk samples, 30\\u00a0\\u03bcg of proteins from coring samples, and\",\n",
            "          \"from bulk samples, 30\\u00a0\\u03bcg of proteins from coring samples, and 6\\u00a0\\u03bcg of proteins from LMD samples, were labeled with 10-plex TMT reagents as described previously. Briefly, TMT reagents were added to each sample, and the mixtures were then incubated at RT\",\n",
            "          \"to each sample, and the mixtures were then incubated at RT for 1\\u00a0h, and quenched with 5% hydroxylamine at RT for 15\\u00a0min. Labeled peptides in each TMT set were desalted on reversed-phase C18 SPE columns, dried using Speed-Vac, and dissolved in 900 \\u03bcL of\",\n",
            "          \"SPE columns, dried using Speed-Vac, and dissolved in 900 \\u03bcL of buffer A (5\\u00a0mM ammonium formate in 2% ACN). Samples from each TMT set were fractionated by the basic reversed-phase liquid chromatography (bRPLC) with a 4.6\\u00a0mm\\u2009\\u00d7\\u2009250\\u00a0mm Zorbax Extend-C18\",\n",
            "          \"chromatography (bRPLC) with a 4.6\\u00a0mm\\u2009\\u00d7\\u2009250\\u00a0mm Zorbax Extend-C18 analytical column (3.5\\u00a0\\u03bcm beads, Agilent) lined up with an Agilent 1220 Series HPLC. A pooled sample from all tumors and NATs was also included in each TMT set as reference. Peptides were\",\n",
            "          \"was also included in each TMT set as reference. Peptides were separated using a non-linear gradient with buffer B (5\\u00a0mM ammonium formate in 90% ACN) as follows: 0% buffer B for 7\\u00a0min, 0\\u201316% buffer B for 6\\u00a0min, 16\\u201340% buffer B for 60\\u00a0min, 40\\u201344% buffer B\",\n",
            "          \"buffer B for 6\\u00a0min, 16\\u201340% buffer B for 60\\u00a0min, 40\\u201344% buffer B for 4\\u00a0min, 44\\u201360% buffer B for 5\\u00a0min, and holding at 60% buffer B for 14\\u00a0min. Fractions were concatenated into 24 fractions as described previously. Samples were resuspended in 3% ACN (0.1%\",\n",
            "          \"described previously. Samples were resuspended in 3% ACN (0.1% FA) prior to ESI-LC\\u2013MS/MS analysis. ESI-LC\\u2013MS/MS for global proteome data-dependent analysis (DDA) The TMT-labeled fractions were analyzed using Orbitrap Fusion Lumos mass spectrometer (Thermo\",\n",
            "          \"analyzed using Orbitrap Fusion Lumos mass spectrometer (Thermo Scientific). Approximately 0.8\\u00a0\\u03bcg of peptides were separated on an in-house packed 28\\u00a0cm\\u2009\\u00d7\\u200975\\u00a0mm diameter C18 column (1.9\\u00a0mm Reprosil-Pur C18-AQ beads (Dr. Maisch GmbH); Picofrit 10\\u00a0mm opening\",\n",
            "          \"C18-AQ beads (Dr. Maisch GmbH); Picofrit 10\\u00a0mm opening (New Objective)) lined up with an Easy nLC 1200 UHPLC system (Thermo Scientific). The column was heated to 50\\u00a0\\u00b0C using a column heater (Phoenix-ST). The flow rate was set at 200\\u00a0\\u00b5l/min. Buffer A [3%\",\n",
            "          \"(Phoenix-ST). The flow rate was set at 200\\u00a0\\u00b5l/min. Buffer A [3% ACN (0.1% FA)] and buffer B [90% ACN (0.1% FA)] were used. The peptides were separated with a 6\\u201330% buffer B gradient in 84\\u00a0min, eluted from the column and nanosprayed directly into the mass\",\n",
            "          \"eluted from the column and nanosprayed directly into the mass spectrometer in a data-dependent mode. Parameters for global proteomic samples were set as follows: MS1 resolution\\u201360,000, mass range\\u2013350 to 1800\\u00a0m/z, RF Lens\\u201330%, AGC Target\\u20134.0e5, Max\",\n",
            "          \"mass range\\u2013350 to 1800\\u00a0m/z, RF Lens\\u201330%, AGC Target\\u20134.0e5, Max injection time\\u201350\\u00a0ms, charge state include\\u20132\\u20136, dynamic exclusion\\u201345\\u00a0s. The cycle time was set to 2\\u00a0s, and within this 2\\u00a0s the most abundant ions per scan were selected for MS/MS in the\",\n",
            "          \"the most abundant ions per scan were selected for MS/MS in the orbitrap. MS2 resolution\\u201350,000, high-energy collision dissociation activation energy\\u201337, isolation width (m/z)\\u20130.7, AGC Target\\u20132.0e5, max injection time\\u2013105\\u00a0ms. Proteomics data processing\",\n",
            "          \"max injection time\\u2013105\\u00a0ms. Proteomics data processing Data were searched for peptides and proteins against a human RefSeq protein fasta database using the MS-GF\\u2009+\\u2009search engine and MS-PyCloud pipeline. MS/MS spectra were searched using a precursor-ion\",\n",
            "          \"pipeline. MS/MS spectra were searched using a precursor-ion mass tolerance of 10\\u00a0ppm. The cysteine carbamidomethylation (+\\u200957.0215), lysine and peptide N-terminal TMT labeling (+\\u2009229.1629), were specified as fixed modifications. The methionine oxidation\",\n",
            "          \"were specified as fixed modifications. The methionine oxidation (+\\u200915.9949) was specified as variable modifications. The search was restricted to tryptic peptides, allowing up to two missed cleavage sites. All the other parameters were set as default.\",\n",
            "          \"cleavage sites. All the other parameters were set as default. Quantification was based on a similar Unique\\u2009+\\u2009Razor peptide approach as described in our previous studies. The search results were then filtered by controlling the final protein-level FDR\",\n",
            "          \"were then filtered by controlling the final protein-level FDR to\\u2009<\\u20091%. PSMs from all TMT sets were utilized when assigning peptides to protein groups. TMT corrections were applied for the accurate PSM-level quantification. PSMs that passed all filtering\",\n",
            "          \"PSM-level quantification. PSMs that passed all filtering criteria were then rolled up to log2 ratio- and abundance-level expression matrices and all samples were then median normalized. Comparison of altered proteins in different sampling groups Proteomic\",\n",
            "          \"of altered proteins in different sampling groups Proteomic data generated in tumor and NAT samples from three sampling methods (bulk, coring, and LMD) were analyzed by OmicsOne. The fold change of the log2 value of absolute abundances were compared\",\n",
            "          \"change of the log2 value of absolute abundances were compared between samples. Due to the limited number of samples, proteins with fold changes\\u2009\\u2265\\u20092.0 were considered as altered proteins. Significantly up- and down-regulated proteins were determined if the\",\n",
            "          \"up- and down-regulated proteins were determined if the fold change\\u2009\\u2265\\u20092.0 and adjusted p values\\u2009<\\u20090.05 via Benjamini\\u2013Hochberg approach. The principal component analysis (PCA) was also utilized to evaluate the performance of three sampling methods for\",\n",
            "          \"to evaluate the performance of three sampling methods for differentiating between tumors and NATs. Missing values were not used in the PCA analysis. DNA extraction and genomic analysis DNA was isolated and subjected to Whole Exome Sequencing (WES) as\",\n",
            "          \"was isolated and subjected to Whole Exome Sequencing (WES) as previously described. Eight cases of tissues prepared by bulk, cored, and LMD PDAC underwent WES. Somatic mutations were called by the Somaticwrapper pipeline v1.6\",\n",
            "          \"mutations were called by the Somaticwrapper pipeline v1.6 (https://github.com/ding-lab/somaticwrapper), which included four different callers, i.e., Strelka v.2, MUTECT v1.7, VarScan v.2.3.8, and Pindel v.0.2.5from WES. Exonic SNVs was called by any two\",\n",
            "          \"and Pindel v.0.2.5from WES. Exonic SNVs was called by any two callers among MUTECT v1.7, VarScan v.2.3.8, and Strelka v.2; and indels was called by any two callers among VarScan v.2.3.8, Strelka v.2, and Pindel v.0.2.5. 14X and 8X coverage cutoff were\",\n",
            "          \"v.2, and Pindel v.0.2.5. 14X and 8X coverage cutoff were applied for merged SNVs and indels in tumor and NAT, respectively. SNVs and indels were filtered by a minimal variant allele frequency (VAF) of 0.05 in tumors and a maximal VAF of 0.02 in NAT\",\n",
            "          \"(VAF) of 0.05 in tumors and a maximal VAF of 0.02 in NAT samples. Any SNV within 10\\u00a0bp of an indel in the same tumor sample was filtered. The percent of VAFs of KRAS mutation were calculated and compared among bulk, LMD and coring WES. Results\",\n",
            "          \"calculated and compared among bulk, LMD and coring WES. Results Morphological features of analyzed tumors and normal adjacent tissues Schematic diagram of the workflow. A Three cases of treatment na\\u00efve PDAC and tumor-matched NATs were prospectively\",\n",
            "          \"treatment na\\u00efve PDAC and tumor-matched NATs were prospectively collected. The tumor area and tumor-matched NAT were marked and matched to corresponding tissue blocks. All tissue blocks were sampled using three techniques, including bulk sampling the tumor\",\n",
            "          \"using three techniques, including bulk sampling the tumor (entire section of the block), laser microdissection (LMD) of selected area, and coring of the selected area. B Morphological features of study cases. C3L-01032 demonstrated over 80% of tumor\",\n",
            "          \"of study cases. C3L-01032 demonstrated over 80% of tumor cellularity with a minimal amount of desmoplastic stroma. The case C3L-01158 revealed less than 10% of tumor cellularity. In the case C3L-01160, the tumor demonstrated an approximate 40% of tumor\",\n",
            "          \"C3L-01160, the tumor demonstrated an approximate 40% of tumor cellularity. In addition, the NAT samples also represented a wide range of pancreatic tissue, including predominately acinar cells, benign ductal structures (i.e. C3L-01032) and a low-grade\",\n",
            "          \"benign ductal structures (i.e. C3L-01032) and a low-grade pancreatic intraepithelial neoplasia (i.e. C3L-01160) In the proteomic analysis, a total of six tissue blocks (three pairs of PDAC tissues and tumor-matched NATs) were sampled using three\",\n",
            "          \"PDAC tissues and tumor-matched NATs) were sampled using three techniques, including bulk, LMD, and coring. Overall, tumor and NAT tissues were analyzed using the workflow described in Fig.\\u00a01A: (1) identifying representative areas of ductal carcinoma in\",\n",
            "          \"(1) identifying representative areas of ductal carcinoma in tumor blocks and normal tissues in NATs blocks on the H&E stained slides, and matching targeted area to the tissue blocks, (2) collecting three types of samples from the same tissue block using\",\n",
            "          \"three types of samples from the same tissue block using three approaches (bulk sectioning, LMD dissecting, and coring the targeted area), (3) characterizing three types of samples using proteomic and genomic analyses. The pathological features of the\",\n",
            "          \"and genomic analyses. The pathological features of the cancers represented a spectrum of commonly seen morphology in PDACs (Fig.\\u00a01B). Based on the histomorphological review of tumor images, C3L-01032 had over 80% neoplastic cellularity with a minimal\",\n",
            "          \"C3L-01032 had over 80% neoplastic cellularity with a minimal amount of desmoplastic stroma, C3L-01160 had approximately 40% neoplastic cellularity with abundant desmoplastic stroma and scattered inflammatory cells, and C3L-01158 had less than 10%\",\n",
            "          \"scattered inflammatory cells, and C3L-01158 had less than 10% neoplastic cellularity with significant and dense desmoplastic stroma and foci of tumor necrosis. In the five additional tumors used for genomic analysis, their neoplastic cellularity ranged\",\n",
            "          \"used for genomic analysis, their neoplastic cellularity ranged from 10 to 30% (Additional file 1: Table S1). Finally, the NAT samples represented a wide range of pancreatic tissue, including predominately acinar cells, benign ductal structures (i.e.\",\n",
            "          \"predominately acinar cells, benign ductal structures (i.e. C3L-01032) and pancreatic intraepithelial neoplasia (i.e. C3L-01160). Proteomic analysis in bulk, LMD, and coring tissue samples Correlation coefficient of quantified proteins in individual cases.\",\n",
            "          \"coefficient of quantified proteins in individual cases. A Pearson correlation coefficients of individual case in three types of samples and TMT sets. B Comparison of quantified proteins in bulk, coring and LMD samples Proteomic analyses of samples from\",\n",
            "          \"bulk, coring and LMD samples Proteomic analyses of samples from three sampling groups were characterized using TMT-labeling-based proteomics. In each tumor and/or NAT, over 8500 proteins were quantified (Additional file 1: Table S2). To evaluate the\",\n",
            "          \"were quantified (Additional file 1: Table S2). To evaluate the impact of three sampling techniques on proteomics, we compared protein abundances quantified from each tumor with NAT samples. Pearson correction coefficients of proteins from each tumor and\",\n",
            "          \"Pearson correction coefficients of proteins from each tumor and NAT sample obtained from three sampling techniques ranged from 0.83 to 0.98 among six samples (three tumors and three NATs) (Fig.\\u00a02A). In the analysis of the same tissue obtained by different\",\n",
            "          \"In the analysis of the same tissue obtained by different sampling techniques, the best correlation was found between bulk and cored samples, whereas the cored and LMD came as the second, and bulk and LMD samples showed the least correlation (Fig.\\u00a02B). In\",\n",
            "          \"bulk and LMD samples showed the least correlation (Fig.\\u00a02B). In the pair-wise analysis of tumor or NAT tissues, tumor tissues showed higher correlation than NATs regardless which sampling technique was used to obtain the tissue sample (Fig.\\u00a02A, B). Taken\",\n",
            "          \"was used to obtain the tissue sample (Fig.\\u00a02A, B). Taken together, the LMD samples had lower correlation with bulk samples than did the cored samples, indicating LMD technique might have a stronger impact on the proteome composition compared to the coring\",\n",
            "          \"impact on the proteome composition compared to the coring technique. Protein expressional patterns in bulk, LMD, and coring samples Principal component analysis and differential analysis of proteomic data from tumors and NATs from three sampling methods.\",\n",
            "          \"data from tumors and NATs from three sampling methods. A Analysis of tumors and NATs in bulk samples. B Analysis of tumors and NATs in coring samples. C Analysis of tumors and NATs in LMD samples, D Volcano plot of differentially expressed proteins in\",\n",
            "          \"samples, D Volcano plot of differentially expressed proteins in bulk samples, E Volcano plot of differentially expressed proteins in coring samples; F Volcano plot of differentially expressed proteins in LMD samples Based upon the above quantified\",\n",
            "          \"proteins in LMD samples Based upon the above quantified proteins, we investigated the protein expression patterns in the three sampling groups. PCA was performed and illustrated distinctive patterns between tumors and NATs (Fig.\\u00a03). All three sampling\",\n",
            "          \"patterns between tumors and NATs (Fig.\\u00a03). All three sampling methods produced profiles that were separable from NATs (Fig.\\u00a03A\\u2013C). A pair-wise comparison of the proteomic profile of tumor and NAT in the three sampling groups was conducted. In the\",\n",
            "          \"and NAT in the three sampling groups was conducted. In the analysis, the proteins considered differentially abundant and sampling-technique-associated proteins were those proteins with a log2 fold change\\u2009\\u2265\\u20092 and adjusted p-value\\u2009<\\u20090.05. In the bulk\",\n",
            "          \"a log2 fold change\\u2009\\u2265\\u20092 and adjusted p-value\\u2009<\\u20090.05. In the bulk tumors, 811 proteins were significantly up-regulated, and 742 proteins were significantly down-regulated in the tumors compared to NAT (Fig.\\u00a03D). In the cored tumor samples, 443 proteins were\",\n",
            "          \"to NAT (Fig.\\u00a03D). In the cored tumor samples, 443 proteins were significantly up-regulated, and 368 proteins were significantly down-regulated in the tumors compared to NAT (Fig.\\u00a03E). In the LMD tumor samples, 305 proteins were significantly up-regulated,\",\n",
            "          \"tumor samples, 305 proteins were significantly up-regulated, and 345 proteins were significantly down-regulated in the tumors compared to NAT (Fig.\\u00a03F). Identifications of changes of proteins in tumors sampled from three sampling methods. A Up-regulated\",\n",
            "          \"in tumors sampled from three sampling methods. A Up-regulated proteins in tumors from the three sampling methods. B Down-regulated proteins in tumors from the three sampling methods To evaluate the proteomic profile of tumor-associated proteins in\",\n",
            "          \"evaluate the proteomic profile of tumor-associated proteins in different sample types, we compared significantly up- and down-regulated proteins in tumor samples with NATs (Fig.\\u00a04). Among up-regulated proteins, 115 and 96 tumor-associated proteins were\",\n",
            "          \"proteins, 115 and 96 tumor-associated proteins were uniquely identified in LMD and cored samples, which were not up-regulated in bulk samples. Only two proteins (CHMP1A and CHMP1B), members of the Endosomal Sorting Complex Required for Transport\\u2013III\",\n",
            "          \"of the Endosomal Sorting Complex Required for Transport\\u2013III (ESCRT-III) family, showed a consistent elevation in all three sampling groups (Fig.\\u00a04A). Among down-regulated proteins, 52 and 92 tumor-associated proteins were uniquely identified in LMD and\",\n",
            "          \"tumor-associated proteins were uniquely identified in LMD and cored samples, which were not seen in bulk samples. Five proteins, including CEL, CLPS, CPA2, CPB1 and CTRC, showed a consistent down-regulated pattern in all three sampling groups (Fig.\\u00a04B).\",\n",
            "          \"down-regulated pattern in all three sampling groups (Fig.\\u00a04B). To further evaluate the potential utility of cored and LMD for the enrichment of tumor cells, we compared proteomic signatures of bulk, cored, and LMD samples with a combined Pancreatic Cancer\",\n",
            "          \"bulk, cored, and LMD samples with a combined Pancreatic Cancer Database, including 2796 gene names of potential PDAC biomarkers. The database was used in our previous study. The same two CHMP proteins were recorded in the Pancreatic Cancer Database\",\n",
            "          \"CHMP proteins were recorded in the Pancreatic Cancer Database (Fig.\\u00a04A). For tumor-associated proteins identified from the cored and bulk samples, 44 proteins were found in the Database, and 17 of them were commonly identified from bulk and coring\",\n",
            "          \"and 17 of them were commonly identified from bulk and coring techniques. Of tumor-elevated proteins identified in LMD sampling, 4 additional tumor-associated proteins, COL17A1, VSNL1, LYPD3, and VCAN, were reported in the Pancreatic Cancer Database\",\n",
            "          \"and VCAN, were reported in the Pancreatic Cancer Database (Fig.\\u00a04A). In all, these data showed that distinct tumor-associated proteins were observed in samples obtained by different sampling techniques; However, findings might be limited by the small\",\n",
            "          \"techniques; However, findings might be limited by the small sample size and need to be further investigated in a large-scale study. Genomic-sequencing of KRAS mutations Based upon the similarity of global proteomic data in bulk and cored samples and the\",\n",
            "          \"of global proteomic data in bulk and cored samples and the distinct proteomic profiles in LMD samples (Fig.\\u00a02), we further evaluated the enrichment of neoplastic cellularity using the percent of variant allelic frequency (VAF) of KRAS mutations derived\",\n",
            "          \"of variant allelic frequency (VAF) of KRAS mutations derived from WES as a surrogate signature for neoplastic cellularity for tumor tissues prepared by bulk, coring, or LMD. A total of eight PDAC tumors were included in the genomic analysis, including\",\n",
            "          \"PDAC tumors were included in the genomic analysis, including above three tumors and additional five PDACs. Of these additional five PDACs, the tumor cellularity ranged from 10 to 30% based on the pathology review of the digital histology images. The\",\n",
            "          \"on the pathology review of the digital histology images. The percent of variant allele frequencies (VAF%) of KRAS mutations in tumor tissues prepared by bulk, coring, and LMD techniques Case\\tVAF (bulk)%\\tVAF (core)%\\tVAF (LMD)%\\tFold changes (core/bulk)\\tFold\",\n",
            "          \"(bulk)%\\tVAF (core)%\\tVAF (LMD)%\\tFold changes (core/bulk)\\tFold changes (LMD/bulk)\\tKRAS mutations\\t \\tC3L-01032\\t34.2\\t48.1\\t45.9\\t1.4\\t1.3\\tp.G12D\\t \\tC3L-01160\\t29.5\\t51.8\\t30.6\\t1.8\\t1.0\\tp.G12D\\t \\tC3N-01897\\t18.5\\t20.3\\t26.7\\t1.1\\t1.4\\tp.G12R\",\n",
            "          \"C3N-01897\\t18.5\\t20.3\\t26.7\\t1.1\\t1.4\\tp.G12R\\t \\tC3N-02997\\t15.1\\t9.4\\t20.0\\t0.6\\t1.3\\tp.G12V\\t \\tC3N-02591\\t10.3\\t9.1\\t21.2\\t0.9\\t2.1\\tp.G12R\\t \\tC3N-02590\\t7.7\\t12.2\\t33.6\\t1.6\\t4.4\\tp.G12V\\t \\tC3L-01158\\t2.7\\t4.4\\t17.3\\t1.6\\t6.4\\tp.G12R\\t \\tC3L-03624\\t0.8\\t0.3\\t6.0\\t0.4\\t7.6\\tp.Q61H\\t \\t The fold\",\n",
            "          \"C3L-03624\\t0.8\\t0.3\\t6.0\\t0.4\\t7.6\\tp.Q61H\\t \\t The fold changes of VAFs of coring and LMD compared to VAF of the matched bulk tumors were included to show the neoplastic cellularity enrichment efficiencies by the coring and LMD methods In the analysis, we\",\n",
            "          \"efficiencies by the coring and LMD methods In the analysis, we determined and compared the VAFs of the KRAS mutations in tumor samples obtained by bulk, coring, and LMD techniques (Table 1). Two of three PDACs used in our proteomic analysis contained a\",\n",
            "          \"Two of three PDACs used in our proteomic analysis contained a relatively high neoplastic cellularity. In C3L-01032, KRAS percent of VAF were 34.2% in bulk and 48.1% in cored, and 45.9% in LMD samples, respectively, representing 68.4%, 96.2%, and 91.8% of\",\n",
            "          \"samples, respectively, representing 68.4%, 96.2%, and 91.8% of neoplastic cellularity in the bulk, cored, and LMD samples, respectively. A similar pattern was observed in C3L-01160, with KRAS VAF scores of 29.5% in bulk 51.8% in cored, and 30.6% in LMD\",\n",
            "          \"VAF scores of 29.5% in bulk 51.8% in cored, and 30.6% in LMD tumor tissues, respectively, representing 59.0%, 100.0%, and 61.2% neoplastic cellularity in the bulk, cored, and LMD samples. In the third PDAC used in proteomic analysis, C3L-01158, percent of\",\n",
            "          \"third PDAC used in proteomic analysis, C3L-01158, percent of KRAS VAF of 2.7% in bulk, 4.4% in cored, and 17.3% in LMD tumor tissues were observed, respectively, representing 5.4%, 8.8%, and 34.6% neoplastic cellularity in the bulk, cored, and LMD tumors\",\n",
            "          \"34.6% neoplastic cellularity in the bulk, cored, and LMD tumors (Table 1). In additional five tumor cases, KRAS VAFs ranged from 0.8% to 18.5% in bulk samples, 0.3% to 20.3% in cored samples, and 6.0% to 26.7% in LMD tumor samples were observed. The\",\n",
            "          \"and 6.0% to 26.7% in LMD tumor samples were observed. The variant allele frequencies (VAFs) of KRAS mutations in tumors prepared by bulk, coring, and LMD. A The scatterplot of percent of VAF for KRAS mutations based on the paired bulk and coring WES of 8\",\n",
            "          \"for KRAS mutations based on the paired bulk and coring WES of 8 PDAC samples. B The scatterplot of VAFs of KRAS mutations based on the paired bulk and LMD of 8 PDAC samples. C The scatterplot of VAFs of KRAS mutations based on the paired cores and LMD of\",\n",
            "          \"of VAFs of KRAS mutations based on the paired cores and LMD of 8 PDAC samples. D The percent of VAF for KRAS mutations in 8 cases of PDAC tumors prepared by bulk, coring, and LMD methods. E The fold changes of VAFs from tumors prepared by coring or LMD\",\n",
            "          \"The fold changes of VAFs from tumors prepared by coring or LMD comparing to the original VAFs from the bulk tumors The scatterplot of KRAS VAFs between tumor samples prepared by bulk, coring, and LMD showed that cored tumor tissues were most similar to\",\n",
            "          \"and LMD showed that cored tumor tissues were most similar to the bulk tumor (R\\u2009=\\u20090.94, Fig.\\u00a05A), while tumor tissues prepared by LMD showed less similarity to bulk (R\\u2009=\\u20090.73, Fig.\\u00a05B) or cored tissues (R\\u2009=\\u20090.75, Fig.\\u00a05C). These results are consistent with\",\n",
            "          \"tissues (R\\u2009=\\u20090.75, Fig.\\u00a05C). These results are consistent with what we observed from the proteomic data, which showed a similar proteomic pattern between bulk and cored tissues while LMD showed distinct proteomic patterns (Fig.\\u00a02). The VAFs of bulk,\",\n",
            "          \"showed distinct proteomic patterns (Fig.\\u00a02). The VAFs of bulk, cored, and LMD tumors showed that tumor tissues prepared by LMD method contained significantly higher neoplastic tumor cellularity ranging from 12.0% to 91.8% comparing to tumor tissues\",\n",
            "          \"ranging from 12.0% to 91.8% comparing to tumor tissues prepared by bulk or coring methods (Table 1, Fig.\\u00a05D). We further investigated the enrichment of neoplastic cellularity by coring and LMD methods using fold changes of KRAS mutations and found that\",\n",
            "          \"LMD methods using fold changes of KRAS mutations and found that LMD significantly enriched neoplastic cellularity. The enrichment of neoplastic cellularity by LMD is more effective for low cellularity bulk tumor tissues (Fig.\\u00a05E). The coring method showed\",\n",
            "          \"bulk tumor tissues (Fig.\\u00a05E). The coring method showed some enrichment for neoplastic cellularity when the bulk tumors contained high neoplastic cellularity, but failed to enrich the neoplastic cells when the original bulk tumors contained low neoplastic\",\n",
            "          \"cells when the original bulk tumors contained low neoplastic cellularity (Fig.\\u00a05E). Discussion The isolation of neoplastic cells plays a critical role in proteogenomic studies. As reported in the CPTAC PDAC proteogenomics and by others, PDAC tumor tissue\",\n",
            "          \"the CPTAC PDAC proteogenomics and by others, PDAC tumor tissue is notorious for its highly variable tumor cellularity. The heterogeneous character of PDAC may obfuscate the study of tumor-associated proteins. The enrichment of neoplastic cellularity is\",\n",
            "          \"proteins. The enrichment of neoplastic cellularity is the critical initial step in the proteogenomic profiling. Knowledge of the effect and differences of sampling techniques may allow us to refine proteomic profiles and, more importantly, to understand\",\n",
            "          \"refine proteomic profiles and, more importantly, to understand better the nature of PDAC for the early detection, monitoring the disease progression and response to therapy. The purpose of this study was to characterize PDAC tissues isolated from bulk,\",\n",
            "          \"this study was to characterize PDAC tissues isolated from bulk, coring, and LMD using proteomics and genomics; and to evaluate the potential utility of coring and LMD in the enrichment of neoplastic cellularity. In our study, we included cases\",\n",
            "          \"of neoplastic cellularity. In our study, we included cases representing a spectrum of pathological features commonly seen in PDACs, with the neoplastic cellularity ranging from\\u2009<\\u200910% to\\u2009>\\u200980%. In addition, we also profiled pancreatic NAT components to\",\n",
            "          \"In addition, we also profiled pancreatic NAT components to include acinar and ductal cells. In the global proteomic analysis, over 8500 proteins were identified in each sample regardless of sampling techniques. Tumor samples were distinguished from NATs\",\n",
            "          \"sampling techniques. Tumor samples were distinguished from NATs by proteomic data obtained from all three sampling methods. These demonstrated that tumor-associated proteins can be determined using samples obtained by all three sampling techniques. The\",\n",
            "          \"using samples obtained by all three sampling techniques. The finding of comparative protein profiles of coring and bulk tissues demonstrated that cored and bulk tissues presented similar proteomic profiles. Our finding is critical for the analysis of\",\n",
            "          \"proteomic profiles. Our finding is critical for the analysis of clinical tumor samples, since the majority of PDAC patients present as advanced diseases, in whom a surgical resected bulk tumor sample cannot be obtained; as such, a tumor tissue can often\",\n",
            "          \"sample cannot be obtained; as such, a tumor tissue can often be obtained by a biopsy procedure (similar to coring samples) for the characterization of the tumor. To further evaluate tumor-associated proteins, we compared significantly up- and\",\n",
            "          \"tumor-associated proteins, we compared significantly up- and down-regulated proteins in each tumor sample using three sampling methods. Five pancreatic enzymes CEL, CLPS, CPA2, CPB1 and CTC, showed a consistent down-regulation in all three sampling\",\n",
            "          \"CTC, showed a consistent down-regulation in all three sampling methods. This makes sense as these proteins are strongly associated with non-neoplastic acinar cells. Among up-regulated proteins, only two proteins CHMP1A and CHMP1B showed a consistent\",\n",
            "          \"only two proteins CHMP1A and CHMP1B showed a consistent elevation in all three sampling methods. These findings suggest that sampling methods used to enrich neoplastic cells can impact the proteomic findings. Both CHMP1A and CHMP1B are members of the\",\n",
            "          \"proteomic findings. Both CHMP1A and CHMP1B are members of the Endosomal Sorting Complex Required for Transport\\u2013III (ESCRT-III) family, and play critical role in transporting membrane-associated proteins such as receptor proteins into lysosome for\",\n",
            "          \"proteins such as receptor proteins into lysosome for degradation via the formation and sorting of multivesicular bodies. In addition to their transporting/sorting function, a tumor suppression role of CHMP1A in kidney and pancreatic cancers has also been\",\n",
            "          \"role of CHMP1A in kidney and pancreatic cancers has also been suggested. In an earlier study by Li et al., the tumor suppressor role of CHMP1A was indicated by growth inhibition of PanC-1 cells and conversion of non-tumorigenic human embryonic kidney\",\n",
            "          \"cells and conversion of non-tumorigenic human embryonic kidney cells to cells capable of forming xenograft tumors in athymic mice by stable knock-down of Chmp1A. In the same study, authors also examined the expression of CHMP1A in human pancreatic tumors\",\n",
            "          \"examined the expression of CHMP1A in human pancreatic tumors by immunochemical labeling of human pancreatic tumor TMAs, including 10 cases of pancreatic ductal carcinoma (PDACs). They found variable expressional patterns of CHMP1A in human pancreatic\",\n",
            "          \"variable expressional patterns of CHMP1A in human pancreatic tumors, including a prominent expression of CHMP1A in PDACs. They also found that the CHMP1A protein showed a diffuse labeling pattern in PDAC cells compared with an apical labeling pattern in\",\n",
            "          \"in PDAC cells compared with an apical labeling pattern in normal ductal epithelium. The IHC data demonstrated an obvious but mis-localized labeling pattern of CHMP1A in human PDAC. In our study, we used fresh frozen PDAC tumor tissue and identified CHMP1A\",\n",
            "          \"we used fresh frozen PDAC tumor tissue and identified CHMP1A protein to be up-regulated in PDACs in the comparison with NAT. Our proteomic finding is similar to the IHC study by Li et al., that CHMP1A was detectable and elevated in human PDAC.\",\n",
            "          \"et al., that CHMP1A was detectable and elevated in human PDAC. Furthermore, studies have also demonstrated that CHMP1A is the only protein in ESCRT superfamily that contains a nuclear localization signal (NLS) at its N-terminus. To further study the\",\n",
            "          \"signal (NLS) at its N-terminus. To further study the potential function of CHMP1A, Manohar et al. transformed PanC-1 cells with different truncated forms of Chmp1A, and found distinctive functional roles of the proteins. The overexpression of NLS-deleted\",\n",
            "          \"roles of the proteins. The overexpression of NLS-deleted Chmp1A could promote the tumor cell growth, whereas, overexpression of C-terminal deleted Chmp1A could inhibit tumor growth. Taken together, the tumor suppressor function of CHMP1A in cancers has\",\n",
            "          \"the tumor suppressor function of CHMP1A in cancers has been demonstrated in cell lines and animal models, also linked to the aberrant isoform of Chmp1a. However, the tumor suppressor role of CHMP1A in human cancer is still not well-studied, yet. Further\",\n",
            "          \"CHMP1A in human cancer is still not well-studied, yet. Further evidence of its tumor suppressor role is needed. To further understand the potential effect of coring and LMD on the enriched tumor-associated proteins, we compared tumor-associated proteomic\",\n",
            "          \"proteins, we compared tumor-associated proteomic signatures identified in bulk, coring, and LMD samples with the combined Pancreatic Cancer Database constructed from two studies, including 2796 gene names of potential PDAC biomarkers as well as several\",\n",
            "          \"2796 gene names of potential PDAC biomarkers as well as several highly robust biomarkers for the early detection and tumor progression. The database has been used in our recent proteogenomic study of 140 PDACs. In current study, we found the two commonly\",\n",
            "          \"study of 140 PDACs. In current study, we found the two commonly identified proteins, CHMP1A and CHMP1B, were tumor-associated proteins in the Pancreatic Cancer Database. Of 115 tumor-elevated proteins identified in LMD sampling, 4 additional\",\n",
            "          \"proteins identified in LMD sampling, 4 additional tumor-associated proteins, COL17A1, VSNL1, LYPD3, and VCAN, were reported in the Pancreatic Cancer Database. For tumor-associated proteins identified from the comparison of coring and bulk samples, 44\",\n",
            "          \"identified from the comparison of coring and bulk samples, 44 proteins were found in the Database, and 17 of them were to be commonly identified from bulk and coring techniques. These data demonstrated that abundances of tumor-associated proteins from\",\n",
            "          \"demonstrated that abundances of tumor-associated proteins from cored and LMD methods might have distinctive enrichment patterns, suggestive of the impact of sampling techniques on the quantitative measurement of enriched tumor-associated proteins.\",\n",
            "          \"quantitative measurement of enriched tumor-associated proteins. However, our observation has several limitations, including a small number of study cases and suboptimal material in LMD samples. The unique feature of LMD samples is suggestive of a\",\n",
            "          \"samples. The unique feature of LMD samples is suggestive of a LMD-specific proteome and warrants further investigations. Maurer et al. performed in-depth proteogenomic analysis of PDAC using LMD samples and compared the profile with that of bulk tissue.\",\n",
            "          \"LMD samples and compared the profile with that of bulk tissue. In that study, they found unique signatures, and provided evidence of the differential expression and pathways in epithelial cell subtypes. Le Large et al., also identified the unique\",\n",
            "          \"cell subtypes. Le Large et al., also identified the unique proteomic features of laser microdissected PDAC samples in comparison with bulk tissues. They enriched tumor area and tumor-matched stroma using laser-capture microdissection and compared\",\n",
            "          \"stroma using laser-capture microdissection and compared proteomic findings with the bulk tumor samples. They were able to identify 2 tyrosine kinase inhibitors targetable receptor tyrosine kinases (RTKs) in the tumor cell compartment obtained by\",\n",
            "          \"kinases (RTKs) in the tumor cell compartment obtained by laser-capture microdissection, including a previously known epidermal growth factor receptor 1 (EGFR) and a novel ephrin type-A receptor 2 (EPHA2). These proteins were not found in proteomes of\",\n",
            "          \"2 (EPHA2). These proteins were not found in proteomes of stroma and bulk tumor samples, suggestive of tumor compartment-specific pathways. These studies indicate that laser-capture microdissection approaches can be used to enrich a desired area\",\n",
            "          \"microdissection approaches can be used to enrich a desired area (compartment-specific areas) for highly selective proteomics. However, the laser-based microdissection technique requires an extended tissue isolation procedure and prolonged separation time,\",\n",
            "          \"tissue isolation procedure and prolonged separation time, which has a potential impact on the proteome and protein post-translational modifications (PTMs) in tumor-associated proteins and for subsequent multi-omics studies. The impact of LMD procedure\",\n",
            "          \"for subsequent multi-omics studies. The impact of LMD procedure requires further investigation. Nevertheless, LMD is an attractive approach for the LC-MS\\u2013based study of the composition of compartment-specific proteome within the tumor tissue. To further\",\n",
            "          \"proteome within the tumor tissue. To further evaluate the enrichment for neoplastic cells, we analyzed KRAS mutations in the tumor samples. KRAS alterations occur in\\u2009>\\u200990% of PDACs and mutations in KRAS therefore can be useful in estimating neoplastic\",\n",
            "          \"in KRAS therefore can be useful in estimating neoplastic purity. We estimated the score of KRAS VAF as a surrogate of constituent tumor purity in samples. The increased scores of KRAS VAF in cored samples from C3L-0132 and C3L-01160 indicate that an\",\n",
            "          \"in cored samples from C3L-0132 and C3L-01160 indicate that an enrichment of neoplastic purity near 100% can be achieved from bulk tumors containing high neoplastic cellularity by the coring technique, whereas, the KRAS VAF remained low in the other cored\",\n",
            "          \"whereas, the KRAS VAF remained low in the other cored tumor samples from bulk tumors containing lower neoplastic cellularity. The genomic analysis of KRAS mutations revealed that tumor tissues prepared by LMD technique significantly enriched neoplastic\",\n",
            "          \"prepared by LMD technique significantly enriched neoplastic cellularity, especially from bulk tumors with low neoplastic cellularity (Fig.\\u00a05E). Similarly to genomic findings, the significant impact of LMD on proteomic profile could due to the enrichment\",\n",
            "          \"impact of LMD on proteomic profile could due to the enrichment of neoplastic cellularity, however, the impacts of LMD technique to proteins could also play a role in the observed proteomic changes, which warrants further investigation. Taken together,\",\n",
            "          \"changes, which warrants further investigation. Taken together, these data suggest that LMD is most effective in enriching neoplastic cellularity from bulk tissues. Coring can enrich neoplastic cells from cancers containing a relatively high neoplastic\",\n",
            "          \"cells from cancers containing a relatively high neoplastic cellularity. However, the coring technique seems to have a limitation for the isolation of neoplastic cells in cases with a low neoplastic cellularity. In our study, we only have small number of\",\n",
            "          \"cellularity. In our study, we only have small number of cases; a further study with a large cohort is necessary to further evaluate the utility of different sampling techniques. Finally, our study is a preliminary work based on a small number of tumor\",\n",
            "          \"study is a preliminary work based on a small number of tumor cases. The aim of our study is not to identify tumor specific proteins or to select the \\u2018best performer\\u2019 from different sampling techniques, but rather to evaluate the impacts of different\",\n",
            "          \"techniques, but rather to evaluate the impacts of different sampling techniques to proteomics and genomics and to provide insights and knowledge of each sampling technique. Although neoplastic cellularity is still an issue in the PDAC for proteogenomic\",\n",
            "          \"cellularity is still an issue in the PDAC for proteogenomic analysis, especially in the study of tumor-associated proteins, single cell analysis techniques are now available to improve homogeneous cell inputs for quantitative proteogenomic measurements.\",\n",
            "          \"cell inputs for quantitative proteogenomic measurements. Conclusions In this study, we used different sampling techniques, including bulk sectioning of whole tumor tissue, LMD, and coring methods, for the proteogenomic analysis of PDAC. The protein\",\n",
            "          \"methods, for the proteogenomic analysis of PDAC. The protein signatures from cored samples revealed a comparative profile with bulk samples, whereas the protein profile from LMD samples demonstrated a unique signature, indicating the potential impact of\",\n",
            "          \"a unique signature, indicating the potential impact of the sampling techniques on the proteomic findings. In the genomic analysis, effective enrichment of neoplastic cellularity revealed by high KRAS VAF scores was found in LMD as well as cored tumor\",\n",
            "          \"by high KRAS VAF scores was found in LMD as well as cored tumor tissues obtained from certain bulk tumor tissues containing different amount of cellularity. Knowledge of the effects and differences of sampling techniques warrant further investigation.\",\n",
            "          \"of sampling techniques warrant further investigation. Supplementary Information Abbreviations ACN Acetonitrile AGC Automatic gain control bRPLC Basic reversed-phase liquid chromatography CHMP Chromatin-modifying protein/charged multivesicular body protein\",\n",
            "          \"Chromatin-modifying protein/charged multivesicular body protein CPTAC Clinical proteomic tumor analysis consortium DDA Data-dependent acquisition DTT Dithiothreitol; FA: formic acid FDR False discovery rate H&E Hematoxylin and eosin HCD Collision\",\n",
            "          \"False discovery rate H&E Hematoxylin and eosin HCD Collision dissociation activation energy HPA Human Proteome Atlas IAA Iodoacetamide iRT Indexed retention time LMD Laser microdissection MeOH Methanol MS Mass spectrometry NAT Normal adjacent tissues NCE\",\n",
            "          \"Methanol MS Mass spectrometry NAT Normal adjacent tissues NCE Normalized collision energy PCA Principal component analysis PDAC Pancreatic adenocarcinoma PSM Peptide spectrum match QC Quality control RT Room temperature SD Standard deviation TCGA The\",\n",
            "          \"control RT Room temperature SD Standard deviation TCGA The Cancer Genome Atlas TCEP Tris (2-carboxyethyl) phosphine TFA Trifluoroacetic acid TMT Tandem mass tag VAF Variant allele frequency WES Whole-exome sequencing Publisher's Note Springer Nature\",\n",
            "          \"WES Whole-exome sequencing Publisher's Note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Author contributions QKL, HZ, YEH, AIR, GM, MM, OFB, RHH, DWC contributed to experimental\",\n",
            "          \"HZ, YEH, AIR, GM, MM, OFB, RHH, DWC contributed to experimental design, organization of data and drafting the manuscript; YEH, YZL, RL, CZ, LD, BZ, GH, CJN, SDJ, QKL, HZ, MS, LC, contributed to data analysis. All authors read and approved the final\",\n",
            "          \"to data analysis. All authors read and approved the final manuscript. Funding This work is supported in part by the National Institutes of Health, National Cancer Institute, Clinical Proteomics Tumor Analysis Consortium (CPTAC, U24CA210985). Availability\",\n",
            "          \"Tumor Analysis Consortium (CPTAC, U24CA210985). Availability of data and materials All data generated and analyzed during this study are included in this published article and its supplementary information files. Pathology and radiology images can be\",\n",
            "          \"information files. Pathology and radiology images can be accessed via Imaging Data Commons (IDC) at https://portal.imaging.datacommons.cancer.gov/explore/filters/?collection_id=cptac_ccrcc and The Cancer Imaging Archive at 10.7937/K9/TCIA.2018.OBLAMN27.\",\n",
            "          \"The Cancer Imaging Archive at 10.7937/K9/TCIA.2018.OBLAMN27. Declarations Ethics approval and consent to participate These criteria are followed by the guidelines of the Cancer Clinical Proteomics Research, National Cancer Institute  Consent for\",\n",
            "          \"Proteomics Research, National Cancer Institute  Consent for publication This manuscript has been read and approved by all the authors to publish and is not submitted or under consideration for publication elsewhere. Competing interests The authors declare\",\n",
            "          \"publication elsewhere. Competing interests The authors declare no competing interests.\"\n",
            "        ]\n",
            "      }\n",
            "    ]\n",
            "  },\n",
            "  \"PDC000291\": {\n",
            "    \"study_id\": \"a2df3cc3-2c40-4da6-ac5a-ff016cce4bbd\",\n",
            "    \"pdc_study_id\": \"PDC000291\",\n",
            "    \"study_submitter_id\": \"TCGA Breast Cancer CompRef Samples - Phosphoproteome\",\n",
            "    \"program_id\": \"10251935-5540-11e8-b664-00a098d917f8\",\n",
            "    \"project_id\": \"c96eca75-8b73-4018-9bf7-b20e31a965cd\",\n",
            "    \"study_name\": \"TCGA Breast Cancer CompRef Samples - Phosphoproteome\",\n",
            "    \"program_name\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"project_name\": \"CPTAC2 Retrospective\",\n",
            "    \"disease_type\": \"Other\",\n",
            "    \"primary_site\": \"Not Reported\",\n",
            "    \"analytical_fraction\": \"Phosphoproteome\",\n",
            "    \"experiment_type\": \"iTRAQ4\",\n",
            "    \"embargo_date\": null,\n",
            "    \"cases_count\": 2,\n",
            "    \"aliquots_count\": 20,\n",
            "    \"protocol_id\": \"e9812283-3c88-4d7a-990a-15402dc4759d\",\n",
            "    \"protocol_submitter_id\": \"TCGA Breast Cancer CompRef Samples Phosphoproteome\",\n",
            "    \"program_submitter_id\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"protocol_name\": \"TCGA Breast Cancer CompRef Samples Phosphoproteome\",\n",
            "    \"protocol_date\": \"2021-04-07\",\n",
            "    \"publications\": [\n",
            "      {\n",
            "        \"publication_id\": \"44e2e766-89c7-11e8-bcf1-0a2705229b82\",\n",
            "        \"pubmed_id\": \"27251275\",\n",
            "        \"doi\": \"10.1038/nature18003\",\n",
            "        \"title\": \"Proteogenomics connects somatic mutations to signalling in breast cancer\",\n",
            "        \"journal\": \"Nature\",\n",
            "        \"journal_url\": \"https://www.nature.com/articles/nature18003\",\n",
            "        \"year\": \"2016\",\n",
            "        \"abstract\": \"Somatic mutations have been extensively characterized in breast cancer, but the effects of these genetic alterations on the proteomic landscape remain poorly understood. Here we describe quantitative mass-spectrometry-based proteomic and phosphoproteomic analyses of 105 genomically annotated breast cancers, of which 77 provided high-quality data. Integrated analyses provided insights into the somatic cancer genome including the consequences of chromosomal loss, such as the 5q deletion characteristic of basal-like breast cancer. Interrogation of the 5q trans-effects against the Library of Integrated Network-based Cellular Signatures, connected loss of CETN3 and SKP1 to elevated expression of epidermal growth factor receptor (EGFR), and SKP1 loss also to increased SRC tyrosine kinase. Global proteomic data confirmed a stromal-enriched group of proteins in addition to basal and luminal clusters, and pathway analysis of the phosphoproteome identified a G-protein-coupled receptor cluster that was not readily identified at the mRNA level. In addition to ERBB2, other amplicon-associated highly phosphorylated kinases were identified, including CDK12, PAK1, PTK2, RIPK2 and TLK2. We demonstrate that proteogenomic analysis of breast cancer elucidates the functional consequences of somatic mutations, narrows candidate nominations for driver genes within large deletions and amplified regions, and identifies therapeutic targets.\",\n",
            "        \"full_text\": [\n",
            "          \"Proteogenomics connects somatic mutations to signaling in breast cancer Summary Somatic mutations have been extensively characterized in breast cancer, but the effects of these genetic alterations on the proteomic landscape remain poorly understood. We\",\n",
            "          \"on the proteomic landscape remain poorly understood. We describe quantitative mass spectrometry-based proteomic and phosphoproteomic analyses of 105 genomically annotated breast cancers of which 77 provided high-quality data. Integrated analyses allowed\",\n",
            "          \"77 provided high-quality data. Integrated analyses allowed insights into the somatic cancer genome including the consequences of chromosomal loss, such as the 5q deletion characteristic of basal-like breast cancer. The 5q trans effects were interrogated\",\n",
            "          \"breast cancer. The 5q trans effects were interrogated against the Library of Integrated Network-based Cellular Signatures, thereby connecting CETN3 and SKP1 loss to elevated expression of EGFR, and SKP1 loss also to increased SRC. Global proteomic data\",\n",
            "          \"and SKP1 loss also to increased SRC. Global proteomic data confirmed a stromal-enriched group in addition to basal and luminal clusters and pathway analysis of the phosphoproteome identified a G Protein-coupled receptor cluster that was not readily\",\n",
            "          \"a G Protein-coupled receptor cluster that was not readily identified at the mRNA level. Besides ERBB2, other amplicon-associated, highly phosphorylated kinases were identified, including CDK12, PAK1, PTK2, RIPK2 and TLK2. We demonstrate that proteogenomic\",\n",
            "          \"PAK1, PTK2, RIPK2 and TLK2. We demonstrate that proteogenomic analysis of breast cancer elucidates functional consequences of somatic mutations, narrows candidate nominations for driver genes within large deletions and amplified regions, and identifies\",\n",
            "          \"within large deletions and amplified regions, and identifies therapeutic targets. Introduction A central deficiency in our knowledge of cancer concerns how genomic changes drive the proteome and phosphoproteome to execute phenotypic characteristics. The\",\n",
            "          \"and phosphoproteome to execute phenotypic characteristics. The initial proteomic characterization in the TCGA breast study was performed using reversed phase protein arrays; however this approach is restricted by antibody availability. To provide greater\",\n",
            "          \"is restricted by antibody availability. To provide greater analytical breadth, the NCI Clinical Proteomic Tumor Analysis Consortium (CPTAC) is analyzing the proteomes of genome-annotated TCGA tumor specimens using mass spectrometry. Herein we describe\",\n",
            "          \"tumor specimens using mass spectrometry. Herein we describe integrated proteogenomic analyses of TCGA breast cancer samples representing the four principal mRNA-defined breast cancer intrinsic subtypes. Proteogenomic analysis of TCGA samples 105 breast\",\n",
            "          \"subtypes. Proteogenomic analysis of TCGA samples 105 breast tumors previously characterized by the TCGA were selected for proteomic analysis after histopathological documentation (Supplementary Tables 1 and 2). The cohort included a balanced\",\n",
            "          \"(Supplementary Tables 1 and 2). The cohort included a balanced representation of PAM50-defined intrinsic subtypes including 25 basal-like, 29 luminal A, 33 luminal B, and 18 HER2 (ERBB2)-enriched tumors, along with 3 normal breast tissue samples. Samples\",\n",
            "          \"tumors, along with 3 normal breast tissue samples. Samples were analyzed by high-resolution, accurate mass, tandem mass spectrometry (MS) that included extensive peptide fractionation and phosphopeptide enrichment (Extended Data Fig. 1a). An isobaric\",\n",
            "          \"phosphopeptide enrichment (Extended Data Fig. 1a). An isobaric peptide labeling approach (iTRAQ) was employed to quantify protein and phosphosite levels across samples, with 37 iTRAQ 4-plexes analyzed in total. A total of 15,369 proteins (12,405 genes)\",\n",
            "          \"analyzed in total. A total of 15,369 proteins (12,405 genes) and 62,679 phosphosites were confidently identified with 11,632 proteins/tumor and 26,310 phosphosites/tumor on average (Supplementary Tables 3, 4 and Supplementary Methods). After filtering for\",\n",
            "          \"Tables 3, 4 and Supplementary Methods). After filtering for observation in at least a quarter of the samples (Supplementary Methods, Extended Data Fig. 1b) 12,553 proteins (10,062 genes) and 33,239 phosphosites, with their relative abundances quantified\",\n",
            "          \"33,239 phosphosites, with their relative abundances quantified across tumors, were used in subsequent analyses in this study. Stable longitudinal performance and low technical noise were demonstrated by repeated, interspersed analyses of a single batch of\",\n",
            "          \"by repeated, interspersed analyses of a single batch of patient-derived luminal and basal breast cancer xenograft samples (Extended Data Fig. 1d,e). Due to the heterogeneous nature of breast tumors, and because proteomic analyses were performed on tumor\",\n",
            "          \"tumors, and because proteomic analyses were performed on tumor fragments that were different from those used in the genomic analyses, rigorous pre-specified sample and data QC metrics were implemented (Supplementary Discussion and Extended Data Figures 2,\",\n",
            "          \"(Supplementary Discussion and Extended Data Figures 2, 3). Extensive analyses concluded that 28 of the 105 samples were compromised by protein degradation. These samples were excluded from further analysis with subsequent informatics focused on the 77\",\n",
            "          \"further analysis with subsequent informatics focused on the 77 tumor samples and three biological replicates. Genome and transcriptomic variation was observed at the peptide level by searching MS/MS spectra not matched to RefSeq against a patient-specific\",\n",
            "          \"MS/MS spectra not matched to RefSeq against a patient-specific sequence database (Fig. 1a). The database was constructed using the QUILTS software package leveraging RefSeq gene models based on whole exome and RNA-seq data generated from portions of the\",\n",
            "          \"on whole exome and RNA-seq data generated from portions of the same tumors and matched germline DNA (Fig. 1a, Supplementary Table 5). While these analyses detected a number of single amino-acid variants (SAAVs), frameshifts, and splice junctions,\",\n",
            "          \"amino-acid variants (SAAVs), frameshifts, and splice junctions, including splice isoforms that had been detected as only single transcript reads by RNA-seq (Fig. 1b, Supplementary Table 5), the number of genomic and transcriptomic variants that were\",\n",
            "          \"5), the number of genomic and transcriptomic variants that were confirmed as peptides by MS was low (Supplementary Discussion). Sparse detection of individual genomic variants by peptide sequencing has been noted in our previous studies and reflects\",\n",
            "          \"sequencing has been noted in our previous studies and reflects limited coverage at the single amino-acid level with current technology. However quantitative MS analysis of multiple peptides for each protein is used to reliably infer overall protein\",\n",
            "          \"for each protein is used to reliably infer overall protein levels. This is an advantage for MS since antibody-based protein expression analysis is typically based on a single epitope. To illustrate this capability in the current data set an initial\",\n",
            "          \"illustrate this capability in the current data set an initial analysis of three frequently mutated genes in breast cancer (TP53, PIK3CA, and GATA3) and three clinical biomarkers (ER, PGR, and ERBB2) was conducted (Fig. 1c, Supplementary Table 6, 7 and\",\n",
            "          \"and ERBB2) was conducted (Fig. 1c, Supplementary Table 6, 7 and Supplementary Discussion). As expected, TP53 missense mutations were associated with elevated MS-based protein levels, as observed by RPPA (Reverse Phase Protein Array), especially in\",\n",
            "          \"observed by RPPA (Reverse Phase Protein Array), especially in basal-like breast cancer. TP53 nonsense and frame-shift mutations were associated with a decrease in TP53 protein levels that was particularly striking in the MS data. In contrast, the mostly\",\n",
            "          \"particularly striking in the MS data. In contrast, the mostly C-terminal GATA3 frame-shift alterations did not result in decreased protein expression when measured by the median of all GATA3 peptides, suggesting these proteins are expressed despite\",\n",
            "          \"GATA3 peptides, suggesting these proteins are expressed despite truncation. No consistent effect of somatic PIK3CA mutation was observed at the level of protein expression. Good correlations between RNA-seq and MS-protein expression levels were found for\",\n",
            "          \"between RNA-seq and MS-protein expression levels were found for ESR1 (r=0.74), PGR (r=0.74), ERBB2 (r=0.84) and GATA3 (r=0.83) with moderate correlations observed for PIK3CA (r=0.45) and TP53 (r=0.36). Lower TP53 protein abundance levels compared to mRNA\",\n",
            "          \"(r=0.36). Lower TP53 protein abundance levels compared to mRNA levels were especially prevalent in luminal tumors, suggesting post-transcriptional regulatory mechanisms such as proteasomal degradation. To explore this hypothesis a search was made for E3\",\n",
            "          \"To explore this hypothesis a search was made for E3 ligases that showed negative correlation to p53 protein (Supplementary Table 8). These analyses identified UBE3A (r=\\u22120.42; adj. pval=0.05) (Extended Data Fig. 4a), an established TP53 E3 ligase. In\",\n",
            "          \"(Extended Data Fig. 4a), an established TP53 E3 ligase. In comparing CNA, RNA, and protein levels for GATA3, copy number gains in chromosome 10q were anti-correlated with RNA and protein levels in basal-like tumors. This observation prompted a search for\",\n",
            "          \"in basal-like tumors. This observation prompted a search for other gains or losses that were anticorrelated with RNA and/or protein levels (see Extended Data Fig. 4b for further analyses). Overall, six genes were identified that significantly\",\n",
            "          \"Overall, six genes were identified that significantly anti-correlated at an FDR<0.05 on both RNA and protein level to their CNA signals (Extended Data Fig. 4b). GATA3 amplification on 10q in basal-like breast cancer showed the strongest anti-correlation,\",\n",
            "          \"basal-like breast cancer showed the strongest anti-correlation, followed by the hexosamine and glycolysis pathway enzymes GFPT2 and HK3, which are upregulated in basal-like breast cancer despite being subjected to frequent chromosomal deletion on 5q.\",\n",
            "          \"despite being subjected to frequent chromosomal deletion on 5q. Global analysis of the correlation of mRNA-to-protein yielded a median Pearson value of r=0.39, with 6,135 out of 9,302 protein/mRNA pairs (66.0%) correlating significantly at an FDR<0.05\",\n",
            "          \"pairs (66.0%) correlating significantly at an FDR<0.05 (Extended Data Fig. 4c, Supplementary Table 9 and Supplementary Discussion). Similar to the colon cancer analysis metabolic functions such as amino acid, sugar and fatty acid metabolism were found to\",\n",
            "          \"as amino acid, sugar and fatty acid metabolism were found to be enriched among positively correlated genes whereas ribosomal, RNA polymerase and mRNA splicing functions were negatively correlated. Overall these analyses demonstrate the utility of global\",\n",
            "          \"Overall these analyses demonstrate the utility of global proteome correlation analysis for both confirmation of suspected regulatory mechanisms and identification of candidate regulators meriting further investigation. Copy Number Alterations To determine\",\n",
            "          \"further investigation. Copy Number Alterations To determine the consequences of CNA on mRNA, protein and phosphoprotein abundance, both in \\u201ccis\\u201d on genes within the aberrant locus and in \\u201ctrans\\u201d on genes encoded elsewhere, univariate correlation analysis\",\n",
            "          \"on genes encoded elsewhere, univariate correlation analysis was used as previously described. A total of 7,776 genes with CNA, mRNA and protein measurements were analyzed by calculating Pearson correlation and associated statistical significance\",\n",
            "          \"Pearson correlation and associated statistical significance (Benjamini-Hochberg corrected p-value) for all possible CNA-mRNA and CNA-protein pairs (Fig. 2a, Supplementary Table 10, Extended Data Fig. 5a, see Methods). For the phosphoproteome, 4,472\",\n",
            "          \"Data Fig. 5a, see Methods). For the phosphoproteome, 4,472 CNA-phosphoprotein pairs were analyzed (Extended Data Fig. 5b). Significant positive correlations (cis) were observed for 64% of all CNA-mRNA, 31% of all CNA-protein, and 20% of all\",\n",
            "          \"for 64% of all CNA-mRNA, 31% of all CNA-protein, and 20% of all CNA-phosphoprotein pairs Fig. 2b. Proteins and phosphoproteins correlated in cis to CNA were, for the most part, a subset of the cis effects observed in mRNA-to-CNA correlation (Fig. 2b,\",\n",
            "          \"the cis effects observed in mRNA-to-CNA correlation (Fig. 2b, Supplementary Table 10). The fractional difference of well-annotated oncogenes and tumor suppressor genes among the significantly cis-correlated CNA-to-mRNA and -protein gene pairs was\",\n",
            "          \"cis-correlated CNA-to-mRNA and -protein gene pairs was analyzed. Based on a reference list of 487 oncogenes and tumor suppressors (Supplementary Table 10), these cancer relevant genes occur 37.6% more frequently in the subset of genes that correlate both\",\n",
            "          \"more frequently in the subset of genes that correlate both on CNA/mRNA and CNA/protein levels than in the subset that only correlate on CNA/mRNA but not on CNA/protein level (Fisher exact p-value=0.02). This suggests that CNA events with a tumor promoting\",\n",
            "          \"This suggests that CNA events with a tumor promoting outcome more likely lead to cis regulatory effects on both the protein and mRNA level, whereas CNA events with no documented role in tumorigenesis are more likely to be neutralized on the protein level\",\n",
            "          \"are more likely to be neutralized on the protein level than on the RNA level. Trans effects (Fig. 2a) appear as vertical bands, with accompanying frequency histograms (in blue) highlighting \\u201chot spots\\u201d of significant trans effects. Using a minimum\",\n",
            "          \"\\u201chot spots\\u201d of significant trans effects. Using a minimum threshold of 50 trans-affected genes, 68% of the tested genes were associated with trans effects on the mRNA level, whereas only 13% were associated with effects on the protein level and 8% on the\",\n",
            "          \"were associated with effects on the protein level and 8% on the phosphoprotein level. Importantly, CNA-protein correlations appeared to be a reduced representation of CNA-mRNA correlations. Furthermore, for many CNA regions correlations were more\",\n",
            "          \"Furthermore, for many CNA regions correlations were more directionally uniform on the protein level than on the mRNA level. CNA regions exhibiting the most trans associations at the protein level were found on chromosomes 5q (LOH in basal; gain in LumB),\",\n",
            "          \"were found on chromosomes 5q (LOH in basal; gain in LumB), 10p (gain in basal), 12 (gain in basal), 16q (LumA deletion), 17q (LumB amplification), and 22q (LOH in luminal and basal) (Extended Data Fig. 5a). Trans associations are not necessarily direct\",\n",
            "          \"Data Fig. 5a). Trans associations are not necessarily direct consequences of the chromosomal aberration. For example since 5q loss occurs in at least 50% of basal-like breast cancers, many of the trans effects involve genes that mark the basal-subtype. To\",\n",
            "          \"the trans effects involve genes that mark the basal-subtype. To identify candidate driver genes whose copy number alterations are direct drivers of trans effects, results were compared with functional knock-down data on 3,797 genes in the Library of\",\n",
            "          \"functional knock-down data on 3,797 genes in the Library of Integrated Cellular Signatures (LINCS) database (http://www.lincsproject.org/). For any given gene with copy number alterations (\\u201cCNA-gene\\u201d), sets of genes were identified corresponding to\",\n",
            "          \"(\\u201cCNA-gene\\u201d), sets of genes were identified corresponding to proteins that changed where there was gain (\\u201cCNA-gain trans gene set\\u201d) or loss (\\u201cCNA-loss trans gene set\\u201d). These gene sets were then compared to the effects of gene knock down in the LINCS\",\n",
            "          \"then compared to the effects of gene knock down in the LINCS database (see Methods). Queries for 502 different CNA-genes meeting the criteria defined above identified 10 CNA-genes that could be functionally connected to both CNA-gain and CNA-loss trans\",\n",
            "          \"be functionally connected to both CNA-gain and CNA-loss trans protein-level effects (Extended Data Fig. 5c, Supplementary Table 11). A permutation-based approach implemented to test significance (see Supplementary Methods) yielded an FDR < 0.05 for 10\",\n",
            "          \"(see Supplementary Methods) yielded an FDR < 0.05 for 10 genes affected by both CNA gains and losses (Fig. 2c). These proteins were defined as potential regulatory candidates for the CNA trans effects observed on the proteome level in this study, since in\",\n",
            "          \"effects observed on the proteome level in this study, since in a gene-dependent manner on average 17% of these trans effects were consistent with the knockdown profiles. Notably, the established oncogenic receptor tyrosine kinase ERBB2 was functionally\",\n",
            "          \"oncogenic receptor tyrosine kinase ERBB2 was functionally connected only to CNA gain trans effects (Supplementary Table 11). The E3 ligase SKP1 and the ribonucleoprotein export factor CETN3, both located on chromosome arm 5q with frequent losses in\",\n",
            "          \"both located on chromosome arm 5q with frequent losses in basal-like breast cancer and less frequent gains in luminal B breast cancer, were detected as potential regulators affecting the expression of the tyrosine kinase and therapeutic target EGFR, and\",\n",
            "          \"of the tyrosine kinase and therapeutic target EGFR, and SKP1 also was linked to SRC (Extended Data Fig. 5d). Another potential regulator, FBXO7, (a substrate recognition component of the SCF (SKP1-CUL1-F-box protein)-type E3 ubiquitin ligase complex), was\",\n",
            "          \"protein)-type E3 ubiquitin ligase complex), was affected mostly by LOH events on chromosome 22q. Interestingly, in a recent human interaction proteome study SKP1 and FBXO7 were listed as interaction partners. Clustering and Network analysis\",\n",
            "          \"listed as interaction partners. Clustering and Network analysis Transcriptional profiling has converged on four major breast cancer subtypes: Luminal A and B, basal and HER2-enriched. To investigate the extent to which the PAM50 \\u201cintrinsic\\u201d breast cancer\",\n",
            "          \"the extent to which the PAM50 \\u201cintrinsic\\u201d breast cancer classification scheme is reflected or refined on the proteome level in the CPTAC samples, clustering analyses were first restricted to the reduced set of PAM50 genes. When RNA data for the 50 PAM50\",\n",
            "          \"the reduced set of PAM50 genes. When RNA data for the 50 PAM50 genes were clustered directly (without using a classifier), the clustering was similar to the TCGA PAM50 annotation (second annotation bar in Fig. 3a). Restricting both the RNA and proteome\",\n",
            "          \"bar in Fig. 3a). Restricting both the RNA and proteome data to the set of 35 PAM50 genes observed in the proteome produced a similar result (bottom two annotation bars in Fig. 3a), and all the major PAM50 groups were recapitulated in the proteome almost\",\n",
            "          \"major PAM50 groups were recapitulated in the proteome almost as well as in the RNA data. This indicates that although different tissue sections of the same tumors were used for RNA-seq and protein analysis, very similar subtype-defining features can be\",\n",
            "          \"protein analysis, very similar subtype-defining features can be observed in both data types. Global proteome and phosphoproteome data were then used to identify proteome subtypes in an unsupervised manner. Consensus clustering identified basal-enriched,\",\n",
            "          \"manner. Consensus clustering identified basal-enriched, luminal-enriched, and stromal-enriched clusters (Extended Data Figs. 6a\\u2013d, 7a). Unlike the clustering observed with PAM50 genes, mRNA-defined HER2-enriched tumors were distributed across these three\",\n",
            "          \"HER2-enriched tumors were distributed across these three proteomic subgroups. The basal-enriched and luminal-enriched groups showed a strong overlap with the mRNA-based PAM50 basal-like and luminal subgroups, whereas stromal-enriched proteome subtype\",\n",
            "          \"luminal subgroups, whereas stromal-enriched proteome subtype represented a mix of all PAM50 mRNA-based subtypes, and has a significantly enriched stromal signature (Extended Data Fig. 3e). Among the stromal-enriched tumors there was strong representation\",\n",
            "          \"the stromal-enriched tumors there was strong representation of reactive type I tumors as classified by RPPA (Supplementary Table 12), showing agreement between the RPPA and mass spectrometry-based protein analyses for the detection of a tumor subgroup\",\n",
            "          \"protein analyses for the detection of a tumor subgroup characterized by stromal gene expression. Since the basal- and luminal-enriched proteome subgroups are coherent, pathway analyses were conducted on these two subtypes, using the stromal-enriched\",\n",
            "          \"conducted on these two subtypes, using the stromal-enriched subgroup as a control to assess specificity. (Fig. 3c, Extended Data Fig. 7b, Supplementary Table 13). The luminal-enriched subgroup was exclusively enriched for estradiol and ESR1-driven gene\",\n",
            "          \"was exclusively enriched for estradiol and ESR1-driven gene sets. In contrast, multiple gene sets were enriched and upregulated specifically in the basal-like tumors. Particularly extensive basal-like enrichment was seen for MYC target genes; for cell\",\n",
            "          \"basal-like enrichment was seen for MYC target genes; for cell cycle, checkpoint, and DNA repair pathways including regulators AURKA/B, ATM, ATR, CHEK1/2, and BRCA1/2; and for immune response/inflammation, including T-cell, B-cell, and neutrophil\",\n",
            "          \"response/inflammation, including T-cell, B-cell, and neutrophil signatures. The complementarity of transcriptional, proteomic, and phosphoproteomic data was also highlighted in these analyses (Extended Data Fig. 7c, d). Using phosphorylation status as a\",\n",
            "          \"(Extended Data Fig. 7c, d). Using phosphorylation status as a proxy for activity, phosphoproteome profiling can theoretically be used to develop a signaling pathway-based cancer classification. K-means consensus clustering was therefore performed on\",\n",
            "          \"K-means consensus clustering was therefore performed on pathways derived from single sample GSEA analysis of phosphopeptide data (Methods, Supplementary Tables 14 and 15). Of four robustly segregated groups, subgroups 2 and 3 substantially recapitulated\",\n",
            "          \"groups, subgroups 2 and 3 substantially recapitulated the stromal- and luminal-enriched proteomic subgroups, respectively (Fig. 3d, Extended Data Fig. 8a). Subgroup 4 included a majority of tumors from the basal-enriched proteomic subgroup, but was\",\n",
            "          \"of tumors from the basal-enriched proteomic subgroup, but was admixed particularly with luminal-enriched samples. This subgroup was defined by high levels of cell cycle and checkpoint activity. All basal and a majority of non-basal samples in this\",\n",
            "          \"activity. All basal and a majority of non-basal samples in this subgroup had TP53 mutations. Consistent with high levels of cell cycle activity, a multivariate kinase-to-phosphosite abundance regression analysis highlighted CDK1 as one of the most highly\",\n",
            "          \"regression analysis highlighted CDK1 as one of the most highly connected kinases in this study (Extended Data Fig. 8b, Supplementary Table 16). Subgroup 1 was a novel subgroup defined exclusively in the phosphopeptide/pathway activity domain, with no\",\n",
            "          \"in the phosphopeptide/pathway activity domain, with no enrichment for either proteomic or PAM50 subtypes. It was defined by G-protein, G-protein coupled receptor, and inositol phosphate metabolism signatures, as well as ionotropic glutamate signaling\",\n",
            "          \"signatures, as well as ionotropic glutamate signaling (Fig. 3d). Co-expression patterns among genes/proteins across different subgroups were also analyzed using a Joint Random Forest (JRF) method that identified network modules, such as an MMP9 module,\",\n",
            "          \"method that identified network modules, such as an MMP9 module, with different interaction patterns between basal-enriched and luminal-enriched subgroups. These latter patterns appeared specific to the proteome-level data (Extended Data Fig. 8 c\\u2013f,\",\n",
            "          \"specific to the proteome-level data (Extended Data Fig. 8 c\\u2013f, Supplementary Table 17 and Supplementary Methods). Phosphosite markers in PIK3CA and TP53 mutant breast cancer TP53 and PIK3CA are the most recurrently mutated genes in breast cancer, with\",\n",
            "          \"are the most recurrently mutated genes in breast cancer, with frequencies for PIK3CA at 43% in luminal tumors and for TP53 at 84% in basal-like tumors. Most of the PIK3CA missense mutations were gain of function mutations and therefore expected to lead to\",\n",
            "          \"gain of function mutations and therefore expected to lead to an activation of the PI3K signaling cascade, but the extent to which this occurs has been controversial and there is uncertainty which pathway components are effectors. Marker selection analysis\",\n",
            "          \"pathway components are effectors. Marker selection analysis was therefore performed for upregulated phosphosites in PIK3CA-mutated tumors. In total, 62 phosphosites were identified that were positively associated with PIK3CA mutation (FDR<0.05), including\",\n",
            "          \"associated with PIK3CA mutation (FDR<0.05), including the kinases RPS6KA5 and EIF2AK4 (Extended Data Fig. 9a, Supplementary Table 18). Calculating the average phosphorylation signal of these marker phosphosites provided a read-out for PI3K pathway\",\n",
            "          \"these marker phosphosites provided a read-out for PI3K pathway activity in PIK3CA-mutated tumors, with 15 of the 26 mutated tumors (58%) exhibiting an activated PIK3CA mutation signature. Of note, the identified PIK3CA mutant phosphoproteome signature was\",\n",
            "          \"the identified PIK3CA mutant phosphoproteome signature was activated in all tumors harboring helical domain PIK3CA mutations but only 2 of 10 tumors harboring kinase domain mutations. To test if the identified differences in the phosphoproteome of PI3K\",\n",
            "          \"if the identified differences in the phosphoproteome of PI3K mutant versus wild-type tumors could be explained by mutation of PIK3CA, the tumor data were compared to phosphosite signatures derived from isogenic PIK3CA mutant cell lines (Extended Data Fig.\",\n",
            "          \"from isogenic PIK3CA mutant cell lines (Extended Data Fig. 9b, Supplementary Table 18). There was an enrichment of signatures derived from helical domain-mutated isogenic cell lines, but not from kinase domain-mutated cells, supporting the observations in\",\n",
            "          \"kinase domain-mutated cells, supporting the observations in primary tumors. The same strategy was used to identify phosphorylation signaling events connected to TP53 mutation. A total of 56 phosphosites upregulated in TP53 mutant tumors were identified\",\n",
            "          \"phosphosites upregulated in TP53 mutant tumors were identified that were independent of basal-like subtype association (Extended Data Fig. 9c, Supplementary Table 18). Using the average phosphorylation signal of these marker phosphosites as a proxy for\",\n",
            "          \"signal of these marker phosphosites as a proxy for TP53 mutation-driven cell cycle control, 22 of 41 mutated tumors (54%) showed upregulated signals. This TP53 mutant phosphosignature was somewhat enhanced in tumors in which mutations occurred almost\",\n",
            "          \"somewhat enhanced in tumors in which mutations occurred almost exclusively in the DNA binding region compared to those with non-sense/frameshift mutations. In addition to the well-described checkpoint kinase CHEK2, significantly upregulated phosphosites\",\n",
            "          \"checkpoint kinase CHEK2, significantly upregulated phosphosites were identified for the kinases MASTL and EEF2K in TP53-mutated tumors. Single sample GSEA analysis of isogenic p53-mutant phosphosignatures showed an enrichment of a phosphosignature derived\",\n",
            "          \"showed an enrichment of a phosphosignature derived from R273H mutated isogenic cells (Extended Data Fig. 9d), confirming the pronounced effect of missense mutations in the DNA-binding region on phosphorylation pathways. Identification of gene\",\n",
            "          \"region on phosphorylation pathways. Identification of gene amplification and breast cancer subtype-specific activated kinases in human breast cancer CNA spans many driver gene candidates and RNA expression has been frequently used to narrow candidate\",\n",
            "          \"and RNA expression has been frequently used to narrow candidate nominations. Proteogenomic analysis should further promote this nomination process. In this candidate refinement a focus on protein kinases is warranted, since many are drug targets. An\",\n",
            "          \"protein kinases is warranted, since many are drug targets. An in-depth proteogenomic pipeline was developed that flagged kinases, expression levels of which were at least 1.5 interquartile ranges higher than the median (Supplemental Table 19). A\",\n",
            "          \"ranges higher than the median (Supplemental Table 19). A proteogenomic circos-like plot (termed a \\u201cpircos\\u201d plot) was used to map these outlier kinase values onto the genome (Fig. 4a,b, Extended Data Fig. 10a). The ERBB2 locus showed the strongest effect\",\n",
            "          \"Data Fig. 10a). The ERBB2 locus showed the strongest effect of increased phosphoprotein levels associated with gene amplification-driven RNA and protein over expression (Fig. 4a). CDK12 is a positive transcriptional regulator of homologous recombination\",\n",
            "          \"positive transcriptional regulator of homologous recombination repair genes with its partner Cyclin K, and is often encompassed by the ERBB2 amplicon. This gene was also found to be upregulated at the RNA, protein and phosphosite level indicating that\",\n",
            "          \"at the RNA, protein and phosphosite level indicating that CDK12 is highly active in the majority of ERBB2 positive tumors (Fig. 4a). The analysis of the ERBB2 amplicon also uncovered co-outlier phosphorylation status for MED1, GRB7, MSL1, CASC3 and TOP2A,\",\n",
            "          \"phosphorylation status for MED1, GRB7, MSL1, CASC3 and TOP2A, all previously described in association with ERBB2 amplification. To better understand the downstream effects of ERBB2 amplification, additional phosphosite outliers were identified in 41 known\",\n",
            "          \"additional phosphosite outliers were identified in 41 known ERBB2 signaling genes for the 15 samples that had ERBB2 phosphosite outlier expression (Extended Data Fig. 10b). These canonical findings stimulated a proteogenomic analysis to identify\",\n",
            "          \"findings stimulated a proteogenomic analysis to identify additional outlier kinases in the breast cancer genome. A proteogenomic dissection of chromosome 11q based on PAK1 amplification (Fig. 4b,c), a breast cancer driver kinase, illustrated that PAK1 is\",\n",
            "          \"4b,c), a breast cancer driver kinase, illustrated that PAK1 is hyperphosphorylated in PAK1 amplified tumors, along with CLNS1A, RFS1 and GAB2. Additional examples of outlier kinases included PTK2 and RIPK2 in association with amplification of chromosome\",\n",
            "          \"PTK2 and RIPK2 in association with amplification of chromosome 8q (Fig. 4c; Extended Data Fig. 10a,c). PAK1 and TLK2 (17q23) appear to be luminal breast cancer specific events Fig. 4c; Extended Data Fig. 10c). To further examine whether outlier kinases\",\n",
            "          \"Data Fig. 10c). To further examine whether outlier kinases were breast cancer subtype-specific, independent of amplification status, the BH-corrected probability was calculated of finding that number of phosphosite outliers within a subtype, given the\",\n",
            "          \"that number of phosphosite outliers within a subtype, given the total number of outliers across all subtypes, the subtype sample size and the total sample size. (Fig. 4d). These analyses led to the expected identification of ERBB2 in the HER2-enriched\",\n",
            "          \"to the expected identification of ERBB2 in the HER2-enriched subtype at the 5% FDR level, as well as the new finding of CDC42BPG (MRGKG), an effector kinase for RHO-family GTPases. In basal-like breast cancer, two kinases, PRKDC and SPEG, were significant\",\n",
            "          \"breast cancer, two kinases, PRKDC and SPEG, were significant at the 5% FDR level. PRKDC is a non-homologous end-joining (NHEJ) factor that can be phosphorylated by ATM kinase, and is therefore a logical finding in this disease subset. However SPEG, a\",\n",
            "          \"a logical finding in this disease subset. However SPEG, a kinase associated with severe dilated cardiomyopathy when suppressed, has not been previously reported in association with breast cancer. A larger number of subtype-specific kinases were detected\",\n",
            "          \"A larger number of subtype-specific kinases were detected at the 10% FDR level, several of which have recently described relevance in breast cancer, including PRKD3 in basal-like breast cancer, the LKB-regulated SIK3 in luminal A breast cancer and CDK13\",\n",
            "          \"the LKB-regulated SIK3 in luminal A breast cancer and CDK13 in luminal B breast cancer, which, similar to CDK12, can interact with Cyclin K. Discussion The analytic breadth and depth of proteomic and phosphoproteomic analyses displayed in this study\",\n",
            "          \"proteomic and phosphoproteomic analyses displayed in this study demonstrates the strengths of mass spectrometry-based proteomics, but also some of the limitations inherent in proteolytic peptide sequencing (see Supplementary Discussion). An example of how\",\n",
            "          \"sequencing (see Supplementary Discussion). An example of how high-dimensional proteomic analysis provides insight into unresolved genomic issues concerns the study of loss of the long arm of chromosome 5 (5q). Analysis of RNA and protein correlations\",\n",
            "          \"of chromosome 5 (5q). Analysis of RNA and protein correlations narrowed the list of potential trans-deregulated proteins. Orthogonal candidate screening using functional genomics methods identified loss of CETN3 and SKP1 as potential transregulators, with\",\n",
            "          \"loss of CETN3 and SKP1 as potential transregulators, with upregulation of EGFR as a downstream consequence in basal-like breast cancers. While further experimental evidence must be sought for these proposed regulatory relationships, SKP1/Cullin complex\",\n",
            "          \"these proposed regulatory relationships, SKP1/Cullin complex has already been linked to EGFR activation in Glioma. Unfortunately EGFR targeting has not to date proven to be effective therapy in basal-like breast cancer. This might be due to the fact the\",\n",
            "          \"in basal-like breast cancer. This might be due to the fact the SKP1 loss deregulates multiple targets requiring a much broader inhibitory strategy. It is recognized that PIK3CA mutations do not strongly activate canonical downstream effectors. Mass\",\n",
            "          \"do not strongly activate canonical downstream effectors. Mass spectrometry-based phosphoproteomics provides an opportunity for unbiased examination of downstream signaling events consequent upon PIK3CA mutational activation. These studies revealed that\",\n",
            "          \"upon PIK3CA mutational activation. These studies revealed that common PIK3CA mutations affect a large number of targets with diverse functionalities including the kinases RPS6KA5 and EIF2AK4. Thus, the data and analyses reported here extend our knowledge\",\n",
            "          \"Thus, the data and analyses reported here extend our knowledge of the effectors that promote tumorigenesis in response to constitutive activation of PI3 kinase. Similarly, TP53 mutation-associated phosphopeptides point towards novel functionalities,\",\n",
            "          \"phosphopeptides point towards novel functionalities, including regulation of the kinases MASTL and EEF2K. A central goal in breast cancer research has been the identification of druggable kinases beyond HER2. Candidate genes that exhibited similar gene\",\n",
            "          \"beyond HER2. Candidate genes that exhibited similar gene amplification-driven proteogenomic patterns to HER2 included CDK12, TLK2, PAK1 and RIPK2. The proteogenomic link with gene amplification was particularly strong for CDK12, in keeping with its\",\n",
            "          \"was particularly strong for CDK12, in keeping with its location in the ERBB2 amplicon, while the strengths of correlation between DNA amplification, RNA, protein and phosphoprotein for the other examples were more variable. The presence of activated CDK12\",\n",
            "          \"examples were more variable. The presence of activated CDK12 in the ERBB2 amplicon might explain why tumors arising in BRCA1 carriers are usually ERBB2 negative. As a positive transcriptional regulator of BRCA1 and multiple FANC family members, CDK12\",\n",
            "          \"regulator of BRCA1 and multiple FANC family members, CDK12 promotes DNA repair by homologous recombination. CDK12 amplification would, therefore, oppose the functional effects of BRCA1 haploinsufficiency during tumor evolution. Overall, multiple outlier\",\n",
            "          \"during tumor evolution. Overall, multiple outlier kinases generate testable therapeutic hypotheses for which enabling inhibitors are in development. For example PAK1 has recently been confirmed to be a therapeutic target and poor prognosis factor in\",\n",
            "          \"to be a therapeutic target and poor prognosis factor in luminal breast cancer. Although incomplete outcome data and the remarkable heterogeneity of breast cancer are additional relevant constraints, the number of TCGA specimens analyzed here is\",\n",
            "          \"constraints, the number of TCGA specimens analyzed here is insufficient to support conclusive clinical correlations. Only 8 deaths occurred among the 77 patients, which are too few samples to provide sufficient statistical power for association analysis.\",\n",
            "          \"provide sufficient statistical power for association analysis. Adequately powered MS-based clinical investigation will require targeted approaches, especially given the highly limited amount of patient material available from clinical trials and the\",\n",
            "          \"of patient material available from clinical trials and the mostly formalin-fixed nature of the specimens. The current analysis is therefore centered on biological findings and correlations, with orthogonal validation and false discovery concerns addressed\",\n",
            "          \"orthogonal validation and false discovery concerns addressed through an examination of cell-line databases of the effects of individual gene perturbations. Typical of a multi-tiered analysis of this complexity, there are many hypotheses to test, and many\",\n",
            "          \"of this complexity, there are many hypotheses to test, and many findings that require further investigation. In conclusion, this study provides a high-quality proteomic resource for human breast cancer investigation and illustrates technologies and\",\n",
            "          \"breast cancer investigation and illustrates technologies and analytical approaches that provide an important new opportunity to connect the genome to the proteome. Larger-scale exploration of discovery proteomics in the clinical setting will require\",\n",
            "          \"of discovery proteomics in the clinical setting will require improvements in clinical investigation, including acquisition of adequate amounts of optimally collected tumor tissue both before and during therapy as well as advances in MS proteomics to\",\n",
            "          \"and during therapy as well as advances in MS proteomics to reduce sample input and increase sensitivity for low abundance proteins and modified peptides. Extended Data Experimental and data analysis workflows and longitudinal data generation quality\",\n",
            "          \"analysis workflows and longitudinal data generation quality control a, iTRAQ 4-plex global proteome and phosphoproteome analysis workflow. 105 TCGA breast tumors were analyzed in 35 iTRAQ 4-plex experiments (plus 1 replicate and 1 normal sample\",\n",
            "          \"iTRAQ 4-plex experiments (plus 1 replicate and 1 normal sample experiment), with three tumors of different subtypes compared to a fourth common internal reference sample in each experiment. The reference sample comprised 10 individual tumors of each of\",\n",
            "          \"The reference sample comprised 10 individual tumors of each of the 4 major breast cancer intrinsic subtypes and served as an internal standard for all proteins and phosphoproteins quantified in this study. Each iTRAQ MS/MS spectrum measures a peptide from\",\n",
            "          \"this study. Each iTRAQ MS/MS spectrum measures a peptide from 4 samples (3 individual patients and the reference sample mix of 40 patients). More than 400,000 distinct peptides were identified and quantified in ~14 million MS/MS spectra. Personalized\",\n",
            "          \"and quantified in ~14 million MS/MS spectra. Personalized tumor-specific protein databases were generated in the QUILTS software package using whole exome sequencing-derived variant calls and RNAseq-derived transcript information. All mass spectrometry\",\n",
            "          \"RNAseq-derived transcript information. All mass spectrometry data was analyzed using the Spectrum Mill software package. b, Overview of proteome and phosphoproteome datasets. The table provides a summary of the datasets used in specific analyses,\",\n",
            "          \"provides a summary of the datasets used in specific analyses, including the filters applied to derive the proteins and phosphosites/phosphoproteins that constitute each dataset; the protein, phosphosite or phosphoprotein count; and the methods that employ\",\n",
            "          \"or phosphoprotein count; and the methods that employ the respective datasets. c, Distribution of sequence coverage of the identified proteins with tryptic peptides detected by MS/MS, whiskers show the 5\\u201395 percentiles. d and e, Robust and accurate\",\n",
            "          \"show the 5\\u201395 percentiles. d and e, Robust and accurate proteome/phosphoproteome platform. Longitudinal performance was tested by repeated proteome and phosphoproteome analysis of patient-derived xenograft tumors. Scatterplots, histograms and Pearson\",\n",
            "          \"xenograft tumors. Scatterplots, histograms and Pearson correlations comparing individual replicate measurements are shown. Tumor sample quality control (I) a, Remark diagram showing sample processing and partitioning. Initial quality review encompassed\",\n",
            "          \"processing and partitioning. Initial quality review encompassed histopathological examination of H&E stained tissue slices. *For 3 samples no tumor cells were seen on histopathology (BH-A0E9, BH-A0C1, A2-A0SW). These samples were nevertheless included in\",\n",
            "          \"BH-A0C1, A2-A0SW). These samples were nevertheless included in the proteome analysis since other quality control standards were met (see below) and samples with 0% tumor cellularity on top or bottom sections were included in TCGA analyses. b, correlation\",\n",
            "          \"bottom sections were included in TCGA analyses. b, correlation of TCGA (top or bottom sections) and CPTAC histological assessment of neoplastic cellularity for samples (n = 105). The average and range of neoplastic cellularities were identical for CPTAC\",\n",
            "          \"and range of neoplastic cellularities were identical for CPTAC and TCGA histological assessments. Averages (standard deviations) for neoplastic cellularity were 76% (+/\\u2212 17) for CPTAC, 76% (+/\\u2212 15) for TCGA_Top, and 75% (+/\\u2212 18) for TCGA_Bottom\",\n",
            "          \"76% (+/\\u2212 15) for TCGA_Top, and 75% (+/\\u2212 18) for TCGA_Bottom histopathology slides (Supplementary Table 2). Note that in three CPTAC cases where no tumor cells were identified by histopathological assessment, numbers of protein-level somatic variants were\",\n",
            "          \"assessment, numbers of protein-level somatic variants were similar to all other tumors. The identified mutated proteins were TP53_R273C, NOP58_Q23E, TAGLN2_G154R, TUBA1B_D116H, and MRPL48_I173K (Supplementary Table 5), indicating presence of tumor cells\",\n",
            "          \"(Supplementary Table 5), indicating presence of tumor cells in these samples. c, Proteome iTRAQ tumor/internal reference ratio heatmap for all CPTAC samples (8,028 proteins without missing values) including passed and failed proteomic quality control (QC)\",\n",
            "          \"including passed and failed proteomic quality control (QC) samples. d, Global tumor/reference proteome ratio distributions for samples that passed and failed proteomic quality control analysis. e, Degradation-related gene sets were enriched in tumors that\",\n",
            "          \"e, Degradation-related gene sets were enriched in tumors that failed proteomic quality control analysis. f, Variant allele frequency (VAF) analysis of re-sequenced CPTAC tumors and comparison to original TCGA data. Overall VAFs for failed QC samples were\",\n",
            "          \"to original TCGA data. Overall VAFs for failed QC samples were lower compared to passed samples suggesting lower purity. Tumor sample quality control (II) a, There was high concordance (94.6%) between DNA variants reported by TCGA and CPTAC re-sequenced\",\n",
            "          \"between DNA variants reported by TCGA and CPTAC re-sequenced tumors. Most point mutations reported by TCGA could be identified across the 8 re-sequenced samples used in the study. b, A high overall correlation (mean=0.77) was observed for the CPTAC\",\n",
            "          \"high overall correlation (mean=0.77) was observed for the CPTAC Variant Allele Fraction (VAF) (X-axis) and TCGA VAF (Y-axis) across the 8 samples used in the study. c, Agglomerative hierarchical clustering (Supplementary Methods Section 3.8) used to\",\n",
            "          \"clustering (Supplementary Methods Section 3.8) used to co-cluster protein and RNA tumor expression data after filtering to retain 4,291 proteins and genes with moderate to high protein-RNA correlation (Pearson correlation > 0.4) with results displayed as\",\n",
            "          \"(Pearson correlation > 0.4) with results displayed as a circular dendrogram (fanplot). The proteome (.P) and RNA (.R) components of each sample are labeled using the same color. The outer ring shows proteome samples in light grey and RNA samples in dark\",\n",
            "          \"shows proteome samples in light grey and RNA samples in dark grey. High concordance between RNA and protein expression is evident from the color adjacency in the inner ring and alternating color in the outer ring showing that RNA and protein components\",\n",
            "          \"color in the outer ring showing that RNA and protein components co-cluster for a large proportion of samples (62/80). d, Co-clustering of MS and RPPA tumor data. 126 RPPA readouts were mapped to gene names. These genes were intersected with the genes\",\n",
            "          \"to gene names. These genes were intersected with the genes observed in the MS proteome, filtered to 48 proteins with moderate or higher RPPA-MS protein correlation, and analyzed for co-clustering as in c. 47 of 80 RPPA-MS protein pairs co-cluster. While\",\n",
            "          \"as in c. 47 of 80 RPPA-MS protein pairs co-cluster. While this is a smaller proportion than for RNA-protein analysis, the number of genes used in the clustering is significantly smaller for RPPA (48 vs. 4,291 for RNA). e, ESTIMATE tumor purity comparison\",\n",
            "          \"(48 vs. 4,291 for RNA). e, ESTIMATE tumor purity comparison between mRNA, RNAseq, and proteome data. ANOVA is used to assess the difference in distribution (\\u2212log10(p-value)) of ESTIMATE, stromal, immune, and tumor purity scores across mRNA (microarray),\",\n",
            "          \"immune, and tumor purity scores across mRNA (microarray), RNA-seq and proteome data. The only significant p-value (=0.02) is for the Cluster 3 stromal score, and higher stromal scores for the proteome drive that difference. f, Ischemia score analysis.\",\n",
            "          \"the proteome drive that difference. f, Ischemia score analysis. Comparison of ischemia scores of 77 CPTAC tumors, 3 normal samples, and patient-derived xenografts. CPTAC tumors had generally lower ischemia scores than PDX samples subjected to 30 minutes\",\n",
            "          \"lower ischemia scores than PDX samples subjected to 30 minutes of cold ischemia. Median ischemia scores are less than 30 minutes for each subtype and no significant differences were observed across subtypes. Effects due to cold ischemia therefore appear\",\n",
            "          \"across subtypes. Effects due to cold ischemia therefore appear to be negligible in this CPTAC sample collection. Protein-to-Protein, -CNA, and -mRNA correlation analyses a, Identification of UBE3A as an E3 ubiquitin ligase that negatively correlates to\",\n",
            "          \"UBE3A as an E3 ubiquitin ligase that negatively correlates to p53 on the protein level. Pearson correlation and Benjamini-Hochberg corrected p-value are shown. b, Analysis of counter-regulated genes with negative correlation of CNA-to-RNA as well as\",\n",
            "          \"genes with negative correlation of CNA-to-RNA as well as CNA-to-protein levels. Negative Pearson correlations are shown with Benjamini-Hochberg corrected p-values for CNA-to-protein correlations. Depicted genes have significant negative correlations at\",\n",
            "          \"Depicted genes have significant negative correlations at FDR<0.05 in the CNA-to-RNA and CNA-to-protein analyses. c, Global mRNA-to-protein correlation and gene set enrichment analysis. Global CNA effects and comparison of CNA TRANS effects to knockdown\",\n",
            "          \"CNA effects and comparison of CNA TRANS effects to knockdown signatures in the LINCS database a, CNA landscape in the CPTAC tumor collection. The segment-based CNAs of 77 samples were downloaded from TCGA Firehose, including 18 Basal, 12 Her2, 23 Luminal\",\n",
            "          \"from TCGA Firehose, including 18 Basal, 12 Her2, 23 Luminal A and 24 Luminal B subtypes. Copy number amplifications were marked in red and deletions in blue. The bottom color key represents the log2 transformed copy number value, with CNA=2 centered at 0.\",\n",
            "          \"log2 transformed copy number value, with CNA=2 centered at 0. Specific CNA events are seen for chromosome 5q and 10p regions in basal-like tumors. b, Correlations of copy number alterations (x-axis) to phosphoprotein levels (y-axis) highlight new CNA cis\",\n",
            "          \"to phosphoprotein levels (y-axis) highlight new CNA cis and trans effects. Significant (FDR<0.05) positive (red) and negative (green) correlations between CNA and phosphoproteins are indicated. Histograms show the fraction [%] of significant CNA trans\",\n",
            "          \"Histograms show the fraction [%] of significant CNA trans effects for each CNA gene. c, LINCS CMap analysis facilitates identification of novel functional candidates for CNA trans effects. Knockdown profiles were compared with CNA/protein trans effects\",\n",
            "          \"Knockdown profiles were compared with CNA/protein trans effects for 502 genes. Genes with a connectivity score >|90| were considered connected and significant cis effects were annotated at an FDR<0.05. d, Basal-like tumor-specific CNAs are candidate\",\n",
            "          \"at an FDR<0.05. d, Basal-like tumor-specific CNAs are candidate regulatory events for EGFR and SRC expression levels. Oncogenic kinases with significant CNA/protein trans effects (left panel), that were regulated in LINCS shRNA experiments (right panel; 4\",\n",
            "          \"that were regulated in LINCS shRNA experiments (right panel; 4 cell lines,) and directly measured as LINCS landmark genes, are shown alongside candidate regulatory genes CETN3 and SKP1. Clinical ER, PR, and HER2 annotation and PAM50 classification are\",\n",
            "          \"ER, PR, and HER2 annotation and PAM50 classification are shown in the header rows of each column. Proteome cluster heatmap and stability analysis a, K-means consensus clustering of proteome and phosphoproteome data identifies three subgroups:\",\n",
            "          \"proteome and phosphoproteome data identifies three subgroups: basal-enriched, luminal-enriched, and stromal-enriched. The heatmap represents all 1,521 proteins used for clustering (Dataset G8). b, Identification of optimal proteome clusters for QC-passed\",\n",
            "          \"b, Identification of optimal proteome clusters for QC-passed CPTAC breast cancer tumors. Proteome clusters were derived using consensus clustering based on 1000 resampled datasets, exploring the range of 2 to 6 k-means clusters. Visualization of consensus\",\n",
            "          \"range of 2 to 6 k-means clusters. Visualization of consensus matrices from k-means consensus clustering for k=3, 4, 5 and 6 target clusters. Consensus clustering was performed on 1,521 proteins with no missing values and SD>1.5. c, Silhouette plots were\",\n",
            "          \"with no missing values and SD>1.5. c, Silhouette plots were generated to evaluate the coherence of the clustering. Silhouette plots for k=3 and k=4 clusters showing a cleaner separation of clusters for k=3. d, Based on both visual inspection of the\",\n",
            "          \"of clusters for k=3. d, Based on both visual inspection of the consensus matrix and the delta plot assessing change in consensus cumulative distribution function (CDF) area, three robustly segregated groups were observed. Consensus cumulative distribution\",\n",
            "          \"groups were observed. Consensus cumulative distribution function (CDF) and delta area (change in CDF area) plots for 2\\u20136 clusters. Proteome cluster markers and enriched pathways a, Markers (based on SAM analysis; FDR<0.01) discriminate between proteome\",\n",
            "          \"(based on SAM analysis; FDR<0.01) discriminate between proteome clusters 1, 2 and 3 (compare to heatmap of proteins used to derive clusters depicted in Extended Data Fig. 6a). b, Applying a Fisher exact test-based enrichment analysis to the proteome,\",\n",
            "          \"a Fisher exact test-based enrichment analysis to the proteome, phosphoproteome and mRNA data, gene sets from MSigDB were identified that were unique for each proteome cluster. Heat map showing specific pathways comprising dominant biological themes that\",\n",
            "          \"specific pathways comprising dominant biological themes that are significantly differential by enrichment analysis between basal-enriched and luminal-enriched tumors (Fisher Exact Test Benjamini-Hochberg corrected p-values are shown; enrichment test\",\n",
            "          \"corrected p-values are shown; enrichment test performed on marker sets identified using SAM analysis; see Methods; compare to Figure 3c). c, Heatmap showing a selection of gene sets significant in basal-enriched or luminal-enriched tumors exclusively by\",\n",
            "          \"in basal-enriched or luminal-enriched tumors exclusively by mRNA, protein or phosphoprotein expression. Cytokine signatures, for example, were strongly captured at the mRNA level, but were seen to only a limited degree at the global protein level, likely\",\n",
            "          \"to only a limited degree at the global protein level, likely because of their typically low protein abundance. By contrast, the vast majority of significant gene sets annotated as \\\"signaling\\\" were enriched only at the phosphoprotein level. d, Global heat\",\n",
            "          \"were enriched only at the phosphoprotein level. d, Global heat map representing all gene sets significantly enriched in at least one of the proteomic breast cancer subtypes. The stromal-enriched group was characterized by breast cancer normal-like,\",\n",
            "          \"group was characterized by breast cancer normal-like, adipocyte differentiation, smooth muscle, toll-like receptor signaling and endothelin gene sets, supporting the clustering-based annotation of high stromal and/or adipose content in these tumors (see\",\n",
            "          \"of high stromal and/or adipose content in these tumors (see Supplemental Table 13). Phosphoproteome pathway clustering, kinase-phosphosite multivariate regression, and protein co-expression networks a, Phosphoproteome pathway clustering. Using\",\n",
            "          \"networks a, Phosphoproteome pathway clustering. Using phosphorylation state as a proxy for activity, deep phosphoproteome profiling allows development of a breast cancer molecular taxonomy based on signaling pathways. K-means consensus clustering was\",\n",
            "          \"based on signaling pathways. K-means consensus clustering was performed on pathways derived from single sample GSEA analysis of phosphopeptide data (908 pathways shown). Of four robustly segregated groups, subgroups 2 and 3 substantially recapitulated the\",\n",
            "          \"groups, subgroups 2 and 3 substantially recapitulated the stromal- and luminal-enriched proteomic subgroups, respectively. Subgroup 4 included a significant majority of tumors from the basal-enriched proteomic subgroup, but was admixed particularly with\",\n",
            "          \"proteomic subgroup, but was admixed particularly with luminal-enriched samples. This subgroup was defined by high levels of cell cycle and checkpoint activity. All basal and a majority of non-basal samples in this subgroup had TP53 mutations. Subgroup 1\",\n",
            "          \"samples in this subgroup had TP53 mutations. Subgroup 1 was a novel subgroup defined exclusively in the phosphopeptide / pathway activity domain, with no enrichment for either proteomic or PAM50 subtypes. It was defined by G-protein, G-protein coupled\",\n",
            "          \"PAM50 subtypes. It was defined by G-protein, G-protein coupled receptor, and inositol phosphate metabolism signatures, as well as ionotropic glutamate signaling. b, Analysis of the regulatory relationship between outlier kinases (see Supplementary Table\",\n",
            "          \"relationship between outlier kinases (see Supplementary Table 19) and phosphopeptides by regulatory multivariate regression analysis (see Methods) identified CDK1 as the most highly connected of the outlier Cyclin-Dependent Kinases, with highest\",\n",
            "          \"connected of the outlier Cyclin-Dependent Kinases, with highest centrality (based on node-degree; see Methods) among the outlier CDKs and seventh highest centrality among all the outlier kinases considered in the remMap analysis. Each line represents a\",\n",
            "          \"considered in the remMap analysis. Each line represents a phosphosite-kinase relationship. c\\u2013f, Analysis of differences in the co-expression patterns among genes/proteins across different subgroups. A Joint Random Forest (JRF) method was applied to\",\n",
            "          \"subgroups. A Joint Random Forest (JRF) method was applied to simultaneously build gene co-expression and protein co-expression networks (Supplementary Table 17, and Methods). Modules in these networks revealed different interaction patterns between\",\n",
            "          \"these networks revealed different interaction patterns between basal-enriched and luminal-enriched subgroups. c, Network module P1 of the protein co-expression network, defined chiefly in the proteome space. This module contained 12 genes connected by 39\",\n",
            "          \"proteome space. This module contained 12 genes connected by 39 edges, among which 34 were protein-specific and 5 were shared by both the protein and mRNA co-expression networks. Many edges were supported by published information and were contained in the\",\n",
            "          \"supported by published information and were contained in the STRING database. Edges in red are specific to the protein co-expression network; edges in green are shared by both protein and gene co-expression networks; edges indicated by double lines are\",\n",
            "          \"co-expression networks; edges indicated by double lines are contained in the STRING database with confidence score greater than 0.15. MMP9, one of the central proteins in this module, contributes to metastatic progression and is a potential target for\",\n",
            "          \"to metastatic progression and is a potential target for anti-metastatic therapies for basal-like / triple negative breast cancer. d, Heatmaps of the absolute correlation across each pair of genes in module P1 (shown in Panel c), based on either protein or\",\n",
            "          \"in module P1 (shown in Panel c), based on either protein or gene expression data for samples in the basal-enriched and luminal-enriched subgroups, respectively. The MMP9 protein was strongly co-expressed with the other members of the module only in the\",\n",
            "          \"co-expressed with the other members of the module only in the basal-enriched subgroup. Notably, this observation is dependent on protein data; the correlation at the mRNA level for this module was consistently low in both the basal-enriched and luminal\",\n",
            "          \"was consistently low in both the basal-enriched and luminal enriched subgroups indicating that these events coherently occur at the proteomic level. e, Co-expression network based on proteomics data. The network contains 693 proteomic network-specific\",\n",
            "          \"data. The network contains 693 proteomic network-specific edges (grey) and 792 edges shared with the RNAseq network (green). For each module, the most enriched category and corresponding Benjamini-Hochberg adjusted p-value is reported. Pie charts adjacent\",\n",
            "          \"adjusted p-value is reported. Pie charts adjacent to each module show the proportion of proteomics-specific edges (grey area) and edges shared between proteomics and RNAseq data (green area). f, RNAseq network. Phosphoproteome signatures of PIK3CA (a,b)\",\n",
            "          \"f, RNAseq network. Phosphoproteome signatures of PIK3CA (a,b) and TP53 (c,d) mutated tumors highlight activated key regulators and indicate frequency of activation a and c, Phosphosites upregulated in mutated tumors (SAM FDR<0.05 across all tumors and\",\n",
            "          \"in mutated tumors (SAM FDR<0.05 across all tumors and independently also across luminal tumors; average phosphosite signal for all markers shown as bar graph). To avoid confounding by intrinsic subtype-specific distinctions, only markers that were\",\n",
            "          \"intrinsic subtype-specific distinctions, only markers that were significantly identified both in analyses covering all tumors and analyses restricted to luminal tumors were selected (FDR <0.05). Color bars in the margins indicate FDRs for grouped analysis\",\n",
            "          \"Color bars in the margins indicate FDRs for grouped analysis of different mutation classes and indicate kinase substrates of known kinases in the respective pathways. Significantly regulated kinase phosphosites are annotated. The average phosphorylation\",\n",
            "          \"kinase phosphosites are annotated. The average phosphorylation signal of the marker phosphosites provides a read-out for PI3K and TP53 pathway activity in mutated tumors (histogram below heatmap). A 95% prediction confidence interval (indicated by dashed\",\n",
            "          \"A 95% prediction confidence interval (indicated by dashed lines) across the average signal in non-mutated tumors was chosen in order to discriminate active from non-active tumors. The most strongly activated PIK3CA kinase domain mutant tumor differed from\",\n",
            "          \"activated PIK3CA kinase domain mutant tumor differed from the other 9 kinase domain mutant tumors, as it contained an amino acid side chain charge neutral H1047L instead of the more common positively charged H1047R mutation. Among the 62 phosphosites\",\n",
            "          \"positively charged H1047R mutation. Among the 62 phosphosites identified that were significantly upregulated in PIK3CA mutated tumors, 13 phosphosites were found on phosphoproteins that are known substrates of well-annotated kinases in the PIK3CA pathway\",\n",
            "          \"substrates of well-annotated kinases in the PIK3CA pathway (panel a, right column). In the mutant TP53 analysis a total 20 phosphosites were found on phosphoproteins that are known substrates of well annotated kinases in the p53 pathway (panel c, right\",\n",
            "          \"of well annotated kinases in the p53 pathway (panel c, right column). b and d, Upregulated phosphosite sets were derived from isogenic PIK3CA and TP53 mutant versus wild-type cell line pairs and tested for enrichment within mutant versus wild-type CPTAC\",\n",
            "          \"and tested for enrichment within mutant versus wild-type CPTAC tumors using single sample GSEA. Significantly enriched phosphosite sets are shown (p<0.05). PIRCOS plots, kinase outliers and outliers in the ERBB2 pathway a, Pircos (Proteogenomics CIRCOS)\",\n",
            "          \"outliers in the ERBB2 pathway a, Pircos (Proteogenomics CIRCOS) plots for 8q and 17q showing median CNA, RNA, protein, and phosphosite expression for 20 tumors with amplification in 8q based on RIPK2 CNA>1; 23 tumors with amplification in 8q based on PTK2\",\n",
            "          \"RIPK2 CNA>1; 23 tumors with amplification in 8q based on PTK2 CNA>1; 15 tumors with amplification in 17q based on CDK12 CNA >1; and 10 tumors with amplification in 17q based on TLK2 CNA>1. Red indicates expression >1, blue < \\u22121, and grey between \\u22121 and 1.\",\n",
            "          \"indicates expression >1, blue < \\u22121, and grey between \\u22121 and 1. Genes with both copy number amplification (CNA>1) and increased phosphosite expression (p-site>1) are labeled. b, Phosphosite outliers in known ERBB2 signaling genes. To better understand the\",\n",
            "          \"in known ERBB2 signaling genes. To better understand the downstream effects of ERBB2 amplification, phosphosite outliers in known ERBB2 signaling genes (MSigDB pathway set, KEGG_ERBB_SIGNALING PATHWAY) were identified for the 15 samples that had ERBB2\",\n",
            "          \"PATHWAY) were identified for the 15 samples that had ERBB2 phosphosite outlier status. Forty-one genes were identified as having a phosphosite outlier in at least one of the ERBB2 amplified samples. PAK4 and ARAF phosphosite outlier status were found in\",\n",
            "          \"samples. PAK4 and ARAF phosphosite outlier status were found in seven of the 15 ERBB2 kinase outlier samples; GSK3B outliers were found in 6 samples; and EIF4EBP1, MAP2K2, ABL1 and AKT1 outlier status was found in 5 of the 15 samples. c, Proteogenomic\",\n",
            "          \"status was found in 5 of the 15 samples. c, Proteogenomic outlier expression analysis for TLK2 and RIPK2. Samples with outlier phosphosite (red), protein (yellow), RNA (green) and copy number (purple) expression are shown. Phosphosite squares indicate\",\n",
            "          \"(purple) expression are shown. Phosphosite squares indicate per-sample outlier phosphosites. Supplementary Material Supplementary Information is available in the online version of the paper. Author Contributions P.M., D.R.M., M.A.G., K.R.C., and S.A.C.\",\n",
            "          \"Author Contributions P.M., D.R.M., M.A.G., K.R.C., and S.A.C. designed the proteomic analysis experiments, data analysis workflow, and proteomic-genomic data comparisons. P.M., M.A.G., J.W.Q., and S.A.C. directed and performed proteomic analysis of breast\",\n",
            "          \"and S.A.C. directed and performed proteomic analysis of breast tumor and quality control samples. P.M., D.R.M., K.V.R., K.R.C., P.W., X.W., S.C., E.K., F.P., Z.T., J.L., M.L.G., M.W., V.Y., K.H., C.L., M.D.M., P.Y., J.W., B.Z., and D.F. performed\",\n",
            "          \"V.Y., K.H., C.L., M.D.M., P.Y., J.W., B.Z., and D.F. performed proteomic-genomic data analyses. D.R.M., P.W., and S.J.S. provided statistical support. D.R.M., K.V.R., K.R.C., K.K. and D.F. performed analyses of mass spectrometry data and adapted\",\n",
            "          \"D.F. performed analyses of mass spectrometry data and adapted algorithms and software for data analysis. S.R.D., R.R.T and M.J.E. developed and prepared breast xenografts used as quality control samples. P.M. and F.M prepared and analyzed cell lines for\",\n",
            "          \"samples. P.M. and F.M prepared and analyzed cell lines for correlative functional annotation of frequently mutated genes. P.M., D.R.M., M.A.G., and S.A.C designed strategy for quality control analyses. M.A.G., S.R.D., C.R.K., M.M., and H.R. coordinated\",\n",
            "          \"analyses. M.A.G., S.R.D., C.R.K., M.M., and H.R. coordinated acquisition, distribution and quality control evaluation of TCGA tumor samples. P.M., M.A.G., C.P., L.D., A.G.P., and M.J.E. interpreted data in the context of breast cancer biology. P.M.,\",\n",
            "          \"interpreted data in the context of breast cancer biology. P.M., D.R.M, M.A.G., K.R.C., P.W., A.G.P, M.J.E. and S.A.C. wrote the manuscript. All primary mass spectrometry data are deposited at the CPTAC Data Portal as raw and mzML files and complete\",\n",
            "          \"at the CPTAC Data Portal as raw and mzML files and complete protein assembly data sets for public access (https://cptac-data-portal.georgetown.edu/cptac/s/S029). In addition, a set of ancillary files such as dataset G1/P1, G3/P3, G4/P4, G5/P5, G7/P7, CNA\",\n",
            "          \"files such as dataset G1/P1, G3/P3, G4/P4, G5/P5, G7/P7, CNA correlation tables for CNA-mRNA, CNA-proteome and CNA-phosphoproteome; CNA data, RNA-seq expression data have also been deposited at the DCC. Two browsers have been created to assist the\",\n",
            "          \"at the DCC. Two browsers have been created to assist the interested reader in exploring the results. One provides track hubs for viewing the identified peptides in the UCSD genome browser (http://fenyolab.org/cptac_breast_ucsc). The second is an on-line\",\n",
            "          \"The second is an on-line tool for proteogenomic data exploration and can be accessed at http://prot-shiny-vm.broadinstitute.org:3838/BC2016/. See Supplemental Methods for descriptions. The authors declare no competing financial interests. Proteogenomic\",\n",
            "          \"authors declare no competing financial interests. Proteogenomic analysis of human breast cancer. Direct effects of genomic alterations on protein level Overlap of a, protein coding single amino acid variants (SAAVs) and b, RNA splice junctions not present\",\n",
            "          \"acid variants (SAAVs) and b, RNA splice junctions not present in RefSeq v60 detected by DNA exome sequencing, RNA-seq, and LC-MS/MS. Proportions of novel variants are noted. c, Heatmap of mutations/CNA and their effects on RNA and protein expression of\",\n",
            "          \"and their effects on RNA and protein expression of breast cancer-relevant genes across tumor and normal samples. ER, PR, HER2 and PAM50 status are annotated. Median iTRAQ protein abundance ratio and the most frequently detected and differential\",\n",
            "          \"ratio and the most frequently detected and differential phosphosite ratio are shown for each gene. Pearson correlations between MS protein vs RNA-seq and MS protein vs RPPA are indicated. Effects of copy number alterations (CNA) on mRNA, protein, and\",\n",
            "          \"Effects of copy number alterations (CNA) on mRNA, protein, and phosphoprotein abundance a, Correlations of CNA (x-axes) to RNA and protein expression levels (y-axes) highlight new CNA cis and trans effects. Significant (FDR<0.05) positive (red) and\",\n",
            "          \"and trans effects. Significant (FDR<0.05) positive (red) and negative (green) correlations between CNA and mRNAs or proteins are indicated. CNA cis effects appear as a red diagonal line, CNA trans effects as vertical stripes. Histograms show the fraction\",\n",
            "          \"trans effects as vertical stripes. Histograms show the fraction [%] of significant CNA trans effects for each CNA gene. b, Overlap of cis effects observed at RNA, protein, and phosphoprotein levels (FDR<0.05). c, Trans-effect regulatory candidates\",\n",
            "          \"levels (FDR<0.05). c, Trans-effect regulatory candidates identified among those with significant protein cis-effects using LINCS CMap. Bars indicate total numbers of significant CNA/protein trans effects (gray; FDR<0.05) and overlap with regulated genes\",\n",
            "          \"trans effects (gray; FDR<0.05) and overlap with regulated genes in LINCS knock-down profiles (red; 4 cell lines; moderated T-test FDR<0.1). Proteomic and phosphoproteomic subtypes of breast cancer and subtype-specific pathway enrichment a, Unsupervised\",\n",
            "          \"cancer and subtype-specific pathway enrichment a, Unsupervised clustering of RNA-seq and proteomics data restricted to PAM50 genes and subset of 35 detected proteins reveal high similarity to PAM50 (TCGA) sample annotation. b, K-means consensus clustering\",\n",
            "          \"PAM50 (TCGA) sample annotation. b, K-means consensus clustering of proteome and phosphoproteome data identifies basal-enriched, luminal-enriched, and stromal-enriched subgroups. c, Gene set enrichment analysis highlights sets of pathways significantly\",\n",
            "          \"enrichment analysis highlights sets of pathways significantly differential between basal-enriched and luminal-enriched tumors (detailed in Extended Data Fig. 7b). d, K-means consensus clustering performed on pathways derived from single sample GSEA\",\n",
            "          \"performed on pathways derived from single sample GSEA analysis of phosphopeptide data identifies four distinct clusters. Example analyses of aberrantly regulated kinases in human breast cancer a and b, PIRCOS (Proteogenomics CIRCOS) plots showing CNA,\",\n",
            "          \"a and b, PIRCOS (Proteogenomics CIRCOS) plots showing CNA, RNA, protein and phosphosite expression for 17 tumors with amplification in 17q (ERBB2 CNA>1) and 8 tumors with amplification in 11q (PAK1 CNA>1). Labeled genes have CNA>1 and phosphosite>1. c,\",\n",
            "          \"(PAK1 CNA>1). Labeled genes have CNA>1 and phosphosite>1. c, Proteogenomic outlier expression analysis for ERBB2, CDK12, and PAK1. Samples with outlier phosphosite (red), protein (yellow), RNA (green) and copy number (purple) expression are shown.\",\n",
            "          \"RNA (green) and copy number (purple) expression are shown. Phosphosite squares indicate per-sample outlier phosphosites. d, Outlier kinase events by PAM50 subtype (>35% of subtype samples contain a phosphosite outlier; <10% FDR using Benjamini-Hochberg\",\n",
            "          \"a phosphosite outlier; <10% FDR using Benjamini-Hochberg adjusted p-values).\"\n",
            "        ]\n",
            "      }\n",
            "    ]\n",
            "  },\n",
            "  \"PDC000290\": {\n",
            "    \"study_id\": \"57b07c60-c9e1-424f-bfd6-9262e06a77d8\",\n",
            "    \"pdc_study_id\": \"PDC000290\",\n",
            "    \"study_submitter_id\": \"TCGA Breast Cancer CompRef Samples - Proteome\",\n",
            "    \"program_id\": \"10251935-5540-11e8-b664-00a098d917f8\",\n",
            "    \"project_id\": \"c96eca75-8b73-4018-9bf7-b20e31a965cd\",\n",
            "    \"study_name\": \"TCGA Breast Cancer CompRef Samples - Proteome\",\n",
            "    \"program_name\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"project_name\": \"CPTAC2 Retrospective\",\n",
            "    \"disease_type\": \"Other\",\n",
            "    \"primary_site\": \"Not Reported\",\n",
            "    \"analytical_fraction\": \"Proteome\",\n",
            "    \"experiment_type\": \"iTRAQ4\",\n",
            "    \"embargo_date\": null,\n",
            "    \"cases_count\": 2,\n",
            "    \"aliquots_count\": 20,\n",
            "    \"protocol_id\": \"7b457134-224f-4c69-b5a8-f62a99666489\",\n",
            "    \"protocol_submitter_id\": \"TCGA Breast Cancer CompRef Proteome\",\n",
            "    \"program_submitter_id\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"protocol_name\": \"TCGA Breast Cancer CompRef Proteome\",\n",
            "    \"protocol_date\": \"2021-04-07\",\n",
            "    \"publications\": [\n",
            "      {\n",
            "        \"publication_id\": \"44e2e766-89c7-11e8-bcf1-0a2705229b82\",\n",
            "        \"pubmed_id\": \"27251275\",\n",
            "        \"doi\": \"10.1038/nature18003\",\n",
            "        \"title\": \"Proteogenomics connects somatic mutations to signalling in breast cancer\",\n",
            "        \"journal\": \"Nature\",\n",
            "        \"journal_url\": \"https://www.nature.com/articles/nature18003\",\n",
            "        \"year\": \"2016\",\n",
            "        \"abstract\": \"Somatic mutations have been extensively characterized in breast cancer, but the effects of these genetic alterations on the proteomic landscape remain poorly understood. Here we describe quantitative mass-spectrometry-based proteomic and phosphoproteomic analyses of 105 genomically annotated breast cancers, of which 77 provided high-quality data. Integrated analyses provided insights into the somatic cancer genome including the consequences of chromosomal loss, such as the 5q deletion characteristic of basal-like breast cancer. Interrogation of the 5q trans-effects against the Library of Integrated Network-based Cellular Signatures, connected loss of CETN3 and SKP1 to elevated expression of epidermal growth factor receptor (EGFR), and SKP1 loss also to increased SRC tyrosine kinase. Global proteomic data confirmed a stromal-enriched group of proteins in addition to basal and luminal clusters, and pathway analysis of the phosphoproteome identified a G-protein-coupled receptor cluster that was not readily identified at the mRNA level. In addition to ERBB2, other amplicon-associated highly phosphorylated kinases were identified, including CDK12, PAK1, PTK2, RIPK2 and TLK2. We demonstrate that proteogenomic analysis of breast cancer elucidates the functional consequences of somatic mutations, narrows candidate nominations for driver genes within large deletions and amplified regions, and identifies therapeutic targets.\",\n",
            "        \"full_text\": [\n",
            "          \"Proteogenomics connects somatic mutations to signaling in breast cancer Summary Somatic mutations have been extensively characterized in breast cancer, but the effects of these genetic alterations on the proteomic landscape remain poorly understood. We\",\n",
            "          \"on the proteomic landscape remain poorly understood. We describe quantitative mass spectrometry-based proteomic and phosphoproteomic analyses of 105 genomically annotated breast cancers of which 77 provided high-quality data. Integrated analyses allowed\",\n",
            "          \"77 provided high-quality data. Integrated analyses allowed insights into the somatic cancer genome including the consequences of chromosomal loss, such as the 5q deletion characteristic of basal-like breast cancer. The 5q trans effects were interrogated\",\n",
            "          \"breast cancer. The 5q trans effects were interrogated against the Library of Integrated Network-based Cellular Signatures, thereby connecting CETN3 and SKP1 loss to elevated expression of EGFR, and SKP1 loss also to increased SRC. Global proteomic data\",\n",
            "          \"and SKP1 loss also to increased SRC. Global proteomic data confirmed a stromal-enriched group in addition to basal and luminal clusters and pathway analysis of the phosphoproteome identified a G Protein-coupled receptor cluster that was not readily\",\n",
            "          \"a G Protein-coupled receptor cluster that was not readily identified at the mRNA level. Besides ERBB2, other amplicon-associated, highly phosphorylated kinases were identified, including CDK12, PAK1, PTK2, RIPK2 and TLK2. We demonstrate that proteogenomic\",\n",
            "          \"PAK1, PTK2, RIPK2 and TLK2. We demonstrate that proteogenomic analysis of breast cancer elucidates functional consequences of somatic mutations, narrows candidate nominations for driver genes within large deletions and amplified regions, and identifies\",\n",
            "          \"within large deletions and amplified regions, and identifies therapeutic targets. Introduction A central deficiency in our knowledge of cancer concerns how genomic changes drive the proteome and phosphoproteome to execute phenotypic characteristics. The\",\n",
            "          \"and phosphoproteome to execute phenotypic characteristics. The initial proteomic characterization in the TCGA breast study was performed using reversed phase protein arrays; however this approach is restricted by antibody availability. To provide greater\",\n",
            "          \"is restricted by antibody availability. To provide greater analytical breadth, the NCI Clinical Proteomic Tumor Analysis Consortium (CPTAC) is analyzing the proteomes of genome-annotated TCGA tumor specimens using mass spectrometry. Herein we describe\",\n",
            "          \"tumor specimens using mass spectrometry. Herein we describe integrated proteogenomic analyses of TCGA breast cancer samples representing the four principal mRNA-defined breast cancer intrinsic subtypes. Proteogenomic analysis of TCGA samples 105 breast\",\n",
            "          \"subtypes. Proteogenomic analysis of TCGA samples 105 breast tumors previously characterized by the TCGA were selected for proteomic analysis after histopathological documentation (Supplementary Tables 1 and 2). The cohort included a balanced\",\n",
            "          \"(Supplementary Tables 1 and 2). The cohort included a balanced representation of PAM50-defined intrinsic subtypes including 25 basal-like, 29 luminal A, 33 luminal B, and 18 HER2 (ERBB2)-enriched tumors, along with 3 normal breast tissue samples. Samples\",\n",
            "          \"tumors, along with 3 normal breast tissue samples. Samples were analyzed by high-resolution, accurate mass, tandem mass spectrometry (MS) that included extensive peptide fractionation and phosphopeptide enrichment (Extended Data Fig. 1a). An isobaric\",\n",
            "          \"phosphopeptide enrichment (Extended Data Fig. 1a). An isobaric peptide labeling approach (iTRAQ) was employed to quantify protein and phosphosite levels across samples, with 37 iTRAQ 4-plexes analyzed in total. A total of 15,369 proteins (12,405 genes)\",\n",
            "          \"analyzed in total. A total of 15,369 proteins (12,405 genes) and 62,679 phosphosites were confidently identified with 11,632 proteins/tumor and 26,310 phosphosites/tumor on average (Supplementary Tables 3, 4 and Supplementary Methods). After filtering for\",\n",
            "          \"Tables 3, 4 and Supplementary Methods). After filtering for observation in at least a quarter of the samples (Supplementary Methods, Extended Data Fig. 1b) 12,553 proteins (10,062 genes) and 33,239 phosphosites, with their relative abundances quantified\",\n",
            "          \"33,239 phosphosites, with their relative abundances quantified across tumors, were used in subsequent analyses in this study. Stable longitudinal performance and low technical noise were demonstrated by repeated, interspersed analyses of a single batch of\",\n",
            "          \"by repeated, interspersed analyses of a single batch of patient-derived luminal and basal breast cancer xenograft samples (Extended Data Fig. 1d,e). Due to the heterogeneous nature of breast tumors, and because proteomic analyses were performed on tumor\",\n",
            "          \"tumors, and because proteomic analyses were performed on tumor fragments that were different from those used in the genomic analyses, rigorous pre-specified sample and data QC metrics were implemented (Supplementary Discussion and Extended Data Figures 2,\",\n",
            "          \"(Supplementary Discussion and Extended Data Figures 2, 3). Extensive analyses concluded that 28 of the 105 samples were compromised by protein degradation. These samples were excluded from further analysis with subsequent informatics focused on the 77\",\n",
            "          \"further analysis with subsequent informatics focused on the 77 tumor samples and three biological replicates. Genome and transcriptomic variation was observed at the peptide level by searching MS/MS spectra not matched to RefSeq against a patient-specific\",\n",
            "          \"MS/MS spectra not matched to RefSeq against a patient-specific sequence database (Fig. 1a). The database was constructed using the QUILTS software package leveraging RefSeq gene models based on whole exome and RNA-seq data generated from portions of the\",\n",
            "          \"on whole exome and RNA-seq data generated from portions of the same tumors and matched germline DNA (Fig. 1a, Supplementary Table 5). While these analyses detected a number of single amino-acid variants (SAAVs), frameshifts, and splice junctions,\",\n",
            "          \"amino-acid variants (SAAVs), frameshifts, and splice junctions, including splice isoforms that had been detected as only single transcript reads by RNA-seq (Fig. 1b, Supplementary Table 5), the number of genomic and transcriptomic variants that were\",\n",
            "          \"5), the number of genomic and transcriptomic variants that were confirmed as peptides by MS was low (Supplementary Discussion). Sparse detection of individual genomic variants by peptide sequencing has been noted in our previous studies and reflects\",\n",
            "          \"sequencing has been noted in our previous studies and reflects limited coverage at the single amino-acid level with current technology. However quantitative MS analysis of multiple peptides for each protein is used to reliably infer overall protein\",\n",
            "          \"for each protein is used to reliably infer overall protein levels. This is an advantage for MS since antibody-based protein expression analysis is typically based on a single epitope. To illustrate this capability in the current data set an initial\",\n",
            "          \"illustrate this capability in the current data set an initial analysis of three frequently mutated genes in breast cancer (TP53, PIK3CA, and GATA3) and three clinical biomarkers (ER, PGR, and ERBB2) was conducted (Fig. 1c, Supplementary Table 6, 7 and\",\n",
            "          \"and ERBB2) was conducted (Fig. 1c, Supplementary Table 6, 7 and Supplementary Discussion). As expected, TP53 missense mutations were associated with elevated MS-based protein levels, as observed by RPPA (Reverse Phase Protein Array), especially in\",\n",
            "          \"observed by RPPA (Reverse Phase Protein Array), especially in basal-like breast cancer. TP53 nonsense and frame-shift mutations were associated with a decrease in TP53 protein levels that was particularly striking in the MS data. In contrast, the mostly\",\n",
            "          \"particularly striking in the MS data. In contrast, the mostly C-terminal GATA3 frame-shift alterations did not result in decreased protein expression when measured by the median of all GATA3 peptides, suggesting these proteins are expressed despite\",\n",
            "          \"GATA3 peptides, suggesting these proteins are expressed despite truncation. No consistent effect of somatic PIK3CA mutation was observed at the level of protein expression. Good correlations between RNA-seq and MS-protein expression levels were found for\",\n",
            "          \"between RNA-seq and MS-protein expression levels were found for ESR1 (r=0.74), PGR (r=0.74), ERBB2 (r=0.84) and GATA3 (r=0.83) with moderate correlations observed for PIK3CA (r=0.45) and TP53 (r=0.36). Lower TP53 protein abundance levels compared to mRNA\",\n",
            "          \"(r=0.36). Lower TP53 protein abundance levels compared to mRNA levels were especially prevalent in luminal tumors, suggesting post-transcriptional regulatory mechanisms such as proteasomal degradation. To explore this hypothesis a search was made for E3\",\n",
            "          \"To explore this hypothesis a search was made for E3 ligases that showed negative correlation to p53 protein (Supplementary Table 8). These analyses identified UBE3A (r=\\u22120.42; adj. pval=0.05) (Extended Data Fig. 4a), an established TP53 E3 ligase. In\",\n",
            "          \"(Extended Data Fig. 4a), an established TP53 E3 ligase. In comparing CNA, RNA, and protein levels for GATA3, copy number gains in chromosome 10q were anti-correlated with RNA and protein levels in basal-like tumors. This observation prompted a search for\",\n",
            "          \"in basal-like tumors. This observation prompted a search for other gains or losses that were anticorrelated with RNA and/or protein levels (see Extended Data Fig. 4b for further analyses). Overall, six genes were identified that significantly\",\n",
            "          \"Overall, six genes were identified that significantly anti-correlated at an FDR<0.05 on both RNA and protein level to their CNA signals (Extended Data Fig. 4b). GATA3 amplification on 10q in basal-like breast cancer showed the strongest anti-correlation,\",\n",
            "          \"basal-like breast cancer showed the strongest anti-correlation, followed by the hexosamine and glycolysis pathway enzymes GFPT2 and HK3, which are upregulated in basal-like breast cancer despite being subjected to frequent chromosomal deletion on 5q.\",\n",
            "          \"despite being subjected to frequent chromosomal deletion on 5q. Global analysis of the correlation of mRNA-to-protein yielded a median Pearson value of r=0.39, with 6,135 out of 9,302 protein/mRNA pairs (66.0%) correlating significantly at an FDR<0.05\",\n",
            "          \"pairs (66.0%) correlating significantly at an FDR<0.05 (Extended Data Fig. 4c, Supplementary Table 9 and Supplementary Discussion). Similar to the colon cancer analysis metabolic functions such as amino acid, sugar and fatty acid metabolism were found to\",\n",
            "          \"as amino acid, sugar and fatty acid metabolism were found to be enriched among positively correlated genes whereas ribosomal, RNA polymerase and mRNA splicing functions were negatively correlated. Overall these analyses demonstrate the utility of global\",\n",
            "          \"Overall these analyses demonstrate the utility of global proteome correlation analysis for both confirmation of suspected regulatory mechanisms and identification of candidate regulators meriting further investigation. Copy Number Alterations To determine\",\n",
            "          \"further investigation. Copy Number Alterations To determine the consequences of CNA on mRNA, protein and phosphoprotein abundance, both in \\u201ccis\\u201d on genes within the aberrant locus and in \\u201ctrans\\u201d on genes encoded elsewhere, univariate correlation analysis\",\n",
            "          \"on genes encoded elsewhere, univariate correlation analysis was used as previously described. A total of 7,776 genes with CNA, mRNA and protein measurements were analyzed by calculating Pearson correlation and associated statistical significance\",\n",
            "          \"Pearson correlation and associated statistical significance (Benjamini-Hochberg corrected p-value) for all possible CNA-mRNA and CNA-protein pairs (Fig. 2a, Supplementary Table 10, Extended Data Fig. 5a, see Methods). For the phosphoproteome, 4,472\",\n",
            "          \"Data Fig. 5a, see Methods). For the phosphoproteome, 4,472 CNA-phosphoprotein pairs were analyzed (Extended Data Fig. 5b). Significant positive correlations (cis) were observed for 64% of all CNA-mRNA, 31% of all CNA-protein, and 20% of all\",\n",
            "          \"for 64% of all CNA-mRNA, 31% of all CNA-protein, and 20% of all CNA-phosphoprotein pairs Fig. 2b. Proteins and phosphoproteins correlated in cis to CNA were, for the most part, a subset of the cis effects observed in mRNA-to-CNA correlation (Fig. 2b,\",\n",
            "          \"the cis effects observed in mRNA-to-CNA correlation (Fig. 2b, Supplementary Table 10). The fractional difference of well-annotated oncogenes and tumor suppressor genes among the significantly cis-correlated CNA-to-mRNA and -protein gene pairs was\",\n",
            "          \"cis-correlated CNA-to-mRNA and -protein gene pairs was analyzed. Based on a reference list of 487 oncogenes and tumor suppressors (Supplementary Table 10), these cancer relevant genes occur 37.6% more frequently in the subset of genes that correlate both\",\n",
            "          \"more frequently in the subset of genes that correlate both on CNA/mRNA and CNA/protein levels than in the subset that only correlate on CNA/mRNA but not on CNA/protein level (Fisher exact p-value=0.02). This suggests that CNA events with a tumor promoting\",\n",
            "          \"This suggests that CNA events with a tumor promoting outcome more likely lead to cis regulatory effects on both the protein and mRNA level, whereas CNA events with no documented role in tumorigenesis are more likely to be neutralized on the protein level\",\n",
            "          \"are more likely to be neutralized on the protein level than on the RNA level. Trans effects (Fig. 2a) appear as vertical bands, with accompanying frequency histograms (in blue) highlighting \\u201chot spots\\u201d of significant trans effects. Using a minimum\",\n",
            "          \"\\u201chot spots\\u201d of significant trans effects. Using a minimum threshold of 50 trans-affected genes, 68% of the tested genes were associated with trans effects on the mRNA level, whereas only 13% were associated with effects on the protein level and 8% on the\",\n",
            "          \"were associated with effects on the protein level and 8% on the phosphoprotein level. Importantly, CNA-protein correlations appeared to be a reduced representation of CNA-mRNA correlations. Furthermore, for many CNA regions correlations were more\",\n",
            "          \"Furthermore, for many CNA regions correlations were more directionally uniform on the protein level than on the mRNA level. CNA regions exhibiting the most trans associations at the protein level were found on chromosomes 5q (LOH in basal; gain in LumB),\",\n",
            "          \"were found on chromosomes 5q (LOH in basal; gain in LumB), 10p (gain in basal), 12 (gain in basal), 16q (LumA deletion), 17q (LumB amplification), and 22q (LOH in luminal and basal) (Extended Data Fig. 5a). Trans associations are not necessarily direct\",\n",
            "          \"Data Fig. 5a). Trans associations are not necessarily direct consequences of the chromosomal aberration. For example since 5q loss occurs in at least 50% of basal-like breast cancers, many of the trans effects involve genes that mark the basal-subtype. To\",\n",
            "          \"the trans effects involve genes that mark the basal-subtype. To identify candidate driver genes whose copy number alterations are direct drivers of trans effects, results were compared with functional knock-down data on 3,797 genes in the Library of\",\n",
            "          \"functional knock-down data on 3,797 genes in the Library of Integrated Cellular Signatures (LINCS) database (http://www.lincsproject.org/). For any given gene with copy number alterations (\\u201cCNA-gene\\u201d), sets of genes were identified corresponding to\",\n",
            "          \"(\\u201cCNA-gene\\u201d), sets of genes were identified corresponding to proteins that changed where there was gain (\\u201cCNA-gain trans gene set\\u201d) or loss (\\u201cCNA-loss trans gene set\\u201d). These gene sets were then compared to the effects of gene knock down in the LINCS\",\n",
            "          \"then compared to the effects of gene knock down in the LINCS database (see Methods). Queries for 502 different CNA-genes meeting the criteria defined above identified 10 CNA-genes that could be functionally connected to both CNA-gain and CNA-loss trans\",\n",
            "          \"be functionally connected to both CNA-gain and CNA-loss trans protein-level effects (Extended Data Fig. 5c, Supplementary Table 11). A permutation-based approach implemented to test significance (see Supplementary Methods) yielded an FDR < 0.05 for 10\",\n",
            "          \"(see Supplementary Methods) yielded an FDR < 0.05 for 10 genes affected by both CNA gains and losses (Fig. 2c). These proteins were defined as potential regulatory candidates for the CNA trans effects observed on the proteome level in this study, since in\",\n",
            "          \"effects observed on the proteome level in this study, since in a gene-dependent manner on average 17% of these trans effects were consistent with the knockdown profiles. Notably, the established oncogenic receptor tyrosine kinase ERBB2 was functionally\",\n",
            "          \"oncogenic receptor tyrosine kinase ERBB2 was functionally connected only to CNA gain trans effects (Supplementary Table 11). The E3 ligase SKP1 and the ribonucleoprotein export factor CETN3, both located on chromosome arm 5q with frequent losses in\",\n",
            "          \"both located on chromosome arm 5q with frequent losses in basal-like breast cancer and less frequent gains in luminal B breast cancer, were detected as potential regulators affecting the expression of the tyrosine kinase and therapeutic target EGFR, and\",\n",
            "          \"of the tyrosine kinase and therapeutic target EGFR, and SKP1 also was linked to SRC (Extended Data Fig. 5d). Another potential regulator, FBXO7, (a substrate recognition component of the SCF (SKP1-CUL1-F-box protein)-type E3 ubiquitin ligase complex), was\",\n",
            "          \"protein)-type E3 ubiquitin ligase complex), was affected mostly by LOH events on chromosome 22q. Interestingly, in a recent human interaction proteome study SKP1 and FBXO7 were listed as interaction partners. Clustering and Network analysis\",\n",
            "          \"listed as interaction partners. Clustering and Network analysis Transcriptional profiling has converged on four major breast cancer subtypes: Luminal A and B, basal and HER2-enriched. To investigate the extent to which the PAM50 \\u201cintrinsic\\u201d breast cancer\",\n",
            "          \"the extent to which the PAM50 \\u201cintrinsic\\u201d breast cancer classification scheme is reflected or refined on the proteome level in the CPTAC samples, clustering analyses were first restricted to the reduced set of PAM50 genes. When RNA data for the 50 PAM50\",\n",
            "          \"the reduced set of PAM50 genes. When RNA data for the 50 PAM50 genes were clustered directly (without using a classifier), the clustering was similar to the TCGA PAM50 annotation (second annotation bar in Fig. 3a). Restricting both the RNA and proteome\",\n",
            "          \"bar in Fig. 3a). Restricting both the RNA and proteome data to the set of 35 PAM50 genes observed in the proteome produced a similar result (bottom two annotation bars in Fig. 3a), and all the major PAM50 groups were recapitulated in the proteome almost\",\n",
            "          \"major PAM50 groups were recapitulated in the proteome almost as well as in the RNA data. This indicates that although different tissue sections of the same tumors were used for RNA-seq and protein analysis, very similar subtype-defining features can be\",\n",
            "          \"protein analysis, very similar subtype-defining features can be observed in both data types. Global proteome and phosphoproteome data were then used to identify proteome subtypes in an unsupervised manner. Consensus clustering identified basal-enriched,\",\n",
            "          \"manner. Consensus clustering identified basal-enriched, luminal-enriched, and stromal-enriched clusters (Extended Data Figs. 6a\\u2013d, 7a). Unlike the clustering observed with PAM50 genes, mRNA-defined HER2-enriched tumors were distributed across these three\",\n",
            "          \"HER2-enriched tumors were distributed across these three proteomic subgroups. The basal-enriched and luminal-enriched groups showed a strong overlap with the mRNA-based PAM50 basal-like and luminal subgroups, whereas stromal-enriched proteome subtype\",\n",
            "          \"luminal subgroups, whereas stromal-enriched proteome subtype represented a mix of all PAM50 mRNA-based subtypes, and has a significantly enriched stromal signature (Extended Data Fig. 3e). Among the stromal-enriched tumors there was strong representation\",\n",
            "          \"the stromal-enriched tumors there was strong representation of reactive type I tumors as classified by RPPA (Supplementary Table 12), showing agreement between the RPPA and mass spectrometry-based protein analyses for the detection of a tumor subgroup\",\n",
            "          \"protein analyses for the detection of a tumor subgroup characterized by stromal gene expression. Since the basal- and luminal-enriched proteome subgroups are coherent, pathway analyses were conducted on these two subtypes, using the stromal-enriched\",\n",
            "          \"conducted on these two subtypes, using the stromal-enriched subgroup as a control to assess specificity. (Fig. 3c, Extended Data Fig. 7b, Supplementary Table 13). The luminal-enriched subgroup was exclusively enriched for estradiol and ESR1-driven gene\",\n",
            "          \"was exclusively enriched for estradiol and ESR1-driven gene sets. In contrast, multiple gene sets were enriched and upregulated specifically in the basal-like tumors. Particularly extensive basal-like enrichment was seen for MYC target genes; for cell\",\n",
            "          \"basal-like enrichment was seen for MYC target genes; for cell cycle, checkpoint, and DNA repair pathways including regulators AURKA/B, ATM, ATR, CHEK1/2, and BRCA1/2; and for immune response/inflammation, including T-cell, B-cell, and neutrophil\",\n",
            "          \"response/inflammation, including T-cell, B-cell, and neutrophil signatures. The complementarity of transcriptional, proteomic, and phosphoproteomic data was also highlighted in these analyses (Extended Data Fig. 7c, d). Using phosphorylation status as a\",\n",
            "          \"(Extended Data Fig. 7c, d). Using phosphorylation status as a proxy for activity, phosphoproteome profiling can theoretically be used to develop a signaling pathway-based cancer classification. K-means consensus clustering was therefore performed on\",\n",
            "          \"K-means consensus clustering was therefore performed on pathways derived from single sample GSEA analysis of phosphopeptide data (Methods, Supplementary Tables 14 and 15). Of four robustly segregated groups, subgroups 2 and 3 substantially recapitulated\",\n",
            "          \"groups, subgroups 2 and 3 substantially recapitulated the stromal- and luminal-enriched proteomic subgroups, respectively (Fig. 3d, Extended Data Fig. 8a). Subgroup 4 included a majority of tumors from the basal-enriched proteomic subgroup, but was\",\n",
            "          \"of tumors from the basal-enriched proteomic subgroup, but was admixed particularly with luminal-enriched samples. This subgroup was defined by high levels of cell cycle and checkpoint activity. All basal and a majority of non-basal samples in this\",\n",
            "          \"activity. All basal and a majority of non-basal samples in this subgroup had TP53 mutations. Consistent with high levels of cell cycle activity, a multivariate kinase-to-phosphosite abundance regression analysis highlighted CDK1 as one of the most highly\",\n",
            "          \"regression analysis highlighted CDK1 as one of the most highly connected kinases in this study (Extended Data Fig. 8b, Supplementary Table 16). Subgroup 1 was a novel subgroup defined exclusively in the phosphopeptide/pathway activity domain, with no\",\n",
            "          \"in the phosphopeptide/pathway activity domain, with no enrichment for either proteomic or PAM50 subtypes. It was defined by G-protein, G-protein coupled receptor, and inositol phosphate metabolism signatures, as well as ionotropic glutamate signaling\",\n",
            "          \"signatures, as well as ionotropic glutamate signaling (Fig. 3d). Co-expression patterns among genes/proteins across different subgroups were also analyzed using a Joint Random Forest (JRF) method that identified network modules, such as an MMP9 module,\",\n",
            "          \"method that identified network modules, such as an MMP9 module, with different interaction patterns between basal-enriched and luminal-enriched subgroups. These latter patterns appeared specific to the proteome-level data (Extended Data Fig. 8 c\\u2013f,\",\n",
            "          \"specific to the proteome-level data (Extended Data Fig. 8 c\\u2013f, Supplementary Table 17 and Supplementary Methods). Phosphosite markers in PIK3CA and TP53 mutant breast cancer TP53 and PIK3CA are the most recurrently mutated genes in breast cancer, with\",\n",
            "          \"are the most recurrently mutated genes in breast cancer, with frequencies for PIK3CA at 43% in luminal tumors and for TP53 at 84% in basal-like tumors. Most of the PIK3CA missense mutations were gain of function mutations and therefore expected to lead to\",\n",
            "          \"gain of function mutations and therefore expected to lead to an activation of the PI3K signaling cascade, but the extent to which this occurs has been controversial and there is uncertainty which pathway components are effectors. Marker selection analysis\",\n",
            "          \"pathway components are effectors. Marker selection analysis was therefore performed for upregulated phosphosites in PIK3CA-mutated tumors. In total, 62 phosphosites were identified that were positively associated with PIK3CA mutation (FDR<0.05), including\",\n",
            "          \"associated with PIK3CA mutation (FDR<0.05), including the kinases RPS6KA5 and EIF2AK4 (Extended Data Fig. 9a, Supplementary Table 18). Calculating the average phosphorylation signal of these marker phosphosites provided a read-out for PI3K pathway\",\n",
            "          \"these marker phosphosites provided a read-out for PI3K pathway activity in PIK3CA-mutated tumors, with 15 of the 26 mutated tumors (58%) exhibiting an activated PIK3CA mutation signature. Of note, the identified PIK3CA mutant phosphoproteome signature was\",\n",
            "          \"the identified PIK3CA mutant phosphoproteome signature was activated in all tumors harboring helical domain PIK3CA mutations but only 2 of 10 tumors harboring kinase domain mutations. To test if the identified differences in the phosphoproteome of PI3K\",\n",
            "          \"if the identified differences in the phosphoproteome of PI3K mutant versus wild-type tumors could be explained by mutation of PIK3CA, the tumor data were compared to phosphosite signatures derived from isogenic PIK3CA mutant cell lines (Extended Data Fig.\",\n",
            "          \"from isogenic PIK3CA mutant cell lines (Extended Data Fig. 9b, Supplementary Table 18). There was an enrichment of signatures derived from helical domain-mutated isogenic cell lines, but not from kinase domain-mutated cells, supporting the observations in\",\n",
            "          \"kinase domain-mutated cells, supporting the observations in primary tumors. The same strategy was used to identify phosphorylation signaling events connected to TP53 mutation. A total of 56 phosphosites upregulated in TP53 mutant tumors were identified\",\n",
            "          \"phosphosites upregulated in TP53 mutant tumors were identified that were independent of basal-like subtype association (Extended Data Fig. 9c, Supplementary Table 18). Using the average phosphorylation signal of these marker phosphosites as a proxy for\",\n",
            "          \"signal of these marker phosphosites as a proxy for TP53 mutation-driven cell cycle control, 22 of 41 mutated tumors (54%) showed upregulated signals. This TP53 mutant phosphosignature was somewhat enhanced in tumors in which mutations occurred almost\",\n",
            "          \"somewhat enhanced in tumors in which mutations occurred almost exclusively in the DNA binding region compared to those with non-sense/frameshift mutations. In addition to the well-described checkpoint kinase CHEK2, significantly upregulated phosphosites\",\n",
            "          \"checkpoint kinase CHEK2, significantly upregulated phosphosites were identified for the kinases MASTL and EEF2K in TP53-mutated tumors. Single sample GSEA analysis of isogenic p53-mutant phosphosignatures showed an enrichment of a phosphosignature derived\",\n",
            "          \"showed an enrichment of a phosphosignature derived from R273H mutated isogenic cells (Extended Data Fig. 9d), confirming the pronounced effect of missense mutations in the DNA-binding region on phosphorylation pathways. Identification of gene\",\n",
            "          \"region on phosphorylation pathways. Identification of gene amplification and breast cancer subtype-specific activated kinases in human breast cancer CNA spans many driver gene candidates and RNA expression has been frequently used to narrow candidate\",\n",
            "          \"and RNA expression has been frequently used to narrow candidate nominations. Proteogenomic analysis should further promote this nomination process. In this candidate refinement a focus on protein kinases is warranted, since many are drug targets. An\",\n",
            "          \"protein kinases is warranted, since many are drug targets. An in-depth proteogenomic pipeline was developed that flagged kinases, expression levels of which were at least 1.5 interquartile ranges higher than the median (Supplemental Table 19). A\",\n",
            "          \"ranges higher than the median (Supplemental Table 19). A proteogenomic circos-like plot (termed a \\u201cpircos\\u201d plot) was used to map these outlier kinase values onto the genome (Fig. 4a,b, Extended Data Fig. 10a). The ERBB2 locus showed the strongest effect\",\n",
            "          \"Data Fig. 10a). The ERBB2 locus showed the strongest effect of increased phosphoprotein levels associated with gene amplification-driven RNA and protein over expression (Fig. 4a). CDK12 is a positive transcriptional regulator of homologous recombination\",\n",
            "          \"positive transcriptional regulator of homologous recombination repair genes with its partner Cyclin K, and is often encompassed by the ERBB2 amplicon. This gene was also found to be upregulated at the RNA, protein and phosphosite level indicating that\",\n",
            "          \"at the RNA, protein and phosphosite level indicating that CDK12 is highly active in the majority of ERBB2 positive tumors (Fig. 4a). The analysis of the ERBB2 amplicon also uncovered co-outlier phosphorylation status for MED1, GRB7, MSL1, CASC3 and TOP2A,\",\n",
            "          \"phosphorylation status for MED1, GRB7, MSL1, CASC3 and TOP2A, all previously described in association with ERBB2 amplification. To better understand the downstream effects of ERBB2 amplification, additional phosphosite outliers were identified in 41 known\",\n",
            "          \"additional phosphosite outliers were identified in 41 known ERBB2 signaling genes for the 15 samples that had ERBB2 phosphosite outlier expression (Extended Data Fig. 10b). These canonical findings stimulated a proteogenomic analysis to identify\",\n",
            "          \"findings stimulated a proteogenomic analysis to identify additional outlier kinases in the breast cancer genome. A proteogenomic dissection of chromosome 11q based on PAK1 amplification (Fig. 4b,c), a breast cancer driver kinase, illustrated that PAK1 is\",\n",
            "          \"4b,c), a breast cancer driver kinase, illustrated that PAK1 is hyperphosphorylated in PAK1 amplified tumors, along with CLNS1A, RFS1 and GAB2. Additional examples of outlier kinases included PTK2 and RIPK2 in association with amplification of chromosome\",\n",
            "          \"PTK2 and RIPK2 in association with amplification of chromosome 8q (Fig. 4c; Extended Data Fig. 10a,c). PAK1 and TLK2 (17q23) appear to be luminal breast cancer specific events Fig. 4c; Extended Data Fig. 10c). To further examine whether outlier kinases\",\n",
            "          \"Data Fig. 10c). To further examine whether outlier kinases were breast cancer subtype-specific, independent of amplification status, the BH-corrected probability was calculated of finding that number of phosphosite outliers within a subtype, given the\",\n",
            "          \"that number of phosphosite outliers within a subtype, given the total number of outliers across all subtypes, the subtype sample size and the total sample size. (Fig. 4d). These analyses led to the expected identification of ERBB2 in the HER2-enriched\",\n",
            "          \"to the expected identification of ERBB2 in the HER2-enriched subtype at the 5% FDR level, as well as the new finding of CDC42BPG (MRGKG), an effector kinase for RHO-family GTPases. In basal-like breast cancer, two kinases, PRKDC and SPEG, were significant\",\n",
            "          \"breast cancer, two kinases, PRKDC and SPEG, were significant at the 5% FDR level. PRKDC is a non-homologous end-joining (NHEJ) factor that can be phosphorylated by ATM kinase, and is therefore a logical finding in this disease subset. However SPEG, a\",\n",
            "          \"a logical finding in this disease subset. However SPEG, a kinase associated with severe dilated cardiomyopathy when suppressed, has not been previously reported in association with breast cancer. A larger number of subtype-specific kinases were detected\",\n",
            "          \"A larger number of subtype-specific kinases were detected at the 10% FDR level, several of which have recently described relevance in breast cancer, including PRKD3 in basal-like breast cancer, the LKB-regulated SIK3 in luminal A breast cancer and CDK13\",\n",
            "          \"the LKB-regulated SIK3 in luminal A breast cancer and CDK13 in luminal B breast cancer, which, similar to CDK12, can interact with Cyclin K. Discussion The analytic breadth and depth of proteomic and phosphoproteomic analyses displayed in this study\",\n",
            "          \"proteomic and phosphoproteomic analyses displayed in this study demonstrates the strengths of mass spectrometry-based proteomics, but also some of the limitations inherent in proteolytic peptide sequencing (see Supplementary Discussion). An example of how\",\n",
            "          \"sequencing (see Supplementary Discussion). An example of how high-dimensional proteomic analysis provides insight into unresolved genomic issues concerns the study of loss of the long arm of chromosome 5 (5q). Analysis of RNA and protein correlations\",\n",
            "          \"of chromosome 5 (5q). Analysis of RNA and protein correlations narrowed the list of potential trans-deregulated proteins. Orthogonal candidate screening using functional genomics methods identified loss of CETN3 and SKP1 as potential transregulators, with\",\n",
            "          \"loss of CETN3 and SKP1 as potential transregulators, with upregulation of EGFR as a downstream consequence in basal-like breast cancers. While further experimental evidence must be sought for these proposed regulatory relationships, SKP1/Cullin complex\",\n",
            "          \"these proposed regulatory relationships, SKP1/Cullin complex has already been linked to EGFR activation in Glioma. Unfortunately EGFR targeting has not to date proven to be effective therapy in basal-like breast cancer. This might be due to the fact the\",\n",
            "          \"in basal-like breast cancer. This might be due to the fact the SKP1 loss deregulates multiple targets requiring a much broader inhibitory strategy. It is recognized that PIK3CA mutations do not strongly activate canonical downstream effectors. Mass\",\n",
            "          \"do not strongly activate canonical downstream effectors. Mass spectrometry-based phosphoproteomics provides an opportunity for unbiased examination of downstream signaling events consequent upon PIK3CA mutational activation. These studies revealed that\",\n",
            "          \"upon PIK3CA mutational activation. These studies revealed that common PIK3CA mutations affect a large number of targets with diverse functionalities including the kinases RPS6KA5 and EIF2AK4. Thus, the data and analyses reported here extend our knowledge\",\n",
            "          \"Thus, the data and analyses reported here extend our knowledge of the effectors that promote tumorigenesis in response to constitutive activation of PI3 kinase. Similarly, TP53 mutation-associated phosphopeptides point towards novel functionalities,\",\n",
            "          \"phosphopeptides point towards novel functionalities, including regulation of the kinases MASTL and EEF2K. A central goal in breast cancer research has been the identification of druggable kinases beyond HER2. Candidate genes that exhibited similar gene\",\n",
            "          \"beyond HER2. Candidate genes that exhibited similar gene amplification-driven proteogenomic patterns to HER2 included CDK12, TLK2, PAK1 and RIPK2. The proteogenomic link with gene amplification was particularly strong for CDK12, in keeping with its\",\n",
            "          \"was particularly strong for CDK12, in keeping with its location in the ERBB2 amplicon, while the strengths of correlation between DNA amplification, RNA, protein and phosphoprotein for the other examples were more variable. The presence of activated CDK12\",\n",
            "          \"examples were more variable. The presence of activated CDK12 in the ERBB2 amplicon might explain why tumors arising in BRCA1 carriers are usually ERBB2 negative. As a positive transcriptional regulator of BRCA1 and multiple FANC family members, CDK12\",\n",
            "          \"regulator of BRCA1 and multiple FANC family members, CDK12 promotes DNA repair by homologous recombination. CDK12 amplification would, therefore, oppose the functional effects of BRCA1 haploinsufficiency during tumor evolution. Overall, multiple outlier\",\n",
            "          \"during tumor evolution. Overall, multiple outlier kinases generate testable therapeutic hypotheses for which enabling inhibitors are in development. For example PAK1 has recently been confirmed to be a therapeutic target and poor prognosis factor in\",\n",
            "          \"to be a therapeutic target and poor prognosis factor in luminal breast cancer. Although incomplete outcome data and the remarkable heterogeneity of breast cancer are additional relevant constraints, the number of TCGA specimens analyzed here is\",\n",
            "          \"constraints, the number of TCGA specimens analyzed here is insufficient to support conclusive clinical correlations. Only 8 deaths occurred among the 77 patients, which are too few samples to provide sufficient statistical power for association analysis.\",\n",
            "          \"provide sufficient statistical power for association analysis. Adequately powered MS-based clinical investigation will require targeted approaches, especially given the highly limited amount of patient material available from clinical trials and the\",\n",
            "          \"of patient material available from clinical trials and the mostly formalin-fixed nature of the specimens. The current analysis is therefore centered on biological findings and correlations, with orthogonal validation and false discovery concerns addressed\",\n",
            "          \"orthogonal validation and false discovery concerns addressed through an examination of cell-line databases of the effects of individual gene perturbations. Typical of a multi-tiered analysis of this complexity, there are many hypotheses to test, and many\",\n",
            "          \"of this complexity, there are many hypotheses to test, and many findings that require further investigation. In conclusion, this study provides a high-quality proteomic resource for human breast cancer investigation and illustrates technologies and\",\n",
            "          \"breast cancer investigation and illustrates technologies and analytical approaches that provide an important new opportunity to connect the genome to the proteome. Larger-scale exploration of discovery proteomics in the clinical setting will require\",\n",
            "          \"of discovery proteomics in the clinical setting will require improvements in clinical investigation, including acquisition of adequate amounts of optimally collected tumor tissue both before and during therapy as well as advances in MS proteomics to\",\n",
            "          \"and during therapy as well as advances in MS proteomics to reduce sample input and increase sensitivity for low abundance proteins and modified peptides. Extended Data Experimental and data analysis workflows and longitudinal data generation quality\",\n",
            "          \"analysis workflows and longitudinal data generation quality control a, iTRAQ 4-plex global proteome and phosphoproteome analysis workflow. 105 TCGA breast tumors were analyzed in 35 iTRAQ 4-plex experiments (plus 1 replicate and 1 normal sample\",\n",
            "          \"iTRAQ 4-plex experiments (plus 1 replicate and 1 normal sample experiment), with three tumors of different subtypes compared to a fourth common internal reference sample in each experiment. The reference sample comprised 10 individual tumors of each of\",\n",
            "          \"The reference sample comprised 10 individual tumors of each of the 4 major breast cancer intrinsic subtypes and served as an internal standard for all proteins and phosphoproteins quantified in this study. Each iTRAQ MS/MS spectrum measures a peptide from\",\n",
            "          \"this study. Each iTRAQ MS/MS spectrum measures a peptide from 4 samples (3 individual patients and the reference sample mix of 40 patients). More than 400,000 distinct peptides were identified and quantified in ~14 million MS/MS spectra. Personalized\",\n",
            "          \"and quantified in ~14 million MS/MS spectra. Personalized tumor-specific protein databases were generated in the QUILTS software package using whole exome sequencing-derived variant calls and RNAseq-derived transcript information. All mass spectrometry\",\n",
            "          \"RNAseq-derived transcript information. All mass spectrometry data was analyzed using the Spectrum Mill software package. b, Overview of proteome and phosphoproteome datasets. The table provides a summary of the datasets used in specific analyses,\",\n",
            "          \"provides a summary of the datasets used in specific analyses, including the filters applied to derive the proteins and phosphosites/phosphoproteins that constitute each dataset; the protein, phosphosite or phosphoprotein count; and the methods that employ\",\n",
            "          \"or phosphoprotein count; and the methods that employ the respective datasets. c, Distribution of sequence coverage of the identified proteins with tryptic peptides detected by MS/MS, whiskers show the 5\\u201395 percentiles. d and e, Robust and accurate\",\n",
            "          \"show the 5\\u201395 percentiles. d and e, Robust and accurate proteome/phosphoproteome platform. Longitudinal performance was tested by repeated proteome and phosphoproteome analysis of patient-derived xenograft tumors. Scatterplots, histograms and Pearson\",\n",
            "          \"xenograft tumors. Scatterplots, histograms and Pearson correlations comparing individual replicate measurements are shown. Tumor sample quality control (I) a, Remark diagram showing sample processing and partitioning. Initial quality review encompassed\",\n",
            "          \"processing and partitioning. Initial quality review encompassed histopathological examination of H&E stained tissue slices. *For 3 samples no tumor cells were seen on histopathology (BH-A0E9, BH-A0C1, A2-A0SW). These samples were nevertheless included in\",\n",
            "          \"BH-A0C1, A2-A0SW). These samples were nevertheless included in the proteome analysis since other quality control standards were met (see below) and samples with 0% tumor cellularity on top or bottom sections were included in TCGA analyses. b, correlation\",\n",
            "          \"bottom sections were included in TCGA analyses. b, correlation of TCGA (top or bottom sections) and CPTAC histological assessment of neoplastic cellularity for samples (n = 105). The average and range of neoplastic cellularities were identical for CPTAC\",\n",
            "          \"and range of neoplastic cellularities were identical for CPTAC and TCGA histological assessments. Averages (standard deviations) for neoplastic cellularity were 76% (+/\\u2212 17) for CPTAC, 76% (+/\\u2212 15) for TCGA_Top, and 75% (+/\\u2212 18) for TCGA_Bottom\",\n",
            "          \"76% (+/\\u2212 15) for TCGA_Top, and 75% (+/\\u2212 18) for TCGA_Bottom histopathology slides (Supplementary Table 2). Note that in three CPTAC cases where no tumor cells were identified by histopathological assessment, numbers of protein-level somatic variants were\",\n",
            "          \"assessment, numbers of protein-level somatic variants were similar to all other tumors. The identified mutated proteins were TP53_R273C, NOP58_Q23E, TAGLN2_G154R, TUBA1B_D116H, and MRPL48_I173K (Supplementary Table 5), indicating presence of tumor cells\",\n",
            "          \"(Supplementary Table 5), indicating presence of tumor cells in these samples. c, Proteome iTRAQ tumor/internal reference ratio heatmap for all CPTAC samples (8,028 proteins without missing values) including passed and failed proteomic quality control (QC)\",\n",
            "          \"including passed and failed proteomic quality control (QC) samples. d, Global tumor/reference proteome ratio distributions for samples that passed and failed proteomic quality control analysis. e, Degradation-related gene sets were enriched in tumors that\",\n",
            "          \"e, Degradation-related gene sets were enriched in tumors that failed proteomic quality control analysis. f, Variant allele frequency (VAF) analysis of re-sequenced CPTAC tumors and comparison to original TCGA data. Overall VAFs for failed QC samples were\",\n",
            "          \"to original TCGA data. Overall VAFs for failed QC samples were lower compared to passed samples suggesting lower purity. Tumor sample quality control (II) a, There was high concordance (94.6%) between DNA variants reported by TCGA and CPTAC re-sequenced\",\n",
            "          \"between DNA variants reported by TCGA and CPTAC re-sequenced tumors. Most point mutations reported by TCGA could be identified across the 8 re-sequenced samples used in the study. b, A high overall correlation (mean=0.77) was observed for the CPTAC\",\n",
            "          \"high overall correlation (mean=0.77) was observed for the CPTAC Variant Allele Fraction (VAF) (X-axis) and TCGA VAF (Y-axis) across the 8 samples used in the study. c, Agglomerative hierarchical clustering (Supplementary Methods Section 3.8) used to\",\n",
            "          \"clustering (Supplementary Methods Section 3.8) used to co-cluster protein and RNA tumor expression data after filtering to retain 4,291 proteins and genes with moderate to high protein-RNA correlation (Pearson correlation > 0.4) with results displayed as\",\n",
            "          \"(Pearson correlation > 0.4) with results displayed as a circular dendrogram (fanplot). The proteome (.P) and RNA (.R) components of each sample are labeled using the same color. The outer ring shows proteome samples in light grey and RNA samples in dark\",\n",
            "          \"shows proteome samples in light grey and RNA samples in dark grey. High concordance between RNA and protein expression is evident from the color adjacency in the inner ring and alternating color in the outer ring showing that RNA and protein components\",\n",
            "          \"color in the outer ring showing that RNA and protein components co-cluster for a large proportion of samples (62/80). d, Co-clustering of MS and RPPA tumor data. 126 RPPA readouts were mapped to gene names. These genes were intersected with the genes\",\n",
            "          \"to gene names. These genes were intersected with the genes observed in the MS proteome, filtered to 48 proteins with moderate or higher RPPA-MS protein correlation, and analyzed for co-clustering as in c. 47 of 80 RPPA-MS protein pairs co-cluster. While\",\n",
            "          \"as in c. 47 of 80 RPPA-MS protein pairs co-cluster. While this is a smaller proportion than for RNA-protein analysis, the number of genes used in the clustering is significantly smaller for RPPA (48 vs. 4,291 for RNA). e, ESTIMATE tumor purity comparison\",\n",
            "          \"(48 vs. 4,291 for RNA). e, ESTIMATE tumor purity comparison between mRNA, RNAseq, and proteome data. ANOVA is used to assess the difference in distribution (\\u2212log10(p-value)) of ESTIMATE, stromal, immune, and tumor purity scores across mRNA (microarray),\",\n",
            "          \"immune, and tumor purity scores across mRNA (microarray), RNA-seq and proteome data. The only significant p-value (=0.02) is for the Cluster 3 stromal score, and higher stromal scores for the proteome drive that difference. f, Ischemia score analysis.\",\n",
            "          \"the proteome drive that difference. f, Ischemia score analysis. Comparison of ischemia scores of 77 CPTAC tumors, 3 normal samples, and patient-derived xenografts. CPTAC tumors had generally lower ischemia scores than PDX samples subjected to 30 minutes\",\n",
            "          \"lower ischemia scores than PDX samples subjected to 30 minutes of cold ischemia. Median ischemia scores are less than 30 minutes for each subtype and no significant differences were observed across subtypes. Effects due to cold ischemia therefore appear\",\n",
            "          \"across subtypes. Effects due to cold ischemia therefore appear to be negligible in this CPTAC sample collection. Protein-to-Protein, -CNA, and -mRNA correlation analyses a, Identification of UBE3A as an E3 ubiquitin ligase that negatively correlates to\",\n",
            "          \"UBE3A as an E3 ubiquitin ligase that negatively correlates to p53 on the protein level. Pearson correlation and Benjamini-Hochberg corrected p-value are shown. b, Analysis of counter-regulated genes with negative correlation of CNA-to-RNA as well as\",\n",
            "          \"genes with negative correlation of CNA-to-RNA as well as CNA-to-protein levels. Negative Pearson correlations are shown with Benjamini-Hochberg corrected p-values for CNA-to-protein correlations. Depicted genes have significant negative correlations at\",\n",
            "          \"Depicted genes have significant negative correlations at FDR<0.05 in the CNA-to-RNA and CNA-to-protein analyses. c, Global mRNA-to-protein correlation and gene set enrichment analysis. Global CNA effects and comparison of CNA TRANS effects to knockdown\",\n",
            "          \"CNA effects and comparison of CNA TRANS effects to knockdown signatures in the LINCS database a, CNA landscape in the CPTAC tumor collection. The segment-based CNAs of 77 samples were downloaded from TCGA Firehose, including 18 Basal, 12 Her2, 23 Luminal\",\n",
            "          \"from TCGA Firehose, including 18 Basal, 12 Her2, 23 Luminal A and 24 Luminal B subtypes. Copy number amplifications were marked in red and deletions in blue. The bottom color key represents the log2 transformed copy number value, with CNA=2 centered at 0.\",\n",
            "          \"log2 transformed copy number value, with CNA=2 centered at 0. Specific CNA events are seen for chromosome 5q and 10p regions in basal-like tumors. b, Correlations of copy number alterations (x-axis) to phosphoprotein levels (y-axis) highlight new CNA cis\",\n",
            "          \"to phosphoprotein levels (y-axis) highlight new CNA cis and trans effects. Significant (FDR<0.05) positive (red) and negative (green) correlations between CNA and phosphoproteins are indicated. Histograms show the fraction [%] of significant CNA trans\",\n",
            "          \"Histograms show the fraction [%] of significant CNA trans effects for each CNA gene. c, LINCS CMap analysis facilitates identification of novel functional candidates for CNA trans effects. Knockdown profiles were compared with CNA/protein trans effects\",\n",
            "          \"Knockdown profiles were compared with CNA/protein trans effects for 502 genes. Genes with a connectivity score >|90| were considered connected and significant cis effects were annotated at an FDR<0.05. d, Basal-like tumor-specific CNAs are candidate\",\n",
            "          \"at an FDR<0.05. d, Basal-like tumor-specific CNAs are candidate regulatory events for EGFR and SRC expression levels. Oncogenic kinases with significant CNA/protein trans effects (left panel), that were regulated in LINCS shRNA experiments (right panel; 4\",\n",
            "          \"that were regulated in LINCS shRNA experiments (right panel; 4 cell lines,) and directly measured as LINCS landmark genes, are shown alongside candidate regulatory genes CETN3 and SKP1. Clinical ER, PR, and HER2 annotation and PAM50 classification are\",\n",
            "          \"ER, PR, and HER2 annotation and PAM50 classification are shown in the header rows of each column. Proteome cluster heatmap and stability analysis a, K-means consensus clustering of proteome and phosphoproteome data identifies three subgroups:\",\n",
            "          \"proteome and phosphoproteome data identifies three subgroups: basal-enriched, luminal-enriched, and stromal-enriched. The heatmap represents all 1,521 proteins used for clustering (Dataset G8). b, Identification of optimal proteome clusters for QC-passed\",\n",
            "          \"b, Identification of optimal proteome clusters for QC-passed CPTAC breast cancer tumors. Proteome clusters were derived using consensus clustering based on 1000 resampled datasets, exploring the range of 2 to 6 k-means clusters. Visualization of consensus\",\n",
            "          \"range of 2 to 6 k-means clusters. Visualization of consensus matrices from k-means consensus clustering for k=3, 4, 5 and 6 target clusters. Consensus clustering was performed on 1,521 proteins with no missing values and SD>1.5. c, Silhouette plots were\",\n",
            "          \"with no missing values and SD>1.5. c, Silhouette plots were generated to evaluate the coherence of the clustering. Silhouette plots for k=3 and k=4 clusters showing a cleaner separation of clusters for k=3. d, Based on both visual inspection of the\",\n",
            "          \"of clusters for k=3. d, Based on both visual inspection of the consensus matrix and the delta plot assessing change in consensus cumulative distribution function (CDF) area, three robustly segregated groups were observed. Consensus cumulative distribution\",\n",
            "          \"groups were observed. Consensus cumulative distribution function (CDF) and delta area (change in CDF area) plots for 2\\u20136 clusters. Proteome cluster markers and enriched pathways a, Markers (based on SAM analysis; FDR<0.01) discriminate between proteome\",\n",
            "          \"(based on SAM analysis; FDR<0.01) discriminate between proteome clusters 1, 2 and 3 (compare to heatmap of proteins used to derive clusters depicted in Extended Data Fig. 6a). b, Applying a Fisher exact test-based enrichment analysis to the proteome,\",\n",
            "          \"a Fisher exact test-based enrichment analysis to the proteome, phosphoproteome and mRNA data, gene sets from MSigDB were identified that were unique for each proteome cluster. Heat map showing specific pathways comprising dominant biological themes that\",\n",
            "          \"specific pathways comprising dominant biological themes that are significantly differential by enrichment analysis between basal-enriched and luminal-enriched tumors (Fisher Exact Test Benjamini-Hochberg corrected p-values are shown; enrichment test\",\n",
            "          \"corrected p-values are shown; enrichment test performed on marker sets identified using SAM analysis; see Methods; compare to Figure 3c). c, Heatmap showing a selection of gene sets significant in basal-enriched or luminal-enriched tumors exclusively by\",\n",
            "          \"in basal-enriched or luminal-enriched tumors exclusively by mRNA, protein or phosphoprotein expression. Cytokine signatures, for example, were strongly captured at the mRNA level, but were seen to only a limited degree at the global protein level, likely\",\n",
            "          \"to only a limited degree at the global protein level, likely because of their typically low protein abundance. By contrast, the vast majority of significant gene sets annotated as \\\"signaling\\\" were enriched only at the phosphoprotein level. d, Global heat\",\n",
            "          \"were enriched only at the phosphoprotein level. d, Global heat map representing all gene sets significantly enriched in at least one of the proteomic breast cancer subtypes. The stromal-enriched group was characterized by breast cancer normal-like,\",\n",
            "          \"group was characterized by breast cancer normal-like, adipocyte differentiation, smooth muscle, toll-like receptor signaling and endothelin gene sets, supporting the clustering-based annotation of high stromal and/or adipose content in these tumors (see\",\n",
            "          \"of high stromal and/or adipose content in these tumors (see Supplemental Table 13). Phosphoproteome pathway clustering, kinase-phosphosite multivariate regression, and protein co-expression networks a, Phosphoproteome pathway clustering. Using\",\n",
            "          \"networks a, Phosphoproteome pathway clustering. Using phosphorylation state as a proxy for activity, deep phosphoproteome profiling allows development of a breast cancer molecular taxonomy based on signaling pathways. K-means consensus clustering was\",\n",
            "          \"based on signaling pathways. K-means consensus clustering was performed on pathways derived from single sample GSEA analysis of phosphopeptide data (908 pathways shown). Of four robustly segregated groups, subgroups 2 and 3 substantially recapitulated the\",\n",
            "          \"groups, subgroups 2 and 3 substantially recapitulated the stromal- and luminal-enriched proteomic subgroups, respectively. Subgroup 4 included a significant majority of tumors from the basal-enriched proteomic subgroup, but was admixed particularly with\",\n",
            "          \"proteomic subgroup, but was admixed particularly with luminal-enriched samples. This subgroup was defined by high levels of cell cycle and checkpoint activity. All basal and a majority of non-basal samples in this subgroup had TP53 mutations. Subgroup 1\",\n",
            "          \"samples in this subgroup had TP53 mutations. Subgroup 1 was a novel subgroup defined exclusively in the phosphopeptide / pathway activity domain, with no enrichment for either proteomic or PAM50 subtypes. It was defined by G-protein, G-protein coupled\",\n",
            "          \"PAM50 subtypes. It was defined by G-protein, G-protein coupled receptor, and inositol phosphate metabolism signatures, as well as ionotropic glutamate signaling. b, Analysis of the regulatory relationship between outlier kinases (see Supplementary Table\",\n",
            "          \"relationship between outlier kinases (see Supplementary Table 19) and phosphopeptides by regulatory multivariate regression analysis (see Methods) identified CDK1 as the most highly connected of the outlier Cyclin-Dependent Kinases, with highest\",\n",
            "          \"connected of the outlier Cyclin-Dependent Kinases, with highest centrality (based on node-degree; see Methods) among the outlier CDKs and seventh highest centrality among all the outlier kinases considered in the remMap analysis. Each line represents a\",\n",
            "          \"considered in the remMap analysis. Each line represents a phosphosite-kinase relationship. c\\u2013f, Analysis of differences in the co-expression patterns among genes/proteins across different subgroups. A Joint Random Forest (JRF) method was applied to\",\n",
            "          \"subgroups. A Joint Random Forest (JRF) method was applied to simultaneously build gene co-expression and protein co-expression networks (Supplementary Table 17, and Methods). Modules in these networks revealed different interaction patterns between\",\n",
            "          \"these networks revealed different interaction patterns between basal-enriched and luminal-enriched subgroups. c, Network module P1 of the protein co-expression network, defined chiefly in the proteome space. This module contained 12 genes connected by 39\",\n",
            "          \"proteome space. This module contained 12 genes connected by 39 edges, among which 34 were protein-specific and 5 were shared by both the protein and mRNA co-expression networks. Many edges were supported by published information and were contained in the\",\n",
            "          \"supported by published information and were contained in the STRING database. Edges in red are specific to the protein co-expression network; edges in green are shared by both protein and gene co-expression networks; edges indicated by double lines are\",\n",
            "          \"co-expression networks; edges indicated by double lines are contained in the STRING database with confidence score greater than 0.15. MMP9, one of the central proteins in this module, contributes to metastatic progression and is a potential target for\",\n",
            "          \"to metastatic progression and is a potential target for anti-metastatic therapies for basal-like / triple negative breast cancer. d, Heatmaps of the absolute correlation across each pair of genes in module P1 (shown in Panel c), based on either protein or\",\n",
            "          \"in module P1 (shown in Panel c), based on either protein or gene expression data for samples in the basal-enriched and luminal-enriched subgroups, respectively. The MMP9 protein was strongly co-expressed with the other members of the module only in the\",\n",
            "          \"co-expressed with the other members of the module only in the basal-enriched subgroup. Notably, this observation is dependent on protein data; the correlation at the mRNA level for this module was consistently low in both the basal-enriched and luminal\",\n",
            "          \"was consistently low in both the basal-enriched and luminal enriched subgroups indicating that these events coherently occur at the proteomic level. e, Co-expression network based on proteomics data. The network contains 693 proteomic network-specific\",\n",
            "          \"data. The network contains 693 proteomic network-specific edges (grey) and 792 edges shared with the RNAseq network (green). For each module, the most enriched category and corresponding Benjamini-Hochberg adjusted p-value is reported. Pie charts adjacent\",\n",
            "          \"adjusted p-value is reported. Pie charts adjacent to each module show the proportion of proteomics-specific edges (grey area) and edges shared between proteomics and RNAseq data (green area). f, RNAseq network. Phosphoproteome signatures of PIK3CA (a,b)\",\n",
            "          \"f, RNAseq network. Phosphoproteome signatures of PIK3CA (a,b) and TP53 (c,d) mutated tumors highlight activated key regulators and indicate frequency of activation a and c, Phosphosites upregulated in mutated tumors (SAM FDR<0.05 across all tumors and\",\n",
            "          \"in mutated tumors (SAM FDR<0.05 across all tumors and independently also across luminal tumors; average phosphosite signal for all markers shown as bar graph). To avoid confounding by intrinsic subtype-specific distinctions, only markers that were\",\n",
            "          \"intrinsic subtype-specific distinctions, only markers that were significantly identified both in analyses covering all tumors and analyses restricted to luminal tumors were selected (FDR <0.05). Color bars in the margins indicate FDRs for grouped analysis\",\n",
            "          \"Color bars in the margins indicate FDRs for grouped analysis of different mutation classes and indicate kinase substrates of known kinases in the respective pathways. Significantly regulated kinase phosphosites are annotated. The average phosphorylation\",\n",
            "          \"kinase phosphosites are annotated. The average phosphorylation signal of the marker phosphosites provides a read-out for PI3K and TP53 pathway activity in mutated tumors (histogram below heatmap). A 95% prediction confidence interval (indicated by dashed\",\n",
            "          \"A 95% prediction confidence interval (indicated by dashed lines) across the average signal in non-mutated tumors was chosen in order to discriminate active from non-active tumors. The most strongly activated PIK3CA kinase domain mutant tumor differed from\",\n",
            "          \"activated PIK3CA kinase domain mutant tumor differed from the other 9 kinase domain mutant tumors, as it contained an amino acid side chain charge neutral H1047L instead of the more common positively charged H1047R mutation. Among the 62 phosphosites\",\n",
            "          \"positively charged H1047R mutation. Among the 62 phosphosites identified that were significantly upregulated in PIK3CA mutated tumors, 13 phosphosites were found on phosphoproteins that are known substrates of well-annotated kinases in the PIK3CA pathway\",\n",
            "          \"substrates of well-annotated kinases in the PIK3CA pathway (panel a, right column). In the mutant TP53 analysis a total 20 phosphosites were found on phosphoproteins that are known substrates of well annotated kinases in the p53 pathway (panel c, right\",\n",
            "          \"of well annotated kinases in the p53 pathway (panel c, right column). b and d, Upregulated phosphosite sets were derived from isogenic PIK3CA and TP53 mutant versus wild-type cell line pairs and tested for enrichment within mutant versus wild-type CPTAC\",\n",
            "          \"and tested for enrichment within mutant versus wild-type CPTAC tumors using single sample GSEA. Significantly enriched phosphosite sets are shown (p<0.05). PIRCOS plots, kinase outliers and outliers in the ERBB2 pathway a, Pircos (Proteogenomics CIRCOS)\",\n",
            "          \"outliers in the ERBB2 pathway a, Pircos (Proteogenomics CIRCOS) plots for 8q and 17q showing median CNA, RNA, protein, and phosphosite expression for 20 tumors with amplification in 8q based on RIPK2 CNA>1; 23 tumors with amplification in 8q based on PTK2\",\n",
            "          \"RIPK2 CNA>1; 23 tumors with amplification in 8q based on PTK2 CNA>1; 15 tumors with amplification in 17q based on CDK12 CNA >1; and 10 tumors with amplification in 17q based on TLK2 CNA>1. Red indicates expression >1, blue < \\u22121, and grey between \\u22121 and 1.\",\n",
            "          \"indicates expression >1, blue < \\u22121, and grey between \\u22121 and 1. Genes with both copy number amplification (CNA>1) and increased phosphosite expression (p-site>1) are labeled. b, Phosphosite outliers in known ERBB2 signaling genes. To better understand the\",\n",
            "          \"in known ERBB2 signaling genes. To better understand the downstream effects of ERBB2 amplification, phosphosite outliers in known ERBB2 signaling genes (MSigDB pathway set, KEGG_ERBB_SIGNALING PATHWAY) were identified for the 15 samples that had ERBB2\",\n",
            "          \"PATHWAY) were identified for the 15 samples that had ERBB2 phosphosite outlier status. Forty-one genes were identified as having a phosphosite outlier in at least one of the ERBB2 amplified samples. PAK4 and ARAF phosphosite outlier status were found in\",\n",
            "          \"samples. PAK4 and ARAF phosphosite outlier status were found in seven of the 15 ERBB2 kinase outlier samples; GSK3B outliers were found in 6 samples; and EIF4EBP1, MAP2K2, ABL1 and AKT1 outlier status was found in 5 of the 15 samples. c, Proteogenomic\",\n",
            "          \"status was found in 5 of the 15 samples. c, Proteogenomic outlier expression analysis for TLK2 and RIPK2. Samples with outlier phosphosite (red), protein (yellow), RNA (green) and copy number (purple) expression are shown. Phosphosite squares indicate\",\n",
            "          \"(purple) expression are shown. Phosphosite squares indicate per-sample outlier phosphosites. Supplementary Material Supplementary Information is available in the online version of the paper. Author Contributions P.M., D.R.M., M.A.G., K.R.C., and S.A.C.\",\n",
            "          \"Author Contributions P.M., D.R.M., M.A.G., K.R.C., and S.A.C. designed the proteomic analysis experiments, data analysis workflow, and proteomic-genomic data comparisons. P.M., M.A.G., J.W.Q., and S.A.C. directed and performed proteomic analysis of breast\",\n",
            "          \"and S.A.C. directed and performed proteomic analysis of breast tumor and quality control samples. P.M., D.R.M., K.V.R., K.R.C., P.W., X.W., S.C., E.K., F.P., Z.T., J.L., M.L.G., M.W., V.Y., K.H., C.L., M.D.M., P.Y., J.W., B.Z., and D.F. performed\",\n",
            "          \"V.Y., K.H., C.L., M.D.M., P.Y., J.W., B.Z., and D.F. performed proteomic-genomic data analyses. D.R.M., P.W., and S.J.S. provided statistical support. D.R.M., K.V.R., K.R.C., K.K. and D.F. performed analyses of mass spectrometry data and adapted\",\n",
            "          \"D.F. performed analyses of mass spectrometry data and adapted algorithms and software for data analysis. S.R.D., R.R.T and M.J.E. developed and prepared breast xenografts used as quality control samples. P.M. and F.M prepared and analyzed cell lines for\",\n",
            "          \"samples. P.M. and F.M prepared and analyzed cell lines for correlative functional annotation of frequently mutated genes. P.M., D.R.M., M.A.G., and S.A.C designed strategy for quality control analyses. M.A.G., S.R.D., C.R.K., M.M., and H.R. coordinated\",\n",
            "          \"analyses. M.A.G., S.R.D., C.R.K., M.M., and H.R. coordinated acquisition, distribution and quality control evaluation of TCGA tumor samples. P.M., M.A.G., C.P., L.D., A.G.P., and M.J.E. interpreted data in the context of breast cancer biology. P.M.,\",\n",
            "          \"interpreted data in the context of breast cancer biology. P.M., D.R.M, M.A.G., K.R.C., P.W., A.G.P, M.J.E. and S.A.C. wrote the manuscript. All primary mass spectrometry data are deposited at the CPTAC Data Portal as raw and mzML files and complete\",\n",
            "          \"at the CPTAC Data Portal as raw and mzML files and complete protein assembly data sets for public access (https://cptac-data-portal.georgetown.edu/cptac/s/S029). In addition, a set of ancillary files such as dataset G1/P1, G3/P3, G4/P4, G5/P5, G7/P7, CNA\",\n",
            "          \"files such as dataset G1/P1, G3/P3, G4/P4, G5/P5, G7/P7, CNA correlation tables for CNA-mRNA, CNA-proteome and CNA-phosphoproteome; CNA data, RNA-seq expression data have also been deposited at the DCC. Two browsers have been created to assist the\",\n",
            "          \"at the DCC. Two browsers have been created to assist the interested reader in exploring the results. One provides track hubs for viewing the identified peptides in the UCSD genome browser (http://fenyolab.org/cptac_breast_ucsc). The second is an on-line\",\n",
            "          \"The second is an on-line tool for proteogenomic data exploration and can be accessed at http://prot-shiny-vm.broadinstitute.org:3838/BC2016/. See Supplemental Methods for descriptions. The authors declare no competing financial interests. Proteogenomic\",\n",
            "          \"authors declare no competing financial interests. Proteogenomic analysis of human breast cancer. Direct effects of genomic alterations on protein level Overlap of a, protein coding single amino acid variants (SAAVs) and b, RNA splice junctions not present\",\n",
            "          \"acid variants (SAAVs) and b, RNA splice junctions not present in RefSeq v60 detected by DNA exome sequencing, RNA-seq, and LC-MS/MS. Proportions of novel variants are noted. c, Heatmap of mutations/CNA and their effects on RNA and protein expression of\",\n",
            "          \"and their effects on RNA and protein expression of breast cancer-relevant genes across tumor and normal samples. ER, PR, HER2 and PAM50 status are annotated. Median iTRAQ protein abundance ratio and the most frequently detected and differential\",\n",
            "          \"ratio and the most frequently detected and differential phosphosite ratio are shown for each gene. Pearson correlations between MS protein vs RNA-seq and MS protein vs RPPA are indicated. Effects of copy number alterations (CNA) on mRNA, protein, and\",\n",
            "          \"Effects of copy number alterations (CNA) on mRNA, protein, and phosphoprotein abundance a, Correlations of CNA (x-axes) to RNA and protein expression levels (y-axes) highlight new CNA cis and trans effects. Significant (FDR<0.05) positive (red) and\",\n",
            "          \"and trans effects. Significant (FDR<0.05) positive (red) and negative (green) correlations between CNA and mRNAs or proteins are indicated. CNA cis effects appear as a red diagonal line, CNA trans effects as vertical stripes. Histograms show the fraction\",\n",
            "          \"trans effects as vertical stripes. Histograms show the fraction [%] of significant CNA trans effects for each CNA gene. b, Overlap of cis effects observed at RNA, protein, and phosphoprotein levels (FDR<0.05). c, Trans-effect regulatory candidates\",\n",
            "          \"levels (FDR<0.05). c, Trans-effect regulatory candidates identified among those with significant protein cis-effects using LINCS CMap. Bars indicate total numbers of significant CNA/protein trans effects (gray; FDR<0.05) and overlap with regulated genes\",\n",
            "          \"trans effects (gray; FDR<0.05) and overlap with regulated genes in LINCS knock-down profiles (red; 4 cell lines; moderated T-test FDR<0.1). Proteomic and phosphoproteomic subtypes of breast cancer and subtype-specific pathway enrichment a, Unsupervised\",\n",
            "          \"cancer and subtype-specific pathway enrichment a, Unsupervised clustering of RNA-seq and proteomics data restricted to PAM50 genes and subset of 35 detected proteins reveal high similarity to PAM50 (TCGA) sample annotation. b, K-means consensus clustering\",\n",
            "          \"PAM50 (TCGA) sample annotation. b, K-means consensus clustering of proteome and phosphoproteome data identifies basal-enriched, luminal-enriched, and stromal-enriched subgroups. c, Gene set enrichment analysis highlights sets of pathways significantly\",\n",
            "          \"enrichment analysis highlights sets of pathways significantly differential between basal-enriched and luminal-enriched tumors (detailed in Extended Data Fig. 7b). d, K-means consensus clustering performed on pathways derived from single sample GSEA\",\n",
            "          \"performed on pathways derived from single sample GSEA analysis of phosphopeptide data identifies four distinct clusters. Example analyses of aberrantly regulated kinases in human breast cancer a and b, PIRCOS (Proteogenomics CIRCOS) plots showing CNA,\",\n",
            "          \"a and b, PIRCOS (Proteogenomics CIRCOS) plots showing CNA, RNA, protein and phosphosite expression for 17 tumors with amplification in 17q (ERBB2 CNA>1) and 8 tumors with amplification in 11q (PAK1 CNA>1). Labeled genes have CNA>1 and phosphosite>1. c,\",\n",
            "          \"(PAK1 CNA>1). Labeled genes have CNA>1 and phosphosite>1. c, Proteogenomic outlier expression analysis for ERBB2, CDK12, and PAK1. Samples with outlier phosphosite (red), protein (yellow), RNA (green) and copy number (purple) expression are shown.\",\n",
            "          \"RNA (green) and copy number (purple) expression are shown. Phosphosite squares indicate per-sample outlier phosphosites. d, Outlier kinase events by PAM50 subtype (>35% of subtype samples contain a phosphosite outlier; <10% FDR using Benjamini-Hochberg\",\n",
            "          \"a phosphosite outlier; <10% FDR using Benjamini-Hochberg adjusted p-values).\"\n",
            "        ]\n",
            "      }\n",
            "    ]\n",
            "  },\n",
            "  \"PDC000174\": {\n",
            "    \"study_id\": \"b93bb1e9-57b8-11e8-b07a-00a098d917f8\",\n",
            "    \"pdc_study_id\": \"PDC000174\",\n",
            "    \"study_submitter_id\": \"TCGA BRCA Phosphoproteome S015-2\",\n",
            "    \"program_id\": \"10251935-5540-11e8-b664-00a098d917f8\",\n",
            "    \"project_id\": \"48af5040-5546-11e8-b664-00a098d917f8\",\n",
            "    \"study_name\": \"TCGA Breast Cancer Phosphoproteome\",\n",
            "    \"program_name\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"project_name\": \"CPTAC2 Retrospective\",\n",
            "    \"disease_type\": \"Breast Invasive Carcinoma;Other\",\n",
            "    \"primary_site\": \"Breast;Not Reported\",\n",
            "    \"analytical_fraction\": \"Phosphoproteome\",\n",
            "    \"experiment_type\": \"iTRAQ4\",\n",
            "    \"embargo_date\": null,\n",
            "    \"cases_count\": 109,\n",
            "    \"aliquots_count\": 109,\n",
            "    \"protocol_id\": \"1c06aff5-3b26-11e9-9a07-0a80fada099c\",\n",
            "    \"protocol_submitter_id\": \"P-S015-2\",\n",
            "    \"program_submitter_id\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"protocol_name\": \"TCGA_Breast_Cancer_Phosphoproteome\",\n",
            "    \"protocol_date\": \"2013-02-02\",\n",
            "    \"publications\": [\n",
            "      {\n",
            "        \"publication_id\": \"44e2e766-89c7-11e8-bcf1-0a2705229b82\",\n",
            "        \"pubmed_id\": \"27251275\",\n",
            "        \"doi\": \"10.1038/nature18003\",\n",
            "        \"title\": \"Proteogenomics connects somatic mutations to signalling in breast cancer\",\n",
            "        \"journal\": \"Nature\",\n",
            "        \"journal_url\": \"https://www.nature.com/articles/nature18003\",\n",
            "        \"year\": \"2016\",\n",
            "        \"abstract\": \"Somatic mutations have been extensively characterized in breast cancer, but the effects of these genetic alterations on the proteomic landscape remain poorly understood. Here we describe quantitative mass-spectrometry-based proteomic and phosphoproteomic analyses of 105 genomically annotated breast cancers, of which 77 provided high-quality data. Integrated analyses provided insights into the somatic cancer genome including the consequences of chromosomal loss, such as the 5q deletion characteristic of basal-like breast cancer. Interrogation of the 5q trans-effects against the Library of Integrated Network-based Cellular Signatures, connected loss of CETN3 and SKP1 to elevated expression of epidermal growth factor receptor (EGFR), and SKP1 loss also to increased SRC tyrosine kinase. Global proteomic data confirmed a stromal-enriched group of proteins in addition to basal and luminal clusters, and pathway analysis of the phosphoproteome identified a G-protein-coupled receptor cluster that was not readily identified at the mRNA level. In addition to ERBB2, other amplicon-associated highly phosphorylated kinases were identified, including CDK12, PAK1, PTK2, RIPK2 and TLK2. We demonstrate that proteogenomic analysis of breast cancer elucidates the functional consequences of somatic mutations, narrows candidate nominations for driver genes within large deletions and amplified regions, and identifies therapeutic targets.\",\n",
            "        \"full_text\": [\n",
            "          \"Proteogenomics connects somatic mutations to signaling in breast cancer Summary Somatic mutations have been extensively characterized in breast cancer, but the effects of these genetic alterations on the proteomic landscape remain poorly understood. We\",\n",
            "          \"on the proteomic landscape remain poorly understood. We describe quantitative mass spectrometry-based proteomic and phosphoproteomic analyses of 105 genomically annotated breast cancers of which 77 provided high-quality data. Integrated analyses allowed\",\n",
            "          \"77 provided high-quality data. Integrated analyses allowed insights into the somatic cancer genome including the consequences of chromosomal loss, such as the 5q deletion characteristic of basal-like breast cancer. The 5q trans effects were interrogated\",\n",
            "          \"breast cancer. The 5q trans effects were interrogated against the Library of Integrated Network-based Cellular Signatures, thereby connecting CETN3 and SKP1 loss to elevated expression of EGFR, and SKP1 loss also to increased SRC. Global proteomic data\",\n",
            "          \"and SKP1 loss also to increased SRC. Global proteomic data confirmed a stromal-enriched group in addition to basal and luminal clusters and pathway analysis of the phosphoproteome identified a G Protein-coupled receptor cluster that was not readily\",\n",
            "          \"a G Protein-coupled receptor cluster that was not readily identified at the mRNA level. Besides ERBB2, other amplicon-associated, highly phosphorylated kinases were identified, including CDK12, PAK1, PTK2, RIPK2 and TLK2. We demonstrate that proteogenomic\",\n",
            "          \"PAK1, PTK2, RIPK2 and TLK2. We demonstrate that proteogenomic analysis of breast cancer elucidates functional consequences of somatic mutations, narrows candidate nominations for driver genes within large deletions and amplified regions, and identifies\",\n",
            "          \"within large deletions and amplified regions, and identifies therapeutic targets. Introduction A central deficiency in our knowledge of cancer concerns how genomic changes drive the proteome and phosphoproteome to execute phenotypic characteristics. The\",\n",
            "          \"and phosphoproteome to execute phenotypic characteristics. The initial proteomic characterization in the TCGA breast study was performed using reversed phase protein arrays; however this approach is restricted by antibody availability. To provide greater\",\n",
            "          \"is restricted by antibody availability. To provide greater analytical breadth, the NCI Clinical Proteomic Tumor Analysis Consortium (CPTAC) is analyzing the proteomes of genome-annotated TCGA tumor specimens using mass spectrometry. Herein we describe\",\n",
            "          \"tumor specimens using mass spectrometry. Herein we describe integrated proteogenomic analyses of TCGA breast cancer samples representing the four principal mRNA-defined breast cancer intrinsic subtypes. Proteogenomic analysis of TCGA samples 105 breast\",\n",
            "          \"subtypes. Proteogenomic analysis of TCGA samples 105 breast tumors previously characterized by the TCGA were selected for proteomic analysis after histopathological documentation (Supplementary Tables 1 and 2). The cohort included a balanced\",\n",
            "          \"(Supplementary Tables 1 and 2). The cohort included a balanced representation of PAM50-defined intrinsic subtypes including 25 basal-like, 29 luminal A, 33 luminal B, and 18 HER2 (ERBB2)-enriched tumors, along with 3 normal breast tissue samples. Samples\",\n",
            "          \"tumors, along with 3 normal breast tissue samples. Samples were analyzed by high-resolution, accurate mass, tandem mass spectrometry (MS) that included extensive peptide fractionation and phosphopeptide enrichment (Extended Data Fig. 1a). An isobaric\",\n",
            "          \"phosphopeptide enrichment (Extended Data Fig. 1a). An isobaric peptide labeling approach (iTRAQ) was employed to quantify protein and phosphosite levels across samples, with 37 iTRAQ 4-plexes analyzed in total. A total of 15,369 proteins (12,405 genes)\",\n",
            "          \"analyzed in total. A total of 15,369 proteins (12,405 genes) and 62,679 phosphosites were confidently identified with 11,632 proteins/tumor and 26,310 phosphosites/tumor on average (Supplementary Tables 3, 4 and Supplementary Methods). After filtering for\",\n",
            "          \"Tables 3, 4 and Supplementary Methods). After filtering for observation in at least a quarter of the samples (Supplementary Methods, Extended Data Fig. 1b) 12,553 proteins (10,062 genes) and 33,239 phosphosites, with their relative abundances quantified\",\n",
            "          \"33,239 phosphosites, with their relative abundances quantified across tumors, were used in subsequent analyses in this study. Stable longitudinal performance and low technical noise were demonstrated by repeated, interspersed analyses of a single batch of\",\n",
            "          \"by repeated, interspersed analyses of a single batch of patient-derived luminal and basal breast cancer xenograft samples (Extended Data Fig. 1d,e). Due to the heterogeneous nature of breast tumors, and because proteomic analyses were performed on tumor\",\n",
            "          \"tumors, and because proteomic analyses were performed on tumor fragments that were different from those used in the genomic analyses, rigorous pre-specified sample and data QC metrics were implemented (Supplementary Discussion and Extended Data Figures 2,\",\n",
            "          \"(Supplementary Discussion and Extended Data Figures 2, 3). Extensive analyses concluded that 28 of the 105 samples were compromised by protein degradation. These samples were excluded from further analysis with subsequent informatics focused on the 77\",\n",
            "          \"further analysis with subsequent informatics focused on the 77 tumor samples and three biological replicates. Genome and transcriptomic variation was observed at the peptide level by searching MS/MS spectra not matched to RefSeq against a patient-specific\",\n",
            "          \"MS/MS spectra not matched to RefSeq against a patient-specific sequence database (Fig. 1a). The database was constructed using the QUILTS software package leveraging RefSeq gene models based on whole exome and RNA-seq data generated from portions of the\",\n",
            "          \"on whole exome and RNA-seq data generated from portions of the same tumors and matched germline DNA (Fig. 1a, Supplementary Table 5). While these analyses detected a number of single amino-acid variants (SAAVs), frameshifts, and splice junctions,\",\n",
            "          \"amino-acid variants (SAAVs), frameshifts, and splice junctions, including splice isoforms that had been detected as only single transcript reads by RNA-seq (Fig. 1b, Supplementary Table 5), the number of genomic and transcriptomic variants that were\",\n",
            "          \"5), the number of genomic and transcriptomic variants that were confirmed as peptides by MS was low (Supplementary Discussion). Sparse detection of individual genomic variants by peptide sequencing has been noted in our previous studies and reflects\",\n",
            "          \"sequencing has been noted in our previous studies and reflects limited coverage at the single amino-acid level with current technology. However quantitative MS analysis of multiple peptides for each protein is used to reliably infer overall protein\",\n",
            "          \"for each protein is used to reliably infer overall protein levels. This is an advantage for MS since antibody-based protein expression analysis is typically based on a single epitope. To illustrate this capability in the current data set an initial\",\n",
            "          \"illustrate this capability in the current data set an initial analysis of three frequently mutated genes in breast cancer (TP53, PIK3CA, and GATA3) and three clinical biomarkers (ER, PGR, and ERBB2) was conducted (Fig. 1c, Supplementary Table 6, 7 and\",\n",
            "          \"and ERBB2) was conducted (Fig. 1c, Supplementary Table 6, 7 and Supplementary Discussion). As expected, TP53 missense mutations were associated with elevated MS-based protein levels, as observed by RPPA (Reverse Phase Protein Array), especially in\",\n",
            "          \"observed by RPPA (Reverse Phase Protein Array), especially in basal-like breast cancer. TP53 nonsense and frame-shift mutations were associated with a decrease in TP53 protein levels that was particularly striking in the MS data. In contrast, the mostly\",\n",
            "          \"particularly striking in the MS data. In contrast, the mostly C-terminal GATA3 frame-shift alterations did not result in decreased protein expression when measured by the median of all GATA3 peptides, suggesting these proteins are expressed despite\",\n",
            "          \"GATA3 peptides, suggesting these proteins are expressed despite truncation. No consistent effect of somatic PIK3CA mutation was observed at the level of protein expression. Good correlations between RNA-seq and MS-protein expression levels were found for\",\n",
            "          \"between RNA-seq and MS-protein expression levels were found for ESR1 (r=0.74), PGR (r=0.74), ERBB2 (r=0.84) and GATA3 (r=0.83) with moderate correlations observed for PIK3CA (r=0.45) and TP53 (r=0.36). Lower TP53 protein abundance levels compared to mRNA\",\n",
            "          \"(r=0.36). Lower TP53 protein abundance levels compared to mRNA levels were especially prevalent in luminal tumors, suggesting post-transcriptional regulatory mechanisms such as proteasomal degradation. To explore this hypothesis a search was made for E3\",\n",
            "          \"To explore this hypothesis a search was made for E3 ligases that showed negative correlation to p53 protein (Supplementary Table 8). These analyses identified UBE3A (r=\\u22120.42; adj. pval=0.05) (Extended Data Fig. 4a), an established TP53 E3 ligase. In\",\n",
            "          \"(Extended Data Fig. 4a), an established TP53 E3 ligase. In comparing CNA, RNA, and protein levels for GATA3, copy number gains in chromosome 10q were anti-correlated with RNA and protein levels in basal-like tumors. This observation prompted a search for\",\n",
            "          \"in basal-like tumors. This observation prompted a search for other gains or losses that were anticorrelated with RNA and/or protein levels (see Extended Data Fig. 4b for further analyses). Overall, six genes were identified that significantly\",\n",
            "          \"Overall, six genes were identified that significantly anti-correlated at an FDR<0.05 on both RNA and protein level to their CNA signals (Extended Data Fig. 4b). GATA3 amplification on 10q in basal-like breast cancer showed the strongest anti-correlation,\",\n",
            "          \"basal-like breast cancer showed the strongest anti-correlation, followed by the hexosamine and glycolysis pathway enzymes GFPT2 and HK3, which are upregulated in basal-like breast cancer despite being subjected to frequent chromosomal deletion on 5q.\",\n",
            "          \"despite being subjected to frequent chromosomal deletion on 5q. Global analysis of the correlation of mRNA-to-protein yielded a median Pearson value of r=0.39, with 6,135 out of 9,302 protein/mRNA pairs (66.0%) correlating significantly at an FDR<0.05\",\n",
            "          \"pairs (66.0%) correlating significantly at an FDR<0.05 (Extended Data Fig. 4c, Supplementary Table 9 and Supplementary Discussion). Similar to the colon cancer analysis metabolic functions such as amino acid, sugar and fatty acid metabolism were found to\",\n",
            "          \"as amino acid, sugar and fatty acid metabolism were found to be enriched among positively correlated genes whereas ribosomal, RNA polymerase and mRNA splicing functions were negatively correlated. Overall these analyses demonstrate the utility of global\",\n",
            "          \"Overall these analyses demonstrate the utility of global proteome correlation analysis for both confirmation of suspected regulatory mechanisms and identification of candidate regulators meriting further investigation. Copy Number Alterations To determine\",\n",
            "          \"further investigation. Copy Number Alterations To determine the consequences of CNA on mRNA, protein and phosphoprotein abundance, both in \\u201ccis\\u201d on genes within the aberrant locus and in \\u201ctrans\\u201d on genes encoded elsewhere, univariate correlation analysis\",\n",
            "          \"on genes encoded elsewhere, univariate correlation analysis was used as previously described. A total of 7,776 genes with CNA, mRNA and protein measurements were analyzed by calculating Pearson correlation and associated statistical significance\",\n",
            "          \"Pearson correlation and associated statistical significance (Benjamini-Hochberg corrected p-value) for all possible CNA-mRNA and CNA-protein pairs (Fig. 2a, Supplementary Table 10, Extended Data Fig. 5a, see Methods). For the phosphoproteome, 4,472\",\n",
            "          \"Data Fig. 5a, see Methods). For the phosphoproteome, 4,472 CNA-phosphoprotein pairs were analyzed (Extended Data Fig. 5b). Significant positive correlations (cis) were observed for 64% of all CNA-mRNA, 31% of all CNA-protein, and 20% of all\",\n",
            "          \"for 64% of all CNA-mRNA, 31% of all CNA-protein, and 20% of all CNA-phosphoprotein pairs Fig. 2b. Proteins and phosphoproteins correlated in cis to CNA were, for the most part, a subset of the cis effects observed in mRNA-to-CNA correlation (Fig. 2b,\",\n",
            "          \"the cis effects observed in mRNA-to-CNA correlation (Fig. 2b, Supplementary Table 10). The fractional difference of well-annotated oncogenes and tumor suppressor genes among the significantly cis-correlated CNA-to-mRNA and -protein gene pairs was\",\n",
            "          \"cis-correlated CNA-to-mRNA and -protein gene pairs was analyzed. Based on a reference list of 487 oncogenes and tumor suppressors (Supplementary Table 10), these cancer relevant genes occur 37.6% more frequently in the subset of genes that correlate both\",\n",
            "          \"more frequently in the subset of genes that correlate both on CNA/mRNA and CNA/protein levels than in the subset that only correlate on CNA/mRNA but not on CNA/protein level (Fisher exact p-value=0.02). This suggests that CNA events with a tumor promoting\",\n",
            "          \"This suggests that CNA events with a tumor promoting outcome more likely lead to cis regulatory effects on both the protein and mRNA level, whereas CNA events with no documented role in tumorigenesis are more likely to be neutralized on the protein level\",\n",
            "          \"are more likely to be neutralized on the protein level than on the RNA level. Trans effects (Fig. 2a) appear as vertical bands, with accompanying frequency histograms (in blue) highlighting \\u201chot spots\\u201d of significant trans effects. Using a minimum\",\n",
            "          \"\\u201chot spots\\u201d of significant trans effects. Using a minimum threshold of 50 trans-affected genes, 68% of the tested genes were associated with trans effects on the mRNA level, whereas only 13% were associated with effects on the protein level and 8% on the\",\n",
            "          \"were associated with effects on the protein level and 8% on the phosphoprotein level. Importantly, CNA-protein correlations appeared to be a reduced representation of CNA-mRNA correlations. Furthermore, for many CNA regions correlations were more\",\n",
            "          \"Furthermore, for many CNA regions correlations were more directionally uniform on the protein level than on the mRNA level. CNA regions exhibiting the most trans associations at the protein level were found on chromosomes 5q (LOH in basal; gain in LumB),\",\n",
            "          \"were found on chromosomes 5q (LOH in basal; gain in LumB), 10p (gain in basal), 12 (gain in basal), 16q (LumA deletion), 17q (LumB amplification), and 22q (LOH in luminal and basal) (Extended Data Fig. 5a). Trans associations are not necessarily direct\",\n",
            "          \"Data Fig. 5a). Trans associations are not necessarily direct consequences of the chromosomal aberration. For example since 5q loss occurs in at least 50% of basal-like breast cancers, many of the trans effects involve genes that mark the basal-subtype. To\",\n",
            "          \"the trans effects involve genes that mark the basal-subtype. To identify candidate driver genes whose copy number alterations are direct drivers of trans effects, results were compared with functional knock-down data on 3,797 genes in the Library of\",\n",
            "          \"functional knock-down data on 3,797 genes in the Library of Integrated Cellular Signatures (LINCS) database (http://www.lincsproject.org/). For any given gene with copy number alterations (\\u201cCNA-gene\\u201d), sets of genes were identified corresponding to\",\n",
            "          \"(\\u201cCNA-gene\\u201d), sets of genes were identified corresponding to proteins that changed where there was gain (\\u201cCNA-gain trans gene set\\u201d) or loss (\\u201cCNA-loss trans gene set\\u201d). These gene sets were then compared to the effects of gene knock down in the LINCS\",\n",
            "          \"then compared to the effects of gene knock down in the LINCS database (see Methods). Queries for 502 different CNA-genes meeting the criteria defined above identified 10 CNA-genes that could be functionally connected to both CNA-gain and CNA-loss trans\",\n",
            "          \"be functionally connected to both CNA-gain and CNA-loss trans protein-level effects (Extended Data Fig. 5c, Supplementary Table 11). A permutation-based approach implemented to test significance (see Supplementary Methods) yielded an FDR < 0.05 for 10\",\n",
            "          \"(see Supplementary Methods) yielded an FDR < 0.05 for 10 genes affected by both CNA gains and losses (Fig. 2c). These proteins were defined as potential regulatory candidates for the CNA trans effects observed on the proteome level in this study, since in\",\n",
            "          \"effects observed on the proteome level in this study, since in a gene-dependent manner on average 17% of these trans effects were consistent with the knockdown profiles. Notably, the established oncogenic receptor tyrosine kinase ERBB2 was functionally\",\n",
            "          \"oncogenic receptor tyrosine kinase ERBB2 was functionally connected only to CNA gain trans effects (Supplementary Table 11). The E3 ligase SKP1 and the ribonucleoprotein export factor CETN3, both located on chromosome arm 5q with frequent losses in\",\n",
            "          \"both located on chromosome arm 5q with frequent losses in basal-like breast cancer and less frequent gains in luminal B breast cancer, were detected as potential regulators affecting the expression of the tyrosine kinase and therapeutic target EGFR, and\",\n",
            "          \"of the tyrosine kinase and therapeutic target EGFR, and SKP1 also was linked to SRC (Extended Data Fig. 5d). Another potential regulator, FBXO7, (a substrate recognition component of the SCF (SKP1-CUL1-F-box protein)-type E3 ubiquitin ligase complex), was\",\n",
            "          \"protein)-type E3 ubiquitin ligase complex), was affected mostly by LOH events on chromosome 22q. Interestingly, in a recent human interaction proteome study SKP1 and FBXO7 were listed as interaction partners. Clustering and Network analysis\",\n",
            "          \"listed as interaction partners. Clustering and Network analysis Transcriptional profiling has converged on four major breast cancer subtypes: Luminal A and B, basal and HER2-enriched. To investigate the extent to which the PAM50 \\u201cintrinsic\\u201d breast cancer\",\n",
            "          \"the extent to which the PAM50 \\u201cintrinsic\\u201d breast cancer classification scheme is reflected or refined on the proteome level in the CPTAC samples, clustering analyses were first restricted to the reduced set of PAM50 genes. When RNA data for the 50 PAM50\",\n",
            "          \"the reduced set of PAM50 genes. When RNA data for the 50 PAM50 genes were clustered directly (without using a classifier), the clustering was similar to the TCGA PAM50 annotation (second annotation bar in Fig. 3a). Restricting both the RNA and proteome\",\n",
            "          \"bar in Fig. 3a). Restricting both the RNA and proteome data to the set of 35 PAM50 genes observed in the proteome produced a similar result (bottom two annotation bars in Fig. 3a), and all the major PAM50 groups were recapitulated in the proteome almost\",\n",
            "          \"major PAM50 groups were recapitulated in the proteome almost as well as in the RNA data. This indicates that although different tissue sections of the same tumors were used for RNA-seq and protein analysis, very similar subtype-defining features can be\",\n",
            "          \"protein analysis, very similar subtype-defining features can be observed in both data types. Global proteome and phosphoproteome data were then used to identify proteome subtypes in an unsupervised manner. Consensus clustering identified basal-enriched,\",\n",
            "          \"manner. Consensus clustering identified basal-enriched, luminal-enriched, and stromal-enriched clusters (Extended Data Figs. 6a\\u2013d, 7a). Unlike the clustering observed with PAM50 genes, mRNA-defined HER2-enriched tumors were distributed across these three\",\n",
            "          \"HER2-enriched tumors were distributed across these three proteomic subgroups. The basal-enriched and luminal-enriched groups showed a strong overlap with the mRNA-based PAM50 basal-like and luminal subgroups, whereas stromal-enriched proteome subtype\",\n",
            "          \"luminal subgroups, whereas stromal-enriched proteome subtype represented a mix of all PAM50 mRNA-based subtypes, and has a significantly enriched stromal signature (Extended Data Fig. 3e). Among the stromal-enriched tumors there was strong representation\",\n",
            "          \"the stromal-enriched tumors there was strong representation of reactive type I tumors as classified by RPPA (Supplementary Table 12), showing agreement between the RPPA and mass spectrometry-based protein analyses for the detection of a tumor subgroup\",\n",
            "          \"protein analyses for the detection of a tumor subgroup characterized by stromal gene expression. Since the basal- and luminal-enriched proteome subgroups are coherent, pathway analyses were conducted on these two subtypes, using the stromal-enriched\",\n",
            "          \"conducted on these two subtypes, using the stromal-enriched subgroup as a control to assess specificity. (Fig. 3c, Extended Data Fig. 7b, Supplementary Table 13). The luminal-enriched subgroup was exclusively enriched for estradiol and ESR1-driven gene\",\n",
            "          \"was exclusively enriched for estradiol and ESR1-driven gene sets. In contrast, multiple gene sets were enriched and upregulated specifically in the basal-like tumors. Particularly extensive basal-like enrichment was seen for MYC target genes; for cell\",\n",
            "          \"basal-like enrichment was seen for MYC target genes; for cell cycle, checkpoint, and DNA repair pathways including regulators AURKA/B, ATM, ATR, CHEK1/2, and BRCA1/2; and for immune response/inflammation, including T-cell, B-cell, and neutrophil\",\n",
            "          \"response/inflammation, including T-cell, B-cell, and neutrophil signatures. The complementarity of transcriptional, proteomic, and phosphoproteomic data was also highlighted in these analyses (Extended Data Fig. 7c, d). Using phosphorylation status as a\",\n",
            "          \"(Extended Data Fig. 7c, d). Using phosphorylation status as a proxy for activity, phosphoproteome profiling can theoretically be used to develop a signaling pathway-based cancer classification. K-means consensus clustering was therefore performed on\",\n",
            "          \"K-means consensus clustering was therefore performed on pathways derived from single sample GSEA analysis of phosphopeptide data (Methods, Supplementary Tables 14 and 15). Of four robustly segregated groups, subgroups 2 and 3 substantially recapitulated\",\n",
            "          \"groups, subgroups 2 and 3 substantially recapitulated the stromal- and luminal-enriched proteomic subgroups, respectively (Fig. 3d, Extended Data Fig. 8a). Subgroup 4 included a majority of tumors from the basal-enriched proteomic subgroup, but was\",\n",
            "          \"of tumors from the basal-enriched proteomic subgroup, but was admixed particularly with luminal-enriched samples. This subgroup was defined by high levels of cell cycle and checkpoint activity. All basal and a majority of non-basal samples in this\",\n",
            "          \"activity. All basal and a majority of non-basal samples in this subgroup had TP53 mutations. Consistent with high levels of cell cycle activity, a multivariate kinase-to-phosphosite abundance regression analysis highlighted CDK1 as one of the most highly\",\n",
            "          \"regression analysis highlighted CDK1 as one of the most highly connected kinases in this study (Extended Data Fig. 8b, Supplementary Table 16). Subgroup 1 was a novel subgroup defined exclusively in the phosphopeptide/pathway activity domain, with no\",\n",
            "          \"in the phosphopeptide/pathway activity domain, with no enrichment for either proteomic or PAM50 subtypes. It was defined by G-protein, G-protein coupled receptor, and inositol phosphate metabolism signatures, as well as ionotropic glutamate signaling\",\n",
            "          \"signatures, as well as ionotropic glutamate signaling (Fig. 3d). Co-expression patterns among genes/proteins across different subgroups were also analyzed using a Joint Random Forest (JRF) method that identified network modules, such as an MMP9 module,\",\n",
            "          \"method that identified network modules, such as an MMP9 module, with different interaction patterns between basal-enriched and luminal-enriched subgroups. These latter patterns appeared specific to the proteome-level data (Extended Data Fig. 8 c\\u2013f,\",\n",
            "          \"specific to the proteome-level data (Extended Data Fig. 8 c\\u2013f, Supplementary Table 17 and Supplementary Methods). Phosphosite markers in PIK3CA and TP53 mutant breast cancer TP53 and PIK3CA are the most recurrently mutated genes in breast cancer, with\",\n",
            "          \"are the most recurrently mutated genes in breast cancer, with frequencies for PIK3CA at 43% in luminal tumors and for TP53 at 84% in basal-like tumors. Most of the PIK3CA missense mutations were gain of function mutations and therefore expected to lead to\",\n",
            "          \"gain of function mutations and therefore expected to lead to an activation of the PI3K signaling cascade, but the extent to which this occurs has been controversial and there is uncertainty which pathway components are effectors. Marker selection analysis\",\n",
            "          \"pathway components are effectors. Marker selection analysis was therefore performed for upregulated phosphosites in PIK3CA-mutated tumors. In total, 62 phosphosites were identified that were positively associated with PIK3CA mutation (FDR<0.05), including\",\n",
            "          \"associated with PIK3CA mutation (FDR<0.05), including the kinases RPS6KA5 and EIF2AK4 (Extended Data Fig. 9a, Supplementary Table 18). Calculating the average phosphorylation signal of these marker phosphosites provided a read-out for PI3K pathway\",\n",
            "          \"these marker phosphosites provided a read-out for PI3K pathway activity in PIK3CA-mutated tumors, with 15 of the 26 mutated tumors (58%) exhibiting an activated PIK3CA mutation signature. Of note, the identified PIK3CA mutant phosphoproteome signature was\",\n",
            "          \"the identified PIK3CA mutant phosphoproteome signature was activated in all tumors harboring helical domain PIK3CA mutations but only 2 of 10 tumors harboring kinase domain mutations. To test if the identified differences in the phosphoproteome of PI3K\",\n",
            "          \"if the identified differences in the phosphoproteome of PI3K mutant versus wild-type tumors could be explained by mutation of PIK3CA, the tumor data were compared to phosphosite signatures derived from isogenic PIK3CA mutant cell lines (Extended Data Fig.\",\n",
            "          \"from isogenic PIK3CA mutant cell lines (Extended Data Fig. 9b, Supplementary Table 18). There was an enrichment of signatures derived from helical domain-mutated isogenic cell lines, but not from kinase domain-mutated cells, supporting the observations in\",\n",
            "          \"kinase domain-mutated cells, supporting the observations in primary tumors. The same strategy was used to identify phosphorylation signaling events connected to TP53 mutation. A total of 56 phosphosites upregulated in TP53 mutant tumors were identified\",\n",
            "          \"phosphosites upregulated in TP53 mutant tumors were identified that were independent of basal-like subtype association (Extended Data Fig. 9c, Supplementary Table 18). Using the average phosphorylation signal of these marker phosphosites as a proxy for\",\n",
            "          \"signal of these marker phosphosites as a proxy for TP53 mutation-driven cell cycle control, 22 of 41 mutated tumors (54%) showed upregulated signals. This TP53 mutant phosphosignature was somewhat enhanced in tumors in which mutations occurred almost\",\n",
            "          \"somewhat enhanced in tumors in which mutations occurred almost exclusively in the DNA binding region compared to those with non-sense/frameshift mutations. In addition to the well-described checkpoint kinase CHEK2, significantly upregulated phosphosites\",\n",
            "          \"checkpoint kinase CHEK2, significantly upregulated phosphosites were identified for the kinases MASTL and EEF2K in TP53-mutated tumors. Single sample GSEA analysis of isogenic p53-mutant phosphosignatures showed an enrichment of a phosphosignature derived\",\n",
            "          \"showed an enrichment of a phosphosignature derived from R273H mutated isogenic cells (Extended Data Fig. 9d), confirming the pronounced effect of missense mutations in the DNA-binding region on phosphorylation pathways. Identification of gene\",\n",
            "          \"region on phosphorylation pathways. Identification of gene amplification and breast cancer subtype-specific activated kinases in human breast cancer CNA spans many driver gene candidates and RNA expression has been frequently used to narrow candidate\",\n",
            "          \"and RNA expression has been frequently used to narrow candidate nominations. Proteogenomic analysis should further promote this nomination process. In this candidate refinement a focus on protein kinases is warranted, since many are drug targets. An\",\n",
            "          \"protein kinases is warranted, since many are drug targets. An in-depth proteogenomic pipeline was developed that flagged kinases, expression levels of which were at least 1.5 interquartile ranges higher than the median (Supplemental Table 19). A\",\n",
            "          \"ranges higher than the median (Supplemental Table 19). A proteogenomic circos-like plot (termed a \\u201cpircos\\u201d plot) was used to map these outlier kinase values onto the genome (Fig. 4a,b, Extended Data Fig. 10a). The ERBB2 locus showed the strongest effect\",\n",
            "          \"Data Fig. 10a). The ERBB2 locus showed the strongest effect of increased phosphoprotein levels associated with gene amplification-driven RNA and protein over expression (Fig. 4a). CDK12 is a positive transcriptional regulator of homologous recombination\",\n",
            "          \"positive transcriptional regulator of homologous recombination repair genes with its partner Cyclin K, and is often encompassed by the ERBB2 amplicon. This gene was also found to be upregulated at the RNA, protein and phosphosite level indicating that\",\n",
            "          \"at the RNA, protein and phosphosite level indicating that CDK12 is highly active in the majority of ERBB2 positive tumors (Fig. 4a). The analysis of the ERBB2 amplicon also uncovered co-outlier phosphorylation status for MED1, GRB7, MSL1, CASC3 and TOP2A,\",\n",
            "          \"phosphorylation status for MED1, GRB7, MSL1, CASC3 and TOP2A, all previously described in association with ERBB2 amplification. To better understand the downstream effects of ERBB2 amplification, additional phosphosite outliers were identified in 41 known\",\n",
            "          \"additional phosphosite outliers were identified in 41 known ERBB2 signaling genes for the 15 samples that had ERBB2 phosphosite outlier expression (Extended Data Fig. 10b). These canonical findings stimulated a proteogenomic analysis to identify\",\n",
            "          \"findings stimulated a proteogenomic analysis to identify additional outlier kinases in the breast cancer genome. A proteogenomic dissection of chromosome 11q based on PAK1 amplification (Fig. 4b,c), a breast cancer driver kinase, illustrated that PAK1 is\",\n",
            "          \"4b,c), a breast cancer driver kinase, illustrated that PAK1 is hyperphosphorylated in PAK1 amplified tumors, along with CLNS1A, RFS1 and GAB2. Additional examples of outlier kinases included PTK2 and RIPK2 in association with amplification of chromosome\",\n",
            "          \"PTK2 and RIPK2 in association with amplification of chromosome 8q (Fig. 4c; Extended Data Fig. 10a,c). PAK1 and TLK2 (17q23) appear to be luminal breast cancer specific events Fig. 4c; Extended Data Fig. 10c). To further examine whether outlier kinases\",\n",
            "          \"Data Fig. 10c). To further examine whether outlier kinases were breast cancer subtype-specific, independent of amplification status, the BH-corrected probability was calculated of finding that number of phosphosite outliers within a subtype, given the\",\n",
            "          \"that number of phosphosite outliers within a subtype, given the total number of outliers across all subtypes, the subtype sample size and the total sample size. (Fig. 4d). These analyses led to the expected identification of ERBB2 in the HER2-enriched\",\n",
            "          \"to the expected identification of ERBB2 in the HER2-enriched subtype at the 5% FDR level, as well as the new finding of CDC42BPG (MRGKG), an effector kinase for RHO-family GTPases. In basal-like breast cancer, two kinases, PRKDC and SPEG, were significant\",\n",
            "          \"breast cancer, two kinases, PRKDC and SPEG, were significant at the 5% FDR level. PRKDC is a non-homologous end-joining (NHEJ) factor that can be phosphorylated by ATM kinase, and is therefore a logical finding in this disease subset. However SPEG, a\",\n",
            "          \"a logical finding in this disease subset. However SPEG, a kinase associated with severe dilated cardiomyopathy when suppressed, has not been previously reported in association with breast cancer. A larger number of subtype-specific kinases were detected\",\n",
            "          \"A larger number of subtype-specific kinases were detected at the 10% FDR level, several of which have recently described relevance in breast cancer, including PRKD3 in basal-like breast cancer, the LKB-regulated SIK3 in luminal A breast cancer and CDK13\",\n",
            "          \"the LKB-regulated SIK3 in luminal A breast cancer and CDK13 in luminal B breast cancer, which, similar to CDK12, can interact with Cyclin K. Discussion The analytic breadth and depth of proteomic and phosphoproteomic analyses displayed in this study\",\n",
            "          \"proteomic and phosphoproteomic analyses displayed in this study demonstrates the strengths of mass spectrometry-based proteomics, but also some of the limitations inherent in proteolytic peptide sequencing (see Supplementary Discussion). An example of how\",\n",
            "          \"sequencing (see Supplementary Discussion). An example of how high-dimensional proteomic analysis provides insight into unresolved genomic issues concerns the study of loss of the long arm of chromosome 5 (5q). Analysis of RNA and protein correlations\",\n",
            "          \"of chromosome 5 (5q). Analysis of RNA and protein correlations narrowed the list of potential trans-deregulated proteins. Orthogonal candidate screening using functional genomics methods identified loss of CETN3 and SKP1 as potential transregulators, with\",\n",
            "          \"loss of CETN3 and SKP1 as potential transregulators, with upregulation of EGFR as a downstream consequence in basal-like breast cancers. While further experimental evidence must be sought for these proposed regulatory relationships, SKP1/Cullin complex\",\n",
            "          \"these proposed regulatory relationships, SKP1/Cullin complex has already been linked to EGFR activation in Glioma. Unfortunately EGFR targeting has not to date proven to be effective therapy in basal-like breast cancer. This might be due to the fact the\",\n",
            "          \"in basal-like breast cancer. This might be due to the fact the SKP1 loss deregulates multiple targets requiring a much broader inhibitory strategy. It is recognized that PIK3CA mutations do not strongly activate canonical downstream effectors. Mass\",\n",
            "          \"do not strongly activate canonical downstream effectors. Mass spectrometry-based phosphoproteomics provides an opportunity for unbiased examination of downstream signaling events consequent upon PIK3CA mutational activation. These studies revealed that\",\n",
            "          \"upon PIK3CA mutational activation. These studies revealed that common PIK3CA mutations affect a large number of targets with diverse functionalities including the kinases RPS6KA5 and EIF2AK4. Thus, the data and analyses reported here extend our knowledge\",\n",
            "          \"Thus, the data and analyses reported here extend our knowledge of the effectors that promote tumorigenesis in response to constitutive activation of PI3 kinase. Similarly, TP53 mutation-associated phosphopeptides point towards novel functionalities,\",\n",
            "          \"phosphopeptides point towards novel functionalities, including regulation of the kinases MASTL and EEF2K. A central goal in breast cancer research has been the identification of druggable kinases beyond HER2. Candidate genes that exhibited similar gene\",\n",
            "          \"beyond HER2. Candidate genes that exhibited similar gene amplification-driven proteogenomic patterns to HER2 included CDK12, TLK2, PAK1 and RIPK2. The proteogenomic link with gene amplification was particularly strong for CDK12, in keeping with its\",\n",
            "          \"was particularly strong for CDK12, in keeping with its location in the ERBB2 amplicon, while the strengths of correlation between DNA amplification, RNA, protein and phosphoprotein for the other examples were more variable. The presence of activated CDK12\",\n",
            "          \"examples were more variable. The presence of activated CDK12 in the ERBB2 amplicon might explain why tumors arising in BRCA1 carriers are usually ERBB2 negative. As a positive transcriptional regulator of BRCA1 and multiple FANC family members, CDK12\",\n",
            "          \"regulator of BRCA1 and multiple FANC family members, CDK12 promotes DNA repair by homologous recombination. CDK12 amplification would, therefore, oppose the functional effects of BRCA1 haploinsufficiency during tumor evolution. Overall, multiple outlier\",\n",
            "          \"during tumor evolution. Overall, multiple outlier kinases generate testable therapeutic hypotheses for which enabling inhibitors are in development. For example PAK1 has recently been confirmed to be a therapeutic target and poor prognosis factor in\",\n",
            "          \"to be a therapeutic target and poor prognosis factor in luminal breast cancer. Although incomplete outcome data and the remarkable heterogeneity of breast cancer are additional relevant constraints, the number of TCGA specimens analyzed here is\",\n",
            "          \"constraints, the number of TCGA specimens analyzed here is insufficient to support conclusive clinical correlations. Only 8 deaths occurred among the 77 patients, which are too few samples to provide sufficient statistical power for association analysis.\",\n",
            "          \"provide sufficient statistical power for association analysis. Adequately powered MS-based clinical investigation will require targeted approaches, especially given the highly limited amount of patient material available from clinical trials and the\",\n",
            "          \"of patient material available from clinical trials and the mostly formalin-fixed nature of the specimens. The current analysis is therefore centered on biological findings and correlations, with orthogonal validation and false discovery concerns addressed\",\n",
            "          \"orthogonal validation and false discovery concerns addressed through an examination of cell-line databases of the effects of individual gene perturbations. Typical of a multi-tiered analysis of this complexity, there are many hypotheses to test, and many\",\n",
            "          \"of this complexity, there are many hypotheses to test, and many findings that require further investigation. In conclusion, this study provides a high-quality proteomic resource for human breast cancer investigation and illustrates technologies and\",\n",
            "          \"breast cancer investigation and illustrates technologies and analytical approaches that provide an important new opportunity to connect the genome to the proteome. Larger-scale exploration of discovery proteomics in the clinical setting will require\",\n",
            "          \"of discovery proteomics in the clinical setting will require improvements in clinical investigation, including acquisition of adequate amounts of optimally collected tumor tissue both before and during therapy as well as advances in MS proteomics to\",\n",
            "          \"and during therapy as well as advances in MS proteomics to reduce sample input and increase sensitivity for low abundance proteins and modified peptides. Extended Data Experimental and data analysis workflows and longitudinal data generation quality\",\n",
            "          \"analysis workflows and longitudinal data generation quality control a, iTRAQ 4-plex global proteome and phosphoproteome analysis workflow. 105 TCGA breast tumors were analyzed in 35 iTRAQ 4-plex experiments (plus 1 replicate and 1 normal sample\",\n",
            "          \"iTRAQ 4-plex experiments (plus 1 replicate and 1 normal sample experiment), with three tumors of different subtypes compared to a fourth common internal reference sample in each experiment. The reference sample comprised 10 individual tumors of each of\",\n",
            "          \"The reference sample comprised 10 individual tumors of each of the 4 major breast cancer intrinsic subtypes and served as an internal standard for all proteins and phosphoproteins quantified in this study. Each iTRAQ MS/MS spectrum measures a peptide from\",\n",
            "          \"this study. Each iTRAQ MS/MS spectrum measures a peptide from 4 samples (3 individual patients and the reference sample mix of 40 patients). More than 400,000 distinct peptides were identified and quantified in ~14 million MS/MS spectra. Personalized\",\n",
            "          \"and quantified in ~14 million MS/MS spectra. Personalized tumor-specific protein databases were generated in the QUILTS software package using whole exome sequencing-derived variant calls and RNAseq-derived transcript information. All mass spectrometry\",\n",
            "          \"RNAseq-derived transcript information. All mass spectrometry data was analyzed using the Spectrum Mill software package. b, Overview of proteome and phosphoproteome datasets. The table provides a summary of the datasets used in specific analyses,\",\n",
            "          \"provides a summary of the datasets used in specific analyses, including the filters applied to derive the proteins and phosphosites/phosphoproteins that constitute each dataset; the protein, phosphosite or phosphoprotein count; and the methods that employ\",\n",
            "          \"or phosphoprotein count; and the methods that employ the respective datasets. c, Distribution of sequence coverage of the identified proteins with tryptic peptides detected by MS/MS, whiskers show the 5\\u201395 percentiles. d and e, Robust and accurate\",\n",
            "          \"show the 5\\u201395 percentiles. d and e, Robust and accurate proteome/phosphoproteome platform. Longitudinal performance was tested by repeated proteome and phosphoproteome analysis of patient-derived xenograft tumors. Scatterplots, histograms and Pearson\",\n",
            "          \"xenograft tumors. Scatterplots, histograms and Pearson correlations comparing individual replicate measurements are shown. Tumor sample quality control (I) a, Remark diagram showing sample processing and partitioning. Initial quality review encompassed\",\n",
            "          \"processing and partitioning. Initial quality review encompassed histopathological examination of H&E stained tissue slices. *For 3 samples no tumor cells were seen on histopathology (BH-A0E9, BH-A0C1, A2-A0SW). These samples were nevertheless included in\",\n",
            "          \"BH-A0C1, A2-A0SW). These samples were nevertheless included in the proteome analysis since other quality control standards were met (see below) and samples with 0% tumor cellularity on top or bottom sections were included in TCGA analyses. b, correlation\",\n",
            "          \"bottom sections were included in TCGA analyses. b, correlation of TCGA (top or bottom sections) and CPTAC histological assessment of neoplastic cellularity for samples (n = 105). The average and range of neoplastic cellularities were identical for CPTAC\",\n",
            "          \"and range of neoplastic cellularities were identical for CPTAC and TCGA histological assessments. Averages (standard deviations) for neoplastic cellularity were 76% (+/\\u2212 17) for CPTAC, 76% (+/\\u2212 15) for TCGA_Top, and 75% (+/\\u2212 18) for TCGA_Bottom\",\n",
            "          \"76% (+/\\u2212 15) for TCGA_Top, and 75% (+/\\u2212 18) for TCGA_Bottom histopathology slides (Supplementary Table 2). Note that in three CPTAC cases where no tumor cells were identified by histopathological assessment, numbers of protein-level somatic variants were\",\n",
            "          \"assessment, numbers of protein-level somatic variants were similar to all other tumors. The identified mutated proteins were TP53_R273C, NOP58_Q23E, TAGLN2_G154R, TUBA1B_D116H, and MRPL48_I173K (Supplementary Table 5), indicating presence of tumor cells\",\n",
            "          \"(Supplementary Table 5), indicating presence of tumor cells in these samples. c, Proteome iTRAQ tumor/internal reference ratio heatmap for all CPTAC samples (8,028 proteins without missing values) including passed and failed proteomic quality control (QC)\",\n",
            "          \"including passed and failed proteomic quality control (QC) samples. d, Global tumor/reference proteome ratio distributions for samples that passed and failed proteomic quality control analysis. e, Degradation-related gene sets were enriched in tumors that\",\n",
            "          \"e, Degradation-related gene sets were enriched in tumors that failed proteomic quality control analysis. f, Variant allele frequency (VAF) analysis of re-sequenced CPTAC tumors and comparison to original TCGA data. Overall VAFs for failed QC samples were\",\n",
            "          \"to original TCGA data. Overall VAFs for failed QC samples were lower compared to passed samples suggesting lower purity. Tumor sample quality control (II) a, There was high concordance (94.6%) between DNA variants reported by TCGA and CPTAC re-sequenced\",\n",
            "          \"between DNA variants reported by TCGA and CPTAC re-sequenced tumors. Most point mutations reported by TCGA could be identified across the 8 re-sequenced samples used in the study. b, A high overall correlation (mean=0.77) was observed for the CPTAC\",\n",
            "          \"high overall correlation (mean=0.77) was observed for the CPTAC Variant Allele Fraction (VAF) (X-axis) and TCGA VAF (Y-axis) across the 8 samples used in the study. c, Agglomerative hierarchical clustering (Supplementary Methods Section 3.8) used to\",\n",
            "          \"clustering (Supplementary Methods Section 3.8) used to co-cluster protein and RNA tumor expression data after filtering to retain 4,291 proteins and genes with moderate to high protein-RNA correlation (Pearson correlation > 0.4) with results displayed as\",\n",
            "          \"(Pearson correlation > 0.4) with results displayed as a circular dendrogram (fanplot). The proteome (.P) and RNA (.R) components of each sample are labeled using the same color. The outer ring shows proteome samples in light grey and RNA samples in dark\",\n",
            "          \"shows proteome samples in light grey and RNA samples in dark grey. High concordance between RNA and protein expression is evident from the color adjacency in the inner ring and alternating color in the outer ring showing that RNA and protein components\",\n",
            "          \"color in the outer ring showing that RNA and protein components co-cluster for a large proportion of samples (62/80). d, Co-clustering of MS and RPPA tumor data. 126 RPPA readouts were mapped to gene names. These genes were intersected with the genes\",\n",
            "          \"to gene names. These genes were intersected with the genes observed in the MS proteome, filtered to 48 proteins with moderate or higher RPPA-MS protein correlation, and analyzed for co-clustering as in c. 47 of 80 RPPA-MS protein pairs co-cluster. While\",\n",
            "          \"as in c. 47 of 80 RPPA-MS protein pairs co-cluster. While this is a smaller proportion than for RNA-protein analysis, the number of genes used in the clustering is significantly smaller for RPPA (48 vs. 4,291 for RNA). e, ESTIMATE tumor purity comparison\",\n",
            "          \"(48 vs. 4,291 for RNA). e, ESTIMATE tumor purity comparison between mRNA, RNAseq, and proteome data. ANOVA is used to assess the difference in distribution (\\u2212log10(p-value)) of ESTIMATE, stromal, immune, and tumor purity scores across mRNA (microarray),\",\n",
            "          \"immune, and tumor purity scores across mRNA (microarray), RNA-seq and proteome data. The only significant p-value (=0.02) is for the Cluster 3 stromal score, and higher stromal scores for the proteome drive that difference. f, Ischemia score analysis.\",\n",
            "          \"the proteome drive that difference. f, Ischemia score analysis. Comparison of ischemia scores of 77 CPTAC tumors, 3 normal samples, and patient-derived xenografts. CPTAC tumors had generally lower ischemia scores than PDX samples subjected to 30 minutes\",\n",
            "          \"lower ischemia scores than PDX samples subjected to 30 minutes of cold ischemia. Median ischemia scores are less than 30 minutes for each subtype and no significant differences were observed across subtypes. Effects due to cold ischemia therefore appear\",\n",
            "          \"across subtypes. Effects due to cold ischemia therefore appear to be negligible in this CPTAC sample collection. Protein-to-Protein, -CNA, and -mRNA correlation analyses a, Identification of UBE3A as an E3 ubiquitin ligase that negatively correlates to\",\n",
            "          \"UBE3A as an E3 ubiquitin ligase that negatively correlates to p53 on the protein level. Pearson correlation and Benjamini-Hochberg corrected p-value are shown. b, Analysis of counter-regulated genes with negative correlation of CNA-to-RNA as well as\",\n",
            "          \"genes with negative correlation of CNA-to-RNA as well as CNA-to-protein levels. Negative Pearson correlations are shown with Benjamini-Hochberg corrected p-values for CNA-to-protein correlations. Depicted genes have significant negative correlations at\",\n",
            "          \"Depicted genes have significant negative correlations at FDR<0.05 in the CNA-to-RNA and CNA-to-protein analyses. c, Global mRNA-to-protein correlation and gene set enrichment analysis. Global CNA effects and comparison of CNA TRANS effects to knockdown\",\n",
            "          \"CNA effects and comparison of CNA TRANS effects to knockdown signatures in the LINCS database a, CNA landscape in the CPTAC tumor collection. The segment-based CNAs of 77 samples were downloaded from TCGA Firehose, including 18 Basal, 12 Her2, 23 Luminal\",\n",
            "          \"from TCGA Firehose, including 18 Basal, 12 Her2, 23 Luminal A and 24 Luminal B subtypes. Copy number amplifications were marked in red and deletions in blue. The bottom color key represents the log2 transformed copy number value, with CNA=2 centered at 0.\",\n",
            "          \"log2 transformed copy number value, with CNA=2 centered at 0. Specific CNA events are seen for chromosome 5q and 10p regions in basal-like tumors. b, Correlations of copy number alterations (x-axis) to phosphoprotein levels (y-axis) highlight new CNA cis\",\n",
            "          \"to phosphoprotein levels (y-axis) highlight new CNA cis and trans effects. Significant (FDR<0.05) positive (red) and negative (green) correlations between CNA and phosphoproteins are indicated. Histograms show the fraction [%] of significant CNA trans\",\n",
            "          \"Histograms show the fraction [%] of significant CNA trans effects for each CNA gene. c, LINCS CMap analysis facilitates identification of novel functional candidates for CNA trans effects. Knockdown profiles were compared with CNA/protein trans effects\",\n",
            "          \"Knockdown profiles were compared with CNA/protein trans effects for 502 genes. Genes with a connectivity score >|90| were considered connected and significant cis effects were annotated at an FDR<0.05. d, Basal-like tumor-specific CNAs are candidate\",\n",
            "          \"at an FDR<0.05. d, Basal-like tumor-specific CNAs are candidate regulatory events for EGFR and SRC expression levels. Oncogenic kinases with significant CNA/protein trans effects (left panel), that were regulated in LINCS shRNA experiments (right panel; 4\",\n",
            "          \"that were regulated in LINCS shRNA experiments (right panel; 4 cell lines,) and directly measured as LINCS landmark genes, are shown alongside candidate regulatory genes CETN3 and SKP1. Clinical ER, PR, and HER2 annotation and PAM50 classification are\",\n",
            "          \"ER, PR, and HER2 annotation and PAM50 classification are shown in the header rows of each column. Proteome cluster heatmap and stability analysis a, K-means consensus clustering of proteome and phosphoproteome data identifies three subgroups:\",\n",
            "          \"proteome and phosphoproteome data identifies three subgroups: basal-enriched, luminal-enriched, and stromal-enriched. The heatmap represents all 1,521 proteins used for clustering (Dataset G8). b, Identification of optimal proteome clusters for QC-passed\",\n",
            "          \"b, Identification of optimal proteome clusters for QC-passed CPTAC breast cancer tumors. Proteome clusters were derived using consensus clustering based on 1000 resampled datasets, exploring the range of 2 to 6 k-means clusters. Visualization of consensus\",\n",
            "          \"range of 2 to 6 k-means clusters. Visualization of consensus matrices from k-means consensus clustering for k=3, 4, 5 and 6 target clusters. Consensus clustering was performed on 1,521 proteins with no missing values and SD>1.5. c, Silhouette plots were\",\n",
            "          \"with no missing values and SD>1.5. c, Silhouette plots were generated to evaluate the coherence of the clustering. Silhouette plots for k=3 and k=4 clusters showing a cleaner separation of clusters for k=3. d, Based on both visual inspection of the\",\n",
            "          \"of clusters for k=3. d, Based on both visual inspection of the consensus matrix and the delta plot assessing change in consensus cumulative distribution function (CDF) area, three robustly segregated groups were observed. Consensus cumulative distribution\",\n",
            "          \"groups were observed. Consensus cumulative distribution function (CDF) and delta area (change in CDF area) plots for 2\\u20136 clusters. Proteome cluster markers and enriched pathways a, Markers (based on SAM analysis; FDR<0.01) discriminate between proteome\",\n",
            "          \"(based on SAM analysis; FDR<0.01) discriminate between proteome clusters 1, 2 and 3 (compare to heatmap of proteins used to derive clusters depicted in Extended Data Fig. 6a). b, Applying a Fisher exact test-based enrichment analysis to the proteome,\",\n",
            "          \"a Fisher exact test-based enrichment analysis to the proteome, phosphoproteome and mRNA data, gene sets from MSigDB were identified that were unique for each proteome cluster. Heat map showing specific pathways comprising dominant biological themes that\",\n",
            "          \"specific pathways comprising dominant biological themes that are significantly differential by enrichment analysis between basal-enriched and luminal-enriched tumors (Fisher Exact Test Benjamini-Hochberg corrected p-values are shown; enrichment test\",\n",
            "          \"corrected p-values are shown; enrichment test performed on marker sets identified using SAM analysis; see Methods; compare to Figure 3c). c, Heatmap showing a selection of gene sets significant in basal-enriched or luminal-enriched tumors exclusively by\",\n",
            "          \"in basal-enriched or luminal-enriched tumors exclusively by mRNA, protein or phosphoprotein expression. Cytokine signatures, for example, were strongly captured at the mRNA level, but were seen to only a limited degree at the global protein level, likely\",\n",
            "          \"to only a limited degree at the global protein level, likely because of their typically low protein abundance. By contrast, the vast majority of significant gene sets annotated as \\\"signaling\\\" were enriched only at the phosphoprotein level. d, Global heat\",\n",
            "          \"were enriched only at the phosphoprotein level. d, Global heat map representing all gene sets significantly enriched in at least one of the proteomic breast cancer subtypes. The stromal-enriched group was characterized by breast cancer normal-like,\",\n",
            "          \"group was characterized by breast cancer normal-like, adipocyte differentiation, smooth muscle, toll-like receptor signaling and endothelin gene sets, supporting the clustering-based annotation of high stromal and/or adipose content in these tumors (see\",\n",
            "          \"of high stromal and/or adipose content in these tumors (see Supplemental Table 13). Phosphoproteome pathway clustering, kinase-phosphosite multivariate regression, and protein co-expression networks a, Phosphoproteome pathway clustering. Using\",\n",
            "          \"networks a, Phosphoproteome pathway clustering. Using phosphorylation state as a proxy for activity, deep phosphoproteome profiling allows development of a breast cancer molecular taxonomy based on signaling pathways. K-means consensus clustering was\",\n",
            "          \"based on signaling pathways. K-means consensus clustering was performed on pathways derived from single sample GSEA analysis of phosphopeptide data (908 pathways shown). Of four robustly segregated groups, subgroups 2 and 3 substantially recapitulated the\",\n",
            "          \"groups, subgroups 2 and 3 substantially recapitulated the stromal- and luminal-enriched proteomic subgroups, respectively. Subgroup 4 included a significant majority of tumors from the basal-enriched proteomic subgroup, but was admixed particularly with\",\n",
            "          \"proteomic subgroup, but was admixed particularly with luminal-enriched samples. This subgroup was defined by high levels of cell cycle and checkpoint activity. All basal and a majority of non-basal samples in this subgroup had TP53 mutations. Subgroup 1\",\n",
            "          \"samples in this subgroup had TP53 mutations. Subgroup 1 was a novel subgroup defined exclusively in the phosphopeptide / pathway activity domain, with no enrichment for either proteomic or PAM50 subtypes. It was defined by G-protein, G-protein coupled\",\n",
            "          \"PAM50 subtypes. It was defined by G-protein, G-protein coupled receptor, and inositol phosphate metabolism signatures, as well as ionotropic glutamate signaling. b, Analysis of the regulatory relationship between outlier kinases (see Supplementary Table\",\n",
            "          \"relationship between outlier kinases (see Supplementary Table 19) and phosphopeptides by regulatory multivariate regression analysis (see Methods) identified CDK1 as the most highly connected of the outlier Cyclin-Dependent Kinases, with highest\",\n",
            "          \"connected of the outlier Cyclin-Dependent Kinases, with highest centrality (based on node-degree; see Methods) among the outlier CDKs and seventh highest centrality among all the outlier kinases considered in the remMap analysis. Each line represents a\",\n",
            "          \"considered in the remMap analysis. Each line represents a phosphosite-kinase relationship. c\\u2013f, Analysis of differences in the co-expression patterns among genes/proteins across different subgroups. A Joint Random Forest (JRF) method was applied to\",\n",
            "          \"subgroups. A Joint Random Forest (JRF) method was applied to simultaneously build gene co-expression and protein co-expression networks (Supplementary Table 17, and Methods). Modules in these networks revealed different interaction patterns between\",\n",
            "          \"these networks revealed different interaction patterns between basal-enriched and luminal-enriched subgroups. c, Network module P1 of the protein co-expression network, defined chiefly in the proteome space. This module contained 12 genes connected by 39\",\n",
            "          \"proteome space. This module contained 12 genes connected by 39 edges, among which 34 were protein-specific and 5 were shared by both the protein and mRNA co-expression networks. Many edges were supported by published information and were contained in the\",\n",
            "          \"supported by published information and were contained in the STRING database. Edges in red are specific to the protein co-expression network; edges in green are shared by both protein and gene co-expression networks; edges indicated by double lines are\",\n",
            "          \"co-expression networks; edges indicated by double lines are contained in the STRING database with confidence score greater than 0.15. MMP9, one of the central proteins in this module, contributes to metastatic progression and is a potential target for\",\n",
            "          \"to metastatic progression and is a potential target for anti-metastatic therapies for basal-like / triple negative breast cancer. d, Heatmaps of the absolute correlation across each pair of genes in module P1 (shown in Panel c), based on either protein or\",\n",
            "          \"in module P1 (shown in Panel c), based on either protein or gene expression data for samples in the basal-enriched and luminal-enriched subgroups, respectively. The MMP9 protein was strongly co-expressed with the other members of the module only in the\",\n",
            "          \"co-expressed with the other members of the module only in the basal-enriched subgroup. Notably, this observation is dependent on protein data; the correlation at the mRNA level for this module was consistently low in both the basal-enriched and luminal\",\n",
            "          \"was consistently low in both the basal-enriched and luminal enriched subgroups indicating that these events coherently occur at the proteomic level. e, Co-expression network based on proteomics data. The network contains 693 proteomic network-specific\",\n",
            "          \"data. The network contains 693 proteomic network-specific edges (grey) and 792 edges shared with the RNAseq network (green). For each module, the most enriched category and corresponding Benjamini-Hochberg adjusted p-value is reported. Pie charts adjacent\",\n",
            "          \"adjusted p-value is reported. Pie charts adjacent to each module show the proportion of proteomics-specific edges (grey area) and edges shared between proteomics and RNAseq data (green area). f, RNAseq network. Phosphoproteome signatures of PIK3CA (a,b)\",\n",
            "          \"f, RNAseq network. Phosphoproteome signatures of PIK3CA (a,b) and TP53 (c,d) mutated tumors highlight activated key regulators and indicate frequency of activation a and c, Phosphosites upregulated in mutated tumors (SAM FDR<0.05 across all tumors and\",\n",
            "          \"in mutated tumors (SAM FDR<0.05 across all tumors and independently also across luminal tumors; average phosphosite signal for all markers shown as bar graph). To avoid confounding by intrinsic subtype-specific distinctions, only markers that were\",\n",
            "          \"intrinsic subtype-specific distinctions, only markers that were significantly identified both in analyses covering all tumors and analyses restricted to luminal tumors were selected (FDR <0.05). Color bars in the margins indicate FDRs for grouped analysis\",\n",
            "          \"Color bars in the margins indicate FDRs for grouped analysis of different mutation classes and indicate kinase substrates of known kinases in the respective pathways. Significantly regulated kinase phosphosites are annotated. The average phosphorylation\",\n",
            "          \"kinase phosphosites are annotated. The average phosphorylation signal of the marker phosphosites provides a read-out for PI3K and TP53 pathway activity in mutated tumors (histogram below heatmap). A 95% prediction confidence interval (indicated by dashed\",\n",
            "          \"A 95% prediction confidence interval (indicated by dashed lines) across the average signal in non-mutated tumors was chosen in order to discriminate active from non-active tumors. The most strongly activated PIK3CA kinase domain mutant tumor differed from\",\n",
            "          \"activated PIK3CA kinase domain mutant tumor differed from the other 9 kinase domain mutant tumors, as it contained an amino acid side chain charge neutral H1047L instead of the more common positively charged H1047R mutation. Among the 62 phosphosites\",\n",
            "          \"positively charged H1047R mutation. Among the 62 phosphosites identified that were significantly upregulated in PIK3CA mutated tumors, 13 phosphosites were found on phosphoproteins that are known substrates of well-annotated kinases in the PIK3CA pathway\",\n",
            "          \"substrates of well-annotated kinases in the PIK3CA pathway (panel a, right column). In the mutant TP53 analysis a total 20 phosphosites were found on phosphoproteins that are known substrates of well annotated kinases in the p53 pathway (panel c, right\",\n",
            "          \"of well annotated kinases in the p53 pathway (panel c, right column). b and d, Upregulated phosphosite sets were derived from isogenic PIK3CA and TP53 mutant versus wild-type cell line pairs and tested for enrichment within mutant versus wild-type CPTAC\",\n",
            "          \"and tested for enrichment within mutant versus wild-type CPTAC tumors using single sample GSEA. Significantly enriched phosphosite sets are shown (p<0.05). PIRCOS plots, kinase outliers and outliers in the ERBB2 pathway a, Pircos (Proteogenomics CIRCOS)\",\n",
            "          \"outliers in the ERBB2 pathway a, Pircos (Proteogenomics CIRCOS) plots for 8q and 17q showing median CNA, RNA, protein, and phosphosite expression for 20 tumors with amplification in 8q based on RIPK2 CNA>1; 23 tumors with amplification in 8q based on PTK2\",\n",
            "          \"RIPK2 CNA>1; 23 tumors with amplification in 8q based on PTK2 CNA>1; 15 tumors with amplification in 17q based on CDK12 CNA >1; and 10 tumors with amplification in 17q based on TLK2 CNA>1. Red indicates expression >1, blue < \\u22121, and grey between \\u22121 and 1.\",\n",
            "          \"indicates expression >1, blue < \\u22121, and grey between \\u22121 and 1. Genes with both copy number amplification (CNA>1) and increased phosphosite expression (p-site>1) are labeled. b, Phosphosite outliers in known ERBB2 signaling genes. To better understand the\",\n",
            "          \"in known ERBB2 signaling genes. To better understand the downstream effects of ERBB2 amplification, phosphosite outliers in known ERBB2 signaling genes (MSigDB pathway set, KEGG_ERBB_SIGNALING PATHWAY) were identified for the 15 samples that had ERBB2\",\n",
            "          \"PATHWAY) were identified for the 15 samples that had ERBB2 phosphosite outlier status. Forty-one genes were identified as having a phosphosite outlier in at least one of the ERBB2 amplified samples. PAK4 and ARAF phosphosite outlier status were found in\",\n",
            "          \"samples. PAK4 and ARAF phosphosite outlier status were found in seven of the 15 ERBB2 kinase outlier samples; GSK3B outliers were found in 6 samples; and EIF4EBP1, MAP2K2, ABL1 and AKT1 outlier status was found in 5 of the 15 samples. c, Proteogenomic\",\n",
            "          \"status was found in 5 of the 15 samples. c, Proteogenomic outlier expression analysis for TLK2 and RIPK2. Samples with outlier phosphosite (red), protein (yellow), RNA (green) and copy number (purple) expression are shown. Phosphosite squares indicate\",\n",
            "          \"(purple) expression are shown. Phosphosite squares indicate per-sample outlier phosphosites. Supplementary Material Supplementary Information is available in the online version of the paper. Author Contributions P.M., D.R.M., M.A.G., K.R.C., and S.A.C.\",\n",
            "          \"Author Contributions P.M., D.R.M., M.A.G., K.R.C., and S.A.C. designed the proteomic analysis experiments, data analysis workflow, and proteomic-genomic data comparisons. P.M., M.A.G., J.W.Q., and S.A.C. directed and performed proteomic analysis of breast\",\n",
            "          \"and S.A.C. directed and performed proteomic analysis of breast tumor and quality control samples. P.M., D.R.M., K.V.R., K.R.C., P.W., X.W., S.C., E.K., F.P., Z.T., J.L., M.L.G., M.W., V.Y., K.H., C.L., M.D.M., P.Y., J.W., B.Z., and D.F. performed\",\n",
            "          \"V.Y., K.H., C.L., M.D.M., P.Y., J.W., B.Z., and D.F. performed proteomic-genomic data analyses. D.R.M., P.W., and S.J.S. provided statistical support. D.R.M., K.V.R., K.R.C., K.K. and D.F. performed analyses of mass spectrometry data and adapted\",\n",
            "          \"D.F. performed analyses of mass spectrometry data and adapted algorithms and software for data analysis. S.R.D., R.R.T and M.J.E. developed and prepared breast xenografts used as quality control samples. P.M. and F.M prepared and analyzed cell lines for\",\n",
            "          \"samples. P.M. and F.M prepared and analyzed cell lines for correlative functional annotation of frequently mutated genes. P.M., D.R.M., M.A.G., and S.A.C designed strategy for quality control analyses. M.A.G., S.R.D., C.R.K., M.M., and H.R. coordinated\",\n",
            "          \"analyses. M.A.G., S.R.D., C.R.K., M.M., and H.R. coordinated acquisition, distribution and quality control evaluation of TCGA tumor samples. P.M., M.A.G., C.P., L.D., A.G.P., and M.J.E. interpreted data in the context of breast cancer biology. P.M.,\",\n",
            "          \"interpreted data in the context of breast cancer biology. P.M., D.R.M, M.A.G., K.R.C., P.W., A.G.P, M.J.E. and S.A.C. wrote the manuscript. All primary mass spectrometry data are deposited at the CPTAC Data Portal as raw and mzML files and complete\",\n",
            "          \"at the CPTAC Data Portal as raw and mzML files and complete protein assembly data sets for public access (https://cptac-data-portal.georgetown.edu/cptac/s/S029). In addition, a set of ancillary files such as dataset G1/P1, G3/P3, G4/P4, G5/P5, G7/P7, CNA\",\n",
            "          \"files such as dataset G1/P1, G3/P3, G4/P4, G5/P5, G7/P7, CNA correlation tables for CNA-mRNA, CNA-proteome and CNA-phosphoproteome; CNA data, RNA-seq expression data have also been deposited at the DCC. Two browsers have been created to assist the\",\n",
            "          \"at the DCC. Two browsers have been created to assist the interested reader in exploring the results. One provides track hubs for viewing the identified peptides in the UCSD genome browser (http://fenyolab.org/cptac_breast_ucsc). The second is an on-line\",\n",
            "          \"The second is an on-line tool for proteogenomic data exploration and can be accessed at http://prot-shiny-vm.broadinstitute.org:3838/BC2016/. See Supplemental Methods for descriptions. The authors declare no competing financial interests. Proteogenomic\",\n",
            "          \"authors declare no competing financial interests. Proteogenomic analysis of human breast cancer. Direct effects of genomic alterations on protein level Overlap of a, protein coding single amino acid variants (SAAVs) and b, RNA splice junctions not present\",\n",
            "          \"acid variants (SAAVs) and b, RNA splice junctions not present in RefSeq v60 detected by DNA exome sequencing, RNA-seq, and LC-MS/MS. Proportions of novel variants are noted. c, Heatmap of mutations/CNA and their effects on RNA and protein expression of\",\n",
            "          \"and their effects on RNA and protein expression of breast cancer-relevant genes across tumor and normal samples. ER, PR, HER2 and PAM50 status are annotated. Median iTRAQ protein abundance ratio and the most frequently detected and differential\",\n",
            "          \"ratio and the most frequently detected and differential phosphosite ratio are shown for each gene. Pearson correlations between MS protein vs RNA-seq and MS protein vs RPPA are indicated. Effects of copy number alterations (CNA) on mRNA, protein, and\",\n",
            "          \"Effects of copy number alterations (CNA) on mRNA, protein, and phosphoprotein abundance a, Correlations of CNA (x-axes) to RNA and protein expression levels (y-axes) highlight new CNA cis and trans effects. Significant (FDR<0.05) positive (red) and\",\n",
            "          \"and trans effects. Significant (FDR<0.05) positive (red) and negative (green) correlations between CNA and mRNAs or proteins are indicated. CNA cis effects appear as a red diagonal line, CNA trans effects as vertical stripes. Histograms show the fraction\",\n",
            "          \"trans effects as vertical stripes. Histograms show the fraction [%] of significant CNA trans effects for each CNA gene. b, Overlap of cis effects observed at RNA, protein, and phosphoprotein levels (FDR<0.05). c, Trans-effect regulatory candidates\",\n",
            "          \"levels (FDR<0.05). c, Trans-effect regulatory candidates identified among those with significant protein cis-effects using LINCS CMap. Bars indicate total numbers of significant CNA/protein trans effects (gray; FDR<0.05) and overlap with regulated genes\",\n",
            "          \"trans effects (gray; FDR<0.05) and overlap with regulated genes in LINCS knock-down profiles (red; 4 cell lines; moderated T-test FDR<0.1). Proteomic and phosphoproteomic subtypes of breast cancer and subtype-specific pathway enrichment a, Unsupervised\",\n",
            "          \"cancer and subtype-specific pathway enrichment a, Unsupervised clustering of RNA-seq and proteomics data restricted to PAM50 genes and subset of 35 detected proteins reveal high similarity to PAM50 (TCGA) sample annotation. b, K-means consensus clustering\",\n",
            "          \"PAM50 (TCGA) sample annotation. b, K-means consensus clustering of proteome and phosphoproteome data identifies basal-enriched, luminal-enriched, and stromal-enriched subgroups. c, Gene set enrichment analysis highlights sets of pathways significantly\",\n",
            "          \"enrichment analysis highlights sets of pathways significantly differential between basal-enriched and luminal-enriched tumors (detailed in Extended Data Fig. 7b). d, K-means consensus clustering performed on pathways derived from single sample GSEA\",\n",
            "          \"performed on pathways derived from single sample GSEA analysis of phosphopeptide data identifies four distinct clusters. Example analyses of aberrantly regulated kinases in human breast cancer a and b, PIRCOS (Proteogenomics CIRCOS) plots showing CNA,\",\n",
            "          \"a and b, PIRCOS (Proteogenomics CIRCOS) plots showing CNA, RNA, protein and phosphosite expression for 17 tumors with amplification in 17q (ERBB2 CNA>1) and 8 tumors with amplification in 11q (PAK1 CNA>1). Labeled genes have CNA>1 and phosphosite>1. c,\",\n",
            "          \"(PAK1 CNA>1). Labeled genes have CNA>1 and phosphosite>1. c, Proteogenomic outlier expression analysis for ERBB2, CDK12, and PAK1. Samples with outlier phosphosite (red), protein (yellow), RNA (green) and copy number (purple) expression are shown.\",\n",
            "          \"RNA (green) and copy number (purple) expression are shown. Phosphosite squares indicate per-sample outlier phosphosites. d, Outlier kinase events by PAM50 subtype (>35% of subtype samples contain a phosphosite outlier; <10% FDR using Benjamini-Hochberg\",\n",
            "          \"a phosphosite outlier; <10% FDR using Benjamini-Hochberg adjusted p-values).\"\n",
            "        ]\n",
            "      }\n",
            "    ]\n",
            "  },\n",
            "  \"PDC000173\": {\n",
            "    \"study_id\": \"b8da9eeb-57b8-11e8-b07a-00a098d917f8\",\n",
            "    \"pdc_study_id\": \"PDC000173\",\n",
            "    \"study_submitter_id\": \"TCGA BRCA Proteome S015-1\",\n",
            "    \"program_id\": \"10251935-5540-11e8-b664-00a098d917f8\",\n",
            "    \"project_id\": \"48af5040-5546-11e8-b664-00a098d917f8\",\n",
            "    \"study_name\": \"TCGA Breast Cancer Proteome\",\n",
            "    \"program_name\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"project_name\": \"CPTAC2 Retrospective\",\n",
            "    \"disease_type\": \"Breast Invasive Carcinoma;Other\",\n",
            "    \"primary_site\": \"Breast;Not Reported\",\n",
            "    \"analytical_fraction\": \"Proteome\",\n",
            "    \"experiment_type\": \"iTRAQ4\",\n",
            "    \"embargo_date\": null,\n",
            "    \"cases_count\": 109,\n",
            "    \"aliquots_count\": 109,\n",
            "    \"protocol_id\": \"1c06aab9-3b26-11e9-9a07-0a80fada099c\",\n",
            "    \"protocol_submitter_id\": \"P-S015-1\",\n",
            "    \"program_submitter_id\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"protocol_name\": \"TCGA_Breast_Cancer_Proteome\",\n",
            "    \"protocol_date\": \"2013-02-02\",\n",
            "    \"publications\": [\n",
            "      {\n",
            "        \"publication_id\": \"44e2e766-89c7-11e8-bcf1-0a2705229b82\",\n",
            "        \"pubmed_id\": \"27251275\",\n",
            "        \"doi\": \"10.1038/nature18003\",\n",
            "        \"title\": \"Proteogenomics connects somatic mutations to signalling in breast cancer\",\n",
            "        \"journal\": \"Nature\",\n",
            "        \"journal_url\": \"https://www.nature.com/articles/nature18003\",\n",
            "        \"year\": \"2016\",\n",
            "        \"abstract\": \"Somatic mutations have been extensively characterized in breast cancer, but the effects of these genetic alterations on the proteomic landscape remain poorly understood. Here we describe quantitative mass-spectrometry-based proteomic and phosphoproteomic analyses of 105 genomically annotated breast cancers, of which 77 provided high-quality data. Integrated analyses provided insights into the somatic cancer genome including the consequences of chromosomal loss, such as the 5q deletion characteristic of basal-like breast cancer. Interrogation of the 5q trans-effects against the Library of Integrated Network-based Cellular Signatures, connected loss of CETN3 and SKP1 to elevated expression of epidermal growth factor receptor (EGFR), and SKP1 loss also to increased SRC tyrosine kinase. Global proteomic data confirmed a stromal-enriched group of proteins in addition to basal and luminal clusters, and pathway analysis of the phosphoproteome identified a G-protein-coupled receptor cluster that was not readily identified at the mRNA level. In addition to ERBB2, other amplicon-associated highly phosphorylated kinases were identified, including CDK12, PAK1, PTK2, RIPK2 and TLK2. We demonstrate that proteogenomic analysis of breast cancer elucidates the functional consequences of somatic mutations, narrows candidate nominations for driver genes within large deletions and amplified regions, and identifies therapeutic targets.\",\n",
            "        \"full_text\": [\n",
            "          \"Proteogenomics connects somatic mutations to signaling in breast cancer Summary Somatic mutations have been extensively characterized in breast cancer, but the effects of these genetic alterations on the proteomic landscape remain poorly understood. We\",\n",
            "          \"on the proteomic landscape remain poorly understood. We describe quantitative mass spectrometry-based proteomic and phosphoproteomic analyses of 105 genomically annotated breast cancers of which 77 provided high-quality data. Integrated analyses allowed\",\n",
            "          \"77 provided high-quality data. Integrated analyses allowed insights into the somatic cancer genome including the consequences of chromosomal loss, such as the 5q deletion characteristic of basal-like breast cancer. The 5q trans effects were interrogated\",\n",
            "          \"breast cancer. The 5q trans effects were interrogated against the Library of Integrated Network-based Cellular Signatures, thereby connecting CETN3 and SKP1 loss to elevated expression of EGFR, and SKP1 loss also to increased SRC. Global proteomic data\",\n",
            "          \"and SKP1 loss also to increased SRC. Global proteomic data confirmed a stromal-enriched group in addition to basal and luminal clusters and pathway analysis of the phosphoproteome identified a G Protein-coupled receptor cluster that was not readily\",\n",
            "          \"a G Protein-coupled receptor cluster that was not readily identified at the mRNA level. Besides ERBB2, other amplicon-associated, highly phosphorylated kinases were identified, including CDK12, PAK1, PTK2, RIPK2 and TLK2. We demonstrate that proteogenomic\",\n",
            "          \"PAK1, PTK2, RIPK2 and TLK2. We demonstrate that proteogenomic analysis of breast cancer elucidates functional consequences of somatic mutations, narrows candidate nominations for driver genes within large deletions and amplified regions, and identifies\",\n",
            "          \"within large deletions and amplified regions, and identifies therapeutic targets. Introduction A central deficiency in our knowledge of cancer concerns how genomic changes drive the proteome and phosphoproteome to execute phenotypic characteristics. The\",\n",
            "          \"and phosphoproteome to execute phenotypic characteristics. The initial proteomic characterization in the TCGA breast study was performed using reversed phase protein arrays; however this approach is restricted by antibody availability. To provide greater\",\n",
            "          \"is restricted by antibody availability. To provide greater analytical breadth, the NCI Clinical Proteomic Tumor Analysis Consortium (CPTAC) is analyzing the proteomes of genome-annotated TCGA tumor specimens using mass spectrometry. Herein we describe\",\n",
            "          \"tumor specimens using mass spectrometry. Herein we describe integrated proteogenomic analyses of TCGA breast cancer samples representing the four principal mRNA-defined breast cancer intrinsic subtypes. Proteogenomic analysis of TCGA samples 105 breast\",\n",
            "          \"subtypes. Proteogenomic analysis of TCGA samples 105 breast tumors previously characterized by the TCGA were selected for proteomic analysis after histopathological documentation (Supplementary Tables 1 and 2). The cohort included a balanced\",\n",
            "          \"(Supplementary Tables 1 and 2). The cohort included a balanced representation of PAM50-defined intrinsic subtypes including 25 basal-like, 29 luminal A, 33 luminal B, and 18 HER2 (ERBB2)-enriched tumors, along with 3 normal breast tissue samples. Samples\",\n",
            "          \"tumors, along with 3 normal breast tissue samples. Samples were analyzed by high-resolution, accurate mass, tandem mass spectrometry (MS) that included extensive peptide fractionation and phosphopeptide enrichment (Extended Data Fig. 1a). An isobaric\",\n",
            "          \"phosphopeptide enrichment (Extended Data Fig. 1a). An isobaric peptide labeling approach (iTRAQ) was employed to quantify protein and phosphosite levels across samples, with 37 iTRAQ 4-plexes analyzed in total. A total of 15,369 proteins (12,405 genes)\",\n",
            "          \"analyzed in total. A total of 15,369 proteins (12,405 genes) and 62,679 phosphosites were confidently identified with 11,632 proteins/tumor and 26,310 phosphosites/tumor on average (Supplementary Tables 3, 4 and Supplementary Methods). After filtering for\",\n",
            "          \"Tables 3, 4 and Supplementary Methods). After filtering for observation in at least a quarter of the samples (Supplementary Methods, Extended Data Fig. 1b) 12,553 proteins (10,062 genes) and 33,239 phosphosites, with their relative abundances quantified\",\n",
            "          \"33,239 phosphosites, with their relative abundances quantified across tumors, were used in subsequent analyses in this study. Stable longitudinal performance and low technical noise were demonstrated by repeated, interspersed analyses of a single batch of\",\n",
            "          \"by repeated, interspersed analyses of a single batch of patient-derived luminal and basal breast cancer xenograft samples (Extended Data Fig. 1d,e). Due to the heterogeneous nature of breast tumors, and because proteomic analyses were performed on tumor\",\n",
            "          \"tumors, and because proteomic analyses were performed on tumor fragments that were different from those used in the genomic analyses, rigorous pre-specified sample and data QC metrics were implemented (Supplementary Discussion and Extended Data Figures 2,\",\n",
            "          \"(Supplementary Discussion and Extended Data Figures 2, 3). Extensive analyses concluded that 28 of the 105 samples were compromised by protein degradation. These samples were excluded from further analysis with subsequent informatics focused on the 77\",\n",
            "          \"further analysis with subsequent informatics focused on the 77 tumor samples and three biological replicates. Genome and transcriptomic variation was observed at the peptide level by searching MS/MS spectra not matched to RefSeq against a patient-specific\",\n",
            "          \"MS/MS spectra not matched to RefSeq against a patient-specific sequence database (Fig. 1a). The database was constructed using the QUILTS software package leveraging RefSeq gene models based on whole exome and RNA-seq data generated from portions of the\",\n",
            "          \"on whole exome and RNA-seq data generated from portions of the same tumors and matched germline DNA (Fig. 1a, Supplementary Table 5). While these analyses detected a number of single amino-acid variants (SAAVs), frameshifts, and splice junctions,\",\n",
            "          \"amino-acid variants (SAAVs), frameshifts, and splice junctions, including splice isoforms that had been detected as only single transcript reads by RNA-seq (Fig. 1b, Supplementary Table 5), the number of genomic and transcriptomic variants that were\",\n",
            "          \"5), the number of genomic and transcriptomic variants that were confirmed as peptides by MS was low (Supplementary Discussion). Sparse detection of individual genomic variants by peptide sequencing has been noted in our previous studies and reflects\",\n",
            "          \"sequencing has been noted in our previous studies and reflects limited coverage at the single amino-acid level with current technology. However quantitative MS analysis of multiple peptides for each protein is used to reliably infer overall protein\",\n",
            "          \"for each protein is used to reliably infer overall protein levels. This is an advantage for MS since antibody-based protein expression analysis is typically based on a single epitope. To illustrate this capability in the current data set an initial\",\n",
            "          \"illustrate this capability in the current data set an initial analysis of three frequently mutated genes in breast cancer (TP53, PIK3CA, and GATA3) and three clinical biomarkers (ER, PGR, and ERBB2) was conducted (Fig. 1c, Supplementary Table 6, 7 and\",\n",
            "          \"and ERBB2) was conducted (Fig. 1c, Supplementary Table 6, 7 and Supplementary Discussion). As expected, TP53 missense mutations were associated with elevated MS-based protein levels, as observed by RPPA (Reverse Phase Protein Array), especially in\",\n",
            "          \"observed by RPPA (Reverse Phase Protein Array), especially in basal-like breast cancer. TP53 nonsense and frame-shift mutations were associated with a decrease in TP53 protein levels that was particularly striking in the MS data. In contrast, the mostly\",\n",
            "          \"particularly striking in the MS data. In contrast, the mostly C-terminal GATA3 frame-shift alterations did not result in decreased protein expression when measured by the median of all GATA3 peptides, suggesting these proteins are expressed despite\",\n",
            "          \"GATA3 peptides, suggesting these proteins are expressed despite truncation. No consistent effect of somatic PIK3CA mutation was observed at the level of protein expression. Good correlations between RNA-seq and MS-protein expression levels were found for\",\n",
            "          \"between RNA-seq and MS-protein expression levels were found for ESR1 (r=0.74), PGR (r=0.74), ERBB2 (r=0.84) and GATA3 (r=0.83) with moderate correlations observed for PIK3CA (r=0.45) and TP53 (r=0.36). Lower TP53 protein abundance levels compared to mRNA\",\n",
            "          \"(r=0.36). Lower TP53 protein abundance levels compared to mRNA levels were especially prevalent in luminal tumors, suggesting post-transcriptional regulatory mechanisms such as proteasomal degradation. To explore this hypothesis a search was made for E3\",\n",
            "          \"To explore this hypothesis a search was made for E3 ligases that showed negative correlation to p53 protein (Supplementary Table 8). These analyses identified UBE3A (r=\\u22120.42; adj. pval=0.05) (Extended Data Fig. 4a), an established TP53 E3 ligase. In\",\n",
            "          \"(Extended Data Fig. 4a), an established TP53 E3 ligase. In comparing CNA, RNA, and protein levels for GATA3, copy number gains in chromosome 10q were anti-correlated with RNA and protein levels in basal-like tumors. This observation prompted a search for\",\n",
            "          \"in basal-like tumors. This observation prompted a search for other gains or losses that were anticorrelated with RNA and/or protein levels (see Extended Data Fig. 4b for further analyses). Overall, six genes were identified that significantly\",\n",
            "          \"Overall, six genes were identified that significantly anti-correlated at an FDR<0.05 on both RNA and protein level to their CNA signals (Extended Data Fig. 4b). GATA3 amplification on 10q in basal-like breast cancer showed the strongest anti-correlation,\",\n",
            "          \"basal-like breast cancer showed the strongest anti-correlation, followed by the hexosamine and glycolysis pathway enzymes GFPT2 and HK3, which are upregulated in basal-like breast cancer despite being subjected to frequent chromosomal deletion on 5q.\",\n",
            "          \"despite being subjected to frequent chromosomal deletion on 5q. Global analysis of the correlation of mRNA-to-protein yielded a median Pearson value of r=0.39, with 6,135 out of 9,302 protein/mRNA pairs (66.0%) correlating significantly at an FDR<0.05\",\n",
            "          \"pairs (66.0%) correlating significantly at an FDR<0.05 (Extended Data Fig. 4c, Supplementary Table 9 and Supplementary Discussion). Similar to the colon cancer analysis metabolic functions such as amino acid, sugar and fatty acid metabolism were found to\",\n",
            "          \"as amino acid, sugar and fatty acid metabolism were found to be enriched among positively correlated genes whereas ribosomal, RNA polymerase and mRNA splicing functions were negatively correlated. Overall these analyses demonstrate the utility of global\",\n",
            "          \"Overall these analyses demonstrate the utility of global proteome correlation analysis for both confirmation of suspected regulatory mechanisms and identification of candidate regulators meriting further investigation. Copy Number Alterations To determine\",\n",
            "          \"further investigation. Copy Number Alterations To determine the consequences of CNA on mRNA, protein and phosphoprotein abundance, both in \\u201ccis\\u201d on genes within the aberrant locus and in \\u201ctrans\\u201d on genes encoded elsewhere, univariate correlation analysis\",\n",
            "          \"on genes encoded elsewhere, univariate correlation analysis was used as previously described. A total of 7,776 genes with CNA, mRNA and protein measurements were analyzed by calculating Pearson correlation and associated statistical significance\",\n",
            "          \"Pearson correlation and associated statistical significance (Benjamini-Hochberg corrected p-value) for all possible CNA-mRNA and CNA-protein pairs (Fig. 2a, Supplementary Table 10, Extended Data Fig. 5a, see Methods). For the phosphoproteome, 4,472\",\n",
            "          \"Data Fig. 5a, see Methods). For the phosphoproteome, 4,472 CNA-phosphoprotein pairs were analyzed (Extended Data Fig. 5b). Significant positive correlations (cis) were observed for 64% of all CNA-mRNA, 31% of all CNA-protein, and 20% of all\",\n",
            "          \"for 64% of all CNA-mRNA, 31% of all CNA-protein, and 20% of all CNA-phosphoprotein pairs Fig. 2b. Proteins and phosphoproteins correlated in cis to CNA were, for the most part, a subset of the cis effects observed in mRNA-to-CNA correlation (Fig. 2b,\",\n",
            "          \"the cis effects observed in mRNA-to-CNA correlation (Fig. 2b, Supplementary Table 10). The fractional difference of well-annotated oncogenes and tumor suppressor genes among the significantly cis-correlated CNA-to-mRNA and -protein gene pairs was\",\n",
            "          \"cis-correlated CNA-to-mRNA and -protein gene pairs was analyzed. Based on a reference list of 487 oncogenes and tumor suppressors (Supplementary Table 10), these cancer relevant genes occur 37.6% more frequently in the subset of genes that correlate both\",\n",
            "          \"more frequently in the subset of genes that correlate both on CNA/mRNA and CNA/protein levels than in the subset that only correlate on CNA/mRNA but not on CNA/protein level (Fisher exact p-value=0.02). This suggests that CNA events with a tumor promoting\",\n",
            "          \"This suggests that CNA events with a tumor promoting outcome more likely lead to cis regulatory effects on both the protein and mRNA level, whereas CNA events with no documented role in tumorigenesis are more likely to be neutralized on the protein level\",\n",
            "          \"are more likely to be neutralized on the protein level than on the RNA level. Trans effects (Fig. 2a) appear as vertical bands, with accompanying frequency histograms (in blue) highlighting \\u201chot spots\\u201d of significant trans effects. Using a minimum\",\n",
            "          \"\\u201chot spots\\u201d of significant trans effects. Using a minimum threshold of 50 trans-affected genes, 68% of the tested genes were associated with trans effects on the mRNA level, whereas only 13% were associated with effects on the protein level and 8% on the\",\n",
            "          \"were associated with effects on the protein level and 8% on the phosphoprotein level. Importantly, CNA-protein correlations appeared to be a reduced representation of CNA-mRNA correlations. Furthermore, for many CNA regions correlations were more\",\n",
            "          \"Furthermore, for many CNA regions correlations were more directionally uniform on the protein level than on the mRNA level. CNA regions exhibiting the most trans associations at the protein level were found on chromosomes 5q (LOH in basal; gain in LumB),\",\n",
            "          \"were found on chromosomes 5q (LOH in basal; gain in LumB), 10p (gain in basal), 12 (gain in basal), 16q (LumA deletion), 17q (LumB amplification), and 22q (LOH in luminal and basal) (Extended Data Fig. 5a). Trans associations are not necessarily direct\",\n",
            "          \"Data Fig. 5a). Trans associations are not necessarily direct consequences of the chromosomal aberration. For example since 5q loss occurs in at least 50% of basal-like breast cancers, many of the trans effects involve genes that mark the basal-subtype. To\",\n",
            "          \"the trans effects involve genes that mark the basal-subtype. To identify candidate driver genes whose copy number alterations are direct drivers of trans effects, results were compared with functional knock-down data on 3,797 genes in the Library of\",\n",
            "          \"functional knock-down data on 3,797 genes in the Library of Integrated Cellular Signatures (LINCS) database (http://www.lincsproject.org/). For any given gene with copy number alterations (\\u201cCNA-gene\\u201d), sets of genes were identified corresponding to\",\n",
            "          \"(\\u201cCNA-gene\\u201d), sets of genes were identified corresponding to proteins that changed where there was gain (\\u201cCNA-gain trans gene set\\u201d) or loss (\\u201cCNA-loss trans gene set\\u201d). These gene sets were then compared to the effects of gene knock down in the LINCS\",\n",
            "          \"then compared to the effects of gene knock down in the LINCS database (see Methods). Queries for 502 different CNA-genes meeting the criteria defined above identified 10 CNA-genes that could be functionally connected to both CNA-gain and CNA-loss trans\",\n",
            "          \"be functionally connected to both CNA-gain and CNA-loss trans protein-level effects (Extended Data Fig. 5c, Supplementary Table 11). A permutation-based approach implemented to test significance (see Supplementary Methods) yielded an FDR < 0.05 for 10\",\n",
            "          \"(see Supplementary Methods) yielded an FDR < 0.05 for 10 genes affected by both CNA gains and losses (Fig. 2c). These proteins were defined as potential regulatory candidates for the CNA trans effects observed on the proteome level in this study, since in\",\n",
            "          \"effects observed on the proteome level in this study, since in a gene-dependent manner on average 17% of these trans effects were consistent with the knockdown profiles. Notably, the established oncogenic receptor tyrosine kinase ERBB2 was functionally\",\n",
            "          \"oncogenic receptor tyrosine kinase ERBB2 was functionally connected only to CNA gain trans effects (Supplementary Table 11). The E3 ligase SKP1 and the ribonucleoprotein export factor CETN3, both located on chromosome arm 5q with frequent losses in\",\n",
            "          \"both located on chromosome arm 5q with frequent losses in basal-like breast cancer and less frequent gains in luminal B breast cancer, were detected as potential regulators affecting the expression of the tyrosine kinase and therapeutic target EGFR, and\",\n",
            "          \"of the tyrosine kinase and therapeutic target EGFR, and SKP1 also was linked to SRC (Extended Data Fig. 5d). Another potential regulator, FBXO7, (a substrate recognition component of the SCF (SKP1-CUL1-F-box protein)-type E3 ubiquitin ligase complex), was\",\n",
            "          \"protein)-type E3 ubiquitin ligase complex), was affected mostly by LOH events on chromosome 22q. Interestingly, in a recent human interaction proteome study SKP1 and FBXO7 were listed as interaction partners. Clustering and Network analysis\",\n",
            "          \"listed as interaction partners. Clustering and Network analysis Transcriptional profiling has converged on four major breast cancer subtypes: Luminal A and B, basal and HER2-enriched. To investigate the extent to which the PAM50 \\u201cintrinsic\\u201d breast cancer\",\n",
            "          \"the extent to which the PAM50 \\u201cintrinsic\\u201d breast cancer classification scheme is reflected or refined on the proteome level in the CPTAC samples, clustering analyses were first restricted to the reduced set of PAM50 genes. When RNA data for the 50 PAM50\",\n",
            "          \"the reduced set of PAM50 genes. When RNA data for the 50 PAM50 genes were clustered directly (without using a classifier), the clustering was similar to the TCGA PAM50 annotation (second annotation bar in Fig. 3a). Restricting both the RNA and proteome\",\n",
            "          \"bar in Fig. 3a). Restricting both the RNA and proteome data to the set of 35 PAM50 genes observed in the proteome produced a similar result (bottom two annotation bars in Fig. 3a), and all the major PAM50 groups were recapitulated in the proteome almost\",\n",
            "          \"major PAM50 groups were recapitulated in the proteome almost as well as in the RNA data. This indicates that although different tissue sections of the same tumors were used for RNA-seq and protein analysis, very similar subtype-defining features can be\",\n",
            "          \"protein analysis, very similar subtype-defining features can be observed in both data types. Global proteome and phosphoproteome data were then used to identify proteome subtypes in an unsupervised manner. Consensus clustering identified basal-enriched,\",\n",
            "          \"manner. Consensus clustering identified basal-enriched, luminal-enriched, and stromal-enriched clusters (Extended Data Figs. 6a\\u2013d, 7a). Unlike the clustering observed with PAM50 genes, mRNA-defined HER2-enriched tumors were distributed across these three\",\n",
            "          \"HER2-enriched tumors were distributed across these three proteomic subgroups. The basal-enriched and luminal-enriched groups showed a strong overlap with the mRNA-based PAM50 basal-like and luminal subgroups, whereas stromal-enriched proteome subtype\",\n",
            "          \"luminal subgroups, whereas stromal-enriched proteome subtype represented a mix of all PAM50 mRNA-based subtypes, and has a significantly enriched stromal signature (Extended Data Fig. 3e). Among the stromal-enriched tumors there was strong representation\",\n",
            "          \"the stromal-enriched tumors there was strong representation of reactive type I tumors as classified by RPPA (Supplementary Table 12), showing agreement between the RPPA and mass spectrometry-based protein analyses for the detection of a tumor subgroup\",\n",
            "          \"protein analyses for the detection of a tumor subgroup characterized by stromal gene expression. Since the basal- and luminal-enriched proteome subgroups are coherent, pathway analyses were conducted on these two subtypes, using the stromal-enriched\",\n",
            "          \"conducted on these two subtypes, using the stromal-enriched subgroup as a control to assess specificity. (Fig. 3c, Extended Data Fig. 7b, Supplementary Table 13). The luminal-enriched subgroup was exclusively enriched for estradiol and ESR1-driven gene\",\n",
            "          \"was exclusively enriched for estradiol and ESR1-driven gene sets. In contrast, multiple gene sets were enriched and upregulated specifically in the basal-like tumors. Particularly extensive basal-like enrichment was seen for MYC target genes; for cell\",\n",
            "          \"basal-like enrichment was seen for MYC target genes; for cell cycle, checkpoint, and DNA repair pathways including regulators AURKA/B, ATM, ATR, CHEK1/2, and BRCA1/2; and for immune response/inflammation, including T-cell, B-cell, and neutrophil\",\n",
            "          \"response/inflammation, including T-cell, B-cell, and neutrophil signatures. The complementarity of transcriptional, proteomic, and phosphoproteomic data was also highlighted in these analyses (Extended Data Fig. 7c, d). Using phosphorylation status as a\",\n",
            "          \"(Extended Data Fig. 7c, d). Using phosphorylation status as a proxy for activity, phosphoproteome profiling can theoretically be used to develop a signaling pathway-based cancer classification. K-means consensus clustering was therefore performed on\",\n",
            "          \"K-means consensus clustering was therefore performed on pathways derived from single sample GSEA analysis of phosphopeptide data (Methods, Supplementary Tables 14 and 15). Of four robustly segregated groups, subgroups 2 and 3 substantially recapitulated\",\n",
            "          \"groups, subgroups 2 and 3 substantially recapitulated the stromal- and luminal-enriched proteomic subgroups, respectively (Fig. 3d, Extended Data Fig. 8a). Subgroup 4 included a majority of tumors from the basal-enriched proteomic subgroup, but was\",\n",
            "          \"of tumors from the basal-enriched proteomic subgroup, but was admixed particularly with luminal-enriched samples. This subgroup was defined by high levels of cell cycle and checkpoint activity. All basal and a majority of non-basal samples in this\",\n",
            "          \"activity. All basal and a majority of non-basal samples in this subgroup had TP53 mutations. Consistent with high levels of cell cycle activity, a multivariate kinase-to-phosphosite abundance regression analysis highlighted CDK1 as one of the most highly\",\n",
            "          \"regression analysis highlighted CDK1 as one of the most highly connected kinases in this study (Extended Data Fig. 8b, Supplementary Table 16). Subgroup 1 was a novel subgroup defined exclusively in the phosphopeptide/pathway activity domain, with no\",\n",
            "          \"in the phosphopeptide/pathway activity domain, with no enrichment for either proteomic or PAM50 subtypes. It was defined by G-protein, G-protein coupled receptor, and inositol phosphate metabolism signatures, as well as ionotropic glutamate signaling\",\n",
            "          \"signatures, as well as ionotropic glutamate signaling (Fig. 3d). Co-expression patterns among genes/proteins across different subgroups were also analyzed using a Joint Random Forest (JRF) method that identified network modules, such as an MMP9 module,\",\n",
            "          \"method that identified network modules, such as an MMP9 module, with different interaction patterns between basal-enriched and luminal-enriched subgroups. These latter patterns appeared specific to the proteome-level data (Extended Data Fig. 8 c\\u2013f,\",\n",
            "          \"specific to the proteome-level data (Extended Data Fig. 8 c\\u2013f, Supplementary Table 17 and Supplementary Methods). Phosphosite markers in PIK3CA and TP53 mutant breast cancer TP53 and PIK3CA are the most recurrently mutated genes in breast cancer, with\",\n",
            "          \"are the most recurrently mutated genes in breast cancer, with frequencies for PIK3CA at 43% in luminal tumors and for TP53 at 84% in basal-like tumors. Most of the PIK3CA missense mutations were gain of function mutations and therefore expected to lead to\",\n",
            "          \"gain of function mutations and therefore expected to lead to an activation of the PI3K signaling cascade, but the extent to which this occurs has been controversial and there is uncertainty which pathway components are effectors. Marker selection analysis\",\n",
            "          \"pathway components are effectors. Marker selection analysis was therefore performed for upregulated phosphosites in PIK3CA-mutated tumors. In total, 62 phosphosites were identified that were positively associated with PIK3CA mutation (FDR<0.05), including\",\n",
            "          \"associated with PIK3CA mutation (FDR<0.05), including the kinases RPS6KA5 and EIF2AK4 (Extended Data Fig. 9a, Supplementary Table 18). Calculating the average phosphorylation signal of these marker phosphosites provided a read-out for PI3K pathway\",\n",
            "          \"these marker phosphosites provided a read-out for PI3K pathway activity in PIK3CA-mutated tumors, with 15 of the 26 mutated tumors (58%) exhibiting an activated PIK3CA mutation signature. Of note, the identified PIK3CA mutant phosphoproteome signature was\",\n",
            "          \"the identified PIK3CA mutant phosphoproteome signature was activated in all tumors harboring helical domain PIK3CA mutations but only 2 of 10 tumors harboring kinase domain mutations. To test if the identified differences in the phosphoproteome of PI3K\",\n",
            "          \"if the identified differences in the phosphoproteome of PI3K mutant versus wild-type tumors could be explained by mutation of PIK3CA, the tumor data were compared to phosphosite signatures derived from isogenic PIK3CA mutant cell lines (Extended Data Fig.\",\n",
            "          \"from isogenic PIK3CA mutant cell lines (Extended Data Fig. 9b, Supplementary Table 18). There was an enrichment of signatures derived from helical domain-mutated isogenic cell lines, but not from kinase domain-mutated cells, supporting the observations in\",\n",
            "          \"kinase domain-mutated cells, supporting the observations in primary tumors. The same strategy was used to identify phosphorylation signaling events connected to TP53 mutation. A total of 56 phosphosites upregulated in TP53 mutant tumors were identified\",\n",
            "          \"phosphosites upregulated in TP53 mutant tumors were identified that were independent of basal-like subtype association (Extended Data Fig. 9c, Supplementary Table 18). Using the average phosphorylation signal of these marker phosphosites as a proxy for\",\n",
            "          \"signal of these marker phosphosites as a proxy for TP53 mutation-driven cell cycle control, 22 of 41 mutated tumors (54%) showed upregulated signals. This TP53 mutant phosphosignature was somewhat enhanced in tumors in which mutations occurred almost\",\n",
            "          \"somewhat enhanced in tumors in which mutations occurred almost exclusively in the DNA binding region compared to those with non-sense/frameshift mutations. In addition to the well-described checkpoint kinase CHEK2, significantly upregulated phosphosites\",\n",
            "          \"checkpoint kinase CHEK2, significantly upregulated phosphosites were identified for the kinases MASTL and EEF2K in TP53-mutated tumors. Single sample GSEA analysis of isogenic p53-mutant phosphosignatures showed an enrichment of a phosphosignature derived\",\n",
            "          \"showed an enrichment of a phosphosignature derived from R273H mutated isogenic cells (Extended Data Fig. 9d), confirming the pronounced effect of missense mutations in the DNA-binding region on phosphorylation pathways. Identification of gene\",\n",
            "          \"region on phosphorylation pathways. Identification of gene amplification and breast cancer subtype-specific activated kinases in human breast cancer CNA spans many driver gene candidates and RNA expression has been frequently used to narrow candidate\",\n",
            "          \"and RNA expression has been frequently used to narrow candidate nominations. Proteogenomic analysis should further promote this nomination process. In this candidate refinement a focus on protein kinases is warranted, since many are drug targets. An\",\n",
            "          \"protein kinases is warranted, since many are drug targets. An in-depth proteogenomic pipeline was developed that flagged kinases, expression levels of which were at least 1.5 interquartile ranges higher than the median (Supplemental Table 19). A\",\n",
            "          \"ranges higher than the median (Supplemental Table 19). A proteogenomic circos-like plot (termed a \\u201cpircos\\u201d plot) was used to map these outlier kinase values onto the genome (Fig. 4a,b, Extended Data Fig. 10a). The ERBB2 locus showed the strongest effect\",\n",
            "          \"Data Fig. 10a). The ERBB2 locus showed the strongest effect of increased phosphoprotein levels associated with gene amplification-driven RNA and protein over expression (Fig. 4a). CDK12 is a positive transcriptional regulator of homologous recombination\",\n",
            "          \"positive transcriptional regulator of homologous recombination repair genes with its partner Cyclin K, and is often encompassed by the ERBB2 amplicon. This gene was also found to be upregulated at the RNA, protein and phosphosite level indicating that\",\n",
            "          \"at the RNA, protein and phosphosite level indicating that CDK12 is highly active in the majority of ERBB2 positive tumors (Fig. 4a). The analysis of the ERBB2 amplicon also uncovered co-outlier phosphorylation status for MED1, GRB7, MSL1, CASC3 and TOP2A,\",\n",
            "          \"phosphorylation status for MED1, GRB7, MSL1, CASC3 and TOP2A, all previously described in association with ERBB2 amplification. To better understand the downstream effects of ERBB2 amplification, additional phosphosite outliers were identified in 41 known\",\n",
            "          \"additional phosphosite outliers were identified in 41 known ERBB2 signaling genes for the 15 samples that had ERBB2 phosphosite outlier expression (Extended Data Fig. 10b). These canonical findings stimulated a proteogenomic analysis to identify\",\n",
            "          \"findings stimulated a proteogenomic analysis to identify additional outlier kinases in the breast cancer genome. A proteogenomic dissection of chromosome 11q based on PAK1 amplification (Fig. 4b,c), a breast cancer driver kinase, illustrated that PAK1 is\",\n",
            "          \"4b,c), a breast cancer driver kinase, illustrated that PAK1 is hyperphosphorylated in PAK1 amplified tumors, along with CLNS1A, RFS1 and GAB2. Additional examples of outlier kinases included PTK2 and RIPK2 in association with amplification of chromosome\",\n",
            "          \"PTK2 and RIPK2 in association with amplification of chromosome 8q (Fig. 4c; Extended Data Fig. 10a,c). PAK1 and TLK2 (17q23) appear to be luminal breast cancer specific events Fig. 4c; Extended Data Fig. 10c). To further examine whether outlier kinases\",\n",
            "          \"Data Fig. 10c). To further examine whether outlier kinases were breast cancer subtype-specific, independent of amplification status, the BH-corrected probability was calculated of finding that number of phosphosite outliers within a subtype, given the\",\n",
            "          \"that number of phosphosite outliers within a subtype, given the total number of outliers across all subtypes, the subtype sample size and the total sample size. (Fig. 4d). These analyses led to the expected identification of ERBB2 in the HER2-enriched\",\n",
            "          \"to the expected identification of ERBB2 in the HER2-enriched subtype at the 5% FDR level, as well as the new finding of CDC42BPG (MRGKG), an effector kinase for RHO-family GTPases. In basal-like breast cancer, two kinases, PRKDC and SPEG, were significant\",\n",
            "          \"breast cancer, two kinases, PRKDC and SPEG, were significant at the 5% FDR level. PRKDC is a non-homologous end-joining (NHEJ) factor that can be phosphorylated by ATM kinase, and is therefore a logical finding in this disease subset. However SPEG, a\",\n",
            "          \"a logical finding in this disease subset. However SPEG, a kinase associated with severe dilated cardiomyopathy when suppressed, has not been previously reported in association with breast cancer. A larger number of subtype-specific kinases were detected\",\n",
            "          \"A larger number of subtype-specific kinases were detected at the 10% FDR level, several of which have recently described relevance in breast cancer, including PRKD3 in basal-like breast cancer, the LKB-regulated SIK3 in luminal A breast cancer and CDK13\",\n",
            "          \"the LKB-regulated SIK3 in luminal A breast cancer and CDK13 in luminal B breast cancer, which, similar to CDK12, can interact with Cyclin K. Discussion The analytic breadth and depth of proteomic and phosphoproteomic analyses displayed in this study\",\n",
            "          \"proteomic and phosphoproteomic analyses displayed in this study demonstrates the strengths of mass spectrometry-based proteomics, but also some of the limitations inherent in proteolytic peptide sequencing (see Supplementary Discussion). An example of how\",\n",
            "          \"sequencing (see Supplementary Discussion). An example of how high-dimensional proteomic analysis provides insight into unresolved genomic issues concerns the study of loss of the long arm of chromosome 5 (5q). Analysis of RNA and protein correlations\",\n",
            "          \"of chromosome 5 (5q). Analysis of RNA and protein correlations narrowed the list of potential trans-deregulated proteins. Orthogonal candidate screening using functional genomics methods identified loss of CETN3 and SKP1 as potential transregulators, with\",\n",
            "          \"loss of CETN3 and SKP1 as potential transregulators, with upregulation of EGFR as a downstream consequence in basal-like breast cancers. While further experimental evidence must be sought for these proposed regulatory relationships, SKP1/Cullin complex\",\n",
            "          \"these proposed regulatory relationships, SKP1/Cullin complex has already been linked to EGFR activation in Glioma. Unfortunately EGFR targeting has not to date proven to be effective therapy in basal-like breast cancer. This might be due to the fact the\",\n",
            "          \"in basal-like breast cancer. This might be due to the fact the SKP1 loss deregulates multiple targets requiring a much broader inhibitory strategy. It is recognized that PIK3CA mutations do not strongly activate canonical downstream effectors. Mass\",\n",
            "          \"do not strongly activate canonical downstream effectors. Mass spectrometry-based phosphoproteomics provides an opportunity for unbiased examination of downstream signaling events consequent upon PIK3CA mutational activation. These studies revealed that\",\n",
            "          \"upon PIK3CA mutational activation. These studies revealed that common PIK3CA mutations affect a large number of targets with diverse functionalities including the kinases RPS6KA5 and EIF2AK4. Thus, the data and analyses reported here extend our knowledge\",\n",
            "          \"Thus, the data and analyses reported here extend our knowledge of the effectors that promote tumorigenesis in response to constitutive activation of PI3 kinase. Similarly, TP53 mutation-associated phosphopeptides point towards novel functionalities,\",\n",
            "          \"phosphopeptides point towards novel functionalities, including regulation of the kinases MASTL and EEF2K. A central goal in breast cancer research has been the identification of druggable kinases beyond HER2. Candidate genes that exhibited similar gene\",\n",
            "          \"beyond HER2. Candidate genes that exhibited similar gene amplification-driven proteogenomic patterns to HER2 included CDK12, TLK2, PAK1 and RIPK2. The proteogenomic link with gene amplification was particularly strong for CDK12, in keeping with its\",\n",
            "          \"was particularly strong for CDK12, in keeping with its location in the ERBB2 amplicon, while the strengths of correlation between DNA amplification, RNA, protein and phosphoprotein for the other examples were more variable. The presence of activated CDK12\",\n",
            "          \"examples were more variable. The presence of activated CDK12 in the ERBB2 amplicon might explain why tumors arising in BRCA1 carriers are usually ERBB2 negative. As a positive transcriptional regulator of BRCA1 and multiple FANC family members, CDK12\",\n",
            "          \"regulator of BRCA1 and multiple FANC family members, CDK12 promotes DNA repair by homologous recombination. CDK12 amplification would, therefore, oppose the functional effects of BRCA1 haploinsufficiency during tumor evolution. Overall, multiple outlier\",\n",
            "          \"during tumor evolution. Overall, multiple outlier kinases generate testable therapeutic hypotheses for which enabling inhibitors are in development. For example PAK1 has recently been confirmed to be a therapeutic target and poor prognosis factor in\",\n",
            "          \"to be a therapeutic target and poor prognosis factor in luminal breast cancer. Although incomplete outcome data and the remarkable heterogeneity of breast cancer are additional relevant constraints, the number of TCGA specimens analyzed here is\",\n",
            "          \"constraints, the number of TCGA specimens analyzed here is insufficient to support conclusive clinical correlations. Only 8 deaths occurred among the 77 patients, which are too few samples to provide sufficient statistical power for association analysis.\",\n",
            "          \"provide sufficient statistical power for association analysis. Adequately powered MS-based clinical investigation will require targeted approaches, especially given the highly limited amount of patient material available from clinical trials and the\",\n",
            "          \"of patient material available from clinical trials and the mostly formalin-fixed nature of the specimens. The current analysis is therefore centered on biological findings and correlations, with orthogonal validation and false discovery concerns addressed\",\n",
            "          \"orthogonal validation and false discovery concerns addressed through an examination of cell-line databases of the effects of individual gene perturbations. Typical of a multi-tiered analysis of this complexity, there are many hypotheses to test, and many\",\n",
            "          \"of this complexity, there are many hypotheses to test, and many findings that require further investigation. In conclusion, this study provides a high-quality proteomic resource for human breast cancer investigation and illustrates technologies and\",\n",
            "          \"breast cancer investigation and illustrates technologies and analytical approaches that provide an important new opportunity to connect the genome to the proteome. Larger-scale exploration of discovery proteomics in the clinical setting will require\",\n",
            "          \"of discovery proteomics in the clinical setting will require improvements in clinical investigation, including acquisition of adequate amounts of optimally collected tumor tissue both before and during therapy as well as advances in MS proteomics to\",\n",
            "          \"and during therapy as well as advances in MS proteomics to reduce sample input and increase sensitivity for low abundance proteins and modified peptides. Extended Data Experimental and data analysis workflows and longitudinal data generation quality\",\n",
            "          \"analysis workflows and longitudinal data generation quality control a, iTRAQ 4-plex global proteome and phosphoproteome analysis workflow. 105 TCGA breast tumors were analyzed in 35 iTRAQ 4-plex experiments (plus 1 replicate and 1 normal sample\",\n",
            "          \"iTRAQ 4-plex experiments (plus 1 replicate and 1 normal sample experiment), with three tumors of different subtypes compared to a fourth common internal reference sample in each experiment. The reference sample comprised 10 individual tumors of each of\",\n",
            "          \"The reference sample comprised 10 individual tumors of each of the 4 major breast cancer intrinsic subtypes and served as an internal standard for all proteins and phosphoproteins quantified in this study. Each iTRAQ MS/MS spectrum measures a peptide from\",\n",
            "          \"this study. Each iTRAQ MS/MS spectrum measures a peptide from 4 samples (3 individual patients and the reference sample mix of 40 patients). More than 400,000 distinct peptides were identified and quantified in ~14 million MS/MS spectra. Personalized\",\n",
            "          \"and quantified in ~14 million MS/MS spectra. Personalized tumor-specific protein databases were generated in the QUILTS software package using whole exome sequencing-derived variant calls and RNAseq-derived transcript information. All mass spectrometry\",\n",
            "          \"RNAseq-derived transcript information. All mass spectrometry data was analyzed using the Spectrum Mill software package. b, Overview of proteome and phosphoproteome datasets. The table provides a summary of the datasets used in specific analyses,\",\n",
            "          \"provides a summary of the datasets used in specific analyses, including the filters applied to derive the proteins and phosphosites/phosphoproteins that constitute each dataset; the protein, phosphosite or phosphoprotein count; and the methods that employ\",\n",
            "          \"or phosphoprotein count; and the methods that employ the respective datasets. c, Distribution of sequence coverage of the identified proteins with tryptic peptides detected by MS/MS, whiskers show the 5\\u201395 percentiles. d and e, Robust and accurate\",\n",
            "          \"show the 5\\u201395 percentiles. d and e, Robust and accurate proteome/phosphoproteome platform. Longitudinal performance was tested by repeated proteome and phosphoproteome analysis of patient-derived xenograft tumors. Scatterplots, histograms and Pearson\",\n",
            "          \"xenograft tumors. Scatterplots, histograms and Pearson correlations comparing individual replicate measurements are shown. Tumor sample quality control (I) a, Remark diagram showing sample processing and partitioning. Initial quality review encompassed\",\n",
            "          \"processing and partitioning. Initial quality review encompassed histopathological examination of H&E stained tissue slices. *For 3 samples no tumor cells were seen on histopathology (BH-A0E9, BH-A0C1, A2-A0SW). These samples were nevertheless included in\",\n",
            "          \"BH-A0C1, A2-A0SW). These samples were nevertheless included in the proteome analysis since other quality control standards were met (see below) and samples with 0% tumor cellularity on top or bottom sections were included in TCGA analyses. b, correlation\",\n",
            "          \"bottom sections were included in TCGA analyses. b, correlation of TCGA (top or bottom sections) and CPTAC histological assessment of neoplastic cellularity for samples (n = 105). The average and range of neoplastic cellularities were identical for CPTAC\",\n",
            "          \"and range of neoplastic cellularities were identical for CPTAC and TCGA histological assessments. Averages (standard deviations) for neoplastic cellularity were 76% (+/\\u2212 17) for CPTAC, 76% (+/\\u2212 15) for TCGA_Top, and 75% (+/\\u2212 18) for TCGA_Bottom\",\n",
            "          \"76% (+/\\u2212 15) for TCGA_Top, and 75% (+/\\u2212 18) for TCGA_Bottom histopathology slides (Supplementary Table 2). Note that in three CPTAC cases where no tumor cells were identified by histopathological assessment, numbers of protein-level somatic variants were\",\n",
            "          \"assessment, numbers of protein-level somatic variants were similar to all other tumors. The identified mutated proteins were TP53_R273C, NOP58_Q23E, TAGLN2_G154R, TUBA1B_D116H, and MRPL48_I173K (Supplementary Table 5), indicating presence of tumor cells\",\n",
            "          \"(Supplementary Table 5), indicating presence of tumor cells in these samples. c, Proteome iTRAQ tumor/internal reference ratio heatmap for all CPTAC samples (8,028 proteins without missing values) including passed and failed proteomic quality control (QC)\",\n",
            "          \"including passed and failed proteomic quality control (QC) samples. d, Global tumor/reference proteome ratio distributions for samples that passed and failed proteomic quality control analysis. e, Degradation-related gene sets were enriched in tumors that\",\n",
            "          \"e, Degradation-related gene sets were enriched in tumors that failed proteomic quality control analysis. f, Variant allele frequency (VAF) analysis of re-sequenced CPTAC tumors and comparison to original TCGA data. Overall VAFs for failed QC samples were\",\n",
            "          \"to original TCGA data. Overall VAFs for failed QC samples were lower compared to passed samples suggesting lower purity. Tumor sample quality control (II) a, There was high concordance (94.6%) between DNA variants reported by TCGA and CPTAC re-sequenced\",\n",
            "          \"between DNA variants reported by TCGA and CPTAC re-sequenced tumors. Most point mutations reported by TCGA could be identified across the 8 re-sequenced samples used in the study. b, A high overall correlation (mean=0.77) was observed for the CPTAC\",\n",
            "          \"high overall correlation (mean=0.77) was observed for the CPTAC Variant Allele Fraction (VAF) (X-axis) and TCGA VAF (Y-axis) across the 8 samples used in the study. c, Agglomerative hierarchical clustering (Supplementary Methods Section 3.8) used to\",\n",
            "          \"clustering (Supplementary Methods Section 3.8) used to co-cluster protein and RNA tumor expression data after filtering to retain 4,291 proteins and genes with moderate to high protein-RNA correlation (Pearson correlation > 0.4) with results displayed as\",\n",
            "          \"(Pearson correlation > 0.4) with results displayed as a circular dendrogram (fanplot). The proteome (.P) and RNA (.R) components of each sample are labeled using the same color. The outer ring shows proteome samples in light grey and RNA samples in dark\",\n",
            "          \"shows proteome samples in light grey and RNA samples in dark grey. High concordance between RNA and protein expression is evident from the color adjacency in the inner ring and alternating color in the outer ring showing that RNA and protein components\",\n",
            "          \"color in the outer ring showing that RNA and protein components co-cluster for a large proportion of samples (62/80). d, Co-clustering of MS and RPPA tumor data. 126 RPPA readouts were mapped to gene names. These genes were intersected with the genes\",\n",
            "          \"to gene names. These genes were intersected with the genes observed in the MS proteome, filtered to 48 proteins with moderate or higher RPPA-MS protein correlation, and analyzed for co-clustering as in c. 47 of 80 RPPA-MS protein pairs co-cluster. While\",\n",
            "          \"as in c. 47 of 80 RPPA-MS protein pairs co-cluster. While this is a smaller proportion than for RNA-protein analysis, the number of genes used in the clustering is significantly smaller for RPPA (48 vs. 4,291 for RNA). e, ESTIMATE tumor purity comparison\",\n",
            "          \"(48 vs. 4,291 for RNA). e, ESTIMATE tumor purity comparison between mRNA, RNAseq, and proteome data. ANOVA is used to assess the difference in distribution (\\u2212log10(p-value)) of ESTIMATE, stromal, immune, and tumor purity scores across mRNA (microarray),\",\n",
            "          \"immune, and tumor purity scores across mRNA (microarray), RNA-seq and proteome data. The only significant p-value (=0.02) is for the Cluster 3 stromal score, and higher stromal scores for the proteome drive that difference. f, Ischemia score analysis.\",\n",
            "          \"the proteome drive that difference. f, Ischemia score analysis. Comparison of ischemia scores of 77 CPTAC tumors, 3 normal samples, and patient-derived xenografts. CPTAC tumors had generally lower ischemia scores than PDX samples subjected to 30 minutes\",\n",
            "          \"lower ischemia scores than PDX samples subjected to 30 minutes of cold ischemia. Median ischemia scores are less than 30 minutes for each subtype and no significant differences were observed across subtypes. Effects due to cold ischemia therefore appear\",\n",
            "          \"across subtypes. Effects due to cold ischemia therefore appear to be negligible in this CPTAC sample collection. Protein-to-Protein, -CNA, and -mRNA correlation analyses a, Identification of UBE3A as an E3 ubiquitin ligase that negatively correlates to\",\n",
            "          \"UBE3A as an E3 ubiquitin ligase that negatively correlates to p53 on the protein level. Pearson correlation and Benjamini-Hochberg corrected p-value are shown. b, Analysis of counter-regulated genes with negative correlation of CNA-to-RNA as well as\",\n",
            "          \"genes with negative correlation of CNA-to-RNA as well as CNA-to-protein levels. Negative Pearson correlations are shown with Benjamini-Hochberg corrected p-values for CNA-to-protein correlations. Depicted genes have significant negative correlations at\",\n",
            "          \"Depicted genes have significant negative correlations at FDR<0.05 in the CNA-to-RNA and CNA-to-protein analyses. c, Global mRNA-to-protein correlation and gene set enrichment analysis. Global CNA effects and comparison of CNA TRANS effects to knockdown\",\n",
            "          \"CNA effects and comparison of CNA TRANS effects to knockdown signatures in the LINCS database a, CNA landscape in the CPTAC tumor collection. The segment-based CNAs of 77 samples were downloaded from TCGA Firehose, including 18 Basal, 12 Her2, 23 Luminal\",\n",
            "          \"from TCGA Firehose, including 18 Basal, 12 Her2, 23 Luminal A and 24 Luminal B subtypes. Copy number amplifications were marked in red and deletions in blue. The bottom color key represents the log2 transformed copy number value, with CNA=2 centered at 0.\",\n",
            "          \"log2 transformed copy number value, with CNA=2 centered at 0. Specific CNA events are seen for chromosome 5q and 10p regions in basal-like tumors. b, Correlations of copy number alterations (x-axis) to phosphoprotein levels (y-axis) highlight new CNA cis\",\n",
            "          \"to phosphoprotein levels (y-axis) highlight new CNA cis and trans effects. Significant (FDR<0.05) positive (red) and negative (green) correlations between CNA and phosphoproteins are indicated. Histograms show the fraction [%] of significant CNA trans\",\n",
            "          \"Histograms show the fraction [%] of significant CNA trans effects for each CNA gene. c, LINCS CMap analysis facilitates identification of novel functional candidates for CNA trans effects. Knockdown profiles were compared with CNA/protein trans effects\",\n",
            "          \"Knockdown profiles were compared with CNA/protein trans effects for 502 genes. Genes with a connectivity score >|90| were considered connected and significant cis effects were annotated at an FDR<0.05. d, Basal-like tumor-specific CNAs are candidate\",\n",
            "          \"at an FDR<0.05. d, Basal-like tumor-specific CNAs are candidate regulatory events for EGFR and SRC expression levels. Oncogenic kinases with significant CNA/protein trans effects (left panel), that were regulated in LINCS shRNA experiments (right panel; 4\",\n",
            "          \"that were regulated in LINCS shRNA experiments (right panel; 4 cell lines,) and directly measured as LINCS landmark genes, are shown alongside candidate regulatory genes CETN3 and SKP1. Clinical ER, PR, and HER2 annotation and PAM50 classification are\",\n",
            "          \"ER, PR, and HER2 annotation and PAM50 classification are shown in the header rows of each column. Proteome cluster heatmap and stability analysis a, K-means consensus clustering of proteome and phosphoproteome data identifies three subgroups:\",\n",
            "          \"proteome and phosphoproteome data identifies three subgroups: basal-enriched, luminal-enriched, and stromal-enriched. The heatmap represents all 1,521 proteins used for clustering (Dataset G8). b, Identification of optimal proteome clusters for QC-passed\",\n",
            "          \"b, Identification of optimal proteome clusters for QC-passed CPTAC breast cancer tumors. Proteome clusters were derived using consensus clustering based on 1000 resampled datasets, exploring the range of 2 to 6 k-means clusters. Visualization of consensus\",\n",
            "          \"range of 2 to 6 k-means clusters. Visualization of consensus matrices from k-means consensus clustering for k=3, 4, 5 and 6 target clusters. Consensus clustering was performed on 1,521 proteins with no missing values and SD>1.5. c, Silhouette plots were\",\n",
            "          \"with no missing values and SD>1.5. c, Silhouette plots were generated to evaluate the coherence of the clustering. Silhouette plots for k=3 and k=4 clusters showing a cleaner separation of clusters for k=3. d, Based on both visual inspection of the\",\n",
            "          \"of clusters for k=3. d, Based on both visual inspection of the consensus matrix and the delta plot assessing change in consensus cumulative distribution function (CDF) area, three robustly segregated groups were observed. Consensus cumulative distribution\",\n",
            "          \"groups were observed. Consensus cumulative distribution function (CDF) and delta area (change in CDF area) plots for 2\\u20136 clusters. Proteome cluster markers and enriched pathways a, Markers (based on SAM analysis; FDR<0.01) discriminate between proteome\",\n",
            "          \"(based on SAM analysis; FDR<0.01) discriminate between proteome clusters 1, 2 and 3 (compare to heatmap of proteins used to derive clusters depicted in Extended Data Fig. 6a). b, Applying a Fisher exact test-based enrichment analysis to the proteome,\",\n",
            "          \"a Fisher exact test-based enrichment analysis to the proteome, phosphoproteome and mRNA data, gene sets from MSigDB were identified that were unique for each proteome cluster. Heat map showing specific pathways comprising dominant biological themes that\",\n",
            "          \"specific pathways comprising dominant biological themes that are significantly differential by enrichment analysis between basal-enriched and luminal-enriched tumors (Fisher Exact Test Benjamini-Hochberg corrected p-values are shown; enrichment test\",\n",
            "          \"corrected p-values are shown; enrichment test performed on marker sets identified using SAM analysis; see Methods; compare to Figure 3c). c, Heatmap showing a selection of gene sets significant in basal-enriched or luminal-enriched tumors exclusively by\",\n",
            "          \"in basal-enriched or luminal-enriched tumors exclusively by mRNA, protein or phosphoprotein expression. Cytokine signatures, for example, were strongly captured at the mRNA level, but were seen to only a limited degree at the global protein level, likely\",\n",
            "          \"to only a limited degree at the global protein level, likely because of their typically low protein abundance. By contrast, the vast majority of significant gene sets annotated as \\\"signaling\\\" were enriched only at the phosphoprotein level. d, Global heat\",\n",
            "          \"were enriched only at the phosphoprotein level. d, Global heat map representing all gene sets significantly enriched in at least one of the proteomic breast cancer subtypes. The stromal-enriched group was characterized by breast cancer normal-like,\",\n",
            "          \"group was characterized by breast cancer normal-like, adipocyte differentiation, smooth muscle, toll-like receptor signaling and endothelin gene sets, supporting the clustering-based annotation of high stromal and/or adipose content in these tumors (see\",\n",
            "          \"of high stromal and/or adipose content in these tumors (see Supplemental Table 13). Phosphoproteome pathway clustering, kinase-phosphosite multivariate regression, and protein co-expression networks a, Phosphoproteome pathway clustering. Using\",\n",
            "          \"networks a, Phosphoproteome pathway clustering. Using phosphorylation state as a proxy for activity, deep phosphoproteome profiling allows development of a breast cancer molecular taxonomy based on signaling pathways. K-means consensus clustering was\",\n",
            "          \"based on signaling pathways. K-means consensus clustering was performed on pathways derived from single sample GSEA analysis of phosphopeptide data (908 pathways shown). Of four robustly segregated groups, subgroups 2 and 3 substantially recapitulated the\",\n",
            "          \"groups, subgroups 2 and 3 substantially recapitulated the stromal- and luminal-enriched proteomic subgroups, respectively. Subgroup 4 included a significant majority of tumors from the basal-enriched proteomic subgroup, but was admixed particularly with\",\n",
            "          \"proteomic subgroup, but was admixed particularly with luminal-enriched samples. This subgroup was defined by high levels of cell cycle and checkpoint activity. All basal and a majority of non-basal samples in this subgroup had TP53 mutations. Subgroup 1\",\n",
            "          \"samples in this subgroup had TP53 mutations. Subgroup 1 was a novel subgroup defined exclusively in the phosphopeptide / pathway activity domain, with no enrichment for either proteomic or PAM50 subtypes. It was defined by G-protein, G-protein coupled\",\n",
            "          \"PAM50 subtypes. It was defined by G-protein, G-protein coupled receptor, and inositol phosphate metabolism signatures, as well as ionotropic glutamate signaling. b, Analysis of the regulatory relationship between outlier kinases (see Supplementary Table\",\n",
            "          \"relationship between outlier kinases (see Supplementary Table 19) and phosphopeptides by regulatory multivariate regression analysis (see Methods) identified CDK1 as the most highly connected of the outlier Cyclin-Dependent Kinases, with highest\",\n",
            "          \"connected of the outlier Cyclin-Dependent Kinases, with highest centrality (based on node-degree; see Methods) among the outlier CDKs and seventh highest centrality among all the outlier kinases considered in the remMap analysis. Each line represents a\",\n",
            "          \"considered in the remMap analysis. Each line represents a phosphosite-kinase relationship. c\\u2013f, Analysis of differences in the co-expression patterns among genes/proteins across different subgroups. A Joint Random Forest (JRF) method was applied to\",\n",
            "          \"subgroups. A Joint Random Forest (JRF) method was applied to simultaneously build gene co-expression and protein co-expression networks (Supplementary Table 17, and Methods). Modules in these networks revealed different interaction patterns between\",\n",
            "          \"these networks revealed different interaction patterns between basal-enriched and luminal-enriched subgroups. c, Network module P1 of the protein co-expression network, defined chiefly in the proteome space. This module contained 12 genes connected by 39\",\n",
            "          \"proteome space. This module contained 12 genes connected by 39 edges, among which 34 were protein-specific and 5 were shared by both the protein and mRNA co-expression networks. Many edges were supported by published information and were contained in the\",\n",
            "          \"supported by published information and were contained in the STRING database. Edges in red are specific to the protein co-expression network; edges in green are shared by both protein and gene co-expression networks; edges indicated by double lines are\",\n",
            "          \"co-expression networks; edges indicated by double lines are contained in the STRING database with confidence score greater than 0.15. MMP9, one of the central proteins in this module, contributes to metastatic progression and is a potential target for\",\n",
            "          \"to metastatic progression and is a potential target for anti-metastatic therapies for basal-like / triple negative breast cancer. d, Heatmaps of the absolute correlation across each pair of genes in module P1 (shown in Panel c), based on either protein or\",\n",
            "          \"in module P1 (shown in Panel c), based on either protein or gene expression data for samples in the basal-enriched and luminal-enriched subgroups, respectively. The MMP9 protein was strongly co-expressed with the other members of the module only in the\",\n",
            "          \"co-expressed with the other members of the module only in the basal-enriched subgroup. Notably, this observation is dependent on protein data; the correlation at the mRNA level for this module was consistently low in both the basal-enriched and luminal\",\n",
            "          \"was consistently low in both the basal-enriched and luminal enriched subgroups indicating that these events coherently occur at the proteomic level. e, Co-expression network based on proteomics data. The network contains 693 proteomic network-specific\",\n",
            "          \"data. The network contains 693 proteomic network-specific edges (grey) and 792 edges shared with the RNAseq network (green). For each module, the most enriched category and corresponding Benjamini-Hochberg adjusted p-value is reported. Pie charts adjacent\",\n",
            "          \"adjusted p-value is reported. Pie charts adjacent to each module show the proportion of proteomics-specific edges (grey area) and edges shared between proteomics and RNAseq data (green area). f, RNAseq network. Phosphoproteome signatures of PIK3CA (a,b)\",\n",
            "          \"f, RNAseq network. Phosphoproteome signatures of PIK3CA (a,b) and TP53 (c,d) mutated tumors highlight activated key regulators and indicate frequency of activation a and c, Phosphosites upregulated in mutated tumors (SAM FDR<0.05 across all tumors and\",\n",
            "          \"in mutated tumors (SAM FDR<0.05 across all tumors and independently also across luminal tumors; average phosphosite signal for all markers shown as bar graph). To avoid confounding by intrinsic subtype-specific distinctions, only markers that were\",\n",
            "          \"intrinsic subtype-specific distinctions, only markers that were significantly identified both in analyses covering all tumors and analyses restricted to luminal tumors were selected (FDR <0.05). Color bars in the margins indicate FDRs for grouped analysis\",\n",
            "          \"Color bars in the margins indicate FDRs for grouped analysis of different mutation classes and indicate kinase substrates of known kinases in the respective pathways. Significantly regulated kinase phosphosites are annotated. The average phosphorylation\",\n",
            "          \"kinase phosphosites are annotated. The average phosphorylation signal of the marker phosphosites provides a read-out for PI3K and TP53 pathway activity in mutated tumors (histogram below heatmap). A 95% prediction confidence interval (indicated by dashed\",\n",
            "          \"A 95% prediction confidence interval (indicated by dashed lines) across the average signal in non-mutated tumors was chosen in order to discriminate active from non-active tumors. The most strongly activated PIK3CA kinase domain mutant tumor differed from\",\n",
            "          \"activated PIK3CA kinase domain mutant tumor differed from the other 9 kinase domain mutant tumors, as it contained an amino acid side chain charge neutral H1047L instead of the more common positively charged H1047R mutation. Among the 62 phosphosites\",\n",
            "          \"positively charged H1047R mutation. Among the 62 phosphosites identified that were significantly upregulated in PIK3CA mutated tumors, 13 phosphosites were found on phosphoproteins that are known substrates of well-annotated kinases in the PIK3CA pathway\",\n",
            "          \"substrates of well-annotated kinases in the PIK3CA pathway (panel a, right column). In the mutant TP53 analysis a total 20 phosphosites were found on phosphoproteins that are known substrates of well annotated kinases in the p53 pathway (panel c, right\",\n",
            "          \"of well annotated kinases in the p53 pathway (panel c, right column). b and d, Upregulated phosphosite sets were derived from isogenic PIK3CA and TP53 mutant versus wild-type cell line pairs and tested for enrichment within mutant versus wild-type CPTAC\",\n",
            "          \"and tested for enrichment within mutant versus wild-type CPTAC tumors using single sample GSEA. Significantly enriched phosphosite sets are shown (p<0.05). PIRCOS plots, kinase outliers and outliers in the ERBB2 pathway a, Pircos (Proteogenomics CIRCOS)\",\n",
            "          \"outliers in the ERBB2 pathway a, Pircos (Proteogenomics CIRCOS) plots for 8q and 17q showing median CNA, RNA, protein, and phosphosite expression for 20 tumors with amplification in 8q based on RIPK2 CNA>1; 23 tumors with amplification in 8q based on PTK2\",\n",
            "          \"RIPK2 CNA>1; 23 tumors with amplification in 8q based on PTK2 CNA>1; 15 tumors with amplification in 17q based on CDK12 CNA >1; and 10 tumors with amplification in 17q based on TLK2 CNA>1. Red indicates expression >1, blue < \\u22121, and grey between \\u22121 and 1.\",\n",
            "          \"indicates expression >1, blue < \\u22121, and grey between \\u22121 and 1. Genes with both copy number amplification (CNA>1) and increased phosphosite expression (p-site>1) are labeled. b, Phosphosite outliers in known ERBB2 signaling genes. To better understand the\",\n",
            "          \"in known ERBB2 signaling genes. To better understand the downstream effects of ERBB2 amplification, phosphosite outliers in known ERBB2 signaling genes (MSigDB pathway set, KEGG_ERBB_SIGNALING PATHWAY) were identified for the 15 samples that had ERBB2\",\n",
            "          \"PATHWAY) were identified for the 15 samples that had ERBB2 phosphosite outlier status. Forty-one genes were identified as having a phosphosite outlier in at least one of the ERBB2 amplified samples. PAK4 and ARAF phosphosite outlier status were found in\",\n",
            "          \"samples. PAK4 and ARAF phosphosite outlier status were found in seven of the 15 ERBB2 kinase outlier samples; GSK3B outliers were found in 6 samples; and EIF4EBP1, MAP2K2, ABL1 and AKT1 outlier status was found in 5 of the 15 samples. c, Proteogenomic\",\n",
            "          \"status was found in 5 of the 15 samples. c, Proteogenomic outlier expression analysis for TLK2 and RIPK2. Samples with outlier phosphosite (red), protein (yellow), RNA (green) and copy number (purple) expression are shown. Phosphosite squares indicate\",\n",
            "          \"(purple) expression are shown. Phosphosite squares indicate per-sample outlier phosphosites. Supplementary Material Supplementary Information is available in the online version of the paper. Author Contributions P.M., D.R.M., M.A.G., K.R.C., and S.A.C.\",\n",
            "          \"Author Contributions P.M., D.R.M., M.A.G., K.R.C., and S.A.C. designed the proteomic analysis experiments, data analysis workflow, and proteomic-genomic data comparisons. P.M., M.A.G., J.W.Q., and S.A.C. directed and performed proteomic analysis of breast\",\n",
            "          \"and S.A.C. directed and performed proteomic analysis of breast tumor and quality control samples. P.M., D.R.M., K.V.R., K.R.C., P.W., X.W., S.C., E.K., F.P., Z.T., J.L., M.L.G., M.W., V.Y., K.H., C.L., M.D.M., P.Y., J.W., B.Z., and D.F. performed\",\n",
            "          \"V.Y., K.H., C.L., M.D.M., P.Y., J.W., B.Z., and D.F. performed proteomic-genomic data analyses. D.R.M., P.W., and S.J.S. provided statistical support. D.R.M., K.V.R., K.R.C., K.K. and D.F. performed analyses of mass spectrometry data and adapted\",\n",
            "          \"D.F. performed analyses of mass spectrometry data and adapted algorithms and software for data analysis. S.R.D., R.R.T and M.J.E. developed and prepared breast xenografts used as quality control samples. P.M. and F.M prepared and analyzed cell lines for\",\n",
            "          \"samples. P.M. and F.M prepared and analyzed cell lines for correlative functional annotation of frequently mutated genes. P.M., D.R.M., M.A.G., and S.A.C designed strategy for quality control analyses. M.A.G., S.R.D., C.R.K., M.M., and H.R. coordinated\",\n",
            "          \"analyses. M.A.G., S.R.D., C.R.K., M.M., and H.R. coordinated acquisition, distribution and quality control evaluation of TCGA tumor samples. P.M., M.A.G., C.P., L.D., A.G.P., and M.J.E. interpreted data in the context of breast cancer biology. P.M.,\",\n",
            "          \"interpreted data in the context of breast cancer biology. P.M., D.R.M, M.A.G., K.R.C., P.W., A.G.P, M.J.E. and S.A.C. wrote the manuscript. All primary mass spectrometry data are deposited at the CPTAC Data Portal as raw and mzML files and complete\",\n",
            "          \"at the CPTAC Data Portal as raw and mzML files and complete protein assembly data sets for public access (https://cptac-data-portal.georgetown.edu/cptac/s/S029). In addition, a set of ancillary files such as dataset G1/P1, G3/P3, G4/P4, G5/P5, G7/P7, CNA\",\n",
            "          \"files such as dataset G1/P1, G3/P3, G4/P4, G5/P5, G7/P7, CNA correlation tables for CNA-mRNA, CNA-proteome and CNA-phosphoproteome; CNA data, RNA-seq expression data have also been deposited at the DCC. Two browsers have been created to assist the\",\n",
            "          \"at the DCC. Two browsers have been created to assist the interested reader in exploring the results. One provides track hubs for viewing the identified peptides in the UCSD genome browser (http://fenyolab.org/cptac_breast_ucsc). The second is an on-line\",\n",
            "          \"The second is an on-line tool for proteogenomic data exploration and can be accessed at http://prot-shiny-vm.broadinstitute.org:3838/BC2016/. See Supplemental Methods for descriptions. The authors declare no competing financial interests. Proteogenomic\",\n",
            "          \"authors declare no competing financial interests. Proteogenomic analysis of human breast cancer. Direct effects of genomic alterations on protein level Overlap of a, protein coding single amino acid variants (SAAVs) and b, RNA splice junctions not present\",\n",
            "          \"acid variants (SAAVs) and b, RNA splice junctions not present in RefSeq v60 detected by DNA exome sequencing, RNA-seq, and LC-MS/MS. Proportions of novel variants are noted. c, Heatmap of mutations/CNA and their effects on RNA and protein expression of\",\n",
            "          \"and their effects on RNA and protein expression of breast cancer-relevant genes across tumor and normal samples. ER, PR, HER2 and PAM50 status are annotated. Median iTRAQ protein abundance ratio and the most frequently detected and differential\",\n",
            "          \"ratio and the most frequently detected and differential phosphosite ratio are shown for each gene. Pearson correlations between MS protein vs RNA-seq and MS protein vs RPPA are indicated. Effects of copy number alterations (CNA) on mRNA, protein, and\",\n",
            "          \"Effects of copy number alterations (CNA) on mRNA, protein, and phosphoprotein abundance a, Correlations of CNA (x-axes) to RNA and protein expression levels (y-axes) highlight new CNA cis and trans effects. Significant (FDR<0.05) positive (red) and\",\n",
            "          \"and trans effects. Significant (FDR<0.05) positive (red) and negative (green) correlations between CNA and mRNAs or proteins are indicated. CNA cis effects appear as a red diagonal line, CNA trans effects as vertical stripes. Histograms show the fraction\",\n",
            "          \"trans effects as vertical stripes. Histograms show the fraction [%] of significant CNA trans effects for each CNA gene. b, Overlap of cis effects observed at RNA, protein, and phosphoprotein levels (FDR<0.05). c, Trans-effect regulatory candidates\",\n",
            "          \"levels (FDR<0.05). c, Trans-effect regulatory candidates identified among those with significant protein cis-effects using LINCS CMap. Bars indicate total numbers of significant CNA/protein trans effects (gray; FDR<0.05) and overlap with regulated genes\",\n",
            "          \"trans effects (gray; FDR<0.05) and overlap with regulated genes in LINCS knock-down profiles (red; 4 cell lines; moderated T-test FDR<0.1). Proteomic and phosphoproteomic subtypes of breast cancer and subtype-specific pathway enrichment a, Unsupervised\",\n",
            "          \"cancer and subtype-specific pathway enrichment a, Unsupervised clustering of RNA-seq and proteomics data restricted to PAM50 genes and subset of 35 detected proteins reveal high similarity to PAM50 (TCGA) sample annotation. b, K-means consensus clustering\",\n",
            "          \"PAM50 (TCGA) sample annotation. b, K-means consensus clustering of proteome and phosphoproteome data identifies basal-enriched, luminal-enriched, and stromal-enriched subgroups. c, Gene set enrichment analysis highlights sets of pathways significantly\",\n",
            "          \"enrichment analysis highlights sets of pathways significantly differential between basal-enriched and luminal-enriched tumors (detailed in Extended Data Fig. 7b). d, K-means consensus clustering performed on pathways derived from single sample GSEA\",\n",
            "          \"performed on pathways derived from single sample GSEA analysis of phosphopeptide data identifies four distinct clusters. Example analyses of aberrantly regulated kinases in human breast cancer a and b, PIRCOS (Proteogenomics CIRCOS) plots showing CNA,\",\n",
            "          \"a and b, PIRCOS (Proteogenomics CIRCOS) plots showing CNA, RNA, protein and phosphosite expression for 17 tumors with amplification in 17q (ERBB2 CNA>1) and 8 tumors with amplification in 11q (PAK1 CNA>1). Labeled genes have CNA>1 and phosphosite>1. c,\",\n",
            "          \"(PAK1 CNA>1). Labeled genes have CNA>1 and phosphosite>1. c, Proteogenomic outlier expression analysis for ERBB2, CDK12, and PAK1. Samples with outlier phosphosite (red), protein (yellow), RNA (green) and copy number (purple) expression are shown.\",\n",
            "          \"RNA (green) and copy number (purple) expression are shown. Phosphosite squares indicate per-sample outlier phosphosites. d, Outlier kinase events by PAM50 subtype (>35% of subtype samples contain a phosphosite outlier; <10% FDR using Benjamini-Hochberg\",\n",
            "          \"a phosphosite outlier; <10% FDR using Benjamini-Hochberg adjusted p-values).\"\n",
            "        ]\n",
            "      }\n",
            "    ]\n",
            "  },\n",
            "  \"PDC000111\": {\n",
            "    \"study_id\": \"b998098f-57b8-11e8-b07a-00a098d917f8\",\n",
            "    \"pdc_study_id\": \"PDC000111\",\n",
            "    \"study_submitter_id\": \"TCGA COAD Proteome S016-1\",\n",
            "    \"program_id\": \"10251935-5540-11e8-b664-00a098d917f8\",\n",
            "    \"project_id\": \"48af5040-5546-11e8-b664-00a098d917f8\",\n",
            "    \"study_name\": \"TCGA Colon Cancer Proteome\",\n",
            "    \"program_name\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"project_name\": \"CPTAC2 Retrospective\",\n",
            "    \"disease_type\": \"Colon Adenocarcinoma;Rectum Adenocarcinoma\",\n",
            "    \"primary_site\": \"Colon;Rectum\",\n",
            "    \"analytical_fraction\": \"Proteome\",\n",
            "    \"experiment_type\": \"Label Free\",\n",
            "    \"embargo_date\": null,\n",
            "    \"cases_count\": 90,\n",
            "    \"aliquots_count\": 95,\n",
            "    \"protocol_id\": \"1c06b336-3b26-11e9-9a07-0a80fada099c\",\n",
            "    \"protocol_submitter_id\": \"P-S016-1\",\n",
            "    \"program_submitter_id\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"protocol_name\": \"TCGA_Colon_Cancer_Proteome\",\n",
            "    \"protocol_date\": \"2012-08-28\",\n",
            "    \"publications\": [\n",
            "      {\n",
            "        \"publication_id\": \"44e2e50c-89c7-11e8-bcf1-0a2705229b82\",\n",
            "        \"pubmed_id\": \"26110064\",\n",
            "        \"doi\": \"10.1038/sdata.2015.22\",\n",
            "        \"title\": \"Proteomic analysis of colon and rectal carcinoma using standard and customized databases\",\n",
            "        \"journal\": \"Sci Data\",\n",
            "        \"journal_url\": \"https://www.nature.com/articles/sdata201522\",\n",
            "        \"year\": \"2015\",\n",
            "        \"abstract\": \"Understanding proteomic differences underlying the different phenotypic classes of colon and rectal carcinoma is important and may eventually lead to a better assessment of clinical behavior of these cancers. We here present a comprehensive description of the proteomic data obtained from 90 colon and rectal carcinomas previously subjected to genomic analysis by The Cancer Genome Atlas (TCGA). Here, the primary instrument files and derived secondary data files are compiled and presented in forms that will allow further analyses of the biology of colon and rectal carcinoma. We also discuss new challenges in processing these large proteomic datasets for relevant proteins and protein variants.\",\n",
            "        \"full_text\": [\n",
            "          \"Proteomic analysis of colon and rectal carcinoma using standard and customized databases Understanding proteomic differences underlying the different phenotypic classes of colon and rectal carcinoma is important and may eventually lead to a better\",\n",
            "          \"carcinoma is important and may eventually lead to a better assessment of clinical behavior of these cancers. We here present a comprehensive description of the proteomic data obtained from 90 colon and rectal carcinomas previously subjected to genomic\",\n",
            "          \"90 colon and rectal carcinomas previously subjected to genomic analysis by The Cancer Genome Atlas (TCGA). Here, the primary instrument files and derived secondary data files are compiled and presented in forms that will allow further analyses of the\",\n",
            "          \"and presented in forms that will allow further analyses of the biology of colon and rectal carcinoma. We also discuss new challenges in processing these large proteomic datasets for relevant proteins and protein variants. Background & Summary Emerging\",\n",
            "          \"proteins and protein variants. Background & Summary Emerging collections of genomics data for human cancers present the formidable challenge of understanding how genomic abnormalities drive the biological and clinical characteristics of cancer. This task\",\n",
            "          \"biological and clinical characteristics of cancer. This task will be facilitated by proteomic analyses, which provides an intermediate layer of biological information that is more directly connected to phenotype. The National Cancer Institute (NCI)\",\n",
            "          \"connected to phenotype. The National Cancer Institute (NCI) Clinical Proteomic Tumor Assessment Consortium (CPTAC) network employs proteomic technology platforms to analyze colorectal, breast and ovarian tumors previously analyzed by the NCI The Cancer\",\n",
            "          \"and ovarian tumors previously analyzed by the NCI The Cancer Genome Atlas (TCGA) network, with the objective of integrating proteomic and genomic data for the same tissue specimens. Here we describe datasets and data analysis methods for the proteomic\",\n",
            "          \"describe datasets and data analysis methods for the proteomic analysis of colon and rectal tumors from the TCGA, which were combined with TCGA genomics data to produce the first integrated proteogenomic analysis of a human cancer. We analyzed 95 samples\",\n",
            "          \"analysis of a human cancer. We analyzed 95 samples representing 90 TCGA colon and rectal (CRC) tumors, which comprise a subset of the 224 sample collection subjected to multiplatform genomic analyses by the TCGA. Proteomic analyses by multidimensional\",\n",
            "          \"analyses by the TCGA. Proteomic analyses by multidimensional liquid chromatography-tandem mass spectrometry (LC-MS/MS) provide peptide and protein identifications from MS/MS data and estimates of relative abundance, from which expression differences can\",\n",
            "          \"of relative abundance, from which expression differences can be inferred. Our data analysis employed a workflow that is well-established for global proteomics, but is typically used for much smaller datasets. The large size of our proteomic dataset\",\n",
            "          \"much smaller datasets. The large size of our proteomic dataset created unanticipated challenges and required additional evaluation of alternative analyses strategies. The final protein assembly summarizes a total of more than 13 million successfully\",\n",
            "          \"summarizes a total of more than 13 million successfully assigned MS/MS spectra representing more than 7,000 non-redundant human protein identifications. We used this dataset to create a proteomics-based classification of colon and rectal carcinoma and\",\n",
            "          \"classification of colon and rectal carcinoma and demonstrate its use in relation to genomics-based classifications. Chromosomal mapping of protein expression profiles revealed both cis- and trans-effects of chromosomal loci commonly involved in\",\n",
            "          \"cis- and trans-effects of chromosomal loci commonly involved in amplification and deletion events. Protein expression changes may be used to create a better understanding of the driving forces behind such chromosomal changes. In addition, we employed a\",\n",
            "          \"behind such chromosomal changes. In addition, we employed a novel strategy that integrates individual sequence variation as determined by RNA-seq with proteomic identification to improve protein identification and discover single amino acid variation\",\n",
            "          \"protein identification and discover single amino acid variation (SAAVs) in the proteome. The current paper describes this dataset in detail and makes available additional files that will facilitate further analyses based on our proteomics data. These\",\n",
            "          \"facilitate further analyses based on our proteomics data. These additional files include search results from the three different protein-search engines and customized databases containing all sequence variations for each of the 90 TCGA tumors. We also\",\n",
            "          \"all sequence variations for each of the 90 TCGA tumors. We also discuss the different options at each of the data processing steps and the consequences for the eventual protein inventory. Methods Summary Description of data acquisition and processing were\",\n",
            "          \"Summary Description of data acquisition and processing were expanded from our previous work. After receipt of the tumor blocks, protein was extracted from the samples, digested with trypsin, and the resulting peptides were fractionated by off-line\",\n",
            "          \"and the resulting peptides were fractionated by off-line basic-reverse phase high-performance liquid chromatography. Each tumor sample yielded 15 concatenated peptide fractions, which then were analyzed by reverse phase liquid chromatography-tandem mass\",\n",
            "          \"analyzed by reverse phase liquid chromatography-tandem mass spectrometry. Analysis of the 95 carcinoma and 60 normal epithelial samples took 9 months on a single MS instrument. To monitor system stability and performance, we also analyzed benchmark\",\n",
            "          \"system stability and performance, we also analyzed benchmark quality control (QC) samples from one basal and one luminal human breast tumor xenograft. These were analyzed in alternating order after every 5 sets of colorectal carcinoma and normal colon\",\n",
            "          \"after every 5 sets of colorectal carcinoma and normal colon epithelium samples. This produced proteomics QC datasets for both the 16 basal and luminal xenografts (Data Citation 1). The QC dataset not only enabled assessment of platform performance and\",\n",
            "          \"dataset not only enabled assessment of platform performance and consistency throughout the project, but also provided a representative large proteomics dataset with which to evaluate data normalization. Raw MS/MS data were processed for peptide\",\n",
            "          \"data normalization. Raw MS/MS data were processed for peptide identification by database and spectral library searching using three different search engines (Myrimatch, Pepitome and MS-GF+). Identified peptides were assembled as proteins and mapped to\",\n",
            "          \"Identified peptides were assembled as proteins and mapped to gene identifiers for proteogenomic comparisons using IDPicker 3.0. Quantitative comparisons were based on spectral count data. Details for each of the preparation steps are given below. TCGA\",\n",
            "          \"Details for each of the preparation steps are given below. TCGA tumor samples All samples for the current study were obtained through the TCGA Biospecimen Core Resource (BCR) as described previously. Of the 276 samples described in the TCGA study, 95\",\n",
            "          \"previously. Of the 276 samples described in the TCGA study, 95 specimens from 90 patients were available for the current study (5 patients\\u2019s tumors were provided as duplicate pieces). No other selection criteria other than availability were applied for\",\n",
            "          \"selection criteria other than availability were applied for this study. Specimens for proteomic study were sectioned sequentially from the tumor blocks used for genomic studies, hence the \\u2018bottom\\u2019 slides from the genomic studies are representative of the\",\n",
            "          \"slides from the genomic studies are representative of the material used for proteomics. The slides are available from the TCGA data portal (http://tcga-data.nci.nih.gov/tcga/tcgaHome2.jsp). All samples were frozen within 1\\u2009h of collection and\",\n",
            "          \"All samples were frozen within 1\\u2009h of collection and histopathological examination confirmed that they contained at least 60% tumor nuclei as described earlier. Prior CPTAC studies have shown that global protein levels, but not phosphorylation states, are\",\n",
            "          \"that global protein levels, but not phosphorylation states, are stable within this 1\\u2009h time period. Samples were shipped from the TCGA BCR on dry ice and kept frozen at \\u221280\\u2009\\u00b0C until processing. All TCGA colorectal tissue samples were washed prior to\",\n",
            "          \"All TCGA colorectal tissue samples were washed prior to digestion to eliminate any residual optimal cutting temperature compound (OCT). The tissue was placed in a 1.5\\u2009ml micro-tube was washed with 1\\u2009ml 70% EtOH/30% H2O for 30\\u2009s with vortexing. The\",\n",
            "          \"washed with 1\\u2009ml 70% EtOH/30% H2O for 30\\u2009s with vortexing. The supernatant was then discarded and the tissue was washed with 1\\u2009ml of 100% H2O 30\\u2009s with vortexing and again the supernatant was discarded. One milliliter of 70% EtOH/30% H2O was added to the\",\n",
            "          \"discarded. One milliliter of 70% EtOH/30% H2O was added to the tissue sample and incubated for 5\\u2009min at room temperature following by centrifugation at 20,000\\u00d7g for 2\\u2009min at 20\\u2009\\u00b0C. The supernatant was removed, and this wash step was repeated. Next, 1\\u2009ml\",\n",
            "          \"was removed, and this wash step was repeated. Next, 1\\u2009ml of 85% EtOH/15% H2O was added to the tissue and incubated for 5\\u2009min at room temperature following by centrifugation at 20,000\\u00d7g for 2\\u2009min at 20\\u2009\\u00b0C. The supernatant was removed, and the wash was\",\n",
            "          \"2\\u2009min at 20\\u2009\\u00b0C. The supernatant was removed, and the wash was repeated. For the final wash, the tissue was washed in 1\\u2009ml 100% EtOH, incubated 5\\u2009min at room temperature, and centrifuged at 20,000\\u00d7g for 2\\u2009min at 20\\u2009\\u00b0C. The supernatant was removed, and the\",\n",
            "          \"for 2\\u2009min at 20\\u2009\\u00b0C. The supernatant was removed, and the final wash was repeated. Normal colon epithelium Normal colon epithelium biopsies were obtained through the Cooperative Human Tissue Network (CHTN) Western Division at Vanderbilt University. Samples\",\n",
            "          \"(CHTN) Western Division at Vanderbilt University. Samples were collected from screening colonoscopies performed between July 2006 and October 2010 under Vanderbilt University IRB approval #061096. During colonoscopy, multiple pinch biopsies were obtained\",\n",
            "          \"During colonoscopy, multiple pinch biopsies were obtained from both ascending and descending colon and immediately frozen in liquid nitrogen. Biopsies obtained from 30 subjects with completely normal findings during colonoscopy were included in the study.\",\n",
            "          \"normal findings during colonoscopy were included in the study. A total of 60 specimens, one from ascending and one from descending colon, were used for proteomic analysis. Protein extraction and digestion of tissue specimens Following OCT removal, tissue\",\n",
            "          \"and digestion of tissue specimens Following OCT removal, tissue specimens were placed in 1.5\\u2009ml micro-centrifuge tubes and re-suspended 100\\u2009\\u03bcl of trifluoroethanol (TFE) and 100\\u2009\\u03bcl of 100\\u2009mM ammonium bicarbonate, pH 8. If additional buffer was required,\",\n",
            "          \"ammonium bicarbonate, pH 8. If additional buffer was required, equal volumes of TFE and 100\\u2009mM ammonium bicarbonate pH 8.0 were added accordingly. In addition, powderized xenograft tumor tissue representing the Comparison/Reference (CompRef) samples for\",\n",
            "          \"representing the Comparison/Reference (CompRef) samples for Luminal (WHIM16) and Basal (WHIM2) breast cancer subtypes were analyzed in alternating order after every 5 TCGA colorectal tissue samples. Samples were sonicated using a Fisher Scientific Sonic\",\n",
            "          \"samples. Samples were sonicated using a Fisher Scientific Sonic Dismembrator Model 100 at a setting of 20\\u2009watts for 20\\u2009s followed by 30\\u2009s incubation on ice. This sonication step was repeated twice, and samples were placed on ice between sonications. The\",\n",
            "          \"twice, and samples were placed on ice between sonications. The resulting homogenate was heated with shaking at 1,000\\u2009rpm for 1\\u2009h at 60\\u2009\\u00b0C followed by a second series of sonication steps, as described above. A protein measurement then was obtained for each\",\n",
            "          \"above. A protein measurement then was obtained for each sample using the BCA Protein Assay (ThermoFisher Pierce, Rockford, IL) using the manufacturer\\u2019s protocol. An aliquot equivalent to 200\\u2009\\u03bcg was removed and reduced with tris(2-carboxyethyl)phosphine\",\n",
            "          \"was removed and reduced with tris(2-carboxyethyl)phosphine (TCEP, 20\\u2009mM) and dithiothreitol (DTT, 50\\u2009mM) at 60\\u2009\\u00b0C for 30\\u2009min, followed by alkylation with iodoacetamide (IAM, 100\\u2009mM) in the dark at room temperature for 20\\u2009min. The lysate was diluted with\",\n",
            "          \"at room temperature for 20\\u2009min. The lysate was diluted with the appropriate volume of 50\\u2009mM ammonium bicarbonate, pH 8.0, to reduce the TFE concentration to 10%. Trypsin was added at a ratio of 1:50 (w:w) and digested overnight at 37\\u2009\\u00b0C. The digested\",\n",
            "          \"of 1:50 (w:w) and digested overnight at 37\\u2009\\u00b0C. The digested mixture was frozen at \\u221280\\u2009\\u00b0C and lyophilized to dryness. The lyophilized samples were re-suspended in 350\\u2009\\u03bcl of HPLC-grade water and vortexed vigorously for one minute and desalted using an Oasis\",\n",
            "          \"vortexed vigorously for one minute and desalted using an Oasis HLB 96-well \\u03bcElution plate (30\\u2009\\u03bcm, 5\\u2009mg, Waters Corp., Milford, MA), which was pre-washed with 500\\u2009\\u03bcl of acetonitrile and equilibrated with 750\\u2009\\u03bcl of HPLC-grade water. The flow-through was\",\n",
            "          \"with 750\\u2009\\u03bcl of HPLC-grade water. The flow-through was discarded, and the plates were washed with 500\\u2009\\u03bcl of HPLC-grade water. The peptides were eluted with 80% acetonitrile, and the eluates were evaporated to dryness in vacuo. Samples were stored in the\",\n",
            "          \"were evaporated to dryness in vacuo. Samples were stored in the freezer until further analysis. Peptide fractionation by basic reverse phase liquid chromatography (bRPLC) Samples were reconstituted in 300\\u2009\\u03bcl of Solvent A (1.0\\u2009M triethylamine bicarbonate\",\n",
            "          \"in 300\\u2009\\u03bcl of Solvent A (1.0\\u2009M triethylamine bicarbonate (TEAB), pH 7.5). The reconstituted sample was then diluted with an additional 100\\u2009\\u03bcl of Solvent A and the entire 400\\u2009\\u03bcl of solution was injected into the bRPLC column. Tryptic peptides were\",\n",
            "          \"was injected into the bRPLC column. Tryptic peptides were fractionated using bRPLC separation with an XBridge BEH C18, 250\\u2009mm\\u00d74.6\\u2009mm analytical column (130A, 5\\u2009\\u03bcm particle size) equipped with a XBridge BEH C18 Sentry guard cartridge at a flow rate of\",\n",
            "          \"with a XBridge BEH C18 Sentry guard cartridge at a flow rate of 0.5\\u2009ml/min. The solvents were 10\\u2009mM TEAB pH 7.5 in water as mobile phase A and 100% acetonitrile as mobile phase B. Sample fractionation was accomplished using the following multistep linear\",\n",
            "          \"was accomplished using the following multistep linear gradient: from 0 to 5% B in 10\\u2009min, from 5 to 35% B in 60\\u2009min, from 35\\u201370% in 15\\u2009min and held at 70% B for an additional 10\\u2009min before returning to initial conditions. A total of 60 fractions were\",\n",
            "          \"returning to initial conditions. A total of 60 fractions were collected over the 105\\u2009min gradient and concatenated into 15 fractions by combining fractions 1, 16, 31, 46; 2, 17, 32, 47; and so on up to fractions 15, 30, 45, and 60. The samples were\",\n",
            "          \"and so on up to fractions 15, 30, 45, and 60. The samples were evaporated to dryness in a Speed-Vac sample concentrator and stored at \\u221280C until LC-MS/MS analysis. LC-MS/MS analysis Resulting peptide fractions were resuspended in 50\\u2009\\u03bcl of 2%\",\n",
            "          \"Resulting peptide fractions were resuspended in 50\\u2009\\u03bcl of 2% acetonitrile/0.1% formic acid and analyzed using a Thermo LTQ Orbitrap Velos ion trap mass spectrometer equipped with an Eksigent NanoLC 2D pump and AS-1 autosampler. A 2\\u2009\\u03bcl injection volume of\",\n",
            "          \"NanoLC 2D pump and AS-1 autosampler. A 2\\u2009\\u03bcl injection volume of the peptide solution was separated on a packed capillary tip (Polymicro Technologies, 100\\u2009\\u03bcm\\u00d711\\u2009cm) containing Jupiter C18 resin (5\\u2009\\u03bcm, 300\\u2009\\u00c5, Phenomenex) using an in-line solid-phase\",\n",
            "          \"resin (5\\u2009\\u03bcm, 300\\u2009\\u00c5, Phenomenex) using an in-line solid-phase extraction column (100\\u2009\\u03bcm\\u00d76\\u2009cm) packed with the same C18 resin using a frit generated with liquid silicate Kasil. Mobile phase A consisted of 0.1% formic acid and Mobile phase B consisted of\",\n",
            "          \"A consisted of 0.1% formic acid and Mobile phase B consisted of 0.1% formic acid in acetonitrile. A 95\\u2009min gradient was preceded by a 15\\u2009min washing period (100% A) at a flow-rate of 1.5\\u2009\\u03bcl\\u2009min\\u22121\\u2009to remove residual salt. Following the wash, the mobile\",\n",
            "          \"remove residual salt. Following the wash, the mobile phase was programmed to 25% B by 50\\u2009min, followed by an increase to 90% B by 65\\u2009min and held for 9\\u2009min before returning to the initial conditions. A full MS scan was collected for peptides from\",\n",
            "          \"conditions. A full MS scan was collected for peptides from 400\\u20132000\\u2009m/z on the Orbitrap at a resolution of 60,000 followed by eight data-dependent MS/MS scans from lowest to highest signal intensity on the LTQ. Centroided MS/MS scans were acquired using\",\n",
            "          \"on the LTQ. Centroided MS/MS scans were acquired using an isolation width of 3\\u2009m/z, an activation time of 30\\u2009ms, an activation q of 0.250 and 35% normalized collision energy. One microscan with a max ion time of 100 and 1000\\u2009ms was used for each MS/MS and\",\n",
            "          \"a max ion time of 100 and 1000\\u2009ms was used for each MS/MS and full MS scan, respectively. MS/MS spectra were collected using a dynamic exclusion of 60\\u2009s with a repeat of 1 and repeat duration of 1. All TCGA and normal colon epithelial samples were\",\n",
            "          \"of 1. All TCGA and normal colon epithelial samples were analyzed on the same Thermo LTQ Orbitrap Velos instrument, with sample analysis beginning on July 26, 2012 and concluding on June 27, 2013. Each sample comprised the 15 bRPLC fractions prepared as\",\n",
            "          \"2013. Each sample comprised the 15 bRPLC fractions prepared as described above. Benchmark quality control (QC) samples from one basal and one luminal human breast tumor xenograft (Comparison/Reference, \\u2018CompRef\\u2019 samples) were analyzed in alternating order\",\n",
            "          \"\\u2018CompRef\\u2019 samples) were analyzed in alternating order after every five CRC samples. Specifically, five TCGA samples were run on the instrument, followed by a luminal CompRef sample, and then another five TCGA samples followed by a basal CompRef sample for\",\n",
            "          \"five TCGA samples followed by a basal CompRef sample for a total of 32 CompRef samples. Normal epithelial samples were analyzed using the same schema whereby each 5 normal epithelial samples were followed by a CompRef sample. Bovine serum albumin (BSA)\",\n",
            "          \"were followed by a CompRef sample. Bovine serum albumin (BSA) tryptic digest standards were analyzed before and after every ten TCGA samples and were used to monitor instrument sensitivity, BSA standard sequence coverage and chromatographic performance to\",\n",
            "          \"standard sequence coverage and chromatographic performance to determine acceptance or rejection of the acquired data. LC-MS/MS peptide identification Peptide identification employed the RefSeq Human protein sequence database, release version 54, and both\",\n",
            "          \"Human protein sequence database, release version 54, and both database and peptide library search strategies. Two bovine trypsin sequences and one porcine trypsin sequence were appended to these 34,586 sequences. The June 14, 2011 NIST human spectral\",\n",
            "          \"these 34,586 sequences. The June 14, 2011 NIST human spectral library for ion traps (617,000 spectra, counting paired decoys) was indexed against this sequence database. Thermo RAW files were converted to mzML peaklists by the ScanSifter algorithm\\u2009or by\",\n",
            "          \"converted to mzML peaklists by the ScanSifter algorithm\\u2009or by ProteoWizard msConvert. The ScanSifter files were employed by Pepitome 1.0.42 for spectral library search and MyriMatch 2.1.87 for database search, while the msConvert files were used by MS-GF+\",\n",
            "          \"database search, while the msConvert files were used by MS-GF+ v9176. Pepitome and MyriMatch employed precursor tolerances of 10\\u2009ppm, while MS-GF+ used a 20\\u2009ppm window; all three algorithms allowed fragments to vary by up to 0.5\\u2009m/z, and both database\",\n",
            "          \"allowed fragments to vary by up to 0.5\\u2009m/z, and both database search engines considered semi-tryptic peptides equally with fully-tryptic peptides, allowed for isotopic error in precursor ion selection, conducted on-the-fly peptide sequence reversal, and\",\n",
            "          \"selection, conducted on-the-fly peptide sequence reversal, and applied static +57 modifications to cysteines and dynamic +16 oxidations to methionines. MS-GF+ considered acetylation for protein N-termini, while MyriMatch added pyroglutamine modifications\",\n",
            "          \"N-termini, while MyriMatch added pyroglutamine modifications to the N-termini of peptides starting with Gln residues. Pepitome considered any modification variants and trypsin specificities that were included in the spectral library. Protein assembly\",\n",
            "          \"that were included in the spectral library. Protein assembly Spectral identification files from each of the search engines (pepXML for Myrimatch and Pepitome, mzIdent for MS-GF+) were converted to IDPicker 3 SQLite databases (idpDB) using IDP3. The\",\n",
            "          \"to IDPicker 3 SQLite databases (idpDB) using IDP3. The resulting 4,275 idpDB files (95 samples\\u00d715 fractions\\u00d73 search engines) were used for a final protein assembly using IDP3. For initial protein assembly, peptide identification stringency was set at a\",\n",
            "          \"assembly, peptide identification stringency was set at a maximum of 1% reversed peptide matches, i.e., 2% peptide-to-spectrum matches (PSM) FDR and a minimum of 2 distinct peptides to identify a given protein within the full data set. To optimize the\",\n",
            "          \"a given protein within the full data set. To optimize the number of proteins identified, we applied a very stringent filter at 0.1% PSM FDR and a minimum of 2 distinct peptides identified for each protein. This filter resulted in the identification of a\",\n",
            "          \"each protein. This filter resulted in the identification of a total of 94,442 distinct peptides among the 95 samples representing 7,526 protein groups with a protein-level FDR of 2.64%. To rescue high quality PSMs that were excluded by the stringent PSM\",\n",
            "          \"high quality PSMs that were excluded by the stringent PSM FDR threshold, we relaxed the PSM FDR threshold to 1% for the confidently identified proteins. This step increased the number of distinct peptides identified to 124,823, corresponding to a total\",\n",
            "          \"peptides identified to 124,823, corresponding to a total number of 6,299,756 spectra; the rescued PSMs were of high quality, with a median PSM FDR of less than 0.2%, confirming that the integrity of the data set was maintained. To facilitate the\",\n",
            "          \"the integrity of the data set was maintained. To facilitate the integration between genomic and proteomic data, we further assembled the peptides at the gene level. This assembly resulted in 7,211 gene groups with a gene-level FDR of 2.7%. Genes\",\n",
            "          \"in 7,211 gene groups with a gene-level FDR of 2.7%. Genes identified from each sample ranged from 3,372 to 5,456, with a median gene count of 4,656 for the 95 samples. 1,530 genes (21%) were found in all 95 samples, 4,628 genes (64%) in more than half of\",\n",
            "          \"found in all 95 samples, 4,628 genes (64%) in more than half of the samples, and only 10 genes (0.1%) in just a single sample. In addition, we created a combined protein assembly using all TCGA colorectal carcinoma and the 60 normal colon epithelium\",\n",
            "          \"TCGA colorectal carcinoma and the 60 normal colon epithelium samples. This assembly identified 7,548 protein groups with a protein FDR of 2.7%. All IDP3 protein assembly files are available through ProteomeXchange (Data Citation 13) or from the CPTAC data\",\n",
            "          \"ProteomeXchange (Data Citation 13) or from the CPTAC data portal (cptac-data-portal.georgetown.edu/cptacPublic). Customized database construction We developed an R package customProDB \\u2009to annotate variations predicted from RNA-Seq, including mapping to\",\n",
            "          \"variations predicted from RNA-Seq, including mapping to dbSNP135 and COSMIC64 databases. For each sample, customProDB generates a protein FASTA database by appending proteins with nonsynonymous protein coding single nucleotide variants (SNVs) and aberrant\",\n",
            "          \"protein coding single nucleotide variants (SNVs) and aberrant proteins to the end of the standard RefSeq Human protein sequence database. Due to the low coverage of this RNA-Seq data set, we did not remove the low abundance transcripts from the standard\",\n",
            "          \"did not remove the low abundance transcripts from the standard RefSeq database. Peptide identification was performed for each sample separately using the corresponding customized FASTA database with MyriMatch 2.1.87. IDPicker 3 was produced a conjoint\",\n",
            "          \"with MyriMatch 2.1.87. IDPicker 3 was produced a conjoint protein assembly across all the customized database searches, with filtering set at 1% PSM FDR and a minimum of 5 spectra identified per protein. The full data set consisted of 8,352 protein groups\",\n",
            "          \"protein. The full data set consisted of 8,352 protein groups with 1.8% protein FDR. Identified single amino acid variations (SAAVs) were further annotated from the previous report of the TCGA study (i.e., TCGA-somatic variants), existence in the COSMIC64\",\n",
            "          \"study (i.e., TCGA-somatic variants), existence in the COSMIC64 database (i.e., COSMIC-supported variants), and existence in the dbSNP135 database (i.e., dbSNP-supported variants). To identify TCGA-somatic variants, we downloaded the MAF (Mutation\",\n",
            "          \"identify TCGA-somatic variants, we downloaded the MAF (Mutation Annotation Format) files from the FIREHOSE website (Data Citation 2 and Data Citation 3). Since the coordinates in MAF files were based on hg18, liftOver\",\n",
            "          \"Since the coordinates in MAF files were based on hg18, liftOver (http://hgdownload.cse.ucsc.edu/admin/exe/) from UCSC was used to convert genome coordinates to hg19. All identified variant peptides as well as SAAVs and their annotations can be found in\",\n",
            "          \"peptides as well as SAAVs and their annotations can be found in the original publication. Variant peptide identification To identify variant peptides, we performed database searches using the above-constructed sample-specific customized databases as\",\n",
            "          \"the above-constructed sample-specific customized databases as previously described. Identifying SNVs and INDELs from RNA-Seq data BAM files for 86 of the 90 patients were downloaded from CGHub in February 2013. Although 87 samples were analyzed by\",\n",
            "          \"CGHub in February 2013. Although 87 samples were analyzed by RNA-Seq, we were unable to obtain the BAM file for one of the 87 samples. Tophat (v2.0.7) was used to re-align reads to human reference genome (hg19) in a spliced mode using default parameters,\",\n",
            "          \"genome (hg19) in a spliced mode using default parameters, allowing a maximum of 10 hits per read. The resulting BAM files were indexed using samtools (0.1.18, http://samtools.sourceforge.net/). We used an in-house script to summarize the reads mapping\",\n",
            "          \"We used an in-house script to summarize the reads mapping information and calculate the exon coverage. Seventy-six percent of exons were covered by RNA-Seq reads, and 64% had an average coverage greater than 1. Putative single nucleotide variants (SNVs)\",\n",
            "          \"greater than 1. Putative single nucleotide variants (SNVs) and short INDELs were called one library at a time using mpileup and varFilter from samtools. Putative SNVs were further filtered based on the following criteria: (i) SNP quality\\u226520; (ii) mapping\",\n",
            "          \"on the following criteria: (i) SNP quality\\u226520; (ii) mapping quality\\u226520; and (iii) read depth\\u22653 and then put into a VCF file. For short INDEL candidates, gapped reads\\u22653 were required. Protein quantification We used spectral count, or the total number of\",\n",
            "          \"quantification We used spectral count, or the total number of MS/MS spectra acquired for peptides from a given protein in a given LC/LC\\u2212MS/MS analysis, as the basis for protein quantification. Spectral count is linearly correlated with the protein\",\n",
            "          \"Spectral count is linearly correlated with the protein abundance over a large dynamic range. This simple but practical quantification method has found broad application in detecting differential or correlated protein expression, and multiple groups have\",\n",
            "          \"or correlated protein expression, and multiple groups have concluded that spectral counting achieves similar accuracy to more complex methods such as the intensity-based techniques. Previously, we have confirmed proteomic changes detected from spectral\",\n",
            "          \"we have confirmed proteomic changes detected from spectral count data by targeted proteomics with multiple reaction monitoring (MRM) in different data sets. For the CompRef QC sample data set, spectral count data were summarized at the protein group\",\n",
            "          \"set, spectral count data were summarized at the protein group level, where a protein group is defined as the set of proteins that are indiscernible on the basis of the observed peptides. For each group, a random protein was selected to represent the\",\n",
            "          \"For each group, a random protein was selected to represent the group. The spectral count table has data for 8,029 mouse/human protein groups from 32 samples, with 16 basal samples and 16 luminal samples. For the TCGA tumor data set, to facilitate the\",\n",
            "          \"luminal samples. For the TCGA tumor data set, to facilitate the integration between genomic and proteomic data, spectral count data were summarized at the gene group level, where a gene group is defined as the set of genes that are indiscernible on the\",\n",
            "          \"is defined as the set of genes that are indiscernible on the basis of the observed peptides. To ensure reproducibility, the longest protein was selected for each gene in the calculation of protein length. For each gene group, the gene with the shortest\",\n",
            "          \"protein length. For each gene group, the gene with the shortest protein length was selected to represent the group following the Occam\\u2019s razor principle. The final spectral count Table has data for 7,211 genes and 95 samples (5 tumors have duplicated\",\n",
            "          \"data for 7,211 genes and 95 samples (5 tumors have duplicated samples, Supplementary Table S1). For analyses that require only one sample from the duplicates, the sample with a larger total spectral count was selected. Quality control using QuaMeter The\",\n",
            "          \"spectral count was selected. Quality control using QuaMeter The raw data for each of the 95 bRPLC experiments were subjected to quality control assessment in the QuaMeter \\u2018IDFree\\u2019 mode, producing forty-four metrics for each file along with its timestamp.\",\n",
            "          \"forty-four metrics for each file along with its timestamp. Metrics included peak height and width distributions, MS scan rates, precursor charge states, and total ion current distributions, and identifications were not a factor in any of the metrics. The\",\n",
            "          \"identifications were not a factor in any of the metrics. The table of metrics included 1,425 rows, one for each RAW file (Supplementary Table S2). Scripts were created in the R statistical environment for multivariate analysis of these metrics. The\",\n",
            "          \"environment for multivariate analysis of these metrics. The software performed a robust Principal Components Analysis (PCA) on the values, excluding the following fields due to insufficient variation: XIC-FWHM.Q1, XIC.FWHM.Q3, RT.Duration, and all\",\n",
            "          \"variation: XIC-FWHM.Q1, XIC.FWHM.Q3, RT.Duration, and all precursor charge state fields. In PC space, the normalized Euclidean distance between pairs of RAW files was computed for the fifteen files produced for each bRPLC. When a LC-MS/MS experiment\",\n",
            "          \"files produced for each bRPLC. When a LC-MS/MS experiment yields a large median standard distance from the other files, that distance suggests the experiment is an outlier in QC. When all the files of a bRPLC experiment have high median distances, they\",\n",
            "          \"files of a bRPLC experiment have high median distances, they are indicative of high variability in instrument performance for that sample. Code availability Exact copies of all open source software, customized scripts and data processing procedures are\",\n",
            "          \"software, customized scripts and data processing procedures are provided through the ProteomeXchange Consortium via the PRIDE partner repository as 'other' files (Data Citation 13). Data Records Primary MS instrument and database search files Primary\",\n",
            "          \"Records Primary MS instrument and database search files Primary files for this study are in the Thermo \\u2018RAW\\u2019 file format accessible through the ProteomeXchange repository (Data Citation 1 and Data Citation 4, Data Citation 5, Data Citation 6, Data\",\n",
            "          \"1 and Data Citation 4, Data Citation 5, Data Citation 6, Data Citation 7, Data Citation 8, Data Citation 9, Data Citation 10, Data Citation 11, Data Citation 12, Data Citation 13 and Data Citation 14) or from the CPTAC data portal\",\n",
            "          \"Citation 13 and Data Citation 14) or from the CPTAC data portal (cptac-data-portal.georgetown.edu/cptacPublic). Included are data files from the 95 TCGA carcinoma samples, the 60 normal epithelial biopsy samples and the CompRef control samples. A complete\",\n",
            "          \"biopsy samples and the CompRef control samples. A complete list of files is provided in Supplementary Table S3. Protein identification from these files was accomplished through database searching using Myrimatch and MS-GF+, and spectral library searching\",\n",
            "          \"using Myrimatch and MS-GF+, and spectral library searching using Pepitome (fully human samples only). Resulting search files are in the pepXML file format for Myrimatch and Pepitome searches and in the mzID file format for MS-GF+ searches. Output files\",\n",
            "          \"and in the mzID file format for MS-GF+ searches. Output files from all searches using the standard human RefSeq database are provided in a single compressed file. Likewise, all search results from normal epithelium and search results using each of the\",\n",
            "          \"from normal epithelium and search results using each of the customized databases are summarized in single compressed files. These databases are available from the CPTAC data portal. In addition to the IDPicker 3 (IDP3) protein assembly file combining\",\n",
            "          \"to the IDPicker 3 (IDP3) protein assembly file combining search results from the original publication, we here present IDP3 files from each of the individual search engines and the IDP3 file containing only the normal epithelium control samples. IDP3 is\",\n",
            "          \"containing only the normal epithelium control samples. IDP3 is an open-source protein assembly and filtering tool that queries search results summarized in a SQLite database format. The protein databases against which all searches were performed are\",\n",
            "          \"protein databases against which all searches were performed are provided separately: 1) the standard human database file (RefSeq version 54), 2) the accompanying standard spectral library file, 3) a single compressed file containing each of the 86\",\n",
            "          \"file, 3) a single compressed file containing each of the 86 customized databases that include the standard search information plus all variant sequences identified in TCGA RNA-seq data from the CPTAC data portal, and 4) the combined human/mouse RefSeq\",\n",
            "          \"the CPTAC data portal, and 4) the combined human/mouse RefSeq database. The processing and result files have been deposited to the ProteomeXchange Consortium via the PRIDE partner repository (Data Citation 1, Data Citation 13 and Data Citation 14). Data\",\n",
            "          \"(Data Citation 1, Data Citation 13 and Data Citation 14). Data processing strategies One of the goals of bioinformatics analysis of shotgun proteomic dataset is to maximize the number of successfully identified MS/MS spectra without compromising the\",\n",
            "          \"successfully identified MS/MS spectra without compromising the quality of the data. Almost all spectrum identification strategies are based on comparing obtained spectra with predicted spectra from known amino acid peptide sequences (\\u2018database searching\\u2019)\",\n",
            "          \"from known amino acid peptide sequences (\\u2018database searching\\u2019) or with previously observed and identified spectra (\\u2018spectral library searching\\u2019) (for review see). Over the years, several algorithms have been developed that accomplish this goal with\",\n",
            "          \"algorithms have been developed that accomplish this goal with varying levels of sophistication, and these have been incorporated into commercially available or open-source software tools for peptide identification. For instance, most algorithms calculate\",\n",
            "          \"peptide identification. For instance, most algorithms calculate from known amino acid masses where one would expect peptide fragments to appear on a MS/MS spectrum, but not all would take signal intensity into account when matching to the observed\",\n",
            "          \"signal intensity into account when matching to the observed spectrum. Newer algorithms also take complementarity of the fragment ions into account (e.g., MS-GF+), looking for the remaining part of the precursor mass when a given peptide fragment is\",\n",
            "          \"part of the precursor mass when a given peptide fragment is observed. A commonly accepted strategy to avoid false-positive matches is to include distractor (decoy) sequences in the database and to set limits on the proportion of such decoy matches in the\",\n",
            "          \"to set limits on the proportion of such decoy matches in the final list of identified peptides. These distractor sequences can be generated by adding reversed amino acid sequences in the database (reverse-peptides), an approach that has the advantage of\",\n",
            "          \"(reverse-peptides), an approach that has the advantage of preserving the distribution of precursor masses in the (reversed) search space. The various strategies in search algorithms cause differences in their identification profiles and it is helpful to\",\n",
            "          \"in their identification profiles and it is helpful to combine search results from multiple search engines to come to an optimal annotation of a given set of MS/MS spectra. For this study, we chose three open-source search tools (Myrimatch, MS-GF+ and\",\n",
            "          \"we chose three open-source search tools (Myrimatch, MS-GF+ and Pepitome) and combined their results into a single protein assembly using IDP3. Figure 1 shows the distribution of spectra and protein group identifications between the three different search\",\n",
            "          \"group identifications between the three different search engines. Note that even though the number of identified proteins is not greatly increased by combining the data, each of the search engines contributes a significant number of engine-specific\",\n",
            "          \"engines contributes a significant number of engine-specific spectral matches. The spectral library search engine Pepitome matches spectra to previously identified spectra from the library and is better at finding such known spectra than the other two\",\n",
            "          \"and is better at finding such known spectra than the other two search engines. This leads to higher numbers of spectral counts, but not necessarily of distinct peptides or protein groups. On the other hand, MS-GF+ identifies a distinct group of additional\",\n",
            "          \"other hand, MS-GF+ identifies a distinct group of additional peptide-spectrum matches, which leads to a large number of added counts from distinct peptides uniquely discovered by this search engine. These results indicate that it is important to combine\",\n",
            "          \"engine. These results indicate that it is important to combine results from different approaches to increase the variety and number of identifications and thus the universe of proteins that are quantifiable by spectral counting. IDP3 uses an inclusive\",\n",
            "          \"are quantifiable by spectral counting. IDP3 uses an inclusive combination algorithm whereby all PSMs that pass filtering criteria for an individual search engine are included in the assembly, using the highest scoring PSM from the search engines. IDP3\",\n",
            "          \"using the highest scoring PSM from the search engines. IDP3 uses protein parsimony filtering to assemble identified peptide lists into a list of non-redundant proteins (\\u2018protein groups\\u2019) as described previously. To allow easy comparison to genomic data,\",\n",
            "          \"described previously. To allow easy comparison to genomic data, IDP3 layers genome data on top of the RefSeq protein identifications, so that the main output becomes a non-redundant list of RefSeq gene IDs (\\u2018gene groups\\u2019) for which peptide evidence was\",\n",
            "          \"RefSeq gene IDs (\\u2018gene groups\\u2019) for which peptide evidence was obtained. The 95 TCGA colon cancer specimens yielded a dataset of 7,526 protein groups representing 7,211 gene groups. Cut off points for filtering, stringencies PSM/peptide/protein FDR The\",\n",
            "          \"points for filtering, stringencies PSM/peptide/protein FDR The goal of our bioinformatics analysis of the TCGA colorectal cancer dataset was to maximize the number and diversity of identified peptides without sacrificing confidence in the obtained\",\n",
            "          \"peptides without sacrificing confidence in the obtained spectral matches. A common strategy to determine false-positive rates in shotgun proteomics datasets is to include decoy sequences in the database. Decoy sequence identifications enable modeling of\",\n",
            "          \"the database. Decoy sequence identifications enable modeling of the score distribution for false matches in a given dataset. The maximum allowable fraction of decoy peptide-spectrum matches in shotgun proteomic datasets is commonly set at 1 or 2%,\",\n",
            "          \"in shotgun proteomic datasets is commonly set at 1 or 2%, although these are arbitrarily chosen cut-off values. In addition, the requirement of having at least 2 distinct peptide sequences for the identification of a single protein can also limit the\",\n",
            "          \"for the identification of a single protein can also limit the number of falsely identified proteins. However, applying these two criteria may not be sufficient to limit the number of falsely identified proteins in a dataset, as many studies do not report\",\n",
            "          \"identified proteins in a dataset, as many studies do not report protein FDR values in separate calculations. This is especially true with larger protein assemblies where the vast majority of unique peptide sequences have been identified already and any\",\n",
            "          \"unique peptide sequences have been identified already and any additional peptide identifications are therefore more likely to be false. One strategy we employed in moderately large datasets is to increase the minimum number of matched spectra required for\",\n",
            "          \"to increase the minimum number of matched spectra required for protein acceptance into the assembly. This strategy works well and protects against inclusion of proteins that have little spectral evidence, as well as against false-positive peptide\",\n",
            "          \"spectral evidence, as well as against false-positive peptide identifications. The drawbacks of using a minimum number of spectral counts per protein are that low-abundance proteins may be missed and that the number of identified proteins is greatly\",\n",
            "          \"be missed and that the number of identified proteins is greatly reduced. In very large datasets, such as the one reported here, proteins with high expression in just a few samples also may be missed. With the ever increasing size of shotgun experiments\",\n",
            "          \"be missed. With the ever increasing size of shotgun experiments (our TCGA colorectal carcinoma assembly was based on 4,275 search result files), limiting PSM FDR and spectral counts is not sufficient to limit false-discovery rates to acceptable levels. To\",\n",
            "          \"to limit false-discovery rates to acceptable levels. To deal with the false discovery challenge in our large dataset, we developed a new strategy based on the counter-intuitive finding that increasing the stringency of PSMs actually increased the number\",\n",
            "          \"increasing the stringency of PSMs actually increased the number of proteins identified. This is illustrated in Fig. 2, which presents the number of protein groups identified at different PSM and protein false-discovery rates at different minimum numbers\",\n",
            "          \"and protein false-discovery rates at different minimum numbers of spectral counts. The lower curve shows the values obtained at the commonly accepted PSM value of 2%, where a minimum number of spectral counts per protein needs to be set at 175 to achieve\",\n",
            "          \"spectral counts per protein needs to be set at 175 to achieve a protein FDR of less than 5%. Because of the loss of a large number of proteins with lower numbers of spectral counts, this setting also yields the lowest number of protein groups identified\",\n",
            "          \"also yields the lowest number of protein groups identified (~4,000). When PSM FDR rate is set more stringently, lower-cut off values for the minimum number of spectral counts per protein can be selected without sacrificing protein FDR. This leads to\",\n",
            "          \"can be selected without sacrificing protein FDR. This leads to higher numbers of identified protein groups at an acceptable protein FDR, albeit with lower spectral counts per protein than using the more lenient PSM FDR settings. Based on these\",\n",
            "          \"than using the more lenient PSM FDR settings. Based on these considerations, we adopted a 2-step procedure, whereby we initially set a very stringent PSM FDR of 0.1%, which maximized the number of identified protein groups at an acceptable protein FDR.\",\n",
            "          \"of identified protein groups at an acceptable protein FDR. This defined the inventory of proteins that could be confidently identified in this dataset. The second step was then to increase the PSM FDR 10-fold to 1% for only the protein groups already\",\n",
            "          \"the PSM FDR 10-fold to 1% for only the protein groups already identified using the more stringent setting. Thus, additional PSMs could be rescued without adversely affecting the protein FDR, which remained locked at the level from the first step of the\",\n",
            "          \"which remained locked at the level from the first step of the procedure. Examination of the rescued peptides revealed that the majority had PSM FDR well below acceptable levels (median 0.2%), thus demonstrating that these additional peptide matches did\",\n",
            "          \"thus demonstrating that these additional peptide matches did not compromise the accuracy of the protein identifications. Our analysis of this dataset illustrates challenges associated with very large proteomic studies. Although our approach adapts\",\n",
            "          \"with very large proteomic studies. Although our approach adapts established target-decoy methodology, new approaches to this challenge merit further study. Detection of novel single amino acid variants (SAAVs) in the dataset We previously designed and\",\n",
            "          \"acid variants (SAAVs) in the dataset We previously designed and implemented a new strategy for the detection of protein variant sequences by using genomic data for peptide identification. This new approach was used to leverage mRNA sequencing data for the\",\n",
            "          \"new approach was used to leverage mRNA sequencing data for the discovery of 796 SAAVs in the TCGA colorectal carcinoma dataset. About 70% (561 of 796) of these had been previously described as single nucleotide polymorphisms (SNPs) occurring as natural\",\n",
            "          \"as single nucleotide polymorphisms (SNPs) occurring as natural variation in the human population. SNPs leading to amino acid changes in proteins occur with varying frequencies and are generally not associated with disease phenotypes. Of the remaining 235\",\n",
            "          \"not associated with disease phenotypes. Of the remaining 235 SAAVs, 73 were previously described in either the COSMIC database or tagged as somatic mutation in the genomic analysis of the TCGA dataset. Thus, the remaining 162 SAAVs detected in our\",\n",
            "          \"the TCGA dataset. Thus, the remaining 162 SAAVs detected in our analysis could not be explained by existing knowledge and were determined to be \\u2018new\\u2019 (Supplementary Table S4). Given the low false-discovery rate in our proteomic dataset and the fact that\",\n",
            "          \"false-discovery rate in our proteomic dataset and the fact that independent, RNAseq-based, evidence existed, the likelihood that these new SAAVs were real findings was high. To determine the reliability of the identifications we manually examined the top\",\n",
            "          \"reliability of the identifications we manually examined the top SAAVs with at least two spectra identified (N=70) from the dataset using BLAST searches and matching to the most recent National Center for Biotechnology Information (NCBI) database of SNPs.\",\n",
            "          \"Center for Biotechnology Information (NCBI) database of SNPs. The results from this follow up analysis demonstrate that many of the most abundant of these SAAVs are correct assignments, which appear to be truly novel (summarized in Supplementary Table\",\n",
            "          \"appear to be truly novel (summarized in Supplementary Table S4). The analyses also revealed apparently trivial explanations for some incorrect assignments, which nevertheless are instructive. For example, peptide \\u2018INLVQR\\u2019, representing the F76I variant of\",\n",
            "          \"For example, peptide \\u2018INLVQR\\u2019, representing the F76I variant of DWR67 with the highest number of spectral counts (375) is an isobar, indistinguishable by mass spectrometry from the peptide \\u2018LNLVQR\\u2019 from LDHA, an abundant metabolic protein. Three other\",\n",
            "          \"\\u2018LNLVQR\\u2019 from LDHA, an abundant metabolic protein. Three other peptides had a similar L to I variation which, given the RNA-seq data, were likely to be true but could not be confirmed by MS because of the lack of a mass difference. There were 14 peptides\",\n",
            "          \"of the lack of a mass difference. There were 14 peptides that matched to SNPs not present in the original databases and 6 peptides that matched to published immunoglobulin and major histocompatability complex sequence variants. The remaining 46 of the\",\n",
            "          \"complex sequence variants. The remaining 46 of the originally selected 70 peptides appear to be truly novel. Two variant sequences each were found for 2 proteins (MYH9 and PKM), resulting in a total of 44 proteins with novel variant sequences. Future\",\n",
            "          \"in a total of 44 proteins with novel variant sequences. Future research will be needed to examine the biological relevance of each of the observed variant sequences. Furthermore, these results underscore the need to perform additional manual analyses on\",\n",
            "          \"underscore the need to perform additional manual analyses on variant peptides identified through automated procedures as recently reviewed by Nesvizhski. These include manual inspection for trivial explanations, comparisons to sequence databases and\",\n",
            "          \"for trivial explanations, comparisons to sequence databases and comparison with chromatography and spectra obtained from synthetic variant sequence peptides. Germline versus somatic single amino acid variations One observation we originally made on the\",\n",
            "          \"amino acid variations One observation we originally made on the basis of SAAV detection in 10 colorectal carcinoma cell lines was that germline variants were about twice as likely to be detectable at the protein level as somatic variants. The larger set\",\n",
            "          \"at the protein level as somatic variants. The larger set of TCGA colorectal carcinomas provided a second confirmation of this original observation, again, the proportion of germline variant sequences observed in the mass spectrometry data was about twice\",\n",
            "          \"observed in the mass spectrometry data was about twice as high as the proportion of somatic variant sequences that would have been expected based on their abundance in the full dataset. These numbers are potentially influenced by the limitations of\",\n",
            "          \"These numbers are potentially influenced by the limitations of detection by mass spectrometry, although those limitations apply to both types of protein variants. Somatic variants may also be present at lower levels than germline variants because they are\",\n",
            "          \"present at lower levels than germline variants because they are expected to only occur in the tumor cells and not in adjacent normal cells that also contribute to the overall proteomic inventory from the TCGA tumors. It is also possible that somatic\",\n",
            "          \"from the TCGA tumors. It is also possible that somatic variants are only present in a sub-set of the tumor cells due to tumor heterogeneity. However, both lines of reasoning do not apply to our earlier observation in the more homogeneous colorectal\",\n",
            "          \"to our earlier observation in the more homogeneous colorectal carcinoma cell lines. A direct comparison of germline- and somatically-derived variant proteins will be needed to confirm the validity of our original observations. Technical Validation Summary\",\n",
            "          \"of our original observations. Technical Validation Summary We used three quality control strategies throughout the analysis of TCGA colorectal carcinoma samples. The first utilized a standard BSA tryptic digest after every colon carcinoma sample of 15\",\n",
            "          \"BSA tryptic digest after every colon carcinoma sample of 15 fractions, followed by manual inspection to assess peptide peak intensities and chromatographic features. The second was achieved by analyzing a CompRef\\u2009sample after each set of 5 colorectal\",\n",
            "          \"by analyzing a CompRef\\u2009sample after each set of 5 colorectal carcinoma samples and the third was by utilizing the identification-independent quality metric software package QuaMeter. Day-to-day quality control The BSA runs were evaluated by mass\",\n",
            "          \"Day-to-day quality control The BSA runs were evaluated by mass calibration check and using intensity and coverage of the BSA PSMs identified by database search. Mass axis shift was determined by comparing the experimental mass to the known monoisotopic\",\n",
            "          \"by comparing the experimental mass to the known monoisotopic mass of five different peptides found in the BSA standard. These peptides are: SLHTLFGDELCK (z=3), LVNELTEFAK (z=2), HLVDEPQNLIK (z=2), YICDNQDTISSK (z=2), KVPQVSTPTLVEVSR (z=2 and z=3). If the\",\n",
            "          \"YICDNQDTISSK (z=2), KVPQVSTPTLVEVSR (z=2 and z=3). If the mass axis shift exceeded 10\\u2009ppm, the instrument was recalibrated prior to any sample runs. In addition, the BSA base peak chromatogram was viewed to determine that peak shape and retention times\",\n",
            "          \"was viewed to determine that peak shape and retention times are appropriate. Specifically, a total intensity of 107\\u2009derived from the BSA chromatogram and BSA coverage of at least 65% was required for acceptance of a set of colon carcinoma runs. Any\",\n",
            "          \"required for acceptance of a set of colon carcinoma runs. Any failure to achieve these quality control measures prompted instrument maintenance before data acquisition was allowed to continue. Use of standardized CompRef sample CompRef samples obtained\",\n",
            "          \"Use of standardized CompRef sample CompRef samples obtained from WHIM2 (basal type) and WHIM16 (luminal type) breast carcinoma xenografts\\u2009were alternated and analyzed after every 5 colorectal carcinoma and normal epithelium samples throughout the full\",\n",
            "          \"carcinoma and normal epithelium samples throughout the full analysis period of the data set (from July 2012 until August 2013). Each of these runs was searched using MyriMatch against a combined human/mouse RefSeq-based database and the results compared\",\n",
            "          \"human/mouse RefSeq-based database and the results compared to the initial CompRef samples used to establish our shotgun proteomic platform (not published). QuaMeter was used as a tool to evaluate instrument performance for each sample run. The quality\",\n",
            "          \"instrument performance for each sample run. The quality control goals for each of these new runs were: 1) more than 5,000 identified spectra per run, 2) third quartile MS2 density of more than 200, and 3) more than 40% of identified spectra have a charge\",\n",
            "          \"200, and 3) more than 40% of identified spectra have a charge state of 2. These metrics guarantee that sufficient numbers of scans are detected, that they have sufficient signal to allow identification, and that stable precursor charge state ratios were\",\n",
            "          \"and that stable precursor charge state ratios were achieved. One of the main reasons to include the CompRef samples in this dataset was to compare the consistency of the observed biological differences between the basal and luminal subtypes. To test the\",\n",
            "          \"differences between the basal and luminal subtypes. To test the consistency of such observations, we compiled a dataset containing all 32 CompRef samples that were analyzed with the TCGA carcinomas and normal biopsies. This dataset included a total of\",\n",
            "          \"and normal biopsies. This dataset included a total of 8,029 identified protein groups with a protein FDR of 3.7% using a combined human/mouse RefSeq database (provided as an IDP3 database available through ProteomeXchange (Data Citation 13) or from the\",\n",
            "          \"through ProteomeXchange (Data Citation 13) or from the CPTAC data portal (cptac-data-portal.georgetown.edu/cptacPublic)). We compared the WHIM2 basal with the WHIM16 luminal samples using QuasiTel and selected 172 human proteins that distinguished the two\",\n",
            "          \"and selected 172 human proteins that distinguished the two xenograft types. Spectral counts for these proteins were at least 32-fold higher in one group over the other with an FDR- of less than 0.001. The full dataset for this analysis consisted of 5,633\",\n",
            "          \"0.001. The full dataset for this analysis consisted of 5,633 proteins that had at least 45 counts across all 32 samples, according to the 1.4 spectral count average we earlier established to be necessary for appropriate quantification by spectral\",\n",
            "          \"to be necessary for appropriate quantification by spectral counting. We then created comparisons for all 256 possible combinations of WHIM2 and WHIM16 (16\\u00d716 samples) between single pairs of the WHIM2 and WHIM16 samples and obtained spectral count ratios\",\n",
            "          \"the WHIM2 and WHIM16 samples and obtained spectral count ratios for all proteins. The predictive value of the set of 172 differential proteins (the \\u2018proteomic signature\\u2019) was tested against these combinations using Receiver Operating Characteristic (ROC)\",\n",
            "          \"combinations using Receiver Operating Characteristic (ROC) analysis using \\u2018area under the curve\\u2019 (AUC) as a measure of accuracy. The median AUC for this analysis was 0.931 (+s.d.=0.028), indicating that the most prominent differential proteins are\",\n",
            "          \"indicating that the most prominent differential proteins are captured when comparing single runs from each of the CompRef subtypes (Fig. 3). This suggests that biologically relevant phenotype differences remain observable even when individual MS analysis\",\n",
            "          \"differences remain observable even when individual MS analysis results are used. Use of instrument values for quality control The QuaMeter performance metrics have been computed for the full set of 1425 LC-MS/MS experiments for the 95 TCGA colon samples\",\n",
            "          \"set of 1425 LC-MS/MS experiments for the 95 TCGA colon samples (see Methods). The software computed quality metrics for each file that were independent of identifications from Pepitome or any other search engine. These metrics enabled recognition of\",\n",
            "          \"any other search engine. These metrics enabled recognition of outliers among LC-MS/MS experiments for a particular sample and of batch effects from instrument drift over the ten-month interval during which these experiments were conducted. The 95 samples\",\n",
            "          \"during which these experiments were conducted. The 95 samples were run sequentially and are separated into 10 \\u2018batches\\u2019 for presentation purposes, with each sample represented by fifteen LC-MS/MS experiments. Each batch is represented by a different color\",\n",
            "          \"experiments. Each batch is represented by a different color in Fig. 4, with the ten samples each represented by a letter from A to J. QuaMeter reduced each LC-MS/MS experiment to a vector of 44 metrics, and multivariate analysis in the R statistical\",\n",
            "          \"of 44 metrics, and multivariate analysis in the R statistical environment transformed the metrics by robust Principal Components Analysis (PCA). The letters in each batch generally occupy the same region of the principal component space. The first two\",\n",
            "          \"the same region of the principal component space. The first two principal components accounted for 42.5% of the variability (Fig. 4), suggesting diversity in the forces causing variation. The first principal component placed the most positive loading on\",\n",
            "          \"first principal component placed the most positive loading on extracted ion chromatogram height ratios (XIC.Height.Q2, XIC.Height.Q3) and the most negative loading on the number of peaks in MS/MS scans (MS2.Density.Q*). Component two, on the other hand,\",\n",
            "          \"MS/MS scans (MS2.Density.Q*). Component two, on the other hand, emphasized positive loadings on the retention time interval during which the second quartile of MS scans were acquired (RT.MS.Q2) and a negative loading on the third quartile of MS scan\",\n",
            "          \"and a negative loading on the third quartile of MS scan intensity (MS1.TIC.Q3) and of MS scan intensity change (MS1.TIC.Change.Q3). The lack of a clear-cut source of variation indicates that multiple, interacting components determine quality metrics\",\n",
            "          \"that multiple, interacting components determine quality metrics stability, and thus do not point towards any particular part of the procedure that could be stabilized further to improve the quality metrics as determined by QuaMeter. Within each set of\",\n",
            "          \"quality metrics as determined by QuaMeter. Within each set of fifteen LC-MS/MS experiments, statistical scripts were used to determine the distance separating all possible pairs of the 15 fractions (Fig. 5, panel a). For most samples, the median value was\",\n",
            "          \"(Fig. 5, panel a). For most samples, the median value was close to the mean and relatively few distances were substantially lower or higher than the median. For each tumor sample, we compared Euclidian distances of each of the 15 fractions to the sample\",\n",
            "          \"Euclidian distances of each of the 15 fractions to the sample median and used the T2\\u2009statistic in a Chi2-test to identify outlier values. When we set the cut-off at P<0.01, 82 of the 1425 fractions are identified as outliers (indicated in blue). Figure 5\",\n",
            "          \"are identified as outliers (indicated in blue). Figure 5 also shows the number of identified spectra, unique peptides and protein groups for each of the fractions. These plots show the variability within each sample and the variability between fractions.\",\n",
            "          \"within each sample and the variability between fractions. A 2-way ANOVA analysis of these numbers showed that more than 80% of the variability was due to differences between samples, with just 3\\u20135% of the variability due to fraction and the remainder was\",\n",
            "          \"3\\u20135% of the variability due to fraction and the remainder was the error component. Thus, differences in protein composition of the various tumor samples make up the vast majority of the difference observed in spectral counts, distinct peptides and protein\",\n",
            "          \"observed in spectral counts, distinct peptides and protein groups. When we compared the search results from the 82 fractions identified as outliers by the performance metrics, these fractions had lower identifications of spectra, peptides and proteins\",\n",
            "          \"had lower identifications of spectra, peptides and proteins compared to the non-outlier fractions (Fig. 6). However, individual fractions with normal or above-normal identifications were also part of this group. Thus, our conclusion from these analyses is\",\n",
            "          \"part of this group. Thus, our conclusion from these analyses is that outliers identified with the performance metrics do not necessarily identify sub-standard fractions in this dataset, but that they may indicate sources of variability that could be\",\n",
            "          \"but that they may indicate sources of variability that could be better controlled in future studies. Usage Notes To date, our compilation of TCGA colorectal carcinomas yielded the first large proteomic dataset for a human cancer. The use of alternative\",\n",
            "          \"proteomic dataset for a human cancer. The use of alternative analysis strategies on this dataset have the potential to identify a larger number of proteins and protein forms. The datasets included with the original publication consisted of all instrument\",\n",
            "          \"with the original publication consisted of all instrument (RAW) files and the final end product files in the form of IDP3 database (idpDB) files. The pipeline that leads from the source data to the final IDP3 file, described in detail under \\u2018Methods\\u2019,\",\n",
            "          \"to the final IDP3 file, described in detail under \\u2018Methods\\u2019, represents one of the possible strategies of dealing with large proteomic datasets. In this publication, we make available all intermediate files that were generated in the process of analyzing\",\n",
            "          \"files that were generated in the process of analyzing this dataset. These include the standard and individual customized FASTA databases used for protein search, the standard spectral library, and the search result files (pepXML/mzID) from Myrimatch,\",\n",
            "          \"and the search result files (pepXML/mzID) from Myrimatch, Pepitome and MS-GF+ searches. These files can serve as direct input to the various other proteomic analysis tools such as the Trans Proteomic Pipeline (TPP), or commercial packages such as\",\n",
            "          \"Trans Proteomic Pipeline (TPP), or commercial packages such as Scaffold. To facilitate access to the TCGA colorectal dataset we created a Colorectal Cancer Portal in NetGestalt\\u2009that encompasses data from all analyses types performed on this set of\",\n",
            "          \"data from all analyses types performed on this set of clinical specimens (http://www.netgestalt.org), including genomic, epigenomic, transcriptomic, proteomic, and clinical data. The portal also includes microarray gene expression data from the Gene\",\n",
            "          \"also includes microarray gene expression data from the Gene Expression Omnibus GEO for five CRC cohorts; mRNA expression, copy number, mutation, and drug response data for CRC cell lines from the Cancer Cell Line Encyclopedia project; and shRNA-based\",\n",
            "          \"from the Cancer Cell Line Encyclopedia project; and shRNA-based high-throughput screen data for CRC cell lines from the Achilles project. The portal provides for easy data access and visualization, data integration of proteomic data with other colorectal\",\n",
            "          \"data integration of proteomic data with other colorectal carcinoma data types, and biological interrogation against multiple biological knowledge bases (protein interaction networks, MsigDB, KEGG etc.). Examples of such analyses include the integration of\",\n",
            "          \"etc.). Examples of such analyses include the integration of all data types for a single gene or a set of genes that allow comparisons of epigenetic, copy number, mutational, mRNA expression, and protein expression data; the query of protein networks that\",\n",
            "          \"and protein expression data; the query of protein networks that are associated with specific sub-types of colorectal carcinoma; and the prioritization of candidate drivers by integrating multiple types of data. Additional Information How to cite this\",\n",
            "          \"multiple types of data. Additional Information How to cite this article: Slebos, R. J. C. et al. Proteomic analysis of colon and rectal carcinoma using standard and customized databases. Sci. Data 2:150022 doi: 10.1038/sdata.2015.22 (2015). Supplementary\",\n",
            "          \"Data 2:150022 doi: 10.1038/sdata.2015.22 (2015). Supplementary Material Data Citations The authors declare no competing financial interests. Comparison of identified spectra, peptides and proteins by the three search engines Myrimatch, Pepitome and\",\n",
            "          \"proteins by the three search engines Myrimatch, Pepitome and MS-GF+. Over 93% of the proteins in the dataset are identified by all 3 search engines, while the spectra and peptide inventories benefit more from each of the individual contributions of the\",\n",
            "          \"benefit more from each of the individual contributions of the different search engines. The spectral library search engine Pepitome increases the overall spectral count totals by 13% through the identification of previously observed spectra that were not\",\n",
            "          \"the identification of previously observed spectra that were not identified by the other search engines. The contribution of Pepitome is less in the area of unique peptides and proteins. MS-GF+ contributes a large fraction of all spectral counts (9.4%) and\",\n",
            "          \"contributes a large fraction of all spectral counts (9.4%) and distinct peptides (16.1%). Impact of peptide-to-spectrum match (PSM) false discovery rate (FDR) threshold at different levels of overall protein FDR on the total protein inventories.\",\n",
            "          \"levels of overall protein FDR on the total protein inventories. Surprisingly, increased PSM FDR stringency (lower values) increased the number of identifiable proteins. Protein FDR was maintained below 5% by requiring a minimum number of spectral counts\",\n",
            "          \"below 5% by requiring a minimum number of spectral counts for each protein across the dataset. The spectral count minimum requires to maintain a protein FDR below 5% increased with the applied PSM FDR. Performance of a 172 protein signature chosen to\",\n",
            "          \"PSM FDR. Performance of a 172 protein signature chosen to distinguish proteomic differences between single pairs of basal and luminal breast cancer xenografts. The protein signature distinguished the two breast tumor subtypes in almost all paired\",\n",
            "          \"the two breast tumor subtypes in almost all paired combinations of proteomic datasets generated for the two types of xenografts. Principal component analysis of LC-MS/MS system performance metrics. Forty-four metrics from a total of 1,425 LC-MS/MS\",\n",
            "          \"metrics. Forty-four metrics from a total of 1,425 LC-MS/MS experiments were collapsed into two principal components, which accounted for 42.5% of the total variation. Comparison of all TCGA samples with respect to normalized Euclidean distance based on\",\n",
            "          \"samples with respect to normalized Euclidean distance based on performance metrics and numbers of filtered spectra, distinct peptides and protein groups. Comparison of protein identification values between LC-MS/MS experiments classified as \\u2018outlier\\u2019 and\",\n",
            "          \"values between LC-MS/MS experiments classified as \\u2018outlier\\u2019 and \\u2018non-outlier\\u2019 based on performance metrics. The mean values for spectral counts, peptide and protein identification were lower in the analyses classified as outliers.\"\n",
            "        ]\n",
            "      },\n",
            "      {\n",
            "        \"publication_id\": \"44e2e0ce-89c7-11e8-bcf1-0a2705229b82\",\n",
            "        \"pubmed_id\": \"25043054\",\n",
            "        \"doi\": \"10.1038/nature13438\",\n",
            "        \"title\": \"Proteogenomic characterization of human colon and rectal cancer\",\n",
            "        \"journal\": \"Nature\",\n",
            "        \"journal_url\": \"https://www.nature.com/articles/nature13438\",\n",
            "        \"year\": \"2014\",\n",
            "        \"abstract\": \"Extensive genomic characterization of human cancers presents the problem of inference from genomic abnormalities to cancer phenotypes. To address this problem, we analysed proteomes of colon and rectal tumours characterized previously by The Cancer Genome Atlas (TCGA) and perform integrated proteogenomic analyses. Somatic variants displayed reduced protein abundance compared to germline variants. Messenger RNA transcript abundance did not reliably predict protein abundance differences between tumours. Proteomics identified five proteomic subtypes in the TCGA cohort, two of which overlapped with the TCGA 'microsatellite instability/CpG island methylation phenotype' transcriptomic subtype, but had distinct mutation, methylation and protein expression patterns associated with different clinical outcomes. Although copy number alterations showed strong cis- and trans-effects on mRNA abundance, relatively few of these extend to the protein level. Thus, proteomics data enabled prioritization of candidate driver genes. The chromosome 20q amplicon was associated with the largest global changes at both mRNA and protein levels; proteomics data highlighted potential 20q candidates, including HNF4A (hepatocyte nuclear factor 4, alpha), TOMM34 (translocase of outer mitochondrial membrane 34) and SRC (SRC proto-oncogene, non-receptor tyrosine kinase). Integrated proteogenomic analysis provides functional context to interpret genomic abnormalities and affords a new paradigm for understanding cancer biology.\",\n",
            "        \"full_text\": [\n",
            "          \"Proteogenomic characterization of human colon and rectal cancer Summary We analyzed proteomes of colon and rectal tumors previously characterized by the Cancer Genome Atlas (TCGA) and performed integrated proteogenomic analyses. Somatic variants displayed\",\n",
            "          \"integrated proteogenomic analyses. Somatic variants displayed reduced protein abundance compared to germline variants. mRNA transcript abundance did not reliably predict protein abundance differences between tumors. Proteomics identified five proteomic\",\n",
            "          \"between tumors. Proteomics identified five proteomic subtypes in the TCGA cohort, two of which overlapped with the TCGA \\u201cMSI/CIMP\\u201d transcriptomic subtype, but had distinct mutation, methylation, and protein expression patterns associated with different\",\n",
            "          \"and protein expression patterns associated with different clinical outcomes. Although copy number alterations showed strong cis- and trans-effects on mRNA abundance, relatively few of these extend to the protein level. Thus, proteomics data enabled\",\n",
            "          \"extend to the protein level. Thus, proteomics data enabled prioritization of candidate driver genes. The chromosome 20q amplicon was associated with the largest global changes at both mRNA and protein levels; proteomics data highlighted potential 20q\",\n",
            "          \"and protein levels; proteomics data highlighted potential 20q candidates including HNF4A, TOMM34 and SRC. Integrated proteogenomic analysis provides functional context to interpret genomic abnormalities and affords a new paradigm for understanding cancer\",\n",
            "          \"and affords a new paradigm for understanding cancer biology. Introduction The Cancer Genome Atlas (TCGA) has characterized the genomic features of human cancers and thereby presents the challenge of explaining how genomic alterations drive cancers.\",\n",
            "          \"challenge of explaining how genomic alterations drive cancers. Because proteins link genotypes to phenotypes, the Clinical Proteomic Tumor Analysis Consortium (CPTAC) is performing proteomic analyses of TCGA tumor specimens for selected cancer types. Here\",\n",
            "          \"of TCGA tumor specimens for selected cancer types. Here we present the first integrated proteogenomic characterization of human cancer with an analysis of the TCGA colorectal cancer (CRC) specimens. The TCGA study affirmed well-established genomic\",\n",
            "          \"specimens. The TCGA study affirmed well-established genomic features of CRC and described three transcriptional subtypes, 17 chromosomal regions of significant focal amplification and 28 regions of significant focal deletion, and linked genomic features\",\n",
            "          \"of significant focal deletion, and linked genomic features of CRC to critical signaling pathways. The drivers underlying copy number alterations (CNAs) and transcriptional subtypes are largely unknown, and an integrative analysis of both genomic and\",\n",
            "          \"unknown, and an integrative analysis of both genomic and proteomic data may provide a more comprehensive understanding of the information flow from DNA to protein to phenotype. Identification of peptides, proteins and variant sequences We performed liquid\",\n",
            "          \"of peptides, proteins and variant sequences We performed liquid chromatography-tandem mass spectrometry (LC-MS/MS)-based shotgun proteomic analyses on 95 TCGA tumor samples (Extended Data Fig. 1, Supplementary Methods 1\\u20134), the clinical and pathological\",\n",
            "          \"1, Supplementary Methods 1\\u20134), the clinical and pathological characteristics and TCGA datasets for which are summarized in Supplementary Table 1. Benchmark quality control (QC) samples from one basal and one luminal human breast tumor xenograft were\",\n",
            "          \"one basal and one luminal human breast tumor xenograft were analyzed in alternating order after every five CRC samples (Supplementary Methods 2). We identified a total of 124,823 distinct peptides among the 95 samples, corresponding to 6,299,756 spectra\",\n",
            "          \"among the 95 samples, corresponding to 6,299,756 spectra in an assembly of 7,526 protein groups with a protein-level False Discovery Rate (FDR) of 2.64% (Supplementary Methods 5.1\\u20135.2, Extended Data Fig. 2). To facilitate integration between genomic and\",\n",
            "          \"Data Fig. 2). To facilitate integration between genomic and proteomic data, a gene-level assembly of the peptides identified 7,211 genes. A fundamental question in proteogenomics is which protein coding alterations are expressed at the protein level.\",\n",
            "          \"protein coding alterations are expressed at the protein level. Because standard database search approaches cannot identify variant pe3ptides from MS/MS data, we also performed database searches with customized sequence databases from matched RNA-Seq data\",\n",
            "          \"with customized sequence databases from matched RNA-Seq data for individual samples (Supplementary Methods 5.3, Extended Data Fig. 3). We identified 796 single amino acid variants (SAAVs) across all 86 tumors for which matched RNA-Seq data was available\",\n",
            "          \"all 86 tumors for which matched RNA-Seq data was available (Fig. 1a\\u2013b, Supplementary Table 2\\u20133), among which 64 corresponded to somatic variants reported by TCGA and 101 were reported in the COSMIC database (i.e. COSMIC-supported variants). Of the\",\n",
            "          \"in the COSMIC database (i.e. COSMIC-supported variants). Of the remaining SAAVs, 526 were listed in the dbSNP database (i.e. dbSNP-supported variants) and are likely to be germline variants. The 162 previously unreported SAAVs might be explained by novel\",\n",
            "          \"The 162 previously unreported SAAVs might be explained by novel somatic or germline variants, RNA editing, or, in some cases, false discovery. The identified somatic variants were clearly enriched in the hypermutated samples, whereas the germline variants\",\n",
            "          \"in the hypermutated samples, whereas the germline variants showed no association with hypermutation (Fig. 1a). Although 58% of the germline variants occurred in two or more samples, almost all somatic variants occurred in only one sample (Fig. 1c). The\",\n",
            "          \"all somatic variants occurred in only one sample (Fig. 1c). The low identification rate for somatic variants may reflect relatively low sequence coverage in shotgun proteomics; however, somatic variants also might negatively impact protein abundance,\",\n",
            "          \"variants also might negatively impact protein abundance, possibly by reducing translational efficiency or protein stability. Using the protein abundance quantification method described below and detailed in Supplementary Methods 5.4, we found that somatic\",\n",
            "          \"detailed in Supplementary Methods 5.4, we found that somatic variants exerted a significantly stronger negative impact on protein abundance than did dbSNP-supported variants (p<2.2 \\u00d7 10-16, Kolmogorov-Smirnov (KS) test, Fig. 1d, Supplementary Methods\",\n",
            "          \"Kolmogorov-Smirnov (KS) test, Fig. 1d, Supplementary Methods 5.5). The percentage of variants with an impact score of less than \\u22122 was doubled for somatic variants compared to dbSNP-supported variants (p<2.2 \\u00d7 10-16, Chi-squared test, Fig. 1d).\",\n",
            "          \"variants (p<2.2 \\u00d7 10-16, Chi-squared test, Fig. 1d). Cancer-related variant proteins may serve as candidate protein biomarkers or therapeutic targets. The 108 somatic or COSMIC-supported protein variants mapped to 105 genes, including known cancer genes\",\n",
            "          \"variants mapped to 105 genes, including known cancer genes in the Cancer Gene Census database such as KRAS, CTNNB1, SF3B1, ALDH2, and FH. The list also included 14 targets of FDA-approved drugs or drugs in clinical trials, such as ALDH2, HSD17B4, PARP1,\",\n",
            "          \"or drugs in clinical trials, such as ALDH2, HSD17B4, PARP1, P4HB, TST, GAK, SLC25A24, and SUPT16H. A subset of variant peptide sequences, including K-ras (G12D) were verified by targeted analyses of tumor lysates spiked with synthetic, isotope-labeled\",\n",
            "          \"of tumor lysates spiked with synthetic, isotope-labeled peptide standards (Supplementary Methods 6). One example is shown in Extended Data Fig. 4. Quantification of protein abundance To quantify protein abundance, we used spectral counts, which are the\",\n",
            "          \"protein abundance, we used spectral counts, which are the total number of MS/MS spectra acquired for peptides from a given protein (Supplementary Methods 5.4, Supplementary Table 4). Analysis of data from benchmark QC samples demonstrated platform\",\n",
            "          \"of data from benchmark QC samples demonstrated platform reproducibility throughout the analyses and enabled evaluation of data normalization methods (Extended Data Fig. 5a\\u2013b). Based on the minimal spectral count requirement established using the QC data\",\n",
            "          \"spectral count requirement established using the QC data set (Extended Data Fig. 5c), 3,899 genes with a protein-level FDR of 0.43% were used to compare relative protein abundance across tumor samples. mRNA abundance does not reliably predict protein\",\n",
            "          \"tumor samples. mRNA abundance does not reliably predict protein abundance The matched proteomic and RNA-Seq measurements from the TCGA CRC tumors allowed the first global analysis of transcript-protein relationships in a large human tumor cohort\",\n",
            "          \"transcript-protein relationships in a large human tumor cohort (Supplementary Methods 7). First, we compared the steady state mRNA and protein abundance for each gene within individual samples (Supplementary Methods 7.2\\u20137.3, Extended Data Fig. 6a). All\",\n",
            "          \"(Supplementary Methods 7.2\\u20137.3, Extended Data Fig. 6a). All samples showed significant positive mRNA-protein correlation (multiple-test adjusted p value < 0.01, Spearman\\u2019s correlation coefficient) and the average correlation between steady state mRNA and\",\n",
            "          \"and the average correlation between steady state mRNA and protein abundance in individual samples was 0.47 (Fig. 2a), which is comparable to previous reports in multi-cellular organisms. Next, we examined the concordance between mRNA and protein variation\",\n",
            "          \"we examined the concordance between mRNA and protein variation of individual genes across the 87 tumors for which 3,764 genes had both mRNA and protein measurements suitable for relative abundance comparison (Supplementary Methods 7.2, 7.4). Although 89%\",\n",
            "          \"comparison (Supplementary Methods 7.2, 7.4). Although 89% of the genes showed a positive mRNA-protein correlation, only 32% had statistically significant correlations (Fig. 2b). The average Spearman\\u2019s correlation between mRNA and protein variation was\",\n",
            "          \"Spearman\\u2019s correlation between mRNA and protein variation was 0.23, which was comparable to reported values for yeast, mouse and human cell lines. To test whether the concordance between protein and mRNA variation is related to the biological function of\",\n",
            "          \"and mRNA variation is related to the biological function of the gene product, we performed KEGG enrichment analysis (Supplementary Methods 7.5, Supplementary Table 5). Genes involved in several metabolic processes showed concordant mRNA and protein\",\n",
            "          \"several metabolic processes showed concordant mRNA and protein variation, whereas other gene classes showed low or even negative concordance in mRNA and protein variation (Figure 2c). We also found that genes with stable mRNA and stable protein tend to\",\n",
            "          \"found that genes with stable mRNA and stable protein tend to have higher mRNA-protein correlation than those with unstable mRNA and unstable protein (p = 5.27 \\u00d7 10-6, two-sided Wilcoxon rank-sum test, Supplementary Methods 7.6, Extended Data Fig. 6b).\",\n",
            "          \"test, Supplementary Methods 7.6, Extended Data Fig. 6b). mRNA measurements thus are poor predictors of protein abundance variations and both biological functions of the gene products and mRNA and protein stability may govern mRNA-protein correlation.\",\n",
            "          \"mRNA and protein stability may govern mRNA-protein correlation. Impact of copy number alterations on mRNA and protein abundance The TCGA study identified 17 regions of significant focal amplification and 28 regions of significant focal deletion. We\",\n",
            "          \"amplification and 28 regions of significant focal deletion. We examined the impact of CNAs on mRNA and protein abundance, including both cis-effects on the abundance of genes in the same loci and trans-effects on the abundance of genes at other loci in\",\n",
            "          \"and trans-effects on the abundance of genes at other loci in the genome (Supplementary Methods 8). For all 23,125 genes with a CNA measurement in the TCGA data set, we calculated Spearman\\u2019s correlation with mRNA and protein abundance, respectively for the\",\n",
            "          \"with mRNA and protein abundance, respectively for the 3,764 genes with both mRNA and protein measurements (Supplementary Methods 8.1). Examination of the matrix visualizing significant CNA-mRNA correlations (multiple-test adjusted p value < 0.01) revealed\",\n",
            "          \"correlations (multiple-test adjusted p value < 0.01) revealed strong positive correlations along the diagonal (Fig. 3a), suggesting strong cis-effects of CNAs on mRNA abundance. Most of the diagonal signals corresponded to previously reported arm-level\",\n",
            "          \"diagonal signals corresponded to previously reported arm-level changes. In contrast, the diagonal pattern was much weaker for CNA-protein correlations (Fig. 3b). To further investigate the cis-effects of CNAs, we separated all genes with CNA, mRNA, and\",\n",
            "          \"cis-effects of CNAs, we separated all genes with CNA, mRNA, and protein measurements into those in focal amplification regions, focal deletion regions, and non-focal regions (i.e., chromosomal regions without focal amplification or deletion). As shown in\",\n",
            "          \"regions without focal amplification or deletion). As shown in Extended Data Figure 7, CNA-mRNA correlations were significantly higher than CNA-protein correlations for genes in all three groups (p<1.0 \\u00d7 10-10, KS test). Moreover, genes in the focal\",\n",
            "          \"groups (p<1.0 \\u00d7 10-10, KS test). Moreover, genes in the focal amplification regions showed significantly higher CNA-mRNA and CNA-protein correlations than genes in the non-focal regions (p=4.4 \\u00d7 10-4 and 0.02, respectively, KS test). However, the same\",\n",
            "          \"\\u00d7 10-4 and 0.02, respectively, KS test). However, the same trend was not observed for genes in the focal deletion regions. Therefore, focal amplifications have the strongest cis-effects on both mRNA and protein abundance, suggesting that selection for\",\n",
            "          \"both mRNA and protein abundance, suggesting that selection for high protein abundance may drive CNA in regions of focal amplification. On the other hand, many CNA-driven mRNA level increases do not translate into increased abundance of the corresponding\",\n",
            "          \"do not translate into increased abundance of the corresponding proteins. Figure 3a\\u2013b also revealed multiple trans-acting CNA hot spots, defined as chromosomal loci whose amplification is significantly associated with abundance changes of many transcripts\",\n",
            "          \"associated with abundance changes of many transcripts or proteins at other loci. Chromosomes 20q, 18, 16, 13 and 7 contained the five strongest hot spots driving global mRNA abundance variation. These hot spots also were strongest at the protein level.\",\n",
            "          \"These hot spots also were strongest at the protein level. Most hot spot-related transcript changes did not propagate to the protein level, presumably reflecting buffering of protein abundance by post-transcriptional regulation. Notably, many hot\",\n",
            "          \"abundance by post-transcriptional regulation. Notably, many hot spot-associated protein level alterations occurred in the absence of corresponding mRNA alterations, suggesting that the same trans-acting hot spot may exert independent effects at both the\",\n",
            "          \"trans-acting hot spot may exert independent effects at both the transcriptome and proteome levels. The 20q amplification was associated with the largest global changes in both mRNA and protein levels in this univariate analysis. The same conclusion was\",\n",
            "          \"levels in this univariate analysis. The same conclusion was reached with a regularized multivariate regression analysis method, remMap (Supplementary Methods 8.2, Supplementary Table 6\\u20139). These data highlight the importance of 20q amplification in CRC,\",\n",
            "          \"data highlight the importance of 20q amplification in CRC, which has not been well documented in previous studies. Among the 79 genes in the 20q region with quantifiable protein measurements, 67 (85%) showed significant CNA-mRNA correlation, but only 40\",\n",
            "          \"67 (85%) showed significant CNA-mRNA correlation, but only 40 (51%) showed significant CNA-protein correlation (multiple-test adjusted p value < 0.01, Spearman\\u2019s correlation coefficient, Supplementary Table 10). Because significant CNA-protein\",\n",
            "          \"Supplementary Table 10). Because significant CNA-protein correlations identify amplified sequences that translate to high protein abundance, proteomic measurements can help prioritize genes in amplified regions for further examination. Of particular\",\n",
            "          \"in amplified regions for further examination. Of particular interest among the 40 genes is HNF4A (Fig. 3c), a candidate driver gene nominated by TCGA for the 20q13.12 focal amplification peak. HNF4\\u03b1 is a transcription factor with a key role in normal\",\n",
            "          \"peak. HNF4\\u03b1 is a transcription factor with a key role in normal gastrointestinal development and is increasingly being linked to CRC. However, there are contradictory reports on whether HNF4\\u03b1 acts as an oncogene or a tumor suppressor gene in CRC. Upon\",\n",
            "          \"acts as an oncogene or a tumor suppressor gene in CRC. Upon reanalysis of the HNF4A shRNA knockdown data for CRC cell lines from the Achilles project, we found that the dependency of CRC cells on HNF4\\u03b1 correlated significantly with the amplification level\",\n",
            "          \"on HNF4\\u03b1 correlated significantly with the amplification level of HNF4A (Supplementary Methods 8.3, Extended Data Fig. 8), which may partially explain the contradictory roles reported for HNF4\\u03b1 in CRC. Other interesting candidates included TOMM34 (Fig.\",\n",
            "          \"in CRC. Other interesting candidates included TOMM34 (Fig. 3d), which is over-expressed frequently in CRC tumors and is involved in the growth of CRC cells, and SRC (Fig. 3e), which encodes a non-receptor tyrosine kinase implicated in several human\",\n",
            "          \"a non-receptor tyrosine kinase implicated in several human cancers including CRC. Proteomic subtypes of CRC The TCGA study reported three transcriptomic subtypes of CRC, designated \\u201cMSI/CIMP\\u201d (microsatellite instability/CpG island methylator phenotype),\",\n",
            "          \"(microsatellite instability/CpG island methylator phenotype), \\u201cInvasive\\u201d, and \\u201cCIN\\u201d (chromosomal instability). Given the limited correlation between mRNA and protein levels, we asked whether CRC subtypes can be better represented with proteomics data.\",\n",
            "          \"CRC subtypes can be better represented with proteomics data. Using the Consensus Clustering method (Supplementary Methods 9.1\\u20139.2, Extended Data Fig. 9), we identified five major proteomic subtypes in this tumor cohort, with 15, 9, 25, 11, and 19 cases in\",\n",
            "          \"in this tumor cohort, with 15, 9, 25, 11, and 19 cases in subtypes A through E, respectively (Fig. 4a\\u2013b). We tested the association between the subtype classification and established genomic and epigenomic features of CRC using Fisher\\u2019s exact test (Fig.\",\n",
            "          \"and epigenomic features of CRC using Fisher\\u2019s exact test (Fig. 4c, Supplementary Table 11). Almost all hypermutated and MSI-high tumors were included in subtypes B and C, as well as tumors with POLE and BRAF mutations. However, statistically significant\",\n",
            "          \"POLE and BRAF mutations. However, statistically significant association with these features was only observed for subtype B (multiple-test adjusted p value < 0.05). Moreover, whereas subtype B was significantly associated with the TCGA CIMP-H (CIMP-high)\",\n",
            "          \"B was significantly associated with the TCGA CIMP-H (CIMP-high) methylation subtype, subtype C was significantly associated with a non-CIMP subtype (cluster 4). Another unique feature of subtype B was the lack of TP53 mutations and chromosome 18q loss.\",\n",
            "          \"B was the lack of TP53 mutations and chromosome 18q loss. These results clearly established the association between proteomic subtype B and MSI-High and CIMP, but suggest that subtype C might have different biological underpinnings. The remaining three\",\n",
            "          \"have different biological underpinnings. The remaining three subtypes were associated with CIN, another well-accepted genetic property of CRC. In particular, subtype E was significantly associated with both TP53 mutations and 18q loss, genomic features\",\n",
            "          \"with both TP53 mutations and 18q loss, genomic features frequently associated with CIN tumors. Interestingly, subtype E was also associated with HNF4A amplification and relatively higher abundance of HNF4\\u03b1 protein (Fig. 4d). HNF4\\u03b1 abundance was\",\n",
            "          \"abundance of HNF4\\u03b1 protein (Fig. 4d). HNF4\\u03b1 abundance was significantly higher in subtype E tumors compared to normal colon samples (multiple-test adjusted p value = 1.09 \\u00d7 10-6, two-sided Wilcoxon rank-sum test); however, significant up-regulation of\",\n",
            "          \"Wilcoxon rank-sum test); however, significant up-regulation of HNF4\\u03b1 was not observed for other subtypes (Supplementary Methods 10). This result, together with our reanalysis of shRNA knockdown data from the Achilles project (Extended Data Fig. 8),\",\n",
            "          \"data from the Achilles project (Extended Data Fig. 8), suggest that HNF4\\u03b1 dependency might be particularly associated with the subset of tumors or cells with HNF4A amplification. We also examined the association between the subtype classification and\",\n",
            "          \"examined the association between the subtype classification and clinical features and found only that stage II tumors were significantly enriched in subtype C (multiple-test adjusted p value < 0.05, Supplementary Table 11). Supervised statistical analyses\",\n",
            "          \"0.05, Supplementary Table 11). Supervised statistical analyses at the individual protein level for 13 clinical and genomic features also identified few, if any significant protein effects of these features, except for hypermutation status, MSI status, and\",\n",
            "          \"features, except for hypermutation status, MSI status, and 18q loss (Supplementary Table 12), suggesting that the proteomic subtypes identified by the unsupervised clustering analysis captured the major proteome variations across the tumors. Next, we\",\n",
            "          \"the major proteome variations across the tumors. Next, we compared the proteomic subtype classification with the TCGA transcriptional subtype classification for the 62 samples that had both subtype labels. Proteomic subtypes B and C both showed\",\n",
            "          \"had both subtype labels. Proteomic subtypes B and C both showed significant association with the TCGA subtype \\u201cMSI/CIMP\\u201d (Fig. 4b, Supplementary Table 11); however, they differ considerably at genomic, epigenomic, and proteomic levels (Fig. 4b). We also\",\n",
            "          \"at genomic, epigenomic, and proteomic levels (Fig. 4b). We also examined alternative classifications of the TCGA samples based on two recently published transcriptomic subtype classifiers. Proteomic subtype C, but not subtype B, showed enriched overlap\",\n",
            "          \"Proteomic subtype C, but not subtype B, showed enriched overlap with the \\u201cStem-like\\u201d subtype described by Sadanandam et al. and the \\u201cCCS3\\u201d subtype described by De Sousa et al.. Interestingly, tumors with \\u201cStem-like\\u201d and \\u201cCCS3\\u201d classifications both have\",\n",
            "          \"tumors with \\u201cStem-like\\u201d and \\u201cCCS3\\u201d classifications both have poor prognosis, which suggests that proteome subtype C also may be associated with poor prognosis. Therefore, the ability to distinguish subtype B from C through proteomics data is important,\",\n",
            "          \"subtype B from C through proteomics data is important, because MSI-High tumors typically have better prognosis. Protein signatures and networks associated with proteomic subtypes To better understand the biology underlying the proteomic subtypes, we\",\n",
            "          \"understand the biology underlying the proteomic subtypes, we identified protein signatures for each subtype by supervised comparison of protein abundance in that subtype against all others; we also required signature proteins for a subtype to be\",\n",
            "          \"others; we also required signature proteins for a subtype to be significantly different in abundance compared to normal colon samples from 30 individuals analyzed on the same proteome analysis platform (Supplementary Methods 10, Supplementary Table\",\n",
            "          \"platform (Supplementary Methods 10, Supplementary Table 13\\u201314). As shown in Extended Data Figure 10a, all CRC subtypes displayed more than 2,000 (>60%) significant protein abundance differences compared to normal colon. Although a full validation of the\",\n",
            "          \"compared to normal colon. Although a full validation of the proteomic subtypes and protein signatures for the subtypes will require proteomic profiling data from an independent tumor cohort, a low cross-validation error rate of 3.8% demonstrated good\",\n",
            "          \"a low cross-validation error rate of 3.8% demonstrated good generalizability of the subtypes and their signature proteins (Supplementary Methods 11). We performed Gene Ontology enrichment analysis for the subtype signatures using WebGestalt (Supplementary\",\n",
            "          \"for the subtype signatures using WebGestalt (Supplementary Methods 11, Supplementary Table 15). Genes involved in \\u201cresponse to wounding\\u201d were significantly enriched in the up-signature of subtype C (multiple-test adjusted p value < 2.2 \\u00d7 10-16, Fisher\\u2019s\",\n",
            "          \"C (multiple-test adjusted p value < 2.2 \\u00d7 10-16, Fisher\\u2019s exact test). The wound-response gene signature is a powerful predictor of poor clinical outcome in patients with early stage breast cancers. This result further links our subtype C to poor\",\n",
            "          \"breast cancers. This result further links our subtype C to poor prognosis. To better understand the functional networks underlying this subtype with potential clinical importance, we uploaded the up- and down-signatures of subtype C to NetGestalt for\",\n",
            "          \"the up- and down-signatures of subtype C to NetGestalt for enriched protein-protein interaction network module analysis. Four network modules were enriched with genes in the up-signature for subtype C, whereas two modules were enriched with genes in the\",\n",
            "          \"subtype C, whereas two modules were enriched with genes in the down-signature (multiple-test adjusted p value < 0.01, Fisher\\u2019s exact test, Extended Data Fig. 10b). Notably, the down-signature enriched module (III) included the E-cadherin (CDH1)-\\u03b2-catenin\",\n",
            "          \"enriched module (III) included the E-cadherin (CDH1)-\\u03b2-catenin (CTNNB1)-\\u03b1-catenin (CTNNA1) complex (Extended Data Fig. 10c, 10e). E-cadherin, the most under-expressed protein in the sub-network, suppresses invasion in lobular breast carcinoma and is a\",\n",
            "          \"suppresses invasion in lobular breast carcinoma and is a switch for the epithelial-to-mesenchymal transition (EMT), which is associated with poor prognosis in colon cancer. Other components of the module were desmosomal proteins (PKP2, JUP and DSG2) and\",\n",
            "          \"of the module were desmosomal proteins (PKP2, JUP and DSG2) and cytokeratins (KRT18, KRT6A and KRT8). Reduction in both desmosome formation and cytokeratin expression is associated with EMT. Moreover, proteins in the most significantly up-regulated\",\n",
            "          \"EMT. Moreover, proteins in the most significantly up-regulated network module (Extended Data Fig. 10d, 10f) included collagens (COL1A1 and COL3A1) and extracellular matrix glycoproteins (FN1, BGN, FBN1, and FBN2) that also are markers of EMT. These data\",\n",
            "          \"BGN, FBN1, and FBN2) that also are markers of EMT. These data strengthen the association of subtype C with poor prognosis and relate it to EMT activation. Discussion Our proteomic characterization of the genomically-annotated TCGA colon tumors illustrates\",\n",
            "          \"of the genomically-annotated TCGA colon tumors illustrates the power of integrated proteogenomic analysis. The data demonstrate that protein abundance cannot be reliably predicted from DNA or RNA-level measurements. mRNA and protein levels were modestly\",\n",
            "          \"RNA-level measurements. mRNA and protein levels were modestly correlated, as earlier cell and animal model studies suggest, but over two thirds of these correlations were not statistically significant in the TCGA tumor set. Although most CNAs in CRC drive\",\n",
            "          \"in the TCGA tumor set. Although most CNAs in CRC drive mRNA abundance changes, relatively few translated to consistent changes in protein abundance. Genomic and proteomic technologies provide reinforcing data. RNA-Seq data facilitates the discovery of\",\n",
            "          \"reinforcing data. RNA-Seq data facilitates the discovery of variant proteins, which could serve as possible biomarker candidates or therapeutic targets. Combined mRNA and protein profiling data can identify potentially relevant genes in amplified\",\n",
            "          \"data can identify potentially relevant genes in amplified chromosomal regions. This approach, which revealed the importance of chromosome 20q amplification and provided new insights into the role of HNF4\\u03b1 in CRC, can be broadly extended to understand\",\n",
            "          \"the role of HNF4\\u03b1 in CRC, can be broadly extended to understand roles of CNAs in other cancers. Proteomics identified CRC subtypes similar to those detectable by transcriptome profiles, but further captured features not detectable in transcript profiles.\",\n",
            "          \"captured features not detectable in transcript profiles. The separation of the TCGA \\u201cMSI/CIMP\\u201d subtype into distinct proteotypes illustrates the unique potential of proteomics-based subtyping. After validation in independent cohorts, protein subtype\",\n",
            "          \"After validation in independent cohorts, protein subtype signatures could be directly translated into laboratory tests for tumor classification. Integrated proteogenomic analysis, as demonstrated in this study, will enable new advances in cancer biology,\",\n",
            "          \"in this study, will enable new advances in cancer biology, diagnostics and therapeutics. Methods Summary All tumor samples for the current study were obtained through the TCGA Biospecimen Core Resource (BCR) as described previously. No other selection\",\n",
            "          \"Core Resource (BCR) as described previously. No other selection criteria other than availability were applied for this study. Patient-derived xenograft tumors from established Basal and Luminal-B breast cancer intrinsic subtypes  were raised\",\n",
            "          \"and Luminal-B breast cancer intrinsic subtypes  were raised subcutaneously in 8 week old NOD.Cg-Prkdcscid Il2rgtm1Wjl/SzJ mice (Jackson Labs, Bar Harbor, Maine) as previously described. Normal colon biopsies were obtained from screening colonoscopies\",\n",
            "          \"colon biopsies were obtained from screening colonoscopies performed between July 2006 and October 2010 under Vanderbilt University IRB approval #061096. Tissue proteins were extracted and tryptic peptide digests were analyzed by multidimensional liquid\",\n",
            "          \"peptide digests were analyzed by multidimensional liquid chromatography-tandem mass spectrometry. Xenograft QC samples were run after every 5 colorectal tumor samples. Raw data were processed for peptide identification by database and spectral library\",\n",
            "          \"for peptide identification by database and spectral library searching and identified peptides were assembled as proteins and mapped to gene identifiers for proteogenomic comparisons. Quantitative proteomic comparisons were based on spectral count data.\",\n",
            "          \"proteomic comparisons were based on spectral count data. Detailed descriptions of the samples, LC-MS/MS analysis, and data analysis methods can be found in Supplementary Methods. All of the primary mass spectrometry data on TCGA tumor samples are\",\n",
            "          \"of the primary mass spectrometry data on TCGA tumor samples are deposited at the CPTAC Data Coordinating Center as raw and mzML files and complete protein assembly datasets for public access (https://cptac-data-portal.georgetown.edu). Extended Data Mass\",\n",
            "          \"(https://cptac-data-portal.georgetown.edu). Extended Data Mass spectrometry (MS)-based proteomics workflow Protein was extracted from frozen tumor tissue and used to generate tryptic digests. The resulting tryptic peptides were fractionated using off-line\",\n",
            "          \"The resulting tryptic peptides were fractionated using off-line basic-reverse phase high-pressure liquid chromatography (bRPLC). Collected fractions were pooled and used for reverse phase HPLC in-line with a Thermo Orbitrap-Velos MS instrument. Raw data\",\n",
            "          \"in-line with a Thermo Orbitrap-Velos MS instrument. Raw data was processed by MSConvert and then used for database and spectral library searching using three different search engines (Myrimatch, Pepitome and MS-GF+). Identified peptides were assembled\",\n",
            "          \"Pepitome and MS-GF+). Identified peptides were assembled using IDPicker 3 with selected filters as described in the methods. IDPicker 3 stores its protein assemblies for a specified set of filters in the idpDB format. These SQLite databases associate\",\n",
            "          \"filters in the idpDB format. These SQLite databases associate spectra with peptides, peptides with proteins, and LC-MS/MS experiments with a hierarchy of experiments. Relaxing the false discovery rate (FDR) of peptide-spectrum match (PSM) for\",\n",
            "          \"false discovery rate (FDR) of peptide-spectrum match (PSM) for high-confident proteins increases spectral counts To increase spectral counts and improve statistical comparisons, we first created a protein assembly that maximized the number of proteins\",\n",
            "          \"a protein assembly that maximized the number of proteins identified (at 0.1% PSM FDR) and then relaxed the PSM FDR to 1% exclusively for the set of confidently identified proteins. This strategy led to increased spectral counts from 4,896,831 to\",\n",
            "          \"strategy led to increased spectral counts from 4,896,831 to 6,299,756, a 29% increase. a, Spectral count plot of all 7,526 confidently identified proteins demonstrates the increase in the absolute number of spectra identified for each protein, but no\",\n",
            "          \"absolute number of spectra identified for each protein, but no decrease for any of the proteins. Each dot in the figure represents one of the 7,526 proteins; x-axis and y-axis represent the spectral counts obtained in the data sets with 0.1% and 1% PSM\",\n",
            "          \"spectral counts obtained in the data sets with 0.1% and 1% PSM FDR, respectively, both plotted on a log scale. b, Density plot showing the distribution of PSM FDR scores for all rescued PSMs. Rescued PSMs are of high quality with a median PSM FDR score of\",\n",
            "          \"Rescued PSMs are of high quality with a median PSM FDR score of less than 0.2%, indicating the maintained integrity of the data set. Reads mapping, exon coverage and missense somatic variants in RNA-Seq data a, Summary of total RNA-Seq read counts and\",\n",
            "          \"in RNA-Seq data a, Summary of total RNA-Seq read counts and mapping results for individual samples. b, Distribution of percentage sequence coverage in exons for individual samples. Among all 228,157 exons, 76% were expressed, but only 64% had an average\",\n",
            "          \"228,157 exons, 76% were expressed, but only 64% had an average coverage greater than 1. Exons with no coverage were not included in the box plots. c, Number of missense somatic variants detected by RNA-Seq in individual samples. Approximately 54% of the\",\n",
            "          \"by RNA-Seq in individual samples. Approximately 54% of the mutation positions were covered by RNA-Seq reads and only 43% were covered by three or more reads. PRM (Parallel reaction monitoring) validation results a, PRM data for the variant sequence\",\n",
            "          \"validation results a, PRM data for the variant sequence LVVVGADGVGK (KRAS G12D in TCGA-AA-3818). b, PRM data for the variant sequence LVVVGADGVGK (KRAS G12D in TCGA-AG-A00Y). c, PRM data for the variant sequence TPVLFDVYEIK (ANXA11 I278V in TCGA-AF-3400).\",\n",
            "          \"variant sequence TPVLFDVYEIK (ANXA11 I278V in TCGA-AF-3400). d, PRM data for the variant sequence DLEDLFFK (SRSF9 Y35F in TCGA-AA-A01P). Single amino acid variants (SAAVs) identified in the TCGA shotgun data set were validated using PRM analyses. Three\",\n",
            "          \"TCGA shotgun data set were validated using PRM analyses. Three distinct SAAVs in four TCGA samples were selected for validation. The TCGA samples were freshly prepared in the same manner as the original samples analyzed by shotgun proteomics. Each sample\",\n",
            "          \"original samples analyzed by shotgun proteomics. Each sample was spiked with 12.5 fmol/\\u03bcL of a mixture of all isotopically labeled peptides. Using an inclusion list containing the precursor m/z values representing both unlabeled (endogenous) and labeled\",\n",
            "          \"m/z values representing both unlabeled (endogenous) and labeled peptides, each fraction was analyzed by PRM for the variant peptides. For each variant shown in a\\u2013d, the top MS/MS spectra display represents the spectrum identified in the initial shotgun\",\n",
            "          \"represents the spectrum identified in the initial shotgun analyses of the TCGA samples. The two annotated spectra shown below the original spectra represent the MS/MS of the unlabeled endogenous variant peptide and the spiked respective labeled peptide in\",\n",
            "          \"variant peptide and the spiked respective labeled peptide in the PRM analysis of the TCGA sample, respectively. The chromatographic traces show the overlapping transitions and retention time of both the endogenous and labeled variant peptide,\",\n",
            "          \"time of both the endogenous and labeled variant peptide, respectively. Platform evaluation and analysis method selection using quality control (QC) samples a, The lower-left half (uncolored) depicts pair-wise scatter plots of the samples, with x- and\",\n",
            "          \"depicts pair-wise scatter plots of the samples, with x- and y-axes representing quantile-normalized spectral counts for samples in corresponding columns and rows, respectively. The upper-right half (red colored) depicts pair-wise Spearman\\u2019s correlation\",\n",
            "          \"half (red colored) depicts pair-wise Spearman\\u2019s correlation coefficients for the same comparisons. b, For each normalization method (none, global, quantile, and NSAF), we calculated the intraclass correlation coefficients (ICCs) for individual proteins in\",\n",
            "          \"correlation coefficients (ICCs) for individual proteins in the QC data set. The analysis was done for the top 1000, 500, or 100 proteins with the largest variance and the cumulative fraction curves were plotted. In most scenarios, quantile normalization\",\n",
            "          \"curves were plotted. In most scenarios, quantile normalization generated slightly higher ICC scores than global normalization, and both methods clearly outperformed the NSAF normalization. c, We sorted all proteins in the QC data set based on their total\",\n",
            "          \"We sorted all proteins in the QC data set based on their total spectral counts and then divided the proteins into 10 bins with equal number of proteins. Average spectral count ranges for each bin are shown in the brackets in the legend box. For each bin,\",\n",
            "          \"bin are shown in the brackets in the legend box. For each bin, we calculated the ICCs for individual proteins in the bin. The analysis was done for the top 300, 200, or 100 proteins with the largest variance in each bin. The cumulative fraction curves\",\n",
            "          \"largest variance in each bin. The cumulative fraction curves were plotted. Protein bins with spectral counts less than 1.4 showed clearly lower ICC scores, whereas the ICC score curves started to converge when the average spectral count was greater than\",\n",
            "          \"to converge when the average spectral count was greater than 1.4. Extended data for mRNA-protein correlation analysis a, Evaluation of the length-bias in different RNA-Seq-based gene abundance estimation methods. The plot shows the distribution of\",\n",
            "          \"estimation methods. The plot shows the distribution of correlation between gene length and estimated transcript abundance based on FPKM (Fragments Per Kilobase of exon per Million fragments mapped, blue curve) and RSEM (RNA-Seq Expectation Maximization,\",\n",
            "          \"mapped, blue curve) and RSEM (RNA-Seq Expectation Maximization, red curve), respectively. FPKM measure is independent of gene length, whereas the RSEM measure strongly correlates with gene length. b, Relationship between mRNA-protein correlation and the\",\n",
            "          \"b, Relationship between mRNA-protein correlation and the stability of the molecules. Human genes were separated into four categories based on the mRNA and protein half-lives of their mouse orthologs: stable mRNA/stable protein; stable mRNA/unstable\",\n",
            "          \"orthologs: stable mRNA/stable protein; stable mRNA/unstable protein, unstable mRNA/stable protein, and unstable mRNA/unstable protein. Distribution of mRNA-protein correlations for genes in each category was plotted in the box plots. Genes with stable\",\n",
            "          \"each category was plotted in the box plots. Genes with stable mRNA and stable protein showed relatively higher mRNA-protein correlation whereas those with unstable mRNA and unstable protein showed relatively lower mRNA-protein correlation. Only common\",\n",
            "          \"showed relatively lower mRNA-protein correlation. Only common genes in both our study and the mouse study were included in the analysis. The total number of genes in each category (N) is labeled in the figure. The p value indicating correlation difference\",\n",
            "          \"in the figure. The p value indicating correlation difference among the four categories was calculated based on the Kruskal-Wallis non-parametric ANOVA test. The p value indicating correlation difference between the stable mRNA/stable protein group and the\",\n",
            "          \"difference between the stable mRNA/stable protein group and the unstable mRNA/unstable protein group was calculated based on the two-sided Wilcoxon rank-sum test. mRNA and protein-level cis-effect of copy number alterations (CNAs) in focal amplification,\",\n",
            "          \"of copy number alterations (CNAs) in focal amplification, focal deletion and non-focal regions The figure plots cumulative fraction curves of CNA-mRNA (dashed lines) and CNA-protein (solid lines) expression correlations for genes in the focal\",\n",
            "          \"(solid lines) expression correlations for genes in the focal amplification regions (red), focal deletion regions (green), and non-focal regions (blue), respectively. Focal amplification regions were defined in the TCGA study. Any chromosomal regions\",\n",
            "          \"regions were defined in the TCGA study. Any chromosomal regions outside the focal amplification and deletion regions were considered as non-focal regions. CNA-mRNA correlations were significantly higher than CNA-protein correlations for genes in any of\",\n",
            "          \"higher than CNA-protein correlations for genes in any of the three groups. Moreover, genes in the focal amplification regions showed the highest level of CNA-mRNA and CNA-protein correlations among the three groups of genes. P values were based on the\",\n",
            "          \"among the three groups of genes. P values were based on the two-sided kolmogorov-smirnov test. HNF4\\u03b1 isoforms and the effect of HNF4A shRNA on the proliferation of colon cancer cells a, Multiple sequence alignment of the HNF4\\u03b1 isoforms, with peptides\",\n",
            "          \"sequence alignment of the HNF4\\u03b1 isoforms, with peptides detected by shotgun proteomics and sequences corresponding to the shRNA target sequences highlighted. Different colors of the letters indicate different levels of sequence coverage in the shotgun\",\n",
            "          \"indicate different levels of sequence coverage in the shotgun proteomics study, as indicated by the color scale bar. Yellow boxes highlight sequences corresponding to the shRNA target sequences. TRCN0000019193 specifically targets P1 promoter-driven\",\n",
            "          \"TRCN0000019193 specifically targets P1 promoter-driven isoforms, whereas the other two target both types of isoforms. b\\u2013d, The P1- HNF4\\u03b1 specific shRNA showed mixed impacts (b), whereas shRNAs simultaneously targeting both P1- and P2- HNF4\\u03b1 showed a\",\n",
            "          \"shRNAs simultaneously targeting both P1- and P2- HNF4\\u03b1 showed a primarily negative impact on cell proliferation (c,d). Moreover, a stronger negative impact was associated with increased copy number, both for the P1- HNF4\\u03b1 specific shRNA (p=0.04,\",\n",
            "          \"copy number, both for the P1- HNF4\\u03b1 specific shRNA (p=0.04, Spearman\\u2019s correlation [r]) and for all shRNAs (p=0.01, Spearman\\u2019s correlation p-values for individual shRNAs summarized by the Fisher\\u2019s combined probability test). Consensus matrices, the\",\n",
            "          \"Fisher\\u2019s combined probability test). Consensus matrices, the empirical cumulative distribution function (CDF) plot and core sample identification a, Consensus matrices of the 90 CRC samples for k = 2 to k = 8. The consensus matrices show the robustness of\",\n",
            "          \"k = 2 to k = 8. The consensus matrices show the robustness of the discovered clusters to sampling variability (resampling 80% samples) for cluster numbers k = 2 to 8. In each consensus matrix, both the rows and the columns were indexed with the same\",\n",
            "          \"both the rows and the columns were indexed with the same sample order and samples belonging to the same cluster frequently are adjacent to each other. For each pair of samples, a consensus index, which is the percentage of times they belong to the same\",\n",
            "          \"index, which is the percentage of times they belong to the same cluster during 1,000 runs of the clustering algorithm based on resampling was calculated. The consensus index for each pair of samples was represented by color gradient from white (0%) to red\",\n",
            "          \"was represented by color gradient from white (0%) to red (100%) in the consensus matrix. b, CDF plots corresponding to the consensus matrices for k = 2 to k = 8. This plot shows the cumulative distribution of the entries of the consensus matrices within\",\n",
            "          \"distribution of the entries of the consensus matrices within the 0\\u20131 range. Skew toward 0 and 1 indicates good clustering. As k increases, the area under the CDF is hypothesized to increase markedly until k reaches the ktrue. In this case, 7 was\",\n",
            "          \"markedly until k reaches the ktrue. In this case, 7 was considered as ktrue because the change of the area under the CDF was close to zero when k increased from 7 to 8. c, Silhouette plot for core sample identification. For each sample (y-axis), the\",\n",
            "          \"for core sample identification. For each sample (y-axis), the silhouette width (x-axis) compares its similarity to its assigned class and to any other classes. Samples with higher similarity to their assigned class than to any other classes will get\",\n",
            "          \"to their assigned class than to any other classes will get positive silhouette width score and be selected as core samples. Network analysis of the subtype signature proteins a, The number of signature proteins for each subtype. For a given subtype, the\",\n",
            "          \"signature proteins for each subtype. For a given subtype, the red circle represents proteins that were different in abundance between the subtype and all other subtypes, the green circle represents proteins that were different in abundance between the\",\n",
            "          \"proteins that were different in abundance between the subtype and normal colon tissues. The intersection between red and green circles contains the signature proteins for each subtype. b, Visualizing subtype C signature proteins in NetGestalt. Proteins in\",\n",
            "          \"subtype C signature proteins in NetGestalt. Proteins in the iRef protein-protein interaction network are placed in a linear order together with the hierarchical modular organization of the network. Alternating bar colors (green and orange) are used to\",\n",
            "          \"network. Alternating bar colors (green and orange) are used to distinguish neighboring modules. Proteins in the up- and down-signatures of subtype C were visualized as two separate tracks below the network modules, where each bar represents a protein.\",\n",
            "          \"below the network modules, where each bar represents a protein. These proteins are not randomly distributed in the network. Highlighted by red or blue arrows are four Network modules (I, IV, V, VI) significantly enriched with up-signature proteins and two\",\n",
            "          \"VI) significantly enriched with up-signature proteins and two modules (II and III) significantly enriched with down-signature proteins (adjusted p value < 0.01). c\\u2013d, Heat maps depicting relative abundance of down- and up-signature proteins of subtype C\",\n",
            "          \"abundance of down- and up-signature proteins of subtype C in modules III and I, respectively. Tumors are displayed as rows, grouped by normal controls (N) and proteomic subtypes (A\\u2013E) as indicated by different side bar colors. Proteins are displayed as\",\n",
            "          \"by different side bar colors. Proteins are displayed as columns. e\\u2013f, Network diagrams depicting the interaction of down- and up-signature proteins of subtype C in modules III and I, respectively. Node and node-border colors represent relatively higher or\",\n",
            "          \"Node and node-border colors represent relatively higher or lower abundance in the subtype compared to other subtypes and normal colon tissues, respectively. Red and blue in the heat maps and the network diagrams represent relatively higher or lower\",\n",
            "          \"and the network diagrams represent relatively higher or lower abundance, respectively. Supplementary Material Supplementary Information is linked to the online version of this paper at www.nature.com/nature. Author Contributions B.Z., R.J.C.S., D.L.T.,\",\n",
            "          \"Author Contributions B.Z., R.J.C.S., D.L.T., L.J.Z. and D.C.L. designed the proteomic analysis experiments, data analysis workflow, and proteomic-genomic data comparisons. K.F.S., L.J.Z., R.J.C.S. and D.C.L. directed and performed proteomic analysis of\",\n",
            "          \"and D.C.L. directed and performed proteomic analysis of colon tumor and quality control samples. J.W., X.W., J.Z., Q.L., Z.S., P.W., S.W., R.J.C.S. and B.Z. performed proteomic-genomic data analyses. M.C.C., S.K., R.J.C.S. and D.L.T. performed analyses of\",\n",
            "          \"M.C.C., S.K., R.J.C.S. and D.L.T. performed analyses of mass spectrometry data and adapted algorithms and software for data analysis. S.R.D., R.R.T and M.J.C.E. developed and prepared breast xenografts used as quality control samples. S.A.C, K.F.S. and\",\n",
            "          \"xenografts used as quality control samples. S.A.C, K.F.S. and D.C.L. designed strategy for quality control analyses. R.J.C.S., C.R.K, R.C.R, and H.R. coordinated acquisition, distribution and quality control evaluation of TCGA tumor samples. B.Z., J.W.,\",\n",
            "          \"quality control evaluation of TCGA tumor samples. B.Z., J.W., R.J.C.S., R.J.C. and D.C.L. interpreted data in context of colon cancer biology. B.Z., R.J.C.S. and D.C.L. wrote the manuscript. All of the primary mass spectrometry data on TCGA tumor samples\",\n",
            "          \"All of the primary mass spectrometry data on TCGA tumor samples are deposited at the CPTAC Data Coordinating Center as raw and mzML files for public access (https://cptac-data-portal.georgetown.edu). The authors declare no competing financial interests.\",\n",
            "          \"The authors declare no competing financial interests. Reprints and permissions information is available at www.nature.com/reprints. Readers are welcome to comment on the online version of this article at www.nature.com/nature. Summary of detected single\",\n",
            "          \"article at www.nature.com/nature. Summary of detected single amino acid variants (SAAVs) and the impact of single nucleotide variants (SNVs) on protein abundance a, The number of different types of SAAVs (TCGA-reported somatic variants, COSMIC-supported\",\n",
            "          \"of SAAVs (TCGA-reported somatic variants, COSMIC-supported variants, dbSNP-supported variants, and new variants) in individual tumor samples. The samples are ordered by the number of detected somatic variants, then COSMIC-supported variants, and then\",\n",
            "          \"somatic variants, then COSMIC-supported variants, and then dbSNP-supported variants. The Microsatellite instability (MSI) and hypermutation (Hyper) status are labeled below the bar charts for each sample (MSI-High: red, MSI-Low: orange, Microsatellite\",\n",
            "          \"for each sample (MSI-High: red, MSI-Low: orange, Microsatellite Stable: yellow; hypermutated: blue, non-hypermutated: sky blue; no data: grey). The number of somatic variants and COSMIC-supported variants were significantly higher in MSI-High and\",\n",
            "          \"variants were significantly higher in MSI-High and hypermutated tumors, whereas the other two types of SAAVs were randomly distributed across the data set. b, The total numbers for different types of SAAVs and their overlapping relations. All 796 detected\",\n",
            "          \"of SAAVs and their overlapping relations. All 796 detected SAAVs were annotated based on previous reports in dbSNP (left circle), COSMIC (middle circle), or TCGA-reported somatic variants (right circle), and their overlapping relations are shown in the\",\n",
            "          \"circle), and their overlapping relations are shown in the Venn diagram. There are 162 SAAVs that have not been reported previously in these databases (new). c, Distribution of the frequency of occurrence (1 sample: light grey, 2\\u20139 samples: grey, >=10\",\n",
            "          \"of occurrence (1 sample: light grey, 2\\u20139 samples: grey, >=10 samples: dark grey) for different types of SAAVs. Border colors of the pie charts correspond to different SAAV types using the same color scheme as in (a). Whereas 58% of dbSNP-supported\",\n",
            "          \"the same color scheme as in (a). Whereas 58% of dbSNP-supported variants occurred in two or more samples, almost all somatic variants each occurred in only one sample. d, SNVs detected in RNA-Seq data were separated into three categories (dbSNP-supported,\",\n",
            "          \"data were separated into three categories (dbSNP-supported, COSMIC-supported, and TCGA-Somatic). The impact of individual SNVs on protein abundance was calculated (see supplementary methods) and the impact scores for different categories of SNVs were\",\n",
            "          \"and the impact scores for different categories of SNVs were plotted as cumulative fraction curves with two-sided p values from the Kolmogorov-Smirnov test labeled. The percentage of SNVs with an absolute impact score greater than 2 was also plotted as an\",\n",
            "          \"an absolute impact score greater than 2 was also plotted as an inset, with p values from the Chi-squared test. Sample size for the dbSNP-supported, COSMIC-supported and TCGA-Somatic variants were 12184, 7492, and 3302, respectively. Correlations between\",\n",
            "          \"were 12184, 7492, and 3302, respectively. Correlations between mRNA and protein abundance in TCGA tumors a, Steady state mRNA and protein abundance were positively correlated in all 86 samples (multiple-test adjusted p value < 0.01) with a mean Spearman\\u2019s\",\n",
            "          \"(multiple-test adjusted p value < 0.01) with a mean Spearman\\u2019s correlation coefficient of 0.47. b, mRNA and protein variation were positively correlated for most (89.4%) mRNA-protein pairs across the 87 samples, but only 32% showed significant correlation\",\n",
            "          \"the 87 samples, but only 32% showed significant correlation (multiple-test adjusted p value < 0.01), with a mean Spearman\\u2019s correlation coefficient of 0.23. c, mRNA and protein levels displayed dramatically different correlation for genes involved in\",\n",
            "          \"dramatically different correlation for genes involved in different biological processes. Genes encoding intermediary metabolism functions showed high mRNA-protein correlations, whereas genes involved in oxidative phosphorylation, RNA splicing and ribosome\",\n",
            "          \"in oxidative phosphorylation, RNA splicing and ribosome components showed low or negative correlations. Multiple-test adjusted two-sided p-values from the Kolmogorov-Smirnov test were provided in the parentheses following the KEGG pathway names. Red and\",\n",
            "          \"in the parentheses following the KEGG pathway names. Red and green in the figures indicate positive- and negative-correlations, respectively. Effects of copy number alterations (CNAs) on mRNA and protein abundance a,b, The top panels show copy\",\n",
            "          \"on mRNA and protein abundance a,b, The top panels show copy number-abundance correlation matrices for mRNA abundance (a) and protein abundance (b) with significant positive and negative correlations (multiple-test adjusted p value < 0.01, Spearman\\u2019s\",\n",
            "          \"correlations (multiple-test adjusted p value < 0.01, Spearman\\u2019s correlation coefficient) indicated by red and green colors, respectively, and genes ordered by chromosomal location on both x and y-axes. The bottom panels show the frequency of\",\n",
            "          \"on both x and y-axes. The bottom panels show the frequency of mRNAs/proteins associated with a particular copy number alteration, where blue and black bars represent associations specific to mRNA/protein or common to both mRNA and protein, respectively.\",\n",
            "          \"mRNA/protein or common to both mRNA and protein, respectively. c\\u2013e,\",\n",
            "          \"HNF4A, TOMM34 and SRC showed significant CNA-mRNA, mRNA-protein, and CNA-protein correlations (Spearman\\u2019s correlation coefficient). The color grade from light yellow to red indicates relatively low-level to high-level CNA, relative mRNA abundance or\",\n",
            "          \"low-level to high-level CNA, relative mRNA abundance or relative protein abundance among the 85 samples, which were ordered by copy number data. Proteomic subtypes of colon and rectal cancers, associated genomic features, and relative abundance of HNF4\\u03b1\",\n",
            "          \"associated genomic features, and relative abundance of HNF4\\u03b1 a, Figure legends for b, c and d. b, Identification of five proteomic subtypes. Tumors are displayed as columns, grouped by proteomic subtypes as indicated by different colors. Proteins used for\",\n",
            "          \"subtypes as indicated by different colors. Proteins used for the subtype classification are displayed as rows. The heat map presents relative abundance of the proteins (logarithmic scale in base 2) in the 90 tumor cohort. c, Association of proteomic\",\n",
            "          \"in base 2) in the 90 tumor cohort. c, Association of proteomic subtypes with major colorectal cancer-associated genomic alterations and previously published transcriptomic and methylation subtypes. Subtypes significantly overlapped with a transcriptomic\",\n",
            "          \"Subtypes significantly overlapped with a transcriptomic or methylation subtype are highlighted by pink boxes. Both proteomic subtypes B and C showed significant overlap with the TCGA MSI/CIMP subtype. In addition, they showed significant overlap with the\",\n",
            "          \"subtype. In addition, they showed significant overlap with the CCS2 and CCS3 subtypes in the De Sousa et al. classification, respectively. Proteomic subtype B significantly overlapped with the TCGA CIMP-H methylation subtype, whereas subtype C\",\n",
            "          \"with the TCGA CIMP-H methylation subtype, whereas subtype C significantly overlapped with a non-methylation subtype (TCGA cluster 4 methylation subtype). Subtypes over-represented with a specific genomic alteration are also highlighted by pink boxes. The\",\n",
            "          \"genomic alteration are also highlighted by pink boxes. The green box highlights the absence of TP53 mutations and 18q loss in subtype B. d, The top panel shows HNF4A copy number and relative abundance of HNF4\\u03b1 in the five subtypes; the bottom panel\",\n",
            "          \"abundance of HNF4\\u03b1 in the five subtypes; the bottom panel compares relative abundance of HNF4\\u03b1 in the five subtypes to that in normal colon samples, respectively, and the adjP values are based on the two-sided Wilcoxon rank-sum test followed by\",\n",
            "          \"are based on the two-sided Wilcoxon rank-sum test followed by multiple-test adjustment.\"\n",
            "        ]\n",
            "      }\n",
            "    ]\n",
            "  },\n",
            "  \"PDC000289\": {\n",
            "    \"study_id\": \"5d019771-c39c-4dec-b2a9-f0aec54a796f\",\n",
            "    \"pdc_study_id\": \"PDC000289\",\n",
            "    \"study_submitter_id\": \"TCGA Colorectal Cancer CompRef Samples - Proteome\",\n",
            "    \"program_id\": \"10251935-5540-11e8-b664-00a098d917f8\",\n",
            "    \"project_id\": \"c96eca75-8b73-4018-9bf7-b20e31a965cd\",\n",
            "    \"study_name\": \"TCGA Colorectal Cancer CompRef Samples - Proteome\",\n",
            "    \"program_name\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"project_name\": \"CPTAC2 Retrospective\",\n",
            "    \"disease_type\": \"Other\",\n",
            "    \"primary_site\": \"Not Reported\",\n",
            "    \"analytical_fraction\": \"Proteome\",\n",
            "    \"experiment_type\": \"Label Free\",\n",
            "    \"embargo_date\": null,\n",
            "    \"cases_count\": 2,\n",
            "    \"aliquots_count\": 8,\n",
            "    \"protocol_id\": \"26719c58-7efe-42e5-bb7b-c3667f231762\",\n",
            "    \"protocol_submitter_id\": \"TCGA Colorectal Cancer CompRef Proteome\",\n",
            "    \"program_submitter_id\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"protocol_name\": \"TCGA Colorectal Cancer CompRef Proteome\",\n",
            "    \"protocol_date\": \"2021-04-07\"\n",
            "  },\n",
            "  \"PDC000292\": {\n",
            "    \"study_id\": \"fef54f2c-f792-455c-b5d2-41f7cb0b071c\",\n",
            "    \"pdc_study_id\": \"PDC000292\",\n",
            "    \"study_submitter_id\": \"TCGA Ovarian CompRef Samples JHU Proteome\",\n",
            "    \"program_id\": \"10251935-5540-11e8-b664-00a098d917f8\",\n",
            "    \"project_id\": \"c96eca75-8b73-4018-9bf7-b20e31a965cd\",\n",
            "    \"study_name\": \"TCGA Ovarian CompRef Samples JHU Proteome\",\n",
            "    \"program_name\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"project_name\": \"CPTAC2 Retrospective\",\n",
            "    \"disease_type\": \"Other\",\n",
            "    \"primary_site\": \"Not Reported\",\n",
            "    \"analytical_fraction\": \"Proteome\",\n",
            "    \"experiment_type\": \"iTRAQ4\",\n",
            "    \"embargo_date\": null,\n",
            "    \"cases_count\": 2,\n",
            "    \"aliquots_count\": 2,\n",
            "    \"protocol_id\": \"5f78dd60-a969-4e41-93fd-a87a2bf597ce\",\n",
            "    \"protocol_submitter_id\": \"TCGA Ovarian CompRef JHU Proteome\",\n",
            "    \"program_submitter_id\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"protocol_name\": \"TCGA Ovarian CompRef JHU Proteome\",\n",
            "    \"protocol_date\": \"2021-04-08\"\n",
            "  },\n",
            "  \"PDC000294\": {\n",
            "    \"study_id\": \"d865ed3e-1057-4f31-b018-85d87db9dcce\",\n",
            "    \"pdc_study_id\": \"PDC000294\",\n",
            "    \"study_submitter_id\": \"TCGA Ovarian CompRef Samples PNNL Phosphoproteome Velos Qexactive\",\n",
            "    \"program_id\": \"10251935-5540-11e8-b664-00a098d917f8\",\n",
            "    \"project_id\": \"c96eca75-8b73-4018-9bf7-b20e31a965cd\",\n",
            "    \"study_name\": \"TCGA Ovarian CompRef Samples PNNL Phosphoproteome Velos Qexactive\",\n",
            "    \"program_name\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"project_name\": \"CPTAC2 Retrospective\",\n",
            "    \"disease_type\": \"Other\",\n",
            "    \"primary_site\": \"Not Reported\",\n",
            "    \"analytical_fraction\": \"Phosphoproteome\",\n",
            "    \"experiment_type\": \"iTRAQ4\",\n",
            "    \"embargo_date\": null,\n",
            "    \"cases_count\": 2,\n",
            "    \"aliquots_count\": 2,\n",
            "    \"protocol_id\": \"12453f97-3f53-4bb5-9258-d34909d16c77\",\n",
            "    \"protocol_submitter_id\": \"TCGA Ovarian CompRef PNNL Phosphoproteome Velos\",\n",
            "    \"program_submitter_id\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"protocol_name\": \"TCGA Ovarian CompRef PNNL Phosphoproteome Velos\",\n",
            "    \"protocol_date\": \"2021-04-08\"\n",
            "  },\n",
            "  \"PDC000293\": {\n",
            "    \"study_id\": \"6fd7959e-bd78-4e42-b59f-0cfca9bef779\",\n",
            "    \"pdc_study_id\": \"PDC000293\",\n",
            "    \"study_submitter_id\": \"TCGA Ovarian CompRef Samples PNNL Proteome\",\n",
            "    \"program_id\": \"10251935-5540-11e8-b664-00a098d917f8\",\n",
            "    \"project_id\": \"c96eca75-8b73-4018-9bf7-b20e31a965cd\",\n",
            "    \"study_name\": \"TCGA Ovarian CompRef Samples PNNL Proteome\",\n",
            "    \"program_name\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"project_name\": \"CPTAC2 Retrospective\",\n",
            "    \"disease_type\": \"Other\",\n",
            "    \"primary_site\": \"Not Reported\",\n",
            "    \"analytical_fraction\": \"Proteome\",\n",
            "    \"experiment_type\": \"iTRAQ4\",\n",
            "    \"embargo_date\": null,\n",
            "    \"cases_count\": 2,\n",
            "    \"aliquots_count\": 2,\n",
            "    \"protocol_id\": \"236f18e1-b6c5-43af-841c-285358764ac3\",\n",
            "    \"protocol_submitter_id\": \"TCGA Ovarian CompRef PNNL Proteome\",\n",
            "    \"program_submitter_id\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"protocol_name\": \"TCGA Ovarian CompRef PNNL Proteome\",\n",
            "    \"protocol_date\": \"2021-04-08\"\n",
            "  },\n",
            "  \"PDC000112\": {\n",
            "    \"study_id\": \"b9f2ccc5-57b8-11e8-b07a-00a098d917f8\",\n",
            "    \"pdc_study_id\": \"PDC000112\",\n",
            "    \"study_submitter_id\": \"TCGA OV Glycoproteome S020-1\",\n",
            "    \"program_id\": \"10251935-5540-11e8-b664-00a098d917f8\",\n",
            "    \"project_id\": \"48af5040-5546-11e8-b664-00a098d917f8\",\n",
            "    \"study_name\": \"TCGA Ovarian JHU Glycoproteome\",\n",
            "    \"program_name\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"project_name\": \"CPTAC2 Retrospective\",\n",
            "    \"disease_type\": \"Other;Ovarian Serous Cystadenocarcinoma\",\n",
            "    \"primary_site\": \"Not Reported;Ovary\",\n",
            "    \"analytical_fraction\": \"Glycoproteome\",\n",
            "    \"experiment_type\": \"iTRAQ4\",\n",
            "    \"embargo_date\": null,\n",
            "    \"cases_count\": 124,\n",
            "    \"aliquots_count\": 124,\n",
            "    \"protocol_id\": \"1c06b658-3b26-11e9-9a07-0a80fada099c\",\n",
            "    \"protocol_submitter_id\": \"P-S020-1\",\n",
            "    \"program_submitter_id\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"protocol_name\": \"TCGA_Ovarian_JHU_Glycoproteome\",\n",
            "    \"protocol_date\": \"2014-04-01\",\n",
            "    \"publications\": [\n",
            "      {\n",
            "        \"publication_id\": \"44e2e658-89c7-11e8-bcf1-0a2705229b82\",\n",
            "        \"pubmed_id\": \"27372738\",\n",
            "        \"doi\": \"10.1016/j.cell.2016.05.069\",\n",
            "        \"title\": \"Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer\",\n",
            "        \"journal\": \"Cell\",\n",
            "        \"journal_url\": \"https://www.cell.com/cell/fulltext/S0092-8674(16)30673-0?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0092867416306730%3Fshowall%3Dtrue\",\n",
            "        \"year\": \"2016\",\n",
            "        \"abstract\": \"To provide a detailed analysis of the molecular components and underlying mechanisms associated with ovarian cancer, we performed a comprehensive mass-spectrometry-based proteomic characterization of 174 ovarian tumors previously analyzed by The Cancer Genome Atlas (TCGA), of which 169 were high-grade serous carcinomas (HGSCs). Integrating our proteomic measurements with the genomic data yielded a number of insights into disease, such as how different copy-number alternations influence the proteome, the proteins associated with chromosomal instability, the sets of signaling pathways that diverse genome rearrangements converge on, and the ones most associated with short overall survival. Specific protein acetylations associated with homologous recombination deficiency suggest a potential means for stratifying patients for therapy. In addition to providing a valuable resource, these findings provide a view of how the somatic genome drives the cancer proteome and associations between protein and post-translational modification levels and clinical outcomes in HGSC.\",\n",
            "        \"full_text\": [\n",
            "          \"Integrated proteogenomic characterization of human high grade serous ovarian cancer SUMMARY To provide a detailed analysis of the molecular components and underlying mechanisms associated with ovarian cancer, we performed a comprehensive mass\",\n",
            "          \"with ovarian cancer, we performed a comprehensive mass spectrometry-based proteomic characterization of 174 ovarian tumors previously analyzed by The Cancer Genome Atlas (TCGA), of which 169 were high-grade serous carcinomas (HGSC). Integrating our\",\n",
            "          \"169 were high-grade serous carcinomas (HGSC). Integrating our proteomic measurements with the genomic data yielded a number of insights into disease such as how different copy number alternations influence the proteome, the proteins associated with\",\n",
            "          \"influence the proteome, the proteins associated with chromosomal instability, the sets of signaling pathways that diverse genome rearrangements converge on, as well as the ones most associated with short overall survival. Specific protein acetylations\",\n",
            "          \"with short overall survival. Specific protein acetylations associated with homologous recombination deficiency suggest a potential means for stratifying patients for therapy. In addition to providing a valuable resource, these findings provide a view of\",\n",
            "          \"providing a valuable resource, these findings provide a view of how the somatic genome drives the cancer proteome and associations between protein and post-translational modification levels and clinical outcomes in HGSC. INTRODUCTION A comprehensive\",\n",
            "          \"and clinical outcomes in HGSC. INTRODUCTION A comprehensive molecular view of cancer is necessary for understanding the underlying mechanisms of disease, improving prognosis, and ultimately guiding treatment. The Cancer Genome Atlas (TCGA) conducted an\",\n",
            "          \"guiding treatment. The Cancer Genome Atlas (TCGA) conducted an extensive genomic and transcriptomic characterization of ovarian high-grade serous carcinoma (HGSC) aimed at defining the genomic landscape and aiding the development of targeted therapies for\",\n",
            "          \"landscape and aiding the development of targeted therapies for this highly lethal malignancy. Key findings from TCGA were: 1) the prevalent role of TP53 mutations, 2) extensive DNA copy alterations, 3) preliminary transcriptional signatures associated\",\n",
            "          \"3) preliminary transcriptional signatures associated with survival, 4) varied mechanisms of BRCA1/2 inactivation, and lastly, 5) CCNE1 aberrations. Subsequent analysis of genomic data from the TCGA consortium led to the refinement of the\",\n",
            "          \"data from the TCGA consortium led to the refinement of the transcript-defined signatures, improving the statistical association with patient outcome, and integrating microRNA and mRNA expression profiles associated with HGSC to identify candidate microRNA\",\n",
            "          \"profiles associated with HGSC to identify candidate microRNA targets. While the insights from genomic analyses are substantial, functions encoded in the genome are generally executed at the protein level, and are often further modulated by\",\n",
            "          \"at the protein level, and are often further modulated by post-translational modifications (PTMs). For example, TCGA used reverse phase protein array (RPPA) analysis of 172 proteins (including 31 phosphoproteins with phospho-specific antibodies) to\",\n",
            "          \"31 phosphoproteins with phospho-specific antibodies) to generate a signature associated with the risk of tumor recurrence. To obtain a more comprehensive assessment of the complex ovarian HGSC phenotype, the Clinical Proteomic Tumor Analysis Consortium\",\n",
            "          \"phenotype, the Clinical Proteomic Tumor Analysis Consortium (CPTAC) conducted an extensive mass spectrometry (MS)-based proteomic and phosphoproteomic characterization of HGSC tumors characterized by TCGA, providing quantitative measurements for a\",\n",
            "          \"by TCGA, providing quantitative measurements for a combined total of 9,600 proteins from 174 tumors (an average of 7,952 proteins per tumor), and a total of 24,429 phosphosites from 6,769 phosphoproteins in a subset of 69 tumors (an average of 7,677\",\n",
            "          \"phosphoproteins in a subset of 69 tumors (an average of 7,677 phosphosites per tumor). Our results provide insights on HGSC biology and correlate differences in protein and PTM levels with clinical outcome complementary to TCGA genomic analyses. RESULTS\",\n",
            "          \"outcome complementary to TCGA genomic analyses. RESULTS Proteomic analysis of TCGA HGSC samples HGSC biospecimens and clinical data from 174 patients collected by TCGA were analyzed at two independent CPTAC Centers, Johns Hopkins University (JHU) and\",\n",
            "          \"independent CPTAC Centers, Johns Hopkins University (JHU) and Pacific Northwest National Laboratory (PNNL); thirty-two samples were analyzed at both JHU and PNNL. Tumors were selected by examining the associated TCGA metadata to select tumors either 1) on\",\n",
            "          \"the associated TCGA metadata to select tumors either 1) on the basis of putative homologous recombination deficiency (HRD), defined by the presence of germline or somatic BRCA1 or BRCA2 mutations, BRCA1 promoter methylation, or homozygous deletion of PTEN\",\n",
            "          \"BRCA1 promoter methylation, or homozygous deletion of PTEN (122 samples; 67 classified as HRD, 55 as non-HRD by the above criteria; JHU), or 2) to maximize differences in overall survival (84 samples; PNNL). For selection purposes, short survival was\",\n",
            "          \"(84 samples; PNNL). For selection purposes, short survival was defined as overall survival of less than 3 years, and long survival as greater than 5 years. All but five tumors had somatic TP53 mutations, a characteristic feature of HGSC; these five tumors\",\n",
            "          \"mutations, a characteristic feature of HGSC; these five tumors were subsequently reclassified as other than HGSC, and removed from protein functional analyses (e.g., subtyping and survival analyses). The tumor selection criteria and the associated\",\n",
            "          \"analyses). The tumor selection criteria and the associated metadata are provided in the Supplemental Information Table S1, available online. Proteomics measurements used isobaric tags for relative and absolute quantitation (iTRAQ) in conjunction with\",\n",
            "          \"relative and absolute quantitation (iTRAQ) in conjunction with offline liquid chromatography fractionation via high-pH reversed phase liquid chromatography (RPLC) and online RPLC with high-resolution tandem MS to provide broad coverage for peptide and\",\n",
            "          \"tandem MS to provide broad coverage for peptide and protein identification and quantification (Supplemental Information); this also alleviated quantitative interference potentially associated with the use of isobaric tags. We used the relative abundance\",\n",
            "          \"with the use of isobaric tags. We used the relative abundance measurements for each protein in the 32 patient samples analyzed at both JHU and PNNL to normalize across the two analysis sites, and then used clustering, principle component analysis (PCA),\",\n",
            "          \"and then used clustering, principle component analysis (PCA), and statistical tests to identify any significant batch effects associated with the site of analysis (a detailed comparison of within site, between site, between sample measurement variability\",\n",
            "          \"site, between site, between sample measurement variability and the process used to merge the JHU and PNNL data is given in Supplemental Information, Figure S1). As shown in Figure S1C, the median coefficient of variation (CV) between measurements at the\",\n",
            "          \"coefficient of variation (CV) between measurements at the two sites was 16%. A total of 9,600 proteins were identified with high confidence in all tumors, and the relative abundances in each tumor are given in Table S2. Functional analyses and\",\n",
            "          \"in each tumor are given in Table S2. Functional analyses and proteome-transcriptome associations were restricted to 3,586 proteins observed and quantified in all 169 HGSC samples used for protein functional analyses and where sample variability (signal)\",\n",
            "          \"functional analyses and where sample variability (signal) exceeded technical variability (noise) in the merged data (Table S2), calculated as described in Supplemental Information. On average we identify peptides covering 29% of the amino acids in any of\",\n",
            "          \"we identify peptides covering 29% of the amino acids in any of these 3,586 proteins in a given sample, with a range from 10% to 47%. In addition to protein abundance levels, phosphoproteomics data were acquired for 69 tumors with sufficient sample (Table\",\n",
            "          \"data were acquired for 69 tumors with sufficient sample (Table S2). As with proteins, phosphopeptide abundances were calculated relative to the pooled reference sample. Because isobaric labeling was performed prior to splitting the samples for proteome\",\n",
            "          \"was performed prior to splitting the samples for proteome and phosphoproteome analyses, phosphopeptide abundance could be corrected for changes in parent protein abundance to identify differences in the relative extent of phosphorylation at specific sites\",\n",
            "          \"in the relative extent of phosphorylation at specific sites for each protein (Table S2). Overall, we achieved a protein dynamic range encompassing >4 orders of magnitude, ranging from low-level transcription factors to abundant structural proteins i.e.,\",\n",
            "          \"transcription factors to abundant structural proteins i.e., actin and tubulin. Proteogenomic landscape of HGSC The degree to which alterations observed at the genome and transcriptome levels are manifested at the protein level is variable, both\",\n",
            "          \"levels are manifested at the protein level is variable, both qualitatively and quantitatively, and partially driven by multiple levels of post-transcriptional regulation. To identify peptides encoded by single amino acid variants (SAAVs), splice variants,\",\n",
            "          \"encoded by single amino acid variants (SAAVs), splice variants, and novel exons documented by TCGA, mass spectra were searched against a custom graph database containing all peptide variations projected from the TCGA genomic analyses of the cohort using a\",\n",
            "          \"projected from the TCGA genomic analyses of the cohort using a multi-stage analysis pipeline (Supplemental Information). The most frequently observed variant peptides represented SAAVs and gene-level events. The evidence supporting each novel event is\",\n",
            "          \"gene-level events. The evidence supporting each novel event is detailed in Table S2, including event type, genomic location, abundance information, and estimated false discovery rate (FDR). The validity of these variant peptides was further evaluated by\",\n",
            "          \"The validity of these variant peptides was further evaluated by the synthesis of twenty examples selected at random across the entire range of spectrum match scores. We obtained tandem mass spectra for all twenty synthetic peptides that matched spectra\",\n",
            "          \"spectra for all twenty synthetic peptides that matched spectra from our analyses, strongly supporting our observation of these variant peptides; three representative examples are given in Figure S1E. These results demonstrate the ability to confidently\",\n",
            "          \"S1E. These results demonstrate the ability to confidently detect, identify and validate genome-level predictions at the protein level. More novel peptides would likely be observed if there was sufficient sample for more extensive fractionation and/or\",\n",
            "          \"was sufficient sample for more extensive fractionation and/or replicate analyses; these limitations preclude any conclusions about the biological significance of unobserved events. For example, only 2 mutant p53 peptides were identified despite the\",\n",
            "          \"example, only 2 mutant p53 peptides were identified despite the presence of p53 mutations in all tumors examined. Such low coverage can occur for reasons that include: excessively large or small tryptic fragments, inability to distinguish some amino\",\n",
            "          \"or small tryptic fragments, inability to distinguish some amino acids, some possible biases against highly hydrophobic and hydrophilic peptides, or low abundance peptides co-eluting with very high abundance peptides. To assess the potential for\",\n",
            "          \"with very high abundance peptides. To assess the potential for post-transcriptional regulation (e.g., translational efficiency or protein degradation), we compared each protein to its corresponding transcript across all tumors, and correlation was\",\n",
            "          \"corresponding transcript across all tumors, and correlation was assessed for those pairs with reliable measurements for both mRNA and protein, i.e., proteins with a corresponding mRNA measurement observed in all 169 HGSC tumors where sample variability\",\n",
            "          \"observed in all 169 HGSC tumors where sample variability exceeded technical variability (3,196 pairs). We excluded 391 proteins observed without a corresponding mRNA (e.g., HBA1) or discordant gene symbol annotation in the protein database (e.g., THOC4).\",\n",
            "          \"gene symbol annotation in the protein database (e.g., THOC4). Overall, 79.4% of the mRNA:protein pairs showed statistically significant positive Spearman correlations (Benjamini-Hochberg adjusted p value <0.01; Figure 1) when changes in mRNA abundance\",\n",
            "          \"p value <0.01; Figure 1) when changes in mRNA abundance were compared to changes in relative protein abundance. The observed median r value of 0.45 for each mRNA-protein pair across all 169 tumors is similar to observations in colorectal cancer, breast\",\n",
            "          \"tumors is similar to observations in colorectal cancer, breast cancer and mouse tissues, although less than found for cell lines. In comparison, the median correlation of paired protein measurements from the same sample, but measured at two sites, was\",\n",
            "          \"from the same sample, but measured at two sites, was substantially higher, at 0.69 (Figure S1). A wide range of mRNA:protein correlations was observed. In general, weaker correlations were observed for highly stable and abundant proteins associated with\",\n",
            "          \"for highly stable and abundant proteins associated with housekeeping or non-intrinsic functions (e.g., ribosomes, mRNA splicing, oxidative phosphorylation, complement cascade), while more dynamic proteins known to be transcriptionally regulated in\",\n",
            "          \"dynamic proteins known to be transcriptionally regulated in response to nutrient demand or other perturbations (e.g., nucleotide metabolism, amino acid metabolism, acute inflammatory responses) were more highly correlated (Figure 1 and Table S3). This\",\n",
            "          \"were more highly correlated (Figure 1 and Table S3). This result is consistent with a previous colorectal cancer study and supports the hypothesis that while many biological functions are primarily regulated by mRNA abundance, post-transcriptional\",\n",
            "          \"are primarily regulated by mRNA abundance, post-transcriptional mechanisms likely have an important role in regulating certain house-keeping functions. Additionally, we found that 23 mRNAs lacking polyA tails displayed lower correlation (mean \\u22120.15) with\",\n",
            "          \"polyA tails displayed lower correlation (mean \\u22120.15) with their cognate proteins than did polyadenylated mRNAs, consistent with the decreased stability of mRNAs lacking polyA tails. Clustering of tumors based on protein abundance HGSC is the most common\",\n",
            "          \"of tumors based on protein abundance HGSC is the most common of the four major histological subtypes of epithelial ovarian cancer, and is characterized by distinct morphological features. Recent studies using mRNA abundance data have suggested four\",\n",
            "          \"Recent studies using mRNA abundance data have suggested four transcriptomic HGSC subtypes designated as differentiated, immunoreactive, mesenchymal, and proliferative. To build an unbiased molecular taxonomy of ovarian HGSC, we used protein abundance data\",\n",
            "          \"taxonomy of ovarian HGSC, we used protein abundance data from the 169 tumors to identify subtypes that might show biological differences that could be exploited in future studies. Figure 2 shows the resulting clustering analysis of individual tumors\",\n",
            "          \"2 shows the resulting clustering analysis of individual tumors (vertical columns) by protein abundance (horizontal rows). The cluster assignment for each sample is provided in Table S4, and a consensus value matrix for the subtype comparisons is shown in\",\n",
            "          \"consensus value matrix for the subtype comparisons is shown in Figure S2A. The results of a Weighted Gene Co-expression Network Analysis (WGCNA) of protein functional enrichment by subgroups are provided for the protein clusters, in Figure S2B and Table\",\n",
            "          \"are provided for the protein clusters, in Figure S2B and Table S4. The enrichment of KEGG and Reactome ontologies in the WGCNA-derived modules are shown in Figures S2C and S2D, respectively; membership of enriched pathways is provided in Table S3. Four of\",\n",
            "          \"of enriched pathways is provided in Table S3. Four of the proteomic clusters showed a clear correspondence to the mesenchymal, proliferative, immunoreactive, and differentiated subtypes defined by the TCGA transcriptome analysis (Figure 2 and Figure S2E).\",\n",
            "          \"by the TCGA transcriptome analysis (Figure 2 and Figure S2E). A relatively small fifth cluster of tumors significantly enriched in proteins associated with extracellular matrix interactions, erythrocyte and platelet functions, and the complement cascade\",\n",
            "          \"erythrocyte and platelet functions, and the complement cascade was also observed using multiple approaches, including model based clustering with Bayesian information criteria, consensus clustering, and VISDA-based sub-phenotype clustering. This new group\",\n",
            "          \"and VISDA-based sub-phenotype clustering. This new group could not be attributed to tissue source site sampling bias or any other meta-data category, but may be related to tumor characteristics, including vascularization and tumor content, as the Tumor\",\n",
            "          \"including vascularization and tumor content, as the Tumor Purity score for this subtype (and the Mesenchymal subtype) was significantly lower than that of the other three subtypes (Figure S2F). The clinical relevance of these protein-based clusters will\",\n",
            "          \"The clinical relevance of these protein-based clusters will require validation in independent HGSC sample sets, particularly as no significant difference in survival was observed (Figure S2G), similar to mRNA-based clustering analysis. Proteomic analysis\",\n",
            "          \"similar to mRNA-based clustering analysis. Proteomic analysis of CNA effects Chromosomal instability, marked by extensive copy number alterations (CNAs) in each tumor, is a hallmark of HGSC, and a likely source of driver alterations in this disease. CNAs\",\n",
            "          \"and a likely source of driver alterations in this disease. CNAs can affect the abundance of proteins at the same locus (cis effects), and may also act in trans, either directly or indirectly. Hypothesizing that CNAs with strong trans effects are more\",\n",
            "          \"Hypothesizing that CNAs with strong trans effects are more likely to elicit a molecular phenotype and confer selective advantages, we sought to identify those CNA regions that have the broadest effect on protein expression. In all, 29,393 CNA loci (Table\",\n",
            "          \"effect on protein expression. In all, 29,393 CNA loci (Table S5) were compared to our global proteomics data with 950,209 CNA/protein pairs (0.72% of the total) exhibiting significant association (Benjamini-Hochberg adjusted p value <0.01). We provide a\",\n",
            "          \"(Benjamini-Hochberg adjusted p value <0.01). We provide a complete list of the significantly associated proteins for each CNA in Table S5. The diagonal line evident in Figure 3 corresponds to cis effects, and vertical stripes correspond to trans effects\",\n",
            "          \"cis effects, and vertical stripes correspond to trans effects where changes in copy number affect expression of numerous proteins across the genome. We performed the same analysis for mRNA to identify sites where associations are transcriptionally\",\n",
            "          \"mRNA to identify sites where associations are transcriptionally mediated. A similar number of CNA/mRNA pairs were found to be significantly associated (1,113,164 at Benjamini-Hochberg adjusted p value <0.01; Figure 3). This analysis revealed regions on\",\n",
            "          \"p value <0.01; Figure 3). This analysis revealed regions on chromosomes 2, 7, 20, and 22 correlated in trans with more than 200 proteins. In contrast to colorectal cancer where most of the trans-regulation of protein by CNA was accompanied by similar\",\n",
            "          \"trans-regulation of protein by CNA was accompanied by similar changes in mRNA, we observed several loci associated with differences in protein abundance without a corresponding change in mRNA. For example, large regions on chromosome 2 have relatively\",\n",
            "          \"For example, large regions on chromosome 2 have relatively little trans effect on mRNA levels, but are associated with more than 200 proteins in trans. Dissecting the mechanisms by which a specific CNA can alter protein levels in trans, without affecting\",\n",
            "          \"CNA can alter protein levels in trans, without affecting the corresponding mRNA is difficult, given the extent of the amplified or deleted regions and the numerous genes affected at a given locus. Possible mechanisms include cis regulation of proteins\",\n",
            "          \"locus. Possible mechanisms include cis regulation of proteins associated with mRNA stability and translational efficiency, such as microRNAs and RNA binding proteins. Given the complex pattern of CNAs observed in HGSC, it has been difficult to identify a\",\n",
            "          \"of CNAs observed in HGSC, it has been difficult to identify a limited number of high impact genomic alterations that could function as drivers of the disease. We interrogated the trans-affected proteins associated with each putative CNA for common\",\n",
            "          \"proteins associated with each putative CNA for common functions using pathways defined by the KEGG, NCI pathway interaction database (PID) and Reactome databases (Supplemental Information). Proteins associated with cell invasion and migration, and\",\n",
            "          \"Proteins associated with cell invasion and migration, and proteins related to immune function appeared to be enriched in association with multiple CNAs (Figure S3; Table S3). These observations suggest a convergence of multiple CNA targets on a common set\",\n",
            "          \"suggest a convergence of multiple CNA targets on a common set of biological functions, namely motility/invasion and immune regulation, functions that are among the hallmarks of cancer. Association of CNA trans-affected proteins with overall survival\",\n",
            "          \"of CNA trans-affected proteins with overall survival Availability of the TCGA survival data allowed us to use trans-affected protein data from the most influential CNAs (e.g., the four altered regions described on chromosomes 2, 7, 20, and 22; Figure 3)\",\n",
            "          \"regions described on chromosomes 2, 7, 20, and 22; Figure 3) to build a model of overall survival. Because each CNA affects many proteins, we used a regression approach that identifies parsimonious Cox proportional hazards models with maximal predictive\",\n",
            "          \"Cox proportional hazards models with maximal predictive ability from the list of significantly correlated proteins for that CNA. We trained models on the proteomics data from PNNL (82 tumors) and tested the ability of the model to predict survival times\",\n",
            "          \"and tested the ability of the model to predict survival times in the data from JHU (87 tumors, not including the 32 overlapping). Each of the four most influential CNA regions produced models that were strongly associated with patient survival (p value\",\n",
            "          \"that were strongly associated with patient survival (p value <0.01, FDR <0.5% based on randomly selected proteins). A Kaplan-Meier plot illustrating the predictive value of each of the locus-specific models after validation is shown in Figure S4A; for the\",\n",
            "          \"models after validation is shown in Figure S4A; for the Kaplan-Meier plots, \\u2018high\\u2019 and \\u2018low\\u2019 expression were defined relative to the median for that signature across all samples, splitting patients into two equal groups of 45 (Table S6). We examined the\",\n",
            "          \"into two equal groups of 45 (Table S6). We examined the overlap in predictions for patients for the four signatures and found that the predictions were unanimous (either high or low signature) for 62% of patients and 16% were evenly split, with the\",\n",
            "          \"for 62% of patients and 16% were evenly split, with the remaining having one dissenting signature. This suggested the utility of combining these signatures, using a voting method where the number of high or low calls was counted for each patient; this\",\n",
            "          \"number of high or low calls was counted for each patient; this substantially improved prediction of survival time (p value 1.9e-6; Figure 4). We also examined the effects of tumor stage, tumor grade, patient age, surgical outcome, and platinum status, and\",\n",
            "          \"grade, patient age, surgical outcome, and platinum status, and found that our proteomic signatures were not improved by inclusion of these variables. For each of these locus-specific models we analyzed the enrichment of genes and their regulatory\",\n",
            "          \"models we analyzed the enrichment of genes and their regulatory sequences associated with outcome, and found all four models had genes significantly enriched (Benjamini-Hochberg adjusted p value <0.05) in binding sites for the proliferation-associated\",\n",
            "          \"value <0.05) in binding sites for the proliferation-associated serum response factor (SRF), suggesting that SRF activity may be important in ovarian cancer outcome. Additionally, there was significant enrichment of proteins involved in the regulation of\",\n",
            "          \"enrichment of proteins involved in the regulation of actin cytoskeleton, apoptosis, and adherens junction (Benjamini-Hochberg adjusted p value <0.05). We examined the protein abundance and phosphorylation status of SRF between short and long surviving\",\n",
            "          \"phosphorylation status of SRF between short and long surviving groups and observed that they were higher in short surviving patients, but only slightly. Thus, although SRF alone was not predictive, the trends in SRF abundance and phosphorylation are\",\n",
            "          \"predictive, the trends in SRF abundance and phosphorylation are consistent with the observation of enrichment for proteins potentially regulated by SRF binding. Details of the larger gene set analysis are provided in Supplemental Information. Several\",\n",
            "          \"set analysis are provided in Supplemental Information. Several proteins shared across all the signatures are known to be involved in cancer processes. Catenin B2 (CTNNA2) is a cell-cell adhesion protein and tumor suppressor. The Rho GDP dissociation\",\n",
            "          \"adhesion protein and tumor suppressor. The Rho GDP dissociation inhibitor (GDI) beta (ARHGDIB) is involved in invasion and migration in many cancers, and overexpression correlates with progression in pancreatic carcinoma. Protein kinase C and casein\",\n",
            "          \"in pancreatic carcinoma. Protein kinase C and casein kinase substrate in neurons protein 2 (PACSIN2) is a repressor of cellular migration. Finally, the GTP-binding nuclear protein Ran, is a prognostic marker associated with increased survival in\",\n",
            "          \"is a prognostic marker associated with increased survival in epithelial ovarian cancer. The association of these previously described survival-related proteins with genes affected in trans by CNAs suggests a potential mechanism for the parallel activation\",\n",
            "          \"CNAs suggests a potential mechanism for the parallel activation of multiple pathways associated with poor prognosis in HGSC. As a comparison we applied the previously described Provar signature, comprising five protein and four phosphoproteins that showed\",\n",
            "          \"comprising five protein and four phosphoproteins that showed good survival prediction in the TCGA ovarian cancer dataset. We observed all proteins in the Provar signature in our proteomic data, but only one of the phosphosites (EGFR Y1173). Thus, we used\",\n",
            "          \"but only one of the phosphosites (EGFR Y1173). Thus, we used the RPPA data from the original signature (Figure S4B). We found that the Provar signature was prognostic of survival (Benjamini-Hochberg adjusted p value = 0.11) in the 67 patients examined\",\n",
            "          \"adjusted p value = 0.11) in the 67 patients examined here (those with phosphoprotein data and had somatic TP53 mutations), but the statistical power of Provar is not as high as the statistical power of signatures derived from the CNA trans-affected\",\n",
            "          \"power of signatures derived from the CNA trans-affected proteins in this dataset. In addition, integrating the Provar signature with the CNA signatures did not improve survival prediction (Figure S4C). Identification of proteins associated with\",\n",
            "          \"(Figure S4C). Identification of proteins associated with chromosomal structural abnormality The degree of chromosomal instability exhibited by a tumor can be represented by a calculated chromosome instability (CIN) index, as described previously for lung\",\n",
            "          \"instability (CIN) index, as described previously for lung cancer. Identification of proteins associated with CIN may provide information on the processes contributing to chromosomal instability, while the analysis of trans-affected proteins described\",\n",
            "          \"while the analysis of trans-affected proteins described above is more closely related to the downstream consequences of specific CNAs. Using a bootstrapping method described in Supplemental Information, we identified a proteomic signature including 128\",\n",
            "          \"Information, we identified a proteomic signature including 128 proteins (Table S7) showing significant correlation with CIN index (Benjamini-Hochberg adjusted p value <1e-4, Spearman correlation; Figure S5A). Functional annotation of these proteins\",\n",
            "          \"Figure S5A). Functional annotation of these proteins (Figure S5B) showed that proteins up-regulated in tumors with high CIN index were preferentially involved in chromatin organization (p value 6.90e-5), whereas proteins up-regulated in tumors with low\",\n",
            "          \"6.90e-5), whereas proteins up-regulated in tumors with low CIN index were more often associated with cell death (p value 2.13e-5). Correlation analysis of protein abundances and CIN indicated that a small number of proteins could account for the majority\",\n",
            "          \"that a small number of proteins could account for the majority of the CIN (Figure 5); two of the most strongly associated proteins, CHD4 and CHD5, are known to be involved in chromatin organization. Identification of proteins associated with HRD status\",\n",
            "          \"Identification of proteins associated with HRD status Since HRD is associated with susceptibility to PARP inhibitors and improved survival, we sought to elucidate systemic changes associated with HRD and identify biomarkers that might be used to stratify\",\n",
            "          \"with HRD and identify biomarkers that might be used to stratify patients for treatment. We defined tumors with HRD as having either germline or somatic mutations in BRCA1 or BRCA2, or BRCA1 promoter methylation, or homozygous deletion of PTEN, with an\",\n",
            "          \"promoter methylation, or homozygous deletion of PTEN, with an overall survival >1.5 years, while non-HRD patients were defined as lacking these genomic aberrations and with a follow-up or time-to-death <2.5 years; additional selection criteria include\",\n",
            "          \"time-to-death <2.5 years; additional selection criteria include available residual tissue volume and a tumor tissue contamination score estimated using copy number alterations. Applying differential dependency network (DDN) analysis on a set of 171 BRCA1-\",\n",
            "          \"dependency network (DDN) analysis on a set of 171 BRCA1- or BRCA2-related proteins curated from the literature and from the cBio portal, we identified a sub-network of 30 proteins that displayed co-expression patterns differentiating HRD from non-HRD\",\n",
            "          \"co-expression patterns differentiating HRD from non-HRD patients (Table S7; Figure 6). Several of the proteins in these modules are known to be involved in histone acetylation or deacetylation, e.g., HDAC1, RBBP4, RBBP7, and EP300, and HUS1. Although\",\n",
            "          \"e.g., HDAC1, RBBP4, RBBP7, and EP300, and HUS1. Although statistical association cannot distinguish between drivers and consequences of HRD status, the observed enrichment of proteins associated with histone acetylation motivated us to use an effective\",\n",
            "          \"with histone acetylation motivated us to use an effective database search strategy (Supplemental Information) to identify and quantify acetylated peptides from the global proteomic data. Comparative analysis of 399 acetylated peptides identified 15\",\n",
            "          \"Comparative analysis of 399 acetylated peptides identified 15 acetylated peptides with significant differences between the HRD and non-HRD samples (Table S7). Among these, dual acetylation at K12 and K16 of histone H4 showed a significant difference\",\n",
            "          \"at K12 and K16 of histone H4 showed a significant difference between HRD and non-HRD samples (Figure 6); differential acetylation of K12 and K16 was verified using synthetic peptides and targeted analysis using Sequential Windowed data independent\",\n",
            "          \"targeted analysis using Sequential Windowed data independent Acquisition of the Total High-resolution Mass Spectra (SWATH-MS). In cell line data, acetylation of H4 has previously been reported to be involved in the choice of DNA double strand break (DSB)\",\n",
            "          \"to be involved in the choice of DNA double strand break (DSB) repair pathways (homologous recombination or non-homologous end joining. This relationship is regulated partially by HDAC1, a protein also identified in the DDN analysis. We observed a\",\n",
            "          \"a protein also identified in the DDN analysis. We observed a significant enrichment of HDAC1 and its co-regulated proteins in tumors with HRD and low H4 acetylation relative to non-HRD tumors with high H4 acetylation (DDN analysis: permutation tests, p\",\n",
            "          \"with high H4 acetylation (DDN analysis: permutation tests, p value <0.05; differentially acetylated peptides: t-tests, p value <0.05 with an estimated FDR <0.5% by bootstrap/permutation tests). The combined observations of increased HDAC1 and associated\",\n",
            "          \"The combined observations of increased HDAC1 and associated proteins at the pathway level, together with decreased acetylation of H4 in HRD patients at the PTM level, provide insight regarding the potential role of HDAC1 in modulating the choice of DSB\",\n",
            "          \"the potential role of HDAC1 in modulating the choice of DSB repair pathways. Phosphoproteomic analysis of pathways associated with survival An initial set of 24,464 different phosphopeptides (21,298 unique phosphorylation sites) contained within 4,420\",\n",
            "          \"(21,298 unique phosphorylation sites) contained within 4,420 proteins having data on net differences in phosphorylation (Table S2) was obtained. Since ischemia of the TCGA tumor samples may alter phosphopeptide abundance, we removed any phosphorylation\",\n",
            "          \"alter phosphopeptide abundance, we removed any phosphorylation sites previously shown to be altered by ischemia. We also removed three tumor samples having substantially higher than average missing values and two tumor samples lacking somatic TP53\",\n",
            "          \"missing values and two tumor samples lacking somatic TP53 mutations (Supplemental Information). This yielded a final set of 7,675 different phosphopeptides (6,802 unique phosphorylation sites) from 2,324 proteins (Table S2) that were mapped to pathways\",\n",
            "          \"from 2,324 proteins (Table S2) that were mapped to pathways via the NCI PID. The average net phosphorylation of all phosphopeptides mapping to a given pathway (i.e., the statistical average across all phosphoproteins known to be in the pathway, rather\",\n",
            "          \"across all phosphoproteins known to be in the pathway, rather than the statistically significant individual proteins) was used as a measure of pathway activity and compared between short survivors (deceased surviving <3 years; N=17) and long survivors\",\n",
            "          \"(deceased surviving <3 years; N=17) and long survivors (patients surviving >5 years; N=19) using both proteomic and phosphoproteomic data. An issue in ranking the pathways is the distinction between phosphopeptides that are unique to a single pathway,\",\n",
            "          \"between phosphopeptides that are unique to a single pathway, such as the receptor peptides themselves, vs. peptides that map to proteins shared between pathways. In order to address this issue, we performed two analyses: one of all proteins associated\",\n",
            "          \"we performed two analyses: one of all proteins associated with a specific pathway in the NCID database, and a second analysis in which proteins common to multiple pathways were excluded. Figure 7A shows a ranking of activated pathways, as inferred from\",\n",
            "          \"7A shows a ranking of activated pathways, as inferred from the analysis of pathway specific components (i.e., differences in mRNA, protein and phosphoprotein levels) in short vs. long survivors. Fifteen pathways showed increased phosphorylation at\",\n",
            "          \"survivors. Fifteen pathways showed increased phosphorylation at Benjamini-Hochberg adjusted p values <0.01, in contrast with transcriptional data for the same samples which yielded only one statistically significantly increased pathway, androgen receptor\",\n",
            "          \"significantly increased pathway, androgen receptor signaling, which was also increased at the phosphoprotein level (Figure 7A and Supplemental Information). All pathways indicated in Figure 7A were increased in association with short survival relative to\",\n",
            "          \"were increased in association with short survival relative to long survival, though some individual components within a pathway showed opposite changes (e.g. JNK1 transcript and protein levels, see Figure 7B). The RhoA-regulatory, PDGFRB, and\",\n",
            "          \"levels, see Figure 7B). The RhoA-regulatory, PDGFRB, and integrin-like kinase pathways emerged as the most activated in this analysis (Figure 7A). Figure 7B shows the common and unique elements of the PDGFRB pathway, and the trends in mRNA, protein, and\",\n",
            "          \"of the PDGFRB pathway, and the trends in mRNA, protein, and phosphoprotein levels in the comparison of short vs. long survivors, illustrating the benefit of combined protein and phosphoprotein measurements, as well as the substantial contribution of\",\n",
            "          \"measurements, as well as the substantial contribution of protein phosphorylation data to the overall analysis. We compared our results to the 31 phosphoproteins represented on the RPPA arrays used in the original TCGA ovarian cancer analysis; RPPA\",\n",
            "          \"arrays used in the original TCGA ovarian cancer analysis; RPPA identified differential phosphorylation of ERK1, RAF1, and STAT3 in the same patient samples analyzed by MS. For short surviving vs. long surviving patients, ERK1 showed statistically\",\n",
            "          \"vs. long surviving patients, ERK1 showed statistically significantly increased phosphorylation by RPPA, on the same ERK1 phosphopeptides as found using MS, while AKT1, RAF1 and STAT3 showed increased phosphorylation by both RPPA and MS-based analyses, but\",\n",
            "          \"phosphorylation by both RPPA and MS-based analyses, but based upon distinct phosphopeptides (Table S2). Good overlap was observed between RPPA and MS results for phosphoproteins on the RPPA array, but we did not observe three of the specific phosphosites\",\n",
            "          \"but we did not observe three of the specific phosphosites from the Provar survival signature in our MS data, suggesting that proteomics is more sensitive to overall pathway activity than specific regulatory events in the pathway itself. However, many of\",\n",
            "          \"regulatory events in the pathway itself. However, many of the phosphopeptides identified by MS phosphoproteomics as components of the PDGFR-beta pathway were not represented in the RPPA, illustrating the ability to identify phosphorylation events not\",\n",
            "          \"illustrating the ability to identify phosphorylation events not readily accessible with currently available antibodies (Figure 7B). DISCUSSION The addition of proteomic information, including post-translational modifications, to the rich genomic and\",\n",
            "          \"post-translational modifications, to the rich genomic and transcriptomic data available from the ovarian HGSC samples analyzed by TCGA has provided additional insights into the biology of HGSC, including identification of changes at the level of pathway\",\n",
            "          \"including identification of changes at the level of pathway activation. Integration of genomic, proteomic, and phosphoproteomic measurements identified differentially regulated pathways and functional modules that displayed significant associations with\",\n",
            "          \"functional modules that displayed significant associations with patient outcomes, including survival and HRD status. Cox proportional hazard analysis of proteins associated with CNAs identified overall survival signatures enriched for targets of the\",\n",
            "          \"overall survival signatures enriched for targets of the proliferation-associated transcription factor SRF, and integrated proteomic, phosphoproteomic and transcriptomic data identified pathways that differentiated patients on the basis of survival,\",\n",
            "          \"pathways that differentiated patients on the basis of survival, including the PDGFR-beta signaling pathway associated with angiogenesis, the RhoA regulatory and integrin-linked kinase pathways associated with cell mobility and invasion, and pathways\",\n",
            "          \"associated with cell mobility and invasion, and pathways associated with chemokine signaling and adaptive immunity. Proteins associated with cell invasion and motility emerged from both the integration of CNAs with protein abundance data and the\",\n",
            "          \"the integration of CNAs with protein abundance data and the phosphoproteomic investigation of up-regulated pathways in short versus long survivors. Despite the complexity of genomic alterations that characterize HGSC, these analyses suggest a functional\",\n",
            "          \"that characterize HGSC, these analyses suggest a functional convergence on a subset of key pathways. The association of increased invasiveness and motility with short overall survival in this study may help to explain more aggressive mechanisms of\",\n",
            "          \"in this study may help to explain more aggressive mechanisms of dissemination in ovarian cancer, including recently reported hematogenous metastasis, in addition to previously described lymphatic spread and known peritoneal spread by direct extension.\",\n",
            "          \"spread and known peritoneal spread by direct extension. Focusing on functional modules has also revealed potential drivers of HGSC and more robust signatures for potentially stratifying ovarian HGSC patients into distinct cancer phenotypes to inform\",\n",
            "          \"ovarian HGSC patients into distinct cancer phenotypes to inform therapeutic management. For example, the identification of specific acetylation events associated with HRD, e.g., the simultaneous acetylation of K12 and K16 on histone H4, may provide an\",\n",
            "          \"acetylation of K12 and K16 on histone H4, may provide an alternative biomarker of HRD and a rationale for the selection of patients in future clinical trials of HDAC inhibitors alone or in combination with PARP inhibition. This may help to resolve the\",\n",
            "          \"combination with PARP inhibition. This may help to resolve the current discrepancy between the initial observation of limited single agent activity of HDAC inhibitors in ovarian cancer, and more recent findings of a >40% response rate when used in\",\n",
            "          \"and more recent findings of a >40% response rate when used in combination with cytotoxic chemotherapy in platinum-resistant patients. Similarly, comprehensive interrogation of protein phosphorylation identified multiple pathways with significantly\",\n",
            "          \"phosphorylation identified multiple pathways with significantly increased phosphorylation in patients with poor clinical outcomes. Specifically, the observed activation of PDGFR pathways in a subset of patients with short overall survival could\",\n",
            "          \"in a subset of patients with short overall survival could potentially stratify patients for selective enrollment in trials of anti-angiogenic therapy, which is particularly relevant to current controversies over the use of bevacuzimab as a first-line\",\n",
            "          \"controversies over the use of bevacuzimab as a first-line therapy in ovarian cancer. Overall, this work illustrates the ability of proteomics to complement genomics in providing additional insights into pathways and processes that drive ovarian cancer\",\n",
            "          \"insights into pathways and processes that drive ovarian cancer biology and how these pathways are altered in correspondence with clinical phenotypes. The comprehensive proteomic measurements for the HGSC tumor samples provide a public resource of\",\n",
            "          \"for the HGSC tumor samples provide a public resource of information. Importantly, the ability to identify PTMs revealed a strong association between specific histone acetylation events and the HRD phenotype. In addition, the enhanced view of pathway\",\n",
            "          \"the HRD phenotype. In addition, the enhanced view of pathway activity provided by measurements of protein phosphorylation provides a foundation for linking genotype to proteotype, and ultimately to phenotype for understanding the molecular basis of\",\n",
            "          \"to phenotype for understanding the molecular basis of cancer. EXPERIMENTAL PROCEDURES Full methods are available in Extended Experimental Procedures. Tumor Samples All tumor samples for the current study were obtained through the TCGA Biospecimen Core\",\n",
            "          \"current study were obtained through the TCGA Biospecimen Core Resource as described previously. Samples were selected based on overall survival, HRD status and availability. Quantitative Proteomics Tissue proteins were extracted and digested with trypsin;\",\n",
            "          \"Tissue proteins were extracted and digested with trypsin; at each site, a portion of each resulting sample was also used to create a pooled reference in which each tumor sample contributed an equal percentage by peptide mass. The pooled reference sample\",\n",
            "          \"equal percentage by peptide mass. The pooled reference sample was included in each multiplexed, isobaric labeling experiment to enable cross-experiment comparison in the entire sample cohort. The patient samples and the pooled reference sample were each\",\n",
            "          \"The patient samples and the pooled reference sample were each labeled by different iTRAQ reagents (Sciex), combined, fractionated, and split for integrated, quantitative global proteome (10%) and phosphoproteome (90%) analysis using LC-MS/MS on an\",\n",
            "          \"(10%) and phosphoproteome (90%) analysis using LC-MS/MS on an Orbitrap Velos mass spectrometer (Thermo Scientific). Raw data were processed for peptide identification by database searching and identified peptides were assembled as proteins and mapped to\",\n",
            "          \"identified peptides were assembled as proteins and mapped to gene identifiers for proteogenomic comparisons. Quantitation was achieved by comparing the iTRAQ reporter ion intensities in each sample. Comparison of mRNA and Protein Subtypes A model-based\",\n",
            "          \"sample. Comparison of mRNA and Protein Subtypes A model-based clustering approach was used to model the protein abundance data as a mixture of subtypes. Bayesian information criteria, statistical resampling and VISDA-based sub-phenotype clustering\",\n",
            "          \"statistical resampling and VISDA-based sub-phenotype clustering approaches provided similar results with a stable optimization on five clusters as determined by consensus clustering. WGCNA analysis was performed to infer genes or gene networks that drive\",\n",
            "          \"was performed to infer genes or gene networks that drive subtyping into 5 clusters, followed by correlation with subtype as a trait. Integration of Proteomics and CNA data Spearman correlation coefficients and corresponding adjusted p values were\",\n",
            "          \"coefficients and corresponding adjusted p values were calculated for each protein/transcript by CNA locus, and gene set enrichment analysis was used to infer function for groups of proteins significantly correlated with a given CNA locus. Regression\",\n",
            "          \"significantly correlated with a given CNA locus. Regression analysis was applied to the list of trans-affected protein correlated with each CNA to generate parsimonious Cox proportional hazards models with maximal predictive ability, using the PNNL data\",\n",
            "          \"models with maximal predictive ability, using the PNNL data for training and JHU data for testing. Kaplan-Meier plots were used to visualize performance in predicting overall survival and progression-free survival. A CIN index was calculated for each\",\n",
            "          \"progression-free survival. A CIN index was calculated for each sample as the mean absolute values of copy number measurements at the 29,393 selected loci. Bootstrap resampling was used to select proteins correlated with CIN index at high confidence.\",\n",
            "          \"select proteins correlated with CIN index at high confidence. Protein Acetylation and HRD Acetylated peptides were identified by searching the global proteomics data for dynamic acetylation to lysines (+42 Da). Acetylation levels were compared between HRD\",\n",
            "          \"lysines (+42 Da). Acetylation levels were compared between HRD and non-HRD cases by t-test. Targeted proteomics (SWATH) was used to orthogonally quantify the acetylated peptide with synthetic peptides as internal standard. Survival-related Pathways\",\n",
            "          \"peptides as internal standard. Survival-related Pathways Analysis Phosphoproteome data from the short and long survivors were mapped to signaling pathways in the NCI pathway information database,(PID (http://pid.nci.nih.gov) using the gene names. For each\",\n",
            "          \"(http://pid.nci.nih.gov) using the gene names. For each signaling pathway in the PID, relative abundances for all phosphopeptides mapping to any pathway component were identified and separated into short and long survivor groups. The difference in\",\n",
            "          \"into short and long survivor groups. The difference in distributions between the two sets of pathway-specific peptides, i.e., those associated with either short survivors or long long survivors, was then assessed using a two-tailed t test. Similar\",\n",
            "          \"survivors, was then assessed using a two-tailed t test. Similar enrichment analyses were also performed using protein abundance, mRNA abundance, and CNA. Data Repository All of the primary mass spectrometry data on TCGA tumor samples are deposited at the\",\n",
            "          \"spectrometry data on TCGA tumor samples are deposited at the CPTAC Data Coordinating Center as raw and mzML files and complete protein assembly data sets for public access (https://cptac-data-portal.georgetown.edu). Supplementary Material This is a PDF\",\n",
            "          \"Supplementary Material This is a PDF file of an unedited manuscript that has been accepted for publication. As a service to our customers we are providing this early version of the manuscript. The manuscript will undergo copyediting, typesetting, and\",\n",
            "          \"The manuscript will undergo copyediting, typesetting, and review of the resulting proof before it is published in its final form. Please note that during the production process errors may be discovered which could affect the content, and all legal\",\n",
            "          \"may be discovered which could affect the content, and all legal disclaimers that apply to the journal pertain.            Author Contributions          Study conception and design: HZ, TL, ZZ, RDS, DWC, KDR. Investigation: performed experiment or data\",\n",
            "          \"ZZ, RDS, DWC, KDR. Investigation: performed experiment or data collection: TL, SS, FY, JZ, LijunC, PS, PA, YT, MAG, TRC, CC, ST, SN, RJM, HengZ. Computation and statistical analysis: ZZ, SHP, BaiZ, JEM, VAP, LiC, DR, MEM, SWC, SW, JW, DLT, DF, VB, YW,\",\n",
            "          \"BaiZ, JEM, VAP, LiC, DR, MEM, SWC, SW, JW, DLT, DF, VB, YW, BingZ. Data interpretation and biological analysis: TL, HZ, SHP, ZZ, BaiZ, JEM, VAP, LiC, DR, SN, CW, IMS, AP, MPS, DAL, RDS, DWC, KDR. Writing/manuscript preparation and revision: HZ, TL, ZZ,\",\n",
            "          \"KDR. Writing/manuscript preparation and revision: HZ, TL, ZZ, SHP, JEM, DR, VAP, LeslieC, HR, IMS, AP, MPS, DAL, RDS, DWC, KDR. Supervision and administration: HR, ESB, TH, RCR, LS, RDS, DWC, KDR. The authors declare no competing financial interests.\",\n",
            "          \"KDR. The authors declare no competing financial interests.            SUPPLEMENTAL INFORMATION          Supplemental information includes Extended Experimental Procedures, 5 figures, and 8 tables and can be found with this article online. Correlations\",\n",
            "          \"tables and can be found with this article online. Correlations between mRNA and protein abundance in TCGA tumors mRNA and protein were correlated across all the samples, resulting in 90.6% positive correlations, and 79.4% were significantly correlated\",\n",
            "          \"positive correlations, and 79.4% were significantly correlated (Benjamini-Hochberg adjusted p value <0.01), with a mean Spearman\\u2019s correlation of 0.38 and a median value of 0.45 (top panel). Different biological pathways and processes showed significantly\",\n",
            "          \"biological pathways and processes showed significantly different levels of correlation (bottom panel). Metabolic pathways and the interferon response displayed high mRNA-protein correlation, and ribosome, mRNA splicing, oxidative phosphorylation, and\",\n",
            "          \"and ribosome, mRNA splicing, oxidative phosphorylation, and complement and coagulation cascade were poorly correlated. The mean correlation is shown in parentheses followed by Benjamini-Hochberg adjusted p values calculated using a Kolmogorov\\u2013Smirnov test\",\n",
            "          \"adjusted p values calculated using a Kolmogorov\\u2013Smirnov test following the functional group names from MSIGDB. Blue bars indicate positive correlations and yellow indicates negative correlations; individual proteins (represented as bars on the x-axis) are\",\n",
            "          \"individual proteins (represented as bars on the x-axis) are sorted by correlation from low to high (bottom panel). Proteomic subtypes and corresponding driving protein modules Global proteomics abundance is shown in a heatmap with TCGA samples represented\",\n",
            "          \"abundance is shown in a heatmap with TCGA samples represented by columns ordered by protein subtype where rows represent proteins. Color of each cell indicates z-score (log2 of relative abundance scaled by proteins\\u2019 standard deviation) of the protein in\",\n",
            "          \"scaled by proteins\\u2019 standard deviation) of the protein in that sample; red is increased and blue is decreased (relative to the pooled reference). Transcriptome-based subtypes (Verhaak et al., 2013) and the proposed proteomic subtypes are indicated in\",\n",
            "          \"al., 2013) and the proposed proteomic subtypes are indicated in color above the heatmap. WGCNA-derived modules are delineated with the row color panel and annotated according to the pathways based on enrichment of KEGG and Reactome ontologies. Functional\",\n",
            "          \"based on enrichment of KEGG and Reactome ontologies. Functional impact of copy number alterations The top panel shows the correlation of CNAs to protein abundance (right) or mRNA (left) with significant positive correlations in red and negative\",\n",
            "          \"with significant positive correlations in red and negative correlations in blue (Benjamini-Hochberg adjusted p value <0.01, Spearman\\u2019s correlation coefficient). The x-axis plots the 29,393 CNAs obtained from TCGA. The y-axis plots 3,202 proteins. Genes\",\n",
            "          \"CNAs obtained from TCGA. The y-axis plots 3,202 proteins. Genes are ordered by chromosomal location on both the x- and y-axes. The bottom panel shows the summed number of significantly correlated proteins (or mRNA) for each individual CNA. In blue is\",\n",
            "          \"proteins (or mRNA) for each individual CNA. In blue is shown the total number, in black are those genes significantly correlated as both mRNA and protein. Where blue and black lines have similar magnitude, e.g., the hotspot on chromosome 20, the CNA\",\n",
            "          \"similar magnitude, e.g., the hotspot on chromosome 20, the CNA associations are shared between protein and mRNA. Where a strong blue line has no mirrored black line, e.g., the protein hotspot on chromosome 2, the associations with CNA are largely unique.\",\n",
            "          \"on chromosome 2, the associations with CNA are largely unique. Kaplan-Meier plot of overall survival stratified by CNA-derived signatures A survival plot is shown for a consensus of the four best signatures (see Figure S4A and Table S6) identified from\",\n",
            "          \"best signatures (see Figure S4A and Table S6) identified from analysis of proteins affected in trans by CNAs. Models were trained using a lasso-based Cox proportional hazards model on the samples from PNNL, and shown are the survival curves from these\",\n",
            "          \"samples from PNNL, and shown are the survival curves from these models applied to the non-overlapping data from the JHU analysis, with the up (red; above the median) and down (blue, below the median) signatures each representing 45 patients. A vote was\",\n",
            "          \"median) signatures each representing 45 patients. A vote was taken among the four most predictive signatures, and the results of this vote are shown. Probability of death is shown on the y-axis vs. survival time in years on the x-axis. Shaded ribbons\",\n",
            "          \"y-axis vs. survival time in years on the x-axis. Shaded ribbons denote 95% confidence intervals. Number of CNAs statistically explained by proteins significantly associated with CIN index A total of 128 proteins selected for their association with CIN\",\n",
            "          \"A total of 128 proteins selected for their association with CIN were arranged along the chromosomal locations of their corresponding genes (y-axis). The length of the light grey horizontal lines indicates the number of CNAs significantly correlated with\",\n",
            "          \"indicates the number of CNAs significantly correlated with the protein across all chromosomes except for the protein\\u2019s corresponding gene coding region (x-axis). The top-ranked proteins are annotated and highlighted with dark lines to show the\",\n",
            "          \"are annotated and highlighted with dark lines to show the bootstrap-estimated 95% confidence intervals. DDN analysis and lysine-acetylation analysis between HRD and non-HRD patients DDN analysis revealed a sub-network of proteins that displayed distinct\",\n",
            "          \"revealed a sub-network of proteins that displayed distinct co-expression patterns between HRD and non-HRD patients, where purple connections indicate protein correlations that exist only in HRD samples and blue connections indicate protein correlations\",\n",
            "          \"HRD samples and blue connections indicate protein correlations that exist only in non-HRD patients. The links between the nodes are drawn with two different thickness indicative of whether the connections meet the significance threshold of 0.05 (light\",\n",
            "          \"the connections meet the significance threshold of 0.05 (light lines) or 0.01 (bold lines). Proteins with blue dotted circles are known to be involved in histone acetylation or deacetylation. Further, identification and quantitation of lysine-acetylated\",\n",
            "          \"Further, identification and quantitation of lysine-acetylated peptides in global proteomics data showed that acetylation level at K12 and K16 of histone H4 are significantly different between HRD and non-HRD samples, suggesting a role of histone H4\",\n",
            "          \"HRD and non-HRD samples, suggesting a role of histone H4 acetylation (together with HDAC1) in modulating the choice of DSB repair mechanisms. Pathway analysis associated with patient survival A) Pathway components were statistically analyzed using a\",\n",
            "          \"A) Pathway components were statistically analyzed using a two-sided t test between samples from patients surviving <3 years (short survival) and patients surviving >5 years (long survival) for differences in CNA, mRNA expression, protein or\",\n",
            "          \"survival) for differences in CNA, mRNA expression, protein or phosphorylation abundance. All significant pathways for mRNA and protein are shown (Benjamini-Hochberg adjusted p value <0.05) and the most significant pathways for phosphoproteomics are\",\n",
            "          \"and the most significant pathways for phosphoproteomics are plotted on the x axis as the \\u2212log of the p value. Results show that phosphorylation provides additional information about the functional state of tumors. B) Simplified PDGFR-beta pathway showing\",\n",
            "          \"state of tumors. B) Simplified PDGFR-beta pathway showing differences between short and long survivor groups for protein abundance (p), transcript abundance (m), and phosphopeptide abundance (circled P).\"\n",
            "        ]\n",
            "      }\n",
            "    ]\n",
            "  },\n",
            "  \"PDC000113\": {\n",
            "    \"study_id\": \"ba4e17a5-57b8-11e8-b07a-00a098d917f8\",\n",
            "    \"pdc_study_id\": \"PDC000113\",\n",
            "    \"study_submitter_id\": \"TCGA OV Proteome S020-2\",\n",
            "    \"program_id\": \"10251935-5540-11e8-b664-00a098d917f8\",\n",
            "    \"project_id\": \"48af5040-5546-11e8-b664-00a098d917f8\",\n",
            "    \"study_name\": \"TCGA Ovarian JHU Proteome\",\n",
            "    \"program_name\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"project_name\": \"CPTAC2 Retrospective\",\n",
            "    \"disease_type\": \"Other;Ovarian Serous Cystadenocarcinoma\",\n",
            "    \"primary_site\": \"Not Reported;Ovary\",\n",
            "    \"analytical_fraction\": \"Proteome\",\n",
            "    \"experiment_type\": \"iTRAQ4\",\n",
            "    \"embargo_date\": null,\n",
            "    \"cases_count\": 124,\n",
            "    \"aliquots_count\": 124,\n",
            "    \"protocol_id\": \"1c06b8e7-3b26-11e9-9a07-0a80fada099c\",\n",
            "    \"protocol_submitter_id\": \"P-S020-2\",\n",
            "    \"program_submitter_id\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"protocol_name\": \"TCGA_Ovarian_JHU_Proteome\",\n",
            "    \"protocol_date\": \"2014-04-01\",\n",
            "    \"publications\": [\n",
            "      {\n",
            "        \"publication_id\": \"44e2e658-89c7-11e8-bcf1-0a2705229b82\",\n",
            "        \"pubmed_id\": \"27372738\",\n",
            "        \"doi\": \"10.1016/j.cell.2016.05.069\",\n",
            "        \"title\": \"Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer\",\n",
            "        \"journal\": \"Cell\",\n",
            "        \"journal_url\": \"https://www.cell.com/cell/fulltext/S0092-8674(16)30673-0?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0092867416306730%3Fshowall%3Dtrue\",\n",
            "        \"year\": \"2016\",\n",
            "        \"abstract\": \"To provide a detailed analysis of the molecular components and underlying mechanisms associated with ovarian cancer, we performed a comprehensive mass-spectrometry-based proteomic characterization of 174 ovarian tumors previously analyzed by The Cancer Genome Atlas (TCGA), of which 169 were high-grade serous carcinomas (HGSCs). Integrating our proteomic measurements with the genomic data yielded a number of insights into disease, such as how different copy-number alternations influence the proteome, the proteins associated with chromosomal instability, the sets of signaling pathways that diverse genome rearrangements converge on, and the ones most associated with short overall survival. Specific protein acetylations associated with homologous recombination deficiency suggest a potential means for stratifying patients for therapy. In addition to providing a valuable resource, these findings provide a view of how the somatic genome drives the cancer proteome and associations between protein and post-translational modification levels and clinical outcomes in HGSC.\",\n",
            "        \"full_text\": [\n",
            "          \"Integrated proteogenomic characterization of human high grade serous ovarian cancer SUMMARY To provide a detailed analysis of the molecular components and underlying mechanisms associated with ovarian cancer, we performed a comprehensive mass\",\n",
            "          \"with ovarian cancer, we performed a comprehensive mass spectrometry-based proteomic characterization of 174 ovarian tumors previously analyzed by The Cancer Genome Atlas (TCGA), of which 169 were high-grade serous carcinomas (HGSC). Integrating our\",\n",
            "          \"169 were high-grade serous carcinomas (HGSC). Integrating our proteomic measurements with the genomic data yielded a number of insights into disease such as how different copy number alternations influence the proteome, the proteins associated with\",\n",
            "          \"influence the proteome, the proteins associated with chromosomal instability, the sets of signaling pathways that diverse genome rearrangements converge on, as well as the ones most associated with short overall survival. Specific protein acetylations\",\n",
            "          \"with short overall survival. Specific protein acetylations associated with homologous recombination deficiency suggest a potential means for stratifying patients for therapy. In addition to providing a valuable resource, these findings provide a view of\",\n",
            "          \"providing a valuable resource, these findings provide a view of how the somatic genome drives the cancer proteome and associations between protein and post-translational modification levels and clinical outcomes in HGSC. INTRODUCTION A comprehensive\",\n",
            "          \"and clinical outcomes in HGSC. INTRODUCTION A comprehensive molecular view of cancer is necessary for understanding the underlying mechanisms of disease, improving prognosis, and ultimately guiding treatment. The Cancer Genome Atlas (TCGA) conducted an\",\n",
            "          \"guiding treatment. The Cancer Genome Atlas (TCGA) conducted an extensive genomic and transcriptomic characterization of ovarian high-grade serous carcinoma (HGSC) aimed at defining the genomic landscape and aiding the development of targeted therapies for\",\n",
            "          \"landscape and aiding the development of targeted therapies for this highly lethal malignancy. Key findings from TCGA were: 1) the prevalent role of TP53 mutations, 2) extensive DNA copy alterations, 3) preliminary transcriptional signatures associated\",\n",
            "          \"3) preliminary transcriptional signatures associated with survival, 4) varied mechanisms of BRCA1/2 inactivation, and lastly, 5) CCNE1 aberrations. Subsequent analysis of genomic data from the TCGA consortium led to the refinement of the\",\n",
            "          \"data from the TCGA consortium led to the refinement of the transcript-defined signatures, improving the statistical association with patient outcome, and integrating microRNA and mRNA expression profiles associated with HGSC to identify candidate microRNA\",\n",
            "          \"profiles associated with HGSC to identify candidate microRNA targets. While the insights from genomic analyses are substantial, functions encoded in the genome are generally executed at the protein level, and are often further modulated by\",\n",
            "          \"at the protein level, and are often further modulated by post-translational modifications (PTMs). For example, TCGA used reverse phase protein array (RPPA) analysis of 172 proteins (including 31 phosphoproteins with phospho-specific antibodies) to\",\n",
            "          \"31 phosphoproteins with phospho-specific antibodies) to generate a signature associated with the risk of tumor recurrence. To obtain a more comprehensive assessment of the complex ovarian HGSC phenotype, the Clinical Proteomic Tumor Analysis Consortium\",\n",
            "          \"phenotype, the Clinical Proteomic Tumor Analysis Consortium (CPTAC) conducted an extensive mass spectrometry (MS)-based proteomic and phosphoproteomic characterization of HGSC tumors characterized by TCGA, providing quantitative measurements for a\",\n",
            "          \"by TCGA, providing quantitative measurements for a combined total of 9,600 proteins from 174 tumors (an average of 7,952 proteins per tumor), and a total of 24,429 phosphosites from 6,769 phosphoproteins in a subset of 69 tumors (an average of 7,677\",\n",
            "          \"phosphoproteins in a subset of 69 tumors (an average of 7,677 phosphosites per tumor). Our results provide insights on HGSC biology and correlate differences in protein and PTM levels with clinical outcome complementary to TCGA genomic analyses. RESULTS\",\n",
            "          \"outcome complementary to TCGA genomic analyses. RESULTS Proteomic analysis of TCGA HGSC samples HGSC biospecimens and clinical data from 174 patients collected by TCGA were analyzed at two independent CPTAC Centers, Johns Hopkins University (JHU) and\",\n",
            "          \"independent CPTAC Centers, Johns Hopkins University (JHU) and Pacific Northwest National Laboratory (PNNL); thirty-two samples were analyzed at both JHU and PNNL. Tumors were selected by examining the associated TCGA metadata to select tumors either 1) on\",\n",
            "          \"the associated TCGA metadata to select tumors either 1) on the basis of putative homologous recombination deficiency (HRD), defined by the presence of germline or somatic BRCA1 or BRCA2 mutations, BRCA1 promoter methylation, or homozygous deletion of PTEN\",\n",
            "          \"BRCA1 promoter methylation, or homozygous deletion of PTEN (122 samples; 67 classified as HRD, 55 as non-HRD by the above criteria; JHU), or 2) to maximize differences in overall survival (84 samples; PNNL). For selection purposes, short survival was\",\n",
            "          \"(84 samples; PNNL). For selection purposes, short survival was defined as overall survival of less than 3 years, and long survival as greater than 5 years. All but five tumors had somatic TP53 mutations, a characteristic feature of HGSC; these five tumors\",\n",
            "          \"mutations, a characteristic feature of HGSC; these five tumors were subsequently reclassified as other than HGSC, and removed from protein functional analyses (e.g., subtyping and survival analyses). The tumor selection criteria and the associated\",\n",
            "          \"analyses). The tumor selection criteria and the associated metadata are provided in the Supplemental Information Table S1, available online. Proteomics measurements used isobaric tags for relative and absolute quantitation (iTRAQ) in conjunction with\",\n",
            "          \"relative and absolute quantitation (iTRAQ) in conjunction with offline liquid chromatography fractionation via high-pH reversed phase liquid chromatography (RPLC) and online RPLC with high-resolution tandem MS to provide broad coverage for peptide and\",\n",
            "          \"tandem MS to provide broad coverage for peptide and protein identification and quantification (Supplemental Information); this also alleviated quantitative interference potentially associated with the use of isobaric tags. We used the relative abundance\",\n",
            "          \"with the use of isobaric tags. We used the relative abundance measurements for each protein in the 32 patient samples analyzed at both JHU and PNNL to normalize across the two analysis sites, and then used clustering, principle component analysis (PCA),\",\n",
            "          \"and then used clustering, principle component analysis (PCA), and statistical tests to identify any significant batch effects associated with the site of analysis (a detailed comparison of within site, between site, between sample measurement variability\",\n",
            "          \"site, between site, between sample measurement variability and the process used to merge the JHU and PNNL data is given in Supplemental Information, Figure S1). As shown in Figure S1C, the median coefficient of variation (CV) between measurements at the\",\n",
            "          \"coefficient of variation (CV) between measurements at the two sites was 16%. A total of 9,600 proteins were identified with high confidence in all tumors, and the relative abundances in each tumor are given in Table S2. Functional analyses and\",\n",
            "          \"in each tumor are given in Table S2. Functional analyses and proteome-transcriptome associations were restricted to 3,586 proteins observed and quantified in all 169 HGSC samples used for protein functional analyses and where sample variability (signal)\",\n",
            "          \"functional analyses and where sample variability (signal) exceeded technical variability (noise) in the merged data (Table S2), calculated as described in Supplemental Information. On average we identify peptides covering 29% of the amino acids in any of\",\n",
            "          \"we identify peptides covering 29% of the amino acids in any of these 3,586 proteins in a given sample, with a range from 10% to 47%. In addition to protein abundance levels, phosphoproteomics data were acquired for 69 tumors with sufficient sample (Table\",\n",
            "          \"data were acquired for 69 tumors with sufficient sample (Table S2). As with proteins, phosphopeptide abundances were calculated relative to the pooled reference sample. Because isobaric labeling was performed prior to splitting the samples for proteome\",\n",
            "          \"was performed prior to splitting the samples for proteome and phosphoproteome analyses, phosphopeptide abundance could be corrected for changes in parent protein abundance to identify differences in the relative extent of phosphorylation at specific sites\",\n",
            "          \"in the relative extent of phosphorylation at specific sites for each protein (Table S2). Overall, we achieved a protein dynamic range encompassing >4 orders of magnitude, ranging from low-level transcription factors to abundant structural proteins i.e.,\",\n",
            "          \"transcription factors to abundant structural proteins i.e., actin and tubulin. Proteogenomic landscape of HGSC The degree to which alterations observed at the genome and transcriptome levels are manifested at the protein level is variable, both\",\n",
            "          \"levels are manifested at the protein level is variable, both qualitatively and quantitatively, and partially driven by multiple levels of post-transcriptional regulation. To identify peptides encoded by single amino acid variants (SAAVs), splice variants,\",\n",
            "          \"encoded by single amino acid variants (SAAVs), splice variants, and novel exons documented by TCGA, mass spectra were searched against a custom graph database containing all peptide variations projected from the TCGA genomic analyses of the cohort using a\",\n",
            "          \"projected from the TCGA genomic analyses of the cohort using a multi-stage analysis pipeline (Supplemental Information). The most frequently observed variant peptides represented SAAVs and gene-level events. The evidence supporting each novel event is\",\n",
            "          \"gene-level events. The evidence supporting each novel event is detailed in Table S2, including event type, genomic location, abundance information, and estimated false discovery rate (FDR). The validity of these variant peptides was further evaluated by\",\n",
            "          \"The validity of these variant peptides was further evaluated by the synthesis of twenty examples selected at random across the entire range of spectrum match scores. We obtained tandem mass spectra for all twenty synthetic peptides that matched spectra\",\n",
            "          \"spectra for all twenty synthetic peptides that matched spectra from our analyses, strongly supporting our observation of these variant peptides; three representative examples are given in Figure S1E. These results demonstrate the ability to confidently\",\n",
            "          \"S1E. These results demonstrate the ability to confidently detect, identify and validate genome-level predictions at the protein level. More novel peptides would likely be observed if there was sufficient sample for more extensive fractionation and/or\",\n",
            "          \"was sufficient sample for more extensive fractionation and/or replicate analyses; these limitations preclude any conclusions about the biological significance of unobserved events. For example, only 2 mutant p53 peptides were identified despite the\",\n",
            "          \"example, only 2 mutant p53 peptides were identified despite the presence of p53 mutations in all tumors examined. Such low coverage can occur for reasons that include: excessively large or small tryptic fragments, inability to distinguish some amino\",\n",
            "          \"or small tryptic fragments, inability to distinguish some amino acids, some possible biases against highly hydrophobic and hydrophilic peptides, or low abundance peptides co-eluting with very high abundance peptides. To assess the potential for\",\n",
            "          \"with very high abundance peptides. To assess the potential for post-transcriptional regulation (e.g., translational efficiency or protein degradation), we compared each protein to its corresponding transcript across all tumors, and correlation was\",\n",
            "          \"corresponding transcript across all tumors, and correlation was assessed for those pairs with reliable measurements for both mRNA and protein, i.e., proteins with a corresponding mRNA measurement observed in all 169 HGSC tumors where sample variability\",\n",
            "          \"observed in all 169 HGSC tumors where sample variability exceeded technical variability (3,196 pairs). We excluded 391 proteins observed without a corresponding mRNA (e.g., HBA1) or discordant gene symbol annotation in the protein database (e.g., THOC4).\",\n",
            "          \"gene symbol annotation in the protein database (e.g., THOC4). Overall, 79.4% of the mRNA:protein pairs showed statistically significant positive Spearman correlations (Benjamini-Hochberg adjusted p value <0.01; Figure 1) when changes in mRNA abundance\",\n",
            "          \"p value <0.01; Figure 1) when changes in mRNA abundance were compared to changes in relative protein abundance. The observed median r value of 0.45 for each mRNA-protein pair across all 169 tumors is similar to observations in colorectal cancer, breast\",\n",
            "          \"tumors is similar to observations in colorectal cancer, breast cancer and mouse tissues, although less than found for cell lines. In comparison, the median correlation of paired protein measurements from the same sample, but measured at two sites, was\",\n",
            "          \"from the same sample, but measured at two sites, was substantially higher, at 0.69 (Figure S1). A wide range of mRNA:protein correlations was observed. In general, weaker correlations were observed for highly stable and abundant proteins associated with\",\n",
            "          \"for highly stable and abundant proteins associated with housekeeping or non-intrinsic functions (e.g., ribosomes, mRNA splicing, oxidative phosphorylation, complement cascade), while more dynamic proteins known to be transcriptionally regulated in\",\n",
            "          \"dynamic proteins known to be transcriptionally regulated in response to nutrient demand or other perturbations (e.g., nucleotide metabolism, amino acid metabolism, acute inflammatory responses) were more highly correlated (Figure 1 and Table S3). This\",\n",
            "          \"were more highly correlated (Figure 1 and Table S3). This result is consistent with a previous colorectal cancer study and supports the hypothesis that while many biological functions are primarily regulated by mRNA abundance, post-transcriptional\",\n",
            "          \"are primarily regulated by mRNA abundance, post-transcriptional mechanisms likely have an important role in regulating certain house-keeping functions. Additionally, we found that 23 mRNAs lacking polyA tails displayed lower correlation (mean \\u22120.15) with\",\n",
            "          \"polyA tails displayed lower correlation (mean \\u22120.15) with their cognate proteins than did polyadenylated mRNAs, consistent with the decreased stability of mRNAs lacking polyA tails. Clustering of tumors based on protein abundance HGSC is the most common\",\n",
            "          \"of tumors based on protein abundance HGSC is the most common of the four major histological subtypes of epithelial ovarian cancer, and is characterized by distinct morphological features. Recent studies using mRNA abundance data have suggested four\",\n",
            "          \"Recent studies using mRNA abundance data have suggested four transcriptomic HGSC subtypes designated as differentiated, immunoreactive, mesenchymal, and proliferative. To build an unbiased molecular taxonomy of ovarian HGSC, we used protein abundance data\",\n",
            "          \"taxonomy of ovarian HGSC, we used protein abundance data from the 169 tumors to identify subtypes that might show biological differences that could be exploited in future studies. Figure 2 shows the resulting clustering analysis of individual tumors\",\n",
            "          \"2 shows the resulting clustering analysis of individual tumors (vertical columns) by protein abundance (horizontal rows). The cluster assignment for each sample is provided in Table S4, and a consensus value matrix for the subtype comparisons is shown in\",\n",
            "          \"consensus value matrix for the subtype comparisons is shown in Figure S2A. The results of a Weighted Gene Co-expression Network Analysis (WGCNA) of protein functional enrichment by subgroups are provided for the protein clusters, in Figure S2B and Table\",\n",
            "          \"are provided for the protein clusters, in Figure S2B and Table S4. The enrichment of KEGG and Reactome ontologies in the WGCNA-derived modules are shown in Figures S2C and S2D, respectively; membership of enriched pathways is provided in Table S3. Four of\",\n",
            "          \"of enriched pathways is provided in Table S3. Four of the proteomic clusters showed a clear correspondence to the mesenchymal, proliferative, immunoreactive, and differentiated subtypes defined by the TCGA transcriptome analysis (Figure 2 and Figure S2E).\",\n",
            "          \"by the TCGA transcriptome analysis (Figure 2 and Figure S2E). A relatively small fifth cluster of tumors significantly enriched in proteins associated with extracellular matrix interactions, erythrocyte and platelet functions, and the complement cascade\",\n",
            "          \"erythrocyte and platelet functions, and the complement cascade was also observed using multiple approaches, including model based clustering with Bayesian information criteria, consensus clustering, and VISDA-based sub-phenotype clustering. This new group\",\n",
            "          \"and VISDA-based sub-phenotype clustering. This new group could not be attributed to tissue source site sampling bias or any other meta-data category, but may be related to tumor characteristics, including vascularization and tumor content, as the Tumor\",\n",
            "          \"including vascularization and tumor content, as the Tumor Purity score for this subtype (and the Mesenchymal subtype) was significantly lower than that of the other three subtypes (Figure S2F). The clinical relevance of these protein-based clusters will\",\n",
            "          \"The clinical relevance of these protein-based clusters will require validation in independent HGSC sample sets, particularly as no significant difference in survival was observed (Figure S2G), similar to mRNA-based clustering analysis. Proteomic analysis\",\n",
            "          \"similar to mRNA-based clustering analysis. Proteomic analysis of CNA effects Chromosomal instability, marked by extensive copy number alterations (CNAs) in each tumor, is a hallmark of HGSC, and a likely source of driver alterations in this disease. CNAs\",\n",
            "          \"and a likely source of driver alterations in this disease. CNAs can affect the abundance of proteins at the same locus (cis effects), and may also act in trans, either directly or indirectly. Hypothesizing that CNAs with strong trans effects are more\",\n",
            "          \"Hypothesizing that CNAs with strong trans effects are more likely to elicit a molecular phenotype and confer selective advantages, we sought to identify those CNA regions that have the broadest effect on protein expression. In all, 29,393 CNA loci (Table\",\n",
            "          \"effect on protein expression. In all, 29,393 CNA loci (Table S5) were compared to our global proteomics data with 950,209 CNA/protein pairs (0.72% of the total) exhibiting significant association (Benjamini-Hochberg adjusted p value <0.01). We provide a\",\n",
            "          \"(Benjamini-Hochberg adjusted p value <0.01). We provide a complete list of the significantly associated proteins for each CNA in Table S5. The diagonal line evident in Figure 3 corresponds to cis effects, and vertical stripes correspond to trans effects\",\n",
            "          \"cis effects, and vertical stripes correspond to trans effects where changes in copy number affect expression of numerous proteins across the genome. We performed the same analysis for mRNA to identify sites where associations are transcriptionally\",\n",
            "          \"mRNA to identify sites where associations are transcriptionally mediated. A similar number of CNA/mRNA pairs were found to be significantly associated (1,113,164 at Benjamini-Hochberg adjusted p value <0.01; Figure 3). This analysis revealed regions on\",\n",
            "          \"p value <0.01; Figure 3). This analysis revealed regions on chromosomes 2, 7, 20, and 22 correlated in trans with more than 200 proteins. In contrast to colorectal cancer where most of the trans-regulation of protein by CNA was accompanied by similar\",\n",
            "          \"trans-regulation of protein by CNA was accompanied by similar changes in mRNA, we observed several loci associated with differences in protein abundance without a corresponding change in mRNA. For example, large regions on chromosome 2 have relatively\",\n",
            "          \"For example, large regions on chromosome 2 have relatively little trans effect on mRNA levels, but are associated with more than 200 proteins in trans. Dissecting the mechanisms by which a specific CNA can alter protein levels in trans, without affecting\",\n",
            "          \"CNA can alter protein levels in trans, without affecting the corresponding mRNA is difficult, given the extent of the amplified or deleted regions and the numerous genes affected at a given locus. Possible mechanisms include cis regulation of proteins\",\n",
            "          \"locus. Possible mechanisms include cis regulation of proteins associated with mRNA stability and translational efficiency, such as microRNAs and RNA binding proteins. Given the complex pattern of CNAs observed in HGSC, it has been difficult to identify a\",\n",
            "          \"of CNAs observed in HGSC, it has been difficult to identify a limited number of high impact genomic alterations that could function as drivers of the disease. We interrogated the trans-affected proteins associated with each putative CNA for common\",\n",
            "          \"proteins associated with each putative CNA for common functions using pathways defined by the KEGG, NCI pathway interaction database (PID) and Reactome databases (Supplemental Information). Proteins associated with cell invasion and migration, and\",\n",
            "          \"Proteins associated with cell invasion and migration, and proteins related to immune function appeared to be enriched in association with multiple CNAs (Figure S3; Table S3). These observations suggest a convergence of multiple CNA targets on a common set\",\n",
            "          \"suggest a convergence of multiple CNA targets on a common set of biological functions, namely motility/invasion and immune regulation, functions that are among the hallmarks of cancer. Association of CNA trans-affected proteins with overall survival\",\n",
            "          \"of CNA trans-affected proteins with overall survival Availability of the TCGA survival data allowed us to use trans-affected protein data from the most influential CNAs (e.g., the four altered regions described on chromosomes 2, 7, 20, and 22; Figure 3)\",\n",
            "          \"regions described on chromosomes 2, 7, 20, and 22; Figure 3) to build a model of overall survival. Because each CNA affects many proteins, we used a regression approach that identifies parsimonious Cox proportional hazards models with maximal predictive\",\n",
            "          \"Cox proportional hazards models with maximal predictive ability from the list of significantly correlated proteins for that CNA. We trained models on the proteomics data from PNNL (82 tumors) and tested the ability of the model to predict survival times\",\n",
            "          \"and tested the ability of the model to predict survival times in the data from JHU (87 tumors, not including the 32 overlapping). Each of the four most influential CNA regions produced models that were strongly associated with patient survival (p value\",\n",
            "          \"that were strongly associated with patient survival (p value <0.01, FDR <0.5% based on randomly selected proteins). A Kaplan-Meier plot illustrating the predictive value of each of the locus-specific models after validation is shown in Figure S4A; for the\",\n",
            "          \"models after validation is shown in Figure S4A; for the Kaplan-Meier plots, \\u2018high\\u2019 and \\u2018low\\u2019 expression were defined relative to the median for that signature across all samples, splitting patients into two equal groups of 45 (Table S6). We examined the\",\n",
            "          \"into two equal groups of 45 (Table S6). We examined the overlap in predictions for patients for the four signatures and found that the predictions were unanimous (either high or low signature) for 62% of patients and 16% were evenly split, with the\",\n",
            "          \"for 62% of patients and 16% were evenly split, with the remaining having one dissenting signature. This suggested the utility of combining these signatures, using a voting method where the number of high or low calls was counted for each patient; this\",\n",
            "          \"number of high or low calls was counted for each patient; this substantially improved prediction of survival time (p value 1.9e-6; Figure 4). We also examined the effects of tumor stage, tumor grade, patient age, surgical outcome, and platinum status, and\",\n",
            "          \"grade, patient age, surgical outcome, and platinum status, and found that our proteomic signatures were not improved by inclusion of these variables. For each of these locus-specific models we analyzed the enrichment of genes and their regulatory\",\n",
            "          \"models we analyzed the enrichment of genes and their regulatory sequences associated with outcome, and found all four models had genes significantly enriched (Benjamini-Hochberg adjusted p value <0.05) in binding sites for the proliferation-associated\",\n",
            "          \"value <0.05) in binding sites for the proliferation-associated serum response factor (SRF), suggesting that SRF activity may be important in ovarian cancer outcome. Additionally, there was significant enrichment of proteins involved in the regulation of\",\n",
            "          \"enrichment of proteins involved in the regulation of actin cytoskeleton, apoptosis, and adherens junction (Benjamini-Hochberg adjusted p value <0.05). We examined the protein abundance and phosphorylation status of SRF between short and long surviving\",\n",
            "          \"phosphorylation status of SRF between short and long surviving groups and observed that they were higher in short surviving patients, but only slightly. Thus, although SRF alone was not predictive, the trends in SRF abundance and phosphorylation are\",\n",
            "          \"predictive, the trends in SRF abundance and phosphorylation are consistent with the observation of enrichment for proteins potentially regulated by SRF binding. Details of the larger gene set analysis are provided in Supplemental Information. Several\",\n",
            "          \"set analysis are provided in Supplemental Information. Several proteins shared across all the signatures are known to be involved in cancer processes. Catenin B2 (CTNNA2) is a cell-cell adhesion protein and tumor suppressor. The Rho GDP dissociation\",\n",
            "          \"adhesion protein and tumor suppressor. The Rho GDP dissociation inhibitor (GDI) beta (ARHGDIB) is involved in invasion and migration in many cancers, and overexpression correlates with progression in pancreatic carcinoma. Protein kinase C and casein\",\n",
            "          \"in pancreatic carcinoma. Protein kinase C and casein kinase substrate in neurons protein 2 (PACSIN2) is a repressor of cellular migration. Finally, the GTP-binding nuclear protein Ran, is a prognostic marker associated with increased survival in\",\n",
            "          \"is a prognostic marker associated with increased survival in epithelial ovarian cancer. The association of these previously described survival-related proteins with genes affected in trans by CNAs suggests a potential mechanism for the parallel activation\",\n",
            "          \"CNAs suggests a potential mechanism for the parallel activation of multiple pathways associated with poor prognosis in HGSC. As a comparison we applied the previously described Provar signature, comprising five protein and four phosphoproteins that showed\",\n",
            "          \"comprising five protein and four phosphoproteins that showed good survival prediction in the TCGA ovarian cancer dataset. We observed all proteins in the Provar signature in our proteomic data, but only one of the phosphosites (EGFR Y1173). Thus, we used\",\n",
            "          \"but only one of the phosphosites (EGFR Y1173). Thus, we used the RPPA data from the original signature (Figure S4B). We found that the Provar signature was prognostic of survival (Benjamini-Hochberg adjusted p value = 0.11) in the 67 patients examined\",\n",
            "          \"adjusted p value = 0.11) in the 67 patients examined here (those with phosphoprotein data and had somatic TP53 mutations), but the statistical power of Provar is not as high as the statistical power of signatures derived from the CNA trans-affected\",\n",
            "          \"power of signatures derived from the CNA trans-affected proteins in this dataset. In addition, integrating the Provar signature with the CNA signatures did not improve survival prediction (Figure S4C). Identification of proteins associated with\",\n",
            "          \"(Figure S4C). Identification of proteins associated with chromosomal structural abnormality The degree of chromosomal instability exhibited by a tumor can be represented by a calculated chromosome instability (CIN) index, as described previously for lung\",\n",
            "          \"instability (CIN) index, as described previously for lung cancer. Identification of proteins associated with CIN may provide information on the processes contributing to chromosomal instability, while the analysis of trans-affected proteins described\",\n",
            "          \"while the analysis of trans-affected proteins described above is more closely related to the downstream consequences of specific CNAs. Using a bootstrapping method described in Supplemental Information, we identified a proteomic signature including 128\",\n",
            "          \"Information, we identified a proteomic signature including 128 proteins (Table S7) showing significant correlation with CIN index (Benjamini-Hochberg adjusted p value <1e-4, Spearman correlation; Figure S5A). Functional annotation of these proteins\",\n",
            "          \"Figure S5A). Functional annotation of these proteins (Figure S5B) showed that proteins up-regulated in tumors with high CIN index were preferentially involved in chromatin organization (p value 6.90e-5), whereas proteins up-regulated in tumors with low\",\n",
            "          \"6.90e-5), whereas proteins up-regulated in tumors with low CIN index were more often associated with cell death (p value 2.13e-5). Correlation analysis of protein abundances and CIN indicated that a small number of proteins could account for the majority\",\n",
            "          \"that a small number of proteins could account for the majority of the CIN (Figure 5); two of the most strongly associated proteins, CHD4 and CHD5, are known to be involved in chromatin organization. Identification of proteins associated with HRD status\",\n",
            "          \"Identification of proteins associated with HRD status Since HRD is associated with susceptibility to PARP inhibitors and improved survival, we sought to elucidate systemic changes associated with HRD and identify biomarkers that might be used to stratify\",\n",
            "          \"with HRD and identify biomarkers that might be used to stratify patients for treatment. We defined tumors with HRD as having either germline or somatic mutations in BRCA1 or BRCA2, or BRCA1 promoter methylation, or homozygous deletion of PTEN, with an\",\n",
            "          \"promoter methylation, or homozygous deletion of PTEN, with an overall survival >1.5 years, while non-HRD patients were defined as lacking these genomic aberrations and with a follow-up or time-to-death <2.5 years; additional selection criteria include\",\n",
            "          \"time-to-death <2.5 years; additional selection criteria include available residual tissue volume and a tumor tissue contamination score estimated using copy number alterations. Applying differential dependency network (DDN) analysis on a set of 171 BRCA1-\",\n",
            "          \"dependency network (DDN) analysis on a set of 171 BRCA1- or BRCA2-related proteins curated from the literature and from the cBio portal, we identified a sub-network of 30 proteins that displayed co-expression patterns differentiating HRD from non-HRD\",\n",
            "          \"co-expression patterns differentiating HRD from non-HRD patients (Table S7; Figure 6). Several of the proteins in these modules are known to be involved in histone acetylation or deacetylation, e.g., HDAC1, RBBP4, RBBP7, and EP300, and HUS1. Although\",\n",
            "          \"e.g., HDAC1, RBBP4, RBBP7, and EP300, and HUS1. Although statistical association cannot distinguish between drivers and consequences of HRD status, the observed enrichment of proteins associated with histone acetylation motivated us to use an effective\",\n",
            "          \"with histone acetylation motivated us to use an effective database search strategy (Supplemental Information) to identify and quantify acetylated peptides from the global proteomic data. Comparative analysis of 399 acetylated peptides identified 15\",\n",
            "          \"Comparative analysis of 399 acetylated peptides identified 15 acetylated peptides with significant differences between the HRD and non-HRD samples (Table S7). Among these, dual acetylation at K12 and K16 of histone H4 showed a significant difference\",\n",
            "          \"at K12 and K16 of histone H4 showed a significant difference between HRD and non-HRD samples (Figure 6); differential acetylation of K12 and K16 was verified using synthetic peptides and targeted analysis using Sequential Windowed data independent\",\n",
            "          \"targeted analysis using Sequential Windowed data independent Acquisition of the Total High-resolution Mass Spectra (SWATH-MS). In cell line data, acetylation of H4 has previously been reported to be involved in the choice of DNA double strand break (DSB)\",\n",
            "          \"to be involved in the choice of DNA double strand break (DSB) repair pathways (homologous recombination or non-homologous end joining. This relationship is regulated partially by HDAC1, a protein also identified in the DDN analysis. We observed a\",\n",
            "          \"a protein also identified in the DDN analysis. We observed a significant enrichment of HDAC1 and its co-regulated proteins in tumors with HRD and low H4 acetylation relative to non-HRD tumors with high H4 acetylation (DDN analysis: permutation tests, p\",\n",
            "          \"with high H4 acetylation (DDN analysis: permutation tests, p value <0.05; differentially acetylated peptides: t-tests, p value <0.05 with an estimated FDR <0.5% by bootstrap/permutation tests). The combined observations of increased HDAC1 and associated\",\n",
            "          \"The combined observations of increased HDAC1 and associated proteins at the pathway level, together with decreased acetylation of H4 in HRD patients at the PTM level, provide insight regarding the potential role of HDAC1 in modulating the choice of DSB\",\n",
            "          \"the potential role of HDAC1 in modulating the choice of DSB repair pathways. Phosphoproteomic analysis of pathways associated with survival An initial set of 24,464 different phosphopeptides (21,298 unique phosphorylation sites) contained within 4,420\",\n",
            "          \"(21,298 unique phosphorylation sites) contained within 4,420 proteins having data on net differences in phosphorylation (Table S2) was obtained. Since ischemia of the TCGA tumor samples may alter phosphopeptide abundance, we removed any phosphorylation\",\n",
            "          \"alter phosphopeptide abundance, we removed any phosphorylation sites previously shown to be altered by ischemia. We also removed three tumor samples having substantially higher than average missing values and two tumor samples lacking somatic TP53\",\n",
            "          \"missing values and two tumor samples lacking somatic TP53 mutations (Supplemental Information). This yielded a final set of 7,675 different phosphopeptides (6,802 unique phosphorylation sites) from 2,324 proteins (Table S2) that were mapped to pathways\",\n",
            "          \"from 2,324 proteins (Table S2) that were mapped to pathways via the NCI PID. The average net phosphorylation of all phosphopeptides mapping to a given pathway (i.e., the statistical average across all phosphoproteins known to be in the pathway, rather\",\n",
            "          \"across all phosphoproteins known to be in the pathway, rather than the statistically significant individual proteins) was used as a measure of pathway activity and compared between short survivors (deceased surviving <3 years; N=17) and long survivors\",\n",
            "          \"(deceased surviving <3 years; N=17) and long survivors (patients surviving >5 years; N=19) using both proteomic and phosphoproteomic data. An issue in ranking the pathways is the distinction between phosphopeptides that are unique to a single pathway,\",\n",
            "          \"between phosphopeptides that are unique to a single pathway, such as the receptor peptides themselves, vs. peptides that map to proteins shared between pathways. In order to address this issue, we performed two analyses: one of all proteins associated\",\n",
            "          \"we performed two analyses: one of all proteins associated with a specific pathway in the NCID database, and a second analysis in which proteins common to multiple pathways were excluded. Figure 7A shows a ranking of activated pathways, as inferred from\",\n",
            "          \"7A shows a ranking of activated pathways, as inferred from the analysis of pathway specific components (i.e., differences in mRNA, protein and phosphoprotein levels) in short vs. long survivors. Fifteen pathways showed increased phosphorylation at\",\n",
            "          \"survivors. Fifteen pathways showed increased phosphorylation at Benjamini-Hochberg adjusted p values <0.01, in contrast with transcriptional data for the same samples which yielded only one statistically significantly increased pathway, androgen receptor\",\n",
            "          \"significantly increased pathway, androgen receptor signaling, which was also increased at the phosphoprotein level (Figure 7A and Supplemental Information). All pathways indicated in Figure 7A were increased in association with short survival relative to\",\n",
            "          \"were increased in association with short survival relative to long survival, though some individual components within a pathway showed opposite changes (e.g. JNK1 transcript and protein levels, see Figure 7B). The RhoA-regulatory, PDGFRB, and\",\n",
            "          \"levels, see Figure 7B). The RhoA-regulatory, PDGFRB, and integrin-like kinase pathways emerged as the most activated in this analysis (Figure 7A). Figure 7B shows the common and unique elements of the PDGFRB pathway, and the trends in mRNA, protein, and\",\n",
            "          \"of the PDGFRB pathway, and the trends in mRNA, protein, and phosphoprotein levels in the comparison of short vs. long survivors, illustrating the benefit of combined protein and phosphoprotein measurements, as well as the substantial contribution of\",\n",
            "          \"measurements, as well as the substantial contribution of protein phosphorylation data to the overall analysis. We compared our results to the 31 phosphoproteins represented on the RPPA arrays used in the original TCGA ovarian cancer analysis; RPPA\",\n",
            "          \"arrays used in the original TCGA ovarian cancer analysis; RPPA identified differential phosphorylation of ERK1, RAF1, and STAT3 in the same patient samples analyzed by MS. For short surviving vs. long surviving patients, ERK1 showed statistically\",\n",
            "          \"vs. long surviving patients, ERK1 showed statistically significantly increased phosphorylation by RPPA, on the same ERK1 phosphopeptides as found using MS, while AKT1, RAF1 and STAT3 showed increased phosphorylation by both RPPA and MS-based analyses, but\",\n",
            "          \"phosphorylation by both RPPA and MS-based analyses, but based upon distinct phosphopeptides (Table S2). Good overlap was observed between RPPA and MS results for phosphoproteins on the RPPA array, but we did not observe three of the specific phosphosites\",\n",
            "          \"but we did not observe three of the specific phosphosites from the Provar survival signature in our MS data, suggesting that proteomics is more sensitive to overall pathway activity than specific regulatory events in the pathway itself. However, many of\",\n",
            "          \"regulatory events in the pathway itself. However, many of the phosphopeptides identified by MS phosphoproteomics as components of the PDGFR-beta pathway were not represented in the RPPA, illustrating the ability to identify phosphorylation events not\",\n",
            "          \"illustrating the ability to identify phosphorylation events not readily accessible with currently available antibodies (Figure 7B). DISCUSSION The addition of proteomic information, including post-translational modifications, to the rich genomic and\",\n",
            "          \"post-translational modifications, to the rich genomic and transcriptomic data available from the ovarian HGSC samples analyzed by TCGA has provided additional insights into the biology of HGSC, including identification of changes at the level of pathway\",\n",
            "          \"including identification of changes at the level of pathway activation. Integration of genomic, proteomic, and phosphoproteomic measurements identified differentially regulated pathways and functional modules that displayed significant associations with\",\n",
            "          \"functional modules that displayed significant associations with patient outcomes, including survival and HRD status. Cox proportional hazard analysis of proteins associated with CNAs identified overall survival signatures enriched for targets of the\",\n",
            "          \"overall survival signatures enriched for targets of the proliferation-associated transcription factor SRF, and integrated proteomic, phosphoproteomic and transcriptomic data identified pathways that differentiated patients on the basis of survival,\",\n",
            "          \"pathways that differentiated patients on the basis of survival, including the PDGFR-beta signaling pathway associated with angiogenesis, the RhoA regulatory and integrin-linked kinase pathways associated with cell mobility and invasion, and pathways\",\n",
            "          \"associated with cell mobility and invasion, and pathways associated with chemokine signaling and adaptive immunity. Proteins associated with cell invasion and motility emerged from both the integration of CNAs with protein abundance data and the\",\n",
            "          \"the integration of CNAs with protein abundance data and the phosphoproteomic investigation of up-regulated pathways in short versus long survivors. Despite the complexity of genomic alterations that characterize HGSC, these analyses suggest a functional\",\n",
            "          \"that characterize HGSC, these analyses suggest a functional convergence on a subset of key pathways. The association of increased invasiveness and motility with short overall survival in this study may help to explain more aggressive mechanisms of\",\n",
            "          \"in this study may help to explain more aggressive mechanisms of dissemination in ovarian cancer, including recently reported hematogenous metastasis, in addition to previously described lymphatic spread and known peritoneal spread by direct extension.\",\n",
            "          \"spread and known peritoneal spread by direct extension. Focusing on functional modules has also revealed potential drivers of HGSC and more robust signatures for potentially stratifying ovarian HGSC patients into distinct cancer phenotypes to inform\",\n",
            "          \"ovarian HGSC patients into distinct cancer phenotypes to inform therapeutic management. For example, the identification of specific acetylation events associated with HRD, e.g., the simultaneous acetylation of K12 and K16 on histone H4, may provide an\",\n",
            "          \"acetylation of K12 and K16 on histone H4, may provide an alternative biomarker of HRD and a rationale for the selection of patients in future clinical trials of HDAC inhibitors alone or in combination with PARP inhibition. This may help to resolve the\",\n",
            "          \"combination with PARP inhibition. This may help to resolve the current discrepancy between the initial observation of limited single agent activity of HDAC inhibitors in ovarian cancer, and more recent findings of a >40% response rate when used in\",\n",
            "          \"and more recent findings of a >40% response rate when used in combination with cytotoxic chemotherapy in platinum-resistant patients. Similarly, comprehensive interrogation of protein phosphorylation identified multiple pathways with significantly\",\n",
            "          \"phosphorylation identified multiple pathways with significantly increased phosphorylation in patients with poor clinical outcomes. Specifically, the observed activation of PDGFR pathways in a subset of patients with short overall survival could\",\n",
            "          \"in a subset of patients with short overall survival could potentially stratify patients for selective enrollment in trials of anti-angiogenic therapy, which is particularly relevant to current controversies over the use of bevacuzimab as a first-line\",\n",
            "          \"controversies over the use of bevacuzimab as a first-line therapy in ovarian cancer. Overall, this work illustrates the ability of proteomics to complement genomics in providing additional insights into pathways and processes that drive ovarian cancer\",\n",
            "          \"insights into pathways and processes that drive ovarian cancer biology and how these pathways are altered in correspondence with clinical phenotypes. The comprehensive proteomic measurements for the HGSC tumor samples provide a public resource of\",\n",
            "          \"for the HGSC tumor samples provide a public resource of information. Importantly, the ability to identify PTMs revealed a strong association between specific histone acetylation events and the HRD phenotype. In addition, the enhanced view of pathway\",\n",
            "          \"the HRD phenotype. In addition, the enhanced view of pathway activity provided by measurements of protein phosphorylation provides a foundation for linking genotype to proteotype, and ultimately to phenotype for understanding the molecular basis of\",\n",
            "          \"to phenotype for understanding the molecular basis of cancer. EXPERIMENTAL PROCEDURES Full methods are available in Extended Experimental Procedures. Tumor Samples All tumor samples for the current study were obtained through the TCGA Biospecimen Core\",\n",
            "          \"current study were obtained through the TCGA Biospecimen Core Resource as described previously. Samples were selected based on overall survival, HRD status and availability. Quantitative Proteomics Tissue proteins were extracted and digested with trypsin;\",\n",
            "          \"Tissue proteins were extracted and digested with trypsin; at each site, a portion of each resulting sample was also used to create a pooled reference in which each tumor sample contributed an equal percentage by peptide mass. The pooled reference sample\",\n",
            "          \"equal percentage by peptide mass. The pooled reference sample was included in each multiplexed, isobaric labeling experiment to enable cross-experiment comparison in the entire sample cohort. The patient samples and the pooled reference sample were each\",\n",
            "          \"The patient samples and the pooled reference sample were each labeled by different iTRAQ reagents (Sciex), combined, fractionated, and split for integrated, quantitative global proteome (10%) and phosphoproteome (90%) analysis using LC-MS/MS on an\",\n",
            "          \"(10%) and phosphoproteome (90%) analysis using LC-MS/MS on an Orbitrap Velos mass spectrometer (Thermo Scientific). Raw data were processed for peptide identification by database searching and identified peptides were assembled as proteins and mapped to\",\n",
            "          \"identified peptides were assembled as proteins and mapped to gene identifiers for proteogenomic comparisons. Quantitation was achieved by comparing the iTRAQ reporter ion intensities in each sample. Comparison of mRNA and Protein Subtypes A model-based\",\n",
            "          \"sample. Comparison of mRNA and Protein Subtypes A model-based clustering approach was used to model the protein abundance data as a mixture of subtypes. Bayesian information criteria, statistical resampling and VISDA-based sub-phenotype clustering\",\n",
            "          \"statistical resampling and VISDA-based sub-phenotype clustering approaches provided similar results with a stable optimization on five clusters as determined by consensus clustering. WGCNA analysis was performed to infer genes or gene networks that drive\",\n",
            "          \"was performed to infer genes or gene networks that drive subtyping into 5 clusters, followed by correlation with subtype as a trait. Integration of Proteomics and CNA data Spearman correlation coefficients and corresponding adjusted p values were\",\n",
            "          \"coefficients and corresponding adjusted p values were calculated for each protein/transcript by CNA locus, and gene set enrichment analysis was used to infer function for groups of proteins significantly correlated with a given CNA locus. Regression\",\n",
            "          \"significantly correlated with a given CNA locus. Regression analysis was applied to the list of trans-affected protein correlated with each CNA to generate parsimonious Cox proportional hazards models with maximal predictive ability, using the PNNL data\",\n",
            "          \"models with maximal predictive ability, using the PNNL data for training and JHU data for testing. Kaplan-Meier plots were used to visualize performance in predicting overall survival and progression-free survival. A CIN index was calculated for each\",\n",
            "          \"progression-free survival. A CIN index was calculated for each sample as the mean absolute values of copy number measurements at the 29,393 selected loci. Bootstrap resampling was used to select proteins correlated with CIN index at high confidence.\",\n",
            "          \"select proteins correlated with CIN index at high confidence. Protein Acetylation and HRD Acetylated peptides were identified by searching the global proteomics data for dynamic acetylation to lysines (+42 Da). Acetylation levels were compared between HRD\",\n",
            "          \"lysines (+42 Da). Acetylation levels were compared between HRD and non-HRD cases by t-test. Targeted proteomics (SWATH) was used to orthogonally quantify the acetylated peptide with synthetic peptides as internal standard. Survival-related Pathways\",\n",
            "          \"peptides as internal standard. Survival-related Pathways Analysis Phosphoproteome data from the short and long survivors were mapped to signaling pathways in the NCI pathway information database,(PID (http://pid.nci.nih.gov) using the gene names. For each\",\n",
            "          \"(http://pid.nci.nih.gov) using the gene names. For each signaling pathway in the PID, relative abundances for all phosphopeptides mapping to any pathway component were identified and separated into short and long survivor groups. The difference in\",\n",
            "          \"into short and long survivor groups. The difference in distributions between the two sets of pathway-specific peptides, i.e., those associated with either short survivors or long long survivors, was then assessed using a two-tailed t test. Similar\",\n",
            "          \"survivors, was then assessed using a two-tailed t test. Similar enrichment analyses were also performed using protein abundance, mRNA abundance, and CNA. Data Repository All of the primary mass spectrometry data on TCGA tumor samples are deposited at the\",\n",
            "          \"spectrometry data on TCGA tumor samples are deposited at the CPTAC Data Coordinating Center as raw and mzML files and complete protein assembly data sets for public access (https://cptac-data-portal.georgetown.edu). Supplementary Material This is a PDF\",\n",
            "          \"Supplementary Material This is a PDF file of an unedited manuscript that has been accepted for publication. As a service to our customers we are providing this early version of the manuscript. The manuscript will undergo copyediting, typesetting, and\",\n",
            "          \"The manuscript will undergo copyediting, typesetting, and review of the resulting proof before it is published in its final form. Please note that during the production process errors may be discovered which could affect the content, and all legal\",\n",
            "          \"may be discovered which could affect the content, and all legal disclaimers that apply to the journal pertain.            Author Contributions          Study conception and design: HZ, TL, ZZ, RDS, DWC, KDR. Investigation: performed experiment or data\",\n",
            "          \"ZZ, RDS, DWC, KDR. Investigation: performed experiment or data collection: TL, SS, FY, JZ, LijunC, PS, PA, YT, MAG, TRC, CC, ST, SN, RJM, HengZ. Computation and statistical analysis: ZZ, SHP, BaiZ, JEM, VAP, LiC, DR, MEM, SWC, SW, JW, DLT, DF, VB, YW,\",\n",
            "          \"BaiZ, JEM, VAP, LiC, DR, MEM, SWC, SW, JW, DLT, DF, VB, YW, BingZ. Data interpretation and biological analysis: TL, HZ, SHP, ZZ, BaiZ, JEM, VAP, LiC, DR, SN, CW, IMS, AP, MPS, DAL, RDS, DWC, KDR. Writing/manuscript preparation and revision: HZ, TL, ZZ,\",\n",
            "          \"KDR. Writing/manuscript preparation and revision: HZ, TL, ZZ, SHP, JEM, DR, VAP, LeslieC, HR, IMS, AP, MPS, DAL, RDS, DWC, KDR. Supervision and administration: HR, ESB, TH, RCR, LS, RDS, DWC, KDR. The authors declare no competing financial interests.\",\n",
            "          \"KDR. The authors declare no competing financial interests.            SUPPLEMENTAL INFORMATION          Supplemental information includes Extended Experimental Procedures, 5 figures, and 8 tables and can be found with this article online. Correlations\",\n",
            "          \"tables and can be found with this article online. Correlations between mRNA and protein abundance in TCGA tumors mRNA and protein were correlated across all the samples, resulting in 90.6% positive correlations, and 79.4% were significantly correlated\",\n",
            "          \"positive correlations, and 79.4% were significantly correlated (Benjamini-Hochberg adjusted p value <0.01), with a mean Spearman\\u2019s correlation of 0.38 and a median value of 0.45 (top panel). Different biological pathways and processes showed significantly\",\n",
            "          \"biological pathways and processes showed significantly different levels of correlation (bottom panel). Metabolic pathways and the interferon response displayed high mRNA-protein correlation, and ribosome, mRNA splicing, oxidative phosphorylation, and\",\n",
            "          \"and ribosome, mRNA splicing, oxidative phosphorylation, and complement and coagulation cascade were poorly correlated. The mean correlation is shown in parentheses followed by Benjamini-Hochberg adjusted p values calculated using a Kolmogorov\\u2013Smirnov test\",\n",
            "          \"adjusted p values calculated using a Kolmogorov\\u2013Smirnov test following the functional group names from MSIGDB. Blue bars indicate positive correlations and yellow indicates negative correlations; individual proteins (represented as bars on the x-axis) are\",\n",
            "          \"individual proteins (represented as bars on the x-axis) are sorted by correlation from low to high (bottom panel). Proteomic subtypes and corresponding driving protein modules Global proteomics abundance is shown in a heatmap with TCGA samples represented\",\n",
            "          \"abundance is shown in a heatmap with TCGA samples represented by columns ordered by protein subtype where rows represent proteins. Color of each cell indicates z-score (log2 of relative abundance scaled by proteins\\u2019 standard deviation) of the protein in\",\n",
            "          \"scaled by proteins\\u2019 standard deviation) of the protein in that sample; red is increased and blue is decreased (relative to the pooled reference). Transcriptome-based subtypes (Verhaak et al., 2013) and the proposed proteomic subtypes are indicated in\",\n",
            "          \"al., 2013) and the proposed proteomic subtypes are indicated in color above the heatmap. WGCNA-derived modules are delineated with the row color panel and annotated according to the pathways based on enrichment of KEGG and Reactome ontologies. Functional\",\n",
            "          \"based on enrichment of KEGG and Reactome ontologies. Functional impact of copy number alterations The top panel shows the correlation of CNAs to protein abundance (right) or mRNA (left) with significant positive correlations in red and negative\",\n",
            "          \"with significant positive correlations in red and negative correlations in blue (Benjamini-Hochberg adjusted p value <0.01, Spearman\\u2019s correlation coefficient). The x-axis plots the 29,393 CNAs obtained from TCGA. The y-axis plots 3,202 proteins. Genes\",\n",
            "          \"CNAs obtained from TCGA. The y-axis plots 3,202 proteins. Genes are ordered by chromosomal location on both the x- and y-axes. The bottom panel shows the summed number of significantly correlated proteins (or mRNA) for each individual CNA. In blue is\",\n",
            "          \"proteins (or mRNA) for each individual CNA. In blue is shown the total number, in black are those genes significantly correlated as both mRNA and protein. Where blue and black lines have similar magnitude, e.g., the hotspot on chromosome 20, the CNA\",\n",
            "          \"similar magnitude, e.g., the hotspot on chromosome 20, the CNA associations are shared between protein and mRNA. Where a strong blue line has no mirrored black line, e.g., the protein hotspot on chromosome 2, the associations with CNA are largely unique.\",\n",
            "          \"on chromosome 2, the associations with CNA are largely unique. Kaplan-Meier plot of overall survival stratified by CNA-derived signatures A survival plot is shown for a consensus of the four best signatures (see Figure S4A and Table S6) identified from\",\n",
            "          \"best signatures (see Figure S4A and Table S6) identified from analysis of proteins affected in trans by CNAs. Models were trained using a lasso-based Cox proportional hazards model on the samples from PNNL, and shown are the survival curves from these\",\n",
            "          \"samples from PNNL, and shown are the survival curves from these models applied to the non-overlapping data from the JHU analysis, with the up (red; above the median) and down (blue, below the median) signatures each representing 45 patients. A vote was\",\n",
            "          \"median) signatures each representing 45 patients. A vote was taken among the four most predictive signatures, and the results of this vote are shown. Probability of death is shown on the y-axis vs. survival time in years on the x-axis. Shaded ribbons\",\n",
            "          \"y-axis vs. survival time in years on the x-axis. Shaded ribbons denote 95% confidence intervals. Number of CNAs statistically explained by proteins significantly associated with CIN index A total of 128 proteins selected for their association with CIN\",\n",
            "          \"A total of 128 proteins selected for their association with CIN were arranged along the chromosomal locations of their corresponding genes (y-axis). The length of the light grey horizontal lines indicates the number of CNAs significantly correlated with\",\n",
            "          \"indicates the number of CNAs significantly correlated with the protein across all chromosomes except for the protein\\u2019s corresponding gene coding region (x-axis). The top-ranked proteins are annotated and highlighted with dark lines to show the\",\n",
            "          \"are annotated and highlighted with dark lines to show the bootstrap-estimated 95% confidence intervals. DDN analysis and lysine-acetylation analysis between HRD and non-HRD patients DDN analysis revealed a sub-network of proteins that displayed distinct\",\n",
            "          \"revealed a sub-network of proteins that displayed distinct co-expression patterns between HRD and non-HRD patients, where purple connections indicate protein correlations that exist only in HRD samples and blue connections indicate protein correlations\",\n",
            "          \"HRD samples and blue connections indicate protein correlations that exist only in non-HRD patients. The links between the nodes are drawn with two different thickness indicative of whether the connections meet the significance threshold of 0.05 (light\",\n",
            "          \"the connections meet the significance threshold of 0.05 (light lines) or 0.01 (bold lines). Proteins with blue dotted circles are known to be involved in histone acetylation or deacetylation. Further, identification and quantitation of lysine-acetylated\",\n",
            "          \"Further, identification and quantitation of lysine-acetylated peptides in global proteomics data showed that acetylation level at K12 and K16 of histone H4 are significantly different between HRD and non-HRD samples, suggesting a role of histone H4\",\n",
            "          \"HRD and non-HRD samples, suggesting a role of histone H4 acetylation (together with HDAC1) in modulating the choice of DSB repair mechanisms. Pathway analysis associated with patient survival A) Pathway components were statistically analyzed using a\",\n",
            "          \"A) Pathway components were statistically analyzed using a two-sided t test between samples from patients surviving <3 years (short survival) and patients surviving >5 years (long survival) for differences in CNA, mRNA expression, protein or\",\n",
            "          \"survival) for differences in CNA, mRNA expression, protein or phosphorylation abundance. All significant pathways for mRNA and protein are shown (Benjamini-Hochberg adjusted p value <0.05) and the most significant pathways for phosphoproteomics are\",\n",
            "          \"and the most significant pathways for phosphoproteomics are plotted on the x axis as the \\u2212log of the p value. Results show that phosphorylation provides additional information about the functional state of tumors. B) Simplified PDGFR-beta pathway showing\",\n",
            "          \"state of tumors. B) Simplified PDGFR-beta pathway showing differences between short and long survivor groups for protein abundance (p), transcript abundance (m), and phosphopeptide abundance (circled P).\"\n",
            "        ]\n",
            "      }\n",
            "    ]\n",
            "  },\n",
            "  \"PDC000115\": {\n",
            "    \"study_id\": \"bb076b33-57b8-11e8-b07a-00a098d917f8\",\n",
            "    \"pdc_study_id\": \"PDC000115\",\n",
            "    \"study_submitter_id\": \"TCGA OV Phosphoproteome S020-4\",\n",
            "    \"program_id\": \"10251935-5540-11e8-b664-00a098d917f8\",\n",
            "    \"project_id\": \"48af5040-5546-11e8-b664-00a098d917f8\",\n",
            "    \"study_name\": \"TCGA Ovarian PNNL Phosphoproteome Velos Qexactive\",\n",
            "    \"program_name\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"project_name\": \"CPTAC2 Retrospective\",\n",
            "    \"disease_type\": \"Other;Ovarian Serous Cystadenocarcinoma\",\n",
            "    \"primary_site\": \"Not Reported;Ovary\",\n",
            "    \"analytical_fraction\": \"Phosphoproteome\",\n",
            "    \"experiment_type\": \"iTRAQ4\",\n",
            "    \"embargo_date\": null,\n",
            "    \"cases_count\": 70,\n",
            "    \"aliquots_count\": 70,\n",
            "    \"protocol_id\": \"1c06bed3-3b26-11e9-9a07-0a80fada099c\",\n",
            "    \"protocol_submitter_id\": \"P-S020-4-1\",\n",
            "    \"program_submitter_id\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"protocol_name\": \"TCGA_Ovarian_PNNL_Phosphoproteome_Velos\",\n",
            "    \"protocol_date\": \"2013-08-31\",\n",
            "    \"publications\": [\n",
            "      {\n",
            "        \"publication_id\": \"44e2e658-89c7-11e8-bcf1-0a2705229b82\",\n",
            "        \"pubmed_id\": \"27372738\",\n",
            "        \"doi\": \"10.1016/j.cell.2016.05.069\",\n",
            "        \"title\": \"Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer\",\n",
            "        \"journal\": \"Cell\",\n",
            "        \"journal_url\": \"https://www.cell.com/cell/fulltext/S0092-8674(16)30673-0?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0092867416306730%3Fshowall%3Dtrue\",\n",
            "        \"year\": \"2016\",\n",
            "        \"abstract\": \"To provide a detailed analysis of the molecular components and underlying mechanisms associated with ovarian cancer, we performed a comprehensive mass-spectrometry-based proteomic characterization of 174 ovarian tumors previously analyzed by The Cancer Genome Atlas (TCGA), of which 169 were high-grade serous carcinomas (HGSCs). Integrating our proteomic measurements with the genomic data yielded a number of insights into disease, such as how different copy-number alternations influence the proteome, the proteins associated with chromosomal instability, the sets of signaling pathways that diverse genome rearrangements converge on, and the ones most associated with short overall survival. Specific protein acetylations associated with homologous recombination deficiency suggest a potential means for stratifying patients for therapy. In addition to providing a valuable resource, these findings provide a view of how the somatic genome drives the cancer proteome and associations between protein and post-translational modification levels and clinical outcomes in HGSC.\",\n",
            "        \"full_text\": [\n",
            "          \"Integrated proteogenomic characterization of human high grade serous ovarian cancer SUMMARY To provide a detailed analysis of the molecular components and underlying mechanisms associated with ovarian cancer, we performed a comprehensive mass\",\n",
            "          \"with ovarian cancer, we performed a comprehensive mass spectrometry-based proteomic characterization of 174 ovarian tumors previously analyzed by The Cancer Genome Atlas (TCGA), of which 169 were high-grade serous carcinomas (HGSC). Integrating our\",\n",
            "          \"169 were high-grade serous carcinomas (HGSC). Integrating our proteomic measurements with the genomic data yielded a number of insights into disease such as how different copy number alternations influence the proteome, the proteins associated with\",\n",
            "          \"influence the proteome, the proteins associated with chromosomal instability, the sets of signaling pathways that diverse genome rearrangements converge on, as well as the ones most associated with short overall survival. Specific protein acetylations\",\n",
            "          \"with short overall survival. Specific protein acetylations associated with homologous recombination deficiency suggest a potential means for stratifying patients for therapy. In addition to providing a valuable resource, these findings provide a view of\",\n",
            "          \"providing a valuable resource, these findings provide a view of how the somatic genome drives the cancer proteome and associations between protein and post-translational modification levels and clinical outcomes in HGSC. INTRODUCTION A comprehensive\",\n",
            "          \"and clinical outcomes in HGSC. INTRODUCTION A comprehensive molecular view of cancer is necessary for understanding the underlying mechanisms of disease, improving prognosis, and ultimately guiding treatment. The Cancer Genome Atlas (TCGA) conducted an\",\n",
            "          \"guiding treatment. The Cancer Genome Atlas (TCGA) conducted an extensive genomic and transcriptomic characterization of ovarian high-grade serous carcinoma (HGSC) aimed at defining the genomic landscape and aiding the development of targeted therapies for\",\n",
            "          \"landscape and aiding the development of targeted therapies for this highly lethal malignancy. Key findings from TCGA were: 1) the prevalent role of TP53 mutations, 2) extensive DNA copy alterations, 3) preliminary transcriptional signatures associated\",\n",
            "          \"3) preliminary transcriptional signatures associated with survival, 4) varied mechanisms of BRCA1/2 inactivation, and lastly, 5) CCNE1 aberrations. Subsequent analysis of genomic data from the TCGA consortium led to the refinement of the\",\n",
            "          \"data from the TCGA consortium led to the refinement of the transcript-defined signatures, improving the statistical association with patient outcome, and integrating microRNA and mRNA expression profiles associated with HGSC to identify candidate microRNA\",\n",
            "          \"profiles associated with HGSC to identify candidate microRNA targets. While the insights from genomic analyses are substantial, functions encoded in the genome are generally executed at the protein level, and are often further modulated by\",\n",
            "          \"at the protein level, and are often further modulated by post-translational modifications (PTMs). For example, TCGA used reverse phase protein array (RPPA) analysis of 172 proteins (including 31 phosphoproteins with phospho-specific antibodies) to\",\n",
            "          \"31 phosphoproteins with phospho-specific antibodies) to generate a signature associated with the risk of tumor recurrence. To obtain a more comprehensive assessment of the complex ovarian HGSC phenotype, the Clinical Proteomic Tumor Analysis Consortium\",\n",
            "          \"phenotype, the Clinical Proteomic Tumor Analysis Consortium (CPTAC) conducted an extensive mass spectrometry (MS)-based proteomic and phosphoproteomic characterization of HGSC tumors characterized by TCGA, providing quantitative measurements for a\",\n",
            "          \"by TCGA, providing quantitative measurements for a combined total of 9,600 proteins from 174 tumors (an average of 7,952 proteins per tumor), and a total of 24,429 phosphosites from 6,769 phosphoproteins in a subset of 69 tumors (an average of 7,677\",\n",
            "          \"phosphoproteins in a subset of 69 tumors (an average of 7,677 phosphosites per tumor). Our results provide insights on HGSC biology and correlate differences in protein and PTM levels with clinical outcome complementary to TCGA genomic analyses. RESULTS\",\n",
            "          \"outcome complementary to TCGA genomic analyses. RESULTS Proteomic analysis of TCGA HGSC samples HGSC biospecimens and clinical data from 174 patients collected by TCGA were analyzed at two independent CPTAC Centers, Johns Hopkins University (JHU) and\",\n",
            "          \"independent CPTAC Centers, Johns Hopkins University (JHU) and Pacific Northwest National Laboratory (PNNL); thirty-two samples were analyzed at both JHU and PNNL. Tumors were selected by examining the associated TCGA metadata to select tumors either 1) on\",\n",
            "          \"the associated TCGA metadata to select tumors either 1) on the basis of putative homologous recombination deficiency (HRD), defined by the presence of germline or somatic BRCA1 or BRCA2 mutations, BRCA1 promoter methylation, or homozygous deletion of PTEN\",\n",
            "          \"BRCA1 promoter methylation, or homozygous deletion of PTEN (122 samples; 67 classified as HRD, 55 as non-HRD by the above criteria; JHU), or 2) to maximize differences in overall survival (84 samples; PNNL). For selection purposes, short survival was\",\n",
            "          \"(84 samples; PNNL). For selection purposes, short survival was defined as overall survival of less than 3 years, and long survival as greater than 5 years. All but five tumors had somatic TP53 mutations, a characteristic feature of HGSC; these five tumors\",\n",
            "          \"mutations, a characteristic feature of HGSC; these five tumors were subsequently reclassified as other than HGSC, and removed from protein functional analyses (e.g., subtyping and survival analyses). The tumor selection criteria and the associated\",\n",
            "          \"analyses). The tumor selection criteria and the associated metadata are provided in the Supplemental Information Table S1, available online. Proteomics measurements used isobaric tags for relative and absolute quantitation (iTRAQ) in conjunction with\",\n",
            "          \"relative and absolute quantitation (iTRAQ) in conjunction with offline liquid chromatography fractionation via high-pH reversed phase liquid chromatography (RPLC) and online RPLC with high-resolution tandem MS to provide broad coverage for peptide and\",\n",
            "          \"tandem MS to provide broad coverage for peptide and protein identification and quantification (Supplemental Information); this also alleviated quantitative interference potentially associated with the use of isobaric tags. We used the relative abundance\",\n",
            "          \"with the use of isobaric tags. We used the relative abundance measurements for each protein in the 32 patient samples analyzed at both JHU and PNNL to normalize across the two analysis sites, and then used clustering, principle component analysis (PCA),\",\n",
            "          \"and then used clustering, principle component analysis (PCA), and statistical tests to identify any significant batch effects associated with the site of analysis (a detailed comparison of within site, between site, between sample measurement variability\",\n",
            "          \"site, between site, between sample measurement variability and the process used to merge the JHU and PNNL data is given in Supplemental Information, Figure S1). As shown in Figure S1C, the median coefficient of variation (CV) between measurements at the\",\n",
            "          \"coefficient of variation (CV) between measurements at the two sites was 16%. A total of 9,600 proteins were identified with high confidence in all tumors, and the relative abundances in each tumor are given in Table S2. Functional analyses and\",\n",
            "          \"in each tumor are given in Table S2. Functional analyses and proteome-transcriptome associations were restricted to 3,586 proteins observed and quantified in all 169 HGSC samples used for protein functional analyses and where sample variability (signal)\",\n",
            "          \"functional analyses and where sample variability (signal) exceeded technical variability (noise) in the merged data (Table S2), calculated as described in Supplemental Information. On average we identify peptides covering 29% of the amino acids in any of\",\n",
            "          \"we identify peptides covering 29% of the amino acids in any of these 3,586 proteins in a given sample, with a range from 10% to 47%. In addition to protein abundance levels, phosphoproteomics data were acquired for 69 tumors with sufficient sample (Table\",\n",
            "          \"data were acquired for 69 tumors with sufficient sample (Table S2). As with proteins, phosphopeptide abundances were calculated relative to the pooled reference sample. Because isobaric labeling was performed prior to splitting the samples for proteome\",\n",
            "          \"was performed prior to splitting the samples for proteome and phosphoproteome analyses, phosphopeptide abundance could be corrected for changes in parent protein abundance to identify differences in the relative extent of phosphorylation at specific sites\",\n",
            "          \"in the relative extent of phosphorylation at specific sites for each protein (Table S2). Overall, we achieved a protein dynamic range encompassing >4 orders of magnitude, ranging from low-level transcription factors to abundant structural proteins i.e.,\",\n",
            "          \"transcription factors to abundant structural proteins i.e., actin and tubulin. Proteogenomic landscape of HGSC The degree to which alterations observed at the genome and transcriptome levels are manifested at the protein level is variable, both\",\n",
            "          \"levels are manifested at the protein level is variable, both qualitatively and quantitatively, and partially driven by multiple levels of post-transcriptional regulation. To identify peptides encoded by single amino acid variants (SAAVs), splice variants,\",\n",
            "          \"encoded by single amino acid variants (SAAVs), splice variants, and novel exons documented by TCGA, mass spectra were searched against a custom graph database containing all peptide variations projected from the TCGA genomic analyses of the cohort using a\",\n",
            "          \"projected from the TCGA genomic analyses of the cohort using a multi-stage analysis pipeline (Supplemental Information). The most frequently observed variant peptides represented SAAVs and gene-level events. The evidence supporting each novel event is\",\n",
            "          \"gene-level events. The evidence supporting each novel event is detailed in Table S2, including event type, genomic location, abundance information, and estimated false discovery rate (FDR). The validity of these variant peptides was further evaluated by\",\n",
            "          \"The validity of these variant peptides was further evaluated by the synthesis of twenty examples selected at random across the entire range of spectrum match scores. We obtained tandem mass spectra for all twenty synthetic peptides that matched spectra\",\n",
            "          \"spectra for all twenty synthetic peptides that matched spectra from our analyses, strongly supporting our observation of these variant peptides; three representative examples are given in Figure S1E. These results demonstrate the ability to confidently\",\n",
            "          \"S1E. These results demonstrate the ability to confidently detect, identify and validate genome-level predictions at the protein level. More novel peptides would likely be observed if there was sufficient sample for more extensive fractionation and/or\",\n",
            "          \"was sufficient sample for more extensive fractionation and/or replicate analyses; these limitations preclude any conclusions about the biological significance of unobserved events. For example, only 2 mutant p53 peptides were identified despite the\",\n",
            "          \"example, only 2 mutant p53 peptides were identified despite the presence of p53 mutations in all tumors examined. Such low coverage can occur for reasons that include: excessively large or small tryptic fragments, inability to distinguish some amino\",\n",
            "          \"or small tryptic fragments, inability to distinguish some amino acids, some possible biases against highly hydrophobic and hydrophilic peptides, or low abundance peptides co-eluting with very high abundance peptides. To assess the potential for\",\n",
            "          \"with very high abundance peptides. To assess the potential for post-transcriptional regulation (e.g., translational efficiency or protein degradation), we compared each protein to its corresponding transcript across all tumors, and correlation was\",\n",
            "          \"corresponding transcript across all tumors, and correlation was assessed for those pairs with reliable measurements for both mRNA and protein, i.e., proteins with a corresponding mRNA measurement observed in all 169 HGSC tumors where sample variability\",\n",
            "          \"observed in all 169 HGSC tumors where sample variability exceeded technical variability (3,196 pairs). We excluded 391 proteins observed without a corresponding mRNA (e.g., HBA1) or discordant gene symbol annotation in the protein database (e.g., THOC4).\",\n",
            "          \"gene symbol annotation in the protein database (e.g., THOC4). Overall, 79.4% of the mRNA:protein pairs showed statistically significant positive Spearman correlations (Benjamini-Hochberg adjusted p value <0.01; Figure 1) when changes in mRNA abundance\",\n",
            "          \"p value <0.01; Figure 1) when changes in mRNA abundance were compared to changes in relative protein abundance. The observed median r value of 0.45 for each mRNA-protein pair across all 169 tumors is similar to observations in colorectal cancer, breast\",\n",
            "          \"tumors is similar to observations in colorectal cancer, breast cancer and mouse tissues, although less than found for cell lines. In comparison, the median correlation of paired protein measurements from the same sample, but measured at two sites, was\",\n",
            "          \"from the same sample, but measured at two sites, was substantially higher, at 0.69 (Figure S1). A wide range of mRNA:protein correlations was observed. In general, weaker correlations were observed for highly stable and abundant proteins associated with\",\n",
            "          \"for highly stable and abundant proteins associated with housekeeping or non-intrinsic functions (e.g., ribosomes, mRNA splicing, oxidative phosphorylation, complement cascade), while more dynamic proteins known to be transcriptionally regulated in\",\n",
            "          \"dynamic proteins known to be transcriptionally regulated in response to nutrient demand or other perturbations (e.g., nucleotide metabolism, amino acid metabolism, acute inflammatory responses) were more highly correlated (Figure 1 and Table S3). This\",\n",
            "          \"were more highly correlated (Figure 1 and Table S3). This result is consistent with a previous colorectal cancer study and supports the hypothesis that while many biological functions are primarily regulated by mRNA abundance, post-transcriptional\",\n",
            "          \"are primarily regulated by mRNA abundance, post-transcriptional mechanisms likely have an important role in regulating certain house-keeping functions. Additionally, we found that 23 mRNAs lacking polyA tails displayed lower correlation (mean \\u22120.15) with\",\n",
            "          \"polyA tails displayed lower correlation (mean \\u22120.15) with their cognate proteins than did polyadenylated mRNAs, consistent with the decreased stability of mRNAs lacking polyA tails. Clustering of tumors based on protein abundance HGSC is the most common\",\n",
            "          \"of tumors based on protein abundance HGSC is the most common of the four major histological subtypes of epithelial ovarian cancer, and is characterized by distinct morphological features. Recent studies using mRNA abundance data have suggested four\",\n",
            "          \"Recent studies using mRNA abundance data have suggested four transcriptomic HGSC subtypes designated as differentiated, immunoreactive, mesenchymal, and proliferative. To build an unbiased molecular taxonomy of ovarian HGSC, we used protein abundance data\",\n",
            "          \"taxonomy of ovarian HGSC, we used protein abundance data from the 169 tumors to identify subtypes that might show biological differences that could be exploited in future studies. Figure 2 shows the resulting clustering analysis of individual tumors\",\n",
            "          \"2 shows the resulting clustering analysis of individual tumors (vertical columns) by protein abundance (horizontal rows). The cluster assignment for each sample is provided in Table S4, and a consensus value matrix for the subtype comparisons is shown in\",\n",
            "          \"consensus value matrix for the subtype comparisons is shown in Figure S2A. The results of a Weighted Gene Co-expression Network Analysis (WGCNA) of protein functional enrichment by subgroups are provided for the protein clusters, in Figure S2B and Table\",\n",
            "          \"are provided for the protein clusters, in Figure S2B and Table S4. The enrichment of KEGG and Reactome ontologies in the WGCNA-derived modules are shown in Figures S2C and S2D, respectively; membership of enriched pathways is provided in Table S3. Four of\",\n",
            "          \"of enriched pathways is provided in Table S3. Four of the proteomic clusters showed a clear correspondence to the mesenchymal, proliferative, immunoreactive, and differentiated subtypes defined by the TCGA transcriptome analysis (Figure 2 and Figure S2E).\",\n",
            "          \"by the TCGA transcriptome analysis (Figure 2 and Figure S2E). A relatively small fifth cluster of tumors significantly enriched in proteins associated with extracellular matrix interactions, erythrocyte and platelet functions, and the complement cascade\",\n",
            "          \"erythrocyte and platelet functions, and the complement cascade was also observed using multiple approaches, including model based clustering with Bayesian information criteria, consensus clustering, and VISDA-based sub-phenotype clustering. This new group\",\n",
            "          \"and VISDA-based sub-phenotype clustering. This new group could not be attributed to tissue source site sampling bias or any other meta-data category, but may be related to tumor characteristics, including vascularization and tumor content, as the Tumor\",\n",
            "          \"including vascularization and tumor content, as the Tumor Purity score for this subtype (and the Mesenchymal subtype) was significantly lower than that of the other three subtypes (Figure S2F). The clinical relevance of these protein-based clusters will\",\n",
            "          \"The clinical relevance of these protein-based clusters will require validation in independent HGSC sample sets, particularly as no significant difference in survival was observed (Figure S2G), similar to mRNA-based clustering analysis. Proteomic analysis\",\n",
            "          \"similar to mRNA-based clustering analysis. Proteomic analysis of CNA effects Chromosomal instability, marked by extensive copy number alterations (CNAs) in each tumor, is a hallmark of HGSC, and a likely source of driver alterations in this disease. CNAs\",\n",
            "          \"and a likely source of driver alterations in this disease. CNAs can affect the abundance of proteins at the same locus (cis effects), and may also act in trans, either directly or indirectly. Hypothesizing that CNAs with strong trans effects are more\",\n",
            "          \"Hypothesizing that CNAs with strong trans effects are more likely to elicit a molecular phenotype and confer selective advantages, we sought to identify those CNA regions that have the broadest effect on protein expression. In all, 29,393 CNA loci (Table\",\n",
            "          \"effect on protein expression. In all, 29,393 CNA loci (Table S5) were compared to our global proteomics data with 950,209 CNA/protein pairs (0.72% of the total) exhibiting significant association (Benjamini-Hochberg adjusted p value <0.01). We provide a\",\n",
            "          \"(Benjamini-Hochberg adjusted p value <0.01). We provide a complete list of the significantly associated proteins for each CNA in Table S5. The diagonal line evident in Figure 3 corresponds to cis effects, and vertical stripes correspond to trans effects\",\n",
            "          \"cis effects, and vertical stripes correspond to trans effects where changes in copy number affect expression of numerous proteins across the genome. We performed the same analysis for mRNA to identify sites where associations are transcriptionally\",\n",
            "          \"mRNA to identify sites where associations are transcriptionally mediated. A similar number of CNA/mRNA pairs were found to be significantly associated (1,113,164 at Benjamini-Hochberg adjusted p value <0.01; Figure 3). This analysis revealed regions on\",\n",
            "          \"p value <0.01; Figure 3). This analysis revealed regions on chromosomes 2, 7, 20, and 22 correlated in trans with more than 200 proteins. In contrast to colorectal cancer where most of the trans-regulation of protein by CNA was accompanied by similar\",\n",
            "          \"trans-regulation of protein by CNA was accompanied by similar changes in mRNA, we observed several loci associated with differences in protein abundance without a corresponding change in mRNA. For example, large regions on chromosome 2 have relatively\",\n",
            "          \"For example, large regions on chromosome 2 have relatively little trans effect on mRNA levels, but are associated with more than 200 proteins in trans. Dissecting the mechanisms by which a specific CNA can alter protein levels in trans, without affecting\",\n",
            "          \"CNA can alter protein levels in trans, without affecting the corresponding mRNA is difficult, given the extent of the amplified or deleted regions and the numerous genes affected at a given locus. Possible mechanisms include cis regulation of proteins\",\n",
            "          \"locus. Possible mechanisms include cis regulation of proteins associated with mRNA stability and translational efficiency, such as microRNAs and RNA binding proteins. Given the complex pattern of CNAs observed in HGSC, it has been difficult to identify a\",\n",
            "          \"of CNAs observed in HGSC, it has been difficult to identify a limited number of high impact genomic alterations that could function as drivers of the disease. We interrogated the trans-affected proteins associated with each putative CNA for common\",\n",
            "          \"proteins associated with each putative CNA for common functions using pathways defined by the KEGG, NCI pathway interaction database (PID) and Reactome databases (Supplemental Information). Proteins associated with cell invasion and migration, and\",\n",
            "          \"Proteins associated with cell invasion and migration, and proteins related to immune function appeared to be enriched in association with multiple CNAs (Figure S3; Table S3). These observations suggest a convergence of multiple CNA targets on a common set\",\n",
            "          \"suggest a convergence of multiple CNA targets on a common set of biological functions, namely motility/invasion and immune regulation, functions that are among the hallmarks of cancer. Association of CNA trans-affected proteins with overall survival\",\n",
            "          \"of CNA trans-affected proteins with overall survival Availability of the TCGA survival data allowed us to use trans-affected protein data from the most influential CNAs (e.g., the four altered regions described on chromosomes 2, 7, 20, and 22; Figure 3)\",\n",
            "          \"regions described on chromosomes 2, 7, 20, and 22; Figure 3) to build a model of overall survival. Because each CNA affects many proteins, we used a regression approach that identifies parsimonious Cox proportional hazards models with maximal predictive\",\n",
            "          \"Cox proportional hazards models with maximal predictive ability from the list of significantly correlated proteins for that CNA. We trained models on the proteomics data from PNNL (82 tumors) and tested the ability of the model to predict survival times\",\n",
            "          \"and tested the ability of the model to predict survival times in the data from JHU (87 tumors, not including the 32 overlapping). Each of the four most influential CNA regions produced models that were strongly associated with patient survival (p value\",\n",
            "          \"that were strongly associated with patient survival (p value <0.01, FDR <0.5% based on randomly selected proteins). A Kaplan-Meier plot illustrating the predictive value of each of the locus-specific models after validation is shown in Figure S4A; for the\",\n",
            "          \"models after validation is shown in Figure S4A; for the Kaplan-Meier plots, \\u2018high\\u2019 and \\u2018low\\u2019 expression were defined relative to the median for that signature across all samples, splitting patients into two equal groups of 45 (Table S6). We examined the\",\n",
            "          \"into two equal groups of 45 (Table S6). We examined the overlap in predictions for patients for the four signatures and found that the predictions were unanimous (either high or low signature) for 62% of patients and 16% were evenly split, with the\",\n",
            "          \"for 62% of patients and 16% were evenly split, with the remaining having one dissenting signature. This suggested the utility of combining these signatures, using a voting method where the number of high or low calls was counted for each patient; this\",\n",
            "          \"number of high or low calls was counted for each patient; this substantially improved prediction of survival time (p value 1.9e-6; Figure 4). We also examined the effects of tumor stage, tumor grade, patient age, surgical outcome, and platinum status, and\",\n",
            "          \"grade, patient age, surgical outcome, and platinum status, and found that our proteomic signatures were not improved by inclusion of these variables. For each of these locus-specific models we analyzed the enrichment of genes and their regulatory\",\n",
            "          \"models we analyzed the enrichment of genes and their regulatory sequences associated with outcome, and found all four models had genes significantly enriched (Benjamini-Hochberg adjusted p value <0.05) in binding sites for the proliferation-associated\",\n",
            "          \"value <0.05) in binding sites for the proliferation-associated serum response factor (SRF), suggesting that SRF activity may be important in ovarian cancer outcome. Additionally, there was significant enrichment of proteins involved in the regulation of\",\n",
            "          \"enrichment of proteins involved in the regulation of actin cytoskeleton, apoptosis, and adherens junction (Benjamini-Hochberg adjusted p value <0.05). We examined the protein abundance and phosphorylation status of SRF between short and long surviving\",\n",
            "          \"phosphorylation status of SRF between short and long surviving groups and observed that they were higher in short surviving patients, but only slightly. Thus, although SRF alone was not predictive, the trends in SRF abundance and phosphorylation are\",\n",
            "          \"predictive, the trends in SRF abundance and phosphorylation are consistent with the observation of enrichment for proteins potentially regulated by SRF binding. Details of the larger gene set analysis are provided in Supplemental Information. Several\",\n",
            "          \"set analysis are provided in Supplemental Information. Several proteins shared across all the signatures are known to be involved in cancer processes. Catenin B2 (CTNNA2) is a cell-cell adhesion protein and tumor suppressor. The Rho GDP dissociation\",\n",
            "          \"adhesion protein and tumor suppressor. The Rho GDP dissociation inhibitor (GDI) beta (ARHGDIB) is involved in invasion and migration in many cancers, and overexpression correlates with progression in pancreatic carcinoma. Protein kinase C and casein\",\n",
            "          \"in pancreatic carcinoma. Protein kinase C and casein kinase substrate in neurons protein 2 (PACSIN2) is a repressor of cellular migration. Finally, the GTP-binding nuclear protein Ran, is a prognostic marker associated with increased survival in\",\n",
            "          \"is a prognostic marker associated with increased survival in epithelial ovarian cancer. The association of these previously described survival-related proteins with genes affected in trans by CNAs suggests a potential mechanism for the parallel activation\",\n",
            "          \"CNAs suggests a potential mechanism for the parallel activation of multiple pathways associated with poor prognosis in HGSC. As a comparison we applied the previously described Provar signature, comprising five protein and four phosphoproteins that showed\",\n",
            "          \"comprising five protein and four phosphoproteins that showed good survival prediction in the TCGA ovarian cancer dataset. We observed all proteins in the Provar signature in our proteomic data, but only one of the phosphosites (EGFR Y1173). Thus, we used\",\n",
            "          \"but only one of the phosphosites (EGFR Y1173). Thus, we used the RPPA data from the original signature (Figure S4B). We found that the Provar signature was prognostic of survival (Benjamini-Hochberg adjusted p value = 0.11) in the 67 patients examined\",\n",
            "          \"adjusted p value = 0.11) in the 67 patients examined here (those with phosphoprotein data and had somatic TP53 mutations), but the statistical power of Provar is not as high as the statistical power of signatures derived from the CNA trans-affected\",\n",
            "          \"power of signatures derived from the CNA trans-affected proteins in this dataset. In addition, integrating the Provar signature with the CNA signatures did not improve survival prediction (Figure S4C). Identification of proteins associated with\",\n",
            "          \"(Figure S4C). Identification of proteins associated with chromosomal structural abnormality The degree of chromosomal instability exhibited by a tumor can be represented by a calculated chromosome instability (CIN) index, as described previously for lung\",\n",
            "          \"instability (CIN) index, as described previously for lung cancer. Identification of proteins associated with CIN may provide information on the processes contributing to chromosomal instability, while the analysis of trans-affected proteins described\",\n",
            "          \"while the analysis of trans-affected proteins described above is more closely related to the downstream consequences of specific CNAs. Using a bootstrapping method described in Supplemental Information, we identified a proteomic signature including 128\",\n",
            "          \"Information, we identified a proteomic signature including 128 proteins (Table S7) showing significant correlation with CIN index (Benjamini-Hochberg adjusted p value <1e-4, Spearman correlation; Figure S5A). Functional annotation of these proteins\",\n",
            "          \"Figure S5A). Functional annotation of these proteins (Figure S5B) showed that proteins up-regulated in tumors with high CIN index were preferentially involved in chromatin organization (p value 6.90e-5), whereas proteins up-regulated in tumors with low\",\n",
            "          \"6.90e-5), whereas proteins up-regulated in tumors with low CIN index were more often associated with cell death (p value 2.13e-5). Correlation analysis of protein abundances and CIN indicated that a small number of proteins could account for the majority\",\n",
            "          \"that a small number of proteins could account for the majority of the CIN (Figure 5); two of the most strongly associated proteins, CHD4 and CHD5, are known to be involved in chromatin organization. Identification of proteins associated with HRD status\",\n",
            "          \"Identification of proteins associated with HRD status Since HRD is associated with susceptibility to PARP inhibitors and improved survival, we sought to elucidate systemic changes associated with HRD and identify biomarkers that might be used to stratify\",\n",
            "          \"with HRD and identify biomarkers that might be used to stratify patients for treatment. We defined tumors with HRD as having either germline or somatic mutations in BRCA1 or BRCA2, or BRCA1 promoter methylation, or homozygous deletion of PTEN, with an\",\n",
            "          \"promoter methylation, or homozygous deletion of PTEN, with an overall survival >1.5 years, while non-HRD patients were defined as lacking these genomic aberrations and with a follow-up or time-to-death <2.5 years; additional selection criteria include\",\n",
            "          \"time-to-death <2.5 years; additional selection criteria include available residual tissue volume and a tumor tissue contamination score estimated using copy number alterations. Applying differential dependency network (DDN) analysis on a set of 171 BRCA1-\",\n",
            "          \"dependency network (DDN) analysis on a set of 171 BRCA1- or BRCA2-related proteins curated from the literature and from the cBio portal, we identified a sub-network of 30 proteins that displayed co-expression patterns differentiating HRD from non-HRD\",\n",
            "          \"co-expression patterns differentiating HRD from non-HRD patients (Table S7; Figure 6). Several of the proteins in these modules are known to be involved in histone acetylation or deacetylation, e.g., HDAC1, RBBP4, RBBP7, and EP300, and HUS1. Although\",\n",
            "          \"e.g., HDAC1, RBBP4, RBBP7, and EP300, and HUS1. Although statistical association cannot distinguish between drivers and consequences of HRD status, the observed enrichment of proteins associated with histone acetylation motivated us to use an effective\",\n",
            "          \"with histone acetylation motivated us to use an effective database search strategy (Supplemental Information) to identify and quantify acetylated peptides from the global proteomic data. Comparative analysis of 399 acetylated peptides identified 15\",\n",
            "          \"Comparative analysis of 399 acetylated peptides identified 15 acetylated peptides with significant differences between the HRD and non-HRD samples (Table S7). Among these, dual acetylation at K12 and K16 of histone H4 showed a significant difference\",\n",
            "          \"at K12 and K16 of histone H4 showed a significant difference between HRD and non-HRD samples (Figure 6); differential acetylation of K12 and K16 was verified using synthetic peptides and targeted analysis using Sequential Windowed data independent\",\n",
            "          \"targeted analysis using Sequential Windowed data independent Acquisition of the Total High-resolution Mass Spectra (SWATH-MS). In cell line data, acetylation of H4 has previously been reported to be involved in the choice of DNA double strand break (DSB)\",\n",
            "          \"to be involved in the choice of DNA double strand break (DSB) repair pathways (homologous recombination or non-homologous end joining. This relationship is regulated partially by HDAC1, a protein also identified in the DDN analysis. We observed a\",\n",
            "          \"a protein also identified in the DDN analysis. We observed a significant enrichment of HDAC1 and its co-regulated proteins in tumors with HRD and low H4 acetylation relative to non-HRD tumors with high H4 acetylation (DDN analysis: permutation tests, p\",\n",
            "          \"with high H4 acetylation (DDN analysis: permutation tests, p value <0.05; differentially acetylated peptides: t-tests, p value <0.05 with an estimated FDR <0.5% by bootstrap/permutation tests). The combined observations of increased HDAC1 and associated\",\n",
            "          \"The combined observations of increased HDAC1 and associated proteins at the pathway level, together with decreased acetylation of H4 in HRD patients at the PTM level, provide insight regarding the potential role of HDAC1 in modulating the choice of DSB\",\n",
            "          \"the potential role of HDAC1 in modulating the choice of DSB repair pathways. Phosphoproteomic analysis of pathways associated with survival An initial set of 24,464 different phosphopeptides (21,298 unique phosphorylation sites) contained within 4,420\",\n",
            "          \"(21,298 unique phosphorylation sites) contained within 4,420 proteins having data on net differences in phosphorylation (Table S2) was obtained. Since ischemia of the TCGA tumor samples may alter phosphopeptide abundance, we removed any phosphorylation\",\n",
            "          \"alter phosphopeptide abundance, we removed any phosphorylation sites previously shown to be altered by ischemia. We also removed three tumor samples having substantially higher than average missing values and two tumor samples lacking somatic TP53\",\n",
            "          \"missing values and two tumor samples lacking somatic TP53 mutations (Supplemental Information). This yielded a final set of 7,675 different phosphopeptides (6,802 unique phosphorylation sites) from 2,324 proteins (Table S2) that were mapped to pathways\",\n",
            "          \"from 2,324 proteins (Table S2) that were mapped to pathways via the NCI PID. The average net phosphorylation of all phosphopeptides mapping to a given pathway (i.e., the statistical average across all phosphoproteins known to be in the pathway, rather\",\n",
            "          \"across all phosphoproteins known to be in the pathway, rather than the statistically significant individual proteins) was used as a measure of pathway activity and compared between short survivors (deceased surviving <3 years; N=17) and long survivors\",\n",
            "          \"(deceased surviving <3 years; N=17) and long survivors (patients surviving >5 years; N=19) using both proteomic and phosphoproteomic data. An issue in ranking the pathways is the distinction between phosphopeptides that are unique to a single pathway,\",\n",
            "          \"between phosphopeptides that are unique to a single pathway, such as the receptor peptides themselves, vs. peptides that map to proteins shared between pathways. In order to address this issue, we performed two analyses: one of all proteins associated\",\n",
            "          \"we performed two analyses: one of all proteins associated with a specific pathway in the NCID database, and a second analysis in which proteins common to multiple pathways were excluded. Figure 7A shows a ranking of activated pathways, as inferred from\",\n",
            "          \"7A shows a ranking of activated pathways, as inferred from the analysis of pathway specific components (i.e., differences in mRNA, protein and phosphoprotein levels) in short vs. long survivors. Fifteen pathways showed increased phosphorylation at\",\n",
            "          \"survivors. Fifteen pathways showed increased phosphorylation at Benjamini-Hochberg adjusted p values <0.01, in contrast with transcriptional data for the same samples which yielded only one statistically significantly increased pathway, androgen receptor\",\n",
            "          \"significantly increased pathway, androgen receptor signaling, which was also increased at the phosphoprotein level (Figure 7A and Supplemental Information). All pathways indicated in Figure 7A were increased in association with short survival relative to\",\n",
            "          \"were increased in association with short survival relative to long survival, though some individual components within a pathway showed opposite changes (e.g. JNK1 transcript and protein levels, see Figure 7B). The RhoA-regulatory, PDGFRB, and\",\n",
            "          \"levels, see Figure 7B). The RhoA-regulatory, PDGFRB, and integrin-like kinase pathways emerged as the most activated in this analysis (Figure 7A). Figure 7B shows the common and unique elements of the PDGFRB pathway, and the trends in mRNA, protein, and\",\n",
            "          \"of the PDGFRB pathway, and the trends in mRNA, protein, and phosphoprotein levels in the comparison of short vs. long survivors, illustrating the benefit of combined protein and phosphoprotein measurements, as well as the substantial contribution of\",\n",
            "          \"measurements, as well as the substantial contribution of protein phosphorylation data to the overall analysis. We compared our results to the 31 phosphoproteins represented on the RPPA arrays used in the original TCGA ovarian cancer analysis; RPPA\",\n",
            "          \"arrays used in the original TCGA ovarian cancer analysis; RPPA identified differential phosphorylation of ERK1, RAF1, and STAT3 in the same patient samples analyzed by MS. For short surviving vs. long surviving patients, ERK1 showed statistically\",\n",
            "          \"vs. long surviving patients, ERK1 showed statistically significantly increased phosphorylation by RPPA, on the same ERK1 phosphopeptides as found using MS, while AKT1, RAF1 and STAT3 showed increased phosphorylation by both RPPA and MS-based analyses, but\",\n",
            "          \"phosphorylation by both RPPA and MS-based analyses, but based upon distinct phosphopeptides (Table S2). Good overlap was observed between RPPA and MS results for phosphoproteins on the RPPA array, but we did not observe three of the specific phosphosites\",\n",
            "          \"but we did not observe three of the specific phosphosites from the Provar survival signature in our MS data, suggesting that proteomics is more sensitive to overall pathway activity than specific regulatory events in the pathway itself. However, many of\",\n",
            "          \"regulatory events in the pathway itself. However, many of the phosphopeptides identified by MS phosphoproteomics as components of the PDGFR-beta pathway were not represented in the RPPA, illustrating the ability to identify phosphorylation events not\",\n",
            "          \"illustrating the ability to identify phosphorylation events not readily accessible with currently available antibodies (Figure 7B). DISCUSSION The addition of proteomic information, including post-translational modifications, to the rich genomic and\",\n",
            "          \"post-translational modifications, to the rich genomic and transcriptomic data available from the ovarian HGSC samples analyzed by TCGA has provided additional insights into the biology of HGSC, including identification of changes at the level of pathway\",\n",
            "          \"including identification of changes at the level of pathway activation. Integration of genomic, proteomic, and phosphoproteomic measurements identified differentially regulated pathways and functional modules that displayed significant associations with\",\n",
            "          \"functional modules that displayed significant associations with patient outcomes, including survival and HRD status. Cox proportional hazard analysis of proteins associated with CNAs identified overall survival signatures enriched for targets of the\",\n",
            "          \"overall survival signatures enriched for targets of the proliferation-associated transcription factor SRF, and integrated proteomic, phosphoproteomic and transcriptomic data identified pathways that differentiated patients on the basis of survival,\",\n",
            "          \"pathways that differentiated patients on the basis of survival, including the PDGFR-beta signaling pathway associated with angiogenesis, the RhoA regulatory and integrin-linked kinase pathways associated with cell mobility and invasion, and pathways\",\n",
            "          \"associated with cell mobility and invasion, and pathways associated with chemokine signaling and adaptive immunity. Proteins associated with cell invasion and motility emerged from both the integration of CNAs with protein abundance data and the\",\n",
            "          \"the integration of CNAs with protein abundance data and the phosphoproteomic investigation of up-regulated pathways in short versus long survivors. Despite the complexity of genomic alterations that characterize HGSC, these analyses suggest a functional\",\n",
            "          \"that characterize HGSC, these analyses suggest a functional convergence on a subset of key pathways. The association of increased invasiveness and motility with short overall survival in this study may help to explain more aggressive mechanisms of\",\n",
            "          \"in this study may help to explain more aggressive mechanisms of dissemination in ovarian cancer, including recently reported hematogenous metastasis, in addition to previously described lymphatic spread and known peritoneal spread by direct extension.\",\n",
            "          \"spread and known peritoneal spread by direct extension. Focusing on functional modules has also revealed potential drivers of HGSC and more robust signatures for potentially stratifying ovarian HGSC patients into distinct cancer phenotypes to inform\",\n",
            "          \"ovarian HGSC patients into distinct cancer phenotypes to inform therapeutic management. For example, the identification of specific acetylation events associated with HRD, e.g., the simultaneous acetylation of K12 and K16 on histone H4, may provide an\",\n",
            "          \"acetylation of K12 and K16 on histone H4, may provide an alternative biomarker of HRD and a rationale for the selection of patients in future clinical trials of HDAC inhibitors alone or in combination with PARP inhibition. This may help to resolve the\",\n",
            "          \"combination with PARP inhibition. This may help to resolve the current discrepancy between the initial observation of limited single agent activity of HDAC inhibitors in ovarian cancer, and more recent findings of a >40% response rate when used in\",\n",
            "          \"and more recent findings of a >40% response rate when used in combination with cytotoxic chemotherapy in platinum-resistant patients. Similarly, comprehensive interrogation of protein phosphorylation identified multiple pathways with significantly\",\n",
            "          \"phosphorylation identified multiple pathways with significantly increased phosphorylation in patients with poor clinical outcomes. Specifically, the observed activation of PDGFR pathways in a subset of patients with short overall survival could\",\n",
            "          \"in a subset of patients with short overall survival could potentially stratify patients for selective enrollment in trials of anti-angiogenic therapy, which is particularly relevant to current controversies over the use of bevacuzimab as a first-line\",\n",
            "          \"controversies over the use of bevacuzimab as a first-line therapy in ovarian cancer. Overall, this work illustrates the ability of proteomics to complement genomics in providing additional insights into pathways and processes that drive ovarian cancer\",\n",
            "          \"insights into pathways and processes that drive ovarian cancer biology and how these pathways are altered in correspondence with clinical phenotypes. The comprehensive proteomic measurements for the HGSC tumor samples provide a public resource of\",\n",
            "          \"for the HGSC tumor samples provide a public resource of information. Importantly, the ability to identify PTMs revealed a strong association between specific histone acetylation events and the HRD phenotype. In addition, the enhanced view of pathway\",\n",
            "          \"the HRD phenotype. In addition, the enhanced view of pathway activity provided by measurements of protein phosphorylation provides a foundation for linking genotype to proteotype, and ultimately to phenotype for understanding the molecular basis of\",\n",
            "          \"to phenotype for understanding the molecular basis of cancer. EXPERIMENTAL PROCEDURES Full methods are available in Extended Experimental Procedures. Tumor Samples All tumor samples for the current study were obtained through the TCGA Biospecimen Core\",\n",
            "          \"current study were obtained through the TCGA Biospecimen Core Resource as described previously. Samples were selected based on overall survival, HRD status and availability. Quantitative Proteomics Tissue proteins were extracted and digested with trypsin;\",\n",
            "          \"Tissue proteins were extracted and digested with trypsin; at each site, a portion of each resulting sample was also used to create a pooled reference in which each tumor sample contributed an equal percentage by peptide mass. The pooled reference sample\",\n",
            "          \"equal percentage by peptide mass. The pooled reference sample was included in each multiplexed, isobaric labeling experiment to enable cross-experiment comparison in the entire sample cohort. The patient samples and the pooled reference sample were each\",\n",
            "          \"The patient samples and the pooled reference sample were each labeled by different iTRAQ reagents (Sciex), combined, fractionated, and split for integrated, quantitative global proteome (10%) and phosphoproteome (90%) analysis using LC-MS/MS on an\",\n",
            "          \"(10%) and phosphoproteome (90%) analysis using LC-MS/MS on an Orbitrap Velos mass spectrometer (Thermo Scientific). Raw data were processed for peptide identification by database searching and identified peptides were assembled as proteins and mapped to\",\n",
            "          \"identified peptides were assembled as proteins and mapped to gene identifiers for proteogenomic comparisons. Quantitation was achieved by comparing the iTRAQ reporter ion intensities in each sample. Comparison of mRNA and Protein Subtypes A model-based\",\n",
            "          \"sample. Comparison of mRNA and Protein Subtypes A model-based clustering approach was used to model the protein abundance data as a mixture of subtypes. Bayesian information criteria, statistical resampling and VISDA-based sub-phenotype clustering\",\n",
            "          \"statistical resampling and VISDA-based sub-phenotype clustering approaches provided similar results with a stable optimization on five clusters as determined by consensus clustering. WGCNA analysis was performed to infer genes or gene networks that drive\",\n",
            "          \"was performed to infer genes or gene networks that drive subtyping into 5 clusters, followed by correlation with subtype as a trait. Integration of Proteomics and CNA data Spearman correlation coefficients and corresponding adjusted p values were\",\n",
            "          \"coefficients and corresponding adjusted p values were calculated for each protein/transcript by CNA locus, and gene set enrichment analysis was used to infer function for groups of proteins significantly correlated with a given CNA locus. Regression\",\n",
            "          \"significantly correlated with a given CNA locus. Regression analysis was applied to the list of trans-affected protein correlated with each CNA to generate parsimonious Cox proportional hazards models with maximal predictive ability, using the PNNL data\",\n",
            "          \"models with maximal predictive ability, using the PNNL data for training and JHU data for testing. Kaplan-Meier plots were used to visualize performance in predicting overall survival and progression-free survival. A CIN index was calculated for each\",\n",
            "          \"progression-free survival. A CIN index was calculated for each sample as the mean absolute values of copy number measurements at the 29,393 selected loci. Bootstrap resampling was used to select proteins correlated with CIN index at high confidence.\",\n",
            "          \"select proteins correlated with CIN index at high confidence. Protein Acetylation and HRD Acetylated peptides were identified by searching the global proteomics data for dynamic acetylation to lysines (+42 Da). Acetylation levels were compared between HRD\",\n",
            "          \"lysines (+42 Da). Acetylation levels were compared between HRD and non-HRD cases by t-test. Targeted proteomics (SWATH) was used to orthogonally quantify the acetylated peptide with synthetic peptides as internal standard. Survival-related Pathways\",\n",
            "          \"peptides as internal standard. Survival-related Pathways Analysis Phosphoproteome data from the short and long survivors were mapped to signaling pathways in the NCI pathway information database,(PID (http://pid.nci.nih.gov) using the gene names. For each\",\n",
            "          \"(http://pid.nci.nih.gov) using the gene names. For each signaling pathway in the PID, relative abundances for all phosphopeptides mapping to any pathway component were identified and separated into short and long survivor groups. The difference in\",\n",
            "          \"into short and long survivor groups. The difference in distributions between the two sets of pathway-specific peptides, i.e., those associated with either short survivors or long long survivors, was then assessed using a two-tailed t test. Similar\",\n",
            "          \"survivors, was then assessed using a two-tailed t test. Similar enrichment analyses were also performed using protein abundance, mRNA abundance, and CNA. Data Repository All of the primary mass spectrometry data on TCGA tumor samples are deposited at the\",\n",
            "          \"spectrometry data on TCGA tumor samples are deposited at the CPTAC Data Coordinating Center as raw and mzML files and complete protein assembly data sets for public access (https://cptac-data-portal.georgetown.edu). Supplementary Material This is a PDF\",\n",
            "          \"Supplementary Material This is a PDF file of an unedited manuscript that has been accepted for publication. As a service to our customers we are providing this early version of the manuscript. The manuscript will undergo copyediting, typesetting, and\",\n",
            "          \"The manuscript will undergo copyediting, typesetting, and review of the resulting proof before it is published in its final form. Please note that during the production process errors may be discovered which could affect the content, and all legal\",\n",
            "          \"may be discovered which could affect the content, and all legal disclaimers that apply to the journal pertain.            Author Contributions          Study conception and design: HZ, TL, ZZ, RDS, DWC, KDR. Investigation: performed experiment or data\",\n",
            "          \"ZZ, RDS, DWC, KDR. Investigation: performed experiment or data collection: TL, SS, FY, JZ, LijunC, PS, PA, YT, MAG, TRC, CC, ST, SN, RJM, HengZ. Computation and statistical analysis: ZZ, SHP, BaiZ, JEM, VAP, LiC, DR, MEM, SWC, SW, JW, DLT, DF, VB, YW,\",\n",
            "          \"BaiZ, JEM, VAP, LiC, DR, MEM, SWC, SW, JW, DLT, DF, VB, YW, BingZ. Data interpretation and biological analysis: TL, HZ, SHP, ZZ, BaiZ, JEM, VAP, LiC, DR, SN, CW, IMS, AP, MPS, DAL, RDS, DWC, KDR. Writing/manuscript preparation and revision: HZ, TL, ZZ,\",\n",
            "          \"KDR. Writing/manuscript preparation and revision: HZ, TL, ZZ, SHP, JEM, DR, VAP, LeslieC, HR, IMS, AP, MPS, DAL, RDS, DWC, KDR. Supervision and administration: HR, ESB, TH, RCR, LS, RDS, DWC, KDR. The authors declare no competing financial interests.\",\n",
            "          \"KDR. The authors declare no competing financial interests.            SUPPLEMENTAL INFORMATION          Supplemental information includes Extended Experimental Procedures, 5 figures, and 8 tables and can be found with this article online. Correlations\",\n",
            "          \"tables and can be found with this article online. Correlations between mRNA and protein abundance in TCGA tumors mRNA and protein were correlated across all the samples, resulting in 90.6% positive correlations, and 79.4% were significantly correlated\",\n",
            "          \"positive correlations, and 79.4% were significantly correlated (Benjamini-Hochberg adjusted p value <0.01), with a mean Spearman\\u2019s correlation of 0.38 and a median value of 0.45 (top panel). Different biological pathways and processes showed significantly\",\n",
            "          \"biological pathways and processes showed significantly different levels of correlation (bottom panel). Metabolic pathways and the interferon response displayed high mRNA-protein correlation, and ribosome, mRNA splicing, oxidative phosphorylation, and\",\n",
            "          \"and ribosome, mRNA splicing, oxidative phosphorylation, and complement and coagulation cascade were poorly correlated. The mean correlation is shown in parentheses followed by Benjamini-Hochberg adjusted p values calculated using a Kolmogorov\\u2013Smirnov test\",\n",
            "          \"adjusted p values calculated using a Kolmogorov\\u2013Smirnov test following the functional group names from MSIGDB. Blue bars indicate positive correlations and yellow indicates negative correlations; individual proteins (represented as bars on the x-axis) are\",\n",
            "          \"individual proteins (represented as bars on the x-axis) are sorted by correlation from low to high (bottom panel). Proteomic subtypes and corresponding driving protein modules Global proteomics abundance is shown in a heatmap with TCGA samples represented\",\n",
            "          \"abundance is shown in a heatmap with TCGA samples represented by columns ordered by protein subtype where rows represent proteins. Color of each cell indicates z-score (log2 of relative abundance scaled by proteins\\u2019 standard deviation) of the protein in\",\n",
            "          \"scaled by proteins\\u2019 standard deviation) of the protein in that sample; red is increased and blue is decreased (relative to the pooled reference). Transcriptome-based subtypes (Verhaak et al., 2013) and the proposed proteomic subtypes are indicated in\",\n",
            "          \"al., 2013) and the proposed proteomic subtypes are indicated in color above the heatmap. WGCNA-derived modules are delineated with the row color panel and annotated according to the pathways based on enrichment of KEGG and Reactome ontologies. Functional\",\n",
            "          \"based on enrichment of KEGG and Reactome ontologies. Functional impact of copy number alterations The top panel shows the correlation of CNAs to protein abundance (right) or mRNA (left) with significant positive correlations in red and negative\",\n",
            "          \"with significant positive correlations in red and negative correlations in blue (Benjamini-Hochberg adjusted p value <0.01, Spearman\\u2019s correlation coefficient). The x-axis plots the 29,393 CNAs obtained from TCGA. The y-axis plots 3,202 proteins. Genes\",\n",
            "          \"CNAs obtained from TCGA. The y-axis plots 3,202 proteins. Genes are ordered by chromosomal location on both the x- and y-axes. The bottom panel shows the summed number of significantly correlated proteins (or mRNA) for each individual CNA. In blue is\",\n",
            "          \"proteins (or mRNA) for each individual CNA. In blue is shown the total number, in black are those genes significantly correlated as both mRNA and protein. Where blue and black lines have similar magnitude, e.g., the hotspot on chromosome 20, the CNA\",\n",
            "          \"similar magnitude, e.g., the hotspot on chromosome 20, the CNA associations are shared between protein and mRNA. Where a strong blue line has no mirrored black line, e.g., the protein hotspot on chromosome 2, the associations with CNA are largely unique.\",\n",
            "          \"on chromosome 2, the associations with CNA are largely unique. Kaplan-Meier plot of overall survival stratified by CNA-derived signatures A survival plot is shown for a consensus of the four best signatures (see Figure S4A and Table S6) identified from\",\n",
            "          \"best signatures (see Figure S4A and Table S6) identified from analysis of proteins affected in trans by CNAs. Models were trained using a lasso-based Cox proportional hazards model on the samples from PNNL, and shown are the survival curves from these\",\n",
            "          \"samples from PNNL, and shown are the survival curves from these models applied to the non-overlapping data from the JHU analysis, with the up (red; above the median) and down (blue, below the median) signatures each representing 45 patients. A vote was\",\n",
            "          \"median) signatures each representing 45 patients. A vote was taken among the four most predictive signatures, and the results of this vote are shown. Probability of death is shown on the y-axis vs. survival time in years on the x-axis. Shaded ribbons\",\n",
            "          \"y-axis vs. survival time in years on the x-axis. Shaded ribbons denote 95% confidence intervals. Number of CNAs statistically explained by proteins significantly associated with CIN index A total of 128 proteins selected for their association with CIN\",\n",
            "          \"A total of 128 proteins selected for their association with CIN were arranged along the chromosomal locations of their corresponding genes (y-axis). The length of the light grey horizontal lines indicates the number of CNAs significantly correlated with\",\n",
            "          \"indicates the number of CNAs significantly correlated with the protein across all chromosomes except for the protein\\u2019s corresponding gene coding region (x-axis). The top-ranked proteins are annotated and highlighted with dark lines to show the\",\n",
            "          \"are annotated and highlighted with dark lines to show the bootstrap-estimated 95% confidence intervals. DDN analysis and lysine-acetylation analysis between HRD and non-HRD patients DDN analysis revealed a sub-network of proteins that displayed distinct\",\n",
            "          \"revealed a sub-network of proteins that displayed distinct co-expression patterns between HRD and non-HRD patients, where purple connections indicate protein correlations that exist only in HRD samples and blue connections indicate protein correlations\",\n",
            "          \"HRD samples and blue connections indicate protein correlations that exist only in non-HRD patients. The links between the nodes are drawn with two different thickness indicative of whether the connections meet the significance threshold of 0.05 (light\",\n",
            "          \"the connections meet the significance threshold of 0.05 (light lines) or 0.01 (bold lines). Proteins with blue dotted circles are known to be involved in histone acetylation or deacetylation. Further, identification and quantitation of lysine-acetylated\",\n",
            "          \"Further, identification and quantitation of lysine-acetylated peptides in global proteomics data showed that acetylation level at K12 and K16 of histone H4 are significantly different between HRD and non-HRD samples, suggesting a role of histone H4\",\n",
            "          \"HRD and non-HRD samples, suggesting a role of histone H4 acetylation (together with HDAC1) in modulating the choice of DSB repair mechanisms. Pathway analysis associated with patient survival A) Pathway components were statistically analyzed using a\",\n",
            "          \"A) Pathway components were statistically analyzed using a two-sided t test between samples from patients surviving <3 years (short survival) and patients surviving >5 years (long survival) for differences in CNA, mRNA expression, protein or\",\n",
            "          \"survival) for differences in CNA, mRNA expression, protein or phosphorylation abundance. All significant pathways for mRNA and protein are shown (Benjamini-Hochberg adjusted p value <0.05) and the most significant pathways for phosphoproteomics are\",\n",
            "          \"and the most significant pathways for phosphoproteomics are plotted on the x axis as the \\u2212log of the p value. Results show that phosphorylation provides additional information about the functional state of tumors. B) Simplified PDGFR-beta pathway showing\",\n",
            "          \"state of tumors. B) Simplified PDGFR-beta pathway showing differences between short and long survivor groups for protein abundance (p), transcript abundance (m), and phosphopeptide abundance (circled P).\"\n",
            "        ]\n",
            "      }\n",
            "    ]\n",
            "  },\n",
            "  \"PDC000114\": {\n",
            "    \"study_id\": \"baa8ae46-57b8-11e8-b07a-00a098d917f8\",\n",
            "    \"pdc_study_id\": \"PDC000114\",\n",
            "    \"study_submitter_id\": \"TCGA OV Proteome S020-3\",\n",
            "    \"program_id\": \"10251935-5540-11e8-b664-00a098d917f8\",\n",
            "    \"project_id\": \"48af5040-5546-11e8-b664-00a098d917f8\",\n",
            "    \"study_name\": \"TCGA Ovarian PNNL Proteome\",\n",
            "    \"program_name\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"project_name\": \"CPTAC2 Retrospective\",\n",
            "    \"disease_type\": \"Other;Ovarian Serous Cystadenocarcinoma\",\n",
            "    \"primary_site\": \"Not Reported;Ovary\",\n",
            "    \"analytical_fraction\": \"Proteome\",\n",
            "    \"experiment_type\": \"iTRAQ4\",\n",
            "    \"embargo_date\": null,\n",
            "    \"cases_count\": 85,\n",
            "    \"aliquots_count\": 85,\n",
            "    \"protocol_id\": \"1c06bbb7-3b26-11e9-9a07-0a80fada099c\",\n",
            "    \"protocol_submitter_id\": \"P-S020-3\",\n",
            "    \"program_submitter_id\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"protocol_name\": \"TCGA_Ovarian_PNNL_Proteome\",\n",
            "    \"protocol_date\": \"2013-08-31\",\n",
            "    \"publications\": [\n",
            "      {\n",
            "        \"publication_id\": \"44e2e658-89c7-11e8-bcf1-0a2705229b82\",\n",
            "        \"pubmed_id\": \"27372738\",\n",
            "        \"doi\": \"10.1016/j.cell.2016.05.069\",\n",
            "        \"title\": \"Integrated Proteogenomic Characterization of Human High-Grade Serous Ovarian Cancer\",\n",
            "        \"journal\": \"Cell\",\n",
            "        \"journal_url\": \"https://www.cell.com/cell/fulltext/S0092-8674(16)30673-0?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0092867416306730%3Fshowall%3Dtrue\",\n",
            "        \"year\": \"2016\",\n",
            "        \"abstract\": \"To provide a detailed analysis of the molecular components and underlying mechanisms associated with ovarian cancer, we performed a comprehensive mass-spectrometry-based proteomic characterization of 174 ovarian tumors previously analyzed by The Cancer Genome Atlas (TCGA), of which 169 were high-grade serous carcinomas (HGSCs). Integrating our proteomic measurements with the genomic data yielded a number of insights into disease, such as how different copy-number alternations influence the proteome, the proteins associated with chromosomal instability, the sets of signaling pathways that diverse genome rearrangements converge on, and the ones most associated with short overall survival. Specific protein acetylations associated with homologous recombination deficiency suggest a potential means for stratifying patients for therapy. In addition to providing a valuable resource, these findings provide a view of how the somatic genome drives the cancer proteome and associations between protein and post-translational modification levels and clinical outcomes in HGSC.\",\n",
            "        \"full_text\": [\n",
            "          \"Integrated proteogenomic characterization of human high grade serous ovarian cancer SUMMARY To provide a detailed analysis of the molecular components and underlying mechanisms associated with ovarian cancer, we performed a comprehensive mass\",\n",
            "          \"with ovarian cancer, we performed a comprehensive mass spectrometry-based proteomic characterization of 174 ovarian tumors previously analyzed by The Cancer Genome Atlas (TCGA), of which 169 were high-grade serous carcinomas (HGSC). Integrating our\",\n",
            "          \"169 were high-grade serous carcinomas (HGSC). Integrating our proteomic measurements with the genomic data yielded a number of insights into disease such as how different copy number alternations influence the proteome, the proteins associated with\",\n",
            "          \"influence the proteome, the proteins associated with chromosomal instability, the sets of signaling pathways that diverse genome rearrangements converge on, as well as the ones most associated with short overall survival. Specific protein acetylations\",\n",
            "          \"with short overall survival. Specific protein acetylations associated with homologous recombination deficiency suggest a potential means for stratifying patients for therapy. In addition to providing a valuable resource, these findings provide a view of\",\n",
            "          \"providing a valuable resource, these findings provide a view of how the somatic genome drives the cancer proteome and associations between protein and post-translational modification levels and clinical outcomes in HGSC. INTRODUCTION A comprehensive\",\n",
            "          \"and clinical outcomes in HGSC. INTRODUCTION A comprehensive molecular view of cancer is necessary for understanding the underlying mechanisms of disease, improving prognosis, and ultimately guiding treatment. The Cancer Genome Atlas (TCGA) conducted an\",\n",
            "          \"guiding treatment. The Cancer Genome Atlas (TCGA) conducted an extensive genomic and transcriptomic characterization of ovarian high-grade serous carcinoma (HGSC) aimed at defining the genomic landscape and aiding the development of targeted therapies for\",\n",
            "          \"landscape and aiding the development of targeted therapies for this highly lethal malignancy. Key findings from TCGA were: 1) the prevalent role of TP53 mutations, 2) extensive DNA copy alterations, 3) preliminary transcriptional signatures associated\",\n",
            "          \"3) preliminary transcriptional signatures associated with survival, 4) varied mechanisms of BRCA1/2 inactivation, and lastly, 5) CCNE1 aberrations. Subsequent analysis of genomic data from the TCGA consortium led to the refinement of the\",\n",
            "          \"data from the TCGA consortium led to the refinement of the transcript-defined signatures, improving the statistical association with patient outcome, and integrating microRNA and mRNA expression profiles associated with HGSC to identify candidate microRNA\",\n",
            "          \"profiles associated with HGSC to identify candidate microRNA targets. While the insights from genomic analyses are substantial, functions encoded in the genome are generally executed at the protein level, and are often further modulated by\",\n",
            "          \"at the protein level, and are often further modulated by post-translational modifications (PTMs). For example, TCGA used reverse phase protein array (RPPA) analysis of 172 proteins (including 31 phosphoproteins with phospho-specific antibodies) to\",\n",
            "          \"31 phosphoproteins with phospho-specific antibodies) to generate a signature associated with the risk of tumor recurrence. To obtain a more comprehensive assessment of the complex ovarian HGSC phenotype, the Clinical Proteomic Tumor Analysis Consortium\",\n",
            "          \"phenotype, the Clinical Proteomic Tumor Analysis Consortium (CPTAC) conducted an extensive mass spectrometry (MS)-based proteomic and phosphoproteomic characterization of HGSC tumors characterized by TCGA, providing quantitative measurements for a\",\n",
            "          \"by TCGA, providing quantitative measurements for a combined total of 9,600 proteins from 174 tumors (an average of 7,952 proteins per tumor), and a total of 24,429 phosphosites from 6,769 phosphoproteins in a subset of 69 tumors (an average of 7,677\",\n",
            "          \"phosphoproteins in a subset of 69 tumors (an average of 7,677 phosphosites per tumor). Our results provide insights on HGSC biology and correlate differences in protein and PTM levels with clinical outcome complementary to TCGA genomic analyses. RESULTS\",\n",
            "          \"outcome complementary to TCGA genomic analyses. RESULTS Proteomic analysis of TCGA HGSC samples HGSC biospecimens and clinical data from 174 patients collected by TCGA were analyzed at two independent CPTAC Centers, Johns Hopkins University (JHU) and\",\n",
            "          \"independent CPTAC Centers, Johns Hopkins University (JHU) and Pacific Northwest National Laboratory (PNNL); thirty-two samples were analyzed at both JHU and PNNL. Tumors were selected by examining the associated TCGA metadata to select tumors either 1) on\",\n",
            "          \"the associated TCGA metadata to select tumors either 1) on the basis of putative homologous recombination deficiency (HRD), defined by the presence of germline or somatic BRCA1 or BRCA2 mutations, BRCA1 promoter methylation, or homozygous deletion of PTEN\",\n",
            "          \"BRCA1 promoter methylation, or homozygous deletion of PTEN (122 samples; 67 classified as HRD, 55 as non-HRD by the above criteria; JHU), or 2) to maximize differences in overall survival (84 samples; PNNL). For selection purposes, short survival was\",\n",
            "          \"(84 samples; PNNL). For selection purposes, short survival was defined as overall survival of less than 3 years, and long survival as greater than 5 years. All but five tumors had somatic TP53 mutations, a characteristic feature of HGSC; these five tumors\",\n",
            "          \"mutations, a characteristic feature of HGSC; these five tumors were subsequently reclassified as other than HGSC, and removed from protein functional analyses (e.g., subtyping and survival analyses). The tumor selection criteria and the associated\",\n",
            "          \"analyses). The tumor selection criteria and the associated metadata are provided in the Supplemental Information Table S1, available online. Proteomics measurements used isobaric tags for relative and absolute quantitation (iTRAQ) in conjunction with\",\n",
            "          \"relative and absolute quantitation (iTRAQ) in conjunction with offline liquid chromatography fractionation via high-pH reversed phase liquid chromatography (RPLC) and online RPLC with high-resolution tandem MS to provide broad coverage for peptide and\",\n",
            "          \"tandem MS to provide broad coverage for peptide and protein identification and quantification (Supplemental Information); this also alleviated quantitative interference potentially associated with the use of isobaric tags. We used the relative abundance\",\n",
            "          \"with the use of isobaric tags. We used the relative abundance measurements for each protein in the 32 patient samples analyzed at both JHU and PNNL to normalize across the two analysis sites, and then used clustering, principle component analysis (PCA),\",\n",
            "          \"and then used clustering, principle component analysis (PCA), and statistical tests to identify any significant batch effects associated with the site of analysis (a detailed comparison of within site, between site, between sample measurement variability\",\n",
            "          \"site, between site, between sample measurement variability and the process used to merge the JHU and PNNL data is given in Supplemental Information, Figure S1). As shown in Figure S1C, the median coefficient of variation (CV) between measurements at the\",\n",
            "          \"coefficient of variation (CV) between measurements at the two sites was 16%. A total of 9,600 proteins were identified with high confidence in all tumors, and the relative abundances in each tumor are given in Table S2. Functional analyses and\",\n",
            "          \"in each tumor are given in Table S2. Functional analyses and proteome-transcriptome associations were restricted to 3,586 proteins observed and quantified in all 169 HGSC samples used for protein functional analyses and where sample variability (signal)\",\n",
            "          \"functional analyses and where sample variability (signal) exceeded technical variability (noise) in the merged data (Table S2), calculated as described in Supplemental Information. On average we identify peptides covering 29% of the amino acids in any of\",\n",
            "          \"we identify peptides covering 29% of the amino acids in any of these 3,586 proteins in a given sample, with a range from 10% to 47%. In addition to protein abundance levels, phosphoproteomics data were acquired for 69 tumors with sufficient sample (Table\",\n",
            "          \"data were acquired for 69 tumors with sufficient sample (Table S2). As with proteins, phosphopeptide abundances were calculated relative to the pooled reference sample. Because isobaric labeling was performed prior to splitting the samples for proteome\",\n",
            "          \"was performed prior to splitting the samples for proteome and phosphoproteome analyses, phosphopeptide abundance could be corrected for changes in parent protein abundance to identify differences in the relative extent of phosphorylation at specific sites\",\n",
            "          \"in the relative extent of phosphorylation at specific sites for each protein (Table S2). Overall, we achieved a protein dynamic range encompassing >4 orders of magnitude, ranging from low-level transcription factors to abundant structural proteins i.e.,\",\n",
            "          \"transcription factors to abundant structural proteins i.e., actin and tubulin. Proteogenomic landscape of HGSC The degree to which alterations observed at the genome and transcriptome levels are manifested at the protein level is variable, both\",\n",
            "          \"levels are manifested at the protein level is variable, both qualitatively and quantitatively, and partially driven by multiple levels of post-transcriptional regulation. To identify peptides encoded by single amino acid variants (SAAVs), splice variants,\",\n",
            "          \"encoded by single amino acid variants (SAAVs), splice variants, and novel exons documented by TCGA, mass spectra were searched against a custom graph database containing all peptide variations projected from the TCGA genomic analyses of the cohort using a\",\n",
            "          \"projected from the TCGA genomic analyses of the cohort using a multi-stage analysis pipeline (Supplemental Information). The most frequently observed variant peptides represented SAAVs and gene-level events. The evidence supporting each novel event is\",\n",
            "          \"gene-level events. The evidence supporting each novel event is detailed in Table S2, including event type, genomic location, abundance information, and estimated false discovery rate (FDR). The validity of these variant peptides was further evaluated by\",\n",
            "          \"The validity of these variant peptides was further evaluated by the synthesis of twenty examples selected at random across the entire range of spectrum match scores. We obtained tandem mass spectra for all twenty synthetic peptides that matched spectra\",\n",
            "          \"spectra for all twenty synthetic peptides that matched spectra from our analyses, strongly supporting our observation of these variant peptides; three representative examples are given in Figure S1E. These results demonstrate the ability to confidently\",\n",
            "          \"S1E. These results demonstrate the ability to confidently detect, identify and validate genome-level predictions at the protein level. More novel peptides would likely be observed if there was sufficient sample for more extensive fractionation and/or\",\n",
            "          \"was sufficient sample for more extensive fractionation and/or replicate analyses; these limitations preclude any conclusions about the biological significance of unobserved events. For example, only 2 mutant p53 peptides were identified despite the\",\n",
            "          \"example, only 2 mutant p53 peptides were identified despite the presence of p53 mutations in all tumors examined. Such low coverage can occur for reasons that include: excessively large or small tryptic fragments, inability to distinguish some amino\",\n",
            "          \"or small tryptic fragments, inability to distinguish some amino acids, some possible biases against highly hydrophobic and hydrophilic peptides, or low abundance peptides co-eluting with very high abundance peptides. To assess the potential for\",\n",
            "          \"with very high abundance peptides. To assess the potential for post-transcriptional regulation (e.g., translational efficiency or protein degradation), we compared each protein to its corresponding transcript across all tumors, and correlation was\",\n",
            "          \"corresponding transcript across all tumors, and correlation was assessed for those pairs with reliable measurements for both mRNA and protein, i.e., proteins with a corresponding mRNA measurement observed in all 169 HGSC tumors where sample variability\",\n",
            "          \"observed in all 169 HGSC tumors where sample variability exceeded technical variability (3,196 pairs). We excluded 391 proteins observed without a corresponding mRNA (e.g., HBA1) or discordant gene symbol annotation in the protein database (e.g., THOC4).\",\n",
            "          \"gene symbol annotation in the protein database (e.g., THOC4). Overall, 79.4% of the mRNA:protein pairs showed statistically significant positive Spearman correlations (Benjamini-Hochberg adjusted p value <0.01; Figure 1) when changes in mRNA abundance\",\n",
            "          \"p value <0.01; Figure 1) when changes in mRNA abundance were compared to changes in relative protein abundance. The observed median r value of 0.45 for each mRNA-protein pair across all 169 tumors is similar to observations in colorectal cancer, breast\",\n",
            "          \"tumors is similar to observations in colorectal cancer, breast cancer and mouse tissues, although less than found for cell lines. In comparison, the median correlation of paired protein measurements from the same sample, but measured at two sites, was\",\n",
            "          \"from the same sample, but measured at two sites, was substantially higher, at 0.69 (Figure S1). A wide range of mRNA:protein correlations was observed. In general, weaker correlations were observed for highly stable and abundant proteins associated with\",\n",
            "          \"for highly stable and abundant proteins associated with housekeeping or non-intrinsic functions (e.g., ribosomes, mRNA splicing, oxidative phosphorylation, complement cascade), while more dynamic proteins known to be transcriptionally regulated in\",\n",
            "          \"dynamic proteins known to be transcriptionally regulated in response to nutrient demand or other perturbations (e.g., nucleotide metabolism, amino acid metabolism, acute inflammatory responses) were more highly correlated (Figure 1 and Table S3). This\",\n",
            "          \"were more highly correlated (Figure 1 and Table S3). This result is consistent with a previous colorectal cancer study and supports the hypothesis that while many biological functions are primarily regulated by mRNA abundance, post-transcriptional\",\n",
            "          \"are primarily regulated by mRNA abundance, post-transcriptional mechanisms likely have an important role in regulating certain house-keeping functions. Additionally, we found that 23 mRNAs lacking polyA tails displayed lower correlation (mean \\u22120.15) with\",\n",
            "          \"polyA tails displayed lower correlation (mean \\u22120.15) with their cognate proteins than did polyadenylated mRNAs, consistent with the decreased stability of mRNAs lacking polyA tails. Clustering of tumors based on protein abundance HGSC is the most common\",\n",
            "          \"of tumors based on protein abundance HGSC is the most common of the four major histological subtypes of epithelial ovarian cancer, and is characterized by distinct morphological features. Recent studies using mRNA abundance data have suggested four\",\n",
            "          \"Recent studies using mRNA abundance data have suggested four transcriptomic HGSC subtypes designated as differentiated, immunoreactive, mesenchymal, and proliferative. To build an unbiased molecular taxonomy of ovarian HGSC, we used protein abundance data\",\n",
            "          \"taxonomy of ovarian HGSC, we used protein abundance data from the 169 tumors to identify subtypes that might show biological differences that could be exploited in future studies. Figure 2 shows the resulting clustering analysis of individual tumors\",\n",
            "          \"2 shows the resulting clustering analysis of individual tumors (vertical columns) by protein abundance (horizontal rows). The cluster assignment for each sample is provided in Table S4, and a consensus value matrix for the subtype comparisons is shown in\",\n",
            "          \"consensus value matrix for the subtype comparisons is shown in Figure S2A. The results of a Weighted Gene Co-expression Network Analysis (WGCNA) of protein functional enrichment by subgroups are provided for the protein clusters, in Figure S2B and Table\",\n",
            "          \"are provided for the protein clusters, in Figure S2B and Table S4. The enrichment of KEGG and Reactome ontologies in the WGCNA-derived modules are shown in Figures S2C and S2D, respectively; membership of enriched pathways is provided in Table S3. Four of\",\n",
            "          \"of enriched pathways is provided in Table S3. Four of the proteomic clusters showed a clear correspondence to the mesenchymal, proliferative, immunoreactive, and differentiated subtypes defined by the TCGA transcriptome analysis (Figure 2 and Figure S2E).\",\n",
            "          \"by the TCGA transcriptome analysis (Figure 2 and Figure S2E). A relatively small fifth cluster of tumors significantly enriched in proteins associated with extracellular matrix interactions, erythrocyte and platelet functions, and the complement cascade\",\n",
            "          \"erythrocyte and platelet functions, and the complement cascade was also observed using multiple approaches, including model based clustering with Bayesian information criteria, consensus clustering, and VISDA-based sub-phenotype clustering. This new group\",\n",
            "          \"and VISDA-based sub-phenotype clustering. This new group could not be attributed to tissue source site sampling bias or any other meta-data category, but may be related to tumor characteristics, including vascularization and tumor content, as the Tumor\",\n",
            "          \"including vascularization and tumor content, as the Tumor Purity score for this subtype (and the Mesenchymal subtype) was significantly lower than that of the other three subtypes (Figure S2F). The clinical relevance of these protein-based clusters will\",\n",
            "          \"The clinical relevance of these protein-based clusters will require validation in independent HGSC sample sets, particularly as no significant difference in survival was observed (Figure S2G), similar to mRNA-based clustering analysis. Proteomic analysis\",\n",
            "          \"similar to mRNA-based clustering analysis. Proteomic analysis of CNA effects Chromosomal instability, marked by extensive copy number alterations (CNAs) in each tumor, is a hallmark of HGSC, and a likely source of driver alterations in this disease. CNAs\",\n",
            "          \"and a likely source of driver alterations in this disease. CNAs can affect the abundance of proteins at the same locus (cis effects), and may also act in trans, either directly or indirectly. Hypothesizing that CNAs with strong trans effects are more\",\n",
            "          \"Hypothesizing that CNAs with strong trans effects are more likely to elicit a molecular phenotype and confer selective advantages, we sought to identify those CNA regions that have the broadest effect on protein expression. In all, 29,393 CNA loci (Table\",\n",
            "          \"effect on protein expression. In all, 29,393 CNA loci (Table S5) were compared to our global proteomics data with 950,209 CNA/protein pairs (0.72% of the total) exhibiting significant association (Benjamini-Hochberg adjusted p value <0.01). We provide a\",\n",
            "          \"(Benjamini-Hochberg adjusted p value <0.01). We provide a complete list of the significantly associated proteins for each CNA in Table S5. The diagonal line evident in Figure 3 corresponds to cis effects, and vertical stripes correspond to trans effects\",\n",
            "          \"cis effects, and vertical stripes correspond to trans effects where changes in copy number affect expression of numerous proteins across the genome. We performed the same analysis for mRNA to identify sites where associations are transcriptionally\",\n",
            "          \"mRNA to identify sites where associations are transcriptionally mediated. A similar number of CNA/mRNA pairs were found to be significantly associated (1,113,164 at Benjamini-Hochberg adjusted p value <0.01; Figure 3). This analysis revealed regions on\",\n",
            "          \"p value <0.01; Figure 3). This analysis revealed regions on chromosomes 2, 7, 20, and 22 correlated in trans with more than 200 proteins. In contrast to colorectal cancer where most of the trans-regulation of protein by CNA was accompanied by similar\",\n",
            "          \"trans-regulation of protein by CNA was accompanied by similar changes in mRNA, we observed several loci associated with differences in protein abundance without a corresponding change in mRNA. For example, large regions on chromosome 2 have relatively\",\n",
            "          \"For example, large regions on chromosome 2 have relatively little trans effect on mRNA levels, but are associated with more than 200 proteins in trans. Dissecting the mechanisms by which a specific CNA can alter protein levels in trans, without affecting\",\n",
            "          \"CNA can alter protein levels in trans, without affecting the corresponding mRNA is difficult, given the extent of the amplified or deleted regions and the numerous genes affected at a given locus. Possible mechanisms include cis regulation of proteins\",\n",
            "          \"locus. Possible mechanisms include cis regulation of proteins associated with mRNA stability and translational efficiency, such as microRNAs and RNA binding proteins. Given the complex pattern of CNAs observed in HGSC, it has been difficult to identify a\",\n",
            "          \"of CNAs observed in HGSC, it has been difficult to identify a limited number of high impact genomic alterations that could function as drivers of the disease. We interrogated the trans-affected proteins associated with each putative CNA for common\",\n",
            "          \"proteins associated with each putative CNA for common functions using pathways defined by the KEGG, NCI pathway interaction database (PID) and Reactome databases (Supplemental Information). Proteins associated with cell invasion and migration, and\",\n",
            "          \"Proteins associated with cell invasion and migration, and proteins related to immune function appeared to be enriched in association with multiple CNAs (Figure S3; Table S3). These observations suggest a convergence of multiple CNA targets on a common set\",\n",
            "          \"suggest a convergence of multiple CNA targets on a common set of biological functions, namely motility/invasion and immune regulation, functions that are among the hallmarks of cancer. Association of CNA trans-affected proteins with overall survival\",\n",
            "          \"of CNA trans-affected proteins with overall survival Availability of the TCGA survival data allowed us to use trans-affected protein data from the most influential CNAs (e.g., the four altered regions described on chromosomes 2, 7, 20, and 22; Figure 3)\",\n",
            "          \"regions described on chromosomes 2, 7, 20, and 22; Figure 3) to build a model of overall survival. Because each CNA affects many proteins, we used a regression approach that identifies parsimonious Cox proportional hazards models with maximal predictive\",\n",
            "          \"Cox proportional hazards models with maximal predictive ability from the list of significantly correlated proteins for that CNA. We trained models on the proteomics data from PNNL (82 tumors) and tested the ability of the model to predict survival times\",\n",
            "          \"and tested the ability of the model to predict survival times in the data from JHU (87 tumors, not including the 32 overlapping). Each of the four most influential CNA regions produced models that were strongly associated with patient survival (p value\",\n",
            "          \"that were strongly associated with patient survival (p value <0.01, FDR <0.5% based on randomly selected proteins). A Kaplan-Meier plot illustrating the predictive value of each of the locus-specific models after validation is shown in Figure S4A; for the\",\n",
            "          \"models after validation is shown in Figure S4A; for the Kaplan-Meier plots, \\u2018high\\u2019 and \\u2018low\\u2019 expression were defined relative to the median for that signature across all samples, splitting patients into two equal groups of 45 (Table S6). We examined the\",\n",
            "          \"into two equal groups of 45 (Table S6). We examined the overlap in predictions for patients for the four signatures and found that the predictions were unanimous (either high or low signature) for 62% of patients and 16% were evenly split, with the\",\n",
            "          \"for 62% of patients and 16% were evenly split, with the remaining having one dissenting signature. This suggested the utility of combining these signatures, using a voting method where the number of high or low calls was counted for each patient; this\",\n",
            "          \"number of high or low calls was counted for each patient; this substantially improved prediction of survival time (p value 1.9e-6; Figure 4). We also examined the effects of tumor stage, tumor grade, patient age, surgical outcome, and platinum status, and\",\n",
            "          \"grade, patient age, surgical outcome, and platinum status, and found that our proteomic signatures were not improved by inclusion of these variables. For each of these locus-specific models we analyzed the enrichment of genes and their regulatory\",\n",
            "          \"models we analyzed the enrichment of genes and their regulatory sequences associated with outcome, and found all four models had genes significantly enriched (Benjamini-Hochberg adjusted p value <0.05) in binding sites for the proliferation-associated\",\n",
            "          \"value <0.05) in binding sites for the proliferation-associated serum response factor (SRF), suggesting that SRF activity may be important in ovarian cancer outcome. Additionally, there was significant enrichment of proteins involved in the regulation of\",\n",
            "          \"enrichment of proteins involved in the regulation of actin cytoskeleton, apoptosis, and adherens junction (Benjamini-Hochberg adjusted p value <0.05). We examined the protein abundance and phosphorylation status of SRF between short and long surviving\",\n",
            "          \"phosphorylation status of SRF between short and long surviving groups and observed that they were higher in short surviving patients, but only slightly. Thus, although SRF alone was not predictive, the trends in SRF abundance and phosphorylation are\",\n",
            "          \"predictive, the trends in SRF abundance and phosphorylation are consistent with the observation of enrichment for proteins potentially regulated by SRF binding. Details of the larger gene set analysis are provided in Supplemental Information. Several\",\n",
            "          \"set analysis are provided in Supplemental Information. Several proteins shared across all the signatures are known to be involved in cancer processes. Catenin B2 (CTNNA2) is a cell-cell adhesion protein and tumor suppressor. The Rho GDP dissociation\",\n",
            "          \"adhesion protein and tumor suppressor. The Rho GDP dissociation inhibitor (GDI) beta (ARHGDIB) is involved in invasion and migration in many cancers, and overexpression correlates with progression in pancreatic carcinoma. Protein kinase C and casein\",\n",
            "          \"in pancreatic carcinoma. Protein kinase C and casein kinase substrate in neurons protein 2 (PACSIN2) is a repressor of cellular migration. Finally, the GTP-binding nuclear protein Ran, is a prognostic marker associated with increased survival in\",\n",
            "          \"is a prognostic marker associated with increased survival in epithelial ovarian cancer. The association of these previously described survival-related proteins with genes affected in trans by CNAs suggests a potential mechanism for the parallel activation\",\n",
            "          \"CNAs suggests a potential mechanism for the parallel activation of multiple pathways associated with poor prognosis in HGSC. As a comparison we applied the previously described Provar signature, comprising five protein and four phosphoproteins that showed\",\n",
            "          \"comprising five protein and four phosphoproteins that showed good survival prediction in the TCGA ovarian cancer dataset. We observed all proteins in the Provar signature in our proteomic data, but only one of the phosphosites (EGFR Y1173). Thus, we used\",\n",
            "          \"but only one of the phosphosites (EGFR Y1173). Thus, we used the RPPA data from the original signature (Figure S4B). We found that the Provar signature was prognostic of survival (Benjamini-Hochberg adjusted p value = 0.11) in the 67 patients examined\",\n",
            "          \"adjusted p value = 0.11) in the 67 patients examined here (those with phosphoprotein data and had somatic TP53 mutations), but the statistical power of Provar is not as high as the statistical power of signatures derived from the CNA trans-affected\",\n",
            "          \"power of signatures derived from the CNA trans-affected proteins in this dataset. In addition, integrating the Provar signature with the CNA signatures did not improve survival prediction (Figure S4C). Identification of proteins associated with\",\n",
            "          \"(Figure S4C). Identification of proteins associated with chromosomal structural abnormality The degree of chromosomal instability exhibited by a tumor can be represented by a calculated chromosome instability (CIN) index, as described previously for lung\",\n",
            "          \"instability (CIN) index, as described previously for lung cancer. Identification of proteins associated with CIN may provide information on the processes contributing to chromosomal instability, while the analysis of trans-affected proteins described\",\n",
            "          \"while the analysis of trans-affected proteins described above is more closely related to the downstream consequences of specific CNAs. Using a bootstrapping method described in Supplemental Information, we identified a proteomic signature including 128\",\n",
            "          \"Information, we identified a proteomic signature including 128 proteins (Table S7) showing significant correlation with CIN index (Benjamini-Hochberg adjusted p value <1e-4, Spearman correlation; Figure S5A). Functional annotation of these proteins\",\n",
            "          \"Figure S5A). Functional annotation of these proteins (Figure S5B) showed that proteins up-regulated in tumors with high CIN index were preferentially involved in chromatin organization (p value 6.90e-5), whereas proteins up-regulated in tumors with low\",\n",
            "          \"6.90e-5), whereas proteins up-regulated in tumors with low CIN index were more often associated with cell death (p value 2.13e-5). Correlation analysis of protein abundances and CIN indicated that a small number of proteins could account for the majority\",\n",
            "          \"that a small number of proteins could account for the majority of the CIN (Figure 5); two of the most strongly associated proteins, CHD4 and CHD5, are known to be involved in chromatin organization. Identification of proteins associated with HRD status\",\n",
            "          \"Identification of proteins associated with HRD status Since HRD is associated with susceptibility to PARP inhibitors and improved survival, we sought to elucidate systemic changes associated with HRD and identify biomarkers that might be used to stratify\",\n",
            "          \"with HRD and identify biomarkers that might be used to stratify patients for treatment. We defined tumors with HRD as having either germline or somatic mutations in BRCA1 or BRCA2, or BRCA1 promoter methylation, or homozygous deletion of PTEN, with an\",\n",
            "          \"promoter methylation, or homozygous deletion of PTEN, with an overall survival >1.5 years, while non-HRD patients were defined as lacking these genomic aberrations and with a follow-up or time-to-death <2.5 years; additional selection criteria include\",\n",
            "          \"time-to-death <2.5 years; additional selection criteria include available residual tissue volume and a tumor tissue contamination score estimated using copy number alterations. Applying differential dependency network (DDN) analysis on a set of 171 BRCA1-\",\n",
            "          \"dependency network (DDN) analysis on a set of 171 BRCA1- or BRCA2-related proteins curated from the literature and from the cBio portal, we identified a sub-network of 30 proteins that displayed co-expression patterns differentiating HRD from non-HRD\",\n",
            "          \"co-expression patterns differentiating HRD from non-HRD patients (Table S7; Figure 6). Several of the proteins in these modules are known to be involved in histone acetylation or deacetylation, e.g., HDAC1, RBBP4, RBBP7, and EP300, and HUS1. Although\",\n",
            "          \"e.g., HDAC1, RBBP4, RBBP7, and EP300, and HUS1. Although statistical association cannot distinguish between drivers and consequences of HRD status, the observed enrichment of proteins associated with histone acetylation motivated us to use an effective\",\n",
            "          \"with histone acetylation motivated us to use an effective database search strategy (Supplemental Information) to identify and quantify acetylated peptides from the global proteomic data. Comparative analysis of 399 acetylated peptides identified 15\",\n",
            "          \"Comparative analysis of 399 acetylated peptides identified 15 acetylated peptides with significant differences between the HRD and non-HRD samples (Table S7). Among these, dual acetylation at K12 and K16 of histone H4 showed a significant difference\",\n",
            "          \"at K12 and K16 of histone H4 showed a significant difference between HRD and non-HRD samples (Figure 6); differential acetylation of K12 and K16 was verified using synthetic peptides and targeted analysis using Sequential Windowed data independent\",\n",
            "          \"targeted analysis using Sequential Windowed data independent Acquisition of the Total High-resolution Mass Spectra (SWATH-MS). In cell line data, acetylation of H4 has previously been reported to be involved in the choice of DNA double strand break (DSB)\",\n",
            "          \"to be involved in the choice of DNA double strand break (DSB) repair pathways (homologous recombination or non-homologous end joining. This relationship is regulated partially by HDAC1, a protein also identified in the DDN analysis. We observed a\",\n",
            "          \"a protein also identified in the DDN analysis. We observed a significant enrichment of HDAC1 and its co-regulated proteins in tumors with HRD and low H4 acetylation relative to non-HRD tumors with high H4 acetylation (DDN analysis: permutation tests, p\",\n",
            "          \"with high H4 acetylation (DDN analysis: permutation tests, p value <0.05; differentially acetylated peptides: t-tests, p value <0.05 with an estimated FDR <0.5% by bootstrap/permutation tests). The combined observations of increased HDAC1 and associated\",\n",
            "          \"The combined observations of increased HDAC1 and associated proteins at the pathway level, together with decreased acetylation of H4 in HRD patients at the PTM level, provide insight regarding the potential role of HDAC1 in modulating the choice of DSB\",\n",
            "          \"the potential role of HDAC1 in modulating the choice of DSB repair pathways. Phosphoproteomic analysis of pathways associated with survival An initial set of 24,464 different phosphopeptides (21,298 unique phosphorylation sites) contained within 4,420\",\n",
            "          \"(21,298 unique phosphorylation sites) contained within 4,420 proteins having data on net differences in phosphorylation (Table S2) was obtained. Since ischemia of the TCGA tumor samples may alter phosphopeptide abundance, we removed any phosphorylation\",\n",
            "          \"alter phosphopeptide abundance, we removed any phosphorylation sites previously shown to be altered by ischemia. We also removed three tumor samples having substantially higher than average missing values and two tumor samples lacking somatic TP53\",\n",
            "          \"missing values and two tumor samples lacking somatic TP53 mutations (Supplemental Information). This yielded a final set of 7,675 different phosphopeptides (6,802 unique phosphorylation sites) from 2,324 proteins (Table S2) that were mapped to pathways\",\n",
            "          \"from 2,324 proteins (Table S2) that were mapped to pathways via the NCI PID. The average net phosphorylation of all phosphopeptides mapping to a given pathway (i.e., the statistical average across all phosphoproteins known to be in the pathway, rather\",\n",
            "          \"across all phosphoproteins known to be in the pathway, rather than the statistically significant individual proteins) was used as a measure of pathway activity and compared between short survivors (deceased surviving <3 years; N=17) and long survivors\",\n",
            "          \"(deceased surviving <3 years; N=17) and long survivors (patients surviving >5 years; N=19) using both proteomic and phosphoproteomic data. An issue in ranking the pathways is the distinction between phosphopeptides that are unique to a single pathway,\",\n",
            "          \"between phosphopeptides that are unique to a single pathway, such as the receptor peptides themselves, vs. peptides that map to proteins shared between pathways. In order to address this issue, we performed two analyses: one of all proteins associated\",\n",
            "          \"we performed two analyses: one of all proteins associated with a specific pathway in the NCID database, and a second analysis in which proteins common to multiple pathways were excluded. Figure 7A shows a ranking of activated pathways, as inferred from\",\n",
            "          \"7A shows a ranking of activated pathways, as inferred from the analysis of pathway specific components (i.e., differences in mRNA, protein and phosphoprotein levels) in short vs. long survivors. Fifteen pathways showed increased phosphorylation at\",\n",
            "          \"survivors. Fifteen pathways showed increased phosphorylation at Benjamini-Hochberg adjusted p values <0.01, in contrast with transcriptional data for the same samples which yielded only one statistically significantly increased pathway, androgen receptor\",\n",
            "          \"significantly increased pathway, androgen receptor signaling, which was also increased at the phosphoprotein level (Figure 7A and Supplemental Information). All pathways indicated in Figure 7A were increased in association with short survival relative to\",\n",
            "          \"were increased in association with short survival relative to long survival, though some individual components within a pathway showed opposite changes (e.g. JNK1 transcript and protein levels, see Figure 7B). The RhoA-regulatory, PDGFRB, and\",\n",
            "          \"levels, see Figure 7B). The RhoA-regulatory, PDGFRB, and integrin-like kinase pathways emerged as the most activated in this analysis (Figure 7A). Figure 7B shows the common and unique elements of the PDGFRB pathway, and the trends in mRNA, protein, and\",\n",
            "          \"of the PDGFRB pathway, and the trends in mRNA, protein, and phosphoprotein levels in the comparison of short vs. long survivors, illustrating the benefit of combined protein and phosphoprotein measurements, as well as the substantial contribution of\",\n",
            "          \"measurements, as well as the substantial contribution of protein phosphorylation data to the overall analysis. We compared our results to the 31 phosphoproteins represented on the RPPA arrays used in the original TCGA ovarian cancer analysis; RPPA\",\n",
            "          \"arrays used in the original TCGA ovarian cancer analysis; RPPA identified differential phosphorylation of ERK1, RAF1, and STAT3 in the same patient samples analyzed by MS. For short surviving vs. long surviving patients, ERK1 showed statistically\",\n",
            "          \"vs. long surviving patients, ERK1 showed statistically significantly increased phosphorylation by RPPA, on the same ERK1 phosphopeptides as found using MS, while AKT1, RAF1 and STAT3 showed increased phosphorylation by both RPPA and MS-based analyses, but\",\n",
            "          \"phosphorylation by both RPPA and MS-based analyses, but based upon distinct phosphopeptides (Table S2). Good overlap was observed between RPPA and MS results for phosphoproteins on the RPPA array, but we did not observe three of the specific phosphosites\",\n",
            "          \"but we did not observe three of the specific phosphosites from the Provar survival signature in our MS data, suggesting that proteomics is more sensitive to overall pathway activity than specific regulatory events in the pathway itself. However, many of\",\n",
            "          \"regulatory events in the pathway itself. However, many of the phosphopeptides identified by MS phosphoproteomics as components of the PDGFR-beta pathway were not represented in the RPPA, illustrating the ability to identify phosphorylation events not\",\n",
            "          \"illustrating the ability to identify phosphorylation events not readily accessible with currently available antibodies (Figure 7B). DISCUSSION The addition of proteomic information, including post-translational modifications, to the rich genomic and\",\n",
            "          \"post-translational modifications, to the rich genomic and transcriptomic data available from the ovarian HGSC samples analyzed by TCGA has provided additional insights into the biology of HGSC, including identification of changes at the level of pathway\",\n",
            "          \"including identification of changes at the level of pathway activation. Integration of genomic, proteomic, and phosphoproteomic measurements identified differentially regulated pathways and functional modules that displayed significant associations with\",\n",
            "          \"functional modules that displayed significant associations with patient outcomes, including survival and HRD status. Cox proportional hazard analysis of proteins associated with CNAs identified overall survival signatures enriched for targets of the\",\n",
            "          \"overall survival signatures enriched for targets of the proliferation-associated transcription factor SRF, and integrated proteomic, phosphoproteomic and transcriptomic data identified pathways that differentiated patients on the basis of survival,\",\n",
            "          \"pathways that differentiated patients on the basis of survival, including the PDGFR-beta signaling pathway associated with angiogenesis, the RhoA regulatory and integrin-linked kinase pathways associated with cell mobility and invasion, and pathways\",\n",
            "          \"associated with cell mobility and invasion, and pathways associated with chemokine signaling and adaptive immunity. Proteins associated with cell invasion and motility emerged from both the integration of CNAs with protein abundance data and the\",\n",
            "          \"the integration of CNAs with protein abundance data and the phosphoproteomic investigation of up-regulated pathways in short versus long survivors. Despite the complexity of genomic alterations that characterize HGSC, these analyses suggest a functional\",\n",
            "          \"that characterize HGSC, these analyses suggest a functional convergence on a subset of key pathways. The association of increased invasiveness and motility with short overall survival in this study may help to explain more aggressive mechanisms of\",\n",
            "          \"in this study may help to explain more aggressive mechanisms of dissemination in ovarian cancer, including recently reported hematogenous metastasis, in addition to previously described lymphatic spread and known peritoneal spread by direct extension.\",\n",
            "          \"spread and known peritoneal spread by direct extension. Focusing on functional modules has also revealed potential drivers of HGSC and more robust signatures for potentially stratifying ovarian HGSC patients into distinct cancer phenotypes to inform\",\n",
            "          \"ovarian HGSC patients into distinct cancer phenotypes to inform therapeutic management. For example, the identification of specific acetylation events associated with HRD, e.g., the simultaneous acetylation of K12 and K16 on histone H4, may provide an\",\n",
            "          \"acetylation of K12 and K16 on histone H4, may provide an alternative biomarker of HRD and a rationale for the selection of patients in future clinical trials of HDAC inhibitors alone or in combination with PARP inhibition. This may help to resolve the\",\n",
            "          \"combination with PARP inhibition. This may help to resolve the current discrepancy between the initial observation of limited single agent activity of HDAC inhibitors in ovarian cancer, and more recent findings of a >40% response rate when used in\",\n",
            "          \"and more recent findings of a >40% response rate when used in combination with cytotoxic chemotherapy in platinum-resistant patients. Similarly, comprehensive interrogation of protein phosphorylation identified multiple pathways with significantly\",\n",
            "          \"phosphorylation identified multiple pathways with significantly increased phosphorylation in patients with poor clinical outcomes. Specifically, the observed activation of PDGFR pathways in a subset of patients with short overall survival could\",\n",
            "          \"in a subset of patients with short overall survival could potentially stratify patients for selective enrollment in trials of anti-angiogenic therapy, which is particularly relevant to current controversies over the use of bevacuzimab as a first-line\",\n",
            "          \"controversies over the use of bevacuzimab as a first-line therapy in ovarian cancer. Overall, this work illustrates the ability of proteomics to complement genomics in providing additional insights into pathways and processes that drive ovarian cancer\",\n",
            "          \"insights into pathways and processes that drive ovarian cancer biology and how these pathways are altered in correspondence with clinical phenotypes. The comprehensive proteomic measurements for the HGSC tumor samples provide a public resource of\",\n",
            "          \"for the HGSC tumor samples provide a public resource of information. Importantly, the ability to identify PTMs revealed a strong association between specific histone acetylation events and the HRD phenotype. In addition, the enhanced view of pathway\",\n",
            "          \"the HRD phenotype. In addition, the enhanced view of pathway activity provided by measurements of protein phosphorylation provides a foundation for linking genotype to proteotype, and ultimately to phenotype for understanding the molecular basis of\",\n",
            "          \"to phenotype for understanding the molecular basis of cancer. EXPERIMENTAL PROCEDURES Full methods are available in Extended Experimental Procedures. Tumor Samples All tumor samples for the current study were obtained through the TCGA Biospecimen Core\",\n",
            "          \"current study were obtained through the TCGA Biospecimen Core Resource as described previously. Samples were selected based on overall survival, HRD status and availability. Quantitative Proteomics Tissue proteins were extracted and digested with trypsin;\",\n",
            "          \"Tissue proteins were extracted and digested with trypsin; at each site, a portion of each resulting sample was also used to create a pooled reference in which each tumor sample contributed an equal percentage by peptide mass. The pooled reference sample\",\n",
            "          \"equal percentage by peptide mass. The pooled reference sample was included in each multiplexed, isobaric labeling experiment to enable cross-experiment comparison in the entire sample cohort. The patient samples and the pooled reference sample were each\",\n",
            "          \"The patient samples and the pooled reference sample were each labeled by different iTRAQ reagents (Sciex), combined, fractionated, and split for integrated, quantitative global proteome (10%) and phosphoproteome (90%) analysis using LC-MS/MS on an\",\n",
            "          \"(10%) and phosphoproteome (90%) analysis using LC-MS/MS on an Orbitrap Velos mass spectrometer (Thermo Scientific). Raw data were processed for peptide identification by database searching and identified peptides were assembled as proteins and mapped to\",\n",
            "          \"identified peptides were assembled as proteins and mapped to gene identifiers for proteogenomic comparisons. Quantitation was achieved by comparing the iTRAQ reporter ion intensities in each sample. Comparison of mRNA and Protein Subtypes A model-based\",\n",
            "          \"sample. Comparison of mRNA and Protein Subtypes A model-based clustering approach was used to model the protein abundance data as a mixture of subtypes. Bayesian information criteria, statistical resampling and VISDA-based sub-phenotype clustering\",\n",
            "          \"statistical resampling and VISDA-based sub-phenotype clustering approaches provided similar results with a stable optimization on five clusters as determined by consensus clustering. WGCNA analysis was performed to infer genes or gene networks that drive\",\n",
            "          \"was performed to infer genes or gene networks that drive subtyping into 5 clusters, followed by correlation with subtype as a trait. Integration of Proteomics and CNA data Spearman correlation coefficients and corresponding adjusted p values were\",\n",
            "          \"coefficients and corresponding adjusted p values were calculated for each protein/transcript by CNA locus, and gene set enrichment analysis was used to infer function for groups of proteins significantly correlated with a given CNA locus. Regression\",\n",
            "          \"significantly correlated with a given CNA locus. Regression analysis was applied to the list of trans-affected protein correlated with each CNA to generate parsimonious Cox proportional hazards models with maximal predictive ability, using the PNNL data\",\n",
            "          \"models with maximal predictive ability, using the PNNL data for training and JHU data for testing. Kaplan-Meier plots were used to visualize performance in predicting overall survival and progression-free survival. A CIN index was calculated for each\",\n",
            "          \"progression-free survival. A CIN index was calculated for each sample as the mean absolute values of copy number measurements at the 29,393 selected loci. Bootstrap resampling was used to select proteins correlated with CIN index at high confidence.\",\n",
            "          \"select proteins correlated with CIN index at high confidence. Protein Acetylation and HRD Acetylated peptides were identified by searching the global proteomics data for dynamic acetylation to lysines (+42 Da). Acetylation levels were compared between HRD\",\n",
            "          \"lysines (+42 Da). Acetylation levels were compared between HRD and non-HRD cases by t-test. Targeted proteomics (SWATH) was used to orthogonally quantify the acetylated peptide with synthetic peptides as internal standard. Survival-related Pathways\",\n",
            "          \"peptides as internal standard. Survival-related Pathways Analysis Phosphoproteome data from the short and long survivors were mapped to signaling pathways in the NCI pathway information database,(PID (http://pid.nci.nih.gov) using the gene names. For each\",\n",
            "          \"(http://pid.nci.nih.gov) using the gene names. For each signaling pathway in the PID, relative abundances for all phosphopeptides mapping to any pathway component were identified and separated into short and long survivor groups. The difference in\",\n",
            "          \"into short and long survivor groups. The difference in distributions between the two sets of pathway-specific peptides, i.e., those associated with either short survivors or long long survivors, was then assessed using a two-tailed t test. Similar\",\n",
            "          \"survivors, was then assessed using a two-tailed t test. Similar enrichment analyses were also performed using protein abundance, mRNA abundance, and CNA. Data Repository All of the primary mass spectrometry data on TCGA tumor samples are deposited at the\",\n",
            "          \"spectrometry data on TCGA tumor samples are deposited at the CPTAC Data Coordinating Center as raw and mzML files and complete protein assembly data sets for public access (https://cptac-data-portal.georgetown.edu). Supplementary Material This is a PDF\",\n",
            "          \"Supplementary Material This is a PDF file of an unedited manuscript that has been accepted for publication. As a service to our customers we are providing this early version of the manuscript. The manuscript will undergo copyediting, typesetting, and\",\n",
            "          \"The manuscript will undergo copyediting, typesetting, and review of the resulting proof before it is published in its final form. Please note that during the production process errors may be discovered which could affect the content, and all legal\",\n",
            "          \"may be discovered which could affect the content, and all legal disclaimers that apply to the journal pertain.            Author Contributions          Study conception and design: HZ, TL, ZZ, RDS, DWC, KDR. Investigation: performed experiment or data\",\n",
            "          \"ZZ, RDS, DWC, KDR. Investigation: performed experiment or data collection: TL, SS, FY, JZ, LijunC, PS, PA, YT, MAG, TRC, CC, ST, SN, RJM, HengZ. Computation and statistical analysis: ZZ, SHP, BaiZ, JEM, VAP, LiC, DR, MEM, SWC, SW, JW, DLT, DF, VB, YW,\",\n",
            "          \"BaiZ, JEM, VAP, LiC, DR, MEM, SWC, SW, JW, DLT, DF, VB, YW, BingZ. Data interpretation and biological analysis: TL, HZ, SHP, ZZ, BaiZ, JEM, VAP, LiC, DR, SN, CW, IMS, AP, MPS, DAL, RDS, DWC, KDR. Writing/manuscript preparation and revision: HZ, TL, ZZ,\",\n",
            "          \"KDR. Writing/manuscript preparation and revision: HZ, TL, ZZ, SHP, JEM, DR, VAP, LeslieC, HR, IMS, AP, MPS, DAL, RDS, DWC, KDR. Supervision and administration: HR, ESB, TH, RCR, LS, RDS, DWC, KDR. The authors declare no competing financial interests.\",\n",
            "          \"KDR. The authors declare no competing financial interests.            SUPPLEMENTAL INFORMATION          Supplemental information includes Extended Experimental Procedures, 5 figures, and 8 tables and can be found with this article online. Correlations\",\n",
            "          \"tables and can be found with this article online. Correlations between mRNA and protein abundance in TCGA tumors mRNA and protein were correlated across all the samples, resulting in 90.6% positive correlations, and 79.4% were significantly correlated\",\n",
            "          \"positive correlations, and 79.4% were significantly correlated (Benjamini-Hochberg adjusted p value <0.01), with a mean Spearman\\u2019s correlation of 0.38 and a median value of 0.45 (top panel). Different biological pathways and processes showed significantly\",\n",
            "          \"biological pathways and processes showed significantly different levels of correlation (bottom panel). Metabolic pathways and the interferon response displayed high mRNA-protein correlation, and ribosome, mRNA splicing, oxidative phosphorylation, and\",\n",
            "          \"and ribosome, mRNA splicing, oxidative phosphorylation, and complement and coagulation cascade were poorly correlated. The mean correlation is shown in parentheses followed by Benjamini-Hochberg adjusted p values calculated using a Kolmogorov\\u2013Smirnov test\",\n",
            "          \"adjusted p values calculated using a Kolmogorov\\u2013Smirnov test following the functional group names from MSIGDB. Blue bars indicate positive correlations and yellow indicates negative correlations; individual proteins (represented as bars on the x-axis) are\",\n",
            "          \"individual proteins (represented as bars on the x-axis) are sorted by correlation from low to high (bottom panel). Proteomic subtypes and corresponding driving protein modules Global proteomics abundance is shown in a heatmap with TCGA samples represented\",\n",
            "          \"abundance is shown in a heatmap with TCGA samples represented by columns ordered by protein subtype where rows represent proteins. Color of each cell indicates z-score (log2 of relative abundance scaled by proteins\\u2019 standard deviation) of the protein in\",\n",
            "          \"scaled by proteins\\u2019 standard deviation) of the protein in that sample; red is increased and blue is decreased (relative to the pooled reference). Transcriptome-based subtypes (Verhaak et al., 2013) and the proposed proteomic subtypes are indicated in\",\n",
            "          \"al., 2013) and the proposed proteomic subtypes are indicated in color above the heatmap. WGCNA-derived modules are delineated with the row color panel and annotated according to the pathways based on enrichment of KEGG and Reactome ontologies. Functional\",\n",
            "          \"based on enrichment of KEGG and Reactome ontologies. Functional impact of copy number alterations The top panel shows the correlation of CNAs to protein abundance (right) or mRNA (left) with significant positive correlations in red and negative\",\n",
            "          \"with significant positive correlations in red and negative correlations in blue (Benjamini-Hochberg adjusted p value <0.01, Spearman\\u2019s correlation coefficient). The x-axis plots the 29,393 CNAs obtained from TCGA. The y-axis plots 3,202 proteins. Genes\",\n",
            "          \"CNAs obtained from TCGA. The y-axis plots 3,202 proteins. Genes are ordered by chromosomal location on both the x- and y-axes. The bottom panel shows the summed number of significantly correlated proteins (or mRNA) for each individual CNA. In blue is\",\n",
            "          \"proteins (or mRNA) for each individual CNA. In blue is shown the total number, in black are those genes significantly correlated as both mRNA and protein. Where blue and black lines have similar magnitude, e.g., the hotspot on chromosome 20, the CNA\",\n",
            "          \"similar magnitude, e.g., the hotspot on chromosome 20, the CNA associations are shared between protein and mRNA. Where a strong blue line has no mirrored black line, e.g., the protein hotspot on chromosome 2, the associations with CNA are largely unique.\",\n",
            "          \"on chromosome 2, the associations with CNA are largely unique. Kaplan-Meier plot of overall survival stratified by CNA-derived signatures A survival plot is shown for a consensus of the four best signatures (see Figure S4A and Table S6) identified from\",\n",
            "          \"best signatures (see Figure S4A and Table S6) identified from analysis of proteins affected in trans by CNAs. Models were trained using a lasso-based Cox proportional hazards model on the samples from PNNL, and shown are the survival curves from these\",\n",
            "          \"samples from PNNL, and shown are the survival curves from these models applied to the non-overlapping data from the JHU analysis, with the up (red; above the median) and down (blue, below the median) signatures each representing 45 patients. A vote was\",\n",
            "          \"median) signatures each representing 45 patients. A vote was taken among the four most predictive signatures, and the results of this vote are shown. Probability of death is shown on the y-axis vs. survival time in years on the x-axis. Shaded ribbons\",\n",
            "          \"y-axis vs. survival time in years on the x-axis. Shaded ribbons denote 95% confidence intervals. Number of CNAs statistically explained by proteins significantly associated with CIN index A total of 128 proteins selected for their association with CIN\",\n",
            "          \"A total of 128 proteins selected for their association with CIN were arranged along the chromosomal locations of their corresponding genes (y-axis). The length of the light grey horizontal lines indicates the number of CNAs significantly correlated with\",\n",
            "          \"indicates the number of CNAs significantly correlated with the protein across all chromosomes except for the protein\\u2019s corresponding gene coding region (x-axis). The top-ranked proteins are annotated and highlighted with dark lines to show the\",\n",
            "          \"are annotated and highlighted with dark lines to show the bootstrap-estimated 95% confidence intervals. DDN analysis and lysine-acetylation analysis between HRD and non-HRD patients DDN analysis revealed a sub-network of proteins that displayed distinct\",\n",
            "          \"revealed a sub-network of proteins that displayed distinct co-expression patterns between HRD and non-HRD patients, where purple connections indicate protein correlations that exist only in HRD samples and blue connections indicate protein correlations\",\n",
            "          \"HRD samples and blue connections indicate protein correlations that exist only in non-HRD patients. The links between the nodes are drawn with two different thickness indicative of whether the connections meet the significance threshold of 0.05 (light\",\n",
            "          \"the connections meet the significance threshold of 0.05 (light lines) or 0.01 (bold lines). Proteins with blue dotted circles are known to be involved in histone acetylation or deacetylation. Further, identification and quantitation of lysine-acetylated\",\n",
            "          \"Further, identification and quantitation of lysine-acetylated peptides in global proteomics data showed that acetylation level at K12 and K16 of histone H4 are significantly different between HRD and non-HRD samples, suggesting a role of histone H4\",\n",
            "          \"HRD and non-HRD samples, suggesting a role of histone H4 acetylation (together with HDAC1) in modulating the choice of DSB repair mechanisms. Pathway analysis associated with patient survival A) Pathway components were statistically analyzed using a\",\n",
            "          \"A) Pathway components were statistically analyzed using a two-sided t test between samples from patients surviving <3 years (short survival) and patients surviving >5 years (long survival) for differences in CNA, mRNA expression, protein or\",\n",
            "          \"survival) for differences in CNA, mRNA expression, protein or phosphorylation abundance. All significant pathways for mRNA and protein are shown (Benjamini-Hochberg adjusted p value <0.05) and the most significant pathways for phosphoproteomics are\",\n",
            "          \"and the most significant pathways for phosphoproteomics are plotted on the x axis as the \\u2212log of the p value. Results show that phosphorylation provides additional information about the functional state of tumors. B) Simplified PDGFR-beta pathway showing\",\n",
            "          \"state of tumors. B) Simplified PDGFR-beta pathway showing differences between short and long survivor groups for protein abundance (p), transcript abundance (m), and phosphopeptide abundance (circled P).\"\n",
            "        ]\n",
            "      }\n",
            "    ]\n",
            "  },\n",
            "  \"PDC000304\": {\n",
            "    \"study_id\": \"debd14db-1702-4bd2-b3cf-7349739f7c6b\",\n",
            "    \"pdc_study_id\": \"PDC000304\",\n",
            "    \"study_submitter_id\": \"Therapeutic Targets in Breast Cancer Xenografts -  Phosphoproteome\",\n",
            "    \"program_id\": \"10251935-5540-11e8-b664-00a098d917f8\",\n",
            "    \"project_id\": \"8dbcf809-3df2-4c57-a9d5-8d58d5824ffa\",\n",
            "    \"study_name\": \"Therapeutic Targets in Breast Cancer Xenografts -  Phosphoproteome\",\n",
            "    \"program_name\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"project_name\": \"CPTAC2-Other\",\n",
            "    \"disease_type\": \"Other\",\n",
            "    \"primary_site\": \"Not Reported\",\n",
            "    \"analytical_fraction\": \"Phosphoproteome\",\n",
            "    \"experiment_type\": \"iTRAQ4\",\n",
            "    \"embargo_date\": null,\n",
            "    \"cases_count\": 27,\n",
            "    \"aliquots_count\": 27,\n",
            "    \"protocol_id\": \"481c6411-677c-4748-8a6e-b569ab524861\",\n",
            "    \"protocol_submitter_id\": \"Breast Cancer Xenografts -  Phosphoproteome\",\n",
            "    \"program_submitter_id\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"protocol_name\": \"Breast Cancer Xenografts -  Phosphoproteome\",\n",
            "    \"protocol_date\": \"2021-05-06\",\n",
            "    \"publications\": [\n",
            "      {\n",
            "        \"publication_id\": \"718f77fb-b329-11eb-9f63-0a2b20c3a9e9\",\n",
            "        \"pubmed_id\": \"28348404\",\n",
            "        \"doi\": \"10.1038/ncomms14864\",\n",
            "        \"title\": \"Proteogenomic integration reveals therapeutic targets in breast cancer xenografts\",\n",
            "        \"journal\": \"Nature Communications\",\n",
            "        \"journal_url\": \"https://www.nature.com/articles/ncomms14864\",\n",
            "        \"year\": \"2017\",\n",
            "        \"abstract\": \"Recent advances in mass spectrometry (MS) have enabled extensive analysis of cancer proteomes. Here, we employed quantitative proteomics to profile protein expression across 24 breast cancer patient-derived xenograft (PDX) models. Integrated proteogenomic analysis shows positive correlation between expression measurements from transcriptomic and proteomic analyses; further, gene expression-based intrinsic subtypes are largely re-capitulated using non-stromal protein markers. Proteogenomic analysis also validates a number of predicted genomic targets in multiple receptor tyrosine kinases. However, several protein/phosphoprotein events such as overexpression of AKT proteins and ARAF, BRAF, HSP90AB1 phosphosites are not readily explainable by genomic analysis, suggesting that druggable translational and/or post-translational regulatory events may be uniquely diagnosed by MS. Drug treatment experiments targeting HER2 and components of the PI3K pathway supported proteogenomic response predictions in seven xenograft models. Our study demonstrates that MS-based proteomics can identify therapeutic targets and highlights the potential of PDX drug response evaluation to annotate MS-based pathway activities.\",\n",
            "        \"full_text\": [\n",
            "          \"Proteogenomic integration reveals therapeutic targets in breast cancer xenografts Recent advances in mass spectrometry (MS) have enabled extensive analysis of cancer proteomes. Here, we employed quantitative proteomics to profile protein expression across\",\n",
            "          \"quantitative proteomics to profile protein expression across 24 breast cancer patient-derived xenograft (PDX) models. Integrated proteogenomic analysis shows positive correlation between expression measurements from transcriptomic and proteomic analyses;\",\n",
            "          \"measurements from transcriptomic and proteomic analyses; further, gene expression-based intrinsic subtypes are largely re-capitulated using non-stromal protein markers. Proteogenomic analysis also validates a number of predicted genomic targets in\",\n",
            "          \"also validates a number of predicted genomic targets in multiple receptor tyrosine kinases. However, several protein/phosphoprotein events such as overexpression of AKT proteins and ARAF, BRAF, HSP90AB1 phosphosites are not readily explainable by genomic\",\n",
            "          \"HSP90AB1 phosphosites are not readily explainable by genomic analysis, suggesting that druggable translational and/or post-translational regulatory events may be uniquely diagnosed by MS. Drug treatment experiments targeting HER2 and components of the\",\n",
            "          \"Drug treatment experiments targeting HER2 and components of the PI3K pathway supported proteogenomic response predictions in seven xenograft models. Our study demonstrates that MS-based proteomics can identify therapeutic targets and highlights the\",\n",
            "          \"proteomics can identify therapeutic targets and highlights the potential of PDX drug response evaluation to annotate MS-based pathway activities. Patient-derived xenografts recapitulate major genomic signatures and transcriptome profiles of their original\",\n",
            "          \"genomic signatures and transcriptome profiles of their original tumours. Here, the authors, performing proteomic and phosphoproteomic analyses of 24 breast cancer PDX models, demonstrate that druggable candidates can be identified based on a comprehensive\",\n",
            "          \"druggable candidates can be identified based on a comprehensive proteogenomic profiling. Profiling of somatic alteration by next-generation DNA sequencing (NGS) has entered clinical practice with the promise of rapid diagnosis of druggable somatic genomic\",\n",
            "          \"the promise of rapid diagnosis of druggable somatic genomic alterations for personalized cancer treatment. For example, recent analysis of 4,068 samples from 16 cancer types suggested that repurposing approved drugs based on genomic alterations could\",\n",
            "          \"repurposing approved drugs based on genomic alterations could provide individualized treatment options for around 40% of tumours. However, clinical evidence for this proposition is limited and has been slow to develop. Further, the signalling and\",\n",
            "          \"and has been slow to develop. Further, the signalling and biological effects of somatic mutations are not routinely determined in human tumour samples even though this is a consideration for rational drug design, response prediction and target\",\n",
            "          \"for rational drug design, response prediction and target prioritization. Finally, druggable genomic alterations are not detected in the majority of cases tested by NGS. Comprehensive proteomic analyses provide a potentially valuable approach to validate\",\n",
            "          \"analyses provide a potentially valuable approach to validate genomic findings as likely biological drivers and to discover discover opportunities for targeted treatment. Patient-derived xenografts (PDXs) in immunodeficient mice maintain the histological\",\n",
            "          \"(PDXs) in immunodeficient mice maintain the histological and molecular heterogeneity of the progenitor human tumour and cytotoxic drug responsiveness is often a transplantable phenotype. Our previous studies have shown that breast tumour PDXs recapitulate\",\n",
            "          \"studies have shown that breast tumour PDXs recapitulate major genomic signatures and transcriptome profiles of their original breast tumours. Moreover, drug responses to endocrine therapy in breast cancer PDXs resembled that observed in the corresponding\",\n",
            "          \"breast cancer PDXs resembled that observed in the corresponding patient and endocrine therapy resistance patterns were associated with aberrations in the ESR1 gene. While comprehensive proteomic characterization of PDX is still lacking, recent studies\",\n",
            "          \"characterization of PDX is still lacking, recent studies using reverse phase protein array have identified similar protein profiles between PDX and primary tumours. These studies collectively suggest that the PDX approach is a potentially valuable\",\n",
            "          \"suggest that the PDX approach is a potentially valuable preclinical model for identification and testing of therapeutic targets. Recent advances in mass-spectrometry provide an unprecedented opportunity for antibody-independent proteome profiling with\",\n",
            "          \"opportunity for antibody-independent proteome profiling with approximately 80% of all proteins in major human tissues quantifiable by this technique. Analyses of the global proteome in tandem with genomic profiles generated through TCGA have yielded key\",\n",
            "          \"with genomic profiles generated through TCGA have yielded key insights on the molecular etiology of colorectal cancer, breast cancer and ovarian cancer. Our recent proteogenomic characterization of 105 human breast cancers linked 5q-loss to elevated EGFR\",\n",
            "          \"of 105 human breast cancers linked 5q-loss to elevated EGFR expression and identified overexpression of phosphorylated kinases including CDK12, PAK1 and ARAF as potential biological drivers. However, current proteogenomic analyses of human tissues are\",\n",
            "          \"However, current proteogenomic analyses of human tissues are limited by the fact that therapeutic hypotheses were not being addressed in these initial analyses. As patient treatments were heterogeneous, links between treatment and proteogenomic findings\",\n",
            "          \"links between treatment and proteogenomic findings are difficult to establish in banked tumour specimens. The PDX system therefore provides an early opportunity to determine whether a proteogenomic approach can produce more robust therapeutic hypotheses\",\n",
            "          \"approach can produce more robust therapeutic hypotheses and predictive biomarkers for individual patients than that achieved by genomic analysis alone. In this study, we generated high-coverage proteome and phosphoproteome data for 24 PDX models\",\n",
            "          \"proteome and phosphoproteome data for 24 PDX models representing different breast cancer subtypes using both isobaric mass-tag labelling and label-free proteomic methods for cross-validated identification of 12,794 proteins and 56,874 phosphorylation\",\n",
            "          \"identification of 12,794 proteins and 56,874 phosphorylation sites. After filtering for observation in at least 10 out of 24 samples 10,069 proteins and 36,609 phosphorylation sites with their relative abundances quantified across tumours, were used in\",\n",
            "          \"relative abundances quantified across tumours, were used in subsequent analyses in this study. A substantial number of druggable protein/phosphoprotein events were uniquely identified. Further, similar proteomic/phosphoproteomic signatures were observed\",\n",
            "          \"similar proteomic/phosphoproteomic signatures were observed in patient breast cancer samples indicating potential clinical relevance. In selected PDX models exhibiting proteomic and/or phosphoproteomic up-regulation events of HER2 or the PI3K pathway, we\",\n",
            "          \"up-regulation events of HER2 or the PI3K pathway, we validated the efficacy of targeted inhibitors in suppressing PDX growth. Taken together, these observations show that comprehensive proteogenomic profiling has potential to identify new targets for\",\n",
            "          \"profiling has potential to identify new targets for individualized treatment approaches in cancer. Results Proteogenomic coverage of breast cancer xenografts We selected 24 PDX models established from primary or metastatic breast tumours for comprehensive\",\n",
            "          \"from primary or metastatic breast tumours for comprehensive proteogenomic characterization (Fig. 1a). The human patient cohort was composed of 10 basal, 1 claudin-low (CLDN-low), 9 luminal B and 4 HER2-enriched (HER2-E) breast tumours based on PAM50\",\n",
            "          \"B and 4 HER2-enriched (HER2-E) breast tumours based on PAM50 expression subtyping (Supplementary Data 1). We conducted DNA and RNA sequencing respectively for 23 PDX models and in one case Sanger DNA sequencing of hotspot mutations. Isotope Tagging for\",\n",
            "          \"Sanger DNA sequencing of hotspot mutations. Isotope Tagging for Relative and Absolute Quantitation (iTRAQ 4-plex) was completed for all 24 PDXs for discovery and label-free quantification (LFQ) for 18 PDXs for validation and confirmation (Supplementary\",\n",
            "          \"for 18 PDXs for validation and confirmation (Supplementary Data 2). The comprehensive proteogenomic coverage allowed us to systematically assess the somatic mutation profile, copy-number variation (CNV), mRNA expression, protein expression, and\",\n",
            "          \"variation (CNV), mRNA expression, protein expression, and phosphosite levels and localizations. We examined somatic mutations in significantly mutated genes of breast cancer in 12 human tumours and their matching PDXs (Supplementary Data 2). Most key\",\n",
            "          \"and their matching PDXs (Supplementary Data 2). Most key somatic mutations in these genes were preserved (Fig. 1b), validating the genomic fidelity of these PDX models. However, recurrent breast cancer mutations were not detected in two PDX models (WHIM17\",\n",
            "          \"cancer mutations were not detected in two PDX models (WHIM17 and WHIM46). While sequencing data were not available for the matched progenitor human tumours, the germline SNPs of human blood normal samples matched with the PDX tumours, validating their\",\n",
            "          \"normal samples matched with the PDX tumours, validating their patients of origin. Follow-up histological and RNA-seq analyses suggested WHIM17 and WHIM46 are EBV-positive human lymphoproliferative cells arising in the NOD/SCID/\\u03b3 mouse strain (Methods,\",\n",
            "          \"cells arising in the NOD/SCID/\\u03b3 mouse strain (Methods, Supplementary Fig. 1). We then compared the variant allele fractions (VAFs) of exonic somatic mutations in human tumours and derived xenografts when both are available. We found comparable or higher\",\n",
            "          \"when both are available. We found comparable or higher VAFs in xenografts, potentially due to higher tumour purity and selection of some mutant alleles by loss of heterozygosity in PDX models (Fig. 1c). The relatively high, positive VAF correlation\",\n",
            "          \"models (Fig. 1c). The relatively high, positive VAF correlation (R=0.66) implied similarity between human samples and their respective xenografts, consistent with our previous report. Importantly, all PIK3CA and TP53 somatic mutations in the 12\",\n",
            "          \"Importantly, all PIK3CA and TP53 somatic mutations in the 12 originating human tumours were detected in respective xenografts with comparable or increased VAFs (Fig. 1c). Among the 22 breast cancer PDX models, 14, including 7 of the 8 basal tumours,\",\n",
            "          \"cancer PDX models, 14, including 7 of the 8 basal tumours, harboured TP53 mutations, 5 luminal B PDXs had ESR1 mutations, and 5 luminal B and 1 CLDN-low xenografts carried PIK3CA mutations. We quantified mRNA expression of 16,209 unique human genes from\",\n",
            "          \"We quantified mRNA expression of 16,209 unique human genes from RNA-seq data for these 24 models. We also applied two distinct MS data acquisition approaches (Supplementary Fig. 2a, Methods), iTRAQ (Supplementary Data 3) and LFQ (Supplementary Data 4), to\",\n",
            "          \"iTRAQ (Supplementary Data 3) and LFQ (Supplementary Data 4), to quantify the expression ratios of 12,794 human proteins (11,879 genes) and 8,648 proteins (8,035 genes), respectively. A total of 56,874 phosphosites were also confidently identified using\",\n",
            "          \"of 56,874 phosphosites were also confidently identified using iTRAQ (Supplementary Data 5). After filtering for observation in at least 10 out of 24 samples (4 out of 9 of the iTRAQ experiments, Supplementary Fig. 3), the relative abundances of 10,069\",\n",
            "          \"Supplementary Fig. 3), the relative abundances of 10,069 proteins and 36,609 phosphorylation sites were quantified across tumours by iTRAQ and used in subsequent analyses in this study. The technical replicates in the LFQ (WHIM2 and WHIM16) and the iTRAQ\",\n",
            "          \"replicates in the LFQ (WHIM2 and WHIM16) and the iTRAQ (WHIM13) sets showed high correlation in protein expression levels (R>0.85, Supplementary Fig. 2b,c,e). Further, phosphosite expressions also showed high correlations in technical replicates (WHIM13)\",\n",
            "          \"also showed high correlations in technical replicates (WHIM13) of the iTRAQ experiment (R=0.82, Supplementary Fig. 2d), validating the technical reproducibility of our proteomic and phosphoproteomic datasets. Of note, while iTRAQ and LFQ quantification\",\n",
            "          \"datasets. Of note, while iTRAQ and LFQ quantification were conducted separately and based on different features of the LC\\u2013MS data collected from the Q Exactive mass spectrometer (that is, reporter ion intensity and MS1 peak area, respectively), our\",\n",
            "          \"reporter ion intensity and MS1 peak area, respectively), our analysis showed reasonable correlation between the two measurements after normalization (R=0.61). The two datasets utilizing the same PDX models but different workflows enabled cross-method\",\n",
            "          \"same PDX models but different workflows enabled cross-method validation of global proteomic results. Proteogenomic integration and comparison We first evaluated the correlation between mRNA expression and protein abundance measurements from the iTRAQ\",\n",
            "          \"expression and protein abundance measurements from the iTRAQ experiment (Fig. 2a) for these 24 PDX models: 83.6% of the genes with sufficient data showed positive correlations with a median Pearson R=0.536 (Supplementary Fig. 4). We investigated whether\",\n",
            "          \"Pearson R=0.536 (Supplementary Fig. 4). We investigated whether the trend of mRNA-protein correlations were associated with specific KEGG pathways, finding that metabolic pathways involved in house-keeping functions are enriched for genes showing high,\",\n",
            "          \"in house-keeping functions are enriched for genes showing high, positive correlation. For example, genes in the glutathione metabolism pathway showed the highest enrichment for positive correlations (Kolmogorov\\u2013Smirnov test, false discovery rate\",\n",
            "          \"correlations (Kolmogorov\\u2013Smirnov test, false discovery rate (FDR)=5.6e-07). Interestingly, we observed that genes in the ribosome (FDR<2.2e-16), spliceosome (FDR=2.0e-13), RNA transport (FDR=1.7e-5), RNA polymerase (FDR=1.8e-4), and oxidative\",\n",
            "          \"(FDR=1.7e-5), RNA polymerase (FDR=1.8e-4), and oxidative phosphorylation (FDR<2.2e-16) pathways showed relatively lower correlation between mRNA and protein level. These pathways were enriched for genes that do not require translated proteins for their\",\n",
            "          \"for genes that do not require translated proteins for their biological functions. Similar pathway-specific pattern of positive and negative enrichment of correlations were observed by LFQ (Supplementary Fig. 5). The high degree of mRNA\\u2013protein correlation\",\n",
            "          \"Fig. 5). The high degree of mRNA\\u2013protein correlation observed in the PDX samples is consistent with recent results obtained for human breast tumours and colorectal cancer, suggesting that PDXs closely mimic the respective human breast tumours in their\",\n",
            "          \"PDXs closely mimic the respective human breast tumours in their relationship between mRNA and protein. We then examined the correlation of CNV, mRNA, and protein expression levels for several key genes for breast cancer biology: EGFR, ERBB2 (HER2), ESR1,\",\n",
            "          \"key genes for breast cancer biology: EGFR, ERBB2 (HER2), ESR1, GATA3, PGR, PIK3CA, AKT1/2/3, MTOR and TP53. In most cases, we observed consistent relationships between CNV, mRNA, and protein expression levels. Compared with PDXs of non-luminal subtypes,\",\n",
            "          \"expression levels. Compared with PDXs of non-luminal subtypes, luminal B breast cancer xenografts, as expected, showed higher mRNA and protein expression of ESR1 and PGR, consistent with their positive ER and PR status (Fig. 2b). Five out of the six\",\n",
            "          \"their positive ER and PR status (Fig. 2b). Five out of the six PIK3CA mutations were observed in luminal B PDXs, which tended to also show higher mRNA and protein expression levels of GATA3. In contrast, a larger proportion of basal PDXs expressed higher\",\n",
            "          \"In contrast, a larger proportion of basal PDXs expressed higher protein levels of EGFR. Strong HER2 expression at the mRNA, protein and phosphoprotein levels were detected in WHIM8 and WHIM35, both derived from HER2-positive breast cancers. Overall\",\n",
            "          \"WHIM35, both derived from HER2-positive breast cancers. Overall expression patterns for key genes were consistent with the clinical subtype diagnosis across CNV, mRNA, protein and phosphosite analyses (Supplementary Fig. 6). Proteomic subtyping of\",\n",
            "          \"analyses (Supplementary Fig. 6). Proteomic subtyping of xenografts and human breast tumours Molecular subtyping of breast cancer based on mRNA expression profiles has been shown to correlate with prognosis and has treatment implications. As expected,\",\n",
            "          \"with prognosis and has treatment implications. As expected, transcriptome clustering based on PAM50 genes from RNA-seq data largely reproduced intrinsic subtypes of the breast tumours (Fig. 3a; Supplementary Data 6). To explore proteomic and\",\n",
            "          \"(Fig. 3a; Supplementary Data 6). To explore proteomic and phosphoproteomic subtype classifications in xenograft models, we conducted unsupervised clustering of proteomes and phosphoproteomes of the 24 xenograft samples based upon the top 436 variably\",\n",
            "          \"of the 24 xenograft samples based upon the top 436 variably expressed proteins showing a s.d. greater than 2 from the iTRAQ proteome (Supplementary Data 7). Two distinct clusters emerged: one contained all basal tumours and the only CLDN-low tumour\",\n",
            "          \"one contained all basal tumours and the only CLDN-low tumour (WHIM12), while the other included all luminal B and HER2-E breast tumours; HER2-E tumours did not show a proteomic expression profile distinct from luminal B samples (Fig. 3b). Clustering\",\n",
            "          \"profile distinct from luminal B samples (Fig. 3b). Clustering analysis using the same gene markers from the transcriptome (Supplementary Fig. 7a) and the LFQ proteome (Supplementary Fig. 7b; Supplementary Data 8) further supported the separation into\",\n",
            "          \"7b; Supplementary Data 8) further supported the separation into these two proteomic subtypes, although the minor differences between the transcriptome and the proteome clustering suggested distinctions between mRNA and protein levels. The proteomic\",\n",
            "          \"distinctions between mRNA and protein levels. The proteomic subtypes defined by the top 968 most variably expressed mouse host proteins did not segregate based on luminal and basal subtypes (Supplementary Fig. 8; Supplementary Data 9). Besides differing\",\n",
            "          \"(Supplementary Fig. 8; Supplementary Data 9). Besides differing in mRNA expression, luminal and basal breast cancer PDXs also showed consistently distinct proteomic expression profiles, supporting their distinct biological origins. We then utilized the\",\n",
            "          \"their distinct biological origins. We then utilized the iTRAQ phosphopeptide expression data to infer phosphoproteomic subtypes. 1,737 unique phosphosites with s.d. greater than 2.5 were included to conduct hierarchical clustering (Fig. 3c; Supplementary\",\n",
            "          \"to conduct hierarchical clustering (Fig. 3c; Supplementary Data 10). These analyses of the phosphoproteomic data produced two major clusters segregating the luminal B and basal subtypes. Again, the WHIM37/WHIM47/WHIM26 group and the WHIM17/WHIM46 group,\",\n",
            "          \"the WHIM37/WHIM47/WHIM26 group and the WHIM17/WHIM46 group, as observed in the proteome clustering, grouped closely together. Gene and protein expression of lymphoid lineage markers showed high expression of CD20 and JAK3 in WHIM17 and WHIM46 (Methods,\",\n",
            "          \"high expression of CD20 and JAK3 in WHIM17 and WHIM46 (Methods, Supplementary Fig. 9), consistent with their positive EBV status and histological diagnosis (Supplementary Fig. 1) as human lymphoproliferative cells arising in an immunocompromised mouse\",\n",
            "          \"lymphoproliferative cells arising in an immunocompromised mouse background. Overall, while the basal and luminal clusters remain consistent, the hierarchical distances between PDX samples within the two major clusters differ between data types. The\",\n",
            "          \"within the two major clusters differ between data types. The departure of proteomic and phosphoproteome subtypes from mRNA expression-defined subtypes suggests independent layers of molecular heterogeneity provided by distinct proteomic analyses. We then\",\n",
            "          \"heterogeneity provided by distinct proteomic analyses. We then investigated whether human proteomic subtypes could be recapitulated in PDX models by including an additional 77 proteomes from TCGA human tumours that were processed concurrently in the iTRAQ\",\n",
            "          \"human tumours that were processed concurrently in the iTRAQ experiment (Methods). To reduce clustering bias imbued by the mouse contribution to the proteome in the PDX samples, we excluded proteins showing differential expression between human tumour and\",\n",
            "          \"showing differential expression between human tumour and xenograft (t-test, FDR\\u22640.3) (Supplementary Data 11). Additional requirements, including the presence of detectable protein in more than 10 samples and minimum difference of 2 s.d.'s in the merged\",\n",
            "          \"10 samples and minimum difference of 2 s.d.'s in the merged proteome, resulted in 133 proteins qualifying for un-supervised clustering. Consistent with the subtyping analysis of the 24 PDXs alone, we identified two major clusters: one that included all\",\n",
            "          \"alone, we identified two major clusters: one that included all but one basal breast tumour and the other comprised mostly of luminal tumours (Fig. 3d). Similar to the proteomic subtypes of the PDX cohorts, luminal tumours and HER-2 tumours did not show\",\n",
            "          \"the PDX cohorts, luminal tumours and HER-2 tumours did not show clear separation, although several sub-clusters were identified. Importantly, xenograft proteomes clustered adjacently to the human proteomes of their respective subtypes, validating the\",\n",
            "          \"human proteomes of their respective subtypes, validating the fidelity of basal and luminal proteomic signatures discovered in PDXs. To search for defining markers between the basal and luminal B subtypes, we conducted differential expression analysis\",\n",
            "          \"B subtypes, we conducted differential expression analysis between the PDXs of the respective subtypes. We found several proteomic markers that were differentially expressed in both the LFQ and iTRAQ datasets (t-test, FDR\\u22640.05), including SPR, GSTP1 and\",\n",
            "          \"and iTRAQ datasets (t-test, FDR\\u22640.05), including SPR, GSTP1 and SERPINB5 (Supplementary Fig. 10; Supplementary Data 12). We then conducted gene-set enrichment analysis based on the Reactome pathway database to investigate patterns of differential\",\n",
            "          \"pathway database to investigate patterns of differential expression (Methods). The basal subtype breast tumours were up-regulated in most of the significantly differentially-expressed pathways (FDR\\u22640.01) identified by both LFQ and iTRAQ datasets,\",\n",
            "          \"pathways (FDR\\u22640.01) identified by both LFQ and iTRAQ datasets, including extracellular matrix organization, cell cycle, and collagen formation. In comparison, the luminal B breast tumours showed higher expression in genes related to organelle biogenesis\",\n",
            "          \"higher expression in genes related to organelle biogenesis and membrane trafficking (Supplementary Data 13). Activated pathways revealed by phosphorylation profiles Cancer driving somatic events trigger major changes in downstream signalling to launch the\",\n",
            "          \"trigger major changes in downstream signalling to launch the tumorigenic cascade. To search for tumour-specific activated pathways in PDX tumours, we systematically evaluated and compared phosphoproteome profiles of gene sets from KEGG signalling pathways\",\n",
            "          \"profiles of gene sets from KEGG signalling pathways (Methods). Phosphorylation enrichment analysis identified 12 significantly activated pathways, including Ras, MAPK and NF\\u03baB signalling, in 4 xenografts (FDR\\u22640.01, Supplementary Data 14). WHIM9 exhibited\",\n",
            "          \"4 xenografts (FDR\\u22640.01, Supplementary Data 14). WHIM9 exhibited elevated phosphorylation of the MAPK signalling pathway (FDR=9.69e-6; Fig. 4a). Interestingly, WHIM9 carried a recurrent somatic mutation, KRAS p.A146V (refs), which may have driven canonical\",\n",
            "          \"mutation, KRAS p.A146V (refs), which may have driven canonical MAPK pathway activation. Further, WHIM12 exhibited activation of the Ras signalling pathway (FDR=4.28e-5), along with an outlier protein expression of MET, a receptor tyrosine kinase upstream\",\n",
            "          \"protein expression of MET, a receptor tyrosine kinase upstream of the Ras signalling pathway (Fig. 4b). Interestingly, WHIM17 and WHIM46, both harboured BTK and PLCG2 protein overexpression, exhibited overall high phosphorylation of the NF\\u03baB signalling\",\n",
            "          \"exhibited overall high phosphorylation of the NF\\u03baB signalling pathway (Fig. 4c,d; FDR=6.94e-3, 4.53e-5 respectively). This observation further supports the strong similarity between these two PDX models based on protein/phosphoprotein clustering (Fig.\",\n",
            "          \"two PDX models based on protein/phosphoprotein clustering (Fig. 3b,c) and their classification as EBV-positive lymphoproliferation. While pathways such as PI3K/AKT/MTOR are known to be activated across the majority of breast tumours, our analysis shows\",\n",
            "          \"across the majority of breast tumours, our analysis shows that other complementary tumorigenesis-related pathways including RAS/MAPK are also activated in a small set of breast tumours due to specific genomic or proteomic alterations, representing\",\n",
            "          \"due to specific genomic or proteomic alterations, representing alternative treatment opportunities. In addition, our phospho-proteomic analysis revealed activation of signalling pathways not readily predicted by genomic data. Complementary\",\n",
            "          \"pathways not readily predicted by genomic data. Complementary genomic/proteomic druggable targets We examined promising drug targets in each tumour by surveying the genome and expressed proteomes. Specifically, we compiled a list 76 druggable genes\",\n",
            "          \"proteomes. Specifically, we compiled a list 76 druggable genes (Supplementary Data 15), along with their respective drugs, from established public databases (Methods). Six PDXs, representing 20.8% of the tumours in this study, harboured druggable somatic\",\n",
            "          \"20.8% of the tumours in this study, harboured druggable somatic mutations, including PIK3CA p.H1047R and KRAS p.A146V in WHIM9, PIK3CA p.H1047R in WHIM16 and WHIM24, PIK3CA p.E545K in WHIM18, PIK3CA p.E542K in WHIM20 and SF3B1 p.K700E in WHIM26\",\n",
            "          \"in WHIM18, PIK3CA p.E542K in WHIM20 and SF3B1 p.K700E in WHIM26 (Supplementary Data 16). While activating mutations in oncogenes can be targeted by treatments, aberrantly overexpressed or activated protein products, such as HER2, also presents exploitable\",\n",
            "          \"protein products, such as HER2, also presents exploitable treatment opportunities. We sought genomic and proteomic evidence of overexpressed genes/proteins or proteins with highly phosphorylated sites. We defined outliers as expression values exceeding\",\n",
            "          \"sites. We defined outliers as expression values exceeding the 1.5 interquartile ranges (IQR) above the third quartile of the cohort (Variations of Box Plots), and rank ordered them by the outlier score. We further required CNV outliers to be validated\",\n",
            "          \"outlier score. We further required CNV outliers to be validated with outlier expression in either the mRNA or protein level to rule out up-regulation events due to technical artifacts or passenger events (Methods). mRNA and protein expression outlier\",\n",
            "          \"passenger events (Methods). mRNA and protein expression outlier scores showed a moderate positive correlation (Fig. 5a, R=0.516), but mRNA outlier expression did not guarantee high protein expression (for example, AKT2 and FGFR2, Fig. 5a). Similarly, we\",\n",
            "          \"(for example, AKT2 and FGFR2, Fig. 5a). Similarly, we observed a fraction of phosphosite outliers not detected at the protein level (Fig. 5b, R=0.548). Consequently, identifying post-transcriptional and post-translational events to capture potential\",\n",
            "          \"and post-translational events to capture potential druggable treatment opportunities requires consideration of protein expression as well as gene expression. Applying this druggable outlier detection strategy across CNV, mRNA, protein and protein\",\n",
            "          \"detection strategy across CNV, mRNA, protein and protein phosphorylation levels, we identified overexpressed druggable genes in 26.1% and 47.8% of PDXs at the CNV and mRNA levels, respectively (Fig. 6; Supplementary Data 16). These events recapitulated\",\n",
            "          \"(Fig. 6; Supplementary Data 16). These events recapitulated known druggable opportunities, such as the PIK3CA copy-number amplification in WHIM4 and ERBB2 (HER2) copy-number amplification in WHIM35. Expanding to iTRAQ protein expression outliers allowed\",\n",
            "          \"WHIM35. Expanding to iTRAQ protein expression outliers allowed us to uncover druggable targets in 19 out of the 24 PDXs (79.2%), while considering phosphosite outliers covered 22 of the 24 PDXs (91.7%). A significantly high fraction of protein outliers\",\n",
            "          \"PDXs (91.7%). A significantly high fraction of protein outliers overlapped between the LFQ and iTRAQ datasets (Fisher's Exact Test, P=2.013e-05, Supplementary Fig. 11), providing validation of our findings. The identified proteomic outliers included\",\n",
            "          \"of our findings. The identified proteomic outliers included multiple known druggable targets involved in the PI3K, RTK, MAPK signalling pathways and other oncogenic processes (Fig. 6; Supplementary Data 16). For instance, HER2 was found to be the top\",\n",
            "          \"Data 16). For instance, HER2 was found to be the top outlier at the protein expression level in 2 HER2-E xenografts, WHIM8 and WHIM35; the phosphosites of HER2 were also identified as outliers. Immunohistochemistry experiments validated the high HER2\",\n",
            "          \"Immunohistochemistry experiments validated the high HER2 protein expression on the cell membranes of WHIM8 and WHIM35 (Fig. 5c; Supplementary Fig. 12). Further, AKT1, AKT2 and AKT3 proteins were outlier expression candidates in WHIM35, WHIM20 and WHIM43\",\n",
            "          \"were outlier expression candidates in WHIM35, WHIM20 and WHIM43 and WHIM4, respectively, providing proteomic validation of previously observed AKT up-regulation at mRNA expression level in breast cancer. Other outlier proteins included IDH1 in WHIM9 and\",\n",
            "          \"cancer. Other outlier proteins included IDH1 in WHIM9 and WHIM30, FGFR4 in WHIM11 and WHIM26, and both BTK and JAK3 in the EBV-positive WHIM17 and WHIM46. Many of these protein outliers aligned with up-regulation in their corresponding phosphosites,\",\n",
            "          \"aligned with up-regulation in their corresponding phosphosites, including phosphosites on AKT1, AKT3, BRAF, FGFR4 and HSP90AB1 (Fig. 5c, selected phosphosite spectra in Supplementary Fig. 13). We also discovered additional outlier phosphosites in other\",\n",
            "          \"We also discovered additional outlier phosphosites in other proteins including CTNNB1, ARAF, and HSP90B1 (Supplementary Data 16). Finally, we identified outlier FGFR2 in WHIM16 and high RAF1 in WHIM9 and further validated their high protein expression\",\n",
            "          \"in WHIM9 and further validated their high protein expression status of by immunohistochemistry analyses (Fig. 5c; Supplementary Fig. 12). As diverse sets of proteomic outliers are identified across PDXs, effectively inhibiting such activation events will\",\n",
            "          \"across PDXs, effectively inhibiting such activation events will be required when designing targeted treatment strategies to each individual breast tumour. Notably, for 80.6% of proteomic outlier events, we were able to identify human tumours from the 77\",\n",
            "          \"events, we were able to identify human tumours from the 77 TCGA samples showing the same outlier protein expression through outlier score or ranking (Methods, Fig. 6b, Supplementary Data 16). For example, we observed outlier HER2 expression in five HER2-E\",\n",
            "          \"For example, we observed outlier HER2 expression in five HER2-E and four luminal B human breast tumours. Further, a basal human sample carried both the outlier FGFR2 and FGFR4 expression, validating the findings in WHIM16, WHIM26 and WHIM11. Basal human\",\n",
            "          \"the findings in WHIM16, WHIM26 and WHIM11. Basal human breast tumours also carried protein expression outliers in IDH1, EGFR and MAP2K1 (Fig. 6b). Phosphosite outlier expression events showed a moderate rate of validation in the same human cohort (48.8%,\",\n",
            "          \"a moderate rate of validation in the same human cohort (48.8%, Fig. 6c), suggesting its transient nature and potential micro-environmental effects on protein phosphorylation. As expected, HER2-E and a few HER2-positive luminal B tumours carried outliers\",\n",
            "          \"and a few HER2-positive luminal B tumours carried outliers in HER2 phosphosites including p.T701, p.T1240 and p.Y1248. ARAF phosphosite outliers, such as p.S299, were found in both human and PDX samples, validating our previous finding in the human\",\n",
            "          \"and PDX samples, validating our previous finding in the human cohort. Interestingly, we identified outlier phosphosites in genes not previously implicated in breast cancer through genomic profiles, such as BRAF p.S447, p.S750 and HSP90AB1 p.Y56 and p.S169\",\n",
            "          \"such as BRAF p.S447, p.S750 and HSP90AB1 p.Y56 and p.S169 (Fig. 6c). Our results demonstrated that proteomic outlier events, like genomic driver mutations, are consistently observed in PDXs and human tumours. Some protein outlier events might represent\",\n",
            "          \"and human tumours. Some protein outlier events might represent \\u2018proteomic drivers' of tumorigenesis and therefore potential drug targets in breast tumours. Targeted treatments using breast cancer xenograft models To validate the identified druggable\",\n",
            "          \"cancer xenograft models To validate the identified druggable events, we conducted treatment experiments targeting HER2 and PI3K pathways on selected PDX models. Four PDX models were chosen to address HER2 targeting using lapatinib, an oral HER2 kinase\",\n",
            "          \"to address HER2 targeting using lapatinib, an oral HER2 kinase inhibitor. These include 2 HER2-E PDX models WHIM8 and WHIM35, both with high HER2 protein and phosphoprotein expression, and 2 low HER2 expressing, basal-like PDX models WHIM6 and WHIM14.\",\n",
            "          \"2 low HER2 expressing, basal-like PDX models WHIM6 and WHIM14. Western blotting suggested high HER2 expression and phosphorylation of HER2 p.Y1248 in both HER2 positive tumours, WHIM8 and WHIM35 (Fig. 7a; Supplementary Fig. 14). Unexpectedly, high levels\",\n",
            "          \"(Fig. 7a; Supplementary Fig. 14). Unexpectedly, high levels of HER2 p.Y1248 phosphorylation was also detected in WHIM14 by western blotting. In contrast MS-based phosphoproteomics also detected high levels of HER2p.Y1248 in WHIM8 and WHIM35, but not in\",\n",
            "          \"high levels of HER2p.Y1248 in WHIM8 and WHIM35, but not in WHIM14, which was more consistent with the known biology of the models. Antibody-based diagnosis of HER2 activation in WHIM14 was possibly due to cross-reaction of the Y1248 antibody with the\",\n",
            "          \"possibly due to cross-reaction of the Y1248 antibody with the pY1172 site of EGFR that bears high sequence similarity around the pY residue (..GTPTAENPEy1248LGLDVPV-CO2H vs...GSTAENAEy1172LRVAPQ.., Supplementary Fig. 6). As expected, WHIM8 and WHIM35 were\",\n",
            "          \"Supplementary Fig. 6). As expected, WHIM8 and WHIM35 were growth inhibited (Wilcoxon Rank Sum Test, P=2.7e-5, 4.8e-7) by lapatinib, whereas WHIM6 was not (P=0.65) (Fig. 7b). Interestingly, WHIM14 also showed significant reduction in tumour growth by\",\n",
            "          \"WHIM14 also showed significant reduction in tumour growth by lapatinib (P=4.3e-3). Upon further exploration with a lower, clinically achievable dose of lapatinib for chronic treatment (30\\u2009mg\\u2009kg\\u22121 compared with 220\\u2009mg\\u2009kg\\u22121 in the previous experiment) over\",\n",
            "          \"compared with 220\\u2009mg\\u2009kg\\u22121 in the previous experiment) over 48 days, significant but weak tumour growth inhibition was again achieved (P=0.0311, Supplementary Fig. 15). However, WHIM14 did not respond to HER2/HER3 antibodies trastuzumab or pertuzumab\",\n",
            "          \"not respond to HER2/HER3 antibodies trastuzumab or pertuzumab (P=0.250 and 0.181, respectively; Supplementary Fig. 15). Thus, the response of WHIM14 to lapatinib was likely due to inhibition of EGFR not HER2 (refs). While WHIM6 also showed elevated EGFR\",\n",
            "          \"of EGFR not HER2 (refs). While WHIM6 also showed elevated EGFR protein level, WHIM14 showed notably higher EGFR phosphorylation based on both mass spectrometry (Fig. 2b) and western blotting (Fig. 7a), which could account for their different response to\",\n",
            "          \"(Fig. 7a), which could account for their different response to lapatinib. Two basal and one HER2-E human breast cancers harboured outlier EGFR expression (Fig. 6b), suggesting EGFR remains a potential therapeutic target in a subset of breast cancers that\",\n",
            "          \"potential therapeutic target in a subset of breast cancers that has yet to be fully realized clinically. The PI3K/AKT/mTOR signalling pathway is altered in approximately 77% of breast tumours and multiple drugs targeting its components, including Class I\",\n",
            "          \"and multiple drugs targeting its components, including Class I PI3Ks, AKTs, and mTORs, are already in clinical trials. Among these, a combination of everolimus and exemestane has been approved for treating advanced ER-positive breast cancers resistant to\",\n",
            "          \"for treating advanced ER-positive breast cancers resistant to non-steroidal aromatase inhibitors. Promising activity has also been reported for direct inhibitors of PI3K. However, mutations in PIK3CA and other genetic alterations at the genomic level have\",\n",
            "          \"PIK3CA and other genetic alterations at the genomic level have so far failed to closely predict therapeutic responsiveness to PI3K pathway inhibitors. We therefore hypothesized that combined genomic and proteomic indication of PI3K signalling activation\",\n",
            "          \"genomic and proteomic indication of PI3K signalling activation is necessary for the prediction of treatment response. Our integrated approach identified 6 xenografts that harboured complementary genomic and proteomic druggable events in the PI3K-AKT\",\n",
            "          \"genomic and proteomic druggable events in the PI3K-AKT pathway. In particular, WHIM16 harboured a hotspot PIK3CA p.H1047R mutation, whereas WHIM18 and WHIM20 each carried a hotspot PIK3CA p.E545K mutation. WHIM20 also showed an additional outlier protein\",\n",
            "          \"mutation. WHIM20 also showed an additional outlier protein expression of AKT2 and WHIM18 also expressed AKT2 at a near outlier level that may combine to activate PI3K pathway signalling (Fig. 6). Since the treatments were applied to later-passage PDX\",\n",
            "          \"6). Since the treatments were applied to later-passage PDX models relative to the ones we conducted proteogenomic analysis on, we performed immunohistochemistry to validate the phosphorylation of AKT p.S473. WHIM18 and WHIM20 showed detectable AKT p.S473,\",\n",
            "          \"of AKT p.S473. WHIM18 and WHIM20 showed detectable AKT p.S473, which was not observed in WHIM16 (Fig. 7c). We conducted combinatorial treatment experiments by applying an alpha specific PI3K inhibitor (TAK-117) and/or an mTORC1/2 inhibitor (TAK-228) to\",\n",
            "          \"inhibitor (TAK-117) and/or an mTORC1/2 inhibitor (TAK-228) to three PDX models of luminal B breast cancer. Consistent with previous reports, PIK3CA mutation status alone did not accurately predict outcome; WHIM18 and WHIM20 showed reduced tumour growth\",\n",
            "          \"predict outcome; WHIM18 and WHIM20 showed reduced tumour growth upon application of the inhibitors, whereas WHIM16 did not (Fig. 7d). mTOR inhibition repressed tumour growth in WHIM18 and WHIM20 (ANOVA followed by Tukey's post hoc test, P=9.5e-10, 1.6e-03\",\n",
            "          \"(ANOVA followed by Tukey's post hoc test, P=9.5e-10, 1.6e-03 respectively), but not in WHIM16 (P=0.97), showing that inhibition of mTORC1/2 may effectively suppress breast tumours with activated AKTs and validating a previous study showing the efficacy of\",\n",
            "          \"AKTs and validating a previous study showing the efficacy of mTOR inhibition in PDX models of triple-negative breast cancer. Importantly, the combinatorial treatment achieved the greatest effect in WHIM18 and WHIM20 (P<2.2e-16 for both comparisons). While\",\n",
            "          \"in WHIM18 and WHIM20 (P<2.2e-16 for both comparisons). While neither PI3K nor mTOR inhibitor drug treatment alone suppressed tumour growth in WHIM20 completely, the combination of both PI3K and mTOR inhibitors significantly reduced tumour growth to a\",\n",
            "          \"and mTOR inhibitors significantly reduced tumour growth to a nearly static state (Fig. 7d). Based on our proteomic characterization, WHIM20 exhibited the strongest AKT1 and AKT2 protein expression, as well as AKT1 p.S122 and p.S475 phosphorylation\",\n",
            "          \"expression, as well as AKT1 p.S122 and p.S475 phosphorylation signatures followed by WHIM18 and then WHIM16 (Fig. 6b,c). Further, both PI3K and mTOR inhibitors significantly reduced AKT p.S473 (Fig. 7c). Our treatment results showed that the magnitude of\",\n",
            "          \"(Fig. 7c). Our treatment results showed that the magnitude of drug response may be associated with overexpression and phosphorylation of the downstream signalling targets such as AKT proteins. In addition to this validation of druggable hypotheses in\",\n",
            "          \"In addition to this validation of druggable hypotheses in luminal tumours, our previous report also demonstrated the effectiveness of combinatorial therapy of AKT and mTOR inhibitors in two other basal breast cancer xenografts. One of the treated\",\n",
            "          \"in two other basal breast cancer xenografts. One of the treated xenografts, WHIM4, was also characterized in this study, and showed copy-number amplification of PIK3CA and AKT3 protein outlier expression (Fig. 6). Overall, proteogenomic analysis revealed\",\n",
            "          \"expression (Fig. 6). Overall, proteogenomic analysis revealed that the dual activation of PIK3CA at the genomic level and AKTs at the protein level may be a common signature of breast tumours, affecting more than 20% of PDXs in this cohort. Importantly,\",\n",
            "          \"affecting more than 20% of PDXs in this cohort. Importantly, our results demonstrate the potential utility of combinatorial inhibitor treatments to treat breast tumours showing these proteogenomic signatures. Discussion Breast cancer has been\",\n",
            "          \"proteogenomic signatures. Discussion Breast cancer has been traditionally characterized in the clinic through hormone receptor status and selected genes' expressions, and more recently by genomic sequencing. However, druggable genomic driver events are\",\n",
            "          \"sequencing. However, druggable genomic driver events are detectable in only a limited percentage of patients. As the majority of drugs target proteins, a systematic evaluation of breast cancer proteomes would seem ultimately to be necessary for selecting\",\n",
            "          \"proteomes would seem ultimately to be necessary for selecting targeted treatment and predicting drug response. Recent advances in MS-based proteomics allow extensive and quantitative surveys of the global proteome. Here, we have systematically analyzed\",\n",
            "          \"of the global proteome. Here, we have systematically analyzed proteogenomic profiles of 22 patient-derived breast cancer xenografts and 2 EBV-positive lymphoproliferations that are likely artifacts of engraftment of human lymphocytes into NSG mice. This\",\n",
            "          \"of engraftment of human lymphocytes into NSG mice. This study shows that proteogenomic signatures of PDXs resemble most findings from breast cancer patients. While some discrepancies exist, we established a normalization strategy at both the genomic and\",\n",
            "          \"we established a normalization strategy at both the genomic and proteomic levels that enabled direct comparison. PDX tumours recapitulated the proteomic diversity of human breast cancers (Fig. 3). We also identified multiple druggable targets for each\",\n",
            "          \"3). We also identified multiple druggable targets for each tumour model (Fig. 6; Supplementary Data 17). Proteomic events validated a significant number of CNV or mRNA up-regulations. For example, HER2 protein and phosphosite outlier expression was\",\n",
            "          \"example, HER2 protein and phosphosite outlier expression was observed in HER2-E WHIM8 and WHIM35 (Fig. 5), which were effectively treated using lapatinib (Fig. 7b). Interestingly, we also identified overexpressed proteomic events not evident in the\",\n",
            "          \"identified overexpressed proteomic events not evident in the genomic level in both PDX and human samples, including outlier protein expression of EGFR, and outlier phosphosite expressions of ARAF, BRAF, HSP90AB1, PTPN11 and TOP2A (Fig. 6), highlighting\",\n",
            "          \"ARAF, BRAF, HSP90AB1, PTPN11 and TOP2A (Fig. 6), highlighting potential new treatment opportunities in breast cancer. In the two PDX models subsequently diagnosed as EBV-positive lymphoproliferations, we observed outlier BTK expression (Fig. 6) and\",\n",
            "          \"we observed outlier BTK expression (Fig. 6) and activation of the NF\\u03baB pathway (Fig. 4), validating BTK as a druggable target in EBV+ lymphomas. While more than 80% of the proteomic outlier events in PDX were also found in human tumours, a lessor 48.8% of\",\n",
            "          \"in PDX were also found in human tumours, a lessor 48.8% of phosphosite outlier events were validated, potentially due to different tumour micro-environments. Thus, transient phosphoproteomic events identified in PDX tumours would likely require further\",\n",
            "          \"events identified in PDX tumours would likely require further verification in their corresponding primary tumours. Outlier protein expression events can likely lead to downstream pathway activation, such as MET outlier protein expression (Fig. 6) and\",\n",
            "          \"activation, such as MET outlier protein expression (Fig. 6) and activated Ras pathway observed in WHIM12 (Fig. 4). While genomic analysis has utilized mutual occurrence or exclusivity in patient cohorts to deduce pathway relationships, phosphorylation\",\n",
            "          \"cohorts to deduce pathway relationships, phosphorylation profile analyses allowed us to directly interrogate signalling in these established pathways in a single sample. Our pathway activation results suggest these events may be crucial to tumorigenesis\",\n",
            "          \"results suggest these events may be crucial to tumorigenesis and that some are likely the proteomic \\u2018smoking guns' that originally triggered the oncogeneic cascade. Roughly 77% of breast tumours showed alterations in the PI3K-AKT signalling pathway,\",\n",
            "          \"tumours showed alterations in the PI3K-AKT signalling pathway, representing a potentially important path forward for drug-based treatment. Yet, genomic alterations of PI3K pathway components have not shown themselves to predict treatment responsiveness to\",\n",
            "          \"not shown themselves to predict treatment responsiveness to PI3K pathway inhibitors. In this study, we defined complementary druggable targets of the pathway using proteogenomic analysis, including several events of a co-occurring PIK3CA mutation or\",\n",
            "          \"including several events of a co-occurring PIK3CA mutation or copy-number amplification, and AKT protein outlier expression coupled by elevated AKT phosphorylation. In two such breast tumours, we successfully inhibited tumour growth using a combination of\",\n",
            "          \"we successfully inhibited tumour growth using a combination of PI3K and MTOR inhibitors (Fig. 7c). While these results show potential functional implications, additional, systematic treatment experiments are required to validate the identified proteomic\",\n",
            "          \"experiments are required to validate the identified proteomic druggable targets. In conclusion, this initial work using proteogenomic integration coupled with patient-derived xenograft validation, has demonstrated a strategy that, in principle, may enable\",\n",
            "          \"has demonstrated a strategy that, in principle, may enable more accurate prediction of the efficacy of mechanism-based cancer therapeutics. Methods Xenograft model generation Patient-derived xenografts were generated from primary or metastatic breast\",\n",
            "          \"xenografts were generated from primary or metastatic breast tumours using previously described procedures. All human tissues for these experiments were processed in compliance with NIH regulations and institutional guidelines, and approved by the\",\n",
            "          \"regulations and institutional guidelines, and approved by the institutional review board at Washington University. All animal procedures were reviewed and approved by the institutional animal care and use committee at Washington University in St. Louis.\",\n",
            "          \"care and use committee at Washington University in St. Louis. PDX models are available through the application to the Human and Mouse-Linked Evaluation of Tumors core at http://digitalcommons.wustl.edu/hamlet/. We selected 24 of the established breast\",\n",
            "          \"We selected 24 of the established breast tumour samples, including 9 luminal B, 10 basal, 4 HER2-E, and 1 CLDN-low breast tumours for further proteogenomic characterization. Receptor statuses of xenograft tumours were validated using IHC after\",\n",
            "          \"statuses of xenograft tumours were validated using IHC after engraftment. Immunohistochemistry Xenografts were formalin-fixed at least for 24\\u2009h and paraffin-embedded. Sections were evaluated by hematoxylin & eosin (H&E) staining. Immunohistochemistry\",\n",
            "          \"by hematoxylin & eosin (H&E) staining. Immunohistochemistry (IHC) was performed on additional sections for HER2 (Dako), Phospho-Akt (Ser473) (Cell signalling), FGFR2 (Abcam), and Raf-1 (Santa Cruz) following the manufacturer's instructions. Western\",\n",
            "          \"(Santa Cruz) following the manufacturer's instructions. Western blotting of phosphorylated HER2 (p.Y1248) was performed using antibody cat Nr-06-229 (Millipore). In vivo drug treatment experiments For the targeted treatment of the PI3K pathway, PI3K alpha\",\n",
            "          \"For the targeted treatment of the PI3K pathway, PI3K alpha inhibitor TAK-117 and TORC1/2 inhibitor TAK-228 were provided by Millennium Pharmaceuticals, Inc. The compounds were dissolved in Peg400. Tumours were engrafted in NSG (NOD.Cg-Prkdcscid\",\n",
            "          \"in Peg400. Tumours were engrafted in NSG (NOD.Cg-Prkdcscid Il2rgtm1Wjl/SzJ) mice (The Jackson Laboratory) by subcutaneous injection of 2\\u20135 \\u00d7 106 PDX cells in PBS supplemented with 30% Matrigel (BD Biosciences, cat. no. 354234). When tumours reached an\",\n",
            "          \"(BD Biosciences, cat. no. 354234). When tumours reached an average size of 250\\u2013300\\u2009mm3, animals were assigned randomly to control and various treatment groups (n=8\\u20139 each group). For treatment of WHIM16 (passage 7), 18 (passage 8), and 20 (passage 5) were\",\n",
            "          \"of WHIM16 (passage 7), 18 (passage 8), and 20 (passage 5) were used. Tumour bearing mice were gavaged with: (1) Peg400; (2) TAK-117, 140\\u2009mg\\u2009kg\\u22121\\u2009day\\u22121; (3) TAK-228, 1\\u2009mg\\u2009kg\\u22121\\u2009day\\u22121; (4) TAK-117, 140\\u2009mg\\u2009kg\\u22121\\u2009day\\u22121, TAK-228, 1\\u2009mg\\u2009kg\\u22121\\u2009day\\u22121. The mice were\",\n",
            "          \"140\\u2009mg\\u2009kg\\u22121\\u2009day\\u22121, TAK-228, 1\\u2009mg\\u2009kg\\u22121\\u2009day\\u22121. The mice were treated on three consecutive days once daily and then had a 4-day interval. Tumours were measured with external caliper, and volume was calculated as (4\\u03c0/3) \\u00d7 (width/2)2 \\u00d7 (length/2). For the\",\n",
            "          \"was calculated as (4\\u03c0/3) \\u00d7 (width/2)2 \\u00d7 (length/2). For the lapatinib therapeutic experiments of WHIM6 (passage 7), WHIM 14 (passage 11), WHIM 8 (passage 6) and WHIM 35 (passage 6), 1 \\u00d7 106 tumour cells were added to equal volume of 1:1 mixture of\",\n",
            "          \"\\u00d7 106 tumour cells were added to equal volume of 1:1 mixture of Matrigel and 10% RPMI plus 10% FBS to 4th mammary fat pads in female SCID/bg mice (ENVIGO). We then established tumours to an average volume 250\\u2013300\\u2009mm3, and randomized the mice into control\",\n",
            "          \"volume 250\\u2013300\\u2009mm3, and randomized the mice into control and lapatinib groups. We treated the treated group mice with lapatinib chow diet by formulating lapatinib (220\\u2009mg\\u2009kg\\u22121) and processing them into food pellets (by Research Diets Inc), which is\",\n",
            "          \"them into food pellets (by Research Diets Inc), which is supplied for 2 weeks. The numbers of replicates for each group are as followed: WHIM 14 control (n=6) and lapatinib treated (n=5); WHIM 6 control (n=11) and lapatinib treated (n=12); WHIM 8 control\",\n",
            "          \"6 control (n=11) and lapatinib treated (n=12); WHIM 8 control (n=16) and lapatinib treated (n=6); WHIM 35 control (n=17) and lapatinib treated (n=18). We also treated WHIM14 with 100\\u2009mg\\u2009kg\\u22121 lapatinib treatment for 48 days, including both control (n=8)\",\n",
            "          \"lapatinib treatment for 48 days, including both control (n=8) and lapatinib treated (n=9) groups. Further, we tested the effect of trastuzumab (30\\u2009mg\\u2009kg\\u22121 weekly with IP injections) and pertuzumab (30\\u2009mg\\u2009kg\\u22121 weekly with IP injections) on PDX tumour\",\n",
            "          \"pertuzumab (30\\u2009mg\\u2009kg\\u22121 weekly with IP injections) on PDX tumour growth for 48 days. The experimental groups are as following: control group (n=6) treated with physiological saline (vehicle); trastuzumab treated group (n=12); pertuzumab treated group\",\n",
            "          \"trastuzumab treated group (n=12); pertuzumab treated group (n=11). Statistical testing of the resulting data was conducted using the R programming language. Wilcoxon rank sum test was applied to compare the fold change in tumour volumes after two weeks in\",\n",
            "          \"to compare the fold change in tumour volumes after two weeks in the lapatinib treatment experiment, and after 44 days or 48 days for the additional WHIM14 experiment using low-dose lapatinib and trastuzumab/pertuzumab. One-way ANOVA was applied to compare\",\n",
            "          \"trastuzumab/pertuzumab. One-way ANOVA was applied to compare the treated versus control groups in the PI3K targeted-therapy experiments, and a follow-up Tukey's post hoc test was used for the various comparisons in PI3K inhibition experiments. All human\",\n",
            "          \"various comparisons in PI3K inhibition experiments. All human tissues for these experiments were processed in compliance with NIH regulations and institutional guidelines, and approved by the Washington University, University of North Carolina, or Baylor\",\n",
            "          \"Washington University, University of North Carolina, or Baylor College of Medicine Institutional Review Board (IRB). All animal procedures were reviewed and approved by the institutional animal care and use committee at Washington University in St. Louis,\",\n",
            "          \"care and use committee at Washington University in St. Louis, University of North Carolina, or Baylor College of Medicine. Genomic and proteomic data generation. Somatic mutation Sequencing reads were aligned using BWA. Somatic variants were identified\",\n",
            "          \"reads were aligned using BWA. Somatic variants were identified using VarScan2 (refs), GATK and Pindel, and annotated based on Ensembl release 70_37. We then filtered out common variants by using variants from the 1000 Genomes and NHLBI projects. We\",\n",
            "          \"by using variants from the 1000 Genomes and NHLBI projects. We further eliminated mouse contamination by filtering somatic variants that were mapped to mouse reference genome. Somatic mutation calls were validated either using a custom array or manually\",\n",
            "          \"calls were validated either using a custom array or manually reviewed in IGV. The genomic data of 17 out of the 24 PDXs have been presented before in our previous studies (Supplementary Data 2). Copy-number variation The segment-based copy-number data\",\n",
            "          \"2). Copy-number variation The segment-based copy-number data were generated using the whole-genome sequencing and exome sequencing data. We then converted the segment-based copy-number data to the gene-based copy-number data by using the RefSeq database\",\n",
            "          \"to the gene-based copy-number data by using the RefSeq database (version 20130727). The copy-number values were further transformed to the log-R ratio, using the cohort mean for the gene as the reference. mRNA expression and virus detection through\",\n",
            "          \"as the reference. mRNA expression and virus detection through RNA-seq mRNA expression values were calculated from mRNA sequencing data using MapSplice. The resulting RSEM values were normalized within samples to a fixed upper quartile. Upper quartile\",\n",
            "          \"within samples to a fixed upper quartile. Upper quartile normalized RSEM data were log2 transformed and the data were median centred by gene. To quantify virus abundance, we used the VirusScan pipeline (https://github.com/ding-lab/VirusScan) to detect\",\n",
            "          \"pipeline (https://github.com/ding-lab/VirusScan) to detect viruses by numbers of virus-supporting reads from RNA-seq data. LFQ proteome Tumour Sample Generation and Protein Extraction. Patient-derived xenograft breast tumours were processed to\",\n",
            "          \"Patient-derived xenograft breast tumours were processed to cryopulverized powders as described previously. The powders (100\\u2009mg wet weight) were subjected to lysis and protein extraction using a buffer composed of 8 M urea, 50\\u2009mM Tris pH 8.0, 75\\u2009mM NaCl,\",\n",
            "          \"a buffer composed of 8 M urea, 50\\u2009mM Tris pH 8.0, 75\\u2009mM NaCl, 1\\u2009mM MgCl2, and 500 units Benzonase. Approximately 1\\u2009mg of total protein extracted was reduced with DTT and subsequently alkylated with iodoacetamide. The proteins were then subjected to\",\n",
            "          \"with iodoacetamide. The proteins were then subjected to proteolysis with endoproteinase Lys-C (Wako Chemicals, USA) for \\u223c4\\u2009h at 37\\u2009\\u00b0C. The solution was diluted 4-fold with 25\\u2009mM Tris pH 8.0, 1\\u2009mM CaCl2 and further digested with trypsin (Promega) for \\u223c12\\u2009h\",\n",
            "          \"CaCl2 and further digested with trypsin (Promega) for \\u223c12\\u2009h at 37\\u2009\\u00b0C. Digestion was stopped by the addition of TFA to 0.4%, and the precipitate was removed by centrifugation. The peptide solutions were desalted on Sep-Pak Light C18 cartridges (Waters) and\",\n",
            "          \"were desalted on Sep-Pak Light C18 cartridges (Waters) and dissolved in 30% ACN, 0.1% TFA before loading on a 300\\u2009\\u03bcm Source 15\\u2009S (GE Healthcare) column for Basic Reversed Phase Chromatography (bRPLC). A linear LC gradient was performed by increasing\",\n",
            "          \"(bRPLC). A linear LC gradient was performed by increasing buffer B from 0 to 70% within 60\\u2009min, where buffer A was aqueous 10\\u2009mM ammonium formate, and buffer B was 90% ACN in 10\\u2009mM ammonium formate. A total of 30 fractions were collected for each WHIM\",\n",
            "          \"formate. A total of 30 fractions were collected for each WHIM sample (18 WHIMs). Five fractions were then prepared by combining non-contiguously fractions. We analyzed an additional technical replicate for WHIM2 and WHIM16. The fractions were dried and\",\n",
            "          \"replicate for WHIM2 and WHIM16. The fractions were dried and desalted using a stop-and-go-extraction tip (StageTip) protocol containing 4 \\u00d7 1\\u2009mm C18 extraction disk (3M). Liquid Chromatography-Tandem Mass Spectrometry and Protein Identification. Sample\",\n",
            "          \"Mass Spectrometry and Protein Identification. Sample analysis was performed via reversed phase LC-MS/MS using a Proxeon 1000 nano LC system coupled to a Q Exactive mass spectrometer (Thermo Scientific, San Jose, CA). The Proxeon system was configured to\",\n",
            "          \"Scientific, San Jose, CA). The Proxeon system was configured to trap peptides using a C18 column (3\\u2009cm \\u00d7 100\\u2009\\u03bcm i.d.) with a diverted flow rate (5\\u2009\\u03bcl\\u2009min\\u22121) The trap column was placed in line with the analytical column (15\\u2009cm \\u00d7 75\\u2009\\u03bcm i.d., 3.5\\u2009\\u03bcm, 300\\u2009\\u00c5\",\n",
            "          \"with the analytical column (15\\u2009cm \\u00d7 75\\u2009\\u03bcm i.d., 3.5\\u2009\\u03bcm, 300\\u2009\\u00c5 particle C18, Thermo Scientific, San Jose, CA, USA) before gradient elution of peptides. Analytical separation of all the tryptic peptides was achieved with a linear gradient of 2\\u201330% buffer B\",\n",
            "          \"peptides was achieved with a linear gradient of 2\\u201330% buffer B over 240\\u2009min (250\\u2009nl\\u2009min\\u22121), where buffer A was aqueous 0.1% formic acid, and buffer B was acetonitrile in 0.1% formic acid. LC-MS experiments were performed in a data-dependent mode with\",\n",
            "          \"LC-MS experiments were performed in a data-dependent mode with Full-MS (externally calibrated to a mass accuracy of <5 p.p.m., and a resolution of 70,000 at m/z=200) followed by HCD-MS/MS of the top 20 most intense ions. High-energy collision activated\",\n",
            "          \"the top 20 most intense ions. High-energy collision activated dissociation (HCD)-MS/MS was used to fragment peptides at a normalized collision energy of 27\\u2009eV in the presence of nitrogen. One LC-MS run was performed for each fraction (from 1 process\",\n",
            "          \"One LC-MS run was performed for each fraction (from 1 process technical replicate), except for WHIM2 and WHIM16 where 2 LC\\u2013MS runs were conducted (from 3 process technical replicates), resulting in the production of 100 LC\\u2013MS runs for global peptide\",\n",
            "          \"in the production of 100 LC\\u2013MS runs for global peptide analysis. Mass spectra were processed, and peptide identification was performed using the Andromeda search engine found in MaxQuant software ver. 1.5.0.25. (Max Planck Institute, Germany). All protein\",\n",
            "          \"ver. 1.5.0.25. (Max Planck Institute, Germany). All protein database searches were performed against the RefSeq database (version 20140707). Peptides were identified with a target-decoy approach using a combined database consisting of reverse protein\",\n",
            "          \"using a combined database consisting of reverse protein sequences of the RefSeq human, mouse and common repository of adventitious proteins (cRAP). The cRAP database was obtained from the Global Proteome Machine (ftp://ftp.thegpm.org/fasta/cRAP). Peptide\",\n",
            "          \"Proteome Machine (ftp://ftp.thegpm.org/fasta/cRAP). Peptide inference was made with a FDR of 1% while peptides were assigned to proteins with a protein FDR of 5%. A precursor ion mass tolerance of 20 ppm was used for the first search that allowed for m/z\",\n",
            "          \"of 20 ppm was used for the first search that allowed for m/z retention time recalibration of precursor ions that were then subjected to a main search using a precursor ion mass tolerance of 6 p.p.m. and a product ion mass tolerance 0.5\\u2009Da. Search\",\n",
            "          \"of 6 p.p.m. and a product ion mass tolerance 0.5\\u2009Da. Search parameters included up to two missed cleavages at KR on the sequence, and oxidation of methionine, and protein N terminus acetylation as a dynamic modification. Carbamidomethylation of cysteine\",\n",
            "          \"as a dynamic modification. Carbamidomethylation of cysteine residues was considered as a static modification. Peptide identifications are reported by filtering of reverse and contaminant entries and assigning to their leading razor protein according to\",\n",
            "          \"and assigning to their leading razor protein according to the Occams razor principal. The mass spectrometric data are deposited at the CPTAC Data Coordinating Center as raw and mzML files (https://cptac-data-portal.georgetown.edu). Peptide and Protein\",\n",
            "          \"(https://cptac-data-portal.georgetown.edu). Peptide and Protein Quantitation. LFQ was performed based on peak areas. The measured area under the curve of m/z and the retention time-aligned extracted ion chromatograms of peptides were performed via the\",\n",
            "          \"extracted ion chromatograms of peptides were performed via the label-free quantitation module in MaxQuant [ver. 1.5.0.25]. All replicates for each PDX were included in the LFQ experimental design with peptide-level quantitation performed using unique and\",\n",
            "          \"with peptide-level quantitation performed using unique and razor peptide features corresponding to identifications filtered with a posterior error probability (PEP) of 0.01, peptide FDR of 0.01 and protein FDR of 0.05. The expression values were median\",\n",
            "          \"0.01 and protein FDR of 0.05. The expression values were median centred in the Perseus software for further analysis [version 1.5.0.9]. iTRAQ proteome and phosphoproteome We included all 24 of the established breast tumour samples for proteomic\",\n",
            "          \"all 24 of the established breast tumour samples for proteomic characterization using iTRAQ. Tumour tissue samples were maintained in cryovials at \\u221280\\u2009\\u00b0C until cryopulverization using a CP02 Cryprep Pulverizer (Covaris, Woburn, MA). 90\\u2009mg aliquots of\",\n",
            "          \"Cryprep Pulverizer (Covaris, Woburn, MA). 90\\u2009mg aliquots of cryofractured material were prepared for proteomic processing in aluminum weighing dishes on dry ice using spatulas kept cold in liquid nitrogen, with remaining material reserved for other\",\n",
            "          \"in liquid nitrogen, with remaining material reserved for other applications. The 90\\u2009mg target was designed to include 40\\u2009mg for each of the collaborating research teams, with an anticipated yield for each team of 1.5\\u20132\\u2009mg protein based on 4\\u20135% recovery.\",\n",
            "          \"yield for each team of 1.5\\u20132\\u2009mg protein based on 4\\u20135% recovery. To avoid systematic bias, sample processing was block randomized, with each intrinsic subtype proportionally represented in each processing tranche. The reproducibility of the iTRAQ4-plex\",\n",
            "          \"each processing tranche. The reproducibility of the iTRAQ4-plex global proteome and phosphoproteome analysis workflow used in this study has been extensively tested for quantitative reproducibility both within and across laboratories in the CPTAC program.\",\n",
            "          \"both within and across laboratories in the CPTAC program. Over a period of several months 5 iTRAQ4-plex replicates were measured at each of the 3 CPTAC proteome analysis centres. Each of these iTRAQ4-plexes contained duplicate measurements for both a\",\n",
            "          \"these iTRAQ4-plexes contained duplicate measurements for both a basal WHIM2 and a luminal WHIM16 PDX samples that are also part of this study. A high degree of consistency in the number of proteins identified and correlation in the protein expression was\",\n",
            "          \"identified and correlation in the protein expression was obtained. Pearson correlations for replicate proteome and phosphoproteome measurements were very high with a R=0.9 in our previous study and very similar to the correlation observed here for the\",\n",
            "          \"study and very similar to the correlation observed here for the WHIM13 replicate measurement. These data show that our platform provides highly reproducible quantitative measurements for global proteomes and phosphoproteomes. Protein extraction, digestion\",\n",
            "          \"proteomes and phosphoproteomes. Protein extraction, digestion and iTRAQ labelling of peptides from breast cancer tumours. Cryopulverized breast cancer tumour samples tissues (\\u223c2 combined aliquots of 90\\u2009mg tissue weight each) were homogenized in 1,000\\u2009\\u03bcl\",\n",
            "          \"of 90\\u2009mg tissue weight each) were homogenized in 1,000\\u2009\\u03bcl lysis buffer containing 8M urea, 75\\u2009mM NaCl, 1\\u2009mM EDTA in 50\\u2009mM Tris HCl (pH 8), 10\\u2009mM NaF, phosphatase inhibitor cocktail 2 (1:100; Sigma, P5726) and cocktail 3 (1:100; Sigma, P0044), 2\\u2009\\u03bcg\\u2009ml\\u22121\",\n",
            "          \"Sigma, P5726) and cocktail 3 (1:100; Sigma, P0044), 2\\u2009\\u03bcg\\u2009ml\\u22121 aprotinin (Sigma, A6103), 10\\u2009\\u03bcg\\u2009ml\\u22121 Leupeptin (Roche, #11017101001), and 1\\u2009mM PMSF (Sigma, 78830). Lysates were centrifuged at 20,000 g for 10\\u2009min before measuring protein concentration of the\",\n",
            "          \"g for 10\\u2009min before measuring protein concentration of the clarified lysates by BCA assay (Pierce). Protein lysates were subsequently reduced with 5\\u2009mM dithiothreitol (Thermo Scientific, 20291) for 45\\u2009min at room temperature, and alkylated with 10\\u2009mM\",\n",
            "          \"20291) for 45\\u2009min at room temperature, and alkylated with 10\\u2009mM iodoacetamide (Sigma, A3221) for 45\\u2009min in the dark. Samples were diluted 4-fold with 50\\u2009mM Tris HCl (pH 8) prior to digesting them with LysC (Wako, 129-02541) for 4\\u2009h and trypsin (Promega,\",\n",
            "          \"them with LysC (Wako, 129-02541) for 4\\u2009h and trypsin (Promega, V511X) overnight at a 1:50 enzyme-to-protein ratio at room temperature overnight on a shaker. Digested samples were acidified with formic acid (FA; Fluka, 56302) to a final volumetric\",\n",
            "          \"with formic acid (FA; Fluka, 56302) to a final volumetric concentration of 1% or final pH of \\u223c3\\u20135, and centrifuged at 2,000\\u2009g for 5\\u2009min to clear precipitated urea from peptide lysates. Samples were desalted on C18 SepPak columns (Waters, 100\\u2009mg,\",\n",
            "          \"Samples were desalted on C18 SepPak columns (Waters, 100\\u2009mg, WAT036820) and 1\\u2009mg peptide aliquots were dried down using a SpeedVac apparatus. Construction of the Common internal Reference Pool. The proteomic and phosphoproteomic analyses of xenograft\",\n",
            "          \"Pool. The proteomic and phosphoproteomic analyses of xenograft samples were performed as iTRAQ 4-plex experiments. Quantitative comparison between all samples analyzed was facilitated by the use of iTRAQ reporter ion ratios between each individual sample\",\n",
            "          \"use of iTRAQ reporter ion ratios between each individual sample and a common internal reference sample present in each 4-plex. The reference sample was comprised of 16 of the 24 WHIM tumours analyzed in this study with equal contribution for each tumour\",\n",
            "          \"analyzed in this study with equal contribution for each tumour (WHIM numbers 2, 4, 6, 8, 11, 12, 13, 14, 16, 18, 20, 21, 24, 25, 30 and 46). The 24 tumour samples were analyzed in nine independent 4-plex experiments, with three individual samples\",\n",
            "          \"independent 4-plex experiments, with three individual samples occupying the first three channels of each experiment and the 4th channel being reserved for the reference sample. While eight iTRAQ 4-plex experiments were used to analyze the 24 individual\",\n",
            "          \"iTRAQ 4-plex experiments were used to analyze the 24 individual WHIM tumour samples, an additional 4-plex experiment was designed to include the WHIM13 sample for process replicate analysis and also internal reference samples from our human primary breast\",\n",
            "          \"also internal reference samples from our human primary breast cancer study and a taxol drug response study (unpublished) to allow cross-referencing of the different datasets. iTRAQ labelling, high pH reversed-phase separation and phosphopeptide enrichment\",\n",
            "          \"high pH reversed-phase separation and phosphopeptide enrichment of peptide samples. Desalted peptides were labelled with 4-plex iTRAQ reagents according to the manufacturer's instructions (AB Sciex, Foster City, CA). For each 1\\u2009mg peptide from each breast\",\n",
            "          \"Sciex, Foster City, CA). For each 1\\u2009mg peptide from each breast tumour sample, 10 units of labelling reagent were used. Peptides were dissolved in 300\\u2009\\u03bcl of 0.5 M triethylammonium bicarbonate (TEAB) (pH 8.5) solution and labelling reagent was added in\",\n",
            "          \"(TEAB) (pH 8.5) solution and labelling reagent was added in 700\\u2009\\u03bcl of ethanol. After 1\\u2009h incubation, 1.5\\u2009ml of 0.05% TFA was added to stop the reaction. Differentially labelled peptides were mixed and subsequently desalted on 500\\u2009mg tC18 SepPak columns.\",\n",
            "          \"mixed and subsequently desalted on 500\\u2009mg tC18 SepPak columns. The combined 4\\u2009mg iTRAQ samples per experiment were separated into 24 proteome fractions and 12 phosphoproteome fractions using a 4.6\\u2009mm \\u00d7 250\\u2009mm column RP Zorbax 300 A ExtendC18 column\",\n",
            "          \"using a 4.6\\u2009mm \\u00d7 250\\u2009mm column RP Zorbax 300 A ExtendC18 column (Agilent, 3.5\\u2009\\u03bcm bead size) on an Agilent 1100 Series HPLC instrument by basic reversed-phase chromatography as described previously. Peptides were separated according to their hydrophobicity\",\n",
            "          \"Peptides were separated according to their hydrophobicity using solvent A (2% acetonitrile, 5\\u2009mM ammonium formate, pH 10) and a nonlinear increasing concentration of solvent B (90% acetonitrile, 5\\u2009mM ammonium formate, pH 10). Phosphopeptides were enriched\",\n",
            "          \"5\\u2009mM ammonium formate, pH 10). Phosphopeptides were enriched using Ni-NTA superflow agarose beads (Qiagen, #1018611) that were stripped of nickel with 100\\u2009mM EDTA and incubated in an aqueous solution of 10\\u2009mM FeCl3 (Sigma, 451649) as described previously.\",\n",
            "          \"of 10\\u2009mM FeCl3 (Sigma, 451649) as described previously. For phosphopeptide enrichment a 80% acetonitrile/0.1% trifluoroacetic acid binding buffer and a 500\\u2009mM dibasic sodium phosphate, pH 7.0, (Sigma, S9763) elution buffer were used. Enriched samples were\",\n",
            "          \"(Sigma, S9763) elution buffer were used. Enriched samples were desalted on StageTips as described before analysis by LC\\u2013MS/MS. Analysis of tumour samples by high performance liquid chromatography tandem mass spectrometry (LC\\u2013MS/MS). All peptides were\",\n",
            "          \"tandem mass spectrometry (LC\\u2013MS/MS). All peptides were separated with an online nanoflow Proxeon EASY-nLC 1000 UHPLC system (Thermo Fisher Scientific) and analyzed on a benchtop Orbitrap Q Exactive mass spectrometer (Thermo Fisher Scientific) equipped\",\n",
            "          \"Exactive mass spectrometer (Thermo Fisher Scientific) equipped with a nanoflow ionization source (James A. Hill Instrument Services, Arlington, MA, USA). The LC system, column, and platinum wire to deliver electrospray source voltage were connected via a\",\n",
            "          \"to deliver electrospray source voltage were connected via a stainless-steel cross (360\\u2009\\u03bcm, IDEX Health & Science, UH-906x). The column was heated to 50\\u2009\\u00b0C using a column heater sleeve (Phoenix-ST) to prevent overpressurizing of columns during UHPLC\",\n",
            "          \"to prevent overpressurizing of columns during UHPLC separation. 10% of each global proteome sample in a 2\\u2009\\u03bcl injection volume, or 50% of each phosphoproteome sample in a 4 ul injection volume was injected onto an in-house packed 20\\u2009cm \\u00d7 75\\u2009um diameter C18\",\n",
            "          \"was injected onto an in-house packed 20\\u2009cm \\u00d7 75\\u2009um diameter C18 silica picofrit capillary column (1.9\\u2009\\u03bcm ReproSil-Pur C18-AQ beads, Dr Maisch GmbH, r119.aq; Picofrit 10\\u2009um tip opening, New Objective, PF360-75-10-N-5). Mobile phase flow rate was\",\n",
            "          \"New Objective, PF360-75-10-N-5). Mobile phase flow rate was 200\\u2009nl\\u2009min\\u22121, comprised of 3% acetonitrile/0.1% formic acid (Solvent A) and 90% acetonitrile /0.1% formic acid (Solvent B), and the 110-minute LC-MS/MS method consisted of a 10-min\",\n",
            "          \"B), and the 110-minute LC-MS/MS method consisted of a 10-min column-equilibration procedure, a 20-min sample-loading procedure, and the following gradient profile: (min:%B) 0:2; 1:6; 85:30; 94:60; 95;90; 100:90; 101:50; 110:50 (last two steps at\",\n",
            "          \"85:30; 94:60; 95;90; 100:90; 101:50; 110:50 (last two steps at 500\\u2009nl\\u2009min\\u22121 flowrate). Data-dependent acquisition was performed using Xcalibur QExactive v2.1 software in positive ion mode at a spray voltage of 2.00\\u2009kV. MS1 Spectra were measured with a\",\n",
            "          \"at a spray voltage of 2.00\\u2009kV. MS1 Spectra were measured with a resolution of 70,000, an AGC target of 3e6 and a mass range from 300 to 1,800\\u2009m/z. Up to 12 MS2 spectra per duty cycle were triggered at a resolution of 17,500, an AGC target of 5e4, an\",\n",
            "          \"triggered at a resolution of 17,500, an AGC target of 5e4, an isolation window of 2.5\\u2009m/z, a maximum ion time of 120\\u2009msec, and a normalized collision energy of 28. Peptides that triggered MS2 scans were dynamically excluded from further MS2 scans for\",\n",
            "          \"MS2 scans were dynamically excluded from further MS2 scans for 20\\u2009s. Charge state screening was enabled to reject precursor charge states that were unassigned, 1, or >6. Peptide match was enabled for monoisotopic precursor mass assignment. Protein-peptide\",\n",
            "          \"for monoisotopic precursor mass assignment. Protein-peptide identification, phosphosite localization, and quantitation. All MS data were interpreted using the Spectrum Mill software package v5.1 (for comparison with proteomes of human breast tumours from\",\n",
            "          \"(for comparison with proteomes of human breast tumours from our previous study) and v6.0 pre-release (Agilent Technologies, Santa Clara, CA, USA) co-developed by the authors. Similar MS/MS spectra acquired on the same precursor m/z within \\u00b1 45\\u2009s were\",\n",
            "          \"spectra acquired on the same precursor m/z within \\u00b1 45\\u2009s were merged. MS/MS spectra were excluded from searching if they failed the quality filter by not having a sequence tag length>0 (that is, minimum of two masses separated by the in-chain mass of an\",\n",
            "          \"is, minimum of two masses separated by the in-chain mass of an amino acid) or did not have a precursor MH+ in the range of 750\\u20136,000. MS/MS spectra from were searched against a database consisting of RefSeq release 60 containing 31,767 human proteins,\",\n",
            "          \"of RefSeq release 60 containing 31,767 human proteins, 24,821 mouse proteins, and an appended set of 85 common laboratory contaminant proteins (RefSeq.20130727-Human.20130730-MouseNR.mm13.contams). Scoring parameters were ESI-QEXACTIVE-HCD-v2, for whole\",\n",
            "          \"Scoring parameters were ESI-QEXACTIVE-HCD-v2, for whole proteome datasets, and ESI-QEXACTIVE-HCD-v3 parameters were for phosphoproteome datasets. All spectra were allowed \\u00b120 p.p.m. mass tolerance for precursor and product ions, 40% minimum matched peak\",\n",
            "          \"for precursor and product ions, 40% minimum matched peak intensity, and trypsin allow P enzyme specificity with up to four missed cleavages. Fixed modifications were carbamidomethylation at cysteine. iTRAQ labelling was required at lysine, but peptide\",\n",
            "          \"cysteine. iTRAQ labelling was required at lysine, but peptide N-termini were allowed to be either labelled or unlabelled. Allowed variable modifications for whole proteome datasets were acetylation of protein N-termini, oxidized methionine, deamidation of\",\n",
            "          \"of protein N-termini, oxidized methionine, deamidation of asparagine, pyro-glutamic acid at peptide N-terminal glutamine, and pyro-carbamidomethylation at peptide N-terminal cysteine with a precursor MH+ shift range of \\u221218 to 64\\u2009Da. Allowed variable\",\n",
            "          \"a precursor MH+ shift range of \\u221218 to 64\\u2009Da. Allowed variable modifications for phosphoproteome dataset were revised to disallow deamidation and allow phosphorylation of serine, threonine, and tyrosine with a precursor MH+ shift range of 0 to 272\\u2009Da.\",\n",
            "          \"and tyrosine with a precursor MH+ shift range of 0 to 272\\u2009Da. Identities interpreted for individual spectra were automatically designated as confidently assigned using the Spectrum Mill autovalidation module to use target-decoy based FDR estimates to\",\n",
            "          \"module to use target-decoy based FDR estimates to apply score threshold criteria via two-step strategies. For the whole proteome datasets thresholding was done at the spectral and protein levels. For the phosphoproteome datasets thresholding was done at\",\n",
            "          \"For the phosphoproteome datasets thresholding was done at the spectral level. In step 1, peptide autovalidation was done first and separately for each iTRAQ 4-plex experiment consisting of either 25 LC\\u2013MS/MS runs (whole proteome) or 13 LC\\u2013MS/MS runs\",\n",
            "          \"of either 25 LC\\u2013MS/MS runs (whole proteome) or 13 LC\\u2013MS/MS runs (phosphoproteome) using an auto thresholds strategy with a minimum sequence length of 7(whole proteome) or 8 (phosphoproteome), automatic variable range precursor mass filtering, and score\",\n",
            "          \"automatic variable range precursor mass filtering, and score and delta Rank1\\u2013Rank2 score thresholds optimized to yield a spectral level FDR estimate for precursor charges 2 through 4 of <0.6% for each precursor charge state in each LC\\u2013MS/MS run. For\",\n",
            "          \"<0.6% for each precursor charge state in each LC\\u2013MS/MS run. For precursor charges 5\\u20136, thresholds were optimized to yield a spectral level FDR estimate of <0.3% across all runs per iTRAQ 4-plex experiment (instead of each run), to achieve reasonable\",\n",
            "          \"4-plex experiment (instead of each run), to achieve reasonable statistics, since many fewer spectra are generated for the higher charge states. In step 2 for the whole proteome datasets, protein polishing autovalidation was applied separately to each\",\n",
            "          \"protein polishing autovalidation was applied separately to each iTRAQ 4-plex experiment to further filter the PSM's using a target protein-level FDR threshold of zero. The primary goal of this step was to eliminate peptides identified with low scoring\",\n",
            "          \"this step was to eliminate peptides identified with low scoring peptide spectrum matches (PSM's) that represent proteins identified by a single peptide, so-called \\u2018one-hit wonders'. After assembling protein groups from the autovalidated PSM's, protein\",\n",
            "          \"assembling protein groups from the autovalidated PSM's, protein polishing determined the maximum protein level score of a protein group that consisted entirely of distinct peptides estimated to be false-positive identifications (PSM's with negative delta\",\n",
            "          \"to be false-positive identifications (PSM's with negative delta forward-reverse scores). PSM's were removed from the set obtained in the initial peptide-level autovalidation step if they contributed to protein groups that have protein scores below the max\",\n",
            "          \"to protein groups that have protein scores below the max false-positive protein score. In the filtered results each identified protein detected in an iTRAQ 4-plex experiment was comprised of multiple peptides unless a single excellent scoring peptide was\",\n",
            "          \"multiple peptides unless a single excellent scoring peptide was the sole match. For the whole proteome datasets the above criteria yielded FDR of <0.5% at the peptide-spectrum match level and <0.8% at the distinct peptide level for each iTRAQ 4-plex\",\n",
            "          \"and <0.8% at the distinct peptide level for each iTRAQ 4-plex experiment. After assembling proteins with all the PSMs from all the iTRAQ 4-plex experiments together the aggregate FDR estimates were 0.42% at the at the peptide-spectrum match level, 1.5% at\",\n",
            "          \"were 0.42% at the at the peptide-spectrum match level, 1.5% at the distinct peptide level, and <0.01% (1/11,372) at the protein group level. Since the protein level FDR estimate neither explicitly required a minimum number of distinct peptides per protein\",\n",
            "          \"required a minimum number of distinct peptides per protein nor adjusted for the number of possible tryptic peptides per protein, it may underestimate false positive protein identifications for large proteins observed only on the basis of multiple low\",\n",
            "          \"for large proteins observed only on the basis of multiple low scoring PSMs. In calculating scores at the protein level and reporting the identified proteins, redundancy was addressed in the following manner: the protein score was the sum of the scores of\",\n",
            "          \"manner: the protein score was the sum of the scores of distinct peptides. A distinct peptide was the single highest scoring instance of a peptide detected through an MS/MS spectrum. MS/MS spectra for a particular peptide may have been recorded multiple\",\n",
            "          \"for a particular peptide may have been recorded multiple times, (that is, as different precursor charge states, in adjacent bRP fractions, modified by deamidation at Asn or oxidation of Met, or different phosphosite localization) but were still counted as\",\n",
            "          \"different phosphosite localization) but were still counted as a single distinct peptide. When a peptide sequence >8 residues long was contained in multiple protein entries in the sequence database, the proteins were grouped together and the highest\",\n",
            "          \"database, the proteins were grouped together and the highest scoring one and its accession number were reported. In some cases when the protein sequences were grouped in this manner there were distinct peptides that uniquely represented a lower scoring\",\n",
            "          \"distinct peptides that uniquely represented a lower scoring member of the group (isoforms, family members, and different species). Each of these instances spawned a subgroup. Multiple subgroups were reported and counted towards the total number of\",\n",
            "          \"subgroups were reported and counted towards the total number of proteins, and were given related protein subgroup numbers (for example, 3.1 and 3.2: group 3, subgroups 1 and 2). To better dissect the tumour/stroma (human/mouse) origin of orthologous\",\n",
            "          \"dissect the tumour/stroma (human/mouse) origin of orthologous proteins in this xenograft experiment, the inclusion of peptides contributing to each subgroup was restricted by enabling the subgroup-specific (SGS) option in Spectrum Mill. Only\",\n",
            "          \"the subgroup-specific (SGS) option in Spectrum Mill. Only subgroup-specific peptide sequences were counted toward each subgroup's count of distinct peptides and protein level TMT quantitation. The SGS option omits peptides that are shared between\",\n",
            "          \"The SGS option omits peptides that are shared between subgroups. If evidence for BOTH human and mouse peptides from an orthologous protein were observed, then peptides that cannot distinguish the two (shared) were ignored. However, the peptides shared\",\n",
            "          \"the two (shared) were ignored. However, the peptides shared between species were retained if there was specific evidence for only one of the species, thus yielding a single subgroup attributed to only the single species consistent with the specific\",\n",
            "          \"to only the single species consistent with the specific peptides. Furthermore, if all peptides observed for a protein group were shared between species, thus yielding a single subgroup composed of indistinguishable species, then all peptides were retained\",\n",
            "          \"of indistinguishable species, then all peptides were retained (the column in Supplementary Data 3 numSpeciesPresentR1 will have a value of 2 in such cases). Assembly of confidently identified PSM's yielded 20,480 total protein subgroups from 11,372\",\n",
            "          \"PSM's yielded 20,480 total protein subgroups from 11,372 protein groups. Human and mouse ortholog proteins were typically arranged into individual subgroups. In step 2 for the phosphoproteome datasets a phosphosite table were assembled with columns for\",\n",
            "          \"datasets a phosphosite table were assembled with columns for individual iTRAQ 4-plex experiments and rows for individual phosphosites. PSM's were combined into a single row for all non-conflicting observations of a particular phosphosite. (that is,\",\n",
            "          \"observations of a particular phosphosite. (that is, different missed cleavage forms, different precursor charges, confident and ambiguous localizations, different sample handling modifications). For related peptides neither observations with a different\",\n",
            "          \"For related peptides neither observations with a different number of phosphosites nor different confident localizations were allowed to be combined. Selecting the representative peptide from the combined observations was done such that once confident\",\n",
            "          \"the combined observations was done such that once confident phosphosite localization was established, higher identification scores and longer peptide lengths are preferable. After assembling the phosphosite table a polishing step was applied to further\",\n",
            "          \"the phosphosite table a polishing step was applied to further filter the phosphosites with the primary goal of eliminating phosphosites with representative peptides identified through low scoring peptide spectrum matches (PSM's) that were observed in only\",\n",
            "          \"peptide spectrum matches (PSM's) that were observed in only a few experiments. The initial table of representative peptides for 82,030 phosphosites had an aggregate FDR of 3.3% at phosphosite-level. The table was sorted by identification score and then by\",\n",
            "          \"The table was sorted by identification score and then by number of iTRAQ 4-plex experiments in which the phosphosite was observed. The cumulative FDR trend showed inflection points at an identification score of \\u223c8. Phosphosites with an identification\",\n",
            "          \"identification score of \\u223c8. Phosphosites with an identification score<8.0 observed in <3/9 experiments were therefore removed, yielding 68,385 phosphosites with an aggregate FDR of 0.34% at the phosphosite level. While the Spectrum Mill identification\",\n",
            "          \"the phosphosite level. While the Spectrum Mill identification score is based on the number of matching peaks, their ion type assignment, and the relative height of unmatched peaks, the phosphosite localization score is the difference in identification\",\n",
            "          \"localization score is the difference in identification score between the top two localizations. The score threshold for confident localization (>1.1) essentially corresponds to at least 1 b or y ion located between two candidate sites that has a peak\",\n",
            "          \"b or y ion located between two candidate sites that has a peak height 10% of the tallest fragment ion (neutral losses of phosphate from the precursor and related ions as well as immonium and iTRAQ reporter ions are excluded from the relative height\",\n",
            "          \"and iTRAQ reporter ions are excluded from the relative height calculation). The ion type scores for b-H3PO4, y-H3PO4, b-H2O, and y-H2O ion types are all set to 0.5. This prevents inappropriate confident localization assignment when a spectrum lacks\",\n",
            "          \"confident localization assignment when a spectrum lacks primary b or y ions between two possible sites but contains ions that can be assigned as either phosphate loss ions for one localization or water loss ions for another localization. In aggregate,\",\n",
            "          \"or water loss ions for another localization. In aggregate, 66.3% of the reported phosphosites were fully localized to a particular serine, threonine, or tyrosine residue. Relative abundances of proteins and phosphosites were determined in Spectrum Mill\",\n",
            "          \"of proteins and phosphosites were determined in Spectrum Mill using iTRAQ reporter ion intensity ratios from each PSM. A protein-level or phosphosite-level iTRAQ ratio was calculated as the median of all PSM level ratios contributing to a protein subgroup\",\n",
            "          \"of all PSM level ratios contributing to a protein subgroup or phosphosite remaining after excluding those PSM's lacking an iTRAQ label, having a negative delta forward-reverse score (half of all false-positive identifications), or having a precursor ion\",\n",
            "          \"all false-positive identifications), or having a precursor ion purity<50% (MS/MS has significant precursor isolation contamination from co-eluting peptides). Unless stated otherwise for a particular analysis, the following considerations apply to the\",\n",
            "          \"particular analysis, the following considerations apply to the tumour/stroma (human/mouse) origin of a protein in this xenograft experiment. For the proteome dataset, only PSM's from subgroup-specific peptide sequences contributed to the protein level\",\n",
            "          \"peptide sequences contributed to the protein level quantitation (see protein subgrouping description above). A protein detected with all contributing PSM's shared between human and mouse was considered to be human. For the phosphoproteome dataset, a\",\n",
            "          \"was considered to be human. For the phosphoproteome dataset, a phosphosite was considered to be mouse if the contributing PSM's were distinctly mouse and human if they were either distinctly human or shared between human and mouse. A 2-component Gaussian\",\n",
            "          \"human or shared between human and mouse. A 2-component Gaussian mixture model-based normalization approach was used to centre the distribution of iTRAQ log-ratios around zero to nullify the effect of differential protein loading and/or systematic MS\",\n",
            "          \"the effect of differential protein loading and/or systematic MS variation. Downstream analyses presented in the main figures were restricted to proteins/phosphosites quantified in at least 10 out of the 24 samples with non-missing values (Supplementary\",\n",
            "          \"10 out of the 24 samples with non-missing values (Supplementary Fig. 3), with the exception of the previously described mRNA-protein correlation analysis requiring quantification in 30%, or 8 out of 24, PDX samples. Specific filtering procedures are noted\",\n",
            "          \"out of 24, PDX samples. Specific filtering procedures are noted in descriptions of the relevant methods. Bioinformatics analyses. Cross data-type integration All gene names were converted to HUGO Gene Nomenclature Committee's approved gene names for\",\n",
            "          \"to HUGO Gene Nomenclature Committee's approved gene names for comparison across levels and datasets. For mRNA and protein expressions, expression values were collapsed across transcripts or isoforms to the corresponding gene using the highest mean when\",\n",
            "          \"isoforms to the corresponding gene using the highest mean when there were two transcripts or isoforms, or the value with the highest connectivity when there were more than three transcripts or isoforms as implemented in the WGCNA R package. mRNA-protein\",\n",
            "          \"or isoforms as implemented in the WGCNA R package. mRNA-protein correlation Spearman correlations between normalized RSEM values and protein quantifications were calculated for genes that were observed in at least 30% of samples for both RNA-seq and\",\n",
            "          \"were observed in at least 30% of samples for both RNA-seq and mass-spectrometry as previously described. KEGG pathway enrichment analysis of the correlation was carried out using a Kolmogorov\\u2013Smirnov test, and P values were adjusted using the\",\n",
            "          \"a Kolmogorov\\u2013Smirnov test, and P values were adjusted using the Benjamini-Hochberg procedure. Proteomic clustering and subtyping We first applied filtering for protein and phosphosite markers observed in at least 10 samples and sufficient deviations\",\n",
            "          \"observed in at least 10 samples and sufficient deviations across samples. We used a 2 s.d. threshold for iTRAQ proteome and 2.5 s.d. for the phosphoproteome. We applied the same protein marker to conduct LFQ proteome and PDX vs. human proteome\",\n",
            "          \"marker to conduct LFQ proteome and PDX vs. human proteome co-clustering. For the co-clustering with human proteome, we further selected for markers that had higher than 2 s.d.'s in the merged proteome and showed non-differential expression between human\",\n",
            "          \"proteome and showed non-differential expression between human and PDX (FDR>0.3, t-test). The subsequent hierarchical clustering was conducted using the complete agglomeration method of hclust as implemented in the heatmap2 R package. Differential\",\n",
            "          \"hclust as implemented in the heatmap2 R package. Differential expression analysis Differential expression testing of each protein in the LFQ and iTRAQ datasets was conducted using the student's t-test, and P values were adjusted using the\",\n",
            "          \"the student's t-test, and P values were adjusted using the Benjamini-Hochberg procedure. For gene set enrichment analysis, we conducted the Wilcoxon Rank Sum Test to test for changes in the t-statistics ranks of protein members in each of the KEGG\",\n",
            "          \"the t-statistics ranks of protein members in each of the KEGG signalling pathway, and again adjusted P values using the Benjamini-Hochberg procedure. Druggable genes and mutations We compiled a list of druggable genes based on extensive curation of public\",\n",
            "          \"a list of druggable genes based on extensive curation of public databases: Tumor Alterations Relevant for GEnomics-driven Therapy (TARGET, version 3, assessed on 6/15/2015), Personalized Cancer Therapy (PCT, assessed on 3/15/2015), GDKD (Gene-Drug\",\n",
            "          \"Cancer Therapy (PCT, assessed on 3/15/2015), GDKD (Gene-Drug Knowledge Database, version 11.0, assessed on 4/10/2015), CancerDR (assessed on 2/6/2015), My Cancer Genome (assessed on 9/11/2014), and DrugBank (assessed on 9/21/2015). We curated the list\",\n",
            "          \"and DrugBank (assessed on 9/21/2015). We curated the list based on evidence level and literature, as well as IC50 data when available. The final list used for the analysis included 76 druggable genes (Supplementary Data 15). Druggable outlier analysis To\",\n",
            "          \"genes (Supplementary Data 15). Druggable outlier analysis To discover expression outliers, we utilized a strategy incorporating multiple steps: first, we limited our search to genes that are in the druggable gene list. We then narrowed down the list to\",\n",
            "          \"in the druggable gene list. We then narrowed down the list to genes that are observed in at least 10 samples in the dataset under investigation. Outlier expressions were defined as values that are greater than 1.5 interquartile ranges (IQRs) above the\",\n",
            "          \"that are greater than 1.5 interquartile ranges (IQRs) above the third quartile (Q3). To rank order outlier expression for each gene, we calculated an outlier score defined as: By definition, genes with outlier score greater than 1.5 are considered as\",\n",
            "          \"genes with outlier score greater than 1.5 are considered as expression outliers. Outlier score for each gene were ranked within the sample to select the most promising druggable targets. For CNV outliers, we required them to have outlier scores above 1 in\",\n",
            "          \"outliers, we required them to have outlier scores above 1 in at least another expression level. For validation of the proteomic druggable outliers, we counted the numbers of the same protein or phosphosite outliers observed in the parallel-processed\",\n",
            "          \"or phosphosite outliers observed in the parallel-processed cohort of 77 human breast cancer samples. Due to the lower numbers in this larger cohort, we considered both proteins with outlier score greater than 1 and the top 2 outliers of each human sample\",\n",
            "          \"greater than 1 and the top 2 outliers of each human sample as validating outliers. Pathway activation analysis We first collapsed the phosphosites to gene-level phosphorylation values by averaging the phosphosite expressions observed for each gene. Then,\",\n",
            "          \"the phosphosite expressions observed for each gene. Then, we converted the phosphoproteomic expression values from iTRAQ to modified z-scores normalized against the cohort as described in Hoaglin et al. (How to Detect and Handle Outliers). We then used\",\n",
            "          \"et al. (How to Detect and Handle Outliers). We then used the Wilcoxon Rank Sum Test to test for changes in the phosphorylation z-score ranks of protein members in each of the KEGG signalling pathway. The resulting P values were adjusted using the\",\n",
            "          \"pathway. The resulting P values were adjusted using the Benjamini\\u2013Hochberg procedure. Data availability All mass spectra, in the original instrument vendor format, contributing to this study may be downloaded from:\",\n",
            "          \"format, contributing to this study may be downloaded from: https://cptac-data-portal.georgetown.edu/cptacPublic/ for the study name: TCGA Breast Cancer. Additional information How to cite this article: Huang, K.-l. et al. Proteogenomic integration reveals\",\n",
            "          \"article: Huang, K.-l. et al. Proteogenomic integration reveals therapeutic targets in breast cancer xenografts. Nat. Commun. 8, 14864 doi: 10.1038/ncomms14864 (2017). Publisher's note: Springer Nature remains neutral with regard to jurisdictional claims\",\n",
            "          \"Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Supplementary Material C.M. Perou is on the board of directors, has ownership interest (including patents), and is a consultant/advisory board\",\n",
            "          \"(including patents), and is a consultant/advisory board member for bioclassifier LLC. The other authors declare no competing financial interests. Author contributions L.D. directed data analysis. L.D., M.J.E., S.A.C., X.C., H.R. and R.R.T. designed and\",\n",
            "          \"L.D., M.J.E., S.A.C., X.C., H.R. and R.R.T. designed and supervised research and experiment. K.-l.H., S.L., P.M., H.P.G., S.C., C.Y., D.R.M., K.R.C., C.R.K., J.W., K.V.R., D.F., M.D.M., S.M.K., S.S. and B.N. analyzed the data. K.-l.H., M.C.W. and D.R.M.\",\n",
            "          \"S.S. and B.N. analyzed the data. K.-l.H., M.C.W. and D.R.M. performed statistical analysis. S.L. established the PDX models. S.L., Z.G., S.M.K., M.T., J.U., J.E.S., J.H., P.E.-G., P.S., C.M. and C.M.P. performed functional validation experiments. J.W.Q.,\",\n",
            "          \"and C.M.P. performed functional validation experiments. J.W.Q., P.M., C.M., L.X. and H.P.G. performed data collection and tissue acquisition. M.A.W., K.-l.H., C.Y., S.C. and L.D. prepared figures and tables. M.J.E., S.R.D. and S.L. provided samples.\",\n",
            "          \"figures and tables. M.J.E., S.R.D. and S.L. provided samples. K.-l.H. and L.D. wrote the manuscript. M.J.E., M.C.W., P.M., S.A.C., X.C., R.R.T., C.M., K.-l.H. and L.D. revised the manuscript. All authors approved and contributed to the preparation of the\",\n",
            "          \"All authors approved and contributed to the preparation of the manuscript. 04/25/2017 The original version of this Article contained a typographical error in the spelling of the author Beifang Niu, which was incorrectly given as Beifung Niu. This has now\",\n",
            "          \"Niu, which was incorrectly given as Beifung Niu. This has now been corrected in both the PDF and HTML versions of the Article. Modelling human breast cancer with patient-derived xenografts (n=24). (a) Illustration of generation and proteogenomic\",\n",
            "          \"(n=24). (a) Illustration of generation and proteogenomic characterization of breast cancer xenograft models. (b) Somatic mutations of significantly mutated genes of human breast tumour were recapitulated in xenograft models. Mutation data for 23 WHIMs are\",\n",
            "          \"in xenograft models. Mutation data for 23 WHIMs are shown (exome data were not available for WHIM47). (c) Variant allele fraction analysis showed clonal representation was consistent between human breast tumour and xenografts. Genomic driver events,\",\n",
            "          \"human breast tumour and xenografts. Genomic driver events, including missenses and truncations in TP53 and PIK3CA, were retained in the xenograft models. Each colour represents one xenograft sample. Proteogenomic correlation analysis in PDX samples. (a)\",\n",
            "          \"sample. Proteogenomic correlation analysis in PDX samples. (a) Correlation between mRNA and iTRAQ protein expression levels identified pathways with significantly concordant or discordant mRNA-protein expressions. Genes were aligned along the x axis by\",\n",
            "          \"expressions. Genes were aligned along the x axis by the rank of their Spearman correlation coefficient between mRNA and protein expression levels. Each colour represents one significantly associated pathway, and each bar represents one gene in the\",\n",
            "          \"associated pathway, and each bar represents one gene in the pathway. (b) Proteogenomic summary of xenograft shows relationships among mutation, CNV (normalized log-R ratio), mRNA (log-transformed and normalized RSEM values), proteomic (normalized log2\",\n",
            "          \"and normalized RSEM values), proteomic (normalized log2 ratio to reference), and phosphoproteomic expression (normalized log2 ratio to reference) levels of breast cancer-related genes in 24 PDX samples across 4 intrinsic subtypes. Expression values from\",\n",
            "          \"PDX samples across 4 intrinsic subtypes. Expression values from each dataset were calculated as described (Methods) and truncated to a maximum of 10 and a minimum of \\u221210 for visualization. Transcriptomic and proteomic clustering of breast cancer PDX and\",\n",
            "          \"and proteomic clustering of breast cancer PDX and human samples. (a) Transcriptomic clustering of PDX breast tumours based on the PAM50 gene expression markers. (b) Proteomic clustering of PDX breast tumours based on the top 436 variably expressed\",\n",
            "          \"of PDX breast tumours based on the top 436 variably expressed proteins. (c) Phosphoproteomic clustering of PDX breast tumours based on the top 1,737 variably expressed phosphosites. (d) Proteomic clustering using only 133 non-differential expressed\",\n",
            "          \"Proteomic clustering using only 133 non-differential expressed proteins between WHIM and human breast tumour samples. The clustering reproduced the basal-enriched and luminal-enriched clusters, where PDX (n=24) and TCGA human breast tumour samples (n=77)\",\n",
            "          \"where PDX (n=24) and TCGA human breast tumour samples (n=77) cluster based on their subtypes. The non-differentially expressed proteins were identified through a t-test with FDR>0.3 between the PDX and the TCGA human tumour samples. The PDX tumours are\",\n",
            "          \"the PDX and the TCGA human tumour samples. The PDX tumours are labelled by their WHIM IDs whereas the human tumours are not labelled. Activated signalling pathways detected through pathway phosphorylation enrichment analysis. (a) Activation of the MAPK\",\n",
            "          \"phosphorylation enrichment analysis. (a) Activation of the MAPK signalling pathway in WHIM9. (b) Activation of the Ras signalling pathway in WHIM12. (c) Activation of the NF\\u03baB pathway in WHIM17. (d) Activation of the NF\\u03baB pathway in WHIM46.\",\n",
            "          \"in WHIM17. (d) Activation of the NF\\u03baB pathway in WHIM46. Phosphorylation levels of each protein in the pathway relative to the cohort of 24 PDX models are shown by the colour scale of red (high) to blue (low). Proteins with no phosphorylation data are\",\n",
            "          \"(high) to blue (low). Proteins with no phosphorylation data are coloured in green. Druggable events identified by expression outlier analysis. Druggable outlier events identified at (a) the mRNA and protein and (b) protein and phosphopeptide expression\",\n",
            "          \"mRNA and protein and (b) protein and phosphopeptide expression levels. Each colour represents one xenograft sample. Key outlier events validated in this study are labelled by text. (c) Immunochemistry staining verified outlier expression of HER2 in WHIM8\",\n",
            "          \"staining verified outlier expression of HER2 in WHIM8 and WHIM35, RAF1 in WHIM9, and FGFR2 in WHIM16. Scale bar: 50\\u2009\\u03bcm. Druggable targets identified through proteogenomic analysis in PDX and human breast cancer. (a) Outlier analysis revealed potentially\",\n",
            "          \"human breast cancer. (a) Outlier analysis revealed potentially druggable events in the RTK, PI3K, MAPK signalling, genome integrity pathways at various frequency and magnitudes across four breast cancer subtypes. Selected genes with any outlier score\",\n",
            "          \"breast cancer subtypes. Selected genes with any outlier score greater than 2.5 or in the key oncogenic pathways, including the PI3K, RTK, MAPK signalling pathways, are shown. (b) Comparison of protein expression outliers of selected druggable genes in PDX\",\n",
            "          \"protein expression outliers of selected druggable genes in PDX and human breast tumours. (c) Comparison of overexpressed phosphosite outliers of selected druggable genes in PDX and human breast tumours. Key outlier events reaching the outlier definition\",\n",
            "          \"tumours. Key outlier events reaching the outlier definition threshold or validated in this study are labelled by text. For the box plots in (b) and (c), the center line indicates median of the protein/phosphosite expression in the human and PDX cohort.\",\n",
            "          \"the protein/phosphosite expression in the human and PDX cohort. The upper and lower hinges correspond to the 25th and 75th percentiles; the upper whisker corresponds to 1.5 \\u00d7 IQR (inter-quartile range) above the 25 percentile and the lower whisker\",\n",
            "          \"range) above the 25 percentile and the lower whisker corresponds to 1.5 \\u00d7 IQR below the 75 percentile. Targeted treatments of breast cancer xenografts. (a) Western blot of HER2 protein and HER2 p.Y1248 expression levels in 5 WHIM models (WHIM6, WHIM8,\",\n",
            "          \"HER2 p.Y1248 expression levels in 5 WHIM models (WHIM6, WHIM8, WHIM12, WHIM14 and WHIM35). (b) In vivo treatment responses to lapatinib in 4 PDX models including two HER2 positive lines (WHIM8 and WHIM35) and two basal lines (WHIM6 and WHIM14). The\",\n",
            "          \"(WHIM8 and WHIM35) and two basal lines (WHIM6 and WHIM14). The response is measured in fold change (%) of tumour volumes after 2 weeks of vehicle or lapatinib treatment. (c) Immunochemistry staining of AKT phosphorylation status in WHIM16, WHIM18 and\",\n",
            "          \"staining of AKT phosphorylation status in WHIM16, WHIM18 and WHIM20 in response to PI3K inhibitor TAK-117, mTOR inhibitor TAK128, and TAK-117/TAK-128 combined. (d) In vivo treatment responses to PI3K/mTOR inhibitors (TAK-117/TAK-128) of WHIM16, WHIM18 and\",\n",
            "          \"to PI3K/mTOR inhibitors (TAK-117/TAK-128) of WHIM16, WHIM18 and WHIM20 (n=8-9 for each control or experimental group). Values were represented by average tumour volume (mm3) every 3 days following treatment (error bars: s.e.m.).\"\n",
            "        ]\n",
            "      }\n",
            "    ]\n",
            "  },\n",
            "  \"PDC000303\": {\n",
            "    \"study_id\": \"f0f337b3-563b-4335-83d8-7e65146102a0\",\n",
            "    \"pdc_study_id\": \"PDC000303\",\n",
            "    \"study_submitter_id\": \"Therapeutic Targets in Breast Cancer Xenografts -  Proteome\",\n",
            "    \"program_id\": \"10251935-5540-11e8-b664-00a098d917f8\",\n",
            "    \"project_id\": \"8dbcf809-3df2-4c57-a9d5-8d58d5824ffa\",\n",
            "    \"study_name\": \"Therapeutic Targets in Breast Cancer Xenografts -  Proteome\",\n",
            "    \"program_name\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"project_name\": \"CPTAC2-Other\",\n",
            "    \"disease_type\": \"Other\",\n",
            "    \"primary_site\": \"Not Reported\",\n",
            "    \"analytical_fraction\": \"Proteome\",\n",
            "    \"experiment_type\": \"iTRAQ4\",\n",
            "    \"embargo_date\": null,\n",
            "    \"cases_count\": 27,\n",
            "    \"aliquots_count\": 27,\n",
            "    \"protocol_id\": \"93e99c96-49ac-40ee-9466-b113f4c62cf8\",\n",
            "    \"protocol_submitter_id\": \"Breast Cancer Xenografts -  Proteome\",\n",
            "    \"program_submitter_id\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"protocol_name\": \"Breast Cancer Xenografts -  Proteome\",\n",
            "    \"protocol_date\": \"2021-05-06\",\n",
            "    \"publications\": [\n",
            "      {\n",
            "        \"publication_id\": \"718f77fb-b329-11eb-9f63-0a2b20c3a9e9\",\n",
            "        \"pubmed_id\": \"28348404\",\n",
            "        \"doi\": \"10.1038/ncomms14864\",\n",
            "        \"title\": \"Proteogenomic integration reveals therapeutic targets in breast cancer xenografts\",\n",
            "        \"journal\": \"Nature Communications\",\n",
            "        \"journal_url\": \"https://www.nature.com/articles/ncomms14864\",\n",
            "        \"year\": \"2017\",\n",
            "        \"abstract\": \"Recent advances in mass spectrometry (MS) have enabled extensive analysis of cancer proteomes. Here, we employed quantitative proteomics to profile protein expression across 24 breast cancer patient-derived xenograft (PDX) models. Integrated proteogenomic analysis shows positive correlation between expression measurements from transcriptomic and proteomic analyses; further, gene expression-based intrinsic subtypes are largely re-capitulated using non-stromal protein markers. Proteogenomic analysis also validates a number of predicted genomic targets in multiple receptor tyrosine kinases. However, several protein/phosphoprotein events such as overexpression of AKT proteins and ARAF, BRAF, HSP90AB1 phosphosites are not readily explainable by genomic analysis, suggesting that druggable translational and/or post-translational regulatory events may be uniquely diagnosed by MS. Drug treatment experiments targeting HER2 and components of the PI3K pathway supported proteogenomic response predictions in seven xenograft models. Our study demonstrates that MS-based proteomics can identify therapeutic targets and highlights the potential of PDX drug response evaluation to annotate MS-based pathway activities.\",\n",
            "        \"full_text\": [\n",
            "          \"Proteogenomic integration reveals therapeutic targets in breast cancer xenografts Recent advances in mass spectrometry (MS) have enabled extensive analysis of cancer proteomes. Here, we employed quantitative proteomics to profile protein expression across\",\n",
            "          \"quantitative proteomics to profile protein expression across 24 breast cancer patient-derived xenograft (PDX) models. Integrated proteogenomic analysis shows positive correlation between expression measurements from transcriptomic and proteomic analyses;\",\n",
            "          \"measurements from transcriptomic and proteomic analyses; further, gene expression-based intrinsic subtypes are largely re-capitulated using non-stromal protein markers. Proteogenomic analysis also validates a number of predicted genomic targets in\",\n",
            "          \"also validates a number of predicted genomic targets in multiple receptor tyrosine kinases. However, several protein/phosphoprotein events such as overexpression of AKT proteins and ARAF, BRAF, HSP90AB1 phosphosites are not readily explainable by genomic\",\n",
            "          \"HSP90AB1 phosphosites are not readily explainable by genomic analysis, suggesting that druggable translational and/or post-translational regulatory events may be uniquely diagnosed by MS. Drug treatment experiments targeting HER2 and components of the\",\n",
            "          \"Drug treatment experiments targeting HER2 and components of the PI3K pathway supported proteogenomic response predictions in seven xenograft models. Our study demonstrates that MS-based proteomics can identify therapeutic targets and highlights the\",\n",
            "          \"proteomics can identify therapeutic targets and highlights the potential of PDX drug response evaluation to annotate MS-based pathway activities. Patient-derived xenografts recapitulate major genomic signatures and transcriptome profiles of their original\",\n",
            "          \"genomic signatures and transcriptome profiles of their original tumours. Here, the authors, performing proteomic and phosphoproteomic analyses of 24 breast cancer PDX models, demonstrate that druggable candidates can be identified based on a comprehensive\",\n",
            "          \"druggable candidates can be identified based on a comprehensive proteogenomic profiling. Profiling of somatic alteration by next-generation DNA sequencing (NGS) has entered clinical practice with the promise of rapid diagnosis of druggable somatic genomic\",\n",
            "          \"the promise of rapid diagnosis of druggable somatic genomic alterations for personalized cancer treatment. For example, recent analysis of 4,068 samples from 16 cancer types suggested that repurposing approved drugs based on genomic alterations could\",\n",
            "          \"repurposing approved drugs based on genomic alterations could provide individualized treatment options for around 40% of tumours. However, clinical evidence for this proposition is limited and has been slow to develop. Further, the signalling and\",\n",
            "          \"and has been slow to develop. Further, the signalling and biological effects of somatic mutations are not routinely determined in human tumour samples even though this is a consideration for rational drug design, response prediction and target\",\n",
            "          \"for rational drug design, response prediction and target prioritization. Finally, druggable genomic alterations are not detected in the majority of cases tested by NGS. Comprehensive proteomic analyses provide a potentially valuable approach to validate\",\n",
            "          \"analyses provide a potentially valuable approach to validate genomic findings as likely biological drivers and to discover discover opportunities for targeted treatment. Patient-derived xenografts (PDXs) in immunodeficient mice maintain the histological\",\n",
            "          \"(PDXs) in immunodeficient mice maintain the histological and molecular heterogeneity of the progenitor human tumour and cytotoxic drug responsiveness is often a transplantable phenotype. Our previous studies have shown that breast tumour PDXs recapitulate\",\n",
            "          \"studies have shown that breast tumour PDXs recapitulate major genomic signatures and transcriptome profiles of their original breast tumours. Moreover, drug responses to endocrine therapy in breast cancer PDXs resembled that observed in the corresponding\",\n",
            "          \"breast cancer PDXs resembled that observed in the corresponding patient and endocrine therapy resistance patterns were associated with aberrations in the ESR1 gene. While comprehensive proteomic characterization of PDX is still lacking, recent studies\",\n",
            "          \"characterization of PDX is still lacking, recent studies using reverse phase protein array have identified similar protein profiles between PDX and primary tumours. These studies collectively suggest that the PDX approach is a potentially valuable\",\n",
            "          \"suggest that the PDX approach is a potentially valuable preclinical model for identification and testing of therapeutic targets. Recent advances in mass-spectrometry provide an unprecedented opportunity for antibody-independent proteome profiling with\",\n",
            "          \"opportunity for antibody-independent proteome profiling with approximately 80% of all proteins in major human tissues quantifiable by this technique. Analyses of the global proteome in tandem with genomic profiles generated through TCGA have yielded key\",\n",
            "          \"with genomic profiles generated through TCGA have yielded key insights on the molecular etiology of colorectal cancer, breast cancer and ovarian cancer. Our recent proteogenomic characterization of 105 human breast cancers linked 5q-loss to elevated EGFR\",\n",
            "          \"of 105 human breast cancers linked 5q-loss to elevated EGFR expression and identified overexpression of phosphorylated kinases including CDK12, PAK1 and ARAF as potential biological drivers. However, current proteogenomic analyses of human tissues are\",\n",
            "          \"However, current proteogenomic analyses of human tissues are limited by the fact that therapeutic hypotheses were not being addressed in these initial analyses. As patient treatments were heterogeneous, links between treatment and proteogenomic findings\",\n",
            "          \"links between treatment and proteogenomic findings are difficult to establish in banked tumour specimens. The PDX system therefore provides an early opportunity to determine whether a proteogenomic approach can produce more robust therapeutic hypotheses\",\n",
            "          \"approach can produce more robust therapeutic hypotheses and predictive biomarkers for individual patients than that achieved by genomic analysis alone. In this study, we generated high-coverage proteome and phosphoproteome data for 24 PDX models\",\n",
            "          \"proteome and phosphoproteome data for 24 PDX models representing different breast cancer subtypes using both isobaric mass-tag labelling and label-free proteomic methods for cross-validated identification of 12,794 proteins and 56,874 phosphorylation\",\n",
            "          \"identification of 12,794 proteins and 56,874 phosphorylation sites. After filtering for observation in at least 10 out of 24 samples 10,069 proteins and 36,609 phosphorylation sites with their relative abundances quantified across tumours, were used in\",\n",
            "          \"relative abundances quantified across tumours, were used in subsequent analyses in this study. A substantial number of druggable protein/phosphoprotein events were uniquely identified. Further, similar proteomic/phosphoproteomic signatures were observed\",\n",
            "          \"similar proteomic/phosphoproteomic signatures were observed in patient breast cancer samples indicating potential clinical relevance. In selected PDX models exhibiting proteomic and/or phosphoproteomic up-regulation events of HER2 or the PI3K pathway, we\",\n",
            "          \"up-regulation events of HER2 or the PI3K pathway, we validated the efficacy of targeted inhibitors in suppressing PDX growth. Taken together, these observations show that comprehensive proteogenomic profiling has potential to identify new targets for\",\n",
            "          \"profiling has potential to identify new targets for individualized treatment approaches in cancer. Results Proteogenomic coverage of breast cancer xenografts We selected 24 PDX models established from primary or metastatic breast tumours for comprehensive\",\n",
            "          \"from primary or metastatic breast tumours for comprehensive proteogenomic characterization (Fig. 1a). The human patient cohort was composed of 10 basal, 1 claudin-low (CLDN-low), 9 luminal B and 4 HER2-enriched (HER2-E) breast tumours based on PAM50\",\n",
            "          \"B and 4 HER2-enriched (HER2-E) breast tumours based on PAM50 expression subtyping (Supplementary Data 1). We conducted DNA and RNA sequencing respectively for 23 PDX models and in one case Sanger DNA sequencing of hotspot mutations. Isotope Tagging for\",\n",
            "          \"Sanger DNA sequencing of hotspot mutations. Isotope Tagging for Relative and Absolute Quantitation (iTRAQ 4-plex) was completed for all 24 PDXs for discovery and label-free quantification (LFQ) for 18 PDXs for validation and confirmation (Supplementary\",\n",
            "          \"for 18 PDXs for validation and confirmation (Supplementary Data 2). The comprehensive proteogenomic coverage allowed us to systematically assess the somatic mutation profile, copy-number variation (CNV), mRNA expression, protein expression, and\",\n",
            "          \"variation (CNV), mRNA expression, protein expression, and phosphosite levels and localizations. We examined somatic mutations in significantly mutated genes of breast cancer in 12 human tumours and their matching PDXs (Supplementary Data 2). Most key\",\n",
            "          \"and their matching PDXs (Supplementary Data 2). Most key somatic mutations in these genes were preserved (Fig. 1b), validating the genomic fidelity of these PDX models. However, recurrent breast cancer mutations were not detected in two PDX models (WHIM17\",\n",
            "          \"cancer mutations were not detected in two PDX models (WHIM17 and WHIM46). While sequencing data were not available for the matched progenitor human tumours, the germline SNPs of human blood normal samples matched with the PDX tumours, validating their\",\n",
            "          \"normal samples matched with the PDX tumours, validating their patients of origin. Follow-up histological and RNA-seq analyses suggested WHIM17 and WHIM46 are EBV-positive human lymphoproliferative cells arising in the NOD/SCID/\\u03b3 mouse strain (Methods,\",\n",
            "          \"cells arising in the NOD/SCID/\\u03b3 mouse strain (Methods, Supplementary Fig. 1). We then compared the variant allele fractions (VAFs) of exonic somatic mutations in human tumours and derived xenografts when both are available. We found comparable or higher\",\n",
            "          \"when both are available. We found comparable or higher VAFs in xenografts, potentially due to higher tumour purity and selection of some mutant alleles by loss of heterozygosity in PDX models (Fig. 1c). The relatively high, positive VAF correlation\",\n",
            "          \"models (Fig. 1c). The relatively high, positive VAF correlation (R=0.66) implied similarity between human samples and their respective xenografts, consistent with our previous report. Importantly, all PIK3CA and TP53 somatic mutations in the 12\",\n",
            "          \"Importantly, all PIK3CA and TP53 somatic mutations in the 12 originating human tumours were detected in respective xenografts with comparable or increased VAFs (Fig. 1c). Among the 22 breast cancer PDX models, 14, including 7 of the 8 basal tumours,\",\n",
            "          \"cancer PDX models, 14, including 7 of the 8 basal tumours, harboured TP53 mutations, 5 luminal B PDXs had ESR1 mutations, and 5 luminal B and 1 CLDN-low xenografts carried PIK3CA mutations. We quantified mRNA expression of 16,209 unique human genes from\",\n",
            "          \"We quantified mRNA expression of 16,209 unique human genes from RNA-seq data for these 24 models. We also applied two distinct MS data acquisition approaches (Supplementary Fig. 2a, Methods), iTRAQ (Supplementary Data 3) and LFQ (Supplementary Data 4), to\",\n",
            "          \"iTRAQ (Supplementary Data 3) and LFQ (Supplementary Data 4), to quantify the expression ratios of 12,794 human proteins (11,879 genes) and 8,648 proteins (8,035 genes), respectively. A total of 56,874 phosphosites were also confidently identified using\",\n",
            "          \"of 56,874 phosphosites were also confidently identified using iTRAQ (Supplementary Data 5). After filtering for observation in at least 10 out of 24 samples (4 out of 9 of the iTRAQ experiments, Supplementary Fig. 3), the relative abundances of 10,069\",\n",
            "          \"Supplementary Fig. 3), the relative abundances of 10,069 proteins and 36,609 phosphorylation sites were quantified across tumours by iTRAQ and used in subsequent analyses in this study. The technical replicates in the LFQ (WHIM2 and WHIM16) and the iTRAQ\",\n",
            "          \"replicates in the LFQ (WHIM2 and WHIM16) and the iTRAQ (WHIM13) sets showed high correlation in protein expression levels (R>0.85, Supplementary Fig. 2b,c,e). Further, phosphosite expressions also showed high correlations in technical replicates (WHIM13)\",\n",
            "          \"also showed high correlations in technical replicates (WHIM13) of the iTRAQ experiment (R=0.82, Supplementary Fig. 2d), validating the technical reproducibility of our proteomic and phosphoproteomic datasets. Of note, while iTRAQ and LFQ quantification\",\n",
            "          \"datasets. Of note, while iTRAQ and LFQ quantification were conducted separately and based on different features of the LC\\u2013MS data collected from the Q Exactive mass spectrometer (that is, reporter ion intensity and MS1 peak area, respectively), our\",\n",
            "          \"reporter ion intensity and MS1 peak area, respectively), our analysis showed reasonable correlation between the two measurements after normalization (R=0.61). The two datasets utilizing the same PDX models but different workflows enabled cross-method\",\n",
            "          \"same PDX models but different workflows enabled cross-method validation of global proteomic results. Proteogenomic integration and comparison We first evaluated the correlation between mRNA expression and protein abundance measurements from the iTRAQ\",\n",
            "          \"expression and protein abundance measurements from the iTRAQ experiment (Fig. 2a) for these 24 PDX models: 83.6% of the genes with sufficient data showed positive correlations with a median Pearson R=0.536 (Supplementary Fig. 4). We investigated whether\",\n",
            "          \"Pearson R=0.536 (Supplementary Fig. 4). We investigated whether the trend of mRNA-protein correlations were associated with specific KEGG pathways, finding that metabolic pathways involved in house-keeping functions are enriched for genes showing high,\",\n",
            "          \"in house-keeping functions are enriched for genes showing high, positive correlation. For example, genes in the glutathione metabolism pathway showed the highest enrichment for positive correlations (Kolmogorov\\u2013Smirnov test, false discovery rate\",\n",
            "          \"correlations (Kolmogorov\\u2013Smirnov test, false discovery rate (FDR)=5.6e-07). Interestingly, we observed that genes in the ribosome (FDR<2.2e-16), spliceosome (FDR=2.0e-13), RNA transport (FDR=1.7e-5), RNA polymerase (FDR=1.8e-4), and oxidative\",\n",
            "          \"(FDR=1.7e-5), RNA polymerase (FDR=1.8e-4), and oxidative phosphorylation (FDR<2.2e-16) pathways showed relatively lower correlation between mRNA and protein level. These pathways were enriched for genes that do not require translated proteins for their\",\n",
            "          \"for genes that do not require translated proteins for their biological functions. Similar pathway-specific pattern of positive and negative enrichment of correlations were observed by LFQ (Supplementary Fig. 5). The high degree of mRNA\\u2013protein correlation\",\n",
            "          \"Fig. 5). The high degree of mRNA\\u2013protein correlation observed in the PDX samples is consistent with recent results obtained for human breast tumours and colorectal cancer, suggesting that PDXs closely mimic the respective human breast tumours in their\",\n",
            "          \"PDXs closely mimic the respective human breast tumours in their relationship between mRNA and protein. We then examined the correlation of CNV, mRNA, and protein expression levels for several key genes for breast cancer biology: EGFR, ERBB2 (HER2), ESR1,\",\n",
            "          \"key genes for breast cancer biology: EGFR, ERBB2 (HER2), ESR1, GATA3, PGR, PIK3CA, AKT1/2/3, MTOR and TP53. In most cases, we observed consistent relationships between CNV, mRNA, and protein expression levels. Compared with PDXs of non-luminal subtypes,\",\n",
            "          \"expression levels. Compared with PDXs of non-luminal subtypes, luminal B breast cancer xenografts, as expected, showed higher mRNA and protein expression of ESR1 and PGR, consistent with their positive ER and PR status (Fig. 2b). Five out of the six\",\n",
            "          \"their positive ER and PR status (Fig. 2b). Five out of the six PIK3CA mutations were observed in luminal B PDXs, which tended to also show higher mRNA and protein expression levels of GATA3. In contrast, a larger proportion of basal PDXs expressed higher\",\n",
            "          \"In contrast, a larger proportion of basal PDXs expressed higher protein levels of EGFR. Strong HER2 expression at the mRNA, protein and phosphoprotein levels were detected in WHIM8 and WHIM35, both derived from HER2-positive breast cancers. Overall\",\n",
            "          \"WHIM35, both derived from HER2-positive breast cancers. Overall expression patterns for key genes were consistent with the clinical subtype diagnosis across CNV, mRNA, protein and phosphosite analyses (Supplementary Fig. 6). Proteomic subtyping of\",\n",
            "          \"analyses (Supplementary Fig. 6). Proteomic subtyping of xenografts and human breast tumours Molecular subtyping of breast cancer based on mRNA expression profiles has been shown to correlate with prognosis and has treatment implications. As expected,\",\n",
            "          \"with prognosis and has treatment implications. As expected, transcriptome clustering based on PAM50 genes from RNA-seq data largely reproduced intrinsic subtypes of the breast tumours (Fig. 3a; Supplementary Data 6). To explore proteomic and\",\n",
            "          \"(Fig. 3a; Supplementary Data 6). To explore proteomic and phosphoproteomic subtype classifications in xenograft models, we conducted unsupervised clustering of proteomes and phosphoproteomes of the 24 xenograft samples based upon the top 436 variably\",\n",
            "          \"of the 24 xenograft samples based upon the top 436 variably expressed proteins showing a s.d. greater than 2 from the iTRAQ proteome (Supplementary Data 7). Two distinct clusters emerged: one contained all basal tumours and the only CLDN-low tumour\",\n",
            "          \"one contained all basal tumours and the only CLDN-low tumour (WHIM12), while the other included all luminal B and HER2-E breast tumours; HER2-E tumours did not show a proteomic expression profile distinct from luminal B samples (Fig. 3b). Clustering\",\n",
            "          \"profile distinct from luminal B samples (Fig. 3b). Clustering analysis using the same gene markers from the transcriptome (Supplementary Fig. 7a) and the LFQ proteome (Supplementary Fig. 7b; Supplementary Data 8) further supported the separation into\",\n",
            "          \"7b; Supplementary Data 8) further supported the separation into these two proteomic subtypes, although the minor differences between the transcriptome and the proteome clustering suggested distinctions between mRNA and protein levels. The proteomic\",\n",
            "          \"distinctions between mRNA and protein levels. The proteomic subtypes defined by the top 968 most variably expressed mouse host proteins did not segregate based on luminal and basal subtypes (Supplementary Fig. 8; Supplementary Data 9). Besides differing\",\n",
            "          \"(Supplementary Fig. 8; Supplementary Data 9). Besides differing in mRNA expression, luminal and basal breast cancer PDXs also showed consistently distinct proteomic expression profiles, supporting their distinct biological origins. We then utilized the\",\n",
            "          \"their distinct biological origins. We then utilized the iTRAQ phosphopeptide expression data to infer phosphoproteomic subtypes. 1,737 unique phosphosites with s.d. greater than 2.5 were included to conduct hierarchical clustering (Fig. 3c; Supplementary\",\n",
            "          \"to conduct hierarchical clustering (Fig. 3c; Supplementary Data 10). These analyses of the phosphoproteomic data produced two major clusters segregating the luminal B and basal subtypes. Again, the WHIM37/WHIM47/WHIM26 group and the WHIM17/WHIM46 group,\",\n",
            "          \"the WHIM37/WHIM47/WHIM26 group and the WHIM17/WHIM46 group, as observed in the proteome clustering, grouped closely together. Gene and protein expression of lymphoid lineage markers showed high expression of CD20 and JAK3 in WHIM17 and WHIM46 (Methods,\",\n",
            "          \"high expression of CD20 and JAK3 in WHIM17 and WHIM46 (Methods, Supplementary Fig. 9), consistent with their positive EBV status and histological diagnosis (Supplementary Fig. 1) as human lymphoproliferative cells arising in an immunocompromised mouse\",\n",
            "          \"lymphoproliferative cells arising in an immunocompromised mouse background. Overall, while the basal and luminal clusters remain consistent, the hierarchical distances between PDX samples within the two major clusters differ between data types. The\",\n",
            "          \"within the two major clusters differ between data types. The departure of proteomic and phosphoproteome subtypes from mRNA expression-defined subtypes suggests independent layers of molecular heterogeneity provided by distinct proteomic analyses. We then\",\n",
            "          \"heterogeneity provided by distinct proteomic analyses. We then investigated whether human proteomic subtypes could be recapitulated in PDX models by including an additional 77 proteomes from TCGA human tumours that were processed concurrently in the iTRAQ\",\n",
            "          \"human tumours that were processed concurrently in the iTRAQ experiment (Methods). To reduce clustering bias imbued by the mouse contribution to the proteome in the PDX samples, we excluded proteins showing differential expression between human tumour and\",\n",
            "          \"showing differential expression between human tumour and xenograft (t-test, FDR\\u22640.3) (Supplementary Data 11). Additional requirements, including the presence of detectable protein in more than 10 samples and minimum difference of 2 s.d.'s in the merged\",\n",
            "          \"10 samples and minimum difference of 2 s.d.'s in the merged proteome, resulted in 133 proteins qualifying for un-supervised clustering. Consistent with the subtyping analysis of the 24 PDXs alone, we identified two major clusters: one that included all\",\n",
            "          \"alone, we identified two major clusters: one that included all but one basal breast tumour and the other comprised mostly of luminal tumours (Fig. 3d). Similar to the proteomic subtypes of the PDX cohorts, luminal tumours and HER-2 tumours did not show\",\n",
            "          \"the PDX cohorts, luminal tumours and HER-2 tumours did not show clear separation, although several sub-clusters were identified. Importantly, xenograft proteomes clustered adjacently to the human proteomes of their respective subtypes, validating the\",\n",
            "          \"human proteomes of their respective subtypes, validating the fidelity of basal and luminal proteomic signatures discovered in PDXs. To search for defining markers between the basal and luminal B subtypes, we conducted differential expression analysis\",\n",
            "          \"B subtypes, we conducted differential expression analysis between the PDXs of the respective subtypes. We found several proteomic markers that were differentially expressed in both the LFQ and iTRAQ datasets (t-test, FDR\\u22640.05), including SPR, GSTP1 and\",\n",
            "          \"and iTRAQ datasets (t-test, FDR\\u22640.05), including SPR, GSTP1 and SERPINB5 (Supplementary Fig. 10; Supplementary Data 12). We then conducted gene-set enrichment analysis based on the Reactome pathway database to investigate patterns of differential\",\n",
            "          \"pathway database to investigate patterns of differential expression (Methods). The basal subtype breast tumours were up-regulated in most of the significantly differentially-expressed pathways (FDR\\u22640.01) identified by both LFQ and iTRAQ datasets,\",\n",
            "          \"pathways (FDR\\u22640.01) identified by both LFQ and iTRAQ datasets, including extracellular matrix organization, cell cycle, and collagen formation. In comparison, the luminal B breast tumours showed higher expression in genes related to organelle biogenesis\",\n",
            "          \"higher expression in genes related to organelle biogenesis and membrane trafficking (Supplementary Data 13). Activated pathways revealed by phosphorylation profiles Cancer driving somatic events trigger major changes in downstream signalling to launch the\",\n",
            "          \"trigger major changes in downstream signalling to launch the tumorigenic cascade. To search for tumour-specific activated pathways in PDX tumours, we systematically evaluated and compared phosphoproteome profiles of gene sets from KEGG signalling pathways\",\n",
            "          \"profiles of gene sets from KEGG signalling pathways (Methods). Phosphorylation enrichment analysis identified 12 significantly activated pathways, including Ras, MAPK and NF\\u03baB signalling, in 4 xenografts (FDR\\u22640.01, Supplementary Data 14). WHIM9 exhibited\",\n",
            "          \"4 xenografts (FDR\\u22640.01, Supplementary Data 14). WHIM9 exhibited elevated phosphorylation of the MAPK signalling pathway (FDR=9.69e-6; Fig. 4a). Interestingly, WHIM9 carried a recurrent somatic mutation, KRAS p.A146V (refs), which may have driven canonical\",\n",
            "          \"mutation, KRAS p.A146V (refs), which may have driven canonical MAPK pathway activation. Further, WHIM12 exhibited activation of the Ras signalling pathway (FDR=4.28e-5), along with an outlier protein expression of MET, a receptor tyrosine kinase upstream\",\n",
            "          \"protein expression of MET, a receptor tyrosine kinase upstream of the Ras signalling pathway (Fig. 4b). Interestingly, WHIM17 and WHIM46, both harboured BTK and PLCG2 protein overexpression, exhibited overall high phosphorylation of the NF\\u03baB signalling\",\n",
            "          \"exhibited overall high phosphorylation of the NF\\u03baB signalling pathway (Fig. 4c,d; FDR=6.94e-3, 4.53e-5 respectively). This observation further supports the strong similarity between these two PDX models based on protein/phosphoprotein clustering (Fig.\",\n",
            "          \"two PDX models based on protein/phosphoprotein clustering (Fig. 3b,c) and their classification as EBV-positive lymphoproliferation. While pathways such as PI3K/AKT/MTOR are known to be activated across the majority of breast tumours, our analysis shows\",\n",
            "          \"across the majority of breast tumours, our analysis shows that other complementary tumorigenesis-related pathways including RAS/MAPK are also activated in a small set of breast tumours due to specific genomic or proteomic alterations, representing\",\n",
            "          \"due to specific genomic or proteomic alterations, representing alternative treatment opportunities. In addition, our phospho-proteomic analysis revealed activation of signalling pathways not readily predicted by genomic data. Complementary\",\n",
            "          \"pathways not readily predicted by genomic data. Complementary genomic/proteomic druggable targets We examined promising drug targets in each tumour by surveying the genome and expressed proteomes. Specifically, we compiled a list 76 druggable genes\",\n",
            "          \"proteomes. Specifically, we compiled a list 76 druggable genes (Supplementary Data 15), along with their respective drugs, from established public databases (Methods). Six PDXs, representing 20.8% of the tumours in this study, harboured druggable somatic\",\n",
            "          \"20.8% of the tumours in this study, harboured druggable somatic mutations, including PIK3CA p.H1047R and KRAS p.A146V in WHIM9, PIK3CA p.H1047R in WHIM16 and WHIM24, PIK3CA p.E545K in WHIM18, PIK3CA p.E542K in WHIM20 and SF3B1 p.K700E in WHIM26\",\n",
            "          \"in WHIM18, PIK3CA p.E542K in WHIM20 and SF3B1 p.K700E in WHIM26 (Supplementary Data 16). While activating mutations in oncogenes can be targeted by treatments, aberrantly overexpressed or activated protein products, such as HER2, also presents exploitable\",\n",
            "          \"protein products, such as HER2, also presents exploitable treatment opportunities. We sought genomic and proteomic evidence of overexpressed genes/proteins or proteins with highly phosphorylated sites. We defined outliers as expression values exceeding\",\n",
            "          \"sites. We defined outliers as expression values exceeding the 1.5 interquartile ranges (IQR) above the third quartile of the cohort (Variations of Box Plots), and rank ordered them by the outlier score. We further required CNV outliers to be validated\",\n",
            "          \"outlier score. We further required CNV outliers to be validated with outlier expression in either the mRNA or protein level to rule out up-regulation events due to technical artifacts or passenger events (Methods). mRNA and protein expression outlier\",\n",
            "          \"passenger events (Methods). mRNA and protein expression outlier scores showed a moderate positive correlation (Fig. 5a, R=0.516), but mRNA outlier expression did not guarantee high protein expression (for example, AKT2 and FGFR2, Fig. 5a). Similarly, we\",\n",
            "          \"(for example, AKT2 and FGFR2, Fig. 5a). Similarly, we observed a fraction of phosphosite outliers not detected at the protein level (Fig. 5b, R=0.548). Consequently, identifying post-transcriptional and post-translational events to capture potential\",\n",
            "          \"and post-translational events to capture potential druggable treatment opportunities requires consideration of protein expression as well as gene expression. Applying this druggable outlier detection strategy across CNV, mRNA, protein and protein\",\n",
            "          \"detection strategy across CNV, mRNA, protein and protein phosphorylation levels, we identified overexpressed druggable genes in 26.1% and 47.8% of PDXs at the CNV and mRNA levels, respectively (Fig. 6; Supplementary Data 16). These events recapitulated\",\n",
            "          \"(Fig. 6; Supplementary Data 16). These events recapitulated known druggable opportunities, such as the PIK3CA copy-number amplification in WHIM4 and ERBB2 (HER2) copy-number amplification in WHIM35. Expanding to iTRAQ protein expression outliers allowed\",\n",
            "          \"WHIM35. Expanding to iTRAQ protein expression outliers allowed us to uncover druggable targets in 19 out of the 24 PDXs (79.2%), while considering phosphosite outliers covered 22 of the 24 PDXs (91.7%). A significantly high fraction of protein outliers\",\n",
            "          \"PDXs (91.7%). A significantly high fraction of protein outliers overlapped between the LFQ and iTRAQ datasets (Fisher's Exact Test, P=2.013e-05, Supplementary Fig. 11), providing validation of our findings. The identified proteomic outliers included\",\n",
            "          \"of our findings. The identified proteomic outliers included multiple known druggable targets involved in the PI3K, RTK, MAPK signalling pathways and other oncogenic processes (Fig. 6; Supplementary Data 16). For instance, HER2 was found to be the top\",\n",
            "          \"Data 16). For instance, HER2 was found to be the top outlier at the protein expression level in 2 HER2-E xenografts, WHIM8 and WHIM35; the phosphosites of HER2 were also identified as outliers. Immunohistochemistry experiments validated the high HER2\",\n",
            "          \"Immunohistochemistry experiments validated the high HER2 protein expression on the cell membranes of WHIM8 and WHIM35 (Fig. 5c; Supplementary Fig. 12). Further, AKT1, AKT2 and AKT3 proteins were outlier expression candidates in WHIM35, WHIM20 and WHIM43\",\n",
            "          \"were outlier expression candidates in WHIM35, WHIM20 and WHIM43 and WHIM4, respectively, providing proteomic validation of previously observed AKT up-regulation at mRNA expression level in breast cancer. Other outlier proteins included IDH1 in WHIM9 and\",\n",
            "          \"cancer. Other outlier proteins included IDH1 in WHIM9 and WHIM30, FGFR4 in WHIM11 and WHIM26, and both BTK and JAK3 in the EBV-positive WHIM17 and WHIM46. Many of these protein outliers aligned with up-regulation in their corresponding phosphosites,\",\n",
            "          \"aligned with up-regulation in their corresponding phosphosites, including phosphosites on AKT1, AKT3, BRAF, FGFR4 and HSP90AB1 (Fig. 5c, selected phosphosite spectra in Supplementary Fig. 13). We also discovered additional outlier phosphosites in other\",\n",
            "          \"We also discovered additional outlier phosphosites in other proteins including CTNNB1, ARAF, and HSP90B1 (Supplementary Data 16). Finally, we identified outlier FGFR2 in WHIM16 and high RAF1 in WHIM9 and further validated their high protein expression\",\n",
            "          \"in WHIM9 and further validated their high protein expression status of by immunohistochemistry analyses (Fig. 5c; Supplementary Fig. 12). As diverse sets of proteomic outliers are identified across PDXs, effectively inhibiting such activation events will\",\n",
            "          \"across PDXs, effectively inhibiting such activation events will be required when designing targeted treatment strategies to each individual breast tumour. Notably, for 80.6% of proteomic outlier events, we were able to identify human tumours from the 77\",\n",
            "          \"events, we were able to identify human tumours from the 77 TCGA samples showing the same outlier protein expression through outlier score or ranking (Methods, Fig. 6b, Supplementary Data 16). For example, we observed outlier HER2 expression in five HER2-E\",\n",
            "          \"For example, we observed outlier HER2 expression in five HER2-E and four luminal B human breast tumours. Further, a basal human sample carried both the outlier FGFR2 and FGFR4 expression, validating the findings in WHIM16, WHIM26 and WHIM11. Basal human\",\n",
            "          \"the findings in WHIM16, WHIM26 and WHIM11. Basal human breast tumours also carried protein expression outliers in IDH1, EGFR and MAP2K1 (Fig. 6b). Phosphosite outlier expression events showed a moderate rate of validation in the same human cohort (48.8%,\",\n",
            "          \"a moderate rate of validation in the same human cohort (48.8%, Fig. 6c), suggesting its transient nature and potential micro-environmental effects on protein phosphorylation. As expected, HER2-E and a few HER2-positive luminal B tumours carried outliers\",\n",
            "          \"and a few HER2-positive luminal B tumours carried outliers in HER2 phosphosites including p.T701, p.T1240 and p.Y1248. ARAF phosphosite outliers, such as p.S299, were found in both human and PDX samples, validating our previous finding in the human\",\n",
            "          \"and PDX samples, validating our previous finding in the human cohort. Interestingly, we identified outlier phosphosites in genes not previously implicated in breast cancer through genomic profiles, such as BRAF p.S447, p.S750 and HSP90AB1 p.Y56 and p.S169\",\n",
            "          \"such as BRAF p.S447, p.S750 and HSP90AB1 p.Y56 and p.S169 (Fig. 6c). Our results demonstrated that proteomic outlier events, like genomic driver mutations, are consistently observed in PDXs and human tumours. Some protein outlier events might represent\",\n",
            "          \"and human tumours. Some protein outlier events might represent \\u2018proteomic drivers' of tumorigenesis and therefore potential drug targets in breast tumours. Targeted treatments using breast cancer xenograft models To validate the identified druggable\",\n",
            "          \"cancer xenograft models To validate the identified druggable events, we conducted treatment experiments targeting HER2 and PI3K pathways on selected PDX models. Four PDX models were chosen to address HER2 targeting using lapatinib, an oral HER2 kinase\",\n",
            "          \"to address HER2 targeting using lapatinib, an oral HER2 kinase inhibitor. These include 2 HER2-E PDX models WHIM8 and WHIM35, both with high HER2 protein and phosphoprotein expression, and 2 low HER2 expressing, basal-like PDX models WHIM6 and WHIM14.\",\n",
            "          \"2 low HER2 expressing, basal-like PDX models WHIM6 and WHIM14. Western blotting suggested high HER2 expression and phosphorylation of HER2 p.Y1248 in both HER2 positive tumours, WHIM8 and WHIM35 (Fig. 7a; Supplementary Fig. 14). Unexpectedly, high levels\",\n",
            "          \"(Fig. 7a; Supplementary Fig. 14). Unexpectedly, high levels of HER2 p.Y1248 phosphorylation was also detected in WHIM14 by western blotting. In contrast MS-based phosphoproteomics also detected high levels of HER2p.Y1248 in WHIM8 and WHIM35, but not in\",\n",
            "          \"high levels of HER2p.Y1248 in WHIM8 and WHIM35, but not in WHIM14, which was more consistent with the known biology of the models. Antibody-based diagnosis of HER2 activation in WHIM14 was possibly due to cross-reaction of the Y1248 antibody with the\",\n",
            "          \"possibly due to cross-reaction of the Y1248 antibody with the pY1172 site of EGFR that bears high sequence similarity around the pY residue (..GTPTAENPEy1248LGLDVPV-CO2H vs...GSTAENAEy1172LRVAPQ.., Supplementary Fig. 6). As expected, WHIM8 and WHIM35 were\",\n",
            "          \"Supplementary Fig. 6). As expected, WHIM8 and WHIM35 were growth inhibited (Wilcoxon Rank Sum Test, P=2.7e-5, 4.8e-7) by lapatinib, whereas WHIM6 was not (P=0.65) (Fig. 7b). Interestingly, WHIM14 also showed significant reduction in tumour growth by\",\n",
            "          \"WHIM14 also showed significant reduction in tumour growth by lapatinib (P=4.3e-3). Upon further exploration with a lower, clinically achievable dose of lapatinib for chronic treatment (30\\u2009mg\\u2009kg\\u22121 compared with 220\\u2009mg\\u2009kg\\u22121 in the previous experiment) over\",\n",
            "          \"compared with 220\\u2009mg\\u2009kg\\u22121 in the previous experiment) over 48 days, significant but weak tumour growth inhibition was again achieved (P=0.0311, Supplementary Fig. 15). However, WHIM14 did not respond to HER2/HER3 antibodies trastuzumab or pertuzumab\",\n",
            "          \"not respond to HER2/HER3 antibodies trastuzumab or pertuzumab (P=0.250 and 0.181, respectively; Supplementary Fig. 15). Thus, the response of WHIM14 to lapatinib was likely due to inhibition of EGFR not HER2 (refs). While WHIM6 also showed elevated EGFR\",\n",
            "          \"of EGFR not HER2 (refs). While WHIM6 also showed elevated EGFR protein level, WHIM14 showed notably higher EGFR phosphorylation based on both mass spectrometry (Fig. 2b) and western blotting (Fig. 7a), which could account for their different response to\",\n",
            "          \"(Fig. 7a), which could account for their different response to lapatinib. Two basal and one HER2-E human breast cancers harboured outlier EGFR expression (Fig. 6b), suggesting EGFR remains a potential therapeutic target in a subset of breast cancers that\",\n",
            "          \"potential therapeutic target in a subset of breast cancers that has yet to be fully realized clinically. The PI3K/AKT/mTOR signalling pathway is altered in approximately 77% of breast tumours and multiple drugs targeting its components, including Class I\",\n",
            "          \"and multiple drugs targeting its components, including Class I PI3Ks, AKTs, and mTORs, are already in clinical trials. Among these, a combination of everolimus and exemestane has been approved for treating advanced ER-positive breast cancers resistant to\",\n",
            "          \"for treating advanced ER-positive breast cancers resistant to non-steroidal aromatase inhibitors. Promising activity has also been reported for direct inhibitors of PI3K. However, mutations in PIK3CA and other genetic alterations at the genomic level have\",\n",
            "          \"PIK3CA and other genetic alterations at the genomic level have so far failed to closely predict therapeutic responsiveness to PI3K pathway inhibitors. We therefore hypothesized that combined genomic and proteomic indication of PI3K signalling activation\",\n",
            "          \"genomic and proteomic indication of PI3K signalling activation is necessary for the prediction of treatment response. Our integrated approach identified 6 xenografts that harboured complementary genomic and proteomic druggable events in the PI3K-AKT\",\n",
            "          \"genomic and proteomic druggable events in the PI3K-AKT pathway. In particular, WHIM16 harboured a hotspot PIK3CA p.H1047R mutation, whereas WHIM18 and WHIM20 each carried a hotspot PIK3CA p.E545K mutation. WHIM20 also showed an additional outlier protein\",\n",
            "          \"mutation. WHIM20 also showed an additional outlier protein expression of AKT2 and WHIM18 also expressed AKT2 at a near outlier level that may combine to activate PI3K pathway signalling (Fig. 6). Since the treatments were applied to later-passage PDX\",\n",
            "          \"6). Since the treatments were applied to later-passage PDX models relative to the ones we conducted proteogenomic analysis on, we performed immunohistochemistry to validate the phosphorylation of AKT p.S473. WHIM18 and WHIM20 showed detectable AKT p.S473,\",\n",
            "          \"of AKT p.S473. WHIM18 and WHIM20 showed detectable AKT p.S473, which was not observed in WHIM16 (Fig. 7c). We conducted combinatorial treatment experiments by applying an alpha specific PI3K inhibitor (TAK-117) and/or an mTORC1/2 inhibitor (TAK-228) to\",\n",
            "          \"inhibitor (TAK-117) and/or an mTORC1/2 inhibitor (TAK-228) to three PDX models of luminal B breast cancer. Consistent with previous reports, PIK3CA mutation status alone did not accurately predict outcome; WHIM18 and WHIM20 showed reduced tumour growth\",\n",
            "          \"predict outcome; WHIM18 and WHIM20 showed reduced tumour growth upon application of the inhibitors, whereas WHIM16 did not (Fig. 7d). mTOR inhibition repressed tumour growth in WHIM18 and WHIM20 (ANOVA followed by Tukey's post hoc test, P=9.5e-10, 1.6e-03\",\n",
            "          \"(ANOVA followed by Tukey's post hoc test, P=9.5e-10, 1.6e-03 respectively), but not in WHIM16 (P=0.97), showing that inhibition of mTORC1/2 may effectively suppress breast tumours with activated AKTs and validating a previous study showing the efficacy of\",\n",
            "          \"AKTs and validating a previous study showing the efficacy of mTOR inhibition in PDX models of triple-negative breast cancer. Importantly, the combinatorial treatment achieved the greatest effect in WHIM18 and WHIM20 (P<2.2e-16 for both comparisons). While\",\n",
            "          \"in WHIM18 and WHIM20 (P<2.2e-16 for both comparisons). While neither PI3K nor mTOR inhibitor drug treatment alone suppressed tumour growth in WHIM20 completely, the combination of both PI3K and mTOR inhibitors significantly reduced tumour growth to a\",\n",
            "          \"and mTOR inhibitors significantly reduced tumour growth to a nearly static state (Fig. 7d). Based on our proteomic characterization, WHIM20 exhibited the strongest AKT1 and AKT2 protein expression, as well as AKT1 p.S122 and p.S475 phosphorylation\",\n",
            "          \"expression, as well as AKT1 p.S122 and p.S475 phosphorylation signatures followed by WHIM18 and then WHIM16 (Fig. 6b,c). Further, both PI3K and mTOR inhibitors significantly reduced AKT p.S473 (Fig. 7c). Our treatment results showed that the magnitude of\",\n",
            "          \"(Fig. 7c). Our treatment results showed that the magnitude of drug response may be associated with overexpression and phosphorylation of the downstream signalling targets such as AKT proteins. In addition to this validation of druggable hypotheses in\",\n",
            "          \"In addition to this validation of druggable hypotheses in luminal tumours, our previous report also demonstrated the effectiveness of combinatorial therapy of AKT and mTOR inhibitors in two other basal breast cancer xenografts. One of the treated\",\n",
            "          \"in two other basal breast cancer xenografts. One of the treated xenografts, WHIM4, was also characterized in this study, and showed copy-number amplification of PIK3CA and AKT3 protein outlier expression (Fig. 6). Overall, proteogenomic analysis revealed\",\n",
            "          \"expression (Fig. 6). Overall, proteogenomic analysis revealed that the dual activation of PIK3CA at the genomic level and AKTs at the protein level may be a common signature of breast tumours, affecting more than 20% of PDXs in this cohort. Importantly,\",\n",
            "          \"affecting more than 20% of PDXs in this cohort. Importantly, our results demonstrate the potential utility of combinatorial inhibitor treatments to treat breast tumours showing these proteogenomic signatures. Discussion Breast cancer has been\",\n",
            "          \"proteogenomic signatures. Discussion Breast cancer has been traditionally characterized in the clinic through hormone receptor status and selected genes' expressions, and more recently by genomic sequencing. However, druggable genomic driver events are\",\n",
            "          \"sequencing. However, druggable genomic driver events are detectable in only a limited percentage of patients. As the majority of drugs target proteins, a systematic evaluation of breast cancer proteomes would seem ultimately to be necessary for selecting\",\n",
            "          \"proteomes would seem ultimately to be necessary for selecting targeted treatment and predicting drug response. Recent advances in MS-based proteomics allow extensive and quantitative surveys of the global proteome. Here, we have systematically analyzed\",\n",
            "          \"of the global proteome. Here, we have systematically analyzed proteogenomic profiles of 22 patient-derived breast cancer xenografts and 2 EBV-positive lymphoproliferations that are likely artifacts of engraftment of human lymphocytes into NSG mice. This\",\n",
            "          \"of engraftment of human lymphocytes into NSG mice. This study shows that proteogenomic signatures of PDXs resemble most findings from breast cancer patients. While some discrepancies exist, we established a normalization strategy at both the genomic and\",\n",
            "          \"we established a normalization strategy at both the genomic and proteomic levels that enabled direct comparison. PDX tumours recapitulated the proteomic diversity of human breast cancers (Fig. 3). We also identified multiple druggable targets for each\",\n",
            "          \"3). We also identified multiple druggable targets for each tumour model (Fig. 6; Supplementary Data 17). Proteomic events validated a significant number of CNV or mRNA up-regulations. For example, HER2 protein and phosphosite outlier expression was\",\n",
            "          \"example, HER2 protein and phosphosite outlier expression was observed in HER2-E WHIM8 and WHIM35 (Fig. 5), which were effectively treated using lapatinib (Fig. 7b). Interestingly, we also identified overexpressed proteomic events not evident in the\",\n",
            "          \"identified overexpressed proteomic events not evident in the genomic level in both PDX and human samples, including outlier protein expression of EGFR, and outlier phosphosite expressions of ARAF, BRAF, HSP90AB1, PTPN11 and TOP2A (Fig. 6), highlighting\",\n",
            "          \"ARAF, BRAF, HSP90AB1, PTPN11 and TOP2A (Fig. 6), highlighting potential new treatment opportunities in breast cancer. In the two PDX models subsequently diagnosed as EBV-positive lymphoproliferations, we observed outlier BTK expression (Fig. 6) and\",\n",
            "          \"we observed outlier BTK expression (Fig. 6) and activation of the NF\\u03baB pathway (Fig. 4), validating BTK as a druggable target in EBV+ lymphomas. While more than 80% of the proteomic outlier events in PDX were also found in human tumours, a lessor 48.8% of\",\n",
            "          \"in PDX were also found in human tumours, a lessor 48.8% of phosphosite outlier events were validated, potentially due to different tumour micro-environments. Thus, transient phosphoproteomic events identified in PDX tumours would likely require further\",\n",
            "          \"events identified in PDX tumours would likely require further verification in their corresponding primary tumours. Outlier protein expression events can likely lead to downstream pathway activation, such as MET outlier protein expression (Fig. 6) and\",\n",
            "          \"activation, such as MET outlier protein expression (Fig. 6) and activated Ras pathway observed in WHIM12 (Fig. 4). While genomic analysis has utilized mutual occurrence or exclusivity in patient cohorts to deduce pathway relationships, phosphorylation\",\n",
            "          \"cohorts to deduce pathway relationships, phosphorylation profile analyses allowed us to directly interrogate signalling in these established pathways in a single sample. Our pathway activation results suggest these events may be crucial to tumorigenesis\",\n",
            "          \"results suggest these events may be crucial to tumorigenesis and that some are likely the proteomic \\u2018smoking guns' that originally triggered the oncogeneic cascade. Roughly 77% of breast tumours showed alterations in the PI3K-AKT signalling pathway,\",\n",
            "          \"tumours showed alterations in the PI3K-AKT signalling pathway, representing a potentially important path forward for drug-based treatment. Yet, genomic alterations of PI3K pathway components have not shown themselves to predict treatment responsiveness to\",\n",
            "          \"not shown themselves to predict treatment responsiveness to PI3K pathway inhibitors. In this study, we defined complementary druggable targets of the pathway using proteogenomic analysis, including several events of a co-occurring PIK3CA mutation or\",\n",
            "          \"including several events of a co-occurring PIK3CA mutation or copy-number amplification, and AKT protein outlier expression coupled by elevated AKT phosphorylation. In two such breast tumours, we successfully inhibited tumour growth using a combination of\",\n",
            "          \"we successfully inhibited tumour growth using a combination of PI3K and MTOR inhibitors (Fig. 7c). While these results show potential functional implications, additional, systematic treatment experiments are required to validate the identified proteomic\",\n",
            "          \"experiments are required to validate the identified proteomic druggable targets. In conclusion, this initial work using proteogenomic integration coupled with patient-derived xenograft validation, has demonstrated a strategy that, in principle, may enable\",\n",
            "          \"has demonstrated a strategy that, in principle, may enable more accurate prediction of the efficacy of mechanism-based cancer therapeutics. Methods Xenograft model generation Patient-derived xenografts were generated from primary or metastatic breast\",\n",
            "          \"xenografts were generated from primary or metastatic breast tumours using previously described procedures. All human tissues for these experiments were processed in compliance with NIH regulations and institutional guidelines, and approved by the\",\n",
            "          \"regulations and institutional guidelines, and approved by the institutional review board at Washington University. All animal procedures were reviewed and approved by the institutional animal care and use committee at Washington University in St. Louis.\",\n",
            "          \"care and use committee at Washington University in St. Louis. PDX models are available through the application to the Human and Mouse-Linked Evaluation of Tumors core at http://digitalcommons.wustl.edu/hamlet/. We selected 24 of the established breast\",\n",
            "          \"We selected 24 of the established breast tumour samples, including 9 luminal B, 10 basal, 4 HER2-E, and 1 CLDN-low breast tumours for further proteogenomic characterization. Receptor statuses of xenograft tumours were validated using IHC after\",\n",
            "          \"statuses of xenograft tumours were validated using IHC after engraftment. Immunohistochemistry Xenografts were formalin-fixed at least for 24\\u2009h and paraffin-embedded. Sections were evaluated by hematoxylin & eosin (H&E) staining. Immunohistochemistry\",\n",
            "          \"by hematoxylin & eosin (H&E) staining. Immunohistochemistry (IHC) was performed on additional sections for HER2 (Dako), Phospho-Akt (Ser473) (Cell signalling), FGFR2 (Abcam), and Raf-1 (Santa Cruz) following the manufacturer's instructions. Western\",\n",
            "          \"(Santa Cruz) following the manufacturer's instructions. Western blotting of phosphorylated HER2 (p.Y1248) was performed using antibody cat Nr-06-229 (Millipore). In vivo drug treatment experiments For the targeted treatment of the PI3K pathway, PI3K alpha\",\n",
            "          \"For the targeted treatment of the PI3K pathway, PI3K alpha inhibitor TAK-117 and TORC1/2 inhibitor TAK-228 were provided by Millennium Pharmaceuticals, Inc. The compounds were dissolved in Peg400. Tumours were engrafted in NSG (NOD.Cg-Prkdcscid\",\n",
            "          \"in Peg400. Tumours were engrafted in NSG (NOD.Cg-Prkdcscid Il2rgtm1Wjl/SzJ) mice (The Jackson Laboratory) by subcutaneous injection of 2\\u20135 \\u00d7 106 PDX cells in PBS supplemented with 30% Matrigel (BD Biosciences, cat. no. 354234). When tumours reached an\",\n",
            "          \"(BD Biosciences, cat. no. 354234). When tumours reached an average size of 250\\u2013300\\u2009mm3, animals were assigned randomly to control and various treatment groups (n=8\\u20139 each group). For treatment of WHIM16 (passage 7), 18 (passage 8), and 20 (passage 5) were\",\n",
            "          \"of WHIM16 (passage 7), 18 (passage 8), and 20 (passage 5) were used. Tumour bearing mice were gavaged with: (1) Peg400; (2) TAK-117, 140\\u2009mg\\u2009kg\\u22121\\u2009day\\u22121; (3) TAK-228, 1\\u2009mg\\u2009kg\\u22121\\u2009day\\u22121; (4) TAK-117, 140\\u2009mg\\u2009kg\\u22121\\u2009day\\u22121, TAK-228, 1\\u2009mg\\u2009kg\\u22121\\u2009day\\u22121. The mice were\",\n",
            "          \"140\\u2009mg\\u2009kg\\u22121\\u2009day\\u22121, TAK-228, 1\\u2009mg\\u2009kg\\u22121\\u2009day\\u22121. The mice were treated on three consecutive days once daily and then had a 4-day interval. Tumours were measured with external caliper, and volume was calculated as (4\\u03c0/3) \\u00d7 (width/2)2 \\u00d7 (length/2). For the\",\n",
            "          \"was calculated as (4\\u03c0/3) \\u00d7 (width/2)2 \\u00d7 (length/2). For the lapatinib therapeutic experiments of WHIM6 (passage 7), WHIM 14 (passage 11), WHIM 8 (passage 6) and WHIM 35 (passage 6), 1 \\u00d7 106 tumour cells were added to equal volume of 1:1 mixture of\",\n",
            "          \"\\u00d7 106 tumour cells were added to equal volume of 1:1 mixture of Matrigel and 10% RPMI plus 10% FBS to 4th mammary fat pads in female SCID/bg mice (ENVIGO). We then established tumours to an average volume 250\\u2013300\\u2009mm3, and randomized the mice into control\",\n",
            "          \"volume 250\\u2013300\\u2009mm3, and randomized the mice into control and lapatinib groups. We treated the treated group mice with lapatinib chow diet by formulating lapatinib (220\\u2009mg\\u2009kg\\u22121) and processing them into food pellets (by Research Diets Inc), which is\",\n",
            "          \"them into food pellets (by Research Diets Inc), which is supplied for 2 weeks. The numbers of replicates for each group are as followed: WHIM 14 control (n=6) and lapatinib treated (n=5); WHIM 6 control (n=11) and lapatinib treated (n=12); WHIM 8 control\",\n",
            "          \"6 control (n=11) and lapatinib treated (n=12); WHIM 8 control (n=16) and lapatinib treated (n=6); WHIM 35 control (n=17) and lapatinib treated (n=18). We also treated WHIM14 with 100\\u2009mg\\u2009kg\\u22121 lapatinib treatment for 48 days, including both control (n=8)\",\n",
            "          \"lapatinib treatment for 48 days, including both control (n=8) and lapatinib treated (n=9) groups. Further, we tested the effect of trastuzumab (30\\u2009mg\\u2009kg\\u22121 weekly with IP injections) and pertuzumab (30\\u2009mg\\u2009kg\\u22121 weekly with IP injections) on PDX tumour\",\n",
            "          \"pertuzumab (30\\u2009mg\\u2009kg\\u22121 weekly with IP injections) on PDX tumour growth for 48 days. The experimental groups are as following: control group (n=6) treated with physiological saline (vehicle); trastuzumab treated group (n=12); pertuzumab treated group\",\n",
            "          \"trastuzumab treated group (n=12); pertuzumab treated group (n=11). Statistical testing of the resulting data was conducted using the R programming language. Wilcoxon rank sum test was applied to compare the fold change in tumour volumes after two weeks in\",\n",
            "          \"to compare the fold change in tumour volumes after two weeks in the lapatinib treatment experiment, and after 44 days or 48 days for the additional WHIM14 experiment using low-dose lapatinib and trastuzumab/pertuzumab. One-way ANOVA was applied to compare\",\n",
            "          \"trastuzumab/pertuzumab. One-way ANOVA was applied to compare the treated versus control groups in the PI3K targeted-therapy experiments, and a follow-up Tukey's post hoc test was used for the various comparisons in PI3K inhibition experiments. All human\",\n",
            "          \"various comparisons in PI3K inhibition experiments. All human tissues for these experiments were processed in compliance with NIH regulations and institutional guidelines, and approved by the Washington University, University of North Carolina, or Baylor\",\n",
            "          \"Washington University, University of North Carolina, or Baylor College of Medicine Institutional Review Board (IRB). All animal procedures were reviewed and approved by the institutional animal care and use committee at Washington University in St. Louis,\",\n",
            "          \"care and use committee at Washington University in St. Louis, University of North Carolina, or Baylor College of Medicine. Genomic and proteomic data generation. Somatic mutation Sequencing reads were aligned using BWA. Somatic variants were identified\",\n",
            "          \"reads were aligned using BWA. Somatic variants were identified using VarScan2 (refs), GATK and Pindel, and annotated based on Ensembl release 70_37. We then filtered out common variants by using variants from the 1000 Genomes and NHLBI projects. We\",\n",
            "          \"by using variants from the 1000 Genomes and NHLBI projects. We further eliminated mouse contamination by filtering somatic variants that were mapped to mouse reference genome. Somatic mutation calls were validated either using a custom array or manually\",\n",
            "          \"calls were validated either using a custom array or manually reviewed in IGV. The genomic data of 17 out of the 24 PDXs have been presented before in our previous studies (Supplementary Data 2). Copy-number variation The segment-based copy-number data\",\n",
            "          \"2). Copy-number variation The segment-based copy-number data were generated using the whole-genome sequencing and exome sequencing data. We then converted the segment-based copy-number data to the gene-based copy-number data by using the RefSeq database\",\n",
            "          \"to the gene-based copy-number data by using the RefSeq database (version 20130727). The copy-number values were further transformed to the log-R ratio, using the cohort mean for the gene as the reference. mRNA expression and virus detection through\",\n",
            "          \"as the reference. mRNA expression and virus detection through RNA-seq mRNA expression values were calculated from mRNA sequencing data using MapSplice. The resulting RSEM values were normalized within samples to a fixed upper quartile. Upper quartile\",\n",
            "          \"within samples to a fixed upper quartile. Upper quartile normalized RSEM data were log2 transformed and the data were median centred by gene. To quantify virus abundance, we used the VirusScan pipeline (https://github.com/ding-lab/VirusScan) to detect\",\n",
            "          \"pipeline (https://github.com/ding-lab/VirusScan) to detect viruses by numbers of virus-supporting reads from RNA-seq data. LFQ proteome Tumour Sample Generation and Protein Extraction. Patient-derived xenograft breast tumours were processed to\",\n",
            "          \"Patient-derived xenograft breast tumours were processed to cryopulverized powders as described previously. The powders (100\\u2009mg wet weight) were subjected to lysis and protein extraction using a buffer composed of 8 M urea, 50\\u2009mM Tris pH 8.0, 75\\u2009mM NaCl,\",\n",
            "          \"a buffer composed of 8 M urea, 50\\u2009mM Tris pH 8.0, 75\\u2009mM NaCl, 1\\u2009mM MgCl2, and 500 units Benzonase. Approximately 1\\u2009mg of total protein extracted was reduced with DTT and subsequently alkylated with iodoacetamide. The proteins were then subjected to\",\n",
            "          \"with iodoacetamide. The proteins were then subjected to proteolysis with endoproteinase Lys-C (Wako Chemicals, USA) for \\u223c4\\u2009h at 37\\u2009\\u00b0C. The solution was diluted 4-fold with 25\\u2009mM Tris pH 8.0, 1\\u2009mM CaCl2 and further digested with trypsin (Promega) for \\u223c12\\u2009h\",\n",
            "          \"CaCl2 and further digested with trypsin (Promega) for \\u223c12\\u2009h at 37\\u2009\\u00b0C. Digestion was stopped by the addition of TFA to 0.4%, and the precipitate was removed by centrifugation. The peptide solutions were desalted on Sep-Pak Light C18 cartridges (Waters) and\",\n",
            "          \"were desalted on Sep-Pak Light C18 cartridges (Waters) and dissolved in 30% ACN, 0.1% TFA before loading on a 300\\u2009\\u03bcm Source 15\\u2009S (GE Healthcare) column for Basic Reversed Phase Chromatography (bRPLC). A linear LC gradient was performed by increasing\",\n",
            "          \"(bRPLC). A linear LC gradient was performed by increasing buffer B from 0 to 70% within 60\\u2009min, where buffer A was aqueous 10\\u2009mM ammonium formate, and buffer B was 90% ACN in 10\\u2009mM ammonium formate. A total of 30 fractions were collected for each WHIM\",\n",
            "          \"formate. A total of 30 fractions were collected for each WHIM sample (18 WHIMs). Five fractions were then prepared by combining non-contiguously fractions. We analyzed an additional technical replicate for WHIM2 and WHIM16. The fractions were dried and\",\n",
            "          \"replicate for WHIM2 and WHIM16. The fractions were dried and desalted using a stop-and-go-extraction tip (StageTip) protocol containing 4 \\u00d7 1\\u2009mm C18 extraction disk (3M). Liquid Chromatography-Tandem Mass Spectrometry and Protein Identification. Sample\",\n",
            "          \"Mass Spectrometry and Protein Identification. Sample analysis was performed via reversed phase LC-MS/MS using a Proxeon 1000 nano LC system coupled to a Q Exactive mass spectrometer (Thermo Scientific, San Jose, CA). The Proxeon system was configured to\",\n",
            "          \"Scientific, San Jose, CA). The Proxeon system was configured to trap peptides using a C18 column (3\\u2009cm \\u00d7 100\\u2009\\u03bcm i.d.) with a diverted flow rate (5\\u2009\\u03bcl\\u2009min\\u22121) The trap column was placed in line with the analytical column (15\\u2009cm \\u00d7 75\\u2009\\u03bcm i.d., 3.5\\u2009\\u03bcm, 300\\u2009\\u00c5\",\n",
            "          \"with the analytical column (15\\u2009cm \\u00d7 75\\u2009\\u03bcm i.d., 3.5\\u2009\\u03bcm, 300\\u2009\\u00c5 particle C18, Thermo Scientific, San Jose, CA, USA) before gradient elution of peptides. Analytical separation of all the tryptic peptides was achieved with a linear gradient of 2\\u201330% buffer B\",\n",
            "          \"peptides was achieved with a linear gradient of 2\\u201330% buffer B over 240\\u2009min (250\\u2009nl\\u2009min\\u22121), where buffer A was aqueous 0.1% formic acid, and buffer B was acetonitrile in 0.1% formic acid. LC-MS experiments were performed in a data-dependent mode with\",\n",
            "          \"LC-MS experiments were performed in a data-dependent mode with Full-MS (externally calibrated to a mass accuracy of <5 p.p.m., and a resolution of 70,000 at m/z=200) followed by HCD-MS/MS of the top 20 most intense ions. High-energy collision activated\",\n",
            "          \"the top 20 most intense ions. High-energy collision activated dissociation (HCD)-MS/MS was used to fragment peptides at a normalized collision energy of 27\\u2009eV in the presence of nitrogen. One LC-MS run was performed for each fraction (from 1 process\",\n",
            "          \"One LC-MS run was performed for each fraction (from 1 process technical replicate), except for WHIM2 and WHIM16 where 2 LC\\u2013MS runs were conducted (from 3 process technical replicates), resulting in the production of 100 LC\\u2013MS runs for global peptide\",\n",
            "          \"in the production of 100 LC\\u2013MS runs for global peptide analysis. Mass spectra were processed, and peptide identification was performed using the Andromeda search engine found in MaxQuant software ver. 1.5.0.25. (Max Planck Institute, Germany). All protein\",\n",
            "          \"ver. 1.5.0.25. (Max Planck Institute, Germany). All protein database searches were performed against the RefSeq database (version 20140707). Peptides were identified with a target-decoy approach using a combined database consisting of reverse protein\",\n",
            "          \"using a combined database consisting of reverse protein sequences of the RefSeq human, mouse and common repository of adventitious proteins (cRAP). The cRAP database was obtained from the Global Proteome Machine (ftp://ftp.thegpm.org/fasta/cRAP). Peptide\",\n",
            "          \"Proteome Machine (ftp://ftp.thegpm.org/fasta/cRAP). Peptide inference was made with a FDR of 1% while peptides were assigned to proteins with a protein FDR of 5%. A precursor ion mass tolerance of 20 ppm was used for the first search that allowed for m/z\",\n",
            "          \"of 20 ppm was used for the first search that allowed for m/z retention time recalibration of precursor ions that were then subjected to a main search using a precursor ion mass tolerance of 6 p.p.m. and a product ion mass tolerance 0.5\\u2009Da. Search\",\n",
            "          \"of 6 p.p.m. and a product ion mass tolerance 0.5\\u2009Da. Search parameters included up to two missed cleavages at KR on the sequence, and oxidation of methionine, and protein N terminus acetylation as a dynamic modification. Carbamidomethylation of cysteine\",\n",
            "          \"as a dynamic modification. Carbamidomethylation of cysteine residues was considered as a static modification. Peptide identifications are reported by filtering of reverse and contaminant entries and assigning to their leading razor protein according to\",\n",
            "          \"and assigning to their leading razor protein according to the Occams razor principal. The mass spectrometric data are deposited at the CPTAC Data Coordinating Center as raw and mzML files (https://cptac-data-portal.georgetown.edu). Peptide and Protein\",\n",
            "          \"(https://cptac-data-portal.georgetown.edu). Peptide and Protein Quantitation. LFQ was performed based on peak areas. The measured area under the curve of m/z and the retention time-aligned extracted ion chromatograms of peptides were performed via the\",\n",
            "          \"extracted ion chromatograms of peptides were performed via the label-free quantitation module in MaxQuant [ver. 1.5.0.25]. All replicates for each PDX were included in the LFQ experimental design with peptide-level quantitation performed using unique and\",\n",
            "          \"with peptide-level quantitation performed using unique and razor peptide features corresponding to identifications filtered with a posterior error probability (PEP) of 0.01, peptide FDR of 0.01 and protein FDR of 0.05. The expression values were median\",\n",
            "          \"0.01 and protein FDR of 0.05. The expression values were median centred in the Perseus software for further analysis [version 1.5.0.9]. iTRAQ proteome and phosphoproteome We included all 24 of the established breast tumour samples for proteomic\",\n",
            "          \"all 24 of the established breast tumour samples for proteomic characterization using iTRAQ. Tumour tissue samples were maintained in cryovials at \\u221280\\u2009\\u00b0C until cryopulverization using a CP02 Cryprep Pulverizer (Covaris, Woburn, MA). 90\\u2009mg aliquots of\",\n",
            "          \"Cryprep Pulverizer (Covaris, Woburn, MA). 90\\u2009mg aliquots of cryofractured material were prepared for proteomic processing in aluminum weighing dishes on dry ice using spatulas kept cold in liquid nitrogen, with remaining material reserved for other\",\n",
            "          \"in liquid nitrogen, with remaining material reserved for other applications. The 90\\u2009mg target was designed to include 40\\u2009mg for each of the collaborating research teams, with an anticipated yield for each team of 1.5\\u20132\\u2009mg protein based on 4\\u20135% recovery.\",\n",
            "          \"yield for each team of 1.5\\u20132\\u2009mg protein based on 4\\u20135% recovery. To avoid systematic bias, sample processing was block randomized, with each intrinsic subtype proportionally represented in each processing tranche. The reproducibility of the iTRAQ4-plex\",\n",
            "          \"each processing tranche. The reproducibility of the iTRAQ4-plex global proteome and phosphoproteome analysis workflow used in this study has been extensively tested for quantitative reproducibility both within and across laboratories in the CPTAC program.\",\n",
            "          \"both within and across laboratories in the CPTAC program. Over a period of several months 5 iTRAQ4-plex replicates were measured at each of the 3 CPTAC proteome analysis centres. Each of these iTRAQ4-plexes contained duplicate measurements for both a\",\n",
            "          \"these iTRAQ4-plexes contained duplicate measurements for both a basal WHIM2 and a luminal WHIM16 PDX samples that are also part of this study. A high degree of consistency in the number of proteins identified and correlation in the protein expression was\",\n",
            "          \"identified and correlation in the protein expression was obtained. Pearson correlations for replicate proteome and phosphoproteome measurements were very high with a R=0.9 in our previous study and very similar to the correlation observed here for the\",\n",
            "          \"study and very similar to the correlation observed here for the WHIM13 replicate measurement. These data show that our platform provides highly reproducible quantitative measurements for global proteomes and phosphoproteomes. Protein extraction, digestion\",\n",
            "          \"proteomes and phosphoproteomes. Protein extraction, digestion and iTRAQ labelling of peptides from breast cancer tumours. Cryopulverized breast cancer tumour samples tissues (\\u223c2 combined aliquots of 90\\u2009mg tissue weight each) were homogenized in 1,000\\u2009\\u03bcl\",\n",
            "          \"of 90\\u2009mg tissue weight each) were homogenized in 1,000\\u2009\\u03bcl lysis buffer containing 8M urea, 75\\u2009mM NaCl, 1\\u2009mM EDTA in 50\\u2009mM Tris HCl (pH 8), 10\\u2009mM NaF, phosphatase inhibitor cocktail 2 (1:100; Sigma, P5726) and cocktail 3 (1:100; Sigma, P0044), 2\\u2009\\u03bcg\\u2009ml\\u22121\",\n",
            "          \"Sigma, P5726) and cocktail 3 (1:100; Sigma, P0044), 2\\u2009\\u03bcg\\u2009ml\\u22121 aprotinin (Sigma, A6103), 10\\u2009\\u03bcg\\u2009ml\\u22121 Leupeptin (Roche, #11017101001), and 1\\u2009mM PMSF (Sigma, 78830). Lysates were centrifuged at 20,000 g for 10\\u2009min before measuring protein concentration of the\",\n",
            "          \"g for 10\\u2009min before measuring protein concentration of the clarified lysates by BCA assay (Pierce). Protein lysates were subsequently reduced with 5\\u2009mM dithiothreitol (Thermo Scientific, 20291) for 45\\u2009min at room temperature, and alkylated with 10\\u2009mM\",\n",
            "          \"20291) for 45\\u2009min at room temperature, and alkylated with 10\\u2009mM iodoacetamide (Sigma, A3221) for 45\\u2009min in the dark. Samples were diluted 4-fold with 50\\u2009mM Tris HCl (pH 8) prior to digesting them with LysC (Wako, 129-02541) for 4\\u2009h and trypsin (Promega,\",\n",
            "          \"them with LysC (Wako, 129-02541) for 4\\u2009h and trypsin (Promega, V511X) overnight at a 1:50 enzyme-to-protein ratio at room temperature overnight on a shaker. Digested samples were acidified with formic acid (FA; Fluka, 56302) to a final volumetric\",\n",
            "          \"with formic acid (FA; Fluka, 56302) to a final volumetric concentration of 1% or final pH of \\u223c3\\u20135, and centrifuged at 2,000\\u2009g for 5\\u2009min to clear precipitated urea from peptide lysates. Samples were desalted on C18 SepPak columns (Waters, 100\\u2009mg,\",\n",
            "          \"Samples were desalted on C18 SepPak columns (Waters, 100\\u2009mg, WAT036820) and 1\\u2009mg peptide aliquots were dried down using a SpeedVac apparatus. Construction of the Common internal Reference Pool. The proteomic and phosphoproteomic analyses of xenograft\",\n",
            "          \"Pool. The proteomic and phosphoproteomic analyses of xenograft samples were performed as iTRAQ 4-plex experiments. Quantitative comparison between all samples analyzed was facilitated by the use of iTRAQ reporter ion ratios between each individual sample\",\n",
            "          \"use of iTRAQ reporter ion ratios between each individual sample and a common internal reference sample present in each 4-plex. The reference sample was comprised of 16 of the 24 WHIM tumours analyzed in this study with equal contribution for each tumour\",\n",
            "          \"analyzed in this study with equal contribution for each tumour (WHIM numbers 2, 4, 6, 8, 11, 12, 13, 14, 16, 18, 20, 21, 24, 25, 30 and 46). The 24 tumour samples were analyzed in nine independent 4-plex experiments, with three individual samples\",\n",
            "          \"independent 4-plex experiments, with three individual samples occupying the first three channels of each experiment and the 4th channel being reserved for the reference sample. While eight iTRAQ 4-plex experiments were used to analyze the 24 individual\",\n",
            "          \"iTRAQ 4-plex experiments were used to analyze the 24 individual WHIM tumour samples, an additional 4-plex experiment was designed to include the WHIM13 sample for process replicate analysis and also internal reference samples from our human primary breast\",\n",
            "          \"also internal reference samples from our human primary breast cancer study and a taxol drug response study (unpublished) to allow cross-referencing of the different datasets. iTRAQ labelling, high pH reversed-phase separation and phosphopeptide enrichment\",\n",
            "          \"high pH reversed-phase separation and phosphopeptide enrichment of peptide samples. Desalted peptides were labelled with 4-plex iTRAQ reagents according to the manufacturer's instructions (AB Sciex, Foster City, CA). For each 1\\u2009mg peptide from each breast\",\n",
            "          \"Sciex, Foster City, CA). For each 1\\u2009mg peptide from each breast tumour sample, 10 units of labelling reagent were used. Peptides were dissolved in 300\\u2009\\u03bcl of 0.5 M triethylammonium bicarbonate (TEAB) (pH 8.5) solution and labelling reagent was added in\",\n",
            "          \"(TEAB) (pH 8.5) solution and labelling reagent was added in 700\\u2009\\u03bcl of ethanol. After 1\\u2009h incubation, 1.5\\u2009ml of 0.05% TFA was added to stop the reaction. Differentially labelled peptides were mixed and subsequently desalted on 500\\u2009mg tC18 SepPak columns.\",\n",
            "          \"mixed and subsequently desalted on 500\\u2009mg tC18 SepPak columns. The combined 4\\u2009mg iTRAQ samples per experiment were separated into 24 proteome fractions and 12 phosphoproteome fractions using a 4.6\\u2009mm \\u00d7 250\\u2009mm column RP Zorbax 300 A ExtendC18 column\",\n",
            "          \"using a 4.6\\u2009mm \\u00d7 250\\u2009mm column RP Zorbax 300 A ExtendC18 column (Agilent, 3.5\\u2009\\u03bcm bead size) on an Agilent 1100 Series HPLC instrument by basic reversed-phase chromatography as described previously. Peptides were separated according to their hydrophobicity\",\n",
            "          \"Peptides were separated according to their hydrophobicity using solvent A (2% acetonitrile, 5\\u2009mM ammonium formate, pH 10) and a nonlinear increasing concentration of solvent B (90% acetonitrile, 5\\u2009mM ammonium formate, pH 10). Phosphopeptides were enriched\",\n",
            "          \"5\\u2009mM ammonium formate, pH 10). Phosphopeptides were enriched using Ni-NTA superflow agarose beads (Qiagen, #1018611) that were stripped of nickel with 100\\u2009mM EDTA and incubated in an aqueous solution of 10\\u2009mM FeCl3 (Sigma, 451649) as described previously.\",\n",
            "          \"of 10\\u2009mM FeCl3 (Sigma, 451649) as described previously. For phosphopeptide enrichment a 80% acetonitrile/0.1% trifluoroacetic acid binding buffer and a 500\\u2009mM dibasic sodium phosphate, pH 7.0, (Sigma, S9763) elution buffer were used. Enriched samples were\",\n",
            "          \"(Sigma, S9763) elution buffer were used. Enriched samples were desalted on StageTips as described before analysis by LC\\u2013MS/MS. Analysis of tumour samples by high performance liquid chromatography tandem mass spectrometry (LC\\u2013MS/MS). All peptides were\",\n",
            "          \"tandem mass spectrometry (LC\\u2013MS/MS). All peptides were separated with an online nanoflow Proxeon EASY-nLC 1000 UHPLC system (Thermo Fisher Scientific) and analyzed on a benchtop Orbitrap Q Exactive mass spectrometer (Thermo Fisher Scientific) equipped\",\n",
            "          \"Exactive mass spectrometer (Thermo Fisher Scientific) equipped with a nanoflow ionization source (James A. Hill Instrument Services, Arlington, MA, USA). The LC system, column, and platinum wire to deliver electrospray source voltage were connected via a\",\n",
            "          \"to deliver electrospray source voltage were connected via a stainless-steel cross (360\\u2009\\u03bcm, IDEX Health & Science, UH-906x). The column was heated to 50\\u2009\\u00b0C using a column heater sleeve (Phoenix-ST) to prevent overpressurizing of columns during UHPLC\",\n",
            "          \"to prevent overpressurizing of columns during UHPLC separation. 10% of each global proteome sample in a 2\\u2009\\u03bcl injection volume, or 50% of each phosphoproteome sample in a 4 ul injection volume was injected onto an in-house packed 20\\u2009cm \\u00d7 75\\u2009um diameter C18\",\n",
            "          \"was injected onto an in-house packed 20\\u2009cm \\u00d7 75\\u2009um diameter C18 silica picofrit capillary column (1.9\\u2009\\u03bcm ReproSil-Pur C18-AQ beads, Dr Maisch GmbH, r119.aq; Picofrit 10\\u2009um tip opening, New Objective, PF360-75-10-N-5). Mobile phase flow rate was\",\n",
            "          \"New Objective, PF360-75-10-N-5). Mobile phase flow rate was 200\\u2009nl\\u2009min\\u22121, comprised of 3% acetonitrile/0.1% formic acid (Solvent A) and 90% acetonitrile /0.1% formic acid (Solvent B), and the 110-minute LC-MS/MS method consisted of a 10-min\",\n",
            "          \"B), and the 110-minute LC-MS/MS method consisted of a 10-min column-equilibration procedure, a 20-min sample-loading procedure, and the following gradient profile: (min:%B) 0:2; 1:6; 85:30; 94:60; 95;90; 100:90; 101:50; 110:50 (last two steps at\",\n",
            "          \"85:30; 94:60; 95;90; 100:90; 101:50; 110:50 (last two steps at 500\\u2009nl\\u2009min\\u22121 flowrate). Data-dependent acquisition was performed using Xcalibur QExactive v2.1 software in positive ion mode at a spray voltage of 2.00\\u2009kV. MS1 Spectra were measured with a\",\n",
            "          \"at a spray voltage of 2.00\\u2009kV. MS1 Spectra were measured with a resolution of 70,000, an AGC target of 3e6 and a mass range from 300 to 1,800\\u2009m/z. Up to 12 MS2 spectra per duty cycle were triggered at a resolution of 17,500, an AGC target of 5e4, an\",\n",
            "          \"triggered at a resolution of 17,500, an AGC target of 5e4, an isolation window of 2.5\\u2009m/z, a maximum ion time of 120\\u2009msec, and a normalized collision energy of 28. Peptides that triggered MS2 scans were dynamically excluded from further MS2 scans for\",\n",
            "          \"MS2 scans were dynamically excluded from further MS2 scans for 20\\u2009s. Charge state screening was enabled to reject precursor charge states that were unassigned, 1, or >6. Peptide match was enabled for monoisotopic precursor mass assignment. Protein-peptide\",\n",
            "          \"for monoisotopic precursor mass assignment. Protein-peptide identification, phosphosite localization, and quantitation. All MS data were interpreted using the Spectrum Mill software package v5.1 (for comparison with proteomes of human breast tumours from\",\n",
            "          \"(for comparison with proteomes of human breast tumours from our previous study) and v6.0 pre-release (Agilent Technologies, Santa Clara, CA, USA) co-developed by the authors. Similar MS/MS spectra acquired on the same precursor m/z within \\u00b1 45\\u2009s were\",\n",
            "          \"spectra acquired on the same precursor m/z within \\u00b1 45\\u2009s were merged. MS/MS spectra were excluded from searching if they failed the quality filter by not having a sequence tag length>0 (that is, minimum of two masses separated by the in-chain mass of an\",\n",
            "          \"is, minimum of two masses separated by the in-chain mass of an amino acid) or did not have a precursor MH+ in the range of 750\\u20136,000. MS/MS spectra from were searched against a database consisting of RefSeq release 60 containing 31,767 human proteins,\",\n",
            "          \"of RefSeq release 60 containing 31,767 human proteins, 24,821 mouse proteins, and an appended set of 85 common laboratory contaminant proteins (RefSeq.20130727-Human.20130730-MouseNR.mm13.contams). Scoring parameters were ESI-QEXACTIVE-HCD-v2, for whole\",\n",
            "          \"Scoring parameters were ESI-QEXACTIVE-HCD-v2, for whole proteome datasets, and ESI-QEXACTIVE-HCD-v3 parameters were for phosphoproteome datasets. All spectra were allowed \\u00b120 p.p.m. mass tolerance for precursor and product ions, 40% minimum matched peak\",\n",
            "          \"for precursor and product ions, 40% minimum matched peak intensity, and trypsin allow P enzyme specificity with up to four missed cleavages. Fixed modifications were carbamidomethylation at cysteine. iTRAQ labelling was required at lysine, but peptide\",\n",
            "          \"cysteine. iTRAQ labelling was required at lysine, but peptide N-termini were allowed to be either labelled or unlabelled. Allowed variable modifications for whole proteome datasets were acetylation of protein N-termini, oxidized methionine, deamidation of\",\n",
            "          \"of protein N-termini, oxidized methionine, deamidation of asparagine, pyro-glutamic acid at peptide N-terminal glutamine, and pyro-carbamidomethylation at peptide N-terminal cysteine with a precursor MH+ shift range of \\u221218 to 64\\u2009Da. Allowed variable\",\n",
            "          \"a precursor MH+ shift range of \\u221218 to 64\\u2009Da. Allowed variable modifications for phosphoproteome dataset were revised to disallow deamidation and allow phosphorylation of serine, threonine, and tyrosine with a precursor MH+ shift range of 0 to 272\\u2009Da.\",\n",
            "          \"and tyrosine with a precursor MH+ shift range of 0 to 272\\u2009Da. Identities interpreted for individual spectra were automatically designated as confidently assigned using the Spectrum Mill autovalidation module to use target-decoy based FDR estimates to\",\n",
            "          \"module to use target-decoy based FDR estimates to apply score threshold criteria via two-step strategies. For the whole proteome datasets thresholding was done at the spectral and protein levels. For the phosphoproteome datasets thresholding was done at\",\n",
            "          \"For the phosphoproteome datasets thresholding was done at the spectral level. In step 1, peptide autovalidation was done first and separately for each iTRAQ 4-plex experiment consisting of either 25 LC\\u2013MS/MS runs (whole proteome) or 13 LC\\u2013MS/MS runs\",\n",
            "          \"of either 25 LC\\u2013MS/MS runs (whole proteome) or 13 LC\\u2013MS/MS runs (phosphoproteome) using an auto thresholds strategy with a minimum sequence length of 7(whole proteome) or 8 (phosphoproteome), automatic variable range precursor mass filtering, and score\",\n",
            "          \"automatic variable range precursor mass filtering, and score and delta Rank1\\u2013Rank2 score thresholds optimized to yield a spectral level FDR estimate for precursor charges 2 through 4 of <0.6% for each precursor charge state in each LC\\u2013MS/MS run. For\",\n",
            "          \"<0.6% for each precursor charge state in each LC\\u2013MS/MS run. For precursor charges 5\\u20136, thresholds were optimized to yield a spectral level FDR estimate of <0.3% across all runs per iTRAQ 4-plex experiment (instead of each run), to achieve reasonable\",\n",
            "          \"4-plex experiment (instead of each run), to achieve reasonable statistics, since many fewer spectra are generated for the higher charge states. In step 2 for the whole proteome datasets, protein polishing autovalidation was applied separately to each\",\n",
            "          \"protein polishing autovalidation was applied separately to each iTRAQ 4-plex experiment to further filter the PSM's using a target protein-level FDR threshold of zero. The primary goal of this step was to eliminate peptides identified with low scoring\",\n",
            "          \"this step was to eliminate peptides identified with low scoring peptide spectrum matches (PSM's) that represent proteins identified by a single peptide, so-called \\u2018one-hit wonders'. After assembling protein groups from the autovalidated PSM's, protein\",\n",
            "          \"assembling protein groups from the autovalidated PSM's, protein polishing determined the maximum protein level score of a protein group that consisted entirely of distinct peptides estimated to be false-positive identifications (PSM's with negative delta\",\n",
            "          \"to be false-positive identifications (PSM's with negative delta forward-reverse scores). PSM's were removed from the set obtained in the initial peptide-level autovalidation step if they contributed to protein groups that have protein scores below the max\",\n",
            "          \"to protein groups that have protein scores below the max false-positive protein score. In the filtered results each identified protein detected in an iTRAQ 4-plex experiment was comprised of multiple peptides unless a single excellent scoring peptide was\",\n",
            "          \"multiple peptides unless a single excellent scoring peptide was the sole match. For the whole proteome datasets the above criteria yielded FDR of <0.5% at the peptide-spectrum match level and <0.8% at the distinct peptide level for each iTRAQ 4-plex\",\n",
            "          \"and <0.8% at the distinct peptide level for each iTRAQ 4-plex experiment. After assembling proteins with all the PSMs from all the iTRAQ 4-plex experiments together the aggregate FDR estimates were 0.42% at the at the peptide-spectrum match level, 1.5% at\",\n",
            "          \"were 0.42% at the at the peptide-spectrum match level, 1.5% at the distinct peptide level, and <0.01% (1/11,372) at the protein group level. Since the protein level FDR estimate neither explicitly required a minimum number of distinct peptides per protein\",\n",
            "          \"required a minimum number of distinct peptides per protein nor adjusted for the number of possible tryptic peptides per protein, it may underestimate false positive protein identifications for large proteins observed only on the basis of multiple low\",\n",
            "          \"for large proteins observed only on the basis of multiple low scoring PSMs. In calculating scores at the protein level and reporting the identified proteins, redundancy was addressed in the following manner: the protein score was the sum of the scores of\",\n",
            "          \"manner: the protein score was the sum of the scores of distinct peptides. A distinct peptide was the single highest scoring instance of a peptide detected through an MS/MS spectrum. MS/MS spectra for a particular peptide may have been recorded multiple\",\n",
            "          \"for a particular peptide may have been recorded multiple times, (that is, as different precursor charge states, in adjacent bRP fractions, modified by deamidation at Asn or oxidation of Met, or different phosphosite localization) but were still counted as\",\n",
            "          \"different phosphosite localization) but were still counted as a single distinct peptide. When a peptide sequence >8 residues long was contained in multiple protein entries in the sequence database, the proteins were grouped together and the highest\",\n",
            "          \"database, the proteins were grouped together and the highest scoring one and its accession number were reported. In some cases when the protein sequences were grouped in this manner there were distinct peptides that uniquely represented a lower scoring\",\n",
            "          \"distinct peptides that uniquely represented a lower scoring member of the group (isoforms, family members, and different species). Each of these instances spawned a subgroup. Multiple subgroups were reported and counted towards the total number of\",\n",
            "          \"subgroups were reported and counted towards the total number of proteins, and were given related protein subgroup numbers (for example, 3.1 and 3.2: group 3, subgroups 1 and 2). To better dissect the tumour/stroma (human/mouse) origin of orthologous\",\n",
            "          \"dissect the tumour/stroma (human/mouse) origin of orthologous proteins in this xenograft experiment, the inclusion of peptides contributing to each subgroup was restricted by enabling the subgroup-specific (SGS) option in Spectrum Mill. Only\",\n",
            "          \"the subgroup-specific (SGS) option in Spectrum Mill. Only subgroup-specific peptide sequences were counted toward each subgroup's count of distinct peptides and protein level TMT quantitation. The SGS option omits peptides that are shared between\",\n",
            "          \"The SGS option omits peptides that are shared between subgroups. If evidence for BOTH human and mouse peptides from an orthologous protein were observed, then peptides that cannot distinguish the two (shared) were ignored. However, the peptides shared\",\n",
            "          \"the two (shared) were ignored. However, the peptides shared between species were retained if there was specific evidence for only one of the species, thus yielding a single subgroup attributed to only the single species consistent with the specific\",\n",
            "          \"to only the single species consistent with the specific peptides. Furthermore, if all peptides observed for a protein group were shared between species, thus yielding a single subgroup composed of indistinguishable species, then all peptides were retained\",\n",
            "          \"of indistinguishable species, then all peptides were retained (the column in Supplementary Data 3 numSpeciesPresentR1 will have a value of 2 in such cases). Assembly of confidently identified PSM's yielded 20,480 total protein subgroups from 11,372\",\n",
            "          \"PSM's yielded 20,480 total protein subgroups from 11,372 protein groups. Human and mouse ortholog proteins were typically arranged into individual subgroups. In step 2 for the phosphoproteome datasets a phosphosite table were assembled with columns for\",\n",
            "          \"datasets a phosphosite table were assembled with columns for individual iTRAQ 4-plex experiments and rows for individual phosphosites. PSM's were combined into a single row for all non-conflicting observations of a particular phosphosite. (that is,\",\n",
            "          \"observations of a particular phosphosite. (that is, different missed cleavage forms, different precursor charges, confident and ambiguous localizations, different sample handling modifications). For related peptides neither observations with a different\",\n",
            "          \"For related peptides neither observations with a different number of phosphosites nor different confident localizations were allowed to be combined. Selecting the representative peptide from the combined observations was done such that once confident\",\n",
            "          \"the combined observations was done such that once confident phosphosite localization was established, higher identification scores and longer peptide lengths are preferable. After assembling the phosphosite table a polishing step was applied to further\",\n",
            "          \"the phosphosite table a polishing step was applied to further filter the phosphosites with the primary goal of eliminating phosphosites with representative peptides identified through low scoring peptide spectrum matches (PSM's) that were observed in only\",\n",
            "          \"peptide spectrum matches (PSM's) that were observed in only a few experiments. The initial table of representative peptides for 82,030 phosphosites had an aggregate FDR of 3.3% at phosphosite-level. The table was sorted by identification score and then by\",\n",
            "          \"The table was sorted by identification score and then by number of iTRAQ 4-plex experiments in which the phosphosite was observed. The cumulative FDR trend showed inflection points at an identification score of \\u223c8. Phosphosites with an identification\",\n",
            "          \"identification score of \\u223c8. Phosphosites with an identification score<8.0 observed in <3/9 experiments were therefore removed, yielding 68,385 phosphosites with an aggregate FDR of 0.34% at the phosphosite level. While the Spectrum Mill identification\",\n",
            "          \"the phosphosite level. While the Spectrum Mill identification score is based on the number of matching peaks, their ion type assignment, and the relative height of unmatched peaks, the phosphosite localization score is the difference in identification\",\n",
            "          \"localization score is the difference in identification score between the top two localizations. The score threshold for confident localization (>1.1) essentially corresponds to at least 1 b or y ion located between two candidate sites that has a peak\",\n",
            "          \"b or y ion located between two candidate sites that has a peak height 10% of the tallest fragment ion (neutral losses of phosphate from the precursor and related ions as well as immonium and iTRAQ reporter ions are excluded from the relative height\",\n",
            "          \"and iTRAQ reporter ions are excluded from the relative height calculation). The ion type scores for b-H3PO4, y-H3PO4, b-H2O, and y-H2O ion types are all set to 0.5. This prevents inappropriate confident localization assignment when a spectrum lacks\",\n",
            "          \"confident localization assignment when a spectrum lacks primary b or y ions between two possible sites but contains ions that can be assigned as either phosphate loss ions for one localization or water loss ions for another localization. In aggregate,\",\n",
            "          \"or water loss ions for another localization. In aggregate, 66.3% of the reported phosphosites were fully localized to a particular serine, threonine, or tyrosine residue. Relative abundances of proteins and phosphosites were determined in Spectrum Mill\",\n",
            "          \"of proteins and phosphosites were determined in Spectrum Mill using iTRAQ reporter ion intensity ratios from each PSM. A protein-level or phosphosite-level iTRAQ ratio was calculated as the median of all PSM level ratios contributing to a protein subgroup\",\n",
            "          \"of all PSM level ratios contributing to a protein subgroup or phosphosite remaining after excluding those PSM's lacking an iTRAQ label, having a negative delta forward-reverse score (half of all false-positive identifications), or having a precursor ion\",\n",
            "          \"all false-positive identifications), or having a precursor ion purity<50% (MS/MS has significant precursor isolation contamination from co-eluting peptides). Unless stated otherwise for a particular analysis, the following considerations apply to the\",\n",
            "          \"particular analysis, the following considerations apply to the tumour/stroma (human/mouse) origin of a protein in this xenograft experiment. For the proteome dataset, only PSM's from subgroup-specific peptide sequences contributed to the protein level\",\n",
            "          \"peptide sequences contributed to the protein level quantitation (see protein subgrouping description above). A protein detected with all contributing PSM's shared between human and mouse was considered to be human. For the phosphoproteome dataset, a\",\n",
            "          \"was considered to be human. For the phosphoproteome dataset, a phosphosite was considered to be mouse if the contributing PSM's were distinctly mouse and human if they were either distinctly human or shared between human and mouse. A 2-component Gaussian\",\n",
            "          \"human or shared between human and mouse. A 2-component Gaussian mixture model-based normalization approach was used to centre the distribution of iTRAQ log-ratios around zero to nullify the effect of differential protein loading and/or systematic MS\",\n",
            "          \"the effect of differential protein loading and/or systematic MS variation. Downstream analyses presented in the main figures were restricted to proteins/phosphosites quantified in at least 10 out of the 24 samples with non-missing values (Supplementary\",\n",
            "          \"10 out of the 24 samples with non-missing values (Supplementary Fig. 3), with the exception of the previously described mRNA-protein correlation analysis requiring quantification in 30%, or 8 out of 24, PDX samples. Specific filtering procedures are noted\",\n",
            "          \"out of 24, PDX samples. Specific filtering procedures are noted in descriptions of the relevant methods. Bioinformatics analyses. Cross data-type integration All gene names were converted to HUGO Gene Nomenclature Committee's approved gene names for\",\n",
            "          \"to HUGO Gene Nomenclature Committee's approved gene names for comparison across levels and datasets. For mRNA and protein expressions, expression values were collapsed across transcripts or isoforms to the corresponding gene using the highest mean when\",\n",
            "          \"isoforms to the corresponding gene using the highest mean when there were two transcripts or isoforms, or the value with the highest connectivity when there were more than three transcripts or isoforms as implemented in the WGCNA R package. mRNA-protein\",\n",
            "          \"or isoforms as implemented in the WGCNA R package. mRNA-protein correlation Spearman correlations between normalized RSEM values and protein quantifications were calculated for genes that were observed in at least 30% of samples for both RNA-seq and\",\n",
            "          \"were observed in at least 30% of samples for both RNA-seq and mass-spectrometry as previously described. KEGG pathway enrichment analysis of the correlation was carried out using a Kolmogorov\\u2013Smirnov test, and P values were adjusted using the\",\n",
            "          \"a Kolmogorov\\u2013Smirnov test, and P values were adjusted using the Benjamini-Hochberg procedure. Proteomic clustering and subtyping We first applied filtering for protein and phosphosite markers observed in at least 10 samples and sufficient deviations\",\n",
            "          \"observed in at least 10 samples and sufficient deviations across samples. We used a 2 s.d. threshold for iTRAQ proteome and 2.5 s.d. for the phosphoproteome. We applied the same protein marker to conduct LFQ proteome and PDX vs. human proteome\",\n",
            "          \"marker to conduct LFQ proteome and PDX vs. human proteome co-clustering. For the co-clustering with human proteome, we further selected for markers that had higher than 2 s.d.'s in the merged proteome and showed non-differential expression between human\",\n",
            "          \"proteome and showed non-differential expression between human and PDX (FDR>0.3, t-test). The subsequent hierarchical clustering was conducted using the complete agglomeration method of hclust as implemented in the heatmap2 R package. Differential\",\n",
            "          \"hclust as implemented in the heatmap2 R package. Differential expression analysis Differential expression testing of each protein in the LFQ and iTRAQ datasets was conducted using the student's t-test, and P values were adjusted using the\",\n",
            "          \"the student's t-test, and P values were adjusted using the Benjamini-Hochberg procedure. For gene set enrichment analysis, we conducted the Wilcoxon Rank Sum Test to test for changes in the t-statistics ranks of protein members in each of the KEGG\",\n",
            "          \"the t-statistics ranks of protein members in each of the KEGG signalling pathway, and again adjusted P values using the Benjamini-Hochberg procedure. Druggable genes and mutations We compiled a list of druggable genes based on extensive curation of public\",\n",
            "          \"a list of druggable genes based on extensive curation of public databases: Tumor Alterations Relevant for GEnomics-driven Therapy (TARGET, version 3, assessed on 6/15/2015), Personalized Cancer Therapy (PCT, assessed on 3/15/2015), GDKD (Gene-Drug\",\n",
            "          \"Cancer Therapy (PCT, assessed on 3/15/2015), GDKD (Gene-Drug Knowledge Database, version 11.0, assessed on 4/10/2015), CancerDR (assessed on 2/6/2015), My Cancer Genome (assessed on 9/11/2014), and DrugBank (assessed on 9/21/2015). We curated the list\",\n",
            "          \"and DrugBank (assessed on 9/21/2015). We curated the list based on evidence level and literature, as well as IC50 data when available. The final list used for the analysis included 76 druggable genes (Supplementary Data 15). Druggable outlier analysis To\",\n",
            "          \"genes (Supplementary Data 15). Druggable outlier analysis To discover expression outliers, we utilized a strategy incorporating multiple steps: first, we limited our search to genes that are in the druggable gene list. We then narrowed down the list to\",\n",
            "          \"in the druggable gene list. We then narrowed down the list to genes that are observed in at least 10 samples in the dataset under investigation. Outlier expressions were defined as values that are greater than 1.5 interquartile ranges (IQRs) above the\",\n",
            "          \"that are greater than 1.5 interquartile ranges (IQRs) above the third quartile (Q3). To rank order outlier expression for each gene, we calculated an outlier score defined as: By definition, genes with outlier score greater than 1.5 are considered as\",\n",
            "          \"genes with outlier score greater than 1.5 are considered as expression outliers. Outlier score for each gene were ranked within the sample to select the most promising druggable targets. For CNV outliers, we required them to have outlier scores above 1 in\",\n",
            "          \"outliers, we required them to have outlier scores above 1 in at least another expression level. For validation of the proteomic druggable outliers, we counted the numbers of the same protein or phosphosite outliers observed in the parallel-processed\",\n",
            "          \"or phosphosite outliers observed in the parallel-processed cohort of 77 human breast cancer samples. Due to the lower numbers in this larger cohort, we considered both proteins with outlier score greater than 1 and the top 2 outliers of each human sample\",\n",
            "          \"greater than 1 and the top 2 outliers of each human sample as validating outliers. Pathway activation analysis We first collapsed the phosphosites to gene-level phosphorylation values by averaging the phosphosite expressions observed for each gene. Then,\",\n",
            "          \"the phosphosite expressions observed for each gene. Then, we converted the phosphoproteomic expression values from iTRAQ to modified z-scores normalized against the cohort as described in Hoaglin et al. (How to Detect and Handle Outliers). We then used\",\n",
            "          \"et al. (How to Detect and Handle Outliers). We then used the Wilcoxon Rank Sum Test to test for changes in the phosphorylation z-score ranks of protein members in each of the KEGG signalling pathway. The resulting P values were adjusted using the\",\n",
            "          \"pathway. The resulting P values were adjusted using the Benjamini\\u2013Hochberg procedure. Data availability All mass spectra, in the original instrument vendor format, contributing to this study may be downloaded from:\",\n",
            "          \"format, contributing to this study may be downloaded from: https://cptac-data-portal.georgetown.edu/cptacPublic/ for the study name: TCGA Breast Cancer. Additional information How to cite this article: Huang, K.-l. et al. Proteogenomic integration reveals\",\n",
            "          \"article: Huang, K.-l. et al. Proteogenomic integration reveals therapeutic targets in breast cancer xenografts. Nat. Commun. 8, 14864 doi: 10.1038/ncomms14864 (2017). Publisher's note: Springer Nature remains neutral with regard to jurisdictional claims\",\n",
            "          \"Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. Supplementary Material C.M. Perou is on the board of directors, has ownership interest (including patents), and is a consultant/advisory board\",\n",
            "          \"(including patents), and is a consultant/advisory board member for bioclassifier LLC. The other authors declare no competing financial interests. Author contributions L.D. directed data analysis. L.D., M.J.E., S.A.C., X.C., H.R. and R.R.T. designed and\",\n",
            "          \"L.D., M.J.E., S.A.C., X.C., H.R. and R.R.T. designed and supervised research and experiment. K.-l.H., S.L., P.M., H.P.G., S.C., C.Y., D.R.M., K.R.C., C.R.K., J.W., K.V.R., D.F., M.D.M., S.M.K., S.S. and B.N. analyzed the data. K.-l.H., M.C.W. and D.R.M.\",\n",
            "          \"S.S. and B.N. analyzed the data. K.-l.H., M.C.W. and D.R.M. performed statistical analysis. S.L. established the PDX models. S.L., Z.G., S.M.K., M.T., J.U., J.E.S., J.H., P.E.-G., P.S., C.M. and C.M.P. performed functional validation experiments. J.W.Q.,\",\n",
            "          \"and C.M.P. performed functional validation experiments. J.W.Q., P.M., C.M., L.X. and H.P.G. performed data collection and tissue acquisition. M.A.W., K.-l.H., C.Y., S.C. and L.D. prepared figures and tables. M.J.E., S.R.D. and S.L. provided samples.\",\n",
            "          \"figures and tables. M.J.E., S.R.D. and S.L. provided samples. K.-l.H. and L.D. wrote the manuscript. M.J.E., M.C.W., P.M., S.A.C., X.C., R.R.T., C.M., K.-l.H. and L.D. revised the manuscript. All authors approved and contributed to the preparation of the\",\n",
            "          \"All authors approved and contributed to the preparation of the manuscript. 04/25/2017 The original version of this Article contained a typographical error in the spelling of the author Beifang Niu, which was incorrectly given as Beifung Niu. This has now\",\n",
            "          \"Niu, which was incorrectly given as Beifung Niu. This has now been corrected in both the PDF and HTML versions of the Article. Modelling human breast cancer with patient-derived xenografts (n=24). (a) Illustration of generation and proteogenomic\",\n",
            "          \"(n=24). (a) Illustration of generation and proteogenomic characterization of breast cancer xenograft models. (b) Somatic mutations of significantly mutated genes of human breast tumour were recapitulated in xenograft models. Mutation data for 23 WHIMs are\",\n",
            "          \"in xenograft models. Mutation data for 23 WHIMs are shown (exome data were not available for WHIM47). (c) Variant allele fraction analysis showed clonal representation was consistent between human breast tumour and xenografts. Genomic driver events,\",\n",
            "          \"human breast tumour and xenografts. Genomic driver events, including missenses and truncations in TP53 and PIK3CA, were retained in the xenograft models. Each colour represents one xenograft sample. Proteogenomic correlation analysis in PDX samples. (a)\",\n",
            "          \"sample. Proteogenomic correlation analysis in PDX samples. (a) Correlation between mRNA and iTRAQ protein expression levels identified pathways with significantly concordant or discordant mRNA-protein expressions. Genes were aligned along the x axis by\",\n",
            "          \"expressions. Genes were aligned along the x axis by the rank of their Spearman correlation coefficient between mRNA and protein expression levels. Each colour represents one significantly associated pathway, and each bar represents one gene in the\",\n",
            "          \"associated pathway, and each bar represents one gene in the pathway. (b) Proteogenomic summary of xenograft shows relationships among mutation, CNV (normalized log-R ratio), mRNA (log-transformed and normalized RSEM values), proteomic (normalized log2\",\n",
            "          \"and normalized RSEM values), proteomic (normalized log2 ratio to reference), and phosphoproteomic expression (normalized log2 ratio to reference) levels of breast cancer-related genes in 24 PDX samples across 4 intrinsic subtypes. Expression values from\",\n",
            "          \"PDX samples across 4 intrinsic subtypes. Expression values from each dataset were calculated as described (Methods) and truncated to a maximum of 10 and a minimum of \\u221210 for visualization. Transcriptomic and proteomic clustering of breast cancer PDX and\",\n",
            "          \"and proteomic clustering of breast cancer PDX and human samples. (a) Transcriptomic clustering of PDX breast tumours based on the PAM50 gene expression markers. (b) Proteomic clustering of PDX breast tumours based on the top 436 variably expressed\",\n",
            "          \"of PDX breast tumours based on the top 436 variably expressed proteins. (c) Phosphoproteomic clustering of PDX breast tumours based on the top 1,737 variably expressed phosphosites. (d) Proteomic clustering using only 133 non-differential expressed\",\n",
            "          \"Proteomic clustering using only 133 non-differential expressed proteins between WHIM and human breast tumour samples. The clustering reproduced the basal-enriched and luminal-enriched clusters, where PDX (n=24) and TCGA human breast tumour samples (n=77)\",\n",
            "          \"where PDX (n=24) and TCGA human breast tumour samples (n=77) cluster based on their subtypes. The non-differentially expressed proteins were identified through a t-test with FDR>0.3 between the PDX and the TCGA human tumour samples. The PDX tumours are\",\n",
            "          \"the PDX and the TCGA human tumour samples. The PDX tumours are labelled by their WHIM IDs whereas the human tumours are not labelled. Activated signalling pathways detected through pathway phosphorylation enrichment analysis. (a) Activation of the MAPK\",\n",
            "          \"phosphorylation enrichment analysis. (a) Activation of the MAPK signalling pathway in WHIM9. (b) Activation of the Ras signalling pathway in WHIM12. (c) Activation of the NF\\u03baB pathway in WHIM17. (d) Activation of the NF\\u03baB pathway in WHIM46.\",\n",
            "          \"in WHIM17. (d) Activation of the NF\\u03baB pathway in WHIM46. Phosphorylation levels of each protein in the pathway relative to the cohort of 24 PDX models are shown by the colour scale of red (high) to blue (low). Proteins with no phosphorylation data are\",\n",
            "          \"(high) to blue (low). Proteins with no phosphorylation data are coloured in green. Druggable events identified by expression outlier analysis. Druggable outlier events identified at (a) the mRNA and protein and (b) protein and phosphopeptide expression\",\n",
            "          \"mRNA and protein and (b) protein and phosphopeptide expression levels. Each colour represents one xenograft sample. Key outlier events validated in this study are labelled by text. (c) Immunochemistry staining verified outlier expression of HER2 in WHIM8\",\n",
            "          \"staining verified outlier expression of HER2 in WHIM8 and WHIM35, RAF1 in WHIM9, and FGFR2 in WHIM16. Scale bar: 50\\u2009\\u03bcm. Druggable targets identified through proteogenomic analysis in PDX and human breast cancer. (a) Outlier analysis revealed potentially\",\n",
            "          \"human breast cancer. (a) Outlier analysis revealed potentially druggable events in the RTK, PI3K, MAPK signalling, genome integrity pathways at various frequency and magnitudes across four breast cancer subtypes. Selected genes with any outlier score\",\n",
            "          \"breast cancer subtypes. Selected genes with any outlier score greater than 2.5 or in the key oncogenic pathways, including the PI3K, RTK, MAPK signalling pathways, are shown. (b) Comparison of protein expression outliers of selected druggable genes in PDX\",\n",
            "          \"protein expression outliers of selected druggable genes in PDX and human breast tumours. (c) Comparison of overexpressed phosphosite outliers of selected druggable genes in PDX and human breast tumours. Key outlier events reaching the outlier definition\",\n",
            "          \"tumours. Key outlier events reaching the outlier definition threshold or validated in this study are labelled by text. For the box plots in (b) and (c), the center line indicates median of the protein/phosphosite expression in the human and PDX cohort.\",\n",
            "          \"the protein/phosphosite expression in the human and PDX cohort. The upper and lower hinges correspond to the 25th and 75th percentiles; the upper whisker corresponds to 1.5 \\u00d7 IQR (inter-quartile range) above the 25 percentile and the lower whisker\",\n",
            "          \"range) above the 25 percentile and the lower whisker corresponds to 1.5 \\u00d7 IQR below the 75 percentile. Targeted treatments of breast cancer xenografts. (a) Western blot of HER2 protein and HER2 p.Y1248 expression levels in 5 WHIM models (WHIM6, WHIM8,\",\n",
            "          \"HER2 p.Y1248 expression levels in 5 WHIM models (WHIM6, WHIM8, WHIM12, WHIM14 and WHIM35). (b) In vivo treatment responses to lapatinib in 4 PDX models including two HER2 positive lines (WHIM8 and WHIM35) and two basal lines (WHIM6 and WHIM14). The\",\n",
            "          \"(WHIM8 and WHIM35) and two basal lines (WHIM6 and WHIM14). The response is measured in fold change (%) of tumour volumes after 2 weeks of vehicle or lapatinib treatment. (c) Immunochemistry staining of AKT phosphorylation status in WHIM16, WHIM18 and\",\n",
            "          \"staining of AKT phosphorylation status in WHIM16, WHIM18 and WHIM20 in response to PI3K inhibitor TAK-117, mTOR inhibitor TAK128, and TAK-117/TAK-128 combined. (d) In vivo treatment responses to PI3K/mTOR inhibitors (TAK-117/TAK-128) of WHIM16, WHIM18 and\",\n",
            "          \"to PI3K/mTOR inhibitors (TAK-117/TAK-128) of WHIM16, WHIM18 and WHIM20 (n=8-9 for each control or experimental group). Values were represented by average tumour volume (mm3) every 3 days following treatment (error bars: s.e.m.).\"\n",
            "        ]\n",
            "      }\n",
            "    ]\n",
            "  },\n",
            "  \"PDC000278\": {\n",
            "    \"study_id\": \"3ea1fbb1-0f05-447e-853e-639338442830\",\n",
            "    \"pdc_study_id\": \"PDC000278\",\n",
            "    \"study_submitter_id\": \"VU Normal Colon Epithelium - Proteome\",\n",
            "    \"program_id\": \"10251935-5540-11e8-b664-00a098d917f8\",\n",
            "    \"project_id\": \"48af5040-5546-11e8-b664-00a098d917f8\",\n",
            "    \"study_name\": \"VU Normal Colon Epithelium - Proteome\",\n",
            "    \"program_name\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"project_name\": \"CPTAC2 Retrospective\",\n",
            "    \"disease_type\": \"Other\",\n",
            "    \"primary_site\": \"Not Reported\",\n",
            "    \"analytical_fraction\": \"Proteome\",\n",
            "    \"experiment_type\": \"Label Free\",\n",
            "    \"embargo_date\": null,\n",
            "    \"cases_count\": 30,\n",
            "    \"aliquots_count\": 60,\n",
            "    \"protocol_id\": \"67e1fa76-3932-42cd-ba8a-01d71090674e\",\n",
            "    \"protocol_submitter_id\": \"VU_Protocol_Summary_S019\",\n",
            "    \"program_submitter_id\": \"Clinical Proteomic Tumor Analysis Consortium\",\n",
            "    \"protocol_name\": \"VU_Protocol_Summary_S019\",\n",
            "    \"protocol_date\": \"2021-02-22\"\n",
            "  }\n",
            "}\n"
          ]
        }
      ],
      "source": [
        "# Step 1: Fetch the study and protocol data\n",
        "study_protocol_url = \"https://pdc.cancer.gov/graphql\"\n",
        "study_protocol_query = \"\"\"\n",
        "query($acceptDUA: Boolean!) {\n",
        "  study(acceptDUA: $acceptDUA) {\n",
        "    study_id\n",
        "    pdc_study_id\n",
        "    study_submitter_id\n",
        "    program_id\n",
        "    project_id\n",
        "    study_name\n",
        "    program_name\n",
        "    project_name\n",
        "    disease_type\n",
        "    primary_site\n",
        "    analytical_fraction\n",
        "    experiment_type\n",
        "    embargo_date\n",
        "    cases_count\n",
        "    aliquots_count\n",
        "  }\n",
        "  protocolPerStudy {\n",
        "    protocol_id\n",
        "    protocol_submitter_id\n",
        "    study_id\n",
        "    pdc_study_id\n",
        "    study_submitter_id\n",
        "    program_id\n",
        "    program_submitter_id\n",
        "    protocol_name\n",
        "    protocol_date\n",
        "  }\n",
        "}\n",
        "\"\"\"\n",
        "variables = {\n",
        "    \"acceptDUA\": True\n",
        "}\n",
        "response = requests.post(study_protocol_url, json={'query': study_protocol_query, 'variables': variables})\n",
        "data = response.json()\n",
        "\n",
        "# Step 2: Merge study and protocol data\n",
        "merged_data = {}\n",
        "\n",
        "for study in data['data']['study']:\n",
        "    pdc_study_id = study['pdc_study_id']\n",
        "    if pdc_study_id not in merged_data:\n",
        "        merged_data[pdc_study_id] = study\n",
        "    else:\n",
        "        merged_data[pdc_study_id].update(study)\n",
        "\n",
        "for protocol in data['data']['protocolPerStudy']:\n",
        "    pdc_study_id = protocol['pdc_study_id']\n",
        "    if pdc_study_id not in merged_data:\n",
        "        merged_data[pdc_study_id] = protocol\n",
        "    else:\n",
        "        for key, value in protocol.items():\n",
        "            if key not in merged_data[pdc_study_id]:\n",
        "                merged_data[pdc_study_id][key] = value\n",
        "\n",
        "# Step 3: Fetch publication data\n",
        "publication_url = \"https://pdc.cancer.gov/graphql\"\n",
        "publication_query = \"\"\"\n",
        "{\n",
        "  getPaginatedUIPublication(offset: 0, limit: 60) {\n",
        "    uiPublication {\n",
        "      publication_id\n",
        "      pubmed_id\n",
        "      doi\n",
        "      title\n",
        "      journal\n",
        "      journal_url\n",
        "      year\n",
        "      abstract\n",
        "      studies {\n",
        "        pdc_study_id\n",
        "        submitter_id_name\n",
        "      }\n",
        "    }\n",
        "  }\n",
        "}\n",
        "\"\"\"\n",
        "pub_response = requests.post(publication_url, json={'query': publication_query})\n",
        "pub_data = pub_response.json()\n",
        "\n",
        "# Step 4: Create a mapping of pdc_study_id to publication info\n",
        "publication_mapping = {}\n",
        "\n",
        "for publication in pub_data['data']['getPaginatedUIPublication']['uiPublication']:\n",
        "    for study in publication['studies']:\n",
        "        pdc_study_id = study['pdc_study_id']\n",
        "        if pdc_study_id not in publication_mapping:\n",
        "            publication_mapping[pdc_study_id] = []\n",
        "        publication_mapping[pdc_study_id].append({\n",
        "            'publication_id': publication['publication_id'],\n",
        "            'pubmed_id': publication['pubmed_id'],\n",
        "            'doi': publication['doi'],\n",
        "            'title': publication['title'],\n",
        "            'journal': publication['journal'],\n",
        "            'journal_url': publication['journal_url'],\n",
        "            'year': publication['year'],\n",
        "            'abstract': publication['abstract'],\n",
        "        })\n",
        "\n",
        "# Step 5: Add publication info to merged data\n",
        "for pdc_study_id, publications in publication_mapping.items():\n",
        "    if pdc_study_id in merged_data:\n",
        "        merged_data[pdc_study_id]['publications'] = publications\n",
        "    else:\n",
        "        merged_data[pdc_study_id] = {'publications': publications}\n",
        "\n",
        "# Convert merged_data back to a list if needed\n",
        "merged_list = list(merged_data.values())\n",
        "\n",
        "# Step 6: Fetch the list of PubMed IDs\n",
        "publication_query_pubmed_ids = \"\"\"\n",
        "{\n",
        "  getPaginatedUIPublication(offset: 0, limit: 60) {\n",
        "    total\n",
        "    uiPublication {\n",
        "      pubmed_id\n",
        "    }\n",
        "  }\n",
        "}\n",
        "\"\"\"\n",
        "pub_response_pubmed_ids = requests.post(publication_url, json={'query': publication_query_pubmed_ids})\n",
        "response_json_pubmed_ids = pub_response_pubmed_ids.json()\n",
        "\n",
        "# Extract the list of publications\n",
        "publications = response_json_pubmed_ids['data']['getPaginatedUIPublication']['uiPublication']\n",
        "\n",
        "# Extract pubmed_id from each publication and create a list\n",
        "pubmed_ids = [publication['pubmed_id'] for publication in publications]\n",
        "\n",
        "# Step 7: Fetch article JSON from PubMed API and extract text with sentence splitting\n",
        "def fetch_article_json(pubmed_id):\n",
        "    url = f\"https://www.ncbi.nlm.nih.gov/research/bionlp/RESTful/pmcoa.cgi/BioC_json/{pubmed_id}/unicode\"\n",
        "    response = requests.get(url)\n",
        "    response.raise_for_status()  # Check for request errors\n",
        "    return response.json()\n",
        "\n",
        "def extract_article_text(article_json):\n",
        "    article_text = []\n",
        "    for document in article_json[0]['documents']:\n",
        "        for passage in document['passages']:\n",
        "            if passage['infons'].get('section_type') != 'REF':\n",
        "                article_text.append(passage['text'])\n",
        "    return \" \".join(article_text)\n",
        "\n",
        "def split_article_text(article_text, chunk_size=256, chunk_overlap=64):\n",
        "    splitter = RecursiveCharacterTextSplitter(chunk_size=chunk_size, chunk_overlap=chunk_overlap)\n",
        "    return splitter.split_text(article_text)\n",
        "\n",
        "all_articles = []\n",
        "for pubmed_id in pubmed_ids:\n",
        "    try:\n",
        "        article_json = fetch_article_json(pubmed_id)\n",
        "        article_text = extract_article_text(article_json)\n",
        "        article_text_split = split_article_text(article_text)\n",
        "        all_articles.append({ 'pubmed_id': pubmed_id, 'full_text': article_text_split })\n",
        "    except requests.exceptions.RequestException as e:\n",
        "        all_articles.append({ 'pubmed_id': pubmed_id, 'full_text': 'Full text is not available in this publication' })\n",
        "        print(f\"Failed to fetch data for PubMed ID {pubmed_id}: {e}\")\n",
        "\n",
        "# Step 8: Merge all_articles with merged_list based on pubmed_id\n",
        "pubmed_to_full_text = {article['pubmed_id']: article['full_text'] for article in all_articles}\n",
        "\n",
        "for study in merged_list:\n",
        "    if 'publications' in study:\n",
        "        valid_publications = []\n",
        "        for publication in study['publications']:\n",
        "            pubmed_id = publication['pubmed_id']\n",
        "            if pubmed_id in pubmed_to_full_text:\n",
        "                publication['full_text'] = pubmed_to_full_text[pubmed_id]\n",
        "                valid_publications.append(publication)\n",
        "        study['publications'] = valid_publications\n",
        "\n",
        "# Output the merged data\n",
        "full_data = {\n",
        "    study['pdc_study_id']: study for study in merged_list\n",
        "}\n",
        "print(json.dumps(full_data, indent=2))\n"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "GcLaUUciuq-_"
      },
      "source": [
        "# Connecting Weaviate"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 6,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "zS5NAd5ujccN",
        "outputId": "89307bb2-d4e8-40da-d835-ee4e48370119"
      },
      "outputs": [
        {
          "metadata": {
            "tags": null
          },
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "INFO:weaviate-client:Binary /root/.cache/weaviate-embedded did not exist. Downloading binary from https://github.com/weaviate/weaviate/releases/download/v1.26.1/weaviate-v1.26.1-Linux-amd64.tar.gz\n",
            "INFO:weaviate-client:Started /root/.cache/weaviate-embedded: process ID 2157\n"
          ]
        }
      ],
      "source": [
        "client = weaviate.Client(\n",
        "  embedded_options=EmbeddedOptions(),\n",
        "  additional_headers={\n",
        "    \"X-HuggingFace-Api-Key\": \"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\"\n",
        "  }\n",
        ")"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "DFevzugFt5QR"
      },
      "source": [
        "# Create the Schema"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "from wasabi import msg\n",
        "\n",
        "# Define a schema for the data\n",
        "class_obj ={\n",
        "    \"classes\": [\n",
        "        {\n",
        "            \"class\": \"Document\",\n",
        "            \"vectorizer\": \"text2vec-huggingface\",\n",
        "            \"moduleConfig\": {\n",
        "                \"text2vec-huggingface\": {\n",
        "                    \"model\": \"microsoft/BiomedNLP-BiomedBERT-base-uncased-abstract-fulltext\",\n",
        "                    \"vectorizeClassName\": False\n",
        "                }\n",
        "            },\n",
        "            \"properties\": [\n",
        "                {\n",
        "                    \"name\": \"study_id\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"A unique identifier for a study.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"pdc_study_id\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"The PDC-specific study ID (e.g., PDC000121).\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"study_submitter_id\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"An identifier submitted by the study’s submitter.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"program_id\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"Identifier for the program associated with the study.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"project_id\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"Identifier for the project within the program.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"study_name\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"Name of the study.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"program_name\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"Name of the program.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"project_name\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"Name of the project.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"disease_type\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"Type of disease being studied.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"primary_site\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"Anatomic site of the tumor or disease.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"analytical_fraction\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"The type of analytical fraction used (e.g., proteome, phosphoproteome).\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"experiment_type\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"Type of labeling reagent used. Indicate number of channels.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"embargo_date\",\n",
        "                    \"dataType\": [\"date\"],\n",
        "                    \"description\": \"Date when study data becomes publicly accessible (null if not applicable).\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"cases_count\",\n",
        "                    \"dataType\": [\"int\"],\n",
        "                    \"description\": \"Number of cases in the study (null if not available).\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"aliquots_count\",\n",
        "                    \"dataType\": [\"int\"],\n",
        "                    \"description\": \"Number of aliquots (samples) in the study (null if not available).\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"protocol_id\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"Identifier for the experimental protocol.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"protocol_submitter_id\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"Identifier submitted by the protocol submitter.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"program_submitter_id\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"Identifier submitted by the program submitter.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"protocol_name\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"Name of the experimental protocol.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"protocol_date\",\n",
        "                    \"dataType\": [\"date\"],\n",
        "                    \"description\": \"Date protocol was created or updated.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"publication_id\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"Publication identifier.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"pubmed_id\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"PubMed identifier.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"doi\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"Digital Object Identifier (DOI) for the publication.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"author\",\n",
        "                    \"dataType\": [\"text\"],\n",
        "                    \"description\": \"Author(s) of the publication.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"title\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"Title of the publication.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"journal\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"Journal where the publication appeared.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"journal_url\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"URL to the journal.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"year\",\n",
        "                    \"dataType\": [\"string\"],\n",
        "                    \"description\": \"Year of publication.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"abstract\",\n",
        "                    \"dataType\": [\"text\"],\n",
        "                    \"description\": \"Abstract of the publication.\"\n",
        "                },\n",
        "                {\n",
        "                    \"name\": \"full_text\",\n",
        "                    \"dataType\": [\"text\"],\n",
        "                    \"description\": \"Full text of the publication.\"\n",
        "                }\n",
        "            ]\n",
        "        }\n",
        "    ]\n",
        "}\n",
        "\n",
        "\n",
        "if not client.schema.exists(\"Document\"):\n",
        "    client.schema.create(class_obj)\n",
        "    msg.warn(f\"Document class was created because it didn't exist.\")\n",
        "else:\n",
        "    # WARNING THIS DELETES ALL Document AND CREATES A NEW Document CLASS\n",
        "    client.schema.delete_class(\"Document\")\n",
        "    msg.info(f\"Document class was removed because it already exists\")\n",
        "    client.schema.create(class_obj)"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "75OlGPW-eZdg",
        "outputId": "18199e5a-cda3-4b35-ade6-f50e7ce00ccb"
      },
      "execution_count": 7,
      "outputs": [
        {
          "metadata": {
            "tags": null
          },
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "\u001b[38;5;3m⚠ Document class was created because it didn't exist.\u001b[0m\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "def jprint(json_in):\n",
        "    import json\n",
        "    print(json.dumps(json_in, indent=2))\n",
        "jprint(client.schema.get())"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "G_T6lPzye1kF",
        "outputId": "0263acb4-e846-4c25-f1db-ffc64cdb9b4f"
      },
      "execution_count": 8,
      "outputs": [
        {
          "metadata": {
            "tags": null
          },
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "{\n",
            "  \"classes\": [\n",
            "    {\n",
            "      \"class\": \"Document\",\n",
            "      \"invertedIndexConfig\": {\n",
            "        \"bm25\": {\n",
            "          \"b\": 0.75,\n",
            "          \"k1\": 1.2\n",
            "        },\n",
            "        \"cleanupIntervalSeconds\": 60,\n",
            "        \"stopwords\": {\n",
            "          \"additions\": null,\n",
            "          \"preset\": \"en\",\n",
            "          \"removals\": null\n",
            "        }\n",
            "      },\n",
            "      \"moduleConfig\": {\n",
            "        \"text2vec-huggingface\": {\n",
            "          \"model\": \"microsoft/BiomedNLP-BiomedBERT-base-uncased-abstract-fulltext\",\n",
            "          \"vectorizeClassName\": false\n",
            "        }\n",
            "      },\n",
            "      \"multiTenancyConfig\": {\n",
            "        \"autoTenantActivation\": false,\n",
            "        \"autoTenantCreation\": false,\n",
            "        \"enabled\": false\n",
            "      },\n",
            "      \"properties\": [\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"A unique identifier for a study.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"study_id\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"The PDC-specific study ID (e.g., PDC000121).\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"pdc_study_id\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"An identifier submitted by the study\\u2019s submitter.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"study_submitter_id\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"Identifier for the program associated with the study.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"program_id\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"Identifier for the project within the program.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"project_id\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"Name of the study.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"study_name\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"Name of the program.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"program_name\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"Name of the project.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"project_name\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"Type of disease being studied.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"disease_type\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"Anatomic site of the tumor or disease.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"primary_site\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"The type of analytical fraction used (e.g., proteome, phosphoproteome).\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"analytical_fraction\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"Type of labeling reagent used. Indicate number of channels.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"experiment_type\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"date\"\n",
            "          ],\n",
            "          \"description\": \"Date when study data becomes publicly accessible (null if not applicable).\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": false,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"embargo_date\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"int\"\n",
            "          ],\n",
            "          \"description\": \"Number of cases in the study (null if not available).\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": false,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"cases_count\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"int\"\n",
            "          ],\n",
            "          \"description\": \"Number of aliquots (samples) in the study (null if not available).\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": false,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"aliquots_count\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"Identifier for the experimental protocol.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"protocol_id\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"Identifier submitted by the protocol submitter.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"protocol_submitter_id\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"Identifier submitted by the program submitter.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"program_submitter_id\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"Name of the experimental protocol.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"protocol_name\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"date\"\n",
            "          ],\n",
            "          \"description\": \"Date protocol was created or updated.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": false,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"protocol_date\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"Publication identifier.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"publication_id\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"PubMed identifier.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"pubmed_id\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"Digital Object Identifier (DOI) for the publication.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"doi\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"Author(s) of the publication.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"author\",\n",
            "          \"tokenization\": \"word\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"Title of the publication.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"title\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"Journal where the publication appeared.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"journal\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"URL to the journal.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"journal_url\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"Year of publication.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"year\",\n",
            "          \"tokenization\": \"whitespace\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"Abstract of the publication.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"abstract\",\n",
            "          \"tokenization\": \"word\"\n",
            "        },\n",
            "        {\n",
            "          \"dataType\": [\n",
            "            \"text\"\n",
            "          ],\n",
            "          \"description\": \"Full text of the publication.\",\n",
            "          \"indexFilterable\": true,\n",
            "          \"indexRangeFilters\": false,\n",
            "          \"indexSearchable\": true,\n",
            "          \"moduleConfig\": {\n",
            "            \"text2vec-huggingface\": {\n",
            "              \"skip\": false,\n",
            "              \"vectorizePropertyName\": false\n",
            "            }\n",
            "          },\n",
            "          \"name\": \"full_text\",\n",
            "          \"tokenization\": \"word\"\n",
            "        }\n",
            "      ],\n",
            "      \"replicationConfig\": {\n",
            "        \"asyncEnabled\": false,\n",
            "        \"factor\": 1\n",
            "      },\n",
            "      \"shardingConfig\": {\n",
            "        \"actualCount\": 1,\n",
            "        \"actualVirtualCount\": 128,\n",
            "        \"desiredCount\": 1,\n",
            "        \"desiredVirtualCount\": 128,\n",
            "        \"function\": \"murmur3\",\n",
            "        \"key\": \"_id\",\n",
            "        \"strategy\": \"hash\",\n",
            "        \"virtualPerPhysical\": 128\n",
            "      },\n",
            "      \"vectorIndexConfig\": {\n",
            "        \"bq\": {\n",
            "          \"enabled\": false\n",
            "        },\n",
            "        \"cleanupIntervalSeconds\": 300,\n",
            "        \"distance\": \"cosine\",\n",
            "        \"dynamicEfFactor\": 8,\n",
            "        \"dynamicEfMax\": 500,\n",
            "        \"dynamicEfMin\": 100,\n",
            "        \"ef\": -1,\n",
            "        \"efConstruction\": 128,\n",
            "        \"flatSearchCutoff\": 40000,\n",
            "        \"maxConnections\": 32,\n",
            "        \"pq\": {\n",
            "          \"bitCompression\": false,\n",
            "          \"centroids\": 256,\n",
            "          \"enabled\": false,\n",
            "          \"encoder\": {\n",
            "            \"distribution\": \"log-normal\",\n",
            "            \"type\": \"kmeans\"\n",
            "          },\n",
            "          \"segments\": 0,\n",
            "          \"trainingLimit\": 100000\n",
            "        },\n",
            "        \"skip\": false,\n",
            "        \"sq\": {\n",
            "          \"enabled\": false,\n",
            "          \"rescoreLimit\": 20,\n",
            "          \"trainingLimit\": 100000\n",
            "        },\n",
            "        \"vectorCacheMaxObjects\": 1000000000000\n",
            "      },\n",
            "      \"vectorIndexType\": \"hnsw\",\n",
            "      \"vectorizer\": \"text2vec-huggingface\"\n",
            "    }\n",
            "  ]\n",
            "}\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "ItYG5B45tX48"
      },
      "source": [
        "# Import Data to Weaviate"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "from transformers import AutoTokenizer, AutoModel\n",
        "import torch\n",
        "import logging\n",
        "import numpy as np\n",
        "import time\n",
        "\n",
        "# Embed Text Data using neuml/pubmedbert-base-embeddings\n",
        "tokenizer = AutoTokenizer.from_pretrained(\"microsoft/BiomedNLP-BiomedBERT-base-uncased-abstract-fulltext\")\n",
        "model = AutoModel.from_pretrained(\"microsoft/BiomedNLP-BiomedBERT-base-uncased-abstract-fulltext\")\n",
        "\n",
        "def generate_embedding(text):\n",
        "    inputs = tokenizer(text, return_tensors=\"pt\", truncation=True, padding=True, max_length=512)\n",
        "    with torch.no_grad():\n",
        "        outputs = model(**inputs)\n",
        "    return outputs.last_hidden_state.mean(dim=1).squeeze().numpy().tolist()\n",
        "\n",
        "\n",
        "# Configure logging\n",
        "logging.basicConfig(level=logging.INFO)\n",
        "msg = logging.getLogger()\n",
        "\n",
        "with client.batch as batch:\n",
        "    batch.batch_size = 100\n",
        "    for i, (pdc_study_id, doc) in enumerate(full_data.items()):\n",
        "        msg.info(f\"({i+1}/{len(full_data)}) Importing Study {doc.get('pdc_study_id', 'Unknown ID')}\")\n",
        "\n",
        "        # Handle multiple publications if present\n",
        "        documents = []\n",
        "        study_info = (\n",
        "            f\"PDC Study ID: {doc['pdc_study_id']}, \"\n",
        "            f\"Study Submitter ID: {doc['study_submitter_id']}, \"\n",
        "            f\"Program ID: {doc['program_id']}, \"\n",
        "            f\"Project ID: {doc['project_id']}, \"\n",
        "            f\"Study Name: {doc['study_name']}, \"\n",
        "            f\"Program Name: {doc['program_name']}, \"\n",
        "            f\"Project Name: {doc['project_name']}, \"\n",
        "            f\"Disease Type: {doc['disease_type']}, \"\n",
        "            f\"Primary Site: {doc['primary_site']}, \"\n",
        "            f\"Analytical Fraction: {doc['analytical_fraction']}, \"\n",
        "            f\"Experiment Type: {doc['experiment_type']}, \"\n",
        "            f\"Embargo Date: {doc['embargo_date']}, \"\n",
        "            f\"Cases Count: {doc['cases_count']}, \"\n",
        "            f\"Aliquots Count: {doc['aliquots_count']}, \"\n",
        "            f\"Protocol ID: {doc['protocol_id']}, \"\n",
        "            f\"Protocol Submitter ID: {doc['protocol_submitter_id']}, \"\n",
        "            f\"Program Submitter ID: {doc['program_submitter_id']}, \"\n",
        "            f\"Protocol Name: {doc['protocol_name']}, \"\n",
        "            f\"Protocol Date: {doc['protocol_date']}\"\n",
        "        )\n",
        "\n",
        "        # Adding publication information if available\n",
        "        if 'publications' in doc and doc['publications']:\n",
        "            for pub in doc['publications']:\n",
        "                pub_info = (\n",
        "                    f\"Publication ID: {pub['publication_id']}, \"\n",
        "                    f\"PubMed ID: {pub['pubmed_id']}, \"\n",
        "                    f\"DOI: {pub['doi']}, \"\n",
        "                    f\"Title: {pub['title']}, \"\n",
        "                    f\"Journal: {pub['journal']}, \"\n",
        "                    f\"Journal URL: {pub['journal_url']}, \"\n",
        "                    f\"Year: {pub['year']}, \"\n",
        "                    f\"Abstract: {pub['abstract']} \"\n",
        "                )\n",
        "                study_info += f\", Publication: ({pub_info})\"\n",
        "\n",
        "        documents.append(study_info)\n",
        "        embedding = generate_embedding(' '.join(documents))\n",
        "\n",
        "        properties = {\n",
        "            \"study_id\": doc.get(\"study_id\", \"\"),\n",
        "            \"pdc_study_id\": doc.get(\"pdc_study_id\", \"\"),\n",
        "            \"study_submitter_id\": doc.get(\"study_submitter_id\", \"\"),\n",
        "            \"program_id\": doc.get(\"program_id\", \"\"),\n",
        "            \"project_id\": doc.get(\"project_id\", \"\"),\n",
        "            \"study_name\": doc.get(\"study_name\", \"\"),\n",
        "            \"program_name\": doc.get(\"program_name\", \"\"),\n",
        "            \"project_name\": doc.get(\"project_name\", \"\"),\n",
        "            \"disease_type\": doc.get(\"disease_type\", \"\"),\n",
        "            \"primary_site\": doc.get(\"primary_site\", \"\"),\n",
        "            \"analytical_fraction\": doc.get(\"analytical_fraction\", \"\"),\n",
        "            \"experiment_type\": doc.get(\"experiment_type\", \"\"),\n",
        "            \"cases_count\": doc.get(\"cases_count\", None),\n",
        "            \"aliquots_count\": doc.get(\"aliquots_count\", None),\n",
        "            \"text_to_embed\": ' '.join(documents)\n",
        "        }\n",
        "\n",
        "        if 'publications' in doc and doc['publications']:\n",
        "            for pub in doc['publications']:\n",
        "                properties.update({\n",
        "                    \"publication_id\": pub.get(\"publication_id\", \"\"),\n",
        "                    \"pubmed_id\": pub.get(\"pubmed_id\", \"\"),\n",
        "                    \"doi\": pub.get(\"doi\", \"\"),\n",
        "                    \"author\": pub.get(\"author\", \"\"),\n",
        "                    \"title\": pub.get(\"title\", \"\"),\n",
        "                    \"journal\": pub.get(\"journal\", \"\"),\n",
        "                    \"journal_url\": pub.get(\"journal_url\", \"\"),\n",
        "                    \"year\": pub.get(\"year\", \"\"),\n",
        "                    \"abstract\": pub.get(\"abstract\", \"\")\n",
        "                })\n",
        "\n",
        "        client.batch.add_data_object(properties, \"Document\", vector=embedding)\n",
        "        msg.info(\"Data imported\")"
      ],
      "metadata": {
        "id": "6TDHEjwYgOb2",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 232,
          "referenced_widgets": [
            "f2effa109824477394e9cdf4d02e2ef8",
            "f493e172d62449a0af6d6e17356577ee",
            "7c89b29f901f40afa432f2a0a0ee2b7f",
            "8bff5b1a19dd4457a6740ab1ee3d88c4",
            "f82f2ddf2eba4e319186fa015fbf909d",
            "79c2349768134eab82edecef1b81d032",
            "86c5e01197c4457491a236cc6136a54a",
            "a4af9a54a0384db7b2265ff701b71a65",
            "f18699c03dc3448d82e48bd94131970e",
            "7716bd03cb844d37a3947921a045b8c6",
            "cb4d8a8e54dd4275bb3a027ddb31fff1",
            "81d50ba5d6424cb7b548bf0d747b386c",
            "ff1e48f3bdc548bd8b0dc80df657287d",
            "4da8303f8a8c4d6f98cfb8aa44fbf197",
            "cdf9c1546be84fe387cf49e306f1aaaa",
            "0f9a4f95f0884cd58f6b9f1cdf6f01a2",
            "e94c2a2991884a82b75425d5fe15c0a4",
            "fa22c8f44d0646bbb568371809df90af",
            "47f966d5dcb64dd7a26d077823c9d10e",
            "b7cb1815160744afa231f61acccd1280",
            "69204280d0bd4a9a8068597343386839",
            "c0254ede3b1e4efb8978924de358ac4c",
            "b93f7d774e044573a89e819eb76fe92a",
            "e6ff09e32fd941dbac5cbf819a79c6b7",
            "2d71c1900ead44fd89e3fd89dd7040ec",
            "e1124d2f9264473d88768b3d9a5c7e83",
            "aa6e53eb2868430d9c7df06c256265a4",
            "7f42e38709fc435eb08c67af1ca811ca",
            "7f5cc813c63549fa8444bea0e370096e",
            "647606068f9b4a55872fe5b89dc499b2",
            "cdfadf7836194184aa2b3bb5b2fb9251",
            "6cd91df2cd5342249dd3952c7df21c08",
            "cd5460bedbd34255bb38fd472a9ad1cb",
            "6a31bd80d106457f9fb9815c4abfacab",
            "ffa3da38deef4d69bfd7568549ec448d",
            "0c3e8f1d60ad4e14a8f2fadc29bf7c4a",
            "8810615977d44056baeae312b2cb3949",
            "ee74b2ffd9e643888b3cd6edc291091a",
            "52eec76de628416daf23b5a604fe23e5",
            "66a53cf98b9c418598807707caf26427",
            "ef5251e61e21485c8043d80127de7503",
            "c14aaaa4a7bb49cfaa9907b0c12a41dd",
            "3c6a08489c164ef4ae3e89f24d8b4373",
            "c7b2a9fc66f5415e89f82bb5b21f02fe"
          ]
        },
        "outputId": "c22862ca-9cf6-47f7-dafc-513c5535928b"
      },
      "execution_count": 9,
      "outputs": [
        {
          "metadata": {
            "tags": null
          },
          "name": "stderr",
          "output_type": "stream",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/huggingface_hub/utils/_token.py:99: UserWarning: \n",
            "Error while fetching `HF_TOKEN` secret value from your vault: 'Requesting secret HF_TOKEN timed out. Secrets can only be fetched when running from the Colab UI.'.\n",
            "You are not authenticated with the Hugging Face Hub in this notebook.\n",
            "If the error persists, please let us know by opening an issue on GitHub (https://github.com/huggingface/huggingface_hub/issues/new).\n",
            "  warnings.warn(\n"
          ]
        },
        {
          "data": {
            "application/vnd.jupyter.widget-view+json": {
              "model_id": "f2effa109824477394e9cdf4d02e2ef8",
              "version_major": 2,
              "version_minor": 0
            },
            "text/plain": [
              "tokenizer_config.json:   0%|          | 0.00/28.0 [00:00<?, ?B/s]"
            ]
          },
          "metadata": {},
          "output_type": "display_data"
        },
        {
          "data": {
            "application/vnd.jupyter.widget-view+json": {
              "model_id": "81d50ba5d6424cb7b548bf0d747b386c",
              "version_major": 2,
              "version_minor": 0
            },
            "text/plain": [
              "config.json:   0%|          | 0.00/385 [00:00<?, ?B/s]"
            ]
          },
          "metadata": {},
          "output_type": "display_data"
        },
        {
          "data": {
            "application/vnd.jupyter.widget-view+json": {
              "model_id": "b93f7d774e044573a89e819eb76fe92a",
              "version_major": 2,
              "version_minor": 0
            },
            "text/plain": [
              "vocab.txt:   0%|          | 0.00/226k [00:00<?, ?B/s]"
            ]
          },
          "metadata": {},
          "output_type": "display_data"
        },
        {
          "data": {
            "application/vnd.jupyter.widget-view+json": {
              "model_id": "6a31bd80d106457f9fb9815c4abfacab",
              "version_major": 2,
              "version_minor": 0
            },
            "text/plain": [
              "pytorch_model.bin:   0%|          | 0.00/440M [00:00<?, ?B/s]"
            ]
          },
          "metadata": {},
          "output_type": "display_data"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "# Check data import"
      ],
      "metadata": {
        "id": "kCgTyvNPYfPQ"
      }
    },
    {
      "cell_type": "code",
      "execution_count": 10,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "FlwxlUVxnkPd",
        "outputId": "15974d0e-d013-496a-b7ef-15058167795c"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "{\n",
            "    \"data\": {\n",
            "        \"Get\": {\n",
            "            \"Document\": [\n",
            "                {\n",
            "                    \"pdc_study_id\": \"PDC000403\",\n",
            "                    \"primary_site\": \"Hematopoietic and reticuloendothelial systems;Not Reported\",\n",
            "                    \"study_name\": \"AML Ex Vivo Drug Response - Combination Treatment - Phosphoproteome\"\n",
            "                },\n",
            "                {\n",
            "                    \"pdc_study_id\": \"PDC000402\",\n",
            "                    \"primary_site\": \"Hematopoietic and reticuloendothelial systems;Not Reported\",\n",
            "                    \"study_name\": \"AML Ex Vivo Drug Response - Combination Treatment - Proteome\"\n",
            "                },\n",
            "                {\n",
            "                    \"pdc_study_id\": \"PDC000399\",\n",
            "                    \"primary_site\": \"Hematopoietic and reticuloendothelial systems;Not Reported\",\n",
            "                    \"study_name\": \"AML Ex Vivo Drug Response - Primary Cohort - Phosphoproteome\"\n",
            "                },\n",
            "                {\n",
            "                    \"pdc_study_id\": \"PDC000398\",\n",
            "                    \"primary_site\": \"Hematopoietic and reticuloendothelial systems;Not Reported\",\n",
            "                    \"study_name\": \"AML Ex Vivo Drug Response - Primary Cohort - Proteome\"\n",
            "                },\n",
            "                {\n",
            "                    \"pdc_study_id\": \"PDC000401\",\n",
            "                    \"primary_site\": \"Hematopoietic and reticuloendothelial systems;Not Reported\",\n",
            "                    \"study_name\": \"AML Ex Vivo Drug Response - Sorafenib Treatment - Phosphoproteome\"\n",
            "                },\n",
            "                {\n",
            "                    \"pdc_study_id\": \"PDC000400\",\n",
            "                    \"primary_site\": \"Hematopoietic and reticuloendothelial systems;Not Reported\",\n",
            "                    \"study_name\": \"AML Ex Vivo Drug Response - Sorafenib Treatment - Proteome\"\n",
            "                },\n",
            "                {\n",
            "                    \"pdc_study_id\": \"PDC000318\",\n",
            "                    \"primary_site\": \"Hematopoietic and reticuloendothelial systems;Not Reported\",\n",
            "                    \"study_name\": \"AML Gilteritinib Resistance -  Phosphoproteome\"\n",
            "                },\n",
            "                {\n",
            "                    \"pdc_study_id\": \"PDC000315\",\n",
            "                    \"primary_site\": \"Hematopoietic and reticuloendothelial systems;Not Reported\",\n",
            "                    \"study_name\": \"AML Gilteritinib Resistance -  Proteome\"\n",
            "                },\n",
            "                {\n",
            "                    \"pdc_study_id\": \"PDC000319\",\n",
            "                    \"primary_site\": \"Hematopoietic and reticuloendothelial systems;Not Reported\",\n",
            "                    \"study_name\": \"AML Gilteritinib TimeCourse -  Phosphoproteome\"\n",
            "                },\n",
            "                {\n",
            "                    \"pdc_study_id\": \"PDC000316\",\n",
            "                    \"primary_site\": \"Hematopoietic and reticuloendothelial systems;Not Reported\",\n",
            "                    \"study_name\": \"AML Gilteritinib TimeCourse -  Proteome\"\n",
            "                }\n",
            "            ]\n",
            "        }\n",
            "    }\n",
            "}\n"
          ]
        }
      ],
      "source": [
        "where_filter= {\n",
        "    \"path\":[\"text_to_embed\"],\n",
        "    \"operator\":\"Like\",\n",
        "    \"valueText\":\"*leukemia*\"\n",
        "}\n",
        "result1 = (\n",
        "    client.query\n",
        "    .get(\"Document\",[\"pdc_study_id\", \"study_name\", \"primary_site\"])\n",
        "    .with_limit(10)\n",
        "    .with_where(where_filter)\n",
        "    .do()\n",
        ")\n",
        "print(json.dumps(result1, indent=4))\n"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "def retrieve_documents(client, query_text, top_k=5):\n",
        "    query_embedding = generate_embedding(query_text)\n",
        "\n",
        "    # Define the properties to retrieve\n",
        "    properties = [\n",
        "        \"study_id\", \"pdc_study_id\", \"study_submitter_id\", \"study_name\",\n",
        "        \"program_name\", \"project_name\", \"disease_type\", \"primary_site\",\n",
        "        \"analytical_fraction\", \"experiment_type\", \"cases_count\",\n",
        "        \"aliquots_count\", \"pubmed_id\", \"doi\", \"author\", \"title\",\n",
        "        \"journal\", \"journal_url\", \"year\", \"abstract\", \"text_to_embed\"\n",
        "    ]\n",
        "\n",
        "    try:\n",
        "        results = client.query.get(\n",
        "            \"Document\",\n",
        "            properties=properties\n",
        "        ).with_near_vector({\n",
        "            \"vector\": query_embedding,\n",
        "            \"certainty\": 0.7\n",
        "        }).with_limit(top_k).do()\n",
        "\n",
        "        response = results['data']['Get']['Document']\n",
        "\n",
        "        # Create a list of dictionaries with only the desired fields\n",
        "        return [{'pdc_study_id': doc.get('pdc_study_id'), 'study_name': doc.get('study_name'), 'primary_site': doc.get('primary_site')} for doc in response]\n",
        "\n",
        "    except Exception as e:\n",
        "        logging.error(f\"An error occurred while retrieving documents: {e}\")\n",
        "        return []\n",
        "\n",
        "# Example usage\n",
        "question = \"Provide studies related to breast cancer.\"\n",
        "retrieve_documents(client, query_text=question)\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "jGQFfNJvkXHz",
        "outputId": "8b4a7333-d784-4714-f786-0a712898710e"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "[{'pdc_study_id': 'PDC000526',\n",
              "  'study_name': 'CPTAC PDAC Proteins in Serum - Proteome',\n",
              "  'primary_site': 'Not Applicable;Pancreas'},\n",
              " {'pdc_study_id': 'PDC000325',\n",
              "  'study_name': 'Microscaled Proteogenomic Methods for Precision Oncology DP1 Clinical Trial - Proteome',\n",
              "  'primary_site': 'Breast;Not Reported'},\n",
              " {'pdc_study_id': 'PDC000329',\n",
              "  'study_name': 'Microscaled Proteogenomic Methods for Precision Oncology PDX cores - Proteome',\n",
              "  'primary_site': 'Not Reported'},\n",
              " {'pdc_study_id': 'PDC000327',\n",
              "  'study_name': 'Microscaled Proteogenomic Methods for Precision Oncology PDX bulk - Proteome',\n",
              "  'primary_site': 'Not Reported'},\n",
              " {'pdc_study_id': 'PDC000303',\n",
              "  'study_name': 'Therapeutic Targets in Breast Cancer Xenografts -  Proteome',\n",
              "  'primary_site': 'Not Reported'}]"
            ]
          },
          "metadata": {},
          "execution_count": 20
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "BwRjM_-b8Cum"
      },
      "source": [
        "# Load llm"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 11,
      "metadata": {
        "id": "VWJ-pEwe8Zu-"
      },
      "outputs": [],
      "source": [
        "from langchain_community.llms import LlamaCpp\n",
        "from langchain.chains import RetrievalQA, LLMChain"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 12,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "collapsed": true,
        "id": "MKDFEMDSS9ri",
        "outputId": "aa4ea128-7719-419c-8482-f086d3924474"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "llama_model_loader: loaded meta data with 21 key-value pairs and 291 tensors from /content/drive/MyDrive/HIDS-7950/llm/BioMistral-7B.Q4_K_M.gguf (version GGUF V3 (latest))\n",
            "llama_model_loader: Dumping metadata keys/values. Note: KV overrides do not apply in this output.\n",
            "llama_model_loader: - kv   0:                       general.architecture str              = llama\n",
            "llama_model_loader: - kv   1:                               general.name str              = hub\n",
            "llama_model_loader: - kv   2:                       llama.context_length u32              = 32768\n",
            "llama_model_loader: - kv   3:                     llama.embedding_length u32              = 4096\n",
            "llama_model_loader: - kv   4:                          llama.block_count u32              = 32\n",
            "llama_model_loader: - kv   5:                  llama.feed_forward_length u32              = 14336\n",
            "llama_model_loader: - kv   6:                 llama.rope.dimension_count u32              = 128\n",
            "llama_model_loader: - kv   7:                 llama.attention.head_count u32              = 32\n",
            "llama_model_loader: - kv   8:              llama.attention.head_count_kv u32              = 8\n",
            "llama_model_loader: - kv   9:     llama.attention.layer_norm_rms_epsilon f32              = 0.000010\n",
            "llama_model_loader: - kv  10:                       llama.rope.freq_base f32              = 10000.000000\n",
            "llama_model_loader: - kv  11:                          general.file_type u32              = 15\n",
            "llama_model_loader: - kv  12:                       tokenizer.ggml.model str              = llama\n",
            "llama_model_loader: - kv  13:                      tokenizer.ggml.tokens arr[str,32000]   = [\"<unk>\", \"<s>\", \"</s>\", \"<0x00>\", \"<...\n",
            "llama_model_loader: - kv  14:                      tokenizer.ggml.scores arr[f32,32000]   = [0.000000, 0.000000, 0.000000, 0.0000...\n",
            "llama_model_loader: - kv  15:                  tokenizer.ggml.token_type arr[i32,32000]   = [2, 3, 3, 6, 6, 6, 6, 6, 6, 6, 6, 6, ...\n",
            "llama_model_loader: - kv  16:                tokenizer.ggml.bos_token_id u32              = 1\n",
            "llama_model_loader: - kv  17:                tokenizer.ggml.eos_token_id u32              = 2\n",
            "llama_model_loader: - kv  18:            tokenizer.ggml.unknown_token_id u32              = 0\n",
            "llama_model_loader: - kv  19:                    tokenizer.chat_template str              = {{ bos_token }}{% for message in mess...\n",
            "llama_model_loader: - kv  20:               general.quantization_version u32              = 2\n",
            "llama_model_loader: - type  f32:   65 tensors\n",
            "llama_model_loader: - type q4_K:  193 tensors\n",
            "llama_model_loader: - type q6_K:   33 tensors\n",
            "llm_load_vocab: special tokens cache size = 3\n",
            "llm_load_vocab: token to piece cache size = 0.1637 MB\n",
            "llm_load_print_meta: format           = GGUF V3 (latest)\n",
            "llm_load_print_meta: arch             = llama\n",
            "llm_load_print_meta: vocab type       = SPM\n",
            "llm_load_print_meta: n_vocab          = 32000\n",
            "llm_load_print_meta: n_merges         = 0\n",
            "llm_load_print_meta: vocab_only       = 0\n",
            "llm_load_print_meta: n_ctx_train      = 32768\n",
            "llm_load_print_meta: n_embd           = 4096\n",
            "llm_load_print_meta: n_layer          = 32\n",
            "llm_load_print_meta: n_head           = 32\n",
            "llm_load_print_meta: n_head_kv        = 8\n",
            "llm_load_print_meta: n_rot            = 128\n",
            "llm_load_print_meta: n_swa            = 0\n",
            "llm_load_print_meta: n_embd_head_k    = 128\n",
            "llm_load_print_meta: n_embd_head_v    = 128\n",
            "llm_load_print_meta: n_gqa            = 4\n",
            "llm_load_print_meta: n_embd_k_gqa     = 1024\n",
            "llm_load_print_meta: n_embd_v_gqa     = 1024\n",
            "llm_load_print_meta: f_norm_eps       = 0.0e+00\n",
            "llm_load_print_meta: f_norm_rms_eps   = 1.0e-05\n",
            "llm_load_print_meta: f_clamp_kqv      = 0.0e+00\n",
            "llm_load_print_meta: f_max_alibi_bias = 0.0e+00\n",
            "llm_load_print_meta: f_logit_scale    = 0.0e+00\n",
            "llm_load_print_meta: n_ff             = 14336\n",
            "llm_load_print_meta: n_expert         = 0\n",
            "llm_load_print_meta: n_expert_used    = 0\n",
            "llm_load_print_meta: causal attn      = 1\n",
            "llm_load_print_meta: pooling type     = 0\n",
            "llm_load_print_meta: rope type        = 0\n",
            "llm_load_print_meta: rope scaling     = linear\n",
            "llm_load_print_meta: freq_base_train  = 10000.0\n",
            "llm_load_print_meta: freq_scale_train = 1\n",
            "llm_load_print_meta: n_ctx_orig_yarn  = 32768\n",
            "llm_load_print_meta: rope_finetuned   = unknown\n",
            "llm_load_print_meta: ssm_d_conv       = 0\n",
            "llm_load_print_meta: ssm_d_inner      = 0\n",
            "llm_load_print_meta: ssm_d_state      = 0\n",
            "llm_load_print_meta: ssm_dt_rank      = 0\n",
            "llm_load_print_meta: model type       = 7B\n",
            "llm_load_print_meta: model ftype      = Q4_K - Medium\n",
            "llm_load_print_meta: model params     = 7.24 B\n",
            "llm_load_print_meta: model size       = 4.07 GiB (4.83 BPW) \n",
            "llm_load_print_meta: general.name     = hub\n",
            "llm_load_print_meta: BOS token        = 1 '<s>'\n",
            "llm_load_print_meta: EOS token        = 2 '</s>'\n",
            "llm_load_print_meta: UNK token        = 0 '<unk>'\n",
            "llm_load_print_meta: LF token         = 13 '<0x0A>'\n",
            "llm_load_print_meta: max token length = 48\n",
            "llm_load_tensors: ggml ctx size =    0.14 MiB\n",
            "llm_load_tensors:        CPU buffer size =  4165.37 MiB\n",
            ".................................................................................................\n",
            "llama_new_context_with_model: n_batch is less than GGML_KQ_MASK_PAD - increasing to 32\n",
            "llama_new_context_with_model: n_ctx      = 16384\n",
            "llama_new_context_with_model: n_batch    = 32\n",
            "llama_new_context_with_model: n_ubatch   = 32\n",
            "llama_new_context_with_model: flash_attn = 0\n",
            "llama_new_context_with_model: freq_base  = 10000.0\n",
            "llama_new_context_with_model: freq_scale = 1\n",
            "llama_kv_cache_init:        CPU KV buffer size =  2048.00 MiB\n",
            "llama_new_context_with_model: KV self size  = 2048.00 MiB, K (f16): 1024.00 MiB, V (f16): 1024.00 MiB\n",
            "llama_new_context_with_model:        CPU  output buffer size =     0.12 MiB\n",
            "llama_new_context_with_model:        CPU compute buffer size =    68.00 MiB\n",
            "llama_new_context_with_model: graph nodes  = 1030\n",
            "llama_new_context_with_model: graph splits = 1\n",
            "AVX = 1 | AVX_VNNI = 0 | AVX2 = 1 | AVX512 = 0 | AVX512_VBMI = 0 | AVX512_VNNI = 0 | AVX512_BF16 = 0 | FMA = 1 | NEON = 0 | SVE = 0 | ARM_FMA = 0 | F16C = 1 | FP16_VA = 0 | WASM_SIMD = 0 | BLAS = 0 | SSE3 = 1 | SSSE3 = 1 | VSX = 0 | MATMUL_INT8 = 0 | LLAMAFILE = 1 | \n",
            "Model metadata: {'tokenizer.chat_template': \"{{ bos_token }}{% for message in messages %}{% if (message['role'] == 'user') != (loop.index0 % 2 == 0) %}{{ raise_exception('Conversation roles must alternate user/assistant/user/assistant/...') }}{% endif %}{% if message['role'] == 'user' %}{{ '[INST] ' + message['content'] + ' [/INST]' }}{% elif message['role'] == 'assistant' %}{{ message['content'] + eos_token + ' ' }}{% else %}{{ raise_exception('Only user and assistant roles are supported!') }}{% endif %}{% endfor %}\", 'tokenizer.ggml.unknown_token_id': '0', 'tokenizer.ggml.eos_token_id': '2', 'general.architecture': 'llama', 'llama.rope.freq_base': '10000.000000', 'llama.context_length': '32768', 'general.name': 'hub', 'llama.embedding_length': '4096', 'llama.feed_forward_length': '14336', 'llama.attention.layer_norm_rms_epsilon': '0.000010', 'llama.rope.dimension_count': '128', 'tokenizer.ggml.bos_token_id': '1', 'llama.attention.head_count': '32', 'llama.block_count': '32', 'llama.attention.head_count_kv': '8', 'general.quantization_version': '2', 'tokenizer.ggml.model': 'llama', 'general.file_type': '15'}\n",
            "Available chat formats from metadata: chat_template.default\n",
            "Guessed chat format: mistral-instruct\n"
          ]
        }
      ],
      "source": [
        "llm = LlamaCpp(\n",
        "    model_path= \"/content/drive/MyDrive/HIDS-7950/llm/BioMistral-7B.Q4_K_M.gguf\",\n",
        "    temperature=0.3,\n",
        "    max_tokens=2048,\n",
        "    n_ctx=16384,\n",
        "    top_p=1)"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "wlpKk32t80Cc"
      },
      "source": [
        "# Set Prompt"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 13,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "XfwZZjEh8zYo",
        "outputId": "9d284661-70b1-4dee-ee8e-af7f8ccf646c"
      },
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "PromptTemplate(input_variables=['question'], template='<<SYS>>\\n    You are a helpful assistant eager to assist with providing better search results.\\n    <</SYS>>\\n\\n    [INST] Provide an answer to the following question. Ensure that the answer is informative,             relevant, and concise:\\n            {question}\\n    [/INST]')"
            ]
          },
          "metadata": {},
          "execution_count": 13
        }
      ],
      "source": [
        "from langchain.chains import LLMChain\n",
        "from langchain.chains.prompt_selector import ConditionalPromptSelector\n",
        "from langchain.prompts import PromptTemplate\n",
        "\n",
        "DEFAULT_LLAMA_SEARCH_PROMPT = PromptTemplate(\n",
        "    input_variables=[\"question\"],\n",
        "    template=\"\"\"<<SYS>>\n",
        "    You are a helpful assistant eager to assist with providing better search results.\n",
        "    <</SYS>>\n",
        "\n",
        "    [INST] Provide an answer to the following question. Ensure that the answer is informative, \\\n",
        "            relevant, and concise:\n",
        "            {question}\n",
        "    [/INST]\"\"\",\n",
        ")\n",
        "\n",
        "DEFAULT_SEARCH_PROMPT = PromptTemplate(\n",
        "    input_variables=[\"question\"],\n",
        "    template=\"\"\"You are a helpful assistant eager to assist with providing better search results. \\\n",
        "        Provide an answer to the following question. Ensure that the answer is informative, \\\n",
        "        relevant, and concise: \\\n",
        "        {question}\"\"\",\n",
        ")\n",
        "\n",
        "QUESTION_PROMPT_SELECTOR = ConditionalPromptSelector(\n",
        "    default_prompt=DEFAULT_SEARCH_PROMPT,\n",
        "    conditionals=[(lambda llm: isinstance(llm, LlamaCpp), DEFAULT_LLAMA_SEARCH_PROMPT)],\n",
        ")\n",
        "\n",
        "prompt = QUESTION_PROMPT_SELECTOR.get_prompt(llm)\n",
        "prompt"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "ADSv5_wne8sj"
      },
      "source": [
        "# Test LLM"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "s-LepcEt9oGM",
        "outputId": "0c8be6fe-27da-4819-caba-5fb5a64a26ab"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.10/dist-packages/langchain_core/_api/deprecation.py:139: LangChainDeprecationWarning: The class `LLMChain` was deprecated in LangChain 0.1.17 and will be removed in 1.0. Use RunnableSequence, e.g., `prompt | llm` instead.\n",
            "  warn_deprecated(\n",
            "\n",
            "llama_print_timings:        load time =    3013.80 ms\n",
            "llama_print_timings:      sample time =      59.24 ms /    93 runs   (    0.64 ms per token,  1569.99 tokens per second)\n",
            "llama_print_timings: prompt eval time =   44381.35 ms /    83 tokens (  534.72 ms per token,     1.87 tokens per second)\n",
            "llama_print_timings:        eval time =   57974.08 ms /    92 runs   (  630.15 ms per token,     1.59 tokens per second)\n",
            "llama_print_timings:       total time =  102521.96 ms /   175 tokens\n"
          ]
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "{'question': 'What is Acute Myeloid Leukemia?',\n",
              " 'text': ' Acute myeloid leukemia (AML) is a type of cancer that affects the blood cells in the bone marrow . It is characterized by an overproduction of immature white blood cells, which accumulate in the bone marrow and interfere with the production of normal blood cells. The exact cause of AML is unknown, but it is thought to be related to genetic mutations that affect the development of the blood cells.'}"
            ]
          },
          "metadata": {},
          "execution_count": 16
        }
      ],
      "source": [
        "llm_chain = LLMChain(prompt=prompt, llm=llm)\n",
        "question = \"What is Acute Myeloid Leukemia?\"\n",
        "llm_chain.invoke({\"question\": question})"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "1Yd2rCvIirKU"
      },
      "source": [
        "# Retriever"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": 14,
      "metadata": {
        "id": "qOHmTJUPTWRq"
      },
      "outputs": [],
      "source": [
        "from langchain.retrievers import WeaviateHybridSearchRetriever\n",
        "\n",
        "# Configuration of the retriever\n",
        "retriever = WeaviateHybridSearchRetriever(\n",
        "    client=client,\n",
        "    index_name=\"Document\",\n",
        "    text_key=\"text_to_embed\",\n",
        "    attributes=[\n",
        "        \"study_id\", \"pdc_study_id\", \"study_submitter_id\", \"study_name\",\n",
        "        \"program_name\", \"project_name\", \"disease_type\", \"primary_site\",\n",
        "        \"analytical_fraction\", \"experiment_type\", \"cases_count\",\n",
        "        \"aliquots_count\", \"pubmed_id\", \"doi\", \"author\", \"title\",\n",
        "        \"journal\", \"journal_url\", \"year\", \"abstract\", \"text_to_embed\"\n",
        "    ],\n",
        "    create_schema_if_missing=True,\n",
        ")\n"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "# RAG chain"
      ],
      "metadata": {
        "id": "asQqdzJTTsWB"
      }
    },
    {
      "cell_type": "code",
      "execution_count": 15,
      "metadata": {
        "id": "zgsUQ4jvfpBP"
      },
      "outputs": [],
      "source": [
        "from langchain.chains import RetrievalQA\n",
        "qa = RetrievalQA.from_chain_type(\n",
        "    llm=llm,\n",
        "    chain_type=\"stuff\",\n",
        "    retriever=retriever,\n",
        "    verbose=True\n",
        ")"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "YXlyTZncf7Nh",
        "outputId": "1a196752-4ab5-4579-a324-01a1dae92f35"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\n",
            "\n",
            "\u001b[1m> Entering new RetrievalQA chain...\u001b[0m\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Llama.generate: prefix-match hit\n",
            "\n",
            "llama_print_timings:        load time =    3013.80 ms\n",
            "llama_print_timings:      sample time =      18.43 ms /    35 runs   (    0.53 ms per token,  1899.59 tokens per second)\n",
            "llama_print_timings: prompt eval time =  823218.81 ms /  1872 tokens (  439.75 ms per token,     2.27 tokens per second)\n",
            "llama_print_timings:        eval time =   24499.30 ms /    34 runs   (  720.57 ms per token,     1.39 tokens per second)\n",
            "llama_print_timings:       total time =  847858.43 ms /  1906 tokens\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\n",
            "\u001b[1m> Finished chain.\u001b[0m\n"
          ]
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "{'query': 'What is the disease_type of PDC000220 ?',\n",
              " 'result': ' The disease type for PDC000220 is Lung Adenocarcinoma;Lung Squamous Cell Carcinoma;Other.'}"
            ]
          },
          "metadata": {},
          "execution_count": 20
        }
      ],
      "source": [
        "query = \"What is the disease_type of PDC000220?\"\n",
        "qa.invoke(query)\n",
        "#take 14 mins and correct"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "query1 = \"Do you have any kidney cancer studies in the PDC database?\"\n",
        "qa.invoke(query1)\n",
        "#take 25 mins"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "ek46Gsd5fsqh",
        "outputId": "6b9a6bb3-bf41-488d-e4f5-59d0ea4c0dfc"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\n",
            "\n",
            "\u001b[1m> Entering new RetrievalQA chain...\u001b[0m\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Llama.generate: prefix-match hit\n",
            "\n",
            "llama_print_timings:        load time =    3013.80 ms\n",
            "llama_print_timings:      sample time =      85.04 ms /   146 runs   (    0.58 ms per token,  1716.80 tokens per second)\n",
            "llama_print_timings: prompt eval time = 1395695.52 ms /  3099 tokens (  450.37 ms per token,     2.22 tokens per second)\n",
            "llama_print_timings:        eval time =  112205.57 ms /   145 runs   (  773.83 ms per token,     1.29 tokens per second)\n",
            "llama_print_timings:       total time = 1508340.73 ms /  3244 tokens\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\n",
            "\u001b[1m> Finished chain.\u001b[0m\n"
          ]
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "{'query': 'Do you have any kidney cancer studies in the PDC database?',\n",
              " 'result': ' The PDC does not currently have any kidney cancer studies in the database. However, there are several ongoing studies that will be added to the database as soon as the data is made public. In the meantime, there are a number of other resources available for researchers interested in studying kidney cancer using proteomics data. One example is The Cancer Proteome Atlas (TCPA), which provides large-scale proteomic datasets from human cancers including kidney cancer. Another resource is the Clinical Proteomic Tumor Analysis Consortium (CPTAC), which has published several papers on kidney cancer using proteomics data. These resources may be helpful in designing future studies or analyzing existing data.'}"
            ]
          },
          "metadata": {},
          "execution_count": 28
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "query2 = \"Name studies in the PDC that focus on Metabolomics.\"\n",
        "qa.invoke(query2)\n",
        "#take 25 mins"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "JnvYOIv2fspB",
        "outputId": "09648127-5733-4a70-b84f-0a61673addb3"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\n",
            "\n",
            "\u001b[1m> Entering new RetrievalQA chain...\u001b[0m\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Llama.generate: prefix-match hit\n",
            "\n",
            "llama_print_timings:        load time =    3013.80 ms\n",
            "llama_print_timings:      sample time =      12.08 ms /    20 runs   (    0.60 ms per token,  1656.31 tokens per second)\n",
            "llama_print_timings: prompt eval time = 1434714.44 ms /  3040 tokens (  471.95 ms per token,     2.12 tokens per second)\n",
            "llama_print_timings:        eval time =   15787.77 ms /    20 runs   (  789.39 ms per token,     1.27 tokens per second)\n",
            "llama_print_timings:       total time = 1450595.12 ms /  3060 tokens\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\n",
            "\u001b[1m> Finished chain.\u001b[0m\n"
          ]
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "{'query': 'Name studies in the PDC that focus on Metabolomics.',\n",
              " 'result': ' The following are some examples of studies in the PDC that focus on metabolomics:'}"
            ]
          },
          "metadata": {},
          "execution_count": 33
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "query3 = \"What is the experiment_type of PDC000403?\"\n",
        "qa.invoke(query3)\n",
        "#take 27 mins and correct"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "VBHDAgOPy7hP",
        "outputId": "5787116d-5df8-4d9d-fd53-aa658ea7cb8a"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\n",
            "\n",
            "\u001b[1m> Entering new RetrievalQA chain...\u001b[0m\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Llama.generate: prefix-match hit\n",
            "\n",
            "llama_print_timings:        load time =    3013.80 ms\n",
            "llama_print_timings:      sample time =      10.46 ms /    19 runs   (    0.55 ms per token,  1817.31 tokens per second)\n",
            "llama_print_timings: prompt eval time = 1630997.24 ms /  3408 tokens (  478.58 ms per token,     2.09 tokens per second)\n",
            "llama_print_timings:        eval time =   15647.07 ms /    19 runs   (  823.53 ms per token,     1.21 tokens per second)\n",
            "llama_print_timings:       total time = 1646741.76 ms /  3427 tokens\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\n",
            "\u001b[1m> Finished chain.\u001b[0m\n"
          ]
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "{'query': 'What is the experiment_type of PDC000403 ?',\n",
              " 'result': ' The experiment_type of PDC000403 is TMT11'}"
            ]
          },
          "metadata": {},
          "execution_count": 39
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "query4 = \"How many studies are related to leukemia?\"\n",
        "qa.invoke(query4)\n",
        "#take 27 mins"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "RAXisbTs6GUL",
        "outputId": "7163d2e8-810f-437f-87bf-aa3d2993d82a"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\n",
            "\n",
            "\u001b[1m> Entering new RetrievalQA chain...\u001b[0m\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "\n",
            "llama_print_timings:        load time =    3516.18 ms\n",
            "llama_print_timings:      sample time =       2.70 ms /     4 runs   (    0.68 ms per token,  1481.48 tokens per second)\n",
            "llama_print_timings: prompt eval time = 2035386.64 ms /  3835 tokens (  530.74 ms per token,     1.88 tokens per second)\n",
            "llama_print_timings:        eval time =    2541.64 ms /     3 runs   (  847.21 ms per token,     1.18 tokens per second)\n",
            "llama_print_timings:       total time = 2038411.88 ms /  3838 tokens\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\n",
            "\u001b[1m> Finished chain.\u001b[0m\n"
          ]
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "{'query': 'How many studies are related to leukemia?', 'result': ' 21'}"
            ]
          },
          "metadata": {},
          "execution_count": 18
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "query5 = \"How many cases and aliquots are there in study PDC000303?\"\n",
        "qa.invoke(query5)\n",
        "# correct"
      ],
      "metadata": {
        "id": "a-xTrLr06GS1",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "37988a2d-f0fb-45f9-9438-37a00021683f"
      },
      "execution_count": null,
      "outputs": [
        {
          "metadata": {
            "tags": null
          },
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "\n",
            "\n",
            "\u001b[1m> Entering new RetrievalQA chain...\u001b[0m\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "\n",
            "llama_print_timings:        load time =   37129.41 ms\n",
            "llama_print_timings:      sample time =      20.73 ms /    25 runs   (    0.83 ms per token,  1206.16 tokens per second)\n",
            "llama_print_timings: prompt eval time = 1875669.23 ms /  3126 tokens (  600.02 ms per token,     1.67 tokens per second)\n",
            "llama_print_timings:        eval time =   23405.78 ms /    24 runs   (  975.24 ms per token,     1.03 tokens per second)\n",
            "llama_print_timings:       total time = 1899574.14 ms /  3150 tokens\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\n",
            "\u001b[1m> Finished chain.\u001b[0m\n"
          ]
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "{'query': 'How many cases and aliquots are there in study PDC000303?',\n",
              " 'result': ' There are 27 cases and 27 aliquots in study PDC000303.'}"
            ]
          },
          "metadata": {},
          "execution_count": 24
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [],
      "metadata": {
        "id": "axF6NXlbGhus"
      },
      "execution_count": null,
      "outputs": []
    }
  ]
}